<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-15 09:29:40 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>56</td>
          <td>44</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The development of drug-resistant cell lines is essential for understanding the mechanisms of drug resistance and identifying strategies to overcome treatment failure in cancer therapy. Resistance models enable preclinical evaluation of novel compounds, repurposed drugs, and combination therapies. To generate resistant cells, parental cancer cell lines are repeatedly exposed to incrementally increasing concentrations of the target drug over several weeks. Cells that survive and proliferate at each stage are selected, expanded, and exposed to higher drug doses. The development of resistance is confirmed by quantifying and comparing the half-maximal inhibitory concentration (IC50) values between parental and resistant cells using cell viability assays and nonlinear regression analysis. Significantly increased IC50 values indicate successful adaptation to drug pressure and the development of resistance. These drug-resistant cell lines are available for comprehensive analysis, such as microarray and single-cell sequencing, as well as various in vitro or in vivo experiments. These models provide valuable tools for investigating potential therapeutic strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb7439fafb8c53daf4f72d4753cdd3f7c5b1b9f" target='_blank'>
              Development of Drug-resistant Cell Lines for Experimental Procedures.
              </a>
            </td>
          <td>
            S. Kadomoto, G. Shelley, Atsushi Mizokami, Evan T. Keller
          </td>
          <td>2025-08-12</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="KRAS is mutationally activated in 45–50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRASG12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, J. Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species.

 Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures.

 Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign.

 Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1223372b66c898bfc66493d1d8cf7781f04f51" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Leigh Ellis, Beatriz German, Katherine L. Morel, Teia Noel, Nadia Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qiu, H. Long, Stefan DiFazio, S. Baca, Ayesha Shafi, Matthew L Freedman, Himisha Beltran, Christopher J Sweeney, H. He, Myles Brown, Jasmine T Plummer, Simon Knott, David Labbé
          </td>
          <td>2025-08-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a genetically and clinically heterogeneous malignancy marked by poor prognosis and limited therapeutic options, especially in older patients. While conventional treatments such as the “7 + 3” chemotherapy regimen and allogeneic stem cell transplantation remain standard care options, the advent of next-generation sequencing (NGS) has transformed our understanding of AML’s molecular complexity. Among the emerging hallmarks of AML, metabolic reprogramming has gained increasing attention for its role in supporting leukemic cell proliferation, survival, and therapy resistance. Distinct AML subtypes—shaped by specific genetic alterations, including FLT3, NPM1, and IDH mutations—exhibit unique metabolic phenotypes that reflect their underlying molecular landscapes. Notably, FLT3-ITD mutations are associated with enhanced reactive oxygen species (ROS) production and altered energy metabolism, contributing to disease aggressiveness and poor clinical outcomes. This review highlights the interplay between metabolic plasticity and genetic heterogeneity in AML, with a particular focus on FLT3-driven metabolic rewiring. We discuss recent insights into how these metabolic dependencies may be exploited therapeutically, offering a rationale for the development of metabolism-targeted strategies in the treatment of FLT3-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdd39e68513bba88417b983cb0b3114bb567964" target='_blank'>
              Metabolic Signature of FLT3-Mutated AML: Clinical and Therapeutic Implications
              </a>
            </td>
          <td>
            C. Banella, Gianfranco Catalano, Maura Calvani, Eleonora Candi, N. Noguera, S. Travaglini
          </td>
          <td>2025-09-08</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Cancer genomic profiling has revolutionized cancer research and clinical practice by facilitating personalized medicine approaches. Next-generation sequencing technologies have played a pivotal role in expanding the possibilities of cancer genomics, providing a comprehensive view of the genomic landscape of tumors and identifying potential therapeutic targets. In Japan, the approval of several cancer-related gene panel tests has accelerated the implementation of cancer genomic medicine. However, there remains a need for evidence-based selection criteria for the different panel tests available. Among these tests, ACTOnco+® stands out as a comprehensive cancer genomic profiling tool. In this study, we compared 110 samples from 29 different tumor types using FoundationOne® CDx (324 genes for DNA analysis) and ACTOnco+® (440 genes for DNA and 31 genes for RNA analysis). Overall, the mutation profiles between the 2 assays exhibited 82.8% positive agreement in reported sequence alterations in clinically actionable genes, including single nucleotide variants and insertions–deletions. Copy number gains showed concordance of 76.9%, and copy number losses in 66.7%. In the case of KIAA1549-BRAF fusion-positive astrocytoma, the fusion event was not detected by DNA only test, but RNA sequencing identified the rearrangement. The patient exhibited clinical benefit from MEK inhibitor treatment. Tumor mutational burden and microsatellite instability (MSI) demonstrated high concordance across various cancer types. ACTOnco+® identified a total of 329 heterozygous deletions, which were subsequently validated for reliability using FISH analysis. Gene profiling with ACTOnco+® exhibited comparable results to FoundationOne® CDx, thereby contributing to future personalized medicine endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584447fccf78026ffcd29a7adc016edfe5f11d6e" target='_blank'>
              Characterization of a novel comprehensive genomic profiling test with better detection of heterozygous deletions and RNA-based gene fusion analysis
              </a>
            </td>
          <td>
            Ryouta Kakuta, Akira Takemoto, Kousuke Tanimoto, Rika Noji, Ryo Kudo, Yoshihito Kano, Yayoi Ando, Tatsunori Simoi, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-08-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background. Cancer drug resistance represents an obvious problem in clinical oncology. Among various intracellular pathways activated during drug resistance development, proinflammatory cascades considered to be involved as well. The inflammatory process may also play a key role in the formation of melanoma resistance to the chemotherapeutic agent dacarbazine. The aim. To characterize differentially expressed genes and signaling pathways associated with cancer resistance to alkylating agent dacarbazine in melanoma cells in vivo with the use of high-throughput sequencing for transcriptomic profiling. Methods. The effect of dacarbazine on melanoma cell gene expression was studied in a C57Bl6/B16 melanoma model in vivo. Mice were injected intraperitoneally with dacarbazine (50 mg/kg) on days 8, 10, and 12 after melanoma tumor transplantation. Total RNA was extracted from tumor nodes on the 14th day after melanoma transplantation to animals and the transcriptome was analyzed using a next generation sequencing method. Bioinformatic analysis was applied to identify differentially expressed genes and corresponding signal pathways. Results. Twenty one differentially expressed genes were identified, of which an increase in expression was observed in 10 genes, and a decrease in expression activity was observed in 11 genes involved in signaling pathways associated with the inflammatory process “Comprehensive IL-17A signaling”, “Oxidative stress and redox pathway”, “TNF alpha NF-kB signaling pathway”. Conclusions. Dacarbazine alters the expression of genes regulating inflammation, which may play a role in the development of chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf6cf90e3f143305c6185b55f86bc396a3a3d9" target='_blank'>
              Inflammation-associated signal pathways activation during drug resistance development in cancer cells
              </a>
            </td>
          <td>
            E. Z. Lapkina, I. S. Zinchenko, Е. I. Bondar, E. Sergeeva, T. Ruksha
          </td>
          <td>2025-07-22</td>
          <td>Acta Biomedica Scientifica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Nucleoside analogues are crucial in treating non-small cell lung cancer (NSCLC), but resistance hampers patient outcomes. The cytidine analogue RX-3117 shows promise in gemcitabine-resistant cancers, yet mechanisms underlying acquired resistance to this drug remain unexplored. This study includes a comprehensive investigation into RX-3117 resistance mechanisms by leveraging new preclinical models and cutting-edge genomic tools, including a CRISPR-Cas9 knockout screen and transcriptomics. Methods NSCLC cell lines A549 and SW1573 were exposed to stepwise increasing concentrations of RX-3117 to establish stable resistant subclones, confirmed by SRB and clonogenic assays. Intracellular RX-3117 nucleotide levels were measured via LC/MS-MS, prompting the evaluation and modulation of the expression of key metabolic enzymes by Western blot and siRNA. A CRISPR-Cas9 screen identified genes whose loss increased RX-3117 sensitivity, while RNA-sequencing with differential expression analyses revealed resistance-related pathways, further investigated through cell cycle distribution, knock-out, and ELISA assays. Results Resistant clones exhibited decreased accumulation of RX-3117 nucleotides, which however, was not associated to reduced expression of activation enzymes (UCK2, UMPK, CMPK, NME1/NDPK, RR1 and RR2). Instead, increased expression was observed in certain DNA repair and deactivation enzymes (NT5C3) but pharmacological inhibition and silencing of the latter did not circumvent resistance. Remarkably, a comprehensive approach with CRISPR-Cas9 screen highlighted DNA-repair and cell cycle determinants as key sensitizing genes. XL-PCR and RNA-sequencing confirmed aberrations in DNA-repair and pathways involved in cell cycle regulation. Knock-out and pharmacological inhibition validated the role of PKMYT1, a protein kinase involved in G2/M transition and genomic stability. RX-3117-resistant A549 cells showed enhanced sensitivity to the PKMYT1 inhibitor lunresertib and its synergism with RX-3117, suggesting further studies, especially in patients with high PKMYT1 expression who have significantly shorter survival rates, as observed in public databases and validated in an internal cohort of NSCLC patients. Conclusion By integrating CRISPR-Cas9 with functional assays and transcriptomics, our study established a framework for decoding resistance mechanisms and highlights potential therapeutic strategies to enhance RX-3117 efficacy in NSCLC. We demonstrated for the first time that aberrant DNA repair and cell cycle dysregulation led resistance, identifying PKMYT1 as a promising target. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03470-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a070b145fa4db64737d5a930cbf0a0bb16c66539" target='_blank'>
              Unraveling resistance mechanisms to the novel nucleoside analog RX-3117 in lung cancer: insights into DNA repair, cell cycle dysregulation and targeting PKMYT1 for improved therapy
              </a>
            </td>
          <td>
            M. Vahabi, Geng Xu, Dzjemma Sarkisjan, B. E. Hassouni, G. Mantini, Valentina Donati, Bing Wang, Giulia Lencioni, R. Honeywell, Dongmei Deng, S. Strano, Godefridus J. Peters, G. Blandino, E. Giovannetti
          </td>
          <td>2025-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) is a key biological process driving tumor invasion and metastasis, and its activation is widely regarded as one of the core mechanisms leading to tumor resistance to multiple therapeutic approaches. Current research indicates that EMT mediates drug resistance through multiple pathways, such as conferring stem cell-like properties to cancer cells, miRNA delivery via exosomes, and promoting drug efflux. The underlying mechanisms of related signaling pathways and key transcription factors are being intensively investigated. However, effectively translating the basic research findings targeting EMT to overcome drug resistance into clinical applications still faces significant challenges. This article systematically reviews the core molecular mechanisms by which EMT mediates tumor drug resistance, with a focus on how EMT confers drug-resistant phenotypes to cancer cells by regulating tumor stem cell characteristics and drug pharmacokinetics. It also reviews experimental strategies targeting key EMT molecules to reverse drug resistance and their preclinical/clinical research evidence. This review provides a theoretical framework for a deeper understanding of the role of EMT in tumor drug resistance and lays the foundation for developing novel combination treatment regimens based on EMT intervention. Future research urgently needs to explore more selective EMT-targeting strategies, identify reliable predictive biomarkers, and optimize their synergistic effects with traditional radiotherapy, chemotherapy, and immunotherapy to ultimately improve the prognosis of patients with refractory tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8627c99bba513b22e7affef08f860cf5a9870601" target='_blank'>
              Research on Tumor Drug Resistance Mediated by Epithelial-mesenchymal Transition
              </a>
            </td>
          <td>
            You Wu
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Although precision-targeted therapies and tyrosine kinase inhibitors (TKIs) have significantly improved outcomes in non-small-cell lung cancer (NSCLC), patients with EGFR-mutant NSCLC with concurrent TP53 mutations often develop drug resistance and experience poor clinical outcomes. This study aims to investigate the molecular mechanisms underlying this aggressive subtype using single-cell RNA sequencing. Methods Formalin-fixed paraffin-embedded (FFPE) tumor samples were obtained from 40 hospitalized NSCLC patients. Somatic mutation profiles were determined using a targeted 23-gene next-generation sequencing (NGS) panel. Four samples harboring concurrent EGFR and TP53 mutations were selected for single-cell transcriptomic profiling using the 10x Genomics platform. Results Two dominant malignant epithelial cell populations were identified: C1_EGFR+, associated with proliferation and invasion, and C2_STAT1+, linked to immunosuppression and drug resistance. These tumor subtypes cooperatively drive CD8+ T cell exhaustion through the MDK–(ITGA4+ITGB1), MIF–(CD74+CXCR4), and TGF-β signaling pathways. In addition, antigen-presenting cancer-associated fibroblasts (apCAFs) recruit regulatory T cells via the CCL5–CCR4 axis, collectively establishing an immune-excluded tumor microenvironment. Mechanistically, a STAT1/ETS1-centered transcriptional program regulates the expression of key immunosuppressive (e.g., MDK, MIF, TGFB1) and resistance-associated genes (e.g., ERBB2, JAK2). Conclusion These findings reveal a coordinated transcriptional network that promotes immune evasion and therapeutic resistance in EGFR/TP53 co-mutated NSCLC. Targeting the STAT1/ETS1 axis, in combination with EGFR-TKIs or immune checkpoint inhibitors, may provide a novel strategy to overcome resistance and improve patient outcomes. Further validation in larger patient cohorts and functional studies is warranted to confirm these observations and support clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dfb7de9c158ecd7f633c8ed6b345d4fa18555ab" target='_blank'>
              Dissecting the immune evasion and therapeutic resistance mechanisms in EGFR/TP53 co-mutated non-small cell lung cancer: implications for targeted and immunotherapy strategies
              </a>
            </td>
          <td>
            Haiyan Shi, Kun Xu, Xueying Kong, Weining Xie, Yingying Chen, Ding He, Zufu Cheng, Xianshan Huo, Ke Gao, Mingshuang Song, Ning Tian
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance and distant metastases are leading causes of mortality in colorectal cancer (CRC), yet the molecular mechanisms linking these processes remain elusive. In this study, we demonstrate that acquired resistance to oxaliplatin, a first-line chemotherapeutic in CRC, enhances metastatic potential through transcriptional reprogramming. Using a clinically relevant dosing regimen, we generated oxaliplatin-resistant CRC cells that displayed increased metastatic potential. Integrated transcriptomic and phenotypic analyses revealed that dysregulated cholesterol biogenesis amplifies TGF-β signaling, which in turn drives expression of SERPINE1, which serves as a key effector of both oxaliplatin resistance and metastasis. Furthermore, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that robustly predicts overall and relapse-free survival across distinct patient cohorts. Notably, RESIST-M stratifies a high-risk subtype of CMS4/iCMS3-fibrotic patients that display the poorest prognosis, underscoring its clinical relevance. Targeting of SERPINE1 or cholesterol biosynthesis re-sensitized resistant, pro-metastatic cells to oxaliplatin in mouse xenograft models. Altogether, this study uncovers a mechanistic link between metabolic rewiring and transcriptional plasticity underlying therapy-induced metastasis in primary CRC. Additionally, it also reveals actionable vulnerabilities that offer both prognostic value and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461233a4140dd31d355e46865c3174f542e2818f" target='_blank'>
              Modeling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Kenzom Tenzin, Niranjan Shirgaonkar, Huiwen Chua, Lin Xuan Chee, Ahpa Sae Yeoh, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, Ethan Subel, M. Thangavelu, Jane Vin Chan, Choon-Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj Dasgupta
          </td>
          <td>2025-07-16</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer continues to be the primary cause of cancer-related deaths among women globally, with increased rates of incidence and mortality, highlighting the critical need for effective treatment strategies. Recent developments have introduced a variety of treatment options that address the molecular diversity of breast cancer; nonetheless, drug resistance remains a significant barrier to achieving favorable results. This review explains the crucial role of genetic and epigenetic changes in contributing to therapeutic resistance, in addition to other factors such as increased drug efflux, enhanced DNA repair, evasion of senescence, tumor heterogeneity, the tumor microenvironment (TME), and epithelial-to-mesenchymal transition (EMT). Genetic modifications, including mutations in oncogenes and tumor suppressor genes, disrupt essential signaling pathways, facilitating resistance to chemotherapy and targeted therapies. At the same time, epigenetic modifications - like DNA methylation, alterations to histones, and dysregulation of non-coding RNAs - reprogram gene expression, supporting adaptive resistance mechanisms. These molecular abnormalities contribute to the plasticity of tumors, allowing cancer cells to evade therapeutic approaches. This review consolidates recent discoveries regarding how these genetic and epigenetic modifications affect treatment responses and resistance in breast cancer, highlighting their interaction with disease advancement. By pinpointing new drug targets, including immunotherapeutic strategies, this article seeks to shed light on the molecular underpinnings of chemoresistance, aiding in the refinement of existing treatment protocols. A more profound understanding of these mechanisms offers the potential for developing precision therapies to overcome resistance, reduce relapse rates, and improve clinical outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f843678da290bd94b0c5bd14d4941f6b50d5f9a" target='_blank'>
              Decoding breast cancer treatment resistance through genetic, epigenetic, and immune-regulatory mechanisms: from molecular insights to translational perspectives
              </a>
            </td>
          <td>
            Suryendu Saha, Samikshya Mahapatra, Sinjan Khanra, Barnalee Mishra, Biswajit Swain, Diksha Malhotra, Swarnali Saha, Venketesh K Panda, Kavita Kumari, Sarmistha Jena, Sandeep Thakur, Pawan K. Singh, Gopal C Kundu
          </td>
          <td>2025-07-21</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="KRAS mutations are among the most prevalent oncogenic drivers in non-small cell lung cancer (NSCLC), yet the mechanisms of therapeutic resistance to KRAS inhibitors in these cancers remains poorly understood. Here, we deploy high-throughput CRISPR base editing screens to systematically map resistance mutations to three mechanistically distinct KRAS-targeted therapies, including KRAS-G12C(OFF) inhibitor (adagrasib), RAS(ON) G12C-selective tri-complex inhibitor (RMC-4998), and RAS(ON) multi-selective tri-complex inhibitor (RMC-7977). Using both a saturation Kras tiling approach and cancer-associated mutation library, we identify common and compound-selective second-site resistance mutations in Kras, as well as gain-of-function and loss-of-function variants across cancer-associated genes that rewire signaling networks in a context-dependent manner. Notably, we identify a recurrent missense mutation in capicua (Cic), that promotes resistance to RMC-7977 in vitro and in vivo. Moreover, we show that targeting NFκB signaling in CIC-mutant cells can resensitize them to RAS pathway inhibition and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597a0301e9cbbccb635fb6b4893ce93089ff3713" target='_blank'>
              Genetic mechanisms of resistance to targeted KRAS inhibition
              </a>
            </td>
          <td>
            Bianca J. Diaz, Max Kops, Sara Bernardo, Henri Schmidt, Elizabeth Grankowsky, Adrian Vega, Chen Zhang, Matthew Bott, Maria Skamagki, Aidan C A Tomlinson, Nicole A Vita, Alyna Katti, Mark Labrecque, Ida Aronchik, Mallika Singh, Lukas E. Dow
          </td>
          <td>2025-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51250a57c51180b7973c2f5e6b3bbb56fc3acbc6" target='_blank'>
              Personalizing cancer therapy: the role of pharmacogenetics in overcoming drug resistance and toxicity.
              </a>
            </td>
          <td>
            Hossein Mardani -Nafchi, Hormoz Solatzadeh, Elnaz Hajimaghsoudi, Emad Babakhanzadeh
          </td>
          <td>2025-08-02</td>
          <td>Molecular biology reports</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer in which disease initiation and relapse are driven by leukemic cells with stem-like properties, known as leukemic stem cells (LSCs). The LSC compartment is highly heterogenous and this contributes to differences in therapy response. This heterogeneity is determined by genetic and nongenetic factors including somatic mutations, the cell of origin, transcriptional and epigenetic states as well as phenotypic plasticity. While this complicates the identification and eradication of LSCs, it also presents an opportunity to tailor therapeutic strategies to the phenotypic and functional states of LSCs present in a patient, exploiting their specific vulnerabilities. The emergence of single-cell multiomics technologies has transformed our ability to dissect cellular heterogeneity in AML, enabling simultaneous interrogation of genomic, transcriptomic, epigenomic and proteomic layers and providing high-resolution molecular snapshots of individual cells. In this review, we discuss causes and consequences of LSC heterogeneity, highlight advances in single-cell multiomics technologies to resolve it and outline how they can address shortcomings in our understanding of LSC heterogeneity and plasticity to revolutionize diagnostics and disease monitoring of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07654c74db7467e7588089114a2ee5e54a7d1bf9" target='_blank'>
              Resolving leukemic stem cell heterogeneity and plasticity with single-cell multiomics.
              </a>
            </td>
          <td>
            Frank Y Huang, Andreas Trumpp, Patrick Stelmach
          </td>
          <td>2025-08-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="With advancements in technology, single-cell RNA sequencing has emerged as a power-ful tool in cancer drug discovery. This technique enables the construction of gene expression profiles at the individual cell level, offering detailed insights into cellular heterogeneity and molecular path-ways involved in tumor development. It enables researchers to gain a deeper understanding of tumor heterogeneity. Researchers can study cell subpopulations and gene expression patterns. This under-standing helps in identifying potential drug targets. Additionally, it aids in predicting therapeutic responses. This high-resolution gene expression analysis provides a new perspective and oppor-tunity for cancer drug development, which is expected to accelerate the discovery and development process of new anti-cancer drugs. This article provides a comprehensive overview of the basic pro-cesses and developmental trajectory of single-cell RNA sequencing technology, with a particular emphasis on its applications in various aspects of cancer drug discovery. It also addresses the chal-lenges faced by single-cell RNA sequencing and potential future directions. This review aims to enhance readers' understanding of single-cell sequencing, inspire new ideas for oncology drug de-velopment, and support the translation of clinical research into practice, ultimately enabling physi-cians to design more precise and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f74b7c7821b0c57a9649d29962bfb630eb54595" target='_blank'>
              Single-Cell Transcriptomics: Technical Advances, Applications and Challenges in Cancer Drug Discovery.
              </a>
            </td>
          <td>
            Yueying Yang, Lingyu Meng, Teng Zhang, Jianjun Tan
          </td>
          <td>2025-07-30</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The addition of CDK4/6 inhibitors to endocrine therapy has significantly improved outcomes in HR+/HER2- breast cancer. However, variable patient responses and acquired resistance remain a clinical challenge. We therefore defined the comprehensive molecular response to palbociclib, the most clinically used CDK4/6 inhibitor. Global analyses of gene expression, protein abundance, splicing, and chromatin accessibility revealed broad patterns and specific changes that result from CDK4/6-inhibition in breast cancer cells. We uncovered unexpected feedback between CDK4/6 and estrogen-response signaling, which has clear clinical implications. We also revealed a widespread alternative splicing program that partially overlapped with genes whose expression is regulated, and which is expected to impact protein function. These molecular changes nominated combination therapies that interfere with the activation of CDKs or ERα. Accordingly, co-targeting CDK7, which regulates CDK2, CDK4/6 and ERα, additively impacted cell fitness. Collectively, these data reveal a complex, multi-tiered response to CDK4/6 inhibition, with implications for therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceb1151f3964ee516b8b4bec2c0e19c51711b54" target='_blank'>
              Charting the Multi-level Molecular Response to Palbociclib in ER-Positive Breast Cancer
              </a>
            </td>
          <td>
            Archishma Kavalipati, Amy Aponte, Michael E Sullivan, Sarah L Whittington, José C Martínez, Grant A. Goda, Maria M. Aleman, Michael J Emanuele, Daniel Dominguez
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Melanoma, the deadliest human skin cancer, frequently harbors activating BRAF mutations, with V600E being the most prevalent. These alterations drive constitutive activation of the MAPK pathway, promoting uncontrolled cell proliferation, survival, and dissemination. The advent of BRAFi and MEKi has significantly improved outcomes in BRAF V600-mutant melanoma. However, therapeutic resistance remains a major clinical barrier. Methods: This review integrates recent findings from preclinical and clinical studies to delineate resistance mechanisms to BRAF-targeted therapy. It categorizes resistance into primary (intrinsic), adaptive, and acquired forms, and analyzes their molecular underpinnings, including genetic and epigenetic alterations, pathway reactivation, and microenvironmental interactions. Results: Primary resistance is linked to pre-existing genetic and epigenetic changes that activate alternative signaling pathways, such as PI3K-AKT. Adaptive and acquired resistance includes secondary BRAF mutations, pathway redundancy, phenotype switching, and immune and stromal interactions. High-throughput sequencing has revealed novel mutations, including NRAS, NF1, and PTEN alterations, that contribute to resistance. Discussion: Understanding the multifaceted nature of resistance is critical to improving outcomes in advanced melanoma. This review highlights emerging strategies to overcome resistance, including combinatorial therapies, metabolic targeting, and biomarker-driven approaches, aiming to inform future therapeutic development and precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fe52d09f6a6cd013c45f83f652a44de19330f7" target='_blank'>
              Molecular Basis of BRAF Inhibitor Resistance in Melanoma: A Systematic Review
              </a>
            </td>
          <td>
            Ilaria Cosci, V. Salizzato, P. Del Fiore, J. Pigozzo, Valentina Guarneri, Simone Mocellin, A. Ferlin, Sara Mathlouthi, L. Piccin, Mariangela Garofalo
          </td>
          <td>2025-08-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a9aba01e0cb2e8c3213b471e7d8f34414b59cc" target='_blank'>
              Functional phenotyping of genomic variants using joint multiomic single-cell DNA-RNA sequencing.
              </a>
            </td>
          <td>
            Dominik Lindenhofer, Julia R. Bauman, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Mikael Marttinen, M. Kueblbeck, Judith B. Zaugg, K. Noh, Sascha Dietrich, W. Huber, O. Stegle, Lars M. Steinmetz
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, , Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Cancer metabolic reprogramming is a fundamental hallmark that enables tumor cells to sustain their malignant behaviors. Beyond its role in supporting growth, invasion, and migration, metabolic rewiring actively contributes to anticancer drug resistance. Cancer cells not only reshape their own metabolism but also engage in aberrant metabolic crosstalk with nonmalignant components within the tumor microenvironment (TME). These metabolic alterations create multiple barriers to the efficacy of drug therapies, including chemotherapy, targeted therapy, and immunotherapy. Despite growing evidence, an integrated understanding of how metabolic reprogramming contributes to the development of drug resistance and how it may be therapeutically targeted to overcome the resistance remains incomplete. This review summarizes recent progresses in tumor‐intrinsic and TME‐associated metabolic alterations that contribute to drug resistance by sustaining metabolic needs and modulating nonmetabolic processes and explores the upstream regulatory mechanisms driving these changes, focusing particularly on glucose, lipid, and amino acid metabolism. We also discuss the current advances in the integration of small molecule inhibitors targeting cancer metabolism to address drug resistance. By consolidating mechanistic insights and therapeutic opportunities, this review highlights metabolic reprogramming as a promising intervention point to overcome anticancer drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779e78bcb053e66126c8246e6269dc9bb736c67" target='_blank'>
              Metabolic Reprogramming: A Crucial Contributor to Anticancer Drug Resistance
              </a>
            </td>
          <td>
            Yunhan Zhu, Weijie Yan, Lingfeng Tong, Jie Yang, Shengfang Ge, Jiayan Fan, Renbing Jia, X. Wen
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id=", Gastric cancer (GC), the fourth leading cause of global cancer mortality, remains clinically problematic despite advances in multimodal treatments including surgery, chemotherapy, and immunotherapy. The consistently poor outcome in advanced stages needs new treatment approaches. Cancer-associated fibroblasts (CAFs) have emerged as important regulators of gastric tumour growth by dynamic interactions inside the tumour microenvironment (TME). These heterogeneous stromal elements, which are produced from several precursors, show continuous phenotypic evolution during carcinogenesis. CAFs mediate tumour-stroma crosstalk by extracellular matrix remodelling, paracrine signalling via growth factors/cytokines, and extracellular vesicle communication. Notably, CAFs and malignant cells show therapeutic pressure-driven co-evolution, and CAF-derived IL-6/IL-8 activate STAT3/NF-κB pathways to promote metabolic reprogramming while also creating drug-resistant ECM barriers. We aim to provide a transformative framework for understanding CAFs biology in gastric oncology, analysing molecular mechanisms of invasion, explaining tumour-CAF co-evolution dynamics, and mapping resistance-related regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3e328f639dd3ec99f2bec01c4994942b99494c" target='_blank'>
              Decoding the Plasticity of Cancer‐Associated Fibroblasts: Mechanistic Insights and Precision Targeting Strategies in Gastric Cancer Progression and Therapeutic Resistance
              </a>
            </td>
          <td>
            Shiyang Deng, Yong Zhen Chen, Jiang Du
          </td>
          <td>2025-07-23</td>
          <td>Cell Proliferation</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Cancer remains one of the leading health threats globally, with therapeutic resistance being a long-standing challenge across chemotherapy, radiotherapy, targeted therapy, and immunotherapy. In recent years, the association between epigenetic modification abnormalities and therapeutic resistance in tumors has garnered widespread attention, spurring interest in the development of approaches to target epigenetic factors. In this review, we explore the widespread dysregulation and crosstalk of various types of epigenetic modifications, including DNA methylation, histone modifications, and non-coding RNA changes, which interact through complex regulatory networks in tumors. Clinically, single-targeted therapy based on epigenetic modification usually has its limited effect against cancer. However, the combination of epigenetic drugs with other treatment modalities, such as chemotherapy, targeted therapy, or immunotherapy, shows potential for synergistically enhancing efficacy and reducing drug resistance. Therefore, we evaluate the possibility and potential mechanisms of targeting epigenetic modifications to overcome resistance in cancer therapy, and discuss the challenges and opportunities in moving epigenetic therapy into clinical practice. Moreover, the application of multi-omics technologies will aid in identifying core epigenetic factors from complex epigenetic networks, enabling precision treatment and overcoming therapeutic resistance in tumors. Furthermore, the development of spatial multi-omics technologies, by providing spatial coordinates of cellular and molecular heterogeneity, revolutionizes our understanding of the tumor microenvironment, offering new perspectives for precision therapy. In summary, the combined application of epigenetic therapies and the integration of multi-omics technologies herald a new direction for cancer treatment, holding the potential to achieve more effective personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8def3b2fb87b962d999f696cd613916b46b863b4" target='_blank'>
              Targeting epigenetic regulators as a promising avenue to overcome cancer therapy resistance
              </a>
            </td>
          <td>
            Jiawei Song, Ping Yang, Canting Chen, Weiqun Ding, O. Tillement, Hao Bai, Shuyu Zhang
          </td>
          <td>2025-07-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>4</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a cornerstone in the treatment of early-stage high-risk breast cancers (BC), particularly in triple-negative, HER2-positive, and selected hormone receptor-positive subtypes. However, its effectiveness is frequently hindered by intrinsic or acquired resistance, resulting in a significant residual cancer burden (RCB) in more than half of patients. Despite extensive genomic profiling, reliable predictive biomarkers for treatment response remain limited, impeding the development of personalized therapeutic strategies. Emerging evidence highlights tumor metabolic reprogramming as a key non-genetic mechanism contributing to NAC resistance. In this review, we critically examine current advances in metabolic imaging and metabolomics as tools to predict NAC response in BC. We also discuss the role of the tumor microenvironment (TME), including hypoxia and acidosis, in shaping metabolic plasticity and fostering treatment resistance. Furthermore, we explore novel therapeutic strategies targeting metabolic pathways, ranging from enzyme inhibition to dietary interventions, and the use of advanced preclinical models. Together, these insights offer a metabolic framework for overcoming NAC resistance and advancing precision oncology in early-stage BC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03500-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de78332c9b34ec581f5a78aef2b8dfd8a219776" target='_blank'>
              Resistance to neoadjuvant chemotherapy in breast cancers: a metabolic perspective
              </a>
            </td>
          <td>
            Manon Desgres, Melis Poyraz, Buse Sari, Francois P. Duhoux, Cédric van Marcke, C. Corbet
          </td>
          <td>2025-08-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome instability in induced pluripotent stem cells (IPSC) poses a significant challenge for their use in research and medicine. Cataloging and precisely describing all the identified aberrations that arise during cell reprogramming, expansion, and differentiation is essential for improving approaches to instability prevention and ensuring genetic quality control. We report the karyotypic analysis of 65 cell lines derived from skin fibroblasts, urinal sediment, and peripheral blood mononuclear cells of 33 individuals, 82% of whom suffer from monogenic genetic disorders not associated with genetic instability. Trisomy of chromosomes 20 and 8 was revealed recurrently, while the 1q arm was the most frequently affected region involved in interstitial duplications and unbalanced translocations with chromosomes 15 and 18. The localization of rearrangement breakpoints identified by SNP arrays within the large DCC gene and histone gene clusters links genetic instability in IPSCs to replication-stress-induced chromosome breakage at common and early replicating fragile sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4802c78d378e6001adc2d0cfa2a7308e860155" target='_blank'>
              Chromosomal Aberrations in Induced Pluripotent Stem Cells: Identification of Breakpoints in the Large DCC Gene and HIST2 Histone Gene Cluster
              </a>
            </td>
          <td>
            D. Zheglo, V. Pozhitnova, A. V. Kislova, Zh.G. Markova, Danila Kiselev, P.S. Sviridov, Valeria Sviridova, Lyajsan I. Gumerova, S. Smirnikhina, Almaqdad Alsalloum, S. V. Pylina, S. Kutsev, Ekaterina Sergeevna Voronina
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The prognosis of low-grade glioma (LGG) exhibits significant heterogeneity, and the optimal management strategy remains controversial. Therefore, identifying biomarkers associated with glioma prognosis is necessary. Methods Hub genes were identified and a prognostic risk signature was constructed in the TCGA cohort using LASSO Cox regression. The predictive significance of this model was assessed and confirmed in the independent CGGA cohort. Single-cell transcriptional profiling characterized the hub gene expression patterns across cell types. Results We screened four hub genes (SERPINE1, KIF2C, SLC16A1, FABP5). Patients were stratified into high- and low-risk groups using this model. The high-risk group had significantly lower survival rates in both TCGA and CGGA cohorts. The model demonstrated strong predictive power (1/2/3-year AUCs: 0.85/0.85/0.83 in TCGA; 0.73/0.71 for 2/3 years in CGGA). Functionally, the high-risk group exhibited activation of pro-tumorigenic pathways, showed higher levels of immune infiltration across multiple lymphocyte and myeloid subsets, and demonstrated higher sensitivity to drugs such as AZD5582 and AZD8055, evidenced by significantly lower IC50 values. At the single-cell level, FABP5 was highly expressed in myeloid cells, and FABP5 + tumor-associated macrophages (TAMs) were enriched in lipid metabolism and antigen presentation pathways. Pseudotime analysis suggested increasing FABP5 expression during TAMs differentiation. Conclusions The constructed 4-gene hypoxia-lactylation prognostic model can effectively predict survival risk in LGG patients. The high-risk group is characterized by activation of pro-tumorigenic pathways, high immune infiltration, and sensitivity to specific targeted drugs. FABP5 + TAMs participate in LGG progression by regulating lipid metabolism and inflammatory responses, representing a potential therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03459-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193ed959679c670c14a31c6b0e0120de559bd31d" target='_blank'>
              Integrated single-cell and bulk RNA-sequencing data reveal prognosis and therapeutic response in low-grade glioma based on hypoxia-lactylation related genes
              </a>
            </td>
          <td>
            Hongyu Tang, Ruijie Zhang, Ying Xu, Xu Wang, Jingyan Gu, Lianping Gu, Yao Liu
          </td>
          <td>2025-08-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c33305648009e1896ce514a1bce98da6447d96" target='_blank'>
              Epigenetic dysregulation in cancer: mechanisms, diagnostic biomarkers and therapeutic strategies.
              </a>
            </td>
          <td>
            Komal Imran, Muhammad Javed Iqbal, Muhammad Mushtaq Ahmed, Ammara Khalid, H. Cortés, O. Reyes-Hernández, Gabriela Figueroa-González, G. Leyva-Gómez, L. Falzone, Massimo Libra, Federica Longo, Javad Sharifi‐Rad, D. Calina
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Oral cancer, a formidable public health challenge, demands innovative approaches for early detection and effective treatment. This review explores the transformative role of transcriptomics in oral cancer research, leveraging cutting-edge technologies to decode the RNA landscape of tumors. We delve into RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), long non-coding RNA sequencing (lncRNA-seq), microarray analysis, and small RNA profiling, showcasing their unique contributions to unraveling the complexities of oral cancer. By mapping gene expression patterns, identifying biomarkers, and understanding tumor heterogeneity at an unprecedented resolution, transcriptomics offers powerful insights into the molecular mechanisms driving oral cancer. Furthermore, we highlight how these approaches pave the way for personalized medicine, enabling precise therapeutic interventions and innovative treatment strategies. The review also addresses the future directions and challenges in the field, emphasizing the need for advanced computational tools and interdisciplinary collaboration to fully harness the potential of transcriptomics in transforming oral cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11589d7a62e5f0b71eace88c7076fdf589259998" target='_blank'>
              Role of transcriptomics in the study of oral cancer
              </a>
            </td>
          <td>
            P. Natarajan, V. Umapathy
          </td>
          <td>2025-07-28</td>
          <td>Frontiers in Oral Health</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, , Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Acute myeloid leukaemia-normal karyotype (AML-NK) exhibits heterogeneity in expression profiles, influencing the treatment response and survival outcome. Transcriptome sequencing allows a comprehensive analysis of differentially expressed genes (DEGs) and dysregulated pathways in AML-NK, shedding light on the molecular mechanisms and their implications in patients’ management. DEG analyses utilising transcriptome sequencing were conducted using a customised DESeq2 pipeline on 51 AML-NK patients at diagnosis (DX), 12 AML-NK patients who attained first remission (CR1) and 12 healthy controls. The transcriptomic sequencing of AML-NK compared to healthy controls revealed 5,126 DEGs, comprising 85.8% coding genes and 14.2% non-coding elements across 37 pathway categories. The AML-NK DX versus CR1 identified 5,621 DEGs consisting of 84.7% coding genes and 15.3% non-coding elements affecting 20 categories of pathways. Gene set enrichment analysis in this study revealed consistent upregulation of proliferative pathways, including cell cycle and DNA replication. In contrast, immune-related pathways, such as cytokine-cytokine receptor interactions and MHC antigen presentation pathways, were downregulated. Overexpression of oncogenes (FLT3, MYB, DNMT3B, and MYCN) in DX vs CR1 samples reinforces their usefulness in minimal residual disease monitoring, especially in AML-NK with no genetic aberrations. These findings reiterate the known hallmarks of cancers and validate the transcriptomic dysregulation in the pathogenesis of AML-NK. The robustness of the transcriptome sequencing findings was confirmed by RT-qPCR validation of six genes that were not reported in AML-NK patients. The comprehensive analyses of pathways with dysregulation of a myriad of genes led to an understanding of AML-NK pathogenesis and highlighted the markers for minimal residual disease. In summary, this study performed the first transcriptome-wide analysis of AML-NK in a Malaysian cohort and underscored pathways that are candidates for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33caffbddb1bfda3bf17661b32ca7747a9b3aa" target='_blank'>
              Gene expression profiling and pathway analysis in acute myeloid leukaemia-normal karyotype patients
              </a>
            </td>
          <td>
            A. Ambayya, Rozaimi Razali, S. Sulong, Y. Y. Yap, Veena Selvaratnam, J. Sathar, Rosline Hassan
          </td>
          <td>2025-09-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Targeted therapies have significantly improved the prognosis and productivity of non-small-cell lung cancer (NSCLC) patients carrying driver mutations, but drug resistance is inevitable. Histological transformation, an important resistance mechanism, is often manifested as transformation into small-cell lung cancer, large-cell neuroendocrine carcinoma, squamous cell carcinoma, and sarcomatoid carcinoma. The mechanisms involved are complex, including RB1/TP53 inactivation, epithelial–mesenchymal transition, and microenvironmental changes. Post-transformation tumors are often more aggressive and drug-resistant, with limited therapeutic options and poorer prognosis. In this paper, we systematically review the histological transformation types, molecular mechanisms, and therapeutic strategies of NSCLC after resistance to targeted therapy, with the aim of providing a reference for clinical decision-making and promoting the development of individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71911ca0beebb701fd3deee5a9b4ea23c86ca3e7" target='_blank'>
              Histologic transformation after targeted therapy resistance in driver gene-positive NSCLC: mechanisms and therapeutic challenges
              </a>
            </td>
          <td>
            Xinyue Li, K. Ding, Dujiang Liu, Yuxia Wang, Yanjun Xu
          </td>
          <td>2025-08-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pediatric leukemia is a leading cause of cancer-related mortality in children, necessitating a deeper understanding of its molecular mechanisms. This study identifies key regulatory networks driving pediatric leukemia using RNA sequencing (RNA-seq) and data from the European Nucleotide Archive (ENA). Peripheral blood samples from pediatric leukemia patients and healthy controls were analyzed. The computational pipeline included quality control via FastQC, read alignment to the reference genome using STAR and differential gene expression analysis with DESeq2. Total 250 differentially expressed genes (DEGs) were identified between leukemia and control samples. Key pathways associated with leukemogenesis including cell cycle regulation, apoptosis and immune response, were significantly enriched. Transcription factors such as MYC and NF- κB were highlighted as central regulators of these networks. Gene ontology (GO) and pathway enrichment analysis were performed using DAVID and KEGG databases, revealing dysregulated immune signaling as a prominent feature of pediatric leukemia. Weighted gene co-expression network analysis (WGCNA) was employed to identify gene modules strongly correlated with leukemic phenotypes.

This study provides a comprehensive overview of dysregulated gene networks in pediatric leukemia, leveraging publicly available ENA data and advanced computational techniques. The results offer potential biomarkers for early diagnosis and new therapeutic targets in pediatric leukemia, contributing to a better understanding of its molecular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262581edb4d5fd27a9d692131e4d1de93fe755b3" target='_blank'>
              RNA-seq Insights into Major Regulatory Networks in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ruchi Yadav
          </td>
          <td>2025-07-31</td>
          <td>Research Journal of Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="One of the biggest obstacles to successful cancer treatment is still therapeutic resistance, which frequently leads to recurrence and unsatisfactory clinical outcomes. MicroRNA‐200c (miR‐200c), one of several molecular regulators, has grown into a crucial modulator of treatment efficacy by affecting processes including apoptosis, drug efflux, epithelial‐mesenchymal transition, and cancer stem cell properties. Despite extensive research on miR‐200c's roles in drug resistance, there is lack of comprehensive reviews summarizing these findings. This review gathers the most recent data on the complex functions of miR‐200c in mediating chemotherapy and radiotherapy resistance across various cancer types. Its potential clinical aspects as a biomarker and therapeutic target are further discussed. Finally, existing knowledge gaps are outlined, and future research directions are proposed to support development of miR‐200c‐based strategies for overcoming therapeutic resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f0c12a9123a2f950b3a37d371b3428388745665" target='_blank'>
              Understanding miR‐200c: an Important Player in Resistance to Cancer Treatment
              </a>
            </td>
          <td>
            Fatma Şanli, Omer Faruk Karatas
          </td>
          <td>2025-07-23</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6adfd64895599aa6bae6a1f8896e93b0d56bc5ac" target='_blank'>
              ADAR1-mediated RNA editing in breast cancer: molecular mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Si-Yu Chen, Si-Ying Chen, Shu Yang, Yi Li, Shengyong Yang
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485aaa6c6ae7c10944eff4bdd9fd536b93adfcd9" target='_blank'>
              Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
              </a>
            </td>
          <td>
            R. Edahiro, Go Sato, Tatsuhiko Naito, Y. Shirai, R. Saiki, K. Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, S. Namba, Noa Sasa, Genta Nagao, Qingbo S. Wang, Yugo Takahashi, Takanori Hasegawa, T. Kishikawa, Ken Suzuki, Yu-Chen Liu, D. Motooka, Ayako Takuwa, Hiromu Tanaka, S. Azekawa, H. Namkoong, R. Koike, Akinori Kimura, S. Imoto, Satoru Miyano, Takanori Kanai, K. Fukunaga, Mamoru Uemura, T. Morita, Yasuhiro Kato, H. Hirata, Y. Takeda, Y. Doki, H. Eguchi, Daisuke Okuzaki, Shuhei Sakakibara, Seishi Ogawa, Atsushi Kumanogoh, Yukinori Okada
          </td>
          <td>2025-07-28</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common types of malignancy diagnosed globally. Metastasis plays a major role in most of the cancer-related mortality among affected patients. Despite the advances in the areas of early detection and localized treatment modalities, there prevail several challenges which the therapeutic strategies encounter, like drug resistance, tumor heterogeneity, and drug delivery. This review presents a comprehensive and detailed overview of organ-specific metastasis that occur in BC, specifically emphasizing key sites such as the bone, liver, lung, and brain. It also outlines the significance of various therapies like chemotherapies, endocrine therapies, targeted therapies and immunotherapies that have been clinically approved to date. The review specifically emphasizes the molecular mechanisms by which non-coding RNAs (ncRNAs) act to exert their effects in regulating drug resistance. It also addresses the new advances in nanotechnology-based drug delivery systems (DDS) that function to enhance the specificity of treatments while simultaneously reducing systemic toxicity. Beyond ncRNAs, this review also explores other critical mechanisms of drug resistance in metastatic BC, including efflux transporter activity, target gene mutations, and micro-environmental factors, to mention a few. Moreover, the review also discusses the clinical significance of combination therapies and new therapeutic strategies, including the use of repurposed drugs and the concepts of personalized medicine. A greater understanding of the ncRNA-mediated signaling pathways, in combination with the latest advances in drug delivery systems, has the potential to greatly improve therapeutic efficacy and could result in more favorable clinical outcomes in the treatment of metastatic BC (MBC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2fe23da14c71b938c6019fd6995e4bfcec570b" target='_blank'>
              Bridging molecular insights and clinical application: non-coding RNAs, targeted drug delivery, and metastatic breast cancer therapy
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Docetaxel is a chemotherapeutic agent widely used for breast cancer treatment; however, its efficacy is often limited by drug resistance and associated toxicity. This review examines the molecular mechanisms of docetaxel resistance in breast cancer and discusses research advances and future directions for overcoming this challenge. Key resistance mechanisms include alterations in drug targets (microtubules), increased drug efflux, suppression of apoptosis, activation of survival signalling pathways, epithelial-to-mesenchymal transition (EMT), and cancer stem cell enrichment. An evolutionary perspective distinguishes between intrinsic and acquired resistance, emphasising the need for adaptive therapeutic strategies. Recent advances in genomic profiling, non-coding RNA research, novel drug combinations, and biomarker-guided therapies have also been reviewed. Emerging approaches, such as targeting the tumour microenvironment, harnessing immunotherapy, and implementing adaptive dosing schedules, have been discussed. This review emphasises the understanding of resistance as a multifactorial phenomenon that requires multipronged interventions. Research has aimed to identify predictive biomarkers, develop targeted agents to reverse resistance, and design rational combination strategies to improve patient outcomes. Progress in deciphering and targeting docetaxel resistance mechanisms holds promise for enhancing treatment responses and extending survival in patients with breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb368d7af0b16140c6442111947c4c43ba088b35" target='_blank'>
              Docetaxel Resistance in Breast Cancer: Current Insights and Future Directions
              </a>
            </td>
          <td>
            F. Postigo-Corrales, Asunción Beltrán-Videla, A. Lázaro-Sánchez, Ana María Hurtado, P. Conesa-Zamora, Ana Belén Arroyo, G. Luengo-Gil
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) poses a considerable global health challenge, notably due to the resistance to sorafenib therapy, which significantly impedes effective treatment strategies. This study aimed to identify potential resistance-associated genes and develop a robust prognostic model to predict patient outcomes. Methods We utilized transcriptomic data from the gene expression omnibus (GEO) database, focusing on sorafenib-resistant Huh7 and MHCC97H cell lines (GSE94550 and GSE176151), and integrated expression, mutation, and clinical data from the cancer genome atlas (TCGA) and international cancer genome consortium (ICGC) databases. Single-cell RNA sequencing data (GSE149614) were processed with the Seurat and Harmony R packages for quality control and integration. Differential gene expression analysis, consensus clustering, and principal component analysis were performed to identify significant genes and stratify patients based on prognostic outcomes. Results The analysis revealed 305 potential resistance-associated genes, with a seven-gene (ANAPC13, NCAPD2, KIF2C, CDK5RAP2, MANBA, PPAT, and LPCAT1) risk model demonstrating significant prognostic capability, indicated by area under curve values of 0.824, 0.746, and 0.717 for 1, 3, and 5-year survival predictions, respectively. Notably, immune cell infiltration analyses highlighted significant correlations between risk scores and specific immune cell types, suggesting potential therapeutic targets. Drug sensitivity analysis further identified various compounds with lower IC50 values in high-risk groups. To facilitate clinical application, a nomogram plot was designed. Conclusion This study provides a comprehensive framework for understanding sorafenib resistance in HCC, alongside a validated prognostic model that holds potential for clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6acc2fd61529281885a6db1b562f4fbba92f2fe" target='_blank'>
              Identifying genes associated with Sorafenib resistance in hepatocellular carcinoma to develop risk model
              </a>
            </td>
          <td>
            Kuan He, Hongyan Hui, Fang Zhou, Kai Liu, Xi Zhao, Jun Liu
          </td>
          <td>2025-08-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common malignancies worldwide. Its complex molecular mechanisms make traditional treatments ineffective in some patients. In recent years, advances in genetic engineering technologies have opened up new possibilities for precision medicine in BC. Researchers have attempted to intervene in cancer cell proliferation and induce apoptosis through various genetic strategies. Preclinical studies are increasing, and therapeutic targets and delivery technologies are expanding. However, current research has largely focused on specific cases or single technologies, lacking a systematic integration and analysis of various genetic strategies from a mechanistic perspective. This paper reviews three major genetic engineering strategies for BC based on their therapeutic mechanisms: RNA interference, gene compensatory expression, and gene repair and deletion. Representative studies, key technologies, challenges, and development trends are analyzed for each. Furthermore, this paper summarizes common challenges across these different mechanisms regarding delivery efficiency, stability, and targeting, and proposes potential directions for future synergistic strategies. This paper helps clarify current research approaches in BC gene therapy from a mechanistic perspective, providing a reference for the design of subsequent cross-mechanism integration and precision intervention strategies. However, issues such as mechanism overlap and diversification of delivery platforms still need further in-depth exploration. Future research can focus more on multi-mechanism combination strategies, individualized treatment pathways, and feasibility assessment of clinical transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38bb75ae7dd66792d49d2225441735e8c43eeb18" target='_blank'>
              Applications of Three Gene Engineering Mechanisms in Breast Cancer Therapy
              </a>
            </td>
          <td>
            Yifan Tian
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b8053e41e58296473fbe96b5f5b8ab91cf715" target='_blank'>
              Reversible therapeutic resistance in EGFR mutant lung cancer caused by RB1 loss induced lineage plasticity.
              </a>
            </td>
          <td>
            Letian Zhang, M. Chinnam, Bradley Balk, Xiaojing Zhang, Yanqing Wang, Tatiana Shaurova, Shamshad Alam, Saraswati Pokharel, Prashant Singh, Justine Jacobi, Jianming Wang, Eduardo Cortes Gomez, Michael Habitzruther, Mukund Seshadri, B. Atanassov, Wei Wu, E. Benevolenskaya, C. Blakely, Pamela Hershberger, Mark Long, T. Bivona, David W. Goodrich
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Thyroid cancer pathogenesis involves complex interactions between genetic predisposition and alterations in the tumor microenvironment. The causal relationships between inflammatory gene variants and thyroid cancer risk remain poorly understood, as does the cellular heterogeneity within the tumor ecosystem. This study aimed to investigate the causal associations between inflammatory protein genes and thyroid cancer risk, and to characterize the cellular composition and differentiation trajectories within the thyroid cancer microenvironment. Methods We employed a two-pronged approach combining Mendelian randomization (MR) analysis and single-cell RNA sequencing (scRNA-seq). MR analyses were conducted using genetic variants associated with the expression of inflammatory proteins (4EBP1_EIF4EBP1, ADA_ADA, ARTN_ARTN, AXIN1_AXIN1, and Beta-NGF_NGF) as instrumental variables to assess their causal effects on thyroid cancer risk. Multiple MR methods (MR Egger, weighted median, inverse variance weighted, simple mode, and weighted mode) were used to enhance robustness. For the cellular characterization, scRNA-seq was performed on thyroid cancer samples, followed by dimensionality reduction, clustering analysis, cell type annotation, and pseudotime trajectory inference. Results MR analyses revealed a significant positive causal association between AXIN1_AXIN1 expression and thyroid cancer risk (weighted median: OR = 1.396, p < 0.05; inverse variance weighted: OR = 1.291, p < 0.05), while ADA_ADA showed protective effects (simple mode: OR = 0.731, p < 0.05). The scRNA-seq analysis identified six major cell populations within the thyroid cancer microenvironment: epithelial cells, T cells, natural killer cells, fibroblasts, stromal cells, and macrophages. Pseudotime analysis revealed distinct differentiation trajectories with natural killer cells and macrophages appearing in early pseudotime, while epithelial cells and fibroblasts demonstrated multiple developmental states. Gene expression profiling identified four distinct cellular states with unique molecular signatures, including immune/inflammatory, stromal, and vascular components. Conclusion Our findings suggest that inflammatory protein genes, particularly AXIN1, have causal effects on thyroid cancer risk, providing potential targets for risk prediction and intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbfdc7a38a023cb6812db2d719e9e3e9bf3c6626" target='_blank'>
              The role of cellular heterogeneity in thyroid cancer: a multi-modal analysis approach with Mendelian randomization
              </a>
            </td>
          <td>
            Yichun Qian, Wei Chen, Xinyuan Chen, Shuai Cheng, Fangzhou Liu
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Late diagnosis and chemotherapy resistance, particularly to 5-fluorouracil (5-FU), contribute to the low survival rate in cholangiocarcinoma (CCA) patients. Identifying relevant genes and pathways, as well as novel targeted molecules, is crucial to overcoming 5-FU resistance and improving treatment outcomes for CCA patients.


OBJECTIVES
This study aimed to determine the potential molecules associated with 5-FU resistance in CCA cells.


MATERIAL AND METHODS
Transcriptomic datasets from 4 stable 5-FU-resistant cell lines and their corresponding parental lines were retrieved from the Gene Expression Omnibus. A series of bioinformatics analyses were conducted to identify key genes upregulated in 5-FU-resistant cells compared to their parental counterparts. The expression levels of candidate genes identified through bioinformatics analysis were validated in CCA tissues and cell lines.


RESULTS
Differential gene expression, protein-protein interaction, and Hub genes analysis revealed 8 genes that were significantly upregulated in 5-FU resistance cells compared to their parental cells. Six of the 8 genes, including TCP1, RPS6, RPS29, HSPA5, RPS15A, and NOTCH1, were upregulated in patient CCA tissues. Using real-time PCR, only the expression levels of NOTCH1 and TCP1 were significantly higher in the 5-FU insensitive CCA cell lines, KKU-213A and KKU-213B, than that of the 5-FU sensitive CCA cell line, KKU-055. A similar result was observed in stable 5-FU-resistant cell lines (KKU-213A-FR and KKU-213B-FR) compared to their parental cells.


CONCLUSIONS
The bioinformatic analysis and PCR results revealed that NOTCH1 and TCP1 might be associated with 5-FU resistance and serve as potential molecular targets to enhance 5-FU sensitivity in CCA cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8104f35b6140638792fb57197dadae1949d4723" target='_blank'>
              Bioinformatics analysis identified TCP1 and NOTCH1 as potential target molecules to overcome 5-fluorouracil resistance in cholangiocarcinoma.
              </a>
            </td>
          <td>
            Sonexai Kidoikhammouan, Nopkamol Kanchanangkul, Worachart Lert-itthiporn, Raksawan Deenonpoe, Charupong Saengboonmee, S. Obchoei, S. Wongkham, Wunchana Seubwai
          </td>
          <td>2025-09-04</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Bladder cancer pathogenesis is closely linked to epigenetic alterations, particularly DNA methylation and demethylation processes. Environmental carcinogens and persistent inflammatory stimuli—such as recurrent urinary tract infections—can induce aberrant DNA methylation, altering gene expression profiles and contributing to malignant transformation. This review synthesizes current evidence on the role of DNA methyltransferases (DNMT1, DNMT3a, DNMT3b) and the hypermethylation of key tumour suppressor genes, including A2BP1, NPTX2, SOX11, PENK, NKX6-2, DBC1, MYO3A, and CA10, in bladder cancer. It also evaluates the therapeutic application of DNA-demethylating agents such as 5-azacytidine and highlights the impact of chronic inflammation on epigenetic regulation. Promoter hypermethylation of tumour suppressor genes leads to transcriptional silencing and unchecked cell proliferation. Urine-based DNA methylation assays provide a sensitive and specific method for non-invasive early detection, with single-target approaches offering high diagnostic precision. Animal models are increasingly employed to validate these findings, allowing the study of methylation dynamics and gene–environment interactions in vivo. DNA methylation represents a key epigenetic mechanism in bladder cancer, with significant diagnostic, prognostic, and therapeutic implications. Integration of human and experimental data supports the use of methylation-based biomarkers for early detection and targeted treatment, paving the way for personalized approaches in bladder cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f706e4ed85cef4708cafccd425d482bfabf2666" target='_blank'>
              DNA Methylation in Bladder Cancer: Diagnostic and Therapeutic Perspectives—A Narrative Review
              </a>
            </td>
          <td>
            D. Puia, M. Ivănuță, C. Pricop
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Broad-spectrum multi-target tyrosine kinase inhibitors (mTKIs) are clinically approved for the treatment of soft tissue sarcomas (STS). However, acquired resistance inevitably arises in the majority of STS patients. There is therefore an urgent need to identify new strategies to overcome resistance and achieve durable treatment responses. Here we show that STS cells that acquire resistance to clinically relevant mTKIs are cross-resistant to one another and sequential treatment does not delay the acquisition of drug resistance. Instead, we find that en route to acquiring drug resistance, STS cells develop collateral sensitivities to alternative drugs. We demonstrate that the mTKI sitravatinib rapidly induces collateral sensitivity to the FGFR inhibitor infigratinib which can be exploited for adaptive therapy to suppress STS cell growth. This study provides proof-of-principle that collateral sensitivity may be an effective strategy for overcoming resistance to mTKIs and this novel approach should be explored in the design of future trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecc73e9d17d4b9da701d49b02c116223f8d29d2" target='_blank'>
              Exploiting collateral sensitivity in the evolution of resistance to tyrosine kinase inhibitors in soft tissue sarcomas
              </a>
            </td>
          <td>
            Mark L Elms, Andrew D Jenks, Avirup Chowdhury, Lukas Krasny, Madhumeeta Chadha, Kaan Low, Peter T. Harrison, William Kerrison, Robin L Jones, P. Huang
          </td>
          <td>2025-08-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Drug resistance critically limits the long-term efficacy of MAPK-targeted therapy in melanoma. While resistance mechanisms at genetic, epigenetic, and transcriptional scales are well-documented, post-transcriptional splicing regulation remains poorly understood. By analyzing patient-matched pre-treatment and resistant melanoma biopsies, we uncover widespread alternative splicing alterations during therapy resistance. Splicing perturbations are most pronounced in MAPK and PI3K-AKT pathway genes. We identify a splicing switch of AKT2 from isoform 210 to 206 in 29.55% (13/44) of disease-progressive biopsies. This splicing switch induces AKT2 kinase hyperactivity by restoring the activated fragment A-loop. Functional validations confirm that AKT2-206 confers BRAF inhibitor resistance in melanoma cells by activating S6 kinase. Further, the splicing factor hnRNPK likely drives the splicing switch of AKT2 during acquired resistance. Our results not only provide insights into splicing-mediated regulation of drug resistance but also highlight the importance of alternative splicing isoforms as targets for clinical diagnosis and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4a2e884a937abc2345c5e03e509d49bf56bc09" target='_blank'>
              Transcriptome-wide decoding the roles of aberrant splicing in melanoma MAPK-targeted resistance evolution
              </a>
            </td>
          <td>
            Jing Yu, Xiujing He, Xueyan Wang, Chune Yu, Xian Jiang, Yanna Li, Xinyu Liu, Ya Luo, Xuemei Chen, Sisi Wu, Lu Si, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-07-18</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here, we showed that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells displayed increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence was rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimicked the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slowed the growth of multiple PAX3-FOXO1+ tumor xenograft models but not the fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba839c9a2b458336e1dd8e4d0285ec2394738d6" target='_blank'>
              PAX3-FOXO1 Drives Targetable Cell State-Dependent Metabolic Vulnerabilities in Rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Angela M Montero, Jacob A Boyer, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, A. Kung, Daoqi You, F. D. Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-09-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Immune-related genes (IRGs) play a pivotal role in the tumor microenvironment of triple-negative breast cancer (TNBC). This study aimed to uncover novel prognostic markers for TNBC based on IRGs. Differential expression analysis, univariate Cox regression, and machine learning algorithms identified BPI, GPHA2, KIR3DL3, MASP1, RASGRP1, and SEMA4A as potential prognostic genes. A random survival forest (RSF) model was developed and subsequently validated, alongside the creation of a prognostic nomogram to assess gene diagnostic performance. Further, epithelial cells, fibroblasts, and T cells were recognized as the primary cell types through which RASGRP1 exerts its effects. Finally, an analysis of the immune microenvironment and drug predictions provided fresh insights for the treatment of TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15157-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5a164db4384d30a7ce7d87a791eadd2b7becb9" target='_blank'>
              Immune microenvironment and prognostic genes of triple-negative breast cancer in the context of transcriptome and single-cell sequencing
              </a>
            </td>
          <td>
            Xiangqing Huang, Wei Chen, Zhiwu Lin, Hui Zhang, Ruijuan Wang, Cailing Lin
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0a59124a03b9df5aeccdd7b10806e57fa0f866" target='_blank'>
              In vivo screening reveals cell-intrinsic mediators of solid tumor resistance to CAR T cell-therapy.
              </a>
            </td>
          <td>
            Julia Fröse, Charles A Whittaker, Paul Leclerc, Evelyn Chen, Sean Doherty, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Tuomas Tammela, Michael T. Hemann
          </td>
          <td>2025-07-29</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Objectives: Medulloblastoma is the most common malignant brain tumor in children and comprises four molecular subtypes—WNT, SHH, Group 3, and Group 4—each with distinct genetic, epigenetic, and metabolic features. Increasing evidence highlights the critical role of metabolic reprogramming and epigenetic alterations in driving tumor progression, therapy resistance, and clinical outcomes. This review aims to explore the interplay between metabolic and epigenetic mechanisms in medulloblastoma, with a focus on their functional roles and therapeutic implications. Methods: A comprehensive literature review was conducted using PubMed and relevant databases, focusing on recent studies examining metabolic pathways and epigenetic regulation in medulloblastoma subtypes. Particular attention was given to experimental findings from in vitro and in vivo models, as well as emerging preclinical therapeutic strategies targeting these pathways. Results: Medulloblastoma exhibits metabolic adaptations such as increased glycolysis, lipid biosynthesis, and altered amino acid metabolism. These changes support rapid cell proliferation and interact with the tumor microenvironment. Concurrently, epigenetic mechanisms—including DNA methylation, histone modification, chromatin remodeling, and non-coding RNA regulation—contribute to tumor aggressiveness and treatment resistance. Notably, metabolic intermediates often serve as cofactors for epigenetic enzymes, creating feedback loops that reinforce oncogenic states. Preclinical studies suggest that targeting metabolic vulnerabilities or epigenetic regulators—and particularly their combination—can suppress tumor growth and overcome resistance mechanisms. Conclusions: The metabolic–epigenetic crosstalk in medulloblastoma represents a promising area for therapeutic innovation. Understanding subtype-specific dependencies and integrating biomarkers for patient stratification could facilitate the development of precision medicine approaches that improve outcomes and reduce long-term treatment-related toxicity in pediatric patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733445d34f0ee82899b45eeeaeabd1e9ff5c7ba8" target='_blank'>
              Deciphering Medulloblastoma: Epigenetic and Metabolic Changes Driving Tumorigenesis and Treatment Outcomes
              </a>
            </td>
          <td>
            Jenny Bonifacio-Mundaca, Sandro Casavilca-Zambrano, Christophe Desterke, Íñigo Casafont, Jorge Mata-Garrido
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer continues to be the most common cause of cancer-related disease and mortality among men worldwide, especially in the advanced stages, notably metastatic castration-resistant prostate cancer (mCRPC), which poses significant treatment challenges. Docetaxel, a widely used chemotherapeutic agent, has long served as the standard treatment, offering survival benefits and mitigation. However, its clinical impact is frequently undermined by the development of chemoresistance, which is a formidable challenge that leads to treatment failure and disease progression. The mechanisms driving docetaxel resistance are diverse and complex, encompassing modifications in androgen receptor signaling, drug efflux transporters, epithelial-mesenchymal transition (EMT), microtubule alterations, apoptotic pathway deregulation, and tumor microenvironmental influences. Recent evidence suggests that extracellular RNAs influence drug responses, further complicating the resistance landscape. This review offers a broad discussion on the mechanisms of resistance and explores novel therapeutic approaches to address them. These include next-generation taxanes, targeted molecular inhibitors, immunotherapies, and combination regimens that can be designed to counteract specific resistance pathways. By broadening our understanding of docetaxel resistance, this review highlights potential strategies to improve therapeutic efficacy and the potential to enhance outcomes in patients with advanced treatment-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538206b7fcdd2ed8debda33969b113a8ee54b834" target='_blank'>
              Beyond Docetaxel: Targeting Resistance Pathways in Prostate Cancer Treatment
              </a>
            </td>
          <td>
            Tayo Alex Adekiya
          </td>
          <td>2025-08-01</td>
          <td>BioChem</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Late recurrence of breast cancer poses a considerable threat to patient survival, often attributed to breast cancer dormancy. Dormancy, characterized by cancer cells remaining quiescent for extended periods, is influenced by genetic factors and modifications that directly impact cellular phenotype. Alterations in gene expression dynamically shape cellular behavior, often mediated through receptor signaling pathways. Moreover, interactions within the tumor microenvironment play a pivotal role, fostering either cancer cell dormancy or promoting their escape from dormancy. This review endeavors to provide a comprehensive synthesis of recent advancements in understanding breast cancer dormancy across genetic, receptor molecular, and cellular dimensions. By elucidating the intricate mechanisms underlying dormancy, we aim to shed light on potential therapeutic strategies to prevent late recurrences. Furthermore, we anticipate future research directions that may uncover novel insights into this complex phenomenon, ultimately improving patient outcomes and refining clinical management strategies for breast cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6a18864245f8cad5e8cc957c1b2db50aef88c" target='_blank'>
              Unraveling the Mystery of Breast Cancer Dormancy: Insights into Genetic, Receptor, and Cellular Interactions Driving Late Recurrence
              </a>
            </td>
          <td>
            Haochen Ma, Bingqiang Zhang, Mengmeng Chen, Zhendi Song, Yi Zhao
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific KRAS inhibitors in colorectal cancer (CRC). Here, we investigated whether broad-spectrum, active-state RAS inhibitors meet similar limitations. We found that KRAS-mutant CRC cell lines were sensitive to the RAS(ON) multiselective RAS inhibitor RMC-7977, given that treatment resulted in RAS-RAF-MEK-ERK pathway inhibition; halted proliferation; and, in some cases, induced apoptosis. RMC-7977 initially reduced the activity of a compartment-specific, dual-color reporter of ERK activity, with reporter reactivation emerging after long-term dose escalation. These drug-resistant cell populations exhibited distinct patterns of phospho-protein abundance, transcriptional activities, and genomic mutations, including a Y71H mutation in KRAS and an S257L mutation in RAF1. Transgenic expression of KRASG13D, Y71H or RAF1S257L in drug-sensitive CRC cells induced resistance to RMC-7977. CRC cells that were resistant to RMC-7977 and harboring RAF1S257L exhibited synergistic sensitivity to concurrent inhibition of RAS and RAF. Our findings demonstrate the power of reporter-assisted screening together with single-cell analyses for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract the emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062fb5066a8ceecc8901f69c1d9a2835ca85a048" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibitors in colorectal cancer.
              </a>
            </td>
          <td>
            Oleksandra Aust, Moritz R T Thiel, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Francisca Böhning, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, C. Sers, C. Der, M. Morkel
          </td>
          <td>2025-07-22</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Hormone receptor-positive (HR + )/HER2−negative (HER2 − ) breast cancer is the most common subtype, with biomarker-driven therapies improving outcomes. Circulating tumor DNA (ctDNA) analysis enables minimally invasive assessment of somatic alterations to guide therapy. However, assay choice impacts clinical utility, and access remains inconsistent. This study compares single-gene and panel-based sequencing for assessing PIK3CA mutations and broader genomic profiling. We conducted a prospective, multicenter study analyzing 161 plasma samples from 146 patients before initiating a new line of palliative therapy using the SiMSen-Seq (SSS) assay for PIK3CA hotspot mutations, the AVENIO ctDNA Expanded assay (77 genes) and mFAST-SeqS for tumor fraction estimation. High concordance (92.6%) was observed between SSS and AVENIO for PIK3CA mutations. AVENIO identified additional actionable alterations, including ESR1 (17.5%) and PI3K pathway alterations (40.6%), and together with tumor fraction estimation, improved interpretation of negative liquid biopsy findings. These findings support broader ctDNA profiling in clinical practice while highlighting accessibility challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e5152da00e15846285751e550fbaaef5313e05" target='_blank'>
              Clinical impact of single-gene vs. panel sequencing in advanced HR + /HER2− breast cancer: insights and implications
              </a>
            </td>
          <td>
            E. Klocker, N. Dobrić, Ricarda Graf, Christine Beichler, D. Hlauschek, C. Suppan, L. Pancheri, D. Egle, Carmen Albertini, R. Bartsch, A. Starzer, P. Jost, G. Rinnerthaler, Ellen Heitzer, N. Dandachi, M. Balic
          </td>
          <td>2025-08-07</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Introduction Next‐generation sequencing‐based comprehensive cancer genomic profiling (CGP) tests are beneficial for refining diagnosis and personalized treatment of various cancers. However, the clinical impact of CGP, as covered by public health insurance in the management of sarcomas, remains unknown. Especially, the data on the utility of the newly emerging dual DNA–RNA panel compared to the conventional DNA‐only panel in clinical settings is lacking. Therefore, we evaluated the utility of CGP in routine clinical practice for sarcoma treatment. Patients and Methods In this study, three types of DNA panel and one DNA–RNA panel, reimbursed by Japanese public health insurance, were utilized. We detected oncogenic and druggable gene mutations and genotype‐matched therapies. Results One hundred and thirty‐six patients were included in this study. Based on the detection of highly histology‐specific translocations in the sequencing results, 2.2% of patients were re‐classified. In patients with translocation‐related sarcomas, a DNA–RNA panel identified more histology‐specific fusion genes than DNA panels (p = 0.0035). Specifically, 86.8% and 39.0% of patients had oncogenic and druggable genomic alterations, respectively. Of these, 9.6% underwent genotype‐matched therapy, with a 36.3% response rate and an 81.8% disease control rate. Patients who were administered genomically matched therapy had better overall survival (OS) than those who did not in patients with metastatic or advanced sarcoma with no prior chemotherapy (3‐year OS: 83.3% vs. 48.0%, p = 0.42). Patients with TP53 and RB1 mutations had worse OS than those without. Germline findings were detected in 11.0% of the patients, one of whom had a truly germline origin. Conclusions This study suggests that publicly reimbursed CGP tests, particularly the dual DNA–RNA panel, could be beneficial for refining diagnostic precision in selected sarcoma subtypes, treatment decisions, detecting the germline findings, and prognosis prediction in routine clinical settings for sarcoma. The implementation of genotype‐matched therapies showed favorable clinical outcomes and improved the prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f63a84e1907839e26eef4d1f8d8db48869e5a3" target='_blank'>
              Real‐World Data of Comprehensive Cancer Genomic Profiling Tests Performed in the Routine Clinical Setting in Sarcoma
              </a>
            </td>
          <td>
            E. Nakata, D. Ennishi, Tatsunori Osone, K. Ninomiya, Shuta Tomida, Takuto Itano, T. Fujiwara, T. Kunisada, Naoyuki Ida, Hideki Yamamoto, Mashu Futagawa, T. Shimoi, Hiroyuki Yanai, Akira Hirasawa, S. Toyooka, Masahiro Tabata, Toshifumi Ozaki
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd134b7250b3edd45d5610ddc418d22bf933baf" target='_blank'>
              Integrative Genomic and Immune Profiling to Identify and Characterize High-Risk Subgroups in Acute Myeloid Leukemia: Development of a 20-Gene Predictive Signature and Its Clinical Implications.
              </a>
            </td>
          <td>
            Kubra Karagoz, M. G. Roukens, Lizzy Comijn, Asuman Celik, Lauren K Brady, Brandon W Higgs, Han Si
          </td>
          <td>2025-09-01</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Lung cancer remains one of the most prevalent and lethal malignancies worldwide, responsible for nearly 1.8 million deaths annually, which accounts for approximately 18.7% of global cancer-related mortality. Cisplatin, a highly effective and widely utilized anticancer drug, is particularly effective against solid tumors and serves as a cornerstone of adjuvant chemotherapy for lung cancer. Despite continuous optimization of cisplatin-based chemotherapy regimens, the emergence of cisplatin resistance frequently results in treatment failure, significantly limiting its clinical utility and therapeutic efficacy. To address this challenge, researchers have extensively investigated the biological mechanisms underlying cisplatin resistance, including impaired DNA repair pathways and inhibition of apoptosis. Among these mechanisms, epigenetic regulation—encompassing DNA methylation, histone modifications, and noncoding RNA (ncRNA) regulation—has emerged as a critical factor in mediating cisplatin resistance by modulating gene expression and signaling pathways. This review comprehensively explores the role of epigenetic mechanisms in cisplatin resistance in lung cancer, highlighting recent research findings and their potential implications for developing strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b34c6c87fbf11d6f9ef8e5eb14596e031f42344" target='_blank'>
              Regulation of cisplatin resistance in lung cancer by epigenetic mechanisms
              </a>
            </td>
          <td>
            Xiaoyan Yu, Luyu Jia, Qing Tang, Qichun Zhou, Genshu Wang, Sumei Wang
          </td>
          <td>2025-08-24</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96cb29c7c4ee18f843780f85e8c5d424d7abe77" target='_blank'>
              Epigenetic modulation in cancer drug discovery: promising targets and clinical applications.
              </a>
            </td>
          <td>
            S. Vickram, Shofia Saghya Infant, M. Sivasubramanian, Saravanan Anbalagan, Mathan Muthu Chinnakannu Marimuthu, Hitesh Chopra
          </td>
          <td>2025-09-02</td>
          <td>Pharmacological reports : PR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor that requires challenging treatments. Tumor Treating Fields (TTFields), an FDA-approved therapy for glioblastoma (GBM), pleural mesothelioma, and platinum-refractory metastatic nonsmall cell lung cancer (in combination with PD-1/PD-L1 inhibitors or docetaxel), employs specific frequency electric fields to disrupt cell division and enhance treatment efficacy. However, their molecular mechanisms remain unclear. This study aimed to elucidate these mechanisms and optimize the therapeutic potential of TTFields through quantitative proteomics, phosphoproteomics, and glycoproteomics. Pathway analysis of the proteomics revealed that TTFields impact the cell cycle, DNA repair, autophagy, and DNA replication. Phosphoproteomic studies further demonstrated a marked decline in the activity of key kinases ABL1 and PDK1, while glycoproteomics highlighted disruptions in cell adhesion and ECM-receptor interactions. Notably, proteomic analysis identified an upregulation of PARP1 and BRD4 protein levels, suggesting a previously unrecognized resistance mechanism. Consistently, combining TTFields with inhibitors targeting these proteins significantly enhanced the treatment efficacy in U87 cells. Thus, this study uncovers comprehensive molecular mechanisms underlying TTFields' effects on GBM cells and supports the development of concomitant therapies to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23e36a333192c5464239bf92b14bd9554e97f70e" target='_blank'>
              Multiomics Insights into the Mechanism and Enhanced Efficacy of Tumor Treating Fields (TTFields) Therapy in Glioblastoma.
              </a>
            </td>
          <td>
            Yanmei Gao, Chenxu Wang, Guobin Liu, Ruilin Zhang, Xuelian Ren, Guangyuan Hu, Qi Mei, He Huang
          </td>
          <td>2025-09-01</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction CDK4/6 inhibitors are cornerstone therapies for advanced HR+/HER2- breast cancer, yet treatment response heterogeneity remains a major clinical challenge. This study integrates single-cell regulatory landscapes with multi-omics data to decode resistance mechanisms and develop predictive biomarkers for CDK4/6 inhibitor response stratification. Methods Single-cell RNA-seq data (GSE158724, n=14 samples) and bulk multi-omics profiles (TCGA-BRCA, n=1,059; GSE186901, n=90) were analyzed. Gene regulatory networks were reconstructed using SCENIC to identify resistance-specific regulons. The Tumor Prognostic Regulon Index (TPRI) was derived from five prognostic transcription factors and validated in independent cohorts. Experimental validation including qPCR of core TFs was performed in patient-derived samples. Multimodal predictive models integrating TPRI, differentially expressed genes, and miRNAs were developed using logistic regression, with performance assessed via ROC/AUC analysis. Results We identified 86 resistance-associated regulons and established TPRI based on five prognostic TFs (ATF1, TEAD4, NFIL3, FOXO1, ETV3). TPRI significantly stratified patients into high/low-risk groups with differential overall survival and treatment response (Fisher’s exact test P=0.0237). qPCR confirmed elevated expression of these TFs in resistant tumors (P<0.01). High-risk patients exhibited increased stemness indices (mRNAsi, P<2.2e-16) and mTOR pathway activation. The multimodal model (TPRI + top 30 DEGs + top 30 miRNAs) achieved superior prognostic accuracy (95%CI:0.6575-0.75). Discussion This study establishes TPRI as a novel biomarker for CDK4/6 inhibitor response prediction, validated through multi-omics integration and qPCR confirmation. The model provides actionable risk stratification, where high-risk patients may benefit from combinatorial mTOR-targeted therapies. Limitations include sample size constraints for methylation integration. Future studies should validate these findings in prospective clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af1d91144e6176cc73b2a5ca97b747f7333a1d7" target='_blank'>
              Integrating single-cell regulatory atlas and multi-omics data for differential treatment response and multimodal predictive modeling in CDK 4/6 inhibitor-treated breast cancer
              </a>
            </td>
          <td>
            Li Yan, Sijie Chen, Ran Ran, Shidi Zhao, Jing Huang, Jin Yang
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Colorectal cancer (CRC) continues to be a major contributor to cancer-associated death, with metastatic disease posing substantial therapeutic challenges. The epithelial-mesenchymal transition (EMT) orchestrates the transformation of polarized epithelial cells into motile mesenchymal phenotypes, characterized by enhanced migratory capacity and invasive properties. EMT is central to CRC metastasis and progression, particularly concerning its contribution to invasion, internal infiltration, and colonization. Beyond metastasis, EMT facilitates cancer cells’ adaptation to diverse microenvironments, gain of stem cell-like characteristics, metabolic reprogramming, and evasion of therapeutic interventions. EMT signatures are emerging as potential prognostic biomarkers, offering valuable insights for real-time disease surveillance and personalized therapeutic strategies. Targeting EMT presents a promising therapeutic avenue to improve drug sensitivity and counteract resistance in CRC. This review systematically examines the molecular mechanisms regulating EMT in CRC, including key transcription factors; post-translational and epigenetic modifications; non-coding RNAs; and pivotal signaling pathways. Additionally, we evaluate the clinical implications of EMT in CRC progression and metastasis and critically assess emerging therapeutic strategies targeting EMT. This study lays the groundwork for developing more efficient interventions to mitigate metastasis and enhance treatment outcomes and patient survival by elucidating the intricate molecular networks that govern EMT and its contributions to CRC pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b5df8807fc3ff4533957ba728d9f34643cd5f8" target='_blank'>
              Epithelial-mesenchymal transition in colorectal cancer metastasis and progression: molecular mechanisms and therapeutic strategies
              </a>
            </td>
          <td>
            Fangfang Nie, Xue Sun, Jizhuo Sun, Jingdong Zhang, Yuanhe Wang
          </td>
          <td>2025-07-22</td>
          <td>Cell Death Discovery</td>
          <td>3</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9194070a3830c48d1eb33aa8623f50b92cbe3b84" target='_blank'>
              Molecular and epigenetic oncogenesis in synovial sarcoma: implications for cancer biology, diagnosis and treatment.
              </a>
            </td>
          <td>
            Amy Xueqi Wang, Kevin B. Jones, Torsten O. Nielsen
          </td>
          <td>2025-08-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor dormancy represents a clinically significant but poorly understood state in which disseminated cancer cells persist in a quiescent, non-proliferative state, evading conventional therapies and driving late relapse. This review summarizes recent advancements in experimental models—both in vitro and in vivo—that recapitulate the full spectrum of dormancy, including its induction, maintenance, and reactivation. Crucial intrinsic pathways such as ERK/p38 signaling shifts, epigenetic remodeling, and metabolic adaptations and microenvironmental and immune-mediated cues that regulate dormant cell fate are discussed. Therapeutic strategies aimed at maintaining dormancy, reactivating dormant cells for elimination, or directly targeting their survival pathways have been highlighted. By integrating insights from model systems, molecular regulation, and therapy, this review aims to provide a comprehensive framework that informs future efforts to target dormant cancer cells and ultimately reduce recurrence and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/780c88da1c462505181b638ab331143519a6fc32" target='_blank'>
              Understanding Tumor Dormancy: from Experimental Models to Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Jaebeom Cho
          </td>
          <td>2025-08-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genome-wide association studies (GWAS) relevant to osteoporosis have identified hundreds of loci; however, understanding how these variants influence the phenotype is complicated because most reside in non-coding DNA sequence that serves as transcriptional enhancers and repressors. To advance knowledge on these regulatory elements in osteoclasts (OCs), we performed Micro-C analysis, which informs on the genome topology of these cells and integrated the results with transcriptome and GWAS data to further define loci linked to BMD. Using blood cells isolated from 4 healthy participants aged 31-61 yr, we cultured OC in vitro and generated a Micro-C chromatin conformation capture dataset. We characterized chromatin loops (CLs) in OC from among more than 69 million chromatin interactions identified in the genome. Of the CL identified in OC, >16 000 were unique compared to precursor cells. When sentinel single nucleotide polymorphisms from osteoporosis and bone-related GWAS and those in linkage disequilibrium at r2 > 0.6 were mapped to CL for OC, 12 588 of these variants were observed within chromatin contact regions. Notable in differential gene ontology enrichment analyses of the topology data for OC and precursors were pathways regulating pluripotency of stem cells, Wnt signaling, nucleotide-binding oligomerization domain (NOD)-like receptor signaling and chemokine signaling. These data, in combination with other 3D genome architecture and epigenetic data (eg, histone modifications and chromatin accessibility), will be useful in modeling to predict genome-wide, which enhancers regulate which genes in OC. This data will therefore also be informative for resolving GWAS hits. In conclusion, we have generated a high-resolution genome topology dataset for human OC and have used this to identify CLs relevant to studies of the genetics of osteoporosis. This data will serve as a powerful resource to inform future functional studies of OC biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db75c088b9f9ba0973d277c93aec7ad4cfeffd1" target='_blank'>
              Genome topology analysis and transcriptomics of human osteoclasts reveals enhancer–promoter interactions at loci for bone traits and diseases
              </a>
            </td>
          <td>
            Scott G Wilson, Purdey J Campbell, Dhanya Sooraj, Kassandra M. Leatherbarrow, B. Mullin, Suzanne J Brown, K. Zhu, S. Mullin, Bryan K Ward, Jordan Zhang, Jonathon Torchia, F. Dudbridge, Jiake Xu, N. Pavlos, David Chandler, John P Walsh
          </td>
          <td>2025-07-16</td>
          <td>JBMR Plus</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) displays remarkable cell state plasticity, a major contributor to therapeutic resistance and tumor progression. While epigenetic mechanisms play a central role in driving this plasticity, the key regulators remain poorly understood, and developing effective therapeutic strategies targeting them has been challenging.


METHODS
We investigated the role of BRD2, a key regulator of NF-κB mediated mesenchymal (MES) transition, using GBM patient-derived xenograft (PDX) cell lines, CRISPR-mediated knock-in/knockout approaches, RNA-seq, and in vitro and in vivo modeling. BET inhibitors were employed to target MES gene expression and sensitize GBM to radiation therapy.


RESULTS
We found that PTEN loss induces RelA chromatin localization and acetylation-mediated recruitment of BRD2 to the MES gene promoters. BRD2 binding is essential for maintaining MES gene expression and phenotype. Genetic ablation or loss-of-function mutation of BRD2 bromodomains reverses MES transition, enhances radiation sensitivity, and improves survival in orthotopic xenograft models. Additionally, treatment with a brain-penetrant BD2-selective inhibitor suppresses the MES phenotype and increases radiation sensitivity of GBM stem cells in vitro.


CONCLUSION
Our study identifies BRD2 as a key mediator of MES transition in GBM, with its bromodomains playing a crucial role in driving cell state plasticity. Targeting BRD2 with BD2-selective inhibitors offers a promising therapeutic strategy to overcome radiation resistance and improve outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa778fbdf5d6872b2c63bfc97f4589c5f7dfb10" target='_blank'>
              BRD2 Bromodomain-Mediated Regulation of Cell State Plasticity Modulates Therapy Response in Glioblastoma.
              </a>
            </td>
          <td>
            Raghavendra Vadla, Brett Taylor, Yohei Miyake, Benjamin Lin, Daisuke Kawauchi, Shunichiro Miki, Nidhi Nathwani, Brandon M Jones, Yashpreet Kaur, Abhinaba Banerjee, Philip Pham, J. Tsang, Albert Baldwin, David A Nathanson, Donald P. Pizzo, C. Miller, Frank B Furnari
          </td>
          <td>2025-07-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="There is a lack of genetic biomarkers for predicting response to immune checkpoint blockade (ICB) therapy in cancer. The recent discovery that loss‐of‐function mutations in the gene encoding the protein phosphatase 2A (PP2A) scaffold protein PPP2R1A confer sensitivity to immune checkpoint blockade in ovarian clear cell carcinoma, therefore represents a breakthrough. Mechanistically, mutations in the PPP2R1A gene induce a strong interferon gamma response in tumor cells, which enhances infiltration of activated CD8+ T cells into the tumor. The activity of these T cells is then fortified by ICB. Furthermore, preclinical studies have shown that PP2A inhibition leads to the generation of neoantigens by disrupting RNA splicing, and PP2A inhibition can remodel the immune microenvironment of tumors to enhance responses to ICB. The finding that loss‐of‐function PPP2R1A mutations predict benefit from immunotherapy also suggests that pharmacological inhibition of PP2A may act synergistically with ICB therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8541b81fbcfabd62dc3e05a2a27e83fb1d85025" target='_blank'>
              A single gene mutation predicts response to immune checkpoint blockade in ovarian clear cell carcinoma
              </a>
            </td>
          <td>
            M. H. Dias, R. Bernards
          </td>
          <td>2025-07-30</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Despite the development of various effective anaplastic lymphoma kinase tyrosine kinase inhibitors (ALK-TKIs), therapeutic resistance remains a major challenge. Both on-target and off-target mechanisms have been identified as key contributors to resistance. With the popularization of genetic testing and the development of precision therapies, the prognosis and survival of patients with ALK-positive non-small cell lung cancer (NSCLC) have improved. However, even with second- and third-generation ALK-TKIs, overcoming resistance remains difficult. Resistance frequently arises during approved treatments, underscoring the need for further research to elucidate the molecular events and resistance mechanisms associated with ALK-positive lung cancer. The discovery of anaplastic lymphoma kinase (ALK) rearrangement as an actionable oncogenic driver in NSCLC has established a biomarker-driven treatment paradigm for advanced disease. This article summarizes current knowledge of the mechanisms of resistance to ALK-targeted therapy in lung cancer, including both primary and acquired mechanisms, treatment strategies following resistance, recent therapeutic advances, and the impact of the immune system and tumor microenvironment. A deeper understanding of ALK-targeted therapy resistance is critical for developing new treatment strategies and may provide important insights to guide the diagnosis, treatment, and management of patients with resistant ALK+ lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880963ca5d4f45d24c1ab8fad996bba2e1305930" target='_blank'>
              New advances in understanding the mechanisms and treatment challenges of ALK-targeted therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Mengle Long, Shixuan Peng, Qingyang Wen, Zhijian Yin, Xinwen Zhang, Haoyu Tan, Yun Xu, Yongjun Wu
          </td>
          <td>2025-08-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin Wlodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A. Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) remains incurable; gaps in our understanding of MM molecular pathogenesis and drugs’ resistance mechanisms are involved in the failure of therapies. This study aims to identify proteins significantly impacting MM patients’ response to commonly used therapeutic regimens. Bone marrow CD138+ selected plasma cells were isolated from patients who had achieved Response (Responders, R) and those who were Non‐Responders (NR) to their primary MM therapy. We used LC‐MS/MS to investigate the proteomic profile of MM samples, followed by bioinformatics analysis. We identified 1190 proteins, of which 230 showed a statistically significant difference between R and NR, with 27 proteins being upregulated and 203 downregulated in R compared to NR. Pathway enrichment analysis identified pathways related to the immune response and protein synthesis regulation, closely associated with MM progression and response to therapy. The results were validated through individual RNA dataset analysis, corroborating the differential expression of several proteins, including proteins associated with MM (e.g., MIF, ILF3) as well as novel findings (e.g., DCPS and SET). Collectively, proteomics data obtained from R and NR to MM therapy displayed significant changes in the immune system and protein synthesis regulation, supporting their potential role in progression and therapeutic response of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da55e0e2199913a64ea869848da54bd159ba5889" target='_blank'>
              Proteomic Analysis of Bone Marrow CD138+ Cells to Identify Proteins Associated With the Response of Multiple Myeloma Patients to Commonly Used Therapeutic Regimens
              </a>
            </td>
          <td>
            Foteini Paradeisi, A. Tserga, V. Lygirou, M. Makridakis, R. Stroggilos, Grigoris Georgiou, George M. Spyrou, Ioannis V. Kostopoulos, C. Liacos, Aikaterini Termentzi, M. Dimopoulos, O. Tsitsilonis, A. Vlahou, E. Kastritis, J. Zoidakis
          </td>
          <td>2025-08-01</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Liver kinase B1 (LKB1/STK11) is a key tumor suppressor that regulates cellular metabolism, epigenetic states, and multiple signaling pathways in prostate cancer (PCa). Recent studies reveal that both genetic and non-genetic LKB1 loss drives metabolic reprogramming, lineage plasticity, and treatment resistance, mainly through dysregulation of the AMP-activated protein kinase (AMPK)/mechanistic target of rapamycin (mTOR), signal transducer and activator of transcription 3 (STAT3), and Hedgehog (Hh) pathways. This review summarizes current evidence on LKB1-centered networks in PCa, highlighting the potential link between LKB1 inactivation, epigenetic remodeling, and aggressive tumor phenotypes. Special attention is given to recent studies on the impact of combined LKB1 and Phosphatase and Tensin Homolog (PTEN) loss on tumor differentiation. Finally, we discuss emerging therapeutic strategies aimed at the metabolic and epigenetic features of LKB1-deficient PCa, with a focus on the prospects for biomarker-driven precision medicine to address resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d293b1cfeae36ff64f881eea447225c95abdea07" target='_blank'>
              The role of LKB1 in prostate cancer: implications for tumor progression and therapy
              </a>
            </td>
          <td>
            Yuwei Liang, Hongliang Cao, Zhijun Tang, Shuxin Li, Gang Yang, Shuai Dong, Hao Du, Jinguo Wang
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Cancer remains a major global health challenge, with high prevalence and mortality rates emphasizing the urgent need for innovative treatment strategies. Although precision oncology offers tailored therapies based on genetic profiles, the clinical translation of genomic insights has been slow. Drug repurposing, using existing FDA‐approved drugs for new indications, presents a cost‐effective and time‐efficient alternative. This study investigates MGAM as a potential direct target of alpha‐glucosidase inhibitors in colorectal cancer (CRC), explores its biomarker potential, and evaluates gene expression patterns across diverse cancers. Using RNA‐Seq data from Recount3, Firebrowse, and gene set co‐expression analysis databases, we analyzed the differential expression of MGAM and its paralog MGAM2 across 33 cancer types. We examined mutation profiles, methylation status, survival impact, immune cell infiltration, and drug‐mRNA interactions. Validation was performed via real‐time PCR and whole‐exome sequencing (WES) in CRC patients. MGAM and MGAM2 showed differential expression across multiple cancers, with MGAM2 upregulated and MGAM downregulated in gastrointestinal tumors. Both genes were linked to key cancer‐related pathways, including metabolism, apoptosis, cell cycle regulation, and epithelial‐mesenchymal transition. MGAM exhibited frequent mutations and aberrant methylation in several cancers. Their expression correlated with immune cell infiltration and drug sensitivity, highlighting potential for therapy planning. Diagnostic modeling showed over 80% accuracy. In CRC patients, MGAM downregulation was confirmed in 64 samples, and WES revealed a novel MGAM mutation (rs2960746). These findings underscore MGAM and MGAM2 as promising biomarkers and therapeutic targets, supporting their relevance in advancing personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0138c7685899dec8663add8d3359263dd4a2f1f8" target='_blank'>
              Pan‐cancer analysis and oncogenic implications of MGAM and MGAM2: Toward precision oncology and drug repurposing in colorectal cancer
              </a>
            </td>
          <td>
            Rawaa Chasib Mezher, H. Azari, Reza Khayami, Hamid Fiuji, Farzaneh Alizadeh, L. Goshayeshi, Majid Rajabian, Ibrahim Saeed Gataa, Seyed Mahdi Hassanian, Amir Avan
          </td>
          <td>2025-08-27</td>
          <td>Journal of Cell Communication and Signaling</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Growth hormone (GH) signaling has been implicated in tumor progression and therapy resistance across multiple cancer types, yet its role in bladder cancer remains largely unexplored. In this study, we investigated the impact of GH and its receptor (GHR) on therapy resistance and disease progression in urothelial carcinoma (UC) through integrated transcriptomic and in vitro analyses. Transcriptomic profiling of The Cancer Genome Atlas bladder cancer cohort revealed that high tumoral GHR expression was associated with differential upregulation of genes involved in drug efflux, epithelial-to-mesenchymal transition (EMT), and extracellular matrix (ECM) remodeling. Notably, elevated GHR levels correlated with significantly reduced overall survival in patients with UC. In parallel, in vitro experiments demonstrated that GH promotes chemoresistance in UC cell lines via upregulation of ATP-binding cassette-containing (ABC) transporters and activation of EMT. GH also modulated ECM-remodeling-associated genes in a chemotherapy-dependent manner, including matrix metalloproteinases and tissue inhibitors of metalloproteinases. Importantly, these effects were abrogated by Pegvisomant, a GHR antagonist, indicating the functional relevance of GH/GHR signaling in the mediation of these phenotypes. Collectively, our findings support a mechanistic role for GH signaling in driving therapy resistance and tumor aggressiveness in bladder cancer and suggest GHR antagonism as a potential therapeutic strategy to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f8b97219a1f5f0a9e16e7142057f197a9d88cb" target='_blank'>
              Growth Hormone Signaling in Bladder Cancer: Transcriptomic Profiling of Patient Samples and In Vitro Evidence of Therapy Resistance via ABC Transporters and EMT Activation
              </a>
            </td>
          <td>
            Emily Davis, Lydia J. Caggiano, Hannah Munholland, R. Basu, D. Berryman, J. Kopchick
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="ABSTRACT Germline pathogenic variants (GPVs, ‘mutations’) causing inherited susceptibility to certain cancers (cancer susceptibility genes, CSGs) broadly belong to one of two main classes—loss of function variants in tumour suppressor genes (TSGs) or gain of function variants in proto‐oncogenes (an over‐simplification). Genomic analyses of tumours identify ‘driver mutations’ promoting tumour growth and somatic variants which contribute to ‘mutation signatures’ which, with histopathology, can be used to subclassify cancers with implications for causality and treatment. The identification of susceptible individuals is important, as they and their relatives may be at elevated risk of tumours, and this can influence optimal cancer treatment. Classically, cancer risk assessment utilises family history, lifestyle/environment factors, and any non‐neoplastic clinical findings, followed by genetic testing of high/moderate penetrance CSGs. In cancer cases not caused by highly penetrant CSGs, multiple variants conferring relatively small risks play a major role. These were discovered by genome‐wide association (GWAS) studies. The utility of polygenic risk scores (PRS) derived from multiple such variants for clinical risk profiling is being assessed. Access to genetic tests is improved by widening eligibility criteria for testing and empowering non‐genetic clinicians to identify CSG GPVs and manage carriers. This will contribute to expanding programmes of screening, prevention and early detection (SPED), with personalised surveillance and prophylactic interventions, and exploit knowledge of the molecular mechanisms of cancer susceptibility to develop novel cancer therapies. In some jurisdictions, population testing is being considered, but GPV penetrance in this setting can be unclear, and the public health implications are complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee21b45930d11565d54a3022b9abc97a84ec0804" target='_blank'>
              Inherited Susceptibility to Cancer: Past, Present and Future
              </a>
            </td>
          <td>
            Shirley V Hodgson, W. Foulkes, E. Maher, C. Turnbull
          </td>
          <td>2025-07-21</td>
          <td>Annals of Human Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic changes, such as DNA methylation, chromatin remodeling, and histone modifications, regulate gene expression without altering the DNA sequence. This review systematically analyzed over 500 studies including human cell line experiments (n>200), animal models (n>50), clinical cohort studies (n>100), and bioinformatics analyses retrieved from PubMed, Web of Science, and TCGA (The Cancer Genome Atlas). Studies increasingly show that genes involved in glucose and lipid metabolism, energy production, and modulation of metabolic hormones are regulated through epigenetic mechanisms. On the other hand, various metabolites participate in epigenetic modifications as coenzymes or substrates. Therefore, a greater understanding of the crosstalk between metabolism and epigenetics in cancer-related pathways could lead to the identification of key signaling molecules for targeted therapies, and raise the possibility of using dietary interventions to modulate epigenetic markers for individualized treatment. In this review, we have summarized the metabolic and epigenetic regulatory networks in cancer development, including glycolipid metabolic reprograming, the role of metabolites produced by the glut flora and tumor microenvironment, and key epigenetic drivers such as non-coding RNAs (ncRNAs). Data were curated from peer-reviewed articles, grounded in mechanistic studies using cell lines (SW480, MCF7 (Michigan cancer foundation-7)) and animal models (APC-mutant mice), with a focus on mechanistic studies, omics analyses, and translational research. Furthermore, we have discussed the potential of therapeutically targeting these pathways, along with the current challenges and future research directions, and a new strategy for reversing therapeutic drug resistance based on metabolism and epigenetic interaction was systematically explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447a75110b1468dd9ba2d78ee2627a7a828fee6" target='_blank'>
              Metabolism and epigenetics in cancer: toward personalized treatment
              </a>
            </td>
          <td>
            Xiaoman Zhang, Dequan Liu, Sulan Yin, Yaru Gao, Xiaorui Li, Guangzhen Wu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="CRISPR-Cas9 technology has rapidly advanced as a transformative genome-editing platform, facilitating precise genetic modifications and expanding therapeutic opportunities across various diseases. This review explores recent developments and clinical translations of CRISPR applications in oncology, genetic and neurological disorders, infectious diseases, immunotherapy, diagnostics, and epigenome editing. CRISPR has notably progressed in oncology, where it enables the identification of novel cancer drivers, elucidation of resistance mechanisms, and improvement of immunotherapies through engineered T cells, including PD-1 knockout CAR-T cells. Clinical trials employing CRISPR-edited cells are demonstrating promising results in hematologic malignancies and solid tumours. In genetic disorders, such as hemoglobinopathies and muscular dystrophies, CRISPR-Cas9 alongside advanced editors like base and prime editors show significant potential for correcting pathogenic mutations. This potential was affirmed with the FDA's first approval of a CRISPR-based therapy, Casgevy, for sickle cell disease in 2023. Neurological disorders, including Alzheimer's, ALS, and Huntington's disease, are increasingly targeted by CRISPR approaches for disease modelling and potential therapeutic intervention. In infectious diseases, CRISPR-based diagnostics such as SHERLOCK and DETECTR provide rapid, sensitive nucleic acid detection, particularly valuable in pathogen outbreaks like SARS-CoV-2. Therapeutically, CRISPR systems target viral and bacterial genomes, offering novel treatment modalities. Additionally, CRISPR-mediated epigenome editing enables precise regulation of gene expression, expanding therapeutic possibilities. Despite these advances, significant challenges remain, including off-target effects, delivery methodologies, immune responses, and long-term genomic safety concerns. Future improvements in editor precision, innovative delivery platforms, and enhanced safety assessments will be essential to fully integrate CRISPR-based interventions into standard clinical practice, significantly advancing personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef2c68c64e6d5ba82e4b21a0830f080c5ac867e1" target='_blank'>
              CRISPR Technology in Disease Management: An Updated Review of Clinical Translation and Therapeutic Potential.
              </a>
            </td>
          <td>
            Bahareh Farasati Far, Marziyeh Akbari, Mohammad Amin Habibi, Morteza Katavand, Sherko Nasseri
          </td>
          <td>2025-07-20</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Purpose: Although the combination of androgen deprivation therapy with docetaxel or abiraterone acetate and prednisone has become a standard treatment for metastatic hormone-sensitive prostate cancer (mHSPC) and has shown improved overall survival, a subset of patients still progress to castration-resistant disease. However, the underlying molecular features in these patients remain poorly understood.Materials and Methods: We performed single-cell RNA sequencing (scRNA-seq) on 12 tissue samples, including 2 mHSPC samples, 7 primary prostate cancer (PCa) samples, and 3 matched normal samples from 7 patients. Raw sequencing data was processed by Cell Ranger software (10X Genomics) and aligned to the human reference genome (GRCh38). A comprehensive analysis of samples from patients with mHSPC was also conducted and validated using a cohort of 52 patients with mHSPC.Results: Our results identified distinct subpopulations within luminal and mononuclear phagocyte (MNP) clusters characterized by proliferative activation associated with unfavorable clinical outcomes. Furthermore, we observed that the MNP cluster exhibited significant proliferation activity. To understand the underlying mechanisms associated with aggressiveness, we conducted cell-cell interaction, copy number variation and pseudotime analysis. Utilizing 87 network genes from scRNA-seq, we classified 52 mHSPC patients into 2 molecular subtypes and demonstrated their correlation with distinct transcriptomic profiles and survival outcomes. Importantly, a 14-gene signature derived from 3 distinct gene sets exhibited a strong association with patient survival and drug response. The prognostic value of our findings was further validated in large-scale cohorts comprising localized PCa, metastatic PCa, mHSPC, and metastatic castration-resistant PCa patients.Conclusion: This study provides valuable insights into the identification of high-risk patients, novel biomarkers, and potential therapeutic targets for individuals with mHSPC. Furthermore, the results in this study can serve as a basis for future investigations aimed at refining prognostic strategies and developing targeted therapies for patients with mHSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450db38b74f006113af6f667e3cedb5cf553cec7" target='_blank'>
              Determinant of Aggressive Phenotype in Metastatic Hormone-Sensitive Prostate Cancer Depends on an Intrinsic, Highly Aggressive Cell Cluster: Integrated Single-Cell RNA and Whole Transcriptomic Sequencing Analyses
              </a>
            </td>
          <td>
            Minyong Kang, B. Jee, Jiwoong Yu, Soohyun Hwang, Kyunghee Park, Kyung Yeon Han, Wan Song, H. Sung, H. Jeon, Byong Change Jeong, S. Seo, Se Hoon Park, Woong-Yang Park, S. Jeon
          </td>
          <td>2025-07-31</td>
          <td>Journal of Urologic Oncology</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Lenvatinib, a multiple-receptor tyrosine kinase inhibitor, has gained recent approval for its use as a first-line treatment of hepatocellular carcinoma (HCC). While lenvatinib demonstrates notable therapeutic efficacy, the drug resistance undermines its sustained tumor control potential. The restricted clinical utility of lenvatinib underscores the imperative necessity to elucidate the mechanisms underpinning drug resistance. We established lenvatinib-resistant cell lines and investigated the changes in their biological characteristics. Next-generation sequencing was performed to identify genes associated with lenvatinib resistance. Western blots were utilized to confirm the involvement of these genes. Using lentiviral technology, we generated cell lines with lowered nuclear receptor coactivator 5 (NCOA5), a pivotal drug resistance-related gene, to explore the underlying resistance mechanism. Moreover, we developed a subcutaneous HCC xenograft tumor model to explore strategies for reversing drug resistance. Our study showed that HCC cells acquire resistance to lenvatinib through the activation of NCOA5, thereby stimulating the NCOA5-Protein Kinase B-mammalian target of rapamycin (AKT-mTOR) axis. Furthermore, the clinical evaluation of HCC specimens established a correlation between the activation of the NCOA5 pathway and the response to lenvatinib treatment. Everolimus, an mTOR inhibitor, in combination with lenvatinib and everolimus, exerted significant synergistic effects against HCC in vivo and in vitro. HCC cells develop resistance to lenvatinib by activating the NCOA5-AKT-mTOR pathway. The combination therapy of lenvatinib with everolimus is a promising strategy to overcome acquired resistance, thereby enhancing the clinical efficacy of lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4645ca240d89c81caab05ab94ee824d557e8c782" target='_blank'>
              Inhibition of nuclear receptor coactivator 5 overcomes acquired lenvatinib resistance driven by protein kinase B-mammalian target of rapamycin signaling in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Hongyuan Zhou, Qin Zhang, Lu Yang, Zhao-Long Pan, Haijing Zheng, Zewu Zhang, Dongyang Li, Guangtao Li, Xiaomeng Liu, Xu Bao, Chen Liu, Wei Zhang
          </td>
          <td>2025-07-31</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer remains one of the most prevalent malignancies worldwide, and radiation therapy is a central component of its management. However, intrinsic or acquired resistance to radiation significantly compromises therapeutic efficacy. This systematic review aimed to identify and evaluate molecular mechanisms and interventions that influence radiation sensitivity in breast cancer models. A comprehensive PubMed search was conducted using the terms “breast cancer” and “radiation resistance” for studies published between 2002 and 2024. Seventy-nine eligible studies were included. The most frequently investigated mechanisms included the dysregulation of the PI3K/AKT/mTOR and MAPK signaling pathways, enhanced DNA damage repair via non-homologous end joining (NHEJ), and the overexpression of cancer stem cell markers such as CD44+/CD24−/low and ALDH1. Several studies highlighted the role of non-coding RNAs, particularly the lncRNA DUXAP8 and microRNAs such as miR-21, miR-144, miR-33a, and miR-634, in modulating radiation response. Components of the tumor microenvironment, including cancer-associated fibroblasts and immune regulators, also contributed to radiation resistance. By synthesizing current evidence, this review provides a consolidated resource to guide future mechanistic studies and therapeutic development. This review highlights promising molecular targets and emerging strategies to enhance radiosensitivity and offers a foundation for translational research aimed at improving outcomes in radiation-refractory breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a4cbc0cd4ca4c1e106a823d69b14920681deda3" target='_blank'>
              Molecular Mechanisms of Radiation Resistance in Breast Cancer: A Systematic Review of Radiosensitization Strategies
              </a>
            </td>
          <td>
            Emma Mageau, Ronan Derbowka, Noah Dickinson, Natalie Lefort, A. T. Kovala, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-07-24</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Understanding how genes influence drug responses is critical for advancing personalized cancer treatments. However, identifying these gene-drug interactions in a physiologically relevant human system remains a challenge, as it requires a model that reflects the complexity and heterogeneity among individuals. Here we show that large-scale CRISPR-based genetic screens, including knockout, interference (CRISPRi), activation (CRISPRa), and single-cell approaches, can be applied in primary human 3D gastric organoids to systematically identify genes that affect sensitivity to cisplatin. Our screens uncover genes that modulate cisplatin response. By combining CRISPR perturbations with single-cell transcriptomics, we resolve how genetic alterations interact with cisplatin at the level of individual cells and uncover an unexpected link between fucosylation and cisplatin sensitivity. We identify TAF6L as a regulator of cell recovery from cisplatin-induced cytotoxicity. These results highlight the utility of human organoid models for dissecting gene-drug interactions and offer insights into therapeutic vulnerabilities in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/306fe56d4d5b9a4de12be73a14dcdbee9c2f3137" target='_blank'>
              Large-scale CRISPR screening in primary human 3D gastric organoids enables comprehensive dissection of gene-drug interactions
              </a>
            </td>
          <td>
            Yuan-Hung Lo, Hudson T Horn, Mo-Fan Huang, Wei-Chieh Yu, Chia-Mei Young, Qing Liu, Madeline Tomaske, Martina Towers, Antonia Dominguez, M. Bassik, Dung-Fang Lee, L. S. Qi, Jonathan S. Weissman, Jin Chen, Calvin J. Kuo
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="Background and Objective Parental chromosomal structural variations (SVs) represent a primary genetic factor contributing to recurrent spontaneous abortion (RSA). Individuals carrying SVs with complex chromosomal rearrangements (CCRs) typically exhibit a normal phenotype but are at an increased risk of miscarriage. Current standard clinical detection methods are insufficient for the identification and interpretation of all SV types, particularly complex and occult SVs, thereby presenting a significant challenge for clinical genetic counseling. Leveraging the high-resolution capabilities of optical genome mapping (OGM) technology, this study aims to rapidly and accurately identify complex SVs in RSA couples. Furthermore, it seeks to conduct an in-depth analysis of the genetic information within the breakpoint regions, thereby providing a more comprehensive scientific foundation for genetic counseling of RSA couples at both the cellular and genetic levels. Material and Methods This study involved the selection of nine subjects from two families who underwent genetic counseling at our hospital. Family 1 comprised a couple with the wife as a SVs carrier, and both her parents and brother were simultaneously analyzed for chromosomal karyotype. Family 2 included a couple with the husband as the SVs carrier, with his parents also undergoing chromosomal karyotype analysis. For SVs carriers whose karyotype analysis did not elucidate the recombination pattern, optical genome mapping (OGM) technology was utilized for further investigation, followed by Sanger sequencing to validate the OGM findings. Results In Family 1, only the wife was identified as an SVs carrier. Initial chromosomal karyotype analysis suggested a karyotype of 46,XX,t (5; 6;8; 13; 15) (?). However, OGM analysis ultimately confirmed the karyotype as 46,XY,der (5)t (5; 13) (q35.2; q21.32), der (6)t (6; 8) (q25.3; q13.1)ins (6; 13) (q25.3; q21.32q21.33),der (8)t (6; 8) (q26; q13.1)ins (8; 13) (q13.1; q21.33q22.1),der (13)t (13; 15) (q21.32; q26.1)ins (13; 6) (q21.32; q25.3q26), der (15)t (5; 15) (q35.2; q26.1). Furthermore, OGM identified a novel translocation variant of the KIF7 gene that is associated with recurrent miscarriage. In Family 2, both the husband and his maternal parent were identified as SVs carriers. Nuclear type analysis revealed a karyotype of 46,XY,?t (1; 6) (q42; p21) (husband) and 46,XX,?t (1; 2) (p31.1; q24.1),?t (1; 6) (q42; p21) (mother). Through OGM detection and analysis, the final karyotype was determined to be 46,XY,ins (1; 6) (q42.2; p22.3p11.3) (husband) and 46,XX,der (1)t (1; 2) (p31.1; q24.1)ins (1; 6) (q42.2; p22.3p11.3), der (2) t (1; 2), der (6)ins (1; 6) (mother). Conclusion OGM technology facilitates the rapid and precise identification of complex chromosomal structural variations, effectively overcoming the limitations associated with traditional karyotype G-banding techniques in detecting intricate and cryptic SVs. This advancement substantially enhances the diagnostic rates of genetic etiology in patients experiencing RSA. The present study elucidates the specific manifestations of complex SVs using OGM technology, accurately pinpointing breakpoints and interpreting affected gene information. This provides novel reference approaches and evidence for disease assessment and genetic counseling in RSA patients. However, it is important to acknowledge certain limitations of this research: the study’s inclusion of only two RSA family cohorts (comprising nine participants) may limit the generalizability of its conclusions due to the small sample size, necessitating further validation through large-scale studies. Additionally, the causal relationship between KIF7 gene dysfunction and recurrent miscarriage remains to be experimentally verified in subsequent research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c823d1db805757e9a370ed45922b739e08e31bb6" target='_blank'>
              Analysis of complex chromosomal structural variants through optical genome mapping integrated with karyotyping
              </a>
            </td>
          <td>
            Xiaoxi Zhu, Huiling Zheng, Xue Wan, Hang Duan, Ying Qi, Weijia Tang, Fan Yang, Limei Yu
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) critically regulate tumor progression, angiogenesis, metastasis, and therapeutic resistance. This study investigated the characteristics of CAFs in glioblastoma (GBM) and developed a CAF-based risk signature to predict patient prognosis. The single-cell RNA sequencing (scRNA-seq) data were sourced from the Gene Expression Omnibus (GEO) database, whereas the bulk RNA-seq datasets were retrieved from The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA), respectively. The Seurat R package processed scRNA-seq data to identify CAF clusters using established markers. Prognostic genes were screened through univariate Cox regression, with Lasso regression constructing the final risk model. A nomogram incorporating clinical parameters was subsequently developed. Immunohistochemical validation was performed using the Human Protein Atlas (HPA) for the signature genes. The qRT-PCR validation was conducted in U251 and HA cells. ScRNA-seq analysis revealed five CAF clusters in GBM, including three prognostically relevant subtypes. Three key genes were refined to construct a risk signature functionally enriched in the the IL6_JAK_STAT3 signaling, P53 pathway, and inflammatory response. The signature correlated strongly with stromal and immune scores. Multivariate analysis confirmed risk signature independent prognostic value (P < 0.0001), followed by age (P = 0.005). The CAF-derived nomogram demonstrated superior predictive accuracy for 1-/2-year survival compared to clinical factors alone. The signature genes were validated in the HPA database and experimental validation. This study proposes CAF-derived molecular signatures as potential predictors of glioblastoma prognosis worthy of clinical validation. Systematic characterization of CAF heterogeneity may offer insights for interpreting GBM immunotherapy responses, providing a foundation for future exploration of stroma-targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00548-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d7a7b7ca4435c9538e14b7aed765c936957688" target='_blank'>
              Integrated single-cell and bulk transcriptomic profiling reveals cancer-associated fibroblast heterogeneity in glioblastoma and establishes a clinically actionable prognostic model and preliminary experimental validation
              </a>
            </td>
          <td>
            Wenhua Zhang, Yaxiong Li, Conghui Li, Qian Huang
          </td>
          <td>2025-08-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular senescence is a double-edged sword in cancer biology, functioning as both a tumor-suppressive mechanism and a driver of malignancy. Initially, senescence acts as a protective barrier by arresting the proliferation of damaged or oncogene-expressing cells via pathways such as oncogene-induced senescence and the DNA damage response. However, persistent senescence-associated secretory phenotype and metabolic reprogramming in senescent cells create a pro-inflammatory, immunosuppressive tumor microenvironment, fueling cancer progression, therapy resistance, and metastasis. This comprehensive review systematically examines the molecular mechanisms of senescence across diverse cancers, spanning digestive, reproductive, urinary, respiratory, nervous, hematologic, endocrine, and integumentary systems, and elucidates its context-dependent roles in tumor suppression and promotion. We highlight groundbreaking therapeutic innovations, including precision senolytics, senomorphics, and combinatorial strategies integrating immunotherapy, metabolic interventions, and epigenetic modulators. The review also addresses microenvironment remodeling and cutting-edge technologies for dissecting senescence heterogeneity, epigenetic clocks for biological age prediction, and microbiome engineering to modulate senescence. Despite their promise, challenges such as off-target effects, biomarker limitations, and cellular heterogeneity underscore the need for precision medicine approaches. Finally, we propose future directions to harness senescence as a dynamic therapeutic target, offering transformative potential for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9e8f75a567edbd4415256ca44b46f90ddbb5c2" target='_blank'>
              Cellular senescence in cancer: from mechanism paradoxes to precision therapeutics
              </a>
            </td>
          <td>
            Tiejun Feng, Fuda Xie, Leo M.Y. Lee, Zhiqiang Lin, Yifan Tu, Yang Lyu, Peiyao Yu, Jialin Wu, Bonan Chen, Ge Zhang, Gary M K Tse, K. To, Wei Kang
          </td>
          <td>2025-08-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, , Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc32b216cd93b5ffb3fc4906eb7ae996906a5999" target='_blank'>
              POFUT1 as a cancer biomarker: insights into its oncogenic mechanisms and clinical relevance.
              </a>
            </td>
          <td>
            Oumaima Mazour, Sébastien Legardinier, B. Badaoui, A. Germot
          </td>
          <td>2025-08-07</td>
          <td>Journal of molecular medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background While the RNA-binding protein HuR is implicated in individual cancers, its comprehensive diagnostic, prognostic, and immunological roles across diverse cancer types remain unexplored. Methods We performed an integrated pan-cancer analysis of HuR using public datasets. This encompassed expression profiling, survival analysis, diagnostic accuracy assessment, immune microenvironment characterization, and drug sensitivity prediction. We investigated HuR’s regulatory mechanisms through pathway correlation and differential gene expression analyses. Results HuR expression was consistently elevated across multiple cancers and correlated with poor patient prognosis. It demonstrated high diagnostic accuracy (>85%) via TMB/PD-L1 biomarkers. High HuR expression was associated with an immunosuppressive tumor microenvironment and reduced efficacy of immune checkpoint inhibitors, establishing it as a key immunoregulatory biomarker. HuR also predicted sensitivity to cell cycle inhibitors and other pathway-targeted drugs. Mechanistically, HuR drives malignancy by dysregulating core processes: cell cycle progression, immune evasion, and cellular metabolism. Conclusions Our pan-cancer analysis establishes HuR as a consistently upregulated oncogenic driver across malignancies, functioning as a potential universal biomarker for prognosis and diagnosis. Its critical roles in modulating the immune response and predicting therapeutic sensitivity highlight its importance for personalized cancer treatment strategies. HuR orchestrates tumorigenesis and malignant progression by integrally regulating vital cellular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee36dd4154d3b1fc037a47fa32d256e8c8ba2af" target='_blank'>
              Integrated pan-cancer analysis of RNA binding protein HuR investigates its biomarker potential in prognosis, immunotherapy, and drug sensitivity
              </a>
            </td>
          <td>
            Jian Peng, Jichuan Quan, Xiaoru Wang
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive malignancy of the bile ducts, and resistance to gemcitabine, a first-line chemotherapy, significantly complicates treatment. Despite extensive research, the molecular mechanisms underlying gemcitabine resistance in iCCA are not fully understood. This study aims to identify key genes associated with gemcitabine resistance in iCCA, investigate the role of lactylation, and propose potential therapeutic targets.A comprehensive bioinformatics analysis was conducted using publicly available transcriptomic data from gemcitabine-resistant iCCA cell lines and patient samples. Differential expression analysis was performed to identify upregulated and downregulated genes. GSEA were used to explore relevant molecular pathways. Immune landscape analysis was carried out using CIBERSORT to assess immune cell infiltration in the tumor microenvironment. Key resistance-related genes were identified through Lasso, RF, and SVM-REF analyses. ITGB4 function was further validated in vitro by siRNA knockdown in HUCCT1 and RBE cells, followed by cell viability and apoptosis assays with or without gemcitabine treatment.Pathway analysis revealed the involvement of cell cycle regulation, DNA replication, and p53 signaling in gemcitabine resistance. The high group associated with resistance showed significantly worse survival outcomes, with a positive correlation between resistance and lactylation levels. Immune landscape analysis indicated altered immune cell infiltration, including increased M2 macrophages and decreased CD8+ T cells in the high group. Key resistance-related genes, including CDC20, H2AX, HK2, and ITGB4, were identified as critical in drug resistance. Experimentally, ITGB4 knockdown markedly enhanced gemcitabine’s antiproliferative and pro-apoptotic effects on cholangiocarcinoma cells, supporting its role in mediating resistance. Molecular docking revealed Dioscin and Deacetyllanatoside C as potential ITGB4-interacting compounds.This study sheds light on the molecular mechanisms of gemcitabine resistance in iCCA, emphasizing lactylation’s role and the significance of immune modulation. ITGB4 is identified as a promising therapeutic target, and the findings suggest that targeting these genes could help overcome resistance in iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea10bda53b7cbe7c506a45067aed62ce5a1a0fe" target='_blank'>
              Integrated transcriptomic analysis identifies lactylation-linked gemcitabine resistance and therapeutic targets in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Wenwei Xie, Jialiang Hu, Hanmei Xu
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="AIDS progression varies widely among individuals, influenced by genetic and molecular factors affecting immune responses and viral replication. Identifying genetic variants associated with disease progression can reveal host-pathogen interactions and potential therapeutic targets. This study analyzes genome-wide association study (GWAS) data and performs functional enrichment analyses to explore key genetic determinants of AIDS progression. GWAS data were analyzed to identify single nucleotide polymorphisms (SNPs), significantly associated with AIDS progression. Functional enrichment through Gene Ontology (GO), KEGG pathways and CellMarker databases classified genes by biological functions, immune cell associations and metabolic pathways. Unsupervised machine learning algorithms, including hierarchical clustering and principal component analysis (PCA) were applied to validate patterns.

Significant SNPs were identified in genes such as PARD3B, LTF, CCRL2, NBPF14, DGKI, RPH3AL and H2AFY, linked to immune regulation, inflammation and viral replication control. PARD3B was associated with slower disease progression while DGKI highlighted metabolic involvement through phosphatidylinositol signaling. Hierarchical clustering revealed natural SNP groupings and PCA confirmed distinct genomic patterns influencing AIDS progression. This study highlights genetic variants influencing AIDS progression and identifies potential biomarkers for future therapeutic strategies. Validation and further exploration of personalized treatment approaches are recommended.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34bf558f9d01070512eaeba6ab1b07218bbc9e77" target='_blank'>
              Genetic and Molecular Insights into AIDS Progression: A Genome-Wide Association Data-Driven Analysis
              </a>
            </td>
          <td>
            Deepthi C. Sravana, Eddula Venkata Ravikanth, U. Adiga, S. Vasishta, B. S. Sindhu
          </td>
          <td>2025-07-31</td>
          <td>Research Journal of Biotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Vascular diseases such as atherosclerosis, aneurysms, and peripheral arterial disease remain leading causes of morbidity and mortality, with current treatments primarily managing symptoms rather than addressing underlying molecular drivers. Gene therapy offers a promising avenue for targeted intervention, and recent advances in multi-omics approaches-including genomics, transcriptomics, and epigenetics-are enhancing the precision and efficacy of these therapies. High-throughput sequencing and integrative omics analyses have facilitated the identification of causal genes, non-coding RNAs, and epigenetic regulators involved in vascular pathology. This review examines how multi-omics frameworks inform gene therapy design, from genomic editing of cardiovascular disease loci to transcriptome-guided RNA therapies and epigenetic modulation of disease states. We highlight emerging applications such as CRISPR-based interventions, RNA therapeutics, and individualized precision medicine strategies. Additionally, we address analytical challenges, implementation hurdles, and ethical considerations in translating multi-omics-driven gene therapies into clinical practice. By integrating systems biology and advanced computational methods, the convergence of multi-omics and gene therapy holds transformative potential for vascular medicine, offering new avenues for disease modification and patient-specific therapeutic solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a11c68f0e7401720c5e3271796816c820e6e436" target='_blank'>
              Multi-Omics Approaches in Gene Therapy for Vascular Diseases: Bridging Genomics, Transcriptomics, and Epigenetics.
              </a>
            </td>
          <td>
            Jianxin Dong, Ming Sun, Yanmei Li, Zhilei Xie
          </td>
          <td>2025-08-05</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common primary bone cancer of mesenchymal origin in children and young adolescents, remains a challenge due to metastasis and resistance to chemotherapy. It displays severe aneuploidy and a high mutation frequency which drive tumor initiation and progression; however, recent studies have highlighted the role of epigenetic modifications as a key driver of OS pathogenesis, independent of genetic mutations. DNA and RNA methylation, histone modifications and non-coding RNAs are among the major epigenetic modifications which can modulate the expression of oncogenes. Abnormal activity of these mechanisms contributes to gene dysregulation, metastasis and immune evasion. Therapeutic targeting against these epigenetic mechanisms, including inhibitors of DNA and RNA methylation as well as regulators of RNA modifications, can enhance tumor suppressor gene activity. In this review, we examine recent studies elucidating the role of epigenetic regulation in OS pathogenesis and discuss emerging drugs or interventions with potential clinical utility. Understanding of tumor- specific epigenetic alterations, coupled with innovative therapeutic strategies and AI-driven biomarker discovery, could pave the way for personalized therapies based on the molecular profile of each tumor and improve the management of patients with OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69115f288109b08c5dc3c9431da8115c0747b13f" target='_blank'>
              Epigenetic Modifications in Osteosarcoma: Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Maria A. Katsianou, Dimitrios Andreou, Penelope Korkolopoulou, E. Vetsika, C. Piperi
          </td>
          <td>2025-07-28</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Nucleosome assembly protein 1-like 5 (NAP1L5), a critical regulator of gene transcription and nucleosome assembly, has been implicated in the progression and poor prognosis of various cancers. However, its specific role and molecular mechanisms in acute myeloid leukemia (AML) remain largely unexplored. Methods To identify key genes associated with AML, we analyzed gene expression profiles from AML patients and healthy controls using microarray datasets obtained from the GEO database. Differential expression analysis was performed to identify differentially expressed genes (DEGs), among which NAP1L5 emerged as a critical candidate based on its expression patterns and prognostic relevance, and we validated NAP1L5 expression in clinical AML samples. To elucidate the functional role of NAP1L5, we conducted Gene Set Enrichment Analysis (GSEA) and Gene Ontology (GO) analysis, which revealed its involvement in specific signaling pathways and biological processes. Furthermore, we constructed an interaction network and predictive model for NAP1L5, complemented by an assessment of its role in immune infiltration and drug sensitivity. Finally, we conducted in vitro experiments to explore its biological functions and underlying molecular mechanisms. Results In AML, elevated expression of NAP1L5 was significantly associated with reduced overall survival, underscoring its prognostic relevance. GSEA revealed that NAP1L5 was prominently enriched in pathways related to apoptosis and DNA replication. GO analysis further indicated that its co-expressed genes were closely linked to autophagy and stress response mechanisms. Interaction network analysis revealed that NAP1L5 engages in complex regulatory interactions with multiple genes, miRNAs, transcription factors (TFs), and RNA-binding proteins (RBPs). Notably, high NAP1L5 expression correlated with increased infiltration of resting CD4+ memory T cells, implicating its potential influence on the tumor immune microenvironment. A predictive model integrating NAP1L5 expression and clinical AML features exhibited robust prognostic utility. Drug sensitivity analysis identified NAP1L5 overexpression as a marker of resistance to Zibotentan, along with associations with 49 additional therapeutic agents. In vitro functional assays demonstrated that NAP1L5 overexpression promoted cellular proliferation, migration, and colony formation while concurrently inhibiting apoptosis, highlighting its oncogenic potential in AML pathogenesis. Conclusions NAP1L5 emerges as a promising prognostic biomarker and therapeutic target in AML, offering potential for improved patient outcomes and precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be458e5cdd1834f6b055dae4c08b4d9e77e28c87" target='_blank'>
              NAP1L5 in acute myeloid leukemia: a prognostic biomarker and potential therapeutic target
              </a>
            </td>
          <td>
            Meng Wang, Zhibin Xie, YuanYuan Tan, Yan Zhou, TingTing Zhang, Yuqing Du, Huan Wu, Lili Zhou, Jian Ge
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Epidermal growth factor receptor (EGFR) inhibitors are used to treat some patients with metastatic colorectal cancer, but many tumors do not respond or develop resistance during therapy. This review explores how understanding the molecular mechanisms of resistance can help guide treatment decisions. We highlight the concept of negative hyperselection, which uses extended genetic profiling to exclude tumors unlikely to benefit from EGFR inhibitors. In addition, we discuss the role of circulating tumor DNA, a blood-based test that monitors tumor evolution in real time. Together, these strategies form a precision oncology platform that supports more personalized and effective care. While these advances offer promise, challenges remain regarding cost, access, and demonstrating meaningful survival benefit in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716984fb2012cf7cb0f4128ef11a21a1a21d5293" target='_blank'>
              Anti-EGFR Therapy in Metastatic Colorectal Cancer: Identifying, Tracking, and Overcoming Resistance
              </a>
            </td>
          <td>
            Luís Felipe Leite, M. Noronha, Júnior Samuel Alonso de Menezes, Lucas Diniz da Conceição, Luiz F. Costa Almeida, Anelise Poluboiarinov Cappellaro, M. Belotto, Tiago Biachi de Castria, R. D. Peixoto, T. B. C. Megid
          </td>
          <td>2025-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Treatment resistance is a major hurdle in diffuse large B-cell lymphoma (DLBCL) therapy. Here, we assessed the relevance of the Musashi (MSI) RNA-binding protein family for DLBCL treatment efficacy. As important gene expression regulators, these proteins have previously been associated with tumorigenesis, treatment failure, and reduced survival in other malignancies, making them promising candidates for assessment in the context of DLBCL outcome and therapy resistance. Methods We first leveraged publicly available gene expression studies to determine expression and prognostic relevance of MSI1 and MSI2 in DLBCL. We then characterized MSI2 co-expressed therapy-relevant signaling. After performing MSI2 knockdown experiments we investigated subsequent effects on DLBCL gene expression in vitro using qPCR, Western blot, protein arrays, and flow cytometry. Finally, cell viability assays and clonogenic assessments were used to assess resistance to radiation, vincristine, and doxorubicin chemotherapy. Results MSI2 was overexpressed and prognostically unfavorable in univariable and multivariable analyses in DLBCL while MSI1 showed very low expression. High MSI2 expression was associated with increased stemness and DNA repair signaling. MSI2 knockdown led to a loss of stemness-associated markers and compromised DNA repair protein activation while increasing radiation-induced DNA double-strand break levels. Cell survival after either radiotherapy, vincristine or doxorubicin chemotherapy was impaired after MSI2 knockdown in follow-up analyses, suggesting a radio- and chemosensitizing effect. Discussion We propose that MSI2, a prognostic marker, may modulate the susceptibility of DLBCL towards genotoxic therapy. Suppressing MSI2 may hold promise to sensitize DLBCL to DNA-targeted treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a873df352503c3224c8f6ec89d594d3543c9ab8" target='_blank'>
              Prognostic marker Musashi-2 modulates DNA damage response and radioresistance in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Timo Habig, Lasse Reichstein, Kathrin A. Brücksken, M. Sicking, Jan Labisch, M. Oertel, Eberhard Korsching, Georg Lenz, Stephan Hailfinger, Burkhard Greve, Fabian M Troschel, H. Eich
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The serine protease kazal-type inhibitor (SPINK) family is central to the regulation of proteolytic function, the establishment of physiological homeostasis, and the development of many disease states, including cancer. Emerging research has identified that members of the SPINK family are commonly overexpressed in most malignancies and are deeply implicated in pivotal oncogenic pathways like cell growth, epithelial-to-mesenchymal transition (EMT), metastasis, and drug resistance. This review provides an in-depth examination of structural and functional characteristics of SPINK proteins and their involvement in the onset and development of multiple cancers, which include prostrate, pancreatic, and colorectal carcinomas. Significantly, SPINK proteins regulate major signalling pathways, including EGFR, NF-κB, and MAPK, highlighting their role as prognostic biomarkers and therapeutic targets. The review underscores the most recent advancements in therapeutic strategies for SPINK-related pathways and outlines the bottlenecks that have restricted their use in the clinic. By integrating current evidence, this work signals the potential of SPINK proteins as good precision oncology candidates with novel options for cancer prognosis, treatment, and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a4480f232504606aade84651cdbbefe0d8d0851" target='_blank'>
              The SPINK Protein Family in Cancer: Emerging Roles in Tumor Progression, Therapeutic Resistance, and Precision Oncology
              </a>
            </td>
          <td>
            Zitin Wali, Neha, Anas Shamsi, Syed Tasqeruddin, Saleha Anwar
          </td>
          <td>2025-08-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Engineering mammalian cells with synthetic circuits drives innovation in next-generation biotherapeutics and industrial biotechnology. However, applications often depend on cellular productivity, which is constrained by finite cellular resources. Here, we harness computational biology to identify drugs that boost productivity without additional genetic modifications. We perform RNA-sequencing on cells expressing an incoherent feed-forward loop (iFFL), a genetic circuit that enhances operational capacity. To find drugs that mimic this effect, we use DECCODE (Drug Enhanced Cell COnversion using Differential Expression), an unbiased method that matches our transcriptional data with thousands of drug-induced profiles. Among the compound candidates, we select Filgotinib, that enhances expression of both transiently and stably expressed genetic payloads across various experimental scenarios and cell lines, including AAV and lentivirus transduction. Our results reveal cell-specific responses, underscoring the context dependency of small-molecule treatments. Altogether, we present a versatile tool for biomedical and industrial applications requiring enhanced productivity from engineered cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c75449ffa28b6703ced9f8f9d9192404f80572" target='_blank'>
              Computational identification of small molecules for increased gene expression by synthetic circuits in mammalian cells
              </a>
            </td>
          <td>
            M. Pisani, F. Calandra, A. Rinaldi, F. Cella, F. Tedeschi, I. Boffa, D. Vozzi, N. Brunetti-Pierri, A. Carissimo, F. Napolitano, V. Siciliano
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a major cause of cancer-related deaths worldwide, often diagnosed at advanced stages with limited treatment options. Despite advancements in systemic therapies, including the use of lenvatinib, the survival rate for advanced HCC remains low due to drug resistance and tumor heterogeneity. Method This study employed single-cell sequencing and spatial transcriptomics to investigate intra-tumor heterogeneity and identify subpopulations of malignant cells with inherent drug resistance. Tumor samples were obtained from patients undergoing lenvatinib, and differential analysis was performed to identify biomarkers associated with drug resistance. Results Analysis revealed significant differences in treatment response between different HCC nodules within the same patient, indicating intra-tumor heterogeneity. Single-cell sequencing and spatial transcriptomics identified distinct cell clusters within tumors, with specific subpopulations showing inherent resistance to lenvatinib. Differential analysis identified 17 common upregulated genes, including PCK1 and ALDH1A1, which were significantly associated with drug resistance. Pseudotime analysis further supported the role of PCK1 and ALDH1A1 as potential biomarkers of inherent drug resistance in HCC. This study highlights the importance of tumor heterogeneity and identifying biomarkers associated with drug resistance in HCC. Conclusions The findings suggest that PCK1 and ALDH1A1 may serve as potential biomarkers for predicting treatment response and guiding therapeutic strategies in HCC patients. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03481-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5d2cb3017ef7eb96af09bc83b104b7725e1bc2c" target='_blank'>
              PCK1 and ALDH1A1 are identified as biomarkers for inherent drug resistance in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei Chen, Cheng Zhou, Gao Liu, Xinyu Wang, Yong Yi, Yang Xu, Shuangjian Qiu
          </td>
          <td>2025-08-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b37ee620d5697800e8d511cd10bb157702e6caa" target='_blank'>
              Clinical and analytical validation of MI Cancer Seek®, a companion diagnostic whole exome and whole transcriptome sequencing-based comprehensive molecular profiling assay.
              </a>
            </td>
          <td>
            V. Domenyuk, Kasey Benson, Peggy Carter, Dan Magee, Jian Zhang, Nitin Bhardwaj, Hongseok Tae, James Wacker, Foram Rathi, Siobhan Miick, Aastha Kohli, Joshua Carroll, Lori Cuyugan, Evelyn Perez, Wayeesha Zhang, John Collins, Patrick Kennedy, Jeremy Ellis, A. Stark, Andrey Loskutov, Brittany Cuttone, Blake Taylor, Rebecca A Feldman, Jeff Swenson, David Bryant, R. Hahn-Lowry, Raunaq Kaushal, Jennifer R. Ribeiro, J. Abraham, M. Radovich, George W Sledge, M. Oberley, D. Spetzler
          </td>
          <td>2025-08-13</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma (NB) is the most common extracranial solid tumor in early childhood and frequently presents with bone marrow (BM) metastasis, particularly in high-risk cases. Metastatic NB cells residing in the BM exhibit distinct biological behaviors and are closely associated with treatment resistance and poor prognosis. Emerging evidence suggests that metabolic reprogramming is a hallmark of NB progression; however, its regulatory landscape within the bone marrow microenvironment remains poorly understood. This study aimed to systematically elucidate the molecular mechanisms underlying metabolic reprogramming in bone marrow–infiltrating metastatic NB cells by integrating single-cell transcriptomic data, bulk RNA-sequencing profiles, and in vitro functional validation. Single-cell RNA-sequencing data from 17 bone marrow aspirates of NB patients—with and without marrow infiltration—were retrieved from the GEO database, while bulk RNA-seq data from 155 NB tumor samples were obtained from the TARGET database. A suite of bioinformatics tools, including Seurat, Harmony, CellChat, and SCENIC, was employed to perform multi-dimensional analyses such as cell subtype annotation, intercellular communication mapping, and transcription factor regulon construction. AUCell scoring, differential gene expression analysis, random survival forest modeling, and Kaplan–Meier survival analysis were conducted to identify key genes involved in metabolic reprogramming and to characterize their immune infiltration features. Gene Set Enrichment Analysis (GSEA), Gene Set Variation Analysis (GSVA), and single-cell pathway scoring were applied to predict the metabolic and immune pathways associated with these candidate genes. Furthermore, co-expression network analysis was used to evaluate the relationships between candidate genes and known NB regulatory factors. Finally, in vitro experiments were carried out to validate the roles of selected genes in regulating NB cell proliferation, migration, invasion, and mitochondrial function. Single-cell transcriptomic analysis of bone marrow–derived NB samples identified nine distinct cell subpopulations with diverse intercellular communication networks. Among these, the ligand–receptor pair MDK–NCL emerged as a key mediator of cell–cell signaling. Regulatory network analysis revealed five critical regulons—JUND, JUNB, FOS, E2F1, and KLF16—closely associated with metabolic reprogramming in NB. The “Neuroblastoma cell” cluster displayed markedly elevated metabolic activity. Through integrated analyses, five core metabolic reprogramming genes (MRPL21, NHP2, RPL13, RPL18A, and RPL27A) were identified and shown to be significantly associated with poor prognosis. High expression of these genes correlated with an immunosuppressive tumor microenvironment, characterized by reduced infiltration of monocytes, M1 macrophages, and T cells. Functional enrichment analysis revealed that these genes were primarily enriched in oxidative phosphorylation, MYC targets, PI3K-Akt, and p53 signaling pathways. Co-expression network analysis further demonstrated that MRPL21 and NHP2 positively correlated with known NB regulatory genes TP53, NRAS, and NKIF1B. In vitro assays confirmed that MRPL21 knockdown significantly impaired NB cell proliferation, migration, invasion, and mitochondrial oxidative phosphorylation. This study identified five key genes involved in metabolic reprogramming of bone marrow–infiltrating NB cells, which are closely associated with immunosuppressive microenvironment formation and enrichment in tumor-associated metabolic pathways. Among these, MRPL21 plays a pivotal role in regulating NB cell proliferation and mitochondrial function, underscoring its potential as a promising therapeutic target. These findings suggest that integrated strategies targeting both tumor metabolism and the immune microenvironment may provide new avenues for the treatment of high-risk, metastatic NB. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13626-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b395447d21c8867ac393546695c70f8056f2bca" target='_blank'>
              Integrated single-cell and transcriptomic analysis of bone marrow-derived metastatic neuroblastoma reveals molecular mechanisms of metabolic reprogramming
              </a>
            </td>
          <td>
            Jing Chu, Rong Qin, Shu-Jing Wang, Qiang Wang, Qiang Wu
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite the high sensitivity of estrogen receptor positive (ER+) breast cancer to endocrine therapy, many patients have primary resistance or develop resistance to endocrine therapies. Acquired resistance to endocrine therapy is a great challenge in the treatment of ER+ breast cancer patient. Here we showed that Response Gene to Complement (RGC)-32 expression is higher in breast cancer than paired normal tissues, which was a poor predictive factor. RGC-32 overexpression resulted in tamoxifen resistance, whereas knockdown of RGC-32 in tamoxifen-resistant cells restored tamoxifen sensitivity. Tamoxifen resistance mediated by RGC-32 was shown to be partially dependent on FoxM1 expression. Mechanistically, RGC-32 could activated PI3K signaling pathway, and then enhanced estrogen receptor alpha (ERα) activity. ERα activation is essential for RGC-32-mediated the expression of FoxM1. These data support that targeting RGC-32 could effectively mitigate cancer progression and tamoxifen resistance, offering a complementary therapeutic approach to reduce acquired endocrine resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8ff0c9ab0a8e724cafedf2beedbe10119c0d4" target='_blank'>
              Response Gene to Complement 32 promotes cell proliferation and tamoxifen resistance in breast cancer via elevated FoxM1 expression
              </a>
            </td>
          <td>
            Xinlei Li, Yan Liu, Zhiqian Wang, Xiaocui Bu, Yu Wang, Wei Zhang, Peng Zhao
          </td>
          <td>2025-07-28</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861fd3ffe0c8c1bb6ed63c62ce2660ccaa3e238b" target='_blank'>
              miRNA-mediated resistance mechanisms in prostate cancer: implications for targeted therapy and metastatic progression.
              </a>
            </td>
          <td>
            Mostafa M. Mostafa, M. K. A. El-Aziz, D. Ellakwa
          </td>
          <td>2025-08-29</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background
Copy number variations (CNVs) play an important role in human genetic disorders. Detection of exon-level CNVs is crucial for accurate clinical diagnosis. The CytoScan XON Array, a high-resolution microarray, was recently developed to detect exonic CNVs of various genes.


Methods
We evaluated the clinical performance of the CytoScan XON Array using 59 patient samples with previously identified CNVs, confirmed via methods including multiple ligation-dependent probe amplification (MLPA), gene-dose PCR, and mRNA assay. Concordance between CytoScan XON and orthogonal methods was evaluated in target regions, and diagnostic utility was compared with that of genome sequencing (GS)-based CNV calling tools through analysis of false-positive CNVs in non-target genomic regions.


Results
For target regions, the CytoScan XON Array achieved concordance rates of 89.8% and 92.5% at the exon and gene levels, respectively, for all CNV calls. Concordance was higher for multi-exon CNVs (100%) than that for single-exon CNVs (82.6%, P =0.03). For non-target regions, false-positive CNV calls were reduced to fewer than 0.01 per gene per person through filtering strategies. The array exhibited false-positive detection rates within dosage-sensitive genes comparable with those of GS-based tools.


Conclusions
The CytoScan XON Array, a reliable tool for detecting exon-level CNVs in target regions, can serve as a complementary approach to GS-based CNV calling tools for genome-wide CNV screening with high resolution. However, its performance for single-exon CNVs requires further optimization. Cross-validation with GS-based CNV calling tools is recommended to improve diagnostic accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a08a61a6f8683a7068bd9bbb76ec77849b8894" target='_blank'>
              High-resolution Chromosomal Microarray with Diagnostic Potential for Detecting Exon-level Copy Number Variations Using Targeted and Non-targeted Approaches.
              </a>
            </td>
          <td>
            Yeseul Kim, Jee-Soo Lee, Boram Kim, M. Kim, Sung Im Cho, Seung-Won Chae, H. Shin, Hoyeon Lee, Ji Yeon Kim, Moon-Woo Seong
          </td>
          <td>2025-07-30</td>
          <td>Annals of laboratory medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most common and aggressive brain derived tumor. It often shows genetic alterations in kinase signaling pathways, such as the Pi3K/mTOR and RAS/MAPK pathways, that frequently converge onto oncogenic processes. However, it is unknown to what extend co-vulnerabilities exist within this network and which kinase drug targets are promising for GB treatment. We investigated the drug sensitivity of GB cell line models to monotherapy and synergy effects in dual combination therapy to targeting components of Pi3K/mTOR and RAS/MAPK pathways. In addition, we examined cell line drug sensitivities in relation to their individual genetic tumor driving lesions (i.e. NF1 alterations as well as transcriptomic defined GB subtypes). Synergy levels were correlated to in-lab generated phospoproteomic data. Lastly, serial or simultaneous addition of MEK and mTOR inhibitors were investigated in longitudinal experiments. Dual inhibition of MEK and mTOR resulted in synergistic effects, which associated with NF1-deficiency. Strong synergy effects was also associated with the mesenchymal subtype. Dual inhibition of MEK and mTOR led to prolonged growth inhibition in GB spheroids. In addition, sequential drug treatment resulted in similar growth inhibitory effects compared to simultaneous combination therapies. Our findings highlight the potential of dual inhibition strategies targeting multiple kinases for the treatment of GB, particularly in NF1-deficient and mesenchymal tumors, the most lethal subtype of GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049087437ba8fac6737126641a3cf7ff0b99a6f7" target='_blank'>
              Combined inactivation of MEK and mTOR can lead to synergistic cell death in glioblastoma models and associates with NF1-deficiency and a mesenchymal subtype.
              </a>
            </td>
          <td>
            F. Cornelissen, Yoran Broersma, R. Narayan, Rogier Dik, S. Piersma, Richard de Goeij-de Haas, T. Pham, D. Noske, W. Vandertop, Connie R. Jimenez, Bart A Westerman
          </td>
          <td>2025-08-01</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related morbidity and mortality worldwide. Its marked heterogeneity - encompassing molecular subtypes, histological characteristics, and variable therapeutic responses - continues to pose persistent clinical challenges Although advances in surgery, hormone therapy, chemotherapy, and targeted therapies have significantly improved patient outcomes, issues such as therapeutic resistance and disease relapse are still common, underscoring the need for novel molecular targets. Within this context, non-coding RNAs (ncRNAs) have emerged as pivotal regulators of breast cancer biology and hold promise as diagnostics and therapeutic agents. These non-protein-coding RNA molecules include diverse subclasses, such as long non-coding RNAs (lncRNAs), circular RNAs (circRNAs), and small non-coding RNAs (sncRNAs), each characterized by distinct structural features and biological functions. Mounting evidence implicates ncRNAs in key oncogenic processes - such as tumor initiation, progression, metastasis, immune evasion, and treatment resistance - often in a subtype-specific manner. Importantly, ncRNA expression profiles differ significantly across BC subtypes, and their stability in body fluids underscores their potential utility in liquid biopsy-based diagnostics. This review provides an integrated overview of the multifaceted roles of ncRNAs in BC, emphasizing their mechanisms of action, contributions to tumor heterogeneity, and translational potential as both biomarkers and therapeutic targets. Understanding ncRNAs complexity and context-specific functions may pave the way toward more precise, personalized interventions for BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5de9a0dd3e6ab3c3ee75baeac6b79a2d1c7dff" target='_blank'>
              Non-coding RNAs as novel biomarkers and therapeutic targets in breast cancer
              </a>
            </td>
          <td>
            V. Barbi, Sara De Martino, A. Aiello, M. Gottardi Zamperla, Sara Negri, Luca Cis, V. Pecci, Simona Nanni, A. Farsetti, Fabio Martelli, C. Gaetano, S. Atlante
          </td>
          <td>2025-08-29</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens.
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="TP53 mutations drive oncogenesis and therapeutic resistance in myelodysplastic syndromes (MDSs) and acute myeloid leukemia (AML), impairing p53-regulated functions such as apoptosis, immune surveillance, and genomic stability, leading to immune evasion and metabolic reprogramming. The tumor microenvironment in TP53-mutated MDS and AML fosters leukemic progression through cytokine dysregulation, altered metabolism, and immune suppression. Current therapies, including chemotherapy and hypomethylating agents, offer limited efficacy, resulting in poor overall survival rates for these high-risk patients. However, novel therapeutic approaches provide promising avenues, including MDM2 inhibitors, p53-reactivating agents, pathway-targeted inhibitors (Hedgehog, Wnt, NF-κB), immune modulation (checkpoint inhibitors, CAR-T therapy), metabolic interventions (fatty acid metabolism, glycolysis), and gene-editing technologies (CRISPR/Cas9, base editing). This review explores the mechanisms of immune dysfunction in TP53-mutated MDS and AML while highlighting emerging therapeutic strategies, emphasizing the integration of targeted, metabolic, and immune-modulating therapies as a transformative approach to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd676eb4edce0f1928207f8f6cec1ec2c9dedb2" target='_blank'>
              TP53-mutated MDS and AML: immune dysregulation, tumor microenvironment, and emerging therapeutic strategies
              </a>
            </td>
          <td>
            Marwah Albakri
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Advances in genetic and transcriptomic technologies have identified large numbers of genes and variants of potential importance to human disease. Determining the function of these genes and variants is a critical bottleneck in understanding disease etiology. Variants of uncertain significance (VUS) are highly prevalent in our genomes, but our ability to identify them significantly outpaces our ability to determine their molecular and clinical consequences. We developed a genetically tractable induced pluripotent stem cell (iPSC) based platform to investigate gene variant pathogenicity in lung disease, using primary ciliary dyskinesia (PCD) as a model. We identified an individual with a clinical diagnosis of PCD and a VUS in the gene Multiciliate differentiation and DNA synthesis associated cell cycle protein (MCIDAS). Through gene-editing of iPSC-derived airway basal stem cells (iBCs), we precisely defined the molecular and cellular pathogenicity of the variant providing a successful application of the iPSC system to diagnose a lung disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1192b86ac009b9b71e1d325755d48cbbdf2ac789" target='_blank'>
              A stem cell-based platform for functional analysis of genetic variants in lung disease
              </a>
            </td>
          <td>
            D. Wallman, Sachiko T Homma, Madeleine Finton, MaryLou Beermann, Anjali Jacob, S. Choksi, S. Pletcher, Huihui Xu, P. S. Bawa, Jeremy F. Reiter, A. Horani, Steven L. Brody, A. Berical, F.J. Hawkins
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The rarity of recurrent somatic mutations poses a challenge for the targeted treatment of neuroblastoma (NB). Differentiation therapy is an encouraging prospect, with cyclin-dependent kinase inhibitors (CDKis) representing a promising avenue for promoting NB differentiation. This study investigated three CDKis (abemaciclib, fadraciclib, and dinaciclib) alone or combined with retinoic acid (RA) to assess the effects on morphology, growth, gene expression, and the induction of immunogenic cell death in NB cell lines with (LAN-1 and CHLA-90) and without (CHLA-172) MYCN amplification. All cell lines demonstrated sensitivity to CDK inhibition. Notably, low-dose abemaciclib promoted cellular differentiation, as evidenced by the emergence of stromal-like morphological features and upregulation of the differentiation markers STMN4 and ROBO2. Treatment with abemaciclib or fadraciclib led to the upregulation of calnexin and holocytochrome C, which are part of the global stress response, along with the protein p27, which arrests the cell cycle. Molecularly, CDKis sensitivity correlated with an increased CDK4-specific copy number, along with a partial deletion of CDKN2a in two cases (LAN-1, CHLA-172). The addition of RA augmented the effects of the monotherapy, particularly in LAN-1 cells, in both 2D and 3D culture, and both treatments triggered immunogenic cell death, evidenced by calreticulin translocation. Transcriptomic analysis of LAN-1 and CHLA-90 cells revealed that genes deregulated by monotherapy (fadraciclib or RA) were re-regulated in the presence of the second drug. Combination therapy significantly downregulated CRABP2 and CYP26B1, both of which are involved in RA metabolism and its degradation. Furthermore, CCNE2, MYBL2, and MCM4 were strongly suppressed in the fadraciclib/RA combination, confirming the induction of cell cycle arrest. CDKi treatments promote NB differentiation via ER stress, with cytotoxicity enhanced by RA co-treatment. This may increase NB immunogenicity and support immunotherapy eligibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85ca193304bf6f6433a4c52b2f5a75f164abeec5" target='_blank'>
              CDK inhibitors promote neuroblastoma cell differentiation and increase sensitivity to retinoic acid—a promising combination strategy for therapeutic intervention
              </a>
            </td>
          <td>
            Fatemeh Shokraie, Larissa Lechermeier, Pia Bordihn, Philipp Kaps, Steffen Möller, Anna Sophie Schulz, Björn Schneider, Dirk Koczan, Samira Khanipour Roshan, Holger N Lode, C. Classen, Olga Hahn, S. Troschke-Meurer, C. Maletzki
          </td>
          <td>2025-08-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Resistance to CDK4/6 inhibitors (CDK4/6i) is a major challenge in treating hormone receptor-positive, HER2-negative metastatic breast cancer. This study aimed to identify a biomarker predictive of resistance that could also serve as a therapeutic target. Circulating IL-6 levels in 166 patients significantly increased at progression, making IL-6 a non-invasive biomarker to predict CDK4/6i resistance. Knockdown of IL-6 re-sensitized resistant cells to palbociclib and endocrine therapy, underscoring IL-6’s critical role in maintaining resistance. Patient-derived xenograft models from patients who progressed within 3 months versus ≥6 months of palbociclib therapy revealed distinct transcriptomic profiles, with later progressors exhibiting IL-6/STAT3 activation, epithelial-to-mesenchymal transition, and immune evasion signatures. Treatment with TTI-101, a STAT3 inhibitor, significantly reduced tumor growth and improved survival in these models, providing preclinical validation for targeting the IL-6/STAT3 axis. These findings support using IL-6 to guide personalized treatment and combining STAT3 inhibitors with CDK4/6i as a transformative strategy to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08735c14db476a16d97e533ff51d162f3e014327" target='_blank'>
              IL-6 predicts CDK4/6 inhibitor resistance, identifying STAT3 as a target in HR + /HER2-negative metastatic breast cancer
              </a>
            </td>
          <td>
            N. Kettner, Tuyen N Bui, Juliana Navarro-Yepes, Sanaz Ghotbaldini, Bethanie Quintela, Catherine K. Luo, Nghi Lam, X. Rao, A. Raghavendra, Yan Wang, Nancy Azizian, T. Kris Eckols, Moses M. Kasembeli, K. Evans, Min Yi, Hannah F Wingate, Jing Wang, Aysegul A. Sahin, Funda Meric-Bernstam, Kelly K Hunt, S. Damodaran, David J Tweardy, Debu Tripathy, K. Keyomarsi
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gastric cancer (GC) continues to rank among the leading causes of cancer-related mortality globally, with treatment resistance and recurrence posing significant clinical hurdles. While surgical interventions, chemotherapy, and targeted therapies are available, their efficacy in managing advanced or metastatic forms of the disease remains constrained. This review provided an overview of the role of glycolytic reprogramming in gastric cancer, emphasizing the complex regulation by epigenetic mechanisms, non-coding RNAs, post-translational modifications, and oncogenic signaling pathways. This review discusses how epigenetic mechanisms, including m6A methylation and ceRNA networks involving circRNAs and microRNAs, modulate key glycolytic enzymes such as PKM2, HK2, and PGK1, thereby promoting tumor growth, metastasis, and chemoresistance. The study also emphasizes the impact of post-translational modifications like succinylation and ubiquitination on enzyme activity, affecting glycolytic flux and tumor adaptability. Additionally, the article details the crosstalk between glycolytic pathways and oncogenic signaling networks, including hypoxia-inducible factors and YAP/TAZ transcriptional regulators, which sustain tumor stemness and immune evasion. Therapeutic strategies targeting these metabolic vulnerabilities—such as inhibiting m6A regulators, disrupting ceRNA interactions, and modulating enzyme modifications—are discussed as potential approaches to improve gastric cancer treatment. Overall, we underscores the complexity of metabolic regulation in gastric cancer and proposes that targeting its epigenetic and signaling networks offers promising avenues for innovative therapies to overcome resistance and hinder tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324b5f927de56ddba626c03ffecb44889a215f99" target='_blank'>
              Harnessing glycolysis in gastric cancer: molecular targets, therapeutic strategies, and clinical horizons
              </a>
            </td>
          <td>
            Zexing Shan, Yefu Liu
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Prostate cancer (PCa) is an extremely heterogeneous cancer and is highly prevalent in the older male population. Since intra‐tumour heterogeneity (ITH) commonly results in PCa chemotherapy resistance and recurrence, it is critical to explore its effects on tumour behaviour. Prognostic genes related to ITH were identified, and a signature was constructed using Cox regression analyses and multiple machine learning algorithms. Single‐cell RNA sequencing data extracted from PCa and CRPC samples were analysed via sub‐clustering, pseudotime, cell communication and drug sensitivity approaches to elucidate their function. The oncogenic potential of hub genes was confirmed by immunohistochemistry and cell proliferation assays. An 11‐gene signature underlying a prostate cancer meta‐program (PCMP) was generated by selecting an optimal combination of machine learning methods. Survival assays and multivariate Cox regression analyses conducted in multiple cohorts revealed the superior prognostic value of the PCMP signature. Functional enrichment analyses indicated that it dysregulates the cell cycle. Using trajectory and cell–cell communication analyses, we illustrated that PCMP genes exert oncogenic effects by enhancing the proliferation and oxidative phosphorylation of epithelial cells. Intra‐cellular assays also demonstrated that CENPA and CKS1B had promising malignant potential. In summary, our research not only establishes the association between the PCMP signature and reveals its malignant characteristics, but also deepens our understanding of the mechanisms underlying PCa progression and ITH. It holds promise for the development of targeted therapeutic interventions, thereby offering clinical benefits to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045ffd0cd32bcb5869978af9b2509598525fb2d8" target='_blank'>
              Single‐Cell and Bulk RNA Sequencing Highlights Intra‐Tumoral Heterogeneity and Malignant Progression Mechanisms in Prostate Cancer
              </a>
            </td>
          <td>
            Junchao Wu, Ziqi Chen, Wentian Wu, Jiaxuan Qin, Rongfang Zhong, Jialin Meng, Yu Yin, Peng Guo, Song Fan
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, N. Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis through their capacity for self-renewal and multilineage differentiation. However, the isolation and functional characterization of HSCs remain challenging due to their cellular heterogeneity and dynamically regulated transcriptional and epigenetic landscapes. Advances in experimental and computational biology, including single-cell RNA sequencing (scRNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq), network inference algorithms, and machine learning, have improved our ability to resolve transcriptional states, trace lineage trajectories, and reconstruct gene regulatory networks (GRN) at single-cell resolution. These approaches enable the discovery of novel HSC subtypes and regulatory factors, and facilitate the integration of multi-omics data to uncover epigenetic and transcriptional mechanisms that drive stem cell fate decisions. Additionally, machine learning models trained on high-throughput datasets provide predictive power for identifying novel enhancers, transcription factors, and therapeutic targets. This review underscores the synergistic role of computational tools in deciphering HSC biology and highlights their potential to improve stem cell therapies and precision treatments for hematologic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3c26c697c565c33ced832e7a44009a59ac4768" target='_blank'>
              Unlocking hematopoietic stem cell potential: integrative computational approaches for genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            P. Raghav, Basudha Banerjee, Rajni Chadha
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) treatment remains challenging due to genetic heterogeneity and resistance mechanisms. To address this, we developed a drug discovery pipeline using patient-derived primary CRC cultures with diverse genomic profiles. These cultures closely resemble certain molecular characteristics of primary and metastatic CRC, highlighting their promise as a translational platform for therapeutic evaluation. Importantly, our engineered model and patient-derived cells reflect the complexity and heterogeneity of primary tumors, not observed with standard immortalized cell lines, offering a more clinically relevant system, although further validation is needed. High-throughput screening (HTS) of 4255 compounds identified 33 with selective efficacy against CRC cells, sparing normal, healthy epithelial cells. Among the tested combinations, everolimus (mTOR inhibitor) and uprosertib (AKT inhibitor) demonstrated promising synergy at clinically relevant concentrations, with favorable therapeutic windows confirmed across tested patient-derived cultures. Notably, this synergy, revealed through advanced models, might have been overlooked in traditional immortalized cell lines, highlighting the translational advantage of patient-derived systems. Furthermore, the integration of machine learning into the HTS pipeline significantly improved scalability, cost-efficiency, and predictive accuracy. Our findings underscore the potential of patient-derived materials combined with machine learning-enhanced drug discovery to advance personalized therapies. Specifically, mTOR-AKT inhibition emerges as a promising strategy for CRC treatment, paving the way for more effective and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b44bc31581c51d6eaf637f51510217abbfb946c" target='_blank'>
              Integrated transcriptomic and functional modeling reveals AKT and mTOR synergy in colorectal cancer
              </a>
            </td>
          <td>
            M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Szklarczyk, Olga Dracz, A. Kurzejamski, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Karolina Pyziak, Joanna Drozdowska, Krzysztof Baczyński, Konrad Wojtowicz, Maurycy Chronowski, Krzysztof Rataj, Magdalena Otrocka, M. Mikula, Paula Feliksiak, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka, Andrzej Mazan
          </td>
          <td>2025-07-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-10</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background The APOBEC (apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like) family plays a vital mutagenic role in diverse human malignancies. Nevertheless, the biological characteristics of APOBEC family members and their clinical significance in cancer have not been comprehensively explored. Our primary objective was to characterize the distribution and clinical relevance of APOBEC family members and their mutations across multiple cancer types, with a particular focus on thyroid carcinoma (THCA). Methods In this study, we employed an integrated, multi-faceted approach combining diverse statistical and bioinformatics methods. Data sources included whole-exome sequencing (WES), targeted next-generation sequencing (NGS), bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), Western blot assays, drug sensitivity analyses, and in vivo animal models. Results APOBEC mutations occupy a significant position in the mutation landscape of THCA. High APOBEC mutation enrichment scores were strongly associated with poor prognosis, immune evasion, and increased risk of malignant progression in THCA patients. These mutations contribute to tumor advancement and influence cellular differentiation within the tumor microenvironment. Additionally, we developed a prognostic APOBEC mutagenesis model using machine learning, which was validated across multiple THCA cohorts. In vitro and in vivo experiments demonstrated that APOBEC2 inhibition effectively suppressed tumor proliferation, metastasis, and glycolytic activity, while simultaneously enhancing immune activation and boosting the efficacy of immune checkpoint inhibitors. Conclusion Our study systematically characterizes APOBEC mutations across various cancer types and underscores their promise as biomarkers for aggressive tumor phenotypes, prognostic assessment, and immunotherapy response to THCA. These findings were accomplished through multi-omics sequencing and validated through comprehensive in vitro and in vivo experiments. Supplementary Information The online version contains supplementary material available at 10.1186/s12575-025-00288-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95761133ff7f07fa29822ab7f50151c4ca2027f9" target='_blank'>
              Comprehensive Insights into APOBEC Mutations in Thyroid Cancer: Prognostic and Therapeutic Discoveries
              </a>
            </td>
          <td>
            Wei Luo, Feng Liu, Mengyu Li, Jialong Yu, Ziyun Liu, Xuan Cheng, Yue Huang, Yu Liu, Mei Tao, Yuqi Wang, Yiping Zou, Xiaobin Shang, Chao Yang, X. Ruan, Yanchao Qin, X. Zheng
          </td>
          <td>2025-07-26</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            Zhi-cheng Huang, Yili Feng, Qian Liu, Ruo-Dan Chen, Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c51fa89f4862ae8ddeae87716369c8a92a2e111" target='_blank'>
              Systematic characterization of full-length RNA isoforms in human colorectal cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Ping Lu, Yu Zhang, Yueli Cui, Y. Liao, Zhenyu Liu, Zhi-Jie Cao, Jun-E Liu, Lu Wen, Xin Zhou, Wei Fu, Fuchou Tang
          </td>
          <td>2025-07-22</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Abstract Background Tumor-associated macrophages have been implicated in the progression and treatment resistance of glioblastoma (GBM). This study aimed to identify macrophage-related genes associated with prognosis and therapeutic response in GBM. Materials and methods Bulk RNA-seq data from 533 patients with GBM were downloaded from the Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) databases. Bioinformatic tools were used to detect the co-expression gene modules associated with the infiltration of immune cells, identify a prognostic macrophage-related gene signature, and explore their association with sensitivity to chemotherapeutic drugs and immune checkpoint blockade. Single-cell RNA-seq data and multiplexed immunofluorescence were used to validate ISG20 expression (a member of the identified gene signature) in macrophages. Results We detected gene modules associated with macrophages and identified a signature consisting of three macrophage-related genes (ISG20, PARP12 and IFIT5) in the discovery set (TCGA–GBM, n = 159), and validated its prognostic value in the validation set (CGGA–GBM, n = 374). This gene signature demonstrated favorable accuracy in predicting prognosis and resistance of immuno- and chemo-therapy. The co-expression of ISG20 and PD-1 in macrophages was verified by single-cell RNA-seq data and multiplex immunofluorescence. Conclusions This study presents a macrophage-related gene signature to predict prognosis and therapeutic response in GBM. ISG20, PARP12 and IFIT5 are interferon-stimulated genes, and further investigations may provide new insights into the interplay between macrophages and interferon signaling in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f715f9dcfc9de2bd4b3f11c175f24e467cbcae2" target='_blank'>
              Identification of a prognostic signature consisting of three macrophage-related genes for glioblastoma based on bulk and single-cell transcriptomes analyses
              </a>
            </td>
          <td>
            Qiyan Wu, Chuanbiao Chen, Lina Li, Hongyu Liu, Lingling Cui, Yang Bai, Liping Zhang, Lingxiong Wang, Yaojun Peng
          </td>
          <td>2025-08-14</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edaab282aa5c143118f3f10f0e8f0e87fd84909" target='_blank'>
              Single-cell transcriptome atlas and genome-wide Mendelian randomization reveal chemokine involvement in diverse immune cells in type 2 diabetes.
              </a>
            </td>
          <td>
            Yang Liu, Tao Wang, Rong Wu, Jiapeng Gao, Jiexun Cai, Lixia Huo, Xiaoyu Li, Jingwen Li, Jue Wang, Zhen Wang, Xiaoyi Wang, Yunliang Yao
          </td>
          <td>2025-07-20</td>
          <td>International journal of obesity</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc486e9a397957b4d882241501f31a8794a03818" target='_blank'>
              Precisely defining disease variant effects in CRISPR-edited single cells.
              </a>
            </td>
          <td>
            Y. Baglaenko, Zepeng Mu, Michelle Curtis, Hafsa M Mire, Vidyashree Jayanthi, Majd Al Suqri, Cassidy Liu, Ryan Agnew, A. Nathan, A. Mah-Som, David R. Liu, Gregory A. Newby, Soumya Raychaudhuri
          </td>
          <td>2025-07-23</td>
          <td>Nature</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Simple Summary Cancers involving RAS gene mutations are among the most difficult to treat, particularly in advanced stages. These genes for RAS—KRAS, HRAS, and NRAS—control normal cellular processes, but when mutated, they help tumors grow, spread, and resist treatment. Decades of research have targeted RAS genes and Ras proteins, yet many therapies still fail due to oversimplified signaling interpretations and poor translational reliability. This review compares how Ras functions in healthy and cancerous cells and evaluates model systems across human, mice, and fly studies. We highlight the limitations of current model systems and the need for more physiologically relevant research tools. The goal is to push research toward more effective tools, refined models, and improved outcomes for patients with late-stage RAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796354d7dffc674d220d0fc486fafe630423dfaf" target='_blank'>
              RESEARCH CHALLENGES IN STAGE III AND IV RAS-ASSOCIATED CANCERS: A Narrative Review of the Complexities and Functions of the Family of RAS Genes and Ras Proteins in Housekeeping and Tumorigenesis
              </a>
            </td>
          <td>
            Richard A. McDonald, A. Varela-Ramírez, Amanda K Ashley
          </td>
          <td>2025-07-25</td>
          <td>Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genomic medicine is revolutionizing genetic disease diagnosis and therapy; has a major impact on clinical practice, particularly in diagnosis and treatment. In addition, next-generation sequencing (NGS) has transformed diagnostics. These advances have made genome profiling costeffective and fast, helping us find pathogenic variations that cause a variety of genetic illnesses. Given its influence on diagnostic methods, NGS mutation detection accuracy and reliability must be assessed. In therapeutics, genomic medicine has introduced precision methods. CRISPR-Cas9 gene editing, and new RNA-based therapies are being evaluated for the treatment of genetic mutations Pharmacogenomics' capacity to customize medication regimens to genetic profiles, optimizing therapeutic results while minimizing side effects, is also evaluated. Although genetic medicine has transformational promise, its widespread acceptance is difficult. Obtaining widespread acceptance of genetic medicine is difficult because of worries around ethical implications, privacy problems, and the possibility for genetic information to be misused. Ethics and privacy issues surrounding genetic information usage require considerable thought. Genomic data integration into clinical practice requires robust regulatory frameworks. The influence of NGS technology and precision treatments on genetic disease diagnosis and therapy is significant. This review emphasizes the importance of assessing diagnostic tools, comprehending novel therapy modalities, and addressing ethical and regulatory issues to enable responsible and successful clinical integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45fb4ba10eea702680c101739b29f0747b84e941" target='_blank'>
              Genomic Medicine: A Critical Review of its Impact on Diagnosing and Treating Genetic Disorders.
              </a>
            </td>
          <td>
            V. Sundaram, Sindhu Kaviya Rengarajan, Sivasubarmanian Manikandan, Saravanan Anbalagan, V. Sivakumar, Thamarai Packiyam, Hitesh Chopra
          </td>
          <td>2025-07-22</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The protein p53, often referred to as the “guardian of the genome,” is essential for preserving cellular balance and preventing cancerous transformations. As one of the most commonly altered genes in human cancers, its impaired function is associated with tumor initiation, development, and resistance to treatment. Exploring the diverse roles of p53, which include regulating the cell cycle, repairing DNA, inducing apoptosis, reprogramming metabolism, and modulating immunity, provides valuable insights into cancer mechanisms and potential treatments. This review integrates recent findings on p53′s dual nature, functioning as both a tumor suppressor and an oncogenic promoter, depending on the context. Wild-type p53 suppresses tumors by inducing cell cycle arrest or apoptosis in response to genotoxic stress, while mutated variants often lose these functions or gain novel pro-oncogenic activities. Emerging evidence highlights p53′s involvement in non-canonical pathways, such as regulating tumor microenvironment interactions, metabolic flexibility, and immune evasion mechanisms. For instance, p53 modulates immune checkpoint expression and influences the efficacy of immunotherapies, including PD-1/PD-L1 blockade. Furthermore, advancements in precision diagnostics, such as liquid biopsy-based detection of p53 mutations and AI-driven bioinformatics tools, enable early cancer identification and stratification of patients likely to benefit from targeted therapies. Therapeutic strategies targeting p53 pathways are rapidly evolving. Small molecules restoring wild-type p53 activity or disrupting mutant p53 interactions, such as APR-246 and MDM2 inhibitors, show promise in clinical trials. Combination approaches integrating gene editing with synthetic lethal strategies aim to exploit p53-dependent vulnerabilities. Additionally, leveraging p53′s immunomodulatory effects through vaccine development or adjuvants may enhance immunotherapy responses. In conclusion, deciphering p53′s complex biology underscores its unparalleled potential as a biomarker and therapeutic target. Integrating multi-omics analyses, functional genomic screens, and real-world clinical data will accelerate the translation of p53-focused research into precision oncology breakthroughs, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6708657f0ff7cd8075abb4af281defbd7a1577" target='_blank'>
              The Multifaceted Role of p53 in Cancer Molecular Biology: Insights for Precision Diagnosis and Therapeutic Breakthroughs
              </a>
            </td>
          <td>
            Bolong Xu, Ayitila Maimaitijiang, Dawuti Nuerbiyamu, Zhengding Su, Wenfang Li
          </td>
          <td>2025-07-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, 
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Kirsten Rat Sarcoma (KRAS) copy number amplification has emerged as an oncogenic driver in colorectal cancer (CRC), in addition to KRAS mutation. However, its clinical significance remains poorly understood. Notably, CRC patients with wild-type KRAS but harboring KRAS amplification have shown resistance to anti-EGFR therapy, representing an unmet clinical need. Methods In this study, we comprehensively investigated the impact of KRAS amplification—alone and in conjunction with mutation—on clinicopathological characteristics, immune infiltration, and therapeutic response. KRAS copy number variation (CNV) was classified into amplification and non-amplification groups. KRAS mutational status was determined as wild-type (WT) or mutant (MUT) using qPCR and Sanger sequencing. CD8⁺ T lymphocyte infiltration was evaluated by immunohistochemistry in resected CRC specimens. Clinical, immune, and survival data were analyzed in association with KRAS status. RNA-seq was performed to identify differentially expressed genes (DEGs) and enriched pathways. Patient-derived xenograft (PDX) models were used to assess responses to targeted therapies. Results We found that KRAS amplification was more frequent in WT KRAS CRC (21.4%) than in MUT KRAS CRC (6.5%). In the WT subgroup, KRAS amplification was associated with poor prognosis and increased KRAS protein expression and downstream pathway activation. In contrast, amplification had little effect in mutant KRAS tumors. KRAS copy number was inversely correlated with CD8⁺ T cell infiltration, suggesting a role in immune evasion. Importantly, low CD8⁺ T cell density combined with KRAS amplification predicted adverse outcomes. Therapeutically, KRAS-amplified PDX models were resistant to anti-EGFR treatment. However, combined MEK and CDK4/6 inhibition overcame this resistance. Conclusions KRAS amplification constitutes a distinct oncogenic driver in CRC and a potential biomarker for therapeutic stratification. Our findings support the clinical rationale for dual MEK and CDK4/6 inhibition in KRAS-amplified CRC and highlight the need for biomarker-guided clinical trials to optimize treatment strategies in this subset of patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06864-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd01174f41ba37a6094968fc4a8b39ec9e5e025b" target='_blank'>
              KRAS amplification in colorectal cancer: correlations with clinicopathological features and prognosis in patients and prediction of response to targeted therapy
              </a>
            </td>
          <td>
            Xuanhua Yang, Shiwei Feng, Qi Zhang, Wenxin Li, Shujuan Li, Min Zhi, Ping Lan, Sijing Cheng
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancerous transcriptome alterations in carcinoma cells could be originated from either genetic copy number changes or epigenetic reprogramming. Ovarian cancer (OV) is the most malignant gynecologic tumor, known for high aneuploidy with robust copy number alterations. However, low aneuploidy ovarian tumors are also frequently found, indicating an essential contribution of epigenetic factors during tumorigenesis and cancer development. Chromatin remodeling modulates the transcriptome epigenetically in a variety of cancer types, but its role in OV is still unclear. Hence, we investigated a cohort of 102 OV patients, analyzed transcriptomic and clinical data from public databases, and performed cellular experiments. We found that RUVBL2, a subunit of the INO80 complex, functions as the key oncogenic chromatin remodeler in OV. RUVBL2 is upregulated in tumors, particularly in low-aneuploidy cases, and is associated with poor prognosis. RUVBL2 drives nucleosome dynamics and elevates chromatin accessibility selectively at promoter regions. The landscape of RUVBL2-dependent modulation of chromatin accessibility and the transcriptome exhibits activation of various transcription factors, especially the AP-1 family, and upregulation of a series of key genes, including CDKN3, MYBL2, and ZNF144, resulting in mediation of cell cycle and Hippo signaling pathway to promote DNA synthesis and cell proliferation. Hence, RUVBL2-dependent chromatin remodeling plays a key role in oncogenic reprogramming of the transcriptome in OV. These findings provide novel insights into the molecular etiology of OV and disclose potential biomarkers and drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2407b3becd0c884fdbbc4e31bad36bc2c5c367" target='_blank'>
              Profiling of RUVBL2-Induced Transcriptome Alterations Highlights a Critical Role for Chromatin Remodeling in Ovarian Cancer.
              </a>
            </td>
          <td>
            Renhao Xue, Yingjie Wang, Xiaomei Luo, Hao Zhang, Dongcheng Guan, Shuo Shi, Yu Wang
          </td>
          <td>2025-07-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer has recently overtaken other conditions as the foremost cause of death worldwide. While numerous
conventional treatment modalities and cytotoxic immunotherapies are available, the intricate nature of tumor biology—
driven by a multitude of genetic and cellular factors underlying tumorigenesis and metastasis—necessitates the
development of advanced treatment approaches capable of acting on cellular as well as genetic fronts. Amid these
advancements, Chimeric Antigen Receptor (CAR) T-cell therapy has risen as a pioneering frontier in the domain of T-cell
engineering. Here, T cells obtained from the patient are engineered ex vivo to carry synthetic receptors designed to
recognize tumor-associated antigens. A key feature is that these modified receptors can identify tumor antigens without
requiring MHC presentation. Recently, CAR-T cell therapy has achieved significant clinical milestones, inducing
remission in up to 80% of patients with hematologic malignancies, particularly those with acute lymphoblastic leukemia
(ALL) and certain subtypes of non-Hodgkin lymphoma, such as large B-cell lymphoma. The anti-CD19 CAR design,
UCART19, has demonstrated remarkable therapeutic potency in managing relapsed or refractory hematologic
malignancies. Furthermore, other surface antigens, including CD20 and CD22, prevalent in various leukemias and
lymphomas, are being actively explored as therapeutic targets, with multiple clinical trials underway. Although its
application is presently concentrated on blood cancers, the integration of advanced modalities—such as bispecific CARs,
tandem CARs, inhibitory CARs, multi-antigen targeting, CRISPR-based gene editing, and nanoparticle-mediated
delivery—holds the promise of significantly enhancing its efficacy. These advancements may extend its utility to both
hematologic and solid tumors, offering a treatment paradigm that is not only rapid and precise but also safer compared to
traditional modalities. The purpose of this review is to offer an in-depth analysis of the benefits and emerging
developments in CAR T-cell immunotherapy, while underscoring its rising advantage over conventional treatments like
chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5c74588893737227ed28d56c33635129d1547e" target='_blank'>
              Targeting Cancer at its Core: Cellular and Genetic Approaches Via CAR-T Cell Therapy CAR-T Cell Therapy of Cancer
              </a>
            </td>
          <td>
            Bandaru Aashritha, Ambati Akshara, Nandini Kongara, P. V. Babu
          </td>
          <td>2025-09-04</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sacrococcygeal teratomas (SCTs) are the most common neonatal tumors, yet their cellular origins, clinical stratification, and sex bias–occurring three times more in XX than XY individuals—remain poorly understood. To address these gaps, we examined six postnatal (one male and five female) and two prenatal (both female) SCTs by single nuclei RNA-seq and spatial transcriptomics. We identified five broad cellular lineages in SCTs: stroma, epithelia, endothelia, neuroectoderm, and immune. The transcriptomes and lineage compositions showed significant heterogeneity, which offer a framework for future molecular stratification. SCTs are thought to originate from and be propagated by pluripotent cells, notably however, we did not detect these populations. Among female tumors, a subset of cells exhibited biallelic expression of X-linked genes, consistent with X-inactivation failure or reactivation of the once inactivated X-chromosome. These biallelic cells were enriched for developmental and neuronal programs, whereas cells with single-allelic X-chromosome preferentially expressed immune-related genes. Biallelic X-chromosome activation, which can occur only in female cells, may result in transcriptomic features that favor survival of tumor cells, contributing to the sex bias of SCTs. Our findings reveal a link between X-chromosome inactivation and SCT cell identity, suggesting that X-dosage dysregulation may influence SCT heterogeneity and immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee83296f6240c6d4cef47d40571146d063d815c6" target='_blank'>
              Integrated single-nuclei and spatial transcriptomic profiling of human sacrococcygeal teratomas reveals heterogeneity in cellular composition and X-chromosome inactivation
              </a>
            </td>
          <td>
            Ernesto J. Rojas, K. Giannikou, Benjamin J. Huang, Soo-Jin Cho, Marco Cordero, Deion Pena, Lan Vu, Aditya Bagrodia, S. Derderian, T. Mackenzie, D. Laird
          </td>
          <td>2025-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a unique subtype of breast cancer characterized by high invasiveness, high metastasis rates, and poor prognosis, making it an important focus within global malignancies. Due to the absence of estrogen receptor, progesterone receptor, and HER2 expression, TNBC presents significant challenges in treatment. Metastatic progression markedly increases treatment complexity, drastically reducing patient survival rates. The metastatic and drug resistance processes of TNBC involve complex, multi-step biological mechanisms regulated through various molecular mechanisms and signaling pathways within and outside tumor cells. In recent years, immunotherapy has brought new hope for TNBC. Compared to other breast cancer subtypes, TNBC demonstrates higher immunogenicity, often accumulating a higher mutational burden that generates more neoantigens, thus typically resulting in a tumor microenvironment (TME) enriched with tumor-infiltrating lymphocytes (TILs). Additionally, PD-L1 expression is significantly higher in TNBC compared to other subtypes, closely correlating with TIL abundance. These characteristics position TNBC as a strong candidate for immune checkpoint inhibitor (ICI) therapy. Clinical trials have demonstrated promising efficacy of ICIs in TNBC, overturning previous beliefs that breast cancer is generally insensitive to immunotherapy. This review summarizes recent advances regarding resistance types, molecular mechanisms, associated genes and pathways, the role of the tumor microenvironment, and clinical strategies related to immunotherapy resistance in the neoadjuvant setting of TNBC, aiming to provide insights and guidance for future research exploration and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03cacb55534d35ef52652d44531e93baccfb5003" target='_blank'>
              Immunotherapy resistance in triple-negative breast cancer: Molecular mechanisms, tumor microenvironment, and therapeutic implications
              </a>
            </td>
          <td>
            Zijian Zhou, Qin Zhou
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22cb899bdeb7ede5aa5cc85f918b0eb5b16429d8" target='_blank'>
              Long-read RNA sequencing of transposable elements from single cells using CELLO-seq.
              </a>
            </td>
          <td>
            Sophie A Marlow, Lauryn A Deaville, Rebecca V Berrens
          </td>
          <td>2025-07-16</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Bone metastasis remains a significant cause of morbidity and diminished quality of life in patients with advanced breast, prostate, and lung cancers. Emerging research highlights the pivotal role of reversible epigenetic alterations, including DNA methylation, histone modifications, chromatin remodeling complex dysregulation, and non-coding RNA networks, in orchestrating each phase of skeletal colonization. Site-specific promoter hypermethylation of tumor suppressor genes such as HIN-1 and RASSF1A, alongside global DNA hypomethylation that activates metastasis-associated genes, contributes to cancer cell plasticity and facilitates epithelial-to-mesenchymal transition (EMT). Key histone modifiers, including KLF5, EZH2, and the demethylases KDM4/6, regulate osteoclastogenic signaling pathways and the transition between metastatic dormancy and reactivation. Simultaneously, SWI/SNF chromatin remodelers such as BRG1 and BRM reconfigure enhancer–promoter interactions that promote bone tropism. Non-coding RNAs, including miRNAs, lncRNAs, and circRNAs (e.g., miR-34a, NORAD, circIKBKB), circulate via exosomes to modulate the RANKL/OPG axis, thereby conditioning the bone microenvironment and fostering the formation of a pre-metastatic niche. These mechanistic insights have accelerated the development of epigenetic therapies. DNA methyltransferase inhibitors (e.g., decitabine, guadecitabine) have shown promise in attenuating osteoclast differentiation, while histone deacetylase inhibitors display context-dependent effects on tumor progression and bone remodeling. Inhibitors targeting EZH2, BET proteins, and KDM1A are now advancing through early-phase clinical trials, often in combination with bisphosphonates or immune checkpoint inhibitors. Moreover, novel approaches such as CRISPR/dCas9-based epigenome editing and RNA-targeted therapies offer locus-specific reprogramming potential. Together, these advances position epigenetic modulation as a promising axis in precision oncology aimed at interrupting the pathological crosstalk between tumor cells and the bone microenvironment. This review synthesizes current mechanistic understanding, evaluates the therapeutic landscape, and outlines the translational challenges ahead in leveraging epigenetic science to prevent and treat bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cc3437004b76dc15ae235bf5424272f30c35dcd" target='_blank'>
              Epigenetic Modulation and Bone Metastasis: Evolving Therapeutic Strategies
              </a>
            </td>
          <td>
            Mahmoud Zhra, Jasmine Holail, Khalid S. Mohammad
          </td>
          <td>2025-07-31</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcaf09592f5e3c1f92fc05933cdb4d9765107436" target='_blank'>
              Evolutionary Overview and Future Perspectives: ESR1 Mutations, Liquid Biopsy, and Artificial Intelligence for a New Era of Personalized Medicine in ER+ Breast Cancer.
              </a>
            </td>
          <td>
            Serafina Martella, Giacomo Cusumano, T. H. Senevirathne, Dimitrios Stylianakis, Enrico Palmas, N. Denaro, Chiara Tommasi, M. Scartozzi, Lorenzo Gerratana, C. Solinas
          </td>
          <td>2025-08-30</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast cancer, a major health concern worldwide, involves diverse molecular subtypes and complex gene expression patterns. This study conducted a comprehensive bioinformatics analysis of breast cancer, analyzing 10 gene expression datasets from the Gene Expression Omnibus archive to find common genes that exhibit differential expression (DEGs). Then, we conducted pan-cancer and functional enrichment analyses, including single-cell level investigations of DEGs. To identify potential hub genes, we built protein–protein interaction networks, which were further analyzed for methylation patterns and expression in various cancer stages. The study also explored interactions between these hub genes with microRNAs and transcription factors, along with their correlation with cytokine expression and infiltration of immune cells in the tumor microenvironment. The TCGA BRCA dataset was used for hub gene survival analysis. Key findings include the identification of 36 shared DEGs, with distinct expression patterns in principal component analysis, suggesting their potential as molecular signatures for diagnosis and prognosis. Investigation at the single-cell level revealed upregulated DEGs mainly expressed by tumor-associated fibroblast cells. Gene ontology unveiled upregulated genes mainly involved in the extracellular matrix, but most of the downregulated genes are tumor suppressors. The protein–protein interaction network analysis highlighted 7 hub genes: CAV1, FN1, BGN, CXCL12, SPTBN1, COL11A1, and INHBA. Methylation analysis revealed intricate epigenetic regulation of these genes. MicroRNA and transcription factor interaction analysis underscored the complex regulatory networks influencing gene expression. Cytokine profile analysis in tumor microenvironment and its correlation with hub gene expression provided insights into the immunological landscape of breast cancer. Survival analysis indicated that both upregulated and downregulated hub genes are linked with patient overall survival outcomes, although it is not statistically significant. This study provides a detailed view of the underlying molecular mechanisms in breast cancer, suggesting potential biomarkers and therapeutic targets. Its findings can contribute significantly to understanding the complexity of the breast cancer microenvironment, leading to the development of more sophisticated and targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf4528f9aeae9acc9b474eba2a50c1c17c8b995" target='_blank'>
              Identification of potential hub genes and molecular mechanisms in breast cancer microenvironment: A comprehensive transcriptomics approach
              </a>
            </td>
          <td>
            MS Abdullah Al Noman, MS a Abdullah Al Saba, MS a Mohammad Sayem, PhD Tahirah Yasmin, PhD A. H. M. Nurun Nabi, A. N. Nabi
          </td>
          <td>2025-08-29</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer.
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome integrity is a critical driver of cellular stability, and defects in the processes that maintain genome health are potent sources of cancer progression. Homologous recombination deficiency (HRD), which damages cells through absent or erroneous repair of double-stranded DNA breaks, is a prime example of such cellular dysfunction. Poly(ADP-ribose) polymerase (PARP) inhibitors exploit these aberrancies in the cellular repair process by arresting the ability of the PARP enzyme to repair cellular and genetic damage, inducing the accumulation of DNA damage and cancer cell death. While the utility of PARP inhibitors has been established in many HRD-associated tumors — particularly breast, pancreatic, ovarian, and prostate cancer — less robust evidence exists for the efficacy of PARP inhibitors in melanoma. Increasing efforts are underway to investigate PARP inhibitors as a viable treatment option for advanced and metastatic melanoma, both as monotherapy and in combination with other agents such as immune checkpoint inhibitors and BRAF/MEK inhibitors. Though several gaps in our knowledge of the use of PARP inhibitors in melanoma still exist, promising headway is being made in our understanding of its efficacy and safety. Here, we present a review of the utility of PARP inhibitors in melanoma, current clinical trials, and future avenues for further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e9847a79c6d01c47c9645097708f725557ad2a" target='_blank'>
              PARP inhibitors in melanoma treatment: potential, challenges, and future directions
              </a>
            </td>
          <td>
            Angela Anaeme, Karen Moussa, Abdallah N. Mansour, George Nassief, George Ansstas
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metabolic reprogramming is a key feature of clear cell renal cell carcinoma (ccRCC), and metabolic abnormality can lead to significant changes in gene expression, resulting in the immunosuppressive microenvironment. In this study, we used a combination of single-cell RNA sequencing and bulk RNA sequencing to investigate the relationships between ccRCC metabolic reprogramming and immune exhaustion. Metabolic subtypes of ccRCC patients were constructed using bulk RNA sequencing. Tumor cells of different metabolic subtypes were analyzed and extracted by the Scissor algorithm, using single-cell RNA sequencing. The molecular mechanisms of abnormal metabolic regulating tumor immunity were explored using cell–cell communication analysis. In addition, the correlations between relevant molecules and immune exhaustion signals were verified in ccRCC by immunohistochemistry. The molecular mechanisms of metabolic abnormalities leading to immune exhaustion were validated via Western blotting, ELISA, cell co-culture and immunotherapy models. ccRCC patients can be divided into MT1 and MT2 metabolic subtypes. The MT2 subtype has a poorer prognosis and lower response to immunotherapy. Abnormal metabolism of arachidonic acid is a prominent feature of the MT2 subtype, and activates the MDK signaling pathway. As a secreted protein, MDK can further recruit immunosuppressive cells, such as Treg, Tex, and TAM. Blocking the arachidonic acid COX metabolic pathway significantly reduces the expression and secretion levels of MDK, thereby reprogramming the tumor microenvironment to promote anti-tumor immunity. Abnormal metabolism of arachidonic acid plays an important role in promoting immune exhaustion by activating the MDK signaling pathway. MDK may serve as an important biomarker for predicting the immune therapy response in ccRCC. By reducing MDK secretion, targeting blockade of arachidonic acid metabolism may be an effective treatment strategy to enhance the efficacy of immunotherapy in ccRCC. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01807-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cdee7e8710af1e672e045bcfb43ec330de5b" target='_blank'>
              Metabolic reprogramming of arachidonic acid in clear cell renal carcinoma promotes an immunosuppressive microenvironment by activating MDK signaling pathway
              </a>
            </td>
          <td>
            Jiaxi Yao, Tong Xu, Chenyuan Wang, Junfeng Xie, Qing Jiang
          </td>
          <td>2025-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13ed8b839c57dd2cae305f096f9c5a1a8fff6df" target='_blank'>
              Targeting mitochondrial metabolism to overcome hormone resistance in breast cancer.
              </a>
            </td>
          <td>
            Abbas S. Neamah, F. M. Lafta, Al-Hassan Soliman Wadan
          </td>
          <td>2025-08-20</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94fbc0a79acc6816ca49ca82a2bba0755e7fbb9" target='_blank'>
              Deciphering the molecular landscape of acute myeloid leukemia initiation and relapse: a systems biology approach.
              </a>
            </td>
          <td>
            Atefeh Bahmei, Hanieh Fadakar, Gholamhossein Tamaddon
          </td>
          <td>2025-09-10</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most prevalent extracranial solid malignancy in children, poses significant therapeutic challenges, particularly concerning high-risk subtypes characterized by an immunologically “cold” phenotype. These tumors evade immune surveillance through mechanisms such as impaired antigen presentation and immunosuppressive microenvironment formation. Despite the incorporation of immunotherapies (e.g., GD2 monoclonal antibodies) into international clinical guidelines, the 5-year survival rate of patients with NB persistently remains lower than 50%. Small-molecule targeted agents, distinguished by their low molecular weight and superior chemical stability, offer advantages over macromolecular antibody therapies by effectively penetrating cell membranes to engage intracellular targets. Epigenetic regulation, a DNA sequence-independent gene expression modulation system, plays a pivotal role in cell fate determination via dynamic DNA methylation, histone covalent modifications, chromatin spatial reorganization, and non-coding RNA-mediated pathways. Emerging evidence has highlighted a strong correlation between epigenetic dysregulation and NB progression, establishing a molecular rationale for novel therapeutic strategies. Current epigenetic research in NB primarily focuses on histone deacetylase inhibitors and DNA methyltransferase inhibitors. These drugs exhibit unique translational potential because of their accelerated development timelines and cost-effective production, significantly enhancing therapeutic accessibility. This review systematically examines the current landscape of epigenetic modulators in NB treatment and discusses their transformative potential in improving outcomes for high-risk patients with NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ed0206bc0a5ccb2cde86f23fd7b8f058a1635a" target='_blank'>
              Emerging frontiers in epigenetic-targeted therapeutics for pediatric neuroblastoma
              </a>
            </td>
          <td>
            Xiaokang Wang, Gengrui Xu, Hongyan Ma, Xiaoyan Deng, Guiping Ma
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 200,000 deaths each year. Despite recent advances in treating HGSC with neoadjuvant chemotherapy, the majority of patients ultimately develop chemotherapy resistance. HGSC is characterized by TP53 mutations and widespread copy number alterations and occurs frequently in the setting of deleterious germline BRCA1/2 variations, but many cases lack putative driver mutations. Here, we performed whole-exome, whole-genome, and whole transcriptome sequencing along with mass spectrometry to characterize the molecular landscape of HGSC in 22 paired samples obtained before and after neoadjuvant chemotherapy. Responsiveness to chemotherapy was determined for each patient. Evidence at the DNA, RNA, and protein level revealed numerous defects in cell-cell and cell-matrix interactions as well as disruption of cell polarity and cytoskeletal regulation in HGSC, indicating that defects in epithelial integrity were present in the majority of HGSC patients. Non-responsive HGSC harbored subclones with putative survival mutations. Additionally, ineffective nonsense mediated decay resulted in the persistence of transcripts with frameshift mutations that were translated into aberrant proteins detectable in HGSC samples. Together, these findings suggest that HGSC may arise through defects in maintenance of epithelial integrity that lead to shedding of malignant cells throughout the peritoneum, and the presence of resistant subclones prior to chemotherapy may decrease the chemosensitivity of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/896a702c0b394e4d17c91e5aa33566545b16c4dc" target='_blank'>
              Multi-omic Characterization of Pre- and Post-neoadjuvant Chemotherapy Treated Ovarian Cancer Reveals Mediators of Tumorigenesis and Chemotherapy Response.
              </a>
            </td>
          <td>
            Mark C Valentine, Mary M Mullen, E. Kotnik, Matthew A Powell, Premal H Thaker, C. McCourt, L. Kuroki, Andrea R Hagemann, Rebecca C Arend, A. Holehouse, R. Mitra, D. Mutch, Katherine C Fuh
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer is the most common in the world with multifaceted and diverse molecular etiology. Lung cancer initiation, progression and metastasis are orchestrated by a network of enzymes and proteins. They include TP53, KRAS, EGFR and ALK, which are expressed and mutated at abnormally high levels. These molecules control signaling in various cellular pathways including growth, survival, differentiation and invasion.

This review focuses on the malignantly regulated TP53, which is commonly and mutated most frequently in non-small cell lung cancer Overall, we also highlighted the role of TP53 in lung cancer development, the existing therapeutic approaches against TP53 and its relation about the efficacy and adverse events such as resistance and side effects and the next steps to achieve better outcomes in TP53 -positive lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24095e3a50d4594724b1890874f6611abcf39d57" target='_blank'>
              The Role of TP53 Mutations in Lung Cancer: Molecular Mechanisms, Therapeutic Implications and Future Directions
              </a>
            </td>
          <td>
            Thirunavukkarasu Muthu Kumar
          </td>
          <td>2025-08-31</td>
          <td>Research Journal of Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer (CC) poses a substantial global health challenge and it ranks as the fourth most prevalent malignancy among women worldwide. Management strategies include surgical intervention, radiotherapy, chemotherapy and emerging systemic treatments. Although advancements in immunotherapy and targeted therapies have been achieved, the aggressive metastatic nature of the disease, coupled with immune evasion and drug resistance, continues to limit overall survival rates. Therefore, there remains an urgent need to identify novel treatment modalities and more effective therapeutic agents. As fundamental regulators of epigenetic modifications, histone alterations serve a critical role in controlling gene expression, DNA repair mechanisms and cellular differentiation. These modifications include acetylation, methylation, phosphorylation, ubiquitination, ADP-ribosylation and glycosylation, as well as the more recently identified lactylation and palmitoylation. By restructuring chromatin and facilitating interactions among histones, DNA and regulatory proteins, these modifications exert a substantial influence on cellular functions. Aberrant histone modifications contribute to tumorigenesis, tumor heterogeneity and resistance to conventional anticancer therapies, making them a key focus of oncological research. In recent years, therapeutic strategies targeting histone modifications have gained increasing attention in the treatment of CC. Among these epigenetic alterations, histone acetylation and deacetylation have been extensively studied, with numerous histone deacetylase inhibitors showing promise in preclinical studies. The present review explores the patterns of histone modifications in CC, emphasizing their molecular roles in tumor progression, metastasis and therapeutic resistance. Additionally, histone modification-driven therapeutic targets are examined, laying the groundwork for future precision medicine approaches in CC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc39c2ad958de58c571126dd5fb3dbfb68daa22" target='_blank'>
              Histone modifications in cervical cancer: Epigenetic mechanisms, functions and clinical implications (Review)
              </a>
            </td>
          <td>
            Xuewei Li, Min Zhou, Jing Yu, Shaohui Yu, Zheng Ruan
          </td>
          <td>2025-08-01</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary KRAS G12C mutations represent a niche but important target in oncology, with implications across several tumor types. This review provides a comprehensive overview of current knowledge, spanning biological mechanisms, translational research, and clinical advances related to KRAS G12C-mutant cancers. By collecting and organizing all known evidence in a pancancer perspective, we aim to offer an integrated view of the progress made so far. Furthermore, we highlight the major challenges that remain to be addressed, such as resistance to therapies and optimization of treatment strategies, with the goal of guiding future research and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fa02712af1eb6b648d3ae5ff58db6c50814014" target='_blank'>
              KRAS G12C Inhibition in Solid Tumors: Biological Breakthroughs, Clinical Evidence, and Open Challenges
              </a>
            </td>
          <td>
            P. P. Vitiello, A. A. Valsecchi, Eleonora Duregon, Paola Francia di Celle, P. Cassoni, Mauro Papotti, A. Bardelli, M. Di Maio
          </td>
          <td>2025-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Oncolytic virotherapy is a rapidly evolving approach to cancer treatment. Our group previously designed VV-GMCSF-Lact, a recombinant oncolytic vaccinia virus targeting solid tumors including gliomas. In this study, we used single-cell RNA sequencing to compare transcriptional responses in human glioma cells, non-malignant brain cells, and immortalized glioblastoma U87 MG cells following infection with this oncolytic virus. We found that proneural glioblastoma cells and microglia-like cells from patient-derived glioma cultures were the most susceptible to VV-GMCSF-Lact. Increased expressions of histones, translational regulators, and ribosomal proteins positively correlated with viral load at the transcript level. Furthermore, higher viral loads were accompanied by a large-scale downregulation of genes involved in mitochondrial translation, metabolism, and oxidative phosphorylation. Levels of early vaccinia virus transcripts are also positively correlated with infection intensity, suggesting that the fate of cells is determined at the early stage of infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0ce99f16e9e13e522700a614d5b988909f5218" target='_blank'>
              Single-Cell Transcriptomic Changes in Patient-Derived Glioma and U87 Glioblastoma Cell Cultures Infected with the Oncolytic Virus VV-GMCSF-Lact
              </a>
            </td>
          <td>
            D. Semenov, N. Vasileva, M. Menyailo, S. Mishinov, Yulya I Savinovskaya, A. Ageenko, A. S. Chesnokova, M. Dymova, G. Stepanov, G. Kochneva, Vladimir A. Richter, E. Kuligina
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea68d175c763147c781a0448b1e766b436554f3d" target='_blank'>
              Therapeutic strategies for fragile X syndrome and implications for other gene-silencing disorders.
              </a>
            </td>
          <td>
            Aseel Gadban, Keith M Gunapala, Verdon Taylor, N. Benvenisty
          </td>
          <td>2025-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Thalassemia, a prevalent single-gene inherited disorder, relies on hematopoietic stem cell or bone marrow transplantation as its definitive treatment. However, the scarcity of suitable donors and the severe complications from anemia and iron overload pose significant challenges. An immediate need exists for a therapeutic method that addresses both the illness and its associated complications. Advancements in stem cell technology and gene-editing methods, such as clustered regularly interspaced short palindromic repeats along with its associated protein (CRISPR/Cas), offer encouraging prospects for a therapy that could liberate patients from the need for ongoing blood transfusions and iron chelation treatments. The potential of genetic reprogramming using induced pluripotent stem cells (iPSCs) to address thalassemia is highly promising. Furthermore, mesenchymal stem cells (MSCs), recognized for their capacity to self-renew and differentiate into multiple lineages that include bone, cartilage, adipose tissue, and liver, demonstrate potential in alleviating several complications faced by thalassemia patients, including osteoporosis, cirrhosis, heart conditions, respiratory issues, and immune-related disorders. In this review, we synthesize and summarize relevant studies to assess the therapeutic potential and predict the curative effects of these cellular approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983b6b15c2e2e9d04817e5e4d461fe2aeabe9943" target='_blank'>
              CRISPR/Cas-edited iPSCs and mesenchymal stem cells: a concise review of their potential in thalassemia therapy
              </a>
            </td>
          <td>
            Jiaojiao Shu, Xin Xie, Sixi Wang, Zuochen Du, Pei Huang, Yan Chen, Zhixu He
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, S. Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Immune expression profiling in colorectal lesions may provide insights into the origins of antitumor immunity and senescence. Optimal approaches for analyzing samples with lower quality RNA from molecularly diverse lesions are lacking. Therefore, we developed a NanoString nCounter-based approach for quality control (QC), normalization, and differential expression (DE) analysis, optimized for FFPE samples in contexts of high biologic heterogeneity. Methods The approach incorporates a colon specific positive control gene set (11 genes) to minimize sample exclusions. We evaluated three normalization methods Removal of Unwanted Variation (RUVg), NanoStringDiff (NSDiff), and nSolver using a 277 gene immune panel to compare 100 samples, including sessile serrated lesions (SSLs) (n = 25), tubulovillous and villous adenomas (TVs) (n = 27), and tubular adenomas (TAs) (n = 48) We assessed Type I error rates, computational efficiency, and gene significance via FDR-corrected q-values. Results Incorporating the colon-specific QC set reduced sample exclusions by 63% compared to standard methods (13 vs. 35 sample exclusions). All three normalization approaches identified DE genes between SSLs and TAs (e.g., TFF1, MUC5AC, MUC6). For TVs vs. TAs, only RUVg and NSDiff detected significant DE genes, revealing wide-spread under-expression of innate and adaptive genes. While NSDiff labeled twice as many significant genes as RUVg, suggesting greater sensitivity, it also exhibited higher Type I error rates and increased computational demand. Conclusions RUVg achieved a balance between computational efficiency and low Type I error, while NSDiff was more sensitive but computationally demanding and exhibited higher Type I error. Our approach provides a robust framework for profiling immune genes in heterogeneous lesions. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07383-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabf843cf30ad6b2aa8a657a57bee406e0c4d696" target='_blank'>
              Tissue gene expression analysis approach in a context of high technical and biological heterogeneity
              </a>
            </td>
          <td>
            Evan Bagley, Souvik Seal, Lauren R. Fanning, Jean-Sebastian Anoma, Tami Crawford, Benjamin G. Vincent, Elizabeth L. Barry, M. Shrubsole, Erin Kirk, John A. Baron, Dale C. Snover, David N. Lewin, Todd A. Mackenzie, Xiaohua Gao, Melissa A. Troester, Alexander V. Alekseyenko, Kristin Wallace
          </td>
          <td>2025-07-22</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="INTRODUCTION
Metastatic colorectal cancer (mCRC) remains a leading cause of cancer mortality worldwide, with limited long-term survival despite therapeutic advances. The increasing understanding of its molecular heterogeneity has paved the way for precision medicine approaches aiming to optimize treatment efficacy and reduce unnecessary toxicity.


AREAS COVERED
This review provides an in-depth analysis of the current and emerging molecular targets in mCRC, including RAS, BRAF, HER2, and microsatellite instability. We discuss the clinical relevance of tumor sidedness, hyperselection panels, EGFR ligand expression, and rare alterations such as NTRK, RET, and ALK fusions. The review also explores the evolving role of KRAS G12C inhibitors, HER2-targeted therapies, and the application of liquid biopsy - particularly circulating tumor DNA (ctDNA) - in treatment monitoring, rechallenge strategies, and resistance detection. Literature was selected through a comprehensive review of recent clinical trials, consensus guidelines, and translational studies.


EXPERT OPINION
Personalized treatment is now an attainable goal in mCRC. While promising, broader implementation of molecular-driven strategies requires overcoming challenges such as resistance mechanisms, assay standardization, and equitable access. The integration of innovative agents with real-time molecular monitoring holds the key to a more dynamic and effective management of mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d84552ed9fbf737b1d558e20dc110dcd8268ee" target='_blank'>
              Personalized therapy in metastatic colorectal cancer: biomarker-driven use of biologics.
              </a>
            </td>
          <td>
            F. Sullo, Chiara Gallio, Laura Matteucci, Alessandro Bittoni, M. Muratore, Luca Esposito, Bianca Ceredi, Graziana Gallo, Paola Ulivi, I. Rapposelli, A. Passardi
          </td>
          <td>2025-09-04</td>
          <td>Expert opinion on biological therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The purpose of the study was a comprehensive analysis of molecular genetic changes in tumor and normal tissues of patients with colorectal cancer using whole-genome sequencing to identify de novo mutations, microsatellite instability patterns, and mutational signatures associated with tumor development.Material and Methods. A single-time cross-sectional study was conducted in the Kaliningrad region. Colorectal cancer samples were analyzed using whole-genome next-generation sequencing (WGS). The human reference genome GRCh38.p14 was used for read mapping. Bioinformatic analysis of signatures was conducted separately for each patient. 71 pairs of samples were sequenced: complete genomes of tumor biopsies and DNA isolated from blood samples for 71 patients. The average age among the analyzed patients was 69 years, with a median of 71 years.Results. Genetic alterations associated with tumor development and progression, as well as potential therapeutic targets were identified. Using the Oncovar database, 151 driver mutations were detected in 62 patients. Using the CIVIC database, 70 marker mutations were found in 45 patients. It was found that the most common driver mutations in the study cohort were mutations in the KRAS gene (KRAS:G35T, KRAS:G35A, KRAS:G38A), associated with tumor resistance to Panitumumab and Cetuximab. The frequency of the most common driver mutations in the study cohort was low, indicating high heterogeneity of colorectal cancer. Analysis of the tumor mutational burden (TMB) revealed a high average value of 5.5, suggesting a significant potential of immunotherapy in colorectal cancer treatment. In the studied population, the mutational signatures SBS1, SBS5, and SBS39 were the most common.Conclusion. The identified diversity of genomic changes may be associated with DNA replication errors, exposure to external factors, and genetic predisposition. The pattern of mutational signatures was very similar among all patients in the studied samples, suggesting a single mutagenic factor is involved in the development of colorectal cancer in all patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410fb5988f2bc832f5b1b887492d03eded83e97b" target='_blank'>
              Genetic landscape of colorectal cancer. A pilot study in the Russian Federation
              </a>
            </td>
          <td>
            E. V. Semina, K. A. Kubasova, V. Kakotkin, S. V. Rodimov, M. Agapov
          </td>
          <td>2025-08-04</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mature T-cell leukemias and lymphomas (mTCL) comprise a clinically and genetically heterogeneous group of lymphoid malignancies. Most subtypes of peripheral T-cell lymphomas and leukemic T-cell malignancies show an aggressive clinical course and poor prognosis. Thus, these diseases urgently require novel therapeutic strategies. Taking advantage of recent progress deciphering the genetic basis of mTCL, we generated a comprehensive database of genetic alterations from 1 825 patients with mTCL and utilized bioinformatic methodology developed to support treatment decisions in molecular tumorboards to identify novel potential therapeutics. Based on the results of this approach, we performed a systematic in vitro drug screen in molecularly characterized cell lines of mTCL to assess the activity of potential therapeutics. Of these, the cell cycle regulator WEE1 stood out as a novel therapeutic target in a genetically defined subgroup of mTCL. Indeed, WEE1 kinase inhibitors potently induced replication stress, premature mitotic entry, accumulation of DNA damage and induction of apoptosis in mTCL cell lines. Building on the single-agent activity of clinical grade WEE1 inhibitors, we next explored potential drug combination strategies through mechanistic studies. With this, we identified strong synergistic effects of combined WEE1 and JAK inhibition in genetically defined subtypes of mTCL, including primary patient samples of T-cell prolymphocytic leukemia. In summary, our study provides a comprehensive overview of the genetic landscape in mTCL and potential therapeutics in genetically defined subtypes. As such, we identified synergistic effects of dual WEE1 and JAK inhibition in cases with altered JAK/STAT signaling as a blueprint for clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a1d19178c29c05a81757105f01cb9db0fb931d" target='_blank'>
              Discovery of molecularly-informed therapeutic strategies for mature T-cell leukemias and lymphomas.
              </a>
            </td>
          <td>
            N. Schmidt, Kevin Kornrumpf, L. Ries, K. Markus, N. Ksionsko, P. Mazzeo, M. Lohrberg, Maren Sitte, Gabriela Salinas-Riester, T. Braun, Annika Dechow, Marco Herling, A. Ianevski, T. Aittokallio, B. Chapuy, Holger Bastians, Gerald Wulf, Tim Beissbarth, Jürgen Dönitz, R. Koch
          </td>
          <td>2025-08-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Purpose Drug resistance severely compromises therapeutic efficacy in hepatocellular carcinoma (HCC); however, the selection of precise treatment strategies for patients remains a critical unmet clinical need. This study investigated PANoptosis-related mechanisms underlying HCC progression to identify actionable therapeutic targets and optimize patient-specific treatment outcomes. Patients and Methods Multi-omics analysis (single-cell/bulk RNA sequencing) combined with machine learning was used to identify the PANoptosis-related prognostic features. The association of PANoptosis-related expression with the tumor immune microenvironment and drugs was explored using bioinformatic analysis and experimental studies. Results High PANoptosis risk exhibited immunosuppressive microenvironments and therapeutic resistance. The PANoptosis-related gene YIF1B has emerged as a dual prognostic biomarker and tumor driver that promotes proliferation, and is linked to immune dysfunction and drug resistance. Conclusion YIF1B may be a promising therapeutic target. This PANoptosis framework bridges molecular mechanisms to clinical management, offering strategies for personalized HCC therapy and overcoming treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24c213b38fc84a86ff385bba02fbc05b8cafc8c" target='_blank'>
              Single-Cell RNA Sequencing Integrated with Bulk-RNA Sequencing Analysis Reveals Prognostic Signatures Based on PANoptosis in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Jiyin Wang, Xue Yin, Ziyi Li, Pu Liang, Yipeng Wang, Xinglin Li, Wenying Qiao, Chaoyang Xiong, Minghang Yu, Xiaoyan Ding, Xi Wang
          </td>
          <td>2025-07-29</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background DDX27, a member of the DEAD-box RNA helicase family, plays a pivotal role in RNA metabolism and is essential for diverse cellular processes, including transcription, pre-mRNA splicing, translation, and ribosome biogenesis. Recent findings have implicated DDX27 as a substantial contributor to tumorigenesis and cancer progression across various malignancies, establishing its significance as a molecular hub that interacts with key oncogenic partners such as major vault protein (MVP) and nucleophosmin 1 (NPM1). Methods We conducted systematic search in the following comprehensive academic databases: PubMed, MEDLINE or Web of Science. The keywords such as DDX27, DEAD-box protein 27, RNA helicase DDX27 and cancer, tumor or carcinoma were used for searching. This review consolidates the existing literature on DDX27, examining its structural features and biological functions within the context of tumorigenesis. We systematically explore the molecular mechanisms by which DDX27 influences tumor development and progression, focusing particularly on its roles across different cancer types, including colorectal cancer (CRC), gastric cancer (GC), breast cancer (BC), hepatocellular carcinoma (HCC), and oral squamous cell carcinoma (OSCC). Key molecular mechanisms such as NF-κB activation and ERK1/2 phosphorylation involved in DDX27-related pathways are discussed. Results Our comprehensive summary elucidates the context-dependent roles of DDX27 across various cancers, highlighting its associations with advanced disease stages, metastasis, and therapeutic resistance. We also assess the potential of DDX27 as a diagnostic and prognostic biomarker, correlating its expression levels with negative clinical outcomes. Conclusion Novel therapeutic strategies targeting DDX27 are proposed, including RNA interference techniques (siRNA and shRNA), miRNA-based therapies (miR-617 mimics), pathway modulation, and synthetic lethality approaches. Furthermore, we identify notable limitations in current research surrounding DDX27 and offer potential avenues for future investigation. These innovative strategies present significant promise for the development of precision cancer therapies aimed at improving treatment outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67002d435ab4b67914e4176d8c14913d0cf8ce8b" target='_blank'>
              DDX27 in cancer: molecular mechanisms, clinical implications, and therapeutic potential
              </a>
            </td>
          <td>
            Le Yang, Simon Wing Fai Mok, Hua Li, Io Nam Wong, Li Jun Yang
          </td>
          <td>2025-08-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal, aggressive malignancy. Little is known about the evolutionary trajectories of HCC and how clinical decision-making could be informed based on biopsies of the initial tumour. Here, we report the whole-exome sequencing of a unique series of resected HCC tumours and matched recurrences. This cohort included patients who received a liver transplant and who were immunosuppressed at time of recurrence, in comparison to patients who underwent liver resection for HCC and immunocompetent at time of recurrence, therefore facilitating analyses of immune selection in driving evolutionary divergence. We find extensive evolutionary divergence between baseline and recurrent tumours, with the majority of mutations in our cohort being private, in the process informing sampling guidelines for precision oncology in this disease. We also find no evidence that immunosuppression relaxes immune selection pressures, given the absence of a genomic footprint reflecting the presentation of neoantigens or altered dynamics of genomic evolution. We attribute this to the presence of genetic lesions that confer the capabilities of immune evasion in these tumours prior to transplantation, and then validate the link between immune selection pressures and the emergence of these lesions in publicly available HCC datasets. Our findings point to HCC as a cancer with extensive evolutionary divergence over time, partly defined by an irreversible, genetically determined trajectory towards immune escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9c9a24625027fee25b759620a813b537ff8ac7" target='_blank'>
              Evolutionary dynamics of recurrent hepatocellular carcinoma under divergent immune selection pressures
              </a>
            </td>
          <td>
            A. Chakravarthy, Elisa Pasini, Xun Zhao, Jeffrey To, Shu Yi (Roxana) Shen, Sandra Fischer, A. Ghanekar, Arndt Vogel, Robert C. Grant, Jennifer J Knox, Gonzalo Sapisochin, Gregory J. Gores, Daniel De Carvalho, Mamatha Bhat
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a common feature of human cancers and is linked to tumour progression, drug resistance, and metastasis1–6. Here we examine the impact of WGD on somatic evolution and immune evasion at single-cell resolution in patient tumours. Using single-cell whole-genome sequencing, we analysed 70 high-grade serous ovarian cancer samples from 41 patients (30,260 tumour genomes) and observed near-ubiquitous evidence that WGD is an ongoing mutational process. WGD was associated with increased cell–cell diversity and higher rates of chromosomal missegregation and consequent micronucleation. We developed a mutation-based WGD timing method called doubleTime to delineate specific modes by which WGD can drive tumour evolution, including early fixation followed by considerable diversification, multiple parallel WGD events on a pre-existing background of copy-number diversity, and evolutionarily late WGD in small clones and individual cells. Furthermore, using matched single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signalling and cGAS-STING pathway activation result from ongoing chromosomal instability, but this is restricted to predominantly diploid tumours (WGD-low). By contrast, predominantly WGD tumours (WGD-high), despite increased missegregation, exhibited cell-cycle dysregulation, STING1 repression, and immunosuppressive phenotypic states. Together, these findings establish WGD as an ongoing mutational process that promotes evolvability and dysregulated immunity in high-grade serous ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Background Spermatogenesis is a pivotal biological process for the precise transmission of paternal genetic information, governed by a highly complex and dynamically regulated testicular microenvironment. Although mammalian research has characterized germ cell development at the single-cell level, differences in reproductive strategies limit the relevance of these findings to avian species. Results We employed single-cell RNA sequencing (scRNA-seq) to analyze the testes of the “Yufen 1” H line roosters at five distinct developmental stages: birth, rapid testicular development, sexual maturity, physical maturity, and senescence. By constructing a single-cell transcriptomic atlas, we identified ten somatic cell subtypes and four germ cell subtypes, thereby elucidating the dynamic changes in gene expression during spermatogenesis. Notably, our findings indicate that meiosis initiates relatively early in chickens, with the formation of the blood-testis barrier being closely associated with pachytene spermatocytes. Additionally, the testicular microenvironment undergoes age-related adaptive changes. Furthermore, we observed that support cells at 20 and 80 weeks of age exhibit similar transcriptional profiles, while macrophages and T cells play a pivotal role in the formation of the testicular cords and vascular networks during the early developmental stages. Conclusion This study offers a comprehensive atlas of testicular development in chickens, elucidating the sequential cell fate transitions from spermatogonial stem cells to mature sperm, alongside the dynamic and intricate developmental trajectories of somatic cells within the testicular microenvironment. These findings present novel insights into avian testicular development and establish a theoretical foundation for future research in reproductive biology and breeding strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11944-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318aed180cb59e08dde10af23502e4ddc7594b71" target='_blank'>
              Single-cell RNA sequencing uncovers dynamic roadmap during chicken spermatogenesis
              </a>
            </td>
          <td>
            Haishan Guo, Jianing Liu, Mingzhen Xu, Tingqi Zhu, Shuangxing Liu, Pingquan Liu, Huayuan Liu, Shi Tang, Zhunan Li, Weiwei Jin, Donghua Li, Yadong Tian, Xiangtao Kang, Guirong Sun
          </td>
          <td>2025-08-13</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id=" 
Background: Accurate and early identification of lung cancer biomarkers is crucial for effective management yet remains a clinical challenge. CRISPR-based technologies provide highly sensitive and specific tools that can be adapted for both diagnostic and therapeutic use in oncology.
Methods: A prospective study was conducted including 120 patients with confirmed lung cancer and 60 healthy individuals as controls at a tertiary care center. A CRISPR-Cas12a diagnostic platform was developed to detect EGFR exon 19 deletions, KRAS G12C mutations, and ALK rearrangements in tumor tissue and plasma samples. In parallel, CRISPR-Cas9 editing was applied to patient-derived lung cancer cell cultures to selectively disrupt EGFR mutant alleles. The diagnostic performance of the CRISPR assay was compared against conventional PCR and next-generation sequencing (NGS).
Results: The CRISPR-Cas12a assay successfully detected as few as 10 DNA copies/µL within one hour. In tissue samples, the assay achieved 96.7% sensitivity and 98.3% specificity relative to NGS. In circulating tumor DNA extracted from plasma, sensitivity reached 92.1%. Functional assays demonstrated that CRISPR-Cas9-mediated disruption of EGFR mutations reduced cell viability by approximately 65% and suppressed downstream oncogenic signaling pathways.
Conclusion: CRISPR-based strategies demonstrate considerable promise for both precise detection and targeted intervention in lung cancer. The integration of CRISPR diagnostics with gene-editing applications could significantly advance precision oncology by enabling earlier diagnosis and patient-tailored therapeutic options.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66e666da19106418ca11e5bfbe956b66a8dd301" target='_blank'>
              CRISPR-BASED APPROACHES FOR THE DETECTION AND TARGETING OF LUNG CANCER BIOMARKERS: A NEW ERA IN PRECISION ONCOLOGY
              </a>
            </td>
          <td>
            Ibtasam Mazhar, Maimona Sadia, Muhammad Hassam Saleem, Ateeqah Siddique, Areeba Musferah, Areej Safdar, Anood Malik
          </td>
          <td>2025-08-31</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 p53 is a transcription factor that regulates the expression of target genes involved in anti-proliferative processes such as cell cycle arrest, apoptosis, and senescence, thereby establishing its canonical role as a tumor suppressor. Due to its substantial network of target genes, p53 is also involved in non-canonical tumor suppression pathways that are yet to be fully explored. During the search of these pathways, we identified one of the methyltransferases as a novel p53 target gene. It’s been reported that this methyltransferase is downregulated in various cancers, suggesting potential tumor suppressive function. However, the specific target of this methyltransferase and its role in the mechanism of p53-mediated tumor suppression remains elusive. In this study, we show that p53 directly binds to intron 1 of this methyltransferase gene and activates its transcription and protein expression. Furthermore, we identify dihydrolipoic acid (DHLA), an important redox cofactor, as a novel target of this methyltransferase. We show that this methyltransferase enhances cellular sensitivity to oxidative stress-induced cell death by potentially methylating DHLA. Our studies reveal a novel role for p53 in regulating the well-known antioxidant DHLA to control cell death and suggest potential therapeutic strategies in promoting cancer cell death through p53 activation drugs.



 Fatima Alaydaroos, Yoko Itahana, Koji Itahana. p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P18.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1c8311abf9a4ec56ed8b5af3129410222f89d5" target='_blank'>
              Abstract P18: p53-induced Methyltransferase Increases Cancer Cell Vulnerability to Cell Death
              </a>
            </td>
          <td>
            Fatima Alaydaroos, Yoko Itahana, Koji Itahana
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Cancer remains a major global health challenge. This review focuses on DNA-PKcs, a protein that enables cancer cells to repair DNA damage caused by radiotherapy and chemotherapy, promoting tumor survival. We synthesize recent evidence on how DNA-PKcs drives tumor progression and therapy resistance, highlighting its potential as a therapeutic target. Current advances in DNA-PKcs inhibitors are examined, including their promise in combination therapies and hurdles in clinical development. Uniquely, this review connects DNA-PKcs to emerging roles in immunomodulation and cancer metabolism, evaluates next-generation inhibitors, and proposes solutions to overcome translational challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f0939416a1ca4dadfa5d4cc33f441bbed022050" target='_blank'>
              Unlocking the Therapeutic Potential of DNA-PKcs in Cancer: Comprehensive Insights into Mechanisms and Clinical Applications
              </a>
            </td>
          <td>
            Tong Zheng, Chao Sun, Cijun Yun, Hui Wang, Xiongxiong Liu
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The prevalence and death rates of early-onset colorectal cancer (EOCRC) have been increasing at an alarming rate since 1994. Compared with late-onset colorectal cancer (LOCRC), EOCRC is more aggressive and resistant to treatment. Despite
 KRAS
 mutations and the mitogen-activated protein kinase/extracellular signal-related kinase (MAPK/ERK) pathway's significant role in colorectal cancer tumorigenesis and progression, these mutations are less frequent in patients with EOCRC.




 In this study, we conducted proteotranscriptomic profiling of two independent cohorts (MD Anderson for testing and The Cancer Genome Atlas for validation), comparing EOCRC and LOCRC tumors. Our focus was on sporadic, microsatellite-stable, treatment-naïve patients with early-onset (<50 years) and late-onset (>60 years) colorectal adenocarcinomas.




 Our analysis revealed consistent molecular differences between EOCRC and LOCRC, particularly in pathway enrichment profiles. The two pathways with the highest levels of significant gene set enrichment analysis in EOCRC tumors were the downregulation of epithelial-mesenchymal transition and upregulation of Wnt/β-catenin signaling pathways. Both testing and validation cohorts showed a consistent enrichment in hallmark KRAS signaling. However, using the novel KRAS-ERK gene signatures to better capture the pathway signaling functionality, we identified a consistent upregulation of MAPK signaling specifically in EOCRC tumors. Further analysis of this enrichment in wild-type RAS GTPase (
 KRAS
 ,
 NRAS
 , and
 HRAS
 ) tumors confirmed significant MAPK upregulation in EOCRC wild-type RAS tumors (
 P
 < .001). Proteomic analysis further supported these transcriptomic findings, reinforcing the molecular distinctions in EOCRC.




 This study not only elucidates the unique molecular attributes of EOCRC but also lays the groundwork for exploring MAPK-targeted therapies specifically for this age group. Given the increasing prevalence of EOCRC, our findings highlight the need for larger investigations into MAPK upregulation in EOCRC and its potential as a target for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5365b8787a6860734caebe70b8eefa54d1dc39aa" target='_blank'>
              Proteotranscriptomic Analyses of Microsatellite-Stable Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            J. Alshenaifi, Giulia Maddalena, Jessica C. Lal, Oluwadara Coker, Hey Min Lee, O. Villarreal, G. Manyam, Alaa M. Mohamed, G. Vetere, Micheal White, N. Ajami, Preeti Kanikarla Marie, Alexey Sorokin, David Menter, Y. N. You, Andy Futreal, S. Kopetz
          </td>
          <td>2025-08-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Mutant KRAS has been implicated in driving a quarter of all cancer types. Although inhibition of the KRASG12C mutant protein has shown clinical promise, there is still a need for therapies that overcome resistance and target non-KRASG12C mutations. KRAS activates downstream MYC, which is also a challenging-to-drug oncoprotein. We have developed an "inverted" RNAi molecule with the passenger strand of a MYC-targeting siRNA fused to the guide strand of a KRAS-targeting siRNA. The chimeric molecule simultaneously inhibits KRAS and MYC, showing marked improvements in efficacy beyond the individual siRNA components. This effect is mediated by 5'-dT overhangs following endosomal metabolism. The synergistic RNAi activity led to a >10-40-fold improvement in inhibiting cancer viability in vitro. When conjugated to an epidermal growth factor receptor (EGFR)-targeting ligand, the chimeric siRNA was delivered to and internalized by tumor cells. As compared with individual targeting siRNAs, the chimeric design resulted in considerably improved metabolic stability in tumors, enhanced silencing of both oncogenes, and reduced tumor progression in multiple cancer models. This inverted chimeric design establishes proof-of-concept for ligand-directed, dual-silencing of KRAS and MYC in cancer and constitutes an innovative molecular strategy for co-targeting any two genes of interest, which has broad implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1afbec09737211f6f2ff3789e350810d140c4a0" target='_blank'>
              Inverted chimeric RNAi molecules synergistically co-target MYC and KRAS in KRAS-driven cancers.
              </a>
            </td>
          <td>
            Y. Chareddy, H. Huggins, S. S. Sahoo, Lyla Stanland, C. Gutierrez-Ford, Kristina M. Whately, Lincy Edatt, S. Azam, Matthew C Fleming, Jonah Im, A. Porrello, Imani Simmons, Jillian Perry, A. Bowers, M. Egli, C. Pecot
          </td>
          <td>2025-07-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic progression models of cancer continue to determine the roadmap of carcinogenesis, although the sequence of genetic events is inferred rather than empirically determined through longitudinal analyses. Here, we present a unique longitudinal study of oral leukoplakia lesions that transformed into carcinoma. Lesions were followed from initial diagnosis through to malignant transformation. For each lesion, biopsies at baseline, the carcinoma, and all available intermediate biopsies were studied for the presence of dysplasia, genomic copy number aberrations, and mutations in selected head and neck cancer genes. Using this information, the phylogenetic history of all carcinomas was reconstructed within the context of applied interventions. In total, 71 biopsies of 21 lesions were studied. The median time to malignant transformation was 60 months. Oral carcinogenesis emerged as a multi-(sub)clonal process. Treatment interventions appeared to impact clonal selection, although lesions always remained after excision. Notably, the lesions demonstrated different routes of progression. A canonical pattern of progression was observed, characterized by longitudinal accumulation of abnormal morphology and genetic changes. However, some lesions followed an alternative, stable pattern of oncogenic progression, with no apparent increase in morphological changes or accumulation of genetic aberrations. This latter pattern appeared to be associated with the development of previously identified 'copy number quiet' head and neck tumors. This study provides a novel perspective on the temporal evolution of oral leukoplakia and the different routes to cancer progression and highlights the key role of multiclonal field cancerization in lesion recurrence and cancer development. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ac89f69d77b6dd7fcbbebb782a481d2786ada6" target='_blank'>
              New insights into the genetic progression of cancer through longitudinal analysis of oral lesions.
              </a>
            </td>
          <td>
            L. Wils, J. Poell, A. Brink, Laura A N Peferoen, I. Evren, E. Brouns, Jan G. de Visscher, Erik H van der Meij, Elisabeth Bloemena, R. Brakenhoff
          </td>
          <td>2025-07-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE Tumor comprehensive genomic profiling (CGP) may detect potential germline pathogenic/likely pathogenic (P/LP) alterations as secondary findings. We analyzed the frequency of potentially germline variants and large rearrangements (LRs) in the RATIONAL study, an Italian multicenter, observational clinical trial that collects next-generation sequencing–based tumor profiling data, and evaluated how these findings were managed by the enrolling centers. PATIENTS AND METHODS Patients prospectively enrolled in the pathway-B of the RATIONAL study and undergoing CGP with the FoundationOne CDx assays were included in the analysis. Potentially germline variants detected in 40 cancer susceptibility genes (CSGs) were classified in three classes with different actionability, most (MA), high (HA), and standard (SA), on the basis of penetrance, mutational spectrum, and intervention for prevention/early detection. RESULTS On the basis of the European Society of Medical Oncology recommendations, we identified 225 potentially germline P/LP variants in 193/1,339 (14.4%) enrolled patients. In particular, 62/225 (27.5%) variants were detected in genes classified as MA-CSG class, 53/225 (23.6%) in genes belonging to the HA-CSG class, and 110/225 (48.9%) in the SA-CSG class. In addition, we detected 58 LRs in the 16/40 CSGs in 53/1,339 (3.95%) patients. Information about germline-focused analysis and follow-up was available for 99 patients with potentially germline variants. Surprisingly, 95/99 (96%) patients were not referred to oncogenetic consultation and follow-up, including 30/32 (93.75%) patients with variants in the MA-CSG class. CONCLUSION Our data confirm the utility of CGP for the identification of potentially germline variants in CSGs, highlighting the importance of reporting LRs in addition to single-nucleotide variants and insertions/deletions. However, our findings also demonstrate a relative lack of knowledge of the implications of germline findings detected on tumor-only sequencing among oncologists and underline the need for specific training in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3205dd2d3d3b99613c9aba83f72f519f1c6e231" target='_blank'>
              Germline Findings From Tumor-Only Comprehensive Genomic Profiling in the RATIONAL Study: A Missed Opportunity?
              </a>
            </td>
          <td>
            R. Esposito Abate, Alessandro Morabito, Michele Milella, F. Tabbò, Valentina Guarneri, G. Pelizzari, I. Rapposelli, R. Berardi, Lucio Buffoni, Elisa Bennicelli, F. Zanelli, C. Tondini, Laura Attademo, T. Latiano, Salvatore Corallo, Giancarlo Pruneri, F. Marmorino, O. Caffo, L. Antonuzzo, Simona Tessitore, Silvia Novello, G. Curigliano, Carmine Pinto, Nicola Normanno, Antonella De Luca
          </td>
          <td>2025-09-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer.
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Abstract Cytosine DNA methylation patterns vary widely across eukaryotes, with its ancestral roles being understood to have included both transposable element (TE) silencing and host gene regulation. To further explore these claims, in this study, we reevaluate the evolutionary origins of DNA methyltransferases and characterize the roles of cytosine methylation on underexplored lineages, including the amoebozoan Acanthamoeba castellanii, the glaucophyte Cyanophora paradoxa, and the heterolobosean Naegleria gruberi. Our analysis of DNA methyltransferase evolution reveals a rich ancestral eukaryotic repertoire, with several eukaryotic lineages likely subsequently acquiring enzymes through lateral gene transfer (LGT). In the three species examined, DNA methylation is enriched on young TEs and silenced genes, suggesting an ancestral repressive function, without the transcription-linked gene body methylation of plants and animals. Consistent with this link with silencing, methylated genomic regions co-localize with heterochromatin marks, including H3K9me3 and H3K27me3. Notably, the closest homologs of many of the silenced, methylated genes in diverse eukaryotes belong to viruses, including giant viruses. Given the widespread occurrence of this pattern across diverse eukaryotic groups, we propose that cytosine methylation was a silencing mechanism originally acquired from bacterial donors, which was used to mitigate the expression of both transposable and viral elements, and that this function may persist in creating a permissive atmosphere for LGT in diverse eukaryotic lineages. These findings further highlight the importance of epigenetic information to annotate eukaryotic genomes, as it helps delimit potentially adaptive LGTs from silenced parasitic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48579c32d97033a0fd54ac2bedb8cb3bebb700ed" target='_blank'>
              Repressive Cytosine Methylation is a Marker of Viral Gene Transfer Across Divergent Eukaryotes
              </a>
            </td>
          <td>
            Luke A. Sarre, Giselle Azucena Gastellou Peralta, Pedro Romero Charria, Vladimir Ovchinnikov, Alex de Mendoza
          </td>
          <td>2025-07-25</td>
          <td>Molecular Biology and Evolution</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Chimeric Antigen Receptor (CAR)-T cell therapy represents a transformative breakthrough in cancer immunotherapy by harnessing the adaptive immune system to selectively eradicate cancer cells. Pioneering advances in the treatment of hematological malignancies have led to the FDA approval of several CAR-T cell therapies, particularly for patients with relapsed or refractory disease. This success is a result of continuous refinements in CAR architecture, which have evolved from early prototypes with limited therapeutic efficacy to advanced next-generation receptors that incorporate co-stimulatory domains, cytokine signaling, safety switches, and precision control mechanisms. This review elucidates the fundamental rationale behind CAR-T cell development and addresses key biological challenges encountered. Advances in receptor engineering, metabolic reprogramming, and optimized immune signaling have markedly enhanced the persistence, antitumor activity, and safety profiles of CAR-T cells. Additionally, emerging genetic engineering tools, including CRISPR, base editing, prime editing, and RNA and epigenome editing, hold promise for reducing immunogenicity and minimizing the risk of graft-versus-host disease (GVHD). However, CAR-T cell therapy continues to face several challenges, including severe side effects such as cytokine release syndrome (CRS) and neurotoxicity, inconsistent therapeutic responses, and high production costs. To overcome these barriers, novel approaches are under development that include generating CAR-T cells in vivo, utilizing logic-gated CAR systems, and expanding CAR platforms to include other immune effector cells, such as natural killer cells (CAR-NK) and macrophages (CAR-M). The future of CAR-based therapies is expected to integrate synthetic biology, immune checkpoint modulation, and innovative delivery methods to enhance both therapeutic efficacy and safety. This review synthesizes current knowledge and emerging strategies to guide future advancements aimed at expanding the applicability of CAR therapy to various cancer types and potentially other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39a6e89e45ceb363e268bc4fe2c0c1a9ccb9b20e" target='_blank'>
              Tracing the development of CAR-T cell design: from concept to next-generation platforms
              </a>
            </td>
          <td>
            A. Alsaieedi, Kawther A Zaher
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CDKN2A/MTAP co-deletion occurs frequently in non-small cell lung cancer and other solid tumors, including glioblastoma and pancreatic ductal adenocarcinoma. Lung cancer remains the leading cause of cancer-related mortality, and fewer than 15% of glioblastoma or pancreatic cancer patients survive 5 years, underscoring the need for more effective therapies. PRMT5 is a synthetic-lethal dependency in MTAP-null tumors and an attractive therapeutic target for CDKN2A/MTAP-deleted cancers. A new revolutionary class of inhibitors, referred to as MTA-cooperative PRMT5 inhibitors, has shown promising results in ongoing early phase clinical trials. Nonetheless, effective cancer treatment typically requires therapeutic combinations to improve response rates and defeat emergent resistant clones. Thus, we sought to determine whether perturbation of other pathways could improve the efficacy of MTA-cooperative PRMT5 inhibitors. A paralog and single gene targeting CRISPR library was used to screen MTAP-deleted cancers in the presence or absence of MTA-cooperative PRMT5 inhibitors. Loss of several genes sensitized to PRMT5 inhibition, including members of the MAPK pathway. Chemical inhibition of MAPK pathway members using KRAS, MEK, ERK, and RAF inhibitors synergized with PRMT5 inhibition to kill CDKN2A/MTAP-null, RAS-active tumors. Further, MTA-cooperative PRMT5 inhibitors combined with either KRAS or RAF inhibitors led to complete responses in vivo, emphasizing the potential benefit for patients. Lastly, cell lines resistant to KRAS inhibition were not resistant to MTA-cooperative PRMT5 inhibitors and vice versa, suggesting non-cross-reactive mechanisms of resistance. Overall, this study identifies therapeutic combinations with MTA-cooperative PRMT5 inhibitors that may offer significant benefit to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa2f9d69996913d906ee879ebdccd11283112c02" target='_blank'>
              CRISPR-Drug Combinatorial Screening Identifies Effective Combination Treatments for MTAP-deleted Cancer.
              </a>
            </td>
          <td>
            Nikola Knoll, Sarah Masser, Blanka Bordas, R. Ebright, Guangyan Li, Devishi Kesar, E. Destefanis, Nicholas Kania, Diego J Rodriguez, Jayu Jen, Sydney E Zagar, Caleb Mensah, Zixin Chen, Samuel J Moffitt, Erhumuoghene M Enakireru, Yao He, Baomou Feng, Mira K Chokshi, Cyrus Y Jin, Srivatsan Raghavan, William R Sellers, K. Mulvaney
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="INTRODUCTION
Parkinson's disease (PD) is a prevalent neurodegenerative disorder characterized by the progressive loss of dopaminergic neuron. Although the role of immunity in PD has been increasingly recognized, the immunogenetic mechanisms underpinning its progression remain largely unresolved.


METHODS
We employed an integrative approach combining Mendelian randomization (MR), expression quantitative trait loci analysis, and single-cell RNA sequencing to investigate immune cell infiltration and transcriptional regulation in PD. Immune cell composition, pathway activation, and gene regulatory networks were assessed through single-cell gene set enrichment analysis and transcriptional correlation analyses.


RESULTS
Immune profiling revealed significant increases in naive B cells (1.22-fold), plasma cells (3.00-fold), switched memory B cells (2.85-fold), and unswitched memory B cells (6.70- fold) in PD patients compared to controls (p < 0.001). MR analysis identified five causal genes- CYTH4, FGR, LRRK2, RIN3, and SAT1- associated with monocyte, neutrophil, and B cell infiltration. SAT1 (OR: 1.529; 95% CI: 1.018-2.297) and RIN3 (OR: 1.222; 95% CI: 1.039- 1.437) showed strong associations with PD risk (p < 0.01). SAT1 positively correlated with PARK7 and regulated reactive oxygen species signaling, while FGR negatively correlated with ABCA4, influencing lipid metabolism and immune responses.


DISCUSSION
These findings highlight distinct immunogenetic mechanisms driving PD progression. The SAT1-PARK7 axis appears to modulate oxidative stress and neuroinflammation, whereas the FGR-ABCA4 interaction may affect metabolic and immune pathways. While the study is limited by population heterogeneity and the challenges of inferring causality, it provides mechanistic insights into immune contributions to PD.


CONCLUSION
Our integrative genomic analysis identified novel regulatory networks involving immune-related genes in PD, offering potential targets for mechanistic understanding and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ada28f9fa3780d00cdec8a4ef08863337dfd3db" target='_blank'>
              Exploring Immunogenetic Mechanisms in Parkinson's Disease Using Single-Cell Transcriptomics and Mendelian Randomization.
              </a>
            </td>
          <td>
            Dongyuan Xu, Yu Lei, Ji Wu, Keyu Chen, Songshan Chai, Nanxiang Xiong
          </td>
          <td>2025-07-21</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The impact of MYC point mutations on cancer development and progression is poorly explored, particularly compared with other MYC genetic alterations such as translocations and amplifications. MYC point mutations were first observed more than 40 years ago in lymphoid malignancies, and some of these were functionally characterized. In the following decades, only a few studies on MYC point mutations were reported until recently, when analyses of the myriad cancer-related high-throughput sequencing studies brought new life to this research topic and expanded the range of malignancies involved. However, to date, all this information can only be retrieved consulting the specific literature or navigating publicly available databases, and a proper collection and systematic description of these genetic changes is urgently needed. In this review, we run through the steps of the MYC point mutations line of research with a comprehensive illustration of the recurrent variants occurring at the MYC coding, non-coding and regulatory regions in cancer. With this work, we also aim to highlight the current gaps of knowledge to stimulate the research in this field which could ultimately result in the release of its translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d123406ed96af6e46fc4b8cbc92394a082d3e153" target='_blank'>
              MYC Point Mutations in Cancer: A Reboot and a Sequel.
              </a>
            </td>
          <td>
            Davide De Luca, Cristina Munafò, L. Lorenzi, Francesco Cucco
          </td>
          <td>2025-07-24</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Previous studies majorly focused on the role of driver gene variation in the occurrence, development and treatment of glioma. Here, we revealed the effect of signaling pathway abnormality by accumulated genetic mutations on patient survival and explored potential mechanism in glioma. We found that abnormalities of three signaling pathways, including autophagy-animal, focal adhesion and neuroactive ligand-receptor interaction, were associated with progression-free survival (PFS) in glioma. Based on the three pathways, we constructed and validated a biological pathway score (BPS) model to predict patient prognosis. Both PFS and overall survival (OS) were poorer in glioma patients with high BPS. In the glioma patients with high BPS, mutation frequencies of PTEN, TTN and EGFR were significantly increased. Immune checkpoints (PD-1, PD-L1 and TIM-3) were strikingly expressed in the high BPS patients with glioma, despite observed high levels of infiltrating immune cells. Differential expression genes in the high BPS patients were enriched in proliferation-related pathways. These results revealed potential reasons of poor prognosis in the high BPS patients with glioma. In addition, we found that low BPS patients with glioma might benefit from sevoflurane during surgery anesthesia. Overall, this study exhibited that the abnormalities of PFS-related signaling pathways by genetic mutations impacted survival of patients and revealed potential mechanism of poor PFS in glioma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-12346-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c3a6594417935ac589e785911e02c60fd97e52" target='_blank'>
              Mutational accumulation drives pathway dysregulation to predict glioma patient survival
              </a>
            </td>
          <td>
            Mengjiao Wan, Caiping Zhuang, Wei Li, Rongrong Yan, Wen Chen, Jing Cheng, Chengjin Yue, Lin Chen, Yingyao Quan
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play complex roles in the tumor microenvironment (TME) of melanoma. However, their impact on prognosis and treatment response in melanoma remains unclear. In this study, ScRNA-seq data (GSE115978) were utilized to characterize CAF heterogeneity and identify marker genes in melanoma. Prognostic CAF genes were identified from the TCGA dataset and employed to construct a risk signature, which was subsequently validated in an independent cohort (GSE65904). Mutation, copy number variation (CNV), pathway enrichment, immune infiltration, and drug sensitivity were analyzed to determine the signature’s clinical relevance. Immunohistochemistry (IHC), immunofluorescence (IF), and qPCR were performed to validate the expression of CAF signatures on clinical melanoma samples. We identified CAFs in patients with melanoma through single-cell RNA sequencing data. A 28-gene CAF signature was constructed using the Least Absolute Shrinkage and Selection Operator (LASSO) regression based on 271 prognostic CAF genes. This signature demonstrated excellent prediction accuracy for survival, with area under the curve (AUC) values of 0.737, 0.737, and 0.779 for 1-year, 3-year, and 5-year survival, respectively. The signature was an independent prognostic factor and was correlated with CNVs, and immunosuppressive TME features (reduced CD8+ T cells, M1 macrophages). Additionally, our CAF signature could predict the efficacy of multiple chemotherapy drugs and serve as a potential prognostic marker for immunotherapy. Experimental validation confirmed the expression of CAF signature genes in melanoma tissue. Our model may help predict the prognosis and response to chemotherapy and immunotherapy in patients diagnosed with melanoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14979-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2590463eef329c26ec20b4fd4f71fd42d9d7d1d3" target='_blank'>
              Combining single-cell and bulk RNA sequencing to identify CAF-related signature for prognostic prediction and treatment response in patients with melanoma
              </a>
            </td>
          <td>
            Jing Wang, Ying Song, Zifu Li, Tianxiang Gao, Wei Shen, Zijian Kang, Chong Xu
          </td>
          <td>2025-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenic alterations in fibroblast growth factor receptor (FGFR)-family proteins occur across cancers, including pediatric gliomas. Our genomic analysis of 11,635 gliomas across ages finds that 5.3% of all gliomas harbor FGFR alterations, with an incidence of almost 9% in pediatric gliomas. Alterations in FGFR proteins are differentially enriched by age, tumor grade, and histology, with FGFR1 alterations associated with glioneuronal histologies. Leveraging isogenic systems, we confirm FGFR1 alterations to induce downstream Mitogen Activated Protein Kinase (MAPK) and mTOR signaling pathways, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway and pan-FGFR inhibitors. Finally, we perform a retrospective analysis of clinical responses in children diagnosed with FGFR-altered gliomas and find that treatment with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328f68005932db97028a1454f9fd6fef0c86bb93" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations suggests potential therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            April A Apfelbaum, Eric Morin, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe Power, Margaret M Cusick, Dana Novikov, Prem Prabhakar, Robert E Jones, J. Vogelzang, Connor C Bossi, Seth Malinowski, Lewis Woodward, Tania A Jones, John Jeang, Sarah W Lamson, Jared Collins, Kelly Y Cai, Jacquelyn S Jones, Sehee Oh, Hyesung Jeon, Jinhua Wang, A. Cameron, P. Rechter, Angela De Leon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, Cornelis M. van Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, S. Buhrlage, T. Bale, Amy Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael J Eck, Lissa C. Baird, Olaf Witt, Claudia L Kleinman, Quang-De Nguyen, Denise Sheer, S. Alexandrescu, David T W Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2025-07-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad16f7c674380966cee80a45864fc4919cc810eb" target='_blank'>
              Multiplexed assays of variant effect for clinical variant interpretation.
              </a>
            </td>
          <td>
            Abbye E. McEwen, Malvika Tejura, Shawn Fayer, Lea M. Starita, Douglas M. Fowler
          </td>
          <td>2025-07-21</td>
          <td>Nature reviews. Genetics</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47887f5c85f735acc88fc608cab90528d242a67" target='_blank'>
              Baseline expression of c-Myc defines the tissue specificity of oncogenic K-Ras.
              </a>
            </td>
          <td>
            Olesja Popow, Qing Yu, Clarance Yapp, Shannon Hull, Joao A. Paulo, Benjamin Hanna, Shidong Xu, Y. Kuang, Carlie Sigel, C. Paweletz, Steve P Gygi, Kevin M. Haigis
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Background Tumor mutation burden (TMB), the total number of nonsynonymous mutations in the tumor genome, is a well-established biomarker for predicting responses to immune checkpoint blockade (ICB) therapy across various cancers. Patients with high TMB tend to exhibit better responses to ICB. Recently, targeted gene panels have been developed to estimate TMB before treatment. These panels are enriched for calcium ion-binding genes. However, a direct link between TMB and calcium ion-binding genes has not been reported in the literature to date. Methods The association between TMB and calcium ion-binding genes was analyzed using mutation data from The Cancer Genome Atlas (TCGA) database. In addition, a pan-cancer model was constructed to estimate TMB based solely on calcium ion-binding genes. The model’s predictive power for ICB response was validated using independent datasets. Finally, enrichment analysis was performed to investigate the biological connections between calcium ion-binding genes and TMB. Results Calcium ion-binding genes were enriched among the TMB-predictive model genes in 27 out of 33 cancer types. Among these, 19 cancer types exhibited strong predictive performance, with R² values greater than 0.5 in our pan-cancer model based on calcium ion-binding genes. The model effectively estimated TMB and identified ICB responders in independent datasets, including lung adenocarcinoma and melanoma. Enrichment analysis further suggested that calcium ion-binding genes may influence TMB through signal transduction pathways. Conclusions These findings establish a novel association between calcium ion-binding genes and TMB, demonstrating the feasibility of a pan-cancer TMB estimation model based on calcium ion-binding genes. This approach may enhance TMB estimation and improve ICB response prediction across multiple cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03184-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a4ff34e1fd9730096e49cd5425ebee584e0fa0" target='_blank'>
              Calcium ion-binding genes can predict tumor mutation burden and immune checkpoint blockade response in a pan-cancer model
              </a>
            </td>
          <td>
            Wan-Yu Lin, Chien-jung Huang, Yu-Chao Wang
          </td>
          <td>2025-07-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="OBJECTIVE
Immune checkpoint inhibitors (ICIs) made a big change in the treatment of hepatocellular carcinoma (HCC), thereby bringing fresh hope in terms of treatment to patients with few options until that time.


METHODS
However, ICIs still encounter resistance mechanisms in a substantial number of patients, either in primary or secondary forms. This phenomenon of resistance occurs due to various reasons that comprise but are not restricted to epigenetic alterations, immunosuppressive tumor microenvironments, and activation of TGF-β and VEGF signaling pathways.


RESULTS
In this study, we will discuss the mechanisms of ICI resistance in HCC and these emerging avenues of therapy as an approach to circumvent the resistance. The rationale and current status of ongoing trials to combine ICIs with anti-angiogenic agents, epigenetic modulating agents, and local therapies will be discussed.


DISCUSSION
In a nutshell, we indicate an immediate need for reliable predictive biomarkers and personalized treatment approaches to improve clinical outcomes. Therefore, by managing challenges and knowledge gaps, this review presents a futurist outlook while charting a course for subsequent clinical trials designed to optimize immunotherapeutic results in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db9b43b506232397a848007c0bb278eb1fda635" target='_blank'>
              Immune checkpoints in hepatocellular carcinoma and the challenges of therapeutic resistance.
              </a>
            </td>
          <td>
            Xin Tao, Xiaoxia Yang
          </td>
          <td>2025-08-31</td>
          <td>Immunopharmacology and immunotoxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objective The development of acquired endocrine resistance and reduced chemosensitivity in oestrogen receptor-positive (ER+) breast cancer presents significant challenges. Microtubule-based process-related genes (MBPRGs) play essential biological roles in the cell cycle and the development of migration. This study aimed to establish a novel prognostic signature based on MBPRGs to improve patient outcomes and offer additional treatment options for those with ER+ breast cancer. Methods Clinical data along with relevant RNA information with ER+ breast cancer were sourced from The Cancer Genome Atlas and the Molecular Taxonomy of Breast Cancer International Consortium. Consensus clustering was subsequently utilised to identify new molecular subgroups. Evaluations of the tumour immune microenvironment and immune status of these subgroups were performed via ESTIMATE, CIBERSORT, MCP, and ssGSEA. Additionally, functional analyses were conducted to investigate the underlying mechanisms involved. Prognostic risk models were developed via random forest, support vector machines and the least absolute shrinkage and selection operator algorithm. Single-cell analysis revealed differences in the expression levels of key genes among various cell types. Western blotting was used to measure protein levels in breast cancer cell lines. Immunohistochemical staining was used to assess protein expression in paraffin-embedded tissues, and Kaplan–Meier survival curves were generated to evaluate survival differences between the high- and low-expression groups of key genes. Transwell and cell viability assays were used to examine the biological functions of CHORDC1. Results Two molecular subgroups with significantly different survival outcomes were identified. Longer survival was linked to a high immune score, low tumour purity, a greater presence of immune infiltrating cells, and an overall positive immune status. Risk models derived from MBPRGs exhibited strong potential for predicting survival in patients with ER+ breast cancer. Key genes had elevated protein levels in differentiated breast cancer cell lines, and elevated CHORDC1 expression was linked to a tendency towards a worse outcome in patients with ER+ breast cancer. Silencing CHORDC1 inhibited cell viability and invasion, reducing sensitivity to tamoxifen and paclitaxel in vitro. Conclusion MBPRG expression is linked to the immune microenvironment and drug resistance in ER+ breast cancer patients, providing a reliable prognostic indicator for this group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ae1cb68449d2685b386413614d1719f45655c6" target='_blank'>
              Association of microtubule-based processes gene expression with immune microenvironment and its predictive value for drug response in oestrogen receptor-positive breast cancer
              </a>
            </td>
          <td>
            Zhenfeng Huang, Minghui Zhang, Nana Zhang, Mengyao Zeng, Yao Qian, Meng Zhu, Xiangyan Meng, Ming Shan, Guoqiang Zhang, Feng Liu
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31234e022d4f1b74becb6000088fd6648f5d37a8" target='_blank'>
              The Diagnostic Role and Potential Pharmacological Value of DDR1 in Pan-Cancer.
              </a>
            </td>
          <td>
            Yi Yu, Yonggang Tian, DeKui Zhang
          </td>
          <td>2025-07-16</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743f697012228792a8f0e4e8d5e5913ecf0ed66d" target='_blank'>
              Targeting epigenetic modifications as an emerging immunotherapeutic strategy for cancers.
              </a>
            </td>
          <td>
            R. I. Kumar, Kavya Jain, Karan Raj Rai, Harshnna Gururajan, Koustav Sarkar
          </td>
          <td>2025-08-19</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal diagnostics have evolved significantly over the past several decades, from classical karyotyping to high-resolution genomic technologies. This review traces the development and integration of cytogenetic toolskaryotyping, fluorescence in situ hybridization (FISH), chromosomal microarray analysis (CMA), and whole genome sequencing (WGS)in modern clinical diagnostics. We examine each technologys diagnostic utility, limitations, and role in clinical workflows. Importantly, insights from a practicing clinical cytogeneticist highlight the increasing automation of laboratory processes and the continued relevance of karyotyping in routine diagnostics due to its cost-effectiveness and structural resolution. As genome-wide testing becomes more accessible, we explore future directions including the rise of artificial intelligence (AI), non-invasive screening technologies, and emerging ethical challenges surrounding incidental findings, data privacy, and equitable access. Together, these perspectives underscore the importance of combining technological innovation with clinical expertise and ethical foresight to guide the future of precision diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d4b14a05606752d08d57c2ff217d9ddb756ec" target='_blank'>
              From Chromosome Karyotyping to Genomic Precision: The Evolution of Diagnostic Technologies in Medicine
              </a>
            </td>
          <td>
            Run Liu
          </td>
          <td>2025-08-13</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) accounts for approximately 15% of primary lung carcinomas and has the poorest outcome in all subtypes of lung cancer. The major hurdle for SCLC treatment failure is resistance to platinum-based chemotherapy. Therefore, an unmet need is to discover new targets that promote SCLC progression and chemoresistance. Based on the signature of ubiquitination-related genes (URGs), differentially expressed genes between cisplatin-resistant and cisplatin-sensitive SCLC cell lines were identified using the Genomics of Drug Sensitivity in Cancer (GDSC) database. The URGs associated with the prognosis were further screened by Cox and LASSO regression analyses, as well as a Kaplan-Meier survival analysis. The E3 ligase NEDD4 was identified to be associated with cisplatin resistance, poor prognosis and tumor metastasis in SCLC. The functional enrichment analysis indicated that the functions and pathways regulated by NEDD4 were enriched in cell proliferation, cell invasion, as well as ubiquitination and PI3K-AKT pathways in SCLC. The knockdown and overexpression of NEDD4 demonstrated that NEDD4 induced the phosphorylation of AKT in SCLC cells. Cell viability, wound healing and transwell invasion assays demonstrated that NEDD4 promoted the proliferation, chemoresistance and invasion of SCLC cells. These results suggest that NEDD4 is a biomarker of a poor prognosis for SCLC, and that it promotes AKT activation, SCLC progression and chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d5f3f59a703f4ed1023547136b525bc0f4e2e7" target='_blank'>
              NEDD4 is a biomarker of a poor prognosis that contributes to the progression and chemoresistance in small cell lung cancer.
              </a>
            </td>
          <td>
            Rong Gao, Yuting Bian, Yongguang Wang, Yani Zhang, Qizhi Zhu, Jinfu Nie, Zongtao Hu, Hongzhi Wang, Bo Hong
          </td>
          <td>2025-08-06</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim. To identify clinical, molecular, genetic, and immunological predictors of response to PARP inhibitor olaparib therapy in patients with metastatic castration-resistant prostate cancer (mCRPC) harboring germline and somatic mutations in homologous recombination repair (HRR) genes.Materials and methods. A prospective analysis was conducted on data from 39 patients with mCRPC and HRR gene mutations (BRCA1/2, ATM, CHEK2, CDK12, among others) receiving оlaparib therapy after prior treatment with androgen receptor signaling inhibitors. Diagnoses were histologically verified according to the current WHO classification (5th edition). Genetic status was determined by next-generation sequencing (NGS). The tumor immune microenvironment was evaluated by immunohistochemical analysis assessing expression of CD4, CD8, CD68, CD163 markers and tumor-infiltrating lymphocyte (TILs) levels. Statistical analysis involved Cox proportional-hazards regression and Kaplan–Meier survival analysis.Results. The BRCA2 mutation was the most frequently detected alteration (61 %), correlating with increased expression of CD163 and decreased CD68 expression. The most significant clinical predictors associated with improved progressionfree survival during оlaparib therapy included BRCA2 mutation positivity, patient age under 64 years, initially metastatic disease presentation, and ≥90 % reduction in prostate-specific antigen levels at 3 months post-treatment initiation. Immunological markers indicative of favorable therapeutic response included elevated levels of M2-polarized macrophages (CD163 ≥30 %) and a high CD163/CD68 ratio (≥2.83). Conversely, robust prior response to еnzalutamide therapy and administration of docetaxel during hormone-sensitive stages significantly worsened the prognosis for olaparib efficacy.Conclusion. Olaparib therapy is effective in patients with HRRm mCRPC, with particularly effectiveness in BRCA2m tumors.  Clinical (age, disease dissemination pattern, prostate-specific antigen dynamics), immunological (CD163 expression and CD163/CD68 ratio), and molecular and genetic tumor characteristics should be considered when selecting patients for olaparib treatment, enhancing personalized therapeutic strategies in mCRPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bedca89d6cf003aa836f25f27df6fe20a943841" target='_blank'>
              HRR gene mutations and characteristics of the tumor microenvironment as predictors of response to olaparib therapy in metastatic castration-resistant prostate cancer
              </a>
            </td>
          <td>
            A. I. Stukan, S. Vtorushin, A. P. Bogdan, Т. Y. Semiglazova, Kh. R. Tovbulatova, V. Bodnya, V. Porkhanov, A. A. Dovlatbekyan, M. A. Chagiev, D. V. Khoreva
          </td>
          <td>2025-08-06</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gliomas are the most common primary malignant tumors of the central nervous system (CNS), and despite progress in molecular diagnostics and targeted therapies, their prognosis remains poor. In recent years, immunotherapy has emerged as a promising treatment modality in cancer therapy. However, the inevitable immune evasion by tumor cells is a key barrier affecting therapeutic efficacy. Epigenetic regulation, such as DNA methylation, histone modification, and non-coding RNA expression, plays a crucial role in the occurrence, development, and immune evasion of gliomas. These modifications can dynamically regulate gene expression, leading to the silencing of tumor-associated antigens, dysregulation of pro-inflammatory cytokines, and dynamic modulation of immune checkpoints (such as PD-L1). This review systematically elucidates the key mechanisms by which epigenetic regulation promotes immune evasion in gliomas and details three interconnected mechanisms: 1) epigenetic silencing of tumor-associated antigens and antigen-presenting machinery; 2) dysregulation of pro-inflammatory cytokine secretion; and 3) dynamic modulation of PD-L1 expression through chromatin remodeling. We emphasize the potential of combining epigenetic therapies with immunotherapies to enhance anti-tumor immune responses and overcome treatment resistance in gliomas. Future research should focus on developing biomarker-driven epigenetic immunotherapies and exploring the complex interplay between epigenetic modifications, glioma cells, and the tumor immune microenvironment to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd05063be53a8d164f1a96c934eb33dcd3607ec2" target='_blank'>
              Unraveling epigenetic drivers of immune evasion in gliomas: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Dan Wu, Dongen Ju, Yujia Zhao, Wenna Liu, Qingqing Liu, Ying Liang
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive malignancy of immature T cells, driven by dysregulated transcriptional networks and oncogenic signaling pathways. Here, we present the first comprehensive analysis of the expression and regulation of TSPAN32, a tetraspanin implicated in lymphocyte homeostasis, in T-ALL. Methods: Transcriptomic data from the Leukemia MILE study (GSE13159) were analyzed to assess TSPAN32 expression across leukemic subtypes. Gene Set Enrichment Analysis (GSEA) was performed to explore biological pathways associated with TSPAN32-correlated genes. For mechanistic validation, HPB-ALL cells were used as a model, with NOTCH signaling inhibited by γ-secretase inhibitor (GSI) treatment and TAL1–LMO1 overexpression induced through doxycycline-inducible lentiviral vectors. Gene expression changes were quantified by RT-qPCR. Results: TSPAN32 was frequently downregulated in T-ALL compared to healthy bone marrow, although expression was retained in a subset of cases. GSEA revealed that TSPAN32-correlated genes were inversely associated with cell cycle–related programs, consistent with its established role as a negative regulator of T cell proliferation. Mechanistically, TAL1–LMO1 overexpression strongly induced TSPAN32, while GSI-mediated NOTCH inhibition partially reactivated its expression. Interestingly, GSI treatment also increased TAL1 levels despite downregulating LMO1. Conversely, TAL1–LMO1 overexpression suppressed NOTCH1 and NOTCH3, highlighting a reciprocal regulatory interplay between NOTCH and TAL1/LMO1 oncogenic circuits that shapes TSPAN32 expression dynamics in T-ALL. Conclusions: This study identifies TSPAN32 as a novel transcriptional target under the influence of key leukemogenic pathways and suggests its potential role as a modulator of leukemic T cell proliferation, with implications for therapeutic strategies targeting TAL1 and NOTCH signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21eb2ec9bb0fa4dcf030ae56b2109616513baa5a" target='_blank'>
              Transcriptional Control of TSPAN32 in T-ALL Reveals Interplay Between TAL1 and NOTCH1
              </a>
            </td>
          <td>
            Grazia Scuderi, Antonio Arcidiacono, E. Cavalli, M. Basile, Antonella Nardo, Ferdinando Nicoletti, P. Fagone
          </td>
          <td>2025-08-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The BRAF protein regulates cell growth and division through key signaling pathways. Mutations in BRAF, particularly the V600E variant, are frequently observed in colorectal cancer (CRC) and are associated with poor prognosis and therapeutic challenges. Tumors harboring certain BRAF mutations often exhibit primary resistance to BRAF inhibitor monotherapies. Over time, these tumors can also develop acquired resistance, further complicating treatment. In this study, we employed replica exchange molecular dynamics simulations combined with machine learning techniques to investigate the structural alterations induced by BRAF mutations and their contribution to drug resistance. Our analyses revealed that conformational changes in mutant BRAF proteins associated with dabrafenib residues psi494, phi600, phi644, phi663, psi675, and phi677 were sufficient for classifying drug-resistant vs. drug-sensitive variants. Similarly, for vemurafenib, residues psi450, phi484, phi495, phi518, psi622, and phi622 were the key residues that influence drug binding and resistance mechanisms. These residues are located in the N-lobe of CR3, which is responsible for ATP binding and the regulation of BRAF kinase activity. These findings offer deeper insights into the molecular basis of BRAF-driven resistance and provide predictive models for phenotypic outcomes of various BRAF mutations. The study underscores the importance of targeting specific BRAF variants for more effective, personalized therapeutic strategies in drug-resistant CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcf6858b6f2686bd3d3197f896e957a981c35dc" target='_blank'>
              Combining Molecular Dynamics and Machine Learning to Predict Drug Resistance Causing Variants of BRAF in Colorectal Cancer
              </a>
            </td>
          <td>
            Longsheng Xie, Christopher Lockhart, Dmitri K. Klimov, M. S. Jafri
          </td>
          <td>2025-08-30</td>
          <td>Molecules</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="CRISPR/Cas9 gene-editing technology has revolutionized biomedical research by enabling precise, efficient, and cost-effective genome manipulation. This paper focuses on the application of CRISPR/Cas9 in cancer research, with an emphasis on triple-negative breast cancer (TNBC), a particularly aggressive and treatment-resistant subtype of breast cancer that lacks expression of estrogen, progesterone, and HER2 receptors. Breast cancer is the most commonly diagnosed cancer among women worldwide, and TNBC represents one of its most challenging forms due to the absence of effective targeted therapies. While hormone receptor-positive and HER2-positive subtypes benefit from endocrine and targeted treatments, TNBC patients are often limited to chemotherapy, which is associated with high toxicity and limited efficacy. This review synthesizes recent CRISPR-based studies that identify oncogenes and tumor suppressors relevant to TNBC, explores advanced delivery systems such as non-viral nanoparticles, and evaluates the therapeutic promise of this technology. Despite current limitations—including off-target effects, delivery challenges, and ethical concerns—CRISPR/Cas9 offers a promising path toward developing personalized, gene-targeted treatments for TNBC and other hard-to-treat cancers.

Key words: CRISPR/Cas9, gene editing, breast cancer, triple negative breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c17f19abfb2d0fe3b181a39613fc90c46bf2f21" target='_blank'>
              Review of Current Applications of CRISPR Gene Editing Techniques in Breast Cancer Therapeutics Development
              </a>
            </td>
          <td>
            Yichen Zhang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accurate determination of lineage or evolutionary pathways is critical for understanding the dynamic developmental and temporal progression patterns observed in single cell omics data. In this study, we present a computational approach designed to infer progression patterns in cell populations that are actively evolving, differentiating or progressing along dynamic pathways. Our method works at single cell resolution using single cell RNA-Seq data. We determine the optimal cell order based on the cellular transcriptional profiles to uncover the progression of cell populations along differentiation, signaling, or tumor evolution trajectories. To achieve this, we developed a seriation-based method for progression pattern inference, leveraging optimally reordered hierarchies. We provide advanced visualization tools using principal curves to represent the inferred pathways in a three-dimensional latent space derived from scRNA-Seq data. Furthermore, we propose new metrics for evaluating the accuracy of the reconstructed order and identified pathways. The effectiveness of our approach is demonstrated through applying the analysis to real human colon samples transcriptomics data that we generated specifically for this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/004d6d59d63d9011d09efaebc9853e9f6c9a5089" target='_blank'>
              Inference of Tumor Progression Patterns in Colon Cancer using Optimal Cell Order Analysis in Single Cell Resolution.
              </a>
            </td>
          <td>
            Marmar Moussa, Charles H. Street
          </td>
          <td>2025-07-24</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Comprehensive genomic profiling (CGP) is a molecular diagnostic tool with increasing use in cancer research and treatment. There are several commercialized CGP assays with variable targeted genes, while the differences between medium- and large-sized panels in refining genomic profiling strategies, optimizing resource allocation and enhancing clinical applications require evaluation. In the present study, patients with triple-negative breast cancer (TNBC) from the Veterans General Hospital TAipei-Yung-Ling foundation sinO-canceR study were initially assayed using a medium-sized CGP panel (Oncomine comprehensive panel), and the remaining nucleic acid specimens were re-sequenced with a large-sized panel (TruSight Oncology 500). The molecular profiling between the two sequencing panels was compared. A total of 108 breast cancer samples were successfully assayed using both platforms and 272 variants were reported at least once by either type of CGP. The reported variants were among actionable genes (AKT1, BRCA1/2, PALB2, ERBB2, PIK3CA and PTEN), which can be acted upon or have clinical relevance for therapeutic intervention and TP53. The concordance rate between the medium- and large-sized panels was 34.6%, which was enhanced to 58.9% after excluding polymorphisms, out-of-targeted region variants and those with low variant allele frequency (#x003C;10%), with variants of TP53, ERBB2 and PTEN being mostly enhanced. A majority of discordance came from TruSight Oncology 500-detected only variants, especially BRCA1, BRCA2 and PALB2. In conclusion, the results indicated that only one-third of actionable mutations could be detected consistently between the medium- and large-sized CGP panels using the default analytical pipelines, while extensive bioinformatics analyses improved concordance substantially. The large-sized panel detected more variants, thereby enhancing clinical actionability. With more therapeutic targets revealed in the future, CGP may be particularly impactful in refining strategies for TNBC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8f8c51dffb87429763b41f009f6f95cce17938" target='_blank'>
              Comprehensive genomic profiling of Taiwanese triple-negative breast cancer samples with medium- and large-sized sequencing panels: A comparative study implicating treatment allocations
              </a>
            </td>
          <td>
            Chi-Cheng Huang, Yi-Chen Yeh, Y. Tsai, Yen-Shu Lin, Ta-Chung Chao, Chun-Yu Liu, Hsiang-Ling Ho, Ling-Ming Tseng
          </td>
          <td>2025-08-05</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Head and neck squamous cell carcinoma (HNSCC) is a common cancer with poor outcomes despite intensive treatment. While immunotherapy using immune checkpoint inhibitors offers new hope, its effectiveness remains limited. In this study, we experimentally examined whether combining epigenetic drugs with anti-PD-L1 therapy could enhance treatment efficacy. Epigenetic treatment changed the expression of several immune-related genes, including increased levels of STAT1, STAT3, and PD-L1. In a syngeneic mouse model, the combination of epigenetic drugs and anti-PD-L1 antibodies significantly reduced tumor growth. Tumors treated with epigenetic drugs showed significantly higher expression of STAT1, STAT3, and PD-L1 compared to untreated tumors. The increased expression of PD-L1 is presumed to enhance the effectiveness of anti-PD-L1 therapy. These results suggest that epigenetic treatment can reprogram tumors to be more responsive to immunotherapy, highlighting a promising new strategy for improving outcomes in patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efb472fa1affb2f3bd1a4c6972db280727452592" target='_blank'>
              Epigenetic Treatment Alters Immune-Related Gene Signatures to Increase the Sensitivity of Anti PD-L1 Drugs
              </a>
            </td>
          <td>
            Chonji Fukumoto, Pritam Sadhukhan, Masahiro Shibata, Muhammed T. Ugurlu, Rachel Goldberg, David Sidransky, L. Marchionni, Fenna C. M. Sillé, Mohammad O. Hoque
          </td>
          <td>2025-07-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a3a7fc693a064b9e09834f53317ec734d6a1c07" target='_blank'>
              Single-cell transcriptome profiling reveals blast cell heterogeneity and identifies novel therapeutic target IKZF2 in t(8;21) acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Yu Liu, Wenbing Liu, Yihan Mei, Qianqian Huang, Xiaoyu Liu, Chengcai Guo, Manling Chen, Junli Mou, Shangshang Wang, Wanqing Xie, Zheng Tian, Kejing Tang, H. Xing, Ying Wang, Hui Wei, R. Gu, Qing Rao, Min Wang, Shaowei Qiu, Jianxiang Wang
          </td>
          <td>2025-08-08</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="INTRODUCTION
Pancreatic Cancer (PC) is recognized as a highly aggressive malignancy and is anticipated to become the second leading cause of cancer-associated deaths across the United States by 2030. Owing to its late-stage diagnosis and the substantial risk of metastasis, current therapeutic strategies exhibit limited efficacy, resulting in a five-year survival rate below 10%. Consequently, identifying reliable biomarkers and therapeutic approaches remains imperative for enhancing treatment effectiveness.


METHODS
In this study, using the data of Cancer Genome Atlas (TCGA) and Genome-Wide Association Study (GWAS), we identified macrophage subsets and key gene arrb2 closely related to the progression of Pancreatic Adenocarcinoma (PAAD) via the scpagwas method combined with single- cell and bulk transcriptome analysis.


RESULTS
The results showed that in macrophage subpopulations closely related to disease progression, the ARRB2 gene was significantly associated with the prognosis of patients, and low expression suggested poor survival outcomes. The high-expression subgroup of ARRB2 exhibited higher sensitivity to a variety of drugs, and i-bet-762 showed strong molecular binding ability with ARRB2. The experimental detection further confirmed the low expression of ARRB2 in PAAD tissue, which provided the basis for its use as a prognostic marker and potential therapeutic target.


DISCUSSION
The results of this study suggest that ARRB2 is not only a prognostic biomarker of PAAD but may also be involved in the regulation of metabolic and immune pathways, thereby affecting drug responsiveness. There was a significant difference in drug sensitivity between the high and low-expression subgroups of ARRB2, suggesting that there may be a potential mechanism between the signal pathway and the therapeutic effect, which warrants further functional research. Considering the heterogeneity of the macrophage population and its dual role in tumor promotion and inhibition, in-depth analysis of the context-dependent function of ARRB2 in the immune-rich microenvironment is expected to provide new insights for the development of combination therapy, especially the potential of combining it with BET inhibitors (such as i-bet-762).


CONCLUSION
Macrophages, along with ARRB2, serve an essential function in the progression of PAAD and immune regulation. The identification of ARRB2 as a prognostic biomarker and its involvement in critical oncogenic pathways furnish a theoretical foundation for targeted therapeutic interventions. These discoveries contribute to the ongoing exploration of diagnostic, prognostic, and treatment strategies for PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/420334015532528f89d5ba5aeb1438dfd993bcbb" target='_blank'>
              Identifying ARRB2 as a Prognostic Biomarker and Key Player in the Tumor Microenvironment of Pancreatic Cancer through scPagwas Methodology.
              </a>
            </td>
          <td>
            Gaonan Tian, Chengcheng Liu, Chang Che, Shiqi Ren, Kun Zhang, Pengcheng Zhou, Kaidong Wang, Guanyi Lu, Yuchen Xia, YIfan Wang, Kailai Li, Li Yang, Xiangjun Fan, Lei Wang
          </td>
          <td>2025-09-08</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND/AIM
Comprehensive genomic profiling (CGP) with tissue- and blood-based next-generation sequencing (NGS) is integral to the delivery of personalized medicine for targeted cancer therapy. This study aimed to evaluate the variant concordance for somatic variants using two clinical NGS systems for conducting both tissue- and blood-based analyses: Genexus-OCA v3 (OCA) vs. FoundationOne CDx (F1) for tissues and Genexus OPA (OPA) vs. FoundationOne CDx Liquid (F1L) for blood.


PATIENTS AND METHODS
The concordance of genomic alterations between the two NGS analyses was compared in six patients with breast, head, and neck cancers using tissue and circulating tumor DNA biopsies.


RESULTS
A total of 130 genes were common between F1 and OCA, and 41 between F1L and OPA. When comparing FoundationOne to Genexus for common genes, the sensitivity and specificity of OCA and OPA were 55% and 99%, respectively. Nine single-nucleotide variants (SNVs), one copy number alteration (CNA), and one fusion were detected by both Genexus and FoundationOne. However, one SNV (MAP2K1 F53V), two CNAs (AKT3 and MYC), and one fusion (ESR-CCDC170) were detected only in Genexus, whereas two SNVs (TP53 Q331* and KRAS G12V) were detected only in FoundationOne.


CONCLUSION
The two cancer genome panels were equivalent but not perfect in terms of the detection of variants using tissue and blood, indicating that different assays and analytical methods may have influenced the results. When performing CGPs, it is important to consider the characteristics of each NGS-based CGP test and the genetic variants associated with each disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe1f8fa1288486304d422d7baf60accd4be21e6" target='_blank'>
              Comparison of Comprehensive Genomic Profiling Testing "Ion Torrent Genexus Sequencer" With FoundationOne.
              </a>
            </td>
          <td>
            Kenji Fujiyoshi, Rie Sugihara, Naoki Miyamoto, Yoriko Watanabe, Tomoya Sudo, Sanae Numata, Jun Akiba, H. Abe, Yuka Ichinose, Kenji Inoue, Shuich Ozono, T. Ono, Kentaro Orioka, Masaki Kashihara, Ryousuke Kajiwara, Hiroyuki Kawano, Akihiko Kawahara, Ryuta Takase, Uhi Toh, Kazuaki Hashimoto, T. Hisaka, Shingo Hirai, M. Mitsuoka, Daiki Miyazaki, Fumi Yoshitomi, Ken Yamamoto, H. Umeno, Masahisa Nomura, Yoshiki Naito
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258c585d858690cd52a8f16499c3f52ebcd32f02" target='_blank'>
              Genetic regulation of cell type-specific chromatin accessibility shapes immune function and disease risk
              </a>
            </td>
          <td>
            A. Xue, J. Fan, O. A. Dong, H. L. Huang, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, B. Bowen, A. Cuomo, A. Henry, H. Tanudisastro, Z. Qiao, L. Chen, E. Ben-David, K. Farh, D. Neavin, A. S. Lee, A. Senabouth, C. Bartie, R. A. McCloy, V. Chin, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Natural killer cells (NK cells) are important immune cells within the tumor microenvironment (TME) and are considered the first line of defense in tumor immunity. Although many studies have focused on the role of NK cells in tumor therapy, the heterogeneity of NK cells complicates the investigation of the complex mechanisms within the tumor microenvironment. Single-cell sequencing technology, with its high-resolution capability, reveals the gene expression profiles of individual NK cells, highlighting their heterogeneity and providing more accurate information for NK cell therapy. This article begins by addressing the mechanisms underlying the formation of NK cell heterogeneity, emphasizing the significance of differentiation, development, and tissue residency in establishing this heterogeneity. It also summarizes the advances in the study of NK cell heterogeneity under physiological conditions and in tumor environments using single-cell sequencing technology. Finally, it analyzes the dynamic changes of NK cells within the tumor microenvironment under various therapeutic approaches to explore drug effects and resistance mechanisms, as well as to optimize therapeutic options. Investigating the mechanisms of tumor progression and drug intervention at the single-cell level will provide new perspectives for personalized treatment strategies centered around NK cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60507aa9211c6d7d18d4e28dd353dc3d06e0a4a9" target='_blank'>
              Unraveling NK cell heterogeneity through single-cell sequencing: insights from physiological and tumor contexts for clinical applications
              </a>
            </td>
          <td>
            Mingxin Shen, Yutong Liu, Liang Shao, Meng Qu, Shixin Song, Wei Sun, Hao Zhang
          </td>
          <td>2025-07-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af84640a910b5c7e72c72fa38163e38d00a66c4" target='_blank'>
              Morphological map of under- and overexpression of genes in human cells.
              </a>
            </td>
          <td>
            S. Chandrasekaran, Eric Alix, John Arevalo, Adriana Borowa, Patrick J. Byrne, William G. Charles, Zitong S. Chen, B. Cimini, Boxiong Deng, John G Doench, Jessica D. Ewald, Briana Fritchman, Colin J. Fuller, Jedidiah Gaetz, Amy Goodale, Marzieh Haghighi, Yu Han, Zahra Hanifehlou, Holger Hennig, Desiree Hernandez, Christina B. Jacob, Tim James, Tomasz Jetka, Alexandr A. Kalinin, B. Komalo, Maria Kost-Alimova, Tomasz Krawiec, Brittany A. Marion, Glynn Martin, Nicola Jane McCarthy, Lisa Miller, Arne Monsees, Nikita Moshkov, Al'an F. Munoz, Arnaud Ogier, Magdalena Otrocka, Krzysztof Rataj, D. Root, Francesco Rubbo, Simon Scrace, Douglas W Selinger, Rebecca Senft, Peter Sommer, A. Thibaudeau, Sarah Trisorus, Rahul Valiya Veettil, William J. Van Trump, Sui Wang, Michał Warchoł, Erin Weisbart, A. Weiss, Michael Wiest, Agata Zaremba, Andrei Zinovyev, Shantanu Singh, A. Carpenter
          </td>
          <td>2025-08-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="The occurrence and development of malignant tumors involve abnormalities in complex molecular regulatory networks, among which the abnormal activation of the transcriptional regulator hairy and enhancer of split 1 (Hes1) has attracted significant attention in recent years and is closely associated with prognosis in various malignancies. Hes1 exhibits high expression in various solid tumors and hematological malignancies, where it participates in alterations involving diverse immune cells, inflammatory factors, and the immune microenvironment, thereby promoting tumor cell proliferation, invasion, metastasis, and resistance to treatment. Recent studies have widely investigated the potential of targeting Hes1 and inhibiting its expression as a cancer therapeutic strategy, although its precise mechanisms of action are not yet fully elucidated. Hes1 interacts with critical pathways including Notch, JAK/STAT, PI3K/AKT/mTOR, and Wnt/β-catenin. These interactions form complex crosstalk networks that drive malignant transformation and progression. Furthermore, Hes1 plays a central role in the formation of an immunosuppressive tumor microenvironment (TME) and immune escape by regulating the expression of immune checkpoint-associated proteins, extracellular matrix (ECM) remodeling, and other processes, making it a highly promising therapeutic target. Notably, the expression level of Hes1 is significantly correlated with tumor clinical stage, prognosis, and drug resistance. This review comprehensively introduces the mechanisms of Hes1 in the progression of malignant tumors, with a particular focus on discussing its application and underlying mechanisms in tumor immunotherapy. It integrates the latest clinical evidence and preclinical research perspectives. The goal is to highlight the translational potential of Hes1 as a novel biomarker and molecular target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca4a5e14b5411c1cf587b921ebef2a69839952a" target='_blank'>
              Hes1 in malignant tumors: from molecular mechanism to therapeutic potential
              </a>
            </td>
          <td>
            Liping Zhang, Qian Zhang, Cheng Guo, Zixin Ru, Zetian Yang, Yi Geng, Junjie Yang, Daigui Zhang, Zhenhuai Yang, Shuicai Huang
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Bladder cancer (BC) is a disease that predominantly affects older adults, with aging playing a critical role in its onset and progression. Age-associated phenomena, including immunosenescence and chronic inflammation, form a pro-tumor milieu, while genomic instability and epigenetic drift further increase cancer risk. The review highlights the dual role of DNA methylation in BC: global hypomethylation can activate transposable elements and oncogenes, whereas focal hypermethylation silences tumor-suppressor genes like CDKN2A, especially detrimental in older tissues that rely on these genes for senescence control. In parallel, frequent mutations in chromatin modifiers (e.g., KDM6A, KMT2D) and overexpression of histone-modifying enzymes (e.g., EZH2) alter the tumor epigenome to promote immune evasion and tumor aggressiveness. At the non-coding RNA level, dysregulated microRNAs (miRNAs) and long non-coding RNAs (lncRNAs) in BC contribute to aberrant proliferation, metastatic potential, and immune suppression, with aging-associated declines in miRNA processing further exacerbating these effects. Collectively, the accumulation of epigenetic alterations in older patients appears to facilitate both tumor progression and resistance to therapy. Looking forward, epigenetic biomarkers may improve early detection and risk stratification. Furthermore, “epigenetic therapies,” such as DNA methyltransferase inhibitors (DNMTi), EZH2 inhibitors (EZH2i), or histone deacetylases inhibitors (HDACi), hold promise to restore tumor-suppressor function and enhance immunogenicity, offering an attractive avenue for improving outcomes in older patients with BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bc5ece634826938f00d72c674db3193d520a4b" target='_blank'>
              Epigenetic regulation of bladder cancer in the context of aging
              </a>
            </td>
          <td>
            Xuewei Liu, Guofeng Ding, Yifan Liu, Xiaoli Yan, Yan Zhao, Hailin Lv, Xiaojuan Xu
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8f50562c127ac79c0cdedb311535fc46568766" target='_blank'>
              Longitudinal single-cell analysis reveals treatment-resistant stem and mast cells with potential treatments for pediatric AML.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Alejandro De Janon Gutierrez, D. DeRyckere, Katherine E. Ferguson, Franklin Chien, William C. Pilcher, Sarthak Satpathy, Sean Jordan, Douglas K Graham, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The HOXA gene locus coordinates body patterning, hematopoiesis, and differentiation. While studying blood phenotype-associated variation within the HOXA locus, we identified a genetic variant, rs17437411, associated with globally reduced blood counts, protection from blood cancers, and variation in anthropometric phenotypes. We find that this variant disrupts the activity of a previously unstudied antisense long non-coding RNA (lncRNA) located between HOXA7 and HOXA9, which we have named HOTSCRAMBL. The HOTSCRAMBL variant disrupts lncRNA function and reduces human hematopoietic stem cell (HSC) self-renewal. Mechanistically, HOTSCRAMBL enables appropriate expression and splicing of HOXA genes in HSCs, most notably HOXA9, in an SRSF2-dependent manner. Given the critical role of HOXA gene expression in some blood cancers, we also demonstrate that HOTSCRAMBL variation or deletion compromises HOXA-dependent acute myeloid leukemias. Collectively, we show how insights from human genetic variation can uncover critical regulatory processes required for effective developmental gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854c6d558bc4abf3b6a29b21c9a18aa5b2bee350" target='_blank'>
              Genetic variation reveals a homeotic long noncoding RNA that modulates human hematopoietic stem cells
              </a>
            </td>
          <td>
            Peng Lyu, Gaurav Agarwal, Chun-Jie Guo, Tianyi Ye, Chen Weng, Mateusz Antoszewski, Samantha Joubran, Alexis Caulier, Michael Poeschla, V. G. Sankaran
          </td>
          <td>2025-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The advent of CRISPR-Cas9 has revolutionized gene editing, offering vast possibilities for disease treatment. This review dissects its mechanisms, structural aspects, and classification, underlining its superiority to preceding technologies such as ZFNs and TALENs. The evolution of CRISPR’s applications is traced from bacterial immunity to the forefront of genome editing. Delivery methods—both viral and non-viral—are assessed, with detailed exploration of emerging solutions like lipid nanoparticles and DNA nanocarriers. CRISPR/Cas9 demonstrates significant therapeutic value in oncology by inactivating oncogenes, restoring tumor suppressors, and enhancing immunotherapeutic strategies. Furthermore, promising results have emerged in treating inherited disorders such as beta-thalassemia and sickle cell anaemia, as well as in clinical studies focused on T-cell therapies and viral diseases. Although these developments are encouraging, obstacles persist, including off-target genetic modifications and the intricacies of delivery. Innovations, notably base editing and prime editing, show promise to resolve these barriers. This review highlights the immense therapeutic prospects of CRISPR/Cas9 and stresses the importance of sustained research to fully unlock its clinical benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937dc42e57cb7d44775b157991f2406ddf45a511" target='_blank'>
              CRISPR-Cas9 PROTEIN: PHARMACEUTICAL FRONTIERS IN GENETIC RESTORATION
              </a>
            </td>
          <td>
            Senthil Prabhu R, Gunaseelan M, J. R.
          </td>
          <td>2025-08-30</td>
          <td>International Journal of Pharmaceutical Sciences and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Fusobacterium nucleatum and activating mutations in the Kirsten rat sarcoma virus oncogene homolog (KRAS) are increasingly recognized as cooperative drivers of colorectal cancer (CRC). F. nucleatum promotes tumorigenesis via adhesion to epithelial cells, modulation of the immune microenvironment, and delivery of virulence factors, while KRAS mutations—present in 60% of CRC cases—amplify proliferative signaling and inflammatory pathways. Here, we review the molecular interplay by which F. nucleatum enhances KRAS-driven oncogenic cascades and, conversely, how KRAS mutations reshape the tumor niche to favor bacterial colonization. We further discuss the use of KRAS as a prognostic biomarker and explore promising non-antibiotic interventions—such as phage therapy, antimicrobial peptides, and targeted small-molecule inhibitors—aimed at selectively disrupting F. nucleatum colonization and virulence. This integrated perspective on microbial–genetic crosstalk offers novel insights for precision prevention and therapy in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8da0aa66d6c92fb8118636330446f0e6b4fde83" target='_blank'>
              The Impact of Fusobacterium nucleatum and the Genotypic Biomarker KRAS on Colorectal Cancer Pathogenesis
              </a>
            </td>
          <td>
            Ahmed Dewan, Ivan Tattoli, M. Mascellino
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: The PTEN gene is one of the main tumour suppressors, which is essential in regulating cell growth and preventing carcinogenesis. In endometrial cancer, mutations or loss of function of PTEN are frequently associated with disease progression, making this gene a relevant target for studies on molecular mechanisms and potential therapeutic approaches. Uncontrolled activation of the PI3K/AKT pathway, resulting from PTEN inactivation, favours cell proliferation and resistance to apoptosis, contributing to tumour aggressiveness. Methology: This study is based on a systematic review of the scientific literature, analysing articles published between 2010 and 2025 in databases such as PubMed, Scopus and Web of Science. Studies that investigate the relationship between PTEN mutations and the development of endometrial cancer, considering molecular, clinical and therapeutic aspects, were included. The selection of articles followed strict inclusion and exclusion criteria, prioritising research with significant samples and well-established methodologies. Results: The data analysed indicate that PTEN mutations are present in approximately 50% of endometrial cancer cases, being more frequent in high-grade tumours. These mutations compromise the function of the PTEN protein, allowing cells with DNA damage to avoid apoptosis and continue to proliferate. In addition, tumours with PTEN mutations are more resistant to conventional treatments, such as chemotherapy and radiotherapy. Recent studies suggest that inhibition of the PI3K/AKT pathway may be a promising strategy to contain tumour progression in patients with PTEN mutations. Discussion: The presence of PTEN mutations in endometrial cancer reinforces its importance as a prognostic and therapeutic biomarker. Uncontrolled activation of the PI3K/AKT pathway contributes to exacerbated cell proliferation, resistance to apoptosis, and tumour angiogenesis, favouring cancer dissemination. Therapeutic strategies aimed at modulating this pathway are being investigated, including specific PI3K inhibitors and targeted therapies. However, challenges such as heterogeneity of mutations and the interaction of PTEN with other molecular pathways require more personalised approaches for the treatment of the disease. Conclusion: The PTEN gene plays an essential role in regulating cell growth and preventing endometrial cancer. Its mutation or loss of function is directly related to tumour progression, therapeutic resistance, and poor prognosis of the disease. Understanding these mechanisms paves the way for the development of new therapeutic strategies, including targeted therapies and PI3K/AKT pathway inhibitors, which may improve clinical outcomes for patients. Future studies should focus on identifying biomarkers associated with PTEN and improving targeted therapies to optimise the management of endometrial cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/830421e6c05eb941747f7017645e77bb052d1fb7" target='_blank'>
              The Role of PTEN Tumor Suppressor Genes in Endometrial Cancer
              </a>
            </td>
          <td>
            Caio Siqueira De La Paz, Laíza Virnia Cortes Viana, Maísa Marques Bueno
          </td>
          <td>2025-08-06</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gene signatures predictive of chemotherapeutic response have the potential to extend the reach of precision medicine by allowing oncologists to optimize treatment for individuals. Most published predictive signatures are only capable of predicting response for individual drugs, but most chemotherapy regimens utilize combinations of different agents. We propose a unified framework, called the chemogram, that uses predictive signatures to rank the relative predicted sensitivity of different drugs for individual tumors. Using this approach, providers could efficiently screen against many therapeutics to optimize chemotherapy at any time, whether it be for a treatment-naive tumor or a chemo-resistant tumor requiring a new treatment strategy. To demonstrate the utility of the chemogram, we used predictive signatures (extracted from a previously established method) in our framework to rank predicted sensitivity among drugs within cell lines. We then compared the rank order of predicted and observed response against each drug. Across most cancer types, chemogram-generated predictions were more accurate than predictions made by randomly generated gene signatures, signatures extracted from differential expression alone, and was comparable to another established method of drug response prediction. Our framework demonstrates the ability of transcriptomic signatures to not only predict chemotherapeutic response, but also correctly assign rankings of drug sensitivity on an individual basis. Additionally, scaling the chemogram to include more drugs does not compromise accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2faa968db922d9bf628e93088cb5fa3dc44c7a8" target='_blank'>
              Personalizing chemotherapy drug selection using a novel transcriptomic chemogram.
              </a>
            </td>
          <td>
            Kristi Lin-Rahardja, J. Scarborough, Jacob G Scott
          </td>
          <td>2025-09-10</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Recently, cancer has become a leading cause of death worldwide, prompting increased research to understand the pathways involved in cancer development and to identify solutions for its treatment. The PI3K/AKT pathway has garnered significant attention because of its involvement in promoting cell proliferation and inhibiting programmed cell death. The protein phosphatase and tensin homolog on chromosome 10 (PTEN) plays a crucial role in inhibiting this pathway, thereby limiting uncontrolled cell proliferation. PTEN gene expression is strictly regulated at the transcriptional, posttranscriptional, and posttranslational levels, and recent research has focused on PTEN due to its reduced levels in cancer cells. This review aims to provide a deep understanding of the PTEN protein from structural and regulatory perspectives, its mutated forms, and its interactions with the occurrence of various malignant tumors to summarize the recent work performed to combat cancers via molecular strategies to enhance PTEN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006440d6b28c83ed5863d6ee8a11b4692d78170c" target='_blank'>
              PTEN defects in cancer, from gene to protein molecular causes and therapeutic targets
              </a>
            </td>
          <td>
            Ghaith Khasarah, Darina Al Nabilsi, Odai Madani, Farah Khazem, Ranwa Alsayed
          </td>
          <td>2025-08-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The efficacy of cancer immunotherapy relies on the recruitment and activation of cytotoxic T cell responses against solid tumors by type 1 conventional dendritic cells (cDC1s). However, the generation of cDC1s for cancer immunotherapy faces significant limitations, including poor cell yield, functional heterogeneity, and susceptibility to immunosuppression in the tumor microenvironment (TME). We recently developed an immunotherapy modality based on in vivo reprogramming of cancer cells into immunogenic cDC1-like cells, which enabled cancer cells to present tumor antigens as cDC1s and elicited polyclonal cytotoxic T cell responses and durable systemic anti-tumor immunity. Here, we describe a tractable protocol to generate cDC1-like cells within the TME by overexpressing the minimal cDC1-specific gene regulatory network-PU.1, IRF8, and BATF3 (collectively referred to as PIB)-in cancer cells, followed by subcutaneous implantation of a mixture of transduced and parental cells. PIB overexpression drives the gradual acquisition of the hematopoietic marker CD45 and the professional antigen presentation complex MHC class II on tumor cells, serving as cell surface readouts for in vivo cDC1 reprogramming. When compared to the reprogramming process in vitro, reprogramming of the YUMM1.7 mouse melanoma model in vivo demonstrated faster kinetics and higher efficiency. cDC1-like cells induced rapid remodeling of the TME by recruiting host immune cells within the first 3 days and leading to the formation of tertiary lymphoid structure by day 9. Reprogrammed cDC1-like cells persisted in tumors for at least 9 days but were undetected at day 15. The in vivo cDC1 reprogramming protocol described here provides a tractable and robust method to effectively transform "immune-cold" tumors into "immune-hot". Overall, it offers a powerful platform to study the mechanisms underlying cDC1-mediated anti-tumor immunity and uncover synergistic combinations with other cancer immunotherapy modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123daf2d749b83399e744626e1c06c270f48bb14" target='_blank'>
              Tractable In Vivo Reprogramming of Tumor Cells to Type 1 Conventional Dendritic Cell-like Cells.
              </a>
            </td>
          <td>
            Ervin Ascic, Gaia Fontanari, Maria Thrasyvoulou, César Pérez Bucio, Tommaso Ballocci, Carlos-Filipe Pereira
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d735b44b6ee6328d790d08642f35e3daac2ffe" target='_blank'>
              Integrated in vivo combinatorial functional genomics and spatial transcriptomics of tumours to decode genotype-to-phenotype relationships.
              </a>
            </td>
          <td>
            Marco Breinig, Artem Lomakin, Elyas Heidari, M. Ritter, G. Rukhovich, Lio Böse, Luise Butthof, Lena Wendler-Link, Hendrik Wiethoff, Tanja Poth, Felix Sahm, Peter Schirmacher, Oliver Stegle, M. Gerstung, Darjus F. Tschaharganeh
          </td>
          <td>2025-07-28</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>