<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-10 09:26:28 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>19</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity, the capacity of cells to transition between distinct phenotypic and lineage states over time, is a genetically and epigenetically encoded trait essential for normal development and adult tissue homeostasis. In cancer, phenotypic plasticity programs can be deployed aberrantly to enable disease progression and acquired therapeutic resistance. Cancer phenotypic plasticity is a current barrier to achieving cures for advanced cancers using available molecularly targeted therapies. This review summarizes the complex and interconnected molecular pathways implicated in phenotypic plasticity, both in the context of normal tissue homeostasis and cancer. Molecular pathways convergent between these contexts are highlighted while pathways enabling plasticity are distinguished from those that specify the phenotype of already plastic cells. Key unresolved questions in the field are discussed along with emerging technologies that may be used to help answer them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d0b45e5336c08fda7f8b88d683482dae5405ff" target='_blank'>
              The molecular determinants of phenotypic plasticity in homeostasis and neoplasia
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2024-12-13</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying rapid adaptation to stress is a fundamental question in evolutionary biology. A common mechanism is copy number changes, which can affect single genes, larger genomic regions, or even entire chromosomes in the case of eukaryotes (aneuploidy). Bacterial species with genomes comprising more than one chromosome are common, yet the role of adaptive copy number changes affecting entire chromosomes is unknown. Here, by using a bacterium with two chromosomes (chrI and chrII) as model system and combining evolution experiments in the presence of severe nutrient stress with whole-genome sequencing, we show that chromosomal copy number changes can drive rapid adaptation. We detected increased copy number of the entire chrII in almost half of the sequenced clones, isolated from twelve independent populations. In most cases, copy number increases were driven by mutations in the replication protein of chrII. A higher chrII copy number is likely to be beneficial because chrII codes for the operon responsible for metabolizing D-gluconic acid, the only carbon source present in the medium. Our results revealed a novel mechanism for rapid adaptation in bacteria. Given the prevalence of bacteria with secondary chromosomes, and the enrichment of secondary chromosomes in genes associated with interaction with the environment, this may be an important mechanism for adapting to novel environments that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e1750352edfd93ae418d5a02e63e98887bb1a2" target='_blank'>
              Chromosomal plasticity can drive rapid adaptation in bacteria
              </a>
            </td>
          <td>
            Luis Vega-Cabrera, O. Skovgaard, Macarena Toll-Riera
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Copy number variants (CNVs) are an important source of genetic variation underlying rapid adaptation and genome evolution. Whereas point mutation rates vary with genomic location and local DNA features, the role of genome architecture in the formation and evolutionary dynamics of CNVs is poorly understood. Previously, we found the GAP1 gene in Saccharomyces cerevisiae undergoes frequent amplification and selection in glutamine-limitation. The gene is flanked by two long terminal repeats (LTRs) and proximate to an origin of DNA replication (autonomously replicating sequence, ARS), which likely promote rapid GAP1 CNV formation. To test the role of these genomic elements on CNV-mediated adaptive evolution, we evolved engineered strains lacking either the adjacent LTRs, ARS, or all elements in glutamine-limited chemostats. Using a CNV reporter system and neural network simulation-based inference (nnSBI) we quantified the formation rate and fitness effect of CNVs for each strain. Removal of local DNA elements significantly impacts the fitness effect of GAP1 CNVs and the rate of adaptation. In 177 CNV lineages, across all four strains, between 26% and 80% of all GAP1 CNVs are mediated by Origin Dependent Inverted Repeat Amplification (ODIRA) which results from template switching between the leading and lagging strand during DNA synthesis. In the absence of the local ARS, distal ones mediate CNV formation via ODIRA. In the absence of local LTRs, homologous recombination can mediate gene amplification following de novo retrotransposon events. Our study reveals that template switching during DNA replication is a prevalent source of adaptive CNVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488ecf1ac91854034846373133d03867e6680f28" target='_blank'>
              Template switching during DNA replication is a prevalent source of adaptive gene amplification
              </a>
            </td>
          <td>
            Julie N. Chuong, Nadav Ben Nun, Ina Suresh, J. Matthews, Titir De, Grace Avecilla, Farah Abdul-Rahman, Nathan Brandt, Yoav Ram, David Gresham
          </td>
          <td>2025-02-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Copy number variants (CNVs) are prevalent in both diploid and haploid genomes, with the latter containing a single copy of each gene. Studying CNVs in genomes from single or few cells is significantly advancing our knowledge in human disorders and disease susceptibility. Low-input including low-cell and single-cell sequencing data for haploid and diploid organisms generally displays shallow and highly non-uniform read counts resulting from the whole genome amplification steps that introduce amplification biases. In addition, haploid organisms typically possess relatively short genomes and require a higher degree of DNA amplification compared to diploid organisms. However, most CNV detection methods are specifically developed for diploid genomes without specific consideration of effects on haploid genomes. Challenges also reside in reference samples or normal controls which are used to provide baseline signals for defining copy number losses or gains. In traditional methods, references are usually pre-specified from cells that are assumed to be normal or disease-free. However, the use of pre-defined reference cells can bias results if common CNVs are present. Here, we present the development of a comprehensive statistical framework for data normalization and CNV detection in haploid single- or low-cell DNA sequencing data called HapCNV. The prominent advancement is the construction of a novel genomic location specific pseudo-reference that selects unbiased references using a preliminary cell clustering method. This approach effectively preserves common CNVs. Using simulations, we demonstrated that HapCNV outperformed existing methods by generating more accurate CNV detection, especially for short CNVs. Superior performance of HapCNV was also validated in detecting known CNVs in a real P. falciparum parasite dataset. In conclusion, HapCNV provides a novel and useful approach for CNV detection in haploid low-input sequencing datasets, with easy applicability to diploids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40d70f46141697934edb1592beae132a61d8c33" target='_blank'>
              HapCNV: A Comprehensive Framework for CNV Detection in Low-input DNA Sequencing Data
              </a>
            </td>
          <td>
            Xuanxuan Yu, Fei Qin, Shiwei Liu, Noah Brown, Qing Lu, Guoshuai Cai, J. Guler, Feifei Xiao
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The molecular non-genetic changes of resistance to sotorasib are currently uncertain. The aim of this study was to generate a sotorasib-resistant cell line via selective pressure and systematically examine the molecular and phenotypic alterations caused by resistance.Mutant NCI-H358 (KRASG12C) were exposed to incremental doses (2–512 nM) of sotorasib. Then, resistant clones were separated by single-cell sorting. Proliferation was analyzed in real-time by xCELLigence; protein profiles were quantified by protein arrays; and mRNA expression profile was measured using the PanCancer Pathways panel by NanoString. In silico analyses were conducted from a database comprising patient-derived xenograft (PDX) models and cell lines resistant to sotorasib. AKT and p38. The synergistic effect of combining AKT, p38, and EGFR inhibitors was assessed using the SynergyFinder platform. Additionally, AKT and p38 genes were silenced using esiRNA.Sotorasib-resistant H358-R cell line displayed markers of the mesenchymal-epithelial transition and loss of cell adhesion. Were identified 30 overexpressed genes in the resistance model, implicating in signaling pathways that leads to AKT activation and heightened protein expression levels of phosphorylated AKT and p38. To identify potential therapeutic strategies for overcoming sotorasib resistance, we investigated the combination of AKT and p38 inhibitors. Notably, combined inhibition of AKT (MK2206) and p38 (adezmapimod) restored sensitivity to sotorasib in resistant cell lines, as did silencing AKT expression.These findings underscore the importance of adaptive mechanisms in sotorasib resistance in NSCLC cells contributing by EMT activation and demonstrates synergic combination with AKT and p38 inhibitors to restore sotorasib sensitivity in KRASG12C cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2b84647241a84761deba81b95c6e8effb443d" target='_blank'>
              Sotorasib resistance triggers epithelial-mesenchymal transition and activates AKT and P38-mediated signaling
              </a>
            </td>
          <td>
            R. A. Megid, G. G. Ribeiro, I. Gomes, A. C. Laus, L. Ferro Leal, Luciane Sussuchi da Silva, A. A. Ariwoola, J. M. Dias, R. Reis, Renato Jose da Silva-Oliveira
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Resistance to endocrine therapy (ET) is common in estrogen receptor (ER) positive breast cancer. Multiple studies have demonstrated that upregulation of MAPK signaling pathways contributes to ET resistance. Herein we show that vandetanib treatment enhances sensitivity to ET in ET-sensitive and –resistant ER+ breast cancer models. Using a CRISPR knockout model, we demonstrate that vandetanib effects are partially mediated by RET receptor tyrosine kinase. Vandetanib treatment alters the gene expression program of ER+ breast cancer cells resulting in a less proliferative and more estrogen responsive Luminal-A like character. Tyrosine kinase network reprogramming was assessed using multiplexed kinase inhibitor beads-mass spectrometry (MIB/MS) assay to identify adaptive resistance mechanisms to vandetanib treatment, which identified upregulation of HER2 activity. Co-treatment to inhibit HER2 with lapatinib enhanced sensitivity to vandetanib, demonstrating biologic activity of HER2 upregulation. Finally, we use our operating room-to-laboratory assay that measures drug response in individual primary tumor cells in short term cultures to demonstrate conserved gene expression changes, including increased HER2 activity signatures, in vandetanib treated cells, and identify features associated with vandetanib response. These results support future investigation of RET targeting strategies considering reprogrammed networks, such as activated HER2, in patients with ET resistant ER+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/163cc944feb5c6e8c8bb5e07eb1e26b3c782d784" target='_blank'>
              Kinase Plasticity in Response to Vandetanib Enhances Sensitivity to Tamoxifen and Identifies Co-Treatment Strategies in Estrogen Receptor Positive Breast Cancer
              </a>
            </td>
          <td>
            Rasha T. Kakati, Austin A. Whitman, Santiago Haase, Attila T. Szenasi, Christine Hnc Thai, Elizabeth Brunk, Denis O. Okumu, Michael P. East, C. Perou, Gary L. Johnson, Philip M. Spanheimer
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>161</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes in many organisms. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs that are present in only a few genomes across a population are often overlooked but important; if beneficial under specific conditions, a de novo CNV that arises in a single genome can expand during selection to create a larger population of cells with novel characteristics. While the reach of single cell methods to study de novo CNVs is increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich P. falciparum genome readily accumulates CNVs that facilitate rapid adaptation to new drugs and host environments. We employed a low-input genomics approach optimized for this unique genome as well as specialized computational tools to evaluate the de novo CNV rate both before and after the application of stress. We observed a significant increase in genome-wide de novo CNVs following treatment with a replication inhibitor. These stress-induced de novo CNVs encompassed genes that contribute to various cellular pathways and tended to be altered in clinical parasite genomes. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c09296cb530f9bd4da8646286b7568688223753" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The success of cancer therapy has been significantly hampered by various mechanisms of therapeutic resistance. Chief among these mechanisms is the presence of clonal heterogeneity within an individual tumor mass. The introduction of the concept of cancer stem cells (CSCs)—a rare and immature subpopulation with tumorigenic potential that contributes to intratumoral heterogeneity—has deepened our understanding of drug resistance. Given the characteristics of CSCs, such as increased drug‐efflux activity, enhanced DNA‐repair capacity, high metabolic plasticity, adaptability to oxidative stress, and/or upregulated detoxifying aldehyde dehydrogenase (ALDH) enzymes, CSCs have been recognized as a theoretical reservoir for resistant diseases. Implicit in this recognition is the possibility that CSC‐targeted therapeutic strategies might offer a breakthrough in overcoming drug resistance in cancer patients. Herein, we summarize the generation of CSCs and our current understanding of the mechanisms underlying CSC‐mediated therapeutic resistance. This extended knowledge has progressively been translated into novel anticancer therapeutic strategies and significantly enriched the available options for combination treatments, all of which are anticipated to improve clinical outcomes for patients experiencing CSC‐related relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467b9483651e22480e8aee88bab2b5c598d31695" target='_blank'>
              Unraveling the underlying mechanisms of cancer stem cells in therapeutic resistance for optimizing treatment strategies
              </a>
            </td>
          <td>
            Yunhan Tan, Siyuan Qin, Zhe Zhang, Yongen Liu, Li Zhou, Bo Li, Edouard C. Nice, Yuanyuan Zhang, Jing Jing
          </td>
          <td>2025-01-10</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Aim: Mutations in the mitochondrial (mt) genome contribute to metabolic dysfunction and their accumulation relates to disease progression and resistance development in cancer cells. This study explores the mutational status of the mt genome of cisplatin-resistant vs. -sensitive testicular germ cell tumor (TGCT) cells and explores its association with their respiration parameters, expression of respiratory genes, and preferences for metabolic pathways to reveal new markers of therapy resistance in TGCTs. Methods: Using Illumina sequencing with Twist Enrichment Panel, the mutations of mt genomes of sensitive 2102EP, H12.1, NTERA-2, T-cam and resistant 2102EP Cis, H12.1 ODM, 1411HP, 1777NRpmet, NTERA-2 Cis and T-cam Cis cell lines were identified. The mt respiration of the cells was assessed using high-resolution respirometry method (O2k-respirometer Oroboros) and the differential expression profiles of mt respiratory genes were determined using RT-qPCR. Associated preferences for metabolic pathways were compared using Glycolysis/OXPHOS assay. Results: In resistant TGCT cells, new mutations in mt genes MT-ND1-6, MT-RNR, MT-CO1-3, MT-ATP6, and MT-CYB were recognized. The respiratory rates of the 1777NRpmet cell line were the highest, while those of the 1411HP line the lowest; rates of the control and all other TGCT cell lines fell between these two lines. The statistically significant differences in gene expression of the respiratory genes were recorded only in NTERA-2 Cis and T-cam Cis cell lines. Sensitive cell lines NTERA-2 and 2102EP preferred oxidative phosphorylation (OXPHOS), while glycolysis was typical for resistant NTERA-2 Cis, 2102EP Cis and 1411HP cell lines. Metastatic 1777NRpmet cells seem to utilize both. An isogenic pair of cell lines H12.1 and H12.1ODM showed the opposite dependence, sensitive H12.1 preferring glycolysis, while resistant H12.1ODM OXPHOS. Conclusion: In summary, our study identified new mutations in mt genes of resistant TGCT cell lines that are associated with different mt respiration parameters, gene expression patterns and preferences for metabolic pathways, providing potential novel molecular biomarkers that distinguish the resistant TGCT phenotype or specify its histological classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec03801d589beba72991767df646f63f72e7a8" target='_blank'>
              Mitochondrial genome variability and metabolic alterations reveal new biomarkers of resistance in testicular germ cell tumors
              </a>
            </td>
          <td>
            Pavlina Kabelikova, Danica Ivovic, Z. Sumbalová, M. Karhánek, Lucia Tatayova, Martina Skopkova, Michal Cagalinec, V. Bruderová, J. Roska, Dana Jurkovicova
          </td>
          <td>2024-12-18</td>
          <td>Cancer Drug Resistance</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Antimicrobial combination therapy is widely used to combat Mycobacterium tuberculosis (Mtb), yet resistance rates continue to rise. Mutator strains, with defects in DNA repair genes, are a known source of resistance in other bacterial infections, but their global role in Mtb remains unclear. Here, we provide a comprehensive overview of the contribution of single nucleotide polymorphisms (SNPs) in DNA Repair, Replication, and Recombination (3R) genes to resistance in Mtb. Through large-scale bioinformatics analysis of 53,589 whole-genomes, we identified 18 novel SNPs linked to genotypic drug resistance in 3R genes, covering 12.5% of clinical isolates/strains with available genome sequences. Notably, a number of the detected SNPs were positively selected during Mtb evolution. Experimental mutation frequency tests further indicated functional defects of these sequence variants in key DNA repair pathways. Our findings indicate that the role of mutability in 3R genes plays a significant role in Mtb resistance at the global level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8834cea19a0a1579b6f2abeb80d3493166b930f" target='_blank'>
              Genome wide analyses reveal the role of mutator phenotypes in the emergence of drug resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            R. Zein-Eddine, A. Le Meur, S. Skouloubris, L. Jelsbak, G. Refrégier, H. Myllykallio
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genetic variation leads to phenotypic variability in pluripotent stem cells that presents challenges for regenerative medicine. Although recent studies have investigated the impact of genetic variation on pluripotency maintenance and differentiation capacity, less is known about how genetic variants affecting the pluripotent state influence gene regulation in later stages of development. Here, we characterized expression of more than 12,000 genes in 127 donor-matched Diversity Outbred (DO) mouse embryonic stem cell (mESC) and neural progenitor cell (mNPC) lines. Quantitative trait locus (QTL) mapping identified 2,947 expression QTL (eQTL) unique to DO mNPCs and 1,113 eQTL observed in both mNPCs and mESCs with highly concordant allele effects. We mapped three eQTL hotspots on Chromosomes (Chrs) 1, 10, and 11 that were unique to mNPCs. Target genes of the Chr 1 hotspot were overrepresented for those involved in mRNA processing, DNA repair, chromatin organization, protein degradation, and cell cycle. Mediation analysis of the Chr 1 hotspot identified Rnf152 as the best candidate mediator expressed in mNPCs, while cross-cell type mediation using mESC gene expression along with partial correlation analysis strongly implicated genetic variant(s) affecting Pign expression in the mESC state as regulating the mNPC Chr 1 eQTL hotspot. Together these findings highlight that many local eQTL confer similar effects on gene expression in multiple cell states; distant eQTL in DO mNPCs are numerous and largely unique to that cell state, with many co-localizing to mNPC-specific hotspots; and mediation analysis across cell types suggests that expression of Pign early in development (mESCs) shapes the transcriptome of the more specialized mNPC state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ad28abad19bc73c3c14d84322dd6bba9b2687" target='_blank'>
              Cross cell-type systems genetics reveals the influence of eQTL at multiple points in the developmental trajectory of mouse neural progenitor cells
              </a>
            </td>
          <td>
            Selcan Aydin, Daniel A. Skelly, Hannah Dewey, J. M. Mahoney, Ted Choi, L. Reinholdt, Christopher L. Baker, Steven C. Munger
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Bladder cancer is one of the most common malignancies of the urinary system and has always presented great challenges in treatment due to its intricate biological features and high recurrence rates. Although great developments were achieved in immunotherapy and targeted therapies within the last decade, therapeutic outcomes for a great number of patients remain unsatisfactory, particularly as to long-term efficacy. Review discusses the molecular mechanisms developed during the process of bladder cancer progression: genetic and epigenetic alterations, dynamics of the tumor microenvironment (TME), and dysregulation and abnormal activation of various signaling pathways—all contributing to therapeutic resistance. It is genetic mutation, especially in both low- and high-grade tumors, that, alongside epigenetic modifications, plays a considerable role in tumor aggressiveness and drug resistance. TME, comprising cancer-associated fibroblasts (CAFs), immunosuppressive cells, and different components of the extracellular matrix (ECM), orchestrates a setting that fosters tumor growth and immune evasion and confers resistance on any therapeutic regime that might be used. The review also provides an overview of PI3K/AKT and MAPK signaling pathways in the progression of bladder cancer and the development of targeted therapies against them. Further, it discusses the challenges and mechanisms of resistance to immunotherapy, including those involving immune checkpoint inhibitors. Other promising approaches include the development of new therapeutic strategies that target not only the signaling pathways but also immune checkpoints in combination therapies. This review aims to contribute to the elaboration of more effective and personalized treatment strategies by fully understanding the underlying mechanisms involved in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee79f09b15911cd50c65c34f5cba7cd83105949" target='_blank'>
              Redefining bladder cancer treatment: innovations in overcoming drug resistance and immune evasion
              </a>
            </td>
          <td>
            Peng Huang, Jie Wang, Zongze Yu, Jiaan Lu, Zhou Sun, Zhigui Chen
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a neoplasm of antibody-producing plasma cells and is the second most prevalent hematological malignancy worldwide. Development of drug resistance and disease relapse significantly impede the success of MM treatment, highlighting the critical need to discover novel therapeutic targets. In a custom CRISPR/Cas9 screen targeting 197 DNA damage response-related genes, Protein Arginine N-Methyltransferase 1 (PRMT1) emerged as a top hit, revealing it as a potential therapeutic vulnerability and survival dependency in MM cells. PRMT1, a major Type I PRMT enzyme, catalyzes the asymmetric transfer of methyl groups to arginine residues, influencing gene transcription and protein function through post-translational modification. Dysregulation or overexpression of PRMT1 has been observed in various malignancies including MM and is linked to chemoresistance. Treatment with the Type I PRMT inhibitor GSK3368715 resulted in a dose-dependent reduction in cell survival across a panel of MM cell lines. This was accompanied by reduced levels of asymmetric dimethylation of arginine (ADMA) and increased arginine monomethylation (MMA) in MM cells. Cell cycle analysis revealed an accumulation of cells in the G0/G1 phase and a reduction in the S phase upon GSK3368715 treatment. Additionally, PRMT1 inhibition led to a significant downregulation of genes involved in cell proliferation, DNA replication, and DNA damage response (DDR), likely inducing genomic instability and impairing tumor growth. This was supported by Reverse Phase Protein Array (RPPA) analyses, which revealed a significant reduction in levels of proteins associated with cell cycle regulation and DDR pathways. Overall, our findings indicate that MM cells critically depend on PRMT1 for survival, highlighting the therapeutic potential of PRMT1 inhibition in treating MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6449b92554ba5747c0bbd79109710a289b5244d5" target='_blank'>
              Therapeutic Potential of PRMT1 as a Critical Survival Dependency Target in Multiple Myeloma
              </a>
            </td>
          <td>
            Tabish Hussain, Sharad Awasthi, Farid Shahid, S. Yi, Nidhi Sahni, Marcelo Aldaz, C. M. Aldaz
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f873b54c9840f07bddb284c4e229ec7800362c" target='_blank'>
              Genetic advancements in breast cancer treatment: a review
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Sadaf Sedaghatshoar, Homaira Arian, Milad Mokarami, Fatemeh Habibi, Fatemeh Bamarinejad
          </td>
          <td>2025-02-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Metabolic reprogramming, particularly lipid metabolism, is a hallmark of cancer progression and a critical vulnerability in prostate cancer (PCa). Two pivotal studies, one by Lu et al. and the other by Dairo et al., have illuminated the roles of HOXB13, a transcriptional regulator, and fatty acid synthase (FASN), a key enzyme in lipid biosynthesis, in the metabolic dysregulation of PCa. The Lu et al. study highlights HOXB13’s role in androgen receptor (AR)-independent PCa, where it regulates lipid metabolism via epigenetic mechanisms involving histone deacetylase HDAC3. A G84E mutation in HOXB13 disrupts this regulation, leading to increased lipid synthesis and a pro-metastatic phenotype. The Dairo et al. study demonstrates that FASN hypomethylation, coupled with increased expression, drives lipid biosynthesis critical for tumor growth. Both studies establish a link between HOXB13 mutations and FASN dysregulation, underscoring their interplay in PCa biology. Therapeutically, pharmacological inhibition of FASN mitigates the aggressive features of HOXB13-deficient or mutant PCa, highlighting lipid metabolism as a promising target. Despite their strengths, including robust methodologies, limitations include reliance on preclinical models and the need for broader patient diversity. These studies collectively emphasize the potential of metabolic and epigenetic interventions for precision medicine in PCa, paving the way for novel therapies targeting lipid metabolism in patients with specific genetic and epigenetic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc58c2230658f37cc79a630362aeb84c369fdaa5" target='_blank'>
              Metabolic Reprogramming in Prostate Cancer: The Roles of HOXB13 and FASN in Tumor Progression and Therapy
              </a>
            </td>
          <td>
            Faisal Saeed, Lara Harik
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Comparative genomics has emerged as a pivotal methodology for elucidating genetic variations in microbial studies. However, conventional analytical approaches for diploid and polyploid microorganisms have demonstrated limited efficacy in discriminating new mutations from background heterozygosity. This study presents an innovative microbial comparative genomics framework adapted from tumor-normal sequencing methodology. Our approach establishes the original strain as the “normal sample” and the derived strain as the “tumor sample”, enabling precise identification of new mutations (“somatic variants”) while filtering pre-existing heterozygous sites (“germline variations”). The analytical pipeline also includes assessment of loss-of-heterozygosity (LOH) events and genome-wide detection of copy number variations (CNVs) with resolution to identify both regional CNV and whole-chromosome aneuploidy through integrated CNV and variant allele frequency analysis. We validated this framework using diploid Saccharomyces cerevisiae strains before successfully extending its application to Kluyveromyces marxianus, Candida spp., and Hortaea werneckii, encompassing haploid, diploid, polyploid, and aneuploid states. The methodology revealed previously undetected variations across experimental evolution studies, demonstrating superior resolution compared to conventional approaches. This adaptable platform establishes a new paradigm for microbial genome studies, particularly for organisms with diploid or polyploid states where traditional comparative genomics methods prove inadequate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to randomly generated nucleotide barcodes. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple-negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ba3313bee111d463805c5f2fb966ea7f8b649e" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Y. Li, C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Organisms constantly face environmental stressors that threaten their cellular and genomic integrity. In their response, pathogen-associated molecular patterns (PAMPs) and/or damage-associated molecular patterns (DAMPs) are detected by pattern recognition receptors (PRRs) and trigger the innate immune response. In this study we tested the hypothesis that DAMPs contribute to radiation-induced cellular plasticity in Glioblastoma (GBM). GBM is known to be organized hierarchically with a small number of glioma-initiating cells (GICs) driving treatment resistance and recurrences. Using patient-derived GBM specimens, we employed sphere forming capacity assays and in vitro extreme limiting dilution assays to examine how innate immune receptor signaling impacts the maintenance and self-renewal of GICs. By leveraging an imaging system for putative GICs we determined de novo induction of GICs from non-stem glioma cells. We find that GIC maintenance after irradiation is mediated by cGAS-independent STING signaling, possibly involving signaling through TLR4 and TLR9. Induction of radiation-induced plasticity involves TLR3 signaling, with potential roles for other receptors and processes modulated by MyD88. These findings suggest that targeting innate immune signaling could prevent radiation-induced cellular plasticity for potential therapeutic benefit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5ddc1e74601131515f0f5cca20f0a178a3981ff" target='_blank'>
              Targeting Innate Immune Signaling in Glioma-Initiating Cells Impairs Self-Renewal and Radiation-Induced Cellular Plasticity
              </a>
            </td>
          <td>
            Anjelica M. Cardenas, Linda Azizi, A. Ioannidis, Kruttika Bhat, Ling He, Mahya Mohammadi, Frank Pajonk
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3833ef2c0218061364fcad4c0598f346a7f4c6cb" target='_blank'>
              Synthetic lethality of mRNA quality control complexes in cancer.
              </a>
            </td>
          <td>
            Vivian Prindle, Adam E Richardson, Kimberly R Sher, Sarah Kongpachith, Kaitlin Kentala, Sakina M. Petiwala, Dong Cheng, Deborah Widomski, Phuong Le, Maricel Torrent, Anlu Chen, Stephen Walker, Marianne B. Palczewski, Diya Mitra, V. Manaves, Xu Shi, Charles Lu, Stephanie Sandoval, Zoltan Dezso, F. Buchanan, Daniel Verduzco, Brian Bierie, Jonathan A Meulbroek, W. Pappano, Joshua P Plotnik
          </td>
          <td>2025-02-05</td>
          <td>Nature</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Genomics has revolutionised the study of invasive species, allowing evolutionary biologists to dissect mechanisms of invasion in unprecedented detail. Botanical research has played an important role in these advances, driving much of what we currently know about key determinants of invasion success (e.g. hybridisation, whole-genome duplication). Despite this, a comprehensive review of plant invasion genomics has been lacking. Here, we aim to address this gap, highlighting recent discoveries that have helped progress the field. For example, by leveraging genomics in natural and experimental populations, botanical research has confirmed the importance of large-effect standing variation during adaptation in invasive species. Further, genomic investigations of plants are increasingly revealing that large structural variants, as well as genetic changes induced by whole-genome duplication such as genomic redundancy or the breakdown of dosage-sensitive reproductive barriers, can play an important role during adaptive evolution of invaders. However, numerous questions remain, including when chromosomal inversions might help or hinder invasions, whether adaptive gene reuse is common during invasions, and whether epigenetically induced mutations can underpin the adaptive evolution of plasticity in invasive populations. We conclude by highlighting these and other outstanding questions that genomic studies of invasive plants are poised to help answer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37cdc62894d0b05ba9dc0a9393c69deaf8d1c768" target='_blank'>
              The genomic secrets of invasive plants.
              </a>
            </td>
          <td>
            K. Hodgins, Paul Battlay, Dan G. Bock
          </td>
          <td>2025-01-02</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Genomic data is essential for clinical decision-making in precision oncology. Bioinformatic algorithms are widely used to analyze next-generation sequencing (NGS) data, but they face two major challenges. First, these pipelines are highly complex, involving multiple steps and the integration of various tools. Second, they generate features that are human-interpretable but often result in information loss by focusing only on predefined genetic properties. This limitation restricts the full potential of NGS data in biomarker extraction and slows the discovery of new biomarkers in precision oncology. Methods We propose an end-to-end deep learning (DL) approach for analyzing NGS data. Specifically, we developed a multiple instance learning DL framework that integrates somatic mutation sequences to predict two compound biomarkers: microsatellite instability (MSI) and homologous recombination deficiency (HRD). To achieve this, we utilized data from 3,184 cancer patients obtained from two public databases: The Cancer Genome Atlas (TCGA) and the Clinical Proteome Tumor Analysis Consortium (CPTAC). Results Our proposed deep learning method demonstrated high accuracy in identifying clinically relevant biomarkers. For predicting MSI status, the model achieved an accuracy of 0.98, a sensitivity of 0.95, and a specificity of 1.00 on an external validation cohort. For predicting HRD status, the model achieved an accuracy of 0.80, a sensitivity of 0.75, and a specificity of 0.86. Furthermore, the deep learning approach significantly outperformed traditional machine learning methods in both tasks (MSI accuracy, p-value = 5.11×10−18; HRD accuracy, p-value = 1.07×10−10). Using explainability techniques, we demonstrated that the model’s predictions are based on biologically meaningful features, aligning with key DNA damage repair mutation signatures. Conclusion We demonstrate that deep learning can identify patterns in unfiltered somatic mutations without the need for manual feature extraction. This approach enhances the detection of actionable targets and paves the way for developing NGS-based biomarkers using minimally processed data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad073d80b11f20c88ed32369a6393b6a5510e50a" target='_blank'>
              Deep Learning for Biomarker Discovery in Cancer Genomes
              </a>
            </td>
          <td>
            Michaela Unger, C. Loeffler, L. Žigutytė, Srividhya Sainath, Tim Lenz, Julien Vibert, Andreas Mock, S. Fröhling, Trevor A. Graham, Zunamys I. Carrero, J. N. Kather
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, Guokang Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Objectives Unlike other diseases, cancer is not just a genome disease but should broadly be viewed as a disease of the cellular machinery. Therefore, integrative multifaceted approaches are crucial to understanding the complex nature of cancer biology. Bcl-2 (B-cell lymphoma 2), encoded by the human Bcl-2 gene, is a critical anti-apoptotic protein that regulates cell death pathways, primarily by inhibiting apoptosis. It plays a pivotal role in maintaining cellular homeostasis by preventing premature or excessive cell death. Genetic variations and dysregulation of Bcl-2 are particularly significant in cancer, as they disrupt the normal apoptotic machinery, enabling cancer cells to evade programmed cell death. Single nucleotide polymorphisms (SNPs) are considered viable diagnostic and therapeutic biomarkers for various cancers. Therefore, this study explores the association between SNPs in Bcl-2 and the structural, functional, protein-protein interactions (PPIs), drug binding and dynamic characteristics. Methods Comprehensive cross-validated bioinformatics tools and molecular dynamics (MD) simulations. Multiple sequence, genetic, structural and disease phenotype analyses were applied in this study. Results Analysis revealed that out of 130 mutations, approximately 8.5% of these mutations were classified as pathogenic. Furthermore, two particular variants, namely, Bcl-2G101V and Bcl-2F104L, were found to be the most deleterious across all analyses. Following 500 ns, MD simulations showed that these mutations caused a significant distortion in the protein conformational, protein-protein interactions (PPIs), and drug binding landscape compared to Bcl-2WT. Conclusion Despite being a predictive study, the findings presented in this report would offer a perspective insight for further experimental investigation, rational drug design, and cancer gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f67dee38c9954b65ace573045ef349b37771a6" target='_blank'>
              Integrative genomic analyses combined with molecular dynamics simulations reveal the impact of deleterious mutations of Bcl-2 gene on the apoptotic machinery and implications in carcinogenesis
              </a>
            </td>
          <td>
            G. Elamin, Zhichao Zhang, Depika Dwarka, K. Kasumbwe, John Mellem, Nompumelelo P. Mkhwanazi, P. Madlala, Mahmoud E. S. Soliman
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="FLT3 mutations are among the most common genetic alterations in acute myeloid leukemia (AML) and are associated with poor prognosis. Significant advancements have been made in developing FLT3 inhibitors (FLT3Is), such as quizartinib, which have improved treatment outcomes in both newly diagnosed and relapsed/refractory AML. Resistance to FLT3Is remains a major clinical challenge, driven by diverse mechanisms including FLT3 point mutations, cellular escape pathways, and the influence of the bone marrow microenvironment. Sustained STAT5 phosphorylation, AXL upregulation, and CXCR4 signaling have been identified as key factors in FLT3I resistance. Additionally, metabolic adaptations have been shown to support the survival of FLT3I-resistant cells. Ongoing clinical trials are investigating various combination regimens, including quizartinib with chemotherapy, Bcl-2 inhibitors, hypomethylating agents, and immune-modulatory drugs, with promising preliminary results. The European LeukemiaNet 2022 guidelines recommend incorporating FLT3Is into treatment regimens; however, questions remain regarding the best timing for the administration of each FLT3I. Additional studies are required to determine the optimal FLT3I-based combinations, reduce resistance emergence, and improve outcomes. This review highlights the current state of FLT3I therapy, ongoing challenges with resistance, and future directions in optimizing treatment for FLT3-mutated AML, focusing on quizartinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3878f024ad9767de696002db14bad327a4d61cfe" target='_blank'>
              Advances and Challenges in Quizartinib-Based FLT3 Inhibition for Acute Myeloid Leukemia: Mechanisms of Resistance and Prospective Combination Therapies.
              </a>
            </td>
          <td>
            A. Bruzzese, E. Martino, Caterina Labanca, F. Mendicino, E. Lucia, Virginia Olivito, Teresa Rossi, A. Neri, F. Morabito, E. Vigna, Massimo Gentile
          </td>
          <td>2025-01-06</td>
          <td>European journal of haematology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Gene therapy has long been a cornerstone in the treatment of rare diseases and genetic disorders, offering targeted solutions to conditions once considered untreatable. As the field advances, its transformative potential is now expanding into oncology, where personalized therapies address the genetic and immune-related complexities of cancer. This review highlights innovative therapeutic strategies, including gene replacement, gene silencing, oncolytic virotherapy, CAR-T cell therapy, and CRISPR-Cas9 gene editing, with a focus on their application in both hematologic malignancies and solid tumors. CRISPR-Cas9, a revolutionary tool in precision medicine, enables precise editing of cancer-driving mutations, enhancing immune responses and disrupting tumor growth mechanisms. Additionally, emerging approaches target ferroptosis—a regulated, iron-dependent form of cell death—offering new possibilities for selectively inducing tumor cell death in resistant cancers. Despite significant breakthroughs, challenges such as tumor heterogeneity, immune evasion, and the immunosuppressive tumor microenvironment (TME) remain. To overcome these barriers, novel approaches like dual-targeting, armored CAR-T cells, and combination therapies with immune checkpoint inhibitors and ferroptosis inducers are being explored. Additionally, the rise of allogeneic “off-the-shelf” CAR-T therapies offers scalable and more accessible treatment options. The regulatory landscape is evolving to accommodate these advancements, with frameworks like RMAT (Regenerative Medicine Advanced Therapy) in the U.S. and ATMP (Advanced Therapy Medicinal Products) in Europe fast-tracking the approval of gene therapies. However, ethical considerations surrounding CRISPR-based gene editing—such as off-target effects, germline editing, and ensuring equitable access—remain at the forefront, requiring ongoing ethical oversight. Advances in non-viral delivery systems, such as lipid nanoparticles (LNPs) and exosomes, are improving the safety and efficacy of gene therapies. By integrating these innovations with combination therapies and addressing regulatory and ethical concerns, gene therapy is poised to revolutionize cancer treatment, providing durable, effective, and personalized solutions for both hematologic and solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a9f2e1247ab5090aa522e48935aeedefea03c5" target='_blank'>
              Enhancing precision in cancer treatment: the role of gene therapy and immune modulation in oncology
              </a>
            </td>
          <td>
            Emile Youssef, Brandon Fletcher, Dannelle Palmer
          </td>
          <td>2025-01-13</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Genetic variation underpins evolutionary change, but accumulation of slightly deleterious mutations also increases mutation load. There are multiple factors affecting the extent of load such as population size and breeding system, yet other potential determinants remain unexplored. A common macromutation, whole-genome duplication (WGD) occurs broadly across Eukaryotes, yet we lack a clear understanding of how WGD impacts neutral and selective processes within a population. Using forward simulations and empirical analysis of 632 short- and 16 long-read sequenced individuals of Arabidopsis arenosa (23 diploid and 42 natural autotetraploid populations), we test for the effects of WGD on genome-wide diversity and mutation load. Our simulations show how genetic variation gradually rises in autotetraploids due to increase of mutational target size. Moreover, mutation load increases due to relaxed purifying selection when deleterious mutations are masked by additional chromosome copies. Empirical data confirm these patterns, showing significant increase in nucleotide diversity, ratios of non-synonymous to synonymous SNPs, and number of indels and large structural variants in A. arenosa autotetraploids. However, a rather modest increase in load proxies together with a broad distribution and niche of autotetraploids suggests load accumulation has not (yet) limited their successful expansion. Overall, we demonstrate a complex interplay between neutral processes and purifying selection in shaping genetic variation following WGD and highlight ploidy as an important determinant of genetic diversity and mutation load in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86bbacd2db01892da50f5a93e5f2a705f35e9543" target='_blank'>
              Whole-genome duplication increases genetic diversity and load in outcrossing Arabidopsis
              </a>
            </td>
          <td>
            Jakub Vlček, Tuomas Hämälä, Cristina Vives Cobo, Emma Curran, G. Šrámková, T. Slotte, R. Schmickl, Levi Yant, Filip Kolář
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Targeted therapies like Venetoclax have increased the options available to acute myeloid leukemia (AML) patients, but survival remains poor due to drug resistance and disease relapse. We found that the translation initiation factor EIF4A1, which unwinds complex mRNA structures in the 5’ UTR of oncogenic transcripts, is highly expressed in AML stem- and progenitor-like cells. Inhibiting eIF4A with the small molecule Zotatifin reduces translation of transcripts related to the cell cycle and survival. This results in downregulation of AKT, STAT-5, and MCL-1 and underlies synergy of Zotatifin with Venetoclax. The drug combination promotes apoptosis across AML genotypes, while the effect on healthy blood cells is limited. Using in vivo relapsed and refractory AML patient-derived xenograft models, the combination significantly suppressed tumor burden and prolonged survival of xenografted mice. These results support eIF4A-mediated protein translation as a therapeutic target in AML. SIGNIFICANCE Despite advances in targeted therapies, the 5-year survival rate for acute myeloid leukemia remains around 32%. The efficacy of existing treatments may be improved with the addition of Zotatifin, an inhibitor targeting the translation initiation factor eIF4A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8672d4c71b8c253c15ee73780df3c698e781744" target='_blank'>
              Research Brief: Translation Initiation Represents an Acute Myeloid Leukemia Cell Vulnerability That Can Be Co-Targeted With BCL-2 Inhibition
              </a>
            </td>
          <td>
            Y. Lee, Jonathan D. Good, Noha Awais, Benjamin K. Eschle, Maia S. Lineberry, Jessica L. Teo, Sarah Bibeau, Morgan Wollman, George Karadimov, Nurefsan Sariipek, Jean Acosta, Jenny Noel, David Cucchi, E. DePasquale, Prafulla Gokhale, Fadi J. Najm, Richard M. Stone, Jacqueline S. Garcia, Andrew A. Lane, Peter van Galen
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetics plays a critical role relative to other branches of genetics, as it makes it possible to regulate gene expression without changing the nucleotide sequence of DNA molecules. This process allows cells to respond to external signals and adapt to  changes in  the  environment while keeping their genetic information intact. The  main mechanisms of  epigenetic regulation include DNA methylation, posttranslational modifications of  histones, chromatin remodeling and  regulation through non-coding RNAs. These processes play a key role in fundamental biological processes such as cellular differentiation, organismal development, and adaptation to environmental conditions.Disturbances in epigenetic mechanisms can lead to various pathologies, including cancer, neurological and  autoimmune diseases. Understanding the  mechanisms of  epigenetic regulation opens new avenues for  the  development of  targeted therapies that can correct abnormal epigenetic profiles without changing the DNA structure itself.In  recent years, the  development and  application of  innovative technologies, such  as  the  CRISPR/Cas9 genome editing system, have significantly expanded the ability to study epigenetic mechanisms and their relationship with diseases. These technologies allow not only a deeper understanding of epigenetic changes, but also the development of new therapeutic approaches, especially in the field of oncology. Research in  epigenetics is  also focusing on  the  interaction between epigenetic changes and the immune system, which opens new perspectives for the development of immunotherapies. The search for new markers of epigenetic disorders and therapeutic agents may lead to the development of individualized treatments that take into account the unique epigenetic profile of each patient.The role of epigenetic modifications in the development of diseases and the creation of new therapeutic strategies cannot be overestimated. Recent research in this area is revealing the potential of epigenetic approaches to treat a wide range of diseases, ushering in a new era in medicine where understanding and correcting epigenetic changes will be the key to effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d047bbd846068847a5bb8e163e24692ce5b5c48" target='_blank'>
              Epigenetic modulation in medicine: Regulation of gene expression in the context of pathogenesis and therapy
              </a>
            </td>
          <td>
            M. A. Omarov, A. R. Mulyukov, R. V. Khalitov, S. I. Safarov, G. U. Ayupova, O. N. Demianenko, V. V. Butenko, A. A. Safronova, R. R. Zainetdinova, O. Y. Ianturina, A. A. Miroshnikov
          </td>
          <td>2024-12-28</td>
          <td>Acta Biomedica Scientifica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras—particularly in diseases such as oral squamous cell carcinoma and glioblastoma—now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, T. Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Despite advancements in the treatment of advanced prostate cancer, including metastatic castration-resistant prostate cancer (mCRPC), the disease remains lethal, often developing resistance to conventional therapies such as androgen deprivation therapy (ADT) and chemotherapy. Therapy resistance poses a significant challenge, particularly in aggressive forms of prostate cancer. Data on TP53 mutations, especially gain-of-function (GOF) mutations, and their specific impact on prostate cancer metabolism remain limited, underscoring the need for a deeper understanding to uncover new therapeutic avenues. This study aimed to explore the metabolic consequences of TP53 GOF mutations (R175H and R273H) in prostate cancer progression using metabolomics analysis. Approximately 300 metabolites were significantly altered across PC3-GFP (KO), PC3-TP53WT (WT), and PC3-TP53 mutant (R175H & R273H) models. Key findings revealed dysregulation of essential metabolites, such as α-ketoglutarate and spermidine, which support cancer cell survival and progression. These insights not only enhance our understanding of the interplay between TP53 mutations and metabolic alterations but also identify potential biomarkers for prognosis and therapeutic targeting in therapy-resistant prostate cancer. In the TP53-R175H mutant, numerous essential metabolites were dysregulated (downregulated: glutamine, GAP/DHAP, and α-ketoglutarate; Upregulated: deoxyuridine, spermidine, and pseudouridine) when compared with the control (KO) thereby causing metabolic shifts that favor cancer cell survival. Conversely, metabolites such as GlaNAC/GlcNAC, adenosine, acetoacetic acid, acetylcholine, octopamine, itaconic acid, lactic acid, and NADP+ are all downregulated in the R273H mutant. While spermine, spermidine, N-acetyl-DL-serine, pseudouridine, deoxyguanosine, sucrose, and AMP/dGMP are upregulated by the R273H mutant but downregulated in GFP (KO). These findings showed that TP53 GOF mutations (R175H & R273H) cause metabolic alterations in numerous metabolites essential for tumor survival and progression. Furthermore, it may provide therapeutic benefits by identifying appropriate biomarkers that could be used as diagnostic and/or prognostic tools for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6235a81e6d316a547376d1bb83acc9177dd3e97b" target='_blank'>
              Impact of TP53 Gain-of-Function Mutations on Metabolic Reprogramming in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Farida Jazuli Usman, D. G. Gupta, S. Abdulkadir, Ibrahim Muhammad Usman, Adamu Jibrin Alhassan, Young A Yoo, Haris Jaafar Bello, Binta G. Kurfi, K. Babagana
          </td>
          <td>2025-01-10</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="With the advancement of single-cell sequencing, we are now capable of analyzing the heterogeneity within tumors with increased precision, revealing the complex interactions between the tumor microenvironment and immune cell subgroups. Studies mentioned in the articles demonstrate significant cell heterogeneity in various cancers at genomic, transcriptomic, and proteomic levels. In particular, single-cell technologies aid in identifying cell-type-specific genetic lineage effects, for example, in the context of influenza infection and T cell surveillance. Current study focuses on the application of single-cell sequencing in cancer immunotherapy and the exploration of novel antigen identification and optimization of therapeutic strategies. The introduction of new strategies, such as the refinement of electrochemical immuno-sensors, synthesis of novel small molecule drugs, and optimized detection of genetic mutations, contributes collectively to the accurate identification of tumor markers and innovative development of therapeutic methods. The integration of treatment schemes and patient survival analysis further guides the implementation of personalized medicine. In summary, the application of single-cell sequencing and its integration with the optimization of therapeutic strategies promises revolutionary advancements in the field of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607f39c834cbc474d463e6168f60ce125e47f1f7" target='_blank'>
              Research On the Identification of Novel Antigens and The Optimization of Therapeutic Strategies in Cancer Immunotherapy Through Single-Cell Sequencing
              </a>
            </td>
          <td>
            Peixuan Li
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Patients with epidermal growth factor receptor mutant nonsmall cell lung cancer (NSCLC) often fail to treat gefitinib because of secondary drug resistance. The development of tumor drug resistance is closely related to variations in cancer cell metabolism. Single-cell metabolomics analysis can provide unique information about tumor drug resistance. Herein, we constructed a platform to study the secondary resistance of tumor cells based on single-cell metabolomics (sSRTC-scM). A gefitinib-resistant NSCLC cell line (PC9GR) was constructed by increasing the dose step by step. The metabolic profiles of parental PC9 cells and PC9GR cells with different drug resistance levels were detected by intact living-cell electrolaunching ionization mass spectrometry at the single-cell level. The data were analyzed by statistical methods such as t-SNE, variance, volcano plot, heat map, and metabolic pathway analysis. Using this platform, we found that the metabolic fingerprints of PC9GR cells can evaluate drug resistance degrees. The metabolic fingerprints continue to be altered with the increase of drug resistance. We revealed 19 metabolic markers of secondary resistance by variance analysis and clarified that the glycerophospholipid metabolic pathway of PC9GR cells changed significantly. In addition, we found that with the increase in drug resistance levels, the heterogeneity of single-cell metabolism became greater and the number of cells with weak drug resistance gradually decreased. This phenomenon can be utilized to illustrate the drug resistance degrees of PC9GR cells. This study provides diagnostic markers for evaluating the drug resistance of tumors and gives new insight into overcoming the secondary resistance of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84899e62659612e5b6164a21695dadcc8005bf4e" target='_blank'>
              Single-Cell Mass Spectrometry Studies of Secondary Drug Resistance of Tumor Cells.
              </a>
            </td>
          <td>
            Guizhen Zhu, Wenmei Zhang, Yaoyao Zhao, Guangyun Wang, Hanyu Yuan, Guangsheng Guo, Xiayan Wang
          </td>
          <td>2024-12-20</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d406237f2977fc66d322658b801ab000ef5018c5" target='_blank'>
              KEAP1 mutations as key crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors
              </a>
            </td>
          <td>
            Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, N. Sun, Jie He, Yan Wang
          </td>
          <td>2025-01-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT, TP53, and CTNNB1, alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070c22030bfb86f8cf3bc0c2063075e7b4258116" target='_blank'>
              Identification of the clinical and genetic characteristics of gliomas with gene fusions by integrated genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            Guo-zhong Yi, Hua-yang Zhang, Tian-shi Que, Shan-qiang Qu, Zhi-yong Li, Song-tao Qi, Wen-yan Feng, Guang-long Huang
          </td>
          <td>2025-01-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="N6-methyladenosine is one of the most common and reversible post-transcriptional modifications in eukaryotes, and it is involved in alternative splicing and RNA transcription, degradation, and translation. It is well known that cancer cells acquire energy through metabolic reprogramming to exhibit various biological behaviors. Moreover, numerous studies have demonstrated that m6A induces cancer metabolic reprogramming by regulating the expression of core metabolic genes or by activating metabolic signaling pathways. Meanwhile, m6A modifications and related regulators are key targets in the regulation of immune effects. We further summarize how m6A modifications contribute to tumor metabolism, and how these events affect the tumor immune microenvironment, with a specific focus on different cell types. Finally, we focus on the specific applications of this field to tumor immunotherapy. We review the potential role of m6A in metabolic reprogramming of tumor immune microenvironment and its regulatory mechanism, with the aim of providing new targets for tumor metabolic regulation and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863477d0bd2f28e887ca2ce1de6b299ba12060bf" target='_blank'>
              N6-methyladenosine RNA methylation, a new hallmark of metabolic reprogramming in the immune microenvironment
              </a>
            </td>
          <td>
            Xiao-Yun Li, Lin Peng, Xue Yang, Jing Luo, Jianmei Wang, Kelin Mou, Huan Zhou, Yuhao Luo, Li Xiang
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant cause of cancer-related mortality worldwide. Despite advancements in surgery, chemotherapy, and radiotherapy, the effectiveness of these conventional treatments is limited, particularly in advanced cases. Therefore, transition to novel treatment is urgently needed. Immunotherapy, especially immune checkpoint inhibitors (ICIs), has shown promise in improving outcomes for CRC patients. Notably, patients with deficient mismatch repair (dMMR) or microsatellite instability-high (MSI-H) tumors often benefit from ICIs, while the majority of CRC cases, which exhibit proficient mismatch repair (pMMR) or microsatellite-stable (MSS) status, generally show resistance to this approach. It is assumed that the MSI phenotype cause some changes in the tumor microenvironment (TME), thus triggering antitumor immunity and leading to response to immunotherapy. Understanding these differences in the TME relative to MSI status is essential for developing more effective therapeutic strategies. This review provides an overview of the TME components in CRC and explores current approaches aimed at enhancing ICI efficacy in MSS CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dbecea08702d890a6154fcbf611f4e3db06227c" target='_blank'>
              The characteristics of the tumor immune microenvironment in colorectal cancer with different MSI status and current therapeutic strategies
              </a>
            </td>
          <td>
            Qingzhe Wang, Min Yu, Shuang Zhang
          </td>
          <td>2025-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors exhibit extensive genomic, epigenomic, and transcriptional diversity, with significant intratumoral heterogeneity, complicating standard treatment approaches involving radiation (RT) and the DNA-alkylating agent temozolomide (TMZ). In this study, we employed an integrative multi-omics approach, including targeted proteomics, transcriptomics, genomics, and DNA methylation profiling, to investigate the response of a representative panel of GBM patient-derived cancer stem cells (CSCs) to astrocytic differentiation and RT and TMZ treatments. Differentiated CSC progenies retained the expression of key stemness genes and survival pathways, while activating the BMP-Smad signaling pathway and upregulating extracellular matrix components. This was associated with increased resistance to TMZ, though not to RT, across all models. We identified TP53 status as a critical determinant of transcriptional response to both RT and TMZ, which was also modulated by the differentiation state and treatment modality in wildtype (wt) p53 GBM cells. Both mutant and wt p53 models exhibited significant activation of the DNA-damage associated interferon (IFN) response in CSCs and differentiated cells, implicating this pathway in the GBM response to therapy. We observed that activation of NF-κB was positively correlated with the levels of O-6-methylguanine-DNA methyltransferase (MGMT) protein, a direct DNA repair enzyme leading to TMZ resistance, regardless of MGMT promoter methylation status, further supporting the clinical potential for inhibition of NF-kB signaling in GBM treatment. Our integrative analysis of the impact of GBM cell developmental states, in the context of genomic and molecular diversity of patient-derived models, provides valuable insights for pre-clinical studies aimed at optimizing treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a242e45dbcf0e04c69a3eb415e835a6082edb4c" target='_blank'>
              Impact of developmental state, p53 status, and interferon signaling on glioblastoma cell response to radiation and temozolomide treatment
              </a>
            </td>
          <td>
            A. Berezovsky, O. Nuga, Indrani Datta, Kimberly Bergman, Thais S Sabedot, Katherine Gurdziel, S. Irtenkauf, L. Hasselbach, Yuling Meng, Claudius Mueller, Emanuel F. Petricoin, Stephen Brown, N. Purandare, Sidhesh Aras, Tom Mikkelsen, Laila M Poisson, H. Noushmehr, Douglas Ruden, A. deCarvalho
          </td>
          <td>2025-02-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We lack tools to edit DNA sequences at scales necessary to study 99% of the human genome that is noncoding. To address this gap, we applied CRISPR prime editing to insert recombination handles into repetitive sequences, up to 1697 per cell line, which enables generating large-scale deletions, inversions, translocations, and circular DNA. Recombinase induction produced more than 100 stochastic megabase-sized rearrangements in each cell. We tracked these rearrangements over time to measure selection pressures, finding a preference for shorter variants that avoided essential genes. We characterized 29 clones with multiple rearrangements, finding an impact of deletions on expression of genes in the variant but not on nearby genes. This genome-scrambling strategy enables large deletions, sequence relocations, and the insertion of regulatory elements to explore genome dispensability and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a90f87137d367e42fa5bf7ac26c12b1be111991" target='_blank'>
              Randomizing the human genome by engineering recombination between repeat elements.
              </a>
            </td>
          <td>
            Jonas Koeppel, Raphael Ferreira, T. Vanderstichele, Lisa M. Riedmayr, Elin Madli Peets, Gareth Girling, Juliane Weller, Pierre Murat, Fabio Giuseppe Liberante, Tom Ellis, George M. Church, Leopold Parts
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="The biological process of aging is influenced by a complex interplay of genetic, environmental, and epigenetic factors. Recent advancements in the fields of epigenetics and senolytics offer promising avenues for understanding and addressing age-related diseases. Epigenetics refers to heritable changes in gene expression without altering the DNA sequence, with mechanisms like DNA methylation, histone modification, and non-coding RNA regulation playing critical roles in aging. Senolytics, a class of drugs targeting and eliminating senescent cells, address the accumulation of dysfunctional cells that contribute to tissue degradation and chronic inflammation through the senescence-associated secretory phenotype. This scoping review examines the intersection of epigenetic mechanisms and senolytic therapies in aging, focusing on their combined potential for therapeutic interventions. Senescent cells display distinct epigenetic signatures, such as DNA hypermethylation and histone modifications, which can be targeted to enhance senolytic efficacy. Epigenetic reprogramming strategies, such as induced pluripotent stem cells, may further complement senolytics by rejuvenating aged cells. Integrating epigenetic modulation with senolytic therapy offers a dual approach to improving healthspan and mitigating age-related pathologies. This narrative review underscores the need for continued research into the molecular mechanisms underlying these interactions and suggests future directions for therapeutic development, including clinical trials, biomarker discovery, and combination therapies that synergistically target aging processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99720a4dac2f698c148a925c719d60e5caa0c9f8" target='_blank'>
              The Intersection of Epigenetics and Senolytics in Mechanisms of Aging and Therapeutic Approaches
              </a>
            </td>
          <td>
            Daiana Burdusel, Thorsten R. Doeppner, R. Surugiu, D. Hermann, Denissa-Greta Olaru, Aurel Popa-Wagner
          </td>
          <td>2024-12-26</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract High-risk multiple myeloma is genomically unstable, comprising heterogeneous populations of tumor cells that evolve over time. Light chain escape (LCE) is a clinical phenomenon observed when light chains rise separately from M-spike values, which implies divergent tumor evolution. We sought to understand LCE by performing high-depth transcriptomic and phenotypic studies. We performed single-cell RNA-sequencing (scRNA-seq) and ex vivo drug sensitivity profiling on serial bone marrow biopsies from a patient with LCE at diagnosis, first relapse, and relapsed/refractory timepoints. scRNA-seq revealed distinct transcriptomic subpopulations with phenotypes that could be tracked separately by clinical serum light chain and M-spike values. Genes differentially expressed between subpopulations were assessed for generalizable effects on prognosis from the Multiple Myeloma Research Foundation CoMMpass and GSE24080 datasets. Notably, the LCE subpopulation exhibited gene expression profile featuring prominent LAMP5 overexpression, which was associated with risk for osteolytic bone lesions. Ex vivo drug sensitivity testing displayed differential sensitivity of the subpopulations. Copy number variant inference showed that the transcriptomic subpopulation underlying LCE was related to a genetic subclone that evolved over time. Our findings illustrate that malignant subpopulations underly LCE in multiple myeloma. These studies imply that LCE and LAMP5 gene overexpression portends for increased risk of osteolytic bone disease and adverse prognosis, findings that were confirmed in the subset of patients from the CoMMpass database with LCE. Significance: scRNA-seq was used to study a patient with high-risk multiple myeloma featuring LCE. LCE was rooted in a transcriptomic subpopulation that corresponded to a genetic subclone and established novel links between LCE and LAMP5 overexpression to osteolysis and prognosis, validated in RNA-seq databases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e33173187a2b1d3d0f9422813ed4bc1d3b6c7d8" target='_blank'>
              Single-Cell RNA Sequencing before and after Light Chain Escape Reveals Intrapatient Multiple Myeloma Subpopulations with Divergent Osteolytic Gene Expression
              </a>
            </td>
          <td>
            Denis Ohlstrom, Z. Walker, Abhishek Pandey, Lorraine N Davis, Krysta Engel, Zenggang Pan, Peter A Forsberg, T. Mark, Austin E. Gillen, Daniel W. Sherbenou
          </td>
          <td>2024-12-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Cancer research has historically focused on the somatic mutation theory, viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information-processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking cancer evolution: from genetic mutations to complex information systems in tumor reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Ionizing radiation is mutagenic and carcinogenic, and it is reported to induce primary and secondary tumors with intestinal tumors being one of the most commonly observed. However, the pathological and molecular mechanism(s) underlying the radiation-associated tumorigenesis remain unclear. A link between radiation and somatic tumorigenesis partly through genetic, epigenetic alteration and/or regulation of mismatch repair (MMR) genes has been hypothesized for the first time within this review. Clinical observations and experimental findings provide significant support for this association including MMR mutations as well as altered MMR RNA and protein expressions that occurred post-exposure, although existing evidence in published literature is sparse in this niche area. Some speculative mechanisms are suggested with this review to inform future research. Further studies are needed to understand the roles of the MMR system in response to radiation and to test this possible connection which could potentially provide useful and urgently needed information for clinical guidance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2500a813ee65c9f47a96bf1021c2ef49642fe4f1" target='_blank'>
              Ionizing Radiation May Induce Tumors Partly Through the Alteration or Regulation of Mismatch Repair Genes
              </a>
            </td>
          <td>
            Mingzhu Sun, Kevin Monahan, Jayne Moquet, Stephen Barnard
          </td>
          <td>2025-02-07</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            Seungyeul Yoo, , S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Cancer remains a significant global health challenge due to its genetic complexity and the diverse mutations, epigenetic modifications, and dysregulated signaling pathways involved. This complexity complicates effective treatment strategies. The CRISPR system, particularly CRISPR - Cas13, presents a promising therapeutic tool, offering precise RNA targeting with high accuracy and sensitivity. This paper examines the potential of Cas 13 in cancer therapy, focusing on four key areas: the mechanism of Cas13, Cas13  applications in cancer treatment, challenges and limitations. The results of the study showed that CRISPR-Cas13 can not only be used as a potential cancer diagnostic tool, but also regulate abnormal mRNA mutations and reduce cancer risk. By knocking down oncogenic lncRNA, CRISPR-Cas13 can inhibit the proliferation and migration of cancer cells. CRISPR-Cas13 has the potential to reduce cancer incidence and reduce cancer mortality through early diagnosis and intervention. In the future, technical optimization to address the off-target effects of Cas13 will be the key to its application in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64e7fa238661b59668d9ff9e403d41b35f3f94d6" target='_blank'>
              The Potential and Future Perspectives of CRISPR-Cas 13 Technology in Cancer Therapy Application
              </a>
            </td>
          <td>
            Yuhe Chen
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of reversible drug-tolerant cell states, thereby accelerating the evolution of drug resistance. Determining how to optimally balance the competing goals of killing the tumor bulk and delaying resistance evolution in this scenario is a nontrivial question of high clinical importance. In this work, we use a combined mathematical and computational approach to study optimal dosing of anti-cancer drug treatment under drug-induced cell plasticity. Our results show that the optimal treatment steers the tumor into a fixed equilibrium composition while balancing the trade-off between cell kill and tolerance induction in a precisely quantifiable way. Under linear induction of tolerance, a low-dose constant strategy is optimal in equilibrium, while under uniform induction of tolerance, alternating between a large dose and no dose is best. The directionality of drug induction, whether the drug elevates transitions from sensitivity to tolerance or inhibits transitions back, significantly affects optimal dosing. To demonstrate the applicability of our approach, we use it to identify an optimal low-dose strategy for colorectal cancer using publicly available in vitro data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586103632d689f330d569f057f914a090bf47bb0" target='_blank'>
              Optimal dosing of anti-cancer treatment under drug-induced plasticity
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Benedikt Vilji Magn'usson, J. Foo
          </td>
          <td>2024-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The CRISPR/Cas system has emerged as a transformative genome-editing technology, enabling precise modifications of genetic material. This paper explores the potential applications of CRISPR/Cas technology in the treatment of gliomas, particularly glioblastoma (GBM), the most aggressive and lethal brain tumor. With over 3,000 genes linked to disease-causing mutations, CRISPR/Cas offers a novel approach to address the underlying genetic alterations in cancer. Recent studies demonstrate the efficacy of CRISPR/Cas12a in knocking out miR-21, a microRNA inversely correlated with glioblastoma survival, resulting in reduced tumor growth in mouse models. Furthermore, we introduce a novel "genome shredding" approach using CRISPR/Cas9 to target repetitive DNA sequences in GBM, leading to extensive cell death, independent of the tumor's genetic profile. Despite these promising advancements, challenges remain in optimizing CRISPR's efficacy and delivery mechanisms to improve patient outcomes. This review highlights the significant potential of CRISPR/Cas systems in revolutionizing glioma therapy while addressing the need for continued research to mitigate off-target effects and enhance therapeutic delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500664bef54808a00d4dd7947aa8294abce819eb" target='_blank'>
              CRISPR/Cas gene editing and its application in the treatment of Gliomas in the brain
              </a>
            </td>
          <td>
            Raymond Sun
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            Ze-Hao Liu, Peng Ma, Ying He, Yuejiao Zhang, Zuo Mou, Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-free gene expression (CFE) systems empower synthetic biologists to build biological molecules and processes outside of living intact cells. The foundational principle is that precise, complex biomolecular transformations can be conducted in purified enzyme or crude cell lysate systems. This concept circumvents mechanisms that have evolved to facilitate species survival, bypasses limitations on molecular transport across the cell wall, and provides a significant departure from traditional, cell-based processes that rely on microscopic cellular “reactors.” In addition, cell-free systems are inherently distributable through freeze-drying, which allows simple distribution before rehydration at the point-of-use. Furthermore, as cell-free systems are nonliving, they provide built-in safeguards for biocontainment without the constraints attendant on genetically modified organisms. These features have led to a significant increase in the development and use of CFE systems over the past two decades. Here, we discuss recent advances in CFE systems and highlight how they are transforming efforts to build cells, control genetic networks, and manufacture biobased products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66dfd8e3013123535721c036d0642b5350410cbb" target='_blank'>
              Cell-Free Gene Expression: Methods and Applications
              </a>
            </td>
          <td>
            Andrew Hunt, Blake J. Rasor, Kosuke Seki, Holly M Ekas, Katherine F. Warfel, Ashty S. Karim, M. Jewett
          </td>
          <td>2024-12-19</td>
          <td>Chemical Reviews</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Metabolic rewiring is required for cancer cells to survive in harsh microenvironments and is considered to be a hallmark of cancer. Specific metabolic adaptations are required for a tumor to become invasive and metastatic. Cell division and metabolism are inherently interconnected, and several cell cycle modulators directly regulate metabolism. Here, we report that TBK1, which is a noncanonical IKK kinase with known roles in cell cycle regulation and TLR signaling, affects cellular metabolism in cancer cells. While TBK1 is reported to be overexpressed in several cancers and its enhanced protein level correlates with poor prognosis, the underlying molecular mechanism involved in the tumor-promoting role of TBK1 is not fully understood. In this study, we show a novel role of TBK1 in regulating cancer cell metabolism using combined metabolomics, transcriptomics, and pharmacological approaches. We find that TBK1 mediates the regulation of nucleotide and energy metabolism through aldo-keto reductase B10 (AKRB10) and thymidine phosphorylase (TYMP) genes, suggesting that this TBK1-mediated metabolic rewiring contributes to its oncogenic function. In addition, we find that TBK1 inhibitors can act synergistically with AKRB10 and TYMP inhibitors to reduce cell viability. These findings raise the possibility that combining these inhibitors might be beneficial in combating cancers that show elevated levels of TBK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a02a736e9562fec8efc444bbd368f2506195967" target='_blank'>
              TBK1 Reprograms Metabolism in Breast Cancer: An Integrated Omics Approach.
              </a>
            </td>
          <td>
            M. Maan, Neha Jaiswal, Min Liu, Harold I Saavedra, S. Chellappan, Mainak Dutta
          </td>
          <td>2024-12-13</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative splicing (AS) is a mechanism that generates translational diversity within a genome. Equally important is the dynamic adaptability of the splicing machinery, which can give preference to one isoform over others encoded by a single gene. These isoform preferences change in response to the cell’s state and function. Particularly significant is the impact of physiological alternative splicing in T lymphocytes, where specific isoforms can enhance or reduce the cells’ reactivity to stimuli. This process makes splicing isoforms defining features of cell states, exemplified by CD45 splice isoforms, which characterize the transition from naïve to memory states. Two developments have accelerated the use of AS dynamics for therapeutic interventions: advancements in long-read RNA sequencing and progress in nucleic acid chemical modifications. Improved oligonucleotide stability has enabled their use in directing splicing to specific sites or modifying sequences to enhance or silence particular splicing events. This review highlights immune regulatory splicing patterns with potential significance for enhancing anticancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaaff8ba550e599aee86ffb2e27658c31cc7d97" target='_blank'>
              Alternative splicing of modulatory immune receptors in T lymphocytes: a newly identified and targetable mechanism for anticancer immunotherapy
              </a>
            </td>
          <td>
            Shay Tzaban, Ori Stern, Elad Zisman, G. Eisenberg, Shiri Klein, S. Frankenburg, Michal Lotem
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Oncolytic herpes simplex viruses (oHSVs) are a type of biotherapeutic utilized in cancer therapy due to their ability to selectively infect and destroy tumor cells without harming healthy cells. We sought to investigate the functional genomic response and altered metabolic pathways of human cancer cells to oHSV-1 infection and to elucidate the influence of these responses on the relationship between the virus and the cancer cells. Methods Two datasets containing gene expression profiles of tumor cells infected with oHSV-1 (G207) and non-infected cells from the Gene Expression Omnibus (GEO) database were processed and normalized using the R software. Common differentially expressed genes between datasets were selected to identify hub genes and were further analyzed. Subsequently, the expression of hub genes was verified by real-time polymerase chain reaction (qRT-PCR) in MDA-MB-231 (a breast cancer cell line) infected with oHSV-1 and non-infected cells. Results The results of our data analysis indicated notable disparities in the genes associated with the proteasome pathway between infected and non-infected cells. Our ontology analysis revealed that the proteasome-mediated ubiquitin-dependent protein catabolic process was a significant biological process, with a p-value of 5.8E−21. Additionally, extracellular exosomes and protein binding were identified as significant cellular components and molecular functions, respectively. Common hub genes with degree and maximum neighborhood component (MNC) methods, including PSMD2, PSMD4, PSMA2, PSMD14, PSMD11, PSMC3, PSMC2, PSMD8, and PSMA4, were also identified. Analysis of gene expression by qRT-PCR and differential gene expression revealed that GADD45g genes can be effective genes in the proliferation of oncolytic HSV-1 virus. Conclusion The transcriptome changes in tumor cells infected by oHSV-1 may be utilized to predict oncolytic efficacy and provide rational strategies for designing next-generation oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0ce55c7be9d2abb820866201dacdc5afac972e" target='_blank'>
              Rational strategies for designing next-generation oncolytic viruses based on transcriptome analysis of tumor cells infected with oncolytic herpes simplex virus-1
              </a>
            </td>
          <td>
            Naeme Javid, Shahriyar Abdoli, Majid Shahbazi
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c49e06286441ee42660f276752afffa0be81c30" target='_blank'>
              Dysregulation of GAS5-miRNA-Mediated Signaling Pathways in Cancer Pathobiology: A Comprehensive Exploration of Pathways Influenced by this Axis.
              </a>
            </td>
          <td>
            Enwa Felix Oghenemaro, Abdulrahman Qais Khaleel, J. Rizaev, R. Roopashree, Muath Suliman, Syeda Wajida Kazmi, Ahmed Hjazi, Pranchal Rajput, Yasser Fakri Mustafa, M. Abosaoda
          </td>
          <td>2024-12-24</td>
          <td>Biochemical genetics</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyu Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tetraploidy is a condition in which the entire set of chromosomes doubles, most often due to errors during cell division. Tetraploidy can lead to genomic instability and significant consequences, in particular metastasis and treatment failure in tumours, including radiotherapy. The development of new strategies to sensitise these cells to treatment is of great importance. In our study, we investigated the in vitro combination of chemical treatment with the kinase inhibitor SP600125 and irradiation on diploid versus metastatic tetraploid RKO colon cancer clones. We assessed mitochondrial transmembrane potential, cell cycle and subG1 population by flow cytometry and performed clonogenic assays to evaluate cell sensitivity. We found that the combination overcomes irradiation resistance in metastatic tetraploid clones. To identify the main pathway involved in cell sensitivity, we screened the Harvard Medical School KINOMEscan library and performed a gene ontology biological process analysis. We found that the major kinases inhibited by SP600125 were ANKK1, BIKE, IKKA, JNK1, MP2K3, MP2K4, MKNK2, MYLK, PLK4, RPS6KA4(Kin,Dom,1), MYLK4 and TTK, and the pathways involved in clone sensitivity were DNA damage repair, radiation resistance and apoptosis, through JNK pathway inhibition. Finally, our main finding was that combined treatment with SP600125 and radiotherapy reduced the resistance of metastatic tetraploid cells to treatment, essentially by inhibiting the JNK pathway. This result supports a promising anti-cancer strategy to overcome the resistance of tetraploid cancer cells to irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0380fc971abb59db901b3b88a1c597ae6b670f0" target='_blank'>
              JNK Inhibition Overcomes Resistance of Metastatic Tetraploid Cancer Cells to Irradiation-Induced Apoptosis
              </a>
            </td>
          <td>
            Mohamed Jemaà, N. Setti Boubaker, Nesrine Kerkeni, Stephan M. Huber
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5ba64df098cc1a4c221cfe3104b836d138b7b9" target='_blank'>
              Genome-wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2025-01-15</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H. Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Genomic integrity is critical for cellular homeostasis, preventing the accumulation of mutations that can drive diseases such as cancer. Among the mechanisms safeguarding genomic stability, the Base Excision Repair (BER) pathway plays a pivotal role in counteracting oxidative DNA damage caused by reactive oxygen species. Central to this pathway are enzymes like 8-oxoguanine glycosylase 1 (OGG1), which recognize and excise 8-oxo-7,8-dihydro-2′-deoxyguanosine (8-oxodG) lesions, thereby initiating a series of repair processes that restore DNA integrity. BER inhibitors have recently been identified as a promising approach in cancer therapy, increasing the sensitivity of cancer cells to radiotherapy and chemotherapy. By exploiting tumor-specific DNA repair dependencies and synthetic lethal interactions, these inhibitors could be used to selectively target cancer cells while sparing normal cells. This review provides a robust reference for scientific researchers, offering an updated perspective on small-molecule inhibitors targeting the 8-oxodG-BER pathway and highlighting their potential role in expanding cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e72df5cdd609ef616fe0cccc3b1963c63863d272" target='_blank'>
              Targeting the 8-oxodG Base Excision Repair Pathway for Cancer Therapy
              </a>
            </td>
          <td>
            Anna Piscone, Francesca Gorini, Susanna Ambrosio, Anna Noviello, Giovanni Scala, Barbara Majello, Stefano Amente
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer is among the most prevalent malignant tumors globally, with carboplatin serving as a standard treatment option. However, resistance often compromises its efficacy. DNA damage repair (DDR) pathways are crucial in determining responses to treatment and are also associated with immune infiltration. This study aimed to identify the DDR genes involved in carboplatin resistance and to elucidate their effects on prognosis, immune infiltration, and drug sensitivity in breast cancer patients.A 3D-culture model resistant to carboplatin was constructed and sequenced. Co-expressed DDR genes were analyzed to develop a predictive model. Immune infiltration analysis tools were employed to assess the immune microenvironment of patients with varying expression levels of these risk genes. Additionally, drug sensitivity predictions were made to evaluate the efficacy of other DNA damage-related drugs across different risk groups. Molecular assays were performed to investigate the role of the key gene TONSL in breast cancer.By integrating data from public database, we established a prognostic signature comprising thirteen DDR genes. Our analysis indicated that this model is associated with immune infiltration patterns in breast cancer patients, particularly concerning CD8+ T cells and NK cells. Additionally, it demonstrated a significant correlation with sensitivity to other DDR-related drugs, suggesting its potential as a biomarker for treatment efficacy. Compared to the control group, TONSL-knockdown cell lines exhibited a diminished response to DNA-damaging agents, marked by a notable increase in DNA damage levels and enhanced drug sensitivity. Furthermore, single-cell analysis revealed elevated TONSL expression in dendritic and epithelial cells, particularly in triple-negative breast cancers.Carboplatin resistance-related DDR genes are associated with prognosis, immune infiltration, and drug sensitivity in breast cancer patients. TONSL may serve as a potential therapeutic target for breast cancer, particularly in triple-negative breast cancer, indicating new treatment strategies for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d51e511dd91c5a50202d42e560936d1506bea71" target='_blank'>
              Carboplatin-resistance-related DNA damage repair prognostic gene signature and its association with immune infiltration in breast cancer
              </a>
            </td>
          <td>
            Shuwen Dong, Anqi Li, Ruixin Pan, Jin Hong, Zheng Wang, Kunwei Shen
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Organisms regulate gene expression in response to environmental cues, a process known as plasticity, to adjust to changing environments. Research into natural variation and the evolution of plasticity frequently studies cis-regulatory elements with theory suggesting they are more important evolutionarily than trans-regulatory elements. Genome-wide association studies have supported this idea, observing a predominance of cis-loci affecting plasticity. However, studies in structured populations provide a contrasting image, raising questions about the genetic architecture of natural variation in plasticity. To circumvent potential statistical difficulties present in genome-wide association studies, we mapped loci underlying transcriptomic plasticity in response to salicylic acid using recombinant inbred lines generated from two random Arabidopsis thaliana accessions. We detected extensive transgressive segregation in the salicylic acid response, suggesting that plasticity to salicylate in Arabidopsis is polygenic. Most loci (>75%) underlying this variation act in trans, especially for loci influencing plasticity. Trans-acting loci were enriched in genome hotspots, with predominantly small effect sizes distributed across many genes. This could potentially explain their under-discovery in genome-wide association studies. This work reveals a potentially important role for trans-acting loci in plastic expression responses, with implications for understanding plant adaptation to different environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6242103d645ac263c51e56f8b61d6b0403cdeb3b" target='_blank'>
              Trans-regulatory loci shape natural variation of gene expression plasticity in Arabidopsis
              </a>
            </td>
          <td>
            Mariele Lensink, Grey Monroe, Dan Kliebenstein
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3899cef9922a83c1043ba54cdf83bc829243942" target='_blank'>
              The role of genetic diversity, epigenetic regulation, and sex-based differences in HIV cure research: a comprehensive review
              </a>
            </td>
          <td>
            Punitha Letchumanan, Kumitaa Theva Das
          </td>
          <td>2025-01-03</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Chemotherapy is a potent tool against cancer, but drug resistance remains a major obstacle. To combat this, understanding the molecular mechanisms behind resistance in cancer cells and the protein expression changes driving these mechanisms is crucial. Targeting the Ubiquitin‐Proteasome System (UPS) has proven effective in treating multiple myeloma and shows promise for solid tumours. Despite initial success with the proteasome inhibitor bortezomib, acquired resistance soon after treatment poses a significant challenge to its efficacy. In this study, we explored proteins potentially involved in acquired resistance to bortezomib using label‐free nLC–MS/MS proteomic analysis. The investigation revealed 299 proteins with notable differences in expression levels in the bortezomib‐resistant PC3 prostate cancer cell line. Using bioinformatics tools, we illustrated the top 10 gene ontology (GO) processes [e.g., translational initiation (p = 5.964E‐10), CRD‐mediated mRNA stabilisation (p = 1.636E‐5), and hydrogen ion transmembrane transport (p = 6.46E‐5)] and the top 20 KEGG [e.g., metabolic pathways (p = 7.601E‐13), biosynthesis of amino acids (p = 3.834E‐12), and chemical carcinogenesis—reactive oxygen species (p = 1.891E‐4)] and REACTOME [e.g., metabolism (p = 4.182E‐21), translation (p = 9.484E‐18), and Nonsense‐Mediated Decay (NMD) (p = 1.829E‐8)] pathways in the PC3‐resistant cells. We further refined our results by comparing them with globally validated TCGA datasets. We correlated the 299 proteins identified through proteomic analysis with tumour aggressiveness and resistance by comparing them with the TCGA nodal metastasis N0 vs. N1 datasets using the UALCAN portal and identified 37 proteins consistent with our results. We believe that a combination of bortezomib with chemotherapeutics targeting these proteins could be effective in overcoming the resistance developed against bortezomib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb7e9b77b4e308aef46ff7c96b68009d3d9b402" target='_blank'>
              Comparative Analysis of Acquired Resistance to Bortezomib in Prostate Cancer Cells Using Proteomic and Bioinformatic Tools
              </a>
            </td>
          <td>
            Semih Seker, B. Sahin, A. Yerlikaya
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The goal of precision oncology is to find effective therapeutics for every patient. Through the inclusion of emerging therapeutics in a high‐throughput drug screening platform, our functional genomics pipeline inverts the common paradigm to identify patient populations that are likely to benefit from novel therapeutic strategies. Approach Utilizing drug screening data across a panel of 46 cancer cell lines from 11 tumor lineages, we identified an ovarian cancer‐specific sensitivity to the first‐in‐class CRL4 inhibitors KH‐4‐43 and 33‐11. CRL4 (i.e., Cullin‐4 RING E3 ubiquitin ligase) is known to be dysregulated in a variety of cancer contexts, making it an attractive therapeutic target. Unlike proteasome inhibitors that are associated with broad toxicity, CRL4 inhibition offers the potential for tumor‐specific effects. Results We observed that CRL4 inhibition negatively regulates core gene signatures that are upregulated in ovarian tumors and significantly slowed tumor growth as compared to the standard of care, cisplatin, in OVCAR8 xenografts. Building on this, we performed combination drug screening in conjunction with proteomic and transcriptomic profiling to identify ways to improve the antitumor effects of CRL4 inhibition in ovarian cancer models. CRL4 inhibition consistently resulted in activation of the mitogen‐activated protein kinase (MAPK) signaling cascade at both the transcriptomic and protein levels, suggesting that survival signaling is induced in response to CRL4 inhibition. These observations were concordant with the results of the combination drug screens in seven ovarian cancer cell lines that showed CRL4 inhibition cooperates with MEK inhibition. Preclinical studies in OVCAR8 and A2780 xenografts confirmed the therapeutic potential of the combination of KH‐4‐43 and trametinib, which extended overall survival and slowed tumor progression relative to either single agent or the standard of care. Conclusions Together, these data demonstrate the prospective utility of functional modeling pipelines for therapeutic development and underscore the clinical potential of CRL4 inhibition in the ovarian cancer context. Highlights A precision medicine pipeline identifies ovarian cancer sensitivity to CRL4 inhibitors. CRL4 inhibition induces activation of MAPK signalling as identified by RNA sequencing, proteomics, and phosphoproteomics. Inhibitor combinations that target both CRL4 and this CRL4 inhibitor‐induced survival signalling enhance ovarian cancer sensitivity to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c6bcbc9392bde91ee7e96491c8582460ba6dd" target='_blank'>
              Functional genomics pipeline identifies CRL4 inhibition for the treatment of ovarian cancer
              </a>
            </td>
          <td>
            S. Claridge, Shalini Nath, Anneliese Baum, Richard Farias, J. Cavallo, Nile M Rizvi, Lamberto De Boni, Eric Park, Genesis Lara Granados, Matthew Hauesgen, Ruben Fernandez-Rodriguez, Eda N Kozan, E. Kanshin, Khoi Q Huynh, Peng-Jen Chen, Kenneth Wu, Beatrix Ueberheide, Juan Miguel Mosquera, Fred R Hirsch, Robert J. DeVita, O. Elemento, Chantal Pauli, Zhen-Qiang Pan, Benjamin D Hopkins
          </td>
          <td>2025-01-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Every protein progresses through a natural lifecycle from birth to maturation to death; this process is coordinated by the protein homeostasis system. Environmental or physiological conditions trigger pathways that maintain the homeostasis of the proteome. An open question is how these pathways are modulated to respond to the many stresses that an organism encounters during its lifetime. To address this question, we tested how the fitness landscape changes in response to environmental and genetic perturbations using directed and massively parallel transposon mutagenesis in Caulobacter crescentus. We developed a general computational pipeline for the analysis of gene-by-environment interactions in transposon mutagenesis experiments. This pipeline uses a combination of general linear models, statistical knockoffs, and a nonparametric Bayesian statistical model to identify essential genetic network components that are shared across environmental perturbations. This analysis allows us to quantify the similarity of proteotoxic environmental perturbations from the perspective of the fitness landscape. We find that essential genes vary more by genetic background than by environmental conditions, with limited overlap among mutant strains targeting different facets of the protein homeostasis system. We also identified 146 unique fitness determinants across different strains, with 19 genes common to at least two strains, showing varying resilience to proteotoxic stresses. Experiments exposing cells to a combination of genetic perturbations and dual environmental stressors show that perturbations that are quantitatively dissimilar from the perspective of the fitness landscape are likely to have a synergistic effect on the growth defect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d35fab11c10334ca080d2ea7ac3de9caeecc10" target='_blank'>
              Discovering genetic modulators of the protein homeostasis system through multilevel analysis
              </a>
            </td>
          <td>
            Vishal Sarsani, Berent Aldikacti, Tingting Zhao, Shai He, Peter Chien, Patrick Flaherty
          </td>
          <td>2024-12-23</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ebbd60130879f07ccf62c91d033d068147d46a" target='_blank'>
              A safe haven for cancer cells: tumor plus stroma control by DYRK1B
              </a>
            </td>
          <td>
            Miriam Ems, Anna Brichkina, Matthias Lauth
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The understanding of genetic and epigenetic regulation of gene expression in endothelial and vascular smooth muscle cells remains fragmented with limited experimental validation. Methods Chromatin conformation (Micro-C), DNA methylation (RRBS), chromatin accessibility (ATAC-seq), and transcriptome profiles (RNA-seq) were mapped in human induced pluripotent stem cell (hiPSC)-derived, isogenic endothelial and vascular smooth muscle cells (iECs and iVSMCs). CTCF and RAD21 were depleted to assess the functional relevance of chromatin architecture, and genome editing was used to evaluate the allelic effect of a blood pressure-associated single nucleotide polymorphism (SNP). Results Significant correlations were identified between gene expression levels and chromatin interactions, chromatin accessibility, and DNA methylation in iECs and iVSMCs, with chromatin interactions showing the strongest association. Chromatin contact regions displayed distinct epigenetic landscapes depending on the types of regulatory element interactions involved. Perturbation of CTCF and RAD21 revealed their differential regulatory effects, particularly on the expression of genes overlapping chromatin contacts, with RAD21 exhibiting a broader regulatory impact. SNPs associated with several vascular traits were enriched in chromatin loops or accessible regions in iECs or iVSMCs. Precise genome editing demonstrated allele-dependent effects of SNP rs9833313 on the expression of SHOX2 located 247.4 kbp away but within the same chromatin loops as the SNP. Conclusion This study provides an extensive epigenetic landscape of vascular cells that may drive novel research on the role of genetic and epigenetic mechanisms of vascular function and disease as demonstrated by our targeted experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cca08a65eb2683798ce82f8cb05c4cd45763cf1" target='_blank'>
              Global and genetic regulation of gene expression in human endothelial and vascular smooth muscle cells
              </a>
            </td>
          <td>
            Pengyuan Liu, Atrayee Ray, Yong Liu, Manoj K. Mishra, Qiongzi Qiu, Rajan Pandey, Bhavika Therani, Jing Huang, Jiayi Ren, C. Stelloh, Xiaowen Bai, Andrew S Greene, Allen W Cowley, Aron M. Geurts, Sridhar Rao, Mingyu Liang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: Colorectal cancer (CRC) remains a leading cause of cancer mortality globally, with KRAS mutations occurring in 30–40% of cases, contributing to poor prognosis and resistance to anti-EGFR therapy. This review explores the biological significance, clinical implications, and therapeutic targeting of KRAS mutations in CRC. Methods: A comprehensive analysis of the existing literature and clinical trials was performed, highlighting the role of KRAS mutations in CRC pathogenesis, their impact on prognosis, and recent advancements in targeted therapies. Specific attention was given to emerging therapeutic strategies and resistance mechanisms. Results: KRAS mutations drive tumor progression through persistent activation of MAPK/ERK and PI3K/AKT signaling pathways. These mutations influence the tumor microenvironment, cancer stem cell formation, macropinocytosis, and cell competition. KRAS-mutant CRC exhibits poor responsiveness to anti-EGFR monoclonal antibodies and demonstrates primary and acquired resistance to KRAS inhibitors. Recent breakthroughs include the development of KRAS G12C inhibitors (sotorasib and adagrasib) and promising agents targeting G12D mutations. However, response rates in CRC remain suboptimal compared to other cancers, necessitating combination therapies and novel approaches, such as vaccines, nucleic acid-based therapeutics, and macropinocytosis inhibitors. Conclusions: KRAS mutations are central to CRC pathogenesis and present a significant therapeutic challenge. Advances in KRAS-targeted therapies offer hope for improved outcomes, but resistance mechanisms and organ-specific differences limit efficacy. Continued efforts in personalized treatment strategies and translational research are critical for overcoming these challenges and improving patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2803fdbec8d915a47df49692d27cd1e6edb65e91" target='_blank'>
              The Role of KRAS Mutations in Colorectal Cancer: Biological Insights, Clinical Implications, and Future Therapeutic Perspectives
              </a>
            </td>
          <td>
            Mitsunobu Takeda, Shoma Yoshida, Takuya Inoue, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, N. Miyoshi, Mamoru Uemura, Hirofumi Yamamoto, Y. Doki, H. Eguchi
          </td>
          <td>2025-01-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="The phenomenon of adaptive mutation in bacteria presents several challenges to classical models of evolution, particularly regarding the observed coordination of mutation patterns across populations. Here, we examine statistical evidence from evolved Escherichia coli populations showing mutation enrichment up to 256-fold above background rates in key metabolic genes, with remarkable temporal stability that appears to transcend known bacterial communication mechanisms. We propose a theoretical framework suggesting that bacteria may utilize quantum coherent oscillations, potentially mediated through synchronized membrane potential fluctuations, to achieve this degree of coordination in adaptive mutation. Building on Sprouffske et al.’s findings that high mutation rates can limit adaptive evolution in E. coli, we examined mutation patterns in four key metabolic genes (pykF, topA, cspC, and rpoC) previously identified as targets of selection. While Sprouffske found that extremely high mutation rates impaired adaptation, our analysis reveals that these genes show non-random enrichment patterns (p < 1.76×10-34 for pykF) that maintain remarkable temporal stability across multiple timepoints. Recent advances in quantum biology have demonstrated sustained quantum coherence in biological systems, we present a model for how quantum-mediated bacterial signaling could potentially contribute to adaptive mutation. The framework makes several experimentally testable predictions about mutation patterns, population dynamics, and coherence times in bacterial populations under stress conditions. While substantial experimental validation remains necessary, this work provides specific approaches for investigating potential quantum contributions to bacterial adaptation, with implications for understanding evolutionary mechanisms and bacterial stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c15202aa7837996110092bdd84ad8961fcfb080a" target='_blank'>
              Could Quantum-Mediated Bacterial Signaling Explain Adaptive Mutation?
              </a>
            </td>
          <td>
            Patrick Ross
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) comprise a group of genetically heterogeneous hematological malignancies that result in the abnormal growth of leukemic cells and halt the maturation process of normal hematopoietic stem cells. Despite using molecular and cytogenetic risk classification to guide treatment decisions, most AML patients survive for less than five years. A deeper comprehension of the disease’s biology and the use of new, targeted therapy approaches could potentially increase cure rates. RAS oncogene mutations are common in AML patients, being observed in about 15–20% of AML cases. Despite extensive efforts to find targeted therapy for RAS-mutated AMLs, no effective and tolerable RAS inhibitor has received approval for use against AMLs. The frequency of RAS mutations increases in the context of AMLs’ chemoresistance; thus, novel anti-RAS strategies to overcome drug resistance and improve patients’ therapy responses and overall survival are the need of the hour. In this article, we aim to update the current knowledge on the role of RAS mutations and anti-RAS strategies in AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607005d4dc391e9d5c0e31b65105281e28950846" target='_blank'>
              Rat Sarcoma Virus Family Genes in Acute Myeloid Leukemia: Pathogenetic and Clinical Implications
              </a>
            </td>
          <td>
            Shaimaa Khattab, Adriatik Berisha, Natalia Baran, P. Piccaluga
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Genomic analysis has played a significant role in the identification of driver mutations that are linked to disease progression and response to drug treatment in ovarian cancer. A prominent example is the stratification of epithelial ovarian cancer (EOC) patients with homologous recombination deficiency (HRD) characterized by mutations in DNA damage repair genes such as BRCA1/2 for treatment with PARP inhibitors. However, recent studies have shown that some epithelial ovarian tumors respond to PARP inhibitors irrespective of their HRD or BRCA mutation status. An exclusive focus on the genome overlooks the significant insight that can be gained from other biological analytes, including proteins, which carry out cellular functions. Proteogenomics is the integration of genomics, transcriptomics, epigenomics and proteomics data. This review paper provides novel insight into the role of proteogenomics as an analytical approach to identify predictive biomarkers of drug treatment response in epithelial ovarian cancer. Proteogenomic analysis can facilitate the identification of predictive biomarkers of drug treatment response, consequently greatly improving the stratification of patients with EOC for treatment towards a goal of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1220cfea040dac3859c8764ba1d49996aa4caab" target='_blank'>
              Opportunities for predictive proteogenomic biomarkers of drug treatment sensitivity in epithelial ovarian cancer
              </a>
            </td>
          <td>
            T. Philips, Britt K. Erickson, Stefani N. Thomas
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ddc2e4963aa356dbc3de332034a633ed4d7780" target='_blank'>
              Antisense oligonucleotides as a targeted therapeutic approach in model of acute myeloid leukemia.
              </a>
            </td>
          <td>
            D. Maceckova, L. Vaňková, J. Bufka, Petr Hošek, Jiří Moravec, Pavel Pitule
          </td>
          <td>2024-12-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background. Colorectal carcinomas (CRCs) are seldom eradicated by cytotoxic chemotherapy. Cancer cells with stem-like functional properties, often referred to as “cancer stem cells” (CSCs), display preferential resistance to several anti-tumor agents used in cancer chemotherapy, but the molecular mechanisms underpinning their selective survival remain only partially understood. Methods. In this study, we used Transcription Factor Target Genes (TFTG) enrichment analysis to identify transcriptional regulators (activators or repressors) that undergo preferential activation by chemotherapy in CRC cells with a “bottom-of-the-crypt” phenotype (EPCAM+/CD44+/CD166+; CSC-enriched) as compared to CRC cells with a “top-of-the-crypt” phenotype (EPCAM+/CD44neg/CD166neg; CSC-depleted). The two cell populations were purified in parallel by fluorescence-activated cell sorting (FACS) from a patient-derived xenograft (PDX) line representative of a moderately differentiated human CRC, following in vivo chemotherapy with irinotecan (CPT-11). The transcriptional regulators identified as differentially activated were tested for differential expression in normal vs. cancer tissues, and in cell populations enriched in stem/progenitor cell-types as compared to differentiated lineages (goblet cells, enterocytes) in the mouse colon epithelium. Finally, the top candidate was tested for mechanistic contribution to drug-resistance by selective down-regulation using short-hairpin RNAs (shRNAs). Results. Our analysis identified E2F4 and TFDP1, two core components of the DREAM transcriptional repression complex, as transcriptional modulators preferentially activated by irinotecan in EPCAM+/CD44+/CD166+ as compared to EPCAM+/CD44neg/CD166neg cancer cells. The expression levels of both genes (E2F4, TFDP1) were found up-regulated in CRCs as compared to human normal colon tissues, and in a sub-population of mouse colon epithelial cells enriched in stem/progenitor elements (Epcam+/Cd44+/Cd66alow/Kitneg) as compared to other sub-populations enriched in either goblet cells (Epcam+/Cd44+/Cd66alow/Kit+) or enterocytes (Epcam+/Cd44neg/Cd66ahigh). Most importantly, E2F4 down-regulation using shRNAs dramatically enhanced the sensitivity of human CRCs to in vivo treatment with irinotecan, across three independent PDX models. Conclusions. Our data identified E2F4 and the DREAM repressor complex as critical regulators of human CRC resistance to irinotecan, and as candidate targets for the development of chemo-sensitizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3c8fee4cf42a4e76a7d265977fa7fcfba7541" target='_blank'>
              The E2F4 transcriptional repressor is a key mechanistic regulator of colon cancer resistance to irinotecan (CPT-11)
              </a>
            </td>
          <td>
            Junichi Matsubara, Y. F. Li, Sanjay Koul, Junko Mukohyama, Luis E. Valencia Salazar, Taichi Isobe, D. Qian, Michael F. Clarke, Debashis Sahoo, R. Altman, Piero Dalerba
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Type 2 diabetes mellitus (T2DM) is a chronic metabolic disorder characterized by hyperglycemia. Current therapeutic strategies primarily manage symptoms, leaving a substantial unmet need for curative interventions. This review explores the potential of CRISPR-dCas9-mediated CpG island editing as a promising therapeutic approach for T2DM. CpG islands, DNA regions enriched in cytosine-guanine dinucleotides, play a pivotal role in gene regulation. Their methylation status significantly influences gene expression. By targeting specific CpG islands within genes involved in glucose metabolism and insulin signaling, CRISPR-dCas9 can modulate gene expression and restore metabolic homeostasis. A particular focus is placed on the TXNIP gene, implicated in T2D pathogenesis. Reprogramming TXNIP expression using CRISPR-dCas9 offers potential therapeutic benefits, including protecting pancreatic beta cells, enhancing insulin sensitivity, and mitigating inflammation. While the potential of CRISPR-dCas9-mediated CpG island editing is clear and evident now, further steps are imperative to translate this approach into effective and safe therapies for T2DM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/383b7fe07ebbb6081b1bfffef778fbc335d6d9c3" target='_blank'>
              CRISPR-dCas9-mediated CpG island editing: A potential game-changer for diabetes treatment
              </a>
            </td>
          <td>
            K. K. Dutta
          </td>
          <td>2024-12-21</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Questions in cancer have engaged systems biologists for decades. During that time, the quantity of molecular data has exploded, but the need for abstractions, formal models, and simplifying insights has remained the same. This review brings together classic breakthroughs and recent findings in the field of cancer systems biology, focusing on cancer-cell pathways for tumorigenesis and therapeutic response. Cancer cells mutate and transduce information from their environment to alter gene expression, metabolism, and phenotypic states. Understanding the molecular architectures that make each of these steps possible is a long-term goal of cancer systems biology pursued by iterating between quantitative models and experiments. We argue that such iteration is the best path to deploying targeted therapies intelligently so that each patient receives the maximum benefit for their cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8544584d0cff558bd248c81e02267385e425b55" target='_blank'>
              Systems Biology of the Cancer Cell.
              </a>
            </td>
          <td>
            Kevin A. Janes, M. Lazzara
          </td>
          <td>2024-12-17</td>
          <td>Annual review of biomedical engineering</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gene expression signatures predictive of chemotherapeutic response have the potential to greatly extend the reach of precision medicine by allowing medical providers to plan treatment regimens on an individual basis for patients with and without actionable mutations. Most published gene signatures are only capable of predicting response for individual drugs, but currently, a majority of chemotherapy regimens utilize combinations of different agents. We propose a unified framework, called the chemogram, that uses predictive gene signatures to rank the relative predicted sensitivity of different drugs for individual tumor samples. Using this approach, providers could efficiently screen against many therapeutics to identify the drugs that would fit best into a patient’s treatment plan at any given time. This can be easily reassessed at any point in time if treatment efficacy begins to decline due to therapeutic resistance. To demonstrate the utility of the chemogram, we first extract predictive gene signatures using a previously established method for extracting pan-cancer signatures inspired by convergent evolution. We derived 3 signatures for 3 commonly used cytotoxic drugs (cisplatin, gemcitabine, and 5-fluorouracil). We then used these signatures in our framework to predict and rank sensitivity among the drugs within individual cell lines. To assess the accuracy of our method, we compared the rank order of predicted response to the rank order of observed response (fraction of surviving cells at a standardized dose) against each of the 3 chemotherapies. Across a majority of cancer types, chemogram-generated predictions were consistently more accurate than randomized prediction rankings, as well as prediction rankings made by randomly generated gene signatures. In addition to the chemogram’s ability to rank relative sensitivity for any given tumor, this framework is easily scalable for any number of drugs for which a predictive signature exists. We repeated the process described above for 10 drugs and found that the accuracy of the predicted sensitivity rankings was maintained as the number of drugs in the chemogram’s screen increased. Our proposed framework demonstrates the ability of transcriptomic signatures to not only predict chemotherapeutic response but correctly assign rankings of drug sensitivity on an individual basis. With further validation, the chemogram could be easily integrated in a clinical setting, as it only requires gene expression data, which is less expensive than an extensive drug screen and can be performed at scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff761858a9cb7260f3bcdb566718490d8fb20c4" target='_blank'>
              Personalizing chemotherapy drug selection using a novel transcriptomic chemogram
              </a>
            </td>
          <td>
            Kristi Lin-Rahardja, J. Scarborough, Jacob G. Scott
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy characterized by the clonal expansion of myeloid precursor cells. Advances in genomic profiling have enhanced our understanding of AML pathogenesis, leading to the identification of recurrent mutations, including TP53, FLT3, MUC4, RAS, and IDH1/2. These mutations significantly influence treatment response and prognosis, with TP53 mutations conferring poor outcomes and resistance to conventional therapies. Despite the advent of venetoclax-based regimens, resistance mechanisms persist, necessitating the development of novel therapeutic strategies. This study aims to investigate the efficacy of drug combinations for the treatment of AML using both in vitro AML cell lines and an in vivo zebrafish embryo xenograft model. Specifically, we focus on two drug combinations; the pan-RAF inhibitor LY3009120 combined with the mTOR inhibitor sapanisertib (designated as LS), and the JAK1/2 inhibitor ruxolitinib combined with the ERK inhibitor ulixertinib (designated as RU). The study integrates real-time cell viability assays, xenograft imaging, and genomic analyses to assess drug efficacy and explore correlations between treatment responses and mutational profiles, particularly TP53, FLT3, and MUC4 mutations. Both combinations, LS and RU, demonstrated superior efficacy compared to venetoclax-based treatments in reducing cell viability across AML cell lines. The LS combination showed significant reductions in cell viability in MOLM16 and SKM cells, while RU exhibited comparable efficacy with lower toxicity profiles. In zebrafish embryos, the LS combination effectively inhibited the proliferation of xenografted human AML cells, as evidenced by decreased fluorescence signals, indicating cell death. The RU combination also disrupted survival signaling pathways, showing promise as a therapeutic strategy. Furthermore, a correlation was identified between drug response and mutational profiles, with TP53, FLT3 and MUC4 mutations significantly influencing sensitivity to the LS and RU combinations. These findings support the further development of LS and RU as effective alternatives to current clinical regimens, with implications for personalized AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e928f03f37069a717618c39ee33259e9551d5324" target='_blank'>
              Targeting AML Resistance with LY3009120-Sapanisertib and Ruxolitinib-Ulixertinib Combinations Demonstrate Superior Efficacy in FLT3, TP53, and MUC4 mutations
              </a>
            </td>
          <td>
            Elham Gholizadeh, Ehsan Zangene, A. Parsons, M. Kontro, Caroline A. Heckman, Mohieddin Jafari
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d077c0e04d5c20a662dd5e8ac2b16b1dbaf335" target='_blank'>
              Integrative pan-cancer genomic analysis highlights mitochondrial protein p32 as a potential therapeutic target in Myc-driven tumorigenesis.
              </a>
            </td>
          <td>
            Qiufen Bi, Jun Nie, Qiang Wu, Liang Sun, Shuang Zhu, Jin Bai, Yong Liu, Fang Huang, Keli Chai
          </td>
          <td>2025-02-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epithelial ovarian cancer (EOC) is a cancer that affects the female reproductive system and is highly lethal. It poses significant challenges in terms of treatment and often has a poor prognosis. In recent years, with the advent of PARPi, the treatment of ovarian cancer has entered a new stage of full-process management. Although more and more drugs have been approved, the therapeutic effect of PARPi is still very limited. With the rapid development of PD-1/PD-L1, CTLA-4, oncolytic viruses, cancer vaccines, adoptive cell therapy, etc., tumor immunotherapy has provided new opportunities for the treatment of ovarian cancer.This study used comprehensive transcriptome analysis across multiple databases to gather gene transcripts and clinical features of normal ovarian samples and tissue samples from ovarian cancer. The aim was to explore the mechanisms underlying tumor immunotherapy resistance and to reveal the relationship between ovarian cancer’s immune microenvironment and genes linked to inflammation. Various R packages were used for differential gene analysis, enrichment analysis, co-expression network construction, and prognostic model building.It has been found that the prognosis of ovarian cancer patients is closely associated with sets of genes involved in inflammation. The immune infiltration microenvironment, clinicopathological features, and survival rates differed significantly between two inflammatory gene expression patterns identified using cluster and immune microenvironment analyses. Further analysis revealed that the high-risk group had a higher abundance of M2-type macrophage infiltration, more active anti-tumor immune response, higher tumor stemness score, potentially worse prognosis, and lower response rates to multiple chemotherapy drugs and immune checkpoint inhibitors.These findings provide new perspectives and potential targets for immunotherapy and prognostic evaluation of ovarian cancer and offer new strategies and directions for clinical treatment and patient management. This study provides crucial information to further our comprehension of drug response mechanisms and tumor immunotherapy. It offers new strategies and methods for the treatment and prognostic improvement of ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a9999d5dd5f48a0fabbe04bc7e87dced9b12813" target='_blank'>
              Transcriptome analysis of ovarian cancer uncovers association between tumor-related inflammation/immunity and patient outcome
              </a>
            </td>
          <td>
            Jingfang Wang, Wenrui Zhu, Xia Li, Yuanyuan Wu, Wenhui Ma, Yangzhou Wang, Weihong Zhao, Fang Wei, Wenhao Wang
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11afec2e2fbad72ef42b2a5b045ab6400e1d976a" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies ITGA8 responsible for abivertinib sensitivity in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Xuan-Guang Li, Guangsheng Zhu, Pei-Jun Cao, Hua Huang, Yu-Hao Chen, Chen Chen, Pei-Jie Chen, Di Wu, Chen Ding, Zi-He Zhang, Rui-Hao Zhang, Zi-Xuan Hu, Wen-Hao Zhao, Ming-Hui Liu, Yong-Wen Li, Hongyu Liu, Jun Chen
          </td>
          <td>2025-01-14</td>
          <td>Acta pharmacologica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e585936250f3de7ab4e7a29d9b3f79e176e62ba4" target='_blank'>
              The molecular impact of miR-326 in acute lymphoblastic leukemia and its cross talk with P53.
              </a>
            </td>
          <td>
            Saba Shafieizadegan, N. Aberuyi, S. Rahgozar
          </td>
          <td>2025-02-05</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Bacterial genomic DNA inversions, which govern molecular phase-variations, provide the bacteria with functional plasticity and phenotypic diversity. These targeted rearrangements enable bacteria to respond to environmental challenges, such as bacteriophage predation, evading immune detection or gut colonization. This study investigated the short- and long-term effects of the lytic phage Barc2635 on the functional plasticity of Bacteroides fragilis, a gut commensal. Germ-free mice were colonized with B. fragilis and exposed to Barc2635 to identify genomic alterations driving phenotypic changes. Phage exposure triggered dynamic and prolonged bacterial responses, including significant shifts in phase-variable regions (PVRs), particularly in promoter orientations of polysaccharide biosynthesis loci. These shifts coincided with increased entropy in PVR inversion ratios, reflecting heightened genomic variability. In contrast, B. fragilis in control mice exhibited stable genomic configurations after gut adaptation. The phase-variable Type 1 restriction-modification system, which affects broad gene expression patterns, showed variability in both groups. However, phage-exposed bacteria displayed more restrained variability, suggesting phage-derived selection pressures. Our findings reveal that B. fragilis employs DNA inversions to adapt rapidly to phage exposure and colonization, leveraging genomic variability for resilience. This study emphasizes gut bacterial genomic and phenotypic plasticity upon exposure to the mammalian host and to bacteriophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f81af1bcd2a68ada21239ed5d5a4363a88cc20" target='_blank'>
              Bacteriophage-driven DNA inversions shape bacterial functionality and long-term co-existence in Bacteroides fragilis
              </a>
            </td>
          <td>
            S. Carasso, Roni Keshet-David, Jia Zhang, Haitham Hajjo, Dana Kadosh-Kariti, T. Gefen, N. Geva-Zatorsky
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cytarabine, daunorubicin, and etoposide (ADE) have been the standard backbone of induction chemotherapy regimens for acute myeloid leukemia (AML) patients for over five decades. However, chemoresistance is still a major concern, and a significant proportion of AML becomes resistant to ADE treatment leading to relapse and poor survival. Therefore, there is a significant need to identify mechanisms mediating drug resistance to overcome chemoresistance. Herein, we performed genome-wide synthetic lethal CRISPR/Cas9 screens using araC, daunorubicin and etoposide. We further integrated significant findings from the CRISPR screen with outcome in pAML patients treated with standard ADE regimen on 3 independent clinical trials to identify drug response biomarkers of prognostic significance. We identified seveal mediators that would represent clinically and biologically significant genes for ADE treatment, such as BCL2, CLIP2, and VAV3, which are resistant genes with high expression associated with poor outcomes in pAML treated with ADE, and GRPEL1, HCFC1, and TAF10, which are sensitive genes with high expression showing beneficial outcomes. Notably, knockdowns of the BCL2, CLIP2, and VAV3 genes sensitize the ADE component in AML cell lines, suggesting that these genes should be further studied as potential therapeutic targets to overcome chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979aaf7e330314833a544cf8bca5acd47b995421" target='_blank'>
              Genome-wide CRISPR screen identifies AraC-Dauno-Eto (ADE) response modulators that predicts outcome in pediatric AML.
              </a>
            </td>
          <td>
            Nam Nguyen, Roya Rafiee, P. K. Parcha, A. Tagmount, J. Rubnitz, Raul C Ribeiro, Xueyuan Cao, S. Pounds, Christopher Vulpe, J. Lamba
          </td>
          <td>2024-12-23</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ca816cc15851962002a66601f774cc9384fd1d" target='_blank'>
              Comprehensive genome-scale CRISPR knockout screening of CHO cells
              </a>
            </td>
          <td>
            Sungwook Shin, Su Hyun Kim, Aghiles Gasselin, Gyun Min Lee, Jae Seong Lee
          </td>
          <td>2025-01-15</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Background Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment. Methods Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real‐time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts. Results The study comprised 12 patients, all under the age of 18. Post‐treatment RT‐PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR‐129‐5p, and miR‐9‐5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive. Conclusion MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP‐Binding Cassette Transporter Genes and microRNAs in Pediatric B‐Cell ALL: Expression Insights
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of Clinical Laboratory Analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood. Methods We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts. Results Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis. Conclusions DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            Tian-Hang Yu, Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu- Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Currently, mankind is fiercely struggling with cancer. Recently, we have been winning the battle against cancer through precision medicine and accompanying diagnostic methods, and we are raising many hopes with blockbuster drugs. It would be even better if we could read the cancer nucleotide sequence, identify them in advance, and suggest treatments simultaneously. However, this may be an impossible dream because it takes a lot of time and effort to diagnose and ensure all the long gene sequences of cancer at once. Thus, victory will be even closer if a rapid and accurate diagnosis of the cancer-specific gene biomarkers that will soon be imprinted can be made. With the advent of nanotechnology, a new short cancer diagnostic toolkit has been proposed to achieve the goal. This review presents a small diagnostic device that detects genetic fragments (simply 'Gizmo') of certain cancers. The development of numerous diagnostic methods has focused on (1) directly detecting pre-selectively targeted genes using novel diagnostic systems, and (2) indirectly detecting substantial improvements in diagnostic sensitivity only through detection signal amplification without existing gene amplification steps. Our fight against cancer is not a dream, but the result of success, and it is assumed that victory will accelerate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eccae0a3a741529fc24b2b02ce5646b06a22df12" target='_blank'>
              Engineering Gizmos for Short Cancer Genetic Fragments Discrimination.
              </a>
            </td>
          <td>
            Jisoo Yuk, Jeonghun Kim, Sunghwan Jung, Soong Ho John Um
          </td>
          <td>2025-02-05</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT The discovery at the end of the 20th century of genes that induce cell death revolutionised the biocontaintment of genetically manipulated bacteria for environmental or agricultural applications. These bacterial ‘killer’ genes were then assayed for their potential to target and control malignant cells in human cancers. The identification of the bacteriomes in different human organs and tissues, coupled with the observation that bacteria tend to accumulate near tumours, has opened new avenues for anti‐cancer strategies. This progress, along with recent insights into how cancer cells evade immune response, has prompted innovative therapeutic approaches. Tumour microenvironments are typically nutrient‐rich, characterised by low oxygen tensions and very resistant to immune responses. Two recent studies in MBT highlight the promise of using Salmonella typhimurium and Escherichia coli as vectors in novel cancer treatments. Engineered S. typhimurium strains can generate adjuvant flagellin‐antigen complexes that function as in situ vaccines, hence increasing the immunogenic responses within tumour environment. Similarly, gut E. coli can be used as vectors to targert tumour cells in colon cancer, enabling both diagnostic applications and localised treatments. Both approaches hold significant potential to improve patient survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bba9ffcefc81fe9e0a146a68f1324d903a6bef1" target='_blank'>
              Bacteria as Precision Tools for Cancer Therapy
              </a>
            </td>
          <td>
            C. Michán, José Prados, Juan-Luis Ramos
          </td>
          <td>2025-01-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic mechanism that helps define and maintain cellular functions. It is influenced by many factors, including environmental exposures, genotype, cell type, sex, and aging. Since age is the primary risk factor for developing neurodegenerative diseases, it is important to determine if aging-related DNA methylation is retained when cells are reprogrammed to an induced Pluripotent Stem Cell (iPSC) state. Here, we selected peripheral blood mononuclear cells (PBMCs; n = 99) from a cohort of diverse and healthy individuals enrolled in the Genetic and Epigenetic Signatures of Translational Aging Laboratory Testing (GESTALT) study to convert to iPSCs. After reprogramming we evaluated the resulting iPSCs for DNA methylation signatures to determine if they reflect the confounding factors of age and environmental factors. We used genome-wide DNA methylation arrays in both cell types to show that the epigenetic clock is largely reset to an early methylation age after conversion of PBMCs to iPSCs. We further examined the epigenetic age of each cell type using an Epigenome-wide Association Study (EWAS). Finally, we identified a set of methylation Quantitative Trait Loci (methQTL) in each cell type. Our results show that age-related DNA methylation is largely reset in iPSCs, and each cell type has a unique set of methylation sites that are genetically influenced. Graphical Abstract Highlights Generation of a population-level set of iPSC lines from healthy individuals across the lifespan Aging-related features were reset based on epigenetic markers of cytosine methylation and telomere length By comparing methQTLs in iPSC vs. their donor PBMCs, we find that detection of methQTLs reflect biological functions of different cell types">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399ecd8f3690914bdcf9c8d820f3abd8488f2bee" target='_blank'>
              Characterization of DNA methylation in PBMCs and donor-matched iPSCs shows methylation is reset during stem cell reprogramming
              </a>
            </td>
          <td>
            X. Reed, Cory A. Weller, S. Sáez-Atienzar, A. Beilina, Sultana Solaiman, Makayla K Portley, Mary Kaileh, Roshni Roy, Jinhui Ding, A. Moore, D. T. Whitaker, Bryan J. Traynor, J. R. Gibbs, Sonja W. Scholz, Mark R. Cookson
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background/Objectives: Treatment with tyrosine kinase inhibitors (TKIs) in chronic myeloid leukemia (CML) has revolutionized disease management and has transformed CML from a life-threatening disease to a chronic condition for many patients. However, overcoming resistance, particularly related to leukemic stem cells (LSC) that can persist even when the bulk of the leukemic cells are eliminated, remains a significant challenge. Methods: K562 and KU812 cell lines were treated in vitro with the TKI Imatinib (IM). Gene expression, protein analysis, and metabolomic screening were conducted to investigate the ability of the drug to enhance stem cell (SC) features. Moreover, a gene ontology analysis was performed on different available datasets, to further consolidate our data. Results: 48 h of IM treatment can significantly increase the expression of genes related to SC self-renewal, particularly SOX2 and OCT 3/4. Interestingly, these modulations occur in cells that remain alive after drug treatment and that displayed features consistent with leukemia stem-like CML cells, suggesting that SC genes levels are crucial even in cell population survived upon TKI treatment. Moreover, after in silico analysis of available data, we observed an enrichment of SOX2/NANOG and OCT 3/4 signatures after TKI treatment, thus strengthening our results. Conclusions: Our results confirmed the relevance of LSC features after TKI treatment, highlighting the need for more effective and potentially curative strategies targeting LSCs to overcome resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28fb6d29d24fd74f86c83c9e97c902987037fdab" target='_blank'>
              Tyrosine Kinase Inhibitor Therapy Enhances Stem Cells Profile and May Contribute to Survival of Chronic Myeloid Leukemiastem Cells
              </a>
            </td>
          <td>
            Simone Rocco, A. Maglione, Valentina Schiavo, Alessandro Ferrando, Carmen Fava, D. Cilloni, Barbara Pergolizzi, C. Panuzzo
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1622b2dfde4aac03266cbf9b4a4aa33c5ec0f83" target='_blank'>
              A TBX2-driven signaling switch from androgen receptor to glucocorticoid receptor confers therapeutic resistance in prostate cancer.
              </a>
            </td>
          <td>
            Sayanika Dutta, Hamed Khedmatgozar, G. Patel, Daniel Latour, Jonathan Welsh, Mainak Mustafi, Antonina Mitrofanova, Manisha Tripathi, S. Nandana
          </td>
          <td>2024-12-20</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de04bf4687911966a6407dbe9d44e6b84ed97d20" target='_blank'>
              NFKB1 as a key player in Tumor biology: from mechanisms to therapeutic implications
              </a>
            </td>
          <td>
            Zixuan Song, Zheng Feng, Xiaoxue Wang, Jingying Li, Dandan Zhang
          </td>
          <td>2025-01-11</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc5699f8d775d7c0b21dfb5b4e13161b3a6de2f" target='_blank'>
              Appraisal of CRISPR Technology as an Innovative Screening to Therapeutic Toolkit for Genetic Disorders.
              </a>
            </td>
          <td>
            A. Shahid, Ambreen Zahra, S. Aslam, Amen Shamim, Waqas Rafique Ali, Bilal Aslam, Sultan Habibullah Khan, Muhammad Imran Arshad
          </td>
          <td>2025-02-02</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer is the most commonly diagnosed cancer and a leading cause of cancer-related mortality among men in the United States. Androgen deprivation therapy (ADT) has long been the cornerstone of treatment for advanced prostate cancer. However, most patients eventually develop resistance, leading to progression into the lethal castration-resistant prostate cancer (CRPC) stage, which is associated with poor survival rates. While next-generation androgen receptor signaling inhibitors (ARSIs), such as enzalutamide and abiraterone, have revolutionized CRPC treatment, resistance to these therapies remains a significant challenge, rendering the disease incurable. Recent studies have identified Polo-like kinase 1 (PLK1), a master regulator of mitosis, as a key driver of prostate cancer progression, particularly in CRPC. PLK1 activation has been linked to critical pathways, including oxidative stress responses, lipid metabolism, and androgen receptor (AR) signaling, which collectively promote tumor growth and ARSIs resistance. Moreover, PLK1 can regulate chromosomal stability and DNA damage repair pathways, contributing to sensitivity to irradiation therapy. Additionally, PLK1 plays a role in driving resistance to targeted therapies, such as PARP inhibitors, in BRCA-mutant CRPC. Compelling evidence further suggests that combining PLK1 inhibition with paclitaxel could serve as an effective therapeutic strategy to overcome resistance in CRPC by targeting microtubule dynamics. This review examines the mechanistic role and broader implications of PLK1 in CRPC treatment, offering valuable insights into potential combination therapies to improve efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1739c67f1f729180f6441e8a334eb27ea83080" target='_blank'>
              Plk1, a promising THERAPEUTIC target for prostate cancer treatment
              </a>
            </td>
          <td>
            Jia Peng, Mansoureh Nouri, Hamed Maasoumyhaghighi, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-17</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Multicopy plasmids are widespread in nature and compose a common strategy for spreading beneficial traits across microbes. However, the role of plasmids in supporting the evolution of encoded genes remains underexplored due to challenges in experimentally manipulating key parameters such as plasmid copy number and mutation rate. In this work, we developed a strategy for controlling copy number in the plasmid-based continuous evolution system, OrthoRep, and used our resulting capabilities to investigate the evolution of a conditionally essential gene under varying CN and mutation rate conditions. Our results show that low CN facilitates the faster enrichment of beneficial alleles while high CN promotes robustness through the maintenance of allelic diversity. High CN also slows the removal of deleterious mutations and increased the fraction of non-functional alleles that could hitchhike during evolution. This study highlights the nuanced relationships between plasmid CN, mutation rate, and evolutionary outcomes, providing insights into the adaptive dynamics of genes encoded on multicopy plasmids and nominating OrthoRep as a versatile tool for studying plasmid evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb00b500620346799e0e3cbc0de16d117d0474e" target='_blank'>
              A System to Explore the Adaptive Dynamics of Multicopy Plasmids: The Role of Copy Number and Mutation Rate in Evolutionary Outcomes
              </a>
            </td>
          <td>
            A. Pisera, Chang C. Liu
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743e24a828ab3b6aa71556c7a8a2010607a05574" target='_blank'>
              Classifications of triple-negative breast cancer: insights and current therapeutic approaches
              </a>
            </td>
          <td>
            Ziqi Chen, Yumeng Liu, Minchuan Lyu, Chi Ho Chan, Meiheng Sun, Xin Yang, Shuangying Qiao, Zheng Chen, Sifan Yu, Meishen Ren, Aiping Lu, Ge Zhang, Fangfei Li, Yuanyuan Yu
          </td>
          <td>2025-02-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc6bfc5e3065b396e263e4793dde2e7908cde4e" target='_blank'>
              CRISPR-StAR enables high-resolution genetic screening in complex in vivo models.
              </a>
            </td>
          <td>
            E. Uijttewaal, Joonsun Lee, Annika Charlotte Sell, Naomi Botay, Gintautas Vainorius, Maria Novatchkova, J. Baar, Jiaye Yang, Tobias Potzler, Sophie van der Leij, Christopher Lowden, Julia Sinner, Anais Elewaut, Milanka Gavrilovic, Anna Obenauf, Daniel Schramek, Ulrich Elling
          </td>
          <td>2024-12-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="WD repeat domain 4 (WDR4) is an essential member of the WD-repeat protein family, known for its regulatory roles in cellular processes critical to cancer development, including RNA modification, protein stability, cell cycle progression, and apoptosis. Studies have shown that WDR4 plays a pivotal role in tumorigenesis across various cancer types, with a particular focus on breast cancer in this study, where its overexpression is closely associated with aggressive tumor characteristics and poorer patient outcomes. As a scaffold protein, WDR4 is involved in N7-methylguanosine tRNA methylation and ubiquitin-mediated protein degradation, thereby regulating RNA stability, protein synthesis, and cell survival. This review provides a comprehensive analysis of WDR4’s molecular mechanisms, its oncogenic functions across different cancer types, and its interactions with other key factors in the tumor microenvironment, further exploring its potential role in tumor progression. As research on WDR4 progresses, we not only gain a deeper understanding of its complex role in tumor biology but also uncover new therapeutic avenues. In particular, the potential of WDR4 as a biomarker and therapeutic target is increasingly recognized. Despite the challenges faced in its clinical application, such as the difficulty in developing targeted therapies and managing side effects, the future prospects of WDR4 in cancer diagnosis and treatment remain promising, and it is expected to emerge as an effective therapeutic target in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c4a55ac88e695fbd2b794797f3e5107d58181d" target='_blank'>
              WD repeat domain 4 in tumorigenesis: Molecular mechanisms, cancer-type specific roles, and therapeutic potential
              </a>
            </td>
          <td>
            Xun Zou, Ling Tao, Bin Liu
          </td>
          <td>2025-02-05</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Following the development of therapeutic probiotics, there is an emerging demand for constraining engineered microbial activities to ensure biosafety. Many biocontainment studies developed genetic devices that involve cell death and growth inhibition on the engineered microbes, which often create basal levels of cytotoxicity that hamper cell fitness and performance on therapeutic functions; furthermore, these toxic pathways may promote genetic instability that leads to mutations and breakdown of biocontainment circuit. To address this issue, here we explore a circuit design that destroys the engineered genetic materials in a probiotic strain, instead of killing these cells, under non-permissive conditions. Our safeguard circuit involves a two-layered transcriptional regulatory circuit to control the expression of a CRISPR system that targets the engineered genes for degradation. In Escherichia coli Nissle 1917 (EcN), the biocontainment system continuously scavenged and destroyed the target until no engineered cellular function could be detected, suggesting this strategy has the potential to avoid escapee formation. Additionally, this safeguard circuit did not affect EcN cell fitness. We further demonstrated that the engineered probiotics inhabited in mouse guts and continued the engineered activities for at least 7 days when the permissive signal was supplied constantly; when the permissive signal was not provided, the engineered activities became undetectable within two days. Together, these studies support that our safeguard design is feasible for synthetic probiotic applications. HIGHLIGHTS Our safeguard system only destroys target genes and does not kill the host microbes It terminated engineered activities in guts in response to a loss of a signal This safeguard allowed synthetic probiotics to inhabit in guts for at least a week Cellobiose has great potential to serve as a continuous genetic signal in guts">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b48bd958d10822279d23f2cb90fbf35745cbf30" target='_blank'>
              A genetic safeguard for eliminating target genes in synthetic probiotics in response to a loss of the permissive signal in a gut environment
              </a>
            </td>
          <td>
            Nhu Nguyen, Miaomiao Wang, Lin Li, Clement T Y Chan
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Cell cycle dysregulation and the corresponding metabolic reprogramming play significant roles in tumor development and progression. CDK9, a kinase that regulates gene transcription and cell cycle, also induces oncogene transcription and abnormal cell cycle in AML cells. The function of CDK9 for gene regulation in AML cells requires further exploration. In this study, we knocked down the CDK9 to investigate its effects on the growth and survival of AML cells. Through RNA-seq analysis, we identified that in U937 cells CDK9 regulates numerous genes involved in proliferation and apoptosis, including mTOR, SREBF1, and Bcl-2. Furthermore, our results demonstrated that both CDK9 and FASN are crucial for the proliferation and survival of Kasumi-1 and U937 cells. Mechanistically, MCL1, c-Myc, and Akt/mTOR/SREBF1 may be critical factors and pathways in the combined therapy of NVP-2 and Orlistat. In summary, our study revealed that CDK9 and FASN are vital for maintaining AML cell survival and proliferation. Treatment with NVP-2 and Orlistat may be a promising clinical candidate for patients with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985b606d62bac6448e6bc281a807101bb710d0ae" target='_blank'>
              NVP-2, in combination with Orlistat, represents a promising therapeutic strategy for acute myeloid leukemia
              </a>
            </td>
          <td>
            Qing Zhu, Jia Cheng, Yuqing Gao, Zimu Zhang, Jian Pan, Xin Su, Danhong Fei, Linbo Cai, Juanjuan Yu, Yanling Chen, Wanyan Jiao, Di Wu, Xiaolu Li, Peifang Xiao
          </td>
          <td>2025-01-12</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda44ec94f8b435093cf034210fe1f5d7c4827e5" target='_blank'>
              Targeting MYC for the treatment of breast cancer: use of the novel MYC-GSPT1 degrader, GT19630.
              </a>
            </td>
          <td>
            Minhong Tang, J. Crown, Michael J Duffy
          </td>
          <td>2025-01-28</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: KRAS-mutated NSCLC has been targeted using monoclonal antibody (mAb) or tyrosine kinase inhibitor (TKI) therapies. However, in time, these mutations appear to develop resistance against the targeted antibodies and TKI treatments. One possible explanation is the activation of pro apoptotic pathways through the AXL–SRC–Akt axis. In this study, we identify AXL as the bypass resistant gene and investigate its role with KRAS and SRC activity. Methods: In this study, we use Dasatinib and SGI-7079 to co-inhibit SRC and AXL respectively. In vitro studies were conducted using four cell lines, and AXL suppression was achieved using siRNA and in CRISPR-Cas9 mediated knockout models. Subsequently, we studied gene-protein expression analysis using Western blot, apoptotic markers using a cytochrome release assay and cytotoxicity using an MTT assay. A549 xenografts were studied for in vivo validation of our proposed hypothesis. Results: The results suggest that dual inhibition of AXL and SRC significantly reversed this resistance, both in in vivo and in vitro studies. Conclusions: Co-inhibition of AXL and SRC synergistically reduced KRAS activity and induced apoptosis in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883777f56e0243ff75313c52129b0f777f7d49f2" target='_blank'>
              Synergistic Inhibition of Drug Resistant KRAS Mutant Non-Small Cell Lung Cancer by Co-Targeting AXL and SRC
              </a>
            </td>
          <td>
            Soumavo Mukherjee, D. Suresh, Ajit P. Zambre, Sairam Yadavilli, Shreya Ghoshdastidar, Anandhi Upendran, R. Kannan
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="MOTIVATION
The emergence of the "telomere-to-telomere" genomics brought the challenge of identifying segmental duplications (SDs) in complete genomes. It further opened a possibility for identifying the differences in SDs across individual human genomes and studying the SD evolution. These newly emerged challenges require algorithms for reconstructing SDs in the most complex genomic regions that evaded all previous attempts to analyze their architecture, such as rapidly-evolving immunoglobulin loci.


RESULTS
We describe the GenomeDecoder algorithm for inferring SDs and apply it to analyzing genomic architectures of various loci in primate genomes. Our analysis revealed that multiple duplications/deletions led to a rapid birth/death of immunoglobulin genes within the human population and large changes in genomic architecture of immunoglobulin loci across primate genomes. Comparison of immunoglobulin loci across primate genomes suggests that they are subjected to diversifying selection.


AVAILABILITY AND IMPLEMENTATION
GenomeDecoder is available at https://github.com/ZhangZhenmiao/GenomeDecoder. The software version and test data used in this paper is uploaded to https://doi.org/10.5281/zenodo.14753844.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e06706d15b6b81ace23df04147d261b78926995" target='_blank'>
              GenomeDecoder: Inferring Segmental Duplica-tions in Highly-Repetitive Genomic Regions.
              </a>
            </td>
          <td>
            Zhenmiao Zhang, Ishaan Gupta, Pavel Pevzner
          </td>
          <td>2025-02-05</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="INTRODUCTION
Approximately one-third of all AML patients have a mutation in the Fms-like tyrosine kinase 3 (FLT3) gene, which is associated with a poor prognosis in these individuals. The 2017 approval of midostaurin, the first FLT3 inhibitor, spurred extensive development of more potent and selective inhibitors with an improved safety profile.


AREAS COVERED
This review analyzes patent inventions for the treatment of AML using FLT3 inhibitors, covering developments from the earliest to the most recent, disclosed in 2024. Our search using the global Espacenet database identified numerous compounds with low nanomolar inhibitory concentrations against FLT3-ITD and FLT3-TKD mutants. These compounds have shown promise in preclinical studies. Co-inhibition strategies and combinatorial therapies to overcome resistance and enhance anti-leukemic efficacy are also discussed.


EXPERT OPINION
Recent patents highlight advances in the field of FLT3 inhibitors with a focus on overcoming resistance, improving selectivity and potency. Future strategies may include third-generation inhibitors such as type III allosteric inhibitors, irreversible inhibitors, or PROTACs. Personalized medicine approaches utilizing genetic profiling to tailor therapies are emphasized. Exploration of novel combination regimens with emerging therapies like CAR T-cell therapy, immune checkpoint inhibitors and small molecules targeting critical AML pathways is ongoing to further enhance anti-leukemic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10fb2e6d07356cabc94588ceb638fa353df881c" target='_blank'>
              Strategies for the treatment of acute myeloid leukemia with FLT3 mutations: a patent review.
              </a>
            </td>
          <td>
            L. Górecki, E. Řezníčková, V. Kryštof, Martina Rezacova, Martina Ceckova, J. Korábečný
          </td>
          <td>2024-12-24</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87eccc46e0a8b0d1964c263841b403eaf8af0f71" target='_blank'>
              CXCR4 promotes tumor stemness maintenance and CDK4/6 inhibitors resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Qiangfeng Shi, Wang Yang, Yiye Ouyang, Yujie Liu, Zijie Cai
          </td>
          <td>2025-01-28</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Human mitochondrial DNA (mtDNA) mutations are causally implicated in maternally inherited mitochondrial respiratory disorders; however, the role of somatic mtDNA mutations in both late-onset chronic diseases and cancer remains less clear. Although recent advances in mtDNA base editing technology have the potential to model and characterize many of these mutations, current editing approaches are complicated by the potential for multiple unintentional edits (bystanders) that are only identifiable through empirical ‘trial and error’, thereby sacrificing valuable time and effort towards suboptimal construct development. Results We developed MitoEdit, a novel tool that incorporates empirical base editor patterns to facilitate identification of optimal target windows and potential bystander edits. MitoEdit allows users to input DNA sequences in a text-based format, specifying the target base position and its desired modification. The program generates a list of candidate target windows with a predicted number of bystander edits and their functional impact, along with flanking nucleotide sequences designed to bind TALE (transcription activator-like effectors) array proteins. In silico evaluations indicate that MitoEdit can predict the majority of bystander edits, thereby reducing the number of constructs that need to be tested empirically. To the best of our knowledge, MitoEdit is the first tool to automate prediction of base edits. Availability and implementation MitoEdit is freely available at Kundu Lab GitHub (https://github.com/Kundu-Lab/mitoedit). Contact Corresponding email: Gang.Wu@stjude.org; Mondira.Kundu@stjude.org Supplementary information Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb09ec051ad08fe864f50953db873f84673f691" target='_blank'>
              MitoEdit: a pipeline for optimizing mtDNA base editing and predicting bystander effects
              </a>
            </td>
          <td>
            Devansh Shah, K. McCastlain, Ti-Cheng Chang, Gang Wu, M. Kundu
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcb2c28999037502cd3294be51e745661f588e3" target='_blank'>
              Functional innovation through new genes as a general evolutionary process.
              </a>
            </td>
          <td>
            Shengqian Xia, Jianhai Chen, Deanna Arsala, J. J. Emerson, Manyuan Long
          </td>
          <td>2025-01-28</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="A major challenge in the stem cell biology field is the ability to produce fully functional cells from induced pluripotent stem cells (iPSCs) that are a valuable resource for cell therapy, drug screening, and disease modelling. Here, we developed a novel inducible CRISPR-mediated activation strategy (iCRISPRa) to drive the expression of multiple endogenous transcription factors (TFs) important for in vitro cell fate and differentiation of iPSCs to haematopoietic progenitor cells. This work has identified a key role for IGFBP2 in developing haematopoietic progenitors. We first identified nine candidate TFs that we predicted to be involved in blood cell emergence during development, then generated tagged gRNAs directed to the transcriptional start site of these TFs that could also be detected during single-cell RNA sequencing (scRNAseq). iCRISPRa activation of these endogenous TFs resulted in a significant expansion of arterial-fated endothelial cells expressing high levels of IGFBP2, and our analysis indicated that IGFBP2 is involved in the remodelling of metabolic activity during in vitro endothelial to haematopoietic transition. As well as providing fundamental new insights into the mechanisms of haematopoietic differentiation, the broader applicability of iCRISPRa provides a valuable tool for studying dynamic processes in development and for recapitulating abnormal phenotypes characterised by ectopic activation of specific endogenous gene expression in a wide range of systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ef294ac361f4462935c56ea7ed2968fcd856ba" target='_blank'>
              A novel human pluripotent stem cell gene activation system identifies IGFBP2 as a mediator in the production of haematopoietic progenitors in vitro
              </a>
            </td>
          <td>
            P. Petazzi, T. Ventura, Francesca Paola Luongo, Heather McClafferty, Alisha May, H. Taylor, Mike Shipston, Nicola Romanò, Lesley M. Forrester, Pablo Menéndez, A. Fidanza
          </td>
          <td>2024-12-23</td>
          <td>eLife</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CRISPR-Cas9, awarded the 2020 Nobel Prize in Chemistry for its discovery by Jennifer Doudna and Emmanuelle Charpentier, has revolutionized gene editing due to its versatility, precision, and efficiency. Unlike previous gene editing tools such as Zinc Finger Proteins (ZFP) and TALENs, CRISPR-Cas9 allows targeted editing of any DNA sequence with minimal off-target effects. This article explores the potential of CRISPR-Cas9 technology to edit telomeresprotective DNA structures at chromosome endsand related genes to address telomere-associated diseases and aging mechanisms. Telomeres, consisting of repetitive DNA sequences and associated proteins, are crucial in protecting chromosomes and regulating cellular aging, with their progressive shortening linked to cancer and other age-related diseases. This article mainly discusses the molecular mechanisms of telomere maintenance, the feasibility of using CRISPR-Cas9 for telomere length control, and its implications for cancer, stem cell research, and telomere-associated diseases. While promising, further research is needed to fully understand the roles of regulatory pathways and non-genetic factors in telomere dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c900abb6d0862046c94ca8ef8e27867352e34ebe" target='_blank'>
              Applications of CRISPR-Cas9 Technology in Editing Telomeres and Their Related Control Genes
              </a>
            </td>
          <td>
            Tianye Liu
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PARP inhibitors (PARPi) have received regulatory approval for the treatment of several tumors, including prostate cancer (PCa), and demonstrate remarkable results in the treatment of castration-resistant prostate cancer (CRPC) patients characterized by defects in homologous recombination repair (HRR) genes. Preclinical studies showed that DNA repair genes (DRG) other than HRR genes may have therapeutic value in the context of PARPi. To this end, we performed multiple CRISPR/Cas9 screens in PCa cell lines using a custom sgRNA library targeting DRG combined with PARPi treatment. We identified LIG1, EME1, and FAAP24 losses as PARPi sensitizers and assessed their frequencies from 3 to 6% among CRPC patients. We showed that concomitant inactivation of LIG1 and PARP induced replication stress and DNA double-strand breaks, ultimately leading to apoptosis. This synthetic lethality (SL) is conserved across multiple tumor types (e.g., lung, breast, and colorectal), and its applicability might be extended to LIG1-functional tumors through a pharmacological combinatorial approach. Importantly, the sensitivity of LIG1-deficient cells to PARPi was confirmed in vivo. Altogether, our results argue for the relevance of determining the status of LIG1, and potentially other non-HRR DRG for CRPC patient stratification and provide evidence to expand their therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6742716d8078e4d21c99bceef70e0cdf901ca76" target='_blank'>
              CRISPR/Cas9 screens identify LIG1 as a sensitizer of PARP inhibitors in castration-resistant prostate cancer.
              </a>
            </td>
          <td>
            Giulia Fracassi, F. Lorenzin, F. Orlando, Ubaldo Gioia, Giacomo D’Amato, A. S. Casaramona, T. Cantore, D. Prandi, F. Santer, Helmut Klocker, Fabrizio d'Adda di Fagagna, J. Mateo, F. Demichelis
          </td>
          <td>2024-12-24</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.


METHODS
In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.   Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.


CONCLUSIONS
Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling in the diagnosis of childhood acute myeloid leukemia: a single center experience.
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish journal of pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Horizontal gene transfer (HGT) is a major driver of bacterial evolution, allowing bacteria to acquire new traits from other organisms. It enables the transfer of advantageous traits, such as antibiotic resistance, which can rapidly spread across bacterial populations, affecting public health. Several studies have estimated how widespread HGT can be and suggested that environmental changes are associated with changes in the rate of HGT. It is often considered that environmental changes lead to an increase in HGT rates due to the adaptive benefits of HGT. However, previous studies have been limited to one or a few species and have often focused on specific genes of interest. Here, we systematically investigate the effect of environmental changes on the rate of HGT in a large number of bacterial genomes along a deep phylogeny. We extend the scope of use of existing state-of -the -art methods for HGT inference to infer the transition into new ecosystems and the gain of pathogenicity. Transitions to new environments are associated with a decrease in the rate of HGT, along with the strengthening of purifying selection. This decrease in HGT rates is consistent regardless of genomic relatedness between the donor and recipient. Our results are robust to the use of different methodological frameworks and have implications for understanding the evolution of bacteria and HGT rates in natural microbial communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509a20de1480698aa85c82ec7a8080e8f597bcfe" target='_blank'>
              Horizontal gene transfer rates decrease during transitions to new ecosystems
              </a>
            </td>
          <td>
            Swastik Mishra, Martin J. Lercher
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2962c853fd82d5be105d8daa9d66daafd430b25e" target='_blank'>
              Elongation factor 2 in cancer: a promising therapeutic target in protein translation
              </a>
            </td>
          <td>
            X. Jia, Chuntian Huang, Fangfang Liu, Zigang Dong, Kangdong Liu
          </td>
          <td>2024-12-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Objectives: The TGF-β signaling pathway is widely acknowledged for its role in various aspects of cancer progression, including cellular invasion, epithelial-mesenchymal transition, and immunosuppression. Immune checkpoint inhibitors (ICIs) and pharmacological agents that target TGF-β offer significant potential as therapeutic options for cancer. However, the specific role of TGF-β in prognostic assessment and treatment strategies for breast cancer (BC) remains unclear. Methods: The Cancer Genome Atlas (TCGA) database was utilized to develop a predictive model incorporating five TGF-β signaling-related genes (TSRGs). The GSE161529 dataset from the Gene Expression Omnibus was employed to conduct single-cell analyses aimed at further elucidating the characteristics of these TSRGs. Additionally, an unsupervised clustering algorithm was applied to categorize BC patients into two distinct groups based on the five TSRGs, with a focus on immune response and overall survival (OS). Further investigations were conducted to explore variations in pharmacotherapy and the tumor microenvironment across different patient cohorts and clusters. Results: The predictive model for BC identified five TSRGs: FUT8, IFNG, ID3, KLF10, and PARD6A. Single-cell analysis revealed that IFNG is predominantly expressed in CD8+ T cells. Consensus clustering effectively categorized BC patients into two distinct clusters, with cluster B demonstrating a longer OS and a more favorable prognosis. Immunological assessments indicated a higher presence of immune checkpoints and immune cells in cluster B, suggesting a greater likelihood of responsiveness to ICIs. Conclusion: The findings of this study highlight the potential of the TGF-β signaling pathway for prognostic classification and the development of personalized treatment strategies for BC patients, thereby enhancing our understanding of its significance in BC prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98f9fd471fec796da5a5b1dc52ea20516e6ffafd" target='_blank'>
              Identification of Molecular Subtypes and Prognostic Features of Breast Cancer Based on TGF-β Signaling-related Genes
              </a>
            </td>
          <td>
            Jia Qu, Meihuan Wang, Yuehua Gao, Hua-Wei Zhang
          </td>
          <td>2025-01-01</td>
          <td>Cancer Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The CRISPR/Cas9 system represents an adaptive immunological defense mechanism observed in bacteria and archaea, which has evolved to protect these organisms from persistent viral and plasmid assaults. This system is comprised of clustered regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated (Cas) proteins. By modifying the simplest type II CRISPR system and introducing a special small guide RNA (sgRNA) and Cas9 endonuclease into the cell, it is possible to cut double-stranded DNA at a specific location and realize knockout or knock-in. CRISPR/Cas9 system has many applications in biological and scientific research fields due to its efficient gene editing function, which makes up for the shortcomings of traditional transgenic technology, such as random integration and genetic instability, and has a broad development prospect. This paper comprehensively discusses the CRISPR/Cas9 technology’s applications in drug discovery and the drug development, such as screening and targeted editing of functional genes, screening and validating drug targets, fabricating animal bioreactors, constructing animal models, and treating genetic diseases, etc. It also discusses the risks and prospects of CRISPR/Cas9 technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248b556c3f98dd8379f35db93ac781813dd8bf7b" target='_blank'>
              Advances in Gene Editing Technology in Biomedicine
              </a>
            </td>
          <td>
            Yuhan Wang
          </td>
          <td>2024-12-31</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a genetic anemia caused by the production of an abnormal adult hemoglobin. The clinical severity is lessened by elevated fetal hemoglobin (HbF) production in adulthood. A promising therapy is the transplantation of autologous, hematopoietic stem/progenitor cells (HSPCs) treated with CRISPR/Cas9 to downregulate the HbF repressor BCL11A via generation of double strand breaks (DSBs) in the +58-kb erythroid-specific enhancer. Here, to further enhance HbF production without increasing the mutagenic load, we targeted both +58-kb and +55-kb BCL11A erythroid-specific enhancers using base editors. We systematically dissected DNA motifs recognized by the key transcriptional activators within these regions and identified the critical nucleotides required for activator binding. Multiplex base editing of these residues was efficient and safe and generated no or little DSBs and genomic rearrangements. We observed substantial HbF reactivation, exceeding the levels achieved using the CRISPR/Cas9 nuclease-based strategy, thus efficiently rescuing the sickling phenotype. Multiplex base editing was efficient in long-term repopulating HSPCs and resulted in potent HbF reactivation in vivo. In summary, these results show that multiplex base editing of BCL11A erythroid-specific enhancers is a safe and potent strategy for treating sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949a30362791a74e8f68a2ef1865b96856027877" target='_blank'>
              Multiplex base editing of BCL11A regulatory elements to treat sickle cell disease
              </a>
            </td>
          <td>
            L. Fontana, P. Martinucci, S. Amistadi, T. Felix, M. Mombled, A. Tachtsidi, G. Corre, A. Chalumeau, G. Hardouin, J. Martin, O. Romano, M. Amendola, P. Antoniou, A. Miccio
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Inotuzumab Ozogamicin (InO) is an antibody-calicheamicin conjugate with striking efficacy in B-cell acute lymphoblastic leukemia (B-ALL). However, there is wide inter-patient variability in treatment response, and the genetic basis of this variation remains largely unknown. Using a genome-wide CRISPR screen, we discovered the loss of DNTT as a primary driver of InO resistance. Mechanistically, DNTT downregulation attenuated InO-induced DNA damage response, cell cycle arrest, and mitochondrial apoptotic priming, ultimately leading to leukemia resistance to InO. Ex vivo leukemia InO sensitivity was highly associated with DNTT expression in ALL blasts, with substantial intra-leukemia heterogeneity as revealed by scRNA sequencing. In B-ALL patients enrolled in the COG trial AALL1621, we observed consistent DNTT downregulation in residual blasts post-InO treatment. The selection of DNTT-low blasts by InO therapy was also recapitulated in vivo using patient-derived xenograft models. Collectively, our data indicate that DNTT is a key regulator of calicheamicin response in leukemia and thus a potential biomarker for individualizing InO therapy in B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0d43dca72c75e6c736c5bab87e5881541ce7f" target='_blank'>
              DNTT-mediated DNA Damage Response Drives Inotuzumab Ozogamicin Resistance in B-cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Carolin S Escherich, Takaya Moriyama, Zhenhua Li, Yu-Chih Hsiao, Wenjian Yang, Yizhen Li, Noemi Reyes, Megan Walker, Amit Budhraja, Sheetal Bhatara, E. Diaz-Flores, Wendy Stock, E. Paietta, M. Konopleva, S. Kornblau, M. Litzow, H. Inaba, Ching-Hon Pui, J. Opferman, M. Loh, Jiyang Yu, Maureen O'Brien, William E Evans, Jun J Yang
          </td>
          <td>2024-12-27</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="How nervous systems coordinate the generation of specific neuron types with gene expression plasticity and how these mechanisms impact cell type evolution is unknown. Here we use Caenorhabditis species to study neuron-type robustness, plasticity and evolution, using VC4 and VC5 cholinergic motoneurons as models. In C. elegans, we found that epigenetic silencing through histone 3 lysine 9 methylation (H3K9me) is necessary to suppress the expression of the serotonin reuptake gene mod-5/Sert and a serotonergic phenotype in these cells. In contrast, we observed that VC4 and VC5 neurons in the Angaria group of species of the Caenorhabditis genus have evolved an intense serotonergic staining. This phenotype is caused by the emergence of a new enhancer in the mod-5/Sert locus, which has been recruited to the ancestral neuron-type gene regulatory network. Enhancer transfer from C. angaria is sufficient to impose a constitutive serotonergic fate in C. elegans. Remarkably, acquiring this new trait modulates egg-laying responses to high levels of exogenous serotonin, which can be found in specific environments. Finally, we discovered that the repression of the serotonergic fate in C. elegans VC4 and VC5 neurons is indeed a plastic trait that can be adjusted in specific environmental growth conditions to elicit egg-laying behaviours similar to those observed in Angaria species. Our work identifies gene regulatory mechanisms that coordinate the generation of robust neuron-type-specific programs with plastic gene expression responses. These findings identify a gene regulatory framework underlying the evolution of neuron-type-specific features and the emergence of novel behaviours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2315a11171225e8baca8060e06a35244519fbc76" target='_blank'>
              Gene regulatory mechanisms underlying evolutionary adaptations of homologous neuronal cell types
              </a>
            </td>
          <td>
            Andrea Millán-Trejo, Carlos Mora-Martínez, Adrián Tarazona-Sánchez, Carla Lloret-Fernández, Rafael Alis, Antonio Jordán, A. Barrios, Nuria Flames
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Hypoxia responses are critical for myriad physiological and pathological processes, such as development, tissue repair, would healing, and tumorigenesis. microRNAs (miRNAs) are a class of small non-coding RNAs that exert their functions by inhibiting the expression of their target genes, and miR-210 is the miRNA universally and most conspicuously upregulated by hypoxia in mammalian systems. For its relationship to hypoxia, miR-210 has been studied extensively, yet no consensus exists on the roles and mechanisms of miR-210 in human physiological processes or diseases, and we know little about genuine miR-210 target genes in humans. Methods To better investigate the functions and mechanisms of human miR-210, therefore, we derived the human miR-210 gene knockout (KO) 293T cell lines using the CRISPR/Cas9 technology. We then examined the cellular phenotypes and gene expression profiles of 293T cells under normoxia and hypoxia conditions. Results and Discussion We found that the loss of miR-210 altered a variety of cellular phenotypes including proliferation and apoptosis. Subsequent global gene expression analyses identified plausible mechanisms underlying these phenotypic changes in 293T cells. In particular, we showed that miR-210 might target the expression of BNIP3L as a potential mechanism to suppress apoptosis. Surprisingly, the mRNA levels of most previously reported miR-210 target genes were not induced upon miR-210 KO, suggesting a need to reexamining and studying human miR-210 functions directly and comprehensively. Thus, our work established a human cellular system and opportunity to unravel the complexity of the regulatory networks by miR-210.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de9fca0fc73bea5e4d2b5e6bb3d58b2f4b8c51b" target='_blank'>
              miR-210 loss leads to widespread phenotypic and gene expression changes in human 293T cells
              </a>
            </td>
          <td>
            Xiaoxiao Zhang, Zhen Meng, Chengyong Yang, Chenghao Wang, Kexin Zhang, Anxin Shi, Jingjing Guo, Yong Feng, Yan Zeng
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic functions have evolved over long timescales and can be encoded by multiple genes dispersed in different locations in genomes, and although contemporary molecular biology enables control over single genes, more complex genetic functions remain challenging. Here, we study the restructuring and mobilization of a complex genetic function encoded by 10 genes, originally expressed from four operons and two loci on the Escherichia coli genome. We observe subtle phenotypic differences and reduced fitness when expressed from episomal DNA and demonstrate that mutations in the transcriptional machinery are necessary for successful implementation in different bacteria. The work provides new approaches for advanced genome editing and constitutes a first step toward modularization and genome-level engineering of complex genetic functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9538aea71ef9ff7e8ced7da977249ea2ae6daecf" target='_blank'>
              Restructuring a Complex Genetic Function on Episomal Vectors in Escherichia coli.
              </a>
            </td>
          <td>
            A. Bertelsen, Anja K. Ehrmann, Carolyn N. Bayer, T. Batth, J. Olsen, Morten H. H. Nørholm
          </td>
          <td>2024-12-19</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The mitochondrial genome (mtDNA) is an important source of inherited extranuclear variation. Clonal increases in mtDNA mutation heteroplasmy have been implicated in aging and disease, although the impact of this shift on cell function is challenging to assess. Reprogramming to pluripotency affects mtDNA mutation heteroplasmy. We reprogrammed three human fibroblast lines with known heteroplasmy for deleterious mtDNA point or deletion mutations. Quantification of mutation heteroplasmy in the resulting 76 induced pluripotent stem cell (iPSC) clones yielded a bimodal distribution, creating three sets of clones with high levels or absent mutation heteroplasmy with matched nuclear genomes. iPSC clones with elevated deletion mutation heteroplasmy show altered growth dynamics, which persist in iPSC-derived progenitor cells. We identify transcriptomic and metabolic shifts consistent with increased investment in neutral lipid synthesis as well as increased epigenetic age in high mtDNA deletion mutation iPSC, consistent with changes occurring in cellular aging. Together, these data demonstrate that high mtDNA mutation heteroplasmy induces changes occurring in cellular aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07003adbe30c731047a58b8ae874dca402e30f77" target='_blank'>
              Increased mitochondrial mutation heteroplasmy induces aging phenotypes in pluripotent stem cells and their differentiated progeny.
              </a>
            </td>
          <td>
            A. Vandiver, Alejandro Torres, Amberly Sanden, Thang L Nguyen, Jasmine Gasilla, Mary T Doan, Vahan Martirosian, Austin N. Hoang, J. Wanagat, Michael A Teitell
          </td>
          <td>2024-12-16</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The 5,000 to 8,000 monogenic diseases are inherited disorders leading to mutations in a single gene. These diseases usually appear in childhood and sometimes lead to morbidity or premature death. Although treatments for such diseases exist, gene therapy is considered an effective and targeted method and has been used in clinics for monogenic diseases since 1989. Monogenic diseases are good candidates for novel therapeutic technologies like gene editing approaches to repair gene mutations. Clustered regularly interspaced short palindromic repeats (CRISPR)-based systems, the pioneer and effective gene editing tool, are utilized for ex vivo and in vivo treatment of monogenic diseases. The current review provides an overview of recent therapeutic applications of CRISPR-based gene editing in monogenic diseases in in vivo and ex vivo models. Furthermore, this review consolidates strategies aimed at providing new treatment options with gene therapy, thereby serving as a valuable reference for advancing the treatment landscape for patients with monogenic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0a8862e584d090d20f1389daab0e1853e5ef72" target='_blank'>
              CRISPR/Cas System: A Powerful Strategy to Improve Monogenic Human Diseases as Therapeutic Delivery; Current Applications and Challenges.
              </a>
            </td>
          <td>
            Fatemeh Saberi, Zeinab Yousefi-Najafabadi, F. Shams, Zeinab Dehghan, Sepideh Ahmadi, Tayyebeh Pilehchi, Effat Noori, Zeinab Esmaeelzadeh, Maryam Bazgiri, Rezvan Mohammadi, Farzaneh Khani, M. Sameni, Parisa Moradbeigi, G. Kardar, Mohammad Salehi, Yong Teng, V. Jajarmi
          </td>
          <td>2025-01-07</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Originally derived from the immune system of bacteria, CRISPR-Cas9 technology has rapidly developed into a revolutionary gene editing tool that is widely used in molecular biology, drug development, and gene therapy. This technology not only provides a new treatment approach for genetic diseases such as sickle cell anemia and Duchenne muscular dystrophy, but also brings breakthrough progress in cancer treatment by enhancing chimeric antigen receptor T cell (CAR-T) therapy. CRISPR-Cas9 has also shown great potential in the drug discovery stage, especially in gene screening, development of disease models, and drug resistance research. However, with the rapid development of CRISPR technology, the scientific community still needs to solve challenges such as potential off-target effects in gene editing, inefficient delivery systems, and long-term safety. In addition, the ethical issues of gene editing have also sparked widespread discussion, especially how to ensure fair access when treating genetic diseases. Despite many challenges, CRISPR-Cas9 technology still shows great application prospects in the future of drug development and gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ad9a53620cc6ea8c99c67cb73104e6fa93ba3d" target='_blank'>
              The Pharmaceutical Applications of CRISPR-Cas9 and Its Development Trends
              </a>
            </td>
          <td>
            Yu Sang
          </td>
          <td>2024-12-31</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="CRISPR/Cas9 technologies provide unique capabilities for modeling disease and understanding gene-to-phenotype connections. In cultured cells, chemical-mediated control of Cas9 activity can limit off-target effects and enable mechanistic study of essential genes. However, widely-used Tet-On systems often show “leaky” Cas9 expression, leading to unintended edits, as well as weak activity upon induction. Leakiness can be distinctly problematic in the context of Cas9 nuclease activity, which may result in cumulative DNA damage and degradation of the target cell genome over time. To overcome these deficiencies, we established transgenic platforms that minimize Cas9 functionality in the off-state along with maximized and uncompromised on-state gene editing efficiency. By combining conditional destabilization and inhibition of Cas9, we developed an all-in-one (one or multiple guide RNAs and Cas9) ultra-tight, Tet-inducible system with exceptional dynamic range (on vs. off-state) across various cell lines and targets. As an alternative to Tet-mediated induction, we created a branaplam-regulated splice switch module for low-baseline and robust Cas9 activity control. Lastly, for circumstances where DNA damage needs to be avoided, we constructed a dual-control, Tet-inducible CRISPRi module for tight and potent transcriptional silencing. This upgraded suite of inducible CRISPR systems has broad applications for numerous cell types and experimental conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67f861c2ce0a102cbb4cb3e559a5affcf75ca92" target='_blank'>
              Chemically-inducible CRISPR/Cas9 circuits for ultra-high dynamic range gene perturbation
              </a>
            </td>
          <td>
            Rajini Srinivasan, Tao Sun, Azalia Sandles, Diana Wu, Liang Wang, A. Heidersbach, Clark Ho, Shiqi Xie, Andrew Ng, Benjamin Haley
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
 Colorectal cancer (CRC) is a prevalent malignant tumor characterized by a high mortality rate, with significant challenges persisting in the identification and management of its metastatic stage. The role of checkpoint kinase 1 (CHEK1), a cell cycle checkpoint kinase, in CRC has not been fully clarified. We hypothesize that the upregulation of CHEK1 may enhance the proliferation of CRC cells, indicating its potential as a novel therapeutic target for CRC therapy.
 AIM
 To investigate the expression and function of CHEK1 in CRC, this study utilizes single-cell RNA sequencing and tissue microarray data.
 METHODS
 Single-cell RNA sequencing technology was employed to analyze CRC cells from the GSE144735 dataset, and immunohistochemistry was conducted to confirm the expression of CHEK1 in CRC and adjacent tissues. We also integrated mRNA expression data from multiple public databases to assess global CHEK1 expression in CRC. Molecular docking experiments were performed to explore the interaction between CHEK1 and the potential drug nitidine chloride (NC), as well as to investigate the influence of CHEK1 on CRC cell proliferation.
 RESULTS
 We found comparatively elevated CHEK1 expression in the malignant epithelial cells of CRC, with marked upregulation of its mRNA levels in CRC tissues. Immunohistochemical analysis further confirmed the high expression of CHEK1 in CRC tissues, and the receiver operating characteristic curve demonstrated high accuracy (area under the curve = 0.964) for CHEK1 as a biomarker. Analysis of global datasets indicated a statistically significant overexpression of CHEK1 in CRC (standard mean difference = 1.81, P < 0.01), with summary receiver operating characteristic analysis yielding sensitivity and specificity values of 0.83 and 0.88, respectively. Molecular docking studies indicated that NC specifically targeted CHEK1, while clustered regularly interspaced short palindromic repeats knockout experiments demonstrated that CHEK1 promoted CRC cell proliferation.
 CONCLUSION
 Upregulation of CHEK1 promotes CRC cell proliferation. However, the dataset's diversity is limited, requiring further investigation into its specific mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a566889c9fc5fe520488439e6857e1a25ba38f4b" target='_blank'>
              Checkpoint kinase 1 in colorectal cancer: Upregulation of expression and promotion of cell proliferation
              </a>
            </td>
          <td>
            Yu-yan Pang, Zu-Yuan Chen, Da-Tong Zeng, Dong-Ming Li, Qi Li, Wan-Ying Huang, Bin Li, Jia-Yuan Luo, Bang-Teng Chi, Qiu Huang, Zhen-Bo Feng, Rong-Quan He
          </td>
          <td>2025-03-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Fragment based novel drug identification and its validation through use of molecular dynamics and simulations.Comparing primary microcephaly genes with glioblastoma expression profiles reveals potential oncogenes, with proteins that support growth and survival in neural stem/progenitor cells likely retaining critical roles in glioblastoma. Identifying such proteins in familial and congenital microcephalic disorders offers promising targets for brain tumor therapy. Among these, KIF11, a kinesin motor protein (KSP), stands out as a significant oncogene. Expression analyses across various cancer types, including glioblastoma, demonstrate its overexpression in brain tumor patients. Using a targeted fragment-based drug discovery approach, we explored alternative small molecule inhibitors for KIF11. Existing drugs, such as ispinesib, are limited by side effects and multidrug resistance. Through molecular docking and simulations, we identified three candidate drug fragments. Further analysis confirmed that Mol-121026 exhibits a more stable interaction with KIF11 compared to ispinesib. Detailed analyses indicate that Mol-121026 binds to the same active site as the reference drug, effectively inhibiting KIF11's mechano-chemical activity. Importantly, Mol-121026, a derivative of 3-phenyl-1H-pyrazol-5-carboxylic acid, offers a promising alternative due to its lower molecular complexity, ability to target allosteric sites, and potential for optimization into a potent and effective drug candidate. Our findings identified Mol-121026 as a top candidate with a docking score of -10.2 kcal/mol and MM/GBSA binding energy of -19.10 kcal/mol. Molecular dynamics simulations revealed stable interactions with key residues GLU116 and GLU118, supporting its potential as a promising KIF11 inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9b86eac4f8c7f39109905d3ffc984c8589c2d9" target='_blank'>
              Fragment-Based Drug Discovery of KIF11 Inhibitors for Glioblastoma Treatment: Molecular Insights and Therapeutic Potential.
              </a>
            </td>
          <td>
            Q. Naseer, Xuexian Cao, Deng Yimai, Muhammad Ajmal Khan, Shengxia Chen
          </td>
          <td>2025-01-22</td>
          <td>Drug research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f23f58ff8f0c69ee80ac960c4ff630df39788c5" target='_blank'>
              LGR5: An emerging therapeutic target for cancer metastasis and chemotherapy resistance
              </a>
            </td>
          <td>
            W. Wang, N. Lokman, Simon C Barry, M. Oehler, C. Ricciardelli
          </td>
          <td>2025-01-17</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mammalian genomes contain millions of regulatory elements that control the complex patterns of gene expression. Previously, The ENCODE consortium mapped biochemical signals across many cell types and tissues and integrated these data to develop a Registry of 0.9 million human and 300 thousand mouse candidate cis-Regulatory Elements (cCREs) annotated with potential functions1. We have expanded the Registry to include 2.35 million human and 927 thousand mouse cCREs, leveraging new ENCODE datasets and enhanced computational methods. This expanded Registry covers hundreds of unique cell and tissue types, providing a comprehensive understanding of gene regulation. Functional characterization data from assays like STARR-seq, MPRA, CRISPR perturbation, and transgenic mouse assays now cover over 90% of human cCREs, revealing complex regulatory functions. We identified thousands of novel silencer cCREs and demonstrated their dual enhancer/silencer roles in different cellular contexts. Integrating the Registry with other ENCODE annotations facilitates genetic variation interpretation and trait-associated gene identification, exemplified by discovering KLF1 as a novel causal gene for red blood cell traits. This expanded Registry is a valuable resource for studying the regulatory genome and its impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99941d31f540b8743cd6f7ea4d2eb9891068be3e" target='_blank'>
              An Expanded Registry of Candidate cis-Regulatory Elements for Studying Transcriptional Regulation
              </a>
            </td>
          <td>
            Jill E. Moore, H. Pratt, K. Fan, Nishigandha Phalke, Jonathan Fisher, Shaimae I. Elhajjajy, Gregory Andrews, Mingshi Gao, Nicole Shedd, Yu Fu, Matthew C Lacadie, Jair Meza, Mohit Ganna, Eva Choudhury, Ross Swofford, Nina P. Farrell, Anusri Pampari, Vivekanandan Ramalingam, Fairlie Reese, Beatrice Borsari, Michelle Yu, Eve Wattenberg, M. Ruiz-Romero, Milad Razavi-Mohseni, Jinrui Xu, Timur Galeev, Michael A. Beer, R. Guigó, Mark B. Gerstein, Jesse M. Engreitz, Mats Ljungman, Timothy E. Reddy, M. Snyder, Charles B. Epstein, Elizabeth Gaskell, Bradley E. Bernstein, D. Dickel, Axel Visel, Len A. Pennacchio, A. Mortazavi, A. Kundaje, Zhiping Weng
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="The p53 protein has attracted huge research interest over several decades due to its role as one of the most important tumor suppressors in mammals, which orchestrates a synchronous response from normal cells in the body to various forms of stress. The diverse cellular activities of the p53 protein are regulated mainly via its post-translational modifications (PTMs). PTMs affect p53 on several levels: at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes, at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes; at the level of proteolysis in the absence of stress; and on the contrary, at the level of augmented protein stability in response to stress signals. Disruptions in these regulatory mechanisms can lead to deviations from normal cellular function, boosting tumor initiation and progression. Conversely, targeted interventions in these pathways could prove beneficial for the development of antitumor therapies. Advancing our understanding of p53 modifiers and the proteins involved in its regulation equips researchers with an expanded toolkit for studying cellular processes and for developing biologically active molecules that influence p53-mediated responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3fe1959718caec99cd867c8f3aa0f229e81022" target='_blank'>
              p53: The Multifaceted Roles of Covalent Modifications in Cancer
              </a>
            </td>
          <td>
            Tatiana A. Grigoreva, Angelina A. Romanova, V. Tribulovich, Nikolay B. Pestov, Ruslan A Oganov, Diana K. Kovaleva, T. Korneenko, N. A. Barlev
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and highly lethal malignant tumor. Due to the difficulty in early diagnosis and limited treatment options, the 5-year survival rate is less than 10%. Traditional treatments such as surgery, chemotherapy, and radiotherapy have limited effects on PDAC, which is mainly attributed to its unique tumor microenvironment (TME), common gene mutations (such as KRAS, and TP53), and immunosuppressive properties. In recent years, with the advancement of molecular biology and genomics technologies, the molecular mechanisms of PDAC have been studied more deeply, revealing the gene mutations and regulatory networks associated with its pathogenesis. In particular, KRAS mutations have become an important research direction for targeted therapy. However, the effect of immunotherapy in PDAC is limited by the immune escape characteristics of TME. This article systematically summarizes the key gene mutations in PDAC and their regulatory mechanisms, including epigenetic regulation, microRNA, histone modification and other influencing factors. CRISPR-Cas9 shows great potential in correcting gene mutations and reshaping TME. In addition, combining immunotherapy with targeted drug delivery (such as nanotechnology) can improve the precision of treatment and reduce side effects. These advances provide a comprehensive reference for future personalized and effective treatment options for PDAC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1538e4df893f0e562277858358833ccdc8a4f65" target='_blank'>
              Genetic Mutation and Regulation in Pancreatic Ductal Adenocarcinoma Cancer: Developing Relevant Molecular Biomarkers
              </a>
            </td>
          <td>
            Xiaomi Ai
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857a45248fbd262ae3ffedba1c112578d12a79d8" target='_blank'>
              SEED-Selection enables high-efficiency enrichment of primary T cells edited at multiple loci.
              </a>
            </td>
          <td>
            Christopher R. Chang, Vivasvan S. Vykunta, Jae Hyun J Lee, Ke Li, Clara Kochendoerfer, J. J. Muldoon, Charlotte H. Wang, Thomas Mazumder, Yang Sun, Daniel Goodman, William A. Nyberg, Chang Liu, Vincent Allain, Allison Rothrock, Chun J Ye, Alex Marson, B. Shy, Justin Eyquem
          </td>
          <td>2025-02-05</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="INTRODUCTION
Leukemia is typically categorized into myeloid leukemia and lymphoblastic leukemia based on the origins of leukemic cell. Myeloid leukemia is a group of clonal malignancies characterized by the presence of increased immature myeloid cells in both bone marrow and peripheral blood. Of note, the aberrant expression of specific proteins or the generation of fusion proteins due to chromosomal abnormalities are well established drivers in various forms of myeloid leukemia. Therefore, these oncoproteins represent promising targets for drug development.


AREAS COVERED
In this review, we comprehensively discussed the pathogenesis of typical leukemia oncoproteins and current landscape of small molecule drugs targeting these oncogenic proteins. Additionally, we elucidated novel strategies, including proteolysis-targeting chimeras (PROTACs), hyperthermia and genomic editing, which specifically degrade oncogenic proteins in myeloid malignancies.


EXPERT OPINION
Although small molecule drugs have significantly improved the prognosis of oncoprotein driven myeloid leukemia patients, drug resistance due to the mutations in oncoproteins is still a great challenge in clinic. New approaches such as PROTACs, hyperthermia and genomic editing are considered promising approaches for the treatment of oncoprotein-driven leukemia, especially for drug resistant mutants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea5fe5c5cfd460bcc9a1a8e4861d057f9becb9a" target='_blank'>
              Therapeutic approaches targeting oncogenic proteins in myeloid leukemia: challenges and perspectives.
              </a>
            </td>
          <td>
            Xing Yi Yan, Yuan-yuan Kang, Ze Yan Zhang, Ping Huang, Chang Yang, Hua Naranmandura
          </td>
          <td>2024-12-16</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Simple Summary KTMA2 gene rearrangements (KMT2Ar) frequently occur in acute leukemias (ALs) and portray an aggressive clinical phenotype, a high relapse risk, and a dismal prognosis. In these subsets of AL, the binding of the protein Menin (MEN1) to KMT2A fusion proteins drives the activation of a transcriptional pathway, altering the transcription of various genomic entities, particularly HOX/MEIS1 genes, which are crucial in the development of the leukemic phenotype. In this regard, KMT2Ar is prevalent in acute myeloid leukemia (AML), acute lymphoblastic leukemia, and some rarer AL subtypes. In addition, nucleophosmin-1 (NPM1m) mutations induce a gene expression profile similar to other KMT2Ar ALs, specifically in NPM1m AML. Therefore, inhibiting the interaction between MEN1 and KMT2Ar and NPM1m proteins induces apoptosis and promotes the differentiation of leukemic cells. In this regard, some targeted agents, such as MEN1 inhibitors, have been developed and are in an advanced phase of clinical investigation with encouraging clinical results. They have recently received their first formal recognition from the U.S. and European drug regulatory agencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f6d0074705ca17dc267107710154dd96466e76" target='_blank'>
              Menin Inhibitors: New Targeted Therapies for Specific Genetic Subtypes of Difficult-to-Treat Acute Leukemias
              </a>
            </td>
          <td>
            P. Niscola, Valentina Gianfelici, Marco Giovannini, D. Piccioni, Carla Mazzone, P. Fabritiis
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="In cancer cells, survival genes contribute to uncontrolled growth and the survival of malignant cells, leading to tumor progression. Neurons are post-mitotic cells, fully differentiated and non-dividing after neurogenesis and survival genes are essential for cellular longevity and proper functioning of the nervous system. This review explores recent research findings regarding the role of survival genes, particularly DX2, in degenerative neuronal tissue cells and cancer cells. Survival gene DX2, an exon 2-deleted splice variant of AIMP2 (aminoacyl-tRNA synthetase-interacting multi-functional protein 2), was found to be overexpressed in various cancer types. The potential of DX2 inhibitors as an anti-cancer drug arises from its unique ability to interact with various oncoproteins, such as KRAS and HSP70. Meanwhile, AIMP2 has been reported as a multifunctional cell death-inducing gene, and survival gene DX2 directly and indirectly inhibits AIMP2-induced cell death. DX2 plays multifaceted survival roles in degenerating neurons via various signaling pathways, including PARP 1, TRAF2, and p53 pathways. It is noteworthy that genes that were previously classified as oncogenes, such as AKT and XBP1, are now being considered as curative transgenes for targeting neurodegenerative diseases. A strategic direction for clinical application of survival genes in neurodegenerative disease and in cancer is justified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca4de34e4eb65a10b19b155db29e45032532ad3d" target='_blank'>
              The Dual Role of Survival Genes in Neurons and Cancer Cells: a Strategic Clinical Application of DX2 in Neurodegenerative Diseases and Cancer
              </a>
            </td>
          <td>
            Kyunghwa Baek
          </td>
          <td>2024-12-23</td>
          <td>Biomolecules & Therapeutics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Insertions of the transposable element IS5 into its target sites in response to stressful environmental conditions, DNA structures, and DNA-binding proteins are well studied, but how the genomic contexts near IS5′s native loci impact its transpositions is largely unknown. Here, by examining the roles of all 11 copies of IS5 within the genome of E. coli strain BW25113 in transposition, we reveal that the most significant copy of IS5 is one nested within and oriented in the same direction as the nmpC gene, while two other copies of IS5 harboring point mutations are hardly transposed. Transposition activity is heavily reliant on the upstream nmpC promoter that drives IS5 transposase gene ins5A, with more transpositions resulting from greater promoter activity. The IS5 element at nmpC but not at other loci transcribed detectable amounts of ins5A mRNA. By increasing expression of the ins5CB operon harbored in IS5, we demonstrate that Ins5B and Ins5C appear to exert a stimulatory role in IS5 transposition, suggesting that the downstream genomic regions near the native loci are involved in overall IS5 transposition as well. Using a strain that carries IS5 only at the nmpC locus, we confirm that IS5 primarily uses a copy/paste mechanism for transposition, although we cannot rule out the cut/paste mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/988cbcb664e4b92fe5ef722bec63c56b769819f3" target='_blank'>
              Investigating How Genomic Contexts Impact IS5 Transposition Within the Escherichia coli Genome
              </a>
            </td>
          <td>
            Jonathan Onstead, Zhongge Zhang, Jialu Huo, Jack W. Ord, Sofia Smith, Milton H. Saier
          </td>
          <td>2024-12-01</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4211b81d0b19e4c7f2e0cfa7c734b541dc3fa943" target='_blank'>
              Epigenetics in the modern era of crop improvements.
              </a>
            </td>
          <td>
            Yan Xue, Xiaofeng Cao, Xiangsong Chen, Xian Deng, Xing Wang Deng, Yong Ding, Aiwu Dong, Cheng-Guo Duan, Xiaofeng Fang, L. Gong, Zhizhong Gong, Xiaofeng Gu, Chongsheng He, Hang He, Shengbo He, Xin-Jian He, Yan He, Yuehui He, Guifang Jia, Danhua Jiang, Jianjun Jiang, Jinsheng Lai, Zhaobo Lang, Chenlong Li, Qing Li, Xingwang Li, Bao Liu, Bing Liu, Xiao Luo, Yijun Qi, Weiqiang Qian, Guodong Ren, Qingxin Song, Xianwei Song, Zhixi Tian, Jia-Wei Wang, Yuan Wang, Liang Wu, Zhe Wu, Rui Xia, Jun Xiao, Lin Xu, Zhengyi Xu, Wenhao Yan, Hongchun Yang, J. Zhai, Yijing Zhang, Yusheng Zhao, Xuehua Zhong, Dao-Xiu Zhou, Ming Zhou, Yue Zhou, Bo Zhu, Jian-Kang Zhu, Qikun Liu
          </td>
          <td>2025-01-08</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Diffuse midline gliomas (DMGs) with histone H3K27M mutations represent a devastating paediatric brain cancer characterized by abysmal prognosis and limited treatment options. The only approved treatment is radiotherapy (RT), but most of the tumours relapse with fatal consequences. In this study, we sought to investigate whether irradiation leads to senescence induction and explore the efficacy of senolytics against DMG. Methods We have characterised the senescent phenotype of five genetically heterogeneous H3K27M-altered human DMG cell lines, combining cellular and/or molecular approaches. The sensitivity of senescent cells to Bcl-xL inhibition has been demonstrated in dose/response curves in vitro and in a PDX model of DMG. Results Here, we show that ionizing radiation induces senescence and SASP responses in both TP53 mutant and wild-type H3K27M-altered human DMG cell lines. We identify Navitoclax as a potent senolytic agent that selectively targets senescent DMG cells into apoptosis by inhibiting Bcl-xL. Related compounds, such as a proteolysis-targeting chimera (PROTAC)-mediated Bcl-xL degradation and a galacto-conjugated form of Navitoclax also show an effective senolytic activity in senescent cancer cells. Finally, we show that a combination therapy of irradiation and Navitoclax results in reduced tumor burden and increased mouse survival in an orthotopic xenograft DMG model. Conclusion These results offer a rationale for further clinical development of senolytic therapies as part of multimodal treatment approaches for DMG patients/ Key Points Ionising irradiation induces senescence in human DMG cells independently of the p53 status. Bcl-xL inhibition results in apoptosis of human DMG senescent cells in synergy with irradiation. Combination of irradiation and BcL-xL inhibition reduces tumourigenesis in a PDX model of DMG. Importance of the Study H3K27M-altered DMG are devastating paediatric tumours with an abysmal prognosis. The only approved treatment is radiotherapy but this is palliative and tumours almost always relapse with fatal consequences for the patients. In this study, we show that radiotherapy results in senescence induction in five genomically heterogeneous human DMG cell lines. We identify that drugs targeting the anti-apoptotic protein Bcl-xL show a strong senolytic activity in conjunction with radiotherapy both in vitro in DMG cells and in vivo in a PDX model of H3K27M-altered DMG. Treatment with Bcl-xL inhibitor Navitoclax, or related compounds targeting Bcl-xL protein degradation or containing a galactose conjugated form of Navitoclax results in DMG cancer cell apoptosis. As several of these inhibitors are currently being tested in ongoing clinical trials against other diseases, our data support the use of Bcl-xL inhibition mediated senolytics as an adjuvant therapy to radiotherapy to potentially improve outcomes in this challenging disease setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b580291efecff13cf295c14d6b2926c855610e2" target='_blank'>
              Targeted inhibition of Bcl-xL following radiation reduces tumourigenesis in preclinical models of H3K27M-altered diffuse midline glioma
              </a>
            </td>
          <td>
            Ashley Vardon, S. Haston, Romain Guiho, Diana Carvahlo, Rebecca Carter, J. Boult, Daniel Gharai, Diwakar Santhakumar, Ielyaas Cloete, James Gray, John Apps, O. Ogunbiyi, Daohong Zhou, Guangrong Zheng, Ramón Martínez-Máñez, Mark Lythgoe, Laura Donovan, A. Carcaboso, Owen Williams, Jesus Gil, Thomas Jacques, Jamie Dean, D. Michod, Chris Jones, Darren Hargrave, J. Martínez-Barbera
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba325d5f7e42c0fb0d9165042321e812452f502" target='_blank'>
              Cas12f1 gene drives propagate efficiently in herpesviruses and induce minimal resistance
              </a>
            </td>
          <td>
            Zhuangjie Lin, Qiaorui Yao, Keyuan Lai, Kehua Jiao, Xianying Zeng, Guanxiong Lei, Tongwen Zhang, Hongsheng Dai
          </td>
          <td>2024-12-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="TRIP13 is a member of the large AAA+ ATPase protein superfamily that plays a crucial role in the precise segregation of chromosomes during mitosis. The abnormal function of TRIP13 has diverse functions, including mitotic processes, DNA repair pathways, and spindle assembly checkpoints, which may contribute to chromosomal instability (CIN). Emerging evidence suggests that the overexpression of TRIP13, observed in many cancers, plays a significant role in drug resistance, autophagy, and immune invasion. Recently, significant advances have been made in identifying TRIP13-associated signaling pathways that have been implicated in cancer progression. Several small molecules that specifically inhibit TRIP13 function and reduce cancer cell growth have been developed. Combination treatments, including TRIP13 inhibitors and other anticancer drugs, have shown promising results. While these findings are promising, TRIP13 inhibitors are awaiting clinical trials. This review discusses recent progress in understanding the oncogenic function of TRIP13 and its possible therapeutic targets, which could be exploited as an attractive option for cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31bad330d341da7462c6f7e15404b9c7c4e5e858" target='_blank'>
              Targeting Thyroid Hormone Receptor Interacting Protein (TRIP13) for Cancer Therapy: A Promising Approach
              </a>
            </td>
          <td>
            Surya P. Singh, Krishnendu Goswami, Gopal Pathuri, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-01-06</td>
          <td>DNA</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, but the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a hypermutation phenomenon at transcription start sites in the human germline, spanning several hundred base pairs in both directions. We link this TSS mutational hotspot to divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, revealing a recombination-independent mechanism distinct from known processes. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants often filtered out in de novo mutation calls, indicating that the hotspot arises during early embryogenesis. Mutational signature analysis reinforces these findings and implicates alternative non-homologous end joining and maternal mutation clusters. Our study provides the first detailed description of a germline TSS mutation hotspot, with broad evolutionary and biomedical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3da1619adeabbe4ded6c11c45076e2a69d9a307" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fuelled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, V. Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The diverse radiation types in medical treatments and the natural environment elicit complex biological effects on both cancerous and non-cancerous tissues. Radiation therapy (RT) induces oncological responses, from molecular to phenotypic alterations, while simultaneously exerting toxic effects on healthy tissue. N6-methyladenosine (m6A), a prevalent modification on coding and non-coding RNAs, is a key epigenetic mark established by a set of evolutionarily conserved enzymes. The interplay between m6A modification and radiobiology of cancerous and non-cancerous tissues merits in-depth investigation. This review summarizes the roles of m6A in the biological effects induced by ionizing radiation and ultraviolet (UV) radiation. It begins with an overview of m6A modification and its detection methods, followed by a detailed examination of how m6A dynamically regulates the sensitivity of cancerous tissues to RT, the injury response in non-cancerous tissues, and the toxicological effects of UV exposure. Notably, this review underscores the importance of novel regulatory mechanisms of m6A and their potential clinical applications in identifying epigenetically modulated radiation-associated biomarkers for cancer therapy and estimation of radiation dosages. In conclusion, enzyme-mediated m6A-modification triggers alterations in target gene expression by affecting the metabolism of the modified RNAs, thus modulating progression and radiosensitivity in cancerous tissues, as well as radiation effects on normal tissues. Several promising avenues for future research are further discussed. This review highlights the importance of m6A modification in the context of radiation biology. Targeting epi-transcriptomic molecules might potentially provide a novel strategy for enhancing the radiosensitivity of cancerous tissues and mitigating radiation-induced injury to normal tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b51d3e0ec15a1766878bdbaa0b5458c9eed89844" target='_blank'>
              The interplay between RNA m6A modification and radiation biology of cancerous and non-cancerous tissues: a narrative review
              </a>
            </td>
          <td>
            Yajia Cheng, Yue Shang, Shuqin Zhang, Saijun Fan
          </td>
          <td>2024-12-15</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The heterogeneity of the hematopoietic system was largely veiled by traditional bulk sequencing methods, which measure the averaged signals from mixed cellular populations. In contrast, single-cell sequencing has enabled the direct measurement of individual signals from each cell, significantly enhancing our ability to unveil such heterogeneity. Building on these advances, numerous single-cell multi-omics techniques have been developed into high-throughput, routinely accessible platforms, delineating the precise relationships among the different layers of the central dogma in molecular biology. These technologies have uncovered the intricate landscape of genetic clonality and transcriptional heterogeneity in both normal and malignant hematopoietic systems, highlighting their roles in differentiation, disease progression, and therapy resistance. This review aims to provide a brief overview of the principles of single-cell technologies, their historical development, and a subset of ever-expanding multi-omics tools, emphasizing the specific research questions that inspired their creation. Amidst the evolving landscape of single-cell multi-omics technologies, our main objective is to guide investigators in selecting the most suitable platforms for their research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Alternative NTRK1/TrkA splicing resulting in TrkAIII expression, originally discovered in advanced-stage metastatic neuroblastomas, is also pronounced in prostate, medullary thyroid, glioblastoma multiforme, MCPyV-positive Merkel cell, cutaneous malignant melanoma, and pituitary neuroendocrine tumor subsets. In tumor models, TrkAIII exhibits actionable oncogenic activity equivalent to the TrkT3-fused oncogene, and in tumor cell lines, alternative TrkAIII splicing is promoted by hypoxia, nutrient deprivation, endoplasmic reticulum stress, and SV40 large T antigen, implicating tumor microenvironmental conditions and oncogenic polyoma viruses in tumor-associated TrkAIII expression. Collectively, these observations characterize TrkAIII as a potentially frequent, actionable oncogenic alternative to TrkA gene fusion in different tumor types. Currently, therapeutic approval for efficacious Trk inhibitors is restricted to Trk-fused gene positive tumors and not for tumors potentially driven by TrkAIII. Methods: With the therapeutically relevant aim of improving the identification of tumors potentially driven by TrkAIII, we have developed a TaqMan-based qRT-PCR assay for evaluating TrkAIII expression in tumor cDNAs. Results: This assay, validated using gel-purified fs-TrkA and TrkAIII cDNAs alone and in complex cDNA mixtures, employs primers and probes designed from fs-TrkA and TrkAIII sequences, with specificity provided by a TaqMan probe spanning the TrkAIII exon 5–8 splice junction. It is highly efficient, reproducible, and specific and can detect as few as 10 TrkAIII copies in complex RNAs extracted from either fresh or FFPE tumor tissues. Conclusions: Inclusion of this assay into precision oncology algorithms, when paired with fs-TrkA qRT-PCR and TrkA immune histochemistry, will make it easier to identify patients with therapy-resistant, advanced-stage metastatic Trk-fused gene-negative tumors potentially driven by TrkAIII, for whom approval of third-line effective Trk inhibitors could be extended.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7006115fc6e45a823e5b912453b2538a31bcd05" target='_blank'>
              A TaqMan-Based qRT-PCR Assay for Accurate Evaluation of the Oncogenic TrkAIII Splice Variant in Tumor cDNAs
              </a>
            </td>
          <td>
            Maddalena Sbaffone, A. Farina, Ilaria Martelli, Eugenio Pontieri, Stefano Guadagni, A. Mackay, L. Cappabianca, Veronica Zelli
          </td>
          <td>2025-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Our understanding of the vertebrate immune system is dominated by a few model organisms such as mice. This use of a few model systems is reasonable if major features of the immune systems evolve slowly and are conserved across most vertebrates, but may be problematic if there is substantial macroevolutionary change in immune responses. Here, we present a test of the macroevolutionary stability, across 15 species of jawed fishes, of the transcriptomic response to a standardized immune challenge. Intraperitoneal injection of an immune adjuvant (alum) induces a fibrosis response in nearly all jawed fishes, which in some species contributes to anti-helminth resistance. Despite this conserved phenotypic response, the underlying transcriptomic response is highly inconsistent across species. Although many gene orthogroups exhibit differential expression between saline versus alum-injected fish in at least one species, few orthogroups exhibit consistent differential expression across species. This result suggests that although the phenotypic response to alum (fibrosis) is highly conserved, the underlying gene regulatory architecture is very flexible and cannot readily be extrapolated from any one species to fishes (or vertebrates) more broadly. The vertebrate immune response is remarkably changeable over macroevolutionary time, requiring a diversity of model organisms to describe effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02997822d89638ffcc115ee4c55d0d06267f2668" target='_blank'>
              Macroevolutionary changes in gene expression response to an immune stimulus across the diversity of fishes
              </a>
            </td>
          <td>
            Ben A. Flanagan, Lauren E. Fuess, M. Vrtílek, Andrea Roth-Monzón, Daniel I. Bolnick
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [5, 10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>