<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-14 09:36:37 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>46</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744c450e0fe105bb7eff34d3f9f47cc3b73c8c0c" target='_blank'>
              A patient-derived organoid model captures fetal-like plasticity in colorectal cancer.
              </a>
            </td>
          <td>
            Liang Xiong, Ying Xu, Zhaoya Gao, Jingyi Shi, Yunfan Wang, Xiaodong Wang, Wensheng Huang, Ming Li, Longteng Wang, Jun Xu, Cheng Li, Jin Gu, Hongkui Deng, Molong Qu
          </td>
          <td>2025-07-04</td>
          <td>Cell research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cells that survive therapeutic drug pressure are a significant cause of disease relapse and progression, impeding curative cancer treatment. Drug-triggered Darwinian selection and the emergence of subclones harbouring specific mutations that confer resistance have been well documented and extensively studied. However, these genetic alterations, while important, do not fully explain clinical observations where some patients, after a drug holiday, regain sensitivity to the same treatment despite previous disease progression. This phenomenon highlights the possibility that drug resistance may not solely rely on genetic mutations but could also involve reversible, non-genetic mechanisms. Recent studies have highlighted the existence of drug-tolerant persister cells (DTPs), a subpopulation of cancer cells that can survive short-term therapeutic pressure without acquiring resistance-associated genetic alterations. These cells exhibit a temporary yet reversible tolerance to the initial treatment while also acquiring cross-tolerance to other anti-cancer therapies. The presence of DTPs underscores a dynamic and complex plasticity in tumours, wherein cancer cells can utilise epigenetic rewiring, metabolic reprogramming, and specific signalling pathways to transit between drug-tolerant and drug-sensitive states to adapt to environmental pressures. Furthermore, this adaptive resilience enables DTPs to act as a reservoir for the development of genetically stable resistance, resulting in cancer therapy failure and eventual relapse. In this mini-review, we examine recent evidence on DTPs to provide an overview of their characteristics, development, and survival mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f885f194fb8254e0480e9608eccdddbd6818a86e" target='_blank'>
              Targeting Cancer Drug-Tolerant Persister Cells in Minimal Residual Disease
              </a>
            </td>
          <td>
            Wei Zhang, Sorayut Chattrakarn, Fuhui Chen, Haobin Chai, Mariana Maranga, Jingwei Zhang
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Drug Discovery and Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3f2125e256fc001c7ec119739d1e135a5d03" target='_blank'>
              A Mathematical Model of Persister Cell Plasticity and Its Impact on Adaptive Cancer Therapy
              </a>
            </td>
          <td>
            Adithya Mathews, D. Basanta
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9621f1d2872733a662aa856b54a67a500948b496" target='_blank'>
              Therapeutic targeting of mismatch repair-deficient cancers.
              </a>
            </td>
          <td>
            Paul Johannet, B. Rousseau, Carol Aghajanian, Mike Foote, Luis A. Diaz
          </td>
          <td>2025-07-10</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Osimertinib, a third-generation EGFR tyrosine kinase inhibitor (EGFR-TKI), has dramatically transformed the treatment landscape for patients with EGFR-mutant NSCLC. However, the long-term success of this therapy is often compromised by the onset of acquired resistance, with non-genetic mechanisms increasingly recognized as pivotal contributors. Here, we exploit a multi-omics approach to profile genome-wide chromatin accessibility and transcriptional landscapes between drug sensitive and resistant EGFR-mutant cells. Our findings reveal a robust concordance between epigenetic regulome and transcriptomic changes that characterize the osimertinib resistant state. Through CRISPR-based functional genomics screen targeting epigenetic regulators and transcription factors, we uncover a critical regulatory network featuring key members of the NuRD and PRC2 complexes that mediate resistance. Most critically, our screen identifies FOSL1 and JUN, two subunits of the AP-1 transcription factor within this network, as the most significant hits. Mechanistically, we demonstrate that cis-regulatory elements exhibiting altered chromatin accessibility in the resistant state are enriched for cognate AP-1 binding motifs, enabling AP-1 to orchestrate a gene expression program that underpins the druggable MEK/ERK signaling axis, potentially enhancing cell viability and fitness of resistant cells. Importantly, genetic depletion or pharmacological inhibition of AP-1 reinstates cellular and molecular sensitivity, and reverts resistance-associated phenotypes, such as epithelial-to-mesenchymal transition, upon anti-EGFR rechallenge by repressing AKT and ERK signaling. These findings provide novel insights into the epigenetic and transcriptional control of osimertinib resistance in EGFR-mutant NSCLC, highlighting AP-1 as a targetable vulnerability of resistance-related hallmarks and offering a promising avenue for developing resistance reversal strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98b05c64d117a31d26404e164fc0fbed4525adc" target='_blank'>
              Integrative multi-omics identifies AP-1 transcription factor as a targetable mediator of acquired osimertinib resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            B. Dayanc, Sude Eris, Nazife Ege Gulfirat, Gulden Ozden-Yilmaz, E. Cakiroglu, Ozlem Silan Coskun Deniz, G. Karakülah, Serap Erkek-Ozhan, Serif Senturk
          </td>
          <td>2025-05-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a leading cause of cancer-related mortality in men worldwide, with tumor heterogeneity, therapy resistance, and lineage plasticity posing significant clinical challenges. Androgen deprivation therapy (ADT), while initially effective, often culminates in castration-resistant prostate cancer (CRPC), may fueled by castration-resistant prostate cancer stem cells (CSCs) with adaptive self-renewal and regenerative capacities. Emerging evidence implicates prostate CSCs as pivotal contributors to tumor heterogeneity, drug resistance, and disease recurrence. These stem subpopulations exhibit intrinsic adaptability through genetic, epigenetic, and microenvironmental reprogramming, enabling survival under androgen-deprived conditions and fostering clonal diversification. This review synthesizes current knowledge on the identity, regulation, and functional dynamics of prostate stem cells, emphasizing their role in shaping the tumor ecosystem. By dissecting the genetic drivers, epigenetic alteration and crosstalk with microenvironment, this review underscores critical triggers on prostate CSCs determination and differentiation. Additionally, we discuss emerging strategies to target prostate CSCs-specific vulnerabilities, including molecular drivers of stemness and plasticity, to improve therapeutic outcomes for advanced prostate malignancies. This synthesis underscores the critical need to unravel prostate stem cells biology for developing precision therapies against prostate CSCs-driven adaptation in prostate diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/749e1d03aa4d620992d251430c919ea9331b3e5d" target='_blank'>
              Prostate cancer stem cell dynamics in the evolution of drug resistance.
              </a>
            </td>
          <td>
            Xiaoyu Zhang, Xiaoqing Zhang, Xinpei Deng, Jingna Cao, Qing Bao, Huan Wang, Dong Chen, Hailin Tang
          </td>
          <td>2025-06-04</td>
          <td>International journal of surgery</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Anticancer chemotherapy is an essential part of cancer treatment, but the emergence of resistance remains a major hurdle. Metabolic reprogramming is a notable phenotype associated with the acquisition of drug resistance. Here, we develop a computational framework that predicts metabolic gene targets capable of reverting the metabolic state of drug-resistant cells to that of drug-sensitive parental cells, thereby sensitizing the resistant cells. The computational framework performs single-gene knockout simulation of genome-scale metabolic models that predicts genome-wide metabolic flux distribution in drug-resistant cells, and clusters the resulting knockout flux data using uniform manifold approximation and projection, followed by k-means clustering. From the clustering analysis, knockout genes that lead to the flux data near that of drug-sensitive cells are considered drug sensitization targets. This computational approach is demonstrated using doxorubicin- and paclitaxel-resistant MCF7 breast cancer cells. Drug sensitization targets are further refined based on proteome and metabolome data, which generate GOT1 for doxorubicin-resistant MCF7, GPI for paclitaxel-resistant MCF7, and SLC1A5 as a common target. These targets are experimentally validated where treating drug-resistant cancer cells with small-molecule inhibitors results in increased sensitivity of drug-resistant cells to doxorubicin or paclitaxel. The applicability of the developed framework is further demonstrated using drug-resistant triple-negative breast cancer cells. Taken together, the computational framework predicts drug sensitization targets in an intuitive and cost-efficient manner and can be applied to overcome drug-resistant cells associated with various cancers and other metabolic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7052bad828f26940bd1911c2893c8896c16d13b8" target='_blank'>
              Genome-scale knockout simulation and clustering analysis of drug-resistant breast cancer cells reveal drug sensitization targets.
              </a>
            </td>
          <td>
            JinA Lim, Hae Deok Jung, S. Park, Moonhyeon Jeon, Da Sol Kim, Ryeongeun Cho, Dohyun Han, Han Suk Ryu, Yoo-Ree Kim, Hyun Uk Kim
          </td>
          <td>2025-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Next-generation sequencing technology enables uniform and impartial assessment of cancer diagnoses and prognosis. However, such studies are mostly type-specific, and capturing shared genomic abnormalities responsible for neoplastic transformation and progression is a challenging task. Pan-cancer analysis offers insights into the shared and unique molecular mechanisms driving cancer. We conducted an integrated gene-expression analysis using 10,629 samples from 30 distinct cancer types characterized by The Cancer Genome Atlas (TCGA). A gene co-expression network was constructed and genes overlapping between the selected modules and Differentially Expressed Genes (DEGs) were designated as genes of interest. Following a comprehensive literature review, ATP binding cassette subfamily A member 10 (ABCA10) and ATP binding cassette subfamily B member 5 (ABCB5) were selected as key candidates for downstream analysis due to the absence of systematic pan-cancer analysis of these genes. This study presents a unique contribution as the first comprehensive pan-cancer analysis of ABCA10 and ABCB5, highlighting their roles in tumor biology and clinical outcomes. We employed a variety of bioinformatics tools to explore the role of these genes across different tumors. Our research demonstrated that ABCA10 shows reduced expression, while ABCB5 displays variable expression patterns across tumors, indicating their opposing roles and flexible functions in pan-cancer. In many cancer patients, these expression patterns are correlated with worse survival outcomes. Furthermore, immunotherapy responses and immune infiltration across a variety of tumor types are associated with the expression levels of both ABCA10 and ABCB5. These results imply that ABCA10 and ABCB5 could serve as valuable predictive markers and potential therapeutic targets across various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f1be3a2286bbe9d4105e8c1625c6913bc7d7bf" target='_blank'>
              Integrated pan-cancer analysis revealed therapeutic targets in the ABC transporter protein family
              </a>
            </td>
          <td>
            M. B. E. Masood, Iqra Shafique, Muhammad Inam Rafique, Ayesha Iman, Ariba Abbasi, Mehak Rafiq, Uzma Habib
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Alternative splicing (AS) is a substantial contributor to the high complexity of transcriptomes in multicellular eukaryotes. Fast chemical reprogramming (FCR) system is an innovative approach that facilitates the rapid transition of somatic cells into induced pluripotent stem cells (iPSCs). Results In this study, we used the FCR system to delve into the dynamics of AS during cell fate transition. The trajectory of FCR, as characterized by gene expression profiles, consistently aligned with that observed in AS patterns, revealing a complex interplay between AS and gene expression regulation. Additionally, we discovered that the exon exclusion events were more prevalent than the exon inclusion events, indicating a predominant mode of splicing regulation during FCR. Compared to transcription factor-induced reprogramming (TFR), FCR showed a distinct AS pattern, underscoring the unique regulatory mechanisms governing AS in each reprogramming system. Further investigation uncovered polypyrimidine tract-binding protein 3 (Ptbp3) as an important splicing factor, possibly participating in epigenetic regulation in late stage of FCR by affecting AS of epigenetic regulators. Moreover, we found an abundance of intron retention events caused by decrease in spliceosome activity, potentially contributing to the downregulation of key diapause-related genes in the middle and late stages of FCR. Conclusions This research provided a comprehensive characterization of AS during FCR, highlighting the pivotal roles of AS in regulating cell fate transitions. Our findings advanced the understanding of the molecular mechanisms governing cell fate decisions and offered new insights into the potential of FCR for regenerative medicine and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02264-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c973566b1c953312a6b93c2ff963467d8d2203b5" target='_blank'>
              Deciphering alternative splicing patterns during cell fate transition of fast chemical reprogramming
              </a>
            </td>
          <td>
            Yunkun Lu, Kainan Lin, Yeling Ruan, Junjie Li, Huizhen Zhang, Tianyuan Pan, Qianqian Wang, Lianyu Lin, Sijie Feng
          </td>
          <td>2025-06-09</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771ef0b5061a9c5213ccca9fb3801ba5c0ea8357" target='_blank'>
              Whole Exome Sequencing and Single-Cell DNA Sequencing for Assessment of Clonal Heterogeneity and Evolution in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sadiksha Adhikari, S. Eldfors, Esa Pitkänen, Markus Vähä-Koskela, C. Heckman
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In ~70% of patients with pancreatic ductal adenocarcinoma, the TP53 gene acquires gain-of-function (GOF) mutations leading to rapid disease progression. Specifically, missense p53 (misp53) GOF mutations associate with therapy resistance and worse clinical outcomes. However, the molecular functions of distinct misp53 mutants in plasticity and therapy response remain unclear. Integrating multicenter patient data and multi-omics, we report that the misp53R273H/C mutant is associated with cell cycle progression and a basal-like state compared to the misp53R248W/Q mutant. Loss of misp53R273H/C decreased tumor growth and liver metastasis while prolonging survival in preclinical models. We found that misp53R273H/C specifically regulated the Rb/DREAM axis involved in cell cycle regulation. Notably, a clinical CDK4/6 inhibitor reduced misp53R273H/C mutant expression. However, it triggered MAPK/ERK-mediated resistance mechanisms, enhancing cell survival and resistance to CDK4/6 inhibitors. Combining MAPK/ERK and CDK4/6 inhibitors reduced misp53R273H/C-associated oncogenic functions. Thus, distinct misp53 mutants show unique cell-intrinsic plasticity, therapeutic vulnerabilities, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8366ad2cf60feed0070f18be69dfac6ab275f732" target='_blank'>
              TP53 missense–specific transcriptional plasticity drives resistance against cell cycle inhibitors in pancreatic cancer
              </a>
            </td>
          <td>
            Laura Urbach, Lena Wieland, Frederike Penz, Rebecca Diya Samuel, Stefan Küffer, Lukas Klein, Christof Lenz, Ulrich Sax, Michael Ghadimi, Ramona Schulz-Heddergott, E. Hessmann, V. Ellenrieder, Nelson Dusetti, Shiv K. Singh
          </td>
          <td>2025-07-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e20582


 Background:
 Despite advances in therapies, the development of drug resistance in non-small cell lung cancer (NSCLC) poses a significant challenge. The emergence of neoantigen-based therapies offer a promising new frontier which enhances treatment efficacy and reduces associated costs.
 Methods:
 A total of 24 patients diagnosed with NSCLC were enrolled in this study. Primary tumor tissues and peripheral blood samples were collected from each patient. Paired whole exome and transcriptomic sequencing data were performed to identify somatic mutations and to quantify their respective expression levels. Somatic single nucleotide variants (SNVs) and small insertions or deletions (inDels) were detected using Strelka algorithm from whole exome sequencing (WES) data. Neoantigen prediction focused on human leukocyte antigens (HLA) type 1. HLA alleles were predicted using WES data encoding adjacent non-cancerous tissue samples, identifying four alleles that were present in at least 50% of the patients. Neoantigens were considered potentially immunogenic if their predicted median IC50 binding scores were ≤500nM while expressed transcripts per million (TPM) exceeds 1 in tumor samples.
 Results:
 Using systematic bioinformatics pipeline, we analyzed 24 NSCLC samples with matched tumor tissues and peripheral blood, and detected the 4677 mutations involving 2442 genes, including 2819 missense SNVs, 941 InDels, 229 gene fusion and 688 RNA alternative splicing events, 94.7% of which (4428 mutations involving in 2247 genes) were common with those reported in the TCGA-LUAD dataset. The average number of mutations was 194.8 in NSCLCs. We found that C>T/G>A transitions/ transversions were dominant. Missense mutations were the most frequent types of somatic mutation in the coding sequence regions. 98% of mutated genes were not detected simultaneously with DNA and RNA mutations.
 ADCK5
 ,
 CD151
 and
 EIF3K
 were identified as potential new driver genes. Genomic variation profiling identified 2213 potential neoantigens with a median IC50 binding score of ≤500nM and a transcript expression level of >1 TPM. Among these, 41 were listed in databases TSNAdb, IEDB, and CTDatabase. A total of 14 genes harboring identical mutations were detected, with the identical mutations observed in over 50% of the patient cohort. Notably, mutations in 14 genes were either gene fusions or RNA alternative splicing events, and core antigenic sequences generated were found to be identical in the majority of patients. Of particular interest is the RPRD2 gene, where an RNA exitrons site was detected in all patient samples, along with the identification of the same neoantigen core sequence.
 Conclusions:
 Our analysis suggests that gene fusions and RNA splicing variations are more likely than point mutations to generate shared antigens, underscoring their importance for developing targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1725e99e0357b9a0ce8d33b25d7c68d81214eed0" target='_blank'>
              Comprehensive genomic alterations identification and potential shared neoantigens for non-small cell lung cancer immunotherapy.
              </a>
            </td>
          <td>
            Muchun Zhu, Qing Zhao Ruan, Boyue Zhang, Weiqiang Yin, Run Sheng Ruan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, , , , , J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Colorectal cancer (CRC), the third most common cancer worldwide, often shows limited responsiveness to immunotherapy due to its predominantly immune-excluded phenotype. Despite increasing insights into the complex tumor microenvironment (TME), the metabolic heterogeneity of CRC cells and their interactions with tumor-infiltrating immune cells remain poorly understood. Methods We analyzed 46,374 epithelial cells from 17 CRC patients treated with PD-1 blockade to develop an amino acid (AA) metabolism score using the AUCell algorithm. This score was applied to a separate single-cell RNA sequencing (scRNA-seq) dataset from 23 CRC patients to investigate cell-cell interactions and functions of tumor-infiltrating immune cells, revealing distinct immune TME landscapes shaped by tumor metabolism. An in vitro co-culture assay of CRC cells and CD8+ T cells was performed to validate the findings. Additionally, LASSO and Cox regression analyses were conducted to construct an AA metabolism-related risk score for predicting prognosis and drug sensitivity across multiple bulk transcriptome cohorts. Results This study identified a link between elevated amino acid metabolism in CRC epithelial cells and resistance to PD-1 blockade therapy. A 31-gene AA score was developed by intersecting differentially expressed genes between responders and non-responders to PD-1 blockade with amino acid metabolism-related genes from the Molecular Signature Database (MSigDB). Using this score, 23 additional CRC samples were classified into high and low AA score groups. Comparative analysis revealed that the low AA group exhibited a more robust immune response, characterized by a greater number and stronger cell-cell interactions. Tumor-infiltrating immune cells in this group demonstrated enhanced activation and anti-tumor functions. Furthermore, CD8+ T cells showed increased Granzyme B levels when co-cultured with CRC cells in which Psat1 or Shmt2 was knocked down. Finally, a machine learning-derived risk score based on six genes was established to translate single-cell findings to bulk transcriptomes. This risk score was found to correlate with immune checkpoint expression and immune cell infiltration, with potential implications for predicting prognosis and drug sensitivity. Conclusion Our findings highlight the role of elevated epithelial amino acid metabolism in shaping an immune-suppressive microenvironment, offering insights for patient stratification and therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9bda18d8cb73c7c5646b13e67e456908ec19a2" target='_blank'>
              Single-cell and bulk transcriptome analyses reveal elevated amino acid metabolism promoting tumor-directed immune evasion in colorectal cancer
              </a>
            </td>
          <td>
            Tianyue Sun, Yan Chen, Yingsheng Chen
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Despite extensive research efforts, glioblastoma (GBM) remains a deadly disease with poor prognosis. Although previous studies have identified various cell states within GBM tumors, the molecular mechanism underlying adaptive GBM cell plasticity induced by conventional therapy remains unclear.


METHODS
We used fluorescent reporters for proneural (PN) and mesenchymal (MES) subtypes to monitor GBM cell plasticity in real-time across multiple patient-derived cell lines. This approach revealed cells that concurrently expressed both proneural and mesenchymal markers. To investigate this unique hybrid population, we implemented a comprehensive methodological approach encompassing bulk and single-cell RNA sequencing, single-cell ChIP sequencing, nuclear proteomics, high-resolution imaging, orthotopic mouse models, clinical dataset analysis, and pharmacological and genetic techniques. This multifaceted strategy allowed us to gain functional and molecular insights into this distinct cellular population.


RESULTS
We showed that these hybrid cells are increased by conventional therapies, and are resistant to these therapies. At the molecular level, hybrid cells display significant alterations in chromatin structure and nuclear protein composition, elevated transcriptional activity, Myc activation, and improved transport between the nucleus and cytoplasm. Genetic and pharmaceutical inhibition of the nuclear import/export shuttling machinery, increased in hybrid cells, effectively suppressed adaptive GBM cell plasticity and hybrid identity, thereby enhancing the sensitivity of GBM cells to therapies.


CONCLUSION
Our results indicate that GBM hybrid cells play a crucial role in chemoradiation resistance. The nuclear transport machinery presents a potential therapeutic target for hybrid cells, offering a way to counteract the typical resistance to treatment observed in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6116cf0972c0f5ba1621a1614ba9f7a9a3add325" target='_blank'>
              The SOX gene superfamily in oncogenesis: unraveling links to ncRNAs, key pathways, chemoresistance, and gene editing approaches.
              </a>
            </td>
          <td>
            Ammar Yasir Ahmed, Subasini Uthirapathy, Enwa Felix Oghenemaro, R. M, Rohit Kumawat, Yasser Fakri Mustafa, Muthena Kariem, A. J. Kadhim, Shilpa Sharma, M. R. Kumar
          </td>
          <td>2025-05-20</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="This study investigated whether therapeutic X-ray doses can affect the expression of mismatch repair (MMR) genes and proteins using Lynch syndrome-associated human colorectal cancer cell lines. MMR-deficient cell lines (HCT116, SW48, LoVo) and an MMR-proficient control cell line (HT29) were exposed to X rays [a 2 Gy dose or 2 Gy daily for five consecutive days (10 Gy)]. Reverse transcription quantitative real-time PCR (RT-qPCR) and Western blotting were used to detect the radiation-induced changes in the expression of RNAs and proteins, respectively. RT-qPCR revealed that MLH1 and MSH6 genes were stably expressed regardless of the MMR status of the cell line and the radiation dose. In contrast, the MSH2 gene was either up-regulated or down-regulated after 2 Gy or 10 Gy or both. The expression of PMS2 increased after 10 Gy irradiation in all MMR-deficient cell lines, even though the data were not statistically significant compared to other doses, except for the LoVo cell line. Protein expression analysed using Western blotting demonstrated that MLH1 protein expression was stable, whereas the expression of MSH2 was significantly affected by radiation exposure in both MLH1-deficient cell lines. No correlation between the expression of RNA and protein could be identified. In conclusion, radiation may have significantly differential effects on MMR RNA and protein expression when different cell lines, doses, and specific genes are considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b8835db8d3ae7872bbc2b8269596b668e92380" target='_blank'>
              The Effect of X rays on the Expression of Mismatch Repair Genes and Proteins in Lynch Syndrome Associated Human Colorectal Cancer Cell Lines.
              </a>
            </td>
          <td>
            Mingzhu Sun, L. Cruz-Garcia, Danny Freestone, Kevin Monahan, C. Badie, Yannick Comoglio, Hannah Mancey, J. Moquet, Stephen Barnard
          </td>
          <td>2025-06-20</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PARP inhibitors have profoundly changed treatment options for cancers with homologous recombination repair defects, especially those carrying BRCA1/2 mutations. However, the development of resistance to these inhibitors presents a significant clinical challenge as it limits long-term effectiveness. This review provides an overview of the current understanding of resistance mechanisms to PARP inhibitors and explores strategies to overcome these challenges. We discuss the basis of synthetic lethality induced by PARP inhibitors and detail diverse resistance mechanisms affecting PARP inhibitors, including homologous recombination restoration, reduced PARP trapping, enhanced drug efflux, and replication fork stabilization. The review then considers clinical approaches to combat resistance, focusing on combination therapies with immune checkpoint inhibitors, DNA damage response inhibitors, and epigenetic drugs. We also highlight ongoing clinical trials and potential biomarkers for predicting treatment response and resistance. The review concludes by outlining future research directions, emphasizing the need for longitudinal studies, advanced resistance monitoring technologies, and the development of novel combination strategies. By tackling PARP inhibitor resistance, this review seeks to aid in the development of more effective cancer therapies, with the potential to improve outcomes for patients with homologous recombination-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b586fe710eefe8bf6bedea662472c8b2059e03" target='_blank'>
              Clinical approaches to overcome PARP inhibitor resistance
              </a>
            </td>
          <td>
            Yutian Zou, Hanqi Zhang, Pangzhou Chen, Jiayi Tang, Siwei Yang, Christophe Nicot, Ziyun Guan, Xing Li, Hailin Tang
          </td>
          <td>2025-05-30</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Activation of oncogenes, such as through mutations in Kirsten rat sarcoma viral oncogene homolog (KRAS), triggers profound disruptions in cellular homeostasis that set off a cascade of stress responses. These responses enable cells to cope with the array of challenges encountered during tumorigenesis by activating defense mechanisms that promote adaptation and survival. Key components of this oncogenic stress response include heat shock proteins, the ubiquitin-proteasome system, autophagy, nuclear factor erythroid 2-related factor 2–antioxidant response element signaling, DNA damage response proteins, p53, redox-regulating proteins, and stress granules. This review concentrates on KRAS-driven oncogenic transformation, as KRAS mutations are among the most common in human cancers, accounting for over 90% of pancreatic ductal adenocarcinoma cases, around 30% of lung cancers, and approximately 50% of colorectal cancers. We examine the intricate molecular interplay between oncogenic stress and the associated cellular defense mechanisms, emphasizing the key molecular events that follow KRAS activation. Importantly, the very pathways that allow cancer cells to adapt to oncogenic stress also offer novel therapeutic opportunities. By selectively targeting pivotal regulators within these stress response pathways, we can potentially disrupt the survival mechanisms of cancer cells. This strategy not only promises to enhance the effectiveness of existing treatments but also paves the way for the development of innovative therapies designed to combat tumor progression. In essence, exploiting oncogenic stress responses represents an original and promising therapeutic approach in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a19b0d4dfe04d02b6a9051c867a7e87fe82159" target='_blank'>
              Oncogenic stress response mechanisms as new therapeutic targets in cancer treatment: A review
              </a>
            </td>
          <td>
            J. Iovanna, Matias Estaras, Daniel Grasso, Martin E. Fernández Zapico, J. L. Neira, P. Santofimia-Castaño
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of differentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, differential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and differentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by differential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be sufficient for gene regulation. Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS - The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops - Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis - Chromatin domain boundaries are weakened during cancer progression - Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops - Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression - Prominent changes in chromatin loops occur at a small subset of differentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c327cba87285dc890d1f7b106ff92f090a08a250" target='_blank'>
              Single-Cell Analysis of Meningiomas Reveals Mutation-Associated Tumor and Immune Cell Gene Expression Programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus I Anzaldua-Campos, Gregory J Zipfel, Albert H. Kim, J. Dowling, Eric C. Leuthardt, Joshua W. Osbun, A. Vellimana, Michael R. Chicoine, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Despite the advances in single cell whole genome sequencing, the ability to detect structural variations from mitotic crossovers remains challenging, while sister chromatid exchanges remain undetectable. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic crossover outcomes and resulting genome instability. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational crossovers in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair, structural variations, and genome instability in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867f71c31d23d231530c8f668476d966ca7500c8" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Targeted therapy has markedly improved outcomes for patients with non‑small cell lung cancer (NSCLC). However, the emergence of drug resistance remains a major clinical challenge, limiting long‑term treatment efficacy. Clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9, a revolutionary gene‑editing technology, offers precise and efficient genetic modifications, providing new insights into the mechanisms of drug resistance in NSCLC. The present review explored the application of CRISPR/Cas9 in overcoming resistance associated with key oncogenic drivers, including EGFR, KRAS, ALK, ROS1, MET and BRAF. It summarized recent advances in CRISPR‑based strategies to reverse resistance, enhance targeted therapy effectiveness and develop potential therapeutic interventions. Additionally, it discussed current limitations, including off‑target effects, delivery challenges and ethical concerns, while highlighting future directions for clinical translation. Using CRISPR/Cas9 technology may pave the way for novel, personalized treatment approaches in NSCLC, ultimately improving patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ddd19a50ff46be19254c0572146995de6cd1223" target='_blank'>
              Harnessing CRISPR/Cas9 to overcome targeted therapy resistance in non‑small cell lung cancer: Advances and challenges (Review).
              </a>
            </td>
          <td>
            Jianting Du, Xian Gong, Renjie Huang, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2025-09-01</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla T. O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Alternative splicing (AS) is a pivotal post-transcriptional mechanism that expands the functional diversity of the proteome by enabling a single gene to generate multiple mRNA and protein isoforms. This process, which involves the differential inclusion or exclusion of exons and introns, is tightly regulated by splicing factors (SFs), such as serine/arginine-rich proteins (SRs), heterogeneous nuclear ribonucleoproteins (hnRNPs), and RNA-binding motif (RBM) proteins. These factors recognize specific sequences, including 5′ and 3′ splice sites and branch points, to ensure precise splicing. While AS is essential for normal cellular function, its dysregulation is increasingly implicated in cancer pathogenesis. Aberrant splicing can lead to the production of oncogenic isoforms that promote tumorigenesis, metastasis, and resistance to therapy. Furthermore, such abnormalities can cause the loss of tumor-suppressing activity, thereby contributing to cancer development. Importantly, abnormal AS events can generate neoantigens, which are presented on tumor cell surfaces via major histocompatibility complex (MHC) molecules, suggesting novel targets for cancer immunotherapy. Additionally, splice-switching oligonucleotides (SSOs) have shown promise as therapeutic agents because they modulate splicing patterns to restore normal gene function or induce tumor-suppressive isoforms. This review explores the mechanisms of AS dysregulation in cancer, its role in tumor progression, and its potential as a therapeutic target. We also discuss innovative technologies, such as high-throughput sequencing and computational approaches, that are revolutionizing the study of AS in cancer. Finally, we address the challenges and future prospects of targeting AS for personalized cancer therapies, emphasizing its potential in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2739ebba4691f4fc04db8de970276d260f0dc2" target='_blank'>
              Alternative Splicing in Tumorigenesis and Cancer Therapy
              </a>
            </td>
          <td>
            Huiping Chen, Jingqun Tang, Juanjuan Xiang
          </td>
          <td>2025-05-29</td>
          <td>Biomolecules</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Cancer progression is a complex, multi-stage development process characterized by dynamic changes at the molecular level. Understanding these changes may provide new insights into tumorigenesis and potential therapeutic targets. This study focuses on the evolutionary transcriptomics of cancer, specifically analyzing the Transcriptome Age Index (TAI) across different pathological stages. By examining various cancers at four distinct pathological stages, we identify a significant «hourglass» pattern in TAI indices of ductal carcinoma of the breast, bladder carcinoma, and liver carcinoma, suggesting a conserved evolutionary trajectory during tumor development. The results reveal that early and late stages of these cancers exhibit higher TAI values, indicative of more novel gene expression, while intermediate stages show a dip in TAI, reflecting a more ancient evolutionary origin of expressed genes. This «hourglass» pattern underscores the evolutionary constraints and innovations at play during tumor progression. Our findings contribute to the growing body of evidence that evolutionary principles are deeply embedded in cancer biology, offering new perspectives on the dynamics of gene expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d899ba20c349660eb4cd72c9aa668578d579a2" target='_blank'>
              Evolutionary Transcriptomics of Cancer Development
              </a>
            </td>
          <td>
            Roman Ivanov, D. Afonnikov, Yury Matushkin, S. Lashin
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c141c00d5de3690a0dd4e5aff895db7f355c5a9" target='_blank'>
              Loss of DNA mismatch repair genes leads to acquisition of antibiotic resistance independent of secondary mutations
              </a>
            </td>
          <td>
            David E. Bautista, Cassidy R. Whitehead, Angela M. Mitchell
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2aba01b3716bf8fef6d8f3f3f6e3783dfc01cf" target='_blank'>
              Recent advances in novel targeting mechanisms for colorectal cancer.
              </a>
            </td>
          <td>
            Zhiping Li, Hailing Wang, Yan Liu, Ahequeli Gemingnuer, Yinan Wang, Xin Meng
          </td>
          <td>2025-07-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423474067572f39ac510a609277d4e54dd172eb6" target='_blank'>
              YAP signaling promotes resistance to MEK and AKT inhibition in NF1-related MPNSTs
              </a>
            </td>
          <td>
            L. E. McGee, J. Grit, Curt J. Essenburg, Sophia Agrusa, Elizabeth A. Tovar, Lisa Turner, K. Becker, J. Zbieg, Anwesha Dey, Jennifer E. Klomp, Jeffrey A. Klomp, Marie Adams, Ian Beddows, Emily Wolfrom, Zhen Fu, A. Hirbe, Carrie R. Graveel, Matthew R. Steensma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2537688c884004a06a6382cdb89bfdc63b85ffff" target='_blank'>
              CRISPR-Based Gene Dependency Screens reveal Mechanism of BRAF Inhibitor Resistance in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            Shawn Noronha, Yue Liu, Gaga Geneti, Haojian Li, Xiaolin Wu, David Sun, Vaibhavi Gujar, Takashi Furusawa, Alexei Lobanov, Maggie Cam, Lipika R. Pal, N. Nair, Chi-Ping Day, E. Ruppin, Chandrayee Gosh, Jiangnan Hu, Suresh Kumar, T. Andresson, King Chan, Maura O’Neill, Raj Chari, Y. Pommier, J. Rivero, Urbain Weyemi, E. Kebebew, Myriem Boufraqech
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="CRISPR has become a revolutionary tool for cancer biology and therapy with unprecedented precision for genome editing. Besides cancer, CRISPR is also under research for possible use in the treatment of viral diseases, such as Bovine Viral Diarrhea Virus (BVDV), a serious livestock problem. Through enabling gene modifications to be made to individual genes, CRISPR is used in identifying oncogenic drivers, characterizing tumor suppressor networks, and designing innovative therapeutics. This review focuses on the various uses of CRISPR for cancer studies with special attention to methods like gene knockout, gene activation, and base editing that each has a great potential for correcting oncogenic mutations and restoring tumor suppressor activity. Moreover, CRISPR's capacity to regulate viral replication, as that of BVDV, signifies its dual role in oncology as well as in viral pathogenesis. Interventions based on CRISPR such as chimeric antigen receptor (CAR) T cell therapy and synthetic lethality are essentially revolutionizing cancer therapy by improving immune responses and capitalizing on the specificity that is characteristic of cancer cells. However, off-target effects, tumor heterogeneity, and ethical dilemmas remain onerous challenges for clinical application. CRISPR delivery systems, despite playing a central role in advancing cancer therapies, also have prospects in optimizing the effectiveness of treatment against viral pathogens like BVDV. Advances in delivery systems using nanoparticles and viral vectors are mitigating against these challenges and improving efficacy and specificity of CRISPR reagent in vivo. In addition to this, continued progress in new technologies such as prime editing and base editing is predicted to improve precision and efficacy of CRISPR-based therapy. Since CRISPR technology is continuously developing, CRISPR's potential to treat both cancer and viral infections, like BVDV, simultaneously will be instrumental in precision medicine. This review highlights the revolutionary potential of CRISPR to revolutionize cancer treatment paradigms and brings hope for more effective and individualized therapy. With potential developments to emergent and potentiated CRISPR tools and reagents, it is predicted that they will play a center stage role in precision oncology and promote better patient results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d3748e9db2d0a3114958d899e7fef2d33921ff" target='_blank'>
              CRISPR in Cancer Biology and Bovine Viral Diarrhea Therapy: Precision Genome Editing at the Frontier of Oncology and Viral Pathogenesis
              </a>
            </td>
          <td>
            Jindong Gao, Mengdi Zhang, Linjia Gao, Lei Kuang, Tuniyazi. Maimaiti, Ling Zhao, Qinghua Shang, Changmin Hu
          </td>
          <td>2025-05-19</td>
          <td>Bio Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetically identical bacterial cells often exhibit heterogeneous responses to antibiotics − some survive, others die. Here, we show that this heterogeneity propagates across generations to give rise to phenotypic resistance. Using real-time single-cell tracking, we exposed Escherichia coli to the β-lactam cefsulodin at its clinical breakpoint concentration and analyzed cell fate within genealogical trees statistically. Cell survival was strongly correlated among family members, driving the selective enrichment of robust lineages within an otherwise susceptible population. Our genealogical population model identified heritable phenotypic resistance as a key factor underlying this enrichment, which was validated experimentally. Comparing enrichment dynamics between the wild-type and a tolC knock-out strain, deficient in multidrug efflux, uncovered nuanced changes that increased the intergenerational memory of phenotypic resistance. Our findings provide evidence for heritable phenotypic resistance and demonstrate how its propagation through cell-to-cell heterogeneity enables the survival of minority cells within isogenic populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6715bbe58fe26a64981336caf37053a9a8306b82" target='_blank'>
              Lineage-dependent variations in single-cell antibiotic susceptibility reveal the selective inheritance of phenotypic resistance in bacteria
              </a>
            </td>
          <td>
            Wesley Stine, Tatsuya Akiyama, David S. Weiss, Minsu Kim
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are repetitive DNA sequences that contribute to genetic diversity and play a significant role in disease susceptibility. The human genome contains approximately 1.5 million STR loci, collectively covering around 3% of the total sequence. Certain repeat expansions can significantly impact cellular function by altering protein synthesis, impairing DNA repair, and leading to neurodegenerative and neuromuscular diseases. Traditional short-read sequencing struggles to accurately characterize STRs due to its limited read length, which limits the ability to resolve repeat expansions, increases mapping errors, and reduces sensitivity for detecting large insertions or interruptions. This review examines how long-read sequencing technologies, particularly Oxford Nanopore and PacBio, overcome these limitations by enabling direct sequencing of full STR regions with improved accuracy. We discuss challenges in sequencing, bioinformatics workflows, and the latest computational tools for STR detection. Additionally, we highlight the strengths and limitations of different methods, providing deeper insight into the future of STR genotyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bca970ab6c190594afe0092773766d77d84cc3b" target='_blank'>
              Get ready for short tandem repeats analysis using long reads-the challenges and the state of the art
              </a>
            </td>
          <td>
            Marija Chaushevska, Karmele Alapont-Celaya, Anne Kristine Schack, Lukasz Krych, M. C. Garrido Navas, Anastasia Krithara, Gjorgji Madjarov
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Identifying genetic mutations contributing to solid tumors by altering the biological pathways related to tumor formation and development is essential for the development of targeted therapies. This study aimed to identify commonly mutated genes and altered pathways in canine solid tumors. Four dogs with different types of naturally occurring neoplasias (urothelial carcinoma, adenocarcinoma, rhabdomyosarcoma, and chondrosarcoma) were randomly selected and classified into carcinoma and sarcoma groups based on histopathological findings. Tumor tissues were analyzed using whole-genome sequencing, and significant variants shared within each tumor group were identified. Gene set enrichment analyses were conducted to compare the biological and functional pathways altered by the mutations in each carcinoma and sarcoma group. Forty-three and fifty-eight genes were identified in the carcinoma and sarcoma groups, respectively. Distinctions between the two tumor groups were noted for mutations related to tumor metastatic function. Mutations were identified in genes encoding cell adhesion molecules in the carcinoma group, whereas significant variations in extracellular matrix-related molecules were evident in the sarcoma group. This study revealed mutations and modified pathways associated with immune and tumor metastatic functions in canine carcinoma and sarcoma, indicating their significant relevance to the development and progression of each tumor group. Additionally, the distinctions indicated that different therapeutic approaches were required for each tumor group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ab56a6166113f6d5cb839f0773a111660d3e21" target='_blank'>
              Identifying functional roles and pathways of shared mutations in canine solid tumors by whole-genome sequencing
              </a>
            </td>
          <td>
            YeSeul Jeon, Hyeona Bae, Seung-Wan Woo, Jaemin Kim, Dohyeon Yu
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy.The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments.This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion.This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The DNA damage response comprises a set of imperfect pathways that maintain cell survival following exposure to DNA damaging agents. Cancers frequently exhibit DNA repair pathway alterations that contribute to their intrinsic genome instability. This, in part, facilitates a therapeutic window for many chemotherapeutic agents whose mechanisms of action often converge at the generation of a double-strand DNA break. The development of therapy resistance occurs through countless molecular mechanisms that promote tolerance to DNA damage, often by preventing break formation or increasing repair capacity. This review broadly discusses the DNA damaging mechanisms of action for different classes of chemotherapeutics, how avoidance and repair of double-strand breaks can promote resistance, and strategic directions for counteracting therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77a69284f3f1f8221d2572829a8005590b029476" target='_blank'>
              DNA repair and the contribution to chemotherapy resistance
              </a>
            </td>
          <td>
            K. Nešić, Phoebe Parker, Elizabeth M. Swisher, John J Krais
          </td>
          <td>2025-05-26</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6f92731e4153548f5adf8b4c57b2dcf0276aa2" target='_blank'>
              Single-cell transcriptomics uncovers that cell-to-cell communication controls p53 activation
              </a>
            </td>
          <td>
            L. Castellano, Mark E Robinson, Aleksandra Dabrowska, Niklas Feldhahn, S. Ottaviani, Justin Stebbing
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a09fe0116b674341564f20d065726804460dc81" target='_blank'>
              Identification of hub genes and potential molecular mechanisms of tumor immune microenvironment-related radiotherapy sensitivity in locally advanced cervical cancer.
              </a>
            </td>
          <td>
            Yichen Zan, Qing Liu, Yue Yin, Chongxin Guo, Jing Xu, Junhan Gao, Guanglu Dong
          </td>
          <td>2025-06-30</td>
          <td>Reproductive sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f23b04434bb73a39ffc142be7c6ab3896c53a0" target='_blank'>
              Longitudinal single cell RNA-sequencing reveals evolution of micro- and macro-states in chronic myeloid leukemia
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Lucy Ghoda, Jeffery M. Trent, Stephen J Forman, Adam L. Maclean, Ya-Huei Kuo, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Proliferating cell nuclear antigen (PCNA) is a central regulator of numerous cellular processes impacting DNA metabolism and genetic integrity, including DNA damage, cell cycle progression, and transcriptional regulation. PCNA aberrations manifest in different disease phenotypes, including neoplastic proliferation, chromatin disassembly, genomic instability, and impaired DNA repair. Although the role of PCNA in control of DNA homeostasis has been extensively studied in some cancers, its pan-cancer relevance in tumorigenesis, immune interactions, and therapeutic resistance remains underexplored. Here, we undertook a comprehensive analysis of publicly available databases to assess the relationship between PCNA expression and the immunological, survival, genetic, functional, and drug sensitivity profiles across multiple cancer types. PCNA mRNA levels were altered in across cancers and associated with altered cancer signaling networks constituting Wnt, Hippo, and mTOR pathways. Increased transcript levels were associated with poor overall survival in some cancers, including ACC, CESC, LGG, and THYM. Amplification was the predominant genetic alteration of PCNA in multiple tumors. In several tumors, upregulation of PCNA was linked with differences in tumor-infiltrating lymphocytes, and specific immune-inhibitors, and chemokines. Moreover, elevated PCNA expression was linked with increased sensitivity to several drugs, particularly to Navitoclax, NPK76-II-72-1, and Ciclopirox across cancers. Our study orients the first comprehensive pan-cancer analysis of PCNA, uncovering its prognostic significance and altered expression across various cancers through multiomics data. Unlike previous studies, tumor-specific genetic alterations, such as amplification and hypomethylation, and the paradoxical immune microenvironment linked to PCNA were explored, suggesting potential immune evasion mechanisms. Additionally, new therapeutic avenues reveal PCNA’s relationship with drug sensitivity to agents like Navitoclax and Ciclopirox, providing invaluable insights for pharmacological interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3571f2ad7f72f9a390320cd46deb0b77021902" target='_blank'>
              PCNA in Pan-Cancer: A Prognostic Biomarker Unveiled Through a Data-Driven, Multidimensional Analysis of Transcriptomics, Immunity, and Functional Profiling
              </a>
            </td>
          <td>
            Depanshi Pandit, Ravindranath Bilachi Sanganabasappa, Amardeep Dhillon, Sanjiban Chakrabarty
          </td>
          <td>2025-06-26</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, A. Singh, Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Leukemia, a group of malignant blood disorders, arises from the uncontrolled proliferation of abnormal white blood cells. Genetic mutations play a critical role in the initiation and progression of leukemia. This review aims to provide an overview of the genetic landscape of leukemia, focusing on the most common genetic alterations and their clinical implications. A literature search was conducted to identify relevant studies on genetic mutations in leukemia. The identified studies were critically appraised to assess their methodological quality. The present review highlights the key genetic alterations associated with different types of leukemia, including chromosomal translocations, point mutations, and gene copy number variations. These genetic abnormalities can impact disease prognosis, treatment response, and overall patient survival. A comprehensive understanding of the genetic basis of leukemia is essential for accurate diagnosis, prognostication, and the development of targeted therapies. Future research should focus on identifying novel genetic markers, elucidating the underlying mechanisms of leukemogenesis, and developing innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e966f7c7f21cf8c9b127f6afb40f6c702d2c3" target='_blank'>
              A Review Article: Evaluation of the Frequency of Genetic Mutations in Leukemia.
              </a>
            </td>
          <td>
            A. Hamadi
          </td>
          <td>2025-06-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd84d5c237ed257f4d0a9b4baefbb0a60d2ecfd" target='_blank'>
              Unmasking Human T Cell Receptor Germline Diversity: 335 Novel Alleles Identified in 47 Pangenome Reference Individuals Using the gAIRR Suite
              </a>
            </td>
          <td>
            Yu-Hsuan Yang, C. Yao, Mao-Jan Lin, Kuan-Ta Huang, Yu-Hui Lin, I-Hsuan Chiu, Sheng-Kai Lai, Chih-Yeh Chen, Ya-Chien Yang, Chia-Lang Hsu, Jacob Shu-Jui Hsu, Chien-Yu Chen, Pei-Lung Chen
          </td>
          <td>2025-05-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8[Formula: see text] T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures.
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Somatic cell reprogramming into human induced pluripotent stem cells entails significant intracellular changes, including modifications in mitochondrial metabolism and a decrease in mitochondrial DNA copy number. However, the mechanisms underlying this decrease in mitochondrial DNA copy number during reprogramming remain unclear. Here we aimed to elucidate these underlying mechanisms. Through a meta-analysis of several RNA sequencing datasets, we identified genes responsible for the decrease in mitochondrial DNA. We investigated the functions of these identified genes and assessed their regulatory mechanisms. In particular, the expression of the thymidine kinase 2 gene (TK2), located in the mitochondria and required for mitochondrial DNA synthesis, is decreased in human pluripotent stem cells as compared with its expression in somatic cells. TK2 was significantly downregulated during reprogramming and markedly upregulated during differentiation. Collectively, this decrease in TK2 levels induces a decrease in mitochondrial DNA copy number and contributes to shaping the metabolic characteristics of human pluripotent stem cells. However, contrary to our expectations, treatment with a TK2 inhibitor impaired somatic cell reprogramming. These results suggest that decreased TK2 expression may result from metabolic conversion during somatic cell reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afec2f521a0fb9cff12157659a7e8ef543de136a" target='_blank'>
              Changes in mitochondrial thymidine metabolism and mtDNA copy number during induced pluripotency
              </a>
            </td>
          <td>
            Hyun Kyu Kim, Yena Song, Minji Kye, Byeongho Yu, Hyungkyu Choi, Sung-Hwan Moon, M. Lee
          </td>
          <td>2025-06-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824f0e3b0deb0f0f6058afcd801582f088b41b57" target='_blank'>
              Quantitative analysis of genetic interactions in human cells from genome-wide CRISPR-Cas9 screens
              </a>
            </td>
          <td>
            Maximilian Billmann, M. Costanzo, Mahfuzur Rahman, Katherine Chan, Amy Hin Yan Tong, Henry N. Ward, Arshia Z. Hassan, Xiang Zhang, , Thomas Rohde, A. H. Shaw, Catherine Ross, Jolanda van Leeuwen, Michael Aregger, Keith A. Lawson, Barbara Mair, P. Mero, Matej Ušaj, Brenda J. Andrews, Charles Boone, Jason Moffat, Chad L. Myers
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21690ca2e89ea2f16a09c71f18f6490b7208397c" target='_blank'>
              High Throughput Screening Identifies Small Molecules that Synergize with MRTX1133 Against Acquired Resistant KRAS G12D Mutated CRC
              </a>
            </td>
          <td>
            Natalie Thielen, Ning Wei, Emiko Nagai, Edward Chu, S. Kitamura, Chaoyuan Kuang
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is recognized as a multifactorial disease, where both genetic and environmental factors play critical roles in its development and progression. The identification of pathogenic germline variants has proven to be a valuable tool for early diagnosis, the implementation of surveillance strategies, and the identification of individuals at increased cancer risk. Next-generation sequencing (NGS) has facilitated comprehensive multigene analysis in both hereditary and sporadic cases of CRC. Patients and methods In this study, we analyzed 100 unselected Colombian patients with CRC to identify pathogenic (P) and likely pathogenic (LP) germline variants, classified according to the guidelines established by the American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology (AMP). Using the BoostDM artificial intelligence method, we were able to identify oncodriver germline variants with potential implications for disease progression. We assessed the model’s accuracy in predicting germline variants by comparing its results with the AlphaMissense pathogenicity prediction model. Additionally, a minigene assay was employed for the functional validation of intronic mutations. Results Our findings revealed that 12% of the patients carried pathogenic/likely pathogenic (P/LP) variants according to ACMG/AMP criteria. Using BoostDM, we identified oncodriver variants in 65% of the cases. These results highlight the significance of expanded multigene analysis and the integration of artificial intelligence in detecting germline variants associated with CRC. The average overall AUC values for the comparison between BoostDM and AlphaMissense were 0.788 for the entire BoostDM dataset and 0.803 for the genes within our panel, with individual gene AUC values ranging from 0.606 to 0.983. Functional validation through the minigene assay revealed the generation of aberrant transcripts, potentially linked to the molecular etiology of the disease. Conclusion Our study provided valuable insights into the prevalence and frequency of P/LP germline variants in unselected Colombian CRC patients through NGS. Integrating advanced genomic analysis and artificial intelligence has proven instrumental in enhancing variant detection beyond conventional methods. Our functional validation results provide insights into the potential pathogenicity of intronic variants. These findings underscore the necessity of a multifaceted approach to unravel the complex genetic landscape of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4d7e10934a5ab00c4eed85e52b6a8520151617" target='_blank'>
              Integrating next-generation sequencing and artificial intelligence for the identification and validation of pathogenic variants in colorectal cancer
              </a>
            </td>
          <td>
            Juliana Rodriguez-Salamanca, Mariana Angulo-Aguado, Sarah Orjuela-Amarillo, Catalina Duque, Diana Carolina Sierra-Díaz, Nora Constanza Contreras Bravo, Carlos Figueroa, Carlos Restrepo, Andrés López-Cortés, Rodrigo Cabrera, Adrien Morel, D. Fonseca-Mendoza
          </td>
          <td>2025-05-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Colorectal carcinogenesis involves dynamic interactions between genetic susceptibility and cellular heterogeneity, yet current studies rarely disentangle causal genes from passive associations. While GWAS have mapped numerous risk loci, only a minority colocalize with eQTL/pQTL. A multi-omics framework combining single-cell transcriptomics, transcriptomics, proteomics, and MR is urgently needed to resolve cell-type-specific drivers of colorectal cancer pathogenesis. Methods We integrated GWAS data, eQTL data, pQTL data, and single-cell RNA sequencing differential gene expression profiles from public databases. Subsequent batch Two-sample Mendelian randomization and further SMR analysis aimed to identify key genes in the pathogenesis of colorectal cancer. Results Cluster analysis identified 4909 DEGs across various cell types. We discovered that 428 DEGs had a causal association with colorectal cancer through eQTL, of which 38 genes met the FDR statistical standards, and four of these genes (CTSF, PCSK7, LYZ, LMAN2L) also had causal associations through pQTL. SMR analysis confirmed the reliability of PCSK7 as a disease target. Conclusion By integrating single-cell data, transcriptomic data, proteomic data and GWAS data for MR analysis, we identified CTSF, PCSK7, LYZ, LMAN2L as potential targets for colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02636-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19765189a791bce76c8d451a55c97d45a6c9279a" target='_blank'>
              Integrating single-cell with transcriptome-proteome Mendelian randomization reveals colorectal cancer targets
              </a>
            </td>
          <td>
            Song Wang, Xin Yao, Shenshen Li, Shanshan Wang, Xuyu Huang, Jing Zhou, Xiao Li, Jieying Wen, Weixuan Lan, Yunsi Huang, Hao Li, Yunlong Sun, Xiaoqian Zhao, Qiaoling Chen, Xuedong Han, Ziming Zhu, Xinyue Zhang, Tao Zhang
          </td>
          <td>2025-05-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 4116


 Background:
 Biliary tract cancers (BTC) are rare, highly aggressive malignancies with limited treatment options, leading to consistently poor outcomes. An improved understanding of BTC tumor evolution could inform enhanced screening strategies, identify useful prognostic markers, and discover novel therapeutic targets.
 Methods:
 We performed whole genome and transcriptome sequencing (WGTS) at high depth ( > 80X) on a prospective cohort of BTC tumors. After detecting mutations and mutational signatures, we developed two novel methods for driver identification and evolutionary reconstruction. First, we created new oncogene and tumor suppressor-specific models that integrate copy number profiles, structural variant breakends, and expression changes to distinguish structural drivers from neutral chromosomal rearrangements. Second, we applied population genetics techniques to fit demographic models to tumor allele frequencies across 20 paired primary and metastatic samples.
 Results:
 We analyzed 130 tumor samples from 110 patients, representing the largest BTC whole-genome cohort to date. We identified hypermutated tumors (50-150 mutations/mb) with distinct etiologies, including mismatch repair deficiency, platinum exposure, tobacco use, and aristolochic acid-related damage, the latter of which was associated with response to immunotherapy. Ourintegrated-driver approach identified an association between selection on
 RAD23A
 and an increased structural variant load, resulting in a tandem-duplicator-like mutational phenotype. This also revealed an underappreciated impact of
 SMAD4
 in BTC, which is inactivated through multiple mutation types in 12%. Notably,
 BAP1
 mutations occurred in 24% of cases, including 4% that were inactivated through deletion of the
 BAP1
 promoter that lowered transcript expression, a previously undescribed mechanism. By pairing clinical data to the genomics, we observed a co-occurrence of
 BAP1
 mutations and
 FGFR2
 fusions in small-duct, mass-forming intrahepatic cholangiocarcinomas. These mutations were mutually exclusive with
 TP53
 mutations, which were enriched in patients with primary sclerosing cholangitis. Finally, our novel method for subclonal population reconstruction on paired samples illuminated recent tumor evolutionary dynamics and identified an
 ARID1B
 fusion with a potential role in metastasis.
 Conclusions:
 This study uncovered novel genomic mechanisms underlying the evolutionary origins of BTC beyond those previously identified with exome and panel sequencing, highlighting the value of WGTS. These results highlight the complex genomic heterogeneity of BTC, with potential implications for precision therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bbdd5f85cf47a722dc40e8dbc7f432d73cb4b4" target='_blank'>
              Whole-genome sequencing of biliary tract cancer: Uncovering the genomic origins of evolutionary trajectories.
              </a>
            </td>
          <td>
            Felix Beaudry, Nicholas Light, Duhan Yendi, A. Zorigtbaatar, Maggie Hildebrand, A. Dodd, Roxana Bucur, Julie M. Wilson, Enrique Sanz Garcia, Xin Wang, Erica S. Tsang, G. O’Kane, S. Gallinger, Arndt Vogel, Jennifer J Knox, F. Notta, G. Sapisochin, Robert C Grant
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lanxin Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The circadian clock governs daily rhythms in numerous physiological processes through precise regulation of gene expression and biochemical functions. Dysregulation of the circadian rhythm has been implicated in carcinogenesis and cancer progression. However, the mechanisms by which the circadian clock influences cancer phenotype and chemotherapy resistance, particularly in cholangiocarcinoma (CCA), remain poorly understood. Using cell lines established from primary CCA and metastatic ascites of two male patients, we manipulated core clock genes (BMAL1, PER2, and NR1D1) to evaluate their effects on circadian rhythms. We analyzed alterations in circadian phenotypes at dynamic and single time points and assessed their impact on cancer-related phenotypic changes, including proliferation, apoptosis, cell cycle regulation, migration, invasion, and the expression of epithelial-to-mesenchymal transition (EMT) and cancer stem cell markers. Additionally, we examined the impact of circadian disruption on gemcitabine sensitivity. Genetic deletion of BMAL1, PER2, and NR1D1 disrupted circadian rhythm and significantly altered cancer phenotypes. Notably, BMAL1 and NR1D1 impairment exacerbated cell migration, invasion, and EMT activation in CCA cells. BMAL1 loss also induced gemcitabine resistance. In contrast, PER2 repression enhanced chemosensitivity and inhibited metastasis. The modulation of the circadian gene triggered phenotypic changes in CCA cells, indicating a crucial involvement of core-clock components in the pathological mechanisms hastening bile duct cancer malignancy. Our findings advance the understanding of regulating CCA malignancy and may offer a novel target for its treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63df1b948313a08b9d2b74bdfc742ccab84a19a9" target='_blank'>
              Dysfunction of core clock genes regulates malignant phenotype and gemcitabine sensitivity of cholangiocarcinoma cells.
              </a>
            </td>
          <td>
            Yin Li, Aimin Zheng, Yangang Cui, Tianbao Liu
          </td>
          <td>2025-06-17</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c58caf03a35382e82dc43ac8f2167d8ee1ee67" target='_blank'>
              X-chromosome upregulation operates on a gene-by-gene basis at RNA and protein levels
              </a>
            </td>
          <td>
            Ryan N. Allsop, Jeffrey Boeren, Beatrice F. Tan, Sarra Merzouk, Suresh Poovanthingal, W. V. van Ijcken, Jeroen Demmers, Hegias Mira Bontenbal, C. Gontan, J. Gribnau, V. Pasque
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Essential transcription and cell cycle progression controllers are CDKs, whose dysregulation is a defining trait of many human cancers. CDKs have grown to be very crucial therapeutic targets in cancer. Although traditional CDK inhibitors have demonstrated therapeutic efficacy, they frequently encounter limitations due to resistance mechanisms and off-target effects. Recent developments in targeted protein degradation, like proteolysis-targeting chimeras (PROTACs) and molecular glues, offer creative ways to destroy CDK proteins specifically. These techniques reduce scaffolding activities and slow down kinase activity, hence more completely blocking oncogenic CDK signaling. This paper highlights the clinical and preclinical developments of PROTACs and molecular glues, investigates the current CDK-targeting therapeutic landscape, and studies the molecular basis of CDK dysregulation in cancer. We also address their benefits over conventional inhibitors, current issues, and possibilities for inclusion into precision oncology. These new approaches taken together represent a change in CDK-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1262893392840549bf93c0b9f7515437c79c1d7" target='_blank'>
              Targeting CDKs in cancer therapy: advances in PROTACs and molecular glues
              </a>
            </td>
          <td>
            Hany E. Marei, Khaled Bedair, Anwarul Hasan, Layla Al-mansoori, Alice Gaiba, Andrea Morrione, C. Cenciarelli, Antonio Giordano
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Objective DDX17, an ATP-dependent RNA/DNA helicase, is implicated in the regulation of RNA metabolism and has been linked to tumorigenesis and metastasis in various cancers. While studies have explored the role of DDX17 in specific cancers, further research is needed to understand its mechanisms across different cancer types. Methods We leveraged several public databases, including TIMER, The Cancer Genome Atlas (TCGA), and the Genotype-Tissue Expression (GTEx), to investigate DDX17 mRNA expression across 33 types of tumors. The GEPIA2 database was utilized to assess the impact of DDX17 on overall survival (OS) and disease-free survival (DFS) in patients with these tumors. Additionally, we employed cBioPortal to examine DDX17 gene alterations in various tumor tissues. Further analysis was conducted using the R language to explore the correlation between DDX17 and a range of clinical features, including tumor microenvironment (TME), immune regulatory genes, immune checkpoints, tumor mutational burden (TMB), microsatellite instability (MSI), DNA methylation, RNA methylation, and drug sensitivity. Gene Set Enrichment Analysis (GSEA) was also applied to elucidate the molecular mechanisms mediated by DDX17. Results DDX17 showed significant differences in expression between cancer and normal tissues. Expression of DDX17 was associated with patient prognosis, TMB, MSI, and drug sensitivity in certain cancers. DDX17 is additionally involved in modulating immune functions and influencing the tumor microenvironment. Further analysis of DDX17 mutation sites and types showed that the mutation frequency was highest in endometrial cancer and the major mutations of DDX17 were missense mutations. Conclusion These findings indicated that DDX17 may be considered a potential prognostic biomarker and a promising target for novel immunotherapeutic approaches in cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02955-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c500003015dd54739438713ee9cd7419ce63da8" target='_blank'>
              Prognostic and immunological landscape of DDX17 in pan-cancer analysis: a comprehensive study
              </a>
            </td>
          <td>
            Yu Lei, Xin He, Peng Chen, Yiping Qin, Bin Ge, Yan Liu, Pu Li, Xing Wei
          </td>
          <td>2025-06-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (iPSCs) have transformed biomedical research by enabling the generation of diverse cell types from accessible somatic tissues. However, certain fundamental biological properties, such as the genetic and epigenetic determinants of iPSC proliferation, remain poorly characterized. We measured the growth of iPSC lines derived from 602 unique donors using high-throughput time-lapse imaging, quantified proliferation through a growth Area-Under-the-Curve (gAUC) phenotype, and correlated gAUC with the gene expression and genotype of the cell lines. We identified 3,091 genes associated with gAUC, many of which are well established regulators of cell proliferation. We also found that rare deleterious variants in WDR54 were associated with reduced iPSC growth and that WDR54 was differentially expressed with respect to gAUC. Although no common variants showed a genome-wide association with gAUC, iPSC lines from monozygotic twins were highly correlated, and common genetic variation explained approximately 71-75% of the variance in iPSC growth rates. These results indicate a complex genetic architecture of iPSC growth rates, where rare, large-effect variants in important growth regulators, including WDR54, are layered onto a highly polygenic background. These findings have important implications for the design of pooled iPSC-based studies and disease models, which may be confounded by intrinsic growth differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46302769c03d6ab5c0d658d81d85d4ec438bae8" target='_blank'>
              Genetics of growth rate in induced pluripotent stem cells
              </a>
            </td>
          <td>
            Brian N. Lee, Henry J. Taylor, Filippo Cipriani, Narisu Narisu, Catherine C. Robertson, A. Swift, Neelam Sinha, Tingfen Yan, L. Bonnycastle, Nathan Dale, Annie Butt, Hemant Parsaud, Stefan Semrau, Joshua W Knowles, Ivan Carcamo-Orive, Agnieszka D’Antonio-Chronowska, K. Frazer, Leslie G. Biesecker, S. Noggle, Michael R. Erdos, Dan Paull, Francis S. Collins, D. L. Taylor
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Simple Summary High-risk human papillomaviruses (HR-HPVs), especially types 16 and 18, contribute to the development of various cancers by integrating their genomes into the host DNA, activating oncogenes or disrupting tumor suppressor genes. This review discusses the limitations of short-read sequencing (SR-Seq) for integration analyses and emphasizes the importance of using long-read sequencing (LR-Seq) technologies, which enable high-resolution mapping of complex integration events, including those previously inaccessible for analysis, i.e., repeat-rich genomic regions. This review highlights recent findings from LR-Seq studies on HPV integration and compares its mechanisms with those used by other viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bd3d524ef4cdc3c87b3f69c07be4376d8e44c3" target='_blank'>
              Novel Avenues for the Detection of Cancer-Associated Viral Genome Integrations Using Long-Read Sequencing Technologies
              </a>
            </td>
          <td>
            Larissa-Anna Bergmann, Alicja Pacholewska, Michal R. Schweiger
          </td>
          <td>2025-05-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Castration-resistant prostate cancer (CRPC) remains the leading cause of mortality among prostate cancer patients. While second-generation androgen receptor (AR) pathway-targeted therapies, such as Abiraterone, have significantly improved survival outcomes, resistance to these treatments ultimately emerges, posing a critical challenge. Understanding the mechanisms underlying Abiraterone resistance is essential for developing strategies to enhance patient outcomes. In this study, a super-enhancer (SE)-driven transcriptional regulatory circuit is identified involving BCL6, NFIB, and SMAD3 that facilitates Abiraterone resistance in CRPC. Through comprehensive analyses of SE expression profiles in Abiraterone-resistant CRPC cells and their parental counterparts, it is revealed that this circuit plays a pivotal role in resistance progression. Mechanistically, BCL6, NFIB, and SMAD3 synergistically remodel the transcriptional landscape of resistant CRPC cells, driving resistance by regulating cholesterol biosynthesis and cell cycle pathways. The findings provide critical insights into the transcriptional dysregulation underlying Abiraterone resistance and highlight potential therapeutic strategies to counteract treatment resistance in CRPC, ultimately aiming to improve patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5333f40f3a2272e85aae404e2fe1d3f4d247124b" target='_blank'>
              A Super-Enhancer-Driven Transcriptional Regulatory Circuit Underlying Abiraterone Resistance in Castration-Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Liling Jiang, Jiamin Wang, Guanjie Peng, Haichuan Zhang, Jinxin Fang, Yingyin Gao, Enzhe Lou, Yangzhou Liu, Wa Ding, Bingyuan Liu, Qiong Mao, Lizhen Jiang, Aochu Liu, Xinyue Li, Shiwen Hu, Qiaomin Ma, Yueyuan Zheng, Zhigang Zhao, Xianping Shi
          </td>
          <td>2025-06-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) remains one of the most aggressive and therapeutically challenging subtypes of BC due to its lack of targeted treatment options and high plasticity. Increasing evidence suggests that epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNA regulation, play a pivotal role in facilitating TNBC phenotype by influencing the expression of genes involved in subtype reprogramming. Modulating these epigenetic marks offers a promising strategy to drive TNBC subtype conversion toward less aggressive forms that are more responsive to treatment, thereby enhancing therapeutic efficacy. By targeting enzymes such as DNA methyltransferases and histone modifiers using epigenetic drugs (epidrugs), TNBC cells can be re-sensitized to hormone therapy, chemotherapy, and targeted treatments. This review delves into the role of epigenetic modulation in harnessing the plasticity of TNBC to drive its conversion into subtypes with better prognoses and discusses problems and limitations associated with the use of epidrugs in this context. By promoting subtype conversion, epidrugs offer a promising strategy for providing more personalized and effective therapies for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ada35a90892ec0be34375adfcf309b46149ff70" target='_blank'>
              Epigenetic Therapeutics: Reprogramming Triple-Negative Breast Cancer into Responsive Subtypes.
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, Aya Mudhafar Al-Azawi, L. Lozon, Rawan R. Kawaf, Ragheb Alsheikh Zein, Yahia El-Gharib, Raafat El-Awady
          </td>
          <td>2025-07-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a clonal malignancy of myeloid progenitor cells that demonstrates highly variable responses to current regimens, highlighting the need for precision medicine. However, reliable biomarkers for precision medicine treatment remain elusive due to cellular heterogeneity. Conventional Models based on bulk RNA sequencing and ex vivo assays often fail to capture the intricate molecular pathways and gene networks that underlie treatment response and resistance. Here, NetAML, a novel network-based precision medicine platform that systematically develops 87 drug sensitivity prediction models for 87 clinical drugs using ex vivo drug responses from 520 AML patients with RNA-Seq is presented. The approach leverages network-based analysis and machine learning to identify biologically interpretable gene signatures that capture the complex molecular interactions driving differential drug responses. Notably, the signature genes derived from the models reveal distinct cellular mechanisms. For instance, the co-expression of C19ORF59 and FLT3 is associated with resistance to FLT3 inhibitors. In summary, NetAML offers a powerful strategy for personalized AML treatment by constructing drug-specific models, identifying clinically actionable biomarkers, and supporting the development of optimized, patient-specific therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b29cce61ad93b17c3f11650bbf2734b68498fe27" target='_blank'>
              A Network-Driven Framework for Drug Response Precision Prediction of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Yinyin Wang, Rui Liu, Yinnan Zhang, Xiang Luo, Chengzhuang Yu, Shentong Fang, Ninghua Tan, Jing Tang
          </td>
          <td>2025-07-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lynch syndrome (LS) is the most common inherited disorder predisposing individuals to colorectal cancer (CRC). It results from germline defects in DNA mismatch repair (MMR) genes, which are critical for maintaining genomic integrity. Here, we demonstrate that the expression of the MMR genes MLH1 and PMS2 are significantly reduced in colon tumor tissues from a proband with CRC, potentially resulting from inherited mutations in the MLH1 gene in LS cases. We identified a previously unreported in-frame deletion mutation in the MLH1 gene, classified as a variant of uncertain significance (VUS) due to its undefined role in oncogenesis. The majority of functionally inactive mutants were located in the residues Phe614 to Lys617, which form crucial hydrogen bonds. Taken together, our data reveals a correlation between this mutation and increased susceptibility to LS-associated tumors. The study offers a valuable insight for evaluating cancer susceptibility in carriers of MLH1 mutants, potentially elucidating the functional roles of MLH1 in oncogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14554-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a32f988e02c1ad79822d81fa1f876d387930af3" target='_blank'>
              An in-frame deletion mutation in MLH1 drives Lynch syndrome-associated colorectal cancer
              </a>
            </td>
          <td>
            Xiaojuan Lin, Ce Bian, Dongni Liang, Lin Li, Qingqing Tang, Qingli Li
          </td>
          <td>2025-07-11</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant DNA methylation is a hallmark of colorectal cancer (CRC), contributing to tumor progression through the silencing of tumor suppressor genes and activation of oncogenes. Indicaxanthin (IND), a dietary betalain pigment from Opuntia ficus indica, has shown antiproliferative effects in CRC models, yet its epigenetic impact remains unexplored. In this study, we investigated the effects of IND on the methylome of Caco-2 cells using Reduced Representation Bisulfite Sequencing (RRBS). IND induced a global hypermethylation profile, particularly at gene promoters and CpG islands. Among the differentially methylated genes, 60% were protein-coding, and 10% encoded transcription factors, including PAX5 and TFAP4, both hypermethylated at active enhancers. Functional enrichment analysis revealed pathways beyond canonical intestinal functions, suggesting altered cell identity and plasticity. Transcription factor targets (SOX10, NFKB1, AHR, ARNT) were significantly enriched among the affected genes, several of which are involved in transdifferentiation processes. Methylation changes also indicated potential reprogramming toward epithelial cell types from pulmonary or neuroectodermal origin. Moreover, IND induced selective hypomethylation of Alu elements on chromosome 21 and hypermethylation of rDNA loci, hinting at suppressed ribosomal biogenesis. Overall, these findings highlight the epigenetic remodeling potential of IND and its possible role in modulating cell fate and metabolism in CRC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aad87b105620f84af351d2656d30853c48a91d89" target='_blank'>
              Epigenetic Remodeling of Regulatory Regions by Indicaxanthin Suggests a Shift in Cell Identity Programs in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            M. Ragusa, C. Gentile, Aldo Nicosia, Salvatore Costa, Sara Volpes, Laura Greco, F. Naselli, Fabio Caradonna
          </td>
          <td>2025-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad770e71268541d6923a6a441040fe4dc007ffe" target='_blank'>
              Multi-selective RAS(ON) Inhibition Targets Oncogenic RAS Mutations and Overcomes RAS/MAPK-Mediated Resistance to FLT3 and BCL2 Inhibitors in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bogdan Popescu, Matthew F. Jones, Madison Piao, Elaine Tran, Andrew Koh, Isabelle Lomeli, C. Peretz, Natalia Murad, Sydney Abelson, Carolina E. Morales, Jose M. Rivera, Y. Pikman, Michael L. Cheng, Aaron C. Logan, Elliot Stieglitz, Catherine C. Smith
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f2b837ac11b06bf8917b835488e0f8a39575dc9" target='_blank'>
              Post-translational modifications in DNA damage repair: mechanisms underlying temozolomide resistance in glioblastoma.
              </a>
            </td>
          <td>
            Yike Chen, Kaikai Ding, Shuyu Zheng, Songting Gao, Xiaohui Xu, Haijian Wu, Fengqi Zhou, Yongjie Wang, Jinfang Xu, Chun Wang, C. Ling, Jing Xu, Lin Wang, Qun Wu, G. Giamas, Gao Chen, Jianmin Zhang, Chenggang Yi, Jianxiong Ji
          </td>
          <td>2025-05-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the p53 gene are frequently observed in various cancers, prompting the initiation of efforts to restore p53 function as a therapeutic approach several decades ago. Nevertheless, only a limited number of drug development initiatives have progressed to late-stage clinical trials, and to date, no p53-targeted therapies have received approval in the USA or Europe. This situation can be attributed primarily to the characteristics of p53 as a nuclear transcription factor, which lacks the conventional features associated with drug targets and has historically been considered “undruggable”. In recent years, however, several promising strategies have emerged, including the enhanced iterations of previous approaches and novel techniques aimed at targeting proteins that have traditionally been considered undruggable. There is a growing interest in small molecules that can restore the tumor-suppressive functions of mutant p53 proteins, and the development of drugs specifically designed for particular p53 mutation types is currently underway. Other approaches aim to deplete mutant p53 or exploit vulnerabilities associated with its expression. Additionally, genetic therapy strategy and approaches have rekindled interest. Advances in mutant p53 biology, compound mechanisms, treatment modalities, and nanotechnology have opened up new avenues for p53-based therapies. However, significant challenges remain in clinical development. This review reassesses the progress in targeting p53-mutant cancers, discusses the obstacles in translating these approaches into effective therapies, and highlights p53-based therapies via nanotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811541704b1e89bbef8a11450dee5f1b59768d70" target='_blank'>
              Discovery of Drugs Targeting Mutant p53 and Progress in Nano-Enabled Therapeutic Strategy for p53-Mutated Cancers
              </a>
            </td>
          <td>
            Na Zhang, Zhiyuan Jing, Jie Song, Qiyue Liang, Yuxue Xu, Zhaowei Xu, Longping Wen, Pengfei Wei
          </td>
          <td>2025-05-26</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Imprinted genes are epigenetically regulated in normal tissues to follow monoallelic expression according to the parent of origin of each allele. Some of these patterns are dysregulated in cancer.


RESULTS
We developed a novel computational multi-omic pipeline to evaluate monoallelic and biallelic expression patterns based on matched RNA-seq expression data, whole-exome sequencing information, and copy number data. We analyzed allelic expression of the entire genes, individual isoforms, and each exon of 59,283 autosomal protein-coding and ncRNA genes, with a focus on 94 genes previously reported to be imprinted. We analyzed 108 cell lines from 9 different tumor histologies using molecular data from the DepMap Portal for the Cancer Cell Line Encyclopedia. Allelic expression patterns of imprinted genes and isoforms in tumor cells were variable. We also identified additional genes and isoforms with predominantly monoallelic expression due to a variety of potential mechanisms. We provide a novel public dataset of transcriptome-wide allelic expression patterns in cell lines from diverse tumor categories, which can serve as a resource for future cancer studies. We examined associations of in vitro cell line response to antitumor agents and repurposed drugs with allelic patterns and overall levels of isoform expression of imprinted genes and of additional genes with predominantly monoallelic expression. Drug response was associated with isoform expression patterns of multiple imprinted genes including CPA4, DGCR6, DNMT1, GNAS, GRB10, H19, NAA60, OSBPL5, PHACTR2, and ZFAT, predominantly monoallelically expressed MAP2K5 and BCLAF1, and additional predominantly monoallelically expressed genes. Multiple associations may be related to mechanisms of drug activity, including associations between the response to the DNA damaging agents and allelic expression of ZFAT, CDC27, and BCLAF1 isoforms, and the response to inhibitors of multiple signaling pathways with expression patterns of GNAS isoforms.


CONCLUSIONS
Tumor cells have a range of monoallelic and biallelic expression patterns in both imprinted and non-imprinted genes and are likely affected by the complex interplay among changes in allelic expression, sequence variants, copy number changes, and expression changes of biologically important genes. Multiple isoform-specific patterns of allelic expression were associated with drug response, indicating complex mechanisms of cancer chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e53122e0648c0b26f74137cf51cf8b0a5877a45c" target='_blank'>
              Allelic expression patterns of imprinted and non-imprinted genes in cancer cell lines from multiple histologies
              </a>
            </td>
          <td>
            Julia Krushkal, Travis L. Jensen, George Wright, Yingdong Zhao
          </td>
          <td>2025-05-25</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a deadly disease known for its genetic heterogeneity. LTR12C is an endogenous retrovirus-derived regulator of pro-apoptotic genes and is normally silenced by epigenetic regulation. In this study, we found that the treatment of two glioblastoma cell lines, T98-G and U87-MG, with DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors activated LTR12C expression. Combined treatment with these epigenetic drugs exerted a synergistic action on the LTR12C activation in both cell lines, while treatment with each drug as a single agent had a far weaker effect. A strong induction of the expression of the TP63 gene was seen in both cell lines, with the pro-apoptotic isoform GTA-p63 accounting for most of this increase. Coherently, downstream targets of p63, such as p21 and PUMA, were also induced by the combined treatment. Furthermore, we observed a significant reduction in the GBM cell growth and viability following the dual DNMT/HDAC inhibition. These findings reveal that the reactivation of LTR12C expression has the potential to modulate survival pathways in glioblastoma and provide information regarding possible epigenetic mechanisms that can be used to treat this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b329500c4414f15e1eee2caec6a0bfaed4d1e7f7" target='_blank'>
              Functional Characterization of LTR12C as Regulators of Germ-Cell-Associated TA-p63 in U87-MG and T98-G In Vitro Models
              </a>
            </td>
          <td>
            Lucia Meola, Sohum Rajesh Shetty, A. Peschiaroli, Claudio Sette, Camilla Bernardini
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1edd21201b5143958af0631e20736d1ea970c7b" target='_blank'>
              Heterogeneity of Genetic Sequence within Quasi-species of Influenza Virus Revealed by Single-Molecule Sequencing
              </a>
            </td>
          <td>
            Kenji Tamao, Hiroyuki Noji, K. Tabata
          </td>
          <td>2025-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8860f8e38f6028b324d356789271bd3f9604af9d" target='_blank'>
              Transcriptional Biomarkers in the Diagnosis of Genetic Disorders: Opportunities, Challenges, and Prospects for Application.
              </a>
            </td>
          <td>
            Lidia N. Nefedova, Tatiana N. Krasnova
          </td>
          <td>2025-06-01</td>
          <td>Biochemistry. Biokhimiia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290d92451b26f978216d79d0c80c8e4f6746115f" target='_blank'>
              LaGrACE: estimating gene program dysregulation with latent regulatory network.
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P.V. Benos
          </td>
          <td>2025-06-30</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Although genetic variants associated with bladder cancer (BCa) risk have been identified through hypothesis-driven and genome-wide association studies, a systematic understanding of BCa genetic susceptibility at the gene and pathway levels remains to be achieved.



 In this 2-stage functional genomics study, we used 5 independent tools for genome-wide gene mapping and ranking based on BCa genome-wide association studies summary statistics, followed by a meta-analysis of gene-level significance p values, to obtain a consensus gene ranking in terms of association with BCa. Subsequently, we performed preranked gene-set enrichment analysis to identify the functional pathways involved in BCa genetic susceptibility. Joint analysis with gene-set enrichment analysis, based on somatic alteration frequency, was performed to explore the pathway-level relationships between genetic susceptibility and somatic alterations in BCa.



 Other than the well-known BCa genes (such as FGFR3, MYC, TERT, CCNE1, and TP63), we additionally prioritized a set of novel genes likely to be genetically implicated in BCa development, including SETD2, a possible tumor suppressor gene involved in chromatin remodeling. We further demonstrated convergence between genetic associations and somatic alterations at both the gene (eg, FGFR3 and TERT) and pathway levels (eg, cell cycle and chromatin modification), as well as functional ontologies specifically implicated in germline predisposition to BCa (eg, CD8/TCR signaling, immune checkpoints, and cytokine signaling).



 We identified several novel genes associated with BCa and demonstrated that genetic variants contribute to the development of BCa by affecting antitumor immunity, response to toxic exposure, and RNA and protein homeostasis and synergizing with somatic alterations in various cancer-related pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff12d27723662d274dee934c75d998f194be948" target='_blank'>
              Gene and pathway analysis of genome-wide genetic associations of bladder cancer
              </a>
            </td>
          <td>
            Mingjun Shi, Xiangyu Meng, Xuan Xu, Qiaoli Wang
          </td>
          <td>2025-06-05</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Chromothripsis, a phenomenon of massive genomic rearrangements, introduces extensive mutations in critical genes, affecting cell survival and apoptosis. Among these genes, the BCL2 (B-cell lymphoma 2) gene, which plays a crucial antiapoptotic role in cancer cells, is often subjected to significant alterations. Here, we present a computational pipeline to model and analyze the structural and functional impacts of chromothripsis-induced Single-Nucleotide Polymorphisms (SNPs) within the BCL2 gene. This pipeline integrates mutation simulation, homology modeling, and protein inter-action analysis to evaluate the stability and apoptotic potential of BCL2 mutations. These results indicate that chromothripsis-induced mutations can destabilize the Bcl-2 protein, thereby disrupting its binding affinity with apoptotic regulators, such as Bax. These findings support the potential of ergodic anticancer therapy to exploit such mutations, facilitating the apoptosis of cancer cells. Our computational model offers a novel in silico approach for understanding mutation-driven alterations in cancer biology, aiding the development of therapeutic strategies targeting apoptotic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8abc547c111e4d9c763f84a8f9da96c16d5a06c5" target='_blank'>
              Breaking Boundaries in Cancer Therapy: Harnessing Chromothripsis- Induced Mutations for Targeted BCL2 Protein Destabilization.
              </a>
            </td>
          <td>
            Sergey Shityakov, Michael Nosonovsky, Ekaterina V. Skorb, Viacheslav Kravtsov
          </td>
          <td>2025-06-02</td>
          <td>Current drug discovery technologies</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="There has been a substantial investment in elucidating the mechanism of expansion in hopes of identifying therapeutic targets for Huntington disease (HD). Although an expanded CAG allele is the causal mutation for HD, there is evidence that somatic expansion may not be the only disease driver. We report here that double strand breaks (DSBs) drive HD toxicity by an independent mechanism from somatic expansion. The mutant HD protein inhibits non-homologous end joining (NHEJ) activity, leading to the accumulation of DSBs. DSBs promote transcriptional pathology in mice that cannot expand their CAG tracts somatically. Conversely, Inhibition of DSBs reverses neuronal toxicity in animals that undergo somatic expansion. Although they coexist in neurons, DSBs and somatic expansion are independent therapeutic targets for HD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98e2f4c3e43c43464e34f5bd46068f4d423e6b46" target='_blank'>
              Double strand breaks drive toxicity in Huntington’s disease mice with or without somatic expansion
              </a>
            </td>
          <td>
            Aris A. Polyzos, Ana Cheong, Jung Hyun Yoo, Lana Blagec, Zachary D. Nagel, Cynthia T McMurray
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c635cae84c7bdc8795ab539e7b406a70e03c5a4" target='_blank'>
              A transient mutational burst occurs during yeast colony development.
              </a>
            </td>
          <td>
            Nicolas Agier, Nina Vittorelli, Louis Ollivier, Frédéric Chaux, Alexandre Gillet-Markowska, Samuel O'Donnell, Fanny Pouyet, G. Fischer, Stéphane Delmas
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e17020


 Background:
 Testicular germ cell tumors (TGCT) are the most common malignancy in young adult men, with a globally increasing incidence. While these tumors are highly curable due to their exceptional sensitivity to cisplatin (cDDP), a proportion of patients develops a resistance to this chemotherapy, which results in disease progression and patient death. The cause of cisplatin resistance in TGCT has not been elucidated yet and is supposed to be multifactorial.
 Methods:
 We conducted a CRISPR Knockout (KO) screening in a TGCT cell line (NCCIT) using a lentiviral sgRNAs library (Addgene #101926) targeting 3015 genes involved in cell cycle regulation and cancer development. The screening population was cultured with or without cDDP (1.25 uM) for 7 days. Samples of treated and control cells were harvested before and after cDDP exposure, the sgRNA cassettes were sequenced by massive parallel sequencing, and the differences in sgRNA abundance were determined using the MAGeCK/MAGeCK-Flute pipeline.
 Results:
 The CRISPR-KO screening data analysis identified 59 gene KOs that were significantly positively selected and 7 gene KOs that were significantly negatively selected after cDDP exposure. Among the top positive hits were genes involved in the regulation of apoptosis (
 BMF, CDIP1
 and
 BAX
 ), epigenetic modifiers (
 HDAC8, SSX2
 ), G-protein regulators (
 ADCY8, GNG10
 ), and others (e.g.
 MAGEA2, ACADM, LAMP1
 ) — inactivation of these genes potentially contributes to the development of cDDP resistance. Among the top negative hits were genes implicated in DNA damage repair (
 FANCB
 ,
 ERCC5
 ), sphingolipid metabolism (
 KDSR
 ), and cytokinesis (
 SPECC1L
 ) — inactivation of these genes may increase sensitivity to cDDP.
 Conclusions:
 The results highlight several potential mechanisms of cDPP resistance in TGCT. Some of these genes and pathways are well known and have been previously described in association with platinum resistance in malignant tumors, while others — such as the G-protein regulators, sphingolipid synthesis, and
 SPECC1L
 — are novel in this context and may reflect the unique biology of TGCT progression. SPECC1L is involved in spindle organization and microtubule stabilization; its paralog SPECC1 is highly expressed in testis. Polyenoic sphingolipids are essential for the completion of meiosis in male germ cells and spermatozoa production.

 ADCY8 controls the production of cAMP, which is a key signaling molecule in spermatogenesis. They have also been described in association with malignant proliferation and progression. The identified hits will be validated in subsequent experiments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff267218117b8545c90f6a863002128909b0591f" target='_blank'>
              Use of CRISPR-KO screening to identify novel genes and pathways potentially contributing to cisplatin resistance in testicular germ cell tumors.
              </a>
            </td>
          <td>
            Laura Matouskova, Kristyna Soffrova, V. Bakardjieva-Mihaylova, J. Stuchlý, L. Boublikova, Karolina Skvarova
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5b8cce1b576a2635e4230be0b09f8a3f1a1a138" target='_blank'>
              Metabolite correlation networks reveal complex phenotypes of adaptation-driving mutations
              </a>
            </td>
          <td>
            Sharon Samuel, Weronika Jasinska, Ezra Sternlicht, Matvey Nikelshparg, D. Kleiner, S. Pilosof, Shimon Bershtein
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer (BC) is a highly prevalent malignancy in women and is often resistant to available therapies, calling for urgent investigation of the molecular mechanisms underlying its pathogenesis and progression. BC is thought to result from a complex interplay between genetic and environmental factors. Among key factors, chronic stress has been associated with worse cancer outcomes and can profoundly impact the epigenome. However, both stress and BC phenotypes are complex and heterogeneous, making studies that examine their molecular links challenging. Despite their heterogeneity, stressors trigger a neuroendocrine response that in humans culminates in the release of cortisol, a highly lipophilic hormone that traverses essentially every cell and induces widespread genomic effects. Modeling such effects at the epigenetic level, here we examine whether cellular DNA methylation (DNAm) markers of chronic stress – derived from human fibroblasts undergoing prolonged exposure to physiological stress cortisol levels – distinguish BC phenotypes in two independent human cohorts. Our results show that methylomic signatures of stress are consistently higher in tumor samples as compared to normal samples and in more advanced tumor stages and grades. Follow-up analyses further identify specific DNAm sites driving these associations, which are significantly enriched for cell adhesion pathways in both cohorts. These findings provide insights into the molecular mechanisms linking stress with BC and a proof-of-concept for utilizing cell model-derived disease biomarkers in environmental epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1693e248c789481400e0b6452a28ce9652ad34f" target='_blank'>
              Cell-Derived Epigenomic Markers of Chronic Stress Distinguish Breast Cancer Phenotypes
              </a>
            </td>
          <td>
            Amrita Acharya, Anthony S. Zannas
          </td>
          <td>2025-06-01</td>
          <td>The Yale Journal of Biology and Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abbd185e23d00e7fc13444a119ca584b80d51fd3" target='_blank'>
              LSINCT5: a pivotal oncogenic long non-coding RNA in cancers.
              </a>
            </td>
          <td>
            Sharanya K Sritharan, Amar Harris Arifin, S. Salvamani, R. Koh, B. Gunasekaran
          </td>
          <td>2025-06-02</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: Combinations of endocrine therapy and CDK4/6 inhibitors (CDK4/6i) used in metastatic breast cancer (MBC) result in heterogenous clinical benefit with subsets of patients having disease control ranging anywhere from a few months to more than 5 years. Understanding the mechanisms that account for these differences can enable new pharmacologic approaches to convert more of these cancers into the long-term control subset. Methods: To elucidate the causal factors that underlie the observed clinical variability, we have utilized multi-omic analyses of patient samples as well as experimental modeling using a diverse cohort of ER+ breast cancers. For clinicogenomic analyses, we analyzed 467 patients with ER+/ HER2- MBC treated with first-line CDK4/6i at our institution (MSK) with detailed clinical follow annotation and follow up. For genomic/transcriptomic studies, we analyzed single cell RNA sequencing data, whole genome sequencing data, and Ki67 score from the neoadjuvant CDK4/6i containing arms in the FELINE trial (NCT02712723). Patient derived xenografts, patient derived organoids, isogenic cell lines were used to define the underlying mechanisms. Live cell imaging assays were performed by FUCCI cell cycle sensor to assess cell cycle transitions. Results: Of 467 patients from MSK cohort, 129 (27.6%) had pre-CDK4/6i loss-of-function variants in TP53, corresponding to short PFS (median PFS= 8.7 months, 95% CI: 6.6–10.3; HR = 2.12, 95% CI: 1.68 to 2.68; p = 2.64e-10). This finding was validated in two large independent clinical cohorts outside MSK. In patient derived and isogenic cell lines, p53 loss had no impact on initial drug response but promoted long-term cell outgrowth and prevented emergence of senescence phenotypes. Following cells in detailed time courses revealed the propensity of p53 mutant but not wild type cells to reenter the cell cycle after initial arrest by CDK4/6 inhibition. Gene expression analysis of RNA seq data revealed a significant positive association between TP53 mutation status and DREAM complex genes, assessed by GSEA (NES= 1.51, p = 1e-4) In models, deletion of DREAM complex component, p130-Rbl2 was found to fully recapitulate the effects of p53 loss: promoting long-term outgrowth from CDK4/6i, blocking therapy induced senescence, and promoting cell cycle re-entry. To validate these results, we analyzed the multi-omic and histologic characteristics of serial samples from the FELINE trial. Compared with WT cases, baseline TP53 LOF had a higher amount of proliferating cells at the end of treatment, measured by Ki67 > 10% (4.5% of WT cases vs. 60% of mutant cases, respectively, OR = 26.8, p = 0.014), the per cell proliferation score, measured by ssGSEA (log2FC: 0.2 vs. 2.8, respectively) as well as a higher pseudobulk expression of DREAM complex related genes (GSEA: NES = 2.0, q = 3.7e-4). Finally, to identify strategies that might facilitate stable cell cycle control of p53KO tumors, we investigated which targets of p53 regulate DREAM and identified p21. Given role of p21 in regulating both CDK4/6 and CDK2 we investigated whether CDK2 inhibitors would provide additional benefit to CDK4/6i. Addition of CDK2i was able to fully restore efficacy in p53KO lines as well as a patient derived model with TP53 mutation enabling irreversible cell cycle arrest and long-term growth suppression. Conclusion: Wild type p53 facilitates DREAM complex assembly in response to CDK4/6i in ER+ breast cancer, underlying long-term disease control. p53 loss enables cells to reenter the cell cycle and facilitates the development of CDK4/6i resistance. Combined inhibition of CDK2 with CDK4/6 represents a promising therapeutic strategy for overcoming loss of p53 and enabling long-term antitumor efficacy.
 Citation Format: Rei Kudo, Anton Safonov, Enrico Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, Edaise M. da Silva, Qing Li, Marie Will, Atsushi Fushimi, Harikrishna Nakshatri, Andrew Koff, Britta Weigelt, Qamar J. Khan, Pedram Razavi, Sarat Chandarlapaty. DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c291e911332efcfe859d324b671af4bba597c2" target='_blank'>
              Abstract PS13-04: DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition
              </a>
            </td>
          <td>
            Rei Kudo, A. Safonov, E. Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, E. D. da Silva, Qing Li, Marie Will, A. Fushimi, H. Nakshatri, Andrew Koff, B. Weigelt, Qamar J Khan, P. Razavi, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BRAF and MEK inhibitors (BRAF/MEKi) have radically changed the treatment landscape of advanced BRAF mutation-positive tumours. However, limited efficacy and emergence of drug resistance are major barriers for successful treatments. Here, by using relevant preclinical models, we find that Connexin43 (Cx43), a protein that plays a role in cell-to-cell communication, enhances the effectiveness of BRAF/MEKi by recruiting DNA repair complexes to lamin-associated domains and promoting persistent DNA damage and cellular senescence. The nuclear compartmentalization promoted by Cx43 contributes to genome instability and synthetic lethality caused by excessive DNA damage, which could provide a therapeutic approach for these tumours to overcome drug resistance. Based on these findings, we designed a drug combination using small extracellular vesicles (sEVs) to deliver the full-Cx43 in combination with the BRAF/MEKi. This study reveals Cx43 as a regulator of DNA repair and BRAF/MEKi response, highlighting the therapeutic potential that this approach could eventually have in the clinic to overcome the limitations of current therapies and improve treatment outcomes for patients with advanced BRAF mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e4426f618e52ca064c1a54a223e0ac95626bea" target='_blank'>
              Cx43 enhances response to BRAF/MEK inhibitors by reducing DNA repair capacity
              </a>
            </td>
          <td>
            A. Varela-Vázquez, A. Guitián-Caamaño, P. Carpintero-Fernández, Alexander Carneiro-Figueira, Vanesa Álvarez, M. Varela-Eirín, Teresa Calleja-Chucla, S. B. Bravo-López, Anxo Vidal, Juan Sendón-Lago, Marina Rodríguez-Candela Mateos, José R Caeiro, Victoria Sanz-Moreno, Trond Aasen, Miguel G. Blanco, Guadalupe Sabio, María Quindós, Carmen Rivas, David Santamaría, Carlos Fernandez-Lozano, E. Fonseca, Pablo Huertas, Berta Sánchez-Laorden, Constance Alabert, M. Mayán
          </td>
          <td>2025-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686514f6a242593983edc4eedbd2ea02d17cd369" target='_blank'>
              REpair of heterozygous Mutations independent of Exogenous Donor template with high efficiency (REMEDY) using allele specific CRISPR targeting and HDR enhancers
              </a>
            </td>
          <td>
            Y. Sezgin, Genesis Snyder, Noushin Saljoughian, Colin Maguire, Ebru Erzurumluoglu Gokalp, Devi Jaganathan, Eleonora S D'Ambrosio, B. Ozes, Gregory Wheeler, Ben Kelly, Mark E. Hester, Zarife Sahenk, S. Vaidyanathan, A. Rashnonejad, Jerry Mendell, Nizar Y. Saad, Roshini S. Abraham, Juhi Bagaitkar, Susan D. Reynolds, Allison Bradbury, M. N. Kararoudi
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b16c1cd80bd43a019c039162e9f1b0a15b5cf6d2" target='_blank'>
              Integrative Multi-Omics Analysis Identifies Nuclear Factor I as a Key Driver of Dysregulated Purine Metabolism in DIPG
              </a>
            </td>
          <td>
            I. Mersich, Sunny Congrove, Matthew Horchar, Roman Caceres, Ranjithmenon Muraleedharan, Janki M Desai, Pankaj B Desai, Larry Sallans, J. Reisz, Abby Grier, Matthew R. Hass, Omer Donmez, Cailing Yin, M. Weirauch, Leah C. Kottyan, Charles B. Stevenson, Claire X. Sun, P. D. de Blank, N. Pillay-Smiley, T. Hummel, N. Elumalai, Ali Tavassoli, Ron Firestein, T. Phoenix, A. D'alessandro, Biplab Dasgupta
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 In acute myeloid leukemia (AML), genetic mutations distort hematopoietic differentiation, resulting in the accumulation of leukemic blasts. However, it remains unclear how these mutations intersect with the cellular origins of each patient's disease, and whether distinct sets of mutations converge upon similar differentiation patterns. Single-cell RNA sequencing has enabled high-resolution mapping of the relationship between leukemia and normal hematopoietic cell states. Yet, this application has been hampered by imprecise reference maps of normal hematopoiesis, or by small patient cohort sizes which do not adequately capture the extent of inter-patient heterogeneity in AML. To resolve this, we constructed a reference atlas of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes enriched for rare hematopoietic stem and progenitor cells (HSPCs). The resulting reference spans 55 cellular states and has been benchmarked against independent datasets of purified HSPCs. Using this comprehensive reference atlas, we confidently mapped over 1.2 million single-cell transcriptomes from 318 patient samples spanning AML, mixed-phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) diagnoses. Single-cell composition analysis revealed twelve patient subgroups, each reflecting distinct patterns of aberrant differentiation in AML. Strikingly, some AML samples exhibited virtually no overlap in cell state involvement with one another, likely reflecting their disparate cellular origins. One subgroup was enriched for early lymphoid progenitors and featured co-clustering of AML and MPAL samples. Notably, this early lymphoid subgroup included an AML sample which eventually relapsed with lymphoid disease. To understand the genetic determinants of aberrant differentiation in AML, we quantified leukemia cell state abundance in >1,200 patient samples and evaluated genotype-to-phenotype associations to link >40 genetic driver alterations with their specific impacts on AML differentiation. This identified genetic drivers for unconventional lineage phenotypes in AML, including erythroid lineage priming associated with the co-occurrence of complex cytogenetics with TP53 mutations as well as lymphoid lineage priming associated with bi-allelic RUNX1 mutations. We also identified non-genetic determinants of AML differentiation. For example, we identified two subgroups of KMT2A-rearranged AML with distinct cellular origins reflecting either HSPCs or committed myeloid precursors. Last, we show that distinct leukemia cell hierarchies can co-exist within individual patients, providing insights into AML evolution. Together, single-cell reference mapping of malignant cell states provides a framework for understanding the impact of genetic alterations on hematopoietic differentiation across hundreds of individual AML patients.



 Andy G. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A. Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, Charles G. Mullighan, John E. Dick. Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3806.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37654945a8b87aa1dc552938a4ce7cfc7067cb7a" target='_blank'>
              Abstract 3806: Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML
              </a>
            </td>
          <td>
            Andy G. X. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, C. Mullighan, J. Dick
          </td>
          <td>2025-05-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="The CRISPR-Cas9 system has been harnessed and repurposed into a powerful genome editing tool. By leveraging this technology, researchers can precisely cut, paste, and even rewrite DNA sequences within living cells. Nevertheless, the application of CRISPR screen technology goes far beyond mere experimentation. It serves as a pivotal tool in the fight against genetic diseases, systematically dissecting complex genetic landscapes, empowering researchers to unravel the molecular mechanisms underlying biological phenomena, and enabling scientists to identify and target the root causes of illnesses such as cancer, cystic fibrosis, and sickle cell anemia. Among all, cancer poses a formidable challenge for medicine, spurring eradication efforts. Radiotherapy, as a traditional treatment, yields results but has limitations. It eradicates cancer cells but also damages healthy tissues, causing adverse effects that reduce quality of life. Additionally, not all cancer cells respond to radiotherapy, and some may develop resistance, worsening the condition. To address this, a comprehensive whole-genome CRISPR screen technology is introduced, as it enables the efficient identification of radiosensitive and radioresistant genes, thereby advancing the field of cancer research and treatment. A genome-wide CRISPR screen was conducted in lung adenocarcinoma cells exposed to irradiation following the described protocol, through which both radioresistance- and radiosensitivity-associated genes were identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0b084e039e9bd637d4e61bfb3817e38d13b7fe" target='_blank'>
              Genome-Wide CRISPR Screen for Unveiling Radiosensitive and Radioresistant Genes.
              </a>
            </td>
          <td>
            Yu Yuan, Ziyu Jiang, Yuxin Zeng, Jiawen Tang, Jiang Luo, Conghua Xie, Yan Gong
          </td>
          <td>2025-05-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Melanoma, the deadliest form of skin cancer, exhibits resistance to conventional therapies, particularly in advanced and metastatic stages. Mitochondrial pathways, including oxidative phosphorylation and mitochondrial translation, have emerged as critical drivers of melanoma progression and therapy resistance. This study investigates the mitochondrial proteome in melanoma to uncover novel therapeutic vulnerabilities. Methods Quantitative proteomics was performed on 151 melanoma‐related samples from a prospective cohort and postmortem tissues. Differential expression analysis identified mitochondrial proteins linked to disease aggression and treatment resistance. Functional enrichment analyses and in vitro validation using mitochondrial inhibitors were conducted to evaluate therapeutic potential. Results Mitochondrial translation and oxidative phosphorylation (OXPHOS) were significantly upregulated in aggressive melanomas, particularly in BRAF‐mutant and metastatic tumors. Inhibition of mitochondrial pathways using antibiotics (doxycycline, tigecycline, and azithromycin) and OXPHOS inhibitors (VLX600, IACS‐010759, and BAY 87‐2243) demonstrated dose‐dependent antiproliferative effects in melanoma cell lines, sparing noncancerous melanocytes. These treatments disrupted mitochondrial function, suppressed key metabolic pathways, and induced apoptosis, highlighting the clinical relevance of targeting these pathways. Conclusions This study reveals mitochondrial pathways as critical drivers of melanoma progression and resistance, providing a rationale for targeting mitochondrial translation and OXPHOS in advanced melanoma. Combining mitochondrial inhibitors with existing therapies could overcome treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361245499d6c69af98713d5bd55c9f9ceda286b1" target='_blank'>
              Mitochondrial proteome landscape unveils key insights into melanoma severity and treatment strategies
              </a>
            </td>
          <td>
            Yonghyo Kim, V. Doma, Uğur Çakır, M. Kuras, L. Betancourt, I. Pla, Aniel Sanchez, Y. Sugihara, R. Appelqvist, Henriett Oskolás, Boram Lee, Jéssica Guedes, G. Monnerat, G. Carneiro, Fábio Nogueira, G. Domont, Johan Malm, Bo Baldetorp, E. Wieslander, I. Németh, A. M. Szasz, R. Hong, Krzysztof Pawłowski, M. Rezeli, H. Kwon, József Timár, David Fenyö, S. Kárpáti, Gyorgy Marko-Varga, Jeovanis Gil
          </td>
          <td>2025-06-23</td>
          <td>Cancer</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: SARS-CoV-2, which drove the 2019 COVID-19 pandemic, continues to engender inquiries into the role of host genetic factors in disease susceptibility. Despite the identification of over 1,000 genes potentially associated with SARS-CoV-2 and COVID-19, the mechanisms connecting genetic variants to phenotype remain elusive. To shed light on these mechanisms, we undertook an integrated analysis, merging data from whole-genome association analyses of COVID-19 with methylome and transcriptomic. Methods: Study includes African American adults from the GENE-FORECAST study, encompassing 371 individuals with whole genome sequencing (WGS), 203 with DNA methylation, and 321 with RNA sequencing (RNA-Seq) of blood. 53.3% of participants reported COVID-19. Significant loci associated with COVID-19 were examined within the framework of methylation quantitative trait loci (mQTL) which located near the gene-of-original (cis-mQTL) and expression quantitative trait loci (eQTL) which located near the gene-of-origin (cis-eQTL), enabling analysis to assess mediators between genetic variants and COVID-19 status. Results: Our analysis identified four intronic variants and confirmed a missense variant, rs1052067, in PMF1 associated with COVID-19. Causal mediation analysis revealed that the combination of genetic variants within PMF1, epigenomics, and transcriptomics mapped four pathways influencing COVID-19 status. These pathways include: rs9659072 -> DNAm at chr1:156285845 (annotated to TMEM79) -> ENSG00000198715:13 (annotated to GLMP); rs12083543 -> DNAm at chr1:155951748 (ARHGEF2) -> ENSG00000198715:13 (GLMP); rs1052067 -> DNAm at chr1:155951748 (ARHGEF2)-> ENSG00000198715:13 (GLMP); rs1543294 -> ENSG00000198715:13 (GLMP) -> DNAm at chr1:156077518 (MEX3A). Conclusions: Through integrated multi-omics analyses, we identified genetic variants whose effects on COVID-19 susceptibility are mediated by changes in DNA methylation and mRNA expression. These findings offer insights into potential mechanistic pathways that merit further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00c7bc8e6839adbc276042c4324afad16196de83" target='_blank'>
              Multi-omics analyses of the complex interplay between genetic variants, DNA methylation, and gene expression in COVID-19.
              </a>
            </td>
          <td>
            Guanjie Chen, Lisa DeRoo, Gabriel Goodney, A. Doumatey, Jie Zhou, A. Adeyemo, C. Rotimi, Amadou Gaye
          </td>
          <td>2025-07-01</td>
          <td>American journal of physiology. Heart and circulatory physiology</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f661d74703dc0b2d864c7015c79812dfe307ea" target='_blank'>
              Decoding the Transcriptional Complexity of the Human BRCA2 DNA Repair Gene Using Hybrid-seq.
              </a>
            </td>
          <td>
            P. Adamopoulos, Michaela A. Boti, Konstantina Athanasopoulou, P. Tsiakanikas, Glykeria N. Daneva, Andreas Scorilas
          </td>
          <td>2025-07-10</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Melanomas are genetically heterogeneous, displaying mitogen-activated protein kinase mutations and homozygous loss of tumor suppressor genes. Mouse models combining such mutations produce fast-growing tumors. In contrast, rare, slow-growing tumors arise in mice combining Braf activation with heterozygous loss of Pten. Here we show that similar tumors can arise in albino mice bearing only a Braf mutation. Incidence kinetics suggest a stochastic event underlies tumorigenesis in tumors that arise with only a Braf mutation, yet de novo mutations or structural variants that could explain the incidence of most tumors could not be found. Single-cell transcriptomics of tumors identify a cell type resembling “neural crest-like” cells in human and mouse melanomas. These exist in normal mouse skin, expand upon Braf activation, and persist through serial transplantation; analyses of gene expression suggest they serve as precursors of malignant cells. This state may serve as an intermediate on a slow path to malignancy that may provide a diagnostically and therapeutically important source of cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a8b6d33ff26b499d62c39adf32521b06fa0db2" target='_blank'>
              Uncovering minimal pathways in melanoma initiation
              </a>
            </td>
          <td>
            Hui Xiao, Jessica Shiu, Chi-Fen Chen, Jie Wu, Peijie Zhou, Sahil S. Telang, Rolando Ruiz-Vega, Robert A Edwards, Qing Nie, A. D. Lander, Anand K. Ganesan
          </td>
          <td>2025-06-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Anthracycline-induced cardiotoxicity (ACT) is a significant concern for cancer survivors, while genetic basis of ACT remains unclear. This study employs a murine genetic reference population (GRP) of BXD recombinant inbred strains, derived from DBA/2J (D2) and C57BL/6J (B6) crosses, to map quantitative trait loci (QTLs) linked to doxorubicin (DOX)-induced phenotypes through systems genetics approaches. Methods To model variability in ACT, 58 BXD strains and parental B6 and D2 mice (n ≥ 4 mice/sex/strain, 3–4-month-old) underwent an intraperitoneal injection of DOX (20 mg/kg). Survival and body weight (BW) were monitored for 10 days. Echocardiography was performed before treatment and on Day 5 post-treatment, followed by genetic mapping and Mendelian randomization analyses for identifying QTLs and candidate genes associated with DOX-induced traits and severity. Results Parental B6 strain had 60% survival, whereas 24% of D2 mice survived on Day 10. Among BXD strains, median survival varied, with BXD77 showing the lowest at Day 4. Echocardiography revealed cardiac dysfunction and a small-heart phenotype resembling ACT patients. Significant QTLs on Chromosome 10 (86–94 Mb), Chromosome 19 (52.5–54.2 Mb) and on Chromosome 14 (103–120 Mb) were associated with the survival, mean BW loss, and left ventricular (LV) volumes and ejection fraction (EF%), respectively. MR analysis identified significant causal associations between the genes implicated in BW loss (ADD3, HSPA12 A, SLC18 A2, PDZD8, DUSP5, CASP7) as well as EF% and LV volumes (GPC6, UGGT2, SLAIN1, POU4 F1, MBNL2) in BXD mice post-DOX and heart failure outcomes in humans. Most of the top candidates showed cardiomyocyte specific expression based on scRNA-seq data. Conclusions Survival, BW loss, and echocardiography parameters considerably varied among DOX-treated BXDs, suggesting significant influence of genetic background on expression of those traits. Several candidate genes that may modulate ACT susceptibility and heart failure were identified, providing a foundation for genetic-based risk stratification and therapeutics in cardio-oncology. Supplementary Information The online version contains supplementary material available at 10.1186/s40959-025-00349-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529bb5ece51e282110735b75f13ced4057c145bb" target='_blank'>
              Unraveling the genetic blueprint of doxorubicin-induced cardiotoxicity through systems genetics approaches
              </a>
            </td>
          <td>
            Buyan-Ochir Orgil, A. Bajpai, Neely R. Alberson, Morgan Lander, Batsaikhan Enkhzul, Hugo R. Martinez, Jeffrey A Towbin, Lu Lu, E. Purevjav
          </td>
          <td>2025-06-03</td>
          <td>Cardio-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Canine malignant insulinoma is a rare, highly metastatic and life-threatening neuroendocrine tumour of pancreatic beta cells. To map the single-cell transcriptomic landscape of canine insulinoma for the first time, transcriptomic profiles of 5,532 cells were captured from two spontaneous insulinomas (Patient 1 and 2) and one associated metastasis (Patient 2) in two Boxer dogs. Distinct cancer, endocrine, and immune cell populations were identified. Notably, all three tumour samples contained two transcriptionally distinct insulin-expressing tumour cell populations (INS+ and INS+FOSlow), characterised here for the first time. These two cancer cell populations significantly differed by ~ 8,000 differentially expressed genes (DEGs), particularly tumour suppressor genes (e.g. TP53, EGR1) and cancer-related pathways (e.g., MAPK, p53). In contrast, COX7A2L was one of a few genes ubiquitously expressed and significantly upregulated (> 20-fold) in both insulin-expressing tumour populations compared to other captured populations. Both populations were also characterised by expression of chromogranin/secretogranin neuroendocrine tumour marker genes (e.g. CHGA, SCGN). There were far fewer gene expression differences observed between insulin-expressing tumour cells from the two patients (~ 600 DEGs) than between the two cancer cell populations within each patient. These DEGs included CLTRN, TMSB4X, CSRP2, LGALS2, and C15orf48. Unexpectedly for a tumour of endocrine origin, the metastasis in Patient 2 exhibited > 20–70 fold upregulation of exocrine pancreatic genes including CLPS, PRSS2, PRSS and CTRC. Immune cell analyses identified distinct infiltrating immune populations, including memory T cells and macrophages and revealed likely tumour-immune interactions, including the CD40-CD40L interaction. This study provides the first single-cell RNA sequencing (scRNA-seq) analysis of naturally occurring insulinoma in any species, revealing tumour cell heterogeneity, novel immune microenvironment features, and potential therapeutic targets. Despite its small scale, the findings highlight the utility of scRNA-seq in veterinary oncology and its translational potential for pancreatic neuroendocrine tumours across species. Supplementary Information The online version contains supplementary material available at 10.1186/s44356-025-00026-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e2940ac404526b0febb4d2ad2d3e5b02fd7aaa" target='_blank'>
              Single-cell transcriptomic analysis of canine insulinoma reveals distinct sub-populations of insulin-expressing cancer cells
              </a>
            </td>
          <td>
            M. Wallace, M. E. Herrtage, R. Gostelow, L. Owen, L. Rutherford, K. Hughes, A. Denyer, B. Catchpole, C. A. O’Callaghan, L. Davison
          </td>
          <td>2025-05-26</td>
          <td>Veterinary Oncology (London, England)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Traditional approaches for capturing genomic alterations and DNA methylation require separate assays, complicating clinical workflows and limiting sample utilization, particularly with low-input materials like cell-free DNA. To address these challenges, we introduce a streamlined approach combining mutation and methylation profiling via mutation-protective strand synthesis with modified deoxycytidine triphosphates, demonstrating high concordance with standard enzymatic methyl-seq and DNA-seq in both whole-genome sequencing of cell lines and targeted sequencing of clinical samples. In potential clinical contexts, incorporating multi-omics information with this approach modestly improve circulating tumor DNA (ctDNA) detection by ~12% in pre-treatment lung cancer patients (N = 26) while preserving specificity in healthy controls (N = 13), and reveal relationships between homologous recombination repair (HRR) gene function and homologous recombination deficiency (HRD) mediated by promoter methylation-driven biallelic loss of HRR genes in gynecologic cancer patients (N = 27). For practical convenience, this method was also implemented on qPCR platform with high performance (0.5% limit of detection). With its adaptability and potential utility in ctDNA detection and treatment, this approach holds promise for advancing clinical diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461cfd126a8923a87cd09690f854d762a0619664" target='_blank'>
              A DNA alteration and methylation co-detection method for clinical purpose
              </a>
            </td>
          <td>
            Jiyan Yu, Chunhe Yang, Xintao Zhu, Zhankun Wang, Boping Xu, Ye Cai, Jingbo Zhao, Ruijian Guo, Wuzhou Yuan, Jianqing Wang, Bohao Dong, Frank Ron Zheng, Shuang Yang
          </td>
          <td>2025-06-04</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer is a growing concern around the world with more than 20 million new cases confirmed and counting. This is partly caused by the overuse of Chemotherapeutic drugs against Cancer, leading to its potency. Cancer starts with an accumulation of genetic mutations in the genome, and the ignorance to cell checks by CD8 and NK cells. The Cancer cell utilizes its hallmark capabilities and uses one of its cells to metastasizes in other areas of the body either through the perineural, circulatory, or lymphatic system. p53 is a tumor suppressor gene which repairs the mutations of the cell by conducting cell cycle arrest and inducing apoptosis. However, cancer cells can increase the expression of SB100 which can inhibit p53. One solution is to combine PRIMA-1 and CDDP which restores p53 and it’s wild type functions to induce apoptosis. This effect is increased with the use of adenoviral DN-Akt since it could increase p53 phosphorylation at Ser15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928e7ed5dc7a58f8574294e13f4f74e2d872a648" target='_blank'>
              Evolving Resistance: The Interplay of p53, Chemoresistance, and Metastasis in Cancer's Journey
              </a>
            </td>
          <td>
            Zenith Kwong
          </td>
          <td>2025-06-17</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41db77b7599b92a63937c7f3610506611aa97e8f" target='_blank'>
              A survey of human cancer-germline genes : linking X chromosome localization, DNA methylation and sex-biased expression in early embryos
              </a>
            </td>
          <td>
            A. Loriot, Julie Devis, Laurent Gatto, C. De Smet
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This review article aims to provide a comprehensive overview of genetic mutations and alterations in prostate cancer, highlighting their roles in disease development and progression. The scope of the review encompasses common genetic mutations, such as those in the PTEN and TP53 genes, as well as structural alterations like the TMPRSS2-ERG fusion. It also discusses the impact of these genetic changes on cellular functions, including cell cycle regulation and signaling pathways. Major findings indicate that genetic mutations contribute to the uncontrolled proliferation of prostate cells and influence treatment responses, emphasizing the importance of molecular profiling in guiding personalized therapy. Additionally, the review explores the significance of the tumor microenvironment and epigenetic modifications in prostate cancer progression. The insights gained from this review underscore the necessity for ongoing research into genetic alterations to improve diagnostic methods and therapeutic strategies, ultimately enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58a7054831337d3623636983abbff338a1c3f2ff" target='_blank'>
              Genetic Mutations in Prostate Cancer: Insights and Implications
              </a>
            </td>
          <td>
            Chinenye Mary Okorochi, Frederick Olusegun Akinbo, Gerald Ikechi Eze, Kehinde Elizabeth Olukoya, Enoch Chibuike Okorochi, Austin Ifeanyi Obiozor, Uzochukwu Chimdindu Ibe, Simon Enenche Iyalla, Macpherson Ifeanyi Ukachukwu
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af45bd1d5a17a9444e51cf2c69c1b1a00028176" target='_blank'>
              p53 status determines the epigenetic response to demethylating agents Azacitidine and Decitabine
              </a>
            </td>
          <td>
            Emma Langdale Hands, A. Wallmann, Gabrielle Oxley, Sophie Storrar, Rochelle D’Souza, Mathew Van de Pette
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="One of the strongest signatures of aging is an accumulation of mutant mitochondrial DNA (mtDNA) heteroplasmy. Here we investigate the mechanism underlying this phenomenon by calling mtDNA sequence, abundance, and heteroplasmic variation in human blood using whole genome sequences from ∼750,000 individuals. Our analyses reveal a simple, two-step mechanism: first, individual cells randomly accumulate low levels of “cryptic” mtDNA mutations; then, when a cell clone proliferates, the cryptic mtDNA variants are carried as passenger mutations and become detectable in whole blood. Four lines of evidence support this model: (1) the mutational spectrum of age-accumulating mtDNA variants is consistent with a well-established model of mtDNA replication errors, (2) these mutations are found primarily at low levels of heteroplasmy and do not show evidence of positive selection, (3) high mtDNA mutation burden tends to co-occur in samples harboring somatic driver mutations for clonal hematopoiesis (CH), and (4) nuclear GWAS reveals that germline variants predisposing to CH (such as those near TERT, TCL1A, and SMC4) also increase mtDNA mutation burden. We propose that the high copy number and high mutation rate of mtDNA make it a particularly sensitive blood-based marker of CH. Importantly, our work helps to mechanistically unify three prominent signatures of aging: common germline variants in TERT, clonal hematopoiesis, and observed mtDNA mutation accrual.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a832134834abc3268ec7af8e86bace5b876111" target='_blank'>
              Mechanism of age-related accumulation of mitochondrial DNA mutations in human blood
              </a>
            </td>
          <td>
            Rahul Gupta, Timothy J. Durham, Grant Chau, M. Uddin, Wenhan Lu, K. Karczewski, D. Howrigan, P. Natarajan, Wei Zhou, Benjamin M. Neale, V. Mootha
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Molecular profiling of DNA and RNA from pediatric cancers by next-generation sequencing has been demonstrated to improve diagnosis and prognosis and to identify somatic alterations indicating vulnerability to targeted therapies. Hence, much like in the treatment of adult cancers, molecular profiling is now routinely utilized in clinical workflows for pediatric cancers as a companion to conventional pathology diagnosis. Many variants of unknown significance identified through DNA profiling are being characterized by saturation genome editing, enabled by CRISPR editing technology and clever functional assays. Newer technologies and analytics are revealing additional structural complexity around cancer drivers and gene fusions in pediatric cancer DNA. Similarly, computational methods such as rare variant association studies and polygenic risk scoring are being used to identify novel cancer susceptibility. Together, these advances are expanding our understanding of pediatric cancer's complexity and fueling the development of emerging methods such as liquid biopsy-based monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae78e85702756670907089a42238ceb27fd06f7" target='_blank'>
              Pediatric Cancer Genetics and Genomics.
              </a>
            </td>
          <td>
            Elaine R. Mardis
          </td>
          <td>2025-05-28</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Individuals with germline PTEN variants (PHTS) have increased risks of the seemingly disparate phenotypes of cancer and neurodevelopmental disorders (NDD), including autism spectrum disorder (ASD). Etiology of the phenotypic variability remains elusive. Here, we hypothesized that decreased genomic diversity, manifested by increased homozygosity, may be one etiology. Comprehensive analyses of 376 PHTS patients of European ancestry revealed significant enrichment of homozygous common variants in genes involved in inflammatory processes in the PHTS-NDD group and in genes involved in differentiation and chromatin structure regulation in the PHTS-ASD group. Pathway analysis revealed pathways germane to NDD/ASD, including neuroinflammation and synaptogenesis. Collapsing analysis of the homozygous variants identified suggestive modifier NDD/ASD genes. In contrast, we found enrichment of homozygous ultra-rare variants in genes modulating cell death in the PHTS-cancer group. Finally, homozygosity burden as a predictor of ASD versus cancer outcomes in our validated prediction model for NDD/ASD performed favorably.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4dfbccabcc397005eb5e22df48b7360abc8d678" target='_blank'>
              Genomic diversity in functionally relevant genes modifies neurodevelopmental versus neoplastic risks in individuals with germline PTEN variants
              </a>
            </td>
          <td>
            Adriel Y Kim, L. Yehia, Ch. Eng
          </td>
          <td>2025-05-20</td>
          <td>NPJ Genomic Medicine</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 International efforts focused on discovery of biological targets in inflammatory breast cancer (IBC) have suggested that aberrant inter-cellular relationship instead of genomic aberrations is the key driver of IBC. To identify such IBC-specific aberrations, we generated single nucleus chromatin accessibility and transcriptome atlas of IBCs (9,628 nuclei from three donors) and compared IBC atlas with a similar atlas of the healthy breast (81,735 nuclei from 92 donors). We recently reported single nucleus atlas of breast tissues of healthy women of diverse genetic ancestry (Nature Medicine in press). In that report, we described markers that identify three major epithelial cell subtypes [Basal-myoepithelial (BM), luminal adaptive secretory precursor (LASP), luminal hormone sensing (LHS)], two endothelial cell subtypes, two adipocyte subtypes, fibroblasts, macrophages, and T cells of the healthy breast. We also showed that LHS cells are the likely cells-of-origin of Luminal A, Luminal B and HER2+ breast cancers. Unlike these breast cancer subtypes, IBC does not appear to have a cell-of-origin, as epithelial cells in IBCs shared gene expression pattern across BM, LASP and LHS cells. Similar results were obtained when comparison was restricted to ancestry-specific healthy breast atlas. However, individual gene level expression differences affecting specific signaling pathways were observed in epithelial cells of IBCs. LASP, and LHS cells of IBCs overexpressed GPR137C, a positive regulator of mTORC1 signaling, and HS6ST3, an enzyme required for heparan sulfate synthesis. LASP and LHS cells of IBCs displayed activation of mitotic and matrix metalloproteinase signaling, respectively. Endothelial cells of IBCs, which showed significant gene expression differences compared to endothelial cells of the healthy breast, displayed enhanced integrin cell surface interaction but loss of cell junction organization. Most critically, BM, LASP, LHS, and endothelial cells of IBCs compared to their counterparts in the healthy breast showed downregulation of TEX14, an inter-cellular bridge forming factor in germ cells and a regulator of mitosis. Downregulation of TEX14 expression in these cells was accompanied with changes in the chromatin accessibility patterns of this gene. These results reinforce the notion that defective inter-cellular communication is a hallmark of IBC and TEX14 expression levels may serve as a biomarker of this defect. We suggest that greater attention to vascular biology and vascular-epithelial cell communication has to be given for better understanding of IBC biology and therapeutic targeting.
 Citation Format: Harikrishna Nakshatri, Poornima Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu. Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-05-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4562fd16c140b8e77bf1f62e51f30b753052c4ad" target='_blank'>
              Abstract P3-05-23: Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer
              </a>
            </td>
          <td>
            H. Nakshatri, P. Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated member of the RAS family of small GTPases (RAS). It affects about one-fifth of cancer cases. The tumor microenvironment (TME) is a multifaceted network of immune cells, metabolites, microbiota, stromal components, and extracellular matrix. It creates a dynamic ecosystem that supports malignant initiation, progression, and therapy resistance through bidirectional crosstalk with tumor cells. Emerging evidence reveals distinct TME landscapes shaped by wild-type versus oncogenic KRAS variants. Additionally, TME rewiring occurs during KRAS-targeted therapies. Deciphering these KRAS-dependent TME architectures and their therapeutic vulnerabilities represents a critical frontier for precision oncology. This review synthesizes key milestones and persistent challenges in KRAS inhibitor development. And it systematically evaluates how KRAS mutations orchestrated immunosuppressive niches, metabolic symbiosis, stromal remodeling, and microbiome dysbiosis, supported by mechanistic insights from preclinical and clinical studies. It further explores therapeutic opportunities arising from targeting TME interactions, including rational combinations of KRAS inhibitors with immune checkpoint blockade, metabolic agents, or microbiota-modulating strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2718254ccab31c0317c655b38e3998d0c1a62d" target='_blank'>
              Mechanistic Foundations of KRAS-Driven Tumor Ecosystems: Integrating Crosstalk among Immune, Metabolic, Microbial, and Stromal Microenvironment.
              </a>
            </td>
          <td>
            Jiayao Ma, Shenao Fu, Jun Tan, Ying Han, Yihong Chen, Xiangying Deng, Hong Shen, Shan Zeng, Yinghui Peng, Changjing Cai
          </td>
          <td>2025-06-09</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5870c7c37807f788d6b6da82817760500b862c7f" target='_blank'>
              Genetic modifiers of somatic expansion and clinical phenotypes in Huntington's disease highlight shared and tissue-specific effects.
              </a>
            </td>
          <td>
            Jong-Min Lee, Zachariah L. McLean, Kevin Correia, Jun Wan Shin, Sujin Lee, Jae-Hyun Jang, Yukyeong Lee, Kyung-Hee Kim, Doo Eun Choi, Jeffrey D. Long, D. Lucente, Ihn Sik Seong, Ricardo Mouro Pinto, James V. Giordano, J. Mysore, Jacqueline Siciliano, Emanuela Elezi, Jayla Ruliera, T. Gillis, V. Wheeler, M.E Macdonald, J. Gusella, Anna Gatseva, Marc Ciosi, Vilija Lomeikaite, Hossameldin Loay, Darren G. Monckton, Christopher Wills, Tom Massey, Lesley Jones, P. Holmans, Seung Kwak, Cristina Sampaio, Michael Orth, G. B. Landwehrmeyer, J. Paulsen, E. R. Dorsey, Richard H. Myers
          </td>
          <td>2025-06-09</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>126</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are known to be involved in the manifestation of aggressive and therapy-resistant phenotypes in solid tumors. Nevertheless, the effects of dynamic intervention by TAMs on the DNA damage response of cancer cells are largely unexplored. Herein, we report that TAMs modulate the DNA damage repair pathways of ovarian cancer cells in response to platinum-(Pt) based therapeutic regimen. We demonstrate that coculture of TAMs with cancer cells directly upregulate Pol η, along with RAD18 and REV1 of the Translesion DNA synthesis (TLS) pathway, while concurrently downregulating components of the high-fidelity nucleotide excision repair (NER) mechanism. Consequently, we observed a better survival probability, DNA repair capacity, and enrichment of stemness properties in ovarian cancer cells. DNA bulky adducts produced by cisplatin are resolved through differential activation NER and TLS pathways. However, we elucidated that TAMs provide favorable conditions for activating the error-prone TLS pathway for lesion bypass over damage resolution. Furthermore, cellular crosstalk in cocultured cancer cells stimulates the nuclear translocation and expression of RelA, which recruits Pol η by acting as a potent transcription factor. In fact, with pristimerin-mediated disruption of p65 (RelA) translocation, the cancer cells become more prone to DNA damage-induced cell death and compromised regenerative potential. In both in vitro cell cultures and in vivo mouse xenograft models, cocultured macrophages exhibited predominantly M2-like phenotype with prevalence in the invasive zone of xenograft tumor margins. Taken together, our investigation revealed multifaceted crosstalk-mediated regulation of DNA damage repair between TAMs and ovarian cancer cells. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05731-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df732d214e8a9fdb550d6752112eedb8ef7093af" target='_blank'>
              Tumor-associated macrophages contribute to cisplatin resistance via regulating Pol η-mediated translesion DNA synthesis in ovarian cancer
              </a>
            </td>
          <td>
            Bilash Chatterjee, Mrinmoy Sarkar, Debanjana Ghosh, Sangita Mishra, Subhankar Bose, Md Maqsood Ahamad Khan, Senthil Kumar Ganesan, Nabanita Chatterjee, A. K. Srivastava
          </td>
          <td>2025-05-29</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8b8bad5074e21f22b413609dfa7b0543a9a0c1" target='_blank'>
              The Genetic and Evolutionary Landscape of Pentanucleotide Tandem Repeats in Human Genomes
              </a>
            </td>
          <td>
            Isaac R. L. Xu, D. Pellerin, L. V. D. Vondel, Matt C. Danzi, S. Züchner
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is a major global health challenge, with an estimated 19.3 million new cases by 2025, largely due to lifestyle changes, environmental factors, and increased life expectancy. Despite advances in conventional therapies, issues such as drug resistance, systemic toxicity, and relapse continue to hinder long‐term survival rates. One underexplored molecular mechanism potentially contributing to cancer progression is the enzyme Human tRNA‐dihydrouridine synthase 2 (hDus2). hDus2 plays a pivotal role in tRNA modification by converting uridine residues to dihydrouridine, enhancing tRNA structural flexibility and translational efficiency. Elevated dihydrouridine levels in tRNA have been correlated with cancer progression, highlighting hDus2 as a potential therapeutic target. In this study, we investigated the inhibition of hDus2 as a novel strategy for cancer therapy. Using in silico techniques, we screened a library of FDA‐approved drugs against the 3D structure of hDus2. Top candidate compounds identified through virtual screening with the Glide module were further evaluated using molecular dynamics simulations. Parameters such as RMSD, RMSF, Rg, SASA, and PCA were analyzed. Additionally, binding free energy calculations were performed to validate the stability and interaction strength of the hDus2 ligand complexes. Our results identified DB09050, a cephalosporin antibiotic used for treating complicated intra‐abdominal infections, and DB00650, a folate analog, as promising hDus2 inhibitors. These findings pave the way for further development of anticancer therapeutics post‐experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a55c7e0fe1d71c7c054d254cdfb39eedafd941" target='_blank'>
              Repurposing of FDA‐Approved Drugs Against Leukemia by Targeting hDus2
              </a>
            </td>
          <td>
            Anshuman Chandra, Monika Mukhija, Kanika Devi, Susmita Chaudhuri, P. Yadav, Vijay Kumar Goel
          </td>
          <td>2025-06-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 e14570


 Background:
 Immune checkpoint blockades (ICBs) showed little efficacy in pediatric solid tumors. The reconfiguration of gene regulation in cancers is a major factor in tumor immune microenvironment (TiME) remodeling and ICB resistance. Hypothesis: A subset of pediatric tumors may exhibit an epigenetically altered TiME, potentially responsible for ICB resistance. Objective: classify the TiME of pediatric extracranial solid tumors and describe the gene regulatory networks (GRNs) of immune landscapes.
 Methods:
 We studied bulk tumor gene expression and DNA-methylation from 184 pediatric extracranial solid tumors. Similarity network fusion (SNFtool) individualized TiME phenotypes by dual-omic clustering of immune genes. The clusters were compared by differential analysis for gene expression and methylation. We studied the relationship of enhancer methylation to gene expression (ELMER package) to infer methylation reprogramming. An enrichment study of regulatory binding regions (ReMapEnrich) identified the master regulators that define the GRNs unique to each phenotype.
 Results:
 SNF clustering identified 3 phenotypes with 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 exhibited low expression of immune genes (“cold” phenotype), cl2 overexpressed immune genes (“hot” phenotype), and cl3 featured global hypermethylation (epigenetically “altered” phenotype). Different tumor types were present across all phenotypes, but Ewing sarcoma was more frequent in altered, osteosarcoma and neuroblastoma in hot, and Wilms tumor in cold phenotype (
 p
 <0.05). Both hot and altered phenotypes overexpressed immune checkpoints (
 CD274
 ,
 PDCD1
 ), T-cell activator chemokines (
 CXCL9
 ,
 CXCL10
 ) and pro-inflammatory pathways, central to ICB sensitivity. However, only hot tumors overexpressed genes and pathways essential for antigen-presenting machinery and immune recognition. Methylation regulation was responsible for derepressing MHC-II genes and their regulators in hot tumors. In contrast, gene silencing in altered phenotype relied on hypermethylation/downregulation of the CIITA cofactor domain, the master control of MHC-II genes, and other immune-specific master regulators crucial for anti-tumor immunity and ICB sensitivity. The regulatory landscape of immune gene repression in the altered phenotype involved binding regions specific to the MYC/MAX network and the polycomb-group protein PCR2 (
 i.e.,
 EZH2 and SUZ12).
 Conclusions:
 We demonstrated that GRN reconfiguration participates in TiME reshaping in pediatric extracranial solid tumors. A subset of tumors with epigenetically altered immune phenotype has an immune recognition capacity repressed by MYC/MAX and PCR2 complexes. Future studies should investigate specific inhibitors to reprogram the GRN to foster immune recognition and ICB sensitivity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6a52d83f8f6dace4bda8bd4ea08fd3f518537c" target='_blank'>
              Dual-omic analysis to reveal gene regulatory network of immune landscape in childhood solid tumors and implications for immunotherapy.
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Mona Patoughi, Thibault Mallevaey, Lara Herrmann, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Noel Raynal, Vincent-Philippe Lavallée, Sonia Cellot, T. Tran, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Beta-hemoglobinopathies are severe genetic diseases caused by mutations affecting the production of the adult β-globin chain. The clinical severity is mitigated by the co-inheritance of mutations that reactivate the production of the fetal β-like γ-globin in adults. However, the epigenetic mechanisms underlying the adult-to-fetal hemoglobin (HbA-to-HbF) switching are still not fully understood. Here, we used epigenome editing technologies to dissect the molecular mechanisms underlying γ- and β-globin gene regulation and to develop novel potential therapeutics for β-hemoglobinopathies. Targeted removal of DNA methylation by dCas9-Tet1 (alone or together with the deposition of histone acetylation by CBP-dCas9) at the fetal promoters led to efficient and durable γ-globin reactivation, demonstrating that DNA methylation is a driver for HbF repression. This strategy, characterized by high specificity and a good safety profile, led to a substantial correction of the pathological phenotype in erythroid cells from patients with sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715c667c37284ad9d0a63ea14b0b32d588d39ebc" target='_blank'>
              Dissecting the epigenetic regulation of the fetal hemoglobin genes to unravel a novel therapeutic approach for β-hemoglobinopathies
              </a>
            </td>
          <td>
            S. Amistadi, L. Fontana, Chiara Magnoni, T. Felix, Matteo Kane Charvin, P. Martinucci, Candice Gautier, Lilian Greau, Bettina Bessières, P. Antoniou, O. Romano, Eric Allemand, C. Mussolino, A. Miccio
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Phosphoglycerate dehydrogenase (PHGDH) is the rate-limiting enzyme in the de novo Serine synthesis pathway (SSP), a highly regulated pathway overexpressed in several tumors. Specifically, PHGDH expression is dynamically regulated during different stages of tumor progression, promoting cancer aggressiveness. Previously, we demonstrated that high Serine (Ser) availability, obtained by increased exogenous uptake or increased PHGDH expression, supports 5-Fluorouracil (5-FU) resistance in colorectal cancer (CRC). Beyond its metabolic role in sustaining Ser biosynthesis, different “non-enzymatic roles” for PHGDH have recently been identified. The present study aims to investigate non-enzymatic mechanisms through which PHGDH regulates 5-FU response in CRC. Methods Overexpression and gene silencing approaches have been used to modulate PHGDH expression in human CRC cell lines to investigate the role of this enzyme in 5-FU cellular response. Identified mechanisms have been validated in selected 5-FU resistant cell lines, CRC patients-derived tumor tissue samples, and patients-derived 3D organoids. Transcriptomic analysis was performed on wild-type and PHGDH-silenced cell lines, allowing the identification of pathways responsible for PHGDH-mediated 5-FU resistance. The relevance of identified genes was validated in vitro and in vivo in a CRC xenograft model. Results PHGDH expression is highly variable among CRC tissues and patient-derived 3D organoids. A retrospective analysis of CRC patients highlighted a correlation between PHGDH expression and therapy response. Coherently, the modulation of PHGDH expression by gene silencing/overexpression affects 5-FU sensitivity in CRC cell lines. Transcriptomic analysis on CRC cell lines stably silenced for PHGDH evidenced down regulation in Hedgehog (HH) pathway. Accordingly, in vitro and in vivo studies demonstrated that the combined treatment of 5-FU and HH pathway inhibitors strongly hinders CRC cell survival and tumor growth in CRC xenograft models. Conclusions PHGDH sustains 5-FU resistance in CRC by mediating the upregulation of the HH signaling; targeting the here identified PHGDH-HH axis increases 5-FU susceptibility in different CRC models suggesting the 5-FU/HH-inhibitors combinatorial therapeutic strategy as a valid approach to counteract drug resistance in CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03447-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e4a03b9a2a33e6b19be4d9c774178fc59d033" target='_blank'>
              PHGDH drives 5-FU chemoresistance in colorectal cancer through the Hedgehog signaling
              </a>
            </td>
          <td>
            Caterina Mancini, Giulia Lori, Gianluca Mattei, Marta Iozzo, Dayana Desideri, Fabio Cianchi, Laura Fortuna, Federico Passagnoli, D. Massi, Filippo Ugolini, Luca Messerini, Salvatore Piscuoglio, Antonio Pezone, Francesca Magherini, A. Biagioni, T. Lottini, Demetra Zambardino, G. Truglio, Elena Petricci, Alberto Magi, A. Arcangeli, Luisa Maresca, Barbara Stecca, E. Pranzini, M.L. Taddei
          </td>
          <td>2025-07-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 The Mismatch Repair (MMR) pathway, a crucial tumor suppressor mechanism, is dysregulated across cancer types. We previously identified the loss of the MutL complex of the MMR pathway, as a key factor inducing resistance to endocrine therapy in Estrogen Receptor positive (ER+) breast cancer patients. Preliminary data previously generated in the lab supports a role for MutL loss in promoting metastasis of ER+ breast cancer. A mammary-specific knockout mouse model for MLH1, a principal component of MutL, created in the Haricharan lab develops aggressive mammary tumors that metastasize to multiple organs. To unveil the mechanism by which MLH1 loss triggers metastasis, an unbiased RPPA and RNA-Seq was conducted which revealed cytokine secretion in the immune system as a distinctive signature of MLH1- ER+ breast cancer cells. The tumor secretome, particularly cytokines, can manipulate macrophage plasticity to skew polarization states towards either an anti- or pro- metastatic phenotype. Secreted cytokine levels altered by MLH1 loss were identified in multiple ER+ breast cancer cells lines from concentrated conditioned media. Additionally, in vivo experiments with xenograft MLH1- ER+ tumors in nude mice, revealed an elevated number of CD206 expressing tumor-promoting M2 macrophages in the tumor microenvironment. Furthermore, MLH1- ER+ breast cancer cells grown in the presence of macrophages exhibit an increase in their migratory capacity. Our preliminary data strongly indicate that loss of MLH1 alters the ER+ breast cancer secretome and alters macrophage maturation, suggesting a role for MLH1 in immunomodulation supporting metastatic progression. Investigation of this new role will uncover key tumor-intrinsic drivers of “cold” tumor immune microenvironment profiles and present unique opportunities for therapeutic interventions to convert them into “hot” profiles, thereby improving ER+ breast cancer patient outcomes.
 Citation Format: Megha Raghunathan, Aloran Mazumder, Sabrina Carrell, Daniel Lozano, Nindo Punturi, Svasti Haricharan. MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f9ff414c291cb2d359a6c911b717a2e6733c88" target='_blank'>
              Abstract P1-07-14: MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity
              </a>
            </td>
          <td>
            M. Raghunathan, A. Mazumder, Sabrina Carrell, D. Lozano, Nindo B Punturi, S. Haricharan
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutation spectra vary across genetic and environmental contexts, leading to differences between and within species. Most research on mutation spectrum has focused on the trinucleotide (3-mer) mutation types in mammals, limiting the breadth and depth of variation surveyed. In this study, we use whole-genome resequencing data across 108 eukaryotic species – including mammals, fish, plants, and invertebrates – to characterize pentanucleotide (5-mer) non-coding mutation spectra using a Bayesian approach. Our findings reveal cytosine transition mutability at CpG and (among plants) at CHG sites as the main drivers of variation in mutation spectra across eukaryotes, correlating strongly with genomic CpG and CHG depletion. However, despite the influence of methylation on CpG mutability, genome-wide average CpG methylation levels do not predict CpG transition rates across species and CHG methylation does not predict CHG transition rate, indicating unknown genetic or environmental factors influencing mutation rates at methylated cytosines. Together, our results illustrate the pivotal role of mutagenesis in shaping genome composition across eukaryotes and highlight a gap in knowledge about the mechanisms governing mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840be1a2fe72dda5c8b163dbfd7199ef76a61434" target='_blank'>
              Methylation-associated mutagenesis underlies variation in the mutation spectrum across eukaryotes
              </a>
            </td>
          <td>
            Fabián Ramos-Almodóvar, Ziyue Gao, Benjamin F. Voight, I. Mathieson
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f9af4dcbefbfc2af9850cbacc2d2628f2e0e1e" target='_blank'>
              TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers
              </a>
            </td>
          <td>
            Yun Zhang, Jianqi She, Xueyan Hu, Yueqi Jin, Changyu Tao, Minghao Du, Ence Yang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. 
Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nu- clear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syn- drome in the family, the maternal grandfather had a similar tumor requiring amputation. 
Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-06-20</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983f8601b7f33f2bbd798f78ca234ed6bec2a254" target='_blank'>
              A clinical solution for tracking clonal evolution of acute myeloid leukemia after allogeneic transplantation using bulk next generation sequencing.
              </a>
            </td>
          <td>
            Elizabeth F. Krakow, Nathan Lee, Isaac Jenkins, O. Sala-Torra, Lan W. Beppu, J. Radich, Bryce Fukuda, Brenda M Sandmaier, C. Yeung, Ivana Bozic
          </td>
          <td>2025-05-16</td>
          <td>Bone marrow transplantation</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Over the past half-century of the global effort against cancer, the vast majority of investigations in both tumor basic research and clinical practice have centered on the "somatic mutation" theory, such as in molecular classification, individualized precision medicine strategies, gene therapy approaches, the development of neoantigen-based tumor vaccines, and advancements in sequencing technologies. Even in the extensively studied tumor microenvironment (including tumor immunity), which has garnered significant attention in recent years, the underlying mechanisms frequently revert to specific genes and mutations within tumor cells or microenvironmental cells as the primary driving forces. However, despite the dominance of the "somatic mutation" paradigm, truly effective approaches for curing cancer in clinical settings remain elusive. Undoubtedly, if the prevailing "somatic mutation theory" continues to monopolize cancer research, meaningful progress in understanding and treating cancer will likely remain frustratingly out of reach. At this critical juncture in the evolution of cancer research, a comprehensive re-evaluation of cancer not only is necessary but also imperative, highlighting the urgent need for a profound transformation in our conceptual framework. This article systematically elucidates the novel perspective offered by the "tumor system" for comprehending the essence of cancer, the foundational principles of "tumor ecology" and their potential applications in treatment, and explores in depth the theoretical framework and research significance of the emerging field of "ecological pathology". Beyond merely advocating for the abandonment of the currently dominant linear reductionist paradigm of cancer, this commentary strives to construct a pragmatic and systematically structured framework to guide the trajectory of the "post-genomic revolution in oncology" and the "tumor ecological philosophy", ultimately fostering the realization of the overarching societal goal of eradicating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbba50cb169d2a51d829cb21c736013cc01b71a8" target='_blank'>
              [Rethinking cancer].
              </a>
            </td>
          <td>
            W. R. Luo
          </td>
          <td>2025-05-28</td>
          <td>Zhonghua zhong liu za zhi [Chinese journal of oncology]</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

Recent advances in CRISPR–Cas9–based genome editing have revolutionized cancer
immunotherapy by enabling precise and efficient T-cell engineering. This review focuses on how
CRISPR-mediated modifications enhance T-cell functionality, highlighting key applications, such
as CAR-T cells and TCR-engineered T-cell therapies. By selectively targeting immune-regulating
genes, CRISPR empowers T cells to recognize and eliminate tumor cells with greater specificity
and reduce off-target effects. This precision opens the door to personalized treatment strategies, allowing
T cells to be tailored to each patient’s unique tumor profile. Furthermore, the ability to
rapidly and systematically edit multiple gene targets has accelerated the development of more potent
and durable T-cell therapies, overcoming some of the current limitations in cancer immunotherapy.
Overall, CRISPR’s transformative potential of CRISPR provides a platform for next-
generation patient-centric cancer treatments, paving the way for improved clinical outcomes and
expanded therapeutic options.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcb955649c8f44c414ba7f048441db68da50f99" target='_blank'>
              Innovation and Implication of CRISPR-Mediated T-cell Engineering in Cancer Immunotherapy: A New Insight
              </a>
            </td>
          <td>
            Syeda Areej Imran, H. Sultan, Muhammad Zubair, S. Ghazanfar, Anas Islam, Syed Mustafizur Rahaman, Sumel Ashique
          </td>
          <td>2025-06-03</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abrogating the immunoevasive role of the tumor immune microenvironment (TIME) represents a critical yet still elusive challenge in cancer treatment. Progress in this area has been hampered by both technological limitations and incomplete understanding of TIME-dependent immunoevasion mechanisms. We hypothesize that the immune-evasive role of TIME subpopulations—including regulatory T cells, cancer-associated fibroblasts, and tumor-associated macrophages—is critically mediated by hyperconnected Master Regulator Checkpoint (MRC) modules whose aberrant activity, as induced by paracrine signals, can be abrogated or modulated either genetically or pharmacologically. MRCs are primarily composed of transcription and co-transcription factors, acting downstream of surface receptors and signal transduction cascades to control the transcriptional identity and, ultimately, the phenotype of individual TIME subpopulations. Pharmacological inhibition of subpopulation-specific MRC proteins can thus help reprogram the TIME and potentially abrogate or modulate its immunosuppressive state. This paradigm shift, away from single ligand/receptor targeting, is supported by recent algorithmic, experimental, and clinical advances allowing systematic identification of MRCs and their pharmacological modulators using systems immunology-based approaches. Refocusing the deployment of existing tools and experimental methods that have proven successful in tumor cell contexts to identify and validate MRC-targeting agents capable of remodeling the immunosuppressive cell states of the tumor microenvironment can potentially pave the road to novel combination therapy synergizing with immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/992372e5de760c711f403d3ef7aa2e616073ce3d" target='_blank'>
              Systematic identification and targeting of master regulator checkpoints (MRC) governing tumor microenvironment-mediated immune evasion
              </a>
            </td>
          <td>
            Pasquale Laise, G. Bosker, Mariana Babor, Xiaoyun Sun, Stuart Andrews, Lorenzo Tomassoni, Andrea Califano, Mariano J. Alvarez
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a poor prognosis that is driven primarily by oncogenic KRAS mutations present in > 90% of cases. KRAS mutations, particularly the G12D mutation which dominates in PDAC, fuel tumor initiation, progression, and immune evasion, thereby contributing to therapy resistance. Nevertheless, KRAS has long been considered "undruggable" due to its structure. Recent advances have spurred transformative progress in direct KRAS inhibition. While FDA-approved mutation-specific and pan-KRAS inhibitors show limited efficacy in PDAC, emerging agents (MRTX1133 and RMC-9805) have demonstrated preclinical promise. However, resistance remains a critical hurdle and is driven by pathway reactivation, secondary mutations, and metabolic adaptations. Alternative strategies targeting upstream regulators (SHP2 and SOS1) aim to block KRAS activation and associated resistance mechanisms. Preclinical studies have also highlighted synergistic benefits of combining KRAS inhibitors with MEK, PI3K, or CDK4/6 inhibitors, which are now undergoing clinical evaluation. Immunotherapies, including KRAS-targeted vaccines and adoptive T-cell therapies, have further expanded the therapeutic landscape of enhancing KRAS-targeted therapies in PDAC. The molecular basis of KRAS-driven PDAC, current inhibitors, resistance mechanisms, and innovative strategies are discussed herein to address treatment barriers. Opportunities to improve clinical outcomes are underscored in this challenging malignancy by integrating insights from preclinical and clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8cfbe6080878693ce57a6e261fa2c4f9ddf499" target='_blank'>
              Drugging the 'undruggable' KRAS: breakthroughs, challenges, and opportunities in pancreatic cancer.
              </a>
            </td>
          <td>
            Nawaz Khan, Umar Raza, Syed Aqib Ali Zaidi, Muhadaisi Nuer, Kayisaier Abudurousuli, Yipaerguli Paerhati, Alifeiye Aikebaier, Wenting Zhou
          </td>
          <td>2025-07-07</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recurrent somatic mutations in the key spliceosome component, SF3B1, have been identified at various frequencies across several cancer types. The most common hotspot mutation is the K700E missense mutation, and while its effects on splicing have been well characterised at the molecular level, the mis-spliced genes that contribute to cancer progression and/or dictate responses to therapy are still unclear. Here, we used we use cell line modelling to assess the impact of the SF3B1K700E mutation on the cellular response to various apoptosis-inducing agents. Our data suggest that the SF3B1K700E mutation leads to reduced cFLIP levels, along with defects in the splicing and translation of BCL2, causing a shift in the balance of pro- and anti-apoptotic genes and proteins, which confers greater sensitivity to the bivalent SMAC mimetic, BV-6. As such, BV-6 may represent a therapeutic opportunity for patients with SF3B1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78c6127f794d73c9e1a0f7e0fc6e3deedb616b30" target='_blank'>
              The cancer-associated SF3B1K700E spliceosome mutation confers enhanced sensitivity to BV-6-induced cytotoxicity
              </a>
            </td>
          <td>
            Lydia E Roets, J. Blayney, Hayley McMillan, Patrick J Preston, Alexander Mutch, Ken I Mills, K. Savage, K. Lappin
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c4150fc3d240c36c02c0683f2eda54a083cd7f" target='_blank'>
              MAPK and mTORC1 signaling converge on Cyclin D to enable cell-cycle re-entry in melanoma persister cells
              </a>
            </td>
          <td>
            Varuna Nangia, Humza M. Ashraf, Nasreen Marikar, Victor J. Passanisi, C. R. Ill, Sabrina L. Spencer
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 How can we improve the safety and efficiency of germline gene correction using CRISPR/Cas9 in early mouse embryos?



 We observed a higher frequency of desirable gene editing events when using crRNA XT for CRISPR/Cas9-mediated gene correction during fertilization of mouse oocytes.



 CRISPR/Cas9 induces double-stranded DNA breaks, which can be repaired via homology-directed repair (HDR) using a repair template to achieve precise gene editing. Undesired events may occur, including non-homologous end joining, leading to error-prone repair causing insertions or deletions (indels), unrepaired DNA breaks resulting in large deletions or partial chromosomal loss, or HDR using the untargeted parental chromosome (gene conversion), causing loss of heterozygosity.
 HDR frequencies in early embryos are typically much lower compared to gene editing outcomes in somatic cell lines. In this study, we aimed to improve HDR efficiency by using chemically modified crRNA optimized for high nuclease environments.



 To achieve gene correction with reduced mosaicism, we co-injected CRISPR/Cas9 along with mutant C57BL/6J mouse sperm (carrying a homozygous Plcz1 deletion, causing failed fertilization after ICSI) into wild-type B6D2F1 and DBA/2 mouse oocytes for subsequent developmental and detailed genetic analysis, respectively.
 We analysed 59 embryos injected with crRNA XT and 60 embryos injected with classical crRNA. The repair template used contained synonymous variants, which, if detected, indicate successful gene correction through HDR with template use.



 We injected CRISPR/Cas9 ribonucleoprotein complexes, designed for correcting the Plcz1 deletion, simultaneously with mutant mouse sperm into wild-type mouse oocytes during piezo-assisted ICSI, followed by artificial activation. We compared the use of crRNA XT, containing additional modifications for improved functional stability, to classical crRNA. After four days of in vitro culture, DNA was extracted and used for targeted sequencing, whole genome amplification, and copy number variation (CNV) analysis to evaluate the events following gene editing.



 After gene correction using classical crRNA, targeted sequencing revealed that the deletion was undetectable in 8 out of 60 embryos (13.3%). However, none of these embryos contained the synonymous variants which would indicate HDR with the provided repair template. In contrast, using crRNA XT, the deletion was no longer detected in 22 out of 59 embryos (37.3%). The synonymous variants were found in 50% of the embryos with undetectable deletions, suggesting a remarkable increase in HDR rates with the repair template. These results indicate that crRNA XT, with its enhanced stability, leads to a remarkable improvement in the frequency of desired gene editing events.
 To explain further the events in which the deletion is no longer detected without incorporation of the repair template’s synonymous variants, we performed whole-genome amplification and copy number variation (CNV) analysis using shallow whole-genome sequencing. In the embryos edited using classical crRNA, we observed partial chromosomal loss of chromosome 6 (carrying the Plcz1 gene) in 1 out of 7 analysed embryos and aneuploidy with partial chromosomal loss of chromosome 6 in 1 out of 7 analysed embryos. CNV analysis of embryos edited with crRNA XT is still ongoing to complete our comparison.



 The resolution of the shallow whole-genome sequencing is limited to 5 Mbp. To detect larger deletions smaller than this threshold, we will perform long-read sequencing.



 Targeted sequencing of embryos undergoing early gene correction for the Plcz1 deletion revealed that the use of crRNA XT, which offers greater stability, significantly enhanced the frequency of HDR with the repair template compared to classical crRNA. These findings contribute to a deeper understanding of germline gene editing events.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797fea7f3b3a355f785c9bfcca4bad7300fab085" target='_blank'>
              P-804 Improving CRISPR/Cas9 gene correction in early mouse embryos using RNase-resistant crRNA XT
              </a>
            </td>
          <td>
            S. Jooris, A. Rybouchkin, F. Jalali, J. Parrington, F. Vanden Meerschaut, T. Hochepied, K. Claes, D. Stoop, B. Heindryckx, B. Menten
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations and gene rearrangements are crucial for the diagnosis and subtyping of acute myeloid leukemia (AML). However, the contribution of non-coding genetic variants, particularly those within cis-regulatory elements (CREs), to AML pathophysiology and heterogeneity remains poorly understood. In this study, we characterize the single-cell chromatin accessibility landscapes of 10 bone marrow samples from AML patients at diagnosis. Additionally, we develop eMut, an integrated computational pipeline for detecting, imputing, and functionally characterizing non-coding mutations in CREs at the single-cell level. Our analysis identifies 2878 potential somatic non-coding mutations, highlighting the extensive mutational heterogeneity in the non-coding genome of AML patients, with recurrent non-coding mutations displaying cell type-specific patterns. We show that mutated CREs are enriched with blood-related genetic variants, potentially linked to AML-associated genes, and harbor a higher abundance of functional CREs, suggesting their functional relevance in leukemogenesis. Importantly, we pinpoint candidate functional non-coding mutations that associate with alteration of target gene expression in AML. Collectively, our work provides a comprehensive resource of single-cell chromatin accessibility in AML and introduces an integrative approach to identify candidate functional non-coding mutations contributing to cellular heterogeneity in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cc3c9c6e3317c1eed73191b4e5e0c7a9dcb592" target='_blank'>
              Discovery of candidate functional non-coding mutations in acute myeloid leukemia using single-cell chromatin accessibility sequencing
              </a>
            </td>
          <td>
            Ming Zhu, Jiali Zhu, Zhijuan Zhu, Yiding Yang, Jinxian Dai, Hua Li, Nainong Li, Jialiang Huang
          </td>
          <td>2025-05-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The objective of this article is to promote abnormal methylation enzymes (MEs) as a critical cause of cancer and the destabilization of abnormal MEs as a new approach to direct cancer therapy. MEs are a ternary enzyme complex consisting methionine adenosyltransferase (MAT)-methylatransferase (MT)-S-adenostkhomocysteine hydrolase (SAHH) which plays a pivotal role on the regulation of cell grow and differentiation. Normal cell growth is under the allosteric regulation of MEs by steroid hormone. In telomerase expressing cells such as pluripotent stem cells, MEs become associated with telomerase that changes kinetic properties of MAT-SAHH isozyme pair by increasing Km values and the regulation greatly in favor of cell growth. Embryonic development of the fetus depends on abnormal MEs to regulate exceptional growth. Abnormal MEs do not cause problem to cells expressing telomerase, because the operation of abnormal MEs is strictly under the control of safety mechanisms such as contact inhibition, ten-eleven translocator-1 enzyme and chemo-surveillance. If safety mechanisms become dysfunctional, clinical symptoms will arise, cancer being the worst case. Wound healing requires the proliferation and the terminal differentiation of progenitor stem cells (PSCs) which are pluripotent embryonic stem cells.  Wound unhealing is mostly attributable to the collapse of chemo-surveillance. But the nature does not have a mechanism to correct the collapse of chemo-surveillance. Instead, PSCs are forced to evolve into cancer stem cells (CSCs) by the silencing of TET-1 enzyme to escape contact inhibition that limits the extent PSCs can proliferate. The proliferation of CSCs still cannot heal the wound, which are then forced to progress to faster growing CCs by chromosomal abnormalities to activate oncogenes or to inactivate suppressor genes, eventually forcing CSCs to become full blown CCs. CSCs became known in 1997. The discovery of CSCs unraveled CSCs as the most important issue of cancer, because these are the cells to initiate cancer growth, and the cells responsible to cause the fatality of cancer. Elimination of CSCs is essential to the success of cancer therapy. The evolution of CSCs is critically linked to wound unhealing. Therefore, induction of terminal differentiation of CSCs is the only option to solve the issue of CSCs. CSCs are unresponsive to cytotoxic chemotherapy, radiotherapy or immunotherapy, because these cells are protected by drug resistance, anti-apoptosis mechanisms and devoid of program death antigen. Besides, these therapies contribute to the destruction of chemo-surveillance. CDA formulations are the best drugs to eliminate CSCs, which are the nature’s creation for the perfection of wound healing to avoid and to cure cancer. CDA formulations can also eliminate CCs by turning CCs into terminally differentiated cells unable to replicate. Therefore, destabilization of abnormal MEs is the right indication to direct cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0c9d46f2f459e5031816891f43fc6e5b352356" target='_blank'>
              Targeting Abnormal Methylation Enzymes: A New Approach to Cancer Therapy
              </a>
            </td>
          <td>
            Ming Liau, Chrinstine L. Craig, L. Baker
          </td>
          <td>2025-05-24</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Glioblastoma is the most aggressive type of brain tumor and is associated with a poor prognosis. First-line treatment is surgical resection followed by radiotherapy and temozolomide-based chemotherapy. However, the duration of treatment with temozolomide is limited due to both its toxicity and the development of drug resistance. The prognostic and predictive factor for response to temozolomide is the methylation status of the MGMT promoter. Indeed, loss of MGMT promoter methylation is a major cause of chemoresistance. However, the development of drug resistance is not only associated with changes in MGMT methylation. The entire epigenome changes and acquires specific properties necessary for tumor progression. Methods To study epigenetic alterations associated with temozolomide exposure, we generated a TMZ-resistant cell model. We investigated epigenetic alterations in the cell model and in a cohort of patients with recurrent glioblastoma using genome-wide methylome approaches (Epic Arrays 850 k). Results We investigated the epigenetic changes associated with temozolomide exposure. Therefore, we generated a TMZ-resistant cell model and studied the epigenetic features acquired after selective temozolomide pressure. Our next step was to investigate the epigenomic differences between primary and recurrent tumors in a small cohort of patients. Finally, we performed a cross-analysis between the epigenetic differences found in TMZ-resistant cells and recurrent glioblastomas to identify common signatures that could be used to guide future resistance-overcoming studies. Conclusions Temozolomide induces significant epigenetic changes in glioblastoma, which may contribute to treatment resistance and increased tumor aggressiveness. The results suggest that further research into DNA methylation changes associated with TMZ resistance is crucial. The use of primary tumor cells in resistance models may help identify strategies to overcome chemoresistance in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06767-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862b534a5409db78fa687bac68e4b9867edb5de0" target='_blank'>
              DNA methylation remodeling in temozolomide resistant recurrent glioblastoma: comparing epigenetic dynamics in vitro and in vivo
              </a>
            </td>
          <td>
            Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Federica Trio, Sara Ferraro, O. Affinito, Alberto De Bellis, Maria Laura Del Basso De Caro, Roberta Visconti, Lorenzo Chiariotti, Giuseppe Catapano, R. Della Monica
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The PIK3R1 gene encodes the class IA PI3K regulatory subunit p85α, which is frequently altered in cancer. PIK3R1 functions as a tumor suppressor by stabilizing and inhibiting the catalytic activity of p110, and it directly binds to and enhances the activity of the PTEN lipid phosphatase. Aberrations in the PIK3R1 gene are associated with poor prognosis in cancer; available data underscore the significant role of PIK3R1 mutations in mediating tumorigenesis by promoting the signaling of the PI3K/AKT/mTOR pathway. Moreover, copy number variations, driver mutations, and epigenetic alterations in PIK3R1 contribute to tumorigenesis and progression through distinct mechanisms. This article reviews the cancer-promoting effects of PIK3R1 gene aberrations across major cancer types and elucidates their underlying mechanisms. It also discusses the targeted therapies for related aberrations, aiming to provide a comprehensive understanding of the dynamic interplay of PIK3R1 in cancer, thereby advancing precision medicine and the development of targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1f74c89d744eac6c65e6a4ebb2236a33792d6e" target='_blank'>
              Cancer-Associated PIK3R1 Genetic Aberrations and Precision Medicine
              </a>
            </td>
          <td>
            Huan Xie, Yirong Li, Xinran Li
          </td>
          <td>2025-06-12</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d0252edfafbc65dacd73e410e1f6e287ac0ef7" target='_blank'>
              Pathway Sculptor for Compact and Versatile Combinatorial Genetic Perturbation
              </a>
            </td>
          <td>
            Bo Gu, James M. Linton, Brice Graham Hendrickson, Hengyu Li, Ron Hadas, Gal Manella, Jan Gregrowicz, M. Elowitz
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, Alice Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The CRISPR–Cas system has transformed molecular biology by providing precise tools for genome editing and pathogen detection. Originating from bacterial adaptive immunity, CRISPR technology identifies and cleaves genetic material from pathogens, thereby preventing infections. CRISPR–Cas9, the most widely utilized variant, creates double-stranded breaks in the target DNA, enabling genetic disruptions or edits. This approach has shown significant potential in antiviral therapies, addressing chronic infections, such as HIV, SARS-CoV-2, and hepatitis viruses. In HIV, CRISPR–Cas9 edits the essential viral genes and disrupts latent reservoirs, while CCR5 gene modifications render the T cells resistant to viral entry. Similarly, SARS-CoV-2 is targeted using CRISPR–Cas13d to inhibit the conserved viral genes, significantly reducing viral loads. Hepatitis B and C treatments leverage CRISPR technologies to target conserved genomic regions, limiting replication and expression. Emerging innovations, such as the PAC-MAN approach for influenza and base-editing systems to reduce off-target effects, further highlight the therapeutic versatility of CRISPR. Additionally, advances in Cas12a and Cas13 have driven the development of diagnostic platforms like DETECTR and SHERLOCK, which provide rapid and cost-effective viral detection. Innovative tools like AIOD-CRISPR enable accessible point-of-care diagnostics for early viral detection. Experimental approaches, such as targeting latent HSV-1 reservoirs, highlight the transformative potential of CRISPR in combating persistent infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be60256d8ceb6a861f6bef31814f57b4bea13d0d" target='_blank'>
              Progress in CRISPR Technology for Antiviral Treatments: Genome Editing as a Potential Cure for Chronic Viral Infections
              </a>
            </td>
          <td>
            Fatemeh Nouri, Farnaz Alibabaei, Behina Forouzanmehr, Hamed Tahmasebi, Valentyn Oksenych, Majid Eslami
          </td>
          <td>2025-05-20</td>
          <td>Microbiology Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44150a1a39e6364f7361004b5a00696c89dd8c7b" target='_blank'>
              ZMYND8 promotes the Warburg effect and tumorigenesis through c-Myc activation in pancreatic cancer.
              </a>
            </td>
          <td>
            Hui Liu, Zhifeng Zhao, Changle Wu, Jinxin Chen, Zhiwei He, Kai Jiang
          </td>
          <td>2025-06-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e861449b1c1d4d8b590227b0b752be56a345c9e3" target='_blank'>
              In-Silico Characterization of TP53 Splice Mutations in Somatic and Germline Tumours
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, Debina Sarkar, Aaron Jeffs, Tania Slatter, Anthony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the most common cancer diagnosis in women globally, accounting for more than 1 in 10 new cancer diagnoses. While BC has historically been a poor candidate for immune checkpoint blockade (ICB), recent studies in triple-negative BC (TNBC) have demonstrated anti-PD-1 treatment efficacy when employed in conjunction with chemotherapy1. APOBEC3 (A3) enzymes, such as A3A and A3B, catalyze cytosine-to-uracil deamination in single-stranded DNA, leading to signature single base substitution mutations (SBS2 and SBS13). A3 expression and mutagenesis have been associated with poor clinical outcomes, including drug resistance2,3. Murine models for studying ICB and mutagenesis by human A3A and A3B are underdeveloped. Here, we address this gap by generating two independent mammary tumor cell lines and studying responsiveness to ICB. Two novel mammary cancer cell lines, MM001 and MM008, were isolated from spontaneously arising tumors in C57BL/6 MMTV-Cre+;p53fl/+ mice. Each tumor line was passaged serially in the mammary fat pads of C57BL/6 (WT) mice to obtain conditioned cell lines MM001i and MM008i, which formed consistent and reproducible tumors. Tumor immune infiltrates were investigated by immunohistochemistry for CD45 (pan-immune marker), CD3 (pan-T cell), CD8 (cytotoxic T cells), B220 (B cells), Mac-1 or CD11b (macrophages), and PD-L1. Cells were also assessed via flow cytometry for surface expression and interferon inducibility of immune markers PD-L1, MHC-I component H2-Kb, and CD155. Reproducibly growing tumors for each of MM001i and MM008i were achieved, with 50% of the tumors reaching the endpoint of 1000 mm3 around day 18 and day 21, respectively. Interestingly, despite a shared ancestral genetic background, ICB treatment of WT mice injected with MM001i and MM008i revealed differential responses. MM001i was poorly responsive, whereas MM008i showed complete tumor regression following treatment with anti-PD-1. In contrast, both MM001i and MM008i responded rapidly to anti-CTLA-4. These differential responses were not explained by surface expression of PD-L1 and MHC-1 component H2-Kb, which appeared to be basally and inducibly (IFN-β or IFN-γ) expressed at similar levels. Experiments are ongoing to deduce the underlying molecular mechanism, test additional ICB therapies, and study the impact of A3 mutagenesis. Thus, two novel murine mammary breast cancer cell lines have been established to evaluate immunotherapies in fully immune-competent C57BL/6 mice.
 Selected References:
 1) Schmid, P. et. al. Pembrolizumab for Early Triple-Negative Breast Cancer. N Engl J Med, 382, 810-821 (2020).
 2) Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv, 2, e1601737 (2016).
 3) Gupta, A. et. al. APOBEC3 mutagenesis drives therapy resistance in breast cancer. bioRxiv, https://doi.org/10.1101/2024.04.29.591453 (2024).
 Citation Format: Thomas Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, Reuben S. Harris, Harshita B. Gupta. New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627f15e34d39099a44c9ec865f3f46e6de37a2af" target='_blank'>
              Abstract P1-02-25: New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy
              </a>
            </td>
          <td>
            Thomas J Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, R. Harris, Harshita B. Gupta
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98cddbd2f616ad4a59f0f4eab2b9b0463e8944eb" target='_blank'>
              YTHDC1 orchestrates oncogenic splicing via the CLK1-SRSF1 splicing machinery to regulate castration-resistant prostate cancer progression.
              </a>
            </td>
          <td>
            Parth Gupta, Devesh Srivastava, Vinayak Nayak, B. V. Rao, K. Suseela, Rakesh Sharma, S. Rajappa, T. S. Rao, Parul Mishra, Ashish Misra
          </td>
          <td>2025-05-22</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="By offering a precise, effective, and adaptable method for genome editing, CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) technology has completely transformed genetic biotechnology. Through a biochemical lens that highlights the molecular complexities behind its methods and discoveries, this review explores the complex field of CRISPR cloning and its emerging frontiers. We examine the molecular underpinnings of target identification, protospacer adjacent motif (PAM) specificity, and Cas nuclease activation, starting with the groundbreaking discovery of CRISPR-Cas systems in bacterial adaptive immunity. Advances in CRISPR-based cloning techniques, such as the creation of synthetic guide RNAs, Cas variants with specific functions, and high-fidelity and base-editing systems that reduce off-target effects, are also examined in the study. The molecular dynamics of DNA-RNA-protein interactions during gene targeting, activation of DNA repair pathways, and epigenetic state regulation are given special attention. We investigate cutting-edge uses beyond traditional genome editing, including RNA-targeting Cas proteins (like Cas13), CRISPR-based diagnostics and treatments, CRISPR interference (CRISPRi), and CRISPR activation (CRISPRa). Additionally covered is the incorporation of CRISPR with other biochemical technologies, such as gene drives, programmable transcription factors, and synthetic biology circuits. The field's increasing biochemical sophistication is highlighted by new developments like CRISPR-nanoparticle conjugates for targeted delivery and CRISPR screening platforms for functional genomics. In the end, this review emphasizes how a thorough biochemical comprehension of CRISPR systems is essential to maximizing their potential in environmental engineering, biotechnology, medicine, and agriculture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf5fde83fcfa701e2f38df12fc05920c20f45e1" target='_blank'>
              Exploring CRISPR Cloning and Beyond Through a Biochemical Lens in Genetic Biotechnology
              </a>
            </td>
          <td>
            Mariam Mariam, Khizra Ikhlaq, Zulaikha Abid, Fatima Sahar Uraf Maryam, Syed Qasim Mehmood, Ahtasham Ahmad, Sohrab Anwar Abbasi, Imran Ali Sani, Muhammad Adeel
          </td>
          <td>2025-07-02</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rare genetic DNA repair deficiency syndromes can lead to immunodeficiency, neurological disorders, and cancer. In the general population, inter-individual variation in DNA repair capacity (DRC) influences susceptibility to cancer and several age-related diseases. Genome wide association studies and functional analyses show that defects in multiple DNA repair pathways jointly increase disease risk, but previous technologies did not permit comprehensive analyses of DNA repair in populations. To overcome these limitations, we used fluorescence multiplex host cell reactivation (FM-HCR) assays that directly quantify DRC across six major DNA repair pathways. We assessed DRC in phytohemagglutinin-stimulated primary lymphocytes from 56 healthy individuals and validated assay reproducibility in 10 individuals with up to five independent blood draws. We furthermore developed generalized analytical pipelines for systematically adjusting for batch effects and both experimental and biological confounders. Our results reveal significant inter-individual variation in DRC for each of 10 reporter assays that measure the efficiency of distinct repair processes. Our data also demonstrate that correlations between the activities of different DNA repair pathways are relatively weak. This finding suggests that each pathway may independently influence susceptibility to the health effects of DNA damage. We furthermore developed a pipeline for analyzing comet repair kinetics and related our new functional data to previously reported comet assay data for the same individuals. Our pioneering analysis underscores the sensitivity of FM-HCR assays for detecting subtle biological differences between individuals and establishes standardized methodologies for population studies. Our findings and open source analytical tools advance precision medicine by enabling comprehensive exploration of genetic, demographic, clinical, and lifestyle factors and supporting targeted interventions to enhance DNA repair and maintain genomic integrity, thereby promoting personalized healthcare and disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            Ting Zhai, Patrizia Mazzucato, Catherine Ricciardi, David C. Christiani, Liming Liang, Leona D Samson, I. A. Chaim, Z. Nagel
          </td>
          <td>2025-06-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e275fe9c31757491fc33c855fc96ec1cae73962" target='_blank'>
              Offspring effect of maternal preconception exposure to metabolic disruptors
              </a>
            </td>
          <td>
            Carlos Díaz-Castillo, Stephanie R. Aguiar, Raquel Chamorro-Garcia
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS Chemical Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="This review examines the evolutionary trajectory and functional versatility of RNA, beginning with its proposed involvement in the origin of life and culminating in its current application in therapeutic strategies. We explored the complexity of the transcriptome, splicing mechanisms, and the regulatory functions of non-coding RNAs, especially microRNAs. The processes underlying microRNA biogenesis and activity are discussed in the context of their potential as therapeutic tools. Advances in RNA-based technologies have been further illustrated by the development of mRNA vaccines, representing a significant breakthrough in biomedical innovation. Additionally, we explored the growing field of epitranscriptomics—chemical modifications to RNA that modulate its stability, translation, and function—by analyzing the roles of modification enzymes known as writers, erasers, and readers. Focus is given to how these alterations influence immune reactions and guide the strategic development of future modified mRNA vaccines. Collectively, these advances underscore RNA’s multifaceted roles and its transformative potential in the biological and medical sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3636bee0122dc54808211a7a4a5dbb31783c6b90" target='_blank'>
              RNA Through Time: From the Origin of Life to Therapeutic Frontiers in Transcriptomics and Epitranscriptional Medicine
              </a>
            </td>
          <td>
            Cecilia Martínez-Campos, Humberto Lanz-Mendoza, Jorge Cime-Castillo, O. Peralta-Zaragoza, Vicente Madrid-Marina
          </td>
          <td>2025-05-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background The three-prime repair exonuclease 1, TREX1, degrades cytosolic DNA to prevent aberrant immune activation. Its inactivation results in DNA accumulation in the cytosol and induction of the cGAS-STING DNA sensing pathway, interferon signaling, and inflammation. Germline pathogenic TREX1 mutations are known to lead to hereditary autoimmune and autoinflammatory disorders, whereas the consequences of TREX1 mutations in cancer remain poorly understood. Results To assess the importance of human TREX1 amino acid variants, we analyzed protein sequences of the functional TREX1b isoform from 168 mammalian species and integrated available data on TREX1 sequence and copy number alterations in hereditary autoimmune and autoinflammatory disorders, cancer, and in human populations. While the entire TREX1b protein was conserved in placental mammals, egg-laying mammals and marsupials had their own unique C-terminal regions, with each predicted to contain a transmembrane domain. We modeled human TREX1 variants occurring in autoimmune disease and cancer samples at 12 protein positions to evaluate their predicted impact on protein stability and function. Conclusions Our findings provide novel insight into the role of TREX1 molecular variation in cancer, where genetic or epigenetic loss of TREX1 activity may improve susceptibility to treatment. However, TREX1 gene deletion in tumors was associated with unfavorable patient outcomes, most likely due its frequent co-occurrence with the loss of the entire 3p chromosomal arm, which contains known cancer-related genes. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00785-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a6c0f4b15cdd23d8488729c5a843f6b9a322d6" target='_blank'>
              Prevalence and impact of molecular variation in the three-prime repair exonuclease 1 TREX1 and its implications for oncology
              </a>
            </td>
          <td>
            Marwa Shekfeh, Mariam M. Konaté, Julia Krushkal
          </td>
          <td>2025-06-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer genomics has advanced a lot; a main problem is still turning molecular findings into treatments due to the variety of cells in each tumor and gaps in knowing certain pathway functions. Predicting whether a treatment will be effective is difficult right now which means that patient outcomes are often less than ideal. This research tackled the issue by studying whether looking at many genes and oncogenic pathways at the same time helps identify which patients may show cancer progression or treatment resistance. For this study, we looked at the genes, proteins and other molecules from a total of 1,500 patients in six major types of cancer (breast, colorectal, lung, melanoma, ovarian and prostate). We used genomic variant calling, analysis of gene pathways and detailed study of proteins with advanced bioinformatics and machine learning tools. The study found that TP53 mutations (OR: 2.14, 95% CI: 1.72-2.66, p < 0.001) and the PI3K/AKT/mTOR pathway being activated (OR: 1.89, 95% CI: 1.51-2.37, p = 0.003) were strongly linked to cancer not responding to treatment and negative outcomes. Tumors with a high number of mutations (≥ 10/Mb) reacted much better to immunotherapy (OR: 3.02, p < 0.001). Three subgroups of tumors were found through unsupervised analysis and these showed different chances of success: tumors with many mutations and high immune activity performed best (39.1% complete response), but those with mutations in PI3K and TP53 had the worst prognosis (28.5% progressive disease). Patients whose tumors were active in the PI3K/AKT pathway had progression-free survival of 8.2 months, significantly less (log-rank p < 0.001) than those whose tumors were not active. Evidence shows that integrative molecular profiling is better at predicting outcomes than the traditional method of classifying tumors by looking at slides. Study results show which molecular subgroups are most important to oncology and help link their treatment response to which therapy is best,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e766178e1eebb59cae34bf9ab40a3c9e125bd68f" target='_blank'>
              Integrative Molecular Profiling of Oncogenic Pathways and Genetic Mutations in Cancer Progression and Therapeutic Response
              </a>
            </td>
          <td>
            Javeria Taj, Muhammad Noman Ajmal, Tayyaba Arshad, Afifa Dawood, Ali Abbas, Misbah Hafeez, Hamza Rafeeq, Abdul Malik, Aiman Nishat, Mujahid Hussain
          </td>
          <td>2025-06-20</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The incidence of thyroid cancer has been rising in recent years. While tumorigenesis has traditionally been attributed to the accumulation of genetic mutations in oncogenes and tumor suppressor genes, increasing attention has been directed toward the role of epigenetic regulation in cancer development. Since the 1980s, however, it has been acknowledged that the role of another key regulatory system in carcinogenesis: epigenetics, shedding light on the regulation of gene expression without altering the DNA sequence.This review synthesizes current literature on epigenetic alterations in follicular cell-derived thyroid cancers, focusing on DNA methylation, histone modifications, chromatin remodeling, and RNA regulation. Evidence indicates that dysregulation of these epigenetic processes is prevalent in thyroid cancer, influencing tumor initiation, progression, and resistance to therapy. Several epigenetic inhibitors are under development, some demonstrating synergy with existing chemotherapies and immunotherapies. Understanding these mechanisms may facilitate the development of novel, more effective strategies for early detection and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9841779c66dc42db3c61189adbbdb2f2d61cd598" target='_blank'>
              Epigenetic modifications in follicular cell-derived thyroid cancer: new dimensions in pathogenesis and treatment
              </a>
            </td>
          <td>
            Yanmei Han, Bian Wu, Jie Tan, Ruolin Wu, Jingjing Wang, Xiaojing Ren, Yajing Zhang, Zairong Gao, Xiaotian Xia
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Conventional wisdom holds that nuclear oncogenes and tumor suppressors initiate malignant transformation. However, mounting research suggests that mitochondrial dysfunction—rooted in the unique evolutionary history and genetic autonomy of mitochondria—may serve as a more fundamental driver of oncogenesis. This paper proposes a “mitochondria-first” hypothesis of cancer, emphasizing the pivotal role of mitochondrial DNA (mtDNA) mutations, metabolic reprogramming, and immune evasion. By examining the evolutionary conflict between host and mitochondria, evaluating high mtDNA mutation rates, and highlighting the disruptive potential of mitochondrial transfer to immune cells, we outline robust mechanisms through which mitochondria could ignite cancer development. We also discuss emerging diagnostic and therapeutic approaches that target mitochondrial integrity, offering a potential paradigm shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09759d1fd0b99bd9f5921041ad32e37eb247e60" target='_blank'>
              Are Mitochondria the True Origin of Cancer? A Hypothesis-Driven Perspective
              </a>
            </td>
          <td>
            Sergio Da Silva
          </td>
          <td>2025-07-01</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed CHIP. How these mutations result in increased risk for numerous age-related diseases remains poorly understood. We conduct a multiracial meta-analysis of EWAS of CHIP in the Framingham Heart Study, Jackson Heart Study, Cardiovascular Health Study, and Atherosclerosis Risk in Communities cohorts (N = 8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. We functionally validate the EWAS findings using human hematopoietic stem cell models of CHIP. We then use expression quantitative trait methylation analysis to identify transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses reveal 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value < 0.05). Taken together, our study reports the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f304229b20119f9327d580fcbe571fa1fc8351" target='_blank'>
              Epigenome-wide DNA methylation association study of CHIP provides insight into perturbed gene regulation
              </a>
            </td>
          <td>
            Sara Kirmani, T. Huan, J. V. Van Amburg, R. Joehanes, M. Uddin, N. Q. Nguyen, Bing Yu, Jennifer A. Brody, Myriam Fornage, Jan Bressler, N. Sotoodehnia, David A Ong, Fabio Puddu, James S Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen N Conneely, J. Weinstock, April P. Carson, Leslie Lange, Kendra R Ferrier, N. Heard-Costa, Joanne M Murabito, Alexander G. Bick, Daniel Levy
          </td>
          <td>2025-05-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="
 Background: Hormone Receptor Positive (HR+) breast cancer is predominantly managed with endocrine therapies that target the Estrogen Receptor Alpha (ERα), a key mediator of estrogen's role in promoting tumor growth. However, the emergence of endocrine resistance poses a significant clinical challenge, limiting the efficacy of these treatments and impeding optimal patient outcomes. Understanding the mechanisms behind ERα's importance and the development of resistance is crucial for advancing therapeutic strategies against HR+ breast cancer. Methods: In our study, we utilized the GEO database to identify ADP-Ribosylation Factor-Like GTPase 3 (ARL3) as a gene associated with tamoxifen resistance in HR+ breast cancer. We constructed cell lines with ARL3 knockout and overexpression to assess the impact of ARL3 on cell proliferation, migration, and tamoxifen sensitivity. RNA sequencing (RNA-seq) and liquid chromatography-mass spectrometry (LC/MS) were employed to demonstrate the influence of ARL3 knockout on downstream ERα pathways. Furthermore, co-immunoprecipitation (CO-IP), immunofluorescence and Western blot (WB) assays confirmed the role of ARL3 in the ubiquitination and degradation pathway of ERα. Results: In our analysis of TCGA breast cancer subtypes, we also observed that ARL3 is highly expressed in luminal breast cancer. Additionally, our collection of breast cancer tissue mRNA and immunohistochemistry data revealed that ARL3 expression is elevated in HR+ breast cancer, indicating a correlation with estrogen receptors. We found that ARL3 promotes the proliferation and migration of HR+ breast cancer. Furthermore, overexpression of ARL3 was associated with reduced sensitivity to tamoxifen, and in vivo tumorigenesis in nude mice showed that tumor volume and weight were significantly decreased compared to the control group. To investigate the underlying mechanisms, RNA-seq analysis of ARL3 knockout and control groups revealed a pronounced inhibition of the MYC pathway. Western blot (WB) validation confirmed that ARL3 knockout suppresses the expression of downstream pathways of ERα. Our data suggest that the association between ARL3 and ERα primarily occurs in the cytoplasm, mainly binding to the ligand-binding domain (LBD) of ERα, which is consistent with ARL3 not being recruited to the promoters occupied by ERα. We also discovered that the ARL3/ERα cascade promotes mitochondrial autophagy to enhance mitochondrial oxidative phosphorylation. In summary, our study establishes a non-genomic mechanism whereby ARL3 controls the transcription of estrogen-dependent genes associated with breast cancer cell proliferation by stabilizing ERα levels. Conclusion: In our study, we posit that ADP-ribosylation factor-like GTPase 3 (ARL3) exerts a pivotal influence on the proliferative capacity, metastatic potential, and responsiveness to endocrine therapies in hormone receptor-positive (HR+) breast cancer. ARL3 modulates the stability of estrogen receptor alpha (ERα), thereby regulating the activation of downstream signaling cascades and the mitochondrial functionality within neoplastic cells. Our findings suggest that ARL3 represents a promising therapeutic target for HR+ breast cancer. We envision that in-depth investigation into the molecular underpinnings of ARL3's role could pave the way for the development of innovative pharmacological agents capable of overcoming endocrine resistance in patients, ultimately enhancing their clinical outcomes and survival rates.
 Citation Format: Han Li, Yang Liu, Zehao Cai, Dan Shu, Yang Peng, Kang Li, Shengchun Liu. ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-04-10.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aec8ed2ad5da8e6d551e26dd1027baa5c75f898" target='_blank'>
              Abstract P2-04-10: ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization
              </a>
            </td>
          <td>
            Han Li, Yang Liu, Zehao Cai, Dan-hua Shu, Yang Peng, Kang Li, Shengchun Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved by specialized helicases. Previous in vitro studies showed that DEAH-box helicase 36 (DHX36/G4R1/RHAU) has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in the accumulation of cytoplasmic DNA fragments, an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression, and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 messenger RNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd74a217dd008b29b89824a440aee3aba255159" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, Shangming Tang, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Yuh-Hwa Wang
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c3d47e157de191e71468217bfe01f9c8e3d31f0" target='_blank'>
              ILF2 blockade suppresses helicase-mediated R-loop resolution to induce lethality in homologous recombination-deficient cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, Li Hoi Yeung
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 4582


 Background:
 Lynch syndrome (LS) is a hereditary cancer predisposition syndrome caused by DNA mismatch repair (MMR) deficiency and associated with a 10–25% lifetime risk of urothelial cancer (UC), particularly in the upper urinary tract. We aimed to investigate the somatic genomic landscape of LS-associated urothelial cancer (LS-UC) using targeted and whole-exome sequencing (WES).
 Methods:
 We analyzed 41 surgical tumor samples accrued to five Finnish biobanks between April 1987 - June 2022 and 3 urine DNA samples from 34 LS-UC patients, all enrolled in the Finnish Lynch Syndrome Registry. Tumors were profiled using the UroScout assay, targeting 25 UC-associated genes, to identify somatic mutations. Immunohistochemistry was performed to assess MMR protein loss, and WES was conducted on selected cases to investigate the broader mutation landscape. A comparative analysis of the genomic and mutational landscapes in LS-UC versus sporadic UC was performed.
 Results:
 We show that telomerase reverse transcriptase (
 TERT
 ) promoter mutations found in 83% of sporadic UC are almost completely absent (5%) in LS-UC (p < 0.00001). Instead, all LS-UC exhibited a 5-methylcytosine deamination (CG > TG) and microsatellite instability driven mutation landscape, characterized by highly frequent
 ARID1A
 (82%),
 FGFR3
 (80%), and
 KMT2D
 (78%) mutations, as well as preferential usage of CG > TG mutation hotspots. We propose that scarcity of
 TERT
 promoter mutations in LS-UC is due to inability to create the GABP binding motif 5’-GGAA through CG > TG mutation or microsatellite instability. Additionally, many mutation hotspots recurrently mutated in sporadic UC were not present in LS-UC.
 Conclusions:
 Our findings establish LS-UC as a distinct disease entity with a unique genomic signature driven by constrained hypermutation. Our proposed explanation that
 TERT
 mutations are absent in LS-UC due to constrained hypermutation is supported by our discovery that other UC driver genes also exhibit an altered mutation landscape in LS-UC. These insights advance the understanding of LS-UC tumorigenesis and support the development of tailored diagnostic and therapeutic approaches for this patient population.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ccc15478489e03787575f6fc2fa196f9bfcafda" target='_blank'>
              Distinct genomic landscape of Lynch syndrome–associated urothelial cancer.
              </a>
            </td>
          <td>
            Jussi Nikkola, Lauri Ryyppö, Juuso Vuorinen, Lauri Moilanen, Maarit Ahtiainen, K. Pylvänäinen, Hanna Selin, Tuomo Virtanen, M. Nykter, T. Veitonmäki, J. Mecklin, T. Seppälä, M. Annala
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The problem of interpretation of the genetic data from patients with inherited cardiovascular diseases still remains relevant. To date, the clinical significance of approximately 40 % of variants in genes associated with inherited cardiovascular diseases is uncertain, which requires new approaches to the assessment of their pathogenetic contribution. A combination of the induced pluripotent stem cell (iPSC) technology and editing the iPSC genome with CRISPR/Cas9 is thought to be the most promising tool for clarifying variant pathogenicity. A variant of unknown significance in MYH7, p.Met659Ile (c.1977G>A), was previously identified in several genetic screenings of hypertrophic cardiomyopathy patients. In this study, the single nucleotide substitution was corrected with CRISPR/Cas9 in iPSCs generated from a carrier of the variant. As a result, two iPSC lines (ICGi019-B-1 and ICGi019-B-2) were generated and characterized using a standard set of methods. The iPSC lines with the corrected p.Met659Ile (c.1977G>A) variant in MYH7 possessed a morphology characteristic of human pluripotent cells, expressed markers of the pluripotent state (the OCT4, SOX2, NANOG transcription factors and SSEA-4 surface antigen), were able to give rise to derivatives of three germ layers during spontaneous differentiation, and retained a normal karyotype (46,XY). No CRISPR/Cas9 off-target activity was found in the ICGi019-B-1 and ICGi019-B-2 iPSC lines. The maintenance of the pluripotent state and normal karyotype and the absence of CRISPR/Cas9 off-target activity in the iPSC lines with the corrected p.Met659Ile (c.1977G>A) variant in MYH7 allow using the iPSC lines as an isogenic control for further studies of the variant pathogenicity and its impact on the hypertrophic cardiomyopathy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb56531c5e4ea40f0f8c8f05402c5581e5ae495" target='_blank'>
              Generation of the ICGi019-B-1 and ICGi019-B-2 lines via correction of the p.Met659Ile (c.1977G>A) variant in MYH7 of patient-specific induced pluripotent stem cells using CRISPR/Cas9
              </a>
            </td>
          <td>
            A. E. Shulgina, S. Pavlova, J. Minina, S. Zakian, E. V. Dementyeva
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA methylation is a primary layer of epigenetic modification that plays a pivotal role in the regulation of development, aging, and cancer. The concurrent activity of opposing enzymes that mediate DNA methylation and demethylation gives rise to a biochemical cycle and active turnover of DNA methylation. While the ensuing biochemical oscillations have been implicated in the regulation of cell differentiation, their functional role and spatio-temporal dynamics are, however, unknown. In this work, we demonstrate that chromatin-mediated coupling between these local biochemical cycles can lead to the emergence of phase-locked domains, regions of locally synchronized turnover activity, whose coarsening is arrested by genomic heterogeneity. We introduce a minimal model based on stochastic oscillators with constrained long-range and non-reciprocal interactions, shaped by the local chromatin organization. Through a combination of analytical theory and stochastic simulations, we predict both the degree of synchronization and the typical size of emergent phase-locked domains. We qualitatively test these predictions using single-cell sequencing data. Our results show that DNA methylation turnover exhibits surprisingly rich spatio-temporal patterns which may be used by cells to control cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64b62b80c82b947292c6cec2280cb45271854a1" target='_blank'>
              Spatio-temporal patterns of active epigenetic turnover
              </a>
            </td>
          <td>
            Fabrizio Olmeda, Misha Gupta, Onurcan Bektaş, Steffen Rulands
          </td>
          <td>2025-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MYC is an aberrantly regulated transcription factor implicated in approximately 70% of human tumors, where it extensively modulates signaling pathways associated with cancer progression. Inactivating MYC has been shown to inhibit tumor growth and even induce sustained tumor regression across various cancer types, a phenomenon referred to as oncogene addiction. However, in vitro studies reveal that the knockout or knockdown of MYC in numerous tumor cell lines does not necessarily result in cell death, despite these tumors exhibiting MYC addiction in vivo. This discrepancy suggests that the unique tumor microenvironment in vivo may play a critical role in facilitating MYC addiction in cancer cells. MYC is also widely acknowledged for its role in mediating the immune evasion of tumor cells. Nevertheless, due to the extensive regulation of cellular gene expression by MYC and the incomplete understanding of the mechanisms underlying tumor immune escape, the precise pathways through which MYC influences tumor immune evasion remain inadequately elucidated. Recent years have seen the identification of novel tumor immune escape mechanisms, some of which have been demonstrated to be directly or indirectly regulated by MYC. For instance, MYC may contribute to immune evasion by modulating the expression of argininosuccinate synthetase 1 (ASS1), a key enzyme involved in arginine biosynthesis. Herein, in this study, we explore some novel potential mechanisms through which MYC facilitates the immune evasion of tumor cells, alongside a combined therapeutic approach targeting MYC and employing immunotherapy based on this mechanism. Furthermore, we suggest that targeting proteins interacting with MYC to modulate its expression and function may serve as an alternative strategy to direct MYC targeting, thereby expediting the clinical translation of combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61614de2defb80c90945b2feff409bc535c7b7f8" target='_blank'>
              Beyond the Limit: MYC Mediates Tumor Immune Escape
              </a>
            </td>
          <td>
            Zhongyang Hong, Sitong Ming, Xin Luan, Zhe Sun, Weidong Zhang
          </td>
          <td>2025-06-29</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H. Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8533a84bf23dfa93f9b981af3fd2fcfd03e25b" target='_blank'>
              AURKA inhibition amplifies DNA replication stress to foster WEE1 kinase dependency and synergistic antitumor effects with WEE1 inhibition in cancers
              </a>
            </td>
          <td>
            Jong Woo Lee, Jackie Shi, Julian Barrantes, Pratima Chaurasia, Sundong Kim, S. Cruz-Gomez, Cindy Yang, Jianlei Gu, Hongyu Zhao, Dejian Zhao, Benjamin Schiff, Ansley Roche, Elizabeth B. Perry, Jeffrey P. Townsend, Gary M. Kupfer, Katerina Politi, E. Golemis, Barbara Burtness
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Purpose The tumor microenvironment (TME) may provide a useful framework for understanding the heterogeneous behavior of growth hormone (GH) secreting pituitary adenomas. Although the interest in TME in somatotropinomas has increased exponentially over the last few decades, there is limited elucidation of its mechanisms, particularly in relation to genes expression involved in its regulation. Methods A retrospective, observational, single-center study was conducted on 85 subjects: 46 patients diagnosed with acromegaly and 39 controls. After DNA extraction, clinical exome sequencing was performed and genomic alterations were detected, classified, and filtered using a dedicated bioinformatics pipeline. Results 5759 unique genetic variants were found in patients with acromegaly. 33 patients (72%) showed the presence of at least one pathogenic variant in at least one of the following genes: FANCD2, SPTA1, TYRO3, and ZNF335. The enrichment pathway analysis of mutated genes was performed and showed that these genes were included in the same genetic pathway called “regulation of lymphocyte activation” (GO:0051249). Inflammatory infiltrate was analyzed in histological samples in 26 patients. A significantly higher number of CD68 + macrophages (P-value = 0.008), a lower number of CD8 + T lymphocytes (P-value = 0.037) and a higher CD68 + macrophages/ CD8 + T-lymphocytes ratio (P-value = 0.004) were observed in patients with pathogenic variants of genes of “regulation of lymphocyte activation” pathway. Conclusion This study provides new insights into the genetic basis of the TME in somatotropinomas and suggests that genetics may influence immune cells infiltration in acromegaly. Supplementary Information The online version contains supplementary material available at 10.1007/s11102-025-01537-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea366143793ec3bdf679833ee8e098198deac01" target='_blank'>
              Genes of the “regulation of lymphocyte activation” pathway may influence immune cells infiltration in growth hormone secreting pituitary tumors
              </a>
            </td>
          <td>
            S. Chiloiro, Flavia Costanza, G. L. Scaglione, Filippo Russo, Carmela Nardelli, A. Giampietro, P. Mattogno, L. Lauretti, Guido Rindi, L. De Marinis, Marco Gessi, Antonio Bianchi, Francesco Doglietto, E. Capoluongo, Alfredo Pontecorvi
          </td>
          <td>2025-05-26</td>
          <td>Pituitary</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung cancer is among the leading causes of cancer-related death in the U.S. Cigarette smoking remains the leading risk factor for lung cancer and can cause somatic mutations in the critical tumor suppressor gene TP53, among others. Mutations in TP53 not only cause loss of wild-type function but may also introduce oncogenic, gain-of-function (GOF) properties. The frequency of missense mutations at residues p.V157 and p.R158 in p53 increases dramatically in lung cancer relative to other cancers. These p53 mutants exhibit both loss of wild-type p53 function and GOF properties, including broadly rewiring gene expression programs in lung cancer cells. Several pan-cancer hotspot mutations in p53 impart GOF activities that reprogram cellular metabolism. To refine our understanding of the GOF properties conferred by the lung-enriched p53 mutants, the cellular metabolism of cells containing mutant p53 (V157F) was investigated. Untargeted metabolomics revealed that glutathione metabolism is among the top altered metabolic pathways related to mutant p53 (V157F). p53 mutants V157F and R158L provided resistance to oxidative stress induced by both menadione and cigarette smoke extract. The cell death experienced in the absence of mutant p53 (V157F; R158L) was due to the increase in reactive oxygen species (ROS). These findings suggest that the lung-enriched mutations in p53 (V157F; R158L) confer lung cancer cells with resistance to ROS, and ROS accumulation in the absence of mutant p53 causes cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8d3955759f3ec29e6914d7584360c523775fbf" target='_blank'>
              Somatic p53 mutations that are markedly overrepresented in lung cancer confer resistance to ROS-induced cell death.
              </a>
            </td>
          <td>
            Mason A Tracewell, Jonathan Karlin, Samantha M. Barnada, Elizabeth L McDuffie, Charles P Scott, J. Barta, Steven B. McMahon
          </td>
          <td>2025-06-03</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
This study reviews how ionizing radiation (IR) induces alternative splicing (AS) in non-tumor and tumor cells under both ex vivo and in vivo irradiation conditions. The relevance and limitations of IR-induced AS in identifying potential biomarkers are highlighted for two main applications: biodosimetry and radiotherapy.


CONCLUSIONS
Radiation promotes alterations in AS, which may differentially affect the response in both tumor and non-tumor cells. This response can occur in genes that change their overall expression as well as in those that remain unaltered in response to IR. Although cis-regulators modulate AS, trans-regulators like splicing factors are more involved in the IR response. Variants of key genes involved in the DNA damage response (DDR) are regulated in non-tumor cells while they are often deregulated in tumor cells favoring radioresistance. Identifying IR-induced AS variants could enhance the sensitivity of biodosimeters for dose estimation and biomarkers for radiosensitivity, offering potential strategies to personalize radiotherapy and improve outcomes. New and advanced sequencing technologies will allow variant identification important for the field of radiobiological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c740ec1aef5f884cca33c90567c580fecc7177" target='_blank'>
              Alternative splicing in human cells exposed to ionizing radiation: a comprehensive review of ex vivo and in vivo studies.
              </a>
            </td>
          <td>
            Jerónimo Leberle, Marina Perona, A. Choudhury, N. Bellora, Irene L Ibañez, Luisa V. Biolatti
          </td>
          <td>2025-05-22</td>
          <td>International journal of radiation biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Temple syndrome is an imprinting disorder resulting from abnormal genomic or epigenomic aberrations of chromosome 14 including maternal uniparental disomy (matUPD), paternal deletion of 14q32, or aberrant methylation of the imprinting control regions at 14q32. Understanding the underlying molecular mechanism is essential to understanding the recurrence risk and physical effects. Currently, diagnosis requires the detection of aberrant methylation and copy number loss via methylation-sensitive assays such as methylation-specific multiplex ligation-dependent probe amplification, and short tandem repeat analysis to detect matUPD and the presence of epimutation. Therefore, a one-step approach that can detect aberrant methylation and underlying genetic mechanisms would be of high clinical value. Here we use nanopore sequencing to delineate the molecular diagnosis of a case with Temple syndrome. We demonstrate the application of nanopore sequencing to detect aberrant methylation and underlying genetic mechanisms simultaneously in this case, thus providing a proof of concept for a one-step approach for molecular diagnosis of this disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c13f17671e59c8d86a86517592417eb48513c1" target='_blank'>
              Using long-read sequencing to detect and subtype a case with Temple syndrome
              </a>
            </td>
          <td>
            Sarah Dada, Vahid Akbari, Duha Hejla, Yaoqing Shen, Katherine Dixon, S. Choufani, Rosanna Weksberg, Cornelius F Boerkoel, Laura Stewart, Kamilla Schlade-Bartusiak, Emma Strong, Danya Fox, Daniel Gamu, William T. Gibson, Steven J M Jones
          </td>
          <td>2025-05-29</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8569a1e64e6674714c2e763b602f6c886995d4db" target='_blank'>
              Multiple FGFR1 mutations modulate tumorigenic mechanisms in glioneuronal tumors
              </a>
            </td>
          <td>
            Jacopo Boni, Míriam Fernández-González, HyeRim Han, Carla Roca, Cassandra J. Wong, Cristina Rioja, C. Nogué, Leticia Manen-Freixa, J. Boulais, Endika Torres-Urtizberea, Antonio Gómez, M. Hasselblatt, R. Estrada-Tejedor, A. Antolin, Islam E. Elkholi, Nada Jabado, Jean-François Côté, A. Gingras, Barbara Rivera
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="An individual’s genetics contributes to their risk of developing amyotrophic lateral sclerosis (ALS); however, there is still a large proportion of the heritability of ALS to be understood. Part of this missing heritability may lie in complex variants, such as the long interspersed element 1 (L1) retrotransposon, which have yet to be evaluated. The majority of L1 insertions in the human genome are no longer able to retrotranspose, but to date 279 retrotransposition-competent (RC) L1s have been reported. Many RC-L1s are polymorphic for their presence/absence; therefore, each individual will have a different number and complement of RC-L1s. These elements have been hypothesized to be involved in disease processes by multiple mechanisms such as somatic mutation by retrotransposition, the triggering of neuroinflammation and DNA damage. We hypothesize that L1s may influence disease development either through their effects on endogenous genes or through the properties that enable them to retrotranspose. Whole genome sequencing data from the New York Genome Center ALS consortium were used to characterize L1 variation identifying 2,803 polymorphic L1 elements and association analysis was performed in European individuals (ALS/ALS with other neurological disorder (ALSND) n = 2,653, controls n = 320). There were no individual L1 elements associated with disease, but we did identify a significant increase in the number of RC-L1s in ALS/ALSND genomes (p = 0.01) and the presence of ≥46 RC-L1s showed the most significant association (OR = 1.09 (1.02–1.16), p = 0.01) with disease. Analysis of individual L1s and their association with age at onset and survival identified one L1 whose presence was significantly associated with a lower age at onset (52.7 years) compared to homozygous absent individuals (59.2 years) (padj = 0.009). Our study has identified novel genetic factors for both disease risk and age at onset in ALS providing further evidence for the role of L1 retrotransposons in neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215cd8618896e86063f645cabe583a6274cdedb5" target='_blank'>
              Retrotransposition-competent L1s are increased in the genomes of individuals with amyotrophic lateral sclerosis
              </a>
            </td>
          <td>
            A. Pfaff, Sulev Kõks
          </td>
          <td>2025-06-03</td>
          <td>Experimental Biology and Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d072dee701b15034f00be3c11a965c7ef7b90a4" target='_blank'>
              Clonal memory of cell division in humans diverges between healthy haematopoiesis and acute myeloid leukaemia
              </a>
            </td>
          <td>
            A. Donada, G. Hermange, T. Tocci, A. Midoun, G. Prevedello, L. Hadj Abed, D. Dupré, W. Sun, I. Milo, S. Tenreira Bento, C. Pospori, A. Innes, C. Willekens, J. Vargaftig, D. Michonneau, C. Lo Celso, N. Servant, KR Duffy, H. Isambert, PH Cournède, L. Laplane, L. Perie
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 e16019


 Background:
 Homologous recombination repair (HRR), a key pathway in DNA damage response (DDR), is linked to the efficacy of platinum-based chemotherapy, PARP inhibitors, and immunotherapy across cancers. However, its mutational landscape and impact on the immune microenvironment in gastric cancer remain unclear.
 Methods:
 Next-generation sequencing (NGS) and multiplex immunofluorescence were used to analyze the mutational profiles and immune microenvironment of 213 gastric cancer patients, focusing on the impact of HRR mutations.
 Results:
 Of the 213 patients, 67 (31.45%) harbored HRR mutations, including 46 with variants of uncertain significance (VOUS). The most frequently mutated HRR genes were RAD50 (4.69%), BLM (4.23%), and BRCA2(2.35%), with BLM/RAD50 co-mutations being the most common (2.35%). HRR mutations were significantly co-occurring with KMT2D mutations and mutually exclusive with TP53 mutations. We analyzed the association of HRR pathway mutations with immune biomarkers, stratifying patients by pathogenic/likely pathogenic and VOUS mutations. Pathogenic mutations were linked to high tumor mutational burden (TMB; p < 0.0001) and microsatellite instability-high (MSI-H) status but showed no association with PD-L1 expression (CPS = 1/5/10) or EBV infection. Similar trends were seen for VOUS mutations. Further analysis of HRR mutation burden (≥2 mutated HRR genes) demonstrated an association with high TMB and MSI-H status, as well as a tendency toward positive PD-L1 CPS expression (CPS ≥ 1). Immune microenvironment analysis revealed that HRR- patients had higher proportions of M2 macrophages, suggesting polarization toward an immunosuppressive phenotype. However, the M1/M2 macrophage ratio showed no significant difference, potentially due to compensatory effects from other immune cells. HRR- patients also displayed elevated CD56dim NK cells and a higher CD56dim/bright NK cell ratio, indicating a dominance of cytotoxic NK cells. In addition, HRR- patients had more CD4⁺ T cells (p = 0.0004) and a higher CD4/CD8 ratio, though not statistically significant. The observed infiltration of CD4⁺FOXP3⁻ T cells suggests that HRR mutations might influence immune cell composition and contribute to the formation of a non-inflammatory immune microenvironment in HRR-mutant gastric cancers.
 Conclusions:
 In gastric cancer, pathogenic/likely pathogenic HRR mutations are associated with higher TMB levels and MSI, while HRR variants of uncertain significance show a similar trend. Immune microenvironment analysis suggests that HRR pathway mutations may be linked to a non-inflammatory immune microenvironment. These findings indicate that HRR mutations could be relevant to the efficacy of immunotherapy in gastric cancer patients, warranting further investigation into their clinical value.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ceb40e645e4f8a6eec7df1e8de409f9b6d52363" target='_blank'>
              Implications of homologous recombination repair pathway mutations on the immune microenvironment in gastric cancer.
              </a>
            </td>
          <td>
            Feng Jin, Ping Jiang, Huayang Zhong, Zengjin Hu, Ning Gao
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antagonistic activities of the 53BP1 axis and the tumor suppressor BRCA1-BARD1 determine whether DNA double-strand breaks (DSBs) are repaired by end joining or homologous recombination. We show that the CTC1-STN1-TEN1 (CST) complex, a central 53BP1 axis component, suppresses DNA end resection by EXO1 and the BLM-DNA2 helicase-nuclease complex but acts by distinct mechanisms in restricting these entities. Whereas BRCA1-BARD1 alleviates the CST-imposed EXO1 blockade, it has little effect on BLM-DNA2 restriction. CST mutants impaired for DNA binding or BLM-EXO1 interaction exhibit a hyper-resection phenotype and render BRCA1-deficient cells resistant to poly(ADP-ribose) polymerase (PARP) inhibitors. Our findings mechanistically define the crucial role of CST in DNA DSB repair pathway choice and have implications for understanding cancer therapy resistance stemming from dysfunction of the 53BP1 axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edcb82922ba3661ab5becf214c56aecb500e3589" target='_blank'>
              CTC1-STN1-TEN1 controls DNA break repair pathway choice via DNA end resection blockade.
              </a>
            </td>
          <td>
            Cody M Rogers, Hardeep Kaur, Michelle L Swift, Vivek B Raina, Shuo Zhou, Ajinkya S Kawale, Shahrez Syed, Korilynn G Kelly, Angela M Jasper, Sameer Salunkhe, Youngho Kwon, Jeffrey Wang, Aida Badamchi Shabestari, J. Daley, Adam Sacks, M. Gaczynska, P. Osmulski, Yashpal Rawal, N. Tomimatsu, S. Gayther, K. Lawrenson, S. Burma, Elizabeth V. Wasmuth, Shaun K. Olsen, Weixing Zhao, R. Hromas, David S. Libich, Alexander V. Mazin, Daohong Zhou, E. Greene, Dipanjan Chowdhury, Patrick Sung
          </td>
          <td>2025-05-22</td>
          <td>Science</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81a8c4d68f310ed7b4d9b39860d8a3a3880f7f9c" target='_blank'>
              Macroscopic Analyses of RNA-Seq Data to Reveal Chromatin Modifications in Aging and Disease
              </a>
            </td>
          <td>
            Achal Mahajan, Francesca Ratti, Ban Wang, Hana El-Samad, James H. Kaufman, Vishrawas Gopalakrishnan
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 4154


 Background:
 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a dismal prognosis despite advances in treatment. PDAC is characterized by a dense stromal environment with suppressed anti-tumor immunity that contributes to treatment resistance. Oncogenic
 KRAS
 mutations are present in > 90% of PDACs and are known to play a role in modulating the tumor immune microenvironment (TME). In this study, we examined the differences in TME components between
 KRAS
 -mutant (m) and
 KRAS
 -wildtype (wt) PDACs.
 Methods:
 Comprehensive genomic and immune profiling (CGIP), including the RNA-seq based gene expression assessment of 395 immune-associated genes, was performed on 311 PDAC patient samples. Gene expression signatures of tumor immunogenicity (TIGS) and cell proliferation were calculated by averaging the gene expression ranks of 161 immune-associated genes and 10 proliferation genes, respectively. The normalized gene expression rank of 22 immune checkpoint genes and 17 cancer testis antigen (CTA) genes were also calculated. DNA-seq was used to identify
 KRAS
 mutations and to calculated tumor mutational burden (TMB). Continuous variables were compared between subgroups using the Wilcoxon Rank-Sum test and categorical variables were compared between groups using the Chi Squared test. For statistical significance, p < 0.05 was required.
 Results:
 The cohort consisted of 311 PDAC patient samples, comprising 159 females (51.1%) and 152 males (48.9%) with a median age at testing of 69.3 years (38.7-92.6). A total of 264 specimens (84.9%) exhibited a
 KRAS
 mutation, with the most common being G12D and G12V while 47 (15.1%) had
 KRAS
 -wt tumors. No difference in TIGS, CP, or TMB was observed between
 KRAS
 -wt and
 KRAS
 -m tumors.
 KRAS
 -wt tumors exhibited greater expression of 12 of 17 tested CTAs than
 KRAS
 -m tumors, including GAGE13 (p = 0.004) NY-ESO-1 (p = 0.01), MAGEA3 (p = 0.01), and LAGE1A (p = 0.01). Analysis of the gene expression of 22 immune checkpoint genes showed no difference for most of the genes, though there was higher expression of PD-1 (p = 0.04) and CD27 (p = 0.03) seen in
 KRAS
 -wt compared to
 KRAS
 -m tumors.
 Conclusions:
 KRAS
 -wt PDACs exhibited greater expression of cancer testis antigen genes compared to
 KRAS
 -m tumors, suggesting potential therapeutic susceptibility to immunotherapy and adoptive cell therapies leveraging the expression of CTAs as targets. Assessment of
 KRAS
 status and immunotherapy susceptibility may support future clinical trial selections for therapies targeting the complex interplay of genomic and immune components of pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae6d1fe40ee2b5375e715aee2512cfe9d76d6fb" target='_blank'>
              Unveiling the differences in tumor immune microenvironment between
 KRAS
 -wildtype and
 KRAS
 -mutant pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Heidi Ko, S. Pabla, Michelle F Green, M. Senosain, Hardik Parikh, Erik Van Roey, Shuang Gao, Yamuna Pulivendula, P. DePietro, Stuti Kaushalkumar Patel, Kyle C. Strickland, R. Previs, Erin Newburn, Kamal S Saini, Alicia Dillard, Eric A Severson, Brian Caveney, Taylor J. Jensen, Shakti H. Ramkissoon, R. Seager
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d2630c6a248d7d989516a74e5f3c41a92ecf258" target='_blank'>
              R-loops shape chromatin architecture to promote balanced lineage allocation during differentiation
              </a>
            </td>
          <td>
            Chun-Hao Chao, T. Fazzio
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The canonical DNA damage response (cDDR) maintains genome stability, involving DNA synthesis arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously characterized a noncanonical response specific to nonblocking replication stress, i.e. low-level stress (LoL-DDR), in primary cells. Although LoL-DDR generates replication stress-induced ROS (RIR), it prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a threshold triggers the cDDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: (1) RIR are not detoxified under high-level stress, resulting in a continuous increase in the dose‒response curve of RIR production; (2) RIR are not produced by NADPH oxidases; (3) replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” replication-stress/ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05172-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1826ee141a3bb5d6a8079b9d816c3e423819f87e" target='_blank'>
              Dysregulation of the low-level replication stress response in transformed cell lines
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-06-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Mutational signatures are increasingly used to understand the mechanisms causing cancer. However, their important applications in predicting prognosis and stratifying patients for therapy are hampered by inaccurate inference of the various featureless, dense trinucleotide mutational spectra, which are often confounded with one another. One of them is the homologous recombination deficiency (HRd)-associated signature SBS3, relevant because of its association with prognosis in ovarian and breast cancer and because of its potential as a biomarker for synthetic lethality therapies. Methods Here, we highlight strong benefits of a multimodal approach for mutational signature extraction, applied on top of standard bioinformatic pipelines. By jointly operating on single-base substitution (SBS) and indel (ID) spectra, this method enables accurate identification of various DNA repair deficiency signatures and patient survival prediction. Results Across four different cohorts of whole-genome sequenced high-grade serous ovarian cancers (HGSOC), the multimodal SBS + ID approach correctly distinguished the commonly confused signatures SBS3, SBS5, SBS8, SBS39, and SBS40. Importantly, we robustly identified two different multimodal SBS3 signatures, m-SBS3a and m-SBS3b, with distinct patterns in the indel spectrum. Multimodal SBS3b signature was strongly predictive of longer survival in ovarian cancer patients, replicating across four cohorts, with effect sizes greatly exceeding other genetic markers. Our m-SBS3 also predicted survival in platinum-treated patients with various cancer types, and moreover, the SBS + ID joint inference was successfully applied to mismatch repair-deficient colorectal cancer and immunotherapy response, supporting a general utility of the multimodal mutational signatures approach. Conclusions Overall, combining SBS and ID mutations improves detection of HR deficiency-associated signatures and reveals distinct SBS3 subtypes with prognostic value. This multimodal approach outperforms existing markers and is readily applicable to therapy stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01497-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01c11401f870f4aa8e6266e2fe8e3ab75a0da51" target='_blank'>
              Joint inference of mutational signatures from indels and single-nucleotide substitutions reveals prognostic impact of DNA repair deficiencies
              </a>
            </td>
          <td>
            Patricia Ferrer-Torres, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-07-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the significant progress made in the development of epigenetic age (eAge) clocks designed to estimate the various aspects of aging, currently available models, generated using large DNA methylation microarray datasets, still cannot fully address the issues of batch effects and technical variation. This hinders the use of the publicly available eAge clocks in routine laboratory practice, and it motivates the development of cost-effective, custom epigenetic clocks that are tailored to the given biological subjects and research methods. In this study, we analyzed the local DNA methylation of mesenchymal stem cell samples during culture expansion using high-throughput targeted bisulfite sequencing (BS-seq). Using the obtained data, we trained a minimized eAge model based on a Random Forest Regression with Leave-One-Out Cross-Validation, which determines cell passage with good performance (MAE 1.094 and R2 0.897) and which is comparable to previous solutions. Using the advantage of BS-seq to analyze consecutive CpGs methylation patterns, we demonstrated that combining the analysis of average DNA methylation levels with local methylation heterogeneity scores—thereby reflecting stochastic DNA methylation dynamics—can improve the quality of the epigenetic clock models. Therefore, we propose a research strategy for creating customized epigenetic clocks using targeted BS-seq and provide a mechanistic conceptualization of how information on longitudinal changes in DNA methylation patterns can potentially be used for the assessment of specific aging aspects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162b59e95b20d78aba8a2d1263685e994dd6fbb1" target='_blank'>
              A Novel Framework for the Design of Minimized Epigenetic Clocks Using the Analysis of DNA Methylation Heterogeneity
              </a>
            </td>
          <td>
            S. E. Romanov, Dmitry I. Karetnikov, D. A. Kalashnikova, Denis E. Polivcev, Yakov A. Osipov, D. A. Maksimov, Polina A. Antoshina, Viktor V. Shloma, E. Samoilova, Alina A. Ivanova, Rustam F. Karimov, A. N. Tkalin, Alexander A. Shevchenko, V. Kalsin, V. Baklaushev, P. P. Laktionov
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Cetuximab resistance severely restricts its effectiveness in the treatment of patients with metastatic colorectal cancer (CRC). Previous studies have predominantly focused on the genetic level, with scant attention to the nongenetic aspects. This study aimed to identify the crucial microRNA (miRNA) that is responsible for cetuximab resistance. Methods Key miRNAs were identified using small RNA sequencing analysis. miR-196b’s role and mechanism in cetuximab resistance was explored by in vitro and in vivo experiments. Clinical blood samples were dynamically analyzed using droplet digital polymerase chain reaction (PCR) to assess the predictive value of miR-196b for efficacy. Results We initially discovered that the extracellular signal-regulated kinase (ERK) signaling pathway was progressively activated during the acquisition of cetuximab resistance in CRC cells. Further study determined that miR-196b can inhibit the activity of ERK and protein kinase B (AKT) signaling pathways by downregulating both NRAS and BRAF, which can kill two birds with one stone, thus enhancing the sensitivity of colorectal cancer cells to cetuximab. The expression of miR-196b was found to be significantly downregulated in both cetuximab-resistant cells and the tumor tissues of patients exhibiting resistance. In the presence of cetuximab, overexpression of miR-196b further inhibited the proliferation and migration and promoted the apoptosis of CRC cells, while miR-196b silencing had the opposite effects. Importantly, analysis of clinical blood samples confirmed that miR-196b can serve as a predictive and dynamic biomarker for monitoring the outcomes of patients with CRC treated with cetuximab. Conclusions This study supports that activation of the ERK signaling pathway is a key factor in cetuximab resistance. In addition, miR-196b can modulate and predict the CRC response to cetuximab, holding broad potential applications. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00740-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/276e7963bb9e3ca916c3eb1d74f4c8841c5d3360" target='_blank'>
              miR-196b strictly regulates and reliably predicts the response to cetuximab in colorectal cancer
              </a>
            </td>
          <td>
            Shiyun Chen, Zhaoli Tan, Yanli Lin, Fang Pang, Xiaojie Wu, Xiang Li, Yumeng Cui, Weiling Man, Ying Li, Yanghua Li, Lu Han, Miaomiao Gou, Zhikuan Wang, Guanghai Dai, Youliang Wang
          </td>
          <td>2025-05-25</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c9d768db6c320ae3141a12d174f6fdda8a0e39f" target='_blank'>
              Modified hTERT promoters-driven purine nucleoside phosphorylase-gene therapy in association with chemo- and targeted therapy in the context of ovarian cancer.
              </a>
            </td>
          <td>
            Quoc Manh Nguyen, Pierre-François Dupré, Mathieu Berchel, Marylène Lévêque, Sylvie Choblet-Thery, F. d’Arbonneau, T. Montier
          </td>
          <td>2025-06-27</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutational signatures of single-base substitutions (SBSs) characterize somatic mutation processes which contribute to cancer development and progression. However, current mutational signatures do not distinguish the two independent steps that generate SBSs: the initial DNA damage followed by erroneous repair. To address this modelling gap we developed DAMUTA, a hierarchical Bayesian probabilistic model that infers separate signatures for each process, and captures their sample-specific interaction. We applied DAMUTA to 18,974 pan-cancer whole genome sequencing mutation catalogues from 23 cancer types and show that tissue-specificity in mutation patterns is driven largely by variability in damage processes. We also show that misrepair processes are predictive of DNA damage response deficiencies. Unlike existing approaches, DAMUTA distinguishes damage from misrepair contributions, and we demonstrate significant improvements over a mutational-burden baseline or signatures from the COSMIC database. Our analysis reveals a shared pan-cancer pattern of early clonal transition-mutations which shifts to a more uniform substitution pattern consistent with increased reliance on translesion synthesis for damage tolerance. DAMUTA thus generates a compact set of signatures which resolves redundancies of current signature models, disentangles the effects of DNA damage and misrepair processes, and facilitates improved stratification of tumours, while providing a framework towards a unified pan-cancer model of the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3964588c1648b53d4e30a576d480451079a470ad" target='_blank'>
              Damage and Misrepair Signatures: Compact Representations of Pan-cancer Mutational Processes
              </a>
            </td>
          <td>
            Caitlin F. Harrigan, Kieran R. Campbell, Quaid Morris, Tyler Funnell
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2fb143b1900a1c66ed78148b97787d2aa07d158" target='_blank'>
              Investigation of a germline variant of uncertain significance (MRE11:c.1138C > T) identified by exome sequencing in a cancer-affected individual and co-segregation analysis in affected and unaffected family members.
              </a>
            </td>
          <td>
            Mohammadreza Saberiyan, Mohammad Reza Seyedtaghia, Elham Mojodi, A. Nejatizadeh
          </td>
          <td>2025-05-31</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) is a crucial regulator of genomic integrity, frequently mutated in more than half of all human cancers. These mutations predominantly target the DNA-binding domain (DBD), impairing p53's interaction with DNA and its tumor-suppressive functions. To elucidate the structural and functional consequences of p53 mutations, we investigated 148 missense variants located within its DNA-binding interface using cutting-edge computational approaches. We employed AlphaFold3 (AF3) to predict p53-DNA complex structures, integrating these predictions with molecular dynamics (MD) and force-guided pulling simulations to assess mutation-induced changes in structural stability and DNA-binding properties. Moreover, we compared the results of our study with experimental in vitro enrichment scores (RFS) and Combined Annotation Dependent Depletion (CADD). We identified a moderate negative correlation between plDDT and the pathogenicity of mutant variants, suggesting that mutations causing more significant alterations in the protein tertiary structure have a greater negative impact on cellular function. Moreover, we identified two possible structural mechanisms through which mutations can impair the p53 functionality. Specifically, some mutations, such as R248P and N239S, reduce the binding affinity of the p53-DNA complex, whereas others, such as C238Y and P278R, enhance affinity but compromise the structural stability of the complex. Furthermore, we uncovered mutations with potential rescuing effects, such as E285A and M243T, which preserved structural stability and enhanced the DNA-binding ability. Our findings provide a comprehensive framework for understanding the molecular mechanisms underlying p53 mutations and their role in cancer pathogenesis. This study highlights the value of integrative computational approaches in investigating protein-nucleic acid interactions, providing critical insights that can guide the development of therapeutic strategies targeting p53 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab5ce93ebd4a1fbc747fc251d769b1eab00ffa3" target='_blank'>
              Investigation of Structural Mechanisms Underlying p53 Dysfunction Caused by 148 Missense Mutations Using AlphaFold3 and Molecular Dynamics Simulations.
              </a>
            </td>
          <td>
            Khondamir R Rustamov, J. I. Razzokov, A. Baev
          </td>
          <td>2025-06-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2968d582c4d09600f6a6aab6083cc272395bfdb6" target='_blank'>
              MicroRNA-driven organ-specific metastasis in triple-negative breast cancer: biogenesis, mechanisms, and therapeutic approaches.
              </a>
            </td>
          <td>
            Deepshikha Rathore, Sweny Jain, Shanaya Patel, Nandani Dharwal, N. Shukla, Chirag Desai, Jigna Shah, Heena V. Dave
          </td>
          <td>2025-06-28</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01390d6c99756b411d8005c0d7bc115177a2c238" target='_blank'>
              A Bayesian Approach for Identifying Driver Mutations within Oncogenic Pathways through Mutual Exclusivity
              </a>
            </td>
          <td>
            Xinjun Wang, Caroline E. Kostrzewa, Allison Reiner, Ronglai Shen, Colin B Begg
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance Here, we address a major limitation in the effectiveness of poly(ADP-ribose) polymerase inhibitors (PARPi) in breast cancer gene (BRCA)-mutant prostate cancer treatment: Only ~50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARPi resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss-of-function mutations in the DNA prereplication complex (pre-RC) confer PARPi resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-RC, can restore PARPi sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95a3ba5883a1720490b54e4008aa7d7b0986c0b" target='_blank'>
              BRCA2 reversion mutation–independent resistance to PARP inhibition through impaired DNA prereplication complex function
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, Nikolaus Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles L Sawyers
          </td>
          <td>2025-06-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Multiple sclerosis (MS) is a neuroinflammatory disease where oxidative stress and DNA damage may influence disease progression. We investigated whether defects in base excision repair (BER) pathways contribute to MS by combining functional DNA repair assays, gene expression profiling, and genotype analysis. We collected peripheral blood mononuclear cells from 70 MS patients and 61 healthy controls. These cells were subjected to tert-butyl hydroperoxide (TBH)-induced oxidative stress, and comet assay kinetics were measured over a period of 60 min. Additionally, we quantified the mRNA expression of nine key BER genes and genotyped selected polymorphisms related to DNA repair capacity. Samples from MS patients exhibited significantly higher levels of TBH-induced DNA lesions and displayed a distinct repair trajectory over time, as indicated by area-under-the-curve (AUC) analyses (p < 0.001). The transcripts of MBD4 and NTHL1 were notably reduced in MS patients compared to those in the controls (p < 0.0001). A logistic regression analysis revealed an association between the specific BER-related single nucleotide polymorphisms (SNPs) rs3087404, rs4135054, and rs1052133 and ineffective DNA repair. Subset analyses of B cells, CD4+ cells, and CD8+ cells further supported the presence of altered repair kinetics in MS, even though some subsets exhibited similar baseline lesion levels. Our findings suggest that impaired oxidative DNA repair is present in MS, likely driven by functional deficits in repair kinetics and alterations in the expression of BER genes and polymorphisms. This integrated approach highlights DNA repair pathways as potential therapeutic or prognostic targets in MS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1132f1e6d566b473e04bbd8dc4095ffb1f1f7b35" target='_blank'>
              Oxidative DNA Damage and Repair Dynamics in Multiple Sclerosis: Insights from Comet Assay Kinetics, Base Excision Repair Gene Expression, and Genotype Analysis
              </a>
            </td>
          <td>
            Beata Filipek, A. Macieja, Aleksandra Binda, Rafał Szelenberger, Leslaw Gorniak, Elżbieta Miller, Mariola Świderek-Matysiak, M. Stasiołek, Ireneusz Majsterek, Tomasz Popławski
          </td>
          <td>2025-05-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99ef5f46003666cce9bfbffdcaf4920a36fe552" target='_blank'>
              RNA G-quadruplexes: emerging regulators of gene expression and therapeutic targets.
              </a>
            </td>
          <td>
            Zukela Ruzi, Wei Zha, Huang Yuan Yuan, Jiaorui Liu
          </td>
          <td>2025-07-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f43f79f58977259121120764f54b519011ee5e" target='_blank'>
              RBM5 recruiting MGC32805 in a sandwich mode and inducing ΔFAS neoantigen and triggering FAS properties switch: implication in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhe Wu, Xiaoyun Hu, Yilin Wang, Xianglong Zhu, Qing Zhao, Ying Zhao, Wanlin Cui, Mingrong Zhang, Haishan Zhao, Kai Li, Minjie Wei
          </td>
          <td>2025-06-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>