<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-05 09:41:56 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>80</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8adbd0387ac6e2aa073dbe75080e4f98049b516f" target='_blank'>
              Acute Myeloid Leukemia Relapse after Bromodomain Inhibitor Treatment or Chemotherapy is Characterized by Myc-Ras Transcriptional Remodeling
              </a>
            </td>
          <td>
            Benjamin J. Huang, Jeevitha D’Souza, Alexa Rane Batingana, M. Harris, Xinyue Wang, E. Hwang, Anica M Wandler, Michael R. Burgess, Qing Li, S. Meshinchi, Gideon Bollag, Kevin Shannon
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-29898-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e404655f34f2f760a22785dfe7969af3c340a1ca" target='_blank'>
              Genome-wide CRISPR screens identified C18orf32 as a novel regulator of lipid metabolism that mediates PFOA-induced toxicity
              </a>
            </td>
          <td>
            Chanhee Kim, A. Tagmount, Zhaohan Zhu, W. B. Barbazuk, Rhonda L. Bacher, Christopher D. Vulpe
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Drug tolerant persister cells (DTPs) refer to a transient drug-tolerance sub-population of cancer cells characteristics of phenotype plasticity and heterogeneity. This adaptive cell state is a critical transitional phase, standing on the crossroad that cancer cells reacquire drug sensitivity or enter into the permanent drug resistance. Emerging evidences indicate the epitranscriptomic regulations, particularly RNA methylations are the important mechanism underline post-transcriptional regulations of genes expression across all RNA species. RNA is integral to gene expression as messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA), which play roles in transmitting information from DNA to the synthesis of functional proteins. Methylation modifications on these RNAs are prevalent and represent a well-recognized non-genetic mechanism, exerting multifaceted regulatory effects on nucleic acid metabolism, such as nucleotide precursor availability, RNA processing dynamics, sub-cellular localization, transcript stability and translational fidelity/ efficiency. This review systematically sorts out the relevant references, demonstrating recent advances on the knowledge of the patterns of methylation modifications on mRNA, tRNA and rRNA, and how these modifications drive the generation and development of DTPs, which hallmarks of epithelial-mesenchymal transition, metabolism shift and immune escape. And then clinical strategies are delineated, leveraging pharmacological modulators of RNA-modifying enzymes alongside non-pharmaceutical lifestyle advice, for the development of therapy strategies preventing DTPs-rooted tumor relapse in this anti-tumor armamentarium with cytotoxic reagents, targeted therapies and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098e2b3a70bd6f86f19a2d17ea69012683c3e930" target='_blank'>
              Therapy-induced mRNA, rRNA and tRNA methylation alterations confer tolerance phenotype in tumor cells: mechanism and implications
              </a>
            </td>
          <td>
            Anfeng Jiang, Shujie Liu, Zhiyuan Li, Xiongzhou Zhang, Minghao Duan, Bin Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding how individual genetic backgrounds shape the effects of disease-associated mutations is central to elucidating the biology of complex psychiatric disorders. We developed a scalable ‘village editing’ strategy that enables simultaneous genome editing across multiple induced pluripotent stem cell (iPSC) lines, allowing systematic assessment of how polygenic context modulates the impact of specific mutations. Using pooled CRISPR editing in 15 iPSC lines spanning a range of schizophrenia (SCZ) polygenic risk scores, we generated homozygous and heterozygous knockouts in two known SCZ-associated genes: LRP1, involved in cholesterol import, and NRXN1, a presynaptic adhesion molecule. By mixing all lines prior to editing and de-multiplexing them afterward, we efficiently produced multi-donor knockout neurons at scale. Transcriptomic profiling revealed that LRP1 and NRXN1 loss produce both shared and donor-specific effects on neuronal gene expression, with variable perturbation of neurotransmitter transport and cholesterol biosynthesis pathways across genetic backgrounds. These results demonstrate that village editing enables systematic dissection of gene-background interactions in human neurons, offering a powerful framework for studying the polygenic architecture of psychiatric disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaa852025370979851c5d71b4c4220d7d271580" target='_blank'>
              Gene editing in “cell villages” enables exploring disease-relevant mutations in many genetic backgrounds
              </a>
            </td>
          <td>
            Rachel A. Battaglia, Sonia Bolshakova, Ilinca Mazureac, Dhara Liyanage, Noah Pettinari, Autumn Johnson, Ethan Crouse, Sartaj Habib, Isabel Flessas, Ajay Nadig, Derek Hawes, Matt Tegtmeyer, Caroline Becker, S. Ghosh, G. Genovese, Marina Hogan, Adrianna Maglieri, Lindy E. Barrett, Laurence Daheron, Steven A. McCarroll, R. Nehme
          </td>
          <td>2025-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The reproducibility of scientific research has been increasingly challenged in recent years. While extrinsic factors (e.g., cross-contamination, mycoplasma infection, serum variability) are well-studied, the role of intrinsic attributes like cell passage number remains underexplored. Using two tumor cell lines (ACHN and Renca), we employed RNA sequencing to analyze transcriptomic dynamics across passages (P3 to P39). Results revealed a nonlinear transcriptomic shift: mid-passage cells (P10/P11, P17) showed heightened activity in cell cycle, metabolism, and stress response, whereas low (P3) and high passages (P24/P39) exhibited stable and similar profiles. KEGG analysis indicated significant alterations in signaling, immune, and metabolic pathways during passaging. This study demonstrates that passage number shapes transcriptional landscapes and impacts experimental reproducibility. We advocate for strict passage control and explicit reporting of passage information to enhance reliability. These findings underscore the need for standardized cell culture practices to improve research reproducibility. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-29424-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility
              </a>
            </td>
          <td>
            Shuai Liu, Hai-juan Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer recurrence remains a major clinical challenge, often associated with therapy resistance and altered metabolic states. To define metabolic vulnerabilities of recurrent disease, we performed a CRISPR knockout screen targeting 421 metabolic genes in paired primary and recurrent HER2-driven breast cancer cell lines. While both primary and recurrent tumors shared dependencies on core metabolic pathways, recurrent tumors exhibited selective essentiality for the de novo pyrimidine synthesis pathway, including Cad, Dhodh, and Ctps. Pharmacologic inhibition of the rate-limiting enzyme DHODH with BAY-2402234 selectively impaired the growth of recurrent tumor cells, while primary tumor cells were relatively resistant. BAY treatment robustly inhibited pyrimidine synthesis in all lines, but only recurrent cells underwent iron-dependent lipid peroxidation and ferroptotic cell death. Lipidomic profiling revealed enrichment of polyunsaturated ether phospholipids in recurrent cells, which may predispose them to ferroptosis. A sensitizer CRISPR screen in primary cells further identified nucleotide salvage and lipid metabolic pathways as modifiers of DHODH inhibitor sensitivity. Stable isotope tracing and nutrient depletion experiments showed that primary cells can compensate for DHODH inhibition through nucleotide salvage, whereas recurrent cells exhibit impaired salvage capacity, likely due to reduced expression of Slc28/Slc29 nucleoside transporters. Together, these findings reveal that breast cancer recurrence is associated with increased dependence on de novo pyrimidine synthesis to suppress ferroptosis, highlighting a therapeutically actionable metabolic vulnerability in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768d3fe55602ad3e6e64d51b9bc66ae2727e39e3" target='_blank'>
              TP53 and RB1 are predictive genetic biomarkers for sensitivity to cytarabine in gliomas
              </a>
            </td>
          <td>
            L. Laaniste, Joachim Luginbuehl, Mi Trinh, Rachel Watkins, Gregor Lueg, Grant Neilson, Marjolein Burgers, Kasia Kania, Regina Reynolds, Zoe Smith, Julia Kleniuk, Lauren Gould, Dorota Pregowska, Sophie Mathiesen, Michael Johnson, Louwai Muhammed
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df86684e553e2308bc95a51da59d93a391350fdb" target='_blank'>
              Human CSB-deficient iPSCs exhibit impaired DNA damage repair and stress responses following BPDE exposure in an early developmental model
              </a>
            </td>
          <td>
            Alessia Lofrano, Wasco Wruck, Nina Graffmann, James Adjaye
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, W. Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Understanding how cellular pathways interact is crucial for treating complex diseases like cancer. Individual gene–gene interaction studies have provided valuable insights, but may miss pathways working together. Here we develop a multi-gene approach to pathway mapping which reveals that acute myeloid leukaemia (AML) depends on an unexpected link between complex II and purine metabolism. Through stable-isotope metabolomic tracing, we show that complex II directly supports de novo purine biosynthesis and that exogenous purines rescue AML cells from complex II inhibition. The mechanism involves a metabolic circuit where glutamine provides nitrogen to build the purine ring, producing glutamate that complex II metabolizes to sustain purine synthesis. This connection translates into a metabolic vulnerability whereby increasing intracellular glutamate levels suppresses purine production and sensitizes AML cells to complex II inhibition. In a syngeneic AML mouse model, targeting complex II leads to rapid disease regression and extends survival. In individuals with AML, higher complex II gene expression correlates with resistance to BCL-2 inhibition and worse survival. These findings establish complex II as a central regulator of de novo purine biosynthesis and a promising therapeutic target in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97c8d8e5a5e06c05f0708f44487c55edde76d75" target='_blank'>
              Pathway coessentiality mapping reveals complex II is required for de novo purine biosynthesis in acute myeloid leukaemia
              </a>
            </td>
          <td>
            Amy E. Stewart, Derek K. Zachman, Pol Castellano-Escuder, Lois M. Kelly, Ben Zolyomi, M. D. Aiduk, Christopher D Delaney, Ian C. Lock, Claudie Bosc, John Bradley, Shane T. Killarney, J. D. Stuart, Paul A Grimsrud, Olga R. Ilkayeva, Christopher B. Newgard, Navdeep S. Chandel, Alexandre Puissant, Kris C. Wood, M. Hirschey
          </td>
          <td>2025-12-05</td>
          <td>Nature Metabolism</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The budding yeast Saccharomyces cerevisiae is a well-established model for studying the genetic basis of complex traits, and it is a powerful system for investigating mechanisms of aging. Here, we examine the genomic and transcriptomic factors contributing to increased replicative age in recombinant yeast populations harboring standing genetic variation. Using Fluorescence-Activated Cell Sorting (FACS), we isolated young and aged cohort pairs across twelve biological replicates and sequenced their progeny to assess patterns of differentiation at the nucleotide and transcription levels. Most differentiated alleles were located in coding regions, including significant variants within 132 unique genes. Transcriptomic analysis revealed 60 differentially expressed genes in aged populations, including 18 genes with increased expression in aged cohorts, and 42 genes with decreased expression. Although only two genes (RFA3 and WSC4) were implicated in both genomic and transcriptomic analyses, functional overlap associated with protein homeostasis, DNA repair, and cell cycle regulation was evident across datasets. Notably, we found no strong evidence that differentially expressed genes were more likely to occur in close proximity to significant gene variants. This suggests that late-life survival is not predominantly governed by local cis-regulatory interactions (e.g. variants within or near coding regions). These findings underscore the power of integrating genomic and transcriptomic data to elucidate the genetics of complex traits such as aging, demonstrating how multi-omics approaches can reveal functional relationships that may be overlooked by single-layer analyses. Significance Statement While many individual genes contributing to aging and lifespan have been identified, our understanding of the polygenic interactions and regulatory processes that contribute to phenotypic variation in these traits is much more limited. Using a recombinant population of yeast, we identify novel links between genetic variation and the phenotype of replicative age. Additionally, we find little evidence for local cis-regulatory interactions, suggesting that downstream regulation or trans-regulatory processes may serve more dominant roles in modulating aging. These results reveal new insights into the role of polygenicity in the evolution and regulation of age-associated phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5421275a5bb574745168cdf381917f8c4a541df8" target='_blank'>
              Age-specific genomic and transcriptomic variation reveals limited evidence for cis-regulatory interactions modulating aging in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            K. McHugh, Felipe S Barreto, Molly K Burke
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Berberine (BBR), a natural compound with diverse anticancer properties, exerts potent inhibitory effects on lung cancer cell proliferation. However, its particular molecular targets remain unknown. The present study aimed to identify the key genetic determinants that mediate the cellular response to BBR in lung cancer. To achieve this, genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) knockout screening was conducted in A549 cells, followed by functional validation and mechanistic assays. In the study, the genome-wide CRISPR/Cas9 knockout screening in A549 lung cancer cells identified claudin-1 (CLDN1) as a critical modulator of BBR sensitivity. Notably, whilst CLDN1 was enriched in the positive selection screen, its knockout markedly increased the sensitivity of A549 cells to BBR, thus leading to enhanced G1-phase arrest and reduced proliferation. These findings suggest that CLDN1 could serve a dual role, promoting cellular resistance under selective pressure, while simultaneously demonstrating a therapeutic vulnerability when directly inhibited. Overall, the present study identified CLDN1 as a key mediator of the anticancer effects of BBR, thus providing a foundation for its potential development as a therapeutic target to optimize lung cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4f6034fc6b67d610759292e57a0e8b62f65752" target='_blank'>
              Genome-wide CRISPR screening identifies CLDN1 as a central node in the anticancer mechanisms of berberine and as a therapeutic target in lung cancer
              </a>
            </td>
          <td>
            Xiaogang Wang, Benzhong Mi, Jinfeng Lv, Xin Tang, Jingyu Huang, Fei Liu, Yajing Xing, Xingjian Wen, Yongde Wang, Yadong Zhong
          </td>
          <td>2025-12-01</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapeutic failure in cancer often arises from the ability of tumor cells to adapt to external stressors such as drug exposure, metabolic and oxidative stress, and immune surveillance within the tumor microenvironment (TME). This adaptive capacity is a significant factor contributing to poor clinical outcomes and the lethality associated with many aggressive cancer types. During tumor progression, a subpopulation of cells may acquire stem-like characteristics, leading to the development of cancer stem cells (CSCs), which play a pivotal role in resistance and disease recurrence. A key driver of this process is epigenetic reprogramming, which alters histone modifications and chromatin architecture in response to environmental stimuli. These modifications activate genes associated with stemness by creating transcriptionally permissive chromatin regions, while simultaneously silencing genes involved in differentiation, thereby maintaining the tumor cells’ undifferentiated state. This cellular plasticity enables cancer cells to undergo transdifferentiation or dedifferentiation, enhancing their capacity for survival and adaptation. Importantly, distinct transcriptional states are often regulated by specific epigenetic signatures, some of which are valuable for tumor subtype classification, prognosis assessment, and treatment response prediction. Furthermore, certain epigenetic biomarkers show considerable promise for detecting tumor recurrence and monitoring minimal residual disease with high sensitivity. In this review, we examine how epigenetic reprogramming influences chromatin plasticity to promote stemness and therapeutic resistance. Additionally, we discuss ongoing translational research aimed at leveraging these mechanisms to advance precision oncology, with the ultimate goal of improving diagnostic accuracy and therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) persists as a leading contributor to global cancer-associated mortality. Identifying key oncogenic drivers is crucial for improving therapeutic strategies. This study aimed to investigate the role of TICRR in LUAD progression and its potential as a therapeutic target. Methods Hub genes were screened through integration of DepMap CRISPR-Cas9 data, TCGA expression profiles, and survival analysis. TICRR expressions was assessed in LUAD tissues, adjacent controls, and cell lines by RT-qPCR and immunohistochemistry. Functional roles were examined using MTT, colony formation, transwell, wound healing, and EdU assays in vitro. Bioinformatics analyses, including GSEA, somatic mutation profiling, immune correlation, CMap drug sensitivity, and CT-based radiomics, were performed to explore mechanisms, therapeutic potential and clinical correlation. Results TICRR expression was observed to be significantly elevated in LUAD tissues and cell lines, and its higher levels correlated with unfavorable patient outcomes and enrichment of malignant pathways, including EMT, E2F targets, and PI3K/AKT/mTOR signaling. High TICRR expression correlated with distinct somatic mutation patterns, increased tumor mutation burden, and elevated immune checkpoint expression. In vitro, TICRR knockdown suppressed Lung cancer progression. CMap analysis identified KU0063794 and CDK inhibitors as potential therapeutic agents targeting TICRR. Using a CT-based radiomics approach, the predictive model revealed a positive correlation between TICRR infiltration and radiological features in LUAD patients. Conclusion TICRR functions as a critical oncogenic driver in LUAD, promoting proliferation, invasion, and immune evasion. Targeting TICRR may represent a novel strategy for personalized treatment of LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8879ea7af18a8373ac313167b49d629d7e64cc7" target='_blank'>
              Multiomics analysis demonstrated that TOPBP1 interacting checkpoint and Replication Regulator may serve as an immune-related biomarker indicative of poor prognosis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Chengbin Lin, Ziyi Chen, Wenbin Song, Long He, Hongyan Yu, Gaofeng Liang, Keyun Zhu, Jinxian He, Cangchang Shi
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer drug resistance, whether intrinsic or acquired, underlies most relapses and treatment failures. Reliable preclinical models are crucial to define resistance mechanisms and test counterstrategies. This overview article concisely compares three model classes: (1) clinical, patient‑derived xenografts that retain tumor heterogeneity and can mirror patient resistance; (2) induced‑resistance models produced by prolonged drug selection in vivo or in vitro that recapitulate tumor evolution under therapy; and (3) engineered isogenic cell lines that isolate specific resistance drivers. We summarize some key resistance mechanisms revealed and potential therapeutic approaches informed by these models, including rational combinations, mutation‑targeted inhibitors/degraders, efflux/epigenetic modulators, and immune‐related combinations. Each model has trade‑offs, but integrating them accelerates mechanistic insight and translational drug development. This overview guides selection of preclinical models and design of strategies to overcome cancer drug resistance. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/329bb2e5a0a76225e84bcf3ca5e48cf0783cbcff" target='_blank'>
              Overview of Preclinical Drug‐Resistant Tumor Models and Their Application in Drug Discovery
              </a>
            </td>
          <td>
            Bin Li, Ting Ni, Dong Wang, Gaoxiang Liu, Wenting Shi, Zhixiang Zhang, Qingyang Gu, Jian Xiang
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cellular stresses induce transcription readthrough, whereby RNA polymerase II elongates past a gene’s polyadenylation cleavage site without RNA cleavage. Readthrough has been reported in several cancer types. Here, we use long-read sequencing of nascent RNA to quantify transcriptional readthrough in chronic myelogenous leukemia (CML) cells and characterize early responses to the targeted therapeutic, imatinib. We show that the amount, length, and gene-specificity of readthrough increase within 1 hour, while gene expression and alternative splicing alterations emerge later. Strikingly, imatinib-dependent mRNA isoform changes involved “readthrough chimeras”, in which exons from an upstream gene are alternatively spliced to exons in a downstream gene. Modifications in mRNA isoforms and chimera levels detected at 18 hours were also found in imatinib-resistant K562 as well as CML patient cells, suggesting a cascade of early changes in the fidelity of transcription and splicing, leading to long-term adjustment in gene expression and the development of therapy resistance. Teaser Precision RNA sequencing was used to discover the earliest response of leukemia cells treated with targeted therapy: transcriptional readthrough.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b375d141f32c298d7ef188a6d1771b933355525" target='_blank'>
              Transcriptional readthrough precedes alternative splicing programs triggered in CML cells by imatinib
              </a>
            </td>
          <td>
            P. Podszywałow-Bartnicka, Morgan Shine, Jing Lin, Karla M. Neugebauer
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7247a45454e42ff687acafe0ee75660b9dd942d" target='_blank'>
              A modular lentiviral system for multiplexed gene perturbation and functional analysis reveals interdependence of hormone receptors in breast cancer growth in vivo
              </a>
            </td>
          <td>
            Seher Alam, Daria Matvienko, Johan Boström, Maria J. Pires, Aikaterini Papadimitropoulou, Saeed Eshtad, Kumar Sanjiv, Hong Qian, Nicholas C. K. Valerie, C. Brisken, Mikael Altun
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bef70c3f25aeda45fe8b2f85d9f47b8192a3581b" target='_blank'>
              Epigenetic silencing of transposable elements by IRF2BP2 is a selective dependency of myeloid leukemia
              </a>
            </td>
          <td>
            Jingru Xu, Jessica Whittle, Liam Clayfield, M. Lie-a-ling, Eve Leddy, Duncan Smith, Robert Sellers, Ozgen Deniz, G. Lacaud
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader.
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c23f53baca0dc3b384bd4e216c9b8503ff2be41" target='_blank'>
              Pooled single-cell CRISPRa/i screens for functional genomics in bacteria at scale
              </a>
            </td>
          <td>
            Jacob R. Brandner, Quoc Tran, Yujia Huang, Dmitry Sutormin, Karl D. Gaisser, Georg Seelig, J. Zalatan, James M. Carothers, A. Kuchina
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Alternative splicing (AS) is one of the principal mechanisms of post-transcriptional regulation that confers transcriptomic plasticity and proteomic diversity in cancer, thereby enabling tumor adaptation to therapeutic pressure. However, two obstacles impede the translation of these findings into clinical benefit: The absence of systematic functional annotation of the numerous splice variants associated with drug resistance and the paucity of biomarkers capable of distinguishing de novo from acquired splice-mediated resistance. In the present review, the current mechanistic understanding of AS-driven drug resistance was briefly synthesized, and it was evaluated how existing strategies address these challenges. It was also described how knowledge of dysregulated splicing networks, due to mutations in cis-regulatory elements such as ESS, overexpression of trans-acting factors such as SRSF1, as well as mechanisms such as alternative trans-splicing, in which the spliceosome interacts with splice sites on two distinct RNA molecules and which can be driven by complementary sequences or other trans-acting factors, could be used to more accurately identify tumors dependent on aberrant splicing for survival. In addition, it was outlined how targeting aberrant splice variants to overcome therapeutic resistance can be achieved, such as through spliceosome inhibition (for example, H3B-8800) or antisense oligonucleotides directed to a specific exon or splice junction (for example, targeting exon 2 of MET, which is implicated in cis-regulated AS isoforms, or alternatively spliced isoforms of BCL2L1, BRAF and CDYL). However, therapeutic strategies to target adaptive resistance mechanisms such as AS remain limited, as intratumoral heterogeneity may facilitate the emergence of resistant subpopulations, and as most spliceosome inhibitors are not spliceosome-specific, they exhibit off-target effects. Importantly, it was also discussed how pan-cancer splicing databases and single-cell isoform expression profiling can be integrated with deep-learning models, thereby informing the design of therapeutic strategies to overcome splicing-mediated adaptive drug resistance. Notably, such integration will enable the rational design of isoform-specific combination regimens to dismantle drug-resistance circuits. It is anticipated that the present review will assist the scientific community, including both basic and translational researchers, in translating these findings into interventions that mitigate therapeutic failure in recalcitrant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f0892d48c3453d0a0ca7ac5b857f619a850559" target='_blank'>
              Alternative splicing in cancer drug resistance: Mechanisms and therapeutic prospects (Review)
              </a>
            </td>
          <td>
            Wenpu Zhu, Zikun Wu, Chai Luv, Banghe Bao, Weijie Zhao, Guanglong Chen, Hang Yang, Huihan Ai, Faqiang Liu, Zhi Li
          </td>
          <td>2025-12-05</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumour suppressor TP53 (tumor protein p53) is a master regulator of cell cycle, DNA repair, and apoptosis, and its mutation is a hallmark of cancer, with individual mutations exerting distinct effects on tumour biology. Despite accounting for ~7% of all TP53 variants, splice site mutations remain the least studied class, and their functional and clinical consequences are poorly understood. We analyzed 25,058 TP53 variants (18,562 somatic; 6496 germline) to characterize the frequency, molecular impact, transcriptional effects, genomic instability, and clinical outcomes of splice mutations. These alterations showed distinct distributions and substitution patterns between germline and somatic contexts and were frequently associated with copy number alterations, reduced TP53 mRNA, and variable protein expression. Transcriptomic profiling identified two transcriptional phenotypes: one with global suppression of canonical p53 target genes and another with mixed activation and repression independent of tumour type. Genomic instability was elevated in a subset of splice-mutant tumours, correlating with increased relapse risk, while other splice mutations showed lower instability but divergent clinical outcomes, including unexpectedly poor prognoses. Our findings fill a critical knowledge gap, defining the biological and clinical spectrum of TP53 splice site mutations and highlighting their potential as prognostic biomarkers and therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df743368c74812062cf79c30615a68f562fc982" target='_blank'>
              TP53 Splice Mutations Have Tumour-Independent Effects on Genomic Stability and Prognosis: An In Silico Study
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, Debina Sarkar, Aaron Jeffs, T. Slatter, Antony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aneuploidy induces chromosomal scale alterations in gene dosage, impacting organismal proliferation yet serving as a driver for adaptive evolution. The complexity of gene dosage effects makes it challenging to elucidate the causal genetic basis of aneuploid consequence. Here, using loss-of-function screening with Synthetic Chromosome Rearrangement and Modification by LoxP-mediated Evolution (SCRaMbLE) in synthetic aneuploid yeast, in conjunction with gain-of-function testing in euploid yeast, we established a sufficient and necessary framework and discovered cases of nonlinear polygenic dosage effects in driving aneuploid phenotypes. We identified an emergent effect resulting from copy number alterations in a locus of five genes, which enhance trehalose biosynthesis and confer heat tolerance in aneuploid yeast with additional chromosome III. Additionally, a gene dosage-dependent antagonistic epistasis effect of two genes YCL039W and YCL037C determines rapamycin resistance in aneuploid yeast by regulating the Ras signaling pathway. Moreover, several cases of sufficiency-necessity asymmetries were found for other aneuploid traits. Together, our findings provide direct evidence of various dosage-dependent nonlinear polygenic interactions in shaping aneuploid phenotypes and advance understanding of the genetic basis of cellular adaptive evolution. Elucidating the genetic basis of aneuploid phenotypes has posed a challenge. Here, the authors use loss-of-function screening in synthetic aneuploid yeast and gain-of-function testing in euploid yeast to identify various dosage-dependent nonlinear polygenic interactions driving aneuploid traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aa66abdbe2f26aef5c2e61643d28e6d2cd19105" target='_blank'>
              Inducible chromosomal rearrangement reveals nonlinear polygenic dosage effects in driving aneuploid yeast traits
              </a>
            </td>
          <td>
            Hongyi Yin, Zhou Guo, Chu Zhou, Lu Ma, Yi Wu, Ying-jin Yuan
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539b2ee1d2d15c9295867b71d2600ec2c2b281c5" target='_blank'>
              MELK controls tumor metabolism to promote resistance to melanoma therapy
              </a>
            </td>
          <td>
            A. C. B. Sant’Anna-Silva, P. Abbe, Sophie Pagnotta, N. Amoedo, Michael D. Torno, Fanny Bonnard, T. Botton, Frédéric Larbret, Cyril Ronco, H. Montaudié, I. Ben-Sahra, Caroline Robert, Thierry Passeron, Rodrigue Rossignol, Michaël Cerezo, Stéphane Rocchi
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edbcbe66a598eaa27dab20a758b907c70d1442c" target='_blank'>
              Adaptive Responses Directed by CREB Control Epithelial-Mesenchymal Plasticity in Cancer
              </a>
            </td>
          <td>
            Kshitij Parag-Sharma, Harish Bharambe, John J. Powers, W. Gong, Daniel J. Purcell, Caleb Mensah, Chloe Twomey, Jay H. Mehta, Samy Lamouille, Antonio L. Amelio
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bladder cancer (BLCA) is the predominant type of urothelial carcinoma in urinary system, and resistance to cisplatin-based chemotherapy substantially worsens clinical outcomes, presenting a major therapeutic obstacle. In this study, we integrated genome-wide association study (GWAS) data with expression quantitative trait loci (eQTL) analyses, and applied Mendelian randomization (MR) to assess the causal effects of eQTLs from 19,942 genes on BLCA. By incorporating scRNA-seq data, our study also identifies differentially expressed genes (DEGs) in cisplatin-resistant BLCA cells and examined their causal associations with BLCA, aiming to elucidate genetic drivers of chemoresistance and tumor progression. Through this integrated approach, we identified the eQTL of the ARHGEF12 gene as a key mediator of cisplatin resistance. Bioinformatic analysis revealed that elevated ARHGEF12 expression was strongly associated with activation of the PI3K/Akt signaling pathway. To define ARHGEF12’s role in cisplatin resistance, we established a cisplatin-resistant UM-UC-3/DDP model. Silencing ARHGEF12 markedly reduced chemoresistance, increased apoptotic cell death, and induced pronounced morphological changes. Pharmacological modulation with the ROCK inhibitor Y-27632 and a rescue assay with the Akt activator SC79 supported a model in which ARHGEF12 drives chemoresistance via RhoA/ROCK-dependent activation of the PI3K/AKT axis. This study is the first to integrate MR with single-cell transcriptomics to explore the genetic contribution to cisplatin resistance in BLCA. Our results uncover a novel mechanistic role of ARHGEF12 in BLCA progression and chemoresistance and suggest it as a potential therapeutic target for precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542b443be27492e921709c204aeb3560eab221c4" target='_blank'>
              Integrative multi-omics and single-cell transcriptomics reveal ARHGEF12 driving chemoresistance in bladder cancer
              </a>
            </td>
          <td>
            Kunyao Zhu, Zhejiao Zhang, Tinghao Li, Yan Sun, Junlong Zhu, Linfeng Wu, Xiaoyu Zhang, Hang Tong, Zijia Qin, Aimin Wang, Weiyang He
          </td>
          <td>2025-11-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by complex genomic background and treatment resistance. We first defined “Titin (TTN) inactivation”, a state of TTN expression deficiency or mutation, affecting tumor progression. However, how TTN inactivation regulates immune escape and affects therapeutic resistance remains unclear. Using whole-exome sequencing, single-cell transcriptome sequencing, and spatial transcriptome sequencing, we screened the clinical features of TNBC patients with TTN inactivation who accepted neoadjuvant therapy. Meanwhile, we used CRISPR-Cas9 technology to construct various mutant TNBC cell lines. Lentiviral vector carrying TTN and delta-like ligand 4 (DLL4) was validated in vivo to verify potential mechanism. Myeloid-derived suppressor cells (MDSCs) metabolic function was measured using glycolysis-related molecular experiments. Immunotherapeutic agents against TNBC with TTN inactivation were explored in the orthotopic MCT4fl/fl genetically modified mouse model. DLL4-regulated ecological niche was established in TNBC with TTN inactivation. Mechanistically, TTN deficiency and mutation led to DLL4 secretion in TNBC. DLL4 enhanced MCT4-mediated glycolysis via MDSCs-derived NOTCH2 signaling pathway, driving the malignant function and lactate acid excretion of MDSCs. DLL4-derived MDSCs promoted stemness-mediated drug resistance by inducing histone lactate modification in TNBC, suppressing the anti-tumor activities of CD8+T cells. Blocking the DLL4-MCT4 axis stimulated anti-tumor immunity and synergized with anti-PD-1, improving response rates for first-line neoadjuvant therapy in TNBC. Our study revealed intrinsic mechanism by which TTN regulates the tumor immune microenvironment and provided a potential target for immunotherapy in TNBC with TTN inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf824ac3335bab02348fceec44a41f866d2add7b" target='_blank'>
              Delta-like ligand 4 mediated myeloid-derived suppressor cell metabolic reprogramming promotes neoadjuvant therapy resistance in titin-inactivated triple-negative breast cancer
              </a>
            </td>
          <td>
            Yanfang Yang, Ziyun Liu, Parhat Kaysar, Yuxi Han, Bo Ni, Linwei Li, Lina Zhang, Xiaobin Shang, Yaoyao Zhou, Yongjie Xie, Zhansheng Jiang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Elucidating mechanisms of drug resistance is key for overcoming resistance, guiding drug design, and enabling accurate resistance prediction. Recently, disease metabolic reprogramming has emerged as a novel mechanism of resistance, which enables disease cells to adapt to therapeutic resistance by altering energy production pathways, cellular signaling, and biosynthesis processes. Moreover, protein structure alterations also play a pivotal role in resistance study, facilitating mechanistic understanding, and structure-based target discovery. In other words, integrating these recently accumulated critical data is essential for enriching the landscape of drug resistance data. Therefore, in this study, DRESIS was a significant update by providing (i) 236 molecules that drive metabolic reprogramming and confer resistance to 168 drugs, together with a detailed mechanism, (ii) 2228 protein structural variants implicated in resistance to 671 drugs across 238 diseases, and (iii) greatly expanded landscapes of drug resistance information, now featuring 398 newly added key drug-resistant molecules, 356 drugs with the latest published resistance mechanisms, and 81 new drug-resistant disease categories. All in all, DRESIS 2.0 is expected to serve as a valuable resource for the scientific community and provide important support in tackling the global challenge of drug resistance, which is now publicly accessible at https://idrblab.org/dresis/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafc45a66260eff1536a05a8101c61e45dec038b" target='_blank'>
              DRESIS 2.0: the comprehensive landscape of drug resistance information.
              </a>
            </td>
          <td>
            Xiuna Sun, Zhangle Wei, Xinyuan Yu, Kaixuan Liu, Shun Yao, Zheng Ni, Hanbing Wang, Ziming Zhao, Yinpeng Zhang, Yuxuan Liu, Hanlu Ding, Yintao Zhang, Zhiguo Liu, Mang Xiao, Feng Zhu
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains clinically challenging due to its molecular heterogeneity and poor outcomes, highlighting the urgent need for novel biomarkers and therapeutic targets. Purpose This study aims to identify and characterize the role of leucine-rich α-2-glycoprotein 1 (LRG1) in AML, evaluating its potential as both a prognostic biomarker and a therapeutic target. Methods We conducted an integrated basic and clinical investigation of LRG1 in AML. Methods included analysis of LRG1 expression in patient samples versus controls and pre- versus post-treatment, assessment of its clinical correlations with mutations and subtypes, and evaluation of its prognostic impact. Functional validation was performed using LRG1 knockdown models to assess effects on colony formation, apoptosis, and differentiation. Single-cell RNA profiling was utilized to identify LRG1-enriched cell populations and explore its role in microenvironmental crosstalk. Results Integrated analysis revealed significantly elevated LRG1 expression in AML patients compared to controls (P<0.001), with levels decreasing post-treatment (P<0.001). High LRG1 expression correlated with FLT3 mutations (P<0.01), M3-M5 AML subtypes (M0&M1&M2 VS M3, P<0.001; M0&M1&M2 VS M4, P<0.01; M0&M1&M2 VS M5, P<0.001; M3 VS M5, P<0.05; M4 VS M5, P<0.05), and worse survival (P<0.01). Functionally, LRG1 knockdown impaired colony formation (P<0.001), increased apoptosis (P<0.001), and disrupted differentiation (P<0.01). Single-cell profiling identified LRG1 enrichment in hematopoietic stem and progenitor cells (HSPCs) and myeloid progenitors, where it facilitated microenvironmental crosstalk via Macrophage Migration Inhibitory Factor (MIF), Galactoside-binding lectin (GALECTIN), and Cyclophilin A (CypA) signals. Conclusion Our findings establish LRG1 as a robust prognostic biomarker and a key functional regulator of AML maintenance through myeloid progenitor dysregulation, presenting it as a promising target for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c85b059f315aeac16b816e67ba97682c83e790" target='_blank'>
              LRG1 Drives AML Progression by Disrupting Myeloid Progenitor Regulation
              </a>
            </td>
          <td>
            Rongxia Guo, Peng Wu, Shuxin Yao, Fumei Liu
          </td>
          <td>2025-12-01</td>
          <td>Blood and Lymphatic Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As a highly fatal and heterogeneous disease, ovarian cancer (OC) lacks sufficient treatment choices. Over the past decade, RNA sequencing (RNA-seq) has profoundly changed research methods in molecular biology, enabling comprehensive analysis of transcriptomes, somatic mutations, and tumour heterogeneity. In OC research, the application of RNA-seq technologies has significantly enhanced scientists’ understanding of the biological mechanisms underlying the disease, providing deeper insights into the intrinsic mechanisms of key processes such as tumour initiation, metastasis progression, drug resistance formation, and immune evasion. This breakthrough not only expands our understanding of OC but also holds promise for the discovery of novel diagnostic biomarkers, the development of innovative therapies, and the establishment of precise prognostic assessment systems. This review summarizes RNA-seq technologies and their applications in OC research, while also discussing the challenges and opportunities for their broader implementation in cancer studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a5fc4d3383fd0f8d4c8ba650bad14685267c27" target='_blank'>
              RNA sequencing in ovarian cancer research: a comprehensive review
              </a>
            </td>
          <td>
            Fang Shen, Chan Li, Yue Li, Ying Shen, Fengjuan Han
          </td>
          <td>2025-11-24</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80392d28b6df8cb89d54d6fc74584c4d84504f02" target='_blank'>
              Multiple Molecular Pathways to Longevity: Opposing Gene Expression Programs Define Distinct Aging Strategies
              </a>
            </td>
          <td>
            Zenith D. Rudich, Aura A. Tamez González, Jiaxi Guan, Grant Booth, Sonja K. Soo, Ulrich Anglas, Meeta Mistry, M. Senchuk, J. V. Van Raamsdonk
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres, stochastically segregate during cell division3–6 and persist over many generations. It has been more than 40 years since ecDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei, but the mechanism underlying this process remains unclear3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. Using Retain-seq, a genome-scale assay that we developed, we reveal thousands of human retention elements that confer generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromosome conformation capture show that retention elements physically interact with mitotic chromosomes at regions that are mitotically bookmarked by transcription factors and chromatin proteins. This activity intermolecularly recapitulates promoter–enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated. Targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, which suggests that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, and reveal the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4655d7bd6dd69864ccca69202d1967eb2da923e5" target='_blank'>
              Transcriptome-wide mapping reveals an RNA-dependent mechanism of platinum cancer drugs
              </a>
            </td>
          <td>
            Arun Krishnaraj, Xinrui Wei, Richi Thakral, Weiyi Guo, Siva Bala Subramaniyan, Xiaopei Zhang, Katelyn R. Alley, Anqi Feng, Andrew Hoy, Douglas Kung, Purushottam B. Tiwari, Aykut Üren, Rodrigo Maillard, Victoria J DeRose, Sreejith J Nair
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cell state plasticity drives metastasis and therapy resistance in cancers. In melanoma, these behaviors map onto a melanocytic-to-mesenchymal-like continuum regulated by AP-1 transcription factors. However, how the AP-1 network encodes a limited set of discrete states, why their distributions vary across tumors, and what drives phenotypically consequential AP-1 state transitions remain unclear. We develop a mechanistic ODE model of the AP-1 network capturing their dimerization-controlled, co-regulated, competitive interactions. Calibrated to heterogeneous single-cell data across genetically diverse melanoma populations and combined with statistical learning, the model reveals network features explaining population-specific AP-1 state distributions. These features correlate with MAPK activity across tumor lines and with variability within clones, linking MAPK signaling to AP-1 states. The model predicts and experiments validate adaptive AP-1 reconfiguration under MAPK inhibition, inducing a dedifferentiated, therapy-resistant state that can be blocked by model-guided AP-1 perturbations. These results establish AP-1 as a configurable network and provide a computational framework for predicting and modulating AP-1 driven cell state plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80053bfbfe501e1acc3e16cb2b130e2f5342e9b1" target='_blank'>
              Cell state plasticity emerging from co-regulated, competitive, and configurable interactions within the AP-1 network
              </a>
            </td>
          <td>
            Yonatan N. Degefu, Magda Bujnowska, Douglas G. Baumann, Mohammad Fallahi-Sichani
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B. Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Background Cervical cancer, as one of the most common types of malignant tumors in women, poses a significant threat to the health and survival of women worldwide. Tumor drug resistance has emerged as a critical bottleneck severely limiting the efficacy of current cervical cancer treatment strategies, making it imperative to delve deeper into the underlying mechanisms for improved therapeutic outcomes. Methods Spatial transcriptomics and single-cell sequencing technologies were employed. Single-cell sequencing was used to uncover the heterogeneity in cervical cancer development. Pseudotime analysis, exploration of cell communication pathways, and spatial transcriptomics were further applied to investigate the roles of relevant molecules. Results This study reveals an abundance of PI3 + neoplastic (NEO) in tumors and SLC40A1 + NEO cells in HSIL samples. These cell populations may play crucial roles in the development of drug resistance, as they could potentially alter the tumor microenvironment and interfere with the efficacy of therapeutic agents. Pseudotime analysis, cell communication pathways, and spatial transcriptomics highlight the pivotal role of LGALS9 and its ligands, offering new insights for early diagnosis, molecular typing, prognosis evaluation, and personalized treatment of cervical cancer. Conclusion Our findings reveal that high expression of LGALS9 suppresses T cell function, fosters a strongly immunosuppressive tumor microenvironment, and is associated with chemotherapy resistance, ineffective immunotherapy, and poor prognosis. LGALS9 may serve as a biomarker for predicting immunotherapy response. This knowledge has the potential to facilitate early diagnosis, precise molecular typing, accurate prognosis evaluation, and the development of personalized treatment strategies that can overcome tumor drug resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiplexed assays of variant effect (MAVEs) systematically measure variant function but have been limited to cancer cell lines rather than disease-relevant cell types. We developed saturation genome editing in human iPSCs (iPSC-SGE) to introduce variant libraries into a single allele of a target gene while programming the genetic background of the second allele, enabling variant assessment across differentiated cell types and genetic contexts at scale. We edited 1,137 variants into MYBPC3 and measured protein abundance in cardiomyocytes and cardiac organoids, accurately identifying pathogenic variants, and resolving variants of uncertain significance. Highlighting the importance of genetic context, we edited 437 POLG variants in two genetic backgrounds and identified loss-of-function and dominant-negative variants. Finally, we illuminate a path for scaling iPSC-SGE by identifying 443 disease genes essential for iPSC or iPSC-derived neuron growth. iPSC-SGE enables systematic assessment of variants in specialized human cell types, advancing MAVEs to empower genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbf73f9abec939337ae228a1f947d752013698c" target='_blank'>
              Editing stem cell genomes at scale to measure variant effects in diverse cell and genetic contexts
              </a>
            </td>
          <td>
            S. Fayer, R. Garge, M. Hopkins, C. E. Friedman, A. V. McGee, J. Rico, R. Powell, E. McDermot, N. T. Smith, S. Pendyala, M. E. Richardson, E. D. Smith, B. M. Bowen, R. Resnick, P. Gupta, A. B. Stergachis, C. Gifford, S. Pinglay, K.-C. Yang, D. M. Fowler, L. Starita
          </td>
          <td>2025-11-14</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK), a member of the receptor tyrosine kinase family, plays a central oncogenic role in the initiation and progression of diverse malignancies. Aberrant ALK activation generally results from structural alterations or dysregulated expression, leading to persistent activation of downstream signaling pathways that drive tumor cell proliferation, survival, and metastasis. ALK gene abnormalities predominantly encompass fusions, point mutations, and amplifications, with EML4-ALK-positive non–small cell lung cancer representing a canonical example. The advent of ALK-targeted inhibitors has constituted a major therapeutic milestone for ALK-positive tumors. From first-generation Crizotinib to third-generation Lorlatinib, successive agents have been refined for target selectivity, central nervous system penetration, and coverage of resistance-associated mutations, substantially improving patient survival and intracranial disease control. Nonetheless, the emergence of acquired resistance remains an overarching challenge, mediated by secondary kinase domain mutations, activation of bypass signaling pathways, and tumor phenotypic transformation. This review presents an integrative synthesis of ALK-targeted therapeutic developments, elucidates underlying resistance mechanisms, and surveys emerging strategies, providing a comprehensive perspective on current advances and future directions in precision management of ALK-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ec5cc87a27a59b857a6b6506e0949481a4fc3f" target='_blank'>
              ALK-Targeted Therapy: Resistance Mechanisms and Emerging Precision Strategies
              </a>
            </td>
          <td>
            Ya‐kun Zhang, Jianbo Tong, M. Luo, Zhi‐Peng Qin, Rong Wang
          </td>
          <td>2025-11-27</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Homozygous deletion of the 9p21.3 genomic locus spanning the CDKN2A/B and MTAP genes is an event affecting 15% of cancers. While CDKN2A is a well-established tumor suppressor gene, the role of MTAP in tumorigenesis varies across cancer types. MTAP codes for methylthioadenosine phosphorylase, a key enzyme in the methionine salvage pathway, and its loss has been associated with several downstream synthetic vulnerabilities. Despite multiple efforts to exploit MTAP loss for targeted therapies, none of these efforts have yielded substantial results in clinical trials. In this review, we consolidate the existing literature along with our systematic analysis to provide an updated perspective on the incidence of MTAP loss in different cancers and elucidate its impact on metabolism, immune microenvironment, and tumor progression. In addition, we summarize the therapeutic strategies that have been investigated preclinically on MTAP-null tumors before and after the advent of functional genomic screening tools. We further assess the current landscape of clinical trials investigating MTAP-targeted inhibitors, evaluating their limitations and potential avenues for improvement. The insights gained from this review will inform future research directions beyond the promising PRMT5/MAT2A axis for rational combination therapies that would work synergistically to eradicate this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846311f0527655d852715f68535274117ddc12d9" target='_blank'>
              MTAP-Null Tumors: A Comprehensive Review on Synthetic Vulnerabilities and Therapeutic Strategies
              </a>
            </td>
          <td>
            Bavani Subramaniam, Wai Chin Chong, Aylar Babaei, M. Bornhorst, Chunchao Zhang, R. Packer, Javad Nazarian
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f4006d95d1215e3a398196888c62daed834131" target='_blank'>
              A Pan-Cancer Blueprint of Early Tumor Microenvironment Reprogramming
              </a>
            </td>
          <td>
            Bhavya Bhavya, Shivangi Agrawal, Ekta Pathak, Rajeev Mishra
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d073de6aab304481d646203a70779deeef6e2a89" target='_blank'>
              CANVAS causing AAGGG repeat expansions cause tissue-specific reduction in RFC1 expression and increase sensitivity to DNA damage
              </a>
            </td>
          <td>
            R. Currò, Natalia Dominik, Stefano Facchini, R. Schnekenberg, Cecilia Perini, Riccardo Ronco, Bianca Rugginini, Arianna Ghia, Silvia Bione, Nadia Tagliaferri, G. P. Grupelli, Simon A. Lowe, Amy R. Hicks, E. Vegezzi, Roberto Simone, Alessandro Bertini, Elena Abati, R. Velasco, Maria Sereno, G. Gutiérrez-Gutiérrez, Simone Thomas, P. Alberti, Vikram Khurana, Johannes Attems, C. Troakes, E. Gustavsson, G. Lignani, Yichen Qiu, J. Sleigh, A. Tucci, P. Fratta, A. Isaacs, Y. M. Lim, Z. Jaunmuktane, S. Brandner, David L. Bennett, Serge Przedborski, P. Opal, A. Hoke, Sheng-Han Kuo, Mary M Reilly, H. Houlden, Mina Ryten, Guido Cavaletti, A. Argyriou, Jordi Bruna, C. Briani, E. Crespan, J. E. Jepson, Andrea Cortese
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Sequestosome 1 (p62/SQSTM1) is a multifunctional adaptor protein whose dysregulation promotes tumorigenesis through autophagy, metabolic reprogramming, and immune modulation. However, its role across cancer types and impact on the tumor immune microenvironment remain poorly defined. Here, we performed a comprehensive pan-cancer analysis to delineate the molecular and immunological landscape of p62 across human malignancies. TCGA analysis revealed rare mutations and limited prognostic impact of genomic alterations but marked transcriptomic upregulation in LIHC, LUAD, BRCA, KIRP, KICH, KIRC, and READ, correlating with poor survival, advanced stage, and higher T classification, particularly in BRCA and LUAD. Pathway analysis showed strong positive associations between p62 and metabolic adaptation and stress tolerance pathways, including oxidative phosphorylation, reactive oxygen species, and DNA repair, in most cancers. Notably, high p62 expression inversely correlated with immune cell infiltration in most epithelial cancers, such as BRCA, COAD, ESCA, HNSC, KIRC, LIHC, LUAD, LUSC, PAAD, PRAD, READ, THCA, while coinciding with elevated expression of immunosuppressive checkpoints such as PD-L1, B7-H3, EBAG9, PVR, and TGFB1, supporting a link between p62-driven metabolic remodeling and an immune-excluded tumor microenvironment. In contrast, GBM, LGG, OV, SARC, and TGCT showed positive correlations between p62 and immunoscore, with enrichment of interferon and pro-inflammatory pathways, reflecting a distinct immune-activated phenotype. Collectively, these findings identify p62 as a central regulator of tumor metabolism and immunity, suggesting that its context-dependent activity may dictate the balance between immune suppression and activation across cancers, and highlight two natural-product-derived PB1 inhibitors (ZINC70669789 and ZINC08877690) as promising candidates for therapeutic development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-04135-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716e845c24cbd3bb41e4b234ab94e5c67dc1b5b3" target='_blank'>
              Comprehensive pan-cancer analysis of p62 reveals its contribution to shaping tumor microenvironment and anti-tumor immunity
              </a>
            </td>
          <td>
            Zahra Nayeri, Elahe Tavakol, M. Rahmati, Marco Cordani, M. Djavaheri-Mergny, Vahid Shariati, M. Moosavi
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background: ATRX is a tumor suppressor and key regulator of transcriptionally repressive epigenetic states. We investigate the impact of ATRX loss on gene expression, DNA methylation, and transposable element (TE) expression in common soft tissue sarcoma tumors, a family of rare cancers. Methods: Using data from The Cancer Genome Atlas (TCGA), we analyzed 234 tumors from patients with dedifferentiated liposarcoma (DDLPS), undifferentiated pleomorphic sarcoma (UPS), soft tissue leiomyosarcoma (LMS), uterine LMS (uLMS), and myxofibrosarcoma (MFS). Corresponding clinical outcome and patient data, DNA sequencing, RNA sequencing, and 450K methylation data were integrated to assess ATRX-dependent features. Results: ATRX loss was associated with significantly altered gene expression with the greatest impact in LMS/uLMS with 31 (33.0%) genes downregulated and 63 (66.0%) genes upregulated (FDR < 0.05), consistent with a role for ATRX in transcriptional silencing. Methylation profiling identified 269 differentially methylated CpGs (FDR<0.05), with a marked hypomethylation effect. ATRX loss was associated with loss of TE silencing with 97% (n=93) of differentially expressed TEs upregulated, indicating that ATRX loss contributes to a marked derepression of TEs. The median overall survival in LMS patients was 81.0 vs 34.9 months for tumors with ATRX retention and loss (plog-rank=0.0023). Conclusions: ATRX loss in sarcomas leads to DNA hypomethylation, increased expression of TEs, as well as transcriptional dysregulation affecting key oncogenic pathways. ATRX status may serve as a potential biomarker for prognosis and therapeutic stratification. Future clinical trials investigating epigenetic therapies could offer novel treatment strategies for ATRX-deficient sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff090d0a0cb56557fe9a6d37cfd5b8f1167336b" target='_blank'>
              ATRX loss in sarcomas is associated with dysregulated gene and transposable element expression, loss of DNA methylation, and worse survival
              </a>
            </td>
          <td>
            Kathryn Demanelis, Julia Leix, Melissa A Burgess, B. Nacev
          </td>
          <td>2025-11-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Targeted therapy has revolutionized the treatment landscape of hepatocellular carcinoma (HCC), offering unprecedented hope to patients. However, despite its promise, significant challenges arise in the form of drug resistance. Only a fraction of HCC patients respond to targeted therapy, and even those who respond often develop resistance over time. Sorafenib and lenvatinib, the sole first-line targeted therapeutic drugs for HCC, face severe clinical limitations due to drug resistance. Understanding the mechanisms underlying sorafenib/lenvatinib resistance is crucial for maximizing treatment efficacy. Recent studies have highlighted the pivotal role of epigenetic regulation in drug resistance. Cancer is recognized as both a genetic and epigenetic disease, with epigenetic factors influencing various aspects of tumor cell biology, especially drug resistance. This review systematically summarizes the mechanisms of epigenetic-mediated sorafenib/lenvatinib resistance, encompassing non-coding RNA (ncRNA) regulation, DNA methylation, RNA methylation, and histone modification. These abnormal epigenetic processes typically influence biological activities, including escaped programmed cell death, tumor metabolic reprogramming, formation and maintenance of drug-resistant cells, uncontrolled cell proliferation signaling pathways, and abnormal transport processes, ultimately culminating in profound drug resistance. By comprehensively summarizing the latest discoveries in epigenetic regulation mechanisms, this review highlights potential strategies to overcome drug resistance, paving the way for future advancements in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8f7f2274db91d56455b33e92f98f5f8697e415" target='_blank'>
              Resistance of first-line targeted drugs in hepatocellular carcinoma: the epigenetic regulation mechanisms
              </a>
            </td>
          <td>
            Ding Qi, Yongqing Qin, Haidong Zhu, Yong Li, Shisong Han
          </td>
          <td>2025-11-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, limiting the efficiency and precision of genome editing in many clinically relevant tissues. Here, we address this barrier by using induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. CRISPR editing outcomes differ dramatically in neurons compared to genetically identical dividing cells: neurons take longer to fully resolve this damage, and upregulate non-canonical DNA repair factors in the process. Manipulating this response with chemical or genetic perturbations allows us to direct DNA repair toward desired editing outcomes in nondividing human neurons, cardiomyocytes, and primary T cells. By studying DNA repair in clinically relevant cells, we reveal unforeseen challenges and opportunities for precise therapeutic editing. Genome editing tools can precisely introduce a specified lesion into the DNA, but ultimately rely on the cell’s DNA repair machinery to determine the editing outcome. Here, authors demonstrate how neurons’ unique DNA repair pathways impact the safety, efficiency, and precision of CRISPR edits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3be0abd5edf19f4fcddf4c37efa7ec4d61c80" target='_blank'>
              Characterizing and controlling CRISPR repair outcomes in nondividing human cells
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J. Namaganda, Manasi M. Kumar, J. Hamilton, Rohit Sharma, Karena G. Chow, Luke A. Workley, Bria L. Macklin, Mengyuan Sun, Alvin Ha, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Philip H. Dierks, Rudra Bose, Julianne Jin, Barbara S. Perez, Cindy R. Sandoval Espinoza, Madeline P. Matia, Serena H. Lu, Luke M. Judge, Brian R. Shy, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2025-11-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC) is the second most common form of non-melanoma skin cancer (NMSC), with a steadily increasing global incidence, especially in populations with prolonged ultraviolet (UV) exposure. Advanced or metastatic cSCC carries a poor prognosis, while the application of targeted therapies and immunotherapies remains in the exploratory stage, due to limited understanding of the underlying mechanisms of cSCC occurrence and progression. Epigenetic dysregulation plays important roles in the progression of cSCC. However, how multi-dimensional regulatory networks reshape the epigenetic landscape and contribute to dysregulated gene expression in cSCC remains unclear. In this study, we performed parallel RNA m6A sequencing, 850K DNA methylation arrays, whole transcriptome sequencing, and ATAC-seq chromatin accessibility profiling on samples from normal skin, actinic keratosis (AK), and cSCC. We analyzed the regulatory networks and pathways of epigenetic modifications on gene expression. We further explored the crosstalk of epigenetic regulatory networks by correlation analysis. By integrating single-cell RNA-seq data, we identified epigenetically upregulated candidate genes and confirmed their expression and functions with experimental methods. Our integrated multi-omics analysis provides a comprehensive and dynamic epigenetic map of cSCC progression. Further analysis revealed that DNA methylation and m6A modification jointly regulate gene expression through independent and synergistic ways. The identified epigenetically upregulated candidate genes IDO1, IFI6, and OAS2 were validated to be overexpressed in cSCC tissues and cell lines, and functional assays confirmed their potential key roles in regulating the processes of cell proliferation, migration and invasion in cSCC. By integrating multi-omics data, this study systematically highlights the multi-layered epigenetic alterations and regulatory mechanisms involved in cSCC development. This multi-stage, multi-omics, and multi-resolution integrated analysis provides a theoretical basis and new insights for future personalized treatment strategies for cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdeb7e8c78575b1d7e37855326ebd04eb68da2c1" target='_blank'>
              Multi-omics analysis reveals the crosstalk of epigenetic regulatory networks in cutaneous squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Ya-Zhou Sun, Dan-Dan Zou, Xin-Jie Li, Xiao-Li Wang, Jia-Hao Feng, Jia-Ying Liu, Ke He, Yu-Tong He, Xian-Xin Lai, Xin Li, Xuan Jiang, Tian-Shun Gao, Li He
          </td>
          <td>2025-11-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600bc28120bc59b1b37b42c37c760d59334a28c6" target='_blank'>
              CRISPR/Cas9-mediated generation of a de novo C9ORF72 knock-in isogenic iPSC cell bank to model ALS and FTD
              </a>
            </td>
          <td>
            Niamh L. O’Brien, Erin C. Hedges, R. Nair, AJ Cammack, M. Carcolé, Deniz Vaizoglu, Sara Tacconelli, Katie Sutherland, Chloe L. Fisher-Ward, A. R. Awan, Simon Topp, A. Iacoangeli, A. Isaacs, Thomas J. Cunningham, M. Ruepp, Sarah Mizielinska
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Structural variations in cancer predisposition genes significantly contribute to the pool of pathogenic variants underlying heritable cancer susceptibility. In this report, we describe and functionally characterize a novel transposon-mediated germline pathogenic insertion identified within the coding region of the BRCA1 gene. The inserted sequence comprises the entire processed transcript of RPL18A, a ribosomal protein-coding gene, and may have been generated through a target-primed reverse transcription event. Robust molecular testing confirmed the heritability of the variant and provided evidence of its correlation with the observed phenotype. This is the first documented case of a germline cancer susceptibility variant arising through this unique mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab6faf5664bd66e2e317ebc80d8ac230de46d4a" target='_blank'>
              Processed Transcript Insertion as a Novel Germline Mutational Mechanism in BRCA1-Associated Hereditary Breast Cancer
              </a>
            </td>
          <td>
            A. Bozsik, H. Butz, V. Grolmusz, Petra Nagy, T. Pócza, Erika Tóth, E. Csernák, Attila Patócs, János Papp
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01974-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. Methods We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Results Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. Conclusion This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene amplification is thought to be common in bacterial populations, providing a rapid and reversible mode of adaptation to diverse stresses, including the acquisition of antibiotic resistance. We previously showed that the opportunistic pathogen Staphylococcus aureus evolves resistance to the dual-targeting fluoroquinolone delafloxacin (DLX) that inhibits both the DNA gyrase and DNA topoisomerase IV via amplification of an efflux pump encoding gene sdrM. However, the pathways that control gene amplification, and consequently adaptive trajectories, remain understudied, especially in gram-positive bacteria like S. aureus. Here, we show that specific DNA repair and chromosomal separation proteins alter the frequency of gene amplification and selection of amplified regions in S. aureus. Through a screen of 40 mutants deficient in various DNA processes, we determined that while sdrM amplification was still the almost universal path to DLX resistance, other mutations that increased sdrM expression reduced the selection frequency of sdrM amplification, demonstrating the critical role of sdrM in DLX resistance. We found that similar to other bacteria, both sdrM amplification and loss of amplified gene copies required a functional RecA recombinase, but multiple other mutants in pathways required for amplification in other species still exhibited frequent sdrM amplification, suggesting that S. aureus may have alternate routes of gene amplification. Finally, loss of function mutants of the tyrosine recombinase XerC, that is known to play a role in chromosomal separation, were deficient for sdrM amplification, indicating that XerC is a novel modulator of gene amplification, or the maintenance or selection of amplified gene copies. Thus, our work sheds light on genetic factors that alter gene amplification-mediated evolutionary trajectories to antibiotic resistance in S. aureus and can potentially unlock mechanisms by which such evolution of resistance can be inhibited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875b466f28280308897b4f66650c929c02cc31e9" target='_blank'>
              Modulators of gene amplification alter evolution of antibiotic resistance in Staphylococcus aureus.
              </a>
            </td>
          <td>
            Kalinga Pavan T Silva, Anthony M Martini, Anupama Khare
          </td>
          <td>2025-12-31</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The DNA Damage Repair (DDR) signaling pathway serves as a crucial molecular hub that regulates chemotherapy efficacy, offering significant translational value in the field of precision oncology. This review systematically analyzes the molecular mechanisms of five core DDR pathways (Homologous Recombination Repair, Non‐Homologous End Joining, Base Excision Repair, Nucleotide Excision Repair, and Mismatch Repair) in mediating chemotherapy resistance in tumors, and thoroughly elucidates the correlation between key molecular events—such as BRCA1/2 deficiency, MMR functional abnormalities, and Ataxia Telangiectasia Mutated/Ataxia Telangiectasia and Rad3‐related (ATM/ATR) signaling pathway dysregulation—and chemotherapy sensitivity. The DDR deficiency biomarker system established through the integration of multi‐omics data provides molecular classification tools for predicting the efficacy of platinum‐based drugs. This study focuses on the mechanism by which Poly ADP‐Ribose Polymerase inhibitors reverse Homologous Recombination‐Deficient tumor resistance through “synthetic lethality” effects while also revealing the synergistic anti‐tumor effects of ATM/ATR inhibitors in combination with chemotherapeutic agents. The research presents an innovative molecular synergy model between DDR regulation and Immune Checkpoint Blockade, confirming that tumor neoantigen release induced by DDR deficiency can enhance immunotherapy responses. This article also provides perspectives on multidimensional intervention strategies based on the DDR network, including the development of inhibitors targeting novel DDR targets, the establishment of DDR pathway functional assessment systems based on multidimensional biomarkers, and the investigation of synergistic paradigms between DDR and novel therapeutic modalities. Additionally, we explore the dynamic evolution mechanisms of DDR‐mediated chemotherapy resistance by analyzing the interactions between DDR and metabolic reprogramming, as well as other related processes. These breakthrough advances provide theoretical foundations and innovative directions for overcoming chemotherapy resistance and advancing personalized treatment, marking a new era in cancer therapy characterized by precision targeting of DDR pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/944e141ebbf24071dde5281d6930372251bae23d" target='_blank'>
              Translational Aspects of DNA Damage Repair in Optimizing Cancer Chemotherapy
              </a>
            </td>
          <td>
            Anqi Lin, Jinyue He, Aimin Jiang, Jian Zhang, Quan Cheng, Hengguo Zhang, Wenjun Mao, Peng Luo
          </td>
          <td>2025-12-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease despite therapeutic advancements extending survival. Relapses driven by drug resistance and minimal residual disease underscore the need for novel treatment strategies. Natural Killer (NK) cells play a key role in MM immunity, yet their function is suppressed by inhibitory cytokines and metabolites from the tumor microenvironment. Developing anticancer drugs with immunomodulatory properties, such as enhancing tumor sensitivity to NK cell recognition, remains a critical challenge. MM cells exhibit high protein synthesis rates, making them vulnerable to proteostasis disruption. Dysregulated ribosome function and aberrant mRNA translation contribute to proteasome inhibitor resistance. RNA Polymerase I (RNA Pol I)-mediated rDNA transcription, the rate-limiting step in ribosome biogenesis (RiBi), is significantly upregulated in MM. Targeting rDNA transcription and inducing nucleolar stress response (NSR) presents a promising therapeutic approach, though its immunomodulatory role is not well understood. Our study examined two “first-in-class” RNA Pol I inhibitors, CX-5461 and BMH-21, which differentially regulate NK cell-activating and inhibitory ligand expression in MM. BMH-21 enhanced NK cell degranulation and increased IFN-γ and TNF-α secretion, demonstrating stronger immunostimulatory effects than CX-5461. Conversely, CX-5461 induced a significant DNA damage response (DDR) and senescence, leading to HLA-E upregulation and suppressing NK cell activity. Mechanistic analyses revealed that HLA-E presentation is governed by ATR/AKT/mTORC1/S6K signaling and Pioneer Round of Translation (PRT), linking its regulation to DDR. This effect was modulated by Lenalidomide and Panobinostat. Moreover, RNA Pol I inhibition enhanced Daratumumab-mediated antibody-dependent cellular cytotoxicity (ADCC) of NK cells against MM, uncovering novel immuno-mediated antitumor mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eac9f710ed72751582b0e2b38db7a164c640592" target='_blank'>
              Targeting RNA polymerase I to boost natural killer cell anticancer activity in multiple myeloma
              </a>
            </td>
          <td>
            Elena Sproviero, Eleonora Gnocchini, Tommaso Cipollone, Sara Petillo, Chiara Cassone, R. Molfetta, A. Zingoni, A. Soriani, C. Cerboni, M. Petrucci, F. Fazio, R. Paolini, Gabriella Palmieri, M. Cippitelli
          </td>
          <td>2025-11-28</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC), an epigenetic modification derived from the oxidation of 5-methylcytosine (5mC) by the ten-eleven translocation (TET) family of dioxygenases, plays a pivotal role in the regulation of gene expression, cellular differentiation, and developmental plasticity. Once considered an intermediate in DNA demethylation, 5hmC is now recognized as a stable and functionally significant epigenetic mark with distinct genomic distributions and significant regulatory implications. This review provides a comprehensive analysis of the biological functions of 5hmC in normal cellular processes, including its role in maintaining tissue-specific gene expression, lineage commitment, and genomic integrity. We also describe its role in cancer, the mechanistic underpinnings of its loss or redistribution in tumor cells, and how these changes contribute to oncogenic signaling pathways, epithelial-mesenchymal transition, and tumor heterogeneity. Furthermore, we explore the utility of 5hmC as a biomarker in cancer diagnostics and prognostics, supported by recent advances in sequencing technologies and cell-free DNA profiling. We also examine the intersection of 5hmC and chemotherapy, highlighting how aberrant 5hmC levels can influence drug resistance and sensitivity, and assess the therapeutic potential of targeting TET enzymes and associated pathways. By integrating insights from basic epigenetics, cancer biology, and therapeutic research, this review underscores the multifaceted role of 5hmC in human malignancies and outlines the translational opportunities for exploiting 5hmC-related mechanisms in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3164793ee65481d7edac6287a5b81d63211bc" target='_blank'>
              5-Hydroxymethylcytosine: a key epigenetic mark in cancer and chemotherapy response
              </a>
            </td>
          <td>
            S. S. Kharat, Shyam K. Sharan
          </td>
          <td>2025-11-17</td>
          <td>Epigenetics & Chromatin</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are driven by complex molecular interactions between genomic instability and immune modulation. Defects in the DNA damage response (DDR) not only promote tumor heterogeneity but also shape the tumor immune landscape through the generation of neoantigens, activation of the cGAS–STING pathway, and modulation of immune checkpoints. This review provides an integrative overview of the molecular mechanisms linking DDR dysfunction to immune crosstalk, emphasizing how these processes influence tumor evolution and response to therapy. We discuss emerging therapeutic strategies that exploit DDR–immune interactions, including PARP and ATR inhibitors, synthetic lethality approaches, and combination regimens with immune checkpoint blockade. Understanding the bidirectional connection between DNA repair pathways and immune signaling unveils new translational opportunities for precision oncology and offers a framework for developing combinatorial therapies capable of overcoming resistance and improving long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855e668433efed3e8846b48896d51f3b8c724981" target='_blank'>
              Targeting DNA Damage Response and Immune Crosstalk in Cancer: Mechanistic Insights and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Lavinia Marcut, Roxana Brata, A. Barb, Alexia Manole, Dan Gabriel Stef, C. Dumitru, Flavia Zară, Raul Pătrașcu
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel C. de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cancer therapy remains an active field of investigation, particularly in understanding and overcoming therapy resistance. Small non-coding RNAs, such as microRNAs (miRNAs), are emerging as key regulators of cancer survival, progression, proliferation, invasion, migration, and metastasis. Although many studies have linked miRNAs to cancer therapy outcomes, significant questions remain regarding their precise molecular and cellular roles in therapy resistance. Increasing evidence shows that miRNAs influence critical pathways such as apoptosis, immune evasion, and other signaling cascades. However, there have been many setbacks because of the limitations in knowledge of each specific miRNA’s function. A deeper understanding of miRNA expression and function may enhance the development of more effective cancer therapeutics and improve overall survival of patients. This review explores the role of miRNA expression as a key regulator of therapeutic resistance in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2eecd6c06795aaec7cf5be8c8739a720372740" target='_blank'>
              MicroRNAs as key regulators of cancer drug resistance: insights and future directions in chemotherapy, targeted-therapy, radiotherapy, and immunotherapy
              </a>
            </td>
          <td>
            Michael A Attathikhun, A. Jurj, George A. Calin
          </td>
          <td>2025-12-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee41f49d06aaeb019cdfc5399ce87f114ac10230" target='_blank'>
              Clinical Implications of SRPK1 Expression in Human Tumours: A Comprehensive Pan-Cancer Analysis Based on Multi-Omics Databases
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most lethal of all brain cancers, resists therapy by rewiring metabolism and relying on GTP signaling to promote DNA repair and radiation therapy (RT) resistance. How GBM modulates GTP levels for this signaling in response to RT-induced DNA damage, and the therapeutic tractability of this metabolic activity in the context of standard-of-care chemoradiation therapy, remain unaddressed. Here, we identify acute changes in glioma metabolism within hours of RT, including an acute post-RT rewiring of guanylate synthesis driven by nuclear translocation of the rate-limiting de novo guanylate synthesis enzyme IMPDH1. This subcellular IMPDH1 re-localization and nuclear GTP accumulation are dependent on the DNA damage signal kinase DNA-PK. Targeting intracranial GTP synthesis with the FDA-approved inhibitor mycophenolate mofetil (MMF) slows repair of DNA damage and extends survival of orthotopic murine models treated with combined RT and temozolomide. Extending our findings to humans, we performed a phase 0 clinical trial revealing that oral MMF administration leads to active intracranial drug concentrations, with target engagement indicated by reversal of IMPDH upstream and downstream metabolites in recurrent GBM tumors. Together, these findings implicate IMPDH as a potential metabolic target in GBM whose pharmacological inhibition is feasible and could complement standard-of-care chemoradiation therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a030935c6094e7ea3aa4ea85bf4e1b478d3311d7" target='_blank'>
              Adaptive nuclear GTP synthesis promotes glioblastoma treatment resistance and is a targetable vulnerability in patients
              </a>
            </td>
          <td>
            A. Scott, Ningning Liang, W. Al-Holou, Jie Xu, Alexandra M O'Brien, Angelica Lin, V. Pareek, Zhou Sha, Zitong Zhao, John Z Yang, A. Yang, Ayesha U. Kothari, Kari Wilder-Romans, Zhe Wu, Jiane Feng, Navyateja Korimerla, A. Elaimy, Sravya Palavalasa, Jiali Chen, Lu Wang, Li Zhang, A. Andren, Peter Sajjakulnukit, Bernard Marini, J. Heth, Meredith A Morgan, Theodore S. Lawrence, S. Benkovic, Deepak Nagrath, Sriram Chandrasekaran, Nathan R. Qi, D. Sun, B. Wen, Manjunath P. Pai, N. Clarke, K. Suresh, Y. Umemura, C. Lyssiotis, Weihua Zhou, Daniel R. Wahl
          </td>
          <td>2025-11-13</td>
          <td>medRxiv</td>
          <td>1</td>
          <td>71</td>
        </tr>

        <tr id="


 Dose–response curves (DRCs) are a standard protocol in experimental biology to assess the effect of anticancer agents in vitro. Yet, therapy itself can reshape tumor composition, driving the gradual evolution of the DRC. Monitoring such changes can reveal the emergence of resistance, as well as evolutionary trade-offs that may be exploited to enhance therapeutic efficacy. Here, we investigate the evolution of DRCs in response to three different ALK inhibitors (ALKi) in ALK-positive anaplastic large cell lymphoma (ALCL) using patient-derived cell lines. Over the course of six months, ALKi concentrations were gradually escalated, allowing ALCL cells to adapt and survive at higher drug levels. Strikingly, at the same time, these resistant populations exhibited reduced fitness in the absence of the drug, indicating an evolutionary cost. This phenomenon, known as “drug addiction” or dependence, has been previously observed in melanoma following resistance to MAPK inhibitors and suggests that treatment holidays could optimize tumor control in the long term. Guided by these findings, we developed a mathematical model to capture the temporal evolution of DRCs and to test how therapy design might exploit addiction in ALCL. Calibrated with experimental data, the model simulated tumor growth under alternative dosing regimens. Simulations revealed that continuous therapy rapidly selects for resistance, leading to tumor regrowth within ∼150 days. In contrast, intermittent dosing schedules maintained long-term control for more than 300 days. Moreover, the model identified an adaptive strategy: switching from continuous to intermittent dosing precisely when tumor burden reaches its minimum. This evolutionarily informed approach prevents resistant clones from dominating while leveraging the fitness disadvantage associated with drug addiction. Preliminary experiments with naïve and resistant ALCL cell lines, cultured under intermittent dosing in both monoculture and co-culture systems, support the model’s predictions. These experiments also enable the refinement of the model's parameters to incorporate tumor heterogeneity. While additional validation is required, our findings demonstrate that integrating mathematical modeling with experimental DRCs provides a quantitative framework for predicting the evolutionary dynamics of resistance and addiction. Collectively, this study highlights the potential of adaptive and evolution-based therapy in ALCL. Optimized treatment schedules informed by evolutionary trade-offs can reduce tumor burden, delay progression, and exploit vulnerabilities generated by resistance mechanisms.



 Franco Pradelli, Lucy Hare, Jamie Matthews, Roxane Chen, Aiindrila Dhara, G. A. Amos Burke, Suzanne D. Turner, Jeffrey West. Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b1f1eb717bb47b50718cff89eef53948102c75" target='_blank'>
              Abstract B037: Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies
              </a>
            </td>
          <td>
            Franco Pradelli, Lucy Hare, J. Matthews, Roxane Chen, Aiindrila Dhara, G. A. A. Burke, Suzanne D. Turner, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Extracellular leucine rich repeat and fibronectin type III domain containing 1 (ELFN1), a transmembrane protein implicated in tumorigenesis and therapy resistance, remains mechanistically undefined as a pan-cancer target. In this study, we aimed to elucidate the function and potential mechanism of action of ELFN1 across cancers. Methods Through integrative analysis of TCGA and GTEx datasets, we systematically characterized ELFN1 across 33 cancer types, including its expression patterns, prognostic value, mutation landscape, methylation modifications, protein-protein interaction (PPI) networks, and the relationship between ELFN1 expression and immune infiltration. KEGG enrichment analysis was also performed to predict the functions and associated cellular pathways of ELFN1. In addition, the molecular docking tool was used to analyze the affinities between ELFN1 protein and drugs. Finally, we assessed the effect of ELFN1 knockdown on colorectal cancer (CRC) cells using in vitro experiments. Results Our study revealed significant dysregulation of ELFN1 across various cancer types, with notable diagnostic and prognostic utility in most cancers analyzed. Mechanistically, ELFN1 expression was associated with DNA methylation, DNA repair, genomic instability, and tumor microenvironment (TME) scores in multiple cancer types. Furthermore, Drug sensitivity profiling linked ELFN1 to ABT-737 susceptibility and benzaldehyde resistance through molecular docking. In CRC cells, ELFN1 knockdown significantly inhibited tumor proliferation, migration, and motility. Conclusion The expression level of ELFN1 may provide insights into tumor development and progression in multiple cancers, including CRC, highlighting its potential utility as an effective prognostic biomarkers and immunotherapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c3fe475dcaffa41259e73c95a1f5f7e78a3e04" target='_blank'>
              Pan-cancer analysis reveals ELFN1 as a novel prognostic biomarker and immunotherapeutic target associated with tumor microenvironment remodeling and promoting malignant phenotypes in colorectal cancer
              </a>
            </td>
          <td>
            Sha-Sha Hu, Tianyuan Tan, Wei Yan, Yu Wu, Xinnian Li, Fu-Jin Liu, Bo Wang
          </td>
          <td>2025-11-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="TP53 mutations are among the worst prognostic factors in acute myeloid leukemia (AML), with affected patients facing relapse-free survival of just five-to-six months compared to TP53 wild-type patients. A major barrier to improving outcomes lies in the dearth of effective therapies, as TP53 mutant patients remain refractory to conventional cytotoxic chemotherapies, targeted therapies, and even allogeneic stem cell transplantation. In this review, we first summarize current clinical strategies and the major setbacks of p53 activators, MDM2/X regulators, and immunotherapy, highlighting the disconnect between promising pre-clinical studies and limited durable clinical responses. We next discuss the mechanisms of therapy resistance in TP53 mutant AML, with specific emphasis on dysfunction in the mitochondrial apoptotic pathway and clonal evolution of TP53 mutant hematopoietic stem cells. We then outline a roadmap for developing tailored therapies that may finally redefine prognosis for this high-risk patient population, including apoptotic activators, cell-cycle modulators, and immune- and metabolic-based therapies. We lastly call attention to new biomarker-driven approaches that can improve patient stratification and optimize identification of responders. By connecting mechanistic understanding with translational insights, this review underscores both the formidable challenges and the emerging opportunities in TP53 mutant AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b2b9dab76dc863b017965cd1c9f9debef78076" target='_blank'>
              Pandora’s Box of AML: How TP53 Mutations Defy Therapy and Hint at New Hope
              </a>
            </td>
          <td>
            E. Olesinski, Shruti Bhatt
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c702cdf51be4f8a5bb8e9bacfe163f51cdb987fa" target='_blank'>
              Mitochondrial genetics defines anti-tumour immunity through mitochondrial ROS and PD-1 signalling
              </a>
            </td>
          <td>
            Gennaro Prota, Raquel Justo-Méndez, U. Gileadi, Macarena de Andrés-Laguillo, J. Cabrera-Alarcon, María del Mar Muñoz-Hernández, Sofía C Khouili, Raquel Martínez-de-Mena, Juan Pellico, Rui Benedito, J. Ruíz-Cabello, D. Sancho, José Antonio Enríquez, A. V. Lechuga-Vieco
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362966d97ea996aa6c3590d2dcc1c33fc19cf534" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular and molecular changes in EGFR-positive lung adenocarcinoma before and after Furmonertinib treatment.
              </a>
            </td>
          <td>
            Jian Chen, Minghui Cai, Lifei Meng, Ze Hong, Wentao Hu
          </td>
          <td>2025-11-27</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, Allen L Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15241-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Compared to classical drug screening, single-cell screening not only significantly enhances throughput but also provides richer transcriptional response information. In this study, we employed the high-throughput and high-sensitive single-nucleus sequencing platform, snHH-seq, to screen clinical drug combinations with anti-hepatocellular carcinoma (HCC) activity. Single-cell transcriptomics analysis revealed that the HY combination (HHT and YM155) exhibited the strongest suppression of tumour cell proliferation, a finding validated by both in vitro and in vivo functional assays. Further investigation suggested that HY triggers ferroptosis, as evidenced by rescue from cell death upon co-treatment with the ferroptosis inhibitor Fer-1. Subcluster analysis identified distinct tumour cell subclusters' responses to HY treatment. A gene regulatory network analysis highlighted JUN as a key regulator mediating proliferation inhibition, primarily active in the apoptotic cell subcluster. These findings illustrate how integrating high-throughput screening with mechanistic dissection can accelerate the discovery of targeted drug combination therapies, and offer a blueprint for precise interventions using pathway vulnerabilities and cellular heterogeneity in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca11c3593d3f82ffcaff76f20d6518bb533fce26" target='_blank'>
              Multiplexed Transcriptomics for Screening Drug Combinations and Defining the Mechanism of Action of HCC Therapeutics at Single-Cell Resolution.
              </a>
            </td>
          <td>
            Mengmeng Jiang, Haide Chen, Guoxia Wen, Yuqing Mei, Wenzhao Zhou, Bin Xu, Tingyue Zhang, Guangyan Li, Junqing Wu, Xiaoping Han, Xudong Fu, Guoji Guo, Jingjing Wang
          </td>
          <td>2025-11-27</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, Laura Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-24</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract CRISPR activation is a powerful tool to upregulate a vast array of genes in many different contexts. However, there are few dynamic CRISPR transcriptional programs, which limit its usage in the creation of living biosensors, self-regulating microbial factories, or conditional therapeutics. Here, we address this limitation by embedding a molecular switch directly into a guide RNA to create a combined sensor–actuator called a metabolite-responsive scaffold RNA (MR-scRNA). We demonstrate the regulatory potential for MR-scRNAs by conditionally activating genes in three different kingdoms of life. We create MR-scRNAs responsive to two distinct metabolites, theophylline and tryptophan, by swapping the molecular switch used. MR-scRNAs respond quickly in a dose-dependent manner specifically to their target metabolite and enhance biochemical production when used as a dynamic regulator of pathway enzyme expression. The broad functionality and ease of design of the MR-scRNAs offer a promising tool for dynamic cellular regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d51648e536b87b3b15eaf74f33b906ab381943e" target='_blank'>
              Metabolite-responsive scaffold RNAs for dynamic CRISPR transcriptional regulation
              </a>
            </td>
          <td>
            Anthony Stohr, Helena Hansen, B. Richards, Hayeon Park, Antonio G Goncalves, Ayushi Agrawal, Mark A. Blenner, Wilfred Chen
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Early recurrence remains a major challenge in the management of hepatocellular carcinoma (HCC), yet its molecular mechanisms are not fully understood. In this study, we applied an integrative multi‐omics strategy at single‐cell resolution to explore potential drivers of early HCC recurrence (recurrence time < 2 years) and to develop a predictive framework. By combining single‐cell RNA sequencing, proteomics, transcriptomics, and clinical feature analysis, we identified 14 relapse‐associated proteins, including CD274, B2M, MYC, and CASP3, as candidate risk factors. Transcriptomic profiling suggested the enrichment of pathways such as MYC‐TARGETS‐V2 and INTERFERON‐GAMMA‐RESPONSE. Single‐cell analysis indicated reduced immune cell infiltration in recurrent tumors, with myeloid cells (particularly cDC2 and macrophages) showing B2M‐associated reprogramming characterized by HLA downregulation and altered GAS6/PROS1 signaling, consistent with tumor‐associated macrophage‐like phenotypes. A LASSO regression model based on cDC2 and macrophage signature genes demonstrated moderate predictive performance in both the training and validation cohorts (AUC > 0.65). Drug sensitivity analyses further suggested that vandetanib may have the potential to inhibit recurrence by targeting B2M‐related pathways. These findings provide evidence that B2M may contribute to remodeling of the immune microenvironment in recurrent HCC. Our integrative single‐cell multi‐omics approach highlights a possible mechanism of early recurrence and offers a preliminary predictive tool with therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea78ab762a4ffddad742cec57b0d57d2bd1edeac" target='_blank'>
              Single‐Cell Multi‐Omics Reveals B2M‐Mediated Myeloid Reprogramming and Constructs a Predictive Model for Early Hepatocellular Carcinoma Recurrence
              </a>
            </td>
          <td>
            Zhenyao Tan, Yezhen Tang, Bendong Chen, Yongxue Lv, Bozhi Zhang, Kejun Liu, Yang Bu
          </td>
          <td>2025-12-22</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="RNA repair is critical for cellular function. The Rtc system maintains RNA integrity within the translational machinery of bacteria. In E. coli, Rtc expression enables cells to rescue growth and survive treatment by conferring transient resistance to ribosome-targeting antibiotics, yet the mechanisms underpinning this resistance remain obscure. Here, we present a computational model of Rtc-regulated repair of translational RNAs. Integrating model predictions with experimental validations, we uncover notable cell-to-cell heterogeneity in rtc expression that impacts on translational capacity, indicating that rtc may induce a form of heteroresistance. We moreover identify Rtc targets that may reduce the translational capacity of cells and so potentiate antibiotic effects. Our findings elucidate a complex response underpinning resistance conferred by Rtc, offering alternate routes for addressing resistance in E. coli and other relevant pathogens. RNA repair helps bacteria survive antibiotic stress. Here, authors show that Rtc-driven repair causes cell-to-cell variation in resistance levels, revealing a potential form of heteroresistance, and identify key Rtc targets to enhance antibiotic effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd19ca4b27ee23689628c6e23e1bb197f23a5f1c" target='_blank'>
              Heterogeneity in responses to ribosome-targeting antibiotics mediated by bacterial RNA repair
              </a>
            </td>
          <td>
            Hollie J Hindley, Zechuan Gong, S. Moradian, M. G. Giuliano, Andrei V. Sapelkin, I. Kotta-Loizou, Martin Buck, C. Engl, Andrea Y Weiße
          </td>
          <td>2025-11-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, , M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gliomas are primary malignant brain tumors with poor prognosis. Conventional therapies have achieved limited improvements, and there is an urgent need for new treatments. Homologous recombination repair (HRR) defects are observed in many types of cancer and confer increased sensitivity to platinum-based chemotherapies and poly-ADP ribose polymerase (PARP) inhibitors. To our knowledge, HRR mutations in gliomas have not been extensively reported. To this end, we analyzed alterations in 39 HRR related genes and divided patients with glioma into different groups according to HRR LOF mutation status. We then analyzed the differences in genetic mutation landscape, TMB, CNV burden, prognosis, and immune-related features between the HRR-LOF group and the NonHRR-LOF group. The results demonstrated that compared to the non-HRR-LOF group, the HRR-LOF group exhibited a higher mutation frequency of IDH1, increased TMB, and poor prognosis in IDH-mutant patients. In summary, gliomas patients with HRR LOF have relatively more unstable genomes and might benefit from the combination and sequential use of PARP inhibitors, radiotherapy, and immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15314-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5956393145980da1f1d1ed7db0b4722bae053cbb" target='_blank'>
              Mutation patterns and prognostic potentiality of homologous recombination repair genes in gliomas
              </a>
            </td>
          <td>
            Hongmin Bai, Yaqin Liu, Cheng Ji, Qin Zhang, Ningning Luo, Xing Zhang, Zhiming Zheng
          </td>
          <td>2025-12-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents, including doxorubicin, are common. Therefore, this project aims to characterize the evolution of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from the ATRT cell line, BT16, and RMC cell line, PEDS005T, utilizing a dose escalation approach. This resulted in DR cell lines that are 10-to-30-fold resistant compared to the parental cell line. Furthermore, we generated two independent DR cell lines from each parental cell line, enabling the capture of distinct evolutionary trajectories leading to resistance. We profiled the transcriptomes by bulk RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1, in 3 of 4 models. To assess whether ABCB1 upregulation alone was sufficient for DR, we treated them with an ABCB1 pharmacological inhibitor and found that ABCB1-upregulated DR cells were re-sensitized to doxorubicin, whereas an ABCB1-negative BT16 DR cell line remained resistant. To determine a shared mechanism of resistance between these models, we profiled the methylomes by Oxford Nanopore Technologies long-read sequencing. We observed global hypermethylation when compared to the parental counterparts. Treatment with a hypomethylating agent reduced cell viability up to 75%. Future studies will focus on identifying and therapeutically targeting these commonalities to overcome resistance regardless of evolutionary trajectory. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there are different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, reduced cell viability resulting from treatment with a hypomethylating agent suggests there are common vulnerabilities despite differing evolutionary trajectories.



 Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong. The evolution of doxorubicin resistance in SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebc129836d1140b7ad59722551b94ec70c22ad5" target='_blank'>
              Abstract B017: The evolution of doxorubicin resistance in
 SMARCB1
 -deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The MTAP (methylthioadenosine phosphorylase) gene, located on chromosome 9p21, plays a crucial role in the methionine salvage pathway and is frequently co-deleted with CDKN2A in various malignancies. Loss of MTAP expression leads to the accumulation of methylthioadenosine (MTA), which selectively inhibits protein arginine methyltransferase 5 (PRMT5) and creates a unique metabolic vulnerability in MTAP-deficient tumors. These alterations have emerged as promising therapeutic targets in precision oncology. Recent advances highlight the potential of exploiting MTAP loss through synthetic lethality approaches using PRMT5 and methionine adenosyltransferase 2A (MAT2A) inhibitors. Preclinical and early clinical data indicate that targeting these pathways can selectively impair tumor growth while sparing MTAP-proficient cells. Moreover, MTAP deletion has been associated with specific molecular and immunologic profiles that may influence treatment response and tumor microenvironment characteristics. This review summarizes current knowledge on the biological functions of MTAP, the mechanisms linking its loss to oncogenesis, and the evolving landscape of therapeutic strategies targeting MTAP-deficient cancers. Understanding these molecular dependencies offers novel opportunities for the development of precision-based therapies across diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40708c2db41533a7db6997687c20695f0be103e4" target='_blank'>
              MTAP Deletion as a Therapeutic Vulnerability in Cancer: From Molecular Mechanism to Clinical Targeting
              </a>
            </td>
          <td>
            P. Krawczyk, K. Wojas-Krawczyk
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Exome and genome sequencing have greatly improved the diagnosis of rare genetic disorders but remain limited in their ability to identify and classify non-coding variants, including intronic variants, cryptic splice-site alterations, and disruptions in regulatory regions. RNA sequencing (RNA-seq) has emerged as a powerful tool to bridge this gap by providing functional insights into genomic variants that disrupt splicing or gene expression, thereby aiding in variant interpretation and classification. We retrospectively reviewed 30 cases from the Utah Penelope Program and the Undiagnosed Diseases Network over a three-year period, in which RNA-seq was performed on whole blood and/or fibroblasts following either negative DNA sequencing or the identification of candidate variants requiring functional assessment. In these cases, RNA-seq identified exon skipping, cryptic splice-site activation, and intron retention, leading to transcript disruption. Additionally, positional enrichment analysis clarified X-inactivation patterns and dosage effects, confirming the pathogenicity of copy number variants. By detecting these transcript-level alterations, RNA-seq provided functional evidence supporting the reclassification of multiple variants of uncertain significance, contributing to diagnostic resolution in selected cases. This study underscores the clinical utility of RNA-seq in detecting splicing and regulatory defects that DNA sequencing and predictive tools alone cannot resolve. Integrating RNA-seq into clinical workflows can support variant classification, aid in diagnostic resolution for selected cases, and provide mechanistic insights into genetic disorders, contributing to patient care and genetic counseling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79a1c19c659e1e0a97e71a33fc57cb1b89e9e984" target='_blank'>
              RNA sequencing provides functional insights and diagnostic resolution in previously unsolved rare disease cases
              </a>
            </td>
          <td>
            Robert G Lewis, John M. O’Shea, Lucilla Pizzo, Ting Wen, Makenzie L. Fulmer, Jian Zhao, Jan Verheijen, Chaofan Zhang, Matt Velinder, Thomas J. Nicholas, S. Boyden, Alistair Ward, Erin Baldwin, A. Andrews, Joselín Hernández Ruiz, Marco Marchetti, D. Viskochil, John C. Carey, S. Bleyl, Russell Butterfield, V. Taliercio, L. Botto, Rong Mao, P. Bayrak-Toydemir
          </td>
          <td>2025-11-14</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id=". The average survival duration for affected patients ranges from 14 to 20 months. This study aims to elucidate the molecular underpinnings of radiation resistance in GBM by integrating bioin-formatic analyses with experimental laboratory approaches. Materials and Methods: Analysis of gene expression datasets identified four candidate genes (KLF10, THBS1, NFYB, OSR1). Human GBM cells were cultured, irradiated with 6 Gy, and processed for RNA isolation and qPCR to evaluate gene expression. Results: Our results showed that OSR1 expression was significantly increased, while THBS1 levels were decreased in radiation-resistant cells compared to control samples; however, KLF10 and NFYB did not exhibit significant changes. Conclusion: These findings implicate OSR1 and THBS1 as potential biomarkers of radiation resistance. Further investigations into their pathways could help in developing targeted therapies for GBM.M">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a55eb4efe451733391590a11447a057c12b3f458" target='_blank'>
              Identifying Mechanisms of Radiation Resistance in Glioblastoma Using Bioinformatic and Molecular-Cellular Approaches
              </a>
            </td>
          <td>
            Arezou Eshaghadadi, Taereh Foroutan, Ali Gorji, Noormohammad Meshkinkhood
          </td>
          <td>2025-12-01</td>
          <td>The Neuroscience Journal of Shefaye Khatam</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Globally, breast cancer (BC) is the most prevalent cancer in women, and drug resistance is a foremost challenge in BC management that contributes to 90% of BC-related deaths. Drug resistance arises as cellular adaptation during treatment, but the underlying mechanisms remain complex and are not yet fully understood. Research studies have shown that most anticancer drugs are effective against the mass of tumor cells rather than Cancer Stem Cells (CSCs). They have the ability to become dormant upon exposure to anticancer drugs and acquire drug resistance over a period of time after silently modifying themselves to resist the drug environment. The maintenance of the stem cell phenotype is mostly driven by stem cell transcription factors (e.g., KLFs, Myc, ZEB2, RUNX1, TWIST1, OCT4, NANOG, FOXO3, SOX2, E2Fs, etc.), which are mediated by epigenetic changes in tumor cells at many levels. These recognized factors/pathways have been demonstrated to be promising therapeutic targets. Therefore, knowledge of epigenetics in regulating cancer stem cells via stem cell transcription factors may be a promising solution for the reversal of therapy-resistant BC. In this review, we emphasize stem cell transcription factors, cancer stem cells, and epigenetic regulation as critical drivers of drug resistance in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bb0cff2c052008da468be3376723f26d8ad1c7" target='_blank'>
              Interplay between Epigenetic and Transcription Factors Mediating Drug Resistance via Stem Cells in Breast Cancer
              </a>
            </td>
          <td>
            Komal Barhela, A. Chaudhary, Richa Mishra, Nupur Nupur, Sri Krishna Jayadev Magani, H. Rudayni, Sanjay Kumar, Binayak Kumar
          </td>
          <td>2025-12-09</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Analysis of coding areas has long been used to study monogenic illnesses, but despite the extensive use of whole-exome sequencing (WES), up to half of suspected cases remain genetically unexplained. Variants outside coding areas can alter splicing, transcript stability, or gene regulation, compromising normal gene activity. These include mutations in noncoding RNAs, promoters, enhancers, deep intronic sequences, and untranslated regions (UTRs). Several well-known disorders have been linked to these mechanisms, including β-thalassemia caused by deep intronic mutations leading to aberrant splicing, familial hypercholesterolemia caused by promoter defects affecting LDLR expression, and inherited retinal diseases driven by noncoding variants influencing retinal gene regulation. These instances show that pathogenic variation is not limited to the exome and can have significant clinical implications. This review summarizes current understanding of noncoding and regulatory variants in monogenic diseases, discusses how they influence diagnosis and therapy, and highlights integrative approaches combining genomic, transcriptomic, and epigenomic data. Multi-layered research has increased diagnostic accuracy and unveiled new therapeutic potentials, although noncoding variations make the connection between genotype and phenotype more complex. Noncoding regions will need to be incorporated into standard diagnostic procedures to convert molecular insights into concrete therapeutic applications in the future. Predictive algorithms, patient-derived model systems, and functional validation testing will all help to simplify this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffd879f4fd9d85153dc94e05643890e26eb43d7" target='_blank'>
              Beyond the Exome: The Role of Noncoding and Regulatory Variants in Monogenic Diseases
              </a>
            </td>
          <td>
            Efthalia Moustakli, Nektaria Zagorianakou, Stylianos Makrydimas, Andreas Miltiadous, Alexandros T. Tzallas, G. Makrydimas
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Ovarian cancer is a lethal disease with high mortality due to late diagnosis, recurrence, and chemotherapy resistance. There is an urgent need for better diagnostic and therapeutic strategies. Recently, protein post-translational modifications (PTMs) have gained significant attention for their role in the onset, progression, and treatment of ovarian cancer. PTMs (including phosphorylation, ubiquitination, methylation, ADP-ribosylation, and others) significantly influence tumor cell proliferation, metastasis, and drug resistance by modulating cellular signal transduction, DNA repair mechanisms, and metabolic processes. PARP inhibitors block the active site of PARP1 in BRCA-mutant ovarian cancer, disrupting ADP-ribosylation, inducing apoptosis in cancer cells, and extending progression-free survival. However, the emergence of drug resistance, such as BRCA reversal mutations, and the insufficient specificity of targets remain significant limitations to therapeutic efficacy. Strategies targeting other modification pathways, such as ubiquitination and methylation, provide promising options for expanding treatments. Combination therapies, like integrating PARP inhibitors with chemotherapy or immunotherapy, and precision medicine also show potential to overcome therapeutic challenges. This article systematically examines the pivotal role of PTMs in ovarian cancer pathogenesis, outlines therapeutic strategies and associated challenges in targeting PTMs, and offers a comprehensive overview with strategic guidance for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7787d1c8ae8d6c5881a1febad8059703b970c76a" target='_blank'>
              Therapeutic targeting of post-translational modifications in ovarian cancer: mechanisms and clinical applications
              </a>
            </td>
          <td>
            Ling Ke, Ying Zhang
          </td>
          <td>2025-11-11</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell sequencing (SCS) techniques have the potential to offer precise insights into cellular diversity by revealing unique genetic and transcriptomic profiles at the single-cell level. This advanced technology has been used extensively in research, but it has still not translated to clinical diagnostics, despite its potential. SCS provides more accurate granular information about heterogeneous cell populations and the creation of personalized treatment strategies. However, the integration of SCS into clinical practice is challenging. This review discusses the potential of SCS technologies in improving clinical molecular diagnostics in various clinical areas including oncology, genetics and rare diseases, infectious diseases, and autoimmune disorders and inflammation. We review recent advances, current uses, integration challenges, and their contribution to the development of these fields. SCS provides significant opportunities in oncology to analyze tumor heterogeneity and develop personalized treatments. In autoimmune and rare diseases, SCS has helped to define novel biomarkers and understand complex immune pathways. SCS has also been used to understand pathogen diversity and host-pathogen interactions in the context of infectious diseases, leading to targeted therapeutic approaches. Despite this progress, there remain challenges in data analysis, standardization, and routine clinical application. The future of SCS in clinical molecular diagnostics is promising. Further technological and research developments in SCS are expected to increase the precision and personalization of medical diagnostics and treatment. To overcome current limitations, interdisciplinary cooperation and innovative approaches to data analysis are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f80640c729e34d2cf6d8aaba32d154d9a103af" target='_blank'>
              Single-cell sequencing in molecular diagnostics: Transformative yet untapped potential
              </a>
            </td>
          <td>
            D. Elshiekh, Nour Farchoukh, Reema El Hassan, Sara Al-Marzooqi, W. Aldosari, Haissam Abou-Saleh, Amal Al-Haidose, H. Zayed, Atiyeh M. Abdallah
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is paradigmatic for therapeutic resistance driven by genetic heterogeneity, epigenetic plasticity and microenvironmental protection. Over the past decade, six targeted or pathway-directed small molecules—midostaurin, gilteritinib, quizartinib, ivosidenib, enasidenib, venetoclax and glasdegib—have changed frontline and relapsed/refractory (R/R) practice in genomically defined subgroups or in patients unfit for intensive chemotherapy. Yet primary refractoriness and early relapse remain common, frequently via adaptive rewiring of apoptotic dependencies, clonal evolution and differentiation resistance. Here we integrate mechanistic insights with clinical evidence to: (i) map resistance biology onto targetable nodes (apoptosis control; signalling kinases; chromatin/lineage programmes; RNA splicing; DNA-damage response; nuclear export; niche adhesion and innate immune evasion); (ii) summarise the clinical trajectory and current limits of approved and emerging small molecules (including menin and LSD1 inhibitors); (iii) propose rules for rational doublets and triplets that are biologically orthogonal yet clinically tolerable; (iv) outline a regulatory timeline for key AML small molecules; and (v) prioritise where drug development should go next, including next-generation BH3 toolkits, clonal-pressure-aware designs, minimal residual disease (MRD)–adapted trials and therapy guided by dynamic functional profiling. The review closes with cross-platform challenges—myelosuppression, infectious risk, resistance monitoring and trial design—and a pragmatic framework for moving beyond incrementalism toward durable control and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f83b4532e166032da2b89eea2b4afa25aa7b8b" target='_blank'>
              Acute myeloid leukemia drug resistance: targetable nodes and the clinical trajectory of small-molecule inhibitors
              </a>
            </td>
          <td>
            Xixi Zhang, Yao Peng, Yina Tian, Shichao Chen, Yijiu Jia, Mengtong Liu, Li Zhang
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract The clonal evolution of breast cancer involves a complex dialogue between tumor cells and their environment. In this process, epigenetic mechanisms play a crucial role in regulating the cellular transcriptome without altering the underlying DNA sequence. Here, we provide an updated summary of three main epigenetic mechanisms: histone modifications, long non-coding RNAs (lncRNAs), and higher-order chromatin structures. Post-translational modifications of DNA or histones influence gene expression by altering chromatin accessibility and/or recruiting regulatory protein complexes. This process is dynamically regulated by enzymes that add or remove these marks, as well as by reader proteins that recognize them. Dysregulated expression or malfunction of these regulators creates an aberrant epigenetic landscape and gene expression profile, contributing to breast cancer initiation, metastasis, and drug resistance. Notably, the donor molecules for chromatin modifications are largely derived from intermediate metabolites shaped by environmental cues, highlighting the intricate crosstalk between epigenetic regulation and both cellular and systemic metabolic states. DNA and histone modifications are further interrelated with lncRNAs and higher-order chromatin architectures, which have been actively investigated in breast carcinogenesis. We also briefly introduce the role of epigenetics in other chromatin-associated events such as replication initiation. Aberrant replication initiation can drive gene duplication and genomic alterations resembling those observed in clinical breast cancer, endowing tumor cells with growth advantages and therapeutic resistance. Finally, we summarize emerging therapeutic strategies that target epigenetic vulnerabilities in breast cancer and discuss their current limitations and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502ad7aff3505dc7bf1c94f78c3fe440208426d" target='_blank'>
              Advances in the epigenetic regulation of breast cancer
              </a>
            </td>
          <td>
            Xiao Liu, Xin Wu, Jianan Ding, Xiaoying Qiao, Jing Liang
          </td>
          <td>2025-11-21</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human body is capable of mutating a single gene to produce a wide range of debilitating disorders. Genomic editing for disease prevention via phenotypic reversal was a significant challenge prior to the development of clustered regulatory interspaced short tandem repeats (CRISPR) and CRISPR-associated protein (Cas) systems. Gene therapy—editing a patient’s DNA to correct a particular mutation—and treating human diseases that have not responded to conventional medicine are two areas where CRISPR/Cas9 technology shows the most promise as a therapeutic tool. This powerful instrument has shown great promise in muscle-related illnesses, offering new insights into muscle biology and developing more effective treatment techniques. Discoveries about the hereditary causes of the majority of inherited myopathies and muscular dystrophies (MDs) have emerged over the last two decades. Additionally, skeletal muscles weaken and degenerate over time due to a group of hereditary disorders known as MDs. The field of skeletal muscle diseases and associated genetic alterations is seeing remarkable progress in developing therapeutic vectors to fix these mutations. Myopathies, MDs, and neuromuscular disorders are just a few examples of the many genetic abnormalities related to muscles that have sparked renewed interest in the potential of genome editing as a therapeutic tool due to its efficiency, adaptability, and relative ease of use in targeted genome editing. Consequently, CRISPR/Cas9 has garnered much interest and is used more often in therapeutic techniques due to its potential capacity to cure various human ailments. To pave the way for more effective and personalized therapies, this review article provides a thorough overview of the revolutionary role of CRISPR/Cas9 in improving our understanding and treatment of genetic disorders related to muscles by combining present knowledge with future perspectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58510c371e81379695caa087647d6ea2ee2145ab" target='_blank'>
              A review of recent studies on CRISPR/Cas9-mediated genome editing in a variety of muscle-related genetic disorders
              </a>
            </td>
          <td>
            Guangchao Lyu, Pengjun Li, Weian Lang
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The mismatch repair (MMR) system plays a crucial role in correcting replication errors; when the MMR system is deficient (dMMR), replication errors (particularly within microsatellites) accumulate throughout the genome, leading to microsatellite instability (MSI). The key MMR genes include MLH1, MSH2, MSH6, and PMS2. Germline mutations in these genes are associated with Lynch syndrome, whereas in sporadic solid tumors, dMMR is often caused by hypermethylation of the MLH1 promoter. As the clinical use of dMMR/MSI expands, the importance of reliable testing for dMMR or MSI in companion diagnostics continues to increase. dMMR/MSI is diagnosed using immunohistochemistry (IHC) for MMR proteins, polymerase chain reaction with fragmentation analysis, or next-generation sequencing. Although IHC has technical limitations, it requires less tissue, has a short processing time, and is cost-effective. Experienced or specialized pathologists and educational efforts are helpful for reliable diagnosis, in addition to the technical aspects. Solid tumors with dMMR/MSI exhibit distinct clinicopathological features, including prognostic significance and a predictive role in adjuvant cytotoxic chemotherapy. Solid tumors with dMMR/MSI are also characterized by a higher tumor mutational burden and abundant immune cell infiltration, making them promising candidates for immune checkpoint inhibitor therapy. However, the oncogenic processes and immune microenvironment are not identical across the organs of origin, between patients, and even within the same patient, which should be considered in future studies. This review provides an overview of the practical aspects of dMMR/MSI testing, along with the molecular mechanisms and immune microenvironments associated with dMMR/MSI solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a00c3e7692fb4f04438233e60ff738a9ba38c4e7" target='_blank'>
              Diagnostic Challenges and Clinical Implications of Microsatellite Instability/Mismatch Repair Deficiency in Solid Tumors.
              </a>
            </td>
          <td>
            Y. Kwak, Jeong Mo Bae, Hye Seung Lee
          </td>
          <td>2025-12-26</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The occurrence and progression of tumors are closely linked to the abnormal metabolic mechanism of fatty acids. In tumor cells, fatty acids are highly expressed and provide molecular structural substances for cells, thereby sustaining tumor growth. In previous experiments, researchers cultured cells in a fatty acid-deficient environment and analyzed the growth status of the cells in batches. These studies confirmed that fatty acids play crucial roles in tumorigenesis and have spurred research on targeted regulation for tumor therapy. However, traditional cellular and molecular experiments have certain limitations, making it difficult to resolve the key issue of metabolic heterogeneity among different subpopulations of tumor cells. This limitation has contributed to the failure of targeted therapies to achieve long-lasting therapeutic effects in the treatment of some solid tumors. In contrast, single-cell RNA sequencing (scRNA-seq) technology can analyze gene expression profiles at the single-cell level, overcoming the constraints of traditional methods, and has thus been widely applied in research on tumor metabolism. This technology can accurately identify tumor cell subpopulations with specific phenotypes (such as cancer cell subpopulations with high expression of fatty acid synthase and radiotherapy-resistant glioblastoma cell subpopulations dependent on fatty acid oxidation), clarify the remodeling effects of metabolic heterogeneity on the tumor microenvironment, and provide a detailed molecular mechanism basis for research on targeted tumor therapy. Consequently, it can effectively address the problem of metabolic heterogeneity in tumor cells. In the future, we can integrate scRNA-seq with technologies such as lipidomics and spatial transcriptomics to promote its clinical translation, guide personalized targeted therapy, and provide a precise and efficient new strategy for metabolic targeted therapy in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50efd5c2ed9ab9479446983a9295d29161e54dbd" target='_blank'>
              Application of Single-Cell RNA Sequencing Technology: Advancing Targeted Fatty Acids for Antitumor Therapy
              </a>
            </td>
          <td>
            Zenan He
          </td>
          <td>2025-11-07</td>
          <td>Journal of Medical and Health Studies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29ff943c6c6477ff522907b96d9f6098e2b606d" target='_blank'>
              An AI-driven multi-omics framework identifies lactylation-mediated therapeutic targets to overcome drug resistance in ovarian cancer.
              </a>
            </td>
          <td>
            Lijia Zhang, Qi Guo, Xue Lei, Xinyu Yin, Yun Ling, Ye Liu, Songjiang Liu
          </td>
          <td>2025-12-20</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Resistance to anti-cancer drugs remains a major challenge in chemotherapy. Combination therapy using sensitizers has emerged as a promising strategy to restore drug sensitivity in resistant cancer cells. However, experimental screening of sensitizers is laborious and costly, highlighting the need for computational methods that enable systematic and efficient prediction. We developed SOCAR, a network-based computational framework that predicts sensitizer drugs by integrating transcriptome profiles with molecular interaction networks. SOCAR identifies resistance-associated genes and network modules, and quantifies each drug's potential to reverse these resistance mechanisms. Applied to 4,009 drugs, SOCAR accurately predicted candidate sensitizers for tamoxifen-resistant breast cancer (AUROC = 0.90). In vitro assays validated that all twelve top-ranked candidates significantly reduced cell viability (p < 0.005) when co-administered with tamoxifen. Furthermore, protein activity analyses showed that resistance-module proteins were markedly altered after acquiring resistance but were restored to normal levels following combined treatment (p < 0.05). Collectively, SOCAR provides a systems-level framework for discovering novel sensitizers and elucidating mechanisms of resistance reversal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1acf5805cb5dd4152a1362e84cfaff70e9641d" target='_blank'>
              SOCAR: Network-based Computational Framework to Overcome Acquired Tamoxifen Resistance of MCF7 Cells.
              </a>
            </td>
          <td>
            Mijin Kwon, Young Min Woo, Woochang Hwang, Jong-Won Kim, Doheon Lee
          </td>
          <td>2025-12-05</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) assays are being widely adopted in oncology and maternal-fetal medicine. Patterns of cfDNA fragmentation can provide useful information about gene regulation and expression in human disease from a blood draw. Here, we demonstrate that enhancer RNA expression – a marker of enhancer activity – can be inferred from local patterns of cfDNA fragmentation. We define a transcriptional activation score (TAS) that predicts expression of enhancers and genes based on cfDNA fragment sizes and positions near transcriptional start sites (TSSs). The TAS identifies activity of cancer-associated enhancers in patients with cancer, distinguishes clinically relevant cancer subtypes, and identifies activation of enhancers associated with treatment resistance and therapy response. We propose a simple model to account for our findings based on chromatin fiber structure and the depletion of H1 histone proteins near active TSSs. Our model provides a unified framework that reconciles seemingly conflicting observations from prior fragmentomics studies. Broadly, this work enables blood-based assessments of gene regulation in cancer and non-oncologic diseases to inform pathobiology, diagnosis, and treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac0e05f4646699f356f6a03f11451dcdcf9d0ec7" target='_blank'>
              Breakthroughs in genetics and genomics research in the first 25 years of the new millenia
              </a>
            </td>
          <td>
            Somnath Paul
          </td>
          <td>2025-11-08</td>
          <td>The Nucleus</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background X chromosome inactivation (XCI) refers to silencing of genes on one copy of the X chromosome in XX females, resulting in dosage compensation between XX females and XY males. Genes can escape this silencing, potentially leading to sex-based differences in disease. Results Across three primary tissue types, we determined XCI status by integrating whole-genome sequencing with bulk RNA-Seq data to assess allele-specific expression (ASE) at heterozygous SNPs. Across all genes and tissues, the average percentage of individuals showing escape was 4.7%. We show that models of full dosage compensation and no dosage compensation are strongly correlated over most parameter space. For G6PD deficiency, we illustrate that in G6PD*B/G6PD*A- heterozygotes, even if silencing is complete and escape is not associated with disease risk, the allele that is expressed can affect mRNA abundance. Conclusions With respect to studies of the genetic architecture of complex traits, our results suggest that a model of full dosage compensation, although not strictly correct for much of chromosome X, is more appropriate than a model of no dosage compensation. We conclude that uncertainty about the model of dosage compensation should not be an impediment to analysis of chromosome X in genetic epidemiology studies. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-12352-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b5cebb4ca442286d049b6f6ea57e87b062a952" target='_blank'>
              X chromosome inactivation across primary human tissues is mostly complete, with significant implications for genetic and clinical studies
              </a>
            </td>
          <td>
            D. Shriner, A. Doumatey, Lin Lei, Mateus H. Gouveia, K. Meeks, Elisabeth F Heuston, Jie Zhou, A. Adeyemo, C. Rotimi
          </td>
          <td>2025-11-24</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly specialized intratumoral compartment responsible for tumor initiation, metastatic dissemination, therapeutic resistance, and disease recurrence. A central conceptual challenge in CSC biology is their capacity to oscillate between a quiescent G0 state and a proliferative, stem-like phenotype, reflecting a high degree of phenotypic plasticity. Although dysregulation of the G1/S checkpoint is a hallmark of malignant transformation, its mechanistic contribution to CSC identity and plastic behavior remains poorly defined.This review outlines a conceptual model that integrates aberrant G1/S control with CSC state transitions. We propose that defective checkpoint regulation accelerates CSC proliferation, leading to the progressive intracellular accumulation of Cyclin D, which in turn drives a self-reinforcing, rapid G1 progression through phosphorylation-dependent pathways that operate independently of the slower, transcription-driven Cyclin D-Rb-E2F regulatory axis. With continued cycling, depletion of key E2F-regulated DNA replication factors ensues, eventually forcing CSCs into a quiescent, biosynthetic restoration phase. During this interval, essential genomic replication and cell cycle machinery are replenished until microenvironmental or intracellular cues trigger reentry into the proliferative cycle, giving rise to another burst of accelerated division.Through these cyclical perturbations in the Cyclin D/E2F balance, CSCs undergo temporally governed shifts between quiescent and proliferative states, thereby sustaining plasticity, intratumoral heterogeneity, and treatment-resistant phenotypes. This model also identifies potential therapeutic strategies, such as leveraging stimuli-responsive delivery systems that exploit cyclic CSC vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53543eb088f0d567f8bdf7288c39cb199fa8756" target='_blank'>
              Dysregulation of the Cyclin D/E2F activity as a core mechanism driving cancer stem cell plasticity and cell cycle dynamics.
              </a>
            </td>
          <td>
            Yasin Ahmadi, Trefa Mohammed, Sakhavat Abolhasani, Shwan Abdullah Hamad, Tahran Faiq
          </td>
          <td>2025-12-14</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard J Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, Ana P. Gomes, Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing.
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer.
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background IDH1-mutant gliomas often undergo malignant transformation over time and progress into treatment-resistant, high-grade tumors. While mutant IDH1 inhibitors have shown some efficacy against lower-grade gliomas, their effects on high-grade gliomas remain limited. Under these circumstances, there is an urgent need to identify novel therapeutic targets from the perspective of synthetic lethality. Methods To identify genes exhibiting synthetic lethality with mutant IDH1 inhibitors, we performed a genome-wide CRISPR/Cas9 screen. Although treatment with the mutant IDH1 inhibitor alone effectively suppressed intracellular D-2-hydroxyglutarate production, it had minimal impact on cell proliferation. Therefore, we aimed to discover gene alterations that could elicit antitumor effects not achievable by monotherapy. Results The screening identified 176 gene candidates that demonstrated synthetic lethality with the mutant IDH1 inhibitor. Gene Ontology analysis revealed that many of these genes were associated with metabolic pathways. Among them, we focused on KMT5A, a gene involved in amino acid metabolism for which small-molecule inhibitors are already available. Knockdown of KMT5A using siRNA alone did not affect cell proliferation; however, its combination with the mutant IDH1 inhibitor significantly suppressed cell growth. Furthermore, the use of a KMT5A inhibitor in combination therapy led to a marked reduction in cell viability, G0/G1 cell cycle arrest, and induction of late apoptosis. These combinatorial effects were consistently observed across two different cell lines, suggesting potential broad applicability of the strategy. Conclusion This study successfully identified novel synthetic lethal targets that enhance the efficacy of mutant IDH1 inhibitors in IDH1-mutant gliomas. In particular, combination therapy with KMT5A inhibition emerged as a promising treatment strategy. Ongoing investigations aim to elucidate the underlying molecular mechanisms, including epigenomic alterations. These findings may contribute to the development of precision medicine approaches to overcome resistance to mutant IDH1 inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15140c48b89ec4eac656649a6fb680b7745e1be" target='_blank'>
              10092-CB-3 Resistance Mechanisms and Target Identification sssin Mutant IDH Inhibitor Therapy
              </a>
            </td>
          <td>
            Sachi Maeda, Kosuke Aoki, F. Ohka, S. Deguchi, Yoshiki Shiba, Yuhei Takido, Shohei Ito, Shigeaki Nawa, Ryuta Saito
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ca5c42693a477fde2aa581c6c760a192f8faa2" target='_blank'>
              The cumulative impact of passenger mutations on cancer development
              </a>
            </td>
          <td>
            Akshatha Nayak, Candace S. Y. Chan, Ioannis Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Activating KRAS mutations are highly relevant to various cancers, and KRAS is the most frequently altered oncogenic protein in solid tumors. While historically considered undruggable, two KRASG12C inactive state-selective inhibitors are currently approved for treating patients with non-small cell lung cancer. However, these agents only demonstrate a 30–40% response rate and a median progression-free survival of approximately 6 months, with the inevitable emergence of resistance mechanisms, hence remaining far from achieving a cure. Additionally, several cancers with poor prognostic outcomes, such as pancreatic adenocarcinoma, are driven by other non-G12C KRAS mutations and thus have no effective targeted therapies. Improvements in understanding RAS signaling, RNA, and nucleic acid chemistry, as well as the role of the tumor microenvironment, have sparked a paradigm shift in the approach to KRAS inhibition and suggested the potential for several novel combination therapies. In this review, we provide an overview of the RAS pathway and discuss the ongoing development and status of therapeutic strategies for targeting the oncogenic RAS. We further delve into the challenges of resistance mechanisms to better understand the rationale behind these developing strategies, describe their mechanisms of action, and offer insights into the current clinical trial status of each of these approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c697e95b8beed852f306ce44fd5e98ffcf2023be" target='_blank'>
              Targeting KRAS mutations: orchestrating cancer evolution and therapeutic challenges
              </a>
            </td>
          <td>
            Khalil Choucair, Hafsa Imtiaz, Md. Hafiz Uddin, M. Nagasaka, M. N. Al-Hallak, P. Philip, B. El-Rayes, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-11-28</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Summary SISTEM is a software package and mathematical framework for simulating tumor evolution and cell migrations at single-cell resolution. Unlike existing frameworks which simulate cancer cell populations under the neutral coalescent or using simple birth–death models, SISTEM simulates tumor populations under somatic clonal selection using an agent-based framework. SISTEM can generate mutation profiles, read counts, and DNA sequencing reads along with ground truth cell lineages and migration graphs under a number of easily customizable mutation and selection models. For improved realism, SISTEM allows for cell fitness to be driven by genomic events of various scales including single nucleotide variants, segmental gains and losses, whole-chromosomal and chromosome-arm aberrations, and whole-genome duplications. SISTEM also includes numerous migration models to simulate metastatic cancers, facilitating the exploration and evaluation of diverse migration patterns. Availability and implementation SISTEM is written in Python and is freely available open-source under GNU GPLv3 from: https://github.com/samsonweiner/sistem">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6ea45c6bd4d361ab9e67581ccc6b02aecffc37" target='_blank'>
              SISTEM: simulation of tumor evolution, metastasis, and DNA-seq data under genotype-driven selection
              </a>
            </td>
          <td>
            Samson Weiner, M. Bansal
          </td>
          <td>2025-11-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Heritable breast cancer (BC) is influenced by well-known high-penetrance genes such as BRCA1, BRCA2, PALB2, and CHEK2, but the contribution of many moderate- and low-penetrance genes remains unclear. This study used large, multi-ethnic genomic datasets, including the UK Biobank and FinnGen, to refine the set of genes associated with BC predisposition. By integrating multiple association approaches (GWAS, TWAS, and PWAS) and removing likely false positives, we reported on 38 high-confidence BC predisposition genes. These include established DNA repair genes and new candidate genes supported by independent evidence. PWAS also highlighted genes with potential recessive effects that are often missed by standard GWAS. Findings were most consistent in European ancestry populations. Overall, this work provides a conservative, gene-focused framework to clarify biological pathways and prioritize candidate genes for further functional study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f8ebf2ad2ff2b0876594246e484e2180346625" target='_blank'>
              Integrative Gene-Centric Analysis Reveals Cellular Pathways Associated with Heritable Breast Cancer Predisposition
              </a>
            </td>
          <td>
            R. Zucker, S. Schreiber, A. Stern, M. Linial
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Among primary malignant brain tumors in adults, glioblastoma is the most common and the most aggressive, characterized by profound metabolic reprogramming. This metabolic shift is essential for sustaining relentless proliferation and adapting to the challenging tumor microenvironment (TME). Central to this adaptation in glioma is the Warburg effect, which leads to excessive lactate production and accumulation, even in the presence of oxygen. This metabolic divergence significantly impacts the tumor immune microenvironment, promoting the recruitment of immunosuppressive cells and weakening the anti-tumor immune response. Core Content: This review provides a comprehensive analysis of the multifaceted roles of lactate in IDHwt glioma pathogenesis. It explores how lactate serves as a critical nexus connecting aberrant metabolism, epigenetic reprogramming (notably via histone lactylation), and immune evasion. The review delves into the molecular mechanisms by which lactate, particularly through the post-translational modification known as lactylation, directly modulates the epigenetic landscape to promote oncogene expression. Furthermore, it examines lactate’s role in acidifying the TME, promoting the immunosuppressive M2 polarization of glioma-associated macrophages (GAMs), and inhibiting the cytotoxic activity of T lymphocytes. Conclusions: This “lactate-centric” framework provides a unifying model that links metabolic dysregulation directly to malignant progression and therapeutic resistance (e.g., to TMZ). By elucidating this metabolic–epigenetic–immune axis, the review highlights a critical dependency that fuels glioma aggression. Finally, it discusses emerging therapeutic strategies aimed at targeting lactate production (LDHAi), transport (MCTi), and downstream epigenetic signaling (HDACi/p300i), offering novel avenues for integrative immunometabolic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b50b3cdb0eb39f78be330d84ff6e266ce7350786" target='_blank'>
              Lactate-Mediated Epigenetic and Immunometabolic Reprogramming in Glioma: An Emerging Axis Linking Metabolism to Tumor Progression
              </a>
            </td>
          <td>
            Xinyi Xie, Wenjing Zhou, Yin Ku, Shasha Li, Yunhao Yang, Xiaohu Hao, Yaohui Chen
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Antisense genes (usually suffixed by -AS) represent a class of long non-coding RNAs (lncRNAs) transcribed from the opposite strand of annotated human genes or exon(s). A total of ~2236 human antisense genes exist in the human genome. Their genomic locations with respect to the corresponding sense genes, their dysregulated expression patterns in cancer specimens, and clinical associations with patient outcomes reveal their potential importance in clinical settings. As of today, there lacks a comprehensive review of HNC-associated antisense genes/transcripts to help move forward the antisense field for genetic biomarker development or future drug research. In total, 2.3% (52/2236 antisense genes) of all known human antisense genes have been investigated in head and neck cancer (HNC). Thus, we perform a comprehensive review of the genomic aberrations (mutations, copy number changes, RNA-expression dysregulation, and single nucleotide polymorphisms) associated with HNC patient prognosis, disease progression, cancer cell signaling, drug sensitivity, and radio-resistance. Four antisense genes, namely HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1, have been clinically cross-validated and have consistently demonstrated to be associated with patient outcomes in multiple independent cohorts by different research teams, with clear evidence for the prioritization of clinical biomarker development in HNC. Single nucleotide polymorphisms (SNPs) of antisense genes with evidence for HNC risk or outcomes should be further validated in different ethnic groups, for potential global HNC applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb801fbeee9cde00558b26b3b11efef6455c2950" target='_blank'>
              Genomic Aberrations of Antisense Gene Transcripts in Head and Neck Cancer
              </a>
            </td>
          <td>
            Jishi Ye, Stacy Magdalene Abbang, Yuen-Keng Ng, V. Lui
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly resistant brain tumor with a dismal prognosis despite standard treatments like temozolomide (TMZ) and radiation therapy (RT). This review delineates the molecular mechanisms driving this resistance to inform future therapeutic strategies. A narrative review was conducted using PubMed, synthesizing studies from the year 2000 up to March 2025 on DNA repair enzymes, TMZ and RT resistance pathways, anti-apoptosis mechanisms, genetic heterogeneity, and cell cycle dysregulation in GBM. Five key resistance mechanisms were identified: (1) DNA repair enzymes (e.g., MGMT, APNG) counteract TMZ-induced damage, regulated by NF-κB and CHD4; (2) TMZ and RT resistance depends on MGMT methylation and MMR functionality; (3) anti-apoptotic pathways (e.g., Bcl-2, PI3K/Akt) protect GBM cells, mitigated by TRAIL sensitization; (4) genetic mutations (EGFR, PTEN, TP53) confer adaptability; and (5) quiescent and proliferative GBM stem cells (GSCs) sustain recurrence. Novel therapies targeting HK2 and neural stem cells show promise. GBM resistance is multifaceted, requiring personalized, multi-targeted approaches. Future research should prioritize epigenetic modulation and combination therapies to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d28a9446444f9c597df4c24c1f9700021704068b" target='_blank'>
              Molecular Insights into Glioblastoma Resistance and Implications for Treatment Advancements: A Systematic Review
              </a>
            </td>
          <td>
            Kaustov Chakrabarti, Amanda M. Mohabir, Mahesh Nukala, Sara M. Safiullah, Tierney R. Woitas
          </td>
          <td>2025-11-21</td>
          <td>Scholarly Research In Progress</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metabolic disorders caused by defects in energy metabolism can lead to many life-threatening diseases; their therapy remains elusive in most cases. Conventional gene therapy relies on the “one gene for one genetic defect” strategy. Here, we demonstrate a more efficient strategy to target multiple genetic defects with a single gene intervention. Specifically, we used a bacterial lipoate protein ligase involved in protein lipoylation to rescue mitochondrial dysfunctions in human lipoylation pathway (LIPT2, LIAS, and LIPT1), lipoyl precursor supply (MECR), and sulfur insertion accessary partner (FDX1). The efficacy and safety of Escherichia coli–derived LplA or Bacillus subtilis–derived LplJ were validated in human cells and mouse models. LplA knock-in mice exhibited normal health with enhanced energy expenditure. Overexpressing LplA through a mating strategy rescued embryonic lethality in Lipt1−/− mutants, yielding viable offspring with normal body weight, energy expenditure, tissue morphology, and biochemical profile. Our work highlights how evolutionary differences in biosynthetic pathways between humans and bacteria can be leveraged for cross-species therapeutic innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc8e96d21e3505f923c0e27e1ea85c93abd04cc" target='_blank'>
              Targeting multiple genetic defects of mitochondrial diseases with a single bacterial lipoate protein ligase
              </a>
            </td>
          <td>
            Zhijuan Hu, Junru Yu, Ziwei Liu, Min Jiang, Anping Zeng
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="AIMS
Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease.


MATERIALS AND METHODS
Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines.


RESULTS
We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression.


CONCLUSIONS
We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome.
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Abstract Resistance to single-agent therapy in acute myeloid leukemia (AML) often arises from compensatory epigenetic circuits. We present a causal inference framework that treats therapy as an intervention (do-operator) and defines resistance as the deviation of dual perturbations from the additive expectation of monotherapies. By modeling the causal structure of the leukemia stem-cell epigenetic landscape that maintains identity and fitness, the framework shows that synergistic and antagonistic interactions can be inferred directly from single perturbations, suggesting interaction inference may be simpler than previously assumed. We applied this model to Perturb-seq of 16 epigenetic regulators in KMT2A-rearranged AML (>31,000 single-cell transcriptomes). From single-knockout profiles, the framework correctly predicted synergistic pairs (Menin+KAT6A, Menin+DOT1L) that enhanced differentiation and induced cell death, as well as an antagonistic pair (DOT1L + PCGF1) that conferred resistance. Predictions were validated by pharmacologic inhibition and bulk RNA-seq, confirming the model’s predictive accuracy. This work presents a mechanism-guided approach for rational prioritization of epigenetic drug combinations in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057014eca0b4705ea6ee698244a083b962878940" target='_blank'>
              Causal interaction inference of compensatory structures from single-cell perturb-seq in AML
              </a>
            </td>
          <td>
            Changde Cheng, Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA is continuously exposed to endogenous and exogenous factors that induce oxidative modifications leading to mutations and genomic instability. Oxidative DNA damage plays a dual role, contributing to physiological signaling at low levels while promoting mutagenesis, carcinogenesis and degenerative diseases when unpaired. Among various lesions, an oxidized base, such as 8-oxo-2′-deoxyguanosine (8-oxodG), is one of the major biomarkers of oxidative stress and genomic damage. Cells have evolved sophisticated repair processes, including base excision repair (BER), nucleotide excision repair (NER), and mismatch repair (MMR), to maintain genomic integrity. Dysregulation or polymorphism of these repair genes has been linked with cancer, neurologic, and cardiovascular disorders. This review discusses an overview of what is presently known concerning oxidative DNA damage and repair mechanisms, particularly emphasizing their molecular players, signaling routes, and human disease implications. It further refers to the latest advances in CRISPR-based technologies and multi-omics approaches that are redefining our understanding of DNA damage response (DDR) networks and creating new frontiers for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f647de037f61e5dbeca9c5a033dbdfb5daa6ba" target='_blank'>
              Cellular and Molecular Mechanisms of Oxidative DNA Damage and Repair
              </a>
            </td>
          <td>
            A. Ayna, Cuneyt Caglayan, S. Taysi
          </td>
          <td>2025-11-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a7357fb0ebfb407c6351b54f23dbabe20891f79" target='_blank'>
              Unveiling prognostic genes and regulatory mechanisms of exosome in prostate cancer: an integrated analysis of bulk transcriptomics and single-cell RNA sequencing data
              </a>
            </td>
          <td>
            C. Pu
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Melanoma, the most deadly type of skin cancer that develops from melanocytes, has seen its sharp increase in occurrence worldwide, especially with increased UV radiation exposure., making it a serious public health problem. Despite significant improvements in early diagnosis and treatment techniques, effectively treating melanoma patients, particularly those with metastatic cancer. Resistance to programmed cell death is a characteristic of many malignancies, including melanoma, which affects cellular development, survival, metastasis, and treatment resistance. Non-coding RNAs have been shown in recent research to be essential regulators of several biological processes, including programmed cell death. Research indicates that ncRNAs can have either pro-tumorigenic (causing resistance to cell death) or anti-tumorigenic (causing cell death) effects, making them attractive targets for therapy as well as prognostic and diagnostic biomarkers. This paper thoroughly examines the role of ncRNAs in controling among other types of designed cell death in melanoma, such as anoikis, pyroptosis, ferroptosis, autophagy, necroptosis, apoptosis, cuproptosis, and netosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d1ff8341f2b3acb9b9ae22131e369b5822b8b7" target='_blank'>
              Non-coding RNAs in programmed cell death regulation in melanoma: mechanisms and therapeutic insights
              </a>
            </td>
          <td>
            Yang Liu, Chenqi Guo, Yunshen He, Yun Pu, Xinming Chen, Yu Zhang
          </td>
          <td>2025-11-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Epitranscriptomic modifications represent a fundamental regulatory layer in cancer biology, with RNA methylation emerging as a pivotal mechanism governing transcriptomic dynamics. Among these, 5-methylcytosine (m5C) RNA methylation—a ubiquitous and conserved epitranscriptomic mark—has been identified across diverse RNA species, including mRNAs, rRNAs, tRNAs, and mitochondrial RNAs. Notably, the RNA m5C “writers”—enzymes responsible for installing this modification onto target RNAs—have emerged as central regulators of tumorigenesis, with NSUN (NOP2/Sun RNA methyltransferase) proteins playing a particularly pivotal role. We synthesize current knowledge of the cellular localization, substrate specificity, and biological functions of m5C-modifying enzymes, focusing predominantly on the NSUN family in the cancer context. We first dissect the spatiotemporal regulation patterns of NSUN proteins—from their nuclear roles in pre-mRNA processing to cytoplasmic functions in mRNA decay and translation—and their conserved methyltransferase domains that dictate target RNA recognition. This review further explores the molecular mechanisms by which NSUN proteins govern tumor progression, metastasis, and therapeutic responses, emphasizing their dual roles in both initiating oncogenic programs and maintaining cancer cell plasticity. Finally, we discuss the translational implications of targeting NSUN-mediated m5C pathways, highlighting small-molecule inhibitors designed against NSUN substrate specificity, combinatorial strategies with conventional chemotherapy or immunotherapy, and the promise of epitranscriptomic diagnostics and prognostic based on NSUN expression signatures. By positioning NSUN proteins as integral nodes in the RNA epigenomic network, this synthesis not only deepens our understanding of cancer pathogenesis but also identifies novel epitranscriptomic targets for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27c9e5c606d98501332df4288112a7acba40a71" target='_blank'>
              NSUNs-driven dysregulation: the next frontier in targeted cancer therapy?
              </a>
            </td>
          <td>
            Ying Zhao, Yuying Zhang, Haonan Zhou, Lei Zhang, Jinyu Guo, Yue Tan, Ting Wu, Yi Peng, Ying Che, Zhanlei Pei, Jun Li, Boshi Fu, Minjie Wei, Gang Wu, Xiaoyun Hu, Huizhe Wu
          </td>
          <td>2025-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Malignant tumors developing in the conditions of chronic hypoxia, unlike normal tissues, quickly acquire resistance to decreased oxygen level. Hypoxia-resistant cells in malignant disease obtain various features promoting their survival: they alter metabolism to aerobic glycolysis, activate angiogenesis-stimulating programs, and rearrange signaling cascades to adapt to hypoxia. At the same time, tumor cells become resistant to chemotherapy and radiation therapy and effectively colonize metastatic niches. 
This review analyzes signaling pathways which activate in the conditions of chronic hypoxia and underlie tumor adaptation to insufficient oxygen. Apart from the main pathway associated with activation of hypoxia-inducible factor 1α (HIF-1α), such cascades as STAT3, Snail cascade associated with epithelial-mesenchymal transition, NRF2 factor responsible for adaptation to reactive oxygen species, and stemness factors OCT4, SOX2 and NANOG are also considered. Effective killing of tumor cells requires simultaneous inhibition of several parts of signaling pathways because disruption of just one will lead to switching of signaling proteins to bypass the blocked one. The article lists various inhibitors of signaling pathways patricipating in tumor cell adaptation to chronic hypoxia and describes their mechanisms of action and targets. Development of successful strategies for treatment of hypoxia-resistant tumors requires identification of an effective combination of the above-mentioned and other inhibitors of the key signaling cascades participating in adaptation to hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21710e96836bbca03c72f6e2b505d714f5b359ca" target='_blank'>
              Rearrangement of signaling pathways and adaptation of tumor cells to hypoxia
              </a>
            </td>
          <td>
            O. E. Andreeva, E. I. Mikhaevich, M. A. Krasil’nikov
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastrointestinal (GI) cancers pose a significant global health burden, driven by complex molecular alterations and microenvironmental interactions. Advances in molecular pathogenesis have elucidated recurrent driver gene mutations, such as KRAS, TP53, and APC, alongside dysregulated signaling pathways including Wnt, RAS-MAPK, and PI3K-AKT, which collectively underpin tumor initiation and progression. Complementing genetic changes, epigenetic alterations—such as DNA hypermethylation, histone modifications, and regulatory non-coding RNAs—further contribute to malignant evolution by reshaping chromatin architecture and gene expression. These mechanisms not only promote uncontrolled proliferation but also reinforce therapeutic resistance by dynamically modifying the tumor microenvironment (TME). Molecular subtyping efforts, including The Cancer Genome Atlas (TCGA) classification for gastric cancer (GC) and the Consensus Molecular Subtypes (CMS) for colorectal cancer (CRC), have delineated disease heterogeneity, revealing distinct pathogenic pathways and enabling refined prognostic stratification. Such insights provide the biological rationale for diagnostic techniques and targeted interventions. For instance, anti-EGFR and anti-VEGF monoclonal antibodies disrupt oncogenic signaling and tumor angiogenesis, respectively, and have demonstrated substantial clinical efficacy in selected patient populations. In parallel, immunotherapy has emerged as a transformative modality in oncology. Immune checkpoint inhibitors targeting PD-1/PD-L1 and CTLA-4 reinvigorate antitumor immunity and have reshaped standard-of-care protocols for several GI malignancies. Beyond conventional immunotherapies, innovative strategies such as CAR-T cell therapy and neoantigen-based vaccines are being actively investigated. These approaches aim to overcome immune evasion mechanisms and enhance tumor-specific targeting, offering promise for patients with resistant or advanced disease. This review comprehensively analyzes the evolving molecular landscape of GI cancers and the corresponding development of targeted and immunotherapeutic agents. It highlights a balanced integration of mechanistic discovery and clinical translation, underscoring their synergistic roles in advancing precision oncology and improving survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5602f60be23d19b8d126a3c7ae3c86407dda38e" target='_blank'>
              Gastrointestinal cancer: molecular pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Yang Jin, Xiaobo He, Yanfeng Wu
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-19</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72271f0d34d2f8f24354ade5fddf8b10f1a093fc" target='_blank'>
              Whole-genome landscapes of 1,364 breast cancers.
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, Seokhwi Kim, Jeong Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, P. Paik, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyungjung Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeong Seok Lee, Woo-Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park
          </td>
          <td>2025-12-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human cells possess a complex network of regulatory systems——the DNA damage repair pathway (DDR), to deal with various DNA damage to prevent the inheritance of deleterious mutations. DDR perform its functions via cell cycle checkpoint activation. Mutations in the DDR gene are also the basis for tumor progression, metastasis, and therapeutic effect to treatments that cause DNA damage. Consequently, DDR has received high attention for its role in the therapy against cancer, and the related experiments have been carried out to further explore. Up to now, PARP inhibitors such as Olaparib, Rucaparib, Niraparib, Talazoparib, etc. have been approved for marketing and clinical use, other DDR inhibitors (DDRi) such as ATMi/ATRi/DNA-PKi, Chk1/2i, and WEE1i have partially entered clinical trials. Synthetic lethality has led to the exploration of combination of DDRi with other anti-tumor therapies, in particular, the combination of DDRi with immunotherapies such as chemoradiotherapy and immune checkpoint inhibitors (ICIs) has achieved potential efficacy. This article reviewed the DDR inhibitors that are currently in development or in clinical therapy, analyzed the limitations and future development prospects, and stated the mechanisms and clinical trials in antitumor and immunotherapy, to provide ideas for the subsequent development of DDRi and enhance the efficacy of antitumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9a6e3b57be514835a193a70ee6587199732e497" target='_blank'>
              Research progress of DNA damage repair (DDR) and DDR inhibitors in tumor immunotherapy
              </a>
            </td>
          <td>
            Qi Liu, Chunmei Zhang, Yixuan Gao, Dongmei Feng, Duo Deng, Yun Pan
          </td>
          <td>2025-12-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-throughput sequencing (HTS) has revolutionized tumor immunology by enabling precise dissection of tumor-immune interactions, directly informing the development of precision immunotherapies. This review highlights key advances in HTS technologies—including whole genome sequencing (WGS), RNA sequencing, assay for transposase-accessible chromatin using sequencing, and single-cell immunogenomics (scTCR-seq/scBCR-seq)—and their clinical translation in personalized cancer vaccines, engineered T-cell therapies, and combination regimens. We discuss how these tools decode tumor-specific mutations, immune evasion mechanisms, and therapeutic targets, while addressing challenges in data standardization, sample processing, and computational integration. Emerging breakthroughs such as spatial multiomics, real-time monitoring, and artificial intelligence-driven discovery are transforming the field by enabling dynamic, personalized treatment strategies. Finally, we outline future directions to overcome current barriers and expand equitable access to HTS-driven precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1666fd68dbba4d54d4ecfda7e481cfc47c3e80e1" target='_blank'>
              High-throughput sequencing unveils tumor-immune interactions: From genomic alterations to clinical translation
              </a>
            </td>
          <td>
            Ling Yin
          </td>
          <td>2025-11-07</td>
          <td>Global Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pooled CRISPR screens with single-cell RNA sequencing readout (Perturb-seq) have emerged as a key technique to determine the functionality of a gene by directly perturbing the DNA of the gene. One of the most intriguing recent problems is quantifying the similarity between CRISPR perturbations, for example, whether they upregulate the same set of downstream genes. In this context, genetic convergence refers to the phenomenon where CRISPR disruptions of different genes lead to a similar downstream outcome. Existing methods are mostly heuristic. We present XConTest, a two-step, cross-validated procedure for assessing the genetic convergence problem. The test statistics calculated from that procedure are approximately standard normal when the two perturbations have an orthogonal influence on the cell expression profile. We apply XConTest to two studies: an investigation of the common impact of a suite of autism genes, and a large-scale study of genes associated with immune response to determine sets of genes with common functionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b54b98a0428e3369db12e1caf803f7f624aee31" target='_blank'>
              Genetic Convergence Analysis of CRISPR Perturbations Deciphers Gene Functional Similarity
              </a>
            </td>
          <td>
            Tianyu Zhang, Ergan Shang, Kathryn Roeder
          </td>
          <td>2025-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: There are important gaps in describing the associations between variants found by GWAS and various phenotypes. Prior reports suggest that SNPs in regulatory regions should be further investigated to uncover these associations. Thus, this study involved a novel approach, along with Pharmacoepigenomics, prompting a new coined term “CpG-PGx SNP”. Methods: The rationale behind our analysis strategy was based on the impact of SNPs playing dual roles both in the CpG site disruption/formation and having PGx associations. Thus, we employed GeneCards (relevance score), PharmGKB (significant p-value), and GWAS catalog data for each gene (p < 5 × 10−8). Following the obtainment of the 25 best-scored genes of four major epigenetic processes (methylation, demethylation, acetylation, and deacetylation), we generated two lists of candidate genes, including potential CpG-PGx SNPs and possible CpG-PGx SNPs. Results: Among 2900 significant PGx annotations, we found 99 potential CpG-PGx SNPs related to 16 genes. CYP2B6, CYP2C19, CYP2D6, and COMT genes were the top genes. Additionally, we found 1230 significant GWAS-based SNPs, among them 329 CpG-SNPs related to 48 genes with at least one CpG site disruption/formation. The top gene with the highest CpG-SNPs was TET2, followed by JMJD1C and HDAC9. Importantly, we detected some synonymous variants in the Epigenetically Modifiable Accessible Region (EMAR), which can provide insights into undiscovered roles of these SNPs. We identified 173 CpG-Disruptive SNPs, 155 CpG-Forming SNPs, and just 1 CpG SNP with both impacts. Conclusions: In conclusion, here we introduce CpG-PGx SNP for the first time and suggest three major genes playing crucial roles in Pharmacoepigenomics (PEpGx), CYP2D6 as the heart of PEpGx, and TET2 with the highest possibility of having CPG-PGx SNPs. We believe that this approach will help the scientific community to utilize “CpG-PGx SNP” to unravel complex disease-driven genetic and epigenetic interactions, yielding therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55afff7dd8ab511c66ce56ce8978bd4898622295" target='_blank'>
              Pharmacoepigenomics in Personalized Medicine: A Hypothesis-Generating Approach to Introduce CpG-PGx SNPs as New Candidates for a Systematic Insight into Genomic-Epigenomic-Phenomic-Pharmacogenomics (G-E-Ph-PGx) Axis
              </a>
            </td>
          <td>
            Alireza Sharafshah, Kenneth Blum, K. Lewandrowski, I. Elman, Brian S. Fuehrlein, David Baron, Albert Pinhasov, P. K. Thanos, R. Fiorelli, S. Schmidt, Eliot L. Gardner, Morgan P. Lorio, Alexander P. L. Lewandrowski, Mark S. Gold
          </td>
          <td>2025-11-29</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a heterogeneous tumor, ranging from cases with spontaneous regression (MS stage) to high risk (HR) tumors. Resistance to therapy presents a major challenge in the treatment and management of HR-NB, contributing to a poor prognosis. In these patients, the resistance to conventional treatment is also related to non-genetic features, such as the cellular plasticity. Novel studies have demonstrated the presence of two distinct cell phenotypes: adrenergic (ADRN) and mesenchymal (MES), which reflect the heterogeneity of NB. MicroRNAs (miRNAs) are small, endogenous and non-coding RNAs with the ability to regulate gene expression and may have a crucial role in controlling cell plasticity. However, the role of miRNAs in NB plasticity has not been investigated yet. We investigated miRNA signature in NB cells subtypes (MES and ADRN) and in the extracellular vesicles (EVs) released by them, to identify potential MES related biomarkers. Differentially expressed miRNAs were identified by RT-qPCR and subjected to gene ontology, KEGG pathway, and protein-protein interaction network analyses. Candidate miRNAs were validated in plasma-derived EVs from NB patients. We identified miR-199a-3p as strongly upregulated in the MES cells subtype. Moreover, its expression levels were significantly higher in primary cell lines derived from HR patients compared to low-risk (LR) ones. This was confirmed by a bioinformatics analysis in patient tissue obtained from TARGET NB dataset. Protein-protein interaction analysis uncovered a complex network, with FN1, CD44, and YAP1, identified as key genes upregulated by miR-199a-3p, all of which are closely associated with the MES phenotype. Among the miRNAs significantly upregulated in EVs derived from MES cell lines, miR-584a-5p was significantly higher in EVs isolated from plasma of HR and L/Intermediate(I)R patients compared to MS. MiR-584-5p is typically considered a tumor suppressor; to support this role miR-584a-3p resulted significantly upregulated in L/IR tumor tissue, both in the INSS and COG classifications (TARGET NB database). Our findings identified specific miRNAs as MES phenotype related biomarkers. Further studies should investigate the potential impact of miRNAs on plasticity-related pathways in order to open new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b8406e9d606c161b955f6866c60bbec146aba97" target='_blank'>
              Identification of a miRNAs signature as potential biomarker of mesenchymal phenotype in neuroblastoma patients
              </a>
            </td>
          <td>
            Silvia Lampis, Alessandro Paolini, Virginia Di Paolo, A. Galardi, Salvatore Raieli, E. Miele, Lauriane Lemelle, F. Fabozzi, A. Serra, A. Mastronuzzi, M. A. De Ioris, Andrea Masotti, Franco Locatelli, A. Di Giannatale
          </td>
          <td>2025-11-26</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Epigenetic regulation is a fundamental mechanism controlling gene expression and cellular function, primarily mediated through reversible modifications such as DNA methylation, histone acetylation, and chromatin remodeling. Dysregulation of critical epigenetic enzymes, including histone deacetylases (HDACs), DNA methyltransferases (DNMTs), and bromodomain and extraterminal domain (BET) proteins, has been closely associated with tumor initiation, progression, metastasis, immune evasion, and resistance to conventional therapies. Targeting these epigenetic regulators with small-molecule inhibitors or degraders has emerged as a promising therapeutic strategy, capable of reprogramming aberrant transcriptional networks and reshaping the tumor microenvironment. Beyond direct cytotoxic effects, epigenetic drugs have demonstrated the ability to enhance antitumor immunity by restoring antigen presentation, promoting immunogenic cell death, modulating cytokine profiles, and reversing local immune suppression. Recent preclinical and clinical studies have highlighted the potential of combining epigenetic therapies with immune checkpoint inhibitors to achieve synergistic antitumor responses and overcome resistance mechanisms. This review provides a comprehensive summary of the mechanisms of action, pharmacological characteristics, and clinical applications of epigenetic drugs, with a focus on innovative combination strategies and ongoing translational advancements. We also discuss future directions, emphasizing the need to improve drug specificity, minimize off-target effects, integrate personalized immunotherapeutic approaches, and identify predictive biomarkers to optimize patient selection and clinical outcomes. Overall, epigenetic therapy represents a versatile and evolving avenue for precision oncology with broad implications for tumor control and immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1142f91bb95f6983a1d418601e7e6fb2c4af66" target='_blank'>
              Epigenetic drugs in cancer therapy: mechanisms, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Chen Ma, Junkai Cheng, Jian Gu, Qin Wang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background/Objectives: The RET (Rearranged during Transfection) gene encodes a receptor tyrosine kinase. RET plays a critical role in embryonic development and postnatal physiology. This review provides a comprehensive overview of RET-associated disorders, focusing on the molecular mechanisms of RET activation, associated clinical phenotypes and therapeutic implications. In addition, we present an updated RET mutation database. Methods: RET mutation database is built through the integration and curation of data from two major RET mutation repositories: the Leiden Open Variation Database (LOVD) and the Cancer Knowledge Base (CKB) as well as information derived from the ClinVar database. Results: To date, 78 pathogenic RET mutations have been identified, among these, 71 (91.0%) are single nucleotide substitutions (missense variants), 2 (2.6%) are deletions, 1 (1.3%) are indels, 2 (2.6%) are nonsense mutations and 1 (1.3%) mutation affecting the introns. A pronounced clustering was observed in exons 10–11, accounting for ~60% of cases, suggesting a potential mutational hotspot with structural or functional relevance. Conclusions: Aberrant RET activation, resulting from activating missense variants, gene fusions, or overexpression, underlies a wide spectrum of human diseases. These include multiple endocrine neoplasia type 2A (MEN2A), medullary thyroid carcinoma (MTC), Hirschsprung disease, and pheochromocytoma. The existence and use of a database classifying variants in the RET gene plays a fundamental role in molecular diagnostics and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abf390ac8f6c6ce13e6f1d602040b530d5fbd8fc" target='_blank'>
              RET Gene Alterations in Clinical Practice: A Comprehensive Review and Database Update
              </a>
            </td>
          <td>
            Claudio Ricciardi Tenore, Eugenia Tulli, Alessia Perrucci, Roberto Bertozzi, Ludovica Fortuna, Giulia Maneri, C. Santonocito, Andrea Urbani, M. De Bonis, A. Minucci
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="RNA-binding proteins (RBPs) are important regulators of post-transcriptional gene expression. Understanding which and how RBPs promote cancer progression is crucial for cancers that lack effective targeted therapies, such as triple negative breast cancer (TNBC). Here, we employed both in vitro and in vivo pooled CRISPR/Cas9 screening to identify 50 RBP candidates essential for TNBC cell survival. Integrated eCLIP and RNA-sequencing analysis identified that poly(U)-binding splicing factor 60 (PUF60) drives exon inclusion within proliferation-associated transcripts that, when mis-spliced, induce cell cycle arrest and DNA damage. Furthermore, disrupting PUF60 interactions with 3' splice sites via a substitution in its RNA-binding domain caused widespread exon skipping, leading to downregulation of proliferation-associated mRNAs and inducing apoptosis in TNBC cells. Knockdown of PUF60 or disruption of PUF60-RNA interactions inhibited TNBC cell proliferation and shrunk tumor xenografts in multiple models. Together, these findings reveal the molecular mechanism by which PUF60 supports cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9daeb0dc546824319e10c4e8987e741e2cbcfc11" target='_blank'>
              Integrative CRISPR Screening and RNA Analyses Discover an Essential Role for PUF60 Interactions with 3' Splice Sites in Cancer Progression.
              </a>
            </td>
          <td>
            Alexandra T. Tankka, Yuhan Zhang, J. Einstein, Cathy Zhou, Vivian N Pham, Jack T Naritomi, Grady G Nguyen, Amaya N Mendez-Molina, Zhimin Hu, Orel Mizrahi, Mark Perelis, Joseph Sarsam, Frederick E Tan, D. Kaufman, Jing Yang, Corina E Antal, Gene W. Yeo
          </td>
          <td>2025-12-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Although ∼20% of early-onset colorectal cancer (EOCRC, age at diagnosis <50y) cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. EOCRC differs molecularly from late-onset colorectal cancer (LOCRC, age at diagnosis > 70y), with EOCRC tumors more frequently exhibiting high-grade histology, immune-related signatures, and microsatellite instability, while LOCRC is marked by DNA damage and oxidative stress pathways. Although consensus molecular subtypes (CMS) offer a framework for classification, they do not fully explain the rising EOCRC incidence, highlighting the need for integrative multiomic approaches to uncover underlying genetic, epigenetic, and environmental drivers for EOCRC.



 We leveraged genomic, transcriptomic, and clinical data from 1,135 sporadic microsatellite stable colorectal cancer (CRC) patients enrolled in the Total Cancer Care protocol and included in the Oncology Research Information Exchange Network (ORIEN) Avatar program across seven U.S. cancer centers, using standardized protocols for biospecimen collection, sequencing, and data harmonization. Whole exome sequencing (WES) and transcriptomic profiling (RNA-Seq) was conducted using standardized pipelines, followed by normalization and filtering. We characterized biological differences across EOCRC and LOCRC using differential expression, molecular subtyping, immune deconvolution, survival analysis, and integrated pathway analyses combining RNA-seq and WES data.



 Our cohort included 27.8% EOCRC, 53.5% average-onset (50-69y) and 18.9% LOCRC cases. EOCRC patients were more likely to present with rectal tumors (24% vs. 14%), advanced stage (75% vs. 57%), and receive treatments at a higher proportion (radiotherapy: 35% vs. 23%; adjuvant therapy: 51% vs. 37%) compared to LOCRC. No significant differences were observed in common CRC mutations or tumor mutational burden. EOCRC cases were significantly enriched for the mesenchymal CMS4 subtype and depleted in CMS2 and CMS3 (p < 9.46 × 10-6), though 5-year survival did not differ by CMS (p-value=0.23). Notably, we observed that EOCRC cases with CMS2 or CMS4 were more likely to be overweight (BMI >= 25kg/m2) as compared to LOCRC (OR > 2). Transcriptomic analysis identified 328 differentially expressed genes (306 up-regulated and 22 down-regulated in EOCRC); GSEA analysis showed enrichment of Hedgehog and calcium signaling pathways (FDR < 0.1) in EOCRC.



 This comparison of EOCRC and LOCRC cases demonstrates clear differences in CMS subtypes, reveals specific associations with environmental factors, and suggests that calcium channel signaling and hedgehog signaling may play a crucial role in the development and progression of EOCRC compared to LOCRC.



 Sheetal Hardikar, Griffin Caryotakis, David A. Nix, Aaron Atkinson, Jamie Teer, Vaia Florou, Andreana Holowatyj, Michelle L. Churchman, David M. McKean, Phaedra Agius, Bodour Salhia, Ning Jin, Daniel Spakowicz, Micha Cavnar, Emily Baiyee. Toegel, Tiago Biachi de Castria, Patrick M. Boland, Ahmad Tarhini, Bryan P. Schneider, Matthew Reilley, Deepak Vadehra, Michele M. Gage, Howard Colman, Courtney Scaife, Jessica N. Cohan, Biljana Gigic, Adetunji Toriola, Christopher I. Li, Jane Figueiredo, Dorotha Byrd, David Shibata, Cornelia M. Ulrich, Aik Choo Tan, Erin M. Siegel. Transcriptomic and pathway analyses patterns in early-onset and late-onset microsatellite stable colorectal cancer: Results from the ORIEN Network [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: The Rise in Early-Onset Cancers—Knowledge Gaps and Research Opportunities; 2025 Dec 10-13; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(23_Suppl):Abstract nr PR012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef5d723b4ea4c2d0685105fa8ade2e819ad2cc77" target='_blank'>
              Abstract PR012: Transcriptomic and pathway analyses patterns in early-onset and late-onset microsatellite stable colorectal cancer: Results from the ORIEN Network
              </a>
            </td>
          <td>
            S. Hardikar, Griffin R. G. Caryotakis, David Nix, Aaron Atkinson, Jamie K. Teer, V. Florou, A. Holowatyj, M. Churchman, David M. McKean, Phaedra Agius, B. Salhia, N. Jin, Daniel J. Spakowicz, Micha Cavnar, Emily Baiyee. Toegel, Tiago Biachi de Castria, P. Boland, Ahmad A. Tarhini, Bryan P. Schneider, M. Reilley, D. Vadehra, Michele M. Gage, H. Colman, Courtney L. Scaife, J. Cohan, Biljana Gigic, A. Toriola, Christopher I Li, Jane C Figueiredo, Dorotha Byrd, David Shibata, C. Ulrich, Aik Choo Tan, E. Siegel
          </td>
          <td>2025-12-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Human embryonic kidney cells HEK293 are widely used in biopharmaceutical manufacturing, with a recent surge particularly in recombinant adeno-associated virus production. Despite their industrial relevance, comprehensive data on their genomic background and stability remains limited. Here, we systematically analyze the genetic landscape of various HEK293 cell lines in response to cultivation conditions, clonal selection, genetic manipulation and over time in culture. Adherent HEK293 were adapted to suspension growth using different serum-free media. Whole genome sequences from these cell lines were analyzed together with previously published data from additional variants in common use. All data sets were aligned against the human reference genome, enabling the assessment of genome stability by evaluation of variants and revealing a conserved genetic core across all lines, regardless of cultivation history or phenotypic divergence. Evaluation of the functional implications of conserved core mutations identified an enrichment in genes related to cellular structure, morphology and cellular connectivity. The distribution of structural variants and single nucleotide polymorphisms indicated a gradual accumulation of mutations over time in culture rather than abrupt shifts in response to environmental changes. Notably, the integrated adenoviral genes remained highly conserved with respect to copy number, integration site and sequence integrity. These findings provide insight into the genomic evolution of HEK293 cells and offer a foundation for further multi-omics studies aimed at optimizing HEK293 cells for applications in biopharmaceutical production.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8796a2f28d4401b9a025247d57029fc3975fe09" target='_blank'>
              Comparative Analysis of HEK293 Genomic Variability.
              </a>
            </td>
          <td>
            Georg Smesnik, Nikolaus Virgolini, Maria Toth, A. Dürauer, N. Borth
          </td>
          <td>2025-11-10</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a70cf29a8102e7c49d2b80d42e52230a68a64c" target='_blank'>
              Integrative multi-omics reveals PARP14 as a key IFNγ-regulated mediator of metastatic progression in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Sánchez-Serra, M. Chicón-Bosch, Núria Martorell-Villanueva, Eduardo Candeal, Ignasi Jarne-Sanz, J. Bech-Serra, D. Giménez-Llorente, Manuel Gris-Lorente, P. Monaco, L. Santana-Viera, Silvia Mateo-Lozano, Jaume Mora, Mario Niepel, Carolina de la Torre Gomez, Rebeca Sanz-Pamplona, Ana Cuadrado, Ana Losada, R. López-Alemany, Ò. M. Tirado
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Purpose of Review Clonal hematopoiesis (CH) arises from the expansion of a single hematopoietic stem cell harboring somatic mutations that confer growth advantage. Recent studies highlight a substantial heritable component to CH, implicating germline mutations in DNA damage repair (DDR) genes. These genes are essential for maintaining genomic integrity and pathogenic variants in key DDR genes are well-established genetic underpinnings of several hereditary cancer syndromes. This review synthesizes current data linking germline DDR mutations – including ATM, CHEK2, TP53, PPM1D, BRCA1/2, and PARP1 – to CH and the development of myeloid malignancies. Recent Findings Emerging evidence suggests that germline perturbations in DDR pathway contribute to CH, though mechanisms remain incompletely defined. Large scale genome-wide association studies (GWAS) have identified strong associations between ATM and CHEK2 variants and CH. Assessing prevalence and CH risk in individuals with germline TP53 variants presents unique challenges, as distinguishing between somatic and constitutional lesions is often complex and requires careful tissue evaluation. The link between germline BRCA1/2 and CH remains inconclusive, confounded by concurrent diagnosis of solid malignancy and prior exposure to chemoradiation therapy in studied patient populations. Although germline mutations in PPM1D and PARP1 are rare, a potential germline predisposition to CH cannot be excluded. Summary The totality of current evidence suggests that germline DDR pathway mutations not only predispose to well-established solid malignancy syndromes but also to CH, which independently increases the risk of hematologic malignancies. Recognizing germline contributions to CH has broad implications for risk assessment, surveillance strategies, and development of preventive strategies in myeloid neoplasia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7301780c127edf4d26a740d2302af1182d86f99" target='_blank'>
              Impact of Germline DNA Repair Mutations on Clonal Hematopoiesis and Myeloid Neoplasm Development
              </a>
            </td>
          <td>
            Kateryna Fedorov, Leo Y. Luo, Alexander G. Bick, M. Savona
          </td>
          <td>2025-12-01</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related mortality worldwide. This narrative review integrates recent advances in the molecular circuits orchestrating metastatic progression, encompassing epithelial–mesenchymal transition (EMT), organotropism, extracellular matrix remodeling, angiogenesis, hypoxia-inducible signaling, tumor-cell migration modes, and tumor–immune interactions through expert-guided literature selection. We examined therapeutic innovations that disrupt these pathways, including EMT modulators, matrix metalloproteinase inhibitors, VEGF/VEGFR-targeted regimens, hypoxia-activated prodrugs, and next-generation immunotherapies such as immune checkpoint blockade and chimeric antigen receptor T cells. Additionally, we discuss established nanotechnology-based delivery systems, advancing multi-omics integration, evolving single-cell analyses, and emerging CRISPR-Cas9 gene-editing applications as tools for improving metastasis detection, monitoring, and treatment. Despite this progress, translational obstacles persist, particularly regarding intratumoral heterogeneity, adaptive resistance, and limited preclinical model fidelity. Addressing these challenges requires biomarker-guided, multi-target therapeutic combinations, interdisciplinary collaboration, and globally inclusive clinical trials. This evidence underscores the importance of integrated strategies that simultaneously target intrinsic tumor plasticity and microenvironmental support to transform metastatic cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7d3de1a04e69b49fb547b3a441611b4041aca5" target='_blank'>
              Deciphering molecular pathways driving cancer invasion and metastasis: advances and therapeutic prospects
              </a>
            </td>
          <td>
            Aziza Alshahrani, Arwa Alsubait, Z. Asiri, Sahar Alghamdi, Sarah Bin Saqyah, Tariq Alqahtani, Rawan Fitaihi, Njoud Altuwaijri, Yahya F. Jamous
          </td>
          <td>2025-12-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Colorectal cancer (CRC) persists as a significant global health challenge, distinguished by intricate molecular modifications and a notable propensity for resistance to standard therapeutic interventions. Among the regulatory factors contributing to CRC pathogenesis, microRNAs (miRNAs) have emerged as pivotal regulators of gene expression, presenting innovative prospects for diagnostic and therapeutic advancements. Notably, microRNA-143 (miR-143) has attracted considerable attention as a tumor-suppressive miRNA, exhibiting diverse functions in the development, progression, and therapeutic response of CRC. This review delineates an exhaustive examination of the molecular mechanisms by which miR-143 modulates critical oncogenic pathways, encompassing KRAS signaling, epithelial-mesenchymal transition, and metabolic reprogramming. We underscore recent progress in the molecular biosciences that position miR-143 as a promising biomarker for the early detection and prognosis of CRC. Furthermore, we investigate its emergent function in the modulation of sensitivity to chemotherapeutic and targeted therapeutic agents, emphasizing its potential utility in predicting and mitigating drug resistance in CRC cells. By synthesizing contemporary findings within the domains of molecular diagnostics and therapeutic interventions, this review accentuates the clinical potential of targeting miR-143 in the personalized management of CRC and the prediction of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8c35da017b5ecf9023804e3cb13c799f4993a0" target='_blank'>
              Targeting microRNA-143 in colorectal cancer: advances in molecular biosciences for biomarker-based diagnostics, therapeutic strategies, and drug resistance prediction
              </a>
            </td>
          <td>
            Xue Wang, Lei Wang, Long Ke
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Md Shiblee Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage-replication/fusion process explains complex rearrangements and segmental DNA amplification.
              </a>
            </td>
          <td>
            , Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2809e2ddcf9b03a077a182b28f9e8cebc2c7cc" target='_blank'>
              Towards CRISPR-based editing of the mitochondrial genome in yeast
              </a>
            </td>
          <td>
            Sifei Yin, Daniel F. Jarosz, Alice Y. Ting
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Approximately 50% of melanoma patients carry a mutation in the BRAF gene, and over 90% of these mutations lead to the substitution of valine 600 with glutamic acid (V600E). Vemurafenib is an FDA-approved kinase inhibitor for BRAFV600E; while the drug elicits effective remission of metastatic melanoma, relapse typically occurs within several months after therapy. Recent studies documented critical roles of reversible modifications in RNA in modulating resistance to cancer therapy. Herein we explored the contributions of epitranscriptomic alterations to vemurafenib resistance by assessing the differential expression of epitranscriptomic reader, writer and eraser (RWE) proteins in IGR37 metastatic melanoma cells and the isogenic vemurafenib-resistant cells (IGR37xp). Our results revealed altered expressions of multiple epitranscriptomic RWE proteins, including markedly elevated expressions of MTO1 and TRMU─which act sequentially to produce 5-taurinomethyl-2-thiouridine (τm5s2U) at the 34th position of human mitochondrial (mt) tRNAGlu, tRNAGln and tRNALys─in the resistant line. We also observed elevated oxidative phosphorylation in IGR37xp relative to IGR37 cells. Moreover, we found that genetic depletion of TRMU in IGR37xp cells results in diminished oxidative phosphorylation and resensitizes IGR37xp cells to vemurafenib. Together, we uncovered a role of TRMU in conferring vemurafenib resistance in melanoma through modulating oxidative phosphorylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51b3dbd227eca206b5d710e54127d5f9303055" target='_blank'>
              TRMU Confers Resistance of Melanoma Cells to Vemurafenib through Modulating Mitochondrial Activities.
              </a>
            </td>
          <td>
            Shiyuan Guo, Tianyu F Qi, Yinsheng Wang
          </td>
          <td>2025-11-10</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains a major contributor to cancer-related morbidity and mortality among women globally. While systemic therapies have significantly advanced, the emergence of drug resistance continues to hinder durable clinical benefit, leading to treatment failure and disease relapse. Long non-coding RNAs (lncRNAs) have gained recognition as key regulators in BC biology and therapeutic response, with growing evidence implicating them in resistance to chemotherapy, endocrine therapy, targeted agents, and immunotherapy. This review provides a detailed overview of the molecular mechanisms through which specific lncRNAs promote resistance, emphasizing their roles in modulating processes such as programmed cell death, epithelial-mesenchymal transition, and remodeling of the tumor microenvironment. We further highlight common regulatory axes - such as competing endogenous RNA networks and the phosphatidylinositol 3-kinase/protein kinase B pathway - that serve as converging points across distinct resistance mechanisms. In addition, we discuss recent progress in harnessing lncRNAs as liquid biopsy biomarkers and examine the key challenges and opportunities in translating lncRNA-based targets into therapeutic strategies. Together, these insights offer a foundation for future research aimed at overcoming drug resistance in BC through novel lncRNA-centered interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfac8b955e147276484ab116353ca6e5f90be4a2" target='_blank'>
              The role of long non-coding RNAs in breast cancer drug resistance: a comprehensive review
              </a>
            </td>
          <td>
            Yingkun Xu, Jingyu Ma, Changyuan Ding, Weili Liang, Q. Zeng, Bin Lv
          </td>
          <td>2025-12-10</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a5fb32298b06f7826592aac1dafed7fb246a78" target='_blank'>
              Genotype-phenotype correlations and de novo induction of cancer stem cells in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epitranscriptomics, the study of dynamic chemical modifications on RNA mediated by “writers,” “erasers,” and “readers,” has emerged as a pivotal discipline in elucidating the intricate interplay between cancer and immune regulation. These reversible modifications (e.g. m6A, m5C, Ψ) govern RNA metabolism, stability, and translation, thereby exerting spatiotemporal control over immune cell differentiation, activation, and function. Dysregulation of RNA-modifying proteins disrupts immune surveillance, enhances tumor cell survival under stress, and promotes chemoradiotherapy resistance by altering RNA splicing, translation, and stress adaptation pathways. This review summarized the recent progress in the regulatory mechanisms profoundly influencing the tumor microenvironment (TME), modulating immune checkpoints, antigen presentation pathways, and the activity of immune cells. Furthermore, we discussed the therapeutic strategies and challenges in targeting epitranscriptomic regulators and epitranscriptomic editing technologies to enhance anti-tumor immune responses and overcome therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c0776f1a44885afebca84621fac676e8beb717" target='_blank'>
              Epitranscriptomic control of cancer immunity and therapy resistance
              </a>
            </td>
          <td>
            Xingsen Zhao, Suhua Guan
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Precision therapy for glioma remains a major challenge due to tumor heterogeneity. The Origin Recognition Complex Subunit 6 (ORC6) is a crucial regulator of DNA replication initiation. This study aims to investigate the expression of ORC6 in gliomas and its relationship with survival rates and malignancy, while screening potential drugs targeting its functional network. By integrating multiple bioinformatics approaches with structure-based virtual screening, retrospective RNA sequencing data analysis was performed using patients from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. A protein-protein interaction (PPI) network was constructed from ORC6-coexpressed genes to identify core hubs. Molecular docking was employed to screen a library of natural compounds and known drugs against these hub targets. Research has revealed that ORC6 is significantly upregulated in high-grade gliomas, with its elevated expression associated with poor survival outcomes and immune inflammatory responses. Network analysis identified five core hub genes (ORC1, ORC2, MCM2, MCM6, CDC45) central to DNA replication. Molecular docking revealed that several compounds, including the natural flavonoid Baicalein and the FDA-approved drug Palbociclib, exhibited high binding affinity to these hub targets. ORC6 represents a highly promising novel target for precision therapy in glioma. Potential approaches to target this pathway include disrupting the ORC6-replication axis using existing drugs (such as palbociclib) or natural products (such as baicalin).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324196dc2699eff3ae974af550ed77475be4c5ee" target='_blank'>
              High ORC6 expression is a prognostic indicator of poor survival in glioma patients
              </a>
            </td>
          <td>
            Mengjie Wang, Song Feng, Chen Zhang, Feng Jin
          </td>
          <td>2025-11-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8fae2bcd154582c2189d775e6e129baa231cb3" target='_blank'>
              A multi-omics analysis of human fibroblasts overexpressing an Alu transposon reveals widespread disruptions in aging-associated pathways.
              </a>
            </td>
          <td>
            Juan I. Bravo, Eyael Tewelde, Christina D. King, J. Bons, Samah Shah, Jacob Rose, Judith Campisi, Birgit Schilling, B. Benayoun
          </td>
          <td>2025-12-11</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Monogenic diabetes represents approximately 1-5% of all diabetes cases globally, with mutations in over 40 genes causing various forms of maturity onset diabetes of the young (MODY) and neonatal diabetes mellitus. The emergence of clustered regularly interspaced short palindromic repeats (CRISPR) technology has revolutionized therapeutic approaches for genetic disorders, offering unprecedented precision in correcting disease-causing mutations. This review examined the current state of CRISPR based gene editing applications for monogenic diabetes, analyzing therapeutic strategies, molecular mechanisms, and clinical translation challenges. A comprehensive literature search was conducted using PubMed and Web of Science databases from 2012-2025, focusing on peer reviewed articles describing CRISPR applications in monogenic diabetes models and clinical studies. Recent advances demonstrated successful correction of pathogenic variants in HNF1A, HNF4A, INS, and KCNJ11 genes using base editing and prime editing technologies, with correction efficiencies ranging from 15-85% in cellular models. Clinical applications showed promise for treating MODY subtypes, with patient derived induced pluripotent stem cells (iPSCs) serving as valuable platforms for personalized therapy development. Current limitations include delivery challenges, off target effects, and regulatory considerations for germline editing. CRISPR based approaches represent a paradigm shift toward precision medicine for monogenic diabetes, with base editing and prime editing emerging as safer alternatives to traditional nuclease-based systems for therapeutic applications.

Keywords: CRISPR gene editing, Monogenic diabetes, Precision medicine, Base editing, Prime editing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13ddf5815b52ccaf46c35e119de2106b2f61ca78" target='_blank'>
              CRISPR-Based Gene Editing Approaches for Monogenic Diabetes Therapy and Precision Medicine
              </a>
            </td>
          <td>
            Namirimu Sandrah
          </td>
          <td>2025-11-26</td>
          <td>IDOSR JOURNAL OF APPLIED SCIENCES</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acbdbd54a52a897e14acda45d5d2ae5113465461" target='_blank'>
              Genomic characterization of colorectal tumors: insights into significantly mutated genes, pathways, and survival outcomes.
              </a>
            </td>
          <td>
            T. Harrison, Syed H Zaidi, Hang Yin, R. Steinfelder, C. Qu, E. Aglago, Sonja I. Berndt, Lisa A. Boardman, Hermann Brenner, Daniel D. Buchanan, P. Campbell, Yin Cao, A. T. Chan, S. Chanock, K. Doheny, David A. Drew, J. C. Figueiredo, Amy J. French, S. Gallinger, P. Georgeson, M. Giannakis, Ellen L. Goode, S. Gruber, A. Gsur, Marc J. Gunter, Sophia Harlid, M. Hoffmeister, Wen-Yi Huang, Meredith A J Hullar, J. R. Huyghe, Mark A Jenkins, Yi Lin, Victor Moreno, Neil Murphy, Polly A Newcomb, Christina C. Newton, J. Nowak, M. Obón-Santacana, Shuji Ogino, T. Shelford, Mingyang Song, Claire E. Thomas, A. Toland, T. Ugai, Caroline Y Um, B. van Guelpen, Quang M Trinh, Wei Sun, T. Hudson, L. Hsu, Ulrike Peters, Amanda I. Phipps
          </td>
          <td>2025-12-18</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d049dde01e4ae93dabdffce09d14b8271cee14ed" target='_blank'>
              Emerging Anti-Cancer and Repurposed Therapies for Overcoming Multidrug Resistance in Lung Cancer.
              </a>
            </td>
          <td>
            N. Solanki, Pratham Shah, Sargam Kewalramani, Umang Shah, Mehul Patel, Swayamprakash Patel, Rajesh Maheshwari
          </td>
          <td>2025-12-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome instability (CIN) remains among the most important problems in modern cancer research. In this study, we conducted a genome-wide RNAi screen to identify genes that contribute to CIN. To achieve this, we used a human artificial chromosome in a novel sensitized screen to measure CIN. We screened 18,658 genes for their roles in maintaining chromosomal stability and identified 834 candidates as potential CIN genes. A secondary RNAi screen identified 44 genes with the most pronounced CIN phenotypes. In guilt-by-association analysis using a published set of 8,498 proteins across a panel of 949 cancer cell lines, this cohort of 44 genes displayed a striking correlation with mitotic regulators. Furthermore, altered expression of these proteins was associated with a poor prognosis across multiple cancer types. Specifically, downregulation of AMY2B, ALAD, PDGFRA, PPIE, VEZ1, and TTC19 is associated with poor survival in two or more of the following: small cell lung cancer, lung adenocarcinoma, adrenocortical carcinoma, ovarian cancer, and breast cancer. The genes identified in this screen hold potential as prognostic markers for patient survival across several cancer types and could potentially serve as targets for the development of new therapeutic approaches aimed at mitigating CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48402b9da158c79dbc9aec979b9f8a854d30f0b3" target='_blank'>
              A genome-wide RNAi screen for novel CIN genes using human artificial chromosome.
              </a>
            </td>
          <td>
            Mikhail Liskovykh, N. Y. Kochanova, Chih-Yuan Chiang, A. Dhall, V. Aksenova, Yu-Chi Chen, William C Reinhold, M. Dasso, Anish Thomas, K. Cheng, Y. Pommier, William C. Earnshaw, Vladimir Larionov, N. Kouprina
          </td>
          <td>2025-12-01</td>
          <td>PNAS nexus</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The process of aging is fundamentally driven by genomic instability and the accumulation of DNA damage, which progressively impair cellular and tissue function. In order to counteract these challenges, cells rely on the DNA damage response (DDR), a multilayered signaling and repair network that preserves genomic integrity and sustains homeostasis. Within this framework, nucleases and helicases have pivotal and complementary roles by remodeling aberrant DNA structures, generating accessible repair intermediates, and determining whether a cell achieves faithful repair, undergoes apoptosis, or enters senescence. Defects in these enzymes are exemplified in human progeroid syndromes, where inherited mutations lead to premature aging phenotypes. This phenomenon is also replicated in genetically engineered mouse models that exhibit tissue degeneration, stem cell exhaustion, and metabolic dysfunction. Beyond their canonical repair functions, helicases and nucleases also interface with the epigenome, as DNA damage-induced chromatin remodeling alters enzyme accessibility, disrupts transcriptional regulation, and drives progressive epigenetic drift and chronic inflammatory signaling. Moreover, their dysfunction accelerates the exhaustion of adult stem cell populations, such as hematopoietic, neural, and mesenchymal stem cells. As a result, tissue regeneration is undermined, establishing a self-perpetuating cycle of senescence, impaired repair, and organismal aging. Current research is focused on developing therapeutic strategies that target the DDR–aging axis on several fronts: by directly modulating repair pathways, by regulating the downstream consequences of senescence, or by preventing DNA damage from accumulating upstream. Taken together, evidence from human disease, animal models, molecular studies, and pharmacological interventions demonstrates that nucleases and helicases are not only essential for genome maintenance but also decisive in shaping aging trajectories. This provides valuable knowledge into how molecular repair pathways influence organismal longevity and age-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9382a44469cc78462704e76a62b5f9104801fc" target='_blank'>
              The Converging Roles of Nucleases and Helicases in Genome Maintenance and the Aging Process
              </a>
            </td>
          <td>
            Aikaterini Margariti, Persefoni Daniil, T. Rampias
          </td>
          <td>2025-11-01</td>
          <td>Life</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous loci linked to late-onset Alzheimer’s disease (LOAD), but the pan-brain regional effects of these loci remain largely uncharacterized. To address this, we systematically analyzed all LOAD-associated regions reported by Bellenguez et al. using the FILER functional genomics catalog across 174 datasets, including enhancers, transcription factors, and quantitative trait loci. We identified 41 candidate causal variant-effector gene pairs and assessed their impact using enhancer–promoter interaction data, variant annotations, and brain cell-type-specific gene expression. Notably, the LOAD risk allele of rs74504435 at the SEC61G locus was computationally predicted to increase EGFR expression in LOAD-related cell types: microglia, astrocytes, and neurons. Functional validation using promoter-focused Capture C, ATAC-seq, and CRISPR interference in the HMC3 human microglia cell line confirmed this regulatory relationship. Our findings reveal a microglial enhancer regulating EGFR in LOAD, suggesting EGFR inhibitors as a potential therapeutic avenue for the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d07f394fd7eb383df0b9014ab8e1eef1a6183ec" target='_blank'>
              Integrated genomic analysis and CRISPRi implicates EGFR in Alzheimer’s disease risk
              </a>
            </td>
          <td>
            Y. Leung, Pavel P. Kuksa, Luke Carter, Jeffrey Cifello, Emily Greenfest-Allen, O. Valladares, Louisa Boateng, Shannon Laub, Natalia Tulina, Sofia Moura, Aura Maria Ramirez, K. Celis, Fulai Jin, R. Feng, Gao Wang, Phil De Jager, Jeff Vance, Liyong Wang, Struan F. A. Grant, Gerard D. Schellenberg, A. Chesi, Li-San Wang
          </td>
          <td>2025-12-16</td>
          <td>Npj Dementia</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Kinesin family member 4 A (KIF4A) is a multifunctional motor protein essential for chromosome condensation, spindle dynamics, and cytokinesis. Beyond its classical mitotic functions, emerging evidence positions KIF4A as a central regulator of tumorigenesis, therapy resistance, metabolic reprogramming, and immune modulation across diverse cancer types. However, no comprehensive review has integrated its molecular mechanisms with its roles in both oncological and non-oncological diseases, nor clarified its context-dependent behavior, including paradoxical tumor-suppressive effects in cervical cancer. In this review, we synthesize current advances spanning structural biology, transcriptional and post-translational regulation, and pathway-level interactions involving PI3K/AKT, TGF-β/Smad, Hippo-YAP, metabolic remodeling, and DNA damage response networks. We summarize KIF4A’s expression and functions across more than 30 malignant tumors and multiple non-neoplastic conditions—including neurodevelopmental disorders, autoimmune diseases, viral infections, fibrotic diseases, and congenital anomalies—highlighting shared molecular themes and disease-specific distinctions. A notable finding is KIF4A’s context dependency: while generally oncogenic, high KIF4A expression in cervical cancer correlates with improved survival, suggesting HPV-specific transcriptional rewiring, altered phosphorylation states, or compensatory genome stabilization as potential mechanisms.We further evaluate the translational implications of KIF4A as a biomarker for diagnosis, prognosis, and treatment response, and we critically examine therapeutic strategies targeting KIF4A—ranging from small-molecule inhibitors and gene-silencing approaches to miRNA therapeutics, exosome-based delivery systems, and neoantigen-directed immunotherapy. Finally, we outline major challenges to clinical translation, including its essential roles in mitosis and neuronal integrity, the need for tumor-selective delivery platforms, and incomplete understanding of its tissue-specific functions. Collectively, this review provides a unified mechanistic and translational framework for understanding KIF4A across human diseases, identifies key knowledge gaps, and proposes future research directions to enable safe and effective targeting of this biologically indispensable protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/443b5589fedb44be8b5987c370d8852efc1fee88" target='_blank'>
              KIF4A in disease pathogenesis and therapeutics: from molecular mechanisms to clinical translation
              </a>
            </td>
          <td>
            Yi Liu, Yunhua Li, Chunrong Tang, Honghua Wen, Jingxian Tang, Gangwen Chen, Yongkang Wu
          </td>
          <td>2025-12-08</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian cancer is the gynecologic malignancy that bears the highest mortality rate in the Western world. This is attributed to late diagnosis and limited therapeutic progress. Recent advances in molecular oncology have highlighted the pivotal role of epigenetic modifications—including DNA methylation, histone modifications, non-coding RNAs, chromatin remodeling, and RNA methylation—in ovarian cancer development, progression, and treatment resistance. DNA methylation patterns affect key tumor suppressors and oncogenes, while histone modifications alter chromatin accessibility, influencing gene expression. Chromatin remodeling complexes, particularly the SWI/SNF complex, are frequently mutated in specific ovarian cancer subtypes, which is central in shaping their biological behavior. Non-coding RNAs, including microRNAs and long non-coding RNAs, further regulate tumor cell behavior and the immunosuppressive tumor microenvironment. Epigenetic profiles vary among histological subtypes and hold promise for biomarker development, early detection, prognosis, and therapeutic monitoring. Liquid biopsy approaches leveraging circulating tumor DNA methylation show diagnostic potential superior to conventional markers. Moreover, targeting epigenetic regulators—such as DNMT and HDAC inhibitors, EZH2 antagonists, and RNA-modifying enzymes—offers novel avenues for treatment, particularly in reversing chemoresistance and sensitizing tumors to immunotherapy. While promising, these strategies require further validation through clinical research to translate into effective clinical interventions. This review aims to summarize the current literature and highlights potential applications of epigenetic manipulation in day-to-day practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e682da74ce847ab8714dc4327d4b867b8d126a5" target='_blank'>
              Epigenetics in Ovarian Cancer: A Review of Current Knowledge and Future Perspectives
              </a>
            </td>
          <td>
            Nikolaos Dedes, Michael Liontos, Dimitrios Haidopoulos, Flora Zagouri, K. Angelou, A. Svarna, A. Michas, Aikaterini Aravantinou Fatorou, Angeliki Andrikopoulou, Meletios A. Dimopoulos
          </td>
          <td>2025-11-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions — metabolism, epigenetics, immunology, and proteostasis — to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs), commonly referred to as “jumping genes,” are discrete DNA fragments capable of mobilization within the genome, often creating new copies during transposition. Once considered non-functional “junk DNA,” TEs are now recognized as essential regulators of genome evolution, gene networks, and epigenetic processes. They contribute to genome innovation through gene duplication, exon shuffling, regulatory rewiring, and stress-responsive plasticity. Conversely, their uncontrolled activation can disrupt genetic integrity. Such aberrant TE activity has been associated with immune dysfunction, cancer, neurodegeneration, and reproductive disorders. 
In livestock, recent studies have identified TE-derived regulatory elements that influence growth, reproduction, immunity, and adaptation. Notably, TEs constitute approximately 45–60% of the genomes of major livestock species, including pigs, sheep, and cattle, highlighting their extensive regulatory impact. Advances in genome editing and artificial intelligence have facilitated their functional characterization. This allows a sharper focus on TE-driven regulation in animal genomes without repeatedly specifying the livestock context. 
Emerging technologies such as CRISPR–transposon systems, programmable epigenome editing, and AI-driven TE mapping offer new opportunities for precise genome engineering, functional genomics, and livestock breeding. This review consolidates current knowledge on TE classification, molecular mechanisms, evolutionary roles, disease associations, and biotechnological  applications, with emphasis on livestock genomics. It also outlines existing challenges, biosafety considerations, and future directions toward harnessing TEs for precision animal breeding and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434f9971fc3a63cead1542771f75e88db7b4209d" target='_blank'>
              Jumping Genes (Transposable Elements): A Comprehensive Review of Their Molecular Mechanisms, Evolutionary Impact and Applications in Livestock and Biomedicine
              </a>
            </td>
          <td>
            Dhruv Bhatia, Prakashji Thakor, Pankti J Naik, Brijal Barot
          </td>
          <td>2025-12-22</td>
          <td>Journal of Experimental Agriculture International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lineage plasticity is a cancer hallmark that drives disease progression and treatment resistance1,2. Plasticity is often mediated by epigenetic mechanisms that may be reversible; however, there are few examples of such reversibility. In castration-resistant prostate cancer (CRPC), plasticity mediates resistance to androgen receptor (AR) inhibitors and progression from adenocarcinoma to aggressive subtypes, including neuroendocrine prostate cancer (CRPC-NE)3–5. Here we show that plasticity-associated treatment resistance in CRPC can be reversed through the inhibition of NSD2, a histone methyltransferase6. NSD2 upregulation in CRPC-NE correlates with poor survival outcomes, and NSD2-mediated H3K36 dimethylation regulates enhancers of genes associated with neuroendocrine differentiation. In prostate tumour organoids established from genetically engineered mice7 that recapitulate the transdifferentiation to neuroendocrine states, and in human CRPC-NE organoids, CRISPR-mediated targeting of NSD2 reverts CRPC-NE to adenocarcinoma phenotypes. Moreover, a canonical AR program is upregulated and responses to the AR inhibitor enzalutamide are restored. Pharmacological inhibition of NSD2 with a first-in-class small molecule reverses plasticity and synergizes with enzalutamide to suppress growth and promote cell death in human patient-derived organoids of multiple CRPC subtypes in culture and in xenografts. Co-targeting of NSD2 and AR may represent a new therapeutic strategy for lethal forms of CRPC that are currently recalcitrant to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31a72e85b6954f413f71412e13d4f1a6449106e" target='_blank'>
              NSD2 targeting reverses plasticity and drug resistance in prostate cancer
              </a>
            </td>
          <td>
            Jia J Li, A. Vasciaveo, Dimitris Karagiannis, Zhen Sun, Kristjan H. Gretarsson, Xiao Chen, O. Ouerfelli, F. Socciarelli, Ziv Frankenstein, Hanyang Dong, Min Zou, W. Yuan, Guangli Yang, Gabriel M Aizenman, Tania Pannellini, Xinjing Xu, H. Beltran, Yu Chen, Kevin L Gardner, Brian D. Robinson, J. D. De Bono, O. Gozani, Cory Abate-Shen, Mark A. Rubin, M. Loda, Charles L. Sawyers, Andrea Califano, Chao Lu, Michael M. Shen
          </td>
          <td>2025-11-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Mosaic loss of the Y chromosome (LOY) is a somatic, age‐related phenomenon associated with an increased risk of developing Alzheimer's disease (AD) by our group and others. While the underlying biological mechanism driving this risk remains unknown, genome wide association studies (GWAS) of LOY outcomes identified ∼150 loci involved in cell cycle regulation, mitotic pathways, and DNA damage response. These variants also predict X chromosome loss in women, suggesting that LOY is a marker of genomic instability in males that precedes the AD clinical phenotype. Method To understand the mechanism by which LOY may contribute to AD risk, we sought to identify both affected and neurologically normal individuals with LOY from genotyping data using the MADloy program. We reprogrammed peripheral blood mononuclear cells (PBMCs) from three of these individuals and used the CytoTune iPS 2.0 Sendai Reprogramming Kit to create induced pluripotent stem cells (iPSCs) lines. Result 1,068 males with genotyping data were screened for loss of the Y chromosome; 41 (3.83%) had detectable LOY in the blood. As expected, LOY was positively correlated with the age of the individual at the time of sample collection. Three LOY samples (2 cognitively normal individuals and 1 individual with Alzheimer's disease) were chosen for iPSC reprogramming. Samples were independently shown to have LOY through either CNV TaqMan assays or whole genome sequencing data analyzed via the Control‐FREEC software. iPSC reprogramming was performed using Sendai virus. Clones were identified from each sample that either carried or were missing the Y chromosome, thereby generating isogenic pairs of cell lines. Conclusion We have generated paired iPSC cell lines derived from AD and cognitively normal individuals that allow for explicit examination of the impact of LOY on AD‐relevant cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fdfe3fdcfff07d9fa3235623a05b3a308822589" target='_blank'>
              Generation of Isogenic Induced Pluripotent Stem Cell Lines Derived From Individuals Mosaic for Loss of the Y Chromosome
              </a>
            </td>
          <td>
            H. N. Cukier, Yeunjoo E. Song, Lauren E. Coombs, Eugene Tang, Tianjie Gu, Brooke A. DeRosa, Derek M. Dykxhoorn, Jonathan L. Haines, M. Pericak-Vance, William S Bush, Anthony J Griswold
          </td>
          <td>2025-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent efforts to construct disease-specific multimodal omics databases at single-cell resolution, along with advances in reconstructive technologies such as brain organoids, have opened up opportunities to elucidate the molecular basis of complex human neuropsychiatric diseases. In this study, we performed a meta-analysis to characterize disease-associated regulatory modules by performing single-cell transcriptome analysis of developing neurons from reciprocal human ESC models of CNV in the distal 1q21.1 region. As a result, we observed significant directional enrichment of a series of genes in neuronal cells associated with autism spectrum disorder (ASD), bipolar disorder, and schizophrenia. Correlation analyses revealed that the disease-associated signature primarily targeted epigenetic regulatory mechanisms. We also identified Bromodomain PHD Finger Transcription Factor (BPTF), a key component of the NURF chromatin remodeling complex, as a potential target responsible for transcriptome changes related to human neuropsychiatric diseases, including ASD. We provide a practical and straightforward analytical workflow for utilizing both public data and in-house single-cell omics data from disease models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7d367adb9e191d1bc2c8d90aaed35322e32690" target='_blank'>
              Single-Cell Transcriptome Meta-Analysis Reveals Epigenomic and Chromatin Dysregulation in Developing Neurons Derived From Human ESCs With 1q21.1 CNVs.
              </a>
            </td>
          <td>
            Kosuke Torigata, J. Nomura, Toru Takumi
          </td>
          <td>2025-12-24</td>
          <td>Autism research : official journal of the International Society for Autism Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetic modifications such as DNA methylation play a fundamental role in oncogenesis and the progression of neoplasms neoplasias. DNA methyltransferase inhibitors (DNMTi) constitute a family of therapeutic agents that impede the methylation at the 5‐position on cytosine nucleotides, thereby modulating the epigenetic regulation of tumor suppressor genes, oncogenes, and other key regulatory genes. The first‐generation DNMTi azacitidine and decitabine have demonstrated substantial efficacy in the treatment of medically non‐fit, older patients with acute myeloid leukemia (AML) or myelodysplastic syndrome (MDS) ineligible for intensive chemotherapy (IC), by virtue of their favorable safety profile. Despite these clinical achievements, however, single‐agent DNMTi treatment has faced challenges such as limited, non‐durable response rates and remissions as well as the emergence of secondary resistance. These limitations have driven broad efforts to identify more effective, dual treatment combinations, such as now attained with the DNMTi‐BCL‐2 (B‐cell lymphoma 2) inhibitor combination. This review aims to provide a comprehensive overview and analysis of the pivotal role of DNMTi in both mono‐ and combination therapies for myeloid malignancies over the last 40 years, while also exploring their potential applicability in lymphoid malignancies. Additionally, this review assesses the therapeutic potential of DNMTi in the management of solid tumors. Through these discussions, we intend to enhance the understanding of the mechanistic and therapeutic implications of DNMTi across a diverse array of malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bcdee98f9dc8853fe622239d843945349566587" target='_blank'>
              DNA methyltransferase inhibitors in hematological malignancies and solid tumors
              </a>
            </td>
          <td>
            Valentin Wenger, G. Garcia-Manero, R. Zeiser, Michael Lübbert
          </td>
          <td>2025-11-07</td>
          <td>International Journal of Cancer</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Introduction Transfer RNA-derived small RNAs (tsRNAs) have emerged as critical regulators in cancer biology, influencing gene expression, protein synthesis, and cellular signaling. Their unique expression patterns and functional diversity highlight their potential for diagnostic, prognostic, and therapeutic potential, particularly in hepatocellular carcinoma (HCC), a major cause of cancer-related mortality worldwide. Areas covered This review provides the first integrative overview of tsRNAs in HCC, encompassing their biogenesis, classification, molecular functions, and involvement in tumor hallmarks, including proliferation, metastasis, apoptosis, metabolism, angiogenesis, and immune modulation. We also summarize current advances in detection methods and databases and highlight the translational potential of tsRNAs as diagnostic, prognostic, and therapeutic targets. This work emphasizes unexplored dimensions of tsRNA-mediated regulation, connecting mechanistic insights with clinical applications. Expert opinion Although tsRNAs show great promise in HCC diagnosis, prognosis, and therapy, clinical translation remains hindered by gaps in mechanistic understanding, technical challenges in detection, and a lack of large-scale validation. Overcoming these limitations through standardized methodologies and multi-omics integration could unlock their full potential in precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad865e85ec3c0a41a64171ec5b4682d2f6b506a" target='_blank'>
              Emerging role of transfer RNA-derived small RNAs (tsRNAs) in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Abdelhamid M. Abdelhamid, Tasneem Abaza, Waleed T Kotb, Adham H Mohamed, Mohamed Z. Gad, Ayman M Metwally, Cédric Coulouarn, R. A. Youness
          </td>
          <td>2025-12-02</td>
          <td>Expert Review of Molecular Diagnostics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Advanced prostate cancer (PrCa) remains a leading cause of cancer-related death among men due to disease progression in nearly all patients on standard of care therapy targeting the androgen receptor. An important mechanism driving therapeutic resistance is lineage plasticity, which enables PrCa cells to reprogram into lineage variants no longer dependent on androgen receptor signaling. As inhibitors of the histone methyltransferase EZH2 are being evaluated clinically for the treatment of advanced PrCa, we investigated here how EZH2 affects PrCa lineage plasticity. Data from genetically engineered mice and human clinical samples demonstrated that genetic or pharmacological suppression of EZH2 altered chromatin to expand active transcription factor programs. These changes in gene expression during PrCa progression increased the diversity of PrCa lineage variants that arose. EZH2 suppression did not inhibit disease progression or therapeutic resistance in this context. These findings advance the current understanding of PrCa lineage plasticity and suggest EZH2 inhibitors may be less effective in treating PrCa prone to lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee66b00f883047cd2ed8c292d9f0f040e78b6ffe" target='_blank'>
              EZH2 Suppression Diversifies Prostate Cancer Lineage Variant Evolution and Lacks Efficacy in Inhibiting Disease Progression.
              </a>
            </td>
          <td>
            Justine J Jacobi, K. Wadosky, Neha Jaiswal, Xiaojing Zhang, Yanqing Wang, Prashant K Singh, Jie Wang, Eduardo Cortes Gomez, Jianmin Wang, Bo Xu, Mark D Long, Paloma Cejas, Shweta Kukreja, H. Long, David W. Goodrich
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Persistent fetal gene expression in childhood neoplasms is usually explained by a maturation block originating in the prenatal phase. In contrast, reactivation of fetal genes in adult malignancies is considered a consequence of oncofetal reprogramming (OFR) and is associated with aggressive disease. By reconstructing epigenetic ontogeny in juvenile myelomonocytic leukemia (JMML), we identified a postnatal maturation state of JMML stem cells with high transcriptional plasticity indicative of OFR in high-risk disease. Similarly, post-natal activation of oncogenic signaling by inducible Ptpn11E76K mutation in mice, triggered molecular plasticity and reactivation of fetal gene expression. Integrative multi-omics analysis revealed aberrant CD52 expression as a feature of high-risk JMML stem cells. Anti-CD52 treatment depleted JMML stem cells and blocked disease propagation in xenograft models. Our results challenge the prevailing maturation-block model of pediatric leukemogenesis and establish RAS-associated stem-cell plasticity as a determinant of OFR and potential therapeutic vulnerabilities in high-risk JMML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c603c5b4c0adc417c6958428e24696c22fabe88d" target='_blank'>
              Molecular Plasticity Results in Oncofetal Reprogramming and Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia.
              </a>
            </td>
          <td>
            M. Hartmann, M. Schönung, J. Rajak, Valentin J. Maurer, Ling Hai, K. Bauer, M. Hakobyan, Sina Stäble, Jens Langstein, Laura Jardine, R. Roelz, S. Bohler, Eleonora Khabirova, Abdul-Habib Maag, Dominik Vonficht, D. Lebrecht, K. Bernt, Kai Tan, Changya Chen, Fatemeh Alikarami, Julia A. Meyer, Jun Wang, Tobias Boch, Viktoria Flore, P. Lutsik, M. Milsom, S. Raffel, Christian Buske, Simon Haas, M. Haniffa, Jan-Philipp Mallm, S. Behjati, M. Bonder, Stefan Frohling, Elliot Stieglitz, Charlotte M. Niemeyer, Joschka Hey, C. Flotho, C. Plass, M. Erlacher, Matthias Schlesner, D. Lipka
          </td>
          <td>2025-12-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Cancer reversion therapy represents a paradigm shift in oncology, focusing on reprogramming malignant cells to a non-malignant state rather than destroying them. This narrative review synthesizes current evidence, emerging technologies, and future directions in this promising field. Cancer reversion is founded on key biological observations: somatic cell reprogramming, spontaneous cancer regression, and microenvironmental influences on malignant behavior. Current approaches include epigenetic reprogramming using HDAC inhibitors and DNA methyltransferase inhibitors; microenvironmental modulation through extracellular matrix manipulation and vascular normalization; differentiation therapy exemplified by all-trans retinoic acid in acute promyelocytic leukemia; and targeting oncogene addiction as demonstrated in BCR-ABL-driven leukemias. Emerging technologies accelerating progress include single-cell analyses that reveal cancer heterogeneity and cellular state transitions; CRISPR-based approaches enabling precise genetic and epigenetic manipulation; patient-derived organoids that model tumor complexity; and artificial intelligence applications that identify novel reversion-inducing agents. Critical evaluation reveals that many reported “reversion” phenomena represent stimulus-dependent plasticity or transient growth arrest rather than stable phenotypic normalization. True cancer reversion requires durable, heritable phenotypic changes that persist after treatment withdrawal, with evidence of epigenetic consolidation and functional restoration. Despite promising advances, significant challenges remain: cancer cell plasticity facilitating therapeutic escape, difficulties in establishing stable reversion states, delivery challenges for solid tumors, and the need for combination approaches to address tumor heterogeneity. Future directions include integrated multi-omics analyses to comprehensively map cellular state transitions, studies of natural regression phenomena to identify reversion mechanisms, advanced nanodelivery systems for targeted therapy, and synthetic biology approaches creating intelligent therapeutic systems. By redirecting rather than destroying cancer cells, reversion therapy offers the potential for reduced toxicity and resistance, potentially transforming cancer from a deadly disease to a manageable condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ef16378f672c1c6324bdae132a2624ad963a92" target='_blank'>
              Cancer Reversion Therapy: Prospects, Progress and Future Directions
              </a>
            </td>
          <td>
            Emmanuel O. Oisakede, D. Olawade, Oluwakemi Jumoke Bello, Claret Chinenyenwa Analikwu, Eghosasere Egbon, Oluwaseun Fapohunda, S. Boussios
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb26e8cb8a062b2f863229f4de1f7a20095c8169" target='_blank'>
              The role of m6A modification in non-small cell lung cancer: functional insights and impact on therapy resistance.
              </a>
            </td>
          <td>
            Yufei Yang, Yifan Shi, Ziqi Tao, Yingzi Ma, Jinliang Zhao
          </td>
          <td>2025-12-29</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies.
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints.
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, M. Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin M. Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="During a lifetime, normal cells accumulate thousands of changes in their genome sequence. These changes, termed somatic mutations, have mostly been studied in the context of cancer, but their presence in normal tissues is ubiquitous and widespread. Somatic mutation accompanies the aging process and is influenced by genetic and environmental factors. Differently from gene expression or imaging data, which fluctuate over time, somatic variants are non-reversible marks in the genome and accumulate over time. This property can be exploited to track the history of a cell, from conception to old age, providing information that cannot be acquired via classical histological tissue inspection nor other types of omics data. Mutations can track embryonic development, measure how clones compete in a tissue over time, or report the mutational processes active in cells and tissues throughout life. We discuss selected examples and emphasize how somatic mutation analysis can enable expanding applications at the service of physiology and cell biology, as well as a deeper understanding of the aging process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e1754e49deb7eef5ac28d0fb31ba15c272ffd93" target='_blank'>
              Reconstructing the lifelong history of cells and tissues via somatic mutation analysis
              </a>
            </td>
          <td>
            Sipontina Faienza, J. P. Margaria, Irene Franco
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffc85985364a8e0c2b935cb6a150df86f47c5ef" target='_blank'>
              Loss of ELF2 drives topotecan resistance in retinoblastoma revealed by genome-wide CRISPR-Cas9 screening.
              </a>
            </td>
          <td>
            Jingyi Jiang, Zihua Jiang, Qian Luo, Xi Chen, Jiejie Zhuang, Jiaxin Chen, Qingqing Mu, Jin Qiu, Yan Li, Shuxia Chen, Ping Zhang, Keming Yu, Shuilian Chen, Guei-Sheung Liu, Zhuang Jing
          </td>
          <td>2025-12-23</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b50c04873ca9e1681d29cd5b99d8fd8703831da" target='_blank'>
              XC-ID: De novo identification of the active X chromosome in single-cell RNA-seq
              </a>
            </td>
          <td>
            Jennifer Li Hui Jiang, Jesse Gillis
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
 Synthetic messenger RNA (mRNA) vaccines have raised concerns regarding prolonged spike protein expression, immune activation, and potential off-target effects.
 AIM
 To investigate transcriptomic alterations in individuals with new-onset adverse events or cancer following mRNA coronavirus disease 2019 vaccination.
 METHODS
 Bulk RNA sequencing was performed on peripheral blood from two patient groups: (1) Individuals with new-onset nonmalignant adverse events; and (2) Individuals newly diagnosed with cancer post-vaccination. A control group of normal individuals was used for comparison. Differential gene expression was analyzed using DESeq2, and Gene Set Enrichment Analysis was conducted using the MSigDB database and custom gene sets.
 RESULTS
 Both vaccine patient groups displayed widespread transcriptional dysregulation. In the nonmalignant adverse event group, hallmark enrichments included mitochondrial dysfunction, proteasome-mediated stress, transcriptomic instability, and systemic inflammation. The cancer group exhibited additional hallmarks of genomic instability and epigenetic reprogramming. Nonsense-mediated decay, ribosomal stress, and myelocytomatosis oncogene activation were prominent in both groups, while immune signaling via toll-like receptors and type I interferons was particularly elevated in cancer patients. The observed transcriptomic profiles indicate cellular stress responses, mitochondrial dysfunction, and immune dysregulation following exposure to mRNA vaccines, potentially in susceptible individuals.
 CONCLUSION
 Shared and distinct molecular signatures in both cohorts demonstrate underlying mechanisms contributing to post-vaccine symptomatology and complications, including oncogenesis and or progression of malignant disease. These findings underscore the need for a deeper investigation into the long-term safety of mRNA vaccines and host response variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a28e301bd6430c7a6e743e763b659faf3780026d" target='_blank'>
              Synthetic messenger RNA vaccines and transcriptomic dysregulation: Evidence from new-onset adverse events and cancers post-vaccination
              </a>
            </td>
          <td>
            Natalia L. von Ranke, Wei Zhang, Philipp Anokhin, Nicolas Hulscher, Kevin McKernan, Peter Mccullough, J. Catanzaro
          </td>
          <td>2025-12-20</td>
          <td>World Journal of Experimental Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA damage and repair are central to the onset of cancer, aging, and aging-related diseases. Rare genetic defects in the nucleotide excision repair pathway, such as those causing the cancer-prone disorder xeroderma pigmentosum (XP) or the progeroid condition Cockayne syndrome, highlight the dramatic consequences of unrepaired DNA lesions. In this issue of the JCI, two related papers from Ogi and coworkers — Fassihi et al. and Nakazawa et al. — describe a new XP clinical entity, XP-J, linked to a pathogenic variant in the p52 subunit of the transcription-repair complex TFIIH. The studies’ characterization of XP-J and the p52ΔC variant opened unexpected possibilities to ameliorate the molecular defect in another subunit of TFIIH that causes a different, more severe repair syndrome: trichothiodystrophy. This commentary provides a broader historical, medical, and molecular context for the intricate genotype-phenotype relationship between compromised repair and its clinical consequences and discusses next steps for the advances reported.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47309874361d974d2143fc65fd785375287e5c39" target='_blank'>
              Expanding the landscape of nucleotide excision repair disorders: from discovery to therapy
              </a>
            </td>
          <td>
            A. Theil, J. H. J. Hoeijmakers
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Neuroblastoma is an embryonal tumour that arises from the malignant transformation of neural crest cells and remains one of the deadliest malignancies in children under five. Neural crest development is regulated by dynamic switches in transcriptional programmes, guided by a variety of growth factors. Due to its developmental origin, neuroblastoma is unique in that these tumours often retain overactivation of growth factor signalling, which can be targeted by receptor tyrosine kinase (RTK) inhibitors. However, mutations in kinases, except for ALK, are extremely rare in neuroblastoma. Furthermore, the high degree of intratumoural heterogeneity often renders RTK inhibition ineffective as a monotherapy. For high-risk tumours, which lack effective treatment options, there remains an unmet need for targeted therapies. This review summarises the roles of growth factor receptors in neural crest and neuroblastoma development in light of recent single-cell studies. We provide a systematic overview of RTK inhibitors that can target growth factor signalling in neuroblastoma and detail their current status in clinical development. We also explore the role of intratumoural heterogeneity in resistance to RTK inhibitors, focusing on the adrenergic-to-mesenchymal transition, which drives a switch in growth factor receptor expression. Finally, we discuss strategies to overcome RTK inhibitor resistance by targeting neuroblastoma cell plasticity, disrupting downstream signalling pathways, or inhibiting escape mechanisms from cell death. This review provides a theoretical basis for developing novel combination therapies incorporating RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251f60bb5eddba4ba1d386fbe214011bbf9166f4" target='_blank'>
              Advances in Targeting Growth Factor Signalling in Neuroblastoma and Overcoming Drug Resistance
              </a>
            </td>
          <td>
            K. Ivanenko, Ruslan Shaymardanov, Vladimir S Prassolov, Timofey Lebedev
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Genomic integrity is crucial to the cellular life cycle, which involves a tightly regulated process where cells progress through specific phases to ensure that fully replicated, undamaged DNA is inherited by daughter cells. Any dysfunction in this process or unrepaired DNA damage leads to cell cycle arrest and programmed cell death. Cancer cells are known to exploit these mechanisms to continue dividing. Usually, DNA damage arrests replication, allowing the DNA Damage Response (DDR) pathway to activate, which repairs the DNA or bypasses the damage to support cell survival and preserve genome integrity. For DNA damage bypass or translesion synthesis (TLS), a group of low-fidelity polymerases perform error-prone DNA synthesis opposite damaged bases, where REV1 functions as the main scaffolding protein. Previously, we reported non-TLS functions of REV1, including its role in triggering DNA damage-dependent specific DNA metabolic processes. Methods and Results: In this study, we demonstrate that REV1 plays a significant role in cell cycle progression and that its loss causes arrest at the G2/M phase in flow cytometry analysis. This unexpected phenotype includes dysregulation of G2/M regulators, such as Cyclin B1 and tubulins, in REV1-deficient cells compared to controls, as quantified by Western blot. Additionally, phosphorylation of histone H3 at serine 28 was significantly reduced in these REV1-deficient cells. These G2/M arrest features were even more pronounced in REV1-deficient cells treated with the tubulin inhibitor colchicine. Conclusions: Overall, this study reveals a previously unrecognized link between REV1 TLS polymerase inhibition and the G2/M cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/349f4407e91b1d8708aa93b1aa069a7dcc5ae83b" target='_blank'>
              REV1 Loss Triggers a G2/M Cell-Cycle Arrest Through Dysregulation of Mitotic Regulators
              </a>
            </td>
          <td>
            Brailey Buntin, Madison Guyette, Vihit Gupta, Kanayo Ikeh, Sombodhi Bhattacharya, Erica N. Lamkin, Allison Lafuze, Roxana del Rio-Guerra, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-12-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA methylation (DNAm) is an epigenetic modification involved in gene regulation. DNAm quantitative trait loci (mQTLs) have been identified in many tissues, but bulk-tissue studies obscure cell type-specific effects. Here, we present the first multi-tissue landscape of cell type-dependent regulation of DNAm in humans by mapping cell type-interaction mQTLs (imQTLs) across seven tissue types (breast, colon, lung, ovary, prostate, kidney, and whole blood), identifying 3,150 imQTLs. Inter-individual variability in cell type proportion, rather than mean proportion, was most associated with imQTL discovery. The cell type with the most imQTLs tended to have interaction effects directionally consistent with mQTL marginal effects from bulk-tissue. imQTLs exhibited biologically relevant effect sharing across related cell types. Compared to cell-agnostic mQTLs, imQTLs exhibited stronger enrichment in regulatory elements and higher colocalization with eQTLs and GWAS loci. Our cell type deconvolution-based approach provides a scalable alternative to single-cell DNAm profiling for uncovering the cellular contexts of genetic regulation of DNAm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9156b5f2cddfd87e50484abcf953648e65b081c5" target='_blank'>
              Mapping the landscape of cell type-dependent genetic regulation of DNA methylation across human tissues
              </a>
            </td>
          <td>
            James L. Li, Niyati Jain, Jason Kiat, Lin Tong, Meritxell Oliva, Kathryn Demanelis, Jasmine Farzana, M. Kibriya, Habibul Ahsan, Lin Chen, Andrew E. Teschendorff, Brandon Pierce
          </td>
          <td>2025-11-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite efforts to understand breast cancer biology, metastatic disease remains a clinical challenge. Identifying suppressors of breast cancer progression and mechanisms of transition to more invasive phenotypes could provide game changing therapeutic opportunities. Transcriptional dysregulation is central to all malignancies, highlighted by the extensive reprogramming of regulatory elements that underlie oncogenic programs. Among these, super-enhancers (SEs) stand out due to their enrichment in genes controlling cancer hallmarks. To reveal novel breast cancer dependencies, we integrated the analysis of the SE landscape with master regulator activity inference for a series of breast cancer cell lines. As a result, we identified T-helper-inducing Poxviruses and Zinc-finger (POZ)/Krüppel-like factor (ThPOK, ZBTB7B), a CD4+ cell lineage commitment factor, as a breast cancer master regulator that is recurrently associated with a SE. ThPOK expression is highest in luminal breast cancer but is significantly reduced in the basal subtype. Manipulation of ThPOK levels in cell lines shows that its repressive function restricts breast cancer cells to an epithelial phenotype by suppressing the expression of genes involved in the epithelial-mesenchymal transition (EMT), WNT/β-catenin target genes, and the pro-metastatic TGFβ pathway. Our study reveals ThPOK as a master transcription factor that restricts the acquisition of metastatic features in breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e6e21f52ee04a7b9c66e53ed4eb4fd24051aae" target='_blank'>
              Super-enhancer profiling reveals ThPOK/ZBTB7B, a CD4+ cell lineage commitment factor, as a master regulator that restricts breast cancer cells to a luminal non-migratory phenotype
              </a>
            </td>
          <td>
            Camila D. Arcuschin, Kamin Kahrizi, R. Sayaman, Carolina DiBenedetto, Pedro J Salaberry, Roxana Pirker, Yizhuo Shen, O. Zakraoui, Cecilia Schwarzer, Alessandro Scapozza, Joseph A. Caruso, Paola Betancur, Julie D. Saba, Jean-Philippe Coppé, M. Barcellos-Hoff, Dietmar Kappes, L. J. van ’t Veer, Ignacio E Schor, Denise P Muñoz
          </td>
          <td>2025-11-13</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Conventional methods available for genome editing have proven non-specific, labour-intensive, and time-consuming. In this context, CRISPR/Cas technology represents a significant breakthrough. It is derived from a sophisticated microbial defence system consisting of clustered regularly interspaced short palindromic repeats, or CRISPR, and the RNA-guided DNA endonuclease Cas. Beyond its original role in genome editing, CRISPR continues to play a major role in the field of proteomics, functional genomics, and molecular therapy. Animal models, including mice, Drosophila, zebrafish, etc., have substantially benefited from CRISPR in uncovering protein function through reverse genetics approaches, including knock-in, knockout, CRISPRi, and indel mutation strategies. On the clinical front, CRISPR gene therapy has also seen successes, including applications in sickle cell disease, hypercholesterolemia, and cancer immunotherapy. However, notable challenges remain, including in vivo packaging and delivery efficiency, toxicity, and genomic off-target effects. Ongoing efforts to overcome these include the development of novel delivery formulations (e.g., nanoparticles, exosomes), artificial intelligence-guided experimental design, and miniaturization of Cas proteins. This review focuses on CRISPR/Cas gene editing mechanisms and explores its state-of-the-art applications in the field of proteomics and theranostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1a9ca0f931dbe7af3d8145ca249e827bef013f" target='_blank'>
              Expanding the CRISPR/Cas toolkit: applications in proteomics and theranostics
              </a>
            </td>
          <td>
            Ashwini Punde, Saurabh Dey, Riya Pandire, Arindam Bhattacharjee, Chinmoy Patra
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Splicing dysregulation is a relevant mechanism of pathogenicity for variants in disease susceptibility genes. Variants affecting the critical intronic +1 and +2 GT nucleotides of the 5' splice sites (5'ss) are generally strong indicators of pathogenicity. However, some +2 T variants create functional noncanonical 5'ss that generate wildtype transcripts, hampering accurate variant interpretation and genetic counseling. We previously showed that variants PALB2 c.108+2T > C and ATM c.1898+2T > G generated significant levels of full-length (FL) transcripts by creating functional atypical GC and GG donor sites, respectively. In this study, we aimed to investigate the splicing impact of +2T variants in the breast cancer susceptibility genes ATM, BRCA1, and PALB2. For this purpose, five minigenes encompassing 29 exons of ATM, BRCA1, and PALB2 were employed. A total of 30 +2T > C/G/A variants were introduced into these constructs by site-directed mutagenesis and analyzed in MCF-7 cells. Four +2T > C variants (ATM c.6347+2T > C, BRCA1 c.5193+2T > C and c.5277+2T > C, and PALB2 c.2748+2T > C) and ATM variants c.6347+2T > A/G produced FL-transcripts (4%-81% of the overall expression). All +2T > C leaky variants conserved a central core of 6 nucleotides (AGgcaa). Variants were assessed according to the ClinGen specifications of the American College of Medical Genetics and Genomics/Association for Molecular Pathology (ACMG/AMP) interpretation guidelines. Two variants (ATM c.6347+2T > C and BRCA1 c.5193+2T > C) were classified as likely benign, consistent with predictions based on their respective ACMG/AMP-based gene specifications. Conversely, two variants (ATM c.6347+2T > G and BRCA1 c.4675+2T > C), initially predicted as likely pathogenic, were reclassified as variant of uncertain significance (VUS). In conclusion, a significant proportion of +2T variants can create functional noncanonical 5'ss, resulting in the production of FL-transcripts that may preserve gene function. Variant-splicing assays provide essential data for accurate clinical classification and for the development of effective clinical management strategies for patients and their families. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/265cac1e77a3aa8b3a6883bebd0ac8a57ede92ec" target='_blank'>
              Generation of functional noncanonical donor splice sites by +2T variants in breast cancer susceptibility genes: impact on clinical interpretation.
              </a>
            </td>
          <td>
            Inés Llinares-Burguet, Lara Sanoguera-Miralles, Elena Bueno-Martínez, Alicia García-Álvarez, Alberto Valenzuela-Palomo, P. Pérez-Segura, M. de la Hoya, Eladio A Velasco-Sampedro
          </td>
          <td>2025-11-13</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Epitranscriptomic regulation, particularly N6-methyladenosine (m ⁶ A) RNA methylation, has emerged as a pivotal layer of gene expression control, complementing traditional epigenetic mechanisms. As the most prevalent internal modification in eukaryotic messenger RNA, m ⁶ A dynamically modulates RNA splicing, stability, translation, and localization through the coordinated actions of writer, eraser, and reader proteins. Recent studies have illuminated the critical role of m ⁶ A in kidney disease and renal fibrosis. Dysregulation of m ⁶ A machinery has been shown to influence key pathogenic pathways in kidney disease development. Moreover, m ⁶ A-modified non-coding RNAs further integrate epitranscriptomic signals into fibrogenic and immune regulatory networks. Therapeutically, targeting m ⁶ A pathways offers promising avenues, with small-molecule inhibitors, RNA-based therapeutics, and clustered regularly interspaced short palindromic repeats (CRISPR)-based m ⁶ A editing technologies under active investigation. However, challenges remain in achieving tissue-specific modulation and avoiding systemic toxicity. This review synthesizes current knowledge on m ⁶ A-mediated regulation in kidney disease, highlighting mechanistic insights, disease-specific alterations, and emerging therapeutic strategies. As the field of epitranscriptomics continues to evolve, m ⁶ A represents a compelling frontier for understanding renal pathophysiology and developing precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572e2ad51a46ae61a256c5f951a93b67ca897db" target='_blank'>
              Epitranscriptomics in kidney disease: insights and therapeutic potential
              </a>
            </td>
          <td>
            Yu-Cheng Tsai, Ming-Chang Tsai, Shuo-Ming Ou, Chih-Ching Lin, Szu-yuan Li
          </td>
          <td>2025-12-12</td>
          <td>Kidney Research and Clinical Practice</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Sickle cell anemia (SCA) is a monogenic inherited disease caused by a single point mutation in the HBB gene. Traditional treatments are ineffective in fundamentally correcting the genetic defect. In recent years, CRISPR-Cas9, an emerging gene editing technology, has shown great potential in the treatment of SCA. This article systematically reviews the mechanism of action of CRISPR-Cas9 and compares it with other editing tools. We focus on its application in SCA treatment, including strategies such as repairing the HBB gene mutation or targeting the HBG1/2 promoter to activate fetal hemoglobin. Furthermore, based on data from cell-based experiments, animal models, and clinical trials, we discuss the efficacy and limitations of representative therapies such as CTX001, Casgevy, and Lyfgenia. Results have shown that CRISPR-Cas9 can significantly improve red blood cell function in patients, and some therapies have received clinical approval, marking a significant breakthrough in its transition from laboratory research to clinical application. However, challenges remain, such as off-target effects, insufficient delivery efficiency, high treatment costs, and transplant-related complications, which need to be addressed. In the future, emerging strategies such as RNA editing are expected to expand into broader areas, including tumor treatment, while improving safety, providing safer and more accessible treatment options for monogenic diseases such as SCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f63e17e30e7bab825eb7a3cacdb558c0d1f25" target='_blank'>
              CRISPR-Cas9 Technology in the Treatment of Sickle Cell Anemia (SCA)
              </a>
            </td>
          <td>
            Zihao Wang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal human malignancies with poor survival outcome. It is characterized by late diagnosis, rapid growth and limited therapeutic choices. Increasing evidence has suggested that gene expression may undergo extensive changes during the process of normal cells to malignant transformation which may play a critical role in the initiation and progression of human PDAC. In this study, we applied single-cell RNA sequencing (scRNA-seq) technology to comprehensively compared the Peripheral Blood Mononuclear Cells (PBMCs) and tissue samples from pancreatic ductal adenocarcinoma patients and normal individuals. We characterized transcriptional heterogeneity and identified differentially expressed genes as well as the changes of immune microenvironment between PDAC and normal individuals through integrative bioinformatic analysis. We believe that our results will provide new insights into the cellular and molecular characterization of PDAC and may provide potential implications for future therapeutic approaches and biomarker discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6870646ac74ff8174d279d7777e8b7094a0c5c48" target='_blank'>
              Single-cell sequencing analysis of the pancreatic cancer samples
              </a>
            </td>
          <td>
            Kaisheng Ma
          </td>
          <td>2025-12-25</td>
          <td>Journal of Clinical Technology and Theory</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A correlation between karyotype diversity and species richness was first observed in mammals in 1980, and subsequently confirmed after controlling for phylogenetic signal. The correlation was attributed to submicroscopic factors, presumably operating at the level of the genome. At the same time, an unexpected association between mutation rates and substitution rates has been observed in all eukaryotes so far examined. One hypothesis to explain the latter observation proposed that neutral mutation (dS) and non-neutral (dN) substitution rates in gene codons co-vary according to genomic position, or location in the genome. Later, it was found that mutation and substitution rates in eukaryotes increase with DNA replication timing during the Synthetic phase, or S phase, of the cell cycle. In 1991, Motoo Kimura proposed a molecular theory of non-adaptive radiation (NAR). Accordingly, genetic drift plays a significant role in speciation, albeit in parallel to and in conjunction with natural selection. The following will examine the contribution of DNA replication timing and DNA repair factors to the relationship between species richness and karyotype diversity, and by extension, to the large variation in species richness across the mammalian Tree of Life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8f06330c7331a025a26ac3aac8d62db4f0ac6f" target='_blank'>
              DNA Replication Timing, Genome Stability and Non-adaptive Radiation
              </a>
            </td>
          <td>
            John Herrick
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer.




 This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6
 low
 (negative/weak) or SMC6
 high
 (moderate/strong).





 SMC6
 low
 tumours tend to show higher CNV. SMC6
 high
 group presented poorer disease-free survival than the SMC6
 low
 group (
 P
 =0.050), mainly for the luminal subtype (
 P
 =0.005). SMC6
 low
 /ER
 pos
 were protective biomarkers for recurrence.




 There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-12-31</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [17, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>