<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-20 09:25:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>14</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity, the capacity of cells to transition between distinct phenotypic and lineage states over time, is a genetically and epigenetically encoded trait essential for normal development and adult tissue homeostasis. In cancer, phenotypic plasticity programs can be deployed aberrantly to enable disease progression and acquired therapeutic resistance. Cancer phenotypic plasticity is a current barrier to achieving cures for advanced cancers using available molecularly targeted therapies. This review summarizes the complex and interconnected molecular pathways implicated in phenotypic plasticity, both in the context of normal tissue homeostasis and cancer. Molecular pathways convergent between these contexts are highlighted while pathways enabling plasticity are distinguished from those that specify the phenotype of already plastic cells. Key unresolved questions in the field are discussed along with emerging technologies that may be used to help answer them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d0b45e5336c08fda7f8b88d683482dae5405ff" target='_blank'>
              The molecular determinants of phenotypic plasticity in homeostasis and neoplasia
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2024-12-13</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In animals, adaptive transcription is a crucial mechanism to connect environmental stimulation to changes in gene expression and subsequent organism remodeling. Adaptive transcriptional programs involving molecules such as CREB, SRF, MEF2, FOS, and EGR1 are central to a wide variety of organism functions, including learning and memory, immune system plasticity, and muscle hypertrophy, and their activation increases cellular resilience and prevents various diseases. Yet, they also form the basis for many maladaptive processes and are involved in the progression of addiction, depression, cancer, cardiovascular disorders, autoimmune conditions, and metabolic dysfunction among others and are thus prime examples for mediating the adaptation-maladaptation dilemma. They are implicated in the therapeutic effects of major treatment modalities such as antidepressants and can have negative effects on treatment, for example, contributing to therapy resistance in cancer. This review examines the universal role of adaptive transcription as a mechanism for the induction of adaptive cell state transitions in health and disease and explores how many medical disorders can be conceptualized as caused by errors in cellular adaptation goals. It also considers the underlying principles in the basic structure of adaptive gene programs such as their division into a core and a directional program. Finally, it analyses how one might best reprogram cells via targeting of adaptive transcription in combination with complex stimulation patterns to leverage endogenous cellular reprogramming dynamics and achieve optimal health of the whole organism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89e34d32f73826dcd87dbd42ca5759bd4b0fabbc" target='_blank'>
              The universal role of adaptive transcription in health and disease.
              </a>
            </td>
          <td>
            Thomas Lissek
          </td>
          <td>2024-11-28</td>
          <td>The FEBS journal</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aim: Resistance to hormonal and targeted therapies in breast cancer limits treatment efficacy. Epigenetic alterations, including changes mediated by DNA methyltransferases, play a key role in this process. Previously, we identified that resistance to tamoxifen and rapamycin is associated with the suppression of DNMT3A. This study aims to further explore the mechanisms underlying this suppression, with a focus on identifying NR6A1 as a novel regulatory factor. Methods: Acquisition of resistant breast cancer cell sublines, MTT-test, immunoblotting, transient transfection and reporter analysis, lentiviral infection, qRT-PCR, and analysis of methylation using bisulfite pyrosequencing. Results: Our findings indicate that the development of cross-resistance in breast cancer cells to hormonal and targeted therapies involves a shift in cell signaling to alternative AKT pathways, marked by a localized suppression of the NR6A1/DNMT3A axis and associated DNA methylation changes. We demonstrated the critical role of NR6A1 downregulation in resistance development. Additionally, we observed activation of Snail - a key regulator in the epithelial-mesenchymal transition - as a mediator of the effects of NR6A1 depletion, establishing a direct link between Snail expression and resistance formation. Conclusion: The coordinated suppression of NR6A1 and DNMT3A may contribute to sustaining the resistant phenotype in breast cancer cells. This pathway could serve as a predictive marker, helping guide the selection of optimal therapeutic strategies for breast cancer treatment in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24584557816dae92ae33446c23b68005521f3e" target='_blank'>
              Breast cancer cell resistance to hormonal and targeted therapeutics is correlated with the inactivation of the NR6A1 axis
              </a>
            </td>
          <td>
            O. Andreeva, D. Sorokin, S. Vinokurova, P. Kopnin, N. V. Elkina, A. Katargin, R. S. Faskhutdinov, D. Salnikova, A. M. Scherbakov, M. Krasil'nikov
          </td>
          <td>2024-11-23</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce561b805defee98bfc023234f32c34a242254a4" target='_blank'>
              Elucidating the role of pyrimidine metabolism in prostate cancer and its therapeutic implications
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Kan Liu, Zhihao Ming, Hong Shan
          </td>
          <td>2025-01-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3eb15125d0d2e4a0c6a634c10b2bffbe96d5f8" target='_blank'>
              Epigenetic modifications and metabolic gene mutations drive resistance evolution in response to stimulatory antibiotics.
              </a>
            </td>
          <td>
            Hui Lin, Donglin Wang, Qiaojuan Wang, Jie Mao, Lutong Yang, Yaohui Bai, Jiuhui Qu
          </td>
          <td>2025-01-16</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d50475b8002866903aae4806efbfc187f7cb121" target='_blank'>
              Canonical and noncanonical NOTCH signaling in the nongenetic resistance of cancer: distinct and concerted control.
              </a>
            </td>
          <td>
            Xianzhe Huang, Wenwei Chen, Yanyan Wang, Dmytro Shytikov, Yanwen Wang, Wangyi Zhu, Ruyi Chen, Yuwei He, Yanjia Yang, Wei Guo
          </td>
          <td>2025-01-02</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The success of cancer therapy has been significantly hampered by various mechanisms of therapeutic resistance. Chief among these mechanisms is the presence of clonal heterogeneity within an individual tumor mass. The introduction of the concept of cancer stem cells (CSCs)—a rare and immature subpopulation with tumorigenic potential that contributes to intratumoral heterogeneity—has deepened our understanding of drug resistance. Given the characteristics of CSCs, such as increased drug‐efflux activity, enhanced DNA‐repair capacity, high metabolic plasticity, adaptability to oxidative stress, and/or upregulated detoxifying aldehyde dehydrogenase (ALDH) enzymes, CSCs have been recognized as a theoretical reservoir for resistant diseases. Implicit in this recognition is the possibility that CSC‐targeted therapeutic strategies might offer a breakthrough in overcoming drug resistance in cancer patients. Herein, we summarize the generation of CSCs and our current understanding of the mechanisms underlying CSC‐mediated therapeutic resistance. This extended knowledge has progressively been translated into novel anticancer therapeutic strategies and significantly enriched the available options for combination treatments, all of which are anticipated to improve clinical outcomes for patients experiencing CSC‐related relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467b9483651e22480e8aee88bab2b5c598d31695" target='_blank'>
              Unraveling the underlying mechanisms of cancer stem cells in therapeutic resistance for optimizing treatment strategies
              </a>
            </td>
          <td>
            Yunhan Tan, Siyuan Qin, Zhe Zhang, Yongen Liu, Li Zhou, Bo Li, Edouard C. Nice, Yuanyuan Zhang, Jing Jing
          </td>
          <td>2025-01-10</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Carcinogenesis often involves significant alterations in the cancer genome, marked by large structural variants (SVs) and copy number variations (CNVs) that are difficult to capture with short-read sequencing. Traditionally, cytogenetic techniques are applied to detect such aberrations, but they are limited in resolution and do not cover features smaller than several hundred kilobases. Optical genome mapping (OGM) and nanopore sequencing [Oxford Nanopore Technologies (ONT)] bridge this resolution gap and offer enhanced performance for cytogenetic applications. Additionally, both methods can capture epigenetic information as they profile native, individual DNA molecules. We compared the effectiveness of the two methods in characterizing the structural, copy number and epigenetic landscape of a clear cell renal cell carcinoma tumor. Both methods provided comparable results for basic karyotyping and CNVs, but differed in their ability to detect SVs of different sizes and types. ONT outperformed OGM in detecting small SVs, while OGM excelled in detecting larger SVs, including translocations. Differences were also observed among various ONT SV callers. Additionally, both methods provided insights into the tumor’s methylome and hydroxymethylome. While ONT was superior in methylation calling, hydroxymethylation reports can be further optimized. Our findings underscore the importance of carefully selecting the most appropriate platform based on specific research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a68c85c3d1dfc77fbfc1f8f772053fa84be16b" target='_blank'>
              Long-read structural and epigenetic profiling of a kidney tumor-matched sample with nanopore sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Sapir Margalit, Zuzana Tulpová, Tahir Detinis Zur, Yael Michaeli, J. Deek, Gil Nifker, Rita Haldar, Y. Gnatek, Dorit Omer, Benjamin Dekel, H. Baris Feldman, Assaf Grunwald, Yuval Ebenstein
          </td>
          <td>2025-01-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Drug resistance is the chief cause of treatment failure for therapies targeting chronic and infectious diseases. Whether the emergence of resistance is accelerated by environmental exposure to low levels of therapeutics remains controversial. Here, we report a non-genetic mechanism of stress adaptation that promotes heritable resistance to the widely used antifungal drug fluconazole. In the human fungal pathogen Candida albicans, transient exposure to subtherapeutic fluconazole doses induces a protective response that we term para-resistance. Like conventional resistance mechanisms, para-resistance is heritable. However, it does not arise from genetic mutations and can revert spontaneously. Systematic analyses of para-resistant isolates suggest that its key regulators include the stress-activated MAP kinase Hog1, the histone deacetylase subunit Snt1, the chromatin regulator Rap1, and the Sko1 transcriptional factor. Notably, molecules that disrupt biomolecular condensation and prion propagation – crucial for the inheritance of protein assemblies – block the induction of para-resistance, whereas inhibiting histone deacetylases facilitates its induction. We find that para-resistance is common in clinical isolates and, remarkably, passage through the mammalian gut triggers its acquisition, compromising fluconazole’s therapeutic efficacy. Our work defines a pervasive, prion-like epigenetic mechanism of stress adaptation and highlights potential strategies to mitigate the rapid emergence of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6a3a4969dd38cce29e6bd3dee1a41bff2be4085" target='_blank'>
              Stress-driven emergence of heritable non-genetic drug resistance
              </a>
            </td>
          <td>
            Jinglin L. Xie, Sifei Yin, Theodore S. Yang, Kiran Chandrasekher, Luke Hanson, Sang Hu Kim, Lucas Esqueda, Catherine A. Hogan, Niaz Banaei, June L. Round, Kyla S. Ost, Judith Berman, Dan Jarosz
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="This study aimed to generate Car- and Pac-resistant cell lines from the human lung adenocarcinoma H1792 cell line, designated as H1792/Car and H1792/Pac, and perform transcriptome sequencing to identify potential targets. Common differentially expressed genes (Co-DEGs) in both resistant cell lines were identified, followed by hub gene identification. Online validation was conducted through GEPIA and Kaplan–Meier Plotter platforms, with experimental validation performed using real-time quantitative PCR (RT-qPCR). After six months, the H1792/Car and H1792/Pac cell lines exhibited a 10.7-fold and 5.6-fold increase in resistance to Car and Pac, respectively. Flow cytometry analysis demonstrated that both resistant cell lines were resistant to cell cycle arrest and apoptosis induced by Car or Pac. Transcriptomic sequencing identified 123 Co-DEGs, including 72 upregulated and 51 downregulated genes, consistently expressed in both H1792/Car and H1792/Pac cell lines. Among these, 13 hub genes were identified, with colony-stimulating factor 3 (CSF3) uniquely associated with post-progression survival (PPS) in adenocarcinoma patients undergoing chemotherapy. Notably, CSF3 expression was significantly elevated in both H1792/Car and H1792/Pac compared to parental cells. These findings underscore the value of drug-resistant models in uncovering critical biomarkers. CSF3 emerges as a promising guiding marker or potential molecular target for optimizing Car- and Pac-based doublet regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/319b42d1b63fc345fe5d5188434196b326a0a009" target='_blank'>
              Transcriptomic Profiling of Carboplatin- and Paclitaxel-Resistant Lung Adenocarcinoma Cells Reveals CSF3 as a Potential Biomarker for the Carboplatin Plus Paclitaxel Doublet Regimens
              </a>
            </td>
          <td>
            Pritsana Raungrut, Suchanan Tanyapattrapong, Thipphanet Masjon, Saowanee Maungchanburi, P. Thongsuksai
          </td>
          <td>2024-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e385b264839697ab3b154e181f427818a7fd2d8b" target='_blank'>
              Biological and genetic characterization of a newly established human primary multidrug-resistant distal cholangiocarcinoma cell line, CBC3T-6
              </a>
            </td>
          <td>
            Mingzhen Bai, Ruoshui Wang, Chongfei Huang, Ruyang Zhong, Ningzu Jiang, Wenkang Fu, Ningning Mi, Long Gao, Yuyao Jin, Haidong Ma, Jie Cao, Haiying Yu, Qiang Jing, Chao Zhang, Ping Yue, Yong Zhang, Yanyan Lin, Hengwei Zhang, Wenbo Meng
          </td>
          <td>2024-11-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer is a heterogeneous disease that can be molecularly classified based on the expression of hormone receptors and the overexpression of the HER2 receptor (ERBB2). Targeted therapies for HER2-positive breast cancer, including trastuzumab, antibody drug conjugates (ADCs) and tyrosine kinase inhibitors, have significantly improved patient outcomes. However, both primary and acquired resistance to these treatments pose challenges that can limit their long-term efficacy. Addressing these obstacles is vital for enhancing therapeutic strategies and patient care. Alternative splicing, a post-transcriptional mechanism that enhances transcript diversity (isoforms) within a cell, can result in isoform-encoded proteins with varied functions, cellular localizations, or binding properties. In this study, we undertook a comprehensive characterization of the alternative splicing isoforms of HER2, assessed their expression levels in primary breast tumors and cell lines, and explored their role in resistance to anti-HER2 therapies. Our results have significantly expanded the catalog of known HER2 protein-coding isoforms from 13 to 90, revealing distinct patterns of protein domains, cellular localization, and protein structures, as well as mapping their antibody-binding sites. Additionally, by profiling expression in 561 primary breast cancer samples and analyzing mass spectrometry data for translation evidence, we discovered a complex landscape of splicing isoform expression in primary tumors, revealing novel isoforms that were previously unrecognized and are not evaluated in routine clinical practice. This extends beyond the traditional profile based solely on HER2 gene expression and translation. Finally, by assessing HER2 isoform expression in cell cultures that are either sensitive or resistant to trastuzumab and ADCs (T-DM1 or T-DXd), we found that drug-resistant tumor cells shifted their expression toward splicing isoforms that lack the antibody-binding domains. Our results substantially broaden the understanding of HER2 protein-coding isoforms, revealing distinct mechanisms of potential resistance to anti-HER2 therapies, particularly ADCs, by uncovering a new dimension of splicing isoform diversity. This expanded landscape of HER2 isoforms, marked by unique domain patterns and altered antibody-binding sites, emphasizes the crucial role of alternative splicing investigations in advancing precision-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1c7501f2f280bbf5f50689bdfd7420891ac09d" target='_blank'>
              Beyond HER2 expression in breast cancer: Investigating alternative splicing profiles as a mechanism of resistance to anti-HER2 therapies
              </a>
            </td>
          <td>
            G. D. A. Guardia, C. H. dos Anjos, A. R. Pozzo, F. F. dos Santos, A. Birbrair, P. Asprino, A. A. Camargo, P. A. F. Galante
          </td>
          <td>2024-11-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae4e9e70a4e6d9c16865710a48fa98c28a99bb4" target='_blank'>
              The role of gene copy number variation in antimicrobial resistance in human fungal pathogens
              </a>
            </td>
          <td>
            Adarsh Jay, David F. Jordan, Aleeza Gerstein, Christian R. Landry
          </td>
          <td>2025-01-06</td>
          <td>npj Antimicrobials and Resistance</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ae77252749b1c986ef7ff79cc410ca86b264c4" target='_blank'>
              Comparison of baseline global gene expression profiles of prostate cancer cell lines LNCaP and DU145
              </a>
            </td>
          <td>
            Khalid Ahmed, Zhannur Omarova, Alisalman Sheikh, G. Abuova, K. Ghias, Syed Hani Abidi
          </td>
          <td>2024-12-31</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81cf24082e18b8a822679c8c6c602531a443fe70" target='_blank'>
              Nucleotide metabolism-associated drug resistance gene NDUFA4L2 promotes colon cancer progression and 5-FU resistance
              </a>
            </td>
          <td>
            Hongxin He, Shiyao Zheng, Shangkun Jin, Weijie Huang, Enhao Wei, Shen Guan, Chunkang Yang
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Osteosarcoma is a malignant tumor originating from mesenchymal bone tissue, characterized by high malignancy and poor prognosis. Despite progress in comprehensive treatment approaches, the five-year survival rate remains largely unchanged, highlighting the need to clarify its underlying mechanisms and discover new therapeutic targets. Methods This study utilized RNA sequencing data from multiple public databases, encompassing osteosarcoma samples and healthy controls, along with single-cell RNA sequencing data. Various methods were utilized, such as differential expression analysis of genes, analysis of metabolic pathways, and weighted gene co-expression network analysis (WGCNA), to pinpoint crucial genes. Using this list of genes, we developed and validated a prognostic model that incorporated risk signatures, and we evaluated the effectiveness of the model through survival analysis, immune cell infiltration examination, and drug sensitivity evaluation. Results We analyzed gene expression and metabolic pathways in nine samples using single-cell sequencing data. Initially, we performed quality control and clustering, identifying 21 statistically significant cell subpopulations. Metabolic analyses of these subpopulations revealed heterogeneous activation of metabolic pathways. Focusing on the osteoblastic cell subpopulation, we further subdivided it into six groups and examined their gene expression and differentiation capabilities. Differential expression and enrichment analyses indicated that tumor tissues were enriched in cytoskeletal and structural pathways. Through WGCNA, we identified core genes negatively correlated with four highly activated metabolic pathways. Using osteosarcoma patient data, we developed a risk signature model that demonstrated robust prognostic predictions across three independent cohorts. Ultimately, we performed a thorough examination of the model, which encompassed clinical and pathological characteristics, enrichment analysis, pathways associated with cancer markers, and scores of immune infiltration, highlighting notable and complex disparities between high-risk and low-risk populations. Conclusion This research clarifies the molecular mechanisms and metabolic features associated with osteosarcoma and how they relate to patient outcomes, offering novel perspectives and approaches for targeted therapy and prognostic assessment in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c754f2161a5b2138f06ad7cb757b73f892978d" target='_blank'>
              Exploring the heterogeneity of osteosarcoma cell characteristics and metabolic states and their association with clinical prognosis
              </a>
            </td>
          <td>
            Sen Qin, YaoFeng Hu, RuCui Deng, Zhe Wang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-cell fusion has been implicated in various physiological and pathological processes, including cancer progression. This study investigated the role of cell-cell fusion in non-small cell lung cancer (NSCLC), focusing on its contribution to chemoresistance and tumor evolution. By co-culturing drug-sensitive and drug-resistant NSCLC cell lines, we observed spontaneous cell-cell fusion events, particularly under gefitinib selection. These fused cells exhibited enhanced fitness and a higher degree of chemoresistance compared to parental lines across a panel of 12 chemotherapeutic agents. Further analysis, including fluorescence imaging and cell cycle analysis, confirmed nuclear fusion and increased DNA content in the fused cells. Bulk RNA sequencing revealed genomic heterogeneity in fused cells, including enrichment of gene sets associated with cell cycle progression and epithelial-mesenchymal transition, hallmarks of cancer malignancy. These findings demonstrate that cell-cell fusion contributes significantly to therapeutic resistance and the promotion of aggressive phenotypes in NSCLC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78ae5e6d6b40de2fc2ee9f6021e6a3e7d5997a6" target='_blank'>
              Cell-Cell Fusion in NSCLC Confers a Fitness Benefit Under Drug Selection
              </a>
            </td>
          <td>
            Paulameena V. Shultes, D. Tadele, A. Durmaz, D. Weaver, R. Barker-Clarke, Mina N. Dinh, Siqing Liu, Endalkachew A. Alemu, Simon Rayner, Jacob G. Scott
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The human gut microbiome within the gastrointestinal tract continuously adapts to variations in diet, medications, and host physiology. A central strategy for genetic adaptation is epigenetic phase variation (ePV) mediated by bacterial DNA methylation, which can regulate gene expression, enhance clonal heterogeneity, and enable a single bacterial strain to exhibit variable phenotypic states. Genome-wide and site-specific ePV have been well characterized in human pathogens’ antigenic variation and virulence factor production. However, the role of ePV in facilitating adaptation within the human microbiome remains poorly understood. Here, we comprehensively cataloged genome-wide and site-specific ePV in human infant and adult gut microbiomes. First, using long-read metagenomic sequencing, we detected genome-wide ePV mediated by complex structural variations of DNA methyltransferases, highlighting the ones associated with antibiotics or fecal microbiota transplantation. Second, we analyzed an extensive collection of public short-read metagenomic sequencing datasets, uncovering a greater prevalence of genome-wide ePV in the human gut microbiome. Third, we quantitatively detected site-specific ePVs using single-molecule methylation analysis to identify dynamic variations associated with antibiotic treatment or probiotic engraftment. Finally, we performed an in-depth assessment of an Akkermansia muciniphila isolate from an infant, highlighting that ePV can regulate gene expression and enhance the bacterial adaptive capacity by employing a bet-hedging strategy to increase tolerance to differing antibiotics. Our findings indicate that epigenetic modifications are a common and broad strategy used by bacteria in the human gut to adapt to their environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e96b214d76adf4974b5d16217427cbdb1f860f3" target='_blank'>
              Epigenetic phase variation in the gut microbiome enhances bacterial adaptation
              </a>
            </td>
          <td>
            Mi Ni, Yu Fan, Yujie Liu, Yangmei Li, Wanjin Qiao, Lauren E. Davey, Xue-Song Zhang, Magdalena Ksiezarek, Edward Mead, Alan Touracheau, Wenyan Jiang, M. J. Blaser, Raphael H. Valdivia, Gang Fang
          </td>
          <td>2025-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ace1a7b748e6be8dfa6d49964b342cccfe62af" target='_blank'>
              PCK1 as a target for cancer therapy: from metabolic reprogramming to immune microenvironment remodeling
              </a>
            </td>
          <td>
            Na Liu, Xiao-Ren Zhu, Chang-Ying Wu, Yuan-Yuan Liu, Minbin Chen, Jin-Hua Gu
          </td>
          <td>2024-11-22</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f970e4e2fc5fa77ad8483f91874195b589fe0b" target='_blank'>
              Single-cell RNA-seq reveals diverse molecular signatures associated with Methotrexate resistance in primary central nervous system lymphoma cells.
              </a>
            </td>
          <td>
            Rosuke Osako, A. Hayano, Atsushi Kawaguchi, R. Yamanaka
          </td>
          <td>2024-12-05</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The main objective of this study was to explore and identify new genetic targets in small-cell lung cancer (SCLC) through transcriptomics analysis and Mendelian randomization (MR) analysis, which will help in the subsequent development of new therapeutic interventions.In this study, we extracted the SCLC dataset from the Gene Expression Omnibus (GEO) database, processed the data, and screened out differentially expressed genes (DEGs) using R software. Based on expression quantitative trait loci data and the genome-wide association study data of SCLC, MR analysis was used to screen the genes closely related to SCLC disease, which intersect with DEGs to obtain co-expressed genes (CEGs), and the biological functions and pathways of CEGs were further explored by enrichment analysis. In addition, the CIBERSORT algorithm was applied to assess the level of immune cell infiltration in SCLC and to analyze the correlation between CEGs and immune cells. Meanwhile, we performed a survival analysis on these five CEGs using an independent cohort of SCLC patients. Finally, the results for the target genes were validated.In this study, 857 DEGs were identified, including 443 up-regulated and 414 down-regulated genes, and 5 CEGs (PSAT1, PSRC1, COLEC12, PLLP, HP) that were significantly associated with SCLC were identified through further intersecting. The results of enrichment analyses indicated that CEGs play important roles in several key functions and pathways. Immune-cell-related analysis revealed the unique distribution of immune cell infiltration in SCLC and the mechanism of immune cell regulation by CEGs. Survival analysis results indicated that PSRC1 was significantly correlated with the overall survival of SCLC, and the survival rate of the high-expression group was markedly lower than that of the low-expression group. Finally, the consistency of the results between the validation group analyses and MR analysis confirmed that the results of this study is reliable.The CEGs and their associated functions and pathways screened in this study may be potential targets of therapeutic intervention in SCLC by targeting specific molecular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77842bf1a95d4aa6340eee7f38794ec66937f841" target='_blank'>
              Exploring potential therapeutic targets for small cell lung cancer based on transcriptomics combined with Mendelian randomization analysis
              </a>
            </td>
          <td>
            Zhicheng Liao, Pengcheng Jia, Yifan Li, Zhihui Zheng, Jizhou Zhang
          </td>
          <td>2025-01-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is caused by mutations that drive aberrant growth, proliferation, and invasion, thus overriding regulatory mechanisms that normally link these processes to organismal needs and cellular physiology. This imposes demands for the production of energy and biomass and for survival in microenvironments that are often nonphysiologic and nutrient-poor, which are met by rewiring of cellular metabolism. The resultant dependence of tumor cells on altered metabolism can induce sensitivity to specific metabolic perturbations that can be exploited for cancer therapy. Some cancers are caused by mutations that impart a novel function to metabolic enzymes, leading to the production of a tumor-promoting metabolite that is dispensable in normal cells, representing an ideal therapeutic target. Tumors can also exploit metabolic regulation of cellular immunity to evade antitumor immune responses, and deciphering this biology has revealed potential targets for therapeutic intervention. Here, we discuss a number of illustrative examples highlighting the therapeutic potential and the challenges of targeting metabolism for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096951e0d89ec39b14a1cc5abbb929dff9f4e14a" target='_blank'>
              Cancer Therapies Targeting Cellular Metabolism.
              </a>
            </td>
          <td>
            Benjamin Morris, Alejandro Gutierrez
          </td>
          <td>2024-11-25</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is considered as the most frequent cancer among women and a significant contributor to mortality. The CRISPR/Cas9 gene-editing tool has promising applications for BC drug resistance. It is a unique and creative approach that has lately attracted attention and can be used to fix gene alterations related to multidrug resistance. Recent research has effectively investigated and targeted particular genes linked to BC treatment resistance using CRISPR/Cas9 gene editing, including those linked to hormone receptor signaling, drug efflux transporters, and DNA repair pathways. The CRISPR/Cas9 technology's selective disruption or mutation of these genes provides valuable information about their role in resistance and paves the path for cutting-edge treatment options. CRISPR/Cas9 gene editing can overcome BC treatment resistance by identifying crucial genetic variables and revealing new therapeutic targets Despite the advantages, there are limitations in the study on CRISPR/Cas9-based gene editing for BC treatment resistance, for example, off-target effects and the improvement of delivery techniques are still major issues. Successful clinical translation depends on methods to improve the specificity and effectiveness of CRISPR/Cas9 editing and to solve these constraints. This review aims to explore the possibility of CRISPR/Cas9 gene editing as an innovative method of combating BC medication resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4162c264e1c96e6bc11bcded7869da58b56f7ba" target='_blank'>
              Molecular Mechanisms of Breast Cancer Drug Resistance and CRISPR/Cas9 Strategies to Overcome
              </a>
            </td>
          <td>
            B. Hussen, Bnar Saleh Ismael, Saman S. Abdulla, Noor Haval Jamal, S. A. Mustafa, Zana Baqi Najmalddin, Mohammed Fatih Rasul
          </td>
          <td>2024-12-30</td>
          <td>BioMed Target Journal</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feaea0d43cfc2d7247e33264b3b0a5a3472dbf9e" target='_blank'>
              Mutational signature analysis predicts bacterial hypermutation and multidrug resistance
              </a>
            </td>
          <td>
            Kalen M. Hall, Leonard G. Williams, Richard D Smith, Erin A Kuang, Robert K. Ernst, Christine M Bojanowski, William C Wimley, L. Morici, Z. Pursell
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Y. Li, C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Adenosine-to-Inosine (A-to-I) RNA editing is a critical post-transcriptional modification that diversifies the transcriptome and influences various cellular processes. Despite its significance, the regulatory mechanisms controlling A-to-I editing remain largely unknown. In this study, we present two complementary CRISPR-based genetic screening platforms: CREDITS (CRISPR-based RNA EDITing regulator Screening), which enables genome-scale identification of editing regulators using an RNA recorder-based reporter system, and scCREDIT-seq (single-cell CRISPR-based RNA EDITing sequencing), which provides multiplexed single-cell characterization of transcriptome and editome changes for pooled perturbations on dozens of selected genes. Through screening 1,350 RNA-binding proteins, we identified a series of known and novel A-to-I editing regulators. Detailed mechanistic investigation revealed DDX39B as a global repressor of A-to-I editing, which functions by preventing double-stranded RNA accumulation through its helicase and ATPase activities. We demonstrate that targeting DDX39B significantly enhances the efficiency of RNA editing-based tools like CellREADR and LEAPER, and represents a promising strategy for anti-HDV therapy by modulating viral genome editing. These technological advances not only expand our understanding of RNA editing regulation but also provide powerful tools for exploring tissue-specific and context-dependent RNA modification mechanisms, with broad implications for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02f7d393c0da3066112a870364908fe61bba6dd" target='_blank'>
              Multimodal CRISPR screens uncover DDX39B as a global repressor of A-to-I RNA editing
              </a>
            </td>
          <td>
            Tianzi Wei, Jiaxuan Li, Xiang Lei, Risheng Lin, Qingyan Wu, Zhenfeng Zhang, Shimin Shuai, Ruilin Tian
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most prevalent malignant bone tumor in adolescents and young adults. OS cells grow in a permissive local microenvironment which modulates their behavior and facilitates all steps in tumor development (e.g., proliferation/quiescence, invasion/migration, and drug resistance) and contributes to their intrinsic heterogeneity. The lung parenchyma is the most common metastatic site in OS, and metastatic foci are frequently associated with a poor clinical outcome. Although multiple factors may be responsible for the disease, including genetic mutations (e.g., Rb and p53), the molecular mechanism of development of OS remains unclear, and the conventional treatment for OS is still based on a sequential approach that combines chemotherapy and surgery. Also, despite the increase in clinical trials, the survival rates for OS have not improved. Non-specific targeting therapies thus show poor therapeutic effects, along with side effects at high doses. For these reasons, many efforts have been made to characterize the complex genome of OS thanks to the whole-exome analysis, with the aim of identifying predictive biomarkers to give these patients a better therapeutic option. This review aims to summarize and discuss the main recent advances in OS molecular research for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62807be22de865f1f377913f1a5b07abd346d218" target='_blank'>
              Whole-Exome Analysis and Osteosarcoma: A Game Still Open
              </a>
            </td>
          <td>
            C. Chiappetta, Carlo Della Rocca, Claudio Di Cristofano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tubo-ovarian high-grade serous carcinoma (HGSC) is the most lethal gynecological malignancy and frequently responds to platinum-based chemotherapy because of common genetic and somatic impairment of DNA damage repair (DDR) pathways. The mechanisms of clinical platinum resistance are diverse and poorly molecularly defined. Consequently, there are no biomarkers or medicines that improve patient outcomes. Herein we use single cell mass cytometry (CyTOF) to systematically evaluate the phosphorylation and abundance of proteins known to participate in the DNA damage response (DDR). Single cell analyses of highly characterized HGSC cell lines that phenocopy human patients show that cells with comparable levels of intranuclear platinum, a proxy for carboplatin uptake, undergo different cell fates. Unsupervised analyses revealed a continuum of DDR responses. Decompositional methods were used to identify eight distinct protein modules of carboplatin resistance and sensitivity at single cell resolution. CyTOF profiling of primary and secondary platinum-resistance patient models shows that a complex DDR sensitivity module is strongly associated with response, suggesting it as a potential tool to clinically characterize complex drug resistance phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1700b92ee74968435b1e4e6c96fff15f73c54ccf" target='_blank'>
              Coordinated protein modules define DNA damage responses to carboplatin at single cell resolution in human ovarian carcinoma models
              </a>
            </td>
          <td>
            Jacob S. Bedia, Ying-Wen Huang, Antonio Delgado Gonzalez, Veronica D. Gonzalez, I. Funingana, Zainab Rahil, Alyssa Mike, Alexis Lowber, M. Vias, Alan Ashworth, James D. Brenton, W. Fantl
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Despite extensive research on antibiotic resistance, the potential effects of antibiotic treatments on bacterial tolerance and resistance remain a significant concern. Although bacterial cells adopt a variety of mutational strategies to resist unfavorable circumstances, it is still unclear how antibiotic tolerance and resistance mechanisms affect bacterial fitness characteristics and whether evolved mutants exhibit similar properties across different cell populations subjected to the same conditions. Here, we used Escherichia coli, a fluoroquinolone antibiotic (ofloxacin), and adaptive laboratory evolutionary experiments to demonstrate that ofloxacin tolerance and resistance can evolve independently across different cell populations exposed to identical conditions. Fitness attributes, such as lag score, doubling time, competition score, and other metabolic features, were variably affected by antibiotic tolerance and resistance mechanisms. However, we did not observe strong and apparent correlations between fitness trade-offs and antibiotic tolerance and resistance. While our whole-genome sequencing identified some shared mutations, such as single nucleotide polymorphisms in the icd gene (a crucial citric acid cycle gene), evolved cell populations exhibited diverse genetic mutations without a clear pattern of a conserved evolutionary pathway. Our study also identifies unique phenotypes, such as those displaying significantly lower minimum inhibitory concentration levels compared to the parental strain yet showing remarkably high tolerance to the same antibiotic. Altogether, our study, examining the phenotypic and mutational landscapes of fluoroquinolone-induced strains, contributes to our understanding of complex bacterial adaptation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d5760f3238ce410965ca0f13f5c55e28d885a91" target='_blank'>
              The Diverse Phenotypic and Mutational Landscape Induced by Fluoroquinolone Treatment
              </a>
            </td>
          <td>
            S. G. Mohiuddin, Pouria Kavousi, Diego Figueroa, Sreyashi Ghosh, Mehmet A. Orman
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Drug resistance is a major obstacle to the successful treatment of non-small cell lung cancer (NSCLC), the most common subtype of lung cancer. The purpose of this research was to look at possible treatment approaches and the processes behind drug resistance in non-small cell lung cancer.
Methods: From March 2022 to February 2024, a prospective observational research was carried out at Lady Reading Hospital in Peshawar. 128 NSCLC patients in all were enrolled. Next-generation sequencing, DNA methylation tests, and immunological profiling were used to examine the genetic, epigenetic, and environmental components linked to medication resistance. Multivariate logistic regression, t-tests, chi-square tests, and Kaplan-Meier survival analysis were among the statistical techniques used.
Results: T790M mutations contributed to resistance, with EGFR mutations found in 35% of patients. Ten percent of patients had MET amplifications, while seven percent of patients had ALK rearrangements. In resistant cancers, epigenetic changes such as hypoacetylation of histones and hypermethylation of CDKN2A were common. Immunosuppressive cells and cancer-associated fibroblasts were abundant in the tumor microenvironment. The median overall survival (OS) of patients with drug-resistant NSCLC was 14 months, a substantial decrease from the 22 months seen in patients without resistance (p < 0.001).
Conclusions: The research emphasizes how genetic, epigenetic, and microenvironmental variables all play a role in the complex process of medication resistance in non-small cell lung cancer. Molecular profiling and targeted medicines along with personalized treatment techniques are critical to overcome resistance and enhancing patient outcomes. Larger, more varied populations and the creation of cutting-edge treatment approaches should be the main topics of future study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96863261655834ce5a911e129e2f5a3a8ee64f44" target='_blank'>
              Understanding the Mechanisms of Drug Resistance in Non-Small Cell Lung Cancer Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Afnan Ali Shah, Waqar Ahmad Tahir, Owais Nawaz Khan, A. Abid, Yasar Umar Khan, Talha Mazhar
          </td>
          <td>2024-12-31</td>
          <td>Innovative Research in Applied, Biological and Chemical Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Cellular senescence plays a significant role in tissue aging. Senescent cells, which resist apoptosis while remaining metabolically active, generate endogenous DNA-damaging agents, primarily reactive oxygen species. Efficient DNA repair is therefore crucial in these cells, especially when they undergo senescence escape, resuming DNA replication and cellular proliferation. To investigate whether senescent cell transcriptomes reflect adequate DNA repair capacity, we conducted a comprehensive meta-analysis of 60 transcriptomic datasets comparing senescent to proliferating cells. Our analysis revealed a striking downregulation of genes encoding essential components across DNA repair pathways in senescent cells. This includes pathways active in different cell cycle phases such as nucleotide excision repair, base excision repair, nonhomologous end joining and homologous recombination repair of double-strand breaks, mismatch repair and interstrand crosslink repair. The downregulation observed suggests a significant accumulation of DNA lesions. Experimental monitoring of DNA repair readouts in cells that underwent radiation-induced senescence supported this conclusion. This phenomenon was consistent across various senescence triggers and was also observed in primary cell lines from aging individuals. These findings highlight the potential of senescent cells as ‘ticking bombs’ in aging-related diseases and tumors recurring following therapy-induced senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e8d9a9bd09842f0e1e6fb75af8a57d50e982d1" target='_blank'>
              Broad repression of DNA repair genes in senescent cells identified by integration of transcriptomic data
              </a>
            </td>
          <td>
            Yann Frey, M. Haj, Y. Ziv, Ran Elkon, Yosef Shiloh
          </td>
          <td>2024-12-31</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Surveillance of drug resistance and the discovery of novel targets-key objectives in the fight against malaria-rely on identifying resistance-conferring mutations in Plasmodium parasites. Current approaches, while successful, require laborious experimentation or large sample sizes. To elucidate shared determinants of antimalarial resistance that can empower in silico inference, we examined the genomes of 724 Plasmodium falciparum clones, each selected in vitro for resistance to one of 118 compounds. We identified 1448 variants in 128 recurrently mutated genes, including drivers of antimalarial multidrug resistance. In contrast to naturally occurring variants, those selected in vitro are more likely to be missense or frameshift, involve bulky substitutions, and occur in conserved, ordered protein domains. Collectively, our dataset reveals mutation features that predict drug resistance in eukaryotic pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f30299235adf20cff0919e33302a042257592b" target='_blank'>
              Systematic in vitro evolution in Plasmodium falciparum reveals key determinants of drug resistance.
              </a>
            </td>
          <td>
            M. Luth, Karla P. Godinez-Macias, Daisy Chen, John Okombo, Vandana Thathy, Xiu Cheng, Sindhu Daggupati, Heledd Davies, Satish K. Dhingra, Jan M Economy, R. C. Edgar, Maria G. Gomez-Lorenzo, E. Istvan, J. Jado, Gregory M. LaMonte, Bruno Melillo, Sachel Mok, Sunil K Narwal, T. Ndiaye, S. Ottilie, Sara Palomo Diaz, Heekuk Park, S. Peña, Frances Rocamora, Tomoyo Sakata-Kato, Jennifer L. Small-Saunders, Robert L Summers, Patrick K. Tumwebaze, M. Vanaerschot, Guoqin Xia, Tomas Yeo, Ashley You, F. Gamo, Daniel E. Goldberg, M. Lee, C. McNamara, Daouda Ndiaye, Philip J. Rosenthal, Stuart L. Schreiber, G. Serra, J. L. De Siqueira-Neto, T. Skinner-Adams, A. Uhlemann, Nobutaka Kato, A. Lukens, D. Wirth, D. Fidock, E. Winzeler
          </td>
          <td>2024-11-29</td>
          <td>Science</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40243d83674d54172095c39c31dd740a67953065" target='_blank'>
              Comparative transcriptomics suggests a highly species-specific nature of the phenotypic plasticity associated with the outbreaks of the two main pest locusts
              </a>
            </td>
          <td>
            N. Bakkali, S. Saadi, A. Badih, M. Bakkali
          </td>
          <td>2024-12-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The P-glycoprotein efflux pump, encoded by the MDR1 gene, is an ATP-driven transporter capable of expelling a diverse array of compounds from cells. Overexpression of this protein is implicated in the multi-drug resistant phenotype observed in various cancers. Numerous studies have attempted to decipher the impact of genetic variants within MDR1 on P-glycoprotein expression, functional activity, and clinical outcomes in cancer patients. Among these, three specific single nucleotide polymorphisms—T1236C, T2677G, and T3435C - have been the focus of extensive research efforts, primarily through in vitro cell line models and clinical cohort analyses. However, the findings from these studies have been remarkably contradictory. In this study, we employ a computational, data-driven approach to systematically evaluate the effects of these three variants on principal stages of the gene expression process. Leveraging current knowledge of gene regulatory mechanisms, we elucidate potential mechanisms by which these variants could modulate P-glycoprotein levels and function. Our findings suggest that all three variants significantly change the mRNA folding in their vicinity. This change in mRNA structure is predicted to increase local translation elongation rates, but not to change the protein expression. Nonetheless, the increased translation rate near T3435C is predicted to affect the protein’s co-translational folding trajectory in the region of the second ATP binding domain. This potentially impacts P-glycoprotein conformation and function. Our study demonstrates the value of computational approaches in elucidating the functional consequences of genetic variants. This framework provides new insights into the molecular mechanisms of MDR1 variants and their potential impact on cancer prognosis and treatment resistance. Furthermore, we introduce an approach which can be systematically applied to identify mutations potentially affecting mRNA folding in pathology. We demonstrate the utility of this approach on both ClinVar and TCGA and identify hundreds of disease related variants that modify mRNA folding at essential positions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3513112218d83dd11f2ce2e7d98d7d9261c6c5e8" target='_blank'>
              Computational Analysis of MDR1 Variants Predicts Effect on Cancer Cells via their Effect on mRNA Folding
              </a>
            </td>
          <td>
            T. Gutman, T. Tuller
          </td>
          <td>2024-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27645ec5d75c6606ae8cc897c50e3164a6947fb9" target='_blank'>
              Exploring new mechanisms in cancer molecular pathways and pathogenic cell transformation: PIP4K2A as a prognostic marker and therapeutic target in cutaneous malignant melanoma
              </a>
            </td>
          <td>
            Wen-fei Luo, Ding-ming Song, Tong Shen, Yi-bo He, Hong-yang Du, Ming-Jue Si, Lu-Wei Fang
          </td>
          <td>2024-11-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4883d3048c2471ea2cd7612bfae069249c9382eb" target='_blank'>
              Integrated analysis of cell cycle and p53 signaling pathways related genes in breast, colorectal, lung, and pancreatic cancers: implications for prognosis and drug sensitivity for therapeutic potential
              </a>
            </td>
          <td>
            Jiyauddin Khan, Priyanjana Ghosh, U. Bajpai, Kountay Dwivedi, D. Saluja
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract DNA transformation is key for phenotypic diversity and adaptation of Streptococcus pneumoniae including in the emergence of multidrug resistance (MDR). Under laboratory conditions, DNA transformation is facilitated by the artificial triggering of competence by the competence stimulating peptide (CSP). In ongoing DNA transformation work, we observed that exogenous CSP was dispensable depending on the combination of strains and culture media. Here, we carried out a chemogenomic screen to select for S. pneumoniae mutants capable of natural transformation in medium that normally would not sustain natural transformation. Our chemogenomic screen relied on chemical mutagenesis followed by selection of mutants with increased DNA transformation capacities. Sequencing the genome of these mutants revealed an abundance and diversity of mutated genes proven experimentally to increase natural transformation. A genome wide association study between MDR and sensitive clinical isolates revealed gene mutations associated with MDR, many of which intersected with those pinpointed by our chemogenomic screens and that were proven to increase natural transformation. S. pneumoniae has adopted DNA transformation as its lifestyle and can select for mutations facilitating DNA transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f2df0477e405f95c7a8519b39b2e83d06e7bf40" target='_blank'>
              Point mutations in functionally diverse genes are associated with increased natural DNA transformation in multidrug resistant Streptococcus pneumoniae
              </a>
            </td>
          <td>
            Flora Peillard-Fiorente, N. Pham, H. Gingras, Chantal Godin, Jie Feng, P. Leprohon, Marc Ouellette
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Evolution by natural selection is expected to be a slow and gradual process. In particular, the mutations that drive evolution are predicted to be small and modular, incrementally improving a small number of traits. However, adaptive mutations identified early in microbial evolution experiments, cancer, and other systems often provide substantial fitness gains and pleiotropically improve multiple traits at once. We asked whether such pleiotropically adaptive mutations are common throughout adaptation or are instead a rare feature of early steps in evolution that tend to target key signaling pathways. To do so, we conducted barcoded second-step evolution experiments initiated from 5 first-step mutations identified from a prior yeast evolution experiment. We then isolated hundreds of second-step mutations from these evolution experiments, measured their fitness and performance in several growth phases, and conducted whole genome sequencing of the second-step clones. Here, we found that while the vast majority of mutants isolated from the first-step of evolution in this condition show patterns of pleiotropic adaptation—improving both performance in fermentation and respiration growth phases—second-step mutations show a shift towards modular adaptation, mostly improving respiration performance and only rarely improving fermentation performance. We also identified a shift in the molecular basis of adaptation from genes in cellular signaling pathways towards genes involved in respiration and mitochondrial function. Our results suggest that the genes in cellular signaling pathways may be more likely to provide large, adaptively pleiotropic benefits to the organism due to their ability to coherently affect many phenotypes at once. As such, these genes may serve as the source of pleiotropic adaptation in the early stages of evolution, and once these become exhausted, organisms then adapt more gradually, acquiring smaller, more modular mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c50e1354d98f18f12597024756458bd2f24fea92" target='_blank'>
              A high-resolution two-step evolution experiment in yeast reveals a shift from pleiotropic to modular adaptation
              </a>
            </td>
          <td>
            Grant Kinsler, Yuping Li, Gavin Sherlock, Dmitri Petrov
          </td>
          <td>2024-12-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Next‐generation sequencing (NGS) has emerged as a transformative technology in oncology, revolutionizing cancer diagnostics and personalized treatment strategies. By providing comprehensive insights into the genetic landscape of tumors, NGS enables the identification of critical somatic and germline mutations, copy number variations (CNVs), and gene fusions. Over the past decade, advancements in NGS platforms have led to greater accuracy, speed, and cost‐effectiveness, making it an integral part of cancer research and clinical diagnostics. Despite its widespread adoption, significant challenges remain, including the need for improved methods to detect minimal residual disease (MRD) and accurately profile tumor heterogeneity. This review explores the evolution of NGS technologies and their pivotal role in cancer biology, from early diagnostics to therapeutic guidance. It delves into the application of NGS in identifying CNVs and gene fusions, monitoring MRD, and the increasing relevance of targeted NGS and spatial genomics. Furthermore, the integration of spatial transcriptomics is highlighted as a frontier in understanding the tumor microenvironment. By addressing these critical aspects, this review provides a comprehensive overview of how NGS is shaping the future of cancer research and treatment, offering a complete overview of potential NGS applications in scientific and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec16577d76cdb54f1382e600b551c2d7b5e57b" target='_blank'>
              Role of next‐generation sequencing in revolutionizing healthcare for cancer management
              </a>
            </td>
          <td>
            O. S. Sahoo, Hiya Aidasani, A. Nayek, Smita Tripathi, Joyeeta Talukdar, Anamta Gul, Deepak Kumar, Ruby Dhar, S. Karmakar
          </td>
          <td>2024-11-23</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Klebsiella pneumoniae, an opportunistic pathogen, is pervasively distributed across the world. Its escalating antibiotic resistance poses a serious threat to global public health. The mechanisms behind this resistance remain largely elusive. In this study, we performed antibiotic susceptibility testing on several clinical isolates of Klebsiella pneumoniae, and a reference strain ATCC13883, and then analysed their transcriptomic profiles to identify genes and pathways associated with antibiotic resistance. Our results showed that a clinical isolate DY16KPN may counteract antibiotics by enhancing the biosynthesis of building blocks of bacterial cell, such as fatty acids, proteins, and DNA, and reducing transmembrane transport. Increased butanoate metabolism and lipopolysaccharide biosynthesis may also contribute to the drug-resistance of Klebsiella pneumoniae. Additionally, we identified resistance-promoting mutations in gene promoter regions, which regulate the activity of downstream drug-resistant genes and pathways. Our results also demonstrated that DY16KPN counterbalances the trimethoprim/sulfamethoxazole-mediated inhibitory effect on the synthesis of tetrahydrofolates and DNA by up-regulating the expression of targeted enzymes of trimethoprim/sulfamethoxazole, dihydrofolate reductase and dihydropteroate synthase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd7409d1c446b41d87b6fccc9e04701e699807c" target='_blank'>
              Transcriptomic analysis reveals pathways underlying the multi-antibiotic resistance of Klebsiella pneumoniae.
              </a>
            </td>
          <td>
            Ying Liu, Zhihui Niu, Rile Wu, Dezhi Yang, Jun Chen, Guoqing Liu, Jun Zhao
          </td>
          <td>2024-12-17</td>
          <td>IET systems biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96442cd68e41fa673e831da90c37e37f248ca02b" target='_blank'>
              A multiomic atlas identifies a treatment-resistant, bone marrow progenitor-like cell population in T cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Jason Xu, Changya Chen, Jonathan H Sussman, S. Yoshimura, T. Vincent, P. Pölönen, Jianzhong Hu, S. Bandyopadhyay, Omar Elghawy, Wenbao Yu, Joseph S. Tumulty, Chia-Hui Chen, Elizabeth Li, Caroline Diorio, R. Shraim, H. Newman, Lahari Uppuluri, A. Li, Gregory M. Chen, David W Wu, Yang-yang Ding, Jessica Xu, Damjan Karanfilovski, Tristan Lim, Miles Hsu, Anusha Thadi, Kyung Jin Ahn, Chi-Yun Wu, Jacqueline Peng, Yusha Sun, Alice Wang, Rushabh Mehta, David Frank, Lauren Meyer, M. Loh, E. Raetz, Zhiguo Chen, Brent L Wood, M. Devidas, K. Dunsmore, S. Winter, Ti-Cheng Chang, Gang Wu, S. Pounds, Nancy R. Zhang, William L Carroll, S P Hunger, Kathrin M Bernt, Jun J Yang, C. Mullighan, Kai Tan, D. Teachey
          </td>
          <td>2024-11-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="Background V-raf murine sarcoma viral oncogene homolog B1 (BRAF) inhibitor (BRAFi) therapy resistance affects approximately 15% of cancer patients, leading to disease recurrence and poor prognosis. The aim of the study was to develop a machine-learning based method to identify patients who are at high-risk of BRAFi resistance and potential biomarker. Methods From Cancer Cell Line Encyclopedia (CCLE) and Genomics of Drug Sensitivity in Cancer (GDSC) databases, we collected RNA sequencing and half maximal inhibitory concentration (IC50) data from 235 pan-cancer cell lines and then identified 37 significant differential expression genes associated with BRAFi resistance. Employing machine learning (ML) models, we successfully classified cell lines into resistant and sensitive groups, achieving robust performance in external validation datasets. Results AOX1 may play a vital part in BRAFi metabolism and resistance. Further, we found that higher mRNA expression of OXTR, H2AC13, and TBX2, and lower mRNA of SLC2A4, as detected by PCR in WM983B and SKMEL-5 cell lines, were independent risk factors for BRAFi resistance and were associated with poor prognosis. Conclusions We established a gene-expression model using ML methods, consisting of 37 variables based on RNA-seq database, which was externally validated and could be used to predict BRAFi resistance. Meanwhile, our findings provide valuable insights into the molecular mechanisms of BRAFi resistance, enabling the identification of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b0592d6ba9ad8ea2cb503fad3bc14ad7271d3d" target='_blank'>
              Machine learning-based pan-cancer study of classification and mechanism of BRAF inhibitor resistance
              </a>
            </td>
          <td>
            Yuhang Zhao, Kai Yang, Yujun Chen, Zexi Lv, Qing Wang, Yuanyuan Zhong, Xiqun Chen
          </td>
          <td>2024-12-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Targeted cancer therapies often induce a reversible drug-tolerant state in subpopulations of cells, akin to bacterial persistence. Precise characterization of these “cancer persisters” may inform the design of more effective treatment strategies. A previous investigation into the transition to persistence of colorectal cancer cell lines has revealed a distinct dependence on drug presence and concentration, not typical of bacterial systems. Leveraging these findings, this study uses mathematical modeling to explore intermittent treatment protocols aimed at diminishing the long-term fitness of the treated population, thereby enhancing therapeutic efficacy. We adapt a mathematical model originally designed for bacteria to describe colorectal cancer population dynamics in response to a series of treatment and release cycles. The model predicts the long-term increase or reduction of a treated population, as well as its asymptotic composition, leading us to identify success and failure regions within a clinically accessible parameter space, also in combination with hypothetical drugs that act on persisters. Strikingly, our analysis suggests, perhaps counter-intuitively, that optimal treatment outcomes may be achieved in correspondence of non-zero recovery periods and lower than currently administered in the clinics drug concentrations. Furthermore, by incorporating patient drug pharmacokinetics in the model, we demonstrate that intermittent dosing strategies currently explored in clinical trials can be optimized to potentially rival the efficacy of continuous dosing regimens. These findings underscore the potential of mathematical models in guiding the design of optimal treatment protocols by fine-tuning non-trivial decisional trade-offs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41c142151d9a1f87899953c865fcf3fb7c41366" target='_blank'>
              Optimal treatment for drug-induced cancer persisters involves release periods and intermediate drug doses
              </a>
            </td>
          <td>
            Mattia Corigliano, Arianna Di Bernardo, M. Cosentino Lagomarsino, Simone Pompei
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT , TP53 , and CTNNB1 , alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Population-scale resources of genetic, molecular, and cellular information form the basis for understanding human genomes, charting the heritable basis of disease, and tracing the effects of mutations. Pooled perturbation assays applied to cellular models, probing the effect of many perturbations coupled with an scRNA-seq readout (Perturb-seq), are especially potent references for interpreting disease-linked mutations or gene expression changes. However, the utility of existing maps has been limited by the comprehensiveness of perturbations possible, and the relevance of their cell line context. Here, we present the first genome-scale CRISPR interference (CRISPRi) perturbation map with single-cell RNA sequencing readout across many human genetic backgrounds in human pluripotent cells. To do so, we establish large-scale CRISPRi screening in human induced pluripotent stem cells from healthy donors, using over 20,000 guide RNAs to target 7,226 genes across 34 cell lines from 26 genetic backgrounds, and gather expression data from nearly 2 million cells. We comprehensively map trans expression changes induced by the target knockdowns, which complement co-expression patterns in unperturbed cells and facilitate the functional annotation of target genes to biological processes and complexes. Consistency of targeting protein complex members point to protein complexes as a nexus for aggregating transcriptional variation, revealing novel interaction partners. We characterise variation in perturbation effects across donors, with expression quantitative trait loci linked to higher genetic modulation of perturbation effects but overall low replication of trans effects due to knockdown of their corresponding cis regulators. This study pioneers population-scale CRISPR perturbations with single cell readouts that will fuel foundation models for the future of effective modulation of cellular disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5800ff9ee76e49e133d6ead67bfba5c5d87d6239" target='_blank'>
              A genome-scale single cell CRISPRi map of trans gene regulation across human pluripotent stem cell lines
              </a>
            </td>
          <td>
            Claudia Feng, Elin Madli Peets, Yan Zhou, Luca Crepaldi, S. Usluer, Alistair S. Dunham, Jana M. Braunger, Jing Su, Magdalena E Strauss, Daniele Muraro, Kimberly Ai Xian Cheam, M. Bonder, Edgar Garriga Nogales, Sarah Cooper, Andrew Bassett, Steven Leonard, Yong Gu, Bo Fussing, David Burke, Leopold Parts, O. Stegle, B. Velten
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0297441e9d4a24ced84d1909df04d7019094986" target='_blank'>
              Functional screening reveals genetic dependencies and diverging cell cycle control in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Daniel J. Merk, F. Tsiami, S. Hirsch, B. Walter, L. Haeusser, Jens D. Maile, Aaron Stahl, M. Jarboui, Anna Lechado-Terradas, Franziska Klose, S. Babaei, Jakob Admard, N. Casadei, C. Roggia, Michael Spohn, J. Schittenhelm, Stephan Singer, U. Schüller, F. Piccioni, N. Persky, Manfred Claassen, M. Tatagiba, Philipp J. Kahle, D. Root, M. Templin, Ghazaleh Tabatabai
          </td>
          <td>2024-12-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Bacterial genomic mutations in Staphylococcus aureus (S. aureus) have been detected in isolated resistant clinical strains, yet their mechanistic effect on the development of antimicrobial resistance remains unclear. The resistance-associated regulatory systems acquire adaptive mutations under stress conditions that may lead to a gain of function effect and contribute to the resistance phenotype. Here, we investigate the effect of a single-point mutation (T331I) in VraS histidine kinase, part of the VraSR two-component system in S. aureus. VraSR senses and responds to environmental stress signals by upregulating gene expression for cell wall synthesis. A combination of enzyme kinetics, microbiological, and transcriptomic analysis revealed the mechanistic effect of the mutation on VraS and S. aureus. Michaelis Menten’s kinetics show that the VraS mutation caused an increase in the autophosphorylation rate of VraS and enhanced its catalytic efficiency. The introduction of the mutation through recombineering coupled with CRISPR-Cas9 counterselection to the Newman strain wild-type (WT) genome doubled the minimum inhibitory concentration of three cell wall-targeting antibiotics. The mutation caused an enhanced S. aureus growth rate at sub-lethal doses of the antibiotics, confirming the causative effect of mutation on bacterial persistence. Transcriptomic analysis showed a genome-wide alteration in gene expression levels and protein-protein interaction network of the mutant compared to the WT strain after exposure to vancomycin. The results suggest that vraS mutation causes several mechanistic changes at the protein and cellular levels that favor bacterial survival under antibiotic stress and cause the mutation-harboring strains to become the dominant population during infection. Importance Rising antimicrobial resistance (AMR) is a global health problem. Mutations in the two- component system have been linked to drug- resistance in Staphylococcus aureus, yet the exact mechanism through which these mutations work is understudied. We investigated the T331I mutation in the vraS gene linked to sensing and responding to cell wall stress. The mutation caused changes at the protein level by increasing the catalytic efficiency of VraS kinase activity. Introducing the mutation to the genome of an S. aureus strain resulted in changes in the phenotypic antibiotic susceptibility, growth kinetics, and genome-wide transcriptomic alterations. By a combination of enzyme kinetics, microbiological, and transcriptomic approaches, we highlight how small genetic changes can significantly impact bacterial physiology and survival under antibiotic stress. Understanding the mechanistic basis of antibiotic resistance is crucial to guide the development of novel therapeutic agents to combat AMR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0ce015d74667f80681dd7299da3e08caeb3120" target='_blank'>
              A mechanistic understanding of the effect of Staphylococcus aureus VraS histidine kinase single point mutation on antibiotic resistance
              </a>
            </td>
          <td>
            Liaqat Ali, Salima Karki, G. Boorgula, Amir Mekakda, Brittnee Cagle-White, Shrijan Bhattarai, Robert Beaudoin, Aryanna Blakeney, Sanjay Singh, Shashikant Srivastava, May H. Abdelaziz
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32d455915f14c9cee5327c8bd03503f5d7e8b33" target='_blank'>
              SNAI1 promotes epithelial-mesenchymal transition and maintains cancer stem cell-like properties in thymic epithelial tumors through the PIK3R2/p-EphA2 Axis
              </a>
            </td>
          <td>
            H. E, Lei Zhang, Zhenhua Yang, Long Xu, Tao Wang, Junhong Guo, Lang Xia, Juemin Yu, Heyong Wang, Y. She, Junqi Wu, Yue Zhao, Chang Chen, Deping Zhao
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract High-risk multiple myeloma is genomically unstable, comprising heterogeneous populations of tumor cells that evolve over time. Light chain escape (LCE) is a clinical phenomenon observed when light chains rise separately from M-spike values, which implies divergent tumor evolution. We sought to understand LCE by performing high-depth transcriptomic and phenotypic studies. We performed single-cell RNA-sequencing (scRNA-seq) and ex vivo drug sensitivity profiling on serial bone marrow biopsies from a patient with LCE at diagnosis, first relapse, and relapsed/refractory timepoints. scRNA-seq revealed distinct transcriptomic subpopulations with phenotypes that could be tracked separately by clinical serum light chain and M-spike values. Genes differentially expressed between subpopulations were assessed for generalizable effects on prognosis from the Multiple Myeloma Research Foundation CoMMpass and GSE24080 datasets. Notably, the LCE subpopulation exhibited gene expression profile featuring prominent LAMP5 overexpression, which was associated with risk for osteolytic bone lesions. Ex vivo drug sensitivity testing displayed differential sensitivity of the subpopulations. Copy number variant inference showed that the transcriptomic subpopulation underlying LCE was related to a genetic subclone that evolved over time. Our findings illustrate that malignant subpopulations underly LCE in multiple myeloma. These studies imply that LCE and LAMP5 gene overexpression portends for increased risk of osteolytic bone disease and adverse prognosis, findings that were confirmed in the subset of patients from the CoMMpass database with LCE. Significance: scRNA-seq was used to study a patient with high-risk multiple myeloma featuring LCE. LCE was rooted in a transcriptomic subpopulation that corresponded to a genetic subclone and established novel links between LCE and LAMP5 overexpression to osteolysis and prognosis, validated in RNA-seq databases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e33173187a2b1d3d0f9422813ed4bc1d3b6c7d8" target='_blank'>
              Single-Cell RNA Sequencing before and after Light Chain Escape Reveals Intrapatient Multiple Myeloma Subpopulations with Divergent Osteolytic Gene Expression
              </a>
            </td>
          <td>
            Denis Ohlstrom, Z. Walker, Abhishek Pandey, Lorraine N Davis, Krysta Engel, Zenggang Pan, Peter A Forsberg, T. Mark, Austin E. Gillen, Daniel W. Sherbenou
          </td>
          <td>2024-12-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, questions remain about how to minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. Here, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol, in which two treatments are applied sequentially to first contract the population to a vulnerable state and then push it to extinction, and compare it to a combination therapy protocol. We investigate how factors such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the doses of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait affect the outcome. Our results show that the timing of switching to the second strike has a marked effect on the likelihood of driving the cancer to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals when and why a second strike will succeed or fail. Finally, we demonstrate that our conclusions hold whether we model resistance as a binary trait or as a quantitative, multi-locus trait.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9eae66855ec0b28076c2f378c35dec6bc945dff" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Sperm are traditionally viewed as transcriptionally and translationally silent cells. However, observations that components of the cellular machinery of gene expression are maintained in ejaculated sperm are increasingly cited as challenges to this fundamental assumption. Here, we critically evaluate these arguments and present three lines of evidence from both model and non-model systems that collectively raise the question of whether ejaculated sperm may be capable of active gene expression. First, and critical for arguments surrounding the possibility of differential gene expression, we review recent evidence that spermatozoa may retain the capacity to transcribe and translate their genomes. Second, we highlight how sperm cells can exhibit differential transcript quantities across different post-ejaculation environments. Third, we ask whether the accumulating evidence of remarkable phenotypic plasticity in post-ejaculatory sperm phenotypes could be mechanistically underpinned by changes in sperm gene expression. While these lines of evidence are indirect and do not definitively show transcription of sperm genomes, we highlight how emerging technologies may enable us to test this hypothesis explicitly. Our review advocates for progress in this field and highlights several important evolutionary, ecological and practical implications that will probably transcend disciplines to the clinical and applied reproductive sectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a850ef55df1e8b91ae196bda58256829cfd12d6" target='_blank'>
              Silent cells? Potential for context-dependent gene expression in mature sperm
              </a>
            </td>
          <td>
            Rowan A. Lymbery, Francisco Garcia-Gonzalez, Jonathan P. Evans
          </td>
          <td>2025-01-01</td>
          <td>Proceedings of the Royal Society B: Biological Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte-SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomic analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in 3-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8536f19d4a9f802b3a9ad739c338954491fd6245" target='_blank'>
              Deciphering Colorectal Cancer-Hepatocyte Interactions: A Multiomic Platform for Interrogation of Metabolic Crosstalk in the Liver-Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E. Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Background Genome‐wide association studies (GWAS) have identified close to one hundred loci associated with Alzheimer’s disease (AD) risk. However, for most of these loci we do not understand the underlying mechanism leading to disease. Crispr genome editing in human induced pluripotent stem cells (hiPSCs) provides a model system to study the effects of these genetic variants in a disease relevant cell type. Method We used integrative genomics to identify the cell types and genes that are most likely to explain the disease risk identified in GWAS and the direction of effect associated with elevated or reduced risk. We used crispr genome editing to modify expression of these candidate genes and tested the impact of modulating gene expression on myeloid cell function and AD associated phenotypes in vitro and in vivo. Result AD risk alleles are enriched in myeloid cell enhancers but also in certain gene families. We have focused our functional studies on two such gene families: MS4A4A/MS4A6A and LACTB/B2. MS4A4A/MS4A6A are implicated in TREM2 signaling, and are major regulators of sTREM2 levels in cerebrospinal fluid (CSF) whereas LACTB/B2 are lactamases expressed in mitochondria. LACTB is the major modulator of succinyl carnitine levels, a metabolite in CSF associated with AD. Conclusion iPSC provide powerful models for understanding the molecular mechanisms underlying genetic risk for AD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1575374a49618e38fd6db8a54bcdf541c0140aad" target='_blank'>
              Use of human iPSC lines to model Alzheimer’s genetic risk and understand disease mechanism
              </a>
            </td>
          <td>
            Alison M. Goate, Alexandra Münch, Carmen Romero Molina, Anna Podleśny-Drabiniok, Grace Peppler, Francesca Garretti, Tulsi Patel, Hyo Lee, Edoardo Marcora
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Proteins that emerge de novo from noncoding DNA could negatively or positively influence cellular physiology in the sense of providing a possible adaptive advantage. Here, we employ two approaches to study such effects in a human cell line by expressing random sequences and mouse de novo genes that lack homologs in the human genome. We show that both approaches lead to differential growth effects of the cell clones dependent on the sequences they express. For the random sequences, 53% of the clones decreased in frequency, and about 8% increased in frequency in a joint growth experiment. Of the 14 mouse de novo genes tested in a similar joint growth experiment, 10 decreased, and 3 increased in frequency. When individually analysed, each mouse de novo gene triggers a unique transcriptomic response in the human cells, indicating mostly specific rather than generalized effects. Structural analysis of the de novo gene open reading frames (ORFs) reveals a range of intrinsic disorder scores and/or foldability into alpha-helices or beta sheets, but these do not correlate with their effects on the growth of the cells. Our results indicate that de novo evolved ORFs could easily become integrated into cellular regulatory pathways, since most interact with components of these pathways and could therefore become directly subject to positive selection if the general conditions allow this.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43878c4781a378a056438c19941008a1a26fd3df" target='_blank'>
              Expression of Random Sequences and de novo Evolved Genes From the Mouse in Human Cells Reveals Functional Diversity and Specificity
              </a>
            </td>
          <td>
            Silvia Aldrovandi, Johana Fajardo Castro, K. Ullrich, Amir Karger, Victor Luria, D. Tautz
          </td>
          <td>2024-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4cb54307571359f46baeb0ac619041e8a5efa2f" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to construct a neutrophil prognostic model for predicting prognosis and immune response in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Jinhang Wang, Zifeng Cui, Qiwen Song, Kaicheng Yang, Yanping Chen, Shixiong Peng
          </td>
          <td>2024-12-26</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00fa5cfb6a662e61bba556733a03184f141da281" target='_blank'>
              From Subtypes to Solutions: Integrating CMS Classification with Precision Therapeutics in Colorectal Cancer.
              </a>
            </td>
          <td>
            Xinyi Ding, Hao Huang, Zhang Fang, Jingting Jiang
          </td>
          <td>2024-11-26</td>
          <td>Current treatment options in oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            G. E. Rech, A. C. Lau, R. L. Goldfeder, R. Maurya, A. V. Danilov, C.-L. Wei
          </td>
          <td>2025-01-15</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cae0c3f9678085fee30003f0aae12e5e2f38618" target='_blank'>
              Multiple myeloma long-term survivors exhibit sustained immune alterations decades after first-line therapy
              </a>
            </td>
          <td>
            R. Lutz, F. Grünschläger, Malte Simon, M. Awwad, M. Bauer, S. Yousefian, N. Beumer, L. Jopp-Saile, Anastasia Sedlmeier, Llorenç Solé-Boldo, Bogdan Avanesyan, Dominik Vonficht, Patrick Stelmach, Georg Steinbuss, T. Boch, Simon Steiger, M. Baertsch, N. Prokoph, K. Rippe, B. Durie, Claudia Wickenhauser, A. Trumpp, Carsten Müller-Tidow, D. Huebschmann, Niels Weinhold, M. Raab, B. Brors, Hartmut Goldschmidt, Charles D. Imbusch, M. Hundemer, Simon Haas
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Multiple myeloma (MM), a clonal B-cell neoplasia, is an incurable and heterogeneous disease where survival ranges from a few months to more than 10 years. The clinical heterogeneity of MM arises from multiple genomic events that result in tumour development and progression. Recurring genomic abnormalities including cytogenetic abnormalities, gene mutations and abnormal gene expression profiles in myeloma cells have a strong prognostic power. With the advancement in technologies and the development of novel drugs, the prognostic factors and treatment paradigms of MM have been fast evolving over the past few years. Following the introduction of new highthroughput cytogenomic technologies such as array comparative genome hybridisation (aCGH) or single nucleotide polymorphism array (SNP array) and molecular techniques such as gene expression profiling (GEP) and massively parallel genomic sequencing, the prediction of survival in MM no longer solely depends on conventional cytogenetics and interphase fluorescence in situ hybridisation (iFISH) analysis findings. These new technologies enable screening for all possible chromosomal aberrations and other genomic alterations, identifying each aberration on a case-bycase basis and discovering new aberrations that are relevant in unraveling the tumor cells' complex biology. This in turn allows a better understanding of the disease complexity and heterogeneity. The objective of this review on the genetic architecture of MM is to discuss the latest developments on the cytogenetic/cytogenomic-based risk classification of MM that are currently in use and their prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f51a37b55e510ee5aeb00a67198a8c2f2e860c2a" target='_blank'>
              Genetic architecture of Multiple Myeloma and its prognostic implications - An updated review.
              </a>
            </td>
          <td>
            E. Foong, R. Ankathil
          </td>
          <td>2024-12-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of reversible drug-tolerant cell states, thereby accelerating the evolution of drug resistance. Determining how to optimally balance the competing goals of killing the tumor bulk and delaying resistance evolution in this scenario is a nontrivial question of high clinical importance. In this work, we use a combined mathematical and computational approach to study optimal dosing of anti-cancer drug treatment under drug-induced cell plasticity. Our results show that the optimal treatment steers the tumor into a fixed equilibrium composition while balancing the trade-off between cell kill and tolerance induction in a precisely quantifiable way. Under linear induction of tolerance, a low-dose constant strategy is optimal in equilibrium, while under uniform induction of tolerance, alternating between a large dose and no dose is best. The directionality of drug induction, whether the drug elevates transitions from sensitivity to tolerance or inhibits transitions back, significantly affects optimal dosing. To demonstrate the applicability of our approach, we use it to identify an optimal low-dose strategy for colorectal cancer using publicly available in vitro data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586103632d689f330d569f057f914a090bf47bb0" target='_blank'>
              Optimal dosing of anti-cancer treatment under drug-induced plasticity
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Benedikt Vilji Magn'usson, J. Foo
          </td>
          <td>2024-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1b1de4a2bc0d1d7dad1a7c12515d499e7bb94f6" target='_blank'>
              The tumor suppressor SALL2 opposes chemotherapeutic resistance in breast cancer.
              </a>
            </td>
          <td>
            Qiji Li, Chenxin Li, Yuhao Zhang, Zihan Zheng, Yun Wang, Yingqian Yang, Qingqing Zhu, Rui Wang, Wanhui Xu, Chengming Zhu, Qin Tian, Meng Wang, Liping Ye
          </td>
          <td>2024-11-22</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pseudomonas aeruginosa is an opportunistic human pathogen causing various severe infections. Understanding genetic mechanisms of its metabolic versatility aids in developing novel antibacterial drugs and therapeutic strategies to address multidrug-resistant P. aeruginosa infections. The metabolism of nucleotides and amino acids contributes to the cycle of two key biological macromolecules in the genetic central dogma. Guanine deaminase (GuaD) catalyzes the deamination of guanine to produce xanthine to maintain the homeostasis of the nucleotide pool, and transporters specific to BCAAs (termed as BraT) import BCAAs to keep its intracellular availability level. However, little is known about the evolution of GuaD and BraT in P. aeruginosa population. Here, two copies turned out to be widespread in P. aeruginosa population for each of GuaD and BraT. The phylogenic analysis demonstrated that GuaD1 and BraB were inherited from the ancestor of Pseudomonas, while GuaD2 and BraZ were additionally acquired via evolutionary events in the ancestors of P. aeruginosa. The functional divergence of two copies was supported by different distribution patterns of dN/dS ratios, divergent expression levels, differentially co-expressed genes, and their functional enrichment modules with few intersections. Besides, some co-expressed genes with known functions are involved in infecting hosts, forming biofilm and resisting antibiotic treatment. Taken together, functional divergence following copy number increase and differentiation of co-expression networks might confer greater metabolic potential to P. aeruginosa, especially in response to host immune responses and antibiotic treatments in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7927b5b28ef799e2d95504a663925f20ba1760b6" target='_blank'>
              Evolution of two metabolic genes involved in nucleotide and amino acid metabolism in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Yutong Wu, Yuqi Shi, Xiaohui Liang
          </td>
          <td>2024-12-17</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8a77c179e3e4f5ecf72af9fc5a5da53d8facac" target='_blank'>
              Overview of a comparative analysis of microsatellite instability and standard mismatch repair protein-deficiency tests in a large cancer cohort.
              </a>
            </td>
          <td>
            András Kiss, M. L. Nádorvári, J. Kulka, T. Barbai, E. Raso, István Kenessey, Gábor Lotz, József Tímár
          </td>
          <td>2024-12-04</td>
          <td>Pathologie</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96318a0747c39abf7e72e7d04c3793937c72bd73" target='_blank'>
              Multidrug resistance in Pseudomonas aeruginosa: genetic control mechanisms and therapeutic advances
              </a>
            </td>
          <td>
            Yuanjing Zhao, Haoran Xu, Hui Wang, Ping Wang, Simin Chen
          </td>
          <td>2024-11-27</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Abstract BACKGROUND Breast cancer is the most common malignancy in women; brain metastasis is a marked cause of mortality. Recent genomic studies have identified distinct genetic profiles in brain metastases, suggesting unique mechanisms underlying treatment resistance. This study aimed to compare gene expression in primary breast tumors and the corresponding brain metastases to identify potential biomarkers and therapeutic targets. METHODS Eight paired samples of primary breast tumors and brain metastases from patients who developed brain metastases after radical mastectomy were analyzed. RNA sequencing was used to comprehensively assess gene expression differences between the two tumor types. RESULTS RNA sequencing revealed 391 downregulated and 474 upregulated genes in brain metastases compared to primary tumors. Gene set enrichment analysis revealed that epithelial-mesenchymal transition (EMT)-associated genes (SPP, GREM1, and MMP3) were upregulated in primary breast tumors. Conversely, reactive oxygen species (ROS)-related genes (GPX4 and PRDX family members) were upregulated in brain metastases. CONCLUSION High expression of EMT-related genes in primary tumors indicates that these tumors prepare for metastasis by inducing EMT during radical resection, enabling tumors to invade the surrounding tissues and metastasize to distant organs. Additionally, ROS-related genes are highly expressed in metastatic brain tumors, suggesting that these genes contribute to tumor enhancement at the metastatic site. These findings could aid in development of biomarkers for predicting brain metastasis and for identifying novel therapeutic targets. In future, we aim to analyze tumor localization and functions of these gene products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e41314c75680ba4a96e915950728faec65dcd87" target='_blank'>
              10168- GEN-11 GENETIC ANALYSIS OF PRIMARY BREAST CANCER TUMORS AND BRAIN METASTASIS TUMORS WITH RNA-SEQ
              </a>
            </td>
          <td>
            Ayaka Sakakibara, Masayo Kaneko, M. Shirahata, Tetsuya Sato, Masanori Yasuda, Akihiko Osaki, K. Mishima
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10915882394b88f99450ed5973df2cdfcc3bd788" target='_blank'>
              Expression quantitative trait loci influence DNA damage-induced apoptosis in cancer
              </a>
            </td>
          <td>
            Jessica Bigge, Laura L Koebbe, Ann-Sophie Giel, Dorothea Bornholdt, Benedikt Buerfent, Pouria Dasmeh, Alexander M. Zink, Carlo Maj, Johannes Schumacher
          </td>
          <td>2024-12-02</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The CRISPR/Cas genome editing approach in non-model organisms poses challenges that remain to be resolved. Here, we demonstrated a generalized roadmap for a de novo genome annotation approach applied to the non-model organism Macrobrachium rosenbergii. We also addressed the typical genome editing challenges arising from genetic variations, such as a high frequency of single nucleotide polymorphisms, differences in sex chromosomes, and repetitive sequences that can lead to off-target events. For the genome editing of M. rosenbergii, our laboratory recently adapted the CRISPR/Cas genome editing approach to embryos and the embryonic primary cell culture. In this continuation study, an annotation pipeline was trained to predict the gene models by leveraging the available genomic, transcriptomic, and proteomic data, and enabling accurate gene prediction and guide design for knock-outs. A next-generation sequencing analysis demonstrated a high frequency of genetic variations in genes on both autosomal and sex chromosomes, which have been shown to affect the accuracy of editing analyses. To enable future applications based on the CRISPR/Cas tool in non-model organisms, we also verified the reliability of editing efficiency and tracked off-target frequencies. Despite the lack of comprehensive information on non-model organisms, this study provides an example of the feasibility of selecting and editing specific genes with a high degree of certainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9f9b90d707bd226bf9a4714aba878076437ac1" target='_blank'>
              Roadmap and Considerations for Genome Editing in a Non-Model Organism: Genetic Variations and Off-Target Profiling
              </a>
            </td>
          <td>
            Hanin Wattad, Jonathan Molcho, Rivka Manor, S. Weil, E. Aflalo, V. Chalifa-Caspi, Amir Sagi
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Adult B-cell acute lymphoblastic leukemia (B-ALL) is characterized by genetic heterogeneity and high relapse rate. Forty-four adult B-ALL patients were studied at both diagnosis and relapse by next-generation sequencing (NGS) in order to identify genetic alteration driving relapse. Four main genetic pathways leading to relapse in adults were identified: IKZF1plus genetic profile, RAS mutations and TP53 alterations in Philadelphia chromosome (Ph)-negative B-ALL and acquisition of ABL1 mutations in Ph-positive patients. In addition, three clonal evolution patterns were identified: the persistent clone trajectory (25%), the expanding clone trajectory (11%) and the therapy-boosted trajectory (48%). Our results reveal the presence of preferential biological pathways leading to relapse in adult B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54416c39b5f84bc7f01293f1564700b87fc9be40" target='_blank'>
              Preferential Genetic Pathways Lead to Relapses in Adult B-Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Josgrey Navas-Acosta, Alberto Hernández-Sánchez, Teresa González, Ángela Villaverde Ramiro, Sandra Santos, Cristina Miguel, J. Ribera, Isabe L Granada, M. Morgades, Ricardo Sánchez, E. Such, S. Barrena, Juana Ciudad, Julio Dávila, Natalia de Las Heras, Alfonso García-de Coca, J. Labrador, J. Queizán, Sandra Martín, Alberto Orfao, J. Ribera, Rocío Benito, J. Hernández-Rivas
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The development of sequencing technologies and their increased accessibility in clinical services and genetic laboratories have considerably accelerated the identification of genetic variants associated with rare diseases (RDs). Among these, Mendelian disorders of the epigenetic machinery (MDEM) are rare monogenic diseases characterized by the presence of mutations in genes encoding epigenetic regulators that play a key role in organismal development and cellular functions. Loss of function of these regulators is expected to lead to epigenetic modifications that profoundly affect genome expression and cellular identity. Disruptions in DNA methylation profiles have been documented in MDEMs, emerging as a useful diagnostic tool. The current challenge is to determine whether and how these epigenomic alterations drive the mechanisms underlying the clinical manifestations in patients suffering from this class of diseases. Studying MDEMs may therefore shed light on the important role of epigenetic information in health and disease, particularly the mechanisms involved in the development and understanding of complex pathologies, such as neurodevelopmental disorders and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b00b4062f8130453f8f77acda434d081dca0a7cd" target='_blank'>
              [Genetic diseases of epigenetic machinery].
              </a>
            </td>
          <td>
            Maud de Dieuleveult, Guillaume Velasco
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79d34836731e3bc98f26657fe4be3cd3ea9eb09" target='_blank'>
              The systemic evolutionary theory of the origin of cancer (SETOC): an update
              </a>
            </td>
          <td>
            Antonio Mazzocca, Giovanni Ferraro, Giovanni Misciagna
          </td>
          <td>2025-01-14</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The association between RET and multiple endocrine neoplasia type 2 was established in 1993 and remains one of the very few oncogenes for which distinct phenotypes (medullary thyroid cancer or phaeochromocytoma) are associated with the same hot-spot variants occurring either in germline or somatic DNA. Somatic RET fusion events have also been described in several cancers, including papillary thyroid cancer, non-small cell lung cancer, breast cancer, salivary gland cancer and pancreatic cancer. Highly selective RET inhibitors have improved outcomes in RET-altered cancers and have been well tolerated. Nevertheless, primary and acquired drug resistance has been observed, arising from distinct genomic alterations either in RET (on target resistance) or via alternate oncogenic pathways (bypass resistance). The same mechanisms of resistance have been observed across multiple cancer types, which implies RET-altered cancers evolve away from RET addiction via stochastic subclonal events. Understanding these mechanisms is crucial for identifying therapeutic opportunities to overcome resistance. Successful treatment targeting bypass oncogenes has been reported in several instances, at least for short-term outcomes; in contrast, although several compounds have been reported to overcome on-target RET alterations, none have yet been translated into routine clinical practice and this remains an area of urgent clinical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4b80f3267e487a6f53e42db32a243722c94787" target='_blank'>
              Mechanisms of resistance to RET-directed therapies.
              </a>
            </td>
          <td>
            R. J. Clifton-Bligh
          </td>
          <td>2024-12-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Significant progress in determining the molecular origins and resistance mechanisms of mantle cell lymphoma (MCL) has improved our understanding of the disease's clinical diversity. These factors greatly impact prognosis in MCL patients. Given the dynamic alterations in MCL clones and disease evolution, it is crucial to recognize high-risk prognostic factors at diagnosis and relapse. Clinical factors include a high MCL International Prognostic Index score with a high Ki-67 proliferation index; early disease progression within 24 months of first-line treatment; >3 prior lines of therapy at relapse; and an aggressive (blastoid or pleomorphic) histology. Molecular aberrations include dysregulated cyclin D1; an aberrant SOX11-CD70 axis; upregulated Musashi-2; MYC rearrangement; metabolic reprogramming; and epigenetic changes. Other factors contributing to high-risk MCL include an immune-depleted microenvironment and clone adaptability with complex chromosomal anomalies and somatic mutations in TP53, NSD2, CCND1, CDKN2A, BIRC3, SP140, KMT2D, NFkBIE, SMARCA4, and NOTCH2. Ultra-high-risk MCL is indicated by the coexistence of multiple high-risk prognostic factors in the relapse setting and can portend very short progression-free survival. As MCL treatments advance towards cellular therapies, resistance to anti-CD19 chimeric antigen receptor T-cell therapy is also observed. These findings necessitate revisiting the prognostic impact of high-risk factors, current management strategies, new bi- and tri-specific T-cell engagers, combination therapies, novel therapeutic targets, and next-generation clinical trials for high-risk MCL patients. This article provides a comprehensive update on recognizing and managing high-risk MCL, encompassing current practices and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03364deb8d8c1c44077292ebb1c2031c1c0a3c37" target='_blank'>
              High-Risk MCL: Recognition and Treatment.
              </a>
            </td>
          <td>
            P. Jain, Michael L Wang
          </td>
          <td>2024-12-30</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Motivation Genomic biotechnology has rapidly advanced, allowing for the inference and modification of genetic and epigenetic information at the single-cell level. While these tools hold enormous potential for basic and clinical research, they also raise difficult issues of how to design studies to deploy them most effectively. In designing a genomic study, a modern researcher might combine many sequencing modalities and sampling protocols, each with different utility, costs, and other tradeoffs. This is especially relevant for studies of somatic variation, which may involve highly heterogeneous cell populations whose differences can be probed via an extensive set of biotechnological tools. Efficiently deploying genomic technologies in this space will require principled ways to create study designs that recover desired genomic information while minimizing various measures of cost. Results The central problem this paper attempts to address is how one might create an optimal study design for a genomic analysis, with particular focus on studies involving somatic variation that occur most often with application to cancer genomics. We pose the study design problem as a stochastic constrained nonlinear optimization problem. We introduce a Bayesian optimization framework that iteratively optimizes for an objective function using surrogate modeling combined with pattern and gradient search. We demonstrate our procedure on several test cases to derive resource and study design allocations optimized for various goals and criteria, demonstrating its ability to optimize study designs efficiently across diverse scenarios. Availability and implementation https://github.com/CMUSchwartzLab/StudyDesignOptimization">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/143147eb452ae222669c29fe9d6382ddb00144ad" target='_blank'>
              Optimizing design of genomics studies for clonal evolution analysis
              </a>
            </td>
          <td>
            Arjun Srivatsa, Russell Schwartz
          </td>
          <td>2024-12-02</td>
          <td>Bioinformatics Advances</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74432db7e7bddd7e5f3b4f353ed49acd74ffe66d" target='_blank'>
              Getting the right combination to break the epigenetic code.
              </a>
            </td>
          <td>
            Seda S. Tolu, A. Viny, Jennifer E Amengual, Barbara Pro, Susan E Bates
          </td>
          <td>2024-12-02</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3899cef9922a83c1043ba54cdf83bc829243942" target='_blank'>
              The role of genetic diversity, epigenetic regulation, and sex-based differences in HIV cure research: a comprehensive review
              </a>
            </td>
          <td>
            Punitha Letchumanan, Kumitaa Theva Das
          </td>
          <td>2025-01-03</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Chemotherapy is a potent tool against cancer, but drug resistance remains a major obstacle. To combat this, understanding the molecular mechanisms behind resistance in cancer cells and the protein expression changes driving these mechanisms is crucial. Targeting the Ubiquitin‐Proteasome System (UPS) has proven effective in treating multiple myeloma and shows promise for solid tumours. Despite initial success with the proteasome inhibitor bortezomib, acquired resistance soon after treatment poses a significant challenge to its efficacy. In this study, we explored proteins potentially involved in acquired resistance to bortezomib using label‐free nLC–MS/MS proteomic analysis. The investigation revealed 299 proteins with notable differences in expression levels in the bortezomib‐resistant PC3 prostate cancer cell line. Using bioinformatics tools, we illustrated the top 10 gene ontology (GO) processes [e.g., translational initiation (p = 5.964E‐10), CRD‐mediated mRNA stabilisation (p = 1.636E‐5), and hydrogen ion transmembrane transport (p = 6.46E‐5)] and the top 20 KEGG [e.g., metabolic pathways (p = 7.601E‐13), biosynthesis of amino acids (p = 3.834E‐12), and chemical carcinogenesis—reactive oxygen species (p = 1.891E‐4)] and REACTOME [e.g., metabolism (p = 4.182E‐21), translation (p = 9.484E‐18), and Nonsense‐Mediated Decay (NMD) (p = 1.829E‐8)] pathways in the PC3‐resistant cells. We further refined our results by comparing them with globally validated TCGA datasets. We correlated the 299 proteins identified through proteomic analysis with tumour aggressiveness and resistance by comparing them with the TCGA nodal metastasis N0 vs. N1 datasets using the UALCAN portal and identified 37 proteins consistent with our results. We believe that a combination of bortezomib with chemotherapeutics targeting these proteins could be effective in overcoming the resistance developed against bortezomib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb7e9b77b4e308aef46ff7c96b68009d3d9b402" target='_blank'>
              Comparative Analysis of Acquired Resistance to Bortezomib in Prostate Cancer Cells Using Proteomic and Bioinformatic Tools
              </a>
            </td>
          <td>
            Semih Seker, B. Sahin, A. Yerlikaya
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Every protein progresses through a natural lifecycle from birth to maturation to death; this process is coordinated by the protein homeostasis system. Environmental or physiological conditions trigger pathways that maintain the homeostasis of the proteome. An open question is how these pathways are modulated to respond to the many stresses that an organism encounters during its lifetime. To address this question, we tested how the fitness landscape changes in response to environmental and genetic perturbations using directed and massively parallel transposon mutagenesis in Caulobacter crescentus. We developed a general computational pipeline for the analysis of gene-by-environment interactions in transposon mutagenesis experiments. This pipeline uses a combination of general linear models, statistical knockoffs, and a nonparametric Bayesian statistical model to identify essential genetic network components that are shared across environmental perturbations. This analysis allows us to quantify the similarity of proteotoxic environmental perturbations from the perspective of the fitness landscape. We find that essential genes vary more by genetic background than by environmental conditions, with limited overlap among mutant strains targeting different facets of the protein homeostasis system. We also identified 146 unique fitness determinants across different strains, with 19 genes common to at least two strains, showing varying resilience to proteotoxic stresses. Experiments exposing cells to a combination of genetic perturbations and dual environmental stressors show that perturbations that are quantitatively dissimilar from the perspective of the fitness landscape are likely to have a synergistic effect on the growth defect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d35fab11c10334ca080d2ea7ac3de9caeecc10" target='_blank'>
              Discovering genetic modulators of the protein homeostasis system through multilevel analysis
              </a>
            </td>
          <td>
            Vishal Sarsani, Berent Aldikacti, Tingting Zhao, Shai He, Peter Chien, Patrick Flaherty
          </td>
          <td>2024-12-23</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ebfde7a932000ba8510eefbb0bb716c35e7fa" target='_blank'>
              The landscape of fusion transcripts in plants: a new insight into genome complexity
              </a>
            </td>
          <td>
            P. Chitkara, Ajeet Singh, Rashmi Gangwar, Rohan Bhardwaj, Shafaque Zahra, Simran Arora, Fiza Hamid, Ajay Arya, Namrata Sahu, Srija Chakraborty, Madhulika Ramesh, Shailesh Kumar
          </td>
          <td>2024-12-04</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="RNA velocity has emerged as a popular approach for modeling cellular change along the phenotypic landscape but routinely omits regulatory interactions between genes. Conversely, methods that infer gene regulatory networks (GRNs) do not consider the dynamically changing nature of biological systems. To integrate these two currently disconnected fields, we present RegVelo, an end-to-end dynamic, interpretable, and actionable deep learning model that learns a joint model of splicing kinetics and gene regulatory relationships and allows us to perform in silico perturbation predictions. When applied to datasets of the cell cycle, human hematopoiesis, and murine pancreatic endocrinogenesis, RegVelo demonstrates superior predictive power for interactions and perturbation simulations, for example, compared to methods that focus solely on dynamics or GRN inference. To leverage RegVelo’s full potential, we studied the dynamics of zebrafish neural crest development and underlying regulatory mechanisms using our Smart-seq3 dataset and shared gene expression and chromatin accessibility measurements. Using RegVelo’s in silico perturbation predictions, validated by CRISPR/Cas9-mediated knockout and single-cell Perturb-seq, we establish transcription factor tfec as an early driver and elf1 as a novel regulator of pigment cell fate and propose a gene-regulatory circuit involving tfec and elf1 interactions via the toggle-switch model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb495aa52d71e37d301ee44c41f833b2bc276eae" target='_blank'>
              RegVelo: gene-regulatory-informed dynamics of single cells
              </a>
            </td>
          <td>
            Weixu Wang, Zhiyuan Hu, P. Weiler, Sarah Mayes, Marius Lange, Jingye Wang, Zhengyuan Xue, Tatjana Sauka-Spengler, F. Theis
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In the realm of hospital-acquired and chronic infections, Pseudomonas aeruginosa stands out, demonstrating significant associations with increased morbidity, mortality, and antibiotic resistance. Antibiotic-resistant strains are believed to contribute to thousands of deaths each year. Chronic and latent infections are associated with the bacterial toxin-antitoxin (TA) system, although the mechanisms involved are poorly understood. This study focuses on a novel type II TA system, parDEPa, identified in the genome of P. aeruginosa ATCC 27853. We explored its structural features, functional relationships, and genetic configurations. Our research identified parDEPa homologs in P. aeruginosa, clarified their interactions, and highlighted connections to essential cellular metabolic processes. Notably, homologs of the ParDPa antitoxin were found to be more conserved than the ParEPa toxin. Structural models of the ParEPa toxin and ParDPa antitoxin confirmed their integrity. Through docking and molecular dynamics simulations, we showed that the ParEPa toxin binds to DNA gyrase, inhibiting replication. The stability of the ParDPa-ParEPa complex is primarily driven by hydrophobic interactions (-1763.2 kcal/mol), while the ParEPa-GyrAPa interaction is sustained by strong electrostatic forces (-1294.9 kcal/mol). The RMSD scores indicated greater stability for the ParDPa-ParEPa complex (1.11 Å) than the ParEPa-GyrAPa complex (1.16 Å). RMSF analysis identified key residues involved in the ParDPa-ParEPa complex (Leu59, Gly60, Arg115, Asn116, Arg117) and the ParEPa-GyrAPa complex (Pro48, Gln49, Ser55, Asp94, Gln95). These findings significantly enhance our understanding of the structural and metabolic roles of the chromosomally encoded parDEPa TA module in P. aeruginosa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf54488160a60c3b3fa339a70a51f3c3a270437" target='_blank'>
              Genomic profiling and molecular dynamics analysis of parDEPa toxin-antitoxin homologs targeting DNA gyrase in Pseudomonas aeruginosa: insights from computational investigations.
              </a>
            </td>
          <td>
            Nomita Gupta, Mohit Yadav, Waseem Ali, Garima Singh, Shobhi Chaudhary, Sonam Grover, S. Chandra, Jitendra Singh Rathore
          </td>
          <td>2025-01-01</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Background Amplification of the programmed cell death-ligand 1 gene (CD274) is highly prevalent and associated with a high response rate to immune checkpoint inhibitors (ICIs) in lymphomas, and is also a potential biomarker for ICI treatment of solid tumors. However, the efficacy of ICIs for solid tumors with CD274 amplification identified by comprehensive genomic profiling (CGP) has been unclear. We here examined ICI efficacy for solid tumors with CD274 amplification identified by CGP in a national database. Methods We retrospectively analyzed data from the Center for Cancer Genomics and Advanced Therapeutics database containing 60,155 CGP test results for individuals with solid tumors. Only clinical data from patients treated with ICIs alone (not those undergoing concomitant therapy with molecularly targeted or cytotoxic chemotherapeutic agents) were evaluated. We matched 48 patients in the CD274 amplification-positive group with 170 patients in the CD274 amplification-negative group in a 1:4 ratio based on tumor type, histology, treatment, and age. Overall survival (OS), time to next treatment (TTNT), and response rate were evaluated as treatment outcomes in the two groups. Results OS was similar in the CD274-amplified and matched CD274-non-amplified groups (median of 22.1 vs 26.3 months, respectively; HR of 0.92 with a 95% CI of 0.55 to 1.54; p=0.075). TTNT tended to be longer in the CD274-amplified group than in the matched CD274-non-amplified group (median of 16.5 vs 14.0 months; HR of 0.63 with a 95% CI of 0.37 to 1.08; p=0.091). The objective response rate was 33.3% and 18.4% (difference of 14.9%, with a 95% CI of –0.2% to 31.6%), and the disease control rate was 63.9% and 41.1% (difference of 22.8%, with a 95% CI of 5.1% to 40.4%), in the CD274-amplified and matched CD274-non-amplified groups, respectively. Conclusions The number of patients with solid tumors positive for CD274 amplification in this analysis is the largest to date, and our results suggest that such gene amplification may be associated with the outcome of ICI treatment in such individuals. CD274 amplification identified by CGP may therefore be a predictor of ICI efficacy for solid tumors. Trial registration number UMIN000029779.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e69b62b239b4cbdbf362661323ab2f6c2f830f1f" target='_blank'>
              Evaluation of immune checkpoint inhibitor efficacy for solid tumors with CD274 (PD-L1 gene) amplification identified by comprehensive genomic profiling: retrospective study based on a nationwide database
              </a>
            </td>
          <td>
            Tomohiro Nakayama, Takayuki Takahama, Y. Chiba, Naoki Shiraishi, H. Kawakami, K. Yonesaka, Kazuhiko Nakagawa, H. Hayashi
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The phenomenon of adaptive mutation in bacteria presents several challenges to classical models of evolution, particularly regarding the observed coordination of mutation patterns across populations. Here, we examine statistical evidence from evolved Escherichia coli populations showing mutation enrichment up to 256-fold above background rates in key metabolic genes, with remarkable temporal stability that appears to transcend known bacterial communication mechanisms. We propose a theoretical framework suggesting that bacteria may utilize quantum coherent oscillations, potentially mediated through synchronized membrane potential fluctuations, to achieve this degree of coordination in adaptive mutation. Building on Sprouffske et al.’s findings that high mutation rates can limit adaptive evolution in E. coli, we examined mutation patterns in four key metabolic genes (pykF, topA, cspC, and rpoC) previously identified as targets of selection. While Sprouffske found that extremely high mutation rates impaired adaptation, our analysis reveals that these genes show non-random enrichment patterns (p < 1.76×10-34 for pykF) that maintain remarkable temporal stability across multiple timepoints. Recent advances in quantum biology have demonstrated sustained quantum coherence in biological systems, we present a model for how quantum-mediated bacterial signaling could potentially contribute to adaptive mutation. The framework makes several experimentally testable predictions about mutation patterns, population dynamics, and coherence times in bacterial populations under stress conditions. While substantial experimental validation remains necessary, this work provides specific approaches for investigating potential quantum contributions to bacterial adaptation, with implications for understanding evolutionary mechanisms and bacterial stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c15202aa7837996110092bdd84ad8961fcfb080a" target='_blank'>
              Could Quantum-Mediated Bacterial Signaling Explain Adaptive Mutation?
              </a>
            </td>
          <td>
            Patrick Ross
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The global population's health is significantly and severely threatened by antimicrobial resistance. Consequently, the scientific community allocates substantial of resources and efforts to confront this challenge. By contrast, the majority of these endeavours are focused on antibiotics, and research on antifungal resistance (AFR) is significantly underrepresented. Fungal pathogens acquire drug resistance through a variety of mechanisms. Innovative antifungal treatments and the enhancement of the efficacy of existing antifungals can be facilitated by a comprehensive understanding of the mechanisms by which fungal infections acquire drug resistance. Chromatin structure and gene expression regulation are critical components of fungal species' adaptation to antifungal stress, which suggests a potential therapeutic approach to AFR. This suggests that developing strategies that concentrate on these mechanisms may be a viable approach for controlling antifungal resistance. For the regulation of a diverse array of fungal biology components, epigenetic pathways are indispensable in medical mycology. The development process and the capacity to modify and adapt physical characteristics and resistance to antifungals that are used to treat fungal infections are critically dependent on these methods. The development process and the ability to modify and adapt physical characteristics and resistance to antifungals that are used to treat fungal infections are critically dependent on these methods. A significant concern is increasing resistance to the limited therapeutic options that are available to manage invasive fungal infections, such as histone acetylation and methylation, chromatin remodelling, and gene silencing through heterochromatin, which inhibit prevailing drug-resistance mechanisms. This review discusses the significance of epigenetic pathways in mediating drug resistance in fungi as well as mechanisms of antifungal drug resistance">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b714470b16b023b24367f2fc71a3ef61a0e65b4" target='_blank'>
              The Role of Epigenetic Mechanisms in Modulating Antifungal Resistance in Fungi
              </a>
            </td>
          <td>
            Aya A Tarek, Mohamed N Hassan, Yasmine Tartor
          </td>
          <td>2024-12-01</td>
          <td>Zagazig Veterinary Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4de250806833a072de9bb7a783936d616bd0d5d" target='_blank'>
              cis- and trans-regulatory contributions to a hierarchy of factors influencing gene expression variation
              </a>
            </td>
          <td>
            Siddhant Kalra, S. Lanno, Guadalupe Sanchez, J. Coolon
          </td>
          <td>2024-11-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Saccharomyces cerevisiae, a model organism in telomere biology, has been instrumental in pioneering a comprehensive understanding of the molecular processes that occur in the absence of telomerase across eukaryotes. This exploration spans investigations into telomere dynamics, intracellular signaling cascades, and organelle-mediated responses, elucidating their impact on proliferative capacity, genome stability, and cellular variability. Through the lens of budding yeast, numerous sources of cellular heterogeneity have been identified, dissected, and modeled, shedding light on the risks associated with telomeric state transitions, including the evasion of senescence. Moreover, the unraveling of the intricate interplay between the nucleus and other organelles upon telomerase inactivation has provided insights into eukaryotic evolution and cellular communication networks. These contributions, akin to milestones achieved using budding yeast, such as the discovery of the cell cycle, DNA damage checkpoint mechanisms, and DNA replication and repair processes, have been of paramount significance for the telomere field. Particularly, these insights extend to understanding replicative senescence as an anticancer mechanism in humans and enhancing our understanding of eukaryotes' evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad17fda304fdcc31fddbd809f9bdbe3fec485b81" target='_blank'>
              Life and Death without Telomerase: The Saccharomyces cerevisiae Model.
              </a>
            </td>
          <td>
            Veronica Martinez-Fernandez, A. Barascu, M. Teixeira
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BRCA1, a critical tumor suppressor gene, plays an essential role in maintaining genomic stability through its involvement in DNA double-strand break repair, particularly via homologous recombination. Loss or impairment of BRCA1 function disrupts this repair pathway, resulting in genomic instability and significantly increased susceptibility to breast and ovarian cancers. To elucidate the molecular mechanisms by which BRCA1 mutations contribute to tumorigenesis, we employed quantitative mass spectrometry-based proximity labeling and affinity purification to identify cancer-specific protein-protein interactions (PPIs). Our integrated omics and visualization analysis of interactors revealed that the BRCA1-Y1853ter mutant, through its interaction with BARD1, perturbs the interactome and impacts cellular processes within the cytoplasm and nucleoplasm. Structural data further corroborated these findings, showing enhanced binding between the mutant BRCA1 and specific interactors, suggesting an altered functional profile. Together, these observations raise the hypothesis that the BRCA1-Y1853ter mutant may exhibit gain-of-function characteristics, providing new insights into the molecular and cellular effects of mutations in the BRCA1 C-Terminal (BRCT) domain and their implications for the pathogenesis of breast and ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7081d3dc4a7fe94ebd83524e06e557bb05ca412" target='_blank'>
              Differential Remodeling of the BRCA1 Interactome by a Unique Minimal Clinical Truncation Mutation
              </a>
            </td>
          <td>
            Jingjing Chen, Mihaela E. Sardiu, M. Rekowski, Jules Nde, Zachary Clark, S. Anant, Michael P. Washburn, Roy A Jensen
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This article summarizes the current understanding of epigenetic regulation in grapevines, emphasising their significance in a clonally propagated plant with limited genetic diversity. Key epigenetic processes, including DNA methylation and histone modifications, shape chromatin structure, influencing gene expression. The grapevine leaf methylome reveals similarities with other clonally propagated plants, emphasizing low methylation levels in specific contexts. Epigenetic regulation contributes to grapevine phenotypic plasticity, clonal diversity, and an intriguing dialogue between grafted partners. These mechanisms form a vital part of plant memory, especially in the face of climate change. Despite the potential resetting during plant regeneration, recent evidence suggests the persistence of parental epigenetic imprints in progeny. Understanding how environmental conditions affect epigenetic imprints in grapevine clones is crucial. In Uruguay, where the wine industry faces climate challenges, Tannat stands as an emblematic variety adapted for our environmental production systems. However, climate change predictions in the region include rising temperatures, altered precipitation patterns, and increased extreme events, which could impact some aspects of its adaptation (yields, berry quality and typicity, among others). Vineyard management strategies, along with plant breeding, are essential for adaptation. Adding epigenetic diversity for breeding strategies enhances adaptability, contributing to sustainable viticulture in the face of climate change. The article calls for urgently developing innovative strategies utilizing heritable epigenetic variations, presenting a faster and more efficient approach to grapevine breeding for stress tolerance in the era of climate change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2d71a4282d4f9fa875ff168ae89aee2162dbe7" target='_blank'>
              Epigenetic regulation and memories
              </a>
            </td>
          <td>
            Mercedes Fourment, P. Gallusci
          </td>
          <td>2024-11-26</td>
          <td>Agrociencia Uruguay</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Acute Promyelocytic Leukaemia (APL) arises from an aberrant chromosomal translocation involving the Retinoic Acid Receptor Alpha (RARA) gene, predominantly with the Promyelocytic Leukaemia (PML) or Promyelocytic Leukaemia Zinc Finger (PLZF) genes. The resulting oncoproteins block the haematopoietic differentiation program promoting aberrant proliferative promyelocytes. Retinoic Acid (RA) therapy is successful in most of the PML::RARA patients, while PLZF::RARA patients frequently become resistant and relapse. Recent studies pointed to various underlying molecular components, but their precise contributions remain to be deciphered. We developed a logical network model integrating signalling, transcriptional, and epigenetic regulatory mechanisms, which captures key features of the APL cell responses to RA depending on the genetic background. The explicit inclusion of the histone methyltransferase EZH2 allowed the assessment of its role in the resistance mechanism, distinguishing between its canonical and non-canonical activities. The model dynamics was thoroughly analysed using tools integrated in the public software suite maintained by the CoLoMoTo consortium (https://colomoto.github.io/). The model serves as a solid basis to assess the roles of novel regulatory mechanisms, as well as to explore novel therapeutical approaches in silico.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ae703b1eebb6bdd0c13cb7345ded3d089090d5" target='_blank'>
              Predictive modelling of acute Promyelocytic leukaemia resistance to retinoic acid therapy
              </a>
            </td>
          <td>
            J. A. Sánchez-Villanueva, L. N’Guyen, M. Poplineau, E. Duprez, Elisabeth Remy, Denis Thieffry
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f0ac70633e21f19fc53cbb0d98239a472be098" target='_blank'>
              The diverse landscape of RNA modifications in cancer development and progression.
              </a>
            </td>
          <td>
            Hyung Seok Kim, Jung Woo Eun, Se Ha Jang, Ji Yun Kim, Jee-Yeong Jeong
          </td>
          <td>2024-12-06</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epigenetic modifications play a pivotal role in cellular differentiation and the onset of complex diseases by regulating gene expression without altering the underlying DNA sequence. Mechanisms such as DNA methylation, histone modification, and chromatin remodeling orchestrate the transition from pluripotent stem cells to specialized cell types, ensuring tissue-specific functionality. This research delves into the intricate interplay between these epigenetic regulators and their involvement in developmental processes. Moreover, it examines how aberrations in epigenetic landscapes contribute to the pathogenesis of diseases such as cancer, neurodegenerative disorders, and autoimmune conditions. By employing advanced techniques like single-cell epigenomics and CRISPR-based epigenetic editing, this study aims to uncover novel biomarkers and therapeutic targets. The findings highlight the dynamic and reversible nature of epigenetic marks, offering promising avenues for personalized medicine. Understanding the role of epigenetic modifications in cellular differentiation and disease development is essential for advancing diagnostics, therapeutics, and our broader understanding of human biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05dc2e90b2c758b79765cdc1b955a8ef36a6ccb9" target='_blank'>
              The Role of Epigenetic Modifications in Cellular Differentiation and the Development of Complex Diseases
              </a>
            </td>
          <td>
            Rea Barlas
          </td>
          <td>2024-11-28</td>
          <td>Next Frontier For Life Sciences and AI</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Exosomes can regulate the malignant progression of tumors by carrying a variety of genetic information and transmitting it to target cells. Recent studies indicate that exosomal circular RNAs (circRNAs) regulate multiple biological processes in carcinogenesis, such as tumor growth, metastasis, epithelial–mesenchymal transition, drug resistance, autophagy, metabolism, angiogenesis, and immune escape. In the tumor microenvironment (TME), exosomal circRNAs can be transferred among tumor cells, endothelial cells, cancer‐associated fibroblasts, immune cells, and microbiota, affecting tumor initiation and progression. Due to the high stability and widespread presence of exosomal circRNAs, they hold promise as biomarkers for tumor diagnosis and prognosis prediction in blood and urine. In addition, designing nanoparticles targeting exosomal circRNAs and utilizing exosomal circRNAs derived from immune cells or stem cells provide new strategies for cancer therapy. In this review, we examined the crucial role of exosomal circRNAs in regulating tumor‐related signaling pathways and summarized the transmission of exosomal circRNAs between various types of cells and their impact on the TME. Finally, our review highlights the potential of exosomal circRNAs as diagnostic and prognostic prediction biomarkers, as well as suggesting new strategies for clinical therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418f9affefc784073dd2ac61b2954c1231677942" target='_blank'>
              Exosomal circular RNAs in tumor microenvironment: An emphasis on signaling pathways and clinical opportunities
              </a>
            </td>
          <td>
            Junshu Li, Wencheng Zhou, Huiling Wang, Meijuan Huang, Hongxin Deng
          </td>
          <td>2024-11-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metastatic melanoma remains a major clinical challenge. Large-scale genomic sequencing of melanoma has identified bona fide activating mutations in RAC1, which are associated with resistance to BRAF-targeting therapies. Targeting the RAC1-GTPase pathway, including the upstream activator PREX2 and the downstream effector PI3Kβ, could be a potential strategy for overcoming therapeutic resistance, limiting melanoma recurrence, and suppressing metastatic progression. Here, we used genetically engineered mouse models and patient-derived BRAFV600E-driven melanoma cell lines to dissect the role of PREX2 in melanomagenesis and response to therapy. While PREX2 was dispensable for the initiation and progression of melanoma, its loss conferred sensitivity to clinically relevant therapeutics targeting the MAPK pathway. Importantly, genetic and pharmacological targeting of PI3Kβ phenocopied PREX2 deficiency, sensitizing model systems to therapy. These data reveal a druggable PREX2/RAC1/PI3Kβ signaling axis in BRAF-mutant melanoma that could be exploited clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d015227301e9822a2f9c55706223dee9a9babf44" target='_blank'>
              Targeting the PREX2/RAC1/PI3Kβ Signaling Axis Confers Sensitivity to Clinically Relevant Therapeutic Approaches in Melanoma.
              </a>
            </td>
          <td>
            Catriona A. Ford, Dana Koludrovic, Patricia P. Centeno, M. Foth, Elpida Tsonou, Nikola Vlahov, Nathalie Sphyris, K. Gilroy, Courtney Bull, Colin Nixon, Bryan Serrels, Alison F. Munro, John C Dawson, Neil O. Carragher, Valeria Pavet, David C Hornigold, Philip D. Dunne, Julian Downward, Heidi C. E. Welch, Simon T Barry, Owen J. Sansom, A. Campbell
          </td>
          <td>2024-12-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8adac51915476cb4a951cd92dae63dbf918abebe" target='_blank'>
              CRISPR-based genetic screens in human pluripotent stem cells derived neurons and brain organoids.
              </a>
            </td>
          <td>
            Yu Guo, Xinyu Zhao
          </td>
          <td>2024-11-25</td>
          <td>Cell and tissue research</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Intron retention is a type of alternative splicing in which introns remain unspliced in mature RNA transcripts. In order to explore the landscape and consequences of genetically regulated intron retention, we perform an intron retention quantitative trait loci (irQTL) analysis in 49 human tissues across 838 individuals. We identify 8,624 unique intron retention events associated with genetic polymorphisms. 1,369 irQTLs (16%) are also associated with genome-wide association study (GWAS) traits, suggesting that genetically regulated intron retention may play a causal role in the regulation of these GWAS traits. 1,999 irQTLs (23%) colocalize with eQTLs to their respective gene, suggesting that intron retention and steady-state gene expression levels are regulated by shared sets of genetic variants. Through a colocalization analysis of irQTL:eQTL events, we demonstrate that irQTLs are sufficient to generate eQTLs when one of the alternatively spliced transcripts is preferentially targeted by the nonsense mediated decay (NMD) pathway. Surprisingly, for intron retention events whose potential NMD effects can be confidently predicted based on their positions within known gene annotations, we find that 58.8% (923/1570) of the colocalized irQTL and eQTL pairs show effect size directions that are discordant with the NMD model. Moreover, we find that irQTLs are significantly more likely to occur in the same gene with the same effect size direction as compared to exon skipping QTLs. Through mathematical modeling and analysis of experimental perturbation data, we provide evidence that eQTLs are able to generate irQTLs by altering the steady state ratios of spliced and unspliced transcripts, and we postulate that this mechanism may partially underlie the widespread intron retention observed previously in various biological conditions. Taken together, these results show that intron retention and steady state gene expression levels are closely intertwined to regulate phenotypic traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7207fff2c80c573e0b32d3cba5d95f5f23832382" target='_blank'>
              Origins and Implications of Intron Retention Quantitative Trait Loci in Human Tissues
              </a>
            </td>
          <td>
            Eddie Park, Yi Xing
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The growing prevalence of antibiotic-resistant bacteria within the human microbiome has become a pressing global health crisis. While antibiotics have revolutionized medicine by significantly reducing mortality and enabling advanced medical interventions, their misuse and overuse have led to the emergence of resistant bacterial strains. Key resistance mechanisms include genetic mutations, horizontal gene transfer, and biofilm formation, with the human microbiota acting as a reservoir for antibiotic resistance genes (ARGs). Industrialization and environmental factors have exacerbated this issue, contributing to a rise in infections with multidrug-resistant (MDR) bacteria, such as methicillin-resistant Staphylococcus aureus (MRSA) and carbapenem-resistant Enterobacteriaceae. These resistant pathogens compromise the effectiveness of essential treatments like surgical prophylaxis and chemotherapy, increase healthcare costs, and prolong hospital stays. This crisis highlights the need for a global One-Health approach, particularly in regions with weak regulatory frameworks. Innovative strategies, including next-generation sequencing (NGS) technologies, offer promising avenues for mitigating resistance. Addressing this challenge requires coordinated efforts, encompassing research, policymaking, public education, and antibiotic stewardship, to safeguard current antibiotics and foster the development of new therapeutic solutions. An integrated, multidimensional strategy is essential to tackle this escalating problem and ensure the sustainability of effective antimicrobial treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a3dd99c233a325d1af9d14880833e0f1ff7573e" target='_blank'>
              From Cure to Crisis: Understanding the Evolution of Antibiotic-Resistant Bacteria in Human Microbiota
              </a>
            </td>
          <td>
            Hamed Tahmasebi, Neda Arjmand, Marzieh Monemi, Ali Babaeizad, Farnaz Alibabaei, Negar Alibabaei, A. Bahar, Valentyn Oksenych, Majid Eslami
          </td>
          <td>2025-01-09</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose Endocrine therapy in combination with CDK4/6 inhibition doubles the progression-free survival of patients with advanced ER+ breast cancer, but resistance is inevitable, leaving patients with limited treatment options. Experimental Design We performed unbiased genome-wide CRISPR/Cas9 knockout screens using ER+ breast cancer cells to identify novel drivers of resistance to combination endocrine therapy (tamoxifen) and CDK4/6 inhibitor (palbociclib) treatment. Screen hits were validated by CRISPR/Cas9 knockout models, mechanistic analyses and evaluation of patient samples. Results Our screens identified the inactivation of JNK signalling, including loss of the kinase MAP2K7, as a key driver of combination resistance. We developed multiple CRISPR/Cas9 knockout ER+ breast cancer cell lines (MCF-7 and T-47D) to investigate the effects of MAP2K7, MAPK8 and MAPK9 loss. MAP2K7 knockout increased metastatic burden in vivo and led to impaired JNK-mediated stress responses, as well as promoting cell survival and reducing senescence entry following endocrine therapy and CDK4/6 inhibitor treatment. Mechanistically, this occurred via loss of the AP-1 transcription factor c-JUN, leading to an attenuated response to combination endocrine therapy plus CDK4/6 inhibition. Furthermore, we analysed ER+ advanced breast cancer patient cohorts and found that inactivation of the JNK pathway was associated with increased metastatic burden, and low pJNKT183/Y185 activity correlated with a poorer response to systemic endocrine and CDK4/6 inhibitor therapies. Conclusions Overall, we demonstrate that suppression of JNK signalling enables persistent growth during combined endocrine therapy and CDK4/6 inhibition. Furthermore, our data provide a pre-clinical rationale to screen patients’ tumours for JNK signalling deficiency prior to receiving combined endocrine therapy and CDK4/6 inhibition. STATEMENT OF TRANSLATIONAL RELEVANCE Resistance to CDK4/6 inhibitors in the context of endocrine therapy resistance presents an urgent clinical challenge for the management of estrogen receptor positive (ER+) breast cancer. However, the mechanisms driving resistance to this therapeutic combination remain poorly understood. Here, we have identified inactivation of JNK signalling, specifically loss of the JNK kinase MAP2K7, as a major determinant of resistance to combined endocrine therapy and CDK4/6 inhibition in ER+ breast cancer. We uncovered that MAP2K7 loss augments tumour survival in vitro and in vivo. This occurs via disruption of activator protein-1 (AP-1) transcription factors, and thus prevents the induction of therapy-induced senescence and JNK-induced stress response. These findings reveal a critical tumour suppressor role for the JNK pathway in ER+ breast cancer, highlighting the importance of identifying patients with deficient JNK signalling and cautioning against the development of JNK inhibitors for this setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea27aa25966a8a8e422d458bc9b0ffc44d3b78e" target='_blank'>
              JNK pathway suppression drives resistance to combination endocrine therapy and CDK4/6 inhibition in ER+ breast cancer
              </a>
            </td>
          <td>
            Sarah Alexandrou, Christine S. Lee, Kristine J. Fernandez, Celine E. Wiharja, Leila Eshraghi, John Reeves, Daniel A. Reed, N. Portman, Zoe Phan, H. Milioli, Iva Nikolic, Antonia L. Cadell, David R. Croucher, K. Simpson, Elgene Lim, Theresa E Hickey, Ewan K. A. Millar, Carla L. Alves, Henrik J Ditzel, C. E. Caldon
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is a multifactorial process involving genetic, epigenetic, physiological, and metabolic changes. The ability of tumours to regulate new reactive pathways is essential for their survival. A key aspect of this involves the decision-making process of cancer cells as they balance the exploitation of surrounding and distant tissues for their own benefit while avoiding the rapid destruction of the host. Nutrition plays a central role in these processes but is inherently limited. Understanding how tumour cells interact with non-tumoural tissues to acquire nutrients is crucial. In this review, we emphasise the utility of Drosophila melanogaster as a model organism for dissecting the complex oncogenic networks underlying these interactions. By studying various levels—from individual tumour cells to systemic markers—we can gain new insights into how cancer adapts and thrives. Moreover, developing innovative technologies, such as high-throughput methods and metabolic interventions, enhances our ability to explore how tumours adapt to different conditions. These technological advances allow us to explore tumour adaptations and open new opportunities for potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc422f043ced142c0695662e09ec939b17ed8e3" target='_blank'>
              Metabolic Adaptations in Cancer and the Host Using Drosophila Models and Advanced Tools
              </a>
            </td>
          <td>
            Ernesto Saez-Carrion, Mario Aguilar-Aragon, Lucía García-López, Maria Dominguez, M. L. Uribe
          </td>
          <td>2024-11-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) patients that fail to achieve a complete metabolic response with frontline immunochemotherapy have a poor prognosis. Genomic profiling has led to a broader understanding of the molecular drivers in DLBCL, but it is unknown how well current classifiers identify patients that will experience primary treatment resistance (PTR). Using whole exome and RNA sequencing data from newly diagnosed DLBCL patients, we evaluated the genomic landscape of PTR and compared it to that of non‐PTR DLBCL. We found a significant increase in the frequency of TP53 (34% vs. 15%, p = 0.005) and ARID1A mutations (21% vs. 7%, p = 0.007) in PTR cases, with pathway analysis further demonstrating a downregulation of TP53 and an increase in chromatin modifying pathways. These results suggest that TP53 and ARID1A may be key mediators of PTR and important pathways contributing to the poor outcomes. We found that the current molecular classifiers were unable to identify PTR cases at diagnosis. However, our newly identified high‐risk signature identified 46% of PTR cases at diagnosis. Overall, these results contribute to our understanding of the genomic landscape of patients with primary treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dad9b8db809f309807b25afc6dc6fdb4c12b2eb" target='_blank'>
              Molecular Features of Diffuse Large B‐Cell Lymphoma Associated With Primary Treatment Resistance
              </a>
            </td>
          <td>
            A. M. Bock, K. Wenzl, J. Novak, M. Stokes, M. Hopper, J. Krull, A. Dropik, V. Sarangi, María Ortiz, Nicholas Stong, C. Huang, M. Maurer, Rebecca L. King, U. Farooq, Yucai Wang, T. Witzig, S. Ansell, T. Habermann, J. Cerhan, A. K. Gandhi, Grzegorz Nowakowski, A. Novak
          </td>
          <td>2024-11-29</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="ABSTRACT Background Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment. Methods Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real‐time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts. Results The study comprised 12 patients, all under the age of 18. Post‐treatment RT‐PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR‐129‐5p, and miR‐9‐5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive. Conclusion MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP‐Binding Cassette Transporter Genes and microRNAs in Pediatric B‐Cell ALL: Expression Insights
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of Clinical Laboratory Analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cellular differentiation and homeostasis rely on complex mechanisms to control gene expression, enabling the different cell lineages of an organism to establish and then "memorize" different epigenetic states. The processes that control gene expression are centered on chromatin, a complex of DNA, histone proteins and RNA, whose structure is finely regulated. Targeted epigenomic engineering tools make it possible to interfere with and study these processes, revealing the logic of epigenetic memory mechanisms. This article reviews the main classes of targeted epigenome modification tools and illustrates how they can be used to better understand and modify the epigenome of cells, paving the way for potentially revolutionary therapeutic prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bf48ee21647af051d470ca1e8d40b7d4279ad6" target='_blank'>
              [Targeted epigenome engineering].
              </a>
            </td>
          <td>
            Hedvika Martin, Michel Wassef
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immunomodulatory agents (IMiDs) are a major class of drugs for treating multiple myeloma (MM); however, acquired resistance to IMiDs remains a significant clinical challenge. While alterations in cereblon (CRBN) and its pathway are known to contribute to IMiD resistance, they account for only 20-30% of cases, and the underlying mechanisms in the majority of the resistance cases remain unclear. Here, we identified ADAR1 as a novel driver of lenalidomide resistance in MM. We showed that lenalidomide activates the MDA5-mediated dsRNA-sensing pathway in MM cells, leading to interferon (IFN)-mediated apoptosis, with ADAR1 as the key regulator. Mechanistically, ADAR1 loss increased lenalidomide sensitivity through endogenous dsRNA accumulation, which in turn triggered dsRNA-sensing pathways and enhanced IFN responses. Conversely, ADAR1 overexpression reduced lenalidomide sensitivity, attributed to increased RNA editing frequency, reduced dsRNA accumulation and suppression of the dsRNA-sensing pathways. In summary, we report the involvement of ADAR1-regulated dsRNA-sensing in modulating lenalidomide sensitivity in MM. These findings highlight a novel RNA-related mechanism underlying lenalidomide resistance and underscore the potential of targeting ADAR1 as a novel therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0d2cf7eab8ed9db1ca8ad24d91b4f52aff132ba" target='_blank'>
              ADAR1-regulated cytoplasmic dsRNA-sensing pathway is a novel mechanism of lenalidomide resistance in multiple myeloma.
              </a>
            </td>
          <td>
            Mun Yee Koh, Tae-Hoon Chung, Nicole Xin Ning Tang, S. Toh, Jianbiao Zhou, T. Tan, Leilei Chen, W. Chng, Phaik Ju Teoh
          </td>
          <td>2024-12-09</td>
          <td>Blood</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="In recent years, gene editing technologies have rapidly evolved to enable precise and efficient genomic modification. These strategies serve as a crucial instrument in advancing our comprehension of genetics and treating genetic disorders. Of particular interest is the manipulation of large DNA fragments, notably the insertion of large fragments, which has emerged as a focal point of research in recent years. Nevertheless, the techniques employed to integrate larger gene fragments are frequently confronted with inefficiencies, off-target effects, and elevated costs. It is therefore imperative to develop efficient tools capable of precisely inserting kilobase-sized DNA fragments into mammalian genomes to support genetic engineering, gene therapy, and synthetic biology applications. This review provides a comprehensive overview of methods developed in the past five years for integrating large DNA fragments with a particular focus on burgeoning CRISPR-related technologies. We discuss the opportunities associated with homology-directed repair (HDR) and emerging CRISPR-transposase and CRISPR-recombinase strategies, highlighting their potential to revolutionize gene therapies for complex diseases. Additionally, we explore the challenges confronting these methodologies and outline potential future directions for their improvement with the overarching goal of facilitating the utilization and advancement of tools for large fragment gene editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05aaf5b1a8b083f9c6bd065129f6dea59d703d40" target='_blank'>
              CRISPR/Cas-Based Gene Editing Tools for Large DNA Fragment Integration.
              </a>
            </td>
          <td>
            Shuhan Yang, Guang Hu, Jianming Wang, Jie Song
          </td>
          <td>2024-12-16</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The complex signaling network within the breast tumor microenvironment is crucial for its growth, metastasis, angiogenesis, therapy escape, stem cell maintenance, and immunomodulation. An array of secretory factors and their receptors activate downstream signaling cascades regulating breast cancer progression and metastasis. Among various signaling pathways, the EGFR, ER, Notch, and Hedgehog signaling pathways have recently been identified as crucial in terms of breast cancer proliferation, survival, differentiation, maintenance of CSCs, and therapy failure. These receptors mediate various downstream signaling pathways such as MAPK, including MEK/ERK signaling pathways that promote common pro-oncogenic signaling, whereas dysregulation of PI3K/Akt, Wnt/β-catenin, and JAK/STAT activates key oncogenic events such as drug resistance, CSC enrichment, and metabolic reprogramming. Additionally, these cascades orchestrate an intricate interplay between stromal cells, immune cells, and tumor cells. Metabolic reprogramming and adaptations contribute to aggressive breast cancer and are unresponsive to therapy. Herein, recent insights into the novel signaling pathways operating within the breast TME that aid in their advancement are emphasized and current developments in practices targeting the breast TME to enhance treatment efficacy are reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fb511b18a1724eebed53dc79d3ad283387011" target='_blank'>
              Molecular Insights on Signaling Cascades in Breast Cancer: A Comprehensive Review
              </a>
            </td>
          <td>
            Venketesh K. Panda, Barnalee Mishra, Samikshya Mahapatra, Biswajit Swain, Diksha Malhotra, Suryendu Saha, Sinjan Khanra, Priyanka Mishra, Sambhunath Majhi, Kavita Kumari, Angitha N. Nath, Swarnali Saha, Sarmistha Jena, Gopal C. Kundu
          </td>
          <td>2025-01-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Despite remarkable progress in Pediatric Acute Myeloid Leukemia (pAML) treatments, the relapsed disease remains difficult to treat, making it pertinent to identify novel biomarkers of prognostic/therapeutic significance. Material and methods Bone marrow samples from 21 pAML patients were analyzed using single cell RNA sequencing, functional assays with ARMH1 knockdown and overexpression were performed in leukemia cell lines to evaluate impact on proliferation and migration, and chemotherapy sensitivity. Mitochondrial function was assessed via Seahorse assay, ARMH1 interacting proteins were studied using co-immunoprecipitation. Bulk RNA-seq was performed on ARMH1knockdown and over expressing cell lines to evaluate the pathways and networks impacted by ARMH1. Results Our data shows that ARMH1, a novel cancer-associated gene, is highly expressed in the malignant blast cells of multiple pediatric hematologic malignancies, including AML, T/B-ALL, and T/B-MPAL. Notably, ARMH1 expression is significantly elevated in blast cells of patients who relapsed or have a high-risk cytogenetic profile (MLL) compared to standard-risk (RUNX1, inv (16)). ARMH1 expression is also significantly correlated with the pediatric leukemia stem cell score of 6 genes (LSC6) associated with poor outcomes. Perturbation of ARMH1 (knockdown and overexpression) in leukemia cell lines significantly impacted cell proliferation and migration. The RNA-sequencing analysis on multiple ARMH1 knockdown and overexpressing cell lines established an association with mitochondrial fatty acid synthesis and cell cycle pathways.The investigation of the mitochondrial matrix shows that pharmacological inhibition of a key enzyme in fatty acid synthesis regulation, CPT1A, resulted in ARMH1 downregulation. ARMH1 knockdown also led to a significant reduction in CPT1A and ATP production as well as Oxygen Consumption Rate. Our data indicates that downregulating ARMH1 impacts cell proliferation by reducing key cell cycle regulators such as CDCA7 and EZH2. Further, we also established that ARMH1 is a key physical interactant of EZH2, associated with multiple cancers. Conclusion Our findings underscore further evaluation of ARMH1 as a potential candidate for targeted therapies and stratification of aggressive pAML to improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9c3fb31bae52e61839b92c250ccd940a2b13b6" target='_blank'>
              ARMH1 is a novel marker associated with poor pediatric AML outcomes that affect the fatty acid synthesis and cell cycle pathways
              </a>
            </td>
          <td>
            Mojtaba Bakhtiari, Sean C Jordan, Hope L. Mumme, Richa Sharma, Mala Shanmugam, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Prostate cancer is a malignant tumor caused by the malignant proliferation of epithelial cells, which is highly heterogeneous and drug-resistant, and neuroendocrine prostate cancer (NEPC) is an essential cause of drug resistance in its late stage. Elucidating the evolution of NEPC and the resistance process of enzalutamide, a novel antiandrogen, will be of great help in improving the prognosis of patients. As a research hotspot in the field of molecular biology in recent years, the wide range of biological functions of long noncoding RNAs (lncRNAs) has demonstrated their position in the therapeutic process of many diseases, and a large number of studies have revealed their critical roles in tumor progression and drug resistance. Therefore, elucidating the involvement of lncRNAs in the formation of NEPCs and their interrelationship with enzalutamide resistance may provide new ideas for a deeper understanding of the development of this disease and the occurrence of enzalutamide resistance and give a new direction for reversing the therapeutic dilemma of advanced prostate cancer. This article focuses on lncRNAs that regulate enzalutamide resistance and the neuroendocrine transition of prostate cancer through epigenetic, androgen receptor (AR) signaling, and non-AR pathways that act as “molecular sponges” interacting with miRNAs. Some insights into these mechanisms are used to provide some help for subsequent research in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50743b295e6011754fbae884a2dd3b94e2ad534d" target='_blank'>
              Long noncoding RNA mediates enzalutamide resistance and transformation in neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Zhe Zhu, Wenjing Xuan, Chaohui Wang, Chancan Li
          </td>
          <td>2024-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cellular differentiation is controlled by intricate layers of gene regulation, involving the modulation of gene expression by various transcriptional regulators. Due to the complexity of gene regulation, identifying master regulators across the differentiation trajectory has been a longstanding challenge. To tackle this problem, a computational framework, single-cell Boolean network inference and control (BENEIN), is presented. Applying BENEIN to human large intestinal single-cell transcriptome data, MYB, HDAC2, and FOXA2 are identified as the master regulators whose inhibition induces enterocyte differentiation. It is found that simultaneous knockdown of these master regulators can revert colorectal cancer cells into normal-like enterocytes by synergistically inducing differentiation and suppressing malignancy, which is validated by in vitro and in vivo experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c7fb2686fa44d6df149bd4d9108d57834cc43d" target='_blank'>
              Control of Cellular Differentiation Trajectories for Cancer Reversion.
              </a>
            </td>
          <td>
            Jeong-Ryeol Gong, Chun-Kyung Lee, Hoon-Min Kim, Juhee Kim, Jaeog Jeon, Sunmin Park, Kwang-Hyun Cho
          </td>
          <td>2024-12-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer is a complex and multifaceted group of diseases characterized by highly intricate mechanisms of tumorigenesis and tumor progression, which complicates diagnosis, prognosis, and treatment. In recent years, targeted therapies have gained prominence by focusing on specific mutations and molecular features unique to each tumor type, offering more effective and personalized treatment options. However, it is equally critical to explore the genetic commonalities across different types of cancer, which has led to the rise of pan-cancer studies. These approaches help identify shared therapeutic targets across various tumor types, enabling the development of broader and potentially more widely applicable treatment strategies. This review aims to provide a comprehensive overview of key concepts related to tumors, including tumorigenesis processes, the tumor microenvironment, and the role of extracellular vesicles in tumor biology. Additionally, we explore the molecular interactions and mechanisms driving tumor progression, with a particular focus on the pan-cancer perspective. To achieve this, we conducted an in silico analysis using publicly available datasets, which facilitated the identification of both common and divergent genetic and molecular patterns across different tumor types. By integrating these diverse areas, this review offers a clearer and deeper understanding of the factors influencing tumorigenesis and highlights potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5b3ea0ad38a337f0df54ee493320c36455e356" target='_blank'>
              Exploring the Complexity of Pan-Cancer: Gene Convergences and in silico Analyses
              </a>
            </td>
          <td>
            Leandro Teodoro, Ana Claudia Carreira, M.C. Sogayar
          </td>
          <td>2024-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103b75e315d591fa4510717dd10f578400ede160" target='_blank'>
              Central control of dynamic gene circuits governs T cell rest and activation.
              </a>
            </td>
          <td>
            Maya M. Arce, Jennifer M. Umhoefer, Nadia Arang, Sivakanthan Kasinathan, Jacob W. Freimer, Zachary Steinhart, Haolin Shen, Minh T N Pham, Mineto Ota, Anika Wadhera, Rama Dajani, Dmytro Dorovskyi, Y. Chen, Qi Liu, Yuan Zhou, D. Swaney, K. Obernier, B. Shy, J. Carnevale, Ansuman T. Satpathy, N. Krogan, Jonathan K. Pritchard, Alex Marson
          </td>
          <td>2024-12-11</td>
          <td>Nature</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92776a13bf2bfc07c4e1f9b75c01119340528b43" target='_blank'>
              Elevated mutation rates in multi-azole resistant Aspergillus fumigatus drive rapid evolution of antifungal resistance
              </a>
            </td>
          <td>
            Michael J. Bottery, N. van Rhijn, H. Chown, J. Rhodes, Brandi N. Celia-Sanchez, Marin T. Brewer, M. Momany, Matthew C. Fisher, Christopher G. Knight, M. Bromley
          </td>
          <td>2024-12-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7fa17db238d722caee80a374931a8a07aedd8d3" target='_blank'>
              ASXL1 truncating variants in BOS and myeloid leukemia drive shared disruption of Wnt-signaling pathways but have differential isoform usage of RUNX3
              </a>
            </td>
          <td>
            Isabella Lin, Zain Awamleh, Mili Sinvhal, Andrew Wan, Leroy Bondhus, Angela Wei, Bianca E Russell, Rosanna Weksberg, VA Arboleda
          </td>
          <td>2024-11-29</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer stem cells are characterized by self-renewal, clonal tumor initiation capacity, and treatment resistance, which play essential roles in the tumor progression of prostate cancer (PCa). In this study, we aim to explore the features of cancer stemness and characterize the expression of stem cell-related genes for PCa. Methods We downloaded RNA-seq data and related clinical information from The Cancer Genome Atlas (TCGA) database. The mRNA stemness index (mRNAsi) was analyzed for various clinical features, overall survival (OS), and disease-free survival (DFS), and a weighted gene co-expression network analysis (WGCNA) was performed to identify crucial gene modules and key genes, which may play a role in CSCs. The key gene functions were verified using multiple databases, including the TCGA and Gene Expression Omnibus database (GEO). Next, we explored the potential function of the modules and genes obtained using WGCNA using an enrichment analysis. Finally, we performed in vitro experiments for further verification. Results We found that mRNAsi were higher in PCa tissues than in normal tissues, and the mRNAsi were closely related to the clinical characteristics of PCa. A total of 16 key genes associated with the mRNAsi scores were identified by WGCNA analysis, including NCAPG, NEK2, DLGAP5, CENPA, CENPF, TPX2, GTSE1, KIF4A, NEIL3, CDC25C, UBE2C, CDCA5, MELK, SKA3, NUF2, and BIRC5. These genes were explicitly highly expressed in PCa across TCGA cancers and were validated in 3 independent GEO PCa datasets. The functional annotations of the key genes were linked with the cell proliferation processes. NUF2 may be a potential biomarker for PCa. In vitro experiments showed that knockdown NUF2 reduced the proliferation and migration of PCa cells. Conclusion The 16 key genes identified in this study significantly correlate with PCa stem cell characteristics and showed prognosis-oriented effects in PCa patients. Further, the NUF2 gene may be used as a drug target for treating PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc9466fa18820a99adb15c098ec91082dee0a3" target='_blank'>
              NUF2 is associated with cancer stem cell characteristics and a potential drug target for prostate cancer
              </a>
            </td>
          <td>
            Dongxu Zhang, Pu Liang, Qi Wang, Bowen Xia, Liqian Yu, Xiaopeng Hu
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human pathogen Streptococcus pneumoniae forms multiple epigenetically and phenotypically distinct intra-populations by invertase PsrA-driven inversions of DNA methyltransferase hsdS genes in the colony opacity-determinant (cod) locus. As manifested by phase switch between opaque and transparent colonies, different genome methylation patterns or epigenomes confer pathogenesis-associated traits, but it is unknown how the pathogen controls the hsdS inversion orientations. Here, we report our finding of the SpxA1-TenA toxin-antitoxin (TA) system that regulates the orientations of hsdS inversions, and thereby bacterial epigenome and associated traits (e.g., colony opacity) by targeting pneumococcal protein synthesis. SpxA1 and TenA were found to constitute a highly conserved type II TA system in S. pneumoniae, primarily based on the observation that overexpressing toxin TenA led to growth arrest in E. coli and enhanced autolysis in S. pneumoniae, and the antitoxin SpxA1 repressed the transcription of the spxA1-tenA operon. When the transcription of tenA was de-repressed by a spontaneous AT di-nucleotide insertion/deletion in the promoter region of the spxA1-tenA operon, TenA bound to the ribosome maturation factor RimM, and thereby reduced the cellular level of alternative sigma factor ComX (known for the activation of natural transformation-associated genes). Attenuation of ComX expression in turn enhanced the transcription of the invertase gene psrA, which favored the formation of the transparent colony phase-associated hsdS allelic configurations in the cod locus. Phenotypically, moderate expression of TenA dramatically reshaped pneumococcal epigenome and colony opacity. Because spontaneous variations frequently occur during bacterial growth in the number of the AT di-nucleotides in the promoter region of the spxA1-tenA operon, this locus acts as a programmed genetic switch that generates pneumococcal subpopulations with epigenetic and phenotypic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd33d7ab497299e98fba78410719174886a039f7" target='_blank'>
              The SpxA1-TenA toxin-antitoxin system regulates epigenetic variations of Streptococcus pneumoniae by targeting protein synthesis
              </a>
            </td>
          <td>
            Shaomeng Wang, Xiu-Yuan Li, Mengran Zhu, Haiteng Deng, Juanjuan Wang, Jing-Ren Zhang
          </td>
          <td>2024-12-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has increased at an alarming rate amongst younger (< 50 years) individuals. Such early-onset colorectal cancer (EOCRC) has been particularly notable within the Hispanic/Latino population. Yet, this population has not been sufficiently profiled in terms of two critical elements of CRC -- the MYC proto-oncogene and WNT signaling pathway. Here, we performed a comprehensive multi-omics analysis on 30 early-onset and 37 late-onset CRC ([≥] 50 years) samples from Hispanic/Latino patients. Our analysis included DNA exome sequencing for somatic mutations, somatic copy number alterations, and global and local genetic similarity. Using RNA sequencing, we also assessed differential gene expression, cellular pathways, and gene fusions. We then compared our findings from early-onset Hispanic/Latino patient samples with publicly available data from Non-Hispanic White cohorts. Across all early-onset patients, which had a median 1000 Genomes Project Peruvian-in-Lima-like (1KG-PEL-like) genetic similarity proportion of 60%, we identified 41 WNT pathway genes with significant mutations. Six important examples were APC, TCF7L2, DKK1, DKK2, FZD10, and LRP5. Notably, patients with mutations in DKK1 and DKK2 had the highest 1KG-PEL-like proportion (79%). When we compared the Hispanic/Latino cohort to the Non-Hispanic White cohorts, four of these key genes -- DKK1, DKK2, FZD10, and LRP5 -- were significant in both risk association analyses and differential gene expression. Interestingly, early-onset tumors (vs. late-onset) exhibited distinct somatic copy number alterations and gene expression profiles; the differences included MYC and drug-targetable WNT pathway genes. We also identified a novel WNT gene fusion, RSPO3, in early-onset tumors; it was associated with enhanced WNT signaling. This integrative analysis underscores the distinct molecular features of EOCRC cancer in the Hispanic/Latino population; reveals potential avenues for tailored precision medicine therapies; and emphasizes the importance of multi-omics approaches in studying colorectal carcinogenesis. We expect this data to help contribute towards reducing cancer health disparities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0668a8b4dd0379f87ebaa20ad3d4dabb63bc8" target='_blank'>
              Assessment of MYC Gene and WNT Pathway Alterations in Early-Onset Colorectal Cancer Among Hispanic/Latino Patients Using Integrated Multi-Omics Approaches
              </a>
            </td>
          <td>
            F. Carranza, B. Waldrup, Y. Yin, Y. Amzaleg, M. Postel, D. Craig, J. Carpten, B. Salhia, D. Hernandez, N. Gutierrez, C. Ricker, J. Culver, C. Chavez, M. Stern, L. Baezconde-Garbanati, H.-J. Lenz, E. Velazquez-Villarreal
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The concept of genetic code expansion (GCE) has revolutionized the field of chemical and synthetic biology, enabling the site-specific incorporation of noncanonical amino acids (ncAAs) into proteins, thus opening new avenues in research and applications across biology and medicine. In this review, we cover the principles of GCE, including the optimization of the aminoacyl-tRNA synthetase (aaRS)/tRNA system and the advancements in translation system engineering. Notable developments include the refinement of aaRS/tRNA pairs, enhancements in screening methods, and the biosynthesis of noncanonical amino acids. The applications of GCE technology span from synthetic biology, where it facilitates gene expression regulation and protein engineering, to medicine, with promising approaches in drug development, vaccine production, and gene editing. The review concludes with a perspective on the future of GCE, underscoring its potential to further expand the toolkit of biology and medicine. Through this comprehensive review, we aim to provide a detailed overview of the current state of GCE technology, its challenges, opportunities, and the frontier it represents in the expansion of the genetic code for novel biological research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8f20128f59397264338dcf03afc1ecef23ea9" target='_blank'>
              Genetic Code Expansion: Recent Developments and Emerging Applications.
              </a>
            </td>
          <td>
            Yujia Huang, Pan Zhang, Haoyu Wang, Yan Chen, Tao Liu, Xiaozhou Luo
          </td>
          <td>2024-12-31</td>
          <td>Chemical reviews</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0b020550b9e80b754c00294f225c54a57ba237" target='_blank'>
              Epigenetics-targeted drugs: current paradigms and future challenges
              </a>
            </td>
          <td>
            Wanlin Dai, Xinbo Qiao, Yuanyuan Fang, Renhao Guo, Peng Bai, Shuang Liu, Tingting Li, Yutao Jiang, Shuang Wei, Zhijing Na, Xue Xiao, Da Li
          </td>
          <td>2024-11-26</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5–34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6b74de7021622cacaa229773949fb0310d8a9a" target='_blank'>
              A practice-oriented genome-profiling study for acute myeloid leukemia using the novel HANDLE system: HM-screen-JAPAN02.
              </a>
            </td>
          <td>
            H. Arai, N. Hosono, Sung-Gi Chi, Kentaro Fukushima, Daisuke Ikeda, Satoshi Iyama, Akihiko Gotoh, Takayuki Ikezoe, Chikashi Yoshida, Goichi Yoshimoto, Junya Kanda, Naoto Takahashi, E. Sakaida, Kensuke Usuki, Takahiro Yamauchi, Yosuke Minami
          </td>
          <td>2024-12-16</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Oncogenes are proteins promoting cancer formation. Cancers caused by mutations in the RAS oncogene have been a major focus of cancer research for decades. Despite better understanding of how the RAS oncogene works, finding an effective way to treat RAS-driven cancers remains a challenge. This review explores how RAS affects DNA replication and genome stability. It also describes how RAS can lead to inflammatory responses and senescence. Finally, we discuss how these molecular insights have led to potential new lines of treatments to target RAS-mutated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb79941bb5aaefe1e6a485ef57471bc907fec75" target='_blank'>
              Oncogenic RAS in Cancers from the DNA Replication Stress and Senescence Perspective
              </a>
            </td>
          <td>
            Hervé Técher, Samira Kemiha, Xieraili Aobuli, A. M. Kolinjivadi
          </td>
          <td>2024-11-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c580172e55394858107b5db74bf7f577e6d7f63" target='_blank'>
              DNMT1 prolonged absence is a tunable cellular stress that triggers cell proliferation arrest to protect from major DNA methylation loss
              </a>
            </td>
          <td>
            Salvatore Martino, Serena Gargano, P. Carollo, A. Di Leonardo, V. Barra
          </td>
          <td>2024-12-18</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e14091bb37f5209a2b92498963038115c6336e4" target='_blank'>
              Epigenetic dysregulation in glioblastoma: potential pathways to precision medicine.
              </a>
            </td>
          <td>
            Vijeta Prakash, R. Gabrani
          </td>
          <td>2024-11-25</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7ecb008360a3e79fcfe183be595899effbacee" target='_blank'>
              N6-methyladenosine RNA modification in head and neck squamous cell carcinoma (HNSCC): current status and future insights.
              </a>
            </td>
          <td>
            Pramodha Janakiraman, Jayasree Peroth Jayaprakash, Sridhanya Velayudham Muralidharan, Kumar Pranav Narayan, P. Khandelia
          </td>
          <td>2024-11-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Discovered more than four decades ago, nonsense-mediated mRNA decay (NMD) plays a fundamental role in the regulation of gene expression and is a major contributor to numerous diseases. With advanced technologies, several novel approaches aim to directly circumvent the effects of disease-causing frameshift and nonsense mutations. Additional therapeutics aim to globally dampen the NMD pathway in diseases associated with pathway hyperactivation, one example being Fragile X Syndrome. In other cases, therapeutics have been designed to hijack or inhibit the cellular NMD machinery to either activate or obviate transcript-specific NMD by modulating pre-mRNA splicing. Here, we discuss promising approaches employed to regulate NMD for therapeutic purposes and highlight potential challenges in future clinical development. We are optimistic that the future of developing target-specific and global modulators of NMD (inhibitors as well as activators) is bright and will revolutionize the treatment of many genetic disorders, especially those with high unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8d79dce5cfe95746b7b429088778df04c8485b" target='_blank'>
              Exploring the therapeutic potential of modulating nonsense-mediated mRNA decay.
              </a>
            </td>
          <td>
            Mary McMahon, L. Maquat
          </td>
          <td>2024-12-12</td>
          <td>RNA</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Philadelphia chromosome-like B-cell acute lymphoblastic leukemia (Ph-like ALL) is driven by genetic alterations that induce constitutive kinase signaling and is associated with chemoresistance and high relapse risk in children and adults. Preclinical studies in the most common CRLF2-rearranged/JAK pathway-activated Ph-like ALL subtype have shown variable responses to JAK inhibitor-based therapies, suggesting incomplete oncogene addiction and highlighting a need to elucidate alternative biologic dependencies and therapeutic vulnerabilities, while the ABL-class Ph-like ALL subtype appears preferentially sensitive to SRC/ABL- or PDGFRB-targeting inhibitors. Which patients may be responsive versus resistant to tyrosine kinase inhibitor (TKI)-based precision medicine approaches remains a critical knowledge gap. Using bulk and single-cell multiomics analyses, we profiled residual cells from CRLF2-rearranged or ABL1-rearranged Ph-like ALL patient-derived xenograft models treated in vivo with targeted inhibitors to identify TKI-resistant subpopulations and potential mechanisms of therapeutic escape. We detected a specific MYC dependency in Ph-like ALL cells and defined a new leukemia cell subpopulation with senescence-associated stem cell-like features regulated by AP-1 transcription factors. This dormant ALL subpopulation was effectively eradicated by dual pharmacologic inhibition of BCL-2 and JAK/STAT or SRC/ABL pathways, a clinically-relevant therapeutic strategy. Single cell-derived molecular signatures of this senescence and stem/progenitor-like subpopulation further predicted poor clinical outcomes associated with other high-risk genetic subtypes of childhood B-ALL and thus may have broader prognostic applicability beyond Ph-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b87bb41065d73fc1bc25a49641c54717955deab" target='_blank'>
              Targeting Senescent Stemlike Subpopulations in Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Yang-yang Ding, Jonathan H Sussman, Kellyn Madden, J. Loftus, Robert K Chen, C. D. Falkenstein, D. A. Barcenas Lopez, David A Hottman, Benjamin Mathier, Wenbao Yu, Jason Xu, Changya Chen, Chia-Hui Chen, Bing He, S. Bandyopadhyay, Zhan Zhang, DongGeun Lee, Hong Wang, Junmin Peng, Chi V. Dang, Kai Tan, S. Tasian
          </td>
          <td>2025-01-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5f657e821743bf515a8df904c0de712f1e97ea" target='_blank'>
              Understanding the biological processes of kidney carcinogenesis: an integrative multi-omics approach
              </a>
            </td>
          <td>
            Ricardo Cortez Cardoso Penha, Alexandra Sexton Oates, S. Senkin, Hanla A. Park, Joshua Atkins, I. Holcatova, A. Hornakova, S. Savic, Simona Ognjanovic, Beata Świątkowska, J. Lissowska, D. Zaridze, A. Mukeria, V. Janout, Amélie Chabrier, Vincent Cahais, C. Cuenin, G. Scelo, M. Foll, Z. Herceg, Paul Brennan, K. Smith-Byrne, N. Alcala, James D. McKay
          </td>
          <td>2024-11-26</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Abstract Youthful gene expression and function can be restored in old cells by brief activation of transcription factors (TFs), a process known as partial reprogramming. Partial reprogramming holds promise as a therapeutic approach, but current methods have only employed a small set of pluripotency TFs that also reprogram cell identity in undesirable ways. It remains unknown if other TFs might be able to reprogram cell age, how long these effects last, and how often TFs act in concert to promote youthful phenotypes. To address these questions, we developed a screening technology called Partial-seq to measure the effect of partial reprogramming on gene expression across thousands of TF sets at once. We used Partial-seq to screen >4000 TF sets in aged human T cells, a keystone cell type of the immune system that declines in function with age. We discovered that many TFs beyond pluripotency factors restored youthful gene expression and cell states, without abolishing T cell identity. TF combinations were more likely than individual factors to reverse cell age, and many of these effects persisted after TF activation ceased. Our results suggest that the phenotypic age of human T cells can be durably reversed by partial reprogramming with diverse TFs. The diversity of TF sets discovered in our screens suggests that reprogramming cell age might be accomplished without perturbing cell identity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fce7d6f2ff1adcedf00e09609177688f7d0c53" target='_blank'>
              PARTIAL REPROGRAMMING REVERSES FEATURES OF AGING IN HUMAN T LYMPHOCYTES
              </a>
            </td>
          <td>
            Jacob Kimmel
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses.
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9efc51b7707d06e4363cf205a25f12d35ff74015" target='_blank'>
              Coupling cellular drug-target engagement to downstream pharmacology with CeTEAM
              </a>
            </td>
          <td>
            Nicholas C. K. Valerie, Kumar Sanjiv, O. Mortusewicz, S. Zhang, Seher Alam, Maria J. Pires, Hannah Stigsdotter, A. Rasti, M. Langelier, D. Rehling, Adam Throup, Oryn Purewal-Sidhu, M. Desroses, Jacob Onireti, Prasad Wakchaure, I. Almlöf, J. Boström, Luka Bevc, Giorgia Benzi, P. Stenmark, J. Pascal, T. Helleday, B. Page, M. Altun
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>82</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most prevalent causes of cancer-related morbidity and mortality worldwide. Late-stage detection and the complex molecular mechanisms driving tumor progression contribute significantly to its poor prognosis. Dysregulated R-loops, three-stranded nucleic acid structures associated with genome instability, play a key role in the malignant characteristics of various tumors. However, the detailed role and mechanism of R-loops in HCC progression remain elusive and require further exploration. This study aimed to construct an R-loop scoring signature centered on prognosis and lipid metabolism, thereby enhancing our understanding of HCC progression and identifying potential therapeutic targets.In this study, we utilized the single-cell RNA-sequencing (scRNA-seq) data from HCC patients (GSE149614 and CRA002308) to construct an R-loop scoring model based on the identified R-loop regulator genes (RLRGs) related to HBV infection through WGCNA analysis. We also explored the tumor microenvironment and intercellular communication related to R-loop score. Additionally, a prognostic risk model based on the fatty acid metabolism-associated RLRGs was constructed using data from the TCGA database, and its association with immune infiltration, mutations, and drug sensitivity was analyzed. In vitro and in vivo experiments were performed to investigate the role of RLRG CLTC in lipid metabolism and HCC progression.Using scRNA-seq data from HCC, we established an R-loop scoring model based on identified RLRGs related to HBV infection. Moreover, the more suppressive tumor immune microenvironment and stronger intercellular communication were displayed in malignant cells with high R-loop scores. The cell trajectory and cellular metabolism analysis exhibited a significant association between lipid metabolism and RLRGs. Additionally, we constructed a prognostic risk model consisting of 8 RLRGs related to fatty acid metabolism, which effectively evaluated the prognostic value, status of tumor immune microenvironment, gene mutations, and chemotherapeutic drug sensitivity for HCC patients. Notably, validation experiments suggested that CLTC could regulate lipid metabolism through R-loop formation and facilitate tumor progression in HCC.Collectively, our study proposes an R-loop scoring model associated with tumor immune microenvironment, lipid metabolism and prognostic value. CLTC, an R-loop regulator, emerges as a promising prognostic biomarker and therapeutic target, offering new insights into potential treatment strategies for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cbaecb70e4d3bf1b08fe7b03162bc2296819cdc" target='_blank'>
              Integrated single-cell and bulk transcriptome analysis of R-loop score-based signature with regard to immune microenvironment, lipid metabolism and prognosis in HCC
              </a>
            </td>
          <td>
            Long Chen, Houzhi Yang, Xianfu Wei, Jianchao Liu, Xiuxin Han, Chao Zhang, Yongheng Liu, Yan Zhang, Yao Xu, Yiqin Li, Guowen Wang, Jinyan Feng
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f23f58ff8f0c69ee80ac960c4ff630df39788c5" target='_blank'>
              LGR5: An emerging therapeutic target for cancer metastasis and chemotherapy resistance.
              </a>
            </td>
          <td>
            W. Wang, N. Lokman, Simon C Barry, M. Oehler, C. Ricciardelli
          </td>
          <td>2025-01-17</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Microtubules play essential roles in diverse cellular processes and are important pharmacological targets for treating human disease. Here, we sought to identify cellular factors that modulate the sensitivity of cells to antimicrotubule drugs. We conducted a genome-wide CRISPR/Cas9-based functional genetics screen in human cells treated with the microtubule-destabilizing drug nocodazole or the microtubule-stabilizing drug paclitaxel. We further conducted a focused secondary screen to test drug sensitivity for ∼1,400 gene targets across two distinct human cell lines and to additionally test sensitivity to the KIF11 inhibitor, STLC. These screens defined gene targets whose loss enhances or suppresses sensitivity to antimicrotubule drugs. In addition to gene targets whose loss sensitized cells to multiple compounds, we observed cases of differential sensitivity to specific compounds and differing requirements between cell lines. Our downstream molecular analysis further revealed additional roles for established microtubule-associated proteins and identified new players in microtubule function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe92d3ab4eb0f4d2bf377007f003eebe7871111" target='_blank'>
              Functional genetics reveals modulators of antimicrotubule drug sensitivity.
              </a>
            </td>
          <td>
            Kuan-Chung Su, Elena Radul, Nolan K Maier, Mary-Jane Tsang, Claire Goul, Brittania Moodie, Océane Marescal, Heather R Keys, I. Cheeseman
          </td>
          <td>2024-11-21</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies.We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process.Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway.Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Antimicrobial resistance is a threat to public health with millions of deaths linked to drug resistant infections every year. To mitigate resistance, common strategies that are used are combination therapies and therapy switching. However, the stochastic nature of pathogenic mutation makes the optimization of these strategies challenging. Here, we propose a two-scale stochastic model that considers the effective evolution of therapies in a multidimensional efficacy space, where each dimension represents the efficacy of a specific drug in the therapy. The diffusion of therapies within this space is subject to stochastic resets, representing therapy switches. The boundaries of the space, inferred from coarser pathogen-host dynamics, can be either reflecting or absorbing. Reflecting boundaries impede full recovery of the host, while absorbing boundaries represent the development of antimicrobial resistance, leading to therapy failure. We derive analytical expressions for the average absorption times, accounting for both continuous and discrete genomic changes using the frameworks of Langevin and Master equations, respectively. These expressions allow us to evaluate the relevance of times between drug-switches and the number of simultaneous drugs in relation to typical timescales for drug resistance development. We also explore realistic scenarios where therapy constraints are imposed to the number of administered therapies and/or their costs, finding non-trivial optimal drug-switching protocols that maximize the time before antimicrobial resistance develops while reducing therapy costs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9354b874f9656a724261081aae3fb63a9b25217d" target='_blank'>
              Optimal switching strategies in multi-drug therapies for chronic diseases
              </a>
            </td>
          <td>
            J. Magalang, Javier Aguilar, J. Esguerra, 'Edgar Rold'an, Daniel Sanchez-Taltavull
          </td>
          <td>2024-11-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mutations in the TP53 tumor suppressor genes are prevalent in aggressive cancers. Pharmacological reactivation of dysfunctional p53 due to mutations is a promising strategy for treating such cancers. Recently, a multifunctional proline- and glutamine-rich protein, PTB-associated splicing factor (PSF), was identified as a key driver of aggressive cancers. PSF promotes the expression of numerous oncogenes by modulating epigenetic and splicing mechanisms. We previously screened a small-molecule library and discovered compound No.10-3 as a potent PSF inhibitor. Here, we report the discovery of a No.10-3 analog, C-30, as a potent PSF inhibitor. Compared to No.10-3, C-30 treatment specifically suppressed the growth and induced apoptosis of mutant p53-bearing and therapy-resistant cancer cells. Interestingly, C-30 activated a set of p53-regulated genes in therapy-resistant cancer cells. A comprehensive analysis of PSF and p53 binding regions demonstrated a higher level of PSF-binding potential in mutant p53-expressing cancer cells around genomic regions identified as p53-binding peaks in p53-wild type cancer cells. Treatment of mutant p53-expressing cancer cells with C-30 decreases PSF binding around these sites, leading to activated histone acetylation. We further demonstrated that C-30 impaired tumor growth and increased the expression of p53-target genes in vivo. These results suggested that C-30 produces tumor-suppressive effects similar to the functional reactivation of p53, providing a rationale for the inhibition of PSF activity as a promising therapy against treatment-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a883a5ac34ea9851ff495a97f5000312eee25b0f" target='_blank'>
              Inhibition of PSF activity overcomes resistance to treatment in cancers harboring mutant p53.
              </a>
            </td>
          <td>
            Ken‐ichi Takayama, Tomohiro Sato, Teruki Honma, Minoru Yoshida, Satoshi Inoue
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The integrated stress response (ISR) is a conserved eukaryotic signaling pathway that responds to diverse stress stimuli to restore proteostasis. The strength and speed of ISR activation must be tuned properly to allow protein synthesis while maintaining proteostasis. Here, we describe how genetic perturbations change the dynamics of the ISR in budding yeast. We treated ISR dynamics, comprising timecourses of ISR activity across different levels of stress, as a holistic phenotype. We profiled changes in ISR dynamics across thousands of genetic perturbations in parallel using CRISPR interference with barcoded expression reporter sequencing (CiBER-seq). We treated cells with sulfometuron methyl, a titratable inhibitor of branched-amino acid synthesis, and measured expression of an ISR reporter. Perturbations to translation such as depletion of aminoacyl-tRNA synthetases or tRNA biogenesis factors reduced cell growth and caused a strikingly proportionate activation of the ISR activation. In contrast, impaired ribosome biogenesis reduced basal ISR activity and weakened ISR dynamics. Reduced ribosome capacity may lower the demand for amino acids and thereby explain these changes. Our work illustrates how CiBER-seq enables high-throughput measurements of complex and dynamic phenotypes that shed light on adaptive and homeostatic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5725328bae6e97ada63ca77f5c500e9dd0a3db19" target='_blank'>
              Mapping the Genetic Architecture of the Adaptive Integrated Stress Response in S. cerevisiae
              </a>
            </td>
          <td>
            Rachel Baum, Jinyoung Kim, Ryan Y. Muller, Nicholas T. Ingolia
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Metabolic reprogramming is a hallmark of cancer, including alterations in the hexosamine biosynthesis pathway (HBP). Glutamine-fructose-6-phosphate transaminase 1 (GFPT1) is the key regulatory enzyme in the HBP; however, its role in invasive breast carcinoma remains underexplored. Methods This study utilized integrated data from The Cancer Genome Atlas (TCGA) to assess GFPT1 expression in breast cancer (BRCA) patients. Functional enrichment and mutational landscape analyses were performed, along with chemosensitivity predictions. In vitro experiments were conducted by silencing GFPT1 in malignant breast epithelial cells to evaluate changes in proliferation, migration, and apoptosis. Results Elevated GFPT1 expression was linked to advanced-stage breast cancer and identified as an independent prognostic marker for overall survival (OS). High GFPT1 levels were associated with increased cytoplasmic translation, activation of oncogenic pathways, and infiltration of M2 macrophages. The GFPT1-High group also showed a higher mutational burden, with frequent TP53 mutations. Chemosensitivity analysis revealed increased IC50 values for chemotherapy drugs in this group. GFPT1 silencing led to reduced cell proliferation and migration, along with enhanced apoptosis. Conclusion These findings indicate that GFPT1 is a novel prognostic biomarker and a predictive indicator of chemotherapy response in invasive breast carcinoma. GFPT1 influences mRNA translation, cell cycle regulation, and M2 macrophage infiltration, thereby promoting cancer cell proliferation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6655d004010dcffbbdb0a2efe7ca8624f1e53e" target='_blank'>
              Bioinformatics insights into the role of GFPT1 in breast invasive carcinoma: implications for tumor prognosis, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Jianghui Liang, Xiaolian Deng, Yingyi Zhang, Tianchi Fei, Muzi Ouyang, Chengjie Yu, Yang Xiang, Dongwei Jia, Fangfang Duan
          </td>
          <td>2024-11-22</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Autism Spectrum Disorder (ASD) is a complex and heterogeneous disorder, which has been reported affected by many different causative genes. In recent years, single-nucleus RNA sequencing (snRNA-seq) have been widely used to reveal variation in gene expression across cell types during the pathology of ASD. However, the vast majority of current studies only consider information on the single modality of gene expression profile obtained from snRNA-seq. Recent bioinformatics studies have successfully captured key transcriptomic information on alternative polyadenylation (APA) from single-cell RNA-seq (scRNA-seq) data to reveal APA dynamics between cell types. Here we integrated multimodal information of gene expression and APA to gain a more comprehensive understanding of the molecular mechanisms of ASD. APA sites were first identified at the single cell level in patients with ASD and healthy populations under different cell types. Gene features were selected by two unsupervised methods using profiles of APA usages and gene expression, respectively. Then these gene features were integrated to build a cell-type-specific classification model based on multiview privileged support vector machines to predict whether a cell is from a ASD patient or a healthy control. With this model, the contribution of an individual modality was calculated, and genes from different modalities were ranked by importance score to identify high-risk genes that are highly associated with ASD. The functions of these high-risk genes were found to be associated with ASD-related biological functions and significantly different across cell types, highlighting the cell-type-specificity of ASD. These high confidence risk genes can serve as potential biomarkers for the diagnosis and treatment of ASD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12cf9da755d59daf8c7da747343466f69076be65" target='_blank'>
              Identification of cell-type-specific genes associated with autism by integrating gene expression and alternative polyadenylation profiles
              </a>
            </td>
          <td>
            Xingyu Bi, Liping Kang, Xiaohui Wu
          </td>
          <td>2024-12-03</td>
          <td>2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the precise mechanisms by which DDAs induce cell death are not fully understood and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with the sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and downregulating tRNALeu(TAA). Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), leading to ER stress and subsequent cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11 knockout or transfection of tRNALeu(TAA). Our proteomic analysis suggests that tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells, and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs, and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811a0f920ef5bd12afa937061ab8c997f4e1a5be" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, Atsuo T. Sasaki, Akio Kanai
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.


METHODS
In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.   Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.


CONCLUSIONS
Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling in the diagnosis of childhood acute myeloid leukemia: a single center experience.
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish journal of pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Horizontal gene transfer (HGT) is a major driver of bacterial evolution, allowing bacteria to acquire new traits from other organisms. It enables the transfer of advantageous traits, such as antibiotic resistance, which can rapidly spread across bacterial populations, affecting public health. Several studies have estimated how widespread HGT can be and suggested that environmental changes are associated with changes in the rate of HGT. It is often considered that environmental changes lead to an increase in HGT rates due to the adaptive benefits of HGT. However, previous studies have been limited to one or a few species and have often focused on specific genes of interest. Here, we systematically investigate the effect of environmental changes on the rate of HGT in a large number of bacterial genomes along a deep phylogeny. We extend the scope of use of existing state-of -the -art methods for HGT inference to infer the transition into new ecosystems and the gain of pathogenicity. Transitions to new environments are associated with a decrease in the rate of HGT, along with the strengthening of purifying selection. This decrease in HGT rates is consistent regardless of genomic relatedness between the donor and recipient. Our results are robust to the use of different methodological frameworks and have implications for understanding the evolution of bacteria and HGT rates in natural microbial communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509a20de1480698aa85c82ec7a8080e8f597bcfe" target='_blank'>
              Horizontal gene transfer rates decrease during transitions to new ecosystems
              </a>
            </td>
          <td>
            Swastik Mishra, Martin J. Lercher
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2962c853fd82d5be105d8daa9d66daafd430b25e" target='_blank'>
              Elongation factor 2 in cancer: a promising therapeutic target in protein translation
              </a>
            </td>
          <td>
            X. Jia, Chuntian Huang, Fangfang Liu, Zigang Dong, Kangdong Liu
          </td>
          <td>2024-12-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Introduction Hypoxia responses are critical for myriad physiological and pathological processes, such as development, tissue repair, would healing, and tumorigenesis. microRNAs (miRNAs) are a class of small non-coding RNAs that exert their functions by inhibiting the expression of their target genes, and miR-210 is the miRNA universally and most conspicuously upregulated by hypoxia in mammalian systems. For its relationship to hypoxia, miR-210 has been studied extensively, yet no consensus exists on the roles and mechanisms of miR-210 in human physiological processes or diseases, and we know little about genuine miR-210 target genes in humans. Methods To better investigate the functions and mechanisms of human miR-210, therefore, we derived the human miR-210 gene knockout (KO) 293T cell lines using the CRISPR/Cas9 technology. We then examined the cellular phenotypes and gene expression profiles of 293T cells under normoxia and hypoxia conditions. Results and Discussion We found that the loss of miR-210 altered a variety of cellular phenotypes including proliferation and apoptosis. Subsequent global gene expression analyses identified plausible mechanisms underlying these phenotypic changes in 293T cells. In particular, we showed that miR-210 might target the expression of BNIP3L as a potential mechanism to suppress apoptosis. Surprisingly, the mRNA levels of most previously reported miR-210 target genes were not induced upon miR-210 KO, suggesting a need to reexamining and studying human miR-210 functions directly and comprehensively. Thus, our work established a human cellular system and opportunity to unravel the complexity of the regulatory networks by miR-210.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de9fca0fc73bea5e4d2b5e6bb3d58b2f4b8c51b" target='_blank'>
              miR-210 loss leads to widespread phenotypic and gene expression changes in human 293T cells
              </a>
            </td>
          <td>
            Xiaoxiao Zhang, Zhen Meng, Chengyong Yang, Chenghao Wang, Kexin Zhang, Anxin Shi, Jingjing Guo, Yong Feng, Yan Zeng
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary In addition to being essential for DNA replication and repair, proliferating cell nuclear antigen (PCNA) has recently been linked to regulation of signalling, metabolism and apoptosis. As PCNA is a new and unexplored target for cancer treatment, we have here investigated the regulatory role of PCNA in 10 multiple myeloma (MM) cell lines using a multi-omics approach and a PCNA-targeting peptide, ATX-101, currently in clinical development. We have found that ATX-101’s efficacy is linked to PCNA’s role in regulation of proteasomal and ER stress in MM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84301979821512ede666f571de223ed396c3f64" target='_blank'>
              Multiple Myeloma Cells with Increased Proteasomal and ER Stress Are Hypersensitive to ATX-101, an Experimental Peptide Drug Targeting PCNA
              </a>
            </td>
          <td>
            Camilla Olaisen, L. M. Røst, Animesh Sharma, C. K. Søgaard, T. Khong, Sigrid Berg, Mi Jang, A. Nedal, Andrew Spencer, P. Bruheim, M. Otterlei
          </td>
          <td>2024-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa136c947da96c060fe9e8b5c25af5b195959fd" target='_blank'>
              Developing novel Lin28 inhibitors by computer aided drug design
              </a>
            </td>
          <td>
            Victor M Matias-Barrios, Mariia Radaeva, Graciella Rosellinny, Qiongqiong Jia, Ning Xie, Monica T L Villanueva, Hanadi Ibrahim, Jason Smith, Martin Gleave, N. Lallous, Suzana K Straus, Artem Cherkasov, Xuesen Dong
          </td>
          <td>2025-01-12</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98e96a8cc9646f040d8ba0ebf38bb7db617ee46a" target='_blank'>
              Low dose DNA methyltransferase inhibitors potentiate PARP inhibitors in homologous recombination repair deficient tumors
              </a>
            </td>
          <td>
            R. Pacaud, Scott Thomas, Sibapriya Chaudhuri, Ann Lazar, Luika Timmerman, P. Munster
          </td>
          <td>2025-01-16</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA repair involves various intricate pathways that work together to maintain genome integrity. XPF (ERCC4) is a structural endonuclease that forms a heterodimer with ERCC1 that is critical in both single-strand break repair (SSBR) and double-strand break repair (DSBR). Although the mechanistic function of ERCC1/XPF has been established in nucleotide excision repair (NER), its role in long-patch base excision repair (BER) has recently been discovered through the 5′-Gap pathway. This study briefly explores the roles of XPF in different pathways to emphasize the importance of XPF in DNA repair. XPF deficiency manifests in various diseases, including cancer, neurodegeneration, and aging-related disorders; it is also associated with conditions such as Xeroderma pigmentosum and fertility issues. By examining the molecular mechanisms and pathological consequences linked to XPF dysfunction, this study aims to elucidate the crucial role of XPF in genomic stability as a repair protein in BER and provide perspectives regarding its potential as a therapeutic target in related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec8dd83ccab56542b88608ecabf7dc8acc0d8d5" target='_blank'>
              Beyond Nucleotide Excision Repair: The Importance of XPF in Base Excision Repair and Its Impact on Cancer, Inflammation, and Aging
              </a>
            </td>
          <td>
            Dhara Gohil, Rabindra Roy
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clinical risk scores in chronic myeloid leukemia (CML) are inadequate for identifying chronic phase (CP) patients at high-risk of blast crisis (BC) progression. The lack of accurate predictive tests hamper timely interventions, including stem cell transplants, that are more effective in early disease. By interrogating a single cell atlas of primary imatinib resistance for a BC-like gene expression signature, we identify aberrant CD42A+ megakaryocytic and CD10+CD19+ lymphoid progenitor expansion, as well as STAT1- and IFNγ-related inflammatory programs, as consistent features in the bone marrow and peripheral blood of CP patients at high-risk of BC transformation. We develop multi-color flow cytometry-based tests (MFC) to detect these features in CP patients at the time of diagnosis. Validating our MFC panels on a combined Australia-Singapore cohort comprising 28 CP patients, including 14 who underwent transformation, we demonstrate their ability to detect 100% of CP patients who transform with no false positives. Our findings highlight small populations of inflamed hematopoietic stem and progenitor cells, present at CP diagnosis, as powerful harbingers of future BC transformation. The ability of MFC panels to detect these cells at diagnosis support their inclusion as accurate risk-assessment tools to improve management of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695b8c7eb368f693a97922122a5ffdc4d6cd3154" target='_blank'>
              Single-cell transcriptomics-guided development of flow cytometric tests predicting chronic myeloid leukemia blast crisis transformation at chronic phase diagnosis
              </a>
            </td>
          <td>
            Mengge Yu, Vaidehi Krishnan, Pavithra Shyamsundar, K. Lee, Florian Schmidt, Xi Ren, Charles T.H. Chuah, Shyam Prabhakar, V. Saunders, D. Yeung, N. Shanmuganathan, Agnes S. M. Yong, Timothy P. Hughes, S. T. Ong
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/915fa8f258f231baa925ca4227c99c648ae059fe" target='_blank'>
              Radioresistance-related gene signatures identified by transcriptomics characterize the prognosis and immune landscape of pancreatic cancer patients
              </a>
            </td>
          <td>
            Dandan Wang, Jun Cao, Yanhui Chen, Lisha Zhang, Chan Zhou, Litao Huang, Yanliang Chen
          </td>
          <td>2024-12-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell states evolve through the combined activity of signaling pathways and gene regulatory networks. While transcription factors can direct cell fate, these factors rely on a cell state that is receptive to transitions in cell identity. How signaling levels contribute to the emergence of receptive cell states remains poorly defined in primary cells. Using a well-defined model of direct conversion, we examined how levels of the MAPK-activating oncogene HRASG12V influence direct conversion of primary fibroblasts to induced motor neurons. We demonstrate that an optimal ‘Goldilocks’ level of MAPK signaling efficiently drives cell-fate programming. Rates of direct conversion respond biphasically to increasing HRASG12V levels. While intermediate HRASG12V levels increase the rate of conversion, high levels of HRASG12V induce senescence. Through chemogenetic tuning, we set optimal MAPK activity for high rates of conversion in the absence of HRAS mutants. As MAPK pathways influence cell-fate transitions in development and disease, our results highlight the need to tune therapeutic interventions within a non-monotonic landscape that is shaped by genetics and levels of gene expression. Highlights MAPK signaling drives proliferation and conversion of fibroblasts to motor neurons Cell-fate programming responds biphasically to HRASG12V expression High HRASG12V expression induces senescence, which reduces conversion Chemogenetic tuning of MAPK activity increases conversion rates A small-molecule MAPK inducer drives high rates of conversion in the absence of HRASG12V">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa4a4e600cf88c5b704f193ed57b3b5cfc188f0" target='_blank'>
              Chemogenetic tuning reveals optimal MAPK signaling for cell-fate programming
              </a>
            </td>
          <td>
            Brittany A. Lende-Dorn, Jane C. Atkinson, Yunbeen Bae, Kate E. Galloway
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Engineering cell fate is fundamental for optimizing stem cell-based therapies aimed at replacing cells in patients suffering from trauma or disease. By timely administering molecular regulators—such as transcription factors, RNAs, or small molecules—in a process that mimics in vivo embryonic development, stem cell differentiation can be guided toward a specific cell fate. A significant challenge in scaling up these therapies is that such differentiation strategies often result in mixed cellular populations. While synthetic biology approaches have been proposed to increase the yield of desired cell types, designing gene circuits that effectively redirect cell fate decisions requires mechanistic insight into the dynamics of endogenous regulatory networks that govern decision-making. In this work, we present a biomolecular adaptive controller based on an Incoherent Feedforward Loop (IFFL)-like topology designed to favor a specific cell fate. This controller requires minimal knowledge of the endogenous network as it exhibits adaptive, non-reference-based behavior. The synthetic circuit operates through a sequestration mechanism and a delay introduced by an intermediate species, producing an output that asymptotically approximates a discrete temporal derivative of its input, provided there is a sufficiently fast sequestration rate. By allowing the controller to actuate over a target species involved in the decision-making process, a tunable, synthetic bias is created that favors the production of the desired species with minimal alteration to the overall equilibrium landscape of the endogenous network. Through theoretical and computational analysis, we provide design guidelines for the controller’s optimal operation, evaluate its performance under parametric perturbations, and extend its applicability to various examples of common multistable systems in biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b540a64fb3aa27b889f01e2b17a8cadd0ea0d35" target='_blank'>
              Engineering cell fate with adaptive feedback control
              </a>
            </td>
          <td>
            Frank Britto Bisso, Giulia Giordano, Christian Cuba Samaniego
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4d6b17f62c34eb4147f40c895b9a78313eb007" target='_blank'>
              C3+ cancer-associated fibroblasts promote tumor growth and therapeutic resistance in gastric cancer via activation of the NF-κB signaling pathway
              </a>
            </td>
          <td>
            Zhenxiong Zhao, S. Xiong, Jianpeng Gao, Yingjing Zhang, Ergang Guo, Yakai Huang
          </td>
          <td>2024-12-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The p53 protein has attracted huge research interest over several decades due to its role as one of the most important tumor suppressors in mammals, which orchestrates a synchronous response from normal cells in the body to various forms of stress. The diverse cellular activities of the p53 protein are regulated mainly via its post-translational modifications (PTMs). PTMs affect p53 on several levels: at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes, at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes; at the level of proteolysis in the absence of stress; and on the contrary, at the level of augmented protein stability in response to stress signals. Disruptions in these regulatory mechanisms can lead to deviations from normal cellular function, boosting tumor initiation and progression. Conversely, targeted interventions in these pathways could prove beneficial for the development of antitumor therapies. Advancing our understanding of p53 modifiers and the proteins involved in its regulation equips researchers with an expanded toolkit for studying cellular processes and for developing biologically active molecules that influence p53-mediated responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3fe1959718caec99cd867c8f3aa0f229e81022" target='_blank'>
              p53: The Multifaceted Roles of Covalent Modifications in Cancer
              </a>
            </td>
          <td>
            Tatiana A. Grigoreva, Angelina A. Romanova, V. Tribulovich, Nikolay B. Pestov, Ruslan A Oganov, Diana K. Kovaleva, T. Korneenko, N. A. Barlev
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Renal clear cell carcinoma (ccRCC) presents a unique landscape of genetic and epigenetic modifications, the understanding of which is crucial for the development of targeted therapies and improved prognostication. This study explores the differential expression of histone‐related genes (HRGs) in ccRCC and correlates these findings with patient clinical outcomes. By leveraging the Cancer Genome Atlas (TCGA) data, we performed a robust multidimensional analysis of HRG expression profiles in ccRCC versus adjacent normal tissues. Our results demonstrate a significant upregulation of several key epigenetic regulators, including UHRF1, KDM5A, EZH2, PRDM6, and TWIST1, in tumor samples, with statistical significance suggesting their involvement in tumorigenesis and progression. Paired expression analysis within patient‐matched samples confirmed the consistency of overexpression in tumors. The prognostic relevance of these genes was underscored through survival analyses, which revealed a clear stratification of patients into distinct risk categories based on their expression profiles. The integration of these genetic markers with clinical parameters facilitated the development of a predictive nomogram, yielding a quantifiable tool for survival prediction. Our comprehensive analysis elucidates the profound impact of epigenetic dysregulation in ccRCC and proposes a novel set of biomarkers for disease diagnosis, prognostic stratification, and potential therapeutic targeting, marking a significant stride toward precision oncology in renal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057b9174164151a3b1cfe3112a48500f0dca6ee3" target='_blank'>
              Epigenetic Alterations in Renal Clear Cell Carcinoma: A Comprehensive Expression and Clinical Outcome Analysis
              </a>
            </td>
          <td>
            Dafei Weng, Zhenqian Qin, Xuefeng Yuan, Xuping Jiang, Hua Rui
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Pharmacy and Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNMT3A and TET2 are epigenetic regulator genes commonly mutated in age-related clonal hematopoiesis (CH). Despite having opposed epigenetic functions, these mutations are associated with increased all-cause mortality and a low risk for progression to hematological neoplasms. While individual impacts on the epigenome have been described using different model systems, the phenotypic complexity in humans remains to be elucidated. Here we make use of a natural inflammatory response occurring during coronavirus disease 2019 (COVID-19), to understand the association of these mutations with inflammatory morbidity (acute respiratory distress syndrome-ARDS) and mortality. We demonstrate the age-independent, negative impact of DNMT3A mutant CH on COVID-19-related ARDS and mortality. Using single cell (sc-) proteogenomics we show that DNMT3A mutations involve myeloid and lymphoid lineage cells. Using single cell multiomics sequencing, we identify cell-specific gene expression changes associated with DNMT3A mutations, along with significant epigenomic deregulation affecting enhancer accessibility, resulting in overexpression of IL32, a proinflammatory cytokine that can result in inflammasome activation in monocytes and macrophages. Finally, we show with single cell resolution that the loss of function of DNMT3A is directly associated with increased chromatin accessibility in mutant cells. Hence, we demonstrate the negative prognostic impact of DNMT3Amt CH on COVID-19 related ARDS and mortality. DNMT3Amt CH in the context of COVID-19, was associated with inflammatory transcriptional priming, resulting in overexpression of IL32. This overexpression was secondary to increased chromatic accessibility, specific to DNMT3Amt CH cells. DNMT3Amt CH can thus serve as a potential biomarker for adverse outcomes in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6f734acb174ae5d3944a4074bf5564fa474087" target='_blank'>
              Single cell multiomics reveal divergent effects of DNMT3A and TET2 mutant clonal hematopoiesis in inflammatory response.
              </a>
            </td>
          <td>
            Wazim Mohammed Ismail, Jenna A. Fernandez, M. Binder, T. Lasho, Minsuk Kim, Susan M. Geyer, Amelia Mazzone, C. Finke, A. Mangaonkar, Jeong-Heon Lee, Liguo Wang, Kwan Hyun Kim, Vernadette A. Simon, Fariborz Rakhshan Rohakthar, Amik Munankarmy, Seul Kee Byeon, S. Schwager, Jonathan J Harington, Melissa R. Snyder, Keith D Robertson, Akhilesh Pandey, Eric D. Wieben, Nicholas Chia, Alexandre Gaspar-Maia, Mirinal S Patnaik
          </td>
          <td>2024-12-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Over the course of the last decade, genomic studies in the context of normal human hematopoiesis have provided new insights into the early pathogenesis of myeloproliferative neoplasms (MPN). A preclinical phase of MPN, termed clonal hematopoiesis (CH) was identified and subsequent lineage tracing studies revealed a multi-decade long time interval from acquisition of an MPN phenotypic driver mutation in a hematopoietic stem cell (HSC) to the development of overt MPN. Multiple germline variants associated with MPN risk have been identified through genome-wide association studies (GWAS) and in some cases functional interrogation of the impact of the variant has uncovered new insights into HSC biology and MPN development. Increasingly sophisticated methods to study clonal contributions to human hematopoiesis and measure HSC fitness have helped discern the biology underlying the tremendous clinical heterogeneity observed in MPN. Despite these advances, significant knowledge gaps remain particularly with respect to germline genetic contributors to both MPN pathogenesis and phenotypic diversity, as well as limitations in the ability to prospectively quantify rates of clonal expansion in individual MPN patients. Ultimately, we envisage a personalized approach to MPN care in the future, where an individualized genetic assessment can predict MPN trajectory and this information will be used to inform and guide therapy. MPN is particularly amenable to precision medicine strategies and our increased understanding of the evolution of MPN from normal blood stem cells provides a unique opportunity for early therapeutic intervention approaches and potentially MPN prevention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd917b44573f25d09202b5ca694e87a98f431b63" target='_blank'>
              Evolution of myeloproliferative neoplasms from normal blood stem cells.
              </a>
            </td>
          <td>
            S. Hormoz, Vijay G Sankaran, Ann Mullally
          </td>
          <td>2024-12-05</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Defining consequential differences in intestinal epithelial stem cells in healthy humans versus those with inflammatory bowel disease (Crohn’s disease and ulcerative colitis) is essential for the development of much needed therapies to restore the epithelial barrier and maintain its fidelity. Employing single cell transcriptomic/epigenomic approaches and colonoid models from children and adults with Crohn’s disease led us to identify an inflammatory secretory progenitor (ISP) cell state present almost exclusively in patients with Crohn’s disease compared to control subjects. ISPs exhibit gene expression profiles consistent with normal secretory progenitor cells but concomitantly express a suite of distinguishing pro-inflammatory genes. Mechanistically, ISPs exhibit open chromatin and gene expression of ISP signature genes. While these ISP-specific genes are not expressed in intestinal stem cells, their chromatin is accessible in Crohn’s disease stem cells suggesting that ISP genes are epigenetically poised in stem cells and are transcriptionally activated in ISP cells in the presence of inflammatory stimuli. Consistently, Crohn’s disease colonoids exhibit sustained ISP gene expression that can be elicited further with pro-inflammatory cytokines or via co-culture with pro-inflammatory macrophages. In summary, we define differences in the epithelial stem and progenitor compartment of patients with Crohn’s disease suggesting aberrant stem cell differentiation and inflammatory gene expression arises during disease. HIGHLIGHTS We identify an inflammatory secretory progenitor cell state in endoscopically non-inflamed tissue from pediatric and adult patients with Crohn’s disease. Crohn’s disease epithelial stem and progenitor cells display increased chromatin accessibility preceding the emergence of inflammatory secretory progenitor cell states. Crohn’s disease colonoids from non-inflamed regions contain inflammatory secretory progenitor cells in the absence of inflammatory stimuli Inflammatory secretory progenitor cells expand in colonoid culture in response to cytokine stimulation or co-culture with pro-inflammatory macrophages">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1c97ff6cf153588b934b86c33ad5324b1f6d33" target='_blank'>
              An epigenetic basis for sustained inflammatory epithelial progenitor cell states in Crohn’s disease
              </a>
            </td>
          <td>
            Tatiana A. Karakasheva, Clara Morral Martinez, Yusen Zhou, Jingya Qui, Xinyi E. Chen, Gloria E. Soto, Shaneice K. Nettleford, Olivia T. Hix, Daana M. Roach, Alyssa M. Laguerta, Anusha Thadi, Rachael M. Edwards, Daniel Aleynick, Sarah Weinbrom, Elizaveta Borodyanskaya, Oliver Pickering, Marykate Fulton, Chia- Chen, Bella V. Peterson, Erik B. Hagen, Ian P. Yannuzzi, Zainab Haider, Zvi Cramer, Maire C Conrad, Ning Li, Meenakshi Bewtra, Yasin Uzun, Kai Tan, J. Kelsen, Andy J. Minn, Christopher Joachim Lengner, Kathryn E. Hamilton
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease resulting from an interplay of genetic, microbial, and environmental factors. Cell-type-specific contributions to CD etiology and genetic risk are incompletely understood. Here we built a comprehensive atlas of cell-type-resolved chromatin accessibility comprising 557,310 candidate cis-regulatory elements (cCREs) in terminal ileum and ascending colon from patients with active and inactive CD and healthy controls. Using this atlas, we identified cell-type-, anatomic location-, and context-specific cCREs and characterized the regulatory programs underlying inflammatory responses in the intestinal mucosa of CD patients. Genetic variants that disrupt binding motifs of cell-type-specific transcription factors significantly affected chromatin accessibility in specific mucosal cell types. We found that CD heritability is primarily enriched in immune cell types. However, using fine-mapped non-coding CD variants we identified 29 variants located within cCREs several of which were accessible in epithelial and stromal cells implicating cell types from additional lineages in mediating CD risk in some loci. Our atlas provides a comprehensive resource to study gene regulatory effects in CD and health, and highlights the cellular complexity underlying CD risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7deb97e1df8220d609c097e56053afc325d9d96d" target='_blank'>
              Cell-type-resolved chromatin accessibility in the human intestine identifies complex regulatory programs and clarifies genetic associations in Crohn’s disease
              </a>
            </td>
          <td>
            Y. Zhao, R. Zhou, Z. Mu, P. Carbonetto, X. Zhong, B. Xie, K. Luo, C. Cham, J. Koval, X. He, A. W. Dahl, X. Liu, E. B. Chang, A. Basu, S. Pott
          </td>
          <td>2024-12-11</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/353f79910da257122b65972ea713833d046175ca" target='_blank'>
              Advancements in gene therapies targeting mutant KRAS in cancers.
              </a>
            </td>
          <td>
            Yuhang Wang, Thuy Anh Bui, Xinpu Yang, G. Hutvagner, Wei Deng
          </td>
          <td>2025-01-17</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Our understanding of the vertebrate immune system is dominated by a few model organisms such as mice. This use of a few model systems is reasonable if major features of the immune systems evolve slowly and are conserved across most vertebrates, but may be problematic if there is substantial macroevolutionary change in immune responses. Here, we present a test of the macroevolutionary stability, across 15 species of jawed fishes, of the transcriptomic response to a standardized immune challenge. Intraperitoneal injection of an immune adjuvant (alum) induces a fibrosis response in nearly all jawed fishes, which in some species contributes to anti-helminth resistance. Despite this conserved phenotypic response, the underlying transcriptomic response is highly inconsistent across species. Although many gene orthogroups exhibit differential expression between saline versus alum-injected fish in at least one species, few orthogroups exhibit consistent differential expression across species. This result suggests that although the phenotypic response to alum (fibrosis) is highly conserved, the underlying gene regulatory architecture is very flexible and cannot readily be extrapolated from any one species to fishes (or vertebrates) more broadly. The vertebrate immune response is remarkably changeable over macroevolutionary time, requiring a diversity of model organisms to describe effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02997822d89638ffcc115ee4c55d0d06267f2668" target='_blank'>
              Macroevolutionary changes in gene expression response to an immune stimulus across the diversity of fishes
              </a>
            </td>
          <td>
            Ben A. Flanagan, Lauren E. Fuess, M. Vrtílek, Andrea Roth-Monzón, Daniel I. Bolnick
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The World Health Organization's system for classifying and grading primary tumors of the Central Nervous System conjectures the clinical-biological course of the oncogenic process based on morphological, genetic, histological, and immunohistochemical parameters. These principles are fundamental for a progression in the classification of these tumors, to guarantee the promotion of a more precise diagnosis. In this sense, it is important to understand the process of oncotic cell formation, which is the result of mutations in intra and extracellular control pathways. In this way, genes that act to induce the cell cycle, under normal conditions, when mutated, can result in a dysregulation of the progress of the cycle, causing alterations in the control factors and, consequently, phenotypic transformations in the cell. Thus, to understand the role of genes in modulating primary tumors in the Central Nervous System, mutations in the genes most prevalently related to Gliomas, Meningiomas, and Medulloblastomas were addressed highlighting their influences on the development of these tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a2d441a03edf17b4beaa6aa54a6e378832f0e6" target='_blank'>
              Cellular Regulation and Oncogenesis of Primary Tumors in the Central Nervous System
              </a>
            </td>
          <td>
            Leonnardo Altoé Miranda Lemos, Caroline Stadler, Izabelly Constantinov Lemos, Junio Pereira Pardins, Victoria Beatriz Podolan Sauka
          </td>
          <td>2024-12-01</td>
          <td>Arquivos Brasileiros de Neurocirurgia: Brazilian Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC), known for its hostile nature and limited treatment modalities, has spurred researchers to explore novel approaches for enhancing clinical outcomes. Here, the study aimed to analyze transcriptomics data to identify immune-related hub genes associated with TNBC that might serve as prognostic biomarkers. Initially, we determined genes that were differentially expressed between TNBC and normal tissues by integrating microarray and RNA sequencing data. Then, through protein-protein interaction and module analysis, we identified five putative hub genes: AURKA, CCNB1, CDCA8, GAPDH, and TOP2A. Subsequently, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses revealed that the hub genes were primarily involved in the progesterone-mediated oocyte maturation signaling pathway and oocyte meiosis. Additionally, we observed that these five hub genes were significantly elevated at both protein and mRNA levels in TNBC tissues and contributed to worse survival. Furthermore, the expression of these hub genes exhibited a strong positive association with immune-invading cells such as CD8 T cells, CD4 T cells, and dendritic cells. The analysis of the regulatory network revealed three transcription factors (YBX-1, E2F1, and E2F3) and three posttranscriptional regulators (hsa-mir-25-3p, hsa-mir-92a-3p, and hsa-let-7b-5p) of hub genes. Finally, we explored potential drug candidates for the hub genes using Drug-Gene Interaction Database and discovered that there are no FDA-approved drugs for CCNB1 and CDCA8, highlighting a promising area for future research. Taken together, our results will be of immense importance in addressing the intricacies of TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf529bc9549c109c7e5d52dedaf7a1033e57f352" target='_blank'>
              Deciphering the prognostic landscape of triple-negative breast cancer: A focus on immune-related hub genes and therapeutic implications.
              </a>
            </td>
          <td>
            H. Krishnamoorthy, Ramanathan Karuppasamy
          </td>
          <td>2024-11-25</td>
          <td>Biotechnology and applied biochemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background/Objectives: Treatment with tyrosine kinase inhibitors (TKIs) in chronic myeloid leukemia (CML) has revolutionized disease management and has transformed CML from a life-threatening disease to a chronic condition for many patients. However, overcoming resistance, particularly related to leukemic stem cells (LSC) that can persist even when the bulk of the leukemic cells are eliminated, remains a significant challenge. Methods: K562 and KU812 cell lines were treated in vitro with the TKI Imatinib (IM). Gene expression, protein analysis, and metabolomic screening were conducted to investigate the ability of the drug to enhance stem cell (SC) features. Moreover, a gene ontology analysis was performed on different available datasets, to further consolidate our data. Results: 48 h of IM treatment can significantly increase the expression of genes related to SC self-renewal, particularly SOX2 and OCT 3/4. Interestingly, these modulations occur in cells that remain alive after drug treatment and that displayed features consistent with leukemia stem-like CML cells, suggesting that SC genes levels are crucial even in cell population survived upon TKI treatment. Moreover, after in silico analysis of available data, we observed an enrichment of SOX2/NANOG and OCT 3/4 signatures after TKI treatment, thus strengthening our results. Conclusions: Our results confirmed the relevance of LSC features after TKI treatment, highlighting the need for more effective and potentially curative strategies targeting LSCs to overcome resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28fb6d29d24fd74f86c83c9e97c902987037fdab" target='_blank'>
              Tyrosine Kinase Inhibitor Therapy Enhances Stem Cells Profile and May Contribute to Survival of Chronic Myeloid Leukemiastem Cells
              </a>
            </td>
          <td>
            Simone Rocco, A. Maglione, Valentina Schiavo, Alessandro Ferrando, Carmen Fava, D. Cilloni, Barbara Pergolizzi, C. Panuzzo
          </td>
          <td>2025-01-10</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d24c4e944e46fe6552525ff683c3906cad05e1d" target='_blank'>
              KDELR1 regulates chondrosarcoma drug resistance and malignant behavior through Intergrin-Hippo-YAP1 axis
              </a>
            </td>
          <td>
            Huabin Yin, Dongjie Jiang, Yongai Li, Wenjun Chen, Jie Zhang, Xinghai Yang, Jinbo Hu, Haifeng Wei
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic processes play key roles in regulating gene expression, genome stability, and metabolic output in organisms across the tree of life. Yet, the role epigenetics plays in regulating genomes and behaviors remains underdeveloped for microalgae, particularly as new species are identified and characterized. This is likely due to the cumbersome nature and species-dependent attributes of epigenetic wet-lab methodologies, which preclude the rapid identification of epigenetic modifications and modulators. However, there is extraordinary conservation of epigenetic processes from budding yeast to humans; in many cases, one may infer how behavior and function are epigenetically regulated in novel species by simply identifying epigenetic modulators, or the proteins responsible for conferring epigenetic modifications. To this end, we have developed a graphical software package, titled PERCEPTIVE (pipeline for the prediction of epigenetic modulators in novel species). This novel platform solely uses the genomic sequence of an organism, and preexisting information from model species, to predict the epigenetic modulators and associated modifications in a novel species. Predictions are presented to the user in a graphical interface, which provides literature-based interpretation of results, enabling users to quickly understand potential epigenetic processes in their species of interest and plan follow-up experiments. To test PERCEPTIVE, we predicted epigenetic modulators in several feedstock candidate algae species. To validate these predictions, wet-lab studies were performed, including mass spectrometry; these results underscore the high accuracy of PERCEPTIVE predictions. Overall, PERCEPTIVE represents a powerful tool for the research and manipulation of algal species, which does not require a priori knowledge of epigenetics and is accessible to a broad set of investigators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb34ee3f1ae2f57ecfe1bc910338d94272b5ae12" target='_blank'>
              PERCEPTIVE: an R Shiny pipeline for the prediction of epigenetic modulators in novel species
              </a>
            </td>
          <td>
            Eric M Small, Christina R. Steadman
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62dfdb941313e3b0f5c59cb8565ae7dba1feb83d" target='_blank'>
              Tipping the balance of cell death: alternative splicing as a source of MCL-1S in cancer
              </a>
            </td>
          <td>
            Mariusz L. Hartman
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [19, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>