<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-03 09:28:09 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>23</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) have demonstrated remarkable efficacy in treating non-small cell lung cancer (NSCLC), but acquired resistance greatly reduces efficacy and poses a significant challenge to patients. While numerous studies have investigated the mechanisms underlying EGFR-TKI resistance, its complexity and diversity make the existing understanding still incomplete. Traditional approaches frequently struggle to adequately reveal the process of drug resistance development through mean value analysis at the overall cellular level. In recent years, the rapid development of single-cell RNA sequencing technology has introduced a transformative method for analyzing gene expression changes within tumor cells at a single-cell resolution. It not only deepens our understanding of the tumor microenvironment and cellular heterogeneity associated with EGFR-TKI resistance but also identifies potential biomarkers of resistance. In this review, we highlight the critical role of single-cell RNA sequencing in lung cancer research, with a particular focus on its application to exploring the mechanisms of EGFR-TKI-acquired resistance in NSCLC. We emphasize its potential for elucidating the complexity of drug resistance mechanism and its promise in informing more precise and personalized treatment strategies. Ultimately, this approach aims to advance NSCLC treatment toward a new era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57a8423fcdeff9dbe3c23c8eb5bf687e8a733ce" target='_blank'>
              Single-Cell RNA Sequencing in Unraveling Acquired Resistance to EGFR-TKIs in Non-Small Cell Lung Cancer: New Perspectives
              </a>
            </td>
          <td>
            Lin Peng, Siyou Deng, Jinjie Li, Yujie Zhang, Li Zhang
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer is a major cause of death worldwide. Next-generation sequencing (NGS) has dramatically increased the sequencing data output and transformed biomedical investigations. NGS enables the generations of genetic data specific to patients from tumor tissue samples so that targeted therapies can be used. The obtained data further allows the prioritization of effective therapies based on the tumor-specific genotype. Practitioners in the field of clinical genomics can make the best use of testing facilities while lessening the possible off-targets by choosing a priori gene set. Therefore, targeted sequencing has arisen as a more affordable technique for the genomic profiling of tumors. Drug resistance is commonly observed in cancer because of mutations. Thus, precise genetic and molecular profiling of tumors ought to be routinely done prior to the use of targeted therapy or precision cancer therapy. NGS already has the capacity to ameliorate genetic screening in families with previous histories of the high occurrence of various cancer-associated genes, including TP53, APC, BRCA2, and BRCA1. By using NGS system, researchers detected increased variants in cancer cells with greater specificity and sensitivity than conventional diagnostic approaches, which suggest the potential of NGS in diagnosis. The field of precision cancer therapy is continuously growing and because of their specificity at the molecular level has improved the management and treatment of numerous cancers. These therapies are less toxic and more efficient compared to conventional chemotherapies used in cancer treatment. The field of precision cancer therapy is likely to significantly expand as NGS system advances. This review provides extensive information regarding current advances in the NGS field in terms of methods, clinical applications, genomic profiling, and the role of NGS of precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42985c9d2d1f73b46baa3e0ccc12da6c1288b30" target='_blank'>
              Recent advances in the methods and clinical applications of next-generation sequencing in genomic profiling and precision cancer therapy
              </a>
            </td>
          <td>
            A. Alsaiari
          </td>
          <td>2025-01-03</td>
          <td>EXCLI Journal</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Synthetic lethality represents a novel paradigm in molecular targeted cancer therapy. In synthetic lethality, perturbation of one gene alone does not hinder cell viability, yet simultaneous perturbation of both genes results in a loss of cellular viability. The presence of gene mutations in cancer cells, as opposed to normal cells, provides an opportunity for targeted therapies that mimic the effects of the second genetic mutation, enabling selective eradication of cancer cells. Recent advances in high-throughput screening technologies, such as CRISPR-Cas9 and RNA interference, have significantly enhanced the identification of synthetic lethal interactions, expanding the potential targets for therapeutic intervention. Challenges in exploiting synthetic lethality for cancer treatment include the complexities of tumor biology, limited comprehension of synthetic lethal interactions, drug resistance, and impediments in screening and clinical translation. Emerging strategies, such as combination therapies and novel drug designs, are being developed to overcome these obstacles. By virtue of its selective lethality towards cancer cells bearing specific genetic alterations, targeting synthetic lethal genes holds the promise to provide wider therapeutic windows compared to traditional cytotoxic chemotherapy. This review describes the current state of synthetic lethality applications in cancer treatment, encompassing both biological and methodological perspectives. It highlights the latest advancements in synthetic lethality with emerging interventional strategies. Furthermore, it explores future directions for research and clinical implementation, aiming to refine and expand the therapeutic potential of synthetic lethality in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c9ad6ae81a9b46e74e1e6708428804d26e2ad" target='_blank'>
              Advances in Synthetic Lethality in Potential Oncology Therapeutic Approaches.
              </a>
            </td>
          <td>
            Feifei Yang, Huiyu Wang, Shule Fan, Huiran Qiu, Xiangzhi Li, Guangyao Shi, Zihao Li, Xiaotian Luan, Haigang Wu
          </td>
          <td>2025-01-30</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All cancers are diseases of the genome, since the cancer cell genome typically consists of 10,000s of passenger alterations, 5-10 biologically relevant alterations, and 1-2 "actionable" alterations. Therefore, somatic mutations in cancer cells can have diagnostic, prognostic, and predictive value. Traditional methods are widely used for testing, such as immunohistochemistry, Sanger sequencing, and allele-specific PCR. However, due to the low throughput, these methods are focused exclusively on testing the most common mutations in target genes. The modern next generation sequencing (NGS) is a technology that enables precision oncology in its current form. ESCAT and ESMO Guidelines defined NGS for routine use in patients with advanced cancers such as non-squamous non-small cell lung cancer, prostate cancer, ovarian cancer, and cholangiocarcinoma. The high sensitivity of the NGS method allows it to be used to search for specific mutations in circulating tumor DNA in blood plasma and other body fluids. NGS testing has evolved from hotspot panels, actionable gene panels, and disease-specific panels to more comprehensive panels. The exome and whole genome sequencing approaches are just beginning to emerge, that is why panel-based testing remains most optimal in oncology practice. NGS is also widely used to identify new and rare mutations in cancer genes and detect inherited cancer mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207a8f97841f6a0acdc06c228c5f1967ea8ee827" target='_blank'>
              Application of Next-Generation Sequencing to Realize Principles of Precision Therapy in Management of Cancer Patients.
              </a>
            </td>
          <td>
            N. Khranovska, O. Gorbach, O. Skachkova, G. Klimnyuk
          </td>
          <td>2025-02-20</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, Manoj Kumar Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance.Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation.We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation.This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In recent years, significant advancements in biochemistry, materials science, engineering, and computer-aided testing have driven the development of high-throughput tools for profiling genetic information. Single-cell RNA sequencing (scRNA-seq) technologies have established themselves as key tools for dissecting genetic sequences at the level of single cells. These technologies reveal cellular diversity and allow for the exploration of cell states and transformations with exceptional resolution. Unlike bulk sequencing, which provides population-averaged data, scRNA-seq can detect cell subtypes or gene expression variations that would otherwise be overlooked. However, a key limitation of scRNA-seq is its inability to preserve spatial information about the RNA transcriptome, as the process requires tissue dissociation and cell isolation. Spatial transcriptomics is a pivotal advancement in medical biotechnology, facilitating the identification of molecules such as RNA in their original spatial context within tissue sections at the single-cell level. This capability offers a substantial advantage over traditional single-cell sequencing techniques. Spatial transcriptomics offers valuable insights into a wide range of biomedical fields, including neurology, embryology, cancer research, immunology, and histology. This review highlights single-cell sequencing approaches, recent technological developments, associated challenges, various techniques for expression data analysis, and their applications in disciplines such as cancer research, microbiology, neuroscience, reproductive biology, and immunology. It highlights the critical role of single-cell sequencing tools in characterizing the dynamic nature of individual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8242b8eb0f45c6f5b5cf16b8ec7b8ce82d36e19b" target='_blank'>
              Advancements in single-cell RNA sequencing and spatial transcriptomics: transforming biomedical research
              </a>
            </td>
          <td>
            Getnet Molla Desta, A. Birhanu
          </td>
          <td>2025-02-05</td>
          <td>Acta Biochimica Polonica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Adoptive immune cell therapies, such as CAR T, have revolutionized how we can approach cancer therapy, however challenges remain to increase their efficacy, response rates, and resistance to dysfunction. The systematic investigation of gene programs governing human T cell function would be advantageous for reprogramming cell products for clinical benefit. Recent studies have demonstrated the power of CRISPR-based forward genetic screens in primary human T cells to nominate candidate perturbations for enhanced T cell function, however combinations of perturbations will likely be necessary to overcome the most challenging tumors. To enable the systematic and high-throughput interrogation of combinations of gain and loss-of-function perturbations, we have developed a bidirectional CRISPR activation and interference (CRISPRai) system, where pairs of genes can be simultaneously activated and repressed in primary human T cells. Leveraging hits from previous genome-wide CRISPRa and CRISPRi screens, we constructed a library of >66,000 sgRNA combinations and screened for regulators of acute T cell stimulation responses, including cytokine production and proliferation. Reassuringly, these screens identified expected gene epistasis interactions (e.g. rescue of PLCG1 repression by PLCG2 activation), while also discovering novel genetic interactions, providing mechanistic insights into the regulatory networks governing stimulation response. To further identify gene “building blocks” for evaluation as combinations, we completed genome-wide CRISPRa and CRISPRi screens for regulators of human CD8+ CAR T cell exhaustion. These screens identified known potential enhancements (e.g. gain of BATF, loss of MED12), and also uncovered novel regulators of T cell function, with CRISPRa discovering genes not normally expressed in T cells but whose overexpression could cause resistance to exhaustion by synthetic mechanisms. Gene expression programs controlled by perturbation of key regulators were mapped by pairing CRISPR with single-cell RNA-seq (Perturb-seq) and arrayed functional assays. Finally, a top gain-of-function candidate was evaluated for in vivo tumor control, using an antigen-inducible Synthetic Notch system. Together, this work demonstrates how advanced forward genetics approaches can discover and elucidate both natural and synthetic programs that regulate T cell states, and the nomination of perturbations to encode in logic-gated synthetic biology designs for cell therapies.
 Citation Format: Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole Roybal, Brian Shy, Alexander Marson. Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, K. Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="INTRODUCTION
Globally, prostate cancer (CaP) is a leading cause of death and disability among men and a substantial public health burden. Despite advancements in cancer treatment, chemoresistance remains a significant issue in cancer therapy, accounting for the majority of patient relapses and poor survival. Cancer stem cells (CSCs) are considered the main cause of cancer recurrence, chemoresistance, and poor survival of patients. These CSCs acquire stemness and chemoresistance by certain mechanisms such as enhanced DNA repair processes, increased expression of drug efflux pumps, resistance to apoptosis, and altered cell cycle and tumor microenvironment (TME).


AREA COVERED
We cover the latest developments in this field and give an overview of future research directions.


EXPERT OPINION
CSCs show dysregulation of several signaling pathways, mostly related to conferring chemoresistance phenotype, such as high drug efflux, apoptotic resistance, quiescent cell cycle, tumor microenvironment, and DNA repair. There are several research articles published on this topic. However, still, this field warrants further investigations to identify the therapeutic molecule that can either chemosensitize CSCs or kill them effectively. This can only be possible when we know the complete mechanisms to comprehend the fundamental causes of cancer stemness and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/756692242b3daaa338fbef439d9939f7beaddfd4" target='_blank'>
              An update understanding of stemness and chemoresistance of prostate cancer.
              </a>
            </td>
          <td>
            Afiya Wahab, Hifzur R Siddique
          </td>
          <td>2025-02-11</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 The molecular mechanisms underlying estrogen receptor (ER)-positive breast carcinogenesis and drug resistance remain incompletely understood. Elevated expression of CCND1 is linked to enhanced invasiveness, poorer prognosis, and resistance to drug therapies in ER-positive breast cancer. In this study, we identify a highly expressed circular RNA (circRNA) derived from
 FOXK2
 , called circFOXK2, which plays a key role in stabilizing
 CCND1
 mRNA, thereby promoting cell cycle progression, cell growth, and endocrine therapy resistance in ER-positive breast cancer cells. Mechanistically, circFOXK2 binds directly to
 CCND1
 mRNA via RNA–RNA pairing and recruits the RNA-binding protein ELAVL1/HuR, stabilizing the
 CCND1
 mRNA and enhancing CCND1 protein levels. This results in activation of the CCND1–CDK4/6–p-RB–E2F signaling axis, driving the transcription of downstream E2F target genes and facilitating the G1/S transition during cell cycle progression. Notably, targeting circFOXK2 with antisense oligonucleotide (ASO-circFOXK2) suppresses ER-positive breast cancer cell growth both in vitro and in vivo. Moreover, combination therapy with ASO-circFOXK2 and tamoxifen exhibits synergistic effects and restores tamoxifen sensitivity in tamoxifen-resistant cells. Clinically, high circFOXK2 expression is positively correlated with
 CCND1
 levels in both ER-positive breast cancer cell lines and patient tumor tissues. Overall, our findings reveal the critical role of circFOXK2 in stabilizing the oncogene
 CCND1
 and promoting cancer progression, positioning circFOXK2 as a potential therapeutic target for ER-positive breast cancer in clinical settings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, Xiao-sha Wen, Di-xian Luo, Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Systematic screening of gain- or loss-of-function genetic perturbations can be used to characterize the genetic dependencies and mechanisms of regulation for essentially any cellular process of interest. These experiments typically involve profiling from a pool of single gene perturbations and how each genetic perturbation affects the relative cell fitness. When applied in the context of drug efficacy studies, often called chemo-genetic profiling, these methods should be effective at identifying drug mechanisms of action. Unfortunately, fitness-based chemo-genetic profiling studies are ineffective at identifying all components of a drug response. For instance, these studies generally fail to identify which genes regulate drug-induced cell death. Several issues contribute to obscuring death regulation in fitness-based screens, including the confounding effects of proliferation rate variation, variation in the drug-induced coordination between growth and death, and, in some cases, the inability to separate DNA from live and dead cells. MEDUSA is an analytical method for identifying death-regulatory genes in conventional chemo-genetic profiling data. It works by using computational simulations to estimate the growth and death rates that created an observed fitness profile rather than scoring fitness itself. Effective use of the method depends on optimal tittering of experimental conditions, including the drug dose, starting population size, and length of the assay. This manuscript will describe how to set up a chemo-genetic profiling study for MEDUSA-based analysis, and we will demonstrate how to use the method to quantify death rates in chemo-genetic profiling data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Deletions and translocations affecting WWOX accompanied by loss of expression are frequently observed in B cell neoplasms and are linked to poor prognosis. Our previous research showed that Wwox deletion early in B cell development induces genomic instability, neoplastic transformation, and monoclonal gammopathies in mice. In this study, by crossing Cd19 Wwox knockout (KO) with Vk*MYC myeloma model mice, we generated a model with concurrent Wwox deletion and MYC activation, reproducing two common oncogenic alterations in B and plasma cell cancers. We observed that Vk*MYC:Wwox KO mice exhibited significantly reduced survival rates primarily due to the development of plasmablastic plasmacytomas and lymphomas. Transcriptome profiling from bone marrow derived Cd138+ plasma cells and plasmablastic tumors revealed enrichment of biofunctions related to tumorigenic phenotype and inflammation activation upon Wwox deletion in Vk*MYC mice. Wwox KO plasmablastic tumors displayed mutations affecting classical cancer genes, DNA damage response (DDR) genes, as well as overexpression of Aid/Apobec family members associated to hypermutation and DDR mutational signatures. These findings illustrate the significant pathobiological effects of B cell specific Wwox deletion and support a relevant role for WWOX loss of function in B cell neoplastic progression towards more aggressive phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5f5eab1861b5f39965ad87997666d862606529" target='_blank'>
              B-cell-Specific Wwox Deletion Promotes Plasmablastic Tumor Development and Pro-Inflammatory Signatures in a Myeloma Mouse Model
              </a>
            </td>
          <td>
            Tabish Hussain, Matthew D Bramble, Bin Liu, M. C. Abba, M. Chesi, C. M. Aldaz
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Comparative genomics has emerged as a pivotal methodology for elucidating genetic variations in microbial studies. However, conventional analytical approaches for diploid and polyploid microorganisms have demonstrated limited efficacy in discriminating new mutations from background heterozygosity. This study presents an innovative microbial comparative genomics framework adapted from tumor-normal sequencing methodology. Our approach establishes the original strain as the “normal sample” and the derived strain as the “tumor sample”, enabling precise identification of new mutations (“somatic variants”) while filtering pre-existing heterozygous sites (“germline variations”). The analytical pipeline also includes assessment of loss-of-heterozygosity (LOH) events and genome-wide detection of copy number variations (CNVs) with resolution to identify both regional CNV and whole-chromosome aneuploidy through integrated CNV and variant allele frequency analysis. We validated this framework using diploid Saccharomyces cerevisiae strains before successfully extending its application to Kluyveromyces marxianus, Candida spp., and Hortaea werneckii, encompassing haploid, diploid, polyploid, and aneuploid states. The methodology revealed previously undetected variations across experimental evolution studies, demonstrating superior resolution compared to conventional approaches. This adaptable platform establishes a new paradigm for microbial genome studies, particularly for organisms with diploid or polyploid states where traditional comparative genomics methods prove inadequate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The integration of whole-genome sequencing (WGS) and whole-transcriptome sequencing (WTS) has revolutionized cancer diagnostics, enabling comprehensive molecular profiling of tumours. While WGS uncovers genomic alterations such as single nucleotide variants (SNVs), structural variants (SVs), and copy number changes, WTS provides complementary insights into transcriptomic changes, including gene expression levels, alternative splicing, and fusion events. Together, these technologies offer unparalleled potential to guide precision oncology by identifying actionable biomarkers and stratifying patients for targeted therapies or clinical trials. However, the integration of WGS and WTS data remains a significant bioinformatics challenge due to biological variability, tumour-specific complexities, and the lack of harmonized reference datasets. To address this, we developed RNAsum, an open-source tool designed to combine and interpret WGS and WTS data from individual cancer patient samples. RNAsum compares patient data against curated cohorts from The Cancer Genome Atlas (TCGA) and integrates quantitative expression data with genomic insights to enhance diagnostic accuracy, validate genomic findings, and prioritize clinically relevant alterations. We demonstrate the utility of RNAsum through case studies, showcasing its ability to confirm clinically reportable variants, refine diagnoses, and identify novel therapeutic targets. By bridging the gap between genome and transcriptome analyses, RNAsum represents a step forward in the routine clinical application of multi-omics for personalized cancer care. The tool is freely available as an R package on GitHub, supporting accessibility and scalability for diverse research and clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82fa64d9bf4f88f6a630cd910ec88f706feaf7b1" target='_blank'>
              RNAsum: A tool for personalised genome and transcriptome interpretation for improved cancer diagnostics
              </a>
            </td>
          <td>
            S. Kanwal, J. Marzec, J. Vissers, P. Diakumis, L. Varghese, L. Tork, O. Hofmann, S. Grimmond, S. J. Luen
          </td>
          <td>2025-01-12</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bdfee01c97ef1e7e57c28ea058914128289212e" target='_blank'>
              Establishment and transcriptomic characteristics of radio-resistant meningioma cell lines.
              </a>
            </td>
          <td>
            Jinxiu Yu, Leihao Ren, Tianqi Wu, Lingyang Hua, Daijun Wang, Yang Wang, Qing Xie, Jiaojiao Deng, Ye Gong
          </td>
          <td>2025-02-28</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21cbb2f5aa4a29eac0f638906ce12441220483a" target='_blank'>
              Deciphering single-cell genomic architecture: insights into cellular heterogeneity and regulatory dynamics
              </a>
            </td>
          <td>
            Byunghee Kang, Hyeonji Lee, Tae-Young Roh
          </td>
          <td>2025-02-11</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose
The epidermal growth factor receptor (EGFR) is a therapeutic target with confirmed clinical efficacy for several cancer types. We aimed to identify EGFR aberrations and their associations with other genomic alterations in patients with metastatic diseases of various cancers.


Materials and Methods
We used real-world data from the next-generation sequencing (NGS) of 3,286 patients with metastatic cancer at the Samsung Medical Center. We analyzed the distribution of EGFR amplification, mutation, and fusion, as well as their correlations with microsatellite instability (MSI), tumor mutation burden (TMB), and other gene aberrations.


Results
A total of 3,286 patients were tested using NGS of a panel covering 523 cancer-related genes (TSO500, Illumina) as part of clinical practice between October 2019 and October 2022. Patients with lung cancer and gliomas were not included in the analysis. Of the 3,286 patients, 175 (5.3%) had EGFR amplification, 38 (1.2%) had EGFR mutations, and 8 (0.2%) had EGFR fusion. All 175 patients with EGFR amplifications had microsatellite-stable (MSS) tumors, but 102 had co-amplifications in other cancer-related genes, and 78 had mutations with clinical significance (tier I/II). Among the 38 patients with EGFR mutations, three (8%) showed MSI-high status, and eleven (29%) demonstrated high TMB (≥ 10 mutations/mb). Among eight patients with EGFR fusion, three exhibited possible functionalities of the EGFR gene.


Conclusion
EGFR aberrations, mainly amplification, followed by mutation and fusion, were present in 6.4% of patients with metastatic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae0df005b2d8a0f47444591e98d3dc4bb550830" target='_blank'>
              Epidermal Growth Factor Receptor Aberrations Identified by Next-generation Sequencing in Patients with Metastatic Cancers.
              </a>
            </td>
          <td>
            Minkyue Shin, , Jaeyun Jung, Deokgeun Kim, Sung Hee Lim, Seung-Tae Kim, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Kyoung-Mee Kim, Jeeyun Lee
          </td>
          <td>2025-02-21</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Pancreatic cancer is the result of mutations in crucial genetic markers like KRAS and TP53 that make treatment challenging. This article discusses how CRISPR Cas9 technology can be combined with these markers to create treatments. CRISPR allows for the alteration or repair of these mutations, with the aim of restoring gene function or blocking cancer-causing pathways. For instance, CRISPR has the potential to fix mutations in TP53 or CDKN2A genes and restore SMAD4 signaling or target the KRAS oncogene in the body’s cells. However promising, it may be that CRISPR encounters obstacles like unintentional effects and challenges in effectively delivering it to pancreatic tumor cells. Furthermore, ethical concerns, especially related to the editing of the germline, need consideration. As techniques based on CRISPR advance, there is a chance for them to transform the treatment landscape for cancer by offering personalized therapies. More studies are needed to enhance how treatments are administered accurately and safely through methods and targeted testing for effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d180ed2014a91157721c0376d7e2ca0a9fe65e1" target='_blank'>
              Integrating CRISPR Technology with Key Genetic Markers in Pancreatic Cancer: A New Frontier in Targeted Therapies
              </a>
            </td>
          <td>
            Dimitrios Stefanoudakis
          </td>
          <td>2025-01-06</td>
          <td>SynBio</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Genomic data is essential for clinical decision-making in precision oncology. Bioinformatic algorithms are widely used to analyze next-generation sequencing (NGS) data, but they face two major challenges. First, these pipelines are highly complex, involving multiple steps and the integration of various tools. Second, they generate features that are human-interpretable but often result in information loss by focusing only on predefined genetic properties. This limitation restricts the full potential of NGS data in biomarker extraction and slows the discovery of new biomarkers in precision oncology. Methods We propose an end-to-end deep learning (DL) approach for analyzing NGS data. Specifically, we developed a multiple instance learning DL framework that integrates somatic mutation sequences to predict two compound biomarkers: microsatellite instability (MSI) and homologous recombination deficiency (HRD). To achieve this, we utilized data from 3,184 cancer patients obtained from two public databases: The Cancer Genome Atlas (TCGA) and the Clinical Proteome Tumor Analysis Consortium (CPTAC). Results Our proposed deep learning method demonstrated high accuracy in identifying clinically relevant biomarkers. For predicting MSI status, the model achieved an accuracy of 0.98, a sensitivity of 0.95, and a specificity of 1.00 on an external validation cohort. For predicting HRD status, the model achieved an accuracy of 0.80, a sensitivity of 0.75, and a specificity of 0.86. Furthermore, the deep learning approach significantly outperformed traditional machine learning methods in both tasks (MSI accuracy, p-value = 5.11×10−18; HRD accuracy, p-value = 1.07×10−10). Using explainability techniques, we demonstrated that the model’s predictions are based on biologically meaningful features, aligning with key DNA damage repair mutation signatures. Conclusion We demonstrate that deep learning can identify patterns in unfiltered somatic mutations without the need for manual feature extraction. This approach enhances the detection of actionable targets and paves the way for developing NGS-based biomarkers using minimally processed data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad073d80b11f20c88ed32369a6393b6a5510e50a" target='_blank'>
              Deep Learning for Biomarker Discovery in Cancer Genomes
              </a>
            </td>
          <td>
            Michaela Unger, C. Loeffler, Laura Žigutytė, Srividhya Sainath, Tim Lenz, Julien Vibert, Andreas Mock, S. Fröhling, Trevor A. Graham, Zunamys I. Carrero, J. N. Kather
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31160a8616dac1a93a1309bf2063539c61bdfcd2" target='_blank'>
              Discovery and characterization of a novel telomerase alternative splicing isoform that protects lung cancer cells from chemotherapy induced cell death
              </a>
            </td>
          <td>
            Jeongjin J. Kim, Alexander Ahn, Jeffrey Y Ying, Andrew T. Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell cycle checkpoints are stringent quality control mechanisms that regulate cell cycle progression and division. Cancer cells often develop a dependency on the G2/M cell cycle checkpoint to facilitate DNA repair and resolve intrinsic or therapy-induced DNA damage. This dependency leads to therapy resistance, continuous cell division, and disease progression. Targeting G2/M checkpoints has been heavily pursued over the past two decades and has progressed into clinical studies. Recent genome-scale functional genomic studies have revealed that Myt1 kinase, an essential but previously overlooked molecule for the G2/M checkpoint, is a promising target for multiple types of cancers. In this work, we summarize the latest discoveries in molecular targeting of Myt1 kinase and discuss the challenges and limitations in expanding its clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17deb173d71c056d6283f804746aaf46a913ddbb" target='_blank'>
              Myt1 kinase: An Emerging Cell Cycle Regulator for Cancer Therapeutics.
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Javeria Zaheer, Diego Luis Ribeiro, Chunzhang Yang
          </td>
          <td>2025-01-16</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objectives Unlike other diseases, cancer is not just a genome disease but should broadly be viewed as a disease of the cellular machinery. Therefore, integrative multifaceted approaches are crucial to understanding the complex nature of cancer biology. Bcl-2 (B-cell lymphoma 2), encoded by the human Bcl-2 gene, is a critical anti-apoptotic protein that regulates cell death pathways, primarily by inhibiting apoptosis. It plays a pivotal role in maintaining cellular homeostasis by preventing premature or excessive cell death. Genetic variations and dysregulation of Bcl-2 are particularly significant in cancer, as they disrupt the normal apoptotic machinery, enabling cancer cells to evade programmed cell death. Single nucleotide polymorphisms (SNPs) are considered viable diagnostic and therapeutic biomarkers for various cancers. Therefore, this study explores the association between SNPs in Bcl-2 and the structural, functional, protein-protein interactions (PPIs), drug binding and dynamic characteristics. Methods Comprehensive cross-validated bioinformatics tools and molecular dynamics (MD) simulations. Multiple sequence, genetic, structural and disease phenotype analyses were applied in this study. Results Analysis revealed that out of 130 mutations, approximately 8.5% of these mutations were classified as pathogenic. Furthermore, two particular variants, namely, Bcl-2G101V and Bcl-2F104L, were found to be the most deleterious across all analyses. Following 500 ns, MD simulations showed that these mutations caused a significant distortion in the protein conformational, protein-protein interactions (PPIs), and drug binding landscape compared to Bcl-2WT. Conclusion Despite being a predictive study, the findings presented in this report would offer a perspective insight for further experimental investigation, rational drug design, and cancer gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f67dee38c9954b65ace573045ef349b37771a6" target='_blank'>
              Integrative genomic analyses combined with molecular dynamics simulations reveal the impact of deleterious mutations of Bcl-2 gene on the apoptotic machinery and implications in carcinogenesis
              </a>
            </td>
          <td>
            G. Elamin, Zhichao Zhang, Depika Dwarka, K. Kasumbwe, John Mellem, Nompumelelo P. Mkhwanazi, P. Madlala, Mahmoud E. S. Soliman
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, , Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Multiple Myeloma (MM) is the second most common hematologic malignancy, which exhibits strong resistance to bortezomib, the first-line treatment. Circular RNAs (circRNAs) are increasingly considered as important drivers of drug resistance across various cancers, but their roles in multiple myeloma are not well understood. Aim To investigate and identify potential circRNA targets and their roles in the mechanisms of bortezomib resistance. Methods Bortezomib-resistant MM patient-specific circRNAs were screened using Arraystar circRNA microarrays. The MM circRNA dataset from the GEO database was analyzed with GEO2R to identify candidate circRNAs associated with MM progression and drug resistance. CircRNA-forming and loop-forming sites, along with their structures, were identified via Sanger sequencing. The identified circRNA was validated by qRT-PCR in MM patients with and without bortezomib resistance. Bioinformatic analysis through CircInteractome was conducted to predict potential miRNA and RBP binding for the core circRNAs. Metascape was employed to perform RBP pathway analysis to identify specific biological processes in circRNAs. Results The hsa_circ_0058191 was found to be overexpressed in bortezomib-resistant MM patient samples, suggesting its pivotal role in drug resistance mechanisms. The interaction of hsa_circ_0058191 with miR-660 and AGO2 as determined through bioinformatic predictions, indicated that it regulates RNA modification and mRNA regulation pathways. These molecular interactions expand our understanding of the mechanisms of drug resistance in multiple myeloma. Conclusion This study identified the role of hsa_circ_0058191 in the development of drug resistance in MM, which provides a theoretical foundation for designing potential therapeutic strategies to prevent drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48ee97c88b1ae7597bcfc9d335948e8abc169c8" target='_blank'>
              Comprehensive Analysis Identifies Hsa_circ_0058191 as a Potential Drug Resistance Target in Multiple Myeloma
              </a>
            </td>
          <td>
            Huiye Yang, Jie Zhu, Xiaotao Wang
          </td>
          <td>2025-02-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bruton's tyrosine kinase (BTK) inhibitors are effective for the treatment of chronic lymphocytic leukemia (CLL) due to BTK's role in B cell survival and proliferation. Treatment resistance is most commonly caused by the emergence of the hallmark BTK C481S mutation that inhibits drug binding. In this study, we aimed to investigate cancer subclones harboring a BTK C481S mutation and identify cells with co-occurring CLL driver mutations. In addition, we sought to determine whether BTK-mutated subclones exhibit distinct transcriptomic behavior when compared to other cancer subclones. To achieve these goals, we use scBayes, which integrates bulk DNA sequencing and single-cell RNA sequencing (scRNA-seq) data to genotype individual cells for subclone-defining mutations. While the most common approach for scRNA-seq includes short-read sequencing, transcript coverage is limited due to the vast majority of the reads being concentrated at the priming end of the transcript. Here, we utilized MAS-seq, a long-read scRNA-seq technology, to substantially increase transcript coverage and expand the set of informative mutations to link cells to cancer subclones in six CLL patients who acquired BTK C481S mutations during BTK inhibitor treatment. In two patients who developed two independent BTK-mutated subclones, we found that most BTK-mutated cells have an additional CLL-driver gene mutation. When examining subclone-specific gene expression, we found that in one patient, BTK-mutated subclones are transcriptionally distinct from the rest of the malignant B cell population with an overexpression of CLL-relevant genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lin28, a conserved RNA-binding protein, promotes cancer stem cell features, epithelial-to-mesenchymal transition, and treatment resistance. Lin28 regulates mRNA translation, RNA stability, and transcription, which improve tumor plasticity and treatment resistance, in addition to suppressing let-7 microRNA production. Lin28 is an interesting cancer target due to its many functions; however, its structural complexity makes treatment development difficult. Lin28's proven and new non-canonical activities in cancer biology are examined in this work, focusing on cancer stem cell maintenance, metastasis, and treatment resistance. We highlight computational drug discovery advances targeting Lin28 utilizing virtual screening and machine learning. Generative artificial intelligence has made it easier to develop inhibitors for challenging targets like Lin28. This work integrates current knowledge and technology to demonstrate Lin28's therapeutic potential and outline future techniques to overcome RNA-binding protein targeting challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f268ca999e90fb84abf8448f8bf69292bbf18883" target='_blank'>
              Targeting Lin28: Insights into Biology and Advances with AI-Driven Drug Development
              </a>
            </td>
          <td>
            Victor M Matias-Barrios, Ekaterina Manskaia, Artem Cherkasov, Xuesen Dong
          </td>
          <td>2025-01-18</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The transcriptome, the complete set of RNA molecules within a cell, plays a critical role in regulating physiological processes. The advent of RNA sequencing (RNA‐seq) facilitated by Next Generation Sequencing (NGS) technologies, has revolutionized transcriptome research, providing unique insights into gene expression dynamics. This powerful strategy can be applied at both bulk tissue and single‐cell levels. Bulk RNA‐seq provides a gene expression profile within a tissue sample. Conversely, single‐cell RNA sequencing (scRNA‐seq) offers resolution at the cellular level, allowing the uncovering of cellular heterogeneity, identification of rare cell types, and distinction between distinct cell populations. As computational tools, machine learning techniques, and NGS sequencing platforms continue to evolve, the field of transcriptome research is poised for significant advancements. Therefore, to fully harness this potential, a comprehensive understanding of bulk RNA‐seq and scRNA‐seq technologies, including their advantages, limitations, and computational considerations, is crucial. This review provides a systematic comparison of the computational processes involved in both RNA‐seq and scRNA‐seq, highlighting their fundamental principles, applications, strengths, and limitations, while outlining future directions in transcriptome research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3e50ca242d05b7e586bb1a356f89c10a9bb9d7" target='_blank'>
              Bioinformatics perspectives on transcriptomics: A comprehensive review of bulk and single-cell RNA sequencing analyses
              </a>
            </td>
          <td>
            Jorge A. Tzec‐Interián, Daianna González‐Padilla, Elsa B. Góngora‐Castillo
          </td>
          <td>2025-01-06</td>
          <td>Quant. Biol.</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The ability to identify gene functions and interactions in specific cellular contexts has been greatly enabled by functional genomics technologies. CRISPR-based genetic screens have proven invaluable in elucidating gene function in mammalian cells. Single-cell functional genomics methods, such as Perturb-seq and Spear-ATAC, have made it possible to achieve high-throughput mapping of the functional effects of gene perturbations by profiling transcriptomes and DNA accessibility, respectively. Combining single-cell chromatin accessibility and transcriptomic data via multiomic approaches has facilitated the discovery of novel cis and gene regulatory interactions. However, pseudobulk readouts from cell populations can often cloud the interpretation of results due to a heterogeneous response from cells receiving the same genetic perturbation, which could be mitigated by using transcriptional profiles of single cells to subset the ATAC-seq data. Existing methods to capture CRISPR guide RNAs to simultaneously assess the impact of genetic perturbations on RNA and ATAC profiles require either cloning of gRNA libraries in specialized vectors or implementing complex protocols with multiple rounds of barcoding. Here, we introduce CAT-ATAC, a technique that adds CRISPR gRNA capture to the existing 10X Genomics Multiome assay, generating paired transcriptome, chromatin accessibility and perturbation identity data from the same individual cells. We demonstrate up to 77% guide capture efficiency for both arrayed and pooled delivery of lentiviral gRNAs in induced pluripotent stem cells (iPSCs) and cancer cell lines. This capability allows us to construct gene regulatory networks (GRNs) in cells under drug and genetic perturbations. By applying CAT-ATAC, we were able to identify a GRN associated with dasatinib resistance, indirectly activated by the HIC2 gene. Using loss of function experiments, we further validated that the gene, ZFPM2, a component of the predicted GRN, also contributes to dasatinib resistance. CAT-ATAC can thus be used to generate high-content multidimensional genotype-phenotype maps to reveal novel gene and cellular interactions and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5994854c014d82e61919b765b3c023100d1d1160" target='_blank'>
              Simultaneous single-cell CRISPR, RNA, and ATAC-seq enables multiomic CRISPR screens to identify gene regulatory relationships
              </a>
            </td>
          <td>
            Kaivalya Shevade, Yeqing Angela Yang, Kevin Feng, Karl Mader, V. Sevim, Jacob Parsons, Gunisha Arora, Hasnaa Elfawy, Rachel Mace, S. Federman, Rustam Esanov, Shawn Shafer, Eric D Chow, Laralynne Przybyla
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation, such as X-rays used during radiotherapy treatment, can kill cancer cells, but it can also cause various types of cancers depending on the treatment site that is exposed; however, many underlying mechanisms remain unclear. Here, we propose a hypothesis based on published evidence that one of the DNA repair pathways, mismatch repair, may be involved in the normal cells turning cancerous. This pathway has not been fully investigated and its roles may be underestimated in response to radiation exposure. As part of our research efforts towards characterizing radiation effects in Lynch syndrome patients, this review can inform future studies and potentially contribute to improved treatment and long-term care and management of these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2500a813ee65c9f47a96bf1021c2ef49642fe4f1" target='_blank'>
              Ionizing Radiation May Induce Tumors Partly Through the Alteration or Regulation of Mismatch Repair Genes
              </a>
            </td>
          <td>
            Mingzhu Sun, Kevin Monahan, J. Moquet, Stephen Barnard
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06ca40e33657af743c749690c1200f7526a655d0" target='_blank'>
              NKX6.3 modulation of mitotic dynamics and genomic stability in gastric carcinogenesis
              </a>
            </td>
          <td>
            J. H. Yoon, J. Kim, J. Eun, H. Ashktorab, Duane T. Smoot, S. Nam, Won Sang Park
          </td>
          <td>2025-01-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Tumor resistance to chemotherapy is a common cause of cancer recurrence in patients with head and neck squamous cell carcinoma. The goal of this study is to establish and characterize a chemoresistant laryngeal cancer cell model and test its potential utility for chemosensitizing therapy. At the genotypic level, RNA sequencing confirmed that the cells acquired putative resistance with upregulated docetaxel‐resistant (DR) genes (e.g., TUBB3, CYP24A1) and signaling pathways (e.g., PI3K/mTOR, autophagy). For phenotypic analysis, DR cells were co‐cultured with laryngeal fibroblasts in a 2‐channel microfluidic chip that mimics a hypoxic tumor core in vivo. A drug sensitivity test with a chemosensitizer, metformin (MTF), was performed on the laryngeal tumor‐on‐a‐chip. Compared to non‐treated controls, MTF‐primed cancer cells exhibit higher sensitivity to docetaxel (DTX), that is, cell death. Collectively, this resistance‐acquired cell model displayed presumed genotypic and phenotypic profiles of chemoresistance providing a viable option for testing new therapeutic strategies for restoring tumor sensitivity to DTX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6777a39362ca5671b1a3fd98435a3160aca77da" target='_blank'>
              Establishment of a chemoresistant laryngeal cancer cell model to study chemoresistance and chemosensitization responses via transcriptomic analysis and a tumor‐on‐a‐chip platform
              </a>
            </td>
          <td>
            Christian R. Moya‐Garcia, Meghana Munipalle, Alain Pacis, Nader Sadeghi, Maryam Tabrizian, N. Y. Li-Jessen
          </td>
          <td>2025-01-22</td>
          <td>Bioengineering &amp; Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eefe391651ce22dbb5bada06a6b81eaabe2e0d4" target='_blank'>
              Cancer-associated fibroblasts promote EGFR-TKI resistance via the CTHRC1/glycolysis/H3K18la positive feedback loop.
              </a>
            </td>
          <td>
            Chen Zhang, Wenxin Zhou, Hai Xu, Jiali Xu, Jun Li, Xinyin Liu, Xiyi Lu, Jiali Dai, Yuqin Jiang, Wei Wang, Erbao Zhang, Renhua Guo
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Small noncoding RNAs (smRNAs) play critical roles in regulating various cellular processes, including development, stress response, and disease pathogenesis. However, functional characterization of smRNAs remains limited by the scale and simplicity of phenotypic readouts. Recently, single-cell perturbation screening methods, which link CRISPR-mediated genetic perturbations to rich transcriptomic profiling, have emerged as foundational and scalable approaches for understanding gene functions, mapping regulatory networks, and revealing genetic interactions. However, a comparable approach for probing the regulatory consequences of smRNA perturbations is lacking. Here, we present Decoy-seq as an extension of this approach for high-content, single-cell perturbation screening of smRNAs. This method leverages U6-driven tough decoys (TuD), which form stable duplexes with their target smRNAs, for inhibition in the cell. Lentiviral-encoded TuDs are compatible with conventional single-cell RNA-sequencing (scRNA-seq) technologies, allowing joint identification of the smRNA perturbation in each cell and its associated transcriptomic profile. We applied Decoy-seq to 336 microRNAs (miRNAs) and 196 tRNA-derived fragments (tRFs) in a human breast cancer cell line, demonstrating its ability to uncover complex regulatory pathways and novel functions of these smRNAs. Notably, we show that tRFs influence mRNA polyadenylation and regulate key cancer-associated processes, such as cell cycle progression and proliferation. Therefore, Decoy-seq provides a powerful framework for exploring the functional roles of smRNAs in normal physiology and disease, and holds promise for accelerating future discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98aed8337c8669c29add6c06f708bd3a96665623" target='_blank'>
              Decoy-seq unlocks scalable genetic screening for regulatory small noncoding RNAs
              </a>
            </td>
          <td>
            Benedict Choi, Sushil Sobti, Larisa M. Soto, Trey Charbonneau, Aiden M. Sababi, A. Navickas, H. Najafabadi, Hani Goodarzi
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Drug resistance is a significant challenge in pancreatic ductal adenocarcinoma (PDAC), where stromal elements such as adipose-derived mesenchymal stem cells (ASCs) contribute to a chemoresistant tumor microenvironment (TME). This study explored the effects of oxaliplatin (OXP) and 5-fluorouracil (5-FU) on PDAC cells (Capan-1) and ASCs to investigate the mechanisms of chemoresistance. While OXP and 5-FU reduced Capan-1 viability in a dose- and time-dependent manner, ASCs demonstrated high resistance, maintaining > 90% viability even at cytotoxic doses. Transcriptomic analyses revealed OXP-induced transcriptional reprogramming in ASCs, with over 7000 differentially expressed genes, highlighting the pathways related to DNA damage response, cell cycle regulation, and stress-related signaling. In contrast, 5-FU elicited limited transcriptional changes, affecting only 192 genes. Cytokine proteome profiling revealed that OXP-treated ASCs significantly influenced the tumor microenvironment by promoting immune evasion (via IL-4, GM-CSF, IP-10, and GROα) and driving extracellular matrix remodeling (through EMMPRIN and DPPIV). In contrast, 5-FU induced comparatively weaker effects, primarily limited to hypoxia-related pathways. Although OXP reduced angiogenic factors, it paradoxically activated pro-survival pathways, thereby enhancing ASC-mediated tumor support. These findings underscore ASCs as modulators of chemoresistance via secretome alterations and stress adaptation. Therefore, future strategies should prioritize the precise targeting of tumor cells while also focusing on the development of personalized treatments to achieve durable therapeutic responses in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b0fcf2c370b9504bde08abaf1ee73d2dffcfaa3" target='_blank'>
              Chemoresistance in Pancreatic Cancer: The Role of Adipose-Derived Mesenchymal Stem Cells and Key Resistance Genes
              </a>
            </td>
          <td>
            Shahram Parvaneh, Vanda Miklós, Zoltán G Páhi, Diána Szűcs, Tamás Monostori, S. Póliska, V. Venglovecz, Tibor Pankotai, L. Kemény, Zoltán Veréb
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Biological and dynamic mechanisms by which Drug-tolerant persister (DTP) cells contribute to the development of acquired drug resistance have not been fully elucidated. Here, by integrating multidimensional data from drug-treated PC9 cells, we developed a novel multiscale mathematical model from an evolutionary perspective that encompasses epigenetic and cellular population dynamics. By coupling stochastic simulation with quantitative analysis, we identified epigenetic instability as the most prominent kinetic feature related to the emergence of DTP cell subpopulations and the effectiveness of intermittent treatment. Moreover, we revealed the optimal schedule for intermittent treatment, including the optimal area for therapeutic time and drug holidays. By leveraging single-cell RNA-seq data characterizing the drug tolerance of lung cancer, we validated the predictions made by our model and further revealed previously unrecognized biological features of DTP cells, such as cell autophagy and migration, as well as new biomarker genes of therapeutic tolerance. Our work not only provides a paradigm for the integration of multiscale mathematical models with newly emerging genomics data but also improves our understanding of the crucial roles of DTP cells and offers guidance for developing new intermittent treatment strategies against acquired drug resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b2c7b90ea2e9a9d298ad9d7420ea2b6b5b8ec4" target='_blank'>
              Integrating multiscale mathematical modeling and multidimensional data reveals the effects of epigenetic instability on acquired drug resistance in cancer
              </a>
            </td>
          <td>
            Shun-guan Wang, Jinzhi Lei, Xiufen Zou, Suoqin Jin
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, A. M. Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Prostate cancer (PCa) is a heterogenous malignancy in men, representing the second most diagnosed cancer and the fifth leading cause of cancer-related death globally in 2022. PCa incidence and progression is dependent on several predisposing factors, most significantly genetics and germline/somatic mutations. With the development of whole-genome sequencing (WGS) technology, it is now possible to determine the mutational statuses of PCa-related genes for improved clinical decision making and individualized patient care. The National Comprehensive Cancer Network (NCCN) Guidelines now recommend germline multigene testing followed by post-test genetic counseling. Depending on the germline mutations in homologous recombination (HR) repair genes or in mismatch repair (MMR) genes, specific treatment options, including poly-ADP-ribose polymerase (PARP) inhibition or PD-1 blockade, may be of more clinical benefit. This review highlights specific PCa-related germline mutations in racially diverse populations and therapeutic approaches to treat PCa patients with DNA damage repair gene alterations in advanced disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f685df01c211531229e0572b44cf7405f9d7cc" target='_blank'>
              Ancestry-Specific DNA Damage Repair Gene Mutations and Prostate Cancer
              </a>
            </td>
          <td>
            Talaibek Borbiev, Kevin Babcock, K. Sinopole, Gregory T. Chesnut, G. Petrovics
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f43e9b27ccadb97b1a4a5f9973cf63bf20b5294" target='_blank'>
              Single-cell RNA-sequencing and genome-wide Mendelian randomisation along with abundant machine learning methods identify a novel B cells signature in gastric cancer
              </a>
            </td>
          <td>
            Qi Ma, Jie Gao, Yuan Hui, Zhi-Ming Zhang, Yu-Jie Qiao, Bin-Feng Yang, Ting Gong, Duo-Ming Zhao, Bang-Rong Huang
          </td>
          <td>2025-01-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Gene therapy and genome editing have emerged as transformative approaches in the management of a diverse range of genetic and acquired diseases. This evaluation offers a thorough examination of the present state and prospects of these innovative technologies. Gene therapy is a prospective approach to the treatment and prevention of a variety of conditions, including complex cancers and inherited genetic disorders, which entail the introduction, removal, or modification of genetic material within a patient's cells. Genome editing, particularly through techniques such as CRISPR-Cas9, enables targeted corrections of genetic defects and opens new possibilities for personalized medicine by allowing for precise modifications at the DNA level. The review addresses the ethical implications, clinical applications, and significant advancements of these technologies. This article endeavors to underscore the substantial influence of gene therapy and genome editing on contemporary medicine by assessing the most recent research and clinical trials, thereby emphasizing their potential to revolutionize disease treatment and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Precision medicine has revolutionized the treatment of colorectal cancer by enabling a personalized approach tailored to each patient’s unique genetic characteristics. Genomic profiling allows for the identification of specific mutations in genes such as KRAS, BRAF, and PIK3CA, which play a crucial role in cell signaling pathways that regulate cell proliferation, apoptosis, and differentiation. This information enables doctors to select targeted therapies that inhibit specific molecular pathways, maximizing treatment effectiveness and minimizing side effects. Precision medicine also facilitates adaptive monitoring of tumor progression, allowing for adjustments in therapy to maintain treatment effectiveness. While challenges such as high costs, limited access to genomic technology, and the need for more representative genomic data for diverse populations remain, collaboration between researchers, medical practitioners, policymakers, and the pharmaceutical industry is crucial to ensure that precision medicine becomes a standard of care accessible to all. With continued advances and support, precision medicine has the potential to improve treatment outcomes, reduce morbidity and mortality rates, and enhance the quality of life for colorectal cancer patients worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ce8f60143b7348cbfaa508b5bbdc79c01b3a" target='_blank'>
              Precision medicine in colorectal cancer: genomics profiling and targeted treatment
              </a>
            </td>
          <td>
            Adi Muradi Muhar, A. J. Velaro, Arya Tjipta Prananda, S. E. Nugraha, Princella Halim, R. A. Syahputra
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69cfefb50db5bb84559d86b5f736278ac09bb363" target='_blank'>
              LINC00626 drives tamoxifen resistance in breast cancer cells by interaction with UPF1
              </a>
            </td>
          <td>
            Hui Yuan, Lianbang Zhou, Wei Hu, Min Yang
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            Mohamed J Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rationale: Anaplastic thyroid carcinoma (ATC) is an extraordinarily aggressive form of thyroid cancer, frequently presenting with locally advanced infiltration or distant metastases at the time of initial diagnosis, thus missing the optimal window for surgical intervention. Consequently, systemic chemotherapy and targeted therapies are vital for improving the prognosis of ATC. However, ATC exhibits significant resistance to conventional treatments, highlighting the need to elucidate the biological mechanisms underlying this drug resistance and identify novel therapeutic targets to overcome it. Methods: We conducted a comprehensive analysis of both bulk and single-cell RNA sequencing (scRNA-seq) data from ATC samples to screen for m 5 C modification–related genes associated with multidrug resistance (MDR). We then performed IC 50 assays, flow cytometry, and employed a spontaneous tumorigenic ATC mouse model with Nsun2 knockout to demonstrate that NSUN2 promotes MDR in ATC. To investigate the mechanisms of NSUN2-mediated drug resistance, we generated NSUN2-knockout ATC cell lines and performed transcriptomic, proteomic, and MeRIP-seq analyses. Additionally, RNA sequencing and alternative splicing analyses were conducted to determine global changes upon NSUN2 knockout. We further explored the underlying mechanisms of the NSUN2/SRSF6/UAP1 axis through glycoprotein staining, denaturing IP ubiquitination, nuclear–cytoplasmic fractionation, and PCR. Lastly, we evaluated the synergistic effects of a small-molecule NSUN2 inhibitor with anticancer agents both in vitro and in vivo . Results: Our findings reveal that NSUN2 expression correlates significantly with MDR in ATC. NSUN2 operates as a "writer" and ALYREF as a "reader" of m 5 C on SRSF6 mRNA, inducing alternative splicing reprogramming and redirecting the splice form of the UAP1 gene from AGX1 to AGX2. As a result, AGX2 enhances the N-linked glycosylation of ABC transporters, stabilizing them by preventing ubiquitination-mediated degradation. Furthermore, an NSUN2 inhibitor reduces NSUN2 enzymatic activity and diminishes downstream target expression, presenting a novel, promising therapeutic approach to overcome MDR in ATC. Conclusions: These findings suggest that the NSUN2/SRSF6/UAP1 signaling axis plays a vital role in MDR of ATC and identify NSUN2 as a synergistic target for chemotherapy and targeted therapy in ATC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de612cc82a82113a9f1d339ec0d2251cf86a8e92" target='_blank'>
              NSUN2-mediated m5C modification drives alternative splicing reprogramming and promotes multidrug resistance in anaplastic thyroid cancer through the NSUN2/SRSF6/UAP1 signaling axis
              </a>
            </td>
          <td>
            Xiukun Hou, Qiman Dong, Jie Hao, Min Liu, Junya Ning, Mei Tao, Zhongyu Wang, Fengli Guo, Dongmei Huang, Xianle Shi, Ming Gao, Dapeng Li, Xiangqian Zheng
          </td>
          <td>2025-01-27</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The human peripheral blood displays diverse molecular characteristics across populations, understanding the drivers and underlying mechanisms of which remains challenging. Here, we introduce the Chinese Immune Multi-Omics Atlas (CIMA), elucidating sex-, age-, and genetic-related molecular variations by analyzing multi-omics data from 428 adults with over 10 million immune cells. CIMA generated an enhancer-driven gene regulatory network, identifying 237 high-quality regulons and revealing cell type-specific regulatory mechanisms. Additionally, 11,521 lead cis-expression quantitative trait loci (eQTLs) and 46,339 chromatin accessibility QTLs (caQTLs) were identified at cell type level. CIMA also uncovered pleiotropic associations among immune-related disease risk loci, eQTLs, and caQTLs in a cell type-specific manner. Lastly, a novel cell language model, CIMA-CLM, was developed to predict chromatin accessibility and noncoding variant effects using chromatin sequences and gene expressions. This work represents a population-scale multi-omics resource of human immune cells, providing a valuable reference for future investigation of immune-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f8b66ae23a1d254045e2d47bbe65620395c404" target='_blank'>
              Single-Cell Genomics Elucidates Molecular Variations and Regulatory Mechanisms in Circulating Immune Cells
              </a>
            </td>
          <td>
            Jianhua Yin, Yuhui Zheng, Zhuoli Huang, Wenwen Zhou, Yue Yuan, Pengfei Cai, Yong Bai, Shichen Yang, Yue Gao, Shanshan Duan, Yang Wang, Wenxi Zhang, Xinyu Zhang, Yilin Wei, Zekai Xu, Yaling Huang, Ying Liu, Weikai Wang, Tao Yang, Jingzhi Lv, Zhongjin Zhang, Xiaoya Chen, Xiru Zhang, Fupeng Li, Yan Zhang, Guodan Zeng, Xue Wang, Wen Ma, Guixue Hou, Shijie Hao, Chang Liu, Yiwei Lai, Bo Wang, Yuxiang Li, Wenwei Zhang, Peng Gao, Jun Xie, Miguel A. Esteban, Ying Gu, Jiansong Ji, Ting Qi, Boxiang Liu, Jian Wang, Jian Yang, Xun Xu, Longqi Liu, Xin Jin, Chuanyu Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            S. Yoo, Ayushi S. Patel, S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4bf9a38d58c2137fd833608580452700c7fd92b" target='_blank'>
              Comprehensive single-cell analysis of triple-negative breast cancer based on cDC1 immune-related genes: prognostic model construction and immunotherapy potential
              </a>
            </td>
          <td>
            Linan Huang, Yiran Liu, Yulin Shi, Qi Sun, Huayao Li, Changgang Sun
          </td>
          <td>2025-02-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The cell fate transition is a fundamental characteristic of living organisms. By introducing external perturbations, it is possible to artificially intervene in cell fate and trigger cell reprogramming. Revealing the general principle underlying the induced phenotypic reshaping of cell populations remains a central focus in the field of cell biology. In this study, we investigate the energetic and dynamic features of induced cell phenotypic transition from differentiated somatic state to pluripotent state by constructing a Boolean genetic network model. The simulation and experimental results highlight the critical role of genetic frustration in initiating cell fate transitions, although the two ending phenotypic states are typically featured by minimal frustration. In addition, the altered gene expression profiles exhibit a scale-free distribution, suggesting that there exist a small number of critical genes responsible for the cell fate transition. This study provides important insights into the dynamic principles governing effective cell reprogramming caused by artificial or exogenous interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e17321e8898e722005a2009e4fe82616f4dc5" target='_blank'>
              Role of genetic frustrations in cell reprogramming
              </a>
            </td>
          <td>
            Yuxiang Yao, Jieying Zhu, Wenfei Li, Duanqing Pei
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Regulation of protein production in response to physiological signals is achieved through precise control of Eukaryotic Elongation Factor 2 (eEF2), whose distinct translocase function is crucial for cell survival. Phosphorylation of eEF2 at its Thr56 (T56) residue inactivates this function in translation. Using genetically modified paralogue of a colon cancer cell line, HCT116 which carries a point mutation at Ser595-to-Alanine in the eEF2 gene we were able to create a constitutively active form of eEF2. Then, we compared these cells with their wild-type counterparts  under complete growth and nutrient deprivation conditions. The mutant forms were created by using CRISPR/Cas9-mediated gene cassette knock-in (KI) strategy. The samples were analyzed by a microarray based approach. Carrying out a WikiPath analysis, we identified distinct signaling pathways and gene expression patterns that are differentially regulated in the mutant cells (HCT116KI), but not in wild-type HCT116 (HCT116WT) cells, particularly under nutrient deprived conditions. Our results suggest that the constitutive activation of eEF2 enables increase in the malignant gene expression in colon cancer cells compared to the wild type counterparts under nutrient deprived conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7a023295ed87a1f8110a5f37ed0be7b05014ddf" target='_blank'>
              Improved Malignancy of Colon Cancer Cells at Gene Expression Level with Constitutive Activation of the Eukaryotic Elongation Factor 2 (eEF2) Under Nutrition Deficient Conditions.
              </a>
            </td>
          <td>
            A. H. Deniz, E. Aydemir, Nur Ekimci Gürcan, Abdulahi Dabo Jakir, Ö. Bayrak, F. Şahin, Furkan Ayaz
          </td>
          <td>2025-01-13</td>
          <td>Chemistry & biodiversity</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W. Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M. Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Natural Killer (NK) cell therapy has emerged as a transformative approach in cancer immunotherapy, driven by decades of research since their discovery in the 1970s. NK cells possess a unique ability to target tumor cells through antigen-independent recognition, which has been expanded with the discovery of antigen-specific memory and licensing mechanisms. This review explores the evolution of NK cell biology, highlighting key findings such as the enhanced response of NK cells upon re-exposure to cytomegalovirus (CMV), defying conventional immune paradigms. Recent advancements in gene editing, particularly with CRISPR technology, have led to the development of "armored" NK cells. For example, knock-in strategies have enabled the insertion of chimeric antigen receptors (CAR), enhancing tumor specificity, while knock-out approaches have improved NK cell activation by removing inhibitory molecules like PD-1 and NKG2A. Early clinical trials demonstrate improved tumor-targeting efficacy and reduced resistance. However, challenges remain, including high R&D costs, limited clinical data, and technical uncertainties such as off-target effects. In China, NK cell therapy is in its nascent stages, with increasing interest but limited clinical applications due to regulatory and funding barriers. This review provides a comprehensive analysis of NK cell evolution, CRISPR-based advancements, and future directions for overcoming current limitations and optimizing therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is one of the most prevalent causes of cancer-related morbidity and mortality worldwide. Late-stage detection and the complex molecular mechanisms driving tumor progression contribute significantly to its poor prognosis. Dysregulated R-loops, three-stranded nucleic acid structures associated with genome instability, play a key role in the malignant characteristics of various tumors. However, the detailed role and mechanism of R-loops in HCC progression remain elusive and require further exploration. This study aimed to construct an R-loop scoring signature centered on prognosis and lipid metabolism, thereby enhancing our understanding of HCC progression and identifying potential therapeutic targets. Methods In this study, we utilized the single-cell RNA-sequencing (scRNA-seq) data from HCC patients (GSE149614 and CRA002308) to construct an R-loop scoring model based on the identified R-loop regulator genes (RLRGs) related to HBV infection through WGCNA analysis. We also explored the tumor microenvironment and intercellular communication related to R-loop score. Additionally, a prognostic risk model based on the fatty acid metabolism-associated RLRGs was constructed using data from the TCGA database, and its association with immune infiltration, mutations, and drug sensitivity was analyzed. In vitro and in vivo experiments were performed to investigate the role of RLRG CLTC in lipid metabolism and HCC progression. Results Using scRNA-seq data from HCC, we established an R-loop scoring model based on identified RLRGs related to HBV infection. Moreover, the more suppressive tumor immune microenvironment and stronger intercellular communication were displayed in malignant cells with high R-loop scores. The cell trajectory and cellular metabolism analysis exhibited a significant association between lipid metabolism and RLRGs. Additionally, we constructed a prognostic risk model consisting of 8 RLRGs related to fatty acid metabolism, which effectively evaluated the prognostic value, status of tumor immune microenvironment, gene mutations, and chemotherapeutic drug sensitivity for HCC patients. Notably, validation experiments suggested that CLTC could regulate lipid metabolism through R-loop formation and facilitate tumor progression in HCC. Conclusion Collectively, our study proposes an R-loop scoring model associated with tumor immune microenvironment, lipid metabolism and prognostic value. CLTC, an R-loop regulator, emerges as a promising prognostic biomarker and therapeutic target, offering new insights into potential treatment strategies for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cbaecb70e4d3bf1b08fe7b03162bc2296819cdc" target='_blank'>
              Integrated single-cell and bulk transcriptome analysis of R-loop score-based signature with regard to immune microenvironment, lipid metabolism and prognosis in HCC
              </a>
            </td>
          <td>
            Long Chen, Houzhi Yang, Xianfu Wei, Jianchao Liu, Xiuxin Han, Chao Zhang, Yongheng Liu, Yan Zhang, Yao Xu, Yiqin Li, Guowen Wang, Jinyan Feng
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Based on the notion that hypomorphic germline genetic variants are linked to autoimmune diseases, we reasoned that novel targets for cancer immunotherapy might be identified through germline variants associated with greater T-cell infiltration into tumors. Here, we report that while investigating germline polymorphisms associated with a tumor immune gene signature, we identified PKCδ as a candidate. Genetic deletion of PKCδ in mice resulted in improved endogenous antitumor immunity and increased efficacy of anti-PD-L1. Single-cell RNA sequencing revealed myeloid cell expression of Prkcd, and PKCδ deletion caused a shift in macrophage gene expression from an M2-like to an M1-like phenotype. Conditional deletion of PKCδ in myeloid cells recapitulated improved tumor control that was augmented further with anti-PD-L1. Analysis of clinical samples confirmed an association between PRKCD variants and M1/M2 phenotype, as well as between a PKCδ KO-like gene signature and clinical benefit from anti-PD-1. Our results identify PKCδ as a candidate therapeutic target that modulates myeloid cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31ff12050b59b63f4c02e53380b59da37e0eb557" target='_blank'>
              PKCδ germline variants and genetic deletion in mice augment antitumor immunity through regulation of myeloid cells.
              </a>
            </td>
          <td>
            Kyle R Cron, A. Sivan, K. Aquino-Michaels, Andrea Ziblat, Emily F. Higgs, R.F. Sweis, Ruxandra Tonea, Seoho Lee, Thomas F Gajewski
          </td>
          <td>2025-01-14</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9cc66b33283e972f3bd943131a689f9bed390bc" target='_blank'>
              Oncogenic TFE3 fusions drive OXPHOS and confer metabolic vulnerabilities in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Jiao Li, Kaimeng Huang, Meha Thakur, Fiona McBride, Ananthan Sadagopan, Daniel S. Gallant, Prateek Khanna, Y. N. Laimon, Bingchen Li, R. Mohanna, Maolin Ge, Cary N. Weiss, M. Achom, Qingru Xu, Sayed Matar, G. M. Lee, Kun Huang, Miao Gui, Chin-Lee Wu, Kristine M. Cornejo, T. Choueiri, Birgitta A. Ryback, S. Signoretti, L. Bar-Peled, S. Viswanathan
          </td>
          <td>2025-02-06</td>
          <td>Nature metabolism</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) carries a poor prognosis, and new therapeutic strategies are necessary to improve outcomes for patients with this disease. Alkylating chemotherapies including temozolomide (TMZ) and lomustine (CCNU) are critical for treating GBM, but resistance mechanisms, including hypomethylation of O6-methylguanine-DNA methyltransferase (MGMT) promoter, undermine treatment. CRISPRoff is a programmable epigenetic memory editor that can induce stable and heritable gene silencing after transient delivery, and we hypothesize that CRISPRoff could potentiate the activity of TMZ and CCNU through long term suppression of target genes.


METHODS
We transiently delivered CRISPRoff mRNA along with sgRNAs against target genes using both electroporation and lipid nanoparticles (LNPs) into established GBM cell lines, patient-derived primary GBM cultures, and orthotopic GBM xenografts. Gene repression, specificity, and stability was measured by RT-qPCR, Western blot, bisulfite sequencing, and RNA-sequencing. Sensitivity to chemotherapies was measured by cell viability dose response, microscopy, and bioluminescence imaging. Genome-wide mapping of CCNU sensitizers was performed using CRISPRi screens.


RESULTS
CRISPRoff induced complete suppression of MGMT and sensitization to TMZ that were stable for over 8 months of continuous cell propagation. GBM orthotopic tumors treated with CRISPRoff against MGMT demonstrated sensitivity to TMZ in vivo, and CRISPRoff delivery resulted in chemosensitivity in patient-derived primary GBM. Genome-wide CRISPRi screens identified combinatorial genetic vulnerabilities (BRIP1, FANCE) that were targetable by multiplexed CRISPRoff to achieve sensitization to CCNU.


CONCLUSION
Transient delivery of a site-specific epigenetic memory can induce stable, complete, and multiplexed suppression of target genes for therapeutic application in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The translation of mRNA into proteins in multicellular organisms needs to be carefully tuned to changing proteome demands in development and differentiation, and defects in translation often have a disproportionate impact in distinct cell types. Here we used inducible CRISPR interference screens to compare the essentiality of genes with functions in mRNA translation in human induced pluripotent stem cells (hiPSC) and hiPSC-derived neural and cardiac cells. We find that core components of the mRNA translation machinery are broadly essential, but the consequences of perturbing translation-coupled quality control factors are highly cell type-dependent. Human stem cells critically depend on pathways that detect and rescue slow or stalled ribosomes, and on the E3 ligase ZNF598 to resolve a novel type of ribosome collisions at translation start sites on endogenous mRNAs with highly efficient initiation. Our findings underscore the importance of cell identity for deciphering the molecular mechanisms of translational control in metazoans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936e5c6a732cad6aa95052ce36d49c24924935ca" target='_blank'>
              Comparative CRISPRi screens reveal a human stem cell dependence on mRNA translation-coupled quality control
              </a>
            </td>
          <td>
            Geraldine Rodschinka, Sergio Forcelloni, Andrew Behrens, Henrick Riemenschneider, D. Edbauer, Sascha Wani, Danny D Nedialkova
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Alternative splicing (AS) is a mechanism that generates translational diversity within a genome. Equally important is the dynamic adaptability of the splicing machinery, which can give preference to one isoform over others encoded by a single gene. These isoform preferences change in response to the cell’s state and function. Particularly significant is the impact of physiological alternative splicing in T lymphocytes, where specific isoforms can enhance or reduce the cells’ reactivity to stimuli. This process makes splicing isoforms defining features of cell states, exemplified by CD45 splice isoforms, which characterize the transition from naïve to memory states. Two developments have accelerated the use of AS dynamics for therapeutic interventions: advancements in long-read RNA sequencing and progress in nucleic acid chemical modifications. Improved oligonucleotide stability has enabled their use in directing splicing to specific sites or modifying sequences to enhance or silence particular splicing events. This review highlights immune regulatory splicing patterns with potential significance for enhancing anticancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaaff8ba550e599aee86ffb2e27658c31cc7d97" target='_blank'>
              Alternative splicing of modulatory immune receptors in T lymphocytes: a newly identified and targetable mechanism for anticancer immunotherapy
              </a>
            </td>
          <td>
            Shay Tzaban, Ori Stern, Elad Zisman, G. Eisenberg, Shiri Klein, S. Frankenburg, Michal Lotem
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Oncolytic herpes simplex viruses (oHSVs) are a type of biotherapeutic utilized in cancer therapy due to their ability to selectively infect and destroy tumor cells without harming healthy cells. We sought to investigate the functional genomic response and altered metabolic pathways of human cancer cells to oHSV-1 infection and to elucidate the influence of these responses on the relationship between the virus and the cancer cells. Methods Two datasets containing gene expression profiles of tumor cells infected with oHSV-1 (G207) and non-infected cells from the Gene Expression Omnibus (GEO) database were processed and normalized using the R software. Common differentially expressed genes between datasets were selected to identify hub genes and were further analyzed. Subsequently, the expression of hub genes was verified by real-time polymerase chain reaction (qRT-PCR) in MDA-MB-231 (a breast cancer cell line) infected with oHSV-1 and non-infected cells. Results The results of our data analysis indicated notable disparities in the genes associated with the proteasome pathway between infected and non-infected cells. Our ontology analysis revealed that the proteasome-mediated ubiquitin-dependent protein catabolic process was a significant biological process, with a p-value of 5.8E−21. Additionally, extracellular exosomes and protein binding were identified as significant cellular components and molecular functions, respectively. Common hub genes with degree and maximum neighborhood component (MNC) methods, including PSMD2, PSMD4, PSMA2, PSMD14, PSMD11, PSMC3, PSMC2, PSMD8, and PSMA4, were also identified. Analysis of gene expression by qRT-PCR and differential gene expression revealed that GADD45g genes can be effective genes in the proliferation of oncolytic HSV-1 virus. Conclusion The transcriptome changes in tumor cells infected by oHSV-1 may be utilized to predict oncolytic efficacy and provide rational strategies for designing next-generation oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0ce55c7be9d2abb820866201dacdc5afac972e" target='_blank'>
              Rational strategies for designing next-generation oncolytic viruses based on transcriptome analysis of tumor cells infected with oncolytic herpes simplex virus-1
              </a>
            </td>
          <td>
            Naeme Javid, Shahriyar Abdoli, Majid Shahbazi
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are a class of noncoding RNAs that regulate gene expression in cancer cells and Cancer Stem Cells (CSCs), and play an indispensable role in the development of resistance to radiotherapy and chemotherapy in cancer treatment. The dysregulation of miRNAs in CSCs plays a vital role in influencing the features of CSCs and their resistance mechanisms in different forms of cancer. These miRNAs are also engaged in regulating important signaling pathways, such as Notch and Wnt, which have an impact on the behavior of CSCs and their response to treatment. The intricate network of miRNAs and their interactions with essential proteins and major pathways highlight their potential as therapeutic targets for controlling cancer stem cell behavior and improving the outcomes of cancer therapy by understanding their effects. Comprehending the complex relationship among the production of microRNA, the environment surrounding the tumor, and the processes by which cells communicate can provide vital knowledge for developing successful treatments to overcome resistance to chemotherapy and radiation in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/762e3db35874e67846ed4cc1432dc55fa5b78f07" target='_blank'>
              MicroRNA-Mediated Regulation of Cancer Stem Cells: Implications for Radioresistance and Chemoresistance in Cancer Treatment.
              </a>
            </td>
          <td>
            Vasanth Kanth Thasma Loganathbabu, Harin N Ganesh, Ravi Gor, K. N. Aruljothi, Satish Ramalingam
          </td>
          <td>2025-02-19</td>
          <td>MicroRNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tetraploidy is a condition in which the entire set of chromosomes doubles, most often due to errors during cell division. Tetraploidy can lead to genomic instability and significant consequences, in particular metastasis and treatment failure in tumours, including radiotherapy. The development of new strategies to sensitise these cells to treatment is of great importance. In our study, we investigated the in vitro combination of chemical treatment with the kinase inhibitor SP600125 and irradiation on diploid versus metastatic tetraploid RKO colon cancer clones. We assessed mitochondrial transmembrane potential, cell cycle and subG1 population by flow cytometry and performed clonogenic assays to evaluate cell sensitivity. We found that the combination overcomes irradiation resistance in metastatic tetraploid clones. To identify the main pathway involved in cell sensitivity, we screened the Harvard Medical School KINOMEscan library and performed a gene ontology biological process analysis. We found that the major kinases inhibited by SP600125 were ANKK1, BIKE, IKKA, JNK1, MP2K3, MP2K4, MKNK2, MYLK, PLK4, RPS6KA4(Kin,Dom,1), MYLK4 and TTK, and the pathways involved in clone sensitivity were DNA damage repair, radiation resistance and apoptosis, through JNK pathway inhibition. Finally, our main finding was that combined treatment with SP600125 and radiotherapy reduced the resistance of metastatic tetraploid cells to treatment, essentially by inhibiting the JNK pathway. This result supports a promising anti-cancer strategy to overcome the resistance of tetraploid cancer cells to irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0380fc971abb59db901b3b88a1c597ae6b670f0" target='_blank'>
              JNK Inhibition Overcomes Resistance of Metastatic Tetraploid Cancer Cells to Irradiation-Induced Apoptosis
              </a>
            </td>
          <td>
            Mohamed Jemaà, N. Setti Boubaker, Nesrine Kerkeni, Stephan M. Huber
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Neoadjuvant, endocrine, and targeted therapies have significantly improved the prognosis of breast cancer (BC). However, due to the high heterogeneity of cancer, some patients cannot benefit from existing treatments. Increasing evidence suggests that amino acids and their metabolites can alter the tumor malignant behavior through reshaping tumor microenvironment and regulation of immune cell function. Breast cancer cell lines have been identified as methionine-dependent, and methionine restriction has been proposed as a potential cancer treatment strategy. Methods We integrated transcriptomic and single-cell RNA sequencing (ScRNA-seq) analyses based on The Cancer Genome Atlas (TCGA) database and Gene Expression Omnibus (GEO) datasets. Then we applied weighted gene co-expression network analysis (WGCNA) and Cox regression to evaluate methionine metabolism-related genes (MRGs) in BC, constructing and validating a prognostic model for BC patients. Immune landscapes and immunotherapy were further explored. Finally, in vitro experiments were conducted to assess the expression and function of key genes APOC1. Results In this study, we established and validated a prognostic signature based on eight methionine-related genes to predict overall survival (OS) in BC patients. Patients were further stratified into high-risk and low-risk groups according to prognostic risk score. Further analysis revealed significant differences between two groups in terms of pathway alterations, immune microenvironment characteristics, and immune checkpoint expression. Our study shed light on the relationship between methionine metabolism and immune infiltration in BC. APOC1, a key gene in the prognostic signature, was found to be upregulated in BC and closely associated with immune cell infiltration. Notably, APOC1 was primarily expressed in macrophages. Subsequent in vitro experiments demonstrated that silencing APOC1 reduced the generation of tumor-associated macrophages (TAMs) with an M2 phenotype while significantly decreasing the proliferation, invasion, and migration of MDA-MB-231 and MDA-MB-468 breast cancer cell lines. Conclusion We established a prognostic risk score consisting of genes associated with methionine metabolism, which helps predict prognosis and response to treatment in BC. The function of APOC1 in regulating macrophage polarization was explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d865c02ec071f3b620933e096d65cab7e82d29" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to explore prognostic value and immune landscapes of methionine metabolism-related signature in breast cancer
              </a>
            </td>
          <td>
            Yanxian Gao, Ziyu Feng, Hailong Zhao, Xinghai Liu, Muyu Zhu, Xiafei Yu, Xiaoan Liu, Xian Wu, Jing Tao
          </td>
          <td>2025-01-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Purpose: Small cell lung cancer (SCLC) is a highly aggressive and deadly malignancy. Two major factors contributing to the high mortality of SCLC are early metastasis and rapid development of therapy resistance. Recent research suggests upregulation of the epithelial-mesenchymal transition (EMT) program and the EMT transcription factor Twist1 correlated with accelerated tumor progression and chemoradiation (CRT) resistance in SCLC. However, a causal relationship between Twist1 and these aspects of SCLC biology has not been rigorously studied. Here, we investigated whether Twist1 upregulation could promote SCLC tumorigenesis, metastasis, and resistance to CRT. Materials and Methods: We analyzed transcriptomic data from the IMpower133 phase 3 trial of extensive stage SCLC stratified by median TWIST1 levels using Kaplan-Meir statistic. To investigate the roles of Twist1 directly in SCLC biology, we have generated a novel genetically engineered mouse model (GEMM) termed RPGT (Rb1Flox; Trp53Flox; ROSA26LSL-rtTA-IRES-EGFP; Twist1-TetO7-Luc). The RPGT GEMM enables the generation of autochthonous SCLC tumors after induction with Cre recombinase adenovirus. By withdrawing or providing doxycycline to mice, we can control Twist1 expression to activate or inactivate EMT in tumors. By characterizing mouse tumor samples with histological, immunostaining, and transcriptomic profiling analyses, we evaluated the impact of Twist1 and EMT upregulation on SCLC tumor biology and plasticity as well as metastatic spread and outgrowth. To evaluate the impact of EMT activation on SCLC sensitivity to CRT, we performed in vitro viability assays on primary tumor cell lines established from RPGT tumors. We also treated RPGT mice with vs. without Twist1 overexpression with CRT to validate our in vitro data. Results: In the chemotherapy alone control arm of IMpower133 there was a trend towards a difference in overall survival (OS) between TWIST1-high or -low patients (p=025 by log-rank) and in the NMF1/NEUROD1 subtype, patients with TWIST1-high had statistically significantly inferior OS (p=0.01). We then observed that both control (no Twist1 overexpression) and Twist1-overexpressing RPGT mice developed neuroendocrine SCLC tumors. While SCLC-ASCL1 was the predominant subtype in both cohorts, RPGT tumors exhibited more plasticity, with features associated with SCLC-NEUROD1 subtype. Furthermore, Twist1 overexpression dramatically increased the metastatic incidence in RPGT compared to control animals, indicating an important role of EMT in SCLC dissemination. Transcriptomic profiling of primary tumors and matching metastases in RPGT mice also revealed downregulation of Twist1 and EMT in metastases, suggesting that EMT suppression was necessary for metastatic outgrowth. Furthermore, we found that repressing Twist1 expression enhanced SCLC susceptibility to CRT both in vitro and in vivo. Conclusions: Overall, our data suggest that TWIST1/Twist1 plays an important role in promoting SCLC plasticity, metastasis, and treatment resistance.
 Citation Format: Triet Nguyen, Jinhee Chang, Kathleen Gabrielson, Amol Shetty, Yang Song, Apaala Chatterjee, Audrey Lafargue, Yoo Sun Kim, Danielle Council, Aaron Chan, Dipanwita Dutta Chowdhury, Muhammad Ajmal Khan, Nick Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, Christine Hann, Phuoc T. Tran. TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a1d1d7c8c25abea1123d9456e9598c5d89538a" target='_blank'>
              Abstract B029: TWIST1 Associates with resistance to treatment and Twist1 drives tumor progression in vivo for small cell lung cancer
              </a>
            </td>
          <td>
            Triet Nguyen, J. Chang, Kathleen L Gabrielson, A. Shetty, Yang Song, Apaala Chatterjee, A. Lafargue, Yoo Sun Kim, Danielle N. Council, Aaron Chan, D. Chowdhury, Muhammad Ajmal Khan, N. Connis, Daniel Sforza, Eric Gardner, Christopher McFarland, Mohammad Rezaee, Nitin Roper, C. Hann, Phuoc T Tran
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/051f9e93e924c3e30e3096e0cd89e4a17082501b" target='_blank'>
              Integrative analysis of genetic variability and functional traits in lung adenocarcinoma epithelial cells via single-cell RNA sequencing, GWAS, bayesian deconvolution, and machine learning.
              </a>
            </td>
          <td>
            Chenggen Gao, Jintao Wu, Fangyan Zhong, Xianxin Yang, Hanwen Liu, Junming Lai, Jing Cai, Weimin Mao, Huijuan Xu
          </td>
          <td>2025-02-24</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The epithelial–mesenchymal transition (EMT) is a dynamic transdifferentiation of epithelial cells into mesenchymal cells. EMT programs exhibit great diversity, based primarily on the distinct impact of molecular activities of the EMT transcription factors. Using a panel of cancer cell lines and a series of 71 triple-negative primary breast tumors, we report that the EMT transcription factor ZEB1 modulates site-specific chemical modifications of ribosomal RNA (rRNA). Overexpression of ZEB1 and ZEB2, but not TWIST1, decreased the level of 2′-O-ribose methylation (2′Ome) of 28S rRNA at position Um2402. ZEB1 overexpression specifically reduced the expression of the corresponding C/D box small nucleolar RNAs (snoRNAs) SNORD143/144, which guide the rRNA 2′Ome complex at the 28S_Um2402 site. During ZEB1-induced EMT induction/reversion, the levels of both 2′Ome at 28S_Um2402 and SNORD143/144 were dynamically comodulated. Taken together, these data demonstrate that 2′Ome rRNA epitranscriptomics is a novel marker of ZEB1-induced EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3736219d26e530108dcb3de1bf5b51cdbebaad" target='_blank'>
              Specific modulation of 28S_Um2402 rRNA 2′-O-ribose methylation as a novel epitranscriptomic marker of ZEB1-induced epithelial–mesenchymal transition in different mammary cell contexts
              </a>
            </td>
          <td>
            Chloé Morin, Hermes Paraqindes, Flora Nguyen Van Long, Caroline Isaac, Emilie Thomas, D. Pedri, Carlos Ariel Pulido-Vicuña, A. Morel, V. Marchand, Y. Motorin, Marjorie Carrere, Jessie Auclair, V. Attignon, Roxane M. Pommier, Emmanuelle Ruiz, Fleur Bourdelais, Frédéric Catez, Sébastien Durand, Anthony Ferrari, Alain Viari, J. Marine, Alain Puisieux, Jean-Jacques Diaz, C. Moyret-Lalle, V. Marcel
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Phenotypic outcomes can be heavily affected by environmental factors. In this study, we exploited the previously observed nutrient-dependency of cell biological phenotypic features, captured by a cross-condition image-based profiling assay of Escherichia coli deletion strains, to examine this in more detail. We identified several general principles, including the existence of a spectrum of deviating phenotypes across nutrient conditions (i.e., from nutrient- or feature-specific to pleiotropic phenotypic deviations), limited conservation of phenotypic deviations across nutrient conditions (i.e., limited phenotypic robustness), and a subset of nutrient-independent phenotypic deviations (indicative of consistent genetic determinants of specific phenotypic features). In a subsequent step, we used this cross-condition dataset to identify five genes of unknown function of which the deletion displayed either nutrient-independent phenotypic deviations or phenotypic similarities to genes of known function: yibN, yaaY, yfaQ, ybiJ, and yijD. These genes showed different levels of phylogenetic conservation, ranging from conserved across the tree of life (yibN) to only present in some genera of the Enterobacterales (yaaY). Analysis of the structural properties of the proteins encoded by these y-genes, identification of structural similarities to other proteins, and the examination of their subcellular localization yielded new insights into their contribution to E. coli cell morphogenesis, cell cycle progression and cell growth. Together, our approach showcases how bacterial image-based profiling assays and datasets can serve as a gateway to reveal the function of uncharacterized proteins. Importance Despite unprecedented access to genomic information, predicting phenotypes based on genotypes remains notoriously difficult. One major confounding factor is the environment and its ability to modulate phenotypic outcomes. Another is the fact that a large fraction of protein-coding genes in bacterial genomes remains uncharacterized and have no known function. In this work, we use a large-scale cross-condition image-based profiling dataset to characterize nutrient-dependent phenotypic variability of E. coli deletion strains and exploit it to provide insight into the cellular role of genes of unknown function. Through our analysis, we identified five genes of unknown function that we subsequently further characterized by examining their phylogenetic conservation, predicted structural properties and similarities, and their intracellular localization. Combined, this approach highlights the potential of cross-condition image-based profiling, which extracts many cell biological phenotypic readouts across multiple conditions, to better understand nutrient-dependent phenotypic variability and uncover protein function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e64fdb79587408307645dda0d8efcc1410cda8f" target='_blank'>
              Complex interplay between gene deletions and the environment uncovers cellular roles for genes of unknown function in Escherichia coli
              </a>
            </td>
          <td>
            Kaat Sondervorst, Kristina Nesporova, Matthew Herdman, Bart Steemans, Joëlle Rosseels, Sander K. Govers
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79d34836731e3bc98f26657fe4be3cd3ea9eb09" target='_blank'>
              The systemic evolutionary theory of the origin of cancer (SETOC): an update
              </a>
            </td>
          <td>
            Antonio Mazzocca, Giovanni Ferraro, Giovanni Misciagna
          </td>
          <td>2025-01-14</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, K. Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa8620cd6b72ee59d0d49a55c5c679295c6c197" target='_blank'>
              Long noncoding RNA GDIL acts as a scaffold for CHAC1 and XRN2 to promote platinum resistance of colorectal cancer through inhibition of glutathione degradation
              </a>
            </td>
          <td>
            Xuan Deng, Lu Chang, Lingyu Tang, Haoqin Jiang, Xiao Xu, Xin-ju Zhang, Jian Chen, Liu Dong, Qianqian Xu, Ruoshui Cao, Jianbin Xiang, Ming Guan
          </td>
          <td>2025-02-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Constitutively active mutations of KRAS are prevalent in non-small cell lung cancer (NSCLC). However, the relationship between these mutations and resistance to platinum-based chemotherapy and the underlying mechanisms remain elusive. In this study, we demonstrated that KRAS mutants confer resistance to platinum in NSCLC. Mechanistically, KRAS mutants mediate platinum resistance in NSCLC cells by activating ERK/JNK signaling, which inhibits ALKBH5 m6A demethylase activity by regulating post-translational modifications (PTMs) of ALKBH5. Consequently, the KRAS mutant leads to a global increase in m6A methylation of mRNAs, particularly DDB2 and XPC, which are essential for nucleotide excision repair. This methylation stabilized the mRNA of these two genes, thus enhancing NSCLC cells' ability to repair platinum-induced DNA damage and avoid apoptosis, thereby contributing to drug resistance. Furthermore, blocking KRAS-mutant-induced m6A methylation, either by overexpressing a SUMOylation-deficient mutant of ALKBH5, or by inhibiting METTL3 pharmacologically, significantly sensitizes KRAS-mutant NSCLC cells to platinum drugs in vitro and in vivo. Collectively, our study uncovers a previously unrecognized mechanism that mediates KRAS mutant-induced chemoresistance in NSCLC cells by activating DNA repair through the modulation of the ERK/JNK/ALKBH5 PTMs-induced m6A modification in DNA damage repair-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b30cd92ed6de2a868acc5ae23a99dadaae25a64" target='_blank'>
              KRAS Mutants Confer Platinum Resistance by Regulating ALKBH5 Post-translational Modifications in Lung Cancer.
              </a>
            </td>
          <td>
            Fang Yu, Shikan Zheng, Chunjie Yu, Sanhui Gao, Zuqi Shen, Rukiye Nar, Zhexin Liu, Shuang Huang, Lizi Wu, Tongjun Gu, Zhijian Qian
          </td>
          <td>2025-02-04</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CART) cell therapy is an innovative form of targeted immunotherapy that has revolutionized the treatment of cancer. However, the durable response remains limited. Recent studies have shown that the epigenetic landscape of preinfusion CART cell products can influence response to therapy, and gene editing has been proposed as a solution. However, more work needs to be done to determine the optimal gene editing strategy. Genome-wide CRISPR screens have become popular tools to both investigate mechanisms of resistance and optimize gene editing strategies. Yet their application to primary cells presents many challenges. Here we describe a method to complete a genome-wide CRISPR knockout screen in CART cells from healthy donors. As a proof-of-concept model, we designed this method to investigate the development of exhaustion in CART cells targeting the CD19 antigen. However, we believe that this approach can be used to address a variety of mechanisms of resistance to therapy in different CAR constructs and tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bc121c6fca52318c6e934753748ac46a988822d" target='_blank'>
              Integrating single-cell RNA-Seq and bulk RNA-Seq data to explore the key role of fatty acid metabolism in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hua Dai, Xin Tao, Yuansen Shu, Fanrong Liu, Xiaoping Cheng, Xiushen Li, Bairui Shu, Hongcheng Luo, XuXiang Chen, Zhaorui Cheng
          </td>
          <td>2025-01-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) comprise a group of genetically heterogeneous hematological malignancies that result in the abnormal growth of leukemic cells and halt the maturation process of normal hematopoietic stem cells. Despite using molecular and cytogenetic risk classification to guide treatment decisions, most AML patients survive for less than five years. A deeper comprehension of the disease’s biology and the use of new, targeted therapy approaches could potentially increase cure rates. RAS oncogene mutations are common in AML patients, being observed in about 15–20% of AML cases. Despite extensive efforts to find targeted therapy for RAS-mutated AMLs, no effective and tolerable RAS inhibitor has received approval for use against AMLs. The frequency of RAS mutations increases in the context of AMLs’ chemoresistance; thus, novel anti-RAS strategies to overcome drug resistance and improve patients’ therapy responses and overall survival are the need of the hour. In this article, we aim to update the current knowledge on the role of RAS mutations and anti-RAS strategies in AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607005d4dc391e9d5c0e31b65105281e28950846" target='_blank'>
              Rat Sarcoma Virus Family Genes in Acute Myeloid Leukemia: Pathogenetic and Clinical Implications
              </a>
            </td>
          <td>
            Shaimaa Khattab, Adriatik Berisha, Natalia Baran, P. Piccaluga
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05abf56ac7febcd59ec911620962ed978222a49b" target='_blank'>
              Copy number amplification of FLAD1 promotes the progression of triple-negative breast cancer through lipid metabolism
              </a>
            </td>
          <td>
            Xiao-Qing Song, Tian-Jian Yu, Ou-Yang Yang, Jianlong Ding, Yizhou Jiang, Zhi-Ming Shao, Yi Xiao
          </td>
          <td>2025-02-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that continued expression is required for maintaining their leukemic phenotype and they therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the leading causes of cancer-related mortality worldwide. In recent years, circular RNAs (circRNAs), a novel class of non-coding RNA molecules, have emerged as a research focus due to their unique stability and functional roles. CircRNAs regulate tumor-related signaling pathways through interactions with microRNAs (miRNAs) and proteins, playing key roles in tumorigenesis, progression, invasion, metastasis, and chemoresistance. This review summarizes the role of circRNAs in CRC, particularly their mechanisms in cell proliferation, migration, apoptosis, tumor microenvironment (TME) remodeling, and immune evasion. Aberrant expression of circRNAs holds great potential as diagnostic and prognostic biomarkers as well as therapeutic targets for CRC. Additionally, natural products such as flavonoids and glycosides, by modulating circRNA-miRNA-mRNA networks, offer promising therapeutic strategies. The article also discusses the current technical challenges in circRNA research and its future application prospects in CRC, highlighting the need for further investigation into the role of circRNAs in tumor immune microenvironments and drug resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39f8c7e9b28a1eb7bde937c631440ba71c34967c" target='_blank'>
              CircRNAs in colorectal cancer: potential roles, clinical applications, and natural product-based regulation
              </a>
            </td>
          <td>
            Jingjing Yang, Qiqi Fan, Yiyun Wang, Yuanyue Liu, Xiaoning Xu, Yeqi Liang, Jiakang Xie, Jiajie Li, Fengting Ai, Yong Cao, Shangzhen Yu, Jinman Liu
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Patients with metastatic castration-resistant prostate cancer (mCRPC) are generally resistant to immune checkpoint inhibitors (ICIs), partly due to the presence of immunosuppressive myeloid cells within tumors. However, broad targeting strategies, such as CSF1R antagonism, have faced clinical limitations given the heterogeneity of these cells. We hypothesize that a more precise understanding of specific immunosuppressive myeloid subsets and their molecular mechanisms at the single-cell level will be essential in the development of effective immunotherapies. Methods: We performed single-cell profiling on tumor biopsies across patients with localized (n=13), metastatic hormone-sensitive (n=24), and metastatic castration-resistant (n=6) prostate cancer. Results from these analyses were reverse-translated into multi syngeneic mouse models of prostate cancer, where we conducted multi-omic single-cell assessments and mechanistic studies. Findings were subsequently correlated with clinical outcomes. Results: Single-cell analysis of patient biopsies identified a distinct subset of tumor-associated macrophages with high expression of SPP1 (SPP1 hi-TAMs), encoding the osteogenic protein osteopontin (OPN), which were more prevalent in mCRPC than in earlier disease stages. These macrophages showed elevated immunosuppressive molecular programs and significantly lower CSF1R expression than other macrophages, potentially explaining the limited efficacy of CSF1R blockade in mCRPC patients. Multi-omic profiling of syngeneic mouse models of prostate cancer (MyC-CaP and TRAMP-C2) identified an analogous macrophage subset resistant to CSF1R blockade, which strongly suppressed CD8+ T cell activity in co-culture. Adoptive transfer of Spp1 hi-TAMs into CRPC increased the frequency of exhausted CD8+ T cells, promoted ICI resistance and worsened survival outcomes. Pathway analysis implicated adenosine signaling as a possible mechanism. Blocking adenosine pathways, either with the A2AR inhibitor ciforadenant or a CD73-blocking antibody, significantly reversed CD8+ T cell suppression by Spp1 hi-TAMs in vitro and enhanced the sensitivity of CRPC cells to PD-1 blockade in vivo. Moreover, we observed a reduction in Spp1 hi-TAMs following A2AR blockade, consistent with the high expression of Adora2a transcripts (encoding A2AR) and activation of downstream pathways, suggesting a feed-forward loop of adenosine pathways contribute to ICI resistance mediated by SPP1 hi-TAMs. Clinical correlation showed that a subset of mCRPC patients treated with combined adenosine receptor and PD-L1 blockade had improved anti-tumor efficacy compared to those on monotherapy, with SPP1 hi-TAMs potentially serving as predictive biomarkers for treatment response. Conclusions: Our studies reveal a significant increase in SPP1 hi-TAMs abundance with disease progression, contributing to ICI resistance through adenosine pathways. These findings support the therapeutic targeting of SPP1 hi-TAMs and their associated pathways as promising strategies to overcome resistance.
 Citation Format: Aram Lyu, Zenghua Fan, Matthew Clark, Averey Lea, Diamond Luong, Ali Setayesh, Alec Starzinski, Rachel Wolters, Marcel Arias-Badia, Kate Allaire, Kai Wu, Vibha Gurunathan, Laura Valderrábano, Xiao Wei, Richard Miller, Eliezer Van Allen, Lawrence Fong. A feed-forward loop in the adenosine signaling pathway drives myeloid-mediated resistance to immune checkpoint inhibition in prostate cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d917567d521f771de6596d03f0f8c85e1a9bdcbb" target='_blank'>
              Abstract PR003: A feed-forward loop in the adenosine signaling pathway drives myeloid-mediated resistance to immune checkpoint inhibition in prostate cancer
              </a>
            </td>
          <td>
            Aram Lyu, Zenghua Fan, Matt Clark, Averey Lea, Diamond Luong, Ali Setayesh, Alec Starzinski, Rachel Wolters, Marcel Arias-Badia, Kathryn M Allaire, Kai Wu, Vibha Gurunathan, Laura Valderrábano, Xiao X Wei, Richard A. Miller, E. Van Allen, Lawrence Fong
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067a6a45cdd25069d169d43fc6989270e4c626cb" target='_blank'>
              SMARCB1-driven EGFR-GLI1 epigenetic alterations in lung cancer progression and therapy are differentially modulated by MEOX2 and GLI-1.
              </a>
            </td>
          <td>
            Octavio A Trejo-Villegas, Priscila Pineda-Villegas, Leonel Armas-López, C. Mendoza-Milla, I. Peralta-Arrieta, Oscar Arrieta, I. Heijink, Joaquín Zúñiga, F. Ávila-Moreno
          </td>
          <td>2025-02-19</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c10b0a74142199e8c7ea18f31540a06dfe3ffe" target='_blank'>
              Aneuploidy-driven gene expression profiling in human blastocysts: insights from RNA-Seq analysis.
              </a>
            </td>
          <td>
            M. Bazrgar, F. Khajehoseini, P. Eftekhari-Yazdi, M. Bakhtiarizadeh, H. Gourabi, Parishad Saei, R. Pazhoomand, S. Hosseinishenatal, R. Mohammadi
          </td>
          <td>2025-01-03</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa0f09862c96ff9cec51a61499fdcc67366281e" target='_blank'>
              Analysis of microarray and single-cell RNA-seq identifies gene co-expression, cell–cell communication, and tumor environment associated with metabolite interconversion enzyme in prostate cancer
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Ali Shakeri Abroudi, Nadia Forghani, Sobhan Bavandi, Melika Djamali, Hamoon Baghaei, Sana Shafaeitilaki, Ehsan HasanZadeh
          </td>
          <td>2025-02-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, Gregor Warsow, Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8153c4934bac7993fe656260b2c6dc203c33aaba" target='_blank'>
              Identification of new reference genes with stable expression patterns for cell cycle experiments in human leukemia cell lines
              </a>
            </td>
          <td>
            Otília Tóth, Gergely Rácz, Eszter Oláh, Máté Tóth, E. Szabó, G. Várady, Beáta G. Vértessy, Nikolett T. Nagy
          </td>
          <td>2025-01-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8844ef9ba3f77a8686c18b6b08bd0f64a9f348" target='_blank'>
              Towards an interpretable deep learning model of cancer
              </a>
            </td>
          <td>
            Avlant Nilsson, N. Meimetis, Douglas A Lauffenburger
          </td>
          <td>2025-02-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The adaptation of malignancy to therapy presents a significant challenge in cancer treatment. The cell cycle plays a crucial role in regulating the evolution of radio- and chemo-resistance in tumor cells. Cancer stem cells (CSCs) are the primary source of therapy resistance, with CD133 being one of the most recognized and valuable cell surface markers of CSCs. Evidence increasingly suggests that CD133 is associated with cancer resistance. The current understanding of the molecular biological function of CD133 is limited, leading to ongoing debates about its role in cancer biology. In this review, we explore recent research and emerging trends related to CD133 through extensive literature and content analysis. It was summarized that new insights into the relationships of CD133 and cell cycle signaling pathways in resistant CSCs. The aim of this review is to provide a foundational understanding of how these signaling pathways and their interactions impact cancer prognosis and inform treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a397e129ea20400495da1c11c4058f7fa2fed81" target='_blank'>
              Unveiling the impact of CD133 on cell cycle regulation in radio- and chemo-resistance of cancer stem cells
              </a>
            </td>
          <td>
            Luyao Wu, T. Katsube, Xiaofei Li, Bing Wang, Yi Xie
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Public Health</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-coding RNA (ncRNA) has been recognized to be an essential regulator of cellular processes and gene expression in cancer. The present study covers the various roles of ncRNAs, including circular RNAs (circRNAs), long non-coding RNAs (lncRNAs), and microRNAs (miR-NAs), that affect cancer properties. Oncogenesis, metastasis, and treatment resistance are all pro-cesses involving ncRNAs, which have tremendous potential as new therapeutic agents and tar-gets. The review covers the broad spectrum of ncRNAs in cancer biology, including their types and activities, epigenetic control, function in metastasis and angiogenesis, detection and profiling ap-proaches, potential as biomarkers, and therapeutic possibilities. Recent advancements in next-gener-ation sequencing and other molecular methods have helped us better understand how ncRNAs work and their potential therapeutic uses. However, there are still challenges to standardizing detection technologies and producing effective RNA-based therapeutics. Therefore, further studies are needed to solve important issues in this sector. Standardization efforts are also essential to developing iden-tical methods for ncRNA collection, quantification, and analysis throughout multiple laboratories and ensuring the findings are reliable and comparable. Large-scale, multi-recentre studies are required to verify the diagnostic usefulness of ncRNA biomarkers across a wide range of patient groups. Also, more detailed mechanistic knowledge is necessary for understanding the particular molecular mech-anisms by which ncRNAs affect cancer growth, metastasis, and treatment response. This review high-lights the complex relationships between ncRNAs and cancer biology and also focuses on their po-tential effect on cancer diagnosis and treatment. It also highlights the necessity for more studies to fully understand the therapeutic potential of ncRNAs in cancer. As studies advance, using ncRNA results in clinical practice might change cancer treatment by novel opportunities for specific therapy and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d029995f049d5872068ca5f1e2ab3843bccfc74" target='_blank'>
              An Overview of Unlocking the Power of Non-Coding RNAs in Revolutionizing Cancer Therapy.
              </a>
            </td>
          <td>
            Sagarika Kabra, Neetu Sharma, Ankita Kumari, Saba Khan, Rashmi Pathak, Himanshu Sharma, Abdullah Al Noman
          </td>
          <td>2025-02-26</td>
          <td>Current drug discovery technologies</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diabetes mellitus (DM) is a global health crisis affecting millions, with islet transplantation emerging as a promising treatment strategy to restore insulin production. This review synthesizes the current research on single-cell and spatial transcriptomics in the context of islet transplantation, highlighting their potential to revolutionize DM management. Single-cell RNA sequencing, offers a detailed look into the diversity and functionality within islet grafts, identifying specific cell types and states that influence graft acceptance and function. Spatial transcriptomics complements this by mapping gene expression within the tissue’s spatial context, crucial for understanding the microenvironment surrounding transplanted islets and their interactions with host tissues. The integration of these technologies offers a comprehensive view of cellular interactions and microenvironments, elucidating mechanisms underlying islet function, survival, and rejection. This understanding is instrumental in developing targeted therapies to enhance graft performance and patient outcomes. The review emphasizes the significance of these research avenues in informing clinical practices and improving outcomes for patients with DM through more effective islet transplantation strategies. Future research directions include the application of these technologies in personalized medicine, developmental biology, and regenerative medicine, with the potential to predict disease progression and treatment responses. Addressing ethical and technical challenges will be crucial for the successful implementation of these integrated approaches in research and clinical practice, ultimately enhancing our ability to manage DM and improve patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d2a871c9156c1f2749ccafc8536877666a5a4e" target='_blank'>
              Single-cell genomics and spatial transcriptomics in islet transplantation for diabetes treatment: advancing towards personalized therapies
              </a>
            </td>
          <td>
            Lisha Mou, Tony Bowei Wang, Yuxian Chen, Ziqi Luo, Xinyu Wang, Zuhui Pu
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) accumulate somatic mutations over time, some conferring a fitness advantage that can lead to clonal hematopoiesis (CH). Mutations in DNMT3A, particularly at hotspot R882, are the most prevalent in CH and carry an increased risk of acute myeloid leukemia (AML). Although DNMT3A R882 mutations are linked to global DNA hypomethylation, the mechanisms underlying their selective advantage remain unclear. Here, we show that Dnmt3a-R882H mutant HSCs exhibit resilience under inflammatory and genotoxic stress. During IL-1β-induced emergency granulopoiesis, Dnmt3a R882H/+ HSCs uncouple increased proliferation from stem cell exhaustion. In contrast, wild-type HSCs rapidly progress to terminal differentiation. We link this phenotype to a delayed activation of the p53-p21-DREAM axis, that allows mutant HSCs to avoid attrition, despite increased replication stress. Similarly, mutant HSCs exhibit delayed Trp53 activation following irradiation, but eventually recover a physiological Trp53 response. Analysis of patient data reveals shared phenotypic features between DNMT3A and monoallelic TP53 mutations in CH and myeloid neoplasms, highlighting potential functional similarities. Collectively, these findings suggest that the expansion of DNMT3A-mutant clones is affected by impaired TP53 signaling, which confers resilience against stressors. Therapeutic strategies targeting inflammatory pathways or the p53-p21-DREAM axis may reduce DNMT3A-CH expansion and/or progression and its associated risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d28c424d36dd039343d97e3e9dacf850657da3e" target='_blank'>
              Crosstalk between non-coding RNAs and programmed cell death in colorectal cancer: implications for targeted therapy
              </a>
            </td>
          <td>
            S. R. Taha, Mehdi Karimi, Bahar Mahdavi, Milad Yousefi Tehrani, Ali Bemani, Shahriar Kabirian, Javad Mohammadi, Sina Jabbari, Meysam Hushmand, Alireza Mokhtar, M. H. Pourhanifeh
          </td>
          <td>2025-01-15</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ca816cc15851962002a66601f774cc9384fd1d" target='_blank'>
              Comprehensive genome-scale CRISPR knockout screening of CHO cells
              </a>
            </td>
          <td>
            Sungwook Shin, Su Hyun Kim, Aghiles Gasselin, Gyun Min Lee, Jae Seong Lee
          </td>
          <td>2025-01-15</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b690f44fb31641c4b057abb18dc7da4f6e88c306" target='_blank'>
              The widely used Ucp1-Cre transgene elicits complex developmental and metabolic phenotypes
              </a>
            </td>
          <td>
            M. Halurkar, Oto Inoue, Archana Singh, Rajib Mukherjee, Meghana Ginugu, Christopher Ahn, C. L. Bonatto Paese, Molly Duszynski, Samantha A Brugmann, Hee-Woong Lim, Joan Sanchez-Gurmaches
          </td>
          <td>2025-01-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bae82e2a8f4a414d538a8210faf20fea3eff9d" target='_blank'>
              Molecular and pharmacological heterogeneity of ETV6::RUNX1 acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenhua Li, Huanbin Zhao, Wenjian Yang, Maud Maillard, S. Yoshimura, Yu-Chih Hsiao, Xingyu Huang, Yoshihiro Gocho, L. Rowland, Anthony Brown, Landon Choi, K R Crews, C. Mullighan, Samuel Brady, Cheng Cheng, Ti-Cheng Chang, Gang Wu, M. Loh, A. E. J. Yeoh, Federico Antillon-Klussmann, S. Jeha, H. Inaba, Jiyang Yu, Ching-Hon Pui, Seth E. Karol, William E Evans, Jun J Yang
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Evidence has been mounting over the last decade demonstrating the crucial functions of ncRNAs, including miRNAs and long non-coding RNAs (lncRNAs), in cellular signalling networks and gene regulation. These ncRNAs are highly expressed and play a crucial role in regulating processes including metastasis and endothelial cell invasiveness. They facilitate crosstalk between interconnected signaling pathways, including PTEN/PI3K/AKT/mTOR, by acting as linking molecules within signal circuits. In addition to gene expression and conventional apical measures, studying miRNAs provides a better understanding of both beneficial and detrimental cellular reactions to external exposures. Additionally, miRNAs—which may be found in extracellular and intracellular environments, such as biofluids—have become potential early and precise molecular indicators of tissue and cellular conditions. This review highlights the regulatory functions of ncRNAs, their involvement in cellular homeostasis, and their potential as biomarkers, shedding light on their significance in gene regulation, toxicological studies, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b11830d2e556b399c1b5c04bd83303250065b32" target='_blank'>
              The Role of Non-Coding RNAs in Gene Regulation and Cellular Signaling Pathways
              </a>
            </td>
          <td>
            K. A. kumari
          </td>
          <td>2025-02-22</td>
          <td>International Journal of Innovations in Science Engineering And Management</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/487f335d997ac1c0a2d89897372deaa3e498deda" target='_blank'>
              Radiation therapy-induced normal tissue damage: involvement of EMT pathways and role of FLASH-RT in reducing toxicities.
              </a>
            </td>
          <td>
            Dhasarathdev Srinivasan, R. Subbarayan, M. Krishnan, Ranjith Balakrishna, Pooja Adtani, Rupendra Shrestha, Ankush Chauhan, S. Babu, Arunkumar Radhakrishnan
          </td>
          <td>2025-01-06</td>
          <td>Radiation and environmental biophysics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metabolic rewiring in cancer cells facilitates the provision of essential precursors for the unbridled growth of tumors. Exploring these cancer-specific metabolic changes offers potential selective therapeutic strategies. However, targeting a single essential gene in cancer treatment often faces challenges, including resistance, lack of targetable oncogenes, and potential harm to non-tumor cells. Attacking multiple targets is hypothesized as a solution to overcome these issues, e.g., a synthetic lethal set, defined as a minimal combination of non-lethal genetic mutations leading to cell death. This study examined the potential of synthetic lethal sets to identify selective drug targets for 13 cancers and the corresponding non-tumor tissues, utilizing context-specific genome-scale metabolic models. To ensure the minimization of therapeutic side effects, this work introduced the concept of strictly-selective drug targets (SSDTs) and the harmlessness of identified targets in all 13 different non-tumor tissues was meticulously verified. Accordingly, for 13 types of cancers, over 500 SSDTs were identified, predominantly including higher-order synthetic lethal sets with more than two targets in each set. Interestingly, for specific cancers where single essential or synthetic lethal genes could not provide acceptable solutions, SSDTs were provided by higher-order synthetic lethal sets. Therefore, for the first time, this study successfully showed that leveraging higher-order synthetic lethal sets holds the key to promising strictly-selective solutions. Furthermore, nine quadruple SSDTs were identified which commonly target five different cancers without harming any of the 13 non-tumor tissues. Further experimental validation of these findings is required to select the most promising treatments for clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699db54aa68bb5b5b6c0164aa363c1a5c2a8cf05" target='_blank'>
              Higher order synthetic lethals are keys to minimize cancer treatment effects on non-tumor cells
              </a>
            </td>
          <td>
            Mehdi Dehghan Manshadi, P. Setoodeh, Amin Ramezani, A. Rajabzadeh, Habil Zare
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Significance Understanding the multifaceted genetic disorder ataxia–telangiectasia (A-T) offers new insights into the functions of the ATM protein kinase, the loss of which underlies this disease. A-T exemplifies the connection between genome instability, cellular senescence, and aging. This study elucidates the pathways leading to a hallmark of A-T’s cellular phenotype: the premature senescence of primary fibroblasts derived from affected individuals. Our findings provide a mechanistic understanding of how inherited genome instability accelerates cellular senescence, offering potential explanations for several A-T symptoms. The work introduces murine primary lung fibroblasts growing under 3% oxygen as a valuable experimental system to explore the genome instability-senescence link and the molecular events leading to the neoplastic transformation of primary diploid cells without any directed manipulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d92deb9c47fa1db095e272225b8766e094ed2771" target='_blank'>
              The cGAS–STING, p38 MAPK, and p53 pathways link genome instability to accelerated cellular senescence in ATM-deficient murine lung fibroblasts
              </a>
            </td>
          <td>
            M. Haj, Yann Frey, Amit Levon, Avishai Maliah, Tal Ben-Yishay, Rachel Slutsky, Riham Smoom, Y. Tzfati, Uri Ben-David, Carmit Levy, Ran Elkon, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f1d4dd8d3660b6ba4f0f1ce80dbad74f8c1f24" target='_blank'>
              Long non-coding RNA MSC-AS1 confers imatinib resistance of gastrointestinal stromal tumor cells by activating FNDC1 and ANLN-mediated PI3K/AKT pathway.
              </a>
            </td>
          <td>
            Lin Chen, Yongjian Gao, Huaixi Yang, Yanzhuo Su, Yunxin Zhang, Lin Lou, Xuefeng Wang, Dayong Ding
          </td>
          <td>2025-01-03</td>
          <td>Human cell</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6693e1cc9fcaaa6e44ac852bdcdb47fc66775d" target='_blank'>
              Double-strand break repair pathways differentially affect processing and transduction by dual AAV vectors
              </a>
            </td>
          <td>
            Anna C Maurer, Brian Benyamini, O. Whitney, Vinson B. Fan, Claudia Cattoglio, Djem U. Kissiov, G. Dailey, X. Darzacq, Matthew D Weitzman, Robert Tjian
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Evidence suggests that increases in ploidy have occurred frequently in the evolutionary history of organisms and can serve adaptive functions to specialized somatic cells in multicellular organisms. However, the sudden multiplication of all chromosome content may present physiological challenges to the cells in which it occurs. Experimental studies have associated increases in ploidy with reduced cell survival and proliferation. To understand the physiological challenges that suddenly increased chromosome content imposes on cells, we used S. cerevisiae to ask how much chromosomal DNA cells may contain and what determines this limit. We generated polyploid cells using 2 distinct methods causing cells to undergo endoreplication and identified the maximum ploidy of these cells, 32–64C. We found that physical determinants that alleviate or exacerbate cell surface stress increase and decrease the limit to ploidy, respectively. We also used these cells to investigate gene expression changes associated with increased ploidy and identified the repression of genes involved in ergosterol biosynthesis. We propose that ploidy is inherently limited by the impacts of growth in size, which accompany whole-genome duplication, to cell surface integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c9419ff84380a674e570b7e466126c557aa8fb" target='_blank'>
              Cell integrity limits ploidy in budding yeast
              </a>
            </td>
          <td>
            Juliet Barker, Andrew W. Murray, Stephen P Bell
          </td>
          <td>2025-01-13</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f63f6a23a3983ad60b3216e3e5876ec300a9938" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9342acbba0e99dc63b9eedf38a334853e0c4e4e7" target='_blank'>
              The role of mitophagy-related genes in prognosis and immunotherapy of cutaneous melanoma: a comprehensive analysis based on single-cell RNA sequencing and machine learning.
              </a>
            </td>
          <td>
            Jun Tian, Lei Zhang, Kexin Shi, Li Yang
          </td>
          <td>2025-01-11</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Dosage-sensitive transcription factors (TFs) underlie altered gene regulation in human developmental disorders, and cell-type specific gene regulation is linked to the reorganization of 3D chromatin during cellular differentiation. Here, we show dose-dependent regulation of chromatin organization by the congenital heart disease (CHD)- linked, lineage-restricted TF TBX5 in human cardiomyocyte differentiation. Genome organization, including compartments, topologically associated domains, and chromatin loops, are sensitive to reduced TBX5 dosage in a human model of CHD, with variations in response across individual cells. Regions normally bound by TBX5 are especially sensitive, while co-occupancy with CTCF partially protects TBX5-bound TAD boundaries and loop anchors. These results highlight the importance of lineage-restricted TF dosage in cell-type specific 3D chromatin dynamics, suggesting a new mechanism for TF-dependent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070213389bd3dded91b65e66eeb390914aae3f41" target='_blank'>
              Dose-dependent sensitivity of human 3D chromatin to a heart disease-linked transcription factor
              </a>
            </td>
          <td>
            Zoe L. Grant, Shuzhen Kuang, Shu Zhang, Abraham J. Horrillo, Kavitha S. Rao, Vasumathi Kameswaran, Carine Joubran, Pik Ki Lau, Keyi Dong, Bing Yang, Weronika M. Bartosik, Nathan R. Zemke, Bing Ren, I. S. Kathiriya, K. S. Pollard, B. G. Bruneau
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the deadliest brain cancer in adults, and all patients succumb to the tumor. While surgery followed by chemoradiotherapy delays disease progression, these treatments do not lead to tumor control, and targeted therapies or biologics have failed to further improve survival. Utilizing a transient radiation-induced state of multipotency, we used the adenylcyclase activator forskolin to alter the fate of irradiated glioma cells. The effects of the combined treatment on neuronal marker expression, cell cycle distribution, and proliferation were studied. Gene expression profiling was conducted using bulk RNA-seq. Changes in cell populations were investigated using single-cell RNA-seq. Effects on glioma stem cells (GSCs) were studied in extreme limiting dilution assays, and the effects on median survival were studied in both syngeneic and PDOX mouse models of GBM. The combined treatment induced the expression of neuronal markers in glioma cells, reduced proliferation, and led to a distinct gene expression profile. scRNA-seq revealed that the combined treatment forced glioma cells into a microglia- and neuron-like phenotype. In vivo, this treatment led to a loss of GSCs and prolonged median survival. Collectively, our data suggest that revisiting a differentiation therapy with forskolin in combination with radiation could lead to clinical benefit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee9b47f25b8f95cc9d3c995fd15ca77ac2b58d9" target='_blank'>
              Radiation-induced cellular plasticity primes glioblastoma for forskolin-mediated differentiation.
              </a>
            </td>
          <td>
            Ling He, Daria J. Azizad, Kruttika Bhat, A. Ioannidis, Carter J. Hoffmann, Evelyn Arambula, M. Eghbali, A. Bhaduri, H. Kornblum, Frank Pajonk
          </td>
          <td>2025-02-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood.In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU).Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome.These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, Simon Zhongyuan Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c56d0e33616064b81832853efdda25a0062176f" target='_blank'>
              Exploring Epigenetic Complexity in Regulation of Hematopoietic Stem Cells Niche: A Mechanistic Journey from Normal to Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            N. A. Yusoff, Zariyantey Abd Hamid, Izatus Shima Taib, Siti Razila Abdul Razak, S. B. Budin
          </td>
          <td>2025-01-23</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Several studies have attempted to find the initiating drivers of small intestinal neuroendocrine tumors (SI-NET) development and the molecular mechanisms driving progression and metastatic spread. For gene expression studies using bulk microarrays and RNA sequencing, researchers commonly use normal intestinal mucosa as a control. The intestine is made up of several different cell types, and using bulk RNA-seq may generate findings that reflect factors other than the tumor transformation. This could potentially contribute to the lack of discoverable treatments and prevention strategies for SI-NETs. We performed scRNA-seq on tissue from two patients that had tumor resection surgery and separated the EC cells from the normal intestinal mucosa and used specific markers to compare them against SI-NET tumors cells. We pinpointed new genes associated with chr18 haploinsufficiency but also others with loss or gain of expression that have not previously been associated with SI-NETs, and which could be potential targets for further functional genomic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly prevalent and deadly type of cancer characterized by intricate molecular pathways that drive tumor development, metastasis, and resistance to conventional treatments. Recently, ferroptosis, a controlled mechanism of cell death instigated by iron-dependent lipid peroxidation, has gained attention for its role in LC progression and treatment. Noncoding RNAs (ncRNAs), such as microRNAs (miRNAs) and long noncoding RNAs (lncRNAs), are emerging as key modulators of ferroptosis, significantly influencing LC biology. This review explores how ncRNAs control ferroptotic pathways and affect tumor growth, metastasis, and therapy resistance in LC. By understanding the dual functions of ncRNAs in both activating and inhibiting ferroptosis, we aim to uncover new therapeutic targets and strategies for LC. These insights provide a promising direction for the development of ncRNA-based treatments designed to induce ferroptosis, potentially improving therapeutic outcomes for patients with LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2604d726c6b3c3745d518fb5d2c37fc2627c33b3" target='_blank'>
              Ferroptosis and noncoding RNAs: exploring mechanisms in lung cancer treatment
              </a>
            </td>
          <td>
            Nadi Rostami Ravari, Farzad Sadri, Mohammad Ali Mahdiabadi, Yaser Mohammadi, Zahra Ourang, Z. Rezaei
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Single‐cell sequencing technologies have revolutionised pharmaceutical research by providing in‐depth insights into human biology at the single‐cell level. These tools enable researchers to identify rare cell types and analyse cellular diversity within tissues, facilitating the discovery of new therapeutic targets and biomarkers. However, their application in nutraceutical research is still in its early stages.Unlike pharmaceuticals, which have well‐defined chemical structures and mechanisms, nutraceuticals are food‐based materials intended for specific medical purposes and often contain complex and diverse food chemicals. These molecules can work synergistically, producing multi‐targeted effects in various tissues. Traditional bulk profiling methods for tissues and tumours do not adequately capture cellular heterogeneity or specific cellular responses to treatments. Therefore, advanced single‐cell sequencing is crucial for dissecting tissues into distinct cell types, helping to clarify the underlying mechanisms at the cellular level. Many derivatives of functional foods have been marketed or assessed, demonstrating health benefits. However, mechanistic insights are lacking, with most current data derived from observational studies or traditional in vitro and in vivo models. Human clinical trials are needed to validate these nutraceutical effects and determine effective and safe dosages. Edible mushrooms have gained attention as nutraceuticals due to their medicinal properties. They have been observed to enhance immunity, reduce inflammation, and combat cancer. These effects have been attributed to their unique bioactive components and historical uses in traditional medicine. Epidemiological studies show that higher consumption of edible mushrooms is linked to a reduced risk of certain cancers.In this review, we share important lessons learned in the design and execution of clinical trials focusing on white button mushrooms as anti‐cancer nutraceuticals. We demonstrate the use of single‐cell RNA sequencing (scRNA‐seq) in nutraceutical research to capture the nuanced biological responses and health effects of dietary foods and their constituents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121f5e0cc5ea30bbc9da6b602e08a3acc9219772" target='_blank'>
              Utilising single‐cell sequencing in clinical nutraceutical research: Recent progress and perspectives
              </a>
            </td>
          <td>
            Xiaoqiang Wang, Yin Chan, David Sadava, Shiuan Chen
          </td>
          <td>2025-01-23</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA methylation plays a critical role in gene regulation, affecting cellular differentiation and disease progression, particularly in non-coding regions. However, predicting the epigenetic consequences of non-coding mutations at single-cell resolution remains a challenge. Existing tools have limited prediction capacity and struggle to capture dynamic, cell-type-specific regulatory changes that are crucial for understanding disease mechanisms. Here, Methven, a deep learning framework designed is presented to predict the effects of non-coding mutations on DNA methylation at single-cell resolution. Methven integrates DNA sequence with single-cell ATAC-seq data and models SNP-CpG interactions over 100 kbp genomic distances. By using a divide-and-conquer approach, Methven accurately predicts both short- and long-range regulatory interactions and leverages the pre-trained DNA language model for enhanced precision in classification and regression tasks. Methven outperforms existing methods and demonstrates robust generalizability to monocyte datasets. Importantly, it identifies CpG sites associated with rheumatoid arthritis, revealing key pathways involved in immune regulation and disease progression. Methven's ability to detect progressive epigenetic changes provides crucial insights into gene regulation in complex diseases. These findings demonstrate Methven's potential as a powerful tool for basic research and clinical applications, advancing this understanding of non-coding mutations and their role in disease, while offering new opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd41b7e43e4befc2efcfbe9286141d3e14f4002" target='_blank'>
              Epigenetic Impacts of Non-Coding Mutations Deciphered Through Pre-Trained DNA Language Model at Single-Cell Resolution.
              </a>
            </td>
          <td>
            Zhe Liu, An Gu, Yihang Bao, Guan Ning Lin
          </td>
          <td>2025-01-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            M. M. Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, S. Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The benefit of adoptive cell transfer (ACT) therapies with T cells engineered to express chimeric antigen receptors (CAR) or T cell receptors (TCR) is limited by T cell dysfunction upon repetitive antigen exposure. We integrated a pooled genome-wide CRISPR knockout screen with in vitro repetitive antigen exposure and phenotypic selection to identify gene knockouts that improved the ability of NY-ESO-1 TCR transgenic T cells to repeatedly kill the antigen- and HLA-matched melanoma cell line M407. In two replicate screens, knocking out of the genes CUL5, RNF7 or ARIH2, which are required for proper cullin 5-RING ligase complex (CRL5) assembly, significantly improved long-term tumor cell killing ability of T cells past five rechallenges (one-way ANOVA p<0.0001). Mechanistically, we discovered that CRL5 interacts with SOCS-box containing proteins to regulate TCR stimulation and cytokine signaling through ubiquitination and proteasomal degradation. Ablation of CRL5 in T cells leads to an increase in polyfunctionality with augmented production and release of cytokines and cytolytic molecules across multiple rounds of killing when compared to control T cells (IFNg/TNFa producing T cells; Challenge 1: 18% versus 10%, Stimulation 3: 6% versus 3%, unpaired t-test p<0.0001). We demonstrated through both CRISPR knockout, and pharmacological inhibition of CRL5 using the neddylation inhibitor MLN4924, the negative regulatory mechanism of CRL5 in CD8+ T cells. These results highlight the previously unrecognized regulation of T cell function through E3 ligase-mediated proteasomal degradation, and that CRL5 may be a target to endow lasting effector function to antigen-specific T cells for cancer immunotherapies.
 Citation Format: Justin Saco, Daniel Karin, Nataly Naser Al Deen, Katie Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas. Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="HIV-1 persists as a global health challenge due to its ability to establish latent reservoirs that evade eradication by antiretroviral therapy (ART). This study introduces a high-diversity barcoded HIV-1 library combined with a novel peptide barcode system to enable single-cell multi-dimensional analysis of viral integration, transcription, splicing, and translation of distinct viral mRNA isoform. Our method revealed over 80 distinct HIV-1 splicing isoforms and uncovered significant temporal variation in mRNA diversity, driven by integration site and chromatin context. By linking HIV-1 5’UTR with peptide barcoding, we quantified the translation efficiency of alternative 5′ UTRs and demonstrated correlations with abundance of each viral mRNA isoform. Additionally, we showed that latency reversal agents, such as SAHA and Bryostatin, selectively reactivate proviruses depending on their genomic and chromatin context and leads to distinct viral RNA splicing program. This platform revealed the diversity of HIV-infected cells at levels of transcripts as well as at proteins, and subsequently the fate determination. The detailed understanding of the stochastic diversity and dynamic nature of HIV-1 latency, inform the development of targeted therapies to eliminate latent reservoirs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b93e98c68b3422ec0ed93ef0f0fe8270c85d96" target='_blank'>
              Extreme high-diversity barcoded HIV library enables single cell multi-omics analysis of viral expression and fate determination
              </a>
            </td>
          <td>
            Tian-hao Zhang, Li Sheng, Xiangzhi Meng, Wenjin Lai, Yushen Du, Yuan Shi, Jing Wen, Masakazu Kamata, S. Chanda, Irvin S. Y. Chen, Ren Sun
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87eccc46e0a8b0d1964c263841b403eaf8af0f71" target='_blank'>
              CXCR4 promotes tumor stemness maintenance and CDK4/6 inhibitors resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Qianfeng Shi, Wang Yang, Yiye Ouyang, Yujie Liu, Zijie Cai
          </td>
          <td>2025-01-28</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d734aed102ed089c1bedbb1c15be484c90edb1" target='_blank'>
              tRNA-derived small RNAs: their role in the mechanisms, biomarkers, and therapeutic strategies of colorectal cancer
              </a>
            </td>
          <td>
            Bo Zhang, Yanru Pan, Zhe Li, Kefeng Hu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Second-generation anti-CD19 chimeric antigen receptor (CAR) T-cells incorporating CD28 (1928z) are now standard of care for treatment of refractory/relapsed large B-cell and mantle cell lymphomas. However, clinical use is limited by high toxicity rates, especially cytokine release syndrome (CRS) and immune effector cell-associated neurotoxicity syndrome (ICANS). Modifying intracellular signalling domains can influence CAR T-cell toxicity risk. Although rare, reports of CAR T-cell-derived malignancies underscore the risks of gene-modified cell therapies and the need to assess transgene insertion sites in new products. Toll-like receptor 2 (TLR2) a pattern recognition receptor, is highly expressed by activated CD8+ human T-cells, and can serve as a T-cell costimulatory signal, enhancing, memory formation, and modifying T-cell cytokine profile. In support of a phase 1 clinical trial (ENABLE-1, NCT04049513) evaluating a new third-generation anti-CD19 CAR T-cell product incorporating CD28 and TLR2 co-stimulatory domains (1928T2z), we explored the functional impact of the TLR2 domain in vitro, and characterised gene insertion sites within the 1928T2z product. Incorporating TLR2-derived Toll/IL-1 receptor resistance domain (TIR) between CD28 and CD3ζ domains in the 1928T2z CAR provided comparable CAR T-cell efficacy while producing lower levels of IFN-γ and GM-CSF, cytokines linked to CRS and ICANS pathogenesis, than 1928z. In contrast, positioning the TLR2 domain at the cytoplasmic tail (1928zT2) did not lower IFN-γ and GM-CSF production. Inhibiting TLR2 TIR function using ortho-vanillin, or through targeted mutations abrogating MYD88 recruitment and/or dimerization, impaired CAR T-cell activity, indicating the TLR2 domain contributes to 1928T2z CAR function. Gene expression identified downregulation of IL-6 signalling and upregulation of an anti-inflammatory cytokine pathway in 1928T2z cells compared to comparator CAR T-cells. A phosphoprotein array revealed that a functional TLR2 domain was associated with in higher levels of phosphorylated MEK1/2 (involved in TLR signalling), GATA3 (Th2-linked transcription factor) and CREB (a transcription factor that limits inflammatory responses). Using long-read sequencing and bioinformatics, we characterised insertion sites in a clonal line and polyclonal human CAR T-cells. Results showed mostly intronic or intergenic sites, with less than 10% mapping to exons, aligning with known lentiviral transgene integration profiles. In summary, incorporating a TLR2 co-stimulatory domain into CD28-costimulated anti-CD19 CAR T-cells modulates cytokine profile in a position-dependent manner, with TLR2 placement between CD28 and CD3ζ reducing CRS-and ICANS-linked cytokines while preserving in vitro anti-tumour activity. These findings align with our phase 1 trial data suggesting that 1928T2z CAR T-cells are associated with low incidence of CRS and ICANS, while exhibiting similar clinical efficacy to commercial comparators.
 Citation Format: Nathaniel Dasyam, Yasmin Nouri, Joshua Halpin, Alexander McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, Markus Templin, Qin Le, Peng Li, Ian Hermans, Rachel Perret, Robert Weinkove. Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739d0a74d9a81f832abc4486254c5ebe8688f593" target='_blank'>
              Abstract A018: Functional, transcriptomic and genomic characterisation of CD19-directed CAR T-cells incorporating a TLR2 co-stimulatory domain
              </a>
            </td>
          <td>
            N. Dasyam, Yasmin Nouri, J. Halpin, A. McLellan, David Eccles, Simon Fink, Felix Schaefer-ruoff, M. Templin, Qin Le, Peng Li, I. Hermans, Rachel Perret, R. Weinkove
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Totipotent cells (TSCs), the origin of mammalian life and foundation to early mammalian embryogenesis, are characterized by the highest differentiation capacity and extensive developmental potential. However, none of the existing embryonic databases have provided epigenetic and transcriptomic resources on totipotency, greatly limiting our understanding of the mechanisms governing the establishment and exit of totipotency. Here, we present Toti, a pioneering multi-omics database exclusively developed for totipotency, covering in vivo, in vitro and genome-edited human and mouse embryonic TSCs, TSC-like cells, pluripotent cells (PSCs), and embryos spanning preimplantation stages, with a total of 8,265 samples. Toti facilitates an in-depth exploration of the molecular mechanisms underlying totipotency by offering Search, Browse and Analysis modules available at http://toti.zju.edu.cn/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0143151fabb07316572f667e24a48e297b397f77" target='_blank'>
              Toti: an integrated multi-omics database to decipher the epigenetic regulation of gene expression in totipotent stem cells
              </a>
            </td>
          <td>
            Yi Chai, Ruiying Zhang, Shunze Jia, Danfei Zhu, Siyi Chen, Xudong Fu, Xin Sheng
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63030f8f4a5318cca9879e32ba13ec7a2af3ed4" target='_blank'>
              Chromosome mis-segregation triggers cell cycle arrest through a mechanosensitive nuclear envelope checkpoint
              </a>
            </td>
          <td>
            Solène Hervé, Andrea Scelfo, Gabriele Bersano Marchisio, Marine Grison, Kotryna Vaidžiulytė, Marie Dumont, Annapaola Angrisani, Adib Keikhosravi, Gianluca Pegoraro, Mathieu Deygas, Guilherme P. F. Nader, Anne-Sophie Macé, Matteo Gentili, Alice Williart, N. Manel, Matthieu Piel, Y.A. Miroshnikova, D. Fachinetti
          </td>
          <td>2025-01-08</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well-studied neopolyploids (we use 'neopolyploid' to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole-genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research.
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, Ryan D Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Perturb-seq is a powerful approach to systematically assess how genes and enhancers impact the molecular and cellular pathways of development and disease. However, technical challenges have limited its application in stem cell-based systems. Here, we benchmarked Perturb-seq across multiple CRISPRi modalities, on diverse genomic targets, in multiple human pluripotent stem cells, during directed differentiation to multiple lineages, and across multiple sgRNA delivery systems. To ensure cost-effective production of large-scale Perturb-seq datasets as part of the Impact of Genomic Variants on Function (IGVF) consortium, our optimized protocol dynamically assesses experiment quality across the weeks-long procedure. Our analysis of 1,996,260 sequenced cells across benchmarking datasets reveals shared regulatory networks linking disease-associated enhancers and genes with downstream targets during cardiomyocyte differentiation. This study establishes open tools and resources for interrogating genome function during stem cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a019d0592b40aaea0bb2bdf58ea166a338fcf9" target='_blank'>
              Benchmarking and optimizing Perturb-seq in differentiating human pluripotent stem cells
              </a>
            </td>
          <td>
            Sushama Sivakumar, Yihan Wang, Sean C Goetsch, Vrushali Pandit, Lei Wang, Huan Zhao, Anjana Sundarrajan, Daniel A Armendariz, Chikara Takeuchi, Mpathi Z. Nzima, Wei-Chen Chen, Ashley E. Dederich, Lauretta El Hayek, Taosha Gao, Renad Ghazawi, Ashlesha Gogate, Kiran J. Kaur, Hyung Bum Kim, Melissa K. McCoy, H. Niederstrasser, Seiya Oura, Carolos A Pinzon-Arteaga, Menaka Sanghvi, Daniel A. Schmitz, Leqian Yu, Yanfeng Zhang, Qinbo Zhou, W. Kraus, Lin Xu, Jun Wu, Bruce A. Posner, Maria H. Chahrour, Gary C. Hon, N. Munshi
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Significance: Growing evidence indicates the importance of redox reactions homeostasis, mediated predominantly by reactive oxygen species (ROS) in influencing the development, differentiation, progression, metastasis, programmed cell death, tumor microenvironment, and therapeutic resistance of cancer. Therefore, reviewing the ROS-linked epigenetic changes in cancer is fundamental to understanding the progression and prevention of cancer. Recent Advances: We review in depth the molecular mechanisms involved in ROS-mediated epigenetic changes that lead to alteration of gene expression by altering DNA, modifying histones, and remodeling chromatin and noncoding RNA. Critical Issues: In cancerous cells, alterations of the gene-expression regulatory elements could be generated by the virtue of imbalance in tumor microenvironment. Various oxidizing agents and mitochondrial electron transport chain are the major pathways that generate ROS. ROS plays a key role in carcinogenesis by activating pro-inflammatory signaling pathways and DNA damage. This loss of ROS-mediated epigenetic regulation of the signaling pathways may promote tumorigenesis. We address all such aspects in this review. Future Directions: Developments in this growing field of epigenetics are expected to contribute to further our understanding of human health and diseases such as cancer and to test the clinical applications of redox-based therapy. Recent studies of the cancer-epigenetic landscape have revealed pervasive deregulation of the epigenetic factors in cancer. Thus, the study of interaction between ROS and epigenetic factors in cancer holds a great promise in the development of effective and targeted treatment modalities. Antioxid. Redox Signal. 00, 000-000.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90fb27d72b5f41dadb3feb89a750c00e2da4835a" target='_blank'>
              Epigenetic Control of Redox Pathways in Cancer Progression.
              </a>
            </td>
          <td>
            Vandit Shah, Hiu Yan Lam, Charlene Hoi-Mun Leong, Reo Sakaizawa, Jigna S Shah, Alan Prem Kumar
          </td>
          <td>2025-01-16</td>
          <td>Antioxidants & redox signaling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a956b3ede35f75fb0fcd21ec0971a9a4d9393907" target='_blank'>
              A cohort-based multi-omics identifies nuclear translocation of eIF5B /PD-L1/CD44 complex as the target to overcome Osimertinib resistance of ARID1A-deficient lung adenocarcinoma
              </a>
            </td>
          <td>
            Dantong Sun, H. Hou, Feiyue Feng, Weizheng Wu, Jingyu Tan, Tongji Xie, Jiayu Liu, Jin-song Wang, Haili Qian, Junling Li, P. Xing
          </td>
          <td>2025-01-07</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cutaneous melanoma, one of the most fatal and aggressive types of skin cancer, is a malignant tumor created from pigment-producing cells called melanocytes. The mor-bidity and incidence rate of melanoma are increasing around the world. Considering all developments in diagnostic and therapeutic approaches, early diagnosis and more effective targeted therapies are still urgent for melanoma. Non-coding RNAs (ncRNAs), including microRNAs and long non-coding RNAs, are key role players in fundamental cellular pro-cesses and have various biological properties. It has been demonstrated that the dysregula-tion of these biomarkers contributes to melanoma pathogenesis. Furthermore, programmed cell death (PCD), as an important cellular event, has been shown to play a role in the initi-ation and progression of cutaneous melanoma. Recent investigations have reported that the interplay between ncRNAs and PCD can be considered a novel choice for targeted therapy in melanoma. Thus, this review discusses PCD-related ncRNAs in melanoma, focusing on underlying pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c1c5de5601f2f61c348bb80645910cf0834e556" target='_blank'>
              Regulatory Effect of Non-Coding RNAs on Programmed Cell Death in Melanoma: Novel Therapeutic Crosstalk for Targeted Therapy.
              </a>
            </td>
          <td>
            Andarz Fazlollahpour-Naghibi, S. R. Taha, Mahdieh Shariatzadeh, Danyal Daneshdoust, Kimia Bagheri, T. Farkhondeh, M. Samini, M. H. Pourhanifeh, Saeed Samarghandian
          </td>
          <td>2025-02-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Promoter DNA methylation is a key epigenetic mark, commonly associated with gene silencing. However, we noticed that a positive association between promoter DNA methylation and expression is surprisingly common in cancer. Here, we use hit-and-run CRISPR/dCas9 epigenome editing to evaluate how deposition of DNA methylation can regulate gene expression dependent on pre-existing chromatin environment. While the predominant effect of DNA methylation in non-bivalent promoters is gene repression, we show that in bivalent promoters this often leads to gene activation. We demonstrate that gain of DNA methylation leads to reduced MTF2 binding and eviction of H3K27me3, a repressive mark that guards bivalent genes against activation. Our cancer patient data analyses reveal that in cancer, this mechanism likely leads to activation of a large group of transcription factors regulating pluripotency, apoptosis, and senescence signalling. In conclusion, our study uncovers an activating role of DNA methylation in bivalent promoters, with broad implications for cancer and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4036111a96308ac464169ce403c17e9659fbafc5" target='_blank'>
              De-novo DNA Methylation of Bivalent Promoters Induces Gene Activation through PRC2 Displacement
              </a>
            </td>
          <td>
            H. O’Geen, Alina Mihalovits, Benjamin D. Brophy, He Yang, Maxwell W. Miller, Chelsey J. Lee, David J. Segal, M. Tomková
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="A large fraction of the human genome is transcribed in RNA molecules that do not encode for proteins but that do have a crucial role in regulating almost every level of gene expression and, thus, define the specific phenotype of each cell. These non-coding RNAs include well-characterized microRNAs and thousands of less-defined longer transcripts, named long non-coding RNAs. Both types markedly affect the onset and the progression of numerous pathologies, ranging from cancer to vascular and neuro-degenerative diseases. In recent years, a substantial effort has been made to design drugs targeting ncRNAs, and promising advancements have been produced from micro-RNA mimics and inhibitors. Each ncRNA controls several targets, and the overall effect of its inhibition or overexpression depends on the function of the set of genes it regulates. Therefore, in selecting the most appropriate target, and predicting the final outcome of ncRNA-based therapies, it is crucial to have and utilize detailed and accurate knowledge of their functional interactions. In this review, I recapitulate the principal resources which collect information on microRNA and lncRNA networks, focusing on the non-homogeneity of the data that result from disparate approaches. I highlight the role of RNA identifiers and interaction evidence standardization in helping the user to filter and integrate data derived from different databases in a reliable functional web of regulative relations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45039e8fa4423e21f993b2688bba0179d8a2c57" target='_blank'>
              The Relevance of the Accurate Annotation of Micro and Long Non-Coding RNA Interactions for the Development of Therapies
              </a>
            </td>
          <td>
            Simona Panni
          </td>
          <td>2025-02-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer represents the most common type of cancer in females worldwide. The survival rates for breast cancer patients have been increasing since 1990. However, in 2023 breast cancer is still the second most common cause of malignancy-associated death in women. One decisive reason is the increase of treatment resistance and low therapy response. Therefore, new therapy targets and predictive markers for the response to treatment are needed. The present study analyzed the potential effects triggered by different breast cancer treatments on the transcriptional expression of 12 pre-selected long non-coding (lnc) RNAs and the proliferation markers Cyclin D1 and Ki-67 in six different breast cancer cell lines (BT-474, MDA-MB-231, BT-20, T-47D, SKBR-3 and MCF-7). The results revealed that lncRNA cytoskeleton regulator RNA may be an appropriate biomarker for the response to treatment with both epirubicin and gemcitabine (P<0.001). NF-ĸB interacting lnc RNA may be a marker for therapy response (P<0.001), while HOX transcript antisense RNA overexpression suggested resistance to treatment (P<0.001) with epirubicin. The transcriptional expression of lncRNA BC4 increased during treatment with epirubicin and gemcitabine, which indicated therapy response. Overall, the present data suggested that the aforementioned lncRNAs have a promising potential as biomarkers to detect early therapy response or resistance in and therefore should be analyzed in more detail.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030e7011040033b6c4ff54f51d9cc9c5158b86f4" target='_blank'>
              Long non‑coding RNA signatures in breast cancer: Properties as biomarkers?
              </a>
            </td>
          <td>
            J. Asberger, Isabell Ge, Benjamin Schmidt, M. Jäger, D. Weiss, K. Berner, T. Erbes, I. Juhasz-Böss, Sebastian Mayer
          </td>
          <td>2025-01-22</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Developmental toxicity testing is essential to identify substances that may harm embryonic development. This study aimed to establish a protocol for evaluating developmental toxicity using human induced pluripotent stem cells (iPSCs) by analyzing cellular activity and gene expression changes. Two ICH S5(R3) positive substances, valproic acid (VPA), which is a substance previously detected as positive by other test methods, and thalidomide (Thalido), were examined during early trichoderm differentiation without fetal bovine serum. RNA-seq analysis identified seven candidate genes, including TP63, associated with altered expression following exposure to VPA or Thalido. These genes were implicated in pathways related to tissue development, cell growth, and molecular interactions. While the assay effectively detected VPA and Thalido, its limitations include testing only soluble substances and focusing on early differentiation stages. Nevertheless, the protocol demonstrates potential for the classification and evaluation of emerging modality drugs based on physical properties such as solubility, polarity, and pH. Integration with AI analysis may enhance its capacity to uncover genetic variations and evaluate previously uncharacterized substances. This study provides a foundation for alternative developmental toxicity testing methods, with further refinements in the culture method expected to improve accuracy and applicability in regulatory toxicology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f645cbfc6d27afd26999597f37c144eb6b8f5ec" target='_blank'>
              Comprehensive Search for Genes Involved in Thalidomide Teratogenicity Using Early Differentiation Models of Human Induced Pluripotent Stem Cells: Potential Applications in Reproductive and Developmental Toxicity Testing
              </a>
            </td>
          <td>
            Yu Kato, Takeshi Inaba, Koudai Shinke, Noriko Hiramatsu, Tetsuhiro Horie, Takuya Sakamoto, Yuko Hata, Eiji Sugihara, Tetsuya Takimoto, Noriaki Nagai, Y. Ishigaki, Hajime Kojima, Osamu Nagano, Naoki Yamamoto, Hideyuki Saya
          </td>
          <td>2025-02-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="LMNA-associated congenital muscular dystrophy is a currently incurable rare genetic disorder characterized by early-onset muscle weakness, dilated cardiomyopathy and respiratory failure, resulting from mutations in the LMNA gene. In this study, we assessed the potential of a CRISPR-mediated strategy to eliminate the mutant allele Lmna c.745C>T, p.R249W using a mutation specific guide (sg745T). Results from R249W-mutation-carrying cellular models showed specific activity of the Cas9/sg745T complex towards the mutant allele. This property varied depending on the concentration of CRISPR components, with a loss of specificity observed with increased dosage. We tested this strategy in vivo using adeno-associated virus delivery in LmnaR249W mice. Despite being associated with a modest CRISPR activity, this therapeutic approach resulted in a 10% increase in the survival of R249W homozygous mice. Interestingly, a similar CRISPR activity improved the cardiac pathology developed by Lmna+/R249W animals, significantly extending their median survival. These results represent the first therapeutic validation of a CRISPR/Cas9-mediated gene editing strategy for the treatment of LMNA-associated congenital muscular dystrophy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eff96c5b33b1fd18b969ab18137992e335a5be4" target='_blank'>
              CRISPR/Cas9-mediated elimination of the LMNA c.745C>G pathogenic mutation enhances survival and cardiac function in LMNA-associated congenital muscular dystrophy
              </a>
            </td>
          <td>
            Déborah Gómez-Domínguez, Carolina Epifano, Iván Hernández, Borja Vilaplana-Martí, Sergi César, Antonio de Molina-Iracheta, Miguel Sena-Esteves, Georgia Sarquella-Brugada, Ignacio Pérez de Castro
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
 Colorectal cancer (CRC) is a prevalent malignant tumor characterized by a high mortality rate, with significant challenges persisting in the identification and management of its metastatic stage. The role of checkpoint kinase 1 (CHEK1), a cell cycle checkpoint kinase, in CRC has not been fully clarified. We hypothesize that the upregulation of CHEK1 may enhance the proliferation of CRC cells, indicating its potential as a novel therapeutic target for CRC therapy.
 AIM
 To investigate the expression and function of CHEK1 in CRC, this study utilizes single-cell RNA sequencing and tissue microarray data.
 METHODS
 Single-cell RNA sequencing technology was employed to analyze CRC cells from the GSE144735 dataset, and immunohistochemistry was conducted to confirm the expression of CHEK1 in CRC and adjacent tissues. We also integrated mRNA expression data from multiple public databases to assess global CHEK1 expression in CRC. Molecular docking experiments were performed to explore the interaction between CHEK1 and the potential drug nitidine chloride (NC), as well as to investigate the influence of CHEK1 on CRC cell proliferation.
 RESULTS
 We found comparatively elevated CHEK1 expression in the malignant epithelial cells of CRC, with marked upregulation of its mRNA levels in CRC tissues. Immunohistochemical analysis further confirmed the high expression of CHEK1 in CRC tissues, and the receiver operating characteristic curve demonstrated high accuracy (area under the curve = 0.964) for CHEK1 as a biomarker. Analysis of global datasets indicated a statistically significant overexpression of CHEK1 in CRC (standard mean difference = 1.81, P < 0.01), with summary receiver operating characteristic analysis yielding sensitivity and specificity values of 0.83 and 0.88, respectively. Molecular docking studies indicated that NC specifically targeted CHEK1, while clustered regularly interspaced short palindromic repeats knockout experiments demonstrated that CHEK1 promoted CRC cell proliferation.
 CONCLUSION
 Upregulation of CHEK1 promotes CRC cell proliferation. However, the dataset's diversity is limited, requiring further investigation into its specific mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a566889c9fc5fe520488439e6857e1a25ba38f4b" target='_blank'>
              Checkpoint kinase 1 in colorectal cancer: Upregulation of expression and promotion of cell proliferation
              </a>
            </td>
          <td>
            Yu-yan Pang, Zu-Yuan Chen, Da-Tong Zeng, Dong-Ming Li, Qi Li, Wan-Ying Huang, Bin Li, Jia-Yuan Luo, Bang-Teng Chi, Qiu Huang, Zhen-Bo Feng, Rong-Quan He
          </td>
          <td>2025-03-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="SKA3, an important factor in cell cycle regulation, is involved in spindle assembly and kinetochore function, playing a critical role in maintaining cancer cell proliferation and division. However, its specific roles and regulatory mechanisms in cancer remain not fully understood. Large-scale datasets from multiple public databases, including The Cancer Genome Atlas and Genotype-Tissue Expression, covering various cancer types, were integrated. Systematic analysis revealed that SKA3 exhibits aberrant expression patterns in multiple cancers and is significantly associated with tumor progression and poor patient prognosis in certain cancers. We explored the status of SKA3 gene mutation, gene amplification and promoter region methylation in various tumors. In the context of immunotherapy, we assessed the value of SKA3 in cancer. Analyzing the correlation between SKA3 expression levels and immune checkpoints and immune cell infiltration, we discovered that SKA3 could serve as a novel immunotherapy biomarker across multiple cancers, guiding clinical immunotherapy decisions. Finally, SKA3 knockdown inhibited lung adenocarcinoma cell proliferation and metastasis. In conclusion, this study provides new insights into the role of SKA3 in cancer and offers significant theoretical and experimental evidence for its development as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by the lack of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2). Chemotherapy remains the primary treatment option, yet TNBC frequently develops resistance, leading to relapse and metastasis. Emerging evidence highlights the potential of combining DNA methylation inhibitors with immune checkpoint inhibitors (ICIs). DNA methylation contributes to immune escape by silencing immune-regulatory genes, thereby reducing the tumor’s visibility to immune cells. Reversing this epigenetic modification can reinvigorate immune surveillance and enhance the efficacy of immunotherapies. This review discusses the role of DNA methylation in TNBC progression and immune evasion, focusing on recent advances in combination therapies involving DNA methylation inhibitors and ICIs. We discuss the underlying mechanisms that enable these therapeutic synergies, preclinical and clinical evidence supporting the approach, and the challenges posed by tumor heterogeneity, drug resistance, and toxicity. Finally, we explore the potential for personalized treatment strategies incorporating multi-omics data to optimize therapeutic outcomes. The integration of epigenetic therapies and immunotherapy offers a promising avenue for improving survival in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1a1657cbbdb3d35b60d2224060b7074a6c9d9c" target='_blank'>
              DNA methylation and immune evasion in triple-negative breast cancer: challenges and therapeutic opportunities
              </a>
            </td>
          <td>
            Wen-yu Cai, , Yi-ran Fei, Rui Ye, Ding-ming Song, Dan Hu, Wan-wan Zhang, Ming-fei Xia, Xiao-xiao Yang
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Massively parallel sequencing (MPS) of nucleic acids has been a transformative technology for basic and applied genomic science, increasing efficiencies and decreasing costs to enable studies of unprecedented scope and impact. In clinical settings, these technological and scientific advances have led to the development of tests that are increasingly fast, comprehensive, and more frequently employed. Practitioners of genomic medicine have applied these tools across clinical settings, including diagnosis of inherited disorders and cancers and infectious disease detection and surveillance. In recent years, the commercial marketplace for MPS sequencers and reagents has been dominated by a few companies. The growing demand for sequencing has led to the recent emergence of several new sequencing platforms with techniques that may provide alternatives or improvements to existing workflows or allow the adoption of sequencing workflows in new settings. Clinical genomics laboratories will evaluate these platforms from a unique perspective, focusing on how technological advancements can improve patient care.


CONTENT
This review describes short-read sequencing platforms provided by Illumina, Element Biosciences, MGI, PacBio, Singular Genomics, Thermo Fisher Scientific, and Ultima Genomics. This review discusses their innovative approaches, principles, workflows, and applications.


SUMMARY
This review aims to inform laboratory geneticists, clinicians, and researchers about emerging short-read technologies and their applications in clinical genomics. By highlighting their principles and potential contributions, we aim to assist laboratories in selecting suitable solutions for their sequencing needs considering key factors such as applications, throughput, and integration with existing laboratory workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc19721b678ba2d4cedee69c6b99d021887e0026" target='_blank'>
              Innovations in Short-Read Sequencing Technologies and Their Applications to Clinical Genomics.
              </a>
            </td>
          <td>
            Katarzyna Polonis, Joseph H. Blommel, Andrew E O Hughes, DH Spencer, Joseph A Thompson, Molly C Schroeder
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f316b25c9fd66656561befcef17f13bf1d2230" target='_blank'>
              Mcm5 mutation leads to silencing of Stat1-bcl2 which accelerating apoptosis of immature T lymphocytes with DNA damage
              </a>
            </td>
          <td>
            Min Liu, Yuanyuan Li, Zhilin Deng, Ke Zhang, Shuying Huang, Jiamin Xia, Yi Feng, Yundan Liang, Chengfu Sun, Xindong Liu, Shurong Li, B. Su, Yong Dong, Sizhou Huang
          </td>
          <td>2025-02-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Gamma-delta (γδ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09de8f88287444836b0106c88bdec6201177a72" target='_blank'>
              A rapid chemical reprogramming system to generate human pluripotent stem cells.
              </a>
            </td>
          <td>
            Yanglu Wang, Fangqi Peng, Zhihan Yang, Lin Cheng, Jingxiao Cao, Xiaodi Fu, Huanjing He, Ruyi Cai, Weizhen Zeng, Yingshuai Dong, Guanxian Chen, Gongxin Peng, Shi Liuyang, Guan Wang, Jinlin Wang, Rong Mu, Cheng Li, Jingyang Guan, Hongkui Deng
          </td>
          <td>2025-01-03</td>
          <td>Nature chemical biology</td>
          <td>3</td>
          <td>13</td>
        </tr>

        <tr id="Leukemia is characterized by multiple rearrangements of signal transduction genes and overexpression of nonmutated genes, such as transcription factors (TFs) genes. Super-enhancers (SEs) are prevalent in human cancers and are associated with the accumulation of numerous core TFs. SEs drive the expression of core TF genes by delivering robust transcriptional activation signals. Additionally, core TFs sustain the stability and activity of SEs through mutual auto-regulation loops, creating a positive feedback loop known as the Core Transcriptional Regulation Circuit (CRC). Using ChIP-seq data, we identified the involvement of the SE-related gene ZNF217 in acute myeloid leukemia (AML), in which its functional role and underlying mechanism remain unclear. We demonstrated that ZNF217, ELF1, MEF2D, RUNX2, and FOXP1 are likely integral components of the AML CRC through various experimental techniques, including CUT&Tag, short hairpin RNA (shRNA) transduction, and Luciferase reporter assays. Notably, ZNF217 was determined to be indispensable for the proliferation and viability of AML cells both in vitro and in vivo . Subsequent analysis of RNA-seq and CUT&Tag results identified MYB as a key direct target of ZNF217. Overall, our research highlights ZNF217 as a critical oncogene in AML and offers new insights into the transcriptional regulatory mechanisms at play in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/388657aaeaf8efe74b215605c5baaa2e335b1840" target='_blank'>
              Core transcriptional regulatory circuitry molecule ZNF217 promotes AML cell proliferation by up-regulating MYB
              </a>
            </td>
          <td>
            Bi Zhou, Fang Fang, Yongping Zhang, Zhiheng Li, Yixin Hu, Yan Li, Wanyan Jiao, YuMeng Wu, XiaoMei Wan, Ying Yang, Fenli Zhang, Ling Xu, Tongting Ji, Jian Pan, Shaoyan Hu
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07346b8a2c818bb0fe268d0dd9f1b1f14d6eac83" target='_blank'>
              CTHRC1 expresses in cancer-associated fibroblasts and is associated with resistance to anti-androgen therapy in prostate cancer.
              </a>
            </td>
          <td>
            Hai-Qi Liang, Qi-Huan He, Qiu-Ju Wei, Qi-Zhou Mo, Guang-Lin Yang, Fa-Ye Wei, Li-Wei Wei, Yu-Jian Li, Min Qin, 
          </td>
          <td>2025-02-26</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Background Immune response variation among sepsis patients leads to substantial clinical heterogeneity that makes treatment challenging. Bulk RNA-sequencing of circulating immune cells has been used in an attempt to resolve sepsis heterogeneity, and has facilitated classification of patients into different subtypes, or “endotypes”. The Sepsis Response Signature (SRS) classification comprises 3 endotypes: SRS1 represents an immunosuppressed transcriptional profile with high mortality; SRS2 a relatively immunocompetent response with low mortality, and SRS3 is a profile close to a normal immune response. However, because bulk RNA-seq does not distinguish the transcriptional contributions of different cell types, mechanistic insights from these classifications are lacking, thus limiting clinical utility. Single-cell RNA-sequencing (scRNA-seq) enables cell-specific transcriptomics and has the potential to demonstrate the cellular basis of sepsis endotypes. In this study, we performed scRNA-seq on blood from sepsis patients to investigate the differences in immune cell transcriptional states among SRS endotypes.Figure 1. Fractional abundance of monocyte substate 1 (MS1) is highest in the Sepsis Response Signature 1 (SRS1) endotype Fraction of MS1 cells relative to all peripheral blood mononuclear cells (PBMCs) in SRS1 (n= 10), SRS2 (n= 33), and SRS3 (n= 11) patients. Boxes show the median and inter-quartile range (IQR) for each endotype, with whiskers extending to 1.5x the IQR in either direction from the top or bottom quartile. Two-sided Wilcoxon rank-sum test with Benjamini-Hochberg correction used for comparisons; ***p<0.001, ****p<0.0001. Methods Mononuclear immune cells were isolated from the blood of sepsis patients (n=54) at hospital presentation and the transcriptomes of ∼1500 single cells per sample were analyzed. By aggregating gene counts for each patient, we generated pseudobulked transcriptional profiles to approximate whole blood gene expression. We then leveraged the SepstratifieR machine learning framework to stratify patients into SRS endotypes and compared the proportions of immune cell transcriptional states across the endotypes. Results A monocyte transcriptional state – monocyte substate 1 (MS1), was differentially expanded across SRS endotypes (Fig.1). MS1 abundance was significantly higher for patients in the SRS1 endotype compared to SRS2 (p< 0.001) or SRS3 (p< 0.0001). Conclusion Identifying cellular signals underlying endotypes may improve understanding of sepsis immunobiology and open paths to precision therapy. This study demonstrates the expansion of a monocyte transcriptional state in the SRS1 endotype. MS1 cells inhibit T cell activation and proliferation, suggesting a potential mechanistic explanation for the immunosuppressive state of SRS1 patients. Disclosures Michael R. Filbin, MD, Day Zero Diagnostics: Grant/Research Support|Quidel: Grant/Research Support">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bda1ab33c7fae8e980cd6b50f6a865d00d04a191" target='_blank'>
              P-2245. Single cell RNA-sequencing of immune cells demonstrates differential expansion of a monocyte transcriptional state among the Sepsis Response Signature (SRS) endotypes
              </a>
            </td>
          <td>
            P. Ankomah, Alyssa DuBois, Michael R. Filbin, N. Hacohen, R. Bhattacharyya
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b5bbe2c0fdac0090392cf87032a83637a36b8be" target='_blank'>
              Non-coding RNAs and exosomal ncRNAs in colorectal cancer
              </a>
            </td>
          <td>
            Yalda Zhoulideh
          </td>
          <td>2025-02-03</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) using metabolic RNA labeling enables detailed analysis of dynamic gene expression within single cells. However, most studies are limited to in vitro settings, restricting the exploration of in vivo transcriptomic dynamics. To address this, we developed scDyna-seq, a time-resolved scRNA-seq method for in vivo applications using 4-thiouridine (4sU) labeling. scDyna-seq efficiently captures nascent RNA, allowing for precise tracking of gene expression in both in vitro and in vivo contexts, including crossing the blood-brain and blood-fetal barriers. It is also compatible with other single-cell multiomics approaches. In a mouse bladder cancer model, scDyna-seq revealed that cisplatin (cis-diaminodichloroplatinum, CDDP) induced significant dynamic changes in tumor-infiltrating lymphocytes, particularly in genes related to costimulation, effector functions, and exhaustion, which were not detected by conventional methods. When coupled with scTCR-seq, scDyna-seq showed increased TCR clonal expansion linked to CDDP-induced immunogenic death and neoantigen production. In conclusion, scDyna-seq offers safe, precise in vivo RNA labeling as well as single-cell analysis, expanding our understanding of cellular dynamics and facilitating research in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7249f3e8ac6eca3f70d08c8938c34e4b84a4f3" target='_blank'>
              In Vivo Time-Resolved Single-Cell RNA-Seq Reveals Chemotherapy-Induced Transcriptional Dynamics in Tumor Infiltrating Lymphocytes.
              </a>
            </td>
          <td>
            Wenqi Zhu, Xiangyu Wu, Zihan Zhao, Meng Zhou, Tianyang Zhang, Derek Zhao, Ning Jiang, Xin Yang, Hongqian Guo, Nan Fang, Jie P. Li, Rong Yang, Tian Tian
          </td>
          <td>2025-02-05</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) refer to a group of RNA molecules that exceed a length of 200 nucleotides and lack the ability to code proteins. Numerous studies suggest that lncRNAs significantly contribute to the onset and progression of various forms of cancers. A specific lncRNA, known as long non-coding RNA 01296 (LINC01296), is extensively expressed in human malignancies. The level of LINC01296 has been shown to correlate with the progression and prognosis of cancers. Moreover, numerous scientific investigations have provided evidence that the dysregulation of LINC01296 functioning as a competitive endogenous RNA (ceRNA) exerts a profound influence on various aspects of cancer cell behavior, including proliferation, apoptosis, invasion, metastasis, and cell cycle progression, by means of regulating target genes and signaling pathways. An increasing body of data strongly suggests that LINC01296 may serve as a valuable biomarker for predicting cancer prognosis and could represent a promising therapeutic target for cancer intervention. In this comprehensive review, we summarize the recent advancements in our understanding of the role, underlying mechanisms, and clinical significance of LINC01296 in malignant tumors. The findings suggest that LINC01296 may be both a reliable biomarker and a potential therapeutic target for cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcfc7646d19e3cb4d25d1197357b2a9f9017093" target='_blank'>
              The Emerging Role of Long Non-Coding RNA 01296 in Human Malignancies.
              </a>
            </td>
          <td>
            Lei Luo, Fan Yang, Xiaoping Fu, Tingting Yu, Wenqian Tang, Juan Xue
          </td>
          <td>2025-01-31</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f2a7adbad7b64718a49a79d5c94f7b319bf5e0" target='_blank'>
              Single-cell assay for transposase-accessible chromatin sequencing of human clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Wenhao Lu, Yixuan Liu, Jiaping Li, Shengzhu Huang, Zheng Wen, Cheng Su, Zheng Lu, Zengnan Mo, Zhenyuan Yu
          </td>
          <td>2025-02-25</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="TRIP13 is a member of the large AAA+ ATPase protein superfamily that plays a crucial role in the precise segregation of chromosomes during mitosis. The abnormal function of TRIP13 has diverse functions, including mitotic processes, DNA repair pathways, and spindle assembly checkpoints, which may contribute to chromosomal instability (CIN). Emerging evidence suggests that the overexpression of TRIP13, observed in many cancers, plays a significant role in drug resistance, autophagy, and immune invasion. Recently, significant advances have been made in identifying TRIP13-associated signaling pathways that have been implicated in cancer progression. Several small molecules that specifically inhibit TRIP13 function and reduce cancer cell growth have been developed. Combination treatments, including TRIP13 inhibitors and other anticancer drugs, have shown promising results. While these findings are promising, TRIP13 inhibitors are awaiting clinical trials. This review discusses recent progress in understanding the oncogenic function of TRIP13 and its possible therapeutic targets, which could be exploited as an attractive option for cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31bad330d341da7462c6f7e15404b9c7c4e5e858" target='_blank'>
              Targeting Thyroid Hormone Receptor Interacting Protein (TRIP13) for Cancer Therapy: A Promising Approach
              </a>
            </td>
          <td>
            Surya P. Singh, Krishnendu Goswami, Gopal Pathuri, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-01-06</td>
          <td>DNA</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Recent advances in functional genomics tools have ushered in a new era of genetic editing to identify molecular pathways relevant to developmental and disease biology. However, limited model systems are available that adequately mimic cell states and phenotypes associated with human disease pathways. Here, we quantitatively analyzed the founder population bottleneck effect and demonstrated how the population changes from induced pluripotent stem cells (iPSCs) to hematopoietic stem cells and to the final induced macrophage population. We then engineered SAMHD1 knockout (KO) iPSC and characterized the iPSC line with RNA Seq, and induced macrophages from two distinct protocols with functional analysis. We then generated SAMHD1 KO CRISPR-dCAS9 KRAB iPSC through lenti-viral transduction aiming to increase the efficiency of lentiviral mediated gene transfer. We demonstrated increased lenti-viral transduction efficiency in induced macrophage, as well as microglia induced with two distinct protocols. This model allows for efficient gene knock down, as well as large-scale functional genomics screens in mature iPSC-derived macrophages or microglia with applications in innate immunity and chronic inflammatory disease biology. These experiments highlight the broad applicability of this platform for disease-relevant target identification and may improve our ability to run large-scale screens in iPSC-derived myeloid model systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091742b2e12364622b082bd3bd26a326b8f0d5f0" target='_blank'>
              SAMHD1 Knockout iPSC model enables high lenti-viral transduction in myeloid cell types
              </a>
            </td>
          <td>
            Huinan Li, Maliha Afroze, Gunisha Arora, S. Federman, Kaivalya Shevade, Yeqing Angela Yang, Phuong Nguyen, Rustam Esanov, Laralynne Przybyla, Adam Litterman, Shawn Shafer
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Bats have adapted to pathogens through diverse mechanisms, including increased resistance—rapid pathogen elimination, and tolerance—limiting tissue damage following infection. In the Egyptian fruit bat (an important model in comparative immunology), several mechanisms conferring disease tolerance were discovered, but mechanisms underpinning resistance remain poorly understood. Previous studies on other species suggested that the elevated basal expression of innate immune genes may lead to increased resistance to infection. Here, we test whether such transcriptional patterns occur in Egyptian fruit bat tissues through single-cell and spatial transcriptomics of gut, lung, and blood cells, comparing gene expression between bat, mouse, and human. Despite numerous recent loss and expansion events of interferons in the bat genome, interferon expression and induction are remarkably similar to that of mouse. In contrast, central complement system genes are highly and uniquely expressed in key regions in bat lung and gut epithelium, unlike in human and mouse. Interestingly, the unique expression of these genes in the bat gut is strongest in the crypt, where developmental expression programs are highly conserved. The complement system genes also evolve rapidly in their coding sequences across the bat lineage. Finally, the bat complement system displays strong hemolytic activity. Together, these results indicate a distinctive transcriptional divergence of the complement system, which may be linked to bat resistance, and highlight the intricate evolutionary landscape of bat immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a965bca0b5e54bd5045f231d1ba30a815dc790e7" target='_blank'>
              Single-cell and Spatial Transcriptomics Illuminate Bat Immunity and Barrier Tissue Evolution
              </a>
            </td>
          <td>
            Roy Levinger, Dafna Tussia-Cohen, Sivan Friedman, Yan Lender, Yomiran Nissan, Evgeny Fraimovitch, Yuval Gavriel, J. Tearle, Aleksandra A Kolodziejczyk, Kyung-Mee Moon, Tomás Gomes, Natalia Kunowska, M. Weinberg, Giacomo Donati, Leonard J Foster, Kylie R James, Y. Yovel, Tzachi Hagai
          </td>
          <td>2025-01-21</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Resistance to immune checkpoint inhibitors (ICIs) represents a major barrier to effectively treating patients with NSCLC. Increased tumor heterogeneity has been linked with diminished response to ICIs, but the mechanisms underlying such an effect remain underexplored, in part due to the lack of models for studying the impact of heterogeneity. To address this issue, we have developed a novel murine model of NSCLC that utilizes clonal cell lines to modulate tumor heterogeneity. Specifically, we isolated single cells from a murine NSCLC cell line harboring 3 common driver mutations (Kras G12D Tp53 +/- Lkb1 -/- , or “KPL”). We mutagenized KPL cells with N-methyl-N-nitrosourea in order to increase the tumor mutational burden, and then isolated single cells to derive clonal cell lines. Whole exome sequencing and subsequent phylogenetic analysis of tumor mutations yielded categorization of these clones into 5 clusters. Clones within the same cluster demonstrated comparable in vivo growth kinetics and similar response to anti-PD-1 immunotherapy. These clones exhibited unique tumor immune microenvironments that did not fully correlate with genomic factors such as tumor mutation burden. Notably, heterogeneous polyclonal tumors grew more aggressively than monoclonal tumors, and they exhibited response to anti-PD-1 therapy that differed from the response seen in the component single clones. Tracking of individual clones with fluorescent proteins and mutation tracing illustrated clonal shifts induced by ICI treatment. Our results validate this system as a model of NSCLC that can modify tumor heterogeneity and provide insights into the effects of heterogeneity on ICIs and other immunotherapies.
 Citation Format: Michael Oh, Jensen W Abascal, Linh M Tran, Ramin Salehi-Rad, Raymond J Lim, Camelia Dumitras, Cara Yean, Sunny Xiao, Bitta P Kahangi, William P Crosson, Edgar Perez Reyes, Nalani Coleman, Austin K Rennels, Kostyantyn Krysan, Bin Liu, Steven M Dubinett. A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The heterogeneity of the hematopoietic system was largely veiled by traditional bulk sequencing methods, which measure the averaged signals from mixed cellular populations. In contrast, single-cell sequencing has enabled the direct measurement of individual signals from each cell, significantly enhancing our ability to unveil such heterogeneity. Building on these advances, numerous single-cell multi-omics techniques have been developed into high-throughput, routinely accessible platforms, delineating the precise relationships among the different layers of the central dogma in molecular biology. These technologies have uncovered the intricate landscape of genetic clonality and transcriptional heterogeneity in both normal and malignant hematopoietic systems, highlighting their roles in differentiation, disease progression, and therapy resistance. This review aims to provide a brief overview of the principles of single-cell technologies, their historical development, and a subset of ever-expanding multi-omics tools, emphasizing the specific research questions that inspired their creation. Amidst the evolving landscape of single-cell multi-omics technologies, our main objective is to guide investigators in selecting the most suitable platforms for their research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="We report the engineering of lipid nanoparticles (LNPs) to transport CRISPR/Cas9 payloads, including double-stranded DNA (dsDNA) donor templates, designed for homology directed repair (HDR)-mediated site-specific insertion of the cystic fibrosis transmembrane conductance regulator (CFTR) gene to correct cystic fibrosis (CF) in diseased airway epithelium. We screened various nanoparticle formulations, adjusting ratios of Cas9-encoding mRNA, single guide RNAs (sgRNAs), and dsDNA donor templates to optimize gene editing using human bronchial epithelial cells (16HBE14o-) harboring a CF-causing mutation (G542X). Populations of G542X cells edited via LNP delivery of CFTR donors achieved 3 – 3.5% gene integration and yielded comparable CFTR protein expression compared to normal 16HBE14o- controls. These edited populations exhibit restoration of CFTR-dependent Cl- current to ca. 80% of values measured in normal 16HBE14o- cell monolayers. This LNP platform adds capabilities for transporting large gene editing machinery to airway epithelial cells for genomic integration of entire genes, enabling therapeutic solutions that achieve correction of any CF-causing mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0097561835a41509ae48e38631c6aa471820400c" target='_blank'>
              Lipid Nanoparticles for the Delivery of CRISPR/Cas9 Machinery to Enable Site-Specific Integration of CFTR and Mutation-Agnostic Disease Rescue
              </a>
            </td>
          <td>
            Ruth A. Foley, P. Ayoub, Vrishti Sinha, C. Juett, A. Sanoyca, Emily C. Duggan, Lindsay Lathrop, Priyanka Bhatt, Kevin Coote, Beate Illek, B. Gomperts, Donald B. Kohn, S. Jonas
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Transcriptional control of transgene expression can be linked to dynamic changes in cellular states if this is accompanied by differential expression of transcription factors (TFs). Synthetic promoters (SPs) designed to respond to the desired TFs can provide this regulation with compactness, specificity, and orthogonality. T cells display differentially expressed TFs according to the functional state. In solid tumors, the highly immunosuppressive TME and the chronic exposure to antigens lead to a progression of T cells from a functional to a dysfunctional state known as exhaustion (Tex), in which their power against cancer cells is strongly compromised. Importantly, this transition is accompanied by a marked increase of several TFs, among other factors, that drive targeted genetic programs. Strategies to detect and mitigate Tex are extremely needed. Here, we design SPs that respond to TFs differentially expressed in activated and exhausted T cells to enable new classifiers of the functional/dysfunctional states. We developed a library of over 80 SPs responsive to 7 TFs. The SPs showed broad strength of activation of reporter genes or immunomodulatory molecules in HEK293 and Jurkat T cell lines. Moreover, using a transfer learning strategy we show SPs strength predictability. By combining SPs responding to different TFs, we created Boolean logic gates and implemented a feed-forward design that was previously shown to reduce noise in the OFF-state. Finally, as proof of principle, we demonstrate the dynamic activation of the NR4A2-responsive SP according to the T cell state in primary human CD8+ T cells. Collectively we present a sensing platform that provides a versatile tool to study and monitor the dynamic changes occurring in T cells. In perspective the biosensors coupled to therapeutic genes can be used to reprogram the TME and reinvigorate the T cell anti-tumoral functionality, preventing or reverting the exhausted phenotype. Graphical Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396338870509bf01505cfe58f9f60b72b3d35505" target='_blank'>
              Design of novel synthetic promoters to tune gene expression in T cells
              </a>
            </td>
          <td>
            Giuliano Bonfá, Giovanna Martino, Assunta Sellitto, A. Rinaldi, F. Tedeschi, Fabio Caliendo, Loris Melchiorri, Daniela Perna, Freddie Starkey, Evangelos-Marios Nikolados, Filippo Menolascina, V. Siciliano
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive. In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby preserving HSC function and promoting age-dependent expansion. Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increases in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decreases in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging. In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.
 Citation Format: Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Registered systemic treatment options for glioblastoma patients are limited. The phase II REGOMA trial suggested an improvement of median overall survival in progressive glioblastoma by the multi-tyrosine kinase inhibitor regorafenib. This has not been confirmed by GBM AGILE. So far, regorafenib has been administered as monotherapy or as an addition to standard of care in newly diagnosed glioblastoma. Rational combination therapies involving regorafenib might be a reasonable strategy. Here, we aimed at identifying functionally-instructed combination therapies involving regorafenib.


METHODS
We applied a genome-wide CRISPR-Cas9-based functional genomics target discovery approach using activation and knockout screens followed by genetic, pharmacological, functional validations. Regorafenib-induced molecular alterations were assessed by RNAsequencing and DigiWest. We investigated selected functionally-instructed combination therapies in three orthotopic glioma mouse models in vivo (syngeneic SMA560/VM/Dk model and two xenograft models) and performed immunohistochemistry of post-treatment brains.


RESULTS
We identified potential modifiers of regorafenib response including BCL2, BCL2L1, ITGB3, FOXC1, SERAC1, ARAF, and PLCE1. The combination of regorafenib with Bcl-2/Bcl-xL inhibition was superior to both monotherapies alone in vitro, ex vivo and in vivo. We identified regorafenib-induced regulations of the Bcl-2 downstream target chemokine receptor 1 (CCR1) as one potential underlying molecular mediator. Furthermore, regorafenib led to changes in the myeloid compartment of the glioma-associated microenvironment.


CONCLUSION
This preclinical study uses a functional genomics-based target discovery approach with subsequent validations involving regorafenib. It serves as a biological rationale for clinical translation. Particularly, an investigation of the combination of regorafenib plus navitoclax within a clinical trial is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d3a5d392764ac21d48a9e0c727584ef62c0c8" target='_blank'>
              Genome-wide CRISPR-Cas 9 screens identify BCL family members as modulators of response to regorafenib in experimental glioma.
              </a>
            </td>
          <td>
            L. Haeusser, H. Becker, L. Kuhlburger, Marcello Zago, B. Walter, F. Tsiami, Sarah Erdmann, Jil Trampert, Surender Surender, Aaron Stahl, M. Templin, Eileen Wegner, Tobias Schmidt, C. Schmees, N. Casadei, Lisa Sevenich, Manfred Claassen, S. Nahnsen, Susanne Beck, D. Merk, G. Tabatabai
          </td>
          <td>2025-01-04</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Chromosomal aneuploidies are a major cause of developmental failure and pregnancy loss. To investigate the possible consequences of aneuploidy on early embryonic development in vitro, we focused on primed pluripotent stem cells that are relatable to the epiblast of post-implantation embryos in vivo. We used human induced pluripotent stem cells (iPSCs) as an epiblast model and altered chromosome numbers by treating with reversine, a small-molecule inhibitor of monopolar spindle 1 kinase (MSP1) that inactivates the spindle assembly checkpoint, which has been strongly implicated in chromosome mis-segregation and aneuploidy generation. Upon reversine treatment, we obtained cells with varied chromosomal content that retained pluripotency and potential to differentiate into cells of three germ lineages. However, these cells displayed lagging chromosomes, increased micronuclei content, high p53 expression and excessive apoptotic activity. Cell proliferation was not affected. Prolonged in vitro culture of these cells resulted in a selective pool of cells with supernumerary chromosomes, which exhibited cellular hypertrophy, enlarged nuclei, and overproduction of total RNAs and proteins. We conclude that increased DNA damage responses, apoptosis, and improper cellular mass and functions are possible mechanisms that contribute to abnormal epiblast development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d165ff1955e4cb3146937b90c98c9776c59f3b72" target='_blank'>
              Chromosome number alterations cause apoptosis and cellular hypertrophy in induced pluripotent stem cell models of embryonic epiblast cells
              </a>
            </td>
          <td>
            Althea Stella Anil Martis, Loshini Soundararajan, Pallavi Shetty, Syed Moin, Tejashree Vanje, Yogeshwaran Jai Sankar, Shagufta Parveen
          </td>
          <td>2025-01-15</td>
          <td>Biology Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Since the discovery of oncogenic neurotrophic receptor tyrosine kinase (NTRK) gene fusions in colorectal cancer in 1986, their understanding has evolved, particularly in non-small-cell lung cancer (NSCLC) over the past five years. NTRK rearrangements, involving NTRK1, NTRK2, and NTRK3, drive tumorigenesis and have been identified in various adult and pediatric cancers, with over 80 different fusion variants in several type of cancers. Detecting these rearrangements is crucial for targeted therapy strategies. The aim of this study is detect, compare and analyse these mutations in NSCLC patients of a cohort of 482 cases from Karolinska University Hospital. Methods: We conducted an initial screening using pan-TRK immunohistochemistry (IHC) to analyze the material. Positive cases were further examined through whole-exome sequencing (WES) with next-generation sequencing (NGS) to confirm the presence of fusions. Additionally, to deepen our understanding, we utilized Ingenuity Pathway Analysis (IPA) software, an artificial intelligence-driven technology, to explore the molecular pathways involved in lung cancer. Results: TRK overexpression was detected in 4.56% of cases via IHC. Among 15 pan-TRK-positive cases, WES confirmed fusions in 3, revealing a higher prevalence of NTRK1 (6.6%) and NTRK2 (13.3%) fusions, while no NTRK3 fusions were observed. Conclusion: Our findings confirm the low prevalence of these neoplasms as well as the need for a molecular test to confirm rearrangements or other potentially treatable mutations and raise other questions regarding their clinical use. However, there is an acceptable correlation between pan-TRK IHC and NTRK mutations, but not enough to determine NTRK fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ebb427a9c843d7fe3222c86733eb64eff16226" target='_blank'>
              NTRK Gene Expression in Non-Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Jair Gutierrez-Herrera, M. A. Montero-Fernandez, Georgia Kokaraki, L. Petris, Raul Maia Falcão, Manuel Molina-Centelles, Ricardo Guijarro, Simon Ekman, C. Ortiz-Villalón
          </td>
          <td>2025-02-27</td>
          <td>Journal of Respiration</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 technology has rapidly advanced as a pivotal tool in cancer research, particularly in the precision targeting required for both detecting and treating malignancies. Its high specificity and low off-target effects make it exceptionally effective in applications involving Human Papillomavirus (HPV) related diseases, most notably cervical cancer. This approach offers a refined methodology for the rapid detection of viral infections and provides a robust platform for the safe and effective treatment of diseases associated with viral infections through gene therapy. Purpose: Gene therapy, within this context, involves the strategic delivery of genetic material into target cells via a vector. This is followed by the meticulous modulation of gene expression, whether through correction, addition, or suppression, specifically honed to target tumor cells while sparing healthy cells. This dual capacity to diagnose and treat at such a precise level underscores the transformative potential of CRISPR/Cas9 in contemporary medical science, particularly in oncology and virology. Research Design: This article provides an overview of the advancements made in utilizing the CRISPR-Cas9 system as a research tool for HPV-related treatments while summarizing its application status in basic research, diagnosis, and treatment of HPV. Data Collection: Furthermore, it discusses the future prospects for this technology within emerging areas of HPV research and precision medicine in clinical practice, while highlighting technical challenges and potential directions for future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69224d4406f31958e4148b5cda3be164a3c3d00" target='_blank'>
              Research Status of Clustered Regulary Interspaced Short Palindromic Repeats Technology in the Treatment of Human Papillomavirus (HPV) Infection Related Diseases
              </a>
            </td>
          <td>
            Minxue Tang
          </td>
          <td>2025-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bae74b424ea31be0a19ad896060dac632aa75c9" target='_blank'>
              A multiparametric anti-aging CRISPR screen uncovers a role for BAF in protein synthesis regulation
              </a>
            </td>
          <td>
            Sophia Y Breusegem, Jack Houghton, Raquel Romero-Bueno, Adrián Fragoso-Luna, K. Kentistou, Ken K Ong, Anne F J Janssen, Nicholas A Bright, Christian G Riedel, J. R. Perry, P. Askjaer, Delphine Larrieu
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4f25866d80529aa37b7f953a4255c06039896" target='_blank'>
              A Protocol for Detecting DNA Methylation Changes at CpG Sites of Stemness-Related Genes in Aging Stem Cells.
              </a>
            </td>
          <td>
            M. Hodjat, S. M. Daghighi, Mohammad Abdollahi
          </td>
          <td>2025-01-17</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This manuscript explores advancements in antigen-specific immunosuppression and the potential and challenges for gene-editing application in autoimmune diseases which are driven by autoantibodies (AAbs). Current approved treatments fail to reach long-lived plasma cells (LLPCs), which may continue secreting pathogenic AAbs after immunobiological courses in some autoimmune illnesses. New approaches, some tested in vitro and some already undergoing clinical trials, such as the chimeric autoantibody receptor (CAAR)-T cells, BiTEs, affinity matrices, and CRISPR-AAb-untargeting, show promise in selectively eliminating autoreactive B lymphocytes. Challenges remain, particularly in adapting CRISPR for in vivo use. The future of precision medicine for autoimmunity will rely in AI-guided personalized approaches, enabling the design of highly specific therapies to target disease-causing autoreactive cells while preserving immune function, marking a transformative step in autoimmune treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d39bbe27322aea952bd24734f3848ee075cac915" target='_blank'>
              Precision medicine for Autoimmunity: From CAAR-T Cells to AI-Driven CRISPR-based therapies, challenges and perspectives
              </a>
            </td>
          <td>
            G. D. Keppeke
          </td>
          <td>2025-01-09</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Objective This study aims to explore the role of exosome-related genes in breast cancer (BRCA) metastasis by integrating RNA-seq and single-cell RNA-seq (scRNA-seq) data from BRCA samples and to develop a reliable prognostic model. Methods Initially, a comprehensive analysis was conducted on exosome-related genes from the BRCA cohort in The Cancer Genome Atlas (TCGA) database. Three prognostic genes (JUP, CAPZA1 and ARVCF) were identified through univariate Cox regression and Lasso-Cox regression analyses, and a metastasis-related risk score model was established based on these genes. Immune cell infiltration, immune escape and drug sensitivity disparities between high- and low-risk groups were assessed using CIBERSORT and single-sample gene set enrichment analysis (ssGSEA) methods. High- and low-risk cell populations were discerned based on the expression of prognostic genes in BRCA scRNA-seq data. Results M0 and M1 macrophages significantly promote the metastasis of breast cancer (BRCA). The developed prognostic model demonstrates good predictive performance for patient survival at 1, 3 and 5 years, with AUC values of 0.654, 0.602 and 0.635, respectively. Compared to the low-risk group, the high-risk group exhibits increased immune cell infiltration and higher levels of immune evasion. scRNA-seq data reveal that high-risk cells have significantly higher risk scores and exhibit notable differences in signalling pathways and intercellular communication patterns. Conclusions This study presents a novel risk score model based on exosome-related genes, validated by comprehensive analyses including differential expression, survival analysis and external dataset validation. The model’s clinical significance is reinforced through its ability to stratify patients into high- and low-risk groups with distinct survival outcomes and immune landscape characteristics. The integration of RNA-seq and scRNA-seq data highlights the predictive accuracy of the model and underscores its potential for identifying novel therapeutic targets and improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a1ffcb440b848f5eb3e596987e61442355c1386" target='_blank'>
              Integrating RNA-seq and scRNA-seq to explore the prognostic features and immune landscape of exosome-related genes in breast cancer metastasis
              </a>
            </td>
          <td>
            Guanyou Huang, Yong Yu, Heng Su, Hongchuan Gan, Liangzhao Chu
          </td>
          <td>2025-01-23</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mutations found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of healthy adults - a phenomenon termed clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. We assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis, remission and/or relapse (total n=182 patients). FLT3 and NPM1 mutations were generally eliminated at complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, CH-associated mutations that likely precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0b511bf0bfbb32bc9fd1061ccf533beadb0c77" target='_blank'>
              Persistent post-remission clonal hematopoiesis shapes the relapse trajectories of acute myeloid leukemia.
              </a>
            </td>
          <td>
            Ryan D Chow, Priya Velu, Safoora Deihimi, Jonathan P Belman, Angela Youn, Nisargbhai Shah, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2025-02-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>