<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-30 09:31:51 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>43</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer cells that survive therapeutic drug pressure are a significant cause of disease relapse and progression, impeding curative cancer treatment. Drug-triggered Darwinian selection and the emergence of subclones harbouring specific mutations that confer resistance have been well documented and extensively studied. However, these genetic alterations, while important, do not fully explain clinical observations where some patients, after a drug holiday, regain sensitivity to the same treatment despite previous disease progression. This phenomenon highlights the possibility that drug resistance may not solely rely on genetic mutations but could also involve reversible, non-genetic mechanisms. Recent studies have highlighted the existence of drug-tolerant persister cells (DTPs), a subpopulation of cancer cells that can survive short-term therapeutic pressure without acquiring resistance-associated genetic alterations. These cells exhibit a temporary yet reversible tolerance to the initial treatment while also acquiring cross-tolerance to other anti-cancer therapies. The presence of DTPs underscores a dynamic and complex plasticity in tumours, wherein cancer cells can utilise epigenetic rewiring, metabolic reprogramming, and specific signalling pathways to transit between drug-tolerant and drug-sensitive states to adapt to environmental pressures. Furthermore, this adaptive resilience enables DTPs to act as a reservoir for the development of genetically stable resistance, resulting in cancer therapy failure and eventual relapse. In this mini-review, we examine recent evidence on DTPs to provide an overview of their characteristics, development, and survival mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f885f194fb8254e0480e9608eccdddbd6818a86e" target='_blank'>
              Targeting Cancer Drug-Tolerant Persister Cells in Minimal Residual Disease
              </a>
            </td>
          <td>
            Wei Zhang, Sorayut Chattrakarn, Fuhui Chen, Haobin Chai, Mariana Maranga, Jingwei Zhang
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Drug Discovery and Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19dad7b139ef16ca0124619b3b70027a193368b" target='_blank'>
              Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Andrew Liao, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/998e8a8abe04f7f2be63264d103e59a69474a153" target='_blank'>
              Adaptation by stochastic tuning of gene expression in mammalian cells
              </a>
            </td>
          <td>
            Amir Momen-Roknabadi, P. Oikonomou, Saeed Tavazoie
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2397ca3adf8845c9715ac4636c6b9da727b026" target='_blank'>
              Epigenetic dynamics and molecular mechanisms in oncogenesis, tumor progression, and therapy resistance.
              </a>
            </td>
          <td>
            Mohammed Kaleem, Lubna Azmi, N. Shahzad, Murtada Taha, Shiv Kumar, Md. Ali Mujtaba, Abdulaziz Ali H Hazazi, A. Kayali
          </td>
          <td>2025-05-13</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a leading cause of cancer-related mortality in men worldwide, with tumor heterogeneity, therapy resistance, and lineage plasticity posing significant clinical challenges. Androgen deprivation therapy (ADT), while initially effective, often culminates in castration-resistant prostate cancer (CRPC), may fueled by castration-resistant prostate cancer stem cells (CSCs) with adaptive self-renewal and regenerative capacities. Emerging evidence implicates prostate CSCs as pivotal contributors to tumor heterogeneity, drug resistance, and disease recurrence. These stem subpopulations exhibit intrinsic adaptability through genetic, epigenetic, and microenvironmental reprogramming, enabling survival under androgen-deprived conditions and fostering clonal diversification. This review synthesizes current knowledge on the identity, regulation, and functional dynamics of prostate stem cells, emphasizing their role in shaping the tumor ecosystem. By dissecting the genetic drivers, epigenetic alteration and crosstalk with microenvironment, this review underscores critical triggers on prostate CSCs determination and differentiation. Additionally, we discuss emerging strategies to target prostate CSCs-specific vulnerabilities, including molecular drivers of stemness and plasticity, to improve therapeutic outcomes for advanced prostate malignancies. This synthesis underscores the critical need to unravel prostate stem cells biology for developing precision therapies against prostate CSCs-driven adaptation in prostate diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/749e1d03aa4d620992d251430c919ea9331b3e5d" target='_blank'>
              Prostate cancer stem cell dynamics in the evolution of drug resistance.
              </a>
            </td>
          <td>
            Xiaoyu Zhang, Xiaoqing Zhang, Xinpei Deng, Jingna Cao, Qing Bao, Huan Wang, Dong Chen, Hailin Tang
          </td>
          <td>2025-06-04</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Anticancer chemotherapy is an essential part of cancer treatment, but the emergence of resistance remains a major hurdle. Metabolic reprogramming is a notable phenotype associated with the acquisition of drug resistance. Here, we develop a computational framework that predicts metabolic gene targets capable of reverting the metabolic state of drug-resistant cells to that of drug-sensitive parental cells, thereby sensitizing the resistant cells. The computational framework performs single-gene knockout simulation of genome-scale metabolic models that predicts genome-wide metabolic flux distribution in drug-resistant cells, and clusters the resulting knockout flux data using uniform manifold approximation and projection, followed by k-means clustering. From the clustering analysis, knockout genes that lead to the flux data near that of drug-sensitive cells are considered drug sensitization targets. This computational approach is demonstrated using doxorubicin- and paclitaxel-resistant MCF7 breast cancer cells. Drug sensitization targets are further refined based on proteome and metabolome data, which generate GOT1 for doxorubicin-resistant MCF7, GPI for paclitaxel-resistant MCF7, and SLC1A5 as a common target. These targets are experimentally validated where treating drug-resistant cancer cells with small-molecule inhibitors results in increased sensitivity of drug-resistant cells to doxorubicin or paclitaxel. The applicability of the developed framework is further demonstrated using drug-resistant triple-negative breast cancer cells. Taken together, the computational framework predicts drug sensitization targets in an intuitive and cost-efficient manner and can be applied to overcome drug-resistant cells associated with various cancers and other metabolic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7052bad828f26940bd1911c2893c8896c16d13b8" target='_blank'>
              Genome-scale knockout simulation and clustering analysis of drug-resistant breast cancer cells reveal drug sensitization targets.
              </a>
            </td>
          <td>
            JinA Lim, Hae Deok Jung, S. Park, Moonhyeon Jeon, Da Sol Kim, Ryeongeun Cho, Dohyun Han, Han Suk Ryu, Yoo-Ree Kim, Hyun Uk Kim
          </td>
          <td>2025-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Locally advanced esophageal adenocarcinoma remains difficult to treat and the ecological and evolutionary dynamics responsible for resistance and recurrence are incompletely understood. Here, we performed longitudinal multiomic analysis of patients with esophageal adenocarcinoma in the MEMORI trial. Multi-region multi-timepoint whole-exome and paired transcriptome sequencing was performed on 27 patients before, during and after neoadjuvant treatment. We found major transcriptomic changes during treatment with upregulation of immune, stromal and oncogenic pathways. Genetic data revealed that clonal sweeps through treatment were rare. Imaging mass cytometry and T cell receptor sequencing revealed remodeling of the tumor microenvironment during treatment. The presence of genetic immune escape, a less-cytotoxic T cell phenotype and a lack of clonal T cell expansions were linked to poor treatment response. In summary, there were widespread transcriptional and environmental changes through treatment, with limited clonal replacement, suggestive of phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286e0cfb0095dd021b03614405167d3960808941" target='_blank'>
              Evolutionary and immune microenvironment dynamics during neoadjuvant treatment of esophageal adenocarcinoma
              </a>
            </td>
          <td>
            M. Barroux, J. Househam, E. Lakatos, T. Ronel, Ann-Marie Baker, Henrike Salié, M. Mossner, Kane Smith, C. Kimberley, S. Nowinski, Alison Berner, V. Gunasri, Martin Borgmann, S. Liffers, Marnix Jansen, G. Caravagna, K. Steiger, J. Slotta-Huspenina, Wilko Weichert, Luis Zapata, Eleftheria Giota, Sylvie Lorenzen, M. Alberstmeier, Benjamin M Chain, H. Friess, B. Bengsch, Roland M. Schmid, J. Siveke, M. Quante, Trevor A. Graham
          </td>
          <td>2025-05-14</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Next-generation sequencing technology enables uniform and impartial assessment of cancer diagnoses and prognosis. However, such studies are mostly type-specific, and capturing shared genomic abnormalities responsible for neoplastic transformation and progression is a challenging task. Pan-cancer analysis offers insights into the shared and unique molecular mechanisms driving cancer. We conducted an integrated gene-expression analysis using 10,629 samples from 30 distinct cancer types characterized by The Cancer Genome Atlas (TCGA). A gene co-expression network was constructed and genes overlapping between the selected modules and Differentially Expressed Genes (DEGs) were designated as genes of interest. Following a comprehensive literature review, ATP binding cassette subfamily A member 10 (ABCA10) and ATP binding cassette subfamily B member 5 (ABCB5) were selected as key candidates for downstream analysis due to the absence of systematic pan-cancer analysis of these genes. This study presents a unique contribution as the first comprehensive pan-cancer analysis of ABCA10 and ABCB5, highlighting their roles in tumor biology and clinical outcomes. We employed a variety of bioinformatics tools to explore the role of these genes across different tumors. Our research demonstrated that ABCA10 shows reduced expression, while ABCB5 displays variable expression patterns across tumors, indicating their opposing roles and flexible functions in pan-cancer. In many cancer patients, these expression patterns are correlated with worse survival outcomes. Furthermore, immunotherapy responses and immune infiltration across a variety of tumor types are associated with the expression levels of both ABCA10 and ABCB5. These results imply that ABCA10 and ABCB5 could serve as valuable predictive markers and potential therapeutic targets across various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f1be3a2286bbe9d4105e8c1625c6913bc7d7bf" target='_blank'>
              Integrated pan-cancer analysis revealed therapeutic targets in the ABC transporter protein family
              </a>
            </td>
          <td>
            M. B. E. Masood, Iqra Shafique, Muhammad Inam Rafique, Ayesha Iman, Ariba Abbasi, Mehak Rafiq, Uzma Habib
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771ef0b5061a9c5213ccca9fb3801ba5c0ea8357" target='_blank'>
              Whole Exome Sequencing and Single-Cell DNA Sequencing for Assessment of Clonal Heterogeneity and Evolution in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sadiksha Adhikari, S. Eldfors, Esa Pitkänen, Markus Vähä-Koskela, C. Heckman
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 e20582


 Background:
 Despite advances in therapies, the development of drug resistance in non-small cell lung cancer (NSCLC) poses a significant challenge. The emergence of neoantigen-based therapies offer a promising new frontier which enhances treatment efficacy and reduces associated costs.
 Methods:
 A total of 24 patients diagnosed with NSCLC were enrolled in this study. Primary tumor tissues and peripheral blood samples were collected from each patient. Paired whole exome and transcriptomic sequencing data were performed to identify somatic mutations and to quantify their respective expression levels. Somatic single nucleotide variants (SNVs) and small insertions or deletions (inDels) were detected using Strelka algorithm from whole exome sequencing (WES) data. Neoantigen prediction focused on human leukocyte antigens (HLA) type 1. HLA alleles were predicted using WES data encoding adjacent non-cancerous tissue samples, identifying four alleles that were present in at least 50% of the patients. Neoantigens were considered potentially immunogenic if their predicted median IC50 binding scores were ≤500nM while expressed transcripts per million (TPM) exceeds 1 in tumor samples.
 Results:
 Using systematic bioinformatics pipeline, we analyzed 24 NSCLC samples with matched tumor tissues and peripheral blood, and detected the 4677 mutations involving 2442 genes, including 2819 missense SNVs, 941 InDels, 229 gene fusion and 688 RNA alternative splicing events, 94.7% of which (4428 mutations involving in 2247 genes) were common with those reported in the TCGA-LUAD dataset. The average number of mutations was 194.8 in NSCLCs. We found that C>T/G>A transitions/ transversions were dominant. Missense mutations were the most frequent types of somatic mutation in the coding sequence regions. 98% of mutated genes were not detected simultaneously with DNA and RNA mutations.
 ADCK5
 ,
 CD151
 and
 EIF3K
 were identified as potential new driver genes. Genomic variation profiling identified 2213 potential neoantigens with a median IC50 binding score of ≤500nM and a transcript expression level of >1 TPM. Among these, 41 were listed in databases TSNAdb, IEDB, and CTDatabase. A total of 14 genes harboring identical mutations were detected, with the identical mutations observed in over 50% of the patient cohort. Notably, mutations in 14 genes were either gene fusions or RNA alternative splicing events, and core antigenic sequences generated were found to be identical in the majority of patients. Of particular interest is the RPRD2 gene, where an RNA exitrons site was detected in all patient samples, along with the identification of the same neoantigen core sequence.
 Conclusions:
 Our analysis suggests that gene fusions and RNA splicing variations are more likely than point mutations to generate shared antigens, underscoring their importance for developing targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1725e99e0357b9a0ce8d33b25d7c68d81214eed0" target='_blank'>
              Comprehensive genomic alterations identification and potential shared neoantigens for non-small cell lung cancer immunotherapy.
              </a>
            </td>
          <td>
            Muchun Zhu, Qing Zhao Ruan, Boyue Zhang, Weiqiang Yin, Run Sheng Ruan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175ce2c2bc9bbd6350000e462ee520fe8e8d597c" target='_blank'>
              Global propagation of genetic perturbation effects through genome-wide stoichiometry conservation architecture
              </a>
            </td>
          <td>
            Genta Chiba, Ken-ichiro F. Kamei, Arisa H. Oda, Kunihiro Ohta, Y. Wakamoto
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colon cancer (CC) is one of the most common and lethal cancers worldwide, with rising incidence rates in both developed and developing countries. Although advances in treatments such as surgery, chemotherapy, and targeted therapies have been made, prognosis for advanced colon cancer, particularly with metastasis, remains poor. Recent studies highlight the significant role of post-transcriptional modifications like acetylation in cancer biology, affecting processes like gene transcription, metabolism, and tumor progression. Methods This study applied multi-omics analyses, including single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and Mendelian randomization. Data were obtained from public datasets like GSE132465, UCSC Xena, and GeneCards. We focused on acetylation-related genes, specifically NAT10 and GNE, using scoring methods, cell–cell interaction models, and survival analyses to investigate their role in colon cancer development, metastasis, and immune evasion. Results This study identifies that NAT10 is highly expressed in epithelial cells of colorectal cancer (CC) and is closely associated with tumor progression and metastasis. Single-cell RNA sequencing analysis revealed that NAT10-positive epithelial cells exhibited strong interactions with myeloid cells and T cells, with significant differences in cell–cell communication (p < 0.05). Based-on-summary-data Mendelian randomization (SMR) analysis further supports a causal relationship between NAT10 and colorectal cancer. In the MR analysis, a significant positive correlation was observed between NAT10 and colorectal cancer risk using summary data from genome-wide association studies (GWAS) and expression quantitative trait loci (eQTL) studies (β_SMR = 0.004, p_SMR = 0.041, p_HEIDI = 0.737). These findings suggest that NAT10 may serve as a pathogenic factor in colorectal cancer development, providing additional genetic evidence that links this acetylation-related gene to colorectal cancer. Survival analysis further demonstrated that NAT10-positive epithelial cells are associated with poorer prognosis. In the TCGA dataset, patients with NAT10-positive epithelial cells exhibited a significantly shorter disease-free survival (DFS) (p = 0.012). Unlike GNE-positive cells, NAT10-positive epithelial cells exhibited immune escape characteristics, and TIDE analysis indicated that NAT10-positive epithelial cells were associated with a lower response to immune checkpoint blockade therapy (p = 1.3e−5), suggesting that they may impair the efficacy of immunotherapy by promoting immune evasion. In contrast, GNE was also significantly expressed in epithelial cells of colorectal cancer, but its role differs from that of NAT10. GNE-positive epithelial cells demonstrated strong communication with immune cells, particularly in interactions between myeloid cells and T cells through receptor-ligand pairs. Despite the important role of GNE-positive epithelial cells in the tumor microenvironment, their association with immune escape is weaker compared to NAT10. Survival analysis revealed that GNE-positive epithelial cells were associated with a better prognosis (p = 0.015). In the TCGA dataset, patients with GNE-positive epithelial cells displayed longer disease-free survival (DFS), contrary to the results from the SMR analysis. Conclusions Leveraging SMR and multi-omics analysis, this study highlights the significant role of acetylation-related genes, particularly NAT10, in colon cancer. The findings suggest that acetylation modifications in epithelial cells contribute to immune evasion and cancer progression. NAT10 could serve as a promising biomarker and therapeutic target for early diagnosis and targeted therapy, offering new avenues for improving colon cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37348c834755949c2631ebe0e9fc050634d0c0ce" target='_blank'>
              Deciphering the role of acetylation-related gene NAT10 in colon cancer progression and immune evasion: implications for overcoming drug resistance
              </a>
            </td>
          <td>
            Xuancheng Zhou, Xun Sang, Lai Jiang, Shengke Zhang, Chenglu Jiang, Yuheng Gu, Yiping Fu, Guanhu Yang, Jieying Zhang, Hao Chi, Binbin Wang, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5066c2b7365798ed65ee7d8872b1b51cf8054789" target='_blank'>
              Long-read DNA and RNA sequencing for inherited polyposis and colorectal cancer: cryptic intronic variants and multiple mutational mechanisms
              </a>
            </td>
          <td>
            A. Jacobson, Amal AbuRayyan, S. Gulsuner, Haley Slater, Yagiz Anasiz, Sirajummuneer M Ahmad, Ming K. Lee, Jessica B Mandell, Emily J Rettner, E. Konnick, Colin C Pritchard, M. King, T. Walsh, Brian H Shirts
          </td>
          <td>2025-06-26</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mechanisms for resistance to cytotoxic cancer drugs are dependent on dynamic changes in the biochemistry of cellular pathways, information which is hard to obtain at the systems level. Here we use a deep functional proteomics implementation of the Cellular Thermal Shift Assay to reveal a range of induced biochemical responses to gemcitabine in resistant and sensitive diffuse large B cell lymphoma cell lines. Initial responses in both, gemcitabine resistant and sensitive cells, reflect known targeted effects by gemcitabine on ribonucleotide reductase and DNA damage responses. However, later responses diverge dramatically where sensitive cells show induction of characteristic CETSA signals for early apoptosis, while resistant cells reveal biochemical modulations reflecting transition through a distinct DNA-damage signaling state, including opening of cell cycle checkpoints and induction of translesion DNA synthesis programs, allowing bypass of damaged DNA-adducts. The results also show the induction of a protein ensemble, labeled the Auxiliary DNA Damage Repair, likely supporting DNA replication at damaged sites that can be attenuated in resistant cells by an ATR inhibitor, thus re-establishing gemcitabine sensitivity and demonstrating ATR as a key signaling node of this response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3932b7cf1d620393ace8a6959b61b212c07c27aa" target='_blank'>
              MS CETSA deep functional proteomics uncovers DNA repair programs leading to gemcitabine resistance
              </a>
            </td>
          <td>
            Ying Yu Liang, Khalidah Khalid, Hai Van Le, Hui Min Vivian Teo, Mindaugas Raitelaitis, Marc-Antoine Gerault, Jane Jia Hui Lee, Jiawen Lyu, Allison Chan, A. D. Jeyasekharan, Wai Leong Tam, Pär Nordlund, N. Prabhu
          </td>
          <td>2025-05-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next‐generation sequencing. New techniques, such as single‐cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single‐cell and spatial analyses. These developments enable the investigation of cell–cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6992085eb1aec63c000f8c350d79dda9f01e87" target='_blank'>
              A computational strategy to uncover fusion genes in prostate cancer cell lines
              </a>
            </td>
          <td>
            Rebecca A. Morgan, Gary Hardiman
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study investigated whether therapeutic X-ray doses can affect the expression of mismatch repair (MMR) genes and proteins using Lynch syndrome-associated human colorectal cancer cell lines. MMR-deficient cell lines (HCT116, SW48, LoVo) and an MMR-proficient control cell line (HT29) were exposed to X rays [a 2 Gy dose or 2 Gy daily for five consecutive days (10 Gy)]. Reverse transcription quantitative real-time PCR (RT-qPCR) and Western blotting were used to detect the radiation-induced changes in the expression of RNAs and proteins, respectively. RT-qPCR revealed that MLH1 and MSH6 genes were stably expressed regardless of the MMR status of the cell line and the radiation dose. In contrast, the MSH2 gene was either up-regulated or down-regulated after 2 Gy or 10 Gy or both. The expression of PMS2 increased after 10 Gy irradiation in all MMR-deficient cell lines, even though the data were not statistically significant compared to other doses, except for the LoVo cell line. Protein expression analysed using Western blotting demonstrated that MLH1 protein expression was stable, whereas the expression of MSH2 was significantly affected by radiation exposure in both MLH1-deficient cell lines. No correlation between the expression of RNA and protein could be identified. In conclusion, radiation may have significantly differential effects on MMR RNA and protein expression when different cell lines, doses, and specific genes are considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b8835db8d3ae7872bbc2b8269596b668e92380" target='_blank'>
              The Effect of X rays on the Expression of Mismatch Repair Genes and Proteins in Lynch Syndrome Associated Human Colorectal Cancer Cell Lines.
              </a>
            </td>
          <td>
            Mingzhu Sun, L. Cruz-Garcia, Danny Freestone, Kevin Monahan, C. Badie, Yannick Comoglio, Hannah Mancey, J. Moquet, Stephen Barnard
          </td>
          <td>2025-06-20</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The integrity of the genome is maintained by mismatch repair (MMR) proteins that recognize and repair base mismatches and insertion/deletion errors generated during DNA replication and recombination. A defective MMR system results in genome-wide instability and the progressive accumulation of mutations. Tumors exhibiting deficient MMR (dMMR) and/or high levels of microsatellite instability (termed “microsatellite instability high”, or MSI-H) have been shown to possess fundamental differences in clinical, pathological, and molecular characteristics, distinguishing them from their “microsatellite stable” (MSS) counterparts. Molecularly, they are defined by a high mutational burden, genetic instability, and a distinctive immune profile. Their distinct genetic and immunological profiles have made dMMR/MSI-H tumors particularly amenable to treatment with immune checkpoint inhibitors (ICIs). The ongoing development of biomarker-driven therapies and the evaluation of novel combinations of immune-based therapies, with or without the use of conventional cytotoxic treatment regimens, continue to refine treatment strategies with the goals of maximizing therapeutic efficacy and survival outcomes in this distinct patient population. Moreover, the resultant knowledge of the mechanisms by which these features are suspected to render these tumors more responsive, overall, to immunotherapy may provide information regarding the potential optimization of this therapeutic approach in tumors with proficient MMR (pMMR)/MSS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0727d40a1aa873459400ba4134581be209441f5d" target='_blank'>
              Deficient Mismatch Repair and Microsatellite Instability in Solid Tumors
              </a>
            </td>
          <td>
            Joy A Awosika, James L. Gulley, Danielle M. Pastor
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla T. O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Alternative splicing (AS) is a pivotal post-transcriptional mechanism that expands the functional diversity of the proteome by enabling a single gene to generate multiple mRNA and protein isoforms. This process, which involves the differential inclusion or exclusion of exons and introns, is tightly regulated by splicing factors (SFs), such as serine/arginine-rich proteins (SRs), heterogeneous nuclear ribonucleoproteins (hnRNPs), and RNA-binding motif (RBM) proteins. These factors recognize specific sequences, including 5′ and 3′ splice sites and branch points, to ensure precise splicing. While AS is essential for normal cellular function, its dysregulation is increasingly implicated in cancer pathogenesis. Aberrant splicing can lead to the production of oncogenic isoforms that promote tumorigenesis, metastasis, and resistance to therapy. Furthermore, such abnormalities can cause the loss of tumor-suppressing activity, thereby contributing to cancer development. Importantly, abnormal AS events can generate neoantigens, which are presented on tumor cell surfaces via major histocompatibility complex (MHC) molecules, suggesting novel targets for cancer immunotherapy. Additionally, splice-switching oligonucleotides (SSOs) have shown promise as therapeutic agents because they modulate splicing patterns to restore normal gene function or induce tumor-suppressive isoforms. This review explores the mechanisms of AS dysregulation in cancer, its role in tumor progression, and its potential as a therapeutic target. We also discuss innovative technologies, such as high-throughput sequencing and computational approaches, that are revolutionizing the study of AS in cancer. Finally, we address the challenges and future prospects of targeting AS for personalized cancer therapies, emphasizing its potential in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2739ebba4691f4fc04db8de970276d260f0dc2" target='_blank'>
              Alternative Splicing in Tumorigenesis and Cancer Therapy
              </a>
            </td>
          <td>
            Huiping Chen, Jingqun Tang, Juanjuan Xiang
          </td>
          <td>2025-05-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, Reva K Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome-wide association studies have suggested numerous colorectal cancer (CRC) susceptibility genes, but their causality and therapeutic potential remain unclear. To prioritise causal associations between gene expression/splicing and CRC risk (52,775 cases; 45,940 controls), we perform a transcriptome-wide association study (TWAS) across six tissues with Mendelian randomisation and colocalisation, integrating sex- and anatomical subsite-specific analyses. Here we reveal 37 genes with robust causal links to CRC risk, ten of which have not previously been reported by TWAS. Most likely causal genes with evidence of cancer cell dependency show elevated expression linked to risk, suggesting therapeutic potential. Notably, SEMA4D, encoding a protein targeted by an investigational CRC therapy, emerges as a key risk gene. We also identify a female-specific association with CRC risk for CCM2 expression and subsite-specific associations, including LAMC1 with rectal cancer risk. These findings offer valuable insights into CRC molecular mechanisms and support promising therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b470fb91f2d0d137327a26167f01f560bda9b57e" target='_blank'>
              Multi-tissue expression and splicing data prioritise anatomical subsite- and sex-specific colorectal cancer susceptibility genes
              </a>
            </td>
          <td>
            E. Hazelwood, D. Canson, Benedita Deslandes, Xuemin Wang, P. F. Kho, D. Legge, Andrei-Emil Constantinescu, Matthew A Lee, D. T. Bishop, Andrew T. Chan, S. Gruber, J. Hampe, L. Le Marchand, Michael O Woods, Rish K. Pai, S. Schmit, J. C. Figueiredo, Wei Zheng, J. R. Huyghe, Neil Murphy, Marc J. Gunter, T. Richardson, V. L. Whitehall, E. E. Vincent, Dylan M Glubb, Tracy A. O’Mara
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="CRISPR has become a revolutionary tool for cancer biology and therapy with unprecedented precision for genome editing. Besides cancer, CRISPR is also under research for possible use in the treatment of viral diseases, such as Bovine Viral Diarrhea Virus (BVDV), a serious livestock problem. Through enabling gene modifications to be made to individual genes, CRISPR is used in identifying oncogenic drivers, characterizing tumor suppressor networks, and designing innovative therapeutics. This review focuses on the various uses of CRISPR for cancer studies with special attention to methods like gene knockout, gene activation, and base editing that each has a great potential for correcting oncogenic mutations and restoring tumor suppressor activity. Moreover, CRISPR's capacity to regulate viral replication, as that of BVDV, signifies its dual role in oncology as well as in viral pathogenesis. Interventions based on CRISPR such as chimeric antigen receptor (CAR) T cell therapy and synthetic lethality are essentially revolutionizing cancer therapy by improving immune responses and capitalizing on the specificity that is characteristic of cancer cells. However, off-target effects, tumor heterogeneity, and ethical dilemmas remain onerous challenges for clinical application. CRISPR delivery systems, despite playing a central role in advancing cancer therapies, also have prospects in optimizing the effectiveness of treatment against viral pathogens like BVDV. Advances in delivery systems using nanoparticles and viral vectors are mitigating against these challenges and improving efficacy and specificity of CRISPR reagent in vivo. In addition to this, continued progress in new technologies such as prime editing and base editing is predicted to improve precision and efficacy of CRISPR-based therapy. Since CRISPR technology is continuously developing, CRISPR's potential to treat both cancer and viral infections, like BVDV, simultaneously will be instrumental in precision medicine. This review highlights the revolutionary potential of CRISPR to revolutionize cancer treatment paradigms and brings hope for more effective and individualized therapy. With potential developments to emergent and potentiated CRISPR tools and reagents, it is predicted that they will play a center stage role in precision oncology and promote better patient results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d3748e9db2d0a3114958d899e7fef2d33921ff" target='_blank'>
              CRISPR in Cancer Biology and Bovine Viral Diarrhea Therapy: Precision Genome Editing at the Frontier of Oncology and Viral Pathogenesis
              </a>
            </td>
          <td>
            Jindong Gao, Mengdi Zhang, Linjia Gao, Lei Kuang, Tuniyazi. Maimaiti, Ling Zhao, Qinghua Shang, Changmin Hu
          </td>
          <td>2025-05-19</td>
          <td>Bio Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ead68569a8b2470e49064bdb4fedf8ee91a2841" target='_blank'>
              Differential PARP inhibitor responses in BRCA1-deficient and resistant cells in competitive co-culture
              </a>
            </td>
          <td>
            Shiella A Soetomo, Michael F Sharp, W. Crismani
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593951b80dada0fabedb9d7b8680116321df128" target='_blank'>
              Development of Paclitaxel Resistance in Triple-Negative Breast Cancer Is Associated with Extensive DNA Methylation Changes That Are Partially Reversed by Decitabine
              </a>
            </td>
          <td>
            Lenka Trnkova, M. Burikova, A. Soltysova, A. Ficek, Jana Plava, Andrea Cumova, Lucia Rojikova, Kristina Jakic, Eva Sedlackova, Boris Tichý, V. Bystrý, F. Busato, Yimin Shen, M. Matúšková, L. Kucerova, J. Tost, S. Miklikova, M. Cihova, V. Buocikova, B. Smolková
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract The genomes of all organisms encode diverse functional elements, including thousands of genes and essential noncoding regions for gene regulation and genome organization. Systematic perturbation of these elements is crucial to understanding their roles and how their disruption impacts cellular function. Genetic perturbation approaches, which disrupt gene expression or function, provide valuable insights by linking genetic changes to observable phenotypes. However, perturbing individual genomic elements one at a time is impractical. Genetic screens overcome this limitation by enabling the simultaneous perturbation of numerous genomic elements within a single experiment. Traditionally, these screens relied on simple, high-throughput readouts such as cell fitness, differentiation, or one-dimensional fluorescence. However, recent advancements have introduced powerful technologies that combine genetic screens with image-based and single-cell sequencing readouts, allowing researchers to study how perturbations affect complex cellular phenotypes on a genome-wide scale. These innovations, alongside the development of CRISPR–Cas technologies, have significantly enhanced the precision, efficiency, and scalability of genetic screening approaches. In this review, we discuss the genetic screens performed in kinetoplastid parasites to date, emphasizing their application to both coding and noncoding regions of the genome. Furthermore, we explore how integrating image-based and single-cell sequencing technologies with genetic screens holds the potential to deliver unprecedented insights into cellular function and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5825d146111335de4ea8d326bda5fc9057913a" target='_blank'>
              Next generation genetic screens in kinetoplastids
              </a>
            </td>
          <td>
            James Budzak, T. Siegel
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genetically identical bacterial cells often exhibit heterogeneous responses to antibiotics − some survive, others die. Here, we show that this heterogeneity propagates across generations to give rise to phenotypic resistance. Using real-time single-cell tracking, we exposed Escherichia coli to the β-lactam cefsulodin at its clinical breakpoint concentration and analyzed cell fate within genealogical trees statistically. Cell survival was strongly correlated among family members, driving the selective enrichment of robust lineages within an otherwise susceptible population. Our genealogical population model identified heritable phenotypic resistance as a key factor underlying this enrichment, which was validated experimentally. Comparing enrichment dynamics between the wild-type and a tolC knock-out strain, deficient in multidrug efflux, uncovered nuanced changes that increased the intergenerational memory of phenotypic resistance. Our findings provide evidence for heritable phenotypic resistance and demonstrate how its propagation through cell-to-cell heterogeneity enables the survival of minority cells within isogenic populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6715bbe58fe26a64981336caf37053a9a8306b82" target='_blank'>
              Lineage-dependent variations in single-cell antibiotic susceptibility reveal the selective inheritance of phenotypic resistance in bacteria
              </a>
            </td>
          <td>
            Wesley Stine, Tatsuya Akiyama, David S. Weiss, Minsu Kim
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The combination of CDK4/6 inhibitors (CDK4/6i) with endocrine therapy prolongs survival of patients with ER+/HER2- metastatic breast cancer. Nearly all patients with advanced disease treated with this combination eventually progress, highlighting a need for therapeutic interventions that prevent the emergence of drug resistance. Drug tolerant persister (DTP) cells represent a reservoir of surviving cells from which drug-resistant clones eventually emerge. We hypothesize that targeting the DTP state which emerges under selective pressure from CDK4/6i could prevent the emergence of drug resistance. In vitro treatment of MCF7 and T47D ER+/HER2- breast cancer cells with the CDK4/6i Palbociclib and estrogen deprivation for 4 weeks suppressed cell viability with 2-5% of cells surviving treatment and achieving a DTP state. Cell cycle analysis by PI staining revealed that 95% of the DTP cells were arrested in G0/G1 after 4 weeks of continuous therapy. Upon drug washout, the DTP population resumed proliferation within ∼1 week and regained sensitivity to retreatment with CDK4/6i, suggesting epigenetic reprogramming is responsible for the DTP state. Whole exome sequencing did not show enrichment of known genomic alterations associated with CDK4/6i resistance (e.g., RB1, FAT1, PTEN). Bulk RNA sequencing of MCF7 DTP cells revealed transcriptome reprogramming compared to untreated controls, which reverted upon drug washout to that of the treatment naïve state. While cell cycle-related gene signatures such as E2F targets and G2M checkpoint were downregulated, interferon response signatures were upregulated in MCF7 DTP cells relative to untreated controls. MCF7 persisters were characterized by enrichment in senescence and chemotherapy-induced stressed gene signatures, while diapause and MYC target signatures were downregulated. These same signatures were similarly enriched in ER+/HER2- tumors from patients in the NeoPalAna study treated with neoadjuvant palbociclib and the aromatase inhibitor anastrozole compared to baseline biopsies. To study evolution of drug resistance, we transduced MCF7 and T47D cells with the LARRY (Lineage and RNA recovery) barcode library to uniquely label and track each persister cells long-term, and identify programs that are causal to the emergence of drug resistant clones. We performed single cell RNA-seq on barcoded cells treated with palbociclib plus estrogen deprivation for 0 h, 72 h, and 4 weeks. Several clones that arrested in G0/G1 initially eventually expanded at 4 weeks, while other clones were eliminated. We are currently investigating which genes/pathways are upregulated in the barcoded clones enriched at the 4-week time point. We will then validate results from these single cell approach in cells surviving neoadjuvant treatment in the NeoPalAna trial. In summary, a small subset of ER+/HER2- breast cancer cells survive long-term treatment with CDK4/6i plus estrogen deprivation and exhibits reversible transcriptomic features of DTPs. Our ongoing single cell-seq analyses may reveal transcriptional programs that enable the DTP state which, in turn, could be targeted to reduce survival of drug-tolerant cells and prevent the emergence of drug resistance.
 Citation Format: Yuki Matsunaga, Hima Milan Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, Dan Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, Ariella B. Hanker. Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-30.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a0b4864d21ce42a1a87671ea1a257909c574fe" target='_blank'>
              Abstract P3-04-30: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, H. M. Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The DNA damage response comprises a set of imperfect pathways that maintain cell survival following exposure to DNA damaging agents. Cancers frequently exhibit DNA repair pathway alterations that contribute to their intrinsic genome instability. This, in part, facilitates a therapeutic window for many chemotherapeutic agents whose mechanisms of action often converge at the generation of a double-strand DNA break. The development of therapy resistance occurs through countless molecular mechanisms that promote tolerance to DNA damage, often by preventing break formation or increasing repair capacity. This review broadly discusses the DNA damaging mechanisms of action for different classes of chemotherapeutics, how avoidance and repair of double-strand breaks can promote resistance, and strategic directions for counteracting therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77a69284f3f1f8221d2572829a8005590b029476" target='_blank'>
              DNA repair and the contribution to chemotherapy resistance
              </a>
            </td>
          <td>
            K. Nešić, Phoebe Parker, Elizabeth M. Swisher, John J Krais
          </td>
          <td>2025-05-26</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Introduction: Survival rates for patients with breast cancer have improved significantly with widespread screening and effective systemic therapies. However, patients who present with advanced nodal stage at diagnosis have higher rates of distant relapse and require more aggressive therapies than patients with pathologic N1 disease (pN1). Understanding the molecular alterations that occur in primary breast cancer (BC) that drive progression to a higher nodal stage (>pN1) could uncover potential therapeutic targets for patients with regionally metastatic BC. Given that epigenetic mechanisms represent that crucial layer of dynamic change that regulates genome-wide gene expression, we assessed the DNA methylation profiles of primary BC at various nodal stages. Methods: We conducted an exploratory study using a well-annotated, retrospective cohort of ER+/HER2- treatment-naive primary BC from patients with clinically positive nodes and complete pathologic nodal data treated at Duke University (n=49, 30 cases pN1 vs. 19 cases >pN1). DNA was extracted from paraffin-embedded samples, and DNA methylation profiling was performed using the Illumina EPIC BeadChip v1 arrays. The ChAMP package was utilized to correct batch effect, and the Wilcoxon test was used to identify Differentially Methylated Sites (DMS). Tumor content purity was assessed using the LUMP algorithm and Gene Ontology enrichment of the DMSs was examined with the GOmeth tool. Data analysis and visualization were performed using the R packages ggplot2 (v. 3.4.4), circlize (v. 0.4.16), and gplots (v. 3.1.3.1), with data management facilitated by the tidyverse (v. 2.0.0) collection. Results: Tumor purity was similar between the two groups (p>0.05). The comparison revealed distinct DNA methylation patterns in 962 genomic regions between pN1 and >pN1 tumors (differential methylation > 15%, p<0.01); 683 regions were hypermethylated and 279 were hypomethylated in the >pN1 group. There was a significant enrichment of biological processes associated with morphogenesis among the hypermethylated sites, with 22 of the 43 affected molecular pathways linked to cellular morphogenesis. Additionally, epigenetic alterations were predominantly found in pathways related to the response to bone morphogenic protein (BMP). Expression-methylation Quantitative Trait Loci (emQTLs) were linked to epithelial to mesenchymal transition (EMT) genes in >pN1 tumors. Finally, higher methylation of the protocadherin cluster genes (PCDHGA) and regions encompassing DUSP6 and KRT7 genes were identified in tumors of higher nodal stage. These findings suggest inherent or acquired epigenetic alterations in primary BC associated with higher nodal stage. Conclusions: Understanding the mechanisms underlying lymph node invasion in breast cancer progression is crucial for improving the clinical management of patients with BC. This study indicates that ER+/HER2- primary tumors from patients with higher nodal involvement have an epigenetic predisposition to activate morphogenic differentiation and EMT more efficiently than patients with lower nodal involvement. Deeper insight into these alterations could contribute to more accurate prognostic tools, the development of novel therapeutic approaches, and potentially help avoid more aggressive axillary procedures.
 Citation Format: Sookyung Ahn, Andrés F. Bedoya-López, Miquel Ensenyat-Mendez, Pere Llinàs-Arias, Diego M. Marzese, Maggie L. DiNome. Epigenetic Alterations Affecting Cell Morphogenesis and Tumor Plasticity Pathways in Primary Tumors of Breast Cancer Patients with High Nodal Involvement [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-11-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de4a53a7cbc29ade67d8866b12ed7218a58382b" target='_blank'>
              Abstract P4-11-09: Epigenetic Alterations Affecting Cell Morphogenesis and Tumor Plasticity Pathways in Primary Tumors of Breast Cancer Patients with High Nodal Involvement
              </a>
            </td>
          <td>
            Sookyung Ahn, Andrés F. Bedoya-López, Miquel Ensenyat-Mendez, P. Llinàs-Arias, D. Marzese, Maggie L. DiNome
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is among the most frequently diagnosed cancer types, and affected patients frequently experience poor prognostic outcomes and high mortality rates. Many genomic studies of ESCC have been performed in recent years, yet the mutational mechanisms driving ESCC and their clinical implications remain incompletely understood. In this study, paired tumor and normal tissue samples from 22 patients with ESCC were used for whole genome sequencing-based analyses of genome-wide mutational events. These comprehensive analyses enabled the detection and characterization of various mutation subtypes in ESCC including somatic single-nucleotide variants, small insertions and deletions, copy number variations, structural variations, and circular extrachromosomal DNA. Of identified genes harboring non-silent mutations, TP53, NOTCH1, CSMD3, EP300, and FAM135B were the most frequently mutated genes in this study and they were annotated in the COSMIC Cancer Gene Census. With the exception of aging-related signatures, an APOBEC-associated mutational signature was the dominant mutational feature detected in ESCC samples, suggesting that APOBEC-mediated cytidine deamination is likely a major driver of mutations in this cancer type. Notably, our study also detected circular extrachromosomal DNA (ecDNA) events in these ESCC patient samples. The oncogenes COX6C, PVT1, and MMP12 as well as the oncogenic long non-coding RNA AZIN1-AS1 which were detected in ecDNA regions in these analyses may be associated with worse disease-free survival in ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55745d3238cc107d72d837356b7050823d1fc40" target='_blank'>
              Whole genome sequencing revealed esophageal squamous cell carcinoma related biomarkers
              </a>
            </td>
          <td>
            Mingjun Li, Lei Li, Xizi Wang, Yanwei Zhao, Peina Du, Wei Wang, , Yadong Wang, Yanxing Sheng, Mingliang Gu, Xiaodong Jia
          </td>
          <td>2025-06-26</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Leukemia, a group of malignant blood disorders, arises from the uncontrolled proliferation of abnormal white blood cells. Genetic mutations play a critical role in the initiation and progression of leukemia. This review aims to provide an overview of the genetic landscape of leukemia, focusing on the most common genetic alterations and their clinical implications. A literature search was conducted to identify relevant studies on genetic mutations in leukemia. The identified studies were critically appraised to assess their methodological quality. The present review highlights the key genetic alterations associated with different types of leukemia, including chromosomal translocations, point mutations, and gene copy number variations. These genetic abnormalities can impact disease prognosis, treatment response, and overall patient survival. A comprehensive understanding of the genetic basis of leukemia is essential for accurate diagnosis, prognostication, and the development of targeted therapies. Future research should focus on identifying novel genetic markers, elucidating the underlying mechanisms of leukemogenesis, and developing innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e966f7c7f21cf8c9b127f6afb40f6c702d2c3" target='_blank'>
              A Review Article: Evaluation of the Frequency of Genetic Mutations in Leukemia.
              </a>
            </td>
          <td>
            A. Hamadi
          </td>
          <td>2025-06-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Homologous recombination deficiency (HRD) is associated with increased sensitivity to PARP inhibitors (PARPi), which offer improved progression-free survival (PFS) in HRD-positive patients compared to those with homologous recombination proficiency (HRP). Access to PARPi therapy requires confirmation of HRD status, typically determined through tumor DNA sequencing to identify BRCA1/2 mutations or specific patterns of genomic instability indicative of HRD. Several diagnostic assays are available to assess HRD, employing different approaches—either through targeted gene panels or genome-wide analysis. This review focuses on the biological basis and consequences of HRD, the clinical use of PARPi, and the currently available diagnostic tools for evaluating HRD status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ec3928c154d81d683dd34c0a4a05380851b15b" target='_blank'>
              Unraveling Homologous Recombination Deficiency in Ovarian Cancer: A Review of Currently Available Testing Platforms
              </a>
            </td>
          <td>
            Nicola Marconato, Orazio De Tommasi, Dino Paladin, Diego Boscarino, G. Spagnol, Carlo Saccardi, Tiziano Maggino, Roberto Tozzi, Marco Noventa, M. Marchetti
          </td>
          <td>2025-05-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mitochondria and mitochondrial DNA (mtDNA) are crucial for cellular energy metabolism and the adaptive response to environmental changes. mtDNA collaborates with the nuclear genome to regulate mitochondrial function. Dysfunctional mitochondria and mutations in mtDNA are implicated in a wide range of diseases, including mitochondrial disorders, neurodegenerative conditions, age-associated pathologies and cancer. While the nuclear genome has been extensively studied for its role in driving the clonal expansion of oncogenes and other aging-related genetic alterations, knowledge regarding mtDNA remains comparatively limited. However, advances in quantitative analysis have provided information regarding the complex patterns of mtDNA mutations. The present review offers a detailed examination of mtDNA mutations and their classifications in the contexts of aging and cancer, and elucidates the role of mtDNA mutations in these processes. Mutations in mtDNA can be detected as early as the neonatal stage, yet most transition mutations retain a normal cellular phenotype. In contrast to mutations in oncogenes and tumor suppressor genes within the nuclear genome, mtDNA exhibits conserved mutational signatures, irrespective of cancer tissue origin. To adapt to the aging process, mitochondria undergo clonal expansion of advantageous mtDNA mutations, maintaining a dynamic equilibrium among various mitochondrial clones. Over time, however, the loss of strand bias can disrupt this equilibrium, diminishing the pool of adaptive clones. This breakdown in mitochondrial homeostasis may contribute to tumorigenesis. In conclusion, the heterogeneity of mtDNA mutations and the collapse of its homeostasis are pivotal in the progression of age-related diseases, including cancer, underscoring the importance of mtDNA mutations in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f62f8f9a03fb69f861e5c5e08d5ee028d72c9e9" target='_blank'>
              Understanding the impact of mitochondrial DNA mutations on aging and carcinogenesis (Review)
              </a>
            </td>
          <td>
            Hiroshi Kobayashi, Shogo Imanaka
          </td>
          <td>2025-06-03</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer is a complex genetic disease driven by somatic and germline mutations that alter critical cellular functions. With the advancement of Next-Generation Sequencing (NGS), researchers can now identify these mutations at high throughput; however, interpreting their biological and clinical relevance requires robust functional annotation. This study investigates the mutational landscape of prostate, breast, and pancreatic cancers using public RNA-seq datasets and an open-source bioinformatics pipeline. Key oncogenic drivers and tumor suppressor genes, including BRCA1, BRCA2, TP53, and KRAS, were analyzed using tools such as FastQC, Bowtie2, Samtools, FreeBayes, and SnpEff. Our findings highlight the predominance of missense mutations and frequent transition substitutions such as C→T and G→A, with silent and nonsense mutations also contributing to disease mechanisms. These insights emphasize the utility of integrative computational tools in variant annotation and their potential to enhance cancer diagnostics, prognostics, and targeted therapy selection in precision oncology">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0c74326956f2ce86b7c2f753a11ac5532d41d2" target='_blank'>
              Functional Annotation of Cancer Variants: Tools And Techniques
              </a>
            </td>
          <td>
            Alka Manisha, Jyoti Prakash, Ruchi yadav, Rachna Chaturvedi, Akanksha Pandey
          </td>
          <td>2025-05-14</td>
          <td>International Journal of Advanced Research in Science, Communication and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Homologous recombination deficiency (HRD) is associated with sensitivity to platinum-based chemotherapy and PARP inhibitors in BRCA-associated cancers, including ovarian, breast, prostate, and pancreatic cancers. This study explores HRD and homologous recombination repair (HRR) gene alterations in a pan-cancer cohort to guide precision oncology. Methods Clinical and genomic data from 9,262 patients with 17 solid tumor types were analyzed using the OncoScreenTM Plus kit. HRD scores, biallelic HRR and tumor suppressor gene alterations, and their clinical correlations were evaluated. Results HRD scores varied across cancer types, all showing a long tail in distribution. The prevalence of pathogenic alterations in pan-cancer HRR was 21.3%, with 13.7% of the cases having an HRD score ≥42. HRD-related events (LOH, LST, and TAI) exhibited similarities and cancer-specific patterns at the chromosomal arm level. Biallelic loss of HRR genes, especially BRCA1, BRCA2, RAD51D, RAD51 C, and PPP2R2 A was linked to higher HRD scores in BRCA-associated cancers, while BARD1, RAD51D, RAD54L, BRCA1, and MRE11 were associated with elevated HRD scores in in other cancer types (non-BRCA cancers). TP53 biallelic alterations, with or without HRR alterations, were linked to increased HRD scores. Higher HRD scores were associated with late-stage, older, metastatic, PD-L1 positive, non-MSI-H/non-POLE samples were correlated with genomic instability indexes, such as structural chromosomal instability (SCIN), weighted genome instability index (WGII), and whole-genome doubling (WGD). Conclusions This is the largest pan-cancer HRD study in an Asian population, providing insights for future HRD testing and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14267-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e5299d0b32b34fcf922057e19238ea118c6064" target='_blank'>
              Pan-cancer analysis of homologous recombination deficiency and homologous recombination repair–associated gene alterations in solid tumors from a large Asian cohort
              </a>
            </td>
          <td>
            Lili Ren, Runsi Yao, Ting Hou, Chenglin Liu, Fei Zhao, Xiaojun Chen, Zhou Zhang, Yan Huang
          </td>
          <td>2025-05-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Serine metabolism is of growing biologic and therapeutic interest in cancer. Upregulation of the serine synthesis pathway (SSP) can fuel tumor growth, and cancers with this phenotype are often sensitive to SSP inhibitors. In parallel, dietary restriction of serine and glycine (SG) can suppress some cancers, but the determinants of sensitivity to this approach are poorly understood. This is especially true in acute myeloid leukemia (AML), where serine metabolism has been less explored. We report that a subset of human AML cell lines and primary samples are completely dependent on external serine, known as serine auxotrophy. These leukemias consistently suppressed the SSP enzyme PSAT1, failed to synthesize serine, responded to SG restriction in vivo, and were rescued by restoring PSAT1. We also found that AML with an SF3B1 K700E mutation showed additional dependence on the SSP enzyme PHGDH, that SG restriction synergized with venetoclax in serine auxotrophic AML, and that MECOM rearrangement was strongly associated with PSAT1 suppression and serine auxotrophy. These findings define a metabolically distinct AML subtype and nominate it for targeting by SG restriction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c5b66553b8e0f26c1d2319a4806fb04112b3ce" target='_blank'>
              Serine auxotrophy is a targetable vulnerability driven by PSAT1 suppression in AML
              </a>
            </td>
          <td>
            Ilias Sinanidis, Panagiotis Tsakiroglou, Benjamin G. Dubner, Rebecca Foertsch, Julia Gondek, InYoung Choi, Bogdan Paun, Liang Zhao, Gabriel Ghiaur, W. Dalton
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 4116


 Background:
 Biliary tract cancers (BTC) are rare, highly aggressive malignancies with limited treatment options, leading to consistently poor outcomes. An improved understanding of BTC tumor evolution could inform enhanced screening strategies, identify useful prognostic markers, and discover novel therapeutic targets.
 Methods:
 We performed whole genome and transcriptome sequencing (WGTS) at high depth ( > 80X) on a prospective cohort of BTC tumors. After detecting mutations and mutational signatures, we developed two novel methods for driver identification and evolutionary reconstruction. First, we created new oncogene and tumor suppressor-specific models that integrate copy number profiles, structural variant breakends, and expression changes to distinguish structural drivers from neutral chromosomal rearrangements. Second, we applied population genetics techniques to fit demographic models to tumor allele frequencies across 20 paired primary and metastatic samples.
 Results:
 We analyzed 130 tumor samples from 110 patients, representing the largest BTC whole-genome cohort to date. We identified hypermutated tumors (50-150 mutations/mb) with distinct etiologies, including mismatch repair deficiency, platinum exposure, tobacco use, and aristolochic acid-related damage, the latter of which was associated with response to immunotherapy. Ourintegrated-driver approach identified an association between selection on
 RAD23A
 and an increased structural variant load, resulting in a tandem-duplicator-like mutational phenotype. This also revealed an underappreciated impact of
 SMAD4
 in BTC, which is inactivated through multiple mutation types in 12%. Notably,
 BAP1
 mutations occurred in 24% of cases, including 4% that were inactivated through deletion of the
 BAP1
 promoter that lowered transcript expression, a previously undescribed mechanism. By pairing clinical data to the genomics, we observed a co-occurrence of
 BAP1
 mutations and
 FGFR2
 fusions in small-duct, mass-forming intrahepatic cholangiocarcinomas. These mutations were mutually exclusive with
 TP53
 mutations, which were enriched in patients with primary sclerosing cholangitis. Finally, our novel method for subclonal population reconstruction on paired samples illuminated recent tumor evolutionary dynamics and identified an
 ARID1B
 fusion with a potential role in metastasis.
 Conclusions:
 This study uncovered novel genomic mechanisms underlying the evolutionary origins of BTC beyond those previously identified with exome and panel sequencing, highlighting the value of WGTS. These results highlight the complex genomic heterogeneity of BTC, with potential implications for precision therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bbdd5f85cf47a722dc40e8dbc7f432d73cb4b4" target='_blank'>
              Whole-genome sequencing of biliary tract cancer: Uncovering the genomic origins of evolutionary trajectories.
              </a>
            </td>
          <td>
            Felix Beaudry, Nicholas Light, Duhan Yendi, A. Zorigtbaatar, Maggie Hildebrand, A. Dodd, Roxana Bucur, Julie M. Wilson, Enrique Sanz Garcia, Xin Wang, Erica S. Tsang, G. O’Kane, Steven Gallinger, Arndt Vogel, Jennifer J Knox, F. Notta, G. Sapisochin, Robert C Grant
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Introduction and Objective: Genetic variation and its interaction with environmental cues are crucial in the etiology of metabolic diseases, which are strongly linked to adipocyte dysfunction and are highly cell state- and context-specific.
 Methods: Here, we leverage a population-scale biobank (CellGenBank) to conduct pooled natural genetic variation screens in primary human adipocyte villages for single-nucleus transcriptomic and chromatin accessibility profiling under various disease-relevant stimuli.
 Results: Processing 338k nuclei from 118 donors allowed us to identified key cell states characterized by canonical marker genes, including quiescent (PDGFRA) and proliferative (PDGFRA, CDK1, AURKB) adipose tissue-derived mesenchymal stem cells (AMSCs), structural Wnt-regulated adipose tissue-resident (SWAT; DCN, PLAC9, APOD) cells and adipogenic (ADIPOQ, PLIN1) cells, with distinct transcriptional responses to stimuli. By leveraging individual polygenic risk scores, we identified correlations between disease risk and shifts in cell state proportions. These states have been also mapped to metabolic disease relevant traits using single-cell heritability analysis, with body mass index highly enriched in AMSCs, whereas traits informative of metabolic health show strong enrichment in adipogenic cells and a subpopulation of SWAT cells. Furthermore, genome-wide eQTL mapping revealed hundreds of context- and state-specific eQTLs that were enriched in predicted gene-enhancer/promoter regions, linking these variants to their functional transcripts and chromatin accessibility.
 Conclusion: Collectively, the human adipocyte village approach coupled with single-nucleus functional genomics enabled the discovery of genetic mechanisms underlying metabolic diseases, highlighting its high potential as a tool in the development of effective therapeutic targets. (*Yi Huang and Joaquin Perez-Schindler contribute to this work equally)


 Y. Huang: None. J. Perez-Schindler: None. S. Datta: None. B. Min: None. N. Nambrath: None. M. Murali: None. H. Dashti: None. B. Sharma: None. S. SinghPoma: None. P. Kubitz: None. W. Qiu: None. R. Andersson: None. T. Jones: None. M. Claussnitzer: None.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1a482f8b4cba32761087321e7d04e821730314" target='_blank'>
              2126-LB: Mapping Functional Genetic Variants and Disease Heritability on Gene Regulatory Programs in Human Adipocyte Villages under Metabolic Disease–Relevant Stimuli
              </a>
            </td>
          <td>
            YI Huang, Joaquín Pérez-Schindler, Sanchari Datta, Bellis Min, Nivedita Nambrath, Mayank Murali, Hesam Dashti, Bandana Sharma, Sunita Singhpoma, Phil Kubitz, Wei-Lin Qiu, Robin Andersson, Thouis Jones, Melina Claussnitzer
          </td>
          <td>2025-06-13</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lanxin Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 e22507


 Background:
 Currently,Breast Cancer (BC) is not considered a mismatch repair (MMR)-deficiency (MMRd)–spectrum tumor. The BC patients carrying constitutional MMR pathogenic variants (PVs) are not eligible for dedicated screening programs and risk-reducing strategies, which are addressed only to the BC individual carriers of specific Homologous Recombination Repair (HRR) variants. The MMRd status, as well as the high levels of microsatellite instability (MSI-H) that occur upon loss of the expression of MMR genes, may have direct therapeutic implications for the patients, who may benefit from the treatment with immune-checkpoint inhibitors (ICIs).
 Methods:
 This was an observational, hospital-based, cohort study to investigate the prevalence of germline MMR PVs in BC patients who were appropriate candidates for hereditary breast and/or ovarian cancer (HBOC) predisposition evaluation. BC patients were tested through NGS-based multigene panel testing including MMR-associated genes, according to clinical established criteria.
 Results:
 Between January 2016 and December 2024, 6.020 BC patients were referred for genetic counseling and HBOC predisposition evaluation. In the cohort of consecutive 5.595 BC patients tested with multigene panel testing, a subgroup of 26 patients resulted carriers of germline MMR PVs. Interestingly, 6 of 26 MMR-mutated patients (23.1%) had also a personal bilateral BC history. The median age at BC diagnosis was 55 (range 31-82). Mutation prevalence showed that
 PMS2
 was the most mutated MMR gene (10 PVs, 38.5%), followed by
 MLH1
 (6 PVs, 23.1%),
 EPCAM
 (5 PVs, 19.2%),
 MSH2
 (3 PVs, 11.5%), and
 MSH6
 (2 PVs, 7.7%). Notably, in the MMR
 -
 mutated cohort, 73.1% of BC patients (19/26) reported a family history of HRR-associated tumors (BC, ovarian/prostate and/or pancreatic tumors), predominantly BC (78.9%). Conversely, first or second-degree family members with a colorectal or endometrial cancer diagnosis, that represents the main Lynch Syndrome-associate tumors, were reported only for 3 patients (11.5%). The genetic characteristics and the clinical behavior of BC in MMR-,
 BRCA
 - and HRR non-
 BRCA
 -carriers were also collected. The variant of uncertain significance (VUS) prevalence was described.
 Conclusions:
 Our results suggested that, in women who met hereditary BC testing criteria but missed the established Lynch Syndrome clinical or testing criteria, the use of panel testing including MMR genes could lead to the identification of a subgroup of BC patients who are carriers of MMR constitutional PVs, with potential important preventive and therapeutic implications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea6cd0e8c7a31e9b9298c77292b06af1383e9ba" target='_blank'>
              Germline evaluation of mismatch repair (MMR) genes in breast cancer patients missing Lynch syndrome criteria: Clinical behavior, preventive paths and therapeutic implications.
              </a>
            </td>
          <td>
            L. Incorvaia, L. Cortesi, T. D. Bazan Russo, Clarissa Mujacic, Emma Zattarin, A. Perez, Elena Barbieri, Maria Concetta Vitale, I. Marchi, V. Gristina, L. Insalaco, A. Carreca, C. Brando, N. Barraco, A. Galvano, D. Fanale, G. Badalamenti, V. Bazan, I. Carreca, A. Russo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Hypomethylating agents are frontline therapies for myelodysplastic neoplasms (MDS), yet clinical responses remain unpredictable. We conducted a phase 2 trial comparing injectable and oral azacitidine (AZA) administered over one or three weeks per four-week cycle, with the primary objective of investigating whether response is linked to in vivo drug incorporation or DNA hypomethylation. Our findings show that injection results in higher drug incorporation, but lower DNA demethylation per cycle, while global DNA methylation levels in mononuclear cells are comparable between responders and non-responders. However, hematopoietic stem and progenitor cells (HSPCs) from responders exhibit distinct baseline and early treatment-induced CpG methylation changes at regulatory regions linked to tissue patterning, cell migration, and myeloid differentiation. By cycle six—when clinical responses typically emerge—further differential hypomethylation in responder HSPCs suggests marrow adaptation as a driver of improved hematopoiesis. These findings indicate that intrinsic baseline and early drug-induced epigenetic differences in HSPCs may underlie the variable clinical response to AZA in MDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddff8b2603da5617fe3335cc02a0fb84c36a91b1" target='_blank'>
              Clinical response to azacitidine in MDS is associated with distinct DNA methylation changes in HSPCs
              </a>
            </td>
          <td>
            J. Thoms, Feng Yan, Henry R Hampton, S. Davidson, Swapna Joshi, J. Saw, C. Sarowar, Xin Ying Lim, Andrea C. Nunez, P. Kakadia, G. S. Bhuyan, Xiaoheng Zou, Mary Nguyen, E. Ghodousi, Forrest C Koch, Fatemeh Vafaee, I. R. Thompson, Mohammad M Karimi, Russell Pickford, Mark J. Raftery, Sally Hough, Griselda Buckland, Michelle Bailey, Yuvaraj Ghodke, N. Absar, Lachlin S. Vaughan, Leonardo Pasalic, Chun Y Fong, M. Kenealy, D. Hiwase, Rohanna Stoddart, S. Mohammed, Linda Lee, F. Passam, S. Larsen, Kevin J Spring, Kristen K Skarratt, Patricia Rebeiro, P. Presgrave, William S. Stevenson, Silvia Ling, Campbell Tiley, Stephen J Fuller, F. Roncolato, A. Enjeti, D. Hoenemann, C. Lemech, Chris J. Jolly, Stefan K. Bohlander, David J Curtis, J. W. H. Wong, Ashwin Unnikrishnan, M. Hertzberg, Jake Olivier, Mark Polizzotto, J. Pimanda
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="OBJECTIVE
To identify exonic markers that could improve analytic performance characteristics of next-generation sequencing (NGS) in detecting mismatch repair deficiency (dMMR) using colorectal cancer (CRC) as a model.


METHODS
Coding sequences of a target NGS panel (~1.13 megabase) were compared between dMMR CRC and mismatch repair-proficient (pMMR) CRC in a training cohort (41 dMMR CRCs and 213 pMMR CRCs) and a validation cohort (33 dMMR CRCs and 307 pMMR CRCs) with documented mismatch repair status by immunohistochemical and/or microsatellite instability assays.


RESULTS
The dMMR CRC cases showed significantly higher insertion/deletion (indel) mutations within exonic homopolymers (homo-indels), occurring predominantly within longer repeats of 5 to 10 nucleotides (92%, P < .0001), rather than shorter repeats of 2 to 4 nucleotides seen in pMMR CRC (62%). Homo-indels in dMMR CRC were not random. Hotspot loci were consistent between the training and validation cohorts. The dMMR defined by indels within homopolymers of 5 or more nucleotides, homopolymers of 7 or more nucleotides, or a panel of hotspots all showed 100% sensitivity and specificity with a range of cutoffs.


CONCLUSIONS
We propose that this approach allows one to identify highly sensitive and specific markers for detecting dMMR CRC by NGS alone. Further studies are warranted to test whether these markers are applicable to non-CRC neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26edad68bf6dd00a6af6b5832569dd252b00f30f" target='_blank'>
              Highly sensitive and specific markers for detection of mismatch repair deficiency by next-generation sequencing.
              </a>
            </td>
          <td>
            Ming-Tseh Lin, Eric S Christenson, Aparna Pallavajjala, James R. Eshleman
          </td>
          <td>2025-05-03</td>
          <td>American journal of clinical pathology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="“Clustered Regularly Interspaced Short Palindromic Repeats” (CRISPR) genome editing technology has revolutionized molecular biology and therapeutic development since its adaptation from a bacterial immune defense mechanism. This technological innovation has opened new avenues for treating previously intractable diseases, from correcting monogenic disorders like sickle cell disease to engineering immune cells for cancer immunotherapy and targeting persistent viral infections. The objective of this review was to study the role of CRISPR-based gene and cell therapy in the treatment of cancer, genetic and infectious disorders and its implementations to improve the treatment efficacy. A comprehensive search was carried out using data bases such as Cochrane, PubMed and Google Scholar which identified relevant studies published between 2014 and 2024. After screening and applying inclusion-exclusion criteria,18 articles were selected. CRISPR/Cas9, a genome editing tool from bacterial immune systems, shows promise for treating cancer, genetic and infectious diseases. It holds potential for enhancing T-cell therapies and fighting viruses like HPV, hepatitis B and C, and EBV. Challenges include cell toxicity, immune responses, off-target impacts and delivery methods. CRISPR/Cas9 is transforming the way genome is edited and is critical in the progress of cancer research, therapy, and management of infectious diseases. Novel applications include universal CAR T-cells precision gene correction and the elimination of HIV reservoirs. While it holds great promise, many issues still need to be addressed, such as off-target effects and how therapeutic genes are delivered into cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/970922bc423d4174e5d8d893fe2eae1a0e4ca7b3" target='_blank'>
              CRISPR-based gene and cell therapy designs to improve treatment efficacy and safety for patients with genetic disorders, infectious diseases and cancer
              </a>
            </td>
          <td>
            Dr.Sumaira Aziz
          </td>
          <td>2025-06-16</td>
          <td>Journal of Rehman Medical Institute</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
Microsatellite instability (MSI) arises from mismatch repair-deficiency (MMR-d) and is a predictor of immune checkpoint blockade (ICB) therapy response. Although MSI diagnostics typically yield a binary classification (MSI or microsatellite stable), the molecular phenotype likely represents a continuum. We explored whether MSI follows a quantitative spectrum, reflecting the extent and severity of microsatellite alterations, and whether it holds clinical significance.


METHODS
To quantitatively assess MSI, we evaluated the length of MSI peak shifts from polymerase chain reaction-based fragment length data in two cohorts (combined N = 237) of hereditary (Lynch syndrome) and sporadic MMR-d colorectal carcinomas (CRCs). We examined whether MSI peak shift lengths in diagnostic markers BAT25, CAT25, BAT26, and BAT40 correlate with specific MMR defects, histologic and clinical features, and coding microsatellite (cMS) mutations, potentially reflecting antigen load.


RESULTS
MSI peak shift lengths varied among MMR-d CRCs and were influenced by the specific underlying MMR defect, such as germline mutations in MLH1, MSH2, MSH6, or PMS2, or somatic MLH1 promotor hypermethylation. Hereditary MSH6-deficient CRCs exhibited shorter peak shifts compared with hereditary MLH1-deficient and MSH2-deficient CRCs. Longer MSI peak shifts were associated with larger tumor sizes (odds ratio of 1.3 for every 1-cm increase in tumor diameter) and number of cMS mutations per tumor (P = 8.73e-09).


CONCLUSION
Our findings demonstrate that (1) MSI peak shift lengths differ among MMR-d CRCs, (2) this variation is influenced by the specific MMR defect, and (3) this variation correlates with tumor size and number of cMS mutations. These findings suggest that a quantitative MSI classification could enhance clinical utility, with potential, for instance, in predicting disease progression and ICB therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/965661cf50df85d07e004e8b46ed690b5f15e144" target='_blank'>
              Microsatellite Peak Shifts in Polymerase Chain Reaction-Based Fragment Length Data Correlate With Microsatellite Instability Degree and Vary With Mismatch Repair Gene Defects and Tumor Size.
              </a>
            </td>
          <td>
            N. C. Helderman, Saskia Haupt, Fabian Strobel, Lena Bohaumilitzky, T. van Wezel, H. Morreau, Magnus von Knebel Doeberitz, Vincent Heuveline, Maartje Nielsen, A. Ahadova, Matthias Kloor
          </td>
          <td>2025-06-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The circadian clock governs daily rhythms in numerous physiological processes through precise regulation of gene expression and biochemical functions. Dysregulation of the circadian rhythm has been implicated in carcinogenesis and cancer progression. However, the mechanisms by which the circadian clock influences cancer phenotype and chemotherapy resistance, particularly in cholangiocarcinoma (CCA), remain poorly understood. Using cell lines established from primary CCA and metastatic ascites of two male patients, we manipulated core clock genes (BMAL1, PER2, and NR1D1) to evaluate their effects on circadian rhythms. We analyzed alterations in circadian phenotypes at dynamic and single time points and assessed their impact on cancer-related phenotypic changes, including proliferation, apoptosis, cell cycle regulation, migration, invasion, and the expression of epithelial-to-mesenchymal transition (EMT) and cancer stem cell markers. Additionally, we examined the impact of circadian disruption on gemcitabine sensitivity. Genetic deletion of BMAL1, PER2, and NR1D1 disrupted circadian rhythm and significantly altered cancer phenotypes. Notably, BMAL1 and NR1D1 impairment exacerbated cell migration, invasion, and EMT activation in CCA cells. BMAL1 loss also induced gemcitabine resistance. In contrast, PER2 repression enhanced chemosensitivity and inhibited metastasis. The modulation of the circadian gene triggered phenotypic changes in CCA cells, indicating a crucial involvement of core-clock components in the pathological mechanisms hastening bile duct cancer malignancy. Our findings advance the understanding of regulating CCA malignancy and may offer a novel target for its treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63df1b948313a08b9d2b74bdfc742ccab84a19a9" target='_blank'>
              Dysfunction of core clock genes regulates malignant phenotype and gemcitabine sensitivity of cholangiocarcinoma cells.
              </a>
            </td>
          <td>
            Yin Li, Aimin Zheng, Yangang Cui, Tianbao Liu
          </td>
          <td>2025-06-17</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c58caf03a35382e82dc43ac8f2167d8ee1ee67" target='_blank'>
              X-chromosome upregulation operates on a gene-by-gene basis at RNA and protein levels
              </a>
            </td>
          <td>
            Ryan N. Allsop, Jeffrey Boeren, , Sarra Merzouk, Suresh Poovanthingal, W. V. van Ijcken, Jeroen Demmers, Hegias Mira Bontenbal, C. Gontan, J. Gribnau, V. Pasque
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Human papillomavirus (HPV) is a key driver of head and neck squamous cell carcinoma (HNSCC) and cervical squamous cell carcinoma (CESC). Yet, these cancers exhibit distinct molecular and clinical features influenced by HPV status. This study utilizes RNA sequencing data from The Cancer Genome Atlas (TCGA). It employs bioinformatics tools, including DESeq2 for differential gene expression, CIBERSORT for immune profiling, and Kaplan–Meier survival analysis to investigate these differences. Differential expression analysis revealed distinct molecular signatures, with HPV-positive tumors enriched in immune-related pathways such as cytokine-cytokine receptor interactions. In contrast, HPV-negative tumors exhibited upregulation of metabolic pathways, including PPAR signaling. Metaflux analysis further demonstrated contrasting metabolic profiles: HPV-positive tumors showed increased glycolysis and oxidative stress regulation, whereas HPV-negative tumors were characterized by elevated amino acid and nucleotide metabolism. Immune profiling highlighted more significant CD8 + T-cell infiltration in HPV-positive tumors, while HPV-negative tumors were predominantly associated with macrophages, suggesting differing tumor immune environments. Survival analysis identified CXCL11 and STAT1 as potential prognostic biomarkers, with lower expression correlating with poorer survival in both cancers. These findings provide an integrated perspective on the molecular, metabolic, and immune differences associated with HPV status, offering insights into potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5da159097a813e24c0a076a336e809e56b48db1" target='_blank'>
              Human papilloma virus infection drives unique metabolic and immune profiles in head and neck and cervical cancers: implications for targeted therapies and prognostic markers
              </a>
            </td>
          <td>
            Saira Hamid, Mohd Shahnawaz Khan, Meraj Alam Khan, Naoshad Muhammad, Mayank Singh, AMMIRA-SARAH AL-SHABEEB AKIL, Ajaz A. Bhat, M. Macha
          </td>
          <td>2025-05-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a biologically heterogeneous malignancy of clonal plasma cells, often progressing from MGUS or smoldering MM. It causes anemia, bone lesions, and immune dysfunction due to abnormal plasma cell expansion in the bone marrow. Neuroinflammatory and neurotrophic factors may influence MM progression by affecting immune cells and the bone marrow niche. Growing evidence points to a role for neuroimmune regulation in tumor immunity. Despite therapeutic progress, disease heterogeneity and resistance highlight the need for new strategies targeting the tumor microenvironment and neuroimmune axis. Methods This investigation exploited single-cell RNA sequencing (scRNA-seq) to analyze MM and high-risk smoldering multiple myeloma (SMMh) samples, identifying 11 distinct cell types. We examined their transcriptional signatures, stemness, proliferative properties, and metabolic pathways, with particular attention to neuroimmune interactions in the tumor microenvironment. Using trajectory inference tools such as CytoTRACE, Monocle2, and Slingshot, we traced the differentiation paths of MM cell subpopulations and identified key signaling pathways that may influence immune responses and tumor progression. Results The analysis identified four distinct subpopulations of myeloma cells, with the C0 IGLC3+ myeloma cells representing the least differentiated and most proliferative subset. These cells played a critical role in MM progression and may contribute to immune evasion mechanisms. Additionally, receptor-ligand interactions within the tumor microenvironment were identified, which may be influenced by neuroinflammatory and neurotrophic factors. These findings suggest that the nervous system and immune modulation significantly affect tumor biology, highlighting potential therapeutic targets that could be exploited to overcome resistance to conventional therapies. Conclusion This single-cell analysis provided new insights into the cellular diversity and differentiation trajectories in MM, offering a deeper understanding of the complex neuroimmune interactions that drive tumor progression and resistance. By incorporating the role of neuroinflammation and immune modulation, our study suggested novel therapeutic strategies targeting the neuroimmune axis in oncology, ultimately contributing to the development of more effective, personalized treatment approaches for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb42682593bbacba35775b457e4d73a57648a9" target='_blank'>
              Decoding multiple myeloma: single-cell insights into tumor heterogeneity, immune dynamics, and disease progression
              </a>
            </td>
          <td>
            Zhenzhen Zhao, Zhijie Zhao, Zhiheng Lin, Lu Fan, Zhikai Xiahou, Yujiang Dong, Weiying Bao
          </td>
          <td>2025-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>3</td>
          <td>9</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d7a0ecc45c91a460045023891d09c9a49fcad6" target='_blank'>
              Identification and Validation of a Previously Missed Mutational Signature in Colorectal Cancer
              </a>
            </td>
          <td>
            Mariya Kazachkova, Burçak Otlu, M. Díaz-Gay, Ammal Abbasi, Sarah Moody, Sandra Perdomo, David C Wedge, Paul Brennan, Michael R Stratton, Ludmil B. Alexandrov
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ee072bd4b29ca666b639b84ed04de31adafdbe" target='_blank'>
              Discovery and validation of non-canonical antigens for Hepatocellular Carcinoma immunotherapy
              </a>
            </td>
          <td>
            Stephen Li, Yujuan Dong, J. Liu, Shanglin Li, Dandan Pu, Yue Guo, C. Wong, Nathalie Wong, Jason Wing, Hon Wong
          </td>
          <td>2025-06-09</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b9cacdce8242c566ddaec1bdaf52f90bb5d4150" target='_blank'>
              Transcriptome-Driven Constraint-Based Modelling Reveals Metabolic Targets for Ovarian Cancer
              </a>
            </td>
          <td>
            Kate E. Meeson, J. McGrail, Jean-Marc Schwartz, Stephen S. Taylor
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05df4b99ec2d4ca5afa53e36017d7bbb5db0a668" target='_blank'>
              Functional Genomic Screens Reveal RBBP4 as a Key Regulator of Cell Cycle Progression in TMZ-Resistant Glioblastoma
              </a>
            </td>
          <td>
            Ezgi Yagmur Kala, Filiz Senbabaoglu Aksu, Erdem Ercan, Altar Ozbıyık, Ahmet Cingöz, Ozlem Yedier-Bayram, A. C. Aksu, Ayse Derya Cavga, E. Yilmaz, I. Kok, A. Humeyra, Dur Karasayar, Ibrahim Kulac, Hamzah Syed, M. Philpott, AP Cribbs, T. Bagci-Onder
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Microsatellite instability (MSI) is a hallmark of mismatch repair (MMR) deficiency and characterizes a distinct subset of colorectal cancers (CRC). In parallel, telomere length dynamics have emerged as important contributors to genomic stability and tumorigenesis. However, the relationship between MSI status, MMR protein expression, and telomere maintenance remains poorly defined. This study aimed to investigate the association between MSI status and telomere length in CRC cell lines and to evaluate the expression of key MMR proteins (MLH1, MSH2, MSH6, PMS2) to elucidate molecular differences between MSI and microsatellite stable (MSS) phenotypes. A panel of CRC cell lines with known MSI and MSS statuses was used. Telomere length was quantified using real-time quantitative PCR (qPCR) based on the T/S ratio method. MSI status was confirmed via PCR using mononucleotide repeat markers. Western blotting was performed to assess protein expressions of MLH1, MSH2, MSH6, and PMS2. β-actin served as a loading control. qPCR analysis revealed that MSI cell lines exhibited significantly longer telomeres compared to MSS lines (P < 0.05). Western blot results showed reduced or absent expression of MLH1 and PMS2 in MSI cell lines, confirming MMR deficiency. In contrast, MSS cell lines maintained normal expression of all tested MMR proteins. These findings suggest a link between defective MMR function and altered telomere dynamics in MSI-CRC. MSI CRC cell lines exhibit telomere elongation and loss of key MMR proteins, highlighting distinct molecular features compared to MSS counterparts. These insights may inform future strategies for personalized CRC diagnostics and therapeutics, particularly in the context of telomere-targeted or immunomodulatory treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceeae7fc57ed1314ec1ac7e38395f2ecdc9bde9a" target='_blank'>
              Telomere Elongation and Mismatch Repair Deficiency Distinguish MSI Colorectal Cancer Cell Lines
              </a>
            </td>
          <td>
            Rajasekhar Moka, Shreya Upadhya
          </td>
          <td>2025-05-29</td>
          <td>Cancer Research Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 10504


 Background:
 Lynch syndrome (LS), caused by germline pathogenic variants (gPVs) in DNA mismatch repair genes (
 MLH1, MSH2, MSH6, PMS2
 ), is characterized by microsatellite instability (MSI)/mismatch repair deficiency (MMRd) of associated tumors. The impact of the underlying gPV on the prognosis of LS patients with advanced cancer receiving ICB has not been elucidated.
 Methods:
 Consecutive cancer-affected LS pts consented to tumor-normal DNA sequencing via MSK-IMPACT (NCT01775072), from 01/2014-01/2024 with advanced solid tumors and receiving ≥1 ICB dose were identified. In cases of multiple cancers, only the first one was considered. MSI and MMRd was assessed via NGS-derived MSISensor score or IHC staining, respectively. Radiological response rates and survival were assessed.
 Results:
 Among 186 LS patients, 132 had an ICB treated advanced tumor with the following prevalence of gPV:
 MSH2
 41% (N=54),
 MLH1
 28%

 (N=37),
 PMS2
 17% (N=23), and
 MSH6
 14% (N=18). The most represented tumor types were colorectal (35%, N=46), urothelial (14%, N=18), pancreatic/biliary (13%, N=17), upper gastro-intestinal (11%, N=15), endometrial (11%, N=15). Objective response rate was 58%, including 35% with radiological complete response (CR) with higher CR rates in g
 MLH1
 and g
 MSH2
 patients (Table). Median follow-up was 53 months [range:0.3-116.3]. Median overall survival (mOS) / Progression free survival (mPFS) were significantly prolonged in
 non-gPMS2
 patients vs. g
 PMS2
 patients (Table). MMR IHC was available for 107 (80%) tumors identifying 88% (n=94) as MMRd and 12% (n=13) as MMRp tumors. 69% (9/13) of MMRp tumors were in
 PMS2
 patients with 8 being both MMRp and MSS suggesting a sporadic origin rather than LS-related cancer. Stratifying by MMRd and MSI status, mOS was similar in patients with MMRd/MSI and MMRd/MSS tumors (109 months vs not reached (NR); HR=0.86; 95%CI [0.35-2.13]), but significantly diminished in in pts with MMRp as compared to MMRd tumors (20 vs 109 months, HR=0.27; 95%CI [0.77-0.96]). In g
 PMS2
 patients with MMRd tumors, median OS was 27.7 months.
 Conclusions:
 ICB treatment results in high CR rates in LS patients. The shorter mOS amongst LS patients with
 gPMS2
 compared to other LS genes is partially explained by the higher prevalence of sporadic MMRp tumors. Even in the context of an MMRd tumor, patients with g
 PMS2
 have worse outcomes than LS patients with other gPVs.




 Treatment
 Total
 Complete Response% (n)
 Objective Response Rate% (n)
 Disease Control rate% (n)
 mPFS (months)
 mOS (months)





 MSH2

 54
 46% (25)
 65% (35)
 89% (48)
 45
 109



 MLH1

 37
 41% (15)
 65% (24)
 81% (30)
 50
 NR



 PMS2

 23
 9% (2)
 30% (7)
 74% (17)
 11
 20



 MSH6

 18
 22% (4)
 61% (11)
 78% (14)
 109
 NR



 MLH1/MSH2/MSH6

 109
 40% (44)
 64% (70)
 84% (92)
 52
 109



 Non-PMS2
 vs PMS2

 -
 OR=7.195%CI [1.7-31.6]P=0.003
 OR=4.195%CI [1.5-10.3]P=0.005
 OR=1.995%CI [0.64-5.6]P=0.24
 HR=0.21 95%CI [0.13-0.35].P<0.0001
 HR=0.19 95%CI [0.10-0.35]P=0.0005




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21fda5f8e2362a55474e61f5ad84ed0422e3cafb" target='_blank'>
              Gene-specific outcomes in patients with Lynch syndrome treated by immune checkpoint blockade for advanced cancer.
              </a>
            </td>
          <td>
            V. Randrian, O. Artz, Mohammad Abbass, Emily Harrold, Michael B Foote, A. Cercek, D. Mandelker, Ying L. Liu, M. Carlo, Alicia J Latham, Y. Murciano-Goroff, J. Shia, Luis A. Diaz, B. Rousseau, Z. Stadler
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Allele-specific expression (ASE) of somatic mutations can be caused by cis-activation of the mutant allele or silencing of the wildtype allele and has been investigated by examining the enrichment of mutant allele in RNA relative to DNA. Here we show that this mutation-based approach can be confounded by gene expression differences in tumor and normal cells which co-exist in most bulk tumor samples. We modeled mutant allele expression by incorporating tumor/normal expression difference, mutant allele dosage, tumor purity, and nonsense-mediated decay (NMD) efficiency, projecting that the enrichment can occur without ASE. This confounding effect is exacerbated with low tumor purity and is dependent on mutant allele dosage for NMD-sensitive mutations. The model was validated empirically using somatic insertions/deletions (indels) from 9,101 the Cancer Genome Atlas (TCGA) samples, showing three-fold higher enrichment in driver genes vs. non-drivers. To explore if the enriched mutant allele expression can be leveraged to complement DNA-based somatic mutation detection, we performed de novo somatic indel calling using TCGA RNA-Seq, increasing the TCGA driver indel repertoire by ∼14%, mostly in samples with lower tumor purity. Our study not only revealed confounding factors in ASE analysis but also demonstrated the utility of leveraging RNA-Seq for driver mutation analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/224205d6db5d08579b24b382ff5353dc292b7523" target='_blank'>
              Confounding factors in assessing the enriched expression of somatic mutant allele in bulk tumor samples
              </a>
            </td>
          <td>
            Kohei Hagiwara, A. Thrasher, Nadezhda V. Terekhanova, Jinghui Zhang
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In cancer genomics, there is the realization that some very frequently mutated genes like TP53 or KRAS are extremely hard to approach from a therapeutic perspective. In the realm of cancer treatment, the development of agents capable of transforming these alterations into actionable targets would be like capturing unicorns. These unicorns of oncology-drugs targeting undruggable targets that span various cancer types-offer an exciting opportunity for the development of histology-agnostic therapies. Among these, recent developments in targeting TP53 mutations, KRAS alterations, and MTAP loss mark pivotal advancements for innovative treatment strategies. TP53 mutations, which occur in nearly every cancer type, are central to tumorigenesis and contribute to resistance against conventional therapies. KRAS mutations, especially in cancers like pancreatic and lung cancers, are notoriously difficult to target but are now yielding promising therapeutic avenues. Similarly, MTAP is frequently deleted in cancers such as lung, pancreatic, and mesothelioma and plays a critical role in cellular metabolism and epigenetics, making its loss a significant vulnerability in tumor cells. Recent developments in drug discovery provide a broad foundation for the clinical testing of drugs that target common mechanisms across diverse tumor types, offering hope for a more universal approach to cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bc931554d7166b2b731a598949019062e38b61" target='_blank'>
              Capturing Unicorns: Targeting Cancers With TP53 Mutations, RAS Alterations Beyond G12C, and MTAP Loss-No Target Is Out of the Realm of Possibility.
              </a>
            </td>
          <td>
            Pooja A. Shah, K. Wiman, K. Cichowski, J. Rodon Ahnert
          </td>
          <td>2025-06-01</td>
          <td>American Society of Clinical Oncology educational book. American Society of Clinical Oncology. Annual Meeting</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the p53 gene are frequently observed in various cancers, prompting the initiation of efforts to restore p53 function as a therapeutic approach several decades ago. Nevertheless, only a limited number of drug development initiatives have progressed to late-stage clinical trials, and to date, no p53-targeted therapies have received approval in the USA or Europe. This situation can be attributed primarily to the characteristics of p53 as a nuclear transcription factor, which lacks the conventional features associated with drug targets and has historically been considered “undruggable”. In recent years, however, several promising strategies have emerged, including the enhanced iterations of previous approaches and novel techniques aimed at targeting proteins that have traditionally been considered undruggable. There is a growing interest in small molecules that can restore the tumor-suppressive functions of mutant p53 proteins, and the development of drugs specifically designed for particular p53 mutation types is currently underway. Other approaches aim to deplete mutant p53 or exploit vulnerabilities associated with its expression. Additionally, genetic therapy strategy and approaches have rekindled interest. Advances in mutant p53 biology, compound mechanisms, treatment modalities, and nanotechnology have opened up new avenues for p53-based therapies. However, significant challenges remain in clinical development. This review reassesses the progress in targeting p53-mutant cancers, discusses the obstacles in translating these approaches into effective therapies, and highlights p53-based therapies via nanotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811541704b1e89bbef8a11450dee5f1b59768d70" target='_blank'>
              Discovery of Drugs Targeting Mutant p53 and Progress in Nano-Enabled Therapeutic Strategy for p53-Mutated Cancers
              </a>
            </td>
          <td>
            Na Zhang, Zhiyuan Jing, Jie Song, Qiyue Liang, Yuxue Xu, Zhaowei Xu, Longping Wen, Pengfei Wei
          </td>
          <td>2025-05-26</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumors are characterized by global changes in epigenetic modifications such as DNA methylation and histone modifications that are functionally linked to tumor progression. Accordingly, several drugs targeting the epigenome have been proposed for cancer therapy, notably, histone deacetylase inhibitors (HDACi) such as vorinostat and DNA methyltransferase inhibitors (DNMTi) such as zebularine. However, a fundamental challenge with such approaches is the lack of genomic specificity, i.e., the transcriptional changes at different genomic loci can be highly variable, thus making it difficult to predict the consequences on the global transcriptome and drug response. For instance, treatment with DNMTi may upregulate the expression of not only a tumor suppressor but also an oncogene, leading to unintended adverse effect. Methods Given the pre-treatment transcriptome and epigenomic profile of a sample, we assessed the extent of predictability of locus-specific changes in gene expression upon treatment with HDACi using machine learning. Results We found that in two cell lines (HCT116 treated with Largazole at eight doses and RH4 treated with Entinostat at 1µM) where the appropriate data (pre-treatment transcriptome and epigenome as well as post-treatment transcriptome) is available, our model distinguished the post-treatment up versus downregulated genes with high accuracy (up to ROC of 0.89). Furthermore, a model trained on one cell line is applicable to another cell line suggesting generalizability of the model. Conclusions Here we present a first assessment of the predictability of genome-wide transcriptomic changes upon treatment with HDACi. Lack of appropriate omics data from clinical trials of epigenetic drugs currently hampers the assessment of applicability of our approach in clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a8c2bc7236dcb12e89963dabb5f55fdfd61138" target='_blank'>
              Predicting gene expression changes upon epigenomic drug treatment
              </a>
            </td>
          <td>
            P. Agrawal, Vishaka Gopalan, M. A. Rumi, Sridhar Hannenhalli
          </td>
          <td>2025-05-02</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="The human genome encodes 19 adenosine and cytidine deaminase genes, classified as A-to-I versus C-to-U editors. A-to-I editors have been widely identified as a promising therapeutic target in various cancers. Conversely, the investigation into C-to-U editors is relatively limited. This study evaluated RNA-editing genes in prostate cancer (PCa). Notably, the APOBEC3 genes are clustered in terms of their chromosomal locations, and their transcriptional changes exhibit significant positive correlations in both primary PCa and castration-resistant prostate cancer (CRPC). One member of this family, APOBEC3C, is demonstrated here as an androgen receptor (AR)-repressed gene. Consistently, APOBEC3 loci are epigenetically inhibited in PCa progression, with APOBEC3C level lower in PSA-high patients. APOBEC3C-low PCa cohorts exhibit increased resistance to Abiraterone and Enzalutamide. Clinicopathological profiling further confirmed APOBEC3C downregulation along PCa progression to advanced phases (grade IV/V, stage III-IV, and pathological stage T3-4), underscoring its prognostic value. Additionally, APOBEC3C expression inversely correlates with PCa relapse and mortality, and low APOBEC3C levels are linked to unfavorable survival. Notably, integrated analyses identified APOBEC3C as the sole RNA-editing gene with significance in both differential expression and PCa prognosis, and APOBEC3C had the best diagnostic performance among 19 genes. Our efforts provide a foundation for further RNA editors research in PCa diagnosis and therapy, and grant APOBEC3C as a candidate tumor suppressor. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00169-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0fae2048158500534064188ceb1148d343ad87" target='_blank'>
              Characterization of RNA editing gene APOBEC3C as a candidate tumor suppressor in prostate cancer
              </a>
            </td>
          <td>
            Li-Yang Wang, Ji Shi, Mo-Fei Wang, Yi-Meng Liu, Hong-Shan Guo, Jin-Cheng Wang, Shu Jiang, Jia-Qian Liang, Xinghua Liao, Shao-Yong Chen
          </td>
          <td>2025-05-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic variation leads to phenotypic variability in pluripotent stem cells that presents challenges for regenerative medicine. Although recent studies have investigated the impact of genetic variation on pluripotency maintenance and differentiation capacity, less is known about how genetic variants affecting the pluripotent state influence gene regulation later in development. Here, we characterized expression of 12,000 genes in a large panel of donor-matched Diversity Outbred (DO) mouse embryonic stem cell (mESC) and neural progenitor cell (mNPC) lines. Quantitative trait locus (QTL) mapping identified 4,060 expression QTL (eQTL) in mNPCs, including 2,998 local and 1,062 distant eQTL. In a comparison of mNPC and mESC eQTLs, we found that local eQTLs were more likely than distant eQTL to be detected in both cell types. Distant eQTL were largely unique to one cell type, and we mapped three mNPC-specific eQTL hotspots on Chromosomes (Chrs) 1, 10, and 11. Mediation analysis of the Chr 1 hotspot identified Rnf152 as the best candidate mediator expressed in mNPCs, while cross-cell type mediation using mESC gene expression along with partial correlation analysis strongly implicated genetic variant(s) affecting Pign expression in the mESC state as regulating the mNPC Chr 1 hotspot. These findings highlight that local mNPC eQTL are more likely than distant eQTL to be shared with mESCs; distant mNPC eQTL are numerous but largely unique to that cell type, with many co-localizing to mNPC-specific hotspots; and mediation analysis across cell types suggests that expression of Pign in mESCs shapes the transcriptome of the more specialized mNPC state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fec557f5355bb475f19a5571689366bc9c77196d" target='_blank'>
              Systems genetics reveals the influence of eQTLs in mouse embryonic stem cells on transcriptional variation later in differentiated neural progenitor cells.
              </a>
            </td>
          <td>
            Selcan Aydin, Daniel A. Skelly, Hannah Dewey, J. M. Mahoney, Ted Choi, L. Reinholdt, Christopher L. Baker, Steven C. Munger
          </td>
          <td>2025-05-06</td>
          <td>G3</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell-cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti-tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non-neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations.
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of internal medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f9a1913b670620726e7b6370c4ca3f3f02d609" target='_blank'>
              Blocking cancer-fibroblast mutualism inhibits proliferation of endocrine therapy resistant breast cancer.
              </a>
            </td>
          <td>
            Jason I. Griffiths, Feng Chi, Elena Farmaki, Eric F. Medina, Patrick A. Cosgrove, Kimya L Karimi, Jinfeng Chen, V. Grolmusz, Frederick R. Adler, Qamar J Khan, Aritro Nath, Jeffrey T. Chang, Andrea H Bild
          </td>
          <td>2025-05-08</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background T-cell acute lymphoblastic leukemia (T-ALL) is a relatively rare hematological malignancy, characterized by the uncontrolled proliferation of immature T lymphoblasts and associated with a generally unfavorable prognosis. Our previous research has demonstrated that decreased mitochondrial activity is associated with the aggressiveness of T-ALL tumors. However, the mechanisms underlying this phenomenon and its contribution to treatment resistance remain largely elusive. Methods We have built up the largest known T-ALL tumor bank, with a median follow-up of 32 months, including our transcriptomic data from 79 newly sequenced tumors that adds to the 54 publicly accessible samples. Computational analyses and a series of functional assays were performed to investigate the molecular links between altered mitochondrial activity and drug resistance. Results The transcriptomic analysis revealed that down-regulation of mitochondrial activity is a potent driver of ABCB1 activation, a gene strongly associated with multidrug resistance. In tumors with low mitochondrial activity, the impaired fatty acids β-oxidation leads to intracellular lipid accumulation, which is directly involved in ABCB1 activation. Indeed, our data show that lipid neo-synthesis and accumulation promotes the activation of lipogenic transcription factors, liver X receptors (LXRs), which act as drivers of ABCB1 expression. Tumor data analyses confirmed that high ABCB1 expression in tumour samples is indeed associated with reduced mitochondrial gene expression, lipid droplet enrichment, increased tumour aggressiveness, and significantly shorter patient survival. Conclusions Our study demonstrates that reduced mitochondrial activity drives multidrug resistance in adult T-ALL via lipid-mediated activation of ABCB1. These findings enhance our understanding of the biology of aggressive T-ALL and provide insight into mechanisms of resistance to conventional chemotherapy. Consequently, we propose that targeting de novo lipogenesis and restricting dietary fats, such as caprylic acid, may help overcome treatment resistance in patients with T-ALL exhibiting low mitochondrial activity. Trial registration The clinical trial was registered under the identifiers ChiCTR-ONRC-14004968 and ChiCTR2000031553 at ClinicalTrials.gov. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06423-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17840f9a8a2001064a43216b8500d420bfe553ae" target='_blank'>
              Mitochondrial dysfunction fuels drug resistance in adult T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Shanshan Guo, Ekaterina Bourova-Flin, Sophie Rousseaux, Florent Chuffart, Lijun Peng, Duohui Jing, Jian-Qing Mi, Saadi Khochbin, Jin Wang
          </td>
          <td>2025-05-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Bacteria with similar genomes can exhibit different phenotypes due to alternative gene expression patterns. In this study, we analysed four antibiotic-resistant Staphylococcus aureus hospital isolates using transcriptomics, PacBio genome sequencing, and methylomics analyses. Transcriptomic data were obtained from cultures exposed to gentamicin, the iodine-alanine complex CC-196, and their combination. We observed strain-specific expression patterns of core and accessory genes that remained stable under antimicrobial stress – a phenomenon we term the Clonal Gene Expression Stability (CGES) that is the main discovery of the paper. An involvement of epigenetic mechanisms in stabilization of the CGES was hypothesized and statistically verified. Canonical methylation patterns controlled by type I restriction-modification systems accounted for ~ 10% of epigenetically modified adenine residues, whereas multiple non-canonically modified adenines were distributed sporadically due to imperfect DNA targeting by methyltransferases. Protein-coding sequences were characterized by a significantly lower frequency of modified nucleotides. Epigenetic modifications near transcription start codons showed a statistically significant negative association with gene expression levels. While the role of epigenetic modifications in gene regulation remains debatable, variations in non-canonical modification patterns may serve as markers of CGES.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/312d8b17fcc90be9c2950c359123ee70cd25237f" target='_blank'>
              Epigenetic background of lineage-specific gene expression landscapes of four Staphylococcus aureus hospital isolates
              </a>
            </td>
          <td>
            I. Korotetskiy, S. Shilov, T. Kuznetsova, N. Zubenko, L. Ivanova, O. Reva
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1950d0d983eeb2c05d4af1f02b47efd0ad374c" target='_blank'>
              Theoretical investigation of AKT1 mutations in breast cancer: a computational approach to structural and functional insights
              </a>
            </td>
          <td>
            Balu Kamaraj, G. C
          </td>
          <td>2025-05-09</td>
          <td>Journal of computer-aided molecular design</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Thyroid cancer represents the most prevalent malignant neoplasm within the endocrine system, exhibiting a steadily increasing global incidence. Non-coding RNAs (ncRNAs) have emerged as a pivotal focus in thyroid cancer research, demonstrating significant involvement in tumor progression through epigenetic regulation. Contemporary studies reveal widespread dysregulation of ncRNA expression profiles in thyroid malignancies, where differentially expressed ncRNAs exert either tumor-suppressive or oncogenic functions by modulating epithelial-mesenchymal transition (EMT) mechanisms. This review synthesizes current knowledge on EMT-related ncRNA mechanisms driving thyroid carcinogenesis and evaluates their diagnostic and therapeutic potential. By elucidating these molecular interactions, we aim to catalyze discovery of novel ncRNA-mediated pathways, advance targeted treatment strategies, and ultimately enhance clinical outcomes for thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c28d43ba92bf9ff021308e25606ff1ccee6bb23c" target='_blank'>
              The regulatory network of epithelial-mesenchymal transition-associated non-coding RNAs in thyroid cancer: molecular mechanisms, clinical implications, and therapeutic strategies
              </a>
            </td>
          <td>
            Rui Zhao, Jiaju Tian, Jing Peng, Ruiting Ma, Songbo Fu
          </td>
          <td>2025-06-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758af464be2f5fe707681ad668cca7d960076b75" target='_blank'>
              Targetable BIRC5 dependency in therapy-resistant TP53 mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, E. Olesinski, Cheryl Gek Teng Chan, S. Jasdanwala, Xiao Xian Lin, Yichen Wang, Jordan Yong Ming Tan, K. Bhatia, Valeriia Sapozhnikova, Chuqi Wang, Aarthi Nivasini Mahesh, Daniel Tan En Liang, Nishtha Chitkara, P. Mertins, L. Hogdal, Brandon D Brown, T. Haferlach, Camille Lobry, Coleman Lindsley, Alexandre Puissant, Han Kiat Ho, Sanjukta Das, A. Letai, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Homologous recombination (HR) and mismatch repair (MMR) act as guardians of the human genome, and defects in HR or MMR are causative in at least a quarter of all malignant tumors. Although these DNA repair-deficient tumors are eligible for effective targeted therapies, fully reliable diagnostic strategies based on functional assay have yet to be established, potentially limiting safe and proper application of the molecular targeted drugs. Here we show that transient transfection of artificial DNA substrates enables ultrarapid detection of HR and MMR. This finding led us to develop a diagnostic strategy that can determine the cellular HR/MMR status within one day without the need for control cells or tissues. Notably, the accuracy of this method allowed the discovery of a pathogenic RAD51D mutation, which was missed by existing companion diagnostic tests. Our methods, termed HR eye and MMR eye, are applicable to frozen tumor tissues and roughly predict the response to therapy. Overall, the findings presented here could pave the way for accurately assessing malignant tumors with functional defects in HR or MMR, a step forward in accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9695c321b4adb2f24badedc1d3f1003b2ca44567" target='_blank'>
              HR eye & MMR eye: one-day assessment of DNA repair-defective tumors eligible for targeted therapy
              </a>
            </td>
          <td>
            Shinta Saito, Shingo Kato, Usaki Arai, Atsuki En, Jun Tsunezumi, Taichi Mizushima, K. Tateishi, N. Adachi
          </td>
          <td>2025-05-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There has been a substantial investment in elucidating the mechanism of expansion in hopes of identifying therapeutic targets for Huntington disease (HD). Although an expanded CAG allele is the causal mutation for HD, there is evidence that somatic expansion may not be the only disease driver. We report here that double strand breaks (DSBs) drive HD toxicity by an independent mechanism from somatic expansion. The mutant HD protein inhibits non-homologous end joining (NHEJ) activity, leading to the accumulation of DSBs. DSBs promote transcriptional pathology in mice that cannot expand their CAG tracts somatically. Conversely, Inhibition of DSBs reverses neuronal toxicity in animals that undergo somatic expansion. Although they coexist in neurons, DSBs and somatic expansion are independent therapeutic targets for HD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98e2f4c3e43c43464e34f5bd46068f4d423e6b46" target='_blank'>
              Double strand breaks drive toxicity in Huntington’s disease mice with or without somatic expansion
              </a>
            </td>
          <td>
            Aris A. Polyzos, Ana Cheong, Jung Hyun Yoo, Lana Blagec, Zachary D. Nagel, Cynthia T McMurray
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c635cae84c7bdc8795ab539e7b406a70e03c5a4" target='_blank'>
              A transient mutational burst occurs during yeast colony development.
              </a>
            </td>
          <td>
            Nicolas Agier, Nina Vittorelli, Louis Ollivier, Frédéric Chaux, Alexandre Gillet-Markowska, Samuel O'Donnell, Fanny Pouyet, G. Fischer, Stéphane Delmas
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The CRISPR-Cas9 system has emerged as a transformative tool in genetic engineering, enabling scientists to edit genomes with unprecedented precision, efficiency, and simplicity. Since its adaptation from a prokaryotic immune mechanism to a gene-editing platform, CRISPR-Cas9 has rapidly progressed from bench research to translational and clinical applications. This paper presents a comprehensive review of the most recent advancements in CRISPR-Cas9 technology, particularly focusing on innovations that enhance the accuracy, versatility, and therapeutic utility of gene editing. Key breakthroughs include the development of base editing and prime editing technologies, which allow for single-nucleotide substitutions and complex DNA insertions or deletions without creating double-stranded breaks. These tools have significantly expanded the scope of precise genetic modifications and minimized genotoxicity. 
Moreover, the engineering of high-fidelity Cas9 variants—such as eSpCas9, SpCas9-HF1, and HypaCas9—has markedly reduced off-target activity, thereby addressing one of the major limitations in clinical genome editing. Innovations in delivery strategies, including the use of viral vectors (e.g., AAV, lentivirus), lipid nanoparticles, and electroporation of ribonucleoprotein complexes, have improved the in vivo and ex vivo application of CRISPR-Cas9 for therapeutic purposes. Notable clinical progress has been observed in the treatment of genetic disorders such as sickle cell disease, β-thalassemia, and hereditary blindness, with several patients experiencing durable therapeutic benefits. Simultaneously, CRISPR-based immunotherapies are being explored for cancers and viral infections, such as HIV, marking a shift toward more personalized and targeted treatments. 
Despite these advances, challenges remain, including immune responses to Cas9 proteins, potential mosaicism in edited cells, ethical concerns regarding germline editing, and regulatory hurdles that govern clinical translation. Furthermore, the integration of computational tools, including machine learning and deep learning algorithms, has become critical for optimizing guide RNA design, predicting off-target effects, and refining editing outcomes. This integration of biotechnology and computational sciences is propelling the CRISPR field into a new era of precision medicine. 
In conclusion, recent advancements in CRISPR-Cas9 technology are not only pushing the boundaries of what is technically feasible in genome editing but are also laying the groundwork for safe, effective, and personalized therapeutic applications. As the technology continues to mature, it holds the promise to redefine the future of medicine, particularly in the treatment and potential cure of a wide range of genetic and complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c179a051e549adc925ccce1b7d2a4b7dcb7a485" target='_blank'>
              Recent Advancements in CRISPR-Cas9 Technology for Precision Gene Editing and Therapeutic Applications
              </a>
            </td>
          <td>
            Dr. D.Jayarajan, Dr. Amit Kumar Dutta, Dr. T.Deborah Paripuranam, Mrs. M.Krishnaveni, Dr S.Bhavya
          </td>
          <td>2025-05-23</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e17020


 Background:
 Testicular germ cell tumors (TGCT) are the most common malignancy in young adult men, with a globally increasing incidence. While these tumors are highly curable due to their exceptional sensitivity to cisplatin (cDDP), a proportion of patients develops a resistance to this chemotherapy, which results in disease progression and patient death. The cause of cisplatin resistance in TGCT has not been elucidated yet and is supposed to be multifactorial.
 Methods:
 We conducted a CRISPR Knockout (KO) screening in a TGCT cell line (NCCIT) using a lentiviral sgRNAs library (Addgene #101926) targeting 3015 genes involved in cell cycle regulation and cancer development. The screening population was cultured with or without cDDP (1.25 uM) for 7 days. Samples of treated and control cells were harvested before and after cDDP exposure, the sgRNA cassettes were sequenced by massive parallel sequencing, and the differences in sgRNA abundance were determined using the MAGeCK/MAGeCK-Flute pipeline.
 Results:
 The CRISPR-KO screening data analysis identified 59 gene KOs that were significantly positively selected and 7 gene KOs that were significantly negatively selected after cDDP exposure. Among the top positive hits were genes involved in the regulation of apoptosis (
 BMF, CDIP1
 and
 BAX
 ), epigenetic modifiers (
 HDAC8, SSX2
 ), G-protein regulators (
 ADCY8, GNG10
 ), and others (e.g.
 MAGEA2, ACADM, LAMP1
 ) — inactivation of these genes potentially contributes to the development of cDDP resistance. Among the top negative hits were genes implicated in DNA damage repair (
 FANCB
 ,
 ERCC5
 ), sphingolipid metabolism (
 KDSR
 ), and cytokinesis (
 SPECC1L
 ) — inactivation of these genes may increase sensitivity to cDDP.
 Conclusions:
 The results highlight several potential mechanisms of cDPP resistance in TGCT. Some of these genes and pathways are well known and have been previously described in association with platinum resistance in malignant tumors, while others — such as the G-protein regulators, sphingolipid synthesis, and
 SPECC1L
 — are novel in this context and may reflect the unique biology of TGCT progression. SPECC1L is involved in spindle organization and microtubule stabilization; its paralog SPECC1 is highly expressed in testis. Polyenoic sphingolipids are essential for the completion of meiosis in male germ cells and spermatozoa production.

 ADCY8 controls the production of cAMP, which is a key signaling molecule in spermatogenesis. They have also been described in association with malignant proliferation and progression. The identified hits will be validated in subsequent experiments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff267218117b8545c90f6a863002128909b0591f" target='_blank'>
              Use of CRISPR-KO screening to identify novel genes and pathways potentially contributing to cisplatin resistance in testicular germ cell tumors.
              </a>
            </td>
          <td>
            Laura Matouskova, Kristyna Soffrova, V. Bakardjieva-Mihaylova, J. Stuchlý, L. Boublikova, Karolina Skvarova
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5b8cce1b576a2635e4230be0b09f8a3f1a1a138" target='_blank'>
              Metabolite correlation networks reveal complex phenotypes of adaptation-driving mutations
              </a>
            </td>
          <td>
            Sharon Samuel, Weronika Jasinska, Ezra Sternlicht, Matvey Nikelshparg, D. Kleiner, S. Pilosof, Shimon Bershtein
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Exposure to progesterone is a recognized risk factor for breast cancer, and PGR polymorphisms are associated with various malignancies. Two progesterone receptor (PR) isoforms, full length PR-B and truncated PR-A, are expressed from the PGR gene in breast tissue and play crucial roles in normal physiology and breast cancer progression. An imbalance in the expression ratio of these isoforms, favoring increased levels of PR-A, is common in breast cancer and is associated with resistance to tamoxifen in luminal A-type tumors. Notably, PRs have recently been implicated in promoting endocrine resistance and driving the expansion of cancer stem-like cell (CSC) populations. Despite this insight, the isoform-specific molecular and epigenetic mechanisms underlying PR action in estrogen receptor positive (ER+) breast cancers remain understudied. Phenotypic studies of T47D cell lines that express exclusively PR-A or PR-B showed that PR isoforms regulate divergent cell fates. PR-B-expressing cells have a higher proliferation rate, while PR-A-expressing cells produce more mammospheres. We profiled progesterone-driven gene expression in cells grown in both adherent (2D) and mammosphere (3D) growth conditions and found differential gene regulation by PR-A and PR-B that is consistent with the observed divergent phenotypes. Only the PR-A-driven gene signature of ER+ breast cancer cells maintained as non-adherent mammospheres robustly predicted poor clinical outcome in the METABRIC data set. We then performed CUT&RUN to identify the genomic binding patterns unique to each PR isoform and their suite of target genes. Our findings indicate that PR-A acts as a regulator of the cell cycle, while PR-B plays a pivotal role in metabolism and intracellular signaling. Our genomic profiling of PRs in this model system has unveiled novel isoform-specific functions of PR. This work has shifted our prior understanding of the role of PRs in gene regulation, offering potential insights for therapeutic interventions in ER+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce41088c34704ab987d2f4090fccff3affff4423" target='_blank'>
              Isoform-Specific Gene Regulation by Progesterone Receptors Drives Divergent Phenotypes in Breast Cancer Cells
              </a>
            </td>
          <td>
            Noelle E. Gillis, Thu H. Truong, Caroline H Diep, A. Spartz, J. Ostrander, Carol A. Lange
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="As many as 90% of human pancreatic ductal adenocarcinoma (PDAC) tumors harbor gain-of-function mutations in the KRAS oncogene. Recently, inhibitors of the most common KRAS mutation, KRASG12D, have entered the clinical arena. However, early evidence suggests that as monotherapy, KRASG12D inhibitors such as MRTX1133 at best provide brief periods of disease stabilization. Hence, there is a growing interest in understanding the mechanisms through which tumors acquire resistance to KRAS inhibition. In the present study, we generated in vitro models of MRTX1133 resistance and subjected parental and drug-resistant cell lines to RNA sequencing. This suggested that MRTX1133-resistant tumor cells undergo a global shift toward histone acetylation. Inhibition of the histone acetyltransferase EP300 reversed the drug-resistant phenotype in vitro, which subsequent RNA sequencing experiments determined was associated with the suppression of pro-survival FOSL1 signaling. Accordingly, siFOSL1 reversed the MRTX1133-resistant phenotype with similar effects on pro-survival signaling. Given the lack of clinically useful EP300 or FOSL1 inhibitors, we next explored whether inhibitors of the acetylation scanning BET proteins would be similarly effective. The addition of BET inhibitors re-sensitized several resistant cell lines to MRTX1133 and impaired FOSL1-mediated survival signaling in vitro. In murine models of MRTX1133-resistant PDAC, BET inhibition cooperated with MRTX1133 to markedly extend overall survival. As BET inhibitors are currently under clinical testing, the combination of MRTX1133 and BET inhibitors warrants further investigation, particularly in tumors that have developed resistance to KRAS inhibition. SIGNIFICANCE Here, we demonstrate that BET inhibition is effective in PDAC with acquired resistance to KRAS inhibitors. As BET inhibitors are under clinical testing, the combination of KRAS and BET inhibitors warrants consideration in PDAC patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31571cc4d771e5d8cbcfb30a223ba711a12ce0b" target='_blank'>
              Resistance to the KRASG12D Inhibitor MRTX1133 is Associated with Increased Sensitivity to BET Inhibition
              </a>
            </td>
          <td>
            Daniel R. Principe, Jeffrey H Becker, Anastasia E. Metropulos, Alejandra M Marinelarena, Thao D Pham, A. Aissa, H. Munshi
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The discovery and implementation of CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) technology have transformed the landscape of biotechnology and precision medicine. As a revolutionary gene-editing tool, CRISPR enables targeted, efficient, and cost-effective modification of DNA, offering promising therapeutic interventions for a wide range of genetic and acquired diseases. This article provides an in-depth overview of CRISPR’s origins, mechanisms, and applications in drug development, from treating genetic disorders and enhancing cancer immunotherapies to accelerating drug discovery and developing tools against infectious diseases. It also addresses ethical considerations, regulatory frameworks, and future innovations such as base and prime editing. By exploring current trends and interdisciplinary applications, this article underscores the pivotal role CRISPR plays in shaping the future of medicine, agriculture, and synthetic biology">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28265b2b5d50360e8a6fadc5c453494039885eb5" target='_blank'>
              CRISPR and Gene Editing in Drug Development: A Revolution in Precision Medicine
              </a>
            </td>
          <td>
            Khushboo Gupta, Ashley Christeena Thomas, Vandana Santwani, Binit Kumar Sarangi, Ankush Chandrakar, S. Tiwari
          </td>
          <td>2025-05-24</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/276c259bf4b9ccfc8a64aee15d032ce2cc575529" target='_blank'>
              Antibody-Based Targeting of the SPP1-CD44 Axis in Pediatric High-Grade Glioma through Single-Cell and Structural Bioinformatics
              </a>
            </td>
          <td>
            Shiwani Limbu, Ambuj Kumar
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686514f6a242593983edc4eedbd2ea02d17cd369" target='_blank'>
              REpair of heterozygous Mutations independent of Exogenous Donor template with high efficiency (REMEDY) using allele specific CRISPR targeting and HDR enhancers
              </a>
            </td>
          <td>
            Y. Sezgin, Genesis Snyder, Noushin Saljoughian, Colin Maguire, Ebru Erzurumluoglu Gokalp, Devi Jaganathan, Eleonora S D'Ambrosio, B. Ozes, Gregory Wheeler, Ben Kelly, Mark E. Hester, Zarife Sahenk, S. Vaidyanathan, A. Rashnonejad, Jerry Mendell, Nizar Y. Saad, Roshini S. Abraham, Juhi Bagaitkar, Susan D. Reynolds, Allison Bradbury, M. N. Kararoudi
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5efe50139c531598a2ac6d2663615d0ad3cf36f2" target='_blank'>
              Combined genomic and phenotypic classification of inherited and acquired genetic disease with long-read sequencing
              </a>
            </td>
          <td>
            Y. Fontaine, S. Aradhya, S. Ji, E. Masle-Farquhar, I. Bosi, J. Forkgen, Z. Qiao, I. Stevanovski, A. M. Reis, M. Rapadas, D. Suan, P. Hsu, B. Wainstein, A. W. Hewitt, G. A. Figtree, I. W. Deveson, O. M. Siggs
          </td>
          <td>2025-05-09</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 In acute myeloid leukemia (AML), genetic mutations distort hematopoietic differentiation, resulting in the accumulation of leukemic blasts. However, it remains unclear how these mutations intersect with the cellular origins of each patient's disease, and whether distinct sets of mutations converge upon similar differentiation patterns. Single-cell RNA sequencing has enabled high-resolution mapping of the relationship between leukemia and normal hematopoietic cell states. Yet, this application has been hampered by imprecise reference maps of normal hematopoiesis, or by small patient cohort sizes which do not adequately capture the extent of inter-patient heterogeneity in AML. To resolve this, we constructed a reference atlas of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes enriched for rare hematopoietic stem and progenitor cells (HSPCs). The resulting reference spans 55 cellular states and has been benchmarked against independent datasets of purified HSPCs. Using this comprehensive reference atlas, we confidently mapped over 1.2 million single-cell transcriptomes from 318 patient samples spanning AML, mixed-phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) diagnoses. Single-cell composition analysis revealed twelve patient subgroups, each reflecting distinct patterns of aberrant differentiation in AML. Strikingly, some AML samples exhibited virtually no overlap in cell state involvement with one another, likely reflecting their disparate cellular origins. One subgroup was enriched for early lymphoid progenitors and featured co-clustering of AML and MPAL samples. Notably, this early lymphoid subgroup included an AML sample which eventually relapsed with lymphoid disease. To understand the genetic determinants of aberrant differentiation in AML, we quantified leukemia cell state abundance in >1,200 patient samples and evaluated genotype-to-phenotype associations to link >40 genetic driver alterations with their specific impacts on AML differentiation. This identified genetic drivers for unconventional lineage phenotypes in AML, including erythroid lineage priming associated with the co-occurrence of complex cytogenetics with TP53 mutations as well as lymphoid lineage priming associated with bi-allelic RUNX1 mutations. We also identified non-genetic determinants of AML differentiation. For example, we identified two subgroups of KMT2A-rearranged AML with distinct cellular origins reflecting either HSPCs or committed myeloid precursors. Last, we show that distinct leukemia cell hierarchies can co-exist within individual patients, providing insights into AML evolution. Together, single-cell reference mapping of malignant cell states provides a framework for understanding the impact of genetic alterations on hematopoietic differentiation across hundreds of individual AML patients.



 Andy G. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A. Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, Charles G. Mullighan, John E. Dick. Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3806.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37654945a8b87aa1dc552938a4ce7cfc7067cb7a" target='_blank'>
              Abstract 3806: Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML
              </a>
            </td>
          <td>
            Andy G. X. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, C. Mullighan, J. Dick
          </td>
          <td>2025-05-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="The CRISPR-Cas9 system has been harnessed and repurposed into a powerful genome editing tool. By leveraging this technology, researchers can precisely cut, paste, and even rewrite DNA sequences within living cells. Nevertheless, the application of CRISPR screen technology goes far beyond mere experimentation. It serves as a pivotal tool in the fight against genetic diseases, systematically dissecting complex genetic landscapes, empowering researchers to unravel the molecular mechanisms underlying biological phenomena, and enabling scientists to identify and target the root causes of illnesses such as cancer, cystic fibrosis, and sickle cell anemia. Among all, cancer poses a formidable challenge for medicine, spurring eradication efforts. Radiotherapy, as a traditional treatment, yields results but has limitations. It eradicates cancer cells but also damages healthy tissues, causing adverse effects that reduce quality of life. Additionally, not all cancer cells respond to radiotherapy, and some may develop resistance, worsening the condition. To address this, a comprehensive whole-genome CRISPR screen technology is introduced, as it enables the efficient identification of radiosensitive and radioresistant genes, thereby advancing the field of cancer research and treatment. A genome-wide CRISPR screen was conducted in lung adenocarcinoma cells exposed to irradiation following the described protocol, through which both radioresistance- and radiosensitivity-associated genes were identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0b084e039e9bd637d4e61bfb3817e38d13b7fe" target='_blank'>
              Genome-Wide CRISPR Screen for Unveiling Radiosensitive and Radioresistant Genes.
              </a>
            </td>
          <td>
            Yu Yuan, Ziyu Jiang, Yuxin Zeng, Jiawen Tang, Jiang Luo, Conghua Xie, Yan Gong
          </td>
          <td>2025-05-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="INTRODUCTION
The emergence of drug resistance poses a serious threat to cancer chemotherapy. by a single agent. Tumor cell heterogeneity, mutation and/or desensitization of receptor render monotherapy ineffective. Combination encompassing multiple targets or biochemical pathways, seems promising for cancer treatment. Combination of HDAC inhibitor(s) with other inhibitor(s) have shown synergistic activity in cancer chemotherapy by modulating a variety of therapeutic targets including epigenetic target of cancer cells by restoring acetylation and reactivating tumor suppressor genes leading to cell cycle arrest, promoting apoptosis, and thus inhibiting cancer cell proliferation.


AREAS COVERED
A comprehensive published patent literature (2020-present) on rational combinations of HDAC inhibitor(s) for cancer chemotherapy has been retrieved and reviewed from various patent databases including Google Patents, Espacenet, Patentscope, WIPO and USPTO etc. to analyze the rational combinations for better, optimized and precise cancer therapy. In this second part of two-part review, we highlighted the patent published for the combination.


EXPERT OPINION
The HDAC inhibitor(s) in combination with other therapeutically relevant inhibitor(s) such as MAPK, GSK3, PI3K/mTOR, PARP, CDK9, HSP90, BTK, BRD4, JAK, VEGF, ALK, PD-1, or PDE inhibitor etc. showed synergistic anti-cancer activity. These combinations not only overcame drug resistance but also acted against relapsed/refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00980df0bbe9fd6925862c998a7648c2f27f87d9" target='_blank'>
              An updated patent review on rational combinations of HDAC inhibitors for cancer chemotherapy (2020 - present): part 2 - patent published.
              </a>
            </td>
          <td>
            Yugal Kishor Shukla, Vandana, Vivekananda Mandal, V. Asati, Raj K. Keservani, S. Bharti
          </td>
          <td>2025-06-05</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Melanomas are genetically heterogeneous, displaying mitogen-activated protein kinase mutations and homozygous loss of tumor suppressor genes. Mouse models combining such mutations produce fast-growing tumors. In contrast, rare, slow-growing tumors arise in mice combining Braf activation with heterozygous loss of Pten. Here we show that similar tumors can arise in albino mice bearing only a Braf mutation. Incidence kinetics suggest a stochastic event underlies tumorigenesis in tumors that arise with only a Braf mutation, yet de novo mutations or structural variants that could explain the incidence of most tumors could not be found. Single-cell transcriptomics of tumors identify a cell type resembling “neural crest-like” cells in human and mouse melanomas. These exist in normal mouse skin, expand upon Braf activation, and persist through serial transplantation; analyses of gene expression suggest they serve as precursors of malignant cells. This state may serve as an intermediate on a slow path to malignancy that may provide a diagnostically and therapeutically important source of cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a8b6d33ff26b499d62c39adf32521b06fa0db2" target='_blank'>
              Uncovering minimal pathways in melanoma initiation
              </a>
            </td>
          <td>
            Hui Xiao, Jessica Shiu, Chi-Fen Chen, Jie Wu, Peijie Zhou, Sahil S. Telang, Rolando Ruiz-Vega, Robert A Edwards, Qing Nie, A. D. Lander, Anand K. Ganesan
          </td>
          <td>2025-06-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Anthracycline-induced cardiotoxicity (ACT) is a significant concern for cancer survivors, while genetic basis of ACT remains unclear. This study employs a murine genetic reference population (GRP) of BXD recombinant inbred strains, derived from DBA/2J (D2) and C57BL/6J (B6) crosses, to map quantitative trait loci (QTLs) linked to doxorubicin (DOX)-induced phenotypes through systems genetics approaches. Methods To model variability in ACT, 58 BXD strains and parental B6 and D2 mice (n ≥ 4 mice/sex/strain, 3–4-month-old) underwent an intraperitoneal injection of DOX (20 mg/kg). Survival and body weight (BW) were monitored for 10 days. Echocardiography was performed before treatment and on Day 5 post-treatment, followed by genetic mapping and Mendelian randomization analyses for identifying QTLs and candidate genes associated with DOX-induced traits and severity. Results Parental B6 strain had 60% survival, whereas 24% of D2 mice survived on Day 10. Among BXD strains, median survival varied, with BXD77 showing the lowest at Day 4. Echocardiography revealed cardiac dysfunction and a small-heart phenotype resembling ACT patients. Significant QTLs on Chromosome 10 (86–94 Mb), Chromosome 19 (52.5–54.2 Mb) and on Chromosome 14 (103–120 Mb) were associated with the survival, mean BW loss, and left ventricular (LV) volumes and ejection fraction (EF%), respectively. MR analysis identified significant causal associations between the genes implicated in BW loss (ADD3, HSPA12 A, SLC18 A2, PDZD8, DUSP5, CASP7) as well as EF% and LV volumes (GPC6, UGGT2, SLAIN1, POU4 F1, MBNL2) in BXD mice post-DOX and heart failure outcomes in humans. Most of the top candidates showed cardiomyocyte specific expression based on scRNA-seq data. Conclusions Survival, BW loss, and echocardiography parameters considerably varied among DOX-treated BXDs, suggesting significant influence of genetic background on expression of those traits. Several candidate genes that may modulate ACT susceptibility and heart failure were identified, providing a foundation for genetic-based risk stratification and therapeutics in cardio-oncology. Supplementary Information The online version contains supplementary material available at 10.1186/s40959-025-00349-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529bb5ece51e282110735b75f13ced4057c145bb" target='_blank'>
              Unraveling the genetic blueprint of doxorubicin-induced cardiotoxicity through systems genetics approaches
              </a>
            </td>
          <td>
            Buyan-Ochir Orgil, A. Bajpai, Neely R. Alberson, Morgan Lander, Batsaikhan Enkhzul, Hugo R. Martinez, Jeffrey A Towbin, Lu Lu, E. Purevjav
          </td>
          <td>2025-06-03</td>
          <td>Cardio-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Canine malignant insulinoma is a rare, highly metastatic and life-threatening neuroendocrine tumour of pancreatic beta cells. To map the single-cell transcriptomic landscape of canine insulinoma for the first time, transcriptomic profiles of 5,532 cells were captured from two spontaneous insulinomas (Patient 1 and 2) and one associated metastasis (Patient 2) in two Boxer dogs. Distinct cancer, endocrine, and immune cell populations were identified. Notably, all three tumour samples contained two transcriptionally distinct insulin-expressing tumour cell populations (INS+ and INS+FOSlow), characterised here for the first time. These two cancer cell populations significantly differed by ~ 8,000 differentially expressed genes (DEGs), particularly tumour suppressor genes (e.g. TP53, EGR1) and cancer-related pathways (e.g., MAPK, p53). In contrast, COX7A2L was one of a few genes ubiquitously expressed and significantly upregulated (> 20-fold) in both insulin-expressing tumour populations compared to other captured populations. Both populations were also characterised by expression of chromogranin/secretogranin neuroendocrine tumour marker genes (e.g. CHGA, SCGN). There were far fewer gene expression differences observed between insulin-expressing tumour cells from the two patients (~ 600 DEGs) than between the two cancer cell populations within each patient. These DEGs included CLTRN, TMSB4X, CSRP2, LGALS2, and C15orf48. Unexpectedly for a tumour of endocrine origin, the metastasis in Patient 2 exhibited > 20–70 fold upregulation of exocrine pancreatic genes including CLPS, PRSS2, PRSS and CTRC. Immune cell analyses identified distinct infiltrating immune populations, including memory T cells and macrophages and revealed likely tumour-immune interactions, including the CD40-CD40L interaction. This study provides the first single-cell RNA sequencing (scRNA-seq) analysis of naturally occurring insulinoma in any species, revealing tumour cell heterogeneity, novel immune microenvironment features, and potential therapeutic targets. Despite its small scale, the findings highlight the utility of scRNA-seq in veterinary oncology and its translational potential for pancreatic neuroendocrine tumours across species. Supplementary Information The online version contains supplementary material available at 10.1186/s44356-025-00026-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e2940ac404526b0febb4d2ad2d3e5b02fd7aaa" target='_blank'>
              Single-cell transcriptomic analysis of canine insulinoma reveals distinct sub-populations of insulin-expressing cancer cells
              </a>
            </td>
          <td>
            M. Wallace, M. E. Herrtage, R. Gostelow, L. Owen, L. Rutherford, K. Hughes, A. Denyer, B. Catchpole, C. A. O’Callaghan, L. Davison
          </td>
          <td>2025-05-26</td>
          <td>Veterinary Oncology (London, England)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The study of circulating tumor cells (CTCs) provides critical insights into the biological mechanisms underlying metastasis. This study aims to demonstrate the applicability of an integrated DEPArray-based phenotypic analysis combined with transcriptomic characterization to investigate the biology of CTCs isolated from 10 patients with metastatic breast cancer (MBC). The transcriptional profiles of CTCs were consistent with both the cancer type and epithelial characteristics. Gene set enrichment analysis identified pathways associated with synapse organization and calcium channel activity. Furthermore, distinct gene expression profiles were linked to specific metastatic sites, particularly bone metastases. We also report a rare and understudied population of CTCs, characterized by the co-expression of epithelial and leukocyte markers, observed exclusively in patient-derived samples and not in blood samples from healthy volunteers spiked with SKBR-3 and MCF-7 cell lines. This suggests that these double-positive CTCs (dpCTCs) may have a specific role in the metastatic process. The transcriptomic characterization of these rare CTCs enhances our understanding of their biology and potential involvement in metastasis. As a pilot study, our findings underscore the potential of CTC-based transcriptomics as a valuable tool for advancing clinical and biological understanding of MBC, particularly regarding metastatic mechanisms and organotropism. Moreover, it provides preliminary insights into dpCTCs, a poorly characterized population that may play a pivotal role in metastasis but remains largely unexplored. These findings could pave the way for developing targeted therapies aimed at the pathways driving metastasis and for improving patient monitoring through CTC profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c8d5b6cac842e5c06154018a6fe136a8682dd8" target='_blank'>
              Transcriptomic profiling of circulating tumor cells from metastatic breast cancer patients reveals new hints in their biological features and phenotypic heterogeneity
              </a>
            </td>
          <td>
            T. Rossi, Sara Bandini, Michele Zanoni, M. Cortesi, M. Palleschi, E. Bandini, Andrea Rocca, Giulia Gallerani, Ivan Vannini, Meropi Plousiou, L. Gerratana, A. Musolino, G. Tallini, Giovanni Martinelli, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-05-06</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The clustered regularly interspaced short palindromic repeats (CRISPR)/Cas system is a powerful genome editing tool that has revolutionized research. Single nucleotide polymorphisms (SNPs) are the most common form of genetic variation in humans. Only a subset of these SNPs has been shown to be linked to genetic diseases, while the biological relevance of the majority remains unclear. Investigating these variants of unknown significance could provide valuable insights into their roles in biological processes, disease susceptibility, and treatment responses. While CRISPR/Cas has emerged as a transformative technology, its ability to make single nucleotide substitutions remains a significant limitation. Other techniques in single nucleotide editing, such as base editing and prime editing, offer promising possibilities to complement CRISPR/Cas systems, though they also have their own limitations. Hence, alternative approaches are necessary to overcome the limitations of CRISPR. Here, to improve the feasibility of generating single base edits in the genome, we provide a protocol that introduces a multiple expression and dual selection (MEDS) system, which, alongside CRISPR, utilizes the opposing roles of cytosine deaminase/uracil phosphoribosyltransferase (CD/UPRT) for negative selection and neomycin phosphotransferase II (NPT II) for positive selection. As a proof of concept and to demonstrate feasibility of the method, we used MEDS, along with traditional CRISPR-Cas9, to generate sickle hemoglobin by introducing a point mutation (A → T) in the sixth codon of the hemoglobin beta gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea49db880f24cd1e00dae98c95c6cafbd8936b9" target='_blank'>
              CRISPR genome editing using a combined positive and negative selection system
              </a>
            </td>
          <td>
            I. Sharma, Kerisa Hall, Shannon Moonah
          </td>
          <td>2025-05-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d73cdbb862170214d86c7235fa4d0f318184be5" target='_blank'>
              Genome-wide CRISPR screens identify PTGES3 as a novel AR modulator
              </a>
            </td>
          <td>
            Haolong Li, James E. Melnyk, Becky Xu Hua Fu, Raunak Shrestha, Meng Zhang, M. Sjöström, S. Feng, Jasmine A. Anderson, Wanting Han, Lisa N Chesner, Hyun Jin Shin, Tatyanah Farsh, Humberto J. Suarez, Seema Nath, Jonathan Chou, Rajdeep Das, Emily A Egusa, Jun Zhu, Aidan Winters, Ashutosh Maheshwari, Junjie T Hua, M. Alshalalfa, William S. Chen, Marsha Calvert, E. Davicioni, Audrey Kishishita, Abhilash Barpanda, Tianyi Liu, Arun P. Wiita, B. Stohr, Javed Siddiqui, Bo Huang, Eric J. Small, K. Shokat, Peter S. Nelson, David A Quigley, Elizabeth V. Wasmuth, Luke A. Gilbert, Felix Y. Feng
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c2d54e4671afd8b01fe843ed17126ffeb50bcf" target='_blank'>
              Genetic, epigenetic, and molecular determinants of multiple myeloma and precursor plasma cell disorders: a pathophysiological overview.
              </a>
            </td>
          <td>
            Henry Sutanto, Debi Yulia Sandra, Ardea Safira, Galih Januar Adytia, Alief Waitupu, P. Z. Romadhon
          </td>
          <td>2025-06-03</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is a growing concern around the world with more than 20 million new cases confirmed and counting. This is partly caused by the overuse of Chemotherapeutic drugs against Cancer, leading to its potency. Cancer starts with an accumulation of genetic mutations in the genome, and the ignorance to cell checks by CD8 and NK cells. The Cancer cell utilizes its hallmark capabilities and uses one of its cells to metastasizes in other areas of the body either through the perineural, circulatory, or lymphatic system. p53 is a tumor suppressor gene which repairs the mutations of the cell by conducting cell cycle arrest and inducing apoptosis. However, cancer cells can increase the expression of SB100 which can inhibit p53. One solution is to combine PRIMA-1 and CDDP which restores p53 and it’s wild type functions to induce apoptosis. This effect is increased with the use of adenoviral DN-Akt since it could increase p53 phosphorylation at Ser15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928e7ed5dc7a58f8574294e13f4f74e2d872a648" target='_blank'>
              Evolving Resistance: The Interplay of p53, Chemoresistance, and Metastasis in Cancer's Journey
              </a>
            </td>
          <td>
            Zenith Kwong
          </td>
          <td>2025-06-17</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d271ccc22e71dc47432dac9f4b41d18e3a1dd713" target='_blank'>
              Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance.
              </a>
            </td>
          <td>
            E. Dorans, K. Jagadeesh, K. Dey, A. L. Price
          </td>
          <td>2025-06-12</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) remains a leading cause of cancer-related mortality globally, with KRAS mutations present in approximately 20–25% of cases. The KRAS-G12C mutation, occurring in approximately 14% of lung adenocarcinomas, has emerged as a critical target for precision medicine strategies. While KRAS-G12C inhibitors, including sotorasib and adagrasib, have shown promise in clinical trials, their efficacy is limited by primary and acquired resistance mechanisms. This study explored the potential of combining anlotinib, a multi-target tyrosine kinase inhibitor, with KRAS-G12C inhibitors to overcome these resistance challenges in NSCLC treatment. Our results demonstrated that anlotinib improved the sensitivity to KRAS-G12C inhibitors in primary and acquired resistance settings, both in vitro and in vivo. Mechanistically, the combination therapy inhibited c-Myc/ORC2 signaling, leading to cell cycle arrest and apoptosis. These findings suggest that the combination of anlotinib and KRAS-G12C inhibitors represents a promising novel therapeutic approach for KRAS-G12C-mutant NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f150607b82a1e3bb412c0390789a9240eec3f016" target='_blank'>
              Anlotinib enhances the efficacy of KRAS-G12C inhibitors through c-Myc/ORC2 axis inhibition in non-small cell lung cancer
              </a>
            </td>
          <td>
            Hongyu Liu, Chao Zhou, Jun Lu, Yuqing Liu, Peichen Zou, Liang Zhu, Huimin Lei, Baohui Han
          </td>
          <td>2025-05-02</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="One of the strongest signatures of aging is an accumulation of mutant mitochondrial DNA (mtDNA) heteroplasmy. Here we investigate the mechanism underlying this phenomenon by calling mtDNA sequence, abundance, and heteroplasmic variation in human blood using whole genome sequences from ∼750,000 individuals. Our analyses reveal a simple, two-step mechanism: first, individual cells randomly accumulate low levels of “cryptic” mtDNA mutations; then, when a cell clone proliferates, the cryptic mtDNA variants are carried as passenger mutations and become detectable in whole blood. Four lines of evidence support this model: (1) the mutational spectrum of age-accumulating mtDNA variants is consistent with a well-established model of mtDNA replication errors, (2) these mutations are found primarily at low levels of heteroplasmy and do not show evidence of positive selection, (3) high mtDNA mutation burden tends to co-occur in samples harboring somatic driver mutations for clonal hematopoiesis (CH), and (4) nuclear GWAS reveals that germline variants predisposing to CH (such as those near TERT, TCL1A, and SMC4) also increase mtDNA mutation burden. We propose that the high copy number and high mutation rate of mtDNA make it a particularly sensitive blood-based marker of CH. Importantly, our work helps to mechanistically unify three prominent signatures of aging: common germline variants in TERT, clonal hematopoiesis, and observed mtDNA mutation accrual.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a832134834abc3268ec7af8e86bace5b876111" target='_blank'>
              Mechanism of age-related accumulation of mitochondrial DNA mutations in human blood
              </a>
            </td>
          <td>
            Rahul Gupta, Timothy J. Durham, Grant Chau, M. Uddin, Wenhan Lu, K. Karczewski, D. Howrigan, P. Natarajan, Wei Zhou, Benjamin M. Neale, V. Mootha
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Molecular profiling of DNA and RNA from pediatric cancers by next-generation sequencing has been demonstrated to improve diagnosis and prognosis and to identify somatic alterations indicating vulnerability to targeted therapies. Hence, much like in the treatment of adult cancers, molecular profiling is now routinely utilized in clinical workflows for pediatric cancers as a companion to conventional pathology diagnosis. Many variants of unknown significance identified through DNA profiling are being characterized by saturation genome editing, enabled by CRISPR editing technology and clever functional assays. Newer technologies and analytics are revealing additional structural complexity around cancer drivers and gene fusions in pediatric cancer DNA. Similarly, computational methods such as rare variant association studies and polygenic risk scoring are being used to identify novel cancer susceptibility. Together, these advances are expanding our understanding of pediatric cancer's complexity and fueling the development of emerging methods such as liquid biopsy-based monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae78e85702756670907089a42238ceb27fd06f7" target='_blank'>
              Pediatric Cancer Genetics and Genomics.
              </a>
            </td>
          <td>
            Elaine R. Mardis
          </td>
          <td>2025-05-28</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="
 International efforts focused on discovery of biological targets in inflammatory breast cancer (IBC) have suggested that aberrant inter-cellular relationship instead of genomic aberrations is the key driver of IBC. To identify such IBC-specific aberrations, we generated single nucleus chromatin accessibility and transcriptome atlas of IBCs (9,628 nuclei from three donors) and compared IBC atlas with a similar atlas of the healthy breast (81,735 nuclei from 92 donors). We recently reported single nucleus atlas of breast tissues of healthy women of diverse genetic ancestry (Nature Medicine in press). In that report, we described markers that identify three major epithelial cell subtypes [Basal-myoepithelial (BM), luminal adaptive secretory precursor (LASP), luminal hormone sensing (LHS)], two endothelial cell subtypes, two adipocyte subtypes, fibroblasts, macrophages, and T cells of the healthy breast. We also showed that LHS cells are the likely cells-of-origin of Luminal A, Luminal B and HER2+ breast cancers. Unlike these breast cancer subtypes, IBC does not appear to have a cell-of-origin, as epithelial cells in IBCs shared gene expression pattern across BM, LASP and LHS cells. Similar results were obtained when comparison was restricted to ancestry-specific healthy breast atlas. However, individual gene level expression differences affecting specific signaling pathways were observed in epithelial cells of IBCs. LASP, and LHS cells of IBCs overexpressed GPR137C, a positive regulator of mTORC1 signaling, and HS6ST3, an enzyme required for heparan sulfate synthesis. LASP and LHS cells of IBCs displayed activation of mitotic and matrix metalloproteinase signaling, respectively. Endothelial cells of IBCs, which showed significant gene expression differences compared to endothelial cells of the healthy breast, displayed enhanced integrin cell surface interaction but loss of cell junction organization. Most critically, BM, LASP, LHS, and endothelial cells of IBCs compared to their counterparts in the healthy breast showed downregulation of TEX14, an inter-cellular bridge forming factor in germ cells and a regulator of mitosis. Downregulation of TEX14 expression in these cells was accompanied with changes in the chromatin accessibility patterns of this gene. These results reinforce the notion that defective inter-cellular communication is a hallmark of IBC and TEX14 expression levels may serve as a biomarker of this defect. We suggest that greater attention to vascular biology and vascular-epithelial cell communication has to be given for better understanding of IBC biology and therapeutic targeting.
 Citation Format: Harikrishna Nakshatri, Poornima Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu. Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-05-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4562fd16c140b8e77bf1f62e51f30b753052c4ad" target='_blank'>
              Abstract P3-05-23: Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer
              </a>
            </td>
          <td>
            H. Nakshatri, P. Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated member of the RAS family of small GTPases (RAS). It affects about one-fifth of cancer cases. The tumor microenvironment (TME) is a multifaceted network of immune cells, metabolites, microbiota, stromal components, and extracellular matrix. It creates a dynamic ecosystem that supports malignant initiation, progression, and therapy resistance through bidirectional crosstalk with tumor cells. Emerging evidence reveals distinct TME landscapes shaped by wild-type versus oncogenic KRAS variants. Additionally, TME rewiring occurs during KRAS-targeted therapies. Deciphering these KRAS-dependent TME architectures and their therapeutic vulnerabilities represents a critical frontier for precision oncology. This review synthesizes key milestones and persistent challenges in KRAS inhibitor development. And it systematically evaluates how KRAS mutations orchestrated immunosuppressive niches, metabolic symbiosis, stromal remodeling, and microbiome dysbiosis, supported by mechanistic insights from preclinical and clinical studies. It further explores therapeutic opportunities arising from targeting TME interactions, including rational combinations of KRAS inhibitors with immune checkpoint blockade, metabolic agents, or microbiota-modulating strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2718254ccab31c0317c655b38e3998d0c1a62d" target='_blank'>
              Mechanistic Foundations of KRAS-Driven Tumor Ecosystems: Integrating Crosstalk among Immune, Metabolic, Microbial, and Stromal Microenvironment.
              </a>
            </td>
          <td>
            Jiayao Ma, Shenao Fu, Jun Tan, Ying Han, Yihong Chen, Xiangying Deng, Hong Shen, Shan Zeng, Yinghui Peng, Changjing Cai
          </td>
          <td>2025-06-09</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cytosine base editors enable programmable and efficient genome editing using an intermediate featuring a U•G mismatch across from a DNA nick. This intermediate facilitates two major outcomes, C•G to T•A and C•G to G•C point mutations, and it is not currently well-understood which DNA repair factors are involved. Here, we couple reporters for cytosine base editing activity with knockdown of 2015 DNA processing genes to identify genes involved in these two outcomes. Our data suggest that mismatch repair factors facilitate C•G to T•A outcomes, while C•G to G•C outcomes are mediated by RFWD3, an E3 ubiquitin ligase. We also propose that XPF, a 3’-flap endonuclease, and LIG3, a DNA ligase, are involved in repairing the intermediate back to the original C•G base pair. Our results demonstrate that competition and collaboration among different DNA repair pathways shape cytosine base editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc2996866bb9608e216151dc0142fedfafc78c6d" target='_blank'>
              Elucidating the genetic mechanisms governing cytosine base editing outcomes through CRISPRi screens
              </a>
            </td>
          <td>
            Sifeng Gu, Z. Bodai, Rachel A Anderson, Hei Yu Annika So, Quinn T. Cowan, Alexis C. Komor
          </td>
          <td>2025-05-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are known to be involved in the manifestation of aggressive and therapy-resistant phenotypes in solid tumors. Nevertheless, the effects of dynamic intervention by TAMs on the DNA damage response of cancer cells are largely unexplored. Herein, we report that TAMs modulate the DNA damage repair pathways of ovarian cancer cells in response to platinum-(Pt) based therapeutic regimen. We demonstrate that coculture of TAMs with cancer cells directly upregulate Pol η, along with RAD18 and REV1 of the Translesion DNA synthesis (TLS) pathway, while concurrently downregulating components of the high-fidelity nucleotide excision repair (NER) mechanism. Consequently, we observed a better survival probability, DNA repair capacity, and enrichment of stemness properties in ovarian cancer cells. DNA bulky adducts produced by cisplatin are resolved through differential activation NER and TLS pathways. However, we elucidated that TAMs provide favorable conditions for activating the error-prone TLS pathway for lesion bypass over damage resolution. Furthermore, cellular crosstalk in cocultured cancer cells stimulates the nuclear translocation and expression of RelA, which recruits Pol η by acting as a potent transcription factor. In fact, with pristimerin-mediated disruption of p65 (RelA) translocation, the cancer cells become more prone to DNA damage-induced cell death and compromised regenerative potential. In both in vitro cell cultures and in vivo mouse xenograft models, cocultured macrophages exhibited predominantly M2-like phenotype with prevalence in the invasive zone of xenograft tumor margins. Taken together, our investigation revealed multifaceted crosstalk-mediated regulation of DNA damage repair between TAMs and ovarian cancer cells. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05731-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df732d214e8a9fdb550d6752112eedb8ef7093af" target='_blank'>
              Tumor-associated macrophages contribute to cisplatin resistance via regulating Pol η-mediated translesion DNA synthesis in ovarian cancer
              </a>
            </td>
          <td>
            Bilash Chatterjee, Mrinmoy Sarkar, Debanjana Ghosh, Sangita Mishra, Subhankar Bose, Md Maqsood Ahamad Khan, Senthil Kumar Ganesan, Nabanita Chatterjee, A. K. Srivastava
          </td>
          <td>2025-05-29</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer is a major global health challenge, with an estimated 19.3 million new cases by 2025, largely due to lifestyle changes, environmental factors, and increased life expectancy. Despite advances in conventional therapies, issues such as drug resistance, systemic toxicity, and relapse continue to hinder long‐term survival rates. One underexplored molecular mechanism potentially contributing to cancer progression is the enzyme Human tRNA‐dihydrouridine synthase 2 (hDus2). hDus2 plays a pivotal role in tRNA modification by converting uridine residues to dihydrouridine, enhancing tRNA structural flexibility and translational efficiency. Elevated dihydrouridine levels in tRNA have been correlated with cancer progression, highlighting hDus2 as a potential therapeutic target. In this study, we investigated the inhibition of hDus2 as a novel strategy for cancer therapy. Using in silico techniques, we screened a library of FDA‐approved drugs against the 3D structure of hDus2. Top candidate compounds identified through virtual screening with the Glide module were further evaluated using molecular dynamics simulations. Parameters such as RMSD, RMSF, Rg, SASA, and PCA were analyzed. Additionally, binding free energy calculations were performed to validate the stability and interaction strength of the hDus2 ligand complexes. Our results identified DB09050, a cephalosporin antibiotic used for treating complicated intra‐abdominal infections, and DB00650, a folate analog, as promising hDus2 inhibitors. These findings pave the way for further development of anticancer therapeutics post‐experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a55c7e0fe1d71c7c054d254cdfb39eedafd941" target='_blank'>
              Repurposing of FDA‐Approved Drugs Against Leukemia by Targeting hDus2
              </a>
            </td>
          <td>
            Anshuman Chandra, Monika Mukhija, Kanika Devi, Susmita Chaudhuri, Paras Nath Yadav, Vijay Kumar Goel
          </td>
          <td>2025-06-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The cell cycle is a critical part of cellular life, one that has long been studied, both directly, and through its regulatory components. Commonly, cell cycle synchronization or selection experiments are performed in order to study the cell cycle, thus chemically modifying the cells, or selecting them for specific phases. We seek to develop a means to study the cell cycle through the use of single cell RNA sequencing, effectively circumventing the need for such experiments. We utilize a well established pseudotime method, along with the predicted and real expression of genes to calculate the velocity of individual genes. We then utilize statistics and expected biological behaviour to identify genes with significant shifts in velocity within the pseudotime. Additionally we show the ability to observe gene regulatory behaviour such as mRNA splicing and degradation rates. As many cell line based research utilize multiple replicates we implement a merger method for technical replicates to adjust for technical variations, creating a more robust analysis. In summary, our study develops a robust approach to map the velocities of individual, biologically and statistically significant genes throughout the cell cycle’s phases within a cell line experiment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f100f81d203100adf1ed9ffefe6b019843972287" target='_blank'>
              Using pseudotime derivative on single cell RNA sequencing data to identify genes undergoing cell cycle regulation
              </a>
            </td>
          <td>
            Y. Lefol, Geir Amund Svan Hasle, S. Hegre, Helle Samdal, P. Sætrom
          </td>
          <td>2025-05-29</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Research on myeloid neoplasms, a field that has been driving scientific advances in cancer for over 50 years, has yielded many discoveries that have fundamentally reshaped our understanding of cancer biology. These insights, often the product of leukemia research, have been instrumental in developing more mechanism-based treatments in the early 2000s [1]. Recognizing epigenetic dysregulation as a common disease mechanism in myeloid cancers has been groundbreaking regarding recent treatment developments that exploit chromatin-based oncogenic mechanisms. In the case of acute myeloid leukemia (AML), sequencing studies aimed at assessing the complement of genetic alterations demonstrated that more than 60% of AML cases harbored disease-driving mutations in epigenetic regulators. This high prevalence underscores the importance of epigenetic dysregulation in AML pathogenesis [2, 3]. Chromatin regulators commonly control disease-specific transcriptional programs, making them attractive therapeutic targets to manipulate neoplastic gene expression programs, particularly in myeloid neoplasms. Several drugs targeting epigenetic mechanisms and exploiting myeloid disease-specific dependencies have recently been approved for treating myeloid neoplasms. Many additional drugs are currently being investigated in clinical trials, and numerous new compound developments are being studied in preclinical studies. This manuscript will review (1) chromatin-based disease mechanisms, such as DNA methylation, chromatin regulatory complexes, and histone modifications, currently investigated for therapeutic exploitation in myeloid malignancies, and (2) therapeutic developments already approved or investigated for treating these diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9903badc85679a15da9a5a2e6c4c1cc4ffc6496" target='_blank'>
              The evolving landscape of epigenetic target molecules and therapies in myeloid cancers: focus on acute myeloid leukemia and myeloproliferative neoplasms.
              </a>
            </td>
          <td>
            Michael W M Kühn, Naveen Pemmaraju, F. Heidel
          </td>
          <td>2025-05-15</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Aberrant function or overactivation of exonuclease 1 (EXO1) may be associated with cancer tumor development, drug resistance, and response to immunotherapy in female-related cancers. Methods By analyzing RNA-sequencing data from The Cancer Genome Atlas database, combined with validation through quantitative polymerase chain reaction experiments, we explored the expression levels of EXO1 in breast cancer (BRCA) cell lines and assessed its multidimensional roles in various female-related cancers. Results Our experiments revealed elevated expression of EXO1 in BRCA cell lines, consistent with the RNA-sequencing data. The high expression of EXO1 is associated with poor prognosis in various female-related cancers, especially in BRCA and UCEC. It significantly correlates with clinical and pathological characteristics. In specific cancer subtypes like the basal-like subtype of BRCA, high EXO1 expression is associated with a better prognosis. Genetic mutation analysis indicates a higher frequency of EXO1 gene mutations in uterine sarcoma and BRCA. DNA methylation levels may play a role in the regulation of EXO1 gene expression in some cancers. EXO1 expression is correlated with various factors within the tumor immune microenvironment and may be associated with the sensitivity to anticancer drugs. Conclusion EXO1 exhibits multidimensional roles in female-related cancers as a prognostic biomarker and potentially influences tumor immune therapy responses and drug sensitivities. Further studies are needed to fully understand the complex mechanisms underlying these associations and to explore potential therapeutic strategies targeting EXO1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-replicative herpes simplex virus type 1 (nrHSV-1) vectors are promising delivery vehicles for gene therapy due to their large DNA payload capacity and ability to infect a broad range of cell types. However, the genomic deletions made to generate such nrHSV-1 vectors can result in undersized genomes that trigger genomic instability—including rearrangements and size extensions—compromising their therapeutic potential. This study investigates the stabilization of undersized nrHSV-1 vectors through the insertion of stuffer DNA segments. We assess genomic stability, productivity, toxicity, and transgene expression in vitro and in vivo. Our findings demonstrate that nrHSV-1 can accommodate variations in genome size up to 5–6% and highlight the importance of maintaining a genome size close to that of the wild-type HSV-1 for enhanced genomic stability and sustained transgene expression without adverse effects. This strategy offers a promising approach for optimizing nrHSV-1 vectors for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63984927847424424cc3892babc095b69c28460a" target='_blank'>
              Genomic Size Is Critical to Guarantee the Genomic Stability of Non-Replicative HSV1 Vectors
              </a>
            </td>
          <td>
            Justine Basset, Alexandra Seraffin, Julien Ratelade, Yohann Dickx, Tomasz Benedyk, Grzegorz Sarek, Teddy Jégu, Alberto L. Epstein
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Potassium ions (K+) within the tumor microenvironment, along with dysregulation of K+ channels, play critical roles in supporting cancer cell survival and preventing their elimination. Directly monitoring changes in K+ homeostasis within cancer cells is invaluable for understanding these processes. However, achieving high selectivity over other biological metal ions, a detection dynamic range that aligns with intracellular K+ levels, and broad accessibility to research laboratories remain technically challenging for current K+ imaging probes. In this study, we report the in vitro selection of the first K+-specific RNA-cleaving DNAzyme and the development of a K+-specific DNAzyme fluorescent sensor with exceptional selectivity, achieving over 1000-fold selectivity against Na+ and more than 100-fold selectivity over other major biologically relevant metal ions. This sensor has an apparent dissociation constant (105 mM) that is close to the intracellular level of K+, and it has a broad detection range from 21 to 200 mM K+. Using this tool, we reveal a progressive decline in intracellular K+ levels in breast cancer cells with more advanced progression states. Moreover, we demonstrate that elevated extracellular K+ levels interfere with the efficacy of anticancer compounds like ML133 and Amiodarone, suggesting an underappreciated role of microenvironmental K+ in chemoresistance. Notably, blocking the Kir2.1 channel activity restored treatment sensitivity, presenting a potential strategy to overcome chemoresistance in aggressive cancers. These findings underscore the role of K+ homeostasis in tumor progression and support further exploration of ion-channel-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15a5b2af9bd939cc77bcb6b3ee460a1e0556b12" target='_blank'>
              Decoding Potassium Homeostasis in Cancer Metastasis and Drug Resistance: Insights from a Highly Selective DNAzyme-Based Intracellular K+ Sensor.
              </a>
            </td>
          <td>
            Zhenglin Yang, Xiangli Shao, Yuting Wu, Aritra Roy, Elijah Garcia, Annie Farrell, Shreestika Pradhan, Weijie Guo, Heather Gan, Zeynep Korkmaz, Emily Adams, Yi Lu
          </td>
          <td>2025-05-14</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 The Mismatch Repair (MMR) pathway, a crucial tumor suppressor mechanism, is dysregulated across cancer types. We previously identified the loss of the MutL complex of the MMR pathway, as a key factor inducing resistance to endocrine therapy in Estrogen Receptor positive (ER+) breast cancer patients. Preliminary data previously generated in the lab supports a role for MutL loss in promoting metastasis of ER+ breast cancer. A mammary-specific knockout mouse model for MLH1, a principal component of MutL, created in the Haricharan lab develops aggressive mammary tumors that metastasize to multiple organs. To unveil the mechanism by which MLH1 loss triggers metastasis, an unbiased RPPA and RNA-Seq was conducted which revealed cytokine secretion in the immune system as a distinctive signature of MLH1- ER+ breast cancer cells. The tumor secretome, particularly cytokines, can manipulate macrophage plasticity to skew polarization states towards either an anti- or pro- metastatic phenotype. Secreted cytokine levels altered by MLH1 loss were identified in multiple ER+ breast cancer cells lines from concentrated conditioned media. Additionally, in vivo experiments with xenograft MLH1- ER+ tumors in nude mice, revealed an elevated number of CD206 expressing tumor-promoting M2 macrophages in the tumor microenvironment. Furthermore, MLH1- ER+ breast cancer cells grown in the presence of macrophages exhibit an increase in their migratory capacity. Our preliminary data strongly indicate that loss of MLH1 alters the ER+ breast cancer secretome and alters macrophage maturation, suggesting a role for MLH1 in immunomodulation supporting metastatic progression. Investigation of this new role will uncover key tumor-intrinsic drivers of “cold” tumor immune microenvironment profiles and present unique opportunities for therapeutic interventions to convert them into “hot” profiles, thereby improving ER+ breast cancer patient outcomes.
 Citation Format: Megha Raghunathan, Aloran Mazumder, Sabrina Carrell, Daniel Lozano, Nindo Punturi, Svasti Haricharan. MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f9ff414c291cb2d359a6c911b717a2e6733c88" target='_blank'>
              Abstract P1-07-14: MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity
              </a>
            </td>
          <td>
            M. Raghunathan, A. Mazumder, Sabrina Carrell, D. Lozano, Nindo B Punturi, S. Haricharan
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutation spectra vary across genetic and environmental contexts, leading to differences between and within species. Most research on mutation spectrum has focused on the trinucleotide (3-mer) mutation types in mammals, limiting the breadth and depth of variation surveyed. In this study, we use whole-genome resequencing data across 108 eukaryotic species – including mammals, fish, plants, and invertebrates – to characterize pentanucleotide (5-mer) non-coding mutation spectra using a Bayesian approach. Our findings reveal cytosine transition mutability at CpG and (among plants) at CHG sites as the main drivers of variation in mutation spectra across eukaryotes, correlating strongly with genomic CpG and CHG depletion. However, despite the influence of methylation on CpG mutability, genome-wide average CpG methylation levels do not predict CpG transition rates across species and CHG methylation does not predict CHG transition rate, indicating unknown genetic or environmental factors influencing mutation rates at methylated cytosines. Together, our results illustrate the pivotal role of mutagenesis in shaping genome composition across eukaryotes and highlight a gap in knowledge about the mechanisms governing mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840be1a2fe72dda5c8b163dbfd7199ef76a61434" target='_blank'>
              Methylation-associated mutagenesis underlies variation in the mutation spectrum across eukaryotes
              </a>
            </td>
          <td>
            Fabián Ramos-Almodóvar, Ziyue Gao, Benjamin F. Voight, I. Mathieson
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ABSTRACT Introduction Despite decreasing trends in incidence, colorectal cancer (CRC) is still a major contributor to malignancy-related morbidities and mortalities. Groundbreaking advances in immunotherapies and targeted therapies benefit a subset of CRC patients, with sub-optimal outcomes. Hence, there is an unmet need to design and manufacture novel therapies, especially for advanced/metastatic disease. KRAS, the most highly mutated proto-oncogene across human malignancies, particularly in pancreatic adenocarcinoma, non-small cell lung cancer, and CRC, is an on-off switch and governs several fundamental cell signaling cascades. KRAS mutations not only propel the progression and metastasis of CRC but also critically modulate responses to targeted therapies. Areas covered We discuss the impacts of KRAS mutations on the CRC’s tumor microenvironment and describe novel strategies for targeting KRAS and its associated signaling cascades and mechanisms of drug resistance. Expert opinion Drug development against KRAS mutations has been challenging, mainly due to structural properties (offering no appropriate binding site for small molecules), critical functions of the wild-type KRAS in non-cancerous cells, and the complex network of its downstream effector pathways (allowing malignant cells to develop resistance). Pre-clinical and early clinical data offer promises for combining KRAS inhibitors with immunotherapies and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3217dae134c4119cc94589ae992593aeb93f60c2" target='_blank'>
              KRAS mutations in colorectal cancer: impacts on tumor microenvironment and therapeutic implications
              </a>
            </td>
          <td>
            Anita Emami, Pouya Mahdavi Sharif, Nima Rezaei
          </td>
          <td>2025-05-04</td>
          <td>Expert Opinion on Therapeutic Targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Recent large-scale sequencing studies reveal that up to 18% of children with cancer harbor pathogenic variants (PV) in known cancer predisposing genes (CPG). However, many patients exhibit features of an underlying predisposition but lack CPG PV. This observation suggests the presence of unidentified cancer predisposing variants and/or genes. To address this gap, we interrogated genes implicated in DNA damage repair (DDR) as a source of novel cancer predisposing signals that could account for this missing heritability. Methods: We examined germline data from 5,993 childhood cancer cases and 14,477 adult non-cancer controls for damaging variants among 189 DDR genes. Damaging variants were defined as rare (allele frequency <0.05% in the gnomAD v2.1 non-cancer subset), nonsense, frameshift, canonical splice site, and missense with REVEL scores >0.7. Enrichment of damaging variants was determined using logistic and firth regression models. Putative novel CPG were replicated using data from 1,494 additional childhood cancer cases in three independent cohorts. Findings: Pan-cancer analyses revealed enrichment of germline TP53 variants (false discovery rate [FDR]logistic=0.0066, FDRFirth=0.0064). Cancer-specific analyses demonstrated significant associations for TP53 in adrenocortical carcinoma (37%, FDRlogistic<0.0001, FDRFirth=0) and high-grade glioma (2.4%, FDRlogistic=0.0022, FDRFirth=0.1082), as well as BARD1 in neuroblastoma (1.2%, FDRlogistic=0.0341, FDRFirth=0.2682). Three novel gene-tumor associations were identified, including POLL in Ewing sarcoma (1.7%, FDRlogistic=0.0319, FDRFirth=0.3101), SMC5 in medulloblastoma (1.6%, FDRlogistic=0.0005, FDRFirth=0.0499) and SMARCAL1 in osteosarcoma (2.6%, FDRlogistic=0.0250, FDRFirth=0.2180). Among these new putative CPG, enrichment of SMARCAL1 PV in osteosarcoma was replicated across all three additional pediatric cancer cohorts (2.5%, PFisher <0.0001). All osteosarcoma tumors (n=3) with whole genome sequencing data available exhibited deletion of the wild-type SMARCAL1 allele. Interpretation: We report a catalogue of germline SMARCAL1 variants in pediatric osteosarcoma and demonstrate potential molecular effects in tumor formation through the presence of second somatic SMARCAL1 hits. The contribution of germline POLL and SMC5 variants in Ewing sarcoma and medulloblastoma, respectively, warrants further investigation. Our study highlights the power of unbiased genomic investigations to uncover novel CPG, providing insights into tumor biology and creating opportunities for therapeutic intervention, surveillance, and prevention. Funding: Funding was provided by the American Lebanese Syrian Associated Charities and US National Institutes of Health, German Cancer Research Center, German Cancer Consortium.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbfb15fc6aaa9b4386af305cef98303e2f941df" target='_blank'>
              Comprehensive investigation of DNA damage repair genes in children with cancer identifies SMARCAL1 as novel osteosarcoma predisposition gene
              </a>
            </td>
          <td>
            Ninad Oak, Wenan Chen, Alise K Blake, Lynn Harrison, Martha O'Brien, C. Previti, Gnanaprakash Balasubramanian, K. Maass, S. Hirsch, Judith Penkert, Barbara C. Jones, Kathrin Schramm, Michaela Nathrath, K. Pajtler, D. T. Jones, Olaf Witt, U. Dirksen, Jiaming Li, Y. Sapkota, Kirsten K. Ness, Lillian M Guenther, Stefan M. Pfister, C. Kratz, Zhaoming Wang, Greg T Armstrong, M. Hudson, Gang Wu, R. Autry, Kim E. Nichols, Richa Sharma
          </td>
          <td>2025-05-13</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Isoform switching in cancer is a prevalent phenomenon with significant implications for immunotherapy, as actionable neoantigens derived from these cancer-specific events would be applicable to broad categories of patients, reducing the necessity for personalized treatments. By integrating five large-scale transcriptomic datasets comprising over 19,500 samples across 29 cancer and 54 normal tissue types, we identified cancer-associated isoform switching events common to multiple cancer types, several of which involve genes with established mechanistic roles in oncogenesis. The presence of neoantigen-containing peptides derived from these transcripts was confirmed in broad cancer and normal tissue proteome datasets and the binding affinity of predicted neoantigens to the human leukocyte antigen (HLA) complex via molecular dynamics simulations. The study presents strong evidence that isoform switching in cancer is a significant source of actionable neoantigens that have the capability to trigger an immune response. These findings suggest that isoform switching events could potentially be leveraged for broad immunotherapeutic strategies across various cancer types. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f67b4f15a19d7345c9bb37824d41a5eb5aa238" target='_blank'>
              Potential shared neoantigens from pan-cancer transcript isoforms
              </a>
            </td>
          <td>
            Jirapat Techachakrit, Aijaz Ahmad Malik, T. Pisitkun, S. Sriswasdi
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 e22604


 Background:
 The success of PARP inhibitors in BRCA1/2 mutant tumors represents the convergence of germline and somatic molecular profiling as part of routine management of clinical cancer care. Other predictive molecular alterations with matched precision oncology drugs in the germline setting have also emerged. The expansion of germline genetic testing to guide therapy selection has increasingly shifted the burden of germline cancer risk assessment to the point-of-care oncologists who now have the additional challenge of ensuring appropriate interpretation of genetic test results. We therefore sought to expand OncoKB, MSK’s FDA-recognized precision oncology knowledge base, to incorporate annotations for germline variants by partnering with MSK’s Clinical Genetics and Diagnostic Molecular Genetics (DMG) services to provide clinicians and researchers worldwide with an automated, evidence-based tool for interpreting the pathogenicity and clinical actionability of germline variants.
 Methods:
 We reviewed data compiled by MSK’s DMG for over 2,600 unique germline variants in known cancer susceptibility genes. For each variant, we standardized information on pathogenicity, penetrance, mechanism of inheritance, and associated inherited syndromes to enable integration into OncoKB. Using the OncoKB therapeutic levels of evidence framework, we assigned clinical actionability to individual germline variants based on supporting evidence as to whether the variant is predictive of response to standard care or investigational targeted therapy. Additionally, we expanded the OncoKB API to include endpoints for germline variants, enabling the programmatic annotation of germline sequencing reports and research cohorts.
 Results:
 To date, OncoKB has incorporated information on 2631 unique germline variants across 91 genes implicated in germline cancer predisposition and detected by MSK-IMPACT. 24 genes across >10 cancer types are considered Level 1 standard care biomarkers in the germline setting. To date, more than 3400 patients in the MSK-IMPACT cohort have germline alterations that can be annotated with germline OncoKB.
 Conclusions:
 OncoKB’s new annotation of germline variants provides a novel and easily accessible tool for clinicians who utilize germline genetic testing to programmatically determine the effect of the variant based on MSK clinical expertise. Public release of this data is expected in 2025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f63c0ca8de2d3faffe2962c69f406fbdcff953" target='_blank'>
              OncoKB: Automated annotation of the pathogenicity and treatment implications of germline mutations in cancer.
              </a>
            </td>
          <td>
            Moriah H. Nissan, Nikita Mehta, Hongxin Zhang, Amanda Dhaneshwar, Margaret Sheehan, Y. Kemel, Calvin Lu, S. Suehnholz, Nicole Fernandez, Hyeonjin Park, Yirong Li, Ciyu Yang, Bree Martin, Y. Murciano-Goroff, Lauren Banaszak, D. Solit, N. Schultz, Z. Stadler, D. Mandelker, D. Chakravarty
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>126</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90139c8e3ae8545f259aeac6842fc6fa1b255360" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC) is characterized by a complex tumor microenvironment (TME) that drives aggressive progression and treatment resistance. Previous studies have highlighted the roles of cancer-associated fibroblasts (CAFs) and exhausted T (Tex) cells in MIBC, but their interactive mechanisms remain poorly understood. Here, single-cell RNA sequencing of 19 tissue samples from 12 patients—7 MIBC, 3 non-muscle-invasive bladder cancer (NMIBC), and 9 normal tissue samples—identified 13 transcriptionally distinct fibroblast clusters and 10 functionally heterogeneous T-cell subsets. Two interferon (IFN)-responsive fibroblast populations, F-ISG15 (inflammatory CAFs) and F-POSTN (myofibroblastic CAFs), were shown to predominate in the MIBC TME. In vivo experiments demonstrated that IFN-γ secreted by Tex cells polarizes CAFs to secrete CXCL12, which recruits CXCR4-expressing T cells via the CXCL12-CXCR4 chemotactic axis. Spatial analysis revealed a bidirectional loop: Tex-derived IFN-γ sustains CAF activation, whereas CAF-secreted CXCL12 amplifies Tex infiltration. Clinically, activated CAF signatures correlate with advanced disease stages and reduced patient survival in MIBC. These findings establish CXCL12 and IFN signaling as critical therapeutic targets, offering new strategies to disrupt immunosuppressive TME crosstalk and improve outcomes for MIBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d57b4cb9a722a4d3750ff4029d47e49abac05b92" target='_blank'>
              Single-cell RNA sequencing reveals a fibroblast gene signature that promotes T-cell infiltration in muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Zige Liu, Xingning Mao, Yuli Xie, Yunkun Yan, Xiang Wang, Jun-Guo Mi, Hao Yuan, Jiange Zhang, Caisheng Huang, Jianxin Chen, Mujia Jili, Shengzhu Huang, Qingyun Zhang, Fubo Wang, Zengnan Mo, Rirong Yang
          </td>
          <td>2025-05-03</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c852431f33c26dac7a0e107c7a3c701f1454c0" target='_blank'>
              Epigenetic variation can promote adaptation by smoothing rugged fitness landscapes
              </a>
            </td>
          <td>
            Akshat Mall, Christopher J. Marx, Jeremy A. Draghi
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 9520


 Background:
 HRDsig can aid in identifying tumors with DNA repair deficiencies, guiding PARP inhibitor (PARPi) use. It expands treatment options beyond breast cancer gene (BRCA) mutations enabling personalized therapy. Combining PARPi with targeted therapy or immunotherapy shows promise in overcoming resistance and improving outcomes for treatment-resistant ACM.
 Methods:
 Formalin-fixed paraffin-embedded (FFPE) primary tumors and metastatic biopsies from 9,576 ACM cases were analyzed using hybrid capture-based comprehensive genomic profiling (CGP), evaluating all classes of genomic alterations (GA). Central pathology review was conducted for all cases. Microsatellite instability-high (MSIH) status, tumor mutational burden (TMB), genomic ancestry, and mutational signatures were derived from sequencing data. HRDsig was assessed using copy number changes and genomic scars. PD-L1 expression was determined via immunohistochemistry [Dako 22C3; tumor proportion score (TPS)]. Statistical comparisons were made using Fisher’s exact test with Benjamini-Hochberg correction.
 Results:
 Among 9,576 ACM cases, 198 (2.1%) were HRDsig positive (HRDsig+). HRDsig+ cases when compared to HRDsig negative (HRDsig-) were older (median age: 69 vs. 67 years; p=0.034), more often female (49.5% vs. 36.2%; p=0.001), and had more GA per tumor (median: 7 vs. 6; p=0.026). HRDsig+ cases were more often of African (5.6% vs. 1.2%; p<0.0001) or American ancestry (8.1% vs. 4.1%; p=0.026) and less often European (84.3% vs. 94.0%; p<0.0001). GA more common in HRDsig+ included
 IGF1R
 (5.1% vs. 1.3%; p=0.003),
 KIT
 (11.6% vs. 5.6%; p=0.004),
 KRAS
 (7.1% vs. 2.7%; p=0.005),
 NF1
 (38.4% vs. 21.4%; p<0.0001),
 RAD21
 (6.7% vs. 3.0%; p=0.037), and
 TP53
 (40.4% vs. 24.3%; p<0.0001). HRDsig- cases exhibited higher TMB (median: 13.8 vs. 6.1; p<0.0001), more frequent TMB >10 mutations/Mb (60.9% vs. 39.9%; p<0.0001), and more UV light exposure trinucleotide signatures (57.3% vs. 36.4%; p<0.0001). GA more common in HRDsig- cases included
 BRAF
 (including V600E) (44.6% vs. 23.2%; p<0.0001),
 CDKN2A
 (49.2% vs. 36.9%; p=0.003),
 NRAS
 (28.0% vs. 11.6%; p<0.0001), and
 TERT
 (74.4% vs. 29.8%; p<0.0001). In both positive and negative groups, PD-L1 low-level expression (1-49% TPS) was comparable (43.3%-39.7%), while BRCA1/2 alterations (<2.0%) and MSIH status (0.0%-0.1%) remained rare.
 Conclusions:
 HRDsig+ status is rare in ACM but more frequent in non-white genomic ancestries. HRDsig+ ACM cases are less likely to have BRAF GA, more likely to have KIT GA and lower TMB levels. These findings may guide the future development of clinical trials employing combination therapies and PARPi in ACM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162f9c9205bbddc1331c24e0a0b790056a2c6e2c" target='_blank'>
              Homologous recombination deficiency signature (HRDsig) in advanced cutaneous melanoma (ACM): A genomic landscape study.
              </a>
            </td>
          <td>
            Nimisha Srivastava, D. Pavlick, E. Sokol, O. Gjoerup, J. Elvin, Ryon P. Graf, Gerald Li, J. Quintanilha, Sanchit Panda, J. Ross, Jade Homsi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="An R-loop is a complex nucleic acid structure consisting of an RNA–DNA hybrid and an associated single-stranded DNA. This structure plays an important role in many biological processes, including gene regulation, DNA replication, and maintenance of genome stability. Under physiological conditions, the R-loop is in a dynamic balance of generation and decomposition, which is a complex process involving multiple mechanisms of its formation, clearance, and regulation. Emerging evidence indicates that R-loops can drive inflammation and participate in immune processes. This review summarizes immune dysregulation resulting from impaired R-loop clearance and analyzes recent advances in computational methods for R-loop regulation, aiming to identify new avenues for developing R-loop-targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bce6b2f2435a1bf918ec22e6cdf3b596fdd7f0" target='_blank'>
              Unraveling R-loops: The hidden drivers of inflammation and immune dysregulation
              </a>
            </td>
          <td>
            Fan Zhang, Dingyun Wang, Guodong Zhao, Dingmao Wang
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>112</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Mutations in TP53 are mutually exclusive with other known drivers of myeloid transformation and define a distinct molecular subtype within de novo Acute Myeloid Leukemia (AML) that is associated with a complex karyotype, resistance to chemotherapy, and poor prognosis. Although TP53 defects are rare in de novo AML, biallelic mutations are a defining molecular feature of erythroleukemia. The genetic alterations that cooperate with defective TP53 to transform erythroid progenitors remain unknown. We found that loss of BAP1 (BRCA1 Associated Protein-1) co-occurs in one-third of patients with TP53-mutated AML, is associated with an erythroid-primed gene expression signature, and confers an additional adverse effect on overall survival. BAP1 is a tumor suppressor involved in the DNA damage response as well as epigenetic regulation through histone H2AK119 de-ubiquitination. While Bap1KO mice develop myelodysplasia with prominent dyserythropoiesis, combined deletion of Bap1 and Trp53 caused transplantable erythroleukemia, and occasionally mixed AML, mirroring the heterogeneity of human disease. Bulk and single-cell RNA-seq coupled to ChIP-seq in hematopoietic progenitors revealed that Bap1 loss triggers a proinflammatory response and cooperates with Trp53 deficiency to transform erythroid-primed multipotent progenitors. Mechanistically, genomic instability led to the development of erythroleukemia, while epigenetic deregulation caused myelomonocytic skewing suggesting a dichotomous and context dependent role for BAP1. We also demonstrate that BAP1 deficient erythroleukemia is dependent on BCL2L1 expression and is sensitive to BCL-xL inhibitors in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/544862deccf569b5878b747c297d81ba58bf024e" target='_blank'>
              Loss of BAP1 defines a unique subtype of TP53-mutated de novo AML and confers sensitivity to BCL-xL inhibitors.
              </a>
            </td>
          <td>
            Jaclyn Andricovich, Coen J. Lap, A. Tzatsos
          </td>
          <td>2025-06-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

Recent advances in CRISPR–Cas9–based genome editing have revolutionized cancer
immunotherapy by enabling precise and efficient T-cell engineering. This review focuses on how
CRISPR-mediated modifications enhance T-cell functionality, highlighting key applications, such
as CAR-T cells and TCR-engineered T-cell therapies. By selectively targeting immune-regulating
genes, CRISPR empowers T cells to recognize and eliminate tumor cells with greater specificity
and reduce off-target effects. This precision opens the door to personalized treatment strategies, allowing
T cells to be tailored to each patient’s unique tumor profile. Furthermore, the ability to
rapidly and systematically edit multiple gene targets has accelerated the development of more potent
and durable T-cell therapies, overcoming some of the current limitations in cancer immunotherapy.
Overall, CRISPR’s transformative potential of CRISPR provides a platform for next-
generation patient-centric cancer treatments, paving the way for improved clinical outcomes and
expanded therapeutic options.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcb955649c8f44c414ba7f048441db68da50f99" target='_blank'>
              Innovation and Implication of CRISPR-Mediated T-cell Engineering in Cancer Immunotherapy: A New Insight
              </a>
            </td>
          <td>
            Syeda Areej Imran, H. Sultan, Muhammad Zubair, S. Ghazanfar, Anas Islam, Syed Mustafizur Rahaman, Sumel Ashique
          </td>
          <td>2025-06-03</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Embryogenesis and tumorigenesis share several key biological characteristics, such as rapid cell proliferation, high plasticity, and immune evasion. This similarity indicates that developmental pathways can be hijacked, leading to the formation of malignant cell states. With regard to this, cancer can be regarded as a stem cell disease. On the contrary, a fetus, in many ways, has similar characteristics to the “ideal tumor”, such as immune evasion and rapid growth. Therefore, deciphering the molecular mechanisms beneath these phenomena will help us to understand the embryonic origins of cancer. This review discusses the relationship between embryogenesis and tumorigenesis, highlighting the potential roles played by DUX4. DUX4 is involved in the activation of the zygote genome and then facilitates the establishment of totipotency in pre-implantation embryos, whereas the misexpression of DUX4 is associated with different types of cancer. Taken together, this indicates that DUX4 performs analogous functions in these two processes and connects embryogenesis and tumorigenesis. Through examining DUX4, this review underscores the importance of developmental mechanisms in cancer biology, suggesting that the insights gained from studying embryonic processes may provide novel therapeutic strategies. As we continue to explore the complex relationship between cancer and embryogenesis, elucidating the role of DUX4 in linking these two processes will be critical for developing targeted therapies that exploit developmental pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f5d78fbcd896e6b005b6f4da73702c30eda3a6" target='_blank'>
              Embryonic Origins of Cancer: Insights from Double Homeobox 4 Regulation
              </a>
            </td>
          <td>
            B. Fu, Hong Ma, Liang Wang, Zhenhua Guo, Fang Wang, Di Liu, Dongjie Zhang
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abrogating the immunoevasive role of the tumor immune microenvironment (TIME) represents a critical yet still elusive challenge in cancer treatment. Progress in this area has been hampered by both technological limitations and incomplete understanding of TIME-dependent immunoevasion mechanisms. We hypothesize that the immune-evasive role of TIME subpopulations—including regulatory T cells, cancer-associated fibroblasts, and tumor-associated macrophages—is critically mediated by hyperconnected Master Regulator Checkpoint (MRC) modules whose aberrant activity, as induced by paracrine signals, can be abrogated or modulated either genetically or pharmacologically. MRCs are primarily composed of transcription and co-transcription factors, acting downstream of surface receptors and signal transduction cascades to control the transcriptional identity and, ultimately, the phenotype of individual TIME subpopulations. Pharmacological inhibition of subpopulation-specific MRC proteins can thus help reprogram the TIME and potentially abrogate or modulate its immunosuppressive state. This paradigm shift, away from single ligand/receptor targeting, is supported by recent algorithmic, experimental, and clinical advances allowing systematic identification of MRCs and their pharmacological modulators using systems immunology-based approaches. Refocusing the deployment of existing tools and experimental methods that have proven successful in tumor cell contexts to identify and validate MRC-targeting agents capable of remodeling the immunosuppressive cell states of the tumor microenvironment can potentially pave the road to novel combination therapy synergizing with immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/992372e5de760c711f403d3ef7aa2e616073ce3d" target='_blank'>
              Systematic identification and targeting of master regulator checkpoints (MRC) governing tumor microenvironment-mediated immune evasion
              </a>
            </td>
          <td>
            Pasquale Laise, G. Bosker, Mariana Babor, Xiaoyun Sun, Stuart Andrews, Lorenzo Tomassoni, Andrea Califano, Mariano J. Alvarez
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa2044a37bf408a983f6463cafdba0b09b74c94" target='_blank'>
              A resource and computational approach for quantifying gene editing allelism at single-cell resolution
              </a>
            </td>
          <td>
            Matthew Ung, Gabriella Angelini, Ruijia Wang, Alyssa Pyclik, J. Lydeard, J. Ferrucio, Michelle I. Lin, Tirtha Chakraborty, Huanying Gary Ge
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0631bffe60efc9e321a6ea4a06deb1f550391138" target='_blank'>
              Clonal effects of the Ras oncogene revealed by somatic mutagenesis in a Drosophila cancer model
              </a>
            </td>
          <td>
            Takuya Akiyama, M. Gibson
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) was the first member of a broad family of growth factor receptors with intrinsic tyrosine kinase activity. While lung cancer commonly exhibits point mutations and minor insertions within the kinase domain, brain tumours typically exhibit high abundance of EGFR and massive internal deletions. These factors led to the widespread use of EGFR and HER2/ERBB2 as targets for anti-cancer treatments. The association between EGFR expression and cancer prognosis is investigated in this review paper. EGFR, when activated by a ligand, starts a sequence of time-dependent molecular switches that govern genes that determine phenotype. These switches include up-regulation of freshly synthesised mRNAs, down-regulation of a large cohort of microRNAs, and covalent protein changes. Long non-coding RNAs and circular RNAs, in addition to microRNAs, are essential for EGFR signalling. EGFR is a multifaceted driver of metastasis, in addition to driver mutations. Targeted therapies against EGFR, including tyrosine kinase inhibitors and monoclonal antibodies, have significantly improved treatment outcomes in cancers such as non-small cell lung cancer and colorectal cancer. In this review, we discussed about how EGFR overexpression and mutations affect different types of human malignancies. Finally, we evaluated all anti-cancer medications that have received clinical approval that target EGFR and discuss recent advancements in therapy as well as in future possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecba6b1c9e10ebcbb52273329d4eb9e943d6838" target='_blank'>
              EGFR in Oncology: A Narrative Review of Mutations, Overexpression and Treatment Strategies
              </a>
            </td>
          <td>
            Sparsha Dey, Rhitoban Ghosh, Sangita Dan, Shrabona Mukherjee, Soma Sett, Palash Dhara, Subhranshu Mandal, Chandan Mandal
          </td>
          <td>2025-05-13</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Segmental duplications (SDs) are blocks of genomic DNA with high sequence homology that are hotspots for chromosomal rearrangements, coinciding with copy-number and single-nucleotide variations in the population. SDs could represent unstable genomic regions that are susceptible to somatic alterations in human cancers. Here, we aimed to elucidate the genomic locations of SDs in relation to cancer-related genes and their propensity for somatic alterations in cancer. The analysis showed that tumor suppressor genes (TSGs) were less associated with SDs compared to non-cancer genes in nearly all mammalian species. TSGs with SDs were larger in size in humans but only modestly conserved among mammals. In humans, the proportion of non-cancer genes with SDs decreased as the gene age increased. However, for TSGs, a loss of association with SDs was apparent among young genes. Pan-cancer analysis revealed that TSGs with SDs were more prone to deletions and structural variations independently of gene size. Re-analysis of publicly available experimental data further revealed that genes with SDs tended to replicate late and were more likely to undergo the error-prone mitotic DNA synthesis upon replication stress. In conclusion, the loss of SDs from TSGs during mammalian evolution protects against tumor formation by reducing somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The transcription factor PAX5 is a major target of genetic alterations in human B-cell precursor acute lymphoblastic leukemia (B-ALL). Among the alterations, the P80R mutation affecting the DNA-binding domain represents the most frequent PAX5 point mutation in B-ALL. In contrast to other somatic PAX5 mutations, PAX5P80R defines a distinct B-ALL subtype characterized by a unique transcriptional program. Here, we aimed to develop a model to elucidate the mechanism by which PAX5P80R perturbs normal B-cell differentiation and the oncogenic relays involved in PAX5P80R-driven malignant progression. A retroviral complementation approach of Pax5-deficient murine fetal liver cells demonstrated at the functional and molecular levels that PAX5P80R failed to rescue definitive B-cell commitment but maintained the repression of T-cell development. Moreover, PAX5P80R eventually led to clonal B-ALL transformation after transplantation through the acquisition of secondary mutations in genes involved in the JAK/STAT and RAS/MAPK pathways. Finally, transcriptomic analyses combined with pharmacological investigation revealed ectopic activation of HIF2α as a common feature of B-ALL and identified acriflavine as a potent drug against B-ALL. Hence, this study provides a strategy to model the multistep process of B-ALL and sheds light on the biological mechanism by which the PAX5P80R mutation leads to leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0667740b0e5a9a582710e0b5a8d5562613a3f309" target='_blank'>
              Modeling the PAX5P80R Mutation Reveals HIF2α Activation as a Common Feature and Therapeutic Target in B-Cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Manon Bayet, Vincent Fregona, Mathieu Bouttier, Clémence Rouzier, Jérémy Bigot, Laura A Jamrog, Sylvie Hebrard, N. Prade, Stéphanie Lagarde, Christine Didier, Stéphanie Gachet, M. Passet, Laetitia Largeaud, Marlène Pasquet, A. Khamlichi, E. Clappier, Eric Delabesse, C. Broccardo, Bastien Gerby
          </td>
          <td>2025-05-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="CRISPR technology has revolutionized genome editing by enabling precise, permanent modifications to genetic material. To circumvent the irreversible alterations associated with traditional CRISPR methods and facilitate research on both essential and nonessential genes, CRISPR interference or inhibition (CRISPRi) and CRISPR activation (CRISPRa) were developed. The gene-silencing approach leverages an inactivated Cas effector protein paired with guide RNA to obstruct transcription initiation or elongation, while the gene-activation approach exploits the programmability of CRISPR to activate gene expression. Recent advances in CRISPRi technology, in combination with other technologies (e.g., biosensing, sequencing), have significantly expanded its applications, allowing for genome-wide high-throughput screening (HTS) to identify genetic determinants of phenotypes. These screening strategies have been applied in biomedicine, industry, and basic research. This review explores the CRISPR regulation mechanisms, offers an overview of the workflow for genome-wide CRISPR-based regulation for screens, and highlights its superior suitability for HTS across biomedical and industrial applications. Finally, we discuss the limitations of current CRISPRi/a HTS screening methods and envision future directions in CRISPR-mediated HTS research, considering its potential for broader application across diverse fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3860c88a16ddc6c01ca95906456a41db1aa0ff54" target='_blank'>
              CRISPR-Based Regulation for High-Throughput Screening.
              </a>
            </td>
          <td>
            Lingling Jiao, Qi Zhou, Dongchang Sun
          </td>
          <td>2025-05-22</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Yang Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accurate and timely detection of clinically relevant genetic abnormalities, such as CBFB::MYH11 or inversion(16) [inv(16)], is critical for the diagnosis and management of patients with acute myeloid leukemia (AML). Notably, CBFB::MYH11 is a disease-defining mutation in AML and is associated with a favorable prognosis. The current standard-of-care workup, which includes a combination of conventional G-banding karyotyping, fluorescence in situ hybridization (FISH), and/or reverse-transcriptase PCR, poses challenges in detecting variant CBFB::MYH11 translocations. High-resolution, genome-wide technologies capable of accurate and unbiased detection of chromosomal structural aberrations at the gene/exon level, such as optical genome mapping (OGM), will be helpful for the timely detection of clinically actionable abnormalities. This case report presents a patient initially diagnosed with therapy-related myelodysplastic syndrome (MDS) following cytotoxic therapy and treated with a hypomethylating agent, who later experienced progression to AML with CBFB::MYH11. Retrospective analysis of the initial diagnostic sample using OGM revealed a cryptic CBFB::MYH11 abnormality at the time of the first presentation. Furthermore, OGM enabled comprehensive characterization of this novel CBFB::MYH11 transcript with noncanonical breakpoints, which were not detected by standard molecular techniques. This case highlights a critical diagnostic blind spot in the detection of CBF::MYH11 AML, representing a missed opportunity to offer effective frontline therapy to a patient with potentially curable AML-an aberration not recognized by conventional karyotype or FISH at the time of initial diagnosis. The implementation of genome-wide technologies such as OGM as a first-tier diagnostic tool in clinical laboratories for the workup of MDS/AML is essential for detecting clinically impactful cryptic genomic alterations. The discovery of this novel alternate CBFB::MYH11 transcript with noncanonical breakpoints underscores a major limitation in current standard-of-care techniques, warranting further prospective studies to evaluate its clinical actionability in guiding personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869695c5b69f60676fbe0fddad777c8effa5e6e6" target='_blank'>
              A Case of Cryptic CBFB::MYH11 Acute Myeloid Leukemia With Noncanonical Breakpoints Detected by Optical Genome Mapping.
              </a>
            </td>
          <td>
            Eduardo Edelman Saul, Samuel Urrutia, Hui Yang, G. Montalban-Bravo, Guilin Tang, Gokce A. Toruner, K. Patel, R. Luthra, C. Bueso-Ramos, Sa A Wang, K. Chien, K. Sasaki, Himachandana Atluri, Hannah Goulart, B. Thakral, G. Garcia-Manero, R. Kanagal-Shamanna
          </td>
          <td>2025-05-09</td>
          <td>Journal of the National Comprehensive Cancer Network : JNCCN</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="The advent of genome-editing technologies, particularly the RNA-guided the clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated system (Cas) 9, which originates from prokaryotic CRISPR/Cas adaptive immune mechanisms, has revolutionized molecular biology. Renowned for its simplicity, cost-effectiveness, and capacity for multiplexed gene editing, CRISPR/Cas9 has emerged as the most versatile and widely adopted genome-editing platform. Its applications span fundamental research, biotechnology, medicine, and therapeutics. This review highlights recent advancements in CRISPR-based technologies, focusing on CRISPR/Cas9, CRISPR/Cas12a, and CRISPR/Cas12f. It emphasizes precision editing methods like base editing and prime editing, which enable targeted nucleotide changes without double-strand breaks. The specificity of these tools, including on-target accuracy and off-target risks, is critically evaluated. Additionally, recent preclinical and clinical efforts to treat diseases such as cancer and sickle cell disease using CRISPR are summarized. Finally, the challenges and future directions of CRISPR-mediated gene therapy are discussed, emphasizing its potential to integrate with other molecular approaches to address unmet medical needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5091bbb50a58b99c0d0d29fe316c867de14c3f" target='_blank'>
              Progress, Applications and Prospects of CRISPR-Based Genome Editing Technology in Gene Therapy for Cancer and Sickle Cell Disease.
              </a>
            </td>
          <td>
            Sha-Sha Zang, Ruirui Zhang, Jia-Run Zhang, Xi Zhang, Jun Li
          </td>
          <td>2025-05-12</td>
          <td>Human gene therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations and gene rearrangements are crucial for the diagnosis and subtyping of acute myeloid leukemia (AML). However, the contribution of non-coding genetic variants, particularly those within cis-regulatory elements (CREs), to AML pathophysiology and heterogeneity remains poorly understood. In this study, we characterize the single-cell chromatin accessibility landscapes of 10 bone marrow samples from AML patients at diagnosis. Additionally, we develop eMut, an integrated computational pipeline for detecting, imputing, and functionally characterizing non-coding mutations in CREs at the single-cell level. Our analysis identifies 2878 potential somatic non-coding mutations, highlighting the extensive mutational heterogeneity in the non-coding genome of AML patients, with recurrent non-coding mutations displaying cell type-specific patterns. We show that mutated CREs are enriched with blood-related genetic variants, potentially linked to AML-associated genes, and harbor a higher abundance of functional CREs, suggesting their functional relevance in leukemogenesis. Importantly, we pinpoint candidate functional non-coding mutations that associate with alteration of target gene expression in AML. Collectively, our work provides a comprehensive resource of single-cell chromatin accessibility in AML and introduces an integrative approach to identify candidate functional non-coding mutations contributing to cellular heterogeneity in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cc3c9c6e3317c1eed73191b4e5e0c7a9dcb592" target='_blank'>
              Discovery of candidate functional non-coding mutations in acute myeloid leukemia using single-cell chromatin accessibility sequencing
              </a>
            </td>
          <td>
            Ming Zhu, Jiali Zhu, Zhijuan Zhu, Yiding Yang, Jinxian Dai, Hua Li, Nainong Li, Jialiang Huang
          </td>
          <td>2025-05-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Endocrine resistance is frequently encountered in estrogen receptor-positive (ER+) breast cancer, often because of somatic mutations such as neurofibromin 1 (NF1) loss. The mechanisms by which ER-directed proliferation is lost in such cases are unknown, limiting the potential use of additional endocrine treatments. Here, we performed CRISPR-Cas9 knockout (KO) screens and found that nuclear receptor subfamily 2 group F member 2 (NR2F2), an orphan nuclear receptor, was essential for NF1 loss-induced endocrine resistance. Induction of NR2F2 was observed in ER+ cell line models and patient samples and occurred via activation of the mitogen-activated protein kinase (MAPK) pathway upon NF1 loss or other MAPK pathway genetic alterations. Mechanistically, increased NR2F2 orchestrated a repressed ER transcriptional program by repartitioning the ER cistrome, altering the balance of its associated transcriptional coregulators, and modifying global chromatin accessibility. Accordingly, genetic depletion or pharmacologic inhibition of NR2F2 restored sensitivity to hormone therapies in multiple models, including ER+ cell lines, patient-derived xenografts, and patient-derived organoid-based xenografts harboring diverse endocrine-resistance mechanisms such as NF1, AT-rich interactive domain-containing protein 1A (ARID1A), phoshatase and tensin homolog (PTEN) loss, or Kirsten rat sarcoma virus (KRAS) overexpression. Together, these findings underscore NR2F2 as a critical modulator of the hormone response pathway and suggest its inhibition as a promising strategy to overcome endocrine resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ad21fc4c5dc5d1f7daf62e5b90b48131d24680d" target='_blank'>
              Inhibition of NR2F2 restores hormone therapy response to endocrine refractory breast cancers.
              </a>
            </td>
          <td>
            Yanyan Cai, P. Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O'Connell, Emma C. Fink, Matthew M. Miele, Zhuoning Li, E. de Stanchina, Emiliano Cocco, P. Razavi, Eneda Toska, S. Fanning, Guotai Xu, A. Sablina, Maurizio Scaltriti, S. Chandarlapaty
          </td>
          <td>2025-06-04</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686f795b4b66276294de9ae162a3558934271a82" target='_blank'>
              A CRISPR-CAS9 high throughput machine-learning platform for modulation of genes involved in Parkinson’s disease-associated PINK1-mitophagy in iPSC-derived dopaminergic neurons
              </a>
            </td>
          <td>
            M. Soutar, Benedetta Carbone, Petya Kindalova, Rahil Mehrizi, Fernanda Martins Lopes, Nathaniel Lam, Alice Rockliffe, Toni Braybrook, Matthew Taylor, Cuong Nguyen, Fiona Ducotterd, Alastair D. Reith, Lisa Mohamet, Helene Plun-Favreau
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="A deficiency in DNA mismatch repair (MMR) leads to microsatellite instability (MSI), which is associated with a favorable response to immune checkpoint inhibitors (ICIs), and the Promega MSI Analysis System is approved as a companion diagnostic tool for it. In this study, we investigated the MMR status in patients with primary cutaneous lymphoma (PCL) diagnosed at our hospital. MSI was found in 1 of the 29 patients (3.4%), an 87-year-old man diagnosed with subcutaneous panniculitis-like T-cell lymphoma. Only the NR-21 marker was present in both tumor and normal tissue, indicating that the MMR status was MSI-low, and he had a germline mutation of SLC7A8. Our study showed that most PCLs are microsatellite stable tumors. This study is a single-center small-sample investigation and requires validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3239fb330da0bf2943086aa15afb249d0b9058f7" target='_blank'>
              Primary cutaneous lymphoma is a microsatellite stable tumor: An analysis of microsatellite instability.
              </a>
            </td>
          <td>
            S. Maeda-Otsuka, Myangat Tselmeg Mijiddorj, I. Kajihara, S. Sawamura, K. Makino, S. Masuguchi, Satoshi Fukushima
          </td>
          <td>2025-06-25</td>
          <td>Drug discoveries & therapeutics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The CRISPR–Cas system has transformed molecular biology by providing precise tools for genome editing and pathogen detection. Originating from bacterial adaptive immunity, CRISPR technology identifies and cleaves genetic material from pathogens, thereby preventing infections. CRISPR–Cas9, the most widely utilized variant, creates double-stranded breaks in the target DNA, enabling genetic disruptions or edits. This approach has shown significant potential in antiviral therapies, addressing chronic infections, such as HIV, SARS-CoV-2, and hepatitis viruses. In HIV, CRISPR–Cas9 edits the essential viral genes and disrupts latent reservoirs, while CCR5 gene modifications render the T cells resistant to viral entry. Similarly, SARS-CoV-2 is targeted using CRISPR–Cas13d to inhibit the conserved viral genes, significantly reducing viral loads. Hepatitis B and C treatments leverage CRISPR technologies to target conserved genomic regions, limiting replication and expression. Emerging innovations, such as the PAC-MAN approach for influenza and base-editing systems to reduce off-target effects, further highlight the therapeutic versatility of CRISPR. Additionally, advances in Cas12a and Cas13 have driven the development of diagnostic platforms like DETECTR and SHERLOCK, which provide rapid and cost-effective viral detection. Innovative tools like AIOD-CRISPR enable accessible point-of-care diagnostics for early viral detection. Experimental approaches, such as targeting latent HSV-1 reservoirs, highlight the transformative potential of CRISPR in combating persistent infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be60256d8ceb6a861f6bef31814f57b4bea13d0d" target='_blank'>
              Progress in CRISPR Technology for Antiviral Treatments: Genome Editing as a Potential Cure for Chronic Viral Infections
              </a>
            </td>
          <td>
            Fatemeh Nouri, Farnaz Alibabaei, Behina Forouzanmehr, Hamed Tahmasebi, Valentyn Oksenych, Majid Eslami
          </td>
          <td>2025-05-20</td>
          <td>Microbiology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the most common cancer diagnosis in women globally, accounting for more than 1 in 10 new cancer diagnoses. While BC has historically been a poor candidate for immune checkpoint blockade (ICB), recent studies in triple-negative BC (TNBC) have demonstrated anti-PD-1 treatment efficacy when employed in conjunction with chemotherapy1. APOBEC3 (A3) enzymes, such as A3A and A3B, catalyze cytosine-to-uracil deamination in single-stranded DNA, leading to signature single base substitution mutations (SBS2 and SBS13). A3 expression and mutagenesis have been associated with poor clinical outcomes, including drug resistance2,3. Murine models for studying ICB and mutagenesis by human A3A and A3B are underdeveloped. Here, we address this gap by generating two independent mammary tumor cell lines and studying responsiveness to ICB. Two novel mammary cancer cell lines, MM001 and MM008, were isolated from spontaneously arising tumors in C57BL/6 MMTV-Cre+;p53fl/+ mice. Each tumor line was passaged serially in the mammary fat pads of C57BL/6 (WT) mice to obtain conditioned cell lines MM001i and MM008i, which formed consistent and reproducible tumors. Tumor immune infiltrates were investigated by immunohistochemistry for CD45 (pan-immune marker), CD3 (pan-T cell), CD8 (cytotoxic T cells), B220 (B cells), Mac-1 or CD11b (macrophages), and PD-L1. Cells were also assessed via flow cytometry for surface expression and interferon inducibility of immune markers PD-L1, MHC-I component H2-Kb, and CD155. Reproducibly growing tumors for each of MM001i and MM008i were achieved, with 50% of the tumors reaching the endpoint of 1000 mm3 around day 18 and day 21, respectively. Interestingly, despite a shared ancestral genetic background, ICB treatment of WT mice injected with MM001i and MM008i revealed differential responses. MM001i was poorly responsive, whereas MM008i showed complete tumor regression following treatment with anti-PD-1. In contrast, both MM001i and MM008i responded rapidly to anti-CTLA-4. These differential responses were not explained by surface expression of PD-L1 and MHC-1 component H2-Kb, which appeared to be basally and inducibly (IFN-β or IFN-γ) expressed at similar levels. Experiments are ongoing to deduce the underlying molecular mechanism, test additional ICB therapies, and study the impact of A3 mutagenesis. Thus, two novel murine mammary breast cancer cell lines have been established to evaluate immunotherapies in fully immune-competent C57BL/6 mice.
 Selected References:
 1) Schmid, P. et. al. Pembrolizumab for Early Triple-Negative Breast Cancer. N Engl J Med, 382, 810-821 (2020).
 2) Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv, 2, e1601737 (2016).
 3) Gupta, A. et. al. APOBEC3 mutagenesis drives therapy resistance in breast cancer. bioRxiv, https://doi.org/10.1101/2024.04.29.591453 (2024).
 Citation Format: Thomas Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, Reuben S. Harris, Harshita B. Gupta. New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627f15e34d39099a44c9ec865f3f46e6de37a2af" target='_blank'>
              Abstract P1-02-25: New Murine Mammary Tumor Cell Lines for Syngeneic Studies Including Immunotherapy
              </a>
            </td>
          <td>
            Thomas J Kalantzakos, Anusha Soni, Cameron Durfee, Chris Mullally, Benjamin Troness, R. Harris, Harshita B. Gupta
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 e13538


 Background:
 Understanding the molecular profile of cancer-associated genes is crucial in oncology, as it informs targeted therapies and improves patient outcomes. Tumor suppressor genes inhibit the activation of oncogenic signals, while oncogenes promote them. Ras proteins, encoded by KRAS, NRAS, and HRAS genes, are involved in cell growth, differentiation, and survival pathways. This study aims to investigate the frequency of KRAS, NRAS, and HRAS mutations in solid tumors treated at a single academic institution between January 2017 and May 2024. By analyzing these mutations, we seek to enhance our understanding of their distribution across different cancer types in Venezuela, providing valuable insights for personalized cancer treatment strategies.
 Methods:
 We conducted a retrospective, cross-sectional, analytical cohort study. The study included 175 patients, with 179 molecular genetic sequencing studies and 20 RT-PCR analyses. We examined mutations in codons 12, 13, 59, 61, 117, and 146 of the KRAS, NRAS, and HRAS genes.
 Results:
 We identified 43 mutations in the RAS gene family: 37 in KRAS, 5 in NRAS, and 1 in HRAS. Lung cancer accounted for 14 of those mutations, with 13 in KRAS and 1 in HRAS. Colorectal cancer showed 19 mutations, with 16 in KRAS and 3 in NRAS. Other KRAS mutations (13.9%) were found in breast, endometrial, pancreatic cancers, and thymoma. We also recorded 2 NRAS mutations in melanoma and 1 in pancreatic cancer.
 Conclusions:
 This study represents the largest collection of KRAS, NRAS, and HRAS mutations reported in Venezuela.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d60941cc22a3a39b9e5850c1cc1b772e65a8cda" target='_blank'>
              Frequency of RAS family gene mutations in metastatic solid tumors in Venezuela: Experience at a large academic center.
              </a>
            </td>
          <td>
            Guillermo Borga, Oscar A. Sucre Astorga, Santiago Sucre, C. Sucre
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rare genetic DNA repair deficiency syndromes can lead to immunodeficiency, neurological disorders, and cancer. In the general population, inter-individual variation in DNA repair capacity (DRC) influences susceptibility to cancer and several age-related diseases. Genome wide association studies and functional analyses show that defects in multiple DNA repair pathways jointly increase disease risk, but previous technologies did not permit comprehensive analyses of DNA repair in populations. To overcome these limitations, we used fluorescence multiplex host cell reactivation (FM-HCR) assays that directly quantify DRC across six major DNA repair pathways. We assessed DRC in phytohemagglutinin-stimulated primary lymphocytes from 56 healthy individuals and validated assay reproducibility in 10 individuals with up to five independent blood draws. We furthermore developed generalized analytical pipelines for systematically adjusting for batch effects and both experimental and biological confounders. Our results reveal significant inter-individual variation in DRC for each of 10 reporter assays that measure the efficiency of distinct repair processes. Our data also demonstrate that correlations between the activities of different DNA repair pathways are relatively weak. This finding suggests that each pathway may independently influence susceptibility to the health effects of DNA damage. We furthermore developed a pipeline for analyzing comet repair kinetics and related our new functional data to previously reported comet assay data for the same individuals. Our pioneering analysis underscores the sensitivity of FM-HCR assays for detecting subtle biological differences between individuals and establishes standardized methodologies for population studies. Our findings and open source analytical tools advance precision medicine by enabling comprehensive exploration of genetic, demographic, clinical, and lifestyle factors and supporting targeted interventions to enhance DNA repair and maintain genomic integrity, thereby promoting personalized healthcare and disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            T. Zhai, P. Mazzucato, C. Ricciardi, D. C. Christiani, L. Liang, L. D. Samson, I. A. Chaim, Z. D. Nagel
          </td>
          <td>2025-06-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD), caused by bi-allelic germline variants in mismatch repair (MMR) genes, is associated with high cancer incidence early in life. A better understanding of mutational processes driving sequential CMMRD tumors can advance optimal treatment. Here, we describe a genomic characterization on a representative collection of CMMRD-associated tumors consisting of 41 tumors from 17 individuals. Mutational patterns in these tumors appear to be influenced by multiple factors, including the affected MMR gene and tumor type. Somatic polymerase proofreading mutations, commonly present in brain tumors, are also found in a T-cell lymphoblastic lymphoma displaying associated mutational patterns. We show prominent mutational patterns in two second primary hematological malignancies after temozolomide treatment. Furthermore, an indel signature, characterized by one-base pair cytosine insertions in cytosine homopolymers, is found in 54% of tumors. In conclusion, analysis of sequential CMMRD tumors reveals diverse mutational patterns influenced by the affected MMR gene, tumor type and treatment history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c61f9eb1036d6b9dbdcb4d99fcc534a6fcd3f8" target='_blank'>
              Unraveling mutagenic processes influencing the tumor mutational patterns of individuals with constitutional mismatch repair deficiency
              </a>
            </td>
          <td>
            Dilys D Weijers, Snežana Hinić, E. Kroeze, M. Gorris, G. Schreibelt, Sjors Middelkamp, Arjen R Mensenkamp, R. Bladergroen, K. Verrijp, N. Hoogerbrugge, Pieter Wesseling, Rachel S van der Post, Jan L C Loeffen, C. Gidding, M. V. van Kouwen, I. J. D. de Vries, R. van Boxtel, R. D. de Voer, M. Jongmans, Roland P Kuiper
          </td>
          <td>2025-05-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) affects genomic stability and has potential as a biomarker for the effectiveness of poly (ADP-ribose) polymerase (PARP) inhibitors and immune checkpoint inhibitors. However, the clinical and molecular profile of HRD in non-small cell lung cancer (NSCLC), particularly in the Chinese population, remains poorly characterized. Based on the next-generation sequencing data of 158 Chinese NSCLC patients, we analyzed the HRD scores of mutations in homologous recombination repair (HRR) genes and dissected the correlation between HRD state and programmed death-ligand 1 (PD-L1) expression. Alterations in HRR genes were observed in 8.9% of the patients, with ATM and BRCA2 being the most commonly affected genes. HRD-high (HRD-H) status was significantly associated with advanced disease stage (≥III) and lung squamous cell carcinoma (LUSC). Transcriptomic analysis revealed distinct gene expression profiles between HRD-H and HRD-low (HRD-L) subgroups, with HRD-H tumors exhibiting predominantly downregulated genes. While EGFR mutations occurred at similar frequencies across HRD status, TP53 mutations were significantly enriched in HRD-H cases. HRD-H status correlated with higher PD-L1 positivity in NSCLC overall, but not within the lung adenocarcinoma (LUAD) subgroup in our cohort. The Cancer genome atlas analysis showed higher PD-L1 protein expression in HRD-H LUAD, but not in LUSC. Kyoto Encyclopedia of Genes and Genomes analysis identified enrichment of complement and coagulation cascades, ABC transporters, and bile secretion pathways in HRD-H tumors, suggesting links to immune evasion and drug resistance. This study elucidates the genomic landscape of HRD in Chinese NSCLC patients and provides insights into its potential clinical utility for therapeutic targeting. Our findings suggest that integrated HRD scoring may guide the application of PARP inhibitors and immunotherapy in specific NSCLC patient subgroups. Further prospective clinical studies are needed to validate the predictive value of HRD scoring in NSCLC treatment and to optimize patient selection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c403957972723c45a13a1636514a41ce8c5b8d3" target='_blank'>
              Genomic alterations of homologous recombination deficiency in Chinese NSCLC patients
              </a>
            </td>
          <td>
            Shuang Xiang, Changqiong Shen, Chun Huang, Ya-ting Yang, Jing Guo, Yi Liu, Mingzhu Yin, Song Duan
          </td>
          <td>2025-06-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e275fe9c31757491fc33c855fc96ec1cae73962" target='_blank'>
              Offspring effect of maternal preconception exposure to metabolic disruptors
              </a>
            </td>
          <td>
            Carlos Díaz-Castillo, Stephanie R. Aguiar, Raquel Chamorro-Garcia
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS Chemical Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Blood cancers are one of the most common cancers worldwide. These diseases stem from defects in blood components having cytogenetic aberrations and genetic mutations. There have been vast improvements in terms of treatment options and survival outcomes. Nevertheless, due to the clonal nature and heterogeneity of the diseases, the number of cases reported exhibit a rising pattern due to chemoresistance and disease relapse thus posing a healthcare burden. Therefore, the need for more specific forms of targeted therapies is ever-present.


RECENT FINDINGS
CRISPR has emerged as a key player and is the epitome of gene editing technology in this post genomic era. In line with the current trend, numerous studies in blood cancer research have extensively utilized CRISPR-based applications to understand the functional genomics of hematologic malignancies and identify potential therapeutic targets for development of novel therapeutic applications.


SUMMARY
The importance of comprehending the utilities of state-of-the-art technologies such as CRISPR for studying hematologic malignancies has never been more apparent and timelier. Therefore, this review attempts to scrutinize the versatility of CRISPR applications which range from functional genomics to immunotherapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215dcf4713fb2244aa818902ca452469fccb308c" target='_blank'>
              Multifaceted roles of CRISPR technology in blood cancer research.
              </a>
            </td>
          <td>
            Maheswaran Solayappan, Adam Azlan, Kang Zi Khor, Mot Yee Yik, Aswani Jaishanker, Thevendran Ramesh, Mohamed Saleem, N. Yusoff, E. Moses
          </td>
          <td>2025-06-04</td>
          <td>Current opinion in hematology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c7153a387ee13c7f82ab9687192bad17d753d5f" target='_blank'>
              Accurate prediction of gene deletion phenotypes with Flux Cone Learning
              </a>
            </td>
          <td>
            C. Merzbacher, Oisin Mac Aodha, Diego A. Oyarzún
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6b3a707fdcdfce06b9c16b1eadf99680a52469" target='_blank'>
              DNA extrusion size determines pathway choice during CAG repeat expansion
              </a>
            </td>
          <td>
            M. Bhatia, A. Phadte, Anna Lakhina, A. R. Monte Carlo, S. Barndt, A. Pluciennik
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer genomics has advanced a lot; a main problem is still turning molecular findings into treatments due to the variety of cells in each tumor and gaps in knowing certain pathway functions. Predicting whether a treatment will be effective is difficult right now which means that patient outcomes are often less than ideal. This research tackled the issue by studying whether looking at many genes and oncogenic pathways at the same time helps identify which patients may show cancer progression or treatment resistance. For this study, we looked at the genes, proteins and other molecules from a total of 1,500 patients in six major types of cancer (breast, colorectal, lung, melanoma, ovarian and prostate). We used genomic variant calling, analysis of gene pathways and detailed study of proteins with advanced bioinformatics and machine learning tools. The study found that TP53 mutations (OR: 2.14, 95% CI: 1.72-2.66, p < 0.001) and the PI3K/AKT/mTOR pathway being activated (OR: 1.89, 95% CI: 1.51-2.37, p = 0.003) were strongly linked to cancer not responding to treatment and negative outcomes. Tumors with a high number of mutations (≥ 10/Mb) reacted much better to immunotherapy (OR: 3.02, p < 0.001). Three subgroups of tumors were found through unsupervised analysis and these showed different chances of success: tumors with many mutations and high immune activity performed best (39.1% complete response), but those with mutations in PI3K and TP53 had the worst prognosis (28.5% progressive disease). Patients whose tumors were active in the PI3K/AKT pathway had progression-free survival of 8.2 months, significantly less (log-rank p < 0.001) than those whose tumors were not active. Evidence shows that integrative molecular profiling is better at predicting outcomes than the traditional method of classifying tumors by looking at slides. Study results show which molecular subgroups are most important to oncology and help link their treatment response to which therapy is best,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e766178e1eebb59cae34bf9ab40a3c9e125bd68f" target='_blank'>
              Integrative Molecular Profiling of Oncogenic Pathways and Genetic Mutations in Cancer Progression and Therapeutic Response
              </a>
            </td>
          <td>
            Javeria Taj, Muhammad Noman Ajmal, Tayyaba Arshad, Afifa Dawood, Ali Abbas, Misbah Hafeez, Hamza Rafeeq, Abdul Malik, Aiman Nishat, Mujahid Hussain
          </td>
          <td>2025-06-20</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/432419f2032b46caaaf3eb1aa445ee0f4fa38314" target='_blank'>
              Clonal tracing with somatic epimutations reveals dynamics of blood ageing.
              </a>
            </td>
          <td>
            Michael Scherer, Indranil Singh, M. Braun, Chelsea Szu-Tu, Pedro Sanchez Sanchez, Dominik Lindenhofer, N. A. Jakobsen, Verena Körber, Michael Kardorff, Lena Nitsch, Pauline Kautz, Julia Rühle, Agostina Bianchi, Luca Cozzuto, R. Frömel, Sergi Beneyto-Calabuig, Caleb A Lareau, Ansuman T. Satpathy, Renée Beekman, Lars M Steinmetz, S. Raffel, Leif S. Ludwig, Paresh Vyas, Alejo E. Rodriguez-Fraticelli, Lars Velten
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Personalized immunotherapy of pancreatic ductal adenocarcinoma (PDAC) through T-cell mediated targeting of tumor-specific, mutanome-encoded neoepitopes may offer new opportunities to combat this disease, in particular by countering recurrence after primary tumor resection. However, the sensitive and accurate calling of somatic mutations in PDAC tissue samples is compromised by the low tumor cell content. Moreover, the repertoire of immunogenic neoepitopes in PDAC is limited due to the low mutational load of the majority of these tumors. Methods We developed a workflow involving the combined analysis of next-generation DNA and RNA sequencing data from matched pairs of primary tumor samples and patient-derived xenograft models towards the enhanced detection of driver mutations as well as single nucleotide variants encoding potentially immunogenic T-cell neoepitopes. Subsequently, we immunized HLA/human T-cell receptor (TCR) locus-transgenic mice with synthetic peptides representing candidate neoepitopes, and molecularly cloned the genes encoding TCRs targeting these epitopes. Result Application of our pipeline resulted in the identification of greater numbers of non-synonymous mutations encoding candidate neoepitopes with increased confidence. Furthermore, we provide proof of concept for the successful isolation of HLA-restricted TCRs from humanized mice immunized with different neoepitopes, several of which would not have been selected based on mutanome analysis of PDAC tissue samples alone. These TCRs mediate specific T-cell reactivity against the tumor cells in which the corresponding mutations were identified. Conclusion Enhanced mutanome analysis and candidate neoepitope selection increase the likelihood of identifying therapeutically relevant neoepitopes, and thereby support the optimization of personalized immunotherapy for PDAC and other poorly immunogenic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74726107b1853e699ed8135ce18a77b1befecac" target='_blank'>
              Enhanced mutanome analysis towards the induction of neoepitope-reactive T-cell responses for personalized immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            M. Volkmar, Dana Hoser, Claudia Lauenstein, Janina Rebmann, Agnes Hotz-Wagenblatt, Jan Rieger, I. Poschke, Jonas P. Becker, Angelika B. Riemer, M. Sprick, A. Trumpp, Oliver Strobel, Thomas Blankenstein, G. Willimsky, R. Offringa
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Bladder cancer affects over half a million people worldwide each year. Recent advances in early detection allowed a successful management of non-aggressive cancers, yet the recurrence rate remains high. Aggressive muscle-invasive bladder tumours are life-threatening and challenging to cure. Therefore, understanding of key molecular pathways involved in cancer progression is critical for developing of new personalised targeted therapies. Recently, non-coding RNAs (ncRNAs) have emerged as key regulators orchestrating complex biological processes in cancer, yet their function is not fully understood. Here, we compare non-muscle invasive and muscle invasive cell lines and identify a ncRNA gene MIR205HG and its transcript LEADR among the top ncRNAs downregulated in muscle invasive urothelial tumours. We show that LEADR expression is epigenetically regulated by master transcription factor p63. LEADR is localised in the nuclei of non-muscle invasive bladder cancer cells where it dampens hyperactivation of interferon stimulated genes possibly increasing sensitivity of bladder cancer cells to interferon signalling. These findings uncover an anti-tumoral role of non-coding RNA LEADR in mediating immune response in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9079e8e609f16cb2e312120dae481fd8ed5c7862" target='_blank'>
              LEADR, a p63 target, dampens interferon signalling in bladder cancer
              </a>
            </td>
          <td>
            Damiano Barnaba, Mariacristina Franzese Canonico, Manuela Helmer-Citterich, P. Gandellini, G. Melino, Artem Smirnov, E. Candi
          </td>
          <td>2025-06-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Immune evasion is a hallmark of gliomas, yet the genetic mechanisms by which tumors escape immune surveillance remain incompletely understood. In this study, we systematically examined the presence of somatic mutations in human leukocyte antigen (HLA) genes and genes encoding proteins involved in antigen-presentation across isocitrate dehydrogenase wild-type (IDHwt) and mutant (IDHmut) gliomas using targeted next-generation sequencing (NGS). To address the challenges associated with detecting somatic mutations in these highly polymorphic and complex regions of the genome, we applied a combination of short-read and long-read sequencing techniques, extended the genetic region of interest (exons and introns), and applied a tailored bioinformatics analysis pipeline, which enabled an accurate evaluation of comprehensive sequencing data. Our analysis identified mutations in HLA class II and non-classical HLA genes as well as genes associated with antigen presentation, such as TAP1/2 and B2M. Three-dimensional modeling of individual mutations simulated the potential impact of somatic mutations in TAP1 and B2M on the encoded protein configuration. The presence of somatic mutations supports the role of antigen-presenting genes in the pathophysiology and potential immune escape of gliomas. Our data demonstrated an increased frequency of such mutations in recurrent glioblastoma (GBM), potentially resulting from a positive selection or mutagenic enrichment of tumor cells during tumor progression. Taken together, this research generates new insights and hypotheses for the functional analysis and optimization of immunotherapy strategies for gliomas, which may guide personalized treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720b53545edf3b58403eb323f878fcac355077a2" target='_blank'>
              Somatic mutations in HLA class genes and antigen presenting molecules in malignant glioma.
              </a>
            </td>
          <td>
            Sara C. Schulte, Wolfgang Peter, G. Rosenberger, Moritz Schäfer, Cecile L Maire, A. Rünger, A. Ryba, K. Riecken, K. Fita, Jakob Matschke, Nuray Akyüz, Judith Dierlamm, G. Klau, F. Ricklefs, J. Gempt, Manfred Westphal, K. Lamszus, A. Dilthey, M. Mohme
          </td>
          <td>2025-05-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles are unique, allowing sample stratification into ATRT subtypes. The TE activity signature in ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC shows broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC is also unique in having significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and-MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b1746e8b34806ab31721a10a88339f9bbc0904" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="AIMS
This study identifies single-nucleotide polymorphisms (SNPs) associated with cellular response to cyclophosphamide (CTX) using phosphoramide mustard (PM), its primary cytotoxic metabolite, and explores the downstream consequences for breast cancer (BC) patients.


METHODS
We analyzed 1,978,545 SNPs from EBV-transformed lymphoblastic cell lines (LCLs) derived from 53 unrelated European individuals, in a genome-wide association study using cellular PM sensitivity data. We filtered SNPs associated with PM sensitivity (p < 5 × 10-5) predicted to overlap with regulatory elements in breast tissue using a chromatin state prediction model. We then assessed the consequences using LCL transcriptomic data and data from BC patients treated with (ACT-BC; N = 155) and without CTX.


RESULTS
Twenty SNPs were filtered out including rs12408401, which was associated with PM resistance (p = 3.89 × 10-5), potentially disrupted a CTCF-loop, and was associated with increased RFX5 expression (p = 0.036), which was associated with poor disease-free interval in ACT-BC patients (HR = 5.32; p = 0.028); and rs784562, which was associated with improved PM sensitivity (p = 6.41 × 10-6), potentially altered nearby enhancer functionality, and reduced expression of KRT72 which was associated with poor progression-free survival in ACT-BC patients (HR = 3.61; p = 0.040).


CONCLUSION
Our study identifies SNPs significantly associated with cellular CTX response with potential mechanistic and clinical relevance, thereby providing insights toward optimized CTX treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867439245770812a0b6fc8a701b4e620584bcc27" target='_blank'>
              A genome-wide association study using HapMap cell lines reveals modulators of cellular response to cyclophosphamide.
              </a>
            </td>
          <td>
            Mohammed O Gbadamosi, N. Bhise, T. Ghosh, Elizabeth K Molchan, Kathleen Streeks, Jason Puglise, Alyssa Ohaegbulam, Mariana Makarem, Oluwaseyi Olabige, Changlin Yang, Luisel J Ricks-Santi, Duane A Mitchell, Brooke L. Fridley, J. Lamba
          </td>
          <td>2025-05-13</td>
          <td>Future oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 e22601


 Background:
 According to the recent 2022-2024 published data from the GLOBOCAN, India alone contributed 15.5% of these cases with a staggering 26% of Breast carcinoma (BC) cases and 0.85 million people succumbed to the disease. Inherited or spontaneous mutations in HRR group of genes can disrupt DNA damage response signaling, impairing HRR and sensitizing cells to genotoxic insults. The current study brings to fore the spectrum of clinically significant and novel HRR pathway gene alterations in breast carcinoma patients with increased TNBC incidence.
 Methods:
 A total of 301, histologically diagnosed breast carcinoma patients were included for germline molecular assessment of HRR pathway genes via Ion Torrent NGS following manufacturer’s protocol from January,2019 to December,2023. Clinical summary and an informed consent were obtained individually from the patients.
 Results:
 Out of the total of 301 BC patients participated in the study, 119(39.53%) were diagnosed TNBC cases. NGS data revealed 76(25.2%) patients to harbour at least one clinically significant HRR pathway gene variant. Predominantly, BRCA2 was the most mutated gene (47.3%), followed by BRCA1 (43.4%) across patients. 41(34.4%) TNBC cases were detected with a pathogenic gene variant. In the mutation spectrum, exon-10 (27.9%) of BRCA1 and exon-11 (52.7%) of BRCA2 genes were the most mutated regions detected. Additionally, findings revealed 11 novel variants across HRR genes. Six novel variants in BRCA1 (exon-10,
 c.1039_1040insA,
 c.2841delA; exon-16, c
 .5009delG
 (02) and, exon-23,
 C.5519_5573del
 ) (02), four in BRCA2 (exon-10,
 c.1249_1250insT
 ; exon-11,
 c.4287_4288insT
 ; exon-14, c
 .7279_7280insC
 and exon-24,
 c.9241_9242insA
 ) and one in FANCD2 (exon-21, c.1900G>A) genes respectively. Also, BRCA1 founder mutation (185delAG) was also observed in two (2.6%) patients. CHEK2, PIK3CA, FANCD2 and TP53 were the other HRR pathway genes found positive in the cohort.
 Conclusions:
 Examining the complete BRCA gene status in patient cohort has enabled the identification of not only recurring pathogenic variants but also novel mutations. Among BC patients, an elevated frequency of TNBC patients (39.53%) is an alarming finding compared to the global incidence of 15-20%. The findings enable promising targeted therapeutic approaches by matching patients to treatments which is most likely to benefit them. With the views over the years and with the projected trends in the coming years, BC is set to remain in the driving seat, to drive the national and global cancer burden. Hence, population-based screening, population based molecular-genetic study updates and novel approaches in different strata of the disease are needed with the fast-moving cancer trend in the global society.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ac143a16ddd61570651a11bd202fbcfe3a78bb9" target='_blank'>
              Landscape of homologous recombination repair pathway germline NGS gene variants in a TNBC-rich cohort of Indian breast carcinoma patients.
              </a>
            </td>
          <td>
            Amit Roychowdhury, Ghanashyam Biswas, Birendranath Banerjee
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Radiation therapy causes long-term skeletal muscle atrophy and fibrosis in juvenile cancer survivors. The mechanisms responsible for the skeletal muscle late effects of radiation therapy are not well-understood and have prevented the development of effective treatments. Using single-cell RNA sequencing (scRNA-seq), we characterize cellular dynamics and communication in a murine model of therapeutic radiation at 24-hours and 56-days post-irradiation (post-IR). We detected changes in muscle stem (satellite) cells (MuSCs) characterized by an acute preservation of committed MuSCs and long-term relative depletion of deep quiescent MuSCs. A conserved senescence Cdkn1a signature was observed in all muscle-resident cells post-IR. Genes related to fibroblast proliferation were up-regulated and a fibrotic and senescent transcriptome persisted in Fibro-adipogenic progenitors (FAPs) post-IR. Intercellular communication analysis revealed FAPs as the primary contributor of extracellular matrix (ECM) and target of monocyte/macrophage-derived TGF-β signalling post-IR through TGF-βR2 on FAPs. Together, our findings provide insights into the potential mechanisms and intercellular communication responsible for radiation-induced muscle atrophy and fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/697ad09c5bedb5b01070a9a0761c75e4c69a9ec3" target='_blank'>
              Single-cell transcriptomic analysis reveals alterations to cellular dynamics and paracrine signalling in radiation-induced muscle pathology.
              </a>
            </td>
          <td>
            Nicolás Collao, E. Johannsen, J. Just, Michael De Lisio
          </td>
          <td>2025-05-02</td>
          <td>American journal of physiology. Cell physiology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7668e6be2f122a26f394a92f23a38b2f39f2818" target='_blank'>
              Genome Editing in Gynecological Oncology: The Emerging Role of CRISPR/Cas9 in Precision Cancer Therapy.
              </a>
            </td>
          <td>
            Naina Kumar
          </td>
          <td>2025-05-20</td>
          <td>Therapeutic innovation & regulatory science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Monoallelic gene expression is a pivotal phenomenon in developmental biology, notably through the influence of imprinted genes. Our model predicts that monoallelic expression generates expression variability, which we assess by measuring genetic noise and entropy within Shannon’s information theory framework. Analyzing single-cell allele-specific expression across human and mouse datasets, we consistently observe increased expression variability due to monoallelic expression, affecting both imprinted and co-expressed non-imprinted genes. Moreover, we find decreasing variability in developing neurons and increasing variability in glial cells. The discovery of distinct noise patterns in over 80% of analyzed genes between glial and neuronal populations highlights the importance of differential noise in neurodevelopmental processes. Given the critical role of imprinted genes in biological processes such as growth and brain development, disruptions in their expression might contribute to various disorders. Understanding the stochastic nature of monoallelic expression and its genome-wide impact offers new insights into the mechanisms underlying these pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75ef7d7f7ce26afe04cd1b0436e38667096c1fe7" target='_blank'>
              Monoallelic gene expression in developing cells increases genetic noise and Shannon entropy
              </a>
            </td>
          <td>
            Robert Wolff, Edoardo Balzani, Elisa Gelli, Alessia Polito, Angelo Serani, Valter Tucci
          </td>
          <td>2025-06-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Oral squamous cell carcinoma (OSCC) is a prevalent malignancy with poor prognosis, largely attributed to p53 mutations in 65%–85% of cases. Mutant p53 loses tumour-suppressor functions and gains oncogenic roles, promoting cancer progression and therapy resistance. Targeting mutant p53 represents a promising therapeutic avenue.



 The aim of this study was to evaluate the potential of 4-amino benzoic acid (PABA)-substituted pyrimidine derivatives as novel inhibitors of mutant p53 protein through in silico analyses.



 Mutant p53 (PDB ID: 4mzi) was retrieved from the RCSB Protein Data Bank. PABA derivatives (3A, 3B, 3C and 3D) were assessed for drug-likeness, toxicity and binding affinities using Lipinski’s Rule, PROTox-II and AutoDock 4.2. Binding energies, hydrogen bonds and Van der Waals interactions were analysed.



 All derivatives satisfied Lipinski’s Rule with no violations, demonstrating drug-likeness. Toxicity analysis classified the compounds as non-carcinogenic and non-mutagenic. Binding affinities were highest for 3D (−8.8 kcal/mol) with strong interactions at PRO A: 219, GLY A: 199 and ASN A: 200, forming four hydrogen bonds. Compound 3C (−7.9 kcal/mol) also exhibited significant binding. PABA’s anti-cancer properties and stable protein–ligand interactions highlight its therapeutic potential.



 PABA-substituted pyrimidine derivatives, particularly 3D, show promise as lead compounds for targeting mutant p53 in OSCC. Further in vitro and in vivo studies are warranted to validate their efficacy and safety, paving the way for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce980804d4d12fed302c5abe967d43aad6bb40f2" target='_blank'>
              In Silico Development of PABA-substituted Pyrimidine Derivatives Targeting Mutant p53 for Precision Therapy in Oral Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Kathirvel Sekar, Kaviyarasi Renu, Sathishkumar Vinayagam, Devianjana Rajendran, Kamaraj Sattu
          </td>
          <td>2025-05-08</td>
          <td>Journal of Advanced Oral Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Programmable gene integration in human cells has the potential to enable mutation-agnostic treatments for loss-of-function genetic diseases and facilitate many applications in the life sciences. CRISPR-associated transposases (CASTs) catalyze RNA-guided DNA integration but thus far demonstrate minimal activity in human cells. Using phage-assisted continuous evolution (PACE), we generated CAST variants with >200-fold average improved integration activity. The evolved CAST system (evoCAST) achieves ~10 to 30% integration efficiencies of kilobase-size DNA cargoes in human cells across 14 tested genomic target sites, including safe harbor loci, sites used for immunotherapy, and genes implicated in loss-of-function diseases, with undetected indels and low levels of off-target integration. Collectively, our findings establish a platform for the laboratory evolution of CASTs and advance a versatile system for programmable gene integration in living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc7a114e9413b1138dda6d46ec6c355a9e8561f" target='_blank'>
              Programmable gene insertion in human cells with a laboratory-evolved CRISPR-associated transposase.
              </a>
            </td>
          <td>
            Isaac P. Witte, George D. Lampe, Simon Eitzinger, Shannon M Miller, Kiara N Berríos, A. McElroy, Rebeca T. King, Olivia G Stringham, Diego R. Gelsinger, P. Vo, Albert T. Chen, Jakub Tolar, Mark J. Osborn, S. H. Sternberg, David R. Liu
          </td>
          <td>2025-05-15</td>
          <td>Science</td>
          <td>2</td>
          <td>34</td>
        </tr>

        <tr id="
 6528


 Background:
 NPM1
 GA characterize a clinically important subset of AML cases which relapse in more than 50% of treated patients despite being generally sensitive to conventional chemotherapy regimens. In AML, the interactions between GA in
 NPM1
 ,
 KMT2A
 and menin protein have been linked to leukemogenesis and represent new potential targets for anti-tumor therapies.
 Methods:
 4,206 cases of AML underwent comprehensive genomic profiling from 2019 through 2024, using the FoundationOne Heme combined hybrid capture based DNA and RNA sequencing assay. All classes of relevant GA were evaluated. The tumor mutation burden (TMB), homologous recombination deficiency signature (HRDsig) and microsatellite Stability (MSS) status were determined from the sequenced data.
 Results:
 633 (15.1%) of the 4,206 AML featured
 NPM1
 GA (NPM1mut). Short variant mutations were found in >99% of the NPM1mut AML with the W288fs*12 frameshift base substitution accounting for 92.4% of cases. An
 NPM1
 -
 MLF1
 fusion was identified in 1.3% of NPM1mut cases. The NPM1mut+ were more frequently associated with female patients (53.4% vs 41.5%; p<.0001) and had a slightly higher median age compared to the NPM1 wild type (NPM1wt) AML patients (62yrs vs 60yrs; p<.0001). Majority of patients (>60%) were from European decent. There were greater NPM1 GA in patients with European (77.1% vs 68.5%; p<.0001) and lower with African ancestry (9.2% vs 10.2%; p<.0001). MSI High (0% in both groups) status, HRDsig+ (0—0.1%) and elevated TMB (median < 1 mutation/Mb) were extremely uncommon in both groups. GA more frequent in NPM1mut AML compared to the NPM1wt AML cohort included
 DNMT3A
 (39.2% vs 12.6%; p<.0001),
 FLT3
 (54.5% vs 14.7%; p<.0001)
 ,
 IDH1
 (16.1% vs 5.6%; p<.0001),
 IDH2
 (19.0% vs 9.0%; p<.0001),
 TET2
 (23.4% vs 13.5%; p<.0001) and
 WT1
 (12.5% vs 9.4% p=.02). GA more frequent in NPM1wt AML included
 ASXL1
 (17.1% vs 3.6%; p,.0001),
 BCOR
 (7.5% vs 1.6%; p<.0001),
 KMT2A
 (14.7% vs 0.2%; p<.0001),
 RUNX1
 (22.5% vs 1.9%; p-,.0001),
 STAG2
 (6.9% vs 1.6%; p<.0001) and
 TP53
 (19.1% vs 4.1%; p<.0001).
 Conclusions:
 The development of menin inhibitors has recently identified GA in
 NPM1
 as a promising target of therapy for AML patients. Other therapy targets in AML such as
 FLT3
 and
 IDH1/2
 are more frequently identified in NPM1mut than NPM1wt AML, while
 KMT2A
 is more frequently identified in NPM1wt AML. This genomic landscape study reveals significant differences in important GA associated with AML in NPM1mut and NPM1wt cases which may enrich our understanding of the molecular profile in AML and identify additional targets for therapy.


 Pathogenic genomic alterations in NPM1mut and NPM1wt AML.





 NPM1wt AML (n=3573)
 NPM1mut AML (n=633)
 P-value





 ASXL1
 17.1%
 3.6%
 <.0001



 CEBPA
 6.4%
 8.2%
 NS



 DNMT3A
 12.6%
 39.2%
 <.0001



 FLT3
 14.7%
 54.5%
 <.0001



 IDH1
 5.6%
 16.1%
 <.0001



 IDH2
 9.4%
 19.0%
 <.0001



 KMT2A
 14.7%
 0.2%
 <.0001



 RUNX1
 22.5%
 1.9%
 <.0001



 TET2
 13.5%
 23.4%
 <.0001



 TP53
 19.1%
 4.1%
 <.0001





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f4a065a163ca2b1d9ef8d78ae544e2bc89a03fb" target='_blank'>
              Nucleophosmin (
 NPM1
 ) genomic alterations (GA) in acute myeloid leukemia (AML): A genomic landscape study.
              </a>
            </td>
          <td>
            Osama Batayneh, Alexandra Goodman, Devashish Desai, D. Pavlick, Chelsea Marcus, Caleb Ho, J. Ross, Teresa Gentile, Zheng Zhou, Krishna Ghimire
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, Daniel W. Lin, Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8533a84bf23dfa93f9b981af3fd2fcfd03e25b" target='_blank'>
              AURKA inhibition amplifies DNA replication stress to foster WEE1 kinase dependency and synergistic antitumor effects with WEE1 inhibition in cancers
              </a>
            </td>
          <td>
            Jong Woo Lee, Jackie Shi, Julian Barrantes, Pratima Chaurasia, Sundong Kim, S. Cruz-Gomez, Cindy Yang, Jianlei Gu, Hongyu Zhao, Dejian Zhao, Benjamin Schiff, Ansley Roche, Elizabeth B. Perry, Jeffrey P. Townsend, Gary M. Kupfer, Katerina Politi, E. Golemis, Barbara Burtness
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Aneuploidy, characterized by the gain or loss of chromosomes, plays a critical role in both cancer and congenital aneuploidy syndromes. For any aneuploidy, we can distinguish between its general effects and its chromosome-specific effects. General effects refer to the common cellular stresses induced by aneuploidy, such as impaired proliferation, proteotoxic stress, and altered metabolism, which occur regardless of the specific chromosome involved and profoundly impact cellular and organismal functions. These generalized stresses often hinder cell fitness but can also, under certain conditions, contribute to cancer progression. In contrast, chromosome-specific effects arise from the altered dosage of particular genes on the gained or lost chromosome. These effects vary depending on the chromosome involved and can provide specific fitness effects in cancer cells or distinct developmental phenotypes in congenital aneuploidies like Down syndrome. Understanding the interplay between these two levels of effects is crucial for deciphering the outcomes of aneuploidy. This review synthesizes current knowledge and discusses future directions for unraveling the hallmarks of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 e17084


 Background:
 Six-Transmembrane Epithelial Antigen of the Prostate 1 (STEAP1) is a transmembrane protein frequently overexpressed in prostate cancer (PC). Antibody-based therapies targeting STEAP1 have shown promising preclinical and clinical outcomes. Despite its widespread and PC-specific overexpression, the molecular regulation and biological function of STEAP1 remain poorly understood. Elucidating the mechanisms that govern STEAP1 expression and its role in PC progression could provide valuable insights into disease mechanisms, improve patient selection for STEAP1-targeted therapies, and inform combination or sequential treatment strategies.
 Methods:
 To investigate STEAP1 regulation and function, we first compared its expression to established PC driver genes and observed a correlation between STEAP1 and the androgen receptor (AR) as well as several AR-regulated genes. Using pathway analyses in the Decipher (localized prostate cancer) and West Coast Dream Team (WCDT, metastatic prostate cancer) clinical cohorts, we examined dysregulated pathways in patients with high STEAP1 expression. Additionally, we performed a FACS-based whole-genome CRISPR interference (CRISPRi) screen in C4-2B cells using an anti-STEAP1 antibody to identify regulators by comparing STEAP1-high and STEAP1-low expressing subsets.
 Results:
 In the WCDT dataset, STEAP1 was highly expressed in AR-positive metastatic prostate cancer subtypes, irrespective of neuroendocrine differentiation, but was significantly lower in AR-negative subtypes. STEAP1 expression strongly correlated with several AR-regulated genes, including STEAP2, KLK2, NKX3-1, KLK3, ARHGAP6, SLC45A3, and PMEPA1. Pathway analyses in both localized and metastatic PC cohorts revealed dysregulated metabolic pathways, particularly lipid metabolism, in STEAP1-high patients. The CRISPRi screen identified several potential targetable regulators of STEAP1, including MYC, SWI/SNF chromatin remodeling complexes.
 Conclusions:
 Our study establishes a strong association between STEAP1 and AR-driven metastatic castration-resistant prostate cancer subtypes. Furthermore, patient data suggest that STEAP1 is linked to dysregulated metabolic pathways, particularly those involving lipid metabolism. The identification of STEAP1 regulators through the CRISPRi screen provides valuable insights into its regulation and biological function, offering potential avenues for developing combination therapies, strategies to prevent treatment resistance, and improved patient stratification for STEAP1-targeted treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2f4b4ddfd1356bdf18bff71926fd89045fb54e" target='_blank'>
              FACS-based whole-genome CRISPRi screen to uncover STEAP1 regulators in prostate cancer.
              </a>
            </td>
          <td>
            Shan Zhu, Mikhail Dias, Yuxin Yang, Emma Dolan, Thaidy Moreno Rodriguez, Tamilla Nechiporuk, Jun Zhu, Derek Xiang, Tianyi Liu, Tatyanah Farsh, Meng Zhang, Gabriel Eades, Stanley Leung, Lisa N Chesner, Peter Bruno, M. Cooperberg, David A Quigley, F. Feng, Haolong Li, Carissa E. Chu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="PURPOSE
This study reviews how ionizing radiation (IR) induces alternative splicing (AS) in non-tumor and tumor cells under both ex vivo and in vivo irradiation conditions. The relevance and limitations of IR-induced AS in identifying potential biomarkers are highlighted for two main applications: biodosimetry and radiotherapy.


CONCLUSIONS
Radiation promotes alterations in AS, which may differentially affect the response in both tumor and non-tumor cells. This response can occur in genes that change their overall expression as well as in those that remain unaltered in response to IR. Although cis-regulators modulate AS, trans-regulators like splicing factors are more involved in the IR response. Variants of key genes involved in the DNA damage response (DDR) are regulated in non-tumor cells while they are often deregulated in tumor cells favoring radioresistance. Identifying IR-induced AS variants could enhance the sensitivity of biodosimeters for dose estimation and biomarkers for radiosensitivity, offering potential strategies to personalize radiotherapy and improve outcomes. New and advanced sequencing technologies will allow variant identification important for the field of radiobiological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c740ec1aef5f884cca33c90567c580fecc7177" target='_blank'>
              Alternative splicing in human cells exposed to ionizing radiation: a comprehensive review of ex vivo and in vivo studies.
              </a>
            </td>
          <td>
            Jerónimo Leberle, Marina Perona, A. Choudhury, N. Bellora, Irene L Ibañez, Luisa V. Biolatti
          </td>
          <td>2025-05-22</td>
          <td>International journal of radiation biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e525ac03623c2a7ca9ffb9d9864241770659543" target='_blank'>
              Unbiased recording of clonal potency reveals species-specific regulation of mammalian intestine
              </a>
            </td>
          <td>
            Mirazul Islam, Matthew E. Bechard, Yilin Yang, A. Simmons, Yanwen Xu, J. Higginbotham, Ping Zhao, Zheng Cao, Naila Tasneem, Sarah E. Glass, Nicholas O. Markham, Frank Revetta, Marisol Ramirez-Solano, Qi Liu, J. Franklin, Ken S. Lau, Robert J. Coffey
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8569a1e64e6674714c2e763b602f6c886995d4db" target='_blank'>
              Multiple FGFR1 mutations modulate tumorigenic mechanisms in glioneuronal tumors
              </a>
            </td>
          <td>
            Jacopo Boni, Míriam Fernández-González, HyeRim Han, Carla Roca, Cassandra J. Wong, Cristina Rioja, C. Nogué, Leticia Manen-Freixa, J. Boulais, Endika Torres-Urtizberea, Antonio Gómez, M. Hasselblatt, R. Estrada-Tejedor, A. Antolin, Islam E. Elkholi, Nada Jabado, Jean-François Côté, A. Gingras, Barbara Rivera
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, Mike Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d072dee701b15034f00be3c11a965c7ef7b90a4" target='_blank'>
              Clonal memory of cell division in humans diverges between healthy haematopoiesis and acute myeloid leukaemia
              </a>
            </td>
          <td>
            A. Donada, G. Hermange, T. Tocci, A. Midoun, G. Prevedello, L. Hadj Abed, D. Dupré, W. Sun, I. Milo, S. Tenreira Bento, C. Pospori, A. Innes, C. Willekens, J. Vargaftig, D. Michonneau, C. Lo Celso, N. Servant, KR Duffy, H. Isambert, PH Cournède, L. Laplane, L. Perie
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 e16019


 Background:
 Homologous recombination repair (HRR), a key pathway in DNA damage response (DDR), is linked to the efficacy of platinum-based chemotherapy, PARP inhibitors, and immunotherapy across cancers. However, its mutational landscape and impact on the immune microenvironment in gastric cancer remain unclear.
 Methods:
 Next-generation sequencing (NGS) and multiplex immunofluorescence were used to analyze the mutational profiles and immune microenvironment of 213 gastric cancer patients, focusing on the impact of HRR mutations.
 Results:
 Of the 213 patients, 67 (31.45%) harbored HRR mutations, including 46 with variants of uncertain significance (VOUS). The most frequently mutated HRR genes were RAD50 (4.69%), BLM (4.23%), and BRCA2(2.35%), with BLM/RAD50 co-mutations being the most common (2.35%). HRR mutations were significantly co-occurring with KMT2D mutations and mutually exclusive with TP53 mutations. We analyzed the association of HRR pathway mutations with immune biomarkers, stratifying patients by pathogenic/likely pathogenic and VOUS mutations. Pathogenic mutations were linked to high tumor mutational burden (TMB; p < 0.0001) and microsatellite instability-high (MSI-H) status but showed no association with PD-L1 expression (CPS = 1/5/10) or EBV infection. Similar trends were seen for VOUS mutations. Further analysis of HRR mutation burden (≥2 mutated HRR genes) demonstrated an association with high TMB and MSI-H status, as well as a tendency toward positive PD-L1 CPS expression (CPS ≥ 1). Immune microenvironment analysis revealed that HRR- patients had higher proportions of M2 macrophages, suggesting polarization toward an immunosuppressive phenotype. However, the M1/M2 macrophage ratio showed no significant difference, potentially due to compensatory effects from other immune cells. HRR- patients also displayed elevated CD56dim NK cells and a higher CD56dim/bright NK cell ratio, indicating a dominance of cytotoxic NK cells. In addition, HRR- patients had more CD4⁺ T cells (p = 0.0004) and a higher CD4/CD8 ratio, though not statistically significant. The observed infiltration of CD4⁺FOXP3⁻ T cells suggests that HRR mutations might influence immune cell composition and contribute to the formation of a non-inflammatory immune microenvironment in HRR-mutant gastric cancers.
 Conclusions:
 In gastric cancer, pathogenic/likely pathogenic HRR mutations are associated with higher TMB levels and MSI, while HRR variants of uncertain significance show a similar trend. Immune microenvironment analysis suggests that HRR pathway mutations may be linked to a non-inflammatory immune microenvironment. These findings indicate that HRR mutations could be relevant to the efficacy of immunotherapy in gastric cancer patients, warranting further investigation into their clinical value.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ceb40e645e4f8a6eec7df1e8de409f9b6d52363" target='_blank'>
              Implications of homologous recombination repair pathway mutations on the immune microenvironment in gastric cancer.
              </a>
            </td>
          <td>
            Feng Jin, Ping Jiang, Huayang Zhong, Zengjin Hu, Ning Gao
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA damage is a serious threat to cellular viability, and it is implicated as the major cause of normal ageing. Hence, targeting DNA damage therapeutically may counteract age-related cellular dysfunction and disease, such as neurodegenerative conditions and cancer. Identifying novel DNA repair mechanisms therefore reveals new therapeutic interventions for multiple human diseases. In neurons, non-homologous end-joining (NHEJ) is the only mechanism available to repair double-stranded DNA breaks (DSB), which is much more error prone than other DNA repair processes. However, there are no therapeutic interventions to enhance DNA repair in diseases affecting neurons. NHEJ is also a useful target for DNA repair-based cancer therapies to selectively kill tumour cells. Protein disulphide isomerase (PDI) participates in many diseases, but its roles in these conditions remain poorly defined. PDI exhibits both chaperone and redox-dependent oxidoreductase activity, and while primarily localised in the endoplasmic reticulum it has also been detected in other cellular locations. We describe here a novel role for PDI in DSB repair following at least two types of DNA damage. PDI functions in NHEJ, and following DNA damage, it relocates to the nucleus, where it co-localises with critical DSB repair proteins at DNA damage foci. A redox-inactive mutant of PDI lacking its two active site cysteine residues was not protective, however. Hence, the redox activity of PDI mediates DNA repair, highlighting these cysteines as targets for therapeutic intervention. The therapeutic potential of PDI was also confirmed by its protective activity in a whole organism against DNA damage induced in vivo in zebrafish. Hence, harnessing the redox function of PDI has potential as a novel therapeutic target against DSB DNA damage relevant to several human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c93ace9388c019ac0f5bc12962ca74d85497c2d9" target='_blank'>
              The Redox Activity of Protein Disulphide Isomerase Functions in Non-Homologous End-Joining Repair to Prevent DNA Damage.
              </a>
            </td>
          <td>
            S. Shadfar, Fabiha Farzana, S. Saravanabavan, Ashley M Rozario, M. Vidal, Cyril J Jagaraj, S. Parakh, Esmeralda Paric, Kristy C Yuan, Mariana Brocardo, Donna R Whelan, Angela S Laird, J. Atkin
          </td>
          <td>2025-05-15</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 4154


 Background:
 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a dismal prognosis despite advances in treatment. PDAC is characterized by a dense stromal environment with suppressed anti-tumor immunity that contributes to treatment resistance. Oncogenic
 KRAS
 mutations are present in > 90% of PDACs and are known to play a role in modulating the tumor immune microenvironment (TME). In this study, we examined the differences in TME components between
 KRAS
 -mutant (m) and
 KRAS
 -wildtype (wt) PDACs.
 Methods:
 Comprehensive genomic and immune profiling (CGIP), including the RNA-seq based gene expression assessment of 395 immune-associated genes, was performed on 311 PDAC patient samples. Gene expression signatures of tumor immunogenicity (TIGS) and cell proliferation were calculated by averaging the gene expression ranks of 161 immune-associated genes and 10 proliferation genes, respectively. The normalized gene expression rank of 22 immune checkpoint genes and 17 cancer testis antigen (CTA) genes were also calculated. DNA-seq was used to identify
 KRAS
 mutations and to calculated tumor mutational burden (TMB). Continuous variables were compared between subgroups using the Wilcoxon Rank-Sum test and categorical variables were compared between groups using the Chi Squared test. For statistical significance, p < 0.05 was required.
 Results:
 The cohort consisted of 311 PDAC patient samples, comprising 159 females (51.1%) and 152 males (48.9%) with a median age at testing of 69.3 years (38.7-92.6). A total of 264 specimens (84.9%) exhibited a
 KRAS
 mutation, with the most common being G12D and G12V while 47 (15.1%) had
 KRAS
 -wt tumors. No difference in TIGS, CP, or TMB was observed between
 KRAS
 -wt and
 KRAS
 -m tumors.
 KRAS
 -wt tumors exhibited greater expression of 12 of 17 tested CTAs than
 KRAS
 -m tumors, including GAGE13 (p = 0.004) NY-ESO-1 (p = 0.01), MAGEA3 (p = 0.01), and LAGE1A (p = 0.01). Analysis of the gene expression of 22 immune checkpoint genes showed no difference for most of the genes, though there was higher expression of PD-1 (p = 0.04) and CD27 (p = 0.03) seen in
 KRAS
 -wt compared to
 KRAS
 -m tumors.
 Conclusions:
 KRAS
 -wt PDACs exhibited greater expression of cancer testis antigen genes compared to
 KRAS
 -m tumors, suggesting potential therapeutic susceptibility to immunotherapy and adoptive cell therapies leveraging the expression of CTAs as targets. Assessment of
 KRAS
 status and immunotherapy susceptibility may support future clinical trial selections for therapies targeting the complex interplay of genomic and immune components of pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae6d1fe40ee2b5375e715aee2512cfe9d76d6fb" target='_blank'>
              Unveiling the differences in tumor immune microenvironment between
 KRAS
 -wildtype and
 KRAS
 -mutant pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Heidi Ko, S. Pabla, Michelle F Green, M. Senosain, Hardik Parikh, Erik Van Roey, Shuang Gao, Yamuna Pulivendula, P. DePietro, Stuti Kaushalkumar Patel, Kyle C. Strickland, R. Previs, Erin Newburn, Kamal S Saini, Alicia Dillard, Eric A Severson, Brian Caveney, Taylor J. Jensen, Shakti H. Ramkissoon, R. Seager
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d2630c6a248d7d989516a74e5f3c41a92ecf258" target='_blank'>
              R-loops shape chromatin architecture to promote balanced lineage allocation during differentiation
              </a>
            </td>
          <td>
            Chun-Hao Chao, T. Fazzio
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clinical divergence between patients harboring CIC-rearrangements is frequently observed. For example, the prototypical CIC::DUX4 fusion associates with soft tissue tumors while CIC::NUTM1 fusions typically localize to the CNS (brain/spinal cord). The basis for these differences is poorly understood due to a lack of molecular tools. To address this need, we generated patient-informed, synthetic coding sequences for CIC::NUTM1, CIC::LEUTX, and ATXN1::DUX4 and validated them in structure-function studies. We found that CIC::NUTM1 drives a transcriptional program distinct from that of CIC::DUX4 due to a C-terminal NUTM1 functional domain, CIC::LEUTX weakly activates CIC target genes through LEUTX transactivation sequences, and ATXN1::DUX4 upregulates CIC target genes via the ATXN1 AXH domain. Our findings indicate that the CIC fusion binding partner may alter overall fusion oncoprotein activity. Thus, these first generation synthetic tools provide an unprecedented resource to study CIC-family fusions beyond CIC::DUX4 and allow for the dissection of this rare subgroup of cancers. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e6c56fc7116777686ac8289c8f8359830731f5" target='_blank'>
              First Generation Tools for the Modeling of Capicua (CIC) - Family Fusion Oncoprotein-Driven Cancers
              </a>
            </td>
          <td>
            Cuyler Luck, Kyle A. Jacobs, Julia Riad, Christopher D. Macaraig, R. K. Ponce, Ross A. Okimoto
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the significant progress made in the development of epigenetic age (eAge) clocks designed to estimate the various aspects of aging, currently available models, generated using large DNA methylation microarray datasets, still cannot fully address the issues of batch effects and technical variation. This hinders the use of the publicly available eAge clocks in routine laboratory practice, and it motivates the development of cost-effective, custom epigenetic clocks that are tailored to the given biological subjects and research methods. In this study, we analyzed the local DNA methylation of mesenchymal stem cell samples during culture expansion using high-throughput targeted bisulfite sequencing (BS-seq). Using the obtained data, we trained a minimized eAge model based on a Random Forest Regression with Leave-One-Out Cross-Validation, which determines cell passage with good performance (MAE 1.094 and R2 0.897) and which is comparable to previous solutions. Using the advantage of BS-seq to analyze consecutive CpGs methylation patterns, we demonstrated that combining the analysis of average DNA methylation levels with local methylation heterogeneity scores—thereby reflecting stochastic DNA methylation dynamics—can improve the quality of the epigenetic clock models. Therefore, we propose a research strategy for creating customized epigenetic clocks using targeted BS-seq and provide a mechanistic conceptualization of how information on longitudinal changes in DNA methylation patterns can potentially be used for the assessment of specific aging aspects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162b59e95b20d78aba8a2d1263685e994dd6fbb1" target='_blank'>
              A Novel Framework for the Design of Minimized Epigenetic Clocks Using the Analysis of DNA Methylation Heterogeneity
              </a>
            </td>
          <td>
            S. E. Romanov, Dmitry I. Karetnikov, D. A. Kalashnikova, Denis E. Polivcev, Yakov A. Osipov, D. A. Maksimov, Polina A. Antoshina, Viktor V. Shloma, E. Samoilova, Alina A. Ivanova, Rustam F. Karimov, A. N. Tkalin, Alexander A. Shevchenko, V. Kalsin, V. Baklaushev, P. P. Laktionov
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutational signatures of single-base substitutions (SBSs) characterize somatic mutation processes which contribute to cancer development and progression. However, current mutational signatures do not distinguish the two independent steps that generate SBSs: the initial DNA damage followed by erroneous repair. To address this modelling gap we developed DAMUTA, a hierarchical Bayesian probabilistic model that infers separate signatures for each process, and captures their sample-specific interaction. We applied DAMUTA to 18,974 pan-cancer whole genome sequencing mutation catalogues from 23 cancer types and show that tissue-specificity in mutation patterns is driven largely by variability in damage processes. We also show that misrepair processes are predictive of DNA damage response deficiencies. Unlike existing approaches, DAMUTA distinguishes damage from misrepair contributions, and we demonstrate significant improvements over a mutational-burden baseline or signatures from the COSMIC database. Our analysis reveals a shared pan-cancer pattern of early clonal transition-mutations which shifts to a more uniform substitution pattern consistent with increased reliance on translesion synthesis for damage tolerance. DAMUTA thus generates a compact set of signatures which resolves redundancies of current signature models, disentangles the effects of DNA damage and misrepair processes, and facilitates improved stratification of tumours, while providing a framework towards a unified pan-cancer model of the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3964588c1648b53d4e30a576d480451079a470ad" target='_blank'>
              Damage and Misrepair Signatures: Compact Representations of Pan-cancer Mutational Processes
              </a>
            </td>
          <td>
            Caitlin F. Harrigan, Kieran R. Campbell, Quaid Morris, Tyler Funnell
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: Patients exhibiting exceptional response to targeted treatments represent a unique opportunity towards identifying predictive biomarkers of response. The introduction of edits into the genome using Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) creates cancer models that can unravel the mechanisms driving tumorigenesis and response to drug agents. The study aimed to identify candidate molecular alterations associated with exceptional response to CDK inhibitors (CDKi) and validate their effects using genome editing. Methods: Patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy at Departments of Oncology-affiliated with the Hellenic Cooperative Oncology Group (HeCOG), were included. Exceptional response was defined as either complete response (CR) to the treatment regimen or partial response (PR) for 3 years, per physician’s assessment. Formalin-fixed paraffin-embedded (FFPE) tumor tissue samples, obtained prior to CDKi administration from these patients were retrospectively collected. Whole-exome sequencing (WES) was performed to analyze tumor mutational landscape of exceptional responders to CDKi. Using CRISPR-mediated genome editing, a breast cancer cell model harboring a patient-specific SNP and the negative control were prepared, and subsequently evaluated for their responsiveness to palbociclib. Transcriptomic and proteomic analyses were conducted to uncover alterations in network interactions, between the two cell lines, elucidating the exceptional response to CDKi. Results: Overall, 12 women identified as exceptional responders, initiated CDKi treatment from March 2017 to February 2020; median age at diagnosis was 46 (range, 37 to 78). All patients had received the CDKi as first-line treatment; most commonly palbociclib (10 patients, 83.3%). Based on physician assessment, 50% of patients achieved CR with the combination therapy. After a median follow up of 65 months, 10 patients are still alive, and 7 are still receiving treatment with CDKi in combination with endocrine treatment. WES results with adequate mapped reads and acceptable technical characteristics in both tumor and control blood samples were obtained from 6 of 12 patients. A set of germline and somatic single nucleotide polymorphisms (SNPs) commonly shared in this cohort was identified. The focus was on MINDY1, a member of the MINDY family of deubiquitinases (DUBs), known as a deubiquitylase of ERα in breast cancer, the common SNP among all 6 patients. Utilizing CRISPR-based genome editing, a breast cancer cell model harboring a specific MINDY1 SNP was engineered, and their response to palbociclib was subsequently evaluated. Depletion of MINDY1 transcripts through RNA interference with short-hairpin RNA, increased sensitivity to palbociclib in both cell lines, confirming that MINDY1 protein is involved in molecular pathways related to palbociclib response. The functional role of MINDY1 rs771205 was also investigated by combining transcriptomic and proteomic data. Our data demonstrated that the MINDY1 rs771205 SNP, which causes an amino acid substitution, while it does not impact the respective transcript and protein levels, alters the MINDY1 protein’s interaction network. Conclusions: The Rs771205 single mutation alters the MINDY1 protein’s interaction network, sensitizing cells to palbociclib. Our study sheds new light on the underlying mechanisms driving exceptional response in patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy , demonstrating that it is an opportune moment to invest attention and resources towards probing the underlying mechanisms of exceptional response in cancer patients.
 Citation Format: Eleni Balla, Yi Li, Kyriaki Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, Eleni Galani, Rania Romanidou, George Fountzilas, Evangelia Razis, Angelos Koutras, Leonidas Bleris, Elena Fountzilas. MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-03-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea823fc49d305c67d0c88bebfe1f1411182d2dc" target='_blank'>
              Abstract P4-03-26: MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer
              </a>
            </td>
          <td>
            Eleni Balla, Yi Li, K. Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, E. Galani, Rania Romanidou, G. Fountzilas, E. Razis, A. Koutras, L. Bleris, E. Fountzilas
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Targeted inhibition of DNA damage response proteins has received significant clinical attention owing to the success of PARP inhibitors. Due to the loss of the G1/S checkpoint, cancer cells are reliant on the G2/M checkpoint to cope with elevated DNA replication stress. We previously demonstrated a single induction of 8-oxo-guanine at telomeres in cancer cells was sufficient to induce replication stress, but was well tolerated at the cellular level. Here, we found inhibition of ATR, Chk1, or Wee1 after induction of telomere oxidative stress significantly induced genome instability and reduced cell viability. This occurred at doses markedly less than those required to increase instability in non-cancer cells. We determined the mechanism of this instability is due to cells progressing through S-phase with telomere damage and exiting G2-phase prematurely, prolonging their mitosis. This study demonstrates targeted oxidative base damage at telomeres can enhance the therapeutic efficacy of ATR inhibition in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520fb2116ae4feadf96591801b8f58200ddc5cec" target='_blank'>
              Oxidative Base Damage to Telomeres Sensitizes Cancer Cells to ATR Inhibition
              </a>
            </td>
          <td>
            Alex Garbouchian, Natalia Cestari Moreno, A. Dey, Patricia Opresko, Ryan Barnes
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) is a crucial regulator of genomic integrity, frequently mutated in more than half of all human cancers. These mutations predominantly target the DNA-binding domain (DBD), impairing p53's interaction with DNA and its tumor-suppressive functions. To elucidate the structural and functional consequences of p53 mutations, we investigated 148 missense variants located within its DNA-binding interface using cutting-edge computational approaches. We employed AlphaFold3 (AF3) to predict p53-DNA complex structures, integrating these predictions with molecular dynamics (MD) and force-guided pulling simulations to assess mutation-induced changes in structural stability and DNA-binding properties. Moreover, we compared the results of our study with experimental in vitro enrichment scores (RFS) and Combined Annotation Dependent Depletion (CADD). We identified a moderate negative correlation between plDDT and the pathogenicity of mutant variants, suggesting that mutations causing more significant alterations in the protein tertiary structure have a greater negative impact on cellular function. Moreover, we identified two possible structural mechanisms through which mutations can impair the p53 functionality. Specifically, some mutations, such as R248P and N239S, reduce the binding affinity of the p53-DNA complex, whereas others, such as C238Y and P278R, enhance affinity but compromise the structural stability of the complex. Furthermore, we uncovered mutations with potential rescuing effects, such as E285A and M243T, which preserved structural stability and enhanced the DNA-binding ability. Our findings provide a comprehensive framework for understanding the molecular mechanisms underlying p53 mutations and their role in cancer pathogenesis. This study highlights the value of integrative computational approaches in investigating protein-nucleic acid interactions, providing critical insights that can guide the development of therapeutic strategies targeting p53 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab5ce93ebd4a1fbc747fc251d769b1eab00ffa3" target='_blank'>
              Investigation of Structural Mechanisms Underlying p53 Dysfunction Caused by 148 Missense Mutations Using AlphaFold3 and Molecular Dynamics Simulations.
              </a>
            </td>
          <td>
            Khondamir R Rustamov, J. I. Razzokov, A. Baev
          </td>
          <td>2025-06-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genome-editing technologies have enabled the clinical development of allogeneic cellular therapies, yet the optimal gene-editing modality for multiplex editing of therapeutic T cell product manufacturing remains elusive. In this study, we conducted a comprehensive comparison of CRISPR/Cas9 nuclease and adenine base editor (ABE) technologies in generating allogeneic chimeric antigen receptor (CAR) T cells, utilizing extensive in vitro and in vivo analyses. Both methods achieved high editing efficiencies across four target genes, critical for mitigating graft-versus-host disease and allograft rejection: TRAC or CD3E, B2M, CIITA, and PVR. Notably, ABE demonstrated higher manufacturing yields and distinct off-target profiles compared to Cas9, with translocations observed exclusively in Cas9-edited products. Functionally, ABE-edited CAR T cells exhibited superior in vitro effector functions under continuous antigen stimulation, including enhanced proliferative capacity and increased surface CAR expression. Transcriptomic analysis revealed that ABE editing resulted in reduced activation of p53 and DNA damage response pathways at baseline, along with sustained activation of metabolic pathways during antigen stress. Consistently, Assay for Transposase-Accessible Chromatin using sequencing data indicated that Cas9-edited, but not ABE-edited, CAR T cells showed enrichment of chromatin accessibility peaks associated with double-strand break repair and DNA damage response pathways. In a preclinical leukemia model, ABE-edited CAR T cells demonstrated improved tumor control and extended overall survival compared to their Cas9-edited counterparts. Collectively, these findings position ABE as superior to Cas9 nucleases for multiplex gene editing of therapeutic T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b4de0c2df1ae6186d4cb00a4a918147feb7d3e2" target='_blank'>
              Quadruple adenine base-edited allogeneic CAR T cells outperform CRISPR/Cas9 nuclease-engineered T cells.
              </a>
            </td>
          <td>
            N. Engel, Israel Steinfeld, Daniel E. Ryan, Kusala Anupindi, Samuel Kim, N. Wellhausen, Linhui Chen, Katherine Wilkins, D. Baker, Philipp C. Rommel, Danuta Jarocha, M. Gohil, Qian Zhang, Michael C. Milone, J. Fraietta, Megan Davis, R. Young, Carl H. June
          </td>
          <td>2025-05-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer is one of the major diseases threatening human health in the world. According to the latest global cancer statistics from the International Agency for Research on Cancer (IARC), there were approximately 20 million new cancer cases and 10 million cancer deaths worldwide. Amidst this global health concern, branched chain amino acids have emerged as key players, playing an important role in the occurrence and development of cancer. In certain malignancies like colorectal cancer, the average level of BCAA in tumor tissues is twice that in normal tissues. BCAA metabolism is intricately associated with the progression of multiple tumors and is modulated by diverse enzymes, including BCAT, BCKDH, and BCKDK. The metabolism of BCAA involves multiple enzymes and biochemical processes via signaling pathways such as PI3K/AKT/mTOR and AMPK/mTOR, etc. In addition, mTOR inhibitors show potential value in cancer treatment by regulating the metabolism and signaling pathways of tumor cells, which provides a new direction for anticancer efforts. Simultaneously, BCAAs are closely associated with tumor immunity, including NK cells, CD4+ T cells, and CD8+ T cells. At present, the research on BCAA metabolism and related enzymes and signaling pathways is still in progress, aiming at identifying new targets and ideas for cancer therapy, and combined therapy will become an important trend in the future. In this review, we discuss the role and mechanisms of BCAA metabolism in human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99fd20e5b7b551f0c2edb41f3bde9c5f0d1adce" target='_blank'>
              Branched-chain amino acid and cancer: metabolism, immune microenvironment and therapeutic targets
              </a>
            </td>
          <td>
            Hao Xiong, Ruiqi Liu, Keke Xu, Xiaoyan Chen, Luanluan Huang, Yiyi Shou, Yumei Huang, Hailong Sheng, Yanwei Lu, Haibo Zhang
          </td>
          <td>2025-06-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract The homologous recombination (HR) system repairs DNA double-strand breaks produced by the DNA damage response, which is a complex signaling pathway consisting of the key proteins BRCA1/2 and other DNA repair proteins, such as the ATM, PALB2, BARD1, RAD51, and Fanconi anemia proteins. Mutations and epigenetic alterations in HR-related genes may lead to homologous recombination deficiency (HRD), resulting in genomic instability and contributing to the development of certain solid tumors. The biological significance and molecular mechanism of BRCA1/2 mutation-related HRD are well understood, but the relationships of other HR-related genes and their variant forms with HRD have not been sufficiently studied. These genes exhibit multiple forms of variation, including one or more HR genes, germline or somatic mutations, monoallelic or biallelic variants, and not all variants present HRD. Therefore, HRD is usually defined as HR-related gene variation, but recent studies have shown that defining it as the combined score of loss of heterozygosity, LST and TAI, known as the HRD score, can more accurately assess genomic instability. In patients with HRD, platinum-based therapy and poly ADP-ribose polymerase enzyme inhibitor (PARPi) have been shown to perform well in ovarian, breast, and prostate cancers. For gastrointestinal cancer (GI cancer), HRD has been relatively well studied in pancreatic cancer, but its role in other cancers has rarely been reported. Herein, we review the pathogenesis and predictive value of HRD, including the use of platinum drugs, PARPi, and immunotherapy, in digestive system tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b86ad0d0eb3efaff60321645af70d194c1741a2" target='_blank'>
              The prognostic and predictive value of homologous recombination deficiency in gastrointestinal cancer
              </a>
            </td>
          <td>
            Beibei Jiang, Yile Shang, Xiang Zhang, Wenguang He, Hanju Hua, Feng Ye, Xile Zhou, Yandong Li, Weixiang Zhong, Wei-wei Jiang, Guosheng Wu
          </td>
          <td>2025-06-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01390d6c99756b411d8005c0d7bc115177a2c238" target='_blank'>
              A Bayesian Approach for Identifying Driver Mutations within Oncogenic Pathways through Mutual Exclusivity
              </a>
            </td>
          <td>
            Xinjun Wang, Caroline E. Kostrzewa, Allison Reiner, Ronglai Shen, Colin B Begg
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>