<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-19 09:30:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>36</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d685ee84bfeabbf621f354633519c29df6bcad68" target='_blank'>
              Abstract 4399: Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer
              </a>
            </td>
          <td>
            Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew W. Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4abf519afad0178ba59d0cd17f3f428071bf92cb" target='_blank'>
              Abstract 2674: Cellular plasticity facilitates therapy resistance through enhancing adaptability
              </a>
            </td>
          <td>
            Alicia Bjornberg, Xierali Aobuli, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Jacob G. Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, A. Marusyk
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19dad7b139ef16ca0124619b3b70027a193368b" target='_blank'>
              Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Andrew Liao, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, M. Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2397ca3adf8845c9715ac4636c6b9da727b026" target='_blank'>
              Epigenetic dynamics and molecular mechanisms in oncogenesis, tumor progression, and therapy resistance.
              </a>
            </td>
          <td>
            Mohammed Kaleem, Lubna Azmi, N. Shahzad, Murtada Taha, Shiv Kumar, Md. Ali Mujtaba, Abdulaziz Ali H Hazazi, A. Kayali
          </td>
          <td>2025-05-13</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43efa757ce011d3db682c5cad92c0307f3b656c7" target='_blank'>
              The Role of Phenotypic Plasticity in Adaptation to Treatment and Prospective Plasticity Drivers in Hepatoblastoma
              </a>
            </td>
          <td>
            Emilia Chen, Christie English, Alejandro Allo Anido, Siân Hamer, Stefano Cairo, Alejandra Bruna
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9151033e1d6669563430e41bef627c6f841c5064" target='_blank'>
              Abstract 5551: From persistence to resistance: insights into the mechanisms of drug tolerance in cancer therapy using patient-derived organoids
              </a>
            </td>
          <td>
            Yasmine Abouleila, Timo Voskuilen, M. Doorn, Roel Verkerk, Joris Maas, C. Veríssimo, Sylvia F. Boj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecdb6f080369d4f144052961e7b7ab0cd6c99ec" target='_blank'>
              Abstract 539: Secretory EMT fuels ERBB3 axis activation in resistance to EGFR inhibitors
              </a>
            </td>
          <td>
            A. Morselli, Donatella Romaniello, Chiara Miroglio, Martina Mazzeschi, M. Sgarzi, F. Pagano, C. Girone, Francesca Ambrosi, W. Kothalawala, Gyorffy Balazs, A. De Giglio, Michelangelo Fiorentino, Andrea Ardizzoni, Yosef Yarden, M. Lauriola
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chemotherapy remains a prevalent strategy in cancer therapy; however, the emergence of drug resistance poses a considerable challenge to its efficacy. Most drug resistance arises from the accumulation of genetic mutations in a minority of resistant cells. The mechanisms underlying the emergence and progression of cancer resistance from these minority-resistant cells (MRCs) remain poorly understood. This study employs force-induced remnant magnetization spectroscopy (FIRMS) alongside various biological investigations to reveal the mechanical pathways for MRCs fostering drug resistance and tumor progression. The findings show that minority Paclitaxel-resistant cancer cells have enhanced mechanical properties. These cells can transmit high-intensity forces to surrounding sensitive cells (SCs) through the force transducer, Merlin. This force transmission facilitates the assimilation of surrounding SCs, subsequently strengthening the contraction and adhesion of tumor cells. This process is termed "mechano-assimilation," which accelerates the development of drug resistance and tumor progression. Interestingly, disturbances and reductions of mechano-assimilation within tumors can restore sensitivity to Paclitaxel both in vitro and in vivo. This study provides preliminary evidence highlighting the contribution of MRCs to the development of drug resistance and malignancy, mediated through mechanical interactions. It also establishes a foundation for future research focused on integrating mechanical factors into innovative cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f46cb96deeea2cf4ed683e45346444a026b4be6" target='_blank'>
              Mechanisms and Therapeutic Strategies for Minority Cell-Induced Paclitaxel Resistance and Tumor Progression Mediated by Mechanical Forces.
              </a>
            </td>
          <td>
            Xueyan Feng, Di Zhang, Guoxun Wang, Liwei Lu, Feng Feng, Xiuyu Wang, Chanchan Yu, Yahong Chai, Jin Zhang, Wenchao Li, Jing Liu, Hongxia Sun, Li Yao
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer drug resistance arises not only from selection of resistant clones, but also through rapid activation of adaptive transcriptional programs. One mechanism of transcriptional regulation involves N6-methyladenosine (m6A) RNA modification, which dynamically regulates mRNA processing and alternative splicing, ultimately impacting cell fate and differentiation. In prostate cancer (PC), resistance to systemic therapies such as the androgen receptor pathway inhibitor (ARPI) enzalutamide is associated with a host of well-documented androgen receptor (AR) alterations, including amplification, mutation, and alternative splicing. Given these functions, we hypothesized that m6A modifications play a role in the transition to enzalutamide resistance in PC. To test this, we used methyl-RNA-immunoprecipitation followed by sequencing (MeRIP-seq) in parallel with RNA-seq to identify gene transcripts that were both differentially methylated and differentially expressed between enzalutamide-sensitive and enzalutamide-resistant PC cells. We filtered and prioritized these genes using clinical and functional database tools, including Gene Ontology (GO) enrichment analysis and Gene Set Enrichment Analysis (GSEA), The Cancer Genome Atlas (TCGA), and the Oncology Research Information Network (ORIEN) avatar. Using this approach, we identified 487 transcripts that were both differentially methylated and differentially expressed and validated six of the top 12 candidates via targeted qPCR and MeRIP-PCR. One of these, THBS1, was found to have increased m6A level associated with decreased transcript levels in enzalutamide-resistant cells, a finding recapitulated in publicly available preclinical and clinical data. Moreover, in enzalutamide-sensitive cells, depletion of THBS1 by siRNA-knockdown induced resistance to enzalutamide. While THBS1 has previously been implicated in aggressive PC phenotypes, we now show that THBS1 downregulation directly contributes to a rapid transition to enzalutamide resistance, suggesting a novel role for this gene in PC hormonal therapy resistance. These results constitute the first comprehensive epitranscriptomic profiling of ARPI resistance and identify THBS1 as a potential driver of acute resistance in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7bc3dab4afa38c6c5372774a5530ea7583082a1" target='_blank'>
              Combined Transcriptomic and Epitranscriptomic Profiling Identifies THBS1 as A Regulator of Enzalutamide Resistance in Prostate Cancer
              </a>
            </td>
          <td>
            Emmanuelle Hodara, Lisa Swartz, Aubree Mades, Daniel Bsteh, Tong Xu, S. Rhie, A. Goldkorn
          </td>
          <td>2025-04-21</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer has consistently posed a significant challenge on a global scale, with rising incidence and mortality. Despite the tremendous advancements in cancer therapeutics throughout the years, chemotherapy remains the current standard treatment in cancer management. As a result, chemotherapy resistance among cancer patients has emerged as a major clinical challenge, causing unsatisfactory responses to the treatment. Chemotherapy resistance is closely related to the dysregulation of a normal cellular process called ‘pre-mRNA Alternative Splicing’, where one gene can generate various protein forms with different structures and functions. A comprehensive understanding of dysregulated alternative splicing mechanisms is essential to propose novel strategies in cancer research aimed at identifying new and improved methods to maximise treatment efficacy and overcome chemotherapy resistance. Therefore, this review highlights the complexity of chemotherapy resistance and the potential roles of alternative splicing in overcoming resistance to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c315ff50467ff3617156ad8cf2b1b2d1ebcf6f" target='_blank'>
              Aberrant Splicing as a Mechanism for Resistance to Cancer Therapies
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df82979a5771df69f4d6ba63039b4660f6ec17f" target='_blank'>
              Abstract 5065: Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD
              </a>
            </td>
          <td>
            Naomi Mfonfu, F. Braye, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey I. Nikolaev, Andrey A. Yurchenko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/221d45fdda028b3fdec5da9d6b92cc4847a8b483" target='_blank'>
              Understanding the Role of Toggle Genes in Chronic Lymphocytic Leukemia Proliferation
              </a>
            </td>
          <td>
            Olga Sirbu, Gunjan Agarwal, Alessandro Giuliani, Kumar Selvarajoo
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7191d1be8112623ab8fc75b326b24bf447563f16" target='_blank'>
              Abstract 1257: Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer
              </a>
            </td>
          <td>
            J. Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, M. T. Nguyen, Yilin Li, G. Marchi, A. Rajavuori, Heidi Koskela, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Newcastle disease virus (NDV) is a promising oncolytic agent with a non-segmented, negative-sense single-stranded RNA (ssRNA) genome of approximately 15 kb. While NDV selectively replicates in and lyses a wide range of human cancer cells, a subset of these cells develop persistent infections, potentially compromising the therapeutic efficacy of NDV-based treatments. To investigate the molecular basis of persistent infection, we performed transcriptome profiling of TCCSUP bladder cancer cells persistently infected with the NDV AF2240 strain. Deep sequencing using Illumina HiSeq 2000 was conducted in triplicate, and the resulting viral and host reads were separated and analyzed. Using Integrative Genomic Viewer (IGV) software, we identified several nucleotide variants linked to persistence. Specifically, nucleotide alterations included a deletion at 359A→C and a substitution at 1653C→T within the nucleoprotein (NP) gene, as well as an insertion at 3338C→T in the matrix protein (M) gene. Additionally, a GGG base insertion was detected at position 2290 in the phosphoprotein (P) gene. Crucially, we observed truncations in the hemagglutinin-neuraminidase (HN) gene (nt 8263–8390) and the large polymerase (L) gene (nt 6203–6342). These mutations and truncations suggest significant disruptions in viral replication, assembly, and host cell attachment, potentially facilitating viral persistence in bladder cancer cells. Understanding these genomic alterations provides valuable insights into the mechanisms driving viral persistence and could inform strategies to optimize the oncolytic efficacy of NDV in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46c1e75575001a56bcabf1b3e8302abfbe59f19d" target='_blank'>
              Genomic alterations in persistently infecting oncolytic Newcastle disease virus reveal mechanisms of viral persistence in bladder cancer cells
              </a>
            </td>
          <td>
            U. Ahmad, D. Chau, SC Chan, K. Yusoff, A. Veerakumarasivam, S. Abdullah
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is heterogeneous and variability in drug sensitivity is widely documented across cancer types. Adaptive therapy is an emerging modality of cancer treatment that leverages this drug resistance heterogeneity to improve therapeutic outcomes. Current standard treatments typically eliminate a large fraction of drug-sensitive cells, leading to drug-resistant relapse due to competitive release. Adaptive therapy aims to retain some drug-sensitive cells, thereby limiting resistant cell growth by ecological competition. While early clinical trials of such a strategy have shown promise, optimisation of adaptive therapy is a subject of active study. Current methods largely assume cell phenotypes to remain constant, even though cell-state transitions could permit drug-sensitive and -resistant phenotypes to interchange and thus escape therapy. We address this gap using a deterministic model of population growth, in which sensitive and resistant cells grow under competition and undergo cell-state transitions. Based on the model’s steady-state behaviour and temporal dynamics, we identify distinct balances of competition and phenotypic transitions that are suitable for effective adaptive versus constant dose therapy. Our data indicate that under adaptive therapy, models with cell-state transitions show a higher frequency of fluctuations than those without, suggesting that the balance between ecological competition and phenotypic transitions could determine population-level dynamical properties. Our analyses also identify key limitations of applying phenomenological models in clinical practice for therapy design and implementation, particularly when cell-state transitions are involved. These findings provide an overall perspective on the relevance of phenotypic plasticity for emerging cancer treatment strategies using population dynamics as an investigation framework. Significance Statement Drug-sensitive and -resistant cancer cells can compete with each other within the same tumour, and adaptive therapy exploits this competition to control overall tumour growth. The fact that sensitive and resistant cell types can switch phenotypes complicates the implementation of adaptive therapy. Our ODE-based theoretical ecology framework shows that asymmetric competition favouring sensitive cells generally benefits therapy outcomes, while phenotypic plasticity is usually detrimental. Our model also provides cell population-level indicators that can help predict the underlying balance between competition and plasticity. Phenomenological models are limited by whether their parameters can be reliably identified given scarce data, and our study illuminates the scope of such models in understanding cancer population dynamics and the need for mechanistic modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5674ed15d6d0f458d4dbdaa153d7277526c19b0b" target='_blank'>
              Impacts of competition and phenotypic plasticity on the viability of adaptive therapy
              </a>
            </td>
          <td>
            B. Vibishan, Paras Jain, Vedant Sharma, K. Hari, Claus Kadelka, Jason T. George, M. Jolly
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Summary We demonstrate that carcinogenesis and multi-therapy resistance in multiple myeloma (MM)—a treatable yet incurable plasma cell malignancy—are driven by epigenetic dysregulation. In this new paradigm, genomic and cytogenetic events unlock epigenetic plasticity, reshaping MM cell biology to evade tumor microenvironment constraints and therapeutic pressure. These conclusions are derived from a newly assembled cohort of nearly 1,000 patients, spanning premalignant to late-stage refractory MM, comprehensively characterized at molecular and clinical levels. Our findings provide a unifying framework to explain inter-patient genomic heterogeneity and the emergence of therapy resistance in sequential samples without new genomic alterations. In conclusion, we propose targeting epigenetic plasticity-mediated plasma cell evasion as a promising therapeutic strategy in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19a7c4379c96f4e14d05eb168255be930f2fe17c" target='_blank'>
              Epigenetic Plasticity Drives Carcinogenesis and Multi-Therapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            A. Silva, Rafael R. Canevarolo, P. Sudalagunta, M. Meads, Maria Silva, Xiaohong Zhao, Dario Magaletti, R. Alugubelli, Gabriel De Avila, Erez Persi, F. Maura, Elissa Bell, Ryan T Bishop, Christopher Cubitt, S. Sansil, Wei Zhang, Jamie K Teer, Mingxiang Teng, Sean J Yoder, E. Siegel, Bijal Shah, T. Nishihori, Lori Hazlehurst, Conor C Lynch, O. Landgren, Oliver Hampton, Robert Gatenby, Daniel Sullivan, J. Brayer, William S. Dalton, John L Cleveland, Melissa Alsina, Rachid C Baz, K. Shain
          </td>
          <td>2025-04-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b60f1dd8ce9984467648ff9cf2c70d381e498a6" target='_blank'>
              Abstract 7502: Radiation-induced transcriptomic reprogramming of mesenchymal glioblastoma stem cells reveals key pathways and potential targets
              </a>
            </td>
          <td>
            Grace M. Rosner, Abigail Goen, Godwin Peasah-Darkwah, Jonathan Rodgers Gochicoa, Alos B. Diallo, Gokul Srinivasan, Elisa Bu Sha, Joshua Levy, Kelli B. Pointer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0847dafec64111106a227406998f89b030109af3" target='_blank'>
              Abstract LB288: Determination of the MYC- and TEAD-dependent transcriptome in pancreatic cancer
              </a>
            </td>
          <td>
            Brandon L. Mouery, Priya S. Hibshman, A. C. Edwards, Ryan D. Mouery, Run-Wei Yang, Jeffrey A. Klomp, Angie L Mordant, Thomas S. Webb, Aurora Cabrera, Laura E. Herring, Adrienne D. Cox, T. Baslan, C. Der, Clint A. Stalnecker
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a113f78cc11aa5cfca10446523fbea0eb55d404" target='_blank'>
              Abstract 1512: EGR1 regulates phenotypic plasticity to promote breast cancer progression upon metabolic stress through elaborating HERPUD1-mediated unfolded protein response
              </a>
            </td>
          <td>
            Yi-Zhen Wu, Ge-Jyun Liu, Shan-Ting Lei, Wei-Huan Zeng, Mei-Ju Juan, Yu-Shu Liu, Yong-Han Su, David K. Ann, Ching-Ying Kuo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf2305c32fc721604cdb952ddd1c89201b94faf9" target='_blank'>
              Abstract 6625: LinkPrep™ - a comprehensive assay for cancer genome characterization - enabling structural variant detection and discovery of enhancer hijacking
              </a>
            </td>
          <td>
            Ericca Stamper, Jonathon Torchia, Daniel Hwang, Chu Yuan Wu, Ek Danwanichakul, Cory Padilla, J. Z. Sanborn, Lisa Munding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217101cddf06b5ca15708fdd50f15de9f63e9eb9" target='_blank'>
              Abstract 5092: Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Dimitri Joseph, Alexandra Guillaume
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antimicrobial combination therapy is widely used to combat Mycobacterium tuberculosis (Mtb), yet resistance rates continue to rise. Mutator strains, with defects in DNA repair genes, drive resistance in other bacterial infections, but their role in Mtb remains unclear. Here, we study the contribution of single nucleotide polymorphisms (SNPs) in DNA Repair, Replication, and Recombination (3 R) genes to Mtb resistance. Through large-scale bioinformatics analysis of 53,589 whole-genomes, we identified 18 novel SNPs in lineages 2 and 4 linked to genotypic drug resistance in 3 R genes, covering 12.5% of clinical isolates with available genome sequences. Notably, a number of the detected SNPs were positively selected during Mtb evolution. Experimental tests showed that mutM, fpgg2, xthA, and nucS mutants had increased the mutation frequency compared to the wild type. Our findings highlight the role of 3 R gene mutations in resistance, emphasizing the need for surveillance to improve early detection and control strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e0f7d4e080ba3e241530434770bea308ca79e11" target='_blank'>
              Genome wide analyses reveal the role of mutator phenotypes in Mycobacterium tuberculosis drug resistance emergence
              </a>
            </td>
          <td>
            R. Zein-Eddine, A. Le Meur, S. Skouloubris, L. Jelsbak, G. Refrégier, H. Myllykallio
          </td>
          <td>2025-04-29</td>
          <td>npj Antimicrobials and Resistance</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R. Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, S. Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Mechanisms for resistance to cytotoxic cancer drugs are dependent on dynamic changes in the biochemistry of cellular pathways, information which is hard to obtain at the systems level. Here we use a deep functional proteomics implementation of the Cellular Thermal Shift Assay to reveal a range of induced biochemical responses to gemcitabine in resistant and sensitive diffuse large B cell lymphoma cell lines. Initial responses in both, gemcitabine resistant and sensitive cells, reflect known targeted effects by gemcitabine on ribonucleotide reductase and DNA damage responses. However, later responses diverge dramatically where sensitive cells show induction of characteristic CETSA signals for early apoptosis, while resistant cells reveal biochemical modulations reflecting transition through a distinct DNA-damage signaling state, including opening of cell cycle checkpoints and induction of translesion DNA synthesis programs, allowing bypass of damaged DNA-adducts. The results also show the induction of a protein ensemble, labeled the Auxiliary DNA Damage Repair, likely supporting DNA replication at damaged sites that can be attenuated in resistant cells by an ATR inhibitor, thus re-establishing gemcitabine sensitivity and demonstrating ATR as a key signaling node of this response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3932b7cf1d620393ace8a6959b61b212c07c27aa" target='_blank'>
              MS CETSA deep functional proteomics uncovers DNA repair programs leading to gemcitabine resistance
              </a>
            </td>
          <td>
            Ying Yu Liang, Khalidah Khalid, Hai Van Le, Hui Min Vivian Teo, Mindaugas Raitelaitis, Marc-Antoine Gerault, Jane Jia Hui Lee, Jiawen Lyu, Allison Chan, A. D. Jeyasekharan, Wai Leong Tam, Pär Nordlund, N. Prabhu
          </td>
          <td>2025-05-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c9f0f37888749fdd1b5acd90887f2484e915ec3" target='_blank'>
              Abstract 5553: Acquisition of adaptive and reversible drug-tolerance state to cabozantinib in renal cell carcinoma cell lines
              </a>
            </td>
          <td>
            A. Zaccagnino, Hiresh Ayoubian, Kerstin Junker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbfea92c86debe9c5d855419c64075ab48f3f011" target='_blank'>
              Single cell transcriptional evolution of myeloid leukaemia of Down syndrome
              </a>
            </td>
          <td>
            Mi K Trinh, Konstantin Schuschel, Hasan Issa, Rebecca Thomas, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, L. Mamanova, E. Prigmore, Emilia Robertson, A. Hodder, Anna Wenger, Nathaniel D Anderson, Holly J. Whitfield, T. Treger, J. Gonçalves-Dias, Karin C Straathof, David O’Connor, Matthew D. Young, Laura Jardine, Stuart Adams, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd0035313cd95f13c0ad0b1db10655d699eb842" target='_blank'>
              Abstract 2991: Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening
              </a>
            </td>
          <td>
            Chenlin Zhao, Sharif U. Ahmed, Xiyue Hu, R. S. Atwal, Zongjie Wang, Sophie Xiao, S. Kelley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf51075c6ced2d719b16f81522dd0e475d5f9454" target='_blank'>
              Abstract 4605: Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers
              </a>
            </td>
          <td>
            Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Primary mouse renal proximal tubule epithelial cells (moRPTECs) were immortalized by lentivirus transduction to create hTERT or SV40LgT (LgT) cell lines. Prior work showed a more pronounced injury and repair response in LgT versus hTERT cells after chemical challenge. We hypothesized that unique genomic changes occurred after immortalization, altering critical genes and pathways. RNA-seq profiling and whole-genome sequencing (WGS) of parent, hTERT, and LgT cells showed that 92.5% of the annotated transcripts were shared, suggesting a conserved proximal tubule expression pattern. However, the cell lines exhibited unique transcriptomic and genomic profiles different from the parent cells. Three transcript classes were quite relevant for chemical challenge response—Cyps, ion channels, and metabolic transporters—each important for renal function. A pathway analysis of the hTERT cells suggested alterations in intermediary and energy metabolism. LgT cells exhibited pathway activation in cell cycle and DNA repair that was consistent with replication stress. Genomic karyotyping by combining WGS and RNA-seq data showed increased gene copy numbers in chromosome 5 for LgT cells, while hTERT cells displayed gene copy losses in chromosomes 4 and 9. These data suggest that the exaggerated transcriptional responses of LgT cells versus hTERT cells result from differences in gene copy numbers, replication stress, and the unique selection processes underlying LgT or hTERT immortalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5fc2423810ee9671b81c235e2952419eb5bbc3d" target='_blank'>
              hTERT and SV40LgT Renal Cell Lines Adjust Their Transcriptional Responses After Copy Number Changes from the Parent Proximal Tubule Cells
              </a>
            </td>
          <td>
            B. A. Merrick, Ashley M Brooks, Julie F. Foley, N. Martin, Rick D. Fannin, Wesley Gladwell, Kevin E. Gerrish
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca6f30d3b9e64a4de714f44e69384d7201b4a8d" target='_blank'>
              Abstract 440: Dual epigenetic targeting of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kyounghyun Kim, Ponmari Guruvaiya
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/094980891c477076ac9ee63d92403d7dd49a9f93" target='_blank'>
              Abstract 5506: Inactivation of mTOR is a strategy for tumor resistance to ribosome biogenesis inhibition
              </a>
            </td>
          <td>
            Wenjun Fan, Lijing Yang, Hester Liu, Steffie Pitts, Niladri Sinha, N.V. Rajeshkumar, H. Easwaran, Rachel Green, M. Laiho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The integrity of the genome is maintained by mismatch repair (MMR) proteins that recognize and repair base mismatches and insertion/deletion errors generated during DNA replication and recombination. A defective MMR system results in genome-wide instability and the progressive accumulation of mutations. Tumors exhibiting deficient MMR (dMMR) and/or high levels of microsatellite instability (termed “microsatellite instability high”, or MSI-H) have been shown to possess fundamental differences in clinical, pathological, and molecular characteristics, distinguishing them from their “microsatellite stable” (MSS) counterparts. Molecularly, they are defined by a high mutational burden, genetic instability, and a distinctive immune profile. Their distinct genetic and immunological profiles have made dMMR/MSI-H tumors particularly amenable to treatment with immune checkpoint inhibitors (ICIs). The ongoing development of biomarker-driven therapies and the evaluation of novel combinations of immune-based therapies, with or without the use of conventional cytotoxic treatment regimens, continue to refine treatment strategies with the goals of maximizing therapeutic efficacy and survival outcomes in this distinct patient population. Moreover, the resultant knowledge of the mechanisms by which these features are suspected to render these tumors more responsive, overall, to immunotherapy may provide information regarding the potential optimization of this therapeutic approach in tumors with proficient MMR (pMMR)/MSS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0727d40a1aa873459400ba4134581be209441f5d" target='_blank'>
              Deficient Mismatch Repair and Microsatellite Instability in Solid Tumors
              </a>
            </td>
          <td>
            Joy A. Awosika, James L. Gulley, Danielle M. Pastor
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, S. Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dec962af7d1ee05246dee3d378f9152a2d6c3e" target='_blank'>
              Genetic variants in NUDT15 gene their clinical implications in cancer therapy
              </a>
            </td>
          <td>
            Y. Jarrar, Maria Ghishan, Fatima Khirfan, Nancy Hakooz
          </td>
          <td>2025-04-14</td>
          <td>Drug Metabolism and Personalized Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematological malignancy. Patients with wild-type FLT3 relapsed or refractory (R/R) AML face significant therapeutic challenges due to the persistent lack of effective treatments. A comprehensive understanding of the mechanisms underlying chemotherapy resistance is needed to the development of effective treatment strategies. Therefore, we investigated the molecular mechanisms underlying cytarabine (Ara-C) resistance and daunorubicin (DNR) tolerance in Ara-C-resistant RHI-1 cells derived from the wild-type FLT3 AML cell line SHI-1. Quantitative analysis of intracellular drug concentrations, proteomics, and phosphoproteomics showed that DNR resistance in Ara-C-resistant RHI-1 cells is driven by metabolic remodeling toward mitochondrial metabolism, upregulation of DNA repair pathways, and enhanced reactive oxygen species (ROS) detoxification rather than reduced drug uptake. Moreover, targeting these compensatory mechanisms, particularly the OXPHOS complex I proteins, significantly improved the efficacy of both Ara-C and DNR. Conclusively, these findings highlight mitochondrial metabolism and DNA repair as critical factors in chemotherapy resistance and offer valuable insights into potential therapeutic targets for enhancing treatment outcomes in patients with wild-type FLT3 R/R AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49a8e37a3ebb622ad11d194102e12d1292163e3" target='_blank'>
              Mechanisms of chemotherapy failure in refractory/relapsed acute myeloid leukemia: the role of cytarabine resistance and mitochondrial metabolism
              </a>
            </td>
          <td>
            Soo Yeon Chae, Se-Young Jang, Jinhui Kim, Sehyun Hwang, D. Malani, O. Kallioniemi, S. Yun, Jong-Seo Kim, Hugh I Kim
          </td>
          <td>2025-04-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ac37cc5cb3a2b919750ef1ac7fd1c5b0996b8dd" target='_blank'>
              Abstract 2487: LINMAP: A framework for comprehensive cell division lineage reconstruction via molecular barcoding and machine learning
              </a>
            </td>
          <td>
            Lin Wang, Zheng Yin, Wenjuan Dong, J. Sheng, Li Yang, Kun Han, Dingcheng Gao, Stephen T. Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aecf1e2b76e187b7e7357a8c7f6d61cd00b2bcd" target='_blank'>
              Abstract 7503: Detection of genomic alterations in lenvatinib-resistant liver cancer cell lines
              </a>
            </td>
          <td>
            Junghyun Kim, Ji Young Kim, Gamin Han, BoYoon Kim, Hoon Kim, Yeup Yoon, Wonseok Kang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent clinical trials have suggested that solid cancers with mismatch repair (MMR) deficiency are highly responsive to immunotherapy, regardless of cancer types. Previous MMR-related studies on breast cancer have predominantly focused on germline variants. However, the somatic MMR alterations have not been comprehensively characterized in breast cancer. In this study, we integrated genomic, transcriptomic, and clinical data from over 3000 breast cancer cases across six public cohorts. Our findings revealed that 1.2% of breast cancers harbored oncogenic somatic MMR alterations, with triple-negative breast cancer (TNBC) demonstrating the highest mutation rate at 3.1%. Additionally, somatic MMR alterations were significantly associated with microsatellite instability-high (MSI-H) and MMR-related mutational signatures, indicating that somatic MMR alterations led to impaired function of the MMR system. Biallelic inactivation of MMR genes resulted in a more pronounced loss of MMR function compared to monoallelic inactivation. Importantly, these MMR alterations significantly increased the tumor mutational burden (TMB) and neoantigen load in breast cancer, regardless of MSI-H status. These findings indicate that the frequency of MMR alterations is highest in TNBC and that MMR alterations in breast cancer can lead to MMR functional deficiencies, suggesting that some patients harboring such alterations may benefit from immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6303d9f75dded8dc24ee6a6e2566fac41f5275" target='_blank'>
              Comprehensive Analysis of Oncogenic Somatic Alterations of Mismatch Repair Gene in Breast Cancer Patients
              </a>
            </td>
          <td>
            Yin Yan, Yang Wang, Junjie Tang, Xiao-ran Liu, Jichuan Wang, Guohong Song, Huiping Li
          </td>
          <td>2025-04-01</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent advancements in cancer therapies have improved clinical outcomes, yet therapeutic resistance remains a significant challenge because of its complex mechanisms. Among epigenetic factors, m5C RNA modification is emerging as a key player in cancer drug resistance, similar to the well-known m6A modification. m5C affects RNA metabolism processes, including splicing, export, translation, and stability, thereby influencing drug efficacy. This review highlights the critical roles of m5C in modulating resistance to chemotherapy, targeted therapy, radiotherapy, and immunotherapy. This review also discusses the functions of key regulators, including methyltransferases, demethylases, and m5C-binding proteins, as essential modulators of the m5C epigenetic landscape that contribute to its dynamic and complex regulatory network. Targeting these regulatory components offers a promising strategy to overcome resistance. We highlight the need for further research to elucidate the specific mechanisms by which m5C contributes to resistance and to develop precise m5C-targeted therapies, presenting m5C-focused strategies as potential novel anticancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf75ffd5990c032f1c8a0e853ea6d7108db5436" target='_blank'>
              5-Methylcytosine RNA modification and its roles in cancer and cancer chemotherapy resistance
              </a>
            </td>
          <td>
            Fang Li, Tingting Liu, Yajing Dong, Qianqian Gao, Rongzhu Lu, Zhiyong Deng
          </td>
          <td>2025-04-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54ccd0aa3ae124063240ed82d6af46f8d425d72" target='_blank'>
              Aneuploidy generates enhanced nucleotide dependency and sensitivity to metabolic perturbation.
              </a>
            </td>
          <td>
            R. Magesh, Arshia N Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, Jessica B Spinelli, S. Elledge, E. Watson
          </td>
          <td>2025-05-05</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Somatic mutations that increase hematopoietic stem cell (HSC) fitness drive their expansion in clonal hematopoiesis (CH) and predispose to blood cancers. Although CH frequently occurs with aging, it rarely progresses to overt malignancy. Population variation in the growth rate and potential of mutant clones suggests the presence of genetic factors protecting against CH, but these remain largely undefined. Here, we identify a non-coding regulatory variant, rs17834140-T, that significantly protects against CH and myeloid malignancies by downregulating HSC-selective expression and function of the RNA-binding protein MSI2. By modeling variant effects and mapping MSI2 binding targets, we uncover an RNA network that maintains human HSCs and influences CH risk. Importantly, rs17834140-T is associated with slower CH expansion rates in humans, and stem cell MSI2 levels modify ASXL1-mutant HSC clonal dominance in experimental models. These findings leverage natural resilience to highlight a key role for post-transcriptional regulation in human HSCs, and offer genetic evidence supporting inhibition of MSI2 or its downstream targets as rational strategies for blood cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, Chun-Jie Guo, Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, James M Allan, Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 activity promotes the survival and evolution of drug-tolerant persister cells during EGFR inhibitor resistance in lung cancer.
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-05</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef1937cc4baf82f21bbced66813abdd14ff16f89" target='_blank'>
              Application and research of genomic optical mapping technology in disease diagnosis.
              </a>
            </td>
          <td>
            Jing Quan, Yan-Qun Xiao, Da-Ru Lu, Yun Bao
          </td>
          <td>2025-04-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369cd649004283a085be1edef8035acdd6d9bbcf" target='_blank'>
              Abstract 5550: Understanding therapy resistance in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Ahmad Nasimian, Kinjal Shah, Julhash U. Kazi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: Chimeric antigen receptor (CAR) T-cell therapy has transformed cancer management over the past decades, offering new hope to many patients. However, its effectiveness is often limited due to cytokine release syndrome (CRS), a life-threatening inflammatory response. Despite its clinical relevance, the molecular mechanisms underlying CRS, specifically in CAR T-cell products, remain poorly understood. This study aims to identify hub genes and pathways in preinfusion CAR T-cell products associated with CRS development and evaluate their potential as therapeutic targets through drug-gene interaction analysis and immune cell correlation profiling. Methods: We examined gene expression data from 43 preinfusion clusters of differentiation 22 of CAR T-cell samples (CD22+), sourced from the Gene Expression Omnibus dataset GSE200296. Using the linear models for microarray data package, we identified differences in gene expression and conducted enrichment analyses to explore relevant biological pathways, including Kyoto Encyclopedia of Genes and Genomes and Gene Ontology terms. We built protein-protein interaction networks using the Search Tool for Retrieval of Interacting Genes/Proteins database to understand how these genes interact and pinpointed central "hub genes" with Cytoscape and the cytoHubba plugin. Our findings were validated using the GeneCards database (Weizmann Institute of Science, Israel) and an independent CRS-related dataset (GSE164805). Additionally, we analyzed immune cell populations and explored potential drug-gene interactions. Results: Our study identified 24 genes with changed expression levels: 16 were downregulated and eight were upregulated. We identified five hub genes, interleukin (IL)1B, IL15, CD276, NCR2, and CCL17, as key contributors in CRS, which were primarily implicated in immune-related pathways, including cytokine-cytokine receptor interactions, IL17 signaling, and TNF signaling. These genes were especially expressed in monocytes, macrophages, and dendritic cells, confirming that those immune cell types play a critical role in CRS development. Through drug-gene interaction analysis, we found prospective therapies, such as enoblituzumab (targeting CD276) and canakinumab (targeting IL1B), which might assist in reducing CRS severity. Conclusion: The study highlights IL1B, IL15, CD276, NCR2, and CCL17 as key CRS genes in preinfusion CAR T-cell products. Their dysregulation activity may contribute to the increased inflammation noted in CRS, pointing to a loss of regulatory control. Bringing us closer to better patient outcomes, these findings not only suggest that these genes could serve as valuable biomarkers for predicting CRS but also open the way for the development of more precise treatments such as combining drugs such as enoblituzumab and canakinumab, which might assist in reducing CRS severity and making CAR T-cell therapy safer and more effective, ultimately improving patient lives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd6910e44d2ba1a3106170ae84bee8bcf217bd" target='_blank'>
              Identification of Hub Genes and Pathways in Preinfusion Chimeric Antigen Receptor (CAR) T-cell Products Associated With Cytokine Release Syndrome
              </a>
            </td>
          <td>
            Somia A Khalafallah, Anas Mohammed Elzein, Israa M Faris, Gamila A Attaelmanan, Marwa F. Alamin, Alaa Almleeh, Mohammed Bader, Mohamed Alfaki
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11b087a0a9fc0566d0ad6c1ba6f886b3c8f91ce" target='_blank'>
              Abstract 6385: Heterogeneity in sensitivity to second-line inhibitors of therapy-resistant melanoma
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Pavithran T. Ravindran, Arjun Raj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2d4cdd26b00d23c9f2af55b08e343fcfbd17a4" target='_blank'>
              Abstract 1469: Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens
              </a>
            </td>
          <td>
            B. McBean, Priyanka S. Rana, Benjamin Hauk, A. Michmerhuizen, S. Lichtman-Mikol, Reine Abou Zeidane, Vesna Markovic, K. Wilder-Romans, Mingfang Tao, Alan P. Boyle, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9213a2dd4a654c0cc05ece63777d6864d7e262c3" target='_blank'>
              Role of Histone Deacetylases in Drug-Resistant Melanoma: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            B. Kalal
          </td>
          <td>2025-04-21</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaaf06b5fb2fb1345caa571c60786539fe457bf" target='_blank'>
              Somatic TP53 Mutations Drive T and NK Cell Dysfunction in AML and Can be Rescued by Reactivating Wild Type p53
              </a>
            </td>
          <td>
            Li Li, M. Muftuoglu, Edward Ayoub, Jiangxing Lv, Mahesh Basyal, A. Bidikian, Ran Zhao, Menon Prashant, Poo Yee Mak, Rupesh K. Kesharwani, Yuki Nishida, G. Issa, N. Varadarajan, B. Carter, Michael Andreeff
          </td>
          <td>2025-04-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf13a264e1cd78c0d16de17ca799be490bf7e832" target='_blank'>
              Challenges and Pathways in Regulating Next-Gen Biological Therapies.
              </a>
            </td>
          <td>
            Surendra Agrawal, Sunita Vaidya, Jitendra Patel, P. Jirvankar, Pravina Gurjar
          </td>
          <td>2025-04-23</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8de13c6fd974bf26d03a422a6ee56b54180c3" target='_blank'>
              Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, F. Cidre-Aranaz, A. Wenninger-Weinzierl, M. Zylka, Karla Queiroz, D. Kurek, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, T. G. Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="In recent years, synthetic lethality has become an important theme in the field of targeted cancer therapy. Synthetic lethality refers to simultaneous defects in two or more genes leading to cell death, whereas defects in any single gene do not lead to cell death. Taking advantage of the genetic vulnerability that exists within cancer cells, it theoretically has no negative impact on healthy cells and has fewer side effects than non-specific chemotherapy. Currently, targeted cancer therapies focus on inhibiting key pathways in cancer. However, it has been found that over-activation of oncogenic-related signaling pathways can also induce cancer cell death, which is a major breakthrough in the new field of targeted therapies. In this review, we summarize the conventional gene targets in synthetic lethality (PARP, ATR, ATM, WEE1, PRMT) and provide an in-depth analysis of their latest potential mechanisms. We explore the impact of over-activation of pathways such as PI3K/AKT, MAPK, and WNT on cancer cell survival, and present the technical challenges of current research. Important theoretical foundations and insights are provided for the application of synthetic lethal strategies in cancer therapy, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a12057bb33fb82ecea65af89cddf52c4a9e3cd" target='_blank'>
              Perspectives on cancer therapy—synthetic lethal precision medicine strategies, molecular mechanisms, therapeutic targets and current technical challenges
              </a>
            </td>
          <td>
            Shixuan Peng, Mengle Long, Qisheng Chen, Zhijian Yin, Chang Zeng, Wanyong Zhang, Qingyang Wen, Xinwen Zhang, Weiqi Ke, Yongjun Wu
          </td>
          <td>2025-04-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Drug resistance results in poor outcomes for cancer patients. Adaptive therapy is a potential strategy to address drug resistance that exploits competitive interactions between sensitive and resistant subclones. Here, we showed that adapting carboplatin dose according to tumor response (adaptive therapy) significantly prolonged survival of murine ovarian cancer models compared to standard carboplatin dosing, without increasing mean daily drug dose or toxicity. Platinum resistant ovarian cancer cells exhibited diminished fitness when drug was absent in vitro and in vivo, which caused selective decline of resistant populations due to reduced proliferation and increased apoptosis. Conversely fitter, sensitive cells re-grew when drug was withdrawn. Using a bioinformatics pipeline that exploits copy number changes to quantify the emergence of treatment resistance, analysis of cell-free DNA obtained longitudinally from ovarian cancer patients during treatment showed subclonal selection through therapy, and measurements of resistant population growth correlated strongly with disease burden. These preclinical findings pave the way for future clinical testing of personalized adaptive therapy regimens tailored to the evolution of carboplatin resistance in individual ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872bfbc7aed3e8f94862c1f84bd0f838583a3b38" target='_blank'>
              Adaptive Therapy Exploits Fitness Deficits in Chemotherapy-Resistant Ovarian Cancer to Achieve Long-Term Tumor Control.
              </a>
            </td>
          <td>
            H. Hockings, E. Lakatos, Weini Huang, M. Mossner, Mohammed A Khan, Nikolina Bakali, Jacqueline McDermott, Kane Smith, Ann-Marie Baker, Trevor A. Graham, M. Lockley
          </td>
          <td>2025-04-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0da8bbe26d7b85bc1bd170b1562ddebfb5b49ba9" target='_blank'>
              Abstract 6382: DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance
              </a>
            </td>
          <td>
            F. Uddin, Samuel Tischfield, R. Somwar, Alexander Lim, Hyung Jun Woo, Christina J Falcon, Barbara P. Mello, Dennis Kinyua, Harsha Sridhar, P. Manoj, Hong Zhong, Juan Qiu, E. de Stanchina, M. Donoghue, Marc Ladanyi, E. Redin, Charles M. Rudin, Á. Quintanal-Villalonga, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58e72b48369a6d6279bf45ff8835cc869b1ce39" target='_blank'>
              CRISPR screens in the context of immune selection identify CHD1 and MAP3K7 as mediators of cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Alex Watterson, G. Picco, Vivien Veninga, Youhani Samarakoon, C. M. Cattaneo, Sara F Vieira, Emre Karakoc, S. Bhosle, Thomas W Battaglia, Sarah Consonni, Timotheus Y. F. Halim, E. Voest, Mathew J. Garnett, Matthew A. Coelho
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d1574fe2e02ef3f4ee02ae939fc0c878b2510b" target='_blank'>
              Abstract 2392: The use of transcriptomic analysis to identify novel drug targets for DSA drug combination studies in acute myeloid leukemia
              </a>
            </td>
          <td>
            William A Chen, Valery Filippov, Jie Fang, Pedro Ochoa, C. Casiano, Kristopher E Boyle, S. Dovat, Hongjian Jin, Jun Yang, Olivia L Francis-Boyle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dead0a4e6712a90fb575b926eb0a4af6b736d577" target='_blank'>
              Abstract 1259: Studying the role of chromatin accessibility in lineage plasticity of Rb1-deficient EGFR-mutant lung adenocarcinoma
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Lynch syndrome (LS), caused by germline pathogenic variants in the mismatch repair genes, leads to high rates of frameshift-derived neopeptide (FSDN) expression due to microsatellite instability (MSI). While colorectal cancer (CRC) prevention is effective, most LS-related tumors lack such strategies. Cancer vaccines targeting FSDNs offer a promising approach for immune interception in LS. This study aimed to identify and validate LS-related FSDNs to develop vaccines for cancer prevention. Methods We identified LS-related coding MS mutations and predicted FSDN with high coverage on common Human Leukocyte Antigen (HLA)-I and II alleles. We validated FSDN-associated mutations in colorectal adenomas (CrAD), endometrial cancers (EC), and CRC samples from patients with LS, non-LS tumors, and cell lines. Immunogenicity was assessed through interferon (IFN)-γ enzyme-linked immunospot and flow cytometry analysis of tissue-infiltrating lymphocytes (TILs) from LS carriers. Results We prioritized 53 HLA-I and 45 HLA-II FSDNs in MSI tumors using in silico predictions. Validation revealed 86.7% of FSDN-associated mutations present in LS-CRC samples, with a median of 7.67 (6.5–9) mutations in CrADs and 6.02 (2–10) in CRCs. Sequencing of CrAD and EC samples showed 95% and 77.5% of predicted FSDN-associated mutations, respectively. MSI cancer cell lines transcribed 69.8% of FSDNs. Immunogenicity assays showed that 71% of potential FSDNs elicited IFN-γ responses, with a median of 7.37 (1–10.75) HLA-I and 6 (2–5.75) HLA-II FSDNs per patient. After prioritizing 24 FSDN, in a cohort of 19 LS-derived samples (4 CrAD and 15 normal mucosa), 52% (10/19) demonstrated T-cell reactivity to an HLA-I neoantigen pool. CD8+CD137+ activation markers increased significantly (p=0.037) over time and peptide-specific cells were detected by pentamer staining. Conclusions Our predicted FSDN set has optimal coverage among LS carriers and can induce IFN-γ inflammatory responses in LS-derived TILs, offering an opportunity for vaccine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4e379b9e444406c0c4ab08355c1d7dbd836a26" target='_blank'>
              Discovery and validation of frameshift-derived neopeptides in Lynch syndrome: paving the way for novel cancer prevention strategies
              </a>
            </td>
          <td>
            Cristina Bayó, Giancarlo Castellano, Fátima Marín, J. Castillo-Iturra, T. Ocaña, Hardeep Kumari, M. Pellisé, Leticia Moreira, L. Rivero, M. Daca-Alvarez, Oswaldo Ortíz, S. Carballal, Rebeca Moreira, Júlia Canet-Hermida, Marta Pineda, Capella Gabriel, Georgina Flórez-Grau, Manel Juan, D. Benitez-Ribas, Francesc Balaguer
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b24fd08026b81ce4d7ce4581da39b975952905" target='_blank'>
              Abstract 5108: Decoding tumor intrinsic and stroma-related drug resistance in EGFR mutant NSCLC
              </a>
            </td>
          <td>
            Radhika Koranne, Aimee Gleason, Emily Hensley, Xiaodong Wang, Alba Font-Tello, Brian Rabe, Fayaz Seifuddin, Eugene Drokhlyansky, Kartik Subramanian, Martino Colombo, Jason R. Dobson, Mavrakis J. Konstantinos, Leila Dardaei, Aaron N. Hata
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60dc48b94558a55ad5ff2dbcdf85170057c59b44" target='_blank'>
              Natural resistance to cancers in long-lived mammals: genomic mechanisms and experimental evidence to explain Peto's paradox.
              </a>
            </td>
          <td>
            Linxia Sun, Zhikang Xu, Mengqi Shuai, Chengxu Li, Guang Yang, Shixia Xu
          </td>
          <td>2025-03-21</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is the most common type of acute leukemia, accounting for 20% of cases in children and adolescents. Genome-wide studies have identified genes that are commonly mutated in AML, including many epigenetic regulators involved in either DNA methylation (DNMT3A, TET2, IDH1/2) or histone post-translational modifications (ASXL1, EZH2, MLL1). Several cell lines derived from AML patients are widely used in cancer research. Whether important differences in these cell lines exist remains poorly characterised. Results Here, we used RNA sequencing (RNA-Seq) to contrast the transcriptome of four commonly used AML-derived cell lines: THP-1, NOMO-1, MOLM-13 bearing the common initiating t(9;11) translocation, and MV4.11 bearing the t(4;11) translocation. Gene set enrichment analyses and comparison of key transcription and epigenetic regulator genes revealed important differences in the transcriptome, distinguishing these AML models. Among these, we found striking differences in the expression of clusters of genes located on chromosome 19 encoding Zinc Finger (ZNF) transcriptional repressors. Low expression of many ZNF genes within these clusters is associated with poor survival in AML patients. Conclusion The present study offers a valuable resource by providing a detailed comparative characterisation of the transcriptome of cell lines within the same AML subtype used as models for leukemia research. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11415-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a934cae686db7528797f8dea84b54367fe76eeb" target='_blank'>
              Transcriptomic characterisation of acute myeloid leukemia cell lines bearing the same t(9;11) driver mutation reveals different molecular signatures
              </a>
            </td>
          <td>
            Elise Georges, William Ho, Miren Urrutia Iturritza, L. Eory, Kamila Malysz, U. Sobhiafshar, Alan L Archibald, Daniel J. Macqueen, Barbara Shih, David Garrick, Douglas Vernimmen
          </td>
          <td>2025-03-25</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08d157066987d58c8cc7f8f1b1b2c36ec37eca2" target='_blank'>
              Abstract 5521: Integrative analysis of acquired drug resistance to KRAS(G12D) inhibitors
              </a>
            </td>
          <td>
            Ines Pulido Endrino, Laura C. Gunder, Malek G. Massad, Julian Carretero, Takeshi Shimamura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A recent study published in Nature by Maya M. Arce and colleagues unveils the role of central gene circuits in governing the delicate balance between T cell rest and T cell activation. 1 This work bridges fundamental molecular biology with ﬁ ndings in preclinical models, providing insights into context-speci ﬁ c gene regulation and potential therapeutic targets for immune modulation. The immune system relies on highly specialized cells to maintain homeostasis and respond to external threats. Among these, the CD4 + T cell compartment exhibits remarkable versatility, including T cell subsets with almost antagonistic functions, and that are capable of toggling between resting (quiescent) and activated states depending on environmental cues. The IL-2 receptor alpha subunit (IL-2R α or CD25) serves as a critical marker for T cell activation, with distinct expression pro ﬁ les in regulatory T cells (Treg) and effector T cells (Teff), characterized by either its constitutive or transient upregulation upon stimulation, respectively. Despite its importance, the trans-regulatory networks orchestrating these state-speci ﬁ c responses remain incompletely understood. Therefore, screening interventions that modulate IL-2R α expression can serve as a means of identifying genes involved in T cell activation, offering valuable insights by comparing the regulatory mechanisms between Teff and Treg cells. The primary objective of this study was to identify and characterize the regulators of IL-2R α expression across different T cell states. To this end, the authors conducted a series of CRISPR screens in primary human Treg and Teff cells to uncover context-dependent regulators and explore their role in maintaining resting and activated states. Using a designed library of 6,000 sgRNAs targeting over a thousand genes coding for transcription factors and chromatin modi ﬁ ers in resting and stimulated Teff and Treg, they mapped dynamic gene circuits that">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2782135135ae4e2ddd37866f821d202eefe30bbe" target='_blank'>
              From CRISPR screens to circuits: identifying key regulators in T cell activation and state transitions
              </a>
            </td>
          <td>
            Maria Silvia Roman Azcona, Toni Cathomen, C. Mussolino
          </td>
          <td>2025-04-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3627b48e9188c47ac8adb02879b131e72038b2ed" target='_blank'>
              Abstract 1600: Tracking adaptation and response to targeted therapies in EGFR-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Regina Hoo, Aaron C Tan, Dawn Lau, Komal Gupta, Kenneth Chow, Jia-Chi Yeo, Lan Ying Wang, Poh Sheng Joe Yeong, A. Skanderup, D. S. Tan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Acquired drug resistance is a main factor contributing to cancer therapy failure and high cancer mortality, highlighting the necessity to develop novel intervention targets. Circular RNAs (circRNAs), an abundant class of RNA molecules with a closed loop structure, possess characteristics including high stability, which provide unique advantages in clinical application. Growing evidence indicates that aberrantly expressed circRNAs are associated with resistance against various cancer treatments, including targeted therapy, chemotherapy, radiotherapy, and immunotherapy. Therefore, targeting these aberrant circRNAs may offer a strategy to improve the efficiency of cancer therapy. Herein, we present a summary of the most recently studied circRNAs and their regulatory roles on cancer drug resistance. With the advances in artificial intelligence (AI)-based bioinformatics algorithms, circRNAs could emerge as promising biomarkers and intervention targets in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7979b6f763a0f2eb2d4ff99ed47af16e914c8d95" target='_blank'>
              Circular RNAs modulate cancer drug resistance: advances and challenges
              </a>
            </td>
          <td>
            Jinghan Hua, Zhe Wang, Xiaoxun Cheng, Jiaojiao Dai, Ping Zhao
          </td>
          <td>2025-03-28</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6aceef50ec995a57bf7b1b3451c3358ae2d11aa" target='_blank'>
              Abstract 1255: Apoptotic DNase DFFB mediates cancer acquired drug resistance by suppressing persister cell intrinsic interferon signaling
              </a>
            </td>
          <td>
            August F. Williams, David A. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X. Wang, B.E. Mauch, Ariel H. Nguyen, Michelle H. Paw, Rhea Plawat, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J Hangauer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874bff63660c812dc6306123212ce70b86ccda05" target='_blank'>
              Abstract 6789: Identification of DGKA as a regulatory factor for paclitaxel resistance in lung cancer based on CRISPR screen and transcriptome sequencing
              </a>
            </td>
          <td>
            Jiang Luo, Mini Zhang, Jiawen Tang, Qingwei Wang, Yu Yuan, Min Xia, Yufei Yan, Yushuang Tan, Conghua Xie, Yan Gong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background About 40% of relapsed or non-responder tumors exhibit therapeutic resistance in the absence of a clear genetic cause, suggesting a pivotal role of intracellular communication. A deeper understanding of signaling pathways rewiring occurring in resistant cells is crucial to propose alternative effective strategies for cancer patients. Methods To achieve this goal, we developed a novel multi-step strategy, which integrates high sensitive mass spectrometry-based phosphoproteomics with network-based analysis. This strategy builds context-specific networks recapitulating the signaling rewiring upon drug treatment in therapy-resistant and sensitive cells. Results We applied this strategy to elucidate the BCR::ABL1-independent mechanisms that drive relapse upon therapy discontinuation in chronic myeloid leukemia (CML) patients. We built a signaling map, detailing - from receptor to key phenotypes - the molecular mechanisms implicated in the control of proliferation, DNA damage response and inflammation of therapy-resistant cells. In-depth analysis of this map uncovered novel therapeutic vulnerabilities. Functional validation in patient-derived leukemic stem cells revealed a crucial role of acquired FLT3-dependency and its underlying molecular mechanism. Conclusions In conclusion, our study presents a novel generally applicable strategy and the reposition of FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for CML relapsed patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02185-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff332b1ad425972fa586920b8d83716a55d5548" target='_blank'>
              A network-based approach to overcome BCR::ABL1-independent resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böttcher, Thomas Fischer, Livia Perfetto, Francesca Sacco
          </td>
          <td>2025-04-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi K. Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, C. Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD) is a rare and highly aggressive cancer predisposition syndrome caused by biallelic germline mutations in mismatch repair genes. This condition is characterized by early-onset malignancies across multiple organ systems, including central nervous system tumors, hematological cancers, and gastrointestinal malignancies. CMMRD-associated tumors exhibit hypermutation and microsatellite instability, resulting in a high tumor mutation burden and rendering these malignancies responsive to immune checkpoint inhibitors (ICIs). ICIs targeting programmed cell death protein-1 and programmed cell death ligand 1 have demonstrated remarkable efficacy, particularly in hypermutated tumors, providing durable responses and improving survival outcomes. Advances in genetic and molecular diagnostics have enhanced the ability to identify CMMRD early, allowing for the implementation of comprehensive surveillance programs and improved management strategies. A multidisciplinary and individualized approach is essential for managing CMMRD patients. This review underscores the importance of early diagnosis, surveillance, and emerging therapeutic approaches to improve outcomes and quality of life for individuals and families affected by this devastating syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf20191865645f39f87803bf02b422fa40cafb91" target='_blank'>
              Constitutional Mismatch Repair Deficiency, the Most Aggressive Cancer Predisposition Syndrome : Clinical Presentation, Surveillance, and Management
              </a>
            </td>
          <td>
            Eungu Kang, Jin Kyung Suh, Sang-Dae Kim
          </td>
          <td>2025-04-29</td>
          <td>Journal of Korean Neurosurgical Society</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a1965c162640161a9b607e38387b4fb02d284d" target='_blank'>
              Abstract 2104: Epigenetic, transcriptomic and genetic analyses of HR+/HER2- primary and metastatic breast cancer results in cohort stratification by aggressiveness and reveals heterogeneous mechanisms of endocrine therapy resistance
              </a>
            </td>
          <td>
            Jomar Sangalang, Tony Sourisseau, V. Scott, Diep Tran, Nusaibah Ibrahimi, A. Tran-Dien, Hakim Benkirane, N. Servant, Fabrice André, Sergey I. Nikolaev
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) harboring BCR/ABL-T315I mutation has been a challenging obstacle for targeted therapy due to the acquired resistance to tyrosine kinase inhibitor (TKI)-based therapy. Thus, it is especially urgent to investigate more effective therapeutic targets to overcome T315I-induced resistance. Here, we reported that BCR/ABL-T315I mutant CML cells possessed a long-term proliferative capacity and tolerance to metabolic stress. Importantly, we also found that selenoamino acid metabolism was increased in the bone marrows of BCR/ABL-T315I patients compared with non-T315I patients by GSEA from RNA-Seq data. Indeed, GPX1 was highly expressed in T315I mutant cells, while knockout of GPX1 significantly suppressed cell proliferation and triggered apoptosis under glucose-deprived condition. GPX1 knockout showed decreased cell metabolism signaling as well as mitochondrial gene expression by RNA-Seq. Mechanistically, GPX1 maintained mitochondrial activity and oxygen consumption rate (OCR), retaining mitochondrial redox homeostasis and oxidative phosphorylation (OXPHOS). Additionally, mercaptosuccinic acid (MSA), a GPX inhibitor, inhibited CML colony formation and induced cell apoptosis under glucose-free condition. Therefore, GPX1 is a promising therapeutic target to overcome drug resistance induced by the T315I mutation, which provides a novel approach for BCR/ABL-T315I CML treatment by disturbing mitochondrial OXPHOS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9003b9be433ceb088a877bba6ada889eb7e9e1" target='_blank'>
              GPX1 confers resistance to metabolic stress in BCR/ABL-T315I mutant chronic myeloid leukemia cells
              </a>
            </td>
          <td>
            Jun-Dan Wang, Jin-Xing Wang, Zhi-Li Lin, Na Xu, Ling Zhang, Jia-Jun Liu, Rui Gao, Z. Long
          </td>
          <td>2025-05-09</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a common and highly lethal type of primary brain tumor in adults. Therapeutic failure is partly attributed to a fraction of Glioblastoma Stem Cells (GSCs) that show high levels of heterogeneity and plasticity. GSCs exist in a transcriptional gradient between two states: Developmental (Dev) and Injury Response (IR) in which IR-GSCs exhibit more invasive behaviors. While previous studies have identified fitness genes in GSCs, the genes required to establish and maintain the Dev and IR states remain poorly defined. To identify the regulators of the IR GSC state, we performed a phenotypic genome-wide CRISPR-Cas9 knockout (KO) screen in patient-derived GSCs based on cell surface expression of the IR marker CD44. Notably, we found that perturbations of the histone acetyltransferase EP300 in IR GSCs led to decreased CD44 cell surface expression, significant downregulation of gene expression signatures associated with the IR state, and to decreased self-renewal and invasion. Furthermore, genetic targeting of Ep300 in a mouse GBM model delayed tumor initiation and/or progression. Collectively, our results establish EP300 as a regulator of the IR state in GSCs and provide a mechanistic basis for its therapeutic targeting in GBM. Significance A genome-wide phenotypic CRISPR-Cas9 screen in a patient-derived Glioblastoma Stem Cell line identified the genes required to maintain the Injury-Response cellular state, with a focus on the histone acetyl transferase gene EP300. This study suggests how therapeutic targeting of cellular state could reduce the aggressiveness of GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72194daa3f94b0757cee54e56b558726a1fc4778" target='_blank'>
              Identification of Injury-Response State Determinants in Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Fatemeh Molaei, G. Macleod, Shahan Haider, Aleczandria S. Tiffany, Frank M Oteng, Jacob M. Berman, Arianna Skirzynska, Molly S. Shoichet, Daniel Schramek, Peter B. Dirks, Stéphane Angers
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Lynch Syndrome (LS) is an autosomal dominant disease characterized by germline heterozygous mutations in DNA mismatch repair (MMR) genes. High-risk LS patients may proceed to colorectal cancer (CRC). However, the drivers or biomarkers of LS benign colon tissue approaching malignant CRC are not completely understood. The similarity and difference between LS-related and nonLS-related CRC are also not well interrogated (LS-CRC vs. nonLS-CRC). This study aimed to understand the cellular changes during malignant transition in LS. Methods Single-cell RNA sequencing (scRNA-seq) was used to analyze paired biopsy samples from 3 patients with LS (cancer tissues vs. adjacent normal tissues, n=3). Single-nuclear RNA sequencing (snRNA-seq) was used to analyze a frozen biopsy sample from a patient with LS (n=1). scRNA-seq or snRNA-seq datasets from CRC were downloaded from the open source. Integrative computational analysis was performed to conclude the distinct pattern in the single-cell atlas. Immuno-histo-fluorescence staining (IHF) were also performed for three key markers. Results In the single-cell atlas, we observed an increase in primitive cancer stem-cell-like cells with high expression of a general cancer biomarker CarcinoEmbryonic Antigen-related Cell Adhesion Molecule 5 (CEACAM5) in the epithelium of the LS. Infiltration of immune cells and DNA repair biological activity are dramatically increased in LS carcinoma. The burden in LS is fundamentally elevated compared to that in CRC or healthy donors when the mutations are in the coding-sequence-wide range. Furthermore, T cell and macrophage-related tumor immunity is readily mobilized in carcinomas compared to paraCArcinomas. Conclusions This study provides single-cell transcriptomic resource using affected tissues from patients with Lynch Syndrome and describes an integrative profile covering the alterations (cancer stem cell markers, mutation burden, and tumor immunity) during the malignant transition from healthy to Lynch Syndrome and to colorectal cancer at the single-cell level. Highlights and Figure/Table Index (Take-Home Messages) Colon tissues from HD (n=4), patients with LS (n=6) and with CRC (n=3) were collected for snRNA-seq or scRNA-seq analysis; Diagnosis of the 6 LS patients are clearly supported by our pedigree documentations (Fig. 1 and Supp. Fig. 1) and briefed in Table 1; Among the 6 patients with LS, 1 for snRNA-seq analysis (a carcinoma tissue only), 3 for scRNA-seq analysis (3 paired para-carcinoma and carcinoma tissues) and 3 for experimental validations; (Fig. 1) Following studies are focused on the comparisons between HD, LS and CRC; In addition, comparisons between paired carcinoma and para-carcinoma tissues from patients with LS were also performed; (Fig. 2-7) When HD and LS are compared, a cancer stem cell marker CEACAM5 is readily detected by snRNA-seq analysis; (Fig. 5) When carcinoma and para-carcinoma from LS patients are compared, increased immune cell infiltration of and enhanced DNA repair activity in the malignant tissues is readily detected by scRNA-seq analysis; in addition, three upregulated markers in LS carcinoma (BACE2, GRPC5A and OLFM4) were identified and validated by immunohistoinfloresnce; (Fig. 3 and Fig. 4) SComatic algorithm-based mutation calling analysis suggests a comparable mutation burden between carcinoma and para-carcinoma from LS patients, but such mutation burdens are grossly greater than that from HD or that from patients with CRC; (Fig. 6)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c44a229dbcaa552459faeccde9844a06701be" target='_blank'>
              Tracing the stemness and malignant transition in a heritable colorectal cancer Lynch Syndrome by single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Junfeng Xu, Yuhang Li, Zhiqin Wang, Qianru Li, Aijun Liu, Jianqiu Sheng, Ge Dong, Lang Yang, Zhigang Cai
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Germline variants in the HSD3B1 gene influence response to hormone therapies. This study examined the “adrenal-permissive” allele (c.1100 C) and its impact on various somatic features in prostate tumors. Upon inferring HSD3B1 variant status from somatic DNA sequencing data in a cohort of 6550 primary and metastatic prostate tumors, we found that tumors harboring the adrenal-permissive CC genotype, as compared to the adrenal-restrictive AA genotype, had limited somatic changes in their exomes or transcriptomes. However, significant differences were observed when we examined immune regulatory pathways and immune cell populations. This was accompanied by differential gene expression of immunoglobulins, MHC class I and II molecules, and druggable cell surface genes. These features appeared to deviate across different metastatic biopsy sites. Altogether, while tumor-intrinsic features are not regulated by germline HSD3B1 c.1100 alleles, these genotypes instead may impact cell–cell interactions and the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5953257502a6a5fc55756fab20f962c1af1666a8" target='_blank'>
              The Influence of the Germline HSD3B1 Adrenal-Permissive Allele (c.1100 C) on the Somatic Alteration Landscape, the Transcriptome, and Immune Cell Infiltration in Prostate Cancer
              </a>
            </td>
          <td>
            Samuel Kellen, Allison Makovec, Carly D Miller, Shayan Nazari, Andrew Elliott, Aiden Deacon, Emily John, Nikitha Vobugari, N. Agarwal, Rana R McKay, Pedro C. Barata, Charles J. Ryan, Nima Sharifi, Justin H. Hwang, E. S. Antonarakis
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32e4d3a24ff91016f9e42d7a576edaca46c0ba68" target='_blank'>
              Abstract 5339: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, Emma Aleksandrovic, H. M. Patel, D. Sudhan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia Ma, Jeon Lee, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16cb06677804f48c2ac85f36bdf345c2b4df6264" target='_blank'>
              Abstract 431: Novel WRN drug resistant CDX models
              </a>
            </td>
          <td>
            Haiting Dai, Tiejun Bing, Huayi Qu, Huawei Pan, Wen-Jen Yu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26edad68bf6dd00a6af6b5832569dd252b00f30f" target='_blank'>
              Highly sensitive and specific markers for detection of mismatch repair deficiency by next-generation sequencing.
              </a>
            </td>
          <td>
            Ming-Tseh Lin, Eric S Christenson, Aparna Pallavajjala, James R. Eshleman
          </td>
          <td>2025-05-03</td>
          <td>American journal of clinical pathology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, Chung-Chi Hsu, Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aba9f5d2cf8bd1341acacd8ce88421b1a228c70" target='_blank'>
              Abstract 5547: High resolution target identification by unbiased forward genetics
              </a>
            </td>
          <td>
            Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J. Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335901b6f2e363bcd4c769050615cefdadb15cb6" target='_blank'>
              Mechanistic Insights into Proteomic Mutation-Phenotype Linkages from Tiling Mutagenesis Screens
              </a>
            </td>
          <td>
            Wei He, Jen-Wei Huang, Yalong Wang, Samuel B. Hayward, G. Leuzzi, Rongjie Fu, Shuyue Wang, Alina Vaitsiankova, Mark T. Bedford, Raphaël Guérois, Alberto Ciccia, Han Xu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy is a cornerstone of colorectal cancer (CRC) treatment; however, its therapeutic efficacy is often compromised by both intrinsic and acquired resistance in CRC cells. This study employed small interfering RNA (siRNA) technology to elucidate the functional roles of BAMBI, GADD34, NFKBIA, and NFKBID in CRC cell lines SW480 and HCT116. We assessed their impact on key cellular processes and radiation sensitivity. Gene silencing of all four target genes significantly suppressed CRC cell proliferation, migration, and invasion. Moreover, siRNA-mediated knockdown enhanced radiation sensitivity, as evidenced by a substantial increase in apoptosis and a marked reduction in cell viability compared with controls. These findings suggest that BAMBI, GADD34, NFKBIA, and NFKBID serve as critical regulators of CRC progression and radiation resistance. Overall, this study provides a mechanistic foundation for further exploration into the pathways underlying radiation resistance and underscores the potential for developing personalized radiotherapy strategies guided by molecular profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b02786eb68733b852aeed62e7695e4393e02cbb" target='_blank'>
              Mechanistic Insights into Radiation Resistance in Colorectal Cancer: Gene Exploration Study
              </a>
            </td>
          <td>
            Beamjun Park, Soohyeon Lee, Inyoung Jo, Donghyun Kang, Taewan Kim, Jaesung Ryu, Hyejeong Kong, Moojun Baek, Taesung Ahn
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/176561a82959305826a2da9c2604661ae413f694" target='_blank'>
              Abstract 2900: Musashi-1 regulated oncogenic switch enhances therapy resistance to DNA damage in glioblastoma
              </a>
            </td>
          <td>
            Xian Liu, Tsoi Pok Kong, Man Ding Wong, Liang Ting Lin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc24f6cab28b67012c2dbe001093787b08c40ec" target='_blank'>
              Abstract 6713: RB1 and TP53 restrict lineage plasticity in prostate cancer by regulating chromatin accessibility
              </a>
            </td>
          <td>
            Mauricio Rodrigo Flores, David W. Goodrich, Justine J Jacobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9a2666f037bf98ff78850cacfb5af10e4de653" target='_blank'>
              Abstract 1338: System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance
              </a>
            </td>
          <td>
            Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3847b0cc02af732174f2aba89f84c63f9e5ea11" target='_blank'>
              Proof-of-Concept of A DNA-Based Recording System for High-Throughput Functional Gene Screening
              </a>
            </td>
          <td>
            Shoichi Kato, Atsushi Ikemoto, Jun Isayama, Tetsuya Takimoto, Hideyuki Saya, Ken-ichi Hamada
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b82c7744ef91d882615ad02c557cbabf06a4d73" target='_blank'>
              Updates on DNA Repair Gene Deficiency in Colorectal Cancer (dMMR)
              </a>
            </td>
          <td>
            Gustavo Sevá-Pereira, C. S. R. Coy, Carlos Augusto Real Martinez
          </td>
          <td>2024-11-01</td>
          <td>Journal of Coloproctology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b575dc8351428b4d22b9c3382531e97fc82f1bcf" target='_blank'>
              Abstract 545: RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines
              </a>
            </td>
          <td>
            Elizabeth K Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In cancer, mutational processes act in concert with epigenetic reprogramming to endow malignant cells with hallmark properties that underpin tumorigenesis. Compared with the relatively rigid and slow processes of genetic evolution, the plastic nature of chromatin enables cells to adapt to a changing environment more rapidly. Multiple myeloma is characterised by high levels of inter- and intra-patient heterogeneity at both the genetic and epigenetic levels. Understanding the many layers of genetic and non-genetic evolution and their interplay is crucial to improve patient outcomes. In this short review, we discuss the most common and extensively characterised epigenetic alterations that occur during myeloma development. We also touch on emerging approaches to reverse the aberrant epigenome of myeloma cells as a treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb9ecdf7e50a09e0efbdf19061d3f2e07fadfcb" target='_blank'>
              Epigenetic reprogramming in multiple myeloma-Challenges and opportunities.
              </a>
            </td>
          <td>
            Subhasree Kumar, Lev M. Kats, Emily Gruber
          </td>
          <td>2025-04-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Homologous recombination between short repeated sequences, such as Alu sequences, can generate pathogenic chromosomal rearrangements. We used budding yeast to measure homologous recombination between short repeated his3 sequences located on non-homologous chromosomes to identify pathways that suppress spontaneous and radiation-associated translocations. Previous published data demonstrated that genes that participate in RAD9-mediated G2 arrest, the S phase checkpoint, and recombinational repair of double-strand breaks (DSBs) suppressed ectopic recombination between small repeats. We determined whether these pathways are independent in suppressing recombination by measuring frequencies of spontaneous recombination in single and double mutants. In the wild-type diploid, the rate of spontaneous recombination was (3 ± 1.2) × 10−8. This rate was increased 10–30-fold in the rad51, rad55, rad57, mre11, rad50, and xrs2 mutants, seven-fold in the rad9 checkpoint mutant, and 23-fold in the mec1-21 S phase checkpoint mutant. Double mutants defective in both RAD9 and in either RAD51, RAD55, or RAD57 increased spontaneous recombination rates by ∼40 fold, while double mutants defective in both the MEC1 (ATR/ATM ortholog) and RAD51 genes increased rates ∼100 fold. Compared to frequencies of radiation-associated translocations in wild type, radiation-associated frequencies increased in mre11, rad50, xrs2, rad51, rad55 and rad9 rad51 diploid mutants; an increase in radiation-associated frequencies was detected in the rad9 rad51 diploid after exposure to 100 rads X rays. These data indicate that the S phase and G2 checkpoint pathways are independent from the recombinational repair pathway in suppressing homology-directed translocations in yeast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c442344c7555fd746bb68962a7be775830f91d7" target='_blank'>
              Checkpoint and recombination pathways independently suppress rates of spontaneous homology-directed chromosomal translocations in budding yeast
              </a>
            </td>
          <td>
            Li Zeng, Mingzeng Sun, M. Fasullo
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294979244322fee861b6d9b92034605b4639b574" target='_blank'>
              Abstract 5763: How cancer drugs work, and fail: proteome-wide cellular thermal shift assay (CETSA) to define cellular mechanism of action and resistance of clinical cancer drugs
              </a>
            </td>
          <td>
            Pär Nordlund, Sara Lööf, A. Ramos, smaranda Bacanu, Marc-Antoine Gerault, Ying Yu Liang, Wai Leong TAM, N. Prabhu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, M. Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919d6be56b49f799484f0a056c28802836f4c288" target='_blank'>
              Abstract 2771: CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells
              </a>
            </td>
          <td>
            U. Sobhiafshar, N. T. Emre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b49562d5cbbdfc03f5130abae6836f4cec03143" target='_blank'>
              Abstract 4109: Accelerated aging in breast cancer survivors: the role of therapy-induced epigenetic modifications
              </a>
            </td>
          <td>
            Nikita Nikita, Zhengyang Sun, Swapnil Sharma, Amy Shaver, Victoria L Seewaldt, Ana María López, Grace Lu Yao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b05e86f87ff9d548ca0d97272bc2384f88e53a2" target='_blank'>
              Abstract 3643: AI driven discovery of novel synthetic lethal gene pairs for targeted cancer therapy
              </a>
            </td>
          <td>
            Amalia Soenens, Daniel S. Miller, Oliver Vipond, Florencia Skuras, Alfie Brennan, Anna Hercot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab08c733369ba98ccceec430e609487bfe090d5" target='_blank'>
              Abstract 1247: Characterizing the evolution of osimertinib resistance in EGFR mutated non-small cell lung cancer
              </a>
            </td>
          <td>
            Mina N. Dinh, Jinling Wu, Masahiro Hitomi, Maximilian Strobl, Jacob G. Scott
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="It is well established that hematologic malignancies are often considerably radiosensitive, which enables usage of far lower doses of therapeutic radiotherapy. This review summarizes the currently known genomic landscape of hematologic malignancies, particularly as it relates to radiosensitivity and the field of radiation oncology. By tracing the historical development of the modern understanding of radiosensitivity, we focus on the discovery and implications of pivotal mutated genes in hematologic malignancies such as TP53, ATM, and other genes critical to DNA repair pathways. These genetic insights have contributed significantly to the advancement of personalized medicine, aiming to enhance treatment precision and outcomes, and there is an opportunity to extend these insights to personalized radiotherapy. We explore the transition from early discoveries to the current efforts in integrating comprehensive genomic data into clinical practice. Specific examples from Hodgkin lymphoma, non-Hodgkin lymphoma, and plasma cell neoplasms illustrate how genetic mutations could influence radiosensitivity and impact subsequent radiotherapeutic response. Despite the advancements, challenges remain in translating these genetic insights into routine clinical practice, particularly due to the heterogeneity of alterations and the complex interactions within cancer signaling pathways. We emphasize the potential of radiogenomics to address these challenges by identifying genetic markers that predict radiotherapy response and toxicity, thereby refining treatment strategies. The need for robust decision support systems, standardized protocols, and ongoing education for healthcare providers is critical to the successful integration of genomic data into radiation therapy. As research continues to validate genetic markers and explore novel therapeutic combinations, the promise of personalized radiotherapy becomes increasingly attainable, offering the potential to significantly improve outcomes for patients with hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e274aa44601f7df10924839e8b158bfac4f727a9" target='_blank'>
              The art of war: using genetic insights to understand and harness radiation sensitivity in hematologic malignancies
              </a>
            </td>
          <td>
            N. Wijetunga, J. Yahalom, B. Imber, Thomas FitzGerald, Lara Hilal, Ji Hyun Hong, Jonathan Leeman
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532359eb783a00c7e5a0af7f6a8c8009cbbb7c55" target='_blank'>
              Abstract 2758: Methylome plasticity as a biomarker of treatment response in small cell lung cancer
              </a>
            </td>
          <td>
            D. Sacdalan, Sami Ul Haq, L. Zhan, J. J. Li, V. Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b23788a6f827c0b72ecb2ed6905b6d5b0e6258" target='_blank'>
              Abstract 3763: Genome-wide CRISPR screens identify critical targets to enhance CAR-NK cell antitumor potency
              </a>
            </td>
          <td>
            Alexander Biederstädt, R. Başar, J. Park, Merve Dede, John G. Watts, N. Uprety, R. Shrestha, Francia Reyes-Silva, May Daher, Bin Liu, H. Rafei, P. Banerjee, Ping Li, S. Acharya, S. Islam, Huihui Fan, Donghai Xiong, Vernikka C. Woods, Paul Lin, Silvia Tiberti, A. N. Nunez Cortes, X. Jiang, Inci Biederstädt, Patrick Zhang, Ye Li, S. Rawal, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Elizabeth J. Shpall, N. Fowlkes, Ken Chen, Katayoun Rezvani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most prevalent malignancies worldwide. With the increasing incidence of CRC, there is an urgent need for effective strategies for early diagnosis and treatment. Circadian rhythm, a natural biological clock, regulates various physiological processes, and its disruption has been implicated in the onset and progression of cancer. However, the specific roles of circadian rhythm-related genes (CRDGs) in CRC remain unclear. Methods In this study, we analyzed the expression patterns of CRDGs in CRC using single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing data from the GSE178318 dataset. We constructed a CRC prognostic model based on CRD scores. Additionally, we explored the potential mechanisms of CRDGs in tumor progression through weighted gene co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA), and assessed their impact on the response to immune checkpoint inhibitors. Results The analysis revealed that CRDGs were significantly upregulated in liver metastasis samples compared to primary CRC samples and were closely associated with several metabolic and immune-related pathways. The prognostic model based on CRD scores indicated that higher CRD scores were associated with poorer outcomes in immunotherapy. These findings were further validated in multiple datasets, underscoring the potential of CRDGs as prognostic indicators in CRC. Conclusion This study systematically reveals, for the first time, the expression characteristics of CRDGs in CRC and their relationship with tumor progression and response to immunotherapy. CRDGs may serve as effective prognostic biomarkers and therapeutic targets, offering new strategies for the personalized treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b7699b45efdb0dd0f3e56012bf04dabfd7ec73" target='_blank'>
              Integration of scRNA-Seq and Bulk RNA-Seq Identifies Circadian Rhythm Disruption-Related Genes Associated with Prognosis and Drug Resistance in Colorectal Cancer Patients
              </a>
            </td>
          <td>
            Yong Tao, Jun Li, Jianhui Pan, Qing Wang, Ru Ke, D. Yuan, Hongbiao Wu, Yuepeng Cao, Lei Zhao
          </td>
          <td>2025-04-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844d851fd9820543dfb4e679c6a5eddb01a41ffd" target='_blank'>
              Abstract 3311: Uncovering methylation signatures and pathways associated with resistance to anti-EGFR therapy in colorectal cancer
              </a>
            </td>
          <td>
            Ana Regina de Abreu, Yann Heylen, J. Ibrahim, Marc Peeters, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a986ae0bdffed64d9a4eefeb701fd4d7309816c" target='_blank'>
              Abstract SY14-02: Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery
              </a>
            </td>
          <td>
            Francisca Vazquez
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c037285ee33cfd3dd5fa1bd1a95f79d26559f4b5" target='_blank'>
              Abstract 813: In vitro and in vivo modelling of drug tolerance and minimal residual disease to discover effective therapeutic combination strategies for EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Kristyna A. Gorospe, Ming Li, A. Arivajiagane, Nhu-An Pham, R. Navab, Kelsie L. Thu, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08272a17f03be6497869a3622bf7e66968e9d5bd" target='_blank'>
              Abstract 219: Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq
              </a>
            </td>
          <td>
            Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55dbe0088f239e1966f0ecbd4169616a69ee9add" target='_blank'>
              Abstract 5673: Identification of potential therapeutic targets by in vivo CRISPR screening on xenograft models
              </a>
            </td>
          <td>
            Ce Zhong, Lu Bai, Tongtong Wang, Yiying Shao, Lingyun Zhang, Lu Zhang, Jingqi Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma, the most common and aggressive primary malignant brain tumor, is characterized by a high rate of recurrence, disability, and lethality. Therefore, there is a pressing need to develop more effective prognostic biomarkers and treatment approaches for glioblastoma. Lactylation, an emerging form of protein post-translational modification, has been closely associated with lactate, a metabolite of glycolysis. Since the initial identification of lactylation sites in core histones in 2019, accumulating evidence has shown the critical role that lactylation plays in glioblastoma development, assessment of poor clinical prognosis, and immunosuppression, which provides a fresh angle for investigating the connection between metabolic reprogramming and epigenetic plasticity in glioblastoma cells. The objective of this paper is to present an overview of the metabolic and epigenetic roles of lactylation in the expanding field of glioblastoma research and explore the practical value of developing novel treatment plans combining targeted therapy and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4de8e5b16e7caa0a9241085fd7820d5813bb6d8" target='_blank'>
              Lactylation in Glioblastoma: A Novel Epigenetic Modifier Bridging Epigenetic Plasticity and Metabolic Reprogramming
              </a>
            </td>
          <td>
            Qingya Qiu, Hui Deng, Ping Song, Yushu Liu, Mengxian Zhang
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent research has identified a potential protective effect of higher numbers of circulating lymphocytes on colorectal cancer (CRC) development. However, the importance of different lymphocyte subtypes and activation states in CRC development and the biological pathways driving this relationship remain poorly understood and warrant further investigation. Specifically, CD4+ T cells -- a highly dynamic lymphocyte subtype -- undergo remodelling upon activation to induce the expression of genes critical for their effector function. Previous studies investigating their role in CRC risk have used bulk tissue, limiting our current understanding of the role of these cells to static, non-dynamic relationships only. Methods Here, we combined two genetic epidemiological methods -- Mendelian randomisation (MR) and genetic colocalisation -- to evaluate evidence for causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation stage. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across five CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). We repeated analyses stratified by CRC anatomical subsites and sex, and performed a sensitivity analysis to evaluate whether the observed effect estimates were likely to be CD4+ T cell-specific. Results We identified six genes with evidence (FDR-P<0.05 in MR analyses and H4>0.8 in genetic colocalisation analyses) for a causal role of CD4+ T cell expression in CRC development -- FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 . We observed differences in causal estimates of gene expression on CRC risk across different CD4+ T cell subtypes and activation timepoints, as well as CRC anatomical subsites and sex. However, our sensitivity analysis revealed that the genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. Conclusions Our study demonstrates the importance of capturing the dynamic nature of CD4+ T cells in understanding disease risk, and prioritises genes for further investigation in cancer prevention research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3854c3b38125ddae1dd0d632aa93ab78f073f33f" target='_blank'>
              Transcriptome-wide Mendelian randomisation exploring dynamic CD4+ T cell gene expression in colorectal cancer development
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A. Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, A. Lindblom, Shuji Ogino, Veronika Vymetálková, A. Wolk, Anna H. Wu, J. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E Thomas, Rish K Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-04-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da3a36ac2d7f53391b11d9e723da136e269551d" target='_blank'>
              Abstract 6805: DKC1 promotes colorectal cancer progression and therapy resistance by dysregulating sphingolipid biosynthesis
              </a>
            </td>
          <td>
            U. K. Khan, Ayush Goel, Shivansh Nigam, Nazia Chaudhary, Anjali Bajpai, A. Praveen, Akshay Shendre, Shannon Carskadon, Abhimanyu Kapoor, Anjali Tewari, Abhijit Chandra, S. Dalal, N. Palanisamy, Bushra Ateeq
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1834a1d0667a17db6a7427294528dfb9ae37c" target='_blank'>
              Abstract 4358: Single nucleotide polymorphisms located in PK/PD pathway genes of standard AML chemotherapy regimen predicts outcome in pediatric AML
              </a>
            </td>
          <td>
            Aneesha Nath, Richard J Marrero, Xueyuan Cao, J. Rubnitz, S. Pounds, J. Lamba
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ddc9edbf2bf1f968d70d06263a337e58c74158" target='_blank'>
              Abstract 4498: Drug screens identify new therapeutic targets that synergize with EZH1/2 inhibition in adult T-cell leukemia/lymphoma
              </a>
            </td>
          <td>
            Noor B Ghanem, C. Ip, Kit I Tong, Mehran Bakhtiari, Michael Y. He, A. Schimmer, R. Kridel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Long Intergenic Non-Protein Coding RNA 1123 (LINC01123), located on human chromosome 2q13, is a pivotal factor in tumorigenesis, exerting multifaceted oncogenic effects. Its expression strongly correlates with clinicopathological features, patient survival, and disease progression. In vivo and in vitro experiments further demonstrate that LINC01123 influences diverse cellular processes, including proliferation, apoptosis, viability, migration, invasion, stemness, and tumor growth. Notably, it also regulates metabolic reprogramming, immune escape, and tumor cell resistance to treatment. LINC01123 is regulated by multiple transcription factors and participates in gene regulation through protein interactions and competitive endogenous RNA (ceRNA) networks, thereby modulating cancer-promoting effects. This work systematically elucidates its primary functions and molecular mechanisms driving cancer initiation and progression, suggesting that LINC01123 might serve as a novel potential oncogenic driver and biomarker in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca43f615285dd30f7033e316e2f7208e0ef3058" target='_blank'>
              A novel cancer-associated lncRNA, LINC01123, participates in tumor progression, metabolism, immune escape, and resistance
              </a>
            </td>
          <td>
            Qiang Liu, He Huang, Shuwen Zhang, Fangteng Liu, Ting Lou
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8007e204fa82863ac047194325746e4be7c5ae" target='_blank'>
              Abstract 1441: Elevated HOXC gene expression in castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Isaacson B. Adelani, Akira Kurozumi, Tobiloba Owojuyigbe, Sanchit Sanyal, Su Mi Choi, Srinivasan Yegnasubramanian, Shawn E Lupold
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269139543b0ac5ad28ee94dd76e531f1c64685e0" target='_blank'>
              Abstract 5507: Mechanisms of resistance to RAS-GTP inhibition in pancreatic cancer
              </a>
            </td>
          <td>
            Joshua H. Choe, Julien Dilly, Eejung Kim, Lick P. Lai, Nicole Sindoni, Fan Yi Kong, Dennis Gong, Jaclyn Varga-Wiles, Brendan D Parent, Junning Wang, Kyle J Seamon, Jennifer A. Chan, Clint A. Stalnecker, C. Der, W. L. Hwang, B. Wolpin, D. Wildes, J. Mancias, Lewis C Cantley, A. Aguirre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631764abf56c15adfa48bcb7f3a761557f103393" target='_blank'>
              Abstract 444: ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents
              </a>
            </td>
          <td>
            Christopher P. Mill, W. Fiskus, Christine E. Birdwell, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, K. Sasaki, S. Loghavi, T. Kadia, Naval G Daver, C. Dinardo, Lu Wang, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8bc96152bbbb1bdeac5241e7bd6de510c434bdc" target='_blank'>
              Abstract 454: Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia
              </a>
            </td>
          <td>
            Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, E. Ceccacci, Giorgia Parodi, A. Bachi, S. Minucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c34b2c44db5e2413af90c06943f2d16b821f4c" target='_blank'>
              Abstract 6621: Multi-modal profiling reveals the role of somatic structural variations in renal cell carcinoma
              </a>
            </td>
          <td>
            Yuwei Zhang, Kapur B. Dhami, Yize Li, Cody Weimholt, Wagma Caravan, John M Herndon, R. Jayasinghe, Song Cao, Melissa A. Reimers, Russell Pachynski, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5520257da55eb20975e2b76db4a10f023d4be1" target='_blank'>
              Abstract 5543: High resolution lineage tracing for the study of cancer drug persistence
              </a>
            </td>
          <td>
            Binyanim Zhitomirsky, Jideofor Ezike, B. Danysh, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8e473fd0d41652458c66d5982b1c03c7dc5d7e" target='_blank'>
              Abstract 3898: Integrating structural variants and single nucleotide variants to uncover evolutionary trajectories in melanoma
              </a>
            </td>
          <td>
            Xiwen Cui, A. Keskus, S. Malikić, Yuelin Liu, Anton Goretsky, Chi-Ping Day, F. Mehrabadi, M. Kolmogorov, Glenn Merlino, S. C. Sahinalp
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Homologous recombination (HR) and mismatch repair (MMR) act as guardians of the human genome, and defects in HR or MMR are causative in at least a quarter of all malignant tumors. Although these DNA repair-deficient tumors are eligible for effective targeted therapies, fully reliable diagnostic strategies based on functional assay have yet to be established, potentially limiting safe and proper application of the molecular targeted drugs. Here we show that transient transfection of artificial DNA substrates enables ultrarapid detection of HR and MMR. This finding led us to develop a diagnostic strategy that can determine the cellular HR/MMR status within one day without the need for control cells or tissues. Notably, the accuracy of this method allowed the discovery of a pathogenic RAD51D mutation, which was missed by existing companion diagnostic tests. Our methods, termed HR eye and MMR eye, are applicable to frozen tumor tissues and roughly predict the response to therapy. Overall, the findings presented here could pave the way for accurately assessing malignant tumors with functional defects in HR or MMR, a step forward in accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9695c321b4adb2f24badedc1d3f1003b2ca44567" target='_blank'>
              HR eye & MMR eye: one-day assessment of DNA repair-defective tumors eligible for targeted therapy
              </a>
            </td>
          <td>
            Shinta Saito, Shingo Kato, Usaki Arai, Atsuki En, Jun Tsunezumi, Taichi Mizushima, K. Tateishi, Noritaka Adachi
          </td>
          <td>2025-05-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575ec445e704ca863d5507eaa78e755caa175d07" target='_blank'>
              The Use of Mutational Signatures to Decipher the Inter-Relationship of Reactive Oxygen Species with Homologous Recombination and Non-Homologous End-Joining Deficiencies as Well as Their Effects on APOBEC Mutagenesis in Breast Cancer
              </a>
            </td>
          <td>
            Amir Farmanbar, Robert Kneller, Sanaz Firouzi
          </td>
          <td>2025-05-12</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b02574f80309e49880132063bfa30fa50f5b" target='_blank'>
              Abstract 6343: Frameshift neoantigen-based vaccine testing in an intra-cecal implantation mouse model of Lynch syndrome
              </a>
            </td>
          <td>
            Yurong Song, Brandon Somerville, Shaneen S Baxter, Jason D. Marshall, Lei Wei, Chelsea Sanders, Qiang Hu, Song Liu, Alan Hutson, Baktiar Karim, Simone Difilippantonio, Ligia A. Pinto, Shizuko Sei, Robert H. Shoemaker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31571cc4d771e5d8cbcfb30a223ba711a12ce0b" target='_blank'>
              Resistance to the KRASG12D Inhibitor MRTX1133 is Associated with Increased Sensitivity to BET Inhibition
              </a>
            </td>
          <td>
            Daniel R. Principe, Jeffrey H Becker, Anastasia E. Metropulos, Alejandra M Marinelarena, Thao D Pham, A. Aissa, H. Munshi
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0460f2bfed152b0841d1d7ccdc52c08315edba3" target='_blank'>
              In vivo CRISPR screening reveals epigenetic regulators of hepatobiliary plasticity.
              </a>
            </td>
          <td>
            Jonathan H. Sussman, Hector Cure, Salina Yuan, Kenji Ito, Irfan A Asangani, Benjamin A Garcia, Ben Z. Stanger, Takeshi Katsuda
          </td>
          <td>2025-04-01</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d3beaaac78a42fdc6dae5c6e9c46a2476c5705" target='_blank'>
              Abstract 6709: NF1 mutations in lung adenocarcinoma preclinical models and potential targeted therapies: The crucial role of the RAS-MAPK pathway
              </a>
            </td>
          <td>
            J. Giraud, Doriane Gorret, Manuela Ye, Dominique Lallemand, William C. Reinhold, Didier Decaudin, Ingrid Laurendeau, E. Pasmant, Camille Tlemsani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aff6e4c1130ad77a048fffd67082d1b6b987410" target='_blank'>
              Abstract 1480: Unlocking DNA repair mechanisms with dynamic single-molecule analysis
              </a>
            </td>
          <td>
            Paul Pease, Jack O'Sullivan, Trey Simpson, Josephine Yeh, A. Candelli
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4631b336ea353f2fab308e1332f9b36313977696" target='_blank'>
              Abstract 4200: Elevated dNTP pools impair homologous recombination-mediated repair, promoting apoptosis in therapy-resistant glioblastoma
              </a>
            </td>
          <td>
            Dominique Monroe, Mercy Kehinde-Ige, Apeksha Anand, Arilyn Williams, E. Usoro, Matthew Kededa, Aman Kalsi, Christian Fahmy, Ramsha Khanam, Huidong Shi, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9488336a355d0c8f13d6aba1e3fdc05af9aece24" target='_blank'>
              Abstract 910: Chromosome 3q amplification in lung squamous cell carcinoma is associated to iron metabolism and immune cell exclusion
              </a>
            </td>
          <td>
            María Gutiérrez Pérez, J. R. Paradas, David Gómez Sánchez, R. G. Luján, Jon Zugazagoitia, N. Carrizo, Roxolyana Khoroz, Inés Díaz Cano, Laura García Redondo, Ana B Enguitta, Esther Conde, Eva M Garrido Martín, Álvaro C. Ucero Herrería, Luis Paz-Ares
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f09c779b15647c7ef62bbebbbbd96b9880a1d08" target='_blank'>
              PROX1: a key regulator of hepatocyte identity and tumorigenesis.
              </a>
            </td>
          <td>
            T. K. Lee, Stephianie Ma
          </td>
          <td>2025-03-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01701fc1080f2a22b516da6a31e565685ef5a558" target='_blank'>
              μSeq: Universal mutation rate quantification via deep sequencing of a single clonal expansion
              </a>
            </td>
          <td>
            Simone Pompei, Alberto Geroldi, Pietro Rivetti, Elena Grassi, V. Vurchio, Giorgio Tallarico, Giorgio Corti, Lorenzo Tattini, Gianni Liti, Andrea Bertotti, M. Lagomarsino
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The diploid genome of the fungal pathogen Candida albicans is highly heterozygous, with most allele pairs diverging at either the coding or regulatory level. When faced with selection pressure like antifungal exposure, this hidden genetic diversity can provide a reservoir of adaptive mutations through loss of heterozygosity (LOH) events. Validating the potential phenotypic impact of LOH events observed in clinical or experimentally evolved strains can be difficult due to the challenge of precisely targeting one allele over the other. Here, we show that a CRISPR-Cas9 system can be used to overcome this challenge. By designing allele-specific guide RNA sequences, we can induce targeted, directed LOH events, which we validate by whole-genome long-read sequencing. Using this approach, we efficiently recapitulate a recently described LOH event that increases resistance to the antifungal fluconazole. Additionally, we find that the recombination tracts of these induced LOH events have similar lengths to those observed naturally. To facilitate future use of this method, we provide a database of allele-specific sgRNA sequences for Cas9 that provide near genome-wide coverage of heterozygous sites through either direct or indirect targeting. This approach will be useful in probing the adaptive role of LOH events in this important human pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161077a2ca34ace846023a0e504cab47b241a6a0" target='_blank'>
              Targeted loss of heterozygosity in Candida albicans using CRISPR-Cas9
              </a>
            </td>
          <td>
            Philippe C. Després, Nicholas C. Gervais, Meea Fogal, Ruby KJ Rogers, Christina A. Cuomo, Rebecca S. Shapiro
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a9bc7f4f8a4cf97f32bbe5a9d88c7eccc61afe" target='_blank'>
              Abstract 2923: Targeting DNA repair mechanisms to overcome CIC::DUX4 undifferentiated sarcomas
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers that arise from germline mutations of breast cancer associated gene 1 (BRCA1), which is a crucial player in homologous recombination (HR) DNA repair, are vulnerable to DNA-damaging agents such as platinum and PARP inhibitors (PARPis). Increasing evidence suggests that BRCA1 is an essential driver of all phases of the cell cycle, thereby maintaining orderly steps during cell cycle progression. Specifically, loss of BRCA1 activity causes the S-phase, G2/M, spindle checkpoints, and centrosome duplication to be dysregulated, thereby blocking cell proliferation and inducing apoptosis. In vertebrates, loss of HR genes such as BRCA1 and/or BRCA2 is lethal, since HR is a prerequisite for genome integrity. Thus, cancer cells utilize alternative DNA repair pathways such as non-homologous end joining (NHEJ) to cope with the loss of BRCA1 function. In this review, we attempt to update and discuss how these novel components are crucial for regulating DNA damage repair (DDR) in BRCA1-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca96851a56316ea96ea1fc73475ff478d721c03" target='_blank'>
              Overview of Roles of Novel Components in the Regulation of DNA Damage Repair in BRCA1-Deficient Cancers: An Update
              </a>
            </td>
          <td>
            Nhat Nguyen, Dominic Arris, M. T. Tran
          </td>
          <td>2025-04-01</td>
          <td>DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29e60f63dffafbff9d001a2d3645ac87336a0cf" target='_blank'>
              G6PC3 promotes genome maintenance and is a candidate mammary tumor suppressor.
              </a>
            </td>
          <td>
            Xin Li, Maria Rossing, Ana Moisés da Silva, Muthiah Bose, Thorkell Gudjónsson, Jan Benada, J. Thatte, Jens Vilstrup Johansen, J. Börcsök, H. van der Gulden, Ji-Ying Song, Renée Menezes, Asma Tajik, Lucía Sena, Zoltan Szallasi, Morten Frödin, J. Jonkers, F. C. Nielsen, C. S. Sørensen
          </td>
          <td>2025-04-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Chemotherapeutic drugs, like cisplatin, function by damaging genomic DNA, thus inducing cell apoptosis. Cancer cells can enhance their DNA repair capacity, leading to chemotherapeutic resistance. Nucleotide excision repair (NER) involves repairing DNA adducts and crosslinks caused by chemotherapeutic agents. Transforming growth factor-beta (TGF-β) pathway contributes to carcinogenesis, DNA repair alteration, and chemoresistance. However, the connection between TGF-β pathway, NER function alteration, and resistance to cisplatin therapy remains elusive. Therefore, the objective of current study was to fill this gap by assessing the impact of TGF-β inhibition and activation on cisplatin-induced antiproliferation, apoptosis, and DNA damage using the MTT assay, flow cytometry analysis, and COMET assay, respectively. Four NER genes, XPA, XPB, XPC, and XPF, were measured using Real-time Polymerase Chain Reaction (qPCR). MDA-MB-231 cell line was utilized as a model of breast cancer. Blockade of the TGF-β pathway strengthened cisplatin cytotoxicity, whereas induction of the TGF-β pathway suppressed cisplatin cytotoxicity. In cisplatin-treated breast cancer cells, DNA damage significantly increased upon the TGF-β pathway inhibition. Conversely, cisplatin-induced DNA damage decreased significantly upon TGF-β pathway stimulation. Finally, cisplatin caused an overexpression of the four NER genes which was curtailed and augmented by TGF-β inhibition and stimulation, respectively. Overall, this study presented evidence of the impact exerted by TGF-β pathway on NER and cisplatin sensitivity of breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da79b3598334d2968d0a956c3775a3a49b9ba178" target='_blank'>
              Modulation of Chemotherapy Sensitivity of Breast Cancer Cells through Transforming Growth Factor-beta Pathway-mediated Alterations in DNA Damage Response
              </a>
            </td>
          <td>
            Abdullah S. Alhamed, Mohammad S El-Wetidy, Mervat M. Abdelwahed, Sabry M. Attia, Abdulrahman M. Alabkka, Saleh A. Alaraj, Khalid Alhazzani, Ahmed Z Alanazi, Faris Almutairi, Ibrahem A. Alotibi, Mohammed Alqinyah
          </td>
          <td>2025-03-31</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40e791df38798fa882f2c7a57b26f0370087bba" target='_blank'>
              Abstract LB290: Identifying novel mechanisms of resistance to KRAS-inhibitors in NSCLC
              </a>
            </td>
          <td>
            S. Kundu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ab372dd4cae4e9e87fbdd2a61f62f73ea8595e" target='_blank'>
              Abstract 2973: Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies
              </a>
            </td>
          <td>
            Andrzej Mazan, M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Izabella Wieckowska, M. Chmiel, A. Kurzejamski, B. Lipert, Katarzyna Sarad, M. Mikula, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60116964600321d1649e9fb0b1956be9bba4b01" target='_blank'>
              Metabolic buffering suppresses phenotype switching in cancer
              </a>
            </td>
          <td>
            Ana Ramírez-Sánchez, Miguel Jociles-Ortega, J. M. García-Martínez, Irene Torrens-Martínez, J. Martínez-Useros, M. J. Fernández-Aceñero, Teresa Olmos-De Blas, Pakavarin Louphrasitthiphol, Colin R. Goding, C. García-Jiménez, Ana Chocarro-Calvo
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0e4ce978bcbd93656c4cbcf8bd876d6363d193" target='_blank'>
              Analysis of the role of cellular autosomal genetic banding position subtyping on clinical prognosis and immune infiltration in colorectal cancer
              </a>
            </td>
          <td>
            Jiansong Wang, Xin Mao
          </td>
          <td>2025-03-27</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0960349fa1d9c3399276e6b3bfa98752934b018" target='_blank'>
              Abstract 6427: Nous-209 off-the-shelf neoantigen immunotherapy induces robust neoantigen T cell response with the potential to intercept cancer in Lynch syndrome carriers
              </a>
            </td>
          <td>
            Jason Willis, A. D’Alise, Michael J. Hall, Marcia Cruz-Correa, Gregory E. Idos, S. Thirumurthi, Veroushka Ballester, Guido Leoni, I. Garzia, Laura Antonucci, Lorenzo De Marco, E. Micarelli, Sven Gogov, Wenli Dong, J. J. Lee, L. Vornik, Araceli Garcia-Gonzalez, Laura Reyes-Uribe, Ellen Richmond, Asad Umar, Powel H Brown, Luz María Rodríguez, Elisa Scarselli, Eduardo Vilar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2db16c30d79726e181227a7646a42de7a4c218" target='_blank'>
              Abstract 829: ITGAV is a targetable vulnerability in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Gilbert K. Salloum
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/642babaec15e8d22494c611094dc188788c6a876" target='_blank'>
              Abstract 3807: Validation of a functional genomics screen based on lentiviral integration site analysis reveals single-gene disruptions enhancing CAR19 T-cell effectiveness in preclinical models
              </a>
            </td>
          <td>
            Beatriz Coutinho de Oliveira, Saaurav Bari, Jessica Kerr, Victoria Putzbach, Y. Phoon, J. Fraietta, Wei-Ting Hwang, Frederic D. Bushman, J. Melenhorst
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe96f8609d981f1d51f682dd9dde5af13de525a" target='_blank'>
              Abstract 5533: Drug tolerance mechanisms in patient-derived xenograft models with epidermal growth factor receptor mutated tumors treated with tyrosine kinase inhibitors
              </a>
            </td>
          <td>
            A. Arivajiagane, Q. Li, Nhu-An Pham, Ming Li, Katrina Hueniken, Takamasa Koga, N. Radulovich, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f9135b22221853a957c30c494e32f6d68e606" target='_blank'>
              Abstract LB016: Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer
              </a>
            </td>
          <td>
            Debdatta Halder, Utsav Sen, V. Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, A. Vanderwalde, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8992e140899a62dade76814a52a30a531da19" target='_blank'>
              Abstract 5026: Discovery of oncogenic mediator genes in rectal cancer chemotherapy response using gene expression data from matched tumor and patient-derived organoids
              </a>
            </td>
          <td>
            Hanchen Huang, W. Zambare, Chao Wu, A. Colaprico, J. Alvarez, Min Jung Kim, A. Bercz, P. Bleu, Lily Wang, P. Paty, P. Romesser, J. J. Smith, X. Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Telomere biology disorders (TBDs) are genetic diseases caused by defective telomere maintenance. TBD patients often develop bone marrow failure and have an increased risk of myeloid neoplasms. To better understand the factors underlying hematopoietic outcomes in TBD, we comprehensively evaluated acquired genetic alterations in hematopoietic cells from 166 pediatric and adult TBD patients. Of these patients, 47.6% (28.8% of children, 56.1% of adults) had clonal hematopoiesis. Recurrent somatic alterations involved telomere maintenance genes (7.6%), spliceosome genes (10.4%, mainly U2AF1 p.S34), and chromosomal alterations (20.2%), including 1q gain (5.9%). Somatic variants affecting the DNA damage response (DDR) were identified in 21.5% of patients, including 20 presumed loss-of-function variants in ataxia-telangiectasia mutated (ATM). Using multimodal approaches, including single-cell sequencing, assays of ATM activation, telomere dysfunction-induced foci analysis, and cell-growth assays, we demonstrate telomere dysfunction–induced activation of the ATM-dependent DDR pathway with increased senescence and apoptosis in TBD patient cells. Pharmacologic ATM inhibition, modeling the effects of somatic ATM variants, selectively improved TBD cell fitness by allowing cells to bypass DDR-mediated senescence without detectably inducing chromosomal instability. Our results indicate that ATM-dependent DDR induced by telomere dysfunction is a key contributor to TBD pathogenesis and suggest dampening hyperactive ATM-dependent DDR as a potential therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7e03f128958621aaeeb6a79d8a716a78521617" target='_blank'>
              ATM-dependent DNA damage response constrains cell growth and drives clonal hematopoiesis in telomere biology disorders
              </a>
            </td>
          <td>
            Christopher M Sande, Stone Chen, Dana V Mitchell, Ping Lin, Diana M Abraham, Jessie M. Cheng, Talia Gebhard, Rujul Deolikar, Colby Freeman, Mary Zhou, Sushant Kumar, Michael Bowman, Robert L Bowman, Shannon Zheng, Bolormaa Munkhbileg, Qijun Chen, Natasha L. Stanley, Kathy Guo, Ajibike Lapite, Ryan Hausler, Deanne M. Taylor, James Corines, Jennifer J D Morrissette, David B Lieberman, Guang Yang, O. Shestova, Saar Gill, Jiayin Zheng, Kelcy Smith-Simmer, L. Banaszak, Kyle Shoger, Erica F Reinig, Madilynn Peterson, Peter Nicholas, A. Walne, I. Dokal, Justin P. Rosenheck, K. Oetjen, Daniel C. Link, A. Gelman, Christopher R Reilly, Ritika Dutta, R. C. Lindsley, K. Brundige, Suneet Agarwal, Alison A. Bertuch, Jane E. Churpek, L. Tague, F. B. Johnson, Timothy Olson, D. Babushok
          </td>
          <td>2025-04-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence, a stable state of cell cycle arrest induced by various stressors or genomic damage, is recognized as a hallmark of cancer. It exerts a context-dependent dual role in cancer initiation and progression, functioning as a tumor suppressor and promoter. The complexity of senescence in cancer arises from its mechanistic diversity, potential reversibility, and heterogeneity. A key mediator of these effects is the senescence-associated secretory phenotype (SASP), a repertoire of bioactive molecules that influence tumor microenvironment (TME) remodeling, modulate cancer cell behavior, and contribute to therapeutic resistance. Given its intricate role in cancer biology, senescence presents both challenges and opportunities for therapeutic intervention. Strategies targeting senescence pathways, including senescence-inducing therapies and senolytic approaches, offer promising avenues for cancer treatment. This review provides a comprehensive analysis of the regulatory mechanisms governing cellular senescence in tumors. We also discuss emerging strategies to modulate senescence, highlighting novel therapeutic opportunities. A deeper understanding of these processes is essential for developing precision therapies and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286d205296f9a0a8d06235700589eef3dccbbc34" target='_blank'>
              Regulation of cellular senescence in tumor progression and therapeutic targeting: mechanisms and pathways
              </a>
            </td>
          <td>
            Bowei Liu, Zhigang Peng, Hao Zhang, Nan Zhang, Zao-bin Liu, Zhiwei Xia, Shaorong Huang, Peng Luo, Quan Cheng
          </td>
          <td>2025-04-02</td>
          <td>Molecular Cancer</td>
          <td>3</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b855a13f4583cca6b74a017150a15f6334f81973" target='_blank'>
              Abstract 2773: The Gundersen Precision Oncology Cohort: a biospecimen resource for mutation characterization and target discovery
              </a>
            </td>
          <td>
            Peter Feiszt, Paraic A. Kenny
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The molecular control of epigenetic information relies on hundreds of proteins of diverse function, which cooperate in defining chromatin structure and DNA methylation landscapes. While many individual pathways have been characterized, how different classes of epigenetic regulators interact to build a resilient epigenetic regulatory network (ERN) remains poorly understood. Here, we show that most individual regulators are dispensable for somatic cell fitness, and that robustness emerges from multiple layers of functional cooperation and degeneracy among network components. By disrupting 200 epigenetic regulator genes, individually or in combination, we generated network-wide maps of functional interactions for representative regulators. We found that paralogues represent only a first layer of functional compensation within the ERN, with intra- or inter-class interactions buffering the effects of perturbation in a gene-specific manner: while CREBBP cooperates with multiple acetyltransferases to form a subnetwork that ensures robust chromatin acetylation, ARID1A interacts with regulators from across all functional classes. When combined with oncogene activation, the accumulated epigenetic disorder exposes a synthetic fragility and broadly sensitizes ARID1A-deficient cells to further perturbation. Our findings reveal homeostatic mechanisms through which the ERN sustains somatic cell fitness and uncover how the network remodels as the epigenome is progressively deregulated in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74c759f71b52bc8e0f4b18c555786227753fc4c1" target='_blank'>
              Systematic genetic perturbation reveals principles underpinning robustness of the epigenetic regulatory network
              </a>
            </td>
          <td>
            Thomas Stuart Wilson, R. Noberini, Eirini Moysidou, Ifeyinwa Ojukwu, Marta Milan, Ming Jiang, Gavin Kelly, Michael Howell, Tiziana Bonaldi, Paola Scaffidi
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5750dddf572726400a2fad0203c2ec7fc9717955" target='_blank'>
              Abstract 478: Engineering viruses for the next generation targeted cancer therapy
              </a>
            </td>
          <td>
            Ikeda Trashi, Jeremiah Gassensmith
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c906c5a3862aab5cd84c1716303e1ffc00c0344c" target='_blank'>
              NSD3: A Promising Target for Cancer Therapy
              </a>
            </td>
          <td>
            Ting Huang, Bowen Zhang, Yifan Yang, Qiong Lin, Genbao Shao
          </td>
          <td>2025-03-26</td>
          <td>Cell Biochemistry and Function</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic‐type malignancies (SMs) arising in germ cell tumors (GCTs) are aggressive neoplasms resistant to systemic treatment. Most are diagnosed in metastatic sites after chemotherapy; however, they have also been well‐documented in primary testicular GCTs. Historically, SMs were thought to originate in components of teratoma that acquire molecular alterations equivalent to those that characterize their true somatic counterparts. However, recent studies have shown that SMs typically lack the hallmark molecular alterations seen in similar somatic tumors. Additionally, clinicopathologic and molecular data suggest that a subset may derive from yolk sac tumor (YST) rather than teratoma. In this study, we evaluated the relationship between conventional histological types of GCTs and SMs by comparing expression of microRNA (miR)‐371–373 and genomic methylation profiles. A total of 96 samples (including multiple paired conventional GCT–SM samples from individual tumors) were assessed for miR‐371–373 expression by RT‐qPCR and genomic DNA methylation using a clinically validated assay. Expression of miR‐371–373 was higher in conventional GCTs than in SMs (considered as a single category encompassing all histological subtypes). However, miR‐371–373 expression was heterogeneous among SMs, with significantly higher levels in sarcomatoid YST (SYST) and glandular neoplasms than in other SMs. Genomic DNA methylation analysis showed that SMs (considered as a single category) did not form a distinct cluster. Instead, they grouped into multiple clusters that did not show perfect correspondence with histology and often included conventional GCTs. Genome‐wide methylation assessment showed a higher abundance of hypermethylated regions in SMs than in conventional GCTs. Analysis of paired conventional GCT and ‘somatic‐type’ components that did not meet size criteria for SMs dissected from individual tumors demonstrated separation according to histology, suggesting that epigenetic processes play a role in the transition from conventional GCT to ‘somatic‐type’ phenotypes. Gene‐level and pathway‐level analyses identified MAPK/RAS signaling, mitosis/proliferation, differentiation towards neural tissue/neuroectoderm, epithelial‐to‐mesenchymal transition, and DNA repair as key differentially regulated processes in components with somatic‐type histology, suggesting mechanisms of progression from conventional to ‘somatic’ phenotypes in GCT. These results support the hypothesis that a subset of SMs derive from YST and suggest that some subtypes (such as SYST) may represent ‘intermediate’ phenotypes. Additionally, analysis of differentially methylated promoter regions in SM identified genes and biologic processess that may underlie 'somatic tranformation' in GCTs. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bd3895700f513d14429236b145a33de18651c" target='_blank'>
              MicroRNA‐371–373 cluster and methylome analysis suggests that a subset of ‘somatic‐type’ malignancies arising in germ cell tumors may originate in yolk sac tumor components
              </a>
            </td>
          <td>
            João Lobo, Nuno Tiago Tavares, Diana Fonseca, Carmen Jerónimo, Rui Henrique, Nicolas Wyvekens, Yiying Yang, Matija Snuderl, Fiona Maclean, Jennifer B. Gordetsky, C. Fletcher, Michelle S Hirsch, Jason L. Hornick, M. T. Idrees, Katrina Collins, L. Warmke, Thomas Ulbright, Andres M Acosta
          </td>
          <td>2025-03-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15a5b2af9bd939cc77bcb6b3ee460a1e0556b12" target='_blank'>
              Decoding Potassium Homeostasis in Cancer Metastasis and Drug Resistance: Insights from a Highly Selective DNAzyme-Based Intracellular K+ Sensor.
              </a>
            </td>
          <td>
            Zhenglin Yang, Xiangli Shao, Yuting Wu, Aritra Roy, Elijah Garcia, Annie Farrell, Shreestika Pradhan, Weijie Guo, Heather Gan, Zeynep Korkmaz, Emily Adams, Yi Lu
          </td>
          <td>2025-05-14</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca665dc53860039a134586f0a4361eb2c643769" target='_blank'>
              Abstract 5645: Targeting transcription-replication conflict for selective chemotherapy in glioblastoma
              </a>
            </td>
          <td>
            Long Gu, Robert J. Hickey, L. Malkas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2116c40f11199361c9cb93d0a62e0a019f2b44da" target='_blank'>
              Abstract 4681: Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Benjamin Hauk, B. McBean, A. Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, L. Lerner, C. Ward, Kassidy M. Jungles, Daniel Spratt, Lori J. Pierce, Mats Ljungman, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4afa94be560741f52b5090eabfe589892d769b54" target='_blank'>
              Modulating populational variance of methyl-guanine methyl transferase expression through miR-181d degradation: a novel mechanism of temozolomide resistance
              </a>
            </td>
          <td>
            Gatikrushna Singh, Shilpi Singh, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Johnny Akers, Thien Nguyen, Eric Wong, Margot Martinez Moreno, E. Kokkoli, Shobha Vasudevan, Sean E Lawler, W. El-Deiry, Z. Gokaslan, Clark C. Chen
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The adaptive response of cancer cells to hypoxia, a key microenvironmental factor in solid tumors, is intricately orchestrated by Hypoxia-inducible factor 1 (HIF1). Evidence is accumulating on oxygen dynamics in tumor cores, showing that hypoxia is frequently unstable, or cycling, associated with specific phenotypic outcomes for the cancer. Transcriptomic analysis show that most gene expression changes in cycling hypoxia lie in between the change caused by stable hypoxia, suggesting multi-cycle averaging of dosage in the oxygen tension, and likely HIF-1 induced transcription. However, small subset of genes show an oscillation/cycling hypoxia specific response, suggesting that the transcriptional machinery of these genes may interpret cycling HIF-1 activity differently from stably high HIF-1 activity. Here, we model a gene regulatory circuit, the incoherent feed-forward loops (IFFLs) to explore parameter regimes where oscillatory specific transcription is plausible. In these IFFL models, HIF-1 regulates gene transcription of a target gene directly, as well indirectly via another transcription factor with an opposite effect on gene transcription. We have identified several plausible parameter regimes where oscillation specific gene expression can occur, with potentially significant effect on cancer growth and progression. Finally, we experimentally confirmed that HIF-1 can form IFFLs with two key transcription factors p53, and Notch1, resulting in cycling hypoxia specific gene expression linked to breast cancer progression and poor prognosis. Our models mechanistically reveal how temporal fluctuations in the tumor microenvironment may directly inform downstream transcription, identify hitherto unknown HIF-1 driven mechanism of cancer progression contributing to emergent tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65afa29d37135628627c1ec1ee0a105f3db46eb" target='_blank'>
              Cancer Cell Adaptation to Cycling Hypoxia: Integrating Mathematical Modeling with Experimental Validation of HIF-1 Dynamics
              </a>
            </td>
          <td>
            Xihua Qiu, Yamin Liu, Paola Vera-Nicola, E. Agmon, Kshitiz, Yasir Suhail
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801d1a28b5036fffab863a5f513c202209cf4c12" target='_blank'>
              Abstract 4722: Crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors
              </a>
            </td>
          <td>
            Yan Wang, Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, Nan Sun, Jie He
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a9e6faf45fc758543cd3dfb6d7c7bf6cfe5d5f" target='_blank'>
              Abstract 6892: Ribonucleotide reductase is a potential therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Federica Invrea, Consalvo Petti, C. Isella, E. Médico
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Human health depends on perplexing defensive cellular responses against microbial pathogens like Viruses. Despite the major effort undertaken, the (epi)genomic mechanisms that human cells utilize to tailor defensive gene expression programs against microbial attacks have remained inadequately understood, mainly due to a significant lack of recording of the in vivo functional cis-regulatory modules (CRMs) of the human genome. Here, we introduce the virus-responsive fate of the human (epi)genome as characterized in naïve and infected cells by functional genomics, computational biology, DNA evolution, and DNA Grammar and Syntax investigations. We discovered that multitudes of novel functional virus-responsive CRMs (vrCRMs) compose typical enhancers (tEs), super-enhancers (SEs), repetitive-DNA enhancers (rDEs), and stand-alone functional genomic stretches that grant human cells regulatory underpinnings for layering basal immunity and eliminating illogical/harmful defensive responses under homeostasis, yet stimulating virus-responsive genes and transposable elements (TEs) upon infection. Moreover, extensive epigenomic reprogramming of previously unknown SE landscapes marks the transition from naïve to antiviral human cell states and involves the functions of the antimicrobial transcription factors (TFs), including interferon response factor 3 (IRF3) and nuclear factor-κB (NF-κB), as well as coactivators and transcriptional apparatus, along with intensive modifications/alterations in histone marks and chromatin accessibility. Considering the polyphyletic evolutionary fingerprints of the composite DNA sequences of the vrCRMs assessed by TFs-STARR-seq, ranging from the animal to microbial kingdoms, the conserved features of antimicrobial TFs and chromatin complexes, and their pluripotent stimulus-induced activation, these findings shed light on how mammalian (epi)genomes evolved their functions to interpret the exogenous stress inflicted and program defensive transcriptional responses against microbial agents. Crucially, many known human short variants, e.g. single-nucleotide polymorphisms (SNPs), insertions, deletions etc., and quantitative trait loci (QTLs) linked to autoimmune diseases, such as multiple sclerosis (MS), systemic lupus erythematosus (SLE), Crohn’s disease (CD) etc., were mapped within or vastly proximal (±2.5 kb) to the novel in vivo functional SEs and vrCRMs discovered, thus underscoring the impact of their (mal)functions on human physiology and disease development. Hence, we delved into the virus-responsive fate of the human (epi)genome and illuminated its architecture, function, evolutionary origins, and its significance for cellular homeostasis. These results allow us to chart the “Human hyper-Atlas of virus-infection”, an integrated “molecular in silico” encyclopedia situated in the UCSC Genome Browser that benefits our mechanistic understanding of human infectious/(auto)immune diseases development and can facilitate the generation of in vivo preclinical animal models, drug design, and evolution of therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a50dabd4132cc74ddb77db8bd576628b644f734" target='_blank'>
              An evolutionarily conserved constellation of functional cis-elements programs the virus-responsive fate of the human (epi)genome
              </a>
            </td>
          <td>
            Marianna A Koutsi, Marialena Pouliou, Dimitris Chatzopoulos, Lydia Champezou, Konstantinos Zagkas, Marili Vasilogianni, Alexandra G Kouroukli, M. Agelopoulos
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are short non-coding RNAs, which perform a key role in cellular differentiation and development. Most human diseases, particularly cancer, are linked to miRNA functional dysregulation implicated in the expression of tumor-suppressive or oncogenic targets. Cancer hallmarks such as continued proliferative signaling, dodging growth suppressors, invasion and metastasis, triggering angiogenesis, and avoiding cell death have all been demonstrated to be affected by dysregulated miRNAs. Thus, for the treatment of different cancer types, the detection and quantification of this type of RNA is significant. The classical and current methods of RNA detection, including northern blotting, reverse transcription-quantitative PCR, rolling circle amplification and next-generation sequencing, may be effective but differ in efficiency and accuracy. Furthermore, these approaches are expensive, and require special instrumentation and expertise. Thus, researchers are constantly looking for more innovative approaches for miRNA detection, which can be advantageous in all aspects. In this regard, an RNA manipulation tool known as the CRISPR and CRISPR-associated sequence 13 (CRISPR/Cas13) system has been found to be more advantageous in miRNA detection. The Cas13-based miRNA detection approach is cost effective and requires no special instrumentation or expertise. However, more research and validation are required to confirm the growing body of CRISPR/Cas13-based research that has identified miRNAs as possible cancer biomarkers for diagnosis and prognosis, and as targets for treatment. In the present review, current updates regarding miRNA biogenesis, structural and functional aspects, and miRNA dysregulation during cancer are described. In addition, novel approaches using the CRISPR/Cas13 system as a next-generation tool for miRNA detection are discussed. Furthermore, challenges and prospects of CRISPR/Cas13-based miRNA detection approaches are described.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7cad934b860ca66c2574dc37104aa0e67005c2" target='_blank'>
              Current updates regarding biogenesis, functions and dysregulation of microRNAs in cancer: Innovative approaches for detection using CRISPR/Cas13-based platforms (Review)
              </a>
            </td>
          <td>
            Abdulaziz A. Aloliqi, Abdullah M. Alnuqaydan, Aqel Albutti, Basmah F. Alharbi, A. Rahmani, Amjad Ali Khan
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c350dc98b4d3786f2ffe26744dbe7c5702a670" target='_blank'>
              Abstract 5246: Chromosomal instability promotes colorectal cancer progression through manipulation of gamma delta T cells
              </a>
            </td>
          <td>
            Paulino Tallón de Lara, Aviad Ben-Shmuel, Wen-Hao Hsu, Chaohao Li, Kyle Labella, Chi Wut Wong, Yonghong Liu, Shan Jiang, Xiaoying Shang, S. Kopetz, Ronald A. DePinho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9365edb5d9a7621f7c2fa255adc2d1f2aebf2bfd" target='_blank'>
              Abstract 5334: Proliferative signaling through the ras-erk pathway can be inhibited by the overexpression of dual-specific phosphatase genes in collaboration with other negative regulators of cell cycle progression in the acute myeloid cell line HL-60
              </a>
            </td>
          <td>
            Michael Roberts, Amelia Harper, Kishan Mangru, Katy Meta, Sophia Kovatsis, J. Forrester, Grace Crossland, Cuong Nguyen, S. Bahadur
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The dual genetic control of mitochondrial respiratory function, combined with the high mutation rate of the mitochondrial genome (mtDNA), makes mitochondrial diseases among the most frequent genetic diseases in humans (1 in 5,000 in adults). With no effective treatments available, gene therapy approaches have been proposed. Notably, several studies have demonstrated the potential for nuclear expression of a healthy copy of a dysfunctional mitochondrial gene, referred to as allotopic expression, to help recover respiratory function. However, allotopic expression conditions require significant optimization. We harnessed engineering biology tools to improve the allotopic expression of the COX2-W56R gene in the budding yeast Saccharomyces cerevisiae. Through conducting random mutagenesis and screening of the impact of vector copy number, promoter, and mitochondrial targeting sequence, we substantially increased the mitochondrial incorporation of the allotopic protein and significantly increased recovery of mitochondrial respiration. Moreover, CN-PAGE analyses revealed that our optimized allotopic protein does not impact cytochrome c oxidase assembly, or the biogenesis of respiratory chain supercomplexes. Importantly, the most beneficial amino acid substitutions found in the second transmembrane helix (L93S and I102K) are conserved residues in the corresponding positions of human MT-CO2 (L73 and L75), and we propose that mirroring these changes could potentially help improve allotopic Cox2p expression in human cells. To conclude, this study demonstrates the effectiveness of using engineering biology approaches to optimise allotopic expression of mitochondrial genes in the baker's yeast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b719879ddbf80db7c9d88a52dbf9b4791e0038" target='_blank'>
              A new set of mutations in the second transmembrane helix of the Cox2p-W56R substantially improves its allotopic expression in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Kewin Gombeau, Stefan A Hoffmann, Yizhi Cai
          </td>
          <td>2025-04-01</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3d68234ed3b90b54b46e31e83216b5a77d6d67" target='_blank'>
              Abstract 5055: Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/242255fea6b3955bb0a9ccc76de9eb6a66d735ca" target='_blank'>
              Abstract 2843: Neoantigen evaluation in viably cryopreserved dissociated tissue via large scale proteomic profiling
              </a>
            </td>
          <td>
            Shawn P. Fahl, Casey Smith, Kerri Colwell, Jessica L. Moore, Jaison Arivalagan, Danielle B. Gutierrez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb84576556f09b1b3e9082fc0a15bf2dd37e8f04" target='_blank'>
              Abstract 3343: Centrosomal and mitotic gene expression profiles across cancer types: Implications for tumor progression and survival outcomes
              </a>
            </td>
          <td>
            Christopher L. Farrell, Amy Turner, Alex Feltus, Victoria Cipollino
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434c5eb06ef2c6750531e761d704cb04ab799fe6" target='_blank'>
              Management of T-cell malignancies: Bench-to-bedside targeting of epigenetic biology.
              </a>
            </td>
          <td>
            Ariana Sabzevari, J. Ung, Jeffrey W. Craig, K. Jayappa, Ipsita Pal, D. Feith, Thomas P Loughran, Owen A O'Connor
          </td>
          <td>2025-04-15</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4e4b812f53d84fe2b91abe7b8dd8fa8cfa234" target='_blank'>
              Abstract 1514: CRISPRi screening reveals RNA editing as a major determinant of platinum sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f4f8607796131836a1e01cc0a05131a1e222c3" target='_blank'>
              Abstract 5280: A strategic framework for modeling abnormal quiescence that drives cancer evolution, and for preclinical development of therapies to prevent recurrence and metastasis in high-risk patients
              </a>
            </td>
          <td>
            Balraj Singh, Anthony Lucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction CRISPR/Cas9-edited induced pluripotent stem cells (iPSCs) are valuable research models for mechanistic studies. However, gene conversion between a gene-pseudogene pair that share high sequence identity and form direct repeats in proximity on the same chromosome can interfere with the precision of gene editing. Mutations in the human beta-glucocerebrosidase gene (GBA1) are associated with Gaucher disease, Parkinson’s disease, and Lewy body dementia. During the creation of a GBA1 KO iPSC line, we detected about 70% gene conversion from its pseudogene GBAP1. These events maintained the reading frame and resulted from GBA1-specific cleavage by CRISPR/Cas9, without disrupting the GBA1 gene. Method To increase the percentage of alleles with out-of-frame indels for triggering nonsense-mediated decay of the GBA1 mRNA, we supplied the cells with two single-stranded oligodeoxynucleotide (ssODN) donors as homology-directed repair (HDR) templates. Results We demonstrate that HDR using the ssODN templates effectively competes with gene conversion and enabled biallelic KO clone isolation, whereas the nonallelic homologous recombination (NAHR)-based deletion rate remained the same. Discussion Here, we report a generalizable method to direct cellular DNA repair of double strand breaks at a target gene towards the HDR pathway using exogenous ssODN templates, allowing specific editing of one gene in a gene-pseudogene pair without disturbing the other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522033033b99ee07b46d656d628a06d2bb8f2da5" target='_blank'>
              Efficient GBA1 editing via HDR with ssODNs by outcompeting pseudogene-mediated gene conversion upon CRISPR/Cas9 cleavage
              </a>
            </td>
          <td>
            Joseph S. Lagas, M. Sentmanat, Xiaoxia Cui
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study presents a comprehensive genetic characterization of the H9c2 cell line, a widely used model for cardiac myoblast research. We established a short tandem repeat (STR) profile for H9c2 that is useful to confirm the identity and stability of the cell line. Additionally, we prepared H9c2 metaphase chromosomes and performed karyotyping and molecular cytogenetics to further investigate chromosomal characteristics. The genetic analysis showed that H9c2 cells exhibit chromosomal instability, which may impact experimental reproducibility and data interpretation. Next-generation sequencing (NGS) was performed to analyze the transcriptome, revealing gene expression patterns relevant to cardiac biology. Western blot analysis further validated the expression levels of selected cardiac genes identified through NGS. Additionally, Phalloidin staining was used to visualize cytoskeletal organization, highlighting the morphological features of these cardiac myoblasts. Our findings collectively support that H9c2 cells are a reliable model for studying cardiac myoblast biology, despite some genetic alterations identified resembling sarcoma cells. The list of genes identified through NGS analysis, coupled with our comprehensive genetic analysis, will serve as a valuable resource for future studies utilizing this cell line in cardiovascular medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1647a813c18457ce232ec6ab89eefeaf1dce3b9a" target='_blank'>
              Genetic and Molecular Characterization of H9c2 Rat Myoblast Cell Line
              </a>
            </td>
          <td>
            Thomas Liehr, Stefanie Kankel, Katharina S. Hardt, E. Buhl, H. Noels, Diandra T. Keller, Sarah K. Schröder-Lange, Ralf Weiskirchen
          </td>
          <td>2025-03-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973aff9217cd2f6bd7d72f2139353fd552c0fa4e" target='_blank'>
              SARE: an optimized method for detection functional regulatory elements in genome
              </a>
            </td>
          <td>
            Mohammadjavad Hosseinpoor
          </td>
          <td>2025-04-16</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c4901cd0ac19548c13b53daf9ce1ecb7db4d98" target='_blank'>
              Abstract 1758: ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            Jennifer Castro, Matthew H Daniels, S. Nayak, David Brennan, Cindy Collins, Sophie A. Shen, Monique Laidlaw, Jie Wu, Anugraha Raman, D. Gotur, Kevin E Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J Lockbaum, Kate Newberry, Stephen J Blakemore, P. Boriack-Sjodin, K. Duncan, Stuart Ince, Jason A. Sager, Robert A Copeland, Serena J Silver
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e23d29d8e2038b6c09ac7ec27f60039b41f6aae" target='_blank'>
              Abstract 1411: ADAR1 enhances ribosome biogenesis and triggers oncogenic translation
              </a>
            </td>
          <td>
            Xuliren Wang, Zhibo Shao, Qi Zhang, Han Zhu, B. Xiu, Y. Chi, Jiong Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5392c951f2de30ee4a1335350f2dfe3bff927f11" target='_blank'>
              Circular RNAs: driving forces behind chemoresistance and immune evasion in bladder cancer.
              </a>
            </td>
          <td>
            M. Saadh, Wael Sheet Hussein, Ali Fawzi Al-Hussainy, A. K. Bishoyi, M. Rekha, Mayank Kundlas, V. Kavitha, Zafar Aminov, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq
          </td>
          <td>2025-03-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39cfc9181fe4cf56d8fc7bb1fb13b18f3722bc3" target='_blank'>
              Abstract 6865: Discovery and characterization of small molecule inhibitors targeting EXO1 for BRCAness cancer therapy
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Histone modifications help control gene activity and are important for maintaining the health of spermatogonial stem cells (SSCs), which are necessary for male fertility. In this study, we used advanced genetic analysis to identify key genes related to SSC function and aging. We discovered 2509 genes that exhibited differential expression in SSCs compared to other cells, with important genes including KDM5B, SCML2, SIN3A, and ASXL3 playing roles in how DNA is organized and read. Further analysis showed that these genes play essential roles in maintaining DNA structure and regulating gene activity. We also identified gene groups associated with SSC aging and discovered potential signals that help SSCs remain active and proliferate. These findings could contribute to developing new methods to protect male fertility and advance SSC research and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27053da94cb913a8fb23edd13a7dab3ef09f1fee" target='_blank'>
              Integration of Microarray and Single-Cell RNA-Seq Data and Machine Learning Allows the Identification of Key Histone Modification Gene Changes in Spermatogonial Stem Cells
              </a>
            </td>
          <td>
            Ali Shakeri Abroudi, Hossein Azizi, Melika Djamali, Ali Qorbanee, Thomas Skutella
          </td>
          <td>2025-04-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb9f7e3e34a8c78dbc563d40d2ad93cdb90edab" target='_blank'>
              Engineered protein circuits for cancer therapy
              </a>
            </td>
          <td>
            Andrew C. Lu, Lukas Moeller, Stephen Moore, Shiyu Xia, Kevin Ho, Evan Zhang, Mark W. Budde, Haley Larson, Ali Ahmed Diaz, Bo Gu, James M. Linton, Leslie Klock, Michael J. Flynn, Xiaojing J. Gao, D. Siegwart, Hao Zhu, M. Elowitz
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The ubiquitin–proteasome system enables post-transcriptional protein modification and is a major pathway for the degradation of most of them in eukaryotic cells. Among these, the ubiquitin-specific protease (USP) family is the most extensively studied. As an important member of the USP family, ubiquitin-specific protease 39 (USP39) plays an essential role in RNA splicing and protein regulation. This review comprehensively summarizes the structural characteristics and molecular functions of USP39, emphasizing its pivotal role in the regulation of cellular processes. Dysregulation of USP39 is closely associated with the progression of various cancers through mechanisms such as immune evasion, modulation of oncogenic signaling pathways, and altered RNA splicing. These processes impact key aspects of cancer biology, including proliferation, metastasis, and therapy resistance, underscoring the broad implications of USP39 in tumor progression. Recent studies position USP39 as a promising target for cancer treatment. Future research should explore its upstream regulatory networks, develop small-molecule inhibitors, and evaluate its potential for precision oncology. This review integrates the latest insight into USP39, providing a foundation for its clinical application in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d101a1675c0ce8425b854017f981230f8a30dc4b" target='_blank'>
              Exploring the cancerous nexus: the pivotal and diverse roles of USP39 in cancer development
              </a>
            </td>
          <td>
            Yujing Chen, Jingyi Zhang, Jinfeng Yang, Jiawei Zhao, Xiaotong Guo, Juzheng Zhang, Jinfeng Gan, Weijia Zhao, Siqi Chen, Xinwen Zhang, Yi Lin, Jiamin Jin
          </td>
          <td>2025-05-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a5926479af2b1d3bad59b8a321348441349405" target='_blank'>
              Abstract 3960: Genome-wide CRISPR screens for genes inhibiting T cell-tumor cell interactions identify complex N-glycans
              </a>
            </td>
          <td>
            A. F. de Groot, Zowi R. Huinen, Juan Simon Nieto, D. Peeper
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract The SETD2 tumour suppressor encodes a histone methyltransferase that specifically trimethylates histone H3 on lysine 36 (H3K36me3), a key histone mark implicated in the maintenance of genomic integrity among other functions. We found that SETD2 protein deficiency, mirrored by H3K36me3 deficiency, is a nearly universal event in advanced‐phase chronic myeloid leukemia (CML) patients. Similarly, K562 and KCL22 cell lines exhibited markedly reduced or undetectable SETD2/H3K36me3 levels, respectively. This resulted from altered SETD2 protein turnover rather than mutations or transcriptional downregulation, and proteasome inhibition led to the accumulation of hyper‐ubiquitinated SETD2 and to H3K36me3 rescue suggesting that a functional SETD2 protein is produced but abnormally degraded. We demonstrated that phosphorylation by Aurora‐A kinase and ubiquitination by MDM2 plays a key role in the proteasome‐mediated degradation of SETD2. Moreover, we found that SETD2 and H3K36me3 loss impinges on the activation and proficiency of homologous recombination and mismatch repair. Finally, we showed that proteasome and Aurora‐A kinase inhibitors, acting via SETD2/H3K36me3 rescue, are effective in inducing apoptosis and reducing clonogenic growth in cell lines and primary cells from advanced‐phase patients. Taken together, our results point to SETD2/H3K36me3 deficiency as a mechanism, already identified by our group in systemic mastocytosis, that is reversible, druggable, and BCR::ABL1‐independent, able to cooperate with BCR::ABL1 in driving genetic instability in CML. Key Points Virtually all CML patients in blast crisis display SETD2 loss of function. SETD2 loss seems to be accomplished at the posttranslational level rather than being the result of genetic/genomic hits or transcriptional repression. Phosphorylation by Aurora kinase A and ubiquitination by MDM2 contribute to SETD2 proteasome‐mediated degradation in blast crisis CML patients. Loss of SETD2 results in increased DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58c369299bf404553dfac73a905cd3d8e7f9f95" target='_blank'>
              SETD2 loss of function is a recurrent event in advanced‐phase chronic myeloid leukemia and contributes to genomic instability
              </a>
            </td>
          <td>
            M. Mancini, S. De Santis, C. Monaldi, F. Castagnetti, M. Iezza, A. Iurlo, D. Cattaneo, Sara Galimberti, M. Cerrano, I. Capodanno, Massimiliano Bonifacio, Maura Rossi, Claudio Agostinelli, M. Meggendorfer, Torsten Haferlach, M. Cavo, G. Gugliotta, Simona Soverini
          </td>
          <td>2025-04-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d478b727dab85de298dfc2ae721a0a16fcf9bb42" target='_blank'>
              Abstract 4154: Cancer cells exhibit novel metabolic vulnerabilities when exposed to low lipid environments
              </a>
            </td>
          <td>
            Diya L. Ramesh, Keene L. Abbott, Raphael Ferreira, M. V. Vander Heiden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba50a4897b94e23b9a56ce785c4accfcfef0e23" target='_blank'>
              Abstract 655: Spatiotemporally deciphering the response dynamics of micro-satellite stable rectal cancer to neoadjuvant short-course radiotherapy and PD-1 blockade
              </a>
            </td>
          <td>
            Jingwen Wang, Zhen Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188b7f43eeddcafd2d8d2c5bfae450202c1e2b1" target='_blank'>
              Abstract 4694: (R)evolutionary approach to predicting individual response to radiation therapy in breast cancer
              </a>
            </td>
          <td>
            Naheel Khatri, Raafat Chalar, Saiful Samad, Alexander Stessin, Mehdi Damaghi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0431d2b0b8dd854cb5fd756e0cd0875bd2691218" target='_blank'>
              CRISPR/Cas9-mediated generation of two isogenic CEP290-mutated iPSC lines
              </a>
            </td>
          <td>
            Joana Figueiro-Silva, Melanie Eschment, Michelle Mennel, Affef Abidi, B. Oneda, Anita Rauch, R. Bachmann-Gagescu
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d8faa11ff0e47828029568767559cbff4cf0c6" target='_blank'>
              Abstract CT194: First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER)
              </a>
            </td>
          <td>
            Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T Henry, Jeanne Tie, N. Yamamoto, Gul Y. Serbest
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb1304f7cbde247e546624ce764f580b19d5f83" target='_blank'>
              Abstract LB187: YF087 is a potent and selective inhibitor of WRN, specifically targeting MSI-H cancer cells
              </a>
            </td>
          <td>
            Yaolin Wang, Rongfeng Liu, Yanxin Yu, Hong Yang, Jifang Weng, Yan Weng, Hong Mei, David Dai
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfdfff54622d4227a75b1cb40c4cc408256a9de" target='_blank'>
              DNA Damage Response and Repair Genes and Anthracycline-Induced Cardiomyopathy in Childhood Cancer Survivors: A Report From the Children’s Oncology Group and the Childhood Cancer Survivor Study
              </a>
            </td>
          <td>
            Xuexia Wang, Purnima Singh, Romina B Cejas, Liting Zhou, Noha Sharafeldin, P. Trainor, Wendy Landier, Changde Cheng, L. Hageman, Fan Wang, Y. Sapkota, Yutaka Yasui, M. Hudson, Eric J Chow, S. Armenian, Joseph P. Neglia, D. S. Hawkins, JillP Ginsberg, P. Burridge, Gregory T Armstrong, Smita Bhatia
          </td>
          <td>2025-03-28</td>
          <td>Circulation: Genomic and Precision Medicine</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy.
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739935357776c471010c005564687898973a29a6" target='_blank'>
              Epidemiology, pathogenesis, biology and evolving management of MSI-H/dMMR cancers.
              </a>
            </td>
          <td>
            M. Ambrosini, P. Manca, Vincenzo Nasca, C. Sciortino, F. Ghelardi, Jenny F Seligmann, J. Taieb, F. Pietrantonio
          </td>
          <td>2025-04-03</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is crucial for maintaining genomic stability by repairing DNA damage. Despite its importance, HRR's role in cancer progression is not fully elucidated. Here, this work shows that nuclear-localized branched-chain α-ketoacid dehydrogenase kinase (BCKDK) acts as a modulator of HRR, promoting cell resistance against DNA damage-inducing therapy in breast cancer. Mechanistically, this work demonstrates that BCKDK is localized in the nucleus and phosphorylates RNF8 at Ser157, preventing the ubiquitin-mediated degradation of RAD51, thereby facilitating HRR-mediated DNA repair under replication stress. Notably, aberrant expression of the BCKDK/p-RNF8/RAD51 axis correlates with breast cancer progression and poor patient survival. Furthermore, this work identifies a small molecule inhibitor of BCKDK, GSK180736A, that disrupts its HRR function and exhibits strong tumor suppression when combined with DNA damage-inducing drugs. Collectively, this study reveals a new role of BCKDK in regulating HRR, independent of its metabolic function, presenting it as a potential therapeutic target and predictive biomarker in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac4a9c9be6494a377793b137fd2fa59c3a2efc2e" target='_blank'>
              Nuclear-Localized BCKDK Facilitates Homologous Recombination Repair to Support Breast Cancer Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Haiying Liu, Jiaqian Feng, Tingting Pan, Pinggen Zhang, Ling Ye, Zetan Jiang, Zilong Zhou, Qiankun Mao, Jian Li, Xinyi Yang, P. Gao, De Huang, Huafeng Zhang
          </td>
          <td>2025-04-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca4cdf73484b348317c78d3c6466b5fa0ebab2b" target='_blank'>
              Abstract 7053: Proteomics and integrative pathway analysis to understand the mechanism of action of mithramycin and etoposide combination induced cell death in Ewing sarcoma cells
              </a>
            </td>
          <td>
            C. Lambring, K. Zaman, Elin Stone, Natalie Gierat, Ameya Bhargava, Laszlo Prokai, Riyaz Basha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CRISPR/Cas9 based genome editing has advanced our understanding of a myriad of important biological phenomena. Important challenges to multiplex genome editing in maize include assembly of large complex DNA constructs, few genotypes with efficient transformation systems, and costly/labor-intensive genotyping methods. Here we present an approach for multiplex CRISPR/Cas9 genome editing system that delivers a single compact DNA construct via biolistics to Type I embryogenic calli, followed by a novel efficient genotyping assay to identify desirable editing outcomes. We first demonstrate the creation of heritable mutations at multiple target sites within the same gene. Next, we successfully created individual and stacked mutations for multiple members of a gene family. Genome sequencing found off-target mutations are rare. Multiplex genome editing was achieved for both the highly transformable inbred line H99 and Illinois Low Protein1 (ILP1), a genotype where transformation has not previously been reported. In addition to screening transformation events for deletion alleles by PCR, we also designed PCR assays that selectively amplify deletion or insertion of a single nucleotide, the most common outcome from DNA repair of CRISPR/Cas9 breaks by non-homologous end-joining. The Indel-Selective PCR (IS-PCR) method enabled rapid tracking of multiple edited alleles in progeny populations. The ‘end to end’ pipeline presented here for multiplexed CRISPR/Cas9 mutagenesis can be applied to accelerate maize functional genomics in a broader diversity of genetic backgrounds. Supplementary Information The online version contains supplementary material available at 10.1186/s13007-025-01365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de431ff0e73b17a2417c342121bd7ddd4aecfa44" target='_blank'>
              Efficient mutagenesis and genotyping of maize inbreds using biolistics, multiplex CRISPR/Cas9 editing, and Indel-Selective PCR
              </a>
            </td>
          <td>
            Maruti Nandan Rai, Brian Rhodes, Stephen Jinga, Praveena Kanchupati, Edward Ross, Shawn R. Carlson, Stephen P. Moose
          </td>
          <td>2025-03-25</td>
          <td>Plant Methods</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c26f8d443588f8353e49604dbb91d4086b38223b" target='_blank'>
              High GLP1R gene copy numbers associated with microsatellite instability for multiple cancers and with better survival probabilities for glioblastoma.
              </a>
            </td>
          <td>
            Teresa M Thomas, Michael T Aboujaoude, Vayda R. Barker, Mallika Varkhedi, Rahul Jain, G. Blanck
          </td>
          <td>2025-04-10</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Missense mutations in the BCR::ABL1 kinase domain are found in approximately 12–80% of patients with chronic myeloid leukemia (CML). Clinically significant mutations include T315I, M244V, Y253H/F, E255K/V, V299L, and F359V. The aim of this study was to create a diagnostic system for rapid and inexpensive detection of the above mutations. We used genomic DNA and RNA from peripheral blood and bone marrow cells of 57 patients with a Ph-positive CML diagnosis established in the chronic phase. We have developed a method to detect mutations in the BCR::ABL1 gene based on allele-specific real-time polymerase chain reaction (AS-PCR). In parallel, we analyzed the RNA sequence of the protein kinase domain of the same samples by next-generation sequencing (NGS) covering the points of putative mutations. In this work, we compared the results obtained by both methods for mutation detection and variant allele frequency (VAF) estimation of mutated vs. normal alleles. The sensitivity and specificity of our diagnostic system were also evaluated. It was found that AS-PCR gives reliable results at VAF up to 0.01%. AS-PCR has high sensitivity and may serve as an alternative for the more time-consuming NGS in some cases, as well as for monitoring CML treatment and for analyzing archival material.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc35b97b7ff1497c87471f2609685fa153e61f6d" target='_blank'>
              Allele-Specific PCR for Detection of Missense Mutations in the Chimeric BCR::ABL1 Gene Causing Failure of Tyrosine Kinase Inhibitor Therapy in CML Patients
              </a>
            </td>
          <td>
            Anastasia Skripkina, I. Fevraleva, E. Kuzmina, B. Biderman, E. Stepanova, E. Chelysheva, Anna Turkina, A. Sudarikov
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea10556f7e26587dbcdc0c9ad631944d5c7f3c43" target='_blank'>
              Abstract 3824: Investigating the role of allelic imbalance in pancreatic cancer aggressiveness, survival, and therapeutic resistance
              </a>
            </td>
          <td>
            Enrico Gurreri, Li Zhang, Hania Khan, Lisa Maria Mustachio, Ningping Feng, T. Heffernan, Joseph R Marszalek, Basturk Olca, David Klimstra, Stefano Sioletic, A. Sgambato, G. Tortora, Giulio F. Draetta, Luigi Perelli, G. Genovese
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba26ff57f58d982ea6db2c2b425446ece055593" target='_blank'>
              Exploring the Prognostic and Predictive Impact of Genomic Loss of Heterozygosity and Homologous Recombination Deficiency Alterations in Patients With Metastatic Colorectal Cancer
              </a>
            </td>
          <td>
            R. Moretto, M. Germani, M. Carullo, V. Conca, A. Minelli, M. Giordano, Rossella Bruno, D. Rossini, Eleonora Gusmaroli, M. D. De Grandis, C. Antoniotti, L. Salvatore, A. Passardi, S. Tamberi, M. Scartozzi, F. Pietrantonio, S. Lonardi, Clara Ugolini, G. Masi, C. Cremolini
          </td>
          <td>2025-04-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>