<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-01 09:37:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>70</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Ovarian cancer results in the death of 1 in 6 patients within the first three months of diagnosis, making it the fifth deadliest cancer in the United States. While clinicians have moved towards using targeted therapies, patients often stop responding to the treatment. In this study, we combined both experimental and computational techniques to explore this process by treating OVCAR3 cell line with Olaparib (a PARP inhibitor). We then performed single cell RNA sequencing on both naïve and persistent populations. RNA velocities analysis indicated that there was a trajectory of gene expressions adaptation toward the persister phenotype. To further investigate the persistence mechanism, metabolic pathways were scored on our single cell data, and persistence cells were found to have differentially expressed glutamine metabolism compared to naïve cells. To test these findings, both persistent and naïve cells where cultured in glutamine deprived media and we found persistent cells have higher viability than the naïve cells particularly under Olaparib treatment. We then found that these cells acquired glucose dependency that can be used as vulnerability. This finding suggested increased plasticity of cancer cells under long-term Olaparib treatment. We then built a math model that suggested treatment scheduling could be improved by considering metabolic plasticity in response to Olaparib treatment. ONE SENTENCE SUMMARY Ovarian cancer cells exploit metabolic plasticity to switch their glutamine/glucose metabolism to persist PARP inhibitor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3565fa0fad40d6b64564b524def98cd2cdcfa949" target='_blank'>
              Metabolic plasticity drives mechanisms of PARP inhibitor persistence in ovarian cancer
              </a>
            </td>
          <td>
            Adriana Del Pino Herrera, Joon-Hyun Song, Karla Torres-Arciga, Susoma Halder, Kadin El Bakkouri, Mehdi Damaghi, Meghan C. Ferrall-Fairbanks
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b437f838fbeb4893166e383606b0c1b14d251" target='_blank'>
              Integrated Metabolic Complex Genetic Interaction Network of Chromosome 4p Loss in Basal Breast Cancer
              </a>
            </td>
          <td>
            Lynn Karam, Alain Pacis, Rohan Dandage, Michael Schwartz, Gabriel Alzial, Anastasia Gherghi, H. Djambazian, Gerardo Zapata, Marina A Schapfl, Paria Asadi, Mayerly Castrillón, Alysh Orr, Belen Hernandez, Pegah Kargar, Michelle Vandeloo, Sylvia Santosa, A. Villunger, Traver Hart, Jiannis Ragoussis, Geneviève Deblois, Andreas Bergdahl, Morag Park, Guillaume Bourque, Elena Kuzmin
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The mortality rate of ovarian cancer remains disproportionately high compared to its incidence. This is partly due to a high level of intra-tumoral heterogeneity that promotes disease recurrence and treatment failure. In this study, we describe degrees of heterogeneity revealed by single-cell whole genome sequencing and spatial transcriptomics of five epithelial ovarian carcinomas. At the cellular level, we describe pseudo-diploid cells that match the malignant cell population in both somatic variant and copy number patterns. At the clonal and subclonal levels, we describe diversification associated with copy number gains and whole genome doubling. In multi-clonal samples, we infer evolutionary relationships from single cell copy number, loss of heterozygosity analysis, and somatic variant detection, and correlate these with tissue histology and gene expression programs. In one sample, we identify functionally consequential copy number alterations that contribute to molecular diversity, cell proliferation, and inflammation in a minor clone that persisted without major expansion alongside a more complex major clone. In another, we describe a complex evolutionary history including a spontaneous reversion of a driver mutation in a secondary clone, which correlated with a switch in oncogenic expression programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, Preetesh Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid progression and the frequent emergence of resistance to standard chemotherapeutic agents such as cisplatin (DDP) and etoposide (VP16), resulting in poor clinical outcomes. To elucidate mechanisms underlying chemoresistance, we conducted a genome-wide CRISPR/Cas9 knockout screen, which identified the histone demethylase KDM6B as a critical mediator of drug resistance in SCLC. Genetic silencing of KDM6B significantly reduced IC₅₀ values of DDP and VP16, particularly in H69-AR cells, and enhanced chemotherapy-induced apoptosis. Consistently, pharmacological inhibition of KDM6B using the dual KDM6A/B inhibitor Gskj1 markedly potentiated the effects of DDP and VP16, while exhibiting minimal cytotoxicity as monotherapy. Overexpression of KDM6B rescued the chemosensitizing effect of Gskj1, thereby excluding confounding contributions from KDM6A. In vivo, the combination of Gskj1 with chemotherapy synergistically suppressed tumor growth without detectable systemic toxicity. To explore the downstream regulatory pathways, we performed transcriptome analysis via RNA-seq followed by KEGG pathway enrichment analysis, which revealed that Gskj1 treatment modulates key oncogenic signaling pathways. Integration of RNA-seq with H3K27me3 ChIP-seq data identified EGR3 as a direct epigenetic target of KDM6B inhibition. STRING analysis further suggested that EGR3 is co-expressed with c-FOS. Functional assays, including qRT-PCR, Western blotting, Co-immunoprecipitation (Co-IP), and dual-luciferase reporter assays, confirmed that EGR3 transcriptionally activates c-FOS, establishing an EGR3/c-FOS regulatory axis downstream of KDM6B. Mechanistically, inhibition of this axis enhanced chemosensitivity by promoting apoptosis, as evidenced by activation of caspase signaling, and by inducing ferroptosis through downregulation of GPX4, upregulation of ACSL4, lipid peroxidation, and modulation of HO-1. Rescue experiments with Z-VAD and ferrostatin-1 further validated that both apoptosis and ferroptosis contribute to the chemosensitizing effects of KDM6B inhibition. Finally, in vivo experiments using patient-derived xenograft (PDX) models demonstrated that Gskj1 effectively enhances the antitumor efficacy of chemotherapy in SCLC, providing compelling evidence for the clinical potential of targeting KDM6B to overcome chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a98cf22fed7bec387cd4e67f856e9e79cc0e4f" target='_blank'>
              KDM6B inhibition enhances chemotherapeutic response in small cell lung cancer via epigenetic regulation of apoptosis and ferroptosis
              </a>
            </td>
          <td>
            Zhongliang Wang, Ziyuan Liu, Yufan Yang, Wenyang Wang, Qian Chen, Shumei Liang, Linlang Guo, Man Li
          </td>
          <td>2025-11-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2093f1cec488c24db178be2a0d7e4f15fc0ea7" target='_blank'>
              Leveraging death of drug-sensitive cancer cells to promote immune-mediated bystander killing of subclones of drug-resistant tumour cells
              </a>
            </td>
          <td>
            M. Tomaschko, KangBo Ng, C. Moore, Claire E. Pillsbury, S. Rana, James Campbell, Saptaparna Mukherjee, Ania Mikolajczak, P. Anastasiou, Andrea de Castro, Alicia Alonso de la Vega, Sophie de Carné Trécesson, Nate Goehring, M. Molina-Arcas, J. Downward
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768d3fe55602ad3e6e64d51b9bc66ae2727e39e3" target='_blank'>
              TP53 and RB1 are predictive genetic biomarkers for sensitivity to cytarabine in gliomas
              </a>
            </td>
          <td>
            L. Laaniste, Joachim Luginbuehl, Mi Trinh, Rachel Watkins, Gregor Lueg, Grant Neilson, Marjolein Burgers, Kasia Kania, Regina Reynolds, Zoe Smith, Julia Kleniuk, Lauren Gould, Dorota Pregowska, Sophie Mathiesen, Michael Johnson, Louwai Muhammed
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma plasticity drives immune evasion and therapy resistance through dynamic cell-state transitions beyond genetic alterations. Epigenetic remodeling critically influences such processes, yet its role in reshaping the tumor ecosystem under therapeutic pressure remains unresolved. Here, we profiled longitudinal biopsies from melanoma patients treated in the phase Ib NIBIT-M4 epi-immunotherapy clinical trial (NCT02608437), testing the combination of a DNMT1 inhibitor with anti-CTLA4 using single-cell multiome and high-resolution spatial transcriptomics. Integrated analyses resolved seven malignant meta-programs, including a rare Wnt/β-catenin–driven melanocytic state and a de-differentiated neural crest–like state enriched in non-responders. Spatial modeling revealed that homotypic clustering stabilizes resistant programs, with neural crest–like cells forming compact, centrally localized niches, whereas Wnt/β-catenin subpopulations displayed a bimodal architecture, either cohesive clusters sustained by adhesion or dispersed, transcriptionally plastic cells. Responders exhibited progressive enrichment of an antigen presentation/interferon program and coordinated remodeling of the tumor microenvironment with T and B cell expansion, whereas tumors from non-responder patients maintained stable composition of neural crest–like clusters. Epigenetic therapy reactivated transposable elements, providing both regulatory signals that prime innate immunity within microenvironment and generating antigens that drive immunoediting and immunogenicity of Antigen presentation/interferon cell states in responders. Finally, NFATC2 emerged as a master regulator of neural crests–like transcriptional phenotypes and promoter of resistance to therapeutic interventions in melanoma patients. NFATC2 perturbation was able to shift tumor cells towards more differentiated and immunogenic states. These findings reveal how epigenetic-based immunotherapy reshapes melanoma ecosystems, provide mechanistic insights into how multiple transcriptional programs promote tumor plasticity and resistance to both combinatorial therapies and immune checkpoint blockade, identify spatial clustering as a principle stabilizing resistant niches, and highlight β-catenin and NFATC2 as actionable vulnerabilities to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Aim: This study aimed to elucidate the role of regulator of chromosome condensation 1 (RCC1) in colorectal cancer (CRC) progression, as well as its involvement in chemoresistance. We specifically examined how RCC1 knockdown modulates cellular responses, including cell cycle, apoptosis, and senescence induced by 5-fluorouracil (5-FU) or doxorubicin (Doxo) in both parental and drug-resistant CRC cell lines. Additionally, we assessed the potential of RCC1 inhibition as an adjuvant therapeutic strategy to enhance the efficacy of chemoradiotherapy in CRC. Methods: The expression of RCC1 in colon cancer tissues and corresponding adjacent non-cancerous tissues was evaluated through tissue microarrays, and its correlation with characteristics and patient prognosis was also examined. Subsequently, a series of in vivo and in vitro experiments based on parental and drug-resistant CRC cell lines were conducted to assess the impact of RCC1 knockdown on sensitivity to 5-FU or Doxo. Finally, transcriptomic analysis and subsequent validation assays were performed to explore the underlying molecular mechanisms. Results: RCC1 knockdown significantly enhanced the antitumor efficacy of 5-FU and Doxo in both CRC and drug-resistant CRC cells. In xenograft models, RCC1 knockdown in combination with 5-FU or Doxo suppressed tumor growth with no evident systemic toxicity observed. Transcriptomic profiling and experimental verification revealed that RCC1 knockdown may impair DNA repair by downregulating key repair proteins, thereby leading to more severe and sustained DNA damage. Conclusion: Our results indicate that RCC1 downregulation enhances the responsiveness of both parental and drug-resistant CRC cells to 5-FU and Doxo, highlighting its potential as a therapeutic adjunct to improve clinical outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8afbd5b9e059c22aa8009a12c6871018f16b21" target='_blank'>
              RCC1 knockdown sensitizes drug-resistant colorectal cancer to 5-fluorouracil or doxorubicin by impairing DNA repair
              </a>
            </td>
          <td>
            Jing Li, Ya Meng, Xumei Ouyang, Xiaowen Lin, Yangzhe Wu, Hang Fai Kwok
          </td>
          <td>2025-11-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, Wei Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d87abfb015ce97c63d2404657d94409eac97df" target='_blank'>
              A Sequential Triple-Drug Strategy for Selective Targeting of p53-Mutant Cancers
              </a>
            </td>
          <td>
            M. Alruwaili, Yanqi Guo, Justin Zonneville, T. Melendy, R. Straubinger, Barbara A. Foster, Priya T. Rajan, Henry G. Withers, Sarah N. Chatley, Renuka Iyer, C. Fountzilas, A. Bakin
          </td>
          <td>2025-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) presents significant therapeutic challenges, particularly due to the development of resistance to standard chemotherapeutic agents such as irinotecan. In this study, we aimed to investigate the molecular and phenotypic mechanisms underlying irinotecan resistance in CRC using the LIM1215 cell line model. Transcriptomic analysis demonstrated that drug withdrawal induced major transcriptional reprogramming, characterized by downregulation of ABC transporters (ABCB1 and ABCG2), extracellular matrix-related genes, and markers of epithelial-to-mesenchymal transition (EMT), alongside reactivation of cell cycle pathways. Drug screening further indicated that resistant cells maintained under irinotecan pressure exhibited a multidrug-resistant phenotype, while withdrawn cells regained sensitivity, particularly to tyrosine kinase inhibitors. Supplementation with the efflux inhibitor Elacridar partially restored drug sensitivity in resistant cells, emphasizing the role of transporter-mediated efflux in maintaining resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3002080effb079c5d4239d8ceb95fa35b9194a90" target='_blank'>
              Phenotypic plasticity including drug efflux drives reversible irinotecan resistance in LIM1215 colorectal cancer cells.
              </a>
            </td>
          <td>
            Tugce Di̇lber, Gizem Damla Yalcin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-10-28</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is an aggressive malignancy with a poor prognosis and limited treatment options. Previous studies have shown that selective downregulation of the MADD (MAP-kinase-activating death domain-containing protein) gene isoform increases ATC cell susceptibility to TRAIL-induced apoptosis. However, the existence of multiple MADD gene isoforms raised the possibility of functional compensation. This study aimed to definitively evaluate the role of MADD in ATC by employing CRISPR-Cas9 to Cas9 to target exon 3, a conserved exon expressed in all known MADD isoforms, resulting in functional knockout of MADD expression. CRISPR-Cas9-mediated MADD knockout, performed in three ATC cell lines (8505 C, C643, and HTH7) with distinct mutational backgrounds, significantly impaired ATC cell function in vitro, as indicated by reduced viability, increased apoptosis, decreased migration, and G0/G1 cell cycle arrest. RNA-seq analysis revealed alterations in genes related to cell survival, proliferation, and metastasis. In the orthotopic ATC mouse model, MADD deletion dramatically suppressed tumor growth, reduced lung metastases, and prolonged survival. Our findings demonstrate that MADD plays a crucial role in ATC cell survival, proliferation, and metastasis. The consistent effects observed across multiple cell lines and in vivo models suggest that MADD may represent a promising therapeutic target for this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c491eafd711a4e0b81ae9148ad43109c7f7479cf" target='_blank'>
              CRISPR/Cas9-mediated deletion of MADD induces cell cycle arrest and apoptosis in anaplastic thyroid cancer cells
              </a>
            </td>
          <td>
            Velavan Bakthavachalam, Mark A. Sanborn, Manikannan Mathayan, Satyajeet S. Salunkhe, , M. Maienschein-Cline, Suman Setty, B. Kabeer, Jalees Rehman, Bellur S. Prabhakar
          </td>
          <td>2025-11-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Mitochondria are essential organelles involved in energy production, cellular metabolism, and signal transduction. They have important impacts on tumorigenesis and cancer progression. Nevertheless, the associations between mitochondrial metabolic processes and chemotherapy resistance in colorectal cancer (CRC) are not well understood. Methods We generated a chemotherapy-resistant colorectal cancer cell line, HCT-15/DOX, via doxorubicin (DOX) induction. We then performed proteomic and metabolomic analyses via LC-MS/MS technology on both the parental and the DOX-resistant cell lines. Additionally, transmission electron microscopy was used to examine changes in mitochondrial morphology between the two cell lines. Results The results revealed significant dysregulation of 185 proteins and 1099 metabolites in HCT-15/DOX cells relative to parental cells, highlighting the impact of chemotherapy resistance on cellular processes. The key functional proteins that were identified included upregulated SDHA, BCKDHB, CRYZ, NUDT6, CPT1A, and POLG, and downregulated CRAT, FDPS, SFXN1, and ATAD3B. Additionally, through combined multiomics pathway enrichment analysis, pyrimidine metabolism, purine metabolism, ascorbate and aldarate metabolism, propanoate metabolism, and the citrate cycle (TCA cycle) were identified as important metabolic processes associated with CRC chemotherapy resistance. Transmission electron microscopy analysis revealed that HCT-15/DOX cells had increased mitochondrial number, length, and area. Conclusions This research highlights notable differences in mitochondrial morphology and diverse mitochondrial metabolic functions between parental and DOX-resistant HCT-15 CRC cells. The findings of the present study provide insights into the mitochondrial metabolic changes associated with CRC chemotherapy resistance, offering valuable insights into the mechanisms underlying these changes and identifying potential therapeutic targets for addressing CRC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47edb9f4c564452e7f562c5c1875b89a970a0bb4" target='_blank'>
              Multiomics analyses of human colorectal cancer reveal changes in mitochondrial metabolism associated with chemotherapy resistance
              </a>
            </td>
          <td>
            Shiyi Chen, Qian Li, Wei Zheng
          </td>
          <td>2025-11-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, O. Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, Mariano J. Alvarez, M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, Wesley S. Lu, Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oxaliplatin resistance remains a critical barrier to effective colorectal cancer treatment. The molecular mechanisms underlying this resistance are not fully understood, highlighting the need to define the transcriptional alterations that contribute to therapeutic failure. Accordingly, a comparative transcriptome analysis was performed on oxaliplatin-resistant colorectal cancer cells (HCT-116-ROx) and their parental counterparts (HCT-116) using RNA sequencing in this study. Differentially expressed gene (DEG) analysis was conducted using a quasi-likelihood negative binomial model. Gene Ontology (GO) enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment were carried out using the topGO and clusterProfiler packages, respectively. To confirm the robustness of the transcriptomic data, the genes with the most significant expression changes, based on false discovery rate-adjusted P value less than 0.05 and a |logFC| > 2 thresholds, were selected for validation by quantitative real-time PCR (qRT-PCR). A total of 313 DEGs were identified, including ALDH3A1 and TACSTD2 (upregulated) and IFITM1 (downregulated); these three genes were chosen for validation by qRT-PCR. Gene Ontology enrichment revealed significant changes in cell motility, redox regulation, and extracellular matrix remodeling. KEGG analysis indicated upregulation of ferroptosis, glutathione metabolism, and lysosome-related pathways, and downregulation of p53 signaling, oxidative phosphorylation, and cancer-specific pathways. Oxaliplatin-resistant colorectal cancer cells undergo multifaceted transcriptional reprogramming that promotes redox homeostasis, metabolic adaptation, and structural plasticity while suppressing apoptotic and mitochondrial functions. These changes support chemoresistance and may represent potential therapeutic targets to restore drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5ab501b314c92e8ff167ffcb8251950e164cb1" target='_blank'>
              Multifaceted transcriptional reprogramming supports oxaliplatin chemoresistance in colorectal cancer cells.
              </a>
            </td>
          <td>
            Gizem Calibasi-Kocal, Hasan Kurter, Zerrin Isik
          </td>
          <td>2025-10-09</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/137a07d5d39a7f965a9829b9a10c7a07db82c57e" target='_blank'>
              Multi-omics-driven kinetic modeling reveals metabolic adaptations and vulnerabilities in BRCA1-deficient ovarian cancer
              </a>
            </td>
          <td>
            Ilias Toumpe, M. Masid, V. Hatzimanikatis, L. Miskovic
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Ovarian cancer (OC) is lethal due to near universal development of resistance to platinum-based chemotherapy. Metabolic adaptations can play a pivotal role in therapy resistance. Here, we aimed to identify key metabolic pathways that regulate platinum response and represent potential therapeutic targets. Transcriptomic and metabolomic analyses in cisplatin sensitive and resistant ovarian cancer cells identified enrichment of pyrimidine metabolism related to upregulated de novo pyrimidine synthesis. 15N-glutamine flux analysis confirmed increased de novo pyrimidine synthesis in cisplatin resistant cells. Targeting this pathway using brequinar (BRQ), an inhibitor of the key enzyme dihydroorotate dehydrogenase (DHODH), decreased cell viability, delayed G2/M cell cycle progression, and altered expression of genes related to mitochondrial electron transport in resistant cells. Under basal conditions, cisplatin resistant cells had a lower oxygen consumption rate (OCR) and spare respiratory capacity (SRC) than sensitive cells. BRQ suppressed OCR in both sensitive and resistant but only inhibited SRC in resistant cells. In cell line-derived and patient-derived xenograft models, BRQ attenuated the growth of cisplatin resistant ovarian tumors and enhanced the inhibitory effects of carboplatin. Together, these results identify metabolic reprogramming in cisplatin resistant ovarian cancer that induces an acquired dependency on de novo pyrimidine synthesis, which can be targeted to sensitize tumors to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d708b6c5cad44de1ded3e0fd0aa39875db711a4c" target='_blank'>
              Metabolic Dependency on De Novo Pyrimidine Synthesis is a Targetable Vulnerability in Platinum Resistant Ovarian Cancer.
              </a>
            </td>
          <td>
            H. Cardenas, Yinu Wang, Guangyuan Zhao, Delan Xingyue Hao, A. Isac, Vanessa Hernandez, Ujin Kim, Wenan Qiang, Hao F. Zhang, Daniela Matei
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9cba62bd659ab0fed0834794015eb81e0685b90" target='_blank'>
              Integrative Multi-Omics Identifies CDK1 as a Key Signaling Regulator of CD4+ T Cell Effector Function
              </a>
            </td>
          <td>
            N. Servaas, H. G. Bauersachs, Luisa Abreu, A. Claringbould, I. Berest, Jennifer J. Schwarz, Frank Stein, Maria Fälth-Savitski, Lena Eismann, James P. Reddington, Judith B. Zaugg
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, 
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="It has been more than 20 years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding the biology of mammalian diseases. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps and only partially represents many biomedically relevant loci. Here we present the first telomere-to-telomere (T2T) mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal substantial variability in telomere and centromere sizes and structural organization. We thus add an additional 213 Mb of new sequence to the reference genome, which contains 517 protein-coding genes. We also examined two important but incomplete loci in the mouse genome—the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc-finger protein loci. We identified distant locations of the PAR boundary, different copy numbers and sizes of segmental duplications and a multitude of amino acid substitution mutations in PAR genes. Telomere-to-telomere assemblies of two mouse inbred strains, C57BL/6J and CAST/EiJ, offer improvements over the current mouse reference genome by adding telomere and centromere sequences that lead to insights into variability in telomere and centromere sizes and organization, and the discovery of 225 and 355 new genes for C57BL/6J and CAST/EiJ, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C. Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00da2b68fdb702605bdeb1d0cfe9d2ae87812c76" target='_blank'>
              Co-targeting MERTK and EGFR with a Bispecific Antibody Overcomes Drug Resistance Across Mutations in Exons 19, 20, and 21
              </a>
            </td>
          <td>
            Suvendu Giri, Boobash-Raj Selvadurai, Arturo Simoni-Nieves, Nitin Gupta, Moshit Lindzen, Rishita Chatterjee, Alessandro Genna, Marieke Van Daele, Deepthi Ramesh-Kumar, Mirie Zerbib, R. Oren, Isabelle Sophie Meijer, Elisabeth Franziska Wippich, Yahel Avraham, Rony Dahan, Laura E. Kilpatrick, Simon Platt, Stephen J. Hill, L.R. Pal, E. Ruppin, Donatella Romaniello, M. Lauriola, Yosef Yarden
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are central architects of the prostate cancer (PCa) microenvironment, yet their phenotypic diversity and druggable vulnerabilities remain largely uncharted. Here, we present an integrative multi-omics characterization of primary ex vivo CAFs from seven treatment-naïve PCa patients. Using single-cell RNA sequencing (scRNA-seq), we uncover substantial transcriptional heterogeneity among CAFs, with distinct gene expression programs related to extracellular matrix remodeling, inflammation, immune modulation, and metabolic reprogramming. This phenotypic diversity was further supported by variable expression of canonical stromal markers, including FAP, SULF1, VIM, CAV1, and αSMA. Transcription factor network analysis revealed SOX, FOX, and STAT3 family members as key regulators of pro-tumorigenic CAF states. To probe therapeutic vulnerabilities, we performed high-throughput drug sensitivity and resistance testing (DSRT) across 396 oncology compounds. CAFs exhibited broad sensitivity to multikinase inhibitors, with dasatinib, midostaurin, and FGFR inhibitors (AZD4547, erdafitinib) emerging as top stromal-directed candidates. These findings underscore the plasticity of prostate CAFs and reveal actionable vulnerabilities, supporting the development of targeted stromal therapies to disrupt tumor-stroma interactions in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0342abe6ecbf94b88ae457a3f680a1e8d054b454" target='_blank'>
              Molecular profiling of ex vivo prostate cancer CAF models captures stromal heterogeneity and drug vulnerabilities
              </a>
            </td>
          <td>
            Frida Rantanen, A. Murumägi, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-11-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT The DNA damage response (DDR), an evolutionarily conserved surveillance mechanism enabling cells to detect genomic lesions and cell cycle arrest for repair, coordinates genomic stability through key effector kinases. In Candida albicans, our prior work characterized Rad53-mediated transcriptional reprogramming based on the RAD53 deletion strain, establishing its canonical DDR functions. However, Rad53 activation dynamically orchestrates cell cycle synchronization and repair coordination during genotoxic challenge. Investigating cellular modifications resulting from increased Rad53 functionality is essential for a deeper understanding of its regulatory response. To this end, we overexpressed Rad53 revealing its dual regulatory capacity in stress adaptation and developmental plasticity. Global transcriptomic profiling demonstrated that RAD53 overexpression enhances methyl methanesulfonate (MMS) resistance while paradoxically stimulating filamentous growth, accompanied by upregulation of DNA repair/replication genes (RFA3, POL30) and non-canonical DDR targets including cell wall remodeling factors. Comparative analysis of RAD53 deletion and overexpression transcriptomes identified enriched transcription factors, notably Sfl1, Yox1, and Rfx1. In particular, Rfx1 mediated the resistance to genotoxic stress resulting from RAD53 overexpression or deletion. Surprisingly, Rad53 overexpression substantially hindered hyphal formation and virulence, whereas its kinase-dead mutation restored these deficiencies. Notably, Rad53’s role in mediating morphogenesis was independent of Rfx1. This multi-modal functionality positions Rad53 as a critical node interfacing genomic stability, developmental plasticity, and pathogenicity in C. albicans. Our study enhances the understanding of the DDR pathway in C. albicans, providing a potential target for treating this fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ff870ecbc422dc2701275aa24d5074e975660" target='_blank'>
              Rad53 orchestrates divergent pathways governing genotoxic resistance, morphogenesis, and virulence in Candida albicans: Mechanistic decoupling through overexpression analysis
              </a>
            </td>
          <td>
            Jia Wang, Shaling Yang, Yi Zhou, Huaxin Cai, Jinrong Feng
          </td>
          <td>2025-10-06</td>
          <td>Virulence</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45dd551579bb493b99c7ec5a7c44f7e95f00ebc0" target='_blank'>
              Single cell whole genome and transcriptome sequencing links somatic mutations to cell identity and ancestry
              </a>
            </td>
          <td>
            Abhiram Natu, Mrunal K. Dehankar, Reenal Pattni, M. Šuvakov, Dmitrii Olisov, Livia Tomasini, Yeongjun Jang, Yiling Huang, Eva Benito-Garragori, Patrick Hasenfeld, Jan O. Korbel, Alexander E. Urban, A. Abyzov, F. Vaccarino
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Bladder cancer (BLCA) is the predominant type of urothelial carcinoma in urinary system, and resistance to cisplatin-based chemotherapy substantially worsens clinical outcomes, presenting a major therapeutic obstacle. In this study, we integrated genome-wide association study (GWAS) data with expression quantitative trait loci (eQTL) analyses, and applied Mendelian randomization (MR) to assess the causal effects of eQTLs from 19,942 genes on BLCA. By incorporating scRNA-seq data, our study also identifies differentially expressed genes (DEGs) in cisplatin-resistant BLCA cells and examined their causal associations with BLCA, aiming to elucidate genetic drivers of chemoresistance and tumor progression. Through this integrated approach, we identified the eQTL of the ARHGEF12 gene as a key mediator of cisplatin resistance. Bioinformatic analysis revealed that elevated ARHGEF12 expression was strongly associated with activation of the PI3K/Akt signaling pathway. To define ARHGEF12’s role in cisplatin resistance, we established a cisplatin-resistant UM-UC-3/DDP model. Silencing ARHGEF12 markedly reduced chemoresistance, increased apoptotic cell death, and induced pronounced morphological changes. Pharmacological modulation with the ROCK inhibitor Y-27632 and a rescue assay with the Akt activator SC79 supported a model in which ARHGEF12 drives chemoresistance via RhoA/ROCK-dependent activation of the PI3K/AKT axis. This study is the first to integrate MR with single-cell transcriptomics to explore the genetic contribution to cisplatin resistance in BLCA. Our results uncover a novel mechanistic role of ARHGEF12 in BLCA progression and chemoresistance and suggest it as a potential therapeutic target for precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542b443be27492e921709c204aeb3560eab221c4" target='_blank'>
              Integrative multi-omics and single-cell transcriptomics reveal ARHGEF12 driving chemoresistance in bladder cancer
              </a>
            </td>
          <td>
            Kunyao Zhu, Zhejiao Zhang, Tinghao Li, Yan Sun, Junlong Zhu, Linfeng Wu, Xiaoyu Zhang, Hang Tong, Zijia Qin, Aimin Wang, Weiyang He
          </td>
          <td>2025-11-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a923c0764db3429c39a600f86e6a4ae2c3ce26" target='_blank'>
              Regulation of ribosomal gene expression and senescence by a PML-mTOR-RONIN nuclear complex in triple-negative breast cancer
              </a>
            </td>
          <td>
            Younes Medkour, C. Dufour, Lingwei Han, Phillipe Hutton, Mirna Farhat, Anthony Alfonso, A. Rambur, Mathieu Vernier, Vincent Giguère
          </td>
          <td>2025-11-08</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37222bf4d60567842a4aa73e1f34998e136ade83" target='_blank'>
              Translatome and translation dynamics analysis of a RiboCancer cell line panel reveals that leukemia-associated Rps15 mutations rewire translation through codon-specific tRNA accommodation defects
              </a>
            </td>
          <td>
            Anaïs Astier, Marino Caruso, Stijn Vereecke, Paulo E. Santo, Coralie Capron, C. Froment, Dana Rinaldi, David Cabrerizo Granados, Marine Leclercq, Jonathan Royaert, J. Verbeeck, Naomy Pasau, L. Plassart, Daniele Pepe, Steven Verbruggen, Hermes Paraqindes, Simon Lebaron, V. Marcel, Sébastien Durand, Clément Chapat, G. Menschaert, Pierre Close, F. Rapino, Frédéric Catez, J. Marcoux, Célia Plisson-Chastang, Kim De Keersmaecker
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Chemoresistance represents a significant challenge in the chemotherapy of colorectal cancer (CRC), limiting the effectiveness. In this regard, gene expression heterogeneity plays a critical role, influencing cancer cell adaptability and survival under chemotherapy. Our previous data revealed that ribosomal protein uL3 positively correlates with both chemoresistance and poor prognosis in CRC patients. This study explores the combination of 5-fluorouracil (5-FU), the first-line treatment of CRC, with G-quadruplex (G4) ligands, which have recently emerged as promising candidates for cancer therapy, to overcome uL3-mediated chemoresistance. We found that resistant p53-deficient and uL3-silenced CRC cells showed increased levels of G4 structures compared to both sensitive p53-deficient and p53-proficient cells, thereby exhibiting vulnerability to the cytotoxic effects of two well-established G4 ligands, pyridostatin (PDS) and RHPS4. The combination of 5-FU with PDS or RHPS4 exhibited a synergistic effect, selectively targeting tumor cells. This approach enabled a >10-fold reduction in the 5-FU dose, improving treatment efficacy. The effectiveness of this combination was further validated in vivo using uL3-silenced CRC cell-derived xenografts in a chicken embryo model. Overall, our findings highlight a novel and promising combination strategy that targets chemoresistance in CRCs characterized by non-functional p53 and reduced levels of uL3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026391aec8bb902ae2019fd3d4037ad920903a6e" target='_blank'>
              Reversing uL3-mediated chemoresistance through synergistic combination of 5-FU and G-quadruplex ligands
              </a>
            </td>
          <td>
            Anna Di Porzio, Annalisa Pecoraro, Camilla Danisi, Carolina Persico, Ludovica D’Auria, Marcello Germoglio, Nunzia Iaccarino, Isidoro Feliciello, C. Giancola, Antonio Randazzo, G. Russo, A. Russo
          </td>
          <td>2025-11-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, S. Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gemcitabine is the primary chemotherapeutic agent used as a first-line treatment for pancreatic cancer (PC), underscoring its pivotal role in cancer treatment. This drug induces apoptosis through multiple mechanisms, with the disruption of cellular pyrimidine metabolism being a crucial mode of action. However, gemcitabine resistance significantly limits the long-term efficacy of PC treatments. Our study aimed to identify potential targets for gemcitabine sensitization. We identified a gene set associated with pyrimidine metabolism through cluster analysis of The Cancer Genome Atlas Pancreatic Adenocarcinoma database. Intersecting this set of differentially upregulated genes in PC with normal tissues revealed inhibin βA (INHBA) as a gene related to pyrimidine metabolism. Quantitative reverse transcription-polymerase chain reaction, western blotting, and immunohistochemistry analyses confirmed INHBA overexpression in PC cells. INHBA’s effects of INHBA on cell proliferation and chemoresistance in PC cells were validated in vitro using a colony counting kit-8 assay, drug sensitivity analysis, EdU incorporation, flow cytometry, and colony formation assay. The proliferation and chemoresistance promotion by INHBA in PC cells was validated in vivo using a mouse xenograft model. Downstream targets of INHBA were identified using immunoprecipitation mass spectrometry. INHBA, a pyrimidine metabolism-associated gene, was highly expressed in PC, and its expression correlated with gemcitabine resistance. INHBA interacts with cytidine triphosphate synthase 1 (CTPS1) and competitively inhibits SMAD Specific E3 Ubiquitin Protein Ligase 1 mediated ubiquitination, enhancing CTPS1 stability and promoting pyrimidine metabolism. INHBA promotes gemcitabine resistance in PC by stabilizing CTPS1 and enhancing pyrimidine metabolism, making it a potential target for chemosensitization in PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aea2b2ecd7e98f320aad3799b76a99f906f124b0" target='_blank'>
              INHBA promotes chemoresistance in pancreatic cancer by enhancing CTPS1 stability and mediating pyrimidine metabolism
              </a>
            </td>
          <td>
            Xinghao Sun, Zikai Wei, Xingkun Liao, Dijie Zheng, Futang Li, Liwen Chen, Yanyu Gong, Suye Ran, Xiaozhao Han, Zhiwei He, Chao Yu
          </td>
          <td>2025-11-14</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elucidating mechanisms of drug resistance is key for overcoming resistance, guiding drug design, and enabling accurate resistance prediction. Recently, disease metabolic reprogramming has emerged as a novel mechanism of resistance, which enables disease cells to adapt to therapeutic resistance by altering energy production pathways, cellular signaling, and biosynthesis processes. Moreover, protein structure alterations also play a pivotal role in resistance study, facilitating mechanistic understanding, and structure-based target discovery. In other words, integrating these recently accumulated critical data is essential for enriching the landscape of drug resistance data. Therefore, in this study, DRESIS was a significant update by providing (i) 236 molecules that drive metabolic reprogramming and confer resistance to 168 drugs, together with a detailed mechanism, (ii) 2228 protein structural variants implicated in resistance to 671 drugs across 238 diseases, and (iii) greatly expanded landscapes of drug resistance information, now featuring 398 newly added key drug-resistant molecules, 356 drugs with the latest published resistance mechanisms, and 81 new drug-resistant disease categories. All in all, DRESIS 2.0 is expected to serve as a valuable resource for the scientific community and provide important support in tackling the global challenge of drug resistance, which is now publicly accessible at https://idrblab.org/dresis/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafc45a66260eff1536a05a8101c61e45dec038b" target='_blank'>
              DRESIS 2.0: the comprehensive landscape of drug resistance information.
              </a>
            </td>
          <td>
            Xiuna Sun, Zhangle Wei, Xinyuan Yu, Kaixuan Liu, Shun Yao, Zheng Ni, Hanbing Wang, Ziming Zhao, Yinpeng Zhang, Yuxuan Liu, Hanlu Ding, Yintao Zhang, Zhiguo Liu, Mang Xiao, Feng Zhu
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Glioma, the most common primary brain tumor, shows poor prognosis and resistance to first-line therapies, like temozolomide (TMZ), lomustine (LOM) and panobinostat-marizomib (PM), complicating treatment and diminishing disease-free survival. It is critical to investigate the molecular mechanisms that underlie chemoresistance to develop effective therapies. This study examines common transcriptional networks that facilitate chemoresistance to identify novel therapeutic targets that may overcome resistance.



 The pediatric high-grade glioma KNS42 cell line was chronically treated with increasing doses of TMZ, LOM, or PM to induce drug resistance (RES); control cells received vehicle (DMSO) treatment. Bulk-RNA sequencing was performed followed by differential gene expression analyses (PartekFlow-ANOVA/DESeq2) followed by pathway and comparison analysis (QIAGEN IPA, |FDR|<0.05).



 RNA-sequencing analysis followed by comparison-analysis revealed differential gene expression in treatment-resistant glioma cells as compared to control cells, including 3,839 differentially expressed genes (DEGs) in LOM RES, 3,641 in TMZ RES, and 3,445 in PM RES groups compared to control group. Among these, 374, 417, and 360 DEGs were downregulated in LOM, TMZ, and PM RES cells, respectively, while 265 (LOM and TMZ RES) and 216 (PM RES) DEGs were upregulated (log2FC > 2; p < 0.05). Interestingly, when the top 10 upregulated genes from each resistant group were compared, six genes (VCAN, FOXA1, SOX2, PDGFRA, ERBB4, and COL4A) were found to be commonly upregulated in all 3 resistant groups. We further confirmed that SOX2, PDGFRA, and ERBB4 are upregulated at the protein level. Ongoing studies will test whether these 6 DEGs are required for mediating resistance against diverse drugs.



 RNA-sequencing identified common genes linked to TMZ, LOM, and PM chemoresistance, revealing key genetic networks involved in acquiring and maintaining drug resistance. Targeting these pathways, along with epigenetic silencing and/or combinatorial therapies, may resensitize chemoresistant cells and allow for effective treatment of glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5824551e4b1f9c0b4ac4e374b69dbac7316d823" target='_blank'>
              DDDR-78. Identifying common transcriptional mechanisms in treatment-resistant glioma
              </a>
            </td>
          <td>
            N. B. Rao, Aaron R Johnson, Subrahmanya D. Vallabhapurapu, Tracy A Miller, Ian F Pollack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immune checkpoint blockade (ICB) therapy, which has revolutionized cancer treatment, has been approved for the treatment of triple-negative breast cancer (TNBC). Unfortunately, most patients with TNBC are either not eligible for treatment or exhibit resistance, resulting in limited overall survival benefits. There is an urgent need to elucidate the mechanisms of resistance and enhance therapeutic efficacy. Here, via CRISPR activation (CRISPRa) screening, we identified family with sequence similarity 114 member A1 (FAM114A1) as a key mediator of immune evasion and ICB resistance in TNBC. Mechanistically, FAM114A1 binds p85α to disrupt the p85α/p110α protein complex, thus activating the PI3K/AKT pathway and simultaneously preventing condensate formation of E2F Transcription Factor 4 (E2F4) to promote E2F4-driven Metadherin (MTDH) transcription. Upregulation of these FAM114A1-mediated pathways suppresses tumor antigen presentation and consequently attenuates antitumor immunity in TNBC. Moreover, targeting FAM114A1 improves the therapeutic effectiveness of anti-PD-1 therapy in mouse models, and a FAM114A1-based signature shows strong predictive performance for identifying patients with TNBC who may benefit from ICB. Collectively, our findings not only reveal that FAM114A1 is an immune evasion driver but also highlight it as a promising biomarker and therapeutic target. Our study provides new insights into TNBC immune evasion and outlines a potential avenue to improve the effectiveness of ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6cebf1ccb54b2528f79a6a1b67182cda5a3f55" target='_blank'>
              Family with sequence similarity 114 member A1 orchestrates immune evasion in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wenhao Zhang, Yanzhi Gai, Mengxue Qiao, Michelle Rowicki, Yong Wei, Xiang Hang, Zhengkai Wei, He Yang, Xifu Ye, Hang Ju, Yi Lu, Yibin Kang, Minhong Shen
          </td>
          <td>2025-11-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has shown remarkable success in hematologic malignancies but faces significant challenges in solid tumors due to the immunosuppressive tumor microenvironment (TME). Among these, hypoxia plays a vital role, yet the molecular mediators that link hypoxia to CAR-T dysfunction remain incompletely understood. Anti-mesothelin (MSLN) CAR-T cells were cultured under normoxic (21% O2) and hypoxic (1% O2) conditions for six days. We assessed cell expansion, phenotypes, cytotoxicity, and metabolic features. RNA sequencing was conducted to identify key gene expression changes induced by hypoxia. Findings were further validated using anti-CD70 CAR-T cells. Hypoxia reduced CAR-T proliferation, increased apoptosis, lowered memory phenotypes, raised exhaustion, and weakened cytotoxicity in short-term and long-term assays. Transcriptomic and metabolic analyses showed metabolic reprogramming with increased glycolysis and reduced oxidative phosphorylation. Among the dysregulated genes, the serine/threonine-protein kinase PIM3 emerged as a previously underexplored mediator of hypoxia-driven dysfunction. Genetic or pharmacologic inhibition of PIM3 counteracted hypoxia-induced impairment, enhancing memory phenotypes of CAR-T cells, and improving their anti-tumor activity both in vitro and in vivo. This work identifies PIM3 as a previously underexplored target that links hypoxia to CAR-T cell dysfunction and demonstrates that PIM3 inhibition can reverse these effects. These findings provide a mechanistic rationale for incorporating PIM3 inhibition into CAR-T cell manufacturing or engineering to improve their therapeutic potential in hypoxic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adcb89e4cbae164c14c49d77ccdb838d38a89aa" target='_blank'>
              Metabolic reprogramming through PIM3 inhibition reverses hypoxia-induced CAR-T cell dysfunction in solid tumors
              </a>
            </td>
          <td>
            Muya Zhou, Luxia Xu, Jinhua Hu, Wenwen Chen, Jialu Hong, Mufeng Wang, Zhi-Cheng Guo
          </td>
          <td>2025-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74bd0c1fdd61877244de8ec8adaa6272cd0c1476" target='_blank'>
              Loss of PIMREG impairs double-strand break signaling and repair
              </a>
            </td>
          <td>
            Maria Vitória de Rizzo Gasparini, Laís do Carmo, Giovanna Gonçalves de Almeida, Mariana Siqueira Lacerda Mamede, Isabela Spido Dias, Jennifer Ann Black, V. C. Arfelli, Yun-Chien Chang, S. Heinzlmeir, Bernhard Kuster, Philipp A. Greif, Leticia Fröhlich Archangelo
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epstein-Barr virus associated gastric cancer (EBVaGC) accounts for ∼9–10% of gastric cancers worldwide and is defined by a distinctive molecular profile, including extreme hypermethylation of the DNA. Targeting this aberrant methylation may be a potential therapeutic strategy. EBV+ gastric cancer cell lines (YCCEL1, SNU719) and EBV- lines (AGS, SNU16, MKN74) were treated with a DNA methyl-transferase inhibitor (DNMT), decitabine (DCB), for three days followed by RNA sequencing to identify EBV-specific responses. DNA methylation profiling by reduced representation bisulfite sequencing (RRBS) was performed in EBV+ cell-lines and integrated with expression data to identify epigenetically regulated networks. While DCB induced broad transcriptional changes across all lines, EBV+ cells exhibited the strongest transcriptional response, sharing many upregulated genes. Many of these EBV+ specific genes were expressed at lower baseline levels in EBV+ tumors from TCGA. DCB predominantly reduced methylation at highly methylated intergenic CpGs, with a subset of promoters undergoing significant demethylation. Integrated analysis revealed a strong inverse correlation between promoter demethylation and gene expression, implicating multiple cancer-relevant pathways. Upstream regulator analysis and motif enrichment indicated that regions losing methylation were enriched for retinoic acid receptor α (RARα) binding motifs, suggesting that DCB-mediated demethylation restores RA pathway accessibility and transcriptional activity. Further, inhibiting RAR signaling reduced DCB induced apoptosis. Although DCB can induce both host gene re-expression and viral lytic gene activation in EBV-positive tumors, its impact on RA signaling in EBVaGC has not been studied. Decitabine promotes extensive epigenetic reprogramming in EBVaGC, with preferential effects in CIMP-positive, EBV-infected cell lines. Importance EBV+ gastric cancer contains hypermethylated DNA and despite this distinct molecular phenotype there are currently no EBV-specific treatments available. Using an FDA approved inhibitor to target hypermethylated DNA and multi-omics approach to study the cellular response, we uncovered epigenetically altered transcriptional networks that may be further exploited to improve potential therapy. Among the pathways disrupted, retinoic acid signaling is of particular interest, as retinoid receptors such as RARα and RARβ are frequently hypermethylated and repressed in EBVaGC. Our findings indicate that DNMT inhibition can partially reverse RA receptor silencing, supporting further investigation of DNMTi–RA combination strategies as a novel therapy for EBV+ gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74a4f19fc0f137d7c09390e6a4bff845ed706aa" target='_blank'>
              Integrated transcriptomic and methylome analysis reveals retinoic acid pathway activation after decitabine treatment in EBV associated gastric cancer
              </a>
            </td>
          <td>
            Sarah Preston-Alp, Yin Wang, Matteo Bergonzoni, Andrew V Kossenkov, Samantha S. Soldan, L. B. Caruso, Davide Maestri, Paul M. Lieberman, B. Gewurz, I. Tempera
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA), a probable human carcinogen, induces toxic and mutagenic O6-methylguanine lesions that are repaired by the O6-methylguanine methyltransferase (MGMT). To elucidate mechanisms of NDMA-induced liver cancer progression, we performed longitudinal analyses of phenomic, transcriptomic, and phosphoproteomic changes in wild-type and MGMT-deficient mice, observing amplified responses in the deficient genotype. Early molecular rewiring indicative of a DNA damage response was detected by phosphoproteomic and transcriptomic profiling within days post-exposure. Transcriptomic analyses identified a persistent and robust interferon response as the dominant activated pathway. This chronic interferon signaling, which remained unresolved, correlated with extensive clonal expansion, an early hallmark of oncogenesis. Spatial transcriptomics further revealed pathway alterations favoring tumorigenesis within clonally expanded cells. These findings delineate the cascade of molecular events triggered by acute early-life NDMA exposure, culminating in cancer development months later. Our study unveils potential predictive biomarkers and strategies for disease mitigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily M. Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408e4356836091a48672426b49d0e709f36de3ec" target='_blank'>
              The transcriptional regulatory circuit as a driver and therapeutic target in CML blast crisis
              </a>
            </td>
          <td>
            Liling Jiang, Enzhe Lou, Jinxin Fang, Lizhen Jiang, Guanjie Peng, Jinghong Chen, Bingyuan Liu, Bo Lu, Yi Meng, Haichuan Zhang, Aochu Liu, Qiong Mao, Peilong Lai, Yueyuan Zheng, Jinbao Liu, Xianping Shi
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division3–6 yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Solid tumors collectively drive the global cancer burden, with profound molecular heterogeneity demanding precision and molecularly informed management. Advances in sequencing technologies have established molecular taxonomy as a cornerstone of clinical oncology, progressively superseding traditional histopathological classifications. Sanger sequencing remains the gold standard for validating guideline mandated actionable variants. Next-generation sequencing (NGS) has revolutionized early cancer detection through liquid biopsy applications and enabled the reclassification of diagnostically challenging tumor subtypes. Emerging long-read platforms offer unique capabilities to resolve complex genomic rearrangements, structural variants, and therapy-induced epigenetic remodeling. Consequently, therapeutic strategies are shifting from organ-centric approaches to mutation-specific interventions, exemplified by non-small-cell lung cancer, where molecular stratification drives substantial improvements in treatment response. Nevertheless, temporal tumor heterogeneity, biological contamination, and computational limitations highlight the urgent need for robust, integrated verification systems. Collectively, this evolution positions sequencing as the operational backbone of adaptive precision oncology across solid tumors. Here, we synthesize our laboratory findings with the current literature to comprehensively review the diagnostic, therapeutic, and prognostic applications of first- through fourth-generation sequencing technologies and discuss future directions in this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70aff33af4435f12308495dd30d443469c249256" target='_blank'>
              Shaping Precision Medicine: The Journey of Sequencing Technologies Across Human Solid Tumors
              </a>
            </td>
          <td>
            Wanwen Li, Chanyu Xiong, Chen Chu, Yun Zhang, Zihao Wang, Zunmin Wan, Peng Tang, Shikai Zhu, Yu Zhou
          </td>
          <td>2025-10-29</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D. Anderson, Felipe Luz Torres Silva, Mi K. Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec13fad98d61dff3c83ecb3a4a35f96c11ec562" target='_blank'>
              Targeting MORF4L1-mediated DNA repair potentiates RT-induced antitumor immunity via cGAS-STING activation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si-wei Wang, Wei-Feng Hong, Yi-Lan Huang, Yi-Min Zheng, Qiu-Yi Zheng, Jun-Jie Cheng, Bu-Fu Tang, Gen-Wen Chen, Bu-Gang Liang, Jia-Cheng Lu, Li Yuan, Shujung Hsu, Yang Zhang, Xiao-Bin Zheng, Zhaochong Zeng, Jian Zhou, Jia Fan, Aiwu Ke, Chao Gao, Shi-Suo Du
          </td>
          <td>2025-11-05</td>
          <td>Cellular & Molecular Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Germ cell tumors (GCTs) are rare, heterogeneous neoplasms derived from primordial germ cells. Although they typically develop in the gonads, they may also arise in extragonadal locations along the midline of the body. Approximately 90% of patients respond well to cisplatin-based chemotherapy; however, ~30% exhibit treatment resistance. Epithelial-mesenchymal plasticity (EMP), a recognized hallmark of cancer, has been implicated in promoting metastasis and chemoresistance. Nonetheless, studies investigating the specific role of EMP in GCT treatment resistance remain limited. The present review compiles key studies on GCTs, EMP markers and cisplatin resistance using both in vitro and in vivo models; it highlights the roles of associated genes, transcription factors and proteins, identifying potential therapeutic targets. Advancing our understanding of EMP and identifying novel therapeutic targets may support the development of treatment strategies that complement or replace cisplatin. This, in turn, could improve survival outcomes and create new avenues for molecular research and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731201ecd287c2222a955e4ad4ca52b43518aba0" target='_blank'>
              Epithelial-mesenchymal plasticity and cisplatin resistance in germ cell tumors: Mechanisms and emerging therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Eleni Solange De Brito Gomes, Tainá De Souza, Ana Laura Oliveira, Ana Flavia Bezerra, Ingridy Izabella Vieira Cardoso, Lenilson Silva, Luiz Fernando Lopes, Marcela Nunes Rosa, Mariana Tomazini Pinto
          </td>
          <td>2025-10-13</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains difficult to cure despite recent advances. Off-target side effects of drugs currently used lead to significant morbidity and mortality. There is recognition that in AML, there is an increased dependence on OXPHOS metabolism, especially in the leukemia stem cell compartment (AML-LSC). It is also recognized that there is potential to exploit this vulnerability to treat AML. Drug re-purposing screens have suggested the potential use of artesunate (ART) to inhibit mitochondrial respiration. We have explored the potential role of ART as an additive agent in treating AML in combination with conventional therapy. Through in-vitro and in-vivo mouse model studies, we demonstrate the mechanism and efficacy of these combinations and their potential to overcome venetoclax resistance. We further demonstrate the specificity of these combinations with minimal off-target effects on normal hematopoietic stem cells (HSC). These observations warrant exploration of the additive role of ART in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f30bb0a803c2c360d413ebfd59a4fc9ab44ee4" target='_blank'>
              A novel strategy to target metabolic dependencies in acute myeloid leukemia
              </a>
            </td>
          <td>
            Nithya Balasundaram, Hamenth Kumar Palani, Arvind Venkatraman, Yolanda Augustin, Shruthi Pichandi, Clement Regnault, Majeela Solomon, Abirami Rajasekaran, M. Yasar, Swathy Palani Kumar, Reeshma Nair Radhakrishnan, A. Korula, U. Kulkarni, E. Edison, P. Balasubramanian, Biju George, A. Abraham, Sanjeev Krishna, Vikram Mathews
          </td>
          <td>2025-11-04</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Non-small cell lung cancer (NSCLC) harboring activating mutations in the Epidermal Growth Factor Receptor (EGFR) has been effectively treated with EGFR tyrosine kinase inhibitors (TKIs). However, the clinical efficacy of these targeted therapies is invariably limited by the development of acquired resistance. While secondary mutations like T790M and bypass pathway activation are well-documented mechanisms, there is a growing appreciation for the profound role of epigenetic regulators, particularly microRNAs (miRNAs), in orchestrating the resistant phenotype. This review provides a comprehensive and detailed analysis of the multifaceted roles of miRNAs in the emergence and maintenance of EGFR-TKI resistance in NSCLC, including their regulation of alternative receptor tyrosine kinase signaling pathways, driving phenotypic plasticity, specifically the epithelial-mesenchymal transition (EMT) and the acquisition of cancer stem cell (CSC) characteristics, as well as dysregulating core cellular processes, such as apoptosis. We further examine the complex interplay within competing endogenous RNA (ceRNA) networks, where long non-coding RNAs and circular RNAs sequester miRNAs, thereby modulating the expression of resistance-associated genes. Finally, the potential of specific miRNAs as circulating biomarkers for monitoring treatment response and as therapeutic targets to overcome resistance is discussed. This review underscores the central role of miRNA-mediated gene regulation as a critical layer of complexity in EGFR-TKI resistance, highlighting a sophisticated network that governs the fate of cancer cells under therapeutic pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec44c69790c0243dc2e3a1211fa70c429a0a3017" target='_blank'>
              The Intricate Web of MicroRNAs in Modulating EGFR-TKI Resistance in Non-Small Cell Lung Cancer: A Comprehensive Review
              </a>
            </td>
          <td>
            Ying Xu, Jie Huang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Glioma, the most prevalent primary brain tumor, exhibits dysregulated ribosome biogenesis closely linked to malignant behavior. However, the role of ribosome biogenesis in glioma and prognosis remains incompletely understood. This study aimed to construct a molecular signature based on ribosome biogenesis-related genes to predict patient survival and therapeutic response in glioma. Methods Utilizing The Cancer Genome Atlas (TCGA) glioma cohort data, we constructed a ribosome biogenesis-related genes (RBRGs) signature using LASSO regression and multivariate Cox analyses, and subsequently validating its prognostic value in independent cohorts. We systematically evaluated the signature’s associations with clinicopathological features, tumor immunity, genomic instability, tumor stemness, and therapeutic sensitivity. The oncogenic role of the key gene UTP20 was experimentally validated in U87 and U251 glioma cell lines through MTS, colony formation, and transwell assays. Results We established a four-gene RBRGs signature (NOP10, UTP20, SHQ1, and PIH1D2). Elevated RBRGs score significantly correlated with shortened overall survival and adverse clinical characteristics, including advanced age, high WHO grade, IDH wild-type status, and absence of 1p/19q codeletion. A nomogram incorporating the RBRGs score demonstrated excellent predictive performance (C-index = 0.841). RBRGs-associated genes were enriched in immune regulatory pathways. The high-risk group exhibited increased infiltration of immunosuppressive cells (macrophages, myeloid-derived suppressor cells [MDSCs], and cancer-associated fibroblasts [CAFs]), upregulation of immunosuppressive checkpoints, and resistance to immunotherapy. Furthermore, the RBRGs signature correlated with genomic alterations, heterogeneity, tumor stemness, and therapeutic sensitivity. Crucially, UTP20 knockdown significantly suppressed glioma cell proliferation and invasion in vitro. Conclusion The RBRGs signature was successfully developed and validated as an independent prognostic biomarker and predictor of therapeutic response in glioma, highlighting its extensive association with tumor heterogeneity. Furthermore, this study identified UTP20 as a key oncogenic driver that facilitates glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9861e316b5d2145ed1a2316d77e1536ce5d1fb6d" target='_blank'>
              Ribosome biogenesis-related gene signature predicts prognosis and immune landscape in glioma and identifies UTP20 as a therapeutic target
              </a>
            </td>
          <td>
            Yadan Li, Xiaolong Tang, Wenhui Zhao, Xiuwen Si, Yanfang Cui, Yanbin Dong, Yongshuo Liu
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f4006d95d1215e3a398196888c62daed834131" target='_blank'>
              A Pan-Cancer Blueprint of Early Tumor Microenvironment Reprogramming
              </a>
            </td>
          <td>
            Bhavya Bhavya, Shivangi Agrawal, Ekta Pathak, Rajeev Mishra
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Anticancer drug resistance remains a significant challenge to the efficacy of cancer treatment, with DNA repair enzymes contributing to this resistance. We hypothesized that thymine DNA glycosylases (TDGs) may be involved in anticancer drug resistance given their dual function of DNA repair and demethylation as well as investigated their possible involvement in the induction of β-catenin in SNUC5 cells resistant to 5-fluorouracil (SNUC5/5-FUR) and oxaliplatin (SNUC5/OXTR). The expression of TDG and phospho-β-catenin increased in both resistant cell types when compared to that in SNUC5 cells. Moreover, knockdown of TDG significantly suppressed phospho-β-catenin expression in both resistant cell types, resulting in enhanced sensitivity to anticancer drugs. TDG binding to the β-catenin promoter was stronger in both resistant cell types than in SNUC5 cells, showing a decreased methylation pattern in the CpG islands of the β-catenin promoter. Furthermore, another DNA demethylase, ten-eleven translocation 1 (TET1), showed the same pattern as TDG in both resistant cell types. Additionally, TDG significantly interacted more with TET1 in both resistant cell types than in SNUC5 cells, enhancing binding to the same locus in the β-catenin promoter. These findings suggest that TDG may be a promising target molecule for overcoming drug resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def655ba0cc554974966a66e65dd66380f445b21" target='_blank'>
              Involvement of the DNA Demethylase Thymine DNA Glycosylase in Anticancer Drug Resistance
              </a>
            </td>
          <td>
            K. Kang, M. Piao, Herath Mudiyanselage Maheshika Madhuwanthi Senavirathna, Hye-Jin Boo, Sang-Pil Yoon, Joo Mi Yi, Jinny Park, J. Hyun
          </td>
          <td>2025-10-15</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genome instability, tumour-promoting inflammation, and immune escape are three distinct hallmarks of cancer. However, accumulating scientific and clinical evidence over the past decade have uncovered a multifaceted interplay of complex dynamic network of interactions between genome instability, the DNA damage response (DDR), and tumour immunogenicity. Fuelled by the clinical successes of immune checkpoint blockers (ICB), growing interest for immuno-oncology and recent cancer biology discoveries have allowed a better understanding of the underlying biology and clinical opportunities brought by this interplay—which is yet, still only in its infancy. The cooperative nature of tumour cell-intrinsic and -extrinsic mechanisms involved suggests that harnessing genomic instability in cancer does not only hamper cancer cells fitness but also stimulate the anti-tumour immune response, thereby paving the way to the development of DDR-based immunomodulatory therapeutic strategies applicable to a variety of molecular and histological cancer types. Here, we review the various aspects of this crosstalk between genome instability and tumour immunogenicity, including feedforward and feedback mechanisms affecting either side of this interplay, as well as the specific consequences of chromosomal instability. We further discuss emerging DDR-based predictive biomarkers of response to ICB therapies, and finally examine the latest clinical developments of therapeutic combinations that exploit the DDR–immunity interplay in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd59247f293c0351f5d4a4afa84e86e0d39c9e9" target='_blank'>
              Genome instability and crosstalk with the immune response
              </a>
            </td>
          <td>
            Roman M. Chabanon, F. Danlos, Kaïssa Ouali, Sophie Postel-Vinay
          </td>
          <td>2025-11-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Most mutant adenomatous polyposis coli (APC) gene produced truncated APC protein (Trunc-APC), which has been shown to function as an oncogene in colorectal cancer (CRC) pathogenesis; however, its role in modulating innate immune responses within tumor cells remains unexplored. We utilized CRISPR-Cas9 to knockout mutant APC and performed transcriptome sequencing across multiple CRC cell lines to investigate the immunomodulatory function of Trunc-APC. Subcellular fractionation, proteinase K protection assays, and immunofluorescence were employed to determine Trunc-APC subcellular localization. Protein interaction studies, ubiquitination assays, and aggregation analyses were conducted to elucidate Trunc-APC binding to MAVS and its impact on MAVS ubiquitination and RIG-I association. The effects of Trunc-APC deletion, alone or in combination with 5-azacytidine and trichostatin A, were evaluated on type I interferon activation, apoptosis, and tumor growth both in vitro and in vivo. We found that Trunc-APC partially localizes to the mitochondrial outer membrane and attenuates type I interferon signaling by binding to MAVS, suppressing its K63-linked polyubiquitination, and disrupting MAVS-RIG-I interactions. Deletion of Trunc-APC, particularly when combined with 5-azacytidine and trichostatin A, enhanced innate immune activation, promoted tumor cell apoptosis, and significantly inhibited CRC tumor growth both in vitro and in vivo. Our study reveals a previously unrecognized role of Trunc-APC in dampening tumor-intrinsic innate immunity and suggests that co-targeting Trunc-APC with epigenetic therapy may offer a promising strategy to enhance anti-tumor immune responses in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0e1308090a0906847a31cc6ff0f2170860b1e8" target='_blank'>
              Truncated APC impairs innate immune response by targeting MAVS on mitochondria in colorectal cancer
              </a>
            </td>
          <td>
            Si Li, Xin-Yi Wang, Jing Wang, Jinghua Cao, Yong-rui Lv, Dan Xie, Fengwei Wang
          </td>
          <td>2025-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary The von Hippel–Lindau Syndrome, an inherited condition that predisposes to clear cell renal cell carcinoma (ccRCC), affects developmental pathways of the kidney. Studying these early stages could help develop customized, targeted treatments. One approach is to reverse-engineer cancer cells into a stem cell-like state and then grow them into organ models. We attempted this with clear cell renal cell carcinoma (ccRCC) cells and adjacent normal (AN) healthy kidney cells using cell reprogramming. While the adjacent normal cells successfully reprogrammed to induce pluripotent stem cells (iPSCs), the cancer cells did not. We then restored the VHL gene—often missing in ccRCC. This reintroduced normal cell characteristics, but the cells upon reprogramming could not maintain a stable stem cell state. Our findings suggest that while restoring VHL is necessary for reprogramming, other genetic issues block ccRCC cells from fully returning to a healthy, stem-like state, indicating the importance of other, yet to be determined factors. Similarities between cancer cells and stem cells have been recognized, as common genes can influence both outcomes. In this study, we demonstrate that the VHL gene can be added to this group, as it is involved in both cancer development and stem cell regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e849d414c2da1724a3bcbca925e3a62cc9f656a" target='_blank'>
              VHL Gene Restoration Supports RCC Reprogramming to iPSCs but Does Not Ensure Line Stability
              </a>
            </td>
          <td>
            Zsuzsanna Lichner, Y. Shamshirgaran, Katarzyna Pieczonka, A. Jonebring, M. Kibschull, O. Shynlova, Jalna Meens, R. Kim, Laurie Ailles, B. Bilican, Ryan Hicks, Ian M. Rogers
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC) is the second most common form of non-melanoma skin cancer (NMSC), with a steadily increasing global incidence, especially in populations with prolonged ultraviolet (UV) exposure. Advanced or metastatic cSCC carries a poor prognosis, while the application of targeted therapies and immunotherapies remains in the exploratory stage, due to limited understanding of the underlying mechanisms of cSCC occurrence and progression. Epigenetic dysregulation plays important roles in the progression of cSCC. However, how multi-dimensional regulatory networks reshape the epigenetic landscape and contribute to dysregulated gene expression in cSCC remains unclear. In this study, we performed parallel RNA m6A sequencing, 850K DNA methylation arrays, whole transcriptome sequencing, and ATAC-seq chromatin accessibility profiling on samples from normal skin, actinic keratosis (AK), and cSCC. We analyzed the regulatory networks and pathways of epigenetic modifications on gene expression. We further explored the crosstalk of epigenetic regulatory networks by correlation analysis. By integrating single-cell RNA-seq data, we identified epigenetically upregulated candidate genes and confirmed their expression and functions with experimental methods. Our integrated multi-omics analysis provides a comprehensive and dynamic epigenetic map of cSCC progression. Further analysis revealed that DNA methylation and m6A modification jointly regulate gene expression through independent and synergistic ways. The identified epigenetically upregulated candidate genes IDO1, IFI6, and OAS2 were validated to be overexpressed in cSCC tissues and cell lines, and functional assays confirmed their potential key roles in regulating the processes of cell proliferation, migration and invasion in cSCC. By integrating multi-omics data, this study systematically highlights the multi-layered epigenetic alterations and regulatory mechanisms involved in cSCC development. This multi-stage, multi-omics, and multi-resolution integrated analysis provides a theoretical basis and new insights for future personalized treatment strategies for cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdeb7e8c78575b1d7e37855326ebd04eb68da2c1" target='_blank'>
              Multi-omics analysis reveals the crosstalk of epigenetic regulatory networks in cutaneous squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Ya-Zhou Sun, Dan-Dan Zou, Xin-Jie Li, Xiao-Li Wang, Jia-Hao Feng, Jia-Ying Liu, Ke He, Yu-Tong He, Xian-Xin Lai, Xin Li, Xuan Jiang, Tian-Shun Gao, Li He
          </td>
          <td>2025-11-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), the most aggressive and prevalent type of brain cancer, presents a formidable therapeutic challenge due to its high degree of drug resistance. Despite advances in chemotherapy, targeted therapy and immunotherapy, patient outcomes remain dismal, with a 5-year survival rate of only 7%. Resistance mechanisms are multifactorial, including the restrictive nature of the blood-brain barrier (BBB), tumor heterogeneity and adaptive responses within the tumor microenvironment (TME). The BBB limits drug delivery, while efflux transporters further reduce therapeutic efficacy. Additionally, the vast molecular genetic and cellular heterogeneity of a glioblastoma enables the survival of resistant subpopulations, such as glioblastoma stem cells, that evade treatment. Chemotherapy resistance in GBM, particularly to temozolomide, is driven by factors such as O6-methylguanine-DNA methyltransferase upregulation, defective mismatch repair, hypoxia-induced gene expression and activation of several signaling pathways, such as the NF-κB, Hippo and Wnt pathways. Targeted therapies have shown limited success due to activation of compensatory pathways and tumor plasticity, while immunotherapeutic approaches are hindered by an immunosuppressive TME. Recently identified resistance mechanisms, including exosomal transfer of noncoding RNAs and metabolic reprogramming, further complicate treatment. Future directions should emphasize overcoming these challenges through combination therapies, enhanced drug delivery systems and precision medicine approaches. Emerging strategies include targeting persister cells, leveraging metabolic vulnerabilities and integrating AI-driven drug discovery approaches and nanotechnology. Robust patient stratification and biomarker-driven interventions are critical for tailoring therapies and improving outcomes. The present review highlights the urgent need for innovative, multidisciplinary approaches to address the complexity of GBM resistance and advance therapeutic strategies for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7873b8c2ed7832370f4bef19a80cb17ac426667" target='_blank'>
              Drug resistance in glioblastoma: Challenges, mechanisms and therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Yash Durga, Raju Pusapati
          </td>
          <td>2025-11-06</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Cancer develops through a complex series of genetic and molecular changes. Modern sequencing technologies now allow scientists and physicians to study these changes in great detail, helping them understand why each person’s cancer behaves differently. By examining the entire genome, exome, or transcriptome of a tumor, researchers can uncover genetic alterations that guide personalized treatment decisions. This article explains how these sequencing methods, together with data analysis tools and artificial intelligence, help identify the best targeted and immune-based therapies for different cancer types. By linking genetic findings to approved drugs, the study aims to make precision cancer care more effective and accessible for patients while also guiding future research directions in oncology. Abstract The rapid evolution of sequencing technologies has profoundly advanced precision oncology. Whole-exome sequencing (WES), whole-genome sequencing (WGS), and whole-transcriptome sequencing (RNA-Seq) enable comprehensive characterization of tumor biology by detecting actionable mutations, gene fusions, splice variants, copy number alterations, and pathway dysregulation. These approaches also provide critical insights into biomarkers such as homologous recombination deficiency (HRD), tumor mutational burden (TMB), and microsatellite instability (MSI), which are increasingly essential for guiding therapeutic decisions. Importantly, comprehensive genomic profiling not only refines patient stratification for targeted therapies but also sheds light on tumor–immune interactions and the tumor microenvironment, paving the way for more effective immunotherapeutic combinations. WGS is considered the gold standard for detecting germline mutations and complex structural variants, while WES remains central for detecting somatic driver mutations that guide targeted therapies. RNA-Seq complements these methods by capturing gene expression dynamics, identifying clinically relevant fusions, and revealing mechanisms of resistance. Together with advances in bioinformatics and artificial intelligence, these tools translate molecular data into actionable strategies for patient care. This review integrates insights from WGS, WES, and RNA-Seq with an overview of FDA- and EMA-approved targeted therapies, organized by tumor type, and highlights the molecular signaling pathways that drive cancer development and treatment. By bridging genomic profiling with regulatory-approved therapies, we outline current advances and future perspectives in delivering personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dfbd87c701593d0e68f1b2d6aa4002ef2eff85" target='_blank'>
              Advances in Precision Oncology: From Molecular Profiling to Regulatory-Approved Targeted Therapies
              </a>
            </td>
          <td>
            P. Brlek, V. Škaro, Nenad Hrvatin, Luka Bulić, Ana Petrović, Petar Projić, Martina Smolić, Parth Shah, D. Primorac
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Alterations in chromatin-associated proteins contribute to the rewiring of transcriptional networks, promoting lymphomagenesis, disease progression, and resistance to therapies. Thus, epigenetic modulators offer a promising therapeutic avenue. Marginal zone lymphoma (MZL) is characterized by methylation changes, recurrent mutations in genes encoding chromatin remodelers and transcription regulators. Epigenetic reprogramming can sustain resistance to targeted agents. To identify novel therapeutic targets in MZL, we took advantage of the Structural Genomics Consortium (SGC) collection of chemical probes that inhibit or antagonize proteins involved in epigenetic signaling (www.thesgc.org). We exposed MZL cell lines, including models of secondary resistance to BTK, BCL2 and PI3K inhibitors, to 46 epigenetic compounds, spanning molecules targeting bromodomain (BRD), methyltransferases (MT), lysine demethylases (KDM) and histone acetyltransferases (HAT).



 Cells were exposed to the SGC library compounds. Cell viability was assessed after 72h. Compounds giving >40% reduction in cell proliferation versus control were considered active. Selected hits were validated in full dose-response assays.



 Two MZL cell lines (VL51, Karpas1718) and four derivatives with secondary resistance to BTK, PI3K, and BCL2 inhibitors (VL51-Ide, VL51-Copa, VL51-Ibru, Karpas1718-Ibru) were exposed to the SGC library compounds at 100 nM and 5 μM. Resistant derivatives were generally more sensitive to various epigenetic compounds than their parental cells. Only the BET inhibitor (i) JQ1 and the DOT1L-i SGC 0946 showed activity in the parental cells (both 100 nM and 5 μM, both models). Conversely, six compounds were active in resistant models at 100 nM: JQ1 and SGC 0946 (4/4), the PRMT5-i LLY 283 (2/4), and the BRPF2/TAF1-i BAY 299 (1/4). At 5 μM, 13 compounds were active: JQ1 and SGC 0946 in 4/4, 8 compounds in at least 2 models (targeting PRMT5, BRPF2/TAF1, G9a/GLP, JMJD3/UTX, BRD9, p300/CBP, EZH2) and 3 compounds in at least 1 model (PRMT7, p300/CBP, PRMT1/3/4/6/8). We then performed validation MTT experiments for 8 compounds (BAY 299, BI 9564, GSK 2801, I-BRD9, LLY 238, SGC 0946, SGC 3027, UNC 0642) using 6 concentrations of each molecule in all models. Higher activity in resistant derivatives was largely confirmed for the Karpas1718 model, and not in VL51, in which similar activity was seen in parental and resistant cells. The most active compounds were BAY 299 (BRPF2/TAF1; median IC50 6 nM), SGC 3027 (PRMT7; 229 nM) and SGC 0946 (DOT1L, 380 nM), exhibiting dose-dependent cytotoxicity across parental and resistant MZL models, followed by the others, all with median IC50s lower than or equal to 2 μM.



 Our findings reveal novel potential epigenetic vulnerabilities (PRMT7, DOT1L, and BRPF2/TAF1) in lymphoma models, also with acquired resistance to targeted therapies, besides confirming PRMT5 and BET bromodomains as valid targets. Compounds targeting these proteins should be further explored as single agents and in combinations.



 Alberto Jesus. Arribas, Luca Licenziato, Eleonora Cannas, Luca Aresu, Afua Adjeiwaa. Mensah, Francesco Bertoni. Epigenetic vulnerabilities in marginal zone lymphoma [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbf4328ab2c7b5c9c31fe8bd744d96696423fe" target='_blank'>
              Abstract C027: Epigenetic vulnerabilities in marginal zone lymphoma
              </a>
            </td>
          <td>
            A. Arribas, L. Licenziato, Eleonora Cannas, Luca Aresu, A. Mensah, Francesco Bertoni
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/967791c9e248760bca34f5e1970d8fd507b14a7e" target='_blank'>
              AP-1 mediated chromatin changes govern alveolar type 2 cell transition in lung injury-repair
              </a>
            </td>
          <td>
            Anne M Lynch, Tala Noun, Sujuan Yang, Tieling Zhou, Scott E Evans, H. Kadara, Jichao Chen
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Epigenetic modifications parallel multiple memory processes, but evidence that the epigenetic makeup of a single site can guide learnt behaviors has so far been lacking. Here, we developed CRISPR-based epigenetic editing tools to address this question in a cell type-specific, locus-restricted and temporally controllable manner in the adult mouse brain. Focusing on learning-induced neuronal populations, we provide a proof-of-principle that site-specific epigenetic dynamics are causally implicated in memory expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a43e190d0186c8da05a2c5b2c34a962bcd551a" target='_blank'>
              Cell-type- and locus-specific epigenetic editing of memory expression
              </a>
            </td>
          <td>
            Davide M. Coda, Lisa Watt, Liliane Glauser, M. Batiuk, Allison M. Burns, Cora L. Stahl, Lok Y Wong, Johannes Gräff
          </td>
          <td>2024-08-18</td>
          <td>Nature Genetics</td>
          <td>2</td>
          <td>20</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, Céline Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers. Methods An in silico study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i. Limma was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content. Results Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission. Conclusion The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc94606b4c77ff4f31472a0aaf128e44eef19aa7" target='_blank'>
              Mutants of p53 sustain tumor growth under mechanical compression
              </a>
            </td>
          <td>
            M. Di-Luoffo, S. Arcucci, N. Therville, Tristan Marty, R. D’Angelo, Maria Chaouki, B. Thibault, Pascal Swider, Pauline Assemat, Morgan Delarue, J. Guillermet-Guibert
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is extensively studied, yet its origins and progression remain unclear. A key question is why tumors of the same type vary in aggressiveness and treatment response. In colorectal cancer (CRC), the third most common cancer, this variability led to the identification of molecular subtypes (CMS). However, the tumor microenvironment remains poorly understood and may be crucial for understanding carcinogenesis and drug resistance. A promising approach is analyzing cell interactions through ligand-receptor expression. This study used bioinformatics to examine CRC in different anatomical locations, identify diagnostic and prognostic biomarkers, and propose targeted drugs. Expression data were obtained from the TCGA-COAD database. All samples were filtered based on the tumor’s region of origin and purity. RNA-seq expression analysis was then conducted to assess molecular differences according to tumor location and purity, identifying region-specific ligands and receptors using the secretome list as a reference. Once these differences were identified, an interactome was constructed to depict cell interactions within the tumor microenvironment. The most relevant genes were then evaluated for their prognostic potential through survival analysis, and their susceptibility to pharmacological modulation was assessed to identify potential new drug candidates for CRC treatment. The integration of secretome data and the construction of the interactome proved to be a valuable approach for detecting novel biomarkers specific to right- and left-sided CRC. Through this approach, FGFR4, FLT1, and WNT5A were identified as key biomarkers involved in tumor carcinogenesis, modulating distinct processes in each region, such as fibroblast recruitment and cell division. Based on these biomarkers, Dovitinib and Nintedanib were predicted as potential therapeutic agents, as they target multiple identified markers. This study highlights FGFR4, FLT1, and WNT5A as key diagnostic and therapeutic biomarkers for CRC, with their relevance varying based on the tumor’s site of origin. Leveraging these findings, we propose Dovitinib and Nintedanib as promising targeted therapies for CRC. These insights can enhance current treatment strategies and pave the way for future in vivo and in vitro studies, driving progress in CRC research and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86a3d9f8b9f730ccd2b60a1bbaaeddf8c7f0fad" target='_blank'>
              Integrating the secretome and interactome to identify novel biomarkers and therapeutic targets in colorectal cancer
              </a>
            </td>
          <td>
            Gabriel Henrique Caxali, Mirian Carolini Esgoti Aal, Catherine Wesselka Garcia Osvaldo, Jakeline Santos Oliveira, L. Bidinotto, Robson Francisco Carvalho, F. K. Delella
          </td>
          <td>2025-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Multiple myeloma (MM) is characterized by the clonal proliferation of plasma cells in the bone marrow. Although the precise molecular mechanisms differentiating men and women in MM are not fully understood, uncovering these differences is crucial for improving personalized therapeutic approaches. Here, we show sex-specific dysregulation of exosomal non-coding RNAs (ncRNAs) in MM. We conducted an in-depth analysis of dysregulated ncRNAs in male and female patients, as well as MM cell lines, revealing distinct expression signatures across multiple clinical contexts, including newly diagnosed, relapse, progression, Hyperdiploid, non-Hyperdiploid, and treatment exposure. Our findings highlight the pivotal roles of lncRNAs and miRNAs in MM pathogenesis, detecting alterations in enriched pathways that influence key biological processes such as cellular proliferation, apoptosis, and gene regulation. We established a panel of ncRNAs with distinct sex-specific expression patterns, significant effects on mRNA regulation, and involvement in MM-associated biological pathways. Our results demonstrate that exosomes provide enhanced analytical resolution for detecting non-coding RNAs, enabling more sensitive and precise identification of transcriptomic alterations. These results suggest that sex-specific dysregulation of ncRNAs may contribute to differences in MM progression and therapy response. Ultimately, this study underscores the importance of exosomal ncRNA profiling in designing sex-tailored therapeutic strategies targeting dysregulated ncRNAs, paving the way for personalized medicine in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3076fd0b79a566fa62815457cb49df2d96414995" target='_blank'>
              Sex-specific dysregulation of exosomal non-coding RNAs drives multiple myeloma progression
              </a>
            </td>
          <td>
            S. Maleknia, Sanam Rezaei Benam, Greg Ahmann, Rafael Fonseca, Diane F Jelinek, Reza Shahbazi
          </td>
          <td>2025-10-16</td>
          <td>Blood Cancer Journal</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The search for novel tumor-specific markers and therapeutic targets is driving the development of more effective and personalized treatment strategies for cancer patients. This article focuses on investigating a promising new source of biomarkers—readthrough transcripts, or downstream-of-gene (DoG) transcripts. These transcripts are extended products of gene transcription that continue into intergenic regions and can overlap neighboring genes, sometimes giving rise to cis-splicing of adjacent gene (cis-SAGe) transcripts. Recent studies suggest that besides frequently being a “transcriptional noise”, DoG transcripts can perform regulatory functions, serve as a source of novel protein products, and act as prognostic markers of patient survival across various cancers. This article aims to investigate the regulatory mechanisms and functional significance of readthrough transcripts in tumors, to identify currently known tumor-specific variants with potential utility as cancer biomarkers or therapeutic targets, and to evaluate the most suitable approaches for their detection. The knowledge gained through this research may provide a foundation for the development of diagnostic test systems and the design of new anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/005f6133a4fcac56d4aa30b3f8401ba10e194f84" target='_blank'>
              Regulatory Mechanisms and Functional Roles of Readthrough Transcripts in Tumorigenesis
              </a>
            </td>
          <td>
            Aleksander Modestov, G. Zakharova, E. Poddubskaya, Anton Buzdin
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Abstract KRAS, once considered undruggable, has become actionable across specific alleles, with KRAS-G12C inhibitors now approved and next-generation approaches—including pan-KRAS/pan-RAS inhibitors, targeted degraders and RNA-based strategies—progressing rapidly. However, clinical benefit remains limited due to the frequent emergence of resistance. Escape mechanisms include on-target secondary mutations, pathway reactivation, epithelial–mesenchymal transition, lineage plasticity and metabolic rewiring within an immunosuppressive tumour microenvironment. Emerging evidence supports rational combination strategies, including parallel inhibition of epidermal growth factor receptor, protein tyrosine phosphatase non-receptor type 11 or SOS1 and vertical blockade of the mitogen-activated protein kinase–extracellular signal-regulated kinase or phosphatidylinositol 3-kinase–mechanistic target of rapamycin cascades; immunotherapies such as checkpoint blockade, T-cell receptor (TCR)-T cells, bispecific T-cell engagers or cytokine-armed oncolytic viruses; metabolic interventions targeting macropinocytosis or autophagy; as well as radiotherapy. Such combination therapies can transform primarily cytostatic effects into more durable antitumour responses, although with potential toxicity constraints. Precision approaches that integrate multiomics profiling with longitudinal circulating tumour DNA analysis enable biomarker-guided patient selection (eg, based on STK11 and KEAP1 comutations) and support therapeutic adaptations, including sequencing strategies and intermittent dosing. Thus, network-level KRAS interception combined with biomarker-driven, clonal evolution-informed trial design offers a path towards sustained control of KRAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44096ce38e05c7f3967ab9cd3f14f4e387fdb46b" target='_blank'>
              KRAS-targeted therapies in cancer: novel approaches and overcoming resistance
              </a>
            </td>
          <td>
            Daolin Tang, Guido Kroemer, Rui Kang
          </td>
          <td>2025-11-01</td>
          <td>BMJ Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) activates pro-inflammatory cGAS/STING signaling, positioning it as a biomarker for combining immune checkpoint blockade (ICB) and PARP inhibition (PARPi). However, the consequences of HRD on the immune landscape across cancers remain unclear. Here, we applied a pan-cancer HRD classifier to >10,000 tumors from The Cancer Genome Atlas and uncovered striking heterogeneity in immune activity. Compared to HR-proficient tumors, HRD tumors showed elevated inflammation in breast, ovarian, and endometrial cancers. These tumors exhibited robust activation of innate and adaptive immune pathways (IFN, NF-κB) and transcriptional hallmarks of senescence, angiogenesis, and adenosine signaling. In contrast, lung, head and neck, and melanoma HRD tumors displayed suppressed inflammation and evidence of immune escape through large-scale loss-of-heterozygosity (LOH) at IFNA/B, STING, and other loci. These tumors also frequently presented HLA LOH and oncogene amplifications, suggesting selection under immune pressure and replication stress. Together, our study resolves HRD tumors into two immune archetypes, immune-inflamed and immune-evasive, linked to chromosomal instability and lineage, informing biomarker-driven evaluation of immune checkpoint blockade/PARPi combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Next-generation sequencing of canine spontaneous cancer is a powerful approach in both comparative oncology and veterinary genomics. We encountered a unique case with concurrent mammary tumors. Using whole-genome sequencing (WGS), we profiled the tumor-specific landscape of somatic mutations across multiple tumor subtypes, providing unprecedented evidence within an identical genetic background. Of the seven mammary gland tumors (MGTs) isolated, two were diagnosed as benign and five as malignant. High-quality WGS (34.5X average sequencing depth, ≥ 20X coverage across 94.9% of the genome) on tumors and a blood sample revealed missense mutations in human breast cancer-related genes such as BRCA2 and TP53. Furthermore, we found evidence of canine-specific somatic mutations depending on the tumor subtypes, including HECTD4 in malignant tumors and NIPBL in epithelial-derived malignant tumors. This unique case study provides novel insights into the genomic heterogeneity, clonal evolution, and subtype-specific pathogenesis of naturally occurring canine MGTs. Despite some inherent limitations and potential for individual-specific variation, our results emphasize and guide future large-scale, longitudinal studies to further elucidate the clinical and biological significance of key somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa805022d59e7e8ebaa159b214972178e6ca2e8d" target='_blank'>
              Comparative genomic analysis of multiple mammary tumors from a single dog: whole-genome sequencing study
              </a>
            </td>
          <td>
            Keon Kim, Tae-Hoon Shin, Sin-Wook Park, Sang-Ik Park, Yoon-Jung Do, Woong-Bin Ro, Chang-Min Lee
          </td>
          <td>2025-10-30</td>
          <td>Irish Veterinary Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Thyroid carcinomas are phenotypically heterogeneous malignancies. Advances in molecular and cellular technologies have revealed genetic, epigenetic, and nongenetic factors underlying this heterogeneity. Our study aimed to assess the impact of single and combined treatments with anticancer agents (Carboplatin, Doxorubicin, Paclitaxel, Avastin), natural compounds (Quercetin), and epigenetic modulators (suberoylanilide hydroxamic acid and 5-Azacytidine) on the expression of long noncoding RNAs, methylation regulators, and functional features in the human thyroid cancer cell line K1. Methods: Treated and untreated K1 cells were used throughout experiments to evaluate the drug-induced cytotoxicity, apoptosis, cell cycle distribution, cytokine release, gene expression, and global DNA methylation levels. Results: Some single- and combined-drug treatments modulated both cell cycle progression and apoptotic events, demonstrating anti-tumor activity of the tested compounds. Gene expression analysis showed treatment-specific regulation of target genes and lncRNAs, including both upregulation and downregulation across different drug combinations. All treatments resulted in increased global DNA methylation levels compared to the untreated controls. Several combinations significantly upregulated DNMT1 and DNMT3B, while concomitantly decreased EZH2 levels. Conclusions: These coordinated epigenetic changes highlight the therapeutic potential of combining epigenetic modulators with chemotherapeutic agents, suggesting a strategy to prevent or reverse treatment resistance and improve outcomes in thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1eba8f910f8930ce4647663ca6adba36af18" target='_blank'>
              Epigenetic Remodeling in Thyroid Cancer: New Dimensions of Targeted Therapy Through lncRNA Modulation
              </a>
            </td>
          <td>
            A. Albulescu, A. Fudulu, M. Mihăilă, I. Iancu, Adriana Pleșa, M. Bostan, A. Botezatu, L. Brasoveanu, C. Hotnog
          </td>
          <td>2025-10-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Regorafenib, a multikinase inhibitor, is widely used to treat hepatocellular carcinoma. However, chemoresistance poses a significant challenge to its long-term efficacy. This study conducted a genome-wide CRISPR/Cas9 knockout screen in liver cancer cell lines to identify key regulators of regorafenib resistance and elucidate the underlying molecular mechanisms. The screen identified ARHGAP35 as a critical negative regulator of regorafenib resistance. ARHGAP35 depletion conferred resistance in HepG2 and Huh7 cells, while regorafenib-resistant variants (HepG2-R and Huh7-R) exhibited decreased ARHGAP35 expression. Reintroducing ARHGAP35 restored drug sensitivity. Further analysis revealed that reduced ARHGAP35 expression facilitated epithelial-mesenchymal transition (EMT) by activating the RhoA signaling pathway. Notably, RhoA inhibition reversed EMT and restored regorafenib sensitivity. These findings highlight ARHGAP35 as a key modulator of regorafenib resistance through RhoA suppression, offering potential therapeutic targets to combat chemoresistance in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b3a4892777e73e2cf92f82b25659fdd83d51c4" target='_blank'>
              A genome-wide screen identified ARHGAP35 as a regulator of regorafenib resistance in liver cancer.
              </a>
            </td>
          <td>
            Kun Chen, Miaomiao Zhang, Yuexin Liu, Zhengnan Dong, Yaoji Liang, Jinzhang Zeng, Jie Liu
          </td>
          <td>2025-10-16</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary The majority of non-small cell lung cancer patients present with advanced disease, whereby survival is less than 2 years. We report a woman with metastatic non-small cell lung cancer who survived 12 years, benefiting from serial tissue analyses that applied genomic and functional platforms to select therapies. Each recurrence manifested distinct biologic features that responded to drugs selected by genotypic and phenotypic analyses. Following laboratory-directed chemotherapy and subsequent immunotherapy, the patient was found positive for a BRAF V600E mutation and responded to Dabrafenib plus Trametinib. Subsequent progression revealed an EGFR del19 mutation, not present in initial genomic analysis, as the previously identified BRAF mutation disappeared. Serial interrogations identified clonal expansions with distinct phenotypes. Each intervention selected highly adapted resistant subpopulations that revealed new therapeutic vulnerabilities. The case represents the application of serial tissue analyses to identify therapies, offering hope for more effective personalized cancer strategies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb66a2f50798ff6be9a24ac5af17d9ad46fa3951" target='_blank'>
              Serial Functional and Genomic Analyses Illuminate Clonal Evolution in Metastatic NSCLC with 12-Year Survival
              </a>
            </td>
          <td>
            Vikrant S. Bakaya, Sabina A. Schneider, Tracy Nguyen, Derrick Phu, Lucas A. Alvarez, S. S. Evans, P. Bernard, F. Francisco, Adam J. Nagourney, Luisa Torres, John Henry, P. D’Amora, R. Nagourney
          </td>
          <td>2025-11-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Y. Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            , Si-Jia He, Xiao-yu Jing, , Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Functional loss of RB1 is a common genetic alteration in triple-negative breast cancer (TNBC) and is associated with poor response to targeted therapies, including CDK4/6 inhibitors. In this study, we perform an unbiased drug screen and identify that co-targeting distinct cell cycle processes such as DNA repair and mitosis induce synthetic lethality selectively in RB-deficient models. While RB loss promotes replication stress and mitotic dysregulation, the selective lethality observed with these combinations arises from an alternate mechanism. Under RB-deficient conditions, cells undergo rapid apoptosis in response to cellular stress induced by cell cycle inhibition. This pro-apoptotic response is further augmented by using a pharmacological agent, birinapant that targets XIAP, which is an endogenous inhibitor of the apoptotic pathway. Birinapant in combination with CHK1 or AURKA inhibitors results in selective cell killing in RB-deficient TNBC models and yields durable disease control via apoptosis in vivo. In conclusion, RB loss in TNBC displays an enhanced vulnerability to pro-apoptotic signaling that can enable the effective implementation of new targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/480b2f05a58b5c00a2a188f4a7abf7234badb233" target='_blank'>
              RB loss sensitizes triple-negative breast cancer to apoptosis induced by cellular stress
              </a>
            </td>
          <td>
            A. Witkiewicz, Subrahmanya Anirudh Kaligotla Venkata, Erik S. Knudsen, Vishnu Kumarasamy
          </td>
          <td>2025-11-24</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite the initial responses to the tyrosine kinase inhibitor (TKI) for cancer therapy, many patients often relapse with no curative regimens available. Further, the ability to target therapeutic agents to cancer cells with appropriate doses remains challenging in the clinic, especially for leukemia. Here, we show that naïve CML cells are dynamically heterogeneous in colony formation. Larger clones expand while smaller ones diminish and eventually disappear. Compared to resistant cells, parental populations, including CD44+ stem cells, form a greater number of larger, solid spheroids. Upregulation of fat mass and obesity associated protein (FTO), an RNA N6-methyladenosine demethylase, and stem cell markers (e.g., CD44, CD133, CD25) is more obvious in resistant cells compared to parental cells. FTO inhibitors (e.g., CS1, FB23-2) appreciably impair the growth of resistant cells either alone or in combination with nilotinib. FTO protein expression is unexpectedly upregulated by CS1 or FB23-2 treatment in multiple leukemia cell lines. We then constructed RNA nanoparticles encapsulating FTO siRNAs and conjugated with anti-CD133 RNA aptamers. We showed that, compared to negative control, these nanoparticles were taken up much more efficiently by resistant cells that highly express CD133. Treatment with the CD133-guided FTO siRNA nanoparticles efficiently silenced FTO expression in resistant cells, which leads to a significant reduction in their colony and spheroid formation. These findings offer new insights into cancer drug resistance and advance the application of RNA nanotechnology for treating leukemia. The research provides a foundation for developing novel, targeted therapies for resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610e20fd12fcf52179a62018bf638a0c7e862c87" target='_blank'>
              CD133-Guided RNA Nanoparticle Delivery of FTO siRNA Impairs Leukemia Resistance to Tyrosine Kinase Inhibitor Therapy
              </a>
            </td>
          <td>
            Huiqin Bian, Changli Zhou, Hiroaki Koyama, Wen Gao, Sicheng Bian, Tao Cheng, Xiaonan Han, William Tse, Shujun Liu, Alexander Miron
          </td>
          <td>2025-10-15</td>
          <td>RNA NanoMed</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554ff6de17aca3fc14055b3c5a6203467aaf1fe3" target='_blank'>
              A gap-free telomere-to-telomere pig reference genome provides insight in centromere evolution and breed-specific selection
              </a>
            </td>
          <td>
            Dong Li, Yulong Wang, Tiantian Yuan, Xiang Li, Yu Quan, Wansheng Liu, K. Schachtschneider, Lawrence B Schook, R. Crooijmans, M. Groenen, M. Derks, Taiyong Yu
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract Disclosure: S. Gadad: None. M. Sedano: None. Triple-negative breast cancer (TNBC) accounts for approximately 12-20% of all new breast cancer cases diagnosed annually and is known to be associated with poorer clinical outcomes compared to other molecular subtypes of breast cancer. Patients with TNBC typically present with larger tumors and higher tumor grades, resulting in decreased overall and recurrence-free survival rates compared to individuals with non-TNBC tumors. The strategies involved in treating TNBC include chemotherapy agents such as doxorubicin, an anthracycline chemotherapeutic agent; however, its efficacy may vary due to the development of resistance. The specific mechanisms behind the constantly evolving resistance of TNBC to chemotherapy remain unclear, emphasizing the need for further investigation. In our study, we utilized two TNBC cell lines engineered with gradually increasing concentrations of doxorubicin to develop resistance. We conducted integrative genomic analyses to uncover the cellular mechanisms involved, identified altered molecular pathways relevant to cancer, and ascertained the factors contributing to the phenotypic characteristics of doxorubicin-resistant TNBC. Our findings indicate that many evolutionarily young transcripts with coding or limited coding potential are differentially expressed compared to non-resistant TNBC cells. Furthermore, analyses of adjacent genomic regions containing aberrantly expressed genes revealed that these genes are located in areas implicated in cellular plasticity, allowing cancer cells to adapt to the presence of doxorubicin, potentially through transcriptional reprogramming. We are also investigating the genetic and epigenetic alterations that may have facilitated the evolution of TNBC cells into doxorubicin-resistant variants. Additionally, we are assessing the clinical implications of the identified factors by studying them in vivo using mouse models with genetically engineered chemo-sensitive and -resistant cells, linking the outcomes to changes observed in patient samples associated with clinical prognosis. By elucidating the specific mechanisms contributing to the development of doxorubicin resistance in TNBC, we aim to uncover the underlying processes driving resistance and recurrence, potentially paving the way for novel therapeutic strategies. S.S.G. is a CPRIT Scholar in Cancer Research. S.S.G. was supported by a First-time faculty recruitment award from the Cancer Prevention and Research Institute of Texas (CPRIT; RR170020). S.S.G. is supported by the The American Cancer Society (RSG-22-170-01-RMC), NIH 1R16GM149497 grants and CPRIT-TREC (RP230420). Presentation: Sunday, July 13, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b4373af8394b05d1e7e23058cecf5a8e12738f" target='_blank'>
              SUN-210 The role of transcriptional reprogramming and genomic alterations in the transition from chemo-sensitive to chemoresistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Shrikanth S. Gadad, Melina J. Sedano
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background The identification and characterisation of somatic cancer driver mutations in the non-coding genome remains challenging. Objective To broadly characterise non-coding driver mutations for pancreatic ductal adenocarcinoma (PDAC). Design Using mutation calls from whole-genome sequence data in PDACs and genome-scale maps of accessible gene regulatory regions in normal-derived and tumour-derived pancreatic samples, we analysed enrichment of non-coding mutations in gene regulatory regions relevant to normal-derived and tumour-derived pancreatic contexts. Functional follow-up of potential driver mutations was performed using chromatin interaction analyses, massively parallel reporter assays (MPRA) and targeted analysis of selected non-coding somatic mutations (NCSMs). Results We first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3614 NCSMs. Chromatin interaction analysis identified 416 potential target genes and MPRA revealed 178 NCSMs impacting reporter activity (19.45% of those tested). Targeted luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near ZFP36L2 and CDKN2A. For the former, CRISPR interference identified ZFP36L2 as a target gene (16.0–24.0% reduced expression, p=0.023–0.0047), and growth inhibition after overexpression of ZFP36L2 (4.1–14.1-fold reduction, p=6.0×10–4 − 3.2×10–3) implicates a possible tumour suppressor function. Conclusion Our integrative approach provides a catalogue of potential non-coding driver mutations and nominates ZFP36L2 as a novel PDAC driver gene with a likely tumour suppressor function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4adc9cdafe8f3e335c7d27583752292d933404cf" target='_blank'>
              Large-scale multiomic analysis identifies non-coding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal B Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Sudipto Das, Maura J. O’Neill, Courtney D Dill, Andrew D. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E Connelly, Jianxin Shi, Efsun Arda, Jason W Hoskins, L. Amundadottir
          </td>
          <td>2025-10-08</td>
          <td>Gut</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumour cells to flexibly adapt to fluctuating microenvironmental conditions, sustain uncontrolled proliferation, and acquire resistance to conventional therapies. Tumour metabolism is not limited to the classical Warburg effect but encompasses a dynamic interplay between glycolysis, oxidative phosphorylation (OXPHOS), fatty acid metabolism, and amino acid utilisation, each fine-tuned according to tissue context, tumour type, and stage of progression. Central regulators such as hypoxia-inducible factor-1 (HIF-1), MYC, p53, peroxisome proliferator-activated receptors (PPARs), oestrogen receptor (ER), and sterol regulatory element-binding proteins (SREBPs) orchestrate these pathways, linking nutrient availability to oncogenic signalling and transcriptional control. This review synthesises current evidence on these interconnected metabolic circuits and critically evaluates existing controversies, such as the dual reliance on glycolysis and OXPHOS, metabolic plasticity under therapeutic pressure, and the role of stromal-tumor metabolic crosstalk. Beyond established pathways, emerging areas are transforming our understanding of tumour metabolism. Single-cell metabolic profiling and spatial metabolomics reveal profound intratumoral heterogeneity, while immunometabolism highlights the bidirectional influence of cancer cells and immune cells within the tumour microenvironment (TME). Epigenetic regulation, driven by metabolites that serve as cofactors for chromatin-modifying enzymes, further integrates metabolic states with transcriptional reprogramming and therapy response. Translationally, targeting metabolic dependencies remains challenging; promising therapeutic opportunities are being developed, including inhibitors of lactate transporters, fatty acid oxidation, and glutamine metabolism. This review integrates mechanistic insights with translational perspectives, providing conceptual models, summary tables, and schematic illustrations to clarify complex networks and highlight clinically relevant opportunities. By mapping the evolving landscape of cancer metabolism, we aim to illuminate both the challenges and the therapeutic potential of exploiting metabolic vulnerabilities in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e702d799086ec509201036b103e5a842964f431" target='_blank'>
              Molecular Mediators of Metabolic Reprogramming in Cancer: Mechanisms, Regulatory Networks, and Therapeutic Strategies.
              </a>
            </td>
          <td>
            S. Ahuja, S. Zaheer
          </td>
          <td>2025-10-08</td>
          <td>Immunology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Ovarian cancer is a highly lethal malignancy, with treatment efficacy frequently compromised by the development of cisplatin resistance. This study investigates the molecular mechanisms driving cisplatin resistance, focusing on interferon-induced protein with tetratricopeptide repeats 1 (IFIT1), a key interferon-stimulated gene. Through comprehensive transcriptome profiling, we systematically compared gene expression patterns between cisplatin-sensitive and cisplatin-resistant ovarian cancer cells, identifying a consistent upregulation of IFIT1 in resistant cells. Functional assays provided evidence that IFIT1 plays a central role in mediating cisplatin resistance. Mechanistically, IFIT1 was found to be associated with activation of inflammatory signaling pathways, suggesting its involvement in resistance-related immune regulation. Clinically, elevated IFIT1 expression significantly correlated with an unfavorable prognosis in ovarian cancer patients. Moreover, we demonstrated that hypoxia-inducible factor-1 alpha transcriptionally regulates IFIT1, linking its expression to hypoxia response. Overall, these findings identify IFIT1 as a key driver of cisplatin resistance in ovarian cancer, shedding light on its clinical significance and the regulatory networks underpinning therapy resistance. IFIT1 holds promise both as a prognostic indicator and as a target for therapeutic intervention. However, its clinical translation will depend on rigorous validation and in-depth exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3e993b5a5faff584d9a192dda2fe082dc88120" target='_blank'>
              Overcoming cisplatin resistance in ovarian cancer by targeting IFIT1-mediated inflammation responses
              </a>
            </td>
          <td>
            Min Hu, Xuejiao Zhao, Xiuming Li, Bin Liu, Chunluan Yuan
          </td>
          <td>2025-11-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ca54f63f6feb36492a0d66c69a169994ddd7a" target='_blank'>
              Leukemia Risk Factor ARID5B Coordinates HDAC-Mediated Transcriptional Repression
              </a>
            </td>
          <td>
            A. P. Kutschat, Fabian Frommelt, Brianda L Santini, Sophie Müller, P. Batty, G. Karbon, Giulio Superti-Furga, Davide Seruggia
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b270922941e97bc91f75cdb1ab9aa350f3916bd" target='_blank'>
              Mitotic bypass and endocycling promote cancer cell survival after genotoxic chemotherapy
              </a>
            </td>
          <td>
            Kevin Truskowski, Louis T A Rolle, George Butler, Margaret E Yang, Kenneth J. Pienta, S. Amend
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="It is unclear how CRISPR editing outcomes vary across the genome and whether undesirable events such as structural variants (SVs) are predictable or preventable. To define a genome-wide map of editability, we performed whole-genome sequencing on 1875 budding yeast clones edited across 16 chromosomes by CRISPR-Cas9 and donor-templated repair. We found that unintended edits, including short indels and SVs, were enriched in specific genomic and sequence contexts. We developed a predictive model, SCORE (System for CRISPR Outcome and Risk Evaluation), which revealed 4.8% of the genome as SV prone, consisting of 562 SV hotspots. Donor repair-enhancing strategies suppressed SV formation in regions with moderate, but not high, predicted risk. Applying SCORE to the Sc2.0 synthetic yeast genome revealed a markedly altered SV landscape due to the removal of endogenous repetitive elements and the insertion of loxP sites. Our study provides the genome-scale map of SV hotspots after CRISPR editing and predictive and experimental tools to mitigate their formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c353e50628925b1279c5062797bfebe0f4ebe5a" target='_blank'>
              The editable landscape of the yeast genome reveals hotspots of structural variant formation
              </a>
            </td>
          <td>
            Shengdi Li, Sibylle C. Vonesch, Kevin R. Roy, Chelsea Szu Tu, Friederike Steudle, Michelle Nguyen, Cosimo Jann, Lars M. Steinmetz
          </td>
          <td>2025-10-31</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Drug resistance remains a significant challenge in achieving successful cancer treatment, often leading to disease recurrence and reduced patient survival. While traditional tissue biopsies provide valuable insights into tumor biology, they are invasive, infrequent, and may fail to capture the full complexity of tumor heterogeneity and dynamic molecular changes. In contrast, liquid biopsy has emerged as a minimally invasive, real-time approach for monitoring tumor evolution through the analysis of circulating biomarkers. Among these biomarkers, circular RNAs (circRNAs) - a distinct class of non-coding RNAs characterized by covalently closed-loop structures - have gained attention due to their remarkable stability, abundance in body fluids, and functional involvement in gene regulation. Increasing evidence supports the role of circRNAs in mediating drug resistance through mechanisms such as inhibition of apoptosis, epithelial-mesenchymal transition, autophagy, and drug efflux, largely via interactions with microRNAs or proteins. Advanced detection methods, including quantitative reverse transcription polymerase chain reaction, droplet digital polymerase chain reaction, and RNA sequencing, combined with computational tools, enable precise profiling of circRNAs in plasma or exosomes. CircRNA-based liquid biopsies offer a dynamic, non-invasive strategy for early detection of therapeutic resistance and may guide personalized treatment decisions. This review highlights the technological advancements, biological relevance, and clinical promise of circRNAs as circulating biomarkers, emphasizing their potential in precision oncology and future collaborative translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df152da0476d5f6e5b6795f9a52718cb169ae7e7" target='_blank'>
              Circular RNA-based liquid biopsy: a promising approach for monitoring drug resistance in cancer
              </a>
            </td>
          <td>
            Desh Deepak Singh, Dharmendra Kumar Yadav, Dongyun Shin
          </td>
          <td>2025-10-30</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA double-strand break (DSB) clusters are a hallmark of high-linear energy transfer (high-LET) radiation and are associated with pronounced biological effects, including reduced cell survival and elevated genomic instability. Our previous work in Chinese hamster cells, engineered with variably designed clusters of I-SceI recognition sites, integrated at multiple genomic locations, revealed that DSB clusters suppress classical non-homologous end-joining (c-NHEJ) and induce chromosomal translocations that ultimately increase cell lethality. Here, we extend this line of investigation to human cell lines and generate DSB clusters using alternative approaches that do not require prior genetic manipulation of the test cell lines. We employ CRISPR/Cas9-technology to generate DSB clusters of specific design at a selected genomic locus and examine their consequences on locus integrity. We target Exon 3 of the human HPRT (hHPRT) gene and introduce single DSBs or DSB clusters of varying numbers and inter-DSB distances. Alterations at the locus reflecting hHPRT gene inactivation, are quantified as mutations causing resistance to 6-thioguanine (6TG). Our results show that DSB clusters are markedly more potent inducers of mutations than single DSBs and that DSBs spaced within ~ 600 base pairs synergize in mutation induction. Mechanistic analyses using small-molecule inhibitors and engineered gene knockout cell lines reveal that the increased mutagenicity of clustered DSBs is primarily mediated by DNA end resection and PARP1-dependent alternative end-joining (alt-EJ) pathways. These findings reinforce the biological relevance of DSB clusters as a severe form of complex DNA damage and provide mechanistic insights into high-LET radiation-induced increased cell killing and genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82859607e3d79ce6936e2d753c4009b872b5c5d9" target='_blank'>
              CRISPR/Cas9 generated DSB clusters mimic complex lesions induced by high-LET radiation and shift repair from c-NHEJ to mutagenic repair pathways
              </a>
            </td>
          <td>
            E. Mladenov, M. Kallies, M. Stuschke, Eleni Gkika, G. Iliakis
          </td>
          <td>2025-10-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy. ALKBH8 is a tRNA methyltransferase, specifically modifying the wobble uridine (U34) in tRNAs. Here, the authors discover that ALKBH8 is a direct transcriptional target of β-catenin, and that mediates intestinal tumorigenesis through regulation of the translation of KRAS in a codon-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Homologous recombination (HR) and non-homologous end joining (NHEJ) are two main repair pathways of DNA double-strand breaks (DSBs). The regulation of repair pathway choice is crucial for maintaining genomic stability and preventing carcinogenesis. Consequently, there is increasing interest in elucidating the molecular mechanisms that govern DSB repair pathway selection. Here, we identify chromosome 8 open reading frame 33 (C8orf33) as a novel regulator of DSB repair choice. We show that C8orf33 is a nuclear protein localized predominantly to the nucleolus and is recruited to DSB sites within both nuclear and nucleolar regions. We demonstrate that C8orf33 promotes the recruitment of 53BP1, thus channeling DSB repair toward NHEJ. Consequently, C8orf33 inhibits DNA end resection and counteracts the recruitment of HR factors, BRCA1 and RAD51, to DSB sites. Mechanistically, chromatin profiling analysis reveals that C8orf33 antagonizes the chromatin association of KAT8 acetyltransferase at DSB sites leading to reduced histone 4 lysine 16 acetylation (H4K16ac) levels. Accordingly, the loss of C8orf33 enhances KAT8 chromatin binding that increases H4K16ac levels. This promotes the recruitment of HR factors while suppressing the accumulation of NHEJ factors at DSB sites, thereby favoring HR over NHEJ. Additionally, we demonstrate that the elevated HR activity in C8orf33-deficient cells causes genomic instability, as evidenced by accelerated loss of ribosomal DNA repeats and increases cell death. Collectively, our findings establish C8orf33 as a critical regulator of DSB repair pathway choice, safeguarding genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1d5cfef9dd20c22a9cd88671f578821b21baa" target='_blank'>
              C8orf33 dictates DNA double-strand break repair choice by modulating KAT8-mediated H4K16 acetylation
              </a>
            </td>
          <td>
            Laila A. Bishara, Enas R. Abu-Zhayia, Marian Nicola, N. Ayoub
          </td>
          <td>2025-11-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The integration of chimeric antigen receptor (CAR) therapies with precision medicine holds potential to impact the treatment landscape for acute myeloid leukemia (AML). Genetic mutations play a role in the efficacy of CAR-T and CAR-NK cells, influencing their crucial role in determining the effectiveness of these cells, as well as their proliferation, persistence, resistance, and safety. This review examines how mutations in FLT3, DNMT3A, NPM1, TP53, TET2, gene fusions involving RUNX1 and KMT2A and other key genes modulate CAR-based immunotherapies, highlighting both vulnerabilities and resistance mechanisms. Recent findings demonstrate that mutations in genes such as DNMT3A and NPM1 enhance antigen expression, thereby improving CAR targeting. In contrast, mutations in TP53 drive immune escape and resistance to therapy. Understanding these mutation-specific effects is essential for tailoring CAR therapies to individual patients, optimizing efficacy while minimizing toxicity. By leveraging genomic profiling and personalized engineering approaches, CAR therapies can be refined to overcome resistance and enhance precision in AML treatment. Future research should focus on integrating multiomic data to develop mutation-adapted CAR strategies, ensuring that patients receive the most effective and personalized immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fed72fbe58eba50628755381ec963c392e48c54" target='_blank'>
              Precision medicine with car cells in acute myeloid leukemia: where are we?
              </a>
            </td>
          <td>
            Larissa C. Zanetti, Victória Tomaz, Ingrid Ferreira de Souza, P. Campregher, Nelson Hamerschlak, L. Kerbauy
          </td>
          <td>2025-11-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Recurrent GBM poses significant treatment challenges due to limited therapeutic options and reliance on primary tumor molecular profiles, often overlooking treatment-induced molecular and phenotypic changes in recurrent tumors. This study investigates genomic, DNA methylation, and transcriptomic differences between a primary GBM (HF3016) and its matched recurrent tumor post-temozolomide and radiotherapy (HF3016R), using neurosphere cultures and patient-derived xenografts (PDXs) to assess radiotherapy (RT) response. Genomic DNA was analyzed via whole-genome sequencing, whole-exome sequencing, and EPIC DNA methylation arrays. Bulk RNA sequencing was performed on HF3016 and HF3016R cancer stem cell (CSC) cultures (n=4) and orthotopic PDX models (n=3). Differential gene expression was analyzed using NOISeq (q=0.95, fold change >2) on TMM-normalized data, with pathway enrichment via Metascape 3.1. Tumor growth and RT sensitivity were evaluated in NCRNU athymic nude mice implanted intracranially with HF3016 or HF3016R cells (3x10^5 cells/mouse, N=9-10 per group), receiving 5 Gy RT daily for 3 days or no treatment. Survival was assessed using Kaplan-Meier curves and Log-rank tests. No genomic driver alterations (CDK4, EGFR, MYC ecDNA amplification, PTEN, p53 mutations) or DNA methylation mesenchymal subgroup changes were observed between primary and recurrent models. However, a transcriptional shift from proneural to mesenchymal subtype occurred in recurrent models, with enriched epithelial-mesenchymal transition and interferon response pathways in HF3016R, while nervous system development pathways were enriched in HF3016. HF3016R PDXs exhibited faster growth (p=0.002) and greater RT resistance (p=0.0021 for HF3016, p=0.061 for HF3016R) compared to HF3016. These findings highlight the importance of considering treatment-induced molecular changes in recurrent GBM for optimizing therapeutic strategies. This study reveals that while genomic and DNA methylation profiles remain stable between primary and recurrent GBM models, significant transcriptional reprogramming occurs. The enrichment of epithelial-mesenchymal transition and interferon response pathways in recurrent models is associated with their aggressive growth and increased radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec6b57a9c052c7471268a4341bec91fcb4af93" target='_blank'>
              EXTH-73. Identifying therapeutic targets for recurrent glioblastoma using patient-derived longitudinal models
              </a>
            </td>
          <td>
            Indrani Datta, O. Nuga, A. Berezovsky, Laila M. Poisson, Ana deCarvalho
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, Sameh Youssef, Alain de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) poses a significant global health challenge, yet immune checkpoint blockade (ICB) therapy benefits only a small subset of patients with mismatch repair-deficient (dMMR) or microsatellite instability-high (MSI-H) tumours. Through analyses of public single-cell and spatial transcriptomic datasets, primary mouse cell sorting and adoptive transfer experiments, flow cytometry, multiplex immunofluorescence, immunohistochemistry, and coimmunoprecipitation, we revealed that sentrin-specific protease 7 (SENP7) promotes regulatory B-cell (Breg) differentiation and inhibits senescence by activating the expression of the NAD-dependent protein deacetylase sirtuin-1 (SIRT1) via deSUMOylation, thereby enhancing the expression of genes such as interleukin-10 (IL-10). Notably, targeting SENP7 in B cells improved the antitumour efficacy of anti-PD-1 therapy. These findings suggest that inhibiting SENP7 may offer a promising strategy to sensitize immunologically “cold” tumours to immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf8df97840e103aaf23b0df7a5e6b5ffd3a725b" target='_blank'>
              A SENP7-SIRT1-IL-10 Axis Driven by DeSUMOylation Promotes Breg Differentiation and Immune Evasion in Colorectal Cancer
              </a>
            </td>
          <td>
            Yuhan Liao, Xinghua Zhuo, Yuan Huang, Huimeng Xu, Zhe Hao, Lanhui Huang, Haoxuan Zheng, Jun Zhou
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Fms-like tyrosine kinase 3 (FLT3), a class III receptor tyrosine kinase essential for hematopoiesis, is a well-established oncogenic driver in acute myeloid leukemia (AML). Canonical internal tandem duplications (ITD) and tyrosine kinase domain (TKD) mutations inform prognosis and guide targeted therapy. Recent evidence highlights FLT3 as a critical oncogenic hub in acute lymphoblastic leukemia (ALL), where its alterations extend beyond ITD/TKD mutations to include non-canonical mutations with only partially explored functional implications. Moreover, recently discovered regulatory mechanisms, mostly acting on the FLT3 locus, drive FLT3 overexpression in ALL, including transcriptional regulation by rearranged ZNF384, epigenetic modifications, novel circular-RNA URAD::FLT3 fusions, and 13q12.2 deletions leading to enhancer hijacking and topologically associated domain (TAD)-boundary disruptions. The impact of these alterations on leukemogenesis and the possibility to target them in ALL subtypes is discussed here. Data from the Functional Omics Resource of Acute Lymphoblastic Leukemia (FORALL) across B- and T-ALL cell line subtypes drug screening, and from preclinical and clinical evidence reveals a variable efficacy in FLT3-mutated and FLT3-overexpressing ALL subtypes, supporting a molecularly guided treatment approach. Building on the success of FLT3 inhibitors in mutated AML and in light of the emerging results in patients lacking FLT3-ITD and in FLT3-like AML cases, presenting with a gene expression pattern similar to FLT3-mutated ones despite the absence of mutations, we discuss their potential in ALL and we consider novel therapeutic strategies, including new FLT3 inhibitors, antibody-based approaches, FLT3 CAR-T therapy, and synergistic drug combinations, such as FLT3 and BCL2 inhibition. These new insights reviewed here may redefine FLT3 as a pan-leukemic target, with ALL-specific activation mechanisms offering unique therapeutic windows. The implementation of FLT3 expression profiling and full-coding mutation screening in ALL (and in AML) diagnostics could unlock precision medicine approaches. By bridging the AML experience with ALL innovations, this review outlines a roadmap for FLT3-targeted therapies and combination strategies, underscoring the urgency of biomarker-driven clinical trials to optimize FLT3-directed interventions in acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b724e5e3a02c4639c7dca6522dee54c5f66974d4" target='_blank'>
              Novel molecular mechanisms of FLT3 deregulation: from the acute myeloid leukemia experience to therapeutic insights in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            F. Lo Schiavo, C. Salvesi, M. Jandoubi, F. Pirini, J. Garbetta, G. Martinelli, G. Simonetti, A. Ferrari
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: The elucidation of the human genome has fundamentally transformed our understanding of disease etiology, positioning gene mutations as central players in susceptibility, pathogenesis, and progression across a vast spectrum of human disorders. The translation of this genetic knowledge into targeted therapeutic strategies represents the cornerstone of precision medicine, heralding a new era in clinical management.
Objective: This narrative review aims to synthesize the current landscape of how specific gene mutations influence disease development and progression, and to explore how these discoveries are shaping the development and application of novel, targeted therapeutic approaches.
Main Discussion Points: The review thematically explores the paradigm of oncogenic mutations in driving targeted cancer therapies, such as tyrosine kinase inhibitors and PARP inhibitors, while also examining the role of germline mutations in hereditary cancer syndromes. It further expands into non-oncological domains, including cardiology and neurology, highlighting the development of treatments like PCSK9 inhibitors and antisense oligonucleotides. The discussion also covers groundbreaking advanced therapies, such as gene replacement and gene editing, using examples from spinal muscular atrophy and sickle cell disease. Critical analysis is provided on the challenges of therapeutic resistance, variants of uncertain significance (VUS), and issues of health equity.
Conclusion: The collective evidence firmly establishes that targeting specific gene mutations is a powerful and transformative therapeutic strategy. However, realizing the full potential of precision medicine requires overcoming significant hurdles, including resistance mechanisms, the high cost of therapies, and a lack of diversity in genetic research. Future efforts must focus on innovative trial designs, long-term safety monitoring, and equitable implementation to ensure these advances benefit all patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382be60380435d45208a6a43c1baabce83169b3e" target='_blank'>
              NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES: A NARRATIVE REVIEW.NARRATIVE REVIEW ON THE IMPACT OF GENE MUTATIONS IN DISEASE SUSCEPTIBILITY, PROGRESSION, AND TARGETED THERAPEUTIC APPROACHES
              </a>
            </td>
          <td>
            Noor Ul, Ain Khaliq¹, Muhammad Yasir Yasin², Talha Waheed³, Atif Maqsood⁴, Ayesha Maalik⁵, Musa Khan⁶
          </td>
          <td>2025-10-04</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM), as one of the most common and aggressive primary brain tumors, pose significant challenges in diagnosis and treatment, highlighting the urgent need for the identification of novel biomarkers, NFKBIE, which may play a critical role in tumor progression, immune modulation, and therapeutic response. We investigated the role of NFKBIE in GBM through both bioinformatics analysis and experimental validation. Initially, bioinformatics analysis based on public databases revealed significant differential expression of NFKBIE across various cancer types, particularly in GBM, where elevated expression was correlated with poor patient prognosis, including overall survival, disease-specific survival, and progression-free survival. Furthermore, genetic alterations in NFKBIE, such as copy number variations (CNVs) and tumor mutational burden (TMB), were significantly associated with tumor progression. In the experimental validation phase, we utilized small interfering RNA (siRNA) technology to silence NFKBIE expression in GBM cell lines, including U87 and T98G. Functional assays were then performed to assess the impact of NFKBIE knockdown. Cell proliferation assays, including CCK-8 and clonogenic assays, demonstrated that silencing NFKBIE significantly inhibited the proliferation of GBM cells. Migration and invasion assays, including wound healing and Transwell assays, further confirmed that NFKBIE knockdown markedly suppressed the migratory and invasive abilities of GBM cells. Additionally, apoptosis assays, such as TUNEL staining, revealed that NFKBIE silencing significantly promoted apoptosis in GBM cells. Western blot analysis confirmed the changes in the expression of key markers associated with proliferation, stemness, migration, invasion, and apoptosis. Furthermore, in a mouse xenograft model, NFKBIE knockdown significantly slowed tumor growth. Histological analysis of the excised tumors confirmed that reduced NFKBIE expression was closely associated with the inhibition of tumor growth. Notably, NFKBIE silencing also led to the downregulation of components of the Hedgehog signaling pathway, suggesting that NFKBIE may regulate tumor progression through modulation of this critical pathway. In conclusion, high NFKBIE expression is strongly associated with GBM progression and poor prognosis, and targeting its expression may offer a promising therapeutic strategy for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567a01090acb470e17dfe52d5931751c0e81fc63" target='_blank'>
              NFKBIE as a prognostic biomarker and therapeutic target for GBM: role in Hedgehog signaling activation
              </a>
            </td>
          <td>
            Feng Mo, Yang Li, Ce Dong, Lijun Yang, Xiaoli Ma, Baogen Pan, Z. Fan
          </td>
          <td>2025-10-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a52f82067df3b6fe8b772cfcc0c82d13aa9a9b1" target='_blank'>
              Integrative transcriptomic analysis of non-small cell lung cancer: Microenvironment, circulation, and metastasis
              </a>
            </td>
          <td>
            Adnan Khosravi, Sharareh Seifi, Babak Salimi, Maryam Mabani, Parsa Rostami, Masoumeh Nomani
          </td>
          <td>2025-11-11</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Summary Humans exhibit significant phenotypic differences from other great apes, yet pinpointing the underlying genetic changes has been limited by incomplete reference genomes and a reliance on single assemblies to represent a species. We aligned 20 telomere-to-telomere (T2T) assemblies spanning great ape divergence and variation to define 1,596 Consensus HAQERs (consensus human ancestor quickly evolved regions), regions that diverged rapidly between the human-chimpanzee ancestor and an ancestral node of modern humans. Unlike prior HAQER sets, Consensus HAQERs incorporate population variation, reducing the likelihood of intraspecies variation appearing as interspecies divergence. Consensus HAQERs exhibit signatures of elevated mutation rates, ancient positive selection, bivalent regulatory function, are enriched in disease-linked loci, and often emerged in previously inaccessible repetitive DNA. Through multiplex, single-cell enhancer assays, we identify HAQERs as active enhancers in the developing brain and cardiomyocytes, highlighting their potential contributions to human-specific gene regulation across multiple tissues. Highlights ● Telomere-to-telomere alignments of diverse human and great ape genomes identify 1,596 Consensus HAQERs, regions of rapid sequence divergence separating human ancestors from other great apes. ● Consensus HAQERs exhibit signatures of both elevated mutation rates and ancient positive selection. ● HAQERs are enriched in bivalent regulatory elements and disease-linked loci. ● Multiplex, single-cell gene regulatory assays identify HAQERs as enhancers in the developing heart and brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00bd3c45b8187e63a71fea1f33c5572d84f0153e" target='_blank'>
              Intraspecific sequence variation and complete genomes refine the identification of rapidly evolved regions in humans
              </a>
            </td>
          <td>
            Yanting Luo, Riley J. Mangan, Seth Weaver, Sarah A. Zhao, Federica Mosti, Michael C. Thomas, Ravi Karra, D. Silver, M. Kellis, Craig B. Lowe
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer treatment is a core challenge in clinical oncology. Despite the continuous advancement of chemotherapy, targeted therapy, and immunotherapy, the emergence of tumor drug resistance remains a key factor leading to treatment failure and worsening patient prognosis. Clinical data indicate that approximately 90% of treatment failures in patients with advanced cancer are directly related to drug resistance. Therefore, systematically analyzing the mechanisms of drug resistance and developing targeted strategies to address them are crucial for improving the efficacy of cancer treatment. This article systematically expounds on the main mechanisms of tumor drug resistance from the perspectives of tumor cell biological characteristics, tumor microenvironment regulation, and epigenetic modifications. These include reduced intracellular drug accumulation, mutations or aberrant expression of drug targets, inactivation of apoptotic pathways, cancer stem cell (CSC)-mediated resistance, and the regulatory role of immunosuppressive cells and cytokines in the tumor microenvironment. Furthermore, based on clinical practice, this article summarizes currently used strategies, such as drug development targeting resistance-associated molecules, chemotherapy regimen optimization and combination therapy, synergistic application of immunotherapy and conventional therapies, personalized treatment plan development, and the clinical translation of resistance monitoring technologies. Research has shown that the development of tumor drug resistance is the result of multiple factors and pathways, and a single strategy is unlikely to effectively reverse drug resistance. However, multi-target, multi-modal combination therapy based on resistance mechanisms, combined with dynamic resistance monitoring technology, can significantly improve treatment outcomes for patients with drug-resistant tumors. This article aims to provide a theoretical basis for clinicians to formulate tumor treatment plans and to guide future research related to drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f859514bae2da8d810219f47530fe8e0b51bb170" target='_blank'>
              A Review of the Mechanisms of Tumor Drug Resistance and Clinical Strategies
              </a>
            </td>
          <td>
            Wenqing Tie
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Public Health and Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to genotoxic therapies remains a major contributor to tumor recurrence and treatment failure, yet the mechanisms by which cancer cells escape these therapies through DNA damage response (DDR) activation are not fully understood. Here, we identify a DDR regulatory pathway in which glycogen synthase kinase 3 β (GSK3B), a multifunctional serine/threonine kinase, governs DNA double-strand break (DSB) repair pathway choice by phosphorylating 53BP1 at threonine 334 (T334) — a site distinct from canonical ATM targets. This phosphorylation event disrupts 53BP1’s interaction with nonhomologous end joining (NHEJ) effectors PTIP and RIF1, promoting their dissociation from DSBs and inhibiting 53BP1-driven NHEJ. Simultaneously, T334 phosphorylation facilitates the recruitment of CtIP and RPA32 for DNA end resection and promotes homologous recombination (HR) by enabling BRCA1 and RAD51 loading. Notably, the phospho-deficient T334A mutant of 53BP1, unlike 53BP1 loss, accumulates aberrantly at DSBs along with PTIP/RIF1, impairs end resection, and suppresses HR activity. Importantly, both genetic and pharmacologic disruption of the GSK3B–53BP1 axis sensitizes tumors to PARP inhibitors (PARPi) independently of BRCA1 status. Together, these findings reveal a GSK3B-dependent mechanism that regulates DSB repair pathway choice and provide a rationale for targeting this axis to enhance PARPi efficacy in solid tumors regardless of BRCA1 status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7947f267430312493c565ecaa4ae8dfb1f98a221" target='_blank'>
              GSK3B directs DNA repair choice and determines tumor response to PARP1 inhibition independent of BRCA1
              </a>
            </td>
          <td>
            H. Allam, Scarlett Acklin-Wehnert, Ratan Sadhukhan, M. Patra, F. Xia
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, Mark R. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Iron metabolism has emerged as a critical regulator of cancer biology, with mounting evidence linking iron dysregulation to tumor initiation, progression, and resistance mechanisms. Osteosarcoma (OS) is the most common primary bone malignancy and a leading cause of cancer-related death in children and young adults; recent studies have identified profound alterations in iron homeostasis at both cellular and microenvironmental levels in OS. These include increased iron uptake, disrupted storage and export, and a reliance on iron-dependent metabolic pathways that promote proliferation, metastasis, and immune evasion. Despite advances in surgical and chemotherapeutic approaches, survival outcomes in OS have stagnated, underscoring the need for novel therapeutic strategies. Targeting iron metabolism represents a promising avenue, with strategies such as iron chelation, transferring receptor inhibition, ferroptosis induction, and modulation of ferritinophagy, showing preclinical efficacy. In this review, we provide an updated and integrated overview of the multifaceted role of iron in OS pathogenesis, dissect emerging therapeutic approaches aimed at disrupting iron regulatory networks, and highlight innovative delivery platforms including nanomedicine. By integrating current insights on iron metabolism with the molecular complexity of OS, we present a comprehensive perspective, while acknowledging that the limited clinical translatability of current findings still hinders progress toward clinical application. A deeper understanding of iron-driven mechanisms may guide future studies toward the development of safe and effective iron-targeted therapies for OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fa870def36afdfc1d96b28031d0b94b16663f1" target='_blank'>
              Iron-Related Metabolic Targets in the Treatment of Osteosarcoma: Research Progress and Prospects
              </a>
            </td>
          <td>
            Arianna Buglione, Magda Gioia, Federica Sinibaldi, Stefano Marini, Chiara Ciaccio
          </td>
          <td>2025-11-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beea8538ea0fee6e0373855f4448e35dd9857e52" target='_blank'>
              From Genes to Personalized Cancer Care: A Systematic Review of Toxicity-Associated Genetic Variants in Solid Tumor Treatments
              </a>
            </td>
          <td>
            A. Gonzalez-Hernandez, A. Escamilla-Sanchez, E. Perez-Ruiz, A. Rios, C. A. Frecha, F. Vaca-Paniagua, I. Barragan, S. Perdomo, A. Rueda-Domínguez, J. Oliver
          </td>
          <td>2025-10-24</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical cancer (CC) remains among the top causes of death for women worldwide, especially in low-income countries, where screening strategies are less widespread. Treatment strategies are mainly based on DNA-damaging agents, though resistance mechanisms still pose a substantial challenge. Among the cellular components that mediate treatment resistance, long non-coding RNAs (lncRNAs) stand out because of their broad regulatory effects. They are involved in virtually all drug resistance mechanisms, such as drug efflux, DNA repair, evasion of cell death, and aberrant epigenetic modifications. Although resistance mechanisms are fundamentally similar in most cancers, the underlying regulatory networks vary substantially. Here, we review the literature for lncRNAs involved in treatment resistance mechanisms in general, and then focus on lncRNAs that mediate resistance in CC. We found a broad area of opportunity in lncRNA research in resistant CC, as the lncRNAs involved are still to be described. These master regulators are promising candidates for response markers and therapeutic targets. May this compilation serve as the basis for further descriptions of the regulatory roles of lncRNA in CC treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38374134a894f6b5db043d7b741d396bd485cc0d" target='_blank'>
              Long noncoding RNAs involved in therapeutic response: implications for cervical cancer drug resistance
              </a>
            </td>
          <td>
            Samuel Trujano-Camacho, Carlos Contreras-Romero, Verónica García-Castillo, David Sánchez-Marín, Mercedes Olvera-Valencia, M. Rodríguez-Dorantes, O. Peralta-Zaragoza, David Cantú- De León, E. López-Urrutia, C. Pérez-Plasencia
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Herein, this report presents the case of a 48-year-old female with a history of breast cancer (2013) and subsequent high-grade epithelial ovarian cancer (2020), illustrating the complex evolution of therapeutic resistance in BRCA1-mutated malignancies. Following initial response to paclitaxel-carboplatin chemotherapy and complete surgical debulking, the patient experienced multiple disease relapses transitioning from platinum-sensitive to platinum-resistant states. Comprehensive molecular profiling via Tempus xF+ next-generation sequencing revealed a pathogenic BRCA1 mutation alongside a secondary BRCA1 reversion mutation, conferring partial restoration of homologous recombination repair and resistance to both platinum-based chemotherapy and PARP inhibitors. Subsequent therapies, including pemetrexed and liposomal irinotecan, were employed with limited success. This case underscores the dynamic molecular evolution of recurrent ovarian cancer under therapeutic pressure and highlights the critical role of serial genomic profiling in guiding personalized treatment strategies. Emerging approaches targeting alternative DNA repair mechanisms and novel antibody–drug conjugates may hold promise for overcoming resistance in BRCA-mutated, therapy-refractory ovarian cancer.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63bac7b92dd6ce55648d1d8d363cc2b4553b6628" target='_blank'>
              Recurrent ovarian cancer with BRCA reversion: a case study and comprehensive literature review
              </a>
            </td>
          <td>
            Sadanand M Karandikar, Himanshi Joon, Shailesh Puntambekar, Amit Nehra, Vivek Chaudhari
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Reproduction, Contraception, Obstetrics and Gynecology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8f7f2274db91d56455b33e92f98f5f8697e415" target='_blank'>
              Resistance of first-line targeted drugs in hepatocellular carcinoma: the epigenetic regulation mechanisms.
              </a>
            </td>
          <td>
            Ding Qi, Yongqing Qin, Haidong Zhu, Yong Li, Shisong Han
          </td>
          <td>2025-11-26</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39534227ca18263af64802892a94b5211231b6b" target='_blank'>
              Failure of DNA repair leads to the tumorigenic transformation of a regenerating blastema in Drosophila imaginal discs
              </a>
            </td>
          <td>
            Tanroop Kaur, Ada Repiso, Ying Liu, Paula Doria, Joan Urgell, Luis Rodriguez-Escudero, Katerina Karkali, Yanhui Hu, N. Perrimon, E. Martín-Blanco
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b823debf44456586f41e4e06b190055e610d9" target='_blank'>
              Genetic Ablation and Multi-Omics Profiling Reveal CEP55 as a Key Driver of Tumorigenesis in Diverse Cancer Models
              </a>
            </td>
          <td>
            Behnam Rashidieh, Yuanhao Yang, A. Bain, Phillip Theron, Catharina Vosloo, Parinaz Ahangar, Brydie Bowden, S. Tria, Ben B. Wang, Sandra Isenmann, Nicholas J. Westra van Holthe, Zherui Xiong, Hani Vu, Sowmya Sharma, A. Sengal, J. A. López, John Finnie, Antonella Papa, Pascal H. G. Duijf, Quan H. Nguyen, Pirjo M. Apaja, K. Khanna
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Modeling cancer recurrence and associated conditions such as cancer cell dormancy remains a significant challenge in cancer research. We developed a novel in vitro model based on platinum-resistant tumor cells treated with a platinum drug, in which cells progress through all stages of recurrence within 30–40 days post-exposure. Our results demonstrate that treatment of platinum-resistant colorectal cancer cells with oxaliplatin enriches the population with quiescent, dormant cells arrested in the G0/G1 phase. These cells exhibit increased autophagy, elevated expression of stem cell markers, reduced reactive oxygen species (ROS) levels, and heightened resistance to chemotherapy. A key advantage of our model is the high survival rate of residual cells, enabling the maintenance of a large cell population. This facilitates the application of conventional research techniques that are often limited in other recurrence models due to the small size of residual populations. Overall, this model provides opportunities for a wide range of further research for studying specific recurrence- and dormancy-associated signaling pathways and for identifying novel therapeutic targets for preventing tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4b53e10213b4d15495530a1d5bcc0ce84af24bb" target='_blank'>
              An in vitro tumor recurrence model based on platinum-resistant colon cancer cells as a research tool for studying cancer cell dormancy
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, M. Igotti
          </td>
          <td>2025-10-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background/Aim: Cancer remains a leading cause of mortality globally, driven by complex molecular mechanisms and characterized by significant biological heterogeneity across cancer types. We aimed to discover mitogen-activated protein kinases (MAPKs) family members as both biomarkers and therapeutic targets in different cancer types. MAPKs are key signaling molecules regulating cell proliferation, differentiation, stress response, and apoptosis. Dysregulation of MAPK pathways has been implicated in the onset and progression of multiple cancers, contributing to tumor growth, metastasis, and therapeutic resistance. Given their diverse roles across cancer types, systematic analysis of MAPK gene expression, mutations, and interactions with tumor microenvironment is essential. Materials and Methods: The present study undertakes a comprehensive transcriptomic analysis of MAPKs in nine major cancer types using RNA-Seq datasets from The Cancer Genome Atlas (TCGA). RNA-Seq data were analyzed to identify differentially expressed MAPKs across nine cancer types using DESeq2, NOIseq, and limma. Significant genes (adjusted p<0.05) were subjected to GO and KEGG enrichment (EnrichR), mutational profiling (cBioPortal), and Kaplan-Meier survival analysis. Tumor immune infiltration was further assessed using TIMER2.0 to explore immune–gene interactions. Results: Through rigorous differential expression analysis, we identified key MAPKs that are significantly altered in liver hepatocellular carcinoma (LIHC) and lung adenocarcinoma (LUAD). Specifically, six MAPKs (MAPK3, MAPK7, MAPK9, MAPK10, MAPK12, and MAPK13) were found to be differentially expressed in LIHC, while MAPK6 emerged as the sole significant candidate in LUAD. Functional enrichment and pathway analysis revealed that these MAPKs are involved in critical oncogenic pathways, including MAPK-mediated transcriptional regulation and stress-activated signaling cascades. Mutational profiling and survival analysis further validated the prognostic significance of these genes, with several MAPKs showing strong associations with reduced patient survival. Tumor immune infiltration analysis indicated potential roles of these MAPKs in modulating immune responses within the tumor microenvironment. Also, MAPKs identified in this study are structurally related, suggesting that targeting them collectively may enhance therapeutic efficacy and overcome resistance mechanisms. Conclusion: Our integrated approach underscores the value of MAPK family members as both biomarkers and therapeutic targets in LIHC and LUAD. This study contributes important insights into MAPK-related oncogenic processes and supports the development of targeted therapies under the framework of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b14065bbb6005ef268d03b15d61fb336481cae" target='_blank'>
              Investigating the Roles of MAPKs as Precision Oncology Targets: A Multi-cancer Expression and Survival Study
              </a>
            </td>
          <td>
            Md Nayab Sulaimani, Prithvi Singh, Mohammad Umar Saeed, Sobia Zaidi, Afzal Hussain, M. Alajmi, Anas Shamsi, Ravins Dohare, M. Hassan
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The onset of acquired resistance limits the efficacy of KRASG12C inhibitors in lung cancer patients. Experiments in preclinical model systems and retrospective analyses of patients treated with these inhibitors have both suggested that the activation of DNA repair pathways and G2/M kinases is common in lung cancers with acquired resistance to KRASG12C inhibitors. Here, we identified a shared vulnerability to the CDK12/13 selective inhibitor SR-4835 in several KRASG12C-mutant cell lines with acquired resistance to sotorasib. In pairs of KRASG12C-mutant parental and sotorasib-resistant cell lines, CDK12/13 inhibition suppressed DNA repair gene expression and induced mitotic arrest, which was exacerbated in sotorasib-resistant cells by failure to activate the G2/M checkpoint and suppression of an oncogenic isoform of TP73. Combined treatment with both sotorasib and SR-4835 yielded an additive response and suppressed the development of acquired resistance to either inhibitor. The inhibitor combination caused additive G1 and G2/M arrest, and the increased sensitivity of emerging drug-resistant cell populations to SR-4835 extended the duration of response over single-agent treatment in cell culture and in mouse models. These results support combining KRAS and CDK12/13 inhibitors to extend the duration of response in KRAS-mutant lung cancer. Patients with acquired resistance to sotorasib may benefit from follow-up monotherapy with CDK12/13 inhibitors, the first of which recently entered clinical trials. Targeting CDK12/13 thus offers a promising strategy to overcome or prevent resistance to KRAS inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5df2549b13618f9401530fb07b8c41d871bb9c6" target='_blank'>
              Targeting CDK12/13 Drives Mitotic Arrest to Overcome Resistance to KRASG12C Inhibitors.
              </a>
            </td>
          <td>
            Yaakov E. Stern, Pompom Ghosh, John Peroza, Hitendra S Solanki, Denis Imbody, Liznair Bridenstine, Thiyagamurthy Pandurangan, Sylvia M Frydman, Kathryn N Nickens, Hannah L Walker-Mimms, Kruthi Suvarna, Neelkamal Chaudhary, Andrii Monastyrskyi, Derek Duckett, E. Haura
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis, creating new combinations of the genome while also introducing mutations, and influencing GC content. Here we report direct detection of recombination events using highly accurate long-read sequencing from testis tissue of 16 individuals across six primate species and three human sperm samples. Based on methylation patterns, we classify sequencing reads as originating from either somatic or germline cells. We identify 2881 crossovers, 2314 simple gene conversions, and 555 complex events, and analyze their chromosomal distribution. Crossovers are more telomeric, showing stronger concordance with recombination maps than gene conversions. Human samples align with a double-strand break map, whereas other species differ, consistent with variation in PRDM9-directed breaks, although the recombination process is otherwise conserved. Gene conversion tracts are short and of similar length across species (mean 22–95 bp), implying that most non-crossover events are undetectable. We observe GC-biased gene conversion for both single and multiple-SNV events, including sites flanking crossovers. We infer longer gene conversion tracts associated with crossovers (318–688 bp) than with non-crossovers. Highly accurate long-read sequencing combined with methylation-based classification of reads to specific cell types provides a powerful way of studying recombination events in single individuals of any mammalian species. Homologous recombination rearranges genetic information during meiosis to generate new combinations in the genome. Here, the authors report direct, single-individual detection of recombination in primates and humans using long-read sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be14431c3fb0a34e6bd96874bd9859d957c5877b" target='_blank'>
              Long-read sequencing of primate testis and human sperm allows identification of recombination events in individuals
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. Winge, C. Hvilsom, Carmen Oroperv, Lasse Thorup Hansen, J. Berner, Marta Pelizzola, S. Laurentino, N. Neuhaus, Asger Hobolth, Thomas Bataillon, S. Besenbacher, Kristian Almstrup, M. Schierup
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second leading cause of cancer‐related deaths worldwide, with drug resistance being a major challenge in limiting the available treatment options. This review focuses on the signaling pathways (Hedgehog, Hippo, Notch, Wnt/β‐catenin, PI3K/Akt, and MAPK/ERK pathways) that are implicated in the development of drug resistance mechanisms in CRC, such as DNA repair systems, tumor microenvironments, autophagy, metabolic reprogramming, evasion of apoptosis, and efflux pumps. Furthermore, recent advancements in targeted and combinatorial diagnostic‐therapeutic approaches are highlighted. This review delineates the significance of signaling cascades and their clinical relevance, offering novel perspectives on the formulation of therapeutic strategies to mitigate drug resistance. In light of the increasing prevalence of drug‐resistant CRC and the lack of effective solutions, it addresses the persistent demand for improved therapeutic modalities and provides a comprehensive framework for forthcoming research in this domain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5e85f40389452bec5ccebda49ee18d686f79d59" target='_blank'>
              Critical Contribution of Various Signaling Pathways in the Development of Drug Resistance in Colorectal Cancer: An Update
              </a>
            </td>
          <td>
            Mehran Molavand, M. Majidinia, Bahman Yousefi
          </td>
          <td>2025-11-01</td>
          <td>IUBMB Life</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73f73f3e37aa93d53554d82182a30e140df7f94" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies CLK1 inhibition as a strategy to restore PARP inhibitor sensitivity via ERCC1 isoform switching.
              </a>
            </td>
          <td>
            Chaohua Liu, Xu Fei, Yutuan Wu, Li Jiana, Mengdong Ni, Xia Siyu, Chen Lihua, Haiyun Zhao, Yu Min, Zhou Yuqi, Meiqin Zhang, Jiajia Li, Xiaohua Wu, Huang Yan, Zhu Tao, Xiaojun Chen
          </td>
          <td>2025-11-04</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ca5c42693a477fde2aa581c6c760a192f8faa2" target='_blank'>
              The cumulative impact of passenger mutations on cancer development
              </a>
            </td>
          <td>
            Akshatha Nayak, Candace S. Y. Chan, Ioannis Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Medulloblastoma is molecularly characterized into 4 main subgroups consisting of WNT, SHH, Grp 3 and Grp 4, which are further divided into 13 subtypes. Medulloblastoma manifests as a clinically heterogeneous tumor with varying prognoses depending on the molecular subgroup. Treatments for high-risk patients include craniospinal irradiation (36Gy) after surgical resection and chemotherapy. Group 3 tumors with upfront radiation therapy relapse in a shorter amount of time at 1 year. Currently, there are limited therapeutic strategies that address medulloblastoma tumor recurrence which has detrimental effects on patient outcome with <5% long term survival. Subsequently, there is a critical need to investigate the molecular mechanisms underlying therapeutic resistance in medulloblastoma and identify the best strategies to combat it. To understand what causes radiation resistance in medulloblastoma, we analyzed matched primary and relapsed patient tumors by single cell ATAC seq + gene expression. We further evaluated radiation resistance in an orthotopic mouse model and analyzed gene expression changes by single-cell RNA sequencing. We identify a resistant cell population that exhibits a distinctive gene signature comprising stem and metabolic gene transcription programs. We observe elevated expression of metabolic marker genes such as wtIDH1 and stem cell markers, TEAD1 and SOX6, within the recurrent cell population. We speculate that radiation resistance in group 3 medulloblastoma is facilitated by wtIDH1 metabolic reprogramming, leading to epigenetic adaptation and cell persistence. Thus, we evaluate how IDH1 changes the epigenetic landscape of radiation-resistant and sensitive cells by examining histone modifications (H3K27ac, H3K27me3, H3K4me3) and the broader chromatin architecture in response to IDH1 inhibition by Cut and Run and ATAC-sequencing. We observe that the manipulation of wtIDH1 reduces MB cell growth, stemness characteristics, and changes the chromatin landscape. Further, we find inhibition of wtIDH1 enhances susceptibility to re-irradiation and provides a survival benefit. Our results highlight evidence of the development of radiation resistance and provide a potential therapeutic alternative for treatment of relapsed MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d6602a0e48e41dffae7101ad80db0948939a5f" target='_blank'>
              DDDR-30. Targeting wtIDH1 in radiation-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, Rajeev Vibhakar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This comprehensive review investigates the complex interactions between oncogenesis and stem cells, underscoring the crucial importance of cancer stem cells (CSCs) in the mechanisms of tumorigenesis, metastatic dissemination, and resistance to therapeutic interventions. CSCs constitute a distinct subset within neoplasms, defined by unique characteristics such as the ability to self-renew, the potential to differentiate into various cellular phenotypes, and an inherent resistance to conventional treatment strategies. It is hypothesized that these properties are regulated by a multitude of signaling pathways, including Wnt/β-catenin, Notch, and Hedgehog signaling cascades. The tumor microenvironment (TME) significantly influences CSC dynamics by providing a nurturing niche that enhances CSC survival, enables immune evasion, and contributes to resistance against therapies. Key components of the TME, such as cancer-associated fibroblasts and immune-modulating exosomes, play a crucial role in augmenting CSC plasticity and metastatic capabilities. Furthermore, the hypoxic environment prevalent within the TME enhances CSC stemness through both metabolic and epigenetic alterations. Despite the existence of promising CSC-targeted therapeutic strategies, including immunotherapy, small-molecule inhibitors, and combinatorial approaches, notable challenges persist. These obstacles include CSC plasticity and heterogeneity, as well as the potential for immune rejection of CSC-targeted therapies. Advancements in precision medicine, genomics, and novel therapeutic interventions instill hope for the improvement of CSC-targeted strategies. A comprehensive understanding of the fluid interactions between CSCs, standard stem cells, and the TME is vital for advancing personalized cancer treatments that enhance patient success and reduce recurrence rates. This review highlights the need for continued research to bridge existing knowledge gaps and facilitate the translation of foundational scientific discoveries into clinical practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65aea70f8623678ce094b97b023361a5efc723c" target='_blank'>
              Biology of stem cell paradox: a double-edged sword-implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Yitayew Demessie, M. Kinde, Kalkidan Getnet, A. Bitew, Kassahun Berrie, Wagaw Sendeku, Asnakew Mulaw Berihun, A. Abat, Atsede solomon Mebratu, Melaku Getahun Feleke, Nesibu Tilahun Yesist, M. M. Ferede, Melkie Dagnaw Fenta
          </td>
          <td>2025-10-28</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Activating KRAS mutations are highly relevant to various cancers, and KRAS is the most frequently altered oncogenic protein in solid tumors. While historically considered undruggable, two KRASG12C inactive state-selective inhibitors are currently approved for treating patients with non-small cell lung cancer. However, these agents only demonstrate a 30–40% response rate and a median progression-free survival of approximately 6 months, with the inevitable emergence of resistance mechanisms, hence remaining far from achieving a cure. Additionally, several cancers with poor prognostic outcomes, such as pancreatic adenocarcinoma, are driven by other non-G12C KRAS mutations and thus have no effective targeted therapies. Improvements in understanding RAS signaling, RNA, and nucleic acid chemistry, as well as the role of the tumor microenvironment, have sparked a paradigm shift in the approach to KRAS inhibition and suggested the potential for several novel combination therapies. In this review, we provide an overview of the RAS pathway and discuss the ongoing development and status of therapeutic strategies for targeting the oncogenic RAS. We further delve into the challenges of resistance mechanisms to better understand the rationale behind these developing strategies, describe their mechanisms of action, and offer insights into the current clinical trial status of each of these approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c697e95b8beed852f306ce44fd5e98ffcf2023be" target='_blank'>
              Targeting KRAS mutations: orchestrating cancer evolution and therapeutic challenges
              </a>
            </td>
          <td>
            Khalil Choucair, Hafsa Imtiaz, Md Hafiz Uddin, M. Nagasaka, M. N. Al-Hallak, P. Philip, B. El-Rayes, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-11-28</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Traditional population sequencing methods struggle to directly correlate gene perturbations with transcriptome changes at the single-cell level, due to limitations such as long cycle times and significant batch effects. Single-cell sequencing (SCS) can reveal the diversity and dynamics of cell populations, while CRISPR gene editing and screening technologies enable large-scale functional gene discovery. The combination of these two approaches has led to the development of ScCRISPR-seq. This article reviews the principles of CRISPR screening and SCS technologies, introduces the development of ScCRISPR-seq, and, drawing on research on lung cancer and intrahepatic cholangiocarcinoma (ICC), summarizes its key applications in cancer research, including deciphering tumor heterogeneity, elucidating mechanisms of drug resistance, and modulating the immune microenvironment. This article further explores challenges with this technology, including experimental workflow, data sparsity, and technical noise. Solutions are proposed, including data completion using AI algorithms, eliminating amplification bias using UMI tags, and optimizing gene-phenotype associations through network modeling. However, clinical translation is limited by unclear mechanisms of action and high barriers to entry in drug development. Future research may consider combining ScCRISPR-seq with network pharmacology and molecular docking techniques to improve the efficiency of new cancer drug development and advance precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1262cf91f44f77bd99618ed1bc278657dd28dee" target='_blank'>
              The Application and Prospect of Single-cell CRISPR Screening Technology in Tumor Research
              </a>
            </td>
          <td>
            Zineng Zhou
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="B‐cell precursor acute lymphoblastic leukemia (B‐ALL) is a hematologic malignancy characterized by the uncontrolled proliferation of immature B lymphoblasts. Despite significant advancements in treatment, chemoresistance remains a major challenge. Previous studies, including those involving Colombian patient cohorts, have identified a gene signature involving ID1, ID3, and IGJ, which correlates with poor prognosis; however, the underlying mechanisms remain unclear. This study integrates metabolic profiling with gene expression analysis of ID1, ID3, and IGJ in patient‐derived lymphoblasts. It explores the role of IGJ overexpression in NALM‐6 cells to assess its potential contribution to chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e198116e17749ec5fbd2ea6f272323e0f4abd014" target='_blank'>
              Integration of Metabolic Profiling and Functional Genomics Suggests IGJ as a Driver of Chemoresistance in B‐ALL
              </a>
            </td>
          <td>
            Karen Lizeth Pachón-Meza, Camilo Ernesto Moreno-Cristancho, José Luis Padilla-Agudelo, Ana Isabel Ramos-Murillo, Diana Londoño-Barbosa, Gustavo Salguero, S. Sanabria-Barrera, Claudia Chica, Nataly Cruz-Rodriguez, Rubén Darío Godoy-Silva, J. A. Gutiérrez-Triana
          </td>
          <td>2025-10-06</td>
          <td>Pediatric Blood & Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Antibiotic resistance remains one of the most formidable challenges to modern medicine, threatening to outpace therapeutic innovation and undermine decades of clinical progress. While resistance was once viewed narrowly as a clinical phenomenon, it is now understood as the outcome of complex ecological and molecular interactions that span soil, water, agriculture, animals, and humans. Environmental reservoirs act as silent incubators of resistance genes, with horizontal gene transfer and stress-induced mutagenesis fueling their evolution and dissemination. At the molecular level, advances in genomics, structural biology, and systems microbiology have revealed intricate networks involving plasmid-mediated resistance, efflux pump regulation, integron dynamics, and CRISPR-Cas interactions, providing new insights into the adaptability of pathogens. Simultaneously, the environmental dimensions of resistance, from wastewater treatment plants and aquaculture to airborne dissemination, highlight the urgency of adopting a One Health framework. Yet, alongside this growing threat, novel therapeutic avenues are emerging. Innovative β-lactamase inhibitors, bacteriophage-based therapies, engineered lysins, antimicrobial peptides, and CRISPR-driven antimicrobials are redefining what constitutes an “antibiotic” in the twenty-first century. Furthermore, artificial intelligence and machine learning now accelerate drug discovery and resistance prediction, raising the possibility of precision-guided antimicrobial stewardship. This review synthesizes molecular insights, environmental drivers, and therapeutic innovations to present a comprehensive landscape of antibiotic resistance. By bridging ecological microbiology, molecular biology, and translational medicine, it outlines a roadmap for surveillance, prevention, and drug development while emphasizing the need for integrative policies to safeguard global health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2665fac6c17a4051c07ada66840ae2833282cd2b" target='_blank'>
              Beyond the Resistome: Molecular Insights, Emerging Therapies, and Environmental Drivers of Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-10-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA is continuously exposed to endogenous and exogenous factors that induce oxidative modifications leading to mutations and genomic instability. Oxidative DNA damage plays a dual role, contributing to physiological signaling at low levels while promoting mutagenesis, carcinogenesis and degenerative diseases when unpaired. Among various lesions, an oxidized base, such as 8-oxo-2′-deoxyguanosine (8-oxodG), is one of the major biomarkers of oxidative stress and genomic damage. Cells have evolved sophisticated repair processes, including base excision repair (BER), nucleotide excision repair (NER), and mismatch repair (MMR), to maintain genomic integrity. Dysregulation or polymorphism of these repair genes has been linked with cancer, neurologic, and cardiovascular disorders. This review discusses an overview of what is presently known concerning oxidative DNA damage and repair mechanisms, particularly emphasizing their molecular players, signaling routes, and human disease implications. It further refers to the latest advances in CRISPR-based technologies and multi-omics approaches that are redefining our understanding of DNA damage response (DDR) networks and creating new frontiers for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f647de037f61e5dbeca9c5a033dbdfb5daa6ba" target='_blank'>
              Cellular and Molecular Mechanisms of Oxidative DNA Damage and Repair
              </a>
            </td>
          <td>
            A. Ayna, Cuneyt Caglayan, S. Taysi
          </td>
          <td>2025-11-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, with the most consequential mutations occurring in coding exons and regulatory regions. However, the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a mutational hotspot at transcription start sites (TSSs) in the human germline, spanning several hundred base pairs in both directions. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants, many of which are excluded from de novo mutation calls, indicating that the hotspot partly arises during early embryogenesis. We associate the TSS mutational hotspot with divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, suggesting a recombination-independent mechanism distinct from known processes. Our findings are reinforced by mutational signature analysis, which highlights alternative double-strand break repair and transcription-associated mutagenesis. These insights reveal a germline mutational phenomenon with evolutionary and biomedical implications, particularly affecting genes linked to cancer and developmental phenotypes. The impact of transcription on germline mutagenesis remains poorly understood. Here, the authors identify a mutational hotspot at transcription start sites in the human germline that is significantly enriched with early mosaic variants; they also analyse the molecular mechanisms that could cause it and explore its potential consequences for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de526e07a78fe794f2e553a66798f6d31a5db32" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fueled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, Vladimir Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor exon 20 insertion mutations (EGFRex20ins) are a unique molecular subtype of non-small cell lung cancer (NSCLC) linked to resistance to EGFR tyrosine kinase inhibitors of the first and second generations. Until recently, these patients had few and frequently ineffective treatment options. Amivantamab, a bispecific antibody targeting both EGFR and mesenchymal–epithelial transition factor, has become a novel therapeutic strategy for this population. This review explores the mechanism of action of Amivantamab and its clinical efficacy and safety as demonstrated in clinical trials. Additionally, the clinical development of the subcutaneous formulation of amivan-tamab, real-world evidence and its regulatory status were evaluated. Lastly, we contex-tualise Amivantamab in the current treatment landscape by contrasting it with mobocer-tinib and highlighting current studies that aim to improve central nervous system activity and overcome resistance mechanisms. This review highlights the therapeutic benefit of Amivantamab in EGFRex20ins-mutated NSCLC and offers guidance for future research in this quickly developing area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac73581dcae1cceea8c975ae407a4672d857585f" target='_blank'>
              Evolving treatment strategies for EGFRex20ins-mutated NSCLC: a comprehensive review of Amivantamab’s role and future directions
              </a>
            </td>
          <td>
            Rafael Alvim Pereira, M. Tumelero, W. Schwengber, G. Lenz
          </td>
          <td>2025-11-17</td>
          <td>ecancermedicalscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers exhibit genetic diversity between individual cancer cells. Previous work shows greater diversity than heretofore expected (1) and that also increases more quickly during a patient’s clinical course than previously thought (1, 2). Rare subclones will harbor pre-existing resistance to any single agent and may cause medium to late term relapse (1), which may evolve further variants that have simultaneous resistance to non-cross resistant therapies (3, 4). We now present a probability distribution function (PDF) of the variant allele fraction (VAF) or prevalence of a rare subclone, derived from previous evolutionary theory (1, 2). We show that current clinical sequencing protocols fail to detect the vast majority of rare subclones. By the time of detection, simultaneous multiple resistance may evolve. We then apply the PDF to simulation of dynamic precision medicine (DPM) (3), an evolutionary guided precision medicine paradigm that attempts to proactively eliminate singly-resistant subclones before they evolve multiple resistance, with significant potential to extend survival. We show that the simulated benefit of DPM with perfect information is degraded by inability to detect rare subclones if they are assumed to be absent when undetectable. But this benefit is restored if the PDF is used to calculate the likelihood of the subclone being present below the level of detection and incorporated into the DPM simulation and therapy recommendations in a probabilistic fashion. Moreover, other common statistical distributions are less effective. This theoretical advance facilitates DPM and potentially other evolutionary guided approaches to precision cancer medicine in spite of the limitations of clinical sequencing. Significance Statement Cancers contain many cells, each genetically unique. These variations can include pre-existing resistance to therapy, enabling relapse. DNA sequencing cannot detect minority cellular populations (subclones) below a certain size, by which time they may have evolved simultaneous resistance to multiple therapies. We present a mathematical approach that enables assigning a risk that a subclone is present, and at what prevalence in the cellular population, even when undetected. We simulate applying this approach to dynamic precision medicine (DPM), which attempts to proactively eliminate singly resistant cells before they become resistant to multiple therapies. Using this probability distribution, we can retain the benefit of DPM even when most rare subclones are undetectable, in contrast to just assuming undetected subclones are absent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common primary malignant bone tumor, remains a therapeutic challenge because of its high metastatic potential, chemoresistance, and poor prognosis. Mutations in the TP53 tumor suppressor gene, including loss-of-function (LOF) and gain-of-function (GOF) mutations, play a central role in OS progression by disrupting cell cycle regulation, DNA repair, and apoptosis and promoting immune evasion and metabolic reprogramming. This review provides an in-depth analysis of p53 biology in OS, highlighting its impact on therapeutic resistance and tumor progression. We discuss advancements in radiotherapy, chemotherapy, and immunotherapy, emphasizing strategies targeting mutant TP53 and its associated pathways. Emerging approaches, including metabolic reprogramming, noncoding RNA regulation, and precision biomarkers such as miRNAs and histone modifications, offer promising tools for diagnosis, risk stratification, and treatment optimization. By linking the molecular mechanisms of p53 with novel therapeutic strategies, this review underscores opportunities for translational research aimed at improving the clinical outcomes of OS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4de5949ca03e6677f6b90cc0b23d966a37a92dc" target='_blank'>
              Involvement of TP53 in osteosarcoma - challenges and prospects
              </a>
            </td>
          <td>
            Yue Shen, Shuzhou Huang, Geng Chen, Guangda Wang, L. Sui
          </td>
          <td>2025-11-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="One of the main challenges in cancer treatment is addressing the metabolic reprogramming of tumor cells, which require more energy and biomolecules than healthy cells. Cancer cells alter their metabolism by switching between glycolysis and oxidative phosphorylation (OXPHOS). These processes depend on transmembrane proteins that respond to the extracellular environment. Our research identified the transmembrane form of Chloride Intracellular Channel 1 (tmCLIC1) as a marker of malignancy and a potential therapeutic target. tmCLIC1 levels are increased in several solid tumors, supporting cancer growth and progression, whereas they are mostly absent in healthy cells. We confirmed that tmCLIC1 is the specific target of the antidiabetic drug metformin, an OXPHOS inhibitor in cancer cells. tmCLIC1 is the primary target of metformin in glioblastoma-initiating cells, as shown by single-channel patch-clamp recordings and NMR experiments. Various patient-derived glioblastoma cells with different genetic backgrounds were used to demonstrate that CLIC1 CRISPR-Cas9 knockout and/or its point mutation at arginine 29 removes metformin’s antitumor effects. Functional assays were used to assess the effects on proliferation, mitochondrial metabolism, and tumor growth in vitro and in vivo, using zebrafish and murine xenograft models. Metformin inhibits the function of tmCLIC1 through direct and specific binding involving arginine 29 in the tmCLIC1 sequence. Additionally, during hypoglycemia, metformin promotes glioblastoma cell apoptosis by inhibiting the Cancerous Inhibitor of Protein Phosphatase 2 A (CIP2A) and activating the PP2A B56δ subunit. This leads to the dephosphorylation of Glycogen Synthase Kinase 3 Beta (GSK3β), resulting in the breakdown of the pro-survival protein MCL-1 and subsequent cell death. Inhibition of tmCLIC1 is crucial for this metformin-driven antineoplastic effect, mainly through regulating the PP2A-GSK3β-MCL-1 pathway under hypoglycemic conditions. The chronic presence of metformin within the tumors impairs in vivo growth at nanomolar concentrations. The therapeutic role of metformin to treat brain tumors remains debated. Our findings show that drug delivery is essential, as in vivo, tumor growth decreases at concentrations below 10 nanomolar. We propose that sustained CNS metformin levels may improve tmCLIC1 inhibition, providing a basis for optimizing interactions with metformin or related compounds to enhance therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775ada65f91c82f60f8e562cfdc0b58db53cdcbe" target='_blank'>
              Chronic administration of metformin exerts cytostatic and cytotoxic effects via the PP2A-GSK3β-MCL-1 pathway by inhibiting the tmCLIC1 membrane protein in glioblastoma-initiating cells
              </a>
            </td>
          <td>
            F. Cianci, I. Verduci, R. Cazzoli, G. Cannavale, G. Rey, Marina Veronesi, B. Balboni, M. Ranucci, L. Palloni, Federico Ballabio, N. Barsotti, Giorgia Ailuno, Alice Balboni, S. Baldassari, G. Caviglioli, C. Tacconi, C. Camilloni, S. Girotto, Federica Barbieri, Alessandro Fantin, A. Cavalli, M. Pasqualetti, Tullio Florio, S. Minucci, Michele Mazzanti
          </td>
          <td>2025-11-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa35e38064580145b981499573e43bdd54174f5" target='_blank'>
              Allele-specific correction of ATAD3A pathogenic variants via template-free CRISPR-Cas9 editing and gene conversion
              </a>
            </td>
          <td>
            Taejeong Bae, Yohan Park, Anel LaGrone, M. Šuvakov, Ping Zhang, Hyekyung Park, Holly Van Remmen, James R. Lupski, Tamar Harel, Jean J. Kim, A. Abyzov, Wan Hee Yoon
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Genomic alterations in germ cell tumors (GCTs) have been identified in the RTK, MAPK, and PI3K pathways through whole exome sequencing, with a prevalence of 71% in germinomas but only 33% in non-germinomas GCTs (NGGCTs). Despite evolution in treatment, GCTs present therapeutic challenges and lack molecularly targeted strategies.



 Whole genome sequencing was performed on 71 cases (37 germinomas and 34 NGGCTs) from the iGCT Genome Analysis Consortium and the Japan Children’s Cancer Group.



 Putative driver mutations were identified in 83% of tumors, primarily involving RAS/MAPK followed by mTOR/Akt pathways. Germinomas exhibited a higher mutation frequency than NGGCTs (84% vs 62%), with KIT enriched in germinomas and USP28 in NGGCTs. All USP28 alterations were bi-allelic and highly specific to CNS GCTs, with only one diffuse midline glioma out of 1448 cases of pediatric cancers affected. Variant allele frequency and clonal analyses revealed that these mutations occurred early, preceding copy number alterations. Germline analysis identified Kleinfelter or Down syndrome in 5.6% of cases, particularly in NGGCTs lacking other alterations.



 Many cases exhibited driver mutations present in major clonal populations and could serve as therapeutic targets. There were notable differences based on histological type, with a diverse range of single nucleotide variants/structural variants observed in NGGCTs. A timeline of tumorigenesis from SNVs/SVs to copy number abnormalities was observed. The utility of understanding germline variants was also implicated in the pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6633a5d8f98dc245c8a327ae1335afe0892cad2" target='_blank'>
              PTHP-06. Molecular Mechanisms and Developmental Pathways of CNS Germ Cell Tumors Elucidated through Whole Genome Analysis
              </a>
            </td>
          <td>
            Hirokazu Takami, S. Kato, Yuki Yamagishi, Eiji Sugihara, Y. Matsushita, Takako Yoshioka, Junko Hirato, Miho Kato, Hajime Okita, Tomonari Suzuki, Masayuki Kanamori, Akio Asai, Akitake Mukasa, Toshihiro Kumabe, Akihide Kondo, Yuichi Hirose, M. Nagane, Kazuhiko Sugiyama, T. Shibata, Kenichi Yoshida, Y. Shiraishi, Takaaki Sato, Ryo Nishikawa, Motohiro Kato, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The integration of multi-omics technologies is transforming the landscape of cancer management, offering unprecedented insights into tumor biology, early diagnosis, and personalized therapy. This review provides a comprehensive overview of the current state of omics approaches, with a particular focus on the application of genomics, NMR-based metabolomics, and radiomics in non-small cell lung cancer (NSCLC). Genomics currently represents one of the most established omics technologies in oncology, as it enables the identification of genetic alterations that drive tumor initiation, progression, and therapeutic response. Interestingly, genomic analyses have revealed that many tumors harbor mutations in genes encoding metabolic enzymes, thus establishing a tight connection between genomics and tumor metabolism. In parallel, metabolomics profiling—by capturing the metabolic phenotype of tumors—has, in recent years, identified specific biomarkers associated with tumor burden, progression, and prognosis. Such findings have catalyzed growing interest in metabolomics as a complementary approach to better characterize cancer biology and discover novel diagnostic and therapeutic targets. Moreover, radiomics, through the extraction of quantitative features from standard imaging modalities, captures tumor heterogeneity and contributes predictive information on tumor biology, treatment response, and clinical outcomes. As a non-invasive and widely available technique, radiomics has the potential to support longitudinal monitoring and individualized treatment planning. Both metabolomics and radiomics, when integrated with genomic data, could support a more comprehensive understanding of NSCLC and pave the way for the development of non-invasive, predictive models and personalized therapeutic strategies. In addition, we explore the specific contributions of these technologies in enhancing clinical decision-making for lung cancer patients, with particular attention to their potential in early diagnosis, treatment selection, and real-time monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3446e3e91ac7b3e6259e583b7f7fc73de7c2244f" target='_blank'>
              Advancing Non-Small-Cell Lung Cancer Management Through Multi-Omics Integration: Insights from Genomics, Metabolomics, and Radiomics
              </a>
            </td>
          <td>
            Martina Pierri, G. Ciani, M. C. Brunese, G. Lauro, S. Terracciano, Maria Iorizzi, Valerio Nardone, M. Chini, Giuseppe Bifulco, Salvatore Cappabianca, A. Reginelli
          </td>
          <td>2025-10-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Protein kinases are fundamental regulators of cellular function, orchestrating proliferation, survival, and differentiation through reversible phosphorylation. Their importance extends beyond normal physiology: dysregulated kinase activity is a defining feature of cancer, where constitutive signaling sustains malignant growth and therapy resistance. Among them, receptor tyrosine kinases occupy a central role by transmitting extracellular cues into potent intracellular programs. One such receptor, FMS-like tyrosine kinase 3 (FLT3), is expressed at high levels in hematopoietic stem and progenitor cells, where it maintains homeostasis and supports immune development, particularly dendritic cell biology. In acute myeloid leukemia (AML), FLT3 mutations—most often internal tandem duplications (ITDs) or tyrosine kinase domain substitutions (TKDs)—constitute some of the most frequent genetic alterations and are strongly associated with poor prognosis. The therapeutic targeting of FLT3 represents a paradigm for translating kinase biology into clinical progress. Decades of preclinical and clinical work culminated in the approval of three potent FLT3 inhibitors—midostaurin, gilteritinib, and quizartinib—that, together with stem cell transplantation, have dramatically improved outcomes for patients with FLT3-mutant AML. Yet, as with many kinase inhibitors, resistance remains a barrier, fueling efforts toward rational combination therapies and measurable residual disease-directed strategies. The FLT3 story illustrates both the promise and the ongoing challenges of kinase inhibition in modern cancer care. Abstract FMS-like tyrosine kinase 3 (FLT3) is a crucial regulator of normal hematopoiesis, with high expression in hematopoietic stem and progenitor cells. Beyond its role in stem cell survival and proliferation, FLT3 signaling is essential for immune regulation, particularly dendritic cell differentiation and NK cell expansion. In acute myeloid leukemia (AML), FLT3 mutations—most commonly internal tandem duplications (FLT3-ITD) and tyrosine kinase domain (FLT3-TKD) substitutions—are among the most frequent genetic alterations, driving constitutive activation of proliferative and antiapoptotic pathways and conferring adverse prognosis. The clinical development of FLT3 inhibitors has been a decades-long endeavor. Early multikinase agents established proof-of-concept but were hampered by off-target effects and incomplete efficacy. The subsequent generation of potent and selective inhibitors has transformed outcomes, culminating in FDA approvals of midostaurin, quizartinib, and gilteritinib. Together with allogeneic transplantation, these agents have reshaped the treatment paradigm for FLT3-mutant AML, converting a historically high-risk subset into one with realistic prospects for long-term survival. Despite these advances, challenges remain. Resistance emerges through cell-intrinsic mechanisms such as acquisition of secondary TKD or RAS pathway mutations, metabolic reprogramming, and antiapoptotic shifts, as well as cell-extrinsic mechanisms mediated by the bone marrow microenvironment, including cytokine support, stromal CYP3A4 metabolism, and retinoid inactivation. These pathways sustain measurable residual disease (MRD), the key predictor of relapse. Rational combination strategies and MRD-directed approaches are therefore essential to fully realize the curative potential of FLT3 inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d84ff453602bf5fb9f5f0df83860106bca5b8365" target='_blank'>
              FLT3: A 35-Year Voyage from Discovery to the Next Generation of Targeted Therapy in AML
              </a>
            </td>
          <td>
            Maria-Camelia Stancioaica, D. Coriu, Gabriel Ghiaur
          </td>
          <td>2025-10-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The endosomal pathway plays a crucial role in eukaryotes, and its conservation across evolution underscores its importance in fundamental processes, such as endocytosis, cytokinesis, and autophagy. Dysregulation of the Notch signaling system, essential for growth and development, contributes to cancers like leukemia. In Drosophila melanogaster (DM), specific endosomal genes function as tumor suppressors, regulating Notch signaling, either through promoting Notch degradation or directing its recycling. Nevertheless, it is still unanswered how endosomal genes specifically, lethal giant discs (lgd/CC2D1A), affect Notch-1-mediated transduction in human malignancies, such as T-cell acute lymphoblastic (T-ALL). This study aimed to comprehensively characterize the (lgd/CC2D1A) CC2D1A/lgd in human T-ALL by performing mutational analysis and sequencing and evaluating its gene and protein expression levels. These analyses were conducted to elucidate better the role of (lgd/CC2D1A)CC2D1A in the pathophysiology of T-ALL and to explore its potential association with the Notch-1 signaling pathway. In this study, we sequenced the lgd and investigated its expression in peripheral blood samples of human T-ALL samples. We used molecular approaches to analyse the Lgd/CC2D1A gene for mutations and characterize its gene and protein expression across genomic and proteomic datasets. Sequencing data revealed the presence of the lgd variant (CC2D1A) in T-ALL patients and healthy individuals. Lgd and Notch-1 expression were significantly upregulated in T-ALL patients, demonstrating a strong positive correlation at both the genomic and proteomic levels. Through this analysis, we reported the presence of Lgd with an altered nucleotide sequence in T-ALL patients. We also found its elevated expression compared with healthy controls, which might affect disease progression. Our investigation results also lay the foundation for the possible development of Lgd as a predictive indication for the early identification of T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b8e3e49ce5c7f6dd3ebfcedabaf637bb4da449" target='_blank'>
              Characterization, expression, and mutational analysis of endosomal protein lethal giant disc (Lgd/CC2D1A) as a biomarker in patients with T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sandeep Pandey, Ranjana Singh, Nimra Habib, Rashmi Kushwaha, A. A. Mahdi
          </td>
          <td>2025-11-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9762d6dd940b73e9b0f65cdd56efc96e1c86d4a" target='_blank'>
              Genome-wide profiling of short tandem repeat somatic instability reveals associations with age, sex and brain-related traits
              </a>
            </td>
          <td>
            J. Hu, R. Mantey, H. Guo, A. M. Sidky, M. Touhidinia, T. Butt, R. Sobhani, S. Estrada, K. Haendler, S. Zhao, X. Shang, E. De Domenico, M. Beyer, M. M. Breteler, N. A. Aziz
          </td>
          <td>2025-11-15</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7bcb7a0494ca2d2085913525073bd8291d09b9" target='_blank'>
              EGFR suppression and drug-induced potentiation are widespread features of oncogenic RTK fusions
              </a>
            </td>
          <td>
            Yuzhi (Carol) Gao, D. Gonzalez-Martinez, Sofia Wissert, Hana Bader, Nidhi Sahni, Anh Le, R. Doebele, Lukasz J. Bugaj
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Epitranscriptomics, the study of dynamic chemical modifications on RNA mediated by “writers,” “erasers,” and “readers,” has emerged as a pivotal discipline in elucidating the intricate interplay between cancer and immune regulation. These reversible modifications (e.g. m6A, m5C, Ψ) govern RNA metabolism, stability, and translation, thereby exerting spatiotemporal control over immune cell differentiation, activation, and function. Dysregulation of RNA-modifying proteins disrupts immune surveillance, enhances tumor cell survival under stress, and promotes chemoradiotherapy resistance by altering RNA splicing, translation, and stress adaptation pathways. This review summarized the recent progress in the regulatory mechanisms profoundly influencing the tumor microenvironment (TME), modulating immune checkpoints, antigen presentation pathways, and the activity of immune cells. Furthermore, we discussed the therapeutic strategies and challenges in targeting epitranscriptomic regulators and epitranscriptomic editing technologies to enhance anti-tumor immune responses and overcome therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c0776f1a44885afebca84621fac676e8beb717" target='_blank'>
              Epitranscriptomic control of cancer immunity and therapy resistance
              </a>
            </td>
          <td>
            Xingsen Zhao, Suhua Guan
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) is a reversible growth arrest induced by anticancer treatments, which may contribute to the development of long-term therapy resistance in tumor cells. Senotherapeutics, agents targeting senescent cells, are being tested in clinical trials to improve patient outcomes. Due to the transient nature of TIS, we hypothesized that senolytics would be most effective when administered at the appropriate time. We created a reliable TIS cell line model in triple-negative breast cancer (TNBC) using experimental drug YM155. We observed that a single dose of YM155 triggers a brief senescence, leading to a persistent drug-tolerant state that cannot be reversed by redosing. This reversibility is not limited to cancer cells. It extends to noncancerous human cells and live zebrafish larvae, suggesting a rapid adaptation mechanism against xenobiotics. We identified transforming growth factor-β (TGF-β), a cytokine linked to TNBC chemoresistance, as being expressed alongside the emergence of drug tolerance. We inhibited TGF-β signaling to eliminate the tolerant phenotype and promote the clearance of cancer cells by immune cells. However, this was most effective within a specific time window after TIS induction. We suggest that the timely use of senotherapeutics could improve the effectiveness of anticancer drugs in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ec0acb6fede0ff85bcba84d195e45333ad41d7" target='_blank'>
              Mapping the Progression of Therapy-Induced Senescence to Therapy Tolerance: An Evolutionarily Conserved Mechanism for Optimizing Cancer Treatment with Senotherapeutics.
              </a>
            </td>
          <td>
            Gargi Mukherjee, Neha Dutta, Muthumeena Ramanathan, Kankonika Bhattacharyya, Alabhya Das, Shaista Haider, Goutam Chowdhury, Anindita Chakrabarty
          </td>
          <td>2025-10-10</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Precision therapy for glioma remains a major challenge due to tumor heterogeneity. The Origin Recognition Complex Subunit 6 (ORC6) is a crucial regulator of DNA replication initiation. This study aims to investigate the expression of ORC6 in gliomas and its relationship with survival rates and malignancy, while screening potential drugs targeting its functional network. By integrating multiple bioinformatics approaches with structure-based virtual screening, retrospective RNA sequencing data analysis was performed using patients from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. A protein-protein interaction (PPI) network was constructed from ORC6-coexpressed genes to identify core hubs. Molecular docking was employed to screen a library of natural compounds and known drugs against these hub targets. Research has revealed that ORC6 is significantly upregulated in high-grade gliomas, with its elevated expression associated with poor survival outcomes and immune inflammatory responses. Network analysis identified five core hub genes (ORC1, ORC2, MCM2, MCM6, CDC45) central to DNA replication. Molecular docking revealed that several compounds, including the natural flavonoid Baicalein and the FDA-approved drug Palbociclib, exhibited high binding affinity to these hub targets. ORC6 represents a highly promising novel target for precision therapy in glioma. Potential approaches to target this pathway include disrupting the ORC6-replication axis using existing drugs (such as palbociclib) or natural products (such as baicalin).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324196dc2699eff3ae974af550ed77475be4c5ee" target='_blank'>
              High ORC6 expression is a prognostic indicator of poor survival in glioma patients
              </a>
            </td>
          <td>
            Mengjie Wang, Song Feng, Chen Zhang, Feng Jin
          </td>
          <td>2025-11-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality among women worldwide, with treatment resistance posing a significant clinical challenge. Circular RNAs (circRNAs), a class of non-coding RNAs, have gained increasing attention as key regulators of gene expression, influencing BC pathogenesis, progression, and therapeutic response. This review explores the mechanistic insights into circRNA functions in BC, focusing on their involvement in tumor proliferation, metabolic reprogramming, epithelial-mesenchymal transition (EMT), angiogenesis, metastasis, and apoptosis. Additionally, we highlight the crosstalk between circRNAs and microRNAs, emphasizing their potential as diagnostic and prognostic biomarkers. Beyond their roles in tumor biology, circRNAs are implicated in drug resistance, modulating responses to chemotherapy, targeted therapy, and endocrine treatment. Despite their promising applications, challenges remain, including the complexity of circRNA interactions, and the development of robust preclinical models. Addressing these challenges through interdisciplinary research integrating genomics, transcriptomics, and functional studies will pave the way for circRNA-based therapeutic strategies and personalized medicine approaches in BC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3e2c40f929a3a40c435801c1f0dae7da3b0326" target='_blank'>
              Circular RNAs orchestrating breast cancer hallmarks: bridging tumor biology and therapy resistance
              </a>
            </td>
          <td>
            Abdelhamid M. Abdelhamid, Shafei Ahmed Mai, Mariam Gamaleldin, Alaa Bassam Heikal, Wesam H Khidr, Nabil Hazza Al-Saadi, Ganna Magdy Kandil, Mahmoud M Omarn, Maha Alhelf
          </td>
          <td>2025-10-13</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly aggressive liver cancer with poor prognosis, often associated with resistance to treatment. MET amplification has been identified as a potential therapeutic target, but resistance to MET inhibitors, such as crizotinib, remains a significant challenge. This study aims to explore the molecular mechanisms underlying resistance to MET inhibitors in MET-amplified HCC. Methods We present a case of advanced HCC in a patient with MET amplification treated with crizotinib. After initial tumor regression, disease progression occurred. Genetic analysis using next-generation sequencing (NGS) was performed on biopsy samples taken before and after progression to identify mutations associated with resistance. Results NGS revealed the loss of MET amplification and identified HGF and NF1 mutations as potential bypass mechanisms. Specifically, a missense mutation in HGF (p.G401A) was observed, which may enhance ligand-receptor binding, while an NF1 mutation (p.M546L) may permit sustained MAPK and PI3K activation despite MET inhibition. These observations are preliminary and require validation in larger patient cohorts. Conclusion Our findings suggest that acquired resistance to MET inhibitors in MET-amplified HCC may involve clonal evolution and activation of compensatory signaling pathways. These insights highlight the need for dynamic molecular surveillance and the development of strategies targeting multiple pathways to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cf103ad37849fe32fd68a89e9da98214e1dcc6" target='_blank'>
              Case Report: HGF and NF1 mutations as putative bypass mechanisms of MET inhibitor resistance in hepatocellular carcinoma: a case study
              </a>
            </td>
          <td>
            Zhitao Chen, Shan Luo, Yangjun Gu, Qiyong Li
          </td>
          <td>2025-11-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The process of aging is fundamentally driven by genomic instability and the accumulation of DNA damage, which progressively impair cellular and tissue function. In order to counteract these challenges, cells rely on the DNA damage response (DDR), a multilayered signaling and repair network that preserves genomic integrity and sustains homeostasis. Within this framework, nucleases and helicases have pivotal and complementary roles by remodeling aberrant DNA structures, generating accessible repair intermediates, and determining whether a cell achieves faithful repair, undergoes apoptosis, or enters senescence. Defects in these enzymes are exemplified in human progeroid syndromes, where inherited mutations lead to premature aging phenotypes. This phenomenon is also replicated in genetically engineered mouse models that exhibit tissue degeneration, stem cell exhaustion, and metabolic dysfunction. Beyond their canonical repair functions, helicases and nucleases also interface with the epigenome, as DNA damage-induced chromatin remodeling alters enzyme accessibility, disrupts transcriptional regulation, and drives progressive epigenetic drift and chronic inflammatory signaling. Moreover, their dysfunction accelerates the exhaustion of adult stem cell populations, such as hematopoietic, neural, and mesenchymal stem cells. As a result, tissue regeneration is undermined, establishing a self-perpetuating cycle of senescence, impaired repair, and organismal aging. Current research is focused on developing therapeutic strategies that target the DDR–aging axis on several fronts: by directly modulating repair pathways, by regulating the downstream consequences of senescence, or by preventing DNA damage from accumulating upstream. Taken together, evidence from human disease, animal models, molecular studies, and pharmacological interventions demonstrates that nucleases and helicases are not only essential for genome maintenance but also decisive in shaping aging trajectories. This provides valuable knowledge into how molecular repair pathways influence organismal longevity and age-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9382a44469cc78462704e76a62b5f9104801fc" target='_blank'>
              The Converging Roles of Nucleases and Helicases in Genome Maintenance and the Aging Process
              </a>
            </td>
          <td>
            Aikaterini Margariti, Persefoni Daniil, T. Rampias
          </td>
          <td>2025-11-01</td>
          <td>Life</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has revolutionized oncology by achieving durable remissions in refractory hematologic malignancies. However, emerging reports link this therapy to second primary malignancies, including CAR+ lymphomas and leukemias, driven by insertional mutagenesis from semi-random viral vector integration near oncogenes or tumor suppressor loci. These rare but serious complications underscore the dual challenge of eradicating primary tumors while mitigating delayed genotoxic risks. Conventional CAR-T cell manufacturing, reliant on gamma-retroviral or lentiviral vectors, introduces genomic instability through integration into fragile sites or transcriptionally active regions. CRISPR/Cas9-mediated genome editing further amplifies risks via off-target double-strand breaks and chromosomal rearrangements. This review evaluates genomic safe harbors (GSHs)—such as AAVS1, TRAC, CCR5, ROSA26 and CLYBL—as loci validated for stable, high-level CAR transgene expression without oncogenic disruption. GSHs meet stringent criteria: distal from cancer-related genes, resistant to epigenetic silencing, and transcriptionally permissive. Preclinical studies demonstrate that site-directed CAR integration into GSHs preserves antitumor efficacy while eliminating malignant transformation risks. Challenges persist in optimizing homology-directed repair efficiency, mitigating residual dsDNA toxicity, and standardizing regulatory frameworks for long-term genomic surveillance. Emerging technologies—base/prime editing, hybrid nucleases, and rigorous monitoring—promise enhanced precision and safety. By reconciling therapeutic innovation with genomic integrity, GSH-engineered CAR-T cells herald a paradigm shift toward precision immunotherapies, offering curative potential while preempting secondary oncogenesis. Collaborative efforts to refine manufacturing, harmonize global standards, and prioritize patient-specific risk stratification will be critical to advancing this transformative approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a919a0552fea16eb7ba7478bb3fc8327302c77cd" target='_blank'>
              Preventing secondary primary malignancies (SPMs) in CAR-T cell therapy through site-specific transgene integration into genomic safe harbors (GSHs)
              </a>
            </td>
          <td>
            Pooria Safarzadeh Kozani, Pouya Safarzadeh Kozani
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Drug resistance poses a major global health challenge, necessitating the development of effective therapeutic strategies. The main challenge is to predict drug-resistant mutations and design drugs that retain efficacy against such evolving targets. Our previous effort in computing the vitality value has provided a framework in assessing drug resistance. While promising, the approach lacked accuracy due to insufficient information about mutation tendencies and protein stability. In this study, we used the Stanford University HIV Drug Resistance Database and observed that drug resistance, usually quantified as [Formula: see text], exhibits a positive correlation with the Maximum Entropy energy, [Formula: see text]. However, both drug resistance and vitality are also correlated with the number of mutations, indicating that the virus cannot easily gain resistance through specific mutational pathways and must sacrifice stability and function to escape inhibition. To overcome this number dependence, we looked for a system with less extensive mutagenesis and chose HCV protease. In this case, resistance substitutions cluster at low [Formula: see text] values, reflecting a limited mutational space. This restricted landscape enables [Formula: see text] to predict evolutionary pathways of resistance and to guide the identification of robust therapeutic candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc38f3191fe3cd0d179e733fa7821cca3748a5b" target='_blank'>
              Exploring evolutionary trajectories of drug resistance.
              </a>
            </td>
          <td>
            Linfeng Hu, Aoxuan Zhang, A. Warshel
          </td>
          <td>2025-11-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="CRISPR screening has revolutionized cancer research by enabling systematic, genome-wide interrogation of gene
function to identify therapeutic vulnerabilities, resistance mechanisms, and immunotherapy determinants. This review
examines methodological approaches for CRISPR screenings using the Cas9 systems, evaluating how system and methodology selection
profoundly influence which biological mechanisms are revealed across therapeutic resistance, metastasis regulation, and immunotherapy
response. Cas9-based screens-from simple one-cell-type (1CT) in vitro approaches to complex autochthonous in vivo systems-excel at
identifying cell-autonomous DNA-level dependencies. Two-cell-type (2CT) co-culture approaches validate immune-dependent mechanisms,
while in vivo screens capture physiological complexity and have identified clinically relevant targets including PTPN2, now in Phase 1 trials.
Critical challenges include incomplete gene editing, false-positive and false-negative hits, context-dependent essentiality, and system-specific
technical limitations. Emerging technologies integrating artificial intelligence, prime editing, single-cell profiling, and patient-derived models
promise to address limitations while enabling functional precision medicine. This review provides a framework for selecting appropriate
systems, interpreting results within methodological constraints, and integrating complementary approaches to maximize biological insight and
translational impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e21f4ff0a121855b3f14e7fcc069197b5d3507" target='_blank'>
              CRISPR- CAS SCREENING IN CANCER RESEARCH: METHODOLOGIES, APPLICATIONS, AND TRANSLATIONAL CHALLENGES
              </a>
            </td>
          <td>
            Ahana Patel
          </td>
          <td>2025-11-01</td>
          <td>INDIAN JOURNAL OF APPLIED RESEARCH</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification has emerged as a critical epigenetic mechanism regulating gene expression in diverse physiological and pathological processes, including cancer. Methyltransferase-like 16 (METTL16), a recently identified m6A methyltransferase, has been shown to influence tumor progression through m6A-dependent regulation of key target genes. Accumulating evidence indicates that METTL16 exerts tumor-suppressive or tumor-promoting roles in a context-dependent manner, affecting cell proliferation, apoptosis, autophagy, and chemotherapeutic response across multiple cancer types such as bladder cancer, lung cancer, colorectal cancer, and acute myeloid leukemia. Mechanistically, METTL16 modifies the mRNA stability and translation of oncogenes or tumor suppressors via recognition of m6A sites, and its expression can be regulated by upstream factors including transcription factors and hypoxia-inducible signals. Recent evidence suggests that METTL16 also modulates the tumor microenvironment (TME), potentially affecting immune cell infiltration, immune checkpoint expression, and tumor immune evasion. Collectively, METTL16 emerges as a pivotal epitranscriptomic regulator linking RNA modification, tumor progression, and immune modulation, offering new avenues for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cac623df0454861cf59d7b80f15a7189f66d09" target='_blank'>
              METTL16 emerges as a pivotal epitranscriptomic regulator, linking RNA modification, tumor progression, and immune modulation
              </a>
            </td>
          <td>
            Qiang Wang, Xiulin Jiang, Yixiao Yuan, Chunhong Li
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Skin cancer encompasses a range of diseases that differ widely in their genetic drivers, clinical behavior, and treatment response. Among these, melanoma is the most aggressive form and includes several uncommon subtypes, such as acral, mucosal, and chronically sun-damaged melanomas, that often contain activating mutations in the KIT gene. These tumors behave differently from other forms of melanoma and frequently demonstrate variable responses to standard targeted or immune-based therapies. The purpose of this review is to synthesize and interpret current evidence on how specific KIT gene mutations shape clinical outcomes, therapeutic sensitivity, and resistance mechanisms. We highlight findings from key clinical studies evaluating targeted tyrosine kinase inhibitors and immune checkpoint inhibitors, emphasizing the most responsive mutation sites, comparative efficacy across agents, and treatment sequencing considerations. The review also explores clinical implications and highlights critical unanswered questions in this evolving field. Together, these insights provide a practical framework to guide ongoing research and inform more effective, personalized therapeutic strategies for patients with KIT-mutant melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a3b1783130151a4e725cc98816d0f672f99e5" target='_blank'>
              KIT-Mutant Melanoma: Understanding the Pathway to Personalized Therapy
              </a>
            </td>
          <td>
            Aditi Kaveti, Ryan J. Sullivan, Hensin Tsao
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer is the most frequently diagnosed type of cancer worldwide, according to GLOBOCAN 2022 statistics. Key genetic alterations involve driver gene mutations that significantly enhance cancer aggressiveness. These include several EGFR mutations, ALK rearrangements, ROS1 rearrangements, RET translocations, MET alterations, NTRK fusions, BRAF mutations and KRAS mutations, such as the KRAS G12C mutation. Naturally, each of these is part of a larger signaling pathway that becomes dysregulated via genetic alterations. We highlight the transduction of EGFR: HER2 via RAS-RAF-MEK-MAPK pathway, PI3K-PTEN-AKT pathway and STAT pathway, of the ALK via PI3K/AKT, MAPK/ERK and JAK/STAT and of KRAS via effectors of the MAPK pathway and of the PI3K pathway. MicroRNAs (miRNAs) interfere at various levels with these pathways, either with pro-oncogenic effects or tumor suppressive effects. For instance, miR-33a is a tumor suppressive miRNA with a role in EGFR-tyrosine kinase inhibitor (TKI) resistance, miR-200c regulates the ALK pathway, and miR-22-3p regulates the MET pathway. The present paper also serves as an integrative work, highlighting the main cancer progression processes regulated by miRNAs, following these mutations. Specifically, we highlight the modulatory roles of miRNA in cancer cell survival and proliferation (miR-28, miR-30b/c), invasion and metastasis (miR-218, miR-182), neoangiogenesis (miR-29c), metabolic reprogramming (miR-124), and therapy resistance (miR-378, miR-328, miR-1244). The broad implications of miRNAs in lung cancer underline their potential real-world utility, as these entities can function as biomarkers for prognosis/diagnosis and even future therapeutic targets or agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b7c143b7d6b1d9c13099f482b9b7a0ba6ec7c9" target='_blank'>
              MicroRNAs and lung cancer: overview of essential pathways and somatic mutations in cancer progression
              </a>
            </td>
          <td>
            Andrei-Alexandru Tirpe, A. Nutu, Constantin Busuioc, Ovidiu-Laurean Pop, I. Berindan‐Neagoe
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Gliomas carry a high disease burden, particularly the aggressive IDH-wildtype subtype. However, some IDH-mutant gliomas can transition from a less aggressive G-CIMP-High (GCH) state to a more malignant G-CIMP-Low (GCL) phenotype, mimicking IDH-wildtype tumors. Understanding the cellular and molecular mechanisms behind this progression is vital for developing effective therapies.



 We propose that single-nuclei resolution analysis can reveal distinct neoplastic cell subtypes driving the GCH-to-GCL transition through activation of stemness- and proliferation-related gene programs.



 We analyzed 22 glioma specimens from 15 patients (including matched primary and recurrent tumors) using single-nuclei RNA- and ATAC-sequencing. After quality control and neoplastic cell identification via somatic CNVs, we applied non-negative matrix factorization (NMF) to identify cell subclusters based on gene expression patterns. We further characterized these clusters using gene set enrichment, protein-protein interaction networks, developmental trajectory analysis, and integration with external RNA-seq datasets. Chromatin accessibility was assessed via snATAC-seq to identify regulatory elements and transcription factor activity.



 We identified six major neoplastic cell subtypes: Astrocytic-Support, Pro-neural Housekeepers, Proteo-synthetic Cells, Mitotic Proliferators, Neural Guidance & Growth Cells, and Mesenchymal-mutant Cells. Each showed distinct gene expression signatures, pathway enrichments, and survival associations. Two subtypes—Mitotic Proliferators and Mesenchymal-mutant Cells—were linked to high-grade tumors and poor patient outcomes. Chromatin accessibility patterns corroborated transcriptional findings, highlighting differential TF regulation in aggressive subtypes.



 This study reveals two aggressive GCL-related neoplastic cell subtypes with distinct stem-like and proliferative features. These findings provide insight into glioma progression mechanisms and uncover potential molecular targets for therapeutic intervention in IDH-mutant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd05421cc91e3cb0411e5625733c706631851341" target='_blank'>
              EPCO-21. SINGLE-NUCLEI ANALYSIS IDENTIFIES NOVEL NEOPLASTIC IDH MUTANT ASTROCYTOMA PROLIFERATIVE AND STEMNESS PHENOTYPES DRIVING POOR PATIENT OUTCOMES
              </a>
            </td>
          <td>
            H. Noushmehr, G. Herrgott, L. Garofano, Ian Lee, J. Snyder, T. Walbert, A. Castro, A. Iavarone
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized cancer treatment, offering durable responses in a subset of patients. Despite this progress, cancer cells have developed sophisticated mechanisms to evade immune surveillance, including genetic mutations, activation of immune escape pathways, alterations in the tumor microenvironment, and metabolic reprogramming. Additionally, the immune system’s ability to effectively target cancer cells can be compromised by immune tolerance and functional exhaustion. There is growing evidence that abnormal activation of cancer signaling pathways, such as the Wnt/β-catenin, Notch, JAK/STAT, p53, and PTEN pathways, acts independently or through crosstalk to form complex regulatory networks that influence tumor immune escape, the therapeutic response, and immune resistance. Consequently, elucidating the mechanisms by which these signaling pathways contribute to tumor immune escape is essential for optimizing current immunotherapy regimens and developing novel therapeutic strategies. Targeting these pathways in combination with immunotherapy represents a promising approach to overcome immune resistance and enhance treatment efficacy. The integration of pathway modulation with immunotherapeutic interventions may reveal new avenues for more effective and durable cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142ff6a5cec86bc497f410ddb74aab33a5d7d49" target='_blank'>
              From intracellular drivers to immune modulators: emerging paradigms in oncogenic pathway-directed immunotherapy optimization
              </a>
            </td>
          <td>
            Dean Rao, Tiantian Wang, Chenan Fu, Yiming Luo, Junli Lu, Zhoubin Sun, Wei Yan, Huifang Liang, Limin Xia, Wenjie Huang
          </td>
          <td>2025-10-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe27e8860b31d75137bdc5403fdf3407806c753" target='_blank'>
              Global re-organisation of genome architecture at the transition to gametogenesis
              </a>
            </td>
          <td>
            Tien-Chi Huang, Maria Rigau, V. Malysheva, Chad Whildling, Stella Siciliani, Jingyu Li, Irina Balaguer Balsells, Pavel Artemov, Camille Dion, M. Spivakov, Juan M. Vaquerizas, Petra Hajkova
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous hematologic malignancy, characterized by complex molecular features and mechanisms of treatment resistance, which lead to a poor prognosis and high relapse rates. The complexity of multi-pathway interactions and the dysregulated dynamics of tumor cell death pathways may contribute to the wide range of clinical outcomes observed despite advancements in current therapies. Most current research focuses on a single form of cell death, neglecting the mechanisms of other death pathways and their synergistic interactions, which hinders the development of novel therapeutic approaches. The present review systematically integrates and compares the molecular features of key cell death modalities in AML, including autophagy, apoptosis, pyroptosis, necroptosis, ferroptosis and cuproptosis. The present review analyzes their specific triggers, signaling hubs and regulatory networks within the metabolic microenvironment, and discusses the dynamic crosstalk among these pathways. A key focus is the therapeutic potential of exploiting this crosstalk to design synergistic combination therapies. To overcome the limitations of conventional treatments and improve patient outcomes, it is essential to further investigate the transition mechanisms of various cell death modes in AML progression, drug resistance and relapse. Additionally, establishing a theoretical foundation for the development of innovative therapies that synergistically regulate multiple death pathways is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c85373acc9cc6f42ac985757b2e5e7f882fbf46" target='_blank'>
              Targeting cell death pathways in acute myeloid leukemia: Molecular mechanisms and clinical implications (Review)
              </a>
            </td>
          <td>
            Reaila Jianati, Haijing Chen, Xi Yang, Lixiang Yan, Yifei Guo, Chenyang Fan, Xiaogang Hao, Gengda Zhu, Zhexin Shi
          </td>
          <td>2025-10-14</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Genomic instability and high replication stress are hallmarks of multiple cancer types and offer a unique therapeutic vulnerability. Specifically, microsatellite instability-high (MSI-H) tumors exhibit a strong dependency on DHX9, an RNA helicase critical for genome maintenance and stress responses. Genetic silencing and phenotype rescue assays in MSI-H cancers show a strong dependency on the helicase activity of DHX9, making it an attractive, tumor-selective target. We present a potent, selective, orally bioavailable and brain-penetrant DHX9 helicase inhibitor, with best-in-class potential, for treating MSI-H and other genomically unstable cancers. STM11315 acts via allosteric inhibition and demonstrates high potency and selectivity in biochemical and cellular assays including DHX9-dependent RNA unwinding (IC50 <5 nM) and ATP-ase activity (IC50 <10 nM), with no effect (up to 30 uM) on 10 other helicases. Mechanistically, STM11315 induces replication stress, cell cycle arrest and apoptosis, leading to selective cell killing in MSI-H tumor cells (<20 nM in LS411N vs >10 uM in SW480 in a cell proliferation assay). STM11315 demonstrates robust single-agent efficacy in multiple MSI-H and other genomically unstable in vitro models, including colorectal, endometrial, and ovarian cancer cell lines. In vivo, STM11315 shows compelling single agent anti-tumor activity, achieving complete and durable tumor regression. The compound also induces dose-dependent induction of pharmacodynamic markers of target engagement and response, correlating with efficacy. Pharmacokinetic profiling confirms oral bioavailability, dose-linear plasma and brain exposure, and strong tumor penetration. Human dose predictions suggest daily oral dosing will give sustained 24h target coverage. In summary, our data support DHX9 inhibition as a promising therapeutic strategy for MSI-H and other molecularly defined genomically unstable cancers, addressing a significant unmet need with precision-targeted therapy. The selective mechanism, strong efficacy, favorable PK, and in vitro safety profile support progression of STM11315 into clinical development.



 Taran Khanam, Helen Harrison, Peter Astles, Elizabeth Anderson, Harry Wing, Angela Shibu, Giles Pergl-Wilson, Rita Chaouni, Richard Fosbeary, Tamsin Samuels, Natalie Webster, Roxine Staats, Beth Thomas, Oliver Rausch. Discovery of an allosteric, potent and selective DHX9 helicase inhibitor with best-in-class potential for the treatment of genomically unstable cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B053.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5266b3c2bf714e51c6e9f7e698674298596cf357" target='_blank'>
              Abstract B053: Discovery of an allosteric, potent and selective DHX9 helicase inhibitor with best-in-class potential for the treatment of genomically unstable cancers
              </a>
            </td>
          <td>
            Taran Khanam, Helen Harrison, Peter C Astles, Elizabeth Anderson, Harry Wing, Angela Shibu, Giles Pergl-Wilson, Rita Chaouni, R. Fosbeary, Tamsin Samuels, N. Webster, Roxine Staats, B. Thomas, O. Rausch
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is one of the most prevalent and lethal cancers worldwide, markedly contributing to cancer-related morbidity and mortality. The development and progression of lung cancer involve intricate signaling pathways that regulate essential cellular processes such as proliferation, survival, metastasis and resistance to therapy. These pathways can be disrupted by genetic mutations, epigenetic alterations or environmental factors including tobacco smoke and air pollution. In the past decade, considerable advancements have been made in elucidating the molecular mechanisms underlying lung cancer pathogenesis. This progress has facilitated the identification of key oncogenic drivers and the development of targeted therapies that have revolutionized treatment options. The present review provides an overview of critical signaling pathways implicated in lung cancer pathogenesis, including EGFR, anaplastic lymphoma kinase, KRAS, PI3K/AKT/mTOR and immune checkpoints. It also examines recent developments in targeted therapies such as next-generation tyrosine kinase inhibitors, monoclonal antibodies and combination strategies along with their clinical implications. By highlighting the molecular complexities associated with lung cancer and emerging therapeutic innovations, the present review highlights the significance of precision medicine in improving patient outcomes and shaping future treatments for lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de655ad32235c17234066e5339bc723f7a810b83" target='_blank'>
              Role of signaling pathways in lung cancer development and advances in targeted therapies (Review)
              </a>
            </td>
          <td>
            Zhang Tao, Weiwei Shao, Haiyang Zhou, Shujing Xia, Jie Zeng, Jing Ren, Xueyang Wang, Hua Zhu
          </td>
          <td>2025-10-14</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dbb38faf877fdc4bb2c148091abcd837c4061f" target='_blank'>
              Pharmacological targeting of HDAC/BET pathway enhances 5-FU efficacy in esophageal squamous cancer cells.
              </a>
            </td>
          <td>
            Xiaqing Xu, Qian Liu, Feifei Yang, Yiqing Zhang, Wanruo Yuan, Wenfang Gao, LiYing Ma, Qi Zhang
          </td>
          <td>2025-11-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Acute myeloid leukemia (AML) is paradigmatic for therapeutic resistance driven by genetic heterogeneity, epigenetic plasticity and microenvironmental protection. Over the past decade, six targeted or pathway-directed small molecules—midostaurin, gilteritinib, quizartinib, ivosidenib, enasidenib, venetoclax and glasdegib—have changed frontline and relapsed/refractory (R/R) practice in genomically defined subgroups or in patients unfit for intensive chemotherapy. Yet primary refractoriness and early relapse remain common, frequently
 via
 adaptive rewiring of apoptotic dependencies, clonal evolution and differentiation resistance. Here we integrate mechanistic insights with clinical evidence to: (i) map resistance biology onto targetable nodes (apoptosis control; signalling kinases; chromatin/lineage programmes; RNA splicing; DNA-damage response; nuclear export; niche adhesion and innate immune evasion); (ii) summarise the clinical trajectory and current limits of approved and emerging small molecules (including menin and LSD1 inhibitors); (iii) propose rules for rational doublets and triplets that are biologically orthogonal yet clinically tolerable; (iv) outline a regulatory timeline for key AML small molecules; and (v) prioritise where drug development should go next, including next-generation BH3 toolkits, clonal-pressure-aware designs, minimal residual disease (MRD)–adapted trials and therapy guided by dynamic functional profiling. The review closes with cross-platform challenges—myelosuppression, infectious risk, resistance monitoring and trial design—and a pragmatic framework for moving beyond incrementalism toward durable control and cure.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f83b4532e166032da2b89eea2b4afa25aa7b8b" target='_blank'>
              Acute myeloid leukemia drug resistance: targetable nodes and the clinical trajectory of small-molecule inhibitors
              </a>
            </td>
          <td>
            Xixi Zhang, Yao Peng, Yina Tian, Shichao Chen, Yijiu Jia, Mengtong Liu, Li Zhang
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="

 Breast cancer remains a major global health challenge because of limitations in early detection and therapeutic outcomes. This study employed bulk and single-cell RNA sequencing to investigate SUMOylation-associated molecular networks, aiming to identify prognostic biomarkers and potential therapeutic applications.




 Transcriptomic profiling was performed on 1,445 breast cancer and 113 normal samples to identify differentially expressed genes. Four hub genes,
 NR3C2
 ,
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , were prioritized using machine learning. Consensus clustering stratified patients into molecular subtypes based on the hub gene expression patterns. Differential immune infiltration analysis was used to evaluate 28 immune cell populations between the subtypes. Hub gene-immune cell interactions were visualized using bubble diagrams. Pharmacogenomic sensitivity profiling was performed using subtype-specific drug response data. Single-cell sequencing identified epithelial subclusters enriched for hub genes, and transcription factor networks were analyzed using SCENIC. Pan-cancer validation was performed to assess the oncogenic role of hub genes in 21 malignancies. Statistical significance was determined using the Student’s
 t
 -test (
 p
 < 0.0001).





 Tumor tissues exhibited significant upregulation of
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , whereas
 NR3C2
 was notably downregulated (
 p
 < 0.0001). Consensus clustering identified two distinct molecular subtypes: Subtype1, characterized by
 NR3C2
 upregulation and poorer prognosis, and Subtype2, distinguished by enhanced expression of
 CDCA8
 ,
 AURKA
 , and
 PLK1
 , correlating with favorable outcomes. Notably, PIK3CA mutations were prevalent in Subtype1, whereas
 TP53
 mutations dominated Subtype2. Immune infiltration profiles differed significantly between the two subtypes for most immune cell types. Pharmacogenomic assessments revealed distinct drug sensitivity profiles for each subtype in response to various therapeutic agents. A pan-cancer analysis of the four hub genes demonstrated consistent expression patterns, immune correlations, and prognostic associations across malignancies.




 Our findings reveal that SUMOylation subtypes in breast cancer exhibit distinct prognostic, immunological and pharmacogenomic profiles. These insights may provide candidate biomarkers for future personalized treatment strategies for breast cancer and potentially for other malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b694520cfc56236ad597a9d4d5eb2e2e73faafb" target='_blank'>
              Significance of SUMOylation in breast cancer progression: a comprehensive investigation using single-cell analysis and bioinformatics
              </a>
            </td>
          <td>
            Wenxing He, Zhengkui Sun, Dongmei Li, Tenghua Yu
          </td>
          <td>2025-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor cells often display aberrant chromosome counts, or aneuploidy, which can be exacerbated by increased chromosome missegregation rates in a condition called chromosomal instability (CIN). Despite the prevalence of CIN in solid tumors and its correlation to poor patient prognosis, understanding how CIN status affects cancer aggression remains a contested issue. In this work, we present a novel mathematical model linking missegregation levels and degree of aneuploidy to the underlying fitness landscape, and examine how these fitness landscapes can affect drug efficacy.



 To model how chromosome counts can shift over time, we use a partial differential equation that contains a diffusion term corresponding to chromosome missegregation and a growth term proportional to the relative fitness of a particular chromosomal distribution. Using this framework, a fitness landscape that connects chromosome count to relative fitness can be inferred from a steady-state cellular population. Fluorescence in situ hybridization (FISH) was used to track chromosome counts in both CIN and non-CIN cell lines over the course of 2 months, with data collected for chromosome 10 individually and for the group of chromosomes 1, 5, and 19. Experimental results were then analyzed using our mathematical framework to determine fitness landscapes for both CIN and non-CIN cells.



 A partial differential equation was developed to model how chromosome counts change over time based on a diffusion-like term corresponding to lagging chromosome rate and a growth rate based on a chromosome count’s contribution to overall cellular fitness. The CIN lines exhibited chromosome count distributions that were statistically closer to normal distributions than non-CIN lines. When connected to our mathematical model, these distributions imply that the CIN lines have a flatter fitness landscape than their non-CIN counterparts. This finding reinforces the notion that CIN lines are tolerant of highly varying degrees of aneuploidy.



 Our work suggests an underlying mechanism for the clinical observation that CIN tumors are more resilient to treatment with taxanes and other microtubule or mitotic targeting drugs. The flatter fitness landscapes associated with CIN provide an explanation for why these drugs are less effective, since increasing or decreasing the number of chromosomes in a CIN line does not lead to a relatively large change in fitness.



 Kelly Brock, Marianna Kleyman, Madison Stoddard, T. Ryan Gregory, Andrew Chen, Arijit Chakravarty. Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN) [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1879dcd4be6c24e75a32134afbe36fcff096e0" target='_blank'>
              Abstract C006: Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN)
              </a>
            </td>
          <td>
            Kelly Brock, Marianna Kleyman, M. Stoddard, T. R. Gregory, Andrew Chen, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The estimation of the in vitro potency of anticancer agents against tumor cell lines in tissue culture is known to be fraught with a high degree of variability. While quality control of experimental conditions has been demonstrated by others to reduce this variability, other sources of variability remain even after standardization of experimental conditions. In this poster, we assess two potential mechanisms for this residual variability: phenotypic stochasticity in response and heritable resistance to treatment.



 We developed three agent-based mathematical models representing different potential scenarios for cellular responses to anticancer therapy: (1) all cells have an equal probability of survival under treatment (phenotypic stochasticity) and future division; (2) sensitive cells deterministically die under treatment while resistant cells survive and may divide to produce resistant offspring (heritable resistance); (3) sensitive subclones have a greater probability of death under treatment compared to resistant subclones (both mechanisms). We implemented these models in MATLAB, running simulated in vitro experiments consisting of large (106) or small (500 or 5000) populations of virtual cells under treatment. The final size of these virtual cell populations serves as the readout of the virtual experiments. To assess the effect of phenotypic stochasticity and heritable resistance on variation in experimental outcomes, we simulated these experiments 10,000 times and report the coefficient of variation (CV).



 Variability in response to treatment was low under most simulated conditions. In the first model simulating phenotypic stochasticity alone, high coefficients of variation in assay outcome were observed only when the probability of survival under treatment was less than 10%. In simulations of deterministic heritable survival under treatment, the coefficient of variation was generally low. However, simulations incorporating both mechanisms exhibited substantial variability in experimental outcome, consistent with the observed phenomenon.



 Stochastic cellular response to treatment or heritable binary resistance alone are insufficient explanations for the variation in in vitro experimental results. On the other hand, the high degree of variation in experimental outcomes is consistent with a combination of heritable resistance and phenotypic stochasticity. Our findings are consistent with a role for Darwinian evolution in the response to anticancer treatment for cancer cells in tissue culture.



 Alexander L. Young, Liam Noronha, Madison Stoddard, T Ryan. Gregory, Arijit Chakravarty. Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of in vitro potency of anticancer agents [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a885f4ecedc46db5c7868acd715fccb8dde4f83" target='_blank'>
              Abstract C073: Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of
 in vitro
 potency of anticancer agents
              </a>
            </td>
          <td>
            Alexander L. Young, Liam Noronha, M. Stoddard, T. R. Gregory, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Advanced prostate cancer (PrCa) remains a leading cause of cancer-related death among men due to disease progression in nearly all patients on standard of care therapy targeting the androgen receptor. An important mechanism driving therapeutic resistance is lineage plasticity, which enables PrCa cells to reprogram into lineage variants no longer dependent on androgen receptor signaling. As inhibitors of the histone methyltransferase EZH2 are being evaluated clinically for the treatment of advanced PrCa, we investigated here how EZH2 affects PrCa lineage plasticity. Data from genetically engineered mice and human clinical samples demonstrated that genetic or pharmacological suppression of EZH2 altered chromatin to expand active transcription factor programs. These changes in gene expression during PrCa progression increased the diversity of PrCa lineage variants that arose. EZH2 suppression did not inhibit disease progression or therapeutic resistance in this context. These findings advance the current understanding of PrCa lineage plasticity and suggest EZH2 inhibitors may be less effective in treating PrCa prone to lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee66b00f883047cd2ed8c292d9f0f040e78b6ffe" target='_blank'>
              EZH2 Suppression Diversifies Prostate Cancer Lineage Variant Evolution and Lacks Efficacy in Inhibiting Disease Progression.
              </a>
            </td>
          <td>
            Justine J Jacobi, K. Wadosky, Neha Jaiswal, Xiaojing Zhang, Yanqing Wang, Prashant K Singh, Jie Wang, Eduardo Cortes Gomez, Jianmin Wang, Bo Xu, Mark D Long, Paloma Cejas, Shweta Kukreja, H. Long, David W. Goodrich
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer genome sequencing efforts have revealed hundreds of genes under positive selection, many of which are now being developed as therapeutic targets. However, positively selected mutations also populate our aging tissues in the absence of cancer. For most mutations, it is currently unknown whether they are recurrently found in cancer genomes because they cause cancer or because they expand during normal tissue evolution and are passively inherited. Here, we develop a mathematical and statistical framework that distinguishes these two factors. We discover – across thousands of cancer and normal tissue genomes – that mutations that most strongly increase cancer risk are enriched in younger patients’ cancers, whereas mutations that are positively selected in normal tissue without causing cancer are enriched in older patients. Focusing on a particularly data-rich cancer type, acute myeloid leukemia, we show that genetic differences between young- and adult-onset cancers can largely be explained by the cumulative effects of normal tissue evolution, contradicting the long-standing notion that childhood cancers require a distinct set of causal mutations. Our framework establishes patient age as a powerful resource for clarifying whether positively selected mutations in cancer genomes are truly disease-promoting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56a3d728c459f75967661fea1d9baa92b4ef2796" target='_blank'>
              Age distinguishes selection from causation in cancer genomes
              </a>
            </td>
          <td>
            David Cheek, Martin Blohmer, Martin A. Nowak, Tibor Antal, Kamila Naxerova
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [2, 15, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>