---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-03 09:27:45 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edff12604e8d13a7b581bfb7b7624565745ae39" target='_blank'>Machine learning-based extrachromosomal DNA identification in large-scale cohorts reveals its clinical implications in cancer</a></td>
          <td>
            Shixiang Wang, Chen-Yi Wu, M. He, Jia-Xin Yong, Yanxing Chen, Li-Mei Qian, Jin-Ling Zhang, Zhao-Lei Zeng, Rui-Hua Xu, Feng Wang, Qi Zhao
          </td>
          <td>2024-02-19</td>
          <td>Nature Communications</td>
          <td>5</td>
          <td>24</td>
        </tr>
    
        <tr id="Focal amplifications in the cancer genome, particularly extrachromosomal DNA (ecDNA) amplifications, are emerging as a pivotal event in cancer progression across diverse cancer contexts, presenting a paradigm shift in our understanding of tumor dynamics. Simultaneously, identification of the various modes of focal amplifications is bioinformatically challenging. We present AmpliconSuite, a collection of tools that enables robust identification of focal amplifications from whole-genome sequencing data. AmpliconSuite includes AmpliconSuite- pipeline; utilizing the AmpliconArchitect (AA) method, and AmpliconRepository.org; a community- editable website for the sharing of focal amplification calls. We also describe improvements made to AA since its initial release that improve its accuracy and speed. As a proof of principle, we utilized publicly available pan-cancer datasets encompassing 2,525 tumor samples hosted on AmpliconRepository.org to illustrate important properties of focal amplifications, showing ecDNA has higher copy number, and stronger oncogene enrichment, compared to other classes of focal amplifications. Finally, we illustrate how AmpliconSuite-pipeline enables delineation of the various mechanisms by which ecDNA forms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3f112dd01867d814b393cd8a3b0372c553a87b" target='_blank'>AmpliconSuite: an end-to-end workflow for analyzing focal amplifications in cancer genomes</a></td>
          <td>
            J. Luebeck, Edwin Huang, Forrest Kim, T. Liefeld, Bhargavi Dameracharla, Rohil Ahuja, Daniel Schreyer, Gino Prasad, Michał Adamaszek, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Thorin Tabor, M. Giurgiu, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>
    
        <tr id="Oncogene amplification, a major driver of cancer pathogenicity, is often mediated through focal amplification of genomic segments. Recent results implicate extrachromosomal DNA (ecDNA) as the primary mechanism driving focal copy number amplification (fCNA) - enabling gene amplification, rapid tumor evolution, and the rewiring of regulatory circuitry. Resolving an fCNA’s structure is a first step in deciphering the mechanisms of its genesis and the subsequent biological consequences. Here, we introduce a powerful new computational method, AmpliconReconstructor (AR), for integrating optical mapping (OM) of long DNA fragments (>150kb) with next-generation sequencing (NGS) to resolve fCNAs at single-nucleotide resolution. AR uses an NGS-derived breakpoint graph alongside OM scaffolds to produce high-fidelity reconstructions. After validating performance by extensive simulations, we used AR to reconstruct fCNAs in seven cancer cell lines to reveal the complex architecture of ecDNA, breakage-fusion-bridge cycles, and other complex rearrangements. By distinguishing between chromosomal and extrachromosomal origins, and by reconstructing the rearrangement signatures associated with a given fCNA’s generative mechanism, AR enables a more thorough understanding of the origins of fCNAs, and their functional consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28a4561b57e335c34ee371c948fa26b8b381895" target='_blank'>AmpliconReconstructor: Integrated analysis of NGS and optical mapping resolves the complex structures of focal amplifications in cancer</a></td>
          <td>
            J. Luebeck, C. Coruh, S. R. Dehkordi, Joshua T. Lange, Kristen M. Turner, Viraj Deshpande, Dave A. Pai, Chao Zhang, Utkrisht Rajkumar, Julie A. Law, P. Mischel, V. Bafna
          </td>
          <td>2020-01-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>103</td>
        </tr>
    
        <tr id="Abstract Cancer cells progressively evolve from a premalignant to a malignant state, which is driven by accumulating somatic alterations that confer normal cells a fitness advantage. Improvements in high-throughput sequencing techniques have led to an increase in construction of tumor phylogenetics and identification of somatic driver events that specifically occurred in different tumor progression stages. Here, we developed the SEECancer database (http://biocc.hrbmu.edu.cn/SEECancer), which aims to present the comprehensive cancer evolutionary stage-specific somatic events (including early-specific, late-specific, relapse-specific, metastasis-specific, drug-resistant and drug-induced genomic events) and their temporal orders. By manually curating over 10 000 published articles, 1231 evolutionary stage-specific genomic events and 5772 temporal orders involving 82 human cancers and 23 tissue origins were collected and deposited in the SEECancer database. Each entry contains the somatic event, evolutionary stage, cancer type, detection approach and relevant evidence. SEECancer provides a user-friendly interface for browsing, searching and downloading evolutionary stage-specific somatic events and temporal relationships in various cancers. With increasing attention on cancer genome evolution, the necessary information in SEECancer will facilitate understanding of cancer etiology and development of evolutionary therapeutics, and help clinicians to discover biomarkers for monitoring tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c50ee01be431fdb0fc32d3a19bb3f5d27bc210c" target='_blank'>SEECancer: a resource for somatic events in evolution of cancer genome</a></td>
          <td>
            Hongyi Zhang, Shangyi Luo, Xinxin Zhang, Jianlong Liao, Fei Quan, E. Zhao, Chenfen Zhou, Fulong Yu, Wenkang Yin, Yunpeng Zhang, Yun Xiao, Xia Li
          </td>
          <td>2017-10-23</td>
          <td>Nucleic Acids Research</td>
          <td>7</td>
          <td>45</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4438021251d9b711c0baeb201445a5a06e8e0" target='_blank'>Methods, bioinformatics tools and databases in ecDNA research: An overview</a></td>
          <td>
            Xinyu Zhao, Huan Zhao, Yupeng Liu, Zhiyun Guo
          </td>
          <td>2023-11-01</td>
          <td>Computers in biology and medicine</td>
          <td>4</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe659e0001f59efd1d4924d1559cee126080ae5" target='_blank'>AmpliconReconstructor integrates NGS and optical mapping to resolve the complex structures of focal amplifications</a></td>
          <td>
            J. Luebeck, C. Coruh, S. R. Dehkordi, Joshua T. Lange, Kristen M. Turner, Viraj Deshpande, Dave A. Pai, Chao Zhang, Utkrisht Rajkumar, Julie A. Law, P. Mischel, V. Bafna
          </td>
          <td>2020-09-01</td>
          <td>Nature Communications</td>
          <td>59</td>
          <td>103</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd217dda14bc5faced17ed3b56a0c56548ca85ad" target='_blank'>ATACAmp: a tool for detecting ecDNA/HSRs from bulk and single-cell ATAC-seq data</a></td>
          <td>
            Hansen Cheng, Wenhao Ma, Kun Wang, Han Chu, Guangchao Bao, Yu Liao, Yawen Yuan, Yixiong Gou, Liting Dong, Jian Yang, Haoyang Cai
          </td>
          <td>2023-11-10</td>
          <td>BMC Genomics</td>
          <td>5</td>
          <td>14</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cdfe871f2a906538ca54506da314022480cca0" target='_blank'>ECGA: A web server to explore and analyze extrachromosomal gene in cancer</a></td>
          <td>
            Nan Zhou, Li Peng, Zhiyu Zhang, Qiqi Luo, Huiran Sun, Jinku Bao, Yuping Ning, Xiaoqing Yuan
          </td>
          <td>2024-11-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr id="The Hi-C technique has been shown to be a promising method to detect structural variations (SVs) in human genomes. However, algorithms that can use Hi-C data for a full-range SV detection have been severely lacking. Current methods can only identify interchromosomal translocations and long-range intrachromosomal SVs (>1 Mb) at less-than-optimal resolution. Therefore, we develop EagleC, a framework that combines deep-learning and ensemble-learning strategies to predict a full range of SVs at high resolution. We show that EagleC can uniquely capture a set of fusion genes that are missed by whole-genome sequencing or nanopore. Furthermore, EagleC also effectively captures SVs in other chromatin interaction platforms, such as HiChIP, Chromatin interaction analysis with paired-end tag sequencing (ChIA-PET), and capture Hi-C. We apply EagleC in more than 100 cancer cell lines and primary tumors and identify a valuable set of high-quality SVs. Last, we demonstrate that EagleC can be applied to single-cell Hi-C and used to study the SV heterogeneity in primary tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c1dd3757008f0744292fa9bfc0f7c36e71ba63" target='_blank'>EagleC: A deep-learning framework for detecting a full range of structural variations from bulk and single-cell contact maps</a></td>
          <td>
            Xiaotao Wang, Yu Luan, Feng Yue
          </td>
          <td>2022-06-01</td>
          <td>Science Advances</td>
          <td>55</td>
          <td>34</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a64951ab3de56b0af72adea21a1af02157560a" target='_blank'>Tracing cancer evolution and heterogeneity using Hi-C</a></td>
          <td>
            Dan D. Erdmann-Pham, S. S. Batra, Timothy K. Turkalo, James Durbin, M. Blanchette, I. Yeh, Hunter Shain, B. Bastian, Yun S. Song, D. Rokhsar, D. Hockemeyer
          </td>
          <td>2023-02-21</td>
          <td>Nature Communications</td>
          <td>7</td>
          <td>105</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>