---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-11 09:34:34 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="The clinical implications of extrachromosomal DNA (ecDNA) in cancer therapy remain largely elusive. Here, we present a comprehensive analysis of ecDNA amplification spectra and their association with clinical and molecular features in multiple cohorts comprising over 13,000 pan-cancer patients. Using our developed computational framework, GCAP, and validating it with multifaceted approaches, we reveal a consistent pan-cancer pattern of mutual exclusivity between ecDNA amplification and microsatellite instability (MSI). In addition, we establish the role of ecDNA amplification as a risk factor and refine genomic subtypes in a cohort from 1015 colorectal cancer patients. Importantly, our investigation incorporates data from four clinical trials focused on anti-PD-1 immunotherapy, demonstrating the pivotal role of ecDNA amplification as a biomarker for guiding checkpoint blockade immunotherapy in gastrointestinal cancer. This finding represents clinical evidence linking ecDNA amplification to the effectiveness of immunotherapeutic interventions. Overall, our study provides a proof-of-concept of identifying ecDNA amplification from cancer whole-exome sequencing (WES) data, highlighting the potential of ecDNA amplification as a valuable biomarker for facilitating personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edff12604e8d13a7b581bfb7b7624565745ae39" target='_blank'>Machine learning-based extrachromosomal DNA identification in large-scale cohorts reveals its clinical implications in cancer</a></td>
          <td>
            Shixiang Wang, Chen-Yi Wu, M. He, Jia-Xin Yong, Yanxing Chen, Li-Mei Qian, Jin-Ling Zhang, Zhao-Lei Zeng, Rui-Hua Xu, Feng Wang, Qi Zhao
          </td>
          <td>2024-02-19</td>
          <td>Nature Communications</td>
          <td>8</td>
          <td>27</td>
        </tr>
    
        <tr id="Focal amplifications in the cancer genome, particularly extrachromosomal DNA (ecDNA) amplifications, are emerging as a pivotal event in cancer progression across diverse cancer contexts, presenting a paradigm shift in our understanding of tumor dynamics. Simultaneously, identification of the various modes of focal amplifications is bioinformatically challenging. We present AmpliconSuite, a collection of tools that enables robust identification of focal amplifications from whole-genome sequencing data. AmpliconSuite includes AmpliconSuite- pipeline; utilizing the AmpliconArchitect (AA) method, and AmpliconRepository.org; a community- editable website for the sharing of focal amplification calls. We also describe improvements made to AA since its initial release that improve its accuracy and speed. As a proof of principle, we utilized publicly available pan-cancer datasets encompassing 2,525 tumor samples hosted on AmpliconRepository.org to illustrate important properties of focal amplifications, showing ecDNA has higher copy number, and stronger oncogene enrichment, compared to other classes of focal amplifications. Finally, we illustrate how AmpliconSuite-pipeline enables delineation of the various mechanisms by which ecDNA forms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3f112dd01867d814b393cd8a3b0372c553a87b" target='_blank'>AmpliconSuite: an end-to-end workflow for analyzing focal amplifications in cancer genomes</a></td>
          <td>
            J. Luebeck, Edwin Huang, Forrest Kim, T. Liefeld, Bhargavi Dameracharla, Rohil Ahuja, Daniel Schreyer, Gino Prasad, Michał Adamaszek, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Thorin Tabor, M. Giurgiu, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>73</td>
        </tr>
    
        <tr id="Oncogene amplification, a major driver of cancer pathogenicity, is often mediated through focal amplification of genomic segments. Recent results implicate extrachromosomal DNA (ecDNA) as the primary mechanism driving focal copy number amplification (fCNA) - enabling gene amplification, rapid tumor evolution, and the rewiring of regulatory circuitry. Resolving an fCNA’s structure is a first step in deciphering the mechanisms of its genesis and the subsequent biological consequences. Here, we introduce a powerful new computational method, AmpliconReconstructor (AR), for integrating optical mapping (OM) of long DNA fragments (>150kb) with next-generation sequencing (NGS) to resolve fCNAs at single-nucleotide resolution. AR uses an NGS-derived breakpoint graph alongside OM scaffolds to produce high-fidelity reconstructions. After validating performance by extensive simulations, we used AR to reconstruct fCNAs in seven cancer cell lines to reveal the complex architecture of ecDNA, breakage-fusion-bridge cycles, and other complex rearrangements. By distinguishing between chromosomal and extrachromosomal origins, and by reconstructing the rearrangement signatures associated with a given fCNA’s generative mechanism, AR enables a more thorough understanding of the origins of fCNAs, and their functional consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28a4561b57e335c34ee371c948fa26b8b381895" target='_blank'>AmpliconReconstructor: Integrated analysis of NGS and optical mapping resolves the complex structures of focal amplifications in cancer</a></td>
          <td>
            J. Luebeck, C. Coruh, S. R. Dehkordi, Joshua T. Lange, Kristen M. Turner, Viraj Deshpande, Dave A. Pai, Chao Zhang, Utkrisht Rajkumar, Julie A. Law, P. Mischel, V. Bafna
          </td>
          <td>2020-01-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>104</td>
        </tr>
    
        <tr id="Abstract Cancer cells progressively evolve from a premalignant to a malignant state, which is driven by accumulating somatic alterations that confer normal cells a fitness advantage. Improvements in high-throughput sequencing techniques have led to an increase in construction of tumor phylogenetics and identification of somatic driver events that specifically occurred in different tumor progression stages. Here, we developed the SEECancer database (http://biocc.hrbmu.edu.cn/SEECancer), which aims to present the comprehensive cancer evolutionary stage-specific somatic events (including early-specific, late-specific, relapse-specific, metastasis-specific, drug-resistant and drug-induced genomic events) and their temporal orders. By manually curating over 10 000 published articles, 1231 evolutionary stage-specific genomic events and 5772 temporal orders involving 82 human cancers and 23 tissue origins were collected and deposited in the SEECancer database. Each entry contains the somatic event, evolutionary stage, cancer type, detection approach and relevant evidence. SEECancer provides a user-friendly interface for browsing, searching and downloading evolutionary stage-specific somatic events and temporal relationships in various cancers. With increasing attention on cancer genome evolution, the necessary information in SEECancer will facilitate understanding of cancer etiology and development of evolutionary therapeutics, and help clinicians to discover biomarkers for monitoring tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c50ee01be431fdb0fc32d3a19bb3f5d27bc210c" target='_blank'>SEECancer: a resource for somatic events in evolution of cancer genome</a></td>
          <td>
            Hongyi Zhang, Shangyi Luo, Xinxin Zhang, Jianlong Liao, Fei Quan, E. Zhao, Chenfen Zhou, Fulong Yu, Wenkang Yin, Yunpeng Zhang, Yun Xiao, Xia Li
          </td>
          <td>2017-10-23</td>
          <td>Nucleic Acids Research</td>
          <td>7</td>
          <td>46</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4438021251d9b711c0baeb201445a5a06e8e0" target='_blank'>Methods, bioinformatics tools and databases in ecDNA research: An overview</a></td>
          <td>
            Xinyu Zhao, Huan Zhao, Yupeng Liu, Zhiyun Guo
          </td>
          <td>2023-11-01</td>
          <td>Computers in biology and medicine</td>
          <td>6</td>
          <td>3</td>
        </tr>
    
        <tr id="Oncogene amplification, a major driver of cancer pathogenicity, is often mediated through focal amplification of genomic segments. Recent results implicate extrachromosomal DNA (ecDNA) as the primary driver of focal copy number amplification (fCNA) - enabling gene amplification, rapid tumor evolution, and the rewiring of regulatory circuitry. Resolving an fCNA’s structure is a first step in deciphering the mechanisms of its genesis and the fCNA’s subsequent biological consequences. We introduce a computational method, AmpliconReconstructor (AR), for integrating optical mapping (OM) of long DNA fragments (>150 kb) with next-generation sequencing (NGS) to resolve fCNAs at single-nucleotide resolution. AR uses an NGS-derived breakpoint graph alongside OM scaffolds to produce high-fidelity reconstructions. After validating its performance through multiple simulation strategies, AR reconstructed fCNAs in seven cancer cell lines to reveal the complex architecture of ecDNA, a breakage-fusion-bridge and other complex rearrangements. By reconstructing the rearrangement signatures associated with an fCNA’s generative mechanism, AR enables a more thorough understanding of the origins of fCNAs. Focal copy number amplifications (fCNAs), which drive cancer pathogenicity, arise by a number of mechanisms and can be challenging to call. Here the authors present AmpliconReconstructor for precise and scalable fCNA reconstruction using optical mapping and next-generation sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe659e0001f59efd1d4924d1559cee126080ae5" target='_blank'>AmpliconReconstructor integrates NGS and optical mapping to resolve the complex structures of focal amplifications</a></td>
          <td>
            J. Luebeck, C. Coruh, S. R. Dehkordi, Joshua T. Lange, Kristen M. Turner, Viraj Deshpande, Dave A. Pai, Chao Zhang, Utkrisht Rajkumar, Julie A. Law, P. Mischel, V. Bafna
          </td>
          <td>2020-09-01</td>
          <td>Nature Communications</td>
          <td>65</td>
          <td>104</td>
        </tr>
    
        <tr id="Background High oncogene expression in cancer cells is a major cause of rapid tumor progression and drug resistance. Recent cancer genome research has shown that oncogenes as well as regulatory elements can be amplified in the form of extrachromosomal DNA (ecDNA) or subsequently integrated into chromosomes as homogeneously staining regions (HSRs). These genome-level variants lead to the overexpression of the corresponding oncogenes, resulting in poor prognosis. Most existing detection methods identify ecDNA using whole genome sequencing (WGS) data. However, these techniques usually detect many false positive regions owing to chromosomal DNA interference. Results In the present study, an algorithm called “ATACAmp” that can identify ecDNA/HSRs in tumor genomes using ATAC-seq data has been described. High chromatin accessibility, one of the characteristics of ecDNA, makes ATAC-seq naturally enriched in ecDNA and reduces chromosomal DNA interference. The algorithm was validated using ATAC-seq data from cell lines that have been experimentally determined to contain ecDNA regions. ATACAmp accurately identified the majority of validated ecDNA regions. AmpliconArchitect, the widely used ecDNA detecting tool, was used to detect ecDNA regions based on the WGS data of the same cell lines. Additionally, the Circle-finder software, another tool that utilizes ATAC-seq data, was assessed. The results showed that ATACAmp exhibited higher accuracy than AmpliconArchitect and Circle-finder. Moreover, ATACAmp supported the analysis of single-cell ATAC-seq data, which linked ecDNA to specific cells. Conclusions ATACAmp, written in Python, is freely available on GitHub under the MIT license: https://github.com/chsmiss/ATAC-amp . Using ATAC-seq data, ATACAmp offers a novel analytical approach that is distinct from the conventional use of WGS data. Thus, this method has the potential to reduce the cost and technical complexity associated ecDNA analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd217dda14bc5faced17ed3b56a0c56548ca85ad" target='_blank'>ATACAmp: a tool for detecting ecDNA/HSRs from bulk and single-cell ATAC-seq data</a></td>
          <td>
            Hansen Cheng, Wenhao Ma, Kun Wang, Han Chu, Guangchao Bao, Yu Liao, Yawen Yuan, Yixiong Gou, Liting Dong, Jian Yang, Haoyang Cai
          </td>
          <td>2023-11-10</td>
          <td>BMC Genomics</td>
          <td>5</td>
          <td>15</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cdfe871f2a906538ca54506da314022480cca0" target='_blank'>ECGA: A web server to explore and analyze extrachromosomal gene in cancer</a></td>
          <td>
            Nan Zhou, Li Peng, Zhiyu Zhang, Qiqi Luo, Huiran Sun, Jinku Bao, Yuping Ning, Xiaoqing Yuan
          </td>
          <td>2024-11-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr id="
 Extrachromosomal DNA (ecDNA) are circular, megabase-scale regions of the genome found outside canonical chromosomes and characterized by high copy-number and frequent oncogene amplification. They are implicated in tumorigenesis, therapeutic resistance, and shortened survival for patients across multiple cancer types. Existing tools to identify and predict ecDNA structures are limited to low throughput, high sensitivity enrichment-based assays or high throughput, low sensitivity computational methods using whole-genome sequencing data. Recent advances in long-read sequencing offer potential improvements in ecDNA detection but such data are currently limited. Evaluating the impact of ecDNA in relation to clinical characteristics requires large cohorts such as The Cancer Genome Atlas (TCGA) or the International Cancer Genome Consortium (ICGC). Hence, there is value in leveraging short-read sequencing data from these existing cohorts while improving confidence in computational predictions of ecDNA presence and structure. Here we present ecDNAInspector, a computational tool that enables systematic selection of high-confidence ecDNA predictions from sequencing data. Each ecDNA prediction is characterized using structure-based metrics: an extreme cycle size Boolean, a breakpoint mapping error Boolean, and adapted classification metrics that assess the frequency of paired ends between neighboring cycle segments with orthogonal structural variant call support. ecDNA predictions are clustered by these metrics and the user can select high-confidence groups. A subsequent filtering step enables rescue of additional ecDNA predictions based on user-defined thresholds. After selection of a final high-confidence group of ecDNA predictions, ecDNAInspector offers modules for structural comparison and analysis of ecDNA within a cohort, including inspection of structural conservation (i.e. maintenance of similar sequence) and feature comparisons across tumors. Applying ecDNAInspector to a breast cancer cohort, we nominated a subset of high-confidence ecDNA with strong orthogonal structural variant support and additional indicators of high quality. We identified both base pair and oncogene inclusion conservation within breast cancer subtype groups, and noted significant variations in structural metrics across subtype groups. Since structure often indicates origin and dictates function, these differences suggest potential biological differences in ecDNA genesis and evolution across breast cancer subtypes. Thus, ecDNAInspector provides a systematic method for selecting high-confidence ecDNA predictions for downstream structural analysis, enabling improved utilization of existing sequencing data in valuable clinical cohorts.
 
 
 Sophia J. Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis. ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3754.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis</a></td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="The Hi-C technique has been shown to be a promising method to detect structural variations (SVs) in human genomes. However, algorithms that can use Hi-C data for a full-range SV detection have been severely lacking. Current methods can only identify interchromosomal translocations and long-range intrachromosomal SVs (>1 Mb) at less-than-optimal resolution. Therefore, we develop EagleC, a framework that combines deep-learning and ensemble-learning strategies to predict a full range of SVs at high resolution. We show that EagleC can uniquely capture a set of fusion genes that are missed by whole-genome sequencing or nanopore. Furthermore, EagleC also effectively captures SVs in other chromatin interaction platforms, such as HiChIP, Chromatin interaction analysis with paired-end tag sequencing (ChIA-PET), and capture Hi-C. We apply EagleC in more than 100 cancer cell lines and primary tumors and identify a valuable set of high-quality SVs. Last, we demonstrate that EagleC can be applied to single-cell Hi-C and used to study the SV heterogeneity in primary tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c1dd3757008f0744292fa9bfc0f7c36e71ba63" target='_blank'>EagleC: A deep-learning framework for detecting a full range of structural variations from bulk and single-cell contact maps</a></td>
          <td>
            Xiaotao Wang, Yu Luan, Feng Yue
          </td>
          <td>2022-06-01</td>
          <td>Science Advances</td>
          <td>63</td>
          <td>35</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>