---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-21 06:32:48 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Genomes mutate and evolve in ways simple (substitution or deletion of bases) and complex (e.g. chromosome shattering). We do not fully understand what types of complex mutation occur, and we cannot routinely characterize arbitrarily-complex mutations in a high-throughput, genome-wide manner. Long-read DNA sequencing methods (e.g. PacBio, nanopore) are promising for this task, because one read may encompass a whole complex mutation. We describe an analysis pipeline to characterize arbitrarily-complex “local” mutations, i.e. intrachromosomal mutations encompassed by one DNA read. We apply it to nanopore and PacBio reads from one human cell line (NA12878), and survey sequence rearrangements, both real and artifactual. Almost all the real rearrangements belong to recurring patterns or motifs: the most common is tandem multiplication (e.g. heptuplication), but there are also complex patterns such as localized shattering, which resembles DNA damage by radiation. Gene conversions are identified, including one between hemoglobin gamma genes. This study demonstrates a way to find intricate rearrangements with any number of duplications, deletions, and repositionings. It demonstrates a probability-based method to resolve ambiguous rearrangements involving highly similar sequences, as occurs in gene conversion. We present a catalog of local rearrangements in one human cell line, and show which rearrangement patterns occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d30e28a0875939246fb9dd82670af95bdd6835" target='_blank'>A bestiary of localized sequence rearrangements in human DNA</a></td>
          <td>
            M. Frith, Sofia Khan
          </td>
          <td>2017-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>
    
        <tr id="Abstract Genomes mutate and evolve in ways simple (substitution or deletion of bases) and complex (e.g. chromosome shattering). We do not fully understand what types of complex mutation occur, and we cannot routinely characterize arbitrarily-complex mutations in a high-throughput, genome-wide manner. Long-read DNA sequencing methods (e.g. PacBio, nanopore) are promising for this task, because one read may encompass a whole complex mutation. We describe an analysis pipeline to characterize arbitrarily-complex ‘local’ mutations, i.e. intrachromosomal mutations encompassed by one DNA read. We apply it to nanopore and PacBio reads from one human cell line (NA12878), and survey sequence rearrangements, both real and artifactual. Almost all the real rearrangements belong to recurring patterns or motifs: the most common is tandem multiplication (e.g. heptuplication), but there are also complex patterns such as localized shattering, which resembles DNA damage by radiation. Gene conversions are identified, including one between hemoglobin gamma genes. This study demonstrates a way to find intricate rearrangements with any number of duplications, deletions, and repositionings. It demonstrates a probability-based method to resolve ambiguous rearrangements involving highly similar sequences, as occurs in gene conversion. We present a catalog of local rearrangements in one human cell line, and show which rearrangement patterns occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b913c0fa64d62b116a38e6432118cedf5fbd602" target='_blank'>A survey of localized sequence rearrangements in human DNA</a></td>
          <td>
            M. Frith, Sofia Khan
          </td>
          <td>2017-12-19</td>
          <td>Nucleic Acids Research</td>
          <td>18</td>
          <td>47</td>
        </tr>
    
        <tr id="License: This open access article is licenced under Creative Commons Attribution-NonCommercial 4.0 International (CC BY-NC 4.0) https://creativecommons.org/licenses/by-nc/4.0/ Abstract: Large-scale structural chromosomal rearrangements or structural variants, such as insertions, deletions, translocations, and inversions may result in the exchange of coding or regulatory DNA/RNA sequences between genes, which can lead to gene fusion or the loss/gain of genetic material. Gene fusion events are common in multiple types of cancer. High-throughput DNA and RNA sequencing methods produce large amounts of genomic data. Due to the massive amounts of data and the fact that structural variants account for just a small fraction of the data, efficient and accurate search methods are required for the detection of chromosomal breakpoints and structural variations. Robust identification of structural variants remains paramount for accurate inference of long-range interactions from high-throughput chromosome conformation capture (Hi-C) data. This chapter is a survey of computational methods based on paired end reading, efficient search techniques and parallel computing to detect structural variants in both whole genome and transcriptome sequences, as well as Hi-C data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796256f0f6211b49465190428cac947392397c1f" target='_blank'>Computational Methods for Detecting Large-Scale Structural Rearrangements in Chromosomes</a></td>
          <td>
            Muneeba Jilani, Nurit Haspel
          </td>
          <td>2021-03-18</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>17</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4438021251d9b711c0baeb201445a5a06e8e0" target='_blank'>Methods, bioinformatics tools and databases in ecDNA research: An overview</a></td>
          <td>
            Xinyu Zhao, Huan Zhao, Yupeng Liu, Zhiyun Guo
          </td>
          <td>2023-11-01</td>
          <td>Computers in biology and medicine</td>
          <td>2</td>
          <td>2</td>
        </tr>
    
        <tr id="Oncogene amplification, a major driver of cancer pathogenicity, is often mediated through focal amplification of genomic segments. Recent results implicate extrachromosomal DNA (ecDNA) as the primary mechanism driving focal copy number amplification (fCNA) - enabling gene amplification, rapid tumor evolution, and the rewiring of regulatory circuitry. Resolving an fCNA’s structure is a first step in deciphering the mechanisms of its genesis and the subsequent biological consequences. Here, we introduce a powerful new computational method, AmpliconReconstructor (AR), for integrating optical mapping (OM) of long DNA fragments (>150kb) with next-generation sequencing (NGS) to resolve fCNAs at single-nucleotide resolution. AR uses an NGS-derived breakpoint graph alongside OM scaffolds to produce high-fidelity reconstructions. After validating performance by extensive simulations, we used AR to reconstruct fCNAs in seven cancer cell lines to reveal the complex architecture of ecDNA, breakage-fusion-bridge cycles, and other complex rearrangements. By distinguishing between chromosomal and extrachromosomal origins, and by reconstructing the rearrangement signatures associated with a given fCNA’s generative mechanism, AR enables a more thorough understanding of the origins of fCNAs, and their functional consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28a4561b57e335c34ee371c948fa26b8b381895" target='_blank'>AmpliconReconstructor: Integrated analysis of NGS and optical mapping resolves the complex structures of focal amplifications in cancer</a></td>
          <td>
            J. Luebeck, C. Coruh, S. R. Dehkordi, Joshua T. Lange, Kristen M. Turner, Viraj Deshpande, Dave A. Pai, Chao Zhang, Utkrisht Rajkumar, Julie A. Law, P. Mischel, V. Bafna
          </td>
          <td>2020-01-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>102</td>
        </tr>
    
        <tr id="Structural variations caused by gene fusion represent a major class of somatically acquired variations in human malignancies, and include deletions, inversions, and translocations. Short fragmented reads are the main source of data from 2nd-generation sequencing, and detecting structural variations from this type of data is different from that of 1st-generation sequencing, where the read length is much longer. Current detection methods are low in specificity and are inefficient. We developed a hybrid algorithm, FirstSV, to meet the clinical demand for fast and accurate structural variation detection. Its main features include cluster analysis, realignment, and local assembly. FirstSV was validated with simulated data, with data from real patient samples, with data from standard testing samples, and with downloaded public data sets. FirstSV outperforms public-available methods in terms of sensitivity, precision, and operational efficiency. FirstSV is freely available at https://github.com/shenjia1/FirstSV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e83037a04251be21933ad412b41f6b3e3b0f34e" target='_blank'>FirstSV: Fast and Accurate Approach of Structural Variations Detection for Short DNA fragments</a></td>
          <td>
            Jiamin Shen, Qiyang Zuo, Rongliang Wang, Xiang Li, Yuanhua Tang
          </td>
          <td>2018-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr id="Background The structural abnormalities in chromosomes are important issues in cancer genomics. Next generation sequencing technologies have big potentials to detect the structural variations precisely and comprehensively. Nevertheless, it is still difficult problem to detect large structural variations from short read sequence data. Major efforts have been achieved with paired-end reads, since discordant pairs directly reflect the existence of large rearrangement. Furthermore, approaches to detect structural variations from single-end reads are still worthwhile challenge because they allow wide choices of sequencing platforms. Result We present ReALLEN, a series of tools to detect genomic rearrangement with base-pair resolution from single-end reads provided by next generation sequencing. We examined the performance of ReALLEN using simulated dataset and real dataset sequenced by Ion Torrent systems. In most cases on the simulated dataset, ReALLEN showed nearly 100% precision and better sensitivity than other major tools. Notably, ReALLEN showed stable scores even if it was on some unfavorable conditions, for example, low coverage or small variant size. On the real dataset sequenced by Ion Torrent systems, ReALLEN accurately found an insertional translocation that was crucial for the diagnosis of chronic myeloid leukemia. Conclusion ReALLEN is useful to researchers in finding genomic rearrangements. It will contribute to discovery of cancer-specific fusion proteins, precise diagnosis of known types of cancers, and understanding of genetic diseases caused by abnormal chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773d766e7fbe8008402447c3eb4d3348535b9228" target='_blank'>ReALLEN: structural variation discovery in cancer genome by sensitive analysis of single-end reads</a></td>
          <td>
            Ryo Kanno, Daisuke Tanaka, H. Nanamiya, T. Isogai
          </td>
          <td>2018-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>