---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-30 06:32:18 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Cellular adaptation to cancer therapy along a resistance continuum.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f2bf0e028aa7d79087ee8ee1bf9c6a6ea3815c4" target='_blank'>Computational quantification and characterization of independently evolving cellular subpopulations within tumors is critical to inhibit anti-cancer therapy resistance</a></td>
          <td>
            H. Alkhatib, A. Rubinstein, S. Vasudevan, E. Flashner-Abramson, S. Stefansky, Sangita Roy Chowdhury, S. Oguche, Tamar Peretz-Yablonsky, Avital Granit, Z. Granot, I. Ben-Porath, K. Sheva, Jon Feldman, Noa E. Cohen, A. Meirovitz, N. Kravchenko‐Balasha
          </td>
          <td>2022-10-20</td>
          <td>Genome Medicine</td>
          <td>13</td>
          <td>35</td>
        </tr>
    
        <tr id="Drug resistance inevitably emerges during the treatment of cancer by targeted therapy. Adaptive resistance is a major form of drug resistance, wherein the rewiring of protein signalling networks in response to drug perturbation allows the drug-targeted protein’s activity to recover, despite the continuous presence of the drug, enabling the cells to survive/grow. Simultaneously, molecular heterogeneity enables the selection of drug-resistant cancer clones that can survive an initial drug insult, proliferate, and eventually cause disease relapse. Despite their importance, the link between heterogeneity and adaptive resistance, specifically how heterogeneity influences protein signalling dynamics to drive adaptive resistance, remains poorly understood. Here, we have explored the relationship between heterogeneity, protein signalling dynamics and adaptive resistance through the development of a novel modelling technique coined Meta Dynamic Network (MDN) modelling. We use MDN modelling to characterise how heterogeneity influences the drug-response signalling dynamics of the proteins that regulate early cell cycle progression and demonstrate that heterogeneity can robustly facilitate adaptive resistance associated dynamics for key cell cycle regulators. We determined the influence of heterogeneity at the level of both protein interactions and protein expression and show that protein interactions are a much stronger driver of adaptive resistance. Owing to the mechanistic nature of the underpinning ODE framework, we then identified a full spectrum of subnetworks that drive adaptive resistance dynamics in the key early cell cycle regulators. Finally, we show that single-cell dynamic data supports the validity of our MDN modelling technique and a comparison between our predicted resistance mechanisms and known CDK4/6 and Estrogen Receptor inhibitor resistance mechanisms suggests MDN can be deployed to robustly predict network-level resistance mechanisms for novel drugs and additional protein signalling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc07dfb613ccafbc973aca41dde40873fb19fcc2" target='_blank'>Systematic Analysis of Network-driven Adaptive Resistance to CDK4/6 and Estrogen Receptor Inhibition using Meta-Dynamic Network Modelling</a></td>
          <td>
            Anthony Hart, Sung-young Shin, Leann Nguyen
          </td>
          <td>2023-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="Abstract Diverse forms of resistance to anticancer drugs can lead to the failure of chemotherapy. Drug resistance is one of the most intractable issues for successfully treating cancer in current clinical practice. Effective clinical approaches that could counter drug resistance by restoring the sensitivity of tumors to the targeted agents are urgently needed. As numerous experimental results on resistance mechanisms have been obtained and a mass of high-throughput data has been accumulated, mathematical modeling and computational predictions using systematic and quantitative approaches have become increasingly important, as they can potentially provide deeper insights into resistance mechanisms, generate novel hypotheses or suggest promising treatment strategies for future testing. In this review, we first briefly summarize the current progress of experimentally revealed resistance mechanisms of targeted therapy, including genetic mechanisms, epigenetic mechanisms, posttranslational mechanisms, cellular mechanisms, microenvironmental mechanisms and pharmacokinetic mechanisms. Subsequently, we list several currently available databases and Web-based tools related to drug sensitivity and resistance. Then, we focus primarily on introducing some state-of-the-art computational methods used in drug resistance studies, including mechanism-based mathematical modeling approaches (e.g. molecular dynamics simulation, kinetic model of molecular networks, ordinary differential equation model of cellular dynamics, stochastic model, partial differential equation model, agent-based model, pharmacokinetic–pharmacodynamic model, etc.) and data-driven prediction methods (e.g. omics data-based conventional screening approach for node biomarkers, static network approach for edge biomarkers and module biomarkers, dynamic network approach for dynamic network biomarkers and dynamic module network biomarkers, etc.). Finally, we discuss several further questions and future directions for the use of computational methods for studying drug resistance, including inferring drug-induced signaling networks, multiscale modeling, drug combinations and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418e09deab969d00f0ec23dec4e41c180b62d9a4" target='_blank'>Mathematical modeling and computational prediction of cancer drug resistance</a></td>
          <td>
            Xiaoqiang Sun, B. Hu
          </td>
          <td>2017-06-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>88</td>
          <td>50</td>
        </tr>
    
        <tr id="Cancer plasticity contributes to tumorigenesis, tumor progression, and drug resistance. Plasticity has traditionally been attributed to stem‐like cells, but recent studies used markers of stemness and differentiation to purify specific phenotypes from a mixed population of cancer cells by flow cytometry, and found that both differentiated and dedifferentiated phenotypes could revert over time to a heterogeneous equilibrium mixture, through stochastic transitions. A given instance of heterogeneity can in theory arise either from multiple static phenotypes or from transitions between non‐static phenotypes, and therefore, understanding plasticity requires measuring transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554119cfe94db8d195c00936a23240ef31a8250e" target='_blank'>Understanding Phenotypic Plasticity Induced by Environmental Factors Using Computational Markov Modeling</a></td>
          <td>
            N. S. Jagannathan, M. O. Ihsan, X. Kin, M. Clément, L. Tucker-Kellogg
          </td>
          <td>2018-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr id="A growing body of experimental evidence indicates a strong link between intratumoral heterogeneity and therapeutic resistance in cancer. In particular, tumor cells may survive therapy by switching their phenotypic identities to more resistant, drug-tolerant states. Computational models of phenotypic plasticity in response to cytotoxic therapy are needed: (1) to strengthen understanding of the interplay between phenotypic heterogeneity and therapeutic resistance, and (2) to identify potential strategies in silico that weaken resistance prior to in vitro testing. This work presents a linear time-invariant model of phenotypic state dynamics to deduce subpopulation-level behavior likely to affect temporal phenotypic composition and thus drug resistance. The model was identified under different therapeutic conditions with authentic biological data from a breast cancer cell line. Subsequent analysis suggested drug-induced effects on phenotypic state switching that could not be deduced directly from empirical observations. A bootstrap algorithm was implemented to identify statistically significant results: reduction in cell division under each therapeutic condition versus control. Further, Monte Carlo simulation was used to evaluate quality of model fit for two-way switching and net switching on synthetically generated data to determine the limitations of the latter assumption for subsequent modeling. Most importantly, the simple model structure initiated a control-theoretic approach for identifying promising combination treatments in silico to guide future laboratory testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763d15c3bfa9d59621b14e79056f68201d678df7" target='_blank'>A model of phenotypic state dynamics initiates a promising approach to control heterogeneous malignant cell populations</a></td>
          <td>
            Margaret P. Chapman, Tyler T. Risom, A. Aswani, Roel Dobbe, R. Sears, C. Tomlin
          </td>
          <td>2016-12-27</td>
          <td>2016 IEEE 55th Conference on Decision and Control (CDC)</td>
          <td>13</td>
          <td>79</td>
        </tr>
    
        <tr id="Phenotypic plasticity of cancer cells can lead to complex cell state dynamics during tumor progression and acquired resistance. Highly plastic stem-like states may be inherently drug-resistant. Moreover, cell state dynamics in response to therapy allow a tumor to evade treatment. In both scenarios, quantifying plasticity is essential for identifying high-plasticity states or elucidating transition paths between states. Currently, methods to quantify plasticity tend to focus on 1) quantification of quasi-potential based on the underlying gene regulatory network dynamics of the system; or 2) inference of cell potency based on trajectory inference or lineage tracing in single-cell dynamics. Here, we explore both of these approaches and associated computational tools. We then discuss implications of each approach to plasticity metrics, and relevance to cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00060c766440bb5c573b31ccd57a8f5bad3b7b7c" target='_blank'>Quantifying cancer cell plasticity with gene regulatory networks and single-cell dynamics</a></td>
          <td>
            Sarah M. Groves, Vito Quaranta
          </td>
          <td>2023-09-04</td>
          <td>Frontiers in Network Physiology</td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            Minsoo Choi, Jue Shi, Yan-ting Zhu, Ruizhen Yang, Kwang-Hyun Cho
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>40</td>
          <td>43</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            Minsoo Choi, Jue Shi, Yan-ting Zhu, Ruizhen Yang, Kwang-Hyun Cho
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>40</td>
          <td>43</td>
        </tr>
    
        <tr id="Introduction: Current personalized cancer medicine tailors therapy to heterogeneity between cancers of the same organ type occurring within different individuals. However, it does not yet address heterogeneity at the single cell level within individual cancers or the dynamics of cancer, due to heritable genetic and epigenetic change, as well as transient functional changes. We established computational methods for evaluating personalized medicine strategies, comparing the current personalized medicine strategy to alternatives. Current personalized medicine matches therapy to a tumor molecular profile at diagnosis and at tumor relapse or progression. This strategy focuses on the average, static, and current properties of the sample. Non-standard strategies also consider minor sub-clones, dynamics, and predicted future tumor states. Previous simulation results in a system with two non-cross resistant agents using non-standard strategies, optimized in single 45 day blocks, demonstrated significantly improved outcomes (Beckman, Schemmann, and Yeang, 109: 14586-91, 2012). The current work explores the effect of long range planning with a time horizon of up to five years on the effectiveness of the strategies, as well as generalizing the model to three non-cross resistant agents.. Methods: We developed a mathematical model of cancer therapy with two non-cross resistant agents incorporating genetic evolutionary dynamics and single cell heterogeneity, and examined simulated clinical outcomes (cell numbers of clones and sub-clones, projected survival). Previously we compared the current personalized medicine strategy to 5 alternative personalized strategies. The latter strategies explicitly considered sub-clones, evolutionary dynamics, and likely future sub-clones in addition to the current predominant clone. Particular emphasis was given to the prevention of incurable, multiply resistant sub-clones. The optimization was performed in single 45 day blocks. In the current work, we extended the work to three drug systems. We also extended the previous single step heuristic strategies to encompass multistep heuristics of up to five 45 day steps, and global optimization in 45 day blocks over a 5 year time horizon, with strategic updates every 45 days. Branch and bound methods were used for pruning decision trees. Parallel processing (23 servers) facilitated computations. Results: Previously we had carried out a computerized virtual clinical trial of over 3 million evaluable cancer “patients”, comparing current personalized medicine and 5 non-standard strategies. The 3 million virtual patients represented a comprehensive survey of likely population structures, growth rates, phenotypic transition rates (by heritable genetic or epigenetic mechanisms), and drug sensitivities. All alternatives tested resulted in an approximate doubling in mean and median survival compared to current personalized medicine and an increase in the apparent cure rate from 0.7% for current personalized medicine to 17-20% for alternatives. In no case was the current personalized medicine strategy superior. In the current work, we found in large simulations that planning ahead led to further increases in cure rates and analyzed several examples where highly complex treatment sequences led to cures which were not possible with single step 45 day optimization. Further, previous and current conclusions applied equally for three non-cross resistant agents. Conclusions: Explicit consideration of intratumoral heterogeneity and evolutionary dynamics, with probabilistic consideration of future outcomes with a long strategy horizon, can potentially lead to markedly improved patient outcomes, including cure rates. Application of knowledge from growing molecular and clinical oncology databases may allow more informative therapeutic simulations than previously possible. Citation Format: Robert A. Beckman, Chen-Hsiang Yeang. Long range personalized cancer treatment strategies incorporating evolutionary dynamics. [abstract]. In: Proceedings of the AACR Special Conference on Computational and Systems Biology of Cancer; Feb 8-11 2015; San Francisco, CA. Philadelphia (PA): AACR; Cancer Res 2015;75(22 Suppl 2):Abstract nr B2-52.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4f08d47e6930740a94de22c0b5e66e9851339a4" target='_blank'>Abstract B2-52: Long range personalized cancer treatment strategies incorporating evolutionary dynamics</a></td>
          <td>
            R. Beckman, Chen-Hsiang Yeang
          </td>
          <td>2015-11-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr id="Cancer has been a paradigmatic example of a complex disease. It is the second leading cause of death worldwide. Knowledge acquired on the mechanisms and potential causes underlying the origins and progression of cancer from single-scale studies have led to the development of personalized treatments, which in turn have decreasedmortality rates in several types of cancer. However, because it is not a single and isolated disease, but rather a heterogeneous set of multiscale alterations, current research requires computational and systems biology approaches that deal with complex systems on gene regulatory networks. Multi-omics is one of these approaches. Aimed at developing multi-scale mathematical modeling, it allows us to integrate the dynamics of biological perturbations for different types of cancer. Moreover, if tissue/organ-specific path is studied we are able to create mechanistic models for a particular type of the disease. That, combined with current low-cost high throughput technologies have made possible massive cohorts of -omics and clinical data to be publicly available. This immeasurable amount of information promotes harnessing data science approaches for the analysis, integration, and mining of multi-omics data. Large and multidisciplinary groups can now focus on creating models and provide theoretical frameworks that describe cancer in a variety of ways. Besides cancer cell transformation or cancer evolution, currently single cell genomics and spatial single cell transcriptomics is moving forward with deeper understanding of the disease. Also, cancer driver mutations and mutational processes, development of targetspecific treatments have been leading to a prediction of drug responses. Notwithstanding, several challenges that remain to be undertaken such asmechanisms ofmetastasis, resistance to treatment, intra-tumoral heterogeneity, molecular, cellular, and metabolic changes during progression stages, epigenetic modifications, to mention a few. This cross-disciplinary interaction at these levels of description have contributed to develop more efficient models that improve the predictive capacity and ultimately help clinicians and medical scientists in the treatment and therapies. OPEN ACCESS">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a5c9568422971ebe3df8c55afef2906c57168e" target='_blank'>Editorial: Cancer systems biology</a></td>
          <td>
            J. Espinal-Enríquez, C. Rangel-Escareño, F. Emmert-Streib
          </td>
          <td>2022-11-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>