---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-21 06:32:54 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Cellular adaptation to cancer therapy along a resistance continuum.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f2bf0e028aa7d79087ee8ee1bf9c6a6ea3815c4" target='_blank'>Computational quantification and characterization of independently evolving cellular subpopulations within tumors is critical to inhibit anti-cancer therapy resistance</a></td>
          <td>
            H. Alkhatib, A. Rubinstein, S. Vasudevan, E. Flashner-Abramson, S. Stefansky, Sangita Roy Chowdhury, S. Oguche, Tamar Peretz-Yablonsky, Avital Granit, Z. Granot, I. Ben-Porath, K. Sheva, Jon Feldman, Noa E. Cohen, A. Meirovitz, N. Kravchenko‐Balasha
          </td>
          <td>2022-10-20</td>
          <td>Genome Medicine</td>
          <td>13</td>
          <td>35</td>
        </tr>
    
        <tr id="Drug resistance inevitably emerges during the treatment of cancer by targeted therapy. Adaptive resistance is a major form of drug resistance, wherein the rewiring of protein signalling networks in response to drug perturbation allows the drug-targeted protein’s activity to recover, despite the continuous presence of the drug, enabling the cells to survive/grow. Simultaneously, molecular heterogeneity enables the selection of drug-resistant cancer clones that can survive an initial drug insult, proliferate, and eventually cause disease relapse. Despite their importance, the link between heterogeneity and adaptive resistance, specifically how heterogeneity influences protein signalling dynamics to drive adaptive resistance, remains poorly understood. Here, we have explored the relationship between heterogeneity, protein signalling dynamics and adaptive resistance through the development of a novel modelling technique coined Meta Dynamic Network (MDN) modelling. We use MDN modelling to characterise how heterogeneity influences the drug-response signalling dynamics of the proteins that regulate early cell cycle progression and demonstrate that heterogeneity can robustly facilitate adaptive resistance associated dynamics for key cell cycle regulators. We determined the influence of heterogeneity at the level of both protein interactions and protein expression and show that protein interactions are a much stronger driver of adaptive resistance. Owing to the mechanistic nature of the underpinning ODE framework, we then identified a full spectrum of subnetworks that drive adaptive resistance dynamics in the key early cell cycle regulators. Finally, we show that single-cell dynamic data supports the validity of our MDN modelling technique and a comparison between our predicted resistance mechanisms and known CDK4/6 and Estrogen Receptor inhibitor resistance mechanisms suggests MDN can be deployed to robustly predict network-level resistance mechanisms for novel drugs and additional protein signalling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc07dfb613ccafbc973aca41dde40873fb19fcc2" target='_blank'>Systematic Analysis of Network-driven Adaptive Resistance to CDK4/6 and Estrogen Receptor Inhibition using Meta-Dynamic Network Modelling</a></td>
          <td>
            Anthony Hart, Sung-young Shin, Leann Nguyen
          </td>
          <td>2023-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="Abstract Diverse forms of resistance to anticancer drugs can lead to the failure of chemotherapy. Drug resistance is one of the most intractable issues for successfully treating cancer in current clinical practice. Effective clinical approaches that could counter drug resistance by restoring the sensitivity of tumors to the targeted agents are urgently needed. As numerous experimental results on resistance mechanisms have been obtained and a mass of high-throughput data has been accumulated, mathematical modeling and computational predictions using systematic and quantitative approaches have become increasingly important, as they can potentially provide deeper insights into resistance mechanisms, generate novel hypotheses or suggest promising treatment strategies for future testing. In this review, we first briefly summarize the current progress of experimentally revealed resistance mechanisms of targeted therapy, including genetic mechanisms, epigenetic mechanisms, posttranslational mechanisms, cellular mechanisms, microenvironmental mechanisms and pharmacokinetic mechanisms. Subsequently, we list several currently available databases and Web-based tools related to drug sensitivity and resistance. Then, we focus primarily on introducing some state-of-the-art computational methods used in drug resistance studies, including mechanism-based mathematical modeling approaches (e.g. molecular dynamics simulation, kinetic model of molecular networks, ordinary differential equation model of cellular dynamics, stochastic model, partial differential equation model, agent-based model, pharmacokinetic–pharmacodynamic model, etc.) and data-driven prediction methods (e.g. omics data-based conventional screening approach for node biomarkers, static network approach for edge biomarkers and module biomarkers, dynamic network approach for dynamic network biomarkers and dynamic module network biomarkers, etc.). Finally, we discuss several further questions and future directions for the use of computational methods for studying drug resistance, including inferring drug-induced signaling networks, multiscale modeling, drug combinations and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418e09deab969d00f0ec23dec4e41c180b62d9a4" target='_blank'>Mathematical modeling and computational prediction of cancer drug resistance</a></td>
          <td>
            Xiaoqiang Sun, B. Hu
          </td>
          <td>2017-06-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>88</td>
          <td>50</td>
        </tr>
    
        <tr id="Tumour recurrence post chemotherapy is an established clinical problem and many cancer types are often observed to be increasingly drug resistant subsequent to chemotherapy treatments. Drug resistance in cancer is a multipart phenomenon which can be derived from several origins and in many cases it has been observed that cancer cells have the ability to possess, acquire and communicate drug resistant traits. Here, we present an in silico framework that can simulate multiple types of drug resistance and study their role in chemotherapeutic treatment. This framework is based on a hybrid multiscale mathematical model. We investigate drug response in cancer cell populations hosting various types of drug resistant phenotypes, arising from inheritance or mutations that are spontaneous, drug-induced or communicated via exosomes. Our study shows that drug response in cancer cell populations is crucially influenced by the drug resistant phenotypes amongst the cells. This indicates that the effect of chemotherapy, and by extension optimal chemotherapy administration, is heavily dependent not only on the existence of drug resistance, but also the type of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c61c22540698a05106a219b62e21c13e24dc65f" target='_blank'>What does not kill a tumour may make it stronger: in silico Insights on Chemotherapeutic Drug Resistance</a></td>
          <td>
            Sara Hamis, P. Nithiarasu, Gibin G. Powathil
          </td>
          <td>2017-12-07</td>
          <td>bioRxiv</td>
          <td>42</td>
          <td>42</td>
        </tr>
    
        <tr id="Cancer plasticity contributes to tumorigenesis, tumor progression, and drug resistance. Plasticity has traditionally been attributed to stem‐like cells, but recent studies used markers of stemness and differentiation to purify specific phenotypes from a mixed population of cancer cells by flow cytometry, and found that both differentiated and dedifferentiated phenotypes could revert over time to a heterogeneous equilibrium mixture, through stochastic transitions. A given instance of heterogeneity can in theory arise either from multiple static phenotypes or from transitions between non‐static phenotypes, and therefore, understanding plasticity requires measuring transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554119cfe94db8d195c00936a23240ef31a8250e" target='_blank'>Understanding Phenotypic Plasticity Induced by Environmental Factors Using Computational Markov Modeling</a></td>
          <td>
            N. S. Jagannathan, M. O. Ihsan, X. Kin, M. Clément, L. Tucker-Kellogg
          </td>
          <td>2018-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr id="A growing body of experimental evidence indicates a strong link between intratumoral heterogeneity and therapeutic resistance in cancer. In particular, tumor cells may survive therapy by switching their phenotypic identities to more resistant, drug-tolerant states. Computational models of phenotypic plasticity in response to cytotoxic therapy are needed: (1) to strengthen understanding of the interplay between phenotypic heterogeneity and therapeutic resistance, and (2) to identify potential strategies in silico that weaken resistance prior to in vitro testing. This work presents a linear time-invariant model of phenotypic state dynamics to deduce subpopulation-level behavior likely to affect temporal phenotypic composition and thus drug resistance. The model was identified under different therapeutic conditions with authentic biological data from a breast cancer cell line. Subsequent analysis suggested drug-induced effects on phenotypic state switching that could not be deduced directly from empirical observations. A bootstrap algorithm was implemented to identify statistically significant results: reduction in cell division under each therapeutic condition versus control. Further, Monte Carlo simulation was used to evaluate quality of model fit for two-way switching and net switching on synthetically generated data to determine the limitations of the latter assumption for subsequent modeling. Most importantly, the simple model structure initiated a control-theoretic approach for identifying promising combination treatments in silico to guide future laboratory testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763d15c3bfa9d59621b14e79056f68201d678df7" target='_blank'>A model of phenotypic state dynamics initiates a promising approach to control heterogeneous malignant cell populations</a></td>
          <td>
            Margaret P. Chapman, Tyler T. Risom, A. Aswani, Roel Dobbe, R. Sears, C. Tomlin
          </td>
          <td>2016-12-27</td>
          <td>2016 IEEE 55th Conference on Decision and Control (CDC)</td>
          <td>13</td>
          <td>79</td>
        </tr>
    
        <tr id="Phenotypic plasticity of cancer cells can lead to complex cell state dynamics during tumor progression and acquired resistance. Highly plastic stem-like states may be inherently drug-resistant. Moreover, cell state dynamics in response to therapy allow a tumor to evade treatment. In both scenarios, quantifying plasticity is essential for identifying high-plasticity states or elucidating transition paths between states. Currently, methods to quantify plasticity tend to focus on 1) quantification of quasi-potential based on the underlying gene regulatory network dynamics of the system; or 2) inference of cell potency based on trajectory inference or lineage tracing in single-cell dynamics. Here, we explore both of these approaches and associated computational tools. We then discuss implications of each approach to plasticity metrics, and relevance to cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00060c766440bb5c573b31ccd57a8f5bad3b7b7c" target='_blank'>Quantifying cancer cell plasticity with gene regulatory networks and single-cell dynamics</a></td>
          <td>
            Sarah M. Groves, Vito Quaranta
          </td>
          <td>2023-09-04</td>
          <td>Frontiers in Network Physiology</td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            Minsoo Choi, Jue Shi, Yan-ting Zhu, Ruizhen Yang, Kwang-Hyun Cho
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>42</td>
          <td>43</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>