---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-18 09:33:48 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Cellular adaptation to cancer therapy along a resistance continuum.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Background Drug resistance continues to be a major limiting factor across diverse anti-cancer therapies. Contributing to the complexity of this challenge is cancer plasticity, in which one cancer subtype switches to another in response to treatment, for example, triple-negative breast cancer (TNBC) to Her2-positive breast cancer. For optimal treatment outcomes, accurate tumor diagnosis and subsequent therapeutic decisions are vital. This study assessed a novel approach to characterize treatment-induced evolutionary changes of distinct tumor cell subpopulations to identify and therapeutically exploit anticancer drug resistance. Methods In this research, an information-theoretic single-cell quantification strategy was developed to provide a high-resolution and individualized assessment of tumor composition for a customized treatment approach. Briefly, this single-cell quantification strategy computes cell barcodes based on at least 100,000 tumor cells from each experiment and reveals a cell-specific signaling signature (CSSS) composed of a set of ongoing processes in each cell. Results Using these CSSS-based barcodes, distinct subpopulations evolving within the tumor in response to an outside influence, like anticancer treatments, were revealed and mapped. Barcodes were further applied to assign targeted drug combinations to each individual tumor to optimize tumor response to therapy. The strategy was validated using TNBC models and patient-derived tumors known to switch phenotypes in response to radiotherapy (RT). Conclusions We show that a barcode-guided targeted drug cocktail significantly enhances tumor response to RT and prevents regrowth of once-resistant tumors. The strategy presented herein shows promise in preventing cancer treatment resistance, with significant applicability in clinical use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f2bf0e028aa7d79087ee8ee1bf9c6a6ea3815c4" target='_blank'>Computational quantification and characterization of independently evolving cellular subpopulations within tumors is critical to inhibit anti-cancer therapy resistance</a></td>
          <td>
            H. Alkhatib, A. Rubinstein, S. Vasudevan, E. Flashner-Abramson, S. Stefansky, Sangita Roy Chowdhury, S. Oguche, Tamar Peretz-Yablonsky, Avital Granit, Z. Granot, I. Ben-Porath, K. Sheva, Jon Feldman, Noa E. Cohen, A. Meirovitz, N. Kravchenko‐Balasha
          </td>
          <td>2022-10-20</td>
          <td>Genome Medicine</td>
          <td>15</td>
          <td>36</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc07dfb613ccafbc973aca41dde40873fb19fcc2" target='_blank'>Systematic Analysis of Network-driven Adaptive Resistance to CDK4/6 and Estrogen Receptor Inhibition using Meta-Dynamic Network Modelling</a></td>
          <td>
            Anthony Hart, Sung-young Shin, Leann Nguyen
          </td>
          <td>2023-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="Understanding tumor cell heterogeneity and plasticity is crucial for overcoming drug resistance. Single-cell technologies enable analyzing cell states at a given condition, but catenating static cell snapshots to characterize dynamic drug responses remains challenging. Here, we propose scStateDynamics, an algorithm to infer tumor cell state dynamics and identify common drug effects by modeling single-cell level gene expression changes. Its reliability is validated on both simulated and lineage tracing data. Application to real tumor drug treatment datasets identifies more subtle cell subclusters with different drug responses beyond static transcriptome similarity and disentangles drug action mechanisms from the cell-level expression changes. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-024-03436-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7e21f7327e57fd7a5664b9a5fd090102737fcd" target='_blank'>scStateDynamics: deciphering the drug-responsive tumor cell state dynamics by modeling single-cell level expression changes</a></td>
          <td>
            Wenbo Guo, Xinqi Li, Dongfang Wang, Nan Yan, Qifan Hu, Fan Yang, Xuegong Zhang, Jianhua Yao, Jin Gu
          </td>
          <td>2024-11-21</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr id="Tumour recurrence post chemotherapy is an established clinical problem and many cancer types are often observed to be increasingly drug resistant subsequent to chemotherapy treatments. Drug resistance in cancer is a multipart phenomenon which can be derived from several origins and in many cases it has been observed that cancer cells have the ability to possess, acquire and communicate drug resistant traits. Here, we present an in silico framework that can simulate multiple types of drug resistance and study their role in chemotherapeutic treatment. This framework is based on a hybrid multiscale mathematical model. We investigate drug response in cancer cell populations hosting various types of drug resistant phenotypes, arising from inheritance or mutations that are spontaneous, drug-induced or communicated via exosomes. Our study shows that drug response in cancer cell populations is crucially influenced by the drug resistant phenotypes amongst the cells. This indicates that the effect of chemotherapy, and by extension optimal chemotherapy administration, is heavily dependent not only on the existence of drug resistance, but also the type of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c61c22540698a05106a219b62e21c13e24dc65f" target='_blank'>What does not kill a tumour may make it stronger: in silico Insights on Chemotherapeutic Drug Resistance</a></td>
          <td>
            S. Hamis, P. Nithiarasu, Gibin G. Powathil
          </td>
          <td>2017-12-07</td>
          <td>bioRxiv</td>
          <td>46</td>
          <td>43</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554119cfe94db8d195c00936a23240ef31a8250e" target='_blank'>Understanding Phenotypic Plasticity Induced by Environmental Factors Using Computational Markov Modeling</a></td>
          <td>
            N. S. Jagannathan, M. O. Ihsan, X. Kin, M. Clément, L. Tucker-Kellogg
          </td>
          <td>2018-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763d15c3bfa9d59621b14e79056f68201d678df7" target='_blank'>A model of phenotypic state dynamics initiates a promising approach to control heterogeneous malignant cell populations</a></td>
          <td>
            Margaret P. Chapman, Tyler T. Risom, A. Aswani, Roel Dobbe, R. Sears, C. Tomlin
          </td>
          <td>2016-12-27</td>
          <td>2016 IEEE 55th Conference on Decision and Control (CDC)</td>
          <td>13</td>
          <td>82</td>
        </tr>
    
        <tr id="Phenotypic plasticity of cancer cells can lead to complex cell state dynamics during tumor progression and acquired resistance. Highly plastic stem-like states may be inherently drug-resistant. Moreover, cell state dynamics in response to therapy allow a tumor to evade treatment. In both scenarios, quantifying plasticity is essential for identifying high-plasticity states or elucidating transition paths between states. Currently, methods to quantify plasticity tend to focus on 1) quantification of quasi-potential based on the underlying gene regulatory network dynamics of the system; or 2) inference of cell potency based on trajectory inference or lineage tracing in single-cell dynamics. Here, we explore both of these approaches and associated computational tools. We then discuss implications of each approach to plasticity metrics, and relevance to cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00060c766440bb5c573b31ccd57a8f5bad3b7b7c" target='_blank'>Quantifying cancer cell plasticity with gene regulatory networks and single-cell dynamics</a></td>
          <td>
            Sarah M. Groves, Vito Quaranta
          </td>
          <td>2023-09-04</td>
          <td>Frontiers in Network Physiology</td>
          <td>4</td>
          <td>7</td>
        </tr>
    
        <tr id="Cancer is a complex disease involving multiple genomic alterations that disrupt the dynamic response of signaling networks. The heterogeneous nature of cancer, which results in highly variable drug response, is a major obstacle to developing effective cancer therapy. Previous studies of cancer therapeutic response mostly focus on static analysis of genome-wide alterations, thus they are unable to unravel the dynamic, network-specific origin of variation. Here we present a network dynamics-based approach to integrate cancer genomics with dynamics of biological network for drug response prediction and design of drug combination. We select the p53 network as an example and analyze its cancer-specific state transition dynamics under distinct anticancer drug treatments by attractor landscape analysis. Our results not only enable stratification of cancer into distinct drug response groups, but also reveal network-specific drug targets that maximize p53 network-mediated cell death, providing a basis to design combinatorial therapeutic strategies for distinct cancer genomic subtypes. Genomic alterations underlie the variability of drug responses between cancers, but our mechanistic understanding is limited. Here the authors use the p53 network to study how rewiring of signalling networks by genomic alterations impact their dynamic response to pharmacological perturbation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            Minsoo Choi, Jue Shi, Yan-ting Zhu, Ruizhen Yang, Kwang-Hyun Cho
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>45</td>
          <td>44</td>
        </tr>
    
        <tr id="An integrated and single-cell-omics dataset–based computing pipeline scPlasticity is introduced to quantify cell plasticity and prioritize master regulators in normal and stress hematopoiesis. Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration, and trauma-to-repair biological process. Single-cell-omics datasets provide an unprecedented resource to empower CP analysis. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP. In this study, we generated high-quality lineage-negative single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named scPlasticity to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells with mutational or environmental challenges. Using embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wild type where the pipeline confirms a low proportion of hybrid cells (Phc, with bias range: 0.4∼0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of hematopoietic stem and progenitor cells was enhanced. We prioritized several transcription factors and signaling pathways, which are responsible for Phc alterations. In silico perturbation suggests knocking out EGR regulons or pathways of IL-1R1 and β-adrenoreceptor partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdd349e556fc552ea2193d11b33873ee6b1af4d" target='_blank'>Computing hematopoiesis plasticity in response to genetic mutations and environmental stimulations</a></td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Dengyi Bao, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-11-13</td>
          <td>Life Science Alliance</td>
          <td>1</td>
          <td>2</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>