---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-10 09:26:03 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>EcDNA hubs drive cooperative intermolecular oncogene expression</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="The Hi-C technique has been shown to be a promising method to detect structural variations (SVs) in human genomes. However, algorithms that can use Hi-C data for a full-range SV detection have been severely lacking. Current methods can only identify interchromosomal translocations and long-range intrachromosomal SVs (>1 Mb) at less-than-optimal resolution. Therefore, we develop EagleC, a framework that combines deep-learning and ensemble-learning strategies to predict a full range of SVs at high resolution. We show that EagleC can uniquely capture a set of fusion genes that are missed by whole-genome sequencing or nanopore. Furthermore, EagleC also effectively captures SVs in other chromatin interaction platforms, such as HiChIP, Chromatin interaction analysis with paired-end tag sequencing (ChIA-PET), and capture Hi-C. We apply EagleC in more than 100 cancer cell lines and primary tumors and identify a valuable set of high-quality SVs. Last, we demonstrate that EagleC can be applied to single-cell Hi-C and used to study the SV heterogeneity in primary tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c1dd3757008f0744292fa9bfc0f7c36e71ba63" target='_blank'>EagleC: A deep-learning framework for detecting a full range of structural variations from bulk and single-cell contact maps</a></td>
          <td>
            Xiaotao Wang, Yu Luan, Feng Yue
          </td>
          <td>2022-06-01</td>
          <td>Science Advances</td>
          <td>54</td>
          <td>34</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd217dda14bc5faced17ed3b56a0c56548ca85ad" target='_blank'>ATACAmp: a tool for detecting ecDNA/HSRs from bulk and single-cell ATAC-seq data</a></td>
          <td>
            Hansen Cheng, Wenhao Ma, Kun Wang, Han Chu, Guangchao Bao, Yu Liao, Yawen Yuan, Yixiong Gou, Liting Dong, Jian Yang, Haoyang Cai
          </td>
          <td>2023-11-10</td>
          <td>BMC Genomics</td>
          <td>4</td>
          <td>14</td>
        </tr>
    
        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and can contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method that combines a breakpoint-graph approach with LASSO regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole-genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyses in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil</a></td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro Gonz√°lez, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome Research</td>
          <td>3</td>
          <td>51</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a64951ab3de56b0af72adea21a1af02157560a" target='_blank'>Tracing cancer evolution and heterogeneity using Hi-C</a></td>
          <td>
            Dan D. Erdmann-Pham, S. S. Batra, Timothy K. Turkalo, James Durbin, M. Blanchette, I. Yeh, Hunter Shain, B. Bastian, Yun S. Song, D. Rokhsar, D. Hockemeyer
          </td>
          <td>2023-02-21</td>
          <td>Nature Communications</td>
          <td>7</td>
          <td>105</td>
        </tr>
    
        <tr id="It has been shown that Hi-C can be used as a powerful tool to detect structural variations (SVs) and enhancer hijacking events. However, there has been no existing programs that can directly visualize and detect such events on a personal computer, which hinders the broad adaption of the technology for intuitive discovery in cancer studies. Here, we introduce the EagleC Explorer, a desktop software that is specifically designed for exploring Hi-C and other chromatin contact data in cancer genomes. EagleC Explorer has a set of unique features, including 1) conveniently visualizing global and local Hi-C data; 2) interactively detecting SVs on a Hi-C map for any user-selected region on screen within seconds, using a deep-learning model; 3) reconstructing local Hi-C map surrounding user-provided SVs and generating publication-quality figures; 4) detecting enhancer hijacking events for any user-suggested regions on screen. In addition, EagleC Explorer can also incorporate other genomic tracks such as RNA-Seq or ChIP-Seq to facilitate scientists for integrative data analysis and making novel discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddde31c431ffe8489dfff5f4e90e563b04aaa49d" target='_blank'>EagleC Explorer: A desktop application for interactively detecting and visualizing SVs and enhancer hijacking on Hi-C contact maps</a></td>
          <td>
            Yihao Fu, Xiaotao Wang, Feng Yue
          </td>
          <td>2023-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03cd8ce6ac493e2c9285920b96ffc888c48e3bd3" target='_blank'>Epiphany: predicting Hi-C contact maps from 1D epigenomic signals</a></td>
          <td>
            Rui Yang, Arnav Das, Vianne R. Gao, Alireza Karbalayghareh, William Stafford Noble, J. Bilmes, C. Leslie
          </td>
          <td>2021-12-03</td>
          <td>Genome Biology</td>
          <td>34</td>
          <td>92</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7023130d35a7dff6797795835ab533d93f4a571d" target='_blank'>EcSeg: Semantic Segmentation of Metaphase Images Containing Extrachromosomal DNA</a></td>
          <td>
            Utkrisht Rajkumar, Kristen M. Turner, J. Luebeck, Viraj Deshpande, Manmohan Chandraker, P. Mischel, V. Bafna
          </td>
          <td>2019-10-21</td>
          <td>iScience</td>
          <td>37</td>
          <td>103</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c326914166281ca840ac654a86ff0a896a7b482e" target='_blank'>Major cell-types in multiomic single-nucleus datasets impact statistical modeling of links between regulatory sequences and target genes</a></td>
          <td>
            F. Leblanc, G. Lettre
          </td>
          <td>2022-09-17</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>71</td>
        </tr>
    
        <tr id="Structural variants can contribute to oncogenesis through a variety of mechanisms, yet, despite their importance, the identification of structural variants in cancer genomes remains challenging. Here, we present an integrative framework for comprehensively identifying structural variation in cancer genomes. For the first time, we apply next-generation optical mapping, high-throughput chromosome conformation capture (Hi-C), and whole genome sequencing to systematically detect SVs in a variety of cancer cells. Using this approach, we identify and characterize structural variants in up to 29 commonly used normal and cancer cell lines. We find that each method has unique strengths in identifying different classes of structural variants and at different scales, suggesting that integrative approaches are likely the only way to comprehensively identify structural variants in the genome. Studying the impact of the structural variants in cancer cell lines, we identify widespread structural variation events affecting the functions of non-coding sequences in the genome, including the deletion of distal regulatory sequences, alteration of DNA replication timing, and the creation of novel 3D chromatin structural domains. These results underscore the importance of comprehensive structural variant identification and indicate that non-coding structural variation may be an underappreciated mutational process in cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56384406aa7af5e9594b02f12e075aeda13c096c" target='_blank'>An Integrative Framework for Detecting Structural Variations in Cancer Genomes</a></td>
          <td>
            Jesse R. Dixon, Jie Xu, V. Dileep, Y. Zhan, F. Song, Victoria T. Le, G. Yardƒ±mcƒ±, Abhijit Chakraborty, Darrin V Bann, Yanli Wang, Royden A. Clark, Lijun Zhang, Hongbo Yang, Tingting Liu, Sriranga Iyyanki, Lin An, Christopher Pool, Takayo Sasaki, Juan Carlos Rivera Mulia, Hakan Ozadam, B. Lajoie, R. Kaul, Michael Buckley, Kristen Lee, Morgan Diegel, Dubravka Pezic, Christina Ernst, Suzana Hadjur, D. Odom, J. Stamatoyannopoulos, J. Broach, R. Hardison, F. Ay, William Stafford Noble, J. Dekker, D. Gilbert, Feng Yue
          </td>
          <td>2017-03-28</td>
          <td>bioRxiv</td>
          <td>15</td>
          <td>92</td>
        </tr>
    
        <tr id="The single-cell assay for transposase accessible chromatin (scATAC) is an invaluable asset to profile the epigenomic landscape of heterogeneous cells populations in complex tissue and organ systems. However, the lack of tools that enable the use of scATAC data to discriminate between malignant and non-malignant cells has prevented the widespread application of this technique to clinical tumor samples. Here we describe Copy-scAT, a new computational tool that uses scATAC data to infer both large-scale and focal copy number alterations. Copy-scAT can call both clonal and subclonal copy number changes, allowing identification of cancer cells and cell populations that putatively constitute the tumor microenvironment. Copy-scAT therefore enables downstream chromatin accessibility studies that focus on malignant or non-malignant cell populations in clinical samples that are profiled by scATAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15880cf60ea5dc913c5c5e1f2973bda3168d77d8" target='_blank'>Copy-scAT: An R package for detection of large-scale and focal copy number alterations in single-cell chromatin accessibility datasets</a></td>
          <td>
            A. Nikoliƒá, D. Singhal, K. Ellestad, Michael J Johnston, Yaoqing Shen, Aaron H. Gillmor, Sorana A. Morrissy, J. Cairncross, S. Jones, M. Lupien, J. Chan, P. Neri, N. Bahlis, M. Gallo
          </td>
          <td>2020-09-22</td>
          <td>bioRxiv</td>
          <td>13</td>
          <td>69</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>