---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-30 09:31:23 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>EcDNA hubs drive cooperative intermolecular oncogene expression</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="The Hi-C technique has been shown to be a promising method to detect structural variations (SVs) in human genomes. However, algorithms that can use Hi-C data for a full-range SV detection have been severely lacking. Current methods can only identify interchromosomal translocations and long-range intrachromosomal SVs (>1 Mb) at less-than-optimal resolution. Therefore, we develop EagleC, a framework that combines deep-learning and ensemble-learning strategies to predict a full range of SVs at high resolution. We show that EagleC can uniquely capture a set of fusion genes that are missed by whole-genome sequencing or nanopore. Furthermore, EagleC also effectively captures SVs in other chromatin interaction platforms, such as HiChIP, Chromatin interaction analysis with paired-end tag sequencing (ChIA-PET), and capture Hi-C. We apply EagleC in more than 100 cancer cell lines and primary tumors and identify a valuable set of high-quality SVs. Last, we demonstrate that EagleC can be applied to single-cell Hi-C and used to study the SV heterogeneity in primary tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c1dd3757008f0744292fa9bfc0f7c36e71ba63" target='_blank'>EagleC: A deep-learning framework for detecting a full range of structural variations from bulk and single-cell contact maps</a></td>
          <td>
            Xiaotao Wang, Yu Luan, Feng Yue
          </td>
          <td>2022-06-01</td>
          <td>Science Advances</td>
          <td>61</td>
          <td>35</td>
        </tr>
    
        <tr id="Background High oncogene expression in cancer cells is a major cause of rapid tumor progression and drug resistance. Recent cancer genome research has shown that oncogenes as well as regulatory elements can be amplified in the form of extrachromosomal DNA (ecDNA) or subsequently integrated into chromosomes as homogeneously staining regions (HSRs). These genome-level variants lead to the overexpression of the corresponding oncogenes, resulting in poor prognosis. Most existing detection methods identify ecDNA using whole genome sequencing (WGS) data. However, these techniques usually detect many false positive regions owing to chromosomal DNA interference. Results In the present study, an algorithm called “ATACAmp” that can identify ecDNA/HSRs in tumor genomes using ATAC-seq data has been described. High chromatin accessibility, one of the characteristics of ecDNA, makes ATAC-seq naturally enriched in ecDNA and reduces chromosomal DNA interference. The algorithm was validated using ATAC-seq data from cell lines that have been experimentally determined to contain ecDNA regions. ATACAmp accurately identified the majority of validated ecDNA regions. AmpliconArchitect, the widely used ecDNA detecting tool, was used to detect ecDNA regions based on the WGS data of the same cell lines. Additionally, the Circle-finder software, another tool that utilizes ATAC-seq data, was assessed. The results showed that ATACAmp exhibited higher accuracy than AmpliconArchitect and Circle-finder. Moreover, ATACAmp supported the analysis of single-cell ATAC-seq data, which linked ecDNA to specific cells. Conclusions ATACAmp, written in Python, is freely available on GitHub under the MIT license: https://github.com/chsmiss/ATAC-amp . Using ATAC-seq data, ATACAmp offers a novel analytical approach that is distinct from the conventional use of WGS data. Thus, this method has the potential to reduce the cost and technical complexity associated ecDNA analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd217dda14bc5faced17ed3b56a0c56548ca85ad" target='_blank'>ATACAmp: a tool for detecting ecDNA/HSRs from bulk and single-cell ATAC-seq data</a></td>
          <td>
            Hansen Cheng, Wenhao Ma, Kun Wang, Han Chu, Guangchao Bao, Yu Liao, Yawen Yuan, Yixiong Gou, Liting Dong, Jian Yang, Haoyang Cai
          </td>
          <td>2023-11-10</td>
          <td>BMC Genomics</td>
          <td>5</td>
          <td>15</td>
        </tr>
    
        <tr id="The role of extrachromosomal circular DNA (eccDNA) has been highlighted. More recently, eccDNA-chromosome interactions were identified, suggesting a potential role of eccDNA in transcriptional regulation. Several databases currently provide valuable resources for the study of eccDNAs. However, these databases are primarily focused on Human eccDNAs and do not provide analysis of eccDNA-chromosome interaction and eccDNA gene expression in different tissues. Herein, to further integrate available resources for eccDNA data across multiple species, we developed the eccDB database. The current version of eccDB has collected a total of 1,317,182 eccDNAs in 424 samples from four species (homo sapiens, mus musculus, saccharomyces cerevisiae, and arabidopsis thaliana). eccDB provides regulatory and epigenetic information on eccDNA, including typical enhancers, super-enhancers, transcription factors, DNA methylation positions, risk SNPs, expression quantitative trait locus, chromatin accessibility regions, and chromHMM states. In particular, eccDB provides eccDNAs intrachromosomal and interchromosomal interaction analysis to predict the transcriptional regulatory functions of eccDNA. Moreover, eccDB identifies eccDNAs from unknown DNA sequences and analyzes the functional and evolutionary relationships of an eccDNA among different species. Overall, eccDB offers web-based analytical tools and a comprehensive resource for biologists and clinicians to decipher the molecular regulatory mechanisms of eccDNA. eccDB is freely available at http://www.xiejjlab.bio/eccDB">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a342b5e252ecb1e48f975fdd5e8efa46835895" target='_blank'>eccDB: a comprehensive repository for eccDNA-mediated chromatin contacts in multi-species</a></td>
          <td>
            Min Yang, Bo Qiu, Guo-You He, Jianyuan Zhou, Haoming Yu, Yuying Zhang, Yanshang Li, Taisong Li, Jinping Guo, Xuecang Li, Jianjun Xie
          </td>
          <td>2022-09-23</td>
          <td>bioRxiv</td>
          <td>13</td>
          <td>13</td>
        </tr>
    
        <tr id="Focal amplifications in the cancer genome, particularly extrachromosomal DNA (ecDNA) amplifications, are emerging as a pivotal event in cancer progression across diverse cancer contexts, presenting a paradigm shift in our understanding of tumor dynamics. Simultaneously, identification of the various modes of focal amplifications is bioinformatically challenging. We present AmpliconSuite, a collection of tools that enables robust identification of focal amplifications from whole-genome sequencing data. AmpliconSuite includes AmpliconSuite- pipeline; utilizing the AmpliconArchitect (AA) method, and AmpliconRepository.org; a community- editable website for the sharing of focal amplification calls. We also describe improvements made to AA since its initial release that improve its accuracy and speed. As a proof of principle, we utilized publicly available pan-cancer datasets encompassing 2,525 tumor samples hosted on AmpliconRepository.org to illustrate important properties of focal amplifications, showing ecDNA has higher copy number, and stronger oncogene enrichment, compared to other classes of focal amplifications. Finally, we illustrate how AmpliconSuite-pipeline enables delineation of the various mechanisms by which ecDNA forms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3f112dd01867d814b393cd8a3b0372c553a87b" target='_blank'>AmpliconSuite: an end-to-end workflow for analyzing focal amplifications in cancer genomes</a></td>
          <td>
            J. Luebeck, Edwin Huang, Forrest Kim, T. Liefeld, Bhargavi Dameracharla, Rohil Ahuja, Daniel Schreyer, Gino Prasad, Michał Adamaszek, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Thorin Tabor, M. Giurgiu, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>72</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/807670030fcdcd4d1e09df57ab926e3510f882f2" target='_blank'>EccBase: A high-quality database for exploration and characterization of extrachromosomal circular DNAs in cancer</a></td>
          <td>
            Haiyang Sun, Xinyi Lu, Lingyun Zou
          </td>
          <td>2023-04-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>11</td>
          <td>9</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc4c9321a9bd59c360d734a2b18de2d86fe3e68" target='_blank'>Genome-wide detection of enhancer-hijacking events from chromatin interaction data in rearranged genomes</a></td>
          <td>
            Xiaotao Wang, Jie Xu, Baozhen Zhang, Ye Hou, F. Song, Huijue Lyu, Feng Yue
          </td>
          <td>2021-06-01</td>
          <td>Nature Methods</td>
          <td>101</td>
          <td>35</td>
        </tr>
    
        <tr id="In cancer, extrachromosomal circular DNA (ecDNA), or megabase-pair amplified circular DNA, plays an essential role in intercellular heterogeneity and tumor cell revolution because of its non-Mendelian inheritance. We developed circlehunter ( https://github.com/suda-huanglab/circlehunter ), a tool for identifying ecDNA from ATAC-Seq data using the enhanced chromatin accessibility of ecDNA. Using simulated data, we showed that circlehunter has an F1 score of 0.93 at 30× local depth and read lengths as short as 35 bp. Based on 1312 ecDNAs predicted from 94 publicly available datasets of ATAC-Seq assays, we found 37 oncogenes contained in these ecDNAs with amplification characteristics. In small cell lung cancer cell lines, ecDNA containing MYC leads to amplification of MYC and cis-regulates the expression of NEUROD1 , resulting in an expression pattern consistent with the NEUROD1 high expression subtype and sensitive to Aurora kinase inhibitors. This showcases that circlehunter could serve as a valuable pipeline for the investigation of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b270d5828c0c647a81a5f5624151881c87a6d379" target='_blank'>Circlehunter: a tool to identify extrachromosomal circular DNA from ATAC-Seq data</a></td>
          <td>
            Manqiu Yang, Shufan Zhang, Rong Jiang, Shaomu Chen, Moli Huang
          </td>
          <td>2023-05-22</td>
          <td>Oncogenesis</td>
          <td>12</td>
          <td>14</td>
        </tr>
    
        <tr id="Extrachromosomal circular DNA (eccDNA) represents a large category of nonmitochondrial and nonplasmid circular extrachromosomal DNA playing an indispensable role in various aspects such as tumorigenesis, immune responses, liquid biopsy, etc. However, characteristic and functions of eccDNA are fragmented, hiding behind abundant literatures and massive whole-genome sequencing (WGS) data, which have not been sufficiently used for identification of eccDNAs. Therefore, establishing an integrated repository portal is essential for identifying and analyzing eccDNAs. Here, we developed eccDNA Atlas (http://lcbb.swjtu.edu.cn/eccDNAatlas), a user-friendly database of eccDNAs that aims to provide a high-quality and integrated resource for browsing, searching and analyzing eccDNAs from multiple species. eccDNA Atlas currently containing 630,434 eccDNAs and 7,774 ecDNAs manually curated from literatures and 1,105 ecDNAs predicted by AmpliconArchitect based on WGS data involved in 66 diseases, 57 tissue and 319 cell lines. The content of each eccDNA entry included multiple aspects such as sequence, disease, function, characteristic, validation strategies, etc. Furthermore, abundant annotations and analyzing utilities were provided to explore existed eccDNAs in eccDNA Atlas or user-defined eccDNAs including oncogenes, typical enhancers, super enhancers, gene expression, survival, and genome visualization. Based on these resources, eccDNA Atlas will significantly improve our understanding of eccDNAs and serve as an important catalyst for future research of eccDNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d20a0db9b1f98cad436777c673672c6524a862" target='_blank'>eccDNA Atlas: a comprehensive resource of eccDNA catalog</a></td>
          <td>
            Tengwei Zhong, Wenqing Wang, Houyan Liu, Mao-Hua Zeng, Xinyu Zhao, Zhiyun Guo
          </td>
          <td>2022-11-08</td>
          <td>bioRxiv</td>
          <td>17</td>
          <td>11</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f879239a8ad5d9ddd955b9e253a086721f6d98e" target='_blank'>Map functional insulators with cross-platform Hi-C meta-analysis</a></td>
          <td>
            Wanying Xu, Jian Cui, Shanshan Zhang, Leina Lu, Xiaoxiao Liu, Yan Li, Fulai Jin
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e46744707cddff45ffabea1a5972ff429db150" target='_blank'>Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D</a></td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>