---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-05 09:41:58 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Spatial epitope barcoding reveals subclonal tumor patch behaviors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="
 
 Multicellular programs in the tumor microenvironment (TME) drive cancer pathogenesis and response to therapy but remain challenging to identify and profile clinically. Here, we present Spatial EcoTyper, a multimodal machine learning framework for large-scale profiling of spatially dependent cell states and multicellular ecosystems, termed spatial ecotypes (SEs).
 
 
 
 and Findings: By integrating five million single-cell and spot-level spatial transcriptomes from 10 diverse human neoplasms, including carcinomas and melanomas, we identified a rich tapestry of conserved regional plasticity across nine TME cell types. We then applied Spatial EcoTyper to deeply dissect these data, revealing nine novel SEs with broad conservation - each with unique biology, TME cell states, geospatial features, and clinical outcome associations. SEs were linked to specific biogeographic domains, including the tumor core, margin, and adjacent stroma, and were remarkably well-validated in 235 held-out tumor samples profiled by either single-cell RNA sequencing, sequencing-based spatial transcriptomics (10x Visium), or high-resolution imaging-based spatial transcriptomics (Vizgen MERSCOPE and NanoString CosMx). By deconvolving SEs from over 1,200 bulk tumor RNA-seq profiles, we discovered striking associations between SE levels and response to immune checkpoint inhibitors (ICIs), outperforming nearly 40 potential correlates of ICI response, including previously described signatures of tertiary lymphoid structures and tumor tissue PD-L1 expression. Notably, baseline levels of a proinflammatory SE - localized to the tumor core and defined by a hub of unique T cell, macrophage, fibroblast, and endothelial cell states - strongly forecasted ICI benefit, while another SE - localized to the tumor margin and defined by myofibroblast and hypoxia-associated endothelial cell states - portended ICI resistance.
 
 
 
 Our data reveal fundamental units of TME organization and demonstrate a platform for large-scale profiling of spatial cellular ecosystems in any tissue, with implications for improved forecasting of immunotherapy response and the development of novel therapeutic strategies.
 
 
 
 Wubing Zhang,Erin L. Brown,Abul Usmani,Noah Earland,Hyun Soo Jeon,Irfan Alahi,Nicholas P. Semenkovich,Janella C. Schwab,Chloé B. Steen,Chloe M. Sachs,Bilge Gungoren,Faridi Qaium,Peter K. Harris,Jacqueline Mudd,Qingyuan Cai,Antonella Bacchiocchi,Ryan C. Fields,Mario Sznol,Ruth Halaban,David Y. Chen,Aadel A. Chaudhuri,Aaron M. Newman. High-resolution profiling of the tumor microenvironment with spatial ecotypes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 153.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483a65f81564c748a7e6be963230e929d2717f09" target='_blank'>Abstract 153: High-resolution profiling of the tumor microenvironment with spatial ecotypes</a></td>
          <td>
            Wubing Zhang, Erin L. Brown, A. Usmani, N. Earland, Hyun Soo Jeon, I. Alahi, N. Semenkovich, Janella Schwab, Chloé B. Steen, Chloe M. Sachs, Bilge Gungoren, F. Qaium, P. Harris, Jacqueline L Mudd, Qingyuan Cai, A. Bacchiocchi, Ryan C. Fields, Mario Sznol, R. Halaban, David Y. Chen, A. Chaudhuri, Aaron M. Newman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00764ef398651a1802a4a862aa98e22aba39a33" target='_blank'>The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution.</a></td>
          <td>
            O. Rozenblatt-Rosen, A. Regev, Philipp Oberdoerffer, T. Nawy, A. Hupalowska, Jennifer E. Rood, O. Ashenberg, E. Cerami, R. Coffey, E. Demir, L. Ding, Edward D Esplin, J. Ford, Jeremy Goecks, Sharmistha Ghosh, J. Gray, J. Guinney, S. Hanlon, Shannon K Hughes, E. Hwang, C. Iacobuzio-Donahue, Judit Jané-Valbuena, B. Johnson, K. Lau, T. Lively, S. Mazzilli, D. Pe’er, S. Santagata, A. Shalek, D. Schapiro, M. Snyder, P. Sorger, A. Spira, S. Srivastava, Kai Tan, Robert West, Elizabeth H. Williams, D. Aberle, S. Achilefu, F. Ademuyiwa, Andrew C. Adey, R. Aft, Rachana Agarwal, Ruben A. Aguilar, Fatemeh Alikarami, V. Allaj, Christopher L Amos, R. Anders, Michael Angelo, K. Anton, J. Aster, O. Babur, Amir Bahmani, Akshay Balsubramani, D. Barrett, J. Beane, D. Bender, K. Bernt, L. Berry, C. Betts, Julie A. Bletz, Katie E. Blise, A. Boire, G. Boland, A. Borowsky, K. Bosse, M. Bott, E. Boyden, James B. Brooks, R. Bueno, Erik A. Burlingame, Q. Cai, Joshua D. Campbell, Wagma Caravan, H. Chaib, J. Chan, Y. Chang, D. Chatterjee, O. Chaudhary, Alyce A. Chen, Bob Chen, Changya Chen, Chia-hui Chen, F. Chen, Yu-An Chen, Milan G. Chheda, K. Chin, R. Chiu, Shih-Kai Chu, R. Chuaqui, Jaeyoung Chun, Luis H. Cisneros, Graham A Colditz, K. Cole, Natalie B Collins, K. Contrepois, L. Coussens, A. Creason, D. Crichton, C. Curtis, Tanja Davidsen, S. Davies, I. Bruijn, L. Dellostritto, A. Marzo, D. DeNardo, D. Diep, S. Diskin, Xengie Doan, Julia L. Drewes, Stephen Dubinett, Michael Dyer, J. Egger, Jennifer Eng, Barbara Engelhardt, Graham S. Erwin, L. Esserman, Alex S. Felmeister, H. Feiler, R. Fields, S. Fisher, K. Flaherty, Jennifer Flournoy, A. Fortunato, Allison Frangieh, Jennifer Frye, R. Fulton, Danielle Galipeau, Siting Gan, Jianjiong Gao, Long Gao, Peng Gao, Vianne R. Gao, Timothy A. Geiger, A. George, G. Getz, M. Giannakis, David L. Gibbs, W. Gillanders, S. Goedegebuure, A. Gould, K. Gowers, W. Greenleaf, Jeremy Gresham, Jennifer L. Guerriero, Tuhin K. Guha, A. Guimaraes, D. Gutman, N. Hacohen, S. Hanlon, C. R. Hansen, O. Harismendy, Kathleen A Harris, A. Hata, Akimasa Hayashi, Cody N. Heiser, K. Helvie, J. Herndon, Gilliam Hirst, F. Hodi, T. Hollmann, Aaron M. Horning, J. Hsieh, Shannon K Hughes, W. J. Huh, S. Hunger, Shelley E. Hwang, H. Ijaz, B. Izar, C. Jacobson, S. Janes, R. Jayasinghe, Lihua Jiang, Brett E. Johnson, Bruce Johnson, T. Ju, H. Kadara, K. Kaestner, J. Kagan, L. Kalinke, R. Keith, Aziz Khan, W. Kibbe, A. Kim, Erika Kim, Junhyong Kim, A. Kolodzie, Mateusz Kopytra, Eran Kotler, Robert Krueger, K. Krysan, Anshul B Kundaje, U. Ladabaum, B. Lake, Huy Lam, Rozelle Laquindanum, Ashley M. Laughney, Hayan Lee, M. Lenburg, Carin Leonard, I. Leshchiner, Rochelle Levy, Jerry Li, C. Lian, K. Lim, Jia-Ren Lin, Yiyun Lin, Qi Liu, Ruiyang Liu, William J. R. Longabaugh, T. Longacre, Cynthia X. Ma, M. C. Macedonia, T. Madison, C. Maher, A. Maitra, Netta Makinen, Danika Makowski, C. Maley, Z. Maliga, D. Mallo, J. Maris, Nicholas O. Markham, J. Marks, Daniel Martinez, R. J. Mashl, Ignas Masilionais, Jennifer L. Mason, J. Massagué, P. Massion, M. Mattar, R. Mazurchuk, L. Mazutis, Eliot T. McKinley, Joshua F. McMichael, Daniel Merrick, M. Meyerson, J. Miessner, G. Mills, M. Mills, Suman B Mondal, Motomi Mori, Y. Mori, Elizabeth Moses, Y. Mossé, J. Muhlich, G. Murphy, N. Navin, M. Nederlof, R. Ness, Stephanie A. Nevins, Milen Nikolov, A. J. Nirmal, G. Nolan, Edward Novikov, Brendan L O'Connell, M. Offin, S. Oh, Anastasiya Olson, A. Ooms, Miguel Ossandon, K. Owzar, Swapnil Parmar, Tasleema Patel, G. Patti, I. Pe’er, Tao Peng, Daniel Persson, Marvin Petty, Hanspeter Pfister, K. Polyak, K. Pourfarhangi, Sidharth V. Puram, Q. Qiu, Á. Quintanal-Villalonga, A. Raj, Marisol Ramirez-Solano, R. Rashid, Ashley N. Reeb, M. Reid, A. Resnick, Sheila M. Reynolds, J. Riesterer, S. Rodig, Joseph T. Roland, Sonia M. Rosenfield, Asaf Rotem, Sudipta Roy, C. Rudin, M. Ryser, Mariarita Santi-Vicini, Kazuhito Sato, D. Schrag, N. Schultz, C. Sears, R. Sears, S. Sen, T. Sen, A. Shalek, Jeff Sheng, Quanhu Sheng, K. Shoghi, M. Shrubsole, Y. Shyr, A. Sibley, Kiara Siex, A. Simmons, D. Singer, Shamilene Sivagnanam, M. Slyper, Artem Sokolov, Sheng-Kwei Song, Austin N. Southard-Smith, A. Spira, J. Stein, P. Storm, E. Stover, Siri H. Strand, Timothy Su, D. Sudar, Ryan Sullivan, Lea F. Surrey, M. Suvà, N. Terekhanova, Luke Ternes, Lisa Thammavong, Guillaume Thibault, G. Thomas, V. Thorsson, E. Todres, L. Tran, Madison A Tyler, Yasin Uzun, A. Vachani, E. Allen, S. Vandekar, D. Veis, S. Vigneau, A. Vossough, A. Waanders, N. Wagle, Liang-Bo Wang, M. Wendl, Robert West, Chi-Yun Wu, Hao Wu, Hung-Yi Wu, Matthew A. Wyczalkowski, Yubin Xie, Xiaolu Yang, C. Yapp, Wenbao Yu, Yinyin Yuan, Dadong Zhang, Kun Zhang, Mianlei Zhang, Nancy R. Zhang, Yantian Zhang, Yanyan Zhao, D. Zhou, Zilu Zhou, Houxiang Zhu, Qin Zhu, Xiangzhu Zhu, Yuankun Zhu, X. Zhuang
          </td>
          <td>2020-04-16</td>
          <td>Cell</td>
          <td>421</td>
          <td>226</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387582e61e90f4446a8f82bd409b53ca8b5a0ffb" target='_blank'>Spatially resolving cancer: from cell states to therapy.</a></td>
          <td>
            Guangsheng Pei, Yang Liu, Linghua Wang
          </td>
          <td>2025-10-01</td>
          <td>Trends in cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr id="Understanding the spatial distribution of cancer cells and how they interact with other cell types in the tumor microenvironment (TME) is crucial to elucidate the mechanisms of cancer development and progression. The identification of cell types in the TME relies on the distinct expression profiles of different cell types. Compared to sequencing-based spatial transcriptomics analysis, multiplex nucleic acid in-situ hybridization (ISH) at a single-molecule level is more focused and cost-effective. Based on ISH technology, RNAscope developed novel approaches to detect multiple gene expression profiles on tissue sections at a single-molecule level. It is highly sensitive and specific and can simultaneously quantify gene expression at sub-cellular levels. Although the technology has been applied to study spatial gene expression in various cancer types, few have applied it to elucidate the association between different markers in different cell compartments within the TME.This protocol describes an image analysis pipeline for multiplex fluorescent signals detected by RNAscope technology. It was developed based on a previous CellProfiler pipeline, facilitating spatial expression analyses across different cell compartments through a flexible, user-friendly interface for non-computational biologists. We describe specific steps to analyze the expression and spatial arrangement of four endogenous housekeeping genes as quality controls (QCs): POLR2A, PPIB, HPRT1, and UBC. We further apply these pipelines to analyze target genes in breast cancer samples, comparing those with and without skin metastasis. The analyses were demonstrated in one sample each using formalin-fixed paraffin-embedded tissue (FFPET) and fresh-frozen tissue (FFT) sections with the ISH steps followed the manufacturer’s protocol.
 
 
 Meng-Shin Shiao, Natini Jinawath, Katherine Copeland, Ariestya Indah Permata Sari. A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6302.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16fe30c659d6f8c68794471b6bb6c1c23bb6c9f4" target='_blank'>Abstract 6302: A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially</a></td>
          <td>
            M. Shiao, N. Jinawath, Katherine Copeland, Ariestya Indah Permata Sari
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine</a></td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>OMICS: A Journal of Integrative Biology</td>
          <td>9</td>
          <td>5</td>
        </tr>
    
        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes</a></td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr id="
 Non-small cell lung cancer (NSCLC) is a complex disease with varying pathological subtypes including adenocarcinomas and squamous cell carcinomas. After diagnosis, tumor staging provides important information about the extent of cancer in the body and anticipated response to treatment. NSCLC patients can have impaired immune responses within the tumor microenvironment (TME), leading to a progression of tumor growth and poorer prognosis. Accurate cell phenotyping combined with spatial profiling of the immune contexture and checkpoint expression, can provide a deeper understanding of complex cellular interactions underpinning the tumor-immune response. The aim of this study was to utilize spatial multiplexed imaging technology and associated data analysis methods to profile the immune contexture, as well as their spatial interactions with the tumor, in a set of tissue cores covering a range of NSCLC subtypes and tumor staging. Formalin-fixed paraffin-embedded (FFPE) NSCLC tissue microarrays (TMA), comprised of n=41 cores containing a range of carcinomas and pathological Tumor-Node-Metastasis (pTNM) stages (I-IV), were stained on a Leica Bond RX™ using the Akoya PhenoCode™ Signature Immuno-contexture Human Protein Panel, which includes markers for CD8, CD68, PD-1, PD-L1, FoxP3, and PanCK as a tumor indicator. Stained TMAs were scanned at 20x magnification on a PhenoImager HT multispectral imaging system. Image analysis was performed using Visiopharm software. Deep learning algorithms were applied to segment tumor and stroma regions of interest (ROI) and to accurately detect and classify specific cell phenotypes. Cell object data files per core were exported for spatial and neighborhood analysis using OracleBio’s proprietary python-based program, PhenoXplore. Immune cell counts, phenotypes and spatial interactions were generated within tumor and stroma ROI per core. Data included total and negative cell phenotype counts, cell density in tumor and stroma, as well as cell distance and neighborhood spatial interactions per core across NSCLC subtypes and stages in the TMA set. The combination of high-quality staining provided by the PhenoCode™ panel, coupled with deep learning quantitative phenotyping and spatial pattern analysis enables detailed characterization of the complex cellular interactions, at both the functional and spatial level, within the TME of NSCLC tissue. Data generated will support a greater understanding of the complex cellular interactions that can contribute to progression of NSCLC and may help guide future precision medicine strategies.
 Citation Format: Lorcan Sherry, Nicole Couper, Bethany Remeniuk, Karen McClymont, Bei Hopkins, Natalie Monteiro, Gabriel Reines March, Darren Locke. Quantitative spatial profiling of NSCLC subtypes across tumor stages using 6-plex multiplex imaging technology and AI-powered phenotyping analysis. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2023; Part 1 (Regular and Invited Abstracts); 2023 Apr 14-19; Orlando, FL. Philadelphia (PA): AACR; Cancer Res 2023;83(7_Suppl):Abstract nr 4633.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da14beea660ae867279ba111d2e9edb0c6727743" target='_blank'>Abstract 4633: Quantitative spatial profiling of NSCLC subtypes across tumor stages using 6-plex multiplex imaging technology and AI-powered phenotyping analysis</a></td>
          <td>
            L. Sherry, Nicole Couper, B. Remeniuk, Karen McClymont, Bei Hopkins, Natalie Monteiro, G. R. March, D. Locke
          </td>
          <td>2023-04-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="Tumors may contain billions of cells including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that is consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a42d55360375575f6b72f5d2adcce6dce183893" target='_blank'>Deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel Brody, Rebecca Eliscu, Brett E. Johnson, T. Mazor, Kevin W. Kelley, M. Potts, M. W. McDermott, E. Huang, D. Lim, R. Pieper, Mitchel S. Berger, J. Costello, J. Phillips, Michael C. Oldham
          </td>
          <td>2023-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>
    
        <tr id="
 Patient-derived xenografts (PDXs) recapitulate intratumoral spatial heterogeneity and simulate a tumor microenvironment in which human immune and stromal cells in the PDX are replaced over passages by murine cells partially lacking immune function. Histological imaging enables exploring the spatial heterogeneity and dynamics of cancer, stromal, and immune cell interactions as correlates of tumor stage and therapeutic response over passages. We created a repository of curated, haematoxylin and eosin (H&E) images as a community resource for addressing these questions.
 Images were generated at five sites within the NCI’s PDX Development and Trial Centers Research Network (PDXNet) and the NCI Patient-Derived Models Repository. Over 900 images, including 739 from PDXs and 190 from paired patients, are hosted on the Seven Bridges Genomics Cancer Genomics Cloud. They represent 42 cancer subtypes, including breast cancer (n=134), colon adenocarcinoma (COAD; n=94), pancreatic cancer (n=87), lung adenocarcinoma (LUAD; n=80), melanoma (n=71), and squamous cell lung cancer (LUSC; n=65). Paired human/PDX images are available for each of these cancers. Human and/or PDX images generated following patient treatment are available for 37 of the subtypes. Most images are from early passages (P0: 158; P1: 292; P2: 152; P3: 69; >P3: 55). Annotations include sex, age, race, ethnicity, and, for most images, pathological assessment of tissue-level percent cancer, stromal, and necrotic cell content (n=639) and tumor stage (n=650). RNA and exome sequencing data are available for 99 and 228 images, respectively, matched at the patient or sample level.
 Quality control was performed using HistoQC. Cells were segmented and labeled as neoplastic, necrotic, immune, stromal, or other using Hover-Net and predictions of total neoplastic cell area correlated with whole-slide pathological assessment of cancer cell percentage (COAD: r=0.51; LUSC: r=0.59). HD-Staining, another classification approach, was applied to a subset of images and our clinical annotations will facilitate validation of this and related methods. Features of 512 x 512 pixel tiles were computed using the Inception V3 convolutional neural network pre-trained on ImageNet. Unsupervised clustering of these features demonstrate inter-patient heterogeneity within pathologist-annotated tumor regions. A classifier developed using pathologist-annotated cancer, stromal, and necrotic regions and trained on the features in LUSC images (n=10 images) achieved a cross-validation accuracy of 96% for cancer tiles across (n=5) LUAD images. Accuracy was lower for stromal classification (90%), likely reflecting current limitations of our small, but growing, labeled training set.
 Our repository of clinically-annotated PDX H&E images should aid the community in studying spatial heterogeneity and in training deep learning-based image analysis methods.
 Citation Format: Brian S. White, Xingyi Woo, Soner Koc, Todd Sheridan, Steven B. Neuhauser, Akshat M. Savaliya, Lacey E. Dobrolecki, John D. Landua, Matthew H. Bailey, Maihi Fujita, Kurt W. Evans, Bingliang Fang, Junya Fujimoto, Maria Gabriela Raso, Shidan Wang, Guanghua Xiao, Yang Xie, Sherri R. Davies, Ryan C. Fields, R Jay Mashl, Jacqueline L. Mudd, Yeqing Chen, Min Xiao, Xiaowei Xu, Melinda G. Hollingshead, Shahanawaz Jiwani, PDXNet Consortium, Yvonne A. Evrard, Tiffany A. Wallace, Jeffrey A. Moscow, James H. Doroshow, Nicholas Mitsiades, Salma Kaochar, Chong-xian Pan, Moon S. Chen, Luis G. Carvajal-Carmona, Alana L. Welm, Bryan E. Welm, Michael T. Lewis, Ramaswamy Govindan, Li Ding, Shunqiang Li, Meenhard Herlyn, Michael A. Davies, Jack A. Roth, Funda Meric-Bernstam, Carol J. Bult, Brandi Davis-Dusenbery, Dennis A. Dean, Jeffrey H. Chuang. A repository of PDX histology images for exploring spatial heterogeneity and cancer dynamics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2022; 2022 Apr 8-13. Philadelphia (PA): AACR; Cancer Res 2022;82(12_Suppl):Abstract nr 1202.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4501da22be82042fdbfb40628240b444058d0f" target='_blank'>Abstract 1202: A repository of PDX histology images for exploring spatial heterogeneity and cancer dynamics</a></td>
          <td>
            Brian S. White, X. Woo, Soner Koc, Todd B Sheridan, Steven B. Neuhauser, Akshat Savaliya, L. Dobrolecki, John D. Landua, Matthew H. Bailey, Maihi Fujita, K. Evans, B. Fang, J. Fujimoto, M. G. Raso, Shidan Wang, Guanghua Xiao, Yang Xie, S. Davies, R. Fields, R. J. Mashl, J. Mudd, Yeqing Chen, M. Xiao, Xiaowei Xu, M. Hollingshead, S. Jiwani, Yvonne A. Evrard, Tiffany A. Wallace, J. Moscow, J. Doroshow, N. Mitsiades, S. Kaochar, C. Pan, Moon S. Chen, L. Carvajal-Carmona, A. Welm, B. Welm, M. Lewis, R. Govindan, Liang Wen Ding, Shunqiang Li, M. Herlyn, M. Davies, J. Roth, F. Meric-Bernstam, C. Bult, Brandi N. Davis-Dusenbery, Dennis A. Dean, Jeffrey H. Chuang
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>123</td>
        </tr>
    
        <tr id="Simple Summary Tumors contain cancerous cells with different mutations and different types of non-cancerous cells. It is important to understand the molecular properties of these cells in order to develop effective treatments for cancers. We describe a new strategy for studying tumor heterogeneity and apply it to human astrocytomas, a type of brain tumor. By slicing astrocytomas into a large number of serial sections and analyzing patterns of mutation frequencies, we reconstruct the evolutionary history of cancer cell mutations in each tumor. By comparing these patterns to gene activity measured in the same sections, we identify molecular features that optimally distinguish different types of cancerous and non-cancerous cells and validate these with different techniques, including analysis of individual nuclei. By integrating analyses of multiple molecular species at multiple scales, our strategy provides a powerful approach for precisely deconstructing intratumoral heterogeneity. Abstract Tumors may contain billions of cells, including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that are consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19f10a20bb87365187cbad7becc484185a46f9af" target='_blank'>Deconstructing Intratumoral Heterogeneity through Multiomic and Multiscale Analysis of Serial Sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel Brody, Rebecca Eliscu, Brett E Johnson, T. Mazor, Kevin W Kelley, Matthew B Potts, Michael W McDermott, Eric J. Huang, Daniel A Lim, Russell O Pieper, Mitchell S. Berger, Joseph F Costello, Joanna J. Phillips, Michael C. Oldham
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>2</td>
          <td>26</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>