---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-02 09:29:56 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Spatial epitope barcoding reveals subclonal tumor patch behaviors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Spatial and genomic heterogeneity of tumors are crucial factors influencing cancer progression, treatment, and survival. However, a technology for direct mapping the clones in the tumor tissue based on somatic point mutations is lacking. Here, we propose Tumoroscope, the first probabilistic model that accurately infers cancer clones and their localization in close to single-cell resolution by integrating pathological images, whole exome sequencing, and spatial transcriptomics data. In contrast to previous methods, Tumoroscope explicitly addresses the problem of deconvoluting the proportions of clones in spatial transcriptomics spots. Applied to a reference prostate cancer dataset and a newly generated breast cancer dataset, Tumoroscope reveals spatial patterns of clone colocalization and mutual exclusion in sub-areas of the tumor tissue. We further infer clone-specific gene expression levels and the most highly expressed genes for each clone. In summary, Tumoroscope enables an integrated study of the spatial, genomic, and phenotypic organization of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope</a></td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E Mold, Xinsong Chen, Johan Hartman, D. Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>5</td>
          <td>36</td>
        </tr>
    
        <tr id="Spatial and genomic heterogeneity of tumors is the key for cancer progression, treatment, and survival. However, a technology for direct mapping the clones in the tumor tissue based on point mutations is lacking. Here, we propose Tumoroscope, the first probabilistic model that accurately infers cancer clones and their high-resolution localization by integrating pathological images, whole exome sequencing, and spatial transcriptomics data. In contrast to previous methods, Tumoroscope explicitly addresses the problem of deconvoluting the proportions of clones in spatial transcriptomics spots. Applied to a reference prostate cancer dataset and a newly generated breast cancer dataset, Tumoroscope reveals spatial patterns of clone colocalization and mutual exclusion in sub-areas of the tumor tissue. We further infer clone-specific gene expression levels and the most highly expressed genes for each clone. In summary, Tumoroscope enables an integrated study of the spatial, genomic, and phenotypic organization of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6a2790cf1b3765aa1bb21c208228815ba2cf26" target='_blank'>Tumoroscope: a probabilistic model for mapping cancer clones in tumor tissues</a></td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Łukasz Rączkowski, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, J. Hartman, D. Nowis, A. Carbone, J. Lagergren, E. Szczurek
          </td>
          <td>2022-09-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>43</td>
        </tr>
    
        <tr id="Understanding the spatial distribution of cancer cells and how they interact with other cell types in the tumor microenvironment (TME) is crucial to elucidate the mechanisms of cancer development and progression. The identification of cell types in the TME relies on the distinct expression profiles of different cell types. Compared to sequencing-based spatial transcriptomics analysis, multiplex nucleic acid in-situ hybridization (ISH) at a single-molecule level is more focused and cost-effective. Based on ISH technology, RNAscope developed novel approaches to detect multiple gene expression profiles on tissue sections at a single-molecule level. It is highly sensitive and specific and can simultaneously quantify gene expression at sub-cellular levels. Although the technology has been applied to study spatial gene expression in various cancer types, few have applied it to elucidate the association between different markers in different cell compartments within the TME.This protocol describes an image analysis pipeline for multiplex fluorescent signals detected by RNAscope technology. It was developed based on a previous CellProfiler pipeline, facilitating spatial expression analyses across different cell compartments through a flexible, user-friendly interface for non-computational biologists. We describe specific steps to analyze the expression and spatial arrangement of four endogenous housekeeping genes as quality controls (QCs): POLR2A, PPIB, HPRT1, and UBC. We further apply these pipelines to analyze target genes in breast cancer samples, comparing those with and without skin metastasis. The analyses were demonstrated in one sample each using formalin-fixed paraffin-embedded tissue (FFPET) and fresh-frozen tissue (FFT) sections with the ISH steps followed the manufacturer’s protocol.
 
 
 Meng-Shin Shiao, Natini Jinawath, Katherine Copeland, Ariestya Indah Permata Sari. A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6302.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16fe30c659d6f8c68794471b6bb6c1c23bb6c9f4" target='_blank'>Abstract 6302: A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially</a></td>
          <td>
            M. Shiao, N. Jinawath, Katherine Copeland, Ariestya Indah Permata Sari
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a95db304a3a2602d501dd51f061db80be330a3c" target='_blank'>Spatial characterization of single tumor cells by proteomics</a></td>
          <td>
            
          </td>
          <td>2022-05-19</td>
          <td>Nature Biotechnology</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr id="
 Non-small cell lung cancer (NSCLC) is a complex disease with varying pathological subtypes including adenocarcinomas and squamous cell carcinomas. After diagnosis, tumor staging provides important information about the extent of cancer in the body and anticipated response to treatment. NSCLC patients can have impaired immune responses within the tumor microenvironment (TME), leading to a progression of tumor growth and poorer prognosis. Accurate cell phenotyping combined with spatial profiling of the immune contexture and checkpoint expression, can provide a deeper understanding of complex cellular interactions underpinning the tumor-immune response. The aim of this study was to utilize spatial multiplexed imaging technology and associated data analysis methods to profile the immune contexture, as well as their spatial interactions with the tumor, in a set of tissue cores covering a range of NSCLC subtypes and tumor staging. Formalin-fixed paraffin-embedded (FFPE) NSCLC tissue microarrays (TMA), comprised of n=41 cores containing a range of carcinomas and pathological Tumor-Node-Metastasis (pTNM) stages (I-IV), were stained on a Leica Bond RX™ using the Akoya PhenoCode™ Signature Immuno-contexture Human Protein Panel, which includes markers for CD8, CD68, PD-1, PD-L1, FoxP3, and PanCK as a tumor indicator. Stained TMAs were scanned at 20x magnification on a PhenoImager HT multispectral imaging system. Image analysis was performed using Visiopharm software. Deep learning algorithms were applied to segment tumor and stroma regions of interest (ROI) and to accurately detect and classify specific cell phenotypes. Cell object data files per core were exported for spatial and neighborhood analysis using OracleBio’s proprietary python-based program, PhenoXplore. Immune cell counts, phenotypes and spatial interactions were generated within tumor and stroma ROI per core. Data included total and negative cell phenotype counts, cell density in tumor and stroma, as well as cell distance and neighborhood spatial interactions per core across NSCLC subtypes and stages in the TMA set. The combination of high-quality staining provided by the PhenoCode™ panel, coupled with deep learning quantitative phenotyping and spatial pattern analysis enables detailed characterization of the complex cellular interactions, at both the functional and spatial level, within the TME of NSCLC tissue. Data generated will support a greater understanding of the complex cellular interactions that can contribute to progression of NSCLC and may help guide future precision medicine strategies.
 Citation Format: Lorcan Sherry, Nicole Couper, Bethany Remeniuk, Karen McClymont, Bei Hopkins, Natalie Monteiro, Gabriel Reines March, Darren Locke. Quantitative spatial profiling of NSCLC subtypes across tumor stages using 6-plex multiplex imaging technology and AI-powered phenotyping analysis. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2023; Part 1 (Regular and Invited Abstracts); 2023 Apr 14-19; Orlando, FL. Philadelphia (PA): AACR; Cancer Res 2023;83(7_Suppl):Abstract nr 4633.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da14beea660ae867279ba111d2e9edb0c6727743" target='_blank'>Abstract 4633: Quantitative spatial profiling of NSCLC subtypes across tumor stages using 6-plex multiplex imaging technology and AI-powered phenotyping analysis</a></td>
          <td>
            L. Sherry, Nicole Couper, B. Remeniuk, Karen McClymont, Bei Hopkins, Natalie Monteiro, G. R. March, D. Locke
          </td>
          <td>2023-04-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.</a></td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>2</td>
          <td>2</td>
        </tr>
    
        <tr id="Tumors may contain billions of cells including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that is consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a42d55360375575f6b72f5d2adcce6dce183893" target='_blank'>Deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel J. Brody, Rebecca Eliscu, Brett E. Johnson, T. Mazor, Kevin W. Kelley, M. Potts, M. W. McDermott, E. Huang, D. Lim, R. Pieper, Mitchel S. Berger, J. Costello, J. Phillips, Michael C. Oldham
          </td>
          <td>2023-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>
    
        <tr id="Cancer genomic medicine relies on targeting driver genes. However, current catalogues of cancer drivers are mostly based on indirect measurements of mutation frequencies, positions or types, rather than their effect on clonal expansions in vivo. Moreover, non-genetic drivers are largely unknown, as are the epigenetic and transcriptomic effects of genetic drivers. Here we perform spatial computational inference on multiomic data with matched whole-genome sequencing, ATAC-seq and RNA-seq. Using 436 samples, we directly quantify the contribution, or lack thereof, of putative driver genes to subclonal expansions in vivo in 30 colorectal carcinomas (4-33 samples per patient, median=15). Although subclonal neutral evolution was widespread (13/26 cases with sufficient data), there were cases with clear evidence of subclonal selection (6/26) in which we measured epigenetic and transcriptomic differences between subclones in vivo. In 7/26 cases we could not distinguish between neutral or selective evolution with the available data. We identified expanding subclones that were not driven by known genetic alterations, and propose candidate epigenetic drivers. We identified the distinguishing patterns of genomic heterogeneity produced in fast, exponentially growing tumours (7/26) versus neoplasms growing only at the periphery (19/26), as well as identifying clonally intermixed (16/28 cases with sufficient data) versus segregated malignancies (10/28). Our model-based approach measures genetic and non-genetic subclonal selection, or lack thereof, in space and time and allows in vivo comparisons of the emergent phenotypic properties of subclones within human tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef941f1c1e1cf7097314709760fd3d658c9657e" target='_blank'>Assessment of the evolutionary consequence of putative driver mutations in colorectal cancer with spatial multiomic data</a></td>
          <td>
            Timon Heide, J. Househam, George D. Cresswell, I. Spiteri, Claire Lynn, C. Kimberley, M. Mossner, L. Zapata, C. Gabbutt, Daniele Ramazzotti, Bingjie Chen, J. Fernández-Mateos, C. James, A. Vinceti, A. Berner, Melissa Schmidt, E. Lakatos, A. Baker, D. Nichol, Helena Costa, Miriam Mitchinson, B. Werner, F. Iorio, M. Jansen, Chris P. Barnes, G. Caravagna, D. Shibata, J. Bridgewater, M. Rodríguez-Justo, L. Magnani, T. Graham, A. Sottoriva
          </td>
          <td>2021-07-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>64</td>
        </tr>
    
        <tr id="
 Patient-derived xenografts (PDXs) recapitulate intratumoral spatial heterogeneity and simulate a tumor microenvironment in which human immune and stromal cells in the PDX are replaced over passages by murine cells partially lacking immune function. Histological imaging enables exploring the spatial heterogeneity and dynamics of cancer, stromal, and immune cell interactions as correlates of tumor stage and therapeutic response over passages. We created a repository of curated, haematoxylin and eosin (H&E) images as a community resource for addressing these questions.
 Images were generated at five sites within the NCI’s PDX Development and Trial Centers Research Network (PDXNet) and the NCI Patient-Derived Models Repository. Over 900 images, including 739 from PDXs and 190 from paired patients, are hosted on the Seven Bridges Genomics Cancer Genomics Cloud. They represent 42 cancer subtypes, including breast cancer (n=134), colon adenocarcinoma (COAD; n=94), pancreatic cancer (n=87), lung adenocarcinoma (LUAD; n=80), melanoma (n=71), and squamous cell lung cancer (LUSC; n=65). Paired human/PDX images are available for each of these cancers. Human and/or PDX images generated following patient treatment are available for 37 of the subtypes. Most images are from early passages (P0: 158; P1: 292; P2: 152; P3: 69; >P3: 55). Annotations include sex, age, race, ethnicity, and, for most images, pathological assessment of tissue-level percent cancer, stromal, and necrotic cell content (n=639) and tumor stage (n=650). RNA and exome sequencing data are available for 99 and 228 images, respectively, matched at the patient or sample level.
 Quality control was performed using HistoQC. Cells were segmented and labeled as neoplastic, necrotic, immune, stromal, or other using Hover-Net and predictions of total neoplastic cell area correlated with whole-slide pathological assessment of cancer cell percentage (COAD: r=0.51; LUSC: r=0.59). HD-Staining, another classification approach, was applied to a subset of images and our clinical annotations will facilitate validation of this and related methods. Features of 512 x 512 pixel tiles were computed using the Inception V3 convolutional neural network pre-trained on ImageNet. Unsupervised clustering of these features demonstrate inter-patient heterogeneity within pathologist-annotated tumor regions. A classifier developed using pathologist-annotated cancer, stromal, and necrotic regions and trained on the features in LUSC images (n=10 images) achieved a cross-validation accuracy of 96% for cancer tiles across (n=5) LUAD images. Accuracy was lower for stromal classification (90%), likely reflecting current limitations of our small, but growing, labeled training set.
 Our repository of clinically-annotated PDX H&E images should aid the community in studying spatial heterogeneity and in training deep learning-based image analysis methods.
 Citation Format: Brian S. White, Xingyi Woo, Soner Koc, Todd Sheridan, Steven B. Neuhauser, Akshat M. Savaliya, Lacey E. Dobrolecki, John D. Landua, Matthew H. Bailey, Maihi Fujita, Kurt W. Evans, Bingliang Fang, Junya Fujimoto, Maria Gabriela Raso, Shidan Wang, Guanghua Xiao, Yang Xie, Sherri R. Davies, Ryan C. Fields, R Jay Mashl, Jacqueline L. Mudd, Yeqing Chen, Min Xiao, Xiaowei Xu, Melinda G. Hollingshead, Shahanawaz Jiwani, PDXNet Consortium, Yvonne A. Evrard, Tiffany A. Wallace, Jeffrey A. Moscow, James H. Doroshow, Nicholas Mitsiades, Salma Kaochar, Chong-xian Pan, Moon S. Chen, Luis G. Carvajal-Carmona, Alana L. Welm, Bryan E. Welm, Michael T. Lewis, Ramaswamy Govindan, Li Ding, Shunqiang Li, Meenhard Herlyn, Michael A. Davies, Jack A. Roth, Funda Meric-Bernstam, Carol J. Bult, Brandi Davis-Dusenbery, Dennis A. Dean, Jeffrey H. Chuang. A repository of PDX histology images for exploring spatial heterogeneity and cancer dynamics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2022; 2022 Apr 8-13. Philadelphia (PA): AACR; Cancer Res 2022;82(12_Suppl):Abstract nr 1202.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4501da22be82042fdbfb40628240b444058d0f" target='_blank'>Abstract 1202: A repository of PDX histology images for exploring spatial heterogeneity and cancer dynamics</a></td>
          <td>
            Brian S. White, X. Woo, Soner Koc, Todd B Sheridan, Steven B. Neuhauser, Akshat Savaliya, L. Dobrolecki, John D. Landua, Matthew H. Bailey, Maihi Fujita, K. Evans, B. Fang, J. Fujimoto, M. G. Raso, Shidan Wang, Guanghua Xiao, Yang Xie, S. Davies, R. Fields, R. J. Mashl, J. Mudd, Yeqing Chen, M. Xiao, Xiaowei Xu, M. Hollingshead, S. Jiwani, Yvonne A. Evrard, Tiffany A. Wallace, J. Moscow, J. Doroshow, N. Mitsiades, S. Kaochar, C. Pan, Moon S. Chen, L. Carvajal-Carmona, A. Welm, B. Welm, M. Lewis, R. Govindan, Liang Wen Ding, Shunqiang Li, M. Herlyn, M. Davies, J. Roth, F. Meric-Bernstam, C. Bult, B. Davis-Dusenbery, Dennis A. Dean, Jeffrey H. Chuang
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>122</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>