---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-07 09:28:53 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Spatial epitope barcoding reveals subclonal tumor patch behaviors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00764ef398651a1802a4a862aa98e22aba39a33" target='_blank'>The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution.</a></td>
          <td>
            O. Rozenblatt-Rosen, A. Regev, Philipp Oberdoerffer, T. Nawy, A. Hupalowska, Jennifer E. Rood, Orr Ashenberg, E. Cerami, R. Coffey, E. Demir, L. Ding, Edward D. Esplin, J. Ford, Jeremy Goecks, Sharmistha Ghosh, J. Gray, J. Guinney, S. Hanlon, Shannon K Hughes, E. Hwang, C. Iacobuzio-Donahue, Judit Jané-Valbuena, B. Johnson, K. Lau, T. Lively, S. Mazzilli, D. Pe’er, S. Santagata, A. Shalek, D. Schapiro, M. Snyder, P. Sorger, A. Spira, S. Srivastava, Kai Tan, Robert West, Elizabeth H. Williams, D. Aberle, S. Achilefu, F. Ademuyiwa, Andrew C. Adey, R. Aft, Rachana Agarwal, Ruben A. Aguilar, Fatemeh Alikarami, V. Allaj, Christopher L Amos, R. Anders, Michael Angelo, K. Anton, J. Aster, O. Babur, Amir Bahmani, Akshay Balsubramani, D. Barrett, J. Beane, D. Bender, K. Bernt, L. Berry, C. Betts, Julie A. Bletz, Katie E. Blise, A. Boire, G. Boland, A. Borowsky, K. Bosse, M. Bott, E. Boyden, James B. Brooks, R. Bueno, Erik A. Burlingame, Q. Cai, Joshua D. Campbell, Wagma Caravan, H. Chaib, J. Chan, Y. Chang, D. Chatterjee, O. Chaudhary, Alyce A. Chen, Bob Chen, Changya Chen, Chia-Hui Chen, F. Chen, Yu-An Chen, M. Chheda, K. Chin, R. Chiu, Shih-Kai Chu, R. Chuaqui, Jaeyoung Chun, Luis H. Cisneros, G. Colditz, K. Cole, Natalie B Collins, K. Contrepois, L. Coussens, A. Creason, D. Crichton, C. Curtis, Tanja Davidsen, S. Davies, I. Bruijn, L. Dellostritto, A. Marzo, D. DeNardo, D. Diep, S. Diskin, Xengie Doan, Julia L. Drewes, Stephen Dubinett, Michael Dyer, J. Egger, Jennifer Eng, Barbara Engelhardt, Graham S. Erwin, L. Esserman, Alex S. Felmeister, H. Feiler, R. Fields, S. Fisher, K. Flaherty, Jennifer Flournoy, A. Fortunato, Allison Frangieh, Jennifer Frye, R. Fulton, Danielle Galipeau, Siting Gan, Jianjiong Gao, Long Gao, Peng Gao, Vianne R. Gao, Timothy A. Geiger, A. George, G. Getz, M. Giannakis, David L. Gibbs, W. Gillanders, S. Goedegebuure, A. Gould, K. Gowers, W. Greenleaf, Jeremy Gresham, Jennifer L. Guerriero, Tuhin K. Guha, A. Guimaraes, D. Gutman, N. Hacohen, S. Hanlon, C. R. Hansen, O. Harismendy, Kathleen A Harris, A. Hata, Akimasa Hayashi, Cody N. Heiser, K. Helvie, J. Herndon, Gilliam Hirst, F. Hodi, T. Hollmann, Aaron M. Horning, J. Hsieh, Shannon K Hughes, W. J. Huh, S. Hunger, Shelley E. Hwang, H. Ijaz, B. Izar, C. Jacobson, S. Janes, R. Jayasinghe, Lihua Jiang, Brett E. Johnson, Bruce Johnson, T. Ju, H. Kadara, K. Kaestner, J. Kagan, L. Kalinke, R. Keith, Aziz Khan, W. Kibbe, A. Kim, Erika Kim, Junhyong Kim, A. Kolodzie, Mateusz Kopytra, Eran Kotler, Robert Krueger, K. Krysan, A. Kundaje, U. Ladabaum, B. Lake, Huy Lam, Rozelle Laquindanum, Ashley M. Laughney, Hayan Lee, M. Lenburg, Carin Leonard, I. Leshchiner, Rochelle Levy, Jerry Li, C. Lian, K. Lim, Jia-Ren Lin, Yiyun Lin, Qi Liu, Ruiyang Liu, William J. R. Longabaugh, T. Longacre, Cynthia X. Ma, M. C. Macedonia, T. Madison, C. Maher, A. Maitra, Netta Makinen, Danika Makowski, C. Maley, Z. Maliga, D. Mallo, J. Maris, Nicholas O. Markham, J. Marks, Daniel Martinez, R. J. Mashl, Ignas Masilionais, Jennifer L. Mason, J. Massagué, P. Massion, M. Mattar, R. Mazurchuk, L. Mazutis, Eliot T. McKinley, Joshua F. McMichael, Daniel Merrick, M. Meyerson, J. Miessner, G. Mills, M. Mills, Suman B Mondal, Motomi Mori, Y. Mori, Elizabeth Moses, Y. Mossé, J. Muhlich, G. Murphy, N. Navin, M. Nederlof, R. Ness, Stephanie A. Nevins, Milen Nikolov, A. Nirmal, G. Nolan, Edward Novikov, Brendan L O'Connell, M. Offin, S. Oh, Anastasiya Olson, A. Ooms, Miguel Ossandon, K. Owzar, Swapnil Parmar, Tasleema Patel, G. Patti, I. Pe’er, Tao Peng, Daniel Persson, Marvin Petty, Hanspeter Pfister, K. Polyak, K. Pourfarhangi, Sidharth V. Puram, Q. Qiu, Á. Quintanal-Villalonga, A. Raj, Marisol Ramirez-Solano, R. Rashid, Ashley N. Reeb, M. Reid, A. Resnick, Sheila M. Reynolds, J. Riesterer, S. Rodig, Joseph T. Roland, Sonia M. Rosenfield, Asaf Rotem, Sudipta Roy, C. Rudin, M. Ryser, Mariarita Santi-Vicini, Kazuhito Sato, D. Schrag, N. Schultz, C. Sears, R. Sears, S. Sen, T. Sen, A. Shalek, Jeff Sheng, Quanhu Sheng, K. Shoghi, M. Shrubsole, Y. Shyr, A. Sibley, Kiara Siex, A. Simmons, D. Singer, Shamilene Sivagnanam, M. Slyper, Artem Sokolov, Sheng-Kwei Song, Austin N. Southard-Smith, A. Spira, J. Stein, P. Storm, E. Stover, Siri H. Strand, Timothy Su, D. Sudar, Ryan Sullivan, Lea F. Surrey, M. Suvà, Nadezhda V. Terekhanova, Luke Ternes, Lisa Thammavong, Guillaume Thibault, G. Thomas, V. Thorsson, E. Todres, L. Tran, Madison A Tyler, Yasin Uzun, A. Vachani, E. Allen, S. Vandekar, D. Veis, S. Vigneau, A. Vossough, A. Waanders, N. Wagle, Liang-Bo Wang, M. Wendl, Robert West, Chi-Yun Wu, Hao Wu, Hung-Yi Wu, Matthew A. Wyczalkowski, Yubin Xie, Xiaolu Yang, C. Yapp, Wenbao Yu, Yinyin Yuan, Dadong Zhang, Kun Zhang, Mianlei Zhang, Nancy R. Zhang, Yantian Zhang, Yanyan Zhao, D. Zhou, Zilu Zhou, Houxiang Zhu, Qin Zhu, Xiangzhu Zhu, Yuankun Zhu, X. Zhuang
          </td>
          <td>2020-04-16</td>
          <td>Cell</td>
          <td>362</td>
          <td>226</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a95db304a3a2602d501dd51f061db80be330a3c" target='_blank'>Spatial characterization of single tumor cells by proteomics</a></td>
          <td>
            
          </td>
          <td>2022-05-19</td>
          <td>Nature Biotechnology</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da14beea660ae867279ba111d2e9edb0c6727743" target='_blank'>Abstract 4633: Quantitative spatial profiling of NSCLC subtypes across tumor stages using 6-plex multiplex imaging technology and AI-powered phenotyping analysis</a></td>
          <td>
            L. Sherry, Nicole Couper, B. Remeniuk, Karen McClymont, Bei Hopkins, Natalie Monteiro, G. R. March, D. Locke
          </td>
          <td>2023-04-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.</a></td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>2</td>
          <td>2</td>
        </tr>
    
        <tr id="Tumors may contain billions of cells including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that is consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a42d55360375575f6b72f5d2adcce6dce183893" target='_blank'>Deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel J. Brody, Rebecca Eliscu, Brett E. Johnson, T. Mazor, Kevin W. Kelley, M. Potts, M. W. McDermott, E. Huang, D. Lim, R. Pieper, Mitchel S. Berger, J. Costello, J. Phillips, Michael C. Oldham
          </td>
          <td>2023-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4501da22be82042fdbfb40628240b444058d0f" target='_blank'>Abstract 1202: A repository of PDX histology images for exploring spatial heterogeneity and cancer dynamics</a></td>
          <td>
            Brian S. White, X. Woo, Soner Koc, Todd B Sheridan, Steven B. Neuhauser, Akshat Savaliya, L. Dobrolecki, John D. Landua, Matthew H. Bailey, Maihi Fujita, K. Evans, B. Fang, J. Fujimoto, M. G. Raso, Shidan Wang, Guanghua Xiao, Yang Xie, S. Davies, R. Fields, R. J. Mashl, J. Mudd, Yeqing Chen, M. Xiao, Xiaowei Xu, M. Hollingshead, S. Jiwani, Yvonne A. Evrard, Tiffany A. Wallace, J. Moscow, J. Doroshow, N. Mitsiades, S. Kaochar, C. Pan, Moon S. Chen, L. Carvajal-Carmona, A. Welm, B. Welm, M. Lewis, R. Govindan, Liang Wen Ding, Shunqiang Li, M. Herlyn, M. Davies, J. Roth, F. Meric-Bernstam, C. Bult, B. Davis-Dusenbery, Dennis A. Dean, Jeffrey H. Chuang
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>120</td>
        </tr>
    
        <tr id="Simple Summary Tumors contain cancerous cells with different mutations and different types of non-cancerous cells. It is important to understand the molecular properties of these cells in order to develop effective treatments for cancers. We describe a new strategy for studying tumor heterogeneity and apply it to human astrocytomas, a type of brain tumor. By slicing astrocytomas into a large number of serial sections and analyzing patterns of mutation frequencies, we reconstruct the evolutionary history of cancer cell mutations in each tumor. By comparing these patterns to gene activity measured in the same sections, we identify molecular features that optimally distinguish different types of cancerous and non-cancerous cells and validate these with different techniques, including analysis of individual nuclei. By integrating analyses of multiple molecular species at multiple scales, our strategy provides a powerful approach for precisely deconstructing intratumoral heterogeneity. Abstract Tumors may contain billions of cells, including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that are consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19f10a20bb87365187cbad7becc484185a46f9af" target='_blank'>Deconstructing Intratumoral Heterogeneity through Multiomic and Multiscale Analysis of Serial Sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel J. Brody, Rebecca Eliscu, Brett E Johnson, T. Mazor, Kevin W. Kelley, Matthew B. Potts, Michael W McDermott, Eric J. Huang, Daniel A. Lim, Russell O Pieper, Mitchell S. Berger, Joseph F Costello, Joanna J Phillips, Michael C. Oldham
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr id="Many malignant cancers like glioblastoma are highly adaptive diseases that dynamically change their regional biology to survive and thrive under diverse microenvironmental and therapeutic pressures. While the concept of intra‐tumoral heterogeneity has become a major paradigm in cancer research and care, systematic approaches to assess and document bio‐variation in cancer are still in their infancy. Here we discuss existing approaches and challenges to documenting intra‐tumoral heterogeneity and emerging computational approaches that leverage artificial intelligence to begin to overcome these limitations. We propose how these emerging techniques can be coupled with a diversity of molecular tools to address intra‐tumoral heterogeneity more systematically in research and in practice, especially across larger specimens and longitudinal analyses. Systematic documentation and characterization of heterogeneity across entire tumor specimens and their longitudinal evolution has the potential to improve our understanding and treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cfa7353f1105e92b3c276a5b736d18e45c4361e" target='_blank'>Standardizing analysis of intra‐tumoral heterogeneity with computational pathology</a></td>
          <td>
            Ameesha Paliwal, K. Faust, Azhar Alshoumer, P. Diamandis
          </td>
          <td>2023-04-17</td>
          <td>Genes</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>