---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-25 12:38:17 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Spatial epitope barcoding reveals subclonal tumor patch behaviors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope</a></td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, Johan Hartman, D. Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>35</td>
        </tr>
    
        <tr id="Spatial and genomic heterogeneity of tumors is the key for cancer progression, treatment, and survival. However, a technology for direct mapping the clones in the tumor tissue based on point mutations is lacking. Here, we propose Tumoroscope, the first probabilistic model that accurately infers cancer clones and their high-resolution localization by integrating pathological images, whole exome sequencing, and spatial transcriptomics data. In contrast to previous methods, Tumoroscope explicitly addresses the problem of deconvoluting the proportions of clones in spatial transcriptomics spots. Applied to a reference prostate cancer dataset and a newly generated breast cancer dataset, Tumoroscope reveals spatial patterns of clone colocalization and mutual exclusion in sub-areas of the tumor tissue. We further infer clone-specific gene expression levels and the most highly expressed genes for each clone. In summary, Tumoroscope enables an integrated study of the spatial, genomic, and phenotypic organization of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6a2790cf1b3765aa1bb21c208228815ba2cf26" target='_blank'>Tumoroscope: a probabilistic model for mapping cancer clones in tumor tissues</a></td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Łukasz Rączkowski, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, J. Hartman, D. Nowis, A. Carbone, J. Lagergren, E. Szczurek
          </td>
          <td>2022-09-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>42</td>
        </tr>
    
        <tr id="ABSTRACT We have devised a barcoding strategy to recapitulate cancer evolution through the emergence of subclonal mutations of interest, whose effects can be monitored in a dynamic manner. This approach can be easily adapted for a variety of applications, including combined modeling of multiple mechanisms of drug resistance or repair of oncogenic driver mutations in addicted cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71a0b178ffc3b6487e5795be58c139e8c681d562" target='_blank'>Modeling intratumor heterogeneity through CRISPR-barcodes</a></td>
          <td>
            A. Guernet, S. Aaronson, Y. Anouar, L. Grumolato
          </td>
          <td>2016-09-06</td>
          <td>Molecular & Cellular Oncology</td>
          <td>5</td>
          <td>110</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00764ef398651a1802a4a862aa98e22aba39a33" target='_blank'>The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution.</a></td>
          <td>
            O. Rozenblatt-Rosen, A. Regev, Philipp Oberdoerffer, T. Nawy, A. Hupalowska, Jennifer E. Rood, Orr Ashenberg, E. Cerami, R. Coffey, E. Demir, L. Ding, E. Esplin, J. Ford, Jeremy Goecks, Sharmistha Ghosh, J. Gray, J. Guinney, S. Hanlon, Shannon K Hughes, E. Hwang, C. Iacobuzio-Donahue, Judit Jané-Valbuena, B. Johnson, K. Lau, T. Lively, S. Mazzilli, D. Pe’er, S. Santagata, A. Shalek, D. Schapiro, M. Snyder, P. Sorger, A. Spira, S. Srivastava, Kai Tan, Robert West, Elizabeth H. Williams, D. Aberle, S. Achilefu, F. Ademuyiwa, Andrew C. Adey, R. Aft, Rachana Agarwal, Ruben A. Aguilar, Fatemeh Alikarami, V. Allaj, Christopher L Amos, R. Anders, Michael Angelo, K. Anton, J. Aster, O. Babur, Amir Bahmani, Akshay Balsubramani, D. Barrett, J. Beane, D. Bender, K. Bernt, L. Berry, C. Betts, Julie A. Bletz, Katie E. Blise, A. Boire, G. Boland, A. Borowsky, K. Bosse, M. Bott, E. Boyden, James B. Brooks, R. Bueno, Erik A. Burlingame, Q. Cai, Joshua D. Campbell, Wagma Caravan, H. Chaib, J. Chan, Y. Chang, D. Chatterjee, O. Chaudhary, Alyce A. Chen, Bob Chen, Changya Chen, Chia-Hui Chen, F. Chen, Yu-An Chen, M. Chheda, K. Chin, R. Chiu, Shih-Kai Chu, R. Chuaqui, Jaeyoung Chun, Luis H. Cisneros, G. Colditz, K. Cole, Natalie B Collins, K. Contrepois, L. Coussens, A. Creason, D. Crichton, C. Curtis, Tanja Davidsen, S. Davies, I. Bruijn, L. Dellostritto, A. Marzo, D. DeNardo, D. Diep, S. Diskin, Xengie Doan, Julia L. Drewes, Stephen Dubinett, Michael Dyer, J. Egger, Jennifer Eng, Barbara Engelhardt, Graham S. Erwin, L. Esserman, Alex S. Felmeister, H. Feiler, R. Fields, S. Fisher, K. Flaherty, Jennifer Flournoy, A. Fortunato, Allison Frangieh, Jennifer Frye, R. Fulton, Danielle Galipeau, Siting Gan, Jianjiong Gao, Long Gao, Peng Gao, Vianne R. Gao, Timothy A. Geiger, A. George, G. Getz, M. Giannakis, David L. Gibbs, W. Gillanders, S. Goedegebuure, A. Gould, K. Gowers, W. Greenleaf, Jeremy Gresham, Jennifer L. Guerriero, Tuhin K. Guha, A. Guimaraes, D. Gutman, N. Hacohen, S. Hanlon, C. R. Hansen, O. Harismendy, Kathleen A Harris, A. Hata, Akimasa Hayashi, Cody N. Heiser, K. Helvie, J. Herndon, Gilliam Hirst, F. Hodi, T. Hollmann, Aaron M. Horning, J. Hsieh, Shannon K Hughes, W. J. Huh, S. Hunger, S. Hwang, H. Ijaz, B. Izar, C. Jacobson, S. Janes, R. Jayasinghe, Lihua Jiang, Brett E. Johnson, Bruce Johnson, T. Ju, H. Kadara, K. Kaestner, J. Kagan, L. Kalinke, R. Keith, Aziz Khan, W. Kibbe, A. Kim, E. Kim, Junhyong Kim, A. Kolodzie, Mateusz Kopytra, Eran Kotler, Robert Krueger, K. Krysan, A. Kundaje, U. Ladabaum, B. Lake, Huy Lam, Rozelle Laquindanum, Ashley M. Laughney, Hayan Lee, M. Lenburg, Carin Leonard, I. Leshchiner, Rochelle Levy, Jerry Li, C. Lian, K. Lim, Jia-Ren Lin, Yiyun Lin, Qi Liu, Ruiyang Liu, William J. R. Longabaugh, T. Longacre, Cynthia X. Ma, M. C. Macedonia, T. Madison, C. Maher, A. Maitra, Netta Makinen, Danika Makowski, C. Maley, Z. Maliga, D. Mallo, J. Maris, Nicholas O. Markham, J. Marks, Daniel Martinez, R. J. Mashl, Ignas Masilionais, Jennifer L. Mason, J. Massagué, P. Massion, M. Mattar, R. Mazurchuk, L. Mazutis, Eliot T. McKinley, Joshua F. McMichael, Daniel Merrick, M. Meyerson, J. Miessner, G. Mills, M. Mills, Suman B Mondal, Motomi Mori, Y. Mori, Elizabeth Moses, Y. Mossé, J. Muhlich, G. Murphy, N. Navin, M. Nederlof, R. Ness, Stephanie A. Nevins, Milen Nikolov, A. Nirmal, G. Nolan, Edward Novikov, Brendan L O'Connell, M. Offin, S. Oh, Anastasiya Olson, A. Ooms, Miguel Ossandon, K. Owzar, Swapnil Parmar, Tasleema Patel, G. Patti, I. Pe’er, Tao Peng, Daniel Persson, Marvin Petty, Hanspeter Pfister, K. Polyak, K. Pourfarhangi, Sidharth V. Puram, Q. Qiu, Á. Quintanal-Villalonga, A. Raj, Marisol Ramirez-Solano, R. Rashid, Ashley N. Reeb, M. Reid, A. Resnick, Sheila M. Reynolds, J. Riesterer, S. Rodig, Joseph T. Roland, Sonia M. Rosenfield, Asaf Rotem, Sudipta Roy, C. Rudin, M. Ryser, Mariarita Santi-Vicini, Kazuhito Sato, D. Schrag, N. Schultz, C. Sears, R. Sears, S. Sen, T. Sen, A. Shalek, Jeff Sheng, Quanhu Sheng, K. Shoghi, M. Shrubsole, Y. Shyr, A. Sibley, Kiara Siex, A. Simmons, D. Singer, Shamilene Sivagnanam, M. Slyper, Artem Sokolov, Sheng-Kwei Song, Austin N. Southard-Smith, A. Spira, J. Stein, P. Storm, E. Stover, Siri H. Strand, Timothy Su, D. Sudar, Ryan Sullivan, Lea F. Surrey, M. Suvà, Nadezhda V. Terekhanova, Luke Ternes, Lisa Thammavong, Guillaume Thibault, G. Thomas, V. Thorsson, E. Todres, L. Tran, Madison A Tyler, Yasin Uzun, A. Vachani, E. Allen, S. Vandekar, D. Veis, S. Vigneau, A. Vossough, A. Waanders, N. Wagle, Liang-Bo Wang, M. Wendl, Robert West, Chi-Yun Wu, Hao Wu, Hung-Yi Wu, Matthew A. Wyczalkowski, Yubin Xie, Xiaolu Yang, C. Yapp, Wenbao Yu, Yinyin Yuan, Dadong Zhang, Kun Zhang, Mianlei Zhang, Nancy R. Zhang, Yantian Zhang, Yanyan Zhao, D. Zhou, Zilu Zhou, Houxiang Zhu, Qin Zhu, Xiangzhu Zhu, Yuankun Zhu, X. Zhuang
          </td>
          <td>2020-04-16</td>
          <td>Cell</td>
          <td>346</td>
          <td>225</td>
        </tr>
    
        <tr id="Introduction Cancer is an evolutionary process in which the stepwise accumulation of genetic alterations is shaped by Darwinian selection. As a result, each tumour is composed of a complex mixture of clonal cell subpopulations containing a partially overlapping, but distinct, pattern of driver and passenger mutations. Such intratumor heterogeneity has dramatic consequences, not only for cancer progression and metastatic spread, but also for resistance to therapy. We recently devised a novel approach based on CRISPR/Cas9 technology to recapitulate and trace the emergence of a new mutation in a subset of cancer cells, thus enabling functional studies on a gene of interest in a context that mimics intratumor heterogeneity. Material and methods We used CRISPR/Cas9 technology to insert a potentially functional modification in the sequence of a gene of interest, coupled to a series of silent point mutations, serving as a genetic label for cell tracing. Because of HDR low efficiency, the modified allele can be introduced only in a small fraction of cells, thus mimicking the emergence of new mutant clones within a cancer mass population. In parallel, a second barcode, consisting of different silent mutations, was used as an internal control for possible CRISPR off-target cleavage. After exposing the cells to a given selective condition, the effects of the mutation of interest can be analysed by measuring the proportion of each barcode in genomic DNA by real-time quantitative PCR or deep sequencing. Results and discussions We used this CRISPR-barcoding strategy to model different mechanisms of non-small-cell lung cancer resistance to EGFR inhibitors, and we established a multiplex system to evaluate the efficacy of combined drug therapies aimed at preventing or delaying the emergence of resistant cells. Through a similar approach, we assessed the effects of repairing oncogenic driver mutations in addicted cancer cells directly at the genome level. Finally, we used a highly complex set of CRISPR-barcodes to trace intratumor heterogeneity, as a convenient alternative strategy to lentiviral barcode libraries. Conclusion CRISPR-barcoding is a fast and highly flexible means to investigate the effects of different kinds of genomic modifications in a broad range of functional assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e82c5aced4133b10bb0a553cfe7355fd594d0e92" target='_blank'>Po-327 Modelling intratumor heterogeneity through CRISPR-barcodes</a></td>
          <td>
            Nad Marzouka, P. Eriksson, David Lindgren, F. Liedberg, C. Bernardo, G. Sjödahl, Håkan Axelson, Mattias Höglund
          </td>
          <td>2018-06-01</td>
          <td>ESMO Open</td>
          <td>0</td>
          <td>39</td>
        </tr>
    
        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.</a></td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="Tumors may contain billions of cells including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that is consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a42d55360375575f6b72f5d2adcce6dce183893" target='_blank'>Deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel J. Brody, Rebecca Eliscu, Brett E. Johnson, T. Mazor, Kevin W. Kelley, M. Potts, M. W. McDermott, E. Huang, D. Lim, R. Pieper, Mitchel S. Berger, J. Costello, J. Phillips, Michael C. Oldham
          </td>
          <td>2023-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>
    
        <tr id="Cancer genomic medicine relies on targeting driver genes. However, current catalogues of cancer drivers are mostly based on indirect measurements of mutation frequencies, positions or types, rather than their effect on clonal expansions in vivo. Moreover, non-genetic drivers are largely unknown, as are the epigenetic and transcriptomic effects of genetic drivers. Here we perform spatial computational inference on multiomic data with matched whole-genome sequencing, ATAC-seq and RNA-seq. Using 436 samples, we directly quantify the contribution, or lack thereof, of putative driver genes to subclonal expansions in vivo in 30 colorectal carcinomas (4-33 samples per patient, median=15). Although subclonal neutral evolution was widespread (13/26 cases with sufficient data), there were cases with clear evidence of subclonal selection (6/26) in which we measured epigenetic and transcriptomic differences between subclones in vivo. In 7/26 cases we could not distinguish between neutral or selective evolution with the available data. We identified expanding subclones that were not driven by known genetic alterations, and propose candidate epigenetic drivers. We identified the distinguishing patterns of genomic heterogeneity produced in fast, exponentially growing tumours (7/26) versus neoplasms growing only at the periphery (19/26), as well as identifying clonally intermixed (16/28 cases with sufficient data) versus segregated malignancies (10/28). Our model-based approach measures genetic and non-genetic subclonal selection, or lack thereof, in space and time and allows in vivo comparisons of the emergent phenotypic properties of subclones within human tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef941f1c1e1cf7097314709760fd3d658c9657e" target='_blank'>Assessment of the evolutionary consequence of putative driver mutations in colorectal cancer with spatial multiomic data</a></td>
          <td>
            Timon Heide, J. Househam, George D. Cresswell, I. Spiteri, Claire Lynn, C. Kimberley, M. Mossner, L. Zapata, C. Gabbutt, Daniele Ramazzotti, Bingjie Chen, J. Fernández-Mateos, C. James, A. Vinceti, A. Berner, Melissa Schmidt, E. Lakatos, A. Baker, D. Nichol, Helena Costa, Miriam Mitchinson, B. Werner, F. Iorio, M. Jansen, Chris P. Barnes, G. Caravagna, D. Shibata, J. Bridgewater, M. Rodríguez-Justo, L. Magnani, T. Graham, A. Sottoriva
          </td>
          <td>2021-07-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>64</td>
        </tr>
    
        <tr id="Simple Summary Tumors contain cancerous cells with different mutations and different types of non-cancerous cells. It is important to understand the molecular properties of these cells in order to develop effective treatments for cancers. We describe a new strategy for studying tumor heterogeneity and apply it to human astrocytomas, a type of brain tumor. By slicing astrocytomas into a large number of serial sections and analyzing patterns of mutation frequencies, we reconstruct the evolutionary history of cancer cell mutations in each tumor. By comparing these patterns to gene activity measured in the same sections, we identify molecular features that optimally distinguish different types of cancerous and non-cancerous cells and validate these with different techniques, including analysis of individual nuclei. By integrating analyses of multiple molecular species at multiple scales, our strategy provides a powerful approach for precisely deconstructing intratumoral heterogeneity. Abstract Tumors may contain billions of cells, including distinct malignant clones and nonmalignant cell types. Clarifying the evolutionary histories, prevalence, and defining molecular features of these cells is essential for improving clinical outcomes, since intratumoral heterogeneity provides fuel for acquired resistance to targeted therapies. Here we present a statistically motivated strategy for deconstructing intratumoral heterogeneity through multiomic and multiscale analysis of serial tumor sections (MOMA). By combining deep sampling of IDH-mutant astrocytomas with integrative analysis of single-nucleotide variants, copy-number variants, and gene expression, we reconstruct and validate the phylogenies, spatial distributions, and transcriptional profiles of distinct malignant clones. By genotyping nuclei analyzed by single-nucleus RNA-seq for truncal mutations, we further show that commonly used algorithms for identifying cancer cells from single-cell transcriptomes may be inaccurate. We also demonstrate that correlating gene expression with tumor purity in bulk samples can reveal optimal markers of malignant cells and use this approach to identify a core set of genes that are consistently expressed by astrocytoma truncal clones, including AKR1C3, whose expression is associated with poor outcomes in several types of cancer. In summary, MOMA provides a robust and flexible strategy for precisely deconstructing intratumoral heterogeneity and clarifying the core molecular properties of distinct cellular populations in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19f10a20bb87365187cbad7becc484185a46f9af" target='_blank'>Deconstructing Intratumoral Heterogeneity through Multiomic and Multiscale Analysis of Serial Sections</a></td>
          <td>
            Patrick G. Schupp, Samuel J. Shelton, Daniel J. Brody, Rebecca Eliscu, Brett E Johnson, T. Mazor, Kevin W. Kelley, Matthew B. Potts, Michael W McDermott, Eric J. Huang, Daniel A. Lim, Russell O Pieper, Mitchell S. Berger, Joseph F. Costello, Joanna J Phillips, Michael C. Oldham
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>