<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-22 09:37:04 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>43</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>89</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>124</td>
          <td>147</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>391</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>340</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells.
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic chromosome instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1, 2–3. Although CAs have been linked to mitotic errors resulting in the emergence of nuclear atypia4, 5, 6–7, the underlying processes and rates of spontaneous CA formation in human cells are underexplored. Here we introduce machine-learning-assisted genomics and imaging convergence (MAGIC)—an autonomously operated platform that integrates live-cell imaging of micronucleated cells, machine learning on-the-fly and single-cell genomics to systematically investigate CA formation. Applying MAGIC to near-diploid, non-transformed cell lines, we track de novo CAs over successive cell cycles, highlighting the common role of dicentric chromosomes as initiating events. We determine the baseline CA mutation rate, which approximately doubles in TP53-deficient cells, and observe that chromosome losses arise more frequently than gains. The targeted induction of DNA double-strand breaks along chromosome arms triggers distinct CA processes, revealing stable isochromosomes, coordinated segregation and amplification of isoacentric segments in multiples of two, as well as complex CA outcomes, influenced by the chromosomal break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The experimentation enabled by MAGIC advances the dissection of DNA rearrangement processes, shedding light on fundamental determinants of chromosomal instability. Coupling live-cell imaging, machine learning and genomic sequencing, the MAGIC platform enables investigation of the cellular context, mutation rates and triggers of spontaneous chromosomal abnormality formation, shedding light on fundamental determinants of chromosomal instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="


 Understanding of the heterogeneity in the formation of the Ph has evolved as technology has advanced. In the pre-TKI era, FISH revealed derivative 9 deletions that were associated with shorter survival. The poor outcomes were overcome with imatinib. Whole exome and transcriptome sequencing revealed greater complexity: inversions; novel 9;22 fusions; sequence fragmentation; and random reassembly. These were termed Ph-associated rearrangements. They occurred on the Ph and/or the derivative 9, were thought to occur at the same time as the Ph and correlated with poorer response to imatinib.



 To use whole genome sequencing (WGS) to examine derivative chromosomes and further resolve the complexity of Ph-associated rearrangements in patients at diagnosis and pan-TKI resistant cell lines.



 WGS was performed for 7 chronic phase CML patients at diagnosis and 7 matched sensitive/resistant Ph-positive cell lines. Resistance was induced by exposure to increasing concentrations of imatinib. Data was analyzed using Oncoanalyzer. Derivative chromosome structure was resolved using the LINX algorithm, which clusters structural variants that are highly unlikely to occur independently. Rearrangements were validated using orthogonal methods, including Optical Genome Mapping and SNP Array. Six of 7 patients had pre-characterized Ph-associated fusions identified using RNAseq.



 For most patients, a balanced reciprocal Ph translocation involves 2 rearrangements: 1 that generates the Ph and BCR::ABL1; and 1 reciprocal derivative 9 with ABL1::BCR. These structures were resolved for 1 patient without pre-characterized Ph-associated rearrangements and for 5 of 7 parental cell lines.
 A distinct class of complex structural rearrangement called chromoplexy was identified involving the Ph translocation in 11 samples. These Ph-associated rearrangements had the highest number of linked structural variants within the individual genomes of 10 of the 11 samples. Chromoplexy involves the formation of extensive chained rearrangements affecting multiple chromosomes that create an intricate series of inter/intra-chromosomal fusions, deletions, duplications and inversions. Many DNA strands were broken and ligated in new configurations in the affected samples. Multiple genes on the derivative chromosomes were disrupted, including recurrent cancer drivers.
 For 3 of 7 matched sensitive/resistant cell lines, Ph-associated rearrangements were acquired in the resistant lines. The most complex events occurred in the LAMA84 resistant line. The parallel sensitive line had the standard 2 rearrangements. However, the resistant line gained 17 rearrangements with highly amplified Ph sequences, inversions and novel genomic fusions. Remarkably, small fragments from 6 chromosomes (2, 4, 10, 13, 17, X) were inserted upstream and downstream of BCR::ABL1, and disrupted the NUP214 driver gene.
 The K562 sensitive line displayed 24 rearrangements with non-contiguous amplified Ph regions of different copy number. Events included NOTCH1 deletion, 9;22 NUP214 fusion and an adjacent 9;13 fusion. Six fragments from chromosome 13 were incorporated into the Ph. The K562 resistant line displayed further evolution with 31 Ph rearrangements.
 The AR230 sensitive line displayed Ph chromoplexy with 9 rearrangements. The resistant line evolved with 19 additional Ph rearrangements. A fragment from chromosome 21 was inserted upstream of BCR::ABL1 and disrupted the MAPK1 driver gene. The LZTR1 tumor suppressor was deleted. All other cell lines maintained the standard Ph configuration.
 Ph-associated chromoplexy was evident in 6 patient samples: average 10 rearrangements. The patient with the highest number (15) progressed to blast phase at 4 months of imatinib. Four Ph-associated subclones were detected at diagnosis with non-contiguous amplified regions, consistent with the evolved cell lines. Furthermore, the LZTR1 tumor suppressor gene was deleted, as detected in AR230. The blast phase sample was unavailable. Deep molecular response was achieved in 1 patient on imatinib and 4 on frontline or switch to nilotinib/dasatinib.



 We have demonstrated that Ph chromoplectic rearrangement can be present at diagnosis but can also be initiated during leukemic progression and disrupt multiple genes, including cancer drivers. The Ph may be prone to extensive DNA breakage. Ph chromoplexy could constitute a new mechanism that contributes to acquired TKI resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed250c9c3a664955ee35598f9870a47e6c84f2b" target='_blank'>
              Whole genome sequencing reveals that the Philadelphia chromosome (Ph) can evolve and gain complex chromoplectic sequence rearrangements associated with tyrosine kinase inhibitor (TKI) resistance
              </a>
            </td>
          <td>
            S. Branford, C. Wadham, N. Shahrin, V. Saunders, Rosalie R. Kenyon, Ming Lin, Dominic Kaczorowski, Muneeza Maqsood, Hamish Scott, John Toubia, N. Shanmuganathan, C. Kok, Timothy Hughes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) fragments in the plasma capture cellular nucleosomal profiles since nucleosome-protected regions escape enzymatic degradation while nucleosome-depleted regions can not. We developed cfOncoXpress, a machine learning framework that uses fragmentation patterns to predict oncogene expression from cfDNA WGS. cfOncoXpress incorporates gene copy number aberrations inferred from cfDNA, including those associated with extrachromosomal DNA. Its application in prostate and breast cancers shows it can predict tumor subtype based on expression of signature genes and activated pathways. cfOncoXpress shows superior performance relative to other state-of-the-art methods and can be used to predict tumor gene expression when tissue biopsies are infeasible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd03a827003a113eb12bee1a6a13f9987699c5c0" target='_blank'>
              cfOncoXpress: Tumor gene expression prediction from cell-free DNA whole-genome sequences
              </a>
            </td>
          <td>
            Weiling Li, M. Roskes, A. Martínez-Fundichely, Ekta Khurana
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Background: Chromothripsis, a catastrophic event of chromosomal shattering and reassembly, is a key driver of oncogenesis and is associated with poor prognosis in multiple myeloma (MM). However, the specific functional consequences and the molecular mechanisms by which it rewires the MM regulatory landscape remained largely unknown.
 Methods: Using previous definitions, we identified chromothripsis (≥10 breakpoints, ≥7 CN oscillations, ≥200 kb segment sizes), chromothripsis-like (5-9 breakpoints and 3-6 CN oscillation) and non-chromothripsis (WT) samples. We utilized multiomic data from the CoMMpass dataset (n=1,066) and an independent Indiana University (IU) dataset (n=134), as well as 14 patient-derived xenograft (PDX) models with short-/long-read whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities and structural variation (SV), as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) to identify 3D chromatin architecture/folding inside the cell and enhancer-target interactions.
 Results: We identified 417 chromothripsis events in 268 patients (25%) from the CoMMpass dataset. Events were identified more frequently in relapsed and refractory (RRMM) samples compared to newly diagnosed (NDMM) samples (30% vs. 22%), suggesting an association between chromothripsis and disease progression. The number of events found in chromosomes highly correlated with chromosome sizes (r=0.63, Pearson correlation). However, events were particularly enriched on chromosomes 1, and 17 (0.26 and 0.30 events per million bps). Breakpoints of these events were enriched on 17q and 1p (odds ratio=2.2 and 1.7, p=0.0003 and 0.001). Chromothriptic samples were associated with inferior progression-free (PFS; 970 vs. 1227 days, p=0.003) and overall survival (2176 vs. not reached, p=0.006) compared to WT. In chromothriptic samples, 34% exhibited breakpoints on >1 chromosome and an higher number of affected chromosomes correlated with worse PFS (≥3 vs. 2, p=0.03; ≥3 vs. 1, p=0.02). Meanwhile, CN oscillations plus breakpoints was an independent factor associated with inferior PFS(p=0.05,: 700 vs. 1358 days). Del(TENT5C), del(TP53)and t(4;14) were enriched in the chromothripsis group (32%, 18% and 20%, odds ratio=2.6, 3.4 and 2.4, p<0.001) and are associated with genomic instability. We applied the same approach and further identified 19 chromothriptic samples from the IU dataset, where t(4;14) and del(TP53) were also enriched (odds ratio=2.6 and inf.) We identified differentially expressed genes (DEGs) between the chromothripsis group and a subset of WT group with balanced frequencies of high-risk events (e.g. t(4;14) & del(TP53)) to generate a chromothripsis signature. Pathway analysis on 37 consistent DEGs between the two datasets identified ‘G2/M checkpoint’ and ‘interferon α/γ responses‘(p<0.05), suggesting selection for a growth advantage and activation of the cGAS/STING pathway. A neuron network model for predicting chromothripsis was trained with DEGs and reached performance of AUC=0.90 and AUPR=0.87.
 We utilized multi-omics data from established PDX models to further examine the functional consequences of chromothripsis events. Among 9 events detected in 7 samples, 5 events exhibited significantly enriched APOBEC mutational signature and kataegis within events. Copy number change alone did not account for changes in gene expression within the chromothripsis regions, with 71% of genes dysregulated through neo-TAD formation and rewiring of the epigenome, specifically through juxtaposition of super-enhancers and allele-specific expression associated with DNA methylation changes. Key dysregulated genes included MYCN (Log2FC=6.3), the tumor suppressor PMAIP1 (Log2FC=-4.9), and cell adhesion proteins (EPCAM/CD326, Log2FC=3.5) that were rarely expressed in other MM samples.
 Conclusion: We revealed novel molecular characteristics of chromothripsis and the mechanisms caused it. We determined that the number of events and CN abnormalities are associated with poor outcome and that epigenetic states are reprogrammed within the events. These findings advance the clinical risk stratification and reveal unique therapeutic vulnerabilities tied to chromothripsis-associated epigenetic dysregulation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eeb625eee780834dfe72e26b192d78e2662c5ec" target='_blank'>
              Chromothripsis drives high-risk multiple myeloma by copy number abnormalities and rewiring the epigenome to activate oncogenic programs.
              </a>
            </td>
          <td>
            Enze Liu, Nathan Becker, Sreejeta Mondal, A. Suvannasankha, D. Coffey, B. Diamond, Abhishek Pandey, Kelvin Lee, D. Kazandjian, R. Abonour, Aneta Mikulasova, G. Morgan, O. Landgren, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Motivation Tumor viruses account for ∼10% of cancer diagnoses. Virally induced tumorigenesis is understood as direct signaling through oncogenes such as E6 and E7 genes in the case of human papillomavirus. Furthermore, pathogen characteristics such as viral oncogene dose may impact the disease course. To our knowledge, no tool has been proposed to assess the intra-viral copy number alterations that define the gene dose of viral oncogenes and associated suppressive pathways native to the pathogen’s normal life cycle. Results We propose an R package, “ELViS,” that analyzes viral copy number changes from DNA sequencing of whole viral genomes. The method adjusts for viral load with 2D transformation and segmentation to offer the relative viral gene doses. Availability and implementation The ELViS R package is available from https://bioconductor.org/packages/ELViS. This article used controlled access data from dbGaP (phs001713.v1.p1).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee403f958df88d61608425c7ffde897d6bfc505e" target='_blank'>
              ELViS: an R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, J. Holt, Xiaobei Zhao, K. Hoadley, D. N. Hayes, Hyo Young Choi, Neil Hayes
          </td>
          <td>2025-11-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) is a notable exception to genome integrity, characterized by significant DNA loss during development. In many nematodes, PDE is initiated by DNA double-strand breaks (DSBs), which lead to chromosome fragmentation and subsequent DNA loss. However, the mechanism of nematode DNA breakage remains largely unclear. Interestingly, in the human and pig parasitic nematode Ascaris, no conserved motif or sequence structures are present at chromosomal breakage regions (CBRs), suggesting the recognition of CBRs may be sequence-independent. Using Hi-C, we revealed that Ascaris CBRs engage in three-dimensional (3D) interactions before PDE, indicating that physical contacts of the break regions may contribute to the PDE process. The 3D interactions are established in both Ascaris male and female germlines, demonstrating inherent genome organization associated with the CBRs and to-be-eliminated sequences. In contrast, in the unichromosomal horse parasite Parascaris univalens, transient and pairwise interactions between two neighboring CBRs that would form the ends of future somatic chromosomes were observed only during PDE. Intriguingly, we found specific spatial compartmentalization changes in the Ascaris chromosomes after PDE converts 24 germline chromosomes into 36 somatic ones. Remarkably, Parascaris PDE generates the same set of 36 somatic chromosomes, and the compartment changes following PDE are consistent between the two species. Overall, our findings contribute to a model that the 3D genome facilitates the recognition of sequence-independent break sites for Ascaris PDE. It also suggests an evolutionary and developmentally conserved 3D genome reorganization of nematode somatic chromosomes following PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Emerging evidence suggests that lineage-specifying transcription factors control the progression of pancreatic ductal adenocarcinoma (PDAC). We have discovered a transcription factor switching mechanism involving the poorly characterized orphan nuclear receptor HNF4G and the putative pioneer factor FOXA1, which drives PDAC progression. Using our unbiased protein interactome discovery approach, we identified HNF4A and HNF4G as reproducible, FOXA1-associated proteins, in both preclinical models and Whipple surgical samples. In the primary tumor context, we consistently find that the dominant transcription factor is HNF4G, where it functions as the driver. A molecular switch occurs in advanced disease, whereby HNF4G expression or activity decreases, unmasking FOXA1’s transcriptional potential. Derepressed FOXA1 drives late-stage disease by orchestrating metastasis-specific enhancer–promoter loops to regulate the expression of metastatic genes. Overall survival is influenced by HNF4G and FOXA1 activity in primary tumor growth and in metastasis, respectively. We suggest that the existence of stage-dependent transcription factor activity, triggered by molecular compartmentalization, mediates the progression of PDAC. Pancreatic cancer progression is driven by a switch from HNF4G-driven transcriptional activity in primary disease to FOXA1-mediated transcription in the metastatic setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d699536709f64e5dfad37d1331732fdf39ca24c" target='_blank'>
              Transcription factor switching drives subtype-specific pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Sean V Flynn, Niklas Krebs, Dominique-Laurent Couturier, Stephanie Mack, Rebecca Brais, Jill Temple, Amy Smith, Evangelia Papachristou, Catarina Pelicano, C. Chilamakuri, Krzysztof Herka, H. Baba, Luay Farah, Phyllis F Cheung, J. Siveke, Stéphane Guerrier, Luca Insolia, Michael Gill, Emily Archer Goode, Steven Kupczak, Yi Cheng, Giacomo Borsari, Duncan Jodrell, Clive S. D’Santos, Alasdair Russell, Barbara Grünwald, Eva Serrao, Igor Chernukhin, Jason S. Carroll
          </td>
          <td>2025-10-30</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Immune system control is a principal hurdle in cancer evolution. The temporal dynamics of immune evasion remain incompletely characterized, and how immune-mediated selection interrelates with epigenome alteration is unclear. Here we infer the genome- and epigenome-driven evolutionary dynamics of tumor-immune coevolution within primary colorectal cancers (CRCs). We utilize a multiregion multiomic dataset of matched genome, transcriptome and chromatin accessibility profiling from 495 single glands (from 29 CRCs) supplemented with high-resolution spatially resolved neoantigen sequencing data and multiplexed imaging of the tumor microenvironment from 82 microbiopsies within 11 CRCs. Somatic chromatin accessibility alterations contribute to accessibility loss of antigen-presenting genes and silencing of neoantigens. Immune escape and exclusion occur at the outset of CRC formation, and later intratumoral differences in immuno-editing are negligible or exclusive to sites of invasion. Collectively, immune evasion in CRC follows a ‘Big Bang’ evolutionary pattern, whereby it is acquired close to transformation and defines subsequent cancer-immune evolution. This study nominates immune escape as an early event in colorectal cancer and shows how this can be driven through both genetic and epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cleavage and polyadenylation (CPA) is a co-transcriptional mRNA processing mechanism that is central to mRNA and protein function. Dysregulation of CPA is widespread in cancer, promotes oncogenic programs, and affects patient outcomes. The CPA machinery is composed of multiple factors, and while prior research has investigated the impact of CPA gene expression on cancer phenotypes, the contribution of genomic alterations, such as mutations and copy number variations, remains largely unexplored. In this study, we conducted a pan-cancer analysis of genomic alterations in CPA genes. While numerous CPA genes harbor somatic mutations, these mutations do not significantly impact mRNA expression or provide prognostic value. In contrast, we found that copy number alterations in CPA genes have substantial clinical relevance. Notably, we identified the cleavage and polyadenylation specificity factor 1 (CPSF1) gene as the most frequently amplified CPA gene in cancer. While amplification of CPSF1 and MYC are co-occuring, CPSF1 amplification independently correlates with poor prognosis. We also found that CPSF1 amplification can impact 3′UTR length regardless of MYC status. Our study highlights the importance of CPSF1 as a promising prognostic factor in cancer and as a therapeutic intervention target to study in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Centromere breakage has been associated with anti-centromere antibodies in systemic sclerosis (SSc), yet the origins of centromere damage and its link to immune activation remain unclear. The bleomycin induced fibrosis model is widely used as an experimental model of SSc. Here, we investigated whether bleomycin selectively disrupts centromeres and whether such damage contributes to chromosome instability and immunogenic chromatin mislocalization. Methods We evaluated centromere integrity and downstream genome damage-response phenotypes across complementary systems: the bleomycin mouse model of skin fibrosis, hTERT-immortalized human fibroblasts, and primary dermal fibroblasts from SSc patients. Centromeric α-satellite copy number, DNA damage and repair markers, micronuclei, nuclear envelope rupture, and colocalization of centromeric chromatin with antigen presentation machinery were assessed by qPCR, RT-qPCR, and immunofluorescence microscopy. Results Bleomycin induced selective depletion of α-satellite repeats at centromeres in mouse skin. In human fibroblasts, bleomycin generated double-strand breaks by γH2AX foci that preferentially colocalized with CENP-A-marked active centromeres and produced persistent centromere loss. Damage signaling required ATM and relied mainly on RAD51-mediated homologous-recombination repair, yet centromere restoration remained incomplete. Bleomycin also increased micronuclei, cytoplasmic centromeric foci, and BANF1-marked nuclear-envelope rupture, consistent with mis-segregation-driven chromatin leakage. CENP-B colocalized with HLA-DRB1 in bleomycin-treated fibroblasts and limited cutaneous SSc (lccSSc) fibroblasts, revealing centromere-derived antigen-presentation signatures. Conclusions This study identifies active centromeres as selectively vulnerable and incompletely repaired targets of bleomycin-induced DNA damage and demonstrates that centromere instability produces mislocalized chromatin capable of engaging antigen-presentation pathways. These findings provide mechanistic insight into SSc autoantibody specificity and link genome instability to immune activation in fibrotic autoimmunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/060c6a6ceec69d2ffe26ac48872dd1fd6fc913c0" target='_blank'>
              Centromere Instability Drives Chromosome Damage and Autoantigen Exposure in Systemic Sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Christine M Wright, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Most animal genomes are partitioned into topologically associating domains (TADs), created by cohesin-mediated loop extrusion and defined by convergently oriented CCCTC-binding factor (CTCF) sites. The dynamics of loop extrusion and its regulation remain poorly characterized in vivo. Here we tracked the motion of TAD anchors in living human cells to visualize and quantify cohesin-dependent loop extrusion across multiple endogenous genomic regions. We show that TADs are dynamic structures whose anchors are brought in proximity about once per hour and for 6–19 min (~16% of the time). Moreover, TADs are continuously extruded by multiple cohesin complexes. Remarkably, despite strong differences in Hi-C patterns across chromatin regions, their dynamics is consistent with the same density, residence time and speed of cohesin. Our results suggest that TAD dynamics is primarily governed by the location and affinity of CTCF sites, enabling genome-wide predictive models of cohesin-dependent chromatin interactions. Live-cell fluorescence microscopy and polymer simulations in human HCT116 cells show similar cohesin density, residence times and extrusion speed across multiple genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715a0a11e407c2847c391eae8db4fee949a2d73b" target='_blank'>
              Uniform dynamics of cohesin-mediated loop extrusion in living human cells
              </a>
            </td>
          <td>
            Thomas Sabaté, Benoît Lelandais, Marie-Cécile Robert, Michael Szalay, J. Tinevez, Edouard Bertrand, Christophe Zimmer
          </td>
          <td>2025-11-14</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline.
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-03</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Prime editing, a flexible "search-and-replace" genome editing technique, has become a potent instrument for accurately altering the genome without the need for donor templates or double-strand DNA breaks. The vast non-coding regulatory genome, which includes enhancers, promoters, untranslated regions (UTRs), and non-coding RNAs, is now understood to be a major driver of tumorigenesis, despite the fact that the majority of cancer research has concentrated on protein-coding mutations. Oncogene dysregulation, tumor suppressor silencing, and treatment resistance can all result from mutations in these areas. The convergence of these two fields is examined in this review, which also describes the use of prime editing to fix particular, harmful non-coding mutations. In order to restore normal gene expression patterns, we describe methods for focusing on regulatory elements, present a landscape of non-coding drivers in cancer, and talk about the fundamentals of prime editing. Lastly, we look at delivery methods, therapeutic factors, and how prime editing might be incorporated into the precision oncology paradigm going forward. We emphasize how prime editing could help usher in a new era of targeted therapy and functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9d66804e5625efa917cbd699bfc770f8da5aa5" target='_blank'>
              Prime Editing the Non-Coding Genome: A Molecular Review of Correcting Regulatory Mutations for Precision Cancer Therapy
              </a>
            </td>
          <td>
            Alina Mansoor, Midhat Narmeen Mustafa, Abdul Ahad Mehboob, Nureen Shafaqat, Sanam Dehraj, Amna Noor, Sehresh Murtaza, Mahnoor Javed
          </td>
          <td>2025-10-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. Here, we demonstrate that Rad27/FEN1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining the stability of the ribosomal DNA (rDNA) repeats. Severe rDNA instability is observed in the rad27∆ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates Okazaki fragments in the rDNA region, without inducing the formation of detectable DSBs. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, Exonuclease 1 and PCNA partially compensate for the loss of Rad27 in rDNA stabilization. These findings highlight the importance of proper Okazaki fragment maturation in the maintenance of rDNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f06a68a9173b59a7c4386ddec1b029c06486a7" target='_blank'>
              Rad27/FEN1 prevents accumulation of Okazaki fragments and ribosomal DNA copy number changes.
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-10-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tandemly repeated non-coding sequences, widely known as satellite DNAs (satDNAs), are extremely diverse and highly variable components of eukaryotic genomes. In recent years, advances in high-throughput sequencing and new bioinformatics platforms have enabled in-depth studies of all (or nearly all) tandem repeats in any genome (the satellitome), while a growing number of telomere-to-telomere assemblies facilitates their detailed mapping. Research performed on a large number of non-model plant and animal species changed significantly the “classical” view on these sequences, both in an organizational and functional sense, from ballast compacted in the form of heterochromatin to elements that are important for structuring the entire genome, as well as for its functions and evolution. The diversity of repeat families, and the complexity of their intraspecies and interspecies distribution patterns, posed new questions, urging for species-by-species comparative analyses. Here we integrate some basic features of different forms of sequences repeated in tandem and rapidly growing data evidencing extensive dispersal of satDNA sequences in euchromatin, their putative roles and evolutionary significance. Importantly, we also present and discuss various issues brought on by the use of new methodological approaches and point out potential threats to the analysis of satDNAs and satellitomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df02ce17404430620feb58c2c132e48c76b2721" target='_blank'>
              Satellite DNA Genomics: The Ongoing Story
              </a>
            </td>
          <td>
            M. Garrido-Ramos, Miroslav Plohl, Eva Šatović-Vukšić
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4228e4a68ff14de2be692e9aac8b9027ae451" target='_blank'>
              Sister chromatid separation determines the proliferative properties upon whole-genome duplication via homologous chromosome arrangement
              </a>
            </td>
          <td>
            Masaya Inoko, Guang Yang, Yuki Tsukada, Ryota Uehara
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Mutations in non-coding regulatory elements are increasingly recognized as critical drivers of cancer development and progression. Among these, structural alterations such as duplications of super-enhancers (SEs), large clusters of enhancers that regulate key oncogenes, have emerged as significant contributors to oncogenesis. In T-cell acute lymphoblastic leukemia (T-ALL), a particularly aggressive hematologic malignancy, SE duplications have been identified in approximately 5% of human cases. These duplications frequently involve a non-coding region located approximately 1.3 megabases downstream of the Myc oncogene, a region known as N-Me. Despite the growing recognition of the role of SE duplications in T-ALL, functional studies to determine their molecular origins and contributions to leukemogenesis have been limited, largely due to the absence of suitable and genetically tractable preclinical models. This gap in knowledge hinders both our understanding of disease pathogenesis and our ability to design targeted therapeutic interventions.
 Here, we report the generation and characterization of a novel genetically engineered mouse model of T-ALL that spontaneously develops oncogenic SE duplications at the Myc locus. This model offers the first spontaneous in vivo system to investigate the origin, regulation, and oncogenic function of SE duplications. This model was created by conditionally stabilizing β-catenin in CD4⁺CD8⁺ double-positive (DP) thymocytes in combination with targeted deletion of the transcription factor HEB. This combination is sufficient to drive the development of aggressive T-cell leukemias that consistently harbor duplications of the N-Me super-enhancer. This finding recapitulates the genomic alteration observed in a subset of human T-ALL and provides a unique platform for mechanistic studies.
 We further demonstrate that the molecular events leading to SE duplication and oncogenic Myc activation unfold in a stepwise manner. At the preleukemic stage, stabilization of β-catenin alone is sufficient to enhance recruitment of HEB to the N-Me region. The simultaneous loss of HEB greatly increased chromatin accessibility at the N-Me enhancer. This altered chromatin landscape permits aberrant binding of transcription factors including TCF-1, which plays a critical role in T-cell development and leads to N-Me SE activation associated with significant three-dimensional chromatin reorganization. Chromosome conformation analyses in the leukemic cells reveal the formation of a novel chromatin loop that brings the activated N-Me SE into close spatial proximity with the Myc promoter, enabling oncogenic Myc transcription.
 These findings uncover a novel mechanistic pathway by which β-catenin and HEB interact to reshape the enhancer landscape and three-dimensional chromatin architecture, ultimately promoting SE duplication and aberrant Myc expression in T-ALL. β-catenin appears to play a dual role, first facilitating HEB recruitment under physiological conditions and then, upon HEB loss, permitting the reprogramming of enhancer activity through chromatin opening and ectopic transcription factor binding. Our data suggest that the cooperative interplay between signaling pathways and transcriptional regulators at enhancer elements can initiate structural genomic changes that drive malignancy.
 This work provides a long-sought preclinical model that faithfully mirrors key features of human T-ALL. Our findings open new avenues for understanding how non-coding regulatory mutations arise and exert oncogenic effects and have the potential to provide insights that may extend beyond T-ALL to other cancers driven by enhancer dysregulation. Moreover, by revealing molecular intermediates such as β-catenin, HEB, and TCF-1 in the regulation of enhancer activity, our findings identify potential therapeutic targets to disrupt the oncogenic enhancer-promoter interactions that underlie Myc-driven leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3354f2ef9255003e9c97bc93c32af47ddbd359" target='_blank'>
              Myc super-enhancer dynamics in T-ALL
              </a>
            </td>
          <td>
            Georgios Tousinas, Vasili Toliopoulos, Melissa Tracy, Akinola Emmanuel, K. Khazaie, Thomais Papamarcaki, P. L. Bergsagel, F. Gounari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Telomeres are known to be important for Leishmania biology but the mechanistic of how the process of telomere maintenance contributes to genome stability remains an unanswered question. Their maintenance is most commonly facilitated by the telomerase ribonucleoprotein complex that elongates telomeres countering their natural shortening due to the incomplete DNA replication in each cell cycle. In some organisms, telomere maintenance is achieved through telomerase-independent mechanisms, such as the Alternative Lengthening of Telomeres (ALT) pathways described in yeasts with dysfunctional telomerase and some rare telomerase-negative human cancer cells. Molecular markers for the ALT pathway include presence of the heterogeneous (in their length and sequence) telomeres, high level of telomeric exchange between the sister chromatids, increased expression of Rad51 and associated proteins, and occurrence of extrachromosomal telomeric repeats that can be present in either linear or circular form. Here, we used third-generation sequencing techniques in combination with other approaches and analyzed telomeres of L. mexicana at unprecedented high-level resolution. We demonstrate that Ku80 ablation-driven telomere elongation varies between chromosomes, possibly due to the chromosome-specific recombination rates, which are sequence/content dependent and associated with the structure of the telomeric tandemly repeated sequence, TTAGGG. Moreover, this telomere length heterogeneity is accompanied by an increased level of C-circles, a subclass of circular telomeric DNA highly specific for ALT activity. Our findings underscore that L. mexicana promastigotes have an inherent ability to utilize ALT, and the loss of Ku80 and/or TERT further enhanced this trait. These proteins work together to maintain telomere integrity, inhibit recombination, and stabilize telomere lengths. Our data suggest that ALT may be a fundamental and readily activated feature of Leishmania biology, and that telomere regulation in this organism significantly differs from what has been observed in other eukaryotic model species, including iconic T. brucei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e070c6fd10f58ebc83ce00a4b0839a86950d886" target='_blank'>
              Leishmania mexicana telomeres at high resolution: Ku80, TERT, and alternative lengthening mechanisms
              </a>
            </td>
          <td>
            Edubiel A. Alpizar-Sosa, Andreu Saura, Petr Fajkus, Ester Poláková, Kateřina Havlová, Amanda T. S. Albanaz, Jíří Fajkus, V. Yurchenko
          </td>
          <td>2025-10-24</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0dc83d3c9d750e9f6591dee2fa83d60f9676308" target='_blank'>
              Dici: a novel DNA transposon reshaping the genome of the opportunistic yeast Diutina catenulata
              </a>
            </td>
          <td>
            F. Bigey, M. Wessner, Martine Pradal, Corinne Cruaud, J. Aury, Cécile Neuvéglise
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Human tissues require a mechanism to generate durable, yet modifiable, transcriptional memories to sustain cell function across a lifetime. Previously, it was demonstrated that nanoscale packing domains couple heterochromatin (cores) and euchromatin (outer zone) into unified reaction volumes that can generate transcriptional memory. In prior work, this framework demonstrates that RNA synthesis occurred within the ideal zone (intermediate density) portions of the domain. Naturally, this creates a question of where genes are positioned in relation to the packing domain architecture and which genetic material fills the domain core to sustain transcription. Here, it is proposed that this can be solved by the encoded positioning of introns, intergenic segments, and exons as a projection of the functional packing layers of domains. This suggests that introns and intergenic segments are coupled to adjacent exons to generate coherent packing domain volumes. How this organization will reconcile contradictions in epigenetic patterns, non-randomness in oncogenic mutations, and produce durable transcriptional memory is illustrated. The study concludes by showing that this genome geometry may have coincided with the rapid evolution of body-plan complexity, suggesting that chromatin geometry could be fundamental to metazoan evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e1c75f6af2c61e333ce8d8a40a65dbb9822c6f" target='_blank'>
              Lysine-specific histone demethylase complex restricts Epstein–Barr virus lytic reactivation
              </a>
            </td>
          <td>
            Yifei Liao, Jinjie Yan, Isabella Y. Kong, Zhixuan Li, Weiyue Ding, Sarah Clark, Davide Maestri, Tetsuya Yoshida, L. Giulino-Roth, B. Gewurz
          </td>
          <td>2025-10-31</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d97982af89e80beb34eb068ca73e5435c3e26dc" target='_blank'>
              High-throughput functional characterization of enhancers in totipotent-like cells
              </a>
            </td>
          <td>
            Lingyue Yang, Tianran Peng, Yating Zhu, Xuzhao Zhai, Boyan Huang, Ling Li, Tao Zhang, Jiekai Chen, Dan Liang, Jiangping He, Man Zhang
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) and its aggressive subtype plasma cell leukemia (PCL) are transcriptionally-driven hematological malignancies, heavily dependent on enhancer activity. Enhancers are distal regulatory elements at which transcription factors (TFs) bind, recruiting coactivators that promote chromatin remodeling to stimulate target gene expression via physical proximity of enhancer and promoter. MM cells show high expression of essential TFs, including IRF4, MYC, PRDM1 and IKZF1/3, which function by binding to and activating oncogenic enhancers.
 A detailed understanding of chromatin regulation in MM patients requires the comparison of histone modifications, chromatin accessibility and TF/cofactor binding distribution within the same cells, which has so far been unachievable. Patient analysis has mainly relied on ATAC-seq and RNA-seq, with limited inference of TF binding and chromatin state, owing to the difficulty in obtaining the large numbers of MM cells required for more detailed characterization. Many observations of enhancer function in MM therefore come from cell lines, which only offer an approximation of MM physiology.
 To overcome the limitations of in vitro models and primary cell availability, we employed TOPmentation (TF-OPtimized ChIPmentation) to generate ChIP-seq-quality data for CD138+ cells from MM and PCL patients. This technique uses as few as 100,000 cells for histone modifications, and 250,000 cells for TFs, allowing us to profile up to 15 features per patient. In addition, we used the base-pair resolution chromosome conformation capture technique micro-capture-C (MCC) to visualize enhancer-promoter interactions in a PCL patient sample. This allowed us to dissect large oncogenic enhancers, identifying the key TF binding sites within each enhancer that directly contact the gene promoter. Using this dual approach, for the first time we have generated integrated maps of TF occupancy, active and repressive histone modifications, and chromatin interactions in MM patient samples.
 We characterized the epigenetic landscapes of two t(11;14) and two t(4;14) patients, finding broadly similar chromatin profiles in each subtype. Key MM enhancers were retained at oncogenes including IRF4, MYC,PRDM1 and IKZF1/3, indicating a convergence of gene regulation in MM originating from distinct initiating genetic events. At many enhancers, we observed multiple TFs co-binding at the same sites, indicating cooperative activation of target genes. By combining TOPmentation with MCC, we found that many of these sites interact with target gene promoters, directly implicating them in gene regulation.
 In MM, patients typically display mutually exclusive upregulation of CCND1 or CCND2. As previously established, we found strong activation at the CCND1 locus by IGH translocation in t(11;14) patient samples, whereas in t(4;14) patients CCND2 was upregulated via an upstream super-enhancer. Surprisingly, in each subtype the silent CCND gene was not actively repressed, but rather existed in a poised, bivalent state marked by active H3K4me3 and repressive H3K27me3 modifications. This argues that intricate regulation of CCND1/2 is required to maintain optimal levels of expression for tumor growth.
 Comparison of patient and cell line data showed broad conservation of epigenetic features. However, we also observed examples of distinct gene regulation, for example at the PRDM1 locus. In several MM cell lines (including KMS12, H929 and MM1S), transcription of PRDM1 initiates from a distal promoter. However, this locus was inactive in the patient samples analyzed, and instead an alternate, proximal promoter was favored.
 This study for the first time reports the genomic binding distribution of TFs and histone modifications in MM patient cells, demonstrating the importance of comprehensive epigenomic analysis to capture the complexity in MM gene regulation. This emphasizes the need for validation of cell line models to provide a more physiologically relevant understanding of enhancer-driven oncogene regulation. As we expand this patient-derived dataset, it will provide a valuable resource to better understand MM biology and inform future strategies for precision epigenetic therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab64ef7d2d4fd86af25053430e4f99417b63dfb" target='_blank'>
              Revealing oncogenic enhancer regulation by epigenomic profiling of multiple myeloma and plasma cell leukemia patient samples
              </a>
            </td>
          <td>
            Katrina Fordwor, Jason Taslim, Sophie Ball, P. Sabbattini, A. Katsarou, T. Milne, A. Karadimitris, A. Chaidos, Nick Crump
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan C Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6212cc21f6c9c548c0deb4b30bef6b44b0cc40c" target='_blank'>
              Analyses of Genome-Wide Recombination in Sulfolobus islandicus
              </a>
            </td>
          <td>
            David J. Krause, Changyi Zhang, Rachel J. Whitaker
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8df6dc2d72a4159b55a77cb1f1691bd8b0d91d08" target='_blank'>
              Click-code-seq reveals strand biases of DNA oxidation and depurination in human genome.
              </a>
            </td>
          <td>
            Vakil Takhaveev, N. Püllen, Navnit K. Singh, Lucie Lefevre, Emilie A Aghajani, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2025-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterizing tissue-specific (TSp) gene expression is crucial for understanding development and disease; however, traditional expression-based methods often overlook the latent “regulatory grammar” embedded in the non-coding DNA, particularly in distal promoter regions. Here, we introduce TSProm, a framework that specializes a DNA foundation model (DNABERT2) to decipher the long-range regulatory logic of TSp promoters at the gene isoform level. The contributions of our work are two-fold. First, we propose a novel comparative design that trains two distinct models, A: for general promoter biology and B: for TSp regulation. These models enable the precise isolation of sequence motifs around the transcription start site that uniquely define tissue identity. Second, we introduce a comprehensive explainable AI (xAI) module that integrates attention-based discovery with model-agnostic SHAP analysis to provide robust, cross-validated interpretations of learned features. Applying this framework to human brain, liver, and testis promoters, we identified and validated clinically relevant transcription factors (TFs) in the brain, including SP1, MYC, and HES6, and confirmed their known roles in diseases such as gliomas and neuroblastomas. Our analysis further revealed that C2H2 Zinc Finger proteins are a dominant feature of the global landscape of TSp gene regulation. TSProm provides a novel and interpretable framework for identifying TSp gene regulatory elements, offering powerful computational tools for the study of tissue-specific gene regulation in normal and disease conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a83e73abe66e3648e75b598545eed1991706b17" target='_blank'>
              TSProm: Deciphering the Genomic Context of Tissue Specificity
              </a>
            </td>
          <td>
            Pallavi Surana, Pratik Dutta, Nimisha Papineni, Rekha Sathian, Zhihan Zhou, Han Liu, Ramana V. Davuluri
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel C. de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Intracranial germ cell tumors (IGCTs), primarily affecting infants and children, exhibit malignancy and prognosis that largely depend on tumor histological subtypes. While somatic coding alterations have revealed activation of the MAPK and PI3K pathways in the germinoma subtype, the genetic basis of the more malignant yolk sac tumor (YST) subtype remains poorly understood. In this study, we analyzed whole-genome sequencing data from blood-tumor matched samples of 69 IGCT patients, identifying a significant enrichment of somatic chromosome 1p36 copy number loss in YST (7 out of 13 YST cases compared to 3 out of 56 non-YST cases). Further analysis of whole-genome level somatic single nucleotide variations revealed a YST-specific single base substitution signature (5 out of 13 YST cases), characterized by frequent ACG to AGG mutations. Alongside this novel signature, we observed a high burden of point mutations, structural variations, and LINE1 retrotransposition, indicating increased genomic instability in this subgroup of IGCTs. Additionally, association analyses revealed that these genomic instability events were significantly linked to the copy number loss of 1p36. These findings elucidate the distinct mutational landscape of YSTs and suggest potential treatment vulnerabilities for this subtype by targeting genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d3aac5eea035f8521957bf70a25b2e906a88c96" target='_blank'>
              EPCO-29. WHOLE-GENOME SEQUENCING REVEALS GENOMIC SIGNATURE OF INTRACRANIAL YOLK SAC TUMORS
              </a>
            </td>
          <td>
            Dong Song, Yang Zhao, Ran Liu, Jie Ma, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous single-nucleotide polymorphisms (SNPs) associated with various diseases, including cancer. However, the mechanisms by which these SNPs contribute to disease susceptibility remain largely unclear. While recent studies have explored the transcriptional impact of disease-associated SNPs, their role in post-transcriptional regulation has been less extensively investigated. In this study, we investigated whether cancer-associated SNPs influence gene expression by altering N6-methyladenosine (m6A) RNA methylation. We collected GWAS-identified SNPs across nine cancer types and integrated these with matched tumor and normal m6A RNA immunoprecipitation sequencing (m6A-seq) and RNA sequencing (RNA-seq) datasets. We first identified differentially methylated m6A sites and assessed whether cancer-associated SNPs were enriched within these regions. These analyses revealed that cancer-associated SNPs were significantly enriched within hypermethylated m6A regions in colon cancer. Integrative analysis revealed that SNPs enriched in m6A-modified regions are associated with altered gene expression and RNA splicing, suggesting that m6A methylation mediates the post-transcriptional impact of genetic variants. Experimental validation further confirmed altered gene expression following ALKBH5 knockdown, consistent with patient-derived data. Collectively, our findings support a novel mechanistic connection between genetic variants and RNA methylation-driven transcriptomic regulation in colorectal cancer, underscoring the epitranscriptome as a potential axis of oncogenic control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8030aadad77b99c149edba7083659d634eaa70" target='_blank'>
              Genetic variants reshape the m6A epitranscriptome and drive transcriptomic reprogramming in colorectal cancer
              </a>
            </td>
          <td>
            Seung Hun Han, Seongmin Jang, Yeongwon Kim, Kun Tan, M. F. Wilkinson, Hyobin Jeong, Junho Choe
          </td>
          <td>2025-11-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, , Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Polycomb targeted loci form long-range chromatin interactions independent of CTCF-cohesion and are demarcated by low DNA methylation(1–3). Polycomb targets form extremely long-range loops (long Polycomb loops) that can separate anchors up to 60 Mb. These loops predominantly occur in cells of self-renewal status, such as human hematopoietic stem cells (HSC) and mouse embryonic stem cells (ESC), but rarely in cell lines. To identify long Polycomb loops in cancer, we initiated a pan-cancer survey of long Polycomb loops in a collection of 264 tumor samples (33 acute myeloid leukemias, 17 T cell lymphoblastic leukemia, 29 Breast cancer samples, 63 pediatric brain tumors, 70 prostate cancers, and 42 colon cancers). We found most cancers, including all prostate cancers and colon cancers, lack long Polycomb loops except pediatric brain tumors and certain AMLs. In pediatric brain tumors, we found 30% of the ependymoma PFA subtype (which shows globally depleted H3K27me3) notably displayed strong long Polycomb loop interactions. AML displayed more diverse levels of long Polycomb loop interactions. Most AMLs lost both long Polycomb loops with Polycomb binding loss at the loop anchors. Whereas 10% of AMLs retain long Polycomb loops as strong as in HSCs. These AMLs recurrently carry mutations in CEBPA and STAG2, which are not associated with the Polycomb complex or DNA methylation machinery. We found that long Polycomb loop strong AML is sensitive to EZH2 inhibition, which induces cell differentiation. Conversely, PRC1 component dependency can be used to predict long Polycomb loop formation in B-cell lymphoblastic leukemia cell lines. Our analysis suggests that the oncogenesis process antagonizes the long Polycomb loop maintenance in most cancers, yet certain cancers may still preserve strong long Polycomb loops from cell-of-origin. The maintenance of long Polycomb loops sensitizes cells to Polycomb inhibition, indicating that such loops could be an epigenomic biomarker for pharmacological or genetic Polycomb inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd40648aae14140db85bd195a593affdaa280e98" target='_blank'>
              Pan-cancer 3D genomic analysis revealed extremely long Polycomb loops as the biomarker for sensitivity to Polycomb inhibition
              </a>
            </td>
          <td>
            Sean Moran, Zhong Fan, M. Zanovello, Feng Fan, David Xue Qing Wang, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Acute promyelocytic leukemia (APL) is a subtype of acute myeloid leukemia (AML), characterized by a fusion between the PML and RARA genes and by a block in the myeloid maturation at the promyelocytic stage. This study investigates the epigenetic landscape of APL by integrating ChIP-seq data on eight histone modifications and RNA-seq in APL as well as non-APL AML. APL showed a distinct chromatin profile that differed from non-APL AML. We describe APL-specific changes in H3K27ac, H3K9me3, and H3K27me3 with impact on enhancer activity, repression of transposable elements, and Polycomb regulated gene repression. The APL-specific H3K27ac pattern identifies APL-specific enhancer and super-enhancer regions, including a subset of enhancers that are bound by the PML-RARA fusion protein. While chromatin bound specifically by PML-RARA were dominantly active, APL was also characterized by gain of APL-specific heterochromatin states with significant gains of H3K9me3 enriched lamina-associated domains and the transposable elements LINE, LTR, and SINE. These findings suggest a unique enhancer and heterochromatin profile in APL, with implications for transcription regulation and treatment response. These findings offer novel insights into the pathogenesis of APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4d4d5354c6d704bfe8c1f882f567114dfdc0fb" target='_blank'>
              Disease-specific epigenetic deregulation of enhancers, transposons, and polycomb targets in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Xiangfu Zhong, Lina Cordeddu, Angelica Gamboa-Cedeno, S. Bengtzén, Karl Ekwall, Andreas Lennartsson, Sören Lehmann
          </td>
          <td>2025-10-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="SF3B1, a component of the U2 snRNP pre-mRNA splicing factor, plays a critical role in splicing and is frequently mutated in cancer, particularly hematologic malignancies. We investigated the effects of the most common SF3B1 mutation, heterozygous substitution of Lysine 700 to Glutamate (K700E), in human embryonic stem cells (hESC), using CRISPR-Cas9 to generate heterozygous SF3B1K700E clones. Interestingly, we observed the upregulation of several key transcription regulators associated with hematopoiesis and a broad range of immune genes in SF3B1K700E hESCs. Despite differences in the transcriptional and splicing profiles between hESC and myelodysplastic syndrome (MDS) cells harboring the SF3B1K700E mutation, several common immune gene programs were identified in both cell types. To elucidate the molecular mechanisms underlying dysregulated gene expression in SF3B1K700E hESCs, we mapped actively engaged RNA polymerase II (RNA Pol II) using Precision Run-On sequencing (PRO-seq). These analyses revealed that the SF3B1K700E mutation alters RNA Pol II elongation properties. Specifically, we observed a general increase in pause release in SF3B1K700E hESCs, consistent with recent work in leukemia cells suggesting that the SF3B1K700E mutation affects early transcription elongation. Taken together, our study identifies several candidate genes that could contribute to the SF3B1 mutated phenotype and clarifies the role for the U2 snRNP and pre-spliceosome assembly on transcription by RNA Pol II. Further, our data suggest that mutations of SF3B1 impact immune gene expression independent of cell type, providing new insights into the role of SF3B1K700E in hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21f566d363570cd1232df0a75a294a1715f4579d" target='_blank'>
              SF3B1K700E mutation in human embryonic stem cells causes aberrant expression of immune-related genes
              </a>
            </td>
          <td>
            M. Dastpak, Claudia A. Mimoso, Moein Farshchian, Christina L. Paraggio, Claudia E. Leonard, Shanye Yin, Binkai Chi, Kelsey W. Nassar, R. G. Rowe, Jiuchun Zhang, Zhonggang Hou, George Q. Daley, Karen Adelman, Robin Reed
          </td>
          <td>2025-12-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is generally thought to be caused by expansion of a single mutant cell1. However, analyses of early colorectal cancer lesions suggest that tumors may instead originate from multiple, genetically distinct cell populations2,3. Detecting polyclonal tumor initiation is challenging in patients, as it requires profiling early-stage lesions before clonal sweeps obscure diversity. To investigate this, we analyzed normal colorectal mucosa, benign and dysplastic premalignant polyps, and malignant adenocarcinomas (123 samples) from six individuals with familial adenomatous polyposis (FAP). Individuals with FAP have a germline heterozygous APC mutation, predisposing them to colorectal cancer and numerous premalignant polyps by early adulthood4. Whole-genome and/or whole-exome sequencing revealed that many premalignant polyps—40% with benign histology and 28% with dysplasia—were composed of multiple genetic lineages that diverged early, consistent with polyclonal origins. This conclusion was reinforced by whole-genome sequencing of single crypts from multiple polyps in additional patients which showed limited sharing of mutations among crypts within the same lesion. In some cases, multiple distinct APC mutations co-existed in different lineages of a single polyp, consistent with polyclonality. These findings reshape our understanding of early neoplastic events, demonstrating that tumor initiation can arise from the convergence of diverse mutant clones. They also suggest that cell-intrinsic growth advantages alone may not fully explain tumor initiation, highlighting the importance of microenvironmental and tissue-level factors in early cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed79fdee85797c4fc9c9d2549d5e711e9476ac0" target='_blank'>
              Polyclonal origins of human premalignant colorectal lesions
              </a>
            </td>
          <td>
            D. Van Egeren, Ryan O. Schenck, Aziz Khan, Aaron M. Horning, Shanlan Mo, Clemens L. Weiß, Edward D Esplin, Winston R. Becker, Si Wu, Casey Hanson, Nasim Barapour, Lihua Jiang, K. Contrepois, Hayan Lee, Stephanie A. Nevins, Tuhin K. Guha, Hao Zhang, Zhen He, Zhicheng Ma, Emma Monte, Thomas V. Karathanos, Rozelle Laquindanum, M. Mills, Hassan Chaib, R. Chiu, Ruiqi Jian, Joanne Chan, M. Ellenberger, Bahareh Bahmani, Basil Michael, Annika K. Weimer, D. G. Esplin, Samuel Lancaster, Jeanne Shen, U. Ladabaum, Teri A. Longacre, Anshul B Kundaje, William J. Greenleaf, Zheng Hu, James M. Ford, Michael P. Snyder, Christina Curtis
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese M. Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, Jangkeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Genomic alterations in germ cell tumors (GCTs) have been identified in the RTK, MAPK, and PI3K pathways through whole exome sequencing, with a prevalence of 71% in germinomas but only 33% in non-germinomas GCTs (NGGCTs). Despite evolution in treatment, GCTs present therapeutic challenges and lack molecularly targeted strategies.



 Whole genome sequencing was performed on 71 cases (37 germinomas and 34 NGGCTs) from the iGCT Genome Analysis Consortium and the Japan Children’s Cancer Group.



 Putative driver mutations were identified in 83% of tumors, primarily involving RAS/MAPK followed by mTOR/Akt pathways. Germinomas exhibited a higher mutation frequency than NGGCTs (84% vs 62%), with KIT enriched in germinomas and USP28 in NGGCTs. All USP28 alterations were bi-allelic and highly specific to CNS GCTs, with only one diffuse midline glioma out of 1448 cases of pediatric cancers affected. Variant allele frequency and clonal analyses revealed that these mutations occurred early, preceding copy number alterations. Germline analysis identified Kleinfelter or Down syndrome in 5.6% of cases, particularly in NGGCTs lacking other alterations.



 Many cases exhibited driver mutations present in major clonal populations and could serve as therapeutic targets. There were notable differences based on histological type, with a diverse range of single nucleotide variants/structural variants observed in NGGCTs. A timeline of tumorigenesis from SNVs/SVs to copy number abnormalities was observed. The utility of understanding germline variants was also implicated in the pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6633a5d8f98dc245c8a327ae1335afe0892cad2" target='_blank'>
              PTHP-06. Molecular Mechanisms and Developmental Pathways of CNS Germ Cell Tumors Elucidated through Whole Genome Analysis
              </a>
            </td>
          <td>
            Hirokazu Takami, S. Kato, Yuki Yamagishi, Eiji Sugihara, Y. Matsushita, Takako Yoshioka, Junko Hirato, Miho Kato, Hajime Okita, Tomonari Suzuki, Masayuki Kanamori, Akio Asai, Akitake Mukasa, Toshihiro Kumabe, Akihide Kondo, Yuichi Hirose, M. Nagane, Kazuhiko Sugiyama, T. Shibata, Kenichi Yoshida, Y. Shiraishi, Takaaki Sato, Ryo Nishikawa, Motohiro Kato, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03533d5a535f8136a5f4c27e74c671f6ec467639" target='_blank'>
              cGAS restricts PARP1-mediated microhomology-mediated end joining by suppressing poly-ADP-ribosylation.
              </a>
            </td>
          <td>
            Haibo Yang, Boya Gao, Bo Wu, Wenjing Li, Xueping Zhu, Laiyee Phoon, Jing Luo, Fu Gui, Weixing Zhao, Li Jia, Li Lan
          </td>
          <td>2025-12-17</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0957eb10bf767ffc74bc27558270080adbc791c6" target='_blank'>
              Pan-cancer analysis reveals mtDNA copy number as a key determinant of mutational load and disease progression in cancer
              </a>
            </td>
          <td>
            Anamika Acharyya, Riddhiman Dhar
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79af111fdda73a352f12cb3e7f1b39f6f8b69b19" target='_blank'>
              Pan-oesophageal and spatially resolved genetic analysis reveals instability signature for earlier diagnosis of squamous cell cancer
              </a>
            </td>
          <td>
            Aisha Yusuf, Shalini Malhotra, Maria O’Donovan, G. Devonshire, Sarah Killcoyne, Maryla Turkot, Andrzej Mróz, M. Lenarcik, Michal Mikula, N. Pilonis, Jarosław Reguła, M. Kaminski, K. Nowicki-Osuch, W. Januszewicz, Rebecca C Fitzgerald
          </td>
          <td>2025-11-30</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell chromatin accessibility data provide important insights into the activity of DNA regulatory elements in health and disease. However, the analysis of these data is made challenging by the lack of a common set of features for use in downstream analysis. This results in individual studies quantifying dataset-specific peak regions that cannot be directly compared to other studies. To address this challenge, we developed a comprehensive set of DNA regulatory element modules (REMO) for the human genome. Here we show how REMO can be applied to single-cell chromatin data to better separate cell states in a low-dimensional space compared to peak matrix quantification, greatly improve the scalability of dimension reduction steps, and enable automated annotation of cell types. This is accompanied by new memory-efficient and scalable software for the quantification of single-cell chromatin accessibility data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eccc71216b694c6ee98748140f0fb8a0dea89a2" target='_blank'>
              Regulatory element modules as universal features for single-cell chromatin analysis
              </a>
            </td>
          <td>
            Chrysania Lim, Javen Tan Yih Ruay, Tim Stuart
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most common (~80%) and lethal ovarian cancer subtype in the United States, characterized by TP53 mutations and DNA repair defects causing chromosomal instability (CIN). KIF18A is an essential cytoskeletal motor protein for cell division in CIN+ cancer cells, but it is not necessary for cell division in normal cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CIN+ cancers. We investigated the use of a novel KIF18A inhibitor ATX020, for selectively targeting CIN+ HGSOC cells using growth inhibition assays, invasion assays, immunoassays, cell cycle analysis, and immunofluorescence techniques. Using DepMap and flow cytometry, we classified a panel of HGSOC cell lines based on aneuploidy scores (AS) and ploidy levels and identified a correlation between these classifications and sensitivity against ATX020. ATX020 induced cytotoxicity through mitotic arrest and DNA damage, and reduced tumor growth in HGSOC with high aneuploidy scores (AS). Mechanistically, ATX020 blocks KIF18A’s plus-end movement on spindle fibers, increasing spindle length, resulting in chromosomal mis-segregation, aneuploidy, and DNA damage. Our findings suggest that ATX020 inhibits CIN+ HGSOC cells mainly by inducing mitotic arrest and DNA damage, disrupting KIF18A’s function crucial for mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f42a6d74edbdfa8c878ba6b61be09681cd9d76" target='_blank'>
              The KIF18A Inhibitor ATX020 Induces Mitotic Arrest and DNA Damage in Chromosomally Instable High-Grade Serous Ovarian Cancer Cells
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Tzu-Ting Huang, Maureen Lynes, Sanjoy Khan, Serena J Silver, Jung-min Lee
          </td>
          <td>2025-11-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh R. Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, Garrett Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Extrachromosomal circular DNA (eccDNA) has emerged as a critical player in cancer biology. Accumulating evidence indicates that eccDNA promotes tumor progression by inducing genomic instability, amplifying oncogenes, and regulating gene expression. However, its biological function and mechanistic role in acute myeloid leukemia (AML) remain unexplored. In this study, we investigated the functional significance and regulatory mechanisms of eccDNA in AML, aiming to identify potential therapeutic targets.



 Bone marrow samples were collected from newly-diagnosed and chemo-resistant AML patients with informed consent. High-throughput eccDNA sequencing was performed to determine eccDNA length distribution. The presence of gene-containing eccDNAs was validated by PCR and fluorescence in situ hybridization (FISH), and synthetic eccDNA constructs were generated to investigate their functional roles in AML. In vivo, we evaluated the impact of eccDNA-encoded LIPIN1 (ecc-LIPIN1) on AML progression and chemoresistance, and developed a PEI/PC7A nanoparticle system for therapeutic intervention. Proteomic and transcriptomic analyses were performed to explore the downstream targets.



 High-throughput eccDNA sequencing revealed abundant eccDNAs (0.3-1 kb) across newly-diagnosed and chemo-resistant AML samples, mapped to diverse genomic regions. Analysis of the GEO dataset (GSE165430) identified genes upregulated in chemo-resistant AML patients. Integrating these data with eccDNA-derived genes highlighted ecc-LIPIN1 as a potential resistance driver.
 PCR and FISH confirmed the presence of eccDNA harboring LIPIN1 nucleotide fragments. Ecc-LIPIN1 levels were significantly elevated in AML samples, particularly in chemo-resistant cases, and associated with poor prognosis. A circular DNA containing LIPIN1 was synthesized based on sequencing data and transfected into AML cells, resulting in elevated LIPIN1 expression. Functional assays demonstrated that ecc-LIPIN1 promoted AML cell proliferation and chemoresistance.
 Given its metabolic role, metabolomic profiling assay revealed that ecc-LIPIN1 primarily regulated lipid metabolism. Fluorescence microscopy and flow cytometry confirmed that ecc-LIPIN1 enhanced lipid droplet formation, contributing to AML progression and chemo-resistance. Proteomic and transcriptomic analyses of ecc-LIPIN1-overexpressed AML cells identified DGAT1, a key enzyme in lipid metabolism, as a downstream effector. Silencing DGAT1 in ecc-LIPIN1-overexpressed AML cells markedly reduced lipid droplet accumulation, indicating that ecc-LIPIN1 drives lipid storage through DGAT1.
 Further analyses revealed that LIPIN1 also modulated endoplasmic reticulum (ER) stress pathways. In LIPIN1-overexpressed AML cells, ER-stress markers CHOP and ATF4 were markedly elevated. Silencing DGAT1 reversed ER-stress activation, suggesting that LIPIN1 promoted AML progression by regulating ER stress via DGAT1.
 To validate the oncogenic function of ecc-LIPIN1, we generated eccDNA constructs containing either wild-type or mutant LIPIN1 nucleotide sequences and transfected them into AML cells. The wild-type ecc-LIPIN1 markedly enhanced AML progression and chemo-resistance both in vitro and in vivo. To counteract this effect, we developed a PEI/PC7A/siLIPIN1 nanoparticle system. Treatment with these nanoparticles efficiently suppressed ecc-LIPIN1 expression, inhibited AML progression, and mitigated chemo-resistance.



 This study provides preclinical evidence that eccDNA plays a critical role in AML progression and chemoresistance by encoding LIPIN1. Mechanistically, ecc-LIPIN1 promotes lipid droplet formation and ER stress via DGAT1, ultimately driving chemoresistance. These findings identify ecc-LIPIN1 as a novel oncogenic factor in AML and highlight its potential as a therapeutic target. Future research into eccDNA biogenesis in AML may uncover new strategies for combating drug resistance and disease progression.



 Extrachromosomal circular DNA; Acute Myeloid Leukemia; drug resistance; lipid metabolism; LIPIN1.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cecd3734536748347f09719465be9ffb74475eb" target='_blank'>
              Eccdna-encoded LIPIN1 drives DGAT1-dependent lipid droplet biogenesis to promote Acute Myeloid Leukemia progression and chemoresistance
              </a>
            </td>
          <td>
            Wancheng Liu, Amin Zhang, Xiangling Xing, Daoxin Ma
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Central nervous system germ cell tumors (CNSGCTs) are the most enigmatic brain tumors. They consist of several histologically distinct types of tumors, which are generally sub-grouped into germinomas or non-germinomatous germ cell tumors (NGGCTs). While germinomas are characterized by alterations in the MAPK/PI3K-pathway genes and global hypomethylation, pathogenesis of NGGCTs is largely unknown. To investigate genetic and epigenetic mechanisms of CNSGCT development, 33 CNSGCTs including 7 germinomas and 26 NGGCTs of various histology collected through the Intracranial Germ Cell Tumor Genome Analysis Consortium and Japan Children’s Cancer Group, together with their matched normal DNA, were subjected to a long-read whole genome sequencing using the PromethION system from Oxford Nanopore Technologies. The mean reading depth was 39X for tumor specimen and 31x for blood DNA. The mean N50 value was 9,565 bp. The mean/median number of structural abnormalities was 16.6/16 for germinomas and 27.9/10.5 for NGGCTs, which was not statistically significantly different. A 2 kb deletion involving an exon of USP28 was observed in a single mixed-germ cell tumor case. Deletions involving PRKN were observed in 4 cases. The mean number of ectopic somatic insertions of the LINE-1 element was 1.8 per case. A single mixed GCT had 21 LINE-1 insertions, one of which was located within an intron of NTRK2. The mean of the whole genome DNA methylation level was 38.2% in germinomas and 75.9% in NGGCTs (p=0.0005). Germinomas mostly had biallelic DNA hypomethylation in the imprinted genes, consistent with their presumed primordial germ cell (PGC) origin. On the other hand, NGGCTs showed various levels of methylation in the imprinted genes, suggesting that they may also have originated from PGCs, however at different stages from that of germinomas. Taken together, our whole genome DNA methylation haplotyping provided a new clue to elucidate the pathogenesis of NGGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a0bfd993cd8d6ea3ee9b1101edf3e65daa2bfa" target='_blank'>
              EPCO-07. A WHOLE GENOME DNA METHYLATION ANALYSIS REVEALED DISTINCT MECHANISMS OF DEVELOPMENT IN DIVERSE TYPES OF CENTRAL NERVOUS SYSTEM GERM CELL TUMORS
              </a>
            </td>
          <td>
            Shoko Yoshimoto, Hirokazu Takami, Erina Ishikawa, Yuta Kuze, Takako Yoshioka, Junko Hirato, Nobuhito Saito, Toshihiro Kumabe, Ryo Nishikawa, Yutaka Suzuki, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pulmonary pleomorphic carcinoma (PPC) is a rare subtype of lung cancer that comprises both epithelial and sarcomatoid components. The molecular basis of PPC, including the cellular dynamics of its components, remains largely unknown. To elucidate potential therapeutic targets for PPC, we perform a multi-omics analysis incorporating digital spatial profiling and single-cell RNA sequencing (scRNA-seq). PPC exhibits diverse driver gene alterations, including MET exon 14 skipping mutation (METex14) and ALK fusion. In spatial transcriptomics, MET gene and protein are overexpressed exclusively within the epithelial component and not in the sarcomatoid component, even in patients harboring METex14. Epithelial-mesenchymal transition (EMT)-related transcriptional changes, along with extracellular matrix (ECM) remodeling between the epithelial and sarcomatoid components, are observed. scRNA-seq identifies cell populations within the epithelial component that contribute to the malignant transformation and differentiation of the sarcomatoid component. They are characterized by an intermediate EMT state with ECM remodeling signature, suggesting their potential as novel therapeutic targets for PPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f421d642675df6e6bdc9b51c7d9b23970c7bec20" target='_blank'>
              Single-cell and spatial transcriptomic characterization of pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Masayoshi Ohki, Kazuya Hisamatsu, Ryota Fujiwara, Kosei Ishimura, Ryunosuke Fujii, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Ryo Yoshichika, Fumiaki Mukohara, Mao Yoshikawa, Yuma Fukumoto, K. Suzawa, Y. Tomioka, Shin Tanaka, Kentaroh Miyoshi, Mikio Okazaki, S. Sugimoto, Yusuke Otani, Atsushi Tanaka, Hirofumi Inoue, Yosuke Togashi, Hidetaka Yamamoto, D. Ennishi, Shinichi Toyooka
          </td>
          <td>2025-12-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d7d2e338e9c2ff093882c73a51e612889ed2af" target='_blank'>
              Histone H1 Promotes Silencing of Unintegrated HIV-1 DNA
              </a>
            </td>
          <td>
            Jiaqi Zhu, Gary Z. Wang, Hugo D. Pinto, Sean E. Healton, Arthur I Skoultchi, Stephen P. Goff
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints.
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Introduction: During the past two decades, our knowledge about the molecular pathogenesis of myeloid neoplasms (MNs), including acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) has dramatically been improved through the identification of major driver alterations using next generation sequencing. However, with the lack of large-scale analysis using whole genome sequencing (WGS) with sufficient sequencing depths, the analyses have mainly focused on the alterations affecting protein-coding sequences, while the role of non-coding alterations and structural variations (SVs) in myeloid leukemogenesis has not fully been investigated. The etiology of mutagenic processes has also been poorly understood. In the current study, we therefore conducted large-scale deep WGS to address these issues.
 Methods: We performed deep WGS with matched normal samples (mean sequence depth 127.0/30.3 for tumor/normal, respectively) for a total of 903 patients with MN (494 AML and 419 MDS), with an additional 263 cases (94 AML and 169 MDS) currently under analysis. Most cases were also analyzed using deep targeted capture sequencing (n=886) of 446 known/putative driver genes. In addition, RNA sequencing was newly performed for nearly half of cases (n=406). Mutation calling was performed using Mutect2 and GRIDSS2, with in-house modifications to control false positivity in the face of frequent contaminations of tumor cells into germline samples.
 Results: We identified 1,159,638 SNVs and 76,842 sInDels with median mutation burden of 0.46 SNVs and 0.030 sInDels per Mb in AML and 0.39 and 0.022 per Mb in MDS. Validation by deep targeted capture sequencing estimated a true-positive rate of 95% and 78.1% sensitivity for known drivers. In the analysis of mutational signatures, we found that clock-like signatures SBS1 and SBS5 were predominant. SBS19 and SBS32 were detected in 62%/14% and 68%/14% of AML and MDS cases, respectively. As expected, chemotherapy related signatures (SBS25, 31 and 99) were significantly enriched in cases with therapy-related MN cases. The SBS18 signature, which is implicated in oxidative stress–induced mutagenesis, was highly enriched in AML cases carrying t(8;21) and inv(16), but rarely found in those with MDS, highlighting the role of oxidative stress in core binding factor leukemia. Other novel signatures such as SBS24, 39 and 40b were detected in a small number of cases (~1%).
 Our study of a large cohort of MNs also revealed novel candidate driver genes. By evaluating mutation enrichment, we identified a total of 82 candidates of driver genes. Among these, 15 genes were not previously reported as driver genes in MNs, including 7 genes listed in COSMIC cancer gene census database as associated with other cancer types. RNA-seq enabled the identification of intronic mutations predicted to cause alternative splicing in known driver genes, including TET2 and DNMT3A, as well as in novel candidates, observed in 2.1% of cases. In addition to coding genes, mutations in non-coding RNAs with potential driver role (e.g. seed region of mir-142) were detected in 3.3% of the cases.
 We identified a total of 11,050 SV events, of which 48 disrupted common tumor suppressor genes, such as RUNX1, ETV6, CBL, and TP53. Chromothripsis events were detected in 5.3% of cases, which were highly enriched in TP53-mutated cases and frequently affected KMT2A, ETS1, ETV6, TP53, EPOR and ERG. Median of 4 chromosomes were affected in a single chromothripsis event.
 We detected a total of 340 fusion events in 22.6% of all cases, of which 105 fusions involved previously known targets, whereas the remaining 235 represented non-recurrent in-frame gene fusions detected in 11.7% of cases. Although such fusion events were significantly associated with TP53 mutations, 5.6% of TP53-wild-type cases also had in-frame gene fusions, some of which were suggestive of oncogenic potential (e.g. KAT6A::NUTM1). Although low in frequency (1.1%), analysis of expression change revealed SVs affecting cis-regulatory elements including potential enhancer hijacking of a RAS pathway gene.
 Conclusion: Through deep WGS of a large cohort of MN cases, we have delineated a comprehensive landscape of driver mutations and detected new mutational signatures and candidates for driver mutations, including those affecting non-coding regions, underscoring the importance of WGS for better understanding of the pathogenesis of MNs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4576f85031bf75aa25ad786753882bdfc5ce2543" target='_blank'>
              Repertoire of driver events in coding and non-coding regions identified by whole-genome sequencing of myeloid neoplasms
              </a>
            </td>
          <td>
            Koji Okazaki, R. Saiki, Y. Ochi, Kenichi Yoshida, K. Katayama, Hidehito Fukushima, Kaito Mimura, M. Nakagawa, Masanori Motomura, A. Yoda, R. Okuda, Lanying Zhao, Motohiro Kato, Tatsuki Ogasawara, Shigeo Fuji, Lee-yung Shih, N. Uoshima, Mitsumasa Watanabe, Yasushi Miyazaki, M. Ichikawa, Takeshi Maeda, Toshiyuki Kitano, Satoshi Yoshihara, Kazunori Imada, Kinuko Mitani, J. Jansen, M. Sakata-Yanagimoto, Luca Malcovati, Akihiko Gotoh, Hitoshi Kiyoi, N. Kanemura, N. Sezaki, N. Hiramoto, S. Kasahara, S. Imoto, A. Takaori-Kondo, Y. Nannya, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Senescence is a pleiotropic phenotype that alternatively suppresses or promotes cancer. The tumor suppressive roles are linked to clearance of damaged cells by the immune system, while cells with tumor promoting functions resist apoptosis, evade immune clearance and either persist and support the tumor microenvironment, or escape and proliferate. What generates these diverse populations is unclear. We investigated this in preneoplastic zebrafish livers where the epigenetic regulator, UHRF1, is overexpressed in hepatocytes. Double strand breaks, DNA methylation repatterning, retrotransposon expression, cell cycle withdrawal and activation of Atm and Tp53 dependent senescence were early responses to UHRF1 overexpression. This evolved to generate diverse populations of senescent cells, some which expressed immune and senescence signatures plus anti-apoptotic markers, and others co-expressed proliferative genes. The fate of these populations was dictated by UHRF1 levels and Tp53, as Tp53 loss enabled proliferation of cells with reduced UHRF1 expression but not in cells expressing high UHRF1. The senolytic Navitoclax targeted only a subset of senescent cells. Thus, the diversity of senescent cells driven by epigenetic changes can generate divergent outcomes. Short summary UHRF1 overexpression in zebrafish hepatocytes induces DNA damage leading to Atm–tp53–dependent senescence with UHRF1 levels dictating whether hepatocytes remain senescent, escape, or undergo senolytic elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Chromosomal loops are CTCF–cohesin-mediated 3D genomic structures in the mammalian nucleus. In addition to these CTCF–cohesin loops, other epigenomic features can also form long-range interactions. Polycomb-targeted loci, regulated by the Polycomb repressive complex and trithorax group during development, form long-range chromatin interactions independent of CTCF–cohesin and are demarcated by regions of low DNA methylation—referred to as DNA methylation canyons.
 We previously identified extremely long Polycomb loops occurring between DNA methylation Canyon demarked Polycomb-targeted loci, spanning distances of up to 60 Mb (Zhang et al, 2020). These loops are found exclusively in self-renewing cells, such as human hematopoietic stem cells and mouse embryonic stem cells.
 In hematopoietic malignancies, both DNA methylation and Polycomb binding are significantly altered. To explore this further, we conducted a pan-cancer survey of long Polycomb loops across 223 tumor samples, with a focus on hematopoietic malignancies—given that their likely cell-of-origin, the hematopoietic stem cell, exhibits strong long Polycomb loops. Our cohort included 32 acute myeloid leukemias (AMLs), 24 T-cell lymphoblastic leukemias, 61 pediatric brain tumors, 80 prostate cancers, and 26 colon cancers. We found that most cancers—including all prostate and colon cancers—lack long Polycomb loops. However, long Polycomb loops are retained in pediatric brain tumors and a subset of AMLs. Interestingly, strong long Polycomb loop interactions are observed in normal developing brain tissue and hematopoietic stem cell. Our data suggest that the presence of long Polycomb loops is likely inherited from the epigenomic state of the cell of origin, as they are also observed in normal pediatric brain tissue and hematopoietic stem cells.
 Loss of long Polycomb loops in primary cancer samples is accompanied by DNA hypermethylation and reduced Polycomb binding at loop anchor loci, and these disruptions are further exacerbated in cultured cell lines. Interestingly, in AML, many of these previously silenced loci—often mesodermal transcription factors—become activated and form de novo 3D interaction anchors. For example, we observed the formation of a new domain around the leukemogenic ZEB2 gene, driven by leukemia-specific HOXA9 binding at an upstream enhancer within the TEX41 locus.
 Notably, AML samples displayed a wide range of long Polycomb loop strength. While most AMLs lose these loops, approximately 12% (4 out of 33) retain strong long Polycomb loops comparable to hematopoietic stem cells. These cases recurrently harbor somatic mutations in CEBPA (2 out of 4) and STAG2 (2 out of 4). These genes are not directly linked to Polycomb or DNA methylation machinery, suggesting alternative mechanisms of loop maintenance. The CEBPA mutant AMLs also exhibited H3K27me3 spreading to non-Polycomb target loci across the genome.
 We therefore tested whether EZH2 inhibition could disrupt long Polycomb loops in these AMLs and impact disease maintenance. Indeed, EZH2 inhibition attenuated long Polycomb loops, reduced colony-forming capacity, and promoted differentiation in long Polycomb loop–retaining AMLs. Treated AML cells activated a macrophage differentiation program and showed downregulation of cell cycle and DNA replication genes. These findings suggest that AMLs retaining long Polycomb loops are dependent on this 3D chromatin architecture. This rare but strong sensitivity to EZH2 inhibition in AML indicates that long Polycomb loops could serve as an epigenomic biomarker for EZH2 or other Polycomb-targeted therapies in cancer rather than the current genomic marker such as EZH2 gain-of-function mutations and SWI/SNF loss-of-function mutations.
 Overall, long Polycomb loops are commonly lost during leukemogenesis due to epigenomic disruption. However, a subset of AMLs maintains these loops from the cell of origin and appears to rely on the Polycomb network to sustain self-renewal.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2c56d775b22a7b9218367e039106ed0000301e" target='_blank'>
              Pan-leukemia 3D genomic profile reveals extreme long polycomb loop in subset of CEBPA mutated AMLs sensitive to EZH2 inhibition
              </a>
            </td>
          <td>
            Sean Moran, M. Zanovello, Zhong Fan, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ab2b82b250c7fe0872534fbb6e35f7aaf3d36d" target='_blank'>
              Cell State Chaos Underpins the Evolution of SMARCA4-Deficient Dedifferentiated Endometrial Cancer
              </a>
            </td>
          <td>
            M. Brandon-Coatham, Julia Vassalakis, Hannah Plummer, Jasleen Kaur, Farzaneh Afzali, Zhihua Xu, Guihua Zhang, Jiahui Liu, Wei Wang, Einav Wajsbrot Renert, Tyler T. Cooper, Alan Dimitriev, Cierra Perron, Andrew Garven, Simatsidk Haregu Abebe, David M Berman, Amber L Simpson, G. Wong, Gilles A. Lajoie, Felix K.F. Kommos, Andreas von Diemlin, Sheela A. Abraham, David J.H.F. Knapp, Alan Underhill, M. Koebel, Cheng Han-Lee, Lynne-Marie Postovit
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objective The calcium-dependent activator protein for secretion (CAPS) has emerged as a protein of interest in tumor biology; however, its functional significance in endometrial carcinoma (EC) remains largely unexplored. Methods We performed an integrative analysis leveraging bulk RNA sequencing data from The Cancer Genome Atlas, GTEx, and publicly available EC datasets, coupled with gene set enrichment analysis (GSEA) and single-cell transcriptomic profiling. The association between CAPS expression and clinicopathological parameters as well as patient prognosis was systematically assessed. In addition, functional enrichment, intercellular communication, and immune infiltration analyses were conducted to elucidate the molecular and microenvironmental roles of CAPS. In vitro assays were employed to validate the biological effects of CAPS knockdown on EC cell proliferation, migration, and invasion. Results CAPS was significantly upregulated in early-stage, low-grade, and endometrioid endometrial carcinoma, and its elevated expression was associated with improved patient survival. GSEA indicated that high CAPS expression was correlated with enhanced ribosome biogenesis and oxidative phosphorylation, along with suppression of proliferative signaling, reflecting a metabolically differentiated phenotype. Single-cell transcriptomic analysis revealed that CAPS was specifically enriched in terminally differentiated secretory epithelial cells, promoting epithelial maturation and oxidative metabolism. Furthermore, CAPS-positive cells exhibited extensive crosstalk with immune cells, suggesting a potential role in facilitating anti-tumor immune responses. Functional assays confirmed that CAPS knockdown markedly enhanced EC cell proliferation, migration, and invasion. Conclusion CAPS may act as a tumor suppressor in EC by stabilizing microtubule architecture, reprogramming cellular metabolism, and modulating immune interactions within the tumor microenvironment. These findings highlight CAPS as a promising prognostic biomarker and potential therapeutic target in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/986f64ba61a7288a4dba4bccd0822c838c05f3b8" target='_blank'>
              Pan-cancer multi-omics characterization of calcyphosine and its revealed links to the immune microenvironment and regulatory networks in endometrial carcinoma
              </a>
            </td>
          <td>
            Xi Chen, Ruiyue Wu, Zijing Wang, Fengfeng Wang, Dan Hu, Xue-han Bi, Xiaolei Liang, Yongxiu Yang
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Age is a dominant risk factor for all major breast cancer subtypes. However, the mechanisms by which aging influences tumor development remain unclear. Using a novel mouse model whereby breast cancer is induced in situ in young and old wild-type mice via intraductal delivery of a lentivirus encoding the HER2/neu oncogene, we found that old mice exhibited a higher oncogene-induced tumor burden than young mice. Old tumor cells showed reduced expression of interferon-related genes, particularly the T cell-recruiting chemokines Cxcl9 and Cxcl10, linked to their altered chromatin accessibility. CXCL9/10 expression also declined with age in human HER2+ tumors. Correspondingly, old tumors exhibited fewer T cells within tumor lesions. Targeted interventions showed that decreased expression of Cxcl9/10 is responsible for reduced T cell infiltration and weakened anti-tumor immunity. These results show how aged tumor cells are impaired in their recruitment of immune cells, leading to a defective anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38011df7fcbd774b6090832583a2578934863ea" target='_blank'>
              Aging alters tumor cell - T cell crosstalk to promote breast cancer progression
              </a>
            </td>
          <td>
            Shanshan Yin, Kay T Yeung, S. Mamde, Kate Lin, Armin Gandhi, Xue Lei, Andrew Davis, Rouven Arnold, Jing Yang, P. Adams
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) is a powerful isothermal strategy for nucleic acid detection, but its linear kinetics and dependence on externally supplied primers limit its sensitivity and programmability. Here, we report an exponential RCA (E-RCA) platform that integrates primer regeneration and signal amplification into a single DNA-encoded system. The design uses a circular DNA template encoding the I-R3 self-cleaving DNAzyme sequence; during RCA, tandem I-R3 units are generated within the DNA amplicons, which then catalyze site-specific cleavage to release new primers. This self-sustained (RCA ↔ DNAzyme) amplification circuit enables robust exponential signal growth using only a circular DNA probe and a DNA polymerase without requiring external primers or protein enzymes. We elucidate the mechanism through biochemical experiments and kinetic modeling and validate the system in multiplexed intracellular microRNA imaging and quantitative dual-marker profiling of clinical breast cancer tissues. The E-RCA strategy achieved high diagnostic accuracy (AUC = 0.914; specificity = 100%; sensitivity = 81.3%), demonstrating its potential for sensitive, programmable, and autonomous molecular analysis in both biological and clinical contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779a0bfc06573a96632b1849e24d829296851cc1" target='_blank'>
              Autonomous Exponential Amplification via Rolling Circle-DNAzyme Feedback Programming.
              </a>
            </td>
          <td>
            Jinhua Shang, Mengdi Yu, Yifei Wang, Shanshan Yu, Xianzheng Zhang, Fuan Wang, Yingfu Li
          </td>
          <td>2025-11-26</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Efficient proteostasis and immune evasion are both critical for tumor progression. The chaperonin TRiC/CCT complex, which mediates the folding of cytoskeletal and signaling proteins, has been associated with oncogenesis; however, the specific role of its subunit CCT4 in tumor–immune interactions remain unclear. Methods To address this gap, we integrated transcriptomics, proteomics, genomics, epigenetics and immunogenomics data. A comprehensive pan-cancer analysis was conducted (including the expression patterns, clinical relevance, prognosis value, immune infiltration of pan-cancer). Then an in-depth analysis of lung adenocarcinoma (LUAD) was carried out through enrichment analysis and single-cell RNA sequencing, and verified through in vitro cell experiments. Results CCT4 was found to be aberrantly upregulated across a majority of tumor types, particularly in LUAD, where elevated expression was associated with advanced stage and inferior survival outcomes. High CCT4 levels were linked to reduced immune cell infiltration and diminished anti-tumor immune signaling, specifically manifested as increased Th2 cell infiltration and decreased Th1 and CD8+ T-cell signatures. Single-cell analyses revealed coordinated overexpression of all CCT subunits in tumor epithelial cells, supporting a global TRiC activation. However, CCT4 was preferentially enriched within highly proliferative subclusters, suggesting partial subunit-specific regulation. CCT4 knockdown suppressed LUAD cell proliferation, migration, and invasion in vitro. Conclusions CCT4 links enhanced proteostasis with immune evasion in LUAD, acting partly through TRiC complex activity and possibly through independent nuclear functions. These findings refine the understanding of how proteostatic machinery contributes to immune modulation in cancer and highlight CCT4 as a potential molecular node bridging tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4892bb2a6429b687729b7113a17a2bf3b326e37" target='_blank'>
              Pan-cancer multi-omics analysis of CCT4 in tumor progression and cancer immunity, with focus on lung adenocarcinoma
              </a>
            </td>
          <td>
            Yonghui Wu, Zhenpeng Wu, Junjie Wen, Desheng Zhou
          </td>
          <td>2025-12-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Waldenström macroglobulinemia (WM) is a lymphoplasmacytic lymphoma characterized by recurrent somatic mutations in MYD88 and CXCR4, which influence response to therapy. Multiomic analysis revealed an expansion of memory B-cells (MBC) in WM, variably impaired in their capacity to differentiate into plasma cells (PCs), allowing classification into two distinct subtypes; MBC-like and PC-like. To deepen understanding of the genomic underpinnings of these subtypes, we performed the largest whole-genome sequencing (WGS) analysis of WM to date. WGS enhanced the capacity to detect and classify somatic mutations, identify single base substitution (SBS) signatures, and characterize structural variants (SVs), providing deeper insights than previously attainable.
 We performed full-depth WGS (80x) on 47 newly diagnosed WM cases using the Illumina NovaSeq 6000 platform, sequencing both B cells and matched T cells. Our harmonized bioinformatics pipeline (MGP1000) enabled robust somatic variant calling. We characterized single nucleotide variants (SNVs), SBS signatures, copy number variations (CNVs), SVs, and reconstructed clonal phylogenies.
 Overall, we found mutations in MYD88 (87%), CXCR4 (28%), CD79B (15%), TP53 (11%), ARID1A (9%), H1-4 (9%), MAP3K14 (NIK) (9%), and KMT2D (4%). These mutations affect B-cell receptor (BCR) signaling, genomic stability, and treatment resistance. The variant allele frequency (VAF) of MYD88 ranged from 20% to 100%, while that of TP53 ranged from 50% to 100%, and other mutations averaged 30–50%. MBC-like (n=10) cases had an increased frequency of mutations in MYD88, CXCR4, CD79B, ARID1A, H1-4, KMT2D, and MAP3K14. In contrast, PC-like (n=22) cases were more frequently associated with gain of 1q and deletion of 6q. Across the different subsets, gain 1q was found in 60% and del of 6q in 40% of MBC-like cases, compared to 73% and 55% of PC-like cases, respectively. SBS signatures showed minimal subgroup variation.
 To contextualize structural variants (SVs) as drivers of WM, we quantified and compared the number and types of SVs with those in other B-cell non-Hodgkin lymphomas (NHL). The average number of deletions per subject was 3.4 in chronic lymphocytic leukemia (CLL), 32.8 in diffuse large B-cell lymphoma (DLBCL), 37.3 in multiple myeloma (MM), and only 8.6 in WM. The average number of duplications was 0.4 in CLL, 8.6 in DLBCL, 7.9 in MM, and 5 in WM. Consistent with this reduced impact, WM also had fewer examples of the complex SV events chromothripsis and chromoplexy. However, subtle differences existed between subtypes, with SVs seen more frequently in MBC-like cases (average deletions per case: 9.7 and duplications: 6.9) compared to PC-like cases (deletions 7.6 and duplications 4.4).
 The size of the dataset and the use of WGS gave us the ability to understand in more detail the impact of del 6q on the pathogenesis of WM both overall and within subtypes by taking account of its ability to map the extent of the deletions to identify minimally deleted regions (MDRs) containing potential driver genes. Detailed analysis of the MDRs across all cases and subtypes revealed differences in the extent of deletion. The MDRs on 6q ran from 73.5 Mb to 80.0 Mb. As deletion of B-cell specific genes, has been suggested previously to be a key driver of progression in WM, we focused our analysis on these genes from chromosome 6 and identified recurrent deletions of BCLAF1, TNFAIP3, HIVEP2, ARID1B, and CCR6. Notably, chromothripsis, a phenomenon involving extensive clustered chromosomal rearrangements, was detected involving a del 6q in one MBC-like case, suggesting a potential mechanism for how del 6q arises.
 We further examined gain of 1q, present in 60% of all cases, as a potential marker of high-risk WM. The minimally gained region on this chromosome as being from 148.5 Mb to 149.7 Mb. Another component of such a high-risk subgroup would include deletions of 17p with copy number loss or mutation of the TP53 tumor suppressor gene. We determined the extent of this group in newly diagnosed WM and found TP53 mutations in 12%, copy number loss of 17p in 11% and biallelic inactivation in 6% of cases.
 To further elucidate disease evolution, we reconstructed clonal phylogenies within and across these subtypes, revealing subtype-specific evolutionary trajectories. Collectively, our findings offer new insights into the genomic architecture of WM and establish a framework for refining subtype-specific therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0d03bf5f658c456cbcb39a1d6824caabef10a7" target='_blank'>
              The molecular landscape of Waldenström macroglobulinemia identified using whole genome sequencing
              </a>
            </td>
          <td>
            Sanghoon Lee, P. Blaney, Dylan C Gagler, Tom Gutman, Di Zhang, Avital Tenenbaum, J. Langton, Marine Armand, Alexandre Eeckhoutte, Amina Joudat, Michael Degaud, Michela Esposito, Gaurav Varma, Yubao Wang, David Kaminetzky, Marc J Braunstein, Louis Williams, F. Nguyen‐Khac, Brian Walker, Damien Roos Weil, Faith Davies, Olivier A Bernard, G. Morgan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 In T-cell acute lymphoblastic leukemia (T-ALL), LMO2 (Lim domain only 2) is frequently overexpressed due to chromosomal translocations or epigenetic changes. LMO2 is a molecular scaffold connecting LDB1 (Lim domain binding 1) to DNA-binding transcription factors. LDB1 is a major chromatin architectural factor involved in long-range enhancer-promoter interactions. Given the prevalence of aberrant LMO2 expression in T-ALL, we hypothesized that LDB1 promotes T-ALL proliferation/survival by connecting oncogenic enhancers to their target genes.
 We performed LDB1 ChIP-seq in 2 T-ALL cell lines with high LMO2 expression: LOUCY (early thymic precursor [ETP]; SET::NUP214) and KOPT K1 (non-ETP; TCRG::LMO2). LDB1 is predominantly recruited to cis-regulatory elements, with 40-60% of peaks localizing to enhancers. However, LDB1 enhancer occupancy differed significantly between cell lines, suggesting subtype-specific regulation of transcriptional programs. Indeed, motif enrichment analysis at LDB1-occupied enhancers demonstrated distinct recruiting factors in LOUCY versus KOPT K1. To enable rapid LDB1 degradation, we tagged endogenous LDB1 with dTAG in both cell lines. Treatment with dTAG-V1 ligand for 4hrs depleted LDB1 by >90%. To assess LDB1-dependent enhancer-promoter connectivity, we treated cells with dTAG-V1 for 4hrs and performed Micro-C sequenced to a depth of ~1 billion valid contacts. Globally, compartmentalization and topologically associated domains remained largely intact in the absence of LDB1. However, we observed widespread changes to loops involving LDB1-occupied enhancers and/or promoters. Importantly, histone modifications and chromatin accessibility at enhancers remained intact upon acute LDB1 depletion, suggesting that architectural changes were directly due to loss of LDB1's looping function rather than indirect consequences of enhancer loss.
 To explore how architectural changes impact gene expression, we measured nascent transcription using TT-seq upon acute LDB1 depletion and observed 328 and 134 differentially expressed genes in LOUCY and KOPT K1, respectively. LDB1-dependent transcripts in LOUCY included critical hematopoietic oncogenes (HHEX, MYB, MYCN, ERG, MEF2C) while those in KOPT K1 were enriched in immune activation pathways (DUSP6, BACH2, CD69). Reduced transcriptional output at these loci was associated with weakened enhancer-promoter connectivity. We analyzed 26 H3K27ac HiChIP datasets from patients with T-ALL and found that LDB1-dependent loops at these oncogenic loci are recapitulated and often correlate with T-ALL differentiation state. These findings suggest that LDB1 connects enhancers with oncogenic driver genes in both cultured and primary T-ALL cells.
 To determine if LDB1 is required for T-ALL proliferation/survival, we passaged LOUCY and KOPT K1 cells in the presence of dTAG-V1 for 15 days and observed loss of cell expansion. Loss of LDB1 in LOUCY induced apoptosis, while in KOPT K1 it elicited a senescent phenotype characterized by beta-galactosidase activation, cell cycle exit, and senescence-associated gene expression. To determine which gene expression changes accounted for these phenotypes, we restored expression of several LDB1-dependent genes in LDB1-depleted cells using CRISPRa. Reactivation of oncogenes like MYB partially rescued cell expansion, suggesting that LDB1 dependency in T-ALL may in part be conveyed through MYB.
 Finally, we compared LDB1-dependent transcriptional signatures from LOUCY and KOPT K1 to transcriptomic data from >1300 patients with T-ALL. The LOUCY signature was enriched in several high-risk T-ALL subtypes, including the recently defined ETP-like and LMO2 γδ-like entities. The KOPT K1 signature was conversely enriched in more differentiated subtypes such as TAL1/DP-like, together suggesting that LDB1 may be required to maintain transcriptional identity of T-ALL subtypes in vivo. Notably, LOUCY signature enrichment conferred worse overall survival (p=0.026) and event-free survival (p=0.00032), driven in part due to LDB1 regulation of high-risk transcriptional features in the ETP-like subtype.
 Together, our study highlights LDB1 as a critical architectural factor that connects T-ALL subtype-specific enhancers and promoters to regulate transcriptional programs required for T-ALL cell expansion. Analyses of patient-derived data corroborate our findings and implicate the prognostic and therapeutic value of LDB1 regulatory networks.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb526de9e9a759ff10b0913b39b5e0ac7836593" target='_blank'>
              LDB1-dependent enhancer networks drive high-risk transcriptional programs in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Rahul Bhansali, Juan Long, Petri Polonen, Shuo Zhang, Mariah C. Antopia, Siqing Wang, Nicholas G. Aboreden, Ahnaf Tausif, Zhuangzhuang Geng, Cheryl Keller, B. Giardine, Ross Hardison, C. Mullighan, Gerd Blobel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="


 TET2 mutations (TET2MT) are among the most frequent somatic lesions in myeloid malignancies, found at ~30% in MDS/AML and over 66% in CMML. TET2MT is also common in clonal hematopoiesis of indeterminate potential (CHIP), a pre-malignant condition that increases the risk of myeloid neoplasms (MN) and other diseases including treatment response in certain solid tumors. However, TET2MT alone is insufficient for malignant transformation, as they are frequently observed in CHIP without apparent disease. Thus, strongly suggesting the presence of additional mechanisms of lineage perturbation and evolution to MN. Chromodomain Helicase DNA-binding Protein 2 (CHD2) is an ATP-dependent DNA helicase that modulates gene expression via nucleosome repositioning. Despite its essential role in transcriptional fidelity and genomic integrity, CHD2 function remains poorly characterized in hematopoiesis and leukemogenesis. Here we report that CHD2 directly interacts with TET2 and co-occupies chromatin regulatory sites to maintain epigenetic homeostatic hematopoiesis. In the absence of TET2, CHD2 is co-opted to sustain oncogenic transcriptional programs that skew differentiation to myeloid precursors and prevent differentiation seen in TET2MT AML.
 Methods. CRISPR-Cas9 was used to generate clonal isogenic TET2 knockout (TET2KO) and TET2WT cells derived from K562 and THP1. Endogenous immunoprecipitation (IP) of CHD2 and TET2 was performed to purify associated protein complexes, followed by analysis and identification of the CHD2-TET2 interactome with high performance liquid chromatography coupled with tandem mass spectrometry (LCMS/MS). To investigate functional consequences of TET2 loss on CHD2 activity, we conducted integrative multi-omics analysis combining proteomics, publicly available chromatin immunoprecipitation sequencing (ChIP-seq), and RNA sequencing (RNA-seq). Genome-wide co-occupancy of CHD2 and TET2 was evaluated through comparative analysis of published ChIP-seq datasets.
 Results. Reciprocal IP of TET2/CHD2 in K562 and CMK cells revealed robust interaction between CHD2-TET2. Global proteomic comparisons using LCMS/MS revealed significant overlap between CHD2 and TET2 interactomes. 268 proteins were unique to CHD2 IPs from TET2WT that reduced to 106 in isogeneic TET2KO cells with CHD2 retaining a conserved overlapping interaction network in both cases. Notably, CHD2 IPs were enriched for components of the CoREST complex (RCOR1, RCOR3, GSE1, KDM1A), suggesting a potential role in transcriptional repression. In contrast, SWI/SNF components were particularly enriched in TET2 IP, highlighting TET2's role as a critical hub for chromatin remodeling. TET2 interactome was markedly broader, comprising over 3,000 proteins, including key regulators of polycomb repressive complexes (PRC1/2) and lineage-defining transcription factors like GATA1, GATA2, RUNX1 and STAT5A/B.
 Our analysis of ChIP-seq data showed 95% overlap between CHD2- and TET2-bound genes. RNAseq analysis in matched isogeneic cells revealed that these co-targeted genes are differentially expressed in the TET2KO cells. The oncogenic pathways were enriched among genes upregulated in TET2KO, while RNA processing and chromatin homeostasis genes were down regulated compared to TET2WT. ChIP-seq track analysis showed CHD2 and TET2 binding at promoter-proximal nucleosome-depleted regions, supporting a cooperative regulatory mechanism. These findings suggest that in TET2-proficient cells, CHD2 is directed to key regulatory elements to maintain transcriptional fidelity that is essential for homoeostasis hematopoiesis. In TET2-deficient cells, CHD2 is hijacked to sustain aberrant transcription that may promote myeloid skewing and leukemogenesis.
 Conclusions. For the first time, here we define a previously unrecognized CHD2-TET2 interactome and its potential functional role in the hematopoiesis. This, in part, may provide a probable mechanism of myeloid skewing in TET2MT HSPCs. These findings may provide rational for the dependency of TET2MT clones on CHD2 for malignant evolution. By integrating proteomic, genomic and functional analyses, we elucidate the role of CHD2 in transcriptional control that define lineage fate in the absence of TET2. In addition, we address a significant knowledge gap in CHD2 biology that may form the basis for novel therapeutic approaches targeting epigenetic vulnerabilities in TET2MT associated disorders.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded1853073e3baae3c88026feb94596db68dae6" target='_blank'>
              Chromodomain helicase DNA binding protein CHD2 interacts with TET2 and regulates lineage defining transcription in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Songa Bae, Xiaorong Gu, Mamta Sumi, Dongxu Jiang, Daniel J Vail, Yahan Zhang, Kacey O'Donnell, Yogenthiran Saunthararajah, Jaroslaw P. Maciejewski, B. Jha
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa4f5ddeab086fa66a7de2d9fed3fa9999b778c" target='_blank'>
              Cryptic multicellularity in wild yeast brings selective advantages under stress
              </a>
            </td>
          <td>
            W. K. Sexton, K. Schmidt, Q. Dickinson, J. Childress, F. Rosenzweig, E. Kroll
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Lamin A/C (LMNA), a key component of the nuclear envelope, is essential for maintaining nuclear integrity and genome organization [W. Xie et al., Curr. Biol. 26, 2651-2658 (2016)]. While LMNA dysregulation has been implicated in genomic instability across cancer and aging, the underlying mechanisms remain poorly understood [S. Graziano et al., Nucleus 9, 258-275 (2018)]. Here, we define a mechanistic role for LMNA in preserving genome stability in small-cell lung cancer (SCLC), a malignancy marked by extreme genomic instability [N. Takahashi et al., Cancer Res. Commun. 2, 503-517 (2022)]. LMNA depletion promotes R-loop accumulation, transcription-replication conflicts, replication stress, DNA breaks, and micronuclei formation. Mechanistically, LMNA deficiency disrupts nuclear pore complex organization, specifically reducing phenylalanine-glycine (FG)-nucleoporin incorporation, resulting in impaired RNA export and nuclear retention of RNA. LMNA expression is repressed by EZH2 and reexpressed during SCLC differentiation from neuroendocrine (NE) to non-NE states, and low LMNA levels correlate with poor clinical outcomes. These findings establish LMNA as a key regulator of nuclear transport and genome integrity, linking nuclear architecture to SCLC progression and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1b936a6a58b416fb0e62e291846a3a4d6fb1c3" target='_blank'>
              Lamin A/C loss promotes R-loop-mediated genomic instability and poor survival in small-cell lung cancer.
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Ravi P Shuklah, Robin Sebastian, Umeshkumar M Vekariya, Shahbaz Ahmed, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yashuhiro Arakawa, Ukhyun Jo, Tomasz Skorski, Beverly A. Teicher, Roshan Shreshta, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dysregulated transcription is a defining hallmark of cancer. Recently, novel chemically induced proximity approaches have enabled the rewiring of transcriptional machinery to drive expression of pro-apoptotic genes using bivalent small molecules. In this work, we demonstrate that this strategy is amenable to relocalizing DNA bound transcriptional machinery, such as fusion transcription factors that commonly drive pediatric malignancies. Targeting fusion transcription factors, such as EWSR1::FLI1 in Ewing sarcoma, with these bivalent compounds may open new therapeutic avenues. Here, we develop a small molecule, EB-TCIP, that recruits FKBP12F36V-tagged EWSR1::FLI1 to DNA sites bound by the transcriptional regulator BCL6, leading to rapid chromatin remodeling and expression of BCL6 target genes. This proof-of-concept study demonstrates that DNA binding proteins with pioneering transcription factor activity, such as EWSR1::FLI1, can be relocalized on chromatin to induce expression of repressed genes. Insights herein will guide the development of future bivalent molecules that rewire DNA binding transcriptional machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f7b6214e9ef05ce0f8a706c26c1ffab5a52705" target='_blank'>
              Rewiring the Fusion Oncoprotein EWSR1::FLI1 in Ewing Sarcoma with Bivalent Small Molecules.
              </a>
            </td>
          <td>
            Michael J Bond, Ryan P. Golden, G. Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, Hannah M Jones, Brendan G. Dwyer, N. Gray, K. Stegmaier
          </td>
          <td>2025-11-27</td>
          <td>Journal of the American Chemical Society</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980482a475765af78491bb69bee337aba4e9cbe1" target='_blank'>
              Discovering the complete enhancer map of human herpesviruses using a natural language processing model.
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Deepanway Ghosal, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-12-02</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Transposable elements (TEs) make up half of the human and mouse genome but remain understudied. Evidence is growing that TEs play important roles in immunity as a subset of TE enhancers regulate innate immunity genes. However, the cis-regulatory contribution of TEs across immune cell development is not well understood. Mining more than 300 chromatin accessibility datasets (ATAC-seq) across mouse hematopoietic cell lineages, we identified two subfamilies of rodent-specific retroviral LTRs, ORR1E and ORR1D2 (ODE), which are enriched in accessible chromatin across all immune cell types. Unsupervised clustering show that ∼2,900 ODEs stratify by cell type-specific accessibility. Each ODE cluster is enriched for a unique combination of sequence motifs for cell lineage-specifying TFs, such as PU.1, IRF8, and RORγ, which we validated with ChIP-seq. By correlating ODE accessibility with gene expression (RNA-seq), we predicted hundreds of ODE-gene interactions. We generated Micro-C (DNA-DNA contact) in CD8+ T cells and validated ∼120 ODE-gene interactions. Predicted target genes are more expressed in mouse than in human CD8+ T cells. Lastly, accessible ODEs have higher sequence conservation across rodents than other ODEs. Together, these data suggest that despite their common origin, ODE LTRs have subfunctionalized into cell type-specific enhancers across mouse hematopoiesis. We propose that ODE enhancers were co-opted during rodent evolution to reinforce cell type identity.



 F31AI183775



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c25d5af8adc02b84a26acb002fe252a06d958df" target='_blank'>
              Functionalization of transposable element-derived enhancers across mouse hematopoiesis 3835
              </a>
            </td>
          <td>
            Jason D. Chobirko, Elizabeth A. Fogarty, Cedric Feschotte, Andrew Grimson
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/930d991209f6f71494e4d0ad31dff292006463ca" target='_blank'>
              A population genetics model explaining overdispersion in active transposable elements
              </a>
            </td>
          <td>
            Adekanmi Daniel Omole, Peter Czuppon
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Eukaryotic DNA has been covalently modified by DNA methylation and folded into a three-dimensional conformation in the nucleus. While the functions of DNA methylation and chromosome organization have been widely discussed, respectively, the interplay between DNA methylation and chromosome organization remains less clear and needs to be further explained. In this review, we first discuss the cross-talk between DNA methylation and chromosome conformation, highlighting the complexity and importance of DNA methylation on chromosome organization. We also summarize the current methodologies that capture DNA methylation and chromosome organization individually or simultaneously in bulk and single cells. These mechanistic and methodological advancements facilitate broad interest in unveiling the interplay between DNA methylation and chromosome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1272377f23956ec52130bb31428bff7433f3f0" target='_blank'>
              Unraveling the interplay of DNA methylation and chromosome organization.
              </a>
            </td>
          <td>
            Yuhe Pei, Guoqiang Li
          </td>
          <td>2025-11-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K. Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>34</td>
        </tr>

        <tr id="


 Introduction: Ribosomal DNA (rDNA) encodes ribosomal RNA (rRNA), which undergoes processing to become ribosomes. Although rDNA dysregulation has been observed in various pathologies, comprehensive epigenetic profiling in cancer has been limited due to its repetitive sequence. To overcome this challenge, a customized genome has been developed to map human and mouse rDNA using high-throughput sequencing data. Using this genome and more than 2,200 ChIP-seq datasets, Antony et al performed a systematic analysis to identify the transcription factors, e.g., C/EBP-A, that support rRNA production in hematopoietic cells. Our previous publications suggested that DNA cytosine modifications influence genomic binding of C/EBP. Accumulated studies, including our own, have reported aberrant DNA methylation and hydroxymethylation in myeloid malignancies. Yet, it remains unclear whether the leukemia-associated DNA methylation dysregulation extends to genomic regions encoding rRNA (rDNA), and how these epigenetic changes in rDNA affect downstream targets.
 Methods: To address these critical knowledge gaps, we employed innovative rDNA assembly methods, along with a DNA methylation analysis pipeline, to evaluate the whole-genome bisulfite sequencing (WGBS) data from 136 cancer patients covering 9 cancer types, aiming to uncover DNA methylation changes in rDNA in cancer. To further investigate unique patterns observed in hematological malignancies, we expanded our analysis to a dataset containing DNA methylation profiles from 292 AML patients. Additional AML datasets were analyzed to validate our initial findings with integrative methylation analysis methods. Furthermore, we analyzed published nanopore long-read data to independently confirm the methylation status of rDNA.
 Results: Our pan-cancer analysis revealed several patterns of rDNA methylation across cancer types, often at odds with non-rDNA methylation patterns. Intriguingly, we discovered that hematological malignancies exhibited distinct DNA methylation signatures in rDNA, with higher DNA methylation in the rDNA-coding region compared with other cancer types. This finding was validated in multiple independent datasets totaling 484 AML samples. Additionally, long-read nanopore sequencing verified these methylation patterns across the entire rDNA tandem array. Further characterization of this hypermethylation in AML patient samples revealed CpG loci that show highly conserved changes. A significant correlation was observed between hypermethylation at these loci and decreased time-to-relapse, suggesting that increased rDNA coding region methylation may serve as a biomarker. Interestingly, along with prognostic significance related to time-to-relapse, we revealed distinct patterns of rDNA methylation in patients treated with hypomethylating agents (HMA). To study potential mechanisms, we first validated published findings that many transcription factors exhibit consistent binding patterns on rDNA. To elucidate the connection between DNA methylation and transcription factor binding within rDNA, we first performed a motif analysis of the rDNA sequence, revealing 4000+ canonical binding motifs. We then used our large AML datasets to identify motifs with differential methylation between healthy bone marrow and AML patients, both at diagnosis and relapse. Across both datasets, we identified several potential transcription factors, e.g., CEBP families, for downstream analysis. We will use multiple leukemia cell lines to validate the correlation between DNA methylation and TF binding at rDNA. We will further use catalytically dead Cas9 (dCas9) mediated epigenome editing to probe the causal relationship to support our hypothesis that rDNA methylation can influence TF binding.
 Conclusions: Our data revealed distinct DNA methylation signatures in rDNA specific to myeloid leukemia. Using multiple published DNA methylation datasets from AML patients, we identified unique DNA methylation changes in rDNA, highlighting its potential diagnostic and prognostic value. At the molecular level, we observed differential DNA methylation levels within several TF binding motifs, which might alter TF binding and impact their transcriptional activity in rDNA, ultimately contributing to aberrant ribosomal biogenesis. Overall, our study provides a comprehensive analysis of previously neglected DNA methylation changes in rDNA in AML, uncovering novel epigenetic mechanisms with potential clinical relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba5a2d1360b7125af2d5a2084348209fa94e181" target='_blank'>
              Hypermethylation of the ribosomal DNA coding sequence and disordered transcription factor binding as hallmarks of acute myeloid leukemia
              </a>
            </td>
          <td>
            James Wengler, Y. Huang, Leng Han, Lei Guo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/267276dd26cb529d07f65beba245c3f404b9cf18" target='_blank'>
              Dysregulated mitochondrial homeostasis and DNA repair in the progression from colon adenoma to cancer.
              </a>
            </td>
          <td>
            Natalie Danesova, Josef Horak, Anna Valickova, Adrian Gil-Korilis, Jorge Ergui-Arbizu, Richard Palek, Jan Bruha, Miroslav Levy, Pavel Škrobánek, Jan Kral, J. Jungwirth, Jiří Neužil, Veronika Vymetálková, P. Vodicka, Sona Vodenkova, Kristyna Tomasova
          </td>
          <td>2025-11-22</td>
          <td>Molecular medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Conjugative plasmids are a major driving force for the dissemination of antimicrobial resistance. During conjugation, plasmid DNA is transferred from the donor cell as a single-stranded (ss) linear molecule. Recent research has highlighted intriguing DNA reactions at the early stages of conjugation that are important for plasmid establishment in the recipient cell, including prompt expression of antidefense genes. However, genomics-based approaches to investigate plasmid establishment have been challenging because the identical fully established plasmids in donor cells mask transconjugant-specific signals. To overcome this limitation, we developed a new method, ED-TA, which exploits a donor mutant hypersensitive to hypoosmotic shock. ED-TA allows unprecedently quick and efficient Elimination of Donor population for Transconjugant Analysis. RNA-seq analyses revealed highly selective and robust induction of plasmid genes during the early stages of conjugation. We experimentally identified transcription start sites of six possible operons that are matched with ssDNA promoter predictions. We also showed that transconjugant gene expression is altered in recipients with perturbed plasmid establishment. As the ED-TA method is straightforward and broadly applicable, it will further our understanding of plasmid establishment processes in the new host cell, including not only gene expression but also ss to double-stranded DNA conversion and plasmid circularization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d989a01b8881b8d61c5bd003f1b710d0d26bbd5" target='_blank'>
              Selective elimination of donor bacteria enables global profiling of plasmid gene expression at early stages of conjugation
              </a>
            </td>
          <td>
            Meng Wen, Nathan Fraikin, Emma Mettouchi, Christian Lesterlin, Elena Espinosa, Y. Yamaichi
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures comprising an RNA/DNA hybrid and a displaced single-stranded DNA. While transient R-loop formation is essential for various physiological processes, their persistent accumulation leads to genomic instability. Cancer cells exhibit elevated R-loop levels due to hypertranscription, replication stress, and impaired DNA repair pathways. In this review, we provide a comprehensive overview of the molecular machinery that resolves R-loops, including chromatin remodelers, transcriptional regulators, nucleases, and helicases. We also highlight the emerging roles of long noncoding RNAs (lncRNAs) in modulating R-loop dynamics and explore how these RNA-based mechanisms cooperate with canonical resolution pathways. Finally, we explore the potential of targeting R-loop regulatory networks as a novel therapeutic strategy in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7591e333b5ca8a5feff5d45b0c90d90e304373" target='_blank'>
              Regulation of R-Loop Dynamics by Proteins and Long Noncoding RNAs: An Emerging Paradigm for Cancer Treatment.
              </a>
            </td>
          <td>
            Miho M. Suzuki, Keiko Shinjo, Tatsunori Nishimura, Yutaka Kondo
          </td>
          <td>2025-11-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant childhood cerebellar tumor comprising molecularly and clinically distinct subgroups. Groups 3 and 4-MB originate from overlapping progenitor pools of the developing rhombic lip when neuronal differentiation hierarchies are subverted through somatic alterations. Mutations suspected to deregulate the activity of chromatin-modifying genes are pervasive in Group 3/4-MB. However, molecular consequences of these alterations remain largely undefined. We characterized the chromatin landscape of fifty-two primary MBs using chromatin immunoprecipitation followed by sequencing (ChIP-seq) for six histone modifications and performed multi-modal integration with sample-matched somatic lesions, genome-wide DNA methylation, and transcriptomic profiles. Analysis of differential chromatin states across MB subgroups revealed significant enrichment of the bivalent enhancer state (EnhBiv; marked by coincident H3K4me1 and H3K27me3) in Group 3/4-MB overlapping neurodevelopmental genes. Integrative bioinformatics coupled with CUT&RUN analysis revealed significant enrichment of KDM2B, a histone lysine demethylase overexpressed in Group 4-MB, in gene promoters marked by the EnhBiv state. CRISPR gene targeting or targeted protein degradation of KDM2B selectively suppressed growth of Group 3 and Group 4-MB models in vitro and in vivo, which was dependent on KDM2B DNA-binding domain but not demethylase activity. Acute and chronic KDM2B degradation impaired EZH2 recruitment through direct physical interactions and indirect KDM2B-mediated PRC1 activity, reducing H3K27me3 deposition and chromatin bivalency, while profoundly derepressing PRC2 targets involved in neuronal differentiation. Our comprehensive characterization of the MB chromatin landscape in a large cohort of primary tumors provides unprecedented insights into the epigenetic basis of MB subgroups, implicating novel mechanisms of tumorigenesis, and disclosing a unique subgroup-specific dependency conferred by KDM2B-mediated Polycomb activity that warrants consideration as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff433ac7a60b68bf5a2b5c1d8d57a287494ba1b0" target='_blank'>
              EPCO-22. DIVERGENT MEDULLOBLASTOMA CHROMATIN STATES DISCLOSE KDM2B AS A SELECTIVE DEPENDENCY
              </a>
            </td>
          <td>
            Ran Tao, Serap Erkek-Ozhan, Beisi Xu, Yurika Matsui, Priya Mittal, Kyle S Smith, Yiran Li, Qingsong Gao, Emily Darrow, Ruijie Xu, Nadhir Djekidel, Richa Bajpai, Rahul Kumar, Colleen Reilly, T. Soliman, Jennifer L. Hadley, Melissa Batts, Natarajan Bhanu, B. Gudenas, Leena Paul, Hong Lin, K. Lowe, L. Kutscher, Benjamin Garcia, S. Pruett-Miller, Xin Zhou, Brent A. Orr, G. Robinson, Gang Wu, Jan O. Korbel, Jamy C Peng, Stefan M. Pfister, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Meningiomas are the most common primary tumors of the central nervous system and are typically treated with surgery or radiation, as targeted therapies remain limited. Despite extensive study, seventeen percent of meningiomas lack known genetic drivers. Our analysis of meningiomas without driver mutations or major chromosomal alterations identifies a subset with recurrent genomic rearrangements involving the FOS and FOSB genes. These tumors exhibit elevated FOS/FOSB protein levels and retain meningothelial lineage. Here we show that FOS/FOSB fusion-positive meningiomas represent a distinct molecular subgroup, defined by unique gene expression patterns, including activation of AP-1 target genes and signatures resembling preadipocyte-like and mast cell–associated profiles. Clinically, these tumors display low-grade behavior and DNA methylation profiles consistent with benign subtypes. Our findings identify a meningioma subgroup with distinct genetic, transcriptomic, and clinical features, expanding the molecular classification of meningiomas and opening new avenues for targeted treatment strategies. Approximately 17% of meningiomas remain genomically uncharacterized. Here, the authors analyze 105 meningiomas without known driver mutations or significant copy number alterations and identify a subgroup of meningiomas, defined by FOS/FOSB gene fusions with distinctive transcriptomic and histopathological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/848d67ce829255f08e0caafaf14bc914e2cca146" target='_blank'>
              Subgroup of meningiomas involving FOS and FOSB gene fusions
              </a>
            </td>
          <td>
            K. Yalcin, Hasan Alanya, B. Gultekin, Diego Samper Figuera, Tanyeri Barak, D. Miyagishima, Mark W. Youngblood, Mateo Gomez Hjerthen, Amos Brooks, Natalia Samuel Lopez, Joseph O’Brien, M. M. Chavez, Alper Dincer, Hao Wu, S. Omay, Katsuhito Yasuno, K. Bilguvar, A. Ercan-Sencicek, K. Mishra-Gorur, D. McGuone, Jennifer Moliterno, Murat Gunel, E. Erson-Omay
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive primary brain tumor, has a dismal prognosis largely due to therapy-resistant stem-like cells that drive recurrence. While N6-methyladenosine modifications in GBM are well-studied, the role of 5-methylcytosine (m5C) modifications specifically in circular RNAs (circRNAs) remains poorly understood. Bioinformatic analysis, qRT-PCR and Western blot assays were used to investigate the expression of NSUN7, circNTRK2, YBX3, STK31, IKZF1 in GBM tissues and cell lines. m5C dot blot assays, m5C-bisulfite sequence assays, Sanger’s sequencing, RNA pull down and RIP assays, in vitro kinase assays, chromatin immunoprecipitation and luciferase reporter gene assays are used to clarify the interaction between factors above. Colony formation assays, sphere formation assays, and stemness marker (OCT4, DCLK1) analysis were utilized to assess the impact of the factors above on GBM stemness. Subcutaneous heterotopic and orthotopic xenograft are utilized to demonstrate the function of YBX3/circNTRK2/STK31/IKZF1 axis in GBM in vivo. NSUN7 and YBX3 are both upregulated in GBM tissues and cell lines. NSUN7 catalyzes m5C modification of circNTRK2. YBX3 decreasing circNTRK2 stability and function via binding m5C of circNTRK2. CircNTRK2 binds to and activates STK31, leading to phosphorylation of IKZF1 at S63, which decreases the O-GlcNAc levels and transcriptional activity of IKZF1, thereby promoting the stem cell characteristics of GBM cells. This study unveils that NSUN7-mediated m5C modification of circNTRK2 fuels GBM stemness via circNTRK2-dependent STK31 activation, thus identifying potential biomarkers for targeted molecular therapy of GBM and providing novel therapeutic targets for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6dd5f2b28a26b1d6df7d811235c13cb3ffa9c9" target='_blank'>
              NSUN7-mediated m5C modification of circNTRK2 regulates stemness properties of glioblastoma cells by activating STK31
              </a>
            </td>
          <td>
            Yubo Zhao, Mengyang Zhang, Jiachun Ma, Hongye Mu, Kaishu Li, Fengjin Li, Qian Peng, Ling Qi
          </td>
          <td>2025-11-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The pathogenesis of koala retrovirus (KoRV) has been explored in various contexts, yet its role in tumorigenesis remains incompletely understood. Unlike acute transforming retroviruses, KoRV lacks a viral oncogene but may contribute to oncogenesis via indirect mechanisms. However, the relationship between KoRV and telomere length, as a potential indicator of telomerase activity, has not been examined. This study investigates the effect of KoRV infection on telomere length in 47 samples from Southern Australian koalas in a novel telomere length quantification method. Telomere lengths of 30 KoRV-negative samples were compared to those of 17 KoRV-positive samples using the Absolute Human Telomere Length Quantification qPCR kit (ScienCell Research Laboratories, California, USA). The telomere length in KoRV-infected WBCs was significantly longer than the uninfected ones (t = −2.059, p-value = 0.045). In line with this, telomere length correlated positively with proviral load (r = 0.421, p-value = 0.003), further linking viral burden to telomere elongation. Furthermore, the effect of age on telomere length differed by infection status (β = −5329.7, p-value = 0.0038); KoRV-positive individuals exhibited longer telomeres at a younger age but experienced more rapid telomere attrition over time compared to KoRV-negative individuals. These results suggest KoRV promotes telomerase elongation ability and modulates age-related telomere dynamics, potentially contributing to subsequent cellular immortality and oncogenesis. These pathways may overlap with other retroviruses, where telomerase dysregulation contributes to their oncogenic potential. This study provides new insights into KoRV pathogenesis and DNA quantification methodology, which could be valuable for future research by identifying predictive markers for tumour progression and potential therapeutic targets in affected koalas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f125ee840ff1c5667ff63226a6db37a09c90835b" target='_blank'>
              Infection-Induced Telomere Length Variation: Insights into Pathogenesis of Koala Retrovirus
              </a>
            </td>
          <td>
            Hiu Ming Cheung, Sze Wing Jamie Lin, Hanh Thi Hong Nguyen, T. Stephenson, N. Speight, F. Hemmatzadeh
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="RhoGDI2 is a RhoGTPase regulator that has roles in cytoskeleton organization and cell survival, amongst others. It is differentially expressed in many cell types and tissues, including several human cancers, where its expression has been correlated with either good or bad prognosis. To identify the underlying mechanisms, we knocked down its expression in human cancer cell lines. We observed that repression of RhoGDI2 expression, but not that of the closely related RhoGDI1, significantly reduces their proliferation rate. In parallel, RhoGDI2 suppression induces supernumerary centrosomes and inhibits ciliogenesis. As RhoGDIs are regulators of GTPases, we checked whether key RhoGTPases are involved in these effects. We found that silencing RhoA partially rescued the induction of supernumerary centrosomes and ciliary defects observed upon RhoGDI2 silencing. It was previously shown that RhoGDI2 is strongly expressed in immune cells and that there are striking similarities between primary cilia and immune synapses. Based on this knowledge, we silenced RhoGDI2 in NK cells and could demonstrate that this strongly affects their immune synapse-related cancer cell-killing activity. Altogether, these data suggest novel roles for RhoGDI2 in centrosome functions in human cancer and immune synapses in immune cells, which provides an explanation for its reported dual role in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c0ac8c4a86b7bca62ea906513e7218a435f267" target='_blank'>
              Reduced RhoGDI2 Expression Disrupts Centrosome Functions and Promotes Mitotic Errors
              </a>
            </td>
          <td>
            M. Tripathi, N. Garbacki, J. Willems, G. Cobraiville, Marianne Fillet, Alain Colige, Christophe F. Deroanne
          </td>
          <td>2025-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Introduction Regulation of gene expression by histone modification is critical in drug resistance and progression. KMT2A-rearranged (KMT2A-r) AML can activate aberrant leukemogenic HOXA and MEIS1 gene signatures through its interaction with menin. KMT2A is part of COMPASS complex along with ASH2L, WDR5, RBBP5, and DPY30, sometimes called 'WRAD’ complex, and functions as writers and readers to catalyze H3K4 methylation and drives transcription of leukemogenic genes such as HOXA9 and MEIS1. ASH2L is part of the core complex and inherently enhances the enzymatic activity amongst the other scaffolding proteins. We hypothesized that ASH2L (a histone methyltransferase writer) and WDR5 (a chromatin reader) constitute an escape, non-canonical resistance mechanism that rewires the leukemogenic transcription program even in the presence of menin inhibitor KMT2A-r AML cells.
 Methods To this note, we aimed to establish an in-vitro resistant model in KMT2Ar leukemia cell lines (MV4-11 and MOLM-13) by chronically exposing them to sequentially escalating, sub-therapeutic doses of a menin inhibitor (ziftomenib, Selleckchem, cat.no:E1290). COMPASS complex gene and downstream transcripts and protein expressions were detected by quantitative RT-qPCR and western blotting. SiRNA-mediated knockdown (kd) of ASH2L and WDR5wasperformed to investigate their functional role in resistance, and western blotting was used to confirm protein depletion and assess downstream effects on MLL1/COMPASS components. An ATP-based cell proliferation assay was performed to assess cell viability and growth inhibition following SiRNA KD and by using WDR5 inhibitor (OICR-9429, Selleckchem, cat.no:S7833) to pharmacologically target the complex. ChIP-qPCR was performed to determine whether resistance is mediated by altered chromatin occupancy by alternate COMPASS components to the HOXA9 promoter in the resistant models.
 Results Incremental exposure of MV4-11 cells to a menin inhibitor (ziftomenib) over six months resulted in a progressive increase in IC₅₀ from baseline, with the cells maintaining proliferative capacity even at continued exposure of 60 nM of the inhibitor. Cell line authentication using short-tandem-repeats verified that the resistant subline was genetically identical to the parental MV4-11 cell line, and samples were processed for sequencing of the MEN1-binding pocket to screen for menin inhibitor resistance-associated mutations. RT-qPCR and immunobloting exposed clear transcriptional and protein level differences in the resistant and sensitive AML cells. Sensitive cells retained low HOXA10 and MEIS1/2 expression whereas resistant cells exhibited reduced menin levels but increased ASH2L, WDR5, and HOXA9 expression, suggesting that HOXA9 upregulation may serve as a potential escape mechanism driving resistance. SiRNA knockdown of ASH2L and WDR5 in resistant AML cells led to a marked decrease in leukemogenic transcripts, including HOXA9 and MEIS1, as confirmed by immunoblotting. In parallel, we generated dose-response curves for WDR5 dependency in sensitive (IC50 for ziftomenib, 14nm) and menin-inhibitor-resistant MV4-11 cells (IC50 for ziftomenib, 65nm), and notably, treatment with a WDR5 inhibitor restored the IC₅₀ of resistant cells to levels comparable to the sensitive parental line (IC50 for WDR5 inhibitor 12nm). ChIP-qPCR with ASH2L immunoprecipitation in menin-inhibitor-resistant cell line showed markedly increased ASH2L occupancy at HOXA9 promoter, suggesting that higher occupancy may contribute to resistance to menin inhibition. Ongoing work examines selective ASH2L inhibition—alone or combined with menin and WDR5 blockade—to overcome resistance in KMT2A-r AML. Genome-wide ChIP-seq to map ASH2L and WDR5 redistribution at promoters/enhancers, extending our HOXA9 data. TurboID proximity labeling to compare protein-interaction networks in sensitive versus resistant cells to uncover COMPASS partners. CRISPR knockdowns and patient cells will be tested in xenografts for therapeutic response.
 Conclusion ASH2L and WDR5 are critical mediators of COMPASS-dependent therapy resistance in KMT2A-r AML. Targeting this complex, particularly ASH2L and WDR5, represents a promising strategy to possibly reverse resistance and improve therapeutic outcomes in resistant KMT2A-r AML.Our preliminary data indicate other components of the COMPASS complex to possibly cause resistance in KMT2A-r AML, and further validations are in progress.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d0d6261eed83d4bc7612c32cd10a8f9626c2b5" target='_blank'>
              Targeting the compass complex to overcome resistance in KMT2A-rearranged Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sandhya Dhiman, Vikramjit Dhillon, J. Aguilar, Julie Boerner, Yang Shi, Jay Yang, S. Balasubramanian
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f8abb290a62c5b3aa53581143c941a3ce167f6" target='_blank'>
              Lenacapavir-induced capsid damage uncovers HIV-1 genomes emanating from nuclear speckles.
              </a>
            </td>
          <td>
            Thorsten G. Müller, Severina Klaus, V. Zila, Bojana Lucic, C. Penzo, Svenja L. Nopper, Gonen Golani, Maria Anders-Össwein, Vera Sonntag-Buck, A. Heuser, U. Schwarz, V. Laketa, M. Lusic, Barbara Müller, H. Kräusslich
          </td>
          <td>2025-12-01</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a524bd974d6e1f504ec3318cf5435988050a086" target='_blank'>
              XIST Drives X-Chromosome Inactivation and Safeguards Female Extraembryonic Cells in Humans
              </a>
            </td>
          <td>
            Amitesh Panda, Léo Carrillo, B. Balaton, Jeanne Brouillet, Solomon Nshemereirwe, Jarne Bonroy, Charbel Alfeghaly, Romina Facchinello, Sherif Khodeer, N. Peredo, R. Boers, G. Castel, Charlie London, Emmanuel Cazottes, Madeleine Moscatelli, Raissa Songwa Tchinda, Thi Xuan Ai Pham, San Kit To, Ryan N. Allsop, Yang Wang, Desislava P. Staneva, P. Rugg-Gunn, Kathy K. Niakan, J. Gribnau, Jean-François Ouimette, C. Rougeulle, Vincent Pasque
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Agrobacterium-mediated transformation (AMT) is the primary means of genetic engineering in plants and many fungi, but the factors that control transformation outcomes—efficiency, transgene insertion number, and transgene integrity—remain poorly characterized. Although transformation outcomes dictate an event’s potential utility in both industrial and academic contexts, AMT remains largely unoptimized for these metrics. Here, we systematically analyze the impact of the transgene-harboring binary vector on transformation outcomes across plant and fungal species. Through a comparison of different plasmid origin of replication (ORI) families and engineered copy number variants, our results reveal that the ORI family—not plasmid copy number—dictates T-DNA insertion number, backbone inclusion, and transformation efficiency, while plasmid copy number tuning alters efficiency without changing ORI family-specific signatures. Independent of plasmid copy number across kingdoms, the most widely used pVS1 ORI-based vectors (e.g. pCambia) result in significantly more insertions per transformant and high levels of transgene silencing compared to the less-utilized pSa ORI family, which enriches for more uniform single insertion events. Furthermore, we demonstrate that ORI-dependent transformation outcomes in yeast predictably reflect those in Arabidopsis. Together, these results lay the foundation for future binary vector design aimed at achieving more predictable, controllable, and optimized transformation outcomes across diverse eukaryotic hosts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c64b7e7100ce9444c4a3bdcd3ae4342c59391d5" target='_blank'>
              Binary vector origin predictably determines Agrobacterium-mediated transformation outcome across eukaryotic kingdoms
              </a>
            </td>
          <td>
            Matthew J. Szarzanowicz, Michael Busche, Ziyu Dai, Nathan Lapp, Gina M. Geiselman, Jiayuan Jia, Lucas M. Waldburger, Yan Liang, Guoliang Yuan, Rylan D. Duong, Kyle R. Pomraning, N. Hillson, Jacob O. Brunkard, Blake A. Simmons, J. Gladden, Joonhoon Kim, Mitchell G. Thompson, Patrick M. Shih
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Epstein-Barr Nuclear Antigen 1 (EBNA1) is essential for the episomal maintenance and DNA replication of Epstein-Barr virus (EBV) in latently infected cells and acts through binding to oriP. The minimal replicative unit of oriP (½DS) contains four EBNA1 binding sites flanked by single telomeric nonamers that recruit shelterin proteins TRF2 and Rap1, but the structural basis for host-factor engagement is not known. Here, we integrate cryo-electron microscopy, zero-length cross-linking mass spectrometry, Alphafold3 modeling, and biochemical binding assays to define the complex formed by EBNA1-TRF2-Rap1 assembly on the ½DS. We find that a highly dynamic complex is formed, with the TRF2 homodimerization domain (TRFH) flexibly interacting with EBNA1 on the surface opposite the DNA-binding region, where there is a large acidic patch in EBNA1 that is unique amongst the herpesvirus episome maintenance proteins. Mutagenesis of this acidic patch abolishes TRFH binding and oriP-dependent plasmid replication. These findings identify a previously uncharacterized acidic patch docking surface on EBNA1 essential for coordinating TRF2-RAP1 at oriP and provide new insights into both EBV and telomere DNA replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b29860c3f466ce401429e4b2e5b0859497b762cd" target='_blank'>
              Structural Basis for TRF2-RAP1 Recruitment by EBNA1 at the EBV origin of replication
              </a>
            </td>
          <td>
            Paul M. Lieberman, Samantha Sustek, Troy Messick, Jayaraju Dheekollu, Coltin Albitz, Christopher Chen, Anneliese Faustino, Hsin-Yao Tang, Hee Jong Kim, Kenji Murakami
          </td>
          <td>2025-10-29</td>
          <td>Research Square</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction.
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-19</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Background Since its inception, RNA sequencing has been pivotal in studying differential gene expression. Despite its extensive results in large-scale oncological studies, differential expression predominantly reflects a response to cancer. Therefore, we introduce differential allelic dispersion (AD) as a more effective measure. AD highlights consistent differences in expression between the 2 alleles of a gene that is, unlike cis-expression quantitative trait loci, independent of normal genetic variation. Such differences can, for example, arise from prevalent copy number alterations or epimutations occurring in the original cancer cell, which are mitotically expanded during cancer growth, making increased AD a marker for allele-specific dysregulation in early carcinogenesis. Findings We present the maelstRom R/C++ software package that enables (differential) AD analysis solely requiring large-scale RNA sequencing data. Using the The Cancer Genome Atlas renal clear cell carcinoma cohort as a case study, we successfully benchmark maelstRom’s AD modeling using known copy number alterations. We also detect increased AD for loci featuring normal random monoallelic expression, including the X chromosome, but demonstrate minimal interference with cancer-specific AD detection. Finally, we identify early dysregulated genes (e.g., FBP1, CCDC8, ECHS1, CLDN7) and pathways in renal cancer, often related to metabolism (e.g., pentose phosphate pathway). Strikingly, many of these genes are known causal contributors to renal carcinogenesis. Conclusions Differential AD clearly indicates early dysregulation in renal cancer, complementing basic differential expression analysis in cancer transcriptomics. AD is also relevant to study random monoallelic expression and may equally detect allele-specific (dys)regulation during early development or in noncancer diseases. maelstRom is available as an open-source software package at github.com/Biobix/maelstRom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e775646c46e5a03c9c6239d832825d07a63f0" target='_blank'>
              Population-level allelic dispersion modeling by maelstRom yields genome-wide maps of allele-specific dysregulation during early carcinogenesis
              </a>
            </td>
          <td>
            Cedric Stroobandt, Louis Coussement, T. Goovaerts, Femke De Graeve, Jeroen Galle, Wim Van Criekinge, Tim De Meyer
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gene redundancy, increasing gene dosage and functional diversity, remains understudied regarding its roles in evolution and clinical infections. Exploring 22,310 prokaryotic genomes with our custom pipeline, we found that redundant genes, though less frequent than in eukaryotes, are widespread and mainly linked to niche specialization. Evolutionary analyses delineated a propensity for gene redundancy expansion with increasing phylogenetic distance, with pathogens accumulating more redundancy than non-pathogens. Redundant genes are always co-duplicated with translation initiation signals and potentially preserve functionality. Time series examining 69 Acinetobacter baumannii isolates from multi-sites in severely infected patients, we identified redundant alcohol dehydrogenase genes and translation initiation signals, introduced by gene islands, as advantageous for invasive urinary tract infection throughout its within-patient development and cross-patient transmission. Mouse peritoneal infection models and plasmid transformation experiments confirmed that the redundancy of frmA, an alcohol dehydrogenase gene, is linked to both enhanced virulence and increased biofilm mass in Acinetobacter baumannii. Additional analysis of 898 Enterobacter cloacae complex genomes revealed that redundant metal ion resistance genes carried by mobile genetic elements may provide selective advantages. This study unveiled a moderate gene redundancy within prokaryotes, providing genetic insights into the adaptive evolution and clinical infection of pathogens. Gene redundancy in prokaryotes remains understudied. This analysis shows redundant genes are widespread, with pathogens accumulating more than non-pathogens, and are linked to virulence of Acinetobacter baumannii.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c88e22113fcc41089f5a7d685891799bf889442" target='_blank'>
              Deciphering gene redundancy in prokaryotic genomes provides evolutionary insights for pathogenicity and its roles in clinical infections
              </a>
            </td>
          <td>
            Peihong Wang, Qian Guo, Xiaoqing Jiang, Ping Lu, Mengru Chang, Longshu Yang, Mo Li, Chunhui Wang, Tingting Xiao, Yonghong Xiao, Huaiqiu Zhu
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Two rounds of whole-genome duplication (WGD) occurred about 500 million years ago and played a major role in the evolution of the vertebrate genomes. Human genes derived from WGD are called “ohnologs”. Ohnologs are involved in fundamental biological processes and significantly contributed to the complexity of the human gene regulatory network. Given the central role of miRNAs in gene regulation, we investigated the contribution of ohnolog miRNAs (ohno-miRNAs) to the human gene regulatory network. We identified intragenic ohno-miRNAs as having higher retention rates compared to intragenic Small Scale Duplicated (SSD) miRNAs. Ohno-miRNAs also show high sequence similarity, a stronger tendency to regulate common target genes and are typically more expressed compared to miRNAs unrelated to WGD events. Analyzing the role of ohno-miRNAs in the human gene regulatory network, we showed that ohno-miRNAs are statistically overrepresented in specific network motifs commonly associated with redundancy and complexity, highlighting their central role in gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c5d0c75cd069a45340fb97f72963794c6c56d0e" target='_blank'>
              Ohno-miRNAs: miRNA pairs derived from whole-genome duplication
              </a>
            </td>
          <td>
            Leonardo Agasso, Ivan Molineris, Michele Caselle
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1bc2b0013bf3f73eea36d3f8f1f1a3ceddf4e" target='_blank'>
              The evolutionary origin of host association and polycistronic transcription in trypanosomatids
              </a>
            </td>
          <td>
            Saurav Mallik, Meir Sylman, Dvir Dahary, Orna Dahan, S. Michaeli, G. Späth, Y. Pilpel
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract Background Medulloblastoma (MB) is the most common malignant pediatric brain tumor and exhibits high heterogeneity in genetic alterations and molecular subgroups. However, current treatment remains largely uniform, and more precise risk stratification is necessary to improve prognosis. Large-scale genomic studies have primarily focused on Western populations, while comprehensive data on Japanese patients remain limited due to low incidence. This study aimed to elucidate molecular genetic characteristics and prognostic factors in Japanese MB through integrative genomic and transcriptomic analyses. Methods Targeted DNA and RNA sequencing was performed on 200 cases from the Japanese Children’s Cancer Group (JCCG), and publicly available whole-genome and RNA sequencing data from 472 Western MB patients were analyzed for comparison. Results Known subgroup-specific driver mutations were identified in the Japanese cohort, and age-adjusted comparison with the Western cohort showed no significant differences in mutation frequencies, suggesting limited racial differences in MB pathogenesis. Deep sequencing detected subclonal mutations, including SMARCA4 in the WNT subgroup. Missense mutations in the MYC functional domain were observed in cases lacking MYC amplification; these are known drivers in hematological malignancies and may also play an oncogenic role in MB. MYC mutations were found in WNT and SHH subgroups, suggesting broader involvement beyond Group 3. Intron retention analysis revealed a non-canonical intronic ZMYM3 mutation in a Group 4 case, highlighting improved driver detection by integrating DNA and RNA data. Whole chromosomal aberrations correlated with better outcomes, supporting their role in risk classification. Conclusions This study identified novel MYC alterations and emphasized the utility of RNA-integrated analysis. Cohort expansion may uncover additional diversity and aid in precision medicine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394acb3917250c67e7f0d75d4a797dbd4bea5da9" target='_blank'>
              10102-GEN-2Integrated Genomic and Transcriptomic Analysis Reveals Molecular Diversity and Novel Drivers in Japanese Medulloblastoma
              </a>
            </td>
          <td>
            Tsubasa Miyauchi, Takuma Nakashima, Miho Kato, Junko Hirato, Takako Yoshioka, Hajime Okita, Y. Kanemura, Yuko Watanabe, Masahiro Toda, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efec579cbc7f789053fbde7e928477ced0b339a" target='_blank'>
              Ensilication preserves high-molecular weight native DNA for clinical long-read sequencing
              </a>
            </td>
          <td>
            Alexis Ferrasse, Rodrigo Mendez, J. Gorzynski, Chloe M Reuter, Jennefer N Carter, Undiagnosed Diseases Network, Jonathan A. Bernstein, Matthew T. Wheeler, James L. Banal, E. A. Ashley
          </td>
          <td>2025-11-03</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Increasing availability and utilisation of high-throughput sequencing techniques has resulted in a rapidly expanding range of uterine mesenchymal lesions harbouring recurrent and nonrecurrent gene rearrangements. Within the literature, 3 molecularly confirmed FOXO1-rearranged uterine corpus tumors have been reported, all representing alveolar rhabdomyosarcomas (ARMS). We report 5 cases of non-ARMS uterine mesenchymal tumors, in patients aged 36 to 71, harbouring novel FOXO1 rearrangements with different fusion partners (JRK, PIK3R4, MEIS1, and ATP7B); in the fifth case, FISH revealed a FOXO1 gene rearrangement with an unknown fusion partner. Although morphologically heterogenous, all 5 cases had a low-grade spindle cell component with 3 cases showing prominent myxoid stroma. Two cases were originally diagnosed as myxoid leiomyosarcoma, one as high-grade endometrial stromal sarcoma, one as an undifferentiated sarcoma with a fibrosarcoma-like appearance, and the other as a myxoid neoplasm of uncertain malignant potential. In 3 cases, the rearrangements showed similar breakpoints to known recurrent FOXO1 gene fusions; 2 rearrangements (JRK::FOXO1 and MEIS1::FOXO1) incorporate both an intact transactivation domain and a DNA-binding domain akin to the rearrangements seen in ARMS, likely representing true oncogenic driver events. Although all 5 cases were confined to the uterine corpus at presentation, recurrences occurred in 2 patients indicating a potential for malignant behaviour and justifying the designation of sarcoma. These cases expand the landscape of FOXO1-rearranged neoplasms and describe a potential new uterine mesenchymal entity. Further study of additional cases is needed to establish whether these rearrangements truly represent an initiating event for a distinct subset of uterine sarcomas, or whether FOXO1 rearrangements simply represent an additional noninitiating/nondriver event within other established tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0ec75f2be8c4a40a6afb3f1c139146fd2939aba" target='_blank'>
              Uterine Sarcomas Harbouring Novel FOXO1 Gene Rearrangements: Report of A Case Series.
              </a>
            </td>
          <td>
            Thomas Pilkington, Chris Watt, J. Dermawan, Asma Haider, Abbas Agaimy, B. Howitt, S. Chiang, C. Antonescu, W. McCluggage
          </td>
          <td>2025-12-11</td>
          <td>The American journal of surgical pathology</td>
          <td>0</td>
          <td>138</td>
        </tr>

        <tr id="


 Chromatin remodelers regulate DNA-nucleosome interactions, influencing chromatin structure, accessibility, and gene expression (Gourisankar, S., Nat Rev Genet, 2024). Their dysregulation is frequently observed in cancer and often represents a therapeutic vulnerability (Johnstone, S. E., Science, 2022). HELLS, a key chromatin remodeler, maintains genome stability and regulates gene expression in aggressive T-cell lymphomas, such as ALK-negative anaplastic large cell lymphoma (ALK- ALCL) (Fragliasso V, Leukemia 2020; Tameni A., Cell Death Dis. 2021). Previous studies have demonstrated that HELLS knockdown sensitizes ALCL cells to chemotherapy, supporting its potential as a therapeutic target (Tameni A., Nucleic Acids Res. 2024).
 This study aimed to define the transcriptional role of HELLS and evaluate its therapeutic relevance in ALCL. To achieve this, we analyzed formalin-fixed, paraffin-embedded biopsies from a cohort of 44 ALCL patients and performed an integrated in vitro multi-omics analysis (RNA-seq, ChIP-seq, ATAC-seq, and Connectivity Map), alongside functional assays.
 To understand how HELLS coordinates and supports transcription, we integrated HELLS Chromatin immunoprecipitation-sequencing (ChIP-seq) data from the TLBR-2 cell line with RNA-sequencing data from TLBR-2 HELLS-knockdown (KD) and control cells. Of the 729 genes significantly affected by HELLS KD, 467 (64%) were identified as direct HELLS targets based on ChIP-seq analysis. Gene-set enrichment analysis of these HELLS-direct genes (HDGs) revealed their association with diverse biological processes, including the JAK/STAT signaling pathway and chemokine- and cytokine-mediated inflammatory signaling. The clinical relevance of HDGs was confirmed by nCounter profiling in a cohort of 44 ALCL patients (15 ALK+, 29 ALK- ALCL).
 To assess the contribution of HELLS to HDG transcription, we investigated the distribution of RNA Polymerase II (RNAPII) via ChIP-seq in TLBR-2 HELLS-KD and control cells. RNAPII ChIP-seq data revealed RNAPII elongation defects in approximately 60% of HDGs and altered RNAPII occupancy in ~40% of HDGs following HELLS depletion. Gene ontology analysis of this latter group of HDGs highlighted an enrichment in immune-related pathways, including T-cell-mediated immunity, cytokine signaling, and JAK/STAT signaling. Furthermore, ATAC-seq and H3K4me3 profiling indicated that HELLS promotes chromatin accessibility and transcriptional activation at immune-related loci.
 To evaluate the therapeutic relevance of HELLS, we employed a drug repurposing approach based on the HDG signature. This led to the identification of 10 classes of synergistic pathways, with PI3K, JAK/STAT, and DNA-PK emerging as top-scoring synergistic targets. In several ALCL cell lines, HELLS depletion synergized with an IC20 dose of Ruxolitinib (a JAK inhibitor) or AZD7648 (a DNA-PK inhibitor), resulting in significant synthetic lethality compared to single-agent treatment or control cells.
 In conclusion, this study demonstrates that HELLS drives the expression of immune-related genes through chromatin remodeling in aggressive ALK- ALCL. Its inhibition uncovers combinatorial vulnerabilities, providing a rationale for dual-targeted therapies involving JAK/STAT or DNA-PK pathway inhibition.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088deae46f89f2511cdb635d4408d6da56d67fdf" target='_blank'>
              Targeting hells‑mediated chromatin accessibility to sensitize ALK‑negative anaplastic large cell lymphoma to JAK/STAT and DNA‑PK inhibition
              </a>
            </td>
          <td>
            Giulia Gambarelli, Selene Mallia, B. Donati, N. Puccio, Magda Zanelli, Emanuele Vitale, Federica Torricelli, V. Manicardi, E. Salviato, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="One of the seminal discoveries from genetic studies of autism spectrum disorder and related neurodevelopmental disorders (NDD) has been that loss-of-function (LoF) mutations in many genes that impact chromatin and transcriptional regulation confer substantial liability to NDD. Haploinsufficiency of the epigenetic regulator POGZ represents one of the strongest such associations; however, little is known about the direct or indirect regulatory targets of POGZ, or the mechanisms by which loss of this chromatin modifier alters early neuronal development and synaptic functions. Here, we created an allelic series of CRISPR-engineered human induced pluripotent stem cell (hiPSC) clones harboring mono- and biallelic POGZ deletions. In hiPSC-derived neural stem cells (NSC) and Neurogenin 2-induced neurons (iN), POGZ LoF altered the expression of genes associated with critical cellular processes and neuronal functions, including synaptic and intracellular signaling and extracellular matrix organization. Our multiomics profiling also showed altered footprinting of critical transcription factors (e.g., activator protein 1 complexes) that were enriched at promoters of differentially expressed genes associated with synaptic function. To further interrogate the shared molecular changes in neuronal development associated with NDD and POGZ regulation, we compared our results to deletions of the transcription factor MEF2C and the sodium channel gene SCN2A that we generated in these same isogenic iN. These analyses revealed strong enrichment of extracellular matrix and intracellular signaling disruption associated with POGZ and MEF2C deletion, whereas POGZ and SCN2A haploinsufficiency exhibited shared transcriptional effects on gene modules enriched for NDD-associated genes with opposing regulatory effects. Notably, we also observed alterations to synaptic firing rate and neurite extension with biallelic deletions, but not heterozygous lines, suggesting subtle effects in neuronal development associated with haploinsufficiency. Overall, these shared molecular consequences suggest key points of convergence that connect epigenetic regulation to neuronal function in the etiology of neurodevelopmental pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed6138b4f1663b7339e409062cbd4f7a6fbbd84" target='_blank'>
              CRISPR-engineered deletion of POGZ alters transcription factor binding at promoters of genes involved in synaptic signaling
              </a>
            </td>
          <td>
            M. Moyses-Oliveira, Yating Liu, Serkan Erdin, Dadi Gao, Riya Bhavsar, Kiana Mohajeri, Kathryn O’Keefe, Philip M. Boone, Gabriela Xavier, Calwing Liao, Aiqun Li, Rachita Yadav, M. Salani, D. Lucente, Benjamin Currall, C. D. de Esch, D. J. Tai, D. Ruderfer, Kristen J. Brennand, James F. Gusella, M. Talkowski
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastomas (GBMs) are highly aggressive, infiltrative, and heterogeneous brain tumors driven by complex genetic alterations. The basic-helix-loop-helix (bHLH) transcription factors ASCL1 and OLIG2 are dynamically co-expressed in GBMs; however, their combinatorial roles in regulating the plasticity and heterogeneity of GBM cells are unclear. Here, we show that induction of somatic mutations in subventricular zone (SVZ) progenitor cells leads to the dysregulation of ASCL1 and OLIG2, which then function redundantly and are required for brain tumor formation in a mouse model of GBM. Subsequently, the binding of ASCL1 and OLIG2 to each other’s loci and to downstream target genes then determines the cell types and degree of migration of tumor cells. Single-cell RNA sequencing (scRNA-seq) reveals that a high level of ASCL1 is key in specifying highly migratory neural stem cell (NSC)/astrocyte-like tumor cell types, which are marked by upregulation of ribosomal protein, oxidative phosphorylation, cancer metastasis, and therapeutic resistance genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f56d67f3fa305af969dda22297460569970ba3" target='_blank'>
              CNSC-16. TRANSCRIPTION FACTORS ASCL1 AND OLIG2 DRIVE GLIOBLASTOMA INITIATION AND CO-REGULATE TUMOR CELL TYPES AND MIGRATION
              </a>
            </td>
          <td>
            Bianca L. Myers, T. Vue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11e6d7e02da2372f0b7447b006156f71a150d88" target='_blank'>
              Evo2HiC: a multimodal foundation model for integrative analysis of genome sequence and architecture
              </a>
            </td>
          <td>
            Tangqi Fang, Xiao Wang, Zhiping Xiao, Shengqi Hang, Ghulam Murtaza, Junwei Yang, Hanwen Xu, Anupama Jha, W. Noble, Sheng Wang
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) is a DNA virus that establishes a persistent asymptomatic infection during childhood and can cause Merkel cell carcinoma (MCC) later in life. Its Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, drive viral replication and tumorigenesis by binding to and perturbing the function of host proteins. LT binds and inhibits RB1, deregulating E2F activity and host cell cycle control to permit viral replication during S phase. While the functions of LT and ST are relatively well characterized, how their expression is controlled is poorly defined. Here, we discovered that E2F1-3/DP1 dimers bind the MCPyV Non-Coding Control Region (NCCR) via a consensus E2 site close to the LT/ST transcriptional start site. Inhibiting E2F-NCCR binding, either by deleting the E2 site or treatment with a E2F small molecule inhibitor, downregulated LT/ST mRNA and protein expression in MCC cells and in 293A cells transfected with MCPyV. Our findings reveal an E2F/LT/RB1 positive feedback loop that appears to have evolved to support viral replication and is hijacked in MCC cells to promote cellular proliferation. Furthermore, we identified E2 sites in the NCCRs of PyVs closely related to MCPyV, including murine PyV, which mediate E2F/DP binding and potentiate viral early transcription. E2F/DP also bound weakly to the SV40 and BKPyV NCCRs, despite lacking an E2 site. Our findings challenge the prevailing model that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV early transcription and replication. Significance statement Polyomaviruses (PyVs) express Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, that drive viral replication and tumorigenesis by binding and perturbing the function of host cells. LTs bind and inhibit RB1 via conserved LxCxE motifs, deregulating E2F activity and host cell cycle control to permit viral replication. Here, we discovered that E2F1-3 bind to the Non-Coding Control Regions (NCCRs) of many, but not all, PyVs to activate LT/ST transcription, revealing an E2F/LT/RB1 positive feedback loop that appears to have evolved to promote viral replication. Our findings challenge the existing paradigm that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV LT/ST transcription and replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92fb2e450770315d4079985e1a3aa56f6892c80a" target='_blank'>
              E2F1–3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Nick Salisbury, Supriya Patil-Amonkar, Ann Roman, Denise A. Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Potassium is the most abundant intracellular cation. Potassium flux across cell membranes is controlled by potassium channels. While cytoplasmic potassium has been extensively studied, very little is known about the function of potassium in the cell nucleus. Here, we discover that a nuclear potassium channel complex composed of KCNG1 and KCNB1 subunits is a regulator of nuclear potassium in medulloblastoma, the most common malignant pediatric brain tumour. Using genetically encoded potassium sensors, we show that KCNG1 regulates nuclear potassium level independently of cytoplasmic potassium level. KCNG1 knockdown alters the genomic architecture of medulloblastoma cells by stabilizing G-quadruplex DNA and inducing DNA damage. Ultimately, KCNG1 deficiency results in nuclear swelling, thereby compromising nuclear envelope integrity, promoting the formation of micronuclei, and inducing DNA spillage into the cytoplasm to activate the immune-stimulatory cGAS/STING pathway. Kcng1 knockout in a genetically engineered mouse model of medulloblastoma remodels the immune microenvironment, reduces tumour growth, and prolongs mouse survival. Altogether, we identified a potassium channel complex that regulates nuclear potassium to govern nuclear integrity, tumour immune microenvironment, and medulloblastoma growth. Our results demonstrate that targeting nuclear potassium can be leveraged to treat medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3156f37831fc433a976d1d47ab5180bff944abd" target='_blank'>
              DNAR-02. Nuclear potassium governs nuclear integrity and tumour growth in medulloblastoma
              </a>
            </td>
          <td>
            Jade Chan, Xi Huang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Immune tolerance mechanisms are shared in cancer and pregnancy. Through cross-analyzing single-cell RNA sequencing data from multiple human cancer types and the maternal-fetal interface, we found B7-H4 (VTCN1) is an onco-fetal immune tolerance checkpoint. We showed that genetic deficiency of B7-H4 resulted in immune activation and fetal resorption in allogeneic pregnancy models. Analogously, B7-H4 contributed to MPA/DMBA-induced breast cancer progression, accompanied by CD8+ T cell exhaustion. Female hormone screening revealed that progesterone stimulated B7-H4 expression in placental and breast cancer cells. Mechanistically, progesterone receptor (PR) bound to a newly identified -58 kb enhancer, thereby mediating B7-H4 transcription via the PR-P300-BRD4 axis. PR antagonist or BRD4 degrader potentiated immunotherapy in a murine B7-H4+ breast cancer model. Thus, our work unravels a mechanistic and biological connection of a female sex hormone (progesterone) to onco-fetal immune tolerance via B7-H4 and suggests the PR-P300-BRD4 axis is targetable for treating B7-H4+ cancer.



 Supported by the NIH/NCI R01 grants (CA217648, CA123088, CA099985, CA193136, and CA152470), and the NIH/NCI through the University of Michigan Rogel Cancer Center (CA46592) to W.Z., and by the UTC-Yale Endowment to L.C..



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2912531adf328b0a2781d2f4a13cd7722dc36929" target='_blank'>
              Progestogen-driven B7-H4 contributes to onco-fetal immune tolerance 2475
              </a>
            </td>
          <td>
            Jiali Yu, Yijian Yan, Shasha Li, Ilona Kryczek, Lieping Chen, Weiping Zou
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g., “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, S. Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Using the UCSC genome browser, this study primarily examined the density of CpG Islands in each gene to investigate epigenetic changes, particularly methylation patterns or profile throughout the genome of TNBC genes. Here the analysis of 10 hub TNBC genes were study for the methylation profiling and additionally focused on its relevance of reprograming to these genes.
 It can also analyze how certain genomic areas are conserved across species, which can reveal information about their functional significance. It also examines the existence of CpG islands, which could be connected to other regulatory elements or gene promoters. Examining the data on histone modifications to find possible promoters or enhancers. Genes such as AURKA, CDK1, EZH2, CCNA2, CTNNB1, MKI67, etc., are known cell cycle regulators, chromatin modifiers, and proliferation markers, many of which are overexpressed in cancer and stem cells, including induced pluripotent stem cells (iPSCs). These 10 genes are reprogramming-supportive genes and indirectly modulated or supported by Yamanaka factors. Their hypomethylation and expression patterns in these results indicate that the cellular environment in TNBC resembles a partially reprogrammed state, which may explain aggressive growth and plasticity — a useful insight for drug targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7b6a886f40c24727327b2385ecbc05a3309cb8" target='_blank'>
              Methylation Profiling of TNBC Hub Genes Using UCSC Genome Browser: A Comparative Study Relevance to Reprogramming
              </a>
            </td>
          <td>
            Shivi Sharma, Prashant Ankur Jain, Eliza Chakraborty
          </td>
          <td>2025-11-07</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Dysregulation of gene expression is an essential feature of most human cancers, including acute myeloid leukemia (AML). Despite distinct biology of different subtypes of AML, there is also increasing evidence for shared pathologic mechanisms. Varied genetic subsets of AML exhibit shared dependencies on specific transcription factors and gene expression regulatory mechanisms. In particular, MYB is aberrantly activated in most studied forms of AML, where it is required for blockade of differentiation and self-renewal of leukemia cells. MYB is rarely mutated and generally not over-expressed in most human myeloid leukemias, presenting a key question concerning mechanisms of its activation that should be targeted therapeutically.
 To elucidate potential mechanisms of MYB activation, we investigated post-translational modifications (PTMs) of endogenous MYB proteins purified from human AML cells. Using comprehensive bottom-up high-resolution mass spectrometry, we achieved 95% MYB peptide sequence coverage and identified 10 unique MYB amino acid residues containing phosphorylated, acetylated, methylated, or ubiquitin-like PTMs. This included phosphorylation of S11 and S12 in the N-terminal domain (NTD) proximal to the DNA-binding domain of MYB, as validated using phospho-specific MYB antibody, mutagenesis, and dephosphorylation experiments. Notably, S11/S12 phosphorylation was preferentially detected across multiple AML cell lines and patient specimens from diverse molecular subtypes as compared to healthy human blood progenitor and stem cells.
 We found that MYB is required for the growth and survival of leukemia cells, which can be rescued by the expression of wild-type MYB but not MYB mutants that cannot be phosphorylated at S11/S12 in its NTD. Using label-free quantitative mass spectrometry combined with affinity purifications, we defined specific protein co-factors that are selectively assembled with phosphorylated as compared to non-phosphorylated MYB in the nuclei of AML cells, as well as cells expressing MYB-PLEKHO1 and MYB-ZFAT fusion oncogenes from blastic plasmacytoid dendritic cell neoplasm (BPDCN) cells. We found that S11-phosphorylated MYB preferentially assembles with other hematopoietic transcription factors including LMO2, LYL1, JUN, and TCF12. A subset of transcription factors including ZEB2, SATB1, IKZF1, SPI1, and RUNX1, 2, and 3 were also found to be co-assembled with BPDCN MYB oncogene fusion proteins. Co-assembled MYB transcription factors were associated with cooperative chromatin complexes regulating genes controlling leukemia cell differentiation, growth and survival, as defined using gene expression, chromatin profiling, and functional genomic assays.
 In all, these studies identify DNA-proximal phosphorylation of MYB and MYB DNA-binding domain fusions as a critical regulatory mechanism required for MYB oncogenesis through cooperative organization of transcription factors. Modifications of DNA-proximal NTD leading to MYB enhanceosome assembly suggest a new target for therapeutic intervention of MYB-dependent transcriptional circuits in AML and other transcription factor-dependent malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97715039497852e85a982a7d2a4af32af301c7f5" target='_blank'>
              DNA-proximal MYB phosphorylation is required for oncogenic transcription in acute myeloid leukemia
              </a>
            </td>
          <td>
            Shuyuan Cheng, Katarzyna Kulej, Christopher A G Booth, Masahiro Uni, Nicole Cruz, Andrew Lane, R. Koche, A. Kentsis
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Radiation-induced damage to nucleic acid and other important cellular constituents could likely cause cancer. The present study has analyzed mutated genes with dysregulated expression signatures due to radiation. We have predicted biomarkers specific to hormone-sensitive cancers (HSCs) namely breast cancer (BC), prostate cancer (PC), ovarian cancer (OC), and endometrial cancer (EC). The study has also attempted to prospect the pathways associated with HSCs and radiation exposure through network analysis and functional enrichment analysis. An overlap of important cell cycle pathways, DNA binding, and transcription activity pathways were found in comparison with HSCs and the radiation hallmark pathways. Hub genes TNF, STAT3, CTNNB1, and MYC in BC; IL1B, CTNNB1, ESR1, and SRC in PC; CTNNB1, BRCA1, JUN, and KRAS in OC; PIK3CA, PTEN, and ESR1 in EC were found as hypoxic signatures due to radiation. Further, the expression survival analysis found MYC and STAT3 in BC, CTNNB1 in PC, JUN, and BRCA1 in OC, and ESR1 in EC were significantly dysregulated and further mutational profiling validated MYC with 18%, STAT3 with 2.6%, and all other genes with 4% mutation. Thus, these genes are proposed as radiation-sensitive diagnostic biomarkers in HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e92757f335699ebbedff521e546648301a15ad2e" target='_blank'>
              Systems biology-driven identification of biomarkers and significant pathways in radiation-induced hormone-sensitive cancers
              </a>
            </td>
          <td>
            Suvitha Anbarasu, Sathyanarayan Balaji, Sudha Ramaiah, A. Anbarasu
          </td>
          <td>2025-11-04</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="This review outlines the mechanisms and regulation of DNA replication in Escherichia coli within the oriC region, focusing on DnaA nucleotide state and stepwise occupation of high and lowaffinity boxes, origin DNA topology, and architectural proteins (IHF, Fis, SeqA). We outline helicase loading by the DnaBDnaC complex, the role of the clamp and Hda in controlling reinitiation, and how small RNAs such as DsrA remodel transcriptional programs that impinge on initiation fidelity through the regulation of proteins associated with the DNA replication process and inhibition of. We discuss coupling between replication and transcription, the contribution of DNA gyrase to the need for unwound DNA to replicate, and strategies cells use to avoid replicationtranscription conflicts. Finally, we highlight synthetic biology applications, including orthogonal replication systems in E. coli that decouple engineered DNA propagation from the host chromosome. These insights provide an integrated view of initiation control and suggest opportunities to engineer replication for biotechnology and genome stability research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1d8ddca5d996affebfe1d9eec679ba452b2314" target='_blank'>
              Genetic Regulatory Network of DNA Replication Initiation in E. coli and Its Biological Significance
              </a>
            </td>
          <td>
            Zenas Deng
          </td>
          <td>2025-10-28</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f741cfc314db9bdb828e55bb38f137b5fb40cdaa" target='_blank'>
              Plasmid copy number affects the DNA methylation-driven expression dynamics of the CfrBI restriction-modification system and impacts phage restriction
              </a>
            </td>
          <td>
            Johannes Gibhardt, Trung Duc Hoang, K. Severinov, Mikhail A. Khodorkovskii, N. Morozova
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Kaposi’s sarcoma–associated herpesvirus (KSHV) causes cancers for which no vaccine exists, and current treatments are often imperfect. Recent work shows that the virus and infected cells produce circular RNAs, which are stable RNA loops that can influence gene activity and build up during active infection. Because these molecules resist degradation and may travel inside virus particles, they can help track disease or shape how the virus behaves. This review explains how viral circular RNAs are produced, how researchers detect and assay them, and what is known about their roles during quiet and active stages of infection in the context of KSHV. It also outlines experimental tools to test what these RNAs do and how to target them. By organizing current knowledge and practical methods, this review aims to guide future studies and speed progress toward better diagnostics and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c49ebc6839a316e30284da0f7440e889a77090bc" target='_blank'>
              Recent Studies on Kaposi’s Sarcoma-Associated Herpesvirus Circular RNAs
              </a>
            </td>
          <td>
            Cristian J. Pagtalunan, Isadora Zhang, Ariella Turley, Fenyong Liu
          </td>
          <td>2025-11-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/234e6b6b6481eb76db70b44e7b7d276eb9ea9a24" target='_blank'>
              Population-scale multiome immune cell atlas reveals complex disease drivers
              </a>
            </td>
          <td>
            M. Kanai, T. Delorey, J. Honkanen, R. S. Rodosthenous, J. Juvila, S. Murphy, I. Teixeira-Soldano, H. S. Hwang, J. Karjalainen, J. Halonen, G. Panagiotaropoulou, Y. Zhang, C. McCabe, E. Chen, K. Nanki, T. Yoshida, K. Liu, M. Glean, N. Mehrotra, E. Finan, D. Chafamo, Y. Zhu, M. Arvas, S. Ruotsalainen, Z. Zheng, FinnGen, M. Reeve, M. Kurki, C. B. Porter, O. Ashenberg, W. Zhou, K. PitkaÌnen, J. Partanen, A. Palotie, D. B. Graham, M. J. Daly, R. J. Xavier
          </td>
          <td>2025-11-27</td>
          <td>None</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-10</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Transcription and replication both utilize genomic DNA as a template, creating the potential for frequent interference between the two machineries. Such transcription–replication conflicts (TRCs) can compromise genome stability due to the abnormal accumulation and persistence of R-loops—three-stranded nucleic acid structures comprising a DNA:RNA hybrid and a displaced single-stranded DNA strand (Bhowmick R., et al., Mol Cell. 2023). To safeguard genome integrity, cells deploy multiple R-loop resolution factors that prevent R-loop-associated replication stress (Brickner J. R., et al., Mol Cell. 2022). Persistent R-loops are also associated with the stalling of RNA polymerase II (RNAPII) on chromatin, which further impedes replication fork progression (Zardoni L., et al., NAR 2021).
 Our observation of cells depleted of the chromatin remodelerHELLS, which displayed an accumulation of R-loops and stalled RNAPII on highly transcribed genes (Tameni A., Mallia S., et al., NAR 2024) in aggressive anaplastic large cell lymphoma (ALCL), led us to hypothesize that HELLS-dependent R-loop accumulation can induce replication stress and TRCs. To test replication forks progression, we performed DNA fiber analysis by sequentially labeling the cells with two thymidine analogs, 5-chloro-2′-deoxyuridine (CldU, red) and 5-iodo-2′-deoxyuridine (IdU, green). While control cells showed a CIdU/ldU ratio close to 1, indicating the absence of stalled forks, TLBR-2 and MAC2A HELLS-KD cells displayed an CIdU/ldU ratio close to 2, strongly indicating the presence of stalled forks in absence of HELLS. Then, we performed proximity-ligation assays (PLA) for measuring the presence of TRCs. Specifically, we used antibodies against phospho-forms of RNAPII, pSer5-RNAPII and pSer2-RNAPII, corresponding to initiation and elongation transcriptional complexes respectively and PCNA, as key component of DNA replicative machinery. We found that PLA signal between PCNA and pSer5-RNAPII, but not pSer2 RNAPII, increased in HELLS-KD cells compared to control, suggesting that the depletion of HELLS increases the collision between RNAPII and replicative complex at promoter-proximal sites before pause release. Importantly, ectopic expression of RNaseH1 in HELLS-KD cells reduced R-loop-dependent TRCs, confirming the functional involvement of R-loops in this process.
 Despite the large number of genes demonstrated or proposed to regulate R-loop homeostasis and, therefore, to impact TRCs and in turn on genome integrity, our knowledge of specific HELLS-regulated factors is still limited. To explore the mechanism contributing to TRCs and thus to genome instability, we performed gene set enrichment analysis focusing on genes belonging to functional categories related to chromatin regulation previously identified as HELLS transcriptional direct genes. Among them, mini chromosome maintenance 5 (MCM5) emerged as a potential candidate. Inducible knockdown of MCM5 significantly reduced cell proliferation, increased TRCs levels in lymphoma cells, as measured by pSer5-RNAPII–PCNA PLA foci, led to forks stall and to enhanced nuclear R-loop accumulation, phenocopying HELLS depletion. Notably, the ectopic overexpression of MCM5 in TLBR-2 HELLS-KDcells fully rescues cell proliferation reducing R-loop formation and the presence of TRCs. Additionally, perturbations observed upon MCM5-KD were dramatically reduced when R-loops were suppressed by overexpression of RNaseH1 indicating that, in absence of MCM5, the persistence of R-loop disrupts the replication fork process through TRCs, mimicking HELLS. These data highligh MCM5 as the downstream target of HELLS in replicative processes.
 Integrative analysis of MCM5, and RNAPII chromatin occupancy (ChIP-seq), alongside chromatin accessibility profiles (ATAC-seq) in TLBR-2 cells, further revealed that MCM5 was excluded from accessible, actively transcribed loci, confirming the not-transcriptional role of MCM5.
 Collectively, these findings position the axis HELLS/MCM5 as a critical regulator of replicative processes in aggressive ALCL. Disruption of HELLS/MCM5 activity induces aberrant TRCs and R-loop accumulation, ultimately compromising genome integrity and highlighting a potential therapeutic vulnerability in lymphoma cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caa8a5ddfc9df7b3d2ed0ec9481cd381c40701c0" target='_blank'>
              The chromatin remodeler hells regulates transcription-replication conflicts and genome integrity in ALK-negative anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Selene Mallia, Giulia Gambarelli, V. Manicardi, B. Donati, Emanuele Vitale, Federica Torricelli, E. Salviato, Magda Zanelli, N. Puccio, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common malignancy in women globally. The NOTCH signaling pathway is aberrantly activated in multiple tumors, and NOTCH1, its core transmembrane receptor, is highly expressed in cervical cancer. However, NOTCH1’s mechanisms in cervical cancer progression and radiotherapy resistance, as well as its interaction with key molecules, remain unclear. Methods This study explored the role of NOTCH1 in cervical cancer from multi-omics perspectives, including single-cell sequencing, cDNA microarrays, high-throughput sequencing, and immunohistochemistry, combined with a series of in vitro and in vivo experiments. Results NOTCH signaling activity was negatively correlated with the overall survival and recurrence-free survival of cervical cancer patients. As the core molecule of this signaling pathway, NOTCH1 was significantly highly expressed in cervical cancer tissues and promoted cervical cancer cell proliferation in vitro. Single-cell analysis revealed that NOTCH1 was relatively highly expressed in CPA6+ and CEL+ malignant cells and involved in cell cycle regulation; further cell cycle detection assays confirmed that NOTCH1 could promote the G1-S phase transition. In addition, patients with high NOTCH1 expression showed decreased plasma cell infiltration in the microenvironment. Cell communication analysis indicated that malignant cells with high NOTCH1 expression might lead to impaired plasma cell differentiation due to the impairment of the MIF ligand-receptor pathway. Finally, NOTCH1 could reduce the radiosensitivity of cervical cancer cells to radiotherapy both in vitro and in vivo; whereas has-miR-449a, as an upstream regulatory miRNA of NOTCH1, could inhibit cervical cancer cell proliferation and enhance radiosensitivity by inhibiting NOTCH1 expression. Conclusions This study clarifies NOTCH1’s role in promoting cervical cancer progression and reducing radiosensitivity, with has-miR-449a as a negative regulator, providing targets for optimizing cervical cancer radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e889481c892941401d223ac892318dcf009227f7" target='_blank'>
              Multi-omics reveals that NOTCH1 promotes cervical cancer progression and reduces radiosensitivity
              </a>
            </td>
          <td>
            Aihua Guo, Zhixiong Su, Enhuan Zhang, Huaqin Lin, Peilin Zhong, Zhiyuan Xie, Qingzhen Zhan, Ting Ye, Yang Sun
          </td>
          <td>2025-11-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a15d6fbe121405ee3a7956d58e8fbc1f6ad9d1" target='_blank'>
              Decoding the role of DNA sequence on protein-DNA co-condensation.
              </a>
            </td>
          <td>
            R. Singh, Pinaki Swain, Mahipal Ganji, Sandeep Choubey
          </td>
          <td>2025-12-18</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/167077c6c722e64c238ff049d56bccfcac548091" target='_blank'>
              Acytota and the evolution of complexity
              </a>
            </td>
          <td>
            E. Kejnovský, Z. Kubat, J. Šponer
          </td>
          <td>2025-10-27</td>
          <td>European Biophysics Journal</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Nucleophosmin mutations are among the most common aberrations in acute myeloid leukemia (AML). We previously uncovered a novel pivotal role for mutant NPM1 (NPM1c) in the nucleus, where it binds chromatin at key self-renewal gene loci, such as HOXA/B and MEIS1, directly regulating oncogenic transcription. How NPM1c regulates the transcriptional activity of its chromatin targets remains unclear. To identify potential nuclear cooperating partners of NPM1c, we employed TurboID proximity labeling of NPM1c and wild-type NPM1 (NPM1wt). Mass spectrometric analysis revealed an enrichment of nucleoporins in the NPM1c interactome, with Nucleoporin 98 (NUP98) emerging as a top candidate.
 NUP98, a key component of the nuclear pore complex (NPC), has been established as a critical oncogenic player in AML as part of oncogenic fusion proteins. In these oncofusions the N-terminal FG (phenylalanine-glycine) repeat domain of NUP98 is preserved and fused to a diverse set of partner genes. NUP98-fusion partners are believed to mediate the chromatin binding specificity of the oncoprotein while the NUP98 FG repeats have been shown to form biomolecular condensates that enhance transcriptional output. Interestingly, wild-type NUP98 (NUP98wt) itself has been linked to gene regulatory functions during embryonic development and normal hematopoiesis. However, the role of NUP98wt in transcriptional regulation in the context of leukemia remains unresolved.
 To explore a potential cooperation between NUP98wt and NPM1c we first validated their interaction by co-immunoprecipitation assays. We next wanted to determine whether NUP98 and NPM1c are recruited to the same chromatin sites. To this end, we performed ChIP-seq in an NPM1c endogenous degrader cell line. These experiments revealed co-occupancy of NUP98 at NPM1c target genes and a significant reduction of NUP98 binding upon degradation of NPM1c. These findings suggest that NPM1c is required for the recruitment of NUP98 to key oncogenic target loci.
 To dissect the functional role of NUP98wt in NPM1c AML, we performed loss of function studies. CRISPR mediated knock-out of NUP98 in NPM1 mutant (OCI-AML3) cells resulted in a significant repression of NPM1c target genes, including HOXA/B and MEIS1. Given the essential nature of NUP98, we also engineered an endogenous NUP98 degrader system in OCI-AML3 and NPM1wt (MV4-11) cells to allow for the study of the effects of rapid protein depletion. Upon degradation of NUP98, OCI-AML3 cells exhibited robust myeloid differentiation (~70% increase in CD11b) and increased apoptosis (~50% increase in Annexin V+). MV411 cells on the other hand did not show the same differentiation phenotype. To characterize the immediate transcriptional consequences of NUP98 depletion, we performed nascent RNAseq after 1 and 3 hours of degradation. In the first hours of degradation we observed a rapid and significant repression of NPM1c target genes such as HOXA/B cluster genes, MEIS1, SMC4, TNRC18, SATB1/2 and UNCX, thus demonstrating the strong dependency of the NPM1c-driven transcriptional program on the presence of NUP98. In contrast, in MV4-11 cells, NUP98 loss had no effect on HOXA/MEIS1 expression, suggesting a leukemia subtype-specific role of NUP98wt.
 To delineate the domains of NUP98 required for its chromatin function, we performed a CRISPR gene tiling screen of full-length NUP98 (635 sgRNAs) and based on this we generated several domain-deletion mutants for rescue experiments in the NUP98-degrader cell lines. Retroviral overexpression of the N-terminal FG-repeat containing domains of NUP98 was sufficient to rescue cell viability and prevent the differentiation phenotype typically observed upon NUP98 degradation. Strikingly, gene expression analysis showed that the NUP98 FG-repeats were sufficient to restore the expression of HOXA10, MEIS1, and HOXB9 expression.
 Overall, our study identifies NUP98wt as a critical cofactor for NPM1c-driven oncogenic transcription. NPM1c recruits NUP98 to its chromatin targets, functionally substituting for an oncogenic fusion partner to concentrate transcriptional condensates at oncogenic loci. This establishes a previously unrecognized, subtype-specific dependency on NUP98wt in leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff1e4b1ddc4587a7bab297c9d60234d4ef12e51" target='_blank'>
              Wildtype NUP98 cooperates with NPM1c to drive oncogenic transcription
              </a>
            </td>
          <td>
            Sara Gama, Marcel Müller, H. González Dammers, Elena Haarer, Sara Peruzzi, José Gonçalves-Dias, J. Klusmann, Lu Yang, Renee Chen, Chun-Wei Chen, Stamatis Papathanasiou, B. Falini, Lorenzo Brunetti, Hannah J Uckelmann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Plasmids are extrachromosomal mobile genetic elements that often carry genes responsible for antimicrobial resistance. Plasmid epidemiology aims to track the evolution and spread of plasmids, but the field currently faces significant barriers that make practical implementation using whole genome sequence data difficult. Hybrid-assembled genomes remain the most reliable way to identify and track complete plasmids; however, most genomic surveillance data exists in the form of short-read sequencing, which lacks the resolution required to accurately resolve plasmids. Despite recent advances, long-read-only assemblies have not yet reached the consistency seen in hybrid assemblies. The ideal approach to plasmid epidemiology using whole genome sequence data would consider the limitations of sequencing technologies and the constraints of existing genomic surveillance infrastructure, in addition to the unique evolutionary biology of plasmids. Here, we present ACCIO (Assembly-based Circular Contig Identification for Outbreaks), a tool which creates a reference plasmid database and uses it to infer which plasmids, and genetically related plasmid groupings, are present in an input assembly (Illumina, Nanopore, or hybrid assembly). We validated ACCIO using an internal dataset of 303 plasmid-harboring bacterial clinical and surveillance isolates collected from a single acute tertiary care center. When highly related database plasmids were grouped together, ACCIO achieved 100% sensitivity and 92.1% positive predictive value (PPV) for detection of plasmid groups using hybrid assemblies, and comparably strong performance for Illumina (93.0% sensitivity, 86.6% PPV) and Nanopore (79.3% sensitivity, 91.4% PPV) assemblies. Evaluation on three external datasets yielded consistently high performance. Finally, when benchmarked against MOB-suite, a tool for reconstruction and typing of plasmids, ACCIO demonstrated superior performance across nearly all assembly types and plasmid grouping levels. By integrating database construction, clustering, and plasmid calling into a single workflow compatible with all major sequencing platforms, ACCIO is intended to help advance plasmid epidemiology beyond its current technological and infrastructural barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b54537a906a5e35f12432cee79bda1bb6b2aad6" target='_blank'>
              ACCIO: An Assembly-Based Tool Enabling Plasmid Detection
              </a>
            </td>
          <td>
            Nathan J. Raabe, M. Griffith, V. Srinivasa, K. Waggle, A. Sundermann, L. Pless, Graham M. Snyder, Maria M. Brooks, D. Van Tyne, Lee H. Harrison
          </td>
          <td>2025-11-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="ABSTRACT One exciting class of future genetic devices could be those deployed in microbes that join complex microbial environments in the wild. We sought to determine whether genetic parts designed for monoculture are predictable when used in co-culture by testing constitutive Anderson promoters driving the expression of chromoproteins from a plasmid. In Escherichia coli monoculture, a high copy number origin of replication causes stochastic expression regardless of promoter strength, and high constitutive Anderson promoter strength leads to selection for inactivating mutations, resulting in inconsistent chromoprotein expression. Medium- and low-strength constitutive Anderson promoters function more predictably in E. coli monoculture but experience an increase in inactivating mutations when grown in co-culture over many generations with Pseudomonas aeruginosa. Expression from regulated promoters instead of constitutive Anderson promoters can lead to stable expression in a complex wastewater culture. Overall, we show intraspecies selection for inactivating mutations due to a competitive growth advantage for E. coli that do not express the genetic device compared to their peers that retain the functional device. We show additional interspecies selection against the functional device when E. coli is co-cultured with another organism. Together, these two selection pressures create a significant barrier to genetic device function in microbial communities that we overcome by utilizing a regulated E. coli promoter. Future strategies for genetic device design in microorganisms that need to function in a complex microbial environment should focus on regulated promoters and/or strategies that give the microorganism carrying the device a selective or growth advantage. IMPORTANCE First-generation biotechnology focused on genetic devices designed for use in monoculture conditions. One class of next-generation biotechnology devices could be designed to function in complex ecosystems with other organisms, so we sought to create conditions where the genetic device retained function when the organism carrying it is in co-culture with other organisms. We discovered that when the genetic device is a significant resource burden on the organism carrying the device, mutations will be selected for due to intraspecies and interspecies selection pressures, and the device will be rendered non-functional. Therefore, genetic device design for complex ecosystems in next-generation biotechnology needs to balance functionality of the genetic device with the need to reduce resource burden on the organism carrying it. First-generation biotechnology focused on genetic devices designed for use in monoculture conditions. One class of next-generation biotechnology devices could be designed to function in complex ecosystems with other organisms, so we sought to create conditions where the genetic device retained function when the organism carrying it is in co-culture with other organisms. We discovered that when the genetic device is a significant resource burden on the organism carrying the device, mutations will be selected for due to intraspecies and interspecies selection pressures, and the device will be rendered non-functional. Therefore, genetic device design for complex ecosystems in next-generation biotechnology needs to balance functionality of the genetic device with the need to reduce resource burden on the organism carrying it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce96777e3b66c893027e25686a79572680c419ea" target='_blank'>
              Effects of intraspecies and interspecies competition on genetic device construction and performance
              </a>
            </td>
          <td>
            Samantha Thompson, A. R. Williams, Veronica Dill, Deven Marshall, Emily Sawyer, Mason Alexander, Lilah Rahn-Lee, J. D. Shih
          </td>
          <td>2025-12-17</td>
          <td>mSphere</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Introduction:Childhood leukemia usually develops via a two-hit mechanism, with a preleukemic genomic lesion arising before birth as the first step. In utero origins of preleukemia have been described for several subtypes of acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML); however, backtracking studies have largely relied on newborn dried bloodspots, precluding identification of preleukemic cells of origin. We developed the “ReCord” study (R01CA262012) to conduct comprehensive backtracking and characterization of preleukemic cells across leukemia subtypes, via recruitment of childhood leukemia patients with banked cord blood (CB) at birth. We present initial results for a set of patients with ETV6::RUNX1 fusions.
 Methods:Childhood leukemia patients with banked CB at birth and available diagnostic leukemia (tumor) samples are enrolled into ReCord through Children's Oncology Group (COG) project AEPI20N1 or as volunteers. Paired tumor-germline whole-genome sequencing (WGS) and tumor RNA-sequencing were conducted to identify molecular subtypes and somatic driver events in each patient. We have first focused on ETV6::RUNX1 fusions, which have established in utero origins. Precise fusion breakpoint locations were obtained from structural variant calling in WGS data. Snapgene was used to generate patient-specific transgenes from which patient-specific droplet digital PCR (ddPCR) primer/probe assays were designed and optimized for backtracking experiments. A control gene ddPCR assay allowed quantification. Patient CB samples were enriched for CD34 expression using microbead selection. CD34-enriched or CD34-depleted cells were FACS-sorted to isolate 5 populations: B cells (CD34-CD19+), other differentiated cells (CD34-CD19-), B progenitors (Lin-CD34+CD38+CD19+), other progenitors (Lin-CD34+CD38+CD19-), and earliest hematopoietic stem/progenitor cells (HSPC) (Lin-CD34+CD38-), where Lin=CD2/3/14/16/56/235+ cells. None of the earliest HSPC were CD19+. Sorted CB populations were analyzed for ETV6::RUNX1 fusions using patient-specific ddPCR assays.
 Results:To date, we have enrolled 58 childhood leukemia patients in ReCord (45 ALL, 12 AML, 1 MPAL), of whom 40 were recruited through COG and 18 were volunteers. CB samples have been received for 28/58 patients, and paired tumor samples obtained for 19 of these. We determined the molecular subtypes and genomic driver events via WGS and RNA-sequencing in 15/19 patients; for 4 patients, leukemia blast count was too low (≤10%) to detect somatic drivers. We initially identified 3 childhood ALL patients with ETV6::RUNX1 fusions, and confirmed fusion breakpoints by PCR and Sanger sequencing. Patient-specific ddPCR assays were optimized in each corresponding tumor sample and then tested in the 5 sorted CB cell populations. For two out of three patients, we detected preleukemic ETV6::RUNX1 fusions in at least one CB cell population. In patient 1, the ETV6::RUNX1 fusion was detected in all three CD34+ CB populations, with the highest clonal frequency seen in B-cell progenitors (73%), then earliest HSPCs (20%), and the lowest frequency found in non-B-cell progenitors (9%). The two CD34- populations had very low detection rates (0.4% & 1.3%). In patient 2, the fusion was only detected in the CD34-CD19+ B-cell compartment at low frequency (0.2%). For patient 3, no preleukemia was detectable in any of the five CB populations. ddPCR assays have been designed and optimized for 5 additional patients with varying driver events to date (n=3 ALL, n=2 AML).
 Discussion:ReCord has amassed, to our knowledge, the largest collection of paired CB and tumor samples from childhood leukemia patients. Initial backtracking experiments in patient CB revealed variation in the cell types that harbor ETV6::RUNX1 preleukemic fusions, with only one of three patients harboring the fusion in the earliest stem and progenitor cell compartment. A second aliquot of CB and the corresponding tumor sample from each of these 3 patients are currently being processed for simultaneous single cell genotyping and transcriptomic assays (TARGET-seq). Molecular characterization of normal, preleukemic and leukemic cells, will throw light on the specific prenatal cell types that transform to leukemia, and the molecular mechanisms involved. The ReCord study will eventually inform the potential for newborn screening of preleukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0a56a2517fb45c8567a82072cd5e37c96d7969" target='_blank'>
              Backtracking the cellular origins of ETV6::RUNX1 childhood acute lymphoblastic leukemia in cord blood (ReCord study)
              </a>
            </td>
          <td>
            A. D. de Smith, N. Elliott, Amiya Tucker, Tanxin Liu, Nawara Ahsan, Adam Mead, Irene Roberts, Joseph L. Wiemels, Michelle Roesler, Logan Spector, Anindita Roy
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2fcd1fe48f0aed8f8d06a8b12971cf0f801b0f5" target='_blank'>
              Deconvolution of haematological cancer methylation patterns reveals a predominantly non-disease related proliferation signal and uncovers true disease associated methylation changes.
              </a>
            </td>
          <td>
            H. Lalchungnunga, H. Atasoy, Edward C Schwalbe, C. M. Bacon, G. Strathdee
          </td>
          <td>2025-10-31</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee5cf6576edac68d63b332c1994530f8d8f03ba" target='_blank'>
              Features of DNA methylome in Capsicum spp.
              </a>
            </td>
          <td>
            Ye Liu, Yingjie Mi, Manru Song, Hongji Yang, Lulu Liu, Qian Yin, Xueyan Yao, Guorong Yu, Ying Fu, Guochen Qin, Feng Liu, Xuan Huang, Yan Xue
          </td>
          <td>2025-12-10</td>
          <td>Epigenetics & chromatin</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Cyclin D1, encoded by CCND1, is a cell cycle regulator, overexpressed in ~15% of multiple myeloma (MM) cases via CCND1::IGH super-enhancer juxtaposition through t(11;14). Cyclin D1 also plays a non-canonical role in transcriptional regulation, but in t(11;14) MM its function in gene regulation remains undefined. We investigated the global transcriptional and epigenomic consequences of CCND1overexpression in t(11;14) MM.
 To define a CCND1 signature, RNA-seq from two independent primary MM patient cohorts, CoMMpass and Mayo, was utilized to compare high CCND1 t(11;14) cases (≥300 transcripts-per-million [TPM], n=271) vs. low CCND1 hyperdiploid cases (<30 TPM, n=233). Two CCND1 knockout (KO) clones in t(11;14) MM U266B1 were also generated by CRISPR/Cas9, introducing a frameshift insertion (A) after chr11:69641478 (hg38) in exon 1. ATAC-seq and ChIP-seq (H3K4me3, H3K4me1, and H3K27ac) were performed in duplicates, and RNA-seq in triplicates in U266B1 wild-type (WT) and KO. Libraries were sequenced on Illumina NextSeq 2000 (2x51 bp for ChIP/ATAC, 2x150 bp for RNA-seq). Reads were mapped to hg38 by BWA (ChIP/ATAC) or STAR (RNA-seq), peaks called by MACS2 (FDR< 0.01), peak and gene raw counts estimated by featureCounts. Differential peaks (fold-change ≥1.5) and differentially expressed genes (fold-change ≥2) were identified by edgeR (FDR<0.05), including peaks detected in ≥2 samples and genes with TPM ≥0.1 in ≥1 sample. Interactions between promoters and distal regulatory regions were identified using publicly available chromatin interaction data in blood cell types. Pathway analysis was performed using Enrichr. Co-immunoprecipitation was performed in t(11;14) KMS12BM cells.
 Differential analysis comparing CCND1-high t(11;14) and CCND1-low MM cases in the two patient cohorts identified 21 upregulated and 21 downregulated genes associated with CCND1 expression which were also captured in the wild-typevs. CCND1 KO U266 cells. Enriched pathways (p<0.05) included Notch signaling, early/late estrogen response, epithelial-mesenchymal transition and reduced mitotic spindle and myogenesis. Dysregulated genes included those related to apoptosis, ubiquitin-proteosome, extracellular matrix, adhesion, and immune response, demonstrating a broader role for cyclin D1 in MM beyond cell cycle control. To explore the epigenomic basis of this transcriptional regulation, for each of the 42 differentially expressed genes we examined the changes of histone marks and chromatin accessibility in promoter and the interacting distal regulatory regions in the U266 KO model. Ten of the 21 genes upregulated in CCND1-high (downloaded in KO; e.g., argininosuccinate synthase 1 [ASS1], PMAIP1, MYEOV, CCR7, and DTX1) and 12 of the 21 genes downregulated in CCND1-high (upregulated in KO; e.g., UACA, RASAL2, PTPRZ1, PPT2, and IL23R) showed concordant changes in histone marks and/or chromatin accessibility, predominantly in distal regulatory regions, suggesting a chromatin-mediated mechanism of gene regulation by cyclin D1. The most prominent changes involved H3K27ac, a mark defining active enhancers. As H3K27ac is deposited by histone acetyltransferases CBP, we tested for the possible cyclin D1-CBP interaction. Co-IP confirmed this interaction, suggesting a role for this complex in chromatin remodeling and transcriptional control in t(11;14) MM.
 One gene notably upregulated in association with CCND1 is ASS1, which encodes a urea cycle enzyme responsible for endogenous arginine synthesis. Metabolomic studies demonstrated KO cells had elevated aspartate and citrulline and lowered arginine and ornithine compared to WT cells (p<0.01) demonstrating impaired urea cycle function. Further, KO cells displayed significantly reduced ASS1 protein expression, cell growth and S-phase entry compared to WT cells in arginine-replete conditions, with both WT and KO cell types further inhibited by arginine depletion (p<0.05). Together, a cyclin D1-associated gene signature was identified supporting its role in t(11;14) MM beyond the CDK4/6 mediated cell cycle control, including a role in metabolic plasticity and survival under arginine depleted conditions. These findings highlight cyclin D1 as a therapeutic vulnerability and open new avenues for personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2041ffc4d8646057f972b418e4b1a71a5a0c2" target='_blank'>
              Cyclin D1 beyond the cell cycle: A new role in t(11;14) multiple myeloma
              </a>
            </td>
          <td>
            Huihuang Yan, Aneta Mikulasova, Suganti Shivaram, Hongwei Tang, Stacey Lehman, Samuel Notch, Anderson Hans, Shulan Tian, Michael D Howe, Abiola Bolarinwa, Cinthya Mendoza Zepeda, Marcella Kaddoura, Rafael Fonseca, P. L. Bergsagel, E. Braggio, W. Gonsalves, S. V. Rajkumar, Shaji K Kumar, Francesco Maura, Daniel Rico, Brian Walker, L. Baughn
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="OBJECTIVE
In addition to pathological diagnosis, the identification of a diagnostic index to determine indolent and index cancers will be useful for prostate cancer diagnosis. The present study aimed to provide evidence that quantification of DNA methylation levels at specific gene positions can serve as an indicator for the diagnosis of prostate cancer malignancy.


METHODS
First, we attempted to identify genomic positions where cancer cells could be discriminated from normal cells using prostate-derived cultured cells by quantification of DNA methylation levels with methylation-specific PCR (MSP). Using needle biopsy tissue from patients with prostate cancer bisected by a tissue divider, we quantified DNA methylation levels at four candidate loci identified in cultured cells and analyzed the pathology. Finally, we examined the correlation between DNA methylation levels and gene expression levels in nearby genomic regions using identical tissue from patients with prostate cancer.


RESULTS
Quantitative analysis using prostate-derived cultured cells and needle biopsy tissue from prostate cancer patients showed that DNA methylation levels at CpG sites near four genes (BCAT1, C3orf37, PCDHA1-8, and RAI1) were significantly higher in cancer than in normal. In particular, DNA methylation levels near the C3orf37 gene were higher in high-grade cancers compared to intermediate-grade cancers, showing a significant correlation with cancer grade. In addition, a correlation tendency between DNA methylation levels and gene expression levels was found at the PCDHA1 locus among the four genetic loci.


CONCLUSIONS
Quantitative analysis of DNA methylation at CpG sites near the C3orf37 gene appears to be a promising approach to distinguish between index and indolent prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c03363414413a94d2b395dfe0740f9f7ff86a2" target='_blank'>
              DNA Methylation Levels at the C3orf37 Loci Correlate With Prostate Cancer Grade.
              </a>
            </td>
          <td>
            Satoshi Sako, Saya Ito, Takashi Ueda, Yusuke Gabata, Hikaru Takahashi, Junki Murashita, Masayoshi Okumi, F. Hongo, Osamu Ukimura
          </td>
          <td>2025-11-27</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Filamentous fungi play key roles in ecosystems, agriculture, biotechnology, symbiosis, and disease, yet the large-scale characterization of gene function in these organisms remains limited by low transformation efficiencies and their multinucleate, syncytial cells, which complicate high-throughput screening strategies. To address the challenge of high-throughput screening in filamentous fungi, we developed methods to construct a genome-wide barcoded insertional mutant library in Trichoderma atroviride, a filamentous fungus widely used as a biocontrol agent against bacterial and fungal plant pathogens. Our strategy leveraged randomly barcoded transfer DNA insertions from plasmid libraries containing hundreds of millions of unique DNA barcodes and a broad host-range drug resistance marker delivered via Agrobacterium tumefaciens into T. atroviride. By optimizing transformation conditions, we achieved up to 600 independent transformants per infection event, resulting in a library of over 31,000 mapped insertions disrupting 7,104 of the 11,863 predicted genes in the T. atroviride genome. This resource establishes a scalable platform for high-throughput functional genomics in filamentous fungi, enabling both fundamental investigations of fungal biology and engineering approaches toward improved medical applications, biotechnology, and sustainable agriculture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de8204781e62a67ceaccfa45320e4f337165165" target='_blank'>
              Construction of a randomly barcoded insertional mutant library in the filamentous fungus Trichoderma atroviride
              </a>
            </td>
          <td>
            Lori B. Huberman, J. M. Villalobos-Escobedo, Jeffrey M. Skerker, Ran Shi, Adriana M. Rico-Ramírez, C. Adams, Adam P. Arkin, A. Deutschbauer, N. L. Glass
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="


 Inactivation of tumor suppressor genes (TSGs) impart a cellular fitness in cancers, including acute myeloid leukemia (AML). The silencing of TSGs without direct mutations presents challenges in cancer therapy but also presents a therapeutic opportunity to restore their function. In this study, we identified the transcriptional repressor ZBTB7A as a TSG that is downregulated in AML and associated with poor survival outcomes. Loss of ZBTB7A amplifies TNF signaling, driving a dysfunctional inflammatory state that accelerates leukemia progression. Mechanistically, the mRNA decay factor ZFP36L2 binds to the 3' untranslated region (3'UTR) of ZBTB7A, promoting its transcript degradation. To uncover therapeutic strategies, we developed a CRISPR-based screening approach coupled with in situ FISH-Flow, pinpointing KDM4 as a vulnerability to restore ZBTB7A function. Pharmacologic inhibition of KDM4 enhanced ZBTB7A expression, promoted terminal differentiation of leukemic cells, and demonstrated broad anti-leukemic activity across AML subtypes while preserving normal hematopoiesis. These findings reveal critical regulatory mechanisms of ZBTB7A and support epigenetic therapy as a promising strategy to reactivate its tumor suppressor function in hematologic cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaf7a74c25b86553fb4e494e6a32f6ab68f9d8f" target='_blank'>
              Epigenetic reactivation of a tumor suppressor program in AML.
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Arnuk, I. Aifantis, Hussein A. Abbas, A. Tsirigos, Sadik Karma, S. Rajendran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3160e385fe96505bcb09a45df54fdae63937a125" target='_blank'>
              NONO as a Sensor of Intracellular Oxidation: Relevance to Neuroblastoma Cell Death
              </a>
            </td>
          <td>
            S. S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10944325f830532ecfb5fb037e3e00b3a58c4a6" target='_blank'>
              Sema7a drives an immunosuppressive microenvironment of breast cancer via Kdm4a-mediated DNA replication regulation.
              </a>
            </td>
          <td>
            Xianan Bai, Shanshan Cai, Jie Jiang, Maojin Tian, Jianxin Du, Peiqing Zhao
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Mitoxantrone (MX) is regularly used to treat several cancers. Despite its long history in the clinic, recent studies continue to unveil novel protein targets. These targets may contribute to the cytotoxic effects of the drug, as well as potential non-canonical antitumor activity. A better understanding of MX’s cellular targets is required to fully comprehend the molecular consequences of treatment and to interpret MX sensitivity in homologous recombination (HR)-deficient cancer. Methods: Here, we evaluated MX activity in HR-deficient UWB1.289 (BRCA1−) ovarian cancer cells and surveyed the binding profile of MX using TMT-labeled quantitative proteomics and chemoproteomics. Results: Mass spectrometry (MS) analysis of cellular extracts from MX-treated BRCA1−UWB1.289 cells revealed unique downregulation of pathways instrumental in maintaining genomic stability, including single-strand annealing. Moreover, the BRCA1− cells exhibited a significant upregulation of proteins involved in ribosome biogenesis and RNA processing. Additional MS analyses following affinity-purification using a biotinylated-mitoxantrone probe corroborated these findings, which showed considerable targeting of proteins involved in genome maintenance and RNA processing. Conclusions: Our results suggest that an interplay of both canonical and non-canonical MX-antitumor activity overwhelms the BRCA1− UWB1.289 cells. Furthermore, this study characterizes the target landscape of MX, providing insights into off-target effects and MX action in HR-deficient cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed8f08582f0dd8776423c5be2d239307bc42082" target='_blank'>
              Surveying the Proteome-Wide Landscape of Mitoxantrone and Examining Drug Sensitivity in BRCA1-Deficient Ovarian Cancer Using Quantitative Proteomics
              </a>
            </td>
          <td>
            Savanna Wallin, Sneha Pandithar, Sarbjit Singh, Siddhartha Kumar, Amar Natarajan, G. Borgstahl, Nicholas T. Woods
          </td>
          <td>2025-11-14</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Aim. The current study aims to investigate changes in the expression of proprotein convertases (PCs) genes during the development of lung tumors. PCs is a family of highly specific serine endopeptidases of mammals that process precursors of various proteins and peptides. Nine genes encoding PCs have been identified in the human genome that are essential for normal body functioning. Additionally, PCs can activate proteins involved in carcinogenesis, as well as the expression levels of their genes were shown to correlate with tumor aggressiveness and patient survival. We previously evaluated the expression of all nine PC genes using quantitative PCR on paired samples of lung tumor and adjacent normal tissue. For the first time, we identified four distinct patterns of PCs gene expression change in tumor tissue, which we have called «scenarios». For three of them, covering more than two thirds of the samples, a dominant change in the expression of one PC gene in the tumor tissue was shown. These results may indicate the existence of a limited number of possible options for changes in PCs gene expression during the malignant transformation of lung cells. However, these results need to be confirmed using expanded cohort of tumor samples. Material and Methods. To confirm our previous findings, we analyzed the expression of PCs genes using modern methods of mathematical statistics and data from three previously published studies, which evaluated gene expression through high-throughput RNA sequencing in paired tumor and normal tissue samples from 194 patients with non-small cell lung cancer. Results. Our meta-analysis confirmed that the changes in PCs gene expression in lung tumor tissue compared to surrounding normal tissue follow a limited set of possible scenarios, each having a unique profile of PCs gene expression. Conclusion. The reasons for implementing each scenario may be linked to the origin of tumors, their mutation status, characteristics of the tumor microenvironment, and other factors. Correspondingly, these scenarios may correlate, for example, with tumor aggressiveness and resistance to therapy, and therefore may potentially be used to choose the treatment approach and/or to predict the course of the disease. However, this issue requires further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3af10872808404c4c77f8841898be6b45125c83" target='_blank'>
              Alterations in proprotein convertase genes’ expression during malignization of human lung cells have a limited set of scenarios
              </a>
            </td>
          <td>
            A. A. Komissarov, I. N. Vlasov, P. A. Slominsky, I. Demidyuk
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a71c8b1d6d6ca39d63271ab34a5a519e0ff9e37e" target='_blank'>
              Development of a Self-Activating Dandelion-Shaped DNA Nanodevice for Amplified Imaging of mRNA and Precise Gene Silencing.
              </a>
            </td>
          <td>
            Yunhong Han, Weijie Tong, Fei Ma, 
          </td>
          <td>2025-12-18</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Background: Fusion transcripts are formed by combining exons from two different genes, often due to structural rearrangements such as deletions, inversions or translocations (genomic rearrangement-dependent, GRD) or through aberrant splicing (genomic rearrangement-independent, GRI). In hematological malignancies, many fusion transcripts act as driver events, playing crucial roles in leukemogenesis and serving as diagnostic markers, such as BCR::ABL1. However, the role and diagnostic significance of other fusion transcripts, like P2RY8::CD99—which has been found in both healthy samples and B-ALL patients—are less clear. Similarly, the clinical relevance of other GRI fusion transcripts, such as SEMA6A::FEM1C, often remains uncertain. Single-cell analysis is a promising approach to detect these fusion transcripts within individual cells and cell populations, shedding light on the subclonal architecture and cells of origin.
 Aim: Evaluating the feasibility of integrating long-read sequencing with single-cell library preparation to detect and characterize GRD and GRI fusion transcripts.
 Patients and Methods: Our cohort consisted of 10 samples diagnosed with B-ALL, featuring various fusion transcripts: BCR::ABL1 (n = 6, GRD), SEMA6A::FEM1C (n = 10, GRI), EBF1::PDGFRB (n = 1, GRD), and P2RY8::CD99 (n = 1, GRI), as identified through bulk whole transcriptome analysis. Cryopreserved cells were processed using the GEM-X Universal 3' Expression Library Prep Kit (10x Genomics) and GEM-X cDNAs with cell-specific labels served as input for the Kinnex single-cell RNA kit (PacBio). GEM-X libraries were sequenced on the NovaSeqX instrument (Illumina) with a median depth of 21,565 reads per cell. Kinnex libraries on the Revio (PacBio) with 3,767,066 mean HiFi reads per sample and a mean HiFi read length of 14.2kb. GEM-X libraries were analyzed using cellranger (v9.0.1) and seurat (v5.2.1). Kinnex libraries were preprocessed with the Iso-Seq workflow (PacBio) and fusion transcripts were called with pbfusion.
 Results: Cells from different samples were merged and clustered based on their gene expression profiles from GEM-X libraries. Cell clusters were annotated as B-cells based on CD10, CD79A, and PAX5 expression; T-cells based on CD3 expression; hematopoietic stem cells as CD34+; and myeloid cells by the absence of lymphatic markers and the presence of CD14, FCER1G, and CEBPD. Within the B-cell population, three distinct subpopulations corresponding to various B-cell developmental states were identified: pro-B, pre-proB, and pro-B VDJ. The pro-B VDJ state, characterized by heavy chain rearrangement, showed increased CD20 expression. Furthermore, cell cycle analysis using 100 genes associated with the S-phase/G2M-phase revealed a subpopulation of cycling pro-B cells marked by high levels of MKI67 (G2M-phase) and MCM4 (S-phase). Our fusion calling pipeline successfully detected BCR::ABL1, SEMA6A::FEM1C, EBF1::PDGFRB, and P2RY8::CD99 fusion transcripts within the long-read dataset, without any false positive calls at the sample level. These fusion transcripts were found exclusively in B precursor cells when mapped to the cellular landscape. Notably, cells in the pro-B VDJ state did not harbour any fusion transcripts. Interestingly, P2RY8::CD99 was identified only in B-cells, not in T-cells or myeloid progenitors, suggesting its association with pathogenic cells. Similarly, SEMA6A::FEM1C, a GDI fusion transcript, was confined to B precursor cells, indicating an association with the B-ALL clone.
 Conclusions: We have demonstrated the feasibility and utility of integrating long-read isoform sequencing with single-cell library preparation for detecting fusion transcripts of diverse origins. This transcriptome-wide approach enables disease-agnostic detection and characterization of fusion transcripts, not only those arising from chromosomal aberrations (GRD) but also those resulting from aberrant splicing (GRI) at the individual cell level. Moreover, this method facilitates a comprehensive analysis of cell-specific gene expression profiles and fusion transcripts and could be expanded to include the detection of single-nucleotide variants and copy number changes, thereby completing the molecular profile. While currently not designed for integration into routine workflows, this method represents a valuable tool for research projects aimed at elucidating the molecular mechanisms underlying various diseases, with the goal to enhance patient care.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae73b6fed764877551a830ff73b15a040a45186" target='_blank'>
              Long-read single-cell isoform sequencing for cell type-specific detection of genomic rearrangement-dependent and -independent fusion transcripts
              </a>
            </td>
          <td>
            W. Walter, W. Kern, A. Stengel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cellular senescence is a hallmark of cancer and induces senescence-associated inflammatory responses in the tumor microenvironment (TME). CDKN1A, an important cellular senescence marker, plays significant roles in cell proliferation, invasion, migration, and apoptosis. Previous studies have implied its drug resistance role in certain cancer types. However, its impact on immunotherapy efficacy in advanced LUAD remains unclear. Using TCGA and SU2C-MARK cohorts, we investigated CDKN1A’s biological features in advanced LUAD, analyzing pathway regulation, immune infiltration, and immunotherapy associations. Single-cell RNA sequencing (GSE148071) validated TME and cellular communication differences between CDKN1A high/low expressing tumors in advanced LUAD. CDKN1A expression was positively associated with immunosuppressive environment including extracellular matrix, cancer-associated fibroblasts (CAFs) and myeloid-derived suppressor cells (MDSCs) in advanced LUAD for both TCGA and SU2C-MARK cohorts (P < 0.05). CDKN1A showed significantly positive correlations with many senescence genes in advanced LUAD (P < 0.05), which were also positively associated with endothelial cells, epithelial cells, fibroblast cells and macrophages, but negatively associated with immune cells (P < 0.05). In multivariable cox regression, patients with high CDKN1A expression had worse OS (HR = 2.74, 95% CI = 1.31–5.73, P = 0.007) and PFS (HR = 1.78, 95% CI = 1.01–3.11, P = 0.045) than those with low CDKN1A expression when treated with immunotherapy. In contrast, the high CDKN1A expression was not associated with PFS and OS in the TCGA cohort, in which the LUAD patients received standard chemotherapy, suggesting the immunotherapy predictive role instead prognostic role of CDKN1A. Moreover, single-cell analysis revealed that CDKN1A highly expressed tumors were accompanied by an enrichment of stromal and endothelial cells within the tumor microenvironment, along with enhanced activity of SMAD3/4, the downstream transcription factors of TGFB signaling. These tumors exhibited increased cell–cell communication with stromal cells (COL1A1/COL1A2-ITGA1/ITGB1/SDC1) and endothelial cells (NAMPT-ITGAS/ITGB1/INSR). CDKN1A expression was associated with cellular senescence, immunosuppressive environment and exhibited resistance to immunotherapy in advanced LUAD, suggesting a potential combination strategy with senolytic or senomorphic therapies to overcome immunotherapeutic resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f34f805e0422258abbde7dbe983d9c6f010ed5" target='_blank'>
              CDKN1A and cellular senescence are associated with immune resistance in advanced lung adenocarcinoma
              </a>
            </td>
          <td>
            Wang Shen, Feidie Duan, Shuiting Fu, Peng Cui, Yezhen Shi, Lei Zhang, Xiao Mi, Yu Xu, Guoqiang Wang, S. Cai, Xuebing Li, Xiaojun Tang
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Over 95% of ependymomas (EPN) that arise in the cortex are characterized by gene fusions commonly involving the zinc finger translocation associated (ZFTA) protein. These fusion oncoproteins (FOs) join ZFTA with either a transcription factor or co-activator, leading to a chimeric protein capable of novel functions. To understand the core mechanism of ZFTA FOs we leveraged super resolution and lattice light sheet microscopy. We show that ZFTA FOs undergo liquid-like phase separation to form dynamic nuclear condensates on chromatin. Nuclear condensate formation is critical for activation of ZFTA FO oncogenic targets and the initiation of brain tumors in mice. Furthermore, we show that DNA binding is necessary for proper condensate distribution in the nucleus and provide the first NMR data of a key DNA binding domain of ZFTA as a resource for drug discovery efforts. Using machine-learning models trained on protein sequences of fusion oncoproteins in pediatric and adult cancers, we identified critical amino acid residues in ZFTA FOs necessary for neoplastic transformation. Critically, insights gained from AI-driven mutagenesis studies enabled the creation of synthetic ZFTA FOs, never seen in biology, and capable of driving oncogenic gene activation and brain tumor development. These findings reveal the molecular building blocks important for the assembly of ZFTA fusion oncoproteins, informing our general understanding of brain tumor initiation and providing opportunities for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07256e768fe7d4150bf338ea18958fe1eedd97dc" target='_blank'>
              TMOD-14. Synthetic ZFTA-fusions pinpoint IDR acquisition as a mechanism of brain tumor development
              </a>
            </td>
          <td>
            A. Arabzade, H. Shirnekhi, S. Varadharajan, S. Ippagunta, Aaron Philips, N. Laboe, D. Baggett, Fnu Wahiduzzaman, M. Jo, T. Zheng, S. Tripathi, R. Pathak, Erik Emanus, A. Bland, A. Kardian, B. Holcomb, T. Wright, T. Bugbee, A. Hancock, Bappaditya Chandra, David W. Ellison, Benjamin Deneen, K. Bertrand, R. Kriwacki, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181f022b28db4f3ed7a11dbea0e2ef041b80de4b" target='_blank'>
              Discovery of the TGCB microprotein translated from the lncRNA CHASERR
              </a>
            </td>
          <td>
            Bernardo Bonilauri, Xiaochun Yang
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, K. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran C. Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Owing to their transcription factor binding sites, endogenous retroviruses (ERVs) can act as cis-regulatory-elements (CREs). By invading genomes in waves, ERVs offer a substrate for lineage-specific adaptations but also, when dysregulated, for lineage-specific disorders. Their role as CREs in rapid placental evolution, and in the human-specific placental disorder preeclampsia, may thus provide a paradigmatic exemplar. Here then we systematically identify ERV-derived CREs controlling human placental gene expression with dysregulation in preeclampsia. We identify 87 ERV-derived CREs located upstream of genes expressed in the placenta. A subset of nine, all belonging to the ERV3-MLT1/2 families and dating to the mouse–human common ancestor, are consistently dysregulated in trophoblasts from preeclampsia samples. Of the nine ERV3-MLT1-linked genes dysregulated in preeclampsia, five are novel candidates, while four were previously associated with preeclampsia, though their ERV-based regulation was not recognized. Focusing on EPS8L1, we predict enhancer activity of upstream MLT1(G1) and validate using reporter assay and genome editing. This vertebrate-specific gene is expressed in progenitor cytotrophoblasts and syncytiotrophoblasts and is overexpressed in preeclampsia, correlating with preeclampsia biomarkers and is not elevated in related pregnancy disorders. A soluble form of EPS8L1 is detectable in maternal plasma as early as between 24 weeks of gestation. EPS8L1 knockout in trophoblast in vitro is lethal, and its overexpression alters trophoblast behaviors characteristic of preeclampsia. We conclude that ERV3-MLT1functions as a trophoblast-specific CRE for several human genes and may be dysregulated in preeclampsia. As EPS8L1 has a form in maternal circulation, it may have utility in diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b721addfcf450b74bec4415619bad752451d80" target='_blank'>
              ERV3-MLT1 provides cis-regulatory elements for human placental functioning and are commonly dysregulated in human-specific preeclampsia
              </a>
            </td>
          <td>
            Rabia Anwar, Amit Pande, Manvendra Singh, Zhi Huang, Eve Hallett, Yiran Xie, A. Gellhaus, F. Herse, O. Nonn, M. Gauster, Tamás Raskó, Matthias Selbach, Stefan Verlohren, A. C. Staff, Ralf Dechend, Ulrich Pecks, S. M. Blois, Laurence D. Hurst, Z. Izsvák
          </td>
          <td>2025-11-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5ca99f1d92208d16804d01a5ade96f5db584369" target='_blank'>
              NAT10-mediated N4-acetylcytidine (ac4C) modification of PIK3R2 mRNA promotes malignant progression of glioblastoma.
              </a>
            </td>
          <td>
            Xiannan Meng, Tao Dong, Jiayu Xu, Kaiwen Tian, Wendong Yang, Zixuan Liu, Yongjing Qian, Dingding Liu, Changxiu Chen, Jin Bai, Hong-Mei Yong, Xiaojin Wu, Zhigang Shen
          </td>
          <td>2025-12-17</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Supratentorial ependymomas are often life-threatening brain tumors that are characterized by significant molecular heterogeneity. Hypoxia contributes to this heterogeneity, but the extent and nature of this relationship remain unclear. This poses challenges for effective treatment strategies, demanding a more holistic and detailed understanding of the underlying biology. In this study, we employed single-nucleus (n = 63) and spatial (n = 30) transcriptomics to delineate the cellular and spatial landscape of supratentorial ependymoma. Characterizing the transcriptional consensus programs of this entity revealed two previously undescribed programs that are associated with the remodeling of the extracellular matrix and the ZFTA fusion identity. These programs were related to highly unfavorable outcome and were exclusive to ZFTA fusion-positive tumors. Developing a set of spatial scoring algorithms that are sensitive to mixed spot transcriptomes enabled the identification of the role of hypoxia as the potential driver of spatial organization. Together with recurrent spatial associations inferred between programs, this informed a generalized model of the higher-order architecture. The functionally and spatially defined zones of this model displayed distinct immune presence, with the highest seen in the hypoxic zone. These results provide insights into the molecular heterogeneity and spatial organization of supratentorial ependymomas, which may aid in the development of targeted future treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6575f9d3eac6558b40e6240ea8f2f812824b5235" target='_blank'>
              Integrated Single-Nucleus and Spatial Transcriptomics Elucidate Heterogeneity and Hypoxia-Driven Organization of Supratentorial Ependymoma.
              </a>
            </td>
          <td>
            Erik Schüftan, I-Na Lu, Flavia W de Faria, Clara Inserte, C. Rössig, Julia Bothe, Adrienn Bagosi, S. Tippelt, Tobias Blau, C. Monoranu, L. Altendorf, D. Spille, Ann-Katrin Bruns, A. Brentrup, Walter Stummer, D. Gorodezki, Martin Ebinger, J. Schittenhelm, C. Thomas, T. Albert, U. Schüller, K. Kerl
          </td>
          <td>2025-10-28</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 The polycomb repressor complex 2 (PRC2) is an epigenetic regulator implicated in cancer immune evasion through silencing of tumor major histocompatibility complex 1 (MHC class 1) in MHC class 1-low or -negative cancers. As glioblastoma (GBM) expresses decreased levels of MHC class 1 compared to normal brain, we sought to determine whether PRC2 activity may aid in GBM immune evasion through suppression of tumor-cell antigen presentation and modulation of the tumor microenvironment. We find that pharmacologic inhibition of enhancer zeste homolog 2 (EZH2), the main catalytic subunit of PRC2, or its functional partner, DNA methyltransferase 1 (DNMT1) in GBM patient-derived cell lines (PDCLs) results in upregulation of MHC class 1 genes and genes involved in antigen peptide loading onto MHC class 1. Genetic perturbation of PRC2 through knockdown of EZH2 or the non-catalytic PRC2 subunit EED similarly results in upregulation of MHC class 1 in GBM PDCLs. Bulk RNAseq data from GBM PDCLs treated with EZH2 inhibitor or DNMT inhibitor demonstrates upregulation of gene sets associated with interferon response, allograft rejection, inflammatory responses and viral infection consistent with a modulation of tumor intrinsic anti-tumor immunity. In human glioblastoma samples, we find that expression of PRC2 components is inversely correlated with expression of MHC class 1 molecules in published single-cell RNA sequencing datasets. Finally, we used highly multiplex immunofluorescence by CODEX to explore spatial relationships between tumor cells as defined by their levels of PRC2 expression and activity with the immune tumor microenvironment. We find complex relationships between tumor PRC2 expression and the immune microenvironment, including correlations between tumor cell EZH2 expression and H3K27 trimethylation/acetylation levels with PD-L1 expression and degree of immunosuppressive neutrophil infiltration. These data suggest that the PRC2 may play a role in modulation of immunogenicity and suggests that PRC2 inhibition may enhance immunotherapy effectiveness in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5967551da92b25343e3693c7e21866cdf2ee17e9" target='_blank'>
              TMIC-32. Polycomb repressor complex 2, its catalytic subunit EZH2 and its functional partner DNA methyltransferase 1 suppress tumoral MHC class 1 and modulate the tumor immune microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Andrew D’Auteuil, Shreya Nakhawa, Jing Ni, Keith L. Ligon, P. Wen, D. Reardon, Kun Huang, Jean Zhao, J. McFaline-Figueroa
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Nasopharyngeal carcinoma (NPC) is endemic to several regions of North Africa and Southeast Asia and has a striking geographic distribution. It is closely linked to the Epstein-Barr virus (EBV) and is still found at an advanced stage that permits metastasis. However, only a limited number of EBV genes, including latent membrane protein 1 (LMP1), have been identified. The 30bp deletion variant of LMP1, which is thought to be the primary EBV oncoprotein, was more prominent in NPC biopsies. The aim of this study was to employ a 30bp deletion in LMP1 as a biomarker to help clinicians diagnose NPC early, improve prognosis, and reduce mortality rate. Polymerase chain reaction (PCR) was used to detect the prevalence of EBV in patients with NPC by amplification of the EBNA-1 and LMP1 genes. β-actin was amplified in all samples as a positive PCR control. The incidence and correlation of the 30bp deletion of LMP1 in 71 paraffin-embedded NPC tissue samples from Jordanian patients (51 males and 16 females) were also evaluated. The results were confirmed by Sanger sequencing and statistically analyzed using SPSS software. EBV was detected in all 71 NPC tissue samples, with positive results for EBNA-1 gene in 58 cases (81.7%) and in combination with LMP1 with or without 30bp deletion in 69 cases (97.18%) whereas, LMP1 30bp deletion was detected in 39 samples (54.9% of all samples). Heterogeneity of mutation was observed in 14.08% of samples. No significant differences were found between the stages with regard to the LMP1 30bp deletion. Statistical analysis showed a significant association between age and LMP1 30bp deletion with p-value = 0.015, and between the source of sample with LMP1 30bp deletion p-value = 0.032, whereas there was no association between LMP1 30bp deletions and patient sex or histological NPC type. A 30bp deletion of LMP1 was found in 54.9% of NPC tissues. This is within the range reported in other studies. There was no significant association between the 30bp deletion of LMP1 and histological type or disease phase. Further research is required to determine how this mutation in LMP1 affects the course and results of patients with NPC in Jordan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28abb19a141082a00b48aeba4beb04a9be49d2d5" target='_blank'>
              Detection of 30bp deletion LMP1 oncogene as potential diagnostic and prognostic Biomarker among nasopharyngeal carcinoma patients
              </a>
            </td>
          <td>
            Aseel Al-Hussien, M. Abusalah, Yaqeen Rjoub, Khaled Al-Qaoud, Anwar Rjoop, Yasmin AlSaidat, Lubna H Khwaileh, Manal Abusalah, Naveed Ahmed
          </td>
          <td>2025-11-19</td>
          <td>Electronic Journal of General Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Embryonal tumors with multilayered rosettes (ETMR) are rare pediatric brain cancers with a median survival of 14 months despite aggressive treatment. The oncogenic driver in ~90% of cases is an amplification and fusion of the chromosome 19 micro-RNA cluster (C19MC), which is yet undruggable. Due to the tumor’s rarity, comprehensive molecular characterization and the identification of therapeutic targets are challenging, and ETMR remains poorly understood.
 We used single cell RNA sequencing and multiplexed protein profiling to study intratumoral heterogeneity and oncogenic signaling in ETMR (n=11). MiRNA-mRNA interactions were investigated by enhanced crosslinking and immunoprecipitation (eCLIP) followed by next-generation sequencing. ETMR cells were treated with locked nucleic acids (LNAs) and small-molecule inhibitors, and their anti-tumor effects assessed through confluency and death assays. Here, we identified three distinct malignant ETMR cell types resembling differentiation stages during early fetal neocortex development: undifferentiated ETMR cells resembling neural-stem cells (NSC), differentiated cells corresponding to mature neurons, and intermediate cells exerting markers of neuronal lineage commitment. C19MC was overexpressed in mitotically active NSC-like cells. Using eCLIP, we uncovered widespread regulatory roles of individual C19MC members. Markers of neuronal differentiation (NR2F2, SOX4, SOX11) and lineage commitment (C2–H2 ZNF family members) were targets of abundant C19MC members. LNA-mediated blockade of key C19MC members significantly reduced ETMR cell confluency and induced apoptosis. Genome-wide cell-cell communication analysis identified fibroblast growth factor receptors (FGFRs) and NOTCH receptors as key mediators of cellular cooperation. Accordingly, FGFR and NOTCH inhibitors had significant anti-tumor effects in vitro, and erdafitinib-supplemented therapy led to partial response in one therapeutically exhausted ETMR patient. Together, we demonstrated intratumoral heterogeneity in ETMR paralleling fetal neurodevelopment. Targeting C19MC in undifferentiated NSC-like cells significantly impaired cell confluency and survival. Additionally, FGFR emerged as a clinically actionable target, showing promising anti-tumor activity in vitro and in one ETMR patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb99e7f0c8810ec820bdec1d791a67a355e38fb" target='_blank'>
              EXTH-92. Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions
              </a>
            </td>
          <td>
            L. Gabler-Pamer, Alexander Beck, G. A. V. Cruzeiro, S. Lambo, McKenzie L. Shaw, Olivia A Hack, Andrezza Nascimento, S. Madlener, Daniela Lötsch, I. Slavc, C. Dorfer, C. Haberler, Lissa C. Baird, Susan N. Chi, S. Alexandrescu, J. Gojo, Kool Marcel, V. Hovestadt, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Nuclear Factor I (NFI) proteins are involved in adenovirus DNA replication and regulate gene transcription, stem cell proliferation, and differentiation. They play key roles in development, cancer, and congenital disorders. Within the NFI family, NFI-X is critical for neural stem cell biology, hematopoiesis, muscle development, muscular dystrophies, and oncogenesis. Here, we present the structural characterization of the NFI transcription factor NFI-X, both alone and bound to its consensus palindromic DNA site. Our analyses reveal a MH1-like fold within NFI-X DNA-binding domain (DBD) and identify crucial structural determinants for activity, such as a Zn²⁺ binding site, dimeric assembly, and DNA-binding specificity. Given the ~85% sequence identity within the NFI DBDs, our structural data are prototypic for the entire family, a NFI Rosetta Stone that allows decoding a wealth of biochemical and functional data and provides a precise target for drug design in a wider disease context. The Nuclear Factor I (NFI) family includes transcription factors key to development, cancer, and genetic disorders. Here, the authors present the structure and DNA-binding properties of NFI-X, a prototype for the entire NFI family.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09b9f75cb385c6026fbb070d6c58ca1ca1038c7" target='_blank'>
              Structural basis of Nuclear Factor 1-X DNA recognition provides prototypic insight into the NFI family
              </a>
            </td>
          <td>
            Michele Tiberi, M. Lapi, L. Gourlay, A. Chaves-Sanjuan, Maurizio Polentarutti, N. Demitri, Miriam Cavinato, Diane Marie Valérie Jeanne Bonnet, Valentina Taglietti, Anna Righetti, Rachele Sala, S. Cauteruccio, Amit Kumawat, R. Russo, A. Barbiroli, N. Gnesutta, C. Camilloni, M. Bolognesi, Graziella Messina, Marco Nardini
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Regulation of gene expression is central to the development of immune cells and their ability to respond to infection. As part of a clinical evaluation, we identified two sisters with recurrent infections, hypogammaglobulinemia, and memory B cell deficiency, diagnosed as common variable immunodeficiency. Whole exome sequencing identified a heterozygous variant (Leu50Ser, L50S) in a conserved region of EZH2, the catalytic subunit of the epigenetic gene repressor Polycomb Repressive Complex 2 (PRC2). EZH2-catalyzed histone H3 lysine 27 methylation (H3K27me) in bulk was not overall significantly disrupted by this variant, in patient samples or cell lines expressing EZH2-L50S. EZH2-L50S protein is expressed similar to wild-type and can form PRC2. However, we find that specific genomic regions that normally have high wild-type levels of H3K27me3 are deficient in the L50S context, particularly around gene promoters. EZH2-L50S is still recruited to these sites, but is not as active. Using recombinant purified PRC2, we determine that L50S affects methylation of nucleosomes and disrupts allosteric stimulation that normally amplifies H3K27me3, consistent with the location of L50 in the allosteric regulatory region of PRC2. Thus, variation of EZH2 L50, occurring at low frequency in the population may interfere with normal B cell gene expression patterns, contributing to immunodeficiency. This study has implications for genetic variation in PRC2 in the general population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6676cb41cac1218e4eeedff2686f0a6921ca0cf8" target='_blank'>
              Characterization of an immunodeficiency-associated EZH2 variant
              </a>
            </td>
          <td>
            Francisco Perez de los Santos, Lily Beck, Emily DeCurtis, April M. Griffin, Maggie M. Balas, Charlotte Marchioni, Attila Kumánovics, Matthew R. G. Taylor, Kimberly R. Jordan, Charles H. Kirkpatrick, Aaron M. Johnson
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
A large scale detection of MLH1 methylation is lacking in esophageal cancer. MLH1 is a well-known mismatch repair gene. The mechanism of MLH1 in DNA double strand break (DSB) repair remains unclear.


METHODS
Esophageal cancer cell lines and 1018 cases of primary cancer samples were employed. Methylation specific PCR, Western Blot, and CRISPR/Cas9 knockout technique were utilized.


RESULTS
Methylation of MLH1 was detected in 3.93%. MLH1 methylation was significantly associated with tumor differentiation, male gender, smoking, and tumor size (all p < 0.05). The median overall survival (OS) was 24.7 months (95% CI 13.4-36.0) and 51.5 months (95% CI 40.4-62.5) in MLH1 methylated and unmethylated groups, respectively. OS was shorter in MLH1 methylated compared to unmethylated group patients (p < 0.01). Multivariate factor analysis indicated that MLH1 methylation is an independent poor prognosis marker (p < 0.05). MLH1 promotes ataxia telangiectasia mutated (ATM), ataxia telangiectasia and RAD3-related (ATR), and non-homologous end-joining repair (NHEJ), while inhibiting microhomology-mediated end joining (MMEJ) repair signaling pathways. Deletion of MLH1 sensitized esophageal cancer cells to novobiocin.


CONCLUSIONS
MLH1 plays important roles in DSB repair and deletion of MLH1 sensitizes ESCC cells to Polθ inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bee021a427dac36fd393a8becff1cde1aed04fc7" target='_blank'>
              Defect of MLH1 expression sensitized esophageal squamous cell carcinoma cells to Polθ inhibitor.
              </a>
            </td>
          <td>
            Bo Zhou, Meiying Zhang, Cheng Zhu, Aiai Gao, Xiaomo Su, Mingzhou Guo
          </td>
          <td>2025-10-30</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Telomerase Reverse Transcriptase promoter (TERTp) mutations enable tumor cell immortality in millions of cancer patients annually. TERTp mutations are the most common non-coding mutations across all cancers, including glioblastoma, oligodendroglioma, medulloblastoma, and high-grade meningioma. A multitude of TERTp mutations, including the two hotspots, create de novo E26 transformation specific transcription factor binding sites. Among the 28 family members, we have shown that only the GA-binding protein (GABP, composed of DNA binding GABPA and transactivating GABPB subunits) activates the mutant TERTp. Prior therapies targeting the telomerase RNA component, TERC, lacked tumor selectivity and were poorly tolerated. The GABP-mediated reactivation of the mutated TERTp presents a unique therapeutic opportunity for tumor specific reversal of cellular immortality. Breakthroughs in the design of biological proteolysis-targeting chimera (bioPROTACs) offer a new approach to selectively degrade previously intractable targets such as transcription factors. We combined in silico protein-protein interaction modeling via AlphaFold and experimental validation to identify a minimal GABPB to bind GABPA and fused it with an E3 ubiquitin ligase. Introduction of this GABPA bioPROTAC into TERTp mutant glioblastoma cells depleted GABPA protein and eliminated GABPA binding to the mutant TERT promoter, reducing transcriptional activating (H3K4me3) and increasing suppressive (H3K27me3) histone marks. Reversion of mutant TERTp to an epigenetically silenced state reduced TERT expression by 73% to 95% and shortened tumor cell telomeres in a promoter mutation-specific manner. The GABP bioPROTAC reduced tumor growth and improved survival of mice bearing an orthotopic xenograft of TERTp mutated glioblastoma cells. The effects of the bioPROTAC were validated in vivo via magnetic resonance imaging of metabolic correlates of TERT expression. These data demonstrate that this GABP bioPROTAC potently degrades GABPA, leading to tumor-specific silencing of telomerase expression, and a reversal of glioblastoma tumor cell immortality. This artificial intelligence-guided approach may be applicable to other intractable cancer specific targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102d24aea88be03925f3847c09d31dfd2d63944f" target='_blank'>
              EXTH-77. An AI-engineered bioPROTAC reverses immortality of
 TERT
 promoter mutant tumors
              </a>
            </td>
          <td>
            Nicholas O. Stevers, Ellen Chen, Joshua S. Wu, T. Patel, Sara Collins, M. Montoya, Samuel H. Wu, G. Batsios, A. Gillespie, Chibo Hong, Noriyuki Kasahara, Pavithra Viswanath, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251a4fb00acc97a635e84c6467a865b534993fc8" target='_blank'>
              PRC2 Restricts Malignant Peripheral Nerve Sheath Tumorigenesis in a Genetically Engineered Mouse Model of MPNST
              </a>
            </td>
          <td>
            Woo Hyun Cho, Amish J. Patel, Makzhuna Khudoynazarova, Sara Warda, Juan Yan, Dan Li, Dana M. Schoeps, Eve Fishinevich, Mohini R Pachai, Cindy J. Lee, Zoe Jacobs, Kae Kristoff, Jessica Sher, C. Antonescu, Yu Chen, Ping Chi
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>138</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Early T-cell precursor (ETP) acute lymphoblastic leukemia/lymphoma (ALL) is an aggressive hematological malignancy characterized by the uncontrolled expansion of T-cell progenitors. Loss-of-function mutations in genes encoding epigenetic regulators, including core components of the Polycomb Repressive Complex 2 (PRC2), such as the EZH2 histone methyltransferase, are frequently observed in ETP-ALL patients.To investigate the role of EZH2 inactivation in human early T-cell leukemogenesis, we optimized a CRISPR/Cas9-based strategy to disrupt EZH2 gene expression by integrating a DNA cassette containing the constitutive human PGK-1 promoter followed by the GFP reporter. Specifically, human CD34⁺ hematopoietic stem/progenitor cells (HSPCs) derived from cord blood were first edited to knock out (KO) EZH2, then transduced with lentiviruses encoding different T-ALL oncogenes or an empty vector control, and maintained in serum-free medium supplemented to support T-cell progenitor expansion. We found that only HOXA9- and LYL1-transduced EZH2KO HSPCs expanded in vitro for up to 30 days, and were enriched for early T-cell subsets, as identified by single-cell RNA sequencing (scRNA-seq) and a 26-color flow cytometry assay. Notably, these same subsets were capable of initiating T-cell leukemia in immunocompromised (NSG) mice following transplantation, suggesting that EZH2 inactivation cooperates with HOXA9 or LYL1 overexpression to drive human T-cell transformation. To assess the relevance of these findings in primary human T-ALLs, we analyzed whether loss-of-function mutations in PRC2 genes (EZH2, EED, SUZ12) were enriched in specific T-ALL subgroups, particularly ETP-ALL, in a cohort of 1,335 patients from the Children's Oncology Group (COG) study. Interestingly, PRC2-mutated T-ALLs were significantly associated with the HOXA subgroup and the ETP-ALL phenotype, and exhibited higher levels of minimal residual disease (MRD) 30 days after therapy compared to other T-ALL cases. To further dissect the molecular effects of EZH2 loss and HOXA9 overexpression in ETP-like T-ALL, we knocked out EZH2 in the human Loucy and PEER cell lines using CRISPR/Cas9. These EZH2KOcells were then transduced with HOXA9-encoding lentiviruses or empty vectors and analyzed by RNA sequencing. In parallel, ChIP-seq analyses were performed to map HOXA9 binding sites and assess histone modifications affected by EZH2 loss. Integrated analysis of RNA-seq and ChIP-seq data revealed that HOXA9 expression in EZH2-deficient ETP-like cells led to the upregulation of a stem-like/immature T-cell gene signature enriched in transcriptional activators and chromatin remodelers (e.g., RUNX2, IRX3, SATB1, ARID5B, CHD1, INO80D).Furthermore, scRNA-seq profiling of primary PRC2-mutated T-ALL samples revealed distinct leukemic subsets with high HOXA9 expression and strong enrichment for the same gene signature, as determined by Gene Set Variation Analysis (GSVA). Importantly, this signature stratified PRC2-mutated T-ALLs and identified a HOXA positive subgroup associated with poor prognosis.
 Collectively, these findings uncover a critical epigenetic circuit involving EZH2 loss and HOXA9-driven transcriptional reprogramming that promotes stem-like, therapy-resistant T-ALL subsets and highlights novel vulnerabilities that could be therapeutically exploited to eliminate high-risk disease at its origin.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d423907fad1e15df50c67eaeeaa6ff45c0caebf" target='_blank'>
              EZH2 inactivation cooperates with HOXA9 to induce leukemic transformation of human t-cell progenitors and sustain high-risk subsets of early t-cell precursor acute lymphoblastic leukemia (ETP-ALL)
              </a>
            </td>
          <td>
            M. Colucci, Costanzo Padovano, Gaja Bruno, E. De Santis, Serena di Iasio, Chiara Di Nunzio, Andrew Weng, Vincenzo Giambra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca6176cd27da7f6583937f65a3ed03caa145ef0" target='_blank'>
              Overexpressed PRR11-SKA2-miR301a/454 bidirectional transcription unit essentially and coordinately promotes PI3K-AKT pathway activation and lung cancer progression.
              </a>
            </td>
          <td>
            Tao Liu, Xiaofeng Zuo, Shijie Sun, Kailong Du, Chuntao Tao, X. Xia, Linli Yu, Chunxue Zhang, Zhengmei Yang, Yitao Wang, Junhong Ye, Youquan Bu
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d84e4ce8cfd881f458277464cbfe840fdb931ee" target='_blank'>
              Biased enrichment of DNA uptake enhancing sequences in Pasteurellaceae and Neisseriaceae
              </a>
            </td>
          <td>
            Stian Aleksander Helsem, Kristian Alfsnes, Stephan A. Frye, O. Ambur, Alexander Hesselberg Løvestad
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH), the clonal expansion of mutant hematopoietic stem and progenitor cells (HSPCs), is highly prevalent in aging populations and is linked to age-related diseases such as cardiovascular disease and malignancy (Jaiswal et al., N. Engl. J. Med., 2014, 2017). DNA methylation regulators, Ten-Eleven Translocation 2 (TET2) and DNA methyltransferase 3A (DNMT3A), are among the most frequently mutated genes in CH (Genovese et al., N. Engl. J. Med. 2014). Recent studies report the essential role of TET2/DNMT3A in regulating DNA methylation in euchromatin particularly in the regulation of key genes involved in hematopoietic stem cell function and lineage specification. DNA methylation also maintains heterochromatin by silencing transposable elements (TEs). Our recent study reported a distinct role for DNMT3A and TET2 in regulating the spatial relocation of H3K9me3-marked heterochromatin during HSPC aging, which contributes to age-related CH (Hong et al., Nat. Aging, 2023). Our data suggested the upregulation of TEs correlated with the activation of interferon-stimulated genes (ISGs) and contributed to the functional decline of aged HSPCs. This decline was mitigated in TET2-mutant HSPCs, increasing their competitive capacity. However, given HSPC heterogeneity, the cell type specific TEs alterations during aging and in the pathogenesis of CH remain largely unknown.
 To further delineate the transcriptional and epigenetic regulation of TEs during aging, we purified Lin- HSPCs from C57BL/6 wild-type (WT) and TET2-knockout (TET2-KO) mice at 7 weeks (7w), 21 months (21m), and 30 months (30m) of age. We then performed single-cell (sc) RNA-seq and scATAC-seq. After preprocessing, cell type labeling, and integration of both scRNA and scATAC datasets, we quantified TE expression and accessibility at both the locus and family levels using MATES, an autoencoder-based methodology (Wang et al., Nat. Commun., 2024). We compared WT and TET2-KO aging by integrative scRNA/scATAC analysis of lineage biases and regulon activities.
 Coding-gene and TE expression profiles independently resolved the same HSPC progenitor subpopulations, indicating cell type specific TE expression. This strongly suggested the tissue specific regulation of TEs, consistent with previous publications regarding embryonic development (Miao, et al, Genome Biol 2020). Furthermore, scATAC-seq analysis revealed similar subpopulation specific TE trends, including distinct, smooth trajectories of TE accessibility along myeloid, lymphoid, and megakaryocyte-erythroid lineages. Integrating our 7w and 21m WT scRNA-seq and scATAC-seq, we found that age-dependent gains in TE accessibility and transcription varied across all major progenitor subpopulations. These age-dependent changes in TE accessibility and transcription occurred in all major classes of TEs including LINEs, SINEs and LTRs, but the changes were mitigated in TET2 deficient cells, strongly suggesting distinct regulatory mechanisms in TE during TET2 deficient HSPCs aging. The age-dependent changes in HSCs were strongly exemplified by many TE elements at the subfamily and locus levels in groups such as the MurERV40int and RLTR10 from the ERV class. Furthermore, we revealed a positive correlation between TE expression and inflammatory genes, suggesting the potential involvement of TEs in regulating intracellular innate immune response during aging. In addition to modulating the age-related changes in TEs, loss of TET2 mitigated age-associated chromatin opening and expression of age-related genes such as Stxbp4 and S100a6 (Svendsen et al., Blood, 2021). Additionally, a regulon-based analysis showed age-related decreases in regulons associated with HSC function and maintenance such as Meis1 (Kocabas, et al. Blood, 2012), this age-related decrease was mitigated in TET2-KO. Parallel analysis of aging human HSPC datasets uncovered similar gene and TE activation patterns, suggesting a conserved mechanism (Hua et al., Blood, 2019).
 Beyond extrinsic inflammation, age-related TE upregulation may trigger innate immune signaling that impairs HSPC self-renewal and specification. Our multi-omics approach elucidates the epigenetic and transcriptional mechanisms underlying TE dysregulation in aging HSPCs. Our study provides new insights into the molecular underpinnings of age-related hematopoietic dysfunction and identified potential therapeutic targets for mitigating CH and associated diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca589ad4135ba9b576c77a6dbfe352981eb6d90e" target='_blank'>
              Single-cell multi-omics uncovers transposable element regulation in hematopoietic stem and progenitor cells during aging
              </a>
            </td>
          <td>
            Logan Rivera, Tingting Hong, Y. Huang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38954ab0551e2a9866f6ea57105246fb32735740" target='_blank'>
              CellSNVReg: a multidimensional resource for SNV-mediated regulatory perturbations in single-cell and spatial omics.
              </a>
            </td>
          <td>
            Jingyi Shi, Shiqian Zhang, Zhenglin Lu, Weiwei Zhou, Weijian Lu, Gengdong Chen, Jianing Li, Wenhui Liu, Jiaqi Huang, Tiantongfei Jiang, Xiyun Jin, Juan Xu, Tingting Shao
          </td>
          <td>2025-10-31</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Children with Down syndrome are predisposed to hematological malignancies due to the presence of an extra copy of chromosome 21. Trisomy 21 (T21) alone has been shown to disrupt fetal hematopoiesis. Further examination identified that the Down syndrome critical region (DSCR) on chromosome 21 plays a key role in T21-driven leukemogenesis due to gene dosage imbalances. DSCR harbors several genes essential for hematopoietic development and leukemogenesis, such as DYRK1A, one of the dosage sensitive genes. As a dual-specificity kinase, DYRK1A plays diverse roles in cell cycle progression, stem cell differentiation, and DNA damage response. Interestingly, DYRK1A has been reported to interact with RNA polymerase II at active promoters, which shows a potential role in transcriptional regulation. DYRK1A overexpression leads to disrupted hematopoiesis, yet its distinct role in ML-DS remains unclear.
 In this study we showed DYRK1A as a novel interactor of EZH2 in CMK, a ML-DS cell line, and 3 distinct relapsed ML-DS PDX samples by co-immunoprecipitation. Given that both proteins are known to function in cytoplasmic and nuclear compartments, we performed co-immunoprecipitation in these subcellular fractions and found that the interaction only occurs in the nuclear fraction in CMK and ML-DS PDX cells. Considering the canonical role of EZH2 as a methyltransferase and of DYRK1A as a dual-specificity kinase, we examined if their canonical roles are necessary for this interaction. Therefore, we treated cells with EZH2 inhibitor (GSK126) or DYRK1A inhibitor (EHT1610) and performed co-immunoprecipitation. The data demonstrated that the interaction between DYRK1A and EZH2 was maintained even in the presence of inhibitors which suggest that the interaction between these two proteins is not dependent on either methyltransferase or kinase activity.
 Since DYRK1A has been implicated in transcriptional regulation and we showed that it interacts with EZH2 in the nucleus only, we hypothesized that DYRK1A may collaborate with EZH2 to shape the chromatin landscape in ML-DS. To explore this possibility, we performed CUT&RUN chromatin profiling of EZH2, DYRK1A, H3K27me3, and H3K4me3 in CMK cells. CUT&RUN using anti-DYRK1A antibody identified 12,348 peaks (p <0.001). These peaks partially overlapped with the H3K4me3 peaks, but they were not enriched by H3K27me3 peaks. Overlapping DYRK1A peaks with EZH2 peaks revealed 186 regions occupied by both, and these regions are highly enriched by H3K4me3, suggesting active gene expression. Pathway enrichment analysis of DYRK1A and EZH2 co-occupied regions indicated enrichment in key signal transduction pathways, including MAPK, ERK, RAS, and neurotrophin (NTRK) signaling (p < 0.05).
 To further investigate the function of EZH2 and DYRK1A in ML-DS, we depleted the expression of EZH2 or DYRK1A in CMK cells by CRISPR-cas9 mediated knockout. While depletion of EZH2 resulted in decreased protein level of DYRK1A in the nucleus, it had no effect on cytoplasmic DYRK1A. Interestingly, the downregulation of DYRK1A protein did not affect the level of EZH2 in both fractions. These data indicate that DYRK1A requires EZH2 for its nuclear localization and likely regulates expression of specific genes required for leukemogenesis.
 Together, these findings suggest that DYRK1A and EZH2 may functionally cooperate to regulate gene expression at transcriptionally active regions in ML-DS cells, especially within key signal transduction networks. This interaction may represent a previously unrecognized mechanism contributing to leukemogenesis in T21, with potential implications for targeting non-canonical EZH2 functions in ML-DS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcb96ecddff82ebd51607a31bc1f76845786142" target='_blank'>
              DYRK1A interacts with EZH2 to regulate transcriptionally active chromatin in myeloid leukemia associated with down syndrome
              </a>
            </td>
          <td>
            Kader Cicek, S. Barwe, Anilkumar Gopalakrishnapillai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract Background The human mitochondrial genome (mtDNA) plays a key role in maintaining cellular functions. However, mtDNA is vulnerable to mutations due to the lack of protective histones and low efficiency in DNA repair mechanisms. Among these mutations, the 4977 bp mtDNA common deletion (mtDNA-CD) encompasses twelve essential genes that are essential for oxidative phosphorylation. Aim In previous studies, mtDNA-CD was identified in nasopharyngeal carcinoma (NPC) patient samples. However, bulk sequencing protocols employed in past studies were unable to distinguish whether those mtDNA-CD were specific to the cancer cells or from other non-cancer cell types in the samples. Single-cell RNA sequencing (scRNA-seq) enables the discovery of different cell types within a population and facilitates the identification of cell-type-specific mutation signatures. In this study, we propose to use scRNA-seq data to identify cell-type-specific deletion signatures in NPC samples. Methods We developed a custom computational pipeline to detect large-scale deletions using scRNA-seq data. The pipeline harnesses both clipped read profiles and secondary alignments to overcome the typical limitations of low sequencing depth and uneven read coverage in scRNA-seq data. We tested our pipeline on an NPC data set consisting of nine patients. Results Our result suggests the presence of mtDNA-CD mutations specific to malignant cancer cells in NPC tumour samples. This study proposes the use of scRNA-seq of cell-type-specific mtDNA-CD in NPC samples, offering new insights into the genetic and molecular pathogenesis of cancer and guiding the development of specific biomarkers for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9ee5d5ecd8271f6680610b609d3e9e7cc092eb" target='_blank'>
              Discovery of cell-type-specific mtDNA common deletion in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Mun Kay Ho, J. W. Ho
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification has emerged as a critical epigenetic mechanism regulating gene expression in diverse physiological and pathological processes, including cancer. Methyltransferase-like 16 (METTL16), a recently identified m6A methyltransferase, has been shown to influence tumor progression through m6A-dependent regulation of key target genes. Accumulating evidence indicates that METTL16 exerts tumor-suppressive or tumor-promoting roles in a context-dependent manner, affecting cell proliferation, apoptosis, autophagy, and chemotherapeutic response across multiple cancer types such as bladder cancer, lung cancer, colorectal cancer, and acute myeloid leukemia. Mechanistically, METTL16 modifies the mRNA stability and translation of oncogenes or tumor suppressors via recognition of m6A sites, and its expression can be regulated by upstream factors including transcription factors and hypoxia-inducible signals. Recent evidence suggests that METTL16 also modulates the tumor microenvironment (TME), potentially affecting immune cell infiltration, immune checkpoint expression, and tumor immune evasion. Collectively, METTL16 emerges as a pivotal epitranscriptomic regulator linking RNA modification, tumor progression, and immune modulation, offering new avenues for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cac623df0454861cf59d7b80f15a7189f66d09" target='_blank'>
              METTL16 emerges as a pivotal epitranscriptomic regulator, linking RNA modification, tumor progression, and immune modulation
              </a>
            </td>
          <td>
            Qiang Wang, Xiulin Jiang, Yixiao Yuan, Chunhong Li
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Dysembryoplastic neuroepithelial tumor (DNT) is a rare glioneuronal tumor that typically arises in childhood and is strongly associated with drug-resistant epilepsy. Diagnosis is often challenging due to histological heterogeneity. Therefore, DNT may be an ideal candidate for genome-wide DNA methylation profiling. However, integrated analyses combining histopathology and genome-wide DNA methylation profiling remain limited.



 We retrospectively analyzed the clinical data, histopathological diagnoses, and molecular findings of patients diagnosed with DNT or low-grade glioneuronal tumors mimicking DNT at the Japan Children’s Cancer Group Central Diagnosis between February 2016 and March 2025.



 Twenty-three patients (12 males and 11 females) were included. The median age at the diagnosis was 11 years (range, 3 to 24 years). The majority were located in the temporal lobes (n=12) and other cortical regions (n=10), but a case was located in the third ventricle. Seizure onset was noted in 15 cases. FGFR1 tyrosine kinase domain internal tandem duplication was identified in 10 cases, and FGFR1 K656E mutation in 5 cases. Genome-wide DNA methylation profiling was available in 16 cases. According to the DKFZ classifier, 14 cases were classified as DNT, one as Pilocytic astrocytoma (PA), hemispheric, and one as Glioneuronal tumour, subtype A (novel). A case of PA, hemispheric was histopathologically diagnosed as DNT complex form and showed focal areas with PA morphology. Interestingly, a case of Glioneuronal tumour, subtype A (novel) showed atypical clinical features such as non-seizure onset and location in the third ventricle. It showed recurrence 5 years after an initial surgery with histopathological changes, including loss of specific glioneuronal elements and growth of multinucleated cells.



 Discrepancy between histopathological diagnosis and DNA methylation classification may result from varied cellular composition. In addition, atypical DNA methylation classification may be associated with clinical signatures, requiring integrated analyses in larger cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1982f5c51b122b194881a102d6ccebfd3f04b696" target='_blank'>
              PTHP-08. Integrated analysis of dysembryoplastic neuroepithelial tumors highlighting discrepancies between histopathological diagnosis and genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Yohei Inoue, Yoshihiro Otani, Joji Ishida, Akira Miyahira, Asuka Fujii, Yuko Hibiya, Y. Matsushita, Yoshiko Nakano, K. Satomi, Kohei Fukuoka, K. Yamasaki, Takako Yoshioka, Junko Hirato, S. Tanaka, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Studies by our lab have demonstrated that DNA Damage Repair (DDR) pathways play a key role in determining response to ICB. To better characterize the effect of these alternations on the tumor immune microenvironment (TME) and define key determinants of ICB response, we created isogenic knockouts of BRCA2 and BRCA1 in a murine 4T1 metastatic TNBC background to assess how these tumor-intrinsic programs influence the TME and poise tumors for immunotherapy response. Differential immune landscapes identified via scRNAseq in Brca2-mutant tumors at baseline showed key differences in the myeloid compartment and interferon-stimulated gene (ISG) expression in tumor-infiltrating myeloid cells from Brca2 mutant tumors. Bulk RNAseq analysis showed increased production of T cell trafficking chemokines that could poise the tumor for response to ICB. Interferon reporter assay results suggested tumor-intrinsic cGAS drives trans-activation of myeloid STING and IFNb1. Further assessment of tumor cell lines via cellular fractionation experiments identified the presence of both DNA and R-loops in the cytoplasm of BRCA2-mutant cell lines that serve to active cGAS/STING. In vivo experiments with tumor intrinsic knockouts showed that tumor STING signaling and cytokine production were dispensable for ICB response while depletion of monocytes via CSF1R blockade showed a complete reversal of ICB response, underlining the essential role of monocytes in poising the tumor for ICB response.



 Supported by NIH Director’s Early Independence Award, the Parker Institute for Cancer Immunotherapy, Burrough’s Welcome Fund, and Mount Sinai’s Cancer Biology T32 Training Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f9189d19bc8a0280db2d343d742431c6842512" target='_blank'>
              Defining the determinants of immune response in DNA homologous recombination deficient tumors 3797
              </a>
            </td>
          <td>
            Natalie Vaninov, R. Samstein
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Introduction Recently, it has been suggested that Tumor Mutation Burden (TMB) for solid tumors is associated with life expectancy after immunotherapy. It is also known that there is a correlation between TMB in gene panels and whole exon analysis. In this preliminary study, we investigated whether the number of tumor gene mutations on the gene panel is associated with prognosis in a small number of patients with initially diagnosed glioblastoma treated with immunotherapy (formalin-fixed autologous tumor vaccine, AFTV). Methods Twenty-seven patients with glioblastoma, 13 in the immunotherapy group and 14 in the control group including most patients with good prognosis, underwent gene cancer panel testing using a next-generation sequencer to identify the presence of 275 known oncogene deletions (deletions) and SNPs (classified as probable damaged, possibly damaged, or benign) were investigated and their association with survival was examined. In 22 representative genes, the number of damaged SNPs/deletion were also analyzed. Results An average of 80 tumor-associated genes were mutated in 275 glioblastoma cases, including 77, 65, and 104 genes in the P53 wildtype (-W), P53-B (benign SNP), and P53-D (probably/possibly damaged SNP or deletion) groups, respectively. The P53-W/B and P53-D groups showed statistically significant differences (p = 0.021), with 77, 65, and 104 genes, respectively. The median survival of the H and L groups was 37 and 21 months, respectively, whereas that of the AFTV group was 15 and 33 months, respectively (no significant difference in either case). The similar results were observed for the analysis of the number of damaged SNPs/deletions in 22 genes. Conclusion The results suggest that, at least for AFTV therapy, a high number of mutations may not be a favorable prognostic factor. The significance of this finding will be discussed in the presentation, including a literature review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fea719b8bf58ee7c937bc6f0c06cb591d2f9aa12" target='_blank'>
              10033-IM-1Association between the number of gene mutations in an oncogene panel and outcome for initially diagnosed glioblastoma treated with autologous tumor vaccine
              </a>
            </td>
          <td>
            Eiichi Ishikawa, Erika Yamada, Shunichiro Miki, N. Sugii, Y. Uemae
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>