<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-10 09:26:16 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>312</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>267</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H. Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Studying the functional consequences of structural variants (SVs) in mammalian genomes is challenging because (i) SVs arise much less commonly than single-nucleotide variants or small indels and (ii) methods to generate, map, and characterize SVs in model systems are underdeveloped. To address these challenges, we developed Genome-Shuffle-seq, a method that enables the multiplex generation and mapping of thousands of SVs (deletions, inversions, translocations, and extrachromosomal circles) throughout mammalian genomes. We also demonstrate the co-capture of SV identity with single-cell transcriptomes, facilitating the measurement of SV impact on gene expression. We anticipate that Genome-Shuffle-seq will be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and three-dimensional nuclear architecture, while also initiating a path toward a minimal mammalian genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd57d0a97983513ebb6d083a8255399d3304d8d" target='_blank'>
              Multiplex generation and single-cell analysis of structural variants in mammalian genomes.
              </a>
            </td>
          <td>
            Sudarshan Pinglay, Jean-Benoît Lalanne, R. Daza, Sanjay Kottapalli, Faaiz Quaisar, Jonas Koeppel, Riddhiman K. Garge, Xiaoyi Li, David S Lee, Jay Shendure
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6c622797b8d89bba676a8b9a3206b6fe5c1df9" target='_blank'>
              A constitutive heterochromatic region shapes genome organization and impacts gene expression in Neurospora crassa
              </a>
            </td>
          <td>
            Andrew T. Reckard, Abhishek Pandeya, Jacob M. Voris, Carlos G. Gonzalez Cruz, Oluwatosin Oluwadare, Andrew D. Klocko
          </td>
          <td>2024-12-20</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Histone deacetylase inhibitors have been investigated as potential therapeutic agents for cancer and other diseases. HDIs are known to promote histone acetylation, resulting in an open chromatin conformation and generally increased gene expression. In previous work, we reported that a subset of genes, particularly those regulated by superenhancers, can be suppressed by the HDAC inhibitor largazole. To elucidate the molecular mechanisms underlying gene repression by largazole, we conducted transposase-accessible chromatin sequencing, ChIP-seq, and RNA-seq studies. Our findings revealed that while largazole treatment generally enhances chromatin accessibility, it selectively decreases the accessibility of a subset of superenhancer regions. These genomic regions, showing the most significant changes in the presence of largazole, were enriched with transcription factor binding motifs for SP1, BRD4, CTCF, and YY1. ChIP-seq analysis confirmed reduced binding of BRD4 and SP1 at their respective sites on chromatin, particularly at superenhancers regulating genes such as ID1, c-Myc and MCMs. Largazole exerts its effects by inhibiting DNA replication, RNA processing, and cell cycle progression, partially through the suppression of SP1 expression. Depletion of SP1 by shRNA mimics several key biological effects of largazole and increases cellular sensitivity to the drug. Specific to cell cycle regulation, we demonstrated that largazole disrupts G/M transition by interfering with chromosome alignment during metaphase, a phenotype also observed with SP1 depletion. Our results suggest that largazole exerts its growth-inhibitory effect by suppressing BRD4 and SP1 at super-enhancers, leading to cytostatic responses and mitotic dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec8d5c07c7619739594c8403fd089fe802e50e6" target='_blank'>
              Histone Deacetylase Inhibitor Largazole Deactivates A Subset of Superenchancers and Causes Mitotic Chromosome Mis-alignment by Suppressing SP1 and BRD4
              </a>
            </td>
          <td>
            Gilson J Sanchez, Zeyu Liu, Samuel Hunter, Quanbin Xu, Jessica Westfall, Graycen E. Wheeler, Cathryn M Toomey, Dylan Taatjes, Mary A. Allen, Robin D. Dowell, Xuedong Liu
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RNase H2 is a heterotrimeric endoribonuclease that resolves RNA:DNA hybrids and genome-embedded ribonucleotides, which are implicated in DNA replication stress and cancer development. Protein and/or mRNA levels of individual RNase H2 subunits are elevated in some cancers, but little is known about the mechanisms or consequences of RNase H2 upregulation. We report that RNase H2 subunits are upregulated at the protein level in response to replication stress induced by oncogenes and chemotherapy drugs in human cancer and non-cancer cell lines. We show that inducible overexpression of the RNASEH2B subunit increases levels of the active RNase H2 heterotrimer. While causing only subtle changes to gene expression, RNASEH2B overexpression is unexpectedly associated with increased RNA:DNA hybrid levels. RNASEH2B overexpression prevents further increases in RNA:DNA hybrid levels by camptothecin or hydroxyurea and reduces replication fork stalling in presence of these drugs. Surprisingly, RNase H2 levels do not strongly impact survival of chemotherapy treatments but appear to have more subtle effects on genome instability and innate immune signalling. In contrast, increased RNase H2 levels in presence of oncogenic HRAS limit not only RAS-induced replication fork stalling but also cell death. Our findings shed new light on the functions of RNase H2 and suggest that upregulation of RNase H2 may be an important aspect of replication stress responses in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54af3146eb7acac3bc5186f9ac5d0287ea7b019a" target='_blank'>
              Human RNase H2 upregulation counteracts oncogene- and chemotherapy-induced replication stress
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, Abirami Kannan, Siobhan A. Plass, Claire Wilson, Richard D. W. Kelly, Claire H. M. Tang, P. Kotsantis, Martin A. M. Reijns, Aditi Kanhere, Eva Petermann
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyu Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="All great apes differ karyotypically from humans due to the fusion of chromosomes 2a and 2b, resulting in human chromosome 2. Yet, the structure, function, and evolutionary history of the genomic regions associated with this fusion remain poorly understood. Here, we analyze finished telomere-to-telomere chromosomes in great apes and macaques to show that the fusion was associated with multiple pericentric inversions, segmental duplications (SDs), and the rapid turnover of subterminal repetitive DNA. We characterized the fusion site at single-base-pair resolution and identified three distinct SDs that originated more than 5 million years ago. These three distinct SDs were differentially distributed among African great apes as a result of incomplete lineage sorting (ILS) and lineage-specific duplication. Most conspicuously, one of these SDs shares homology to a hypomethylated SD spacer sequence present in hundreds of copies in the subterminal heterochromatin of chimpanzees and bonobos. The fusion in human was accompanied by a systematic degradation of the three divergent α-satellite arrays representing the ancestral centromere creating five distinct structural haplotypes in humans. CRISPR/Cas9-mediated depletion of the fusion site in human cell lines significantly alters the expression of 108 genes, indicating a potential regulatory consequence to this human-specific karyotypic change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b16610644425427f06758c18d45678c36f10190" target='_blank'>
              Incomplete lineage sorting of segmental duplications defines the human chromosome 2 fusion site early during African great ape speciation
              </a>
            </td>
          <td>
            Xinrui Jiang, Lu Zhang, Zikun Yang, Xiangyu Yang, Kaiyue Ma, DongAhn Yoo, Yong Lu, Shilong Zhang, Jieyi Chen, Yanhong Nie, Xinyan Bian, Junmin Han, Lianting Fu, Juan Zhang, Guojie Zhang, Qiang Sun, E. Eichler, Yafei Mao
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6960e9d1d69ddbed72b6eb04c8ec6b5d4992a9b7" target='_blank'>
              Structural polymorphism and diversity of human segmental duplications.
              </a>
            </td>
          <td>
            Hyeonsoo Jeong, Philip C. Dishuck, DongAhn Yoo, William T. Harvey, Katherine M. Munson, Alexandra P. Lewis, Jennifer R. Kordosky, Gage H. Garcia, F. Yilmaz, P. Hallast, Charles Lee, T. Pastinen, E. Eichler
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="We lack tools to edit DNA sequences at scales necessary to study 99% of the human genome that is noncoding. To address this gap, we applied CRISPR prime editing to insert recombination handles into repetitive sequences, up to 1697 per cell line, which enables generating large-scale deletions, inversions, translocations, and circular DNA. Recombinase induction produced more than 100 stochastic megabase-sized rearrangements in each cell. We tracked these rearrangements over time to measure selection pressures, finding a preference for shorter variants that avoided essential genes. We characterized 29 clones with multiple rearrangements, finding an impact of deletions on expression of genes in the variant but not on nearby genes. This genome-scrambling strategy enables large deletions, sequence relocations, and the insertion of regulatory elements to explore genome dispensability and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a90f87137d367e42fa5bf7ac26c12b1be111991" target='_blank'>
              Randomizing the human genome by engineering recombination between repeat elements.
              </a>
            </td>
          <td>
            Jonas Koeppel, Raphael Ferreira, T. Vanderstichele, Lisa M. Riedmayr, Elin Madli Peets, Gareth Girling, Juliane Weller, Pierre Murat, Fabio Giuseppe Liberante, Tom Ellis, George M. Church, Leopold Parts
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="Activation of synthetic centromeres on chromosome 4 in maize leads to its breakage and formation of trisomic fragments called neochromosomes. A limitation of neochromosomes is their low and unpredictable transmission rates due to trisomy. Here we report that selecting for dicentric recombinants through male crosses uncovers stabilized chromosome 4 fission events which split it into 4a-4b complementary chromosome pairs, where 4a carries a native centromere and 4b a synthetic one. The cells rapidly stabilized chromosome ends by de novo telomere formation and the new centromeres spread among genes without altering their expression. When both 4a and 4b chromosomes were present in a homozygous state, they segregated through meiosis indistinguishably from wild-type, and gave rise to healthy plants with normal seed set. This work leverages synthetic centromeres to engineer chromosome fission, effectively raising the diploid chromosome number of maize from 20 to 22. TEASER Using synthetic centromeres, we divide one maize chromosome into two chromosomes, and show that both chromosomes are fully functional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 DNA methylation alterations are observed across multiple cancers, occurring early during tumor initiation. However, evolution and drivers of these changes remain poorly understood and difficult to assess at the clonal level. Consequently, we developed a method for profiling genome-wide DNA methylation from low DNA amounts and applied it to the organoid model of BRAFV600E oncogene-induced colon cancer (CC) tumorigenesis. Single organoid clones, derived from single stem cells, were profiled every month for 5 months to map nucleotide-level dynamic heterogeneity in methylation changes. Uninduced control organoids exhibit a pattern of epigenetic drift leading towards clonal convergence not until 5 months. Whereas BRAFV600E-induced organoids exhibit a step-wise divergence throughout the different time points, likely explained by progressive clonal takeovers. The methylation changes in BRAFV600E induced organoids are explained by the age-in-culture associated methylation changes; especially BRAFV600E did not induce epigenetic changes, as previously thought. The CpG sites affected in the control and BRAFV600E comparison were not shared but mapped to the same genomic elements, indicating common drivers of methylation changes regardless of oncogene. Investigation of promoter CpG-island methylation gains revealed that they are decoupled from replication. External validation in other pre-cancerous and CC datasets, and experimental cell line data sets, show that methylation gains at regulatory elements are a function of “biological aging” and independent of replication or cell division cycles. Our studies provide new insights into the early epigenetic changes involved in BRAFV600E driven carcinogenesis rooted in the age-dependent drivers of epigenetic alterations.
 Citation Format: Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, Thomas R. Pisanic II, Stephen B Baylin, Hariharan Easwaran. Promoter methylation gains in aging and cancer are independent of replication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Centromere location is specified by CENP-A, a centromere-specific histone that epigenetically propagates centromere identity. How CENP-A is epigenetically maintained at one location in rapidly evolving centromeric DNA is unknown. Using single cell-derived clones of human cell lines, we demonstrate heterogeneity in CENP-A position within cell populations at neocentromeres and a native centromere. CENP-A heterogeneity is accompanied by heterogenous DNA methylation and H3K9me3 patterns, with DNA methylation shifting according to CENP-A position. We further reveal temporary precise CENP-A maintenance but evolution of CENP-A, DNA methylation and H3K9me3 position over prolonged proliferation, with neocentromeres exhibiting DNA methylation instability, H3K9me3 gain, boundary loss and increased epigenetic heterogeneity. Lastly, prolonged CENP-A and HJURP overexpression leads to neocentromere expansion, gradual CENP-A depletion, neocentromere destabilization and re-localization. This study reveals the evolving epigenetic plasticity of human centromeres and neocentromeres and highlights the importance of repressive chromatin boundaries in maintaining centromere stability across cellular proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37294bb2928f0f4df70a33eed7a5c2638b0c5df2" target='_blank'>
              Epigenetically dynamic human centromeres are maintained within a stable DNA methylation signature
              </a>
            </td>
          <td>
            Megan A. Mahlke, Lior Lumerman, Poulomi Nath, Cy Chittenden, Savannah J. Hoyt, Jonas Koeppel, Yuan Xu, Rebecca Raphael, Kylie Zaffina, P. Hook, W. Timp, K. Miga, Peter J. Campbell, Rachel J. O’Neill, Nicolas Altemose, Yael Nechemia-Arbely
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal inversions play a crucial role in evolution by influencing phenotypes through the linkage of co-adapted alleles. While inversions have been found across a large number of taxa, mapping and characterizing inversion breakpoint regions remains challenging, often due to the presence of complex tandem repeats and transposable elements (TEs). Here, we identify and quantify TEs in the breakpoints of the four large-scale inversions previously reported in Atlantic cod, leveraging on three high-quality long-read-based reference genome assemblies for the Norwegian Coastal cod, the Northeast Arctic cod and Celtic cod ecotypes. We detected a significant enrichment of TE orders and superfamilies with terminal inverted repeats (TIRs) within the inversion breakpoints of chromosomes 1, 7 and 12. Notably, we discovered a tandem accumulation of miniature inverted-repeat transposable elements (MITEs) belonging to a family of hAT transposons, exclusively residing in the breakpoints of the inverted haplotype on chromosomes 1 and 7 found in the Northeast Arctic cod. The accumulation of tandemly arranged TEs in breakpoint regions suggests that they have driven the appearance of inversions through ectopic recombination, further supporting the potential of TEs in facilitating chromosomal reorganizations with large evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c69b85f1eed1a39a185be57ad812eafc541ce3b" target='_blank'>
              Chromosomal inversions mediated by tandem insertions of transposable elements
              </a>
            </td>
          <td>
            Robin Aasegg Araya, William B. Reinar, O. K. Tørresen, C. Goubert, Tara J Daughton, S. Hoff, H. T. Baalsrud, Marine Servane Ono Brieuc, Anna Komisarczuk, S. Jentoft, José Cerca, K.S. Jakobsen
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Rearrangements associated with gene amplification are frequently initiated by the formation of inverted repeats. Among these rearrangements are breakage-fusion-bridge cycles and tandem inverted duplications (TIDs), both of which can be mediated through palindromic sequences. We have investigated spontaneous rearrangements within the genome of the filamentous fungus Aspergillus flavus by high-coverage third-generation sequencing of vegetatively cultured strains. Analysis of sequence data identified a class of rare and transient rearrangements having structures typical of TIDs. These putative TIDs (pTIDs) form both unprocessed and processed species and have a variably sized deletion at the junction between direct and inverted sequences. Most intact junctions contain the predicted palindromic sequences, supporting the TID model. However, palindrome-free junctions are also infrequently observed. pTIDs are highly enriched in heterochromatic centromeric and subtelomeric A+T rich regions, suggesting that this process is a source of genetic instability in these domains. Consistent with this finding, A+T rich regions contain elevated levels of direct, inverted, and perfect palindromic repeats. We propose a replication-based model in which palindromes and short inverted repeats, under condition of replication stress, are responsible for pTID structures. TEASER The dynamic formation of duplication/inversions clustered in fungal chromosomes reveals the genetic instability of centromeric and subtelomeric regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728781bd2be224742eb5106c8451f24b72cd7b56" target='_blank'>
              Clustering of Tandem Inverse Duplications in Centromeric and Subtelomeric Chromosomal Regions of Aspergillus flavus
              </a>
            </td>
          <td>
            Jeffrey W. Cary, Amna Malik, Alexie Smith, Erik K Flemington, Arthur J Lustig
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Embryonal Tumors with Multilayered Rosettes (ETMRs) are relatively rare but aggressive embryonal brain tumors that mainly affect children under the age of 4 and which have a poor outcome. Around 90% of ETMRs harbor amplification of the C19MC miRNA cluster, in most cases fused to the TTHY1 gene. The second most recurrent aberration, present in about half of the C19MC negative cases, are bi-allelic mutations in DICER1. ETMRs are also characterized by overall lower DNA methylation levels and high LIN28A gene expression as compared with other embryonal brain tumors like medulloblastomas (MB) and atypical teratoid rhabdoid tumors (ATRT). However, a more detailed description of the epigenetic landscape marked by DNA methylation and histone modifications, and how this affects the ETMR transcriptome is still lacking. To decode the epi-genome, we have collected whole genome bisulfite sequencing (WGBS) data, ChIP sequencing (ChIP-seq) data of various (active and repressive) histone marks and epigenetic modifiers (EZH2, BRD4, CTCF), and RNA sequencing (RNA-seq) data from a series of ETMR patient tumors, and similar data from a series of MBs and ATRTs covering all molecular subtypes for comparison reasons. Using all these data and performing bioinformatic analyses, we have depicted chromatin states and DNA-methylation-defined genomic segments of ETMR for the first time. ETMR genomes are generally hypomethylated, with wider distributions of un-methylated regions and DNA methylation valleys (DMVs) on its genome compared with MB and ATRT. By integrating these data with all other omics data, we found that BRD4-regulated overexpression of TET demethylases may cause global hypomethylation in ETMRs. Polycomb and TETs are recruited at DMVs to ensure DMVs are longer in ETMRs than MBs and ATRTs. Furthermore, genes involved in embryo development and cell proliferation are enriched in entity-shared DMVs and are highly expressed in ETMRs and other embryonal brain tumors. We also depicted that TET family members, BRD4, and Polycomb complex (EZH2) are potential therapeutic targets for ETMR patients. In conclusion, we have partially described the epi-genome and epigenetic regulations of ETMR, and are currently working on more detailed regulation axes and performing drug test experiments for these potential therapeutic targets to understand better the underlying biology of these tumors and how this may lead to alternative and more effective therapies.
 Citation Format: Shanzheng Wang, Sander Lambo, Monika Mauermann, Robert J Autry, Natalie Jäger, Stefan M Pfister, Marcel Kool. Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR) [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0e28f4d695edfbb9dadc1b324a1b929e51f3786" target='_blank'>
              Abstract B028: Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR)
              </a>
            </td>
          <td>
            Shanzheng Wang, S. Lambo, Monika Mauermann, Robert J Autry, N. Jäger, Stefan M. Pfister, M. Kool
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff76f6b675fe3b53c1d6072d58b13f856a3bdc6" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Significance Mutations in the metabolic enzyme isocitrate dehydrogenase (IDH1) are frequent and considered early events in a subset of adult gliomas. The mutant IDH1 enzyme changes functionally, favoring the production of the oncometabolite 2-hydroxyglutarate, and alters metabolic and epigenetic pathways. However, a comprehensive understanding of the epigenetic landscape in IDH1mut cells and how it contributes to tumorigenesis is lacking. This study presents a high-dimensional single-cell analysis of chromatin alterations driven by mutant-IDH1. Our analysis reveals extensive remodeling of chromatin patterns, highlighting a rapid decrease in histone acetylation marks and increased epigenetic heterogeneity. These changes in histone acetylation, affecting enhancer patterns, are strongly associated with the metabolic abnormalities of IDH1mut cells, underscoring the interplay between chromatin and metabolism dysregulation in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1395bf17f297913f9bbe2a4e1db052179fa9f3f" target='_blank'>
              Oncogenic IDH1mut drives robust loss of histone acetylation and increases chromatin heterogeneity
              </a>
            </td>
          <td>
            N. Furth, Niv Cohen, Avishay Spitzer, T. Salame, B. Dassa, Tevie Mehlman, Alexander Brandis, Arieh Moussaieff, D. Friedmann-Morvinski, Maria G Castro, Jerome Fortin, M. Suvà, I. Tirosh, A. Erez, Guy Ron, E. Shema
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background The understanding of genetic and epigenetic regulation of gene expression in endothelial and vascular smooth muscle cells remains fragmented with limited experimental validation. Methods Chromatin conformation (Micro-C), DNA methylation (RRBS), chromatin accessibility (ATAC-seq), and transcriptome profiles (RNA-seq) were mapped in human induced pluripotent stem cell (hiPSC)-derived, isogenic endothelial and vascular smooth muscle cells (iECs and iVSMCs). CTCF and RAD21 were depleted to assess the functional relevance of chromatin architecture, and genome editing was used to evaluate the allelic effect of a blood pressure-associated single nucleotide polymorphism (SNP). Results Significant correlations were identified between gene expression levels and chromatin interactions, chromatin accessibility, and DNA methylation in iECs and iVSMCs, with chromatin interactions showing the strongest association. Chromatin contact regions displayed distinct epigenetic landscapes depending on the types of regulatory element interactions involved. Perturbation of CTCF and RAD21 revealed their differential regulatory effects, particularly on the expression of genes overlapping chromatin contacts, with RAD21 exhibiting a broader regulatory impact. SNPs associated with several vascular traits were enriched in chromatin loops or accessible regions in iECs or iVSMCs. Precise genome editing demonstrated allele-dependent effects of SNP rs9833313 on the expression of SHOX2 located 247.4 kbp away but within the same chromatin loops as the SNP. Conclusion This study provides an extensive epigenetic landscape of vascular cells that may drive novel research on the role of genetic and epigenetic mechanisms of vascular function and disease as demonstrated by our targeted experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cca08a65eb2683798ce82f8cb05c4cd45763cf1" target='_blank'>
              Global and genetic regulation of gene expression in human endothelial and vascular smooth muscle cells
              </a>
            </td>
          <td>
            Pengyuan Liu, Atrayee Ray, Yong Liu, Manoj K. Mishra, Qiongzi Qiu, Rajan Pandey, Bhavika Therani, Jing Huang, Jiayi Ren, C. Stelloh, Xiaowen Bai, Andrew S Greene, Allen W Cowley, Aron M. Geurts, Sridhar Rao, Mingyu Liang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Matvey Mikhailovich Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, Sergey Viktorovich Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="TAL1 is overexpressed in 40-60% of T-cell acute lymphoblastic leukemia (T-ALL) cases and forms an oncogenic core regulatory circuit (CRC) with other transcription factors such as LMO1, LMO2 and GATA3. In 5% of T-ALL cases an insertion of a consensus GT dinucleotide (MuTE) is observed upstream of the TAL1 gene, driving TAL1 overexpression. Using an in vitro reconstitution DNA pull-down assay combined with quantitative mass spectrometry, we identified proteins that preferentially bound to the MuTE sequence and demonstrated that among the candidates the RNA methyltransferase TARBP1 and the zinc finger proteins ZBTB2, ZBTB25 and ZNF639 form a complex that we term TARZN. Interestingly, the TARZN complex also bound to de novo super enhancer sites upstream of the LMO1 and LMO2 genes in T-ALL cells, indicating a putative common mechanism between these different non-coding driver mutations. Furthermore, knock-down of all TARZN members resulted in lower TAL1 protein expression in MuTE-positive but not in MuTE-negative T-ALL cells. Given TARZN’s methyltransferase activity and the lack of concomitant TAL1 mRNA level changes, we investigated reduced TAL1 translation and identified reduced neo-synthesised TAL1 protein levels upon TARBP1 knockdown. Overall, these data suggest that the TARZN complex promotes oncogenic expression in T-ALL via co-transcriptional RNA methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3cc9fd0c83ba3cc4d7a530bb55531a564291e6" target='_blank'>
              The TARZN complex binds de novo enhancer mutations and promotes oncogenic expression in T-ALL
              </a>
            </td>
          <td>
            Nurkaiyisah Zaal Anuar, C. Goh, Boon Haow Chua, Shi Hao Tan, Joana R Costa, Marc R. Mansour, Takaomi Sanda, Dennis Kappei
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The six subunit Origin Recognition Complex (ORC) is a DNA replication initiator that also promotes heterochromatinization in some species. A multi-omics study in a human cell line with mutations in three subunits of ORC, reveals that the subunits bind to DNA independent of each other rather than as part of a common six-subunit ORC. While DNA-bound ORC2 was seen to compact chromatin and attract repressive histone marks, the activation of chromatin and protection from repressive marks was seen at a large number of sites. The epigenetic changes regulate hundreds of genes, including some epigenetic regulators, adding an indirect mechanism by which ORC2 regulates epigenetics without local binding. DNA-bound ORC2 also prevents the acquisition of CTCF at focal sites in the genome to regulate chromatin loops. Thus, individual ORC subunits are major regulators, in both directions, of epigenetics, gene expression and chromosome structure, independent of the role of ORC in replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f78b4b4e51086a34cc8aa6b1896dde6fc4ca7f3" target='_blank'>
              Regulation of epigenetics and chromosome structure by human ORC2
              </a>
            </td>
          <td>
            Zhangli Su, Mengxue Tian, Etsuko Shibata, Y. Shibata, Tianyi Yang, Zhenjia Wang, Fulai Jin, C. Zang, A. Dutta
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Copy number variants (CNVs) are an important source of genetic variation underlying rapid adaptation and genome evolution. Whereas point mutation rates vary with genomic location and local DNA features, the role of genome architecture in the formation and evolutionary dynamics of CNVs is poorly understood. Previously, we found the GAP1 gene in Saccharomyces cerevisiae undergoes frequent amplification and selection in glutamine-limitation. The gene is flanked by two long terminal repeats (LTRs) and proximate to an origin of DNA replication (autonomously replicating sequence, ARS), which likely promote rapid GAP1 CNV formation. To test the role of these genomic elements on CNV-mediated adaptive evolution, we evolved engineered strains lacking either the adjacent LTRs, ARS, or all elements in glutamine-limited chemostats. Using a CNV reporter system and neural network simulation-based inference (nnSBI) we quantified the formation rate and fitness effect of CNVs for each strain. Removal of local DNA elements significantly impacts the fitness effect of GAP1 CNVs and the rate of adaptation. In 177 CNV lineages, across all four strains, between 26% and 80% of all GAP1 CNVs are mediated by Origin Dependent Inverted Repeat Amplification (ODIRA) which results from template switching between the leading and lagging strand during DNA synthesis. In the absence of the local ARS, distal ones mediate CNV formation via ODIRA. In the absence of local LTRs, homologous recombination can mediate gene amplification following de novo retrotransposon events. Our study reveals that template switching during DNA replication is a prevalent source of adaptive CNVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488ecf1ac91854034846373133d03867e6680f28" target='_blank'>
              Template switching during DNA replication is a prevalent source of adaptive gene amplification
              </a>
            </td>
          <td>
            Julie N. Chuong, Nadav Ben Nun, Ina Suresh, J. Matthews, Titir De, Grace Avecilla, Farah Abdul-Rahman, Nathan Brandt, Yoav Ram, David Gresham
          </td>
          <td>2025-02-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="During classical non-homologous end joining (cNHEJ), DNA-dependent protein kinase (DNA-PK) encapsulates free DNA ends, forming a recruitment platform for downstream end-joining factors including Ligase 4 (LIG4)1. DNA-PK can also bind telomeres and regulate their resection2–4, but does not initiate cNHEJ at this position. How the end joining process is regulated in this context-specific manner is currently unclear. Here we show that the shelterin components TRF2 and RAP1 form a complex with DNA-PK that directly represses its end joining function at telomeres. Biochemical experiments and cryo-electron microscopy reveal that when bound to TRF2, RAP1 establishes a network of interactions with KU and DNA that prevents DNA-PK from recruiting LIG4. In mouse and human cells, RAP1 is redundant with the Apollo nuclease in repressing cNHEJ at chromosome ends, demonstrating that the inhibition of DNA-PK prevents telomere fusions in parallel with overhang-dependent mechanisms. Our experiments show that the end joining function of DNA-PK is directly and specifically repressed at telomeres, establishing a molecular mechanism for how individual linear chromosomes are maintained in mammalian cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797e7286a7a007c6d465328b33c7892e6e693028" target='_blank'>
              Chromosome end protection by RAP1-mediated inhibition of DNA-PK
              </a>
            </td>
          <td>
            Patrik Eickhoff, Ceylan Sonmez, Charlotte E.L. Fisher, Oviya Inian, Theodoros I. Roumeliotis, Angela dello Stritto, Jörg Mansfeld, J. Choudhary, Sebastian Guettler, Francisca Lottersberger, Max E Douglas
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a genetic anemia caused by the production of an abnormal adult hemoglobin. The clinical severity is lessened by elevated fetal hemoglobin (HbF) production in adulthood. A promising therapy is the transplantation of autologous, hematopoietic stem/progenitor cells (HSPCs) treated with CRISPR/Cas9 to downregulate the HbF repressor BCL11A via generation of double strand breaks (DSBs) in the +58-kb erythroid-specific enhancer. Here, to further enhance HbF production without increasing the mutagenic load, we targeted both +58-kb and +55-kb BCL11A erythroid-specific enhancers using base editors. We systematically dissected DNA motifs recognized by the key transcriptional activators within these regions and identified the critical nucleotides required for activator binding. Multiplex base editing of these residues was efficient and safe and generated no or little DSBs and genomic rearrangements. We observed substantial HbF reactivation, exceeding the levels achieved using the CRISPR/Cas9 nuclease-based strategy, thus efficiently rescuing the sickling phenotype. Multiplex base editing was efficient in long-term repopulating HSPCs and resulted in potent HbF reactivation in vivo. In summary, these results show that multiplex base editing of BCL11A erythroid-specific enhancers is a safe and potent strategy for treating sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949a30362791a74e8f68a2ef1865b96856027877" target='_blank'>
              Multiplex base editing of BCL11A regulatory elements to treat sickle cell disease
              </a>
            </td>
          <td>
            L. Fontana, P. Martinucci, S. Amistadi, T. Felix, M. Mombled, A. Tachtsidi, G. Corre, A. Chalumeau, G. Hardouin, J. Martin, O. Romano, M. Amendola, P. Antoniou, A. Miccio
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cis-regulatory elements bridge enhancers and gene promoters to control gene expression via distal DNA interaction and three-dimensional chromosomal conformation organization. The aberrant changes of cis-acting regulatory systems as one type of the epigenetic regulative ways may be connected with human genetic diseases. Klotho, as an antiaging protein, is selectively expressed in kidney tissues and plays a crucial role in preventing chronic kidney disease (CKD) and renal fibrosis. However, the underlying transcription regulatory mechanism of Klotho in CKD is not fully understood. Herein, we analyzed the spatial organization of the chromatin region spanning 2 Mb upstream Klotho in human renal punctured CKD tissues using chromosome conformation capture (3C)-qPCR and identified the distal interaction of the Klotho promoter with certain specific chromatin regions characterized as the regulatory elements. Moreover, we determined that four DNase I hypersensitive sites (DHSs) involved in the regulation of Klotho gene expression lost their activities in CKD tissues compared to control accompanied by the reduction of H3K27ac. Finally, the CCCTC-binding factor (CTCF) sites were validated on the DHSs beyond the Klotho promoter by chromatin looping formation through the recruitment of CTCF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac8a6d42134fe44495915973d3727dff6c48d076" target='_blank'>
              The Long-Range Chromosomal Interaction Controlling Klotho Gene Expression in Human Chronic Kidney Disease
              </a>
            </td>
          <td>
            Pengwei Xu, Minjun Jiang, Jian-chun Chen, Yong-qiang Zhou, Zhenfan Wang
          </td>
          <td>2024-12-18</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b62083d3dd11f079792a9e41da3af07b399c20f" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Olivia Dove Estrella, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="SETD6 is a non-histone lysine methylatransferase, previously shown to participate in several housekeeping signaling pathways such as the NFkB pathway, Wnt signaling pathway, mitosis and more. In the current study we show evidence that SETD6 methylation is involved in the regulation of cytokinesis - the final process that divides cell contents into two daughter cells. SETD6 depleted HeLa cells presented high levels of chromatin bridges and actin patches, which are commonly observed following chromosomal segregation errors. In a proteomic screen we identified Aurora-B as a novel SETD6 substrate. Aurora-B kinase is an essential regulator of cytokinesis, known to actively delay cytokinesis as a response to the presence of chromatin in the midzone. We found that SETD6 binds and methylates Aurora-B on two adjacent lysine residues. Upon replication stress, Aurora-B methylation by SETD6 increases but is abolished when the two lysine methylation targets are substituted. In addition, replication stress led to a high tendency of SETD6 depleted cells to multinucleate, a major chromosomal-instability (CIN) phenotype. We detected a significant reduction in the Aurora-B kinase activity during cytokinesis in SETD6 knockout cells upon replication stress, which could be the mechanism underlying the accumulation of CIN phenotypes in these cells. CIN is a hallmark of cancer and is associated with tumor cell malignancy. Our findings suggest that Aurora-B methylation by SETD6 carries meaningful implications on tumorigenic cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30079fd8588b1378d0602ce8bccaf091eb97d109" target='_blank'>
              AuroraB-kinase methylation by SETD6 regulates cytokinesis and protects cells from chromosomal instability
              </a>
            </td>
          <td>
            Michal Feldman, Anand Chopra, Dikla Nachmias, K. Biggar, Daniel Sevilla, Natalie Elia, D. Levy
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4cf48abf7265148385bfea0bb2728f6fd6101" target='_blank'>
              SMYD3 drives cell cycle and epithelial-mesenchymal transition pathways through dual gene transcriptional repression and activation in HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Madhavi Murali, Abbas Saeed, Sohyoung Kim, K. Burkitt, Hui Cheng, Arfa Moshiri, Jawad Akhtar, D. Tsai, Marie Luff, Baktiar Karim, V. Saloura
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Inducible gene expression circuits offer precise control over target gene activation, making them essential tools for direct reprogramming, where cells are guided to differentiate into specific cell types. However, stable circuit function and consistent expression of key inducible proteins are crucial for effective reprogramming, and DNA methylation-induced silencing hinders this stability. To address this issue, A2-ubiquitous chromatin opening elements (A2UCOE) have gained attention for their ability to maintain open chromatin and prevent methylation, thereby stabilizing gene expression. In this study, we aimed to forward program iPSCs into thymic epithelial cells (TECs) using a compact, all-in-one gene circuit composed of a doxycycline-inducible Tet-On system, 863-bp A2UCOE (0.9 UCOE), and FOXN1, a master transcription factor for TEC differentiation. This compact construct enables site-specific genome integration and stable inducible expression within iPSCs for renewably generating TECs. While the 0.9 UCOE promoted stable expression of constitutively expressed genes, it also caused unintended FOXN1 gene leakage, leading to unprogrammed gradual differentiation of the iPSCs. We generated A2UCOE fragments of varying lengths and found that gene leakage persisted regardless of fragment size. We tested spacer sequences between the A2UCOE and the Tet-On promoter consisting of varying AT-nucleotide content (35, 50, and 65%) and the 65% AT-rich SV40 poly-A terminator sequence and found that only the SV40 poly-A mitigated this leakage, and surprisingly, enhanced desired anti-silencing effects. These findings highlight the benefits and potential risks of using A2UCOE in iPSCs, and provides insights into a regulatory DNA architecture optimal for compact forward programming circuits for controlled iPSC differentiation. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73f3bb6e83ecb23bff92fd8f2a8de439e520396" target='_blank'>
              Termination sequence between an inducible promoter and ubiquitous chromatin opening element (UCOE) reduces gene expression leakage and silencing
              </a>
            </td>
          <td>
            Tomoki Yanagi, Shean Fu Phen, Jonah Ayala, Deniz Ece Aydin, David M. Truong
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6318b7359de932f08bb59735e4b7b9de2484816" target='_blank'>
              CHD6 has poly(ADP-ribose)- and DNA-binding domains and regulates PARP1/2-trapping inhibitor sensitivity via abasic site repair
              </a>
            </td>
          <td>
            Luc Provencher, Wilson Nartey, Peter M Brownlee, Austin W Atkins, J. Gagné, Lou Baudrier, Nicholas S Y Ting, C. Piett, Shujuan Fang, D. D. Pearson, S. Moore, Pierre Billon, Zachary D. Nagel, Guy G. Poirier, Gareth J. Williams, Aaron A Goodarzi
          </td>
          <td>2025-01-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-stranded DNA secondary structures such as G-quadruplexes (G4s) can potentially disrupt transcription, replication, and repair. Using bio-informatic analysis, here we show that BMI1 is enriched at putative G4s flanked by heterochromatin domains, and that BMI1 knockdown in human dermal fibroblasts (HDFs) resulted in heterochromatin relaxation and G4 induction, followed by replication stress and genomic instability. In these cells, G4s co-localized with large 53BP1 and PCNA foci resembling replication catastrophes. Inhibiting transcription partly attenuated DNA damage, suggesting rescue of transcription-replication collisions at difficult-to-replicate sequences. In BMI1 knockdown or pyridostatin-exposed HDFs, the Werner helicase accumulated and co-localized with G4s. Acute WRN knockdown also resulted in G4 induction. In HDFs from Werner and Hutchinson-Gilford progeria syndromes, loss of heterochromatin and nuclear envelope anomalies were associated with G4 induction and DNA damage. Nuclear envelope anomalies were also prominent following BMI1 knockdown. These findings suggests that heterochromatin-mediated repression of G4s attenuates replication stress and genomic instability, and that this mechanism is shared across distinct progeroid models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731920cfa53eda1eb3b00a50ee2eec0629da79d1" target='_blank'>
              BMI1-mediated heterochromatinization represses G-quadruplex DNA formation to maintain genomic stability during replication
              </a>
            </td>
          <td>
            Roy Hanna, Gilbert Bernier
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The interaction between RAD51 and BRCA2 plays a key role in homologous recombination (HR), a critical DNA repair mechanism essential for the survival of cancer cells. Disrupting this interaction increases the sensitivity of cancer cells to chemotherapeutic agents. Here, we employed in silico methods to design a novel class of aptamers—customized single-stranded oligonucleotides—specifically engineered to bind RAD51. These aptamers were developed with the aim of selectively modulating RAD51’s nuclear recruitment and its role in DNA repair processes. The leading candidate displays high affinity for RAD51, competing with BRCA2 for the same interaction site in vitro, as confirmed through biolayer interferometry (BLI) and fluorescence lifetime imaging microscopy (FLIM). We tested the efficacy of the leading aptamer in pancreatic cancer cells and observed that it significantly impedes RAD51 nuclear localization, reduces homologous recombination (HR) efficiency, and increases DNA damage. Critically, our aptamer potentiates the cytotoxicity of the PARP inhibitor olaparib, exploiting synthetic lethality (SL) to induce cancer cell death. Our study showcases an aptamer-based approach for selectively targeting protein interactions within DNA repair pathways, introducing a promising avenue for SL-based treatments applicable to a wide range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc2579e152970b51a457421873665b384bcc5d3" target='_blank'>
              Computationally-designed aptamers targeting RAD51-BRCA2 interaction inhibit RAD51 nuclear recruitment
              </a>
            </td>
          <td>
            Giulia Milordini, E. Zacco, Alexandros Armaos, Francesco Di Palma, Mirco Masi, Martina Gilodi, Jakob Rupert, Laura Broglia, Giulia Varignani, Michele Oneto, Marco Scotto, Roberto Marotta, S. Girotto, Andrea Cavalli, Gian Gaetano Tartaglia
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Protein domains of transposable elements (TEs) and viruses increase the protein diversity of host genomes by recombining with other protein domains. By screening 10 million eukaryotic proteins, we identified several domains that define multi-copy gene families and frequently co-occur with TE/viral domains. Among these, a Tc1/Mariner transposase helix-turn-helix (HTH) domain was captured by F-box genes in the Caenorhabditis genus, creating a new class of F-box genes. For specific members of this class, like fbxa-215, we found that the HTH domain is required for diverse processes including germ granule localisation, fertility, and thermotolerance. Furthermore, we provide evidence that HSF-1 mediates the transcriptional integration of fbxa-215 into the heat-shock response by binding to Helitron TEs directly upstream of the fbxa-215 locus. The interactome of HTH-bearing F-box factors suggests roles in post-translational regulation and proteostasis, consistent with established functions of F-box proteins. Based on AlphaFold2 multimer proteome-wide screens, we propose that the HTH domain may diversify the repertoire of protein substrates that F-box factors regulate post-translationally. We further demonstrate that F-box genes repeatedly and independently captured TE domains throughout eukaryotic evolution, and describe an additional instance in zebrafish. In conclusion, we identify recurrent TE domain captures by F-box genes in eukaryotes and provide insights into how these novel proteins are integrated within host gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6dd85055c13e497054453e00ae392d0e8fafaf" target='_blank'>
              Transposable elements drive regulatory and functional innovation of F-box genes
              </a>
            </td>
          <td>
            M. V. Almeida, Zixin Li, Pedro Rebelo-Guiomar, Alexandra Dallaire, Lukáš Fiedler, Jonathan L. Price, Jan Sluka, Xiaodan Liu, F. Butter, C. Rödelsperger, E. Miska
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Selection of a suitable chromatin environment during retroviral integration is a tightly regulated and multilayered process that involves interplay between viral and host factors. However, whether intrinsic chromatin dynamics during mitosis modulate retroviral genome invasion is currently poorly described. Direct interaction between the spumaretrovirus prototype foamy virus (PFV) Gag protein and cellular chromatin has been described as a major determinant for integration site selection. A previous Gag chromatin-binding site (CBS)–nucleosome co-crystal structure revealed an interaction with the histone H2A-H2B acidic patch via a highly conserved arginine anchor residue. Yet, the molecular mechanisms regulating Gag-chromatin capture during PFV infection remain obscure. Here, we investigated the kinetics of Gag-chromatin interactions during mitosis and proviral integration of PFV-infected synchronized cells. Using Gag CBS variant viruses, we showed that alteration of Gag affinity for nucleosome binding induced untimely chromatin tethering during mitosis, decreased infectivity and redistributed viral integration sites to markers associated with late replication timing of host chromosomes. Mutant Gag proteins were moreover defective in their ability to displace the histone H4 tail from the nucleosome acidic patch of highly condensed mitotic chromatin. These data indicate that the mitotic chromatin landscape during Gag–nucleosome interactions hosts PFV integration site selection determinants and that spumaretroviruses evolved high-affinity chromatin binding to overcome early mitosis chromatin condensation for optimal viral DNA tethering, integration and infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c034dba5335395c865d3f3e018bafaae4ee93b" target='_blank'>
              Timed chromatin invasion during mitosis governs prototype foamy virus integration site selection and infectivity
              </a>
            </td>
          <td>
            F. Lagadec, P. K. Singh, C. Calmels, D. Lapaillerie, D. Lindemann, V. Parissi, P. Cherepanov, A. N. Engelman, P. Lesbats
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA, our findings rather suggest that protein-DNA co-condensates ‘sense’ the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d84d86b264f5c1ad8de5c81e2ae317173c400dd" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Thejas Satheesh, Rohit Kumar Singh, Prakshi Gaur, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Long interspersed element-1 (LINE-1, L1) retrotransposons are the most abundant protein-coding transposable elements (TE) in mammalian genomes, and have shaped genome content over 170 million years of evolution. LINE-1 is self-propagating and mobilizes other sequences, including Alu elements. Occasionally, LINE-1 forms chimeric insertions with non-coding RNAs and mRNAs. U6 spliceosomal small nuclear RNA/LINE-1 chimeras are best known, though there are no comprehensive catalogs of LINE-1 chimeras. To address this, we developed TiMEstamp, a computational pipeline that leverages multiple sequence alignments (MSA) to estimate the age of LINE-1 insertions and identify candidate chimeric insertions where an adjacent sequence arrives contemporaneously. Candidates were refined by detecting hallmark features of L1 retrotransposition, such as target site duplication (TSD). Applying this pipeline to the human genome, we recovered all known species of LINE-1 chimeras and discovered new chimeric insertions involving small RNAs, Alu elements, and mRNA fragments. Some insertions are compatible with known mechanisms, such as RNA ligation. Other structures nominate novel mechanisms, such as trans-splicing. We also see evidence that LINE-1 loci with defunct promoters can acquire regulatory elements from nearby genes to restore retrotransposition activity. These discoveries highlight the recombinatory potential of LINE-1 RNA with implications for genome evolution and TE domestication. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1899bb5dd3c3cee45e88b97674bce81ea6849d66" target='_blank'>
              Comparative Genomics Reveals LINE-1 Recombination with Diverse RNAs
              </a>
            </td>
          <td>
            Cheuk-Ting Law, Kathleen H. Burns
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Transposable elements (TEs) comprise the majority of the human genome. In most somatic tissues, TEs are silenced by DNA methylation and histone modifications. Tumors exhibit changes in DNA methylation including global loss of methylation at regions that are silenced for genome stability, like TEs, and gain of methylation at the promoter regions of tumor suppressor genes. We showed that low doses of DNMT inhibitors (DNMTis) activate type I interferon signaling by reducing methylation and increasing expression of TEs that activate dsRNA sensors. DNMTis increase TEs in murine models of cancer, activating interferon signaling and recruiting CD8+ T cells to kill cancer cells and sensitize to immune checkpoint blockade therapy. We performed a genome-wide analysis of the response to DNMTi in a panel of ovarian cancer cell lines. Alu elements, which directly bind to the dsRNA sensor MDA5, are the most significantly demethylated TEs after DNMTi treatment. Our RNA-sequencing analyses indicated that A-to-I RNA editing of TEs by the RNA editing enzyme ADAR1 was increased after DNMTi treatment. This A-to-I editing makes TE RNA less immunogenic and inhibits the immune response to DNMTi. We find that combining ADAR1 knockdown and DNMTi treatment significantly increases type I interferon signaling and cancer cell secretion of downstream cytokines, including CCL5 and CXCL10. Further, this combination increased recruitment and activation of host immune cells and significantly improved survival in the aggressive ID8 Trp53-/- murine model of ovarian cancer. CITE-Seq (cellular indexing of transcriptomes and epitopes, a type of single-cell RNA sequencing that utilizes antibody tagging of each cell type before sequencing) demonstrated significantly higher infiltration of T and B cells in the tumor microenvironment. To understand the immune cell types causing the DNMTi anti-tumor response, we used antibodies to deplete CD8 T cells and NK cells in the ID8 Trp53-/- model with ADAR1 knockdown and DNMTi treatment. In addition, we performed these experiments in the C57BL/6-Tg(IghelMD4)4Ccg/J mice, which only produce antibodies against “Hel” antigen and have no other B cell responses. Of these lymphocyte populations, depleting T cells had the biggest effect and we observed specific killing of tumor cells by T cells isolated from the mice. We performed immunopeptidomics on cancer cell lines with and without DNMTi and showed that DNMTi upregulate TE peptide presentation by MHC molecules on the surface of tumor cells. Using a novel immuno-engineering approach, we are able to generate antigen-specific T cells targeting TEs and other antigens in murine and human cancer cell line models. These T cells specifically recognize (produce IFNgamma and TNFalpha) and kill the target tumor cells. As DNMTi have also been shown to demethylate genes crucial for T cell function, including IFNgamma and TCF7, we are currently combining these T cells with DNMTis for a novel cell therapy against cancer.
 Citation Format: Katherine B Chiappinelli. Epigenetic regulation of transposable elements to reverse cancer immune evasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras—particularly in diseases such as oral squamous cell carcinoma and glioblastoma—now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, T. Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The nucleolus is a major subnuclear compartment where ribosomal DNA (rDNA) is transcribed and ribosomes are assembled. In addition, recent studies have shown that the nucleolus is a dynamic organizer of chromatin architecture that modulates developmental gene expression. rDNA gene units are assembled into arrays located in the p-arms of five human acrocentric chromosomes. Distal junctions (DJs) are ∼400 kb sequences adjacent to rDNA arrays that are thought to anchor them at the nucleolus, although the underlying regulatory elements remain unclear. Here we show that DJs display a dynamic chromosome conformation profile in human embryonic stem cells (hESCs). We identified a primate-specific, full-length insertion of the retrotransposon long interspersed nuclear element 1 (LINE1) in a conserved position across all human DJs. This DJ-LINE1 locus interacts with specific regions of the DJ and is upregulated in naïve hESCs. CRISPR-based deletion and interference approaches revealed that DJ-LINE1 contributes to nucleolar positioning of the DJs. Moreover, we found that the expression of DJ-LINE1 is required for maintenance of the structure and transcriptional output of the nucleolus in hESCs. Silencing of DJ-LINE1 leads to loss of self-renewal, disruption of the landscape of chromatin accessibility, and derepression of earlier developmental programs in naïve hESCs. This work uncovers specific LINE1 elements with a fundamental role in nucleolar organization in hESCs and provides new insights into how the nucleolus functions as a key genome-organizing hub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6f03971b0054eedbed60fad051bf9b167c6012" target='_blank'>
              LINE1 elements at distal junctions of rDNA repeats regulate nucleolar organization in human embryonic stem cells.
              </a>
            </td>
          <td>
            Lamisa Ataei, Juan Zhang, Simon Monis, Krystyna Giemza, Kirti Mittal, Joshua Yang, Mayu Shimomura, B. McStay, Michael D. Wilson, M. Ramalho-Santos
          </td>
          <td>2024-12-20</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) imposes a significant health burden, necessitating innovative therapeutic strategies to enhance treatment efficacy. Current treatments such as surgery, radiation, and chemotherapy have limited effectiveness and yield severe side effects, emphasizing the need for targeted therapies. We have focused on DNA polymerase beta (Polβ) and its roles in replication stress, cellular responses to DNA damaging therapies, and DNA damage response modifiers. Our investigations reveal a regulatory role for base excision repair (BER) proteins, including Polβ, in the cellular response to inhibitors of poly(ADP-ribose) glycohydrolase (PARG), an enzyme involved in poly(ADP-ribose) (PAR) degradation. The inhibition of PARG, in HNSCC cells, elicits replication stress. Further, this activates the PAR-induced S-phase/ATR checkpoint, leading to a block to replication, cell cycle arrest, and the onset of apoptosis. However, Polβ overexpression mitigates this response, reducing replication-stress induced PAR foci formation, suggesting a modulation of replication checkpoint activation. We found that PARG inhibitor treatment is ineffective on HNSCC cells that overexpress Polβ, implying that the PARG inhibitor-induced PAR and apoptotic response is dependent on the level of Polβ. Further, our in vitro experiments demonstrate that combining PARG and ATR/CHK1 inhibitors overcomes Polβ-mediated treatment resistance in HNSCC cells, producing enhanced effects as compared to the individual treatment conditions. Our findings suggest a possible treatment paradigm for HNSCC, employing ATR or CHK1 inhibitors in combination with PARG inhibitors. This strategy offers a promising path for more effective HNSCC treatments, potentially overcoming Polβ-related resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546af29aab21a1579fd4129cca1f86d70f7aaf27" target='_blank'>
              DNA polymerase beta expression in head & neck cancer modulates the poly(ADP-ribose)-mediated replication checkpoint
              </a>
            </td>
          <td>
            Md Maruf Khan, Wynand P. Roos, Anusha Angajala, Denise Y. Gibbs, Jeffrey C. Liu, Camille Ragin, Robert W. Sobol
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 During solid tumor development, gene-poor and lamina-associated genomic regions undergo progressive loss of DNA methylation, whereas promoter CpG islands become DNA hypermethylated. We report evidence suggesting that the recruitment of DNMT3A by histone modifications contributes to this cancer-associated DNA methylome reprogramming. The genomic targeting of DNMT3A is regulated by the competition between NSD1-mediated histone H3K36 methylation and polycomb-mediated H2AK119 ubiquitination, via DNMT3A's PWWP domain and N-terminal ubiquitin-interacting region, respectively. Mutations in NSD1, DNMT3A reader domains or polycomb complexes perturb DNMT3A recruitment and can recapitulate patterns of DNA methylation observed in patient tumors. Lastly, tumors exhibiting progressive hypomethylation, such as those harboring NSD1 mutations, are paradoxically immune-cold despite the increased expression of retrotransposons. These results suggest that DNA hypomethylation and aberrant retrotransposon activation are significant bottlenecks to tumor development and may represent promising therapeutic targets.
 Citation Format: Chao Lu. Interplay between histone modifications and DNA methylation in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8283805e8d954027bac5a7fd108f9580c6417e4d" target='_blank'>
              Abstract PR004: Interplay between histone modifications and DNA methylation in cancer
              </a>
            </td>
          <td>
            Chao Lu
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer.We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity.UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug.We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The cell line Sarcoma 180, which is also called S-180 (or S180), was established about 110 years ago from a murine axillary sarcoma. It has been applied in >5000 studies but was never genetically characterized in detail; this study fills that gap. The cell line Sarcoma 180 was analyzed for its chromosomal constitution using molecular cytogenetic approaches, specifically murine multicolor banding (mcb). Additionally, array comparative genomic hybridization was performed to characterize copy number alterations. Sarcoma 180 has a near tetraploid karyotype without Y-chromosome material and only two X-chromosomes. The complex karyotype includes dicentrics and simple and complex rearrangements and shows a relatively high chromosomal instability. An in silico translation of the obtained results to the human genome indicated that Sarcoma 180 is suitable as a model for advanced human mesenchymal chondrosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf21550d200fcd748a7836eb6966ec3a47ab78b" target='_blank'>
              Cytogenomic Characterization of Murine Cell Line Sarcoma 180 = S-180
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2025-01-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite the evolutionary importance of supergenes, their properties in polyploids remain unexplored. Polyploid genomes are expected to undergo chromosomal rearrangements and gene losses over time, potentially affecting supergene architecture. The iconic distyly supergene (S-locus), controlling a floral heteromorphism with two self-incompatible morphs, has been well-documented in diploids, but remains unknown in polyploids. Primula, the classic model for distyly since Darwin, is ancestrally diploid and distylous, yet polyploid, homostylous species with a single, self-compatible floral morph evolved repeatedly. The intraspecific loss of distyly is associated with small loss-of-function mutations in the S-locus CYPT gene controlling style length and female self-incompatibility. Over longer timescales, relaxed selection on CYPT should generate greater accumulation of larger mutations, including exon and gene loss. By analyzing the first assembled genome of an allotetraploid, homostylous species (Primula grandis) in a comparative framework, we discovered two, nearly identical S-locus alleles in the same subgenome, suggesting it originated via inter-specific hybridization between a homostylous and a distylous progenitor. Conformant to predictions from theory, the macroevolutionary loss of distyly coincided with considerable degeneration of CYPT, while other S-locus genes remained largely unaffected, suggesting the shift to homostyly preceded and facilitated polyploid establishment. At the whole-genome level, we found minimal subgenome dominance — as expected, given the inferred recent origin of P. grandis — and highly reduced genetic diversity, congruently with its narrow distribution and self-compatibility. This study provides the first comparison of a supergene across ploidy levels and reproductive systems, contributing new knowledge on the previously unknown fate of supergenes in polyploids. SIGNIFICANCE This study advances knowledge on genome evolution by elucidating how supergenes (clusters of tightly linked genes) evolve across species with different sets of chromosomes and reproductive systems. By analyzing the newly assembled genome of the polyploid, self-compatible Primula grandis in a broad framework, we provide the first comparison of the distyly supergene between diploid outcrossers and polyploid self-fertilizers. We discovered one pair of identical supergene alleles in the same subgenome, rather than one pair per subgenome, revealing the species originated via a cross between a self-compatible and a self-incompatible progenitor. Conformant to theory, the gene controlling female self-incompatibility and style length (CYPT) was considerably degenerated, because of relaxed selection over time, with the rest of the supergene largely unaffected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99c78bc08b60ac655953b43bdc71e27d5d4f87" target='_blank'>
              Fate of a supergene in the shift from diploidy to polyploidy
              </a>
            </td>
          <td>
            E. Mora-Carrera, N. Yousefi, G. Potente, R. Stubbs, B. Keller, Étienne Léveillé-Bourret, Grob Stefan, F. Celep, G. Tedoradze, E. Conti
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Mutations in somatic cells are inflicted by both extrinsic and intrinsic sources and contribute over time to cancer. Tobacco smoke contains chemical carcinogens that have been causatively implicated with cancers of the lung and head & neck1,2. APOBEC family DNA cytosine deaminases have emerged as endogenous sources of mutation in cancer, with hallmark mutational signatures (SBS2/SBS13) that often co-occur in tumors of tobacco smokers with an equally diagnostic mutational signature (SBS4)3,4. Here we challenge the dogma that mutational processes are thought to occur independently and with additive impact by showing that 4-nitroquinoline 1-oxide (NQO), a model carcinogen for tobacco exposure, sensitizes cells to APOBEC3B (A3B) mutagenesis and leads to synergistic increases in both SBS2 mutation loads and oral carcinomas in vivo. NQO-exposed/A3B-expressing animals exhibit twice as many head & neck lesions as carcinogen-exposed wildtype animals. This increase in carcinogenesis is accompanied by a synergistic increase in mutations from APOBEC signature SBS2, but not from NQO signature SBS4. Interestingly, a large proportion of A3B-catalyzed SBS2 mutations occurs as strand-coordinated pairs within 32 nucleotides of each other in transcribed regions, suggesting a mechanism in which removal of NQO-DNA adducts by nucleotide excision repair exposes short single-stranded DNA tracts to enzymatic deamination. These highly enriched pairs of APOBEC signature mutations are termed didyma (Greek for twins) and are mechanistically distinct from other types of clustered mutation (omikli and kataegis). Computational analyses of lung and head & neck tumor genomes show that both APOBEC mutagenesis and didyma are elevated in cancers from smokers compared to non-smokers. APOBEC signature mutations and didyma are also elevated in normal lung tissues in smokers prior to cancer initiation. Collectively, these results indicate that DNA adducting mutagens in tobacco smoke can amplify DNA damage and mutagenesis by endogenous APOBEC enzymes and, more broadly, suggest that mutational mechanisms can interact synergistically in both cancer initiation and promotion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de637cc3bf2890bf07d4603dfc1f19004aa5b7c2" target='_blank'>
              Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            Cameron Durfee, E. N. Bergstrom, M. Díaz-Gay, Yufan Zhou, N. A. Temiz, Mahmoud A. Ibrahim, Shuvro P. Nandi, Yaxi Wang, Xingyu Liu, Christopher D. Steele, Joshua Proehl, Rachel I Vogel, P. Argyris, Ludmil B. Alexandrov, R. Harris
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Cancer cells display high levels of oncogene-induced replication stress (RS) and rely on DNA damage checkpoint for viability. This feature is exploited by cancer therapies to either increase RS to unbearable levels or inhibit checkpoint kinases involved in the DNA damage response. Thus far, treatments that combine these two strategies have shown promise but also have severe adverse effects. To identify novel, better-tolerated anticancer combinations, we screened a collection of plant extracts and found two natural compounds from the plant, Psoralea corylifolia, that synergistically inhibit cancer cell proliferation. Bakuchiol inhibited DNA replication and activated the checkpoint kinase CHK1 by targeting DNA polymerases. Isobavachalcone interfered with DNA double-strand break repair by inhibiting the checkpoint kinase CHK2 and DNA end resection. The combination of bakuchiol and isobavachalcone synergistically inhibited cancer cell proliferation in vitro. Importantly, it also prevented tumor development in xenografted NOD/SCID mice. The synergistic effect of inhibiting DNA replication and CHK2 signaling identifies a vulnerability of cancer cells that might be exploited by using clinically approved inhibitors in novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/042cde9aa3dc3d91bb528dfff54349708df45e8a" target='_blank'>
              Synergistic effect of inhibiting CHK2 and DNA replication on cancer cell growth
              </a>
            </td>
          <td>
            Flavie Coquel, Sing-Zong Ho, K. Tsai, Chun-Yen Yang, Antoine Aze, Julie Devin, Ting-Hsiang Chang, Marie Kong-Hap, Audrey Bioteau, J. Moreaux, Domenico Maiorano, Philippe Pourquier, Wen-Chin Yang, Yea-Lih Lin, Philippe Pasero
          </td>
          <td>2025-01-31</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="A central goal of cancer genomics is to identify, in each patient, all the cancer-driving mutations. Among them, point mutations are referred to as cancer-driving nucleotides (CDNs), which recur in cancers. The companion study shows that the probability of i recurrent hits in n patients would decrease exponentially with i; hence, any mutation with i ≥ 3 hits in The Cancer Genome Atlas (TCGA) database is a high-probability CDN. This study characterizes the 50–150 CDNs identifiable for each cancer type of TCGA (while anticipating 10 times more undiscovered ones) as follows: (i) CDNs tend to code for amino acids of divergent chemical properties. (ii) At the genic level, far more CDNs (more than fivefold) fall on noncanonical than canonical cancer-driving genes (CDGs). Most undiscovered CDNs are expected to be on unknown CDGs. (iii) CDNs tend to be more widely shared among cancer types than canonical CDGs, mainly because of the higher resolution at the nucleotide than the whole-gene level. (iv) Most important, among the 50–100 coding region mutations carried by a cancer patient, 5–8 CDNs are expected but only 0–2 CDNs have been identified at present. This low level of identification has hampered functional test and gene-targeted therapy. We show that, by expanding the sample size to 105, most CDNs can be identified. Full CDN identification will then facilitate the design of patient-specific targeting against multiple CDN-harboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb003947f43738ae589b3a6538f7d27fb518829" target='_blank'>
              Characterization of cancer-driving nucleotides (CDNs) across genes, cancer types, and patients
              </a>
            </td>
          <td>
            Lingjie Zhang, Tong Deng, Zhongqi Liufu, Xiangnyu Chen, Shijie Wu, Xueyu Liu, Changhao Shi, Bingjie Chen, Zheng Hu, Qichun Cai, Chenli Liu, Mengfeng Li, Miles E. Tracy, Xuemei Lu, Chung-I Wu, Hai-Jun Wen
          </td>
          <td>2024-12-17</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Transposable elements (TEs), once considered as "junk" DNA, are now recognized for their epigenetic and regulatory influence, especially in cancer. Cryptic regulatory elements within TEs can be epigenetically reactivated in cancer, contributing to oncogenesis and altering gene expression patterns. This reactivation results in TE-derived transcripts, which have the potential to be translated and form neoantigens that can be presented by the human leukocyte antigens (HLA) complex and recognized by T cells, ultimately impacting the tumor immune response. In this study, we analyzed RNA-seq data from melanoma patients undergoing PD1/PDL1 immunotherapy. By identifying tumor specific TE-derived transcripts and utilizing in silico translation, we detected TE-derived neoantigens in patients with differing responses to immunotherapy. Our findings indicate that the number of TE-derived neoantigens is correlated with patients' response to immunotherapy, with a higher number of neoantigens observed in patients who responded to treatment compared to those who did not. These results imply that TE-derived neoantigens may serve as potential biomarkers for predicting the prognosis of PD1/PDL1 immunotherapy treatment. Incorporating TE-derived neoantigen profiles into clinical decision-making could enhance patient selection for immunotherapy, thereby improving treatment outcomes.
 Citation Format: Yixin Wu, Ting Wang, Yonghao Liang. Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a907aabd673387a635b624fb0d79ef5b5ee9835" target='_blank'>
              Retrotrans-genomics identifies aberrant THE1B endogenous retrovirus fusion transcripts in the pathogenesis of sarcoidosis
              </a>
            </td>
          <td>
            Shunsuke Funaguma, A. Iida, Yoshihiko Saito, J. Tanboon, Francia Victoria De Los Reyes, K. Sonehara, Yu-Ichi Goto, Y. Okada, Shinichiro Hayashi, Ichizo Nishino
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, Gregor Warsow, Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chromosomes and their constituent compartments are core elements of the cell division machinery. In mammalian oocytes, defects in the structure or composition of chromosomes are a leading cause of aberrant or failed meiosis, and by extension, can cause miscarriage and infertility. The underlying mechanisms are poorly understood, but are critical for development of novel diagnostics and treatments. The chromosome periphery, the least understood chromosome compartment, has recently emerged as an essential component of mitotic chromosomes and somatic cell division. However, in female meiosis it remains completely unexplored. This study provides the first comprehensive survey of a meiotic chromosome periphery compartment in both human and mouse oocytes. Using a combination of time-lapse microscopy, super-resolution imaging and 3DCLEM we show that removing the chromosome periphery, via Ki67 depletion, has substantial negative impact on chromosome structure, spatial organization and positional awareness, with many oocytes stalling and arresting in meiosis I. Importantly, we also reveal key differences between the mitotic and oocyte meiotic chromosome periphery compartments, most remarkably in the retention of Ki67 in the oocyte through anaphase I, where unwanted chromosomes are stripped of Ki67 before being ejected from the oocyte. This work presents the discovery of an exciting new pathway operating during female meiosis and a provides a platform for future work exploring the meiotic chromosome periphery for therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4e852fd562f1b6718584414157f534cabd865" target='_blank'>
              The Chromosome Periphery is an Essential Compartment of Oocyte Chromosomes
              </a>
            </td>
          <td>
            Eva L Simpson, B. Wetherall, A. Byrska, Liam P. Cheeseman, Tania Mendonca, Xiaomeng Xing, Alison J. Beckett, H. Maiato, Alexandra Sarginson, Ian A. Prior, Geraldine M Hartshorne, A. McAinsh, S. Madgwick, Daniel G. Booth
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="With the practical implementation of the CRISPR/Cas technology for targeted genome editing, it has become possible to carry out genetic engineering manipulations with eukaryotic genomes with high efficiency. One of the key stages of this technology is the targeted induction of site-specific DNA cleavages (breaks). The cell repairs these breaks via one of two pathways: nonhomologous end joining or homologous recombination. The choice of DNA repair pathway is determined by the architecture of the sites at the DNA break area formed as a result of terminal resection and depends on the phases of the cell cycle. Nonhomologous end joining is the main pathway for repair of double-stranded DNA breaks in mammalian cells. It involves a nonspecific ligation reaction, the accuracy of which depends on the structure of the ends of the break, and can result in various insertions or deletions in the target region of the genome. Integration of the desired sequence into the genome occurs along the path of homologous recombination, the implementation of which requires a matrix with homology regions on both sides of the double-strand break. The introduction of a genetic construct into a given location in the genome is an important, but currently complex and labor-intensive task. At the same time, for fundamental studies of gene function and the creation of animal models of human diseases, the choice of the repair pathway can be of fundamental importance. This review is an attempt to combine and structure all known information on approaches to increasing the efficiency of DNA repair involving homologous recombination. The article lists the most effective strategies to shift the balance towards homologous repair, such as the use of inhibitors of the non-homologous end joining mechanism, regulation of key factors of homologous recombination, control of the cell cycle, chromatin status, construction of templates for homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23816bdc012e17808d654e03c85cd07d45917c31" target='_blank'>
              How to shift the equilibrium of dna break repair in favor of homology recombination
              </a>
            </td>
          <td>
            O. A. Averina, S. A. Kuznetsova, O. Permyakov, P. V. Sergiev
          </td>
          <td>2024-12-30</td>
          <td>Molekulârnaâ biologiâ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance.
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, Tianjun Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs) are significant drivers of genome evolution, yet their recent dynamics and impacts within and among species, as well as the roles of host genes and non-coding RNAs in the transposition process, remain elusive. With advancements in large-scale pan-genome sequencing and the development of open data sharing, large-scale comparative genomics studies have become feasible. Here, we performed complete de novo TE annotations and identified active TEs in 310 plant genome assemblies across 119 species and seven crop populations. Using 811 high-quality genomes, we detected 13 844 553 TE-induced structural variants (TE-SVs), providing unprecedented resolution in delineating recent TE activities. Our integrative analysis revealed a mutual evolutionary relationship between TEs and host genomes. On one hand, host genes and ncRNAs are involved in the transposition process, as evidenced by their colocalization and coactivation with TEs, and may play a role in chromatin regulation. On the other hand, TEs drive genetic innovation by promoting the duplication of host genes and inserting into regulatory regions. Moreover, genes influenced by active TEs are linked to plant growth, nutrient absorption, storage metabolism and environmental adaptation, aiding in crop domestication and adaptation. This TE dynamics atlas not only reveals evolutionary and functional features linked to transposition activity but also highlights the role of TEs in crop domestication and adaptation, paving the way for future exploration of TE-mediated genome evolution and crop improvement strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3e6dd3ee1f3baea529da4c3c4ab4bf7659f210" target='_blank'>
              Deciphering recent transposition patterns in plants through comparison of 811 genome assemblies.
              </a>
            </td>
          <td>
            Yan Huang, Sunil Kumar Sahu, Xin Liu
          </td>
          <td>2025-01-10</td>
          <td>Plant biotechnology journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be789fb687d3f78efa3b5343344eaabbca303d35" target='_blank'>
              Chromatin licensing and DNA replication factor 1 as a potential prognostic and diagnostic biomarker for gastric cancer
              </a>
            </td>
          <td>
            Sijia Wang, Min Cui, Na Zhou, Meichen Tong, Shuhua Wu, Hong Li, Zhang Cao
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ku70 and Ku80 form Ku, a ring-shaped protein that initiates the non-homologous end-joining (NHEJ) DNA repair pathway.1 Specifically, Ku binds to double-stranded DNA (dsDNA) ends and recruits other NHEJ factors (e.g., DNA-PKcs and LIG4). While Ku binds to double-stranded RNA (dsRNA)2 and traps mutated-DNA-PKcs on ribosomal RNA in vivo,3,4 the physiological significance of Ku-dsRNA interactions in otherwise wild-type cells remains elusive. Intriguingly, while dispensable for murine development,5,6 Ku is essential in human cells.7 Despite similar genome sizes, human cells express ∼100-fold more Ku than mouse cells, implying functions beyond NHEJ, possibly through a dose-sensitive interaction with dsRNA, which is ∼100 times weaker than with dsDNA.2,8 While investigating the essentiality of Ku in human cells, we found that depletion of Ku - unlike LIG4 - induces profound interferon (IFN) and NF-kB responses reliant on the dsRNA-sensor MDA5/RIG-I and adaptor MAVS. Prolonged Ku-degradation also activates other dsRNA-sensors, e.g. PKR that suppresses protein translation, and OAS/RNaseL that cleaves rRNAs and eventually induces growth arrest and cell death. MAVS, RIG-I, or MDA5 knockouts suppressed IFN signaling and, together with PKR knockouts, partially rescued Ku-depleted human cells. Ku-irCLIP analyses revealed that Ku binds to diverse dsRNA, predominantly stem-loops in primate-specific Alu elements9 at anti-sense orientation in introns and 3’-UTRs. Ku expression rose sharply in higher primates tightly correlating with Alu-expansion (r = 0.94/0.95). Together, our study identified a vital role of Ku in accommodating Alu-expansion in primates by mitigating a dsRNA-induced innate immune response, explaining the rise of Ku levels and its essentiality in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24649fa8cee2fc610da3934361ee2a3da1b227a9" target='_blank'>
              Ku suppresses RNA-mediated innate immune responses in human cells to accommodate primate-specific Alu expansion
              </a>
            </td>
          <td>
            Yimeng Zhu, Angelina Li, Suvrajit Maji, Brian J. Lee, Sophie M. Korn, Jake A. Gertie, Tyler J. Dorrity, Jianhua Wang, Kyle J. Wang, Amandine Pelletier, Daniel F. Moakley, Rachel D. Kelly, Antony B. Holmes, Raul Rabadan, D. Edgell, Caroline Schild Poulter, Mauro Modesti, Anna-Lena Steckelberg, Eric A. Hendrickson, Hachung Chung, Chaolin Zhang, Shan Zha
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Since the discovery of the Musashi (MSI) protein, its ability to affect the mitosis of Drosophila progenitor cells has garnered significant interest among scientists. In the following 20 years, it has lived up to expectations. A substantial body of evidence has demonstrated that it is closely related to the development, metastasis, migration, and drug resistance of malignant tumors. In recent years, research on the MSI protein has advanced, and many novel viewpoints and drug resistance attempts have been derived; for example, tumor protein p53 mutations and MSI-binding proteins lead to resistance to protein arginine N-methyltransferase 5-targeted therapy in lymphoma patients. Moreover, the high expression of MSI2 in pancreatic cancer might suppress its development and progression. As a significant member of the MSI family, MSI2 is closely associated with multiple malignant tumors, including hematological disorders, common abdominal tumors, and other tumor types (e.g., glioblastoma, breast cancer). MSI2 is highly expressed in the majority of tumors and is related to a poor disease prognosis. However, its specific expression levels and regulatory mechanisms may differ based on the tumor type. This review summarizes the research progress related to MSI2 in recent years, including its occurrence, migration mechanism, and drug resistance, as well as the prospect of developing tumor immunosuppressants and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dd0dc9ec8f8cc8ab893bc32f47ba299917d963" target='_blank'>
              Update on the Progress of Musashi-2 in Malignant Tumors.
              </a>
            </td>
          <td>
            Yiting Niu, Tao Zhou, Yanjun Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction The altered expression of genes encoding histone acetyltransferases (HATs) and histone deacetylases (HDACs) has been implicated in the tumorigenesis and progression of various solid tumors. However, systematic characterization of the transcriptomic landscape and clinical relevance of HATs and HDACs in pan-cancer contexts remains lacking. Methods Transcriptome and clinical data of 9,483 patients across 31 tumor types from The Cancer Genome Atlas were collected for systematic pan-cancer analysis. Additional glioma-specific datasets (Chinese Glioma Genome Atlas, GlioVis, GSE43378, and GSE182109) were also collected to validate the transcriptional characteristics of HATs and HDACs in gliomas. Consensus clustering analysis was applied to identify distinct expression patterns of HATs and HDACs. Results Based on the transcriptomic data of 25 genes encoding 9 HATs and 16 HDACs, we identified five major subtypes across 31 cancer types (AC-I to AC-V). Notably, the AC-V subtype comprised over 95% of glioma patients, suggesting glioma patients exhibited distinct expression patterns of histone acetylation-modifying enzymes compared to patients with other solid tumors. Therefore, we re-conducted the consensus clustering analysis specifically within the context of gliomas and identified five subtypes, denoted “AC-GI” to “AC-GV”, which were characterized by differences in HATs/HDACs expression patterns, biological and immune status, genetic alterations, and clinical outcomes. The AC-GII patients exhibited the best prognosis and were sensitive to temozolomide, while AC-GV patients had the poorest prognosis and the lowest sensitivity to temozolomide among all subtypes. Moreover, based on the Connectivity Map database analysis and experimental verification, we identified several pan-HDAC inhibitors that could serve as sensitizers for temozolomide therapy in AC-GV patients, such as panobinostat and scriptaid. Considering the distinctive clinical characteristics of patients with AC-GII and AC-GV, we constructed the “ACG score” model capable of effectively recognizing patients with these subtypes and predicting patient prognosis. Conclusion Herein, we established novel biologically and clinically relevant molecular classifications for pan-solid tumors and gliomas based on transcriptional expression profiles of HATs and HDACs. Moreover, the ACG score model, calculated by the transcriptional expression of 29 genes, was not only an independent prognostic factor for glioma patients, but can also provide valuable references for promoting more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9506518bda05def7ccbdb74a1bb450d215aa14aa" target='_blank'>
              Pan-cancer analysis of the transcriptional expression of histone acetylation enzymes in solid tumors defines a new classification scheme for gliomas
              </a>
            </td>
          <td>
            Junhao Zhang, Lingbo Li, Aiwei Tang, Chucheng Wang, Yupeng Wang, Yongqi Hu, Guangting He, Wangjun Liao, Rui Zhou
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The epigenetic regulation of clustered protocadherin (cPCDH) genes is tightly linked to their function as specific cell surface barcodes for neural self-nonself discrimination. Differential cPCDH DNA methylation in both blood and brain has been implicated in diverse human traits and diseases ranging from BMI, aging, cancer and brain disorders. However, the unique regulation of cPCDH methylation remains poorly understood. Therefore, we performed a genome-wide association study to evaluate the association of >7 million genetic variants with DNA methylation at 607 cPCDH CpGs measured in whole blood of 3777 individuals and validated findings in prefrontal cortex samples obtained from 523 brain donors. We observed concordant cPCDH methylation patterns in blood and prefrontal cortex, which switched between hypo-, intermediate and hypermethylation over short distances with the former overlapping with the promoter regions of each cPCDH member. Through methylation quantitative trait locus (meQTL) analysis in trans, we first confirmed the broad effect of the candidate gene SMCHD1 on cPCDH methylation in blood which validated in prefrontal cortex. Through a genome-wide analysis, we next identified the SENP7 and UTF1/VENTX loci to have widespread, subcluster-specific effects on cPCDH methylation in blood which was validated in prefrontal cortex. While SENP7 can indirectly affect DNA methylation through the deSUMOylation of the chromatin repressor KAP1, UTF1 and VENTX are two genes involved in embryonic development not previously implicated in epigenetic regulation. Our findings shed new light on the processes involved in cPCDH methylation that may underlie associations with disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67582443cbd9a31a5bbe0cd87bef0d2d7219dcf9" target='_blank'>
              Identification of SENP7 and UTF1/VENTX as new loci influencing clustered protocadherin methylation across blood and brain using a genome-wide association study
              </a>
            </td>
          <td>
            Yunfeng Liu, Maja Vukic, E. Hannon, H. Mei, Jonathan Mill, Lucia Daxinger, B. Heijmans
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The natural ends of chromosomes resemble double-strand breaks (DSBs), which would activate the DNA damage response (DDR) pathway without the protection provided by a specialized protein complex called shelterin. Over the past decades, extensive research has uncovered the mechanism of action and the high degree of specialization provided by the shelterin complex to prevent aberrant activation of DNA repair machinery at chromosome ends in somatic cells. However, recent findings have revealed striking differences in the mechanisms of end protection in stem cells compared to somatic cells. In this review, we discuss what is known about the differences between stem cells and somatic cells regarding chromosome end protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64f88771a224944498f40c132aea82d1a6ba4a" target='_blank'>
              Telomere Protection in Stem Cells.
              </a>
            </td>
          <td>
            Marta Markiewicz-Potoczny, E. L. Denchi
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Gene expression is regulated by chromatin DNA methylation and other features, including histone post-translational modifications (PTMs), chromatin remodelers and transcription factor occupancy. A complete understanding of gene regulation will require the mapping of these chromatin features in small cell number samples. Here we describe a novel genome-wide chromatin profiling technology, named as Nicking Enzyme Epitope targeted DNA sequencing (NEED-seq). NEED-seq offers antibody-targeted controlled nicking by Nt.CviPII-pGL fusion to study specific protein–DNA complexes in formaldehyde fixed cells, allowing for both visual and genomic resolution of epitope bound chromatin. When applied to nuclei, NEED-seq yielded genome-wide profile of chromatin-associated proteins and histone PTMs. Additionally, NEED-seq of lamin B1 and B2 demonstrated their association with heterochromatin. Lamin B1- and B2-associated domains (LAD) segregated to three different states, and states with stronger LAD correlated with heterochromatic marks. Hi-C analysis displayed A and B compartment with equal lamin B1 and B2 distribution, although methylated DNA remained high in B compartment. LAD clustering with Hi-C resulted in subcompartments, with lamin B1 and B2 partitioning to facultative and constitutive heterochromatin, respectively, and were associated with neuronal development. Thus, lamin B1 and B2 show structural and functional partitioning in mammalian nucleus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66e32b18b291507cf88421735ce3ee98c268576f" target='_blank'>
              Distinct structural and functional heterochromatin partitioning of lamin B1 and lamin B2 revealed using genome-wide nicking enzyme epitope targeted DNA sequencing
              </a>
            </td>
          <td>
            Sagnik Sen, P. Estève, Karthikeyan Raman, Julie Beaulieu, H. Chin, G. R. Feehery, U. Vishnu, Shuang-yong Xu, James C. Samuelson, S. Pradhan
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Among the many features of cancer cells, they commonly display aberrant accumulation of RNA-DNA hybrids known as R-loops. R-loops are formed in healthy cells during many cellular processes, including transcription and DNA replication, and are crucial for proper maintenance of telomeres and centromeres, as well as for adequate mitochondrial function. However, aberrant accumulation of R-loops can lead to increases in DNA damage and inflammation. R-loop accumulation has been documented in different types of cancer, including BRCA1-mutant breast cancer, diffuse large B cell lymphoma, and myelodysplastic syndrome (MDS). Indeed, accumulation of R-loops has been proposed as a unifying mechanism underlying MDS and other myeloid malignancies. Ten-eleven translocation enzymes (TET1, TET2, TET3) are 5-methylcytosine dioxygenases that undergo frequent mutational or functional inactivation in human cancers. We have shown that mice with profound, acute TET deficiency induced by deletion of all Tet genes (hereafter referred to as Tet iTKO mice) show rapid onset of an uncontrolled malignant myeloid expansion. Expanded myeloid cells from Tet iTKO mice show increased levels of R-loops, together with increased DNA damage and inflammation, compared to control myeloid cells. Using Tet iTKO hematopoietic stem and progenitor cells (HSPC), we performed an in vivo CRISPR knockout screen of genes encoding RNA and DNA sensors, as well as R-loop regulators, to find how loss of genes related to nucleic acid sensing and innate immune response would affect the proliferation of TET-deficient myeloid cells. We observed that loss of enzymes that prevent the accumulation of R-loops impaired the uncontrolled malignant myeloid expansion of Tet iTKO cells. CRISPR-mediated depletion of the gene encoding RNase H1, an enzyme that specifically degrades the RNA strand in R-loops, further increased R-loop levels and DNA damage, and decreased proliferation and in vivo expansion of TET-deficient myeloid cancer cells, compared to Tet iTKO cells treated with non-targeting sgRNAs. Moreover, the accumulation of DNA breaks in Tet iTKO cells upon Rnaseh1 loss was not observed in Tet wild-type cells similarly depleted for RNase H1, suggesting a role for TET enzymes and potentially TET-oxidized 5-methylcytosines in DNA repair, which will be discussed in this presentation. Overall, these findings indicate that further increasing R-loops in TET-deficient cancer cells blocks their proliferation, and suggest that RNase H depletion or inhibition could be a new therapeutic strategy for cancer treatment in cancers that display augmented levels of R-loops, independent of the presence of TET mutations.
 Citation Format: Isaac F López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, Arthur X Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao. Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcf01e01d7fd229a9e37956d4e689cd570037805" target='_blank'>
              Chromosome distribution of four LTR retrotransposons and 18 S rDNA in coffea eugenioides
              </a>
            </td>
          <td>
            Mariana Cansian Sattler, J. C. Silva, Stéfanie Cristina de Oliveira, W. Clarindo
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary This review presents an overview of recent findings on various epigenetic and post-transcriptional modifications of the epidermal growth factor receptor (EGFR), which activates the oncogenes RAS and MYC. Alterations in EGFR, RAS, and MYC play a crucial role in dysregulated processes that contribute to oncogenesis. Focused, targeted research aimed at understanding the distinct effects of these oncogenes and their carcinogenic modifications has facilitated the successful treatment of various malignancies using inhibitors of EGFR tyrosine kinase activity and other therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6048dcc607aa9e108befb3b09588362de65007" target='_blank'>
              Dynamic Multilevel Regulation of EGFR, KRAS, and MYC Oncogenes: Driving Cancer Cell Proliferation Through (Epi)Genetic and Post-Transcriptional/Translational Pathways
              </a>
            </td>
          <td>
            M. Šereš, Katarina Spacayova, Z. Sulová, Jana Špaldová, Albert Breier, L. Pavlikova
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="LINE-1 retrotransposons, comprising 17% of the genome, drive cancer instability through hypomethylation. The DIAMOND assay, targeting LINE-1 hypomethylation with bisulfite sequencing of cell free DNA, achieved AUCs of 88% to 100% across six cancer types, surpassing mutation-based diagnostics and suggesting utility in early cancer detection and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57cf07128fb65aaad9c55c65c3a93161bd10c47" target='_blank'>
              LINE-1 Retrotransposons as Cell-free DNA Biomarkers for Multi-Cancer Early Detection.
              </a>
            </td>
          <td>
            J. Szymanski, Pradeep S Chauhan, A. Chaudhuri
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8360ddf104ac6e4b2843cdfd9b7edf6a857c0d" target='_blank'>
              Dysregulation of transposable elements and PIWI-interacting RNAs in myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Z. Krejcík, D. Kundrát, Jirí Kléma, A. Hruštincová, I. Trsova, M. Belickova, Jaroslav Čermák, A. Jonášová, Jiri Dostal, M. Dostalova Merkerova
          </td>
          <td>2025-01-23</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In mammals, X-linked dosage compensation involves two processes: X-chromosome inactivation (XCI) to balance X chromosome dosage between males and females, and hyperactivation of the remaining X chromosome (Xa-hyperactivation) to achieve X-autosome balance in both sexes. Studies of both processes have largely focused on coding genes and have not accounted for transposable elements (TEs) which comprise 50% of the X-chromosome, despite TEs being suspected to have numerous epigenetic functions. This oversight is due in part to the technical challenge of capturing repeat RNAs, bioinformatically aligning them, and determining allelic origin. To overcome these challenges, here we develop a new bioinformatic pipeline tailored to repetitive elements with capability for allelic discrimination. We then apply the pipeline to our recent So-Smart-Seq analysis of single embryos to comprehensively interrogate whether X-linked TEs are subject to either XCI or Xa-hyperactivation. With regards to XCI, we observe significant differences in TE silencing in parentally driven “imprinted” XCI versus zygotically driven “random” XCI. Chromosomal positioning and genetic background impact TE silencing. We also find that SINEs may influence 3D organization during XCI. In contrast, TEs do not undergo Xa-hyperactivation. Thus, while coding genes are subject to both forms of dosage compensation, TEs participate only in Xi silencing. Evolutionary and functional implications are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9ae3587e3b90516f521d0a832022aafe22d64b" target='_blank'>
              Dosage compensation of transposable elements in mammals
              </a>
            </td>
          <td>
            Chunyao Wei, Barry A. Kesner, Uri Weissbein, Peera Wasserzug-Pash, Priyojit Das, Jeannie T Lee
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="HOXB13 X285K is a frameshift mutant due to the deletion of one nucleotide, c.853delT, within the HOXB13 stop codon, resulting in an extension of HOXB13 protein by 96 amino acids on the C terminus. It was found primarily in men of West African ancestry and associated with early-onset prostate cancer (PCa) and more advanced stage. Whether and how X285K contributes to PCa progression remains largely unknown. Here, we established isogenic 22Rv1 cell lines with heterozygous wildtype/X285K HOXB13, which recapitulates the genotypes of X285K in PCa patients. In addition, using the unique C-terminal region as an antigen, we developed an antibody that specifically recognizes the HOXB13 X285K protein. Lastly, we demonstrated that X285K retained the function of HOXB13 in suppressing target genes, such as FASN and PSA. In summary, our data suggest that the isogenic cell lines and the antibody generated in this study will be useful for a comprehensive investigation of X285K function in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eed5e8e9d2022eb0eb9f170bbcfe9edb5266f2f" target='_blank'>
              Delineating the roles of HOXB13 X285K variant in prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Samantha Ye, Mingyang Liu, Jindan Yu
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="SUMMARYHuman papillomaviruses (HPVs) are small DNA viruses that are responsible for significant disease burdens worldwide, including cancers of the cervix, anogenital tract, and oropharynx. HPVs infect stratified epithelia at a variety of body locations and link their productive life cycles to the differentiation of the host cell. These viruses have evolved sophisticated mechanisms to exploit cellular pathways, such as DNA damage repair (DDR), to regulate their life cycles. HPVs activate key DDR pathways such as ATM, ATR, and FA, which are critical for maintaining genomic integrity but are often dysregulated in cancers. Importantly, these DDR pathways are essential for HPV replication in undifferentiated cells and amplification upon differentiation. The ability to modulate these DDR pathways not only enables HPV persistence but also contributes to cellular transformation. In this review, we discuss the recent advances in understanding the mechanisms by which HPV manipulates the host DDR pathways and how these depend upon enhanced topoisomerase activity and R-loop formation. Furthermore, the strategies to manipulate DDR pathways utilized by high-risk HPVs are compared with those used by other DNA viruses that exhibit similarities and distinct differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3977d9b3bc13e1ba0a9ea26fcf6605bb8b2b35" target='_blank'>
              How human papillomavirus (HPV) targets DNA repair pathways for viral replication: from guardian to accomplice.
              </a>
            </td>
          <td>
            A. Vats, Lou Laimins
          </td>
          <td>2025-01-27</td>
          <td>Microbiology and molecular biology reviews : MMBR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERV) represent nearly 8% of the human genome. Of these, HERV-K subtype HML-2 is a transposable element that plays a critical role in embryonic development and in the pathogenesis of several diseases. Quantification and characterization of these multiple HML-2 insertions in the human chromosome has been challenging due to their size, sequence homology with each other, and their repetitive nature. We examined a cohort of 222 individuals for HML-2 proviruses 6q14.1 and 7p22.1a, two loci that are capable of producing full-length viral proteins and have been previously implicated in several cancers, autoimmune disorders and neurodegenerative diseases, using long-read DNA sequencing. While the reference genome for both regions suggests these two loci are structurally dissimilar, we found that for both loci about 5% of individuals have a unique tandem repeat-like sequence (three long terminal repeat sequences sandwiching two internal, potentially protein coding sequences), while most individuals have a standard proviral structure (one internal region sandwiched by two long terminal repeats). Moreover, both proviruses can make full-length, or nearly full-length, HERV-K proteins in multiple transcription orientations. The amino acid sequences from different loci in the same transcriptional orientation share sequence homology with each other. These results demonstrate a clear, previously unreported, relationship between HML-2 loci 6q14.1 and 7p22.1a and highlight the utility of long-read sequencing to study repetitive elements. Future studies need to determine if these polymorphisms determine genetic susceptibility to diseases that are associated with them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd67bb0a75d66d24c8cfbfff72aa27068e59aecc" target='_blank'>
              Characterization of novel human endogenous retrovirus structures on chromosomes 6 and 7
              </a>
            </td>
          <td>
            Nicholas Pasternack, Ole Paulsen, Avindra Nath
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245a4598675a5f03862517e366728182b100458" target='_blank'>
              Molecular mechanism targeting condensin for chromosome condensation
              </a>
            </td>
          <td>
            Menglu Wang, Daniel Robertson, Juan Zou, Christos Spanos, J. Rappsilber, A. Marston
          </td>
          <td>2024-12-17</td>
          <td>The EMBO Journal</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background/Objectives: The ~1.6 kb NBPF repeat units in neuroblastoma breakpoint family (NBPF) genes are specific to humans and are associated with cognitive capacity in higher primates. While the number of NBPF monomers/Olduvai sequences in humans is approximately 2–3 times greater than in great apes, the difference in copy number values of canonical NBPF 3mer Higher-order repeats (HORs)/Olduvai triplets between humans and great apes is substantially larger. This study aims to analyze the organization and evolutionary significance of NBPF 3mer HORs/Olduvai triplets in fully sequenced primate genomes. Methods: We applied the global repeat map (GRM) algorithm to identify canonical and variant NBPF 3mer HORs/Olduvai triplets in the complete genomes of humans, chimpanzees, gorillas, and orangutans. The resulting monomer arrays were analyzed using the GRMhor algorithm to generate detailed schematic representations of NBPF HOR organization. Results: The analysis reveals a distinct difference in NBPF-related patterns among these primates, particularly in the number of tandemly organized canonical 3mer HORs/Olduvai triplets: 61 tandemly organized canonical NBPF 3mer HORs/Olduvai triplets in humans, compared to 0 in chimpanzees and orangutans, and 9 in gorillas. When considering only tandemly organized 3mer HORs/Olduvai triplets with more than three copies, the numbers adjust to 36 in humans and 0 in great apes. Furthermore, the divergence between individual NBPF monomers in humans and great apes is twice as high as that observed within great apes. Conclusions: These findings support the hypothesis that the tandem organization of NBPF 3mer HORs/Olduvai triplets plays a crucial role in enhancing cognitive capacity in humans compared to great apes, potentially providing a significant evolutionary advantage. This effect complements the impact of the increased number of individual NBPF monomers/Olduvai sequences, together contributing to a synergistic amplification effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fef2cc1f9844b84173159f8df4b772160e9a99" target='_blank'>
              Neuroblastoma Breakpoint Family 3mer Higher Order Repeats/Olduvai Triplet Pattern in the Complete Genome of Human and Nonhuman Primates and Relation to Cognitive Capacity
              </a>
            </td>
          <td>
            Matko Glunčić, I. Vlahović, M. Rosandić, Vladimir Paar
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression.
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b4d59be0a1ccb5ccd42ae33408325f684d6c1d" target='_blank'>
              RRM1 promotes homologous recombination and radio/chemo-sensitivity via enhancing USP11 and E2F1-mediated RAD51AP1 transcription
              </a>
            </td>
          <td>
            Shuai Yang, Ruru Wang, Lingling Liu, Feng Xu, Xipeng Zhao, Zhicheng Yao, Jie Zhang, Xu Cheng, An Xu, Lijun Wu, Guoping Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor cell networks formed by tumor microtubes (TMs) may play a key role in the development of therapy resistance in glioblastoma (GB). TM-mediated detoxification from radiation-induced reactive oxygen species (ROS) may infer radioresistance. We hypothesize that high linear energy transfer (LET) radiation, which describes the amount of energy deposited by radiation per unit length, interacts directly with the DNA backbone to induce complex lesions and thus might be less dependent on TM-mediated resistance mechanisms. Therefore, we sought to systematically investigate the impact of LET-induced complex DNA damage on TM formation and GB survival. To this end, the formation of TMs, radiation-induced nuclear DNA damage repair foci (RIF), and GB survival were correlated with a gradual increase in LET using a dose series of clinical protons (low), helium (intermediate), and carbon (high) ion beams. Consistent with conventional photon/X-rays, low-LET proton irradiation promoted TM formation in a dose- dependent manner. In contrast, an anti-correlation between LET and TM induction was found, i.e., a decreased network connectivity with gradual increase of LET and formation of complex DNA damage. Consequently, LET increase correlated with reduced cell survival, with the most pronounced cell killing observed after high-LET carbon irradiation. Moreover, the inverse correlation between LET and TM density was further confirmed for a broad range of LET modulated within the carbon ion irradiation spectrum. This is the first report on the relevance of LET as a novel mean to overcome TM network-mediated radioresistance in GB, with ramifications for the clinical translation of high-LET particle radiotherapy to further improve outcome in this still devastating disease. Highlights Tumor microtubes (TM) density correlates with glioblastoma cell viability after X-rays irradiation. Induction of DNA damage and its complexity depends on linear energy transfer (LET) and it correlates with decreased TM density. High-LET radiotherapy with carbon ions significantly reduces TM density and GB survival dependency from TM formation. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c77d1ef913ee8a8c60eddee5504f119ad2317c6" target='_blank'>
              High LET Radiation: A Novel Strategy to Overcome Tumor Microtubes - Mediated Radioresistance in Glioblastoma
              </a>
            </td>
          <td>
            I. Dokic, Federica Ciamarone, D. Hoffmann, J. Bojcevski, D. Krunic, T. Tessonnier, F. Winkler, Jürgen Debus, V. Venkataramani, A. Mairani, W. Wick, Amir Abdollahi
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Formalin preservation of museum specimens has long been considered a barrier to molecular research due to extensive crosslinking and chemical modification. However, recent optimisation of hot alkaline lysis and proteinase K digestion DNA extraction methods have enabled a growing number of studies to overcome these challenges and conduct genome-wide re-sequencing and targeted locus-specific sequencing. The newest, and perhaps most unexpected utility of formalin preservation in archival samples is its ability to preserve in situ DNA-protein interactions at a molecular level. Retrieving this signal provides information about the relative compaction or accessibility of the genome to the transcriptional machinery required for gene expression. Thus, exposure to formalin essentially corresponds to taking a snapshot of organism-wide gene expression at the time of death. While DNA methylation and RNA-Seq analyses of dried tissues have provided glimpses into historical gene regulation, these techniques were previously limited to skeletal or desiccated remains, offering only partial insights. By examining fluid-preserved specimens, molecular tools can now be applied to a broader range of tissues, enabling more detailed tissue-specific gene regulation profiling across vertebrates. In this review, we chronicle the historical use of formaldehyde in collections and discuss how targeted chromatin profiling with assays like MNase-seq and FAIRE-seq are surmounting fixation challenges and unlocking invaluable insights into historical genomes and gene expression profiles. The deeper integration of molecular genetics with museum collections bridges the gap between past and present and provides a vital tool that could help us predict and mitigate some of the impacts of future environmental change, novel pathogens, or invasive species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cdced5ded68073de0b09e74b9af4ad05548ef" target='_blank'>
              Reframing Formalin: A Molecular Opportunity Enabling Historical Epigenomics and Retrospective Gene Expression Studies.
              </a>
            </td>
          <td>
            C. E. Holleley, Erin E. Hahn
          </td>
          <td>2025-01-02</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the evolutionary interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial in vivo genome editing identifies widespread epistasis and an accessible fitness landscape during lung tumorigenesis.
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-05</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract Most of the risk factors associated with chronic and complex diseases, such as cancer, stem from exogenous and endogenous exposures experienced throughout an individual’s life, collectively known as the exposome. These exposures can modify DNA, which can subsequently lead to the somatic mutations found in all normal and tumor tissues. Understanding the precise origins of specific somatic mutations has been challenging due to multitude of DNA adducts (i.e. the DNA adductome) and their diverse positions within the genome. Thus far, this limitation has prevented researchers from precisely linking exposures to DNA adducts and DNA adducts to subsequent mutational outcomes. Indeed, many common mutations observed in human cancers appear to originate from error-prone endogenous processes. Consequently, it remains unclear whether these mutations result from exposure-induced DNA adducts, or arise indirectly from endogenous processes or are a combination of both. In this review, we summarize approaches that aim to bridge our understanding of the mechanism by which exposure leads to DNA damage and then to mutation and highlight some of the remaining challenges and shortcomings to fully supporting this paradigm. We emphasize the need to integrate cellular DNA adductomics, long read-based mapping, single-molecule duplex sequencing of native DNA molecules and advanced computational analysis. This proposed holistic approach aims to unveil the causal connections between key DNA modifications and the mutational landscape, whether they originate from external exposures, internal processes or a combination of both, thereby addressing key questions in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdf82ec8445e0cef06c9a8eb260342f4ef9a96b" target='_blank'>
              Investigating the origins of the mutational signatures in cancer
              </a>
            </td>
          <td>
            Gunnar Boysen, Ludmil B. Alexandrov, R. Rahbari, I. Nookaew, Dave Ussery, M. Chao, Chiung-Wen Hu, Marcus S. Cooke
          </td>
          <td>2025-01-07</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Telomere maintenance is crucial for preventing the linear eukaryotic chromosome ends from being mistaken for DNA double-strand breaks, thereby avoiding chromosome fusions and the loss of genetic material. Unlike most eukaryotes that use telomerase for telomere maintenance, Drosophila relies on retrotransposable elements-specifically HeT-A, TAHRE, and TART (collectively referred to as HTT)-which are regulated and precisely targeted to chromosome ends. Drosophila telomere protection is mediated by a set of fast-evolving proteins, termed terminin, which bind to chromosome termini without sequence specificity, balancing DNA damage response factors to avoid erroneous repair mechanisms. This unique telomere capping mechanism highlights an alternative evolutionary strategy to compensate for telomerase loss. The modulation of recombination and transcription at Drosophila telomeres offers insights into the diverse mechanisms of telomere maintenance. Recent studies at the population level have begun to reveal the architecture of telomere arrays, the diversity among the HTT subfamilies, and their relative frequencies, aiming to understand whether and how these elements have evolved to reach an equilibrium with the host and to resolve genetic conflicts. Further studies may shed light on the complex relationships between telomere transcription, recombination, and maintenance, underscoring the adaptive plasticity of telomeric complexes across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc71fe25065e2464b3633178f88c0066a196f35c" target='_blank'>
              Maintaining Telomeres without Telomerase in Drosophila: Novel Mechanisms and Rapid Evolution to Save a Genus.
              </a>
            </td>
          <td>
            S. Cacchione, G. Cenci, Anne-Marie Dion-Côté, D. Barbash, G. Raffa
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, Greta Schmauck, Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human REXO4 is a poorly characterized exonuclease that is overexpressed in human cancers. To better understand the function of REXO4 and its relationship to cellular proliferation, we have undertaken multidisciplinary approaches to characterize its cell cycle phase-dependent subcellular localization and the cis determinants required for this localization, its importance to cell cycle progression and cell viability, its protein-protein association network, and its activity. We show that the localization of REXO4 to the nucleolus in interphase depends on an N-terminal nucleolar localization sequence and that its localization to the perichromosomal layer of mitotic chromosomes is dependent on Ki67. Depletion of REXO4 led to a G1/S cell cycle arrest, and reduced cell viability. REXO4 associated with ribosome components and other proteins involved in rRNA metabolism. We propose a model where REXO4 is important for proper rRNA processing, which is required for ribosome biogenesis, cell cycle progression, and proliferation. SIGNIFICANCE STATEMENT REXO4 is a putative RNA exonuclease with limited characterization. The authors used in silico, cell, and molecular biology approaches to characterize its localization, associations, regulation, and function. They found that during interphase, REXO4 localizes to the nucleolus through an N-terminal nucleolar localization sequence. Whereas during mitosis, REXO4 localized to the perichromosomal layer in a Ki67-dependent manner. REXO4 was required for proper cell cycle progression, and viability. These results indicated that REXO4 is important for regulating cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21fc9df0b1eebef7ec11d9c5b7fce970a02eabc6" target='_blank'>
              Human REXO4 is Required for Cell Cycle Progression
              </a>
            </td>
          <td>
            Kevin M. Clutario, Mai Abdusamad, Ivan Ramirez, Kayla J. Rich, Ankur A. Gholkar, Julian Zaragoza, Jorge Z. Torres
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In many eukaryotes, meiotic recombination occurs preferentially at discrete sites, called recombination hotspots. In various lineages, recombination hotspots are located in regions with promoter-like features and are evolutionarily stable. Conversely, in some mammals, hotspots are driven by PRDM9 that targets recombination away from promoters. Paradoxically, PRDM9 induces the self-destruction of its targets and this triggers an ultra-fast evolution of mammalian hotspots. PRDM9 is ancestral to all animals, suggesting a critical importance for the meiotic program, but has been lost in many lineages with surprisingly little effect on meiosis success. However, it is unclear whether the function of PRDM9 described in mammals is shared by other species. To investigate this, we analyzed the recombination landscape of several salmonids, the genome of which harbors one full-length PRDM9 and several truncated paralogs. We identified recombination initiation sites in Oncorhynchus mykiss by mapping meiotic DNA double-strand breaks (DSBs). We found that DSBs clustered at hotspots positioned away from promoters, enriched for the H3K4me3 and H3K36me3 and the location of which depended on the genotype of full-length Prdm9. We observed a high level of polymorphism in the zinc finger domain of full-length Prdm9, indicating diversification driven by positive selection. Moreover, population-scaled recombination maps in O. mykiss, Oncorhynchus kisutch and Salmo salar revealed a rapid turnover of recombination hotspots caused by PRDM9 target motif erosion. Our results imply that PRDM9 function is conserved across vertebrates and that the peculiar evolutionary runaway caused by PRDM9 has been active for several hundred million years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f09c603402d3b52a99c356601f8730c3715c9b0" target='_blank'>
              PRDM9 drives the location and rapid evolution of recombination hotspots in salmonid fish
              </a>
            </td>
          <td>
            Marie-Laure Raynaud, Paola Sanna, Julien Joseph, J. Clément, Yukiko Imai, J. Lareyre, Audrey Laurent, Nicolas Galtier, F. Baudat, Laurent Duret, P. Gagnaire, B. de Massy
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Background Gene methylation in cells is an important factor in tumorigenesis, and radiotherapy can change DNA methylation in cells. In this study, complete genome methylation sequencing (BS-Seq) technology was used to analyze the genome-wide methylation of patients with cervical cancer before and after radiotherapy. Methods Three pairs of cervical squamous cell carcinoma samples were collected from patients before and after radiotherapy in July 2020. Genome-wide DNA methylation profiles were generated using WGBS. Bioinformatics analysis was conducted to identify differential methylation regions (DMRs) and their associated genes and pathways. The study focused on the methylation changes of LHX2, LHX5, and LHX9 genes, assessing their expression levels using qRT-PCR and correlating these changes with cervical cancer stages. Results MCG was the main way of genomic DNA methylation in the three patients. The DNA methylation level and methylation density on each chromosome varied greatly. As revealed by comparison of methylation before and after radiation in the three patients, 1287, 1261 and 789 differential methylation genes were identified, respectively. 3) Combined with clinical treatment, methylation level difference and correlation enrichment analysis, it was found that LHX2, LHX5 and LHX9 were closely related to the occurrence and development of cervical cancer. After 5-Aza-DC and radiotherapy, the methylation of the CpG islands in LHX2, LHX5 and LHX9 genes in these patients was decreased (p < 0.01), and the mRNA and protein expression levels were relatively increased (p < 0.01). Conclusion In our present work, genome-wide DNA methylation maps of cervical cancer tissues before and after radiotherapy were successfully constructed. We found that LHX5 and LHX9 genes are closely related to cervical cancer. LHX5 and LHX9 have a negative effect on cervical cancer. The migration ability of LHX9 silenced cells was significantly enhanced after irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fea2269d56f3d3d8896922414dbb511f2559dfa8" target='_blank'>
              Whole-Genome Methylation Sequencing Analysis and Functional Verification of LIM-Homeobox Family Genes in Cervical Cancer
              </a>
            </td>
          <td>
            Rong Yu, Qin Yu, Jie Shi, Xue Meng, Zhiyuan Deng, Jing Suo, Hao Yang
          </td>
          <td>2025-01-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase 1 (PARP1) plays a crucial role in DNA repair and genomic stability maintenance. However, the regulatory mechanisms governing PARP1 activity, particularly through deubiquitination, remain poorly elucidated. Using a deubiquitinase (DUB) library binding screen, we identified cylindromatosis (CYLD) as a bona fide DUB for PARP1 in breast cancer cells. Mechanistically, CYLD is recruited by PARP1 to DNA lesions upon genotoxic stress, where it cleaves K63-linked polyubiquitin chains on PARP1 at residues K748, K940, and K949, resulting in compromised PARP1 activation. In a reciprocal manner, PARP1 PARylates CYLD at sites E191, E231, E259, and E509, thereby enhancing its DUB activity. Consequently, depletion of CYLD leads to increased efficiency in base excision repair and confers breast cancer cells with resistance to alkylating agents. Conversely, overexpression of CYLD enhances sensitivity to PARP inhibitors (PARPi) even in homologous recombination-proficient breast cancer cells. These findings offer unique insights into the intricate interplay between CYLD and PARP1 in DNA repair, underscoring the pivotal role of targeting this regulatory axis for breast cancer chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0849c3c6a3caa581334770da3d981fc152217b00" target='_blank'>
              The CYLD-PARP1 feedback loop regulates DNA damage repair and chemosensitivity in breast cancer cells.
              </a>
            </td>
          <td>
            Miaomiao Zheng, Shuo Wang, Kexin Tang, Ruixue Kong, Xuemeng Wang, Jun Zhou, Yan Chen, Yijie Wang
          </td>
          <td>2024-12-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a614193ffb0a35548300d5126509bf2d1bf067aa" target='_blank'>
              Defining ortholog-specific UHRF1 inhibition by STELLA for cancer therapy
              </a>
            </td>
          <td>
            Wenjing Bai, Jinxin Xu, Wenbin Gu, Danyang Wang, Ying Cui, Weidong Rong, Xiaoan Du, Xiaoxia Li, Cuicui Xia, Qingqing Gan, Guantao He, Huahui Guo, Jinfeng Deng, Yuqiong Wu, R. Yen, Srinivasan Yegnasubramanian, S. Rothbart, Cheng Luo, Linping Wu, Jinsong Liu, Stephen B Baylin, Xiangqian Kong
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Y. Li, C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Simple Summary The homeobox transcription factor NKX3.1 is essential for the development of the prostate and has long been thought of as a tumor suppressor. We hypothesize that much like the androgen receptor (AR), NKX3.1 plays a pro-differentiative, tumor suppressive role in early-stage disease and potentially acts as an oncogene in late-stage, AR-driven prostate cancer. Our results indicate that NKX3.1 is a critical co-factor for AR in prostate cancer cells, and knockdown leads to decreased cell survival, while overexpression is enough to promote growth in androgen depleted and AR-inhibited conditions. Finally, we suggest that NKX3.1 may promote oncogenic activity in late-stage disease through upregulation of genes driven by AR at selective androgen response elements. Altogether, these data indicate that NKX3.1 is necessary and sufficient for prostate cancer growth, and like AR, functions as an oncogene in a context-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d23adb1a041976f4f7a7bd36179e78ccc6d3c88" target='_blank'>
              The Homeobox Transcription Factor NKX3.1 Displays an Oncogenic Role in Castration-Resistant Prostate Cancer Cells
              </a>
            </td>
          <td>
            Audris Budreika, John T. Phoenix, R. Kostlan, Carleen D. Deegan, Marina G Ferrari, K. Young, Sean W. Fanning, S. Kregel
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aea14196d457e30b1e0c0858af5e94a5516dad2" target='_blank'>
              RBM47 promotes cell proliferation and immune evasion by upregulating PDIA6: a novel mechanism of pancreatic cancer progression
              </a>
            </td>
          <td>
            Yihui Ma, Enjie Liu, H. Fan, Chenfei Li, Pei Huang, Meiying Cui, Zhengyang Wang, Jing Zhou, Kuisheng Chen
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon.
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Post-transcriptional regulation by RNA-binding proteins (RBPs) is critical for mRNA stability, localization, and translation, primarily through interaction with the 3’-untranslated region. Dysregulation of RBPs has been associated with various cancers, with fragile X-related protein 1 (FXR1) emerging as a critical RBP involved in tumorigenesis through its interactions with target mRNAs. Despite its significance, the specific role of FXRI in cancer progression remains underexplored. In this study, we investigated FXR1’s function using SH-SY5Y cells. RNA immunoprecipitation (RNA-IP) assay was employed to isolate RNA complexes associated with FXR1. We generated stable cell lines with either FXR1 overexpression or silencing to assess the impact of FXRI binding to mRNA complexes. Subsequent analyses, including quantitative reverse transcription polymerase chain reaction, correlation analysis, gene expression profiling, and survival analysis, were performed to validate the interactions of FXR1 with target mRNAs. Our RNA-IP analysis identified several mRNAs significantly enriched in FXR1-bound RNA complexes, including SHISAL1, SLC43A3, NBAT1, PDZK1IP1, ACKR3, KCNN3, NECAB2, ANO5, ATOH8, IGFBP7, LEMD1, GPR35, WNT7A, and F2RL3. Notably, we observed changes in the expression levels of these genes following FXR1 overexpression or depletion, indicating FXR1-mediated functional regulation. Co-expression analysis further supported FXR1’s association with these target mRNAs. These findings highlight the significant role of FXR1 in regulating the stability and expression of key mRNAs implicated in malignancies. The dysregulation of FXR1 and its interaction with these target transcripts suggest that FXR1 plays a critical role in tumor biology, potentially offering new avenues for therapeutic interventions. This study provides a deeper understanding of FXR1’s involvement in cancer and underscores the importance of RBPs in the post-transcriptional regulatory landscape of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68584e4b61388edd6ff9ff31b6a17585ebe8fc4" target='_blank'>
              FXR1 modulates the expression of oncogenes and tumor suppressor genes associated with poor cancer prognosis
              </a>
            </td>
          <td>
            Faiz Ali Khan, , Shaoping Ji, Jingcheng Dong
          </td>
          <td>2025-02-05</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5202cd39039a85ffc734b688a9ee572ce5b91b15" target='_blank'>
              Inhibition of GSK3β is synthetic lethal with FHIT loss in lung cancer by blocking homologous recombination repair
              </a>
            </td>
          <td>
            Shishi Tao, Yue Pu, E. Yang, Guowen Ren, Changxiang Shi, Li-Jie Chen, Liang Chen, J. Shim
          </td>
          <td>2025-01-06</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast cancer is known as one of the most predominant cancers that affect both females and
males worldwide. The most crucial risk factor in breast cancer is the mutations in the RAD51C
gene that have been considered in most hereditary breast cancers. RAD51C, the RAD51
paralogs, is also a deoxyribonucleic acid (DNA) repair protein related to breast and ovarian
cancers. DNA double-strand breaks (DSBs) account for the significant detrimental form of
DNA damage. RAD51C mutants also have been recognized in breast/ovarian cancer patients.
However, the role of the RAD51C protein in hereditary breast cancer and its three-dimensional
(3D) structures remains unclear. Thus, this study was conducted to identify the 3D structure of
RAD51C protein from its amino acid sequences. The homology modeling for the 3D structure
of the RAD51C protein was carried out by using three automated webservers: I-TASSER,
SWISS-MODEL, and Phyre2. PyMOL was applied to visualize the 3D structure of RAD51C
protein. Next, the MolProbity, ProSA, and SAVES v6.0 programs have been employed to
check the stereo-chemical quality of RAD51C protein. The RAD51C-IT models were found to
be the best models for the RAD1C protein after being evaluated and validated, and the models
were constructed using full-length RAD51C protein sequences. Thus, these protein models can
be utilized as a virtual screening tool in discovering potential inhibitors of RAD51C protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be054c5f435a476d5df616bc8bacb7f34a1b8877" target='_blank'>
              Homology Modeling of Human DNA Repair Protein RAD51 Homolog 3 (RAD51C) in Breast Cancer
              </a>
            </td>
          <td>
            Ruzianisra Mohamed
          </td>
          <td>2024-12-30</td>
          <td>International Journal of Pharmaceuticals, Nutraceuticals and Cosmetic Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258ec54986c78fd95aedc9a7b2b989006fafe5e2" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2824e967c6a4d06b3abc17af2f0703099dbde0" target='_blank'>
              Uncovering the whole genome silencers of human cells via Ss-STARR-seq
              </a>
            </td>
          <td>
            Xiusheng Zhu, Lei Huang, Chao Wang, Guoli Li, Biao Deng, Dashuai Kong, Xiaoxiao Wang, Rongrong Chang, Yi Gu, Qiuhan Wen, Siyuan Kong, Yuwen Liu, Yubo Zhang
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c26d06bb50d508e1ca0550e77b81ac840e53150" target='_blank'>
              PARP4 deficiency enhances sensitivity to ATM inhibitor by impairing DNA damage repair in melanoma
              </a>
            </td>
          <td>
            Yuehua Li, Yu Liu, Jingjing Ma, Yuqi Yang, Q. Yue, Guannan Zhu, Weinan Guo, Tianwen Gao, Qiong Shi, Chunying Li
          </td>
          <td>2025-01-30</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Head and Neck Squamous Cell Carcinoma (HNSCC) is the sixth most common cancer worldwide, and local failure of disease following treatment with radiation therapy remains a major challenge for improving patient outcomes. We therefore designed a CRISPR-Cas9 pooled genetic screen to identify signaling mechanisms that regulate HNSCC radiosensitivity. Using a kinome gRNA library (763 genes with 8 gRNAs each), Cal27 and Detroit562 cells were screened using fractionated exposure to ionizing radiation as a selection pressure. Deep sequencing results identified significant enrichment or depletion of gRNAs for DNA damage signaling (ATM and DNAPK) and NFKB signaling (IRAK4, IRAK1, IKKA, IKKB), pathways known to regulate cell survival after radiation. Surprisingly, the screen also identified significant depletion of Janus kinase signaling (JAK1 and TYK2) to be a cause of radiation resistance. Independent knockout (KO) of Janus Kinase 1 (JAK1) in both Cal27 and Detroit562 cells caused radioresistance in vitro and in vivo. JAK1 KO prolongs G2 cell cycle arrest in HNSCC after treatment with radiation or after thymidine synchronization. JAK1 KO cells also have enhanced Rad51 foci formation indicative of homologous recombination usage for DNA repair. Consistent with an enhanced mitotic arrest, JAK1 KO causes the appearance of cells with 4N and 8N DNA content after exposure to radiation as measured by flow cytometry with propidium iodide or following 5-ethynyl-2’deoxyuridine (EdU) labeling. HNSCC cells with JAK1 KO, or treated with the JAK1 specific inhibitor Abrocitinib, formed significantly fewer micronuclei after radiation treatment consistent with a reduction in mitotic errors. Consistent with these findings, live cell imaging demonstrated cells with JAK1 KO undergo fewer mitotic catastrophe events. Loss of JAK1 function did not alter CDK1, but delayed activation of Aurora Kinase A (AURKA) and Polo-like Kinase 1 (PLK1), identifying a cell cycle signaling pathway regulated by JAK1. Live cell imaging also showed a more than doubling of the time required from chromatin condensation to sister chromatid separation, suggesting an associated defect in kinetochore assembly. The effects of inhibiting Kif18a, a kinesin that regulates kinetochore tension, in combination with radiation was therefore tested. Kif18 inhibition significant radiosensitized JAK1 KO HNSCC in clonogenic survival and xenograft models. In summary we have elucidated a JAK1-dependent mechanism for the regulation of AURKA/PLK1 signaling, a protective G2 arrest, and radioresistance. Furthermore, we provide a mechanistic and targeted pharmacologic approach for overcoming therapeutic resistance and enhancing radiation therapy in HNSCC.
 Citation Format: Vanessa Kelley, Marta Baro, William Gasperi, Chatchai Phoomak, Hojin Lee, Joseph Contessa. CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab6adeb5ebe92c5746446bc4ffdb501c4fd87e05" target='_blank'>
              Abstract B024: CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Vanessa Kelley, Marta Baro, William Gasperi, C. Phoomak, Hojin Lee, Joseph Contessa
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most prevalent forms of cancer globally, and has recently become the leading cause of cancer-related mortality in women. BC is a heterogeneous disease comprising various histopathological and molecular subtypes with differing levels of malignancy, and each patient has an individual prognosis. Etiology and pathogenesis are complex and involve a considerable number of genetic alterations and dozens of alterations in non-coding RNA expression. Non-coding RNAs are part of an abundant family of single-stranded RNA molecules acting as key regulators in DNA replication, mRNA processing and translation, cell differentiation, growth, and overall genomic stability. In the context of breast cancer, non-coding RNAs are involved in cell cycle control and tumor cell migration and invasion, as well as treatment resistance. Alterations in non-coding RNA expression may contribute to the development and progression of breast cancer, making them promising biomarkers and targets for novel therapeutic approaches. Currently, the use of non-coding RNAs has not yet been applied to routine practice; however, their potential has been very well studied. The present review is a literature overview of current knowledge and its objective is to delineate the function of diverse classes of non-coding RNAs in breast cancer, with a particular emphasis on their potential utility as diagnostic and prognostic markers or as therapeutic targets and tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0045975d00155c439ba80ba1b588114b3af277" target='_blank'>
              Non-Coding RNAs in Breast Cancer: Diagnostic and Therapeutic Implications
              </a>
            </td>
          <td>
            Roman Beňačka, Daniela Szabóová, Z. Guľašová, Z. Hertelyová
          </td>
          <td>2024-12-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9f4a8e5c101e35acf29f53374f1c86d8b5fc2b" target='_blank'>
              CircRNF13 enhances IGF2BP1 phase separation-mediated ITGB1 mRNA stabilization in an m6A-dependent manner to promote oral cancer cisplatin chemoresistance
              </a>
            </td>
          <td>
            Xuemeng Xu, Qiu Peng, Zongyao Ren, Yaqian Han, Xianjie Jiang, Zhu Wu, Shiming Tan, Wenjuan Yang, L. Oyang, Xia Luo, Jinguan Lin, Longzheng Xia, Mingjing Peng, Nayiyuan Wu, Yanyan Tang, Hao Tian, Yujuan Zhou, Q. Liao
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutation of DNMT3A, encoding a de novo methyltransferase essential for cytosine methylation, is a common early event in clonal hematopoiesis (CH) and adult acute myeloid leukemia (AML). Spontaneous deamination of methylated cytosines damages DNA, which is repaired by the base excision repair (BER) enzymes MBD4 and TDG. Congenital MBD4-deficiency has been linked to early-onset CH and AML, and is marked by exceedingly high levels of DNA damage and mutation of DNMT3A. Strikingly, wildtype (WT) DNMT3A binds TDG, thereby potentiating its repair activity. Since TDG is the only remaining BER enzyme in MBD4-deficient AML patients capable of repairing methylation damage, we investigated whether mutant DNMT3A negatively affects the repair function of TDG. We found that, whereas WT DNMT3A stimulates TDG function, mutant DNMT3A impairs TDG-mediated repair of DNA damage in vitro. In light of this finding and to extrapolate our observations to the broader AML patient population, we investigate here the genetic profiles and survival outcomes of AML patients with single (SM) versus double mutant (DM) DNMT3A. DM DNMT3A AML patients show a characteristic driver mutation landscape and reduced overall survival when compared to SM DNMT3A AML patients. Importantly, whole-genome sequencing showed a trend for increased DNA damage in primary DM DNMT3A AML samples, especially when DNMT3A mutations are located at the DNMT3A-TDG interaction interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba74fc432bc69631e0bcce279545e026f31ce31c" target='_blank'>
              Double mutant DNMT3A AML: a unique subtype experiencing increased DNA damage and poor prognosis.
              </a>
            </td>
          <td>
            E. Boertjes, Sanne Massaar, A. Zeilemaker, J. Konijnenburg, M. Rijken, F. Kavelaars, T. Grob, J. Versluis, Bob Löwenberg, Peter J M Valk, M. Sanders
          </td>
          <td>2024-12-18</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Zachary Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8663834f0e1d3ecd98b4319fc6ee165b1d3d20b" target='_blank'>
              Analysis of mRNA Pentatricopeptide Repeat Domain 1 as a prospective oncogene in clear cell renal cell carcinoma that accelerates tumor cells proliferation and invasion via the Akt/GSK3β/β-catenin pathway
              </a>
            </td>
          <td>
            Z. Zhou, Yulong Li, Y. Chai, Yong Zhang, Pu Yan
          </td>
          <td>2025-01-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The retinoblastoma (RB) gene is an important tumor suppressor gene with a higher mutation frequency than other tumor suppressor genes. The mutation or inactivation of RB has been found in various cancers. The discovery of small molecules to promote RB expression is an effective anti-cancer strategy. Special DNA secondary structures with G-quadruplex and i-motif on the RB promoter could act as “molecular switches” for gene transcriptional regulation and are potentially important targets for the development of new anti-cancer drugs. After extensive screening, we found that the bisacridine derivative A06 had selective binding and destabilization for both the G-quadruplex and i-motif on the RB promoter, which significantly up-regulated RB gene transcription and translation, resulting in the inhibition of tumor cell proliferation and metastasis. A06 exhibited potent anti-tumor activity on Hela cells and strongly suppressed tumor growth on the Hela xenograft mice model without significant toxicity. In comparison, A02 exhibited strong binding and destabilization to the RB promoter G-quadruplex only, which showed a much weaker effect than A06 on regulating RB expression and producing anti-tumor activity. As we know, this is the first study for up-regulating a tumor suppressor gene through destabilization of both the G-quadruplex and i-motif on the gene promoter, which provides a new strategy for innovative anti-cancer drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2432014fea9848b263f5ea65101b4b9216843d82" target='_blank'>
              Selective Up-Regulation of Tumor Suppressor Gene Retinoblastoma by Bisacridine Derivative Through Gene Promoter Quadruplex Structures for Cancer Treatment
              </a>
            </td>
          <td>
            Xiaomin Lin, Jiahui Zhang, Jihai Liang, Dongsheng Ji, , Ding Li
          </td>
          <td>2025-02-07</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 In colon cancer, the BRAFV600E mutation is strongly associated with the CpG island methylator phenotype (CIMP). Here, we characterized the contribution of BRAFV600E to maintenance of aberrant DNA methylation. A reverse CRISPR gene editing approach was applied to revert the V600E mutation in BRAF back to wildtype (E600V) in organoids derived from a late-stage tumour. DNA methylation analyses identified 5187 differentially methylated CpGs within CpG islands, predominantly hypermethylated (82%) in BRAFV600E organoids, including genes associated with CIMP, as well as Polycomb Repressor Complex 2 (PRC2) target genes. RNA sequencing showed that expression of those genes was concordantly repressed. Furthermore, BRAFV600Einduced high expression of PRC2 core components (EZH2, SUZ12, EED), caused PRC2-induced H3K27 trimethylation in promoter regions, and maintained a PRC2-associated embryonic phenotype. This phenotype was lost following mutation correction or pharmacological inhibition of DNA methylation. These findings show that BRAFV600E maintains aberrant DNA and histone methylation patterns in advanced colon cancer, likely preserving the transformed phenotype by silencing of PRC2 target genes. Epigenetic therapies may have value in the treatment of BRAFV600E-mutant colon cancer.
 Citation Format: Layla El Bouazzaoui, Jeroen M Bugter, Emre Küçükköse, André Verheem, Jasmin B Post, Nicola Fenderico, Inne H.M Borel Rinkes, Hugo J.G Snippert, Madelon M Maurice, Onno Kranenburg. BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aeb4732d6e0e4b7c91f83caa7be8cc3275c4570" target='_blank'>
              Abstract B003: BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer
              </a>
            </td>
          <td>
            Layla El Bouazzaoui, J. Bugter, Emre Küçükköse, A. Verheem, J. Post, N. Fenderico, Inne H. M. Borel Rinkes, Hugo J. Snippert, M. Maurice, O. Kranenburg
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
α thalassemia/mental retardation syndrome X-linked (ATRX) serves as a part of the sucrose nonfermenting 2 (SNF2) chromatin-remodeling complex. In interphase, ATRX localizes to pericentromeric heterochromatin, contributing to DNA double-strand break repair, DNA replication, and telomere maintenance. During mitosis, most ATRX proteins are removed from chromosomal arms, leaving a pool near the centromere region in mammalian cells, which is critical for accurate chromosome congression and sister chromatid cohesion protection. However, the function and localization mechanisms of ATRX at mitotic centromeres remain largely unresolved.


METHODS
The clustered regularly interspaced short palindromic repeats with CRISPR-associated protein 9 (CRISPR-Cas9) system and overexpression approaches were employed alongside immunofluorescence to investigate the mechanism of ATRX localization at the centromere. To study the binding mechanism between ATRX and heterochromatin protein 1 (HP1), both full-length and truncated mutants of hemagglutinin (HA)-ATRX were generated for co-immunoprecipitation and glutathione S-transferase (GST)-pull assays. Wild-type ATRX and HP1 binding-deficient mutants were created to investigate the role of ATRX binding to HP1 during mitosis, with the Z-Leu-Leu-Leu-al (MG132) maintenance assay, cohesion function assay, and kinetochore distance measurement.


RESULTS AND CONCLUSIONS
Our research demonstrated that HP1α, HP1β, and HP1γ facilitate the positioning of ATRX within the mitotic centromere area through their interaction with the first two [P/L]-X-V-X-[M/L/V] (PxVxL)motifs at the N-terminus of ATRX. ATRX deficiency causes aberrant mitosis and decreased centromeric cohesion. Furthermore, reducing Wapl activity can bypass the need for ATRX to protect centromeric cohesion. These results provide insights into the mechanism of ATRX's centromeric localization and its critical function in preserving centromeric cohesion by reducing Wapl activity in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2508ee19573d906286f1694bdd429742f1bd10" target='_blank'>
              HP1 Promotes the Centromeric Localization of ATRX and Protects Cohesion by Interfering Wapl Activity in Mitosis.
              </a>
            </td>
          <td>
            Erchen Zhang, Lei Peng, Kejia Yuan, Zexian Ding, Qi Yi
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, E. Robertson
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            Ze-Hao Liu, Peng Ma, Ying He, Yuejiao Zhang, Zuo Mou, Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022ef2a36db22fd62af61dcb734a88029a1a5b21" target='_blank'>
              Celastrol promotes DNA damage and apoptosis in uterine corpus endometrial carcinoma via promotion of KAT2B-mediated RBPJ acetylation and repression of MCM4 transcription
              </a>
            </td>
          <td>
            Lipeng Pei, Yan Zhu, Wenpeng Li, Bin Xu, Jingli Sun
          </td>
          <td>2025-02-03</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: The highly polymorphic Major Histocompatibility Complex (MHC) genomic region, located on the short arm of chromosome 6, is implicated genetically in Parkinson’s disease (PD), a progressive neurodegenerative disorder with motor and non-motor symptoms. Previously, we reported significant associations between SINE-VNTR-Alu (SVA) expression quantitative trait loci (eQTLs) and Human Leucocyte Antigen (HLA) class I genotypes in PD. In this study, we aimed to evaluate SVA associations and their regulatory effects on transposable element (TE) transcription in the MHC class I region. Methods: Transcriptome data from the peripheral blood cells of 1530 individuals in the Parkinson’s Progression Markers Initiative (PPMI) cohort were reanalyzed for TE and gene expression using publicly available bioinformatics tools, including Salmon and Matrix-eQTL. Results: Four structurally polymorphic SVAs regulated the transcription of 18 distinct clusters of 235 TE loci, comprising LINEs (33%), SINEs (19%), LTRs (35%), and ancient transposon DNA elements (12%) located near HLA genes. The transcribed TEs were predominantly short, with an average length of 445 nucleotides. The regulatory effects of these SVAs varied significantly in terms of TE types, numbers, and transcriptional activation or repression. The SVA-regulated TE RNAs in blood cells appear to function as enhancer-like elements, differentially influencing the expression of HLA class I genes, non-HLA genes, and noncoding RNAs. Conclusions: These findings highlight the roles of SVAs and their associated TEs in the complex regulatory networks governing coding and noncoding gene expression in the MHC class I region, with potential implications for immune function and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7af8d419a6f31034c910704dd0806ace1362d2" target='_blank'>
              The Transcription of Transposable Elements Differentially Regulated by SVAs in the Major Histocompatibility Complex Class I Region of a Parkinson’s Progression Markers Initiative Cohort
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2025-01-06</td>
          <td>Journal of Molecular Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N. Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent studies have demonstrated that the production of bidirectional enhancer-derived transcripts (eRNAs) is a characteristic of an active Cis-regulatory element (CRE). Higher levels of eRNA synthesis correlate with the activation of histone modifications, a potentially valuable tool for deciphering the complexity of the gene regulatory network. To understand the changes of CREs during gonadal development in mice, we collected gonadal WT1-positive cells from the piggyBac-Wt1-mCherry-2A-EGFP (PBWt1-RG) reporter strain at E13.5, E16.5, and P0 in both sexes and conducted Cap Analysis of Gene Expression analysis (CAGE) which is capable to capture transcriptional starting site (TSS). We compared the levels of intergenic bidirectional RNA, i.e, potentially eRNA, according to sex at each stage (testis somatic cells vs ovary somatic cells at E13.5, E16.5, and P0) and stage in each sex (E13.5 vs E16.5, E16.5 vs P0, and E13.5 vs P0 in testis somatic cells or ovary somatic cells). Intergenic RNAs with significant changes (|Log2FC| > 1, p < 0.05) were selected. The TSS profile of intergenic RNA changed more profoundly in testis somatic cells than in ovary somatic cells, suggesting embryonic testicular development is driven by larger changes in a transcriptional regulatory network than ovarian development. Based on the profiles of the predicted transcription factors (TFs) that would bind to the active CREs during gonadal development, the NR4A, EGR, and TCF3 families would be novel TFs to play pivotal roles in gonadal development. Identifying active CREs using eRNAs would provide a means to comprehensively understand the transcriptional regulatory system, leading to valuable insights into the gonadal development of male and female individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8877e8a45402ec1d0fc04d7b02b6aa2118971232" target='_blank'>
              Analysis of Functional Cis-regulatory Elements Reveals Novel Transcriptional Regulatory Mechanisms in Gonadal Development.
              </a>
            </td>
          <td>
            S. Kirino, Ryuichi Nakagawa, Maki Gau, K. Takasawa, Yasuhiro Murakawa, H. Kawaji, Y. Hayashizaki, Tomohiro Morio, K. Kashimada
          </td>
          <td>2025-01-20</td>
          <td>Sexual development : genetics, molecular biology, evolution, endocrinology, embryology, and pathology of sex determination and differentiation</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692b233441e797a73e14defb069a03a55d2ca324" target='_blank'>
              DNA methylation profiling at base-pair resolution reveals unique epigenetic features of early-onset colorectal cancer in underrepresented populations
              </a>
            </td>
          <td>
            J. S. Li, Karen Riggins, Li Yang, Chaorong Chen, Patricia Castro, Wedad Alfarkh, Neda Zarrin-Khameh, Michael E. Scheurer, Chad J. Creighton, Benjamin Musher, Wei Li, Lanlan Shen
          </td>
          <td>2025-01-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 High-grade serous carcinoma (HGSC) remains the deadliest gynecological cancer, mostly due to the widespread metastases in abdominal cavities already at diagnosis. To improve therapy success, understanding metastasis routes and their molecular characteristics is crucial. Herein, we used >400 samples subjected to whole-genome bisulfite (WGBS) and RNA sequencing from 125 patients with HGSC, enrolled to prospective, longitudinal, multi-region, observational DECIDER trial (NCT04846933). These real-world epigenetics data allowed, for the first time, to delineate metastatic routes in HGSC. Our results show that cancer cells released from the site-of-origin sites seed to both mid-abdomen and upper abdomen in parallel. The dominant mechanism responsible for almost half of the spread was direct surface-to-surface release of ovarian tumor cells into the peritoneal cavity. The major source of cancer cells in the pelvic area was ovary, from which cancer disseminated to all intra-abdominal organs, including omentum. Tumor cells from pelvic bowel displayed strong tropism for virtually all metastatic deposits, including such distant metastases, as liver, vagina, and lungs, which suggested hematogenous spread. Ascites play a prominent role in fueling colonization, accounting for additional 20% of direct spread. The seeding via ascitic fluid remained localized within the abdominal area, however, ascites colonized primarily para-aortic and inguinal lymph nodes. Lymph node metastases were noted in one-third of possible spreading modes. HGSC dissemination via inguinal lymph nodes represented a rare, but aggressive trajectory, as these tumors, originated directly from tube were prone to invade all distant locations. Comparison of the paired samples before and after chemotherapy revealed that most sensitive to neoadjuvant chemotherapy (NACT) tumors are adnexal and omental tumors. Peritoneum and mesentery showed prominent resistance to treatment, as well as lymph nodes. Thus, patients with metastases in inguinal lymph nodes and bowel are at the highest risk of developing widespread intra-abdominal and distant metastases. Furthermore, the standard-of-care platinum/taxane therapy, along with maintenance relapse therapies are ineffective for eradication of these tumors. Ascites displayed an intriguing case: being virtually non-responding to NACT, epigenetic profiles at relapse exhibited a large change. This implies that epigenetic processes in ascites evolve along with the cancer progression, driving chemoresistance. Taken together, using DNA methylation changes from >400 samples we revealed metastasis routes in HGSC, as well as tissue-specific chemotherapy treatment response. Dissection of the key regulatory pathways of metastatic routes helps identify patients at the highest risk of distant metastases and suggests new targets for therapies.
 Citation Format: Alexandra Lahtinen, Giovanni Marchi, Kari Lavikka, Daria Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, Taru A Muranen, Jaana Oikkonen, Antti Häkkinen, Anni Virtanen, Johanna Hynninen, Sampsa Hautaniemi. DNA methylation reveals metastatic colonization routes in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f704aa07c09b6c48c3c1f8d354bbb48ab6359478" target='_blank'>
              Abstract PR003: DNA methylation reveals metastatic colonization routes in ovarian cancer
              </a>
            </td>
          <td>
            A. Lahtinen, G. Marchi, K. Lavikka, D. Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, T. Muranen, J. Oikkonen, Antti Häkkinen, A. Virtanen, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BRCA1 deficiency is observed in approximately 25% of triple-negative breast cancer (TNBC). BRCA1, a key player of homologous recombination (HR) repair, is also involved in stalled DNA replication fork protection and repair. Here, we investigated the sensitivity of BRCA1-deficient TNBC models to the frequently used replication chain terminator gemcitabine, which does not directly induce DNA breaks. A large fraction of BRCA1-deficient cells was sensitive to gemcitabine, in contrast to their isogenic BRCA1-proficient counterparts. Gemcitabine treated BRCA1-deficient cells accumulated massive levels of single strand DNA (ssDNA) and presented no RPA or RAD51 nuclear foci. The gemcitabine-induced accumulation of ssDNA in BRCA1-deficient cells was strongly diminished by targeting MRE11 with inhibitors and by siRNA attenuation. In contrast, treatment with the PARP1/2 inhibitor olaparib did not result in MRE11 dependent over-resection. Furthermore, a fraction of gemcitabine treated BRCA1-deficient cells that showed massive ssDNA accumulation slipped into mitosis, producing mitotic bridges and strongly stained BrdU and γH2AX micronuclei (MN). The BrdU-positive MN and DNA bridges also stained positively for cGAS. In conclusion, these data suggest that gemcitabine treatment in BRCA1-deficient TNBC exposes unprotected nascent DNA linked to replication fork reversal, which leads to MRE11 over-resection and ssDNA accumulation. Therefore, the observed hypersensitivity to gemcitabine indicates that it could be a beneficial addition to BRCA1-deficient TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d16464aa7109edc218441355d65fc601eddce9b" target='_blank'>
              Replication-Poison Treatment in BRCA1-Deficient Breast Cancer Causes MRE11 Over-Resection that Induces Single-Stranded DNA Accumulation and Mitotic Catastrophe.
              </a>
            </td>
          <td>
            I. Tabet, Esin Orhan, Ermes Candiello, Lise Fenou, C. Velázquez, B. Orsetti, Geneviéve Rodier, William Jacot, C. Ribeyre, Claude Sardet, C. Theillet
          </td>
          <td>2025-01-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10−10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5′-TGT*−3′ motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential. IMPORTANCE Yarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e564005ed56626c695c387291b72229a4171b45" target='_blank'>
              Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a Mendelian condition.
              </a>
            </td>
          <td>
            Mitchell R. Vollger, Jonas Korlach, K. Eldred, Elliott G. Swanson, Jason G. Underwood, Stephanie C. Bohaczuk, Yizi Mao, Y. H. Cheng, Jane Ranchalis, Elizabeth E. Blue, Ulrike Schwarze, Katherine M. Munson, Christopher T. Saunders, Aaron M. Wenger, Aimee Allworth, S. Chanprasert, Brittney L. Duerden, Ian A. Glass, M. Horike‐Pyne, Michelle Kim, K. Leppig, Ian J. McLaughlin, Jessica Ogawa, E. A. Rosenthal, Sam Sheppeard, Stephanie M Sherman, Samuel Strohbehn, Amy L Yuen, Andrew W Stacey, Thomas A Reh, Peter H. Byers, M J Bamshad, F. Hisama, Gail P. Jarvik, Yasemin Sancak, Katrina M. Dipple, A. Stergachis
          </td>
          <td>2025-01-29</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 DNA methylation (DNAm) reflects regions of the genome that are transcriptionally repressed or active depending on the methylation level. Multitudes of DNAm “clocks” have been created to predict biological age and health outcomes. Deviations in biological age from chronological age reflect an alteration in the normal biological aging process. Clonal hematopoiesis of indeterminate potential (CHIP) arises in hematopoietic stem cells that have accumulated somatic mutations in key driver genes which provide a selective advantage for clonal expansion. Subjects with CHIP mutations are more prone to negative clinical outcomes such as coronary heart disease. While DNAm and CHIP have been studied for the past decade, the associations between the two are only now starting to become apparent. In this study we propose an improved measurement of aging based on the Jenson-Shannon distance (JSD) and associate this measurement with CHIP and adverse clinical outcomes. Bisulfite amplicon sequencing of 20 CpG-rich targets that change in methylation with age was performed on an initial cohort of 155 individuals from the National Institute of Neurological Disorders and Stroke (NINDS). JSD was used to construct a DNA methylation chaos clock (DMC) on these targets. DMC provided better predictions of chronological age than DNAm alone (r=0.9; p<0.0001). We validated our findings in 286 individuals from Cooper University Hospital for Management of Trauma where the accuracy of the DMC was confirmed in correlations of predicted vs actual age (r=0.86; p<0.0001). We next examined the association between DMC age acceleration and clinical outcomes. It was found that in individuals over the age of 50, DMC acceleration was associated with smoking and disease incidence (e.g. congestive heart failure and chronic obstructive pulmonary disease) independent of chronological age. The incidence of CHIP mutations in three key driver genes (DNMT3A, TET2, and ASXL1) was assayed in a subset of 345 individuals from both cohorts. Individuals with CHIP mutations in key driver genes had greater deviations in their predicted age than individuals without CHIP mutations (p<0.001). This study demonstrates that the DMC assay is a candidate for improvements in our measurements of biological age and it reflects important changes in an individual’s biological aging process.
 Citation Format: Anthony J Pompetti, Woonbok Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre Issa. DNA methylation variability in aging and Clonal Hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadd86e514d184ff56ec546d04b1399f1bbf484d" target='_blank'>
              Abstract A016: DNA methylation variability in aging and Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Anthony J Pompetti, W. Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitor (ICI) therapy is a promising anti-tumor therapeutic strategy; however, its efficacy in solid tumors is limited. Chromosome missegregation is common in various solid tumors; however, its role in tumor progression remains poorly understood, and its correlation with ICI is yet to be explored. Here, it is found that increased chromosome missegregation promotes tumor immune microenvironment, and eventually immunotherapeutic efficacy, by triggering pyroptosis. yin yang 2 (YY2) is identified as a mitotic checkpoint regulator, which promotes chromosome missegregation by upregulating BUB1B transcription. Increased chromosome missegregation promoted the formation of micronuclei and release of double-stranded DNA (dsDNA) into the cytosol, triggering an AIM2-mediated cytosolic dsDNA response. The subsequent pyroptosis strengthened the tumor immune microenvironment, thereby enhancing immunoinfiltration and cytotoxicity of CD8+ T cells, while preventing their exhaustion. Finally, through in vitro and in vivo experiments, it is demonstrated that combining YY2 overexpression-induced chromosome missegregation/cytosolic dsDNA response and PD-1 inhibitor significantly enhanced the efficacy of ICI immunotherapy in microsatellite instable and microsatellite stable colorectal cancer cells. Together, these findings provide new insights on the role of chromosome missegregation in triggering cytosolic dsDNA response-mediated pyroptosis and modulating the tumor immune microenvironment, suggesting a novel strategy for improving ICI therapeutic efficacy in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/210fffb6c8ca89fde7413dc4cb1951b37eabd3d0" target='_blank'>
              Chromosome Missegregation Triggers Tumor Cell Pyroptosis and Enhances Anti-Tumor Immunotherapy in Colorectal Cancer.
              </a>
            </td>
          <td>
            Wei Duan, R. Hosea, Lingxian Wang, Cao Ruan, Fuqiang Zhao, Jingyi Liu, Hezhao Zhao, Makoto Miyagishi, Shourong Wu, Vivi Kasim
          </td>
          <td>2025-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 247


 Background:
 MET signaling is implicated in the tumorigenesis of many solid tumors, including CRC. MET-targeted therapies are approved in non-small cell lung cancer with novel agents in clinical trials for tumors with
 MET
 exon 14 splice site mutations,
 MET
 amplifications, and recently MET expression. There is emerging therapeutic interest in targeting MET in CRC.
 Methods:
 Tumor samples from 50,500 cases of clinically advanced CRC were analyzed by hybrid capture-based comprehensive genomic profiling (CGP) that evaluated all classes of genomic alterations (GA). MSI-high status, tumor mutational burden (TMB), genomic ancestry, trinucleotide mutational signatures, and homologous recombination deficiency signature (HRDsig) were assessed for patients with activating
 MET
 GA. PD-L1 expression was determined by IHC (Dako 22C3 with TPS scoring system). Results were compared using the Fisher exact test with the Benjamini-Hochberg adjustment.
 Results:
 A total of 502 (1.0%) cases of activating GA of
 MET
 (METmut) were identified, including 418 cases (0.83%) with
 MET
 amplification (METamp), 10 (<0.1%) rearrangements, and 78 (0.1%) short variant (SV) mutations with 19 (<0.1%) cases of
 MET
 exon 14 splice site mutations. When compared with
 MET
 wild-type (METwt) CRC, the METmut CRC were of similar age (median 62 years), and numerically more likely to be male (61.4% vs 56.0%; P=.05). There were no differences in genomic ancestry (72% European ancestry for both groups). The METmut CRC featured more cases with a POLE signature (3.6% vs 0.4%; P=<0.0001) and more GA per tumor (7 vs 5; P<0.0001). MSI-high status was similar in both groups, with 4.2% in METmut and 6.0% in METwt (P=0.28). TMB levels were numerically higher in the METmut cases (TMB ≥ 10 mutations/Mb at 12.2% vs 9.0%; P=0.058). Low level PD-L1 expression (1-49% TPS) was similar in both groups (13.9% in METmut vs 13.3% in METwt), and while not statistically significant, high level PD-L1 expression (>50% TPS) was higher in the METmut group (4.3% vs 1.6%; P=0.056). The frequencies of HRDsig+ CRC were rare and similar in both groups with 1.5% in METmut and 1.7% in METwt. METmut CRC had lower frequencies of GA in
 KRAS
 (34.7% vs 48.8%; P<0.0001),
 NRAS
 (1.8% vs 4.2%; P=0.011),
 APC
 (72.5% vs 78.2%; P=0.009) and
 PIK3CA
 (12.2% vs 18.9%; P=0.0004), and higher frequencies of GA in
 CDK6
 (10.2% vs 0.6%; P<0.0001),
 EGFR
 (5.2% vs 0.2%; P=0.0002),
 ERBB2
 (9.6% vs 5.2%; P=0.0004),
 HGF
 (8.0% vs 0.8%; P<0.0001),
 POLE
 (4.2% vs 0.5%; P=0.005) and
 TP53
 (84.1% vs 75.9%; P<0.0001). In METmut CRC, ERBB2 alterations were 4.1% amplifications, 2.9% sequence mutations, 0.9% with more than one mutation.
 BRAF
 V600E was seen in 5.9% in METmut CRC and 8.4% of METwt.
 Conclusions:
 CGP reveals significant differences in the genomic landscapes of METmut CRC and METwt CRC, which may guide selection of potential rational drug combinations with novel MET-targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbce0e305dedc1251ad0c64f1286171de877cd3f" target='_blank'>
              Landscape of
 MET
 genomic alterations in colorectal cancer (CRC).
              </a>
            </td>
          <td>
            Faiza Yasin, D. Pavlick, J. Ross, M. Cecchini
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a difficulty and bottleneck in the clinical treatment of breast cancer due to a lack of effective therapeutic targets. Herein, we first report that secernin 2 (SCRN2), an uncharacterized gene in human cancer, acts as a novel tumor suppressor in TNBC to inhibit cancer progression and enhance therapeutic sensitivity to poly(ADP-ribose) polymerase (PARP) inhibition both in vitro and in vivo. SCRN2 is downregulated in TNBC through chaperone-mediated autophagic degradation, and its downregulation is associated with poor patient prognosis. Moreover, SCRN2 impedes the proteasomal degradation of histone-lysine N-methyltransferase 2C (KMT2C) by recruiting Bcl2-associated athanogene 2 to block the interaction of KMT2C with E3 ubiquitin-protein ligase CHIP. Consistently, SCRN2 transcriptionally activates Bcl2-modifying factor by amplifying histone H3 monomethylation at lysine 4 at its enhancer, thereby inducing intrinsic apoptosis. Notably, KMT2C knockdown restores the impaired TNBC progression caused by SCRN2 overexpression both in vitro and in vivo. Furthermore, SCRN2 decreases the expression of key DNA repair-related genes and induces endogenous DNA damage, thus conferring therapeutic sensitivity of TNBC cells to PARP inhibition.   Collectively, these findings identify SCRN2 as a novel suppressor of TNBC, reveal its mechanism of action, and highlight its potential role in TNBC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25afa7f93ec640694b6cc33af309a4cb4f092490" target='_blank'>
              Secernin-2 Stabilizes Histone Methyltransferase KMT2C to Suppress Progression and Confer Therapeutic Sensitivity to PARP Inhibition in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Minhua Huang, Jia-Yang Cai, Shaoyou Yang, Qian Zhao, Zhi-Min Shao, Fang-Lin Zhang, Yin-Ling Zhang, A-Yong Cao, Da-Qiang Li
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Non-coding RNA transcribed from active enhancers, known as enhancer RNA (eRNA), is a critical element in gene regulation with a highly specific expression pattern in the regulatory networks of tumor-infiltrating cells. Therefore, eRNA signatures could potentially be applied to represent anti-tumor immune cells and to improve cancer immunotherapy. Herein, we identified thousands of eRNAs that were significantly correlated with infiltrating immune cell abundance in more than 10,000 patient samples across a variety of cancer types. The expression of these eRNAs was mediated by transcription factors with high expression in anti-tumor immune cells, as identified through single-cell assays. An eRNA immunotherapy signature (eRIS) developed using the anti-tumor eRNAs was highly associated with the objective response rate (ORR) of immunotherapy and was elevated in patients who benefited from immune checkpoint blockade (ICB) treatment. In comparison with a signature based on protein-coding genes, the eRIS was more effective in predicting the response to immunotherapy. Integration of the eRIS with pharmacogenomic data revealed hundreds of anti-cancer drugs that have the potential to enhance immunotherapy efficacy. Finally, treatment of a mouse model of IDH mutant glioma with the histone deacetylase inhibitor vorinostat improved the effects of anti-PD-1 immunotherapy through increased abundance of infiltrating immune cells. Taken together, this study developed an eRIS with demonstrated efficacy in predicting immunotherapy response and used the eRIS to identify a series of effective combination drugs, thus highlighting the clinical utility of the eRIS in immunotherapy enhancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb404431d7b2aee513f3b1e9892ee993135259d" target='_blank'>
              Characterization of an Enhancer RNA Signature Reveals Treatment Strategies for Improving Immunotherapy Efficacy in Cancer.
              </a>
            </td>
          <td>
            Chenyang Zhang, Yan-Yan Chen, Shuyu Chen, Yunzhe Wang, Yifan Yuan, Xiwen Yang, Wei Hu, Bo Chen, Zengxin Qi, Jason T Huse, Yun Liu, Bo Wen, Xiuping Liu, Leng Han, Yuxiang Wang, Zhao Zhang
          </td>
          <td>2025-02-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVE
Our study presents a novel analysis of the oncogenes and tumor suppressor proteins directly modulated by E6/E7 of high-risk HPV types 16 and 18, in colorectal cancer (CRC).


METHODS
HCT 116 (KRAS mutant) & HT-29 (TP53 mutant) cell models of CRC were transduced with E6/E7 of HPV16 and HPV18, individually and in combination. Further, we utilized a liquid chromatography mass spectrometry (LC-MS/MS) approach to analyze and compare the proteomes of both CRC cell models.


RESULTS
We generated six stably transduced cell lines. Our data revealed a significantly higher, HPV-induced modulation of oncogenes and tumor suppressor proteins in the TP53 mutant model, as compared to the KRAS mutant model (p ≤ 0.01). Less than 1% of the genes were commonly modulated by HPV, between both models. We also report that HT-29 cells, expressing E6/E7 of both HPV types, significantly reduced the suppression of oncogenes as compared to cells expressing E6/E7 of either HPV types individually (p-value ≤0.00001).


CONCLUSION
Our data imply that HPV coinfections leads to the sustenance of a pro-oncogenic environment in CRC. HPV modulates different oncogenes/tumor suppressor proteins in CRC of varying mutational backgrounds, thus highlighting the importance of personalized therapies for such diseases with mutational heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8413199da82e1f8d1d547e5a890f2a85923ff990" target='_blank'>
              Oncoproteins E6/E7 of the human papillomavirus types 16 & 18 synergize in modulating oncogenes and tumor suppressor proteins in colorectal cancer.
              </a>
            </td>
          <td>
            Queenie Fernandes, Lubna Therachiyil, Shahd M. Younis, Said Dermime, A. Al Moustafa
          </td>
          <td>2025-01-16</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Synthetic lethality approaches in BRCA1/2-mutated cancers have focused on poly(ADP-ribose) polymerase (PARP) inhibitors, which are subject to high rates of innate or acquired resistance in patients. Here, we used CRISPR/Cas9-based screening to identify DNA Ligase I (LIG1) as a novel target for synthetic lethality in BRCA1-mutated cancers. Publicly available data supported LIG1 hyperdependence of BRCA1-mutant cells across a variety of breast and ovarian cancer cell lines. We used CRISPRn, CRISPRi, RNAi, and protein degradation to confirm the lethal effect of LIG1 inactivation at the DNA, RNA, and protein level in BRCA1-mutant cells in vitro. LIG1 inactivation resulted in viability loss across multiple BRCA1-mutated cell lines, whereas no effect was observed in BRCA1/2 wild-type cell lines, demonstrating target selectivity for the BRCA1-mutant context. On-target nature of the phenotype was demonstrated through rescue of viability with exogenous wild-type LIG1 cDNA. Next, we demonstrated a concentration-dependent relationship of LIG1 protein expression and BRCA1 mutant cell viability using a titratable, degradable LIG1 fusion protein. BRCA1 mutant viability required LIG1 catalytic activity, as catalytically dead mutant LIG1K568A failed to rescue viability loss caused by endogenous LIG1 depletion. LIG1 perturbation produced proportional increases in PAR staining in BRCA1 mutant cells, indicating a mechanism consistent with the function of LIG1 in sealing ssDNA nicks. Finally, we confirmed LIG1 hyperdependence in vivo using a xenograft model in which LIG1 loss resulted in tumor stasis in all mice. Our cumulative findings demonstrate that LIG1 is a promising synthetic lethal target for development in patients with BRCA1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831e285c6dc25f79bcb13593b00dacbe2a024fd9" target='_blank'>
              LIG1 is a synthetic lethal target in BRCA1 mutant cancers.
              </a>
            </td>
          <td>
            Lauren Martires, Leanne G Ahronian, Charlotte B Pratt, Nikitha M Das, Xiaobin Zhang, D. Whittington, Hongxiang Zhang, B. Shen, Jon Come, Patrick McCarren, Mu-Sen Liu, Chengyin Min, Tianshu Feng, Haris Jahic, Janid A Ali, Daniel R Aird, Fang Li, Jannik N Andersen, A. Huang, William D Mallender, Hilary E. Nicholson
          </td>
          <td>2025-01-27</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Inhibitors of DNA methylation such as 5-aza-deoxycytidine are widely used in experimental and clinical settings. However, their mechanism of action is such that DNA damage inevitably cooccurs with loss of DNA methylation, making it challenging to discern their respective effects. Here we deconvolute the effects of decreased DNA methylation and DNA damage on cancer cells, by using degron alleles of key DNA methylation regulators (DNMT1 and UHRF1). We report that cancer cells with decreased DNA methylation — but no DNA damage— enter cellular senescence, with G1 arrest, senescence-associated secretory phenotype (SASP) expression, and senescence-associated beta-galactosidase (SA-β-gal) positivity. This senescence is independent of p53 and Rb, but involves p21, which is cytoplasmic and inhibits apoptosis. The suppression of cyclic GMP-AMP synthase (cGAS) alleviates SASP response and lysosomal activity, and this process is independent of stimulator of interferon genes (STING). Xenograft experiments show that tumor cells can be made senescent in vivo by decreasing DNA methylation. These findings reveal the intrinsic effects of loss of DNA methylation in cancer cells and have practical implications for future therapeutic approaches.
 Citation Format: Xiaoying Chen, Kosuke Yamaguchi, Brianna Rodgers, Delphine Goehrig, David Vindrieux, Xavier Lahaye, Matthieu Nolot, Laure Ferry, Nadine Martin, Pierre Dubus, Fumihito Miura, Takashi Ito, Nicolas Manel, Masato Kanemaki, David Bernard, Pierre-Antoine Defossez. DNA methylation protects cancer cells against senescence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324fb87b02d8939636995cfccebb627de6d9cb0d" target='_blank'>
              Abstract B002: DNA methylation protects cancer cells against senescence
              </a>
            </td>
          <td>
            Xiaoying Chen, Kosuke Yamaguchi, Brianna Rodgers, D. Goehrig, D. Vindrieux, Xavier Lahaye, Matthieu Nolot, Laure Ferry, Nadine Martin, Pierre Dubus, F. Miura, Takashi Ito, Nicolas Manel, Masato Kanemaki, David Bernard, P. Defossez
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 RNaseH2 plays a critical role in removing mis-bounded RNA from the DNA double strand helix1. Absence of RNaseH2 results in spontaneous double-stranded DNA breaks and cytosolic release of dsDNA2. We have previously shown that epithelial deficiency of RNaseH2b (H2bΔIEC) leads to a p53-dependent stem cell suppression and additional deficiency of p53 results in spontaneous intestinal carcinogenesis3. Due to the central role of cGAS/STING in sensing dsDNA, we aim to investigate the role of epithelial STING in a mouse model of chronic epithelial DNA damage and tumor prevention in this study.



 DNA damage was assessed using comet assay in ModeK cell line with cGAS and STING knock out. Protein kinase activity were assessed by Pamgene Kinase Assay. Intestinal organoids were generated from H2bΔIEC and H2bΔIEC/Tmem173-/-mice. Colony forming assay and EdU staining were performed in all genotypes to assess the impact of STING on epithelial stem cell function and proliferation were assessed by CellTiter Glo assay.
 RNA sequencing was performed in intestinal organoids. Spontaneous carcinogenesis was assessed in 1-year aged mice.



 Loss of RNaseH2b results in activation of cGAS/STING pathway and subsequent upregulation of IFN-I signature. Deficiency of cGAS/STING lead to impaired IFN-I independent DNA damage repair which were accessed by WB and comet assay. During the DNA damage response, STING but not cGAS mediates the P53 pathway and the P21-CDK axis. Organoid EdU stanning displayed restored proliferation in H2bΔIEC/Tmem173-/- compared to H2bΔIEC. RNA sequencing from H2bΔIEC/Tmem173-/-organoid further confirmed decreased G2/M DNA damage checkpoint and P53 signature compared to H2bΔIEC. In vivo, we show that H2bΔIEC/Tmem173-/- mice have downregulated DNA damage repair and developed spontaneous small bowel adenocarcinoma after 53 weeks. Tumor derived from H2bΔIEC/Tmem173-/- mice show sensitivity to CDK inhibitor consistent with in vitro validation.



 We provide evidence that non-canonical role of epithelial STING mediates DNA damage response and guides P21-CDK1 axis in preventing tumor genesis. Our research provided evidence understand the role of STING in DNA damage response and intestine carcinogenesis.



 1.Reijns, Martin A.M., et al., Enzymatic Removal of Ribonucleotides from DNA Is Essential for Mammalian Genome Integrity and Development. Cell, 2012. 149(5): p. 1008-1022.
 2.Bartsch, K., et al., Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum Mol Genet, 2017. 26(20): p. 3960-3972.
 3.Aden, K., et al., Epithelial RNase H2 Maintains Genome Integrity and Prevents Intestinal Tumorigenesis in Mice. Gastroenterology, 2019. 156(1): p. 145-159 e19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0539a26d1a5ba9dc2728ef562232f66e1b1c39" target='_blank'>
              P0212 Non-canonical STING signaling in intestinal epithelial cells guides DNA damage-driven stem cell function and protects from intestinal carcinogenesis
              </a>
            </td>
          <td>
            H. Xiang, C. Woitaske-Proske, J. Bernardes, G. R. Martini, F. Wottawa, J. Bornhaeuser, N. Kakavand, Y. Guang, Q. Wu, J. Kugler, L. Welz, B. Konukiewitz, C. Peifer, S. Schreiber, M. Tschurtschenthaler, P. Rosenstiel, K. Aden
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive. In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby preserving HSC function and promoting age-dependent expansion. Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increases in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decreases in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging. In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.
 Citation Format: Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Ionizing radiation induces various types of DNA damage, and the reparability and lethal effects of DNA damage differ depending on its spatial density. Elucidating the structure of radiation-induced clustered DNA damage and its repair processes will enhance our understanding of the lethal impact of ionizing radiation and advance progress toward precise therapeutics. Previously, we developed a method to directly visualize DNA damage using atomic force microscopy (AFM) and classified clustered DNA damage into simple base damage clusters (BDCs), complex BDCs and complex double-strand breaks (DSBs). This study investigated the repair of each type of damage in DNA-repair-deficient human TK6 cells and elucidated the association between each type of clustered DNA damage and the pathway responsible for its repair postirradiation with low linear energy transfer (LET) radiation (X-rays) and high-LET radiation (Fe-ion beams) in cells. We found that base excision repair and, surprisingly, nucleotide excision repair restored simple and complex BDCs. In addition, the number of complex DSBs in wild-type cells increases 1 h postirradiation, which was most likely caused by BDC cleavage initiated with DNA glycosylases. Furthermore, complex DSBs, which are likely associated with lethality, are repaired by homologous recombination with little contribution from nonhomologous-end joining.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d06d3a70f8c910aac38121d146c8464277e3a84c" target='_blank'>
              Deciphering repair pathways of clustered DNA damage in human TK6 cells: insights from atomic force microscopy direct visualization
              </a>
            </td>
          <td>
            Toshiaki Nakano, Ken Akamatsu, Masaoki Kohzaki, Masataka Tsuda, R. Hirayama, Akira Sassa, Manabu Yasui, Mahmoud I. Shoulkamy, Takeshi Hiromoto, Taro Tamada, Hiroshi Ide, Naoya Shikazono
          </td>
          <td>2024-12-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Activation of lineage-specific gene expression programs is mediated by recruitment of lineage-specific transcription factors and their coactivators to chromatin. The lineage factor PAX8 drives essential gene expression in ovarian cancer cells and is required for tumor proliferation. However, the molecular details surrounding co-factor recruitment and specific activation of transcription by PAX8 remain unknown. Here, we identify an important functional interaction between PAX8 and the SWItch/Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complex. We show that PAX8 can recruit SWI/SNF complexes to DNA, where they function to open chromatin and facilitate expression of PAX8 target genes. Genetic deletion of PAX8 results in loss of SWI/SNF from PAX8 bound enhancers, loss of expression of associated target genes, and reduced proliferation. These results can be phenocopied by pharmacological inhibition of SWI/SNF ATPase activity. These data indicate that PAX8 mediates the expression of an essential ovarian cancer proliferative program in part by the recruitment of the SWI/SNF complex, highlighting a novel vulnerability in PAX8 dependent ovarian cancer. Implications: PAX8 recruits SWI/SNF complexes to enhancers, to mediate expression of genes essential for ovarian cancer proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3a38e5d065ce1adf6165485026d4c8154f2562" target='_blank'>
              PAX8 interacts with the SWI/SNF complex at enhancers to drive proliferation in ovarian cancer.
              </a>
            </td>
          <td>
            Kostianna Sereti, Anna E Russo, Ryan Raisner, Taylur P Ma, Karen E Gascoigne
          </td>
          <td>2025-02-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Circular RNA (CircRNA)s, a newly discovered type of noncoding RNAs, have been found to play a role in controlling the development and aggressiveness of tumors. Abnormal control of circRNA has been observed in various types of human cancers, including bladder cancer, hepatocellular carcinoma (HCC), breast cancer, and gastric cancer (GC). CircRNAs possess binding sites for microRNAs (miRNAs) and function as miRNA sponges in posttranscriptional regulation. This mechanism has been documented to influence the course of cancer. Significantly, among these putative circRNAs, circular RNA protein tyrosine kinase 2 (circPTK2) exhibited increased expression and displayed a substantial association with adverse clinical characteristics and a negative prognosis. The production of these transcripts occurs via a back-splicing mechanism. The enclosed conformation of circRNAs shields them from destruction and enhances their potential as biomarkers. Gaining insight into the molecular mechanisms involved in these processes would aid in the development of treatment approaches and the discovery of new tumor markers. This article provides a comprehensive assessment of the latest research on the biosynthesis and features of circRNAs. It examines the role of circPTK2 in the diagnosis, treatment, and prognosis evaluation of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd1e91ef7563824a8ebebdc925d90d66cc38471" target='_blank'>
              CircPTK2 as a Valuable Biomarker and Treatment Target in Cancer.
              </a>
            </td>
          <td>
            Chengqiu Yan, Yu Du, Lihong Cui, Han Bao, Hui Li
          </td>
          <td>2025-01-30</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be5c20b4cd0ad9d6c8146c5936042d6f39bb61" target='_blank'>
              CRYβB2 alters cell adhesion to promote invasion in a triple-negative breast cancer cell line
              </a>
            </td>
          <td>
            Amr A. Waly, London Harper, Jodie M. Fleming, Lindsey M. Costantini
          </td>
          <td>2025-01-21</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Viral mimicry refers to an active antiviral response triggered by the activation of endogenous retroviruses (ERVs), usually manifested by the formation of double-stranded RNA (dsRNA) and activation of the cellular interferon response, which activates the immune system and produces anti-tumor effects. Epigenetic studies have shown that epigenetic modifications (e.g. DNA methylation, histone modifications, etc.) play a crucial role in tumorigenesis, progression, and treatment resistance. Particularly, alterations in DNA methylation may be closely associated with the suppression of ERVs expression, and treatment by demethylation may restore ERVs activity and thus strengthen the tumor immune response. Therefore, we propose that viral mimicry can induce immune responses in the tumor microenvironment by activating the expression of ERVs, and that epigenetic alterations may play a key regulatory role in this process. In this paper, we review the intersection of viral mimicry, epigenetics and tumor immunotherapy, and explore the possible interactions and synergistic effects among the three, aiming to provide a new theoretical basis and potential strategies for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916b8820f91cde5f4b984b3eba25d2fd40a973b1" target='_blank'>
              Exploring viral mimicry combined with epigenetics and tumor immunity: new perspectives in cancer therapy
              </a>
            </td>
          <td>
            Ruirui Wang, Xin Dong, Xiongjian Zhang, Jinzhuang Liao, Wei Cui, Wei Li
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CRISPR-Cas9, awarded the 2020 Nobel Prize in Chemistry for its discovery by Jennifer Doudna and Emmanuelle Charpentier, has revolutionized gene editing due to its versatility, precision, and efficiency. Unlike previous gene editing tools such as Zinc Finger Proteins (ZFP) and TALENs, CRISPR-Cas9 allows targeted editing of any DNA sequence with minimal off-target effects. This article explores the potential of CRISPR-Cas9 technology to edit telomeresprotective DNA structures at chromosome endsand related genes to address telomere-associated diseases and aging mechanisms. Telomeres, consisting of repetitive DNA sequences and associated proteins, are crucial in protecting chromosomes and regulating cellular aging, with their progressive shortening linked to cancer and other age-related diseases. This article mainly discusses the molecular mechanisms of telomere maintenance, the feasibility of using CRISPR-Cas9 for telomere length control, and its implications for cancer, stem cell research, and telomere-associated diseases. While promising, further research is needed to fully understand the roles of regulatory pathways and non-genetic factors in telomere dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c900abb6d0862046c94ca8ef8e27867352e34ebe" target='_blank'>
              Applications of CRISPR-Cas9 Technology in Editing Telomeres and Their Related Control Genes
              </a>
            </td>
          <td>
            Tianye Liu
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e63083f2ebe39f75bcbba803ff0b3dd9f4e01b8c" target='_blank'>
              The role of exosomal non-coding RNAs in the breast cancer tumor microenvironment.
              </a>
            </td>
          <td>
            Mohamed J Saadh, O. Q. B. Allela, Radhwan Abdul Kareem, S. Ballal, Mamata Chahar, Suman Saini, G. V. S. Prasad, Hayder Naji Sameer, Atheer Khdyair Hamad, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronic hepatitis B virus (HBV) infection can significantly increase the incidence of cirrhosis and liver cancer, and there is no curative treatment. The persistence of HBV covalently closed circular DNA (cccDNA) is the major obstacle of antiviral treatments. cccDNA is formed through repairing viral partially double-stranded relaxed circular DNA (rcDNA) by varies host factors. However, the detailed mechanisms are not well characterized. To dissect the biogenesis of cccDNA, we took advantage of an in vitro rcDNA repair system to precipitate host factors interacting with rcDNA and identified co-precipitated proteins by mass spectrometry. Results revealed the MRE11–RAD50–NBS1 (MRN) complex as a potential factor. Transiently or stably knockdown of MRE11, RAD50 or NBS1 in hepatocytes before HBV infection significantly decreased viral markers, including cccDNA, while reconstitution reversed the effect. Chromatin immunoprecipitation assay further validated the interaction of MRN complex and HBV DNA. However, MRN knockdown after HBV infection showed no effect on viral replication, which indicated that MRN complex inhibited the formation of cccDNA without affecting its stability or transcriptional activity. Interestingly, Mirin, a MRN complex inhibitor which can inhibit the exonuclease activity of MRE11 and MRN-dependent activation of ATM, but not ATM kinase inhibitor KU55933, could decrease cccDNA level. Likewise, the MRE11 endonuclease activity inhibitor PFM01 treatment decreased cccDNA. MRE11 nuclease assays indicated that rcDNA is a substrate of MRE11. Furthermore, the inhibition of ATR-CHK1 pathway, which is known to be involved in cccDNA formation, impaired the effect of MRN complex on cccDNA. Similarly, inhibition of MRE11 endonuclease activity mitigated the effect of ATR-CHK1 pathway on cccDNA. These findings indicate that MRN complex cooperates with ATR-CHK1 pathway to regulate the formation of HBV cccDNA. In summary, we identified host factors, specifically the MRN complex, regulating cccDNA formation during HBV infection. These findings provide insights into how HBV hijacks host enzymes to establish chronic infection and reveal new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54b77d859792051cb6318950a8aecf132b1dd9b2" target='_blank'>
              Hepatitis B virus hijacks MRE11–RAD50–NBS1 complex to form its minichromosome
              </a>
            </td>
          <td>
            Kaitao Zhao, Jingjing Wang, Zichen Wang, Mengfei Wang, Chen Li, Zaichao Xu, Qiong Zhan, Fangteng Guo, Xiaoming Cheng, Yuchen Xia
          </td>
          <td>2025-01-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="INhibitor of Growth (ING1-5) proteins are epigenetic readers that target histone acetyltransferase (HAT) or histone deacetylase (HDAC) complexes to the H3K4Me3 mark of active transcription. ING5 targets Moz/Morf and HBO1 HAT complexes that alter acetylation of H3 and H4 core histones, affecting gene expression. Previous experiments in vitro indicated that ING5 functions to maintain stem cell character in normal and in cancer stem cells. Here we find that CRISPR/Cas9 ING5 knockout (KO) mice are sub-fertile but show no decrease in lifespan or ability to heal wounds despite indications of depleted stem cell pools in several tissues. ING5 KO mouse embryo fibroblasts accumulate in G2 of the cell cycle, have high levels of abnormal nuclei and show high basal levels of the γH2AX indicator of DNA damage. KO animals also develop severe dermatitis at a 5-fold higher rate that wild-type littermates. Consistent with ING5 serving a tumor suppressive role, ING5 KO mice developed germinal centre diffuse large B-cell lymphomas at a rate 6-fold higher than control mice at 18 months of age. These data suggest that ING5 functions in vivo to maintain stem cell character in multiple organs, that reduction of stem cell populations is not limiting for murine lifespan and that like a subset of other ING family members, ING5 functions as a tumor suppressor in hematopoietic cells in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7856dbb73e87aa51549dc70eedb5371d011208e6" target='_blank'>
              Knockout of the ING5 epigenetic regulator confirms roles in stem cell maintenance and tumor suppression in vivo
              </a>
            </td>
          <td>
            Buthaina Al Shueili, Arthur Dantas, Etienne Mahe, Tak-Ho Chu, Yang Yang, Elodie Labit, E. Kutluberk, Nicolas Lasaleta, Anand Masson, Hiba Omairi, Kenichi Ito, Roman J. Krawetz, R. Midha, Gregory Cairncross, Karl Riabowol
          </td>
          <td>2025-01-09</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18bddf54a0b7a939c01f99932e638ad6022b7e70" target='_blank'>
              CHAF1B promotes the progression of lung squamous-cell carcinoma by inhibiting SETD7 expression.
              </a>
            </td>
          <td>
            Zhuo Zheng, Yongfang Lin, Hua Guo, Zheng Liu, Xiaoliang Jie, Guizhen Wang, Guangbiao Zhou
          </td>
          <td>2025-01-25</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ASXL transcriptional regulator 1 (ASXL1) is one of the three most frequently mutated genes in age-related clonal hematopoiesis (CH), alongside DNA methyltransferase 3 alpha (DNMT3A) and Tet methylcytosine dioxygenase 2 (TET2). CH can progress to myeloid malignancies including chronic monomyelocytic leukemia (CMML) and is also strongly associated with inflammatory cardiovascular disease and all-cause mortality in humans. DNMT3A and TET2 regulate DNA methylation and demethylation pathways, respectively, and loss-of-function mutations in these genes reduce DNA methylation in heterochromatin, allowing derepression of silenced elements in heterochromatin. In contrast, the mechanisms that connect mutant ASXL1 and CH are not yet fully understood. CH/CMML-associated ASXL1 mutations encode C-terminally truncated proteins that enhance the deubiquitinase activity of the ASXL-BAP1 "PR-DUB" deubiquitinase complex, which removes monoubiquitin from H2AK119Ub. Here, we show that ASXL1 mutant proteins interact with the euchromatic histone lysine methyltransferases 1 and 2 (EHMT1-EHMT2) complex, which generates H3K9me1 and me2, the latter a repressive modification in constitutive heterochromatin. Compared to cells from age-matched wild-type mice, we found that expanded myeloid cells from old (≥18-mo-old) Asxl1tm/+ mice, a heterozygous knock-in mouse model of CH, display genome-wide decreases of H3K9me2, H3K9me3, and H2AK119Ub as well as an associated increase in expression of transposable elements (TEs) and satellite repeats. Increased TE expression was also observed in monocytes from ASXL1-mutant CMML patients compared to monocytes from healthy controls. Our data suggest that mutant ASXL1 proteins compromise the integrity of both constitutive and facultative heterochromatin in an age-dependent manner by reducing the levels of H3K9me2/3 and H2AK119Ub. This increase in TE expression correlated with increased expression of nearby genes, including many interferon-inducible (inflammation-associated) genes (ISGs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e7123fc0bf81467e6f3de419413862ec10652" target='_blank'>
              A mutant ASXL1-BAP1-EHMT complex contributes to heterochromatin dysfunction in clonal hematopoiesis and chronic monomyelocytic leukemia.
              </a>
            </td>
          <td>
            Zhen Dong, Hugo Sepulveda, Leo J. Arteaga-Vazquez, Chad Blouin, Jenna A. Fernandez, Moritz Binder, Wen-Chien Chou, H. Tien, M. Patnaik, Geoffrey J Faulkner, Samuel A Myers, Anjana Rao
          </td>
          <td>2025-01-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Hypoxia is a hallmark symptom of solid tumors, including Ewing Sarcoma (EwS), which promotes tumor progression through a variety of complex biochemical and genetic pathways. The development of hypoxic tissues in tumors activates hypoxia inducible factors (HIF) which are believed to increase therapy resistance and metastasis. Endothelial Per-Arnt-Sim (PAS) domain-containing protein 1 (EPAS1) is known to encode an understudied HIF subunit called HIF-2α which may be a potential target for future therapies. A Tibetan population living in the Himalayan belt at high altitude has been shown to possess a Denisovan variation of the EPAS1 gene with reduced expression that contributes to their population’s hypoxia resistance. The relationship between thriving under hypoxic conditions and reduced expression of EPAS1 suggests this gene may be significantly involved in progressing hypoxic cancers. The objective of this study is to investigate the function of the EPAS1 gene under hypoxic conditions and its role in promoting cancer using the Tibetan allele as a convenient model for reduced expression. Three phases of experiments investigating tumor growth, therapy resistance, and metastasis will be performed using spheroid models and scratch wound analysis on EwS cells transfected with the Tibetan allele of EPAS1 (EPAS1t). It is expected that reduced EPAS1 expression will correlate to increased tumor growth, therapy resistance, and cell migration. These results would highlight EPAS1, and consequentially HIF-2α, as a biomarker for patient prognosis and a potential target for future therapies. Future studies may wish to expand this experiment by repeating the study with alternative cell lines, using animal models, or investigating the impacts of upregulated EPAS1 in hypoxic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd13604fabef519fb5b89720cbebb9b333e46457" target='_blank'>
              A Tibetan Allele of EPAS1 as a Model for Investigating the Role of HIF-2α in Ewing Sarcoma
              </a>
            </td>
          <td>
            Julia E. Shaw
          </td>
          <td>2024-12-16</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Increased telomerase activity has been considered as a conspicuous sign of human cancers. The catalytic cores of telomerase involve a reverse transcriptase and the human telomerase RNA (hTR). However, current detection of telomerase is largely limited to its activity at the tissue and single-cell levels. To reveal the precise distribution of subcellular hTR and telomerase activity, here a modular engineered DNA nanodevice (DNA-ND) is designed capable of imaging hTR and telomerase activity in cytoplasm and nucleus, enabling colocalization analysis. DNA-ND is a modular DNA complex comprising hTR and telomerase activity detection modules, which respectively sense intercellular hTR and telomerase activity via target-sensitive allosteric transition of DNA switches, actuating orthogonal activation of fluorescence signals to achieve in situ co-imaging of hTR and telomerase activity. By integrating DNA-ND with specific localized signals, the DNA-ND based precise profiling of colocalization of hTR and telomerase activity in different cell lines as well as their dynamic changes during pharmacological interventions is demonstrated. Notably, the results suggest that the locations of hTR and telomerase activity are not exactly overlapped, indicating the influence of intracellular environment on the binding of hTR to telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a34a17a7f6af1948fe0851efc3da900886ee2f" target='_blank'>
              A Modular Engineered DNA Nanodevice for Precise Profiling of Telomerase RNA Location and Activity.
              </a>
            </td>
          <td>
            Shiyi Zhang, Jian Lv, Ze-Rui Zhou, Peter X Geng, Dawei Li, Ruocan Qian, H. Ju
          </td>
          <td>2024-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Occurrence of degenerative interactions is thought to serve as a mechanism underlying hybrid unfitness in most animal systems. However, the molecular mechanisms underpinning the genetic interaction and how they contribute to overall hybrid incompatibilities are limited to only a handful of examples. A vertebrate model organism, Xiphophorus, is used to study hybrid dysfunction, and it has been shown from this model that diseases, such as melanoma, can occur in certain interspecies hybrids. Melanoma development is due to hybrid inheritance of an oncogene, xmrk, and loss of a co-evolved tumor modifier. It was recently found that adgre5, a G protein-coupled receptor involved in cell adhesion, is a tumor regulator gene in naturally hybridizing Xiphophorus species Xiphophorus birchmanni (X. birchmanni) and Xiphophorus malinche (X. malinche). We hypothesized that 1 of the 2 parental alleles of adgre5 is involved in regulation of cell growth, migration, and melanomagenesis. Accordingly, we assessed the function of adgre5 alleles from each parental species of the melanoma-bearing hybrids using in vitro cell growth and migration assays. In addition, we expressed each adgre5 allele with the xmrk oncogene in transgenic medaka. We found that cells transfected with the X. birchmanni adgre5 exhibited decreased growth and migration compared to those with the X. malinche allele. Moreover, X. birchmanni allele of adgre5 completely inhibited melanoma development in xmrk-transgenic medaka, while X. malinche adgre5 expression did not exhibit melanoma suppressive activity in medaka. These findings provide evidence that adgre5 is a natural melanoma suppressor and provide new insight in melanoma etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35473f18fdca14702fd87c87ffcf454b6421341" target='_blank'>
              Functional test of a naturally occurred tumor modifier gene provides insights to melanoma development
              </a>
            </td>
          <td>
            Mateo Garcia-Olazabal, M. Adolfi, B. Wilde, Anita Hufnagel, Rupesh Paudel, Yuan Lu, Svenja Meierjohann, Gil Rosenthal, Manfred Schartl
          </td>
          <td>2025-01-17</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cell division cycle-associated (CDCA) genes are dysregulated in carcinomas. Our study aims to identify similarities and differences of the clinical roles of CDCAs in breast cancer (BRCA) and to explore their potential mechanisms. In GEPIA, compared to normal tissues, expressions of CDCAs were higher in BRCA and sub-types. In addition, CDCAs were significantly positively related to stages and predicted worse survival in BRCA. In CancerSEA, expression levels of most CDCAs were strongly positively related to cell cycle, DNA damage, DNA repair, and proliferation. In TIMER, CDCAs were linked with immune infiltration levels of BRCA, including Dendritic cell, B cell and so on, and were positively related to most of the common markers of immune cells, especially CD38 of B cell and IL12RB2 of Th1. In GeneMANIA, there were complex interactions and co-expression relationships between CDCAs and cell division-associated genes. In addition, CDCA1, CDCA3, CDCA5, CDCA6 and CDCA8 had a high proportion of amplification in BRCA, and CDCA1, CDCA2, CDCA5, CDCA7 and CDCA8 had high levels of body DNA methylation. Among 11 transcription factors possibly combining promoters of all CDCAs, FOXP3 and YY1 were significantly higher in BRCA in comparison to normal tissues, and both had a positive relationship with all CDCAs in GEPIA and IHC. In addition, silencing FOXP3 or YY1 decreased levels of CDCAs in MDA-MB-231. In summary, CDCAs have various similarities in clinical functions, functional states, immune infiltration, and mechanisms, and they may become novel potential biomarkers for BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0573bd3b9dd89f3992773c13c07581660383688" target='_blank'>
              Roles of the CDCA gene family in breast carcinoma
              </a>
            </td>
          <td>
            Wei Ding, Wei Han, Chun-Tao Shi, Li-qian Yao, Zhi-wei Liang, Ming-hui Zhou, Hao-nan Wang
          </td>
          <td>2025-01-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [4, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>