<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-10 09:22:38 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>30</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>135</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>107</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>317</td>
          <td>135</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>274</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) are circular DNA bodies that play critical roles in tumor progression and treatment resistance by amplifying oncogenes across a wide range of cancer types. ecDNA lack centromeres and are thus not constrained by typical Mendelian segregation, enabling their unequal accumulation within daughter cells and associated increases in copy number. Despite intrinsic links to their oncogenic potential, the fidelity and mechanisms of ecDNA inheritance are poorly understood. Here, we show that ecDNA are protected against cytosolic mis-segregation through mitotic clustering and by tethering to the telomeric and subtelomeric regions of mitotic chromosomes. ecDNA-chromosome tethering depends on BRD4 transcriptional co-activation and mitotic transcription of the long non-coding RNA PVT1, which is co-amplified with MYC in colorectal and prostate cancer cell lines. Disruption of ecDNA-chromosome tethering through BRD4 inhibition, PVT1 depletion, or inhibiting mitotic transcription results in cytosolic mis-segregation, ecDNA reintegration, and the formation of homogeneously staining regions (HSRs). We propose that nuclear inheritance of ecDNA is facilitated by an RNA-mediated physical tether that links ecDNA to mitotic chromosomes and thus protects against cytosolic mis-segregation and chromosomal integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations, and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated copies of large genomic segments, identify multi-way interactions, distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Dear Editor Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark1,2, with a pervasive presence across cancers3,4. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate ecDNA’s behaviors and functions 5-7. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction while we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Chromosomal instability in gastric cancer cells is associated with the amplification of oncogenes that encode receptor tyrosine kinases (RTKs), such as HER2 and FGFR2; such gene amplification varies from cell to cell and manifests as genetic heterogeneity within tumours. The intratumoural genetic heterogeneity of RTK gene amplification causes heterogeneity in RTK protein expression, which has been suggested to be associated with therapeutic resistance to RTK inhibitors; however, the underlying mechanism is not fully understood. Here, we show that extrachromosomal DNA (ecDNA) causes intratumoural genetic heterogeneity in RTKs and drug resistance due to diverse dynamic changes. We analysed the dynamics of FGFR2 and MYC ecDNA in a gastric cancer cell line after single-cell cloning. Similar to those in parental cells, the copy numbers of FGFR2 and MYC in subclones differed significantly between cells, indicating intraclonal genetic heterogeneity. Furthermore, the ecDNA composition differed between subclones, which affected FGFR2 protein expression and drug sensitivity. Interestingly, clone cells that were resistant to the FGFR2 inhibitor AZD4547 presented diverse changes in ecDNA, including chimeric ecDNA, large ecDNA and increased ecDNA numbers; these changes were associated with high expression and rephosphorylation of FGFR2. Conversely, when resistant clone cells were cultured under conditions that excluded AZD4547, the ecDNA status became similar to that of the original clone cells, and the inhibitory effect on cell growth was restored. Implications: Our results show that dynamic quantitative and qualitative changes in ecDNA can drive the intratumoural genetic heterogeneity of RTKs and resistance to RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The most common chemotherapeutics induce DNA damage to eradicate cancer cells, yet defective DNA repair can propagate mutations, instigating therapy resistance and secondary malignancies. Structural variants (SVs), arising from copy-number-imbalanced and -balanced DNA rearrangements, are a major driver of tumor evolution, yet understudied posttherapy. Here, we adapted single-cell template-strand sequencing (Strand-seq) to a HER2+ breast cancer model to investigate the formation of doxorubicin-induced de novo SVs. We coupled this approach with nucleosome occupancy (NO) measurements obtained from the same single cell to enable simultaneous SV detection and cell-type classification. Using organoids from TetO-CMYC/TetO-Neu/MMTV-rtTA mice modeling HER2+ breast cancer, we generated 459 Strand-seq libraries spanning various tumorigenesis stages, identifying a 7.4-fold increase in large chromosomal alterations post-doxorubicin. Complex DNA rearrangements, deletions, and duplications were prevalent across basal, luminal progenitor (LP), and mature luminal (ML) cells, indicating uniform susceptibility of these cell types to SV formation. Doxorubicin further elevated sister chromatid exchanges (SCEs), indicative of genomic stress persisting posttreatment. Altered nucleosome occupancy levels on distinct cancer-related genes further underscore the broad genomic impact of doxorubicin. The organoid-based system for single-cell multiomics established in this study paves the way for unraveling the most important therapy-associated SV mutational signatures, enabling systematic studies of the effect of therapy on cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model.
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            Gabriel E Rech, Alyssa C Lau, Rachel L Goldfeder, Rahul Maurya, Alexey V Danilov, Chia-Lin Wei
          </td>
          <td>2025-01-15</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumors with low expression of Interferon-Stimulated Genes (ISG) and Antigen Presentation (AP) genes respond relatively poorly to current immunotherapies. One of the early hallmarks of cancer is DNA hypomethylation in genomic repeat regions, resulting in the expression of normally silenced endogenous “viral” elements. Such epigenetic changes have the potential to augment anti-tumor immune responses as well as reduce tumor cell fitness through the generation of aberrant nucleic acid species (NAS) and consequent activation of NAS-sensing pathways. Therefore, tumor evolution should favor additional selective events that suppress NAS generation, possibly yielding specific therapeutic vulnerabilities. Here, we show that the Lysine Demethylase 5 (KDM5) family of epigenetic regulatory enzymes suppress R-loop formation in genomic repeat regions in cancer cells. We find that KDM5 inhibition in luminal breast cancer cells results in R-loop-mediated DNA damage, reduced cell fitness and an increase in ISG and AP signatures as well as cell surface Major Histocompatibility Complex (MHC) class I, mediated by RNA:DNA hybrid activation of the CGAS/STING pathway. KDM5 inhibition does not result in DNA damage or activation of the CGAS/STING pathway in normal breast epithelial cells, suggesting that KDM5 inhibitors may enable a wide therapeutic window in this setting, as compared to STING agonists or Type I Interferons. These findings provide new insights into the interplay between epigenetic regulation of genomic repeats, R-loop formation, innate immunity, and cell fitness in the context of cancer evolution and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Human gene expression is controlled from distance via enhancers, which can form longer ‘super-enhancer’-regions of intense regulatory activity. Whether super-enhancers constitute a separate regulatory paradigm remains unclear, largely due to the difficulty of dissecting the contributions and interactions of individual elements within their natural chromosomal context. To address this challenge, we developed enhancer scrambling, a high-throughput strategy to generate stochastic inversions and deletions of targeted enhancer regions by combining CRISPR prime editing insertion of symmetrical loxP sites with Cre recombinase-induced rearrangements. We applied our approach to dissect a distal super-enhancer of the OTX2 gene, generating up to 134 alternative regulatory configurations in a single experiment, and establishing how they drive gene expression and chromatin accessibility, as well as the individual contributions of its elements to this activity. Surprisingly, the presence of the sequence containing a single DNase I hypersensitive site predominantly controls OTX2 expression. Our findings highlight that enhancer-driven regulation of some highly expressed, cell-type-specific genes can rely on an individual element within a cluster of non-interacting, dispensable components, and suggest a simple functional core to a subset of super-enhancers. The targeted randomisation method to scramble enhancers can scale to resolve many super-enhancers and human gene regulatory landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cecb86ee79d0fa4af20cbd670e19b80d640e006e" target='_blank'>
              Resolution of a human super-enhancer by targeted genome randomisation
              </a>
            </td>
          <td>
            Jonas Koeppel, Pierre Murat, Gareth Girling, Elin Madli Peets, Mélanie Gouley, Valentin Rebernig, Anant Maheshwari, Jacob Hepkema, Juliane Weller, Jenie Hannah Johnkingsly Jebaraj, Ronnie Crawford, Fabio Giuseppe Liberante, Leopold Parts
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Monitoring disease progression in patients with high-grade gliomas (HGGs) is challenging due to treatment-related changes on imaging and the requirement for neurosurgical intervention to obtain diagnostic tissue. DNA junctions in HGGs often amplify oncogenes, making these DNA fragments potentially more abundant in blood than monoallelic mutations. Herein, we piloted a cell-free DNA approach for disease detection in plasma of patients with HGGs by leveraging patient-specific DNA junctions associated with oncogene amplifications.


EXPERIMENTAL DESIGN
Whole genome sequencing of grade 3 or 4 IDH-mutant or wild-type astrocytomas was utilized to identify amplified junctions. Individualized qPCR assays were developed using patient-specific primers designed for the amplified junction. ctDNA levels containing these junctions were measured in patient plasma samples.


RESULTS
Unique amplified junctions were evaluated by individualized semi-quantitative PCR assays in presurgical plasma of 18 patients, 15 with tumor-associated focal amplifications and three without. High-copy number junctions were robustly detected in plasma of 14/15 patients (93.3%) with amplified junctions and none of the controls. Changes in junction abundance correlated with disease trajectory in serial plasma samples from five patients, including increased abundance of amplified junctions preceding radiographic disease progression.


CONCLUSION
In patients with grade 3 or 4 astrocytomas who had tumor-associated amplifications, patient-specific amplified junctions were successfully detected in assayed plasma from most patients. Longitudinal analysis of plasma samples correlated with disease trajectory, including cytoreduction and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6be79817ee79c2803379821d2392c413312af98" target='_blank'>
              Personalized tumor-specific amplified DNA junctions in peripheral blood of patients with glioblastoma.
              </a>
            </td>
          <td>
            Mohamed F Ali, C. Riviere-Cazaux, Sarah H Johnson, Rebecca Salvatori, Alan R Penheiter, James B Smadbeck, Stephen J. Murphy, F. Harris, Lex F McCune, Lucas P Carlstrom, Michael T Barrett, F. Kosari, Leila A Jones, Cristiane M Ida, M. Borad, B. Bendok, Alfredo Quiñones-Hinojosa, Alyx B Porter, Maciej M Mrugala, K. Jaeckle, P. Anastasiadis, S. Kizilbash, John C Cheville, David M Routman, Terry C Burns, G. Vasmatzis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Kaposi sarcoma-associated herpesvirus (KSHV) is the causative agent of a number of human neoplasms. KSHV causes lifelong chronic infections by establishing persistent reservoirs of latently infected cells in which the viral double-stranded DNA genome is present in the form of an extrachromosomal episome. Polycomb group protein (PcG) mediated gene repression is detected early after entry of the CpG-rich and epigenetically naïve viral DNA into the host nucleus. Here we show that the establishment and maintenance of KSHV latency is strongly influenced by Polycomb Repressive Complexes 1 and 2 (PRC1, and PRC2, respectively). Using a series of genetic deletion models, we dissect the functional interdependence of PRC1 and PRC2 on KSHV and identify KSHV episomes as a prime target for independent as well as collaborative de novo PRC recruitment. Loss of PRC function leads to a severely altered chromatin state on KSHV and we show that PRC mediated gene repression is not only preventing transcription of viral genomes but also protects KSHV episomes from detection by the human silencing hub (HUSH). Furthermore, we demonstrate that PRC complexes play an important role in ensuring episomal maintenance in dividing cells. Taken together, our results show that absence of PRC complexes creates a metastable state of KSHV latency and suggest early chromatinization as an integral step for KSHV latency establishment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47bd8518f8db94e121fb18dd96d7456a23942c6d" target='_blank'>
              Polycomb group proteins protect latent Kaposi sarcoma-associated herpesvirus from episome clearance and HUSH-dependent chromatin silencing
              </a>
            </td>
          <td>
            Simon Weissmann, Alexis Robitaille, Thomas Günther, Marion Ziegler, A. Grundhoff
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract Alternative end-joining (alt-EJ) is an error-prone DNA repair pathway that cancer cells deficient in homologous recombination rely on, making them vulnerable to synthetic lethality via inhibition of poly(ADP-ribose) polymerase (PARP). Targeting alt-EJ effector DNA polymerase theta (POLθ), which synergizes with PARP inhibitors and can overcome resistance, is of significant preclinical and clinical interest. However, the transcriptional regulation of alt-EJ and its interactions with processes driving cancer progression remain poorly understood. Here, we show that alt-EJ is suppressed by hypoxia while positively associated with MYC (myelocytomatosis oncogene) transcriptional activity. Hypoxia reduces PARP1 and POLQ expression, decreases MYC binding at their promoters, and lowers PARylation and alt-EJ-mediated DNA repair in cancer cells. Tumors with HIF1A mutations overexpress the alt-EJ gene signature. Inhibition of hypoxia-inducible factor 1α or HIF1A expression depletion, combined with PARP or POLθ inhibition, synergistically reduces the colony-forming capacity of cancer cells. Deep learning reveals the anticorrelation between alt-EJ and hypoxia across regions in tumor images, and the predictions for these and MYC activity achieve area under the curve values between 0.70 and 0.86. These findings further highlight the critical role of hypoxia in modulating DNA repair and present a strategy for predicting and improving outcomes centered on targeting alt-EJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2d05c118cddd3cc2623e904498c569be8cdf72" target='_blank'>
              Harnessing transcriptional regulation of alternative end-joining to predict cancer treatment
              </a>
            </td>
          <td>
            Roderic Espín, Ferran Medina-Jover, Javier Sigüenza-Andrade, Sònia Farran-Matas, Francesca Mateo, Agnès Figueras, Rosario T Sanz, Guillermo P Vicent, Arzoo Shabbir, Lara Ruiz-Auladell, Emilio Racionero-Andrés, Irene García, A. Baiges, Lídia Franco-Luzón, Adrián Martínez-Tébar, Miguel Angel Pardo-Cea, M. Martínez-Iniesta, Xieng C. Wang, E. Cuyás, Javier A. Menendez, Marta Lopez-Cerda, Purificacion Muñoz, Ivonne Richaud, Angel Raya, Isabel Fabregat, Alberto Villanueva, Xènia Serrat, Julián Cerón, M. Alemany, Inés Guix, A. Herencia-Ropero, V. Serra, Rehna Krishnan, Karim Mekhail, Razqallah Hakem, Jordi Bruna, M. Barcellos-Hoff, Francesc Viñals, Á. Aytés, M. Pujana
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab99623f5d226080b2ad6d750b3bbe647805883" target='_blank'>
              The landscape of Helicobacter pylori-mediated DNA breaks links bacterial genotoxicity to its oncogenic potential
              </a>
            </td>
          <td>
            Hadas Sibony-Benyamini, Rose Jbara, Tania Shubash Napso, Layan Abu-Rahmoun, Daniel Vizenblit, Michal Easton-Mor, Shira Perez, Alexander Brandis, Tamar Leshem, Avi Peretz, Yaakov Maman
          </td>
          <td>2025-02-24</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Activation of synthetic centromeres on chromosome 4 in maize leads to its breakage and formation of trisomic fragments called neochromosomes. A limitation of neochromosomes is their low and unpredictable transmission rates due to trisomy. Here we report that selecting for dicentric recombinants through male crosses uncovers stabilized chromosome 4 fission events which split it into 4a-4b complementary chromosome pairs, where 4a carries a native centromere and 4b a synthetic one. The cells rapidly stabilized chromosome ends by de novo telomere formation and the new centromeres spread among genes without altering their expression. When both 4a and 4b chromosomes were present in a homozygous state, they segregated through meiosis indistinguishably from wild-type, and gave rise to healthy plants with normal seed set. This work leverages synthetic centromeres to engineer chromosome fission, effectively raising the diploid chromosome number of maize from 20 to 22. TEASER Using synthetic centromeres, we divide one maize chromosome into two chromosomes, and show that both chromosomes are fully functional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In this study, we show that comprehensive DNA profiling using whole genome sequencing (WGS) can offer novel insights into ovarian carcinoma (OC). First, using WGS of 185 OC with HRDetect profiling, a mutational signature-based algorithm for homologous recombination repair deficiency (HRD), we found that patients with HRDetect-high OC had significantly better overall survival, with a median of 6.2 years compared to 4.1 years for those with lower score OC. Second, the rest of the OCs are not simply non-HRD - some carry distinguishing rearrangement signatures that are strongly prognostic. Third, in an independent cohort of 77 patients, we demonstrate predictive value; HRDetect-high patients exhibited better responses to PARP inhibitors (PARPi) (median progression-free survival of 11.1 months versus 7.1 months; overall response rate of 54% versus 22.5%). Critical next steps include fine-tuning therapeutic vulnerabilities associated with the diverse rearrangement signatures, and harnessing the full potential of WGS for OC clinical trial stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc713b72314da7faa35c83a1d8de687d85095e55" target='_blank'>
              HRDetect in Ovarian Carcinoma: Stratification and Therapeutic Implications.
              </a>
            </td>
          <td>
            K. Banda, H. R. Davies, Y. Kumar, Y. Memari, A. Degasperi, M. Radke, I. Rodriguez, T. A. Gooley, F. Menghi, T. Harding, K. Lin, E. T. Liu, S. Nik-Zainal, E. M. Swisher
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The human genome contains regulatory DNA elements, enhancers, that can activate gene transcription over long chromosomal distances. Here, we show that enhancer distance can be critical for gene silencing. We demonstrate that linear recruitment of the normally distal HBB super-enhancer to developmentally silenced HBG promoters, through deletion or inversion of intervening DNA sequences, results in strongly reactivated HBG expression in adult erythroid cells and ex vivo differentiated hematopoietic stem and progenitor cells. A similar observation is made in the HBA locus, where deletion-to-recruit of the distal enhancer strongly reactivates embryonic HBZ expression. Overall, our work assigns function to seemingly non-regulatory genomic segments: by providing linear separation they may support genes to autonomously control their transcriptional response to distal enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a34162ce128443bfc4bf95c06967af0f813ffc9" target='_blank'>
              Reactivation of developmentally silenced globin genes through genomic deletions reveals that enhancer distance matters
              </a>
            </td>
          <td>
            Anna-Karina Felder, S. J. Tjalsma, H. Verhagen, Rezin Majied, M. Verstegen, Thijs C. J. Verheul, Rebecca Mohnani, Richard Gremmen, Peter H. L. Krijger, S. Philipsen, Emile van den Akker, W. de Laat
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Purpose
The epidermal growth factor receptor (EGFR) is a therapeutic target with confirmed clinical efficacy for several cancer types. We aimed to identify EGFR aberrations and their associations with other genomic alterations in patients with metastatic diseases of various cancers.


Materials and Methods
We used real-world data from the next-generation sequencing (NGS) of 3,286 patients with metastatic cancer at the Samsung Medical Center. We analyzed the distribution of EGFR amplification, mutation, and fusion, as well as their correlations with microsatellite instability (MSI), tumor mutation burden (TMB), and other gene aberrations.


Results
A total of 3,286 patients were tested using NGS of a panel covering 523 cancer-related genes (TSO500, Illumina) as part of clinical practice between October 2019 and October 2022. Patients with lung cancer and gliomas were not included in the analysis. Of the 3,286 patients, 175 (5.3%) had EGFR amplification, 38 (1.2%) had EGFR mutations, and 8 (0.2%) had EGFR fusion. All 175 patients with EGFR amplifications had microsatellite-stable (MSS) tumors, but 102 had co-amplifications in other cancer-related genes, and 78 had mutations with clinical significance (tier I/II). Among the 38 patients with EGFR mutations, three (8%) showed MSI-high status, and eleven (29%) demonstrated high TMB (≥ 10 mutations/mb). Among eight patients with EGFR fusion, three exhibited possible functionalities of the EGFR gene.


Conclusion
EGFR aberrations, mainly amplification, followed by mutation and fusion, were present in 6.4% of patients with metastatic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae0df005b2d8a0f47444591e98d3dc4bb550830" target='_blank'>
              Epidermal Growth Factor Receptor Aberrations Identified by Next-generation Sequencing in Patients with Metastatic Cancers.
              </a>
            </td>
          <td>
            Minkyue Shin, Dae-Ho Choi, Jaeyun Jung, Deokgeun Kim, Sung Hee Lim, Seung-Tae Kim, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Kyoung-Mee Kim, Jeeyun Lee
          </td>
          <td>2025-02-21</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 DNA methylation alterations are observed across multiple cancers, occurring early during tumor initiation. However, evolution and drivers of these changes remain poorly understood and difficult to assess at the clonal level. Consequently, we developed a method for profiling genome-wide DNA methylation from low DNA amounts and applied it to the organoid model of BRAFV600E oncogene-induced colon cancer (CC) tumorigenesis. Single organoid clones, derived from single stem cells, were profiled every month for 5 months to map nucleotide-level dynamic heterogeneity in methylation changes. Uninduced control organoids exhibit a pattern of epigenetic drift leading towards clonal convergence not until 5 months. Whereas BRAFV600E-induced organoids exhibit a step-wise divergence throughout the different time points, likely explained by progressive clonal takeovers. The methylation changes in BRAFV600E induced organoids are explained by the age-in-culture associated methylation changes; especially BRAFV600E did not induce epigenetic changes, as previously thought. The CpG sites affected in the control and BRAFV600E comparison were not shared but mapped to the same genomic elements, indicating common drivers of methylation changes regardless of oncogene. Investigation of promoter CpG-island methylation gains revealed that they are decoupled from replication. External validation in other pre-cancerous and CC datasets, and experimental cell line data sets, show that methylation gains at regulatory elements are a function of “biological aging” and independent of replication or cell division cycles. Our studies provide new insights into the early epigenetic changes involved in BRAFV600E driven carcinogenesis rooted in the age-dependent drivers of epigenetic alterations.
 Citation Format: Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, Thomas R. Pisanic II, Stephen B Baylin, Hariharan Easwaran. Promoter methylation gains in aging and cancer are independent of replication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="The DUX4 transcription factor is briefly expressed in the early embryo and is epigenetically repressed in somatic tissues. Loss of epigenetic repression can result in the aberrant expression of DUX4 in skeletal muscle and can cause facioscapulohumeral dystrophy (FSHD). Multiple factors have been identified as necessary to maintain epigenetic silencing of DUX4 in skeletal muscle, but whether specific sequences at the DUX4 locus are sufficient for epigenetic silencing has been unknown. We cloned fragments of the D4Z4 macrosatellite repeat, the DNA region that encompasses the DUX4 retrogene, adjacent to a reporter driven by a constitutive promoter and identified a single fragment sufficient to epigenetically repress reporter gene expression. Previously identified suppressors of DUX4 expression—SETDB1, ATF7IP, SIN3A/B, and LRIF1—were necessary for silencing activity and p38 inhibitors enhanced suppression. These findings identify a key regulatory sequence for D4Z4 epigenetic repression and establish a model system for mechanistic and discovery studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c637d49373bdbc8ffeb399dfd912513e88f9743f" target='_blank'>
              A discrete region of the D4Z4 is sufficient to initiate epigenetic silencing
              </a>
            </td>
          <td>
            Ellen M. Paatela, Faith G. St. Amant, Danielle C Hamm, Sean R. Bennett, Taranjit S. Gujral, Silvère M. van der Maarel, Stephen J. Tapscott
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are widely considered the most cytotoxic DNA lesions occurring in cells because they physically disrupt the connectivity of the DNA double helix. Homologous recombination (HR) is a high-fidelity DSB repair pathway that copies the sequence spanning the DNA break from a homologous template, most commonly the sister chromatid. How both DNA ends, and the sister chromatid are held in close proximity during HR is unknown. Here we demonstrate that the PST repeat region of MDC1 is a mutlivalent nucleosome binding domain, sufficient to tether chromatin in multiple contexts. In mitotic cells the affinity of the PST repeats for chromatin is downregulated by phosphorylation to prevent chromosome missegregation, while still contributing to DNA break tethering by the MDC1-TOPBP1-CIP2A complex. In interphase, the PST repeat region is critical for RAD51 focus formation but not the recruitment of 53BP1 to DNA breaks, consistent with a chromatin tethering function. In total, this work demonstrates that the PST repeat region of MDC1 is a multivalent chromatin binding domain with tunable affinity that contributes to DNA break tethering during HR and in mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dbf6e7c3f0e969876e0503353c0c93165dcec37" target='_blank'>
              The PST repeat region of MDC1 is a tunable multivalent chromatin tethering domain
              </a>
            </td>
          <td>
            Joshua R. Heyza, Maria Mikhova, Gloria I Perez, David G. Broadbent, Jens C. Schmidt
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="
 Embryonal Tumors with Multilayered Rosettes (ETMRs) are relatively rare but aggressive embryonal brain tumors that mainly affect children under the age of 4 and which have a poor outcome. Around 90% of ETMRs harbor amplification of the C19MC miRNA cluster, in most cases fused to the TTHY1 gene. The second most recurrent aberration, present in about half of the C19MC negative cases, are bi-allelic mutations in DICER1. ETMRs are also characterized by overall lower DNA methylation levels and high LIN28A gene expression as compared with other embryonal brain tumors like medulloblastomas (MB) and atypical teratoid rhabdoid tumors (ATRT). However, a more detailed description of the epigenetic landscape marked by DNA methylation and histone modifications, and how this affects the ETMR transcriptome is still lacking. To decode the epi-genome, we have collected whole genome bisulfite sequencing (WGBS) data, ChIP sequencing (ChIP-seq) data of various (active and repressive) histone marks and epigenetic modifiers (EZH2, BRD4, CTCF), and RNA sequencing (RNA-seq) data from a series of ETMR patient tumors, and similar data from a series of MBs and ATRTs covering all molecular subtypes for comparison reasons. Using all these data and performing bioinformatic analyses, we have depicted chromatin states and DNA-methylation-defined genomic segments of ETMR for the first time. ETMR genomes are generally hypomethylated, with wider distributions of un-methylated regions and DNA methylation valleys (DMVs) on its genome compared with MB and ATRT. By integrating these data with all other omics data, we found that BRD4-regulated overexpression of TET demethylases may cause global hypomethylation in ETMRs. Polycomb and TETs are recruited at DMVs to ensure DMVs are longer in ETMRs than MBs and ATRTs. Furthermore, genes involved in embryo development and cell proliferation are enriched in entity-shared DMVs and are highly expressed in ETMRs and other embryonal brain tumors. We also depicted that TET family members, BRD4, and Polycomb complex (EZH2) are potential therapeutic targets for ETMR patients. In conclusion, we have partially described the epi-genome and epigenetic regulations of ETMR, and are currently working on more detailed regulation axes and performing drug test experiments for these potential therapeutic targets to understand better the underlying biology of these tumors and how this may lead to alternative and more effective therapies.
 Citation Format: Shanzheng Wang, Sander Lambo, Monika Mauermann, Robert J Autry, Natalie Jäger, Stefan M Pfister, Marcel Kool. Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR) [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0e28f4d695edfbb9dadc1b324a1b929e51f3786" target='_blank'>
              Abstract B028: Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR)
              </a>
            </td>
          <td>
            Shanzheng Wang, S. Lambo, Monika Mauermann, Robert J Autry, N. Jäger, Stefan M. Pfister, M. Kool
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers. Methods We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line. Results CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells. Conclusion Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d43986e3ec1bf10fa7949b61cdcc071a4afe56" target='_blank'>
              Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, Masashi Kariya, Ayami Osaka, S. Ishino, Masataka Yamauchi, Tomonori Tamura, I. Hamachi, Shoji Takada, Y. Ishino, H. Atomi
          </td>
          <td>2025-02-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861d404b3fd5272bc30487378b974c5ae16b843d" target='_blank'>
              CTCF is selectively required for maintaining chromatin accessibility and gene expression in human erythropoiesis
              </a>
            </td>
          <td>
            Xue Yang, Li Cheng, Ye Xin, Jianxiang Zhang, Xinfeng Chen, Jinchao Xu, Mengli Zhang, Ruopeng Feng, Judith Hyle, Wenjie Qi, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li, Peng Xu
          </td>
          <td>2025-02-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Jr Isaac E. Kim, A. Fola, Enrique Puig, Titus K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Because transposable elements (TEs) can cause heritable genetic changes, past work on TE mobility in Arabidopsis thaliana has mostly focused on new TE insertions in the germline of hypomethylated plants. It is, however, well-known that TEs can also be active in the soma, although the high-confidence detection of somatic events has been challenging. Here, we leveraged the high accuracy of PacBio HiFi long reads to evaluate the somatic mobility of TEs in individuals of an A. thaliana non-reference strain lacking activity of METHYLTRANSFERASE1 (MET1), a major component of the DNA methylation maintenance machinery. Most somatically mobile families coincided with those found in germline studies of hypomethylated genotypes, although the exact TE copies differed. We also discovered mobile elements that had been missed by standard TE annotation methods. Somatic TE activity was variable among individual plants, but also within TE families. Finally, our approach pointed to the possible involvement of alternative transposition as a cause for somatic hypermutability in a region that contains two closely spaced VANDAL21 elements. We conclude that long-read sequencing can reveal widespread TE transposition in the soma of A. thaliana hypomethylated mutants. Assessing somatic instead of germline mobilization is a fast and reliable method to investigate different aspects of TE mobility at the single plant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62607e49f83191e3ad9371253c4b7877fe10686a" target='_blank'>
              Long-read detection of transposable element mobilization in the soma of hypomethylated Arabidopsis thaliana individuals
              </a>
            </td>
          <td>
            Andrea Movilli, Svitlana Sushko, Fernando A. Rabanal, Detlef Weigel
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="DNA methylation at gene promoters is generally considered to be associated with transcriptional repression. However, lack of a clear picture of where promoter methylation is most important for transcriptional regulation has hindered our understanding of this relationship and resulted in the use of a wide variety of arbitrary promoter definitions. We demonstrate here that the use of different promoter definitions can lead to contradictory results between studies of promoter methylation. In response, we have developed Methodical, a computational method that combines RNA-seq and whole genome bisulfite sequencing (WGBS) data to identify genomic regions where DNA methylation is highly correlated with transcriptional activity. We refer to these regions as transcript-proximal methylation-associated regulatory sites (TMRs). We applied Methodical to one normal prostate tissue data set, one prostate tumour dataset and one prostate metastases dataset and characterized the identified TMRs. We show that the region just downstream of the TSS, particularly within the first exon and intron, is the most common location for TMRs and that TMRs are enriched for particular genomic features, chromatin states and transcription factor binding sites. Finally, we demonstrate that the methylation of TMRs is generally strongly correlated with transcription in diverse cancer types and that TMRs are highly subject to altered DNA methylation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229e0f93b401077a9ccbe06fb3844f9ba3a44d0d" target='_blank'>
              Systematic identification of regions where DNA methylation is correlated with transcription refines regulatory logic in normal and tumour tissues
              </a>
            </td>
          <td>
            Richard Heery, Martin H Schaefer
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression from adenoma to adenocarcinoma is associated with global reduction in 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC). DNA hypomethylation continues upon liver metastasis. Here we examine 5hmC changes upon progression to liver metastasis. Results 5hmC is increased in metastatic liver tissue relative to the primary colon tumour and expression of TET2 and TET3 is negatively correlated with risk for metastasis in patients with CRC. Genes associated with increased 5-hydroxymethylcytosine show KEGG enrichment for adherens junctions, cytoskeleton and cell migration around a core cadherin (CDH2) network. Overall, the 5-hydroxymethylcyosine profile in the liver metastasis is similar to normal colon appearing to recover at many loci where it was originally present in normal colon and then spreading to adjacent sites. The underlying sequences at the recover and spread regions are enriched for SALL4, ZNF770, ZNF121 and PAX5 transcription factor binding sites. Finally, we show in a zebrafish migration assay using SW480 CRISPR-engineered TET knockout and rescue cells that reduced TET expression leads to a reduced migration frequency. Conclusion Together these results suggest a biphasic trajectory for 5-hydroxymethyation dynamics that has bearing on potential therapeutic interventions aimed at manipulating 5-hydroxymethylcytosine levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3087cc8f484e07e82cdb51614296a13d7efc4fb4" target='_blank'>
              Colorectal cancer progression to metastasis is associated with dynamic genome-wide biphasic 5-hydroxymethylcytosine accumulation
              </a>
            </td>
          <td>
            Ben Murcott, Floris Honig, Dominic Oliver Halliwell, Yuan Tian, J. L. Robson, Piotr J. Manasterski, Jennifer Pinnell, Thérèse Dix-Peek, Santiago Uribe-Lewis, Ashraf EK Ibrahim, Julia Sero, David Gurovich, Nikolas Nikolai, Adele Murrell
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="During the development of cancer, many changes leading to a series of genetic mutations that cannot control tumor growth. Therefore, understanding the complex interactions between the intrinsic, extrinsic, and systemic mediators of tumor cells and disease progression is crucial for the rational development of effective anti-cancer treatments. In this study, we conducted a multi-omics pan-cancer exploration of the function of JMJD4 using various comprehensive analysis tools and samples collected from relevant cancer databases, to understand how JMJD4 affects pan-cancer development by identifying its clinical features and multi-omics heterogeneity. We discovered that JMJD4 was highly expressed and associated with cancer stemness, epigenetic modulations and DNA repair in pan-cancer, especially in BLCA, BRCA, CHOL, COAD, ESCA. JMJD4 was also related to most immune-infiltrating cells, which indicated that JMJD4 may serve as a diagnostic and immunological biomarker in pan-cancer. Therefore, JMJD4 can be used as an indicator and a new therapeutic target for early tumor diagnosis and a prognostic marker for many types of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f74f88772da2d32d539504f99797675a6a54aee" target='_blank'>
              JMJD4 is a potential diagnostic and immunological biomarker in pan-cancer
              </a>
            </td>
          <td>
            Qinyi Li
          </td>
          <td>2025-03-03</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89eb232b40b0237c425e76f52fd0c4fca33e33a1" target='_blank'>
              Pan‑cancer analysis of the oncogenic role of telomeric repeat binding factor 2 (TERF2) in human tumors and in vitro validation in gastric cancer by TERF2 knockdown
              </a>
            </td>
          <td>
            Qiang Ma, Wen Xu, Xiaolan Guo
          </td>
          <td>2025-02-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            M. M. Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, S. Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Both DNA replication and RNA transcription utilize genomic DNA as their template, necessitating spatial and temporal separation of these processes. Conflicts between the replication and transcription machinery, termed transcription-replication conflicts (TRCs), pose a considerable risk to genome stability, a critical factor in cancer development. While several factors regulating these collisions have been identified, pinpointing primary causes remains difficult due to limited tools for direct visualization and clear interpretation. In this study, we directly visualize TRCs using a proximity ligation assay (PLA), leveraging antibodies specific to PCNA and phosphorylated CTD of RNA polymerase II. This approach allows precise measurement of TRCs between replication and transcription processes mediated by RNA polymerase II. The method is further enhanced through DNA primers conjugated covalently to these antibodies, coupled with PCR amplification using fluorescent probes, providing a highly sensitive and specific means of detecting endogenous TRCs. Fluorescence microscopy enables the visualization of these conflicts, offering a powerful tool to study genome instability mechanisms associated with cancer. This technique addresses the gap in direct TRC visualization, allowing for a more comprehensive analysis and understanding of the underlying processes driving genome instability in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f574fa952d860bc38fdda116785a24f5d5ebcd3f" target='_blank'>
              Proximity Ligation Assay to Study Oncogene-Derived Transcription-Replication Conflicts.
              </a>
            </td>
          <td>
            Linling Ke, Qian Xie, Xiaoman Wang, Yuanhang Gong, Min Li
          </td>
          <td>2025-01-10</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Osteosarcoma (OS) is the most common primary malignant bone tumor in both humans and dogs. Canine and human OS share significant similarities in clinical presentation, histopathologic features, and molecular aberrations which have pioneered the dog as a translationally relevant model to accelerate treatment discovery in human OS. Despite aggressive treatment, up to 40% of children and >90% of dogs with OS die due to multi-drug-resistant metastatic disease, underscoring an urgent need to identify biomarkers which predict metastasis and/or response to therapy. Copy number gains in the cMYC oncogene have recently emerged as a potential biomarker of aggressive disease in human OS, and its genomic amplification has been recently described in primary canine OS tumors and cell lines. MYC is a nuclear transcription factor that drives a variety of tumor intrinsic hallmarks, but more recently has also been linked to regulation of the tumor immune microenvironment (TIME) in rodent models. We therefore hypothesize MYC overexpression in metastatic OS contributes to an immunosuppressive environment by driving expression of chemokines that promote tumor associated macrophage (TAM) enrichment and inhibit T cell recruitment. To characterize the role of oncogenic MYC signaling in the metastatic OS TIME, 42 archived formalin-fixed paraffin-embedded lung metastatic canine OS samples were evaluated for MYC copy number variation (CNV), mRNA, and protein expression via ddPCR, Nanostring analysis, and immunohistochemistry (IHC). 7 samples also underwent GeoMX spatial profiling to more specifically evaluate T cell and macrophage transcriptional profiles based on MYC status. To determine the role of MYC target modulation as a potential therapeutic option, canine and human OS cell lines were treated with a novel MYC inhibitor (MYCi975) and assessed for effects on survival, proliferation, and cytokine profiles. We demonstrate that CNV gains are not a key driver of MYC hyperactivity in canine metastatic OS. However, stratification based on MYC protein expression demonstrates that ‘MYC-high’ tumors are associated with downregulation of cytotoxic effector T-cell associated transcripts and upregulation of TAM and extracellular matrix remodeling transcripts. We also report that MYCi975 treatment of canine and human OS cell lines results in significant inhibition of OS cell survival and proliferation at concentrations which are pharmacologically achievable in mice. Furthermore, we demonstrate MYC inhibition by MYCi975 is associated with reduced pro-metastatic cytokine secretion in OS cell culture models. While MYC overactivity in metastatic canine OS may not be genomically driven, other mechanisms that lead to increased MYC protein expression are associated with transcriptomic profiles supportive of local immunosuppression. Furthermore, pharmacologic targeting of MYC may serve as a strategy to bolster immunotherapeutic options in canine and human metastatic OS.
 Citation Format: Rebecca Makii, Téa Ned, Eric Palmer, Kathryn Cronise, Mary Jane Underdown, Breelyn Wilky, Daniel Regan. The role of oncogenic MYC signaling in shaping the canine metastatic osteosarcoma tumor immune microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2811582e135c424fa6e17581d7991144b4560600" target='_blank'>
              Abstract B062: The role of oncogenic MYC signaling in shaping the canine metastatic osteosarcoma tumor immune microenvironment
              </a>
            </td>
          <td>
            Rebecca Makii, Téa Ned, Eric Palmer, Kathryn E. Cronise, M. J. Underdown, B. Wilky, Daniel Regan
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea01c369ba68e6382727474294c60f827396f68" target='_blank'>
              Super-enhancers in tumors: unraveling recent advances in their role in Oncogenesis and the emergence of targeted therapies
              </a>
            </td>
          <td>
            Yumeng Ji, Baixue Li, Rongjin Lin, Jing Yuan, Yang Han, Yuping Du, Yang Zhao
          </td>
          <td>2025-01-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer, yet the specific genes driving recurrent chromosome-arm losses remain largely unknown. Here, we present a systematic framework integrating gene expression, mutation, and copy number data to identify candidate driver genes of cancer type-specific recurrent chromosome-arm losses across 20 cancer types, using ∼7,500 tumors from The Cancer Genome Atlas. By analyzing focal deletions and point mutations that co-occur with, or are mutually exclusive with, chromosome-arm losses, we pinpoint 311 candidate drivers associated with 160 cancer type-specific recurrent events. Our approach identifies known aneuploidy drivers such as TP53 and PTEN, while revealing multiple novel candidates, including established tumor suppressor genes not previously linked to aneuploidy. Furthermore, we leverage gene expression changes associated with these chromosome-arm losses to propose pathway-level alterations that may drive cancer progression. Integrating these findings highlights key candidate drivers underlying the observed gene expression alterations, thereby reinforcing their biological relevance. This work provides the first comprehensive catalogue of candidate driver genes for recurrently lost chromosome-arms in human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, M. Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) causes most Merkel cell carcinomas (MCC). The virus is one of the few known human tumor viruses, and due to its direct role in this skin cancer development, it is a bona fide model for viral tumorigenesis and tumorigenesis in general. Chronic viruses in humans such as MCPyV are highly adapted to their host and current models to study infection, persistence and pathogenesis are highly limited. We here use an induced pluripotent stem cell (iPSC)-derived hair-bearing skin organoid (SkO) system to demonstrate efficient viral infection, progression and spread of MCPyV. Using bulk-, single cell - and spatial-transcriptomics, combined with immunostaining and nucleic acid hybridization technologies, we show that MCPyV ensures persistence due to a quasi-latency state within the majority of dermal fibroblasts carry the viral genome. Further, we identify the cell type of productive infection with papillary fibroblasts and dermal sheath fibroblasts supporting viral replication and progeny production. Our high-resolution methods demonstrate that the virus in these cells evades the innate immune response, as evidenced by the efficacy of interferon-beta treatment or ruxolitinib, a JAK/STAT inhibitor, in suppressing or stimulating viral replication. We show that iPSC-derived SkOs are able to support infection and long-term persistence of the virus under conditions very similar to those found in humans. Thus, this infection model provides a robust platform for understanding and characterizing the interaction of this virus with the immune system in infection, testing treatment strategies to control reactivation and map processes involved in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The melanocyte lineage determining Microphthalmia-associated transcription factor (MITF) drives proliferation and survival of melanocytic melanoma cells through regulation of both coding genes and long non-coding RNAs (LncRNAs). Here we characterize LINC00520 (hereafter called LncRNA ENhancer of Translation, LENT) regulated by MITF and strongly expressed in melanocytic melanoma cells. LENT is essential for proliferation and survival of cultured melanocytic melanoma cells and xenograft tumours. LENT interacts with the G4 quadruplex resolvase DHX36 and both associate with the ribosome in the 80S and light polysome fractions. LENT modulates DHX36 association with a collection of mRNAs regulating their engagement with polysomes and fine-tuning their subsequent translation. These mRNAs encode proteins involved in endoplasmic reticulum (ER) and mitochondrial homeostasis as well as autophagy. Consequently, LENT silencing leads to extensive autophagy and mitophagy, compromised oxidative metabolic capacity accompanied by an accumulation and mis-localization of mitochondrial proteins leading to proteotoxic stress and apoptosis. The LENT-DHX36 axis therefore fine-tunes translation of proteins involved in ER and mitochondrial homeostasis suppressing autophagy and promoting survival and proliferation of melanoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b920b4f4116633072032263045e09ed81f394c06" target='_blank'>
              Interaction of lncRNA LENT with DHX36 regulates translation and suppresses autophagy in melanoma
              </a>
            </td>
          <td>
            Alexandre Haller, Giovanni Gambi, Mattia D’Agostino, G. Davidson, A. Lallement, G. Mengus, Chadia Nahy, Nadia Messaddeq, Guillaume Bec, Angelita Simonetti, Eric Ennifar, Irwin Davidson
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b62083d3dd11f079792a9e41da3af07b399c20f" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Olivia Dove Estrella, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Epigenetic mechanisms are crucial for developmental programming and can be disrupted by environmental stressors, increasing susceptibility to disease. This has sparked interest in therapies for restoring epigenetic balance, but it remains uncertain whether disordered epigenetic mechanisms can be fully corrected. Disruption of DNA methyltransferase 1 (DNMT1), responsible for DNA methylation maintenance, has particularly devastating biological consequences. Therefore, here we explored if rescuing DNMT1 activity is sufficient to reverse the effects of its loss utilizing mouse embryonic stem cells. However, only partial reversal could be achieved. Extensive changes in DNA methylation, histone modifications, and gene expression were detected, along with transposable element derepression and genomic instability. Reduction of cellular size, complexity, and proliferation rate were observed, as well as lasting effects in germ layer lineages and embryoid bodies. Interestingly, by analyzing the impact on imprinted regions, we uncovered 20 regions exhibiting imprinted-like signatures. Notably, while many permanent effects persisted throughout Dnmt1 inactivation and rescue, others arose from the rescue intervention. Lastly, rescuing DNMT1 after differentiation initiation worsened outcomes, reinforcing the need for early intervention. Our findings highlight the far-reaching functions of DNMT1 and provide valuable perspectives on the repercussions of epigenetic perturbations during early development and the challenges of rescue interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c888744288cbb5c0e8c3ec88a53008ee7b90e2a" target='_blank'>
              Rescuing DNMT1 fails to fully reverse the molecular and functional repercussions of its loss in mouse embryonic stem cells
              </a>
            </td>
          <td>
            Elizabeth Elder, Anthony Lemieux, Lisa-Marie Legault, Maxime Caron, Virginie Bertrand-Lehouillier, Thomas Dupas, Noël J-M Raynal, Guillaume Bourque, Daniel Sinnett, Nicolas Gévry, Serge McGraw
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836896d1f99b83d8b9c8d7d766bdd19d95c66f23" target='_blank'>
              NONO regulates m5C modification and alternative splicing of PTEN mRNAs to drive gastric cancer progression
              </a>
            </td>
          <td>
            Gaichao Zhao, Ruochen Liu, Lingjun Ge, Dan Qi, Qishu Wu, Zini Lin, Houji Song, Liping Zhong, Hongjuan Cui
          </td>
          <td>2025-03-04</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A significant portion of human cancers utilize a recombination-based pathway, Alternative Lengthening of Telomeres (ALT), to maintain telomere length. Targeting the ALT is of growing interest as a cancer therapy, yet a substantial knowledge gap remains regarding the basic features of telomeres in ALT-positive cells. To address this, we adopted END-seq, an unbiased sequencing-based approach, to define the identity and regulation of the terminal sequences of chromosomes in ALT cells. Our data reveal that the terminal portions of chromosomes in ALT cells contain canonical telomeric sequences with the same 5’ terminus bias (-ATC) observed in non-ALT cells. Furthermore, as reported for non-ALT cells, POT1 is required to preserve the precise regulation of the 5’ end in cells that maintain telomere length using the ALT pathway. Thus, the regulation of the terminal 5’ of chromosomes occurs independently of the mechanism of telomere elongation. Additionally, we employed an S1 endonuclease-based sequencing method to determine the presence and origin of single-stranded regions within ALT telomeres. These data shed light on conserved and unique features of ALT telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6318b7359de932f08bb59735e4b7b9de2484816" target='_blank'>
              CHD6 has poly(ADP-ribose)- and DNA-binding domains and regulates PARP1/2-trapping inhibitor sensitivity via abasic site repair
              </a>
            </td>
          <td>
            Luc Provencher, Wilson Nartey, Peter M Brownlee, Austin W Atkins, J. Gagné, Lou Baudrier, Nicholas S Y Ting, C. Piett, Shujuan Fang, D. D. Pearson, S. Moore, Pierre Billon, Zachary D. Nagel, Guy G. Poirier, Gareth J. Williams, Aaron A Goodarzi
          </td>
          <td>2025-01-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The structural integrity of the chromosomes is essential to every functional process within eukaryotic nuclei. Chromosomes are DNA-histone complexes that are essential for the inheritance of genetic information to the offspring, and any defect in them is linked to mitotic errors, cancer growth, and cellular aging. Changes in the mechanical properties of a chromosome could lead to its compromised function and stability, leading to chromosome breaks. Here, we studied the changes in chromosome physical properties using metaphase chromosomes isolated from moderately malignant (MCF7) and highly malignant (MDA-MB-231) human breast cancer cells exposed to valproic acid (VPA), a known epigenetic modifier drug involved in histone hyperacetylation and DNA demethylation. Due to chromosomal structural intricacy and preparative and technical limitations of analytical tools, we employed a label-free atomic force microscopy approach for simultaneously visualizing and mapping single chromosome elasticity and stretching modulus. Additionally, we performed electron transport characteristics through metaphase chromosomes to elucidate the effect of VPA. The chromosomal elasticity and electron transport alterations are manifestations of VPA-mediated chromatin’s epigenetic changes. Our multiparametric strategy, as shown by receiver operating characteristics analyses with the physical properties of chromosomes, offers a new scope in terms of analytical tools for studying chromosomal structural changes/aberrations linked to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4081e3a40a851c1d9b5593b224de0890423bfe7" target='_blank'>
              Epigenetic Modifier Drug Valproic Acid Enhances Cancer Metaphase Chromosome Elasticity and Electron Transport: An Atomic Force Microscopy Approach
              </a>
            </td>
          <td>
            Tanya Agrawal, Debashish Paul, Amita Mishra, Ganesan Arunkumar, T. Rakshit
          </td>
          <td>2025-02-12</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Super-enhancers (SEs) are expansive cis-regulatory elements known for amplifying oncogene expression across various cancers. However, their role in cervical cancer (CC), a remarkable global malignancy affecting women, remains underexplored. Here we applied integrated epigenomic and transcriptomic profiling to delineate the distinct SE landscape in CC by analyzing paired tumor and normal tissues. Our study identifies a tumor-specific SE at the EFNA1 locus that drives EFNA1 expression in CC. Mechanically, the EFNA1 SE region contains consensus sequences for the transcription factor FOSL2, whose knockdown markedly suppressed luciferase activity and diminished H3K27ac enrichment within the SE region. Functional analyses further underlined EFNA1's oncogenic role in CC, linking its overexpression to poor patient outcomes. EFNA1 knockdown strikingly reduced CC cell proliferation, migration, and tumor growth. Moreover, EFNA1 cis-interacted with its receptor EphA2, leading to decreased EphA2 tyrosine phosphorylation and subsequent activation of the Src/AKT/STAT3 forward signaling pathway. Inhibition of this pathway with specific inhibitors substantially attenuated the tumorigenic capacity of EFNA1-overexpressing CC cells in both in vitro and in vivo models. Collectively, our study unveils the critical role of SEs in promoting tumor progression through the FOSL2-EFNA1-EphA2-Src/AKT/STAT3 axis, providing new prognostic and therapeutic avenues for CC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3e483c9466e24512f997df84699534d6031c04" target='_blank'>
              Super-enhancer-driven EFNA1 fuels tumor progression in cervical cancer via the FOSL2-Src/AKT/STAT3 axis.
              </a>
            </td>
          <td>
            Shu-Qiang Liu, Xi-Xi Cheng, Shuai He, Tao Xia, Yi-Qi Li, Wan Peng, Yaqing Zhou, Zi-Hao Xu, Mi-Si He, Yang Liu, P.-P. Wei, Song-Hua Yuan, Chang Liu, Shu-Lan Sun, Dong-Ling Zou, Min Zheng, Chun-Yan Lan, Chun-Ling Luo, Jin-Xin Bei
          </td>
          <td>2025-02-18</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The interaction between RAD51 and BRCA2 plays a key role in homologous recombination (HR), a critical DNA repair mechanism essential for the survival of cancer cells. Disrupting this interaction increases the sensitivity of cancer cells to chemotherapeutic agents. Here, we employed in silico methods to design a novel class of aptamers—customized single-stranded oligonucleotides—specifically engineered to bind RAD51. These aptamers were developed with the aim of selectively modulating RAD51’s nuclear recruitment and its role in DNA repair processes. The leading candidate displays high affinity for RAD51, competing with BRCA2 for the same interaction site in vitro, as confirmed through biolayer interferometry (BLI) and fluorescence lifetime imaging microscopy (FLIM). We tested the efficacy of the leading aptamer in pancreatic cancer cells and observed that it significantly impedes RAD51 nuclear localization, reduces homologous recombination (HR) efficiency, and increases DNA damage. Critically, our aptamer potentiates the cytotoxicity of the PARP inhibitor olaparib, exploiting synthetic lethality (SL) to induce cancer cell death. Our study showcases an aptamer-based approach for selectively targeting protein interactions within DNA repair pathways, introducing a promising avenue for SL-based treatments applicable to a wide range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc2579e152970b51a457421873665b384bcc5d3" target='_blank'>
              Computationally-designed aptamers targeting RAD51-BRCA2 interaction inhibit RAD51 nuclear recruitment
              </a>
            </td>
          <td>
            Giulia Milordini, E. Zacco, Alexandros Armaos, Francesco Di Palma, Mirco Masi, Martina Gilodi, Jakob Rupert, Laura Broglia, Giulia Varignani, Michele Oneto, Marco Scotto, Roberto Marotta, S. Girotto, Andrea Cavalli, Gian Gaetano Tartaglia
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575e0cd67bcf68fb44b85f6259c1ef6837884bb7" target='_blank'>
              Tousled-like kinase loss confers PARP inhibitor resistance in BRCA1-mutated cancers by impeding non-homologous end joining repair
              </a>
            </td>
          <td>
            Min-ah Kim, Banseok Kim, Jihyeon Jeon, Jonghyun Lee, Hyeji Jang, Minjae Baek, Sang-Uk Seo, Dongkwan Shin, Anindya Dutta, Kyung Yong Lee
          </td>
          <td>2025-01-22</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Selection of a suitable chromatin environment during retroviral integration is a tightly regulated and multilayered process that involves interplay between viral and host factors. However, whether intrinsic chromatin dynamics during mitosis modulate retroviral genome invasion is currently poorly described. Direct interaction between the spumaretrovirus prototype foamy virus (PFV) Gag protein and cellular chromatin has been described as a major determinant for integration site selection. A previous Gag chromatin-binding site (CBS)–nucleosome co-crystal structure revealed an interaction with the histone H2A-H2B acidic patch via a highly conserved arginine anchor residue. Yet, the molecular mechanisms regulating Gag-chromatin capture during PFV infection remain obscure. Here, we investigated the kinetics of Gag-chromatin interactions during mitosis and proviral integration of PFV-infected synchronized cells. Using Gag CBS variant viruses, we showed that alteration of Gag affinity for nucleosome binding induced untimely chromatin tethering during mitosis, decreased infectivity and redistributed viral integration sites to markers associated with late replication timing of host chromosomes. Mutant Gag proteins were moreover defective in their ability to displace the histone H4 tail from the nucleosome acidic patch of highly condensed mitotic chromatin. These data indicate that the mitotic chromatin landscape during Gag–nucleosome interactions hosts PFV integration site selection determinants and that spumaretroviruses evolved high-affinity chromatin binding to overcome early mitosis chromatin condensation for optimal viral DNA tethering, integration and infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c034dba5335395c865d3f3e018bafaae4ee93b" target='_blank'>
              Timed chromatin invasion during mitosis governs prototype foamy virus integration site selection and infectivity
              </a>
            </td>
          <td>
            F. Lagadec, P. K. Singh, C. Calmels, D. Lapaillerie, D. Lindemann, V. Parissi, P. Cherepanov, A. N. Engelman, P. Lesbats
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Transposable elements (TEs) comprise the majority of the human genome. In most somatic tissues, TEs are silenced by DNA methylation and histone modifications. Tumors exhibit changes in DNA methylation including global loss of methylation at regions that are silenced for genome stability, like TEs, and gain of methylation at the promoter regions of tumor suppressor genes. We showed that low doses of DNMT inhibitors (DNMTis) activate type I interferon signaling by reducing methylation and increasing expression of TEs that activate dsRNA sensors. DNMTis increase TEs in murine models of cancer, activating interferon signaling and recruiting CD8+ T cells to kill cancer cells and sensitize to immune checkpoint blockade therapy. We performed a genome-wide analysis of the response to DNMTi in a panel of ovarian cancer cell lines. Alu elements, which directly bind to the dsRNA sensor MDA5, are the most significantly demethylated TEs after DNMTi treatment. Our RNA-sequencing analyses indicated that A-to-I RNA editing of TEs by the RNA editing enzyme ADAR1 was increased after DNMTi treatment. This A-to-I editing makes TE RNA less immunogenic and inhibits the immune response to DNMTi. We find that combining ADAR1 knockdown and DNMTi treatment significantly increases type I interferon signaling and cancer cell secretion of downstream cytokines, including CCL5 and CXCL10. Further, this combination increased recruitment and activation of host immune cells and significantly improved survival in the aggressive ID8 Trp53-/- murine model of ovarian cancer. CITE-Seq (cellular indexing of transcriptomes and epitopes, a type of single-cell RNA sequencing that utilizes antibody tagging of each cell type before sequencing) demonstrated significantly higher infiltration of T and B cells in the tumor microenvironment. To understand the immune cell types causing the DNMTi anti-tumor response, we used antibodies to deplete CD8 T cells and NK cells in the ID8 Trp53-/- model with ADAR1 knockdown and DNMTi treatment. In addition, we performed these experiments in the C57BL/6-Tg(IghelMD4)4Ccg/J mice, which only produce antibodies against “Hel” antigen and have no other B cell responses. Of these lymphocyte populations, depleting T cells had the biggest effect and we observed specific killing of tumor cells by T cells isolated from the mice. We performed immunopeptidomics on cancer cell lines with and without DNMTi and showed that DNMTi upregulate TE peptide presentation by MHC molecules on the surface of tumor cells. Using a novel immuno-engineering approach, we are able to generate antigen-specific T cells targeting TEs and other antigens in murine and human cancer cell line models. These T cells specifically recognize (produce IFNgamma and TNFalpha) and kill the target tumor cells. As DNMTi have also been shown to demethylate genes crucial for T cell function, including IFNgamma and TCF7, we are currently combining these T cells with DNMTis for a novel cell therapy against cancer.
 Citation Format: Katherine B Chiappinelli. Epigenetic regulation of transposable elements to reverse cancer immune evasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Patients with poorly differentiated thyroid cancer (PDTC) and anaplastic thyroid cancer (ATC) face a much poorer prognosis than those with differentiated thyroid cancers. Around 25% of PDTCs and 35% of ATCs carry the BRAFV600E mutation, which constitutively activates the MAPK pathway, a key driver of cell growth. Although combining BRAF and MEK inhibitors can shrink tumors, resistance often develops. The exact cause of this resistance remains unclear. We previously found that in PDTC and ATC cells the BRAFV600Emutation is strongly linked to the expression of ETV5, a transcription factor downstream of the MAPK pathway. In the current study, we observed a significant association between ETV5 expression and the activation of p38, a central component of the MAPK14 pathway. Upon reduction of ETV5 levels, p38 expression and activation decreased, along with its upstream regulators MKK3/MKK6. This suggests that the MAPK and p38/MAPK14 pathways are interconnected and that p38 has oncogenic properties in these cancers. Using high-throughput screening, we established that combining p38 inhibitors with the BRAF inhibitor dabrafenib showed strong synergy in vitro, including in cells resistant to dabrafenib and trametinib that had acquired a secondary TP53 mutation. We then tested this combination in a genetically engineered mouse model (GEMM) of ATC. Overall, our findings suggest an oncogenic link between the MAPK and p38/MAPK14 pathways and that combining p38 pathway inhibitors with dabrafenib-targeted therapy could improve treatment outcomes for aggressive thyroid cancers. However, more specific and effective p38 inhibitors are required to fully harness this potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8307a2ffa6398df1dd15e351dd6013ae9af9fab7" target='_blank'>
              Role of the ETV5/p38 signaling axis in aggressive thyroid cancer cells
              </a>
            </td>
          <td>
            Jerry H Houl, R. Bagheri-Yarmand, M. Kunnimalaiyaan, Paola Miranda Mendez, Joseph L. Kidd, A. Dadbin, Andrea Jurado-Ruiz, Parag A Parekh, Ying C. Henderson, N. S. Chari, A. Thennavan, Reid T Powell, Clifford C Stephan, Xiao Zhao, A. Maniakas, R. Nurieva, N. Busaidy, M. Cabanillas, R. Dadu, M. Zafereo, Jennifer R Wang, Stephen Y Lai, Marie-Claude Hofmann
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Primary cilia have been considered tumor-suppressing organelles in cholangiocarcinoma (CCA), though the mechanisms behind their protective role are not fully understood. This study investigates how the loss of primary cilia affects DNA damage response (DDR) and DNA repair processes in CCA. Human cholangiocyte cell lines were used to examine the colocalization of DNA repair proteins at the cilia and assess the impact of experimental deciliation on DNA repair pathways. Deciliation was induced using shRNA knockdown or CRISPR knockout of IFT20, IFT88, or KIF3A, followed by exposure to the genotoxic agents cisplatin, methyl methanesulfonate (MMS), or irradiation. Cell survival, cell cycle progression, and apoptosis rates were evaluated, and DNA damage was assessed using comet assays and γH2AX quantification. An in vivo liver-specific IFT88 knockout model was generated using Cre/Lox recombination. Results showed that RAD51 localized at the cilia base, while ATR, PARP1, CHK1 and CHK2 were found within the cilia. Deciliated cells displayed dysregulation in critical DNA repair. These cells also showed reduced survival and increased S-phase arrest after genotoxic challenges as compared to ciliated cells. Enhanced DNA damage was observed via increased γH2AX signals and comet assay results. An increase in γH2AX expression was also observed in our in vivo model, indicating elevated DNA damage. Additionally, key DDR proteins, such as ATM, p53, and p21, were downregulated in deciliated cells after irradiation. This study underscores the crucial role of primary cilia in regulating DNA repair and suggests that targeting cilia-related mechanisms could present a novel therapeutic approach for CCA. New and Noteworthy: Our findings reveal a novel connection between primary cilia and DNA repair in cholangiocytes. We showed that DDR and DNA repair proteins localize to cilia, and that deciliation leads to impaired cell survival and S-phase arrest under genotoxic stress. Deciliated cells exhibit heightened DNA damage, evidenced by increased γH2AX signals and comet assay results, a phenotype mirrored in in vivo IFT88 knockout mice. Furthermore, key DDR regulators, including ATM, p53, and p21, are downregulated in deciliated cells following irradiation, highlighting a crucial role for primary cilia in maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4692fdd18b494d4b749d2c1b7ef3cd9972515b" target='_blank'>
              Cholangiocytes’ Primary Cilia Regulate DNA Damage Response and Repair
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) play key roles in a range of biological processes and disease progression. Despite their functional significance and therapeutic potential, lncRNAs’ mechanisms of action remain understudied. One such lncRNA is the Second Chromosome Locus Associated with Prostate-1 (SChLAP1). SChLAP1 is overexpressed in malignant prostate cancer and is associated with unfavorable patient outcomes, such as metastasis and increased mortality. In this study, we demonstrated that SChLAP1 possesses distinct structural domains and conserved regions that may contribute to its function. We determined the secondary structure of SChLAP1 using chemical probing methods combined with mutational profiling (DMS-MaP and SHAPE-MaP). Our in vitro secondary structural model revealed that SChLAP1 consists of two distinct secondary-structural modules located at its 5’ and 3’ ends, both featuring regions with a high degree of structural organization. Our in vivo chemical probing identified potential protein-binding hotspots within the two modules. Overexpression of the modules led to a notable increase in cancer cell proliferation and invasion, proving their functional significance on the oncogenicity of SChLAP1. In conclusion, we discovered functionally important, independent modules with well-defined structures of SChLAP1. These results will serve as a guide to explore the detailed molecular mechanisms by which SChLAP1 promotes aggressive prostate cancer, ultimately contributing to the development of SChLAP1 as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2926b467e70c36d1a40c753470b2a48e5f240e99" target='_blank'>
              LncRNA SChLAP1 promotes cancer cell proliferation and invasion via its distinct structural domains and conserved regions
              </a>
            </td>
          <td>
            Mihyun Oh, Roshni Nagesh Kadam, Zahra Sadruddin Charania, Srinivas Somarowthu
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Whole-genome duplications (WGD) and transposable element (TE) activity can act synergistically in genome evolution. WGDs can increase TE activity directly through cellular stress or indirectly by relaxing selection against TE insertions in functionally redundant, duplicated regions. Because TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE activity following WGD are therefore likely to impact evolution of gene regulation. Yet, the role of TEs in genome regulatory evolution after WGDs is not well understood. Here we used Atlantic salmon as a model system to explore how TE activity after the salmonid WGD ~100MYA shaped CRE evolution. We identified 55,080 putative TE-CREs using chromatin accessibility data from liver and brain. Retroelements were both the dominant source of TE-CREs and had higher regulatory activity in MPRA experiments compared to DNA elements. A minority of TE-subfamilies (16%) accounted for 46% of TE-CREs, but these "CRE-superspreaders" were mostly active prior to the WGD. Analysis of individual TE insertions, however, revealed enrichment of TE-CREs originating from WGD-associated TE activity, particularly for the DTT (Tc1-Mariner) DNA elements. Furthermore, coexpression analyses supported the presence of TE-driven gene regulatory network evolution, including DTT elements active at the time of WGD. In conclusion, our study supports a scenario where TE activity has been important in genome regulatory evolution, either through relaxed selective constraints, or strong selection to recalibrate optimal gene expression phenotypes, during a transient period following genome doubling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 During solid tumor development, gene-poor and lamina-associated genomic regions undergo progressive loss of DNA methylation, whereas promoter CpG islands become DNA hypermethylated. We report evidence suggesting that the recruitment of DNMT3A by histone modifications contributes to this cancer-associated DNA methylome reprogramming. The genomic targeting of DNMT3A is regulated by the competition between NSD1-mediated histone H3K36 methylation and polycomb-mediated H2AK119 ubiquitination, via DNMT3A's PWWP domain and N-terminal ubiquitin-interacting region, respectively. Mutations in NSD1, DNMT3A reader domains or polycomb complexes perturb DNMT3A recruitment and can recapitulate patterns of DNA methylation observed in patient tumors. Lastly, tumors exhibiting progressive hypomethylation, such as those harboring NSD1 mutations, are paradoxically immune-cold despite the increased expression of retrotransposons. These results suggest that DNA hypomethylation and aberrant retrotransposon activation are significant bottlenecks to tumor development and may represent promising therapeutic targets.
 Citation Format: Chao Lu. Interplay between histone modifications and DNA methylation in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8283805e8d954027bac5a7fd108f9580c6417e4d" target='_blank'>
              Abstract PR004: Interplay between histone modifications and DNA methylation in cancer
              </a>
            </td>
          <td>
            Chao Lu
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f817e6c27d9a43bff904bfe868d13c1b02f3616e" target='_blank'>
              Varieties of altered TFE3 can occur in MiT-family-related renal cell carcinomas.
              </a>
            </td>
          <td>
            Allison K Kennedy, Hassan D. Kanaan, Kanika Arora, Harry Zhang, Jason M Hafron, Shelly L Kaufman, Mark A Micale, Ping L Zhang
          </td>
          <td>2025-02-18</td>
          <td>International urology and nephrology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The cell line Sarcoma 180, which is also called S-180 (or S180), was established about 110 years ago from a murine axillary sarcoma. It has been applied in >5000 studies but was never genetically characterized in detail; this study fills that gap. The cell line Sarcoma 180 was analyzed for its chromosomal constitution using molecular cytogenetic approaches, specifically murine multicolor banding (mcb). Additionally, array comparative genomic hybridization was performed to characterize copy number alterations. Sarcoma 180 has a near tetraploid karyotype without Y-chromosome material and only two X-chromosomes. The complex karyotype includes dicentrics and simple and complex rearrangements and shows a relatively high chromosomal instability. An in silico translation of the obtained results to the human genome indicated that Sarcoma 180 is suitable as a model for advanced human mesenchymal chondrosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf21550d200fcd748a7836eb6966ec3a47ab78b" target='_blank'>
              Cytogenomic Characterization of Murine Cell Line Sarcoma 180 = S-180
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2025-01-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal rearrangements involving the Mixed Lineage Leukemia gene (MLL1, KMT2A) are defining a genetically distinct subset in about 10% of human acute leukemias. Translocations involving the KMT2A-locus at chromosome 11q23 are resulting in the formation of a chimeric oncogene, where the N-terminal part of KMT2A is fused to a variety of translocation partners. The most frequently found fusion partners of KMT2A in acute leukemia are the C-terminal parts of AFF1, MLLT3, MLLT1 and MLLT10. Unfortunately, the presence of an KMT2A-rearrangements is associated with adverse outcomes in leukemia patients. Moreover, non-rearranged KMT2A-complexes have been demonstrated to be crucial for disease development and maintenance in NPM1-mutated and NUP98-rearranged leukemia, expanding the spectrum of genetic disease subtypes that are dependent on KMT2A. Recent advances in the development of targeted therapy strategies to disrupt the function of KMT2A-complexes in leukemia have led to the establishment of Menin-KMT2A interaction inhibitors that effectively eradicate leukemia in preclinical model systems and show favorable tolerability and significant efficacy in early-phase clinical trials. Indeed, one Menin inhibitor, Revumenib, was recently approved for the treatment of patients with relapsed or refractory KMT2A-rearranged acute leukemia. However, single agent therapy can lead to resistance. In this Review article we summarize our current understanding about the biology of pathogenic KMT2A-complex function in cancer, specifically leukemia, and give a systematic overview of lessons learned from recent clinical and preclinical studies using Menin inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e763a1f504a59703ce0db774f75dc6bbb915a958" target='_blank'>
              Targeting the Menin-KMT2A interaction in leukemia: Lessons learned and future directions.
              </a>
            </td>
          <td>
            Florian Perner, Jayant Y. Gadrey, Scott A. Armstrong, Michael W. M. Kühn
          </td>
          <td>2025-01-30</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae92845b0c6a04dfb07c44a4ee609b10c82fc09" target='_blank'>
              Unveiling the role of chromosome structure morphology on gene function through chromosome conformation analysis
              </a>
            </td>
          <td>
            Yuxiang Zhan, Aslı Yıldırım, L. Boninsegna, Frank Alber
          </td>
          <td>2025-02-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract G-quadruplex (G4) DNAs are alternative nucleic acid structures, proposed to play important roles in regulating DNA replication, gene transcription, and translation. Several specialized DNA helicases are involved in cellular G4 metabolism, in some cases with redundant functions. Among them, human FANCJ/BRIP1, which has orthologs in all metazoans, is one of the most powerful G4 resolvases, believed to act mainly at DNA replication forks. Here, we tested the effects of a set of hydrazone-derivative G4 ligands in a FANCJ-knocked-out HeLa cell line and in a Caenorhabditis elegans strain, where DOG-1, a FANCJ ortholog, was disrupted, as a whole organism model system. Our results revealed that loss of FANCJ specifically sensitized cancer cells to FIM-15, a mono-guanylhydrazone derivative bearing the diimidazopyrimidine core, among the tested hydrazone-based compounds and induced enhanced DNA damage in different chromosomal sites including telomeric ends. Moreover, dietary administration of FIM-15 to dog-1−/− nematodes stabilized G4 structures in gonadal cell nuclei and resulted in compromised embryonic development in the first-generation post-treatment. Collectively, our findings unveil a specific vulnerability of FANCJ-knocked-out cancer cells (and DOG-1-lacking worms) to G4 stabilization by the FIM-15 compound. This study provides an important proof-of-principle for use of G4 ligands in synthetic lethality-based therapeutic approaches targeting FANCJ-defective cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a485077681fc1cddbc3b2edd2dc8610a684c82b" target='_blank'>
              Effects of hydrazone-based G-quadruplex ligands on FANCJ/BRIP1-depleted cancer cells and a Caenorhabditis elegans dog-1−/− strain
              </a>
            </td>
          <td>
            Marcello Germoglio, Federica D'Aria, G. Cortone, Antonello Prodomo, M. Mahtab, Rita Morigi, Jussara Amato, Francesca M. Pisani, C. Giancola
          </td>
          <td>2025-02-08</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The Hippo/YAP1 signaling pathway regulates normal development by controlling contact inhibition of growth. In cancer, YAP1 activation is often dysregulated, leading to excessive tumor growth and metastasis. SRC kinase can cross talk to Hippo signaling by disrupting adherens junctions, repressing the Hippo cascade, or activating YAP1 to promote proliferation. Here, we demonstrate that the IGF2 messenger RNA-binding protein 1 (IGF2BP1) impedes the repression of YAP1 by Hippo signaling in carcinomas. IGF2BP1 stabilizes the YAP1 messenger RNA (mRNA) and enhances YAP1 protein synthesis through an m6A-dependent interaction with the 3′ untranslated region of the YAP1 mRNA, thereby increasing YAP1/TAZ-driven transcription to bypass contact inhibition of tumor cell growth. Inhibiting IGF2BP1–mRNA binding using BTYNB reduces YAP1 levels and transcriptional activity, leading to significant growth inhibition in carcinoma cells and ovarian cancer organoids. In contrast, SRC inhibition with Saracatinib fails to inhibit YAP1/TAZ-driven transcription and cell growth in general. This is particularly significant in de-differentiated, rather mesenchymal carcinoma-derived cells, which exhibit high IGF2BP1 and YAP1 expression, rendering them less reliant on SRC-directed growth stimulation. In such invasive carcinoma models, the combined inhibition of SRC, IGF2BP1, and YAP1/TAZ proved superior over monotherapies. These findings highlight the therapeutic potential of targeting IGF2BP1, a key regulator of oncogenic transcription networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16383314a8de10bc799de0f2ecf7435c95cb185c" target='_blank'>
              The IGF2BP1 oncogene is a druggable m6A-dependent enhancer of YAP1-driven gene expression in ovarian cancer
              </a>
            </td>
          <td>
            Annekatrin Schott, Theresa Simon, Simon Müller, Alexander Rausch, Bianca Busch, Markus Glaß, Danny Misiak, Mohammad Dipto, Hend Elrewany, Lara Meret Peters, Sunita Tripathee, Ehab Ghazy, Florian Müller, R. Rolnik, M. Lederer, Ali Hmedat, Martina Vetter, Markus Wallwiener, W. Sippl, S. Hüttelmaier, N. Bley
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Protein intrinsically disordered regions (IDRs) are critical gene-regulatory components and aberrant fusions between IDRs and DNA-binding/chromatin-associating domains cause diverse human cancers. Despite this importance, how IDRs influence gene expression, and how aberrant IDR fusion proteins provoke oncogenesis, remains incompletely understood. Here we develop a series of synthetic dCas9-IDR fusions to establish that locus-specific recruitment of IDRs can be sufficient to stimulate endogenous gene expression. Using dCas9 fused to the paradigmatic leukemogenic NUP98 IDR, we also demonstrate that IDRs can activate transcription via localized biomolecular condensation and in a manner that is dependent upon overall IDR concentration, local binding density, and amino acid composition. To better clarify the oncogenic role of IDRs, we construct clinically observed NUP98 IDR fusions and show that, while generally non-overlapping, oncogenic NUP98-IDR fusions convergently drive a core leukemogenic gene expression program in donor-derived human hematopoietic stem cells. Interestingly, we find that this leukemic program arises through differing mechanistic routes based upon IDR fusion partner; either distributed intragenic binding and intrachromosomal looping, or dense binding at promoters. Altogether, our studies clarify the gene-regulatory roles of IDRs and, for the NUP98 IDR, connect this capacity to pathological cellular programs, creating potential opportunities for generalized and mechanistically tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de0eb272216630f4634f73b7ac6d3e80cb86fd8" target='_blank'>
              Biomolecular condensation of human IDRs initiates endogenous transcription via intrachromosomal looping or high-density promoter localization
              </a>
            </td>
          <td>
            Jing Li, Shizhe Liu, Sunghwan Kim, Jacob Goell, Zachary Allen Drum, John Patrick Flores, Alex J Ma, Barun Mahata, Mario Escobar, Alex Raterink, Jeong Hyun Ahn, Erik R Terán, R. S. Guerra-Resendez, Yuhao Zhou, Bo Yu, Michael R. Diehl, Gang Greg Wang, Anna-Karin Gustavsson, D. Phanstiel, Isaac B. Hilton
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Transposable elements (TEs), once considered as "junk" DNA, are now recognized for their epigenetic and regulatory influence, especially in cancer. Cryptic regulatory elements within TEs can be epigenetically reactivated in cancer, contributing to oncogenesis and altering gene expression patterns. This reactivation results in TE-derived transcripts, which have the potential to be translated and form neoantigens that can be presented by the human leukocyte antigens (HLA) complex and recognized by T cells, ultimately impacting the tumor immune response. In this study, we analyzed RNA-seq data from melanoma patients undergoing PD1/PDL1 immunotherapy. By identifying tumor specific TE-derived transcripts and utilizing in silico translation, we detected TE-derived neoantigens in patients with differing responses to immunotherapy. Our findings indicate that the number of TE-derived neoantigens is correlated with patients' response to immunotherapy, with a higher number of neoantigens observed in patients who responded to treatment compared to those who did not. These results imply that TE-derived neoantigens may serve as potential biomarkers for predicting the prognosis of PD1/PDL1 immunotherapy treatment. Incorporating TE-derived neoantigen profiles into clinical decision-making could enhance patient selection for immunotherapy, thereby improving treatment outcomes.
 Citation Format: Yixin Wu, Ting Wang, Yonghao Liang. Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, Gregor Warsow, Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance.
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, Tianjun Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ku70 and Ku80 form Ku, a ring-shaped protein that initiates the non-homologous end-joining (NHEJ) DNA repair pathway.1 Specifically, Ku binds to double-stranded DNA (dsDNA) ends and recruits other NHEJ factors (e.g., DNA-PKcs and LIG4). While Ku binds to double-stranded RNA (dsRNA)2 and traps mutated-DNA-PKcs on ribosomal RNA in vivo,3,4 the physiological significance of Ku-dsRNA interactions in otherwise wild-type cells remains elusive. Intriguingly, while dispensable for murine development,5,6 Ku is essential in human cells.7 Despite similar genome sizes, human cells express ∼100-fold more Ku than mouse cells, implying functions beyond NHEJ, possibly through a dose-sensitive interaction with dsRNA, which is ∼100 times weaker than with dsDNA.2,8 While investigating the essentiality of Ku in human cells, we found that depletion of Ku - unlike LIG4 - induces profound interferon (IFN) and NF-kB responses reliant on the dsRNA-sensor MDA5/RIG-I and adaptor MAVS. Prolonged Ku-degradation also activates other dsRNA-sensors, e.g. PKR that suppresses protein translation, and OAS/RNaseL that cleaves rRNAs and eventually induces growth arrest and cell death. MAVS, RIG-I, or MDA5 knockouts suppressed IFN signaling and, together with PKR knockouts, partially rescued Ku-depleted human cells. Ku-irCLIP analyses revealed that Ku binds to diverse dsRNA, predominantly stem-loops in primate-specific Alu elements9 at anti-sense orientation in introns and 3’-UTRs. Ku expression rose sharply in higher primates tightly correlating with Alu-expansion (r = 0.94/0.95). Together, our study identified a vital role of Ku in accommodating Alu-expansion in primates by mitigating a dsRNA-induced innate immune response, explaining the rise of Ku levels and its essentiality in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24649fa8cee2fc610da3934361ee2a3da1b227a9" target='_blank'>
              Ku suppresses RNA-mediated innate immune responses in human cells to accommodate primate-specific Alu expansion
              </a>
            </td>
          <td>
            Yimeng Zhu, Angelina Li, Suvrajit Maji, Brian J. Lee, Sophie M. Korn, Jake A. Gertie, Tyler J. Dorrity, Jianhua Wang, Kyle J. Wang, Amandine Pelletier, Daniel F. Moakley, Rachel D. Kelly, Antony B. Holmes, Raul Rabadan, D. Edgell, Caroline Schild Poulter, Mauro Modesti, Anna-Lena Steckelberg, Eric A. Hendrickson, Hachung Chung, Chaolin Zhang, Shan Zha
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c09ef83e6dfd77bec2d21b9a7e0a720daa40d98" target='_blank'>
              Multiple regulatory events contribute to a widespread circular RNA downregulation in precancer and early stage of colorectal cancer development
              </a>
            </td>
          <td>
            Alessandro Camandona, Amedeo Gagliardi, N. Licheri, S. Tarallo, G. Francescato, E. Budinska, Martina Čarnogurská, B. Zwinsová, Barbara Martinoglio, Lorenzo Franchitti, Gaetano Gallo, S. Cutrupi, Michele De Bortoli, Barbara Pardini, A. Naccarati, Giulio Ferrero
          </td>
          <td>2025-02-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Since the discovery of the Musashi (MSI) protein, its ability to affect the mitosis of Drosophila progenitor cells has garnered significant interest among scientists. In the following 20 years, it has lived up to expectations. A substantial body of evidence has demonstrated that it is closely related to the development, metastasis, migration, and drug resistance of malignant tumors. In recent years, research on the MSI protein has advanced, and many novel viewpoints and drug resistance attempts have been derived; for example, tumor protein p53 mutations and MSI-binding proteins lead to resistance to protein arginine N-methyltransferase 5-targeted therapy in lymphoma patients. Moreover, the high expression of MSI2 in pancreatic cancer might suppress its development and progression. As a significant member of the MSI family, MSI2 is closely associated with multiple malignant tumors, including hematological disorders, common abdominal tumors, and other tumor types (e.g., glioblastoma, breast cancer). MSI2 is highly expressed in the majority of tumors and is related to a poor disease prognosis. However, its specific expression levels and regulatory mechanisms may differ based on the tumor type. This review summarizes the research progress related to MSI2 in recent years, including its occurrence, migration mechanism, and drug resistance, as well as the prospect of developing tumor immunosuppressants and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dd0dc9ec8f8cc8ab893bc32f47ba299917d963" target='_blank'>
              Update on the Progress of Musashi-2 in Malignant Tumors.
              </a>
            </td>
          <td>
            Yiting Niu, Tao Zhou, Yanjun Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Repetitive elements, mostly derived from transposable elements (TEs), account for half the DNA in human and other mammalian genomes. Although epigenetic mechanisms, including DNA methylation and repressive histone modifications, have evolved to suppress TE activities, TEs have substantially shaped the regulatory landscape of the host genome by contributing regulatory sequences to it. TE-derived sequences are often highly repetitive and thus have low mappability, making it difficult to profile the genomics of TEs using short-read sequencing technology. Many specialized bioinformatics tools have been developed for TE-related analysis, but meaningfully visualizing, navigating, and interpreting such data remains challenging. Here, we describe the WashU Repeat Browser to host genomics profiles of human and mouse TEs using data produced by the ENCODE Project and to support the navigation, interactive visualization, integration, comparison, and analysis in the context of TEs. WashU Repeat Browser is a web-based platform allowing users to browse genomic and statistical signals over repetitive elements derived from ENCODE, Roadmap, and FANTOM datasets. The Browser provides a TE-centric view including TE subfamily enrichments, TE subfamily profiling, as well as overviews of genomic signals on individual TE loci where we extend the WashU Epigenome Browser to display user-selected datasets and TE loci. These features could help to close the gaps in our understanding of the repetitive sequences and their putative regulatory functions and aid investigators in formulating new hypotheses by integrating their data with public data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217e299b91fe8b545cc86ee95ab6ebdd4d9d3fb8" target='_blank'>
              Exploring the epigenome profiles of repetitive elements with the WashU Repeat Browser.
              </a>
            </td>
          <td>
            Jiawei Shen, Siyuan Cheng, Deepak Purushotham, Xiaoyu Zhuo, Alan Y. Du, Wenjin Zhang, Daofeng Li, Ting Wang
          </td>
          <td>2025-02-21</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background/Aim In this study, we provide a comprehensive characterization of HPV-positive primary cervical cancers (CC) and HPV-positive head and neck squamous cell carcinomas (HNSCC) through whole genome next-generation sequencing. Human papillomavirus (HPV) infection, recognized as a definitive human carcinogen, is increasingly acknowledged for its role in development of human cancers. HPV-driven cervical cancers are among the leading causes of cancer-related deaths worldwide, while HPV-driven head and neck cancers exhibit distinct biological and clinical characteristics. Recent data has provided convincing evidence that HPV-related cervical cancer, like HPV head and neck cancer also predict better outcomes, with viral integration patterns further predicting disease related outcomes. Materials and Methods We designed an experimental study that encompasses four pairs of HPV-positive patient samples with controls, utilizing state-of-the-art Next Generation Sequencing (NGS) technology including whole genome sequencing, transcriptome sequencing and virus integration. Results Multiple mutated genes, including TTN, COL6A3, and FLNA, were identified shared between CC and HNSCC. Additionally, we observed a notable proportion of pathways affected by oncogenic alterations, particularly in the RTK-RAS and NOTCH pathways, in both CC and HNSCC. Furthermore, we discovered a shared down-regulation of the Hedgehog signaling pathway based on transcriptome expression analysis in KEGG. We also identified RUNX2 and TFPI as sites of virus integration, and upstream as well as downstream pathway modulators, and represent potential targets for therapeutic interventions. Conclusion Overall, this study showed a thorough comparison between CC and HNSCC from multiple aspects, including gene variations, oncogenic pathways, KEGG enrichment and virus integration sites. However, further studies, which involve larger patient cohorts should be undertaken to further support these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31845b9c0bfa1be5cf5eb5b1191d9c516332d2d" target='_blank'>
              Characterization of the Genomic Landscape in HPV‐positive Cervical and Head and Neck Squamous Cell Carcinomas by Whole Genome Next Generation Sequencing
              </a>
            </td>
          <td>
            JIANLAN REN, Nian Ma, Tyler D. Seckar, S. Bassa, N. Zetola, Surbhi Grover, Zhi Wei, Erle Robertson
          </td>
          <td>2025-02-24</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33f540ba07c7d118037674f121aaa35d22c9a7" target='_blank'>
              Prolonged persistence of mutagenic DNA lesions in somatic cells
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A Fabre, A. Ranzoni, Philip S. Robinson, Lori D Kregar, Matthias Wilk, Steffen Boettcher, Krishnaa Mahbubani, Kourosh Saeb Parsy, K. Gowers, Sam M Janes, Stanley W K Ng, Matt Hoare, Anthony R Green, G. Vassiliou, A. Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael R Stratton, J. Nangalia, Tim H. H. Coorens, P. J. Campbell
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="Transposable elements provide material for novel gene formation. In particular, DNA transposons have contributed several essential genes involved in various physiological or pathological conditions. Here, we discuss recent findings by Tu et al. in Molecular Cell that identify Pogo transposon‐derived gene POGK as tumor suppressor in triple‐negative breast cancer (TNBC) by regulating ribosome biogenesis and restricting cell growth. An isoform‐switch in TNBC results in the loss of POGK capacity to recruit the epigenetic corepressor TRIM28 and to exert its repressive functions. These findings shed light on the potential for TE‐derived genes in providing new therapeutic opportunities for highly malignant TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0b7a06aa8e42e86b3ed5439a233f4a0bb817fd" target='_blank'>
              Pogo transposons provide tools to restrict cancer growth
              </a>
            </td>
          <td>
            Elina Zueva, Marianne Burbage
          </td>
          <td>2025-01-15</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aberrant activation of transposable elements (TEs) has been a well-documented source of genomic instability and disease, stemming from their insertion into genes and their imposition of epigenetic effects on nearby loci. However, the extent to which their disruptive effects involve concomitant or subsequent formation of DNA:RNA hybrids (R-loops) remains unknown. Here we used DNA:RNA immunoprecipitation followed by high-throughput sequencing (DRIP-seq) to map the R-loop profiles of TEs and satellites in D. melanogaster ovaries in control and rhino knockout flies, where dozens of TE families are derepressed. We observe that R-loops form primarily in LTR retrotransposons that carry A/T-rich sequence motifs, which are known to favor R-loop formation at genes in Drosophila and other species. We also report evidence of R-loop formation at 11 of 14 highly abundant D. melanogaster DNA satellites. R-loop formation is positively correlated with expression level for both TEs and satellites; however, neither sequence content nor expression fully explain which repeat families form R-loops, suggesting other factors are at play. Finally, by analyzing population frequencies of R-loop forming TEs, we present evidence that TE copies with high R-loop signal may be under stronger negative selection, which suggests that R-loop formation by TEs may be deleterious to their host. Collectively, these results provide insight into the determinants of R-loop formation at repetitive elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23fd0fe606d16164cbf75d226bfad5eec1b690f" target='_blank'>
              Complex determinants of R-loop formation at transposable elements and major DNA satellites.
              </a>
            </td>
          <td>
            Timothy J. Stanek, Adam Kneebone, Matthew A Lawlor, Weihuan Cao, Christopher E Ellison
          </td>
          <td>2025-02-28</td>
          <td>Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03888dff03b1ffdc768bc303dbe00e2fcd98334c" target='_blank'>
              SNORA37/CMTR1/ELAVL1 feedback loop drives gastric cancer progression via facilitating CD44 alternative splicing
              </a>
            </td>
          <td>
            Banghe Bao, Minxiu Tian, Xiaojing Wang, Chunhui Yang, Jiaying Qu, Shunchen Zhou, Yang Cheng, Q. Tong, Liduan Zheng
          </td>
          <td>2025-01-16</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Among the many features of cancer cells, they commonly display aberrant accumulation of RNA-DNA hybrids known as R-loops. R-loops are formed in healthy cells during many cellular processes, including transcription and DNA replication, and are crucial for proper maintenance of telomeres and centromeres, as well as for adequate mitochondrial function. However, aberrant accumulation of R-loops can lead to increases in DNA damage and inflammation. R-loop accumulation has been documented in different types of cancer, including BRCA1-mutant breast cancer, diffuse large B cell lymphoma, and myelodysplastic syndrome (MDS). Indeed, accumulation of R-loops has been proposed as a unifying mechanism underlying MDS and other myeloid malignancies. Ten-eleven translocation enzymes (TET1, TET2, TET3) are 5-methylcytosine dioxygenases that undergo frequent mutational or functional inactivation in human cancers. We have shown that mice with profound, acute TET deficiency induced by deletion of all Tet genes (hereafter referred to as Tet iTKO mice) show rapid onset of an uncontrolled malignant myeloid expansion. Expanded myeloid cells from Tet iTKO mice show increased levels of R-loops, together with increased DNA damage and inflammation, compared to control myeloid cells. Using Tet iTKO hematopoietic stem and progenitor cells (HSPC), we performed an in vivo CRISPR knockout screen of genes encoding RNA and DNA sensors, as well as R-loop regulators, to find how loss of genes related to nucleic acid sensing and innate immune response would affect the proliferation of TET-deficient myeloid cells. We observed that loss of enzymes that prevent the accumulation of R-loops impaired the uncontrolled malignant myeloid expansion of Tet iTKO cells. CRISPR-mediated depletion of the gene encoding RNase H1, an enzyme that specifically degrades the RNA strand in R-loops, further increased R-loop levels and DNA damage, and decreased proliferation and in vivo expansion of TET-deficient myeloid cancer cells, compared to Tet iTKO cells treated with non-targeting sgRNAs. Moreover, the accumulation of DNA breaks in Tet iTKO cells upon Rnaseh1 loss was not observed in Tet wild-type cells similarly depleted for RNase H1, suggesting a role for TET enzymes and potentially TET-oxidized 5-methylcytosines in DNA repair, which will be discussed in this presentation. Overall, these findings indicate that further increasing R-loops in TET-deficient cancer cells blocks their proliferation, and suggest that RNase H depletion or inhibition could be a new therapeutic strategy for cancer treatment in cancers that display augmented levels of R-loops, independent of the presence of TET mutations.
 Citation Format: Isaac F López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, Arthur X Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao. Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a31032197d36af5a23b4c173a00990f8a714fa2" target='_blank'>
              The Shu complex interacts with the replicative helicase to prevent mutations and aberrant recombination
              </a>
            </td>
          <td>
            Adeola A. Fagunloye, Alessio De Magis, Jordan Little, Isabela Contreras, Tanis J Dorwart, Braulio Bonilla, Kushol Gupta, Nathan L. Clark, Theresa Zacheja, K. Paeschke, Kara A. Bernstein
          </td>
          <td>2025-01-21</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106d9fd456a3ff5a33ecce59e35a5f050b56ab0e" target='_blank'>
              RECQL4 requires PARP1 for recruitment to DNA damage, and PARG dePARylation facilitates its associated role in end joining
              </a>
            </td>
          <td>
            Mansoor Hussain, Prabhat Khadka, Komal Pekhale, Tomasz Kulikowicz, Samuel Gray, A. May, D. Croteau, V. A. Bohr
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A. Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract Resistance to radiotherapy remains a critical barrier in treating colorectal cancer (CRC), particularly in cases of locally advanced rectal cancer (LARC). To identify key kinases involved in CRC radioresistance, we employed a kinase‐targeted CRISPR‐Cas9 library screen. This approach aimed to identify potential kinase inhibitors as radiosensitizers. Our screening identified salt‐inducible kinase 2 (SIK2) as a critical factor in CRC radioresistance. Increased SIK2 expression correlated with reduced tumor regression and poorer outcomes in LARC patients undergoing neoadjuvant chemoradiotherapy. The depletion of SIK2 significantly enhanced radiation‐induced apoptosis and tumor regression. Mechanistically, SIK2 interacts with valosin‐containing protein (VCP), promoting its hyperphosphorylation. This modification improves VCP's capacity to extract K48‐linked ubiquitin‐conjugated proteins from chromatin, thus aiding the recruitment of RPA and RAD51 to DNA damage sites. This mechanism strengthens homologous recombination–mediated DNA repair, which contributes to radioresistance. Importantly, ARN‐3236, a SIK2 inhibitor, markedly sensitized CRC cells to radiation both in vivo and in vitro, providing a potential strategy to overcome radioresistance. In summary, our findings reveal a novel mechanism by which SIK2 contributes to the radioresistance of CRC, proposing SIK2 as a potential therapeutic target with its inhibitor significantly enhancing CRC radiotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27d0bffcf12a04202c03cb4fe50e95cbe2133af6" target='_blank'>
              Salt‐inducible kinase 2 confers radioresistance in colorectal cancer by facilitating homologous recombination repair
              </a>
            </td>
          <td>
            Yuan Meng, Shuo Li, Da‐Shan Lu, Xue Chen, Lu Li, You‐fa Duan, Gao‐yuan Wang, Wenlin Huang, Ran-Yi Liu
          </td>
          <td>2025-01-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The nuclear pore complex (NPC) is an evolutionarily conserved structure that maintains the traffic between the nucleus and cytoplasm. Here, we profiled the expression of nucleoporins (NUPs) in glioblastoma stem cells (GSCs) and found that NUP98 promoted GSC maintenance and therapeutic resistance. GSCs preferentially expressed NUP98, which is essential for GSC tumorigenesis in vitro and in vivo. RNA sequencing demonstrated that NUP98 regulated the expression of key DNA damage and repair pathways. NUP98 formed a complex with transcription factor p65 to directly activate genes involved in homologous repair. Attenuation of NUP98 or p65 expression induced unrepaired intrinsic DNA damage and sensitized GSC to ionizing radiation. Clinically, overexpression of NUP98 informs poor clinical outcome among glioblastoma (GBM) patients. Collectively, our results demonstrate that NUP98-p65 represents a novel node in the regulation of DNA repair, suggesting a therapeutic strategy with potential clinical benefits for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ERBB2 (HER2) is a well-studied oncogene with several driver mutations apart from the well-known amplification defect in some breast cancers. We used the GigaAssay to test the functional effect of HER2 missense mutations on its receptor tyrosine kinase function. The GigaAssay is a modular high-throughput one-pot assay system for simultaneously measuring molecular function of thousands of genetic variants at very high accuracy. The activities of 5,886 mutations were classified, significantly more than mutants previously reported. These variants include 112 new in vitro, 10 known, and 9 new in vivo gain-of-function (GOF) mutations. Many of the GOFs spatially cluster in sequence and structure, supporting the activation mechanisms of heterodimerization with EGFR and release of kinase inhibition by the juxtamembrane domain. Retrospective analysis of patient outcomes from the Genomic Data Commons predicts increased survival with the newly identified HER2 GOF variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b136d3cab282e49a197992ab3575cc9958742822" target='_blank'>
              New Gain-of-Function Mutations Prioritize Mechanisms of HER2 Activation
              </a>
            </td>
          <td>
            C. Giacoletto, Liz J. Valente, Lancer Brown, Sara Patterson, Rewatee H Gokhale, S. Mockus, Wayne W. Grody, , Jerome I Rotter, Martin R. Schiller, Heligenics
          </td>
          <td>2025-03-04</td>
          <td>None</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="LINE-1 retrotransposons, comprising 17% of the genome, drive cancer instability through hypomethylation. The DIAMOND assay, targeting LINE-1 hypomethylation with bisulfite sequencing of cell free DNA, achieved AUCs of 88% to 100% across six cancer types, surpassing mutation-based diagnostics and suggesting utility in early cancer detection and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57cf07128fb65aaad9c55c65c3a93161bd10c47" target='_blank'>
              LINE-1 Retrotransposons as Cell-free DNA Biomarkers for Multi-Cancer Early Detection.
              </a>
            </td>
          <td>
            J. Szymanski, Pradeep S Chauhan, A. Chaudhuri
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The Exon Junction Complex (EJC) is formed by the essential eIF4A3, MAGOH and Y14 core proteins. It is universally deposited during splicing at exon-exon junctions. The EJC is known to impact almost every post-transcriptional regulatory step throughout the life of mRNAs including their modifications, splicing, decay and trafficking. Its dysregulation leads to neurodevelopmental pathologies. Here we show that EJC-i, a compound known to block the ATPase activity of eIF4A3, inhibits de novo EJC assembly. EJC-i and targeted knockdown of either eIF4A3 or Y14 core EJC subunits lead to very similar phenotypes by impacting the destiny of mRNAs due to alterations in alternative splicing, nonsense-mediated mRNA decay, genome-wide m6A methylation and proper intracellular addressing of specific transcripts, in particular to the centrosome. Both EJC impairment methods disrupt the centrosome function that might be responsible for mitotic arrest at prometaphase. As a small molecule that readily diffuses into cells, EJC-i is a particularly attractive easy to use and versatile tool to investigate EJC functions in live cells or whole organism that are not prone to genetic manipulation. Indeed, this property was used to disrupt ciliogenesis in primary neural stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d82ec08c9ad540baf17cea17836bedb8e4f45c6" target='_blank'>
              Chemical inhibition of Exon Junction Complex assembly impairs mRNA localization and neural stem cells ciliogenesis
              </a>
            </td>
          <td>
            Tommaso Villa, Oriane Pourcelot, D. Dierks, Marion Faucourt, Cindy Burel, Floric Slimani, Léa Guyonnet, Nathalie Spassky, Schraga Schwartz, Edouard Bertrand, Olivier Bensaude, H. Le Hir
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Introduction Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response. Methods In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment. Results Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure. Discussion The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA damage responses primarily coordinate intracellular processes, but their effects on the cell surface remain poorly understood. We uncovered a DNA damage response at the cell surface that regulates complement dependent cytotoxicity (CDC), a key mechanism of antibody-mediated tumor cell killing. We found that several mCRPs, such as CD46, CD55, and CD59, are upregulated in diffuse large B-cell lymphoma (DLBCL) cells following chemotherapy. These mCRPs protect cells from CDC, including CDC triggered by anti-CD20 immunotherapy. Chemotherapy pre-treatment reduced the efficacy of rituximab-induced CDC, with CD59 exerting a dominant function. A high-throughput kinase screen revealed that Chk1 inhibition restored CDC sensitivity. Mechanistically, chemotherapy-induced CD59 expression was Chk1-dependent and regulated at an Sp1-driven promoter site. Inhibition of Sp1 with Mithramycin A abrogated chemotherapy-induced CD59 transcription. Co-immunoprecipitation mass spectrometry revealed that DNA damage releases transcriptional repressors from Sp1, an effect reversed by Chk1 inhibition. These findings define a Chk1-Sp1-CD59 signaling axis linking genotoxic stress to complement modulation. Our study reveals potential antagonism between chemotherapy and antibody-mediated CDC and highlights consideration in sequencing combination therapies and a therapeutic potential of Chk1 inhibitors in maintaining CDC efficacy during chemoimmunotherapy. Short Summary Rituximab-based chemoimmunotherapy is the first-line treatment for DLBCL. Our study reveals that chemotherapy induces resistance to rituximab-mediated cell killing. We identify this resistance as driven by a Chk1-Sp1-CD59 signaling axis and provide a therapeutic rationale for targeting this pathway to enhance chemoimmunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="SUMMARYHuman papillomaviruses (HPVs) are small DNA viruses that are responsible for significant disease burdens worldwide, including cancers of the cervix, anogenital tract, and oropharynx. HPVs infect stratified epithelia at a variety of body locations and link their productive life cycles to the differentiation of the host cell. These viruses have evolved sophisticated mechanisms to exploit cellular pathways, such as DNA damage repair (DDR), to regulate their life cycles. HPVs activate key DDR pathways such as ATM, ATR, and FA, which are critical for maintaining genomic integrity but are often dysregulated in cancers. Importantly, these DDR pathways are essential for HPV replication in undifferentiated cells and amplification upon differentiation. The ability to modulate these DDR pathways not only enables HPV persistence but also contributes to cellular transformation. In this review, we discuss the recent advances in understanding the mechanisms by which HPV manipulates the host DDR pathways and how these depend upon enhanced topoisomerase activity and R-loop formation. Furthermore, the strategies to manipulate DDR pathways utilized by high-risk HPVs are compared with those used by other DNA viruses that exhibit similarities and distinct differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3977d9b3bc13e1ba0a9ea26fcf6605bb8b2b35" target='_blank'>
              How human papillomavirus (HPV) targets DNA repair pathways for viral replication: from guardian to accomplice.
              </a>
            </td>
          <td>
            A. Vats, Lou Laimins
          </td>
          <td>2025-01-27</td>
          <td>Microbiology and molecular biology reviews : MMBR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Chromosomal aneuploidies are a major cause of developmental failure and pregnancy loss. To investigate the possible consequences of aneuploidy on early embryonic development in vitro, we focused on primed pluripotent stem cells that are relatable to the epiblast of post-implantation embryos in vivo. We used human induced pluripotent stem cells (iPSCs) as an epiblast model and altered chromosome numbers by treating with reversine, a small-molecule inhibitor of monopolar spindle 1 kinase (MSP1) that inactivates the spindle assembly checkpoint, which has been strongly implicated in chromosome mis-segregation and aneuploidy generation. Upon reversine treatment, we obtained cells with varied chromosomal content that retained pluripotency and potential to differentiate into cells of three germ lineages. However, these cells displayed lagging chromosomes, increased micronuclei content, high p53 expression and excessive apoptotic activity. Cell proliferation was not affected. Prolonged in vitro culture of these cells resulted in a selective pool of cells with supernumerary chromosomes, which exhibited cellular hypertrophy, enlarged nuclei, and overproduction of total RNAs and proteins. We conclude that increased DNA damage responses, apoptosis, and improper cellular mass and functions are possible mechanisms that contribute to abnormal epiblast development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d165ff1955e4cb3146937b90c98c9776c59f3b72" target='_blank'>
              Chromosome number alterations cause apoptosis and cellular hypertrophy in induced pluripotent stem cell models of embryonic epiblast cells
              </a>
            </td>
          <td>
            Althea Stella Anil Martis, Loshini Soundararajan, Pallavi Shetty, Syed Moin, Tejashree Vanje, Yogeshwaran Jai Sankar, Shagufta Parveen
          </td>
          <td>2025-01-15</td>
          <td>Biology Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain-and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) remains a leading cause of cancer associated deaths in women owing to its highly metastatic potential and limited treatment options. Recent studies have shown that expression of proteins associated with epigenetic regulation of gene expression are associated with metastatic relapse, however targeting epigenetic regulatory proteins has not resulted in effective therapies for TNBC in the clinic. The ABL tyrosine kinases promote metastasis of breast cancer cells in mouse models. However, a role of ABL kinases in the regulation of epigenetic processes in solid tumor metastasis remains unexplored. Here we show that inactivation of ABL kinases in bone metastatic TNBC cells led to a significant enrichment in gene signatures associated with the PRC2 protein complex, revealing a functional link between ABL kinases and the PRC2 complex. ABL inactivation promotes EZH2-T487 phosphorylation through the regulation of a FAK-CDK1 signaling axis. We find that phosphorylated EZH2 T487 or a phosphomimic EZH2 T487D mutant exhibit increased binding to non-canonical binding partners of EZH2 including c-MYC and ZMYND8. Notably, we identify a therapeutic vulnerability in TNBC cells whereby combination treatment with ABL allosteric inhibitors and EZH2 inhibitors elicits a synergistic decrease in TNBC cell survival in vitro, and impairs TNBC metastasis, prolonging survival of tumor-bearing mice treated with the combination therapy. One Sentence Summary ABL Kinases indirectly impact EZH2 catalytic activity by blocking a signaling cascade that leads to changes in the phosphorylation, protein interactions, and function of the PRC2 catalytic component EZH2 in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73bead74e9165b080991fb6c70e4def4cd0e8e6" target='_blank'>
              ABL Kinases Modulate EZH2 Phosphorylation and Signaling in Metastatic Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Ashley Colemon, Tatiana Prioleau, Clay Rouse, A. Pendergast
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Fungal genome sizes exhibit more than a 100-fold variation, largely driven by the expansion of repetitive sequences such as transposable elements (TEs). Genomic defenses have evolved to counteract TE proliferation through silencing mechanisms operating at the epigenetic or transcript level. In fungi, repeat-induced point mutation (RIP) targets TEs by recognizing repetitive sequences and inducing mutagenesis. However, the prevalence of RIP across the fungal kingdom and the fidelity of the canonical C-to-T mutation signatures remain unclear. In this study, we address these gaps by tracking shifts in genome architecture across the fungal kingdom. We show that the loss of RIP components is associated with an 80% (∼30 Mb) increase in genome size in ascomycetes (class of Leotiomycetes). To track the impact of genome defenses, we designed a quantitative screen for RIP-like mutation signatures. The phylum of ascomycetes was unique in showing enrichment in mutation signatures in non-coding and repetitive sequences, consistent with a phylogenetically restricted occurrence of RIP-like genome defense systems. Then, we gene functions associated in a phylogenetic context with RIP-like mutation signatures. We identified a zinc-finger protein as the strongest candidate underpinning a novel mechanism of genome defenses. Our findings reveal the multifaceted drivers of genome defense systems and their close ties to genome size evolution in fungi, highlighting how proximate molecular mechanisms can shape genome evolution on deep phylogenetic scales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6148de5b7f0cf13898d5f28034ac9ab633d52a0" target='_blank'>
              A systematic screen for genetic factors underpinning transposon defense systems across the fungal kingdom
              </a>
            </td>
          <td>
            Thomas Badet, D. Croll
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352956df3f50dd3a8802a00f3ce2fb6c110258ba" target='_blank'>
              SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
              </a>
            </td>
          <td>
            Zhongqing Ren, Lindsey E. Way, Xindan Wang
          </td>
          <td>2025-01-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) are non-B form secondary DNA structures that are highly conserved across evolution. G4 structures occupy key regulatory sites in the mammalian genomes and are implicated in several cellular processes. However, the mechanisms by which G4s contribute to distinct facets of genome function are not well understood. Here, we conduct a proteomics screen with G4s of diverse topologies to uncover novel G4 binding activities in genomic regulators of nucleosome remodeling, paraspeckle assembly, RNA splicing, and 3D genome organization. Among the most prominent hits, we identify the genomic architectural protein, CTCF, as one of the strongest G4 binders. Building on this discovery, we perform extensive biochemical validation of CTCF-G4 interaction and identify CTCF mutants, with pronounced affinity for G4s over its consensus DNA motif. By implementing well-established approaches and developing new G4 mapping tools, we define a comprehensive catalog of genomic G4s and investigate their association with CTCF binding. Using genetic reconstitution of mouse embryonic stem cells with a G4-specific CTCF mutant, we ascribe genomic functions to CTCF-G4 interactions in regulation of CTCF occupancy, chromatin looping and gene expression. Interestingly, our studies reveal a subset of G4-linked chromatin loop anchors that form “persistent loops” which are retained even upon CTCF depletion. Collectively, our work establishes the G4 binding activity of CTCF and provides new insights into the functional significance of G4 structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92ea67e64484dade77882133de154f4d5bb2f2" target='_blank'>
              CTCF directly binds G-quadruplex structures to regulate genome topology and gene expression
              </a>
            </td>
          <td>
            Daniela Samaniego-Castruita, Isabella Han, Roxroy C Morgan, Samantha Carpenter, Bryce Williams, Abhijit Chakraborty, Ishwar Radhakrishnan, F. Ay, Samuel A. Myers, Anjana Rao, Vipul Shukla
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for the removal of bulky DNA damage such as pyrimidine dimers induced by ultraviolet light. The NER pathway detects such lesions and excises the damaged strand through incisions at 5 ’ and 3’ of the damage. The 5’ incision is catalyzed by a heterodimeric endonuclease composed of XPF (catalytic subunit) and ERCC1 (non-catalytic). Here, we show that the genome of Trypanosoma brucei, the causal agent of human African trypanosomiasis or sleeping sickness, codes for an XPF ortholog. RNAi silencing of TbXPF sensitizes cells to UV irradiation, thus providing evidence that NER operates in these parasites. In addition, TbXPF confers protection against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C respectively. Consistent with a role in DNA repair, XPF localizes to the cell nucleus, and is found associated to nucleoplasmic and nucleolar regions. The presence of a functional NER pathway in trypanosomes suggests that in vivo, they are susceptible to undergo replication and transcription-blocking DNA damages. The results obtained with various antitumor agents provide proof of concept for the potential of NER inhibition as a means to improve antiparasitic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eda64b2f73b09173646bdfd81a7c10d708ef6e" target='_blank'>
              Role of the Nucleotide Excision Repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, , I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="MET fusions (MET-Fs) are oncogenic drivers that remain poorly characterized. Analysis of 56 MET-F-positive tumors from an institutional cohort of 91,119 patients (79,864 DNA-seq plus 11,255 RNA-seq) uncovered two forms of MET-F pathobiology. The first group featured 5' partners with homodimerization domains fused in-frame with MET-tyrosine kinase domain (TKD), primarily originated from translocations, frequently excluded MET exon 14, mediated oncogenesis through cytoplasmic aggregation and constitutive activation, and were markedly sensitive to MET tyrosine kinase inhibitors (TKI) in pre-clinical models and patients with lung cancer. The second group lacked partner homodimerization motifs and retained MET transmembrane and extracellular domains. Their pathogenesis involved intrachromosomal rearrangements, resulting in partner selection for promoter hijacking and fusion allele amplification. Membrane-bound fusions were enriched in gliomas with RTK co-alterations. We provide a framework to comprehend the heterogeneous landscape of MET-Fs, supporting that fusion oncogenicity and MET-TKI sensitivity are determined by structural topology and pathogenomic context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020be0c73016e3bdb9e553a5f923d18e28c49838" target='_blank'>
              Pan-cancer analysis of oncogenic MET fusions reveals distinct pathogenomic subsets with differential sensitivity to MET-targeted therapy.
              </a>
            </td>
          <td>
            Christopher A Febres-Aldana, M. Vojnic, I. Odintsov, Tom Zhang, Ryan Cheng, Catherine Z. Beach, Daniel Lu, M. Mattar, Andrea M Gazzo, Leo Gili, M. Harshan, Ali Ameri, S. Machnicki, Xiuying Xiao, William W Lockwood, Xiaoyan Zhou, Qianlan Yao, A. Drilon, Natasha Rekhtman, Nameeta Shah, Anqi Li, Zebing Liu, Soo-Ryum Yang, M. Davare, Marc Ladanyi, R. Somwar
          </td>
          <td>2025-02-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, Greta Schmauck, Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose: To look into the influence of DNAJC12 knockdown on the progression and radio-sensitivity of esophageal squamous cell carcinoma (ESCC), with a focus on cellular mechanics and tumor microenvironment interactions. Methods: The TCGA database combined with immunohistochemical staining was used to validate the DNAJC12 expression in ESCC patients from the perspective of the clinic. DNAJC12 knockdown was performed in TE-1 and KYSE-150 cell lines to assess changes in proliferation, migration, invasion, apoptosis, and cellular mechanical properties (e.g., stiffness, adhesion, and contractility). The downstream molecule regulated by DNAJC12 was explored using Western blotting and biomechanical assays. The effect of DNAJC12 knockdown on tumor radiosensitivity was evaluated in vivo, with a focus on tumor stiffness and extracellular matrix (ECM) remodeling under irradiated conditions. Results: Upon analyzing the TCGA database and examining tumor tissue samples from patients, it was discovered that DNAJC12 exhibited high expression levels in tissues of ESCC. Vitro experiments showed that DNAJC12 knockdown significantly decreased cellular proliferation and migration (P < 0.05). Biomechanical assays revealed that DNAJC12 knockdown decreased cellular stiffness and contractility, suggesting a role in regulating cytoskeletal dynamics. Molecular analysis showed downregulation of P-ERK, MMP-2, N-Cadherin, P-P38, Snail, Vimentin, β-Catenin, Fibronectin, and Twist alongside upregulation of E-Cadherin (P < 0.05). Overexpression of SNAI1 could restore the proliferative and migratory capabilities of cells with downregulated DNAJC12. In vivo experiments, knockdown of DNAJC12 resulted in faster tumor growth under irradiated conditions (P < 0.05). Conclusion: DNAJC12 knockdown slows ESCC progression by modulating cellular biomechanical properties and molecular pathways. However, it enhances tumor growth post-radiotherapy, potentially due to altered mechanosensitive signaling and ECM remodeling. These findings highlight the interplay between molecular biology and biomechanics in ESCC progression and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba737d66daf461c8c86cfa467ed3f95f6246b1d2" target='_blank'>
              Targeting BRCA1-deficient PARP inhibitor-resistant cells with nickases reveals nick resection as a cancer vulnerability.
              </a>
            </td>
          <td>
            Jenna M Whalen, Jillian Earley, C. Wisniewski, A. Mercurio, Sharon B Cantor
          </td>
          <td>2025-01-21</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/272731c989a5ab1beb52c9be0f816a9c8eb7f104" target='_blank'>
              Homologous recombination promotes non-immunogenic mitotic cell death upon DNA damage
              </a>
            </td>
          <td>
            R. Szmyd, S. Casolin, Lucy French, A.G. Manjon, Melanie Walter, Léa Cavalli, Christopher B. Nelson, Scott G Page, Andrew Dhawan, Eric Hau, Hilda A. Pickett, Harriet Gee, Anthony J. Cesare
          </td>
          <td>2025-01-13</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0676fb55661a5af6ba07cbca79e578a0d145a7" target='_blank'>
              SART3 promotes homologous recombination repair by stimulating DNA-RNA hybrids removal and DNA end resection
              </a>
            </td>
          <td>
            Hui Fu, Min Huang, Honglin Wu, Hui Zheng, Yifei Gong, Lingyu Xing, Juanjuan Gong, Ruiyuan An, Qian Li, Xinyu Jie, Xiaolu Ma, , Caixia Guo
          </td>
          <td>2025-03-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Evidence suggests that increases in ploidy have occurred frequently in the evolutionary history of organisms and can serve adaptive functions to specialized somatic cells in multicellular organisms. However, the sudden multiplication of all chromosome content may present physiological challenges to the cells in which it occurs. Experimental studies have associated increases in ploidy with reduced cell survival and proliferation. To understand the physiological challenges that suddenly increased chromosome content imposes on cells, we used S. cerevisiae to ask how much chromosomal DNA cells may contain and what determines this limit. We generated polyploid cells using 2 distinct methods causing cells to undergo endoreplication and identified the maximum ploidy of these cells, 32–64C. We found that physical determinants that alleviate or exacerbate cell surface stress increase and decrease the limit to ploidy, respectively. We also used these cells to investigate gene expression changes associated with increased ploidy and identified the repression of genes involved in ergosterol biosynthesis. We propose that ploidy is inherently limited by the impacts of growth in size, which accompany whole-genome duplication, to cell surface integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c9419ff84380a674e570b7e466126c557aa8fb" target='_blank'>
              Cell integrity limits ploidy in budding yeast
              </a>
            </td>
          <td>
            Juliet Barker, Andrew W. Murray, Stephen P Bell
          </td>
          <td>2025-01-13</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine.
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, J. Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="SKA3, an important factor in cell cycle regulation, is involved in spindle assembly and kinetochore function, playing a critical role in maintaining cancer cell proliferation and division. However, its specific roles and regulatory mechanisms in cancer remain not fully understood. Large-scale datasets from multiple public databases, including The Cancer Genome Atlas and Genotype-Tissue Expression, covering various cancer types, were integrated. Systematic analysis revealed that SKA3 exhibits aberrant expression patterns in multiple cancers and is significantly associated with tumor progression and poor patient prognosis in certain cancers. We explored the status of SKA3 gene mutation, gene amplification and promoter region methylation in various tumors. In the context of immunotherapy, we assessed the value of SKA3 in cancer. Analyzing the correlation between SKA3 expression levels and immune checkpoints and immune cell infiltration, we discovered that SKA3 could serve as a novel immunotherapy biomarker across multiple cancers, guiding clinical immunotherapy decisions. Finally, SKA3 knockdown inhibited lung adenocarcinoma cell proliferation and metastasis. In conclusion, this study provides new insights into the role of SKA3 in cancer and offers significant theoretical and experimental evidence for its development as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colon cancer progression heavily relies on intricate mechanisms of invasion, metastasis, and migration. Tight junction protein Cldn2 has emerged as a potential regulator of these processes. This study aimed to elucidate the molecular mechanisms linking Clan2 deletion to gene expression changes related to motility, invasion, and metastasis in colon caner. Methods CRISPR/Cas9-mediated knockout of human Cldn2 in HCT116 cells was conducted, and the resulting cells were compared to the wild-type cells using real-time PCR to analyze the expression of genes associated with invasion and metastasis. Results Cldn2-KO resulted in a widespread downregulation of genes linked to motility, invasion, and metastasis, including ZONAB, NDRG1, Cldn14, Cldn23, Bcl2, , P53, and BCL-6. These findings suggest a potential regulatory role of Cldn2 in the expression of these genes, influencing colon cancer cell migration and spread. Conclusion This study identified Claudin-2 as a crucial regulator of genes involved in colorectal cancer metastasis. Downregulation of these genes upon Claudin-2 deletion suggests its inhibitory role in cancer cell motility and invasion. Further investigation into the specific downstream signaling pathways mediated by Claudin-2 could pave the way for novel therapeutic strategies targeting metastasis inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd54734c6f14d3222e6a9fb2a03c4f6f4a65ac00" target='_blank'>
              CRISPR/Cas9-Mediated Cldn2 Knockout in HCT116 Cells, Reveals Its Crucial Role in Colorectal Cancer progression
              </a>
            </td>
          <td>
            R. Alghamdi, Maryam H. Al-Zahrani
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The current treatment of triple-negative breast cancer (TNBC) is still primarily based on platinum-based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin-tolerant cells after platinum administration. Here, it is reported that upregulated N 6-methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase-like 3 (METTL3), facilitating METTL3 interaction with Wilms'-tumor-1-associated protein and subsequently increasing m6A of transcript-associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2-Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon.
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9961e6ed0a6d198d471203dc74fb5135efa471b" target='_blank'>
              CDKN2A, a key gene in copper-induced cell death model, influencing melanoma invasion and apoptosis
              </a>
            </td>
          <td>
            Jing Li, Xi Yang, Cunli Yin, Siru Li, Yan Xu, Bin Liu
          </td>
          <td>2025-02-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Post-transcriptional chemical modifications to RNA, or the epitranscriptome, play important roles in RNA metabolism, gene regulation, and human disease, including viral pathogenesis. Modifications to the RNA viral genome and transcripts of human immunodeficiency virus 1 (HIV-1) have been reported, including methylation of adenosine (m6A) and cytosine (m5C), acetylation of cytosine, pseudouridylation (psi), and conversion of adenosine to inosine, and their effects on virus and host biology have been investigated. However, diverse experimental approaches have been used, making clear correlations across studies difficult to assess. To address this need, we propose the establishment of a reference HIV-1 epitranscriptome. We sequenced the model NL4-3 HIV-1 genome from infected Jurkat CD4+ T cells cells using the latest nanopore chemistry, custom RNA preparation methods, and commercial base-calling algorithms. This resulted in a reproducible sense and preliminary antisense HIV-1 epitranscriptome where m6A, m5C, psi, ands inosine could be identified by multiplexed base-calling. Multiplexed base-calling miscalled modifications due to sequence and neighboring modification contexts, which we demonstrate can be corrected with synthetic HIV-1 RNA fragments. We validate m6A modification sites with a small molecule inhibitor of methyltransferase-like 3 (METTL3), STM2457. We conclude that modifications do not change substantially under combination antiretroviral therapy (cART) treatment or in primary CD4+ T cells. Samples from patients living with HIV reveal conservation of certain modifications, such as m6A. Our approach and reference data offer a straightforward benchmark that can be adopted to help advance rigor, reproducibility, and uniformity across future HIV-1 epitranscriptomics studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01daaec39b10dfae0360e2c425526168147edec5" target='_blank'>
              An HIV-1 Reference Epitranscriptome
              </a>
            </td>
          <td>
            Michael S. Bosmeny, Adrian A. Pater, Li Zhang, Beverly E. Sha, Zidi Lyu, Lydia Larkai, M. Damha, Joao I. Mamede, Keith T. Gagnon
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="During mitosis, stable but dynamic interactions between the centromere DNA and kinetochore complex enable accurate and efficient chromosome segregation. Even though many proteins of the kinetochore are highly conserved, centromeres are among the fastest evolving regions within a genome, showing extensive variation even on short evolutionary timescales. Here, we sought to understand how new types of centromeres emerge and reach fixation by mapping centromere evolution across 138 budding yeast species and over 2,500 natural strain isolates. We show that new centromeres spread progressively via drift and subsequent selection, and that the kinetochore interface, which is evolving slowly in relative terms, determines which new centromere variants are tolerated. Together, our findings provide insight into the evolutionary constraints and trajectories shaping centromere evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1b9e4f4c3d05e240290a3c1358cd60478a138" target='_blank'>
              Centromeres evolve progressively through selection at the kinetochore interface
              </a>
            </td>
          <td>
            Jana Helsen, Kaustubh Ramachandran, Gavin Sherlock, Gautam Dey
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedbdbd3810726b7db1ec867ee9197cd7581b73a" target='_blank'>
              Epigene functional diversity: isoform usage, disordered domain content, and variable binding partners
              </a>
            </td>
          <td>
            Leroy Bondhus, Aileen A. Nava, Isabelle S. Liu, Valerie A. Arboleda
          </td>
          <td>2025-02-01</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Giant tortoises exhibit exceptional longevity, often exceeding the human lifespan. To understand the genomic and epigenomic basis of their longevity, awe analyzed the DNA sequence and methylome of Jonathan, an Aldabra giant tortoise (Aldabrachelys gigantea), estimated to be 192 years old. Relative to other giant tortoises (Aldabrachelys gigantea and Chelonoidis abingdonii), we found Jonathan has gene variants in pathways associated with aging, including DNA repair and telomere regulation. Consistent with his advanced age, Jonathan has significant age-related changes in DNA methylation and methylation entropy, compared with a 5-year-old Aldabra individual. Notably, we found that low entropy regions in Jonathan’s methylome were enriched for genes involved in the electron transport chain. This suggests that high-fidelity transcription of these genes may be crucial for extreme longevity. With this data, we propose a model for aging, that links efficient mitochondrial energy production with nuclear maintenance of low methylation entropy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a4f4477e8b781bb6e194d55ad6b5369b0f8013" target='_blank'>
              Epigenomic insights into extreme longevity in the world’s oldest terrestrial animal, Jonathan
              </a>
            </td>
          <td>
            Benjamin Vaisvil, Daniel P. Schmitt, Angela Jones, V. Kapatral, James M. Ford, Madison L. Taylor, Mathia Colwell, Jonathan Hollins, Sam Pascucci, Konstantin Weissenow, B. Rost, Pascal Notin, Justin Gerlach, Thomas C. Terwilliger, Li-Wei Hung, Lars Juhl Jensen, Steve Horvath, Christopher D. Faulk, Yanjun Ma, Stephen W. Clark
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Post-transcriptional regulation by RNA-binding proteins (RBPs) is critical for mRNA stability, localization, and translation, primarily through interaction with the 3’-untranslated region. Dysregulation of RBPs has been associated with various cancers, with fragile X-related protein 1 (FXR1) emerging as a critical RBP involved in tumorigenesis through its interactions with target mRNAs. Despite its significance, the specific role of FXRI in cancer progression remains underexplored. In this study, we investigated FXR1’s function using SH-SY5Y cells. RNA immunoprecipitation (RNA-IP) assay was employed to isolate RNA complexes associated with FXR1. We generated stable cell lines with either FXR1 overexpression or silencing to assess the impact of FXRI binding to mRNA complexes. Subsequent analyses, including quantitative reverse transcription polymerase chain reaction, correlation analysis, gene expression profiling, and survival analysis, were performed to validate the interactions of FXR1 with target mRNAs. Our RNA-IP analysis identified several mRNAs significantly enriched in FXR1-bound RNA complexes, including SHISAL1, SLC43A3, NBAT1, PDZK1IP1, ACKR3, KCNN3, NECAB2, ANO5, ATOH8, IGFBP7, LEMD1, GPR35, WNT7A, and F2RL3. Notably, we observed changes in the expression levels of these genes following FXR1 overexpression or depletion, indicating FXR1-mediated functional regulation. Co-expression analysis further supported FXR1’s association with these target mRNAs. These findings highlight the significant role of FXR1 in regulating the stability and expression of key mRNAs implicated in malignancies. The dysregulation of FXR1 and its interaction with these target transcripts suggest that FXR1 plays a critical role in tumor biology, potentially offering new avenues for therapeutic interventions. This study provides a deeper understanding of FXR1’s involvement in cancer and underscores the importance of RBPs in the post-transcriptional regulatory landscape of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68584e4b61388edd6ff9ff31b6a17585ebe8fc4" target='_blank'>
              FXR1 modulates the expression of oncogenes and tumor suppressor genes associated with poor cancer prognosis
              </a>
            </td>
          <td>
            F. Khan, Xin-Ying Ji, Shaoping Ji, Jingcheng Dong
          </td>
          <td>2025-02-05</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="LINE-1 (L1) is a type of mobile genetic element (retrotransposon) that can self-copy into distinct parts of its host genome by dynamically shaping its content. In humans, misregulation of these elements have been associated as the potential cause of distinct types of pathologies and neurodevelopmental disorders, including cancer, Rett syndrome and Aicardi Goutierrez disease. However, the precise annotation of these 6 kb long nucleotide sequence is a challenging task due to the high similarity between their internal regions (coding and non-coding) for distinct L1 families. Current L1 reference annotations such as RepeatMasker still lack sequence details and also provide confounding annotated information, that includes thousands of incomplete L1 sequences annotated as representative full-length, and missing information on their regulatory and coding regions (5’UTR, ORF1, intron, ORF2 and 3’UTR). Additionally, current L1 resources also provide annotations based on consensus reference genomes and does not include individual genome information at allele-level and nucleotide-level resolution. Here, we present L1Farm as a complementary databank and annotation resource that contains a comprehensive analysis and annotation of L1 subfamilies identified in the current human reference genome (build Hg38) and in two individual Asian and Caucasian genomes at allele-level and nucleotide-level resolution. L1 elements sequences were also annotated with genomic loci and ranked according to their full-length content and similarity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76f0f9e2e6a72987b0b32fe629f320cea26077d1" target='_blank'>
              A comprehensive identification and annotation of mobile LINE-1 retrotransposons across human reference and personal genomes
              </a>
            </td>
          <td>
            Adriano Ferrasa, R. Herai
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DNA decoys are expected to be among the nucleic acid drugs used to downregulate the transcription process. However, spatially controlling the on/off efficacy of DNA decoys to avoid side effects on normal cells is challenging. To reduce undesired decoy function in normal cells, we adopted catalytic hairpin assembly (CHA) to produce a DNA duplex from a hairpin DNA pair in response to a specific microRNA (miRNA). We designed the DNA hairpin pairs to form a DNA decoy that binds to NF-kB, whose overexpression is related to many diseases, including cancer. The transformation of the DNA hairpin pair to the NF-kB DNA decoy was catalyzed by miR-21. Intracellular CHA progression and the inhibitory effect against NF-kB were observed only in miR-21 overexpressing cancer cells. The intracellular miR-21-catalyzed production of the NF-kB DNA decoy has the potential to reduce side effects on normal cells, thereby strengthening the therapeutic profile of the CHA-decoy system. This study is the first to use the CHA product itself as a selective therapeutic substance. The ability to customize the combination of catalytic miRNA and target transcription factors would allow our technology to serve as a "personalized drug discovery system" for a variety of challenging diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a859188c2bcbbbd269e7e181ed7e2f2850df16d" target='_blank'>
              Amplified Production of a DNA Decoy Catalyzed by Intracellular MicroRNA.
              </a>
            </td>
          <td>
            Soshu Yasuda, K. Morihiro, Shuichiro Koga, Akimitsu Okamoto
          </td>
          <td>2025-02-03</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50e1ae97392a47d5cd3e9a60e099a7e7efba1c9" target='_blank'>
              Massively parallel characterization of transcriptional regulatory elements.
              </a>
            </td>
          <td>
            Vikram Agarwal, Fumitaka Inoue, Max Schubach, Dmitry Penzar, Beth K. Martin, P. Dash, Pia Keukeleire, Zicong Zhang, A. Sohota, Jingjing Zhao, Ilias Georgakopoulos-Soares, W. S. Noble, G. Yardımcı, I. Kulakovskiy, Martin Kircher, Jay Shendure, N. Ahituv
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2824e967c6a4d06b3abc17af2f0703099dbde0" target='_blank'>
              Uncovering the whole genome silencers of human cells via Ss-STARR-seq
              </a>
            </td>
          <td>
            Xiusheng Zhu, Lei Huang, Chao Wang, Guoli Li, Biao Deng, Dashuai Kong, Xiaoxiao Wang, Rongrong Chang, Yi Gu, Qiuhan Wen, Siyuan Kong, Yuwen Liu, Yubo Zhang
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA damage is a common event in cells, resulting from both internal and external factors. The maintenance of genomic integrity is vital for cellular function and physiological processes. The inadequate repair of DNA damage results in the genomic instability, which has been associated with the development and progression of various human diseases. Accumulation of DNA damage can lead to multiple diseases, such as neurodegenerative disorders, cancers, immune deficiencies, infertility and aging. This comprehensive review delves the impact of alterations in DNA damage response genes (DDR) and tries to elucidate how and to what extent the same traits modulate diverse major human diseases, such as cancer, neurodegenerative diseases, and immunological disorders. DDR is apparently the trait connecting important complex disorders in humans. However, the pathogenesis of the above disorders and diseases are different and leading to the divergent consequences. It is important to discover the switch(es) that direct further the pathogenic process either to proliferative, or degenerative diseases. Our understanding the influence of DNA damage on diverse human disorders may enable a development of the strategies to prevent, diagnose, and treat these diseases. In our article, we analysed publicly available GWAS summary statistics from the NHGRI-EBI GWAS Catalog and identified 12,009 single nucleotide polymorphisms (SNPs) associated with cancer. Among these, 119 SNPs were found in DDR pathways, exhibiting significant p-values. Additionally, we identified 44 SNPs linked to various cancer types and neurodegenerative diseases (NDDs), including four located in DDR-related genes: ATM, CUX2, and WNT3. Furthermore, 402 SNPs were associated with both cancer and immunological disorders, with two found in DDR gene RAD51B. This highlights the versatility of the DDR pathway in multifactorial diseases. However, the specific mechanisms that regulate DDR to initiate distinct pathogenic processes remain to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Nonsense-mediated mRNA decay (NMD) eliminates transcripts with premature termination codons, often resulting from mutations or RNA processing errors. While both pro-and anti-tumor NMD activities have been proposed, mechanistic and therapeutic insights into NMD inhibition in cancer have been hindered by the lack of a suitable in vivo model. We address this by combining a liver-specific, conditional Smg6 mutation – disabling the endonuclease that cleaves NMD-regulated transcripts – with a genetic hepatocellular carcinoma (HCC) model. SMG6 inhibition elevates NMD-sensitive mRNA abundance and translation, promotes MHC presentation of unique immunopeptides, and enhances T-cell infiltration. Additionally, mutant SMG6 specifically activates type-I interferon signaling via double-stranded RNA (dsRNA) accumulation and dsRNA-sensor MDA5, boosting innate immune responses. These pathways converge to halt HCC at an inflammatory stage. Our findings establish SMG6 as a key regulator of dsRNA homeostasis and immune activation, offering therapeutic potential and advancing our understanding of NMD in immune regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d439fae46ed61e162c6eb73643a9e6a014615c1" target='_blank'>
              Double-stranded RNA responses, neoantigen presentation and suppression of hepatocellular carcinoma through NMD inhibition via endonuclease SMG6
              </a>
            </td>
          <td>
            Enes S. Arpa, Virginie Ricci, I. E. Shapiro, Marija Jokic, I. González-Menéndez, Leticia Quintanilla-Martinez, Alexandre P. Bénéchet, J. Rehwinkel, S. Luther, M. Bassani-Sternberg, D. Gatfield
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Mosaic chromosomal alterations of the autosomes (aut-mCAs) are large structural somatic mutations which cause clonal hematopoiesis and increase cancer risk. Here, we detected aut-mCAs in 1,011,269 participants across four biobanks. Through integrative analysis of the minimum critical region and inherited genetic variation, we found that proto-oncogenes exclusively drive chromosomal gains, tumor suppressors drive losses, and copy-neutral events can be driven by either. We identified three novel inherited risk loci in CHI3L2, HLA class II, and TERT that modulate aut-mCA risk and ten novel aut-mCA-specific loci. We found specific aut-mCAs are associated with cardiovascular, cerebrovascular, or kidney disease incidence. High-risk aut-mCAs were associated with elevated plasma protein levels of therapeutically actionable targets: NPM1, PARP1, and TACI. Participants with multiple high-risk features such as high clonal fraction, more than one aut-mCA, and abnormal red cell morphology had a 50% cumulative incidence of blood count abnormalities over 2 years. Leveraging inherited variation, we causally established aut-mCAs as premalignant lesions for chronic lymphocytic leukemia. Together, our findings provide a framework integrating somatic mosaicism, germline genetics, and clinical phenotypes to identify individuals who could benefit from preventative interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f202aec9ed88d0660af9bd306eaa9b009384d1c" target='_blank'>
              Genetic drivers and clinical consequences of mosaic chromosomal alterations in 1 million individuals
              </a>
            </td>
          <td>
            , Y. Pershad, H. Poisner, X. Ma, K. Quade, C. Vlasschaert, T. Mack, N. K. Khankari, K. von Beck, J. Brogan, A. Kishtagari, R. Corty, Y. Li, Y. Xu, A. Reiner, P. Scheet, P. Auer, A. G. Bick
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK) fusion tyrosine kinases (TKs) are commonly found in various cancers and are considered as promising targets for therapy due to their intricate biological processes. However, the reasons for the higher frequency of ALKs fusion compared to other TKs are not well elucidated. Physicochemical properties, secondary and tertiary structures, and phylogenetic trees, along with protein sequence alignments of receptor tyrosine kinases (RTKs) and ALK fused partner genes, were examined using the resources provided by the National Center for Biotechnology Information (NCBI) and the Catalogue of Somatic Mutations in Cancer (COSMIC). Sequence alignments were performed to identify common sequences between partner genes and search for common breakpoints within the COSMIC database. ALK is a large, unstable, acidic protein with similarly conservation among RTKs. ALK fusion partners are mostly acidic, unstable proteins, mostly consisting of α-helices and random coil. However, EML4 and NPM1 are the most frequently occurring partner genes and have their own unique structural characteristics. By functional domain analysis, we found that the functions of the first half of the ALK partner gene (the part fused to ALK) are mostly focused on signaling. ALK is identified as a large hydrophilic protein，exhibits a higher proportion of random coils. Compared to other RTKs, ALK has fewer structural domains (PTKC_ALK_LTK domain). Pairwise comparison with fusion partner genes revealed a conserved sequence predicted to have structural stability and act as a common binding site for nucleases. Exon 20 of ALK is a fusion frequent site according to COSMIC database analysis. The structural instability of ALK and partner genes, coupled with the inherent variability of breakpoint sequences, leads to the formation of potent kinase-activated oncogenes, which play a critical role in tumorigenesis. While the occurrence of ALK fusions with partner genes is random, specific combinations lead to the generation of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d901b9b33dfa1544156c22b0678e85042f152af" target='_blank'>
              Genomic landscaping of receptor tyrosine kinase ALK with highly frequent rearrangements in cancers.
              </a>
            </td>
          <td>
            Wei Ye, Wen-Bin Ou
          </td>
          <td>2025-02-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Base of tongue squamous cell carcinoma (BOTSCC) is a prevalent and aggressive form of oral cancer, often associated with poor patient outcomes. The tumor microenvironment (TME) of HPV-positive BOTSCC is critical in influencing cancer progression and treatment response. Objective This study aims to analyze the TME of HPV-positive BOTSCC by examining the expression of key genes involved in various biological processes. Methods We utilized the RT2 Profiler PCR Array to quantify the expression of 168 genes related to inflammation, immunity, oncogenesis, tumor suppression, apoptosis, and angiogenesis. Enrichment analysis of cancer hallmarks was performed on all upregulated genes. Additionally, we investigated the correlation between the expression levels of the ten most highly upregulated genes and survival prognosis in HPV-associated BOTSCC patients. Results Our analysis revealed dysregulation of 42 genes associated with tumor-immune interactions, with 20 genes upregulated and 22 downregulated. Furthermore, we identified 64 genes linked to cancer development, with 33 upregulated and 31 downregulated. High-risk HPV (hr-HPV) genotypes were found in 81% of patients, predominantly HPV-35 and HPV-16. Conclusion This study highlights the complexity of the HPV-positive BOTSCC TME, underscoring the need for further research into molecular pathways and immune interactions to identify new therapeutic targets for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc25ed1594bab521984d64540a18029b5d937fb" target='_blank'>
              Profiling the Tumor Immune Microenvironment of HPV-Associated Base of Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Reham M Alahmadi, M. Awadalla, N. Marraiki, Mohammed Alswayyed, Hajar Alshehri, Amjad Alsahli, H. Khoja, Osamah Khojah, Rawan M Alahmadi, Nada Farid, B. Alosaimi
          </td>
          <td>2025-02-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Mitosis maintains a genome's genetic information in daughter cells by accurately segregating chromosomes. However, chromosome aberrations are common during early mammalian embryogenesis. Chromosomal abnormalities during the early stages of embryogenesis result in the formation of mosaic embryos, wherein cells with normal genomes coexist with cells exhibiting abnormal genomes. The precise frequency and etiology of such abnormalities remain unclear. It is postulated that these aberrations contribute to the etiology of a number of conditions, including infertility and congenital diseases such as Down's syndrome. Methods This review synthesizes current literature and data to elucidate the causes and implications of chromosome aberrations in early mammalian embryos. It places particular emphasis on identifying patterns of mosaicism and investigating the underlying mechanisms responsible for these abnormalities. Main Findings The underlying causes of chromosome abnormalities in early embryos were examined in the context of DNA replication and embryonic development. Conclusion A deeper understanding of chromosome abnormalities in early embryos could help develop new infertility treatments and advance research on cancers caused by these abnormalities. This article reviews current knowledge and gaps in understanding chromosome segregation abnormalities during embryogenesis and future directions in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0411fdbb93ab5a617be60af6f02c7f92304ed6b" target='_blank'>
              Chromosome segregation errors during early embryonic development
              </a>
            </td>
          <td>
            Hirohisa Kyogoku
          </td>
          <td>2025-01-01</td>
          <td>Reproductive Medicine and Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="p27Kip1 is a key cell cycle gatekeeper governing the timing of Cyclin-dependent kinase (CDK) activation/inactivation and, consequently, cell proliferation. Structurally, the protein is largely unfolded, a feature that strongly increases its plasticity and interactors and enhances the number of regulated cellular processes. p27Kip1, like other intrinsically unstructured proteins, is post-translationally modified on several residues. These modifications affect its cellular localization and address p27Kip1 for specific interactions/functions. Several germline or somatic CDKN1B (the p27Kip1 encoding gene) mutations have been demonstrated to be associated with multiple endocrine neoplasia type 4 (MEN4), hairy cell leukemia, small-intestine neuroendocrine tumors, and breast and prostate cancers. Here, we analyzed the effect of four CDKN1B missense and nonsense mutations found in patients affected by MEN4 or cancers, namely, c.349C>T, p.P117S; c.397C>A, p.P133T; c.487C>T, p.Q163*; and c.511G>T, p.E171*. By transfecting breast cancer cell lines, we observed increased growth and cell motility for all the investigated mutants compared to wild-type p27Kip1 transfected cells. Furthermore, we discovered that the mutant forms exhibited altered phosphorylation on key residues and different localization or degradation mechanisms in comparison to the wild-type protein and suggested a possible region as crucial for the lysosome-dependent degradation of the protein. Finally, the loss of p27Kip1 ability in blocking cell proliferation was in part explained through the different binding efficiency that mutant p27Kip1 forms exhibited with Cyclin/Cyclin-dependent Kinase complexes (or proteins involved indirectly in that binding) with respect to the WT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b858b52e6f58495fd391a4dfcc6f1900f00849e" target='_blank'>
              p27Kip1 and Tumors: Characterization of CDKN1B Variants Identified in MEN4 and Breast Cancer
              </a>
            </td>
          <td>
            Debora Bencivenga, E. Stampone, Jahanzaib Azhar, Daniela Parente, Waqar Ali, Vitale Del Vecchio, Fulvio Della Ragione, A. Borriello
          </td>
          <td>2025-01-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are unique nucleic acid structures composed of guanine-rich (G-rich) sequences that can form diverse topologies based on the arrangement of their four strands. G4s have attracted attention for their potential roles in various biological processes and human diseases. In this review, we focus on the G4 structures formed by human telomeric sequences, (GGGTTA)n, and the hexanucleotide repeat expansion, (GGGGCC)n, in the first intron region of the chromosome 9 open reading frame 72 (C9orf72) gene, highlighting their structural diversity and biological significance. Human telomeric G4s play crucial roles in telomere retention and gene regulation. In particular, we provide an in-depth summary of known telomeric G4s and focus on our recently discovered chair-type conformation, which exhibits distinct folding patterns. The chair-type G4s represent a novel folding pattern with unique characteristics, expanding our knowledge of telomeric G4 structural diversity and potential biological functions. Specifically, we emphasize the G4s formed by the (GGGGCC)n sequence of the C9orf72 gene, which represents the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). The thorough structural analysis in this review advances our comprehension of the disease mechanism and provides valuable insights into developing targeted therapeutic strategies in ALS/FTD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4afb7326b8980cb609e9c6033f3c9e1b8d005a43" target='_blank'>
              G-Quadruplex Structures Formed by Human Telomere and C9orf72 GGGGCC Repeats
              </a>
            </td>
          <td>
            Bing Yan, M. C. Suen, Naining Xu, Chao Lu, Changdong Liu, Guang Zhu
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="During zygotic genome activation (ZGA) and early development, hierarchical levels of chromatin structure undergo remarkable perturbations: changes in the nuclear-to-cytoplasmic ratio of various components; changes in chromatin accessibility; histone exchange; and the formation of 3D structures such as loops, topologically associated domains, and compartments. Here, we review the peculiarities, variability, and emergence of the chromatin structural features during ZGA in different organisms. Focusing on newly found structures called fountains, we describe the prerequisites for cohesin loading on DNA and possible mechanisms of genome organization in early development. Fountains resulting from asymmetric bidirectional cohesin extrusion spread from cohesin-loading points in a CTCF-independent manner. We discuss that fountains may not possess specific functions, unlike conventional chromatin structures, and could be found in other biological processes where cohesin loading occurs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35698f05d68e24632b02180e50c1bfb7426f2b2" target='_blank'>
              The establishment of the 3D genome structure during zygotic genome activation.
              </a>
            </td>
          <td>
            Anastasiia L Sivkina, O. Iarovaia, S. V. Razin, S. Ulianov
          </td>
          <td>2025-03-03</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Adaptation to challenging environmental conditions is crucial for the survival/fitness of all organisms. Alongside genetic mutations that provide adaptive potential during environmental challenges, epigenetic modifications offer dynamic, reversible, and rapid mechanisms for regulating gene expression in response to environmental changes in both evolution and daily life, without altering DNA sequences or relying on accidental favorable mutations. The widespread conservation of diverse epigenetic mechanisms - like DNA methylation, histone modifications, and RNA interference across diverse species, including plants - underscores their significance in evolutionary biology. Remarkably, environmentally induced epigenetic alterations are passed to daughter cells and inherited transgenerationally through germline cells, shaping offspring phenotypes while preserving adaptive epigenetic memory. Throughout anthropoid evolution, epigenetic modifications have played crucial roles in: i) suppressing transposable elements and viral genomes intruding into the host genome; ii) inactivating one of the X chromosomes in female cells to balance gene dosage; iii) genetic imprinting to ensure expression from one parental allele; iv) regulating functional alleles to compensate for dysfunctional ones; and v) modulating the epigenome and transcriptome in response to influence from the gut microbiome among other functions. Understanding the interplay between environmental factors and epigenetic processes may provide valuable insights into developmental plasticity, evolutionary dynamics, and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7f5656f1a791fd7b047008e717c5695b95280c" target='_blank'>
              Epigenetics in evolution and adaptation to environmental challenges: pathways for disease prevention and treatment.
              </a>
            </td>
          <td>
            H. Abdolmaleky, S. Nohesara, Jin-Rong Zhou, S. Thiagalingam
          </td>
          <td>2025-02-13</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The DNA double-stranded genome of human adenoviruses (HAdV) is preferentially targeted by host factors involved in chromatin remodeling, such as SPOC1 and KAP1 to inhibit efficient viral gene expression. HAdV genomes undergo alterations through association with host histones and epigenetic modifications; however, the precise underlying mechanism remains elusive. A recently discovered silencing mechanism for retrotransposons and retroviruses involves the Human Silencing Hub (HUSH) complex. This complex of MPP8, TASOR, and PPHLN1 safeguards the human genome by utilizing histone H3 Lys9 trimethylation (H3K9me3) to block transcription. Through the recruitment of SETDB1 and MORC2, the HUSH complex silences host genes and condenses target genomes to combat infections such as HIV, MLV, and AAV. Here, we present evidence that the HUSH complex effectively restricts HAdV infection. To counteract the repressive function of this epigenetic silencing complex, HUSH factors are inhibited through binding of HAdV proteins and subsequent relocalization. We observe that MPP8 is targeted by the adenoviral E3 ubiquitin ligase, thus recruited by the viral early proteins E1B-55K and E4orf6 for proteasomal degradation. In summary, we provide evidence that the HUSH complex is a previously unrecognized host factor that restricts HAdV gene expression and replication. Based on these novel findings, we propose that HUSH represents a promising therapeutic target to combat HAdV infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26334a481a2a25fb7be72ec2c09273c472c1ce03" target='_blank'>
              Intrinsic immunity against HAdV is achieved by a novel epigenetic silencing complex
              </a>
            </td>
          <td>
            Julia Mai, Maryam Karimi, A. Kuderna, Anna Heider, Thomas Günther, C. Friedel, A. Grundhoff, Thomas Stamminger, Sabrina Schreiner
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 In colon cancer, the BRAFV600E mutation is strongly associated with the CpG island methylator phenotype (CIMP). Here, we characterized the contribution of BRAFV600E to maintenance of aberrant DNA methylation. A reverse CRISPR gene editing approach was applied to revert the V600E mutation in BRAF back to wildtype (E600V) in organoids derived from a late-stage tumour. DNA methylation analyses identified 5187 differentially methylated CpGs within CpG islands, predominantly hypermethylated (82%) in BRAFV600E organoids, including genes associated with CIMP, as well as Polycomb Repressor Complex 2 (PRC2) target genes. RNA sequencing showed that expression of those genes was concordantly repressed. Furthermore, BRAFV600Einduced high expression of PRC2 core components (EZH2, SUZ12, EED), caused PRC2-induced H3K27 trimethylation in promoter regions, and maintained a PRC2-associated embryonic phenotype. This phenotype was lost following mutation correction or pharmacological inhibition of DNA methylation. These findings show that BRAFV600E maintains aberrant DNA and histone methylation patterns in advanced colon cancer, likely preserving the transformed phenotype by silencing of PRC2 target genes. Epigenetic therapies may have value in the treatment of BRAFV600E-mutant colon cancer.
 Citation Format: Layla El Bouazzaoui, Jeroen M Bugter, Emre Küçükköse, André Verheem, Jasmin B Post, Nicola Fenderico, Inne H.M Borel Rinkes, Hugo J.G Snippert, Madelon M Maurice, Onno Kranenburg. BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aeb4732d6e0e4b7c91f83caa7be8cc3275c4570" target='_blank'>
              Abstract B003: BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer
              </a>
            </td>
          <td>
            Layla El Bouazzaoui, J. Bugter, Emre Küçükköse, A. Verheem, J. Post, N. Fenderico, Inne H. M. Borel Rinkes, Hugo J. Snippert, M. Maurice, O. Kranenburg
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
α thalassemia/mental retardation syndrome X-linked (ATRX) serves as a part of the sucrose nonfermenting 2 (SNF2) chromatin-remodeling complex. In interphase, ATRX localizes to pericentromeric heterochromatin, contributing to DNA double-strand break repair, DNA replication, and telomere maintenance. During mitosis, most ATRX proteins are removed from chromosomal arms, leaving a pool near the centromere region in mammalian cells, which is critical for accurate chromosome congression and sister chromatid cohesion protection. However, the function and localization mechanisms of ATRX at mitotic centromeres remain largely unresolved.


METHODS
The clustered regularly interspaced short palindromic repeats with CRISPR-associated protein 9 (CRISPR-Cas9) system and overexpression approaches were employed alongside immunofluorescence to investigate the mechanism of ATRX localization at the centromere. To study the binding mechanism between ATRX and heterochromatin protein 1 (HP1), both full-length and truncated mutants of hemagglutinin (HA)-ATRX were generated for co-immunoprecipitation and glutathione S-transferase (GST)-pull assays. Wild-type ATRX and HP1 binding-deficient mutants were created to investigate the role of ATRX binding to HP1 during mitosis, with the Z-Leu-Leu-Leu-al (MG132) maintenance assay, cohesion function assay, and kinetochore distance measurement.


RESULTS AND CONCLUSIONS
Our research demonstrated that HP1α, HP1β, and HP1γ facilitate the positioning of ATRX within the mitotic centromere area through their interaction with the first two [P/L]-X-V-X-[M/L/V] (PxVxL)motifs at the N-terminus of ATRX. ATRX deficiency causes aberrant mitosis and decreased centromeric cohesion. Furthermore, reducing Wapl activity can bypass the need for ATRX to protect centromeric cohesion. These results provide insights into the mechanism of ATRX's centromeric localization and its critical function in preserving centromeric cohesion by reducing Wapl activity in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2508ee19573d906286f1694bdd429742f1bd10" target='_blank'>
              HP1 Promotes the Centromeric Localization of ATRX and Protects Cohesion by Interfering Wapl Activity in Mitosis.
              </a>
            </td>
          <td>
            Erchen Zhang, Lei Peng, Kejia Yuan, Zexian Ding, Qi Yi
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While the mutational landscape across early T-cell precursor acute lymphoblastic leukemia (ETP-ALL) and ETP-like leukemia is known, establishing a unified framework that activates stem cell genes characteristic of these tumors remains elusive. Using complementary mouse and human models, chromatin mapping, and enhancer profiling, we show that the coactivator ZMIZ1 promotes normal and malignant ETP population growth by inducing the transcription factor MYB in feedforward circuits to convergently activate oncogenes (MEF2C, MYCN, and BCL2) through essential enhancers. A key superenhancer, the N-Myc regulating enhancer (NMRE), drives malignant ETP population growth but is dispensable for normal lymphopoiesis. This network of stem cell superenhancers identifies treatment-resistant tumors and poor survival outcomes; unifies diverse ETP-ALLs; and contributes to cardinal features of the recently genomically identified high-risk bone marrow progenitor-like (BMP-like) ETP-ALL tumor-stem cell/myeloid gene expression, inhibited NOTCH1-induced T-cell development, aggressive clinical behavior, and venetoclax sensitivity. Since ZMIZ1 is dispensable for essential homeostasis, it might be possible to safely target this network to treat high-risk diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779b27de9c7947d2736cb472ee433351114fe43" target='_blank'>
              Native stem cell transcriptional circuits define cardinal features of high-risk leukemia.
              </a>
            </td>
          <td>
            Qing Wang, Francesco Boccalatte, Jason Xu, Giovanni Gambi, Bettina Nadorp, Fatema Akter, Carea Mullin, A. Melnick, Elizabeth Choe, Anna C. McCarter, Nicole A Jerome, Siyi Chen, Karena Lin, Sarah Khan, R. Kodgule, Jonathan H. Sussman, P. Pölönen, Javier Rodriguez-Hernaez, Sonali Narang, Kleopatra Avrampou, B. King, A. Tsirigos, R. Ryan, C. Mullighan, D. Teachey, Kai Tan, I. Aifantis, M. Chiang
          </td>
          <td>2025-02-19</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e899cf4a3d1394fc8fe742ffd50bbb8e81d067" target='_blank'>
              Extensive mutual influences of SMC complexes shape 3D genome folding.
              </a>
            </td>
          <td>
            Han Zhao, Lirong Shu, Shiyi Qin, Fangxuan Lyu, Fuhai Liu, En Lin, Sijian Xia, Baiyue Wang, Manzhu Wang, Fengnian Shan, Yinzhi Lin, Lin Zhang, Yufei Gu, Gerd A. Blobel, Kai Huang, Haoyue Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692b233441e797a73e14defb069a03a55d2ca324" target='_blank'>
              DNA methylation profiling at base-pair resolution reveals unique epigenetic features of early-onset colorectal cancer in underrepresented populations
              </a>
            </td>
          <td>
            J. S. Li, Karen Riggins, Li Yang, Chaorong Chen, Patricia Castro, Wedad Alfarkh, Neda Zarrin-Khameh, Michael E. Scheurer, Chad J. Creighton, Benjamin Musher, Wei Li, Lanlan Shen
          </td>
          <td>2025-01-22</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f02778fd06b36e0f2376644c23b8af48578fee" target='_blank'>
              Purine-rich element binding protein alpha: a DNA/RNA binding protein with multiple roles in cancers
              </a>
            </td>
          <td>
            Shiyi Yu, Chengyang Jiang, Yawen Yang, Fei Cheng, Fangchen Liu, Chang Liu, Xue Gong
          </td>
          <td>2025-01-22</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BRCA1 deficiency is observed in approximately 25% of triple-negative breast cancer (TNBC). BRCA1, a key player of homologous recombination (HR) repair, is also involved in stalled DNA replication fork protection and repair. Here, we investigated the sensitivity of BRCA1-deficient TNBC models to the frequently used replication chain terminator gemcitabine, which does not directly induce DNA breaks. A large fraction of BRCA1-deficient cells was sensitive to gemcitabine, in contrast to their isogenic BRCA1-proficient counterparts. Gemcitabine treated BRCA1-deficient cells accumulated massive levels of single strand DNA (ssDNA) and presented no RPA or RAD51 nuclear foci. The gemcitabine-induced accumulation of ssDNA in BRCA1-deficient cells was strongly diminished by targeting MRE11 with inhibitors and by siRNA attenuation. In contrast, treatment with the PARP1/2 inhibitor olaparib did not result in MRE11 dependent over-resection. Furthermore, a fraction of gemcitabine treated BRCA1-deficient cells that showed massive ssDNA accumulation slipped into mitosis, producing mitotic bridges and strongly stained BrdU and γH2AX micronuclei (MN). The BrdU-positive MN and DNA bridges also stained positively for cGAS. In conclusion, these data suggest that gemcitabine treatment in BRCA1-deficient TNBC exposes unprotected nascent DNA linked to replication fork reversal, which leads to MRE11 over-resection and ssDNA accumulation. Therefore, the observed hypersensitivity to gemcitabine indicates that it could be a beneficial addition to BRCA1-deficient TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d16464aa7109edc218441355d65fc601eddce9b" target='_blank'>
              Replication-Poison Treatment in BRCA1-Deficient Breast Cancer Causes MRE11 Over-Resection that Induces Single-Stranded DNA Accumulation and Mitotic Catastrophe.
              </a>
            </td>
          <td>
            I. Tabet, Esin Orhan, Ermes Candiello, Lise Fenou, C. Velázquez, B. Orsetti, Geneviéve Rodier, William Jacot, C. Ribeyre, Claude Sardet, C. Theillet
          </td>
          <td>2025-01-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dc1a99977067c119e44f09575781707edcc62" target='_blank'>
              Interplay of aurora kinase a functional residues and Epstein-barr Nuclear Antigen 1 in Epstein-barr virus associated Gastric cancer using AGS cells
              </a>
            </td>
          <td>
            N. Varshney, Siddharth Singh, M. Kandpal, Vaishali Saini, Erle S. Roberston, Hem Chandra Jha
          </td>
          <td>2025-01-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="FOXP3+ natural regulatory T cells (nTregs) promote resolution of inflammation and repair of epithelial damage following viral pneumonia-induced lung injury, thus representing a cellular therapy for patients with acute respiratory distress syndrome (ARDS). Whether in vitro induced Tregs (iTregs), which can be rapidly generated in substantial numbers from conventional T cells, also promote lung recovery is unknown. nTregs require specific DNA methylation patterns maintained by the epigenetic regulator, ubiquitin-like with PHD and RING finger domains 1 (UHRF1). Here, we tested whether iTregs promote recovery following viral pneumonia and whether iTregs require UHRF1 for their pro-recovery function. We found that adoptive transfer of iTregs to mice with influenza virus pneumonia promotes lung recovery and that loss of UHRF1-mediated maintenance DNA methylation in iTregs leads to reduced engraftment and a delayed repair response. Transcriptional and DNA methylation profiling of adoptively transferred UHRF1-deficient iTregs that had trafficked to influenza-injured lungs demonstrated transcriptional instability with gain of effector T cell lineage-defining transcription factors. Strategies to promote the stability of iTregs could be leveraged to further augment their pro-recovery function during viral pneumonia and other causes of ARDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8259d3d12170176131ead01b0d7705b88e45403" target='_blank'>
              Maintenance DNA methylation is required for induced regulatory T cell reparative function following viral pneumonia
              </a>
            </td>
          <td>
            A. Joudi, J. Gurkan, Qianli Liu, Manuel A. Torres Acosta, K. Helmin, Luisa Morales-Nebreda, N. Mambetsariev, Carla P. Reyes Flores, H. Abdala-Valencia, Elizabeth M. Steinert, S. Weinberg, Benjamin D. Singer
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA methylation is a crucial epigenetic modification that orchestrates chromatin remodelers that suppress transcription, and aberrations in DNA methylation result in a variety of conditions such as cancers and developmental disorders. While it is understood that methylation occurs at CpG-rich DNA regions, it is less understood how distinct methylation profiles are established within various cell types. In this work, we develop a molecular-transport model that depicts the genomic exploration of DNA methyltransferase within a multiscale DNA environment, incorporating biologically relevant factors like methylation rate and CpG density to predict how patterns are established. Our model predicts DNA methylation-state correlation distributions arising from the transport and kinetic properties that are crucial for the establishment of unique methylation profiles. We model the methylation correlation distributions of nine cancerous human cell types to determine how these properties affect the epigenetic profile. Our theory is capable of recapitulating experimental methylation patterns, suggesting the importance of DNA methyltransferase transport in epigenetic regulation. Through this work, we propose a mechanistic description for the establishment of methylation profiles, capturing the key behavioral characteristics of methyltransferase that lead to aberrant methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e458fbb01165e95120272a65d7850681688311a0" target='_blank'>
              Modeling DNA methyltransferase function to predict epigenetic correlation patterns in healthy and cancer cells.
              </a>
            </td>
          <td>
            Ariana Y Tse, A. Spakowitz
          </td>
          <td>2025-01-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) refer to RNA molecules that exceed 200 nucleotides in length. While lncRNAs do not possess the capacity to encode proteins, they play crucial roles in gene expression, chromatin remodeling, and protein relocation. PSMA3 antisense RNA 1 (PSMA3-AS1) is a newly discovered lncRNA located on human chromosome 14q23.1. Convincing evidence shows that it acts as a tumor-promoting factor in several forms of human cancers. Moreover, high expression of PSMA3-AS1 is linked to poor clinical and pathological features and adverse prognosis in eight types of cancer. The molecular mechanisms of PSMA3- AS1 are diverse and complex. Existing evidence demonstrates that PSMA3-AS1 is activated by two transcription factors, PAX5 and YY-1, and influences cancer cell growth, metastasis, apoptosis, drug resistance, oxidative stress, and autophagy by acting as a competing endogenous RNA, activating signaling pathways, directly interacting with RNA or proteins, as well as participating in the epithelial-mesenchymal transition process. Therefore, PSMA3-AS1 holds promise as a biomarker for cancer detection and prediction, as well as a novel therapeutic target. This review explores the expression features, biological roles, potential mechanisms, and clinical significance of PSMA3-AS1 in various human cancers and provides directions for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5575613f9e7c6e034f80189343ae1eb7afbdba" target='_blank'>
              Revisiting the Role of Long Non-coding RNA PSMA3-AS1 in Human Cancers: Current Evidence and Future Directions.
              </a>
            </td>
          <td>
            Jingjie Yang, Kexing Liu, Lihan Chen, Haodong He, Tongtong Li, Li Li, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2025-01-31</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose The mechanisms that lead to pterygium pathogenesis are poorly understood. Ultraviolet (UV) exposure is a leading environmental risk factor. We propose that UV triggers the de-repression of transposable elements (TEs), including endogenous retroviruses (ERVs), and subsequently activate double-stranded RNA and DNA sensors such as RIG-I and cGAS. Methods In the present study, we analyzed publicly available RNA sequencing data from pterygium and healthy conjunctiva specimens. We used Ingenuity Pathway Analysis (IPA) to examine the relative expression of nucleic acid sensing pathways and related processes. We also used Telescope to identify TEs. We then repeated this analysis using RNA collected from cultured human conjunctival fibroblasts exposed over multiple days to a range of UV intensities. Results We found that pathways involved in cellular stress, nucleic acid sensing, and innate immune signaling were upregulated in both cell lines exposed to increased amounts of UV energy as well as pterygium. We also observed differential expression of TEs. Conclusions Taken together, our findings suggest a mechanistic link between environmental factors and pterygium. Understanding the molecular pathways activated by UV exposure may aid in developing therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8234f024c61b7af4ccdfb30cc2f8dd5336b81ea" target='_blank'>
              Differential expression of transposable elements and upregulation of nucleic acid sensing pathways in pterygium
              </a>
            </td>
          <td>
            Patrick C. Demkowicz, Emma Hammes, Despoina Theotoka, Jessica Chow, Mathieu F. Bakhoum
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 DNA methylation (DNAm) reflects regions of the genome that are transcriptionally repressed or active depending on the methylation level. Multitudes of DNAm “clocks” have been created to predict biological age and health outcomes. Deviations in biological age from chronological age reflect an alteration in the normal biological aging process. Clonal hematopoiesis of indeterminate potential (CHIP) arises in hematopoietic stem cells that have accumulated somatic mutations in key driver genes which provide a selective advantage for clonal expansion. Subjects with CHIP mutations are more prone to negative clinical outcomes such as coronary heart disease. While DNAm and CHIP have been studied for the past decade, the associations between the two are only now starting to become apparent. In this study we propose an improved measurement of aging based on the Jenson-Shannon distance (JSD) and associate this measurement with CHIP and adverse clinical outcomes. Bisulfite amplicon sequencing of 20 CpG-rich targets that change in methylation with age was performed on an initial cohort of 155 individuals from the National Institute of Neurological Disorders and Stroke (NINDS). JSD was used to construct a DNA methylation chaos clock (DMC) on these targets. DMC provided better predictions of chronological age than DNAm alone (r=0.9; p<0.0001). We validated our findings in 286 individuals from Cooper University Hospital for Management of Trauma where the accuracy of the DMC was confirmed in correlations of predicted vs actual age (r=0.86; p<0.0001). We next examined the association between DMC age acceleration and clinical outcomes. It was found that in individuals over the age of 50, DMC acceleration was associated with smoking and disease incidence (e.g. congestive heart failure and chronic obstructive pulmonary disease) independent of chronological age. The incidence of CHIP mutations in three key driver genes (DNMT3A, TET2, and ASXL1) was assayed in a subset of 345 individuals from both cohorts. Individuals with CHIP mutations in key driver genes had greater deviations in their predicted age than individuals without CHIP mutations (p<0.001). This study demonstrates that the DMC assay is a candidate for improvements in our measurements of biological age and it reflects important changes in an individual’s biological aging process.
 Citation Format: Anthony J Pompetti, Woonbok Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre Issa. DNA methylation variability in aging and Clonal Hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadd86e514d184ff56ec546d04b1399f1bbf484d" target='_blank'>
              Abstract A016: DNA methylation variability in aging and Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Anthony J Pompetti, W. Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers and thought to drive carcinogenesis by impairing POT1's suppression of aberrant telomere elongation. To classify clinical variants of uncertain significance (VUSs) and identify cancer-driving loss-of-function mutations, we developed a locally haploid human stem cell system to evaluate >1900 POT1 mutations, including >600 VUSs. Unexpectedly, many validated familial cancer-associated POT1 (caPOT1) mutations are haplosufficient for cellular viability, indicating that some pathogenic alleles do not act through a loss-of-function mechanism. Instead, POT1's DNA damage response suppression and telomere length control are genetically separable. ATR inhibition enables isolation of frameshift mutants, demonstrating that the only essential function of POT1 is to repress ATR. Furthermore, comparison of caPOT1 and frameshift alleles reveals a class of caPOT1 mutations that elongate telomeres more rapidly than full loss-of-function alleles. This telomere length-promoting activity is independent from POT1's role in overhang sequestration and fill-in synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aca8c4eb6696cc9c11ef54c9581b2900ee7f04" target='_blank'>
              Active telomere elongation by a subclass of cancer-associated POT1 mutations.
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The presence of specific genetic mutations in patients with glioblastoma multiforme (GBM) is associated with improved survival outcomes. Disruption of the DNA damage response (DDR) pathway in tumor cells enhances the effectiveness of radiotherapy drugs, while increased mutational burden following tumor cell damage also facilitates the efficacy of immunotherapy. The ATRX gene, located on chromosome X, plays a crucial role in DDR. The aim of this research is to elucidate the correlation between ATRX mutations and GBM. Dataset obtained from TCGA-GBM were conducted an analysis on the genomic features, biological characteristics, immunopathological markers, and clinical prognosis of patients carrying ATRX mutations. Our findings revealed a significantly elevated level of microsatellite instability in individuals with ATRX mutants, along with significant alterations in the receptor-tyrosine kinase (RTK)-ras pathway among patients exhibiting combined ATRX mutations. TCGA-GBM patients with concurrent ATRX mutations exhibited sensitivity to 26 chemotherapeutic and anticancer drugs, which exerted their effects by modulating the DDR of tumor cells through highly correlated mechanisms involving the RTK-ras pathway. Additionally, we observed an enrichment of ATRX mutations in specific pathways associated with DDR among TCGA-GBM patients. Our model also demonstrated prolonged overall survival in patients carrying ATRX mutations, particularly showing strong predictive value for 3- and 5-year survival rates. Furthermore, additional protective factors such as younger age, female gender, combined IDH mutations, and TP53 mutations were identified. The results underscore the protective role and prognostic significance of ATRX mutations in GBM as a potential therapeutic target and biomarker for patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89334fb95e9947311852d880d8c780526377835e" target='_blank'>
              ATRX mutation modifies the DNA damage response in glioblastoma multiforme tumor cells and enhances patient prognosis
              </a>
            </td>
          <td>
            Rou Yu, Keru Huang, Xinyan He, Jingwen Zhang, Yushan Ma, Hui Liu
          </td>
          <td>2025-01-10</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-coding RNA transcribed from active enhancers, known as enhancer RNA (eRNA), is a critical element in gene regulation with a highly specific expression pattern in the regulatory networks of tumor-infiltrating cells. Therefore, eRNA signatures could potentially be applied to represent anti-tumor immune cells and to improve cancer immunotherapy. Herein, we identified thousands of eRNAs that were significantly correlated with infiltrating immune cell abundance in more than 10,000 patient samples across a variety of cancer types. The expression of these eRNAs was mediated by transcription factors with high expression in anti-tumor immune cells, as identified through single-cell assays. An eRNA immunotherapy signature (eRIS) developed using the anti-tumor eRNAs was highly associated with the objective response rate (ORR) of immunotherapy and was elevated in patients who benefited from immune checkpoint blockade (ICB) treatment. In comparison with a signature based on protein-coding genes, the eRIS was more effective in predicting the response to immunotherapy. Integration of the eRIS with pharmacogenomic data revealed hundreds of anti-cancer drugs that have the potential to enhance immunotherapy efficacy. Finally, treatment of a mouse model of IDH mutant glioma with the histone deacetylase inhibitor vorinostat improved the effects of anti-PD-1 immunotherapy through increased abundance of infiltrating immune cells. Taken together, this study developed an eRIS with demonstrated efficacy in predicting immunotherapy response and used the eRIS to identify a series of effective combination drugs, thus highlighting the clinical utility of the eRIS in immunotherapy enhancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb404431d7b2aee513f3b1e9892ee993135259d" target='_blank'>
              Characterization of an Enhancer RNA Signature Reveals Treatment Strategies for Improving Immunotherapy Efficacy in Cancer.
              </a>
            </td>
          <td>
            Chenyang Zhang, Yan-Yan Chen, Shuyu Chen, Yunzhe Wang, Yifan Yuan, Xiwen Yang, Wei Hu, Bo Chen, Zengxin Qi, Jason T Huse, Yun Liu, Bo Wen, Xiuping Liu, Leng Han, Yuxiang Wang, Zhao Zhang
          </td>
          <td>2025-02-04</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f65c0d127245d474792c410ee7afae8aec904" target='_blank'>
              Complicated crosstalk between HMGA and non-coding RNAs modulates hallmarks of cancer
              </a>
            </td>
          <td>
            Lijie Zhang, Xiaomin Zhao, Xianghong Gao, Hao Qin, Feng Chen, Zhijuan Lin
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Adoptive immune cell therapies, such as CAR T, have revolutionized how we can approach cancer therapy, however challenges remain to increase their efficacy, response rates, and resistance to dysfunction. The systematic investigation of gene programs governing human T cell function would be advantageous for reprogramming cell products for clinical benefit. Recent studies have demonstrated the power of CRISPR-based forward genetic screens in primary human T cells to nominate candidate perturbations for enhanced T cell function, however combinations of perturbations will likely be necessary to overcome the most challenging tumors. To enable the systematic and high-throughput interrogation of combinations of gain and loss-of-function perturbations, we have developed a bidirectional CRISPR activation and interference (CRISPRai) system, where pairs of genes can be simultaneously activated and repressed in primary human T cells. Leveraging hits from previous genome-wide CRISPRa and CRISPRi screens, we constructed a library of >66,000 sgRNA combinations and screened for regulators of acute T cell stimulation responses, including cytokine production and proliferation. Reassuringly, these screens identified expected gene epistasis interactions (e.g. rescue of PLCG1 repression by PLCG2 activation), while also discovering novel genetic interactions, providing mechanistic insights into the regulatory networks governing stimulation response. To further identify gene “building blocks” for evaluation as combinations, we completed genome-wide CRISPRa and CRISPRi screens for regulators of human CD8+ CAR T cell exhaustion. These screens identified known potential enhancements (e.g. gain of BATF, loss of MED12), and also uncovered novel regulators of T cell function, with CRISPRa discovering genes not normally expressed in T cells but whose overexpression could cause resistance to exhaustion by synthetic mechanisms. Gene expression programs controlled by perturbation of key regulators were mapped by pairing CRISPR with single-cell RNA-seq (Perturb-seq) and arrayed functional assays. Finally, a top gain-of-function candidate was evaluated for in vivo tumor control, using an antigen-inducible Synthetic Notch system. Together, this work demonstrates how advanced forward genetics approaches can discover and elucidate both natural and synthetic programs that regulate T cell states, and the nomination of perturbations to encode in logic-gated synthetic biology designs for cell therapies.
 Citation Format: Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole Roybal, Brian Shy, Alexander Marson. Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Purpose
Molecular characteristics of synchronous colorectal cancers (SCRCs) remain incompletely elucidated, despite their importance in targeted therapy selection. We compared the molecular characteristics and somatic mutations between SCRCs.


Materials and Methods
This retrospective study (2012-2014) included 98 consecutive patients with surgically resected SCRCs. Molecular characteristics, including microsatellite instability (MSI) and tumor-infiltrating lymphocytes (TILs), were analyzed for all cancer lesions. The intertumoral heterogeneity of SCRCs was evaluated using whole-exome sequencing (WES) for 18 cancers from 9 patients with at least one MSI-high (MSI-H) tumor.


Results
Twelve patients had at least one MSI-H tumor; five showed discordant MSI status. Mucinous adenocarcinoma frequency and TIL density were higher in patients with at least one MSI-H tumor than in those with only microsatellite-stable tumors. WES revealed that, except one patient (6.5%), most synchronous cancers shared few variants in each patient (0.09-0.36%). The concordance rates for BRAF, KRAS, NRAS, and PIK3CA in synchronous cancers from each patient were 66.7%, 66.7%, 66.7%, and 55.6%, respectively.


Conclusion
Although synchronous cancers shared a mutated gene, the mutation subtypes differed. SCRCs exhibited 5.1% MSI status discordance rate and a high discordance rate in somatic mutational variants. As intertumoral heterogeneity may affect the targeted therapy response, molecular analysis of all tumors is recommended for patients with SCRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d64426c5ce00acb3fa51471d3649240227f5078" target='_blank'>
              Molecular Mosaics: Unveiling Heterogeneity in Synchronous Colorectal Cancers.
              </a>
            </td>
          <td>
            H. Lee, Yeseul Kim, Mi-Ju Kim, Yeon Wook Kim, Sun-Young Jun, Deokhoon Kim, In Ja Park, Seung-Mo Hong
          </td>
          <td>2025-02-18</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Synthetic lethality approaches in BRCA1/2-mutated cancers have focused on poly(ADP-ribose) polymerase (PARP) inhibitors, which are subject to high rates of innate or acquired resistance in patients. Here, we used CRISPR/Cas9-based screening to identify DNA Ligase I (LIG1) as a novel target for synthetic lethality in BRCA1-mutated cancers. Publicly available data supported LIG1 hyperdependence of BRCA1-mutant cells across a variety of breast and ovarian cancer cell lines. We used CRISPRn, CRISPRi, RNAi, and protein degradation to confirm the lethal effect of LIG1 inactivation at the DNA, RNA, and protein level in BRCA1-mutant cells in vitro. LIG1 inactivation resulted in viability loss across multiple BRCA1-mutated cell lines, whereas no effect was observed in BRCA1/2 wild-type cell lines, demonstrating target selectivity for the BRCA1-mutant context. On-target nature of the phenotype was demonstrated through rescue of viability with exogenous wild-type LIG1 cDNA. Next, we demonstrated a concentration-dependent relationship of LIG1 protein expression and BRCA1 mutant cell viability using a titratable, degradable LIG1 fusion protein. BRCA1 mutant viability required LIG1 catalytic activity, as catalytically dead mutant LIG1K568A failed to rescue viability loss caused by endogenous LIG1 depletion. LIG1 perturbation produced proportional increases in PAR staining in BRCA1 mutant cells, indicating a mechanism consistent with the function of LIG1 in sealing ssDNA nicks. Finally, we confirmed LIG1 hyperdependence in vivo using a xenograft model in which LIG1 loss resulted in tumor stasis in all mice. Our cumulative findings demonstrate that LIG1 is a promising synthetic lethal target for development in patients with BRCA1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831e285c6dc25f79bcb13593b00dacbe2a024fd9" target='_blank'>
              LIG1 is a synthetic lethal target in BRCA1 mutant cancers.
              </a>
            </td>
          <td>
            Lauren Martires, Leanne G Ahronian, Charlotte B Pratt, Nikitha M Das, Xiaobin Zhang, D. Whittington, Hongxiang Zhang, B. Shen, Jon Come, Patrick McCarren, Mu-Sen Liu, Chengyin Min, Tianshu Feng, Haris Jahic, Janid A Ali, Daniel R Aird, Fang Li, Jannik N Andersen, A. Huang, William D Mallender, Hilary E. Nicholson
          </td>
          <td>2025-01-27</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Radiotherapy is an integral component in the treatment of many types of cancer, with approximately half of cancer patients receiving radiotherapy. Systemic therapy applies pressure that can select for resistant tumor subpopulations, underscoring the importance of understanding how radiation impacts tumor evolution to improve treatment outcomes. We integrated temporal genomic profiling of 120 spatially distinct tumor regions from 20 patients with undifferentiated pleomorphic sarcomas (UPS), longitudinal circulating tumor DNA (ctDNA) analysis, and evolutionary biology computational pipelines to study UPS evolution during tumorigenesis and in response to radiotherapy. Most unirradiated UPS displayed initial linear evolution followed by subsequent branching evolution with distinct mutational processes during early and late development. Metrics of genetic divergence between regions provided evidence of strong selection pressures during UPS development that further increased during radiotherapy. Subclone abundance changed following radiotherapy with subclone contraction tied to alterations in calcium signaling, and inhibiting calcium transporters radiosensitized sarcoma cells. Finally, ctDNA analysis accurately measured subclone abundance and enabled non-invasive monitoring of subclonal changes. These results demonstrate that radiation exerts selective pressures on UPS and suggest that targeting radioresistant subclonal populations could improve outcomes after radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdc94aca0e727c5cc3864519a9f72d72e8fbb00" target='_blank'>
              Evolutionary Pressures Shape Undifferentiated Pleomorphic Sarcoma Development and Radiotherapy Response.
              </a>
            </td>
          <td>
            Erik Blomain, Shaghayegh Soudi, Ziwei Wang, Anish Somani, Ajay Subramanian, Serey C L Nouth, E. Oladipo, Chistin New, Deborah E Kenney, Neda Nemat-Gorgani, Thomas Kindler, R. Avedian, Robert J Steffner, D. Mohler, S. Hiniker, Alexander L. Chin, A. Kalbasi, M. Binkley, Marius Fried, Matthias M Gaida, Matt van de Rijn, E. J. Moding
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Noncoding microRNAs tend to evolve within introns of coding genes that provide them with transcriptional opportunity. As an outcome of natural selection, the intragenic position of microRNAs is crucial for their expression, evolution, and functional cooperation with the host gene. Therefore, understanding the evolution of intragenic microRNA structures may bring novel insights into genetic and phenotypic evolution. However, it remains largely unexplored. Here, by analyzing microRNA genomics in 34 metazoan species, we found that the majority (630/1,154) of microRNA families originated from introns of coding genes that provided them with initial transcriptional capacity. The most rapid expansion of intragenic microRNAs happened at the advent of vertebrates when 21 microRNA families emerged from introns of neural genes and reorganized the gene regulatory network, leading to the rise of vertebrate-specific neural crest cells, which transformed the invertebrate head and enabled the ecological shift from filter feeding to active predation. Intragenic microRNAs gradually gain independence from their host genes, which is accelerated by whole-genome duplications. After a whole-genome duplication, the purging of redundant host genes often set an orphaned microRNA “free” to diversify with the transcriptional elements inherited from the host. Whole-genome duplications facilitated a dramatic microRNA diversification during the initial divergence of vertebrates, as the intragenic status of 12 neural crest-regulating microRNAs was retained in jawed vertebrates but was lost in jawless cyclostomes, which diverged their neural crest development. We propose that coding genes not only facilitate the origination of new microRNAs, but also “sacrifice” themselves to help microRNA diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec14a5673e5811f8ba88226799b41e9414365cb1" target='_blank'>
              Coding Genes Helped the Origination and Diversification of Intragenic MicroRNAs
              </a>
            </td>
          <td>
            Chengzhang Liu, Xiaojun Zhang, Jianbo Yuan, J. Xiang, Fuhua Li
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The PPM1D gene and its protein product (serine-threonine protein phosphatase, PPM1D or Wip1) are involved in regulation of cell's DNA damage response, cell cycle control, and repair. Amplification, overexpression, or mutations of the PPM1D gene have a significant impact on cell responses to stress factors and genetic instability as well as impairments of processes of double-strand break repair, nucleotide excision repair, base excision repair, cell cycle, and apoptosis. PPM1D dephosphorylates and thus inactivates p53, proteins that respond to DNA strand integrity damage, cell cycle checkpoint proteins, and apoptotic proteins. This contributes to tumor development, growth, and maintenance of the tumor phenotype. In this review we consider data on the role of the PPM1D gene in the formation and maintenance of various oncological processes, including tumors of the mammary glands, ovaries, prostate gland, esophagus, stomach, intestines, liver and pancreas, hemoblastoses, and others.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the M-phase, the nuclear membrane is broken down, nucleosomes are condensed as mitotic chromosomes, and transcription factors are generally known to be dislocated from their recognition sequences and dispersed to the cytoplasm. However, some transcription factors have recently been reported to remain on mitotic chromosomes and facilitate the rapid re-activation of the target genes in early G1-phase. Paired-like homeobox 2B (PHOX2B) is a transcription factor exhibiting chromosomal localization during M-phase. PHOX2B mutations are associated with congenital central hypoventilation syndrome, Hirschsprung disease, and neuroblastoma. In this study, we investigated PHOX2B chromosomal localization during M-phase through immunostaining and fluorescence recovery after photobleaching analysis to determine whether the chromosomal localization of disease-associated PHOX2B mutants is altered during M-phase. Missense mutations in the homeodomain and the frameshift mutation in the C-terminal domain disrupted the chromosomal localization of PHOX2B in M-phase, leading to its dispersion in the cell. Furthermore, a PHOX2B mutant with polyalanine expansion showed a line-shaped localization to the restricted region of mitotic chromosomes. Our findings suggest an association between the disease-associated mutations and defective chromosomal localization of transcription factors during M-phase. Further investigations of PHOX2B chromosomal localization during M-phase could reveal pathogenic mechanisms of such diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc280a650e26ad4fbfbcabfa8901f734f1fa074d" target='_blank'>
              Chromosomal localization of PHOX2B during M-phase is disrupted in disease-associated mutants.
              </a>
            </td>
          <td>
            Yuki Sato, Shin-ichiro Hayashi, Souichi Oe, T. Koike, Y. Nakano, Ryohei Seki-Omura, Hikaru Iwashita, Y. Hirahara, Masaaki Kitada
          </td>
          <td>2025-02-11</td>
          <td>Development, growth & differentiation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Circular RNA (circRNA), a subtype of noncoding RNA, has emerged as a significant focus in RNA research due to its distinctive covalently closed loop structure. CircRNAs play pivotal roles in diverse physiological and pathological processes, functioning through mechanisms such as miRNAs or proteins sponging, regulation of splicing and gene expression, and serving as translation templates, particularly in the context of various cancers. The hallmarks of cancer comprise functional capabilities acquired during carcinogenesis and tumor progression, providing a conceptual framework that elucidates the nature of the malignant transformation. Although numerous studies have elucidated the role of circRNAs in the hallmarks of cancers, their functions in the development of chemoradiotherapy resistance remain unexplored and the clinical applications of circRNA‐based translational therapeutics are still in their infancy. This review provides a comprehensive overview of circRNAs, covering their biogenesis, unique characteristics, functions, and turnover mechanisms. We also summarize the involvement of circRNAs in cancer hallmarks and their clinical relevance as biomarkers and therapeutic targets, especially in thyroid cancer (TC). Considering the potential of circRNAs as biomarkers and the fascination of circRNA‐based therapeutics, the “Ying‐Yang” dynamic regulations of circRNAs in TC warrant vastly dedicated investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c029c3f0dab07c4c68d6c59a5ca97d4f501ffcd" target='_blank'>
              Circular RNAs in cancer
              </a>
            </td>
          <td>
            Yang Guo, Qiang Huang, Y. Heng, Yujuan Zhou, Hui Chen, Chengzhi Xu, Chunping Wu, Lei Tao, Liang Zhou
          </td>
          <td>2025-02-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Two rounds of whole-genome duplication (WGD) occurred about 500 million years ago and played a major role in the evolution of the vertebrate genomes. Human genes derived from WGD are called “ohnologs”. Ohnologs are involved in fundamental biological processes and significantly contributed to the complexity of the human gene regulatory network. Given the central role of miRNAs in gene regulation, we investigated the contribution of ohnolog miRNAs (Ohno-miRNAs) to the human gene regulatory network. We worked on the identification of miRNA pairs in the human genome derived from the two rounds of whole-genome duplication in early vertebrate lineage and their role in the structure of the human gene regulatory network. By focusing on intragenic miRNAs within ohnolog gene pairs, we identified Ohno-miRNAs as having higher retention rates and sequence similarity compared to miRNAs hosted on paralogue gene pairs from small-scale duplications (SSD). They also show a stronger tendency to regulate common target genes. Analyzing the role of Ohno-miRNAs in the human gene regulatory network, we showed that Ohno-miRNAs are statistically overrepresented in specific network motifs commonly associated with redundancy and complexity, highlighting their central role in this particular layer network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/317e406e602cc05274e82af803c484357008c5a8" target='_blank'>
              Ohno-miRNAs: intragenic miRNA pairs derived from whole-genome duplication
              </a>
            </td>
          <td>
            Leonardo Agasso, Ivan Molineris, M. Caselle
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b8fba2b01c4b6b3b2c8d14a0db3601be28447" target='_blank'>
              BRPF1 inhibition reduces migration and invasion of metastatic ovarian cancer cells, representing a potential therapeutic target
              </a>
            </td>
          <td>
            E. Alexandrova, Marharyta Smal, Domenico Di Rosa, Rosario Nicola Brancaccio, Roberto Parisi, Fabio Russo, Roberta Tarallo, G. Nassa, G. Giurato, Alessandro Weisz, Francesca Rizzo
          </td>
          <td>2025-03-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07346b8a2c818bb0fe268d0dd9f1b1f14d6eac83" target='_blank'>
              CTHRC1 expresses in cancer-associated fibroblasts and is associated with resistance to anti-androgen therapy in prostate cancer.
              </a>
            </td>
          <td>
            Hai-Qi Liang, Qi-Huan He, Qiu-Ju Wei, Qi-Zhou Mo, Guang-Lin Yang, Fa-Ye Wei, Li-Wei Wei, Yu-Jian Li, Min Qin, Ji-Wen Cheng
          </td>
          <td>2025-02-26</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Meiosis in mammalian oocytes is interrupted by a prolonged arrest at the germinal vesicle stage, during which oocytes have to repair DNA lesions to ensure genome integrity or otherwise undergo apoptosis. The FIRRM/FLIP-FIGNL1 complex dissociates RAD51 from the joint DNA molecules in both homologous recombination (HR) and DNA replication. However, as a type of non-meiotic, non-replicative cells, whether this RAD51-dismantling mechanism regulates genome integrity in oocytes remains elusive. Here, we show that FIRRM/FLIP is required for disassembly of RAD51-filaments and maintenance of genome integrity in oocytes. Deletion of FIRRM in oocytes leads to formation of massive nuclear RAD51 foci in oocytes of primordial follicles and activated follicles in mice. These RAD51 foci colocalize with the sites of DNA damage repair, as indicated by RPA2 and EdU, suggesting substantial DNA damage and extensive HR in oocytes. Especially in fully-grown FIRRM-deleted oocytes, RAD51 forms a net-like structure. As a consequence, FIRRM-deleted females are infertile due to aberrant homologous chromosome segregation at metaphase I and primordial follicle insufficiency at young adulthood. Hence, our study demonstrates the physiological importance of HR in maintaining genome integrity in oocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca41eb4ac62e414cd3f1677c195f277ccdccda71" target='_blank'>
              Extensive homologous recombination safeguards oocyte genome integrity in mammals
              </a>
            </td>
          <td>
            Huiwen Cao, Cheng Qiu, Anxuan Fang, Jianzhou Shang, Wei Xu, Lugeng He, Xing Duan, Qianting Zhang, Chao Yu
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20bcfbeb6577568a7ff02bdc23116269869d367" target='_blank'>
              TFAP2A-regulated CRNDE enhances colon cancer progression and chemoresistance via RIPK3 interaction
              </a>
            </td>
          <td>
            Xin Gao, Yanming Huang, Tonghui Wei, Jingmin Xue, Filippov Iurii, Laishou Yang, Liying Wang, Hao Li, Genshen Mo, Yuze Huang, Haonan Xie, Hang Wang, Shenghan Lou, Peng Han
          </td>
          <td>2025-02-15</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Mitochondrial DNA (mtDNA) mutagenesis remains poorly understood despite its crucial role in disease, aging, and evolutionary tracing. In this study, we reconstructed a comprehensive 192-component mtDNA mutational spectrum for chordates by analyzing 118,397 synonymous mutations in the CytB gene across 1,697 species and five classes. This analysis revealed three primary forces shaping mtDNA mutagenesis: (i) symmetrical, replication-driven errors by mitochondrial polymerase (POLG), resulting in C > T and A > G mutations that are highly conserved across classes; (ii) asymmetrical, damage-driven C > T mutations on the single-stranded heavy strand with clock-like dynamics; and (iii) asymmetrical A > G mutations on the heavy strand, with dynamics suggesting sensitivity to oxidative damage. The third component, sensitive to oxidative damage, positions mtDNA mutagenesis as a promising marker for metabolic and physiological processes across various classes, species, organisms, tissues, and cells. The deconvolution of the mutational spectra into mutational signatures uncovered deficiencies in both base excision repair (BER) and mismatch repair (MMR) pathways. Further analysis of mutation hotspots, abasic sites, and mutational asymmetries underscores the critical role of single-stranded DNA damage (components ii and iii), which, uncorrected due to BER and MMR deficiencies, contributes roughly as many mutations as POLG-induced errors (component i).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4269bc12502a52f2f9e09037077d3dcfaff9d7df" target='_blank'>
              Deciphering the Foundations of Mitochondrial Mutational Spectra: Replication-Driven and Damage-Induced Signatures Across Chordate Classes
              </a>
            </td>
          <td>
            D. Iliushchenko, B. Efimenko, A. G. Mikhailova, Victor Shamanskiy, Murat K. Saparbaev, B. Matkarimov, I. Mazunin, Alexander Voronka, Dmitry A Knorre, Wolfram S. Kunz, Philipp Kapranov, S. Denisov, J. Fellay, Konstantin Khrapko, K. Gunbin, K. Popadin
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="MCM2 belongs to the microchromosome maintenance [MCM] family and plays an essential role in initiating DNA replication as well as maintaining normal cellular cycle functions. Recent research indicates that there is the abnormal expression of MCM2 in various cancers, such as breast, cervical, ovarian, lung, hepatocellular carcinoma, nephroblastoma, prostate, and pancreatic cancers, where it shows a strong link to tumorigenesis, growth, invasion, migration, and adverse prognosis. Thus, MCM2 could serve as a significant biomarker for the early identification, diagnosis, and prognostic evaluation of multiple cancers. In addition, targeting MCM2 expression may open new possibilities for a full range of cancer treatments. In this paper, the protein structure, physiological function, and carcinogenic mechanism of MCM2 were reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8b9305ad5bdf4a40999bdb1d8deb688207b86c2" target='_blank'>
              Utilization of DNA Replication factor MCM2 by Cancer.
              </a>
            </td>
          <td>
            Yuwei Yuan, Tian Zeng, Anbo Gao, Yang Guan, Qun-feng Zhang, Yukun Li, Hui Tan, Juan Zou
          </td>
          <td>2025-02-13</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>