<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-19 09:30:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>34</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>71</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>111</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>335</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>293</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a common source of oncogene amplification across many types of cancer. The non-Mendelian inheritance of ecDNA contributes to heterogeneous tumour genomes that rapidly evolve to resist treatment. Here, using single-cell and live-cell imaging, single-micronucleus sequencing, and computational modelling, we demonstrate that elevated levels of ecDNA predisposes cells to micronucleation. Damage on ecDNA, commonly arising from replication stress, detaches ecDNA from the chromosomes upon which they hitchhike during cell division, thereby causing micronucleus formation in daughter cells. Clusters of oncogene-containing, CIP2A-TOPBP1-associated ecDNA molecules form, and asymmetrically segregate into daughter cell micronuclei during cell division. ecDNA chromatin remains highly active during mitosis, but upon micronucleation, it undergoes suppressive chromatin remodeling, largely ceasing oncogene transcription. These studies provide insight into the fate of damaged ecDNA during cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome conformation underlies transcriptional regulation by distal enhancers, and genomic rearrangements in cancer can alter critical regulatory interactions. Here we profiled the three-dimensional genome architecture and enhancer connectome of 69 tumor samples spanning 15 primary human cancer types from The Cancer Genome Atlas. We discovered the following three archetypes of enhancer usage for over 100 oncogenes across human cancers: static, selective gain or dynamic rewiring. Integrative analyses revealed the enhancer landscape of noncancer cells in the tumor microenvironment for genes related to immune escape. Deep whole-genome sequencing and enhancer connectome mapping provided accurate detection and validation of diverse structural variants across cancer genomes and revealed distinct enhancer rewiring consequences from noncoding point mutations, genomic inversions, translocations and focal amplifications. Extrachromosomal DNA promoted more extensive enhancer rewiring among several types of focal amplification mechanisms. These results suggest a systematic approach to understanding genome topology in cancer etiology and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75ec987e99ab172c82d4158473e7b61d31088a2b" target='_blank'>
              Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Kathryn E Yost, Yanding Zhao, King L. Hung, Kaiyuan Zhu, Duo Xu, M. Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, , William J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-05-12</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f629203ff4e6d119bd3d8d7b5f0a9d082c7f87e7" target='_blank'>
              Abstract 5046: Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, W.J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7372198dd7ad93e0ac09354eea55bbe4096844" target='_blank'>
              Human Papillomavirus Genomes Associate with Active Host Chromatin during Persistent Viral Infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7809d316520a84fae49c8004779285724bae28b" target='_blank'>
              Analysis of regulatory mechanisms of enhancers in gastric cancer with double minute chromosomes based on bioinformatics.
              </a>
            </td>
          <td>
            Meng-Ting An, Guan-Lin Guo, Jie Wu, Wen-Jing Sun, 
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc027322f110679344e9de5725b902edc43f13e" target='_blank'>
              Deregulated enhancer-promoter communication in cancer through altered nuclear architecture.
              </a>
            </td>
          <td>
            Isabelle Seufert, Claire Vargas, Sina Jasmin Wille, K. Rippe
          </td>
          <td>2025-04-12</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, P. Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c882aec893125b1655a88dcecde1b32dd7df1e2c" target='_blank'>
              Abstract 4094: Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, E. L. Cadieux, Özgen Deniz, M. Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, Katie Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93434ec63d05d303e79c327f791808c80b48abc" target='_blank'>
              Sustained MYB activity drives emergent enhancer activation and precise enhancer-promoter interactions
              </a>
            </td>
          <td>
            I-Jun Lau, Joe R Harman, Alastair L Smith, N. Denny, Nicole E Jackson, Joseph C Hamley, Paresh Vyas, James Oliver Davies, Jim R. Hughes, Nicholas T Crump, Thomas A. Milne
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c93e7f3cab1fe1fe207b6d98907a185c39cb35" target='_blank'>
              Abstract LB306: 3D epigenomic analysis reveals non-coding DNA regions that can reverse prostate cancer phenotypes
              </a>
            </td>
          <td>
            Huan Cao, Zexun Wu, Baixi Ji, Seolyn Yang, Leonardo Gonzalez-Smith, S. Rhie
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, M. Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M. Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b54c36ea8dc8ae6d831b9cec1f08634da1f520c" target='_blank'>
              Abstract 1152: The landscape of copy number variations following WGD in 5,980 colorectal cancers
              </a>
            </td>
          <td>
            Jierui Xu, H. Walch, Paolo Manca, Maria A Perry, Christopher Fong, Justin Jee, K. Pichotta, E. Vakiani, Neil H. Segal, Adam Price, Allison L Richards, Chai Bandlamudi, M. Donoghue, A. Cercek, R. Yaeger, N. Schultz, W. Chatila
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b6b876ae54e45db144a86ddc16615e044025e57" target='_blank'>
              Abstract 5049: Haplotype-specific copy number profiling from long reads sequencing data
              </a>
            </td>
          <td>
            Tanveer Ahmad, M. Kolmogorov, A. Keskus, Ataberk Donmez, M. Farooqi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90139c8e3ae8545f259aeac6842fc6fa1b255360" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf2305c32fc721604cdb952ddd1c89201b94faf9" target='_blank'>
              Abstract 6625: LinkPrep™ - a comprehensive assay for cancer genome characterization - enabling structural variant detection and discovery of enhancer hijacking
              </a>
            </td>
          <td>
            Ericca Stamper, Jonathon Torchia, Daniel Hwang, Chu Yuan Wu, Ek Danwanichakul, Cory Padilla, J. Z. Sanborn, Lisa Munding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9dfba50807f4d056e7d436de2c34ed7ddd425f" target='_blank'>
              Abstract 1502: Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx
              </a>
            </td>
          <td>
            Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, C. Swanton, M. Jamal-Hanjani, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Exonic enhancers (EEs) occupy an under-appreciated niche in gene regulation. By integrating transcription factor binding, chromatin accessibility, and high-throughput enhancer-reporter assays, we demonstrate that many protein-coding exons possess enhancer activity across species. These EEs exhibit characteristic epigenomic signatures, form long-range interactions with gene promoters, and can be altered by both nonsynonymous and synonymous variants. CRISPR–mediated inactivation demonstrated the involvement of EEs in the cis-regulation of host and distal gene expression. Through large-scale cancer genome analyses, we reveal that EE mutations correlate with dysregulated target-gene expression and clinical outcomes, highlighting their potential relevance in disease. Evolutionary comparisons show that EEs exhibit both strong sequence constraint and lineage-specific plasticity, suggesting that they serve ancient regulatory functions while also contributing to species divergence. Our findings redefine the landscape of functional elements by establishing EEs as a component of gene regulation, while revealing how coding regions can simultaneously fulfil both protein-coding and cis-regulatory roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/163da6e03d81a31e9fe14936d274ce9d2b480af7" target='_blank'>
              Exonic enhancers are a widespread class of dual-function regulatory elements
              </a>
            </td>
          <td>
            Jean-Christophe Mouren, Magali Torres, Antoinette F. van Ouwerkerk, Iris Manosalva, F. Gallardo, Salvatore Spicuglia, Benoît Ballester
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Structural variants (SVs) are increasingly recognized as important contributors to oncogenesis through their effects on 3D genome folding. Recent advances in whole-genome sequencing have enabled large-scale profiling of SVs across diverse tumors, yet experimental characterization of their individual impact on genome folding remains infeasible. Here, we leveraged a convolutional neural network, Akita, to predict disruptions in genome folding caused by somatic SVs identified in 61 tumor types from the Children’s Brain Tumor Network dataset. Our analysis reveals significant variability in SV-induced disruptions across tumor types, with the most disruptive SVs coming from lymphomas and sarcomas, metastatic tumors, and germline cell tumors. Dimensionality reduction of disruption scores identified five recurrently disrupted regions enriched for high-impact SVs across multiple tumors. Some of these regions are highly disrupted despite not being highly mutated, and harbor tumor-associated genes and transcriptional regulators. To further interpret the functional relevance of high-scoring SVs, we integrated epigenetic data and developed a modified Activity-by-Contact scoring approach to prioritize SVs with disrupted genome contacts at active enhancers. This method highlighted highly disruptive SVs near key oncogenes, as well as novel candidate loci potentially implicated in tumorigenesis. These findings highlight the utility of machine learning for identifying novel SVs, loci, and genetic mechanisms contributing to pediatric cancers. This framework provides a foundation for future studies linking SV-driven regulatory changes to cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e24a79ed206cbada1bc6b51069582d272e00ef" target='_blank'>
              Machine learning-predicted chromatin organization landscape across pediatric tumors
              </a>
            </td>
          <td>
            Ketrin Gjoni, Shu Zhang, Rachel E. Yan, Bo Zhang, Daniel Miller, Adam C. Resnick, Nadia Dahmane, K. Pollard
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e63be79d13b386a8293331209bd30415a9a147" target='_blank'>
              Enrichment of Z-DNA-Forming Sequences Within Super-Enhancers: A Computational and Population-Based Study
              </a>
            </td>
          <td>
            Yulia V. Makus, G. A. Ashniev, Alexey V. Orlov, Petr I. Nikitin, Zoia G. Zaitseva, N. Orlova
          </td>
          <td>2025-05-04</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de502f1073df10f45cbf431033ea87108c552c" target='_blank'>
              ARID1A stabilises non-homologous end joining factors at DNA breaks induced by the G4 ligand pyridostatin
              </a>
            </td>
          <td>
            Susana Simões-Sousa, Noa Amin, Karen A. Lane, Alison Harrod, Malin Pedersen, Mercedes Pardo, Christine Rasetti-Escargueil, Z. Kozik, Katheryn A. G. Begg, Alison Ribeiro, Susana Banerjee, Manuel Stucki, Kevin J. Harrington, Jyoti S. Choudhary, Jessica A. Downs
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8be723b80118653ca09b51446fe4768c5505b8" target='_blank'>
              A Guide to Extrachromosomal DNA: Cancer's Dynamic Circular Genome.
              </a>
            </td>
          <td>
            Natasha E Weiser, Thomas B K Watkins, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7517f5d7b5ed8e4f3c0b12985d9cbc0431dd1" target='_blank'>
              Cumulative TF binding and H3K27 Acetylation drive enhancer activation frequency
              </a>
            </td>
          <td>
            Valentina Baderna, Guido Barzaghi, Rozemarijn Kleinendorst, Kasit Chatsirisupachai, Laura Moniot-Perron, Meike Schopp, T. Hochepied, Claude Libert, Duncan T Odom, Judith B. Zaugg, Arnaud Krebs
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Allele-specific DNA methylation (ASM) provides critical insights into the complex genetic and epigenetic mechanisms regulating gene transcription. Emerging evidence suggests that ASM is particularly enriched in gene enhancer regions, and recent studies have demonstrated that ASM is increased in cancer tissues compared with normal tissues. Despite the increasing recognition of ASM as a potential biomarker in tumorigenesis, systematic resources dedicated to identifying and annotating ASMs in cancer contexts remain limited. In this study, we developed CanASM (https://bioinfor.nefu.edu.cn/CanASM/), the first comprehensive database specifically designed to identify and annotate ASM in cancer. In CanASM, ASM sites identified from bisulfite sequencing data across 31 cancer types and their matched normal tissue samples are cataloged. Importantly, CanASM includes extensive regulatory annotations for ASMs, including associated genes, cis-regulatory elements and transcription factor binding colocalizations, transcription factor affinity changes, etc. Users can query and explore ASMs using various parameters, such as single-nucleotide variations (SNVs), chromosomal coordinates, and gene names. The current version of CanASM includes 5,003,877 unique SNV–CpG pairs, including 3,056,776 index SNVs, of which 2,634,406 are single-nucleotide polymorphisms (SNPs), and 4,157,508 CpGs. With an intuitive interface for browsing, querying, analyzing, and downloading, CanASM serves as a valuable resource for researchers investigating cancer-associated genetic variations and epigenetic regulation in cancer. Author summary Identifying target genes and regulatory mechanisms of cancer-associated genetic variants remains a major challenge in the post-GWAS era. Existing databases like Pancan-meQTL and ASMdb have contributed to uncovering the role of genetic variants in cancer by linking DNA variations to methylation. However, they lack comprehensive regulatory annotations for such sites and rely solely on genomic proximity for target gene prediction, overlooking distal interactions. Our analysis using 3D interaction data reveals that over 94.8% of ASMs exhibit distal regulatory relationships, emphasizing their critical role in cancer epigenetics. Furthermore, these databases cover a limited number of cancer types (Pancan-meQTL: 23, ASMdb: 8). To address these limitations, we developed CanASM, a database that provides ASMs identified from BS-seq data for 31 human cancers, along with multidimensional regulatory annotations for ASM sites. This includes target genes, cis-regulatory elements (enhancers, ATAC-seq regions, insulators, 3D interactions, CpG islands), TFBSs, and GWAS cancer trait associations. Additionally, CanASM offers three interactive tools for analyzing TF binding affinity changes, predicting ASM target genes through 3D interactions, and identifying cell marker variations. As a comprehensive resource for SNV-driven epigenetic regulation in cancer, CanASM is helpful for in-depth research in cancer genetics and epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0525da13a9172e31ed829ad891d330567e188ba3" target='_blank'>
              CanASM: A comprehensive database for genome-wide allele-specific DNA methylation identification and annotation in cancer
              </a>
            </td>
          <td>
            Jianmei Zhao, Zeyu zhao, Hongfei Li, Haojie Yu, Hao Lin, Hanqi Chen, Xuecang Li, Di Liu, Yiming Wang, Guohua Wang
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) are circular, double-stranded DNA molecules ubiquitously present across various organisms, playing a critical role in tumorigenesis and tumor progression. However, their precise contribution to prostate cancer (PCa) remains incompletely understood. To elucidate the function of eccDNAs in PCa, eccDNA sequencing and annotation were performed in PCa tissues and cell lines using Circle-seq. Amplified genes on eccDNAs were identified by cross-referencing annotated eccDNA-associated genes with those overexpressed in PCa based on TCGA data. Furthermore, eccDNA profiles were compared between Enzalutamide-sensitive and -resistant cell lines to investigate their role in resistance mechanisms. Notably, VAV2 was detected on both linear and circular DNA, as confirmed by PCR and Sanger sequencing. Functional analyses demonstrated that VAV2 overexpression promotes PCa proliferation and metastasis by activating the PAK1/AKT signaling pathway through PAK1 phosphorylation. Additionally, VAV2 contributes to Enzalutamide resistance by enhancing AR/ARv7 protein stability via reduced ubiquitination, mediated through the recruitment of the deubiquitinating enzyme USP48. These findings establish VAV2, identified through eccDNA sequencing, as a potential oncogene and a promising biomarker for PCa diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin’an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1dc55203f4f61b62677fd9af379d34ddca08f1" target='_blank'>
              Abstract 1479: Defining the interplay between DNAPKcs and shieldin during chromosomal break repair
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mammalian genomes undergo pervasive transcription in both genic and intergenic regions. Tri-methylation of histone H3 lysine 4 (H3K4me3) is a deeply conserved and functionally important histone modification enriched at transcriptionally active promoters, where it facilitates RNA polymerase activity. H3K4me3 is also commonly found in intergenic regions, where its role is poorly understood. We interrogated the role of H3K4me3 at intergenic regulatory elements by using epigenetic editing to efficiently deposit H3K4me3 at intergenic loci. We find that H3K4me3 instructs RNA polymerase activity and is actively remodeled at intergenic regions, shedding light on these important but poorly understood regions of the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fec0f2b2acd349d0dd2600b011de5bca83a1c403" target='_blank'>
              H3K4me3 instructs transcription at intergenic active regulatory elements
              </a>
            </td>
          <td>
            Haoming Yu, Yongyan Zhang, Zhicong Liao, B. W. Walters, Bluma J. Lesch
          </td>
          <td>2025-03-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy J. Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles are unique, allowing sample stratification into ATRT subtypes. The TE activity signature in ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC shows broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC is also unique in having significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and-MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b1746e8b34806ab31721a10a88339f9bbc0904" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction: Hi-C is a widely used technique for mapping chromosomal interactions within a 3D genomic framework, however, its resolution is often constrained by sequencing depth, making it challenging to detect fine-scale interactions. To overcome this limitation, Promoter-Capture Hi-C (PCHi-C), as it selectively enriches for promoter-associated interactions, was employed. This study integrates PCHi-C and Hi-C datasets from colorectal cancer (CRC) models investigate chromosomal interaction dynamics across various regulatory levels, from cis-regulatory elements to topologically associated domains (TADs). The primary goal is to examine how genomic structural alterations shape the epigenomic landscape in CRC and to assess their potential role in colorectal cancer susceptibility. Methods: PCHi-C and Hi-C datasets from multiple colorectal cancer (CRC) studies were integrated to enhance the resolution of chromatin interaction mapping. The analysis focused on identifying fine-scale interactions within topologically associated domains (TADs) while incorporating histone modification landscapes (H3K27ac, H3K4me3) and transcriptomic signatures from CRC cell lines and the TCGA database. For experimental validation, ChIP-quantitative PCR was performed at the promoters of target genes using the highly malignant colorectal cell line HT29 and compared it to an embryonic cell line NT2D1. Results: Our integrated analysis revealed significant genomic structural instability in CRC cells, closely associated with tumor-suppressive transcriptional programs. We identified nine dysregulated genes, including long non-coding RNAs (MALAT1, NEAT1, FTX, and PVT1), small nucleolar RNAs (SNORA26 and SNORA71A), and protein-coding genes (TMPRSS11D, TSPEAR, and DSG4), all of which exhibited a substantial increase in expression in CRC cell lines compared to human embryonic stem cells (hESCs). Additionally, we observed enriched activation-associated histone modifications (H3K27ac and H3K4me3) at the potential enhancer regions of these genes, indicating possible transcriptional activation. ChIP-quantitative PCRs conducted using in the highly malignant CRC cell line HT29, compared to the embryonic cell line NT2D1, further validated these findings, reinforcing the link between altered chromosomal interactions and gene dysregulation in CRC. Discussion: This study sheds light on the dynamic 3D genome organization in CRC, highlighting critical structural changes associated with disease-associated loci. The identification of nine dysregulated genes points to potential biomarkers for colorectal cancer, with implications for diagnostic and therapeutic strategies. The combination of Hi-C and PCHi-C offers a refined approach for detecting chromosomal interactions at a higher resolution, laying the foundation for future studies on cancer-associated chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c3094833c1e043ac48c473ef6e13b0033ea5bf" target='_blank'>
              Integrated promoter-capture Hi-C and Hi-C analysis reveals fine-tuned regulation of the 3D chromatin architecture in colorectal cancer
              </a>
            </td>
          <td>
            A. K. Saw, Ayush Madhok, Anupam Bhattacharya, Soumyadeep Nandi, Sanjeev Galande
          </td>
          <td>2025-03-28</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are the effective lesion of cancer radiotherapy and induce gene editing. 53BP1 accumulates at DSBs and is implicated in end joining (EJ) repair, but its influence on DSB repair is distinct from canonical non-homologous end joining (C-NHEJ). We sought to define the genetic interplay of 53BP1 with C-NHEJ, focusing on the DNA-PKcs kinase. We examined Cas9 DSBs, which largely generates blunt DSBs, since blunt DSB EJ is dependent on C-NHEJ. Loss of 53BP1 does not affect blunt DSB EJ, but causes a reduction in such repair in DNA-PKcs deficient cells. In contrast, disrupting 53BP1 and DNA-PKcs, both alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We found similar effects on EJ with RIF1 loss, which is a downstream effector of 53BP1. Thus, 53BP1/RIF1 appear to play a backup role for DNA-PKcs during blunt DSB EJ, but function in the same pathway to suppress microhomology deletions. In contrast, 53BP1 and DNA-PKcs function independently to suppress homology-directed repair. Finally, DNA-PKcs kinase inhibition causes marked radiosensitivity, which is not additive with loss of 53BP1 and RIF1. Altogether, 53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb1f8550e47aed05c0a51b4b294e0030758aa9f" target='_blank'>
              53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef492b9d87d02d306f99109c4e3ede437c5ca62c" target='_blank'>
              Enhancer control of promoter activity and variability via frequency modulation of clustered transcriptional bursts
              </a>
            </td>
          <td>
            Jana Tünnermann, Gregory Roth, J. Cramard, Luca Giorgetti
          </td>
          <td>2025-03-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62dd3d2788ffe055d52a584d9b884866f23c266" target='_blank'>
              Delayed indel formation after Cas9 cleavage promotes mosaicism and aneuploidy in human embryos
              </a>
            </td>
          <td>
            Alex Robles, Julie Sung, Stepan Jerabek, Jishnu Talukdar, Diego Marin, Jia Xu, Nathan Treff, Dieter Egli
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a04fc6c896abc9b9472141040cfd525f6ea0447a" target='_blank'>
              Abstract 5045: Cancer multiomics: Whole-genome DNA, methylation, RNA-seq, and direct RNA characterization of cancer genomes using Oxford Nanopore sequencing
              </a>
            </td>
          <td>
            Sergey Aganezov, Philipp Rescheneder, Aino Jarvelin, Sean Mckenzie, Ella Kennett, George Dodd, S. Juul
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6be55a5ef5db321d17ba7bfe59475decf1a2ca" target='_blank'>
              Abstract 2855: CDK12 inhibition remodels DNA replication to drive selective gene amplification
              </a>
            </td>
          <td>
            Shivani Yalala, Julia Kutz, Helmut Pospiech, Harri M Itkonen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The molecular control of epigenetic information relies on hundreds of proteins of diverse function, which cooperate in defining chromatin structure and DNA methylation landscapes. While many individual pathways have been characterized, how different classes of epigenetic regulators interact to build a resilient epigenetic regulatory network (ERN) remains poorly understood. Here, we show that most individual regulators are dispensable for somatic cell fitness, and that robustness emerges from multiple layers of functional cooperation and degeneracy among network components. By disrupting 200 epigenetic regulator genes, individually or in combination, we generated network-wide maps of functional interactions for representative regulators. We found that paralogues represent only a first layer of functional compensation within the ERN, with intra- or inter-class interactions buffering the effects of perturbation in a gene-specific manner: while CREBBP cooperates with multiple acetyltransferases to form a subnetwork that ensures robust chromatin acetylation, ARID1A interacts with regulators from across all functional classes. When combined with oncogene activation, the accumulated epigenetic disorder exposes a synthetic fragility and broadly sensitizes ARID1A-deficient cells to further perturbation. Our findings reveal homeostatic mechanisms through which the ERN sustains somatic cell fitness and uncover how the network remodels as the epigenome is progressively deregulated in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74c759f71b52bc8e0f4b18c555786227753fc4c1" target='_blank'>
              Systematic genetic perturbation reveals principles underpinning robustness of the epigenetic regulatory network
              </a>
            </td>
          <td>
            Thomas Stuart Wilson, R. Noberini, Eirini Moysidou, Ifeyinwa Ojukwu, Marta Milan, Ming Jiang, Gavin Kelly, Michael Howell, Tiziana Bonaldi, Paola Scaffidi
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="TEA domain transcription factor 1 (TEAD1), a Hippo pathway transcription factor important in cellular homeostasis and development, is increasingly implicated in cancer biology. Here, we reveal a novel role for TEAD1 in organizing nuclear condensates, independent of active transcription. Using high-resolution imaging, ChIP-seq, RNA-seq and proximity-based proteomics, we demonstrate that in patient-derived renal cell carcinoma cells, TEAD1 forms micron-sized foci by binding to the heterochromatic pericentromeric regions using its DNA-binding domain. These TEAD1 foci do not mediate transcription but instead serve as depots for excess TEAD1. This contrasts with TEAD1 organization in other genomic regions of both RCC and normal kidney cells, where TEAD1 associates with markers of active transcription. Our findings provide a mechanistic framework for TEAD1’s dual regulatory roles, offering new insights into its contribution to transcriptional dysregulation and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf7a61d4683b21760f0a534c0754f40cc6df07c" target='_blank'>
              TEAD1 condensates are transcriptionally inactive storage sites on the pericentromeric heterochromatin
              </a>
            </td>
          <td>
            Jindayi Liang, Yiran Wang, Justin Demmerle, Britney Jiayu He, Christopher J. Ricketts, W. Marston Linehan, Chongzhi Zang, Danfeng Cai
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0036830d7ca0d3dd2d206e6880d31ee221a0279" target='_blank'>
              Advances in single-cell DNA sequencing enable insights into human somatic mosaicism.
              </a>
            </td>
          <td>
            Diane D. Shao, Andrea J. Kriz, Daniel A. Snellings, Zinan Zhou, Yifan Zhao, Liz Enyenihi, Christopher A. Walsh
          </td>
          <td>2025-04-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a4380bb34311ba8e2b2e583f5eb6ff7d05c33e" target='_blank'>
              Abstract 5050: Comparative analysis of somatic copy number variations and allelic imbalance in whole-genome bisulfite sequencing: challenges and insights
              </a>
            </td>
          <td>
            Yiru Gu, Jian Gu, P. Scheet, Justin Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44c370bd7521d24afccebc5dd604dc8d4b6c7b" target='_blank'>
              Abstract 2617: Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing
              </a>
            </td>
          <td>
            Yanling Liu, B. Ju, Li Dong, Melanie Loyd, Yuan Feng, H. Mulder, P. Kolekar, Li Fan, Hanxia Li, S. Foy, Geoffrey Neale, Patrick A. Brown, William L Carroll, M. Loh, John Easton, Xiaotu Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68e6681ead255204363cfd6f85f96a1a0227012" target='_blank'>
              Genome-wide modeling of DNA replication in space and time confirms the emergence of replication specific patterns in vivo in eukaryotes
              </a>
            </td>
          <td>
            Dario D’Asaro, J. Arbona, Vinciane Piveteau, Aurèle Piazza, Cédric Vaillant, Daniel Jost
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96f5ae7db10ebf2238c71827c45c782f6d159f1b" target='_blank'>
              New insights into chromosomal regions 15p11.2 to 15q11.2 by studying submicroscopic variations using molecular cytogenetics.
              </a>
            </td>
          <td>
            Maria N Correia, Stefanie Kankel, I. Carreira, Joana B. Melo, Thomas Liehr
          </td>
          <td>2025-04-17</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="DNA methylation is a significant component in proximal chromatin regulation and plays crucial roles in regulating gene expression and maintaining the repressive state of retrotransposon elements. However, accurate profiling of the proteomics which simultaneously identifies specific DNA sequences and their associated epigenetic modifications remains a challenge. Here, we report a strategy termed SelectID (selective profiling of epigenetic control at genome targets identified by dCas9), which introduces methylated DNA binding domain into dCas9-mediated proximity labeling system to enable in situ protein capture at repetitive elements with 5-methylcytosine (5mC) modifications. SelectID is demonstrated as feasible as dCas9-TurboID system at specific DNA methylation regions, such as the chromosome 9 satellite. Using SelectID, we successfully identify CHD4 as potential repressors of methylated long interspersed nuclear element-1 (LINE-1) retrotransposon through direct binding at the 5’ untranslated region (5’UTR) of young LINE-1 elements. Overall, our SelectID approach has opened up avenues for uncovering potential regulators of specific DNA regions with DNA methylation, which will greatly facilitate future studies on epigenetic regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583fb8bbad6809bf5dd6e0e97904aebaca8e992e" target='_blank'>
              Selective identification of epigenetic regulators at methylated genomic sites by SelectID
              </a>
            </td>
          <td>
            Wenchang Qian, Penglei Jiang, Mingming Niu, Yujuan Fu, Deyu Huang, Dong Zhang, Ying Liang, Qiwei Wang, Yingli Han, Xin Zeng, Yixin Shi, Lingli Jiang, Zebin Yu, Jinxin Li, Huan Lu, Hong Wang, Baohui Chen, Pengxu Qian
          </td>
          <td>2025-04-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d02fce4e58a33328241a1cb2df16862b6b79d6e" target='_blank'>
              Abstract 3901: Charting copy number gain histories from mutational divergence among multi-samples
              </a>
            </td>
          <td>
            Yunong Xia, Ruping Sun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Biomolecular condensates partition various cellular processes including transcription, DNA repair, and RNA metabolism. We report NELFE, a member of the Negative Elongation Factor complex required for Polymerase II (Pol II) pausing, forms distinct foci mediated by two low complexity sequences. We show NELFE is oncogenic in hepatocellular carcinoma (HCC) by undergoing liquid-liquid phase separation (LLPS) with SMARCB1 to modulate chromatin accessibility to downregulate pro-apoptotic genes through Pol II pausing while activating pro-growth signals to promote HCC progression. Our work highlights the importance of NELFE LLPS as a mechanism of chromatin accessibility to regulate both paused and non-paused genes to drive tumorigenesis in hepatocellular carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f26e8eddda4bd9a79c8360f1e8668793e05bfeb6" target='_blank'>
              Phase separation of NELFE modulates chromatin accessibility to promote dichotomous signaling pathways in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hien Dang, Alvaro Lucci, Anna E. Barry, Victoria Johnson, Brittany Ruiz, Laura Reynolds, Adam Wojnar, Kasonde Chewe, Y. Pomyen, Christoph Eckert, Kai Zhang, Jason Hill, Ashesh Shah, Adam Bodzin, Elda Grabocka, Matthias Gaida, Nicolas Fawzi
          </td>
          <td>2025-04-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C-and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract TAQing technologies are based on the restriction enzyme-induced DNA double-strand break (DSB) formation in living cells, which results in large-scale genomic rearrangements and phenotypic alterations. Originally, the TAQing system requires heat treatments to activate restriction enzymes, which sometimes leads to cell toxicity or stress responses. Here, we developed a blue-light-controlled MagTAQing system, which induces DSBs exclusively upon blue-light exposure by assembling the split restriction enzymes via Magnet modules. Application of MagTAQing to mitotic budding yeast cells successfully triggered various genomic rearrangements upon blue-light exposure. Since this technology enables the conditional induction of genomic rearrangements in specific cells or tissues, we employed MagTAQing on meiotic yeast cells lacking the recombinase Spo11 to induce artificial DSBs. Consequently, Spo11-independent meiotic DSBs resulted in aneuploidies and nonallelic homologous recombinations between repetitive sequences such as ribosomal DNA and retrotransposons. These results suggest a pivotal role of Spo11-induced recombination in preventing chromosomal abnormality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c560ade8948ca0b121c00661a4ad61330184b0" target='_blank'>
              Light-controlled Spo11-less meiotic DNA breaks by MagTAQing lead to chromosomal aberrations
              </a>
            </td>
          <td>
            Hideyuki Yone, Yuri Kawashima, Hayato Hirai, Arisa H. Oda, Moritoshi Sato, Hiromitsu Kono, Kunihiro Ohta
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce720a5b79eb9fb996bed883c2b0fde9438b7fa" target='_blank'>
              Genome evolution in parthenogenetic nematodes shaped by chromosome rearrangements and rare sex
              </a>
            </td>
          <td>
            George Chung, Lewis Stevens, Xiaoguang Dai, Scott Hickey, K. Kiontke, Priyesh Rughani, M. Kieninger, Joanna C. Collins, S. Juul, Fabio Piano, David H.A. Fitch, Mark Blaxter, Kristin C. Gunsalus
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The study examines Opisthorchis viverrini (OV)-related cholangiocarcinoma (CCA), a serious malignancy common in Southeast Asia. Through multi-regional whole-exome sequencing of 52 tumor samples and 13 adjacent tissues from 13 patients, significant intratumoral heterogeneity (ITH) and inter-patient heterogeneity are shown. Chronic liver fluke infection induces a distinct mutational landscape, with 48–90% of mutations concentrated in each region of the tumor. The average mutation burden is 95 non-synonymous mutations per area, exceeding previous CCA investigations. Critical driver mutations in TP53, SMAD4, and other genes underscore their significance in pathogenesis. Mutational markers elucidate mechanisms including spontaneous deamination and impaired DNA repair. Unique mutation patterns distinguish OV-associated CCA from other variants. Chromosomal instability in patient K110 signifies aggressive tumor behavior and unfavorable prognosis. Targetable mutations such as ERBB2 underscore the possibility for personalized therapeutics. These findings underscore the necessity for personalized strategies for treatment that target both trunk and branch mutations in endemic areas. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95142-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48c940bd80f998f283a697250c46b3404991cb4" target='_blank'>
              Whole exome sequencing of multi-regions reveals tumor heterogeneity in Opisthorchis viverrini-associated cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, A. Jusakul, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Bin Tean Teh, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In vivo genetic diversifiers have previously enabled efficient searches of genetic variant fitness landscapes for continuous directed evolution. However, existing genomic diversification modalities for mammalian genomic loci exclusively rely on deaminases to generate transition mutations within target loci, forfeiting access to most missense mutations. Here, we engineer CRISPR-guided error-prone DNA polymerases (EvolvR) to diversify all four nucleotides within genomic loci in mammalian cells. We demonstrate that EvolvR generates both transition and transversion mutations throughout a mutation window of at least 40 bp and implement EvolvR to evolve previously unreported drug-resistant MAP2K1 variants via substitutions not achievable with deaminases. Moreover, we discover that the nickase’s mismatch tolerance limits EvolvR’s mutation window and substitution biases in a gRNA-specific fashion. To compensate for gRNA-to-gRNA variability in mutagenesis, we maximize the number of gRNA target sequences by incorporating a PAM-flexible nickase into EvolvR. Finally, we find a strong correlation between predicted free energy changes underlying R-loop formation and EvolvR’s performance using a given gRNA. The EvolvR system diversifies all four nucleotides to enable the evolution of mammalian cells, while nuclease and gRNA-specific properties underlying nickase fidelity can be engineered to further enhance EvolvR’s mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4485500132f714892fcf79f15fa49a1b187bbe0" target='_blank'>
              Nickase fidelity drives EvolvR-mediated diversification in mammalian cells
              </a>
            </td>
          <td>
            Juan E Hurtado, Adam J. Schieferecke, Shakked O Halperin, John Guan, Dylan Aidlen, David V. Schaffer, J. Dueber
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb48bc35dc30e8efcbd8268b117fd6382f30da8" target='_blank'>
              Abstract 2395: The role of exonized mobile elements in chimeric transcript emergence in glioblastoma
              </a>
            </td>
          <td>
            Rafael L. V. Mercuri, T. L. Miller, Filipe F. dos Santos, M. D. de Lima, Aline Rangel-Pozzo, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6af71ed0a5156caa55bfb02af14d3017aff813e" target='_blank'>
              Abstract 2748: Integrative analysis of 3D chromatin organization revealed regulation mechanisms in angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Makoto Iwasaki, Caroline MacVicar, A. B. Rider, A. Sohani, Valentina Nardi, J. Ford, Kusha Chopra, Eric D. Jacobsen, Jeffrey Barnes, Salvia Jain
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Duplication is appreciated as a key source of genomic novelty. In this study, we examined putative enhancers in the human genome to investigate how duplication influences the evolution of these enhancers. We strictly identified a set of 3,336 confident duplicate enhancer pairs and examined their genomic and evolutionary features. We found that, compared to non-duplicated enhancers, duplicated enhancers tend to be longer, more pleiotropic, closer to genes, and harbor greater numbers and more diverse groups of transcription factor binding motifs. These attributes were more pronounced for evolutionarily older duplicate enhancers. Therefore, the regulatory potentials of enhancers may facilitate evolutionary maintenance of duplicated enhancers. Utilizing chimpanzee and rhesus macaque as outgroups, we found that between 30-40% of the examined duplicate enhancers exhibit evidence of asymmetric sequence evolution. Notably, the majority of the “accelerating enhancers” in such pairs gained enhancer activities in novel tissues, particularly in immune-related tissues. Moreover, the accelerating enhancers tended to harbor transcription factor binding motifs previously implicated in human evolution, and enriched in associations with immune functions and stress responses. These findings indicate that duplication may contribute to the proliferation of highly pleiotropic enhancers, as well as gaining novel enhancer activities, and contribute to rapid evolution of the immune system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c437b97a3389f686b6cd64b20f3a5da9a957f687" target='_blank'>
              Evolution of Enhancers through Duplication
              </a>
            </td>
          <td>
            Devika Singh, Taylor Hoyt, Soojin V. Yi
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3738bec14783d53b32ea0221a11feadb11d7ceea" target='_blank'>
              Abstract 176: Simultaneous spatial characterisation of copy number change and immune cell profiling using DNA FISH and multiplexed immunofluorescence
              </a>
            </td>
          <td>
            Eleanor M O'Roberts, Pranauti Panshikar, Xiaoze Li-Wang, Quentin Verron, S. S. Fard, Magda Bienko, Charlotte Stadler
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The interplay between 3D genomic structure and transposable elements (TE) in regulating cell state-specific gene expression program is largely unknown. Here, we explore the utilization of TE-derived enhancers in naïve and expanded pluripotent states by integrative analysis of genome-wide Hi-C-defined enhancer interactions, H3K27ac HiChIP profiling and CRISPR-guided TE proteomics landscape. Results We find that short interspersed nuclear elements (SINEs) are the more involved TEs in the active chromatin and 3D genome architecture. In particular, mammalian-wide interspersed repeat (MIR), a SINE family member, is highly associated with naïve-specific genomic interactions compared to the expanded state. Primarily, in the naïve pluripotent state, MIR enhancer is co-opted by ESRRB for naïve-specific gene expression program. This ESRRB and MIR enhancer interaction is crucial for the formation of loops that build a network of enhancers and super-enhancers regulating pluripotency genes. We demonstrate that loss of a ESRRB-bound MIR enhancer impairs self-renewal. We also find that MIR is co-bound by structural protein complex, ESRRB-YY1, in the naïve pluripotent state. Conclusions Altogether, our study highlights the topological regulation of ESRRB on MIR in the naïve potency state. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03577-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bbebe28792c7bd92836f3fe87c3dd070e1b1ca0" target='_blank'>
              Rewiring of SINE-MIR enhancer topology and Esrrb modulation in expanded and naive pluripotency
              </a>
            </td>
          <td>
            Nadia Omega Cipta, Yingying Zeng, Ka Wai Wong, Ziwen Zheng, Yao Yi, Tushar Warrier, Jian Zhou Teo, Jia Hao Jackie Teo, Y. Kok, Xuezhi Bi, Reshma Taneja, D. Ong, Jian Xu, Florent Ginhoux, Hu Li, Y. Liou, Y. Loh
          </td>
          <td>2025-04-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The precise regulation of the transcription of genes is essential for normal development and for the maintenance of life. Aberrant gene expression changes drive many human diseases. Despite this, we still do not completely understand how precise gene regulation is controlled in living systems. Enhancers are key regulatory elements that enable cells to specifically activate genes in response to environmental cues, or in a stage or tissue-specific manner. Any model of enhancer activity needs to answer two main questions: (1) how enhancers are able to identify and act on specific genes and (2) how enhancers influence transcription. To address these points, we first outline some of the basic principles that can be established from simpler prokaryotic systems, then discuss recent work on aberrant enhancer activity in leukemia. We argue that highly specific protein-protein interactions are a key driver of enhancer-promoter proximity, allowing enhancer-bound factors to directly act on RNA polymerase and activate transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27d90f79b42e3028d27e523c904d5697e499f59" target='_blank'>
              Is Enhancer Function Driven by Protein-Protein Interactions? From Bacteria to Leukemia.
              </a>
            </td>
          <td>
            Nicholas T Crump, Thomas A. Milne
          </td>
          <td>2025-04-08</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor suppressor p53 plays a crucial role in maintaining genome integrity in response to exogenous or endogenous stresses. The dynamics of p53 activation are stimulus- and cell type- dependent and regulate cell fate. Acting as a transcription factor, p53 induces the expression of target genes involved in apoptosis, cell cycle arrest and DNA repair. However, transcription is not a deterministic process, but rather occurs in bursts of activity and promoters switch stochastically between ON and OFF states, resulting in substantial cell-to-cell variability. Here, we characterized how stimulus-dependent p53 dynamics are converted into specific gene regulation patterns by inducing diverse forms of DNA damage ranging from ionizing and UV radiation to clinically relevant chemotherapeutics. We employed single molecule fluorescence in-situ hybridization (smFISH) to quantify the activity of target gene promoters at the single-cell and single-molecule level. To analyse this comprehensive data set, we developed a new framework for determining parameters of stochastic gene expression by Bayesian inference. Using this combined theoretical and experimental approach, we revealed that features of promoter activity are differentially regulated depending on the target gene and the nature and extent of the DNA damage induced. Indeed, stimulus-specific stochastic gene expression is predominantly regulated by promoter activation and deactivation rates. Interestingly, we found that in many situations, transcriptional activity was uncoupled from the total amount of p53 and the fraction bound to DNA, highlighting that transcriptional regulation by p53 is a multi- dimensional process. Taken together, our study provides insights into p53-mediated transcriptional regulation as an example of a dynamic transcription factor that shapes the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8f8f817463be32115a6d7b633baecddd3a2d98" target='_blank'>
              Decoding stimulus-specific regulation of promoter activity of p53 target genes
              </a>
            </td>
          <td>
            Flavia Vigliotti, Nicolai Engelmann, Mark Sinzger-D’Angelo, Heinz Koeppl, Alexander Loewer
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4538bea006263ba940dee6e9fb53dd7cfbcb8f30" target='_blank'>
              Abstract 124: Cell state regulation in early colorectal cancer through transcription factor and chromatin action
              </a>
            </td>
          <td>
            Sweta Sharma, Alireza Lorzadeh, George Ye, Unmesh Jadhav
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA double strand break repair (DSBR) represents a fundamental process required to maintain genome stability and prevent the onset of disease. Whilst cell cycle phase and the chromatin context largely dictate which repair pathway is utilised to restore damaged DNA, it has been recently shown that nuclear actin filaments play a major role in clustering DNA breaks to facilitate DSBR by homologous recombination (HR). However, the mechanism with which nuclear actin and the different actin nucleating factors regulate HR is unclear. Interestingly, patients with biallelic mutations in the actin nucleating factor DIAPH1 exhibit a striking overlap of clinical features with the HR deficiency disorders, Nijmegen Breakage Syndrome (NBS) and Warsaw Breakage Syndrome (WABS). This suggests that DIAPH1 may play a role in regulating HR and that some of the clinical deficits associated with DIAPH1 mutations may be caused by an underlying DSBR defect. In keeping with this clinical similarity, we demonstrate that cells from DIAL (DIAPH1 Loss-of-function) Syndrome patients display an HR repair defect comparable to loss of NBS1. Moreover, we show that this DSBR defect is also observed in a subset of patients with Baraitser-Winter Cerebrofrontofacial (BWCFF) syndrome associated with mutations in ACTG1 (γ-actin) but not ACTB (β-actin). Lastly, we demonstrate that DIAPH1 and γ-actin promote HR-dependent repair by facilitating the relocalisation of the MRE11/RAD50/NBS1 complex to sites of DNA breaks to initiate end-resection. Taken together, these data provide a mechanistic explanation for the overlapping clinical symptoms exhibited by patients with DIAL syndrome, BWCFF syndrome and NBS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fdfe6f0c4067d89c1d4aa10073bf8c75525183b" target='_blank'>
              Inherited deficiency of DIAPH1 identifies a DNA double strand break repair pathway regulated by γ-actin
              </a>
            </td>
          <td>
            Beth L. Woodward, Sudipta Lahiri, Anoop S Chauhan, Marcos Rios Garcia, Lucy E Goodley, Thomas L Clarke, Mohinder Pal, A. Agathanggelou, Satpal S. Jhujh, Anil N Ganesh, Fay M Hollins, Valentina Galassi Deforie, R. Maroofian, S. Efthymiou, Andrea Meinhardt, C. Mathew, Michael A Simpson, Heather C Mefford, E. Faqeih, Sergio D. Rosenzweig, Stefano Volpi, G. Di Matteo, C. Cancrini, A. Scardamaglia, Fiona Shackley, E. G. Davies, Shahnaz Ibrahim, Peter D. Arkwright, Maha S. Zaki, Tatjana Stankovic, A. M. Taylor, A. Mazur, Nataliya Di Donato, H. Houlden, Eli Rothenberg, Grant S Stewart
          </td>
          <td>2025-05-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cc40d7bb5bed72dbfa57e9bb607d9efbd812f2" target='_blank'>
              Abstract 6659: Defective lipid homeostasis induced by QKI loss is a mutator on gliomagenesis
              </a>
            </td>
          <td>
            Yinglu Guan, Takashi Shingu, Kaylene Lu, Ece Kilinc, Traver G. Hart, Jianhua Zhang, Yiwen Chen, Betty Y. S. Kim, Jian Hu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chemo-/radioresistance of malignant tumors hampers cancer control and increases patient mortality. Efficient repair of damaged DNA is critical for the maintenance of genomic integrity and fidelity of genetic information. In reverse, increased DNA repair capability in cancer cells contributes to chemo-/radioresistance of malignant tumors. DNA double-strand break (DSB) is the most serious DNA damage and is also the principal molecular basis of radiotherapy. Upon DNA damage, the Ku80 is recruited and forms a critical DNA-PK complex at the DSB sites with Ku70 and the catalytic subunit (DNA-PKcs) to initiate DNA repair. How DNA-PK is assembled and activated is not fully understood. Based on the identification of radiation-reduced Ku80 K568 crotonylation through quantitative global lysine crotonylome analysis, we reveal that Ku80 K568 is crotonylated by p300-CBP-associated factor (PCAF). Upon DNA damage, the K568cr is decrotonylated by HDAC8 (Histone deacetylase 8). Decrotonylation of K568cr empties this site for the subsequent SUMOylation of Ku80 by CBX4. The conversion of Ku80 from K568 crotonylation to SUMOylation facilitates the assembly of DNA-PK complex and autophosphorylation of DNA-PKcs S2056, consequently activating the DSB repair. Moreover, mutation disrupting the post-translational modification (PTM) of Ku80 K568 site sensitizes cancer cells to radiotherapy in tumor-bearing nude mice models. This study elucidates the conversion model between two different forms of PTMs in the regulation of DNA-PK complex assembly and DSB repair, highlighting this model’s potential in controlling chemo-/radioresistance of malignant tumors, as well as expands the atlas of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e28a599caae10368a0a56703d29a0e46b8a2e" target='_blank'>
              Conversion of Ku80 K568 crotonylation to SUMOylation facilitates DNA non-homologous end joining and cancer radioresistance
              </a>
            </td>
          <td>
            Hongling Zhao, Shan-shan Gao, Yang Han, Dafei Xie, Lihui Xuan, Xin Huang, Jinhua Luo, Q. Ran, Gang Li, Hejiang Guo, Weixiang Hu, Jin Jia, Xiaochang Liu, Yuhao Liu, Jinpeng Tan, Chenjun Bai, Yongqing Gu, Teng Ma, Zhongjun Li, Hua Guan, Ruixue Huang, Ping-Kun Zhou
          </td>
          <td>2025-04-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4868141c39dc560c925882ee816da6d52c4deb" target='_blank'>
              Uev1A counteracts oncogenic Ras stimuli in both polyploid and diploid cells
              </a>
            </td>
          <td>
            Qi Zhang, Yunfeng Wang, Xueli Fu, Ziguang Wang, Yang Zhang, Lizhong Yan, Yuejia Wang, Muhan Yang, Dongze Song, Ruixing Zhang, Hongru Zhang, Shian Wu, Shaowei Zhao
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3714058d5e1278c33cbc0e33134728cb5c41b6d7" target='_blank'>
              Abstract 1738: Pan-inhibition of super-enhancer-driven oncogenic transcription by novel synthetic ecteinascidins yields potent anti-cancer activity
              </a>
            </td>
          <td>
            Julian Obid, Max Cigrang, Maguelone Nogaret, Frédéric Coin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Higher-order chromatin structure and DNA methylation are critical for gene regulation, but how these vary across the human body remains unclear. We performed multi-omic profiling of 3D genome structure and DNA methylation for 86,689 single nuclei across 16 human tissues, identifying 35 major and 206 cell subtypes. We revealed extensive changes in CG and non-CG methylation across almost all cell types and characterized 3D chromatin structure at an unprecedented cellular resolution. Intriguingly, extensive discrepancies exist between cell types delineated by DNA methylation and genome structure, indicating that the role of distinct epigenomic features in maintaining cell identity may vary by lineage. This study expands our understanding of the diversity of DNA methylation and chromatin structure and offers an extensive reference for exploring gene regulation in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc53907293087756d961bef23e16d94d527ce994" target='_blank'>
              Human Body Single-Cell Atlas of 3D Genome Organization and DNA Methylation
              </a>
            </td>
          <td>
            Jingtian Zhou, Yue Wu, Hanqing Liu, Wei Tian, R. Castanon, Anna Bartlett, Zuolong Zhang, Guocong Yao, Dengxiaoyu Shi, Ben Clock, Samantha Marcotte, Joseph R. Nery, Michelle Liem, Naomi Claffey, Lara Boggeman, Cesar Barragan, R. A. e Drigo, Annika K. Weimer, Minyi Shi, Johnathan Cooper-Knock, Sai Zhang, Michael P. Snyder, Sebastian Preissl, Bing Ren, Carolyn O’Connor, Shengbo Chen, C. Luo, Jesse R. Dixon, J. Ecker
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is the leading cause of cancer-related mortality worldwide owing to the lack of effective and early diagnostic tools and therapeutic approaches. DNAJC7, a member of the DnaJ heat shock family, is crucial in protein folding and stability; however, its specific functions and mechanisms in HCC remain unclear. Objective This study aimed to explore the role of DNAJC7 in HCC progression and evaluate its potential clinical significance as a prognostic marker. Methods Public databases (TCGA, ICGC, GEO, and TIMER) were used to assess DNAJC7 expression, correlations with clinical parameters, and related signaling pathways. Proliferation, migration, invasion, and cell cycle assays were performed to evaluate the function of DNAJC7 in HCC. Immune infiltration and associations with checkpoint proteins were analyzed using TIMER, and a Gene Set Enrichment Analysis (GSEA) was used to explore enriched pathways. Results DNAJC7 expression was higher in HCC tissues than in adjacent normal tissues and was associated with advanced malignancy and poor prognosis, including a lower overall survival, progression-free survival, and disease-free survival. DNAJC7 knockdown resulted in reduced malignant behavior of HCC cells, leading to S-phase cell cycle arrest. Increased DNAJC7 expression was associated with immune cell infiltration and the presence of immunological checkpoint molecules, including CTLA4 and PD-1. GSEA highlighted the activation of key pathways, including WNT signaling and cell cycle regulation. Conclusion DNAJC7 regulates tumor cell proliferation, migration, invasion, and immune evasion by acting as an oncogene in HCC. It can serve as a diagnostic and prognostic biomarker and potential treatment target for HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06202-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5d2c109594fa970efe3bba40f733c0078c3d83a" target='_blank'>
              Increased expression of DNAJC7 promotes the progression of hepatocellular carcinoma by influencing the cell cycle and immune microenvironment
              </a>
            </td>
          <td>
            Jiaxing Chen, Zhizhao Yang, Yongqiang Cui, Zhilei Zhao, D. Deng, Zhihao Fu, Xiao Zhang
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial organization of chromatin plays a critical role in gene transcription, but connecting population-averaged HiC data to functional outcomes remains a challenge. We present a computational framework linking HiC contact map to gene transcription. Utilizing a bead-spring polymer model informed by HiC contact maps, we generate an ensemble of 3D conformations for a given genomic locus. These conformations are then coupled to gene transcription levels through a Markov chain model, with transition rates derived from molecular dynamics simulations. The efficacy of this framework is demonstrated by simulating the perturbation of a CTCF-mediated TAD boundary, impacting the expression of sox9 and kcnj2. Our model quantitatively reproduces experimentally observed changes in gene expression, revealing that the increased kcnj2 transcription is a consequence of enhancers within the sox9 TAD becoming accessible upon boundary disruption. Quantifying enhancer impact, our model can also identify functional enhancers. This framework enhances our understanding of the relationship between chromosome spatial architecture and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa702759a3ba01ec0ae87f9b6149d8c7b72f89" target='_blank'>
              Predicting gene expression changes from chromatin structure modification
              </a>
            </td>
          <td>
            Swayamshree Senapati, Inayat Ullah Irshad, Ajeet K Sharma, Hemant Kumar
          </td>
          <td>2025-04-15</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Micronuclei exhibit defective proteomes rendering their chromatin vulnerable to fragmentation. This fragmentation process, known as chromothripsis, promotes tumorigenesis by catalysing the activation of oncogenes and the silencing of tumor suppressors. With this role in mind, micronuclei serve as promising targets for therapeutic intervention. This review will explore recent discoveries regarding how micronuclei form, their function in catalysing chromothripsis and how chromothripsis provides a selective advantage for cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ac4a5ea94375865c4d842a9f93ca345cb31b4e0" target='_blank'>
              Cancer cells' chamber of secrets: the link between micronuclei, chromothripsis and malignancy.
              </a>
            </td>
          <td>
            Truman McNeil, Sweta Sikder, Yamini Dalal
          </td>
          <td>2025-05-01</td>
          <td>Open biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct “fragmentomic” characteristics that are related with their cellular origins. Despite the wealth of low-pass whole genome sequencing data from recent cancer liquid biopsy studies, somatic variants within this data have often been overlooked without detailed characterization. Here, we reported that in low-pass sequencing data, somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, our investigation into cfDNA fragmentomics showed that even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like traits, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilized artificial intelligence to integrate genomic, fragmentomic, and epigenomic features, developing diagnostic models named FreeSV (Fragmentomic and epigenomic examinations of Somatic Variants in cfDNA) and FreeSV-m. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features were also included, the AUCs of FreeSV-m model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64ed6f9ef46f8ac541b4cf9a70df9227b7d0f90" target='_blank'>
              Cancer-like fragmentomic characteristics of somatic variants in cell-free DNA
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Wanqiu Wang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d143808abe3792603e16e2ab02c9eaad5e2640" target='_blank'>
              Abstract 2409: Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways
              </a>
            </td>
          <td>
            Joel Pardo, N. Temiz, Douglas Yee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72e3fdbe0004a1df71caab7b38b9a83bc3ac31c0" target='_blank'>
              Abstract 2750: Alternative enhancer usage of PER2 locus regulating 3D genome architecture in epithelial-mesenchymal states
              </a>
            </td>
          <td>
            Yi-Chia Chiu, Qing You Pang, Tuan Zea Tan, Wendy W. Hwang-Verslues, Ruby Y. J. Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks arising from base excision repair (BER) during the misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination–deficient (HRD) cells. At templated AP sites, cross-links between the DNA and 5-hydroxymethylcytosine binding, embryonic stem cell–specific (HMCES) regulate replication fork speed, avoiding cytotoxic double-strand breaks. While the role of HMCES at the template DNA strand is well studied, its effects on nascent DNA are not. We provide evidence that HMCES–DNA-protein cross-links (DPCs) are detrimental to the BER-mediated removal of 5-hydroxymethyl-2′-deoxycytidine (5hmdC)–derived 5-hydroxymethyl-2′-deoxyuridine from replication forks. HRD cells have heightened HMCES-DPCs, which increase further upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP sites. HMCES depletion substantially suppresses 5hmdC-mediated replication fork defects, chromosomal aberrations, and cell death in HRD cells. This reveals that HMCES-DPCs are a source of BER-initiated single-stranded DNA gaps, which indicates that endogenous DPCs contribute to genomic instability in HRD tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622156d932ecc5d7d331e4371a1b29a1f41c3ef" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination–deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martín, Marta del Rio Oliva, Yodhara Wijesekara Hanthi, Sara Berrada, Raimundo Freire, Jean-Yves Masson, José Carlos Reyes, Vincenzo Costanzo, Iván V. Rosado
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/545b6d8dbebbe17da0f379631a55f84485df7054" target='_blank'>
              The Significance of Chromosome Conformation Capture in 3D Genome
Architecture Comprehension
              </a>
            </td>
          <td>
            Emre Sefer
          </td>
          <td>2025-04-22</td>
          <td>Current Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5717acd2d0393973aea7e3766b9be2a2a3a6876f" target='_blank'>
              One for All and All for One: New Insights into Enhancers Driving MYC Dysregulation.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2025-04-16</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa24230eaf15e64077b2640e42b580f6dcef4855" target='_blank'>
              Abstract 2838: Modeling putative oncogenic variants of human topoisomerase II by genome editing of a near haploid cell line
              </a>
            </td>
          <td>
            K. C. Nitiss, Brigid Conroy, M. N. McCoy, J. Nitiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9f435d244c6af037296ac91414cad5547923bc9" target='_blank'>
              Abstract 1465: RAD52 and ERCC6L/PICH have a compensatory relationship for genome stability
              </a>
            </td>
          <td>
            Beth Osia, Arianna Merkell, F. Lopezcolorado, Xiaoli Ping, Jeremy M. Stark
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Homology, or relationship among characters by common descent, has been notoriously difficult to assess for many morphological features, and cell types in particular. The ontogenetic origin of morphological traits means that the only physically inherited information is encoded in the genomes. However, the complexity of the underlying gene regulatory network and often miniscule changes that can impact gene expression, make it practically impossible to postulate a clear demarcation line for what molecular signature should "define" a homologous cell type between two deeply branching animals. In this Hypothesis article, we propose the use of the recently characterized irreversible genomic states, that occur after chromosomal and sub-chromosomal mixing of genes and regulatory elements, to dissect regulatory signatures of each cell type into irreversible and reversible configurations. While many of such states will be non-functional, some may permanently impact gene expression in a given cell type. Our proposal is that such evolutionarily irreversible, and thus synapomorphic, functional genomic states can constitute a criterion for the timing of the origin of deep evolutionary cell type homologies. Our proposal thus aims to close the gap between the clearly defined homology of the individual genomic characters and their genomic states to the homology at the phenotypic level through the identification of the underlying evolutionarily irreversible and regulatory linked states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1c7265ffd0750ec9e349f309b4f00f20641d114" target='_blank'>
              The application of irreversible genomic states to define and trace ancient cell type homologies
              </a>
            </td>
          <td>
            Oleg Simakov, Günter P. Wagner
          </td>
          <td>2025-05-03</td>
          <td>EvoDevo</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is crucial for maintaining genomic stability by repairing DNA damage. Despite its importance, HRR's role in cancer progression is not fully elucidated. Here, this work shows that nuclear-localized branched-chain α-ketoacid dehydrogenase kinase (BCKDK) acts as a modulator of HRR, promoting cell resistance against DNA damage-inducing therapy in breast cancer. Mechanistically, this work demonstrates that BCKDK is localized in the nucleus and phosphorylates RNF8 at Ser157, preventing the ubiquitin-mediated degradation of RAD51, thereby facilitating HRR-mediated DNA repair under replication stress. Notably, aberrant expression of the BCKDK/p-RNF8/RAD51 axis correlates with breast cancer progression and poor patient survival. Furthermore, this work identifies a small molecule inhibitor of BCKDK, GSK180736A, that disrupts its HRR function and exhibits strong tumor suppression when combined with DNA damage-inducing drugs. Collectively, this study reveals a new role of BCKDK in regulating HRR, independent of its metabolic function, presenting it as a potential therapeutic target and predictive biomarker in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac4a9c9be6494a377793b137fd2fa59c3a2efc2e" target='_blank'>
              Nuclear-Localized BCKDK Facilitates Homologous Recombination Repair to Support Breast Cancer Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Haiying Liu, Jiaqian Feng, Tingting Pan, Pinggen Zhang, Ling Ye, Zetan Jiang, Zilong Zhou, Qiankun Mao, Jian Li, Xinyi Yang, P. Gao, De Huang, Huafeng Zhang
          </td>
          <td>2025-04-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background
 Spatial and temporal control of DNA damage response pathways after DNA damage is crucial for maintenance of genomic stability. Ataxia telangiectasia mutated (ATM) protein plays a central role in DNA damage response pathways. The chain of events following induction of DNA damage that results in full activation of ATM is still evolving. Here we set out to explore the role of CREB-binding protein (CBP), a histone acetyltransferase (HAT), in DNA damage response, particularly in the ATM activation pathway. Results In response to DNA damage, CBP is stabilized and is recruited at sites of DNA double-strand breaks where it acetylates ATM and promotes its kinase activity. Cells deficient in CBP display an impairment in DNA double-strand break repair and high sensitivity to chemo- and radiotherapy. Importantly, re-expressing CBP’s HAT domain in CBP-deficient cells restores the DNA repair capability, demonstrating the essential role of CBP’s HAT domain in repairing DNA double-strand breaks. Conclusions Together, our findings shed the light on CBP as a key participant in the ATM activation pathway and in the subsequent repair of DNA double-strand breaks, which may serve as a potential target to modulate the cellular response to DNA damaging agents in cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03528-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a883f9f1e4c2e6460111ee2a491dce39c437792" target='_blank'>
              The histone acetyltransferase CBP participates in regulating the DNA damage response through ATM after double-strand breaks
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, Samrein B M Ahmed, Iman M Talaat, L. Lozon, S. Mouffak, Timo Gemoll, Wael Y Mansour, Raafat El-Awady
          </td>
          <td>2025-04-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246dd9457357ea0f8f4372c88fc8b6dbeeeee6a1" target='_blank'>
              Extrusion-modulated DnaA activity oscillations coordinate DNA replication with biomass growth
              </a>
            </td>
          <td>
            Dengjin Li, Hai Zheng, Yang Bai, Zheng Zhang, Hao Cheng, Xiongliang Huang, Ting Wei, Arieh Zaritsky, Terence Hwa, Chenli Liu
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54ccd0aa3ae124063240ed82d6af46f8d425d72" target='_blank'>
              Aneuploidy generates enhanced nucleotide dependency and sensitivity to metabolic perturbation.
              </a>
            </td>
          <td>
            R. Magesh, Arshia N Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, Jessica B Spinelli, S. Elledge, E. Watson
          </td>
          <td>2025-05-05</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Homologous recombination deficiency (HRD) is a predictive biomarker for efficacy of PARP (poly ADP-ribose polymerase) inhibition and platinum chemotherapy for cancer patients but remains challenging to detect. The discovery of patients without pathogenic mutations in known HR genes but exhibiting genomic scars indicative of HRD led to the FDA approval of the first scar-based HRD test. Despite advancements in whole genome sequencing (WGS) and integration of large training datasets with machine learning models, current methods lack the sensitivity required for detecting HRD scars in low tumor purity samples, especially in liquid biopsies. Here, we describe DirectHRD, a genomic scar-based HRD classifier based on WGS. Compared to other WGS-based methods, DirectHRD exclusively utilizes a highly specific type of HRD scar—small deletions with microhomology—and its associated signatures in a probabilistic framework. We applied DirectHRD to 501 tumor and 90 cell-free DNA (cfDNA) samples from 4 cancer types: breast, ovarian, prostate, and pancreas. Among all 501 tumor biopsies, DirectHRD achieved 100% detection of HRD with high specificity (>90%). Across all 90 cfDNA samples, the method achieved an area under the curve of 0.87 and demonstrated the ability to detect HRD at tumor fractions as low as 1%, making it 10 times more sensitive than state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44b1868468070d95d0fd70a9ed4a011186d9a1" target='_blank'>
              DirectHRD enables sensitive scar-based classification of homologous recombination deficiency
              </a>
            </td>
          <td>
            Ruolin Liu, Eugenia Roberts, Heather A. Parsons, Elizabeth H. Stover, A. Choudhury, J. Rhoades, T. Blewett, David D. Yang, Joyce F. Liu, Erica L. Mayer, V. Adalsteinsson
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Pluripotent stem cells (PSC) are known to recurrently acquire genetic aberrations including chromosomal aneuploidy during long-term culture of which the consequences remain largely unknown. A recent study reported that xenografting of aneuploid mouse embryonic stem cells (mESC) gives rise to teratomas containing undifferentiated elements with metastatic capacity. This contrasts with benign teratomas composed of fully differentiated tissues from the three embryonic germ layers obtained by xenografting diploid mESCs. The highlighted study thus proposes that general aneuploidy may drive metastasis of PSC-derived teratomas, thus directly correlating it with malignant potential of PSCs. Main The aneuploid mESC lines used in the aforementioned study showed trisomies of chromosomes 6, 8, 11, 15 or combinations thereof. Interestingly, several of these trisomies, specifically in chromosomes 6, 8, and 11, constitute well-known chromosomal aberrations recurrently acquired in mESCs upon (long-term) in vitro culture. This is suggested to be driven by numerous (proto)oncogenes in these regions. Notably, recurrent chromosomal aberrations are also observed in human PSCs (hPSCs), mostly in chromosome 1, 12, 17, 20 and X, and have similarly been associated with the acquisition of a malignant phenotype in vitro, posing a risk to their potential clinical application. There is synteny between the chromosomes recurrently affected by such aberrations in mouse and human PSCs, namely mouse chromosome 6 and human chromosome 12 as well as mouse and human chromosome 1, suggesting a chromosome-specific phenomenon that may transcend the species barrier. Additionally, in the field of cancer, malignant (but not benign) human germ cell tumors (hGCTs), pathologically similar to the tumors derived from the aforementioned aneuploid mouse PSCs, are characterized by supernumerical copies of chromosome 12, highlighting the additional clinical relevance of these findings and the aneuploid mESC models for GCT research and treatment. Conclusions We suggest that chromosome-specific trisomies, rather than general aneuploidy, might drive teratoma metastasis upon mESC xenografting in vivo. We also observed indications of synteny between the recurrent chromosomal aberrations among human and mouse PSCs, suggesting potentially common intraspecies selection mechanisms. We finally reiterate the similarities observed between the PSC and GCT field related to chromosomal aberrations and malignancy, highlighting the relevance of these experimental models in both research fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9961bd4e5533b7648a3a993149a92ce0795a5ee" target='_blank'>
              Chromosome-specific aberrations, rather than general aneuploidy, may drive mouse embryonic stem cell-derived teratoma metastasis
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, Sanne Hillenius, T. F. Eleveld, Daniela C F Salvatori, Leendert H J Looijenga
          </td>
          <td>2025-04-15</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d11e895cea0cf1e967c3e4adda2d90a1183e147" target='_blank'>
              Abstract 215: A novel framework for deciphering 3D genome structure in colorectal cancer
              </a>
            </td>
          <td>
            Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Addiction to oncogene-rewired transcriptional networks is a therapeutic vulnerability in cancer cells, underscoring a need to better understand mechanisms that relay oncogene signals to the transcriptional machinery. Here, using human and mouse T cell acute lymphoblastic leukemia (T-ALL) models, we identify an essential requirement for the endosomal sorting complex required for transport protein CHMP5 in T-ALL epigenetic and transcriptional programming. CHMP5 is highly expressed in T-ALL cells where it mediates recruitment of the coactivator BRD4 and the histone acetyl transferase p300 to enhancers and super-enhancers that enable transcription of T-ALL genes. Consequently, CHMP5 depletion causes severe downregulation of critical T-ALL genes, mitigates chemoresistance and impairs T-ALL initiation by oncogenic NOTCH1 in vivo. Altogether, our findings uncover a non-oncogene dependency on CHMP5 that enables T-ALL initiation and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768bb462ad3f5f7bfdc0d91a1f7eb42525d3db0" target='_blank'>
              The ESCRT protein CHMP5 promotes T cell leukemia by enabling BRD4-p300-dependent transcription
              </a>
            </td>
          <td>
            Katharine Umphred-Wilson, Shashikala Ratnayake, Qianzi Tang, Rui Wang, S. Chaudhary, Devaiah N Ballachanda, Josephine Trichka, Jan Wisniewski, Lan Zhou, Qing-rong Chen, D. Meerzaman, Dinah S. Singer, S. Adoro
          </td>
          <td>2025-05-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Enhancers and promoters interact in three-dimensional (3D) chromatin structures to regulate gene expression. Here we characterize the mechanisms that drive the formation and function of these structures in a lymphoid-to-myeloid transdifferentiation system. Based on analyses at base pair resolution, we demonstrate a close correlation between binding of regulatory proteins, formation of chromatin interactions and gene expression. Multi-way interaction analyses and computational modeling show that tissue-specific gene loci are organized into chromatin hubs, characterized by cooperative interactions between multiple enhancers, promoters and CTCF-binding sites. While depletion of CTCF strongly impairs the formation of these chromatin hubs, the effects of CTCF depletion on gene expression are modest and can be explained by rewired enhancer-promoter interactions. These findings demonstrate a role for enhancer-promoter interactions in gene regulation that is independent of cooperative interactions in chromatin hubs. Together, these results contribute to our understanding of the structure-function relationship of the genome during cellular differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc2107839027b3ea20b094ebcb742df75918711" target='_blank'>
              CTCF depletion decouples enhancer-mediated gene activation from chromatin hub formation.
              </a>
            </td>
          <td>
            Magdalena A. Karpinska, Yi Zhu, Zahra Fakhraei Ghazvini, Shyam Ramasamy, Mariano Barbieri, T. B. N. Cao, Natalie Varahram, Abrar Aljahani, Michael Lidschreiber, A. Papantonis, A. M. Oudelaar
          </td>
          <td>2025-05-13</td>
          <td>Nature structural & molecular biology</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genotoxic therapies such as ionizing radiation eliminate cancer cells by inducing extensive DNA damage but often cause normal tissue toxicity, including cutaneous injury. Extrachromosomal circular DNA (eccDNA) refers to circular DNA fragments outside the chromosomal context, with their formation and persistence linked to DNA damage repair and genomic instability. Despite growing recognition of eccDNA in oncogenesis, its role under genotoxic stress in normal tissues remains poorly understood. Here, eccDNA is profiled in irradiated rat skin using Circle-seq, identifying alterations in eccDNA number and composition. Specifically, radiation induced circle17:44148731-48208624, in which vacuolar protein sorting 41 homolog (VPS41) is the sole radiation-induced amplification gene by semiquantitative PCR and gel electrophoresis. The findings show that eccDNA or VPS41 overexpression reduces radiation-induced skin injury (RISI) in vitro and in vivo. Proteomic and interaction analyses identified metastasis suppressor kangai-1 (KAI1) as a VPS41-interacting partner. Notably, VPS41 overexpression promotes KAI1 lysosomal degradation, protecting against radiation-induced apoptotic cell death. Peptide array analysis pinpoints the VPS41-KAI1 interaction through the K263 residue, consistent with AlphaFold prediction. The findings uncover a novel mechanism in which radiation-induced eccDNA, specifically VPS41, mitigates skin injury by modulating KAI1 degradation. This study highlights the role of eccDNA in cellular defense, providing strategies to enhance tissue resilience to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821c77d7a2a02d4160cf107b29b6628f9d6cb1ed" target='_blank'>
              EccDNA-Driven VPS41 Amplification Alleviates Genotoxic Stress via Lysosomal KAI1 Degradation.
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Jinchen He, Bin Song, Hao Bai, Fengdi Jiang, Yining Zhang, Qian Li, Tao Yan, Wenling Tu, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background
 The incidence of early-onset colorectal cancer (EOCRC) (< 50 years) has been steadily rising, with a parallel increase in metastatic and invasive cases. To elucidate the molecular mechanisms underlying this aggressive phenotype, we performed comprehensive multi-omics profiling to delineate the distinct features of EOCRC, with a focus on key drivers of metastatic and invasive potential. Methods We initially characterized the genome, epigenome, and transcriptome of tumors from 515 (69 EOCRC and 446 late-onset CRC [LOCRC]) cases in The Cancer Genome Atlas. Key candidate molecules were further validated using RNA-seq and scRNA-seq data. Multi-omics profiling revealed PIWIL1/piRNA as a hallmark of EOCRC, with further validation through in vitro functional assays, transcriptomic profiling, and Kaplan-Meier survival analysis. Results EOCRC demonstrated a mutational landscape similar to that of LOCRC, with comparable oncogenic driver mutations and somatic copy-number alterations. However, EOCRC exhibited a higher frequency of deletion in chromosomes 6, 15, and 19 regions, along with metabolic reprogramming favoring aerobic glycolysis and lipid metabolism. Integrative transcriptomic and DNA methylation analyses identified six EOCRC-specific molecules, including PIWIL1. Notably, PIWIL1 was mainly expressed in epithelial cells, with lower expression in EOCRC versus LOCRC. Its downstream piRNAs (FR019019, FR019089, and FR132045) were also downregulated in EOCRC. Functional experiments demonstrated that FR019089/FR019019 overexpression suppressed migration and invasion. Clinically, low FR019089 levels correlated with significantly shorter progression-free and overall survival in EOCRC patients. Additionally, downstream pathways of FR019089 and FR019019 overexpression were enriched in anti-cancer-related signaling pathways. Conclusions Our multi-omics approach yields novel insights into the molecular underpinnings of EOCRC and we characterize the role of PIWIL1-associated piRNAs in modulating EOCRC metastasis and invasion. FR019089 shows promise as a prognostic biomarker with potential clinical utility in the risk stratification and management of EOCRC patients. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04074-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c18e83c7bc9618c0b53a164ccc0c62b690301cf6" target='_blank'>
              Integration of multi-omics data to unveil the molecular landscape and role of piRNAs in early-onset colorectal cancer
              </a>
            </td>
          <td>
            Siyun Zhou, Lili Yu, Jianhui Zhao, Q. Xiao, Jing Sun, Lijuan Wang, Yuan Zhou, Yadong Lu, Malcolm G Dunlop, E. Theodoratou, Honghe Zhang, Kefeng Ding, Xue Li
          </td>
          <td>2025-04-29</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability.
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are widely existing stable DNA secondary structures in mammalian cells. A long-standing hypothesis is that timely resolution of G4s is needed for efficient and faithful DNA replication. In vitro, G4s may be unwound by helicases or alternatively resolved via DNA2 nuclease mediated G4 cleavage. However, little is known about the biological significance and regulatory mechanism of the DNA2-mediated G4 removal pathway. Here, we report that DNA2 deficiency or its chemical inhibition leads to a significant accumulation of G4s and stalled replication forks at telomeres, which is demonstrated by a high-resolution technology: Single molecular analysis of replicating DNA (SMARD). We further identify that the DNA repair complex MutSα (MSH2-MSH6) binds G4s and stimulates G4 resolution via DNA2-mediated G4 excision. MSH2 deficiency, like DNA2 deficiency or inhibition, causes G4 accumulation and defective telomere replication. Meanwhile, G4-stabilizing environmental compounds block G4 unwinding by helicases but not G4 cleavage by DNA2. Consequently, G4 stabilizers impair telomere replication and cause telomere instabilities, especially in cells deficient in DNA2 or MSH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica D Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29460fc2cc42604173072239d7b085ed803a493" target='_blank'>
              Abstract 4305: Targeting the exonuclease domain of POLE to induce synthetic lethality in highly aneuploidic tumor cells
              </a>
            </td>
          <td>
            Meng He, Eun Jung Kim, Yangmeng Wang, A. Chibly, Kristel M. Dorighi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87641a322ac1dbfc7184a0c61b91e897e328415f" target='_blank'>
              Human papillomavirus integration induces oncogenic host gene fusions in oropharyngeal cancers.
              </a>
            </td>
          <td>
            N. Khan, Keiko Akagi, Shiming Jiang, J. D. Dunn, Bo Jiang, Weihong Xiao, Madison P O'Hara, Li Shen, Qi Wang, Vakul Mohanty, Jing Wang, Sara Goodwin, Jamie L Hutchins, Kevin R. Coombes, Jagannadha K Sastry, D. Symer, M. Gillison
          </td>
          <td>2025-05-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Given their rarity and diversity, a fundamental understanding of the genomic underpinnings for many sarcoma subtypes is still lacking. To better define the molecular landscape of this group of diseases, we perform matched whole exome sequencing and RNA sequencing on a cohort of 1340 sarcoma tumor specimens. We identify recurrent somatic mutations and observe an increased mutational burden in metastatic vs. primary samples (p < 0.001). We observe frequent copy number alterations including whole genome doubling, with this feature being more common in metastatic tumors (p = 0.026). Estimation of immune cell abundances followed by hierarchical clustering identifies five immune subtypes ranging from low to high and we observe inferior overall survival in immune deplete clusters compared to immune enriched (p < 0.01). Interestingly, GIST predominantly form a distinct “immune intermediate” cluster that is marked by a specific enrichment for NK cells (FDR < 0.01).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09dedff6b3501b2376796d9aaef10264491ab2ba" target='_blank'>
              Genomic, transcriptomic, and immunogenomic landscape of over 1300 sarcomas of diverse histology subtypes
              </a>
            </td>
          <td>
            Alex C. Soupir, Oscar E. Ospina, Oliver Hampton, Michelle L Churchman, Michael Radmacher, D. Hedges, David M. McKean, Phaedra Agius, Saman Zeeshan, Nathan D Seligson, Raphael Pollock, D. Liebner, James L. Chen, G. Tinoco, B. Salhia, Martin D McCarter, B. Wilky, Benjamin J. Miller, M. Cavnar, John S. Groundland, Bryan P. Schneider, Gregory Riedlinger, Stephen B Edge, C. Moskaluk, Kenneth Cardona, A. R. Naqash, Ricardo J. Gonzalez, J. Mullinax, David M Joyce, O. Binitie, G. Douglas Letson, A. Naghavi, M. Druta, Damon R Reed, E. Siegel, Jamie K Teer, Brooke L. Fridley, Andrew S Brohl
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair.
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="A prominent feature of eukaryotic chromosomes are centromeres, which are specialized regions of repetitive DNA required for faithful chromosome segregation during cell division. In interphase cells, centromeres are non-randomly positioned in the three-dimensional space of the nucleus in a cell type-specific manner. The functional relevance and the cellular mechanisms underlying this localization are unknown, and quantitative methods to measure distribution patterns of centromeres in 3D space are needed. Here, we developed an analytical framework that combines sensitive clustering metrics and advanced modeling techniques for the quantitative analysis of centromere distributions at the single-cell level. To identify a robust quantitative measure for centromere clustering, we benchmarked six metrics for their ability to sensitively detect changes in centromere distribution patterns from high-throughput imaging data of human cells, both under normal conditions and upon experimental perturbation of centromere distribution. We found that Ripley’s K function has the highest accuracy with minimal sensitivity to variations in the number of centromeres, making it the most suitable metric for measuring centromere distributions. As a complementary approach, we also developed and validated spatial models to replicate centromere distribution patterns, and we show that a radially shifted Gaussian distribution best represents the centromere patterns seen in human cells. Our approach creates tools for the quantitative characterization of spatial centromere distributions with applications in both targeted studies of centromere organization and unbiased screening approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174a17b1ef834044537c7e22b9436da4a482c609" target='_blank'>
              Simulation and Quantitative Analysis of Spatial Centromere Distribution Patterns
              </a>
            </td>
          <td>
            Adib Keikhosravi, Krishnendu Guin, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-03-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            V. Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05f154cb832f6f7744c5bac28235208f5d152b7" target='_blank'>
              Mitochondrial DNA copy numbers in gastric cancer tissues: a possible biomarker for estimating cancer progression.
              </a>
            </td>
          <td>
            Hisanori Fukunaga, Mayuko Fukunaga
          </td>
          <td>2025-04-23</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00eb3a597363b589454ef131b68d47400d8e5ecf" target='_blank'>
              Mechanisms and regulation of DNA end resection in the maintenance of genome stability.
              </a>
            </td>
          <td>
            Raphael Ceccaldi, Petr Cejka
          </td>
          <td>2025-03-25</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6462361aae6f3cbb7d9db079e7f43e5e3ea1a7f1" target='_blank'>
              Abstract 1448: An evolutionarily conserved chromatin remodeling factor, FACT, in regulation of chromatin, gene expression and cancer
              </a>
            </td>
          <td>
            Priyanka Barman, S. Bhaumik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8baa6a9cc6acf1a8fc2f2802c0c1373dc7878226" target='_blank'>
              Steroid-Modulated Transcription Synergistically Forms DNA Double-Strand Breaks With Topoisomerase II Inhibitor.
              </a>
            </td>
          <td>
            Ying Zhao, Tetsuro Hisayoshi, Doudou Zhang, Saaya Suzuki, Takashi Watanabe, Atsuo Kobayashi, Qianqian Guo, Y. Momozawa, Takashi Shimokawa, Shunsuke Kato, Yoshio Miki, Shigeaki Sunada
          </td>
          <td>2025-04-15</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Mobile genetic elements evade CRISPR–Cas adaptive immunity by encoding anti-CRISPR proteins (Acrs). Acrs inactivate CRISPR–Cas systems via diverse mechanisms but generally coevolve with a narrow subset of Cas effectors that share high sequence similarity. Here, we demonstrate that AcrIIA11 inhibits Streptococcus pyogenes (Sp), Staphylococcus aureus (Sa), and Francisella novicida (Fn) Cas9s in vitro and in human cells. Single-molecule imaging reveals that AcrIIA11 hinders SaCas9 target search by reducing its diffusion on nonspecific DNA. DNA cleavage is inhibited because the AcrIIA11:SaCas9 complex binds to protospacer adjacent motif (PAM)-rich off-target sites, preventing SaCas9 from reaching its target. AcrIIA11 also greatly slows down DNA cleavage after SaCas9 reaches its target site. A negative-stain electron microscopy reconstruction of an AcrIIA11:SaCas9 RNP complex reveals that the heterodimer assembles with a 1:1 stoichiometry. Physical AcrIIA11–Cas9 interactions across type IIA and IIB Cas9s correlate with nuclease inhibition and support its broad-spectrum activity. These results add a kinetic inhibition mechanism to the phage-CRISPR arms race.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae6d0bcafc0d2a35bdeaef7ab05074639e57301" target='_blank'>
              Mechanism of Cas9 inhibition by AcrIIA11
              </a>
            </td>
          <td>
            Kaylee E. Dillard, Hongshan Zhang, Lianne Z Dubbs, Chia-Wei Chou, Cynthia Terrace, Kamyab Javanmardi, Wantae Kim, Kevin J Forsberg, Ilya J Finkelstein
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purifying selection that limits the transmission of harmful mitochondrial DNA (mtDNA) mutations has been observed in both human and animal models. Yet the precise mechanism underlying this process remains undefined. Here, we present a highly specific and efficient in situ imaging method capable of visualizing mtDNA variants that differ by only a few nucleotides at single-molecule resolution in Drosophila ovaries. Using this method, we revealed that selection primarily occurs within a narrow developmental window during germline cysts differentiation. At this stage, the proportion of the deleterious mtDNA variant decreases without a reduction in its absolute copy number. Instead, the healthier mtDNA variant replicates more frequently, thereby outcompeting the co-existing deleterious variant. These findings provide direct evidence that mtDNA selection is driven by replication competition rather than active elimination processes, shedding light on a fundamental yet previously unresolved mechanism governing mitochondrial genome transmission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a50553c510c0eaecac058a100fd3edea07806d1" target='_blank'>
              Replication competition drives the selective mtDNA inheritance in Drosophila ovary
              </a>
            </td>
          <td>
            Cheng Zhang, Zhe Chen, Hansong Ma, Hong Xu
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aberrations in the regulatory genome play a pivotal role in population-level disease predisposition. Annotation of the regulatory regions using appropriate primary tissues - instead of cell lines affected by selection and other confounding factors - could shed new light into mechanisms underlying common conditions. We test this approach in uterine leiomyomas, highly prevalent benign neoplasms of the myometrium, by creating 15-state chromatin annotations for myometrium and uterine leiomyomas. Integration with RNA-seq, ATAC-seq, HiChIP and methylation data enables us to compare the epigenomes of myometrium and ULs with distinct driver mutations, highlighting the role of bivalent regions in the neoplastic process. Subsequently, a genome wide association study meta-analysis is performed, using three different cohorts. Disease association loci are enriched at active chromatin, especially at enhancers, and harbor tumor- and driver mutation-specific chromatin states. At SATB2 locus we show the effect of the risk genotype already in the normal tissue. Integration of genome-wide association studies and deep regulatory genomics data from the correct tissue type represents a powerful approach in understanding population-level disease predisposition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72f1ed6a98b06ee18a001d9ff397770c4c614283" target='_blank'>
              Chromatin state origins of uterine leiomyoma
              </a>
            </td>
          <td>
            Maritta Räisänen, Eevi Kaasinen, Maija Jäntti, Aurora Taira, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, D. Berta, Niko Välimäki, Lauri A Aaltonen
          </td>
          <td>2025-05-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5545243aab210759476cd520384aa54505acdf5" target='_blank'>
              SAUNA - Simulated Annealing for Unique Nucleosome Arrangements from Cell-Free DNA-Sequencing Data
              </a>
            </td>
          <td>
            Maria Rafaela Lambrinos, Tara Rahimi, Tarun Mehra, Michael P Schmid, Nicola Miglino, A. Romanens, Kirsten D. Mertz, Michael Krauthammer, Ronny Nienhold, Andreas Wicki, Zsolt Balázs
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, J. Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Bleomycin (Bleo) is a glycopeptide with potent antitumor activity that induces DNA double-strand breaks (DSBs) through free radical generation, similar to ionizing radiation (IR). Therefore, Bleo is considered a radiomimetic drug. However, differences in DNA repair mechanisms between IR- and Bleo-induced DNA damage have not been fully elucidated. Therefore, in the present study, we examined a panel of repair-deficient human TK6 cell lines to elucidate the relative contributions of individual repair factors. Results Our comprehensive profiling indicated that both non-homologous end joining (NHEJ) and homologous recombination (HR) contributed to DSB repair induced by X-rays and Bleo. Furthermore, tyrosyl-DNA phosphodiesterase (TDP)-related repair was a significant factor for cellular sensitivity to Bleo treatment. TDP1−/−/TDP2−/− cells exhibited greater sensitivity to Bleo than TDP1−/− or TDP2−/− cells, but not to X-rays. In addition, we determined whether TDP2 is involved in the repair of Bleo-induced DSBs using a neutral comet assay. In TDP1-deficient cells, knockout of TDP2 resulted in a significant delay in the repair kinetics of DSBs induced by Bleo, but not by X-rays. Conclusions The contribution of the TDP-related pathway to DSB repair significantly differed between IR and radiomimetic drugs. The discovery of this novel TDP2-dependent repair of DSBs resulting from radiomimetic drug exposure indicates that TDP1 and TDP2 inhibition in combination with radiomimetic drugs represents a strategy for cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00329-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b896deb7b39c22219dcb649c924ec1a29423e547" target='_blank'>
              Role of TDP2 in the repair of DNA damage induced by the radiomimetic drug Bleomycin
              </a>
            </td>
          <td>
            N. Shimizu, Kazuki Izawa, Mubasshir Washif, Ryosuke Morozumi, Kouji Hirota, Masataka Tsuda
          </td>
          <td>2025-03-28</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA cytosine deaminase APOBEC3B (A3B) is one of the endogenous sources of somatic mutations in many types of human cancers and is associated with tumor progression rather than tumorigenesis. However, it remains uncertain whether APOBEC3B‐induced mutations accelerate tumor progression or not. In this paper, we established a mouse model with A3B overexpression and investigated whether the introduction of A3B overexpression accelerates tumor development in Tp53 hemizygous mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b2e1c29be3a5ab7793714efd1b22c6b4cc4521" target='_blank'>
              APOBEC3B Does Not Promote Tumor Progression in Tp53 Hemizygous Mice
              </a>
            </td>
          <td>
            Yoshihito Horisawa, Tadahiko Matsumoto, June Takeda, Yusuke Tashiro, Ryosuke Nomura, Suguru Takeuchi, Yugo Kawai, Yasuhiro Kazuma, Y. Konishi, H. Yamazaki, Hiroyuki Matsui, Kotaro Shirakawa, A. Takaori-Kondo
          </td>
          <td>2025-04-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Centromeres are pivotal for accurate chromosome segregation, yet their regulation and evolutionary dynamics remain poorly understood. Here, we investigate centromeres of the diploid species Gossypium anomalum (Ga, B-genome) that were transferred into tetraploid cotton G. hirsutum (Gh, AD-genome) as either an additional or integrated chromosome, as well as in synthetic allohexaploid (AABBDD) lines. We demonstrate consistent size reduction for all Ga centromeres in the Gh background. Histone modification profiling across 10 marks revealed heightened levels of both active and repressive chromatin marks within the Ga centromeres when transferred into the Gh background, particularly for H3K36me2. The centromeric histone modification perturbation extended into pericentromeric regions, with variable CENH3-binding domains consistently exhibiting a more pronounced increase in histone modification levels compared to stable centromere regions, highlighting the role of histone modification elevation in centromere dynamics. In addition, we observed enhanced chromatin accessibility and the presence of non-B-form DNA motifs, such as A-phased DNA repeats within stable centromere domains that are correlated with centromere stability. Hi-C analysis reveals a reorganized 3D chromatin architecture within the introgression line centromeres, including the formation of new topologically associating domains linked to H3K36me2 dynamics, emphasizing the importance of H3K36me2 in centromere organization. Together, these findings elucidate epigenetic mechanisms underlying centromere composition following intergenomic hybridization and allopolyploid formation, offering insights into centromere evolution in plants and its myriad epigenetic and potentially functional dimensions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cd7b66a7f1ffe44365de20596b44e77ed523aee" target='_blank'>
              Centromere-size reduction and chromatin state dynamics following intergenomic hybridization in cotton
              </a>
            </td>
          <td>
            Jinlei Han, Guanjing Hu, Yan Dai, Xin Zhang, Jingjing Tian, Jialiang Zhou, Xinqi Xu, Qi Chen, Xiaobing Kou, Lei Xu, Xinyu Wu, Ziying Sun, Jiahui Geng, Lin Li, Chenyu Qiu, Teame Gereziher Mehari, Baohua Wang, Hui Zhang, Xinlian Shen, Zhenzhen Xu, Jonathan F. Wendel, Kai Wang
          </td>
          <td>2025-05-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ce08d08aef5a794575f58122e8dce3a63ce74" target='_blank'>
              Aneuploidy alleviates the cell proliferation defect caused by mutations affecting origin licensing in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Christophe de La Roche Saint-André
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences that make up a sizeable fraction of mammalian genomes yet are often tightly repressed by transcriptional and epigenetic mechanisms. During early development, epigenetic reprogramming selectively loosens TE repression, and TE transcription actively contributes to embryogenesis. This raises the question: how can embryos and embryonic stem cells (ESCs) tolerate TE expression without incurring widespread inflammation or DNA damage? Here, we reveal multiple mechanisms that prevent innate immune activation by TE-derived cytosolic DNA, including reduced cGAS/STING expression and signalling, dampening of Type I interferon responses by pluripotency factors, and post-transcriptional restriction of retrotransposition. These layers of protection are essential, as experimental perturbation triggers loss of ESC self-renewal and pluripotency. Our data explain how early development can be permissive to TE expression while safeguarding against harmful effects of TE activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43e345844c8f33145e79c69db85ef993aa97f99" target='_blank'>
              Inhibition of cytosolic DNA sensing and transposon activity safeguards pluripotency
              </a>
            </td>
          <td>
            Ferran Garcia-Llagostera, Audrey L. K. Putman, Alexandros Choromidis, Bryony J. Leeke, Klaudia Stanik, Antonia Ramos-Guzmán, Benjamin Moyon, Jesus Gil, Alexis R. Barr, Michelle Percharde
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Parvoviruses are small, single-stranded DNA viruses that have evolved sophisticated mechanisms to hijack host cell machinery for replication and persistence. One critical aspect of this interaction involves the manipulation of the host’s DNA Damage Response (DDR) pathways. While the viral genome is comparatively simple, parvoviruses have developed strategies that cause significant DNA damage, activate DDR pathways, and disrupt the host cell cycle. This review will explore the impact of parvovirus infections on host genome stability, focusing on key viral species such as Adeno-Associated Virus (AAV), Minute Virus of Mice (MVM), and Human Bocavirus (HBoV), and their interactions with DDR proteins. Since parvoviruses are used as oncolytic agents and gene therapy vectors, a better understanding of cellular DDR pathways will aid in engineering potent anti-cancer agents and gene therapies for chronic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f8c8e278af4ce203bace432e47ab666e9ddfe2" target='_blank'>
              Small Genomes, Big Disruptions: Parvoviruses and the DNA Damage Response
              </a>
            </td>
          <td>
            Rhiannon R. Abrahams, Kinjal Majumder
          </td>
          <td>2025-03-29</td>
          <td>Viruses</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, Antonio Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c91a033b11a5d910b45df1fe7d4556a0c6c8b6e" target='_blank'>
              Abstract 5585: DNA methylation and transcription factors regulate overexpression of PVT1 in prostate cancer
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Colin Finnegan, Murtaza Barkarar, Oluwabusola Ajagbe, Rachel E. Bonacci, O. Ogunwobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40685785e340da5a1a12204dd296babda47935cc" target='_blank'>
              Abstract 1955: Blood-based mapping of the personalized tumor epigenomic landscape
              </a>
            </td>
          <td>
            Elena Zotenko, Rachel Gittelman, Alan Selewa, Noam Vardi, Tam Banh, Gleb Martovetsky, Oscar Westesson, Emily Tsang, Sara Wienke, D. Kennedy, Meromit Singer, Stefanie Mortimer, Darya I. Chudova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background LRFN4, characterized by leucine-rich repeats and fibronectin type III domains, has been implicated in various human diseases. However, its role in immune regulation and cancer prognosis remains unclear. Methods We performed a comprehensive analysis using datasets from The Cancer Genome Atlas (TCGA), Cancer Cell Line Encyclopedia (CCLE), Genotype-Tissue Expression Project (GTE x), UALCAN, Star Base, and Comparative Toxicogenomics Database (CTD), and observed significant dysregulation of LRFN4 in multiple cancers compared to normal tissues. Results LRFN4 expression was strongly correlated with clinical prognosis, immune subtypes, molecular subtypes, immune checkpoint (ICP) genes, tumor mutational burden (TMB), microsatellite instability (MSI), and immune infiltration, which were measured by ESTIMATE scores. Moreover, LRFN4 expression was associated with the presence of tumor-infiltrating immune cells, particularly in gastrointestinal tumors, reflecting immune cell genetic signatures. Validation through fluorescence multiplex immunohistochemistry confirmed that the association of LRFN4 protein expression with the clinicopathological features and the immune microenvironment of gastric cancer. Flow cytometry analysis indicated that LRFN4 inhibited apoptosis in gastric cancer cell lines while enhancing cell cycle arrest in the S phase. Western Blot analysis demonstrated a positive correlation between the high expression of LRFN4 and the expression levels of cyclin D1 as well as CDK4. In contrast, a negative correlation was observed between the high expression of LRFN4 and the expression level with cleaved-caspase-3 levels. Conclusion These findings suggest that LRFN4 may serve as a novel biomarker for cancer prognosis and a potential target for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c07d23b659bc994c0936e1010f028f57eb1c1bba" target='_blank'>
              A comprehensive pan-cancer analysis of LRFN4: its potential as a prognostic biomarker and therapeutic target for immunotherapy
              </a>
            </td>
          <td>
            Yinmei Xu, Xin Wu, Peng Zhi, Genyu Guo, Yankan Fu, Lukuan You, Siyuan Huai, Jianxiong Li
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d51acfd255e9f2c3851395fb88bf941b1c544b" target='_blank'>
              Abstract 1735: The discovery of PKMYT1 inhibitors to exploit synthetic lethality caused by replication stress
              </a>
            </td>
          <td>
            R. Marais, J. Grabarek, M. Ioannou, L. Leung, A. Raoof, Mike Brown, D. Smithen, C. Fitzpatrick, C. Chester, J. Leigh, M. Aljarah, M. Lisauskaite, O. Grytsai, R. Burgin, D. Niculescu-Duvaz, G. Thomson, C. Springer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d08007dfb89c4d2387e3dfb8d798320475546d" target='_blank'>
              RF-SIRF defining reversed DNA replication forks with single-cell and spatio-temporal resolution reveals a replication stress specific epigenetic code
              </a>
            </td>
          <td>
            Sunetra Roy, Morgan Fimreite, Yue Chen, Francesca Citron, Giulio Draetta, K. Schlacher
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In this study, we established dozens of single provirus-infected cellular clones offering various transcriptional phenotypes of HIV. We proposed that stochastic fluctuations in HIV transcription can appear at, at least, two levels: (1) the chromosomal landscape and (2) the in situ HIV integration site. In the former case, proviruses integrating at different genomic locations demonstrated a variety of transcriptional bursting and can be classified in noise space constructed based on the parameters associated with the coefficient of variation and using a mathematical model fitting a curve of exponential decay. In the latter case, stochastic HIV gene expression tended to be a pure epigenetic phenomenon: the identical provirus demonstrated fluctuations in its transcription with an elevated frequency. We observed similar expression patterns between sense and antisense RNA transcripts. Notably, both HIV long terminal repeats reacted to drug stimulation and may reveal distinct behaviors. Overall, our data suggest that HIV antisense transcripts could be involved in the stochastic nature of HIV transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2887cf5883e567b88fc75ea922207e1c7b94ecf7" target='_blank'>
              The influence of HIV sense and antisense transcripts on stochastic HIV gene expression and reactivation
              </a>
            </td>
          <td>
            Kamil Więcek, Janusz Wiśniewski, Heng-Chang Chen
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Archaea, often thriving in extreme habitats, are believed to have evolved efficient DNA repair pathways to cope with constant insults to their genomes. However, how these organisms repair DNA double-strand breaks (DSBs), the most lethal DNA lesions, remains unclear. Here, we show that replicative primase consisting of the catalytic subunit PriS and the noncatalytic subunits PriL and PriX from the hyperthermophilic archaeon Saccharolobus islandicus is involved in DSB repair. We show that the overproduction or knockdown of PriL increases or decreases, respectively, the rate of survival and mutation frequency of S. islandicus cells following treatment with a DNA damaging agent. The increase in mutation is attributed primarily to an increase in small insertions or deletions. Further, overproduction of PriL enhances the repair of CRISPR-generated DSBs in vivo. These results are consistent with the extraordinary ability of PriSL to promote annealing between DNA strands sharing microhomology in addition to the activity of the heterodimer in terminal transfer and primer extension. The primase-mediated DSB repair is cell-cycle dependent since PriL is barely detectable during the S/G2 transition. Our data demonstrate that replicative primase is involved in DSB repair through microhomology-mediated end joining in Archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41c802b392e2aa09bc0c77d38347ce453e14aa3" target='_blank'>
              Archaeal replicative primase mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Daijiang Xiong, Zhimeng Li, Wen Qi, Shaoying Wang, Junkai Huang, Ningning Zhang, Zhenfeng Zhang, Li Huang
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The development of hematopoietic cell lineages is a highly complex process governed by a delicate interplay of various transcription factors. The expression of these factors is influenced, in part, by epigenetic signatures that define each stage of cell differentiation. In particular, the formation of B lymphocytes depends on the sequential silencing of stemness genes and the balanced expression of interdependent transcription factors, along with DNA rearrangement. We have investigated the impact of Dido3 deficiency, a protein involved in chromatin status readout, on B cell differentiation within the hematopoietic compartment of mice. Our findings revealed significant impairments in the successive stages of B cell development. The absence of Dido3 resulted in remarkable alterations in the expression of essential transcription factors and differentiation markers, which are crucial for orchestrating the differentiation process. Additionally, the somatic recombination process, responsible for generation of antigen receptor diversity, was also adversely affected. These observations highlight the vital role of epigenetic regulation, particularly the involvement of Dido3, in ensuring proper B cell differentiation. This study reveals new mechanisms underlying disruptive alterations, deepening our understanding of hematopoiesis and may potentially lead to insights that aid in the development of therapeutic interventions for disorders involving aberrant B cell development. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01394-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba226143005da64aa14cb154db38e86341f77b0e" target='_blank'>
              The chromatin reader Dido3 is a regulator of the gene network that controls B cell differentiation
              </a>
            </td>
          <td>
            Fernando Gutiérrez del Burgo, María Ángeles García-López, Tirso Pons, Enrique Vázquez de Luis, Carlos Martínez-A, R. Villares
          </td>
          <td>2025-04-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="PURPOSE Regulator of chromosome condensation 1 (RCC1) and RCC2 have been shown to play important roles in the regulation of cell cycle, DNA damage response, and nucleocytoplasmic transport. MATERIALS AND METHODS DNA (592-gene or whole exome) and RNA (whole transcriptome) sequencing was performed at Caris Life Sciences (Phoenix, AZ). Samples were stratified by RCC1 expression quartile thresholds (Q1: low, Q4: high) for small cell lung cancer (SCLC; n = 876), non–small cell lung cancer (NSCLC; n = 21,603), gastric cancer (GC; n = 1,908), pancreatic cancer (PC; n = 5,071), and colorectal cancer (CRC; n = 14,892). Statistical significance was determined using chi-square and Wilcoxon rank-sum tests and adjusted for multiple comparisons (*P < .05). Corresponding analyses were run for RCC2. RESULTS Median RCC1 mRNA expression was highest in SCLC (14.3 transcript per million [TPM]), followed by GC (9.9), NSCLC (9.9), CRC (9.8), and PC (6.9). Similar to RCC1, the median RCC2 expressions were highest in SCLC (36.2 TPM). Tumor mutational burden-high rates were positively associated with increasing RCC1 expression quartiles (Q1-4) in NSCLC (31%-41%), GC (7%-22%), and CRC (5%-17%) and with increasing RCC2 expression in NSCLC and CRC only. Higher expression with RCC1 and RCC2 was associated with worse overall survival in NSCLC (hazard ratio [HR] for RCC1 and RCC2 were 1.3 and 1.3, respectively), PC (HR for RCC1 and RCC2 were 1.5 and 1.12, respectively), and CRC (HR for RCC1 and RCC2 were 1.3 and 1.03, respectively). CONCLUSION RCC1 and RCC2 expression is a negative prognostic marker in NSCLC, PC, and CRC. Further studies to investigate RCC1 and RCC2 function at the molecular level may provide opportunities for novel targeted drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ff6d06df5be38cb6ff5a68f5198616070f9b7f" target='_blank'>
              Multiomic Characterization of RCC1 and RCC2 Expression and Their Association With Molecular Alterations, Immune Phenotypes, and Cancer Outcomes
              </a>
            </td>
          <td>
            M. Nagasaka, Brian Warnecke, S. Ou, Sahar F. Bannoura, Chul Kim, A. Elliott, B. Halmos, Dave S.B. Hoon, S. Darabi, A. Sukari, M. Radovich, Emil Lou, George W Sledge, W. El-Deiry, M. Al-Hallak, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-05-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6032320cee71742e5a132f4662a3a337579e09" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe.
              </a>
            </td>
          <td>
            Joshua M Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-05-03</td>
          <td>Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa57a71626db4f88ef44a82587dc582edcdfe08" target='_blank'>
              Dissecting the Emerging Regulatory and Mechanistic Paradigms of Transcribed Conserved Non-Coding Elements in Breast Cancer
              </a>
            </td>
          <td>
            Wenyong Zhu, Hao Huang, Qiong Li, Yu Gu, Rong Zhang, Huiling Shu, Yunqi Zhao, Hongde Liu, Xiao Sun
          </td>
          <td>2025-04-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7d904ab0bb7bfa986bb2bea4c6f19edb348725" target='_blank'>
              Dysregulation of Locus-Specific Repetitive Elements in TCGA Pan-Cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Inaccurate chromosome segregation can lead to the formation of aneuploid cells that harbor an imbalanced complement of chromosomes. Several fungal species are not only able to tolerate the detrimental effects of aneuploidy but can use it to adapt to environmental pressures. The fungal pathobiont Candida albicans frequently acquires supernumerary chromosomes that enable growth in the presence of antifungal drugs or in specific host niches, yet the transcriptional changes associated with aneuploidy are not globally defined. Here, a karyotypically diverse set of C. albicans strains revealed that expression generally correlated with gene copy number regardless of the strain karyotype. Unexpectedly, aneuploid strains shared a characteristic transcriptional profile that was distinct from a generalized environmental stress response previously defined in aneuploid yeast cells. This aneuploid transcriptional response led to altered growth and oxidative balances relative to euploid control strains. The increased expression of reactive oxygen species (ROS) mitigating enzymes in aneuploid cells reduced the levels of ROS but caused an acute sensitivity to both internal and external sources of oxidative stress. Taken together, our work demonstrates common transcriptional and phenotypic features of aneuploid C. albicans cells with consequences for infection of different host niches and susceptibility to environmental stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e3ff6416e6e4c37bdd4902bf219f1aee84a69b" target='_blank'>
              Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans
              </a>
            </td>
          <td>
            Anna I Mackey, Robert J. Fillinger, P. S. Hendricks, Gregory J. Thomson, Christina A. Cuomo, Richard J. Bennett, Matthew Z. Anderson
          </td>
          <td>2025-04-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e660bf6ed0fa0c7abfd961fbf48ee2e593689402" target='_blank'>
              Abstract 4108: SMARCA4-driven oncogenic enhancer and transcriptional reprogramming in non-small cell lung cancer
              </a>
            </td>
          <td>
            Hye-Ju Yang, Eun-ju Kim, Sang-Hyeon Song, Tae-You Kim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575ec445e704ca863d5507eaa78e755caa175d07" target='_blank'>
              The Use of Mutational Signatures to Decipher the Inter-Relationship of Reactive Oxygen Species with Homologous Recombination and Non-Homologous End-Joining Deficiencies as Well as Their Effects on APOBEC Mutagenesis in Breast Cancer
              </a>
            </td>
          <td>
            Amir Farmanbar, Robert Kneller, Sanaz Firouzi
          </td>
          <td>2025-05-12</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de660154998a356e9b85c8c4e7d59ed34b5f5dd0" target='_blank'>
              Abstract 4668: Prognostic significance and therapeutic potential of TOP2A expression in rare cancers
              </a>
            </td>
          <td>
            Jifa Li, Hua Ye, Xiaoqiong Bao, Wenwen Yu, Xiangxiang Chen, Yangyang Ni, Jincong Wang, Yubo Shi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) present a critical threat to genomic integrity, often precipitating genomic instability and oncogenesis. Repair of DSBs predominantly occurs through homologous recombination (HR) and non-homologous end joining (NHEJ). In HR-deficient cells, DNA polymerase theta (Polθ) becomes critical for DSB repair via microhomology-mediated end joining (MMEJ), also termed theta-mediated end joining (TMEJ). Thus, Polθ is synthetically lethal with BRCA1/2 and other HR factors, underscoring its potential as a therapeutic target in HR-deficient cancers. However, the molecular mechanisms governing Polθ-mediated MMEJ remain poorly understood. Here we present a series of cryo-electron microscopy structures of the Polθ helicase domain (Polθ-hel) in complex with DNA containing different 3′-ssDNA overhangs. The structures reveal the sequential conformations adopted by Polθ-hel during the critical phases of DNA binding, microhomology searching, and microhomology annealing. The stepwise conformational changes within the Polθ-hel subdomains and its functional dimeric state are pivotal for aligning the 3′-ssDNA overhangs, facilitating the microhomology search and subsequent annealing necessary for DSB repair via MMEJ. Our findings illustrate the essential molecular switches within Polθ-hel that orchestrate the MMEJ process in DSB repair, laying the groundwork for the development of targeted therapies against the Polθ-hel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdae7f64677f27faa8b83647b04ea59147fae30b" target='_blank'>
              Structural basis for Polθ-helicase DNA binding and microhomology-mediated end-joining
              </a>
            </td>
          <td>
            Fumiaki Ito, Ziyuan Li, Leonid Minakhin, Htet A. Khant, R. Pomerantz, Xiaojiang S. Chen
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA) is present in food, water, and drugs and is considered a probable human carcinogen by the International Agency for Research on Cancer. The mechanism of action of NDMA involves the generation of carcinogenic methyl lesions such as 3-methyladenine (3MeA) on DNA bases. Alkyladenine DNA Glycosylase (AAG) removes 3MeA to initiate Base Excision Repair, leaving an intermediary lesion that is subsequently resolved by backbone cleavage, nucleotide insertion, and backbone ligation. The intermediate steps following lesion removal produce potentially toxic and mutagenic single-strand DNA breaks. Here, we explored differences between males and females regarding downstream DNA damage, toxicity, mutations and cancer arising from 3MeA in the livers of WT, Aag-/-, and Aag-overexpressing (AagTg) mice. We found that males were more susceptible to NDMA-induced mutations (WT and Aag-/-) and cancer (all genotypes). In contrast, AagTg females were more prone than males to micronucleus induction. As we showed in our prior analyses where data were pooled for males and females, Aag-/- mice were significantly more susceptible to NDMA-induced mutations and cancer, and AagTg mice displayed significantly greater toxicity. Building on these findings, our analyses of sex-related differences show that Aag deficiency and maleness are both susceptibility factors for NDMA-induced liver cancer, while Aag overexpression drives toxicity, potentially with a greater effect on females. By assessing differences between males and females, this study reveals a deeper mechanistic understanding of the underpinnings for a well-known increased risk of liver cancer in men versus women by demonstrating a higher susceptibility of male mice to both mutations and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2ad8f9213bd9ddfa98933a729d0b81e465af3f" target='_blank'>
              Sex and Alkyladenine DNA Glycosylase Expression are Key Susceptibility Factors for NDMA-induced Mutations, Toxicity, and Cancer
              </a>
            </td>
          <td>
            Jennifer E. Kay, J. Corrigan, Lindsay B Volk, Amanda L Armijo, Ilana S Nazari, D. Torous, Svetlana L. Avlasevich, R. G. Croy, D. Wadduwage, S. Dertinger, J. Essigmann, Leona D. Samson, Sebastian E Carrasco, B. Engelward
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, C. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Transcription-coupled repair (TCR) and R-loops are two interrelated processes critical to the maintenance of genome stability during transcription. TCR, a specialized sub-pathway of nucleotide excision repair, rapidly removes transcription-blocking lesions from the transcribed strand of active genes, thereby safeguarding transcription fidelity and cellular homeostasis. In contrast, R-loops, RNA–DNA hybrid structures formed co-transcriptionally, play not only regulatory roles in gene expression and replication but can also contribute to genome instability when persistently accumulated. Recent experimental evidence has revealed dynamic crosstalk between TCR and R-loop resolution pathways. This review highlights current molecular and cellular insights into TCR and R-loop biology, discusses the impact of their crosstalk, and explores emerging therapeutic strategies aimed at optimizing DNA repair and reducing disease risk in conditions such as cancer and neurodegenerative disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb58ac22e35569ba8451a29e77e0d78908989716" target='_blank'>
              Transcription-Coupled Repair and R-Loop Crosstalk in Genome Stability
              </a>
            </td>
          <td>
            Jeseok Jeon, Tae-Hong Kang
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a226dccecb23fcc9aa5b5fc6069240367f4d5b" target='_blank'>
              Abstract 5044: Whole-genome investigation of transcription factors that promote UV damage and mutation
              </a>
            </td>
          <td>
            Bo Chi, Raluca Gordân
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cells are the fundamental structural and functional units of living organisms and the study of these entities has remained a central focus throughout the history of biological sciences. Traditional cell research techniques, including fluorescent protein tagging and microscopy, have provided preliminary insights into the lineage history and clonal relationships between progenitor and descendant cells. However, these techniques exhibit inherent limitations in tracking the full developmental trajectory of cells and elucidating their heterogeneity, including sensitivity, stability and barcode drift. In developmental biology, nucleic acid barcode technology has introduced an innovative approach to cell lineage tracing. By assigning unique barcodes to individual cells, researchers can accurately identify and trace the origin and differentiation pathways of cells at various developmental stages, thereby illuminating the dynamic processes underlying tissue development and organogenesis. In cancer research, nucleic acid barcoding has played a pivotal role in analyzing the clonal architecture of tumor cells, exploring their heterogeneity and resistance mechanisms and enhancing our understanding of cancer evolution and inter-clonal interactions. Furthermore, nucleic acid barcodes play a crucial role in stem cell research, enabling the tracking of stem cells from diverse origins and their derived progeny. This has offered novel perspectives on the mechanisms of stem cell self-renewal and differentiation. The present review presented a comprehensive examination of the principles, applications and challenges associated with nucleic acid barcode technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5cf40687295751922bb2f7725d94b4cd317977" target='_blank'>
              Barcodes based on nucleic acid sequences: Applications and challenges (Review)
              </a>
            </td>
          <td>
            Ying Hong Wei, Faquan Lin
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4f3702444f966febda408aa09a277d286c200" target='_blank'>
              Abstract 1409: Reprogramming the alternative polyadenylation of an oncogenic transcription factor as a therapeutic strategy for prostate cancer
              </a>
            </td>
          <td>
            Giulia Pagani, Chiara Casirati, Stefania Fornezza, Roberta Cacioppo, P. Gandellini
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, S. Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53622534a6ad4bf417cb989b871bcc51c0303f4" target='_blank'>
              Abstract 1420: Long-read sequencing of single-stranded DNA of colorectal cancer patients
              </a>
            </td>
          <td>
            Béla Molnár, K. Rada, Nikolett Szakállas, B. Barták, Alexandra Kalmár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae14464f2ce77e42ffad40f3c6898b5621cece3e" target='_blank'>
              PLK4: Master Regulator of Centriole Duplication and Its Therapeutic Potential.
              </a>
            </td>
          <td>
            Muhammad Hamzah, Franz Meitinger, Midori Ohta
          </td>
          <td>2025-04-21</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="KRAS is a proto-oncogene that contains activating mutations in up to 30% of tumors. Many conventional therapies inhibit both cancerous and normal cells, which may cause toxicity. Thus, programmable mutant-selective targeted inhibitors are needed. Peptide nucleic acids (PNAs) incorporate base sequences analogous to DNA, with modified peptide backbones instead of ribose-phosphate backbones, allowing PNAs to hybridize with DNA with high avidity to suppress transcription. Here, we developed KRAS G12D-selective PNA oligomers with novel cell-penetrating flanking regions. Fluorescein-labeled PNA oligomers displayed high uptake rates in cells and nuclei. Exposure to PNA-delivery peptide conjugates resulted in repression of KRAS G12D mRNA and protein expression within 2 hours and lasting up to 48 hours. Varying cell-penetrating peptide (CPP) compositions and lengths of complementary KRAS sequences were tested using dose-response cell viability assays. These experiments identified configurations that were effective at selectively preventing growth of on-target KRAS G12D cells, while relatively sparing off-target KRAS G12C cells. Electrophoretic mobility shift assays demonstrated in vitro binding and selectivity for KRAS G12D DNA sequences. CPP-PNA-G12D-1 was effective against a panel of pancreatic ductal adenocarcinoma cell lines and patient-derived xenografts in vivo. These results show promise for an enhanced PNA-delivery peptide conjugate strategy as both a tool for studying tumors driven by oncogenic point mutations and as a potential therapeutic strategy to selectively target mutant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/809bc7a2732828bb8a1dec23dc2dcff988fd6804" target='_blank'>
              [Aberrant RNA Splicing Regulation as a Novel Source of Neoantigens].
              </a>
            </td>
          <td>
            Masahiko Ajiro
          </td>
          <td>2025-04-01</td>
          <td>Gan to kagaku ryoho. Cancer & chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2065faaa8c8b01822bceb610976fd9fb9507b1" target='_blank'>
              Mechanism of DNA replication fork breakage and PARP1 hyperactivation during replication catastrophe.
              </a>
            </td>
          <td>
            Pedro Ortega, Elodie Bournique, Junyi Li, Ambrocio Sanchez, Gisselle Santiago, Brooke R. Harris, Josefine Striepen, J. Maciejowski, Abby M. Green, Rémi Buisson
          </td>
          <td>2025-04-18</td>
          <td>Science advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Eukaryotic genomes are physically organized into chromatin, which regulates virtually every dimension of genome biology, including replication, recombination, transcription, and DNA repair. Precocious Dissociation of Sisters 5 (PDS5) protein is a key regulator of the cohesin complex, responsible for generating higher-order chromatin structures within and between chromosomes. In yeast and vertebrates, PDS5 can lead to cohesin establishment or disassembly, depending on its interaction with other proteins. In this study, we explored the diversity of PDS5 proteins across eukaryotes; HEAT-repeats forming a hook-like structure (PDS5 domain), followed by a predicted disordered region of variable lengths, are observed across the Tree of Life. Interestingly, two homolog types (A and B) with signs of varying chromatin localization emerged independently in later branches of animal and plant kingdoms, in vertebrates and vascular plants, respectively (∼400-500 MYA). We also investigated the emergence of two plant-specific features in the PDS5 proteins: 1.) An ancestral acquisition of a Tudor domain with conserved H3K4me1-binding amino acids in land plants 2.) A truncated version of PDS5 proteins that lacks essential features of PDS5 domain and is involved in the RNA-directed DNA methylation (RdDM) plant-specific pathway which emerged in seed plants. This work provides the most comprehensive study of PDS5 diversity, and suggests the emergence of specialized cohesin regulatory mechanisms that could shape distinct strategies for genome organization across eukaryotes. Short summary Here, we analyzed the diversification of PDS5 proteins; sequence, domain content, and structure, providing new observations that evidence different evolutionary trajectories of this protein across the Tree of Life. These results suggest neofunctional evolution in cohesin regulatory proteins that could shape distinct strategies for genome organization across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c1664fc04ea9ae186e4173c40cb2e49fd9d273" target='_blank'>
              Evolutionary trajectories of cohesin gatekeepers across the Tree of Life
              </a>
            </td>
          <td>
            Daniela Quiroz, Alice Pierce, Grey Monroe
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b447161a1815b3267cbfa70059d3d28a80f8d107" target='_blank'>
              Targeting redox-sensitive MBD2-NuRD condensate in cancer cells.
              </a>
            </td>
          <td>
            Heyang Wei, Hongdan Zheng, Siqing Wang, Yun Yang, Ruiqian Zhao, Aihong Gu, Ronggui Hu, Fei Lan, Wenyu Wen
          </td>
          <td>2025-04-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Toxoplasma gondii is an obligate intracellular parasite with a high replication rate that can lead to DNA replicative stress, in turn associated with the generation of DNA double-strand breaks (DSBs). Cells have two main pathways to repair DSBs: non-homologous end joining and homologous recombination repair (NHEJ and HRR respectively). RAD51 is the key recombinase in the HRR pathway. In this work, we achieved endogenous tagging of the RAD51 gene using the Auxin Inducible Degron (AID) system, to generate the clonal line RH RAD51HA-AID. Here we demonstrate that RAD51 is expressed in replicative tachyzoites and establishes damage foci. Auxin-induced knock-down (KD) affects the correct replication of tachyzoites which show loss of synchronization. The use of the RAD51 inhibitor B02 also affects parasite growth, with an IC50 of 4.8 µM. B02 produced alterations in tachyzoite replication and arrest in the S phase of the cell cycle. Additionally, B02 induced tachyzoite to bradyzoite differentiation showing small cyst-like structures. In conclusion, the HRR pathway is necessary for maintaining proper tachyzoite replication under normal growth conditions, supporting that replicative stress occurs during the cell cycle. Our findings also suggest that DNA replication stress can induce bradyzoite differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a99ccd548e29dfb7eccfd9c686305c90fbab532" target='_blank'>
              Toxoplasma gondii RAD51 recombinase is required to overcome DNA replication stress and its inactivation leads to bradyzoite differentiation
              </a>
            </td>
          <td>
            Ana M. Saldarriaga Cartagena, Ayelén Aparicio Arias, C. Cristaldi, Agustina Ganuza, M. M. Gonzalez, María M. Corvi, William J. Sullivan, Laura Vanagas, Sergio O. Angel
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af01fc45afb6596f1391be85d304128cf0bbe91" target='_blank'>
              Abstract 1481: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Bone metastases represent frequent and severe complications in various cancers, notably impacting prognosis and quality of life. This review article delves into the genetic and epigenetic mechanisms underpinning drug resistance in bone metastases, a key challenge in effective cancer treatment. The development of drug resistance in cancer can manifest as either intrinsic or acquired, with genetic heterogeneity playing a pivotal role. Intrinsic resistance is often due to pre-existing mutations, while acquired resistance evolves through genetic and epigenetic alterations during treatment. These alterations include mutations in driver genes like TP53 and RB1, epigenetic modifications such as DNA methylation and histone changes, and pathway alterations, notably involving RANK-RANKL signaling and the PI3K/AKT/mTOR cascade. Recent studies underline the significance of the tumor microenvironment in fostering drug resistance, with components such as cancer-associated fibroblasts and hypoxia playing crucial roles. The interactions between metastatic cancer cells and the bone microenvironment facilitate survival and the proliferation of drug-resistant clones. This review highlights the necessity of understanding these complex interactions to develop targeted therapies that can overcome resistance and improve treatment outcomes. Current therapeutic strategies and future directions are discussed, emphasizing the integration of genomic profiling and targeted interventions in managing bone metastases. The evolving landscape of genetic research, including the application of next-generation sequencing and CRISPR technology, offers promising avenues for novel and more effective therapeutic strategies. This comprehensive exploration aims to provide insights into the molecular intricacies of drug resistance in bone metastases, paving the way for improved clinical management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3b250a10d84b32555ed5529bedbf646475e86e" target='_blank'>
              The genetic architecture of bone metastases: unveiling the role of epigenetic and genetic modifications in drug resistance
              </a>
            </td>
          <td>
            Ahmad Dawalibi, Mohamad Bakir, Khalid S. Mohammad
          </td>
          <td>2025-04-22</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d31436619804913b5e6fe39fc1dbf3eb2207ea0" target='_blank'>
              Abstract 2920: ZBP1 antagonizes MRE11-mediated DNA end resection and confers synthetic lethality to PARP inhibition in Ovarian cancer
              </a>
            </td>
          <td>
            Shen-nan Shi, Qiuyang Xu, Xuejiao Zhao, Gordon B. Mills, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f994b853e5612dbe78f049706800d9c2a5981d" target='_blank'>
              Systemic genome-epigenome analysis captures a lineage-specific super-enhancer for MYB in gastrointestinal adenocarcinoma.
              </a>
            </td>
          <td>
            Fuyuan Li, Shangzi Wang, Lian Chen, Ning Jiang, Xingdong Chen, Jin Li
          </td>
          <td>2025-04-15</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b1ea31e530ff890dfc3d45aab43f96f9c81ec4" target='_blank'>
              Abstract 3825: Divergent evolution of FOXA1 mutations drive prostate tumorigenesis or therapy-resistant intra-luminal plasticity
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Jie Luo, Jean Ching-Yi Tien, Qiuyang Zhang, Matthew Pang, Somnath Mahapatra, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J. Todd, Noshad Hosseini, Ryan J. Rebernick, Xuhong Cao, Rui Wang, Fengyun Su, Rohit Mehra, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>168</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7defb0dc0b2dcfb00df9b73a0a0c72cd7d43d4" target='_blank'>
              Transposable elements as instructors of the immune system.
              </a>
            </td>
          <td>
            Lisa Schmidleithner, Philipp Stüve, Markus Feuerer
          </td>
          <td>2025-04-29</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA metabolism consists of crucial processes occurring in all living cells. These processes include various transactions, such as DNA replication, genetic recombination, transposition, mutagenesis, and DNA repair. While it was initially assumed that these processes might occur in the cytoplasm of prokaryotic cells, subsequent reports indicated the importance of the cell membrane in various DNA transactions. Furthermore, newly identified factors play significant roles in regulating DNA-related cellular processes. One such factor is the Hfq protein, originally discovered as an RNA chaperone but later shown to be involved in several molecular mechanisms. These include DNA transactions and interaction with the cell membrane. Recent studies have suggested that Hfq plays a role in the regulation of DNA replication, mutagenesis, and recombination. In this narrative review, we will focus on the importance of membranes in DNA transactions and discuss the potential role of Hfq-mediated regulation of these processes in Escherichia coli, where the protein is the best characterized. Special attention is given to the affinity of this small protein for both DNA and membranes, which might help explain some of the findings from recent experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd00cd734e448197ffc4468301d796e51c42b3e2" target='_blank'>
              DNA Transactions in Bacteria and Membranes: A Place for the Hfq Protein?
              </a>
            </td>
          <td>
            Sylwia Bloch, R. Sinden, Frank Wien, Grzegorz Węgrzyn, V. Arluison
          </td>
          <td>2025-04-01</td>
          <td>Membranes</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Significance Cells tightly control gene expression through the combined actions of many transcription factors and cofactors responding to developmental and environmental signals. This study uncovers an unexpected mechanism by which inflammatory signals influence gene expression in breast cancer cells by regulating the transcriptional activity of the nuclear receptor corepressor (NCoR)/silencing mediator of retinoic acid and thyroid hormone receptor (SMRT)/histone deacetylase 3 (HDAC3) corepressor complex through tumor necrosis factor receptor-associated factor 6 (TRAF6)-dependent ubiquitination events. These findings reveal an intricate interplay between the mechanisms regulating the function and clearance of chromatin remodeling complexes by immune signaling pathways across subcellular compartments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3624750f674c04efdf16591612bce3664c372f18" target='_blank'>
              Nonproteolytic ubiquitination regulates chromatin occupancy by the NCoR/SMRT/HDAC3 corepressor complex in MCF-7 breast cancer cells
              </a>
            </td>
          <td>
            Giulio Ferrero, M. Cardamone, Francesca Luca, E. Bourk, L. Ricci, Wen Liu, Yuan Gao, Giulia Burrone, Akhirah Muhammad, Stefanie Chan, Emma Smith, T. Fan, S. Cutrupi, Ivan Garcia-Bassets, Michele De Bortoli, Michael G. Rosenfeld, Valentina Perissi
          </td>
          <td>2025-04-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771756b2729241a36b5c4bbcbf190204361bdbc6" target='_blank'>
              Abstract 1427: Characterization of epigenetic regulation of MYC in acute myeloid leukemia
              </a>
            </td>
          <td>
            W. S. Streitfeld, William M. Yashar, Itallia V. Pacentine, Thai T Nguyen, Samantha J. Worme, Gurkan G. Yardimci, Andrew C. Adey, Theodore P. Braun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57ae88cae69176f96ae514176dc70dea1bd4299" target='_blank'>
              Cellular type is a major determinant of R-loop genomic distribution
              </a>
            </td>
          <td>
            K. Oleynikova, N. Zhigalova, A. P. Hutchins, Alexey Ruzov
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA topoisomerase 1 (TOP1) is essential for transcription, replication, and repair. Its function relies on two catalytic steps, DNA breakage and rejoining. Inhibitors of the second step prevent DNA rejoining and lead to persistent DNA breaks, acting as topoisomerase poisons, used as anticancer drugs. However, reliable inhibitors of the first step are not available. Here, we provide genetic and molecular evidence supporting that Patulin and, to a lesser extent, Xestoquinol inhibit the first catalytic step of TOP1 in vitro, in yeast and in human cells. Particularly, Patulin prevents the accumulation of TOP1 cleavage complexes caused by the TOP1 poison camptothecin (CPT) in human cells. Moreover, Patulin pretreatment of human or yeast cells reduces DNA damage and the accumulation of DNA breaks upon CPT exposure. Consistent with the protective role of TOP1 against harmful R-loops, Patulin treatment increases R-loops and R-loop-associated cytotoxicity, mimicking the effect of TOP1 silencing. Altogether our findings indicate that Patulin and Xestoquinol are nonpoisoning inhibitors of TOP1, which should potentiate new research approaches in molecular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c91b4af793ee46a1c397d0e749fad65c6dce6ba" target='_blank'>
              Patulin and Xestoquinol are inhibitors of DNA topoisomerase 1.
              </a>
            </td>
          <td>
            E. Tumini, R. Wellinger, Emilia Herrera-Moyano, Patricia Navarro-Cansino, María L. García-Rubio, Daniel Salas-Lloret, Alejandro Losada, M. Muñoz-Alonso, H. Gaillard, Rosa Luna, A. Aguilera
          </td>
          <td>2025-04-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b4705110ed5255e541024e217bd1b38f97bdc5" target='_blank'>
              Abstract 2802: Circular RNA variants of theNUMB gene modulate the oncogenicity of oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Chung-Hsien Chou, Pin-Yu Chen, Shu-Chun Lin, Kuo-Wei Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="By integrating findings from large-scale omics analyses with experimental tests, this study aims to decipher susceptibility genes and the underlying biological mechanisms involved in the development of colorectal cancer (CRC). We first conducted a trans-ancestry transcriptome-wide association study (TWAS) among 57,402 CRC cases and 119,110 controls, aiming to examine how altered gene expression influences CRC risk in European and Asian populations. Then, functional experiments in (i) CRC cell lines and (ii) tumor xenografts were conducted to examine potential underlying mechanisms involved in colorectal carcinogenesis. Further, a drug sensitivity test was employed to explore possible clinical implications for CRC treatment. The TWAS identified 67 genes highly associated with CRC risk, 23 of which were novel findings. Functional annotation of variants within TWAS-identified loci revealed that the majority (93.6%) showed evidence of transcriptional regulatory mechanisms via proximal promoter or distal enhancer-promoter interactions. Among the identified susceptibility genes, splicing factor 3a subunit 3 (SF3A3) may act as an oncogene on the basis that overexpression of this gene was significantly associated with increased risk of CRC (P = 5.75 × 10−11). Further cell and animal experiments confirmed that SF3A3 plays an oncogenic role in CRC development, and the underlying biological mechanism is likely to be related to its anti-apoptosis effect. The drug sensitivity test suggested that phenethyl isothiocyanate (PEITC) targeting SF3A3 can inhibit CRC progression. This study identified novel CRC susceptibility genes and potential biological mechanisms of SF3A3 involved in CRC development, providing important insight into the etiology and potential leads to the treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aaa1b2ef2174050014b370855a3f1310c6af79c" target='_blank'>
              Trans-ancestry transcriptome-wide association and functional studies to uncover novel susceptibility genes and therapeutic targets for colorectal cancer
              </a>
            </td>
          <td>
            Lijuan Wang, Lidan Hu, Jing Sun, Jianhui Zhao, Siyun Zhou, Lexin Liu, Wei Yu, Yeting Hu, Dan Zhou, Xiangrui Meng, Zhongshang Yuan, Honghe Zhang, Susan Farrington, M. Timofeeva, Kefeng Ding, J. Little, Malcolm G Dunlop, E. Theodoratou, Xue Li
          </td>
          <td>2025-04-29</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Pharmacogenomics is revolutionizing precision medicine by enabling tailored therapeutic strategies based on an individual genetic and molecular profile. Circular RNAs (circRNAs), a distinct subclass of endogenous non-coding RNAs, have recently emerged as key regulators of drug resistance, tumor progression, and therapeutic responses. Their covalently closed circular structure provides exceptional stability and resistance to exonuclease degradation, positioning them as reliable biomarkers and novel therapeutic targets in cancer management. This review provides a comprehensive analysis of the interplay between circRNAs and pharmacogenomics, focusing on their role in modulating drug metabolism, therapeutic efficacy, and toxicity profiles. We examine how circRNA-mediated regulatory networks influence chemotherapy resistance, alter targeted therapy responses, and impact immunotherapy outcomes. Additionally, we discuss emerging experimental tools and bioinformatics techniques for studying circRNAs, including multi-omics integration, machine learning-driven biomarker discovery, and high-throughput sequencing technologies. Beyond their diagnostic potential, circRNAs are being actively explored as therapeutic agents and drug delivery vehicles. Recent advancements in circRNA-based vaccines, engineered CAR-T cells, and synthetic circRNA therapeutics highlight their transformative potential in oncology. Furthermore, we address the challenges of standardization, reproducibility, and clinical translation, emphasizing the need for rigorous biomarker validation and regulatory frameworks to facilitate their integration into clinical practice. By incorporating circRNA profiling into pharmacogenomic strategies, this review underscores a paradigm shift toward highly personalized cancer therapies. circRNAs hold immense potential to overcome drug resistance, enhance treatment efficacy, and optimize patient outcomes, marking a significant advancement in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c91f6707f728019195288d5d968764867ed64f" target='_blank'>
              Unveiling Pharmacogenomics Insights into Circular RNAs: Toward Precision Medicine in Cancer Therapy
              </a>
            </td>
          <td>
            Saud Alqahtani, T. Alqahtani, K. Venkatesan, Durgaramani Sivadasan, Rehab Ahmed, Hassabelrasoul Elfadil, Premalatha Paulsamy, Kalaiselvi Periannan
          </td>
          <td>2025-04-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The supernumerary B chromosome of maize has a drive mechanism to maintain itself in a population despite being dispensible. This involves nondisjunction of the B centromere at the second pollen mitosis that produces the two sperm followed by preferential fertilization of the egg by the B containing sperm during double fertilization. During an introgression of the supernumerary B chromosome into the inbred line B73, an unusually high frequency of trisomies for A chromosomes was observed. Due to parallels to the High Loss phenomenon in which three or more B chromosomes in a specific genetic background cause chromosomal breakage at heterochromatic knob sites during the second pollen mitosis as well as ploidy changes, this phenomenon was revisited. Examination of pollen of the High Loss line revealed a high frequency of single sperm in the presence of the B chromosomes, which was previously not realized. Crosses to tetraploid females confirmed that the single sperm were diploid and functional but also revealed the presence of diploids with their A chromosomes derived solely from the tetraploid parent indicating a "diploid induction". Collectively, the results reveal two backgrounds in which the B drive mechanism is not confined to this chromosome causing detrimental effects by adherence of heterochromatic knobs and apparently A centromeres at the mitosis preceding sperm development. In most genetic backgrounds this process is restricted to the B chromosome but in B73 and the High Loss line, there is spillover to the normal chromosomes in distinct ways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822f11adca9e71b6e0ff40e6fc00f85ee3a4109f" target='_blank'>
              Aneuploidy and ploidy variation conditioned by the B chromosome of maize.
              </a>
            </td>
          <td>
            Patrice S. Albert, Hua Yang, Zhi Gao, Cassidy DeVore, J. Birchler
          </td>
          <td>2025-04-25</td>
          <td>Heredity</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5516fc4406b04626bff5988fb10a8ec93da2c74b" target='_blank'>
              Unraveling the Role of N6-Methyladenosine in Prostate Cancer: Implications for Prognosis and Tumor Aggression
              </a>
            </td>
          <td>
            Xin Xu, Xinqiao Zheng, Housheng He
          </td>
          <td>2025-05-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f151aed2abf27a7b67be8871fbd4b51979dcabff" target='_blank'>
              Abstract 1485: BRCA2 haploinsufficiency leads to mitotic DNA synthesis at common fragile sites in human mammary epithelial cells
              </a>
            </td>
          <td>
            Kavya Vipparthi, Ariana Bellare, Mihriban Karaayvaz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T. Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Huntington’s disease (HD) is a neurological condition caused by an excessive expansion of CAG repeats in the Huntingtin (HTT) gene. Although experiments have shown an altered epigenetic landscape and chromatin architecture upon HD development, the structural consequences on the HTT gene remain elusive. Structural data are only available for model nucleosome systems and yeast systems with human nucleosomes. Here, we use our experimentally validated nucleosome-resolution mesoscale chromatin model to investigate folding changes of the HTT gene associated with HD. We investigate how the histone fold domain of the variant macroH2A1, a biomarker of HD, affects the genome structure by modeling HD-like systems that contain (i) 100% canonical, (ii) 100% macroH2A1, (iii) 50% canonical and 50% macroH2A1, and (iv) 100% hybrid cores (one canonical H2A and one macroH2A1 per nucleosome). Then, we model the mouse HTT gene in healthy and HD conditions by incorporating the CAG expansion and macroH2A1 cores, reducing the linker histone density and tail acetylation levels, and incorporating genomic contacts. Overall, our results show that the histone fold domain of macroH2A1 affects chromatin compaction in a fiber-dependent manner (i.e., nucleosome distribution dependent) and can thus both enhance or repress HTT gene expression. Our modeling of the HTT gene shows that HTT is less compact in the diseased condition, which could accelerate the production of the mutated protein. By suggesting the structural biophysical consequences of the HTT gene under HD conditions, our findings may help in the development of diagnostic and therapeutic treatments for HD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078c117b1be318f2b52bff16f032f54f66f9eb98" target='_blank'>
              Regulation of Genome Architecture in Huntington’s Disease
              </a>
            </td>
          <td>
            Stephanie Portillo-Ledesma, Minna Hang, Tamar Schlick
          </td>
          <td>2025-04-27</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT We investigated the presence of viral DNA and RNA in cutaneous squamous cell carcinoma (cSCC) tumor and normal tissues from nine individuals with a history of hematopoietic stem cell transplantation (HCT). Microbiome quantification through DNA and RNA sequencing (RNA-seq) revealed the presence of 18 viruses in both tumor and normal tissues. DNA sequencing (DNA-seq) identified Torque teno virus, Saimiriine herpesvirus 1, Merkel cell polyomavirus, Human parvovirus B19, Human gammaherpesvirus-4, Human herpesvirus-6, and others. RNA-seq revealed additional viruses such as Tobamovirus, Pinus nigra virus, Orthohepadnavirus, Human papillomavirus-5, Human herpesvirus-7, Human gammaherpesvirus-4, Gammaretrovirus, and others. Notably, DNA-seq indicated that tumor samples exhibited low levels of Escherichia virus in three out of nine subjects and elevated levels of Human gammaherpesvirus-4 in one subject, while normal samples frequently contained Gammaretrovirus and occasionally Escherichia virus. A comparative analysis using both DNA- and RNA-seq captured three common viruses: Abelson murine leukemia virus, Murine type C retrovirus, and Human gammaherpesvirus-4. These findings were corroborated by an independent data set, supporting the reliability of the viral detection methods utilized. The study provides insights into the viral landscape in post-HCT patients, emphasizing the need for comprehensive viral monitoring in this vulnerable population. IMPORTANCE This study is important because it explores the potential role of viruses in the development of cSCC in individuals who have undergone allogeneic HCT. cSCC is common in this population, particularly in those with chronic graft-versus-host disease on long-term immunosuppression. By using advanced metagenomic and metatranscriptomic next-generation sequencing, we aimed to identify viral pathogens present in tumor and adjacent normal tissue. The results could lead to targeted preventive or therapeutic interventions for these high-risk people, potentially improving their outcomes and management of cSCC. This study is important because it explores the potential role of viruses in the development of cSCC in individuals who have undergone allogeneic HCT. cSCC is common in this population, particularly in those with chronic graft-versus-host disease on long-term immunosuppression. By using advanced metagenomic and metatranscriptomic next-generation sequencing, we aimed to identify viral pathogens present in tumor and adjacent normal tissue. The results could lead to targeted preventive or therapeutic interventions for these high-risk people, potentially improving their outcomes and management of cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56b53109651f083205e0fc320c14b9f0267e185" target='_blank'>
              Mining metagenomes and metatranscriptomes unveils viruses associated with cutaneous squamous cell carcinoma in hematopoietic stem cell transplant recipients
              </a>
            </td>
          <td>
            Jana K Dickter, Yuqi Zhao, Vishwas Parekh, Huiyan Ma, Badri G Modi, Wai-Yee Li, S. Armenian, Xiwei Wu, Farah R Abdulla
          </td>
          <td>2025-03-31</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c16d5ebf6f1d5ea4dbccbf5a0178380f13bbf886" target='_blank'>
              Abstract 7429: Illuminating the dark genome in cancer using artificial intelligence
              </a>
            </td>
          <td>
            M. D. M. Álvarez-Torres, Xi Fu, Raul Rabadan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fe83552a51ec8685227ae6ed241734102c7207" target='_blank'>
              Differential interfacial tension between oncogenic and wild-type populations forms the mechanical basis of tissue-specific oncogenesis in epithelia
              </a>
            </td>
          <td>
            Amrapali Datta, Phanindra Dewan, Aswin Anto, Tanya Chhabra, Tanishq Tejaswi, Sindhu Muthukrishnan, Akshar Rao, Sumantra Sarkar, Medhavi Vishwakarma
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377a4743e9ceeda080173ff3bd910fc589bc9196" target='_blank'>
              Abstract 5890: Personalized detection of circulating cell-free tumor DNA in gynecologic cancer
              </a>
            </td>
          <td>
            Andrea B. Moffitt, J. Kendall, Joan Alexander, A. Stepansky, A. Kapedani, Zi-Hua Wang, D. Levy, Kenny Ye, A. Krieger, Marina Frimer, Gary L. Goldberg, Michael Wigler
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>