<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-10 09:34:57 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>40</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>85</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>120</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>377</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>330</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke‐positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single‐cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low‐pass whole‐genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke‐endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability‐associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f47c334737b17de2a5aca09e969a1cbb905e569" target='_blank'>
              E2F3 amplification primes bladder cancer cells for premature mitosis
              </a>
            </td>
          <td>
            Kathryn A. Wierenga, Qingwu Liu, Isabel Nieland, Pepijn Rakers, Anita van den Heuvel, Mara Pateli, Richard W. Wubbolts, Frank Riemers, Saskia van Essen-Dorresteijn, E. V. van Liere, B. Westendorp
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-01</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T‐ and B‐cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B‐cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by‐product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B‐cell leukemia
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f59455ffad8232534ff09a38ff5659981429a5" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes.
              </a>
            </td>
          <td>
            L. G. de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Cryptococcus neoformans is a fungal pathogen of humans that causes life-threatening meningoencephalitis. During infection, enlarged, polyploid titan cells are produced that promote survival, evade immune cells, and generate diverse progeny. These titan cells divide to produce haploid, aneuploid or diploid daughter cells with enhanced stress tolerance; however, how ploidy reduction occurs is poorly understood. Here, we show that titan cells developed from diploid strains predominantly produce diploid daughter cells with haploid daughters observed infrequently. We further demonstrate that meiosis-specific genes, including DMC1 and SPO11, are critical for stable inheritance of a diploid genome in the daughter cells. Specifically, deletion of these genes in a heterozygous diploid background results in: 1) titan cells with a significantly reduced capacity to produce daughter cells; 2) increased phenotypic variation among daughter cells produced by the titan cells, including traits that could be relevant to cell growth and viability; and 3) daughter cells produced by the titan cells exhibiting high levels of loss of heterozygosity (LOH) and aneuploidy, suggested elevated genome instability. Taken together, these findings demonstrate the importance of meiosis-specific genes in the ploidy reduction process of titan cells derived from a heterozygous diploid background in an important human fungal pathogen. SIGNIFICANCE Polyploid titan cells are central to Cryptococcus neoformans persistence during infection, yet how they reduce ploidy to generate daughter cells was unclear. We show that the meiosis-specific factors Dmc1 and Spo11 are critical for titan cells to produce viable daughter cells specifically in a heterozygous diploid context, ensuring chromosome stability without introducing detectable canonical meiotic recombination. Loss of either gene compromises daughter cell production and causes genome instability (LOH, aneuploidy) and phenotypic heterogeneity. These findings reveal a “para-meiosis” safeguard that preserves genome balance during titan cell depolyploidization in C. neoformans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b707195ab1586edfb629b8392c64a813e56aa4" target='_blank'>
              Meiosis-specific genes play roles in ploidy reduction in Cryptococcus neoformans titan cells
              </a>
            </td>
          <td>
            Zhuyun Bian, Kayla Wilhoit, Julian Liber, Anushka Peer, Ziyan Xu, Paul M. Magwene, Sheng Sun, Joseph Heitman
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e712e19562ce359834d0d186aafed4a0127af156" target='_blank'>
              HCFC1 and YY1 mediate recruitment of COMPASS and Integrator to initiate X chromosome inactivation
              </a>
            </td>
          <td>
            Jeffrey Boeren, , Sarra Merzouk, E. Rentmeester, Rien van Haperen, Samuel J. Luchsinger-Morcelle, Cheryl Maduro, Richelle Rietdijk, Büşra Göynük, W. V. van Ijcken, J. Demmers, C. Gontan, Hegias Mira Bontenbal, J. Gribnau
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OncoDB was initially developed to advance cancer research by integrating RNA expression profiles, DNA methylation patterns, clinical annotations, and oncoviral signatures derived from the Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression datasets. We now present OncoDB 2.0, an extensively expanded platform that offers a more comprehensive and integrated view of cancer omics. The updated version includes an atlas of somatic mutations discovered from combined DNA and RNA sequencing, enabling in-depth investigation of mutation patterns across tumor types and their association with clinical features. Furthermore, we have integrated proteomic data from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and chromatin accessibility data from TCGA, offering new dimensions for oncogene regulation studies. OncoDB 2.0 also introduces advanced multi-omics analysis modules that facilitate the combined exploration of RNA expression, DNA methylation, and somatic mutations, allowing researchers to examine complex cross-omic relationships with greater depth and flexibility. Together, these enhancements make OncoDB 2.0 a robust and invaluable tool for the cancer research community. OncoDB 2.0 is freely available at https://oncodb.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2597afaecc32114e7746cc65fb5c9507bfe36bf0" target='_blank'>
              OncoDB 2.0: a comprehensive platform for integrated pan-cancer omics analysis.
              </a>
            </td>
          <td>
            Minsu Cho, Gongyu Tang, Charles S Rogers, Makayla Dove, Xinyi Liu, Yuanxiang Li, Xiaowei Wang
          </td>
          <td>2025-09-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The process of transcription is driven forward by the activity of kinases including CDK7, CDK9 and CDK12. Accordingly, acute inhibition of any of these kinases results in profound downregulation of gene expression. Here, we discover that loss or inhibition of CDK12 also significantly upregulates a set of coding and non-coding loci, whose activation could contribute to the anti-proliferative effects of CDK12 inhibitors. Mechanistically, CDK12 inhibition impairs transcription elongation, leading to increased RNA polymerase II termination or arrest in long genes. However, short genes such as MYC and enhancer RNAs are highly transcribed in the absence of CDK12 activity. Indeed, in HER2+ breast cancer, a malignancy where CDK12 is co-amplified with HER2 and its expression correlates with disease status, CDK12 inhibition markedly elevates MYC expression to induce lethality. The dual effects of CDK12 inhibition elucidated herein clarify its role in transcriptional control and have significant translational implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afcac9f96d92b35a58e058ad138d926ec89e0013" target='_blank'>
              Dual Modes of Gene Regulation by CDK12
              </a>
            </td>
          <td>
            Yubao Wang, Apoorva Baluapuri, Cherubin Manokaran, Jing Ni, Tao Jiang, R. Janssens, J. Marteijn, Karen Adelman, Jean J. Zhao, Thomas M Roberts
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Endometrioid and mucinous ovarian carcinomas represent nearly a fifth of ovarian cancers, but their molecular characteristics and pathologic origins are poorly understood. To identify the genomic and epigenomic alterations characteristic of these ovarian cancer subtypes and evaluate links to morphologically similar tumors from other sites, we performed a combination of sequence, copy number, mutation signature, and rearrangement analyses from tumor samples and matched normal tissues of 133 patients, as well as methylation analyses of these tumors and tissues of 150 patients from The Cancer Genome Atlas. Genomic analyses included samples from patients with ovarian endometrioid (n = 44), ovarian mucinous (n = 43), uterine endometrioid (n = 15), and gastrointestinal mucinous carcinomas (n = 31), including mucinous carcinomas of the stomach, colon, and pancreas. In addition to identifying genes previously known to be involved in these tumors, we identified alterations in RAD51C, NOTCH4, SMARCA1/4, and JAK1 in ovarian endometrioid, ESR1 in uterine endometrioid, and SMARCA4 in ovarian mucinous carcinomas. Whole-genome sequencing revealed rearrangements involving PTEN, NF1, and NF2 in ovarian endometrioid carcinomas and NF1 and MED1 in ovarian mucinous carcinomas. The number of alterations, affected genes, and genome-wide methylation profiles were not distinguishable between ovarian and uterine endometrioid carcinomas, supporting the hypothesis that these tumors share a tissue of origin. In contrast, mutation and methylation patterns in ovarian mucinous carcinomas were different from gastrointestinal mucinous carcinomas. These analyses provide insights into the genomic landscapes and origins of mucinous and endometrioid ovarian carcinomas, providing new avenues for early clinical intervention and management of patients with these cancers. Significance: Integrative multi-omic analyses support a common tissue of origin between ovarian endometrioid and uterine endometrioid carcinomas but not between ovarian mucinous and gastric or pancreatic mucinous carcinomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c6457a2db7e3ebae9f1aae5ab7a9dc09208e54" target='_blank'>
              Genomic Landscapes of Endometrioid and Mucinous Ovarian Cancers and Morphologically Similar Tumor Types
              </a>
            </td>
          <td>
            Dorothy Hallberg, A. Eastman, Shashikant Koul, D. C. Bruhm, E. Papp, Simon Davenport, V. Adleff, Leonardo Ferreira, N. Niknafs, J. Medina, S. Cristiano, C. Hruban, J. Fiksel, Kaui P Lebarbenchon, Luis Aparicio, Nicholas A. Vulpescu, Kuan-Ting Kuo, Nita Ahuja, Ronny Drapkin, Euihye Jung, Sarah H Kim, Mark A. Eckert, E. Lengyel, Kentaro Nakayama, Ayse Ayhan, I.-M. Shih, Tian-Li Wang, O. Lavie, Gadi Rennert, H. Easwaran, Stephen B. Baylin, Michael F Press, V. Velculescu, R. Scharpf
          </td>
          <td>2025-09-22</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>120</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc4a7f8e64bdd86a0871fb665e1084f20e21063" target='_blank'>
              A mobile circular DNA element drives memory-related processes in mice
              </a>
            </td>
          <td>
            Haobin Ren, Hao Gong, W. Liau, Qiongyi Zhao, Alexander P. Walsh, Mason R. B. Musgrove, J. Davies, Esmi L. Zajackowski, Sachithrani U. Madugalle, P. Marshall, Tian Lan, Jichun Shi, Jiazhi Jiang, Wei Wei, Xiang Li, T. Bredy
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histone variant macroH2A1 (mH2A1) has been widely recognized as a suppressor of gene expression. Recently, a cell cycle-dependent deposition of mH2A1 was discovered in mouse cells, but whether this process exists in human chromatin is unclear, which might be crucial for related diseases, particularly cancer. In this study, with native chromatin immunoprecipitation (nChIP-seq), we firstly demonstrate that dynamic mH2A1 domains occur in both normal and cancerous human cells and have conserved enrichment patterns across species. Our findings further provide new epigenetic insights into the role of mH2A1 in malignant proliferation, offering a novel perspective for future cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a188786f3e05b473ebb9d1416685a9aa16e12df" target='_blank'>
              Regional Conservation and Transcriptional Regulation of Tumor-Associated Genes by macroH2A1 Deposition in Mammalian Cells
              </a>
            </td>
          <td>
            Yongzhuo Deng, Zeqian Xu, Le Zhang, Bishan Ye, Zhifeng Shao, Xinhui Li
          </td>
          <td>2025-09-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb77c980017aee37cb31324f45fa9fe45acb96" target='_blank'>
              Moonlighting Role of Meiotic SYCP1 in Breast Cancer: A Chromatin-Bound Regulator of DNA Repair, Transcription, and Drug Resistance
              </a>
            </td>
          <td>
            LC Brennan, OV Grinchuk, M. Pachon, IF Sou, C. Fawcett, CG Nogueira, M. Guthrie, AD Bates, M. Hine, A. Thomaz, Z. Hu, AB Fielding, OR Davies, W-W Tee, UL McClurg
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The genome is compacted in the nucleus through a hierarchical chromatin organization, ranging from chromosome territories to compartments, topologically associating domains (TADs), and individual nucleosomes. Nucleosome remodeling complexes hydrolyze ATP to translocate DNA and thereby mobilize histone proteins. While nucleosome remodeling complexes have been extensively studied for their roles in regulating nucleosome positioning and accessibility, their contributions to higher-order chromatin architecture remain less well understood. Here, we investigate the roles of two key nucleosome remodelers, esBAF and INO80C, in shaping 3D genome organization in mouse embryonic stem cells. Using Hi-C, we find that loss of either remodeler has minimal effects on global compartment or TAD structures. In contrast, subcompartment organization is notably altered, suggesting that esBAF and INO80C contribute to finer-scale chromatin topology. To overcome the limited resolution of Hi-C for detecting regulatory loops, we employed promoter capture Micro-C (PCMC), which revealed that the loss of esBAF or INO80C alters a subset of promoter anchored looping interactions. Although these changes occur at distinct genomic loci for each remodeler, the affected sites are commonly enriched for bivalent chromatin regions bound by OCT4, SOX2, and NANOG (OSN), as well as BRG1 and INO80 themselves. Together, our findings reveal that esBAF and INO80C selectively influence subcompartment identity and enhancer–promoter communication at key regulatory loci, highlighting a previously underappreciated role for nucleosome remodelers in higher-order chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e95264023995f85a35edad63b59e9eef19fae0c" target='_blank'>
              esBAF and INO80C fine-tune subcompartments and differentially regulate enhancer-promoter interactions
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd28764a1bb58dc6ea970c4ddb741f750a9e8bc" target='_blank'>
              Genome-wide DNA bridging by H-NS reshapes the stationary phase nucleoid and transcriptional landscape
              </a>
            </td>
          <td>
            Lindsey E. Way, Xiaofeng Dai, Emma E Wiesler, Lauren A McCarthy, Zhongqing Ren, Gail G Hardy, D. E. Fuller, Labeeb Hossain, Anne S. Meyer, E. Abbondanzieri, Julie S. Biteen, Xindan Wang
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, Andreas E Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background/Objectives: Uterine leiomyomas (ULs) and breast fibroadenomas (FAs) are the most common benign tumors in women, both arising in hormone-responsive mesenchymal tissues and often co-occurring during reproductive years. Despite their shared hormonal sensitivity and frequent MED12 mutations, their downstream molecular evolution remains poorly characterized. This study aimed to investigate whether ULs and FAs, though initiated by similar genetic alterations, diverge in their oncogenic trajectories, thereby addressing the molecular basis for their distinct clinical behaviors. Methods: We performed whole-exome sequencing (WES) on 15 uterine leiomyomas and 7 publicly available fibroadenomas with matched normal controls to compare somatic mutations, copy number alterations (CNAs), and mutational signatures. All UL samples were derived from Korean patients who underwent surgical treatment at a tertiary hospital between 2017 and 2019. Somatic variants were analyzed using MuTect2 and Strelka2. FACETS was used to estimate copy number changes in individual samples, and GISTIC2 to identify recurrent and statistically significant copy number alterations across patient cohorts. Mutational processes were inferred using SigProfiler. Microsatellite instability status was determined with MSIsensor2. The study was approved by the Institutional Review Board (UC17SNSI0092). Results: Comparative whole-exome sequencing of 15 ULs and 7 FAs from matched tumor-normal samples in an East Asian cohort confirmed both tumor types harbor identical MED12 p.G44D mutations, establishing shared molecular initiation. However, post-initiation evolution diverged dramatically: ULs exhibited chromosomal instability with 15 copy number amplifications, 6 of which affected oncogenes, but relatively modest point mutations. In contrast, FAs remained chromosomally stable but hypermutated, harboring 1.47× more variants than ULs despite lower tumor purity. Notably, one histologically benign FA harbored multiple loss-of-function mutations plus an EGFR gain-of-function mutation typically associated with malignant breast cancer, challenging traditional benign-malignant classifications. Conclusions: Despite sharing a common initiating mutation in MED12, ULs and FAs evolve through fundamentally distinct genomic pathways. UL evolves through chromosomal instability, whereas FA evolves through a mutator phenotype, with important implications for understanding tumor biology and molecular-based risk stratification. These findings support a paradigm of tissue-specific oncogenic evolution and underscore the potential clinical utility of genomic profiling in distinguishing benign tumors with atypical molecular features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9889bd66df0b0830e6828b3b935a1992068cb7" target='_blank'>
              Tissue-Specific Genomic Evolution Despite Shared MED12 Mutations in Benign Tumors
              </a>
            </td>
          <td>
            Jeong Namkung, Sang Ho Park, Ayoung Hwang, Hae Seo, Jooyoung Park, Moonyoung Lee, Hyunkyung Kim, Jungmin Choi, Jaeyen Song
          </td>
          <td>2025-10-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. Here, we demonstrate that Rad27/FEN1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining the stability of the ribosomal DNA (rDNA) repeats. Severe rDNA instability is observed in the rad27∆ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates Okazaki fragments in the rDNA region, without inducing the formation of detectable DSBs. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, Exonuclease 1 and PCNA partially compensate for the loss of Rad27 in rDNA stabilization. These findings highlight the importance of proper Okazaki fragment maturation in the maintenance of rDNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f06a68a9173b59a7c4386ddec1b029c06486a7" target='_blank'>
              Rad27/FEN1 prevents accumulation of Okazaki fragments and ribosomal DNA copy number changes.
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-10-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The strength of the repressive histone H3K27me3 signal varies across silencers. Focusing on regions with unusually strong signals—super-silencers—we show that B-cell super-silencers are initially linked to gene upregulation in development, with target genes highly expressed in stem cells. About 13% of B-cell super-silencers convert to super-enhancers in B-cell lymphoma; 22% of these recur in over half of patients. Genes like BCL6 and BACH2 tied to these conversions are downregulated faster by JQ1, a super-enhancer-disrupting anti-cancer agent. Super-silencers are enriched for B-cell cancer-associated variants—both somatic and germline—and translocation breakpoints, exceeding levels in other regulatory elements like CTCF binding sites. Over 80% of B-cell lymphoma t(3;14)(q27;q32) translocations fuse BCL6 super-silencers with enhancer-rich regions. Super-silencer repression depends on CpG content: CpG-rich elements block promoter–enhancer contacts; CpG-poor — inhibit looping. These findings highlight super-silencers’ key role in B-cell regulation and suggest their alteration may be a primary factor of B-cell carcinogenesis. In B-cells, super-silencers originally help activate important genes during early development. Later, they switch roles and become powerful silencers in mature B cells. Super-silencers help control gene activity in healthy B-cells, but when they malfunction or mutate, they may drive the development of B-cell cancers. In healthy, mature B cells, super-silencers keep developmental genes switched off to maintain cell identity. When disrupted by mutation or malfunction, these elements can reverse their role and contribute to the development of B-cell cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab550e747142b856a2c8f404a6e1c22dc14158c" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chromatin remodelers are vital for cellular functions like transcription by modulating nucleosome accessibility. Although biological condensates regulate these processes, the contribution of chromatin remodelers to condensation mechanisms remains poorly understood. Here, we examine the role of the E1321 frameshift mutation in CHD1, a chromatin remodeler, which is often targeted in cancers. This mutation truncates CHD1’s C-terminus, leading to an oncogenic transcriptome and promoting tumorigenesis. This is due to the loss of an intrinsically disordered region (IDR) crucial for forming CHD1 condensates. These condensates are facilitated by the presence of H3K4me3-modified nucleosomes and RNA, guided to active promoters to regulate gene expression. Furthermore, CHD1 condensates contain long noncoding RNA and histone-modifying proteins, revealing an integral role for CHD1 condensates in epigenetic regulation. Among these components, MLL mutations frequently co-occur with CHD1 mutations in various cancers, suggesting a shared pathway in cancer development. These findings underscore the importance of chromatin remodeler condensation as a regulatory hub in various cellular processes and tumor suppression. This study reveals how a cancer-associated mutation in CHD1 leads to the loss of its C-terminal intrinsically disordered region, disrupting condensate formation. This impairment causes improper genome distribution and gene expression, driving cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ad8969965afb8f8a0d4b404e3db51ba160bcaa" target='_blank'>
              Condensation-dependent interactome of a chromatin remodeler underlies tumor suppressor activities
              </a>
            </td>
          <td>
            Yasuhiro Tsukamoto, Atsuki Kawamura, Ayhan Yurtsever, Hidefumi Suzuki, N. Rojas-Chaverra, Hiroki Sato, Daisuke Ino, Takehiko Ichikawa, Weilin Wei, Shojiro Haji, Dominic Chih-Cheng Voon, Akinobu Matsumoto, Kunio Matsumoto, Hidehisa Takahashi, Noriyuki Kodera, Takeshi Fukuma, Yoshihiro Ogawa, Masaaki Nishiyama, Katsuya Sakai
          </td>
          <td>2025-10-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress. How replication stress affects nuclear integrity is not fully understood. Here, the authors report that replication stress induces DNA damage and nuclear envelope reassembly defect during mitosis, which is suppressed by DNA polymerase theta mediated end joining.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Wei-Ke Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human tissues require a mechanism to generate durable, yet modifiable, transcriptional memories to sustain cell function across a lifetime. Previously, it was demonstrated that nanoscale packing domains couple heterochromatin (cores) and euchromatin (outer zone) into unified reaction volumes that can generate transcriptional memory. In prior work, this framework demonstrates that RNA synthesis occurred within the ideal zone (intermediate density) portions of the domain. Naturally, this creates a question of where genes are positioned in relation to the packing domain architecture and which genetic material fills the domain core to sustain transcription. Here, it is proposed that this can be solved by the encoded positioning of introns, intergenic segments, and exons as a projection of the functional packing layers of domains. This suggests that introns and intergenic segments are coupled to adjacent exons to generate coherent packing domain volumes. How this organization will reconcile contradictions in epigenetic patterns, non-randomness in oncogenic mutations, and produce durable transcriptional memory is illustrated. The study concludes by showing that this genome geometry may have coincided with the rapid evolution of body-plan complexity, suggesting that chromatin geometry could be fundamental to metazoan evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f77d30c5b35d3bee66492e606ea00f82e5f1ceb" target='_blank'>
              Argonautes and small RNAs associated with nematode programmed DNA elimination
              </a>
            </td>
          <td>
            M. Zagoskin, Yuanyuan Kang, Richard E Davis, Jianbin Wang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab22d511d2948a185e5a280612430d61e3b4b0a" target='_blank'>
              Linking Germline Telomere Removal to Global Programmed DNA Elimination in Tetrahymena Genome Differentiation
              </a>
            </td>
          <td>
            Kohei Nagao, Kazufumi Mochizuki
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract DNA damage formation and repair are influenced by the genomic landscape, yet how chromatin and transcriptional activity shape these processes at a whole-organism scale remains incompletely understood. Using Caenorhabditis elegans, a widely used model organism to study DNA repair and related processes, we present comprehensive, time-course maps of ultraviolet-induced DNA damage and excision repair, revealing how chromatin context and transcription dictate the spatiotemporal patterns of damage and repair. Of the two repair pathways—global repair and transcription-coupled repair—global repair predominates, removing the majority of the lesions; and notably, (6–4) photoproducts are removed by transcription-coupled repair at an extent comparable to cyclobutane pyrimidine dimers, a feature not previously observed in animals. Integration of damage and repair profiles with chromatin features reveals that, despite non-uniform damage formation, repair efficiency is the primary determinant of residual damage. Finally, repair around accessible regions exhibit nucleosome-size periodicity, reflecting underlying nucleosome architecture. Together, these findings establish C. elegans as a valuable model organism for interrogating damage formation and repair within a chromatin context and reveal species-specific features that broaden our understanding of DNA repair mechanisms across metazoans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb94f64070b41f072e581840675d0dbf1b209a0d" target='_blank'>
              Chromatin context shapes DNA damage formation and nucleotide excision repair dynamics in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Cansu Kose, Cem Azgari, L. Lindsey-Boltz, Ogun Adebali, A. Sancar
          </td>
          <td>2025-10-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract KRAS activating mutations occur in 90%–95% of pancreatic adenocarcinoma (PC) and contribute to tumor progression and resistance to therapy, including radiotherapy. A screen in isogenic cells revealed that KRAS activation positively modulates STN1 expression, a component of the CTC1–STN1–TEN1 (CST) complex. We find that STN1 is significantly upregulated in PC and its elevation is correlated with KRAS oncogenic mutations, while inhibition of KRAS signaling decreases STN1 expression. Interestingly, depletion of STN1 increases DNA damage and replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. STN1 silencing reduces both homologous recombination and non-homologous end joining repair of double-strand breaks (DSBs), suggesting STN1 ensures proper DSB repair. Furthermore, knockdown of STN1 impairs cell cycle arrest at G2/M phase in response to ionizing radiation, which is accompanied by increased mitotic catastrophe. Proteomic analysis reveals that STN1 physically interacts with proteins important for DNA repair, replication, and cell cycle progression, including ATM, DICER, CEP164, and CEP250. In particular, STN1 appears to stabilize ATM expression and promote proper ATM signaling after DNA damage. Our findings have revealed a novel CST complex-independent role of STN1 in DSB repair and suggest STN1 may be a promising target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad779c078ab51c12ad59d64d17f7d8c0438bc38d" target='_blank'>
              KRAS-induced STN1 (OBFC1) promotes proper CTC1–STN1–TEN1 complex-independent DNA double-strand break repair and cell cycle checkpoint maintenance in pancreatic cancer
              </a>
            </td>
          <td>
            Changxian Shen, Tiantian Cui, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, Haiqing Li, J. Karasinska, J. Topham, Daniel J Renouf, David F. Schaeffer, Anthony Fernandez, Xiaoli Ping, Binghui Shen, Jeremy M. Stark, T. Williams
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, N. Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization.
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cyclin-dependent kinases (CDKs) coordinate DNA replication and cell division, and play key roles in tissue homeostasis, genome stability and cancer development. The first step in replication is origin licensing, when minichromosome maintenance (MCM) helicases are loaded onto DNA by CDC6, CDT1 and the origin recognition complex (ORC). In yeast, origin licensing starts when CDK activity plummets in G1 phase, reinforcing the view that CDKs inhibit licensing. Here we show that, in human cells, CDK4/6 activity promotes origin licensing. By combining rapid protein degradation and time-resolved EdU-sequencing, we find that CDK4/6 activity acts epistatically to CDC6 and CDT1 in G1 phase and counteracts RB pocket proteins to promote origin licensing. Therapeutic CDK4/6 inhibitors block MCM and ORC6 loading, which we exploit to trigger mitosis with unreplicated DNA in p53-deficient cells. The CDK4/6-RB axis thus links replication licensing to proliferation, which has implications for human cell fate control and cancer therapy design. Cells must coordinate cell division with genome replication. Here, the authors combine rapid protein depletion, clinical CDK4/6 inhibitors, and genome-wide EdU sequencing to reveal that the CDK4/6-RB axis ensures timely loading of DNA replication factors in G1 phase in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1df687f2a244dc847159544edc08f4fb36a5cf" target='_blank'>
              Temporal control of human DNA replication licensing by CDK4/6-RB signalling and chemical genetics
              </a>
            </td>
          <td>
            Anastasia Sosenko Piscitello, Ann-Sofie Nilsson, Michael Hawgood, Abid H Sayyid, V. Dionellis, Giovanni Giglio, Bruno Urién, Pratikiran Bajgain, Sotirios G. Ntallis, J. Bartek, T. Halazonetis, B. Lemmens
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Whole genome doubling (WGD) triggers profound genomic and epigenetic reorganization, yet the functional dynamics of DNA methylation during this process remain incompletely resolved. Here, we integrate whole genome bisulfite sequencing (WGBS) and three-dimensional chromatin interaction data to display methylation landscapes in autotetraploid Arabidopsis thaliana. Our analysis reveals evolutionarily conserved spatial patterning of DNA methylation after WGD, with centromeric enrichment and telomeric depletion. Chromosome-level profiling identifies Chromosome 2 as the most highly methylated across CG, CHG, and CHH contexts, while Chromosome 1 shows the lowest methylation. Subcontext methylation analysis uncovers increases in methylation levels in autotetraploid Arabidopsis thaliana, most pronounced in the CHH context, yet global distribution patterns remain stable. Comparative methylation profiling around genes and transposable elements (TEs) reveals elevated CHH methylation in autotetraploid gene bodies and flanking regions, whereas TE bodies exhibit minimal changes despite minor flanking hypermethylation. Strikingly, 8% of chromatin compartments were restructured, and B-B interactions weakened in autotetraploid, while DNA methylation remained stable across shifting A/B compartments. Our findings suggest that DNA methylation serves as a resilient epigenetic modification during WGD, even if 3D chromatin architecture undergoes reorganization upon WGD in some degree.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c794ab17ab1c871d4bc77df6d82021f2b6c73a" target='_blank'>
              The Association Between DNA Methylation and Three-Dimensional Genome During Whole Genome Doubling in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Ranze Zhao, Zhongqiu Ni, Dingyu Zhang, Yuda Fang
          </td>
          <td>2025-09-24</td>
          <td>Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome conformation within the nucleus is essential for genome function. These have primarily been studied at the scale of loops and compartments, or at lower spatial resolution using traditional in situ hybridization in chemically fixed samples. However, the mesoscale organization of single chromosomes in vivo, shaped by the interplay between chromatin architectural proteins and histone modifications, remains partially understood. In this study, we interrogated the mesoscale conformations of interphase chromosomes in live human osteoblasts and transformed osteosarcoma cells, focusing on chromosome 19. Chromosome conformations were quantified by the aspect ratio of the principal axes of gyration tensors. In osteoblasts, approximately 81% of chromosome 19 are observed to consist of regions characterized by highly extended organizations, with aspect ratios approximately four times greater than those of spheres. In contrast, in osteosarcoma cells, the chromosome displays an extensively collapsed conformation, with aspect ratios more closely approximately that of a sphere. In both cell types, the chromosome’s conformation is bimodal and the balance between these two modes differs very significantly between the two cell types. While the mesoscopic conformation is considerably stable, it is superimposed on dynamic, smaller scale regions. Additional results reveal that this significant conformational shift is independent of the cell cycle but co-regulated by CTCF, cohesion, and H3K27 modifications. Our findings provide new insights into the coordinated complex regulatory mechanisms governing mesoscale chromosome organization in normal and transformed osteogenic tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f8561f1fa7ab9bbf02e738cec3deefcabedf878" target='_blank'>
              Differential regulation of mesoscale chromosome conformations in osteoblasts and osteosarcoma
              </a>
            </td>
          <td>
            Madhoolika Bisht, Yu-Chieh Chung, Siou-Luan He, Sydney Willey, Benjamin D. Sunkel, Meng Wang, Benjamin Z. Stanton, Li-Chun Tu
          </td>
          <td>2025-09-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genomic organization requires an intricate balance between the compact storage of genetic material and the ability to finely tune gene regulation. Chromatin looping achieves this balance by organizing concordantly regulated groups of genes and their regulatory elements into loops while also condensing DNA to fit into the small volume of a nucleus. A number of DNA-binding and associated proteins, including CTCF and cohesin, act as chromatin looping factors that mediate this process. Given the tight association between chromatin looping and gene expression, disordered genomic organization has been linked to disease development, including cancer. Recurrent mutations in chromatin looping factors are common in cancer, in particular blood cancers such as leukemia and myelodysplastic syndromes. In this review, we describe the evolution of our understanding of the chromatin looping process in healthy and malignant hematopoiesis and discuss the therapeutic potential of targeting chromatin looping factors in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05682afd3dc6513232a3e809f87b0eed8c538362" target='_blank'>
              Role of Chromatin Looping Factors in Leukemia.
              </a>
            </td>
          <td>
            Shira G Glushakow-Smith, Zuzana Tothova
          </td>
          <td>2025-09-15</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Balancing replication fork progression and origin usage is essential to maintain genome stability, but measuring replication fork progression rates and origin usage throughout the genome has been challenging. Here, we use nanopore sequencing combined with DNAscent to measure replication fork progression together with origin and termination site usage with single-molecule precision throughout the Drosophila genome with nearly full genome coverage. We find that replication fork progression rates are not uniform throughout the genome. Rather, fork progression is slowest in euchromatin, and this is not correlated with active transcription. Replication origins are also influenced by chromatin, but the exact position of initiation is highly variable and are often several kilobases away from ORC binding sites. Termination sites lack any chromatin or sequence motifs and appear nearly random throughout the genome. By measuring DNA replication dynamics at near full genome coverage, our work reveals key principles of metazoan replication dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0599dfed7162a33334c7070e7c9fe3430e8ac1ed" target='_blank'>
              Nanopore-based sequencing of active DNA replication reveals key principles of metazoan replication fork progression, origin and termination sites
              </a>
            </td>
          <td>
            Dongsheng Han, Cole Shepherd, Mary Lauren Benton, Jared T Nordman
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aadf2778bea4e8d563036eda473ff737cd085e3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T Tran, Clara Errard, Brian E Hew, Brandon E Johnson, David F Waller, Shanfu Xie, Joseph J Higgins, Raymond Tabibiazar, Jesse B Owens
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3066257378e89ecfacf6bd297cae20bfa3bfa3db" target='_blank'>
              p53 activity licenses transcriptional regulation by YAP/TAZ-TEAD to shape expression landscapes during tumorigenesis
              </a>
            </td>
          <td>
            Cole Martin, William B. Sullivan, Scott Bang, Joseph R. Gomes, A. C. Edwards, C. Der, Cheng Fan, Yusha Liu, John P. Morris
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Exonuclease EXO1 performs multiple roles in DNA replication and DNA damage repair (DDR). However, EXO1 loss is well-tolerated, suggesting the existence of compensatory mechanisms that could be exploited in DDR-deficient cancers. Using CRISPR screening, we find EXO1 loss as synthetic lethal with many DDR genes somatically inactivated in cancers, including Fanconi Anaemia (FA) pathway and BRCA1-A complex genes. We also identify the spliceosome factor and tumour suppressor ZRSR2 as synthetic lethal with loss of EXO1 and show that ZRSR2-deficient cells are attenuated for FA pathway activation, exhibiting cisplatin sensitivity and radial chromosome formation. Furthermore, FA or ZRSR2 deficiencies depend on EXO1 nuclease activity and can be potentiated in combination with PARP inhibitors or ionizing radiation. Finally, we uncover dysregulated replication-coupled repair as the driver of synthetic lethality between EXO1 and FA pathway attributable to defective fork reversal, elevated replication fork speeds, post-replicative single stranded DNA exposure and DNA damage. These findings implicate EXO1 as a synthetic lethal vulnerability and promising drug target in a broad spectrum of DDR-deficient cancers unaddressed by current therapies. EXO1 performs multiple roles in DNA replication and DNA damage repair (DDR), but its role in DDR-deficient cancers remains unclear. Here, the authors find EXO1 loss as synthetic lethal with many DDR genes involved in various cancers, including genes from Fanconi Anaemia pathway, BRCA1-A complex, and spliceosome factor ZRSR2; such interactions represent potential clinical targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef09b9a20a2a66068a4fe7db533a048d468729c" target='_blank'>
              EXO1 as a therapeutic target for Fanconi Anaemia, ZRSR2 and BRCA1-A complex deficient cancers
              </a>
            </td>
          <td>
            Marija Maric, Sandra Segura-Bayona, Raviprasad Kuthethur, Tohru Takaki, V. Borel, Tyler H Stanage, Miroslav P Ivanov, Nishita Parnandi, Graeme Hewitt, Rhona Millar, Carmen Fonseca, Harshil Patel, Miriam Llorian, Scott Warchal, Michael Howell, A. Chaudhuri, Panagiotis Kotsantis, S. Boulton
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3ca6346229918f35167d16749694c1e3b771aa" target='_blank'>
              STAG2 cohesin cooperates with DREAM to maintain quiescence and suppress tumourigenesis in the urothelium
              </a>
            </td>
          <td>
            Maria Ramal, E. Lapi, M. Kalisz, Jaime Martínez de Villarreal, Meritxell Novillo-Font, M. Marqués, E. Zarzuela, Marta Isasa, Osvaldo Graña-Castro, Jianming Xu, Ana Cayuela, François Radvanyi, Mireia Novell, Jorge Martínez-Torrecuadrada, M. Barallobre, I. Bernard-Pierrot, María Marti-Marimon, Maria L Arbonés, M. Marti-Renom, Francisco X. Real
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d535d363c465b65e62fbbcdfa920c1c51d8d6da" target='_blank'>
              Segregating DNA lesions point to high selective advantage of tumor initiating cells
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Maha Shady, Maria A Andrianova, Michael Spencer Chapman, E. V. Van Allen, S. Sunyaev
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91cdceed3d847b45d60afbbef0fb296224e3aa1c" target='_blank'>
              Defective proviruses cause T cell reprogramming through promoter exaptation in HIV-1 infection
              </a>
            </td>
          <td>
            M.V. Hamann, Lisa Brauckmann, Christoph Schwarz, M. Spohn, Ramon Stoeck, Sabrina M. Leddy, Julie Frouard, Maisha Adiba, F. Thiele, Sanamjeet Virdi, A. Grundhoff, Cédric Feschotte, N. Roan, E. Tolosa, Ulrike C Lange
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a28c3b0eeec817792757ba08ffa47e832addea" target='_blank'>
              The mitotic stopwatch limits the proliferation of whole genome doubled cells
              </a>
            </td>
          <td>
            LA Allan, R. Foy, M. Keli, AT Saurin
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2871bba13184fce8594e19be2ca206ed09a430e3" target='_blank'>
              Independent evolution of holocentric chromosomes in an early branching apicomplexan parasite
              </a>
            </td>
          <td>
            Abigail Kimball, Wanyi Huang, Rui Xu, Melanie Key, Lisa J. Funkhouser-Jones, L. D. Sibley, Dr. L. David Sibley
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S. Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres.
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C. Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c454f3d7e351cc169def09b1bb6a18830063a225" target='_blank'>
              PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES
              </a>
            </td>
          <td>
            Rubén Fernández, L. F. Lorenzo-Martín, Víctor Quesada, Xosé R. Bustelo
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/334d0c7aed7d2693b5a9fde9a58459b2772d8def" target='_blank'>
              Rewiring of chromatin regulation underlies the evolution of brown algal multicellularity
              </a>
            </td>
          <td>
            Jérômine Vigneau, Jaruwatana S. Lotharukpong, Pengfei Liu, Rémy Luthringer, Bérangère Lombard, D. Loew, Fabian B. Haas, Michael Borg, Susana M. Coelho
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476d94c7dbe88869f2a1eef006cd2afd67f0a95c" target='_blank'>
              Spatial Organization of Chromatin of ZEB1 Gene Promoter Region in Pancreatic Ductal Adenocarcinoma Cells
              </a>
            </td>
          <td>
            M. Zinovyeva, L. G. Nikolaev
          </td>
          <td>2025-09-22</td>
          <td>Molecular Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genetic variation at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, has been statistically associated with gene expression and other molecular phenotypes in humans. However, the causality and regulatory mechanisms for most of these STRs remains unknown. Massively parallel reporter assays (MPRA) enable testing the regulatory activity of a large number of synthesized variants, but have not been applied to STRs due to experimental and computational challenges. Here, we optimized an MPRA framework based on random barcoding to study the impact of variation in repeat copy number on expression. We first performed an MPRA on sequences derived from 30,516 promoter-proximal STR loci along with up to 152bp of genomic context, testing 3-4 variants with differing repeat copy numbers for each locus in HEK293T cells. We identified 1,366 loci with significant associations between repeat copy number and expression, which were enriched for positive effect sizes (P=2.08e-110). We then designed a second MPRA in which we performed deeper perturbations, including systematic manipulation of the repeat unit sequence, orientation, and copy number, with 200-300 perturbations for each of the 300 loci with the strongest signals. Our results revealed that the repeat unit sequence is the primary driver of differences in the relationship between copy number and expression across loci, whereas orientation and flanking sequence have weaker effects, primarily for AT-rich repeat units. The high resolution of these perturbations enabled us to detect non-linear effects, most notably for AAAC/GTTT repeats, which emerge only beyond a certain copy number threshold. Finally, we observed that a subset of STRs in our library show expression levels that are tightly linked with predicted DNA secondary structure formation. We repeated our perturbation MPRA in HeLa S3 cells under wildtype and RNase H1 knockdown conditions, which, via reduction in RNase H1 activity, are expected to hinder resolution of R-loops. This demonstrated that associations between copy number and expression at G-quadruplex-forming CCCCG/CGGGG repeats are particularly sensitive to loss of RNase H1, providing support for an R-loop mediated mechanism for these repeats. Altogether, we establish STRs as a critical component of the non-coding regulatory grammar and provide a framework for understanding how this dynamic form of genetic variation shapes gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad23e384b94840a5506d6ec02839f2c2721fcd3" target='_blank'>
              Systematic evaluation of the impact of promoter proximal short tandem repeats on expression
              </a>
            </td>
          <td>
            Xuan Zhang, Lingzhi Zhang, Ellice Wang, Susan Benton, Eduardo Modolo, Mikhail Maksimov, Sharona Shleizer-Burko, Qingqing Gong, Catherine Wang, M. Lamkin, E. Mendenhall, Melissa Gymrek, A. Goren
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Human positive coactivator 4 (PC4) is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, autophagy, B-cell differentiation, neurogenesis, DNA repair, etc. Most PC4 is phosphorylated in cells, which interacts with core histones and the linker histone H1 to confer the compact heterochromatin state of the genome. Knocking down PC4 at both cellular and organismal levels leads to significant chromatin decondensation, altered epigenetic landscape, enhanced autophagy, and increased DNA damage susceptibility. Here, we report that besides p300, PC4 also gets acetylated by DNA repair, facilitating lysine acetyltransferase KAT5 (Tip60) at a specific lysine residue (PC4K80) when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells was substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (PC4K80R), defective in KAT5-mediated acetylation. High-resolution microscopy techniques, including transmission electron microscopy and atomic force microscopy, are employed to visualize chromatin structural changes in response to DNA damage and repair in a Tip60-mediated PC4 acetylation-dependent manner. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates access to the damaged DNA by altering chromatin structures at damage sites, thus promoting DNA repair. This process could be highly significant both in cancer and in aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c83d5690dbc70212ddc740cb45975fd69abb4f3" target='_blank'>
              KAT5-mediated acetylation of PC4 facilitates DNA repair by promoting chromatin reorganization
              </a>
            </td>
          <td>
            Aa yushi A gr a w al, eta Sikder, Siddharth Singh, Rakesh Kumar, Nikhil Pallaprolu, Kalyan Mitra, Sourav Mondal, Rupa Mukhopadhy, anta Sarkar, Ramaling am Per aman, Ra vic, Tapas K. Kundu
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41f42377ee927ee8e239912c752adf5b0867faa4" target='_blank'>
              Chromosomal Restructuring and Subgenome Divergence Drive Post-Polyploid Adaptive Diversification in Sinocyclocheilus Cavefish
              </a>
            </td>
          <td>
            Tingru Mao, Yewei Liu, H. Svardal, Mariana M. Vasconcellos, Jian Yang, Liandong Yang, Shunping He, M. Meegaskumbura
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c2cc869774eac711c785244c7b4a415374fa2e8" target='_blank'>
              Saturation genome editing of BARD1 resolves VUS and provides insight into BRCA1-BARD1 tumor suppression
              </a>
            </td>
          <td>
            I. Woo, S. Casadei, M. W. Snyder, , S. Best, M. Tejura, , A. E. McEwen, M. Post, A. Hamm, M. Dawood, A. Hosokai, , R. K. Garge, S. Fayer, T. Brannan, M. E. Richardson, S. Pendyala, S. Heidl, L. Muffley, D. M. Fowler, L. Starita
          </td>
          <td>2025-11-06</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf8d0c3dbf667b289968c8f0d2992da2f2099762" target='_blank'>
              Distinct ATRX functions cooperate with 9-1-1 and CST complexes to safeguard replication and telomere integrity
              </a>
            </td>
          <td>
            Sandra Segura-Bayona, Marija Maric, Tohru Takaki, Zornitsa Manova, Shudong Li, Tyler H Stanage, Aurora I Idilli, Graeme Hewitt, P. Ruis, Rhona Millar, Harshil Patel, Steven Howell, Panagiotis Kotsantis, Michael Howell, S. Boulton
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Ovarian cancer (OVCA) is third most lethal gynecologic cancers and acquired chemoresistance is the key link in the high mortality rate of OVCA patients. Currently, there are no reliable methods to predict chemoresistance in OVCA. In our study, we identify genes, pathways and networks altered by DNA methylation in high-grade serous ovarian carcinoma (HGSC) cells that are associated with chemoresistance and prognosis of HGSC patients. We performed methylome-wide profiling using Illumina Infinium MethylationEPIC BeadChip (HM850K) methylation array on a set of HGSC chemoresistant and chemosensitive cell lines. Differentially Methylated CpG Probes (DMPs) were identified between the resistant and sensitive groups in HGSC. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) over-representation analyses were conducted to identify both common and unique pathways between resistant and sensitive cells. While the HM850K array was used for the discovery phase to identify differentially methylated probes and regions in HGSC cell lines, the publicly available The Cancer Genome Atlas ovarian cancer (TCGA-OV) dataset generated using the Illumina Infinium HumanMethylation27 BeadChip (27 K array) methylation array served as an independent validation cohort for downstream survival and drug sensitivity analyses. Machine learning methods were applied to our dataset to predict drug sensitivity in the TCGA-OV cohort and to investigate associations with overall survival and progression-free survival. Kaplan-Meier analysis was performed to assess the relationship between differentially methylated genes and patient survival outcomes. The overlapping CpG probes shared between the two Illumina platforms were used for machine learning and survival analyses. Data visualization was performed using various R/Bioconductor packages. Our analysis identified a total of 3,641 DMPs spanning 1,617 differentially methylated genes between chemoresistant and sensitive HGSC cells, whereas 80% of them were hypermethylated CpG sites associated with HGSC resistant cells. Approximately half of the DMPs were distributed on chromosomes 1–3, 6, 11–12 and 17 and top identified hypermethylated CpGs were cg21226224 (SOX17, ∆β = 79%, adj.P = 7.73E-03), cg02538901 (ATP1A1, ∆β = 75%, adj.P = 7.6E-03), and cg17032184 (CD58, ∆β = 64%, adj.P = 4.39E-02). Machine learning analysis identified significant association of global hypermethylation in the HGSC chemoresistant cells with poor overall and progression-free survival of HGSC patients. Further analysis identified four differentially methylated genes (CD58, SOX17, FOXA1, ETV1) that were also positively associated with poor prognosis of HGSC OC patients. Functional enrichment analysis showed enrichment of several cancer-related pathways, including phosphatidylinositol signaling, homologous recombination and ECM-receptor interaction pathways. This study supplements the current knowledge of the underlying mechanism behind acquired chemoresistance in OVCA. Four differentially methylated genes identified in this study may have the potential to serve as promising epigenetic clinical biomarkers for HGSC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488e6c0c8f72e40ef843295838b808f3ae35f44c" target='_blank'>
              Global DNA methylation signatures associated with chemoresistance and poor prognosis of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hafiza Bushra Manzoor, Melisa Acun, Hamed Yasavoli-Sharahi, Chantal Matar, A. Mer, Benjamin K. Tsang, Meshach Asare-Werehene
          </td>
          <td>2025-10-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf7e3130dd32248dd76079d5a6fd192ad3103a10" target='_blank'>
              Germline-limited transposon remnants foster somatic genome diversification
              </a>
            </td>
          <td>
            Bozhidar-Adrian Stefanov, M. Nowacki
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin "TE graveyards" can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA "recycling yards." Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards.
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA research : an international journal for rapid publication of reports on genes and genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Although NUP85 is a member of the nuclear pore complex (NPC) and is associated with chromosome variation and tumor regulation, its specific role in cancer development remains unclear and requires further research. This study analyzes the mRNA and protein levels of NUP85 in normal and tumor tissues using the TCGA, GTEx, and HPA databases. We investigated the relationship between NUP85 and survival rates, as well as clinical features, in various tumors by analyzing the TCGA database. The expression pattern of NUP85 in cancer cells is analyzed using single-cell sequencing data from the TISCH database. Two types of prostate cancer cell lines are utilized to investigate the impact of NUP85 on cell proliferation, migration, invasion, and apoptosis, as well as its regulatory mechanism. These analyses aim to uncover the role of NUP85 in cancer, particularly in prostate cancer. NUP85 is observed to exhibit elevated expression levels across multiple malignancies, with its heightened expression showing consistent associations with poorer clinical prognoses. Bioinformatic analyses further reveal that NUP85 expression patterns demonstrate significant correlations with the activity of cancer-related pathways and immunological interactions involving macrophages and T cell populations. Notably, experimental studies using prostate cancer models have documented reduced cellular proliferation following NUP85 knockout, suggesting a potential functional connection warranting further mechanistic investigation. The results indicate that elevated NUP85 expression shows strong correlations with cancer initiation and progression. These findings support its potential utility as a candidate biomarker for disease monitoring across various malignancies, though mechanistic validation remains necessary to establish clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d9fcec4dab152579fb48a48d3474142345170" target='_blank'>
              Comprehensive pan-cancer evaluation of NUP85 prognosis, immune infiltration response, and validation in prostate cancer
              </a>
            </td>
          <td>
            Zhijun Chen, H. Guan, Long Chen, Xinwei Yuan, Wenyan Sun, Ming Chen
          </td>
          <td>2025-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Although there are known sex differences in cancer incidence, severity, and treatment, the sex chromosomes are typically excluded from genomic analyses because of the unique technical challenges associated with assessing their copy number, sequence variation, and expression. Here we assess sex chromosome complement in three widely-used human genomics datasets from normal (non-cancerous) tissues, primary tumors, and cancer cell lines and study the effects on genome-wide gene expression. Expected sex chromosome complements based on reported patient sex were observed in non-cancerous tissues, but about half of tumors and cancer cell lines showed loss of typical sex chromosome gene expression across tissue types with three categories: loss of chromosome Y (LOY), loss of chromosome X (LOX) and reactivation of the inactive X chromosome (XaXa). Genes consistently differentially expressed in tumors with loss of chromosome X, loss of chromosome Y, or loss of X chromosome inactivation are associated with the hallmarks of cancer and include both sex-linked and autosomal genes from nearly all chromosomes, druggable genes, and genes with molecular functions relevant to cancer signaling, such as kinase activity. Strikingly, tumors that are X0, including tumors from female patients that have lost an X chromosome and tumors from male patients that have lost a Y chromosome, cluster together by gene expression profile. Patients with tumors that have LOX or LOY had poorer survival outcomes compared to those with tumors that had maintained their sex chromosome complement. Further, LOX and LOY eliminates nearly all of the differential gene expression between tumors from different patient sexes, affecting sex chromosomal and autosomal gene expression. Going forward, considering patient sex as well as the entire genome, including assessment of the sex chromosome complement, will provide additional insights into personalized tumor etiology, progression, treatment, and patient outcome. Teaser Loss of typical sex chromosome complement is present in primary tumors and cancer cell lines across tissue types. We identify consistent autosomal gene expression changes across multiple cancers when the sex chromosome complement is altered, and show that loss of sex chromosomes reduces tumor gene expression differences between patient sexes. We show that loss of sex chromosomes is also associated with poorer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2224f8588ccaf63733d9224d9f7545eea3b5b510" target='_blank'>
              Reproducible autosomal gene expression changes with loss of typical X and Y complement across tumor types
              </a>
            </td>
          <td>
            Seema B. Plaisier, Robert Phavong, Mason Farmwald, Mariah Lee, Teagen Allen, Malli Swamy, Ilsa Rodriguez, MacKenzie Wells, Nadia Phaneuf, S. Massey, Jared Del Rosario, Juvelyn Hart, Alexander Mangelsdorf, Martin Van Der Jagt, Alex R. DeCasien, K. Buetow, Melissa A. Wilson
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose FOS-like antigen 1 (FRA1), encoded by FOSL1, is an inducible subunit of the AP-1 transcription factor complex and regulates gene expression in response to proliferative and environmental cues. Although FRA1 has been linked to cancer progression, its role in early transformation and radiation responses remains unclear. Methods CRISPR-engineered human CGL1 cells—a hybrid of HeLa and normal fibroblasts—were used to evaluate the impact of FRA1 overexpression and knockout on neoplastic transformation. Transformation frequency, clonogenic survival, DNA damage recognition and repair, and cell cycle distribution were assessed following irradiation. Transcriptomic profiling was performed under baseline and serum-stimulated conditions. Results FRA1 loss markedly increased both spontaneous and radiation-induced transformation frequency, while overexpression suppressed transformation under both conditions. FRA1-deficient cells were sensitized to radiation-induced cell killing, despite intact DNA damage recognition and repair. In contrast, FRA1 overexpression promoted G2/M accumulation post-irradiation, suggesting enhanced checkpoint activation. Transcriptomic profiling revealed that FRA1 remodels AP-1 complex composition and functions as a transcriptional repressor of mitogen- and stress-responsive genes. FRA1-mediated repression was observed across gene networks involved in extracellular matrix remodeling, hypoxia signaling, inflammation, and proliferation, under both baseline and serum-stimulated conditions. Conclusion These findings establish FRA1 as a key modulator of neoplastic transformation and radiation response, acting primarily through transcriptional repression of pro-tumorigenic signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb780bfa4e21042a750aefe2ef72a09f6e8a78f" target='_blank'>
              FRA1 (FOSL1) suppresses neoplastic transformation and modulates radiation responses via transcriptional control of mitogenic and stress-responsive networks
              </a>
            </td>
          <td>
            W. Al-Khayyat, T. Laframboise, Jessica Dougherty, Marc S. Mendonca, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae37de99c671716653a1ca2bda6a664db37f1de" target='_blank'>
              GNL3 SUMOylation is essential for DNA double-strand break repair by homologous recombination
              </a>
            </td>
          <td>
            Yunhan Yang, Yanping Li, Roselyn S. Dai, Canping Chen, Keith Zientek, Ashok Reddy, Zheng Xia, Rosie Sears, Xiao-Xin Sun, Mu-Shui Dai
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6922da00f94f2dd9a584ee514f3fa6786a061103" target='_blank'>
              Genetic landscape of DNMT3A R882H reveals mechanism of aberrant oligomerization
              </a>
            </td>
          <td>
            Emma M. Garcia, Olivia R Lavidor, Shaunak Raval, Nicholas Z. Lue, Jessica K Liang, Maeve McFadden, Derek D Hwang, Marcanthony U Zepeda, Luke W Chen, Steven A. Carr, Malvina Papanastasiou, Brian B Liau
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J. Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The rapid evolution of sequencing technologies has profoundly advanced precision oncology. Whole-exome sequencing (WES), whole-genome sequencing (WGS), and whole-transcriptome sequencing (RNA-Seq) enable comprehensive characterization of tumor biology by detecting actionable mutations, gene fusions, splice variants, copy number alterations, and pathway dysregulation. These approaches also provide critical insights into biomarkers such as homologous recombination deficiency (HRD), tumor mutational burden (TMB), and microsatellite instability (MSI), which are increasingly essential for guiding therapeutic decisions. Importantly, comprehensive genomic profiling not only refines patient stratification for targeted therapies but also sheds light on tumor–immune interactions and the tumor microenvironment, paving the way for more effective immunotherapeutic combinations. WGS is considered the gold standard for detecting germline mutations and complex structural variants, while WES remains central for detecting somatic driver mutations that guide targeted therapies. RNA-Seq complements these methods by capturing gene expression dynamics, identifying clinically relevant fusions, and revealing mechanisms of resistance. Together with advances in bioinformatics and artificial intelligence, these tools translate molecular data into actionable strategies for patient care. This review integrates insights from WGS, WES, and RNA-Seq with an overview of FDA- and EMA-approved targeted therapies, organized by tumor type, and highlights the molecular signaling pathways that drive cancer development and treatment. By bridging genomic profiling with regulatory-approved therapies, we outline current advances and future perspectives in delivering personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dfbd87c701593d0e68f1b2d6aa4002ef2eff85" target='_blank'>
              Advances in Precision Oncology: From Molecular Profiling to Regulatory-Approved Targeted Therapies
              </a>
            </td>
          <td>
            P. Brlek, V. Škaro, Nenad Hrvatin, Luka Bulić, Ana Petrović, Petar Projić, Martina Smolić, Parth Shah, D. Primorac
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The advent of poly (ADP-ribose) polymerase inhibitors (PARPi) over the past decade has significantly altered the management of epithelial ovarian cancer (EOC). We proposed that the etiology of homologous recombination deficiency (HRD) might underlie the variable responses to PARPi observed across patient populations. As part of the phase 2 study of the Chinese HRD Harmonization Project, we developed a genomic methylation sequencing (GM-seq) pipeline facilitated by the TET enzyme for the simultaneous identification of methylated modifications and genetic variations in EOC tumor samples, and compared with established DNA sequencing-based HRD assays. Somatic mutation and HRD scores were confounded by low tumor purity in our cohort of 98 locally advanced/advanced EOC patients. In samples with tumor purity ≥ 30% (n = 45), the GM-seq pipeline showed high consistency with DNA sequencing-based HRD assay, identifying genetic variations in homologous recombination repair (HRR) genes and HRD score with 92.6% (25/27) and 97.1% (33/34) consistency respectively, in addition to conducting methylation profiling. Moreover, different underlying mechanisms of HRD were associated with varying degrees of PARPi efficacy, with BRCA1/2 LOH group having the best efficacy (median PFS, undefined), followed by BRCA1 methylation group (median PFS, 23.4 months), and those with unknown etiology of HRD having the worst efficacy (median PFS, 8.8 months, p < 0.001). Our findings underscore the importance of considering HRD etiology when evaluating PARPi efficacy in EOC patients. The GM-seq pipeline, represents a significant advancement in HRD detection, enabling more accurate predictions of PARPi response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5213220b5e2f3e35a5d8562e34edf897da4714a0" target='_blank'>
              Comprehensive assessment of homologous recombination deficiency via simultaneous methylation and mutation analysis in epithelial ovarian cancer: implications for PARP inhibitors efficacy
              </a>
            </td>
          <td>
            Lin Dong, Huanwen Wu, Ning Li, Wenbin Li, Yan Song, Yu Xiong, Huan Yin, Huan Fang, Rongrong Chen, Xin Yi, Jie Huang, Jianming Ying
          </td>
          <td>2025-10-10</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Centrosome amplification (CA) has been observed in various solid tumors and contributes to chromosomal instability (CIN) and poor clinical prognosis in patients with cancer. CA also inhibits cell proliferation by inducing cell-cycle arrest and cell death. However, the mechanism of regulation of centrosome number in cancer cells and its effect on CIN and cell proliferation remains elusive. Here, we report that tuftelin (TUFT1) is a novel centrosomal protein that localizes to the proximal ends of parent centrioles. TUFT1 prevents CA and mitotic spindle multipolarity by suppressing premature polo-like kinase 1 activation. In addition, TUFT1 is phosphorylated by NIMA-related kinase 2 (NEK2), and the phosphorylation status of TUFT1 is essential for coordinating centrosome number and cell proliferation in cervical and breast cancers. Data from clinical breast cancer samples indicate that the combined detection of TUFT1 and NEK2 expression reflects tumor malignancy and patient survival with higher precision. Overall, these results reveal a crucial role of TUFT1 in the regulation of tumor progression through centrosome number control. Thus, TUFT1 represents a promising target for diagnostic and therapeutic approaches for cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efc995dda20af305e1ea407bedfd835bb75716c" target='_blank'>
              TUFT1 regulates cancer progression by suppressing centrosome amplification and mitotic spindle multipolarity
              </a>
            </td>
          <td>
            Shaojie Feng, Mengmeng Zhao, Donghui Zhang, Yan Zhang, Lingyuan Min, Xianqiang Liu, Huan Shi, Tianning Wang
          </td>
          <td>2025-09-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Many gene fusions have been implicated in the development and progression of human cancers. Our group and others have identified specific gene fusions involving the MAML2 gene in human cancers, and we were interested in searching for additional novel MAML2 fusions. Our analysis with the Caris Life Sciences molecular database demonstrated that most of the novel fusions were associated with increased fusion burden and genomic loss of heterozygosity, which are potentially suggestive of genomic instability, except for ATXN3::MAML2 which warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa520a41486a278de5084f190cbe8e828a82921" target='_blank'>
              Novel MAML2 Fusions in Human Malignancy
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao H. Huang, A. Crymes, E. S. Antonarakis, Andrew Elliott, Matt Oberley, Mark G. Evans
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored.



 We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls.




 AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g.,
 FLT3
 ,
 RUNX1
 , and
 CD33
 ) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML.




 This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgen Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Among them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed a synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2446ce02999836acb1c038122ee522f1878bf0c3" target='_blank'>
              TopBP1 biomolecular condensates as a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, B. Bordignon, C. Hassen-Khodja, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Laurent Chaloin, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2025-10-21</td>
          <td>eLife</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution.
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract NUPTs are DNA sequences of plastid origin that are present in plant nuclear genomes in varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, they are removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analyses from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are: (i) formed by both continuous and episodic mechanisms; (ii) unevenly represented across the plastid genome; (iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA, and regulatory RNA genes; (iv) differentially contributing to structural genes; and (v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which not only involve their continuous formation but also episodic events, highlight their role as a major source of noncoding RNA genes and other genomic features, and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and Ongoing Mechanisms Drive Plastid-Derived Nuclear DNA Evolution in Angiosperms
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily L. Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Centromeres are fundamental chromosomal structures that ensure accurate chromosome segregation during cell division. Despite their conserved and essential role in maintaining genomic stability, centromeres are subject to rapid evolutionary change. At the heart of centromere identity is the histone H3 variant CENP-A, an epigenetic mark that defines and propagates active centromeres and is essential for their function. Recent evidence supports a rapid evolution of centromere DNA sequences but also suggests a certain degree of flexibility in CENP-A deposition and propagation. The phenomenon of centromere drift, recently observed in humans, highlights how the dynamic repositioning of CENP-A and associated epigenetic environment over time maintains a regulated equilibrium, ensuring centromere function despite positional variation. Understanding these processes is crucial for unraveling centromere dynamics and their broader implications for genome stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0e9c5778af501dce621770da4beeabba7210f5" target='_blank'>
              The dynamic centromere
              </a>
            </td>
          <td>
            Angela Enriquez, Yael Nechemia-Arbely
          </td>
          <td>2025-10-16</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ∼2.5×10-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G. Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="ABSTRACT Chromosome organization is critical for the maintenance of genetic integrity. Most studies of bacterial nucleoids have focused on growing rod-shaped organisms. Studying nucleoid dynamics during Myxococcus xanthus development offers the unique opportunity to investigate the localization of two sister chromosomes as rod-shaped cells transition into round spores. During starvation-induced multicellular development, DNA replication is required for M. xanthus rods to transition into spores with two copies of the chromosome. Here, we report novel approaches using confocal fluorescence microscopy to observe the chromosome number and arrangement, and nucleoid localization in developing cells in situ. We discovered that sister chromosomes are present in some rods and transitioning cells (TCs) early in development. The arrangement of the two chromosomes in developing cells was novel compared to predivisional growing cells studied previously. We observed segregated nucleoids in ~40% of TCs and spores. The majority of TCs contained a crescent-shaped nucleoid along one side, perhaps due to ongoing chromosome segregation, whereas most spores appeared to have undergone nucleoid decondensation. During unicellular glycerol-induced sporulation of M. xanthus, we observed segregated nucleoids in only ~10%–20% of TCs and spores. In addition, early in starvation-induced development, we discovered a subpopulation of cells that may be spheroplasts destined for lysis, which is the fate of most cells under these conditions. Chromosome segregation in developing M. xanthus may be a bet-hedging strategy to increase survival under different conditions and/or an evolutionary remnant of ancestral events that included cell division to produce spores with one copy of the chromosome. IMPORTANCE The cell cycle normally involves DNA replication, chromosome segregation, and cell division. During starvation-induced Myxococcus xanthus development, DNA replication is necessary for progression to spore formation, which occurs without cell division, resulting in spores with two copies of the chromosome. The organization of sister chromosomes during the morphological change of rod-shaped cells into round spores was unknown. We discovered that the two nucleoids often segregate during the transition from rods to spores. Mature spores contained decondensed nucleoids. Our observations raise important questions about the mechanism of chromosome segregation during M. xanthus development and the reason for its existence. We also discovered a subpopulation of developing cells with characteristics suggesting they are spheroplasts on the verge of cell death. The cell cycle normally involves DNA replication, chromosome segregation, and cell division. During starvation-induced Myxococcus xanthus development, DNA replication is necessary for progression to spore formation, which occurs without cell division, resulting in spores with two copies of the chromosome. The organization of sister chromosomes during the morphological change of rod-shaped cells into round spores was unknown. We discovered that the two nucleoids often segregate during the transition from rods to spores. Mature spores contained decondensed nucleoids. Our observations raise important questions about the mechanism of chromosome segregation during M. xanthus development and the reason for its existence. We also discovered a subpopulation of developing cells with characteristics suggesting they are spheroplasts on the verge of cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe53b6817f634726006f01b89cdd1037a4dfb40" target='_blank'>
              Segregation of sister chromosomes during the shape change of developing Myxococcus xanthus cells
              </a>
            </td>
          <td>
            Y. Hoang, Yann S Dufour, Lee Kroos
          </td>
          <td>2025-09-26</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Neural tube defects (NTDs) are one of the most common congenital malformations. Folic acid deficiency in pregnant women increases the risk of developing NTDs; however, the underlying etiology and mechanisms remain elusive. In this study, the role of DNA double-strand breaks (DSBs) in 3D genome organization in NTDs with folate deficiency is reported. The NTD mouse model is burdened with abundant DSBs associated with the disruption of 3D genome organization. DSBs occurring in active genes lead to the stalling of RNA polymerase II (Pol II) and formation of R-loops in the 3D genome. The DSB ratios of the genomic regions negatively correlated with the distance from the transcription start sites of the gene. The DSB ratios of the proximal and distal enhancers are significantly higher and induce the displacement of loops with busy anchors. Furthermore, DSB-associated dysregulation of chromatin loops occurs in neural tube closure-associated genes that are abnormally expressed in human NTDs. Taken together, excessive DSB-associated 3D genome organization disruption within NTDs with folate deficiency contributes to the dysregulation of neural tube closure-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8659be399a2187d7c9eb42445c50713cbee2a3dc" target='_blank'>
              Excessive DNA Double-Strand Breaks-Associated 3D Genome Reorganization Contributes to Neural Tube Defects with Folate Deficiency.
              </a>
            </td>
          <td>
            Ting Zhang, Lin Lin, Jianting Li, Caihua Li, Shengjun Liang, Xuemei Bai, Fang Wang, Y. Bao, Dan Guo, Xiaochen Bo, Hao Li, Hebing Chen, Qiu Xie
          </td>
          <td>2025-09-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Bladder cancer is rare in young women, and advanced presentations are exceptionally uncommon. We report a de-identified case of a previously healthy 31-year-old female who developed rapidly progressive stage IV bladder cancer within 12 months of completing a three-dose Moderna mRNA vaccination series (May 2021, June 2021, December 2021). Case Findings: Comprehensive multi-omic profiling was performed using PBIMA (Molecular Surveillance and Individualized Targeted Immunotherapy Peptide Editing) and REViSS (Spike-associated Transcription-al/Translational Instability Surveillance), incorporating analyses of plasma-derived circulating tumor DNA, whole-blood RNA, and urine exosome proteomics. Dysregulated gene expressions were identified across oncogenic driver genes (KRAS, ATM, MAPK1, NRAS, CHD4, PIK3CA, and SF3B1), auxiliary tumor-promoting signals (TOP1, PSIP1, and ERBB2), and broad evidence of genome instability with impaired DNA repair (ATM, MSH2). Within circulating tumor DNA, a host–vector chimeric read mapped to chr19:55,482,637–55,482,674 (GRCh38), in cytoband 19q13.42, positioned ~367 kb downstream of the canonical AAVS1 safe harbor and ~158 kb upstream of ZNF580 at the proximal edge of the zinc-finger (ZNF) gene cluster. This sequence aligned with perfect 20/20 bp identity to a segment (bases 5905–5924) within the Spike open reading frame (ORF) coding region (bases 3674–7480) of the Pfizer BNT162b2 DNA plasmid reference (GenBank accession OR134577.1), despite the patient only receiving Moderna vaccinations. This apparent paradox is best explained by shared Spike ORF sequences within the expression cassette across both vaccine platforms; because Moderna has not deposited its proprietary plasmid sequence in NCBI, BLAST defaults to Pfizer’s published reference as the nearest available match. The integration site was located outside the canonical AAVS1 “safe harbor” and within a gene-dense, recombination-prone regulatory region, raising concern for transcriptional disruption, fusion transcript formation, and oncogenic potential. The probability of a random 20-base sequence perfectly matching a predefined target is approximately 1 in a trillion, making this alignment statistically compelling and highly unlikely to be an incidental artifact. Conclusions: This sentinel case report provides the first documented evidence of genomic integration of mRNA vaccine-derived genetic material in a human subject, documenting a temporal association between COVID-19 mRNA vaccination and aggressive malignancy, reproducible multi-omic evidence of oncogenic signaling, and a non–safe harbor host–vector integration event. While causality cannot be established from a single case, the con-vergence of (i) close temporal proximity to vaccination, (ii) genomic integration of a vaccine plasmid–derived spike gene fragment, and (iii) consistent transcriptomic and proteomic instability across biospecimens represents a highly unusual and biologically plausible pattern. These findings highlight an urgent need for systematic genomic sur-veillance, orthogonal validation with long-read sequencing, and larger cohort studies to rigorously define the im-pact of synthetic mRNA vaccine platforms on genome integrity and cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8848f5c106afb2dfada322a3642c497ad8b443d" target='_blank'>
              Genomic Integration and Molecular Dysregulation in Aggressive Stage IV Bladder Cancer Following COVID-19 mRNA Vaccination
              </a>
            </td>
          <td>
            J. A. Catanzaro, Nicolas Hulscher, Peter A. McCullough
          </td>
          <td>2025-10-05</td>
          <td>International Journal of Innovative Research in Medical Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9f14cca351b80c4eaf8fa50610851b494729d1" target='_blank'>
              Noncanonical chromosomal-end-specific telomeric arrays in naturally telomerase-negative yeasts
              </a>
            </td>
          <td>
            Broňa Brejová, Viktória Hodorová, Hana Lichancová, A. Gafurov, Dominik Bujna, Filip Brázdovič, Filip Červenák, Tomáš Petrík, Eva Hegedűsová, M. Jakúbková, M. Neboháčová, Ľubomír Tomáška, Matthias Sipiczki, T. Vinař, J. Nosek
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1532e1a41f5f79814eb3fb790e35c92b5a006139" target='_blank'>
              The Androgen Receptor and MYC synergise to modulate the synthesis of Siglec-7 ligands in prostate cancer
              </a>
            </td>
          <td>
            Adam Duxfield, Rebecca Garnham, Esme Hutton, Adam A. Dowle, Jamie Wills, Sara Luzzi, Ryan Nelson, Emma Lishman-Walker, Rianna Magee, A. Buskin, A. Hepburn, Emirhan Tekoglu, Kaidan Maloney-Friar, Fiona Frame, Norman Maitland, Ann Hedley, Holly Henderson, Benjamin McCullough, Bharat Gowardhan, K. Sahadevan, S. McCracken, L. Gaughan, Rakesh Heer, Craig N. Robson, Kelly Coffey, Nathan Lack, Nathalie Signoret, M. Fascione, Emma Scott
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DSCC1, a key subunit of the alternative RFC complex, is essential for DNA replication and genome stability. While its role in cancers like breast cancer is documented, its function in LIHC remains largely unexplored. This study comprehensively examines DSCC1's involvement in LIHC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6e78aa5a515dfa7feb5f0abf3158e965c6ce93d" target='_blank'>
              DSCC1 as a Novel Biomarker in Hepatocellular Carcinoma: Diagnostic Potential, Prognostic Impact, Correlation With Key Pathways, and Association With the Immune Landscape
              </a>
            </td>
          <td>
            Abdul Jamil Khan, Shad Man, Shahid Ullah Khan
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Gene Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA), the most abundantly expressed locus in the human genome, is represented by hundreds of units per cell. Each unit includes a 13 kb long region (47S rDNA) containing the genes of ribosomal particles, and a 30 kb long intergenic spacer (IGS). The 47S rDNA is transcribed with varying intensity in different units, some of which remain permanently silent. A key intermediator of this silencing is the promoter-associated RNA (pRNA) produced from a 2 kb long gene situated upstream of the rDNA transcription start site. Recent studies, including ours, suggest that the sequence variability, which normally occurs in mammalian cells, may account for the selective transcription of different rDNA units. The present work is based on the deep sequencing of a pRNA gene fragment and its RNA product and subsequent bioinformatic analysis. We found that a certain SNV, which converts the CCC motif into CCT, as well as deletions which reduce the number of (CCCT) tandem repeats, were significantly more frequent in the DNA than in the respective transcripts. These findings allowed us to establish directly the inhibitory effect of DNA variants on the expression of pRNA and thus (indirectly) the promoting effect on the production of ribosomal RNA. Our results also suggest that (CCCT)n/(GGGA)n DNA repeats and the respective (GGGA)n RNA repeats may form triplex structures, facilitating the function of pRNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891924cd9bf55653e2539c5b0034f6d9fcdd744" target='_blank'>
              Deletions in (CCCT)n repeat regions belonging to the human pRNA gene inhibit its expression
              </a>
            </td>
          <td>
            Nikola Chmúrčiaková, Adam Nógell, E. Smirnov, Dušan Cmarko
          </td>
          <td>2025-10-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1892593b4d09d954d617fd50a949c2f67aa1bce" target='_blank'>
              Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin
              </a>
            </td>
          <td>
            Agnisrota Mazumder, John Cooper, Can Goksal, J. S. Khanduja, Richard I. Joh, Johannes J. Groos, Rosario Y.J. Brockhausen, Junko Kanoh, Mohammad R. Motamedi, Ilya J. Finkelstein, Bassem Al-Sady, S. Braun
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1147c4ec4d5050e89e9fab52610e8f2d66fcc3cc" target='_blank'>
              Integrative multi-modal analysis reveals the contribution of noncoding RNAs to post-treatment progression of IDH-mutant astrocytomas
              </a>
            </td>
          <td>
            A. Hartewig, S. Jaatinen, M. Annala, J. Haapasalo, Wei Zhang, H. Haapasalo, M. Nykter, Xin Lai, K. Rautajoki
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major contributor to female mortality worldwide. Treatment resistance and tumor heterogeneity contribute to the lack of effective therapeutic targets, posing a significant challenge in BCa management. CENPI, a core centromere protein involved in chromosome segregation, has emerging evidence implicating it in oncogenesis across diverse malignancies. However, its functional and molecular mechanisms in BCa remain unclear. We analyzed CENPI expression and its clinical significance by using the BCa dataset from the Cancer Genome Atlas (TCGA) and immunohistochemical staining of 3 human BCa tissue samples. Cellular functional assays and mice xenograft models were utilized to assess the effects of CENPI on BCa growth. RNA sequencing combined with bioinformatics analysis was conducted to elucidate the molecular mechanisms underlying CENPI function, with further validation through Western blotting, immunofluorescence, and TOP/FOP flash assays. CENPI was aberrantly overexpressed in BCa, with elevated expression levels strongly associated with disease progression and poor prognosis. Functional assays demonstrated that CENPI significantly promoted breast carcinogenesis in both cellular and animal models. Mechanistically, CENPI increased BCa progression and malignant phenotypes by modulating the Wnt/β-catenin axis. CENPI is a critical oncogene in BCa, driving tumorigenesis and disease progression via the Wnt/β-catenin axis, which represents a promising biomarker and therapeutic target for BCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33eef316f873dd58287f9f4d71241b7970f53004" target='_blank'>
              Centromere protein I facilitates breast cancer tumorigenesis and disease progression through modulation of Wnt/β-Catenin signaling
              </a>
            </td>
          <td>
            Chaoshen Wu, Yibing Zhou, Yuxiao Mu, Yijie Cheng, Wenqian Xu, Mengna Huang, Jingyuan Xu, Ming Li, Jianfeng Gu, Xuli Meng, Da Qian
          </td>
          <td>2025-10-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mitotic chromosomes give genome portions the required compaction and mechanical stability for faithful inheritance during cell divisions. They are shaped by the chromosomal condensin complex. Here, we record human chromosome dimensions from their appearance in prophase over successive times in a mitotic arrest. Chromosomes first appear long and uniformly thin. Then, individual chromosome arms become discernible, which continuously shorten and thicken—the longer a chromosome arm, the thicker it becomes. In the search for a molecular explanation of this behavior, given uniform condensin density, the popular loop extrusion model provides no obvious means by which longer chromosome arms become thicker. Instead, we find that simulations of an alternative loop capture model recapitulate key features of our observations, with re-arranging chromatin rosettes underpinning the gradually developing arm length-to-width relationship. Our analyses portray chromosomes as out-of-equilibrium structures in the process of transitioning towards, but on biologically relevant time scales not typically reaching, steady state. Chromosomes continuously develop their shape during cell divisions. Mitotic chromosomes gradually shorten and thicken. Longer chromosome arms become thicker than short arms. Anaphase sets in before most longer chromosome arms reach steady state. Loop capture simulations recapitulate observed chromosome shape changes. Mitotic chromosomes gradually shorten and thicken. Longer chromosome arms become thicker than short arms. Anaphase sets in before most longer chromosome arms reach steady state. Loop capture simulations recapitulate observed chromosome shape changes. Chromosomes continuously develop their shape during cell divisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3055d8d7767edb2d5ac8b163761950bc3ca2e716" target='_blank'>
              Progressive chromosome shape changes during cell divisions
              </a>
            </td>
          <td>
            Yasutaka Kakui, Y. Kusano, Tereza Clarence, Maya Lopez, T. Fallesen, T. Hirota, Bhavin S. Khatri, Frank Uhlmann
          </td>
          <td>2025-09-23</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is the most frequent soft tissue sarcoma of childhood. The fusion positive (FP) subtype expressing the chimeric transcriptional factor PAX3-FOXO1, the driver of malignancy, has limited treatment options due to resistance to treatments. As a pediatric cancer, RMS has a low mutational burden with epigenetic transcriptional deregulations playing key roles. Thus, targeting the basic transcriptional machinery including catalytic effectors of transcription is an emerging anti-cancer approach. The transcriptional cyclin-dependent kinase CDK12 regulates the transcription elongation phase, particularly of the DNA damage response (DDR) genes, by phosphorylating Ser2 (pSer) on RNA PolII. In Ewing sarcoma, CDK12 inhibition leads to a DNA repair deficient phenotype and high sensitivity to DNA damaging agents. PAX3-FOXO1 RMS cells display elevated levels of replication stress highly relying to DDR for survival. Therefore, in this study we evaluate the impact of CDK12 inhibition on PAX3-FOXO1 RMS tumorigenic features. We found that CDK12 mRNA levels were highly up-regulated in RMS primary samples compared to healthy control muscles, with one of the highest expression among the adult and pediatric tumors analyzed. Then, we treated PAX3-FOXO1 cell lines with THZ531, an inhibitor of CDK12 and its paralog CDK13. Six hours of THZ531 treatment at nanomolar concentrations decreased pSer2 of RNA PolII and reduced PAX3-FOXO1 transcripts and protein levels. The drug also reduced cell survival in 2D and 3D culture conditions. THZ531-treated cells accumulated in G2/M phase and underwent PARP- and Casp 3/7-dependent apoptosis, increased markers of DNA damage associated to DNA double-strand breaks (neutral comet assay), and reduced the levels of the DDR players ATM, ATR, RAD51, BRCA1 and BRCA2. Similar results were obtained after treatment with another CDK12/13i with Cyclin K degrader ability, SR-4835, and with a CDK12-specific degrader, BSJ-4-116, suggesting on-target effects. The three drugs showed similar low nanomolar concentrations effective on PAX3-FOXO1 RMS organoids. Preliminary RNAseq data in two PAX3-FOXO1 RMS cell lines showed that THZ531 downregulated only a low number of genes, common to the two cell lines. The most downregulated gene families were those of DDR pathways. Moreover, a negative correlation between gene length and gene expression levels was found. CDK12 KO in two RMS cell lines affected cell survival in 2D and 3D conditions compared to control cells suggesting on-target activity of the drugs. Phosphoproteomics analysis after THZ531 treatment showed a significant decrease of a number of serines and threonine phosphopeptides also involving a subset on DDR proteins. THZ531 treatment resulted synergic in vitro with Olaparib (PARPi) and with JQ1 (BRD4i). Interestingly, BRD4 is needed for PAX3-FOXO1 activity on oncogenic super-enhancers. Analysis of nascent RNAs and mapping of RNA PolII to identify direct targets of CDK12 is ongoing. Supported by Associazione Italiana per la Ricerca sul Cancro (AIRC) IG #27794 grant to RR



 Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, Francesca Antonella Aiello, Matteo Cassandri, Silvia Pomella, Adam D. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T Meister, Jarno Drost, Concetta Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, Biagio De Angelis, Kristy R Stengel, Rossella Rota. CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd33df041a8224c723527c37b828d7c14637d9c" target='_blank'>
              Abstract B015: CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, F. Aiello, Matteo Cassandri, S. Pomella, A. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T. Meister, J. Drost, C. Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, B. De Angelis, Kristy Stengel, R. Rota
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d053aeb5c620a529619ef894dd48c56ee4b7d249" target='_blank'>
              Arginine Metabolism Supports De Novo Pyrimidine Biosynthesis to Block DNA Damage and Maintain Epstein-Barr Virus Latency
              </a>
            </td>
          <td>
            Shaowen White, Yifei Liao, Eric M Burton, John M. Asara, B. Gewurz
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Yuheon Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The transcription of satellite DNA is highly sensitive to environmental factors and represents a source of genomic instability. Therefore, tight regulation of (peri)centromeric transcription is essential for genome maintenance. Antibiotics are routinely used for in vitro studies and for medical treatment, however, their effect on pericentromeric satellite DNA transcription was not investigated. Here we show that antibiotics geneticin and hygromycin B, conveniently used in cell culture, as well as rifampicin (along with five other antibiotics), used to treat bacterial infections, increase transcription of a major human pericentromeric alpha satellite DNA in cell lines at standard concentrations. However, response differs among cell lines - maximal increase in A-1235 cells is obtained by rifampicin while in HeLa cells and fibroblasts by geneticin. There is also a positive correlation between antibiotic concentration and the level of alpha satellite transcription. The increase of transcription is accompanied with either H3K9me3 decrease or H3K18ac increase at tandemly arranged alpha satellite arrays while H3K4me2 remains unchanged. Our results suggest that induced alpha satellite DNA transcription upon antibiotic stress could be linked to epigenetic changes - histone modifications H3K9me3 and H3K18ac, which are associated with transcription of heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03ef88bdc2c47f36c0fdbaf887b60122c711579a" target='_blank'>
              Antibiotics induce overexpression of alpha satellite DNA accompanied with epigenetic changes at alpha satellite arrays as well as genome-wide
              </a>
            </td>
          <td>
            Sven Ljubić, Maja Matulić, Damir Đermić, Maria Chiara Feliciello, Alfredo Procino, Đurđica Ugarković, I. Feliciello
          </td>
          <td>2025-09-26</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106cf167da77fcf718e98566d67aa1a48f009f0c" target='_blank'>
              Telomere-to-telomere gap-free genome assembly of a male donkey and the identification of novel SVs associated with functional genes
              </a>
            </td>
          <td>
            Tao Yang, Ran Yang, Yu Liu, Mo Feng, Yuanyuan Li, Hetong Zhang, Xinyu Wang, Ruyu Yao, Jiahui Wu, Wenhui Xing, Shiyu Qian, Chunjiang Zhao
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b870d32a021571695dc3e8913c757c0de67eb433" target='_blank'>
              Deciphering glioma susceptibility
              </a>
            </td>
          <td>
            A. Fratton, B. Bonev
          </td>
          <td>2025-10-01</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="CTCF is a multifunctional protein that mediates long-range cis-DNA interactions in mammalian genomes. Chromatin architecture governs spatial and functional interactions of gene regulatory elements at various loci and is impacted by the ability of CTCF to restrict cohesin complex dependent chromatin extrusion. In addition, at antigen receptor loci, long-range interactions facilitate spatial proximity of gene segments for VDJ recombination that generates functional genes encoding immunoglobulins and T-cell receptors in developing lymphocytes. To investigate the role of CTCF in VDJ recombination, we mutated CTCF binding sites (CBS) of murine Tcrb locus. Our analysis revealed that CBS interspersed in the domain encompassing variable gene segments (Vb) are not redundant. They exhibit independent but additive effects on dynamic chromatin organization leading to distinct VDJ recombination profiles in CBS mutants depending on positions of mutated CBS relative to Vb segments. Further, inversion of a single CBS drastically altered the chromatin loop organization and VDJ recombination profile. Our results demonstrate the critical importance of chromatin extrusion for generation of chromatin loops for VDJ recombination and underscore its dynamic impediment by CTCF binding at specific points within Vb segment domain to be essential to diversify the usage of Vb segments for VDJ recombination at Tcrb locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac90d7bd76dd65f015613a39e4dba67c4c978f" target='_blank'>
              Site Specific and Orientation Dependent CTCF Binding Determines VDJ Recombination at Murine Tcrb Locus.
              </a>
            </td>
          <td>
            Monika Yadav, Samriddhi Sharma, S. R. Rajalekshmi, Nidhi Bhasin, Jeanne Yimdjo, Karl Pfeifer, M. Srivastava
          </td>
          <td>2025-09-18</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fb5c415bb77b25342add7bc3b95281a99e2297" target='_blank'>
              Metagenomic approaches for studying ubiquitous yet diverse nucleoid associated proteins in microbial communities: challenges and advances
              </a>
            </td>
          <td>
            Harsh V Purohit, Joydeep Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>World Journal of Microbiology and Biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The O6-methylguanine-DNA methyltransferase (MGMT) plays a significant role in the pathogenesis and progression of glioma. Numerous enhancer variants, including those within the MGMT gene region and adjacent gene regions, have been found to be associated with cancer development and progression. We investigated the significance of enhancer variants located in the intergenic spacer far from the MGMT gene in relation to glioma susceptibility and progression. We recruited 402 glioma patients and 654 controls for this investigation using Sequenom MassARRAY genotyping. We identified a significantly elevated risk of glioma among carriers with the rs11016629 TG genotype compared to those with the GG genotype (OR = 1.41, 95% CI 1.03–1.93; P = 0.034). Subgroup analyses revealed that rs11016629 was significantly associated with glioma risk in subjects with WHO grade IV tumor (OR = 1.59, 95% CI 1.07–2.38; P = 0.023) and high-grade glioma (OR = 1.57, 95% CI 1.11–2.21; P = 0.011). Patients who underwent gross total resection with TG/TT genotypes exhibited a 2.66-fold higher risk of disease progression than GG carriers (HR = 2.66, 95% CI 1.23–5.79; P = 0.014). The study demonstrates that a MGMT enhancer variant rs11016629 contributes to both glioma susceptibility and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ffd3767daacb020b869e9275c5c5550f74b90c" target='_blank'>
              A MGMT Enhancer Variant is Associated with Glioma Susceptibility and Progression
              </a>
            </td>
          <td>
            Shu Zhang, Yiji Chen, Wenshen Xu, Danfang Yan, Meiqin Chen, Jiayi Zheng, Liming Huang
          </td>
          <td>2025-10-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a83ebfb77b99a082f33dfac7fdc4e834369069a" target='_blank'>
              Gastric epithelium from BRCA1 and BRCA2 carriers harbors increased double-stranded DNA damage and augmented growth
              </a>
            </td>
          <td>
            Kole H. Buckley, Michaela Dungan, Kevin Dinh, Gregory M. Kelly, Ryan Hausler, Kate E. Bennett, Daniel Clay, Julia E. Youngman, Ariana D. Majer, Keely Beyries, Blake A. Niccum, Sydney M. Shaffer, Tatiana A. Karakasheva, Kathryn E. Hamilton, M. Kochman, Gregory G Ginsberg, N. Ahmad, Kara N Maxwell, Bryson W. Katona
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f7c1744fc2f1ed50403a7b109097213e18ec402" target='_blank'>
              Design and Development of DNA Damage Chemical Inducers of Proximity (DD-CIP) for Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Tian Qiu, Yeuan Ting Lee, Brendan G. Dwyer, Yi Jer Tan, Ting Chen, Bryan A. Romero, Yanlan Wang, Jiehui Deng, Tinghu Zhang, G. R. Crabtree, Stephen M. Hinshaw, , N. Gray
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gastric cancer (GC) metastasis remains a major cause of poor prognosis, yet its molecular drivers are poorly understood. Here, we integrated single-cell RNA sequencing (scRNA-seq) of primary tumors and matched metastatic lymph nodes from six GC patients to identify a metastatic epithelial subpopulation characterized by EGR4 overexpression. Kaplan-Meier analysis revealed that high EGR4 expression correlated with reduced survival in GC patients. Mechanistically, chromatin immunoprecipitation sequencing (ChIP-seq) and luciferase assays demonstrated that EGR4 directly bound to the GDF15 promoter, driving its transcriptional activation. Functional studies showed that EGR4 promoted migration and metastasis via GDF15-mediated ErbB3/ErbB1 hetero-dimerization, which activated PI3K/AKT and MAPK/ERK pathways. Furthermore, CellChat analysis identified robust interactions between EGR4+ GC cells and cancer-associated fibroblasts (CAFs), particularly extracellular matrix (ECM)-remodeling eCAFs. Secreted GDF15 induced CAF activation through TGF-β receptor signaling, creating a pro-metastatic niche. Collectively, our study establishes the EGR4/GDF15 axis as a critical driver of GC metastasis, offering possible therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6e5035ca4934c776796dfb40ae16c3c3f2ef43" target='_blank'>
              EGR4 transcriptionally upregulates GDF15 to promote gastric cancer metastasis
              </a>
            </td>
          <td>
            Weiwei Liu, Yanyan Li, Lixin Liang, Lisheng Zheng, Rui Zeng, Congcong Zhang, Zhihao Lin, Wanying Feng, Qingling Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b73f806f958dd1db9fdcf8f058c71382d2407" target='_blank'>
              Antagonistic histone H2A variants and autonomous heterochromatin formation shape epigenomic patterns in Arabidopsis
              </a>
            </td>
          <td>
            Shoko Oda, Sayaka Tominaga, Shumpei Takeuchi, A. Osakabe, Akira Kawabe, Frédéric Berger, T. Kakutani, T. To
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5530646bdb0fb88ef129ade5bf2a556562e0f177" target='_blank'>
              SRF fusion oncogenes encode constitutively activated chimeric transcription factors in myoid soft tissue tumors
              </a>
            </td>
          <td>
            Constance Pirson, Ariane Sablon, A. Loriot, Pierre Autin, A. K. Braat, Marie Karanian, C. Antonescu, Franck Tirode, J. Demoulin
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>136</td>
        </tr>

        <tr id="We have investigated changes in chromosome conformation, nuclear organization, and transcription during differentiation and maturation of control and mutant motor neurons harboring hexanucleotide expansions in the C9orf72 gene that cause amyotrophic lateral sclerosis (ALS). Using an in vitro reprogramming, differentiation and neural maturation protocol, we obtained highly purified populations of post-mitotic motor neurons for both normal and diseased cells. As expected, as fibroblasts are reprogrammed into iPSCs, and as iPSCs differentiate into motor neurons, chromatin accessibility, chromosome conformation, and nuclear organization change along with large-scale alterations in transcriptional profiles. We find that the transcriptome changes extensively during the first three weeks of post-mitotic neuronal maturation, with thousands of genes changing expression, but then is relatively stable for the next three weeks. In contrast, chromosome conformation and nuclear organization continue to change over the entire 6-week maturation period: chromosome territoriality increases, long-range interactions along chromosomes decrease, compartmentalization strength increases, and centromeres and telomeres increasingly cluster. In motor neurons derived from ALS patients such changes in chromosome conformation were much reduced. Chromatin accessibility changes also showed delayed maturation. The transcriptome in these cells matured relatively normally but with notable changes in expression of genes involved in lipid, sterol and mitochondrial function. We conclude that neural maturation is associated with large scale post-mitotic changes in gene expression, chromosome conformation and nuclear organization, and that these processes are defective in motor neurons derived from ALS patients carrying C9orf72 hexanucleotide repeat expansions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ebb664b6f1b21244e0cfd38a3721d45d012b3f" target='_blank'>
              Dynamic changes in chromosome and nuclear architecture during maturation of normal and ALS C9orf72 motor neurons
              </a>
            </td>
          <td>
            Özgün Uyan, Snehal Sambare, Marlies E. Oomen, Nicholas Wightman, A. Schooley, Joseph R. Klim, Houda Belaghzal, Özkan Aydemir, Betul Akgol-Oksuz, Zeynep Sena Agim Uslu, Kevin Eggan, Robert H. Brown, Job Dekker
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Waleed K. Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Trypanosomatid parasites and the diseases they cause affect more than 30 million people annually worldwide. To develop treatments for these diseases, it is critical to understand how trypanosomatid biology protects the parasite, so that these mechanisms may be exploited as drug targets. An important aspect of trypanosomatid survival is protection from oxidative damage inflicted by the host. Reactive oxygen species produced by the host can damage nuclear DNA and kinetoplast, the mitochondrion DNA. DNA damage must be repaired or bypassed for the trypanosomatid to continue to replicate its genome. Trypanosomatids also possess specialized redox pathways that neutralize reactive oxygen species (ROS) from host-derived attacks and endogenous mitochondrion metabolism. This Review Article focuses on how trypanosomatids employ microhomology-mediated end-joining to repair DNA double-strand breaks and translesion DNA synthesis to bypass oxidatively damaged bases in nuclear and kinetoplast DNA. While the deleterious effects of ROS must be managed to protect the parasite’s genome, the redox status generated by oxidative assault is crucial for intracellular signaling, DNA synthesis, and kinetoplast homeostasis. This Review will also comment on the effectiveness of current treatments for trypanosomatid-caused diseases and the role of oxidative damage in trypanosomatid diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/280cba51b176c0cec98145ce4753c330f4d4fd5a" target='_blank'>
              Translesion synthesis and microhomology-mediated end-joining repair in trypanosomatids
              </a>
            </td>
          <td>
            Lea Drogalis Beckham, Anne K. Snyder, Sylvie Doublié, Bruno Martorelli di Genova
          </td>
          <td>2025-10-01</td>
          <td>PLOS Neglected Tropical Diseases</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a7a507c9272e7ca6986d3ffe683c348245fff39" target='_blank'>
              Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner
              </a>
            </td>
          <td>
            Paige V. Blinkiewicz, Nicole C. Hoenes, Myra X. Afzal, Yihong Liu, Ulysses J. Hill, B. Bjarnadóttir, T. E. Burgin, P. Malaney
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a2eaeac8d39964b8ce29ff957548c6f095cc6b" target='_blank'>
              Beckwith-Wiedemann syndrome multiomic analysis of hepatoblastoma uncovers unique tumor heterogeneity and cellular landscapes, including transition cells leading to tumor formation
              </a>
            </td>
          <td>
            Snehal Nirgude, Elisia D Tichy, Yuanchao Zhang, Rose D Pradieu, Michael Xie, Kathrin M. Bernt, Suzanne Macfarland, J. Kalish
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Loss of DNA mismatch repair (MMR) increases mutagenesis and tumorigenesis. mutS homolog 2 (MSH2), a central component of the MMR pathway, is essential for correcting base–base mismatches and insertion/deletion loops during DNA replication. To investigate how Msh2 deficiency cooperates with oxidative stress to drive mutagenesis and tumorigenesis, we employed an rpsL reporter gene assay using normal tissues before tumor development following treatment with an oxidizing agent. The background mutation frequency in the small intestines of Msh2-/- mice was over 20-fold higher than that of wild-type mice. In addition to G > A base substitutions, frequent 1-bp deletions in adenine mononucleotide repeats ((A)n) in the rpsL gene were observed. Potassium bromate treatment further increased the mutation frequency, particularly insertion-deletion mutations (indel), in the normal small intestinal epithelium of Msh2-/- mice before tumor development. Mutation signature analysis from next-generation sequencing data revealed that signatures associated with MMR deficiency (SBS15, SBS44, and ID2) and clock-like processes (SBS1 and SBS5) were consistently detected across all Msh2-/- tumors, similar to those observed in human MMR-deficient cancers. ID2, which involves 1-base deletions occurring in (A/T)n tracts of six bases or longer, supports the findings of the rpsL assay. Microsatellite instability (MSI) analysis showed that indel mutations at (A)n loci detected using the rpsL assay reflect genome-wide MSI. Msh2-/- tumors frequently harbored driver mutations, such as frameshift mutations in short tandem repeats within Apc and G > A substitutions in Ctnnb1, both of which activate the Wnt signaling pathway. Oxidative stress further accelerated these mutational processes. Oxidative stress promotes repeat-associated mutagenesis, which manifests as MSI and base substitutions in MMR-deficient intestinal tissues, thereby enhancing the mutator phenotype and increasing the overall mutation burden. This process can be sensitively captured using our rpsL assay, which serves as a functional indicator of MMR deficiency and replication instability in normal tissues before tumor formation. This increases the likelihood of driver mutations in oncogenes and tumor suppressor genes, ultimately accelerating early tumorigenesis. This study demonstrated that MSH2 is essential for maintaining genome stability under oxidative conditions and functions as a key suppressor of oxidative stress–induced tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c307ffc3ea1fca1fb4bfcae167ad12e85c5a8ce" target='_blank'>
              Oxidative stress accelerates repeat sequence instability and base substitutions promoting gastrointestinal driver mutations in MSH2 deficient mice
              </a>
            </td>
          <td>
            M. Ohno, Noriko Takano, Kyoko Hidaka, Fumiko Sasaki, Yasunobu Aoki, Takehiko Nohmi, Teruhisa Tsuzuki
          </td>
          <td>2025-10-23</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554ff6de17aca3fc14055b3c5a6203467aaf1fe3" target='_blank'>
              A gap-free telomere-to-telomere pig reference genome provides insight in centromere evolution and breed-specific selection
              </a>
            </td>
          <td>
            Dong Li, Yulong Wang, Tiantian Yuan, Xiang Li, Yu Quan, Wansheng Liu, K. Schachtschneider, Lawrence B. Schook, R. Crooijmans, M. Groenen, M. Derks, Taiyong Yu
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0598f4e042a38ac90e8698c02adae86ab3960654" target='_blank'>
              KCTD10 is a sensor for co-directional transcription-replication conflicts.
              </a>
            </td>
          <td>
            Jake A. Kloeber, Bin Chen, Guangchao Sun, Charles S King, Zhiquan Wang, Li Wang, Zheming Wu, Shouhai Zhu, Fei Zhao, Hongran Qin, Yaobin Ouyang, Huaping Xiao, X. Tu, Jing Lu, Yanxia Jiang, K. Luo, Ping Yin, Xinyan Wu, R. Mutter, Jinzhou Huang, Zhenkun Lou
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b04efe93fb72cdef3529bf2ec79fd71db767d8a" target='_blank'>
              DNASE1L3 surveils mitochondrial DNA on the surface of distinct mammalian cells
              </a>
            </td>
          <td>
            Jennifer Porat, Valentina Poli, Karim Almahayni, Benson M. George, F. N. de Luna Vitorino, Li Yi, Carolina Bras Costa, Yitong Zhou, Andrew Crompton, Hongxiu Wen, Leon Zheng, Peter K. Gregersen, Suneet Agarwal, Benjamin A. Garcia, Yixuan Xie, Leonhard Möckl, Ivan Zanoni, K. Simpfendorfer, Ryan A. Flynn
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Reverse transcriptases (RTs) have well-established roles in the replication and spread of retroviruses and retrotransposons. However, recent evidence suggests that RTs have been conscripted by cells for diverse roles in antiviral defence. Here we determine structures of a type I-A retron, which explain how RNA, DNA, RT, HNH-nuclease and four molecules of a structure maintenance of chromosome (SMC)-family ATPase assemble into a 364 kDa complex that provides phage defence. We show that phage-encoded nucleases trigger degradation of the retron-associated DNA, leading to activation of the HNH nuclease. The HNH nuclease cleaves tRNASer, stalling protein synthesis and arresting viral replication. Taken together, these data reveal diverse and paradoxical roles for RTs in the perpetuation and elimination of genetic parasites. In this study, the authors determine the structure of a Type I-A retron from E. coli FORC82 and reveal the functional interplay between Reverse Transcriptases (RTs) and Structural Maintenance of Chromosomes (SMC) ATPases.’">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78259f1a0c5b394b643034f4b28e1301ed4a03ce" target='_blank'>
              Structural basis of antiphage defence by an ATPase-associated reverse transcriptase
              </a>
            </td>
          <td>
            Jerrin Thomas George, Nathaniel Burman, R. Wilkinson, Senuri de Silva, Quynh McKelvey-Pham, M. Buyukyoruk, Adelaide Dale, Hannah Landman, A. Graham, Steven Z DeLuca, B. Wiedenheft
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b15ed383fa686f49d7de283de5e0eaf7d1cc0c" target='_blank'>
              A Rare Multipotent Peg-like Epithelial Cell is a Candidate Cell-of-Origin for High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Megan L. Ritting, Wenmei Yang, Syed Mohamed Musheer Aalam, Hui Zhao, Liang Feng, Jianning Song, Mihai G. Dumbrava, W. Ismail, Dong-Gi Mun, Chunling Hu, Opalina Roy, Keyur Chaludiya, Geng-Xian Shi, Daphne Norma Crasta, Kenneth Schaufelberger, Jeffrey R. Janus, Guruprasad Kalthur, S. Weroha, Scott H. Kaufmann, A. Sadanandam, David J. H. F. Knapp, Chen Wang, Akhilesh Pandey, Alexandre Gaspar-Maia, F. Couch, Mark E. Sherman, J. Bakkum-Gamez, Nagarajan Kannan
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In eukaryotic cells, histone acetylation and chromatin states affect gene expression programs in broad spectrum of cellular function. Comprehending the genomic signature of retinoid X receptor (RXR) will provide crucial insight into the molecular signaling pathways underlying stem cell fate transition. Here, we employ an integrative ChIP-seq approach to investigate the interplay of RXR with MyoD and CTCF in proliferating myoblasts. Our findings suggest an active enhancer distribution of RXR along with MyoD, marked by H3K18 and H3K27 acetylation, potentially mediated by histone acetyltransferase p300. On the other hand, RXR together with CTCF were mainly distributed to active promoters, marked by H3K9 acetylation. CTCF in proximity to RXR was also associated to nuclear receptor regulator NCOA1 and N-COR, suggesting a potential role in chromatin organization. Taken together, our analyses indicate a crucial interplay of MyoD and CTCF with RXR in the regulation of stem cell function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b370781731a41dce45eb74590430590f7a9c7a7" target='_blank'>
              Cistromic insight into the association of retinoid X receptor with MyoD and CTCF in proliferating myoblasts
              </a>
            </td>
          <td>
            Jean-Thomas Clément, Qiao Li
          </td>
          <td>2025-11-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/167077c6c722e64c238ff049d56bccfcac548091" target='_blank'>
              Acytota and the evolution of complexity.
              </a>
            </td>
          <td>
            E. Kejnovský, Z. Kubat, J. Šponer
          </td>
          <td>2025-10-27</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epigenetic regulation, encompassing DNA methylation, histone modifications, and non-coding RNA activities, is a crucial mechanism through which gene expression is modified without corresponding changes in genomic DNA sequences. Dysregulation of these mechanisms can lead to histone and DNA modifications that either suppress or enhance the expression of disease progression-related genes. Among these regulatory processes, histone modifications are particularly significant, as they contribute to genomic stability, DNA repair, and chromatin dynamics, all of which influence breast cancer initiation and progression. This review offers a detailed analysis of the current state of research centered on epigenetic regulatory factors, with a particular focus on the role that histone modifications play in the treatment of breast cancer. It also examines the interplay between epigenetic modifications and the effectiveness of radiotherapy and chemotherapy when treating breast cancer. Lastly, this article explores the potential of epigenetic regulatory factors as viable targets for the future design of new anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379da6773863c74043489d2ccffa9dd0b5e23c2c" target='_blank'>
              Epigenetic regulation espeically histone modifications in breast cancer: A viable and emerging targeted therapeutic strategy
              </a>
            </td>
          <td>
            Yibing Zhou, Haotian Liu, Weimin Hong, Haotian Su, Yuxiao Mu, Yijie Cheng, Chaoshen Wu, Xuli Meng, Da Qian
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44901dfbd5176301295029fab9bb4b40320a082" target='_blank'>
              AAV Kills Dividing Cells by Depleting PARP1 and Other DNA Damage Response Proteins
              </a>
            </td>
          <td>
            Sasha Friese, Junjie Zai, Grace Luzbetak, Nyonika Khanna, Johanna Gesperger, Chien-Hung Liu, Wynand P. Roos, Rasha Q. Al-Rahahleh, Matisse Willardson, Feng Yang, Duoduo Fu, Yiming Han, Nadia Lintag, Jaiden Saykham, Joshua Le, Emeen Al-Delaimy, Nolan Soutipan, Ellen Duong, Jeremy Rich, Maria Carolina Marchetto, Michael Rosenfeld, Robert W. Sobol, Matthew Shtrahman
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1bc2b0013bf3f73eea36d3f8f1f1a3ceddf4e" target='_blank'>
              The evolutionary origin of host association and polycistronic transcription in trypanosomatids
              </a>
            </td>
          <td>
            Saurav Mallik, Meir Sylman, Dvir Dahary, Orna Dahan, S. Michaeli, G. Späth, Y. Pilpel
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce78a1432bf55161afa66b6f861a74035f16861" target='_blank'>
              Low Dose of Lead Causes Activation of Ribosomal Cistrons 15 Chromatids Associated with Malignant Growth
              </a>
            </td>
          <td>
            T. Lezhava, T. Buadze, T. Jokhadze, M. Gaiozishvili, T. Sigua, Khatia Nadiradze, Shorena Sharia
          </td>
          <td>2025-09-21</td>
          <td>Cytology and Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba22ede5cb07736eac6d115a03c89fcfe2e86a6e" target='_blank'>
              Evolutionary Variation of Poxvirus Genome Architecture
              </a>
            </td>
          <td>
            J. Enow, Arkaidy Garber, Betram Jacobs, G. McFadden, Masmudur M. Rahman
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3161e03102a0603b20259b47a78fa5baea5644e5" target='_blank'>
              Decoder-FFPE-seq enables sensitive, genome-wide spatial transcriptomics of archival tissues at single-cell resolution
              </a>
            </td>
          <td>
            Jiao Cao, Jun Zeng, Yiwei Meng, Shanqing Huang, Jing Zhang, Xing Fang, Yang Guo, Weixiong Shi, Yutong Liu, Zhong Zheng, Jianchen Fang, Xuepeng Xiong, Gang Chen, Lingling Wu, Chaoyong Yang
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c809269d9254558fb4308ec6c19a38c6246f527" target='_blank'>
              STING activation reshapes the tumor microenvironment leading to tumor regression in osteosarcoma
              </a>
            </td>
          <td>
            Elizabeth P Young, Christine Johnson, Alex G Lee, Douglass Morris, Jessica Tsui, Marcus R. Breese, Juan Antonio, Camara Serrano, Courtney R Schott, Clare L. Abram, Stanley G. Leung, L. Sayles, Aafrin M. Pettiwala, Vanessa S Gutierrez Vera, Clifford A. Lowell, Alexis J. Combes, T. A. McEachron, A. Sweet-Cordero
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c93dca3eb5be61d1472406ead880481fa79563f" target='_blank'>
              Genetic diversity and evolution of rice centromeres.
              </a>
            </td>
          <td>
            Lingjuan Xie, Yujie Huang, Wei Huang, L. Shang, Yanqing Sun, Quanyu Chen, Shuangtian Bi, Mingyu Suo, Shiyu Zhang, Chentao Yang, Xiaoming Zheng, Weiwei Jin, Qian Qian, Longjiang Fan, Dongya Wu
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b8134c8c64f4cddd987308aa2a1f8568ae01a4c" target='_blank'>
              TRIP13 fosters both transcriptional silencing and DSB repair during meiosis
              </a>
            </td>
          <td>
            Marina Marcet-Ortega, A. Martínez-Marchal, Andros Maldonado-Linares, Mercè Encinas, Anna Esteve Codina, M. Jasin, S. Keeney, I. Roig
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6d95c002d856c172eff01b267c3c0451ed62ab" target='_blank'>
              Stable hypermutators revealed by the genomic landscape of genes involved in genome stability among yeast species.
              </a>
            </td>
          <td>
            Carla Gonçalves, J. Steenwyk, D. Rinker, D. A. Opulente, A. Labella, Marie-Claire Harrison, John F. Wolters, Xiaofan Zhou, Xingxing Shen, S. Covo, M. Groenewald, C. T. Hittinger, A. Rokas
          </td>
          <td>2025-11-03</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="DNA methylation and histone modifications are key epigenetic marks controlling chromatin structure, gene expression and transposable element (TE) activity. In plants, the histone demethylase INCREASE IN BONSAI METHYLATION1 (IBM1) prevents heterochromatic silencing marks from accumulating on actively transcribed genes. Through a genetic screen of mutants defective in IBM1 production, we identified suppressor mutations in genes essential for maintaining balanced genome-wide epigenetic states. The gyrb3 mutation partly reversed DNA hypermethylation in IBM1-deficient plants, revealing a novel role for GyrB3, a nuclear protein combining domains from cyanobacterial gyrases and ELM2/SANT proteins involved in histone acetylation. In gyrb3 mutants, TEs exhibit transcriptional activity, showing reduced DNA methylation and increased histone H3 acetylation, both of which are epigenetic marks associated with expression activation. GyrB3 physically interacts with histone deacetylases like HISTONE DEACETYLASE6 (HDA6), likely mediating their activities at TEs. The functional overlap between HDA6 and GyrB3 is further supported by the observation that, similar to gyrb3, a mutation in hda6 suppresses the Ibm2 phenotype. Our findings reinforce that histone deacetylation is essential for TE silencing and that loss of IBM1 in plants abolished the frontiers between genes and TEs, emphasizing its importance in maintaining epigenomic stability. Key Points IBM1 is a histone demethylase that safeguards actively transcribed genes from inappropriate heterochromatic silencing, thereby preserving their function and epigenomic integrity. A gyrb3 mutant suppresses the developmental abnormalities observed in ibm1 or ibm2 mutants. GyrB3, though misannotated as a plant gyrase, appears to have evolved away from its conventional role and instead functions in epigenetic regulation. GyrB3 is a nuclear protein that interacts with histone deacetylases such as HDA6 to repress transposon activity; mutating GYRB3 results in elevated transposon expression, driven by reduced DNA methylation and increased histone acetylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6d673549c03f874db25b197e723534908d09fd2" target='_blank'>
              The arabidopsis GyraseB3 contributes to transposon silencing by promoting histone deacetylation
              </a>
            </td>
          <td>
            Isabelle Gy, Sébastien Beaubiat, Nicolas Bouché
          </td>
          <td>2025-08-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Recent studies reveal that 5-18% of children with cancer harbor pathogenic variants in known cancer predisposing genes. However, DNA damage repair (DDR) genes, which are frequently somatically altered in pediatric tumors, have not been systematically examined as a source of novel cancer predisposing signals.


METHODS
To address this gap, we interrogated 189 DDR genes for presence of germline predisposing variants (PV) among 5,993 childhood cancer cases and 14,477 adult non-cancer controls (discovery cohort). PV were determined using a tiered approach incorporating ClinVar annotations, InterVar classification, and in silico tools (REVEL, CADD, MetaSVM). Using logistic and firth regression, we identified genes with PV statistically enriched in tumors and replicated findings among 1,497additional childhood cancer cases across three independent cohorts.


FINDINGS
Analysis across all cancers revealed enrichment of TP53 PV. Cancer-specific analyses confirmed known associations including germline TP53 PV in adrenocortical carcinoma, high-grade glioma (HGG), and medulloblastoma (MB), PMS2 in HGG and non-Hodgkin lymphoma (NHL), MLH1 in HGG, BRCA2 in NHL, and BARD1 in neuroblastoma. In addition, four novel associations were uncovered, including BRCA1 in ependymoma, SPIDR in HGG, SMC5 in MB, and SMARCAL1 in osteosarcoma (OS). Importantly, the SMARCAL1:OS association was significant in the discovery (6/230, 2.6%, FDRlogistic=0.0189) as well as all three replication cohorts (Childhood Cancer Survivor Study: 8/275, 2.9%; PFisher<0.0001; German Childhood Cancer Registry: 4/135, 3%, PFisher=0.002; INdividualized Therapy FOr Relapsed Malignancies in Childhood: 4/217, 1.8%, PFisher=0.012). The remaining wildtype SMARCAL1 allele was deleted in three of four OS tumors with available data.


INTERPRETATION
Our study confirms the relevance DDR genetic variation in pediatric cancer risk and establishes SMARCAL1 as a novel OS predisposing gene, providing insights into tumor biology and creating opportunities to optimize care for patients with this challenging tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c07cd6375ece0142b345b31eccaeaa135c76e101" target='_blank'>
              Investigation of DNA damage response genes validates the role of DNA repair in pediatric cancer risk and identifies SMARCAL1 as novel osteosarcoma predisposition gene.
              </a>
            </td>
          <td>
            Ninad Oak, Wenan Chen, Alise K Blake, Lynn Harrison, Martha O'Brien, C. Previti, Gnanaprakash Balasubramanian, K. Maass, S. Hirsch, Judith Penkert, Barbara C Jones, Kathrin Schramm, Michaela Nathrath, K. Pajtler, David T W Jones, Olaf Witt, U. Dirksen, Jiaming Li, Y. Sapkota, Kirsten K. Ness, Lillian M Guenther, Stefan M. Pfister, Christian P. Kratz, Zhaoming Wang, Greg T Armstrong, M. Hudson, Gang Wu, R. Autry, Kim E. Nichols, Richa Sharma
          </td>
          <td>2025-10-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b865648502532b64b511597f51b1acb1d7cb69bb" target='_blank'>
              Disrupting the epigenetic alliance: structural insights and therapeutic strategies targeting DNMT1–UHRF1
              </a>
            </td>
          <td>
            Emadeldin M. Kamel, Mohamed A M Ali, A. A. Allam, Noha A. Ahmed, Adil Abalkhail, Faris F. Aba Alkhayl, Al Mokhtar Lamsabhi
          </td>
          <td>2025-09-17</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Significance Gastric adenocarcinoma and proximal polyposis of the stomach (GAPPS) is an autosomal dominant syndrome characterized by polyposis localized in the gastric body and fundus with a strong tendency for adenocarcinoma. In this study, we demonstrated that somatic mutations of APC occur in carcinoma and polyp, while mutations of KRAS additionally occur in carcinoma. Furthermore, we revealed that APC and KRAS mutations recurrently co-occur in carcinoma both between cases and within subclones of the same case. KRAS mutations could serve as a biomarker for carcinogenesis in GAPPS, potentially contributing to the early diagnosis of carcinoma and determining the appropriate timing for resection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8b03bc09f85ddf947c28f95d505727c55dedcf" target='_blank'>
              Genomic and transcriptomic landscape of carcinogenesis in patients with gastric adenocarcinoma and proximal polyposis of the stomach (GAPPS)
              </a>
            </td>
          <td>
            C. Matsumoto, Kazuki K. Takahashi, Masaaki Iwatsuki, Noriko Yasuda-Yoshihara, A. Niida, Kohei Yamashita, Takeshi Morinaga, K. Eto, S. Iwagami, Satoshi Ida, H. Yano, Yoshihiro Komohara, Yuji Miyamoto, Takaaki Masuda, Yasuhito Tanaka, Koshi Mimori, Hideo Baba
          </td>
          <td>2025-10-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genomic amplification may result in aberrant gene expression and support development of cancer, including chronic myeloid leukemia (CML). In CML cell line K-562, we recently reported overexpression of TBX1 located at chromosomal position 22q11, focally co-amplified together with BCR, part of the CML hallmark fusion gene BCR::ABL1. Here, we extended that study, by identifying genomically amplified and overexpressed MAPK1/ERK2 at 22q11 together with MCTS1 at Xq22. Using pharmacological inhibitors and siRNA-mediated knockdown assays, our data collectively revealed novel regulatory connections between TBX1, MAPK1 and MCTS1, which may play a role in drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cfe5a18303d3c15ad4fb64c39efcc6da10d99d" target='_blank'>
              Aberrant expression of MAPK1 and MCTS1 in chronic myeloid leukemia (CML)
              </a>
            </td>
          <td>
            Leo Kortendick, C. Meyer, S. Nagel
          </td>
          <td>2025-09-18</td>
          <td>microPublication Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="ABSTRACT Guanine quadruplex (G4) structures are secondary structures formed in nucleic acids containing guanine-rich stretches. The regulatory role of G4s in various cellular processes has been studied in both eukaryotes and prokaryotes. Deinococcus radiodurans, an extremophile, shows increased sensitivity to ionizing radiation in the presence of G4-binding ligands during the post-irradiation recovery (PIR) period. Here, for the first time, a detailed study is reported on the intracellular formation and resolution of G4 structures in this organism. The potential G-quadruplex-forming sequences, positioned at different locations within the genes, folded into different topologies in vitro. The divalent cation Mn²+, which plays a crucial role in radioresistance, destabilized G4 structures, whereas Mg²+ supported the G4 formation. Depending on their location at the 5′ or 3′ end of the gene, the native G4 motifs differentially regulated the endogenous gene expression under both normal and PIR conditions. Thioflavin T and anti-G-quadruplex antibodies detected the in vivo formation and dynamics of G4s in response to various DNA-damaging agents, with the highest levels observed following gamma radiation treatment. The presence of G4-binding drugs during the PIR arrested G4 structure dynamics, increased the cellular levels of 8-oxo guanine, and delayed the DNA repair process. Furthermore, the absence of RecQ resulted in the accumulation of more G4 structures within the cells, leading to genome instability. These results indicate that in D. radiodurans, the in vivo formation of G4 structures varies significantly under normal and cellular stress conditions, depending on the levels of Mn²+ and G4 resolving proteins. IMPORTANCE Guanine quadruplexes (G4s) can act as both activators and inhibitors of various cellular functions, depending on their genomic position. While G4 functions are increasingly understood in eukaryotes, their specific roles in bacteria, especially in vivo, are still largely unexplored. Deinococcus radiodurans harbors a GC-rich genome with numerous potential G-quadruplex-forming sequences. In this study, the first direct evidence of G4 formation in vivo and its dynamic nature during the DNA damage response to gamma radiation is shown in a bacterium. The significance of G4 structural dynamics in modulating endogenous gene expression, DNA repair, and genome maintenance processes is demonstrated. These findings provide a novel insight into G4-mediated regulatory mechanisms in prokaryotes and expand our understanding of G4’s role as a dynamic structural switch. Guanine quadruplexes (G4s) can act as both activators and inhibitors of various cellular functions, depending on their genomic position. While G4 functions are increasingly understood in eukaryotes, their specific roles in bacteria, especially in vivo, are still largely unexplored. Deinococcus radiodurans harbors a GC-rich genome with numerous potential G-quadruplex-forming sequences. In this study, the first direct evidence of G4 formation in vivo and its dynamic nature during the DNA damage response to gamma radiation is shown in a bacterium. The significance of G4 structural dynamics in modulating endogenous gene expression, DNA repair, and genome maintenance processes is demonstrated. These findings provide a novel insight into G4-mediated regulatory mechanisms in prokaryotes and expand our understanding of G4’s role as a dynamic structural switch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a18e321742198ffe63625b20778b7010d50310" target='_blank'>
              Structural switches: intracellular G-quadruplexes modulate DNA repair and genome maintenance in Deinococcus radiodurans
              </a>
            </td>
          <td>
            Himani Tewari, Shruti Mishra, Swathi Kota
          </td>
          <td>2025-09-29</td>
          <td>mBio</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a660def78ace40512480c658f652835e9dad99a3" target='_blank'>
              The long non-coding RNA, CyKILRb, augments oncogenic phenotypes via induction of PIK3R2 and activation of the PI3K/AKT axis
              </a>
            </td>
          <td>
            Xiujie Xie, H. Macknight, Amy L. Lu, C. E. Chalfant
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Profilin 1 (Pfn1) has been implicated in cytoskeletal regulation; however, its role in breast cancer progression and DNA replication remains unclear. This study investigated the functional significance of Pfn1 nuclear-cytoplasmic shuttling in breast cancer.


METHODS
We analyzed Pfn1 expression and its correlation with DNA replication, repair, and oncogenic markers in breast cancer cell lines. Chromatin-bound and soluble Pfn1 levels were quantified by western blotting. The effects of nuclear (nuclear localization sequence-Pfn1) and cytoplasmic (nuclear export sequence-Pfn1) localization on cell growth, DNA replication, and stemness were assessed using colony formation, Alamar blue fluorescence, replication protein A 32-kDa foci staining, and DNA fiber assays. Mouse xenografts of breast cancer cells were used to determine the effect of Pfn1 localization on tumor growth in vivo. We identified the direct interactors of nuclear Pfn1 by immunoprecipitation, and their affinity was determined using bio-layer interferometry.


RESULTS
Pfn1 expression was positively correlated with DNA replication, repair, p53, and MYC expression. Chromatin-bound Pfn1 was significantly degraded in breast cancer cell lines compared to that in non-cancerous MCF10a cells. Nuclear Pfn1 inhibited cell growth and DNA replication in SKBR3 cells, while cytoplasmic Pfn1 promoted cell survival and DNA replication in MCF10a cells. Loss of nuclear Pfn1 in SKBR3 cells inhibited their growth in vivo. Additionally, cytoplasmic Pfn1 upregulated stemness markers (c-Myc, B lymphoma Mo-MLV insertion region 1, and Nijmegen breakage syndrome 1). Pfn1 regulated cell stemness by binding to the nucleosome remodeler sucrose non-fermenting 2 homolog.


CONCLUSION
Our findings revealed that nuclear Pfn1 acts as a tumor suppressor by inhibiting DNA replication and cell growth, while cytoplasmic Pfn1 promotes tumorigenesis by enhancing stemness and replication efficiency. These results highlight the dual role of Pfn1 in breast cancer progression, governed by its subcellular localization. They suggested that modulating Pfn1 nuclear-cytoplasmic shuttling may be a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1361eb9668d89561237c423978b3412e7cc545f" target='_blank'>
              Loss of Nuclear Profilin 1 Triggers Oncogenic Reprogramming of Mammary Epithelial Cells Through Dysregulated DNA Replication in Breast Cancer.
              </a>
            </td>
          <td>
            Zheng Zhao, Yingbin Huang, Junhao Mai, Fei Cao, Q. Fang, Di Wu, Ziqian Li, Xuekui Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal of breast cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff9dd6e8b4e7b70e0de9451cb1c5dd37501bb1c" target='_blank'>
              A comprehensive genomic framework for identifying genes predisposing to homologous recombination repair deficient breast cancer
              </a>
            </td>
          <td>
            José Camacho-Valenzuela, Thibaut Matis, Carla Roca, Jorge Luis, Cuamatzi Flores, N. Hamel, Barbara Rivera, Simon Gravel, P. Polak, Carla Daniela, Robles-Espinoza, William D. Foulkes
          </td>
          <td>2025-10-23</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Background Unlike other cancer types the determinants of anti-PD-1 response in clear cell RCC are not well defined; ccRCC exhibits a low mutational burden and CD8+ T cells infiltration is associated with worse prognosis. Cytomegalovirus (CMV) and human papillomavirus (HPV) infections are postulated to modulate the anti-tumor immune responses within the tumor microenvironment. We sought to investigate the clinical and immunologic impact of CMV and HPV viral transcriptomic signatures, as defined by prior studies, in patients with RCC. Methods RNA-seq data from three RCC prospetive trials were upper-quartile normalized, log2-transformed (TPM + 1), and batch-corrected using ComBat. Transcriptomic scores for CMV (75 genes) and HPV (79 genes) were computed using geometric means (71 and 77 genes were found, respectively). CD8 immunofluorescence (IF) was performed on tumor samples. Overall survival (OS) and progression-free survival (PFS) were assessed using multivariable Cox regression, adjusting for age at therapy start, gender, IMDC risk, sarcomatoid/rhabdoid features, and number of previous lines received. Results A high intratumoral CMV transcriptomic score was significantly correlated with T-effector signature (ρ  =  0.54, p = 4.8 × 10−6), IF tumor core CD8+ (ρ  =  0.43, P < .001), IF tumor margin CD8+ (ρ  =  0.41, P < .001), and myeloid signature (ρ  =  0.40, P < .001). The CMV score was significantly higher in infiltrated versus excluded/desert tumors (P = .0079). The CMV score was associated with worse OS among patients receiving immune checkpoint blockade (ICB) (HR 4.98, P = .039). In contrast, the HPV transcriptomic score showed only a modest correlation with myeloid signature (ρ  =  0.24, P = .051) and no association was found with OS nor PFS. Conclusions In metastatic RCC, a high tumor CMV transcriptomic score was associated with a paradoxical immune landscape, marked by CD8+ infiltration yet co-enriched with suppressive myeloid signatures, that predicts inferior overall survival following ICB. These viral signatures within the complex immune architecture of RCC may help understand immune responsiveness and resistance within distinct genomic and therapeutic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c3c656ee910ac2c2d9287a96ba1a9501c97839" target='_blank'>
              39Intratumoral viral transcriptomic signatures stratify immune phenotypes and clinical outcomes in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            Mustafa Saleh, E. Saad, Pablo Barrios, M. Machaalani, Liliana Ascione, W. Daoud Khatoun, Jad El Masri, Razane ElHajj Chehade, Marc Eid, Rashad Nawfal, Karl Semaan, E. Yekeduz, Clara Steiner, Weiwei Bian, Maxine Sun, S. Signoretti, E. V. Van Allen, D. Braun, Alexander Gusev, Toni Choueri
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Abstract UVB radiation (280–320 nm) from sunlight induces skin cancer through DNA damage-induced mutations. Melanomas carry mutational signatures associated with UVB-induced cyclobutane pyrimidine dimers (CPDs). However, there are several other melanoma signatures of unknown origin. To test if these signatures are linked to UVA, we exposed human melanocytes to UVA (340–400 nm) and to UVB for comparison. We mapped DNA damage in the form of CPDs or 8-oxoguanines (8-oxoG) genome-wide at base resolution. We then determined mutational patterns in single melanocyte cell clones by whole genome sequencing. Different from UVB, UVA induces CPDs more selectively at TT sequences resembling melanoma signature SBS7d. We did not observe rising CPD levels after cessation of radiation (dark CPDs). The UVA-induced CPDs were not mutagenic in the mutation analysis. 8-oxoG was present in melanocytes but did not substantially increase after UVA. G/C to T/A mutations were prominent in melanocyte single-cell clones with no major shift after UVA radiation. These mutations matched SBS18, a signature present in melanomas. Although UVA damages DNA, it has a surprisingly limited mutagenic effect on human melanocytes. However, the oxidative base lesions in melanocytes and their associated mutations may be linked to a subset of melanoma mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b21e19c0b878b06400096dc62c3652b28e09c65" target='_blank'>
              UVA-induced DNA damage and mutations in human melanocytes: relevance for melanoma mutations
              </a>
            </td>
          <td>
            , Jennifer Johnson, Paul Y Kim, Gerd P Pfeifer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The rate of DNA synthesis is crucial for full DNA duplication. We report a key role of p21 in controlling this rate. During normal replication, p21 promotes nascent DNA synthesis alongside the DNA polymerase iota (Pol ι)/p53 complex. When p21 is down-regulated but detectable, nascent DNA tracks are longer and discontinuous and rely on primase and DNA polymerase (PrimPol). With the complete elimination of p21, nascent DNA tracks become shorter and continuous and depend on Pol kappa (κ). Endogenous p21 levels are critical for genomic stability, as both PrimPol- and Pol κ–mediated syntheses can induce chromosomal instability. The residual expression of p21 in p53-null cells influences the involvement of PrimPol or Pol κ in nascent DNA synthesis and subsequent chromosomal instability. Our results demonstrate that endogenous levels of p21 in cycling cells, insufficient for cyclin-dependent kinase inhibition, prevent genomic instability through proliferating cell nuclear antigen binding (PCNA), limiting PrimPol and Pol κ’s role in nascent DNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31f81e290a07076ab9200968e4922ab77b5675a" target='_blank'>
              Endogenous p21 levels protect genomic stability by suppressing both excess and restrained nascent DNA syntheses
              </a>
            </td>
          <td>
            N. L. Calzetta, S. Mansilla, Candelaria Mares Ahlers, Agostina P. Bertolin, Lilen I. Caimi, Sofía Venerus Arbilla, Jingkun Zeng, Lisa Wiesmüller, V. Gottifredi
          </td>
          <td>2025-10-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 African Americans (AAs) have higher incidence rates of advanced lung cancer when compared to European Americans (EAs). DNA double-strand breaks (DNA DSBs) are the major cause of genomic instability that initiates cancer. Our previous work found AA lung cancer patients have more germline DNA DSB repair deficiencies than EAs, suggesting unrelated AAs may share genomic risk factors for developing lung cancer. We hypothesize African ancestry drives disease-causing alterations in DNA DSB repair pathway genes.



 ClinVar criteria were used to profile germline pathogenic variants (PVs) across the four major DNA DSB repair pathways: homologous recombination repair (HRR), classical non-homologous end joining (c-NHEJ), alternative end-joining (alt-EJ), and single-strand annealing (SSA). The discovery cohort consisted of populations of African (n=37,545) and European (n=622,057) descent in the Genome Aggregation Database. The validation cohort included study participants in the NIH All of Us Research Program (African, n=50,080) and (European, n=125,860). In silico RNA (SpliceA1, Pangolin, and PhyloP) and protein (VEP and CADD) annotation tools were used to predict functional consequences.



 In the African ancestry populations, 20 PVs across eight HRR genes, nine PVs across six c-NHEJ genes, six PVs across three alt-EJ genes, and 14 PVs across five SSA genes we discovered. In comparison, 115 PVs across 11 HRR genes, 52 PVs across eight c-NHEJ genes, 43 PVs across six alt-EJ genes, and 113 PVs across seven SSA genes were observed in the European ancestry populations. Interestingly, a RAD51 PV was exclusively enriched in African ancestry and not European ancestry populations for the HRR pathway. By contrast, NHEJ1, EXO1 and DCLRE1C PVs were uniquely present in those with European ancestry and not in individuals with African ancestry for the alt-EJ and SSA pathways, respectively. Frameshift mutations were the most common functional protein change in both populations.



 A total of 32 germline PVs across 13 DNA DSB repair genes were observed in African ancestry populations only, suggesting genetic ancestry may drive DNA repair deficiencies that increase susceptibility to developing lung cancer.



 Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell. Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C077.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11a81bc23bab29ba6a3aa864a49c700d5fec40e" target='_blank'>
              Abstract C077: Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry
              </a>
            </td>
          <td>
            Musinu Zakari, E. Aigboeghian, Khadijah A. Mitchell
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT We have previously reported that human adenovirus E1A interacts with ARGLU1, a small disordered cellular protein. The consequences of this interaction for virus replication were unclear. E1A is the first protein produced during adenovirus infection. Via protein-protein interactions, E1A modifies the cellular environment to create optimal conditions for viral replication. To better understand the molecular mechanisms driving viral infection, we further investigated the functional consequences of the interaction between ARGLU1 and E1A. ARGLU1 interacts with E1A directly as determined by a GST-pulldown assay with recombinant proteins. Importantly, ARGLU1 was found to act as a transcriptional repressor when it was localized to viral promoters. Repression was driven by enhanced promoter-proximal RNA polymerase II pausing. Significantly, ARGLU1-induced repression of viral promoters is likely an unintended consequence of ARGLU1 binding to E1A, as a mutant of E1A (dl1102), unable to bind to ARGLU1, did not show reduced viral gene expression. Furthermore, ARGLU1 was found to colocalize with E1A in infected cell nuclei. Finally, the binding of E1A to ARGLU1 appears to reduce DNA damage repair in bleomycin-treated and adenovirus-infected cells, suggesting that E1A binding to ARGLU1 is important for DNA damage response inhibition. Overall, this study underscores the complexity of virus-host interactions and reveals a novel role for ARGLU1 during adenoviral infection. IMPORTANCE This study uncovers a novel antiviral role for ARGLU1, known for its involvement in splicing, transcriptional regulation, and DNA damage repair. Our study demonstrates that ARGLU1 functions as a viral restriction factor by reducing virus growth through the inhibition of viral gene expression and enhanced DNA damage response, leading to reduced viral growth. These findings not only highlight an important role for ARGLU1 in host antiviral mechanisms but also emphasize the utility of adenovirus as a tool to uncover critical cellular pathways. This study uncovers a novel antiviral role for ARGLU1, known for its involvement in splicing, transcriptional regulation, and DNA damage repair. Our study demonstrates that ARGLU1 functions as a viral restriction factor by reducing virus growth through the inhibition of viral gene expression and enhanced DNA damage response, leading to reduced viral growth. These findings not only highlight an important role for ARGLU1 in host antiviral mechanisms but also emphasize the utility of adenovirus as a tool to uncover critical cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e29ce875c83fc5e2a9c3669beab1e24607b11d4" target='_blank'>
              ARGLU1 is a negative regulator of the adenoviral replicative cycle
              </a>
            </td>
          <td>
            Amit Koul, Lauren Fulham, N. Akkerman, Drayson Graves, Esha Kaul, Khadija Khadija, Peter Pelka
          </td>
          <td>2025-11-04</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba892d255c691eee277b20e16deae8de79fd0b76" target='_blank'>
              BCOR mutations deregulate cell cycle and hypoxic adaptation pathways in retinoblastoma.
              </a>
            </td>
          <td>
            Michelle G. Zhang, J. Kuznetsoff, Natalie C Cetta, Sebastian Salazar, Renata L Volonterio, Stefan Kurtenbach, Z. Corrêa, Daniel Pelaez, J. W. Harbour
          </td>
          <td>2025-11-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, Shibasish Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT: Gene editing exploits endogenous DNA repair pathways to introduce precise modifications into the genome. The CRISPR system was first identified in Escherichia coli (1987) and, based on Cas protein architecture, is divided into Class I (types I, III, IV) and Class II (types II, V, VI). The widely adopted CRISPR/Cas9 system comprises a guide RNA (gRNA) and the Cas9 endonuclease, which orchestrate genome editing through a tripartite mechanism: target sequence recognition via gRNA binding to a complementary DNA site adjacent to a protospacer adjacent motif (PAM), double-stranded DNA cleavage by two nuclease domains (RuvC cleaving the non‑target strand and HNH the target strand), and repair of the resulting break by cellular pathways. This approach has been applied to in vitro fertilization-derived embryos and meiotically developing oocytes to disrupt or correct genes, offering potential for eliminating heritable diseases. However, concerns remain regarding off-target effects that introduces unwanted genetic mutations, necessitating improved specificity and ethical scrutiny.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e32b11816cdfe9dbe37c9b7f4fa9f2f1d5d3ed5" target='_blank'>
              CRISPR/CAS9-Mediated Gene Editing in Human Gametes: A Review
              </a>
            </td>
          <td>
            Esha Kumari, Neha Banu, Kathrina Marbaniang, Faridha Jane R.M. Momin, B. C. Hynniewta
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. Y. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Inherited mutations in BRCA1 are among the primary causes of hereditary breast and ovarian cancer (HBOC). Genetic testing has identified numerous pathogenic and benign mutations in BRCA1, but also thousands of variants of uncertain significance (VUS) with unclear functional consequences. Accurate cancer risk prediction for carriers of these VUS requires functional assays to assess their pathogenicity. In this study, we analyzed 16 BRCA1 VUS detected at the Hereditary Cancer Genetic Counseling Laboratory in Salamanca, Spain, with the goal of providing additional data to guide their reclassification. Since homologous recombination (HR) is the primary tumor-suppressive function of BRCA1, we employed two complementary HR repair assays to evaluate HR efficiency. The first, an already established assay, uses a HeLa-DR cell line harboring an HR reporter cassette and involves silencing endogenous BRCA1 followed by complementation with the VUS under study. The second, developed in this work, is also based in complementation assays that restore the green fluorescence protein (GFP) gene, and employs a breast cancer cell line with a similar HR reporter system (HCC1937-HR) that lacks BRCA1 expression, allowing direct complementation. Both assays consistently identified five VUS (p.V11G, p.H888Y, p.G1201S, p.Q1395R, and p.F1734L) as pathogenic due to significantly reduced HR efficiency, whereas the remaining 11 VUS were benign in terms of HR function. Notably, two pathogenic variants (p.H888Y, p.G1201S) were located outside the known functional domains of BRCA1. The five HR-deficient variants were further evaluated for their sensitivity to ionizing radiation, which confirmed the deleterious impact on DNA repair for variants p.V11G, p.H888Y, p.G1201S and p.F1734L. Sensitivity to the PARP inhibitor olaparib revealed hypersensitivity only in cells expressing p.V11G and p.F1734L, suggesting variant-specific mechanistic effects with potential therapeutic relevance. In conclusion, recombination-based functional assays complemented with sensitivity assays are effective tools for assessing the pathogenicity of BRCA1 VUS and can provide valuable information to support clinical decision-making. We propose that these assays can be completed within a timeframe compatible with clinical needs, offering critical insights for genetic counseling and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c220541b6ad477ac28279a59bfc46ba5327a7db4" target='_blank'>
              Functional characterization of BRCA1 variants of unknown significance using homologous recombination repair assays
              </a>
            </td>
          <td>
            Paloma Martín-Bejarano, E. M. Sánchez-Tapia, Pérez Jessica, Teresa Martín-Gómez, Rosario Vidal Tocino, Rogelio González-Sarmiento, A. B. Herrero
          </td>
          <td>2025-10-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy. ALKBH8 is a tRNA methyltransferase, specifically modifying the wobble uridine (U34) in tRNAs. Here, the authors discover that ALKBH8 is a direct transcriptional target of β-catenin, and that mediates intestinal tumorigenesis through regulation of the translation of KRAS in a codon-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="For a positive-strand RNA virus, the encoded viral RNA-dependent RNA polymerase (oRdRP) synthesizes complementary antigenome strand and uses it as a template for amplifying the viral genome, generating various replication intermediates. Structural proteins and viral genome are packaged into virions, but the fate of replication intermediates is underexplored. Here, we investigate Orsay Virus (OV) replication intermediates, including antigenome, oRdRP and double stranded RNA (dsRNA), using PCR and fluorescence-based imaging in C. elegans intestines. As for other positive-strand RNA viruses, we find that genome is in vast excess of antigenome. Antigenome is only visualized in cells when using denaturation protocols, indicating basepaired intermediates. OV antigenome is observed with distinct cytoplasmic and perinuclear localization patterns that depend on factors required for generation of primary, but not secondary, siRNAs. In both wildtype and RNA interference (RNAi) mutants, viral dsRNA is observed in the cytoplasm associated with oRdRP, suggesting cytoplasmic virus replication hubs. Additionally, using antibodies to oRdRP, we observed spherical structures of ∼1μm in diameter defined by oRdRP at their surface; over 75% of infected wildtype animals show these structures, which associate with mitochondria and autophagosomes in an antiviral RNAi- and autophagy-dependent manner, respectively. Our study defines new features of OV replication intermediates in wildtype animals, setting the stage for understanding their connection to the viral life cycle and host antiviral pathways. Author Summary Viruses infect and cause disease in all organisms. Once they enter an organism, or host, viruses make copies of themselves in a process called replication, and this process generates byproducts that are recognized as foreign by the host. This signals the host to begin protecting itself using pathways such as the innate immune response, underscoring the importance of understanding where and when replication byproducts are generated. Here we utilize Caenorhabditis elegans, a microscopic roundworm, to detect the viral replication intermediates of its naturally infecting pathogen, Orsay Virus (OV). For the first time, we detect OV antigenome, a byproduct that is used as a template for viral genome replication. We report a previously uncharacterized antigenome-containing perinuclear granule that is partially dependent on the antiviral RNA interference (RNAi) pathway, an invertebrate immune response. We also identify regions of cytoplasmic viral replication and unusually large (∼1 micron) spherical structures lined with the viral RNA-dependent RNA polymerase, an enzyme made by the virus for its replication. By characterizing important viral replication byproducts, our studies set the stage for understanding mechanistic connections between viral infection and the host response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191f2c795c34a173bf3ec9d8e533cdddf262fc53" target='_blank'>
              Detection of Orsay viral replication intermediates reveals spatial and regulatory links to Caenorhabditis elegans innate immune responses
              </a>
            </td>
          <td>
            Supraja Ranganathan, P. Aruscavage, Brenda L. Bass
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Single-cell epigenetic regulation relies on spatially organized enzymatic activities within nuclear compartments, which establish DNA methylation patterns that dictate gene expression and cellular identity. Given this spatial organization, we present a modular DNA-assembled nanopipette (mDNP) platform for ultrasensitive detection of DNA methyltransferase 1 (DNMT1) within the nucleus of individual cells. The device incorporates tripartite DNA molecular sentinel probes that undergo methylation-dependent sequence discrimination and initiate rolling circle amplification, producing quantifiable ionic current signals within a confined nanofluidic interface. This system offers high analytical precision and a broad linear range, enabling quantitative evaluation of DNMT1 inhibition and differentiation between malignant and nonmalignant cell populations. The regenerable mDNP device provides an integrated platform for single-cell epigenetic profiling and inhibitor screening at subcellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67457ecde67c0610e59fff5e8a9ab63e43837fb" target='_blank'>
              A Modular DNA-Assembled Nanopipette (mDNP) Platform for Spatially Confined Single-Cell Nuclear Detection of Methyltransferase Activity.
              </a>
            </td>
          <td>
            Juan Song, Chao-Wei Xia, Xian Zhang, Jing Xu
          </td>
          <td>2025-09-17</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c57f7b414b240c72f02d9113a840fdad776a055" target='_blank'>
              BRCA2-deficiency Causes Global Transcriptomic Alterations in Endothelial Cells
              </a>
            </td>
          <td>
            Sepideh Nikfarjam, Aman Singh, Shuhan Bu, Hien C. Nguyen, Krishna K. Singh
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The remarkable advances in cancer therapies significantly enhance the survival rates and longevity of cancer patients. Among childhood, adolescent, and young adult female cancer survivors, however, anti-cancer agents frequently cause primary ovarian insufficiency (POI), early menopause, and infertility, primarily due to the depletion of the ovarian reserve. Oocytes, the female germ cells, exhibit a notable susceptibility to DNA damage, given that they remain in meiotic arrest of prophase I for prolonged durations from months to years, which increases the risks of accumulating DNA damage overtime. To counteract this, a tightly controlled DNA damage response (DDR) signaling ensures that only oocytes with an intact genome progress to ovulation, fertilization, and next generations. Chemotherapeutic anti-cancer agents, including doxorubicin, cisplatin, cyclophosphamide, along with irradiation, elicit DNA damage via various mechanisms, including DNA crosslinking, single and double-strand DNA breaks, and oxidative stress. The genotoxic insults activate DDR in the oocytes, which detect and repair DNA damage or initiate apoptosis to eliminate impaired oocytes. Although several protein molecules such as DNA damage-sensing kinases, checkpoint kinases, p53 family transcription factors, and pro-apoptotic molecules, have been discovered, the precise mechanisms of DDR in determining the fate of oocytes, particularly how they differ from those in somatic cells and cancer cells, remain poorly understood. From an oncofertility perspective, the current review analyzes the molecular mechanisms of anti-cancer agent-induced DDR in oocytes and discuss knowledge gaps and urgent future research directions for preserving the ovarian reserve, fertility, and endocrine functions of young female cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman (Sonia) Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epstein-Barr Virus (EBV) establishes life-long latent infection in >90% of adults and is a causal agent for diverse cancers and autoimmune diseases. EBV has a complex life cycle in multiple different tissue types that involve dynamic variations in viral gene expression. These gene expression changes account for the success of the virus in long-term persistence and evading host immune control, as well as its potential for driving cancer evolution and autoimmune disease. Here, we review some of the salient features of EBV gene regulation highlighting the many variations of viral transcription. We review recent advances in our understanding of the factors that bind and regulate EBV gene expression. Based on this diversity of viral transcription patterns, we propose that EBV genome consists of gene modules regulated by local promoter-proximal transcription factor combinations that are further regulated by distal regulatory interactions among the various modules that interact through architectural factors, such as CTCF and cohesion. These modules are likely to represent chromatin architectural domains, and can also interact with host chromosome domains that further regulate viral and host gene expression. We propose that this gene regulatory hierarchy provides EBV with necessary plasticity for viral persistence, as well as a strong potentiator for cancer and autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a251ae41cd508722e48d62d5ca0f4e635efbec9f" target='_blank'>
              Transcriptional regulation of gene modules in Epstein-Barr virus.
              </a>
            </td>
          <td>
            Paul M. Lieberman
          </td>
          <td>2025-09-22</td>
          <td>Transcription</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cervical cancer (CC) is a highly heterogeneous malignancy primarily driven by persistent infection with high-risk human papillomavirus (HPV). However, comprehensive analyses of heterogeneity in the immune microenvironment, particularly its spatial heterogeneity, between HPV-positive and HPV-negative CC remain limited, despite their critical clinical significance. Methods We performed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) sequencing on collected cervical cancer samples, integrating scRNA-seq, ST, and bulk RNA-seq to analyze distinct cell subtypes and characterize their spatial distribution. Multiplex immunofluorescence analysis was further utilized to validate HPV status-specific expression patterns. Cox regression and LASSO regression analyses were used to identify the prognostic signature on the TCGA dataset. Results Through integrative analysis, we found that HPV-positive samples demonstrated elevated proportions of CD4+ T cells and cDC2s, whereas HPV-negative samples exhibited increased CD8+ T cell infiltration. In HPV-positive CC, epithelial cells acted as primary regulators of cDC2s via the ANXA1-FPR1/3 pathway, with cDC2s subsequently modulating CD4+ T cells and interferon-related CD8+ T cell subtypes. In contrast, HPV-negative CC featured epithelial cells predominantly influencing monocytes and macrophages, which then interacted with CD8+ T cells. Notably, the MDK-LRP1 ligand-receptor interaction emerged as a potential key mechanism for recruiting immunosuppressive cells into CC tumors, fostering an immunosuppressive microenvironment. Further, we constructed a risk score model based on an epithelial cell-related signature (ERS), which was significantly associated with patient survival. Noteworthy variations were observed in immune cell infiltration and immune microenvironment among distinct risk groups. Conclusion Based on integrated multi-omics data, we precisely delineated the spatial transcriptional features of the tumor microenvironment in CC with different HPV statuses, including identifying distinct CD8+ T cell states and cell-cell communication. In addition, we developed an ERS closely associated with the immune environment and prognosis of CC. These results increase our understanding of the molecular mechanisms of cervical cancer under different HPV statuses and provide assistance for the precise treatment of cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5152b47b483e0040237d8137dfb459c7f501aef2" target='_blank'>
              Integrated multi-omics analysis of single-cell and spatial transcriptomics reveals distinct hpv-associated immune microenvironment features and prognostic signatures in cervical cancer
              </a>
            </td>
          <td>
            Qiuyue Su, Xiangdong Tian, Fucheng Li, Xi Yu, Wenchen Gong, Yurong Chen, Jianan Wang, Siqi Yang, Shaojun Zhang, Qian Zhang, Shanshan Yang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres.
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Several autonomous mechanisms regulate protein expression, such as transcription, translation, post-translational modifications, and epigenetic changes. Rarely, these processes are controlled by the same molecular player with overlapping roles. Here, we reveal that transcription factor NFATc1 regulates both transcription and degradation of the Ca2+ channel Orai3 in a context-dependent manner. We demonstrate that NFATc1 drives Orai3 transcription in non-metastatic pancreatic cancer cells. In invasive and metastatic pancreatic cancer cells, NFATc1 induces Orai3 lysosomal degradation by transcriptionally enhancing MARCH8 E3-ubiquitin ligase. We show that MARCH8 physically interacts with Orai3 intracellular loop eventually resulting in its ubiquitination at the N-terminal. Mechanistically, the dichotomy in the regulation of Orai3 expression emerges from the differences in MARCH8 epigenetic landscape. We uncover that MARCH8 promoter is hyper-methylated in non-metastatic cells. Importantly, we demonstrate that MARCH8 restricts pancreatic cancer metastasis by targeting Orai3 degradation, thereby highlighting the pathophysiological importance of this signaling module. Taken together, we report a unique and clinically relevant scenario wherein the same transcription factor both enhances and curtails the expression of a target protein in cancer. By harnessing the epigenetic differences in the promoter of MARCH8 E3 ubiquitin ligase, the transcription factor NFATc1 drives Orai3 transcription and protein degradation, regulating pancreatic cancer in context-dependent manner. NFATc1 transcriptionally upregulates Orai3 Ca2+ channel in non-metastatic cells while inducing Orai3 lysosomal degradation via MARCH8 E3 ubiquitin ligase in metastatic cells. MARCH8 physically interacts with Orai3 and induces ubiquitination at the Orai3 N-terminal K2 residue. NFATc1’s dichotomous function derives from differences in the methylation status of the MARCH8 promoter. MARCH8 inhibits pancreatic cancer metastasis by driving Orai3 degradation. NFATc1 transcriptionally upregulates Orai3 Ca2+ channel in non-metastatic cells while inducing Orai3 lysosomal degradation via MARCH8 E3 ubiquitin ligase in metastatic cells. MARCH8 physically interacts with Orai3 and induces ubiquitination at the Orai3 N-terminal K2 residue. NFATc1’s dichotomous function derives from differences in the methylation status of the MARCH8 promoter. MARCH8 inhibits pancreatic cancer metastasis by driving Orai3 degradation. Dichotomous control by NFATc1 defines cell state-specific roles of the Ca2+ channel Orai3 in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc7315e3b7c364310dce4b187714fe3a0717601" target='_blank'>
              NFATc1 drives Orai3 transcription and proteolysis by harnessing epigenome differences in the MARCH8 promoter
              </a>
            </td>
          <td>
            Sharon Raju, Akshay Sharma, Sakshi Dahiya, Gyan Ranjan, Rajender K. Motiani
          </td>
          <td>2025-09-29</td>
          <td>The EMBO Journal</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Breast cancer is the leading cause of death in women under 50. The majority of breast cancers are estrogen receptor α-positive (ER+) and are commonly treated with hormonal therapies such as tamoxifen that inhibit ER activity. The TP53 tumor suppressor gene, encoding the p53 protein, is the most frequently mutated gene in breast cancer, and TP53 mutations are associated with diminished tamoxifen response and worse prognosis for breast cancer patients. Here, we report that in breast cancer cells p53 and ER cooperate to regulate the transcription of a set of genes encoding chromatin modifiers. The net result is a global increase in H3K4me3 and decrease in H3K9me3 chromatin marks. The resultant "open" chromatin is associated with increased transcription of luminal cell identity genes and enhanced tamoxifen sensitivity. Conversely, diminished p53 control of these chromatin modulators is associated with the evolution of tamoxifen resistance and cancer stem cell properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a29e75c12c65ca56f17dc6c777618735a34c52" target='_blank'>
              p53 regulates the expression of histone modifiers to restrict stemness and maintain differentiated luminal identity in breast cancer.
              </a>
            </td>
          <td>
            Yael Aylon, N. Furth, A. Pirona, Adi Lavie, Olga Fedorova, Ori Hassin, Nuno Padrão, Maxime Steinmetz, Avital Sarusi-Potuguez, Liat Fellus-Alyagor, Irit Shimoni, B. Dassa, W. Zwart, E. Shema, Moshe Oren
          </td>
          <td>2025-10-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Osteosarcoma (OS) has a high degree of malignancy, is prone to metastasize. This study aims to identify new biomarkers and natural drugs for OS. Open access OS-related datasets in Gene Expression Omnibus database (GSE39262, GSE21257 and GSE16088) were retrieved and analyzed. The genes related to OS progression were screened through differentially expressed gene analysis, weighted gene co-expression network analysis and univariate COX regression analysis. Immune infiltration analysis was performed using the CIBERSORT and ESTIMATE methods. The binding relationship between DExD-box helicase 39 A (DDX39A) and the natural drug coumerstrol was verified with molecular docking and cellular thermal shift assay. OS cell lines with DDX39A knockdown and overexpression were constructed respectively, to investigate the functions of DDX39A and coumerstrol. Cell viability, proliferation, apoptosis, cell cycle distribution, migration and invasion abilities were analyzed by CCK-8, EdU, flow cytometry and Transwell assays, respectively. DDX39A was highly expressed in OS samples and cell lines, associated with the poor prognosis of the patients. In vitro experiments confirmed that knockdown of DDX39A could inhibit the viability, proliferation, migration and invasion ability of OS cells, causing G1/S phase arrest and apoptosis. DDX39A had a good binding ability with coumerstrol. Coumerstrol treatment could inhibit the expression of DDX39A in OS cells and repressed the malignant behaviors of OS cells, while the overexpression of DDX39A reversed this phenomenon. DDX39A is a promising biomarker and therapy target for OS, and coumestrol exerts tumor suppressive properties in OS via inhibiting DDX39A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/292fa13d4b93b4b604a4c5c668626f4202a606ac" target='_blank'>
              DExD-box helicase 39 A, targeted by coumestrol, facilitates the malignant behaviors of osteosarcoma cells
              </a>
            </td>
          <td>
            Haiqing Wang, Yangli Jin, Zhang Feng
          </td>
          <td>2025-10-28</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The fact that breast cancer exhibits notable individual and racial variations in incidence, prognosis, and response to treatment. This highlights the need for a more thorough examination of the molecular mechanisms underlying these differences. KAISO (ZBTB33), long recognized as a methyl-CpG binding protein and transcriptional repressor, is increasingly understood to have more complex regulatory roles. KAISO has two DNA-binding sites: a methylated CpG motif and a consensus sequence unique to KAISO. Previous ATAC-seq study showed that KAISO can localize to the areas of open chromatin in different cancer cells, indicating possible roles beyond repression. Based on this data, we hypothesized that KAISO might interact with important regulatory proteins and form unique chromatin complexes to contribute to both transcriptional repression and activation. To investigate this, we analyzed MCF7 (luminal, Non-Metastatic, European American origin) and HCC1806 (Basal, Metastatic, African American origin) breast cancer cell lines, which differ in molecular subtype and racial background. We used Cut&Run to construct genome-wide KAISO binding profiles and Rapid Immunoprecipitation Mass Spectrometry of Endogenous Protein (RIME) to map KAISO-interacting protein networks. Our RIME data showed clear distinctions between the two lines: in HCC1806, KAISO strongly associated with molecular chaperones (HSPA8, HSP90AB1), elongation factor (EEF1A1), and immune-related proteins (e.g., C4B, APOC3), while MCF7 interactions favored structural and co-regulatory proteins (e.g., MYH10, CLTC, RBMXL1). Different chromatin engagement patterns were revealed by Cut&Run profiling. With promoter-associated peaks at about 90 locations, MCF7 showed an abundance of intronic (∼160 peaks) and intergenic binding (∼110 peaks), encompassing genes such as ATP9B, LINC01667, and TSPAN9. HCC1806, on the other hand, showed greater intronic enrichment (∼140 peaks) and fewer promoter peaks (∼50). All of these results point to KAISO's potential as a molecular marker for precision oncology by indicating that it is involved in transcriptional regulatory complexes that are unique by subtypes (Basal/Luminal, Metastatic/Non-Metastatic) in breast cancer. In order to further understand the regulatory function of KAISO, ongoing research attempts to expand these investigations to more related cell lines.



 Sazaul Morshed Sazib, Huixian Lin, Aliyu Muhammad, Md. Abu Siddique, Isra Elhussin, Moray Campbell, Shahjalal Chowdhury, Samuel Mwamburi, Malik McRae, Balasubramanyam Karanam, Clayton Yates. Kaiso defines distinct chromatin and protein interaction networks in breast cancer: Implications for precision therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C083.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4449400024ebdc9467232adda56233e690bba18f" target='_blank'>
              Abstract C083: Kaiso defines distinct chromatin and protein interaction networks in breast cancer: Implications for precision therapy
              </a>
            </td>
          <td>
            Sazaul M. Sazib, Huixian Lin, Aliyu Muhammad, Md. Abu Siddique, Isra Elhussin, Moray J. Campbell, Shahjalal Chowdhury, Samuel Mwamburi, Malik McRae, Balasubramanyam Karanam, Clayton C. Yates
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system. The authors present a diploid reference-quality genome assembly of the widely used cell line RPE-1, revealing heightened variation between haplotypes and other human genomes at centromeres. The availability of a reference genome matching the cell line under study promises a new era of high-precision haplotype-resolved functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>27</td>
        </tr>

        <tr id="The MRE11-RAD50-NBS1 (MRN) complex is a central, multifunctional factor in the detection, signaling and nucleolytic processing of DNA double-strand breaks (DSBs). To clarify how human MRN binds generic and telomeric DNA ends and can separate DNA end sensing from nuclease activities, we determined cryo-electron microscopy (cryo-EM) structures of human MRN bound to DNA and to DNA and the telomere protection factor TRF2. MRN senses DSBs through a tight clamp-like sensing state with closed coiled-coil domains, but auto-inhibited MRE11 nuclease. NBS1 wraps around the MRE11 dimer, with NBS1’s ATM recruitment motif sequestered by binding to the regulatory RAD50 S site, necessitating a switch in the NBS1 C helix for ATM activation. At telomeric DNA, TRF2 blocks the second S site via the iDDR motif to prevent nuclease and ATM activation. Our results provide a structural framework for DNA sensing via a gating mechanism and separation of sensing, signaling and processing activities of mammalian MRN. The MRN complex plays a key role in detecting, signalling, and processing DNA double-strand breaks. This study reports the cryo-EM structures of human MRN bound to DNA, alone and with telomeric protection factor TRF2, revealing MRN’s structural transitions and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7645e8c109e5a6f384b9d506f7906e33acb9e2b8" target='_blank'>
              Structural basis for DNA break sensing by human MRE11-RAD50-NBS1 and its regulation by telomeric factor TRF2
              </a>
            </td>
          <td>
            Yilan Fan, Filiz Kuybu, Hengjun Cui, K. Lammens, Jia-Xuan Chen, Michael Kugler, Christophe Jung, K. Hopfner
          </td>
          <td>2025-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceefcc8e25ead58e8d1394b9af76c598697ce059" target='_blank'>
              Nuclear Localization Signals Enable the Cellular Delivery of an Anti-CRISPR Protein to Control Genome Editing
              </a>
            </td>
          <td>
            A. Vera, Franklin J. Avilés-Vázquez, T. Ha, Amit Choudhary, Ronald T. Raines
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Although intragenic CpG dinucleotides are conserved during evolution, they are also sensitive to methylation-dependent mechanisms. Methylation status of the TP53 introns 1, 3 and 4 have been analysed in stage III ovarian carcinoma (OC). In the present study, the methylation of exon 4 (10 CpG pairs) was analysed in advanced-stage OC to investigate TP53 methylation and compare exon and intron 4 methylation patterns. A total of 80 samples from patients with advanced-stage OC and metastatic lesions were examined, along with 80 samples derived from healthy patients who had never been diagnosed with cancer. Methylation analysis of the human A2780 ovarian cancer cell line was also performed. Exon and intron 4 were methylated in OC, corresponding metastases and paired healthy tissue. The DNA from the human A2780 ovarian cancer cell line and the normal samples from healthy subjects was also methylated. The data indicate the existence of an intragenic mechanism of regulation of TP53 activity that involves demethylation/methylation processes. This mechanism provides the ability to alter the response from cell cycle arrest to apoptosis by manipulating only the expression of long or short p53 isoforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba65eba61b813922b785bd71652e055bf783b2a" target='_blank'>
              Exon 4 and intron 4 TP53 are both methylated in advanced-stage ovarian carcinomas
              </a>
            </td>
          <td>
            Wiktor Szewczuk, Oksana Szewczuk, Krzysztof Czajkowski, R. Gromadka, Maciej Waledziak, Marek Semczuk, Andrzej Semczuk
          </td>
          <td>2025-09-12</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c629e7baaceae40832311e536a58f80a6d8ec6a8" target='_blank'>
              Transcriptomic Mutational Profiling of Gastric Adenocarcinoma in Northern Brazil
              </a>
            </td>
          <td>
            Rubem Ferreira da Silva, Diego Pereira, Mateus Silva Tavares, Daniel de Souza Avelar, Jéssica Manoelli Costa da Silva, R. Mourão, Valéria Cristiane Santos da Silva, Juliana Barreto Albuquerque Pinto, Samir Mansour Moraes Casseb, S. Demachki, Tatiane Neotti, Ana Karyssa Mendes Anaissi, Williams Fernandes Barra, A. Vidal, G. Ishak, P. P. de Assumpção, R. Burbano, Fabiano Cordeiro Moreira
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>