<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-15 09:40:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>43</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>88</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>123</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>389</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>340</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>74</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells.
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic chromosome instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1, 2–3. Although CAs have been linked to mitotic errors resulting in the emergence of nuclear atypia4, 5, 6–7, the underlying processes and rates of spontaneous CA formation in human cells are underexplored. Here we introduce machine-learning-assisted genomics and imaging convergence (MAGIC)—an autonomously operated platform that integrates live-cell imaging of micronucleated cells, machine learning on-the-fly and single-cell genomics to systematically investigate CA formation. Applying MAGIC to near-diploid, non-transformed cell lines, we track de novo CAs over successive cell cycles, highlighting the common role of dicentric chromosomes as initiating events. We determine the baseline CA mutation rate, which approximately doubles in TP53-deficient cells, and observe that chromosome losses arise more frequently than gains. The targeted induction of DNA double-strand breaks along chromosome arms triggers distinct CA processes, revealing stable isochromosomes, coordinated segregation and amplification of isoacentric segments in multiples of two, as well as complex CA outcomes, influenced by the chromosomal break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The experimentation enabled by MAGIC advances the dissection of DNA rearrangement processes, shedding light on fundamental determinants of chromosomal instability. Coupling live-cell imaging, machine learning and genomic sequencing, the MAGIC platform enables investigation of the cellular context, mutation rates and triggers of spontaneous chromosomal abnormality formation, shedding light on fundamental determinants of chromosomal instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Understanding of the heterogeneity in the formation of the Ph has evolved as technology has advanced. In the pre-TKI era, FISH revealed derivative 9 deletions that were associated with shorter survival. The poor outcomes were overcome with imatinib. Whole exome and transcriptome sequencing revealed greater complexity: inversions; novel 9;22 fusions; sequence fragmentation; and random reassembly. These were termed Ph-associated rearrangements. They occurred on the Ph and/or the derivative 9, were thought to occur at the same time as the Ph and correlated with poorer response to imatinib.



 To use whole genome sequencing (WGS) to examine derivative chromosomes and further resolve the complexity of Ph-associated rearrangements in patients at diagnosis and pan-TKI resistant cell lines.



 WGS was performed for 7 chronic phase CML patients at diagnosis and 7 matched sensitive/resistant Ph-positive cell lines. Resistance was induced by exposure to increasing concentrations of imatinib. Data was analyzed using Oncoanalyzer. Derivative chromosome structure was resolved using the LINX algorithm, which clusters structural variants that are highly unlikely to occur independently. Rearrangements were validated using orthogonal methods, including Optical Genome Mapping and SNP Array. Six of 7 patients had pre-characterized Ph-associated fusions identified using RNAseq.



 For most patients, a balanced reciprocal Ph translocation involves 2 rearrangements: 1 that generates the Ph and BCR::ABL1; and 1 reciprocal derivative 9 with ABL1::BCR. These structures were resolved for 1 patient without pre-characterized Ph-associated rearrangements and for 5 of 7 parental cell lines.
 A distinct class of complex structural rearrangement called chromoplexy was identified involving the Ph translocation in 11 samples. These Ph-associated rearrangements had the highest number of linked structural variants within the individual genomes of 10 of the 11 samples. Chromoplexy involves the formation of extensive chained rearrangements affecting multiple chromosomes that create an intricate series of inter/intra-chromosomal fusions, deletions, duplications and inversions. Many DNA strands were broken and ligated in new configurations in the affected samples. Multiple genes on the derivative chromosomes were disrupted, including recurrent cancer drivers.
 For 3 of 7 matched sensitive/resistant cell lines, Ph-associated rearrangements were acquired in the resistant lines. The most complex events occurred in the LAMA84 resistant line. The parallel sensitive line had the standard 2 rearrangements. However, the resistant line gained 17 rearrangements with highly amplified Ph sequences, inversions and novel genomic fusions. Remarkably, small fragments from 6 chromosomes (2, 4, 10, 13, 17, X) were inserted upstream and downstream of BCR::ABL1, and disrupted the NUP214 driver gene.
 The K562 sensitive line displayed 24 rearrangements with non-contiguous amplified Ph regions of different copy number. Events included NOTCH1 deletion, 9;22 NUP214 fusion and an adjacent 9;13 fusion. Six fragments from chromosome 13 were incorporated into the Ph. The K562 resistant line displayed further evolution with 31 Ph rearrangements.
 The AR230 sensitive line displayed Ph chromoplexy with 9 rearrangements. The resistant line evolved with 19 additional Ph rearrangements. A fragment from chromosome 21 was inserted upstream of BCR::ABL1 and disrupted the MAPK1 driver gene. The LZTR1 tumor suppressor was deleted. All other cell lines maintained the standard Ph configuration.
 Ph-associated chromoplexy was evident in 6 patient samples: average 10 rearrangements. The patient with the highest number (15) progressed to blast phase at 4 months of imatinib. Four Ph-associated subclones were detected at diagnosis with non-contiguous amplified regions, consistent with the evolved cell lines. Furthermore, the LZTR1 tumor suppressor gene was deleted, as detected in AR230. The blast phase sample was unavailable. Deep molecular response was achieved in 1 patient on imatinib and 4 on frontline or switch to nilotinib/dasatinib.



 We have demonstrated that Ph chromoplectic rearrangement can be present at diagnosis but can also be initiated during leukemic progression and disrupt multiple genes, including cancer drivers. The Ph may be prone to extensive DNA breakage. Ph chromoplexy could constitute a new mechanism that contributes to acquired TKI resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed250c9c3a664955ee35598f9870a47e6c84f2b" target='_blank'>
              Whole genome sequencing reveals that the Philadelphia chromosome (Ph) can evolve and gain complex chromoplectic sequence rearrangements associated with tyrosine kinase inhibitor (TKI) resistance
              </a>
            </td>
          <td>
            S. Branford, C. Wadham, N. Shahrin, V. Saunders, Rosalie R. Kenyon, Ming Lin, Dominic Kaczorowski, Muneeza Maqsood, Hamish Scott, John Toubia, N. Shanmuganathan, C. Kok, Timothy Hughes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) fragments in the plasma capture cellular nucleosomal profiles since nucleosome-protected regions escape enzymatic degradation while nucleosome-depleted regions can not. We developed cfOncoXpress, a machine learning framework that uses fragmentation patterns to predict oncogene expression from cfDNA WGS. cfOncoXpress incorporates gene copy number aberrations inferred from cfDNA, including those associated with extrachromosomal DNA. Its application in prostate and breast cancers shows it can predict tumor subtype based on expression of signature genes and activated pathways. cfOncoXpress shows superior performance relative to other state-of-the-art methods and can be used to predict tumor gene expression when tissue biopsies are infeasible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd03a827003a113eb12bee1a6a13f9987699c5c0" target='_blank'>
              cfOncoXpress: Tumor gene expression prediction from cell-free DNA whole-genome sequences
              </a>
            </td>
          <td>
            Weiling Li, M. Roskes, A. Martínez-Fundichely, Ekta Khurana
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Background: Chromothripsis, a catastrophic event of chromosomal shattering and reassembly, is a key driver of oncogenesis and is associated with poor prognosis in multiple myeloma (MM). However, the specific functional consequences and the molecular mechanisms by which it rewires the MM regulatory landscape remained largely unknown.
 Methods: Using previous definitions, we identified chromothripsis (≥10 breakpoints, ≥7 CN oscillations, ≥200 kb segment sizes), chromothripsis-like (5-9 breakpoints and 3-6 CN oscillation) and non-chromothripsis (WT) samples. We utilized multiomic data from the CoMMpass dataset (n=1,066) and an independent Indiana University (IU) dataset (n=134), as well as 14 patient-derived xenograft (PDX) models with short-/long-read whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities and structural variation (SV), as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) to identify 3D chromatin architecture/folding inside the cell and enhancer-target interactions.
 Results: We identified 417 chromothripsis events in 268 patients (25%) from the CoMMpass dataset. Events were identified more frequently in relapsed and refractory (RRMM) samples compared to newly diagnosed (NDMM) samples (30% vs. 22%), suggesting an association between chromothripsis and disease progression. The number of events found in chromosomes highly correlated with chromosome sizes (r=0.63, Pearson correlation). However, events were particularly enriched on chromosomes 1, and 17 (0.26 and 0.30 events per million bps). Breakpoints of these events were enriched on 17q and 1p (odds ratio=2.2 and 1.7, p=0.0003 and 0.001). Chromothriptic samples were associated with inferior progression-free (PFS; 970 vs. 1227 days, p=0.003) and overall survival (2176 vs. not reached, p=0.006) compared to WT. In chromothriptic samples, 34% exhibited breakpoints on >1 chromosome and an higher number of affected chromosomes correlated with worse PFS (≥3 vs. 2, p=0.03; ≥3 vs. 1, p=0.02). Meanwhile, CN oscillations plus breakpoints was an independent factor associated with inferior PFS(p=0.05,: 700 vs. 1358 days). Del(TENT5C), del(TP53)and t(4;14) were enriched in the chromothripsis group (32%, 18% and 20%, odds ratio=2.6, 3.4 and 2.4, p<0.001) and are associated with genomic instability. We applied the same approach and further identified 19 chromothriptic samples from the IU dataset, where t(4;14) and del(TP53) were also enriched (odds ratio=2.6 and inf.) We identified differentially expressed genes (DEGs) between the chromothripsis group and a subset of WT group with balanced frequencies of high-risk events (e.g. t(4;14) & del(TP53)) to generate a chromothripsis signature. Pathway analysis on 37 consistent DEGs between the two datasets identified ‘G2/M checkpoint’ and ‘interferon α/γ responses‘(p<0.05), suggesting selection for a growth advantage and activation of the cGAS/STING pathway. A neuron network model for predicting chromothripsis was trained with DEGs and reached performance of AUC=0.90 and AUPR=0.87.
 We utilized multi-omics data from established PDX models to further examine the functional consequences of chromothripsis events. Among 9 events detected in 7 samples, 5 events exhibited significantly enriched APOBEC mutational signature and kataegis within events. Copy number change alone did not account for changes in gene expression within the chromothripsis regions, with 71% of genes dysregulated through neo-TAD formation and rewiring of the epigenome, specifically through juxtaposition of super-enhancers and allele-specific expression associated with DNA methylation changes. Key dysregulated genes included MYCN (Log2FC=6.3), the tumor suppressor PMAIP1 (Log2FC=-4.9), and cell adhesion proteins (EPCAM/CD326, Log2FC=3.5) that were rarely expressed in other MM samples.
 Conclusion: We revealed novel molecular characteristics of chromothripsis and the mechanisms caused it. We determined that the number of events and CN abnormalities are associated with poor outcome and that epigenetic states are reprogrammed within the events. These findings advance the clinical risk stratification and reveal unique therapeutic vulnerabilities tied to chromothripsis-associated epigenetic dysregulation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eeb625eee780834dfe72e26b192d78e2662c5ec" target='_blank'>
              Chromothripsis drives high-risk multiple myeloma by copy number abnormalities and rewiring the epigenome to activate oncogenic programs.
              </a>
            </td>
          <td>
            Enze Liu, Nathan Becker, Sreejeta Mondal, A. Suvannasankha, D. Coffey, B. Diamond, Abhishek Pandey, Kelvin Lee, D. Kazandjian, R. Abonour, Aneta Mikulasova, G. Morgan, O. Landgren, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) is a notable exception to genome integrity, characterized by significant DNA loss during development. In many nematodes, PDE is initiated by DNA double-strand breaks (DSBs), which lead to chromosome fragmentation and subsequent DNA loss. However, the mechanism of nematode DNA breakage remains largely unclear. Interestingly, in the human and pig parasitic nematode Ascaris, no conserved motif or sequence structures are present at chromosomal breakage regions (CBRs), suggesting the recognition of CBRs may be sequence-independent. Using Hi-C, we revealed that Ascaris CBRs engage in three-dimensional (3D) interactions before PDE, indicating that physical contacts of the break regions may contribute to the PDE process. The 3D interactions are established in both Ascaris male and female germlines, demonstrating inherent genome organization associated with the CBRs and to-be-eliminated sequences. In contrast, in the unichromosomal horse parasite Parascaris univalens, transient and pairwise interactions between two neighboring CBRs that would form the ends of future somatic chromosomes were observed only during PDE. Intriguingly, we found specific spatial compartmentalization changes in the Ascaris chromosomes after PDE converts 24 germline chromosomes into 36 somatic ones. Remarkably, Parascaris PDE generates the same set of 36 somatic chromosomes, and the compartment changes following PDE are consistent between the two species. Overall, our findings contribute to a model that the 3D genome facilitates the recognition of sequence-independent break sites for Ascaris PDE. It also suggests an evolutionary and developmentally conserved 3D genome reorganization of nematode somatic chromosomes following PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Background and Aims: Genomic instability, a prominent feature of cancer and precancerous conditions, leads to constant acquisition of genomic changes some of which enable these cells to progress through worsening stages of disease, overcome immune surveillance and survive through therapeutic interventions. The purpose of this study was to identify and validate genes and pathways driving genomic evolution in human cancer, with special focus on myeloma (MM).
 Methods: Gene expression and copy number data of ten human cancers (MM, hepatocellular carcinoma, hormone positive breast cancer, triple negative breast cancer and adenocarcinomas of esophagus, stomach, pancreas, lung, prostate and ovary) were used in a stepwise manner in each cancer. First, the extent of genomic instability in each patient was assessed by calculating total copy number events and then integrated with expression data to identify genes whose expression correlated with genomic instability. Subsequently the genes in this signature were investigated for correlation with survival in different MM datasets. Polo-like kinase 4, a member of this signature, was investigated in loss and gain of function studies for its impact on growth and chromosomal integrity.
 Results: Expression of 32 genes correlated with genomic instability in ten human cancers including MM. Functional enrichment analysis demonstrated that microtubule motor, RNA/DNA hybrid ribonuclease, flap endonuclease and 5‘-3‘ exonuclease activities were prominent molecular functions whereas spindle organization, chromosome segregation and nuclear division were major biological processes associated with these genes. Expression of 11 out of 32 genes associated with poor OS and/or EFS in three different MM datasets [MMRF, IFM70 (n=170) and GSE24080 (n=559)]. Of these 11 genes, polo-like kinase 4 (PLK4), with the highest relative expression in MMRF dataset was investigated further. Evaluation by Western blotting confirmed that expression of PLK4 was low in PBMC (n=2) whereas markedly elevated in all 10 MM cell lines tested. PLK4 knockdown significantly inhibited whereas its overexpression increased DNA breaks and genomic instability (as assessed from micronucleus assay) as well as growth rate of MM cells. Evaluation of cell cycle in PLK4-overexpressing MM as well as non-cancerous (HS5) cells identified a smaller third peak after G2 indicating a sub population of cells with DNA content > 4N. Consistently, the karyotypic evaluation of these MM cells demonstrated that after 8 weeks in culture, control cells had only 1 out of 70 (1.4%) abnormal metaphases, whereas 8 out of 70 (11.4%) metaphases in PLK4-overexpressing cells were abnormal. Abnormalities in PLK4-overexpressing cells included fused, broken and polyploid chromosomes. In non-cancerous immortal (HS5) cells, 5 out of 70 (7%) abnormal metaphases were observed in control cells whereas 15 out of 70 (21.4%) were abnormal in PLK4-overexpressing cells. Abnormalities included fused, broken, triradial and polyploid chromosomes. Evaluation by fluorescence in situ hybridization (FISH) demonstrated c-myc amplification (with 3 to > 5 copies) in 30/500 (6%) control cells whereas 56/500 (11.2%) PLK4-overexpressing cells. Similar observations were made when a centromere probe was used to investigate these samples by FISH, indicating an increase in intrachromosomal instability by PLK4. A specific PLK4 inhibitor caused inhibition of cell viability in all 4 MM cell lines tested (with IC50 ranging from 11 nM – 71 nM) with a minimal impact observed in 2 normal PBMC samples (IC50 > 50 M) indicating a high therapeutic index. Furthermore, we observed that PLK4 regulates expression and/or phosphorylation of PBK which in turn phosphorylates FOXM1 leading to increased genomic instability (through expression of DNA repair genes including RAD51) as well as contributes to ongoing proliferation of MM cells(through inhibition of DREAM complex).
 Conclusions: Using an integrated genomic strategy, we identified a global gene signature (comprised of 32 genes) associated with genomic instability and poor survival in human cancers including MM. One of the most prominent amongst these, PLK4, is identified as an important component of this signature impacting development of chromosomal changes including hyperdiploidy, a major defining feature in half of MM . With its role in cell cycle and and chromosome integrity, it is a promising target in MM to inhibit growth and progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a26341b8978a6f53aca199b7ea675509c5318f5" target='_blank'>
              Integrated genomic analysis identifies polo-like kinase 4 (PLK4) as a key regulator of chromosomal stability and growth in multiple myeloma
              </a>
            </td>
          <td>
            Leutz Buon, Poonam Kalhotra, S. Talluri, Jiangning Zhao, Harshini Sriram, C. Chakraborty, Masood Shammas, Nikhil C Munshi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Immune system control is a principal hurdle in cancer evolution. The temporal dynamics of immune evasion remain incompletely characterized, and how immune-mediated selection interrelates with epigenome alteration is unclear. Here we infer the genome- and epigenome-driven evolutionary dynamics of tumor-immune coevolution within primary colorectal cancers (CRCs). We utilize a multiregion multiomic dataset of matched genome, transcriptome and chromatin accessibility profiling from 495 single glands (from 29 CRCs) supplemented with high-resolution spatially resolved neoantigen sequencing data and multiplexed imaging of the tumor microenvironment from 82 microbiopsies within 11 CRCs. Somatic chromatin accessibility alterations contribute to accessibility loss of antigen-presenting genes and silencing of neoantigens. Immune escape and exclusion occur at the outset of CRC formation, and later intratumoral differences in immuno-editing are negligible or exclusive to sites of invasion. Collectively, immune evasion in CRC follows a ‘Big Bang’ evolutionary pattern, whereby it is acquired close to transformation and defines subsequent cancer-immune evolution. This study nominates immune escape as an early event in colorectal cancer and shows how this can be driven through both genetic and epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cleavage and polyadenylation (CPA) is a co-transcriptional mRNA processing mechanism that is central to mRNA and protein function. Dysregulation of CPA is widespread in cancer, promotes oncogenic programs, and affects patient outcomes. The CPA machinery is composed of multiple factors, and while prior research has investigated the impact of CPA gene expression on cancer phenotypes, the contribution of genomic alterations, such as mutations and copy number variations, remains largely unexplored. In this study, we conducted a pan-cancer analysis of genomic alterations in CPA genes. While numerous CPA genes harbor somatic mutations, these mutations do not significantly impact mRNA expression or provide prognostic value. In contrast, we found that copy number alterations in CPA genes have substantial clinical relevance. Notably, we identified the cleavage and polyadenylation specificity factor 1 (CPSF1) gene as the most frequently amplified CPA gene in cancer. While amplification of CPSF1 and MYC are co-occuring, CPSF1 amplification independently correlates with poor prognosis. We also found that CPSF1 amplification can impact 3′UTR length regardless of MYC status. Our study highlights the importance of CPSF1 as a promising prognostic factor in cancer and as a therapeutic intervention target to study in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4935b57359aad70cac663f1f7ac7eb81e40945" target='_blank'>
              AP-1 promotes oncogenic transcription in lung cancer cells by bridging promoter-enhancer interactions.
              </a>
            </td>
          <td>
            Xianglong Tan, Michael Kroneberg, Fei Sun, Kevin Avelar Diaz, Alisha Flora, Michael F Carey
          </td>
          <td>2025-12-03</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Centromere breakage has been associated with anti-centromere antibodies in systemic sclerosis (SSc), yet the origins of centromere damage and its link to immune activation remain unclear. The bleomycin induced fibrosis model is widely used as an experimental model of SSc. Here, we investigated whether bleomycin selectively disrupts centromeres and whether such damage contributes to chromosome instability and immunogenic chromatin mislocalization. Methods We evaluated centromere integrity and downstream genome damage-response phenotypes across complementary systems: the bleomycin mouse model of skin fibrosis, hTERT-immortalized human fibroblasts, and primary dermal fibroblasts from SSc patients. Centromeric α-satellite copy number, DNA damage and repair markers, micronuclei, nuclear envelope rupture, and colocalization of centromeric chromatin with antigen presentation machinery were assessed by qPCR, RT-qPCR, and immunofluorescence microscopy. Results Bleomycin induced selective depletion of α-satellite repeats at centromeres in mouse skin. In human fibroblasts, bleomycin generated double-strand breaks by γH2AX foci that preferentially colocalized with CENP-A-marked active centromeres and produced persistent centromere loss. Damage signaling required ATM and relied mainly on RAD51-mediated homologous-recombination repair, yet centromere restoration remained incomplete. Bleomycin also increased micronuclei, cytoplasmic centromeric foci, and BANF1-marked nuclear-envelope rupture, consistent with mis-segregation-driven chromatin leakage. CENP-B colocalized with HLA-DRB1 in bleomycin-treated fibroblasts and limited cutaneous SSc (lccSSc) fibroblasts, revealing centromere-derived antigen-presentation signatures. Conclusions This study identifies active centromeres as selectively vulnerable and incompletely repaired targets of bleomycin-induced DNA damage and demonstrates that centromere instability produces mislocalized chromatin capable of engaging antigen-presentation pathways. These findings provide mechanistic insight into SSc autoantibody specificity and link genome instability to immune activation in fibrotic autoimmunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/060c6a6ceec69d2ffe26ac48872dd1fd6fc913c0" target='_blank'>
              Centromere Instability Drives Chromosome Damage and Autoantigen Exposure in Systemic Sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Christine M Wright, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline.
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-03</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Prime editing, a flexible "search-and-replace" genome editing technique, has become a potent instrument for accurately altering the genome without the need for donor templates or double-strand DNA breaks. The vast non-coding regulatory genome, which includes enhancers, promoters, untranslated regions (UTRs), and non-coding RNAs, is now understood to be a major driver of tumorigenesis, despite the fact that the majority of cancer research has concentrated on protein-coding mutations. Oncogene dysregulation, tumor suppressor silencing, and treatment resistance can all result from mutations in these areas. The convergence of these two fields is examined in this review, which also describes the use of prime editing to fix particular, harmful non-coding mutations. In order to restore normal gene expression patterns, we describe methods for focusing on regulatory elements, present a landscape of non-coding drivers in cancer, and talk about the fundamentals of prime editing. Lastly, we look at delivery methods, therapeutic factors, and how prime editing might be incorporated into the precision oncology paradigm going forward. We emphasize how prime editing could help usher in a new era of targeted therapy and functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9d66804e5625efa917cbd699bfc770f8da5aa5" target='_blank'>
              Prime Editing the Non-Coding Genome: A Molecular Review of Correcting Regulatory Mutations for Precision Cancer Therapy
              </a>
            </td>
          <td>
            Alina Mansoor, Midhat Narmeen Mustafa, Abdul Ahad Mehboob, Nureen Shafaqat, Sanam Dehraj, Amna Noor, Sehresh Murtaza, Mahnoor Javed
          </td>
          <td>2025-10-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. Here, we demonstrate that Rad27/FEN1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining the stability of the ribosomal DNA (rDNA) repeats. Severe rDNA instability is observed in the rad27∆ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates Okazaki fragments in the rDNA region, without inducing the formation of detectable DSBs. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, Exonuclease 1 and PCNA partially compensate for the loss of Rad27 in rDNA stabilization. These findings highlight the importance of proper Okazaki fragment maturation in the maintenance of rDNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f06a68a9173b59a7c4386ddec1b029c06486a7" target='_blank'>
              Rad27/FEN1 prevents accumulation of Okazaki fragments and ribosomal DNA copy number changes.
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-10-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is among the most prevalent malignancies. However, the regulatory networks involved in tumor occurrence and development are still poorly understood. Human endogenous retroviruses (HERVs), a class of transposable elements, have been implicated in the development and progression of various human cancers. This study presents the first comprehensive locus-specific profiling of the expression of HERV gene transcripts in rectal cancer, revealing significantly dysregulated HERVs. Analysis of data from three Gene Expression Omnibus data sets revealed 25 upregulated HERVs and 7 downregulated HERVs. Dysregulation of HERV6196, a type of HERVH, was validated through reverse transcription quantitative PCR and droplet digital PCR in cells and tissues. Additionally, HERV6196 promoted the proliferation, inhibited the apoptosis, enhanced the colony formation ability, and enhanced the migration capability of rectal cancer cells. Moreover, HERV6196 functioned as an enhancer, promoting the expression of neighboring genes and the development of CRC. In summary, the present results revealed that HERV6196 is involved in the pathogenesis of rectal cancer, indicating the potential contribution of dysregulated HERVs to the development and progression of CRC through gene expression modulation. IMPORTANCE The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis. The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8dcd97edf79418a2a506bad015f01e1c7fbd0b0" target='_blank'>
              HERV6196 as an enhancer with oncogenic potential in rectal cancer
              </a>
            </td>
          <td>
            Yi-Xiu Gan, Xin Jiang, Zhi-Yu Wang, Yi-Lin Yu, Ling-Dong Shao, Jianmin Wang, 
          </td>
          <td>2025-12-08</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Mutations in non-coding regulatory elements are increasingly recognized as critical drivers of cancer development and progression. Among these, structural alterations such as duplications of super-enhancers (SEs), large clusters of enhancers that regulate key oncogenes, have emerged as significant contributors to oncogenesis. In T-cell acute lymphoblastic leukemia (T-ALL), a particularly aggressive hematologic malignancy, SE duplications have been identified in approximately 5% of human cases. These duplications frequently involve a non-coding region located approximately 1.3 megabases downstream of the Myc oncogene, a region known as N-Me. Despite the growing recognition of the role of SE duplications in T-ALL, functional studies to determine their molecular origins and contributions to leukemogenesis have been limited, largely due to the absence of suitable and genetically tractable preclinical models. This gap in knowledge hinders both our understanding of disease pathogenesis and our ability to design targeted therapeutic interventions.
 Here, we report the generation and characterization of a novel genetically engineered mouse model of T-ALL that spontaneously develops oncogenic SE duplications at the Myc locus. This model offers the first spontaneous in vivo system to investigate the origin, regulation, and oncogenic function of SE duplications. This model was created by conditionally stabilizing β-catenin in CD4⁺CD8⁺ double-positive (DP) thymocytes in combination with targeted deletion of the transcription factor HEB. This combination is sufficient to drive the development of aggressive T-cell leukemias that consistently harbor duplications of the N-Me super-enhancer. This finding recapitulates the genomic alteration observed in a subset of human T-ALL and provides a unique platform for mechanistic studies.
 We further demonstrate that the molecular events leading to SE duplication and oncogenic Myc activation unfold in a stepwise manner. At the preleukemic stage, stabilization of β-catenin alone is sufficient to enhance recruitment of HEB to the N-Me region. The simultaneous loss of HEB greatly increased chromatin accessibility at the N-Me enhancer. This altered chromatin landscape permits aberrant binding of transcription factors including TCF-1, which plays a critical role in T-cell development and leads to N-Me SE activation associated with significant three-dimensional chromatin reorganization. Chromosome conformation analyses in the leukemic cells reveal the formation of a novel chromatin loop that brings the activated N-Me SE into close spatial proximity with the Myc promoter, enabling oncogenic Myc transcription.
 These findings uncover a novel mechanistic pathway by which β-catenin and HEB interact to reshape the enhancer landscape and three-dimensional chromatin architecture, ultimately promoting SE duplication and aberrant Myc expression in T-ALL. β-catenin appears to play a dual role, first facilitating HEB recruitment under physiological conditions and then, upon HEB loss, permitting the reprogramming of enhancer activity through chromatin opening and ectopic transcription factor binding. Our data suggest that the cooperative interplay between signaling pathways and transcriptional regulators at enhancer elements can initiate structural genomic changes that drive malignancy.
 This work provides a long-sought preclinical model that faithfully mirrors key features of human T-ALL. Our findings open new avenues for understanding how non-coding regulatory mutations arise and exert oncogenic effects and have the potential to provide insights that may extend beyond T-ALL to other cancers driven by enhancer dysregulation. Moreover, by revealing molecular intermediates such as β-catenin, HEB, and TCF-1 in the regulation of enhancer activity, our findings identify potential therapeutic targets to disrupt the oncogenic enhancer-promoter interactions that underlie Myc-driven leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3354f2ef9255003e9c97bc93c32af47ddbd359" target='_blank'>
              Myc super-enhancer dynamics in T-ALL
              </a>
            </td>
          <td>
            Georgios Tousinas, Vasili Toliopoulos, Melissa Tracy, Akinola Emmanuel, K. Khazaie, Thomais Papamarcaki, P. L. Bergsagel, F. Gounari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfde1f929fb1d5e109604b5adc00a4ea625430b8" target='_blank'>
              Localised activity of reverse gyrase at gene regulatory elements
              </a>
            </td>
          <td>
            Paul Villain, Vladislav Kuzin, Augustin Darennes-Degaugue, Florence Lorieux, Antoine Hocher, Romain Le Bars, Tobias Warnecke, Laura Baranello, Tamara Basta
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Telomeres are known to be important for Leishmania biology but the mechanistic of how the process of telomere maintenance contributes to genome stability remains an unanswered question. Their maintenance is most commonly facilitated by the telomerase ribonucleoprotein complex that elongates telomeres countering their natural shortening due to the incomplete DNA replication in each cell cycle. In some organisms, telomere maintenance is achieved through telomerase-independent mechanisms, such as the Alternative Lengthening of Telomeres (ALT) pathways described in yeasts with dysfunctional telomerase and some rare telomerase-negative human cancer cells. Molecular markers for the ALT pathway include presence of the heterogeneous (in their length and sequence) telomeres, high level of telomeric exchange between the sister chromatids, increased expression of Rad51 and associated proteins, and occurrence of extrachromosomal telomeric repeats that can be present in either linear or circular form. Here, we used third-generation sequencing techniques in combination with other approaches and analyzed telomeres of L. mexicana at unprecedented high-level resolution. We demonstrate that Ku80 ablation-driven telomere elongation varies between chromosomes, possibly due to the chromosome-specific recombination rates, which are sequence/content dependent and associated with the structure of the telomeric tandemly repeated sequence, TTAGGG. Moreover, this telomere length heterogeneity is accompanied by an increased level of C-circles, a subclass of circular telomeric DNA highly specific for ALT activity. Our findings underscore that L. mexicana promastigotes have an inherent ability to utilize ALT, and the loss of Ku80 and/or TERT further enhanced this trait. These proteins work together to maintain telomere integrity, inhibit recombination, and stabilize telomere lengths. Our data suggest that ALT may be a fundamental and readily activated feature of Leishmania biology, and that telomere regulation in this organism significantly differs from what has been observed in other eukaryotic model species, including iconic T. brucei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e070c6fd10f58ebc83ce00a4b0839a86950d886" target='_blank'>
              Leishmania mexicana telomeres at high resolution: Ku80, TERT, and alternative lengthening mechanisms
              </a>
            </td>
          <td>
            Edubiel A. Alpizar-Sosa, Andreu Saura, Petr Fajkus, Ester Poláková, Kateřina Havlová, Amanda T. S. Albanaz, Jiří Fajkus, V. Yurchenko
          </td>
          <td>2025-10-24</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human tissues require a mechanism to generate durable, yet modifiable, transcriptional memories to sustain cell function across a lifetime. Previously, it was demonstrated that nanoscale packing domains couple heterochromatin (cores) and euchromatin (outer zone) into unified reaction volumes that can generate transcriptional memory. In prior work, this framework demonstrates that RNA synthesis occurred within the ideal zone (intermediate density) portions of the domain. Naturally, this creates a question of where genes are positioned in relation to the packing domain architecture and which genetic material fills the domain core to sustain transcription. Here, it is proposed that this can be solved by the encoded positioning of introns, intergenic segments, and exons as a projection of the functional packing layers of domains. This suggests that introns and intergenic segments are coupled to adjacent exons to generate coherent packing domain volumes. How this organization will reconcile contradictions in epigenetic patterns, non-randomness in oncogenic mutations, and produce durable transcriptional memory is illustrated. The study concludes by showing that this genome geometry may have coincided with the rapid evolution of body-plan complexity, suggesting that chromatin geometry could be fundamental to metazoan evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) selectively removes portions of the genome during development, deviating from the genome constancy rule. Our understanding of PDE mechanisms, including the involvement of small RNAs and transposases, is mainly based on studies in the single-cell ciliates, whereas in metazoans, the PDE mechanisms remain largely unknown. In the parasitic nematode Ascaris, PDE removes and remodels all germline chromosome ends, splits fused chromosomes into somatic chromosomes, leading to dramatic karyotype changes. To probe the possible involvement of small RNAs in Ascaris PDE, we identified two Argonautes, AsWAGO-2 and AsWAGO-3, associated with PDE. Using cellular fractionation followed by nuclease-treated immunoprecipitation, we characterized WAGO-associated small RNAs (siRNAs) and their targets in chromatin, nucleoplasm, and cytoplasm. We found that AsWAGO-3 is enriched on eliminated chromatin, with its associated siRNAs mostly targeting genes. However, a subset of AsWAGO-3 siRNAs targets repeats in the eliminated regions specifically located in the middle of chromosomes that originated from the ends of ancestral chromosomes. In contrast, AsWAGO-2 transiently stains the retained chromosomes during elimination mitoses, and the associated siRNAs are enriched in the retained regions only in the chromatin fraction. Overall, our data established a link between WAGOs and Ascaris PDE, demonstrating the flexibility of small RNA pathways that selectively distinguish retained and eliminated DNA. We suggest that some of the targeting may be a response to the karyotype changes associated with chromosome fusion and PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f77d30c5b35d3bee66492e606ea00f82e5f1ceb" target='_blank'>
              Argonautes and small RNAs associated with nematode programmed DNA elimination
              </a>
            </td>
          <td>
            M. Zagoskin, Yuanyuan Kang, Richard E Davis, Jianbin Wang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society Transactions</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) and its aggressive subtype plasma cell leukemia (PCL) are transcriptionally-driven hematological malignancies, heavily dependent on enhancer activity. Enhancers are distal regulatory elements at which transcription factors (TFs) bind, recruiting coactivators that promote chromatin remodeling to stimulate target gene expression via physical proximity of enhancer and promoter. MM cells show high expression of essential TFs, including IRF4, MYC, PRDM1 and IKZF1/3, which function by binding to and activating oncogenic enhancers.
 A detailed understanding of chromatin regulation in MM patients requires the comparison of histone modifications, chromatin accessibility and TF/cofactor binding distribution within the same cells, which has so far been unachievable. Patient analysis has mainly relied on ATAC-seq and RNA-seq, with limited inference of TF binding and chromatin state, owing to the difficulty in obtaining the large numbers of MM cells required for more detailed characterization. Many observations of enhancer function in MM therefore come from cell lines, which only offer an approximation of MM physiology.
 To overcome the limitations of in vitro models and primary cell availability, we employed TOPmentation (TF-OPtimized ChIPmentation) to generate ChIP-seq-quality data for CD138+ cells from MM and PCL patients. This technique uses as few as 100,000 cells for histone modifications, and 250,000 cells for TFs, allowing us to profile up to 15 features per patient. In addition, we used the base-pair resolution chromosome conformation capture technique micro-capture-C (MCC) to visualize enhancer-promoter interactions in a PCL patient sample. This allowed us to dissect large oncogenic enhancers, identifying the key TF binding sites within each enhancer that directly contact the gene promoter. Using this dual approach, for the first time we have generated integrated maps of TF occupancy, active and repressive histone modifications, and chromatin interactions in MM patient samples.
 We characterized the epigenetic landscapes of two t(11;14) and two t(4;14) patients, finding broadly similar chromatin profiles in each subtype. Key MM enhancers were retained at oncogenes including IRF4, MYC,PRDM1 and IKZF1/3, indicating a convergence of gene regulation in MM originating from distinct initiating genetic events. At many enhancers, we observed multiple TFs co-binding at the same sites, indicating cooperative activation of target genes. By combining TOPmentation with MCC, we found that many of these sites interact with target gene promoters, directly implicating them in gene regulation.
 In MM, patients typically display mutually exclusive upregulation of CCND1 or CCND2. As previously established, we found strong activation at the CCND1 locus by IGH translocation in t(11;14) patient samples, whereas in t(4;14) patients CCND2 was upregulated via an upstream super-enhancer. Surprisingly, in each subtype the silent CCND gene was not actively repressed, but rather existed in a poised, bivalent state marked by active H3K4me3 and repressive H3K27me3 modifications. This argues that intricate regulation of CCND1/2 is required to maintain optimal levels of expression for tumor growth.
 Comparison of patient and cell line data showed broad conservation of epigenetic features. However, we also observed examples of distinct gene regulation, for example at the PRDM1 locus. In several MM cell lines (including KMS12, H929 and MM1S), transcription of PRDM1 initiates from a distal promoter. However, this locus was inactive in the patient samples analyzed, and instead an alternate, proximal promoter was favored.
 This study for the first time reports the genomic binding distribution of TFs and histone modifications in MM patient cells, demonstrating the importance of comprehensive epigenomic analysis to capture the complexity in MM gene regulation. This emphasizes the need for validation of cell line models to provide a more physiologically relevant understanding of enhancer-driven oncogene regulation. As we expand this patient-derived dataset, it will provide a valuable resource to better understand MM biology and inform future strategies for precision epigenetic therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab64ef7d2d4fd86af25053430e4f99417b63dfb" target='_blank'>
              Revealing oncogenic enhancer regulation by epigenomic profiling of multiple myeloma and plasma cell leukemia patient samples
              </a>
            </td>
          <td>
            Katrina Fordwor, Jason Taslim, Sophie Ball, P. Sabbattini, A. Katsarou, T. Milne, A. Karadimitris, A. Chaidos, Nick Crump
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fdb1478a9bc7792b93fd92427bd911d727f3989" target='_blank'>
              Resolving the CES1 Genomic Locus with Cas9-Directed Targeted Long-Read Sequencing for Precision Pharmacogenomics
              </a>
            </td>
          <td>
            E. Lekka, A. Ambrodji, A. Nater, A. Ballah, U. Amstutz, A. Ramette, C. R. Largiadèr
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/473839999da7d90946bb0b2160d9978b716dce95" target='_blank'>
              Human DICER1 hotspot mutation induces both loss and gain of miRNA function.
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-11-04</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The c-MYC transcription factor is aberrantly expressed in most human cancers to enhance expression of proliferative gene programs. Owing to its pseudo-ordered structure and reliance on extensive and dynamic protein-protein interactions in distinct transcriptional regulatory complexes, defining context-specific MYC interactors has remained challenging. Therefore, mapping MYC-centered complex topologies in disease relevant models could identify components critical for its function which may serve as therapeutic targets in MYC-driven cancers. Here, we present a matched pair of photoproximity probes coupled with quantitative proteomics which enable context-dependent mapping of protein complex topology inside cells. We applied this spatially resolved, intracellular photoproximity (siPROX) profiling workflow to map MYC interactomes across temporal, spatial and disease-relevant contexts. Basal and inhibitor-treated profiles confirmed interactions with a wide range of known chromatin-associated transcriptional regulatory factors that define the extended MYC transcriptional bubble in live cells. Time-resolved mapping of inhibitor treated cells identified dynamic remodeling of numerous transcriptional regulatory factors and identified several BAF complex members (e.g., PBRM1 and SMARCC1)1 that persist in the presence of bromodomain inhibition. Furthermore, spatial MYC topology maps in small cell lung cancer cells confirmed the presence of BAF complex members under conditions where MYC induced target gene expression, altered cell morphology and enhanced proliferation. Lastly, loss of BAF function via inhibition of SMARCA2/4 ATPase activity resulted in rapid loss of chromatin-bound and nuclear MYC levels, downregulation of MYC-dependent transcripts and MYC-specific cell growth in several cancer cell models. Together, these data highlight the potential for siPROX to identify spatially resolved, dynamic TF interactors and highlight MYC-proximal BAF interactions as a targetable liability to regulate MYC-dependent transcription and proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78a06fd992b15a6520a4d4a8e38c9a04a5e3b92" target='_blank'>
              Spatially-resolved Photoproximity Profiling of MYC Identifies a MYC-BAF Liability in Cancer Cells
              </a>
            </td>
          <td>
            Anthony J. Carlos, Shuyuan Huang, Dongbo Yang, Colin Swenson, Pratyasha Chakraborty, Shaopeng Yu, Benjamin D. Stein, Raymond E. Moellering
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8df6dc2d72a4159b55a77cb1f1691bd8b0d91d08" target='_blank'>
              Click-code-seq reveals strand biases of DNA oxidation and depurination in human genome.
              </a>
            </td>
          <td>
            Vakil Takhaveev, N. Püllen, Navnit K. Singh, Lucie Lefevre, Emilie A Aghajani, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2025-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Characterizing tissue-specific (TSp) gene expression is crucial for understanding development and disease; however, traditional expression-based methods often overlook the latent “regulatory grammar” embedded in the non-coding DNA, particularly in distal promoter regions. Here, we introduce TSProm, a framework that specializes a DNA foundation model (DNABERT2) to decipher the long-range regulatory logic of TSp promoters at the gene isoform level. The contributions of our work are two-fold. First, we propose a novel comparative design that trains two distinct models, A: for general promoter biology and B: for TSp regulation. These models enable the precise isolation of sequence motifs around the transcription start site that uniquely define tissue identity. Second, we introduce a comprehensive explainable AI (xAI) module that integrates attention-based discovery with model-agnostic SHAP analysis to provide robust, cross-validated interpretations of learned features. Applying this framework to human brain, liver, and testis promoters, we identified and validated clinically relevant transcription factors (TFs) in the brain, including SP1, MYC, and HES6, and confirmed their known roles in diseases such as gliomas and neuroblastomas. Our analysis further revealed that C2H2 Zinc Finger proteins are a dominant feature of the global landscape of TSp gene regulation. TSProm provides a novel and interpretable framework for identifying TSp gene regulatory elements, offering powerful computational tools for the study of tissue-specific gene regulation in normal and disease conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a83e73abe66e3648e75b598545eed1991706b17" target='_blank'>
              TSProm: Deciphering the Genomic Context of Tissue Specificity
              </a>
            </td>
          <td>
            Pallavi Surana, Pratik Dutta, Nimisha Papineni, Rekha Sathian, Zhihan Zhou, Han Liu, Ramana V. Davuluri
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Intracranial germ cell tumors (IGCTs), primarily affecting infants and children, exhibit malignancy and prognosis that largely depend on tumor histological subtypes. While somatic coding alterations have revealed activation of the MAPK and PI3K pathways in the germinoma subtype, the genetic basis of the more malignant yolk sac tumor (YST) subtype remains poorly understood. In this study, we analyzed whole-genome sequencing data from blood-tumor matched samples of 69 IGCT patients, identifying a significant enrichment of somatic chromosome 1p36 copy number loss in YST (7 out of 13 YST cases compared to 3 out of 56 non-YST cases). Further analysis of whole-genome level somatic single nucleotide variations revealed a YST-specific single base substitution signature (5 out of 13 YST cases), characterized by frequent ACG to AGG mutations. Alongside this novel signature, we observed a high burden of point mutations, structural variations, and LINE1 retrotransposition, indicating increased genomic instability in this subgroup of IGCTs. Additionally, association analyses revealed that these genomic instability events were significantly linked to the copy number loss of 1p36. These findings elucidate the distinct mutational landscape of YSTs and suggest potential treatment vulnerabilities for this subtype by targeting genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d3aac5eea035f8521957bf70a25b2e906a88c96" target='_blank'>
              EPCO-29. WHOLE-GENOME SEQUENCING REVEALS GENOMIC SIGNATURE OF INTRACRANIAL YOLK SAC TUMORS
              </a>
            </td>
          <td>
            Dong Song, Yang Zhao, Ran Liu, Jie Ma, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous single-nucleotide polymorphisms (SNPs) associated with various diseases, including cancer. However, the mechanisms by which these SNPs contribute to disease susceptibility remain largely unclear. While recent studies have explored the transcriptional impact of disease-associated SNPs, their role in post-transcriptional regulation has been less extensively investigated. In this study, we investigated whether cancer-associated SNPs influence gene expression by altering N6-methyladenosine (m6A) RNA methylation. We collected GWAS-identified SNPs across nine cancer types and integrated these with matched tumor and normal m6A RNA immunoprecipitation sequencing (m6A-seq) and RNA sequencing (RNA-seq) datasets. We first identified differentially methylated m6A sites and assessed whether cancer-associated SNPs were enriched within these regions. These analyses revealed that cancer-associated SNPs were significantly enriched within hypermethylated m6A regions in colon cancer. Integrative analysis revealed that SNPs enriched in m6A-modified regions are associated with altered gene expression and RNA splicing, suggesting that m6A methylation mediates the post-transcriptional impact of genetic variants. Experimental validation further confirmed altered gene expression following ALKBH5 knockdown, consistent with patient-derived data. Collectively, our findings support a novel mechanistic connection between genetic variants and RNA methylation-driven transcriptomic regulation in colorectal cancer, underscoring the epitranscriptome as a potential axis of oncogenic control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8030aadad77b99c149edba7083659d634eaa70" target='_blank'>
              Genetic variants reshape the m6A epitranscriptome and drive transcriptomic reprogramming in colorectal cancer
              </a>
            </td>
          <td>
            Seung Hun Han, Seongmin Jang, Yeongwon Kim, Kun Tan, M. F. Wilkinson, Hyobin Jeong, Junho Choe
          </td>
          <td>2025-11-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Significance Transposable elements (TEs) constitute about half of the vertebrate genome. Previously, we provided evidence that TE silencing by CpG methylation in the germ line and subsequent deamination resulted in strong CpG depletion not only in the TEs but also in the host genome. Present day vertebrate genomes are CpG poor except for the 1% of the genome represented by CpG islands (CGIs). We propose a theory that TEs are indirectly responsible for the existence of CGIs through selection bias. Additionally, the dilution of the CpG sequence in the majority of host DNA allowed for the evolution of regulatory elements (REs). These REs utilize reversible DNA methylation for gene control and include tissue-specific transcription start sites and enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb89f7ec1a712e4f367ad5750458812dca5078d" target='_blank'>
              Roles of transposable elements and DNA methylation in the formation of CpG islands and CpG-depleted regulatory elements
              </a>
            </td>
          <td>
            J. You, Steve Pierce, G. Liang, Peter A Jones
          </td>
          <td>2025-10-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b332eb177c3231afc269f28608f60924be826cc" target='_blank'>
              Pan-cancer analysis of RNA expression signatures associated with cancer tissue architecture.
              </a>
            </td>
          <td>
            Megumi Mogi, Katsushige Kawase, Hiroyoshi Y. Tanaka, Suguru Miyata, Satoko Takemoto, Hideki Ikeda, Eri Katayama, Yuki Nakamura, Ryuta Kojima, Takao Morinaga, Shihori Tanabe, S. Yonekura, M. Kano, Toyoyuki Hanazawa, Masahito Kawazu
          </td>
          <td>2025-10-17</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90cf835255eacd329046c889367761e186d358b" target='_blank'>
              Cohesin forms fountains at active enhancers in C. elegans.
              </a>
            </td>
          <td>
            Bolaji N. Lüthi, Jennifer I Semple, Anja Haemmerli, Saurabh Thapliyal, Kalyan Ghadage, Klement Stojanovski, Dario D’Asaro, Moushumi Das, Nick Gilbert, Dominique A. Glauser, Benjamin D. Towbin, Daniel Jost, Peter Meister
          </td>
          <td>2025-12-11</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability.
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Polycomb targeted loci form long-range chromatin interactions independent of CTCF-cohesion and are demarcated by low DNA methylation(1–3). Polycomb targets form extremely long-range loops (long Polycomb loops) that can separate anchors up to 60 Mb. These loops predominantly occur in cells of self-renewal status, such as human hematopoietic stem cells (HSC) and mouse embryonic stem cells (ESC), but rarely in cell lines. To identify long Polycomb loops in cancer, we initiated a pan-cancer survey of long Polycomb loops in a collection of 264 tumor samples (33 acute myeloid leukemias, 17 T cell lymphoblastic leukemia, 29 Breast cancer samples, 63 pediatric brain tumors, 70 prostate cancers, and 42 colon cancers). We found most cancers, including all prostate cancers and colon cancers, lack long Polycomb loops except pediatric brain tumors and certain AMLs. In pediatric brain tumors, we found 30% of the ependymoma PFA subtype (which shows globally depleted H3K27me3) notably displayed strong long Polycomb loop interactions. AML displayed more diverse levels of long Polycomb loop interactions. Most AMLs lost both long Polycomb loops with Polycomb binding loss at the loop anchors. Whereas 10% of AMLs retain long Polycomb loops as strong as in HSCs. These AMLs recurrently carry mutations in CEBPA and STAG2, which are not associated with the Polycomb complex or DNA methylation machinery. We found that long Polycomb loop strong AML is sensitive to EZH2 inhibition, which induces cell differentiation. Conversely, PRC1 component dependency can be used to predict long Polycomb loop formation in B-cell lymphoblastic leukemia cell lines. Our analysis suggests that the oncogenesis process antagonizes the long Polycomb loop maintenance in most cancers, yet certain cancers may still preserve strong long Polycomb loops from cell-of-origin. The maintenance of long Polycomb loops sensitizes cells to Polycomb inhibition, indicating that such loops could be an epigenomic biomarker for pharmacological or genetic Polycomb inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd40648aae14140db85bd195a593affdaa280e98" target='_blank'>
              Pan-cancer 3D genomic analysis revealed extremely long Polycomb loops as the biomarker for sensitivity to Polycomb inhibition
              </a>
            </td>
          <td>
            Sean Moran, Zhong Fan, M. Zanovello, Feng Fan, David Xue Qing Wang, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, Ivan Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7bcb7a0494ca2d2085913525073bd8291d09b9" target='_blank'>
              EGFR suppression and drug-induced potentiation are widespread features of oncogenic RTK fusions
              </a>
            </td>
          <td>
            Yuzhi (Carol) Gao, D. Gonzalez-Martinez, Sofia Wissert, Hana Bader, Nidhi Sahni, Anh Le, R. Doebele, Lukasz J. Bugaj
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c948196f370f90e50659421ebd98d4fcdbcb30c7" target='_blank'>
              Differentiation-Dependent Proximity Proteomics Identifies Novel Host Factors Linked to HPV16 E2 Function
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, A. T. Prabhakar, Jenny D Roe, Elinor Lu, Austin J. Witt, S. Giri, Molly L Bristol, Phoebe Bridy, Xu Wang, Arjun Rijal, Charles Lyons, I. Morgan
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although NUP85 is a member of the nuclear pore complex (NPC) and is associated with chromosome variation and tumor regulation, its specific role in cancer development remains unclear and requires further research. This study analyzes the mRNA and protein levels of NUP85 in normal and tumor tissues using the TCGA, GTEx, and HPA databases. We investigated the relationship between NUP85 and survival rates, as well as clinical features, in various tumors by analyzing the TCGA database. The expression pattern of NUP85 in cancer cells is analyzed using single-cell sequencing data from the TISCH database. Two types of prostate cancer cell lines are utilized to investigate the impact of NUP85 on cell proliferation, migration, invasion, and apoptosis, as well as its regulatory mechanism. These analyses aim to uncover the role of NUP85 in cancer, particularly in prostate cancer. NUP85 is observed to exhibit elevated expression levels across multiple malignancies, with its heightened expression showing consistent associations with poorer clinical prognoses. Bioinformatic analyses further reveal that NUP85 expression patterns demonstrate significant correlations with the activity of cancer-related pathways and immunological interactions involving macrophages and T cell populations. Notably, experimental studies using prostate cancer models have documented reduced cellular proliferation following NUP85 knockout, suggesting a potential functional connection warranting further mechanistic investigation. The results indicate that elevated NUP85 expression shows strong correlations with cancer initiation and progression. These findings support its potential utility as a candidate biomarker for disease monitoring across various malignancies, though mechanistic validation remains necessary to establish clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d9fcec4dab152579fb48a48d3474142345170" target='_blank'>
              Comprehensive pan-cancer evaluation of NUP85 prognosis, immune infiltration response, and validation in prostate cancer
              </a>
            </td>
          <td>
            Zhijun Chen, H. Guan, Long Chen, Xinwei Yuan, Wenyan Sun, Ming Chen
          </td>
          <td>2025-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute promyelocytic leukemia (APL) is a subtype of acute myeloid leukemia (AML), characterized by a fusion between the PML and RARA genes and by a block in the myeloid maturation at the promyelocytic stage. This study investigates the epigenetic landscape of APL by integrating ChIP-seq data on eight histone modifications and RNA-seq in APL as well as non-APL AML. APL showed a distinct chromatin profile that differed from non-APL AML. We describe APL-specific changes in H3K27ac, H3K9me3, and H3K27me3 with impact on enhancer activity, repression of transposable elements, and Polycomb regulated gene repression. The APL-specific H3K27ac pattern identifies APL-specific enhancer and super-enhancer regions, including a subset of enhancers that are bound by the PML-RARA fusion protein. While chromatin bound specifically by PML-RARA were dominantly active, APL was also characterized by gain of APL-specific heterochromatin states with significant gains of H3K9me3 enriched lamina-associated domains and the transposable elements LINE, LTR, and SINE. These findings suggest a unique enhancer and heterochromatin profile in APL, with implications for transcription regulation and treatment response. These findings offer novel insights into the pathogenesis of APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4d4d5354c6d704bfe8c1f882f567114dfdc0fb" target='_blank'>
              Disease-specific epigenetic deregulation of enhancers, transposons, and polycomb targets in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Xiangfu Zhong, Lina Cordeddu, Angelica Gamboa-Cedeno, S. Bengtzén, Karl Ekwall, Andreas Lennartsson, Sören Lehmann
          </td>
          <td>2025-10-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Defective viral genomes (DVGs) are byproducts of replication that arise during infection with diverse RNA viruses and can impact virus infection and disease outcome. To gain insight into DVG generation during arenavirus infection, we serially passaged Tacaribe virus at a high multiplicity of infection, which led to the generation of both deletion DVGs (del-DVGs) and copyback DVGs (cb-DVGs). Interestingly, specific combinations of start/stop breakpoints were highly overrepresented, resulting in certain DVGs being highly enriched within the population. Functional characterization of the most prevalent del-DVGs suggests that they are able to compete for interaction with the viral RNA synthesis machinery and that this ability is length-dependent. A closer analysis of the DVG breakpoints used to generate highly abundant DVGs revealed a role for local sequence identity in the formation of cb-DVGs, while del-DVG formation was associated with the presence of specific nucleotide triplets (i.e., TAG, AGA, and GAA). Taken together with similar findings from other virus families, this then supports the idea that DVG formation is not a random process, but rather that specific mechanisms promote their formation at certain positions. The characterization of these individual arenavirus DVG sequences, and also the identification of sequence elements associated with their production, will facilitate future work examining their impact on arenavirus biology, and also opens up the possibility of using such sequences as a part of antiviral approaches and/or of modulating their production as a part of virus attenuation strategies. IMPORTANCE Infection with diverse RNA viruses can generate defective viral genomes (DVGs) that, while unable to support productive virus infection on their own, appear to play a crucial role in determining infection outcome. In light of this apparent biological importance, there is an urgent need to better understand the sequence characteristics of individual DVGs and the molecular mechanisms that regulate their formation to study their biological functions. We have now characterized several DVGs that are highly enriched during infection with the arenavirus Tacaribe virus. Functional analysis of a subset of these DVGs showed length-dependent competition for the viral RNA synthesis machinery, while detailed sequence analysis revealed that DVG formation involves either regions of sequence identity within the genome or the presence of specific nucleotide sequences. Understanding these mechanisms opens up the possibility to leverage DVG generation in support of antiviral and/or vaccine attenuation approaches. Infection with diverse RNA viruses can generate defective viral genomes (DVGs) that, while unable to support productive virus infection on their own, appear to play a crucial role in determining infection outcome. In light of this apparent biological importance, there is an urgent need to better understand the sequence characteristics of individual DVGs and the molecular mechanisms that regulate their formation to study their biological functions. We have now characterized several DVGs that are highly enriched during infection with the arenavirus Tacaribe virus. Functional analysis of a subset of these DVGs showed length-dependent competition for the viral RNA synthesis machinery, while detailed sequence analysis revealed that DVG formation involves either regions of sequence identity within the genome or the presence of specific nucleotide sequences. Understanding these mechanisms opens up the possibility to leverage DVG generation in support of antiviral and/or vaccine attenuation approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ae090613229703491d2a14d808168d309835c4" target='_blank'>
              Molecular characterization of arenavirus defective viral genomes reveals sequence features associated with their formation
              </a>
            </td>
          <td>
            T. Hoenen, Patrick Bohn, Sebastian Herndler, Marine Klamke, , A. Groseth
          </td>
          <td>2025-12-09</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff43ee467527f28383e03b026cf9032e05af845" target='_blank'>
              TripLexicon: Prediction and analysis of gene regulatory RNA-DNA interactions.
              </a>
            </td>
          <td>
            T. Warwick, Christina Kalk, Ralf Peter Brandes, Marcel H Schulz
          </td>
          <td>2025-12-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D. Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, JangKeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Sequence divergence within gene regulatory elements has been proposed to play an important role in the evolution of human-specific traits, including cortical expansion. However, the mutational processes that efficiently modify gene regulatory elements and the target genes upon which they act are poorly understood. We investigated the regulatory function and origins of the fastest evolved regions in the human genome, termed Human Ancestor Quickly Evolved Regions (HAQERs), in their native genomic context during human cerebral cortex development. Results We identified 50 HAQERs with accessible chromatin in developing human cortex, largely arising from previously unconstrained ancestral sequences. To test the necessity of these HAQERs for gene regulation, we established an all-in-one CRISPRi lentiviral vector and linked 26 HAQERs to nearby target genes across cell types and Wnt pathway activation contexts. Rapid gains of CpGs distinguished HAQERs active during cortical development and displaying human-specific epigenomic marks. As a high density of CpG sites can drive formation of permissive chromatin, we identified 107 HAQERs with at least 17 human-specific CpG gains per kb, termed HAQER CpG Beacons. These HAQERs emerged via contributions from GC-biased gene conversion (gBGC) with evidence for selection preferentially fixing CpG sites. Notably, the CHL1 and DPP10 loci, both implicated in human neurological disorders, each harbored two gene-linked HAQER CpG Beacons. Conclusions Our findings reveal HAQER target genes and support a model where gBGC and natural selection jointly drive regulatory-altering CpG variants to fixation, forging regulatory innovations in the human cortex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e64bca00a8a0022a427effe630eac24ac36a2d" target='_blank'>
              Interrogating the Regulatory Function of HAQERs during Human Cortical Development
              </a>
            </td>
          <td>
            Yashodara Abeykoon, Natalie Dzikowski, Yanting Luo, Enakshi Sinniah, Seth Weaver, Bryan J. Pavlovic, Jenelle L. Wallace, Riley J. Mangan, Craig B. Lowe, Alex A. Pollen
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0957eb10bf767ffc74bc27558270080adbc791c6" target='_blank'>
              Pan-cancer analysis reveals mtDNA copy number as a key determinant of mutational load and disease progression in cancer
              </a>
            </td>
          <td>
            Anamika Acharyya, Riddhiman Dhar
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d0e12f17e165489871093f9d70ab93d1439628" target='_blank'>
              Dual DNA-binding capability of Cdc13 coordinates with Ku to safeguard telomere integrity
              </a>
            </td>
          <td>
            Zhitong Feng, Jiangchuan Shen, Yuxi Li, Giovanni Gonzalez-Gutierrez, Quan Wang, Hengyao Niu
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 For quite some time, knowledge about mitochondria and the mitochondrial genome has been primarily limited to energy production. However, there is now increasing evidence that they have many important roles in cell function and that synergy between the nuclear and mitochondrial genomes is an essential prerequisite to developmental outcome. This review describes the mitochondrial genome and its contribution to overall cellular genomic content; and discusses mitochondrial DNA (mtDNA) inheritance. mtDNA homoplasmy and heteroplasmy are defined and distinctions between pathogenic and non-pathogenic rearrangements are drawn; how they are transmitted; and their effects on oocyte quality and developmental outcomes. This is followed by analysis of mtDNA replication and changes in mtDNA copy number during development; why they need to happen; and how they influence developmental outcomes. Changes to nuclear DNA methylation events are then discussed in the context of changes to mtDNA replication throughout development. This leads to the concept of ‘genomic balance’, which defines how cells at any stage of development require adjustments to both genomes to ensure successful cellular function and development; and how this process can be perturbed by some of the more invasive assisted reproductive technologies designed to treat infertility and mtDNA disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902806be1c6e8a4fa2af943ea8c04c26b2c9f2b7" target='_blank'>
              The nuclear and mitochondrial genomes need to tango. How is their dance synchronised during development?
              </a>
            </td>
          <td>
            J. S. St. John
          </td>
          <td>2025-12-08</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most common (~80%) and lethal ovarian cancer subtype in the United States, characterized by TP53 mutations and DNA repair defects causing chromosomal instability (CIN). KIF18A is an essential cytoskeletal motor protein for cell division in CIN+ cancer cells, but it is not necessary for cell division in normal cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CIN+ cancers. We investigated the use of a novel KIF18A inhibitor ATX020, for selectively targeting CIN+ HGSOC cells using growth inhibition assays, invasion assays, immunoassays, cell cycle analysis, and immunofluorescence techniques. Using DepMap and flow cytometry, we classified a panel of HGSOC cell lines based on aneuploidy scores (AS) and ploidy levels and identified a correlation between these classifications and sensitivity against ATX020. ATX020 induced cytotoxicity through mitotic arrest and DNA damage, and reduced tumor growth in HGSOC with high aneuploidy scores (AS). Mechanistically, ATX020 blocks KIF18A’s plus-end movement on spindle fibers, increasing spindle length, resulting in chromosomal mis-segregation, aneuploidy, and DNA damage. Our findings suggest that ATX020 inhibits CIN+ HGSOC cells mainly by inducing mitotic arrest and DNA damage, disrupting KIF18A’s function crucial for mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f42a6d74edbdfa8c878ba6b61be09681cd9d76" target='_blank'>
              The KIF18A Inhibitor ATX020 Induces Mitotic Arrest and DNA Damage in Chromosomally Instable High-Grade Serous Ovarian Cancer Cells
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Tzu-Ting Huang, Maureen Lynes, Sanjoy Khan, Serena J Silver, Jung-min Lee
          </td>
          <td>2025-11-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ee418dd0e60cea320d764ace68e662b4dbd193" target='_blank'>
              Structural Analysis of Recombinant AAV Vector Genomes at Single-Molecule Resolution
              </a>
            </td>
          <td>
            David Rouleau, Dimpal Lata, Serena Dollive, Robert E. Bruccoleri, Laura Van Lieshout, Diane Golebiowski, Ify Iwuchukwu
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Extrachromosomal circular DNA (eccDNA) has emerged as a critical player in cancer biology. Accumulating evidence indicates that eccDNA promotes tumor progression by inducing genomic instability, amplifying oncogenes, and regulating gene expression. However, its biological function and mechanistic role in acute myeloid leukemia (AML) remain unexplored. In this study, we investigated the functional significance and regulatory mechanisms of eccDNA in AML, aiming to identify potential therapeutic targets.



 Bone marrow samples were collected from newly-diagnosed and chemo-resistant AML patients with informed consent. High-throughput eccDNA sequencing was performed to determine eccDNA length distribution. The presence of gene-containing eccDNAs was validated by PCR and fluorescence in situ hybridization (FISH), and synthetic eccDNA constructs were generated to investigate their functional roles in AML. In vivo, we evaluated the impact of eccDNA-encoded LIPIN1 (ecc-LIPIN1) on AML progression and chemoresistance, and developed a PEI/PC7A nanoparticle system for therapeutic intervention. Proteomic and transcriptomic analyses were performed to explore the downstream targets.



 High-throughput eccDNA sequencing revealed abundant eccDNAs (0.3-1 kb) across newly-diagnosed and chemo-resistant AML samples, mapped to diverse genomic regions. Analysis of the GEO dataset (GSE165430) identified genes upregulated in chemo-resistant AML patients. Integrating these data with eccDNA-derived genes highlighted ecc-LIPIN1 as a potential resistance driver.
 PCR and FISH confirmed the presence of eccDNA harboring LIPIN1 nucleotide fragments. Ecc-LIPIN1 levels were significantly elevated in AML samples, particularly in chemo-resistant cases, and associated with poor prognosis. A circular DNA containing LIPIN1 was synthesized based on sequencing data and transfected into AML cells, resulting in elevated LIPIN1 expression. Functional assays demonstrated that ecc-LIPIN1 promoted AML cell proliferation and chemoresistance.
 Given its metabolic role, metabolomic profiling assay revealed that ecc-LIPIN1 primarily regulated lipid metabolism. Fluorescence microscopy and flow cytometry confirmed that ecc-LIPIN1 enhanced lipid droplet formation, contributing to AML progression and chemo-resistance. Proteomic and transcriptomic analyses of ecc-LIPIN1-overexpressed AML cells identified DGAT1, a key enzyme in lipid metabolism, as a downstream effector. Silencing DGAT1 in ecc-LIPIN1-overexpressed AML cells markedly reduced lipid droplet accumulation, indicating that ecc-LIPIN1 drives lipid storage through DGAT1.
 Further analyses revealed that LIPIN1 also modulated endoplasmic reticulum (ER) stress pathways. In LIPIN1-overexpressed AML cells, ER-stress markers CHOP and ATF4 were markedly elevated. Silencing DGAT1 reversed ER-stress activation, suggesting that LIPIN1 promoted AML progression by regulating ER stress via DGAT1.
 To validate the oncogenic function of ecc-LIPIN1, we generated eccDNA constructs containing either wild-type or mutant LIPIN1 nucleotide sequences and transfected them into AML cells. The wild-type ecc-LIPIN1 markedly enhanced AML progression and chemo-resistance both in vitro and in vivo. To counteract this effect, we developed a PEI/PC7A/siLIPIN1 nanoparticle system. Treatment with these nanoparticles efficiently suppressed ecc-LIPIN1 expression, inhibited AML progression, and mitigated chemo-resistance.



 This study provides preclinical evidence that eccDNA plays a critical role in AML progression and chemoresistance by encoding LIPIN1. Mechanistically, ecc-LIPIN1 promotes lipid droplet formation and ER stress via DGAT1, ultimately driving chemoresistance. These findings identify ecc-LIPIN1 as a novel oncogenic factor in AML and highlight its potential as a therapeutic target. Future research into eccDNA biogenesis in AML may uncover new strategies for combating drug resistance and disease progression.



 Extrachromosomal circular DNA; Acute Myeloid Leukemia; drug resistance; lipid metabolism; LIPIN1.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cecd3734536748347f09719465be9ffb74475eb" target='_blank'>
              Eccdna-encoded LIPIN1 drives DGAT1-dependent lipid droplet biogenesis to promote Acute Myeloid Leukemia progression and chemoresistance
              </a>
            </td>
          <td>
            Wancheng Liu, Amin Zhang, Xiangling Xing, Daoxin Ma
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Central nervous system germ cell tumors (CNSGCTs) are the most enigmatic brain tumors. They consist of several histologically distinct types of tumors, which are generally sub-grouped into germinomas or non-germinomatous germ cell tumors (NGGCTs). While germinomas are characterized by alterations in the MAPK/PI3K-pathway genes and global hypomethylation, pathogenesis of NGGCTs is largely unknown. To investigate genetic and epigenetic mechanisms of CNSGCT development, 33 CNSGCTs including 7 germinomas and 26 NGGCTs of various histology collected through the Intracranial Germ Cell Tumor Genome Analysis Consortium and Japan Children’s Cancer Group, together with their matched normal DNA, were subjected to a long-read whole genome sequencing using the PromethION system from Oxford Nanopore Technologies. The mean reading depth was 39X for tumor specimen and 31x for blood DNA. The mean N50 value was 9,565 bp. The mean/median number of structural abnormalities was 16.6/16 for germinomas and 27.9/10.5 for NGGCTs, which was not statistically significantly different. A 2 kb deletion involving an exon of USP28 was observed in a single mixed-germ cell tumor case. Deletions involving PRKN were observed in 4 cases. The mean number of ectopic somatic insertions of the LINE-1 element was 1.8 per case. A single mixed GCT had 21 LINE-1 insertions, one of which was located within an intron of NTRK2. The mean of the whole genome DNA methylation level was 38.2% in germinomas and 75.9% in NGGCTs (p=0.0005). Germinomas mostly had biallelic DNA hypomethylation in the imprinted genes, consistent with their presumed primordial germ cell (PGC) origin. On the other hand, NGGCTs showed various levels of methylation in the imprinted genes, suggesting that they may also have originated from PGCs, however at different stages from that of germinomas. Taken together, our whole genome DNA methylation haplotyping provided a new clue to elucidate the pathogenesis of NGGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a0bfd993cd8d6ea3ee9b1101edf3e65daa2bfa" target='_blank'>
              EPCO-07. A WHOLE GENOME DNA METHYLATION ANALYSIS REVEALED DISTINCT MECHANISMS OF DEVELOPMENT IN DIVERSE TYPES OF CENTRAL NERVOUS SYSTEM GERM CELL TUMORS
              </a>
            </td>
          <td>
            Shoko Yoshimoto, Hirokazu Takami, Erina Ishikawa, Yuta Kuze, Takako Yoshioka, Junko Hirato, Nobuhito Saito, Toshihiro Kumabe, Ryo Nishikawa, Yutaka Suzuki, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="SINE variable number tandem repeat Alu elements (SVAs) are a unique group of hominid-specific composite retrotransposons with highly variable internal structure. They represent the youngest TE family in humans and contribute to genetic diversity, evolution, and disease. Recent findings indicate that SVA mobilization rates may exceed previous estimates, and many SVAs exhibit insertion polymorphism. SVAs facilitate transduction (TD) events when transcription initiates upstream of a source element, or when their internal termination signal is bypassed, mobilizing adjacent 5’ and/or 3’ sequence. To investigate features of non-reference SVA elements currently polymorphic in the human genome, we analyzed a structural variant callset built upon 35 diverse human genomes generated by the Human Genome Structural Variation Consortium. In our curated dataset of 543 polymorphic, non-reference SVAs, we identify insertions representing the three youngest subfamilies: D (7%), E (38%), and F (55%). Of the latter, we determine that at least 47% are actually SVA_F1, a more recently discovered human-specific subfamily, indicating that F1 is a major contributor to SVA expansion in the human population. We further uncover that 40% of non-reference SVAs carry a TD on their 5’ and/or 3’ ends. Of these, the majority (69%) harbor sequence originating in a gene, including 14 exonic events and the mobilization of a processed pseudogene, supporting the role of SVA in exon shuffling. In addition, we identified a so-called “orphan” TD, defined by the absence of SVA sequence at the insertion site. Leveraging TD origin coordinates, we identify 55 active source elements, including nine non-reference and 46 across GRCh38 and T2T-CHM13, giving rise to 84% of TD-carrying SVAs. Our analyses indicate that SVA_F1 is more active than previously described and is a main driver of SVA expansion. We find two-fold more TD events compared to previous estimates, with an unexpected bias toward 3’ events. Finally, we postulate that the discrepant SVA mobilization rate may be attributed to inter-individual variation in the presence/absence of source elements, a recent uptick in mobilization supported by overall low allele frequencies, and/or negative selection against deleterious insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c58ab3c46bd44059e8817d00743b60c2481618a6" target='_blank'>
              Structural and transduction patterns of human-specific polymorphic SVA insertions
              </a>
            </td>
          <td>
            Ashley E Kirby, Mark Loftus, Emily C. Golba, , H. Abel, Hufsah Ashraf, P. Audano, A. Basile, M. Bonder, H. Brand, M. Byrska-Bishop, Mark J. P. Chaisson, Junjie Chen, Yu Chen, Zechen Chong, Nelson Chuang, Wayne E. Clarke, A. Corvelo, S. Devine, Peter Ebert, Jana Ebler, U. S. Evani, S. Fairley, Paul Flicek, Mark B. Gerstein, M. Ghareghani, Ira M. Hall, William T. Harvey, Patrick Hasenfeld, Alex R Hastie, Wolfram Höps, Pinghsun Hsieh, Sushant Kumar, Joyce Lee, Alexandra P. Lewis, Chong Li, Yang I Li, Jiadong Lin, Tsung-Yu Lu, Rebecca Serra Mari, Ryan E. Mills, Zepeng Mu, Katherine M. Munson, David Porubsky, Benjamin Raeder, Tobias Rausch, Allison A. Regier, Jingwen Ren, Bernardo Rodriguez-Martin, A. Sanders, Martín Santamarina, Xinghua Shi, Oliver Stegle, Arvis Sulovari, M. Talkowski, Luke J. Tallon, J. Tubío, A. Wenger, Xiaofei Yang, Kai Ye, F. Yilmaz, Xuefang Zhao, Weichen Zhou, Qihui Zhu, Michael C. Zody, Jan O. Korbel, Tobias Marschall, Evan E. Eichler, Charles Lee, Miriam K. Konkel
          </td>
          <td>2025-11-06</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999aa4e6a47ef459adcf0a9e608c45b2e8af7576" target='_blank'>
              A human supergene: BRCA1, CCDC200, and U2 snRNAs
              </a>
            </td>
          <td>
            Pelle Scholten, Michael Segel, David Haig
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Natural chromosomal transformation (NCT) in Bacillus subtilis requires RecA and its accessory proteins including RecX and RecD2. Inactivation of the nucleoid-associated protein (NAP) Hbsu, Rok, or LrpC results in a dual effect: it enhances NCT but exacerbates the NCT defect of ΔrecX cells. Purified EbfC exhibits characteristic features of NAPs: it binds both single- and double-stranded DNA, protects them from degradation, and forms higher-order protein–DNA complexes via its DNA bridging activity. NCT is reduced upon EbfC inactivation but enhanced by ebfC overexpression. hbs55, Δrok, or ΔlrpC mutations suppress the NCT defect of ΔebfC, yet synergistically increase NCT upon ebfC overexpression. The NCT defect in ΔrecD2 cells is worsened by ebfC overexpression or by hbs55, Δrok, or ΔlrpC mutations. The nucleoid was more compacted in ΔebfC cells, an effect counteracted by hbs55, Δrok, or ΔlrpC mutations. EbfC contributes to DNA repair, and ebfC is epistatic to hbs or lrpC in response to DNA damage. We propose that chromosome folding, modulated by NAPs, plays a critical role in NCT and DNA repair. In this context, EbfC, by regulating nucleoid dynamics as a NAP, opposes the functions of Hbsu, Rok, and LrpC in NCT, while their interconnected roles contribute to DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab39c4a3f04975076ada74d7385c75dacff21" target='_blank'>
              Nucleoid structure and dynamics influence natural chromosomal transformation in Bacillus subtilis: the role of EbfC
              </a>
            </td>
          <td>
            Rubén Torres, María López-Sanz, Yuri Ushijima, Kazuya Morikawa, Juan C Alonso
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Introduction: During the past two decades, our knowledge about the molecular pathogenesis of myeloid neoplasms (MNs), including acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) has dramatically been improved through the identification of major driver alterations using next generation sequencing. However, with the lack of large-scale analysis using whole genome sequencing (WGS) with sufficient sequencing depths, the analyses have mainly focused on the alterations affecting protein-coding sequences, while the role of non-coding alterations and structural variations (SVs) in myeloid leukemogenesis has not fully been investigated. The etiology of mutagenic processes has also been poorly understood. In the current study, we therefore conducted large-scale deep WGS to address these issues.
 Methods: We performed deep WGS with matched normal samples (mean sequence depth 127.0/30.3 for tumor/normal, respectively) for a total of 903 patients with MN (494 AML and 419 MDS), with an additional 263 cases (94 AML and 169 MDS) currently under analysis. Most cases were also analyzed using deep targeted capture sequencing (n=886) of 446 known/putative driver genes. In addition, RNA sequencing was newly performed for nearly half of cases (n=406). Mutation calling was performed using Mutect2 and GRIDSS2, with in-house modifications to control false positivity in the face of frequent contaminations of tumor cells into germline samples.
 Results: We identified 1,159,638 SNVs and 76,842 sInDels with median mutation burden of 0.46 SNVs and 0.030 sInDels per Mb in AML and 0.39 and 0.022 per Mb in MDS. Validation by deep targeted capture sequencing estimated a true-positive rate of 95% and 78.1% sensitivity for known drivers. In the analysis of mutational signatures, we found that clock-like signatures SBS1 and SBS5 were predominant. SBS19 and SBS32 were detected in 62%/14% and 68%/14% of AML and MDS cases, respectively. As expected, chemotherapy related signatures (SBS25, 31 and 99) were significantly enriched in cases with therapy-related MN cases. The SBS18 signature, which is implicated in oxidative stress–induced mutagenesis, was highly enriched in AML cases carrying t(8;21) and inv(16), but rarely found in those with MDS, highlighting the role of oxidative stress in core binding factor leukemia. Other novel signatures such as SBS24, 39 and 40b were detected in a small number of cases (~1%).
 Our study of a large cohort of MNs also revealed novel candidate driver genes. By evaluating mutation enrichment, we identified a total of 82 candidates of driver genes. Among these, 15 genes were not previously reported as driver genes in MNs, including 7 genes listed in COSMIC cancer gene census database as associated with other cancer types. RNA-seq enabled the identification of intronic mutations predicted to cause alternative splicing in known driver genes, including TET2 and DNMT3A, as well as in novel candidates, observed in 2.1% of cases. In addition to coding genes, mutations in non-coding RNAs with potential driver role (e.g. seed region of mir-142) were detected in 3.3% of the cases.
 We identified a total of 11,050 SV events, of which 48 disrupted common tumor suppressor genes, such as RUNX1, ETV6, CBL, and TP53. Chromothripsis events were detected in 5.3% of cases, which were highly enriched in TP53-mutated cases and frequently affected KMT2A, ETS1, ETV6, TP53, EPOR and ERG. Median of 4 chromosomes were affected in a single chromothripsis event.
 We detected a total of 340 fusion events in 22.6% of all cases, of which 105 fusions involved previously known targets, whereas the remaining 235 represented non-recurrent in-frame gene fusions detected in 11.7% of cases. Although such fusion events were significantly associated with TP53 mutations, 5.6% of TP53-wild-type cases also had in-frame gene fusions, some of which were suggestive of oncogenic potential (e.g. KAT6A::NUTM1). Although low in frequency (1.1%), analysis of expression change revealed SVs affecting cis-regulatory elements including potential enhancer hijacking of a RAS pathway gene.
 Conclusion: Through deep WGS of a large cohort of MN cases, we have delineated a comprehensive landscape of driver mutations and detected new mutational signatures and candidates for driver mutations, including those affecting non-coding regions, underscoring the importance of WGS for better understanding of the pathogenesis of MNs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4576f85031bf75aa25ad786753882bdfc5ce2543" target='_blank'>
              Repertoire of driver events in coding and non-coding regions identified by whole-genome sequencing of myeloid neoplasms
              </a>
            </td>
          <td>
            Koji Okazaki, R. Saiki, Y. Ochi, Kenichi Yoshida, K. Katayama, Hidehito Fukushima, Kaito Mimura, M. Nakagawa, Masanori Motomura, A. Yoda, R. Okuda, Lanying Zhao, Motohiro Kato, Tatsuki Ogasawara, Shigeo Fuji, Lee-yung Shih, N. Uoshima, Mitsumasa Watanabe, Yasushi Miyazaki, M. Ichikawa, Takeshi Maeda, Toshiyuki Kitano, Satoshi Yoshihara, Kazunori Imada, Kinuko Mitani, J. Jansen, M. Sakata-Yanagimoto, Luca Malcovati, Akihiko Gotoh, Hitoshi Kiyoi, N. Kanemura, N. Sezaki, N. Hiramoto, S. Kasahara, S. Imoto, A. Takaori-Kondo, Y. Nannya, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA), the most abundantly expressed locus in the human genome, is represented by hundreds of units per cell. Each unit includes a 13 kb long region (47S rDNA) containing the genes of ribosomal particles, and a 30 kb long intergenic spacer (IGS). The 47S rDNA is transcribed with varying intensity in different units, some of which remain permanently silent. A key intermediator of this silencing is the promoter-associated RNA (pRNA) produced from a 2 kb long gene situated upstream of the rDNA transcription start site. Recent studies, including ours, suggest that the sequence variability, which normally occurs in mammalian cells, may account for the selective transcription of different rDNA units. The present work is based on the deep sequencing of a pRNA gene fragment and its RNA product and subsequent bioinformatic analysis. We found that a certain SNV, which converts the CCC motif into CCT, as well as deletions which reduce the number of (CCCT) tandem repeats, were significantly more frequent in the DNA than in the respective transcripts. These findings allowed us to establish directly the inhibitory effect of DNA variants on the expression of pRNA and thus (indirectly) the promoting effect on the production of ribosomal RNA. Our results also suggest that (CCCT)n/(GGGA)n DNA repeats and the respective (GGGA)n RNA repeats may form triplex structures, facilitating the function of pRNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891924cd9bf55653e2539c5b0034f6d9fcdd744" target='_blank'>
              Deletions in (CCCT)n repeat regions belonging to the human pRNA gene inhibit its expression
              </a>
            </td>
          <td>
            Nikola Chmúrčiaková, Adam Nógell, E. Smirnov, Dušan Cmarko
          </td>
          <td>2025-10-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We report a novel cell-free technology, ICED (Intra-Circular Expression and Display), for displaying an expressed protein on its encoding circular DNA. The recovered circular DNA, enriched by affinity-based screening of nascent protein, can be directly amplified using RCR, the reconstituted E. coli replication-cycle reaction. Unlike CIS display, which requires the replication initiator RepA and cis-acting elements including the oriR, the cognate binding site of RepA from the R1 plasmid, ICED does not depend on such a specific protein or DNA elements. The display is abolished by linearization of template DNA, inserting a transcription terminator, or treatment with RNaseA as well as puromycin. Its efficiency is enhanced by the addition of magnesium in the selection step. These suggest that the expressed protein remains anchored to the circular DNA via a transcription-translation (TX-TL) complex involving RNA polymerase, mRNA, and ribosome. This previously unrecognized linkage offers new insight into the mechanistic interface in TX-TL. Notably, the system is compatible not only with crude extracts but also with the reconstituted PURE system composed of E. coli or T7 RNA polymerase and purified translation factors. By directly reusing the selected RCR product for subsequent rounds, we achieved 108-fold enrichment by two rounds, surpassing the performance of conventional display platforms. ICED thus provides a more efficient and straightforward platform for cell-free display.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d54219d3deebc6475c9abe696e720bda4153d3c" target='_blank'>
              Cell-Free Transcription and Translation Tether Expressed Peptides to Their Encoding Circular DNA.
              </a>
            </td>
          <td>
            Kenta Ichinoe, Masayuki Su’etsugu
          </td>
          <td>2025-10-17</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Chromosomal loops are CTCF–cohesin-mediated 3D genomic structures in the mammalian nucleus. In addition to these CTCF–cohesin loops, other epigenomic features can also form long-range interactions. Polycomb-targeted loci, regulated by the Polycomb repressive complex and trithorax group during development, form long-range chromatin interactions independent of CTCF–cohesin and are demarcated by regions of low DNA methylation—referred to as DNA methylation canyons.
 We previously identified extremely long Polycomb loops occurring between DNA methylation Canyon demarked Polycomb-targeted loci, spanning distances of up to 60 Mb (Zhang et al, 2020). These loops are found exclusively in self-renewing cells, such as human hematopoietic stem cells and mouse embryonic stem cells.
 In hematopoietic malignancies, both DNA methylation and Polycomb binding are significantly altered. To explore this further, we conducted a pan-cancer survey of long Polycomb loops across 223 tumor samples, with a focus on hematopoietic malignancies—given that their likely cell-of-origin, the hematopoietic stem cell, exhibits strong long Polycomb loops. Our cohort included 32 acute myeloid leukemias (AMLs), 24 T-cell lymphoblastic leukemias, 61 pediatric brain tumors, 80 prostate cancers, and 26 colon cancers. We found that most cancers—including all prostate and colon cancers—lack long Polycomb loops. However, long Polycomb loops are retained in pediatric brain tumors and a subset of AMLs. Interestingly, strong long Polycomb loop interactions are observed in normal developing brain tissue and hematopoietic stem cell. Our data suggest that the presence of long Polycomb loops is likely inherited from the epigenomic state of the cell of origin, as they are also observed in normal pediatric brain tissue and hematopoietic stem cells.
 Loss of long Polycomb loops in primary cancer samples is accompanied by DNA hypermethylation and reduced Polycomb binding at loop anchor loci, and these disruptions are further exacerbated in cultured cell lines. Interestingly, in AML, many of these previously silenced loci—often mesodermal transcription factors—become activated and form de novo 3D interaction anchors. For example, we observed the formation of a new domain around the leukemogenic ZEB2 gene, driven by leukemia-specific HOXA9 binding at an upstream enhancer within the TEX41 locus.
 Notably, AML samples displayed a wide range of long Polycomb loop strength. While most AMLs lose these loops, approximately 12% (4 out of 33) retain strong long Polycomb loops comparable to hematopoietic stem cells. These cases recurrently harbor somatic mutations in CEBPA (2 out of 4) and STAG2 (2 out of 4). These genes are not directly linked to Polycomb or DNA methylation machinery, suggesting alternative mechanisms of loop maintenance. The CEBPA mutant AMLs also exhibited H3K27me3 spreading to non-Polycomb target loci across the genome.
 We therefore tested whether EZH2 inhibition could disrupt long Polycomb loops in these AMLs and impact disease maintenance. Indeed, EZH2 inhibition attenuated long Polycomb loops, reduced colony-forming capacity, and promoted differentiation in long Polycomb loop–retaining AMLs. Treated AML cells activated a macrophage differentiation program and showed downregulation of cell cycle and DNA replication genes. These findings suggest that AMLs retaining long Polycomb loops are dependent on this 3D chromatin architecture. This rare but strong sensitivity to EZH2 inhibition in AML indicates that long Polycomb loops could serve as an epigenomic biomarker for EZH2 or other Polycomb-targeted therapies in cancer rather than the current genomic marker such as EZH2 gain-of-function mutations and SWI/SNF loss-of-function mutations.
 Overall, long Polycomb loops are commonly lost during leukemogenesis due to epigenomic disruption. However, a subset of AMLs maintains these loops from the cell of origin and appears to rely on the Polycomb network to sustain self-renewal.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2c56d775b22a7b9218367e039106ed0000301e" target='_blank'>
              Pan-leukemia 3D genomic profile reveals extreme long polycomb loop in subset of CEBPA mutated AMLs sensitive to EZH2 inhibition
              </a>
            </td>
          <td>
            Sean Moran, M. Zanovello, Zhong Fan, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ab2b82b250c7fe0872534fbb6e35f7aaf3d36d" target='_blank'>
              Cell State Chaos Underpins the Evolution of SMARCA4-Deficient Dedifferentiated Endometrial Cancer
              </a>
            </td>
          <td>
            M. Brandon-Coatham, Julia Vassalakis, Hannah Plummer, Jasleen Kaur, Farzaneh Afzali, Zhihua Xu, Guihua Zhang, Jiahui Liu, Wei Wang, Einav Wajsbrot Renert, Tyler T. Cooper, Alan Dimitriev, Cierra Perron, Andrew Garven, Simatsidk Haregu Abebe, David M Berman, Amber L Simpson, G. Wong, Gilles A. Lajoie, Felix K.F. Kommos, Andreas von Diemlin, Sheela A. Abraham, David J.H.F. Knapp, Alan Underhill, M. Koebel, Cheng Han-Lee, Lynne-Marie Postovit
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Ubiquitin-specific protease 1 (USP1) is a critical regulator of DNA damage response (DDR) pathways, exerting its function via the deubiquitination of key DNA repair substrates, including FANCD2 and proliferating cell nuclear antigen (PCNA). These deubiquitination events are essential for the functional integrity of the Fanconi anemia (FA) pathway and translesion DNA synthesis (TLS), both of which are vital for resolving DNA interstrand crosslinks and replication-associated DNA lesions. DNA-damaging agents, a broad class of therapeutics exert their cytotoxic effects by inducing various forms of DNA lesions, including base modifications, single-strand breaks (SSBs), double-strand breaks (DSBs), and DNA crosslinks. These lesions disrupt DNA replication and transcription, triggering replication stress, genomic instability, and ultimately cell death if left unrepaired. However, cancer cells often activate DDR pathways that coordinate DNA repair, cell cycle checkpoints, and survival responses, which can confer resistance to DNA-damaging treatments by enhancing lesion resolution and maintaining genomic integrity. Our previous studies showed that USP1 inhibition impaired the resolution of DNA lesions, resulting in persistent activation of DNA damage markers such as γH2AX. This indicates that USP1 inhibition could deepens the genotoxic stress imposed by DNA-damaging agents and promotes apoptotic cell death by blocking effective DNA repair In this study, we investigated whether pharmacological inhibition of USP1 could enhance the cellular response to DNA-damaging agents by disrupting key DNA repair mechanisms. Human cancer cell lines were treated with our proprietary USP1 inhibitor, VRTX531, either alone or in combination with DNA-damaging agents. VRTX531 treatment alone led to the accumulation of monoubiquitinated FANCD2 and PCNA, confirming effective USP1 inhibition and disruption of FA and TLS pathway activity. Combined treatment with VRTX531 and DNA-damaging agents resulted in a synergistic increase in cytotoxicity, as reflected by significantly reduced cell viability relative to DNA damage alone. These findings underscore the broader therapeutic potential of USP1 inhibition in not only sensitizing cancer cells to DNA-damaging agents but also overcoming chemo-resistance driven by DDR pathway activation.



 Prashant K. Bhavar, Uday K. Surampudi, Partha Pratim. Sarma, Appaji Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar. USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06d19eca0105605db82afdf528e4bb45ce22555e" target='_blank'>
              Abstract A034: USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells
              </a>
            </td>
          <td>
            P. Bhavar, U. Surampudi, P. Sarma, A. Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Super-enhancers (SEs) are key DNA cis-regulatory elements that play a central role in regulating tissue/cell-specific gene expression, thereby maintaining cellular identity and function. SEdb 3.0 (http://www.licpathway.net/sedb) provides an extensively updated resource of SEs and their regulatory annotations across multiple species. The current version of SEdb now curates 3 478 186 SEs from 5387 H3K27ac ChIP-seq samples across four species. Compared to SEdb 2.0, it has achieved a two-fold expansion in human and mouse SE entries while newly incorporating Arabidopsis thaliana and maize data, significantly enhancing both the database's scale and its utility in plant research. Furthermore, abundant (epi)genomic features have been added, such as enhancer RNAs (eRNAs), binding sites of transcription co-factors (TcoFs), and chromatin regulators (CRs). The inclusion of eRNAs provides insights into SE transcriptional activity. Mapping TcoF binding sites highlights their roles in mediating enhancer-promoter looping and stabilizing transcriptional complexes at SEs. The integration of CRs uncovers how SEs are associated with histone modifications and chromatin remodeling, which are critical for maintaining an open chromatin state. Collectively, these annotations not only reveal the diverse mechanisms by which SEs exert regulatory functions but also enable more detailed investigations into their biological significance and functional roles. Meanwhile, existing annotations have been substantially expanded, such as an approximately five-fold increase in transcription factor (TF) ChIP-seq data, a 2.3-fold rise in TF motifs, and a roughly 1.8-fold growth in SE-associated eQTL-gene regulatory pairs. SEdb 3.0 introduces two advanced inference strategies for associating genes with SEs. Moreover, two newly developed analysis tools are provided in SEdb 3.0, including SE blast alignment analysis and SE-driven core TF enrichment analysis. In summary, SEdb 3.0 represents a significant upgrade over SEdb 2.0, with a substantial expansion in SE coverage across multiple species, alongside enhanced functional annotations encompassing SE upstream/downstream regulatory information, thereby offering a more comprehensive and user-friendly platform for exploring the biological roles of SEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c473d9211ee5f186ffb574ffbc44abe478a16ce8" target='_blank'>
              SEdb 3.0: a comprehensive super-enhancer database across multiple species.
              </a>
            </td>
          <td>
            Shuang Song, Liyuan Liu, Chenchen Feng, Liyuan Xie, Guorui Zhang, Yuexin Zhang, Yichen Gao, Mingxue Yin, Xiuyun Tang, Wenya Pei, Chao Song, Runping Liu, Chunquan Li
          </td>
          <td>2025-12-08</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The modulation of DNA replication dynamics has emerged as a key area of study in understanding genome stability and its perturbations in various physiological and pathological contexts. Replication fork rate is influenced by a variety of factors, including DNA repair pathways, origin firing, chromatin organization, transcription, and oncogenic signaling. This review highlights recent findings on the molecular mechanisms driving replication fork acceleration, focusing on scenarios such as PARP inhibition, oncogene activation, depletion of replication factors, and defects in Okazaki fragment processing. We discuss how reduced origin firing, R-loop resolution, and metabolic changes contribute to fork rate modulation, as well as the involvement of innate immune signaling, particularly through pathways such as cGAS-STING and ISG15. Special attention is given to consequences of accelerated replication forks for genome stability and their role in disease progression, particularly cancer. By unraveling the molecular mechanisms of fork acceleration, this Mini Review underscores its critical role in shaping genome integrity and cellular homeostasis, providing insights into future research directions and therapeutic strategies. A mini review explores molecular drivers of replication fork acceleration and their consequences for genome stability, innate immunity, and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edcef1b767fb22ecf30bb9d1778a6566d3ca870a" target='_blank'>
              The need for speed: drivers and consequences of accelerated replication forks
              </a>
            </td>
          <td>
            Dávid Lukáč, Katarína Chromá, Pavel Moudrý
          </td>
          <td>2025-11-06</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Bladder and urothelial carcinoma are marked by profound genomic diversity. Using a large, multi-institutional dataset, we performed comprehensive genomic profiling of 4631 tumor samples from 4050 individuals. A retrospective analysis of bladder and urothelial cancer was performed using the AACR Project GENIE database. Demographic associations, mutation frequencies, copy number changes, and survival correlations were analyzed with a p-value < 0.05. Frequent mutations were identified in TP53, TERT, KDM6A, KMT2D, ARID1A, and FGFR3. Mutation frequencies varied by sex and race, with specific alterations enriched in female and Asian patients. Distinct patterns of co-occurrence, including TP53 with RB1, and mutual exclusivity, including TP53 with FGFR3 or KDM6A, revealed distinct molecular subtypes. This study highlights the extensive heterogeneity of bladder cancer, and our findings emphasize the clinical importance of molecular stratification and support the need for further mechanistic and prospective studies to inform the development of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f60ee507571a09c3baf6058507fa0927d48e551b" target='_blank'>
              Genomic Characteristics of Bladder Cancer: An AACR Project GENIE Study
              </a>
            </td>
          <td>
            John Paul Braun, Kenneth A. D. Palattao, Elijah Torbenson, Beau Hsia, A. Tauseef
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa4f5ddeab086fa66a7de2d9fed3fa9999b778c" target='_blank'>
              Cryptic multicellularity in wild yeast brings selective advantages under stress
              </a>
            </td>
          <td>
            W. K. Sexton, K. Schmidt, Q. Dickinson, J. Childress, F. Rosenzweig, E. Kroll
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Lamin A/C (LMNA), a key component of the nuclear envelope, is essential for maintaining nuclear integrity and genome organization [W. Xie et al., Curr. Biol. 26, 2651-2658 (2016)]. While LMNA dysregulation has been implicated in genomic instability across cancer and aging, the underlying mechanisms remain poorly understood [S. Graziano et al., Nucleus 9, 258-275 (2018)]. Here, we define a mechanistic role for LMNA in preserving genome stability in small-cell lung cancer (SCLC), a malignancy marked by extreme genomic instability [N. Takahashi et al., Cancer Res. Commun. 2, 503-517 (2022)]. LMNA depletion promotes R-loop accumulation, transcription-replication conflicts, replication stress, DNA breaks, and micronuclei formation. Mechanistically, LMNA deficiency disrupts nuclear pore complex organization, specifically reducing phenylalanine-glycine (FG)-nucleoporin incorporation, resulting in impaired RNA export and nuclear retention of RNA. LMNA expression is repressed by EZH2 and reexpressed during SCLC differentiation from neuroendocrine (NE) to non-NE states, and low LMNA levels correlate with poor clinical outcomes. These findings establish LMNA as a key regulator of nuclear transport and genome integrity, linking nuclear architecture to SCLC progression and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1b936a6a58b416fb0e62e291846a3a4d6fb1c3" target='_blank'>
              Lamin A/C loss promotes R-loop-mediated genomic instability and poor survival in small-cell lung cancer.
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Ravi P Shuklah, Robin Sebastian, Umeshkumar M Vekariya, Shahbaz Ahmed, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yashuhiro Arakawa, Ukhyun Jo, Tomasz Skorski, Beverly A. Teicher, Roshan Shreshta, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Transposable elements (TEs) make up half of the human and mouse genome but remain understudied. Evidence is growing that TEs play important roles in immunity as a subset of TE enhancers regulate innate immunity genes. However, the cis-regulatory contribution of TEs across immune cell development is not well understood. Mining more than 300 chromatin accessibility datasets (ATAC-seq) across mouse hematopoietic cell lineages, we identified two subfamilies of rodent-specific retroviral LTRs, ORR1E and ORR1D2 (ODE), which are enriched in accessible chromatin across all immune cell types. Unsupervised clustering show that ∼2,900 ODEs stratify by cell type-specific accessibility. Each ODE cluster is enriched for a unique combination of sequence motifs for cell lineage-specifying TFs, such as PU.1, IRF8, and RORγ, which we validated with ChIP-seq. By correlating ODE accessibility with gene expression (RNA-seq), we predicted hundreds of ODE-gene interactions. We generated Micro-C (DNA-DNA contact) in CD8+ T cells and validated ∼120 ODE-gene interactions. Predicted target genes are more expressed in mouse than in human CD8+ T cells. Lastly, accessible ODEs have higher sequence conservation across rodents than other ODEs. Together, these data suggest that despite their common origin, ODE LTRs have subfunctionalized into cell type-specific enhancers across mouse hematopoiesis. We propose that ODE enhancers were co-opted during rodent evolution to reinforce cell type identity.



 F31AI183775



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c25d5af8adc02b84a26acb002fe252a06d958df" target='_blank'>
              Functionalization of transposable element-derived enhancers across mouse hematopoiesis 3835
              </a>
            </td>
          <td>
            Jason D. Chobirko, Elizabeth A. Fogarty, Cedric Feschotte, Andrew Grimson
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Eukaryotic DNA has been covalently modified by DNA methylation and folded into a three-dimensional conformation in the nucleus. While the functions of DNA methylation and chromosome organization have been widely discussed, respectively, the interplay between DNA methylation and chromosome organization remains less clear and needs to be further explained. In this review, we first discuss the cross-talk between DNA methylation and chromosome conformation, highlighting the complexity and importance of DNA methylation on chromosome organization. We also summarize the current methodologies that capture DNA methylation and chromosome organization individually or simultaneously in bulk and single cells. These mechanistic and methodological advancements facilitate broad interest in unveiling the interplay between DNA methylation and chromosome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1272377f23956ec52130bb31428bff7433f3f0" target='_blank'>
              Unraveling the interplay of DNA methylation and chromosome organization.
              </a>
            </td>
          <td>
            Yuhe Pei, Guoqiang Li
          </td>
          <td>2025-11-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171126bf14024769a7ffeadfaccab1f7a77e1513" target='_blank'>
              Mitochondrial genome microhomology-mediated editing by donor DNA delivery into mitochondria in human cells
              </a>
            </td>
          <td>
            Vadim V. Maximov, N. Shebanov, Natalia Nikitchina, Rachel Rapoport, Y. Maor, I. Tarassov, O. Pines, N. Entelis
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Sarcomas are a heterogeneous group of over 170 malignant tumours of mesenchymal origin. The poor prognosis highlights the need for novel therapeutic strategies. Preclinical modelling is essential, yet challenging, given that sarcomas differ substantially from carcinomas and resources are very limited. GEMMs allow for the precise modelling of recurrent sarcoma genetics. The Cre-loxP system offer spatial and temporal control over the activation of oncogenes or the loss of tumour suppressors, while the CRISPR-Cas9 system enables the rapid, simultaneous editing of key drivers such as Trp53, Nf1, Kras and Pten. These models reproduce key features of human sarcomas, including their histopathology, the initiation of tumours in specific lineages and sites, and tumour–immune interactions within immune-competent hosts. GEMMs have been used to investigate hypotheses about the cells of origin, to test radiotherapy and immunotherapy, and to compare fusion-driven sarcomas with those with a complex karyotype. Despite variability, GEMMs remain essential tools for investigating the mechanisms of initiation, progression, and response to therapy. GEMMs offer mechanistic fidelity, but their use is limited by factors such as breeding burden, variability in recombination, off-target effects of CRISPR, underrepresentation of genomic complexity and inconsistent metastasis. These weaknesses reduce their predictive value, particularly with regard to advanced disease and immunotherapy. Progress will require the integration of Cre-loxP with CRISPR-Cas9, the standardisation of induction and reporting, and a closer alignment with distinct sarcoma subtypes, in order to enhance translational relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a1f976c30385cd478ed53978a57d862bccd72d" target='_blank'>
              Genetically Modified Mouse Models for Sarcoma Research: A Comprehensive Review
              </a>
            </td>
          <td>
            Piotr Remiszewski, Eryk Siedlecki, M. Wełniak-Kamińska, M. Mikula, A. Czarnecka
          </td>
          <td>2025-10-22</td>
          <td>Current Oncology Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K. Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Introduction: Ribosomal DNA (rDNA) encodes ribosomal RNA (rRNA), which undergoes processing to become ribosomes. Although rDNA dysregulation has been observed in various pathologies, comprehensive epigenetic profiling in cancer has been limited due to its repetitive sequence. To overcome this challenge, a customized genome has been developed to map human and mouse rDNA using high-throughput sequencing data. Using this genome and more than 2,200 ChIP-seq datasets, Antony et al performed a systematic analysis to identify the transcription factors, e.g., C/EBP-A, that support rRNA production in hematopoietic cells. Our previous publications suggested that DNA cytosine modifications influence genomic binding of C/EBP. Accumulated studies, including our own, have reported aberrant DNA methylation and hydroxymethylation in myeloid malignancies. Yet, it remains unclear whether the leukemia-associated DNA methylation dysregulation extends to genomic regions encoding rRNA (rDNA), and how these epigenetic changes in rDNA affect downstream targets.
 Methods: To address these critical knowledge gaps, we employed innovative rDNA assembly methods, along with a DNA methylation analysis pipeline, to evaluate the whole-genome bisulfite sequencing (WGBS) data from 136 cancer patients covering 9 cancer types, aiming to uncover DNA methylation changes in rDNA in cancer. To further investigate unique patterns observed in hematological malignancies, we expanded our analysis to a dataset containing DNA methylation profiles from 292 AML patients. Additional AML datasets were analyzed to validate our initial findings with integrative methylation analysis methods. Furthermore, we analyzed published nanopore long-read data to independently confirm the methylation status of rDNA.
 Results: Our pan-cancer analysis revealed several patterns of rDNA methylation across cancer types, often at odds with non-rDNA methylation patterns. Intriguingly, we discovered that hematological malignancies exhibited distinct DNA methylation signatures in rDNA, with higher DNA methylation in the rDNA-coding region compared with other cancer types. This finding was validated in multiple independent datasets totaling 484 AML samples. Additionally, long-read nanopore sequencing verified these methylation patterns across the entire rDNA tandem array. Further characterization of this hypermethylation in AML patient samples revealed CpG loci that show highly conserved changes. A significant correlation was observed between hypermethylation at these loci and decreased time-to-relapse, suggesting that increased rDNA coding region methylation may serve as a biomarker. Interestingly, along with prognostic significance related to time-to-relapse, we revealed distinct patterns of rDNA methylation in patients treated with hypomethylating agents (HMA). To study potential mechanisms, we first validated published findings that many transcription factors exhibit consistent binding patterns on rDNA. To elucidate the connection between DNA methylation and transcription factor binding within rDNA, we first performed a motif analysis of the rDNA sequence, revealing 4000+ canonical binding motifs. We then used our large AML datasets to identify motifs with differential methylation between healthy bone marrow and AML patients, both at diagnosis and relapse. Across both datasets, we identified several potential transcription factors, e.g., CEBP families, for downstream analysis. We will use multiple leukemia cell lines to validate the correlation between DNA methylation and TF binding at rDNA. We will further use catalytically dead Cas9 (dCas9) mediated epigenome editing to probe the causal relationship to support our hypothesis that rDNA methylation can influence TF binding.
 Conclusions: Our data revealed distinct DNA methylation signatures in rDNA specific to myeloid leukemia. Using multiple published DNA methylation datasets from AML patients, we identified unique DNA methylation changes in rDNA, highlighting its potential diagnostic and prognostic value. At the molecular level, we observed differential DNA methylation levels within several TF binding motifs, which might alter TF binding and impact their transcriptional activity in rDNA, ultimately contributing to aberrant ribosomal biogenesis. Overall, our study provides a comprehensive analysis of previously neglected DNA methylation changes in rDNA in AML, uncovering novel epigenetic mechanisms with potential clinical relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba5a2d1360b7125af2d5a2084348209fa94e181" target='_blank'>
              Hypermethylation of the ribosomal DNA coding sequence and disordered transcription factor binding as hallmarks of acute myeloid leukemia
              </a>
            </td>
          <td>
            James Wengler, Y. Huang, Leng Han, Lei Guo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/267276dd26cb529d07f65beba245c3f404b9cf18" target='_blank'>
              Dysregulated mitochondrial homeostasis and DNA repair in the progression from colon adenoma to cancer.
              </a>
            </td>
          <td>
            Natalie Danesova, Josef Horak, Anna Valickova, Adrian Gil-Korilis, Jorge Ergui-Arbizu, Richard Palek, Jan Bruha, Miroslav Levy, Pavel Škrobánek, Jan Kral, J. Jungwirth, Jiří Neužil, Veronika Vymetálková, P. Vodicka, Sona Vodenkova, Kristyna Tomasova
          </td>
          <td>2025-11-22</td>
          <td>Molecular medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures comprising an RNA/DNA hybrid and a displaced single-stranded DNA. While transient R-loop formation is essential for various physiological processes, their persistent accumulation leads to genomic instability. Cancer cells exhibit elevated R-loop levels due to hypertranscription, replication stress, and impaired DNA repair pathways. In this review, we provide a comprehensive overview of the molecular machinery that resolves R-loops, including chromatin remodelers, transcriptional regulators, nucleases, and helicases. We also highlight the emerging roles of long noncoding RNAs (lncRNAs) in modulating R-loop dynamics and explore how these RNA-based mechanisms cooperate with canonical resolution pathways. Finally, we explore the potential of targeting R-loop regulatory networks as a novel therapeutic strategy in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7591e333b5ca8a5feff5d45b0c90d90e304373" target='_blank'>
              Regulation of R-Loop Dynamics by Proteins and Long Noncoding RNAs: An Emerging Paradigm for Cancer Treatment.
              </a>
            </td>
          <td>
            Miho M. Suzuki, Keiko Shinjo, Tatsunori Nishimura, Yutaka Kondo
          </td>
          <td>2025-11-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant childhood cerebellar tumor comprising molecularly and clinically distinct subgroups. Groups 3 and 4-MB originate from overlapping progenitor pools of the developing rhombic lip when neuronal differentiation hierarchies are subverted through somatic alterations. Mutations suspected to deregulate the activity of chromatin-modifying genes are pervasive in Group 3/4-MB. However, molecular consequences of these alterations remain largely undefined. We characterized the chromatin landscape of fifty-two primary MBs using chromatin immunoprecipitation followed by sequencing (ChIP-seq) for six histone modifications and performed multi-modal integration with sample-matched somatic lesions, genome-wide DNA methylation, and transcriptomic profiles. Analysis of differential chromatin states across MB subgroups revealed significant enrichment of the bivalent enhancer state (EnhBiv; marked by coincident H3K4me1 and H3K27me3) in Group 3/4-MB overlapping neurodevelopmental genes. Integrative bioinformatics coupled with CUT&RUN analysis revealed significant enrichment of KDM2B, a histone lysine demethylase overexpressed in Group 4-MB, in gene promoters marked by the EnhBiv state. CRISPR gene targeting or targeted protein degradation of KDM2B selectively suppressed growth of Group 3 and Group 4-MB models in vitro and in vivo, which was dependent on KDM2B DNA-binding domain but not demethylase activity. Acute and chronic KDM2B degradation impaired EZH2 recruitment through direct physical interactions and indirect KDM2B-mediated PRC1 activity, reducing H3K27me3 deposition and chromatin bivalency, while profoundly derepressing PRC2 targets involved in neuronal differentiation. Our comprehensive characterization of the MB chromatin landscape in a large cohort of primary tumors provides unprecedented insights into the epigenetic basis of MB subgroups, implicating novel mechanisms of tumorigenesis, and disclosing a unique subgroup-specific dependency conferred by KDM2B-mediated Polycomb activity that warrants consideration as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff433ac7a60b68bf5a2b5c1d8d57a287494ba1b0" target='_blank'>
              EPCO-22. DIVERGENT MEDULLOBLASTOMA CHROMATIN STATES DISCLOSE KDM2B AS A SELECTIVE DEPENDENCY
              </a>
            </td>
          <td>
            Ran Tao, Serap Erkek-Ozhan, Beisi Xu, Yurika Matsui, Priya Mittal, Kyle S Smith, Yiran Li, Qingsong Gao, Emily Darrow, Ruijie Xu, Nadhir Djekidel, Richa Bajpai, Rahul Kumar, Colleen Reilly, T. Soliman, Jennifer L. Hadley, Melissa Batts, Natarajan Bhanu, B. Gudenas, Leena Paul, Hong Lin, K. Lowe, L. Kutscher, Benjamin Garcia, S. Pruett-Miller, Xin Zhou, Brent A. Orr, G. Robinson, Gang Wu, Jan O. Korbel, Jamy C Peng, Stefan M. Pfister, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Numerous somatic mutations have been implicated in cancer. However, but the evolutionary dynamics that shape tumor initiation and progression remain incompletely understood. In particular, systematic quantification of how mutation rates and selective pressures change across tumor stages has not previously been achieved. Unlike mutation prevalence, measures of selection intensity directly estimate the survival and proliferative advantage conferred by each variant, providing a means to distinguish drivers of initiation from those of progression.We analyzed thousands of tumor genomes, exomes, and targeted gene sequencing datasets across multiple cancer types to evaluate how mutation rate and selection intensity change from organogenesis to primary tumor development and metastatic spread. We calculated cancer effect sizes—population-genetic scaled selection coefficients—for driver mutations and genes. Mutation spectra were broadly conserved across stages within cancer types, but underlying mutation rates generally escalated with progression. In contrast, selection intensity shifted in gene- and stage-specific patterns. For example, BRAF V600E exhibited orders-of-magnitude stronger selection in metastasis compared to primary melanoma, SPOP mutations played an early initiating role in prostate cancer by enhancing selection on cooperating genes, and RET alterations were strongly selected during metastatic progression in thyroid cancer.These findings reveal that tumor evolution is shaped not only by mutation rate but also by dynamic, stage-specific selective pressures. Systematic quantification of selection intensity provides insight into the gene mutations that are most critical at each stage of cancer development, informing precision strategies for therapeutic targeting matched to disease stage and progression.



 Moein Rajaei, Jeffrey P. Townsend. Variation in selection intensity and mutation rates during tumor evolution across cancer types [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66fe67f238d6a363986f92b91615617f3804af0e" target='_blank'>
              Abstract B039: Variation in selection intensity and mutation rates during tumor evolution across cancer types
              </a>
            </td>
          <td>
            Moein Rajaei, Jeffrey P. Townsend
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The mixed lineage leukemia (MLL) methyltransferase gene is a member of the trithorax group family of factors and plays a significant role in development and hematopoiesis. In a healthy context, MLL is a lysine methyltransferase that catalyzes trimethylation of histone H3 at lysine 4 (H3K4me3), facilitating a transcriptionally permissive environment at promoters. However, defects in MLL are associated with hematopoietic malignancies and leukemias. One class of MLL-related leukemias involves rearrangements of the MLL N-terminal region that manifest as in-frame partial tandem duplications (PTD) of MLL's DNA binding domains, resulting in an elongated protein with aberrant function, with the most common form of MLLPTD containing duplications of exons 2-6. MLLPTD appears in up to 10% of patients with myelodysplastic syndrome (MDS) or acute myeloid leukemia (AML) and has been identified as a strong predictor of adverse outcomes, on par with defects in p53. Despite its clinical importance, the MLLPTD has not been well characterized and its mechanistic functions remain poorly understood. We have characterized the MLLPTD at the transcript, protein, and genomic level. Moreover, using quantitative mass spectrometry, we show that MLLPTD cells have a lower overall MLL copy number per nucleus than MLL WT cells. Despite this low copy number, we observe that MLLPTD exhibits broader binding to chromatin than WT MLL N-term when measured by CUT&Tag. Interestingly, we also observe increased H3K4 trimethylation at genes bound by MLLPTD. We conclude that the increased number of MLLPTD DNA binding domains combined with the intrinsic disorder of large regions of the MLL protein create increased opportunities for MLL to interact with distal chromatin regions, resulting in greater methyltransferase activity at a smaller number of loci. We propose that this altered methylation landscape promotes an aberrant gene regulatory program that drives leukemogenesis in MLLPTD cells. We also hypothesized that MLLPTD facilitates the development of AML by disrupting normal gene expression through differential protein-protein interactions relative to wild-type MLL. We report the identification of a preferential MLLPTD interactor, KAT2A, that can be targeted for acute degradation. Pharmacological degradation of KAT2A has a distinctively deleterious effect on cells expressing MLLPTD. Furthermore, treating MLLPTD cells with a KAT2A degrader slows engraftment when these cells are transplanted into immunodeficient mice. We conclude that KAT2A plays a crucial role in promoting MLLPTD-associated leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41be0cb8707aab9a37bac7d8bb23b376b8f664d5" target='_blank'>
              Deciphering the mechanistic role of mixed lineage leukemia partial tandem duplications (MLLPTD) in MDS and AML
              </a>
            </td>
          <td>
            Roberta Dollinger, Audrey Kopp, Mehar Un Nissa, Omar Arias-Gaguancela, Jeff Ranish, Marjorie Brand
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The pathogenesis of koala retrovirus (KoRV) has been explored in various contexts, yet its role in tumorigenesis remains incompletely understood. Unlike acute transforming retroviruses, KoRV lacks a viral oncogene but may contribute to oncogenesis via indirect mechanisms. However, the relationship between KoRV and telomere length, as a potential indicator of telomerase activity, has not been examined. This study investigates the effect of KoRV infection on telomere length in 47 samples from Southern Australian koalas in a novel telomere length quantification method. Telomere lengths of 30 KoRV-negative samples were compared to those of 17 KoRV-positive samples using the Absolute Human Telomere Length Quantification qPCR kit (ScienCell Research Laboratories, California, USA). The telomere length in KoRV-infected WBCs was significantly longer than the uninfected ones (t = −2.059, p-value = 0.045). In line with this, telomere length correlated positively with proviral load (r = 0.421, p-value = 0.003), further linking viral burden to telomere elongation. Furthermore, the effect of age on telomere length differed by infection status (β = −5329.7, p-value = 0.0038); KoRV-positive individuals exhibited longer telomeres at a younger age but experienced more rapid telomere attrition over time compared to KoRV-negative individuals. These results suggest KoRV promotes telomerase elongation ability and modulates age-related telomere dynamics, potentially contributing to subsequent cellular immortality and oncogenesis. These pathways may overlap with other retroviruses, where telomerase dysregulation contributes to their oncogenic potential. This study provides new insights into KoRV pathogenesis and DNA quantification methodology, which could be valuable for future research by identifying predictive markers for tumour progression and potential therapeutic targets in affected koalas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f125ee840ff1c5667ff63226a6db37a09c90835b" target='_blank'>
              Infection-Induced Telomere Length Variation: Insights into Pathogenesis of Koala Retrovirus
              </a>
            </td>
          <td>
            Hiu Ming Cheung, Sze Wing Jamie Lin, Hanh Thi Hong Nguyen, T. Stephenson, N. Speight, F. Hemmatzadeh
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND Colorectal cancer (CRC) is the second leading cause of cancer-related death, largely due to limited treatment options in advanced stages. Genomic alterations in advanced CRC (aCRC) are complex and not fully characterized, with only 30% of patients benefiting from targeted therapies. AIM To investigate the molecular heterogeneity of primary aCRC in order to identify clinically relevant genomic alterations. METHODS We conducted a retrospective molecular analysis of 73 consecutive patients with histologically confirmed primary aCRC (stage pT4a-b). All molecular findings were correlated with available clinicopathological data. In addition, we performed survival analyses using publicly available datasets and tools. RESULTS Genetic abnormalities identified in primary tumors were most frequently mutations in tumor protein p53 (58% of cases), Kirsten rat sarcoma viral oncogene homolog (52%), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (25%), B-Raf kinase (11%) and fibroblast growth factor receptor 3 (8%), as well as R-spondin 3 (RSPO3) fusions (8%). Alterations in the tumor protein p53 and neuroblastoma RAS viral oncogene homolog genes were predominantly observed in tumors from the left colon, whereas B-Raf kinase mutations and RSPO3 fusions were more frequently detected in the right or transverse colon. We also show a strong association between the presence of RSPO3 rearrangements and patients with small tumors, normal carcinoembryonic antigen levels, and microsatellite stable tumors. Furthermore, aCRC patients with protein tyrosine phosphatase receptor type k::RSPO3 fusions exhibited a higher mortality rate. Elevated RSPO3 gene expression levels were also significantly correlated with poorer OS across two large, independent CRC cohorts. CONCLUSION This study identifies a relatively high incidence of RSPO3 rearrangements in aCRC and a strong association with clinical features. Furthermore, we find that RSPO3 fusions are associated with poorer OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b1b9374204622db5b697e06c7d6c2484792d29" target='_blank'>
              RSPO3 rearrangements in advanced colorectal cancer patients and their relationship with disease characteristics
              </a>
            </td>
          <td>
            Raquel Tur, M. Abad, Elena Filipovich, Maria Belen Rivas, Marta Rodriguez, Juan Carlos Montero, J. Sayagués
          </td>
          <td>2025-11-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="There is an urgent need to find targeted agents for T-cell acute lymphoblastic leukemia (T-ALL). NOTCH1 is the most frequently mutated oncogene in T-ALL, but clinical trials showed that pan-Notch inhibitors caused dose-limiting toxicities. Thus, we shifted our focus to ETS1, which is one of the transcription factors that most frequently co-bind Notch-occupied regulatory elements in the T-ALL context. To identify the most essential enhancers, we performed a genome-wide CRISPR interference screen of the strongest ETS1-dependent regulatory elements. The #1-ranked element is located in an intron of AHI1 that interacts with the MYB promoter and is amplified with MYB in ~8.5% of T-ALL patients. Using mouse models, we showed that this enhancer promotes self-renewal of hematopoietic stem cells and T-cell leukemogenesis, maintains early T-cell precursors, and restrains myeloid expansion with aging. We named this enhancer the hematopoietic stem cell MYB enhancer (H-Me). The H-Me shows limited activity and function in committed T-cell progenitors but is accessed during leukemogenesis. In one T-ALL context, ETS1 binds the ETS motif in the H-Me to recruit cBAF to promote chromatin accessibility and activation. ETS1 or cBAF degraders impaired H-Me function. Thus, we identified a targetable stem cell element that is co-opted for T-cell transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b0d046a416194ebc6608468d656972dd5460330" target='_blank'>
              The hematopoietic stem cell MYB enhancer is essential and recurrently amplified during T-cell leukemogenesis.
              </a>
            </td>
          <td>
            Carea Mullin, Karena Lin, Elizabeth Choe, Cher Sha, Zeel Shukla, Koral Campbell, Anna C. McCarter, Annie Wang, Jannaldo Nieves-Salva, Sarah Khan, Theresa M. Keeley, Shannon Liang, Qing Wang, A. Melnick, Pearl Evans, Alexander C. Monovich, Ashwin Iyer, R. Kodgule, Yamei Deng, Felipe da Veiga Leprevost, Kelly R. Barnett, P. Pölönen, R. Khoriaty, Daniel Savic, D. Teachey, C. Mullighan, Marcin Cieslik, Alexey I. Nesvizhskii, Linda C. Samuelson, Morgan Jones, Qing Li, Russell J H Ryan, M. Chiang
          </td>
          <td>2025-10-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, Mark R. Wilkinson, Lu Wang, J. Klco, L. Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome organization is reproducible and linked to gene expression, but the forces shaping it remain poorly understood. This hypothesis proposes that chromatin positioning is directed by a weak radial electric field generated by the nuclear membrane potential. Although classical models predict rapid charge screening, the confined and viscous nuclear interior, regulated by ion pumps, limits this process and allows a residual field to persist. This field biases the movement of charged macromolecules within the gel‐like nucleoplasm, similar to electrophoresis. DNA is uniformly negative, but chromatin charge varies. GC‐rich regions bind more nucleosomes and are less negative, drawing them inward with positively charged nuclear speckles, hubs of gene expression. Epigenetic modifications further modulate chromatin charge, producing a self‐organized, dynamic radial architecture that regulates transcription. This framework connects the noncoding genome to expression and helps explain variable disease penetrance. Its testable predictions open new avenues for deciphering the logic of genome regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5086d8da2de0a4459dfbcf50eef7981af59ce0" target='_blank'>
              The Electric Genome: A Mechanistic Hypothesis for Nuclear Architecture and Gene Expression Bias
              </a>
            </td>
          <td>
            Ifat Keydar
          </td>
          <td>2025-10-24</td>
          <td>BioEssays</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Polyomaviruses (PyVs) are small, circular double-stranded DNA viruses that establish lifelong persistent infections. PyV infections are typically asymptomatic, but they can cause severe disease particularly in immunosuppressed individuals, including BKPyV-associated nephropathy, and Merkel cell carcinoma (MCC) caused by Merkel cell polyomavirus (MCPyV). While most individuals are infected with PyVs during childhood, it is unclear how these viruses persist in the host and what cellular signals prompt their reactivation. The early regions of PyV genomes encode regulatory proteins that control viral replication and host cell transformation in the case of MCPyV, SV40 and murine polyomavirus (MuPyV). All PyV early regions encode Large and Small Tumor antigens (LT, ST), which are splice variants of a common transcript. PyV LTs bind and inhibit RB1 via a conserved LxCxE motif, deregulating E2F transcriptional activity and cell cycle control to promote viral replication and tumorigenesis. The existing paradigm in the field is during normal PyV infection, LT is expressed and drives the host cell into S phase. However, a recent study revealed that cells infected with BKPyV only express LT and replicate the viral genome when already in S phase, but the mechanisms activating LT expression during S phase are unknown. The Non-Coding Control Region (NCCR), found in all PyVs, contains the viral origin of replication and predicted transcription factor (TF) binding sites, yet only a handful have been experimentally validated and shown to regulate PyV early transcription. Here, we set out to identify host TFs that bind the MCPyV NCCR to regulate early transcription in both MCC cells and in an in vitro model of MCPyV infection. We identified E2F1-3 as critical regulators of MCPyV early transcription, which bind to the MCPyV NCCR via a consensus E2 site just upstream of LT/ST transcriptional start site. We discovered E2 sites in the NCCRs of several other polyomaviruses, including MuPyV but not BKPyV. Deletion of the E2 sites in PyV NCCRs downregulates early transcription in luciferase assays. Despite not containing a consensus E2 site, we detected weak E2F binding to the NCCRs of BKPyV and SV40, suggesting that E2Fs also contribute to transcription of their early genes. Overall, our findings challenge the existing paradigm of PyV infection, that LT expression activates E2F signaling via RB1 inhibition, and suggest that E2Fs must already be active in the PyV-infected host cell for LT/ST expression and viral replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38026f5b1f8b196429ba7f6a171e6ca72452e0dc" target='_blank'>
              E2F1-3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Njh Salisbury, S. Amonkar, A. Roman, DA Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2508a7d83b4a1f4cf3672318d8edc8fabc927a1" target='_blank'>
              Evidence for improved DNA repair in long-lived bowhead whale.
              </a>
            </td>
          <td>
            Denis Firsanov, Max Zacher, Xiao Tian, T. Sformo, Yang Zhao, Greg Tombline, J. Y. Lu, Zhizhong Zheng, Luigi Perelli, Enrico Gurreri, Li Zhang, Jing Guo, Anatoly Korotkov, Valentin Volobaev, S. A. Biashad, Zhihui Zhang, Johanna Heid, Alex Maslov, Shixiang Sun, Zhuoer Wu, Jonathan Gigas, Eric C Hillpot, John C. Martinez, Minseon Lee, Alyssa Williams, Abbey Gilman, Nicholas Hamilton, Ekaterina Strelkova, Ena Haseljic, Avnee Patel, M. Straight, Nalani Miller, J. Ablaeva, L. M. Tam, Chloé Couderc, M. Hoopmann, Robert L. Moritz, Shingo Fujii, Amandine Pelletier, Dan J Hayman, Hongrui Liu, Yuxuan Cai, Anthony K. L. Leung, Zhengdong Zhang, C. B. Nelson, Lisa M. Abegglen, Joshua D. Schiffman, V. Gladyshev, Carlo C. Maley, Mauro Modesti, G. Genovese, M. Simons, J. Vijg, A. Seluanov, Vera Gorbunova
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="PURPOSE
HPV integration (HPVint) is associated with carcinogenesis and tumor progression in HPV-associated cancers, including head and neck squamous cell carcinomas (HNSCC). While its impact on human DNA has been well characterized, its relationship with clinical outcomes remains unconfirmed.


EXPERIMENTAL DESIGN
We analyzed HPVint events from 261 HPV-associated HNSCC bulk and single-cell RNA-seq samples from five cohorts, including 62 from a new University of Michigan cohort, and DNA HPVint events from 102 HPV(+) HNSCC participants in two of the cohorts. We investigated the consequences of HPVint both with respect to human and HPV gene expression and clinical outcomes (recurrence and overall survival).


RESULTS
By leveraging this large meta-cohort of HNSCC, we first reveal an oncogenic gene network based on the recurrent HPV integration locations in the human genome and gene expression alterations, highlighting key recurrent and overexpressed genes including NR4A2, CD274, CCER1 and genes from the CAMK and KLF families. We then stratify HPVint-positive participants by risk using HPV RNA features, specifically spliced HPV-human fusion transcripts (E1* integration) and HPV gene expression ratios (HGER), showing that subsets of participants have worse clinical outcomes based on these two candidate biomarkers.


CONCLUSIONS
By focusing on RNA instead of DNA, we expand our understanding of the carcinogenic mechanisms of HPVint, in part addressing the conflicting findings of whether HPVint is associated with aggressive phenotypes and worse clinical consequences and provide potential biomarkers to advance precision oncology in HPV-associated HNSCC. Newly identified genes with recurrent integration events may serve as candidates for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/737e8ec9fc9860af7b63c3f955141de9c82f430b" target='_blank'>
              HPV integration in head and neck cancer: downstream splicing events and expression ratios linked with poor outcomes.
              </a>
            </td>
          <td>
            Shiting Li, Shaomiao Xia, Maria Lawas, Aishani Kulshreshtha, Bailey F. Garb, Sarah E. Soppe, Chamila Perera, Chen Li, Min Liu, Y. X. Lim, Tingting Qin, Joshua D. Welch, N. D’Silva, Laura S Rozek, Maureen A. Sartor
          </td>
          <td>2025-11-21</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Two rounds of whole-genome duplication (WGD) occurred about 500 million years ago and played a major role in the evolution of the vertebrate genomes. Human genes derived from WGD are called “ohnologs”. Ohnologs are involved in fundamental biological processes and significantly contributed to the complexity of the human gene regulatory network. Given the central role of miRNAs in gene regulation, we investigated the contribution of ohnolog miRNAs (ohno-miRNAs) to the human gene regulatory network. We identified intragenic ohno-miRNAs as having higher retention rates compared to intragenic Small Scale Duplicated (SSD) miRNAs. Ohno-miRNAs also show high sequence similarity, a stronger tendency to regulate common target genes and are typically more expressed compared to miRNAs unrelated to WGD events. Analyzing the role of ohno-miRNAs in the human gene regulatory network, we showed that ohno-miRNAs are statistically overrepresented in specific network motifs commonly associated with redundancy and complexity, highlighting their central role in gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c5d0c75cd069a45340fb97f72963794c6c56d0e" target='_blank'>
              Ohno-miRNAs: miRNA pairs derived from whole-genome duplication
              </a>
            </td>
          <td>
            Leonardo Agasso, Ivan Molineris, Michele Caselle
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efec579cbc7f789053fbde7e928477ced0b339a" target='_blank'>
              Ensilication preserves high-molecular weight native DNA for clinical long-read sequencing
              </a>
            </td>
          <td>
            Alexis Ferrasse, Rodrigo Mendez, J. Gorzynski, Chloe M Reuter, Jennefer N Carter, Undiagnosed Diseases Network, Jonathan A. Bernstein, Matthew T. Wheeler, James L. Banal, E. A. Ashley
          </td>
          <td>2025-11-03</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chromatin remodelers regulate DNA-nucleosome interactions, influencing chromatin structure, accessibility, and gene expression (Gourisankar, S., Nat Rev Genet, 2024). Their dysregulation is frequently observed in cancer and often represents a therapeutic vulnerability (Johnstone, S. E., Science, 2022). HELLS, a key chromatin remodeler, maintains genome stability and regulates gene expression in aggressive T-cell lymphomas, such as ALK-negative anaplastic large cell lymphoma (ALK- ALCL) (Fragliasso V, Leukemia 2020; Tameni A., Cell Death Dis. 2021). Previous studies have demonstrated that HELLS knockdown sensitizes ALCL cells to chemotherapy, supporting its potential as a therapeutic target (Tameni A., Nucleic Acids Res. 2024).
 This study aimed to define the transcriptional role of HELLS and evaluate its therapeutic relevance in ALCL. To achieve this, we analyzed formalin-fixed, paraffin-embedded biopsies from a cohort of 44 ALCL patients and performed an integrated in vitro multi-omics analysis (RNA-seq, ChIP-seq, ATAC-seq, and Connectivity Map), alongside functional assays.
 To understand how HELLS coordinates and supports transcription, we integrated HELLS Chromatin immunoprecipitation-sequencing (ChIP-seq) data from the TLBR-2 cell line with RNA-sequencing data from TLBR-2 HELLS-knockdown (KD) and control cells. Of the 729 genes significantly affected by HELLS KD, 467 (64%) were identified as direct HELLS targets based on ChIP-seq analysis. Gene-set enrichment analysis of these HELLS-direct genes (HDGs) revealed their association with diverse biological processes, including the JAK/STAT signaling pathway and chemokine- and cytokine-mediated inflammatory signaling. The clinical relevance of HDGs was confirmed by nCounter profiling in a cohort of 44 ALCL patients (15 ALK+, 29 ALK- ALCL).
 To assess the contribution of HELLS to HDG transcription, we investigated the distribution of RNA Polymerase II (RNAPII) via ChIP-seq in TLBR-2 HELLS-KD and control cells. RNAPII ChIP-seq data revealed RNAPII elongation defects in approximately 60% of HDGs and altered RNAPII occupancy in ~40% of HDGs following HELLS depletion. Gene ontology analysis of this latter group of HDGs highlighted an enrichment in immune-related pathways, including T-cell-mediated immunity, cytokine signaling, and JAK/STAT signaling. Furthermore, ATAC-seq and H3K4me3 profiling indicated that HELLS promotes chromatin accessibility and transcriptional activation at immune-related loci.
 To evaluate the therapeutic relevance of HELLS, we employed a drug repurposing approach based on the HDG signature. This led to the identification of 10 classes of synergistic pathways, with PI3K, JAK/STAT, and DNA-PK emerging as top-scoring synergistic targets. In several ALCL cell lines, HELLS depletion synergized with an IC20 dose of Ruxolitinib (a JAK inhibitor) or AZD7648 (a DNA-PK inhibitor), resulting in significant synthetic lethality compared to single-agent treatment or control cells.
 In conclusion, this study demonstrates that HELLS drives the expression of immune-related genes through chromatin remodeling in aggressive ALK- ALCL. Its inhibition uncovers combinatorial vulnerabilities, providing a rationale for dual-targeted therapies involving JAK/STAT or DNA-PK pathway inhibition.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088deae46f89f2511cdb635d4408d6da56d67fdf" target='_blank'>
              Targeting hells‑mediated chromatin accessibility to sensitize ALK‑negative anaplastic large cell lymphoma to JAK/STAT and DNA‑PK inhibition
              </a>
            </td>
          <td>
            Giulia Gambarelli, Selene Mallia, B. Donati, N. Puccio, Magda Zanelli, Emanuele Vitale, Federica Torricelli, V. Manicardi, E. Salviato, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Pan-cancer analyses focused on the immunological significance and therapeutic potential of microspherule protein 1 (MCRS1) remain unreported. This study aims to define the pan-cancer immunological significance and therapeutic potential of MCRS1, with focused mechanistic dissection of its epigenetic-driven roles in hepatocellular carcinoma (HCC) progression. Methods An integrated multi-omics approach was employed, encompassing bulk transcriptomics, single-cell RNA-sequencing (scRNA-seq), and functional validation. Data from public repositories including The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx), and the Human Protein Atlas (HPA) were analyzed using tools such as SangerBox, University of Alabama at Birmingham Cancer (UALCAN), Gene Expression Profiling Interactive Analysis (GEPIA), Kaplan-Meier plotter, and Tumor Immune Estimation Resource (TIMER). Expression patterns, prognostic significance, epigenetic regulation, interactions with immune infiltrates, and functional impact of MCRS1 on HCC malignant phenotypes were comprehensively assessed. Results MCRS1 was upregulated in 24 malignancies (including HCC) and correlated with advanced stage, poor differentiation, and reduced survival (P<0.001). Hypomethylation of the MCRS1 promoter drove its overexpression in HCC, strongly associating with tumor progression. MCRS1 recruited M2-polarized macrophages (Rho =0.423, P=1.90e−16) and myeloid dendritic cells (Rho =0.560, P=7.87e−30). Spatial mapping confirmed MCRS1+/CD68+ macrophage colocalization in tumor niches. MCRS1 knockdown suppressed HCC proliferation, migration, and invasion. scRNA-seq revealed MCRS1 enrichment in immunosuppressive clusters expressing VEGFA/TGFB1. Conclusions Our pan-cancer analysis identifies MCRS1 as a key node linking epigenetic dysregulation and immunosuppression in HCC. Its promoter hypomethylation-driven overexpression is associated with an M2 macrophage-polarized, immune-resistant niche. These findings suggest that targeting MCRS1 may represent a strategy to overcome resistance to current immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17f217c352c9e87ecdceb056f6440ba28dcb255" target='_blank'>
              MCRS1 is associated with immunosuppressive microenvironments in pan-cancer and promotes hepatocellular carcinoma malignant phenotypes
              </a>
            </td>
          <td>
            Qiang Wang, Yu-Lu Ye, Mengyu Zhang, Bo Li
          </td>
          <td>2025-11-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Herpesvirales are an ancient viral order that causes lifelong infections in species from mollusks to humans. They export their capsids from the nucleus to the cytoplasm by a noncanonical nuclear egress route that involves capsid budding at the inner nuclear membrane followed by fusion of this temporary envelope with the outer nuclear membrane. Here, using a whole-genome CRISPR screen, we identify ER protein CLCC1 as important for the fusion stage of nuclear egress in herpes simplex virus 1. We also find that the genomes of Herpesvirales that infect mollusks and fish encode CLCC1 genes acquired from host genomes by horizontal gene transfer. In uninfected cells, loss of CLCC1 causes a nuclear blebbing defect, suggesting a role in host nuclear export. We hypothesize that CLCC1 facilitates an ancient cellular membrane fusion mechanism that Herpesvirales have hijacked or co-opted for capsid export and propose a mechanistic model. Herpesvirales utilize a unique nuclear egress route for capsid export. Here the authors show that herpesviruses exploit a cellular membrane protein, once thought to transport chloride, to facilitate membrane fusion and egress from the nucleus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4fb4c70f2d2d09bbfa7bbd4ae09de6478c50bc" target='_blank'>
              ER protein CLCC1 promotes nuclear envelope fusion in herpesviral and host processes
              </a>
            </td>
          <td>
            Bing Dai, Adrian W. Sperl, Lucas Polack, Isabel Mejia, Haley Dame, Tien Huynh, Chloe Deveney, Nathalie Lavoie, Chanyoung Lee, John G Doench, M. Daugherty, E. Heldwein
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="One of the seminal discoveries from genetic studies of autism spectrum disorder and related neurodevelopmental disorders (NDD) has been that loss-of-function (LoF) mutations in many genes that impact chromatin and transcriptional regulation confer substantial liability to NDD. Haploinsufficiency of the epigenetic regulator POGZ represents one of the strongest such associations; however, little is known about the direct or indirect regulatory targets of POGZ, or the mechanisms by which loss of this chromatin modifier alters early neuronal development and synaptic functions. Here, we created an allelic series of CRISPR-engineered human induced pluripotent stem cell (hiPSC) clones harboring mono- and biallelic POGZ deletions. In hiPSC-derived neural stem cells (NSC) and Neurogenin 2-induced neurons (iN), POGZ LoF altered the expression of genes associated with critical cellular processes and neuronal functions, including synaptic and intracellular signaling and extracellular matrix organization. Our multiomics profiling also showed altered footprinting of critical transcription factors (e.g., activator protein 1 complexes) that were enriched at promoters of differentially expressed genes associated with synaptic function. To further interrogate the shared molecular changes in neuronal development associated with NDD and POGZ regulation, we compared our results to deletions of the transcription factor MEF2C and the sodium channel gene SCN2A that we generated in these same isogenic iN. These analyses revealed strong enrichment of extracellular matrix and intracellular signaling disruption associated with POGZ and MEF2C deletion, whereas POGZ and SCN2A haploinsufficiency exhibited shared transcriptional effects on gene modules enriched for NDD-associated genes with opposing regulatory effects. Notably, we also observed alterations to synaptic firing rate and neurite extension with biallelic deletions, but not heterozygous lines, suggesting subtle effects in neuronal development associated with haploinsufficiency. Overall, these shared molecular consequences suggest key points of convergence that connect epigenetic regulation to neuronal function in the etiology of neurodevelopmental pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed6138b4f1663b7339e409062cbd4f7a6fbbd84" target='_blank'>
              CRISPR-engineered deletion of POGZ alters transcription factor binding at promoters of genes involved in synaptic signaling
              </a>
            </td>
          <td>
            M. Moyses-Oliveira, Yating Liu, Serkan Erdin, Dadi Gao, Riya Bhavsar, Kiana Mohajeri, Kathryn O’Keefe, Philip M. Boone, Gabriela Xavier, Calwing Liao, Aiqun Li, Rachita Yadav, M. Salani, D. Lucente, Benjamin Currall, C. D. de Esch, D. J. Tai, D. Ruderfer, Kristen J. Brennand, James F. Gusella, M. Talkowski
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde6df8438fec9eca3014692df8a1930274045c8" target='_blank'>
              A pangenome framework uncovers the role of deletions in repeated evolution of cave-derived traits
              </a>
            </td>
          <td>
            Emma Y. Roback, X. Maggs, Edward S Ricemeyer, Adam Warlen, Rachel A. Carroll, Christine G Elsik, Alex C. Keene, Nicolas Rohner, S. McGaugh, Wes C. Warren
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is an aggressive form of primary brain tumor, characterized by high malignancy and genetic heterogeneity, and standard treatment for GBM has remained unchanged in the last decade. A feature of GBM that makes it particularly difficult to treat is its heterogeneity, in part created by glioma stem cells (GSCs) within the solid tumor. GSCs have high plasticity and the ability to self-renew and differentiate into various heterogeneous cancer cell populations. GSCs are kept in a ‘stem’- like state with unlimited self-replication without differentiation and maturation, which helps maintain tumor heterogeneity and growth. This state also prevents them from being targeted effectively by traditional therapeutics. Histone modifications keep late-glial genes inaccessible by transcription factors, promoting transcription of replication and early-glial genes, maintaining stem-ness. A major player in this process is the imitation switch (ISWI) family of ATP-dependent chromatin remodelers, which have been found to have widespread aberrant expression in cancers, including gliomas. The ISWI complex can change the spacing of histones on the chromatin, and is primarily involved in the repression of gene expression. It contains a catalytic subunits, SMARCA5, and our data support that inhibition of SMARCA5 reprograms GSCs to differentiate. RNA-Seq after genetic depletion and pharmacologic inhibition of SMARCA5 in GSCs leads to loss of expression of early glial lineage genes and increased expression of late glial genes. We have developed a first-in-class small molecule SMARCA5 inhibitor that passes the BBB. In pilot studies, SMARCA5 depletion through genetic and pharmacologic inhibition extended animal survival in GSC-derived xenograft models of GBM. ATAC-Seq and single-cell RNA-Seq of tumors further support that SMARCA5 inhibition facilitated transition from stem/progenitor to differentiated cell states. Additionally, the SMARCA5 pharmacological inhibitor was well tolerated in vivo, suggesting that targeting SMARCA5 may have a favorable therapeutic window and high clinical translation potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea3b1fa5c18282fa0b9ccc1bb0d300ed3d16cc6" target='_blank'>
              STEM-03. SMARCA5 AS A THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Molly Guthrie, Himanshu Dashora, Le Xiong, Meghana Budankayala, Hongyu Zhao, Hariti Shah, Julia Yoo, Tapas Behera, Dalya Khalife, Y. Saunthararajah, Liangqi Xie, Jennifer Yu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Potassium is the most abundant intracellular cation. Potassium flux across cell membranes is controlled by potassium channels. While cytoplasmic potassium has been extensively studied, very little is known about the function of potassium in the cell nucleus. Here, we discover that a nuclear potassium channel complex composed of KCNG1 and KCNB1 subunits is a regulator of nuclear potassium in medulloblastoma, the most common malignant pediatric brain tumour. Using genetically encoded potassium sensors, we show that KCNG1 regulates nuclear potassium level independently of cytoplasmic potassium level. KCNG1 knockdown alters the genomic architecture of medulloblastoma cells by stabilizing G-quadruplex DNA and inducing DNA damage. Ultimately, KCNG1 deficiency results in nuclear swelling, thereby compromising nuclear envelope integrity, promoting the formation of micronuclei, and inducing DNA spillage into the cytoplasm to activate the immune-stimulatory cGAS/STING pathway. Kcng1 knockout in a genetically engineered mouse model of medulloblastoma remodels the immune microenvironment, reduces tumour growth, and prolongs mouse survival. Altogether, we identified a potassium channel complex that regulates nuclear potassium to govern nuclear integrity, tumour immune microenvironment, and medulloblastoma growth. Our results demonstrate that targeting nuclear potassium can be leveraged to treat medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3156f37831fc433a976d1d47ab5180bff944abd" target='_blank'>
              DNAR-02. Nuclear potassium governs nuclear integrity and tumour growth in medulloblastoma
              </a>
            </td>
          <td>
            Jade Chan, Xi Huang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Immune tolerance mechanisms are shared in cancer and pregnancy. Through cross-analyzing single-cell RNA sequencing data from multiple human cancer types and the maternal-fetal interface, we found B7-H4 (VTCN1) is an onco-fetal immune tolerance checkpoint. We showed that genetic deficiency of B7-H4 resulted in immune activation and fetal resorption in allogeneic pregnancy models. Analogously, B7-H4 contributed to MPA/DMBA-induced breast cancer progression, accompanied by CD8+ T cell exhaustion. Female hormone screening revealed that progesterone stimulated B7-H4 expression in placental and breast cancer cells. Mechanistically, progesterone receptor (PR) bound to a newly identified -58 kb enhancer, thereby mediating B7-H4 transcription via the PR-P300-BRD4 axis. PR antagonist or BRD4 degrader potentiated immunotherapy in a murine B7-H4+ breast cancer model. Thus, our work unravels a mechanistic and biological connection of a female sex hormone (progesterone) to onco-fetal immune tolerance via B7-H4 and suggests the PR-P300-BRD4 axis is targetable for treating B7-H4+ cancer.



 Supported by the NIH/NCI R01 grants (CA217648, CA123088, CA099985, CA193136, and CA152470), and the NIH/NCI through the University of Michigan Rogel Cancer Center (CA46592) to W.Z., and by the UTC-Yale Endowment to L.C..



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2912531adf328b0a2781d2f4a13cd7722dc36929" target='_blank'>
              Progestogen-driven B7-H4 contributes to onco-fetal immune tolerance 2475
              </a>
            </td>
          <td>
            Jiali Yu, Yijian Yan, Shasha Li, Ilona Kryczek, Lieping Chen, Weiping Zou
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bfffe74b13f5a75dbf73aa9e042097d2141c09" target='_blank'>
              POLQ variants with aberrant DNA polymerase activity protect against UV-induced cell death
              </a>
            </td>
          <td>
            Steven E Weicksel, Corey Thomas, Ethan Hall, Kylie Davis, Chase Michalczik, Sreerupa Ray, Jamie B Towle-Weicksel
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic cancer marked by clonal expansion of plasma cells in the bone marrow. Although its genomic landscape has been extensively characterized, the transcriptional mechanisms that govern malignant progression and long-term tumor cell survival remain incompletely understood. We integrated single-cell RNA sequencing (scRNA-seq) data from healthy donors (HD), monoclonal gammopathy of undetermined significance (MGUS), smoldering multiple myeloma (SMM), and newly diagnosed multiple myeloma (NDMM) patients, retrieved from GEO datasets GSE124310 and GSE271107, to construct a comprehensive transcriptional landscape of plasma cell differentiation. Pseudotime and enrichment analyses identified PDIA4 as a terminal-state–associated gene. The prognostic significance of PDIA4 was validated using the multiple myeloma research foundation (MMRF) CoMMpass cohort. Functional analyses were performed in vitro and in vivo to validate the role of PDIA4 in MM cell survival and therapeutic response. Pseudotime trajectory analysis revealed progressive upregulation of genes involved in protein processing in the endoplasmic reticulum (ER), with PDIA4 identified as a top candidate in terminal-stage plasma cells. Survival analysis in the MMRF CoMMpass cohort further demonstrated that high PDIA4 expression correlated with poor overall survival. In RPMI-8226 cells, PDIA4 knockout activated the IRE1α/XBP1s branch of the unfolded protein response (UPR), impaired proliferation, and induced G1-phase arrest. PDIA4 depletion also sensitized cells to bortezomib. In vivo, sg-PDIA4 suppressed tumor growth in RPMI-8226 xenografts. PDIA4 is a key regulator of the unfolded protein response and MM cell survival. Targeting PDIA4 may enhance the efficacy of proteasome inhibitors and offers a potential strategy to overcome therapeutic resistance in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881fab1a35c430a7450d4d3fe7ea84a60cfe6f7c" target='_blank'>
              Single-cell transcriptomics identifies PDIA4 as a marker of progression and therapeutic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Wenjie Yu, Jiamin Zhan, Yanxia Wang, Xu Cao, Aoyang Yu, Yuexuan Rao, Can Huang, Xuan Zhang, Shunv Lu, Yanhua Liu, Chong Chen, Zhenyu Li
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Dysregulation of gene expression is an essential feature of most human cancers, including acute myeloid leukemia (AML). Despite distinct biology of different subtypes of AML, there is also increasing evidence for shared pathologic mechanisms. Varied genetic subsets of AML exhibit shared dependencies on specific transcription factors and gene expression regulatory mechanisms. In particular, MYB is aberrantly activated in most studied forms of AML, where it is required for blockade of differentiation and self-renewal of leukemia cells. MYB is rarely mutated and generally not over-expressed in most human myeloid leukemias, presenting a key question concerning mechanisms of its activation that should be targeted therapeutically.
 To elucidate potential mechanisms of MYB activation, we investigated post-translational modifications (PTMs) of endogenous MYB proteins purified from human AML cells. Using comprehensive bottom-up high-resolution mass spectrometry, we achieved 95% MYB peptide sequence coverage and identified 10 unique MYB amino acid residues containing phosphorylated, acetylated, methylated, or ubiquitin-like PTMs. This included phosphorylation of S11 and S12 in the N-terminal domain (NTD) proximal to the DNA-binding domain of MYB, as validated using phospho-specific MYB antibody, mutagenesis, and dephosphorylation experiments. Notably, S11/S12 phosphorylation was preferentially detected across multiple AML cell lines and patient specimens from diverse molecular subtypes as compared to healthy human blood progenitor and stem cells.
 We found that MYB is required for the growth and survival of leukemia cells, which can be rescued by the expression of wild-type MYB but not MYB mutants that cannot be phosphorylated at S11/S12 in its NTD. Using label-free quantitative mass spectrometry combined with affinity purifications, we defined specific protein co-factors that are selectively assembled with phosphorylated as compared to non-phosphorylated MYB in the nuclei of AML cells, as well as cells expressing MYB-PLEKHO1 and MYB-ZFAT fusion oncogenes from blastic plasmacytoid dendritic cell neoplasm (BPDCN) cells. We found that S11-phosphorylated MYB preferentially assembles with other hematopoietic transcription factors including LMO2, LYL1, JUN, and TCF12. A subset of transcription factors including ZEB2, SATB1, IKZF1, SPI1, and RUNX1, 2, and 3 were also found to be co-assembled with BPDCN MYB oncogene fusion proteins. Co-assembled MYB transcription factors were associated with cooperative chromatin complexes regulating genes controlling leukemia cell differentiation, growth and survival, as defined using gene expression, chromatin profiling, and functional genomic assays.
 In all, these studies identify DNA-proximal phosphorylation of MYB and MYB DNA-binding domain fusions as a critical regulatory mechanism required for MYB oncogenesis through cooperative organization of transcription factors. Modifications of DNA-proximal NTD leading to MYB enhanceosome assembly suggest a new target for therapeutic intervention of MYB-dependent transcriptional circuits in AML and other transcription factor-dependent malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97715039497852e85a982a7d2a4af32af301c7f5" target='_blank'>
              DNA-proximal MYB phosphorylation is required for oncogenic transcription in acute myeloid leukemia
              </a>
            </td>
          <td>
            Shuyuan Cheng, Katarzyna Kulej, Christopher A G Booth, Masahiro Uni, Nicole Cruz, Andrew Lane, R. Koche, A. Kentsis
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, L. Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="This review outlines the mechanisms and regulation of DNA replication in Escherichia coli within the oriC region, focusing on DnaA nucleotide state and stepwise occupation of high and lowaffinity boxes, origin DNA topology, and architectural proteins (IHF, Fis, SeqA). We outline helicase loading by the DnaBDnaC complex, the role of the clamp and Hda in controlling reinitiation, and how small RNAs such as DsrA remodel transcriptional programs that impinge on initiation fidelity through the regulation of proteins associated with the DNA replication process and inhibition of. We discuss coupling between replication and transcription, the contribution of DNA gyrase to the need for unwound DNA to replicate, and strategies cells use to avoid replicationtranscription conflicts. Finally, we highlight synthetic biology applications, including orthogonal replication systems in E. coli that decouple engineered DNA propagation from the host chromosome. These insights provide an integrated view of initiation control and suggest opportunities to engineer replication for biotechnology and genome stability research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1d8ddca5d996affebfe1d9eec679ba452b2314" target='_blank'>
              Genetic Regulatory Network of DNA Replication Initiation in E. coli and Its Biological Significance
              </a>
            </td>
          <td>
            Zenas Deng
          </td>
          <td>2025-10-28</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f741cfc314db9bdb828e55bb38f137b5fb40cdaa" target='_blank'>
              Plasmid copy number affects the DNA methylation-driven expression dynamics of the CfrBI restriction-modification system and impacts phage restriction
              </a>
            </td>
          <td>
            Johannes Gibhardt, Trung Duc Hoang, K. Severinov, Mikhail A. Khodorkovskii, N. Morozova
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte-rich malignant tumours with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte-rich tissues, and then performed genome-wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis-regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour-specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D-loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour-specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome-wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas.
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f7970eb0b2f0e2ae86ba00f6f0eaa976b226bb" target='_blank'>
              DAZAP1 promotes cancer progression and chemotherapy resistance by stabilizing PIN1 protein in gastric cancer.
              </a>
            </td>
          <td>
            Peiling Zhang, Lujuan Ma, Yitian Wei, Qian Peng, Hong Xiang, Xisheng Fang, Cheng-yin Weng, Yong Wu, Lin Lu
          </td>
          <td>2025-12-02</td>
          <td>Cell biology and toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/234e6b6b6481eb76db70b44e7b7d276eb9ea9a24" target='_blank'>
              Population-scale multiome immune cell atlas reveals complex disease drivers
              </a>
            </td>
          <td>
            M. Kanai, T. Delorey, J. Honkanen, R. S. Rodosthenous, J. Juvila, S. Murphy, I. Teixeira-Soldano, H. S. Hwang, J. Karjalainen, J. Halonen, G. Panagiotaropoulou, Y. Zhang, C. McCabe, , K. Nanki, T. Yoshida, K. Liu, M. Glean, N. Mehrotra, E. Finan, D. Chafamo, Y. Zhu, M. Arvas, S. Ruotsalainen, Z. Zheng, FinnGen, M. Reeve, M. Kurki, C. B. Porter, O. Ashenberg, W. Zhou, K. PitkaÌnen, J. Partanen, A. Palotie, D. B. Graham, M. J. Daly, R. J. Xavier
          </td>
          <td>2025-11-27</td>
          <td>None</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The onset of acquired resistance limits the efficacy of KRASG12C inhibitors in lung cancer patients. Experiments in preclinical model systems and retrospective analyses of patients treated with these inhibitors have both suggested that the activation of DNA repair pathways and G2/M kinases is common in lung cancers with acquired resistance to KRASG12C inhibitors. Here, we identified a shared vulnerability to the CDK12/13 selective inhibitor SR-4835 in several KRASG12C-mutant cell lines with acquired resistance to sotorasib. In pairs of KRASG12C-mutant parental and sotorasib-resistant cell lines, CDK12/13 inhibition suppressed DNA repair gene expression and induced mitotic arrest, which was exacerbated in sotorasib-resistant cells by failure to activate the G2/M checkpoint and suppression of an oncogenic isoform of TP73. Combined treatment with both sotorasib and SR-4835 yielded an additive response and suppressed the development of acquired resistance to either inhibitor. The inhibitor combination caused additive G1 and G2/M arrest, and the increased sensitivity of emerging drug-resistant cell populations to SR-4835 extended the duration of response over single-agent treatment in cell culture and in mouse models. These results support combining KRAS and CDK12/13 inhibitors to extend the duration of response in KRAS-mutant lung cancer. Patients with acquired resistance to sotorasib may benefit from follow-up monotherapy with CDK12/13 inhibitors, the first of which recently entered clinical trials. Targeting CDK12/13 thus offers a promising strategy to overcome or prevent resistance to KRAS inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5df2549b13618f9401530fb07b8c41d871bb9c6" target='_blank'>
              Targeting CDK12/13 Drives Mitotic Arrest to Overcome Resistance to KRASG12C Inhibitors.
              </a>
            </td>
          <td>
            Yaakov E. Stern, Pompom Ghosh, John Peroza, Hitendra S Solanki, Denis Imbody, Liznair Bridenstine, Thiyagamurthy Pandurangan, Sylvia M Frydman, Kathryn N Nickens, Hannah L Walker-Mimms, Kruthi Suvarna, Neelkamal Chaudhary, Andrii Monastyrskyi, Derek Duckett, E. Haura
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eeb5a1876276ca40bae5f7f5ce7e3154844650e9" target='_blank'>
              Regulation of nucleolar dominance through allopolyploidization of hexaploid wheat
              </a>
            </td>
          <td>
            Yohta Hyuga, Tohru Nakano, Hirokazu Handa
          </td>
          <td>2025-11-06</td>
          <td>Theoretical and Applied Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-10</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Transcription and replication both utilize genomic DNA as a template, creating the potential for frequent interference between the two machineries. Such transcription–replication conflicts (TRCs) can compromise genome stability due to the abnormal accumulation and persistence of R-loops—three-stranded nucleic acid structures comprising a DNA:RNA hybrid and a displaced single-stranded DNA strand (Bhowmick R., et al., Mol Cell. 2023). To safeguard genome integrity, cells deploy multiple R-loop resolution factors that prevent R-loop-associated replication stress (Brickner J. R., et al., Mol Cell. 2022). Persistent R-loops are also associated with the stalling of RNA polymerase II (RNAPII) on chromatin, which further impedes replication fork progression (Zardoni L., et al., NAR 2021).
 Our observation of cells depleted of the chromatin remodelerHELLS, which displayed an accumulation of R-loops and stalled RNAPII on highly transcribed genes (Tameni A., Mallia S., et al., NAR 2024) in aggressive anaplastic large cell lymphoma (ALCL), led us to hypothesize that HELLS-dependent R-loop accumulation can induce replication stress and TRCs. To test replication forks progression, we performed DNA fiber analysis by sequentially labeling the cells with two thymidine analogs, 5-chloro-2′-deoxyuridine (CldU, red) and 5-iodo-2′-deoxyuridine (IdU, green). While control cells showed a CIdU/ldU ratio close to 1, indicating the absence of stalled forks, TLBR-2 and MAC2A HELLS-KD cells displayed an CIdU/ldU ratio close to 2, strongly indicating the presence of stalled forks in absence of HELLS. Then, we performed proximity-ligation assays (PLA) for measuring the presence of TRCs. Specifically, we used antibodies against phospho-forms of RNAPII, pSer5-RNAPII and pSer2-RNAPII, corresponding to initiation and elongation transcriptional complexes respectively and PCNA, as key component of DNA replicative machinery. We found that PLA signal between PCNA and pSer5-RNAPII, but not pSer2 RNAPII, increased in HELLS-KD cells compared to control, suggesting that the depletion of HELLS increases the collision between RNAPII and replicative complex at promoter-proximal sites before pause release. Importantly, ectopic expression of RNaseH1 in HELLS-KD cells reduced R-loop-dependent TRCs, confirming the functional involvement of R-loops in this process.
 Despite the large number of genes demonstrated or proposed to regulate R-loop homeostasis and, therefore, to impact TRCs and in turn on genome integrity, our knowledge of specific HELLS-regulated factors is still limited. To explore the mechanism contributing to TRCs and thus to genome instability, we performed gene set enrichment analysis focusing on genes belonging to functional categories related to chromatin regulation previously identified as HELLS transcriptional direct genes. Among them, mini chromosome maintenance 5 (MCM5) emerged as a potential candidate. Inducible knockdown of MCM5 significantly reduced cell proliferation, increased TRCs levels in lymphoma cells, as measured by pSer5-RNAPII–PCNA PLA foci, led to forks stall and to enhanced nuclear R-loop accumulation, phenocopying HELLS depletion. Notably, the ectopic overexpression of MCM5 in TLBR-2 HELLS-KDcells fully rescues cell proliferation reducing R-loop formation and the presence of TRCs. Additionally, perturbations observed upon MCM5-KD were dramatically reduced when R-loops were suppressed by overexpression of RNaseH1 indicating that, in absence of MCM5, the persistence of R-loop disrupts the replication fork process through TRCs, mimicking HELLS. These data highligh MCM5 as the downstream target of HELLS in replicative processes.
 Integrative analysis of MCM5, and RNAPII chromatin occupancy (ChIP-seq), alongside chromatin accessibility profiles (ATAC-seq) in TLBR-2 cells, further revealed that MCM5 was excluded from accessible, actively transcribed loci, confirming the not-transcriptional role of MCM5.
 Collectively, these findings position the axis HELLS/MCM5 as a critical regulator of replicative processes in aggressive ALCL. Disruption of HELLS/MCM5 activity induces aberrant TRCs and R-loop accumulation, ultimately compromising genome integrity and highlighting a potential therapeutic vulnerability in lymphoma cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caa8a5ddfc9df7b3d2ed0ec9481cd381c40701c0" target='_blank'>
              The chromatin remodeler hells regulates transcription-replication conflicts and genome integrity in ALK-negative anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Selene Mallia, Giulia Gambarelli, V. Manicardi, B. Donati, Emanuele Vitale, Federica Torricelli, E. Salviato, Magda Zanelli, N. Puccio, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769ab003240fead51b76ade414628f11f7033a1a" target='_blank'>
              Dedifferentiation-Driven Oncogenic Stemness Promotes Tumor-Sustaining Adaptability in the Intestinal Epithelium
              </a>
            </td>
          <td>
            Kylee Zgeib, Thomson Hui, Simon Garcia, Zahra Hashemi, Shima Nejati, Crystal Lim, Dahlia Matouba, Atharva Inamdar, Christina Li, Hossein Khiabanian, Binfeng Lu, Ansu Perekatt
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, Gavin Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, G. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d6b8d33286dfd47e06aa024a1a7a5fb5466454" target='_blank'>
              Deciphering APOBEC1 in Avians: Unravelling loss events and functional insights
              </a>
            </td>
          <td>
            Aswin S Soman, N. Vijay
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea9805a426118d493f4c3e4084ab4ad0614ebdb" target='_blank'>
              DNA Repair in Ensuring Genome Stability and Human Health
              </a>
            </td>
          <td>
            N. Rechkunova, O. I. Lavrik
          </td>
          <td>2025-11-01</td>
          <td>Russian Journal of Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Nucleophosmin mutations are among the most common aberrations in acute myeloid leukemia (AML). We previously uncovered a novel pivotal role for mutant NPM1 (NPM1c) in the nucleus, where it binds chromatin at key self-renewal gene loci, such as HOXA/B and MEIS1, directly regulating oncogenic transcription. How NPM1c regulates the transcriptional activity of its chromatin targets remains unclear. To identify potential nuclear cooperating partners of NPM1c, we employed TurboID proximity labeling of NPM1c and wild-type NPM1 (NPM1wt). Mass spectrometric analysis revealed an enrichment of nucleoporins in the NPM1c interactome, with Nucleoporin 98 (NUP98) emerging as a top candidate.
 NUP98, a key component of the nuclear pore complex (NPC), has been established as a critical oncogenic player in AML as part of oncogenic fusion proteins. In these oncofusions the N-terminal FG (phenylalanine-glycine) repeat domain of NUP98 is preserved and fused to a diverse set of partner genes. NUP98-fusion partners are believed to mediate the chromatin binding specificity of the oncoprotein while the NUP98 FG repeats have been shown to form biomolecular condensates that enhance transcriptional output. Interestingly, wild-type NUP98 (NUP98wt) itself has been linked to gene regulatory functions during embryonic development and normal hematopoiesis. However, the role of NUP98wt in transcriptional regulation in the context of leukemia remains unresolved.
 To explore a potential cooperation between NUP98wt and NPM1c we first validated their interaction by co-immunoprecipitation assays. We next wanted to determine whether NUP98 and NPM1c are recruited to the same chromatin sites. To this end, we performed ChIP-seq in an NPM1c endogenous degrader cell line. These experiments revealed co-occupancy of NUP98 at NPM1c target genes and a significant reduction of NUP98 binding upon degradation of NPM1c. These findings suggest that NPM1c is required for the recruitment of NUP98 to key oncogenic target loci.
 To dissect the functional role of NUP98wt in NPM1c AML, we performed loss of function studies. CRISPR mediated knock-out of NUP98 in NPM1 mutant (OCI-AML3) cells resulted in a significant repression of NPM1c target genes, including HOXA/B and MEIS1. Given the essential nature of NUP98, we also engineered an endogenous NUP98 degrader system in OCI-AML3 and NPM1wt (MV4-11) cells to allow for the study of the effects of rapid protein depletion. Upon degradation of NUP98, OCI-AML3 cells exhibited robust myeloid differentiation (~70% increase in CD11b) and increased apoptosis (~50% increase in Annexin V+). MV411 cells on the other hand did not show the same differentiation phenotype. To characterize the immediate transcriptional consequences of NUP98 depletion, we performed nascent RNAseq after 1 and 3 hours of degradation. In the first hours of degradation we observed a rapid and significant repression of NPM1c target genes such as HOXA/B cluster genes, MEIS1, SMC4, TNRC18, SATB1/2 and UNCX, thus demonstrating the strong dependency of the NPM1c-driven transcriptional program on the presence of NUP98. In contrast, in MV4-11 cells, NUP98 loss had no effect on HOXA/MEIS1 expression, suggesting a leukemia subtype-specific role of NUP98wt.
 To delineate the domains of NUP98 required for its chromatin function, we performed a CRISPR gene tiling screen of full-length NUP98 (635 sgRNAs) and based on this we generated several domain-deletion mutants for rescue experiments in the NUP98-degrader cell lines. Retroviral overexpression of the N-terminal FG-repeat containing domains of NUP98 was sufficient to rescue cell viability and prevent the differentiation phenotype typically observed upon NUP98 degradation. Strikingly, gene expression analysis showed that the NUP98 FG-repeats were sufficient to restore the expression of HOXA10, MEIS1, and HOXB9 expression.
 Overall, our study identifies NUP98wt as a critical cofactor for NPM1c-driven oncogenic transcription. NPM1c recruits NUP98 to its chromatin targets, functionally substituting for an oncogenic fusion partner to concentrate transcriptional condensates at oncogenic loci. This establishes a previously unrecognized, subtype-specific dependency on NUP98wt in leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff1e4b1ddc4587a7bab297c9d60234d4ef12e51" target='_blank'>
              Wildtype NUP98 cooperates with NPM1c to drive oncogenic transcription
              </a>
            </td>
          <td>
            Sara Gama, Marcel Müller, H. González Dammers, Elena Haarer, Sara Peruzzi, José Gonçalves-Dias, J. Klusmann, Lu Yang, Renee Chen, Chun-Wei Chen, Stamatis Papathanasiou, B. Falini, Lorenzo Brunetti, Hannah J Uckelmann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="


 Mosaic loss of chromosome Y (LOY) is a highly prevalent somatic event in many cancers (~30% of male tumors) and is frequently observed in males with age-associated clonal hematopoiesis of indeterminate potential (CHIP; ~50% by age 70-80 years). However, the mechanistic role of LOY in cancer remains poorly understood. Some studies suggest LOY is, in part, a manifestation of inherent genomic instability and a marker of biological aging, but mounting evidence supports an alternative hypothesis — that LOY is itself a cancer driver.
 We analyzed >1500 male patients with acute myeloid leukemia (AML) and found that ~10% harbored somatic LOY. Notably, LOY was significantly enriched (p<0.01) in specific subtypes — most strikingly in 54% of male patients with RUNX1::RUNX1T1 (AML1-ETO) AML — including pediatric cases. These data suggest that LOY may actively contribute to the pathogenesis of certain AML subtypes rather than being a mere passenger event or age-related artifact. Supporting clinical relevance, Schlenk et al. (2004) reported that LOY in AML1-ETO AML predicts shorter remission and inferior overall survival.
 Several candidate Y-encoded genes — including UTY/KDM6C, KDM5D, CSF2RA, TMSB4Y, and CRLF2 — have been implicated as drivers of LOY's oncogenic effects across different malignancies. However, loss of multiple genes, rather than any single gene, may underlie the phenotypic consequences of LOY, highlighting the need for robust, lineage-specific, and otherwise isogenic models to study pathogenic mechanisms.
 To investigate this, we immortalized primary murine hematopoietic progenitors in an ex vivo culture system using a retrovirally-expressed estrogen receptor-HoxB8 fusion protein, which maintains hematopoietic cells as immature, multipotent progenitors in the presence of estradiol and allows synchronized differentiation upon estradiol removal. Using two CRISPR/Cas9 strategies targeting ChrY repeat elements, we generated isogenic progenitors with or without ChrY deletion, confirmed by qRT-PCR and DNA PCR. In short-term culture of undifferentiated myeloid progenitors, LOY had no significant effects on growth kinetics, confirmed by flow cytometry cell tracing and cell cycle analysis. However, in longer-term competitive co-culture assays, LOY cells showed significantly reduced growth versus wildtype ChrY (WTY) cells, evident by day 7. Similar paradoxical reduced fitness has been seen previously with single gene models of clonal hematopoiesis-associated mutations. Transcriptomic profiling revealed 195 upregulated and 198 downregulated genes in LOY versus WTY cells (padj<0.05). Downregulated ChrY-encoded genes including Uty, Ddx3y, and Kdm5d validated LOY, and pathway enrichment of non-ChrY genes revealed significant downregulation (p=0.008) of oxidative phosphorylation genes in LOY cells, suggesting metabolic reprogramming.
 Following estradiol withdrawal, which inactivates the HoxB8 stem cell transcription factor and induces terminal differentiation, both WTY and LOY Hoxb8 cells lost the progenitor marker Cd117 (c-Kit) and upregulated mature myeloid markers Gr-1 and Cd11b within four days. However, LOY cells consistently showed accelerated myeloid maturation, confirmed by flow cytometry and morphology.
 In contrast to cells with LOY alone, in cells co-expressing the oncogene AML-ETO9a (AE9a), LOY significantly impaired myeloid differentiation compared to WTY. RNA-seq of AE9a cells with and without LOY revealed 638 up- and 540 down-regulated genes (padj<0.05), with significant reprogramming of translation, stress response, and biosynthetic pathways. Functionally via in vitrotransformation assays, both AE9a WTY and LOY cells proliferated and survived two-fold longer than empty vector controls (6 vs 12 days); however, only AE9a LOY cells sustained long-term viability beyond 21 days, while AE9a WTY cells progressively died off. In methylcellulose colony formation assays, AE9a LOY cells were the only genotype that produced large rapidly growing colonies (>200 cells), while other groups (AE9a WTY, and LOY or WTY without AE9a) formed only microcolonies (<20 cells).Together, our findings suggest that LOY impedes myeloid differentiation and enhances self-renewal in concert with specific leukemia oncogenes, revealing a context-dependent effect to promote clonal expansion and leukemic progression. These results challenge the paradigm that LOY is merely a bystander in leukemia and support further mechanistic study of its role in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c7b5fd6dd10d7134f1746ffc1fb097a24dfdd3" target='_blank'>
              Loss of the Y chromosome drives context-dependent transcriptional reprogramming in myeloid progenitors and AML1-ETO acute myeloid leukemia (AML)
              </a>
            </td>
          <td>
            Hayden Bell, Christopher A G Booth, Andrew Lane
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2fcd1fe48f0aed8f8d06a8b12971cf0f801b0f5" target='_blank'>
              Deconvolution of haematological cancer methylation patterns reveals a predominantly non-disease related proliferation signal and uncovers true disease associated methylation changes.
              </a>
            </td>
          <td>
            H. Lalchungnunga, H. Atasoy, Edward C Schwalbe, C. M. Bacon, G. Strathdee
          </td>
          <td>2025-10-31</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The E7 oncoprotein of mouse papillomavirus (MmuPV1) plays a pivotal role in both viral infection and cancer development. While earlier studies have identified key cellular targets of MmuPV1 E7, such as pRB and PTPN14, the broader impact of MmuPV1 E7 on keratinocyte homeostasis and shared activities with human papillomavirus (HPV) E7 remains unclear. In this study, we employed proteomic and transcriptomic analyses using our established mouse keratinocyte model—previously instrumental in uncovering a novel function of MmuPV1 E6—to investigate the biological consequences of MmuPV1 E7 expression in mouse keratinocytes. Our findings reveal that MmuPV1 E7 induces cellular changes reminiscent of those driven by “high-risk” HPV infection implicated in cervical cancer. Notably, MmuPV1 E7 did not activate canonical E2F-responsive gene expression or promote proliferation, reinforcing the idea that MmuPV1 E6 is the primary driver of cell cycle activation. However, MmuPV1 E7 expression led to a significant accumulation of stress keratin 17, a marker associated with immune evasion and elevated in both HPV16 transgenic models and MmuPV1 infections. Additionally, we observed enhanced PI3K-AKT-mTOR signaling, with increased levels of phosphorylated S6 kinase and heightened sensitivity to epidermal growth factor stimulation. Collectively, these results underscore the role of MmuPV1 E7 in promoting oncogenic phenotypes and highlight its relevance as a model for studying the molecular underpinnings of “high-risk” HPV-driven disease. IMPORTANCE In this study, we determined the ability of the MmuPV1 E7 oncoprotein in promoting disruption of keratinocyte homeostasis in mouse keratinocytes. Using a multiomics approach, we observed that MmuPV1 E7 promoted several phenotypes associated with “high-risk” human papillomavirus (HPV) infection. Specifically, we confirmed that MmuPV1 E7 does not increase E2F-responsive gene expression and proliferation of mouse keratinocytes. We did find that MmuPV1 E7 was able to increase the expression of stress keratin 17, which promotes immune evasion in papillomavirus infections. Finally, MmuPV1 E7 showed increased expression of genes associated with PI3K-AKT-mTOR signaling. Consistent with this observation, MmuPV1 E7-expressing mouse keratinocytes had elevated phosphorylation of S6 kinase. We also found that MmuPV1 E7 potentiates this signaling through increased sensitivity to epidermal growth factor stimulation. Our collective data show that MmuPV1 E7 promotes several phenotypes associated with “high-risk” HPV infection and cancers. In this study, we determined the ability of the MmuPV1 E7 oncoprotein in promoting disruption of keratinocyte homeostasis in mouse keratinocytes. Using a multiomics approach, we observed that MmuPV1 E7 promoted several phenotypes associated with “high-risk” human papillomavirus (HPV) infection. Specifically, we confirmed that MmuPV1 E7 does not increase E2F-responsive gene expression and proliferation of mouse keratinocytes. We did find that MmuPV1 E7 was able to increase the expression of stress keratin 17, which promotes immune evasion in papillomavirus infections. Finally, MmuPV1 E7 showed increased expression of genes associated with PI3K-AKT-mTOR signaling. Consistent with this observation, MmuPV1 E7-expressing mouse keratinocytes had elevated phosphorylation of S6 kinase. We also found that MmuPV1 E7 potentiates this signaling through increased sensitivity to epidermal growth factor stimulation. Our collective data show that MmuPV1 E7 promotes several phenotypes associated with “high-risk” HPV infection and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc11c43dd44972330a70bdfb67ca963643009c95" target='_blank'>
              MmuPV1 E7 promotes phenotypes associated with “high-risk” HPV infection in mouse keratinocytes
              </a>
            </td>
          <td>
            Kayla R. Duxbury, Liyan Zhang, Laura K. Muelhbauer, Mitchell Hayes, Joshua J. Coon, Megha Padi, James C. Romero-Masters
          </td>
          <td>2025-10-30</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Deuterium, a stable isotope of hydrogen present in natural water at ~150 ppm, has been implicated in modulating cellular metabolism and tumor progression. While deuterium-depleted water (DDW) has shown anti-cancer effects in preclinical and clinical studies, the underlying transcriptional mechanisms remain incompletely defined. Here, we profiled gene expression in A549 lung adenocarcinoma cells cultured for 72 h in media containing four graded deuterium concentrations (40, 80, 150, and 300 ppm) using a targeted NanoString panel of 236 cancer-related genes. After stringent quality filtering, 87 genes were retained and classified into nine distinct expression patterns based on fold-change trends relative to the 150 ppm control. High deuterium (300 ppm) induced strong upregulation (up to 2.1-fold) of oncogenic and survival-related genes (e.g., EGFR, CTNNB1, STAT3, CD44), while DDW (40–80 ppm) led to selective downregulation (down to 0.58-fold) of oncogenes (e.g., MYCN, ETS2, IRF1) and drug-resistance genes (e.g., ABCB1). Se-veral genes involved in DNA repair, apoptosis, and extracellular matrix remodeling exhibited dose-dependent responses, suggesting coordinated regulation by deuterium abundance. These findings demonstrate that deuterium concentration functions as a biologically active variable capable of modulating cancer-relevant gene networks. This exploratory dataset refines mechanistic models of DDW action and provides a foundation for future studies incorporating biological replication, functional assays, and in vivo validation. Significance: Deuterium concentration modulation alters oncogenic, apoptotic, and drug-resistance gene networks in lung adenocarcinoma cells, refining prior models of deuterium-depleted water effects. These findings identify deuterium concentration as a biologically active variable warranting further mechanistic and translational investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e32eb13240959435681e9160aca37dba43a34498" target='_blank'>
              Gene Expression Patterns in Lung Adenocarcinoma Cells in Response to Changes in Deuterium Concentration
              </a>
            </td>
          <td>
            Gabor I. Csonka, András Papp, I. Somlyai, G. Somlyai
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Inactivation of tumor suppressor genes (TSGs) impart a cellular fitness in cancers, including acute myeloid leukemia (AML). The silencing of TSGs without direct mutations presents challenges in cancer therapy but also presents a therapeutic opportunity to restore their function. In this study, we identified the transcriptional repressor ZBTB7A as a TSG that is downregulated in AML and associated with poor survival outcomes. Loss of ZBTB7A amplifies TNF signaling, driving a dysfunctional inflammatory state that accelerates leukemia progression. Mechanistically, the mRNA decay factor ZFP36L2 binds to the 3' untranslated region (3'UTR) of ZBTB7A, promoting its transcript degradation. To uncover therapeutic strategies, we developed a CRISPR-based screening approach coupled with in situ FISH-Flow, pinpointing KDM4 as a vulnerability to restore ZBTB7A function. Pharmacologic inhibition of KDM4 enhanced ZBTB7A expression, promoted terminal differentiation of leukemic cells, and demonstrated broad anti-leukemic activity across AML subtypes while preserving normal hematopoiesis. These findings reveal critical regulatory mechanisms of ZBTB7A and support epigenetic therapy as a promising strategy to reactivate its tumor suppressor function in hematologic cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaf7a74c25b86553fb4e494e6a32f6ab68f9d8f" target='_blank'>
              Epigenetic reactivation of a tumor suppressor program in AML.
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Arnuk, I. Aifantis, Hussein A. Abbas, A. Tsirigos, Sadik Karma, S. Rajendran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10944325f830532ecfb5fb037e3e00b3a58c4a6" target='_blank'>
              Sema7a drives an immunosuppressive microenvironment of breast cancer via Kdm4a-mediated DNA replication regulation.
              </a>
            </td>
          <td>
            Xianan Bai, Shanshan Cai, Jie Jiang, Maojin Tian, Jianxin Du, Peiqing Zhao
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Early-stage Luminal A breast cancer generally has a favorable prognosis, yet some patients experience recurrence, presenting a challenge in understanding the underlying genetic factors. This study aimed to identify genetic mutations associated with recurrence in early-stage Luminal A breast cancer patients through whole-exome sequencing (WES). We collected formalin-fixed paraffin-embedded samples from 34 patients and divided them into two groups: 17 patients with recurrence within five years post-surgery (recurrence group) and 17 patients with no recurrence for over five years (control group). The extracted DNA went through library preparation and was subjected to WES. Sequencing data went through quality control, alignment, and mutation identification. Functional enrichment analyses were conducted to explore the biological implications of the mutations. We generated on average ~ 11 Gb raw sequencing data for each sample and identified 7,066 nonsynonymous mutations. The recurrence group exhibited a higher mutation rate (11.48 mutations/Mb) compared with the control group (9.18 mutations/Mb, p < 0.05). A significant negative correlation was observed between disease-free survival time and the number of mutations (p < 0.05). Eight genes (MICALCL, G6PD, OR8U1, PCLO, OR8U8, ZCCHC18, CPED1, HMCN1) were significantly associated with early recurrence (p < 0.05). Functional enrichment analyses revealed that these genes were involved in pathways like mismatch repair and immune response. This study identified specific genetic mutations linked to early recurrence in Luminal A breast cancer, highlighting potential biomarkers for predicting patient outcomes and personalizing cancer treatment. Our study also showed that state-of-the-art WES can extract biologically and clinically meaningful mutation signatures from routinely stored FFPE tissues, unlocking archived specimens for large-scale biomarker discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9b4aba66e01c7c4c06919d370f4457e1aff8f9" target='_blank'>
              Whole-exome sequencing of FFPE samples reveals mutations associated with Luminal A breast cancer recurrence
              </a>
            </td>
          <td>
            Kezhen Lv, Yongxia Chen, Jichun Zhou, Feiyang Ji, Wenxian Hu
          </td>
          <td>2025-11-11</td>
          <td>BMC Biotechnology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="In the 40 years since the discovery of the CENP proteins, many studies have examined the role of these proteins and their interactions with other chromosomal proteins of the centromere and beyond. Together, these studies have yielded vast amounts of sequencing and proteomics data. Typically, each study has focused on a single question and the majority of each dataset remains largely unexplored. Often the interesting details of publicly deposited data are left behind, buried in archives online, while more and more new data are generated. Reanalysing these databases can represent a new paradigm for investigating diverse biological pathways in unprecedented detail. Here, we explore two publicly available pan-cancer proteomic datasets to compare proteins whose abundance correlates with CENP proteins, with a particular focus on CENP-C. Our analysis confirms an expected link between CENP-C and cohesin levels but reveals a surprising and unexpected correlation between CENP-C and proteins of the inner nuclear membrane and the NuMA protein. This guilt-by-association analysis has the potential to identify proteins that act in common pathways but never associate or colocalize and may not even be expressed at the same time in cells. As an example, we show here that it can reveal unexpected links that expand our conception of centromeric chromatin beyond chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b131b447bc4a00bf7e5247f66ecd03a10d2ff6" target='_blank'>
              Sustainable integrative cell biology: CENP-C is guilty by association
              </a>
            </td>
          <td>
            N. Y. Kochanova, I. Samejima, William C. Earnshaw
          </td>
          <td>2025-11-25</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aim. The current study aims to investigate changes in the expression of proprotein convertases (PCs) genes during the development of lung tumors. PCs is a family of highly specific serine endopeptidases of mammals that process precursors of various proteins and peptides. Nine genes encoding PCs have been identified in the human genome that are essential for normal body functioning. Additionally, PCs can activate proteins involved in carcinogenesis, as well as the expression levels of their genes were shown to correlate with tumor aggressiveness and patient survival. We previously evaluated the expression of all nine PC genes using quantitative PCR on paired samples of lung tumor and adjacent normal tissue. For the first time, we identified four distinct patterns of PCs gene expression change in tumor tissue, which we have called «scenarios». For three of them, covering more than two thirds of the samples, a dominant change in the expression of one PC gene in the tumor tissue was shown. These results may indicate the existence of a limited number of possible options for changes in PCs gene expression during the malignant transformation of lung cells. However, these results need to be confirmed using expanded cohort of tumor samples. Material and Methods. To confirm our previous findings, we analyzed the expression of PCs genes using modern methods of mathematical statistics and data from three previously published studies, which evaluated gene expression through high-throughput RNA sequencing in paired tumor and normal tissue samples from 194 patients with non-small cell lung cancer. Results. Our meta-analysis confirmed that the changes in PCs gene expression in lung tumor tissue compared to surrounding normal tissue follow a limited set of possible scenarios, each having a unique profile of PCs gene expression. Conclusion. The reasons for implementing each scenario may be linked to the origin of tumors, their mutation status, characteristics of the tumor microenvironment, and other factors. Correspondingly, these scenarios may correlate, for example, with tumor aggressiveness and resistance to therapy, and therefore may potentially be used to choose the treatment approach and/or to predict the course of the disease. However, this issue requires further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3af10872808404c4c77f8841898be6b45125c83" target='_blank'>
              Alterations in proprotein convertase genes’ expression during malignization of human lung cells have a limited set of scenarios
              </a>
            </td>
          <td>
            A. A. Komissarov, I. N. Vlasov, P. A. Slominsky, I. Demidyuk
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Background: Fusion transcripts are formed by combining exons from two different genes, often due to structural rearrangements such as deletions, inversions or translocations (genomic rearrangement-dependent, GRD) or through aberrant splicing (genomic rearrangement-independent, GRI). In hematological malignancies, many fusion transcripts act as driver events, playing crucial roles in leukemogenesis and serving as diagnostic markers, such as BCR::ABL1. However, the role and diagnostic significance of other fusion transcripts, like P2RY8::CD99—which has been found in both healthy samples and B-ALL patients—are less clear. Similarly, the clinical relevance of other GRI fusion transcripts, such as SEMA6A::FEM1C, often remains uncertain. Single-cell analysis is a promising approach to detect these fusion transcripts within individual cells and cell populations, shedding light on the subclonal architecture and cells of origin.
 Aim: Evaluating the feasibility of integrating long-read sequencing with single-cell library preparation to detect and characterize GRD and GRI fusion transcripts.
 Patients and Methods: Our cohort consisted of 10 samples diagnosed with B-ALL, featuring various fusion transcripts: BCR::ABL1 (n = 6, GRD), SEMA6A::FEM1C (n = 10, GRI), EBF1::PDGFRB (n = 1, GRD), and P2RY8::CD99 (n = 1, GRI), as identified through bulk whole transcriptome analysis. Cryopreserved cells were processed using the GEM-X Universal 3' Expression Library Prep Kit (10x Genomics) and GEM-X cDNAs with cell-specific labels served as input for the Kinnex single-cell RNA kit (PacBio). GEM-X libraries were sequenced on the NovaSeqX instrument (Illumina) with a median depth of 21,565 reads per cell. Kinnex libraries on the Revio (PacBio) with 3,767,066 mean HiFi reads per sample and a mean HiFi read length of 14.2kb. GEM-X libraries were analyzed using cellranger (v9.0.1) and seurat (v5.2.1). Kinnex libraries were preprocessed with the Iso-Seq workflow (PacBio) and fusion transcripts were called with pbfusion.
 Results: Cells from different samples were merged and clustered based on their gene expression profiles from GEM-X libraries. Cell clusters were annotated as B-cells based on CD10, CD79A, and PAX5 expression; T-cells based on CD3 expression; hematopoietic stem cells as CD34+; and myeloid cells by the absence of lymphatic markers and the presence of CD14, FCER1G, and CEBPD. Within the B-cell population, three distinct subpopulations corresponding to various B-cell developmental states were identified: pro-B, pre-proB, and pro-B VDJ. The pro-B VDJ state, characterized by heavy chain rearrangement, showed increased CD20 expression. Furthermore, cell cycle analysis using 100 genes associated with the S-phase/G2M-phase revealed a subpopulation of cycling pro-B cells marked by high levels of MKI67 (G2M-phase) and MCM4 (S-phase). Our fusion calling pipeline successfully detected BCR::ABL1, SEMA6A::FEM1C, EBF1::PDGFRB, and P2RY8::CD99 fusion transcripts within the long-read dataset, without any false positive calls at the sample level. These fusion transcripts were found exclusively in B precursor cells when mapped to the cellular landscape. Notably, cells in the pro-B VDJ state did not harbour any fusion transcripts. Interestingly, P2RY8::CD99 was identified only in B-cells, not in T-cells or myeloid progenitors, suggesting its association with pathogenic cells. Similarly, SEMA6A::FEM1C, a GDI fusion transcript, was confined to B precursor cells, indicating an association with the B-ALL clone.
 Conclusions: We have demonstrated the feasibility and utility of integrating long-read isoform sequencing with single-cell library preparation for detecting fusion transcripts of diverse origins. This transcriptome-wide approach enables disease-agnostic detection and characterization of fusion transcripts, not only those arising from chromosomal aberrations (GRD) but also those resulting from aberrant splicing (GRI) at the individual cell level. Moreover, this method facilitates a comprehensive analysis of cell-specific gene expression profiles and fusion transcripts and could be expanded to include the detection of single-nucleotide variants and copy number changes, thereby completing the molecular profile. While currently not designed for integration into routine workflows, this method represents a valuable tool for research projects aimed at elucidating the molecular mechanisms underlying various diseases, with the goal to enhance patient care.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae73b6fed764877551a830ff73b15a040a45186" target='_blank'>
              Long-read single-cell isoform sequencing for cell type-specific detection of genomic rearrangement-dependent and -independent fusion transcripts
              </a>
            </td>
          <td>
            W. Walter, W. Kern, A. Stengel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181f022b28db4f3ed7a11dbea0e2ef041b80de4b" target='_blank'>
              Discovery of the TGCB microprotein translated from the lncRNA CHASERR
              </a>
            </td>
          <td>
            Bernardo Bonilauri, Xiaochun Yang
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran C. Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, Keith L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="


 Dysregulation of the MAF transcription factor, frequently overexpressed in multiple myeloma (MM) due to adverse translocations like t(14;16), is a well-established driver of poor prognosis. While the mechanisms are complex and involve super-enhancers (SEs), the specific downstream effectors contributing to MAF's oncogenicity remain incompletely understood. To identify critical MAF targets, we integrated genes associated with poor MM prognosis, MAF-regulated genes, and SE-associated genes, pinpointing TRIM47 as a prime candidate. Although TRIM47 is a crucial E3 ubiquitin ligase implicated in various cancers and inflammatory disorders, its functional role in MM remains undefined.
 Knockdown of MAF in MM cell lines (ARP-1, KMS-11) significantly reduced both TRIM47 transcription and protein levels. Mechanistically, ChIP-seq and ATAC-seq analyses revealed that MAF binding extends beyond the TRIM47 promoter. Multiple MAF ChIP-seq peaks co-localized strongly with H3K27ac peaks within a ±10 kb window flanking the TRIM47 coding sequence (CDS), suggesting MAF-mediated SE activation drives TRIM47 expression. Functional validation using the SE inhibitor JQ1 and sgRNA-mediated targeted deletion confirmed four critical SE sites; disruption of these sites markedly diminished both MAF/H3K27ac enrichment and TRIM47 expression. To elucidate TRIM47's function, ubiquitinomics and proteomics screens identified potential interacting partners. Subsequent validation revealed a specific and robust endogenous and exogenous interaction solely between TRIM47 and RARα (Retinoic Acid Receptor Alpha), with their protein levels often showing an inverse correlation across MM cell lines. Notably, RARα activation by all-trans retinoic acid (ATRA) has been linked to enhanced efficacy of CD38-targeted immunotherapy in MM.
 Using truncation mutagenesis, we mapped the interaction: RARα binds the SPRY domain of TRIM47, while TRIM47 binds RARα's ligand-binding domain (LBD). Critically, we demonstrated that TRIM47, via its RING domain, mediates K48-linked polyubiquitination of RARα specifically at lysine 134 (K134), targeting it for proteasomal degradation. Functionally, ATRA treatment induced CD38 expression more readily in MAF-knockdown or TRIM47-knockdown cells. This enhanced response was rescued by re-expressing wild-type TRIM47, but not by a RING domain-deletion mutant (TRIM47-ΔRING).
 We next established that TRIM47 ablation specifically licenses ATRA-driven enhancement of CD38-targeted immunotherapy through integrated in vitro and in vivo models. Critically, co-culture experiments demonstrated that while TRIM47 knockdown alone did not alter baseline CD38-CAR-T cytotoxicity or T-cell activation profiles, ATRA pretreatment of shTRIM47 myeloma cells triggered dramatically enhanced CAR-T-mediated killing accompanied by significantly amplified IFN-γ/TNF-α secretion and robust CAR-T proliferative expansion – revealing TRIM47 targeting as a molecular prerequisite for ATRA-driven CAR-T potentiation. Paralleling these findings, in vivo studies using NSG mice engrafted with luciferase-coupled shTRIM47 myeloma cells showed comparable tumor progression and CD38-CAR-T monotherapy response relative to shCtrl controls; yet when combined with ATRA, the shTRIM47 cohort exhibited profound tumor regression and survival extension unseen in control groups. Mechanistic interrogation across both systems converged on a unified pathway: TRIM47 ablation stabilizes RARα, enabling ATRA to elevate CD38 surface expression which subsequently licenses enhanced CAR-T recognition, triggers amplified effector cytokine production, and drives clonal CAR-T-cell expansion – establishing a self-reinforcing therapeutic circuit.
 In summary, our findings establish a novel MAF-TRIM47-RARα axis in MM pathogenesis: MAF orchestrates super-enhancer remodeling to drive TRIM47 expression, which subsequently targets RARα for degradation. This axis critically modulates the response to ATRA. Importantly, these results suggest that patients with t(14;16) or other high-MAF-expressing MM subtypes may exhibit diminished responsiveness to ATRA-mediated enhancement of CD38-targeted immunotherapy efficacy due to TRIM47-dependent RARα degradation. Targeting this axis represents a potential therapeutic strategy for this adverse-risk patient group.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2636bbc8566cb42ae6aaf4d94c40b29e10850104" target='_blank'>
              Super-enhancer-mediated regulation of CD38 inducibility in multiple myeloma via the MAF/TRIM47/RARα axis
              </a>
            </td>
          <td>
            Zhenfeng Dai, Yunlu Jia, Qiqi Shen, Haoguang Chen, Xinyuan Dai, Shengchun Jin, E. Zhang, Jianbiao Zhou, Gang Xiao, Wee J. Chng, Zhen Cai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 RAD51 is a highly conserved gene whose protein product, RAD51, is instrumental in the maintenance of genome integrity in eukaryotes. RAD51 is well known to function in homologous recombination repair ('HRR') in conjunction with the BRCA1 and BRCA2 gene products. It also has other less well-known functions including protection from replication stress ('RS'). RS occurs downstream of incessant oncogene signaling and is therefore typical of cancers in general, and leukemias in particular, but not normal hematopoietic cells. RS is defined by stalled replication forks, which, if unprotected, are susceptible to endonuclease-mediated digestion leading to potentially lethal double strand DNA breaks. RAD51 gathers around stalled forks and protects them by a variety of processes including fork reversal to allow survival and proliferation of cancer cells. RAD51 is, therefore, a potentially important target in leukemias: its inhibition has potential to selectively kill malignant cells.
 We have shown that lead compounds of the IBR family of RAD51 inhibitors (IBR2, IBR120 and JKYN-1) have low micromolar single agent activity in a variety of cancer cell lines including representatives of both acute ('AML') and chronic ('CML') myeloid leukemia. Furthermore, these compounds often exhibit strong synergy and at least additivity in combination with both on-label and off-label use of FDA-approved compounds, as well as investigational molecularly-targeted agents, in relevant cell lines; these include FLT3 inhibitors quizartinib, gilteritinib and tuspetinib ((in MV-4-11 and HL60 cell lines) and BCR/ABL inhibitors (imatinib and regorafenib) in the K562 cell line (AACR 2025). Interestingly, strong synergy was demonstrated with JKYN1 and quizartinib, and separately, tuspetinib, in the FLT3 WT line HL60, but only additivity between JKYN-1 and gilteritinib in that cell line; all three of these compounds were extremely potent in the FLT3 mutated MV-4-11 cell line; additivity with JKYN-1 was nonetheless demonstrable, but not synergy. IBR2 exhibited low micromolar single agent activity in the K562 CML line and synergized with both imatinib and (separately) the off-label agent regorafenib. Regorafenib was also strongly synergistic with IBR2 in multiple other solid tumor cell lines, both within and beyond its label in colorectal cancer.
 We have confirmed the IBRs inhibit multiple RAD51 functions including hydroxyurea-induced RAD51 focus formation, HRR by DR-GFP assay, and RS protection by RPA focus formation and a DNA fibre assay. They markedly reduce intracellular RAD51 levels. All the IBRs tested to date bind RAD51 strongly by surface plasmon resonance (‘SPR‘), especially JKYN-1, and JKYN-1 is more potent in this and other assays than the standard classical RAD51 inhibitor B02. Our hypothesis is that both the single agent activity and synergy is due primarily to deprotection of RS.
 JKYN-1 is a partially optimized (more soluble and more potent) version of IBR120 but is not resistant to degradation by liver microsomes or sufficiently capable of entry into target cells. We have therefore selected 15-20 IBR analogues amongst a pool of ~35,000 created by generative chemistry (InSilico Medicine, Boston) for synthesis and in vitro testing. Anti-leukemic properties of these and our current IBRs will be tested on primary patient derived cells from genetically diverse AML patients. The assay will target stemness properties of AML patients in a high throughput screen designed to provide clinically actionable data for relapsed AML. Treatment dose and combinations will be pre-screened using a highly predictive cell line as a surrogate of human cancer stem cells for subsequently AML patient panels of leukemic stem cells to be examined. These studies will focus on single agent anti-proliferative activity and synergy with a suite of standard and investigational antileukemic drugs including cytosine arabinoside, daunorubicin, quizartinib, gilteritinib, tuspetinib, venetoclax, and azacitadine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79dc795d42c064ddfb139b88cd12f7483902f754" target='_blank'>
              Inhibitors of RAD51 as potential novel therapies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Mick Bhatia, P. Ferguson, A. Xenocostas, James Koropatnick, Morgan Black, Mark D. Vincent
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410d0943e5dad068f895d42bbe237bbb7cbc02b7" target='_blank'>
              DNA topoisomerase I acts as supercoiling sensor for bacterial transcription elongation.
              </a>
            </td>
          <td>
            Vita Vidmar, Céline Borde, Lisa Bruno, Nataliya Miropolskaya, Maria Takacs, Claire Batisse, C. Saint-André, Chengjin Zhu, O. Espéli, Valérie Lamour, A. Weixlbaumer
          </td>
          <td>2025-12-01</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a major cause of cancer-related mortality globally. Previous studies have reported that oral cancer overexpression 1 (ORAOV1) is overexpressed in HCC and correlated with poor prognosis, yet its molecular mechanisms remain incompletely understood. In this study, ORAOV1 overexpression was confirmed in HCC tissues via tissue microarray analysis and functionally linked to tumor cell proliferation through a positive correlation with Ki-67 expression in the human HCC cell line MHCC-97L. Bioinformatics analyses using The Cancer Genome Atlas (TCGA) and three Gene Expression Omnibus (GEO) HCC datasets further supported these findings. Multiple mechanisms appear to drive ORAOV1 upregulation, including promoter hypomethylation, amplification of the 11q13 region, and a putative ceRNA network involving AC005332.1, AC012615.1, and hsa-miR-100-5p. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses implicated ORAOV1 in various cellular processes, such as abnormal membrane channel function, extracellular matrix–receptor interactions, IL-17 signaling, and peroxisome proliferator-activated receptor (PPAR) signaling. Co-expression analysis identified significant associations between ORAOV1 and the oncogenes TPCN2 and CCND1. Additionally, ORAOV1 expression correlated with enhanced infiltration of immunosuppressive cells, including regulatory T cells, myeloid-derived suppressor cells, and cancer-associated fibroblasts, as well as upregulation of immune checkpoint markers (PD-1, PD-L1, and CTLA-4). These results indicate that ORAOV1 may modulate the immunosuppressive tumor microenvironment and contribute to resistance against immunotherapy, highlighting its potential as a therapeutic target in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b50660a0d2a36a94ce4bc7721bd3f07db2787d9" target='_blank'>
              Overexpression of ORAOV1 and its association with immunotherapy resistance in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yuzhen Huang, Ni Yang, Su Wen, Ziwei Fang, Yucong Zhang, Zonghao Qian, Yi Huang, Tiejun Yin, Cuntai Zhang, Le Zhang
          </td>
          <td>2025-11-25</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Supratentorial ependymomas are often life-threatening brain tumors that are characterized by significant molecular heterogeneity. Hypoxia contributes to this heterogeneity, but the extent and nature of this relationship remain unclear. This poses challenges for effective treatment strategies, demanding a more holistic and detailed understanding of the underlying biology. In this study, we employed single-nucleus (n = 63) and spatial (n = 30) transcriptomics to delineate the cellular and spatial landscape of supratentorial ependymoma. Characterizing the transcriptional consensus programs of this entity revealed two previously undescribed programs that are associated with the remodeling of the extracellular matrix and the ZFTA fusion identity. These programs were related to highly unfavorable outcome and were exclusive to ZFTA fusion-positive tumors. Developing a set of spatial scoring algorithms that are sensitive to mixed spot transcriptomes enabled the identification of the role of hypoxia as the potential driver of spatial organization. Together with recurrent spatial associations inferred between programs, this informed a generalized model of the higher-order architecture. The functionally and spatially defined zones of this model displayed distinct immune presence, with the highest seen in the hypoxic zone. These results provide insights into the molecular heterogeneity and spatial organization of supratentorial ependymomas, which may aid in the development of targeted future treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6575f9d3eac6558b40e6240ea8f2f812824b5235" target='_blank'>
              Integrated Single-Nucleus and Spatial Transcriptomics Elucidate Heterogeneity and Hypoxia-Driven Organization of Supratentorial Ependymoma.
              </a>
            </td>
          <td>
            Erik Schüftan, I-Na Lu, Flavia W de Faria, Clara Inserte, C. Rössig, Julia Bothe, Adrienn Bagosi, S. Tippelt, Tobias Blau, C. Monoranu, L. Altendorf, D. Spille, Ann-Katrin Bruns, A. Brentrup, Walter Stummer, D. Gorodezki, Martin Ebinger, J. Schittenhelm, C. Thomas, T. Albert, U. Schüller, K. Kerl
          </td>
          <td>2025-10-28</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Nasopharyngeal carcinoma (NPC) is endemic to several regions of North Africa and Southeast Asia and has a striking geographic distribution. It is closely linked to the Epstein-Barr virus (EBV) and is still found at an advanced stage that permits metastasis. However, only a limited number of EBV genes, including latent membrane protein 1 (LMP1), have been identified. The 30bp deletion variant of LMP1, which is thought to be the primary EBV oncoprotein, was more prominent in NPC biopsies. The aim of this study was to employ a 30bp deletion in LMP1 as a biomarker to help clinicians diagnose NPC early, improve prognosis, and reduce mortality rate. Polymerase chain reaction (PCR) was used to detect the prevalence of EBV in patients with NPC by amplification of the EBNA-1 and LMP1 genes. β-actin was amplified in all samples as a positive PCR control. The incidence and correlation of the 30bp deletion of LMP1 in 71 paraffin-embedded NPC tissue samples from Jordanian patients (51 males and 16 females) were also evaluated. The results were confirmed by Sanger sequencing and statistically analyzed using SPSS software. EBV was detected in all 71 NPC tissue samples, with positive results for EBNA-1 gene in 58 cases (81.7%) and in combination with LMP1 with or without 30bp deletion in 69 cases (97.18%) whereas, LMP1 30bp deletion was detected in 39 samples (54.9% of all samples). Heterogeneity of mutation was observed in 14.08% of samples. No significant differences were found between the stages with regard to the LMP1 30bp deletion. Statistical analysis showed a significant association between age and LMP1 30bp deletion with p-value = 0.015, and between the source of sample with LMP1 30bp deletion p-value = 0.032, whereas there was no association between LMP1 30bp deletions and patient sex or histological NPC type. A 30bp deletion of LMP1 was found in 54.9% of NPC tissues. This is within the range reported in other studies. There was no significant association between the 30bp deletion of LMP1 and histological type or disease phase. Further research is required to determine how this mutation in LMP1 affects the course and results of patients with NPC in Jordan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28abb19a141082a00b48aeba4beb04a9be49d2d5" target='_blank'>
              Detection of 30bp deletion LMP1 oncogene as potential diagnostic and prognostic Biomarker among nasopharyngeal carcinoma patients
              </a>
            </td>
          <td>
            Aseel Al-Hussien, M. Abusalah, Yaqeen Rjoub, Khaled Al-Qaoud, Anwar Rjoop, Yasmin AlSaidat, Lubna H Khwaileh, Manal Abusalah, Naveed Ahmed
          </td>
          <td>2025-11-19</td>
          <td>Electronic Journal of General Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Embryonal tumors with multilayered rosettes (ETMR) are rare pediatric brain cancers with a median survival of 14 months despite aggressive treatment. The oncogenic driver in ~90% of cases is an amplification and fusion of the chromosome 19 micro-RNA cluster (C19MC), which is yet undruggable. Due to the tumor’s rarity, comprehensive molecular characterization and the identification of therapeutic targets are challenging, and ETMR remains poorly understood.
 We used single cell RNA sequencing and multiplexed protein profiling to study intratumoral heterogeneity and oncogenic signaling in ETMR (n=11). MiRNA-mRNA interactions were investigated by enhanced crosslinking and immunoprecipitation (eCLIP) followed by next-generation sequencing. ETMR cells were treated with locked nucleic acids (LNAs) and small-molecule inhibitors, and their anti-tumor effects assessed through confluency and death assays. Here, we identified three distinct malignant ETMR cell types resembling differentiation stages during early fetal neocortex development: undifferentiated ETMR cells resembling neural-stem cells (NSC), differentiated cells corresponding to mature neurons, and intermediate cells exerting markers of neuronal lineage commitment. C19MC was overexpressed in mitotically active NSC-like cells. Using eCLIP, we uncovered widespread regulatory roles of individual C19MC members. Markers of neuronal differentiation (NR2F2, SOX4, SOX11) and lineage commitment (C2–H2 ZNF family members) were targets of abundant C19MC members. LNA-mediated blockade of key C19MC members significantly reduced ETMR cell confluency and induced apoptosis. Genome-wide cell-cell communication analysis identified fibroblast growth factor receptors (FGFRs) and NOTCH receptors as key mediators of cellular cooperation. Accordingly, FGFR and NOTCH inhibitors had significant anti-tumor effects in vitro, and erdafitinib-supplemented therapy led to partial response in one therapeutically exhausted ETMR patient. Together, we demonstrated intratumoral heterogeneity in ETMR paralleling fetal neurodevelopment. Targeting C19MC in undifferentiated NSC-like cells significantly impaired cell confluency and survival. Additionally, FGFR emerged as a clinically actionable target, showing promising anti-tumor activity in vitro and in one ETMR patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb99e7f0c8810ec820bdec1d791a67a355e38fb" target='_blank'>
              EXTH-92. Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions
              </a>
            </td>
          <td>
            L. Gabler-Pamer, Alexander Beck, G. A. V. Cruzeiro, S. Lambo, McKenzie L. Shaw, Olivia A Hack, Andrezza Nascimento, S. Madlener, Daniela Lötsch, I. Slavc, C. Dorfer, C. Haberler, Lissa C. Baird, Susan N. Chi, S. Alexandrescu, J. Gojo, Kool Marcel, V. Hovestadt, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Nuclear Factor I (NFI) proteins are involved in adenovirus DNA replication and regulate gene transcription, stem cell proliferation, and differentiation. They play key roles in development, cancer, and congenital disorders. Within the NFI family, NFI-X is critical for neural stem cell biology, hematopoiesis, muscle development, muscular dystrophies, and oncogenesis. Here, we present the structural characterization of the NFI transcription factor NFI-X, both alone and bound to its consensus palindromic DNA site. Our analyses reveal a MH1-like fold within NFI-X DNA-binding domain (DBD) and identify crucial structural determinants for activity, such as a Zn²⁺ binding site, dimeric assembly, and DNA-binding specificity. Given the ~85% sequence identity within the NFI DBDs, our structural data are prototypic for the entire family, a NFI Rosetta Stone that allows decoding a wealth of biochemical and functional data and provides a precise target for drug design in a wider disease context. The Nuclear Factor I (NFI) family includes transcription factors key to development, cancer, and genetic disorders. Here, the authors present the structure and DNA-binding properties of NFI-X, a prototype for the entire NFI family.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09b9f75cb385c6026fbb070d6c58ca1ca1038c7" target='_blank'>
              Structural basis of Nuclear Factor 1-X DNA recognition provides prototypic insight into the NFI family
              </a>
            </td>
          <td>
            Michele Tiberi, M. Lapi, L. Gourlay, A. Chaves-Sanjuan, Maurizio Polentarutti, N. Demitri, Miriam Cavinato, Diane Marie Valérie Jeanne Bonnet, Valentina Taglietti, Anna Righetti, Rachele Sala, Silvia Cauteruccio, Amit Kumawat, R. Russo, A. Barbiroli, N. Gnesutta, C. Camilloni, M. Bolognesi, Graziella Messina, Marco Nardini
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Introduction: Diffuse large B-cell lymphoma (DLBCL) is stratified into genetic subtypes (MCD, BN2, A53, N1, EZB, ST2) that differ in their gene expression profiles, oncogenic mechanisms, and response to therapy. Using paired single cell RNA (scRNA) and ATAC (scATAC) sequencing in DLBCL tumors, we previously identified gene expression themes reflecting B cell differentiation, cell growth, and cell cycle that distinguished intratumoral genetic subclones (Wang B, ASH, 2024). Here, we present a global analysis of transcription factor (TF) binding and activity in normal B cells and DLBCL tumors that revealed epigenetic heterogeneity among the DLBCL genetic subtypes, which underpins their divergent therapeutic responses.
 Methods: Paired scRNA and scATAC sequencing was performed on 102 DLBCL cases (504,444 cells) and 3 tonsils (12,227 cells). Gene expression and genetic subtypes were determined from matched bulk samples by RNA and whole exome sequencing. Computational analysis was performed using R/python and custom bioinformatic pipelines.
 Results: By linking TF activators (+/+) and repressors (-/+) to target gene expression using SCENIC+ (Bravo Gonzalez-Blas C, Nat Methods, 2023), we identified gene regulatory networks (GRNs) composed of enhancer-driven Regulons (eRegulons). In tonsillar B cell subpopulations, we identified 173 eRegulons that linked TF binding to 9,456 genomic regions and 3,535 target genes. A subset of these eRegulons were differentially active (p<0.05) in germinal center (GC) B cells (FOXO1, MEF2B, EBF1, PAX5, TCF3), plasma cells (PC; IRF4, XBP1, PRDM1) and memory B cells (KLF2, STAT1, ETV6). Importantly, cell lineage analysis traced the activity of these eRegulons along 3 differentiation trajectories stemming from naïve B cells towards either GC dark zone, PC, or memory B cells.
 Next, we used TF binding to define the epigenetic landscape of the DLBCL genetic subtypes, which could be distinguished from each other using subtype-specific gene expression signatures. Chromatin binding by TFs that regulate PC differentiation (IRF4, POU2F2, TCF4) correlated with the MCD, BN2 and A53 gene expression signatures as well as with gene expression themes reflecting PC differentiation, cell cycle, and cell growth. Binding by another group of TFs (FOXO1, MYBL1, STAT6, PAX5) was associated with the EZB signature and the GC B cell gene expression theme. Binding by BCL6 was anticorrelated with signatures of the N1 subtype and memory differentiation, suggesting that BCL6 antagonizes memory B cell differentiation and the generation of N1 DLBCL.
 To define GRNs in DLBCL genetic subtypes and genetic subclones, we used SCENIC+ to infer 289 eRegulons, comprised of 12,016 TF binding regions and 4,723 target genes. By integrating the eRegulon RNA and ATAC scores using multiomics factor analysis (Argelaguet R, Genome Biol, 2020), we identified major axes of variation that discriminated both DLBCL subtypes and normal B cell populations. The EZB and ST2 subtypes were significantly associated (p<0.05) with eRegulons that typify normal GC B cells (MEF2B, MEF2C, IRF8, FOXO1). Within these subtypes, subclones with REL amplification had significantly greater activity of a REL +/+ eRegulon than those with wild type REL (p<0.001). The MCD, A53 and BN2 subtypes were enriched (p<0.05) for the IRF4 +/+ eRegulon while MCD was additionally associated (p<0.05) with BATF, SPIB, XBP1 and PRDM1 eRegulons. A TBL1XR1 –/+ eRegulon was significantly associated with the N1 subtype (p<0.001), which is notable given that TBL1XR1 is a tumor suppressor that is frequently inactivated in N1. The subtype-associated eRegulons were also differentially active in normal B cell populations, with several MCD eRegulons active in PCs, N1 eRegulons active in memory B cells, and EZB eRegulons active in GC B cells. Accordingly, eRegulon scores correlated with the B cell differentiation themes across DLBCL subclones.
 Conclusions: By paired scRNA and scATAC sequencing, we identified GRNs present in normal and malignant B cells that highlight transcriptional states of DLBCL genetic subtypes which vary along three principal differentiation axes – GC B cell, memory B cell and PC. Our analysis illuminates the biological heterogeneity of DLBCL molecular subtypes and offers rationale targets for future therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f09806b355405a44b654842f8d2b5c5a853ff8" target='_blank'>
              Gene-regulatory axes of diffuse large B-cell lymphoma subtypes
              </a>
            </td>
          <td>
            Julius C Enssle, Boya Wang, George W. Wright, Xin Yu, James D. Phelan, Yandan Yang, Arthur Shaffer, D. Huang, Zana Coulibaly, Michael Kelly, Bao Tran, Stefania Pittaluga, M. Roschewski, Wyndham H Wilson, Thomas Oellerich, G. Inghirami, Louis M Staudt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Telomerase Reverse Transcriptase promoter (TERTp) mutations enable tumor cell immortality in millions of cancer patients annually. TERTp mutations are the most common non-coding mutations across all cancers, including glioblastoma, oligodendroglioma, medulloblastoma, and high-grade meningioma. A multitude of TERTp mutations, including the two hotspots, create de novo E26 transformation specific transcription factor binding sites. Among the 28 family members, we have shown that only the GA-binding protein (GABP, composed of DNA binding GABPA and transactivating GABPB subunits) activates the mutant TERTp. Prior therapies targeting the telomerase RNA component, TERC, lacked tumor selectivity and were poorly tolerated. The GABP-mediated reactivation of the mutated TERTp presents a unique therapeutic opportunity for tumor specific reversal of cellular immortality. Breakthroughs in the design of biological proteolysis-targeting chimera (bioPROTACs) offer a new approach to selectively degrade previously intractable targets such as transcription factors. We combined in silico protein-protein interaction modeling via AlphaFold and experimental validation to identify a minimal GABPB to bind GABPA and fused it with an E3 ubiquitin ligase. Introduction of this GABPA bioPROTAC into TERTp mutant glioblastoma cells depleted GABPA protein and eliminated GABPA binding to the mutant TERT promoter, reducing transcriptional activating (H3K4me3) and increasing suppressive (H3K27me3) histone marks. Reversion of mutant TERTp to an epigenetically silenced state reduced TERT expression by 73% to 95% and shortened tumor cell telomeres in a promoter mutation-specific manner. The GABP bioPROTAC reduced tumor growth and improved survival of mice bearing an orthotopic xenograft of TERTp mutated glioblastoma cells. The effects of the bioPROTAC were validated in vivo via magnetic resonance imaging of metabolic correlates of TERT expression. These data demonstrate that this GABP bioPROTAC potently degrades GABPA, leading to tumor-specific silencing of telomerase expression, and a reversal of glioblastoma tumor cell immortality. This artificial intelligence-guided approach may be applicable to other intractable cancer specific targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102d24aea88be03925f3847c09d31dfd2d63944f" target='_blank'>
              EXTH-77. An AI-engineered bioPROTAC reverses immortality of
 TERT
 promoter mutant tumors
              </a>
            </td>
          <td>
            Nicholas O. Stevers, Ellen Chen, Joshua S. Wu, T. Patel, Sara Collins, M. Montoya, Samuel H. Wu, G. Batsios, A. Gillespie, Chibo Hong, Noriyuki Kasahara, Pavithra Viswanath, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Early T-cell precursor (ETP) acute lymphoblastic leukemia/lymphoma (ALL) is an aggressive hematological malignancy characterized by the uncontrolled expansion of T-cell progenitors. Loss-of-function mutations in genes encoding epigenetic regulators, including core components of the Polycomb Repressive Complex 2 (PRC2), such as the EZH2 histone methyltransferase, are frequently observed in ETP-ALL patients.To investigate the role of EZH2 inactivation in human early T-cell leukemogenesis, we optimized a CRISPR/Cas9-based strategy to disrupt EZH2 gene expression by integrating a DNA cassette containing the constitutive human PGK-1 promoter followed by the GFP reporter. Specifically, human CD34⁺ hematopoietic stem/progenitor cells (HSPCs) derived from cord blood were first edited to knock out (KO) EZH2, then transduced with lentiviruses encoding different T-ALL oncogenes or an empty vector control, and maintained in serum-free medium supplemented to support T-cell progenitor expansion. We found that only HOXA9- and LYL1-transduced EZH2KO HSPCs expanded in vitro for up to 30 days, and were enriched for early T-cell subsets, as identified by single-cell RNA sequencing (scRNA-seq) and a 26-color flow cytometry assay. Notably, these same subsets were capable of initiating T-cell leukemia in immunocompromised (NSG) mice following transplantation, suggesting that EZH2 inactivation cooperates with HOXA9 or LYL1 overexpression to drive human T-cell transformation. To assess the relevance of these findings in primary human T-ALLs, we analyzed whether loss-of-function mutations in PRC2 genes (EZH2, EED, SUZ12) were enriched in specific T-ALL subgroups, particularly ETP-ALL, in a cohort of 1,335 patients from the Children's Oncology Group (COG) study. Interestingly, PRC2-mutated T-ALLs were significantly associated with the HOXA subgroup and the ETP-ALL phenotype, and exhibited higher levels of minimal residual disease (MRD) 30 days after therapy compared to other T-ALL cases. To further dissect the molecular effects of EZH2 loss and HOXA9 overexpression in ETP-like T-ALL, we knocked out EZH2 in the human Loucy and PEER cell lines using CRISPR/Cas9. These EZH2KOcells were then transduced with HOXA9-encoding lentiviruses or empty vectors and analyzed by RNA sequencing. In parallel, ChIP-seq analyses were performed to map HOXA9 binding sites and assess histone modifications affected by EZH2 loss. Integrated analysis of RNA-seq and ChIP-seq data revealed that HOXA9 expression in EZH2-deficient ETP-like cells led to the upregulation of a stem-like/immature T-cell gene signature enriched in transcriptional activators and chromatin remodelers (e.g., RUNX2, IRX3, SATB1, ARID5B, CHD1, INO80D).Furthermore, scRNA-seq profiling of primary PRC2-mutated T-ALL samples revealed distinct leukemic subsets with high HOXA9 expression and strong enrichment for the same gene signature, as determined by Gene Set Variation Analysis (GSVA). Importantly, this signature stratified PRC2-mutated T-ALLs and identified a HOXA positive subgroup associated with poor prognosis.
 Collectively, these findings uncover a critical epigenetic circuit involving EZH2 loss and HOXA9-driven transcriptional reprogramming that promotes stem-like, therapy-resistant T-ALL subsets and highlights novel vulnerabilities that could be therapeutically exploited to eliminate high-risk disease at its origin.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d423907fad1e15df50c67eaeeaa6ff45c0caebf" target='_blank'>
              EZH2 inactivation cooperates with HOXA9 to induce leukemic transformation of human t-cell progenitors and sustain high-risk subsets of early t-cell precursor acute lymphoblastic leukemia (ETP-ALL)
              </a>
            </td>
          <td>
            M. Colucci, Costanzo Padovano, Gaja Bruno, E. De Santis, Serena di Iasio, Chiara Di Nunzio, Andrew Weng, Vincenzo Giambra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca6176cd27da7f6583937f65a3ed03caa145ef0" target='_blank'>
              Overexpressed PRR11-SKA2-miR301a/454 bidirectional transcription unit essentially and coordinately promotes PI3K-AKT pathway activation and lung cancer progression.
              </a>
            </td>
          <td>
            Tao Liu, Xiaofeng Zuo, Shijie Sun, Kailong Du, Chuntao Tao, X. Xia, Linli Yu, Chunxue Zhang, Zhengmei Yang, Yitao Wang, Junhong Ye, Youquan Bu
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH), the clonal expansion of mutant hematopoietic stem and progenitor cells (HSPCs), is highly prevalent in aging populations and is linked to age-related diseases such as cardiovascular disease and malignancy (Jaiswal et al., N. Engl. J. Med., 2014, 2017). DNA methylation regulators, Ten-Eleven Translocation 2 (TET2) and DNA methyltransferase 3A (DNMT3A), are among the most frequently mutated genes in CH (Genovese et al., N. Engl. J. Med. 2014). Recent studies report the essential role of TET2/DNMT3A in regulating DNA methylation in euchromatin particularly in the regulation of key genes involved in hematopoietic stem cell function and lineage specification. DNA methylation also maintains heterochromatin by silencing transposable elements (TEs). Our recent study reported a distinct role for DNMT3A and TET2 in regulating the spatial relocation of H3K9me3-marked heterochromatin during HSPC aging, which contributes to age-related CH (Hong et al., Nat. Aging, 2023). Our data suggested the upregulation of TEs correlated with the activation of interferon-stimulated genes (ISGs) and contributed to the functional decline of aged HSPCs. This decline was mitigated in TET2-mutant HSPCs, increasing their competitive capacity. However, given HSPC heterogeneity, the cell type specific TEs alterations during aging and in the pathogenesis of CH remain largely unknown.
 To further delineate the transcriptional and epigenetic regulation of TEs during aging, we purified Lin- HSPCs from C57BL/6 wild-type (WT) and TET2-knockout (TET2-KO) mice at 7 weeks (7w), 21 months (21m), and 30 months (30m) of age. We then performed single-cell (sc) RNA-seq and scATAC-seq. After preprocessing, cell type labeling, and integration of both scRNA and scATAC datasets, we quantified TE expression and accessibility at both the locus and family levels using MATES, an autoencoder-based methodology (Wang et al., Nat. Commun., 2024). We compared WT and TET2-KO aging by integrative scRNA/scATAC analysis of lineage biases and regulon activities.
 Coding-gene and TE expression profiles independently resolved the same HSPC progenitor subpopulations, indicating cell type specific TE expression. This strongly suggested the tissue specific regulation of TEs, consistent with previous publications regarding embryonic development (Miao, et al, Genome Biol 2020). Furthermore, scATAC-seq analysis revealed similar subpopulation specific TE trends, including distinct, smooth trajectories of TE accessibility along myeloid, lymphoid, and megakaryocyte-erythroid lineages. Integrating our 7w and 21m WT scRNA-seq and scATAC-seq, we found that age-dependent gains in TE accessibility and transcription varied across all major progenitor subpopulations. These age-dependent changes in TE accessibility and transcription occurred in all major classes of TEs including LINEs, SINEs and LTRs, but the changes were mitigated in TET2 deficient cells, strongly suggesting distinct regulatory mechanisms in TE during TET2 deficient HSPCs aging. The age-dependent changes in HSCs were strongly exemplified by many TE elements at the subfamily and locus levels in groups such as the MurERV40int and RLTR10 from the ERV class. Furthermore, we revealed a positive correlation between TE expression and inflammatory genes, suggesting the potential involvement of TEs in regulating intracellular innate immune response during aging. In addition to modulating the age-related changes in TEs, loss of TET2 mitigated age-associated chromatin opening and expression of age-related genes such as Stxbp4 and S100a6 (Svendsen et al., Blood, 2021). Additionally, a regulon-based analysis showed age-related decreases in regulons associated with HSC function and maintenance such as Meis1 (Kocabas, et al. Blood, 2012), this age-related decrease was mitigated in TET2-KO. Parallel analysis of aging human HSPC datasets uncovered similar gene and TE activation patterns, suggesting a conserved mechanism (Hua et al., Blood, 2019).
 Beyond extrinsic inflammation, age-related TE upregulation may trigger innate immune signaling that impairs HSPC self-renewal and specification. Our multi-omics approach elucidates the epigenetic and transcriptional mechanisms underlying TE dysregulation in aging HSPCs. Our study provides new insights into the molecular underpinnings of age-related hematopoietic dysfunction and identified potential therapeutic targets for mitigating CH and associated diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca589ad4135ba9b576c77a6dbfe352981eb6d90e" target='_blank'>
              Single-cell multi-omics uncovers transposable element regulation in hematopoietic stem and progenitor cells during aging
              </a>
            </td>
          <td>
            Logan Rivera, Tingting Hong, Y. Huang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The RB tumor suppressor is a key regulator of cell cycle progression that is often inactivated in triple-negative breast cancer (TNBC). Recent studies indicate that drugs activating RB have multiple tumor-suppressing effects on the tumor and the tumor microenvironment (TME). Here, we utilize a constitutively active RB protein incapable of being phosphorylated and inactivated by CDKs (RBΔCDK) to assess the intrinsic sufficiency of RB activation on tumor suppression. Expression of RBΔCDK in TNBC cell lines uniformly inhibited proliferation. Transcriptomic analysis revealed suppression of cell cycle genes and the induction of genes associated with interferon response. Similarly, tumor growth and metastasis were suppressed in RBΔCDK-expressing human xenograft and mouse syngeneic tumor models. RB activation was sufficient to dramatically alter the TME, wherein tumor growth suppression was mediated by CD8+ T cells. Together, these data indicate that active RB suppresses TNBC progression in cancer cell-autonomous and non-autonomous mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9900b7f402a23a48886cadde2caa2235a93dd00" target='_blank'>
              Intrinsic RB activation induces tumoral and stromal anti-tumor responses that limit triple-negative breast cancer
              </a>
            </td>
          <td>
            Yin Wan, Jianxin Wang, Thomas N. O’Connor, Vishnu Kumarasamy, Ioannis Sanidas, Scott I. Abrams, A. Witkiewicz, Erik S. Knudsen
          </td>
          <td>2025-12-01</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38954ab0551e2a9866f6ea57105246fb32735740" target='_blank'>
              CellSNVReg: a multidimensional resource for SNV-mediated regulatory perturbations in single-cell and spatial omics.
              </a>
            </td>
          <td>
            Jingyi Shi, Shiqian Zhang, Zhenglin Lu, Weiwei Zhou, Weijian Lu, Gengdong Chen, Jianing Li, Wenhui Liu, Jiaqi Huang, Tiantongfei Jiang, Xiyun Jin, Juan Xu, Tingting Shao
          </td>
          <td>2025-10-31</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Children with Down syndrome are predisposed to hematological malignancies due to the presence of an extra copy of chromosome 21. Trisomy 21 (T21) alone has been shown to disrupt fetal hematopoiesis. Further examination identified that the Down syndrome critical region (DSCR) on chromosome 21 plays a key role in T21-driven leukemogenesis due to gene dosage imbalances. DSCR harbors several genes essential for hematopoietic development and leukemogenesis, such as DYRK1A, one of the dosage sensitive genes. As a dual-specificity kinase, DYRK1A plays diverse roles in cell cycle progression, stem cell differentiation, and DNA damage response. Interestingly, DYRK1A has been reported to interact with RNA polymerase II at active promoters, which shows a potential role in transcriptional regulation. DYRK1A overexpression leads to disrupted hematopoiesis, yet its distinct role in ML-DS remains unclear.
 In this study we showed DYRK1A as a novel interactor of EZH2 in CMK, a ML-DS cell line, and 3 distinct relapsed ML-DS PDX samples by co-immunoprecipitation. Given that both proteins are known to function in cytoplasmic and nuclear compartments, we performed co-immunoprecipitation in these subcellular fractions and found that the interaction only occurs in the nuclear fraction in CMK and ML-DS PDX cells. Considering the canonical role of EZH2 as a methyltransferase and of DYRK1A as a dual-specificity kinase, we examined if their canonical roles are necessary for this interaction. Therefore, we treated cells with EZH2 inhibitor (GSK126) or DYRK1A inhibitor (EHT1610) and performed co-immunoprecipitation. The data demonstrated that the interaction between DYRK1A and EZH2 was maintained even in the presence of inhibitors which suggest that the interaction between these two proteins is not dependent on either methyltransferase or kinase activity.
 Since DYRK1A has been implicated in transcriptional regulation and we showed that it interacts with EZH2 in the nucleus only, we hypothesized that DYRK1A may collaborate with EZH2 to shape the chromatin landscape in ML-DS. To explore this possibility, we performed CUT&RUN chromatin profiling of EZH2, DYRK1A, H3K27me3, and H3K4me3 in CMK cells. CUT&RUN using anti-DYRK1A antibody identified 12,348 peaks (p <0.001). These peaks partially overlapped with the H3K4me3 peaks, but they were not enriched by H3K27me3 peaks. Overlapping DYRK1A peaks with EZH2 peaks revealed 186 regions occupied by both, and these regions are highly enriched by H3K4me3, suggesting active gene expression. Pathway enrichment analysis of DYRK1A and EZH2 co-occupied regions indicated enrichment in key signal transduction pathways, including MAPK, ERK, RAS, and neurotrophin (NTRK) signaling (p < 0.05).
 To further investigate the function of EZH2 and DYRK1A in ML-DS, we depleted the expression of EZH2 or DYRK1A in CMK cells by CRISPR-cas9 mediated knockout. While depletion of EZH2 resulted in decreased protein level of DYRK1A in the nucleus, it had no effect on cytoplasmic DYRK1A. Interestingly, the downregulation of DYRK1A protein did not affect the level of EZH2 in both fractions. These data indicate that DYRK1A requires EZH2 for its nuclear localization and likely regulates expression of specific genes required for leukemogenesis.
 Together, these findings suggest that DYRK1A and EZH2 may functionally cooperate to regulate gene expression at transcriptionally active regions in ML-DS cells, especially within key signal transduction networks. This interaction may represent a previously unrecognized mechanism contributing to leukemogenesis in T21, with potential implications for targeting non-canonical EZH2 functions in ML-DS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcb96ecddff82ebd51607a31bc1f76845786142" target='_blank'>
              DYRK1A interacts with EZH2 to regulate transcriptionally active chromatin in myeloid leukemia associated with down syndrome
              </a>
            </td>
          <td>
            Kader Cicek, S. Barwe, Anilkumar Gopalakrishnapillai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract Background The human mitochondrial genome (mtDNA) plays a key role in maintaining cellular functions. However, mtDNA is vulnerable to mutations due to the lack of protective histones and low efficiency in DNA repair mechanisms. Among these mutations, the 4977 bp mtDNA common deletion (mtDNA-CD) encompasses twelve essential genes that are essential for oxidative phosphorylation. Aim In previous studies, mtDNA-CD was identified in nasopharyngeal carcinoma (NPC) patient samples. However, bulk sequencing protocols employed in past studies were unable to distinguish whether those mtDNA-CD were specific to the cancer cells or from other non-cancer cell types in the samples. Single-cell RNA sequencing (scRNA-seq) enables the discovery of different cell types within a population and facilitates the identification of cell-type-specific mutation signatures. In this study, we propose to use scRNA-seq data to identify cell-type-specific deletion signatures in NPC samples. Methods We developed a custom computational pipeline to detect large-scale deletions using scRNA-seq data. The pipeline harnesses both clipped read profiles and secondary alignments to overcome the typical limitations of low sequencing depth and uneven read coverage in scRNA-seq data. We tested our pipeline on an NPC data set consisting of nine patients. Results Our result suggests the presence of mtDNA-CD mutations specific to malignant cancer cells in NPC tumour samples. This study proposes the use of scRNA-seq of cell-type-specific mtDNA-CD in NPC samples, offering new insights into the genetic and molecular pathogenesis of cancer and guiding the development of specific biomarkers for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9ee5d5ecd8271f6680610b609d3e9e7cc092eb" target='_blank'>
              Discovery of cell-type-specific mtDNA common deletion in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Mun Kay Ho, Joshua W K Ho
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification has emerged as a critical epigenetic mechanism regulating gene expression in diverse physiological and pathological processes, including cancer. Methyltransferase-like 16 (METTL16), a recently identified m6A methyltransferase, has been shown to influence tumor progression through m6A-dependent regulation of key target genes. Accumulating evidence indicates that METTL16 exerts tumor-suppressive or tumor-promoting roles in a context-dependent manner, affecting cell proliferation, apoptosis, autophagy, and chemotherapeutic response across multiple cancer types such as bladder cancer, lung cancer, colorectal cancer, and acute myeloid leukemia. Mechanistically, METTL16 modifies the mRNA stability and translation of oncogenes or tumor suppressors via recognition of m6A sites, and its expression can be regulated by upstream factors including transcription factors and hypoxia-inducible signals. Recent evidence suggests that METTL16 also modulates the tumor microenvironment (TME), potentially affecting immune cell infiltration, immune checkpoint expression, and tumor immune evasion. Collectively, METTL16 emerges as a pivotal epitranscriptomic regulator linking RNA modification, tumor progression, and immune modulation, offering new avenues for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cac623df0454861cf59d7b80f15a7189f66d09" target='_blank'>
              METTL16 emerges as a pivotal epitranscriptomic regulator, linking RNA modification, tumor progression, and immune modulation
              </a>
            </td>
          <td>
            Qiang Wang, Xiulin Jiang, Yixiao Yuan, Chunhong Li
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The adenoma-adenocarcinoma pathway represents a crucial mechanism underlying the development of colorectal precancerous lesions, encompassing approximately 85%-90% of colorectal cancer (CRC). Elucidating the molecular mechanisms underlying colorectal cancer progression is of paramount importance for achieving early and accurate diagnosis as well as effective treatment. We collected peripheral blood mononuclear cells (PBMC) from healthy controls, adenoma patients, and adenocarcinoma patients, and performed transcriptomic profiling to characterize dynamic gene expression during carcinogenesis. Diagnostic potential was assessed using receiver operating characteristic (ROC) analysis, and a random-forest model was trained to classify disease status. The screening identified genes with consistent expression changes as potential early diagnostic markers, and further exploration of their functions and significance in CRC is conducted through analysis of the TCGA database. The findings revealed that the progression of precancerous lesions in the “Normal-Adenoma-Cancer” (N-A-C) sequence was accompanied by a sustained enhancement of the immune response. Notably, HECW2, WARS1, SLC16A3, SECTM1, IFITM3, ADAMTSL4, FCGR1A, F2RL1, OPLAH, SERPINA1, FCGR1CP showed consistent upregulation with promising diagnostic performance. In our PBMC cohort, the random-forest classifier achieved an accuracy of 93.62%, indicating potential for distinguishing cancer from precancerous lesions. The bioinformatics analysis revealed a significant association of these genes with DNA methyltransferase, DNA mismatch repair, m6A regulator, tumor mutational burden (TMB) and microsatellite instability (MSI). Furthermore, a detailed analysis was further performed on WARS1. In the “N-A-C” sequence, WARS1 exhibited a significant upregulation in both blood and tissues, demonstrating a positive correlation with augmented infiltration of immune cells, activation of stromal and immune responses, as well as heightened activity during the cancer immune cycle. However, it demonstrates a declining trend in the progression of CRC from stage I to IV, which may be intricately associated with the metastasis of CRC. The WARS1 can serve as a reliable indicator of the immune response in CRC, thereby demonstrating its potential to impede tumorigenesis or metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa1e73dcc984320ef956ec066c98a00971bc8f16" target='_blank'>
              Progressively altered genes in colorectal carcinogenesis link oncogenesis immune cycle and tumor microenvironment
              </a>
            </td>
          <td>
            Bingwen Zhou, Qingrui Liu, Chuyue Huang, Hao Chen, Pei Wang, Yueyang Lu, Shujun Jiang, Desong Kong, Lu Wang, Zhimin Fan
          </td>
          <td>2025-10-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Studies by our lab have demonstrated that DNA Damage Repair (DDR) pathways play a key role in determining response to ICB. To better characterize the effect of these alternations on the tumor immune microenvironment (TME) and define key determinants of ICB response, we created isogenic knockouts of BRCA2 and BRCA1 in a murine 4T1 metastatic TNBC background to assess how these tumor-intrinsic programs influence the TME and poise tumors for immunotherapy response. Differential immune landscapes identified via scRNAseq in Brca2-mutant tumors at baseline showed key differences in the myeloid compartment and interferon-stimulated gene (ISG) expression in tumor-infiltrating myeloid cells from Brca2 mutant tumors. Bulk RNAseq analysis showed increased production of T cell trafficking chemokines that could poise the tumor for response to ICB. Interferon reporter assay results suggested tumor-intrinsic cGAS drives trans-activation of myeloid STING and IFNb1. Further assessment of tumor cell lines via cellular fractionation experiments identified the presence of both DNA and R-loops in the cytoplasm of BRCA2-mutant cell lines that serve to active cGAS/STING. In vivo experiments with tumor intrinsic knockouts showed that tumor STING signaling and cytokine production were dispensable for ICB response while depletion of monocytes via CSF1R blockade showed a complete reversal of ICB response, underlining the essential role of monocytes in poising the tumor for ICB response.



 Supported by NIH Director’s Early Independence Award, the Parker Institute for Cancer Immunotherapy, Burrough’s Welcome Fund, and Mount Sinai’s Cancer Biology T32 Training Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f9189d19bc8a0280db2d343d742431c6842512" target='_blank'>
              Defining the determinants of immune response in DNA homologous recombination deficient tumors 3797
              </a>
            </td>
          <td>
            Natalie Vaninov, R. Samstein
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Rare and short-lived DNA conformations are proposed to be key drivers of mutagenesis, yet assessing their contribution to mutational signatures found in human cancers remains challenging. Here, we developed an approach that quantifies the sequence-dependent propensity to form a rare DNA conformation and compares the resulting fingerprint against cancer mutational signatures. Using 19 F NMR, we measured the propensity for the anionic Watson-Crick-like G•T − conformation across all sixteen triplet sequence contexts and discovered a striking 50-fold variation driven by suboptimal interactions between anionic thymine and its 3' neighbor. Comparing this fingerprint, and those of other rare DNA states, against the COSMIC database uncovered plausible links to mutational processes associated with exposure to damaging agents and therapies. Thus, integrating molecular biophysics with genomic epidemiology provides a powerful framework to explore how DNA’s dynamic properties shape genome stability and influence human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ebb7fd9ba5c7aaf01897f4a9869d4e5ed09b6c2" target='_blank'>
              Assessing the contribution of rare DNA states to cancer mutational signatures using sequence-specific conformational fingerprinting
              </a>
            </td>
          <td>
            Hashim M. Al-Hashimi, Or Szekely, Yeongjoon Lee, A. Rangadurai, Serafima Guseva, Joshua Cooksey, Edgar Faison, Nikita Zalenski, Qi Zhang, Zucai Suo
          </td>
          <td>2025-11-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The synthesis of human genomes and other gigabase-scale genomes will require new strategies. Here, we realized key steps in our pipeline for building synthetic human chromosomes. We established: (i) the facile transfer of human chromosomes from human cells to mouse embryonic stem cells (assembly cells), where they are haploid, are nonessential, and may be operated on; (ii) the transfer of these human chromosomes from monochromosomal hybrids back into human cells to generate defined, synthetic aneuploidies; and (iii) the elimination of the corresponding endogenous human chromosomes to regenerate diploid cells containing a transferred chromosome. All steps were performed in nontransformed cells without chromothripsis and generated minimal structural variants, insertions, deletions, or single-nucleotide variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47de2d2fbef0f5d3fc29497bcce34730dd1a05b6" target='_blank'>
              High-fidelity human chromosome transfer and elimination.
              </a>
            </td>
          <td>
            G. Petris, Simona Grazioli, L. van Bijsterveldt, Pierre Murat, Kim C. Liu, Jakob Birnbaum, Julian E. Sale, Jason W. Chin
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Background: The prognostic impact of specific genomic changes in mantle cell lymphoma (MCL) is not well characterized beyond altered TP53, which is recognized as a high-risk marker and commonly assessed at diagnosis, and the “proliferation signature” developed using gene expression in fresh frozen tissues (Rosenwald et al, Cancer Cell 2003). To bridge this knowledge gap, we applied comprehensive tumor sequencing to investigate how genomic abnormalities affect prognosis in patients (pts) with MCL with and without TP53 alterations.
 Methods: The Atlas of Blood Cancer Genomes project is an international collaborative effort including 25 sites for collecting and sequencing all blood cancers (Love et al, ASH 2021). We recruited MCL pts with detailed clinical data and subjected their tumors to whole exome, whole transcriptome, and targeted sequencing. TP53-aberrant cases (i.e., mutation or deletion) were identified from sequencing and clinical pathology reports. Progression-free survival (PFS) and overall survival (OS) were measured using the Kaplan-Meier method, with statistical comparisons by the log-rank test. Risk for a gene signature was defined as the coefficient from the respective Cox proportional hazard model.
 Results: RNA and DNA sequencing were performed successfully for 252 pts with MCL. Clinical features and treatment regimens were consistent with prior disease descriptions and have been reported previously (Koff et al, ASH 2022). In the entire cohort, median PFS was 38 months, and median OS was not reached. As expected, TP53 abnormalities were associated with inferior OS compared to TP53-wildtype (WT; 5-year OS 48% and 79%, respectively, p<0.001).
 A novel gene expression signature (“immune signature”) was determined by identifying genes that displayed prognostic ability independent of the previously described MCL proliferation signature (Rosenwald et al, Cancer Cell 2003; Scott et al, J Clinical Oncology 2017). This signature was distinct from the proliferation signature both in terms of included genes and ability to discriminate between risk groups; the correlation plot between the immune and proliferation signature scores showed a low R2 value of 0.01. The immune signature appears to derive from differences in tumor microenvironment (TME) CD8+ T and T follicular helper cells: pts designated as high-risk by the immune signature demonstrated lower proportions of these TME subsets as assessed by CIBERSORT (p=0.001). For the cohort with tumors sequenced prior to treatment (n=208), 5-year OS was 45% for pts with a high-risk immune score (bottom quartile), 80% for pts with intermediate-risk immune score (middle 2 quartiles), and 91% for low-risk immune score (top quartile, p <0.001), with improved discrimination compared to the proliferation signature (56%, 79%, and 80% for high-, intermediate-, and low-risk proliferation scores, respectively; p=0.03).
 The immune signature also risk-stratified outcomes in MCL subgroups with and without TP53 alteration. For pts with TP53-WT (n=74), low-risk immune score predicted 5-year OS of 86%, while high-risk immune score was associated with 5-year OS of 38% (p<0.001), compared to 72% vs 82% for low-risk and high-risk proliferation scores (p=0.16). For pts with aberrant TP53 (n=43), low-risk immune score had 5-year OS of 77%, and high-risk immune score had 5-year OS of 9% (p<0.001), compared to 35% vs 52% for low-risk and high-risk proliferation scores (p=0.28). Application of the immune signature also further stratified pts deemed high-risk by the proliferation score (n=52): within this group, low-risk immune score associated with 5-year OS of 76%, and high-risk immune score predicted 5-year OS of 29% (p<0.001). Similar stratification was also observed when the immune signature was applied to pts with low- and intermediate-risk proliferation scores (n=156; 5-year OS of 88% vs 57% for low-risk vs high-risk immune score respectively, p<0.001).
 Conclusions: In this largest-ever study of MCL's genomic landscape, we identify a novel gene expression signature that stratifies risk within and across existing prognostic groups, including TP53-altered cases. Our findings support development of the immune signature as a tool that can be used in routine clinical practice to improve risk stratification of all MCL patients at diagnosis. Additional study is warranted to define therapeutic implications of differential TME T cell subset composition in MCL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b502f44f0a099faad0f0968d56a96b077d77a2" target='_blank'>
              Novel immune gene expression signature risk-stratifies Mantle Cell Lymphoma outcomes independent of TP53 status
              </a>
            </td>
          <td>
            Jean Koff, Veronica S. Russell, Rachel Kositsky, David L. Jaye, M. Churnetski, K. Baird, Colin B O'Leary, C. Flowers, S. Leppä, Shaoying Li, Jessie Xu, Mette Pedersen, A. O. Gang, K. Naresh, R. Au-Yeung, Jennifer Chapman, Amy Chadburn, I. Lossos, Sarah Ondrejka, A. Louissaint, Eric Tse, C. Love, D. Thakkar, Clay Parker, C. Ong, Andrew Evans, E. F. Mason, Yan Jiong, Magdalena Czader, A. Evens, D. Soliman, Y. Fedoriw, Sandeep Dave, Jonathon Cohen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>68</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>