<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-05 09:28:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>33</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>68</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>111</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>329</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>290</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a common source of oncogene amplification across many types of cancer. The non-Mendelian inheritance of ecDNA contributes to heterogeneous tumour genomes that rapidly evolve to resist treatment. Here, using single-cell and live-cell imaging, single-micronucleus sequencing, and computational modelling, we demonstrate that elevated levels of ecDNA predisposes cells to micronucleation. Damage on ecDNA, commonly arising from replication stress, detaches ecDNA from the chromosomes upon which they hitchhike during cell division, thereby causing micronucleus formation in daughter cells. Clusters of oncogene-containing, CIP2A-TOPBP1-associated ecDNA molecules form, and asymmetrically segregate into daughter cell micronuclei during cell division. ecDNA chromatin remains highly active during mitosis, but upon micronucleation, it undergoes suppressive chromatin remodeling, largely ceasing oncogene transcription. These studies provide insight into the fate of damaged ecDNA during cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, , Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f629203ff4e6d119bd3d8d7b5f0a9d082c7f87e7" target='_blank'>
              Abstract 5046: Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, W.J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Aneuploidy is observed as gains or losses of whole chromosomes or chromosome arms and is a common hallmark of cancer. Whereas models for the generation of aneuploidy in cancer invoke mitotic chromosome segregation errors, whole-arm losses might occur simply as a result of centromere breakage. We recently showed that elevated RNA Polymerase II (RNAPII) level over S-phase-dependent histone genes predicts rapid recurrence of human meningioma and is correlated with total whole-arm losses relative to gains. To explain this imbalance in arm losses over gains, we have proposed that histone overexpression at S-phase competes with the histone H3 variant CENP-A, resulting in centromere breaks and whole-arm losses. To test whether centromere breaks alone can drive aneuploidy, we ask whether total whole-arm aneuploids can predict outcome across different cancer types in large RNA and whole-genome sequencing databanks. We find that total whole-arm losses generally predict outcome, suggesting that centromere breakage is a major initiating factor leading to aneuploidy and the resulting changes in the selective landscape that drive most cancers. We also present evidence that centromere breakage alone is sufficient to account for whole-arm losses and gains, contrary to mitotic spindle error models for generation of aneuploidy. Our results suggest that therapeutic intervention targeting histone overexpression has the potential of reducing aneuploidy and slowing cancer progression. Significance Statement Gain or loss of whole chromosome arms following centromere breaks is frequent in cancer, but whether or not there is a common initiating event is unknown. Here we show that the total number of whole-arm losses predicts patient outcomes across cancer types, suggesting a causal relationship. This general excess of losses over gains is not predicted by mitotic error models of aneuploidy but rather suggests that centromere breaks themselves initiate whole-arm aneuploidies. Insofar as aneuploidy reshapes the selective landscapes that drive most cancers, our results have potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c0dc401cc26acc6c4f2c1eb53245c79aaf98cd8" target='_blank'>
              Total whole-arm chromosome losses predict malignancy in human cancer
              </a>
            </td>
          <td>
            Ye Zheng, K. Ahmad, S. Henikoff
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7372198dd7ad93e0ac09354eea55bbe4096844" target='_blank'>
              Human Papillomavirus Genomes Associate with Active Host Chromatin during Persistent Viral Infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7809d316520a84fae49c8004779285724bae28b" target='_blank'>
              Analysis of regulatory mechanisms of enhancers in gastric cancer with double minute chromosomes based on bioinformatics.
              </a>
            </td>
          <td>
            Meng-Ting An, Guan-Lin Guo, Jie Wu, Wen-Jing Sun, 
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f22aef6f2b3fccea5561ffb5ccb8e4b8470eaaa" target='_blank'>
              Hi-C for genome-wide detection of enhancer-hijacking rearrangements in routine lymphoid cancer biopsies
              </a>
            </td>
          <td>
            Jamin Wu, Shih-Chun A. Chu, Jang Cho, Misha Movahed-Ezazi, Kristyn Galbraith, C. Fang, Yiying Yang, Chanel Schroff, Kristin Sikkink, Michelle Perez-Arreola, Logan Van Meter, Savanna Gemus, Jon-Matthew Belton, Valentina Nardi, Abner Louissant, Dennis Shasha, A. Tsirigos, Noah A Brown, Tatyana Gindin, Marcin P. Cieslik, Minji Kim, Anthony D Schmitt, Matija Snuderl, Russell J H Ryan
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Intratumour heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance1,2. Oncogene dosage variation contributes to cell-state transitions and phenotypic heterogeneity3, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNAs drive varying levels of MYC dosage, depending on their regulatory landscape, enabling cancer cells to rapidly and reversibly adapt to microenvironmental changes. In the absence of selective pressure, a high ecDNA copy number imposes a substantial fitness cost on PDAC cells. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors. Our work provides a detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>2</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc027322f110679344e9de5725b902edc43f13e" target='_blank'>
              Deregulated enhancer-promoter communication in cancer through altered nuclear architecture.
              </a>
            </td>
          <td>
            Isabelle Seufert, Claire Vargas, Sina Jasmin Wille, K. Rippe
          </td>
          <td>2025-04-12</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ff813478f3cbaef070da0522b428d7b65243ab" target='_blank'>
              Nucleotide insufficiency induced by p53 deficiency leads to replication stress driving genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, A. Eliassaf, Karen Grimes, M. Irony-Tur Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a97fcf511bf3ac6d39959b0d32a3431fd3b9263" target='_blank'>
              Abstract 7504: Integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH-wildtype integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH wild-type glioblastoma
              </a>
            </td>
          <td>
            Dacheng Zhao, K. Anderson, Amit Gujar, Megan L Costa, Djamel Nehar-Belaid, F. Varn, Elise T. Courtois, Paul Gabrie, Paul Robson, H. Moon, A. Gołȩbiewska, S. Paek, S. Niclou
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad9d941478590c32e5cf0763f3872cb1fd2c287" target='_blank'>
              Satellite DNA Amplification in Advanced Prostate Cancer Is Largely Independent From Euchromatic and Oncogene Amplicons.
              </a>
            </td>
          <td>
            Anja Weise, A. Ornellas, Gilda Alves, Constanze Pentzold, Jenny Holler, Melanie Wolter, Elena Jamali, Bernhard Theis, Thomas Liehr
          </td>
          <td>2025-03-17</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, Katie Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, Markus Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="NOTCH2NL (NOTCH2-N-terminus-like) genes arose from incomplete, recent chromosome 1 segmental duplications implicated in human brain cortical expansion. Genetic characterization of these loci and their regulation is complicated by the fact they are embedded in large, nearly identical duplications that predispose to recurrent microdeletion syndromes. Using nearly complete long-read assemblies generated from 67 human and 12 ape haploid genomes, we show independent recurrent duplication among apes with functional copies emerging in humans ∼2.1 million years ago. We distinguish NOTCH2NL paralogs present in every human haplotype (NOTCH2NLA) from copy number variable ones. We also characterize large-scale structural variation, including gene conversion, for 28% of haplotypes leading to a previously undescribed paralog, NOTCH2tv. Finally, we apply Fiber-seq and long-read transcript sequencing to human cortical neurospheres to characterize the regulatory landscape and find that the most fixed paralogs, NOTCH2 and NOTCH2NLA, harbor the greatest number of paralog-specific elements potentially driving their regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57156ab124ecd643544fdfa8cdaaa9cc74e49805" target='_blank'>
              Genetic diversity and regulatory features of human-specific NOTCH2NL duplications
              </a>
            </td>
          <td>
            Taylor Real, Prajna Hebbar, DongAhn Yoo, F. Antonacci, Ivana Pačar, M. Diekhans, Gregory J. Mikol, Oyeronke G. Popoola, Ben Mallory, Mitchell R. Vollger, Philip C. Dishuck, Xavi Guitart, Allison N. Rozanski, Katherine M. Munson, Kendra Hoekzema, Jane Ranchalis, Shane Neph, Adriana E. Sedeño-Cortés, B. Paten, Sofie R. Salama, A. Stergachis, E. Eichler
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8275563011d6ab6090a01c6b82a9c990741f5" target='_blank'>
              Z-DNA formation induces the totipotent-like state and primes Zscan4-dependent chromatin compartmentalization
              </a>
            </td>
          <td>
            Shireen Shajahan, Yann Loe-Mie, Marion Salmon-Legagneur, Tatiana Traboulsi, Agnès Thierry, Anne Dejean, J. Cossec
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer is fundamentally a disease of the genome, characterized by extensive genomic, transcriptomic, and epigenomic alterations. Most current studies predominantly use short-read sequencing, gene panels, or microarrays to explore these alterations; however, these technologies can systematically miss or misrepresent certain types of alterations, especially structural variants, complex rearrangements, and alterations within repetitive regions. Long-read sequencing is rapidly emerging as a transformative technology for cancer research by providing a comprehensive view across the genome, transcriptome, and epigenome, including the ability to detect alterations that previous technologies have overlooked. In this Perspective, we explore the current applications of long-read sequencing for both germline and somatic cancer analysis. We provide an overview of the computational methodologies tailored to long-read data and highlight key discoveries and resources within cancer genomics that were previously inaccessible with prior technologies. We also address future opportunities and persistent challenges, including the experimental and computational requirements needed to scale to larger sample sizes, the hurdles in sequencing and analyzing complex cancer genomes, and opportunities for leveraging machine learning and artificial intelligence technologies for cancer informatics. We further discuss how the telomere-to-telomere genome and the emerging human pangenome could enhance the resolution of cancer genome analysis, potentially revolutionizing early detection and disease monitoring in patients. Finally, we outline strategies for transitioning long-read sequencing from research applications to routine clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing
              </a>
            </td>
          <td>
            Qiuhui Li, A. Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome Research</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b54c36ea8dc8ae6d831b9cec1f08634da1f520c" target='_blank'>
              Abstract 1152: The landscape of copy number variations following WGD in 5,980 colorectal cancers
              </a>
            </td>
          <td>
            Jierui Xu, H. Walch, Paolo Manca, Maria A Perry, Christopher Fong, Justin Jee, K. Pichotta, E. Vakiani, Neil H. Segal, Adam Price, Allison L Richards, Chai Bandlamudi, M. Donoghue, A. Cercek, R. Yaeger, N. Schultz, W. Chatila
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9dfba50807f4d056e7d436de2c34ed7ddd425f" target='_blank'>
              Abstract 1502: Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx
              </a>
            </td>
          <td>
            Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, C. Swanton, M. Jamal-Hanjani, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) are circular, double-stranded DNA molecules ubiquitously present across various organisms, playing a critical role in tumorigenesis and tumor progression. However, their precise contribution to prostate cancer (PCa) remains incompletely understood. To elucidate the function of eccDNAs in PCa, eccDNA sequencing and annotation were performed in PCa tissues and cell lines using Circle-seq. Amplified genes on eccDNAs were identified by cross-referencing annotated eccDNA-associated genes with those overexpressed in PCa based on TCGA data. Furthermore, eccDNA profiles were compared between Enzalutamide-sensitive and -resistant cell lines to investigate their role in resistance mechanisms. Notably, VAV2 was detected on both linear and circular DNA, as confirmed by PCR and Sanger sequencing. Functional analyses demonstrated that VAV2 overexpression promotes PCa proliferation and metastasis by activating the PAK1/AKT signaling pathway through PAK1 phosphorylation. Additionally, VAV2 contributes to Enzalutamide resistance by enhancing AR/ARv7 protein stability via reduced ubiquitination, mediated through the recruitment of the deubiquitinating enzyme USP48. These findings establish VAV2, identified through eccDNA sequencing, as a potential oncogene and a promising biomarker for PCa diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin’an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22caf0363586789d3ef13130aa7bdb69bf876e9" target='_blank'>
              Nuclear and genome dynamics underlying DNA double-strand break repair.
              </a>
            </td>
          <td>
            I. Chiolo, Matthias Altmeyer, G. Legube, Karim Mekhail
          </td>
          <td>2025-03-17</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy J. Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60e85c34b84d3ee7bcc4641260225bbd7766477" target='_blank'>
              Abstract 6350: Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing
              </a>
            </td>
          <td>
            F. D’Angelo, A. Ko, L. Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, S. Williams, S. Gultekin, Ashish Shah, Ricardo Komotar, Michael Ivan, Corneliu Sologon, A. Laurenge, A. Picca, Amel Dridi-Aloulou, B. Mathon, Franck Bielle, Stephan Zuchner, M. D. L. Fuente, M. Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, A. Lasorella, A. Iavarone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland W. Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A. Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3af6173f4bd387223a85f81701f7e3b9e33817" target='_blank'>
              Abstract 1241: Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies
              </a>
            </td>
          <td>
            P. Itagi, Samantha L. Schuster, Sonali Arora, Patricia Galipeau, Thomas W Persse, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Hung-Ming Lam, Andrew C. Hsieh, Gavin Ha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replicative stress (RS) is emerging as a promising therapeutic target in oncology, yet full exploitation of its potential requires a detailed understanding of the mechanisms and genes involved. Here, we investigated the RNA helicase Senataxin (SETX), an enzyme that resolves RNA-DNA hybrids and R-loops, to address its role in preventing RS by oncogenic Myc. Upon Myc activation, silencing of SETX led to selective engagement of the DNA damage response (DDR) and robust cytotoxicity. Pharmacological dissection of the upstream kinases regulating the DDR uncovered a protective role of the ATR pathway, that once inhibited, boosted SETX driven-DDR. While SETX loss did not lead to a genome-wide increase of R-loops, mechanistic analyses revealed enhanced R-loops localized at DDR-foci and newly replicated genomic loci, compatible with a selective role of SETX in resolving RNA-DNA hybrids to alleviate Myc-induced RS. Genome-wide mapping of DNA double-strand breaks confirmed that SETX silencing exacerbated DNA damage at transcription-replication conflict (TRC) regions at early replicated sites. We propose that SETX prevents Myc-induced TRCs by resolving transcription-associated R-loops that encounter the replisome. The identification of SETX as a genetic liability of oncogenic Myc opens up new therapeutic options against aggressive Myc-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6be55a5ef5db321d17ba7bfe59475decf1a2ca" target='_blank'>
              Abstract 2855: CDK12 inhibition remodels DNA replication to drive selective gene amplification
              </a>
            </td>
          <td>
            Shivani Yalala, Julia Kutz, Helmut Pospiech, Harri M Itkonen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac35aaec3d88c6b3cd4be700d6af5bf25aaebdef" target='_blank'>
              Drosophila SA1 expression prevents brain tumorigenesis and PARP-mediated cell elimination
              </a>
            </td>
          <td>
            Simona Totaro, Antonella Lettieri, Silvia Castiglioni, Francesco Lavezzari, C. Gervasini, Valentina Massa, Thomas Vaccari
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Stem cells have the unique ability to self-renew and differentiate into specialized cell types. Epigenetic mechanisms, including histones and their post-translational modifications, play a crucial role in regulating programs integral to a cell’s identity, like gene expression and DNA replication. However, the transcriptional, chromatin, and replication timing profiles of adult stem cells in vivo remain poorly understood. Containing germline stem cells (GSCs) and somatic cyst stem cells (CySCs), the Drosophila testis provides an excellent in vivo model for studying adult stem cells. However, the small number of stem cells and cellular heterogeneity of this tissue have limited comprehensive genomic studies. In this study, we developed cell type-specific genomic techniques to analyze the transcriptome, histone modification patterns, and replication timing of GSC-like and CySC-like cells. Single cell RNA-sequencing validated previous findings on GSC-CySC intercellular communication and revealed high expression of chromatin regulators in GSCs. To characterize chromatin landscapes, we developed a cell-type-specific chromatin profiling assay to map H3K4me3-, H3K27me3-, and H3K9me3-enriched regions, corresponding to euchromatic, facultative heterochromatic, and constitutive heterochromatic domains, respectively. Finally, we determined cell type-specific replication timing profiles, integrating our in vivo datasets with published data using cultured cell lines. Our results reveal that GSCs display a distinct replication program compared to somatic lineages, that aligns with chromatin state differences. Collectively, our integrated transcriptomic, chromatin, and replication datasets provide a comprehensive framework for understanding genome regulation differences between these in vivo stem cell populations, demonstrating the power of multi-omics in uncovering cell type-specific regulatory features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db91bf6625baf7a6f5b436dd4b6c6b57e16b4fc" target='_blank'>
              Cell type- and chromosome-specific chromatin landscapes and DNA replication programs of Drosophila testicular stem cells
              </a>
            </td>
          <td>
            Jennifer A. Urban, Daniel Ringwalt, John M. Urban, Wingel Xue, Ryan J Gleason, Keji Zhao, Xin Chen
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3537c6c5299b945e50a679b1495982f083cd02" target='_blank'>
              Abstract 5359: Multi-omics characterization of copy number variation in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yuxin Jin, Jing Qian, Michelle G. Webb, David W. Craig, Lynda D. Roman, John D. Carpten, R. Bassiouni
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, P. Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. The ribosomal DNA (rDNA) region is particularly vulnerable to such changes due to its repetitive nature. Here, we demonstrate that Rad27/FEN-1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining rDNA stability. The production of extrachromosomal rDNA circles and severe chromosomal rDNA instability are observed in the rad27Δ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates unprocessed Okazaki fragments in the rDNA region, without inducing DSB formation. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, we show that Exonuclease 1 and PCNA can compensate for the loss of Rad27 function in the rDNA stabilization. These findings highlight the importance of proper Okazaki fragment processing in preventing non-DSB-induced rDNA copy number changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35141ab23e752c4a99955ef6952f7857bccfbbe" target='_blank'>
              Rad27/FEN1 prevents accumulation of unprocessed Okazaki fragments and ribosomal DNA copy number changes
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841194928a5faee8a089ace06821025bbd9fd89a" target='_blank'>
              Abstract LB347: Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, B. Lawson, R. Hillman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68e6681ead255204363cfd6f85f96a1a0227012" target='_blank'>
              Genome-wide modeling of DNA replication in space and time confirms the emergence of replication specific patterns in vivo in eukaryotes
              </a>
            </td>
          <td>
            Dario D’Asaro, J. Arbona, Vinciane Piveteau, Aurèle Piazza, Cédric Vaillant, Daniel Jost
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a clinically challenging and molecularly heterogenous breast cancer subgroup. Here, we investigate the DNA methylation landscape of TNBC. By analyzing tumor methylome profiles and accounting for the genomic context of CpG methylation, we divide TNBC into two epigenetic subtypes corresponding to a Basal and a non-Basal group, in which characteristic transcriptional patterns are correlated with DNA methylation of distal regulatory elements and epigenetic regulation of key steroid response genes and developmental transcription factors. Further subdivision of the Basal and non-Basal subtypes identifies subgroups transcending genetic and proposed TNBC mRNA subtypes, demonstrating widely differing immunological microenvironments, putative epigenetically-mediated immune evasion strategies, and a specific metabolic gene network in older patients that may be epigenetically regulated. Our study attempts to target the epigenetic backbone of TNBC, an approach that may inform future studies regarding tumor origins and the role of the microenvironment in shaping the cancer epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704abc40683ff045fcf6f261937f8b6b3f4d7c59" target='_blank'>
              The DNA methylation landscape of primary triple-negative breast cancer
              </a>
            </td>
          <td>
            Mattias Aine, Deborah F. Nacer, E. Arbajian, Srinivas Veerla, Anna Karlsson, J. Häkkinen, Henrik J Johansson, Frida Rosengren, J. Vallon-Christersson, Åke Borg, J. Staaf
          </td>
          <td>2025-03-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b3cddb316bdce4c6828dd308d722612ae6e4f52" target='_blank'>
              Abstract 1070: siderGrimoire: a database of retroCNVs in human tumors and populations
              </a>
            </td>
          <td>
            T. L. Miller, Helena Beatriz da Conceição, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C-and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2792ac282d27759d5d74152b7ef668d21c563cc" target='_blank'>
              CDK2 Inhibition Produces a Persistent Population of Polyploid Cancer Cells.
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Samuel C Okpechi, Blessing Ogunlade, Y. Alfaro-Mora, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-15</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract The DNA-dependent protein kinase (DNA–PK) complex plays a critical role in nonhomologous end-joining (NHEJ), a template-independent pathway for repairing DNA double-strand breaks (DSBs). The association of Ku70/80 with DSB ends facilitates the assembly of the DNA–PK holoenzyme. However, key mechanisms underlying the attachment and stabilization of DNA–PK at broken DNA ends remain unclear. Here, we identify PRRX1, a homeodomain-containing protein, as a mediator of chromatin localization and subsequent activation of DNA–PK. PRRX1 oligomerizes to simultaneously bind to double-strand DNA and the SAP (SAF-A/B, Acinus, and PIAS) domain of Ku70, thereby enhancing Ku anchoring at DSBs and stabilizing DNA–PK for efficient NHEJ repair. Reduced expression or pathogenic mutations of PRRX1 are associated with genomic instability and impaired NHEJ repair. Furthermore, a peptide that disrupts PRRX1 oligomerization compromises NHEJ efficiency and reduces cell survival following irradiation. These findings provide new insights into the activation of the NHEJ machinery and offer potential strategies for optimizing cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a315c496cdecaadb48e60963bd147f371cc04d" target='_blank'>
              Homeodomain protein PRRX1 anchors the Ku heterodimers at DNA double-strand breaks to promote nonhomologous end-joining
              </a>
            </td>
          <td>
            Yan Wang, Fuyuan Shen, Chen Zhao, Jiali Li, Wen Wang, Yamu Li, Jia Gan, Haojian Zhang, Xuefeng Chen, Qiang Chen, Fangyu Wang, Ying Liu, Yan Zhou
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce720a5b79eb9fb996bed883c2b0fde9438b7fa" target='_blank'>
              Genome evolution in parthenogenetic nematodes shaped by chromosome rearrangements and rare sex
              </a>
            </td>
          <td>
            George Chung, Lewis Stevens, Xiaoguang Dai, Scott Hickey, K. Kiontke, Priyesh Rughani, M. Kieninger, , S. Juul, Fabio Piano, David H.A. Fitch, Mark Blaxter, Kristin C. Gunsalus
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome regulation relies on complex and dynamic interactions between DNA and proteins. Recently, powerful methods have emerged that leverage third-generation sequencing to map protein-DNA interactions genome-wide. For example, Directed Methylation with Long-read sequencing (DiMeLo-seq) enables mapping of protein-DNA interactions along long, single chromatin fibers, including in highly repetitive genomic regions. However, DiMeLo-seq involves lossy centrifugation-based wash steps that limit its applicability to many sample types. To address this, we developed DiMeLo-cito, a single-tube, wash-free protocol that maximizes the yield and quality of genomic DNA obtained for long-read sequencing. This protocol enables the interrogation of genome-wide protein binding with as few as 100,000 cells and without the requirement of a nuclear envelope, enabling confident measurement of protein-DNA interactions during mitosis. Using this protocol, we detected strong binding of CTCF to mitotic chromosomes in diploid human cells, in contrast with earlier studies in karyotypically unstable cancer cell lines, suggesting that CTCF “bookmarks” specific sites critical for maintaining genome architecture across cell divisions. By expanding the capabilities of DiMeLo-seq to a broader range of sample types, DiMeLo-cito can provide new insights into genome regulation and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69924e2da138038fc52ccdade72f6d918fe9612d" target='_blank'>
              DiMeLo-cito: a one-tube protocol for mapping protein-DNA interactions reveals CTCF bookmarking in mitosis
              </a>
            </td>
          <td>
            Nathan Gamarra, Cy Chittenden, Kousik Sundararajan, Jacob P. Schwartz, Sofia Lundqvist, Denise Robles, Oberon Dixon-Luinenburg, Jeremy Marcus, Annie Maslan, J. M. Franklin, Aaron M. Streets, A. Straight, Nicolas Altemose
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In vivo genetic diversifiers have previously enabled efficient searches of genetic variant fitness landscapes for continuous directed evolution. However, existing genomic diversification modalities for mammalian genomic loci exclusively rely on deaminases to generate transition mutations within target loci, forfeiting access to most missense mutations. Here, we engineer CRISPR-guided error-prone DNA polymerases (EvolvR) to diversify all four nucleotides within genomic loci in mammalian cells. We demonstrate that EvolvR generates both transition and transversion mutations throughout a mutation window of at least 40 bp and implement EvolvR to evolve previously unreported drug-resistant MAP2K1 variants via substitutions not achievable with deaminases. Moreover, we discover that the nickase’s mismatch tolerance limits EvolvR’s mutation window and substitution biases in a gRNA-specific fashion. To compensate for gRNA-to-gRNA variability in mutagenesis, we maximize the number of gRNA target sequences by incorporating a PAM-flexible nickase into EvolvR. Finally, we find a strong correlation between predicted free energy changes underlying R-loop formation and EvolvR’s performance using a given gRNA. The EvolvR system diversifies all four nucleotides to enable the evolution of mammalian cells, while nuclease and gRNA-specific properties underlying nickase fidelity can be engineered to further enhance EvolvR’s mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4485500132f714892fcf79f15fa49a1b187bbe0" target='_blank'>
              Nickase fidelity drives EvolvR-mediated diversification in mammalian cells
              </a>
            </td>
          <td>
            Juan E Hurtado, Adam J. Schieferecke, Shakked O Halperin, John Guan, Dylan Aidlen, David V. Schaffer, J. Dueber
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Micronuclei originate from DNA damage generated by clastogenic and/or by aneugenic effects. Depending on the pattern of damage, they may have distinct genomic origin and composition. Sequences of the centromere, telomere and rDNA have been identified in plant micronuclei. However, other DNA sequences may also be present in the micronuclei, as well as their DNA contents may be different. Here, we investigate the DNA content, genomic composition and origin of micronuclei induced in Zea mays by methyl methanesulfonate (MMS). DNA contents showed a wide range of distribution, suggesting their diverse genomic origins and illustrating how much of the nuclear genome can be lost due to mutagen effects. Micronuclei diversity was also evidenced by in situ probing with different DNA sequences (5S and 18S rDNAs, 180-bp knob and Grande LTR-retrotransposon) and by 6-diamidino-2 phenylindole (DAPI) fluorochrome. Perhaps these sequences are hotspots for MMS damage, especially the Grande LTR-retrotransposon, 5S and 18S rDNAs, which are rich in guanine. In addition, probe pools were constructed from individual genomic DNA of two microdissected micronuclei. These probe pools hybridized on all Z. mays chromosomes. However, the centromere, knob and secondary constriction were hybridized by only one probe pool, evidencing the distinct genomic composition of the micronuclei. We illustrate the micronuclei genomic diversity as they originated from several different chromosomes following the MMS treatment, and demonstrate the extent of the genotoxic damage to the genome. We provide some insights into micronuclei structure and diversity, and show that they can be further explored in mutagenesis research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31c4f2a6572ceed6e9b3fda5f1cc0379119cd48" target='_blank'>
              DNA content, repeatome composition and origin of the Zea mays micronuclei
              </a>
            </td>
          <td>
            Nathália Vállery Tostes, Marcos Vitor Rosa Ferreira, Fernanda Aparecida Ferrari Soares, J. C. Silva, L. Bhering, W. Clarindo
          </td>
          <td>2025-04-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb48bc35dc30e8efcbd8268b117fd6382f30da8" target='_blank'>
              Abstract 2395: The role of exonized mobile elements in chimeric transcript emergence in glioblastoma
              </a>
            </td>
          <td>
            Rafael L. V. Mercuri, T. L. Miller, Filipe F. dos Santos, M. D. de Lima, Aline Rangel-Pozzo, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b525ba6493ef6d26b4e42891bed37752a529ff9f" target='_blank'>
              Rewiring the fusion oncoprotein EWS/FLI1 in Ewing sarcoma with bivalent small molecules
              </a>
            </td>
          <td>
            Michael J. Bond, Ryan P Golden, Giulia Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, N. Gray, K. Stegmaier
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a85f979d0564ed820500eb8776afbf0e35e1acb" target='_blank'>
              Wnt10b signaling regulates replication stress-induced chromosomal instability in human cancer
              </a>
            </td>
          <td>
            A. Haas, Friederike Wenz, Janine Wesslowski, Gary Davidson, O. Voloshanenko, Michael Boutros, Sergio Acebron, Holger Bastians
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94444bc68fc048c7923ba3b90250650c889cfa29" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a9bc7f4f8a4cf97f32bbe5a9d88c7eccc61afe" target='_blank'>
              Abstract 2923: Targeting DNA repair mechanisms to overcome CIC::DUX4 undifferentiated sarcomas
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The Mitotic Surveillance Pathway (MSP) monitors the duration of M-phase. Prolonged mitosis, caused by spindle attachment defects or microtubule-targeting drugs such as the taxane paclitaxel, induces the formation of the ternary “mitotic stopwatch” complex consisting of 53BP1, USP28, and p53. This event protects p53 from degradation, resulting in cell cycle arrest or apoptosis in daughter cells. In paclitaxel-resistant cancers, cells bypass the MSP, enabling unchecked proliferation and survival, although the underlying mechanisms remain unknown. Here, we demonstrate that 53BP1 physically interacts with GMCL1 but not its paralog, GMCL2, and we mapped the interaction regions on both proteins. CRL3GMCL1 functions as a ubiquitin ligase that targets 53BP1 for degradation during M phase, impacting p53 levels in daughter cells. High GMCL1 expression significantly correlates with resistance to paclitaxel in cancer cell lines with wild-type p53, including endometrial, breast, and upper aerodigestive tract cancer cells. Loss of GMCL1 restores paclitaxel sensitivity in p53 expressing cells but not in p53 deficient cells. We propose that in cancers with high GMCL1 levels, the CRL3GMCL1-mediated degradation of 53BP1 prevents the formation of the mitotic stopwatch complex, leading to p53 degradation and sustained proliferation. Finally, our results indicate that GMCL1 inhibition represents a novel strategy to restore taxane sensitivity in resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/709ee165cbc090bfd04b08f1a64c12e9f13af2e9" target='_blank'>
              GMCL1 Controls 53BP1 Stability and Modulates Paclitaxel Sensitivity in Cancer
              </a>
            </td>
          <td>
            Yuki Kito, Tania J. González-Robles, Sharon Kaisari, Juhee Pae, Sheena Faye Garcia, Juliana Ortiz-Pacheco, Beatrix Ueberheide, Ruth Lehmann, Antonio Marzio, G. Róna, Michele Pagano
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="During mitosis, pairing of homologous chromosomes can be detrimental and has been correlated with gene misregulation, chromosomal aberrations, and various pathological diseases. We previously demonstrated that homologous chromosomes are spatially segregated, or antipaired, in neonatal human endothelial cells at metaphase/anaphase, which may help prevent abnormal recombination. However, it is unclear if this antipairing persists in adult endothelial cells. To test whether the antipairing, or one homolog per nuclear hemisphere motif, is conserved in adult endothelial cells, we examined human aortic endothelial cells at metaphase. Using ImmunoFISH and high-resolution confocal microscopy to visualize the chromosomes and centrosomes, we found that small homologous chromosomes 13, 15, 17, 19, 21, 22, and the sex chromosomes, XY, exhibit a loss of spatial segregation in human adult aortic endothelial cells. In contrast, fewer adult endothelial cells showed a loss of segregation for the larger chromosomes 1, 4, and XX, suggesting a gradual decline in the fidelity of spatial segregation of homologous chromosomes. Notably, we observed a higher frequency of abnormal pairing in both small and large chromosomes in adult aortic endothelial cells as compared to neonatal umbilical vein endothelial cells. These findings suggest that mechanisms governing chromosome antipairing may decline with aortic endothelial cell age, leading to increased susceptibility to abnormal pairing and cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30619766655f6da41d03dd46682fe70a005b2ede" target='_blank'>
              Higher frequency of homologous chromosome pairing in human adult aortic endothelial cells
              </a>
            </td>
          <td>
            Jemery Morales, Gabriel Quintero Plancarte, Lisa Hua
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor suppressor p53 plays a crucial role in maintaining genome integrity in response to exogenous or endogenous stresses. The dynamics of p53 activation are stimulus- and cell type- dependent and regulate cell fate. Acting as a transcription factor, p53 induces the expression of target genes involved in apoptosis, cell cycle arrest and DNA repair. However, transcription is not a deterministic process, but rather occurs in bursts of activity and promoters switch stochastically between ON and OFF states, resulting in substantial cell-to-cell variability. Here, we characterized how stimulus-dependent p53 dynamics are converted into specific gene regulation patterns by inducing diverse forms of DNA damage ranging from ionizing and UV radiation to clinically relevant chemotherapeutics. We employed single molecule fluorescence in-situ hybridization (smFISH) to quantify the activity of target gene promoters at the single-cell and single-molecule level. To analyse this comprehensive data set, we developed a new framework for determining parameters of stochastic gene expression by Bayesian inference. Using this combined theoretical and experimental approach, we revealed that features of promoter activity are differentially regulated depending on the target gene and the nature and extent of the DNA damage induced. Indeed, stimulus-specific stochastic gene expression is predominantly regulated by promoter activation and deactivation rates. Interestingly, we found that in many situations, transcriptional activity was uncoupled from the total amount of p53 and the fraction bound to DNA, highlighting that transcriptional regulation by p53 is a multi- dimensional process. Taken together, our study provides insights into p53-mediated transcriptional regulation as an example of a dynamic transcription factor that shapes the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8f8f817463be32115a6d7b633baecddd3a2d98" target='_blank'>
              Decoding stimulus-specific regulation of promoter activity of p53 target genes
              </a>
            </td>
          <td>
            Flavia Vigliotti, Nicolai Engelmann, Mark Sinzger-D’Angelo, Heinz Koeppl, Alexander Loewer
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48f47266acdba9744d0c7b280d96e71682e3850" target='_blank'>
              Stable interaction of NONO with DBHS family members upon etoposide-induced DNA damage
              </a>
            </td>
          <td>
            Barbara Trifault, Stephanie Lamer, Andreas Schlosser, Kaspar Burger
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4868141c39dc560c925882ee816da6d52c4deb" target='_blank'>
              Uev1A counteracts oncogenic Ras stimuli in both polyploid and diploid cells
              </a>
            </td>
          <td>
            Qi Zhang, Yunfeng Wang, Xueli Fu, Ziguang Wang, Yang Zhang, Lizhong Yan, Yuejia Wang, Muhan Yang, Dongze Song, Ruixing Zhang, Hongru Zhang, Shian Wu, Shaowei Zhao
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3714058d5e1278c33cbc0e33134728cb5c41b6d7" target='_blank'>
              Abstract 1738: Pan-inhibition of super-enhancer-driven oncogenic transcription by novel synthetic ecteinascidins yields potent anti-cancer activity
              </a>
            </td>
          <td>
            Julian Obid, Max Cigrang, Maguelone Nogaret, Frédéric Coin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks in repetitive sequences is challenging because the abundance of potential templates for homology-directed repair (HDR) exacerbates the risk of ectopic recombination and chromosome rearrangements. Relocalization of repair sites in repetitive sequences to a ‘safe location’ prior to RAD51 loading has been observed in various organisms and is thought to suppress ectopic recombination and chromosomal rearrangements. We characterized this phenomenon in the rDNA repeats that reside within the nucleoli, specialized nuclear compartments where ribosome biosynthesis takes place. DSB induction in the rDNA repeats is associated with large-scale mobilization of the broken rDNA repeats from inside of the nucleoli to the nucleolar periphery where they are repaired by HDR. Here, we show that the two adaptor proteins Treacle and MDC1 are coordinating the sequence of events that ensue in response to rDNA break induction. Recruitment of the HDR machinery to rDNA breaks is dependent on the nucleolar adaptor Treacle, and specifically on its role in rDNA mobilization upon break induction. We demonstrate that following mobilization of the rDNA repeats and subsequent establishment of the γH2AX chromatin domain in the nucleolar periphery, the MDC1-RNF8-RNF168 axis is mediating the recruitment of the BRCA1-PALB2-BRCA2 complex and RAD51 loading. This coordinated sequence of events thus ensures that RAD51 loading is coupled to rDNA break mobilization to the nucleolar periphery, which may prevent ectopic recombination between repeats.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517df0bcb0ecef68788668311a97fb91ddb06252" target='_blank'>
              Treacle and MDC1 coordinate rDNA break repair by homologous recombination
              </a>
            </td>
          <td>
            Andrea Haenel, Johannes Leyrer, Manuel Stucki
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is the leading cause of cancer-related mortality worldwide owing to the lack of effective and early diagnostic tools and therapeutic approaches. DNAJC7, a member of the DnaJ heat shock family, is crucial in protein folding and stability; however, its specific functions and mechanisms in HCC remain unclear. Objective This study aimed to explore the role of DNAJC7 in HCC progression and evaluate its potential clinical significance as a prognostic marker. Methods Public databases (TCGA, ICGC, GEO, and TIMER) were used to assess DNAJC7 expression, correlations with clinical parameters, and related signaling pathways. Proliferation, migration, invasion, and cell cycle assays were performed to evaluate the function of DNAJC7 in HCC. Immune infiltration and associations with checkpoint proteins were analyzed using TIMER, and a Gene Set Enrichment Analysis (GSEA) was used to explore enriched pathways. Results DNAJC7 expression was higher in HCC tissues than in adjacent normal tissues and was associated with advanced malignancy and poor prognosis, including a lower overall survival, progression-free survival, and disease-free survival. DNAJC7 knockdown resulted in reduced malignant behavior of HCC cells, leading to S-phase cell cycle arrest. Increased DNAJC7 expression was associated with immune cell infiltration and the presence of immunological checkpoint molecules, including CTLA4 and PD-1. GSEA highlighted the activation of key pathways, including WNT signaling and cell cycle regulation. Conclusion DNAJC7 regulates tumor cell proliferation, migration, invasion, and immune evasion by acting as an oncogene in HCC. It can serve as a diagnostic and prognostic biomarker and potential treatment target for HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06202-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5d2c109594fa970efe3bba40f733c0078c3d83a" target='_blank'>
              Increased expression of DNAJC7 promotes the progression of hepatocellular carcinoma by influencing the cell cycle and immune microenvironment
              </a>
            </td>
          <td>
            Jiaxing Chen, Zhizhao Yang, Yongqiang Cui, Zhilei Zhao, D. Deng, Zhihao Fu, Xiao Zhang
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d143808abe3792603e16e2ab02c9eaad5e2640" target='_blank'>
              Abstract 2409: Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways
              </a>
            </td>
          <td>
            Joel Pardo, N. Temiz, Douglas Yee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This study explores the multifaceted role of p21 in mediating cellular responses to DNA-damaging agents, with a focus on doxorubicin treatment in HCT116 colon carcinoma cells. We investigated how different doses of doxorubicin affect cells with varied p21 and p53 statuses, revealing distinct roles for p21 depending on the drug dosage. At high doses (HD), p21 is more critical than p53 in mediating apoptosis, whereas at low doses (LD), p21 is essential for preventing mitotic defects and multinucleation. Notably, reintroducing p21 or pharmacologically inhibiting CDK1 reduced multinucleation. The absence of p21 upon LD doxorubicin exposure led to aberrant chromosome segregation, persistent DNA damage response (DDR) activation, and increased non-homologous end-joining (NHEJ) activity, resulting in unrepaired DNA accumulation and multinucleation. Additionally, mitotic defects in p21-deficient cells were associated with mislocalization of key mitotic regulators, Aurora B and mitotic kinesin-like protein 1 (MKLP1), exacerbating defective mitosis. In summary, p21 functions as a dual regulator in response to DNA damage, promoting apoptosis at HD and preventing mitotic failure at LD. These insights have significant implications for cancer therapy, highlighting the potential of targeting the p21 to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62798e3e091c52a2538fbed6da27e2c926098463" target='_blank'>
              Dual role of p21 in regulating apoptosis and mitotic integrity in response to doxorubicin in colon cancer cells
              </a>
            </td>
          <td>
            Heeyeon Kim, Haein Kim, Eunjung Jang, Young-Woo Eom, Gyesoon Yoon, Kyeong Sook Choi, Eunhee Kim
          </td>
          <td>2025-04-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb1e30ecb249d137107e5349811c391ef3198f8" target='_blank'>
              A Cancer-Specific Antigen Drives Histone Acetylation by Stabilizing the Acetyltransferases
              </a>
            </td>
          <td>
            Xuekun Fu, Xu Yang, Jie Huang, P. R. Potts, Chao Liang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks arising from base excision repair (BER) during the misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination–deficient (HRD) cells. At templated AP sites, cross-links between the DNA and 5-hydroxymethylcytosine binding, embryonic stem cell–specific (HMCES) regulate replication fork speed, avoiding cytotoxic double-strand breaks. While the role of HMCES at the template DNA strand is well studied, its effects on nascent DNA are not. We provide evidence that HMCES–DNA-protein cross-links (DPCs) are detrimental to the BER-mediated removal of 5-hydroxymethyl-2′-deoxycytidine (5hmdC)–derived 5-hydroxymethyl-2′-deoxyuridine from replication forks. HRD cells have heightened HMCES-DPCs, which increase further upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP sites. HMCES depletion substantially suppresses 5hmdC-mediated replication fork defects, chromosomal aberrations, and cell death in HRD cells. This reveals that HMCES-DPCs are a source of BER-initiated single-stranded DNA gaps, which indicates that endogenous DPCs contribute to genomic instability in HRD tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622156d932ecc5d7d331e4371a1b29a1f41c3ef" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination–deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martín, Marta del Rio Oliva, Yodhara Wijesekara Hanthi, Sara Berrada, Raimundo Freire, Jean-Yves Masson, José Carlos Reyes, Vincenzo Costanzo, Iván V. Rosado
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/545b6d8dbebbe17da0f379631a55f84485df7054" target='_blank'>
              The Significance of Chromosome Conformation Capture in 3D Genome
Architecture Comprehension
              </a>
            </td>
          <td>
            Emre Sefer
          </td>
          <td>2025-04-22</td>
          <td>Current Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, Chung-Chi Hsu, Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c41f6c6066f9f1caf2b6797a1f6cfc0b6b61d7" target='_blank'>
              Abstract 1958: Chromatin accessibility landscapes of colorectal cancer development and genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, J. Medina, Stephen B. Baylin, Kasper D. Hansen, V. Adleff, R. Fijneman, Michael A. Beer, Robert B Scharpf, G. Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d56727982104a6a78307d0f1220221468bf7fd" target='_blank'>
              Abstract 2740: Mechanisms of phase-separation driven chromatin looping in acute myeloid leukemia
              </a>
            </td>
          <td>
            Tooba Rashid, Z. Drum, Doris Cruz Alonso, Gelila Petros, Yoseli Quiroga, Douglas Phanstiel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa24230eaf15e64077b2640e42b580f6dcef4855" target='_blank'>
              Abstract 2838: Modeling putative oncogenic variants of human topoisomerase II by genome editing of a near haploid cell line
              </a>
            </td>
          <td>
            K. C. Nitiss, Brigid Conroy, Maureen N. McCoy, J. Nitiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827f74e9a451d846709a409162aebd5ef66250d5" target='_blank'>
              Abstract LB468: Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity
              </a>
            </td>
          <td>
            Samantha L. Schuster, P. Itagi, Sonali Arora, Thomas W Persse, Patricia Galipeau, John K. Lee, R. Montgomery, Jonathan L. Wright, P. Grivas, F. Vakar‐López, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Pluripotent stem cells (PSC) are known to recurrently acquire genetic aberrations including chromosomal aneuploidy during long-term culture of which the consequences remain largely unknown. A recent study reported that xenografting of aneuploid mouse embryonic stem cells (mESC) gives rise to teratomas containing undifferentiated elements with metastatic capacity. This contrasts with benign teratomas composed of fully differentiated tissues from the three embryonic germ layers obtained by xenografting diploid mESCs. The highlighted study thus proposes that general aneuploidy may drive metastasis of PSC-derived teratomas, thus directly correlating it with malignant potential of PSCs. Main The aneuploid mESC lines used in the aforementioned study showed trisomies of chromosomes 6, 8, 11, 15 or combinations thereof. Interestingly, several of these trisomies, specifically in chromosomes 6, 8, and 11, constitute well-known chromosomal aberrations recurrently acquired in mESCs upon (long-term) in vitro culture. This is suggested to be driven by numerous (proto)oncogenes in these regions. Notably, recurrent chromosomal aberrations are also observed in human PSCs (hPSCs), mostly in chromosome 1, 12, 17, 20 and X, and have similarly been associated with the acquisition of a malignant phenotype in vitro, posing a risk to their potential clinical application. There is synteny between the chromosomes recurrently affected by such aberrations in mouse and human PSCs, namely mouse chromosome 6 and human chromosome 12 as well as mouse and human chromosome 1, suggesting a chromosome-specific phenomenon that may transcend the species barrier. Additionally, in the field of cancer, malignant (but not benign) human germ cell tumors (hGCTs), pathologically similar to the tumors derived from the aforementioned aneuploid mouse PSCs, are characterized by supernumerical copies of chromosome 12, highlighting the additional clinical relevance of these findings and the aneuploid mESC models for GCT research and treatment. Conclusions We suggest that chromosome-specific trisomies, rather than general aneuploidy, might drive teratoma metastasis upon mESC xenografting in vivo. We also observed indications of synteny between the recurrent chromosomal aberrations among human and mouse PSCs, suggesting potentially common intraspecies selection mechanisms. We finally reiterate the similarities observed between the PSC and GCT field related to chromosomal aberrations and malignancy, highlighting the relevance of these experimental models in both research fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9961bd4e5533b7648a3a993149a92ce0795a5ee" target='_blank'>
              Chromosome-specific aberrations, rather than general aneuploidy, may drive mouse embryonic stem cell-derived teratoma metastasis
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, Sanne Hillenius, T. F. Eleveld, Daniela C F Salvatori, Leendert H J Looijenga
          </td>
          <td>2025-04-15</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d11e895cea0cf1e967c3e4adda2d90a1183e147" target='_blank'>
              Abstract 215: A novel framework for deciphering 3D genome structure in colorectal cancer
              </a>
            </td>
          <td>
            Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent evidence indicates that FAM111B is significantly involved in the progression of various cancers. Nonetheless, the potential pan-cancer implications of FAM111B have not been systematically investigated. In this study, FAM111B’s expression and oncogenic potential were studied using TCGA and GTEx data via GEPIA2, TIMER2.0, and STRING tools. Pathway enrichment analyses with the GO, KEGG, Reactome, and WikiPathways databases were conducted to explore its role in cancer development. The results were validated via multiplex immunofluorescence assays of pancreatic cancer tissues, microarray assays of ovarian cancer tissues, and protein transcriptomics of ovarian cancer cells. The expression levels of FAM111B were elevated in most cancer types and were associated with poor prognostic outcomes. Mechanistically, FAM111B expression was positively correlated with the expression of genes involved in DNA homologous recombination repair and with the infiltration of Th2 CD4+ T cells. These observations were further substantiated in ovarian cancer cell lines and tissue specimens from pancreatic and ovarian cancers. FAM111B functions as a biomarker for the DNA repair pathway and Th2 CD4+ T-cell infiltration in human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe1e275d0931ab0098bd9862f4488d0d9192898d" target='_blank'>
              Unveiling FAM111B: A Pan-Cancer Biomarker for DNA Repair and Immune Infiltration
              </a>
            </td>
          <td>
            Fang Wei, Wanying Li, Ting Zhou, Xianglin Yuan, Lihong Zhang
          </td>
          <td>2025-03-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract APE1 [apurinic/apyrimidinic (AP) endodeoxyribonuclease 1] is the main endonuclease of the base excision repair pathway acting on abasic (AP) sites in DNA. APE1 is an abundant nuclear protein, and improper expression or localization of this factor could lead to the accumulation of toxic DNA intermediates. Altered APE1 subcellular distribution and expression are associated with cancer development, suggesting the importance of a fine-tuning mechanism for APE1 activities. Recent works highlighted the presence of APE1 within nucleoli of cancer cells and the ability of APE1 to form biomolecular condensate. However, whether secondary structures of ribosomal RNA (rRNA) influence the nucleolar localization of APE1 remains poorly understood. Since protein overexpression can result in artificial nucleolar accumulation, it is imperative to have appropriate cellular models to study APE1 trafficking under physiological conditions. To address this issue, we generated a murine embryonic stem cell line expressing endogenous fluorescent-tagged APE1. Live-cell imaging demonstrates that APE1 nucleolar accumulation requires active rRNA transcription and is modulated by different genotoxicants. In vitro experiments showed that APE1 condensate formation depends on RNA-forming G-quadruplex structures and relies on critical lysine residues. This study sheds light on the mechanisms underlying APE1 trafficking to the nucleolus and the formation of RNA-dependent APE1 nucleolar condensates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e2a0fb2c51466df9d445ca234bc9bfccb21a3b" target='_blank'>
              APE1 condensation in nucleoli of non-cancer cells depends on rRNA transcription and forming G-quadruplex RNA structures
              </a>
            </td>
          <td>
            Giuseppe Dall’Agnese, N. Hannett, Kalon J. Overholt, Jesse M. Platt, Jonathan E Henninger, Asier Marcos-Vidal, Zahraa Othman, Gilmar Salgado, G. Antoniali, Gianluca Tell
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43eec0c4a94eb30376e354d956c5d082c137be28" target='_blank'>
              Abstract 228: Loss of TET2 and TET3 mediated DNA demethylation drives intestinal tumor formation
              </a>
            </td>
          <td>
            Oliver Gilliam, Ihab Ansari, Nikolaos Georgakopoulos, Manuel Mastel, Rene Jackstadt, Sebastian Belle, Yehudit Bergman, F. Lyko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="The IER3 gene has a complex role in cancer biology, acting either as a tumor suppressor or an oncogene, depending on the cancer type. This duality underscores the complexity and importance of molecular pathways in modulating cancer behavior. Despite its significance in cancer development, there is a dearth of studies elucidating the exact mechanisms underlying IER3’s involvement in modulating cancer behavior. Here, utilizing cervical carcinoma and neuroblastoma (NB) cell lines as model systems we characterized the pathways that mediate the functional switch between the oncogenic and tumor suppressor roles of IER3. In HeLa cells, IER3 expression promotes an oncogenic program that includes immediate early response pathway genes such as EGR2, FOS, and JUN. However, in NB cells, IER3 suppresses the EGR2-dependent oncogenic program. This differential regulation of EGR2 by IER3 involves epigenetic modulation of the EGR2 promoter. IER3 dependent tumor suppressor pathway in NB cells relies on ADAM19 gene. Thus, our findings uncover the molecular pathways that dictate the context-dependent roles of IER3 in cancer, providing insights into its dual functionality in different cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f45e736911a5471bab3e71faffe39b31598773a" target='_blank'>
              IER3: exploring its dual function as an oncogene and tumor suppressor
              </a>
            </td>
          <td>
            M. Kanduri, Santhilal Subhash, Rossana Putino, Sagar Mahale, C. Kanduri
          </td>
          <td>2025-03-16</td>
          <td>Cancer Gene Therapy</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Loss of chromosome Y (LOY) has recently been proposed to be associated with cancer aggressiveness, altered T-cell function, and poor prognosis in bladder carcinomas. Methods Chromosome Y was analyzed using fluorescence in-situ hybridization on a tissue microarray containing 2,071 urothelial carcinomas of the urinary bladder from male patients, including 487 patients who had undergone cystectomy for muscle-invasive disease and for whom follow-up data were available. Data on tumor microenvironment were obtained from a previous study. Results LOY was found in 26.0% of 1,704 analyzable cancers. In non-invasive cancers, LOY frequency was comparable in pTa G2 (22.8%) and pTa G3 (24.1%, p = 0.8036) carcinomas and slightly increased from pTa to pT2 - 4 carcinomas (23.1% for pTa and 27.2% for pT2 - 4) but these differences were not significant (p = 0.0794). In muscle-invasive cancers, LOY frequency slightly increased from pT2 (25.5%) to pT4 cancers (33.0%), but this association was not significant (p = 0.1814). Among pT2 - 4 cancers, LOY was associated with venous invasion (p = 0.0010) but unrelated to pT, pN, and L-status, as well as to overall, recurrence-free, and cancer-specific survival. Muscle-invasive urothelial carcinomas with and without LOY did not show significant differences in the number of CD8 positive lymphocytes, fraction of CD8 positive intraepithelial lymphocytes, number of macrophages and dendritic cells, and fraction of T helper and T regulatory cells. Conclusion The lack of a clear association of LOY with histopathological parameters of cancer aggressiveness, patient prognosis, and parameters describing the tumor microenvironment strongly argues against the driving role of LOY in bladder cancer progression and cancer-associated immune reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbc1bca8e295978679529dedbaa6c18114de784c" target='_blank'>
              Loss of chromosome Y is unrelated to the composition of the tumor microenvironment and patient prognosis in muscle-invasive urothelial bladder cancers
              </a>
            </td>
          <td>
            H. Plage, V. Ahlburg, S. Hofbauer, K. Furlano, Sarah Weinberger, Annika Fendler, Florian Rossner, S. Schallenberg, S. Elezkurtaj, N. Blessin, M. Lennartz, A. Marx, H. Samtleben, Margit Fisch, M. Rink, M. Słojewski, K. Kaczmarek, T. Ecke, S. Koch, N. Adamini, R. Simon, G. Sauter, H. Zecha, J. Weischenfeldt, T. Klatte, S. Minner, D. Horst, T. Schlomm, M. Kluth
          </td>
          <td>2025-04-14</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="To counteract the damaging effects of DNA interstrand crosslinks (ICLs), cells have evolved various specialized ICL repair pathways. However, how ICL repair impacts genetic integrity remains incompletely understood. Here, we determined the mutagenic consequences of psoralen ICL repair in the animal model C. elegans and identify two mutagenic repair mechanisms: (i) translesion synthesis through POLH and REV1/3-mediated bypass, leading to single nucleotide polymorphisms (SNVs), and (ii) end joining via POLQ or HELQ action resulting in deletions. While we found no role for the Fanconi anemia genes FANCD2 and FANCI, disruption of TRAIP, which triggers unloading of the CMG helicase at sites of blocked replication, led to a strikingly altered repair profile, suggesting a role for DNA replication in the etiology of ICL-induced deletions. TRAIP deficiency did not affect SNV formation; instead, we found these SNVs to depend on the functionality of the Fanconi anemia-associated nuclease FAN1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c8e44dc0dd119d75905686eba6c875d7b3f783" target='_blank'>
              FAN1-mediated translesion synthesis and POLQ/HELQ-mediated end joining generate interstrand crosslink-induced mutations
              </a>
            </td>
          <td>
            Jip Verschuren, R. van Schendel, Ivo van Bostelen, Alex E E Verkennis, P. Knipscheer, M. Tijsterman
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05f154cb832f6f7744c5bac28235208f5d152b7" target='_blank'>
              Mitochondrial DNA copy numbers in gastric cancer tissues: a possible biomarker for estimating cancer progression.
              </a>
            </td>
          <td>
            Hisanori Fukunaga, Mayuko Fukunaga
          </td>
          <td>2025-04-23</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fbe089f3b9ae2dd8bdd29ef8533760180387d9e" target='_blank'>
              Abstract 375: Cancer cells may utilize centrosome amplification to promote survival as a response to chemotherapeutic stress
              </a>
            </td>
          <td>
            Madison M. Purkerson, Sarah R Amend, Kenneth J. Pienta
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differential alternative splicing transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes including vFLIP, vCyclin and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells, and significantly inhibited KSHV-driven proliferation, cellular transformation and tumorigenesis. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. Importance Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1dd5dbd2f3302db90bfa70c7f9ae17b8e6ba3f" target='_blank'>
              KSHV Reprograms Host RNA Splicing via FAM50A to Activate STAT3 and Drive Oncogenic Cellular Transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Karla Paniagua, Ling Ding, Xian Wang, Yufei Huang, Mario A Flores, Shou-Jiang Gao
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a47b29a8de8c8675e9768fffa79f59328b51b0f" target='_blank'>
              Abstract 6616: LMNA-FOSB fusions are a recurrent event in meningiomas
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan Congivaram, P. K. Brastianos, Akash J. Patel, C. Horbinski, James P. Chandler, E. Z. Erson Omay, JenniferA Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark Youngblood
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6462361aae6f3cbb7d9db079e7f43e5e3ea1a7f1" target='_blank'>
              Abstract 1448: An evolutionarily conserved chromatin remodeling factor, FACT, in regulation of chromatin, gene expression and cancer
              </a>
            </td>
          <td>
            Priyanka Barman, S. Bhaumik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Ovarian cancer (OC) is one of the deadliest gynecological malignancies. As the prevalent post-transcriptional regulation, alternative polyadenylation (APA) plays a crucial role in various tumors. Here we identify that the APA regulator NUDT21 is upregulated in OC and promotes malignant progression. We further demonstrate that IGF2BP3 interacts with NUDT21, which suggests m6A modification could regulate APA processing. Mechanistically, IGF2BP3, recognizing the m6A-modified site in intron 32 of SPTBN1, recruits NUDT21 to promote the usage of the SPTBN1 proximal polyadenylation site (PAS), thus increasing the generation of short transcripts in OC cells. Intriguingly, the SPTBN1 long variant demonstrates tumor-suppressive properties, whereas the short variant enhances oncogenic activity in OC. Subsequently, we illustrate that the long isoform inhibits tumor growth and metastasis by binding to CDK1 and blocking the G2/M phase of the cell cycle. In conclusion, this study uncovers a previously unrecognized regulatory mechanism in OC, which could provide potential therapeutic strategies for OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05b972768c51f09fee1989801fedf4505bb59f18" target='_blank'>
              IGF2BP3 recruits NUDT21 to regulate SPTBN1 alternative polyadenylation and drive ovarian cancer progression
              </a>
            </td>
          <td>
            Xin Luo, Lingcui Xie, Xiaohui Ma, Ningxuan Chen, Jiani Xu, Xiaoyan Jiang, Xiaoyi Liu, Yu Yang, Tao Liu, Ping Yi, Jing Xu
          </td>
          <td>2025-04-29</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95cf76b81497b2613567d4d58faa4859bccba0a9" target='_blank'>
              Attenuation of mosaic aneuploidy and erroneous first mitotic division of human embryos
              </a>
            </td>
          <td>
            Cerys E. Currie, A. Byrska, D. Taylor, M. Erent, Daniela N. Bakalova, Xuhui Sun, Constandina Koki, Nigel J. Burroughs, Richard. A. Anderson, A. Marston, Geraldine M. Hartshorne, A. McAinsh
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, Jeanette Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8774da0a06c116c814c2652572a737b8d86a0c02" target='_blank'>
              Genome-wide ribonucleotide detection in Archaea
              </a>
            </td>
          <td>
            Y. Moalic, Maurane Reveil, Deepali L. Kundnani, S. Balachander, Taehwan Yang, Alli L. Gombolay, Farahnaz Ranjbarian, Raphaël Brizard, Patrick Durand, H. Myllykallio, Mohamed Jebbar, Anders Hofer, F. Storici, Ghislaine Henneke
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Bleomycin (Bleo) is a glycopeptide with potent antitumor activity that induces DNA double-strand breaks (DSBs) through free radical generation, similar to ionizing radiation (IR). Therefore, Bleo is considered a radiomimetic drug. However, differences in DNA repair mechanisms between IR- and Bleo-induced DNA damage have not been fully elucidated. Therefore, in the present study, we examined a panel of repair-deficient human TK6 cell lines to elucidate the relative contributions of individual repair factors. Results Our comprehensive profiling indicated that both non-homologous end joining (NHEJ) and homologous recombination (HR) contributed to DSB repair induced by X-rays and Bleo. Furthermore, tyrosyl-DNA phosphodiesterase (TDP)-related repair was a significant factor for cellular sensitivity to Bleo treatment. TDP1−/−/TDP2−/− cells exhibited greater sensitivity to Bleo than TDP1−/− or TDP2−/− cells, but not to X-rays. In addition, we determined whether TDP2 is involved in the repair of Bleo-induced DSBs using a neutral comet assay. In TDP1-deficient cells, knockout of TDP2 resulted in a significant delay in the repair kinetics of DSBs induced by Bleo, but not by X-rays. Conclusions The contribution of the TDP-related pathway to DSB repair significantly differed between IR and radiomimetic drugs. The discovery of this novel TDP2-dependent repair of DSBs resulting from radiomimetic drug exposure indicates that TDP1 and TDP2 inhibition in combination with radiomimetic drugs represents a strategy for cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00329-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b896deb7b39c22219dcb649c924ec1a29423e547" target='_blank'>
              Role of TDP2 in the repair of DNA damage induced by the radiomimetic drug Bleomycin
              </a>
            </td>
          <td>
            N. Shimizu, Kazuki Izawa, Mubasshir Washif, Ryosuke Morozumi, Kouji Hirota, Masataka Tsuda
          </td>
          <td>2025-03-28</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cd7b66a7f1ffe44365de20596b44e77ed523aee" target='_blank'>
              Centromere-size reduction and chromatin state dynamics following intergenomic hybridization in cotton.
              </a>
            </td>
          <td>
            Jinlei Han, Guanjing Hu, Yan Dai, Xin Zhang, Jingjing Tian, Jialiang Zhou, Xinqi Xu, Qi Chen, Xiaobing Kou, Lei Xu, Xinyu Wu, Ziying Sun, Jiahui Geng, Lin Li, Chenyu Qiu, Teame Gereziher Mehari, Baohua Wang, Hui Zhang, Xinlian Shen, Zhenzhen Xu, Jonathan F. Wendel, Kai Wang
          </td>
          <td>2025-05-02</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While DNA:RNA hybrids contribute to multiple genomic transactions, their unscheduled formation is a recognized source of DNA lesions. Here, through a suite of systematic screens, we rather observed that a wide range of yeast mutant situations primarily triggering DNA damage actually leads to hybrid accumulation. Focusing on Okazaki fragment processing, we establish that genic hybrids can actually form as a consequence of replication-born discontinuities such as unprocessed flaps or unligated Okazaki fragments. Strikingly, such “post-lesion” DNA:RNA hybrids neither detectably contribute to genetic instability, nor disturb gene expression, as opposed to “pre-lesion” hybrids formed upon defective mRNA biogenesis, e.g., in THO complex mutants. Post-lesion hybrids similarly arise in distinct genomic instability situations, triggered by pharmacological or genetic manipulation of DNA-dependent processes, both in yeast and human cells. Altogether, our data establish that the accumulation of transcription-born DNA:RNA hybrids can occur as a consequence of various types of natural or pathological DNA lesions, yet do not necessarily aggravate their genotoxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67011b6e27fed1e3b5196a0fb25f35fae9832be" target='_blank'>
              DNA lesions can frequently precede DNA:RNA hybrid accumulation
              </a>
            </td>
          <td>
            Raphaël M Mangione, Steven Pierce, Myriam Zheng, Robert M Martin, Coralie Goncalves, Arun Kumar, S. Scaglione, Cristiana de Sousa Morgado, Arianna Penzo, Astrid Lancrey, Robert J. D. Reid, Ophélie Lautier, P. Gaillard, Peter C Stirling, Sérgio F. de Almeida, Rodney Rothstein, Benoît Palancade
          </td>
          <td>2025-03-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Krüppel-like factor 4 (KLF4) is a highly conserved zinc-finger transcription factor involved in cellular processes such as development, differentiation, and cell cycle regulation. Previous studies show that mouse embryonic fibroblasts (MEFs) null for Klf4 exhibit increased genomic instability. While KLF4 is regarded as a tumor suppressor in many human cancers, its role in DNA repair mechanisms remains unknown. In this study, cultured MEFs wild type (+/+) and null (−/−) for Klf4 and human carcinoma colorectal (RKO) cells were studied as a model for human colorectal cancer. Etoposide, a chemotherapeutic topoisomerase II poison, was employed to investigate KLF4’s role in DNA damage repair. Following etoposide treatment, immunostaining and Western blotting revealed cells expressing Klf4 exhibited lower levels of γ-H2AX, a biomarker for DNA damage, compared to cells without Klf4. Moreover, after DNA damage, cells expressing Klf4 exhibited increased levels of BRCA1 and Rad51, known tumor suppressor genes. Finally, genes involved in DNA damage response (DDR), ATR, and Chk1 were upregulated in cells containing functional KLF4, offering a possible mechanism for KLF4’s role in mediating DDR. Our results indicate that KLF4 plays a crucial role in maintaining genetic stability by enhancing cell DDR, supporting previous findings that KLF4 functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d84ceceef475b01fca18be7952c16f21ed8ae0d3" target='_blank'>
              Krüppel-like Factor 4-Deficient Cells Are Sensitive to Etoposide-Induced DNA Damage
              </a>
            </td>
          <td>
            Maxwell H. Rubinstein, Aidan Conroy, Elisabeth L. Pezzuto, Hadeel Al Qoronz, Patrick Wertimer, Engda G. Hagos
          </td>
          <td>2025-03-21</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4239f86976c6dc02232cf427f0fe3c7ee8f60e6" target='_blank'>
              Abstract 2835: Lamin AC deficiency promotes genomic instability through increased R-loop accumulation
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, Vasilisa Askenova, Elizabeth Giordanio, N. Takahashi, Michael Nirula, Yasuhiro Arakawa, Ukhyun Jo, Beverly A. Teacher, M. Aladjem, Stephen Lockett, Marry Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background The incidence of colorectal cancer (CRC) in young adults (age of diagnosis < 50 years old) has been rapidly increasing. Although ∼20% of early-onset (EO) CRC cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Non-genetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as differences in the EOCRC epigenome, microbiome and immunome. We sought to address this by comparing differences in DNA methylation from the cohort of colorectal cancer patients in The Cancer Genome Atlas (TCGA). Further, we carefully identified intra-tumoral microbes from TCGA and two other datasets and then related the microbes to EOCRC status and deconvolved immune cell abundances. We found that DNA methylation (DNAm) age acceleration by12 years when compared with average-onset CRC (AOCRC) patients. Differentially methylated sites associated with genes are related to CREB signaling in neurons, G protein coupled receptor signaling, phagosome formation and S100 family signaling. These differences were validated in the gene expression from TCGA and a second, larger real-world dataset from the Oncology Research Information Exchange Network (ORIEN). However, no consistent differences were observed in the intra-tumor microbes between EOCRC and AOCRC. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more, larger, positive correlations in EOCRC. These data suggest epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. SIGNIFICANCE We investigated whether environmentally driven factors contribute to early-onset colorectal cancer (EOCRC). We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than average-onset CRC. These data suggest a dysregulation of immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8859cb309a43420dfdd0f2da144b37f5d23d1c73" target='_blank'>
              Epigenetic Modulation, Intra-tumoral Microbiome and Immunity in Early Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Ning Jin, R. Hoyd, Ayse Selen Yilmaz, Jian Zhu, YunZhou Liu, M. J. Singh, Dennis Grencewicz, Xiaokui Mo, Matthew Kalady, Daniel Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos HF Chan, Michelle L Churchman, N. Denko, Frances Di Clemente, Rebecca D Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, A. Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, A. Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, Aik Choon Tan, Daniel J. Spakowicz
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c91a033b11a5d910b45df1fe7d4556a0c6c8b6e" target='_blank'>
              Abstract 5585: DNA methylation and transcription factors regulate overexpression of PVT1 in prostate cancer
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Colin Finnegan, Murtaza Barkarar, Oluwabusola Ajagbe, Rachel E. Bonacci, O. Ogunwobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Activation-induced cytidine deaminase (AID) serves as a critical molecular orchestrator in the germinal center (GC) reaction within secondary lymphoid organs (SLOs), driving the production of high-affinity antibodies through somatic hypermutation. While its pathological implications are well-documented - including ectopic expression in non-B cell populations and transcriptional dysregulation linked to hematological malignancies and solid tumorigenesis - the cellular provenance of AID in solid tumors remains an unresolved paradox. This review advances two principal hypotheses: (1) AID may derive from tertiary lymphoid structures (TLSs), ectopic immune niches mirroring SLO organization, and (2) exhibits context-dependent transcriptional duality, capable of both potentiating and suppressing gene expression based on microenvironmental cues. Through systematic analysis of AID/GC involvement across cancer subtypes, we delineate mechanistic connections between lymphoid neogenesis and tumor progression. Our examination extends to TLS architecture, revealing three critical dimensions: (i) structural organization and cellular heterogeneity, (ii) developmental trajectories, and (iii) bidirectional interactions with tumor microenvironments. Crucially, we establish functional parallels between tumor-infiltrating B cells (TIL-Bs) in SLOs versus TLSs, while elucidating the differential roles of AID in canonical GC versus TLS-associated GC formation. This synthesis ultimately proposes that AID’s functional dichotomy - acting as both oncogenic collaborator and tumor suppressor - underlies the paradoxical prognostic associations observed with TLS presence across malignancies. The review thereby provides a conceptual framework reconciling AID’s dual functionality with the context-dependent immunobiology of tumor-associated lymphoid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b363fcb2d2de4ec9cabc224fdff730fd9d9c0c6b" target='_blank'>
              Activation-induced cytidine deaminase in tertiary lymphoid structures: dual roles and implications in cancer prognosis
              </a>
            </td>
          <td>
            Zhuangwei Lv, J. Jiao, Wuyang Xue, Xiaoyu Shi, Ruihan Wang, Jinhua Wu
          </td>
          <td>2025-04-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40685785e340da5a1a12204dd296babda47935cc" target='_blank'>
              Abstract 1955: Blood-based mapping of the personalized tumor epigenomic landscape
              </a>
            </td>
          <td>
            Elena Zotenko, Rachel Gittelman, Alan Selewa, Noam Vardi, Tam Banh, Gleb Martovetsky, Oscar Westesson, Emily Tsang, Sara Wienke, D. Kennedy, Meromit Singer, Stefanie Mortimer, Darya I. Chudova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcbbcac8a736304b9cab15ef0da632985a22219d" target='_blank'>
              Abstract 2666: Mapping intratumoral heterogeneity in colorectal cancer: Spatially resolved subclones and tumor microenvironment
              </a>
            </td>
          <td>
            YeonJu Lee, Woong-Yang Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebaad9fe8570813573af3bfde1cc0999a786ae2f" target='_blank'>
              Abstract 5884: Cancer subtype classification by cfDNA fragmentomics analysis
              </a>
            </td>
          <td>
            Lisha Zhu, C. Dai, Shuang Gan, S. Jia, Pan Du
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d51acfd255e9f2c3851395fb88bf941b1c544b" target='_blank'>
              Abstract 1735: The discovery of PKMYT1 inhibitors to exploit synthetic lethality caused by replication stress
              </a>
            </td>
          <td>
            R. Marais, J. Grabarek, M. Ioannou, L. Leung, A. Raoof, Mike Brown, D. Smithen, C. Fitzpatrick, C. Chester, J. Leigh, M. Aljarah, M. Lisauskaite, O. Grytsai, R. Burgin, D. Niculescu-Duvaz, G. Thomson, C. Springer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Archaea, often thriving in extreme habitats, are believed to have evolved efficient DNA repair pathways to cope with constant insults to their genomes. However, how these organisms repair DNA double-strand breaks (DSBs), the most lethal DNA lesions, remains unclear. Here, we show that replicative primase consisting of the catalytic subunit PriS and the noncatalytic subunits PriL and PriX from the hyperthermophilic archaeon Saccharolobus islandicus is involved in DSB repair. We show that the overproduction or knockdown of PriL increases or decreases, respectively, the rate of survival and mutation frequency of S. islandicus cells following treatment with a DNA damaging agent. The increase in mutation is attributed primarily to an increase in small insertions or deletions. Further, overproduction of PriL enhances the repair of CRISPR-generated DSBs in vivo. These results are consistent with the extraordinary ability of PriSL to promote annealing between DNA strands sharing microhomology in addition to the activity of the heterodimer in terminal transfer and primer extension. The primase-mediated DSB repair is cell-cycle dependent since PriL is barely detectable during the S/G2 transition. Our data demonstrate that replicative primase is involved in DSB repair through microhomology-mediated end joining in Archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41c802b392e2aa09bc0c77d38347ce453e14aa3" target='_blank'>
              Archaeal replicative primase mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Daijiang Xiong, Zhimeng Li, Wen Qi, Shaoying Wang, Junkai Huang, Ningning Zhang, Zhenfeng Zhang, Li Huang
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination (HR) is a major pathway to repair DNA double-strand breaks. Hereditary breast and ovarian cancer syndrome (HBOC) is caused by germline pathogenic variants of HR-related genes, such as BRCA1 and BRCA2 (BRCA1/2). Cancer cells with HR deficiency are sensitive to poly(ADP-ribose) polymerase (PARP) inhibitors. Therefore, accurate evaluation of HR activity is helpful to diagnose HBOC and predict the effects of PARP inhibitors. The direct-repeat GFP (DR-GFP) assay has been utilized to evaluate cellular HR activity. However, evaluation by the DR-GFP assay tends to be qualitative and requires the establishment of stable cell lines. Therefore, we developed an assay to quantitatively measure HR activity called Assay for Site-Specific HR Activity (ASHRA), which can be performed by transiently transfecting two plasmids. In ASHRA, we use Cas9 endonuclease to create DNA double-strand breaks at specific sites in the genome, enabling the targeting of any endogenous loci. Quantification of HR products by real-time PCR using genomic DNA allows HR activity evaluated at the DNA level. Thus, ASHRA is an easy and quantitative method to evaluate HR activity at any genomic locus in various samples. Here, we present the protocols for adherent cells, suspension cells, and tumor tissues. Key features • This assay quantitatively evaluates homologous recombination (HR) activity. • This assay can measure HR activity in adherent cells, suspension cells, and tumor tissues. • This real-time PCR-based assay does not require a flow cytometer or next-generation sequencer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6041469126e683215dc13399e4746d11c09375" target='_blank'>
              Assay for Site-Specific Homologous Recombination Activity in Adherent Cells, Suspension Cells, and Tumor Tissues
              </a>
            </td>
          <td>
            Yuki Yoshino, Shin Kikuta, N. Chiba
          </td>
          <td>2025-04-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f499ed27ee797b62ddf192d5f361132033db2e" target='_blank'>
              Abstract 5080: Investigation of exome-wide tumor heterogeneity on colorectal tissue-based single cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, B. Barták, Béla Molnár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The development of hematopoietic cell lineages is a highly complex process governed by a delicate interplay of various transcription factors. The expression of these factors is influenced, in part, by epigenetic signatures that define each stage of cell differentiation. In particular, the formation of B lymphocytes depends on the sequential silencing of stemness genes and the balanced expression of interdependent transcription factors, along with DNA rearrangement. We have investigated the impact of Dido3 deficiency, a protein involved in chromatin status readout, on B cell differentiation within the hematopoietic compartment of mice. Our findings revealed significant impairments in the successive stages of B cell development. The absence of Dido3 resulted in remarkable alterations in the expression of essential transcription factors and differentiation markers, which are crucial for orchestrating the differentiation process. Additionally, the somatic recombination process, responsible for generation of antigen receptor diversity, was also adversely affected. These observations highlight the vital role of epigenetic regulation, particularly the involvement of Dido3, in ensuring proper B cell differentiation. This study reveals new mechanisms underlying disruptive alterations, deepening our understanding of hematopoiesis and may potentially lead to insights that aid in the development of therapeutic interventions for disorders involving aberrant B cell development. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01394-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba226143005da64aa14cb154db38e86341f77b0e" target='_blank'>
              The chromatin reader Dido3 is a regulator of the gene network that controls B cell differentiation
              </a>
            </td>
          <td>
            Fernando Gutiérrez del Burgo, María Ángeles García-López, Tirso Pons, Enrique Vázquez de Luis, Carlos Martínez-A, R. Villares
          </td>
          <td>2025-04-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c9b2ddba58024f3c35959c6bf71d2fc72ae818" target='_blank'>
              Abstract 1456: Tumor lineage plasticity blockade by therapeutic targeting of an epigeneticregulator
              </a>
            </td>
          <td>
            Xiong Zhang, Yatian Yang, E. Corey, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2b1de2dc9f84fd1d329975218308dac529d161" target='_blank'>
              Obligate intracellular Orientia tsutsugamushi impedes TP53 expression to inhibit DNA damage-induced apoptosis
              </a>
            </td>
          <td>
            Paige E. Allen, David L. Armistead, Svetlana Blinova, J. Carlyon
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7d904ab0bb7bfa986bb2bea4c6f19edb348725" target='_blank'>
              Dysregulation of Locus-Specific Repetitive Elements in TCGA Pan-Cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Inaccurate chromosome segregation can lead to the formation of aneuploid cells that harbor an imbalanced complement of chromosomes. Several fungal species are not only able to tolerate the detrimental effects of aneuploidy but can use it to adapt to environmental pressures. The fungal pathobiont Candida albicans frequently acquires supernumerary chromosomes that enable growth in the presence of antifungal drugs or in specific host niches, yet the transcriptional changes associated with aneuploidy are not globally defined. Here, a karyotypically diverse set of C. albicans strains revealed that expression generally correlated with gene copy number regardless of the strain karyotype. Unexpectedly, aneuploid strains shared a characteristic transcriptional profile that was distinct from a generalized environmental stress response previously defined in aneuploid yeast cells. This aneuploid transcriptional response led to altered growth and oxidative balances relative to euploid control strains. The increased expression of reactive oxygen species (ROS) mitigating enzymes in aneuploid cells reduced the levels of ROS but caused an acute sensitivity to both internal and external sources of oxidative stress. Taken together, our work demonstrates common transcriptional and phenotypic features of aneuploid C. albicans cells with consequences for infection of different host niches and susceptibility to environmental stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e3ff6416e6e4c37bdd4902bf219f1aee84a69b" target='_blank'>
              Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans
              </a>
            </td>
          <td>
            Anna I Mackey, Robert J. Fillinger, P. S. Hendricks, Gregory J. Thomson, Christina A. Cuomo, Richard J. Bennett, Matthew Z. Anderson
          </td>
          <td>2025-04-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors.
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a59b50595ad67358ac03c0f14d767d313308a1" target='_blank'>
              Abstract 3753: Differential roles of Wnt pathway mutations in colorectal cancer
              </a>
            </td>
          <td>
            Y. Inoue, N. Kakiuchi, Yasuhito Nanya, Kenichi Yoshida, Koichi Watanabe, Y. Takeuchi, Y. Fujii, Yuichi Shiraishi, K. Chiba, T. Yoshizato, Satoshi Miyano, Satoshi Nagayama, K. Obama, Seishi Ogawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, Moritz J Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63affd7e1896b137912e86c03b4cbb1a8fc8724f" target='_blank'>
              Abstract 6243: Mathematical modeling of cell-free DNA fragmentation to map chromatin landscapes in health and cancer
              </a>
            </td>
          <td>
            Brian Sun Liu, Mengran Zhang, Takeshi Sugio, M. Esfahani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de660154998a356e9b85c8c4e7d59ed34b5f5dd0" target='_blank'>
              Abstract 4668: Prognostic significance and therapeutic potential of TOP2A expression in rare cancers
              </a>
            </td>
          <td>
            Jifa Li, Hua Ye, Xiaoqiong Bao, Wenwen Yu, Xiangxiang Chen, yangyang Ni, Jincong Wang, Yubo Shi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d431bff55b178eb615284d91d43a85ede147802" target='_blank'>
              Abstract 5488: Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay
              </a>
            </td>
          <td>
            Darian Williams, Andrew McFadden, Austin J. Hepperla, Min Shen, Tino W Sanchez, Gordon M. Stott, M. Henderson, Ian Davis, Samantha Pattenden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, C. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197e6d30a9ae12cbdea22ce3dfece89aab97864e" target='_blank'>
              Abstract 2491: Conditional generation of mutational profiles in cancer
              </a>
            </td>
          <td>
            Andy J. Wu, Hannan Wong, Aakansha Narain, Vedant Sandhu, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4f3702444f966febda408aa09a277d286c200" target='_blank'>
              Abstract 1409: Reprogramming the alternative polyadenylation of an oncogenic transcription factor as a therapeutic strategy for prostate cancer
              </a>
            </td>
          <td>
            Giulia Pagani, Chiara Casirati, Stefania Fornezza, Roberta Cacioppo, P. Gandellini
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0ba571a5943c70a943f3ca480bbb6fa72ef3ee" target='_blank'>
              Molecular Mechanism of Cross-regulation between DNA Repair and Apoptosis
              </a>
            </td>
          <td>
            Huaping Huang
          </td>
          <td>2025-04-21</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2100d708c16d89d6aad0011f4f25b72f1e562e77" target='_blank'>
              Abstract LB311: Dissecting and targeting EZH2’s non-canonical oncogenic activity in cancers
              </a>
            </td>
          <td>
            Arum Kim, G. G. Wang
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e879216882ba777a52dac02a65cb65b0e331d4" target='_blank'>
              Abstract 5361: Clinical implications of The Cancer Genome Atlas molecular classification system in esophagogastric cancer
              </a>
            </td>
          <td>
            H. Walch, Raktim Borpatragohain, Justin Jee, W. Chatila, Christopher Fong, S. Maron, Geoffrey Y Ku, D. Ilson, Yelena Y Janjigian, Abraham J Wu, Pari Shah, Dan Coit, M. Bains, Valerie W Rusch, B. Park, M. Bott, Katherine Gray, David R. Jones, Michael F. Berger, N. Schultz, V. Strong, D. Molena, S. Sihag
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1fb8f2f98183aa7c1fbc3339ae6c67ad7f2ba6" target='_blank'>
              Abstract 2774: Pan-cancer chromatin state profiling suggests enhancer-heterogeneity and precision medicine approach
              </a>
            </td>
          <td>
            M. Maitituoheti, Emre Arslan, Ayush T. Raman, Christopher Terranova, Kunal Rai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bde3f65de807bb9cec86aaa0dfc6c09be8467d" target='_blank'>
              Abstract 1942: Copy number aberrations in the ctDNA uncover clinical and biological heterogeneity in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Maare Arffman, L. Meriranta, J. Jørgensen, M. Karjalainen-Lindsberg, K. Beiske, Mette Pedersen, Kristina DrottØystein Fluge, S. Jyrkkiö, Peter Brown, Harald Holte, S. Leppä
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae14464f2ce77e42ffad40f3c6898b5621cece3e" target='_blank'>
              PLK4: Master Regulator of Centriole Duplication and Its Therapeutic Potential.
              </a>
            </td>
          <td>
            Muhammad Hamzah, Franz Meitinger, Midori Ohta
          </td>
          <td>2025-04-21</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological cancer globally, with epithelial ovarian cancer (EOC) comprising up to 90% of cases. A molecular characterization linking the histological subtypes with tumor grade in EOC has been suggested. Variations in genetic biomarkers such as BRCA1/2, MSH2, MLH1/6, BRIP1, and RAD51C/D have been studied in EOC. In addition, molecular characteristics, including DNA methylation and RNA transcription, are being explored as potential new biomarkers for the diagnosis and prognosis of this type of neoplasia. The present review focused on the role of DNA methylation and non-coding RNA expression in the development of ovarian carcinomas and their association with diagnosis, prognosis, and the resistance of cancer cells to radiotherapy and chemotherapy. The present review considered the transition from the DNA structure to the RNA expression in ovarian carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e068de948072497de4cda02de43b3f132918d12" target='_blank'>
              Role of DNA methylation and non‑coding RNAs expression in pathogenesis, detection, prognosis, and therapy‑resistant ovarian carcinoma (Review)
              </a>
            </td>
          <td>
            Victor M Del Castillo Falconi, Jenny Godinez Rodriguez, Verónica Fragoso‑Ontiveros, Laura Contreras-Espinosa, Abraham Pedroza‑Torres, José Díaz‑chávez, Luis A Herrera
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="KRAS is a proto-oncogene that contains activating mutations in up to 30% of tumors. Many conventional therapies inhibit both cancerous and normal cells, which may cause toxicity. Thus, programmable mutant-selective targeted inhibitors are needed. Peptide nucleic acids (PNAs) incorporate base sequences analogous to DNA, with modified peptide backbones instead of ribose-phosphate backbones, allowing PNAs to hybridize with DNA with high avidity to suppress transcription. Here, we developed KRAS G12D-selective PNA oligomers with novel cell-penetrating flanking regions. Fluorescein-labeled PNA oligomers displayed high uptake rates in cells and nuclei. Exposure to PNA-delivery peptide conjugates resulted in repression of KRAS G12D mRNA and protein expression within 2 hours and lasting up to 48 hours. Varying cell-penetrating peptide (CPP) compositions and lengths of complementary KRAS sequences were tested using dose-response cell viability assays. These experiments identified configurations that were effective at selectively preventing growth of on-target KRAS G12D cells, while relatively sparing off-target KRAS G12C cells. Electrophoretic mobility shift assays demonstrated in vitro binding and selectivity for KRAS G12D DNA sequences. CPP-PNA-G12D-1 was effective against a panel of pancreatic ductal adenocarcinoma cell lines and patient-derived xenografts in vivo. These results show promise for an enhanced PNA-delivery peptide conjugate strategy as both a tool for studying tumors driven by oncogenic point mutations and as a potential therapeutic strategy to selectively target mutant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2065faaa8c8b01822bceb610976fd9fb9507b1" target='_blank'>
              Mechanism of DNA replication fork breakage and PARP1 hyperactivation during replication catastrophe.
              </a>
            </td>
          <td>
            Pedro Ortega, Elodie Bournique, Junyi Li, Ambrocio Sanchez, Gisselle Santiago, Brooke R. Harris, Josefine Striepen, J. Maciejowski, Abby M. Green, Rémi Buisson
          </td>
          <td>2025-04-18</td>
          <td>Science advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b447161a1815b3267cbfa70059d3d28a80f8d107" target='_blank'>
              Targeting redox-sensitive MBD2-NuRD condensate in cancer cells.
              </a>
            </td>
          <td>
            Heyang Wei, Hongdan Zheng, Siqing Wang, Yun Yang, Ruiqian Zhao, Aihong Gu, Ronggui Hu, Fei Lan, Wenyu Wen
          </td>
          <td>2025-04-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Toxoplasma gondii is an obligate intracellular parasite with a high replication rate that can lead to DNA replicative stress, in turn associated with the generation of DNA double-strand breaks (DSBs). Cells have two main pathways to repair DSBs: non-homologous end joining and homologous recombination repair (NHEJ and HRR respectively). RAD51 is the key recombinase in the HRR pathway. In this work, we achieved endogenous tagging of the RAD51 gene using the Auxin Inducible Degron (AID) system, to generate the clonal line RH RAD51HA-AID. Here we demonstrate that RAD51 is expressed in replicative tachyzoites and establishes damage foci. Auxin-induced knock-down (KD) affects the correct replication of tachyzoites which show loss of synchronization. The use of the RAD51 inhibitor B02 also affects parasite growth, with an IC50 of 4.8 µM. B02 produced alterations in tachyzoite replication and arrest in the S phase of the cell cycle. Additionally, B02 induced tachyzoite to bradyzoite differentiation showing small cyst-like structures. In conclusion, the HRR pathway is necessary for maintaining proper tachyzoite replication under normal growth conditions, supporting that replicative stress occurs during the cell cycle. Our findings also suggest that DNA replication stress can induce bradyzoite differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a99ccd548e29dfb7eccfd9c686305c90fbab532" target='_blank'>
              Toxoplasma gondii RAD51 recombinase is required to overcome DNA replication stress and its inactivation leads to bradyzoite differentiation
              </a>
            </td>
          <td>
            Ana M. Saldarriaga Cartagena, Ayelén Aparicio Arias, C. Cristaldi, Agustina Ganuza, M. M. Gonzalez, María M. Corvi, William J. Sullivan, Laura Vanagas, Sergio O. Angel
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8572ed5aac9e75313380c3e2cd2f1a7e899e36b7" target='_blank'>
              Abstract 1903: Actionable mutational landscape in solid tumors: a case study of ovarian and colon cancer
              </a>
            </td>
          <td>
            Amit Kumar, Samir Agarwal, Pankaj Khanna, Shobit Gupta, Rama Chandran, Ananth S. Bommakanti
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63aefe6d3e777332427bd46910b51b4d2635b3ae" target='_blank'>
              Abstract LB476: Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            R. Harris
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284e8736a9d8069a665b78b16285eb6fd004904b" target='_blank'>
              Escape from X-chromosome inactivation at KDM5C is driven by promoter-proximal DNA elements and enhanced by domain context.
              </a>
            </td>
          <td>
            Samantha B Peeters, S. Baldry, Andrea J. Korecki, Aditi Srinivasan, W. Wasserman, Elizabeth M Simpson, Carolyn J Brown
          </td>
          <td>2025-04-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7defb0dc0b2dcfb00df9b73a0a0c72cd7d43d4" target='_blank'>
              Transposable elements as instructors of the immune system.
              </a>
            </td>
          <td>
            Lisa Schmidleithner, Philipp Stüve, Markus Feuerer
          </td>
          <td>2025-04-29</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f25c99a3e3eab72506356fccfee90b80d5b5a8" target='_blank'>
              The role of DNA methylation and histone modifications in the pathogenesis of hematological malignancies and solid cancers: mechanisms, clinical implications, and therapeutic potential
              </a>
            </td>
          <td>
            Shafee Ur Rehman, Muhammad Abdullah, Zarak Khurshid Khan, Moriom Shurovi
          </td>
          <td>2025-04-14</td>
          <td>Asian Journal of Medical and Biological Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21340650146a334354ebce820c4c26cb9ab3ba1b" target='_blank'>
              Abstract LB386: Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer
              </a>
            </td>
          <td>
            Aashirwad Shahi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e843a00844275db4fbe08c936ffad2c5534db0e" target='_blank'>
              Abstract 99: HMGB1deficiency occurs in a broad range of human cancers and is often associated with unfavorable tumor phenotype
              </a>
            </td>
          <td>
            Viktoria Chirico, Hena Sharifi, M. Tsourlakis, Şeyma Büyücek, Clara Marie von Bargen, Katharina Möller, F. Lutz, D. Dum, C. Hube-Magg, E. Bady, Lisa Hornsteiner, Georgia Makrypidi-Fraune, Piero Caneve, M. Lennartz, Morton Freytag, S. Rico, S. Kind, V. Reiswich, E. Burandt, T. Clauditz, P. Lebok, C. Fraune, T. Krech, S. Minner, A. Marx, W. Wilczak, R. Simon, G. Sauter, S. Steurer, K. Jansen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The accurate control of DNA replication is crucial for the maintenance of genomic stability and cell viability. In this study, we explore the consequences of depleting the replicative DNA Polymerase α (POLA) in the wing disc of Drosophila melanogaster. Our findings reveal that reduced POLA activity induces DNA replication stress and activates the replication checkpoint in vivo. Consistent with this, we demonstrate that dATR, a key component in DNA replication checkpoint signaling, is essential for the maintenance of tissue integrity under conditions of compromised POLA activity. We show that cells within the wing disc exhibiting reduced POLA activity arrest in the G2 phase and undergo p53-dependent apoptosis. We also reveal a critical role for DNA Ligase 4 in sustaining cell viability when POLA function is impaired. Most notably, we report the appearance of oncogenic traits in wing disc cells with diminished POLA activity when apoptosis is suppressed. In this context, the overexpression of the oncogene cdc25/string enhances the oncogenic phenotype. These results indicate that a combination of oncogenic activation, replication stress, and suppression of apoptosis is sufficient to promote the emergence of hallmarks of tumorigenesis, highlighting major implications for cancer development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db1ced614bf54d24af87f355b9a2a18f09d3472" target='_blank'>
              Replication stress promotes cellular transformation in Drosophila epithelium
              </a>
            </td>
          <td>
            Maria Molano-Fernández, I. Hickson, Héctor Herranz
          </td>
          <td>2025-03-12</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise approximately half of eukaryotic genomes and significantly contribute to genome plasticity. In this study, we focused on a specific TE, MERVL, which exhibits particular expression during the 2-cell stage and commonly serves as an indicator of embryonic totipotency. However, its precise role in embryo development remains mysterious. We utilized DRUG-seq to investigate the effects of oxidative damage on genes and TEs expression. Our findings revealed that exposure to hydrogen peroxide (H2O2) could induce DNA damage, apoptosis, and incomplete DNA demethylation in embryos, which were potentially associated with MERVL expression. To further explore its function, antisense nucleotides (ASO) targeting MERVL were constructed to knockdown the expression in early embryos. Notably, this knockdown led to the occurrence of DNA damage and apoptosis as early as the 2-cell stage, consequently reducing the number of embryos that could progress to the blastocyst stage. Moreover, we discovered that MERVL exerted an influence on the reprogramming of embryonic DNA methylation. In MERVL-deficient embryos, the activity of the DNA demethylase ten-eleven translocation 3 (TET3) was suppressed, resulting in impaired demethylation when compared to normal development. This impairment might underpin the mechanism that impacts embryonic development. Collectively, our study not only verified the crucial role of MERVL in embryonic development but also probed its regulatory function in DNA methylation reprogramming, thereby laying a solid foundation for further investigations into MERVL's role. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01143-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ed27bb6383161df0bd1281e88568674842389e" target='_blank'>
              Transposition element MERVL regulates DNA demethylation through TET3 in oxidative-damaged mouse preimplantation embryos
              </a>
            </td>
          <td>
            Lihong Liu, Siyao Ha, Dan Cao, MingQing Li, Zhiling Li
          </td>
          <td>2025-03-12</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Hematologic malignancies (HMs) encompass a diverse spectrum of cancers originating from the blood, bone marrow, and lymphatic systems, with myeloid malignancies representing a significant and complex subset. This review provides a focused analysis of their classification, prevalence, and incidence, highlighting the persistent challenges posed by their intricate genetic and epigenetic landscapes in clinical diagnostics and therapeutics. The genetic basis of myeloid malignancies, including chromosomal translocations, somatic mutations, and copy number variations, is examined in detail, alongside epigenetic modifications with a specific emphasis on DNA methylation. We explore the dynamic interplay between genetic and epigenetic factors, demonstrating how these mechanisms collectively shape disease progression, therapeutic resistance, and clinical outcomes. Advances in diagnostic modalities, particularly those integrating epigenomic insights, are revolutionizing the precision diagnosis of HMs. Key approaches such as nano-based contrast agents, optical imaging, flow cytometry, circulating tumor DNA analysis, and somatic mutation testing are discussed, with particular attention to the transformative role of machine learning in epigenetic data analysis. DNA methylation episignatures have emerged as a pivotal tool, enabling the development of highly sensitive and specific diagnostic and prognostic assays that are now being adopted in clinical practice. We also review the impact of computational advancements and data integration in refining diagnostic and therapeutic strategies. By combining genomic and epigenomic profiling techniques, these innovations are accelerating biomarker discovery and clinical translation, with applications in precision oncology becoming increasingly evident. Comprehensive genomic datasets, coupled with artificial intelligence, are driving actionable insights into the biology of myeloid malignancies and facilitating the optimization of patient management strategies. Finally, this review emphasizes the translational potential of these advancements, focusing on their tangible benefits for patient care and outcomes. By synthesizing current knowledge and recent innovations, we underscore the critical role of precision medicine and epigenomic research in transforming the diagnosis and treatment of myeloid malignancies, setting the stage for ongoing advancements and broader clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1633c8ceb2748c3ad6ca551f7099ac2f4cee0994" target='_blank'>
              Epigenomic insights and computational advances in hematologic malignancies
              </a>
            </td>
          <td>
            Carolyn Lauzon-Young, Ananília Silva, B. Sadikovic
          </td>
          <td>2025-04-12</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e04034cb839d11700bfc365e9345aa1519c5848" target='_blank'>
              Abstract 6712: Epigenetic reactivation of a tumor suppressor program in leukemia
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Calderon, I. Aifantis, A. Tsirigos, Abimbo Lawal, S. H. Rajendran, Sadik Karma, Hussein A. Abbas, M. Y. Yassouf, Zhiping Zhang, Pedro Miura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="DNA topoisomerase 1 (TOP1) is essential for transcription, replication, and repair. Its function relies on two catalytic steps, DNA breakage and rejoining. Inhibitors of the second step prevent DNA rejoining and lead to persistent DNA breaks, acting as topoisomerase poisons, used as anticancer drugs. However, reliable inhibitors of the first step are not available. Here, we provide genetic and molecular evidence supporting that Patulin and, to a lesser extent, Xestoquinol inhibit the first catalytic step of TOP1 in vitro, in yeast and in human cells. Particularly, Patulin prevents the accumulation of TOP1 cleavage complexes caused by the TOP1 poison camptothecin (CPT) in human cells. Moreover, Patulin pretreatment of human or yeast cells reduces DNA damage and the accumulation of DNA breaks upon CPT exposure. Consistent with the protective role of TOP1 against harmful R-loops, Patulin treatment increases R-loops and R-loop-associated cytotoxicity, mimicking the effect of TOP1 silencing. Altogether our findings indicate that Patulin and Xestoquinol are nonpoisoning inhibitors of TOP1, which should potentiate new research approaches in molecular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c91b4af793ee46a1c397d0e749fad65c6dce6ba" target='_blank'>
              Patulin and Xestoquinol are inhibitors of DNA topoisomerase 1.
              </a>
            </td>
          <td>
            E. Tumini, R. Wellinger, Emilia Herrera-Moyano, Patricia Navarro-Cansino, María L. García-Rubio, Daniel Salas-Lloret, Alejandro Losada, M. Muñoz-Alonso, H. Gaillard, Rosa Luna, A. Aguilera
          </td>
          <td>2025-04-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b4705110ed5255e541024e217bd1b38f97bdc5" target='_blank'>
              Abstract 2802: Circular RNA variants of theNUMB gene modulate the oncogenicity of oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Chung-Hsien Chou, Pin-Yu Chen, Shu-Chun Lin, Kuo-Wei Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Simple Summary Ductal adenocarcinoma of the pancreas is a cancer with exceedingly dismal prognosis as less than 5% of all patients survive more than 5 years. Therefore, targeted molecular therapies are urgently needed. MTAP deficiency is currently one of the most promising molecular targets for tumor therapies. MTAP deficiency—often caused by homozygous 9p21 deletion—is easily detectable by immunohistochemistry because it is characterized by a complete MTAP expression loss. It makes tumor cells more susceptible to PRMT5- and MAT2A-inhibiting drugs as well as to pemetrexed (antifolate therapy). However, the success of targeted therapies critically depends on the extent of heterogeneity of the targeted tumor cell trait in clinical tumors. In this study, intratumoral heterogeneity of MTAP deficiency was thus assessed in more than 200 ductal adenocarcinomas of the pancreas in a tissue microarray format and whole-section analysis was carried out by immunohistochemistry and fluorescence in situ hybridization. MTAP deficiency was found in almost 40% of all cancers, was always caused by homozygous deletion, and was rarely heterogenous in TMA and large-section analyses (less than 10%). Pancreatic adenocarcinomas may thus represent an ideal cancer type for studying new drugs targeting MTAP-deficient cancer cells in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa497701c09c398746a0513a639855972ce4cb6a" target='_blank'>
              Deficiency of MTAP Is Frequent and Mostly Homogeneous in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            N. Gorbokon, Katharina Teljuk, V. Reiswich, M. Lennartz, S. Minner, R. Simon, G. Sauter, W. Wilczak, T. Clauditz, Nina Schraps, T. Hackert, Faik G. Uzunoglu, M. Kluth, L. Bubendorf, Matthias Matter, F. Viehweger, Morton Freytag, Frank Jacobsen, Katharina Möller, S. Steurer
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86f33c29e42254bdc63ed12ff517dd771563902" target='_blank'>
              Abstract 3319: Evaluating and targeting DNA repair defects in pediatric high grade glioma
              </a>
            </td>
          <td>
            Evan Savage, Ke Xu, Miaomiao Li, Jason R Myers, E. Rampersaud, Rebekah DeVries, Kamran Mohammad Zahid, Krista Millican, Yawei Hui, Ti-Cheng Chang, Gang Wu, A. Shelat, Christopher L Tinkle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f70cae7350af93e8189b0a48977a508877000d4" target='_blank'>
              Abstract SY27-02: Exploring the significance of somatic mtDNA mutations in the context of acute leukemia
              </a>
            </td>
          <td>
            M. Kundu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pathogenic gene dysregulation can be attributed to chromatin state change that pre-transcriptionally regulates expression. Recent breakthroughs elucidating the rules governing this DNA control layer, an epigenetic code, unlock a modality in precision medicine to target gene dysregulation across myriad diseases. Here we present a modular platform to design programmable mRNA therapeutics, Epigenomic Controllers (EC), that control gene expression through directed epigenetic change. By leveraging natural mechanisms, ECs tune expression levels of one or multiple genes with durable effect of weeks-to-months in female mice following a single dose. We design and characterize ECs to multiple target genes and identify an EC that effectively inhibits the cancer- and inflammatory-disorder-associated multi-gene cluster CXCL1-8. With precision targeting of NF-kB signaling and identification of homologous murine surrogates, ECs significantly reduce neutrophil migration in vivo during acute lung inflammation in female mice. A platform approach to EC design for epigenomic modulation expands treatment frontiers for diverse gene targets, including those considered “undruggable.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c55ad387d82ce60d43ef750c8b35debeea4164" target='_blank'>
              Programmable mRNA therapeutics for controlled epigenomic modulation of single and multiplexed gene expression in diverse diseases
              </a>
            </td>
          <td>
            Charles W O'Donnell, J. Farelli, Houda Belaghzal, Justin Chen, Lauren Beech, James Sullivan, Chevaun D. Morrison-Smith, Stephen Siecinski, Adam Katz, Samuel Mildrum, Mayur Gurnani, Prachi Dhanania, Caitlyn R Webb, Giuliana Castello Coatti, Pranjali Rumale, Daniel F G Costa, Marcus I Gibson, Yaoyu E. Wang, J. Newman, T. McCauley
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b995a18ffa49845c7801e75f621e56fe9ecdff48" target='_blank'>
              Probability-based sequence comparison finds the oldest ever nuclear mitochondrial DNA segments in mammalian genomes
              </a>
            </td>
          <td>
            Muyao Huang, Martin C. Frith
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Epigenetics, encompassing DNA and RNA modifications, has emerged as a prominent area of research in the post-genomic era. Numerous studies have elucidated the impact of epigenetics on tumor regulation. However, the methylation patterns of 5-methylcytosine in cervical cancer as well as its role in the tumor microenvironment and immunotherapy remain poorly understood. Methods Utilizing a comprehensive dataset encompassing samples from 306 patients with cervical cancer from The Cancer Genome Atlas and Gene Expression Omnibus repositories, we conducted an in-depth analysis to evaluate the potential association between the modification patterns of 5-methylcytosine and the infiltration of cells within the tumor microenvironment, taking into account 11 regulators of 5-methylcytosine modification. Subsequently, we employed stepwise regression and Least Absolute Shrinkage and Selection Operator Cox regression to quantify 5-methylcytosine modification patterns in patients with cervical squamous cell carcinoma and endocervical adenocarcinoma, yielding the 5-methylcytosine score. Our study explored the link between the 5-methylcytosine score and clinical characteristics as well as prognostic outcomes in patients with cervical squamous cell carcinoma and endocervical adenocarcinoma. Results A comprehensive analysis of 306 patients with cervical cancer revealed two distinct 5-methylcytosine modification patterns, henceforth labeled as 5-methylcytosine clusters A and B. These clusters exhibited distinct immunological profiles and biological attributes, with 5-methylcytosine cluster A exhibiting a higher degree of immune cell infiltration. Utilizing univariate Cox regression analysis, we identified 367 genes regulated by 5-methylcytosine that were significantly correlated with patient prognosis. This analysis further stratified the samples into three distinct genomic subtypes. Survival analyses indicated that patients belonging to gene cluster C exhibited more favorable survival outcomes than those belonging to gene clusters A and B. Intriguingly, most 5-methylcytosine regulatory factors had higher expression levels in gene cluster B than in gene cluster A. Gene set enrichment analysis of a single sample revealed elevated immune cell infiltration within gene cluster B, indicating a stronger immune response in this cluster. The 5-methylcytosine score feature was utilized to determine the 5-methylcytosine modification pattern in cervical cancer, revealing that patients with low 5-methylcytosine scores exhibited better survival rates, whereas those with high scores had increased mutation frequencies and better treatment responses. Conclusions This research underscores the key role of 5-methylcytosine modification patterns in cervical cancer. Analysis of these patterns will deepen our understanding of the cervical cancer tumor microenvironment, paving the way for the development of more refined and effective immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765be572dba74d9a6436a1344db98db66126fd88" target='_blank'>
              5-Methylcytosine methylation predicts cervical cancer prognosis, shaping immune cell infiltration
              </a>
            </td>
          <td>
            Luyang Su, Ren Xu, Yanan Ren, Shixia Zhao, Liyun Song, Cuiqiao Meng, Weilan Liu, Xuan Zhou, Zeqing Du
          </td>
          <td>2025-04-01</td>
          <td>The Journal of International Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy among women, characterized by extensive heterogeneity stemming from molecular and genetic alterations. This review explores the intricate epigenetic landscape of BC, highlighting the significant role of epigenetic modifications—particularly DNA methylation, histone modifications, and the influence of non-coding RNAs—in the initiation, progression, and prognosis of the disease. Epigenetic alterations drive crucial processes, including gene expression regulation, cell differentiation, and tumor microenvironment interactions, contributing to tumorigenesis and metastatic potential. Notably, aberrations in DNA methylation patterns, including global hypomethylation and hypermethylation of CpG islands, have been associated with distinct BC subtypes, with implications for early detection and risk assessment. Furthermore, histone modifications, such as acetylation and methylation, affect cancer cell plasticity and aggressiveness by profoundly influencing chromatin dynamics and gene transcription. Finally, non-coding RNAs contribute by modulating epigenetic machinery and gene expression. Despite advances in our knowledge, clinical application of epigenetic therapies in BC is still challenging, often yielding limited efficacy when used alone. However, combining epi-drugs with established treatments shows promise for enhancing therapeutic outcomes. This review underscores the importance of integrating epigenetic insights into personalized BC treatment strategies, emphasizing the potential of epigenetic biomarkers for improving diagnosis, prognosis, and therapeutic response in affected patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T. Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59bc43762365a9fef4fc08c5fe2fbf001a4b471" target='_blank'>
              Application of salinomycin in treatment of different types of cancer
              </a>
            </td>
          <td>
            Saba Noreen, Hussan Bin Umar, Rameesha Asghar, Fariha Naz, Farah Naz
          </td>
          <td>2025-04-12</td>
          <td>Journal of Agriculture and Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2712e3b94f1d3c9bfa21826517cacd075e15f89f" target='_blank'>
              Focus on numbers: characterizing protein accumulation at DNA double strand breaks.
              </a>
            </td>
          <td>
            G. Modica, Joannie Roy, Antoine G Godin, Hugo Wurtele, Santiago Costantino
          </td>
          <td>2025-05-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43e1e965f05b28c3994c4e8f0fb6a1f9448be497" target='_blank'>
              Abstract 2856: Pks+ E. coli trigger intestinal stem cell plasticity and early onset colorectal cancer
              </a>
            </td>
          <td>
            Stefanie Gerstberger, Melissa Lumish, S. Hartner, Farheen Shah, Seongmin Choi, Krystal Lum, Christopher Cowley, A. Luthra, Qingwen Jiang, Hyung Jun Woo, Ahmed Mahmoud, H. Walch, Asha Saxena, Tavis J. Reed, A. Cercek, R. Yaeger, Ileana Cristea, Andrew McPherson, F. Sánchez-Vega, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731f9e63512b3fed7df73e42f3ab82567dc1e2b4" target='_blank'>
              Abstract 6440: Spatial transcriptomics of Ewing sarcoma: evolution of tumor associated extracellular matrix from localized to metastasis-associated primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, Masanori Hayashi, Micha SB Raredon, James F. Amatruda
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fe83552a51ec8685227ae6ed241734102c7207" target='_blank'>
              Differential interfacial tension between oncogenic and wild-type populations forms the mechanical basis of tissue-specific oncogenesis in epithelia
              </a>
            </td>
          <td>
            Amrapali Datta, Phanindra Dewan, Aswin Anto, Tanya Chhabra, Tanishq Tejaswi, Sindhu Muthukrishnan, Akshar Rao, Sumantra Sarkar, Medhavi Vishwakarma
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cd802b620e803fe61dfbfe7d21f2c3b02ece22" target='_blank'>
              Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer.
              </a>
            </td>
          <td>
            Guangsheng Pei, Jimin Min, K. Rajapakshe, Vittorio Branchi, Yunhe Liu, B. Selvanesan, F. Thege, Dorsay Sadeghian, Daiwei Zhang, Kyung Serk Cho, Yanshuo Chu, Enyu Dai, Guangchun Han, Mingyao Li, Cassian Yee, Kazuki Takahashi, Bharti Garg, Hervé Tiriac, Vincent Bernard, A. Semaan, Jean L. Grem, T. Caffrey, Jared K. Burks, Andrew M Lowy, Andrew J. Aguirre, P. Grandgenett, Michael A. Hollingsworth, Paola A. Guerrero, Linghua Wang, Anirban Maitra
          </td>
          <td>2025-04-23</td>
          <td>Nature</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="As in many diploid organisms with genetic sex determination, haploid-dominant organisms have also evolved sex chromosomes or extensive genomic regions that lack genetic recombination. An understanding of sex chromosome evolution should explain the causes and consequences of such regions in both diploids and haploids. However, haploids have been little studied, even though differences from sex chromosomes in diploids carry implications concerning the evolution of suppressed recombination in diploid organisms, and make predictions about genome evolution in the sex-linked regions of haploids that can now be tested by approaches using genome sequences. I review these ideas, and the current empirical evidence concerning them, in more detail than recent reviews focusing on progress in understanding the mechanisms involved in sex determination. I also discuss evidence that one specific prediction, that genetic degeneration should be minor in haploids, is not upheld. I suggest that this prediction does not take account of all processes leading to gene loss from sex-linked regions and that profound degeneration may evolve if sex-linked genes become duplicated to autosomes, a process that also appears to occur in diploids. I emphasize types of data that are needed to make progress in testing several of the ideas described.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a631e9206e73bfe8bfc9b31ba9839d98080a985c" target='_blank'>
              Sex chromosome evolution in haploid plants: Microchromosomes, disappearing chromosomes, and giant chromosomes.
              </a>
            </td>
          <td>
            Deborah Charlesworth
          </td>
          <td>2025-04-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f68977168d050d5c5f7c3fa27b15c83e3b09b0a" target='_blank'>
              Abstract 1011: Transposable element patterns across genetic ancestries in prostate cancer patients
              </a>
            </td>
          <td>
            Nicholas Korvink, Lucas Ueta, Hannah Park, Vinay Kumar, Farah Rahmatpanah
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b13792be21e8e149fad24f47b79b2c67bef2640" target='_blank'>
              The protein deacetylase HDAC10 controls DNA replication in malignant lymphoid cells.
              </a>
            </td>
          <td>
            Andreas O Mieland, Giuseppe Petrosino, Mario Dejung, Jia-Xuan Chen, Amitkumar Fulzele, F. Mahmoudi, J.-W. Tu, Al-Hassan M. Mustafa, Yanira Zeyn, Christoph Hieber, Matthias Bros, T. Schnöder, F. Heidel, Sara Najafi, Ina Oehme, Ilse Hofmann, Mike Schutkowski, Sebastian Hilscher, Christian Kosan, F. Butter, S. Bhatia, W. Sippl, Oliver H. Krämer
          </td>
          <td>2025-04-29</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Proline-rich transmembrane protein 4 (PRRT4) has been infrequently studied, with limited literature suggesting its potential as a prognostic marker for gastric cancer. This study aims to investigate the prognostic value of the PRRT4 gene in pan-cancer.


METHODS
We acquired and analyzed data from several platforms, including The Cancer Genome Atlas (TCGA), Genotype Tissue Expression Project (GTEx), Cancer Cell Line Encyclopedia (CCLE), cBioPortal, HPA, and TIMER 2.0. In addition, we have further analyzed the data using multivariate analyzes and RT-qPCR. In vitro experiments were performed to detect the proliferation and apoptosis of AML cells before and after PRRT4 knockdown.


RESULTS
PRRT4 exhibited low expression in 10 types of cancers and high expression in 3 types, and this expression was significantly correlated with tumor stage, age, and gender across various cancer types. PRRT4, identified as a potential independent prognostic factor for overall survival (OS) in several cancers including LAML, PAAD, SKCM, STAD, THYM, and UVM, and exhibited a high frequency of mutation in UCEC. Moreover, PRRT4 was found to be correlated with DNA methylation and immune infiltration in various cancers. Ultimately, in the multivariate analysis model, PRRT4 was discerned as an independent prognostic biomarker for AML, predicated on the statistics based from our institution. After PRRT4 knockdown, the proliferation ability of THP1 cells was significantly enhanced, and the apoptosis ratio was significantly decreased.


CONCLUSION
PRRT4 may serve as a potential therapeutic target and prognostic marker for various malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66047aa9da0269c010edbf7b80afbec9051b4fd1" target='_blank'>
              Pan-cancer analysis reveals PRRT4 is a potential prognostic factor of AML.
              </a>
            </td>
          <td>
            Wenqiong Xiang, Fangjun Chen, Hao-Yu Zhou, Guilin Ren, Guangliang Qiang, Li Wang
          </td>
          <td>2025-04-25</td>
          <td>Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35acfa591bf2d48662dc328bde4741026068f8bb" target='_blank'>
              Neoantigens and Stochastic Fluctuations Regulate T Cell Proliferation in Primary and Metastatic Malignant Brain Tumors
              </a>
            </td>
          <td>
            Maheshwor Poudel, William Stewart, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f608a4fe82676c1f869676898474a93427b4d" target='_blank'>
              Abstract LB449: InTraSeq unveils transcriptomic and proteomic landscapes of Th17 differentiation
              </a>
            </td>
          <td>
            Majd M. Ariss
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0640112dbbdf90099adbfa03bf4452da14a83a22" target='_blank'>
              Abstract 2664: Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, K. Suzawa, Hidetaka Yamamoto, Shinichi Toyooka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Epigenetic regulation of gene expression is essential for cellular development and differentiation processes in higher eukaryotes. Modifications of cytosine, in particular 5-methylcytosine (5mdC), in DNA play a central role through impacting chromatin structure, repressing transposons, and regulating transcription. DNA methylation is actively installed by DNA methyltransferases and reversed through Tet-dioxygenase-mediated oxidation of 5mdC to 5-hydroxylmethylcytosine (5hmdC), 5-formylcytosine (5fdC), and 5-carboxycytosine (5cadC). It is crucial to understand the role of these epigenetic DNA modifications in cellular differentiation and developmental processes, as well as in disease state mapping and tracing of 5mdC and its oxidized forms. In bisulfite sequencing, which has been the benchmark for mapping 5mdC for the last few decades, degradation of the majority of genetic material occurs through harsh chemical treatment. Alternative sequencing methods often utilize Tet-enzyme-mediated oxidation of 5mdC to locate 5mdC and 5hmdC in genomic DNA. Herein, we report the development of novel Tet3-variants for oxidation-based bisulfite-free 5mdC- sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e804da833f12d96ee4c49d305fc6d792d26acb4a" target='_blank'>
              Novel Tet3 enzymes for next-generation epigenetic sequencing.
              </a>
            </td>
          <td>
            Özge Simsir, Tobias Walter, Hanife Sahin, Thomas Carell, Sabine Schneider
          </td>
          <td>2025-03-10</td>
          <td>RSC chemical biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CRISPR-associated (Cas) transposases (CAST) are RNA-guided systems capable of programmable integration of large segments of DNA without creating double-strand breaks. Engineered Cascade CAST function in human cells but are challenging to deploy due to the complexity of the targeting components. Unlike Cascade, which require three Cas proteins, type V-K CAST require a single Cas12k effector for targeting. Here, we show that compact type V-K CAST from uncultivated microbes are repurposable for programmable DNA integration into the genome of human cells. Engineering for nuclear localization and function enables integration of a therapeutically relevant transgene at a safe-harbor site in multiple human cell types. Notably, off-targets are rare events reproducibly found in specific genomic regions. These CAST advancements are expected to accelerate applications of genome editing to therapeutic development, biotechnology, and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729b5eccbfaf3e212cbd8bcab377e318bdda9fa9" target='_blank'>
              Integration of therapeutic cargo into the human genome with programmable type V-K CAST
              </a>
            </td>
          <td>
            Jason Liu, Daniela S Aliaga Goltsman, Lisa M. Alexander, Khak Khak Khayi, Jennifer H Hong, Drew T Dunham, Christine A Romano, Morayma M. Temoche-Diaz, Shailaja Chadha, Rodrigo Fregoso Ocampo, Jennifer Oki-O'Connell, Owen P Janson, K. Turcios, Liliana Gonzalez-Osorio, Jared Muysson, Jenat Rahman, Sarah M Laperriere, Audra E. Devoto, C. Castelle, Cristina N. Butterfield, Gregory J Cost, Christopher T. Brown, Brian C. Thomas
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a30e97ab73c3d9f10f09249e3cecf21c25bbdf3" target='_blank'>
              Establishment of SHON Gene Knockout Breast Cancer Cell Line using CRISPR-Cas9 Technology
              </a>
            </td>
          <td>
            Alyssa Clemas
          </td>
          <td>2025-03-26</td>
          <td>Rangahau Aranga: AUT Graduate Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d685ee84bfeabbf621f354633519c29df6bcad68" target='_blank'>
              Abstract 4399: Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer
              </a>
            </td>
          <td>
            Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew W. Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The transcription factor Prdm14 is a potent oncogene implicated in the initiation of many cancers. PRDM14 resets and maintains the pluripotent state in normal cells, but the molecular mechanisms through which PRDM14 drives oncogenesis are poorly understood. Here, we interrogated the heterogeneity of Prdm14-expressing cells in a T cell lymphoblastic leukemia/lymphoma mouse model. Using mass cytometry (CyTOF) of bone marrow at a pre-leukemic timepoint, an unexpected abnormal progenitor B cell population was identified. Prdm14-expressing progenitor B cells demonstrated short-term self-renewal and a block in differentiation when transferred to syngeneic hosts. Consistently, aged host mice succumb to a highly penetrant B-LL. Single-cell RNA-seq analyses suggests that the expression signature of these pre-leukemia cells is more consistent with that of B-1 cells than B-2 cells. B-1 cells are a self-renewing population of unconventional B cells established during embryonic development. Overlaying the chromatin binding of transcriptional marks H3K4me1 and H3K4me3 with PRDM14 suggests that PRDM14 initiates cancers through promiscuous DNA binding, activating oncogenic pathways and skewing development towards a self-renewing B-1-like phenotype. Together, our data show that Prdm14 can initiate premature T and B cell cancer programs when expressed in hematopoietic progenitor cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93043-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d78c4f8c37e578dfe03b8165770afdb4f53da3e6" target='_blank'>
              Functional and molecular single-cell analyses implicate PRDM14 in the initiation of B cell leukemia in mice
              </a>
            </td>
          <td>
            Lauren J. Tracey, Mahmoud El-Maklizi, Dustin J. Sokolowski, Miki S Gams, Travis Brooke-Bisschop, Julie Ruston, Christine Taylor, Alexandra Khozin, S. Rajakumar, Michael D. Wilson, C J Guidos, Monica J. Justice
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>