<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-25 12:38:02 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>63</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>306</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>260</td>
          <td>157</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f686dc6f9303431a89daa1a7f74014cc18c394d" target='_blank'>
              Coordinated inheritance of extrachromosomal DNAs in cancer cells
              </a>
            </td>
          <td>
            King L. Hung, Matthew G Jones, I. Wong, Ellis J Curtis, Joshua T. Lange, B. J. He, J. Luebeck, R. Schmargon, Elisa Scanu, Lotte Brückner, Xiaowei Yan, Rui Li, Aditi Gnanasekar, R. Chamorro González, J. Belk, Zhonglin Liu, Bruno Melillo, V. Bafna, J. R. Dörr, Benjamin Werner, Weini Huang, Benjamin F. Cravatt, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 The epidermal growth factor receptor (EGFR) is the most frequently altered gene in GBM, with EGFR amplification occurring in approximately 40% of all glioblastomas, half of which exhibit the EGFRvIII deletion variant. EGFR amplification often occurs through the formation of extrachromosomal DNA (ecDNA), where EGFR typically co-amplifies with upstream enhancers. ecDNA is commonly found in a wide range of cancer types, particularly Glioblastoma (GBM), but is rare in normal cells. The uneven segregation of ecDNAs results in oncogene copy number variations in subclones, producing highly heterogeneous tumor cells and providing a selective advantage to cancer cells in response to changing environmental conditions. Meanwhile, ecDNAs keep evolving through fusion, deletion, or integration of new fragments, resulting in alterations to their structure and sequence. Some ecDNAs can reintegrate into chromosomal DNA, disrupting the integrity of chromosomal genes or rewiring chromosomal gene expression via ecDNA resident enhancers. However, the dynamics of EGFR-containing ecDNA (ecEGFR) have not been thoroughly investigated across primary and recurrent GBM. In this study, we profiled 47 glioblastoma specimens (both primary and recurrent) using a multi-omic approach, including WGS, RNA-seq, and ATAC-seq, identifying ecDNA in 14 samples, 10 of which carried EGFR. EGFR deletions were observed in 4 ecEGFR-positive samples, with 2 being the vIII variant. Interestingly, EGFR deletions predominantly occurred in the primary GBM. These ecEGFR samples exhibited significantly high EGFR copy number, gene expression, and chromatin accessibility. Additionally, there were samples with ecEGFR in the primary tumor but not in the recurrent tumor, or ecEGFR appears only in recurrent tumor. One patient showed an ecEGFR deletion in the primary tumor, while the recurrent tumor exhibited wild type ecEGFR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed628ab91baba42cb0f02631625fb1fc581085a" target='_blank'>
              EPCO-52. INVESTIGATING EGFR CONTAINING EXTRACHROMOSOMAL DNA DYNAMICS IN GBM
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Kadir Akdemir
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA), highly amplified circular DNA, is widespread in human cancers and serves as a primary location for oncogene amplification. Due to its dynamic structural rearrangement and asymmetric inheritance during cell division, ecDNA increases tumor genome complexity, contributing to drug resistance and shortened patient survival. Here, we show that ecDNA-mediated genomic amplification is highly associated with transcript fusion events, which are prevalent in cancer and often lead to tumor development. We discovered that transcripts with the highest fusion events originate from genes amplified on ecDNA across ecDNA(+) cell lines. Notably, PVT1 (Plasmacytoma Variant Translocation 1) long noncoding RNA, which is known to be a hotspot for chromosomal translocation, is the most frequently fused RNA species in MYC/PVT1-amplified ecDNA(+) cancer cells. Exon 1 of PVT1 is the predominant fusion partner and confers RNA stability, increasing the RNA abundance of the partner oncogene. Using a model cell line with PVT1-MYC fusion on ecDNA, we found that RNA expression of PVT1-MYC increases in vivo in an ecDNA-dependent manner, while canonical MYC does not show a prominent increase. Additionally, ectopic expression of PVT1- MYC in MYC-depleted cancer cells provides higher rescue efficiency than canonical MYC. In contrast, a PVT1-MYC mutant that is unable to enhance RNA stability fails to rescue, highlighting the functional significance of PVT1-mediated RNA stabilization in cancer. This study unveils the link between PVT1 fusion and ecDNA in cancer, providing insights for developing diagnostic and therapeutic approaches tailored to target ecDNA.
 Citation Format: Hyerim Yi, Shu Zhang, Matthew G Jones, Julia A Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, Paul S Mischel, Howard Y Chang. PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b099680e1d89fbf4dffca6c72e1bc67243614d39" target='_blank'>
              Abstract PR014: PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Matthew G Jones, J. Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 Introduction: Myeloma has the most complex genomic architecture among blood cancers. Central to this complexity are structural variants (SVs), large-scale genomic changes with the capacity to disrupt chromatin organization. SVs can modify the epigenomic landscape near DNA breakpoints, leading to gene dysregulation without directly altering coding sequences or DNA copy number. Here, we elucidate SVs involving the previously hidden regions of the human genome, namely compact heterochromatin including centromeric regions. These dense and repetitive regions, traditionally dismissed as “junk DNA”, emerge as critical areas of investigation.
 Methods: Short-read whole-genome sequencing (srWGS) data from 23 CoMMpass cases were mapped to the CHM13v2 genome assembly by BWA-MEM. RNA-seq CoMMpass (N = 928) and control GSE148924 (N = 12) sets were processed by DESeq2 normalization, variance-stabilizing transformation, and limma batch correction. Gene under-/over-expression was assessed by Z-score (>2/<-2) and log2-fold change (>1/<-1). Patient-derived xenografts (PDXs, N = 13) and KMS27 and PCM6 cell lines were studied by multiomics: srWGS (Illumina), long-read HiFi WGS (PacBio), and Micro-C (CantanaBio); including cases with t(11;14), t(4;14), t(14;16), and hyperdiploidy. Raw reads were aligned to the CHM13v2 assembly. HiFi alignments were phased by WhatsHap. In-house algorithms were built for the analysis of SVs involving repetitive regions in srWGS and DNA methylation within the HiFi reads. Copy-number variants were assessed by CNVRobot. Micro-C data were processed with Juicer tools.
 Results: SVs affecting compact heterochromatin were detected in 61% of CoMMpass cases (14/23) and frequently involved (peri)centromeres of chromosomes 8, 12, 15, and 16. Breakpoints at partner loci often disrupted tumor-suppressor genes (CYLD, GAS8, UNC5D) and recombined near proto-oncogenes (MYC, NFKB1), which increased gene expression compared to control plasma cells.
 In a PDX case, we resolved the +1q as “jumping translocations” between 1q12 pericentromeric heterochromatin and 6q15, 15p11.2, and 18q11.2. The previous hg38 assembly could not resolve t(1;6) and t(1;15) due to DNA breakpoints being present in gaps of this reference genome. HiFi reads enabled breakpoint mapping at 1q (impossible by srWGS), and Micro-C data confirmed derivative chromosome der(1)t(1;18), der(6)t(1;6), and der(15)t(1;15) interactions, and were associated with loss of the partner chromosome arm. DNA methylation analysis of HiFi phased data showed significant hypomethylation of partner loci juxtaposed to the 1q pericentromere. We identified this pattern as a mark of compact heterochromatin rearrangements, confirming an “open chromatin” state, consistent with increased transcription in the CoMMpass data.
 Allele-specific DNA methylation changes were also observed in key chromosomal SVs involving super-enhancers, such as t(11;14) CCND1-IGH, t(4;14) NSD2-IGH, t(14;16) IGH-MAF, and t(6;8) TXNDC5-MYC. For example, in a t(11;14) PDX case, we detected hypomethylation (~2 kb) at the chr11 breakpoint, followed by 225 kb hypermethylation and subsequent hypomethylation spanning CCND1. Micro-C data showed interactions between these hypomethylated regions on chr11 and the IGH promoter through introduction of a de novo translocation-specific regulatory TAD loop absent in non-t(11;14) samples, allowing close interaction of the translocation breakpoint with CCND1. No methylation changes were observed in other SVs lacking super-enhancers or compact heterochromatin involvement.
 The level of compact heterochromatin breakage was not linked to global hypomethylation. However, global methylation analysis showed a higher proportion of methylated CpG sites in the t(4;14) group (71.8%, N = 4) compared to other molecular groups: t(11;14) (44.8%, N = 4), t(14;16) (46.1%, N = 2), and hyperdiploidy (34.1%, N = 4). This supports the role of NSD2 in forming H3K36me2 histone marks, essential for maintaining DNA methylation.
 Conclusions: Utilizing novel computational algorithms, complete genome assembly, and multiomics data integration, we provide the first genomic evidence of SVs affecting compact heterochromatin as common changes in the myeloma genome. These rearrangements alter the DNA topology, epigenomic code and gene expression in juxtaposed loci, suggesting a driving role in myeloma genome instability.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce365e86100cc20873a8250a59c923a1a8ca733" target='_blank'>
              Multiomics Data Integration in the Complete Myeloma Genome Reveals Frequent Centromeric Rearrangements and Their Epigenomic Consequences
              </a>
            </td>
          <td>
            A. Mikulasova, Enze Liu, Nathan Becker, Parvathi Sudha, R. Abonour, Brian A Walker
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through controlled double-strand breaks. While TOP2A is a chemotherapy target in proliferating cells, the ubiquitously expressed TOP2B is a potential off-target. Here we explore roles of TOP2B in mutagenesis by generating DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analysing these maps for driver mutations and mutational processes in 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants (SVs), especially at evolutionary conserved sites displaying high transcription and long-range chromatin interactions. TOP2B binding underlies SVs and hotspot mutations in cancer-driving genes such as TP53, MYC, FOXA1, and VHL, and many cis-regulatory elements. We show that the TOP2B-bound mutational hotspot at RMRP drives tumor initiation and growth in vivo. These data highlight TOP2B as a protector of the genome from topological challenges whose aberrant activity promotes driver and passenger mutations in cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0cbfffd4039fee7d6be03df4eefb1698ccef5df" target='_blank'>
              Topoisomerase IIb binding underlies frequently mutated elements in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, S. Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, D. Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, Daniel Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) can drive oncogene amplification, gene expression and intratumor heterogeneity, representing a major force in cancer initiation and progression. The phenomenon becomes even more intricate as distinct types of ecDNA present within a single cancer cell. While exciting as a new and significant observation across various cancer types, there is a lack of a general framework capturing the dynamics of multiple ecDNA types theoretically. Here, we present novel mathematical models investigating the proliferation and expansion of multiple ecDNA types in a growing cell population. By switching on and off a single parameter, we model different scenarios including ecDNA species with different oncogenes, genotypes with same oncogenes but different point mutations and phenotypes with identical genetic compositions but different functions. We analyse the fraction of ecDNA-positive and free cells as well as how the mean and variance of the copy number of cells carrying one or more ecDNA types change over time. Our results showed that switching does not play a role in the fraction and copy number distribution of total ecDNA-positive cells, if selection is identical among different ecDNA types. In addition, while cells with multiple ecDNA cannot be maintained in the scenario of ecDNA species without extra fitness advantages, they can persist and even dominate the ecDNA-positive population if switching is possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f328238baa1a06ddbda4cdafb65854f0db985125" target='_blank'>
              Population dynamics of multiple ecDNA types
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-11-21</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09784c909701bec6584bd91b9c974eb6800207e8" target='_blank'>
              Enhancing transcription–replication conflict targets ecDNA-positive cancers
              </a>
            </td>
          <td>
            Jun Tang, Natasha E Weiser, Guiping Wang, Sudhir Chowdhry, Ellis J Curtis, Yanding Zhao, I. Wong, Georgi K. Marinov, Rui Li, Philip Hanoian, Edison Tse, Salvador Garcia Mojica, Ryan J. Hansen, Joshua Plum, Auzon Steffy, S. Milutinovic, S. T. Meyer, J. Luebeck, Yanbo Wang, Shu Zhang, Nicolas Altemose, Christina Curtis, W.J. Greenleaf, V. Bafna, S. J. Benkovic, Anthony B Pinkerton, S. Kasibhatla, Christian A Hassig, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Somatic genomic rearrangements are hallmarks of cancer. Complex genomic rearrangements (CGRs) involving multiple intertwined structural alterations are often present in tumor genomes. CGRs frequently harbor oncogenic drivers, but their genomic architectures and etiologies are poorly understood. We used deep-coverage optical mapping technology to profile the genomic landscapes of normal lung tissues, benign pulmonary lesions, carcinoma in situ, and advanced carcinomas to examine the patterns of genome disorganization and instability in different stages of carcinogenesis in lung. Large rearrangements and CGRs were prevalent in the carcinomas. We developed omcplR to resolve the architecture of CGRs and predict their genesis from optical mapping data using genome-graph concept. We found that CGRs often arose from hierarchical combinations of multiple, localized simple structural variations, and harbored allelic heterogeneity at the affected loci. Rearrangement patterns and associated genomic signatures suggested that chromoanasynthesis was a likely prevalent mechanism driving complex genomic rearrangements. The early rearrangement junctions in intra-chromosomal CGRs were usually localized within the same chromatin domains, but the late junctions in advanced tumors had more heterogeneous contexts suggesting progressive organizational heterogeneity. The CGRs, especially the late events therein were under positive selection. A composite signature of genomic alterations including the CGRs captured the trajectory of progressive genomic disorganization and instability with carcinogenesis in lung and underscored the extent of genomic structural heterogeneity among the in-situ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0b28163735146a5869b061b33efdd0283e9308" target='_blank'>
              Deconstructing evolutionary histories of complex genomic rearrangements in lung malignancies
              </a>
            </td>
          <td>
            Xiaoju Hu, Vignesh Venkat, Gregory R Riedlinger, Zhiyuan Shen, Jyoti Malhotra, Subhajyoti De
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc1dada885aaee19cff1e9a95787aee6cf49a1c" target='_blank'>
              Comprehensive profiling of extrachromosomal circular DNAs in colorectal cancer progression
              </a>
            </td>
          <td>
            Lu Lu, Mingjie Chen, Guicheng Zhang, Yujing Liu, Xiangyuan Xu, Zenghua Jiang, Yangxian Xu, Tao Liu, Fan Yang, Guang Ji, Hanchen Xu
          </td>
          <td>2024-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="

 Introduction: Complex structural variants (SVs) are important markers of high-risk disease in multiple myeloma, and include chromoplexy, templated insertions, and chromothripsis.Despite being able to identify these events in whole genome sequencing data, we still know relatively little about how these are generated and their selective advantage. Chromothripsis is particularly complicated and often involves dozens of SVs and copy number changes, though the functional impact has not been systematically investigated.
 Methods: Paired short-read (Illumina, average depth 75x), long-read HiFi (PacBio, 16x) and Micro-C (Cantata Bio, 934M reads) data were sequenced from 13 patient-derived xenograft (PDX) samples and 2 cell lines, covering t(11;14), t(4;14), and hyperdiploid subgroups. Reads were aligned to the hg38 reference while haplotype phasing was conducted using small variants detected from HiFi data (deepvariant). De novo assembly was conducted for haplotypes using Micro-C and HiFi data. Chromothripsis events were reconstructed by integrating short-read (Manta), HiFi (pbsv), Micro-C (EagleC) data and assembled haplotypes. Dysregulated genes at the SV region were subsequently identified with RNA-sequencing. Moreover, reconstructed chromosome organization including loops/topologically associated domains (TADs) were identified from Micro-C data (Neoloopfinder). CUT&Tag-IT was conducted for six histone marks to determine chromatin states as well as identifying enhancers/repressors. Additionally, haplotype-level DNA methylation was determined from HiFi data while differential methylation regions were calculated and mapped onto breakpoints.
 Results: Three chromothripsis events were found in three PDX samples. One occurred on chromosome 2p which was shattered into 32 large fragments (>500 kb), 10 and 13 of which were further amplified or deleted, respectively. 66% of breakpoints did not have homologous joins (N<=1), which, combined with pathway analysis indicated that non-homologous end joining was the dominant DNA damage repair mechanism. SNP B allele frequencies indicated chromothripsis only occurred on one allele.
 Analysis of chromosome 2 breakpoints and interactions indicated that the chromothripsis event occurred through chromosome shattering, shuffling, and amplification of regions. This event was followed by a swap of material resulting in the insertion of the chromothripsis event into chr 18 and a 10 Mb region of chr 18 into chr 2, thereby affecting multiple chromosomes. Interestingly, chromothripsis-derived insertions into other chromosomes were also observed in 67% of 133 samples with chromothripsis events in the CoMMpass cohort, indicating that duplication of chromothripsis SVs onto other chromosomes is a common mechanism.
 Among 236 protein coding genes in the region affected by chromothripsis, 109 were significantly dysregulated (|Log2FC|>0.49, p<0.05, T-test). 84 were affected by copy number (CN) loss or gain, including 3 tumor suppressor genes (ASXL2, CN=1) and 4 oncogenes (MYCN, CN=4). Super enhancers (SE) dysregulated 59 genes including a de novo SE defined by H3K27ac detected in a neo-chromatin loop with MYCN which was also hypomethylated at the DNA level (10% vs. 23%, p=0.02, Mann Whitney U test). Also, a SE near KCNS3 was juxtaposed to CAD (an oncogene involved in nucleotide synthesis and proliferation), leading to its over-expression (Log2FC=0.7 p=0.01, CN=2); another SE near MTA3 was juxtaposed to EPCAM, a cell surface adhesion protein, leading to a significant upregulation (Log2FC=3.7, p=1x10-8, CN=4); a t(2;18) breakpoint was found in the promoter of the tumor suppressor FOXN2, resulting in downregulation (Log2FC=-0.5, p=0.08, CN=2); significant allele-specific hypomethylation (2% vs. 37%, p=0.008) was found on the chromothripsis allele in the promoter of oncogene SOS1, which was significantly over-expressed (Log2FC=2.1, p=3x10-6, CN=4).
 Conclusion:
 Reconstruction of a chromothripsis event revealed various mechanisms that lead to the dysregulation of multiple oncogenes and tumor suppressors. Our analysis indicated a highly dynamic environment that chromothripsis created to allow MM cells to manipulate gene regulation to gain a proliferative advantage, explaining the negative prognostic impact of these events.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd643141122f40f0a74fd3c728eaec1ce12cfce" target='_blank'>
              Reconstructing Catastrophic Chromothripsis Events Using Multiomic Data Reveals Their Functional Impact in Multiple Myeloma
              </a>
            </td>
          <td>
            Enze Liu, Nathan Becker, Parvathi Sudha, A. Mikulasova, R. Abonour, Brian A Walker
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Independently of the pathways or circuits deregulated in cancer cells, these present altered transcription patterns, often also direct consequence of deregulation of transcription factors. In this sense, also the RNA Polymerase complexes responsible for transcription can be affected in cancers. We find that upregulations of RNA Polymerase II subunits, especially the largest ones, correlates with poor cancer patients’ outcome across a range of tumor types, presenting increased genome instability. Overexpressing the subunits RPB1, RPB3 and RPB4 in cells we find that these induce DNA damage. However, the mechanisms behind this increased genome instability are specific for each subunit, linked to the unique transcription alterations generated by the subunit overexpression. Importantly, we find significant overlap between the genes with more DNA damage in our cell line models and those more affected in cancers with subunit upregulation, indicating that upregulations could be responsible for some of the phenotypes present in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1c9c056673cff0cd8a0e04f5d4022f59e547e9" target='_blank'>
              RNA Polymerase II subunits overexpressions induce genome instability and deregulate transcription
              </a>
            </td>
          <td>
            Martina Mustè Sadurnì, Megan Louise Jones, Lucas Pavlou, Mohamed Radi, Smaragda Kompocholi, Paolo Passaretti, Agnieszka Gambus, Lea H. Gregersen, Marco Saponaro
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd2a73c90ca9b2dfa75e988092f2839d133a616" target='_blank'>
              Origins and impact of extrachromosomal DNA
              </a>
            </td>
          <td>
            Chris Bailey, O. Pich, K. Thol, T. Watkins, J. Luebeck, A. Rowan, Georgia Stavrou, Natasha E Weiser, Bhargavi Dameracharla, R. Bentham, Wei-Ting Lu, Jeanette Kittel, S. Y. C. Yang, B. Howitt, Natasha Sharma, Maria Litovchenko, R. Salgado, King L. Hung, A. Cornish, David A Moore, R. Houlston, V. Bafna, Howard Y. Chang, S. Nik-Zainal, N. Kanu, N. Mcgranahan, Adrienne M. Flanagan, P. Mischel, M. Jamal-Hanjani, C. Swanton
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Purpose of the study: to determine the role of retroelements in chromoanagenesis mechanisms in cancer etiopathogenesis.Material and Methods. The search for relevant sources was carried out in the Scopus, Web of Science, PubMed, Elibrary systems, including publications from February 2002 to December 2023. Of the 864 scientifc articles found, 60 were used to write a systematic review.Results. According to original works and meta-analyses results, the cause of complex chromosomal rearrangements during cancer development may be retroelement pathological activation. Chromoanagenesis involves LINE1, SVA, Alu, HERV, which cause double-stranded DNA breaks, insertions in tumor suppressor genes region, the formation of chimeric oncogenes due to retroelement use as new promoters, and function as molecular “band-aids” in non-homologous end junctions and form bridges of distal DNA fragments. Global structural rearrangements of chromosomes observed during chromoanagenesis may be consequences of retroelements activation, which participate in non-allelic homologous recombination and in microhomology-mediated joining of ends characteristic. Certain types of neoplasms, such as colon cancer, are characterized by both high levels of chromothripsis and retroelement activity. In head and neck squamous cell carcinoma, chromoplexy is specifc, the sources of sequences at the breakpoints of which are retroelements. During chromoanagenesis, activation of proto-oncogenes and inactivation of tumor suppressor genes are observed, which is also a consequence of retroelement activation. This is due to the presence of retroelement sequences in proto-oncogenes promoter regions and introns (which become the basis for chimeric oncogene formation) and hot spots of insertional mutagenesis in tumor suppressor genes (transpositions into these regions inactivate these genes).Conclusion. The results obtained on the driver effect of retroelements in chromothripsis, chromoplexy and chromoanasynthesis mechanisms, which are the basis for the formation of clonal evolution of tumors, indicate promise of targeted therapy aimed at silencing the activity of retroelements in cancer patients treatment. For this purpose, it is possible to use microRNAs complementary to retroelements, which are also involved in tumor development, as tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3dd701cb78af4fbe30529399084bdaa803a811" target='_blank'>
              Participation of retroelements in chromoanagenesis in cancer development
              </a>
            </td>
          <td>
            R. N. Mustafin
          </td>
          <td>2024-11-15</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer genomes frequently carry APOBEC (apolipoprotein B mRNA editing catalytic polypeptide-like)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and/or evolution. However, the pure mutagenic impacts of the specific enzymes among this family that are responsible for APOBEC-associated mutagenesis remain unclear, particularly the comparative mutagenic activities of APOBEC3A and APOBEC3B. Here, we investigated the mutagenic contributions of these enzymes through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC3A or APOBEC3B cassettes. Our findings demonstrated that transcriptional APOBEC3A upregulation led to the acquisition of a massive number of genomic mutations in a few cell cycles. By contrast, APOBEC3B upregulation did not generate a substantial number of mutations in gastric epithelium. APOBEC3B-associated mutagenesis remained insignificant even after a combined inactivation of TP53. Based on the spectrum of acquired mutations after APOBEC3A upregulation, we further analyzed APOBEC3A-associated mutational signatures, encompassing indels mainly composed of 1bp deletions, characteristics of clustered mutations, and selective pressures operative on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4025cc28df4328626e6c695274ccd359914520" target='_blank'>
              APOBEC3A, not APOBEC3B, drives deaminase mutagenesis in human gastric epithelium
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Traditional drug discovery efforts have largely focused on targeting rapid, reversible protein-mediated adaptations to undermine cancer cells’ resistance to therapy. However, cancer cells also exploit DNA-based strategies, typically viewed as slow, irreversible, and unpredictable changes like point mutations or the selection of drug-resistant clones. Contrary to this perception, extrachromosomal DNA (ecDNA) represents a form of DNA alteration that is rapid, reversible, and predictable, playing a crucial role in cancer’s adaptive response. In this study, we present a novel post-processing pipeline for the automated detection and quantification of ecDNA in Fluorescence in situ Hybridization (FISH) images using the Microscopy Image Analyzer (MIA) tool. Our approach is particularly designed to monitor ecDNA dynamics during drug treatment, providing a quantitative framework to understand how ecDNA enables cancer cells to swiftly and reversibly adapt to therapeutic pressure. This pipeline not only offers a valuable resource for researchers aiming to automate ecDNA detection in FISH images but also sheds light on the adaptive mechanisms of ecDNA in response to epigenetic remodeling agents like JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a33656032abe4b14c89c4da5e33bf1e9ab825af" target='_blank'>
              Leveraging AI to Automate Detection and Quantification of Extrachromosomal DNA (ecDNA) to Decode Drug Responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Correct chromosome organization in the cell nucleus is essential for genome function. However, dynamics and regulations of large-scale chromosomal conformations beyond single compartments at larger than ten megabases in vivo in single cells remain largely unknown. Here we use CRISPR-Sirius, a high-resolution and high-sensitivity real-time imaging technique, to directly visualize distinctions in large-scale chromosomal conformations between live osteosarcoma (OS) cells and osteoblasts, suggesting extensive chromatin reorganization during cell transformation. A surprising discovery is that chromosome 19 long arm is primarily extended in osteoblasts and maintained by H3K27me3. Extended chromosome conformation has been reported in fly and mouse but not in human cells yet. However, in OS cells, chromosome 19 primarily folded into collapsed conformations, which reshape in minutes and are regulated by the chromosome architectural proteins CTCF and cohesin in the presence of H3K27ac. Changes in chromosome conformations by knocking down the cohesin subunit RAD21 resulted in altered gene expression, including proto-oncogenes. Transcription inhibition by a small molecule inhibitor did not have detectable effects on large-scale chromosome conformation, suggesting that local transcription events have limited effects on large-scale chromosomal architecture. Our results provide unique insights into the complex regulatory mechanisms of endogenous large-scale chromosome organization in normal and transformed osteogenic tissues. One-sentence summary Regulations of chromosome territory organization in normal and cancer bone cells visualized by CRISPR-based live-cell imaging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7533cb6aa378e00ab386aa48ab39b24e7989134" target='_blank'>
              Differential Regulation of Large-scale Chromosome Conformations in Osteoblasts and Osteosarcoma
              </a>
            </td>
          <td>
            Madhoolika Bisht, Y. Chung, Siou-Luan He, Sydney Willey, Benjamin D. Sunkel, Meng Wang, Benjamin Z. Stanton, Li-Chun Tu
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Completion of DNA replication before chromosome segregation is essential for the stable maintenance of the genome. Under replication stress, DNA synthesis may persist beyond S phase, especially in genomic regions that are difficult to proceed with the replication processes. Incomplete replication in mitosis emerges as non-disjoined segment in mitotic chromosomes leading to anaphase bridges. The resulting chromosome rearrangements are not well characterized, however. Here, we report that incomplete replication due to SMC5/6 deficiency impairs sister chromatid disjunction at difficult-to-replicate regions, including common fragile sites. These non-disjoined regions manifest as cytologically defined symmetric gaps, causing anaphase bridges. These bridges break at the gaps, leading to telomere loss, micronucleation, and fragmentation. Subsequently, fusions between telomere-deficient chromosomes generate complex chromosomal rearrangements, including dicentric chromosomes, suggesting the occurrence of breakage-fusion-bridge cycle. Additionally, chromosomes in micronuclei were pulverized, indicative of chromothripsis. Our findings suggest that incomplete replication facilitates complex chromosomal rearrangements, which may contribute to genomic instability in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb56da5499cf4165223ba91582989de2546ca468" target='_blank'>
              Chromosomal rearrangements associated with SMC5/6 deficiency in DNA replication.
              </a>
            </td>
          <td>
            Y. Kusano, Y. Kinugasa, Satoshi Tashiro, Toru Hirota
          </td>
          <td>2024-11-14</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The 2-micron plasmid residing within the host budding yeast Saccharomyces cerevisiae nucleus serves as a model system for understanding the mechanism of segregation and stable maintenance of circular endogenously present extrachromosomal DNA in eukaryotic cells. The plasmid is maintained at a high average copy number (40-60 copies per yeast cell) through generations despite there is no apparent benefit to the host. Notably, the segregation mechanism of 2-micron plasmid shares significant similarities with those of bacterial low-copy-number plasmids and episomal forms of viral genomes in mammalian cells. These similarities include formation of a complex where the plasmid- or viral encoded proteins bind to a plasmid- or viral genome-borne locus, respectively and interaction of the complex with the host proteins. These together form a partitioning system that ensures stable symmetric inheritance of both these genomes from mother to daughter cells. Recent studies with substantial evidence showed that the 2-micron plasmid, like episomes of viruses such as Epstein-Barr virus, relies on tethering itself to the host chromosomes in a non-random fashion for equal segregation. This review delves into the probable chromosome hitchhiking mechanisms of 2-micron plasmid during its segregation, highlighting the roles of specific plasmid-encoded proteins and their interactions with host proteins and the chromosomes. Understanding these mechanisms provides broader insights into the genetic stability and inheritance of extrachromosomal genetic elements across diverse biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf1d5837f159323449ccdc148cbe5bd08f01794" target='_blank'>
              Chromosome hitchhiking: a potential strategy adopted by the selfish yeast plasmids to ensure symmetric inheritance during cell division.
              </a>
            </td>
          <td>
            Deepanshu Kumar, S. Ghosh
          </td>
          <td>2024-12-13</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal inversions (INVs) are particularly challenging to detect due to their copy-number neutral state and association with repetitive regions. Inversions represent about 1/20 of all balanced structural chromosome aberrations and can lead to disease by gene disruption or altering regulatory regions of dosage-sensitive genes in cis. Short-read genome sequencing (srGS) can only resolve ∼70% of cytogenetically visible inversions referred to clinical diagnostic laboratories, likely due to breakpoints in repetitive regions. Here, we study 12 inversions by long-read genome sequencing (lrGS) (n = 9) or srGS (n = 3) and resolve nine of them. In four cases, the inversion breakpoint region was missing from at least one of the human reference genomes (GRCh37, GRCh38, T2T-CHM13) and a reference agnostic analysis was needed. One of these cases, an INV9 mappable only in de novo assembled lrGS data using T2T-CHM13 disrupts EHMT1 consistent with a Mendelian diagnosis (Kleefstra syndrome 1; MIM#610253). Next, by pairwise comparison between T2T-CHM13, GRCh37, and GRCh38, as well as the chimpanzee and bonobo, we show that hundreds of megabases of sequence are missing from at least one human reference, highlighting that primate genomes contribute to genomic diversity. Aligning population genomic data to these regions indicated that these regions are variable between individuals. Our analysis emphasizes that T2T-CHM13 is necessary to maximize the value of lrGS for optimal inversion detection in clinical diagnostics. These results highlight the importance of leveraging diverse and comprehensive reference genomes to resolve unsolved molecular cases in rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43691c884ea796d103539d35e9071d519b6ff152" target='_blank'>
              Leveraging the T2T assembly to resolve rare and pathogenic inversions in reference genome gaps
              </a>
            </td>
          <td>
            Kristine Bilgrav Saether, J. Eisfeldt, J. Bengtsson, M. Y. Lun, Christopher M. Grochowski, M. Mahmoud, Hsiao-Tuan Chao, Jill A. Rosenfeld, Pengfei Liu, Marlene Ek, Jakob Schuy, Adam Ameur, Hongzheng Dai, J. Hwang, F. Sedlazeck, W. Bi, Ronit Marom, Josephine Wincent, Ann Nordgren, C. M. Carvalho, A. Lindstrand
          </td>
          <td>2024-11-01</td>
          <td>Genome Research</td>
          <td>1</td>
          <td>51</td>
        </tr>

        <tr id="RNase H2 is a heterotrimeric endoribonuclease that resolves RNA:DNA hybrids and genome-embedded ribonucleotides, which are implicated in DNA replication stress and cancer development. Protein and/or mRNA levels of individual RNase H2 subunits are elevated in some cancers, but little is known about the mechanisms or consequences of RNase H2 upregulation. We report that RNase H2 subunits are upregulated at the protein level in response to replication stress induced by oncogenes and chemotherapy drugs in human cancer and non-cancer cell lines. We show that inducible overexpression of the RNASEH2B subunit increases levels of the active RNase H2 heterotrimer. While causing only subtle changes to gene expression, RNASEH2B overexpression is unexpectedly associated with increased RNA:DNA hybrid levels. RNASEH2B overexpression prevents further increases in RNA:DNA hybrid levels by camptothecin or hydroxyurea and reduces replication fork stalling in presence of these drugs. Surprisingly, RNase H2 levels do not strongly impact survival of chemotherapy treatments but appear to have more subtle effects on genome instability and innate immune signalling. In contrast, increased RNase H2 levels in presence of oncogenic HRAS limit not only RAS-induced replication fork stalling but also cell death. Our findings shed new light on the functions of RNase H2 and suggest that upregulation of RNase H2 may be an important aspect of replication stress responses in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54af3146eb7acac3bc5186f9ac5d0287ea7b019a" target='_blank'>
              Human RNase H2 upregulation counteracts oncogene- and chemotherapy-induced replication stress
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, Abirami Kannan, Siobhan A. Plass, Claire Wilson, Richard D. W. Kelly, Claire H. M. Tang, P. Kotsantis, Martin A. M. Reijns, Aditi Kanhere, Eva Petermann
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2ef81377921558c83081b12d35bab26833147b" target='_blank'>
              Extrachromosomal circular DNA containing DTX1 promotes cell growth in hydroquinone-induced malignantly transformed cells by regulating the transcription of DTX1
              </a>
            </td>
          <td>
            X. Ling, Qunfang Jiao, Daifan Lin, Jialong Chen, Yali Han, Jinxue Meng, Bohuan Zhong, He Zhang, Gongda Zhang, Fangling Zhu, Jiheng Qin, Yongdui Ruan, Linhua Liu
          </td>
          <td>2024-11-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ffead5506fa79af05a88145b919b97ef5788bd8" target='_blank'>
              The genetic mechanism of B chromosome drive in rye illuminated by chromosome-scale assembly
              </a>
            </td>
          <td>
            Jianyong Chen, Jan Bartoš, A. Boudichevskaia, Anna Voigt, M. T. Rabanus-Wallace, Steven Dreissig, Zuzana Tulpová, H. Šimková, J. Macas, Gihwan Kim, Jonas Buhl, Katharina Bürstenbinder, F. Blattner, Jörg Fuchs, Thomas Schmutzer, A. Himmelbach, V. Schubert, Andreas Houben
          </td>
          <td>2024-11-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a4f47822586a1fd8d669e38036044aa6da804e" target='_blank'>
              Chromothripsis in cancer.
              </a>
            </td>
          <td>
            Milena Simovic-Lorenz, Aurélie Ernst
          </td>
          <td>2024-11-15</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy nearly half of the human genome and play diverse biological roles. Despite their abundance, the extent to which TEs contribute to three-dimensional (3D) genome structure remains unclear. To investigate this, we developed te_hic, a computational pipeline that integrates genomic data with chromatin interactions. Our analysis reveals that TE sequences are responsible for 3D genome structure in interphase nuclei. This phenomenon is mediated by the recruitment of specific transcription factors to TEs, which enables both the formation and disruption of chromatin contacts. We computationally identified known contact-forming proteins (CTCF, RAD21, SMC3) and breaking proteins (RNF2), as well as novel candidate contact-formers (SMARCA4, MAFK). Knockdown of the predicted contact-formers SMARCA4 and MAFK decreased contacts and loops at and between TEs. Notably, SMARCA4 knockdown selectively reduced short-range contacts, highlighting its role in maintaining 3D genome structure mediated by TE binding. Overall, our findings demonstrate that TEs are crucial determinants of 3D genome organization in mammalian cells. Key findings TEs determine 48% of the 3D genome structure alone, and 70% of heterotypic contacts. A/B compartments, TADs, and loop signatures can be retrieved using TE-mapped reads only. TFs can be divided into contact formers and breakers at TEs. SMARCA4 and MAFK build chromatin contacts between TE sequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d79bb0690215dabbc840c511519efb306a3da77" target='_blank'>
              Dissecting the Contribution of Transposable Elements to Interphase Chromosome Structure
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhouqi Huang, Li Sun, Xuemeng Zhou, Jiangping He, Alexander Strunnikov, Andrew P. Hutchins
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA-dependent protein kinase (DNA-PK) is a key effector of non-homologous end joining (NHEJ)-mediated double-strand break (DSB) repair. Since its identification, a substantial body of evidence has demonstrated that DNA-PK is frequently overexpressed in cancer, plays a critical role in tumor development and progression, and is associated with poor prognosis in cancer patients. Recent studies have also uncovered novel functions of DNA-PK, shifting the paradigm of the role of DNA-PK in oncogenesis and renewing interest in targeting DNA-PK for cancer therapy. To gain genetic insight into the cellular pathways requiring DNA-PK activity, we used a CRISPR/Cas9 screen to identify genes in which defects cause hypersensitivity to DNA-PK inhibitors. We identified over one hundred genes involved in DNA replication, cell cycle regulation, and RNA processing that promoted cell survival when DNA-PK kinase activity was suppressed. This gene set will be useful for characterizing novel biological processes that require DNA-PK activity and identifying predictive biomarkers of response to DNA-PK inhibition in the clinic. We also validated several genes from this set and reported previously undescribed genes that modulate the response to DNA-PK inhibitors. In particular, we found that compromising the mRNA splicing pathway led to marked hypersensitivity to DNA-PK inhibition, providing a possible rationale for the combined use of splicing inhibitors and DNA-PK inhibitors for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b81329cc49dba729d43bb24bfcaa92943d47c86" target='_blank'>
              Disruption of RNA Splicing Increases Vulnerability of Cells to DNA-PK Inhibitors
              </a>
            </td>
          <td>
            A. P. Kovina, Artem V. Luzhin, V. Tatarskiy, Dmitry A. Deriglazov, Natalia V Petrova, N. V. Petrova, L. Kondratyeva, O. Kantidze, S. V. Razin, A. K. Velichko
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosomal rearrangements on the short arm of chromosome 8 cause 8p syndrome, a rare developmental disorder characterized by neurodevelopmental delays, epilepsy, and cardiac abnormalities. While significant progress has been made in managing the symptoms of 8p syndrome and other conditions caused by large-scale chromosomal aneuploidies, no therapeutic approach has yet been demonstrated to target the underlying disease-causing chromosome. Here, we establish a two-step approach to eliminate the abnormal copy of chromosome 8 and restore euploidy in cells derived from an individual with a complex rearrangement of chromosome 8p. Transcriptomic analysis revealed 361 differentially expressed genes between the proband and the euploid revertant, highlighting genes both within and outside the 8p region that may contribute to 8p syndrome pathology. Our work demonstrates the feasibility of using chromosome engineering to correct complex aneuploidies in vitro and suggests a potential therapeutic avenue for disorders caused by chromosomal rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9c5433c28b14d1239c7306435b8438d42ccf6" target='_blank'>
              Chromosome engineering to restore euploidy in cells harboring a complex rearrangement of chromosome 8
              </a>
            </td>
          <td>
            Sophia N. Lee, Lu Qiao, Sarah L. Thompson, R. Hagenson, Teresa Davoli, J. Sheltzer
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The chromosome 5p15.33 region, which encodes telomerase reverse transcriptase (TERT), harbors multiple germline variants identified by genome-wide association studies (GWAS) as risk for some cancers but protective for others. We characterized a variable number tandem repeat within TERT intron 6 (VNTR6-1, 38-bp repeat unit) and observed a strong association between VNTR6-1 alleles (Short: 24-27 repeats, Long: 40.5-66.5 repeats) and GWAS signals within TERT intron 4. Specifically, VNTR6-1 fully explained the GWAS signals for rs2242652 and partially for rs10069690. VNTR6-1, rs10069690 and their haplotypes were associated with multi-cancer risk and age-related telomere shortening. Both variants reduce TERT expression through alternative splicing and nonsense-mediated decay: rs10069690-T increases intron 4 retention and VNTR6-1-Long expands a polymorphic G quadruplex (G4, 35-113 copies) within intron 6. Treatment with G4-stabilizing ligands decreased the fraction of the functional telomerase-encoding TERT full-length isoform, whereas CRISPR/Cas9 deletion of VNTR6-1 increased this fraction and apoptosis while reducing cell proliferation. Thus, VNTR6-1 and rs10069690 regulate the expression and splicing of TERT transcripts encoding both functional and nonfunctional telomerase. Altered TERT isoform ratios might modulate cellular longevity and replicative potential at homeostasis and in response to environmental factors, thus selectively contributing to the reduced or elevated cancer risk conferred by this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697821a17f5330940d9a6d91cd15fa1550a4da2" target='_blank'>
              Genetic regulation of TERT splicing contributes to reduced or elevated cancer risk by altering cellular longevity and replicative potential
              </a>
            </td>
          <td>
            O. Florez-Vargas, M. Ho, M. Hogshead, C.-H. Lee, B. Papenberg, K. Forsythe, K. Jones, W. Luo, K. Teshome, C. Blauwendraat, K. Billingsley, M. Kolmogorov, M. Meredith, B. Paten, R. Chari, C. Zhang, J. Schneekloth, M. Machiela, S. Chanock, S. Gadalla, S. Savage, S. Mbulaiteye, L. Prokunina-Olsson
          </td>
          <td>2024-11-05</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis.
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcd0f4cc9961826189852b167ab5b2ef51c1fec" target='_blank'>
              Integrating Circle-Seq with transcriptomics reveals genome-wide characterization of extrachromosomal circular DNA for dilated cardiomyopathy
              </a>
            </td>
          <td>
            Zhenhao Lin, Fangjie Dai, Bo Li, Yongchao Zhao, Changqian Wang
          </td>
          <td>2024-11-29</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="Aberrant meiosis in plants often leads to aneuploidy, genetic instability, and sterility. This can occur due to several factors, including chromosome misalignment, defective synapsis or environmental factors that may result in unusual genetic combinations in the offsprings. Unusual chromosome behavior during male meiosis in Humulus lupulus is linked to irregular chromosome segregation and genome instability. However, the origin of meiotic instability remains unclear. We analyzed the centromeric landscape of H. lupulus to determine its role in aberrant chromosomal segregation during cell division. Using a combination of bioinformatic, molecular and cytogenetic approaches, we identified new centromeric repeats and revealed two types of centromeric organizations. Cytogenetic localization on metaphase chromosomes confirmed the genomic distribution of major repeat arrays and revealed unique features that contribute to aberrant segregation. Two centromeric types are composed of the major repeats SaazCEN and SaazCRM1 which are further accompanied by chromosome-specific centromeric satellites, Saaz40, Saaz293, Saaz85, and HuluTR120. Chromosome 2 displays unbalanced segregation during the cell division, implicating an important role for its centromere structure in segregation patterns. Moreover, Saaz293 is a new marker for studying aneuploidy in hop. Our findings provide new insights on chromosome segregation in hop and highlight the diversity and complexity of the centromere organization in H. lupulus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc46ea72029acc6577e0f7b5a251b7322d20a30" target='_blank'>
              Centromeric repeat diversity underlies non-Mendelian segregation pattern in hop (Humulus lupulus)
              </a>
            </td>
          <td>
            Lucie Horáková, R. Čegan, Pavel Jedlička, Pavla Navrátilová, Hiroyuki Tanaka, Atsushi Toyoda, Takehiko Itoh, Takashi Akagi, Eiichiro Ono, Vojtěch Hudzieczek, Josef Patzak, Jan Šafář, R. Hobza, Václav Bačovský
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genome duplication, critical for cell survival and identity, requires precise origin activation to ensure accurate DNA replication and chromatin structure maintenance. In colorectal cancer cells, prolonged replication stress does not hinder cell proliferation, although the mechanisms driving cancer cell adaptability remain largely unclear. Here, we demonstrate that upon recovery, cancer cells are able to activate new replication origins in distinct domains, causing persistent changes in chromatin accessibility, nuclear morphology, and replication timing, ultimately promoting chromatin instability. Chromatin accessibility, particularly in promoter regions, increased following replication stress in a subset of cells, correlating with altered gene expression and nuclear expansion. Additionally, some genes with enhanced promoter accessibility displayed sustained expression changes, further suggesting a transcriptional shift linked to stress adaptation. Our findings reveal that colorectal cancer cells recover from severe replication stress through new origin activation, a mechanism that not only maintains cell proliferation under stress but may also accelerate tumour heterogeneity. This research underscores replication origin activation as a potential therapeutic target in combating cancer cell resilience. GRAPHICAL ABSTRACT MAIN HIGHLIGHTS Colorectal cancer cells under severe replication stress activate new replication origins upon recovery. The firing of new origins in distinct replication domains results in long-lasting changes in chromatin accessibility, especially in promoter regions, and is associated with alterations in nuclear morphology and replication timing. Increased chromatin accessibility in certain promoter regions correlates with prolonged gene expression changes. The capacity of CRC cells to recover from severe replication stress fosters chromatin instability and tumour heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cc1950d705dd8a02a8a5d1f21cfa69a97d0c9c" target='_blank'>
              Adaptive replication origin activation alters chromatin dynamics and stability in cancer cells
              </a>
            </td>
          <td>
            Alejandro Pérez-Venteo, Fernando Unzueta, Sonia Feu, P. Llinàs-Arias, Caroline Mauvezin, Neus Agell
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioma is an aggressive cancer of the central nervous system that is genetically characterized by aneuploidy and frequent structural variation (SV). We sought to investigate the role of telomere dysfunction (TD) in shaping the SV landscape in glioma. Normal human astrocytes (NHA) were engineered to express viral E6E7 oncoproteins which are known to spontaneously trigger TD by suppressing p53/pRb. We hypothesized that NHA-E6E7 cells gradually accumulate SVs and aneuploidy over time because of progressive TD. We therefore longitudinally tracked bulk and single-cell clones of NHA-E6E7 cells for a period of over 6 months since viral infection. Overall, NHA-E6E7 cells demonstrate a growth benefit over primary NHA cells. We obtained Illumina short-read sequencing for NHA-E6E7 cells at multiple time points. Three time points were additionally selected for both Oxford Nanopore and PacBio HiFi long-read sequencing using the single-molecule chromatin fiber sequencing (fiber-seq) protocol. Copy number variation (CNV) analysis of short-read data showed deletion events in chromosome 13 as early as population doubling level (PDL) 48. K-mer based CNV analysis using the long-read data confirmed this finding and also detected deletion events in chromosome 7 at PDL 50. From the long-read data, we found an elevated number of SVs in NHA-E6E7 cells compared to the primary NHA cells. In particular, we detected telomere junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells compared to primary cells. Next, chromatin accessibility and regulatory elements were predicted from fiber-seq data with exogenous N6-methyladenosine (m6A) stencils. We observed large-scale chromatin actuation changes in various regulatory elements across the genome that intensified in later passages of NHA-E6E7 cells. For example, we found a decrease in the accessibility in FBXO7 which is a gene that maintains chromosome stability. Our preliminary results indicated that TD is an important mediator of structural variation in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd253150a754aa7476459ba7121045b1c6e00121" target='_blank'>
              EPCO-23. LONGITUDINAL TRACKING AND LONG-READ CHROMATIN FIBER SEQUENCING OF HUMAN ASTROCYTES WITH TELOMERE DYSFUNCTION REVEALED ALTERED CHROMATIN ACCESSIBILITY AND INCREASED STRUCTURAL VARIATION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle Fukushima, Maria Kyriakidou, Michael E. Berens, Floris P Barthel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce605e6029ae60fb80c4a4396abcd104f0b8dd64" target='_blank'>
              Large DNA deletions occur during DNA repair at 20-fold lower frequency for base editors and prime editors than for Cas9 nucleases.
              </a>
            </td>
          <td>
            Gue-Ho Hwang, Seok-Hoon Lee, Minsik Oh, Segi Kim, Omer Habib, Hyeon-Ki Jang, Heon Seok Kim, Youngkuk Kim, Chan Hyuk Kim, Sun Kim, Sangsu Bae
          </td>
          <td>2024-11-04</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying rapid adaptation to stress is a fundamental question in evolutionary biology. A common mechanism is copy number changes, which can affect single genes, larger genomic regions, or even entire chromosomes in the case of eukaryotes (aneuploidy). Bacterial species with genomes comprising more than one chromosome are common, yet the role of adaptive copy number changes affecting entire chromosomes is unknown. Here, by using a bacterium with two chromosomes (chrI and chrII) as model system and combining evolution experiments in the presence of severe nutrient stress with whole-genome sequencing, we show that chromosomal copy number changes can drive rapid adaptation. We detected increased copy number of the entire chrII in almost half of the sequenced clones, isolated from twelve independent populations. In most cases, copy number increases were driven by mutations in the replication protein of chrII. A higher chrII copy number is likely to be beneficial because chrII codes for the operon responsible for metabolizing D-gluconic acid, the only carbon source present in the medium. Our results revealed a novel mechanism for rapid adaptation in bacteria. Given the prevalence of bacteria with secondary chromosomes, and the enrichment of secondary chromosomes in genes associated with interaction with the environment, this may be an important mechanism for adapting to novel environments that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e1750352edfd93ae418d5a02e63e98887bb1a2" target='_blank'>
              Chromosomal plasticity can drive rapid adaptation in bacteria
              </a>
            </td>
          <td>
            Luis Vega-Cabrera, O. Skovgaard, Macarena Toll-Riera
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In this study, artificial transformation experiments were performed to investigate how the A. castellanii genome responds to foreign DNA presented in both circular and linear plasmid form. Nanopore sequencing was used as a high throughput method to screen for transgene DNA in the resulting transformant cultures, and candidate transgene integrations were identified. Molecular biology experiments were performed to validate the sequence data and provide additional context on the fate of transgenes. A method was devised to estimate the rate of read chimerism in nanopore sequencing runs and accurately account for the effects of read chimerism in identifying putative transgene integrations. Based on the experimental data in hand, a potential mechanism for transgene maintenance in A. castellanii is proposed, one in which incoming foreign DNA is tandemly duplicated and telomeres are added to the ends. This nascent linear molecule is maintained as a transgene-bearing minichromosome, while also allowing for chromosomal integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec4ccd82afc79294f23c601721a20e74fbd23cf" target='_blank'>
              The fate of artificial transgenes in Acanthamoeba castellanii
              </a>
            </td>
          <td>
            Morgan J. Colp, Cédric Blais, Bruce A. Curtis, John M. Archibald
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The presence of nuclei defines eukaryotes, enabling compartmentalization of macromolecules and cellular regulation. Inside the nucleus, chromosome numbers vary greatly across organisms, both in terms of ploidy status and haploid chromosome number. For cells harboring multiple nuclei as in many fungal mycelia and animal muscle cells, each healthy nucleus is traditionally believed to carry at least one haploid set of chromosomes. Abnormal chromosome numbers in nuclei are often associated with aging, diseases such as cancer, developmental disorders or lethality. Here we report a surprising discovery that chromosomes in haploid cells of the fungal species Sclerotinia sclerotiorum and Botrytis cinerea are segregated into separate nuclei, with each nucleus containing only a fraction of the chromosomes. This is the first report of eukaryotic cells that partition chromosome subsets into distinct nuclei, bringing new questions and opening fresh avenues for chromosome biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0dcedf2d2e70bf41eb84b0c81857481a63a869" target='_blank'>
              Distribution of haploid chromosomes into separate nuclei in two pathogenic fungi
              </a>
            </td>
          <td>
            Yan Xu, Lei Tian, Jinyi Tan, Josh Li, Nigel O’Neil, Martin Hirst, Phil Hieter, Yuelin Zhang, Xin Li
          </td>
          <td>2024-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107c0d6e681a7d833da603e062a1adf5d877660c" target='_blank'>
              MLL oncoprotein levels influence leukemia lineage identities
              </a>
            </td>
          <td>
            D. Janssens, Melodie Duran, Dominik J. Otto, Weifang Wu, Yiling Xu, Danielle C Kirkey, C. Mullighan, Joanna S Yi, S. Meshinchi, Jay Sarthy, K. Ahmad, S. Henikoff
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair, and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd4d063ed0793e8f2d3e58d28ea55c516edbff0" target='_blank'>
              Chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, Estanislao Peixoto, Asad Khan, Yong Li, Jason S Romero Neidigk, Olivia Decker, Azait Imtiaz, Brianna A Jones, Yangfeng Zhang, Etsuko Shiabata, S. Gradilone, Zachary A. Lewis, Rafael Contreras-Galindo, Arko Sen, A. Dutta, Wioletta Czaja
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="

 In contrast to RNA expression, which captures the current state of a cell, DNA is the only biomolecule in a cell that is not constantly replaced but copied and passed on during cell divisions. As a result, mutations that occur in the DNA code are fixed and gradually accumulate forming an archive of the life history of cells. DNA from pediatric cancers usually do not have high mutational loads. However, this is not the case with pediatric lymphomas which tend to harbor many mutations. This makes pediatric lymphomas an ideal model disease to study mutation accumulation. Here, we have applied state-of-the-art single-cell whole genome sequencing on pediatric Burkitt, T-lymphoblastic and Hodgkin lymphoma using the Primary Template-directed Amplification technique. Using this method, we have compared the genomes of malignant and non-malignant cells from the same tumors. We have also included patients in our cohort which have multiple-site involvement to investigate clonal dynamics across tissues. Using in-depth mutational analyses, we have generated detailed phylogenetic trees of each patient, identified processes unique to tumorigenesis and timed the occurrence of driver mutations and mutagenic processes. This type of analysis has not been applied to lymphoma before. Outcome of lymphoma in children is excellent (>90% survival), whereas survival in adults is much lower (~50%), because children can be treated with higher doses of chemotherapy. Consequently, treatment is extremely toxic, and relapses are mostly lethal. We envision that our research will contribute to improved diagnostics and ultimately the development of more targeted, less toxic treatment and a strongly improved quality of life.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6e32c0cce75f04e37d7e5c5ae5c58506c63d3b" target='_blank'>
              Single-Cell Whole Genome Sequencing Reveals the Evolution and Migration of Lymphoma
              </a>
            </td>
          <td>
            A. Steemers, Vera M Poort, R. van Boxtel, F. Meyer-Wentrup, R. Hagelaar, N. Groenen
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Base editors create precise genomic edits by directing nucleobase deamination or removal without inducing double-stranded DNA breaks. However, a vast chemical space of other DNA modifications remains to be explored for genome editing. Here, we harness the bacterial anti-phage toxin DarT2 to append ADP-ribosyl moieties to DNA, unlocking distinct editing outcomes in bacteria versus eukaryotes. Fusing an attenuated DarT2 to a Cas9 nickase, we program site-specific ADP-ribosylation of thymines within a target DNA sequence. In tested bacteria, targeting drives efficient homologous recombination in tested bacteria, offering flexible and scar-free genome editing without base replacement nor counterselection. In tested eukaryotes including yeast, plants and human cells, targeting drives substitution of the modified thymine to adenine or a mixture of adenine and cytosine with limited insertions or deletions, offering edits inaccessible to current base editors. Altogether, our approach, called append editing, leverages the addition of a chemical moiety to DNA to expand current modalities for precision gene editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/829cd57086ac21616ffa14eb1545d71ebc09b84d" target='_blank'>
              Targeted DNA ADP-ribosylation triggers templated repair in bacteria and base mutagenesis in eukaryotes
              </a>
            </td>
          <td>
            Constantinos Patinios, Darshana Gupta, H. V. Bassett, Scott P. Collins, Charlotte Kamm, Anuja Kibe, Yanyan Wang, Chengsong Zhao, Katie Vollen, C. Toussaint, Kathryn M. Polkoff, Thuan Nguyen, Irene Calvin, Angela Migur, Ibrahim S. Al’Abri, Tatjana Achmedov, Alessandro Del Re, A. Saliba, Nathan Crook, A. Stepanova, Jose M. Alonso, Chase L. Beisel
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract In contrast to the typified view of genome cycling only between haploidy and diploidy, there is evidence from across the tree of life of genome dynamics that alter both copy number (i.e. ploidy) and chromosome complements. Here, we highlight examples of such processes, including endoreplication, aneuploidy, inheritance of extrachromosomal DNA, and chromatin extrusion. Synthesizing data on eukaryotic genome dynamics in diverse extant lineages suggests the possibility that such processes were present before the last eukaryotic common ancestor. While present in some prokaryotes, these features appear exaggerated in eukaryotes where they are regulated by eukaryote-specific innovations including the nucleus, complex cytoskeleton, and synaptonemal complex. Based on these observations, we propose a model by which genome conflict drove the transformation of genomes during eukaryogenesis: from the origin of eukaryotes (i.e. first eukaryotic common ancestor) through the evolution of last eukaryotic common ancestor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3030ffec030decedf471a2b0091572180ba9a577" target='_blank'>
              Diverse Genome Structures among Eukaryotes May Have Arisen in Response to Genetic Conflict
              </a>
            </td>
          <td>
            Elinor G. Sterner, Auden Cote-L’Heureux, Xyrus X. Maurer-Alcalá, Laura A. Katz
          </td>
          <td>2024-11-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Spatial genome organization refers to the conformation of the chromosomes and their relative positioning within the nucleus. In mammals, fertilization entails intense chromatin remodeling of parental genomes, as well as large-scale structural changes in nuclear organization of the newly formed zygote over the first two cell cleavages. The molecular characteristics, mechanisms and functionality of spatial genome organization during the early steps of development in mouse have been extensively studied and will be presented in this review, with a specific focus on heterochromatin. Concomitantly to maturation of genomic architecture, the embryonic genome activation occurs in transient waves of transcription. Here, we will also discuss the putative link between heterochromatin organization and the regulation of genome expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c60421185a6e2d944eeb443125fbc94412bbed" target='_blank'>
              Non-canonical spatial organization of heterochromatin in mouse preimplantation embryos.
              </a>
            </td>
          <td>
            Amelie Bonnet-Garnier, Katia Ancelin
          </td>
          <td>2024-11-01</td>
          <td>Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b517f5a3e621705cee3f04181fbdcd06f0c41b9" target='_blank'>
              Gradual chromosomal lagging drive programmed genome elimination in hemiclonal fishes from the genus Hypseleotris
              </a>
            </td>
          <td>
            Dmitrij Dedukh, Zuzana Majtánová, P. Ráb, Tariq Ezaz, Peter J Unmack
          </td>
          <td>2024-11-06</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Aims Mammalian genomes encode 12 proteins that contain a CXXC zinc finger domain. Most members of this family are large multi-domain proteins that function in the control of DNA methylation and histone methylation patterns. CXXC5 is a smaller member of the family, along with its closest homologue CXXC4. These two proteins lack known catalytic domains. Here, we have characterized CXXC5 in mouse embryonic stem (ES) cells. Materials & Methods We used gene knockouts, RNA sequencing, and DNA methylation analysis by whole-genome bisulfite sequencing. Results & Conclusions We show that CXXC5 is a nuclear protein that interacts with 5-methylcytosine oxidases (TET proteins). Removal of CXXC5 from ES cells leads to very few changes in gene expression. CXXC5 extensively colocalizes with TET1 and TET2 at CpG islands. CXXC5 inactivation leads to a substantial reduction of DNA methylation levels that affects all genomic compartments including genic and intergenic regions and CpG island shores. We propose a model in which CXXC5 serves as an anchor for TET proteins at CpG islands. In the absence of CXXC5, the 5-methylcytosine oxidases become dislodged from CpG islands and are enabled to induce genome-scale DNA demethylation. Thus, CXXC5 serves as a stabilizer of DNA methylation patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5906d688b460efabf6402d1ee097374746669" target='_blank'>
              CXXC5 stabilizes DNA methylation patterns in mouse embryonic stem cells
              </a>
            </td>
          <td>
            , Jennifer Johnson, Zhijun Huang, Wei Cui, Thomas Dunwell, Gerd P Pfeifer
          </td>
          <td>2024-11-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Phased telomere to telomere (T2T) genome assemblies are revolutionising our understanding of long hidden genome biology “dark matter” such as centromeres, rDNA repeats, inter-haplotype variation, and allele specific expression (ASE). Yet insights into dikaryotic fungi that separate their haploid genomes into distinct nuclei is limited. Here we explore the impact of dikaryotism on the genome biology of a long-term asexual clone of the wheat pathogenic fungus Puccinia striiformis f. sp. tritici. We use Oxford Nanopore (ONT) duplex sequencing combined with Hi-C to generate a T2T nuclear-phased assembly with >99.999% consensus accuracy. We show that this fungus has large regional centromeres enriched in LTR retrotransposons, with a single centromeric dip in methylation that suggests one kinetochore attachment site per chromosomes. The centromeres of chromosomes pairs are most often highly diverse in sequence and kinetochore attachment sites are not always positionally conserved. Each nucleus carries a unique array of rDNAs with >200 copies that harbour nucleus-specific sequence variations. The inter-haplotype diversity between the two nuclear genomes is caused by large-scale structural variations linked to transposable elements. Nanopore long-read cDNA analysis across distinct infection conditions revealed pervasive allele specific expression for nearly 20% of all heterozygous gene pairs. Genes involved in plant infection were significantly enriched in ASE genes which appears to be mediated by elevated CpG gene body methylation of the lower expressed pair. This suggests that epigenetically regulated ASE is likely a previously overlooked mechanism facilitating plant infection. Overall, our study reveals how dikaryotism uniquely shapes key eukaryotic genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe895c868a2f8ac32e5a5a400c1fdc993b92b30" target='_blank'>
              Long-read genomics reveal extensive nuclear-specific evolution and allele-specific expression in a dikaryotic fungus
              </a>
            </td>
          <td>
            Rita Tam, Mareike Möller, Runpeng Luo, Zhenyan Luo, Ashley W. Jones, Sambasivam K. Periyannan, J. Rathjen, B. Schwessinger
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA methylation (DNAm) is a key epigenetic mark that modulates regulatory elements and gene expression, playing a crucial role in mammalian development and physiological function. Despite extensive characterization of DNAm profiles across species, little is known about its evolutionary conservation. Here, we conducted a comparative epigenome-wide analysis of great apes to identify and characterize sequence- and methylation-conserved CpGs (MCCs). Using 202 DNAm arrays, alongside 6 matched genotype and 13 matched transcriptomic datasets, we identified 11,500 MCCs for which methylation was evolutionarily related to sequences of CpGs and methylation quantitative trait loci. MCCs were the most stable across healthy human tissues and exhibited weaker genetic associations than other CpGs. Moreover, MCCs showed minimal associations with demographic, environmental factors, and noncancer diseases, yet demonstrated stronger associations with certain cancers than other CpGs, particularly gastrointestinal cancers. Functional enrichment analysis revealed that genes associated with MCC methylation in cancer were enriched for cancer driver genes and canonical cancer pathways, highlighting a significant regulatory role for MCCs in tumorigenesis. Collectively, our findings reveal the extent of DNAm conservation in great ape evolution, its association with genetic conservation, and its relevance to human diseases. These integrative analyses offer evolutionary insights into epigenetic variation and its functional implications in human populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ff2d13f4df89f2515822a3ea179e1bce4a04b6" target='_blank'>
              Genetic Basis, Quantitative Nature, and Functional Relevance of Evolutionarily Conserved DNA Methylation
              </a>
            </td>
          <td>
            Zheng (Joe) Dong, Samantha Schaffner, Maggie Fu, Joanne Whitehead, Julia L. MacIsaac, David H. Rehkopf, W. T. Boyce, Luis Rosero-Bixby, L. Quintana-Murci, E. Patin, Gregory E. Miller, Keegan Korthauer, Michael S. Kobor
          </td>
          <td>2024-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5-34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks.
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The DNA damage checkpoint is crucial for maintaining genome stability after genotoxic stress; without it, excess DNA replication origins are activated, stalled DNA replication forks cannot restart normally, high levels of DNA damage and single-stranded DNA (ssDNA) accumulate and cells cannot complete S phase. Preventing excess origin firing suppresses all these effects. Here we show that when replication is not restrained by a functional checkpoint, excess DNA synthesis sequesters the processivity factor PCNA and its loader RFC, preventing normal fork restart. Nascent DNA ends unprotected by RFC/PCNA are attacked by the Helicase-Like Transcription Factor (HLTF), causing irreversible replication fork collapse and hyperaccumulation of single-stranded DNA. This explains how the checkpoint stabilizes stalled replication forks and has implications for how origin firing is normally coordinated with fork progression. Loss of HLTF suppresses fork collapse and cell lethality in checkpoint-deficient cells, which has implications for how resistance to anti-checkpoint therapies may arise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e472b56a57aa9912c2f88673648732ecc582c3db" target='_blank'>
              The mechanism of checkpoint-dependent DNA replication fork stabilization in human cells
              </a>
            </td>
          <td>
            Agostina P. Bertolin, Berta Canal, M. Yekezare, Jingkun Zeng, R. Instrell, Michael Howell, J. Diffley
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="For long, genome-wide coverage has been used as a measure of sequencing quality and quantity, but the biology hidden beneath has not been fully exploited. Here we performed comparative analyses on genome-wide coverage profiles between nucleus genome DNA (gDNA) samples from the 1000 Genomes Project (n=3,202) and cell-free DNA (cfDNA) samples from healthy controls (n=113) or cancer patients (n=362). Regardless of sample type, we observed an overall conserved landscape with coverage segmentation, where similar levels of coverage were shared among adjacent windows of genome positions. Besides GC-content, we identified protein-coding gene density and nucleosome density as major factors affecting the coverage of gDNA and cfDNA, respectively. Differential coverage of cfDNA vs gDNA was found in immune-receptor loci, intergenic regions and non-coding genes, reflecting distinct genome activities in different cell types. A further rise in coverage at non-coding genes/intergenic regions and a further drop of coverage at protein-coding genes/genic regions within cancer cfDNA samples suggested a relative loss of contribution by normal cells. Importantly, we observed the distinctive convergence of coverage in cancer-derived cfDNA, with the extent of convergence positively correlated to stages. Based on the findings we developed and validated an outlier-detection approach for cfDNA-based cancer screening without the need of cancer samples for training. The method achieved 97% sensitivity on pediatric sarcomas (n=241) and 44% sensitivity on early-stage lung cancers (n=36) with >90% specificity for condition-matched tasks, 100% sensitivity on late-stage cancers (n=85) for condition-unmatched tasks, outperforming current benchmarks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58763bc618f0e854c5267409a63aba70151b9e8e" target='_blank'>
              Coverage landscape of the human genome in nucleus DNA and cell-free DNA
              </a>
            </td>
          <td>
            Jiaqi Luo, Shuaicheng Li
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the most common cancer in women and the 2nd most common cancer worldwide, yearly impacting over 2 million females and causing 650 thousand deaths. It has been widely studied, but its epigenetic variation is not entirely unveiled. We aimed to identify epigenetic mechanisms impacting the expression of breast cancer related genes to detect new potential biomarkers and therapeutic targets. We considered The Cancer Genome Atlas database with over 800 samples and several omics datasets such as mRNA, miRNA, DNA methylation, which we used to select 2701 features that were statistically significant to differ between cancer and control samples using the Monte Carlo Feature Selection and Interdependency Discovery algorithm, from an initial total of 417,486. Their biological impact on cancerogenesis was confirmed using: statistical analysis, natural language processing, linear and machine learning models as well as: transcription factors identification, drugs and 3D chromatin structure analyses. Classification of cancer vs control samples on the selected features returned high classification weighted Accuracy from 0.91 to 0.98 depending on feature-type: mRNA, miRNA, DNA methylation, and classification algorithm. In general, cancer samples showed lower expression of differentially expressed genes and increased β-values of differentially methylated sites. We identified mRNAs whose expression is well explained by miRNA expression and differentially methylated sites β-values. We recognized differentially methylated sites possibly affecting NRF1 and MXI1 transcription factors binding, causing a disturbance in NKAPL and PITX1 expression, respectively. Our 3D models showed more loosely packed chromatin in cancer. This study successfully points out numerous possible regulatory dependencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471e2fdeae8d795259c36bd54b5931925e83e88a" target='_blank'>
              Unveiling epigenetic regulatory elements associated with breast cancer development
              </a>
            </td>
          <td>
            Marta Jardanowska-Kotuniak, Michał Dramiński, Michał Wlasnowolski, Marcin Łapiński, Kaustav Sengupta, Abhishek Agarwal, Adam Filip, Nimisha Ghosh, Vera Pancaldi, Marcin Grynberg, I. Saha, Dariusz Plewczyński, Michal J Dabrowski
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 The stepwise progression of colorectal cancer (CRC) from healthy epithelium, to premalignant adenoma, to cancer is accompanied by genome-wide epigenetic reprogramming which may be reflected in circulating cell-free DNA (cfDNA). However, current analyses of these processes have been hampered by complex mixtures of cells in normal, adenoma and cancer tissue samples, as well as absence of suitable adenoma model systems. Cultured colorectal organoids, comprised of pure epithelial cells, could enable insights on the epigenetic changes leading to progression from healthy tissue to cancer. Such features could be instrumental in the early cancer detection using genome-wide analyses of cfDNA fragmentation. Using 43 patient-derived colorectal organoids derived from healthy, adenoma, and cancer tissues, as well as 10 freshly collected peripheral blood mononuclear cell (PBMC) control samples, we examined the chromatin landscape of tumor progression using transposase accessible chromatin analyses with next-generation sequencing. We defined a consensus set of nucleosome depleted regions for each disease state and identified loci with differential accessibility between PBMCs, colon healthy, colon adenoma, and colon cancer tissues. We analyzed these differential loci for fragmentation characteristics in the circulating cfDNA of 250 healthy individuals and 51 patients with metastatic CRC. Across all organoid samples, we identified >35 million nucleosome depleted regions that we coalesced into a consensus set of 62,708 regions. As 78% of these regions could be linked to genes, we assessed whether these signatures identify known pathways involved in colorectal cancer. Within these regions, we identified unique colon tumor-related signatures comprising genes of known and novel pathways, including those involved in WNT, hippo, and RAS signaling. While the majority of chromatin accessibility differences were shared between adenomas and cancers, we identified 895 chromatin changes in adenomas that were not present in either healthy or malignant tissues. In healthy individuals, we found that the cfDNA coverage was negatively correlated with accessibility of white blood cell accessibility loci (Spearman ρ= - 0.60) while in individuals with cancer, changes in chromatin accessibility were associated with independent circulating tumor DNA abundance by droplet digital PCR (R2=0.69, p<0.001). Chromatin accessibility profiles were correlated with genome-wide cell-free DNA fragmentation features and could be used to distinguish healthy individuals from those with colorectal cancer (AUC=0.97). These analyses are consistent with dynamic chromatin remodeling during colorectal tumor development and may provide an avenue for evaluating these through genome-wide analyses of cfDNA fragmentation.
 Citation Format: Nicholas A. Vulpescu, Zachariah H. Foda, Pieter H.A. Wisse, Christopher Cherry, Jaime E. Medina, Vilmos Adleff, Remond J.A. Fijneman, Robert B. Scharpf, Gerrit A. Meijer, Beatriz Carvalho, Victor E. Velculescu. Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37db1a1abcfb5adfff7334fbd173d8e5404d5cef" target='_blank'>
              Abstract PR016: Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, Jaime E. Medina, V. Adleff, R. Fijneman, Robert B Scharpf, Gerrit A Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Transposable elements (TEs) provide sequences that are powerful cis-regulatory drivers of gene expression programmes. This is particularly apparent during early development when many TEs become de-repressed. MERVL elements are highly yet transiently upregulated in mouse totipotent 2-cell (2C) embryos during major zygotic genome activation (ZGA), and in 2C-like cells in vitro. One of the most powerful activators of MERVL is the pioneer transcription factor, Dux. However, apparent differences lie in the requirement for Dux versus MERVL activation in embryos, for unclear reasons. Moreover, sustained Dux activation causes cell toxicity in multiple cell types, which may or may not be linked to MERVL activation. Using a CRISPR-activation, 2C-GFP reporter system, we have unpicked the relative role of Dux and MERVL in ZGA, totipotent-like characteristics and cell toxicity. We find that direct MERVL activation comprises only a portion of the Dux-dependent transcriptome, and which is sufficient for expanded fate potential, but not other totipotency features. Conversely, Dux-induced pathology is independent of MERVL activation and involves induction of the pro-apoptotic factor, Noxa. Our study highlights the complexity of the Dux-MERVL transcriptional network and uncovers a new player in Dux-driven pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1097bcc0ee4068a4a806cd6193383ebb7f0df5ed" target='_blank'>
              CRISPRa-mediated disentanglement of the Dux-MERVL axis in the 2C-like state, totipotency and cell death
              </a>
            </td>
          <td>
            Paul Chammas, Sheila Q Xie, L. P. Sepulveda-Rincon, Bryony J. Leeke, Marian H Dore, Dirk Dormann, Ryan T Wagner, Michael T McManus, Mohammad M Karimi, George Young, Michelle Percharde
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The mitochondrial genome, which encodes genes essential for respiration and cellular homeostasis, is the target of abundant and highly diverse somatic alterations in cancers. Somatic alterations to mitochondrial DNA (mtDNA) nearly always arise heteroplasmically, producing heterogeneous ensembles of mtDNA within single cells. Here, we review new insights derived from exponential increases in genomic sequencing data that have uncovered the nature of, selective pressure for, and functional consequences of cancer-associated mtDNA alterations. As many discoveries have been limited by their ability to determine cell-to-cell variation in mtDNA genotype, we describe a new generation of single-cell sequencing approaches that resolve otherwise indeterminate models of mtDNA heteroplasmy. In tandem with novel approaches for mtDNA editing and modeling of mutations, these advances foreshadow the quantitative dissection of dosage-dependent mtDNA phenotypes that underlie both tumor evolution and heterogeneous response to therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823dce23cbbddef91125605bfb3305bbb4cdca3d" target='_blank'>
              Single-Cell Technologies for Studying the Evolution and Function of Mitochondrial DNA Heteroplasmy in Cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Erin M. Cumming, Minsoo Kim, Caleb Lareau, Ed Reznik
          </td>
          <td>2024-11-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) refers to a class of circular, non-chromosomal DNA that has recently gained widespread attention due to its potential role in aging and neurodegenerative diseases. The generation of ecDNA is closely associated with processes such as double-strand breaks, micronuclei formation, and the breakage-fusion-bridge (BFB) cycle, all of which are integral to regulation of gene expression, genetic stability, and clonal evolution. In neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, and Huntington's disease, the aberrant formation of ecDNA is closely linked to defects in DNA repair, alterations in synaptic plasticity, and neuronal dysfunction. The distinct distribution and functional roles of ecDNA in these conditions make it a potential diagnostic biomarker and therapeutic target. This review provides an overview of the mechanisms underlying ecDNA formation and its functions in the nervous system. Additionally, it explores the clinical potential of ecDNA in disease diagnosis, targeted therapy, and personalized medicine, offering new insights for future research and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d58ac80378b185b8a630604a71f2ebc6449a88f9" target='_blank'>
              Extrachromosomal DNA: Molecular perspectives in aging and neurodegenerative diseases.
              </a>
            </td>
          <td>
            Yafeng Ma, Ying Xia, Kenji Karako, Peipei Song, Xiqi Hu
          </td>
          <td>2024-11-30</td>
          <td>Intractable & rare diseases research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Philadelphia chromosome (Ph) a hallmark of chronic myeloid leukemia (CML), is a result of reciprocal translocation t(9;22) generating aberrant persistently activate oncogenic tyrosine- kinase (BCR/ABL1) facilitating leukemogenesis. It was demonstrated that simultaneous breakages of two different chromosomes within BCR and ABL1 loci, serve as an oncogenic initiation of Ph+ leukemia. Thus, recurrency of Ph and other translocations in hematopoietic stem cells indicates that chromosomal instability (CIN) is leading pathogenetics of leukemogenesis.
 Surprisingly, despite established diagnostic methods and knowledge on CML, use of Optical Genome Mapping (OGM) identified novel pathogenetic landscape of chromosomal aberrations and generation of BCR/ABL1 p190 fusion gene without formation of Ph chromosome. We found that dominant characteristics of CIN in these leukemia cells is increase of clustered aberrations at distinct breakpoints throughout a genome with proximity to microhomology-enriched segments. We detected significant accumulation of double-strand breaks (DSBs) at near-overlapping origins of replications. OGM and chromosomal analyses strongly correlate with induced distinct expression profile and activation of microhomology-mediated break induce repair (MMBIR). MMBIR utilizes microhomology for DSB repair, producing templated insertions that promote complex genomic rearrangements detected in cancer including chromosomal illegitimate recombination, integrations and fusions. Despite growing evidence on accumulation of de novo templated insertions in several cancer types, the underlying molecular and genomic mechanisms, impact on carcinogenesis and heterogeneity, particularly in hematopoietic cells are poorly understood.
 Based on OGM, omics data mining and NGS, we developed sophisticated screening panels and molecular analyses to study BCR, ABL1, corresponding breakpoints loci, and near-overlapping microhomology-enriched elements, to identify and characterize this novel atypical p190 fusion gene. Our results indicate that in some CML patients', with unique CIN signature, induced mechanism of de novo templated insertions generating atypical p190 fusion gene instead of Ph chromosome formation.
 It's well established that different BCR/ABL1 transcripts correspond to distinct CML clinical phenotype. Furthermore, chronic, accelerated and blast phases of CML progression evaluated and measured by expression rate of each Ph+ BCR/ABL1 transcript variant, resulting in prediction of therapy response, and clinical outcomes.
 However, in few CML patients, standard assessments succeeded to detect fusion gene induced by templated insertion and monitor atypical transcripts. Thus, generally, standard tests were insufficient to assess expression rate of atypical p190 transcript initially and in MRD. Importantly, precise definition of the disease is challenging in patients with minimal CML. For example, preliminary screen detected templated insertions with atypical p190 BCR/ABL1 fusion genes in 6 patients. 3 patients were JAK2 V617F positive, while 2 subsequently treated for Essential Thrombocytosis. Third patient diagnosed and treated for p210 Ph+CML. In contrast, spontaneous recovery observed in other 3 patients, with allelic frequency of atypical p190 transcript lower than 0.035%. Thus, detection of minor clones with templated insertions, characterization of related abnormalities provide critical insights during early detection on clinical significance of atypical p190 transcript, necessity of further molecular testing and treatment evaluation. Hence, we developed robust molecular assay to measure atypical p190 transcripts to assist diagnosis and monitoring MRD. In contrast, ongoing study points on importance of MRD monitoring, detection CIN signatures, accumulation of templated insertions promoting transformation.
 In conclusion, our study indicates that distinct CIN signature orchestrates unique DNA repair landscape. Lack of precise molecular assessment, and genetic identification of seemingly similar oncogenes originated from atypical fusion gene, or Ph chromosome can be providing misleading diagnostics. Furthermore, our results suggest that increased heterogeneity observed among CML patients, reflect on clonal ability to repair DNA damage, preserve genome stability and function, affecting carcinogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8037e7571b5a68dcda78fbeaec9945cd834494" target='_blank'>
              Templated Insertion Mediates Generation of p190 BCR/ABL Facilitating CML Heterogeneity Uniquely Detected By Optical Genome Mapping
              </a>
            </td>
          <td>
            C. Ganzel, Y. Ofran, Bayan Alyan, Elina D Shnayder, Shira Vanunu, Zakaria M Sous, Victoria Vinokurov, Veronika Butin Israeli
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235fec0818688f0c80daa0063ad7c555bc1856b" target='_blank'>
              Hypoxia-dependent recruitment of error-prone DNA polymerases to genome replication.
              </a>
            </td>
          <td>
            Ran Yehuda, Ido Dromi, Y. Levin, Thomas Carell, Nicholas Geacintov, Z. Livneh
          </td>
          <td>2024-10-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Eukaryotic nuclei adopt a highly compartmentalized architecture that influences nearly all genomic processes. Understanding how this architecture impacts gene expression has been hindered by a lack of tools for elucidating the molecular interactions at individual genomic loci. Here, we adapt oligonucleotide-mediated proximity-interactome mapping (O-MAP) to biochemically characterize discrete, micron-scale nuclear neighborhoods. By targeting O-MAP to introns within the TTN pre-mRNA, we systematically map the chromatin loci, RNAs, and proteins within a muscle-specific RNA factory organized around the TTN locus. This reveals an unanticipated compartmental architecture that organizes cis- and trans-interacting chromosomal domains, including a hub of transcriptionally silenced chromatin. The factory also recruits dozens of unique RNA-binding and chromatin-scaffolding factors, including QKI and SAFB, along with their target transcripts. Loss of the cardiac-specific splicing factor RBM20—a master regulator of TTN splicing that is mutated in dilated cardiomyopathy—remodels nearly every facet of this architecture. This establishes O-MAP as a pioneering method for probing single-locus, microcompartment-level interactions that are opaque to conventional tools. Our findings suggest new mechanisms by which coding genes can "moonlight" in nuclear-architectural roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627af105d83b03abb917d1910b63bbb83988b3fc" target='_blank'>
              Nascent transcript O-MAP reveals the molecular architecture of a single-locus subnuclear compartment built by RBM20 and the TTN RNA
              </a>
            </td>
          <td>
            Evan E. Kania, Aidan Fenix, Daphnée M Marciniak, Qiaoyi Lin, Sara Bianchi, Borislav Hristov, Shuai Li, Conor K. Camplisson, Rose Fields, Brian J. Beliveau, Devin K. Schweppe, W. S. Noble, Shao-En Ong, Alessandro Bertero, Charles E. Murry, D. Shechner
          </td>
          <td>2024-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Chronically persistent viruses are integral components of the organismal ecosystem in humans and animals 1 2. Many of these viruses replicate and accumulate within the cell nucleus 3. The nuclear location allows viruses to evade cytoplasmic host viral sensors and promotes viral replication 4. One of the unexplored and puzzling aspects of the viral nuclear lifecycle involves the virus’s ability to deal with the physical constraints of nuclear architecture. To replicate and accumulate within the nucleus in large numbers sufficient for infection spreading, DNA viruses need to overcome the spatial limitations imposed by chromatin and the nuclear matrix. We found that one of the most widespread and potentially lethal human viruses, the JC polyomavirus 5, interferes with nuclear heterochromatin to create virus-occupied space. The JC virus’s impact on heterochromatin is mediated by the viral nonstructural protein, Agnoprotein (Agno). Agno’s interference with heterochromatin is governed by structurally diverse mimics of host epigenetic regulators that facilitate virus-induced chromatin reorganization and a dramatic decline in nuclear stiffness in the infected cells. The JCV epigenetic mimicry is critical for the virus infection, as evident from reduced replication of mimic-mutant viruses. Our data suggest that modulation of nuclear mechanical properties is a novel strategy enabling chronicity of the JC and possibly other nuclear virus infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab25f0e5f1471f31142457449e1e4104ce760805" target='_blank'>
              Chromatin mimicry by human JC virus
              </a>
            </td>
          <td>
            Uwe Schaefer, Yekaterina A. Miroshnikova, Wei Xie, Adam G. Larson, Ziyu Lu, Shoudeng Chen, Martina Bradic, Y. Goldgur, Kexin Chen, Ved P. Sharma, Junyue Cao, Dinshaw J. Patel, G. Narlikar, Sara A. Wickström, Alexander Tarakhovsky
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Copy number variation (CNV) is a class of genomic Structural Variation (SV) that underlie genomic disorders and can have profound implications for health. Short-read genome sequencing (sr-GS) enables CNV calling for genomic intervals of variable size and across multiple phenotypes. However, unresolved challenges include an overwhelming number of false-positive calls due to systematic biases from non-uniform read coverage and collapsed calls resulting from the abundance of paralogous segments and repetitive elements in the human genome. Methods To address these interpretative challenges, we developed VizCNV. The VizCNV computational tool for inspecting CNV calls uses various data signal sources from sr-GS data, including read depth, phased B-allele frequency, as well as benchmarking signals from other SV calling methods. The interactive features and view modes are adept for analyzing both chromosomal abnormalities [e.g., aneuploidy, segmental aneusomy, and chromosome translocations], gene exonic CNV and non-coding gene regulatory regions. In addition, VizCNV includes a built-in filter schema for trio genomes, prioritizing the detection of impactful germline CNVs, such as de novo CNVs. Upon computational optimization by fine-tuning parameters to maximize sensitivity and specificity, VizCNV demonstrated approximately 83.8% recall and 77.2% precision on the 1000 Genome Project data with an average coverage read depth of 30x. Results We applied VizCNV to 39 families with primary immunodeficiency disease without a molecular diagnosis. With implemented build-in filter, we identified two de novo CNVs and 90 inherited CNVs >10 kb per trio. Genotype-phenotype analyses revealed that a compound heterozygous combination of a paternal 12.8 kb deletion of exon 5 and a maternal missense variant allele of DOCK8 are likely the molecular cause of one proband. Conclusions VizCNV provides a robust platform for genome-wide relevant CNV discovery and visualization of such CNV using sr-GS data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e76ef14898dd5bfe072cdf32b13a850ad55424cf" target='_blank'>
              VizCNV: An integrated platform for concurrent phased BAF and CNV analysis with trio genome sequencing data
              </a>
            </td>
          <td>
            Haowei Du, M. Y. Lun, Lidiia Gagarina, Michele G. Mehaffey, J. Hwang, S. Jhangiani, Sravya V. Bhamidipati, D. Muzny, Cecilia Poli, Sebastian Ochoa, I. K. Chinn, Anna Linstrand, J. Posey, Richard A Gibbs, J. Lupski, C. M. Carvalho
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="

 Introduction
 The pathogenic role of somatic non-coding (nc) single nucleotide variants (SNVs) is underexplored in multiple myeloma (MM) and additional characterization of these regions has the potential to provide significant new insights into disease progression. To investigate the role of ncSNVs and their relationship with MM pathogenesis, we have characterized a novel series of WGS from 294 cases of MGUS, SMM, and MM using computational tools optimized to identify functional driver SNVs in these regions. We hypothesize the ncSNVs that co-localize to sites of super-enhancers (SEs) active in plasma cells (PC) are sites of high functional impact SNVs and these mutations lead to the dysregulated expression of genes that should be silent at the terminally differentiated stage of PCs, providing a selective advantage that shapes the fitness of the MM genomic landscape.
 Methods
 We analyzed 294 high-coverage matched tumor/normal pairs (80x tumor and 40x normal) derived from 21 MGUS, 86 SMM, 167 NDMM and 40 RRMM. In addition to previously sequenced samples, newly sequenced patients were obtained from NYU Langone Health, Memorial Sloan Kettering Cancer Center, University of Alabama at Birmingham and University of Miami following IRB-approved informed consent. DNA was extracted from CD138+ purified bone marrow plasma cells using standard approaches. We used the MGP1000 (https://github.com/pblaney/mgp1000) to process sequencing data which employs a union-consensus mechanism for determining the final set of variants. We collected enhancer elements (E/SEs) active in GC B-cells, ABC-DLBCL and GBC-DLBCL (Bal, et al. Nature, 2022) and PC specific E/SEs (Loven, et al. Cell 2013). FunSeq2 was used to score ncSNVs. GRAM was used to predict how variants alter TF binding and gene expression in a subset of 6 cases with paired WGS in our series and will expand this analysis on a validation set of 263 NDMM and 93 RRMM cases from University-Hospital Heidelberg enrolled in the GMMG-HD6 clinical trial.
 Results
 We identified 1,952,232 high-quality SNVs with 18,199 (0.93%) accumulating in 280 focal sites we termed “mutationally enriched regions” or MERs. To discern SNVs in MERs with the highest potential for imparting a selective advantage, we clustered mutations based on the density of FunSeq2 impact scores which uses reference data contexts on sequence conservation, modification of transcription factor (TF) binding, and gene linkage to regulatory elements and network hubs from pan-cancer studies. We show that MERs are enriched for ncSNVs with high impact scores (mean=2.57) compared to random similarly-sized regions of the genome (Fisher's exact test P<9.7e-12, OR=10.10 [3.88-37.40]). The high impact ncSNVs (hi-ncSNVs) in MERs (n=220) account for 8.09% of all hi-ncSNVs and are present in 46.3% of cases (136/294) with some patients having up to 7. The proportion of cases carrying a MER hi-ncSNVs was similar across disease stages (MGUS=42.9%, SMM=46.5%, MM=40.7%, and RR=47.5%). Analysis of the VAF of MER hi-ncSNVs shows a minor shift towards more clonal versus subclonal mutations (53.6% vs 46.4% VAF>=0.25) suggesting some high impact mutations are acquired in the GC but additional hi-ncSNV are acquired later in PC development, providing evidence that hi-ncSNVs accumulate in MERs throughout disease progression. We mapped these hi-ncSNVs to associated genes then intersected the loci with E/SEs active in GCBs and PCs to determine their functional context. This approach identified 12 genes affected by hi-ncSNVs including the previously identified nc-drivers NEAT1, MIR142, ZCCHC7, and IGLL5 as well as the PC differentiation genes IRF8, BACH2, proliferation genes RHOH, SPRED2, known coding MM drivers CCND1, ZFP36L1, and chr1q genes BTG2 and CROCC. We predicted the effect of ncSNVs on expression through modulation of TF binding at this set of putative nc drivers in MM and identified a trend towards increase in expression of SPRED2, CCND1, NEAT1, and IGLL5 compared to non-mutated cases (P=0.089). We will present data on the validation of this set of genes on a larger set of data derived from the Heidelberg group.
 Conclusions
 We identified a set of 12 putative nc driver genes in MM that modify expression and have the potential to significantly alter the pathogenesis of MM by providing a selective survival advantage. We will present the impact of these by clinical stage and on outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d27e02bc3368053980f07501df92720db4c496" target='_blank'>
              Mutationally Enriched Regions of the Non-Coding Genome Harbor Functionally Impactful Mutations That Dysregulate Key Genes Involved in Multiple Myeloma Pathogenesis
              </a>
            </td>
          <td>
            P. Blaney, K. Maclachlan, M. Papadimitriou, A. Mikulasova, Dylan Gagler, Marc J Braunstein, O. Lahoud, David Kaminetzky, Yubao Wang, Sanxiong Liu, Di Zhang, Anthony Cirrincione, A. Poos, B. Ziccheddu, Marcella Kaddoura, M. Chojnacka, B. Diamond, N. Korde, A. Lesokhin, Saad Z Usmani, L. Rasche, E. Mai, M. Raab, Niels Weinhold, Elizabeth E. Brown, N. Robine, L. Bergsagel, Brian A Walker, O. Landgren, F. Maura, F. E. Davies, G. Morgan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Missing heritability in hereditary diffuse gastric cancer (HDGC) ranges from 60 to 90%. These HDGC-like families, despite complying with HDGC clinical criteria, lack CDH1 and CTNNA1 actionable germline variants, and are not offered HDGC-targeted life-saving disease prevention measures. Herein, we explored novel HDGC predisposition mechanisms affecting the CDH1-regulatory network. We called single-nucleotide (SNV) and copy-number variants (CNV) from 19 HDGC-like probands from whole-genome sequencing data and performed gene-ontology analysis. Chromatin enhancer marks and CDH1 promoter interactions were evaluated in normal stomach by ChIP-seq, ATAC-seq and 4C-seq, variant causality was assessed by RT-PCR, immunohistochemistry and microsatellite instability (MSI) analysis in tumours. Functional analysis was performed using CRISPR-Cas9, RT-PCR and flow cytometry in cell lines, and enhancer assays using mouse embryos. Within the CDH1 topologically associating domain (TAD), we found two deletions in Family F4 and F9. F4 carried a heterozygous CDH3 20kb-CNV triggering CDH1 mRNA/protein loss in homozygosity by CRISPR-Cas9 editing, similarly to a CDH1 coding deletion. This 20kb sequence encloses two hypomorphic tissue-specific regulatory elements (REs), each contributing 50% to CDH1 expression regulation. F9 carried a heterozygous 39bp-intergenic CNV downstream of CDH1, triggering CDH1 mRNA/protein loss by CRISPR-Cas9. F15, presenting gastric but not colorectal cancer, carried an MLH1 heterozygous 2.7Kb germline CNV overlapping a stomach-specific RE found by ChIP-seq. The gastric tumour of mixed histology displayed Microsatellite instability (MSI), reduced MLH1 mRNA and protein, and reduced CDH1 and E-cadherin protein. CRISPR-Cas9 clones mimicking the MLH1 heterozygous CNV, triggered loss of MLH1 and CDH1/E-cadherin mRNA and protein, similar to a coding deletion. Beyond the CDH1 TAD and tumour risk syndrome genes, multiple deletions of stomach accessible chromatin sequences were found in particularly young-affected individuals from additional 6 families. This oligogenic pattern impaired specifically mucin genes and multiple immune-related pathways. Herein, we pinpointed novel mechanisms behind HDGC predisposition. One involves deletions of CDH1-REs in the TAD or stomach-specific CDH1-REs in the MLH1 locus. The second involves multiple deletions of stomach REs affecting mucin and immune-related genes, favouring a gastric immune-deficient phenotype. Altogether, by combining stomach-specific chromatin accessibility and promoter interactions with whole genome sequencing, we solved the missing heritability in 47% of HDGC-like families within our cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66d13fec63d70d2f045432b7e54527ac3dd2944a" target='_blank'>
              Combined analysis of chromatin accessibility, promoter interactions and whole genome sequencing solved the missing heritability in gastric cancer
              </a>
            </td>
          <td>
            C. São José, M. Ferreira, L. Cordova, A. Pedro, J. Senz, J. Garcia-Pelaez, S. Lobo, I. Gullo, A. Andre, P. Kaurah, F. Puntieri, J. Glaser, D. Huntsman, S. Mundlos, K. A. Schrader, C. Oliveira
          </td>
          <td>2024-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The 22q11.2 deletion syndrome (22q11.2DS) is the most common microdeletion disorder. Why the incidence of 22q11.2DS is much greater than that of other genomic disorders remains unknown. Short read sequencing cannot resolve the complex segmental duplicon structure to provide direct confirmation of the hypothesis that the rearrangements are caused by nonallelic homologous recombination between the low copy repeats on Chromosome 22 (LCR22s). To enable haplotype-specific assembly and rearrangement mapping in LCR22 regions, we combined fiber-FISH optical mapping with whole genome (ultra-)long read sequencing or rearrangement-specific long-range PCR on 24 duos (22q11.2DS patient and parent-of-origin) comprising several different LCR22-mediated rearrangements. Unexpectedly, we demonstrate that not only different paralogous segmental duplicon but also palindromic AT-rich repeats (PATRR) are driving 22q11.2 rearrangements. In addition, we show the existence of two different inversion polymorphisms preceding rearrangement, and somatic mosaicism. The existence of different recombination sites and mechanisms in paralogues and PATRRs which are copy number expanding in the human population are a likely explanation for the high 22q11.2DS incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769f0987422b68baa29d9145bb1d8e87d9879537" target='_blank'>
              Multiple paralogues and recombination mechanisms contribute to the high incidence of 22q11.2 Deletion Syndrome.
              </a>
            </td>
          <td>
            Lisanne Vervoort, Nicolas Dierckxsens, Marta Sousa Santos, Senne Meynants, E. Souche, Ruben Cools, T. Heung, Koen Devriendt, Hilde Peeters, Donna M. McDonald-McGinn, A. Swillen, J. Breckpot, Beverly S Emanuel, H. Van Esch, Anne S. Bassett, J. Vermeesch
          </td>
          <td>2024-11-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="JC polyomavirus (JCPyV) is a ubiquitous pathogen causing progressive multifocal leukoencephalopathy (PML) in immunocompromised individuals. Polyomaviruses (PyVs) utilize complex transcriptional regulation mechanisms to express diverse mRNAs from their compact, double-stranded circular DNA genomes. A recent study employed next-generation sequencing (NGS) to provide a detailed transcriptome atlas of PyVs, including BK polyomavirus and simian virus 40. However, detailed information regarding the JCPyV transcriptome remains limited. Here, we conducted a comprehensive analysis combining short-read and long-read NGS methods to unveil the detailed transcriptome atlas of JCPyV. RNA isolated from IMR-32 and 293 cells transfected with the circular JCPyV genome was analyzed by NGS, leading to the discovery of 39 novel viral transcripts in addition to 12 previously annotated ones. Among these novel transcripts, we identified characteristic wraparound transcripts, conserved across PyVs, which are constructed from long primary transcripts generated through continuous, multiple rounds of circular transcription of the viral genome. These wraparound transcripts included both late transcripts harboring leader-to-leader repeated sequences and SuperT transcripts containing multiple LxCxE motifs. Notably, wraparound transcripts, including a SuperT transcript, were also detected in brain tissues from PML patients. Collectively, this study significantly expands our understanding of the JCPyV transcriptome, revealing the expression of wraparound transcripts in PML lesions. These findings provide valuable insights into the molecular basis of JCPyV gene expression and PML pathogenesis, potentially facilitating the development of effective countermeasures against PML. Author Summary JC polyomavirus (JCPyV) was first identified in 1971 from the brain tissue of a patient with progressive multifocal leukoencephalopathy (PML), a life-threatening neurological disease. Despite over 50 years since the pathogen’s identification, the prognosis for PML remains extremely poor due to the lack of effective therapeutics. Elucidating the JCPyV transcriptome can expand our understanding of viral gene expression and PML pathogenesis, potentially providing targets for novel therapeutic strategies. Here, we employed a combination of three next-generation sequencing technologies to explore novel viral transcripts, creating a detailed transcriptome atlas of JCPyV. We identified 39 novel viral transcripts in addition to the 12 previously annotated ones in cultured cells transfected with the JCPyV genome. These novel transcripts included wraparound transcripts, which arise from continuous, multiple rounds of circular transcription of the viral genome. Wraparound transcripts were characterized by the presence of tandem repeats of approximately 100 nucleotides. Notably, these characteristic wraparound transcripts were also detected in brain tissues from PML patients. Our study provides new insights into JCPyV gene expression, potentially contributing to the development of effective therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84cfe1dc7784f3b6c01808180027476dfefc4c21" target='_blank'>
              Identification of novel wraparound transcripts in JC polyomavirus
              </a>
            </td>
          <td>
            S. Iida, Kenta Takahashi, S. Mine, Tadaki Suzuki, H. Katano
          </td>
          <td>2024-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfeae92a124fdc8f4d3b4b5cb477f237233cfc8" target='_blank'>
              Nuclear porcupine mediates XRCC6/Ku70 S-palmitoylation in the DNA damage response
              </a>
            </td>
          <td>
            Yang Chen, Mingming Xiao, Yaqi Mo, Jinlu Ma, Yamei Han, Qing Li, Qinghua Zeng, R. Boohaker, Joshua S. Fried, Yonghe Li, Han Wang, Bo Xu
          </td>
          <td>2024-11-04</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Nearly half of multiple myeloma (MM) patients have hyperdiploidy (HMM) at diagnosis. Although HMM occurs early, the mutational processes before and after hyperdiploidy are still unclear. Here, we used 72 WGS samples from patients with HMM and identified pre and post HMM mutation to define the chronology of development of hyperdipoidy. A MM cell accumulated on median 0.56 mutations per mb pre-HMM and for every clonal pre-HMM mutation, 1.21 mutations accumulated post-HMM. This analysis using mutations before and after hyperdiploidy show that hyperdiploidy happens after somatic hypermutation, pre-hyperdipoidy muations are AID and age/Clock-like signature driven whereas post-hyperdiploidy mutations are from DNA damage and APOBEC. Interestingly, the first hyperdiploidy event occured within the first 3 decades of life and took a decade to complete. Copy number changes affecting chromosomes 15 and 19 occurred first. Finally, mutations pre initiating event affected chromosomes at different rates while post-initiating event mutational processes affect each chromosomes equally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9c338168c54e27d44b4bae2aaa73a50b779e7" target='_blank'>
              Development of hyperdiploidy starts at an early age and takes a decade to complete.
              </a>
            </td>
          <td>
            M. Samur, Anil Aktas Samur, Parth Shah, Joseph Park, M. Fulciniti, M. Shammas, J. Corre, Kenneth C. Anderson, G. Parmigiani, H. Avet-Loiseau, Nikhil C Munshi
          </td>
          <td>2024-11-21</td>
          <td>Blood</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fecaeaf832acff0ef3a7b992effb9b3cb46a2f35" target='_blank'>
              TORC2 inhibition triggers yeast chromosome fragmentation through misregulated Base Excision Repair of clustered oxidation events
              </a>
            </td>
          <td>
            Kenji Shimada, Cleo V D Tarashev, Stephanie Bregenhorn, C. Gerhold, Barbara van Loon, Gregory Roth, Verena Hurst, Josef Jiricny, S. B. Helliwell, Susan M. Gasser
          </td>
          <td>2024-11-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Alterations in DNA methylation profiles are typically found in cancer cells, combining genome-wide hypomethylation with hypermethylation of specific regions, such as CpG islands, which are normally unmethylated. Driving effects in cancer development have been associated with alteration of DNA methylation in certain regions, inducing, for example, the repression of tumor suppressor genes or the activation of oncogenes and retrotransposons. These alterations represent prime candidates for the development of specific markers for the detection, diagnosis and prognosis of cancer. In particular, these markers, distributed along the genome, provide a wealth of information that offers potential for innovation in the field of liquid biopsy, in particular thanks to the emergence of artificial intelligence for diagnostic purposes. This could overcome the limitations related to sensitivities and specificities, which remain too low for the most difficult applications in oncology: the detection of cancers at an early stage, the monitoring of residual disease and the analysis of brain tumors. In addition, targeting the enzymatic processes that control the epigenome offers new therapeutic strategies that could reverse the regulatory anomalies of these altered epigenomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6734e14b7462be1947bbbaa4edaec73c090d31d" target='_blank'>
              [Epigenetics and cancer: the role of DNA methylation].
              </a>
            </td>
          <td>
            Marine Gorse, Charline Bianchi, Charlotte Proudhon
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="H3K27-altered diffuse midline glioma (DMG) is an aggressive and treatment-resistant form of pediatric high-grade glioma (pHGG). The disease is defined by point mutations in histone H3 that convert lysine 27 to methionine (termed H3K27M), resulting in genome-wide epigenetic changes that drive tumorigenesis. While radiation therapy is the standard of care, subsequent recurrence, often within the high dose radiation field, is universal. We found that the apical DNA damage response (DDR) kinase Ataxia Telangiectasia-Mutated (ATM) was uniquely upregulated in H3K27M-expressing patient tumor samples compared to pHGG expressing only wild-type histone H3. Using a panel of H3K27 isogenic DMG cell lines, we further found that H3K27M was associated with reduced H3K27me3 within the ATM promoter, increased ATM mRNA levels, and elevated DDR signaling, even in the absence of exogenous DNA damage. Consistent with these results, AZD1390, a clinical-grade, CNS-penetrant ATM inhibitor, sensitized H3K27M neurospheres to the long-term effects of radiation on survival, in part due to attenuated repair of radiation-induced DNA damage. Finally, AZD1390 sensitized orthotopic H3K27M mutant tumors to radiotherapy and significantly extended median survival relative to vehicle, AZD1390 or radiation alone (50 days vs 31, 36 or 39 days, respectively) with minimal adverse effects. Taken together, these data provide a direct mechanistic link between the H3K27M mutation and ATM expression and support the clinical investigation of AZD1390 with radiotherapy in H3K27M-altered DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4deb9890cca07ab9724e554ef1a2c801a2ce473" target='_blank'>
              Inhibition of H3K27M-enhanced ATM signaling increases radiation efficacy in diffuse midline glioma
              </a>
            </td>
          <td>
            Erik R Peterson, L. A. Parsels, J. Parsels, Xinyi Zhao, Maria G. Castro, Theodore S. Lawrence, C. Koschmann, Qiang Zhang, Daniel R Wahl, Meredith A. Morgan
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="PURPOSE
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge.


EXPERIMENTAL DESIGN
We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. The DIAMOND (Detection of Long Interspersed Nuclear Element Altered Methylation ON plasma DNA) method targets 30-40,000 young L1 scattered throughout the genome, covering about 100,000 CpG sites and is based on a reference-free analysis pipeline.


RESULTS
Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers (colorectal, breast, lung, ovarian, gastric cancers and uveal melanoma including localized stages) in two independent cohorts (AUC = 88% to 100%, N = 747). DIAMOND can also be used to perform copy number alterations (CNA) analysis which improves cancer detection.


CONCLUSIONS
This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321c21ecf5626d3880de9fbd35f995ec3b32a1f7" target='_blank'>
              Non-invasive multi-cancer detection using DNA hypomethylation of LINE-1 retrotransposons.
              </a>
            </td>
          <td>
            M. Michel, Maryam Heidary, Anissa Mechri, Kévin Da Silva, Marine Gorse, Victoria Dixon, Klaus von Grafenstein, Charline Bianchi, C. Hego, A. Rampanou, C. Lamy, M. Kamal, C. Le Tourneau, M. Séné, I. Bieche, Cécile Reyes, D. Gentien, Marc-Henri Stern, Olivier Lantz, L. Cabel, J-Y Pierga, F. Bidard, Chloé-Agathe Azencott, C. Proudhon
          </td>
          <td>2024-12-02</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="To maintain genome stability, proliferating cells must enact a program of telomere maintenance. While most tumors maintain telomeres through the action of telomerase, a subset of tumors utilize a DNA-templated process termed Alternative Lengthening of Telomeres or ALT. ALT is associated with mutations in the ATRX/DAXX/H3.3 histone chaperone complex, which is responsible for deposition of non-replicative histone variant H3.3 at heterochromatic regions of the genome including telomeres. We wished to better understand the role DAXX plays in ALT suppression, and to determine which disease-associated DAXX mutations are unable to suppress ALT. To answer this question, we have leveraged the G292 cell line, in which ATRX is wild type but DAXX has undergone a fusion event with the non-canonical kinesin KIFC3. Restoration of wild-type DAXX in G292 localizes ATRX and abrogates ALT. Using this model system, we tested the ability of a panel of disease-associated DAXX missense variants to suppress ALT. Missense mutations in the ATRX binding domain, the histone binding domain, and the C-terminal SUMO interaction motif reduce the ability of DAXX to suppress ALT. Unexpectedly, we find that mutations in the DAXX histone binding domain lead to failure of ATRX localization. We conclude that a key function of DAXX in ALT suppression is the localization of ATRX to nuclear foci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecce9b8b3920bbf0f0b14800927fbabcdf8182c8" target='_blank'>
              Cancer-associated DAXX mutations reveal a critical role for ATRX localization in ALT suppression
              </a>
            </td>
          <td>
            Sarah F. Clatterbuck Soper, Robert L. Walker, M. Pineda, Y. Zhu, James Dalgleish, Jasmine Wang, Paul S. Meltzer
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Glioma is the most common and aggressive primary adult brain tumor. Radiation therapy (RT) is a critical part of glioma treatment, but radioresistance severely limits its therapeutic efficacy. To better understand how radioresistance develops, we analyzed 212 paired primary and recurrent gliomas from the Glioma Longitudinal Analysis Consortium. We found a significant increase in mutational signatures associated with the activity of the Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like (APOBEC) family of cytidine deaminases in RT-treated recurrent gliomas (p<0.001, paired Wilcoxon signed-rank test). Further analysis of post-treatment mutations showed that RT increased APOBEC mutational signatures independent of other factors (p<0.001, multivariable log-linear regression), suggesting that APOBEC proteins become activated by radiation. However, their role in promoting tumor evolution and treatment resistance is currently unknown. Further analyses of extended signature contexts and expression data nominated APOBEC3B (A3B) and APOBEC3G (A3G) as candidates for creating these mutational signatures. To investigate their roles in RT response, we knocked out (KO) A3B and A3G individually in patient-derived glioma cell lines before RT. Compared to Control cells, irradiated A3B-KO and A3G-KO cells acquired significantly fewer APOBEC mutational signatures by whole genome sequencing (p<0.05, Kruskal–Wallis test) and exhibited enhanced radiosensitivity in vitro. Interestingly, the KO cells displayed a significant decrease in mutational signatures associated with Non-Homologous End-Joining (NHEJ), one of the main pathways for repairing the RT-induced DNA damage (p<0.05, Kruskal–Wallis test), suggesting that APOBEC deletion impaired DNA repair. This was further supported by persistent gH2AX DNA damage foci 48h post-RT and reduced autophosphorylation of DNA-dependent protein kinase (DNA-PK), a vital component of the NHEJ pathway, particularly in A3G KO cells. Overall, our results identify APOBEC proteins as potential targets for radiosensitizing gliomas and highlight the clinical utility of using mutational signatures to predict and monitor response to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a3ae9f5828a41ce57986e17a653e14abeac0226" target='_blank'>
              DNAR-16. TARGETING APOBEC CYTIDINE DEAMINASES TO ENHANCE RADIOSENSITIVITY IN GLIOMA
              </a>
            </td>
          <td>
            Bianca M Marin, Amit Gujar, E. Kocakavuk, K. Johnson, Samirkumar B Amin, R. Verhaak
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="

 Introduction : Megakaryocytes (MKs) are unique cells that acquire multiple copies of their nuclear genome via endomitosis - an unusual variant of the cell cycle in which failure of late cytokinesis leads to successive rounds of whole genome duplication (WGD). WGD+ cells are genetically unstable. To protect against this, in most human cells, centrosome accumulation triggers activation of a cell apparatus called the PIDDosome. This degrades MDM2, causing p53-mediated cell cycle arrest and apoptosis. How MKs bypass this checkpoint, and the consequences of WGD for MK genome integrity are poorly understood. Of note, MK ploidy varies over human ontogeny and in disease states. While adult MKs are typically 16N, fetal MKs are 2-4N, and low ploidy MKs occur in blood cancers eg myelofibrosis and certain inherited thrombocytopenias. Crucially, WGD is also a common feature of cancer genomes, where it is a catalytic event for cancer evolution. 1/3 cases occur with an intact p53 pathway, similar to MKs. Therefore, understanding the tolerance mechanisms and consequences of WGD in MKs has broad implications for human health.
 Methods : Cell cycle analysis, ploidy (Hoechst), single cell ATAC & RNAseq (SmartSeq-3), and DNA damage was functionally assessed in ex vivo isolated and in vitro differentiated MKs from adult bone marrow (BM), WT and p53 KO murine BM and healthy human cord blood (CB).
 Results: Significantly more γH2AX foci (indicative of double stranded DNA breaks) were found in unperturbed, healthy adult BM polyploid MKs than diploid MKs (% MKs with > 5 γH2AX foci, Diploid = 0.85%, Polyploid = 14.6%, p = 0.03, n = >150 cells). Polyploid MKs also showed impaired resolution of γH2AX foci after low-dose irradiation. These data confirm that WGD is detrimental for MK genome integrity.
 To interrogate this, we analysed >1400 highly purified human MKs ranging 2-16N (n=5 healthy adult donors). Single cell analysis is vital to accurately distinguish MKs from non-MK cells and enable two important corrections - removing doublets by genetic demultiplexing and predicting cell cycle phase to correct for G2M cells ‘masquerading’ as a higher ploidy class. We confirmed that, in contrast to most other human cells, MKs continually progress through cell cycle, even after WGD. While genes encoding essential nuclear proteins increased with ploidy, DNA repair pathway genes were insufficiently scaled with nuclear size, potentially explaining the impaired γH2AX clearance observed.
 We predicted that p53 activation may restrain MK ploidy. Surprisingly, BM MKs from p53KO mice showed no difference in ploidy vs WT MKs, but mature WT MKs showed highly efficient p53-mediated cell cycle arrest upon irradiation. Together, this indicated that p53 is proficient in MKs, but does not respond to WGD.
 We next explored how MKs might uncouple WGD sensing from p53 activation. ANKRD26 activates the PIDDosome upon centrosome amplification, and its expression is downregulated during MK maturation. To test whether this explains the lack of p53 activation in response to WGD, we differentiated MKs from CD34+ HSPCs of patients with ANKRD26-Related Thrombocytopenia (ANKRD26-THC), in whom genetic variants prevent RUNX1-mediated transcriptional repression of ANKRD26. ANKRD26-THC MKs were low ploidy, comparable to CB MKs (% 8N+ Cells, Adult: 10.95%, ANKRD26-THC: 5.62%, CB: 2.43%, n=3-5), and showed strikingly increased p53 activation by imaging. Ongoing interrogation of the molecular regulation of ploidy in healthy adult, CB and ANKRD26-THC MKs will delineate the role of the centrosome-p53 axis across ontogeny/disease states.
 Conclusions : To our knowledge, this study presents the first detailed examination of MK nuclear dynamics and genome integrity. We demonstrate a previously unappreciated accumulation of DNA damage with ploidy, and show that transcriptional repression of ANKRD26 uncouples p53 responses to WGD from DNA damage response, while enforced ANKRD26 expression has profound effects on MK ploidy. Differential PIDDosome activity may also explain the physiological variation in MK ploidy across human ontogeny and in blood disorders. More broadly, hijacking these mechanisms could also provide a mechanism by which p53-intact cancer genomes tolerate WGD, indicating a potential therapeutic vulnerability. This study highlights how interrogating unusual aspects of MK biology is broadly relevant to understanding human physiology and disease states.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfeb596bf95eb612e57257e98bb3559b7cf5c2a3" target='_blank'>
              Tolerance of Whole Genome Duplication in Megakaryocytes, and Consequences for Their Genome Integrity
              </a>
            </td>
          <td>
            A. Rodriguez-Romera, Rong Li, Nah-Young Shin, SA Clark, A. Adamo, Affaf Aliouat, Abdullah Khan, Aude-Anais Olijnik, Jill Brown, Lucia Sommerova, Natalie J Jooss, Akiko Shimamura, Xin Lu, Guanlin Wang, S. Constantinescu, Ketan Patel, A. Mead, Alan B. Cantor, B. Psaila
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras-particularly in diseases such as oral squamous cell carcinoma and glioblastoma-now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy.
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, Tali Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The BRCA2 tumor suppressor protein has roles in homologous recombination DNA repair and replication fork protection and carriers of mutation in the BRCA2 gene (BRCA2mut/+) are at high risk for breast cancers as a consequence of genomic instability. Mammary epithelial cells (MECs) are subjected to significant hormone-driven replication. BRCA2mut/+ MECs must therefore be able to mitigate RS-induced chronic DNA damage that could result in cell death or senescence. In response to acute RS in hTERT-MECs, we have found that the EGFR ligand, amphiregulin (AREG) rapidly undergoes retrograde trafficking to the nucleus and nuclear membrane (NM) in association with lamin A resulting in a transient increase in H3K9me3 heterochromatin most markedly in BRCA2mut/+ hTERT-MECs. RS promoted NM incorporation of prelamin A that was dependent on endogenous but not exogenous AREG. Prelamin A is reported to stabilize SUV39h1 and expression of a C-terminal truncated AREG that translocates to the NM increased SUV39h1, HP1α and H3K9me3. AREG knockdown reduced both HP1α and SUV39h1 accompanied by massive decompaction and reduction in global H3K9me3 heterochromatin. Loss of heterochromatin upon AREG depletion increased DNA damage, reduced replication fork speed and resulted in a significant increase in firing of new and multiple origins. Knockdown of endogenous AREG also impacted the NM resulting in activation of a type-1 interferon (IFN)-like response promoting senescence. Chronic RS also stimulated nuclear AREG along with FOXM1 and ESR1 mRNA and protein. Consistent with this, nuclear AREG, FOXM1 and ERα were detected in hyperplastic regions of human tissue. Overall, our work reveals a critical and unanticipated nuclear role for AREG in maintenance and transient induction of heterochromatin in response to RS preventing excessive DNA damage and senescence in BRCA2mut/+ MECs. Since nuclear AREG also promotes ERα expression, this may link RS with aberrant expansion of estrogen receptor-expressing MECs found in BRCA2mut/+ mammary tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7115be9e7c57e4cfb92a7b960c20389981e78db4" target='_blank'>
              Intracellular amphiregulin is critical for heterochromatin maintenance and genomic stability in response to replication stress in BRCA2mut/+mammary epithelial cells
              </a>
            </td>
          <td>
            Tianqi Jiang, A. Zein, M.A. Christine Pratt
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mammalian RAD52 is a DNA repair factor with strand annealing and recombination mediator activities that appear important in both interphase and mitotic cells. Nonetheless, RAD52 is dispensable for cell viability. To query RAD52 synthetic lethal relationships, we performed genome-wide CRISPR knock-out screens and identified hundreds of candidate synthetic lethal interactions. We then performed secondary screening and identified genes for which depletion causes reduced viability and elevated genome instability (increased 53BP1 nuclear foci) in RAD52-deficient cells. One such factor was ERCC6L, which marks DNA bridges during anaphase, and hence is important for genome stability in mitosis. Thus, we investigated the functional interrelationship between RAD52 and ERCC6L. We found that RAD52 deficiency increases ERCC6L-coated anaphase ultrafine bridges, and that ERCC6L depletion causes elevated RAD52 foci in prometaphase and interphase cells. These effects were enhanced with replication stress (i.e. hydroxyurea) and topoisomerase IIα inhibition (ICRF-193), where post-treatment effect timings were consistent with defects in addressing stress in mitosis. Altogether, we suggest that RAD52 and ERCC6L co-compensate to protect genome stability in mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef49a71354166033d520b118cfb94ac951639e9c" target='_blank'>
              RAD52 and ERCC6L/PICH have a compensatory relationship for genome stability in mitosis
              </a>
            </td>
          <td>
            Beth Osia, Arianna Merkell, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Jeremy M. Stark
          </td>
          <td>2024-11-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Maintenance of the plant organelle genomes involves factors mostly inherited from their bacterial symbiotic ancestors. In bacteria, a major player in genome maintenance is DNA Polymerase I (Pol I), which provides a 5’-3’-exonuclease/flap-endonuclease activity required for multiple replication and repair functions. In plant organelles, DNA polymerases POL1A and POL1B are evolutionarily derived from DNA Pol I but lack this domain. In Arabidopsis, OEX1 and OEX2 (Organellar Exonucleases 1 and 2) represent this missing domain and are targeted to mitochondria and chloroplasts, respectively. An oex1 mutant allele shows developmental and fertility defects that correlate with the differential segregation of mitochondrial DNA (mtDNA) subgenomes generated by recombination, suggesting that OEX1 processes replication and recombination intermediates whose accumulation results in genome instability. Alternative splicing generates two OEX1 isoforms that can differentially interact with POL1A and POL1B and variably affect mtDNA repair. Recombinant OEX1 has 5’-3’-exonuclease and flap endonuclease activities, the latter being a key function in replication and repair. Furthermore, OEX1 has high affinity for RNA:DNA hybrids, rapidly degrading RNA in Okazaki-like structures and R-loops. Consistent with a role in suppressing R-loops, oex1 plants accumulate R-loops in highly transcribed mtDNA regions. Taken together, our results show that OEX1 plays multiple important roles in the processes required to maintain mtDNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48bd0220633827dbb43a1426d794cf0014df2476" target='_blank'>
              R-Loop control and mitochondria genome stability require the 5’-3’ exonuclease/flap endonuclease OEX1
              </a>
            </td>
          <td>
            Déborah Schatz-Daas, Anaïs Le Blevenec, Fabio Moratti, Kin Pan Chung, Pierre Mercier, Rana Khalid Iqbal, Elody Vallet, André Dietrich, Ralph Bock, F. Weber-Lotfi, J. Gualberto
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12abdeefb14dd8084b12c06a67b90a6d3dae88f" target='_blank'>
              CPNE7 promotes colorectal tumorigenesis by interacting with NONO to initiate ZFP42 transcription
              </a>
            </td>
          <td>
            Liangbo Zhao, Xiao Sun, Chenying Hou, Yanmei Yang, Peiwen Wang, Zhaoyuan Xu, Zhenzhen Chen, Xiangrui Zhang, Guanghua Wu, Hong Chen, Hao Xing, Huimin Xie, Luyun He, Shuiling Jin, Benyu Liu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia-telangiectasia-mutated) and ATR (Ataxia-telangiectasia-mutated and Rad3-related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure to UV-B, UV-C and protons induced the microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana.
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background: The Neuro-2a cell line, derived from a murine neuroblastoma (NB), was established as early as 1969 and originates from a transplantable tumor that arose spontaneously in an A/Jax male mouse in 1940. Since then, it has been applied in over 10,000 studies and is used by the World Organization for Animal Health for the routine diagnosis of rabies. Surprisingly, however, Neuro-2a has never been genetically characterized in detail; this study fills that gap. Methods: The Neuro-2a cell line and two of its derivatives, Neuro-2a TR-alpha and Neuro-2a TR-beta, were analyzed for their chromosomal constitution using molecular cytogenetic approaches. Array comparative genomic hybridization was performed to characterize copy number alterations. Results: Neuro-2A has a hyper-tetraploid karyotype with 70 to 97 chromosomes per cell, and the karyotypes of its two examined derivatives were quite similar. Neither of them had a Y-chromosome. The complex karyotype of Neuro-2a includes mitotically stable dicentres, neocentrics, and complex rearrangements resembling chromothripsis events. Although no amplification of euchromatin or oncogenes was detected, there are five derivative chromosomes with the amplification of centromere-near heterochromatic material and 1–5 additional derivatives consisting only of such material. Conclusions: Since satellite DNA amplification has recently been found in advanced human tumors, this finding may be the corresponding equivalent in mice. An in silico translation of the obtained results into the human genome indicated that Neuro-2A is suitable as a model for advanced human NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b64be2f9391134627028421230fd8316c10590" target='_blank'>
              Cytogenomic Characterization of Murine Neuroblastoma Cell Line Neuro-2a and Its Two Derivatives Neuro-2a TR-Alpha and Neuro-2a TR-Beta
              </a>
            </td>
          <td>
            Lioba Hergenhahn, Niklas Padutsch, S. Azawi, Ralf Weiskirchen, Thomas Liehr, Martina Rinčic
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Insertion sequences (ISs) are mobile pieces of DNA that are widespread in bacterial genomes. IS movements typically involve (i) excision of the IS element, (ii) cutting of target site DNA, and (iii) IS element insertion. This process generates a new copy of the IS element and a short duplication at the target site. It has been noted that, for some extant IS copies, no target site duplications (TSDs) are readily identifiable. TSD absence has been attributed to degeneration of the TSD after the insertion event, recombination between identical ISs, or adjacent deletions. Indeed, the latter two-recombination between ISs and adjacent deletions-are frequent causes for the absence of TSDs, which we demonstrate here in an analysis of genome sequence data from the Lenski long-term evolution experiment. Furthermore, we propose that some IS movements-namely, those that occur in association with large-scale genomic rearrangements-do not generate TSDs, and occur without evidence for recombination between ISs or adjacent deletions. In support of this hypothesis, we provide two direct, empirical observations of such IS transposition events: an IS5 movement plus a large deletion in Escherichia coli C, and an IS481 movement occurring with a large duplication in Pseudomonas fluorescens SBW25. Although unlikely, it is possible that the observed deletion and associated IS movement occurred in two successive events in one overnight culture. However, an IS at the center of a large-scale duplication is not readily explained, suggesting that IS element activity may promote both large-scale deletions and duplications.


IMPORTANCE
Insertion sequences are the most common mobile genetic elements found in bacterial genomes, and hence they significantly impact bacterial evolution. We observe insertion sequence movement at the center of large-scale deletions and duplications that occurred during laboratory evolution experiments with Escherichia coli and Pseudomonas fluorescens, involving three distinct types of transposase. We raise the possibility that the transposase does not mediate DNA cleavage but instead inserts into existing DNA breaks. Our research highlights the importance of insertion sequences for the generation of large-scale genomic rearrangements and raises questions concerning the mechanistic basis of these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00001ae08c5c01e65cbd4f4a4b234bdd427cfa24" target='_blank'>
              A more significant role for insertion sequences in large-scale rearrangements in bacterial genomes.
              </a>
            </td>
          <td>
            Wing Y. Ngan, Lavisha Parab, Frederic Bertels, J. Gallie
          </td>
          <td>2024-12-05</td>
          <td>mBio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86fef8bdb07b67c65a912b81891de3ba97bc74cb" target='_blank'>
              Osmotic disruption of chromatin induces Topoisomerase 2 activity at sites of transcriptional stress
              </a>
            </td>
          <td>
            William H. Gittens, Rachal M. Allison, Ellie M Wright, George G B Brown, Matthew J. Neale
          </td>
          <td>2024-12-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) is a retrovirus responsible for adult T-cell leukemia/lymphoma (ATLL) and HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive neurodegenerative disease. Regulation of viral gene expression plays a key role in viral persistence and pathogenesis. However, the molecular mechanisms underlying this fine-tuned regulation remain poorly understood. Little is known regarding RNA chemical modifications of HTLV-1 RNA and how these affect viral biology and disease development. Posttranscriptional chemical modification of RNA is common in eukaryotes, with N6-methyladenosine (m6A) being the most prevalent. In this study, we investigated the role of m6A RNA modifications on HTLV-1 gene expression. Using MeRIP-Seq, we mapped sites of m6A modification to the 3’ end of the viral genome. We found HTLV-1 RNA, as well as viral oncogene transcripts tax and hbz, contained m6A modifications. m6A-depletion in HTLV-1-transformed cells decreased sense-derived viral genes (Tax, Gag, Env) and increased antisense-derived Hbz expression. Tax and hbz transcripts were bound by reader proteins YTHDF1 and YTHDC1 in a panel of HTLV-1 T-cell lines. Using expression vectors and shRNA-mediated knockdown, we found YTHDF1 had opposing effects on viral gene expression – decreasing sense-derived viral genes and increasing antisense-derived Hbz. Upon further study, the YTHDF1 effects on tax abundance were dependent on tax m6A deposition. The nuclear m6A reader protein YTHDC1 affected the abundance of both sense- and antisense-derived viral transcripts and specifically enhanced the nuclear export of tax transcript. Collectively, our results demonstrate global m6A levels and m6A reader proteins YTHDF1 and YTHDC1 regulate HTLV-1 gene expression. Importance Human T-cell leukemia virus type 1 (HTLV-1) persistence and pathogenesis are controlled through tight regulation of viral gene expression. The fate of RNA can be controlled by epigenetic modifications which impact gene expression without altering the DNA sequence. Our study details the impact of N6-methyladenosine (m6A) RNA chemical modifications on HTLV-1 gene expression. We found reductions in global m6A levels affected viral gene expression, decreasing Tax and other sense-derived viral genes, while increasing the antisense-derived Hbz. Our results suggest the oncogenic viral transcripts, tax and hbz, are m6A-modified in cells. We found these viral RNA modifications are interpreted by reader proteins YTHDF1 and YTHDC1, which dictate the fate of the viral RNA. Understanding HTLV-1 RNA chemical modifications offers potential insights into novel therapeutic strategies for HTLV-1-associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6351be4fe50d4418a28feb2032fbd8b3eec3a2ff" target='_blank'>
              YTHDF1 and YTHDC1 m6A reader proteins regulate HTLV-1 tax and hbz activity
              </a>
            </td>
          <td>
            Emily M. King, A. Midkiff, Karsyn McClain, Sanggu Kim, A. Panfil
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="A central goal of cancer genomics is to identify, in each patient, all the cancer-driving mutations. Among them, point mutations are referred to as cancer-driving nucleotides (CDNs), which recur in cancers. The companion study shows that the probability of i recurrent hits in n patients would decrease exponentially with i; hence, any mutation with i ≥ 3 hits in The Cancer Genome Atlas (TCGA) database is a high-probability CDN. This study characterizes the 50–150 CDNs identifiable for each cancer type of TCGA (while anticipating 10 times more undiscovered ones) as follows: (i) CDNs tend to code for amino acids of divergent chemical properties. (ii) At the genic level, far more CDNs (more than fivefold) fall on noncanonical than canonical cancer-driving genes (CDGs). Most undiscovered CDNs are expected to be on unknown CDGs. (iii) CDNs tend to be more widely shared among cancer types than canonical CDGs, mainly because of the higher resolution at the nucleotide than the whole-gene level. (iv) Most important, among the 50–100 coding region mutations carried by a cancer patient, 5–8 CDNs are expected but only 0–2 CDNs have been identified at present. This low level of identification has hampered functional test and gene-targeted therapy. We show that, by expanding the sample size to 105, most CDNs can be identified. Full CDN identification will then facilitate the design of patient-specific targeting against multiple CDN-harboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb003947f43738ae589b3a6538f7d27fb518829" target='_blank'>
              Characterization of cancer-driving nucleotides (CDNs) across genes, cancer types, and patients
              </a>
            </td>
          <td>
            Lingjie Zhang, Tong Deng, Zhongqi Liufu, Xiangnyu Chen, Shijie Wu, Xueyu Liu, Changhao Shi, Bingjie Chen, Zheng Hu, Qichun Cai, Chenli Liu, Mengfeng Li, Miles E. Tracy, Xuemei Lu, Chung-I Wu, Hai-Jun Wen
          </td>
          <td>2024-12-17</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetic regulation mechanisms such as DNA methylation and histone acetylation are important for controlling various biological phenomena by regulating gene expression at the genome level. They are reversible systems that change depending on environmental factors. Epigenetic abnormalities are associated with the onset of various diseases, including developmental and aging abnormalities, neurological disorders, and malignant tumors. Aberrant DNA methylation is an important epigenetic change in the development and progression of colorectal cancer. DNA methylation in tumor tissues occurs mainly in CpG islands in the promoter regions of genes and inactivates gene functions by negatively suppressing transcription. CpG island methylator phenotype(CIMP)is an important carcinogenic mechanism of colorectal cancer related to DNA methylation and is involved in approximately 20% of all colorectal cancers. CIMP is generally judged to be positive when a certain percentage or more of the marker gene set is methylated, and many CIMP markers have been reported so far. However, no established marker has been set to classify colorectal cancer by genome-wide DNA methylation. We developed a new method to assess genome-wide DNA methylation status and obtained pharmaceutical approval as a new in vitro diagnostic drug to predict sensitivity to anti-EGFR antibody drugs in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd19841c8e119be51563341cefa28e2c2deaf96" target='_blank'>
              [Epigenome Diagnosis of Cancer-Focusing on Genome-Wide DNA Methylation Diagnosis of Colorectal Cancer for Predicting Sensitivity of Anti-EGFR Antibody Treatment].
              </a>
            </td>
          <td>
            C. Ishioka, K. Ouchi, Shonosuke Wakayama, Shin Takahashi
          </td>
          <td>2024-11-01</td>
          <td>Gan to kagaku ryoho. Cancer & chemotherapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62a212b47508047875bfe6549eb4ee11cd53c6" target='_blank'>
              Survey for Activating Oncogenic Mutation Variants in Metazoan Germline Genes.
              </a>
            </td>
          <td>
            Karl E Krueger
          </td>
          <td>2024-11-26</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Oncoprotein-driven human cancers exhibit replication stress (RS), leading to an increase in DNA double-strand breaks (DSB). These DSBs are repaired by one of the three main pathways - homologous recombination repair (HR), non-homologous end-joining (NHEJ), and microhomology-mediated end joining (MMEJ). Cyclin D1 (CCND1) is one of the most overexpressed oncoproteins and is universally overexpressed in mantle cell lymphoma (MCL). While CCND1 overexpression drives cells into the S-phase, its specific effects on RS and DSB repair pathways are poorly understood. Here, we investigated the impact of CCND1 overexpression on RS and DSB repair mechanisms in MCL.
 First, the expression level of CCND1 positively correlated with heightened RS, as indicated by elevated p-RPA-S33 and p-RPA S8 levels, and increased DNA damage, assessed by g-H2AX, especially in the mitotic phase, across diverse cell models, including MCL. Since CCND1-overexpressed cells proliferate rapidly, we hypothesized that some DNA repair pathways might be upregulated to meet the demand for repairing the excess RS-driven DSBs. Reporter assays demonstrated that the MMEJ pathway was specifically upregulated upon CCND1 overexpression, whereas no significant changes were observed in the HR and NHEJ pathways. Furthermore, CRISPR interference (CRISPRi)-mediated knockdown of CCND1 in MCL cells led to a significant reduction in MMEJ activity. This finding suggests the specificity of CCND1 in mediating the upregulation of the MMEJ pathway. Consistent with this, increased MMEJ activity, assessed via foci formation and immunoblotting of DNA polymerase theta expression (POLQ, encoded by POLQ gene), the primary polymerase for the MMEJ pathway, correlated with CCND1 overexpression, suggesting the crucial role of MMEJ in DSB repair processes under CCND1-overexpressed conditions.
 Sincethe MMEJ pathwayplays a pivotal role in repairing DSB during mitosis, we assessed the dependency of MCL cells on MMEJ-mediated DSB repair for cell survival. As expected, genetic deletion or pharmacological inhibition (ART558 and novobiocin) of POLQ in MCL cells led to increased RS and increased unrepaired DNA damage during mitosis, ultimately leading to chromosomal instability, mitotic catastrophe, and apoptosis. Additionally, the deficiency of POLQ in MCL cells resulted in increased levels of single-strand DNA, another critical biomarker for RS.
 Second, we found that ATM deficiency, the second most common genetic alteration after CCND1 overexpression in MCL, further increased the cellular dependence on MMEJ. Interestingly, the accumulation of unrepaired DNA damage during mitosis was notably increased in ATM-deficient CCND1-overexpressed cells. When POLQ was inhibited, a significant antitumor effect was evident in MCL cells, including ibrutinib-resistant MCL, both in vitro and in vivo, and this effect was enhanced by ATM deficiency. Further, simultaneous inhibition of both ATM (AZD0156) and POLQ was synthetic lethal in MCL cell lines and primary tumor cells from patients with MCL.
 Mechanistically, we observed the direct binding of CCND1 to the POLQ promoter and a corresponding upregulation of POLQ mRNA in CCND1-overexpressing cells. Consistently, a significant overexpression of POLQ mRNA was observed in primary MCL cells, compared to other non-Hodgkin lymphoma cells without CCND1 overexpression. Moreover, the positive correlation between POLQ and CCND1 expression was further validated using syngeneic B-lymphoma cells with and without CCND1 overexpression. Importantly, CRISPRi-mediated CCND1 knockdown reduced POLQ mRNA expression and the dependency of MCL cells on the MMEJ pathway for survival. In addition, ATM deficiency showed a further increase of endogenous POLQ expression and foci formation in cells with CCND1 overexpression, suggesting ATM negatively regulates the MMEJ pathway, especially in the CCND1 overexpressed background.
 In summary, CCND1 overexpression leads to a substantial increase in RS, intensifying the reliance on MMEJ-mediated mitotic DSB repair. Within MCL, POLQ is crucial in mitigating the adversities of CCND1-driven RS. Our data suggest that targeting the MMEJ pathway via POLQ inhibition, particularly in the presence of CCND1 overexpression and ATM deficiency, emerges as a promising therapeutic approach in oncoprotein-driven hematologic malignancies such as MCL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d732971cc0b21a57895c7d2927fa16e5ff9809f7" target='_blank'>
              Cyclin D1 Overexpression in Mantle Cell Lymphoma Causes Cellular Dependence on Microhomology-Mediated End-Joining
              </a>
            </td>
          <td>
            J. Abeykoon, Shuhei Asada, Guangli Zu, Yuna Hirohashi, Divya R Iyer, K. Parmar, Lige Jiang, Sirisha Mukkavalli, Dongni Yi, M. Manske, Xiaosheng Wu, Kimberly A. Gwin, Thomas E. Witzig, Geoffrey I. Shapiro, Alan D. D'Andrea
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 DNA methylation is a fundamental epigenetic mark that governs gene expression and chromatin organization, thus providing a window into cellular identity and developmental processes. Current datasets typically include only a fraction of methylation sites and are often based either on cell lines that underwent massive changes in culture or on tissues containing unspecified mixtures of cells. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing, allowing fragment-level analysis across thousands of unique markers for 39 cell types sorted from 205 healthy tissue samples. Replicates of the same cell type are more than 99.5% identical, demonstrating the robustness of cell identity programmes to environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny and identifies methylation patterns retained since embryonic development. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely hypermethylated loci are rare and are enriched for CpG islands, Polycomb targets and CTCF binding sites, suggesting a new role in shaping cell-type-specific chromatin looping. The atlas provides an essential resource for study of gene regulation and disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
 Citation Format: Tommy Kaplan. A DNA methylation atlas of normal human cell types [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr IA025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9c66abd8a0c90b855e4304f66a149ab9fbef96" target='_blank'>
              Abstract IA025: A DNA methylation atlas of normal human cell types
              </a>
            </td>
          <td>
            Tommy Kaplan
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Pediatric leukemias are commonly driven by chromosomal translocations which create gene fusions involving hematopoietic transcription factors (TFs) in progenitor lymphoid and myeloid cell populations. Although driver gene fusions and altered signaling pathways across leukemia subtypes have been extensively cataloged, the regulatory mechanisms enabling TF fusions to reprogram the epigenome and arrest hematopoietic differentiation remain unclear.
 In this study, we generated a single nucleus multiomic atlas from pediatric leukemia patient samples and leveraged deep learning models of regulatory DNA sequence to decipher the regulatory logic linking mutant TFs and regulatory elements to downstream genes and pathways in a sample and cell type-specific manner.
 We profiled 22 bone marrow specimens from major diagnostic categories (T-ALL, B-ALL, AML) and including recurrent gene fusions such as ETV6-RUNX1 and RUNX1-RUNX1T1. Using whole genome sequencing (WGS) and multiplexed 10X multiome profiling (single-nucleus RNA+ATAC-seq), we simultaneously profiled gene expression and chromatin accessibility for over 70,000 cells. Unsupervised clustering and cell type annotation revealed 21 distinct clusters, including leukemic and healthy cell populations. Demultiplexing using SNPs from WGS allowed us to recover sample identities and distinguish between malignant and healthy cell populations.
 To determine the sequence basis and downstream functional effects of TF rewiring in leukemia, we trained and interpreted ChromBPNet, a fully convolutional neural network, on ATAC-seq data from healthy and malignant B cell clusters. The model discovered enriched motifs for hematopoietic transcription factors, including RUNX1, ETV6, PAX5, and ERG, in ATAC peak regions of ETV6-RUNX1 B-ALL samples.
 This integrative approach provides new insights into how oncogenic fusions confer blocked differentiation, survival, and proliferation to leukemia cells. Future work will leverage these models to identify novel motifs for TF fusions and fine-map germline variants to identify functional mutations that perturb TF binding and accessibility through motif disruption.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88345dc2ca3b5874225a4a7d37d44b0f47170259" target='_blank'>
              Deep Learning the Cis-Regulatory Code of Epigenomic Rewiring By Fusion Transcription Factors in Pediatric Leukemia
              </a>
            </td>
          <td>
            Shouvik Mani, Fabian Grubert, D. Gratzinger, A. Kundaje, Maya Kasowski
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2100497dbb37ac929dfe5b9230f9192ece871ee3" target='_blank'>
              Inter-chromosomal contacts demarcate genome topology along a spatial gradient
              </a>
            </td>
          <td>
            Milad Mokhtaridoost, Jordan J. Chalmers, Marzieh Soleimanpoor, Brandon J. McMurray, Daniella F. Lato, Son C. Nguyen, Viktoria Musienko, Joshua O. Nash, S. Espeso-Gil, Sameen Ahmed, Kate Delfosse, Jared W. L. Browning, A. R. Barutcu, Michael D. Wilson, Thomas Liehr, Adam Shlien, Samin Aref, Eric F Joyce, A. Weise, Philipp G. Maass
          </td>
          <td>2024-11-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 By integrating whole-genome sequencing (WGS) and RNA-seq data, we found that intracranial germ cell tumors (IGCTs) can be divided into three distinct molecular subtypes: immune-hot, MYC/E2F, and SHH, each characterized by a unique profile of genomic abnormalities. To further define these subtypes, we subsequently performed whole-genome bisulfate sequencing (WGBS) on IGCT tumors



 Within our prior discovery cohort (n=110), DNA from 84 samples was available for profiling methylation levels using Whole Genome Bisulfite Sequencing (WGBS). Fastq data was aligned to the human reference genome (hg19), with duplicates removed, and DNA methylation was extracted using bismark pipeline (ver0.24.0). After quality control, weight methylation level was calculated for each CpG island as the sum of methylated reads divided by coverage. To identify IGCTs molecular groups, we applied hierarchical consensus k-means clustering to the most variant methylated CpG island (n=1000) across 84 IGCTs tumor tissues. The methylation level of RNA-seq subtypes was visualized to further clarify the molecular subtypes of IGCTs



 Unsupervised clustering revealed that IGCTs can be divided into three groups based on the CpG island methylation levels: hypomethylation group (n=32), moderate methylation group (n=27), and hyper methylation group (n=25). The hypomethylation and moderate methylation group are predominantly observed in germinoma (>90%). In contrast, hypermethylation group comprises the majority of non-germinomatous germ cell tumors. Furthermore, these subgroups derived from methylation analysis are highly consistent with the molecular subtypes previously defined through gene transcription profile analysis. We observed that the majority of MYC/E2F subtype (24/33) belonged to the hypomethylation group, whereas the majority of immune-hot subtype (14/23) fell into the moderate methylation group. The SHH subtype was the most common within the hypermethylation group (18/26)



 The three molecular IGCT subtypes showed unique methylation profile. These results indicate different mechanisms of tumorigenesis among the subtypes and pave the way for the clinical application of methylation-based subtyping.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d90945535c5dff7b25f6c9ce6ea42e158ea9d3d" target='_blank'>
              PATH-24. NOVEL MOLECULAR SUBTYPES IDENTIFIED IN INTRACRANIAL GERM CELL TUMORS SHOWED UNIQUE WHOLE-GENOME METHYLATION PROFILE
              </a>
            </td>
          <td>
            Bo Li, Xiaoguang Qiu
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1415b701bc925426d0f6efd42f7dbefb5852dc" target='_blank'>
              Replication stress induces POLQ-mediated structural variant formation throughout common fragile sites after entry into mitosis
              </a>
            </td>
          <td>
            Thomas E Wilson, Samreen Ahmed, Amanda H Winningham, Thomas W. Glover
          </td>
          <td>2024-11-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Jumping translocations (JT) are rare cytogenetic abnormalities associated with progression in myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). Typically, a tri–tetra‐somic 1q chromosome is translocated to two or more recipient chromosomes. In multiple myeloma JT were shown to originate after DNA demethylation and decondensation. Using epigenomics, we investigated sequential samples in an SRSF2‐mutated MDS and AML cohort with normal karyotype at diagnosis and 1qJT at disease evolution after 5′‐azacytidine (AZA). 1qJT breakpoints fell within repetitive DNA at both 1q12 and the translocation partners, namely acrocentrics n. 14, 15, 21, and 22, chromosome 16, and chromosome Y. The global methylome at diagnosis showed hypermethylation at 61% of the differentially methylated regions (DMRs), followed by hypomethylation at 80% of DMRs under AZA, mostly affecting pathways related to immune system, chromatin organization, chromosome condensation, telomere maintenance, rRNA, and DNA repair. At disease evolution, a shift toward hypermethylation, intronic enhancers enrichment and epigenetic involvement of the PI3K/AKT and MAPK signaling emerged. In particular, AKT1 phosphorylation behaved as a hallmark of the progression. Overall, we provided new insights on the characterization of 1qJT in SRSF2‐mutated myeloid neoplasms and first showed that epigenetics is a powerful tool to investigate the molecular landscape of repetitive DNA rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c67408d13a5333628e6fca2329dcdcc54f1fe5a1" target='_blank'>
              Epigenetic Modeling of Jumping Translocations of 1q Heterochromatin in Acute Myeloid Leukemia After 5'‐Azacytidine Treatment
              </a>
            </td>
          <td>
            A. G. Lema Fernandez, C. Nardelli, V. Pierini, B. Crescenzi, F. Pellanera, C. Matteucci, Maria Crocioni, S. Arniani, Valeria di Battista, M. Quintini, G. Mondanelli, C. Orabona, P. Gorello, Cristina Mecucci
          </td>
          <td>2024-11-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA methylation patterning is a consequence of opposing activities of DNA methyltransferases and DNA demethylases. In flowering plants, two distinct female gametes, the egg cell and the central cell, are fertilized, producing what will become the embryo and the endosperm of the seed. In Arabidopsis, a 5-methylcytosine DNA glycosylase, DME, demethylates regions in the central cell genome, leading to methylation differences between maternally- and paternally-inherited endosperm genomes after fertilization. DME is required for endosperm gene imprinting. Homologues of DME include ROS1, DML2 and DML3. It is unknown whether any of these DNA glycosylases are required for endosperm methylation patterning. We show that ROS1 prevents hypermethylation of paternally-inherited alleles in the endosperm at regions that lack maternal or paternal-allele methylation in wild-type. Thus, ROS1 promotes epigenetic symmetry between genomes in the endosperm by preventing paternal genome hypermethylation. We propose that ROS1 and DME act in a parent-of-origin-specific manner at shared endosperm targets, and consider implications for the evolution of imprinting mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec72c6a62fce89adf40409b047cafdf8c99f302e" target='_blank'>
              The 5-methylcytosine DNA glycosylase ROS1 antagonizes parent-of-origin specific DNA methylation in Arabidopsis endosperm
              </a>
            </td>
          <td>
            Elizabeth A. Hemenway, Mary Gehring
          </td>
          <td>2024-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The division of labor among cellular lineages is a pivotal step in the evolution of multicellularity. In mammals, the soma-germline boundary is formed during early embryogenesis, when genes that drive germline identity are repressed in somatic lineages through DNA and histone modifications at promoter CpG islands (CGIs). Somatic misexpression of germline genes is a signature of cancer and observed in select neurodevelopmental disorders. However, it is currently unclear if all germline genes use the same repressive mechanisms and if factors like development and sex influence their dysregulation. Here, we examine how cellular context influences the formation of somatic tissue identity in mice lacking lysine demethylase 5c (KDM5C), an X chromosome eraser of histone 3 lysine 4 di and tri-methylation (H3K4me2/3). We found male Kdm5c knockout (-KO) mice aberrantly express many tissue-specific genes within the brain, the majority of which are unique to the germline. By developing a comprehensive list of mouse germline-enriched genes, we observed Kdm5c-KO cells aberrantly express key drivers of germline fate during early embryogenesis but late-stage spermatogenesis genes within the mature brain. KDM5C binds CGIs within germline gene promoters to facilitate DNA CpG methylation as embryonic stem cells differentiate into epiblast-like cells (EpiLCs). However, the majority of late-stage spermatogenesis genes expressed within the Kdm5c-KO brain did not harbor promoter CGIs. These CGI-free germline genes were not bound by KDM5C and instead expressed through ectopic activation by RFX transcription factors. Furthermore, germline gene repression is sexually dimorphic, as female EpiLCs require a higher dose of KDM5C to maintain germline silencing. Altogether, these data revealed distinct regulatory classes of germline genes and sex-biased silencing mechanisms in somatic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6ca82d739eda98255fede09a6bf7335b693d55" target='_blank'>
              KDM5C is a sex-biased brake against germline gene expression programs in somatic lineages
              </a>
            </td>
          <td>
            Katherine M Bonefas, Ilakkiya Venkatachalam, Shigeki Iwase
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cytosine methylation is an important epigenetic modification that plays a crucial role in genomic regulation. The conventional second generation sequencing based bisulfite conversion methods used to interrogate cytosine methylation require comparing bisulfited-treated reads with a reference genome for methylation calling. Therefore, it cannot characterize the methylation of unaligned regions. Taking advantage of the recent improvement of the third generation sequencing, we investigated the methylation pattern of human lymphoblastoid cell lines (LCLs) of non-reference insertions from the human population, with a focus on polymorphic transposable elements. We first characterized whole-genome CpG methylation using both SMRT and nanopore technology and benchmarked their performance against WGBS using five human LCLs included in the draft Human Pangenome Reference. Both methods are highly correlated with the conventional WGBS results across the genome. The level of differences between PacBio and ONT on the same sample is comparable with that of two replicates of WGBS of the same sample. Using long-read data from the draft Human Pangenome Reference, we characterized CpG methylation of non-reference insertions, especially polymorphic transposable elements. We focused on addressing two questions: 1) do newly inserted TEs adopt the methylation pattern of their genomic context? and 2) do methylation spread from new TE insertions to their flanking regions? We found that most non-TE insertions exhibit DNA methylation pattern consistent with their genomic context, but TE insertions are consistently methylated, with a few exceptions. We also found limited methylation spreading from Alu/L1 insertions to their flanking genomic regions. We investigated INDEL frequency in both hypermethylated and hypomethylated CpG islands and found INDELs are enriched in hypermethylated CpG islands. Our work demonstrated the methylation calling capability of the 3rd generation sequencing and its unique advantage in characterizing epigenomic features within non-reference positions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969f23ef61be78d0dc966d0d35b748cd55ef16c5" target='_blank'>
              Characterizing cytosine methylation of polymorphic human transposable element insertions using human pangenome resources
              </a>
            </td>
          <td>
            Xiaoyu Zhuo, Chad Tomlinson, E. Belter, Prashant Kumar Kuntala, Wesley N. Saintilnord, Tina Lindsay, Juan Macias, Robert S. Fulton, Ting Wang
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND The mitotic DNA integrity checkpoint signaling pathway is potentially involved in cancers that regulate genomic stability where protein kinases play a pivotal role. 16 total protein kinase genes are involved in this pathway: ATM, BRSK1, CDK1, CDK2, CHEK1, CHEK2, MAP3K20, NEK11, PLK1, PLK2, PLK3, PRKDC, STK33, TAOK1, TAOK2, and TAOK3. This study aims to provide pan-cancer profiles of the protein kinases in mitotic DNA integrity checkpoint signaling gene set for potential prognostic and diagnostic purposes, as well as future potential therapeutic targets for cancer in a clinical setting. METHODS Multi-omic data was acquired for the 16 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of SNV, CNV, methylation, mRNA expression, pathway crosstalk, and microRNA regulation networks. RESULTS The SNV profile showed that most of these genes have a high SNV mutation frequency across some cancer types, such as UCEC and SKCM. The CNVs of some of these genes are associated with the survival of UCEC, KIRP, and LGG. BRCA, KIRC, LUAD, and STAD might be affected by the mRNA expression of these genes which might involve regulation of copy number, methylation, and miRNA. In addition, these genes also cross-talk with some known cancer pathways. CONCLUSION The protein kinases in mitotic DNA integrity checkpoint signaling may play a role in cancer development and, with adequate research, could potentially be developed as biomarkers for cancer diagnosis and prognosis. However, further efforts are necessary to validate their clinical value for diagnosis and prognosis and to develop practical applications in clinical settings. Nevertheless, these pan-cancer profiles offer a better overall understanding as well as useful information for future reference regarding mitotic DNA integrity checkpoint signaling in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bfd3e9f554799cf23b8ba08f42b425055efabcc" target='_blank'>
              Pan-Cancer Genetic Profiles of Mitotic DNA Integrity Checkpoint Protein Kinases
              </a>
            </td>
          <td>
            Ayana Meegol Rasteh, Hengrui Liu, Panpan Wang
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric malignancies frequently harbor chromosomal translocations that induce expression of fusion oncoproteins. The EWSR1-FLI1 fusion oncoprotein acts as a neomorphic transcription factor and is the dominant genetic driver of Ewing’s sarcoma. Interrogation of the mechanisms by which EWSR1-FLI1 drives tumorigenesis has been limited by a lack of model systems to precisely and selectively control its expression in patient-derived cell lines and xenografts. Here, we report the generation of a panel of patient-derived EWS cell lines in which inducible protein degrons were engineered into the endogenous EWSR1-FLI1 locus. These alleles enabled rapid and efficient depletion of EWSR1-FLI1. Complete suppression of EWSR1-FLI1 induced a reversible cell cycle arrest at the G1-S checkpoint, and we identified a core set of transcripts downstream of EWSR1-FLI1 across multiple cell lines and degron systems. Additionally, depletion of EWSR1-FLI1 potently suppressed tumor growth in xenograft models validating efforts to directly target EWSR1-FLI1 in Ewing’s sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7730c36245c38a450ad2b9993e2be4d0895d9f0" target='_blank'>
              Endogenous EWSR1-FLI1 degron alleles enable control of fusion oncoprotein expression in tumor cell lines and xenografts
              </a>
            </td>
          <td>
            James H. McGinnis, Alberto Bremauntz Enriquez, Florentina Vandiver, Xin Bai, Jiwoong Kim, Jessica Kilgore, Purbita Saha, Ryan O’Hara, Yang Xie, Laura A. Banaszynski, Noelle S. Williams, David G McFadden
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Introduction: Since DNA strand breaks are required for genomic rearrangements, genes involved in DNA breaks can potentially drive genomic instability and clonal evolution. Therefore, we developed a novel genome-wide functional phenotyping screen to identify genes whose modulation impacts DNA breaks in multiple myeloma (MM).
 Methods: Myeloma cell lines (JIM3 and U266) were transduced (in triplicate) with a whole genome lentiviral expression library comprised of 17,255 open reading frames (ORFs) representing 12,728 genes. Cells selected in puromycin were cultured and stained using a fluorescently labeled antibody against H2AX, a marker of DNA breaks. Fluorescence-activated cell sorting was used to isolate cells with increased H2AX expression representing presence of damaged DNA; and those cells were sequenced to identify the ORFs associated with increased DNA breaks. Selected genes were further investigated in gain as well as loss of function studies.
 Results: Genome wide ORF expression screen identified, in JIM3 and U266 MM cell lines, 789 and 302 genes associated with increased DNA breaks, respectively. We further analyzed 226 genes associated with increased DNA breaks that are common in both MM cell lines by functional enrichment analysis and report 6 distinct groups. These included 1) cell cycle cyclin-dependent kinases and phosphatases; 2) serine/threonine-protein kinases including calcium/calmodulin-dependent protein kinases; 3) DNA repair and recombination proteins; 4) cytoskeleton, chromosome maintenance and mitosis/cytokinesis related proteins; 5) monooxygenases involved in metabolic detoxification of xenobiotics; and 6) RNA helicases, and proteins involved in RNA processing, splicing and degradation. We next utilized 2 large datasets to evaluate clinical impact of high expression of these genes on patient outcome in MM. Elevated expression of 20 and 23 of these genes in MM patients conferred poor overall (OS) and event free (EFS) survival, respectively (GSE24080; n=559). In second MM dataset elevated expression of 14 and 8 of these genes correlated with poor OS and EFS, respectively (IFM70; n=170). Overall, the expression of three genes correlated with both the PFS and OS in both datasets. These included; 1) DTYMK; deoxythymidylate kinase; a kinase, regulating nucleotide biosynthesis and contributes to DNA replication and repair; 2) CDC25A; cell division cycle 25A, a phosphatase involved in G1/S progression and whose degradation in response to DNA damage is essential to prevent cells with chromosomal abnormalities from progressing through cell division; and 3) EXO1; Exonuclease 1, a nuclease with roles in DNA repair and replication.
 We further functionally validated the role of these select genes for their impact on genomic instability. Overexpression of all three genes (DTYMK, CDC25A and EXO1) in non-cancerous bone marrow stromal (HS5) and/or cancer cells led to increase in genomic instability, as assessed by micronucleus assay, by ∼7-fold, 3.6-fold and > 2-fold respectively. For EXO1 and two other genes also identified in the screen (HDAC8 and TPX2) we have confirmed their role in causing spontaneous and chemotherapy-induced DNA breaks, acquisition of newer genomic changes overtime as well as proliferation of MM and/or other cancer cells. Validation of all identified hits in DNA damage and assays to measure genomic instability is currently in progress.
 Conclusion: We report a novel genome wide high throughput screen that can uniquely utilize protein staining properties and FACS to assess impact of individual gene manipulation at functional and phenotypic level. Using this screen, we have identified genes and pathways contributing to increased DNA breaks in MM and their function is further validated in gain and loss-of function studies in spontaneous and chemotherapy-induced DNA breaks, and genomic instability in myeloma cells. These genes have potential to serve as promising new targets to make MM cells genomically static and the functional screen provides a tool to investigate impact of genome wide perturbations on functional and phenotypic changes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2d1194cb644b42ff02bfccb0592ee04f93e978" target='_blank'>
              Deciphering Drivers of Functional and Phenotypic Changes through Genome-Wide Screen (DRIVE-Genome) to Identify Genes Inducing Genomic Instability in Myeloma
              </a>
            </td>
          <td>
            S. Talluri, Leutz Buon, Jiangning Zhao, Daniel Meglino, C. Chakraborty, Chengcheng Liao, M. Samur, David E. Root, M. Shammas, Nikhil C Munshi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="One of the most extensive forms of natural genome editing occurs in ciliates, a group of microbial eukaryotes. Ciliate germline and somatic genomes are contained in distinct nuclei within the same cell. During the massive reorganization process of somatic genome development, ciliates eliminate tens of thousands of DNA sequences from a germline genome copy. Recently, we showed that the chromatin remodeler ISWI1 is required for somatic genome development in the ciliate Paramecium tetraurelia Here, we describe two high similarity paralogous proteins, ICOPa and ICOPb, essential for their genome editing. ICOPa and ICOPb are highly divergent from known proteins; the only domain detected showed distant homology to the WSD (WHIM2+WHIM3) motif. We show that both ICOPa and ICOPb interact with the chromatin remodeler ISWI1. Upon ICOP knockdown, changes in alternative DNA excision boundaries and nucleosome densities are similar to those observed for ISWI1 knockdown. We thus propose that a complex comprising ISWI1 and either or both ICOPa and ICOPb are needed for Paramecium's precise genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab87b9b92cd1938007d275a747e29fba3c93f620" target='_blank'>
              ISWI1 complex proteins facilitate developmental genome editing in Paramecium.
              </a>
            </td>
          <td>
            Aditi Singh, Lilia Häußermann, Christiane Emmerich, Emily Nischwitz, Brandon K. B. Seah, Falk Butter, M. Nowacki, E. Swart
          </td>
          <td>2024-11-14</td>
          <td>Genome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="DNA double-strand break (DSB) repair pathways are crucial for maintaining genome stability and cell viability. However, these pathways can mistakenly recognize chromosome ends as DNA breaks, leading to adverse outcomes such as telomere fusions and malignant transformation. The shelterin complex protects telomeres from activation of DNA repair pathways by inhibiting nonhomologous end joining (NHEJ), homologous recombination (HR), and microhomology-mediated end joining (MMEJ). The focus of this paper is on MMEJ, an error-prone DSB repair pathway characterized by short insertions and deletions flanked by sequence homology. MMEJ is critical in mediating telomere fusions in cells lacking the shelterin complex and at critically short telomeres. Furthermore, studies suggest that MMEJ is the preferred pathway for repairing intratelomeric DSBs and facilitates escape from telomere crisis. Targeting MMEJ to prevent telomere fusions in hematologic malignancies is of potential therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ae75e964c3e7cfe4dbb822826b27c518fe45ac" target='_blank'>
              The Role of Microhomology-Mediated End Joining (MMEJ) at Dysfunctional Telomeres.
              </a>
            </td>
          <td>
            David Billing, Agnel Sfeir
          </td>
          <td>2024-11-05</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="This review explores articles concerning the experimental research cycle on genome instability in cell populations of highly malignant recurrent organotropic rhabdomyosarcoma RA- 2 in rats. Clonal analysis and cloning were pivotal components of this research, which relies on the frequency of cells with micronuclei and internuclear bridges to gauge the intensity of chromothripsis and break-fusion-bridge cycles. The efficacy of cloning, determined by these indicators, stemmed from the deliberate isolation of tumor stem cells, yielding clones in which chromothripsis activity and breakage-fusion-bridge cycles were sustained. Notably, it is plausible that the stem cells themselves, progenitors of these clones, harbor dicentric chromosomes and chromosomal fragments, contributing to the formation of "fatal micronuclei" in their karyotype. Cloning based on bridges and micronuclei has proven effective up to a certain threshold (15%-18%), reaffirming the predicted reproductive extinction of malignant cell populations under mutational pressure and genome chaos, as posited by the genetic theory of cell populations. Furthermore, this review highlights the potential of ergodic cancer therapy as a novel therapeutic strategy. Ergodic therapy offers promising prospects for late-stage and terminal malignant tumors, where conventional treatments may fall short due to advanced progression. Furthermore, by "enhancing chromothripsis" through the induction of additional micronuclei and bridges, ergodic cancer therapy seeks to increase genome chaos to a critical threshold, potentially halting malignant progression. This innovative approach presents opportunities to explore new drugs and targets for chromothripsis-based oncotherapy, addressing the pressing need for effective treatments in advanced stages of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ee83cc51235034563b6307f2bcb528ce64b269" target='_blank'>
              Perspectives on Ergodic Cancer Therapy Derived from Cloning Genome Chaos via In Vivo Rhabdomyosarcoma RA-2 Models: a Narrative Review.
              </a>
            </td>
          <td>
            Sergey Shityakov, Natalia Lubinets, Viacheslav Kravtsov
          </td>
          <td>2024-11-08</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Aneuploidy, characterized by imbalanced chromosome numbers, is a hallmark of cancer and is strongly correlated with lethal progression in prostate cancer. Our recent work identified that gains of chromosome 8q, the most frequently gained chromosome in aneuploidy prostate cancers, drive cancer progression, partially through the cohesin RAD21 gene located on the chr8q24 region. We demonstrated that increased RAD21 expression accelerates lethal progression and oncogenesis in both prostate cancer and Ewing sarcoma by alleviating oncogenic stress during early-stage oncogenesis in both cell line and organoid models. Therefore, we hypothesize that elevated RAD21 levels could enable cancer cells to evade immune surveillance by mitigating oncogenic stress and DNA damage. In this study, we developed isogenic prostate and breast cancer cell culture models with varying levels of RAD21. Using a natural killer (NK) cell co-culture assay, we observed that cells with elevated RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited increased survivorship when co-cultured with NK cells. These findings align with publicly available data showing that tumors with high RAD21 levels have reduced NK cell infiltration in prostate cancer. Our results suggest that increased RAD21 expression, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Understanding the role of RAD21 in immune evasion could reveal novel therapeutic strategies targeting immune surveillance mechanisms in prostate cancer.
 Citation Format: Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, Andrew Elliott, Nicholas A. Zorko, Justin H. Hwang, Xiaofeng A. Su. Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e04861d70a2d0fe00d7781f4f4827f407b579d8" target='_blank'>
              Abstract B021: Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers
              </a>
            </td>
          <td>
            Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, A. Elliott, Nicholas A Zorko, Justin H. Hwang
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Rearrangements between genes can yield neomorphic fusions that drive oncogenesis. Fusion oncogenes are made up of fractional segments of the partner genes that comprise them, with each partner potentially contributing some of its own function to the nascent fusion oncoprotein. Clinically, fusion oncoproteins driving one diagnostic entity are typically clustered into a single molecular subset and are often treated a similar fashion. However, knowledge of where specific fusion breakpoints occur in partner genes, and the resulting retention of functional domains in the fusion, is an important determinant of fusion oncoprotein activity and may differ between patients. This study investigates these phenomena through the example of capicua (CIC)::double homeobox 4 (DUX4), a fusion between transcriptional repressor CIC and DUX4 genes, which drives an aggressive subset of undifferentiated round cell sarcoma. Using a harmonized dataset of more than 100 patient fusion breakpoints from the literature, we show that most bona fide CIC::DUX4 fusions retain the C1 domain, which is known to contribute to DNA binding by wild-type CIC. Mechanistically, deletion or mutation of the C1 domain reduces, but does not eliminate, the activation of CIC target genes by CIC::DUX4. We also find that expression of C1-deleted CIC::DUX4 is capable of exerting intermediate transformation-related phenotypes compared with those imparted by full-length CIC::DUX4 but was not sufficient for tumorigenesis in a subcutaneous mouse model. In summary, our results suggest a supercharging role for the C1 domain in the activity of CIC::DUX4. Significance: We show in mammalian settings that the capicua C1 functional domain is a supercharger for CIC::DUX4, a poorly studied fusion oncoprotein which drives a rare sarcoma with dismal outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33a18cf240663d7acde75f57d78717f3cd1075" target='_blank'>
              The Capicua C1 Domain Is Required for Full Activity of the CIC::DUX4 Fusion Oncoprotein
              </a>
            </td>
          <td>
            Cuyler Luck, Kyle A. Jacobs, Ross A. Okimoto
          </td>
          <td>2024-11-12</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA repair dysregulation is a key driver of cancer development. Understanding the molecular mechanisms underlying DNA repair dysregulation in cancer cells is crucial for cancer development and therapies. Here, we report that enhancer of zeste homolog 2 (EZH2) directly methylates poly(adenosine diphosphate–ribose) polymerase-1 (PARP-1), an essential enzyme involved in DNA repair, and regulates its activity. Functionally, EZH2-catalyzed methylation represses PARP1 catalytic activity, down-regulates the recruitment of x-ray repair cross-complementing group-1 to DNA lesions and its associated DNA damage repair; on the other hand, it protects the cells from nicotinamide adenine dinucleotide overconsumption upon DNA damage formation. Meanwhile, EZH2-mediated methylation regulates PARP1 transcriptional and oncogenic activity, at least in part, through impairing PARP1-E2F1 interaction and E2F1 transcription factor activity. EZH2 and PARP1 inhibitors synergistically suppress prostate cancer growth. Collectively, our findings uncover an insight of EZH2 functions in fine-tuning PARP1 activity during DNA damage repair and cancer progression, which provides a rationale for combinational targeting EZH2 and PARP1 in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e5d1675eeb186efe28f57213417662345e8e12" target='_blank'>
              EZH2 directly methylates PARP1 and regulates its activity in cancer
              </a>
            </td>
          <td>
            Qingshu Meng, Jiangchuan Shen, Yanan Ren, Qi Liu, Rui Wang, Qiaqia Li, Weihua Jiang, Quan Wang, Yixiang Zhang, Jonathan C Trinidad, Xiaotong Lu, Ting-You Wang, Yanqian Li, Chaehyun Yum, Yang Yi, Yongyong Yang, Dongyu Zhao, C. Harris, S. Kalantry, Kaifu Chen, Rendong Yang, Hengyao Niu, Qi Cao
          </td>
          <td>2024-11-27</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a uniformly lethal primary intracranial neoplasm that inevitably recurs despite treatment by DNA damaging agents radiation (RT) and temozolomide (TMZ). Recurrence may be driven by inherently resistant glioma stem-like cells (GSCs). GSCs have been shown to have constitutive upregulation of DNA damage response (DDR) pathways, resulting from elevated DNA replication stress. This baseline elevated DDR has consequently been shown to confer inherent resistance to DNA damaging agents, TMZ/RT. However, factors mediating elevated replication stress and upregulated DDR in GSCs are poorly defined. Previously, we identified the cell-surface protein, THY1, as a mediator of GSC phenotype and treatment resistance in GBM. To determine mechanisms mediating treatment resistance in THY1+ cells, we engineered patient-derived neurosphere and organoid models, to either overexpress or silence THY1 expression. These models were treated with RT and collected at various time points to measure DNA repair efficiency by assessing the formation and resolution of yH2AX foci and ATM/ATR activation. In parallel, patient-derived GBM tissue from over 20 patients was analyzed using single cell RNA sequencing and spatial transcriptomics to assess pathways associated with THY1 expression. Interestingly, spatial transcriptomics and single cell RNA sequencing data demonstrates significant upregulation of GSC pathways, as well as constitutive upregulation of DDR pathways (p<2e-16) in THY1-high cells. Exploiting our patient-derived lines we determined that DNA repair activation was upregulated at baseline, measured using yH2AX foci and immunoblotting. We then measured DDR in response to radiation and found that high THY1 expression mediated rapid repair of DNA. Overall, we demonstrate that THY1 mediates therapy resistance by constitutively upregulating DDR, likely by upregulating DNA replication stress. We are now working to determine the mechanisms mediating the GSC phenotype and DNA replication stress in THY1 cells in hopes of future translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdfb60c03e046f43e9a607b25a101b9359a6482" target='_blank'>
              DNAR-08. THY1 MEDIATES TREATMENT RESISTANCE IN GLIOBLASTOMA THROUGH RAPID REPAIR OF DNA DAMAGE
              </a>
            </td>
          <td>
            Safiulla Basha Syed, Arushi Tripathy, V. Ravikumar, Sahezeel Awadia, Sravya Palavalasa, Dah-Luen Huang, Arvind Rao, Daniel R. Wahl, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="

 Introduction: Optical genome mapping (OGM) is a novel DNA-based technology for comprehensive genome-wide analysis of cytogenomic abnormalities. Multiple Myeloma (MM) is characterized by recurrent cytogenetic abnormalities including copy number variants (CNVs) such as hyperdiploidy, gain of 1q, loss of 1p, del(17p), del(13q); as well as structural variants (SVs) such as chromosomal rearrangements involving IgH. This study assessed the use of OGM compared to fluorescence in situ hybridization (FISH) and karyotyping in clinical MM cases.
 Methods: Bone marrow aspirate (BMA) samples with >20% plasma cells from MM patients (pts) at a single institute (March 2023 - April 2024) were collected, and subjected to OGM, karyotyping, and FISH analysis for rearrangements of IGH/CCND1, IGH/FGFR3, IGH/MAF, MYC, and numerical changes of CDKN2C/CKS1B, CDKN2A/CEP9, RB1/13q34, and TP53/CEP17. Cytogenomic findings detected by OGM were classified as pathogenic/likely pathogenic, or uncertain clinical significance. Clinical data including staging, extramedullary disease and Line of therapy (LOT) were also collected.
 Results: The study cohort included newly diagnosed MM (NDMM, n=10) and relapsed refractory MM (RRMM, n=15). OGM detected all cytogenomic aberrations which were reported by karyotyping and FISH. Furthermore, OGM identified additional cytogenomic abnormalities in 15 (60%) of patients that were not identified by FISH and karyotyping. OGM identified 3 or more CNVs in 7 pts and SVs in 8 pts, and uncharacterized cytogenetic aberrations in 15 patients. Notably del(17p) was detected in 2 pts by FISH but not by OGM, likely due to the small clonal size below OGM's detection limit (20%). MYC rearrangement (eg MYC::IGH) was detected by both FISH and OGM, however with OGM identifying additional MYC partner genes (IGL, PECAM1, NBEA and TENT5C). Furthermore several abnormalities involving novel genes were exclusively detected by OGM only: PTK2, PVT1, and ARIH2.
 Chromoanagenesis was detected only by OGM in 6 pts (n=1 for NDMM; n=5 for RRMM). The NDMM pt with chromoanagenesis achieved only a partial response after induction therapy. RRMM pts with chromoanagenesis demonstrated functionally aggressive clinical features, all required intensive chemotherapy regimens such as hypercytoxan, hyperCVAD, and also novel bispecific antibodies. Chromoanagenesis can present either independently or in combination with known high-risk FISH features.
 Conclusion: OGM, as a novel cytogenomic technique, offers broader genomic coverage compared to current FISH probe sets (with limited targets) and much higher genomic resolution and sensitivity compared to karyotyping. OGM may complement classical cytogenetics for the diagnosis and risk stratification in MM. OGM's ability to detect chromoanagenesis is promising, despite its current limitation in detection sensitivity which could be improved by cell sorting techniques. Chromoanagenesis identified by OGM can potentially serve as a biomarker for complex chromosomal rearrangement and genomic instability, and also for strategizing the subsequent LOTs for more favorable clinical responses. Future prospective studies are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7985e630f2f081e5f31e633527a8662575a79c7c" target='_blank'>
              Unveiling Clinical Potential: Exploring Cytogenomic Aberrations through Optical Genomic Mapping in Multiple Myeloma
              </a>
            </td>
          <td>
            J. C. Ye, Guilin Tang, O. Pasvolsky, M. Becnel, Mahmoud R. Gaballa, K. Patel, Hans C. Lee, S. Thomas, D. Weber, P. Lin, R. Orlowski
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="

 Introduction
 Ionizing radiotherapy (RT) is a mainstay in the local control of lymphoid malignancies. It introduces DNA double-strand breaks (DSB) that result in cell death unless repaired prior to cell division. Resulting mutations are measurable as mutational signatures with whole genome sequencing (WGS). Importantly, detection of a therapy-induced signature in bulk WGS is predicated by the expansion of a single cell bearing therapy-associated mutations that sweeps to clonal dominance. This concept can be leveraged to track cancer evolution relative to treatment (Diamond et al. Blood 2023). The influence of RT, which has been linked with the ID8 indel signature, on patterns of relapse remains understudied, in part, due to the difficulties in isolating RT-induced lesions from those induced by genotoxic chemotherapies, which are often co-administered.
 Methods
 We reworked mmsig for indel signatures (github.com/UM-Myeloma-Genomics) and used 80x WGS to characterize the indels of 514 total WGS samples including 56 relapsed/refractory (RR) and 100 newly diagnosed (ND) large B-cell lymphoma (LBCL) and 266 ND and 98 RR multiple myeloma (MM; 327 patients). Finally, we validated and refined the indel landscape of chemoRT by exposing TP53-mutant lymphoma cells (DHL-4) to cisplatin and RT, then performing 30x bulk WGS of 15 single-cell-derived colonies.
 Results
 Of 514 total WGS, 29 and 16 samples were from MM and LBCL patients exposed to prior RT. Of these 45 cases, 41 (91%) also had prior mutagenic chemo (platinum/melphalan). Indel burden was increased in RR cases compared to ND (LBCL: median 467 vs 1106, p<0.001; MM: 527 vs 591, p<0.001) but there was no difference in indel counts between chemoRT- and chemo-treated cases (p = 0.51). We performed indel and single base substitution (SBS) signatures extraction, decomposition, and fitting for all samples. SBS signatures included platinum- (SBS31, SBS35, E_SBS37) and melphalan-induced (SBS99) mutagenesis in 36/56 (64%) and 71/107 (66%) of exposed samples, respectively. We deconvoluted 7 indel signatures including ID8. ID8 was seen in 18 RR cases (of 154 total, 11.6%) and not in any at baseline. While presence of ID8 was strongly correlated with prior RT (p<0.001), it was also observed in 10 unexposed samples. All of these RT-unexposed cases had evidence of platinum/melphalan mutagenesis.
 As chemo-induced ID8 has not been reported, we performed a single cell expansion experiment wherein DHL-4 cells were exposed daily to platinum or RT. Single cells from days corresponding to a sub-lethal and a half-lethal dose were expanded, and 3 resulting colonies from each time were subjected to 30x WGS (12 colonies). An unexposed expansion was performed as control (3 colonies). RT and platinum expansions had a higher indel burden (mean, 4247) compared to control (mean, 3065; p = 0.004). Where all RT expansions had evidence of ID8 mutagenesis, only sub-lethal-dose platinum harbored ID8, implying a dose threshold must be met, but that DSB from chemotherapy can indeed cause ID8. Indel profiles of RT and sub-lethal-dose platinum colonies had cosine similarity > 0.95.
 To investigate the impact of RT on tumor evolution, we focused on 2 ID8+ cases with RT exposure, but no chemo mutagenesis. One LBCL case had an irradiated neck node prior to CD19 CART. At relapse, 9 months later, an inguinal biopsy was ID8+. One MM case had RT of a clavicular plasmacytoma and a relapse 7 years later with ID8+ iliac crest marrow. Given the requisite expansion of a single cell bearing a unique catalogue of RT-induced indels required for ID8 detection in bulk WGS, this constitutes evidence that, in both cases, a single cell that survived RT seeded systemic relapse (single cell accelerated model; Landau et al. Nat Comm 2020). Finally, ID8-associated deletion motifs were identified in driver genes across these samples including in SOCS1, ACTB, TNFAIP3, and RB1, among others, implicating chemoRT in the potential development of refractoriness.
 Discussion
 ID8 is strongly associated with RT in LBCL and MM. Yet, we show, for the first time, that it can be induced via DSB from chemo. Dissection of therapy-related mutational signatures can dissect the patterns of relapse after each clinical intervention. Here, we see that even a single resistant tumor cell from an RT-exposed lesion can seed systemic relapse. further underscoring the need to eradicate disease in entirety to avoid propagation of resistant subclones.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488eea0c2970adf9a2c557d0b4cd25e4e0c43cbb" target='_blank'>
              Clonal Evolution of Lymphoid Malignancies Following the Mutagenic Impact of Radiotherapy
              </a>
            </td>
          <td>
            B. Diamond, Dhanvantri Chahar, Michael D. Jain, A. Poos, Michael Durante, B. Ziccheddu, Marcella Kaddoura, M. Papadimitriou, K. Maclachlan, T. Jelínek, F. E. Davies, Nicholas B Figura, G. Morgan, E. Mai, Hartmut Goldschmidt, K. Weisel, Roland Fenk, Saad Z Usmani, O. Landgren, Frederick L. Locke, M. Raab, Jonathan H. Schatz, Niels Weinhold, F. Maura
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 The treatment of many solid tumors presents significant challenges with chemotherapy, radiation therapy, and the limited effectiveness of immunotherapy. Targeted therapy offers a promising approach, yet the lack of validated targets limits its applicability across the spectrum of solid tumors. Our past studies have revealed that an R-loop and mRNA-dependent DNA repair (RDDR) pathway, induced by damage at the transcribed regions of the genome, contributes to cell survival and drug resistance in cancer cells. This is particularly due to the high levels of transcription and DNA damage in these cells. We identified the RNA methyltransferase TRDMT1 as the primary modifier of mRNA methyl-5-cytosine in the RDDR pathway. Overcoming technical difficulties in monitoring RNA modifications, we developed the first TRDMT1 inhibitor (TRDMT1i) using in-house built cell-based and in vitro assays. We have identified highly potent and specific lead compounds that significantly reduce TRDMT1 activity at nanomolar concentrations. Our current lead TRDMT1 inhibitors meets most of the criteria that are required for optimal lead. Initial assessments involving the screening of normal and cancer cell lines, xenograft models, and patient specimens indicate that TRDMT1i sensitizes tumors with genomic instability when used as monotherapy. We aim to develop TRDMT1i as an investigational new drug (IND) for future clinical trials, initially targeting ovarian cancer as a monotherapy or in combination therapy for first-line maintenance and systematic therapy. Our goal is to pioneer the discovery of the RDDR pathway, leading to the development of innovative platforms and next-generation medications to revolutionize the targeting of mRNA modifications in cancer treatment.
 Citation Format: Li Lan. Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5c058059942f92471fbbee99fbac2c5e418612" target='_blank'>
              Abstract PR001: Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy
              </a>
            </td>
          <td>
            Li Lan
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77ea6b7a2dc14678d80892bdad28eb8662925eda" target='_blank'>
              Impact of pan-cancer analysis of the exportins family on prognosis, the tumor microenvironment and its potential therapeutic efficacy
              </a>
            </td>
          <td>
            Ying Peng, Youheng Li, Lingmei Wang, Shenglai Lin, Hong Xu
          </td>
          <td>2024-12-21</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elucidating the mechanisms underlying cancer development and proliferation is important for the development of therapeutic methods for the complete cure of cancer. In particular, the identification of diagnostic markers for early detection and new therapeutic strategies for refractory gastrointestinal cancers are needed. Various abnormal phenomena occur in cancer cells, such as functional changes of proteins, led by genomic mutations, and changes in gene expression due to dysregulation of epigenetic regulation. This is no exception for noncoding RNA (ncRNA), which do not encode proteins. Recent reports have revealed that microRNA (miRNA), long noncoding RNA (lncRNA), and circular RNA (circRNA) are deeply involved in cancer progression. These ncRNAs have attracted attention as gene expression regulatory molecules. Recent advances in technology have made it possible not only to read DNA and RNA sequences but also to study the modification state of each base. In particular, comprehensive analysis of N6-methyladenosine (m6A) has been performed by many research groups, with multiple studies reporting that m6A modifications of specific genes are associated with cancer progression. Based on the above, this review examines how ncRNA modifications are related to cancer progression in gastrointestinal cancers such as colorectal and pancreatic cancer. We also discuss enzyme inhibitors that have been reported to have drug discovery potential targeting m6A modifications. By utilizing the new perspective of ncRNA modification, we may be able to accumulate knowledge on the molecular biology of cancer and contribute to human health through diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24e2ef3283a2f1c095d3a859b796d319aaa73ec" target='_blank'>
              Recent advances in noncoding RNA modifications of gastrointestinal cancer.
              </a>
            </td>
          <td>
            T. Hara, Sikun Meng, Yasuko Arao, Yoshiko Saito, Kana Inoue, Sarah Rennie, Ken Ofusa, Y. Doki, H. Eguchi, Toru Kitagawa, Hideshi Ishii
          </td>
          <td>2024-11-01</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="BCR–ABL, the pioneering gene fusion resulting from chromosomal translocation, has marked a major milestone in understanding genetic alterations in cancer. Initially, gene fusions were linked solely to chromosomal rearrangements, serving as diagnostic markers and cancer drivers. However, advancements in high-throughput sequencing and bioinformatics have revealed additional mechanisms underlying gene fusion. The term “gene fusion” primarily refers to fusion events at the DNA level, whereas “chimeric RNA” encompasses a wide range of transcripts containing exons from different parental genes, including gene fusion transcripts. Recent developments have identified numerous chimeric RNAs in various cancer types, extending even to non-cancerous tissues. Chimeric RNAs, originating from events such as trans-splicing, read-through, and intergenic splicing, form a complex landscape with varied functions. While some chimeric RNAs have defined roles and therapeutic potential, a comprehensive understanding of their diverse functions remains a priority. Exploring the full spectrum of chimeric RNA activities is crucial for revealing their clinical and therapeutic implications. In addition, chimeric RNAs are key players in tumorigenesis, affecting cellular processes, and driving cancer progression. Understanding their intricate interactions with cellular pathways is essential for developing targeted therapies and precision medicine approaches. The dynamic nature of chimeric RNAs highlights the need for ongoing research to fully harness their diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0ece1a1fed50713b0199fed280d597f09ffa316" target='_blank'>
              Beyond chromosomal rearrangements: The expanding landscape of gene fusions and chimeric RNAs
              </a>
            </td>
          <td>
            Sangeen Khan, Yue Tang, Zhenguo Song, Xi Chen, Lijun Wang, Chengjuan Zhang, Fujun Qin
          </td>
          <td>2024-10-28</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a36577362842a55eb6cd804fed42f385c47b74" target='_blank'>
              Tissue microarray analyses of the essential DNA repair factors ATM, DNA-PKcs and Ku80 in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            H. Zech, Clara Marie von Bargen, A. Oetting, N. Möckelmann, Christina Möller-Koop, M. Witt, N. Struve, Cordula Petersen, Christian Betz, K. Rothkamm, A. Münscher, T. Clauditz, T. Rieckmann
          </td>
          <td>2024-10-30</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="TAR DNA-binding protein 43 (TDP-43) proteinopathy plays a critical role in neurodegenerative diseases, including amyotrophic lateral sclerosis and frontotemporal dementia (FTD). In our recent discovery, we identified that TDP-43 plays a critical role in DNA double-strand break (DSB) repair via the non-homologous end joining (NHEJ) pathway. Here, we found persistent DNA damage in brains of ALS/FTD patients, primarily in the transcribed regions of the genome. We further investigated the underlying mechanism and found that the activity of polynucleotide kinase 3’-phosphatase (PNKP) was severely impaired in the nuclear extracts of both the patient brains and TDP-43-depleted cells. PNKP is a key player in DSB repair within the transcribed genome, where its 3’-P termini processing activity is crucial for preventing persistent DNA damage and neuronal death. The inactivation of PNKP in ALS/FTD was due to reduced levels of its interacting partner, phosphofructo-2-kinase fructose 2,6 bisphosphatase (PFKFB3), and its biosynthetic product, fructose-2,6-bisphosphate (F2,6BP), an allosteric modulator of glycolysis. Recent work from our group has shown that F2,6BP acts as a positive modulator of PNKP activity in vivo. Notably, exogenous supplementation with F2,6BP restored PNKP activity in both nuclear extracts from ALS/FTD brain samples and in patient-derived induced pluripotent stem (iPS) cells harboring pathological mutations. Our findings underscore the possibility of exploring the therapeutic potential of F2,6BP or its analogs in TDP-43 pathology-associated motor neuron diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb09f31512a9cde9f44a3eb0ee151dbb94ade770" target='_blank'>
              Fructose-2,6-bisphosphate restores TDP-43 pathology-driven genome repair deficiency in motor neuron diseases
              </a>
            </td>
          <td>
            J. Mitra, Anirban Chakraborty, Vikas H. Malojirao, M. Kodavati, S. Mandal, Velmarini Vasquez, Mikita Mankevich, Gourisankar Ghosh, Muralidhar L. Hegde, T. Hazra
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The six subunit Origin Recognition Complex (ORC) loads excess MCM2-7 on chromosomes to promote initiation of DNA replication and is believed to be important for origin specification. Mapping of origins in cancer cell lines engineered to delete three of the subunits, ORC1, ORC2 or ORC5 shows that specific origins are still used and are mostly at the same sites in the genome as in wild type cells. The few hundred origins that were up-regulated in the absence of ORC suggest that GC/TA skewness and simple repeat sequences facilitate, but are not essential for, origin selection in the absence of the six-subunit ORC. Despite the lack of ORC, excess MCM2-7 is still loaded at comparable rates in G1 phase to license reserve origins and is also repeatedly loaded in the same S phase to permit re-replication. Thus, origin specification and excess MCM2-7 loading on origins do not require the six-subunit ORC in human cancer cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20fbdd40cdfd2421e119fa832f03205a7825921a" target='_blank'>
              Specific origin selection and excess functional MCM2-7 loading in ORC-deficient cells
              </a>
            </td>
          <td>
            Y. Shibata, Mihaela Peycheva, Etsuko Shibata, Daniel Malzl, Rushad Pavri, A. Dutta
          </td>
          <td>2024-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs) are a novel class of RNA transcripts, which regulate important cellular functions in health and disease. CircRNAs are covalently joined and characterized by the perturbed arrangement of exons known as back-splicing. Initially regarded as transcriptional byproducts, circRNAs have been shown to regulate mRNA translation by acting as microRNA sponges, and recent studies have revealed their roles in transcription, translation, and various cellular functions. In cancer, circRNAs can function as oncogenes or tumor suppressors, and their stability makes them potential biomarkers for disease. In acute leukemias, circRNAs generated from recurrent chromosomal translocations contribute to leukemogenesis. Here, we investigate the role of circPCMTD1 in chronic myeloid leukemia (CML) in the blast crisis (BC).
 Functional studies using LNA-modified, RNase H-recruiting oligonucleotides (gapmers) targeting circPCMTD1 demonstrated a significant decrease in proliferation and a potent G2/M cell cycle blockade in CML-BC cell lines (K-562 & LAMA-84), both harboring the t(9;22)(q34;q11.2). Quantitative real-time PCR confirmed the specificity of circPCMTD1 depletion without affecting the linear PCMTD1 transcript. CircPCMTD1 knockdown (KD) reduced the viability of leukemic blasts, indicating its essential role in cell survival.
 RNA sequencing after circPCMTD1-KD in K-562 cells identified approximately 150 differentially expressed genes involved in cell cycle control, nuclear organization, and transcriptional regulation, such as SMARCA4, MACM, PCLAF, and SASH1. Gene Set Enrichment analysis highlighted rRNA processing and DNA replication-dependent chromatin function to be notably affected by circPCMTD1 depletion. CyTOF-based cell cycle analysis validated the G2/M blockade. Increased γH2AX levels indicated aberrant DNA damage response, confirmed by western blotting and intracellular flow cytometry. In addition, circPCMTD1-KD led to an increase in the phosphorylation of the CHK1, RPA32, ATR, ATM, and DNA-PK proteins. DNA fiber assays and comet assays further confirmed reduced DNA replication capacity and increased double-stranded DNA breaks upon circPCMTD1 depletion. Taken together, these data underscore the aberrant DNA damage response and the significant increase in genotoxic stress that is triggered by circPCMTD1 depletion.
 We performed targeted circPCMTD1 profiling in CML patients in the chronic, accelerated, and blast crisis phase and found an increased abundance of circPCMTD1 in advanced disease stages, indicating a potential role of higher circPCMTD1 expression in disease progression. In vitro experiments with patient blasts showed that circPCMTD1-KD increased γH2AX levels specifically in BCR::ABL-positive samples. In vivo, targeting circPCMTD1 in mice engrafted with BCR::ABL-positive blasts prolonged survival significantly, with no notable toxicities observed.
 Mechanistically, circPCMTD1 was enriched in the cytoplasm and associated with ribosomes. Polysome profiling suggested its protein-coding capacity, and we identified a cryptic open reading frame within circPCMTD1. Using custom antibodies, we detected a circPCMTD1-derived peptide (~30 KD) localized mainly in the nucleus. Immunoprecipitation followed by mass spectrometry revealed that the peptide interacted with BLM, TOP3A, and RMI1 proteins of the BTR complex. CircPCMTD1 knockdown reduced BTR complex formation. Knockdown of these proteins individually reduced leukemic blast viability, but concomitant depletion mimicked the G2/M blockade seen with circPCMTD1 depletion. Furthermore, treatment with Dasatinib, a tyrosine kinase inhibitor, decreased circPCMTD1-derived peptide levels without affecting the expression levels of the circPCMTD1 transcript and reduced BTR complex formation, linking BCR::ABL activity to circPCMTD1 function.
 In summary, we identify circPCMTD1 as a crucial regulator in BCR::ABL-positive leukemias, affecting DNA damage response, proliferation, and cell cycle progression. Our findings highlight circPCMTD1 as a potential therapeutic target in myeloid malignancies with t(9;22). Future studies should explore the therapeutic implications of targeting circPCMTD1 in combination with existing treatments, potentially offering a novel approach to managing drug resistance and improving outcomes in CML patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f60b79cdd3ac3caea5d248ca34c9af90547920b7" target='_blank'>
              A Protein-Coding Circular RNA That Regulates DNA Damage Response in BCR::ABL-Positive Leukemias
              </a>
            </td>
          <td>
            Amog P. Urs, D. Papaioannou, R. Kulkarni, C. Goda, Rémi Buisson, D. Nicolet, Lauren A. Woodward, K. Mrózek, G. Behbehani, Sakari Kauppinen, Ann-Kathrin Eisfeld, I. Aifantis, Guramrit Singh, A. Dorrance, Ramiro Garzon
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is one of the most devastating childhood cancers with a median survival of < 1year from diagnosis. There is a critical need for new therapeutics for improving treatment outcomes for DMG patients. We performed genome-wide CRISPR/Cas9 screening and identified RAD52 as a potential therapeutic target in DMG cells. RAD52 inhibition, using shRNA-mediated RAD52 depletion and treatment with RAD52 inhibitor, D-I03, suppressed DMG cell proliferation. Western blotting revealed that RAD52 inhibition increased DNA double-starand breaks (DSBs) marker γH2AX while it decreased repair markers BRCA1 and RAD51. Also, RAD52 inhibition causes a sustained level of phosphorylated RAD50 and γH2AX in irradiated DMG cells over 24 hours. Immunocytochemistry (ICC) of γH2AX and repair marker 53BP1 showed that RAD52 inhibition sustained DNA damage with high levels of γH2AX at 24 hours following radiation while the level of 53BPI was decreased, thereby inhibiting DNA DSB repair. DNA repair assay showed that RAD52 inhibition suppressed DNA DSB repair through a homologous recombination pathway. RNA sequencing showed that RAD52 inhibition downregulated genes associated with the DNA repair pathway. Importantly, RAD52 inhibition increased the radiosensitivity of DMG cells. To understand the mechanism of RAD52 inhibition on DNA damage, we performed dot plot assay and ICC of S9.6 which recognizes DNA-RNA hybrids (also known as R-loops) and found accumulation of R-loop formation in DMG cells treated with RAD52 inhibitor. In addition, RAD52 inhibition decreased the expression of RNA polymerase II. Finally, the combination therapy of RAD52 inhibitor and radiation inhibited tumor growth and increased survival of mice bearing DMG patient-derived xenografts, outperforming either monotherapy. Together, our results highlight that RAD52 inhibition induces DNA damage by accumulating R-loops, results in reducing DMG cell proliferation, while also increase DMG radiosensitivty, and providing a rationale for developing combination therapy with radiation for the treatment of DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a1672081dd9284a370714cfd28f2c782a48acc1" target='_blank'>
              DNAR-11. INHIBITION OF RAD52 ACTIVITY INDUCES DNA DAMAGE BY ACCUMULATING R-LOOPS IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Eita Uchida, Yukitomo Ishi, Jun Watanabe, Savneet Kaur, Kentaro Mineji, H. Abdala-Valencia, Benjamin D. Singer, Samantha Gadd, Rintaro Hashizume
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background/Aim Methionine restriction selectively arrests cancer cells during the S-phase of the cell cycle. We hypothesized that DNA damage may occur in S-phase in cancer cells during methionine restriction. To determine if this occurs, we used MiaPaCa-2Tet-On 53BP1-green fluorescent protein (GFP) pancreatic cancer cells, which report GFP fluorescence in real time after DNA-damage response (DDR) in these cells. We also determined whether a chemotherapy drug in combination with methionine restriction increases the rate of DNA damage. Materials and Methods MiaPaCa-2Tet-On 53BP1-GFP cells were used for in vitro experiments. The 25% and 50% inhibitory concentrations (IC25 and IC50, respectively) of recombinant methioninase (rMETase) and paclitaxel on MiaPaCa-2Tet-On 53BP1-GFP pancreatic cancer cells were determined. Cell viability and DDR with rMETase alone, paclitaxel alone, and their combination were measured in MiaPaCa-2Tet-On 53BP1-GFP cells. Results The IC25 of rMETase on MiaPaCa-2Tet-On 53BP1-GFP cells was 1.66 U/ml. The IC25 for paclitaxel on MiaPaCa-2Tet-On 53BP1-GFP cells was 3.31 nM. The combination of rMETase and paclitaxel synergistically reduced the viability of MiaPaCa-2Tet-On 53BP1-GFP cells. The IC50 of paclitacel on MiaPaCa-2Tet-On 53BP1-GFP cells was 5.1 nM. The IC50 of rMETase on MiaPaCa-2Tet-On 53BP1-GFP cells was 2.3 U/ml. The combination of rMETase (IC50) plus paclitaxel (IC50) on MiaPaCa-2Tet-On 53BP1-GFP cells also caused more DNA damage than either agent alone. Conclusion The present study suggests the synergy of methionine restriction and chemotherapy is due, at least in part, to DNA damage of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8068584e9105b6d879c0af48c45959f90d7d2f" target='_blank'>
              Extensive DNA Damage and Loss of Cell Viability Occur Synergistically With the Combination of Recombinant Methioninase and Paclitaxel on Pancreatic Cancer Cells which Report DNA-Damage Response in Real Time
              </a>
            </td>
          <td>
            Sei Morinaga, Qinghong Han, Kohei Mizuta, B. M. Kang, Motokazu Sato, M. Bouvet, N. Yamamoto, Katsuhiro Hayashi, H. Kimura, S. Miwa, K. Igarashi, T. Higuchi, Hiroyuki Tsuchiya, Satoru Demura, Robert M. Hoffman
          </td>
          <td>2024-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract The phosphorylation of histone H2A.X into γH2A.X is a crucial early event in the DNA damage response, marking DNA damage sites and initiating repair processes. While ATM kinase is traditionally recognized as the primary mediator of H2A.X phosphorylation, our study identifies serine/threonine kinase 39 (STK39) as a novel enhancer of this critical signaling pathway. We demonstrate that after DNA damage, STK39 undergoes phosphorylation by the ATM kinase, facilitating its interaction with the Mre11-Rad50-Nbs1 complex and subsequent recruitment to chromatin. This recruitment enables STK39 to further phosphorylate H2A.X, thus amplifying γH2A.X production and promoting homologous recombination repair. Notably, we observe a significant upregulation of STK39 in pancreatic adenocarcinoma (PAAD) tissues, correlating with heightened resistance to PARPi therapy. Furthermore, we demonstrate the synergistic efficacy of combining STK39 inhibition with PARP inhibitors in suppressing and reversing PAAD growth. This study not only provides new insights into the molecular dynamics of H2A.X phosphorylation but also highlights the therapeutic potential of targeting STK39 to enhance PARPi sensitivity in PAAD (created with BioRender).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b5527588649efe249bff697a10bc6cd9dfa473" target='_blank'>
              STK39-mediated amplification of γ-H2A.X promotes homologous recombination and contributes to PARP inhibitor resistance
              </a>
            </td>
          <td>
            Yi Xu, Changying Li, Huan Yin, S. Nowsheen, Xin Xu, Wenjuan Kang, Xin Liu, Lifeng Chen, Zhenkun Lou, Junlin Yi, Min Deng
          </td>
          <td>2024-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Cancer development involves a complex interplay between genetic and epigenetic factors, with emerging evidence highlighting the pivotal role of competitive endogenous RNA (ceRNA) networks in regulating gene expression. However, the influence of ceRNA networks by aberrant DNA methylation remains incompletely understood. In our study, we proposed DMceNet, a computational method to characterize the effects of DNA methylation on ceRNA regulatory mechanisms and apply it across eight prevalent cancers. By integrating methylation and transcriptomic data, we constructed methylation-driven ceRNA networks and identified a dominant role of lncRNAs within these networks in two key ways: (i) 17 cancer-shared differential methylation lncRNAs (DMlncs), including PVT1 and CASC2, form a Common Cancer Network (CCN) affecting key pathways such as the G2/M checkpoint, and (ii) 24 cancer-specific DMlncs construct unique ceRNA networks for each cancer type. For instance, in LUAD and STAD, hypomethylation drives DMlncs like PCAT6 and MINCR, disrupting the Wnt signaling pathway and apoptosis. We further investigated the characteristics of these methylation-driven ceRNA networks at the cellular level, revealing how methylation-driven dysregulation varies across distinct cell populations within the tumor microenvironment. Our findings also demonstrate the prognostic potential of cancer-specific ceRNA relationships, highlighting their relevance in predicting patient survival outcomes. This integrated transcriptomic and epigenomic analysis provides new insights into cancer biology and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7483d8802056f68722b0432bf0cd28325b090cf3" target='_blank'>
              Constructing methylation-driven ceRNA networks unveil tumor heterogeneity and predict patient prognosis.
              </a>
            </td>
          <td>
            Xinyu Li, Chuo Peng, Hongyu Liu, Mingjie Dong, Shujuan Li, Weixin Liang, Xia Li, Jing Bai
          </td>
          <td>2024-11-27</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The remarkable capability of Tardigrade to survive under extreme conditions has been partially attributed to Dsup, an intrinsically disordered, highly positively charged protein. Dsup has been shown to bind to DNA in vitro, a property that has been associated with the capability of Dsup to exhibit stress-protective effects when expressed in mammalian cells. However, DNA binding of Dsup has not been visualized in living cells and expression of Dsup in different cell types was associated with either protective or detrimental effects. In addition, the effect of Dsup expression has not been clearly demonstrated at the organism level. Here we combined molecular dynamics (MD) simulations and fluorescence lifetime imaging microscopy (FLIM)-Förster resonance energy transfer (FRET) to interrogate Dsup-DNA interactions and demonstrated Dsup binding to DNA in living mammalian cells. Furthermore, Dsup expression in both HEK293T cells and yeast enhanced cell survival in the presence of hydrogen peroxide, suggesting that the presence of Dsup allows both mammalian and yeast cells to better cope with oxidative stress conditions. This study provides a better understanding of the property and functional role of Dsup and lays a foundation to explore new approaches to enhance stress resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c6d0a3ae54b0eb2c8b74ea39dd22a5dd18674ac" target='_blank'>
              Tardigrade Dsup: Interactions with DNA and protection of cells from oxidative stress
              </a>
            </td>
          <td>
            Gavin Ni, Hetian Su, Yingqi Zhu, Anshika Dhiman, Huan-Xiang Zhou, Wei Lin, Nan Hao
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) allows cancer cells to metastasize while acquiring resistance to apoptosis and chemotherapeutic agents with significant implications for patients' prognosis and survival. Despite its clinical relevance, the mechanisms initiating EMT during cancer progression remain poorly understood. We demonstrate that DNA damage triggers EMT and that activation of PARP and the PARP-dependent chromatin remodeler ALC1 (CHD1L) was required for this response. Our results suggest that this activation directly facilitates access to the chromatin of EMT transcriptional factors (TFs) which then initiate cell reprogramming. We also show that EMT-TFs bind to the RAD51 promoter to stimulate its expression and to promote DNA repair by homologous recombination (HR). Importantly, a clinically relevant PARP inhibitor reversed or prevented EMT in response to DNA damage while resensitizing tumor cells to other genotoxic agents. Overall, our observations shed light on the intricate relationship between EMT, DNA damage response, and PARP inhibitors, providing potential insights for in cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa6fa387dca2c5c105c906136c5efbe65cd6ac6" target='_blank'>
              DNA damage-induced EMT controlled by the PARP dependent chromatin remodeler ALC1 promotes DNA repair efficiency through RAD51 in tumor cells.
              </a>
            </td>
          <td>
            Fatemeh Rajabi, Rebecca Smith, Win-Yan Liu-Bordes, Michael Schertzer, Sébastien Huet, A. Londoño-Vallejo
          </td>
          <td>2024-11-06</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: Current development in stool- and cell-free DNA (cfDNA)-based technologies have demonstrated promising potentials in colorectal cancer (CRC) diagnosis. However, early detection of CRC and advanced adenoma (AA) remains challenging. Micronuclei (MN) are extranuclear bodies containing chromatin segments resulting from errors in DNA repair. Elevated levels of MN+ erythrocytes have been studied in genotoxicity testing and cancer diagnosis. We further performed whole-genome sequencing on purified and isolated micronuclei DNA (MN-DNA) from erythrocytes in peripheral blood using the approach (WO2021/228246 A1) we previously developed. By comparing MN-DNA from colorectal cancer (CRC) patients and healthy individuals, we identified a series of tumor-associated MN-DNA features (taMN- DNA) with significant differences in read counts and developed a classification model. Here, we established an independent cohort to clinically validate the CRC model and explore its application in early cancer diagnosis. Methods: We launched a prospective observational case- control clinical trial (NCT05875584), which was used as an independent cohort to validate taMN-DNA features in CRC and to perform a comparative analysis with quantitative fecal immunochemical test (qFIT). A total of 598 samples were enrolled in this study (585 was available, including 299 HDs, 206 AAs, and 80 CRCs). We gathered peripheral blood (1-2ml) from a training cohort of 1226 individuals, a test cohort of 309 individuals and this independent cohort of 598 individuals for the MN-DNA isolation from erythrocytes and whole-genome sequencing. Predictive models were developed using distinctive taMN-DNA features identified by genome-wide analysis from training cohort to differentiate between HD and AA or CRC. Results: The predictive CRC model built on taMN-DNA features achieved an AUC of 93% and the detection of AA achieved an AUC of 82% in this clinical validation cohort. Comparing MN-DNA with qFIT, the overall sensitivity was generally consistent (91.3% vs. 87.5%), with MN-DNA achieving 86.4% sensitivity in early-stage (stage I) CRC, significantly higher than 68.2% in qFIT. Additionally, MN-DNA showed superior sensitivity in detecting AA (71.4% vs. 18.4%), with comparable specificity (90.3% vs. 96.3%). Conclusions: Our results demonstrate that MN-DNA within erythrocytes from 1-2ml peripheral blood enable accurate detection of AA and early-stage CRC. Research sponsor: Timing Biotech.
 Citation Format: Yurong Jiao, Xingyun Yao, Haobo Sun, Chengcheng Liu, Xiangxing Kong, Honghao Liang, Fei Meng, Jun Li, Kefeng Ding, Xiaofei Gao. Evaluating the potential of micronuclei DNA from erythrocytes for early detection of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5ef616e28e97fac15d13e7a4fdf0802ddd0c685" target='_blank'>
              Abstract A045: Evaluating the potential of micronuclei DNA from erythrocytes for early detection of colorectal cancer
              </a>
            </td>
          <td>
            Yurong Jiao, Xingyun Yao, Haobo Sun, Chengcheng Liu, Xiangxing Kong, Honghao Liang, Fei Meng, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 H-DNA is an intramolecular DNA triplex formed by homopurine-homopyrimidine mirror repeats. Since its discovery, the field has advanced from characterizing the structure in vitro to discovering its existence and role in vivo. H-DNA interacts with cellular machinery in unique ways, stalling DNA and RNA polymerases and causing genome. The foundational S1 nuclease and chemical probing technologies originally used to show H-DNA formation have been updated and combined with genome-wide sequencing methods for large-scale mapping of secondary structures. There is evidence for triplex H-DNA’s role in polycystic kidney disease, cancer, and numerous repeat expansion diseases. In polycystic kidney disease (PKD), an H-DNA forming repeat region within the PKD1 gene stalls DNA replication and induces fragility. H-DNA- forming repeats in various genes have a role in cancer; the most well-studied examples involve H-DNA-mediated fragility causing translocations in multiple lymphomas. Lastly, H-DNA-forming repeats have been implicated in four repeat expansion diseases: Friedreich's ataxia (FRDA), GAA-FGF14-related ataxia, X-linked Dystonia Parkinsonism (XDP), and cerebellar ataxia, neuropathy and vestibular areflexia syndrome (CANVAS). In this review, we summarize H-DNA’s discovery and characterization, evidence for its existence and function in vivo, and the field's current knowledge on its role in physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d258222649c5f8f00b18b553e105c9ceb00bafd" target='_blank'>
              Triplex H-DNA Structure: The Long and Winding Road from the Discovery to Its Role in Human Disease
              </a>
            </td>
          <td>
            Julia A. Hisey, Chiara Masnovo, S. Mirkin
          </td>
          <td>2024-12-05</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract DNA damage response (DDR) pathway is the coordinated cellular network dealing with the identification, signaling, and repair of DNA damage. It tightly regulates cell cycle progression and promotes DNA repair to minimize DNA damage to daughter cells. Key proteins involved in DDR are frequently mutated/inactivated in human cancers and promote genomic instability, a recognized hallmark of cancer. Besides being an intrinsic property of tumors, DDR also represents a unique therapeutic opportunity. Indeed, inhibition of DDR is expected to delay repair, causing persistent unrepaired breaks, to interfere with cell cycle progression, and to sensitize cancer cells to several DNA‐damaging agents, such as radiotherapy and chemotherapy. In addition, DDR defects in cancer cells have been shown to render these cells more dependent on the remaining pathways, which could be targeted very specifically (synthetic lethal approach). Research over the past two decades has led to the synthesis and testing of hundreds of small inhibitors against key DDR proteins, some of which have shown antitumor activity in human cancers. In parallel, the search for synthetic lethality interaction is broadening the use of DDR inhibitors. In this review, we discuss the state‐of‐art of ataxia‐telangiectasia mutated, ataxia‐telangiectasia‐and‐Rad3‐related protein, checkpoint kinase 1, Wee1 and Polθ inhibitors, highlighting the results obtained in the ongoing clinical trials both in monotherapy and in combination with chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3effb948dd60b6e28e5a1e98f095e523b94a901" target='_blank'>
              Targeting the DNA damage response in cancer
              </a>
            </td>
          <td>
            Guffanti Federica, Chiappa Michela, Damia Giovanna
          </td>
          <td>2024-10-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA is frequently damaged by genotoxic stresses such as ionizing radiation, reactive oxygen species, and nitrogen species. DNA damage is a key contributor to cancer initiation and progression, and thus the precise and timely repair of these harmful lesions is required. Recent studies revealed transcription as a source of genome instability, and transcription-coupled DNA damage has been a focus in cancer research. Impaired mRNA export is closely related to DNA damage through R-loop formation. The molecular machineries of transcription-coupled DNA damage have been extensively analyzed in Saccharomyces cerevisiae. However, the molecular basis of these phenomena in higher eukaryotes remains elusive. In this review, we focus on the relationship between deregulated mRNA export through the transcription-export-2 (TREX-2) complex and cancer development. Particularly, the expression of germinal center-associated nuclear protein (GANP), a molecular scaffold in the TREX-2 complex, is highly associated with tumorigenesis in mice and humans. Although the deregulated expression of other components in the TREX-2 complex might affect cancer development, we have directly demonstrated the significance of GANP in tumorigenesis using genetically modified mice. Additionally, we describe recent evidence for medical applications demonstrating that the downregulation of the other components may be a good candidate for a chemotherapeutic target in terms of reducing the side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e429b3bb884f7cc3d479875de747738fafbdce" target='_blank'>
              Tumorigenesis Caused by Aberrant Expression of GANP, a Central Component in the Mammalian TREX-2 Complex—Lessons from Transcription-Coupled DNA Damages
              </a>
            </td>
          <td>
            Andri Rezano, Naomi Gondo, Yasuhiro Sakai, Yuko Nakamura, S. Phimsen, Tokio Tani, Akihiko Ito, Seiji Okada, Kazuhiko Kuwahara
          </td>
          <td>2024-12-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Hepatocellular carcinoma (HCC) is the most common form of liver cancer and accounts for ∼90% of cases globally. Detection of early-stage HCC is vital as it allows for potentially curative treatment, yet remains a significant challenge. Micronuclei (MN) are a hallmark of genomic instability. An increased frequency of MN is commonly observed in tumor cells as well as other somatic cells, including erythrocytes, in cancer patients. Here, we have established a technique (WO2021/228246 A1), for the isolation and analysis of micronuclei DNA (MN-DNA) in erythrocytes from peripheral blood. Significant changes in read densities at specific genomic locations within MN-DNA were identified in patients, termed as tumor- associated MN-DNA (taMN-DNA) features. This study evaluates the potential of these taMN- DNA features for early-stage HCC detection across human and murine model. Methods: To explore the potential of MN-DNA in cancer detection, we first collected 1-2 mL of whole blood from HDs (N = 53) and HCC patients (N = 53). MN-DNA was isolated and purified from erythrocytes, followed by whole-genome sequencing. Participants were randomly divided into training, test cohorts in an 8:2 ratio. We applied machine learning algorithms to leverage these features for cancer detection in the human model. Additionally, to investigate the formation of taMN-DNA features, we employed a well-established mouse model that mimics HCC development. Mice were injected with the genotoxic agent diethylnitrosamine (DEN) 2 weeks after birth, and developed malignant macroscopic liver nodules at approximately 40 weeks of age. Results: In the human cohort, we enrolled 106 participants, with more than half of the HCC cases were diagnosed at early-stage (stage 0-II). The cancer detection model utilizing taMN- DNA features achieved an overall accuracy of 86.3%, with a sensitivity of 81.8% and specificity of 90.9%. For early-stage, the model demonstrated sensitivities of 54.5% at a specificity of 90.9%. In the DEN-induced HCC mouse model, unsupervised hierarchical clustering based on these selected taMN-DNA features clearly separated WT and tumor mice into two distinct groups. Notably, mice that were not tumorigenic at 36 weeks post-treatment in the DEN-treated group fell into the WT group when classified by the same taMN-DNA features, indicating that the formation of taMN-DNA signatures is associated with the presence of tumors specifically. Conclusions: This pilot study highlights the potential of MN-DNA as a promising tool for early HCC detection. Leveraging these taMN-DNA features can accurately distinguish early-stage HCC patients from HDs with high sensitivity. In both human and murine models, there is a significant relationship between these signatures and tumor presence, suggesting that taMN- DNA features might reflect the diseased state across mammalian species. Research sponsor: Timing Biotech.
 Citation Format: Hui Lin, Haobo Sun, Xingyun Yao, Xiaoxiao Fan, Fei Meng, Honghao Liang, Xiaofei Gao. Leveraging micronuclei DNA from erythrocytes for early detection of hepatocellular carcinoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15221ad2cc8dccf6b314a7408989c4da875e19bd" target='_blank'>
              Abstract B073: Leveraging micronuclei DNA from erythrocytes for early detection of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hui Lin, Haobo Sun, Xingyun Yao, Xiaoxiao Fan, Fei Meng, Honghao Liang, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNA topoisomerases are essential molecular machines that manage DNA topology in the cell and play important roles in DNA replication and transcription. We find that knocking down the enzyme Topoisomerase Top2 or its mammalian homolog Top2b increases the life span of S. cerevisiae, C. elegans, and mice. Top2b reduction also extends the health span of mice and alleviates the pathologies of aging in multiple tissues. At the cellular/molecular level, Top2b reduction attenuates the major hallmarks of aging, such as cellular senescence, de-regulated nutrient sensing, epigenetic alteration, and lysosomal biogenesis. We observed that Top2b reduction significantly changes the epigenetic landscape in various mouse tissues toward those of the young animals, and differentially down-regulates genes with active promoter and high expression. Our observations suggest that Top2 reduction confers longevity effect across species via a conserved mechanism, and may be used as a novel therapeutic strategy for countering aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2158b0d6967f642c67566187862ea04490680967" target='_blank'>
              Reduction of DNA Topoisomerase Top2 reprograms the epigenetic landscape and extends health and life span across species
              </a>
            </td>
          <td>
            Man Zhu, Meng Ma, Lunan Luo, Feiyang Li, Jiashun Zhang, Yan Pan, Lu Yang, Ying Xiao, Ziteng Wang, Bo Xian, Yi Zheng, Hao Li, Jing Yang
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In many organisms, aging is a clear risk factor for increased rates of chromosome mis-segregation, the main source of aneuploidy. Here, we report that old yeast mother cells lose chromosomes by partitioning them asymmetrically to their daughter cells together with the pre-existing (old) Sindle Pole Body (centrosome equivalent in yeast). Strikingly, remodelling of the NPC and the displacement of its nuclear basket triggered these asymmetric segregation events. Concomitantly, basket displacement also caused unspliced pre-mRNAs to leak to the cytoplasm. We show that removing the introns of three genes involved in chromosome segregation was sufficient to fully suppress chromosome loss in old cells. Furthermore, promoting pre-mRNA leakage in young cells also caused asymmetric chromosome partition and loss through the same three introns. Therefore, we propose that basket displacement from NPCs and its ensuing effects on pre-mRNA quality control are a key trigger of aging phenotypes such as aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7e4a2437a944eea036632a9da1081015ec98c9" target='_blank'>
              Dissociation of the nuclear basket triggers chromosome loss in aging yeast
              </a>
            </td>
          <td>
            M. Mirkovic, Jordan McCarthy, A. C. Meinema, J. Parenteau, Sung Sik Lee, Sherif Abou Elela, Yves Barral
          </td>
          <td>2024-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa78a29add9f8f7c58ef77a453eb2141f303497" target='_blank'>
              Differentiation signals induce APOBEC3A expression via GRHL3 in squamous epithelia and squamous cell carcinoma.
              </a>
            </td>
          <td>
            Nicola J Smith, Ian Reddin, Paige Policelli, Sunwoo Oh, Nur Zainal, Emma Howes, Benjamin Jenkins, Ian Tracy, Mark Edmond, Benjamin Sharpe, Damian Amendra, Ke Zheng, Nagayasu Egawa, John Doorbar, Anjali Rao, Sangeetha Mahadevan, Michael A. Carpenter, R. Harris, Simak Ali, Christopher Hanley, Rémi Buisson, Emma King, Gareth J. Thomas, Tim R. Fenton
          </td>
          <td>2024-11-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary In most animals, the genome does not undergo radical changes during ontogenesis. However, in some species, a programmed loss of a portion of the genome has been identified, such as the elimination of entire chromosomes during meiosis in passerine birds; this chromosome is termed the germline-restricted chromosome (GRC). The discovery of the GRC in 1998 in the zebra finch opened new avenues for understanding the phenomenon of DNA elimination. It is now known that the GRC is predominantly maternally inherited, as this chromosome is lost in males at the end of meiosis. In spermatocytes, the GRC univalent forms a distinct chromatin domain at the nuclear periphery. In the present study, immunocytochemistry showed that the components of the proteinaceous scaffold of the prophase I GRC and other chromosomes most likely load asynchronously. This is possibly due to unique aspects of chromatin conformation and transcriptional silencing in the GRC domain, where repressive chromatin marks are present, while transcriptional markers are absent. Nonetheless, some studies indicate gene expression in the GRC of several species. In this study, the molecular machinery of meiotic repair and recombination was found to be functional, as RPA and RAD51 proteins (involved in double-strand break processing) were detected at certain GRC sites. Notably, some RPA foci in the GRC univalent showed telomeric localization. It is in these chromosomal regions that female GRC homologs recombine. The observed meiotic phenomena associated with the GRC make this chromosome unique and a target for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a832ec13ad091b7345447132784aefccb65b8a08" target='_blank'>
              The Germline-Restricted Chromosome of Male Zebra Finches in Meiotic Prophase I: A Proteinaceous Scaffold and Chromatin Modifications
              </a>
            </td>
          <td>
            S. Matveevsky
          </td>
          <td>2024-11-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d596deef9be907cbcf519c92b9380b95ae7639a" target='_blank'>
              PARP1 condensates differentially partition DNA repair proteins and enhance DNA ligation
              </a>
            </td>
          <td>
            Christopher Chin Sang, Gaelen Moore, M. Tereshchenko, Hongshan Zhang, Michael L. Nosella, Morgan Dasovich, T. R. Alderson, Anthony K. L. Leung, Ilya J. Finkelstein, J. Forman-Kay, Hyun O Lee
          </td>
          <td>2024-11-04</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes. The UW/Fred Hutch metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting cell-free DNA (cfDNA) and matched normal tissue, primary tumor, and metastatic tumor samples from advanced BLCA patients, with a focus on these variant subtype tumors. This dataset, consisting of 20 patients with up to five metastases per patient, is a unique opportunity to assess how cfDNA is able to capture heterogeneity across BLCA. We assessed both inter-patient heterogeneity between BLCA subtypes and inter-tumoral heterogeneity within each patient using epigenetic and genomic profiling. Using nucleosome profiling, we identified the activity of lineage markers and transcription factors that distinguish healthy individuals from BLCA patients, and between BLCA subtypes. Furthermore, we conducted a detailed analysis of whole genome (30X) and targeted (4,000X) cfDNA sequencing in their ability to capture the evolutionary history of tumor mutations. Detection of tumor mutations by cfDNA is highly dependent on clonality, with >90% of founder tumor mutations captured by either sequencing method, while capture is decreased for either subclonal mutations or those private to a single metastasis. Importantly, deleterious mutations in BLCA driver genes had close to 100% detection using targeted panel sequencing, even for subclonal mutations. These findings lay the groundwork for the use of cfDNA in clinical BLCA decision-making, including tracking the emergence of pathogenic, targetable mutations and variant subtypes.
 Citation Format: Samantha L Schuster, Pushpa Itagi, Sonali Arora, Patricia C Galipeau, Thomas W Persse, Michael Yang, Allie Kreitman, Alan Min, Funda Vakar-Lopez, John K Lee, Petros Grivas, Robert B Montgomery, Jonathan L Wright, Andrew C Hsieh, Hung- Ming Lam, Gavin Ha. Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb84ec36801f20a2ac043ba0422da74b9249b7" target='_blank'>
              Abstract B040: Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer
              </a>
            </td>
          <td>
            Samantha L Schuster, P. Itagi, Sonali Arora, Patricia Galipeau, Thomas Persse, Michael Yang, Allie Kreitman, Alan Min, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background: Chronic Myeloid Leukemia (CML) is a malignant genetic disorder commonly caused by the translocation of two genes among chromosomes 9 and 22. Nucleophosmin1 (NPM1) is a nuclear protein-coded gene located on chromosome 5q35. NPM1 protein, in association with other tumor suppressor proteins, prevents cell proliferation in an uncontrolled way. Several mutations are located within exon 12 of the NPM1 gene. More than 95% of these mutations are frameshift due to the insertion of tetra‐nucleotides at positions 863 and 864 of exon 12. Three common mutation types (A, B, and D) represent about 90% of NPM1 mutations. Objectives: To analyze the NPM1 gene variants and their association with CML. Methodology: Genomic DNA was extracted from the whole blood of 50 cases and health control subjects. NPM1 (563 bp) was amplified using gene-specific primers by PCR and sequenced using BigDye® Terminator. The mutation was analyzed using Sanger sequencing in an ABI Gene Analyzer (3130XL). The results were analyzed and compared with the reference human NPM1 gene (accession # NG_016018.1). Results: The gene sequencing analysis revealed that NPM1 mutation (type A: Dup TCTG) was not present in DNA-sequenced samples of CML patients. Conclusion: It may be concluded that NPM1 mutations are unlikely to be linked to CML illness in a Pakistani community. Further research with a larger sample size may aid in determining the role of other gene mutations with CML for the effective treatment of this cancer.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc6733160cff5450ae996e96cd43d480700fa23" target='_blank'>
              MUTATION ANALYSIS FOR NUCLEOPHOSMIN-1 (NPM1) GENE VARIANT OF CHRONIC MYELOID LEUKEMIA PATIENTS FROM LAHORE-PAKISTAN
              </a>
            </td>
          <td>
            Sumaira Mehboob, Kokab Durri, Muhammad Farooq Sabar, Hira Muzzamal, Farkhanda Manzoor
          </td>
          <td>2024-11-25</td>
          <td>International Journal of Pharmacy &amp; Integrated Health Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1113f4b65f7bb6543b590129c22b6e32ee76e8" target='_blank'>
              Transposable element 5mC methylation state of blood cells predicts age and disease.
              </a>
            </td>
          <td>
            Francesco Morandini, Jinlong Y Lu, Cheyenne Rechsteiner, A. Shadyab, Ramon Casanova, Beverly M Snively, A. Seluanov, Vera Gorbunova
          </td>
          <td>2024-11-27</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Female gametogenesis in Drosophila requires both differentiation of germline stem cells and the unique cell cycle meiosis. Successful segregation of chromosomes into gametes during meiosis requires the formation of crossovers between homologous chromosomes. The frequency and distribution of crossovers is highly regulated at several levels, including formation of at least one obligate crossover per pair of homologs, crossover interference that keeps crossovers farther apart than expected by chance, and suppression of crossovers in and near the centromere. To achieve crossover patterning, a meiosis specific version of homologous recombination is used to repair programmed double-stranded breaks, and in Drosophila melanogaster, this recombination pathway requires the putative double Holliday junction resolvase MEI-9. Despite our understanding of crossover patterning mechanisms at the phenomenological level, how crossover patterning mechanisms are developmentally controlled in the context of germ cell differentiation is unknown. Here, we take advantage of a hypomorphic mutation in mei-P26, a gene previously associated with regulation of both germ cell mitoses and meiotic induction. We confirm that mei-P261 mutants show extended expression of mitotic cell cycle markers but rarely completely block differentiation, indicating that the mitotic cell cycle is mis-regulated. In this developmentally delayed context, we show that mei-P261 mutants enter meiosis and complete the earliest stages of prophase, including loading of synaptonemal complex proteins onto the centromere and initial loading onto the arms. Most nuclei do not progress past this to make full length synaptonemal complex, yet meiotic double-strand breaks are induced and crossovers form. However, while these crossovers require MEI-9 and exhibit crossover assurance, the centromere effect and interference are lost. We suggest a model where entry into zygotene of prophase is enough to commit pro-oocytes to using the meiotic homologous recombination machinery and thus ensuring crossover assurance, but that full length synaptonemal complex assembly is required for crossover interference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84ace144ba78bce370dee9d1c7b82531d89566c" target='_blank'>
              Analysis of a hypomorphic mei-P26 mutation reveals developmental control of CO patterning mechanisms
              </a>
            </td>
          <td>
            Erica Berent, Ally Solomon, Amanda P. Spencer, Oscar Bautista, Joe Terry, Elizabeth T. Ables, Nicole Crown
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55ad526ec993a486eebae57a301cd30f9bbd46d3" target='_blank'>
              Genome-wide DNA methylation profiling reveals a novel hypermethylated biomarker PRKCB in gastric cancer
              </a>
            </td>
          <td>
            Leyan Li, Xiao Fei, Huan Wang, Sihai Chen, Xinbo Xu, Huajing Ke, Yanan Zhou, Yi Hu, Cong He, Chuan Xie, N. Lu, Jianping Liu, Yin Zhu, Nianshuang Li
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Micronuclei are the extra-nuclear chromatin compartments separated from the primary nucleus and surrounded by their own nuclear envelope. For a long time it has been thought that micronuclei is the final stage of the pathological process in a cell. They have been used as biomarkers of the influence of genotoxic factors as well as of genome instability in various diseases. Nowadays, it is demonstrated that micronuclei could be involved in the cellular activities, affect the nuclear genome and lead to the changes in cell and tissue physiology. It is known that the formation of micronuclei is one of the steps in selective chromatin elimination in the ontogenesis of plant and animal species. The regions to be marked and eliminated from cell nucleus are recognized at the level of genome. This process is often accompanied by modifications with the heterochromatin formation, changes in the chromosome condensation and in the position of chromosomes in the nucleus. The processes observed in selective and non-selective chromatin elimination are similar to a great extent. The fact that the role of micronuclei in the cell functioning is not well-known yet, and the composition of the micronuclei and the ways of chromatin elimination could influence their role in the development of the pathogenesis, emphasizes the importance of additional studies for a more profound investigation of this phenomenon.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeddb8a010fe6c42592e816c3b4480c6ec74d17" target='_blank'>
              The role of micronuclei in chromatin elimination
              </a>
            </td>
          <td>
            Yu. R. Akhmadullina, P. O. Khomenko
          </td>
          <td>2024-11-18</td>
          <td>Žurnal obŝej biologii</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Polyploidization or whole-genome duplication (WGD), followed by genome downsizing is a recurring evolutionary cycle with a significant impact on genome structure. Polyploidy originates from a cell cycle error and in species without a dedicated germline, mitotic as well as meiotic errors can be stably inherited. After the origin, polyploids face immediate challenges, particularly the need to adapt meiotic machinery to prevent improper chromosome pairing and segregation, such as multivalent formation. Over evolutionary timescales, surviving polyploids undergo rediploidization, reorganizing their genomes back to a diploid state. This study focuses on the evolution of a young allotetraploid Arabidopsis suecica, a hybrid species derived from A. thaliana and A. arenosa. Our genomic analyses reveal that A. suecica originated from diploid A. arenosa, contrary to earlier suggestions of a tetraploid origin, indicating that WGD in A. suecica likely occurred somatically rather than being inherited via unreduced gametes. Ecological niche modeling traces the geographic origin of A. suecica to the southeastern edge of the European glacier during the last ice age. We explore how A. suecica adapted to allopolyploidy from diploid ancestors and find evidence of positive selection on meiotic cell cycle genes in the A. arenosa sub-genome. Several of these genes overlap with those involved in autopolyploid adaptation in A. arenosa, but have distinct haplotypes. We also show that purifying selection is relaxed in A. suecica, however, functional compensation between homeologous gene pairs persists, with fewer gene pairs than expected having loss-of-function mutations in both copies. This is consistent with early signs of rediploidization just after ∼16K years since the origin of A. suecica. Our study covers three key aspects of the evolution of allopolyploid A. suecica: origin, meiotic adaptation, and the beginning of rediploidization. Author summary Polyploidization, or whole-genome duplication (WGD), drastically reshapes genomes. Most eukaryotic genomes carry the scars of ancient polyploidy events, reflecting cycles of genome doubling followed by downsizing and eventual return to a diploid state. Here, we examine the cycle of polyploidy in Arabidopsis suecica, a hybrid polyploid (i.e. allopolyploid) of A. thaliana and A. arenosa. Starting at the origin, we reveal that the A. arenosa parent of A. suecica was diploid, not tetraploid as previously hypothesized. Given this, the most probable scenario is that two diploid parents formed a sterile diploid hybrid, which partially restored fertility via somatic WGD. Establishment of the new polyploid species requires further restoration of fertility via meiotic adaptation. We find signatures of positive selection in allopolyploid A. suecica acting on meiosis-related genes. Several of the meiosis-related genes overlap with those involved in meiotic adaptation in autotetraploid A. arenosa, but distinct alleles suggest an independent adaptation. Rediploidization concludes the evolutionary cycle of polyploidy, however, pinpointing the beginning of this process is extremely challenging. We show that despite relaxed purifying selection in A. suecica due to genome-wide genetic redundancy, selection acts to preserve one intact functional gene when the alternate gene is damaged in homeologous gene pairs. Within just 16,000 years, A. suecica appears to be undergoing early stages of rediploidization, with some genes already reverting to a single-copy functional state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa0db07e0a139a9b87065247f114fd356086eaf7" target='_blank'>
              Diploid origins, adaptation to polyploidy, and the beginning of rediploidization in allotetraploid Arabidopsis suecica
              </a>
            </td>
          <td>
            R. Burns, Aboli Kulkarni, Anna Glushkevich, Uliana K Kolesnikova, F. Kolář, A. Scott, P. Novikova
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Down syndrome, or Trisomy 21 (T21), is uniquely associated with transient abnormal myelopoiesis (TAM) and myeloid leukemia of Down syndrome (ML-DS). Both TAM and ML-DS harbor mutations in the erythro-megakaryocytic transcription factor GATA1 that lead to exclusive expression of GATA1s, a truncated isoform lacking the amino terminus. Consistent with the megakaryocytic phenotype and gene signature of TAM and ML-DS, we found that hematopoietic differentiation of human induced pluripotent stem cells (iPSCs) with T21 and GATA1s display absent erythropoiesis but hyperproliferation of megakaryocytes and cytokine independence. Single-cell RNA sequencing of iPSC-derived hematopoietic progenitor cells (HPCs) revealed that T21 and GATA1s drive early commitment to the megakaryocyte lineage and block maturation, recapitulating a leukemic phenotype.
 To investigate the mechanisms underlying these diseases, we combined bulk RNA sequencing, ATAC-seq (assay for transposase-accessible chromatin with sequencing), and CUT&RUN (cleavage under targets and release using nuclease) for GATA1 in T21 HPCs expressing wtGATA1 vs. GATA1s. Multipotent CD41+235+ HPCs were differentiated and flow-purified from isogenic T21/wtGATA1 and T21/GATA1s iPSCs. RNA sequencing of these HPCs identified 1,292 differentially expressed genes (|log2FC| >0, adjusted p <0.05). Gene set enrichment analysis (GSEA) revealed enrichment of cell cycle pathways in T21/GATA1s HPCs, including G2/M checkpoint, E2F targets, and MYC targets. Parallel ATAC-seq identified 19,277 differentially accessible regions (adjusted p <0.05) in T21 HPCs expressing wtGATA1 vs. GATA1s. GSEA of differentially expressed genes with corresponding changes in chromatin accessibility again revealed positive enrichment of G2/M checkpoint genes, E2F targets, and MYC targets as well as negative enrichment of heme metabolism, consistent with the cytokine-independent hyperproliferation of megakaryocytes we observed.
 To determine whether GATA1s chromatin occupancy drives these transcriptional changes, we performed CUT&RUN for GATA1 in CD41+235+ HPCs and CD41+42b+ megakaryocytes differentiated from T21/wtGATA1 and T21/GATA1s iPSCs. In HPCs, GATA1s binding was globally decreased; compared to wtGATA1, 2,340 regions had decreased occupancy whereas only 62 regions had increased binding (false discovery rate ≤0.05). Genes with decreased GATA1s occupancy, chromatin accessibility, and expression in T21/GATA1s HPCs included CCND1 and CDKN1A, both of which inhibit cell cycle progression in G1 phase and play critical roles in the RB pathway. Notably, GATA1 itself plays an important role in this pathway through the GATA1-RB-E2F complex, which represses E2F to control cell proliferation, but GATA1s lacks the motif to form this complex. In stark contrast to the HPCs, GATA1s occupancy was globally increased in T21/GATA1s megakaryocytes (10,299 regions with increased occupancy compared to 279 regions with decreased occupancy; false discovery rate ≤0.05), including at both canonical megakaryocytic and erythroid loci. The essential megakaryocyte gene RUNX1 demonstrated increased chromatin accessibility and GATA1s occupancy, suggesting its role in the megakaryocyte bias and proliferation we observed with GATA1s.
 Taken together, our results suggest that GATA1s in T21 HPCs simultaneously promotes cell proliferation and megakaryopoiesis, yielding hyperproliferative, immature megakaryocytes due to absence of GATA1s binding in HPCs and inappropriate persistence of GATA1s at key loci in megakaryocytes. Our findings recapitulate the hallmarks of malignant transformation and offer mechanistic insights into the progression from T21 to TAM and ML-DS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f408828d9e7e65baf81f6dbf42218d4da5c28b3" target='_blank'>
              Alterations in GATA1s Chromatin Occupancy Lead to Hyperproliferative, Cytokine-Independent Megakaryopoiesis
              </a>
            </td>
          <td>
            K. Takasaki, Kristin Murphy, Eric K. Wafula, Christopher S. Thom, Sara S. Kumar, Ying Ting Sit, Alyssa Gagne, Deborah L. French, Laurie A. Steiner, Stella T. Chou
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer has become the leading cause of death worldwide, and it is the second most common cancer in women and the third most common cancer in men. Accumulating evidence suggests that genetic and epigenetic factors play a key role in the development of colorectal cancer. Cancer Stem Cells (CSC) play an important role in the suppression or development of cancer in various conditions. In recent years, non-coding RNAs (ncRNA) have been the focus, and the association of CSC and non-coding RNA has played a crucial role in the development of human cancers. These non-coding RNAs are known to be expressed in many cancers. Studies have suggested that ncRNAs are dysregulated in colorectal cancer cells, and different factors, like Wnt and Notch, are involved in this dysregulation. ncRNAs play a significant role in cancer initiation, migration, and resistance to therapies. Moreover, long noncoding RNAs are known to regulate tumor suppressor genes or oncogenes. Targeting different ncRNAs like miRNA, circular RNA, long noncoding RNAs, and small interfering RNA may provide efficient, targeted therapeutic strategies for colon cancer treatment. This review aims to briefly discuss the latest findings on the role of noncoding RNAs in the prognosis and diagnosis of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf75cafb36b85765499ce0fa8a4cfb98a12e479" target='_blank'>
              The Role of Noncoding RNAs in the Prognosis and Diagnosis of Colorectal Cancer: An Emerging Biomarker.
              </a>
            </td>
          <td>
            O. S. Vani, Kavitha R Thangaraj, Varshaa Ravichandran, Solomon F D Paul
          </td>
          <td>2024-11-26</td>
          <td>MicroRNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Mantle Cell lymphoma (MCL) is an incurable non-Hodgkin lymphoma (NHL) and has the poorest overall prognoses. Despite intensive therapeutic approaches, the median progression-free survival is 3-5 years. Genetic alterations have been identified in MCL including chromosomal alterations and mutations. However, these changes are not solely responsible for MCL development. Recent studies highlight the biological relevance of human endogenous retroviruses (HERVs), a class of transposable elements (TE), in cancer biogenesis and their potential as novel diagnostic, prognostic, and treatment response biomarkers. However, studies on TE/HERV in lymphoid malignancies are scant and have not been examined in MCL. TEs account for 45% of the human genome, which is broken down into long-interspersed nuclear elements (LINEs), short-interspersed nuclear elements (SINEs), and long-terminal repeats (LTRs). HERV fall into the LTR family and are ancestral remnants of germline retroviral infections, accounting for 8-10% of the human genome.
 To investigate HERV expression in MCL, we performed RNA sequencing (RNA-Seq) in CD19+ B-cells taken from patients with MCL (n=18), and healthy donors (n=10). HERV characterization was conducted by developing an in-house workflow that relied on the telescope, a statistical model to quantify HERVs with a locus-specific resolution. Among the 14,968 HERV genomic loci evaluated in this study, 3345 (22%) were expressed (mean Transcript Per Million (TPM) > 10) across all our MCL samples and, therefore, considered for the differential expression analysis. Such analysis was performed by comparing the MCL samples against the healthy donors. Specifically, a total of 406 and 683 HERVs {(Log2FC) > 0.58 and FDR < 0.05} were found up- and down-regulated in MCL, respectively. From a genomic standpoint, the differentially expressed HERVs were dispersed throughout the genome. However, some chromosomes (chr) such as chr16, chr17, chr19, and chr22, had higher percentages of differentially expressed HERVs calculated by the total number of HERV loci per chromosome. In addition, we identified an increased enrichment of HERV families including HERVL, HERV3, and HERVK in MCL patients vs healthy donors. Among, all the known HERV families, the HERVK family is the most recently integrated into the human genome and the most biologically active in generating retrovirus-like particles. We validated the HERVK RNA-Seq data, by quantitative RT-PCR and confirmed the downregulation of HERVK transcription in 4 MCL cell lines compared with peripheral blood cells from healthy donors. Using RNA-Seq data, we further identified significant downregulation of HERVK subfamilies such as HML2, HML3, and HML-6 in MCL compared to normal control. Mechanistically, HERV transcription is controlled by epigenetic mechanisms including DNA and histone modifications including methylation. We thus interrogated whether DNA methyltransferase inhibitor azacytidine (Aza) or G9a methyltransferase inhibitor (G9ai) controls the HERVK transcription in the MCL cells. We observed that low doses of Aza or G9ai, exhibit variable responses in modulating the transcription of HERVK among the MCL lines. To understand the variability of HERVK upregulation among MCL cells, we examined whether TP53 status affected HERVK expression after in-vitro epigenetic drug treatment. The mutant TP53 MCL cells exhibited significantly less upregulation of HERVK with both epigenetic drugs than the wild-type (wt) TP53 MCL cells. Interestingly, nutlin-3a (MDM2 inhibitor; inhibits MDM2-p53 interaction) treatment significantly increased the HERVK expression in the TP53 wt MCL cells, however, no effect on HERVK expression was seen in TP53 mutant MCL cells. Therapeutically, nutlin-3a was highly sensitive to TP53 wt MCL cells compared to TP53 mutant cells. Combining nutlin-3a with epigenetic drugs Aza or G9ai significantly decreased the proliferation and survival of wt TP53 cells compared to single agents. This is the first study to characterize the expression of HERVs at genomic locus-specific resolution in MCL patients. Our study provides a comprehensive, genome-wide picture of HERV families in MCL along with changes in HERVK after epigenetic treatment and implicates p53 in HERV-K transcriptional regulation. Overall, we provide a novel insight into the biology of HERVs in MCL pathogenesis and epigenetic therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b062a1bdc8a227394bdd63015402f0ed42d991" target='_blank'>
              Transcriptome Analysis at Locus-Specific Resolution Identified Alteration in Human Endogenous Retroviruses, a Class of Transposable Elements in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Alessandro La Ferlita, Nicholas Staudinger, Lara Bayraktar, Katherine Chiappinelli, L. Alinari, Robert A Baiocchi, L. Sehgal, Mamta Gupta
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089aac455387b5d1dba7549c993d3541b59537a0" target='_blank'>
              Circular RNA circPHLPP2 promotes tumor growth and anti-PD-1 resistance through binding ILF3 to regulate IL36γ transcription in colorectal cancer
              </a>
            </td>
          <td>
            Yan Hu, Ze-Rong Cai, Ren-Ze Huang, De-Shen Wang, Huai-qiang Ju, Dong-Liang Chen
          </td>
          <td>2024-12-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Cells release extracellular vesicles (EVs) loaded with functional biomolecules that facilitate intercellular communication in health and disease. These EVs contain DNA (EV-DNA), mirroring the mutational profile of parental cells, yet the mechanism of DNA packaging onto EVs and its role in diseases like cancer remain unclear. Using advanced imaging and biochemical techniques, we discovered that EV-DNA predominantly resides on the vesicle surface, bound to modified histones. Through genome-wide CRISPR knockout screens in cancer cell lines, we identified immune-related pathways and genes crucial for EV-DNA loading, such as APAF1 and NCF1. In several cancer models, loss of APAF1 reduced EV-DNA packaging, and enhancing metastasis, thus implicating EV-DNA in immune surveillance against cancer spread. We demonstrated that EV-DNA uptake by liver macrophages triggers DNA damage response, altering cytokine production and promoting immune activation in pre-metastatic sites. Moreover, quantifying EV-DNA in colorectal cancer biopsies revealed its potential as a predictive metastatic risk biomarker. Our findings shed light on EV-DNA packaging mechanisms and its significance, suggesting that elevated levels of tumor-derived EV-DNA bolster anti-tumor immunity and inhibit metastasis.
 Citation Format: Inbal Wortzel, Han-Sang Kim, David C. Lyden. DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edef13099e4837d7d8441f1fcd7734045284c7c9" target='_blank'>
              Abstract PR014: DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity
              </a>
            </td>
          <td>
            I. Wortzel, H. Kim, David C. Lyden
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancers frequently co-opt lineage-specific transcription factors (TFs) utilized in normal development to sustain proliferation. However, the effects of these TFs on tumor development depend considerably on where in the genome they bind. A new paper by Taylor and colleagues expands on previously developed diamidine compounds that obstruct the DNA binding sites of the pioneer TF PU.1 (SPI1) in acute myeloid leukemia (AML). Immobilization and sequencing of genomic DNA targeted by these compounds revealed that these inhibitors alter the genomic binding patterns of PU.1. The authors report that their strategy constrains the genomic binding preferences of PU.1, leading to redistribution of PU.1 to promoters and other gene-proximal regions with elevated G/C content. Here we discuss recent developments for targeting PU.1 in hematologic malignancies. We also explore the shared functional roles of PU.1 and SWI/SNF ATP-dependent chromatin remodeling complexes, which work together to sustain the enhancer landscape needed for tumor cell proliferation but also have key roles in non-tumor settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241dbf292aad4fcef078a200a7fb9e680896d9df" target='_blank'>
              Pharmacological blockade of a pioneer transcription factor.
              </a>
            </td>
          <td>
            Katerina Cermakova, H. C. Hodges
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Survivin is known for its dual biological role in apoptosis inhibition and mitotic progression. In addition to its being part of the chromosomal passenger complex (CPC), recent findings suggest additional roles for Survivin in the DNA damage response, further contributing to therapy resistance. In this study, we investigated the role of Survivin and the CPC proteins in the cellular response to irradiation with a focus on DNA replication processes. As is known, ionizing radiation leads to an increased expression of Survivin and its accumulation in nuclear foci, which we now know to be specifically localized to centromeric heterochromatin. The depletion of Survivin and Aurora B increases the DNA damage marker γH2AX, indicative of an impaired repair capacity. The presence of Survivin and the CPC in nuclear foci that we already identified during the S phase co-localize with the proliferating cell nuclear antigen (PCNA), further implying a potential role during replication. Indeed, Survivin knockdown reduced replication fork speed as assessed via DNA fiber assays. Mechanistically, we identified a PIP-box motif in INCENP mediating the interaction with PCNA to assist in managing damage-induced replication stress. Survivin depletion forces cells to undergo unphysiological genome replication via mitotic DNA synthesis (MiDAS), resulting in chromosome breaks. Finally, we revealed that Aurora B kinase liberates Pol η by phosphorylating polymerase delta-interacting protein 2 (POLDIP2) to resume the replication of damaged sites via translesion synthesis. In this study, we assigned a direct function to the CPC in the transition from stalled replication forks to translesion synthesis, further emphasizing the ubiquitous overexpression of Survivin particularly in tumors. This study, for the first time, assigns a direct function to the chromosomal passenger complex, CPC, including Survivin, Aurora B kinase, Borealin, and INCENP, in the transition from stalled replication forks (involving PCNA binding) to translesion synthesis (liberating Pol η by phosphorylating POLDIP2), and thus in maintaining genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933e65144a080ac3e72c369a27df2e2dc4c884df" target='_blank'>
              Old Passengers as New Drivers: Chromosomal Passenger Proteins Engage in Translesion Synthesis
              </a>
            </td>
          <td>
            Katharina Falke, E. Schröder, S. Mosel, Cansu N. Yürük, Sophie Feldmann, Désirée Gül, Paul Stahl, Roland H. Stauber, S. Knauer
          </td>
          <td>2024-10-31</td>
          <td>Cells</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The inherent difference between sex chromosomes in males and females drives conflicts in gene expression, leading to adaptations such as Meiotic Sex Chromosome Inactivation (MSCI). In this study, we explore the evolutionary dynamics of MSCI within the Drosophila genus by analyzing transcriptomes across different stages of spermatogenesis in D. melanogaster and its progressively more distant relatives, D. simulans, D. willistoni, and D. mojavensis. Stage-specific bulk RNA sequencing, showing a strong correlation in spermatogenic gene expression patterns among these species, revealed that MSCI dates back to the early evolution of the Drosophila genus, impacting the regulation of both coding and long non-coding RNAs. Notably, for newly evolved genes, X-linked genes show higher expression levels than autosomal genes during mitosis and meiosis, indicating that MSCI predominantly regulates older genes. In contrast, newly evolved autosomal genes exhibit a gradual increase in expression throughout spermatogenesis, reaching their peak in the post-meiotic phase. During this phase, the expression of X-linked new genes decreases, eventually aligning with that of autosomal genes. This expression pattern suggests that haploid selection plays a crucial role in the regulation of new genes, with monoallelic expression of the X chromosome providing an advantage across all stages of germline development, while autosomal gene expression gains a selective edge primarily in the post-meiotic phase. Together, these findings provide new insights into the evolution of sex chromosomes and highlight the critical role of MSCI in shaping gene expression profiles in Drosophila.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324c85d18cb401f6993148f64f1e9de32f4395bf" target='_blank'>
              Meiotic Sex Chromosome Inactivation: conservation across the Drosophila genus
              </a>
            </td>
          <td>
            Camila C. Avelino, Carolina A. Mendonca, Gabriel Goldstein, Carvalho A. Bernardo, M. Vibranovski
          </td>
          <td>2024-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

 Background. Identifying the biologyof high-risk multiple myeloma (MM) is critical to improving outcomes. Current genetic alterations do not faithfully identify high-risk proliferative disease and there are limited data that integrate genetic, epigenetic, and transcriptional information with outcomes. We generated DNA methylation (DNAm) data on 415 MMRF CoMMpass samples and identified distinct epigenetic programs of high-risk disease.
 Methods. Institutional IRB approval was obtained and DNA from CD138+ CoMMpass samples were analyzed by whole genome bisulfite sequencing on Illumina NovoSeq instruments. Reads were aligned to the GRCh38 genome with Bismark and DNAm values were obtained at 16,374,655 CpGs at an average coverage of 24x. WGBS from the BluePrint project were used for comparison to normal plasma cells. CoMMpass whole genome sequencing and RNA-seq were obtained from the Genomic Data Commons and outcome data (IA21) were provided by the MMRF.
 Results. Newly diagnosed MM samples (N=370) exhibited DNA hypomethylation (median 45%, range 23-77%) as compared to normal plasma cells (median 69%), which occurred in late replicating regions of the genome (P=2.8e-135). Dimensionality reduction (t-SNE) of DNAm data clustered samples by subtype, most notably t(4;14) and t(14;16) samples. DNAm also better defined the high-risk Proliferation (PR) subtype as compared to unsupervised RNA analysis. Integration of DNAm and RNA data identified 1,494,744 loci predictive of gene expression (FDR≤0.01). The DNAm loci most tightly correlated with expression included genes highly relevant to MM pathology including cell cycle (e.g. CDKN2C, CCND2), proteasome function (e.g. PSMA8), immune signaling (e.g. CD28, CD27, IL5RA), and immune targets (e.g. GPRC5D).
 Differential DNAm between MM subtypes indicated the high-risk t(4;14) subtype had the most distinct DNAm program with 2,075,489 Differentially Methylated Loci (DML; FDR≤0.01). The majority (92%) of t(4;14) DML had higher levels of DNAm as compared to other subtypes and this was specific to the late replicating regions of the genome (P=3.5e-38). Correlating t(4;14)-specific gene expression and DNAm identified several co-regulated genes including cell adhesion (ITGA7, JAM3) and frizzled (FZD8, FZD2) pathway components.
 The PR subtype had the second most unique DNAm program with 418,474 DML (FDR ≤0.01). In contrast to the t(4;14) subtype, 99.95% of PR DML had less DNAm than other subtypes. PR hypomethylated loci were enriched for motifs of E2F transcription factors, which contain a CpG in their binding motif. Consistent with this, expression of E2F1 and E2F2 were significantly higher in the PR subtype. Analysis of DNAm and E2F1 binding in MM.1S cells showed that E2F1 exclusively bound unmethylated regions of the genome, corroborating the motif analysis of CoMMpass PR subtype DML.
 Conclusion. MM exhibits a dramatically remodeled DNAm program with extensive DNA hypomethylation in late replicating regions of the genome, suggesting this hypomethylation reflects the proliferative history of MM. This phenomenon is consistent with normal plasma cell differentiation and other B cell malignancies. However, the t(4;14) subtype had a distinct DNAm program, which is likely connected to aberrant levels of H3K36me2 catalyzed by NSD2. H3K36me2 is recognized by the PWWP domains of DNA methyltransferases, potentially explaining the higher levels of DNAm in t(4;14). This suggests novel NSD2 inhibitors require time to not only deplete H3K36me2, but also reprogram t(4;14)-specific DNAm. Finally, consistent with DNA hypomethylation reflecting the proliferative history of MM, the PR subtype had the most pronounced DNA hypomethylation. Notably, the PR subtype had reduced DNAm at E2F1 motifs and E2F1 exclusively bound unmethylated regions of the genome, suggesting the DNAm restricts the proliferative program of high-risk MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f454a882da4d31aebb4c4595f4e15c00ee34b62" target='_blank'>
              DNA Methylation Analysis of Commpass Reveals Distinct Epigenetic Programs of High-Risk Disease
              </a>
            </td>
          <td>
            B. Barwick, Robert M Chavez, Doris R. Powell, Yu Zhang, Mark Hamilton, Daniel Auclair, George Mulligan, Karen N Conneely, J. Keats, Paula M. Vertino, L. Boise, S. Lonial
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="
 The majority of the human genome (>70%) contains non-coding DNA consisting of retrotransposable elements (RTE), such as LINE-1, SINE/Alu elements, human endogenous retroviruses (HERV), and satellite repeat DNA (Hsat2 and Hsat3). The expression of these elements is suppressed in normal adult tissues but are abnormally expressed in many cancers. Another key mechanism in cancer initiation and progression is the release of tumor-derived extracellular vesicles (EV) that are long-distance communication vehicles propagating signals from tumors inducing metabolic changes, immune dysregulation and promoting metastases across the body. Our work has uncovered a link between RTE, HERV and Hsat expression in cancer and EV in driving tumor progression and immune dysregulation. W hole transcriptome sequencing of EV derived from several different cancer cell lines and patient samples, including Ewing sarcoma, osteosarcoma, prostate, pancreatic, and breast cancer found that RTE, HERV and Hsat2,3 transcripts are highly enriched in EV versus coding region transcripts and in much higher abundance than in the tumor cell themselves. Tumor-derived EV contained dsRNA, dsDNA and RNA:DNA hybrids from these non-coding regions. Plasma EV isolated from both metastatic and non-metastatic patients also showed an enrichment of these RTE, HERV and Hsat2,3 elements and clearly differentiated healthy, normal donor EV that had little or no detectable signal. Evidence of local dissemination of these non-coding elements in EV was also found in vivo in tumor xenografts that showed transfer of Hsat2,3 transcripts into the murine stroma that naturally lack these satellite repeats. Treatment of fibroblasts and myeloid cells with tumor-derived EV triggered innate immune responses via type I IFN and pro-inflammatory cytokines, including IL-1beta, IL-6, IL-8, and TNFalpha owing to the pathogen-like nucleic acid sequences (viral mimicry) of these elements. These EV-induced chronic inflammatory effects were mainly induced through the cGAS-STING nucleic acid sensing pathway. Evidence of DNA damage in stromal fibroblasts and induced senescence was also found as a result of EV treatment. A key element of RTE is the expression of LINE-1 and HERV derived reverse transcriptase (RT) that propagates the action of RTE via RNA:DNA hybrids, dsDNA and retrotransposition in the genome. In a spontaneous estradiol-driven breast cancer model in ACI rats, long-term treatment with an HIV RT inhibitor (lamivudine/3TC) inhibited breast tumor development associated with reduced RTE activation in the mammary gland and in circulating immune cells as well as reduced chronic systemic innate inflammation. Our results suggest that RTE, HERV and Hsat activation, together with their local and systemic dissemination in EV plays an important role in cancer initiation and progression associated with chronic inflammation. We also suggest that abnormal dissemination of these activated non-coding elements in EV may also play a role in "inflammaging" facilitating not only cancer but also other chronic diseases.
 Citation Format: Laszlo Radvanyi, Valentina Evdokimova, Peter Ruzanov, Zhenbo Zhang, Poul Sorenson, Hendrik Gassmann, Stefan Burdach, Lincoln D. Stein. Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4545e23fd117647dd7aa5ce20199e63c3fd7396" target='_blank'>
              Abstract C012: Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation
              </a>
            </td>
          <td>
            Laszlo Radvanyi, V. Evdokimova, P. Ruzanov, Zhenbo Zhang, Poul Sorenson, H. Gassmann, Stefan Burdach, Lincoln D. Stein
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Introduction: Non-coding regions, constituting a substantial part of the human genome, are crucial for gene regulation. In cancer, somatic non-coding regulatory mutations can significantly alter gene expression by either disrupting or creating transcription factor-binding sites. This phenomenon is notably well-known in diffuse large B-cell lymphoma (DLBCL), where activation-induced cytidine deaminase (AID) misfires during the germinal center (GC) reaction, leading to numerous non-coding mutations that disrupt gene regulation and contribute to malignancy. Classical Hodgkin lymphoma (cHL) is the most prevalent lymphoma among young adults. Although cHL originates from GC B cells, research into AID-associated hypermutation affecting non-coding regulatory regions has not yet been carried out.
 Methods: We investigated somatic mutations in 155 coding genes and thirty-three genomic regions (mostly non-coding) known to be foci of AID-associated hypermutation and contained within regulatory regions of genes involved in GC B cell biology and transformation. ctDNA was analyzed using LyV4.0 ctDNA CAPP-seq in 297 cHL patients from the IOSI-EMA003-NCT03280394 and FIL-RougeBIO-NCT05066555 studies, and in 235 DLBCL for comparison.
 Results: Mutations of in at least one of region of AID-associated hypermutation was detected in 83% of cHL, a prevalence comparable to that observed in DLBCL (88%). The region of AID-associated hypermutation within the BCL6 locus was the most frequently mutated, found in 56% of cHL cases. Other hypermutated regions occurring in >10% of patients mapped in the SOCS1, CD83, RHOH, ST6GAL1, PAX5, BTG1, BCL7A, LRMP and HIST1H1E loci. We then scanned the sequenced non-coding spaces to detect recurrently mutated hotspots. This analysis involved evaluating overlapping 20 bp intervals through a sliding window of one bp while accounting for the nucleotide context. A hotspot located at chr3:187462671-187462703 of BCL6-intron 1 (designated as BCL6 Hotspot 7) was mutated in 30% of cases, a proportion that closely resembled that of DLBCL patients where it is a validated somatic expression quantitative trait locus (seQTL). BCL6 Hotspot 7 aligned with an area of accessible chromatin and heightened H3K27 acetylation in cHL, which was also nominated a super enhancer in 33% cHL cell lines. Moreover, ChIP-seq analysis of BCL6 Hotspot 7 revealed increase in H3K27ac in mutant cHL cell lines that suggested differential binding of transcription factors. BCL6 Hotspot 7 sequence predicted to bind the PRDM1 transcriptional repressor of BCL6, and mutations therein were found to impede BLIMP1 binding in cHL cells. Notably, BCL6 Hotspot 7 mutations were found to co-occur with BCL6 expression in cHL cell lines and HRS cells of primary biopsies, despite the co-expression of PRDM1. BCL6 protein expression ranging from weak to strong was detected in the nucleus of HRS cells of 68% of primary biopsies. The core set of genes that are directly bound and regulated by BCL6 exhibited similar expression levels in GC B cells and in BCL6 expressing cHL cell lines and in primary HRS cells. Furthermore, the core set of genes directly bound and regulated by BCL6 had a nearly identical chromatin accessibility patterns in GC B cells and in BCL6 expressing cHL cell lines. BCL6 protein degradation was observed with BI-3802 in four cHL cell lines expressing BCL6. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was similarly derepressed in cHL cell lines as in the SU-DHL4 DLBCL cell line, indicating that BCL6 is involved in their repression in cHL as in DLBCL. Compared to the BI-5372 control molecule, treatment with BI-3802 significantly decreased proliferation in all cell lines where BCL6 degradation was observed, as well as in the BCL6-dependent SU-DHL4 DLBCL cell line, which was used as a control.
 Conclusion: This study broadens the understanding of known oncogenic mechanisms in cHL development and identifies BCL6 as a vulnerability in a fraction of cHL and as a potential therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04bc769829586d490a6d84a23a5bb0ceea30f412" target='_blank'>
              Non-Coding Mutations of BCL6 in Classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            M. Pirosa, M. Salehi, A. Bruscaggin, L. Terzi di Bergamo, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, Georgia Alice Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Salvatore Annunziata, F. Bergesio, Renzo Boldorini, Eugenio Borsatti, P. Bulian, S. Chauvie, Marco Cuzzocrea, Bernhard Gerber, M. Kurlapski, L. M. Larocca, A. Rinaldi, M. Rodari, Grzegorz Romanowicz, G. Sacchetti, A. Stathis, Georg Stüssi, I. Zangrilli, E. Calabretta, Francesco Corrado, Antonello Pinto, Luca Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, Emanuele Zucca, G. Gaidano, Carmelo Carlo-Stella, A. Tzankov, Luca Ceriani, Davide Rossi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="

 Introduction: The t(4;14) translocation causes overexpression of NSD2 in ~12% of patients with multiple myeloma (MM) and is associated with poor clinical outcomes. NSD2 catalyzes dimethylation of histone H3 lysine 36 (H3K36me2), which is dramatically increased in t(4;14) MM. The molecular mechanisms by which altered H3K36me2 potentiates an oncogenic gene expression program are poorly understood. One function of H3K36me2 is the recruitment of DNA methyltransferase enzymes such as DNMT3B, leading to DNA methylation (DNAm). DNAm plays important roles in both plasma cell differentiation and oncogenesis, yet how DNAm is dysregulated in t(4;14) MM is largely unknown.
 Methods: We analyzed whole genome bisulfite sequencing (WGBS) DNAm data from 415 primary MM samples from the MMRF CoMMpass trial (NCT01454297) which had been integrated with genetic, transcriptional, and clinical information. Differential DNAm analysis between t(4;14) and non-t(4;14) MM samples was performed using the DSS Bioconductor package. KMS11 isogenic cell lines which express either high (NTKO) or low (TKO) levels of NSD2 were profiled for chromatin marks using cleavage under targets and tagmentation (CUT&Tag) and DNAm using WGBS. Three t(4;14) MM cell lines (KMS11, KMS18, H929) were treated with KTX-1031, a catalytic inhibitor of NSD2. Currently, KTX-1001 is a first-in-class catalytic NSD2 inhibitor in a phase 1 clinical trial (NCT05651932). Following 8 days of treatment, chromatin modifications, DNAm, and gene expression were assayed using CUT&Tag, WGBS, and RNAseq. DNMT3B protein levels were assayed by western blot.
 Results: Analysis of CoMMpass samples showed higher levels of global DNAm in the t(4;14) subtype compared to other subtypes (p = 2.9e-8) and >2M differentially methylated loci (DML) (FDR < 0.01), with 92% of DML having higher DNAm in t(4;14) MM samples. A correlation analysis between DNAm and gene expression determined that ~232K DML correlated with RNA levels of the nearest gene (FDR < 0.01). These RNA-correlated DML were enriched near genes identified by Zhan et al. (Blood, 2006) as defining the t(4;14) gene expression subtype (OR = 55.9, p < 1e-16), suggesting that changes in DNAm at these regions may facilitate the t(4;14) MM gene expression program. Epigenetic profiling of KMS11 NTKO (NSD2-high) and TKO (NSD2-low) cells showed higher global DNAm in NTKO cells (p < 1e-9). Furthermore, regions of elevated H3K36me2 in NTKO cells were enriched for increases in DNAm (OR = 1.85, p < 1e-15). Treating t(4;14) MM cell lines with the NSD2 inhibitor KTX-1031 reversed the chromatin changes observed in t(4;14) patients, resulting in reduced H3K36me2 and increased levels of the EZH2 modification H3K27me3. These epigenetic changes corresponded with downregulation of 539 genes and upregulation of 133 genes (FDR < 0.05) in KMS 11 t(4;14) cells. While the NSD2 inhibitor globally depleted H3K36me2 and elevated H3K27me3, these changes were most pronounced proximal to and in the gene bodies of downregulated genes. NSD2 inhibition for 8 days significantly reduced levels of DNMT3B protein, but only slightly reduced global DNAm (~2%). Importantly, the NSD2 inhibitor reversed much of the t(4;14) transcriptional program described by Zhan et al. (FDR = 5.5e-4). Furthermore, gene set enrichment analysis demonstrated that the NSD2 inhibitor resulted in reduced expression of genes involving NFKB (FDR < 0.001) and KRAS (FDR < 0.05) signaling.
 Conclusions: Through analyses of patient and cell line data, we provide evidence that NSD2-mediated increases in H3K36me2 directly dysregulate DNAm in t(4;14) MM, leading to DNAm hypermethylation as compared to non-t(4;14) MM. Many DNAm changes are correlated with transcriptional changes and occur primarily near disease-relevant genes. Treatment with catalytic NSD2 inhibitor for 8 days reverses much of the chromatin and gene expression changes associated with t(4;14) MM. NSD2 inhibition also reduces DNMT3B protein levels after 8 days and may disrupt the t(4;14) DNAm hypermethylation program after more prolonged treatment, resulting in further gene expression changes. We will present these results alongside data from extended NSD2 inhibitor treatments. We posit that elevated NSD2 in t(4;14) MM causes excessive H3K36me2 leading to relative DNA hypermethylation and transcriptional dysregulation which can be reversed through extended pharmacologic inhibition of NSD2.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872f8007ca4b3c7f1153278051a8310364a25bd4" target='_blank'>
              Ectopic NSD2 Remodels H3K36me2 and DNA Methylation to Promote Oncogenic Gene Expression in Multiple Myeloma
              </a>
            </td>
          <td>
            Robert M Chavez, Doris R. Powell, Kiran Lakhani, John Attelah, E. Flynt, Terry Connolly, Mark Hamilton, George Mulligan, Daniel Auclair, J. Keats, Paula M. Vertino, L. Boise, S. Lonial, Karen N Conneely, B. Barwick
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Retrotransposable elements are implicated in genome rearrangements and gene expression alterations that result in various human disorders. In the current study, we sought to investigate the potential effects of long interspersed elements-1 (LINE-1) overexpression on the integrity and methylation of DNA and on the expression of three major pluripotency factors (OCT4, SOX2, NANOG) during the preimplantation stages of human embryo development. Human MI oocytes were matured in vitro to MII and transfected through intracytoplasmic sperm injection (ICSI) either with an EGFP vector carrying a cloned active human LINE-1 retroelement or with the same EGFP vector without insert as control. The occurrence of retrotransposition events was screened by fluorescent microscopy. The in vitro preimplantation development as well as the methylation, pluripotency, and DNA double-strand breaks (DSBs) of the transfected embryos were examined. LINE-1 retrotransposons gave rise to new retrotransposition events in the transfected embryos. LINE-1 injected embryos were characterized by accelerated asymmetrical cell division, multiple cellular fragments, cleavage arrest, and degeneration. Early OCT4 expression remained unaltered, but cleavage arrest and a high fragmentation rate hindered the expression of SOX2/NANOG at the morula stage. Increased DNA DSBs were observed in cleavage-stage blastomeres, while no methylation changes were detected before the cleavage arrest. Our data provide evidence that LINE-1 retrotransposition in human preimplantation embryos may induce DNA DSBs, while at the same time, it appears to interfere with the expression patterns of pluripotency factors. The morphological, structural, and cleavage abnormalities of the transfected embryos show that aberrant retroelement expression may negatively affect human embryo development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c448bfb57d8b4a52f5e6135b0463d8036afab" target='_blank'>
              LINE-1-Induced Retrotransposition Affects Early Preimplantation Embryo DNA Integrity and Pluripotency
              </a>
            </td>
          <td>
            P. Sakaloglou, L. Lazaros, I. Bouba, S. Markoula, Athanasios Zikopoulos, E. Drakaki, Ismini Anagnostaki, Anastasios Potiris, S. Stavros, A. Gerede, Ekaterini Domali, Peter Drakakis, Theodoros Tzavaras, Ioannis Georgiou
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Genomic studies of the malignant brain tumor glioma invariably rely on aligning sequencing reads to a reference genome such as HG38, even though no single reference can capture the diversity of an individual patient or cell line. To begin addressing these biases in genomics research, we sought to construct a chromosome-scale diploid-phased assembly of a normal human astrocyte (NHA) cell line that is frequently used as a basis for developing isogenic glioma cell models. To achieve this, we generated 45x Pacific Biosciences HiFi data (N50: 16kb), 26x Oxford Nanopore Technologies ultra-long data (N50: 80kb), and 2 billion paired-end reads of Hi-C data. The assembler verkko2 was used to integrate these data and build a phased diploid assembly. We achieved an assembly size of 6.01 Gbp, with 17 gapless chromosomes. The two haplotypes contain 214 and 182 contigs, respectively. Our assembly includes 76 fully assembled telomeres and 20 fully assembled centromeres. The contiguity of our assembly is 130 Mbp, which vastly exceeds that of HG38 (N50: 68 Mbp) and approaches that of the recent telomere-to-telomere CHM13 assembly (N50: 155 Mbp). We ran BUSCO on each haplotype of the assembly using the metazoa database to quantify the single-copy genes that are expected to be present in every animal genome. The haplotypes of our assembly contain the complete sequence of 98.6% and 99.9% of these genes respectively. Finally, we used Merqury to perform a k-mer based evaluation of the two haplotypes and found quality values of 59.09 and 59.75 respectively. This diploid genome assembly of NHA will serve as a resource for researchers that wish to develop NHA-derived glioma models. It can be used to call indels or structural variants in a haplotype-specific manner which will further elucidate genotypes underlying glioma phenotypes. Most importantly, it sets the stage for personalized genome assembly and tackling the lack of diversity in genomic studies of glioma to date.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2325a1e30af3626a8d3da00529a91462ed9452ce" target='_blank'>
              EPCO-45. COMPLETE CHROMOSOME-SCALE DIPLOID-PHASED ASSEMBLY OF A NORMAL HUMAN ASTROCYTE CELL LINE
              </a>
            </td>
          <td>
            T. R. Ranallo-Benavidez, Yue Hao, Noelle Fukushima, Ogechukwu Mbegbu, Floris P Barthel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d0309ba86fc8bda71e7e3770fa64ea2730267d" target='_blank'>
              Nuclear IMPDH2 controls the DNA damage response by modulating PARP1 activity
              </a>
            </td>
          <td>
            L. Espinar, Marta Garcia-Cao, Alisa Schmidt, S. Kourtis, Antoni Gañez Zapater, Carla Aranda-Vallejo, R. Ghose, Laura García-López, Ilir Sheraj, N. Pardo-Lorente, Marina Bantula', Laura Pascual-Reguant, Evangelia Darai, M. Guirola, Joan Montero, S. Sdelci
          </td>
          <td>2024-11-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile genomic elements constituting a big fraction of eukaryotic genomes. They ignite an evolutionary arms race with host genomes, which in turn evolve strategies to restrict their activity. Despite being tightly repressed, TEs display precisely regulated expression patterns during specific stages of mammalian development, suggesting potential benefits for the host. Among TEs, the long interspersed nuclear element (LINE-1 or L1) has been found to be active in neurons. This activity prompted extensive research into its possible role in cognition. So far, no specific cause-effect relationship between L1 retrotransposition and brain functions has been conclusively identified. Nevertheless, accumulating evidence suggests that interactions between L1 RNAs and RNA/DNA binding proteins encode specific messages that cells utilize to activate or repress entire transcriptional programs. We summarize recent findings highlighting the activity of L1 RNAs at the non-coding level during early embryonic and brain development. We propose a hypothesis suggesting a mutualistic relationship between L1 mRNAs and the host cell. In this scenario, cells tolerate a certain rate of retrotransposition to leverage the regulatory effects of L1s as non-coding RNAs on potentiating their mitotic potential. In turn, L1s benefit from the cell's proliferative state to increase their chance to mobilize.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc6d372a85f4508f7413dd971b49c4cc08587c1" target='_blank'>
              From the genome's perspective: Bearing somatic retrotransposition to leverage the regulatory potential of L1 RNAs.
              </a>
            </td>
          <td>
            D. Mangoni, Aurora Mazzetti, Federico Ansaloni, Alessandro Simi, Gian Gaetano Tartaglia, Luca Pandolfini, Stefano Gustincich, R. Sanges
          </td>
          <td>2024-11-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Abstract The hyper-unstable Chr9p21 locus, harbouring the interferon gene cluster, oncogenes and C9orf72, is linked to multiple diseases. C9orf72 (GGGGCC)n expansions (C9orf72Exp) are associated with incompletely penetrant amyotrophic lateral sclerosis, frontotemporal dementia and autoimmune disorders. C9orf72Exp patients display hyperactive cGAS-STING-linked interferon immune and DNA damage responses, but the source of immunostimulatory or damaged DNA is unknown. Here, we show C9orf72Exp in pre-symptomatic and amyotrophic lateral sclerosis-frontotemporal dementia patient cells and brains cause the folate-sensitive chromosomal fragile site, FRA9A. FRA9A centers on >33 kb of C9orf72 as highly compacted chromatin embedded in an 8.2 Mb fragility zone spanning 9p21, encompassing 46 genes, making FRA9A one of the largest fragile sites. C9orf72Exp cells show chromosomal instability, heightened global- and Chr9p-enriched sister-chromatid exchanges, truncated-Chr9s, acentric-Chr9s and Chr9-containing micronuclei, providing endogenous sources of damaged and immunostimulatory DNA. Cells from one C9orf72Exp patient contained a highly rearranged FRA9A-expressing Chr9 with Chr9-wide dysregulated gene expression. Somatic C9orf72Exp repeat instability and chromosomal fragility are sensitive to folate deficiency. Age-dependent repeat instability, chromosomal fragility and chromosomal instability can be transferred to CNS and peripheral tissues of transgenic C9orf72Exp mice, implicating C9orf72Exp as the source. Our results highlight unappreciated effects of C9orf72 expansions that trigger vitamin-sensitive chromosome fragility, adding structural variations to the disease-enriched 9p21 locus, and likely elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524754fd5a2c38a8a83f5ecc801de07d52de2482" target='_blank'>
              C9orf72 repeat expansion creates the unstable folate-sensitive fragile site FRA9A
              </a>
            </td>
          <td>
            Mila Mirceta, M. H. Schmidt, Natalie Shum, Tanya Prasolava, Bryanna Meikle, Stella Lanni, Mohiuddin Mohiuddin, P. McKeever, Ming Zhang, Minggao Liang, Ilse M. van der Werf, Stefaan Scheers, P. Dion, Peixiang Wang, Michael D. Wilson, Theresa Abell, Elliot A. Philips, Łukasz J. Sznajder, Maurice S. Swanson, Mustafa Mehkary, Mahreen Khan, Katsuyuki Yokoi, Christine Jung, Pieter J. de Jong, C. Freudenreich, P. McGoldrick, R. Yuen, A. Abrahão, Julia Lee Keith, L. Zinman, Janice Robertson, E. Rogaeva, Guy A Rouleau, R. F. Kooy, Christopher E. Pearson
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="The mechanism underlying the spatiotemporal chromosome organization in Escherichia coli cells remains an open question, though experiments have been able to visually see the evolving chromosome organization in fast- and slow-growing cells. We had proposed [D. Mitra et al., Soft Matter 18, 5615 (2022)1744-683X10.1039/D2SM00734G] that the DNA ring polymer adopts a specific polymer topology as it goes through its cell cycle, which in turn self-organizes the chromosome by entropic forces during slow growth. The fast-growing E. coli cells have four (or more) copies of the replicating DNA, with overlapping rounds of replication going on simultaneously. This makes the spatial segregation and the subsequent organization of the multiple generations of DNA a complex task. Here, we establish that the same simple principles of entropic repulsion between polymer segments which provided an understanding of self-organization of DNA in slow-growth conditions also explains the organization of chromosomes in the much more complex scenario of fast-growth conditions. Repulsion between DNA-polymer segments through entropic mechanisms is harnessed by modifying polymer topology. The ring-polymer topology is modified by introducing crosslinks (emulating the effects of linker proteins) between specific segments. Our simulation reproduces the emergent evolution of the organization of chromosomes as seen in vivo in fluorescent in situ hybridization experiments. Furthermore, we reconcile the mechanism of longitudinal organization of the chromosomes arms in fast-growth conditions by a suitable adaptation of the model. Thus, polymer physics principles, previously used to understand chromosome organization in slow-growing E. coli cells also resolve DNA organization in more complex scenarios with multiple rounds of replication occurring in parallel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77ef47ab0364debc53a506ffe919aa67b66234ab" target='_blank'>
              Topology-mediated organization of Escherichia coli chromosome in fast-growth conditions.
              </a>
            </td>
          <td>
            Shreerang Pande, Debarshi Mitra, A. Chatterji
          </td>
          <td>2024-11-01</td>
          <td>Physical review. E</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zongxiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b14307c5c6357088b57550e695810a4d94ffee" target='_blank'>
              HTS and scRNA-seq revealed that the location and RSS quality of the mammalian TRBV and TRBJ genes impact biased rearrangement
              </a>
            </td>
          <td>
            Yingjie Wu, Fengli Wu, Qingqing Ma, Jun Li, Long Ma, Hou Zhou, Yadong Gong, Xinsheng Yao
          </td>
          <td>2024-10-29</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Cancer is often linked to changes in the regulation of genes, with DNA methylation playing a key role in controlling gene expression and maintaining stability. Many cancers show widespread loss of DNA methylation, particularly in repetitive elements like LINE-1, which can serve as an indicator of overall genomic health. Cell-free DNA (cfDNA), which is present in bodily fluids, offers a non-invasive way to track these changes and could be useful in diagnosing and monitoring cancer. Recent studies suggest that analyzing the methylation of LINE-1 in cfDNA could help predict disease outcomes and evaluate treatments like immunotherapy. This review aims to explore the current research on using LINE-1 methylation in cfDNA as a potential biomarker for cancer detection and therapy monitoring, addressing the need for further investigation into this promising approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/341df9b432e212e6ae303b9a96f7ea2e9a0b4140" target='_blank'>
              LINE-1 cfDNA Methylation as an Emerging Biomarker in Solid Cancers
              </a>
            </td>
          <td>
            U. Gezer, E. Özgür, E. E. Yörüker, Eleni Polatoglou, S. Holdenrieder, A. Bronkhorst
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548909e3fc8f58fb6f1caf3e51e4cd2b9742adb3" target='_blank'>
              Unique structural configuration of EV-DNA primes Kupffer cell-mediated antitumor immunity to prevent metastatic progression.
              </a>
            </td>
          <td>
            I. Wortzel, Yura Seo, Ife Akano, Lee Shaashua, G. Tobias, Jakob Hebert, Kyung-A Kim, DooA Kim, Shani Dror, Yanshen Liu, Griffin Campbell Azrak, Michele Cioffi, Kofi Ennu Johnson, Tammy Hennika, Meshulam Zisha Twerski, Alexis Kushner, Robert Math, Yoon Dae Han, Dai Hoon Han, Minsun Jung, Juyeong Park, Soonmyung Paik, Jeon-Soo Shin, Min Goo Lee, Marco Vincenzo Russo, D. Zakheim, Jesse Barnes, Sunjoy Mehta, Katia Manova, R. Schwartz, B. K. Thakur, Nancy Boudreau, Irina R Matei, Haiying Zhang, Simone Sidoli, Jacqueline Bromberg, Yael David, H. Kim, D. Lyden
          </td>
          <td>2024-12-03</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Efficient DNA replication requires highly coordinated programs for the timely recruitment of protein complexes to DNA replication forks. Defects in this process result in replication stress, which in turn activates cell cycle checkpoints, suppresses cell proliferation and induces apoptosis. In response to persistent cell growth signals that speed up DNA replication processes, cells accelerate the recruitment of DNA replication proteins to avoid DNA replication stress. The mechanisms by which cell growth signals induce processes to facilitate the recruitment of DNA replication proteins onto the replication sites remain unclear. Here, we report that the epidermal growth factor receptor (EGFR) phosphorylates heat shock protein 70 (HSP70) for DNA replication. Such a modification promotes nuclear localization and chromatin association of HSP70, which interacts with the DNA replication coordinator, proliferating cell nuclear antigen (PCNA). HSP70 subsequently facilitates the loading of PCNA onto chromatin. Knockdown or chemical inhibition of HSP70 suppresses PCNA association with chromatin and impairs DNA synthesis and Okazaki fragment maturation, leading to replicative DNA double-strand breaks and apoptosis. Furthermore, chemical inhibition of HSP70 potentiates EGFR-tyrosine kinase inhibitor-induced tumor reduction in vivo. This work expands our understanding of oncogenesis-induced DNA replication processes and provides a foundation for improved treatments for EGFR-mutated lung cancer by simultaneously targeting HSP70.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca2470519b5a8cc8cd3ca52af8ede3fd0298ef0" target='_blank'>
              EGFR-mediated HSP70 phosphorylation facilitates PCNA association with chromatin and DNA replication.
              </a>
            </td>
          <td>
            Yingying Wang, Anthony Fernandez, Xinyu Pei, Bing Liu, Lei Shen, Yao Yan, Hitendra S Solanki, Lin Yang, Mian Zhou, Yuming Guo, Jun Wu, K. Reckamp, Li Zheng, Binghui Shen
          </td>
          <td>2024-10-29</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Adeno-associated viral (AAV) vector-mediated gene therapy has emerged as a promising alternative to liver transplantation for monogenic metabolic hepatic diseases. AAVs are non-integrative vectors that are maintained primarily as episomes in quiescent cells like adult hepatocytes. This quality, while advantageous from a safety perspective due to a decreased risk of insertional mutagenesis, becomes a disadvantage when treating dividing cells, as it inevitably leads to the loss of the therapeutic genome. This is a challenge for the treatment of hereditary liver diseases that manifest in childhood. One potential approach to avoid vector genome loss involves putting scaffold/matrix attachment regions (S/MARs) into the recombinant AAV (rAAV) genome to facilitate its replication together with the cellular genome. We found that the administration of AAVs carrying the human β-interferon S/MAR sequence to neonatal and infant mice resulted in the maintenance of higher levels of viral genomes. However, we also observed that its inclusion at the 3′ end of the mRNA negatively impacted its stability, leading to reduced mRNA and protein levels. This effect can be partially attenuated by incorporating nonsense-mediated decay (NMD)-inhibitory sequences into the S/MAR containing rAAV genome, whose introduction may aid in the development of more efficient and longer-lasting gene therapy rAAV vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c0d46cc95a496eaa7c34f9d64d0726b770f333" target='_blank'>
              In Vivo Selection of S/MAR Sequences to Favour AAV Episomal Maintenance in Dividing Cells
              </a>
            </td>
          <td>
            Andrea Llanos-Ardaiz, Aquilino Lantero, L. Neri, I. Mauleón, M. Ruiz de Galarreta, L. Trigueros-Motos, Nicholas D. Weber, Veronica Ferrer, Rafael Aldabe, Gloria González-Aseguinolaza
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Beta genus human papillomaviruses (β-HPVs) are ubiquitous double stranded DNA (dsDNA) viruses that may promote skin cancers by destabilizing the host genome. Supporting this, expression of the E6 gene from a β-HPV (β-HPV 8 E6) results in increased micronuclei that should induce an innate immune response that eliminates these cells. Yet, β-HPV 8 E6 promotes rather than restricts proliferation. We hypothesize that β-HPV 8 E6 accomplishes this by attenuating the cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS-STING) pathway, an innate immune pathway that becomes activated in response to cytosolic micronuclear dsDNA. Here, we show that in response to stimulation by transfection of pLVX-GFP plasmid, β-HPV 8 E6 reduced the magnitude and intensity of cGAS-STING pathway activation in immunoblot experiments. These data also demonstrate that impairment of the cGAS-STING pathway is strongest downstream of STING phosphorylation. Further, RNA-sequencing suggests that β-HPV 8 E6 downregulates other innate immune pathways. We also show that cGAS is recruited to micronuclei induced by β-HPV 8 E6. These data suggest a mechanism by which β-HPV 8 E6 facilitates proliferation of cells destabilized by micronuclei and support the hypothesis that the prevalence of β-HPV infections is in part due to the impairment of the cGAS-STING innate immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d3fe2a8394ef229dea8d6cc2665539054660d5" target='_blank'>
              Cutaneous human papillomavirus E6 impairs the cGAS-STING pathway
              </a>
            </td>
          <td>
            Emily Tolbert, D. Dacus, Rose Pollina, N. Wallace
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Histone methyltransferases (HMTs) and histone demethylases (HDMs) are critical enzymes that regulate chromatin dynamics and gene expression through the addition and removal of methyl groups on histone proteins. HMTs, such as PRC2 and SETD2, are involved in the trimethylation of histone H3 at lysine 27 and lysine 36, influencing gene silencing and activation. Dysregulation of these enzymes often leads to abnormal gene expression and contributes to tumorigenesis. In contrast, HDMs including KDM7A and KDM2A reverse these methylation marks, and their dysfunction can drive disease progression. In cancer, the aberrant activity of specific HMTs and HDMs can lead to the silencing of tumor suppressor genes or the activation of oncogenes, facilitating tumor progression and resistance to therapy. Conversely, in neurodegenerative diseases, such as Alzheimer’s disease (AD), Parkinson’s disease (PD), and Huntington’s disease (HD), disruptions in histone methylation dynamics are associated with neuronal loss, altered gene expression, and disease progression. We aimed to comprehend the odd activity of HMTs and HDMs and how they contribute to disease pathogenesis, highlighting their potential as therapeutic targets. By advancing our understanding of these epigenetic regulators, this review provides new insights into their roles in cancer and neurodegenerative diseases, offering a foundation for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb434586ac7de7e0c22b593818a08f0a3dc6b5b6" target='_blank'>
              Epigenetic Modifiers: Exploring the Roles of Histone Methyltransferases and Demethylases in Cancer and Neurodegeneration
              </a>
            </td>
          <td>
            Lauren Reed, Janak Abraham, Shay Patel, Shilpa S. Dhar
          </td>
          <td>2024-12-03</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inverted repeats are repetitive elements that can form hairpin and cruciform structures. They are linked to genomic instability, however they also have various biological functions. Their distribution differs markedly across taxonomic groups in the tree of life, and they exhibit high polymorphism due to their inherent genomic instability. Advances in sequencing technologies and declined costs have enabled the generation of an ever-growing number of complete genomes for organisms across taxonomic groups in the tree of life. However, a comprehensive database encompassing inverted repeats across diverse organismal genomes has been lacking. We present InvertiaDB, the first comprehensive database of inverted repeats spanning multiple taxa, featuring repeats identified in the genomes of 118,070 organisms across all major taxonomic groups. The database currently hosts 30,067,666 inverted repeat sequences, serving as a centralized, user-friendly repository to perform searches, interactive visualization, and download existing inverted repeat data for independent analysis. invertiaDB is implemented as a web portal for browsing, analyzing and downloading inverted repeat data. invertiaDB is publicly available at https://invertiadb.netlify.app/homepage.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb8b11bba034f3fdf0fb28bc28f3eae66e67a237" target='_blank'>
              invertiaDB: A Database of Inverted Repeats Across Organismal Genomes
              </a>
            </td>
          <td>
            Kimonas Provatas, Nikol Chantzi, Michail Patsakis, Akshatha Nayak, I. Mouratidis, Georgios A. Pavlopoulos, Ilias Georgakopoulos-Soares
          </td>
          <td>2024-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 Pre-B cell acute lymphoblastic leukemia (ALL) is the most common cancer in children. Although modern treatment protocols cure a majority of patients, relapsed disease remains a leading cause of pediatric death from cancer. Epigenetic dysregulation plays a critical role in ALL therapeutic resistance, with mutations in epigenetic regulators identified at the time of relapse. Epigenetic dysregulation and variability can propel disease evolution by driving leukemia cell heterogeneity and plasticity. To uncover epigenetic drivers of ALL progression, we sought to comprehensively map the DNA methylation landscape in matched diagnosis-relapse ALL specimens using methods that effectively capture epigenetic variability, aiming to identify the regulators and effectors of epigenetic instability in ALL progression. We aimed to construct genome-wide maps encompassing the dynamic DNA methylation landscape in paired diagnosis-relapse samples of a defined genetic subtype of ALL. We carried outwhole genome bisulfite sequencing (WGBS) on 36 samples representing paired diagnosis and relapse specimens from 18 patients with t(12;21) ETV6-RUNX1 pre-B ALL treated on modern Children's Oncology Group protocols. This represents the most common chromosome translocation in pre-B ALL which is typically associated with favorable prognosis, highlighting the importance of identifying epigentic drivers of treatment failure and relapse. We analyzed WGBS data using the informME method, an information-theoretic approach that effectively captures methylation stochasticity and has been previously applied to the identification of non-mutated epigenetically altered driver genes in ALL (Koldobskiy et al., Nature Biomedical Engineering 5: 360-76, 2021). This approach allows the use of WGBS reads to encapsulate epigenetic variability, in order to map genomic regions undergoing evolution from diagnosis to relapse. DNA methylation levels measured by WGBS are used to generate DNA methylation probability distributions, and comparisons of these distributions from diagnosis to relapse are accomplished using an information-theoretic measure of Jensen-Shannon distance (JSD) which measures the distance between two probability distributions and thereby includes stochasticity rather than conventional analyses which only evaluate mean methylation differences. Subsequently, we ranked genomic regions and genes with significant JSD between diagnosis and relapse and tested for functional enrichments. Our analysis revealed significant differences in the DNA methylation landscape between ALL diagnosis and relapse, associated with genome-wide DNA hypomethylation and globally increased methylation entropy in the majority of relapse samples compared to their corresponding diagnosis samples. Focusing on specific genomic regulatory regions, we noted that despite the global trend of genome-wide hypomethylation at relapse, bivalent promoters showed hypermethylation at relapse compared to diagnosis. Mapping methylation discordance revealed a striking enrichment among the target genes of Polycomb Repressive Complex 2 (PRC2), a core epigenetic regulator linked to suppression of gene transcription. Notably, among the 500 genes exhibiting the greatest JSD between diagnosis and relapse, the top four enrichments converged on targets of EED and SUZ12, critical components of PRC2. To further evaluate the role of PRC2 in pre-B ALL, we employed CRISPR-Cas9-mediated genome editing to silence EED, in the B-cell leukemia Reh cell line harboring t(12;21) ETV6-RUNX1 to evaluate the influence of PRC2 modulation on gene expression and the leukemic phenotype. We employed a powerful approach that comprehensively analyzes DNA methylation genome-wide and quantifies epigenetic variability to map evolution of the dynamic epigenetic landscape in a common genetic subtype of ALL from diagnosis to relapse. This approach permits the dissection of gene-regulatory mechanisms underlying leukemic progression. Identification of PRC2 targets as the predominant effectors of epigenetic instability in leukemic progression can inform novel therapeutic strategies, such as approaches aimed at restricting epigenetic variability that contributes to therapeutic resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a83f3fbb398def62708d3edde2848e84e752f91" target='_blank'>
              Mapping the Dynamic DNA Methylation Landscape in Pre-B ALL from Diagnosis to Relapse Reveals a Role for Aberrant PRC2 Regulation
              </a>
            </td>
          <td>
            Athanasia Liapodimitri, Oscar Camacho, Kayleigh Lunsford, Ashley R Tetens, Jordyn Craig-Schwartz, Kegan O. Skalitzky, K. Rabin, Andy Feinberg, Michael A Koldobskiy
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Each cancer is the result of an individually unique evolutionary process. Yet, as demonstrated by extensive genetic analyses of human specimens, the genetic endpoints of cancers across different tissues are remarkably specific, with individual cancer driver genes being typically associated with only a limited set of tumor types. Tissue and cellular contexts thus impose strong and genetically predictable evolutionary constraints on carcinogenesis. Lineage-specific transcription factors, central regulators of organismal development, tissue homeostasis, and regeneration, often also support cancer cell fitness in a lineage-specific manner. In this review, we discuss recent results on the interactions between transcriptional lineage factor programs and oncogenic pathways and how such interactions may determine the oncogenic competence of cancer-associated genetic alterations. These developments are starting to shed light on the molecular basis of tissue specificity in carcinogenesis, with relevance for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f236b52bc63786a8f5d7d9e1f3c0d9457cd40be6" target='_blank'>
              Lineage-Specific Transcription Factors in Carcinogenesis
              </a>
            </td>
          <td>
            Charles J. David, Sakari Vanharanta
          </td>
          <td>2024-10-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent type of liver cancer. Dosage suppressor of NNF1 (DSN1), a component of the MIS12 kinetochore complex, encodes a kinetochore protein crucial for proper mitotic assembly. The role of DSN1 in HCC remains to be elucidated. In this study, we utilized The Cancer Genome Atlas, the Hepatocellular carcinoma Cell Database, and other databases to analyze DSN1 expression and prognosis in samples from patients with HCC. We investigated the signaling pathways regulated by DSN1 and their implications in HCC. Additionally, we engineered siRNAsiRNA/shRNA/shRNA and overexpression vectors for DSN1 and assessed the specific mechanisms of regulatory pathways of DSN1 in hepatoma cell lines and subcutaneous tumor xenograft model. Our findings revealed that DSN1 expression was significantly upregulated in patients with HCC, correlating with decreased survival rates. Elevated DSN1 expression led to the overproduction of cell cycle-related proteins through direct interaction with Centromere Protein T. This interaction contributes to chromosomal instability in patients with HCC, resulting in an aberrant cell cycle and fostering the development and progression of HCC. Increased DSN1 expression is pivotal in HCC initiation and progression. Investigating DSN1 offers valuable insights into the pathogenesis, treatment, and prevention of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162c8061c9e0cfc53558ac673f085aa56ba63f7e" target='_blank'>
              DSN1 Interaction With Centromere-Associated Proteins Promotes Chromosomal Instability in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Hongrui Zhou, Mengxue Zhang, Jiabing Lian, Ruichang Wang, Zhen-Jun Yang, Jin Wang, Xiuli Bi
          </td>
          <td>2024-11-19</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Medulloblastoma (MB) is a common primary brain cancer in children. The sonic hedgehog (SHH) pathway is indispensable for the normal development of the cerebellum, and MB is often caused by persistent SHH activation owing to mutations in pathway components. Patched1 (PTCH1) is the primary receptor for the SHH ligand and a negative regulator of the SHH signal transduction pathway. Mice heterozygous for the Ptch1 gene (Ptch1+/−) are predisposed to MB development. Irradiation of newborn Ptch1+/− mice dramatically increases MB occurrence. A genetic background carrying the Ptch1 mutation significantly influences the risk of developing MB. This study aims to investigate the genetic background-related mechanisms that regulate radiation-induced cellular response and oncogenesis in the cerebellum. We employed multiple approaches, including: (a) analysis of cellular radiosensitivity in granule cell precursors (GCPs), the MB cells of origin, derived from Ptch1 mice with a genetic background that is sensitive (CD1) or resistant (C57Bl/6) to the induction of radiogenic MB; (b) identification of genes differentially expressed in spontaneous and radiation-induced MBs from these two mouse strains; (c) bioinformatic analysis to correlate the expression of radiation-induced genes with survival in MB patients; and (d) examining the expression of these genes in ex vivo MBs induced by single or repeated radiation doses. We have identified a potential gene expression signature—Trp53bp1, Bax, Cyclin D1, p21, and Nanog—that influences tumor response. In ex vivo cultured spontaneous MBs, the expression levels of these genes increase after irradiation in CD1 mice, but not in mice with a C57Bl/6 genetic background, suggesting that this signature could predict tumor response to radiation therapy and help develop strategies for targeting DNA damage repair in tumors. A detailed understanding of the mechanisms behind genetic background-related susceptibility to radiation-induced oncogenic responses is crucial for translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19adfa1196ccdabbcc31c191cdbb05c4c3e4c33" target='_blank'>
              Dissecting the Impact of Genetic Background on Oncogenic Response to Radiation Exposure in the Ptch1+/− Mouse Model
              </a>
            </td>
          <td>
            B. Tanno, Emiliano Fratini, S. Leonardi, Flavia Novelli, Valentina Pisano, Mariateresa Mancuso, S. Pazzaglia
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 Acute lymphoblastic leukemia in infants (iALL) is a high-risk subtype of childhood leukemia with poor survival outcomes despite intensive therapies. Rearrangement of KMT2A (KMT2A-r) occurs in 70% of cases and is associated with chemotherapy resistance, early relapse, and rapid leukemic progression, all of which contribute to a survival rate of about 40%. The most common KMT2A-r in iALL is KMT2A::AFF1 (MLL::AF4), derived from t(4;11)(q21;23). This KMT2A-r generates a driver fusion oncogene which leads to epigenetic dysregulation of target gene transcription. Infant ALL expressing KMT2A::AFF1 is also a unique cancer in that other somatic mutations are uncommon and there are no known genetic risk factors. This leukemia's cell of origin is proposed to be a very early hematopoietic precursor, with transcriptomic studies of iALL blasts showing similarities to hematopoietic stem and progenitor cells (HSPCs), multipotent progenitors and early lymphoid progenitors (ELPs). Despite this knowledge, the factors leading to leukemogenesis remain poorly understood, with research into this devastating disease hampered by the lack of a representative model of iALL Attempts at creation of an iALL models have led to a variety of results including myeloid neoplasms, lymphomas, mixed phenotypic hyperplasia or leukemia, mature B cell neoplasias. Because of these failures, much remains unknown regarding how KMT2A::AFF1 transforms early hematopoiesis and how it alters the severity of the disease.
 To further understand the developmental state of the cell of origin and leukemogenesis of iALL, we have engineered a highly controlled human inducible pluripotent stem (iPS) cell model of KMT2A-r iALL. Using an established doxycycline (dox) regulated platform, we have generated multiple iALL driver transgene constructs, including an epitope tagged human KMT2A::AFF1 cassette and a human-mouse KMT2A::Aff1 cassette. This system targets the dox-regulated transgene and dox regulatory machinery to the endogenous AAVS1 locus into non-diseased human iPS cell lines using CRISPR mediated targeting via homology directed repair. The non-diseases human iPS cells allow for mimicking of the silent mutational landscape found in patient samples. Correct targeting of the KMT2A::Aff1 transgene was confirmed via real-time reverse transcriptase-polymerase chain reaction (RT-PCR). These cells tightly express the KMT2A::Aff1 transgene only in response to dox treatment in both undifferentiated iPS cells and after differentiation to hematopoietic cells.
 Through directed differentiation we have also produced functional HSPCs from iPS cells with multilineage differentiation capacity. Flow cytometry analysis demonstrated enrichment of CD45+CD34+ cells by day 10 of differentiation. Further flow cytometry analysis of colony forming unit assays revealed functional enrichment in lympho-myeloid (CD45RA+) differentiated cells, along with formation of monocyte (CD15+) and erythroid (CD235+) colonies. We also used the 10x Chromium system to perform single-cell RNA sequencing of the differentiated cells. The scRNAseq transcriptomic data was analyzed using Seurat in R. Transcriptomic data confirmed the cells' differentiation ability as they clustered into distinct populations representing erythrocytes, megakaryocytes, and monocytes.
 In conclusion, this model has allowed us to recapitulate hematopoietic ontogeny with the ability to control iALL induction at specific developmental stages. We are utilizing single cell genomics to investigate transcriptomic changes during hematopoietic differentiation and after expression of the KMT2A oncogenic driver. We will compare these results to published data sets of single cell RNA sequencing data from healthy fetal HSPCs and to our existing iALL and pediatric KMT2A-r ALL single cell sequencing data sets. Our iPS cell based iALL model system provides the opportunity to investigate a range of critical and outstanding questions of iALL disease initiation, progression, and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b631981ea1355f736e4203294ab4947ce73e4273" target='_blank'>
              A Genomics Driven, Inducible Pluripotent Stem Cell Model of Infant Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Meagan Vacek, Jacqelyn Nemechek, I. Pushel, Bradley T. Thornton, Molly Leyda, Priyanka Prem Kumar, M. Farooqi, Erin Guest, Jay Vivian, John M Perry
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Drug resistance and unwanted side effects in chemotherapies are major causes of therapeutic complications. To monitor cytotoxic drugs on a personalized scale, fast and easy applicable diagnostics should be developed. Chemotherapeutic agents often induce DNA damage, especially in rapidly dividing (cancer) cells, leading to its death. The detection of γH2AX, a protein involved in DNA damage repair, is considered as an efficient marker for drug-specific cell damage. Furthermore, cell-free DNA formation, a result of drug-induced cell death, may serve as a reliable diagnostic marker. Circulating in the bloodstream, cell-free DNA can be extracted in a minimally invasive way and repeatably quantified during chemotherapy. In time-course studies on drug-treated leukemia T-cells and healthy blood cells, both markers confirmed an enhanced drug response in dividing cells. Additionally, we showed formerly unknown drug-specific differences to timely induce DNA and cell damage. Our combined approach can improve personalized biomonitoring in chemotherapies and provides valuable information for patient biosampling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd00e731e8155737e43bee06b700a5b0614fcec" target='_blank'>
              Heterogeneous Formation of DNA Double-Strand Breaks and Cell-Free DNA in Leukemia T-Cell Line and Human Peripheral Blood Mononuclear Cells in Response to Topoisomerase II Inhibitors
              </a>
            </td>
          <td>
            Christian Linke, Thilo von Hänisch, Julia Schröder, W. Dammermann, P. M. Deckert, Mark Reinwald, Sandra Schwarzlose-Schwarck
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Hematopoietic malignancies emerge through the gradual acquisition of genetic mutations within hematopoietic stem and progenitor cells (HSPCs). Mutations that occur early in disease progression impart a selective growth advantage to HSPCs, which allows them to expand and contribute to a substantial percentage of mature blood cells. This increased expansion is termed clonal hematopoiesis (CH) and is a preleukemic phase associated with an increased risk of developing leukemia. Inhibitor of DNA binding 1 (ID1) protein is a transcriptional regulator of proliferation/differentiation of neuronal, muscle, hematopoietic and other cells, and is frequently overexpressed in cancer. Id1 is expressed at low levels in normal HSCs and is induced by growth factors and other mediators of inflammatory stress and promotes HSPC proliferation in vitro and in vivo. Since chronic inflammation is associated with the progression of hematopoietic malignancies, reducing Id1 expression during CH may be therapeutic. Mutations in TET2 are frequently observed in patients with CH, and Tet2−/−mice develop CH that progress to hematopoietic malignancies. Id1 is upregulated in murine Tet2−/− HSPCs and in AML, CMML and MDS patient samples with TET2 mutations. Genetic ablation of Id1 in Tet2−/− HSPCs reduces HSPC expansion/self-renewal/CH, extramedullary hematopoiesis, myeloid skewing, genetic instability and delays the onset of disease. Mechanistically, p16 expression, senescence and apoptosis were increased and proliferation decreased in Tet2−/−; Id1−/− HSPCs. Thus, ID1 may represent a potential therapeutic target to reduce CH, hematopoietic hyperplasia, and delay the onset of disease. One Sentence Summary Genetic ablation of Id1 in Tet2−/− mice rescues clonal hematopoiesis by increasing CDKI expression, apoptosis, senescence, and differentiation, and reducing cell growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3312a0ebaea58379a205a9959b262e76e84831" target='_blank'>
              Id1 Promotes Clonal Hematopoiesis in Mice with Tet2 Loss of Function
              </a>
            </td>
          <td>
            Shweta Singh, Tanmoy Sarkar, K. Gudmundsson, Brad L. Jakubison, Holly Morris, Sandra Burkett, Karim Baktiar, Gary T. Pauly, D. Sigano, Joel P. Schneider, Jonathan Keller
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represents the most prevalent and deadly primary brain tumor in adults, and it remains highly refractory to current therapeutic interventions. Significant intratumor heterogeneity drives rapid invasion, immune escape, and therapeutic resistance. Therapeutic paradigms based on deconvoluting key molecular mechanisms driving gliomagenesis are urgently needed. Quaking (Qki, encoded by Qk) is frequently altered in GBM patients; Qk mutations or deletions occur in 35% percent of cases, and the Qki protein is absent in 50-60% of cases. Our lab has characterized Qki as important tumor suppressor in GBM, where deletion of Qk enables pre-malignant neural stem cells to expand outside their niche and develop into tumors that recapitulate human GBM. We discovered that Qki functions as a transcriptional coactivator of lipid metabolism genes, whereby Qki loss disrupts lipid homeostasis and compromises the integrity of various membrane structures. Several downstream processes governed by Qki have been delineated; however, upstream regulators of Qki have yet to be identified. Recurrent cancer mutations to specific Qki arginine sites implicate the dysregulation of arginine methylation, a common post-translational modification catalyzed by protein arginine methyltransferases (PRMTs). Leveraging in vitro methylation assays, we demonstrate that Qki is selectively methylated by the PRMT coactivator-associated arginine methyltransferase 1 (CARM1), specifically in the Quaking regulatory domain where cancer mutations are reported. We observe that Qki interacts with known CARM1 substrates, functioning in the transcriptional complex assembled by CARM1. Through mutagenesis studies, we identified arginine 242 (R242) and arginine 256 (R256) as specific CARM1-mediated methylation sites on Qki. Importantly, mutations to these sites compromise Qki’s activity as a transcriptional coactivator. Moreover, brain-specific deletion of CARM1 in vivo results in neurological phenotypes reminiscent of Qki deletion. Altogether, we provide evidence that CARM1 represents a novel tumor suppressor in GBM that functions through the arginine methylation of Qki.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c4f80d5e1592ad297d2b32f6d52b2a08309ce04" target='_blank'>
              EPCO-17. CARM1 METHYLATES GLIOBLASTOMA TUMOR SUPPRESSOR QUAKING TO REGULATE ITS TRANSCRIPTIONAL COACTIVATOR FUNCTION
              </a>
            </td>
          <td>
            Kaylene Lu, Tanner Wright, Seula Shin, Mark T. Bedford, Jian Hu
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Several reports have indicated that impaired mitochondrial function contributes to the development and progression of Huntington’s disease (HD). Mitochondrial genome damage, particularly DNA strand breaks (SBs), is a potential cause for its compromised functionality. We have recently demonstrated that the activity of polynucleotide kinase 3’-phosphatase (PNKP), a critical DNA end-processing enzyme, is significantly reduced in the nuclear extract of HD patients due to lower level of a metabolite fructose-2,6 bisphosphate (F2,6BP), a biosynthetic product of 6-phosphofructo-2-kinase fructose-2,6-bisphosphatase 3 (PFKFB3), leading to persistent DNA SBs with 3’-phosphate termini, refractory to subsequent steps for repair completion. PNKP also plays a pivotal role in maintaining mitochondrial genome integrity. In this report, we provide evidence that both PFKFB3 and F2,6BP, an allosteric modulator of glycolysis, are also present in the mitochondria. Notably, the level of F2,6BP, a cofactor of PNKP, is significantly decreased due to the degradation of PFKFB3 in the mitochondrial extract of HD patients’ brain. PNKP activity is thus severely decreased in the mitochondrial extract; however, addition of F2,6BP restored PNKP activity. Moreover, supplementation of F2,6BP in HD mouse striatal neuronal cells restored mitochondrial genome integrity and partially restored mitochondrial membrane potential and prevented pathogenic aggregate formation. We observed similar restoration of mitochondrial genome integrity in HD drosophila supplemented with F2,6BP. Our findings, therefore, suggest that F2,6BP or its structural analog hold promise as a therapeutic for restoring both nuclear and mitochondrial genome integrity and thereby of organismal health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bbc602a15b70c3d96d75621746c1c9d8c7a5066" target='_blank'>
              F2,6BP restores mitochondrial genome integrity in Huntington’s Disease
              </a>
            </td>
          <td>
            Anirban Chakraborty, S. Mandal, Mikita Mankevich, S. G. Sreenivasmurthy, Muralidhar L. Hegde, Balaji Krishnan, Gourisankar Ghosh, T. Hazra
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Background: Osimertinib is currently the most common precision therapy for patients with non- small cell lung cancer (NSCLC) with EGFR mutations. While it prolongs progression-free survival, resistance eventually develops in many, leading to disease progression. Somatic NGS has identified resistance mutations in the EGFR pathway and other oncogenes, but these account for only 40-45% of cases, suggesting non-genetic factors, such as epigenetic changes, may be involved. This study investigates the epigenetic landscape at diagnosis and progression on first- line osimertinib, focusing on cases with and without canonical resistance mutations. Methods: We defined two cohorts from GuardantINFORM, a clinico-genomics database, restricting to only patients with a test on the Guardant Infinity platform, a comprehensive blood-based NGS assay, that analyzes DNA sequence variants and performs extensive methylation profiling (∼15Mb) in >20,000 differentially methylated regions (DMRs) assessed for sample-specific methylation patterns. (A) Diagnosis Cohort: Comprised of 34 NSCLC patients with blood samples collected within six days prior to the initiation of first-line osimertinib therapy. (B) Progression Cohort: Comprised of 36 patients who were on first-line osimertinib therapy, with blood samples collected within 60 days of discontinuation. The progression cohort was further stratified based on the presence or absence of known resistance mutations identified through somatic variant analysis by Guardant360. Gene set enrichment analysis was conducted on these methylation profiles using the Reactome database to define pathway-level differences between the cohorts. Results: Pathway enrichment analysis in the diagnosis and progression cohorts found expected biological processes in NSCLC such as extracellular matrix organization, G- protein coupled receptor (GPCR) signaling, and FGFR pathways. Additionally, the progression cohort lacking canonical resistance mutations (n=23) exhibited unique enrichment in two specific pathways: Glucose-dependent Insulinotropic Polypeptide and Physiological Factors. These pathways include genes such as GATA4, which has been implicated in the epithelial- mesenchymal transition (EMT)—a process associated with transforming to a small cell phenotype, a known mechanism of resistance to EGFR inhibition, including osimertinib. Conclusion: This study demonstrates the utility of epigenetic profiling via liquid biopsies in uncovering non-genetic mechanisms underlying osimertinib resistance in advanced NSCLC. Identification of unique epigenetic pathways in progressing patients without canonical resistance mutations highlights the potential for discovering novel biomarkers of resistance. Further investigation into these findings may advance our understanding of resistance mechanisms across various cancer therapies, particularly for patients without identified genomic resistance, and inform clinical decision-making and trial design.
 Citation Format: Aaron Hardin, Leslie Bucheit, Amar Das, Craig Eagel, Helmy Eltoukhy. Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9810af5aadd760f6c2c32f19cb5c026e7ca93" target='_blank'>
              Abstract B036: Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay
              </a>
            </td>
          <td>
            Aaron Hardin, L. Bucheit, Amar Das, Craig Eagel, H. Eltoukhy
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="

 Extra-Nodal (NK)/T-cell Lymphoma (ENKTL) is the most common subtype of NK lymphoma derived from EBV-infected NK cells. ENKTL is incurable and the 5-year survival is around 40% (<10% if disseminated) highlighting the unmet need for novel therapies. It has been shown that upregulation of DNA damage repair (DDR) plays a pivotal role in ENKTL survival by promoting genomic stability and resistance to lymphoma conventional DNA damaging therapies as reflected by the limited response rate (<50%). We identified protein arginine methyltransferase (PRMT5) to be oncogenic in many malignancies by supporting the cell cycle, WNT, and PI3K/AKT pathways. PRMT5, also promotes DDR through RNA splicing of TIP60/KAT5 and stabilization of 53BP1, essential for homologous recombination (HR) and non-homologous end joining (NHEJ), respectively. PRMT5-deficiency leads to DNA damage accumulation, cell cycle arrest, and apoptosis. Unbiased analysis of our RNA-seq data on ENKTL patient samples (n=22) show significant upregulation of DDR genes compared to normal NK cells. Interestingly, we also detected PRMT5 overexpression in 95% of this ENKTL cohort by RNA-seq, suggesting that PRMT5 may also be playing a role in the DDR within ENKTL. We confirmed these RNA-seq data and detected PRMT5 overexpression in primary ENKTL patients' samples (n=10) by immunohistochemical (IHC) staining compared to normal tonsils (n=5), in ENKTL-cell lines (NKL, NK92, and YT) and in a patient-derived xenograft (PDX) mouse model of ENKTL (ENKTL1) by western-blot. We hypothesize that PRMT5 inhibition 1) targets DNA repair in ENKTL cells leading to increased DNA damage-induced cell death and 2) sensitizes ENKTL cells to other DNA damage-inducing agents used in the current treatment of ENKTL leading to substantial increases in DNA damage-mediated cell death with combination therapy. To address this hypothesis, we used selective PRMT5-inhibitors, PRT382 and PRT808 (Prelude-Therapeutics) and detected cytotoxicity (IC50 25 - 500nM) in ENKTL-cell lines and ENKTL1. Furthermore, PRT382 significantly prolonged overall survival (OS) in our in vivo PDX mouse model of ENKTL1 (median OS: control 43 days vs PRT382 65 days; p=0.0088). Mechanistically, accumulation of DNA-damage was evident by the increase in phosphorylated forms of 1) H2AX (H2A histone family member X) at Ser139, and 2) Chk1/Chk2 (major DNA check points control) at Ser317/Ser345 and Thr68, respectively, in PRT808-treated cells and upon PRMT5 knock-down. Lastly, PRT808 sensitized NK-lymphoma cells to killing by doxorubicin (DNA-damage agent) and Olaparib (PARP1/2-inhibitor, key regulator of HR). In conclusion, PRMT5-inhibition results in an unrepairable level of DNA damage/subsequent apoptosis and sensitizes NK-lymphoma to other DNA damage-inducing agents. All together, these data justify targeting PRMT5 in ENKTL, as a single agent and in combination with other DNA damage agents which are an essential part of the current ENKTL regimen (Doxorubicin and Etoposide) representing a novel approach for better outcome. Further mechanistic studies investigating the role of PRMT5 in DNA repair and DNA damage in ENKTL cells are ongoing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afcae6b6400f10ddf471f1df0daddfbaee52e50b" target='_blank'>
              Targeting PRMT5-Mediated DNA Repair Pathway in Extra-Nodal NK/T-Cell Lymphoma (ENKTL)
              </a>
            </td>
          <td>
            Y. Youssef, Walter Hanel, Wing Chan, Fiona C. Brown, C. Hinterschied, J. Helmig-Mason, Neha Bhagwat, Kris Vaddi, Peggy Scherle, Amin Noorani, A. Freud, L. Alinari, C. C. Oakes, Bethany L. Mundy-Bosse, Robert A Baiocchi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="

 Resistance to anti-cancer therapeutics remains one of major obstacles to improving survival rates in cancer; it arises in a multitude of ways including accumulation of epigenetic alterations. Recently, we discovered a new mechanism of treatment resistance that is not fully driven by epigenetic remodeling but nevertheless implies an activity switch of an epigenetic enzyme, EZH2. Despite its well-established activity as part of PRC2 for mediating gene repression by H3K27me3 deposition, new evidence points the importance of others, so-called non-canonical activities mediated by variation of PRC2 composition and kinase-related post-translational modifications (PTMs) of EZH2. We previously reported that non-canonical EZH2 activity was associated with retinoic acid (RA) resistance in acute promyelocytic leukemia (APL) (Poplineau, Blood, 2022). RA resistant cells (i) expressed genes involved in DNA repair, replication and proliferation processes, that we called ReP signature, (ii) kept the potential to develop leukemia in vivo and (iii) were marked by high EZH2 expression. Targeting pan-EZH2 activities (canonical/non-canonical) was necessary to eliminate RA resistant cells, which underlies a dependency of these cells on an EZH2 non-canonical and non-methyltransferase activity and the necessity to degrade EZH2 to overcome RA resistance in APL.
 In the present work, we questioned whether EZH2 non-canonical activity may contribute more generally to resistance in non-APL AML. To do so, we questioned non-APL public AML transcriptomic datasets (BeatAML and TCGA). At first, we classified non-APL AML patients with high and low EZH2 expression and revealed that patients with high EZH2 expression were enriched in metabolic signatures reflecting chemotherapy resistance (i-e HighOXPHOS). Next, we computed a new EZH2-related signature (i-e; 61 common genes to BeatAML and TGCA obtained by retaining the 200 most differentially expressed genes in patients with high and low EZH2 expression) and found that this signature was positively correlated with AraC low responder patients. Besides, we showed that our computed non-canonical and non-methyltransferase EZH2 and ReP signatures were associated (i) with HighOXPHOS and leukemic stem cells maintenance signatures in AML patients and (ii) relapse of patients following AraC treatment.
 Next, we investigated the relevance of targeting EZH2 in AML using an EZH2 PROTAC synthetized in our laboratory and derived from the previously published MS177 EZH2 degrader (Wang, Nat. Cell Biol., 2022). Using several leukemic cell lines, we showed that chemo-resistant cell lines, more especially AraC resistant cells, were highly sensitive to EZH2 degradation (fifty times more sensitive than AraC sensitive cells) but were not sensitive to the catalytic EZH2 inhibitor Tazemetostat. We then studied the potential synergistic effect between the EZH2 degrader and AraC and did not observe any additional beneficial effect with the combo therapy. However, we showed that EZH2 degradation has strong synergistic effect with RA in killing AML cells in vitro, suggesting that EZH2 degradation may sensitize chemotherapy resistant cells to RA.
 Finally, by coupling omics approaches on AML cell line resistant to AraC, we are currently investigating mechanisms that dictate EZH2 activities (canonical/non-canonical) and characterizing these activities at the functional level. Proteomics screening revealed specific PTMs on EZH2 and variation in its interacting partners that may contribute to AraC resistance in AML.
 Altogether, these data reinforce (i) the intimate link between EZH2 non-canonical activity and AraC resistance in non-APL AML and (ii) support the need to preclinically investigate the therapeutic value of targeting non-canonical EZH2 activities not only to sensitize non-APL AML to RA but also to overcome chemotherapy resistance in non-APL AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477e541c859f23a419d4cdd61e3b25c660a0006d" target='_blank'>
              Deciphering EZH2 Non-Canonical Activities in AML Resistance
              </a>
            </td>
          <td>
            L. N’Guyen, Emeline Boet, C. Montersino, Claire Burny, S. Audebert, Luc Camoin, R. Castellano, S. Garciaz, Atsushi Iwama, J. Sarry, E. Duprez, M. Poplineau
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a101d4e936af34d1b466e5552558a397fc61b" target='_blank'>
              Carcinogenic Mechanisms of Hexavalent Chromium: From DNA Breaks to Chromosome Instability and Neoplastic Transformation.
              </a>
            </td>
          <td>
            Idoia Meaza, Aggie R Williams, Sandra S Wise, Haiyan Lu, John W Pierce
          </td>
          <td>2024-10-28</td>
          <td>Current environmental health reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Here, we demonstrate that single strand annealing (SSA) can be co-opted for the precise autocatalytic excision of a drive element. We have termed this technology Repeat Mediated Excision of a Drive Element (ReMEDE). By engineering direct repeats flanking the drive allele and inducing a double-strand DNA break (DSB) at a second endonuclease target site within the allele, we increased the utilization of SSA repair. ReMEDE was incorporated into the mutagenic chain reaction (MCR) gene drive targeting the yellow gene of Drosophila melanogaster, successfully replacing drive alleles with wild-type alleles. Sequencing across the Cas9 target site confirmed transgene excision by SSA after pair-mated outcrosses with yReMEDE females, revealing ~4% inheritance of an engineered silent TcG marker sequence. However, phenotypically wild-type flies with alleles of indeterminate biogenesis also were observed, retaining the TGG sequence (~16%) or harboring a silent gGG mutation (~0.5%) at the PAM site. Additionally, ~14% of alleles in the F2 flies were intact or uncut paternally inherited alleles, indicating limited maternal deposition of Cas9 RNP. Although ReMEDE requires further research and development, the technology has some promising features as a gene drive mitigation strategy, notably its potential to restore wild-type populations without additional transgenic releases or large-scale environmental modifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2496fb754fc3f570ffb02c4a20aa804e15a0691b" target='_blank'>
              Repeat mediated excision of gene drive elements for restoring wild-type populations
              </a>
            </td>
          <td>
            Pratima R. Chennuri, Josef Zapletal, Raquel D. Monfardini, M. Ndeffo-Mbah, Zach N. Adelman, Kevin M. Myles
          </td>
          <td>2024-11-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

 Chronic Lymphocytic Leukemia (CLL) is the most common hematological malignancy in the United States, accounting for approximately 25% to 35% of all leukemias. Targeted therapies such as Bruton's Tyrosine Kinase (BTK) and B-cell lymphoma-2 (BCL-2) inhibitors ibrutinib and venetoclax have produced durable responses in CLL however, a subset of patients experience relapse and poor outcomes. This group with a poor prognosis is in urgent need of therapies that address the unique biology of treatment resistant CLL.
 To investigate mutant TP53's contribution to disease progression, we collected samples at baseline from patients entering ibrutinib treatment for front-line and relapsed/refractory CLL. Large-scale exome sequencing was performed on 270 patient samples revealing that 106 patients harbored some TP53 aberration, mostly in the DNA binding domain (DBD). Mutational analysis revealed missense mutations R175H and R248Q were among the most frequently occurring. These variants are known gain of function (GOF) mutations in solid tumors contributing to tumor progression and therapeutic resistance. Eighty-two percent of TP53 mutations in this cohort had a simultaneous deletion of the short arm of chromosome 17 (del17p) shown by fluorescence in situ hybridization (FISH). Del17p serves as an independent predictor of progression on ibrutinib as these patients had an inferior response to ibrutinib. These data lead us to hypothesize that the presence of TP53 mutations in CLL supports a shortened time to initial therapy and progression and relapse compared to patients without mutations in TP53.
 Wild type (WT) p53 is an essential transcription factor functioning as a tumor suppressor responsible for DNA damage repair, regulating apoptosis, and maintaining genomic stability. To describe mutant p53 activity, DNA damage was induced by ionizing radiation (IR) administered at 3Gy for 1 hour. WT but not mutant TP53 samples responded to DNA damage by stabilizing p53 protein expression. After stabilization, WT p53 strongly induced p21 protein, whereas p53 mutant expressing patient samples did not induce p21 protein in response to IR indicating a disrupted DNA repair pathway ex vivo. However, these studies did not show which genes are being targeted transcriptionally by mutant p53 nor how mutant p53 binds to and regulates these targets allowing these affected cells to subvert apoptosis.
 To further investigate this, we independently modeled this system in two cell lines. The HG3 and OSU-CLL lines are patient-derived lines created using CRISPR/Cas9 gene editing to introduce specific mutations at R175H and R248Q into WT p53 CLL. Homozygous knock-in of R175H and R248Q mutations on p53 was confirmed through Sanger sequencing on expanded single-cell clones. A profile of differentially expressed genes was revealed during RNA-seq analysis of mutant TP53 and WT TP53 cell lines. Genes overexpressed in relation to WT include transcription factors such as AFF3 and BCL11b and known tumor suppressors like KLHL14. Interestingly, a strong downregulation was seen in CYP4F3 across mutants whose upregulation is shown to be associated with resistance to chemotherapeutic agents such as fludarabine and targeted agents such as ibrutinib in the mutant TP53 setting. To determine whether p53 mutants in CLL exhibit a unique transcriptional signature like what is seen in solid tumors, we are conducting chromatin immunoprecipitation (ChIP) for p53 in these cell lines.
 In summary, our study reveals that TP53 mutations take advantage of mechanisms of resistance towards apoptosis-inducing therapies and impair the normal tumor-suppressive function of WT p53 which remains to be addressed in the clinic. These significantly worsen patient prognosis and accelerate relapse. Ongoing analysis will identify and characterize the action of these differentially expressed targets in CLL and highlight their role in facilitating survival in TP53 mutant CLL cells during therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2a607318eb84b0272d94a7830a09a85dda87" target='_blank'>
              TP53 Gain-of-Function Mutants Display Unique Transcriptional Profiles in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            C. Williams, Suresh Satpati, J. Ravikrishnan, Bailey Slawin, Daisy Y. Diaz Rohena, Palaniraja Thandapani, K. Rogers, S. Bhat, Kunal Rai, J. Woyach, Deepa Sampath
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Homologous recombination (HR) deficiency upon BRCA2 loss arises from defects in the formation of RAD51 nucleoprotein filaments. Here, we demonstrate that loss of the anti-recombinase FIGNL1 retains RAD51 loading at DNA double-stranded breaks (DSBs) in BRCA2-deficient cells, leading to genome stability, HR proficiency, and viability of BRCA2-deficient mouse embryonic stem cells. Mechanistically, we directly show that strand invasion and subsequent HR defects upon BRCA2 loss primarily arises from the unrestricted removal of RAD51 from DSB sites by FIGNL1, rather than from defective RAD51 loading. Furthermore, we identify that the MMS22L-TONSL complex interacts with FIGNL1 and is critical for HR in BRCA2/FIGNL1 double-deficient cells. These findings identify a pathway for tightly regulating RAD51 activity to promote efficient HR, offering insights into mechanisms of chemoresistance in BRCA2-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cebc2a7a30b95e95e9cd143f7fd6f4463ef94048" target='_blank'>
              FIGNL1 inhibits homologous recombination in BRCA2 deficient cells by dissociating RAD51 filaments
              </a>
            </td>
          <td>
            Raviprasad Kuthethur, A. Acharya, S. Sengodan, Carmen Fonseca, Nupur Nagar, Safa Nasrin VZ, Oluwakemi Ibini, E. Manolika, Kelly de Koning, Stefan Braunshier, Julien Dessapt, Amélie Fradet-Turcotte, Joyce H.G. Lebbink, Roland Kanaar, K. Poluri, Shyam K. Sharan, Petr Cejka, A. Chaudhuri
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Repeat expansion disorders (REDs) encompass over 50 inherited neurological disorders and are characterized by the expansion of short tandem nucleotide repeats beyond a specific repeat length. Particularly intriguing among these are multiple fragile X-associated disorders (FXds), which arise from an expansion of CGG repeats in the 5' untranslated region of the FMR1 gene. Despite arising from repeat expansions in the same gene, the clinical manifestations of FXds vary widely, encompassing developmental delays, parkinsonism, dementia, and an increased risk of infertility. FXds also exhibit molecular mechanisms observed in other REDs, that is, gene- and protein-loss-of-function and RNA- and protein-gain-of-function. The heterogeneity of phenotypes and pathomechanisms in FXds results from the different lengths of the CGG tract. As the number of repeats increases, the structures formed by RNA and DNA fragments containing CGG repeats change significantly, contributing to the diversity of FXd phenotypes and mechanisms. In this review, we discuss the role of RNA and DNA structures formed by expanded CGG repeats in driving FXd pathogenesis and how the genetic instability of CGG repeats is mediated by the complex interplay between transcription, DNA replication, and repair. We also discuss therapeutic strategies, including small molecules, antisense oligonucleotides, and CRISPR-Cas systems, that target toxic RNA and DNA involved in the development of FXds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8069105a08fb0ea3a51654b6cac8adcc45f352f1" target='_blank'>
              Contribution of DNA/RNA Structures Formed by Expanded CGG/CCG Repeats Within the FMR1 Locus in the Pathogenesis of Fragile X-Associated Disorders.
              </a>
            </td>
          <td>
            Izabela Broniarek, Daria Niewiadomska, Krzysztof Sobczak
          </td>
          <td>2024-11-01</td>
          <td>Wiley interdisciplinary reviews. RNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Epigenetic alterations in gene expression have been implicated in cancer development and tumor immune escape, with posttranslational histone or non-histone modifications representing attractive targets for disease surveillance and therapy. SET domain bifurcated 1 (SETDB1) is a histone lysine methyltransferase that reversibly catalyzes the di- and tri-methylation of histone 3 lysine 9 (H3K9) on euchromatin, inhibiting gene transcription within these regions and facilitating the switch from euchromatic to heterochromatic states. Emerging evidence suggests that SETDB1 amplification and aberrant activation are significantly associated with poor prognosis in hepatocellular carcinoma (HCC), and contribute to HCC development, immune escape, and immune checkpoint blockade (ICB) resistance. Here, we provide an updated overview of the cellular and molecular effects of SETDB1 activity in hepatocarcinogenesis and progression and focus on studies linking its function to immunotherapy for HCC, and present current challenges and future perspectives for targeting SETDB1 in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775e9dd157e3b9dd836833865f11fe90ae06aaa7" target='_blank'>
              The function of histone methyltransferase SETDB1 and its roles in liver cancer
              </a>
            </td>
          <td>
            Enxiang Zhang, Pingping He
          </td>
          <td>2024-11-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA replication represents a series of precisely regulated events performed by a complex protein machinery that guarantees accurate duplication of the genetic information. Since DNA replication is permanently faced by a variety of exogenous and endogenous stressors, DNA damage response, repair and replication must be closely coordinated to maintain genomic integrity. HROB has been identified recently as a binding partner and activator of the Mcm8/9 helicase involved in DNA interstrand crosslink (ICL) repair. We identified HROB independently as a nuclear protein whose expression is co-regulated with various DNA replication factors. Accordingly, the HROB protein level showed a maximum in S phase and a downregulation in quiescence. Structural prediction and homology searches revealed that HROB is a largely intrinsically disordered protein bearing a helix-rich region and a canonical oligonucleotide/oligosaccharide-binding-fold motif that originated early in eukaryotic evolution. Employing a flow cytometry Förster resonance energy transfer (FRET) assay, we detected associations between HROB and proteins of the DNA replication machinery. Moreover, ectopic expression of HROB protein led to an almost complete shutdown of DNA replication. The available data imply a function for HROB during DNA replication across barriers such as ICLs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/063ed9d559a592d4694044ef441583042ee83050" target='_blank'>
              HROB Is Implicated in DNA Replication
              </a>
            </td>
          <td>
            Julia Kutz, Hannes Schmietendorf, Sheikh Anika Rahman, Franz Opel, Helmut Pospiech
          </td>
          <td>2024-12-10</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8d3d24f9c6f481141224932a60797062129316" target='_blank'>
              Combatting cellular immortality in cancers by targeting the shelterin protein complex
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2024-11-22</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6c4cb49f91070291b1cb8d9f3de6129d9c0bd5" target='_blank'>
              Base-excision repair pathway shapes 5-methylcytosine deamination signatures in pan-cancer genomes
              </a>
            </td>
          <td>
            André Bortolini Silveira, Alexandre Houy, Olivier Ganier, Begüm Özemek, Sandra Vanhuele, A. Vincent-Salomon, N. Cassoux, Pascale Mariani, G. Pierron, Serge Leyvraz, Damian Rieke, Alberto Picca, Franck Bielle, M. Yaspo, M. Rodrigues, Marc-Henri Stern
          </td>
          <td>2024-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="

 Angioimmunoblastic T-cell lymphoma (AITL) is an aggressive subtype of non-Hodgkin's lymphoma, representing a significant portion of peripheral T-cell lymphomas (PTCL). To elucidate the genomic landscape of PTCL, comprehensive genomic analysis was performed on 25 AITL and 19 PTCL- not otherwise specified (NOS) cases. Our findings reveal distinct genetic alterations in AITL, including a lower mutation burden and an enrichment of reactive oxygen species (ROS)-associated mutational signature (SBS18) when compared to PTCL-NOS. Besides recurrent somatic mutations in TET2 and RhoA, we have discovered frequent novel focal deletion on chromosome 8q24.21, impacting the PVT1 gene. Functional characterization demonstrated that this deletion activates the DNA damage pathway via CDKN1A/p21, potentially conferring a survival advantage to AITL cells under DNA damaging stress. These findings shed light on the molecular mechanisms underlying AITL development and highlight the potential clinical implications of chr8q24.21 focal deletion in this disease. Further studies with larger cohorts are warranted to validate these findings and explore their therapeutic implications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017f87b1d59e8667058766e4b5b6cf42db5e9ebd" target='_blank'>
              Functional Consequences of Chromosomal Alterations in Angioimmunoblastic T-Cell Lymphoma
              </a>
            </td>
          <td>
            Dachuan Huang, Jason Yongsheng Chan, C. Ong, J. Lim
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10-10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5'-TGT*-3' motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential.IMPORTANCEYarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica.
              </a>
            </td>
          <td>
            , Yuan-Chun Fang, Min He, , Lei Qi, Yang Sui, Ke Zhang, , Liang Meng, Ou Li, 
          </td>
          <td>2024-12-23</td>
          <td>Applied and environmental microbiology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 BACKGROUND
 Hematopoietic stem cells (HSCs) constantly acquire somatic mutations at a rate of ~17 per cell per year. Mutations in specific genes offer a growth advantage that may result in clonal hematopoiesis, and eventually, hematological malignancy. Understanding the processes causing these somatic mutations is fundamental to understanding the early origins of malignancy and informing interventions to reduce cancer risk. While some mutations in HSCs are from well-characterised processes, most are of unknown cause. Mutations are usually preceded by DNA damage. In fact, DNA suffers continual damage, with thousands of individual lesions at any moment. The efficiency of DNA repair means that most known classes of lesions have a half-life of minutes to hours, but might DNA damage persist longer? Understanding the dynamics of DNA lesions may help elucidate their origins and mitigate their consequences.
 METHODS
 We hypothesised that examining high-resolution phylogenetic trees of somatic cells would let us infer the persistence of DNA lesions across multiple cell cycles. A given lesion that persisted across several cell divisions would have potential to generate a mutation each time that strand was replicated, and these separate mutations could be detectable in the phylogeny. If different bases were misincorporated opposite the lesion during sequential rounds of DNA replication, closely related clones would carry two alternative mutations at the same position in the genome (‘multi-allelic’ variants; MAVs). If a persistent lesion has the same (incorrect) base misincorporated opposite during different rounds of replication, those mutations may be recognised by their contravention of the consensus phylogeny (‘phylogeny-violating’ variants; PVVs).
 We used seven published sets of somatic phylogenies from whole-genome sequencing of single-cell-derived colonies, organoids or laser-capture microdissections (LCM). The dataset comprised 103 phylogenies from 89 individuals, in total 11,429 genomes. Each phylogeny was generated from a single tissue type: hematopoietic stem and progenitor cells (HSPCs, n=39), bronchial epithelial cells (n=16) or liver parenchyma (n=48). The HSPC phylogenies were either from foetal and cord blood (n=4), healthy adults (n=13), stem cell transplant donor/recipient pairs (n=10), patients with myeloproliferative neoplasms (n=10) or chemotherapy-exposed patients (n=2). We examined all phylogenies to identify MAVs and PVVs caused by persistent DNA lesions.
 RESULTS We identified mutations arising from 816 DNA lesions that persisted across multiple cell cycles in normal human stem cells from the blood, liver and bronchial epithelium.In HSCs, persistent DNA lesions, likely from endogenous sources, generated a characteristic mutational signature, so-called SBS19.The causative lesions predominantly affected guanines in an ApG context.The persistent lesions occurred steadily throughout life, enduring 2.2 years on average, with 25% lasting 3+ years.We estimate that a single HSC has ~8 such lesions at any moment in time, half of which will generate a mutation with each cell cycle.Overall, 16% of mutations in blood cells are attributable to SBS19, and similar proportions of driver mutations in blood cancers exhibit this signature.
 CONCLUSION
 A vast array of DNA lesions emerges from the quotidian chemistry of life coupled with the rather more elective chemistry of our lifestyles. While many lesions are generated frequently and efficiently repaired, our data imply the existence of a family of DNA lesions present in low numbers, but persisting for months to years, generating sizable fractions of cells' mutation burdens.
 HSCs are unusual amongst normal tissues in that a substantial proportion of mutations have a pattern corresponding to SBS19. Our data explain this, as HSCs feature long-lived lesions giving rise specifically to mutations with this profile. Intriguingly, some patients with sickle cell disease have been shown to have increased rates of SBS19, suggesting there is variability in the degree of damage caused.
 Recognizing the unusual dynamics of this DNA damage in HSCs, future work will be well-placed to identify the causative lesion and the reason for its persistence, informing strategies to reduce the mutations it causes. In so doing, we may be able to prevent the earliest origins of blood cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b480333705d27a92ba25d8f06391ec300a70fb6" target='_blank'>
              Mutagenic DNA Lesions in Normal Hematopoietic Stem Cells Persist for Years Causing Distinct Patterns of Mutation
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A. Fabre, A. Ranzoni, Philip S. Robinson, Lori D. Kregar, C. M. Wilk, S. Boettcher, Krishnaa Mahbubani, K. Saeb-Parsy, Stanley Ng, George S. Vassiliou, Kate Gowers, Sam Janes, Matt Hoare, Anthony R Green, Ana Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael Stratton, J. Nangalia, Tim H. H. Coorens, Peter J Campbell
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Abstract Motivation Centromeres are chromosomal regions historically understudied with sequencing technologies due to their repetitive nature and short-read mapping limitations. However, recent improvements in long-read sequencing allow for the investigation of complex regions of the genome at the sequence and epigenetic levels. Results Here, we present Centromere Dip Region (CDR)-Finder: a tool to identify regions of hypomethylation within the centromeres of high-quality, contiguous genome assemblies. These regions are typically associated with a unique type of chromatin containing the histone H3 variant CENP-A, which marks the location of the kinetochore. CDR-Finder identifies the CDRs in large and short centromeres and generates a BED file indicating the location of the CDRs within the centromere. It also outputs a plot for visualization, validation, and downstream analysis. Availability and implementation CDR-Finder is available at https://github.com/EichlerLab/CDR-Finder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8a2c873ad9f738bfcea9319867ca10efdb76a5" target='_blank'>
              Identification and annotation of centromeric hypomethylated regions with CDR-Finder
              </a>
            </td>
          <td>
            Francesco Kumara Mastrorosa, Keisuke K. Oshima, Allison N. Rozanski, William T. Harvey, E. Eichler, Glennis A. Logsdon
          </td>
          <td>2024-12-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d829dac90ae6e0bee2354e14ecf9cb6f0b45ddf8" target='_blank'>
              Polyclonal-to-monoclonal transition in colorectal precancerous evolution.
              </a>
            </td>
          <td>
            Zhaolian Lu, Shanlan Mo, Duo Xie, Xiangwei Zhai, Shanjun Deng, Kantian Zhou, Kun Wang, Xueling Kang, Hao Zhang, Juanzhen Tong, Liangzhen Hou, Huijuan Hu, Xuefei Li, Da Zhou, L. T. Lee, Li Liu, Yaxi Zhu, Jing Yu, Ping Lan, Jiguang Wang, Zhen He, Xionglei He, Zheng Hu
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Metabolic reprogramming from oxidative respiration to glycolysis is generally considered to be advantageous for tumor initiation and progression. However, we found that breast cancer cells forced to perform glycolysis acquired a vulnerability to PARP inhibitors. Small-molecule inhibition of mitochondrial respiration-using glyceollin I, metformin, or phenformin-induced overproduction of the oncometabolite lactate, which acidified the extracellular milieu and repressed the expression of homologous recombination (HR)-associated DNA repair genes. These serial events created so-called "BRCAness," in which cells exhibit an HR deficiency phenotype despite lacking germline mutations in HR genes such as BRCA1 and BRCA2, and, thus, sensitized the cancer cells to clinically available poly(ADP-ribose) polymerase inhibitors. The increase in lactate repressed HR-associated gene expression by decreasing histone acetylation. These effects were selective to breast cancer cells; normal epithelial cells retained HR proficiency and cell viability. These mechanistic insights into the BRCAness-prone properties of breast cancer cells support the therapeutic utility and cancer cell-specific potential of mitochondria-targeting drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d552f07898d1f01f3a3d3064ba56a79d2a0073d0" target='_blank'>
              Metabolically inducing defects in DNA repair sensitizes BRCA-wild-type cancer cells to replication stress.
              </a>
            </td>
          <td>
            Kenji Watanabe, Tatsuro Yamamoto, Tomoko Fujita, Shinjiro Hino, Yuko Hino, K. Yamazaki, Y. Ohashi, Shun Sakuraba, H. Kono, M. Nakao, Koji Ochiai, Shingo Dan, Noriko Saitoh
          </td>
          <td>2024-11-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Centromeres are chromosomal regions historically understudied with sequencing technologies due to their repetitive nature and short-read mapping limitations. However, recent improvements in long-read sequencing allowed for the investigation of complex regions of the genome at the sequence and epigenetic levels. Here, we present Centromere Dip Region (CDR)-Finder: a tool to identify regions of hypomethylation within the centromeres of high-quality, contiguous genome assemblies. These regions are typically associated with a unique type of chromatin containing the histone H3 variant CENP-A, which marks the location of the kinetochore. CDR-Finder identifies the CDRs in large and short centromeres and generates a BED file indicating the location of the CDRs within the centromere. It also outputs a plot for visualization, validation, and downstream analysis. CDR-Finder is available at https://github.com/EichlerLab/CDR-Finder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ea2acfbe88409c0df7154515c53c47ebf174b6" target='_blank'>
              Identification and annotation of centromeric hypomethylated regions with Centromere Dip Region (CDR)-Finder
              </a>
            </td>
          <td>
            F. Mastrorosa, Keisuke K. Oshima, Allison N. Rozanski, William T. Harvey, E. Eichler, Glennis A. Logsdon
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 The cohesin complex is a ring-shaped structure composed of SMC1, SMC3, RAD21, and either STAG1 or STAG2 and plays a crucial role in the topological control of gene expression. Given that STAG2 is recurrently mutated in myeloid malignancies, understanding the role of STAG2 in hematopoiesis could provide novel therapeutic insights. Our prior work identified a specific role for STAG2 in facilitating new short range chromatin loops that facilitate hematopoietic stem cell (HSC) fate commitment and lineage priming. Subsequently, we demonstrated that Stag2 loss results in increased HSC self-renewal and myelodysplasia, primarily due to loss of DNA accessibility and ensuing transcriptional dysregulation. After lineage priming and fate specification, erythropoiesis is characterized by genome wide repression, loss of DNA accessibility, and enucleation. In contrast to HSCs and myeloid progenitors, we demonstrate that loss of Stag2 cohesin alters erythropoiesis at the progenitor stage and during terminal erythroid differentiation (TED) with defects in chromosomal condensation and impaired nuclear extrusion.
 To investigate TED, Stag2WT and Stag2∆ bone marrow cells were stained for Ter119/CD44 and analyzed via flow cytometry, which revealed fewer Ter119+ cells (p<0.001), decreased proerythroblasts (ProE, p<0.05), and increased orthochromatic erythroblasts (OrthoE, p<0.01). As Stag2∆ mice are not anemic, we assayed serum erythropoietin (EPO) and splenic masses, and identified an increase in EPO and splenic mass (p<0.05 and 0.001, respectively) in Stag2∆ mice suggesting orthogonal compensatory mechanisms. Flow cytometry confirmed splenic erythropoiesis with expansion of splenic Ter119+ erythroblasts in Stag2∆ mice (p<0.05). Given that enucleation occurs during the OrthoE stage, Stag2∆ cells were stained with the nuclear binding dye Syto16, revealing fewer enucleated cells relative to Stag2WT controls (p<0.05). Imaging flow cytometry data of Stag2WT and Stag2∆ bone marrow reveals enlarged nuclei in Stag2∆ erythroblasts, though most prominent in ProEs and polychromatic erythroblasts, suggestive of impaired chromatin condensation.
 At the progenitor stage, we observe decreased erythroid maturation via scRNA-seq of murine Stag2∆ Lin- HSPCs. Furthermore, we observe decreased expression of Gypa and Tfrc in Stag2∆ MEPs and CFU-Es. Interestingly, we also noted aberrations to critical components of erythroblast enucleation, which requires caspase-3 mediated cleavage of lamin B to permit nuclear opening and histone release. We observe decreased caspase-3 expression associated with Stag2 loss, suggesting alterations in critical enucleation machinery from the outset of erythropoiesis. Subsequent bulk ATAC-seq of sorted Stag2WT and Stag2∆ CFU-Es (Lin-cKit+Sca1-CD34-Fcγ-CD105+) showed an overall more relaxed chromatin state and hyperaccessibility with Stag2 loss, suggesting defective nuclear condensation from the earliest stages of erythropoiesis. Analysis of scATAC-seq of murine Stag2WT and Stag2∆ Lin- HSPCs reveals failure of chromatin remodeling across the seven progressively mature erythroid populations observed during erythroid maturation. In support of this, we observe decreased accessibility at Casp3 with Stag2∆, implicating enucleation pathways as Stag2-dependent in their transcriptional regulation. Investigations into whether correction of aberrant caspase-3 expression restores normal enucleation is underway to identify a potential common pathway by which Stag2 loss controls erythropoiesis. Overall, these data suggest that loss of Stag2 negatively affects chromosomal condensation, gene regulation, and nuclear extrusion, requiring compensatory EPO secretion and splenic erythropoiesis to maintain appropriate hemoglobin levels. These data expand the role of Stag2 from HSC function and lineage priming to terminal erythropoiesis and define a novel physiologic role for Stag2 in erythropoiesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eaddeff36828e26d4a3aba37f893747a9cca5fe" target='_blank'>
              Erythroid Dysplasia in Stag2 Deficient Murine Models Reveals Novel Erythropoietic Function for Stag2 Cohesin
              </a>
            </td>
          <td>
            Varun S Sudunagunta, Yi Chen, Russell Kunes, Edna M. Stewart, Viviana Scoca, John Pantazi, Hongxia Yan, J. Xu, M. Narla, A. Viny
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract We performed comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses. While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (e.g., EGFR, PTEN, NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Further, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation at recurrence had inferior survival and were significantly enriched in NF1, TP53, and RB1 alterations and the Mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and hypermethylation at four specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes. This talk will describe how GBM undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704f3941c3c62c19e62495b9dd1824a7c49e3146" target='_blank'>
              IL-1 IDHWT GLIOBLASTOMA LONGITUDINAL MOLECULAR EVOLUTION
              </a>
            </td>
          <td>
            C. G. Lucas, N. Al-Adli, J. S. Young, R. Gupta, R. Morshed, J. Wu, A. Ravindranthan, A. Shai, N. Oberheim Bush, J. W. Taylor, J. de Groot, J. Villanueva-Meyer, M. Pekmezci, A. Perry, A. W. Bolen, P. Theodosopoulos, M. Aghi, E. Chang, S. Hervey-Jumper, D. R. Raleigh, A. M. Molinaro, J. Costello, A. A. Diaz, J. Clarke, N. Butowski, J. J. Phillips, S. M. Chang, M. S. Berger, D. A. Solomon
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="

 RNA homeostasis is dysregulated in oncogenesis, impacting cancer progression and therapy resistance. N6-methyladenosine (m6A), the predominant epitranscriptomic modification in eukaryotic mRNA, plays a crucial role in RNA biology, affecting transcript stability, translation, and splicing. These diverse effects suggest a complex interaction network among m6A machinery components, requiring integrative analyses for a profound understanding of methylated adenosine biology, associated enzymes, and therapeutic implications. Despite extensive research, the prognosis for relapsed patients with T-cell acute lymphoblastic leukemia (T-ALL) remains poor, with a median survival of only 4-8 months (Pui, C. H., Lancet, 2008). T-ALL, with its extensive chromatin and transcriptomic changes (Ntziachristos P. et al., Nature, 2014, Zhou Y. et al., Cancer Discovery, 2020), is an ideal model for studying RNA methylation.
 To characterize the m6A landscape in T-cell leukemia, we performed ELISA and enhanced UV crosslinking and immunoprecipitation (eCLIP) sequencing on human T-ALL patient samples and healthy thymic controls. Our findings reveal extensive global RNA methylation changes in the T-ALL transcriptome, primarily characterized by hypomethylation. Integration of gene expression and epitranscriptomics data showed a significant positive correlation between RNA methylation and gene expression changes in T-ALL. Hypermethylated transcripts are predominantly upregulated in T-ALL, while hypomethylation is associated with decreased expression. Enrichment analysis indicated that MYC-regulated transcripts and those involved in mevalonate/cholesterol biosynthesis (e.g., 3-hydroxy-3-methylglutaryl-CoA synthase 1 (HMGCS1)) are highly expressed and enriched with the m6A mark in leukemia, showing increased transcript and protein expression levels in T-ALL.
 Gene expression analysis, gene essentiality data from the Cancer Dependency Map, and our RNA-binding protein (RBP)-focused CRISPR/Cas9 screen identified that the m6A reader heterogeneous nuclear ribonucleoprotein C (HNRNPC) is highly expressed in T-ALL and critical for T-ALL survival compared or cancer types or other m6A readers. HNRNPC directly binds and regulates the MYC and cholesterol biosynthesis pathway transcripts. Silencing HNRNPC significantly reduced these pathways via decreased cell proliferation and increased apoptosis. To study m6A changes in the T-ALL transcriptome and dynamic HNRNPC positioning, we used METTL3 inhibitors in T-ALL cells. METTL3 inhibition with STM2457 (Yankova E. Nature, 2021) compromised global RNA methylation levels. Contrary to studies in AML, MYC(and the cholesterol biosynthesis pathway) showed hypermethylation coupled to increased HNRNPC binding, intensified translation rates, and elevated transcript and protein levels upon METTL3 inhibition. We further show that HNRNPC interacts with ribosomal proteins, further suggesting a potential role of the protein in translation. Ultimately, metabolomic analyses revealed increased intracellular cholesterol levels following METTL3 inhibition. Treatment with a newer inhibitor STM30006 (Guirguis, A.A. Cancer Discovery, 2023) showed a similar increase in MYC and the cholesterol pathway components.
 We then evaluated the levels of RNA methylation machinery components in T-cell leukemia to identify m6A-associated therapeutic vulnerabilities. We show that T-ALL cells present elevated FTO levels compared to physiological T cells or AML, due to direct transcriptional regulation by MYC. MYC also controls HNRNPC and the cholesterol biosynthesis pathway transcripts and inhibition of the cholesterol pathway using statins compromises MYC levels, suggesting a feedback loop in T-ALL. The use of the FTO inhibitors CS1 (bisantrene) and CS2 (brequinar, Su R. et al., Cancer Cell, 2020) for the treatment of cell lines, patient samples ex vivo, and a xenograft T-ALL model showed high efficacy of the drugs in inhibiting T-ALL. Further analyses revealed decreased m6A levels and repression of MYC and the cholesterol pathway upon FTO inhibition. Combining FTO inhibitors with dexamethasone and other chemotherapeutic agents demonstrated synergy in inhibiting T-ALL.
 These findings indicate that T-ALL exhibits dysregulated RNA methylation and demonstrate the potential of targeting HNRNPC and FTO as therapeutic strategies in T-ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fddd228de99db2e5ab10bf8b747cbd448df2607" target='_blank'>
              Hnrnpc and m6a RNA Methylation Control Oncogenic Transcription and Metabolism in T-Cell Leukemia
              </a>
            </td>
          <td>
            Jonas De Kesel, Igor Fijalkowski, T. Pieters, C. Borin, Marino Caruso, Wouter Sleeckx, Giulia Veltri, Nitesh Devinarayan Sharma, Kim De Keersmaecker, S. Goossens, Tom Taghon, V. Serafin, Bruno Palhais, Ksenia Matlawska-Wasowska, Ana Milovanovic, Eva Maria Novoa, Eric Wang, Panagiotis Ntziachristos
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aging constitutes complex and dynamic alterations in molecular and physiological processes and is associated with numerous disorders, in part due to increased genetic instability. The aging population is projected to double by 2050, underscoring the urgent need to better understand the relationships between aging and age-related disorders. Repetitive DNA elements are intrinsic sources of genetic instability and have been found to co-localize with mutation hotspots in human cancer genomes. In this study, we explored the relationship between aging and DNA repeat-mediated genetic instability in vivo using an H-DNA-forming mirror-repeat sequence from the cancer-associated human c-MYC gene. Utilizing a unique mutation-reporter mouse model, we observed tissue-specific effects of aging on H-DNA-induced genetic instability, with mutation frequencies increasing in spleen tissues and remaining unchanged in testis tissues. Analysis of the mutation spectra revealed large deletion mutations as the primary contributor to increasing H-DNA-induced mutations, supported by increased cleavage activity of H-DNA structures in aged spleen tissues. Our findings demonstrate that aging has distinct tissue-specific effects on repeat-mediated, cancer-associated mutations, providing insights into the complex relationship between aging and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8066191467ac61ad7ea1a2f5a55dbdea73e14d68" target='_blank'>
              Tissue-Specific Effects of Aging on Repeat-Mediated Mutation Hotspots In Vivo
              </a>
            </td>
          <td>
            Alexandra M. D’Amico, Tonia T. Li, Karen M. Vasquez
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="This study aimed to address new molecular events occurring in precancerous stages of colorectal cancer (CRC) by integrated analysis of gene expression data and DNA methylation profiles. Whole-transcriptome sequencing analysis was performed on 16 fresh frozen colorectal adenoma and matched mucosa specimens along with validation of candidates in an independent cohort of 20 fresh frozen paired adenoma and adjacent mucosa tissues as well as eight independent public datasets (335 cases). Genome-wide methylation profiles were determined for 5 adenoma pairs and confirmed by pyrosequencing on 20 tissue pairs used for validation. Functional analysis was performed in vitro and in vivo using the inflammation-associated azoxymethane/dextran sodium sulfate (AOM/DSS) and the sporadic colorectal carcinogenesis (six AOM injections) mouse models as well as ApcMin/+ mice. Candidates were investigated by immunohistochemical staining of human adenomas and early-stage (pT1) CRC tumors. A total of 1,917 differentially expressed genes and 148,191 differentially methylated CpG sites were detected in adenomas compared with adjacent mucosa samples. Based on the transcriptome data and relevance to CRC, TACSTD2, MMP7, MMP1, CLDN2, CLDN1, and ETV4, were selected for further validation. TACSTD2 promoter hypomethylation in adenoma tissues was validated in 20 additional tissue pairs and corresponded with increased TACSTD2 expression. TACSTD2 was also overexpressed in an in vitro transformation model of human colonic epithelial cells. The TACSTD2 protein TROP2 was elevated in human adenomas, pT1 tumors, and in murine adenomas, while it was absent in the unaffected adjacent mucosa. TROP2 overexpression might trigger the development of precancerous lesions and could help identify early transformation foci in colon biopsies. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf2c25521e6c5d07fbdce38e2d34466f87e5b71" target='_blank'>
              TACSTD2 expression marks the early transition to colon adenomas
              </a>
            </td>
          <td>
            Anna Siskova, K. Huebner, Giulio Ferrero, A. Gehring, Chuanpit Hampel, A. Naccarati, Barbara Pardini, S. Tarallo, Carol-Immanuel Geppert, Jan Prochazka, J. Tureckova, J. Jungwirth, T. Hucl, J. Kral, L. Vodickova, K. Erlenbach-Wuensch, A. Hartmann, Katerina Honkova, R. Matěj, P. Vodicka, Veronika Vymetálková, R. Schneider-Stock
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is a genetically diverse cancer of the bone marrow that affects approximately 20,000 new patients annually in the US alone. Many mutations drive leukemogenesis, including those in the core binding factor (CFB) complex, which is a heterodimeric transcription factor involved in hematopoietic cell development and differentiation (Pulikkan and Castilla 2018). Mutations in the CBF complex are present in 15% of adult AML cases and can affect either the alpha (RUNX1) or the beta (CBFβ) subunit through the chromosomal aberrations t(8;21) and inv(16), respectively. These mutations are necessary but not sufficient for leukemic initiation.
 CBF mutations co-occur with other mutations, including those affecting the cohesin complex, which is mutated in 10% of all AML cases (Ley et al 2013). The cohesin complex is comprised of four subunits, RAD21, SMC1A, SMC3, and STAG1/2, and regulates chromatin architecture and genomic looping, thereby affecting transcription. Mutations in the cohesin complex in AML affect only one subunit, resulting in haploinsufficiency of the complex and loss of function (Heimbruch et al. 2021).
 Despite CBF leukemias being classified as one group clinically, t(8;21) and inv(16) have different co-mutational profiles (Faber et al. 2016, Opatz et al. 2020, Duployez et al. 2016, Jahn et al. 2020, Qin et al. 2022). Interestingly, cohesin complex mutations have been observed more commonly in t(8;21) AML than in inv(16) AML. We therefore hypothesized that haploinsufficiency of cohesin would be detrimental to the development or maintenance of inv(16) AML. To test this, we utilized the inv(16) model published by Kuo et al. 2006 and the Smc3 haploinsufficiency model published by Viny et al. 2015. Both inv(16) and the loss of a single Smc3 allele are driven by MX1-Cre activation by PIPC injection.
 In contrast to our hypothesis, we found that inv(16);Smc3+/- mice developed AML with a shorter latency (14 weeks) than did inv(16);Smc3+/+ mice (20 weeks), suggesting the two mutations cooperate during leukemic initiation. ATAC sequencing of HSPCs revealed increased global accessibility in inv(16);Smc3+/- vs. inv(16);Smc3+/+ cells, particularly at Ets family transcription factor binding sites. qPCR and western blotting confirmed an upregulation the Ets family member Fli1 in inv(16);Smc3+/- cells. Upregulation of several known Fli1 target genes such as Ccng1, Ccl2, Mdm2, and Rab27b, was also observed. Interestingly, upregulation of Fli1 was not observed in leukemic cells from inv(16);Smc3+/- mice, suggesting cohesin mutation and Fli1 upregulation may contribute to leukemic initiation but not maintenance. In support of this, knockdown of Rad21 induced apoptosis in leukemias from inv(16);Smc3+/+ mice. We next performed single cell RNA sequencing on lin-, Sca1+, c-Kit+ (LSK) sorted cells from both genotypes to identify which cell types exhibited elevated Fli1. We found a significant upregulation of Fli1 in several clusters, primarily in LSKs, ST-HSCs, and MPPs, suggesting that Fli1 upregulation occurs in the most primitive stem cells. We are currently testing if Fli1 upregulation drives a shortened latency using inv(16);Smc3+/+ HSPCs.
 Cohesin mutations have not been readily detected in inv(16) AML, with current data primarily including patients of Chinese and German/Austrian origin (Faber et al. 2016, Opatz et al. 2020, Duployez et al. 2016, Jahn et al. 2020, Qin et al. 2022). Studies that have detected cohesin mutations in inv(16) patients have used a targeted sequencing approach in addition to whole genome or exome sequencing. Collectively, this suggests that cohesin mutations may co-occur with inv(16) at a higher rate than reported, which needs to be confirmed with targeted sequencing of a more diverse population. Another possibility is that cohesin mutations primarily affect the initiation of inv(16) AML but have a negative impact on maintenance. Therefore, selective pressure may no longer exist at the AML stage, leading to mutational loss. It would be interesting to examine the clonal evolution of healthy persons harboring inv(16) mutations in an effort to detect gain or loss of cohesin mutations. Our data support the hypothesis that cohesin mutations drive inv(16) AML initiation through the upregulation of Fli1 and its downstream targets in HSPCs and underline the importance of studying the effect of mutations on both AML initiation and maintenance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e41e1c9777cb8592358eaf5b18f857dcb9ed9d" target='_blank'>
              Cohesin Haploinsufficiency Drives Leukemic Initiation through Fli1 Upregulation in Inv(16) AML
              </a>
            </td>
          <td>
            Alison E Meyer, Katelyn E. Heimbruch, K. Pulakanti, C. Stelloh, Josiah Murray, J. A. Pulikkan, Sridhar Rao
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 GBM stands as the most aggressive brain tumor in adults. The median overall survival is less than 2 years despite conventional treatments. Recent advancements in DNA analysis have delineated the molecular characteristics of gliomas, leading to revisions in the WHO classification based on DNA profiles. However, the role of RNA modifications, epitranscriptomics, remains poorly understood in glioma. In this study, we underscore the significance of tRNA modifications targeting adenosine at position 37 in GBM. Transfer RNA isopentenyl transferase 1 (TRIT1) is identified as an isopentenyl transferase catalyzing the conversion of A into i6A at position 37 of tRNA, a nucleotide adjacent to the anticodon region. Additionally, i6A is further modified to 2-methylthio i6A (ms2i6A) by cyclin-dependent kinase 5 regulatory subunit associated-protein 1 (CDK5RAP1) within mitochondrial tRNA. Analysis of TCGA data reveals elevated TRIT1 mRNA expression levels in GBM compared to non-tumor samples. Furthermore, a low expression group of TRIT1 in gliomas demonstrates improved survival outcomes according to data from the CGGA, implicating TRIT1 in glioma progression. Functional assays involving TRIT1 knockdown in human GBM cell lines result in decreased cell viability, indicating its functional significance. Moreover, our data suggest that this phenotype may be mediated through translational control of selenoprotein W (SEPW1) via tRNA^(Ser)Sec modification. GO analysis of TRIT1 and SEPW1 in TCGA data further supports this proposed mechanism. At position 37 of tRNA, we also found that CDK5RAP1 in mitochondrial tRNAs is indispensable for sustaining glioblastoma initiating cell (GIC)-related traits. CDK5RAP1 maintains the self-renewal capacity, undifferentiated state, and tumorigenic potential of GICs, independent of translational control of mitochondrial proteins. Notably, CDK5RAP1 abrogates the antitumor effect of i6A by converting i6A to ms2i6A and shields GICs from excessive autophagy triggered by i6A. This work underscores the critical role of tRNA modifications in glioma pathogenesis. We will report these findings alongside RNA modification profile data and clinical data from glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4509570a8c26cde1777fa7d05efa5970d2e1b728" target='_blank'>
              DNAR-06. EPITRANSCRIPTOMIC TRNA MODIFICATIONS TARGETING ADENOSINE (A) AT POSITION 37 IN GBM
              </a>
            </td>
          <td>
            Takahiro Yamamoto, Jin Matsuura, Atsushi Fujimura, T. Chujo, Fan-Yan Wei, Naoki Shinojima, J. Kuroda, A. Mukasa, K. Tomizawa
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Aneuploidy, arising from gain or loss of chromosomes due to nondisjunction, is a special class of mutation. It can create significant phenotypic changes by altering abundance of hundreds of genes in a single event, providing material for adaptive evolution. But it can also incur large fitness costs relative to other types of mutations. Understanding mutational dynamics of aneuploidy is important for modeling its impact in nature, but aneuploidy rates are difficult to measure accurately. One challenge is that aneuploid karyotypes may revert back to euploidy, biasing forward mutation rate estimates - yet the rate of aneuploidy reversion is largely uncharacterized. Furthermore, current rate estimates are confounded because fitness differences between euploids and aneuploids are typically not accounted for in rate calculations. We developed a unique fluctuation assay in a wild-yeast model to measure the rate of extra-chromosome loss across three aneuploid chromosomes, while accounting for fitness differences between aneuploid and euploid cells. We show that incorporating fitness effects is essential to obtain accurate estimates of aneuploidy rates. Furthermore, the rate of extra-chromosome loss, separate from karyotype fitness differences, varies across chromosomes. We also measured rates in a strain lacking RNA-binding protein Ssd1, important for aneuploidy tolerance and implicated in chromosome segregation. We found no role for Ssd1 in the loss of native aneuploid chromosomes, although it did impact an engineered chromosome XV with a perturbed centromeric sequence. We discuss the impacts and challenges of modeling aneuploidy dynamics in real world situations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f543ecd0458b30a0801433f2e7a5623276dbbac" target='_blank'>
              On the rate of aneuploidy reversion in a wild yeast model.
              </a>
            </td>
          <td>
            James Hose, Qi Zhang, Nathaniel P. Sharp, A. Gasch
          </td>
          <td>2024-11-25</td>
          <td>Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The sine oculis homeobox homolog (SIX) family, a group of transcription factors characterized by a conserved DNA-binding homology domain, plays a critical role in orchestrating embryonic development and organogenesis across various organisms, including humans. Comprising six distinct members, from SIX1 to SIX6 , each member contributes uniquely to the development and differentiation of diverse tissues and organs, underscoring the versatility of the SIX family. Dysregulation or mutations in SIX genes have been implicated in a spectrum of developmental disorders, as well as in tumor initiation and progression, highlighting their pivotal role in maintaining normal developmental trajectories and cellular functions. Efforts to target the transcriptional complex of the SIX gene family have emerged as a promising strategy to inhibit tumor development. While the development of inhibitors targeting this gene family is still in its early stages, the significant potential of such interventions holds promise for future therapeutic advances. Therefore, this review aimed to comprehensively explore the advancements in understanding the SIX family within gastrointestinal cancers, focusing on its critical role in normal organ development and its implications in gastrointestinal cancers, including gastric, pancreatic, colorectal cancer, and hepatocellular carcinomas. In conclusion, this review deepened the understanding of the functional roles of the SIX family and explored the potential of utilizing this gene family for the diagnosis, prognosis, and treatment of gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e660aee580bc2a10b782aee723fba3778b722194" target='_blank'>
              Sine oculis homeobox homolog family function in gastrointestinal cancer: Progression and comprehensive analysis
              </a>
            </td>
          <td>
            Yang-Zheng Lan, Zheng-Jia Wu, Wen-Jia Chen, Xin-Ning Yu, Hua-Tao Wu, Jing Liu
          </td>
          <td>2025-01-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Meiotic crossover, i.e. the reciprocal exchange of chromosome fragments during meiosis, is a key driver of genetic diversity. Crossover is initiated by the formation of programmed DNA double-strand breaks (DSBs). While the role of ATAXIA-TELANGIECTASIA AND RAD3-RELATED (ATR) kinase in DNA damage signaling is well-known, its impact on crossover formation remains understudied. Here, using measurements of recombination at chromosomal intervals and genome-wide crossover mapping, we showed that ATR inactivation in Arabidopsis (Arabidopsis thaliana) leads to dramatic crossover redistribution, with an increase in crossover frequency in chromosome arms and a decrease in pericentromeres. These global changes in crossover placement were not caused by alterations in DSB numbers, which we demonstrated by analyzing phosphorylated H2A.X foci in zygonema. Using the seed-typing technique, we found that hotspot usage remains mainly unchanged in atr mutants compared with wild-type individuals. Moreover, atr showed no change in the number of crossovers caused by two independent pathways, which implies no effect on crossover pathway choice. Analyses of genetic interaction indicate that while the effects of atr are independent of MMS AND UV SENSITIVE81 (MUS81), ZIPPER1 (ZYP1), FANCONI ANEMIA COMPLEMENTATION GROUP M (FANCM), and D2 (FANCD2), the underlying mechanism may be similar between ATR and FANCD2. This study extends our understanding of ATR's role in meiosis, uncovering functions in regulating crossover distribution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacde0c3932254585c19d65235879a141c6f4b4" target='_blank'>
              The kinase ATR controls meiotic crossover distribution at the genome scale in Arabidopsis
              </a>
            </td>
          <td>
            Longfei Zhu, J. Dluzewska, N. Fernández-Jiménez, Rajeev Ranjan, Alexandre Pelé, Wojciech Dzięgielewski, Maja Szymańska-Lejman, Karolina Hus, Julia Górna, M. Pradillo, Piotr A. Ziolkowski
          </td>
          <td>2024-10-29</td>
          <td>The Plant Cell</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Breast cancer is associated with high morbidity and mortality, which are closely influenced by protein post-translational modifications (PTMs). Lysine crotonylation (Kcr) serves as a newly identified PTM type that plays a role in various biological processes; however, its involvement in breast cancer progression remains unclear. Minichromosome maintenance 6 (MCM6) is a critical component of DNA replication and has been previous confirmed to exhibit a significant role in tumorigenesis. Despite this, a comprehensive analysis of MCM6, particularly regarding its modifications in breast cancer is lacking. In this study, we found MCM6 is upregulated in breast invasive carcinoma (BRCA) and is associated with poorer overall survival by regulating the DNA damage repair mechanisms. Furthermore, MCM6-knockdown resulted in decreased cell proliferation and inhibited the DNA replication, leading to DNA replication stress and sustained DNA damage, thereby enhancing the chemotherapeutic sensitivity of breast cancer. Additionally, SIRT7-mediated crotonylation of MCM6 at K599 (MCM6-K599cr) was significantly upregulated in response to DNA replication stress, primarily due to the disassemebly of the MCM2-7 complex and regulated by RNF8-mediated ubiquitination. Concurrently, kaempferol, which acts as a regulator of SIRT7, was found to enhance the Kcr level of MCM6, reducing tumour weight, particular when combined with paclitaxel, highlighting its potential chemotherapeutic target for BRCA therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc331419a4f2d077914bda931ebc6dc0480b9e3e" target='_blank'>
              Crotonylation of MCM6 enhances chemotherapeutics sensitivity of breast cancer via inducing DNA replication stress.
              </a>
            </td>
          <td>
            Haoyun Song, Zhao Guo, Kun Xie, Xiangwen Liu, Xu Yang, Rong Shen, Degui Wang
          </td>
          <td>2024-10-30</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b3b6f34b71d4b5f3b8e63dfe568f53d1e1b13e" target='_blank'>
              Stabilization of expandable DNA repeats by the replication factor Mcm10 promotes cell viability
              </a>
            </td>
          <td>
            Chiara Masnovo, Zohar Paleiov, Daniel Dovrat, Laurel K Baxter, Sofia Movafaghi, A. Aharoni, S. Mirkin
          </td>
          <td>2024-12-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Epigenetics plays a pivotal role in regulating gene expression and cellular differentiation. DNA methylation, involving the addition of methyl groups to specific cytosine bases, is a well-known epigenetic modification. The recent discovery of 5-hydroxymethylcytosine (5hmC) has provided new insights into cytosine modifications. 5hmC, derived from the oxidation of 5-methylcytosine (5mC), serves as both an intermediate in demethylation and a stable chemical modification in the genome. In this comprehensive review, we summarize the recent research advancements regarding the functions of 5hmC in development and disease. We discuss its implications in gene expression regulation, cellular differentiation, and its potential role as a diagnostic and prognostic marker in various diseases. Additionally, we highlight the challenges associated with accurately detecting and quantifying 5hmC and present the latest methodologies employed for its detection. Understanding the functional role of 5hmC in epigenetic regulation and further advancing our understanding of gene expression dynamics and cellular processes hold immense promise for the development of novel therapeutic strategies and precision medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93e3a1b98db15f76b85238979bccf5d238d99cf3" target='_blank'>
              5-Hydroxymethylcytosine: Far Beyond the Intermediate of DNA Demethylation
              </a>
            </td>
          <td>
            Kaixi Zheng, Zhengbing Lyu, Jianqing Chen, Guo‐Hsian Chen
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80a71e43761517adb11c79a0e4819178fbabb90" target='_blank'>
              Circular RNA circABCC4 as the ceRNA facilitates renal carcinoma progression.
              </a>
            </td>
          <td>
            Yongjie Yao, Tianchen Wang, Shilun Li, Qixiang Song, Kai Yuan
          </td>
          <td>2024-10-30</td>
          <td>World journal of urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Drosophila female germline stem cells (GSCs) complete asymmetric mitosis in the presence of an intact, but permeable, nuclear envelope and nuclear lamina (NL). This asymmetric division requires a modified centrosome cycle, wherein mitotic centrosomes with mature pericentriolar material (PCM) embed in the NL and interphase centrosomes with reduced PCM leave the NL. This centrosome cycle requires Emerin, an NL protein required for GSC survival and germ cell differentiation. In emerin mutants, interphase GSC centrosomes retain excess PCM, remain embedded in the NL and nucleate microtubule asters at positions of NL distortion. Here, we investigate the contributions of abnormal interphase centrosomes to GSC loss. Remarkably, reducing interphase PCM in emerin mutants rescues GSC survival and partially restores germ cell differentiation. Direct tests of the effects of abnormal centrosomes were achieved by expression of constitutively active Polo kinase to drive enlargement of interphase centrosomes in wild-type GSCs. Notably, these conditions failed to alter NL structure or decrease GSC survival. However, coupling enlarged interphase centrosomes with nuclear distortion promoted GSC loss. These studies establish that Emerin maintains centrosome structure to preserve stem cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dcda208a870f5adea9fc32fe1189dfff38a108a" target='_blank'>
              Emerin preserves stem cell survival through maintenance of centrosome and nuclear lamina structure
              </a>
            </td>
          <td>
            Samuel D Jones, Jack E B Miller, Madilynn M Amos, Julianna M Hernández, Katherine M Piaszynski, Pamela K. Geyer
          </td>
          <td>2024-10-28</td>
          <td>Development (Cambridge, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Following eukaryotic genome replication, the two newly synthesised sister chromatids remain paired by the ring-shaped cohesin complex, enabling their faithful segregation to daughter cells during cell divisions1. Cohesin topologically embraces DNA already before DNA replication2–5, and replisome passage through the cohesin ring is thought of as a fail-safe mechanism ensuring that cohesin entraps both replication products6,7. Whether replisomes indeed pass through cohesin rings remains unknown. Here, we use biochemical reconstitution4,8,9 and single molecule fluorescence microscopy to directly visualise replisome-cohesin encounters. We find that the translocating eukaryotic replicative Cdc45-Mcm2-7-GINS (CMG) helicase is frequently an obstacle to cohesin, but that the likelihood with which CMG passes cohesin increases in the presence of replisome components with known sister chromatid cohesion functions7,10–14, or by preventing cohesin from freely sliding along the template DNA. Cohesin retains topological DNA entrapment during CMG passage, suggesting that the helicase passes through the ring. The passage frequency increases further when fully reconstituted replisomes encounter cohesin rings, resulting in successful establishment of cohesion between the two replication products. Our findings demonstrate the existence of a simple mechanism that links genome replication with chromosome segregation, replisome passage through cohesin rings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3cb5799fad1801629c2f96c5a944f38743e23e2" target='_blank'>
              Replisome passage through the cohesin ring
              </a>
            </td>
          <td>
            Samson Glaser, Maxim I Molodtsov, J. Diffley, F. Uhlmann
          </td>
          <td>2024-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Infection by retroviruses and the mobilization of transposable elements cause DNA damage that can be catastrophic for a cell. If the cell survives, the mutations generated by retrotransposition may confer a selective advantage, although, more commonly, the effect of new integrants is neutral or detrimental. If retrotransposition occurs in gametes or in the early embryo, it introduces genetic modifications that can be transmitted to the progeny and may become fixed in the germline of that species. PIWI-interacting RNAs (piRNAs) are single-stranded, 21–35 nucleotide RNAs generated by the PIWI clade of Argonaute proteins that maintain the integrity of the animal germline by silencing transposons. The sequence specific manner by which piRNAs and germline-encoded PIWI proteins repress transposons is reminiscent of CRISPR, which retains memory for invading pathogen sequences. piRNAs are processed preferentially from the unspliced transcripts of piRNA clusters. Via complementary base pairing, mature antisense piRNAs guide the PIWI clade of Argonaute proteins to transposon RNAs for degradation. Moreover, these piRNA-loaded PIWI proteins are imported into the nucleus to modulate the co-transcriptional repression of transposons by initiating histone and DNA methylation. How retroviruses that invade germ cells are first recognized as foreign by the piRNA machinery, as well as how endogenous piRNA clusters targeting the sequences of invasive genetic elements are acquired, is not known. Currently, koalas (Phascolarctos cinereus) are going through an epidemic due to the horizontal and vertical transmission of the KoRV-A gammaretrovirus. This provides an unprecedented opportunity to study how an exogenous retrovirus becomes fixed in the genome of its host, and how piRNAs targeting this retrovirus are generated in germ cells of the infected animal. Initial experiments have shown that the unspliced transcript from KoRV-A proviruses in koala testes, but not the spliced KoRV-A transcript, is directly processed into sense-strand piRNAs. The cleavage of unspliced sense-strand transcripts is thought to serve as an initial innate defense until antisense piRNAs are generated and an adaptive KoRV-A-specific genome immune response is established. Further research is expected to determine how the piRNA machinery recognizes a new foreign genetic invader, how it distinguishes between spliced and unspliced transcripts, and how a mature genome immune response is established, with both sense and antisense piRNAs and the methylation of histones and DNA at the provirus promoter.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f261264317e219034ce10344e394b40699b58a3" target='_blank'>
              piRNA Defense Against Endogenous Retroviruses
              </a>
            </td>
          <td>
            Milky Abajorga, Leonid A Yurkovetskiy, Jeremy Luban
          </td>
          <td>2024-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="RNA methylation adds a second layer of genetic information that dictates the post-transcriptional fate of RNAs. Although various methods exist that enable the analysis of RNA methylation in a site-specific or transcriptome-wide manner, whether biophysical approaches can be employed to such analyses is unexplored. In this study, Fourier-transform infrared (FT-IR) and circular dichroism (CD) spectroscopy are employed to examine the methylation status of both synthetic and cellular RNAs. The results show that FT-IR spectroscopy is perfectly capable of quantitatively distinguishing synthetic m6A-methylated RNAs from un-methylated ones. Subsequently, FT-IR spectroscopy is successfully employed to assess the changes in the extent of total RNA methylation upon the knockdown of the m6A writer, METTL3, in HeLa cells. In addition, the same approach is shown to accurately detect reduction in total RNA methylation upon the treatment of HeLa cells with tumor necrosis factor alpha (TNF-α). It is also demonstrated that m1A and m6A methylation induce quite a distinct secondary structure on RNAs, as evident from CD spectra. These results strongly suggest that both FT-IR and CD spectroscopy methods can be exploited to uncover biophysical properties impinged on RNAs by methyl moieties, providing a fast, convenient and cheap alternative to the existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5d73f7a71f859cb2104e3d803e1a6ec86a1c0d" target='_blank'>
              An Investigation of RNA Methylations with Biophysical Approaches in a Cervical Cancer Cell Model
              </a>
            </td>
          <td>
            Buket Sağlam, Onur Akkuş, Azime Akçaöz-Alasar, Çağatay Ceylan, Günnur Güler, B. Akgül
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 Chronic lymphocytic leukemia (CLL), one of the most common types of leukemia among adults in the US, is characterized by the accumulation of CD5+CD19+ monoclonal B cells. Current clinical treatment approaches for progressive CLL patients focus on novel agents targeting Bruton Tyrosine Kinase (BTK) and Bcl-2. Although clinical trials combining novel agent have shown significant efficacy with high overall response and, in some cases, achievement of high rates of undetectable measurable residual disease (MRD), CLL remains incurable with a majority of patients eventually experiencing relapse of their disease. Therefore, it remains of high interest to expand therapeutic options.
 Transcriptional chemical inducers of proximity (TCIPs) are small molecules designed and developed by Dr. Gerald R. Crabtree at Stanford University. TCIPs are a new class of anticancer drugs that recruit endogenous cancer drivers or a downstream transcription factor to the promoters of cell death genes and activate their expression, resulting in apoptosis of cancer cells. Current TCIPs are produced by covalently linking small molecules that bind B-cell lymphoma 6 (BCL6) to those that bind to transcriptional activators, such as Bromodomain-containing protein 4 (BRD4) or cyclin-dependent kinases 9 (CDK9). The Stanford team has demonstrated TCIPs generate remarkable killing in diffuse large B cell lymphoma (DLBCL) cell lines, including chemotherapy-resistant, TP53-mutant lines, with a half maximal effective concentration (EC50) of 1-10nM at 72hours (h) and exhibited cell-specific and tissue-specific effects (Gourisankar et al, 2023).
 We have initiated TCIPs cytotoxicity assessment in primary leukemic cells from untreated CLL patients. Testing two types of TCIPs with a flow cytometry-based apoptosis assay, we found time and dose-dependent drug-associated killing in primary CLL B cells collected from untreated CLL patients over a 24-72h interval. The IC50 for BRD4-BCL6 TCIP JWZ-7-133 at 72h was 560nM while CDK9-BCL6 TCIP BAK-04-232 at 72h was 10nM. After 24, 48, and 72h exposure of CLL B cells to TCIP JWZ-7-133, the IC50 was 1050, 820 and 560nM, respectively. We also found that TCIP JWZ-7-133 exhibited a similar ability to inhibit leukemic cell viability with an IC50 of 1070nM using a cell viability assay, compared to a flow cytometry-based cell death assay (24h IC50 1050nM).
 CDK9-BCL6 TCIPs are designed and produced by covalently linking BI-3812 (binding to BCL6) and SNS-032 (binding to CDK9) with a rigid linker. BI-3812 is a potent and efficacious BCL6 inhibitor, inhibiting the BTB domain of BCL6, and SNS-032 is a potent and selective inhibitor of CDK9. When CLL B cells were treated with only BI-3812 or SNS-032 or TCIP BAK-04-232 as single agents for 72h, we found that TCIP BAK-04-232 had superior and significant killing effects compared to the single agents, with an IC50 of 10nM. Specifically this killing activity was more than 10-fold compared to SNS-032 alone (IC50 over 100nM), and more than 1,000-fold compared to BI-3812 alone (IC50 over 10,000nM).
 There remains an urgent to identify new categories of potent drug, inhibitor, or small molecules to expand potential clinical treatment options for patients with CLL. The results presented here indicate that TCIPs targeting BCL6 and BRD4/CDK9 have a strong capacity to induce high levels of cell death in CLL B cells at nanomolar concentrations, and this remarkable killing effect may be due to its unique gain-of-function drug mechanism. TCIPs provide a completely new drug direction and strategy for the design of anticancer drugs: to rewire cancer drivers to activate apoptosis in cancer cells. This study is the first of a kind demonstration of TCIPs in robustly targeting CLL B cells. Further studies to fully validate the preclinical activity of TCIPs and investigate the molecular mechanism(s) for TCIP mediated killing of leukemic B cells will help to define the role of TCIPs as a novel therapeutic approach in CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75bfc360f4f5a8565f53d51b915602b5543c4845" target='_blank'>
              Rewiring Cancer Drivers to Induce Apoptosis By Transcriptional Chemical Inducers of Proximity in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Weiguo Han, Aisharya Bhattacharya, Wenzhi Ji, Basel Karim, Roman C. Sarott, Meredith Nix, S. Gourisankar, Andrey Krokhotin, Sutapa Sinha, Zhiquan Wang, T. Shanafelt, S. Parikh, N. Gray, G. R. Crabtree, Neil E. Kay
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="

 Introduction
 The major primary genetic events in multiple myeloma (MM) include translocations of immunoglobulin heavy chain (IgH) and a hyperdiploidy (HRD) karyotype (Kuehl, 2002). The main drivers of pathogenesis in HRD MM are unknown (Fonseca, 2003), therefore further research is needed in this subgroup. Translational research in HRD MM has relied on analysis of genomic and transcriptomic alterations, however changes at the genome and transcriptome level do not always correlate directionally with protein abundance. Because proteins are considered the main and final effectors of most cellular processes, proteogenomics can improve our understanding of the drivers of HRD MM. We employed it here to identify potential biomarkers in HRD MM and determine whether any given trisomy may lead to a selective advantage.
 Methods:
 Protein and mRNA were extracted from CD138+ cells from 47 HRD MM and 14 t(11;14) MM samples. RNA-seq data was processed using MAPR-Seq pipeline using default parameters and gene-wise raw counts were extracted for each sample. Proteomics data was processed using Specranaut and protein-wise intensities were extracted for each sample. A generalized linear model configured with negative binomial distribution was used to analyze gene-wise count data in edgeR software. Genes with an adjusted p-value of <=0.05 were considered significantly differentially expressed. Protein intensity data were normalized using trimmed mean of M-values and log2 transformed. A generalized linear model configured with Gaussian distribution was used to compare protein intensities between the groups. Proteins with an adjusted p-value <= 0.05 were considered significantly differentially expressed. Two separate rank lists for protein and mRNA data were generated for pathway analysis.
 Results
 When comparing our small cohort of HRD MM patient samples to those with t(11;14) MM, out of approximately 16,600 total transcripts and 9,700 proteins evaluated we identified approximately 2,100 significantly variant transcripts and 500 significantly variant proteins in HRD MM. GSEA pathway analysis of the transcriptome and proteome revealed significantly upregulated translational and ribosomal processing pathways in HRD MM. Transcriptional MYC target pathways were also significantly upregulated. RNAseq revealed significantly higher expression of MYC (Rank 24.5, p=5.3E-6) in HRD MM. It is known that patients with t(11;14) MM have increased expression of CCND1 (Kuehl and Bergsagel, 2012). As expected, we found that the HRD MM subgroup had significantly lower expression of CCND1 compared to the t(11;14) MM subgroup (Rank -79.9, p=2.8E-21) and this was also true at the protein level (Rank -21.7, p=2.2E-5). The most significantly abundant protein in the HRD MM subgroup compared to t(11;14) MM was BAHD1, which is a protein that targets H3K27me3 and recruits transcriptional co-repressors (Fan, 2020). BAHD1 has been shown to repress erythroid genes and may recruit HDAC5, which plays a role in osteoblast differentiation (Rank 72.3, p=2.9E-20; Xu, 2021). The second most abundant protein in HRD MM was CKS2, which binds to the catalytic subunit of cyclin-dependent kinases (CDKs) and therefore positively regulates cell proliferation, invasion, and migration and has been shown to be implicated in tumor progression in several solid tumors (Rank 72.3, p=2.9E-20; You, 2015). TRIM proteins promote tumorigenesis and cancer development. TRIM14 mediates cancer development through mediating the JAK/STAT, PI3K/AKT, NFkB, and p53 pathways and was found to be significantly higher in HRD MM (Rank 70.6, p=9.0E-20; Huang, 2022). We also found increased protein abundance of LAMP5 in HRD MM (Rank 20.1, p=6.0E-5). LAMP5 may play an important role in MM progression and has been shown to be highly expressed in patients with mixed-lineage leukemia and is associated with a poor prognosis (Wang, 2023; Gracia-Maldonado, 2022). We identified many proteins (not all discussed here) that have not been well described in HRD MM and thus may be biomarkers/targets of disease pathogenesis of this subgroup.
 Conclusion
 Proteins are the final effectors of most cellular processes. Proteogenomics can improve our understanding of the drivers of HRD MM. Using a proteogenomic approach we identified proteins that may lead to selective advantage and/or pathogenesis of trisomies in HRD MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/276665eba54a5425c632c52984a42df1abd8f60c" target='_blank'>
              A Proteogenomic Approach to Identify Potential Drivers of Hyperdiploid Multiple Myeloma
              </a>
            </td>
          <td>
            J. Wiedmeier-Nutor, Surendra Dasari, Richard K Kandasamy, Kiran Manalaparthi, L. Bruins, G. Ahmann, M. Arribas, P. Bergsagel, Akhilesh Pandey, E. Braggio, Rafael Fonseca
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Non-recombining regions of the genome often have profound effects on evolution, resulting in phenomena such as sex chromosomes and supergenes. Amongst the strangest examples are balanced lethal systems, such as that found in newts of the genus Triturus. These systems halve reproductive output, and the evolution of such a deleterious trait is difficult to explain. For Triturus an intriguing model proposes that the balanced lethal system evolved from an ancestral Y-chromosome. To test this hypothesis, we identify the Y-chromosome of Triturus and verify whether it, or the balanced lethal system, is homologous to the Y-chromosome of its sister genus Lissotriton, which does not possess the balanced lethal system. We identify a set of candidate Y-linked markers in T. ivanbureschi and place them on a high-density linkage map that we construct with 7,233 RADseq markers. We validate male specificity of the markers across the genus, and then place both the Triturus and Lissotriton Y-linked regions within previously constructed target capture linkage maps that include genes linked to the balanced lethal system. We observe that neither the Triturus balanced lethal system, nor the Triturus Y-chromosome are homologous to the Lissotriton Y-chromosome. This is the first molecular evidence of a transition between Y-chromosome systems within salamanders. However, unless additional sex chromosome turnover events are involved, our data does not support a sex chromosome origin of the balanced lethal system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0761b7931e7391a5022fda139613d3d74ab441" target='_blank'>
              Identification of Y-chromosome turnover in newts fails to support a sex chromosome origin for the Triturus balanced lethal system
              </a>
            </td>
          <td>
            James France, W. Babik, Milena Cvijanović, K. Dudek, Ana Ivanović, Tijana Z. Vučić, B. Wielstra
          </td>
          <td>2024-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="

 Introduction
 Acute myeloid leukemia (AML) with mutated TP53 (TP53MT) is resistant to chemotherapy and has a poor prognosis. Prior reports suggest that DNA methyltransferase inhibitors (DNMTi), such as decitabine and azacitidine, form covalent DNA-DNMT1 adducts and invoke a DNA damage response (DDR) characterized by activation of the ATR-CHK1 pathway, including in TP53MT AML samples. This study evaluates whether decitabine exhibits cytotoxic synergy with replication checkpoint modulators (ATRi, CHK1/2i, WEE1i) in pre-clinical models of AML.
 Methods
 Parental cell lines (U937, KG1a, Molm13, MV-4-11) were obtained commercially and genomically validated. The Molm13 TP53-/- cell line was generated via CRISPR/Cas9 knockout. Cells were maintained in RPMI-1640 medium with 10% FBS, 2 mM glutamine, and antibiotics at 37 ºC, 5% CO2. Decitabine, ceralasertib (ATRi), prexasertib (CHK1/2i), MK-8776 (CHK1i), and adavosertib (WEE1i) were reconstituted at 10 mM in DMSO. For drug treatments, cells were cultured for 48 h in medium with 0.2% (v/v) DMSO or drug. Western blots were stained with commercial antibodies followed by peroxidase-coupled secondary antibodies and imaged by chemiluminescence. Apoptosis assays were performed via flow cytometry (BD FACSCanto II) with annexin V-APC (BD Biosciences) and/or propidium iodide (Sigma) staining. Peripheral blood and bone marrow samples were obtained from AML patients after informed consent at Mayo Clinic and Johns Hopkins University. Mononuclear cells were isolated via Histopaque-1077 (Sigma) centrifugation and used immediately. Colony forming assays were performed in Methocult H4435 (Stem Cell Technologies) supplemented with vehicle or drug; leukemic colonies were counted after 12-16 days. Combination indices (CI) were determined by the Chou-Talalay method (CalcuSyn) where values <1 indicate synergy and >1 antagonism. This work was supported in part by R01 CA289285.
 Results
 Western blots of TP53MT AML cell lines (U937, KG1a) and primary samples treated with decitabine (≤200 nM) for 24 h demonstrated increased phosphorylation of H2AX and S296/S345 CHK1, consistent with activation of the ATR-CHK1 DDR pathway. These changes were less pronounced in TP53 wild type cell lines (Molm13, MV-4-11).
 In subdiploid (SubG1) assays, U937 cells exhibited resistance to monotherapy with decitabine (EC50 = 330 ± 200 nM) and variable sensitivity to ceralasertib (250 - 500 nM), prexasertib (1 - 10 nM), MK-8776 (250 - 1,000 nM), and adavosertib (50 - 400 nM). When combined, however, marked synergy was observed between decitabine and both ceralasertib (mean CI 0.15) and adavosertib (CI 0.59). Synergy was also observed with the selective CHK1i, MK-8776 (CI 0.56), but less-so with the dual CHK1/2i, prexasertib (CI 0.93), consistent with the hypothesis of an ATR-CHK1 DNA damage response. Similar results were obtained from decitabine (EC50 = 300 ± 100 nM) and ceralasertib (CI 0.54), adavosertib (CI 0.67), MK-8776 (CI 0.52), and prexasertib (CI 0.81) in annexin V assays.
 These combinations were further evaluated (SubG1 assays) in an isogenic pair of Molm13 and Molm13 TP53-/- cell lines. In Molm13 cells, decitabine (EC50 = 23 ± 11 nM) was again synergistic with adavosertib (CI 0.60), but not ceralasertib (CI 1.3) or prexasertib (CI 1.2). In Molm13 TP53-/- cells, however, very little synergy was observed between decitabine (EC50 = 53 ± 22 nM) and ceralasertib (CI 1.5), prexasertib (CI 4.3), or adavosertib (CI 1.0) except for at higher concentrations of adavosertib (400 nM, CI 0.89). Thus, while biallelic TP53 inactivation conferred resistance to DNMTi and checkpoint inhibitors, the degree of resistance was context-dependent and could be partially overcome by WEE1 inhibition.
 The combinations of decitabine with ceralasertib (n = 2), prexasertib (n = 4), or adavosertib (n = 3) were evaluated in colony forming assays using primary AML samples from newly-diagnosed patients with diverse clinicomolecular profiles. Combination treatment was antagonistic with ceralasertib (CI 2.2), additive with prexasertib (CI 1.0), and synergistic with adavosertib (CI 0.88).
 Conclusions
 In multiple cell lines and diverse primary AML samples, decitabine activated the ATR-CHK1 DDR pathway and synergized with ATRi, CHK1/2i, and WEE1i. Amongst these, only the combination of decitabine plus adavosertib exhibited synergy across all three models, highlighting WEE1 as a potential target in TP53MT AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7efdc400ff1ad0e1032ed03786acd8bf148ae20d" target='_blank'>
              Decitabine Synergizes with Replication Checkpoint Inhibitors in TP53-Mutated Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Clifford M. Csizmar, Antoine N Saliba, Kevin L. Peterson, X. Meng, B. D. Smith, G. Ghiaur, M. Patnaik, Scott H. Kaufmann
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="

 The ribosomal protein RPS15 is recurrently mutated in chronic lymphocytic leukemia (CLL) and confers adverse prognosis. While translational rewiring is a recognized consequence of RPS15 mutation (RPS15-mut), its impact on genome stability and the precise mechanisms governing RPS15-mut driven B-cell leukemogenesis remain unclear. To delineate the sequence of cellular events that underlie malignant transformation and progression in the context of RPS15 mutation, we functionally interrogated a conditional knock-in mouse model of the RPS15-S138F mutation established using the Cd19-Cre/loxP system (Gutierrez et al, ASH 2020). Despite enrichment for Myc target genes in pre-leukemic Rps15-mutB splenocytes, these cells displayed a hypoproliferative phenotype characterized by increased G1 cell cycle checkpoint activation and reduced ex vivo proliferative capacity accompanied by increased apoptosis, compared to Rps15-WT splenocytes.
 We hypothesized that decreased proliferation and increased apoptosis could result from a p53-mediated response to cellular stress arising from Rps15 mutation. Informed by the genomic landscape of CLL developing in older Rps15-mut mice that displayed an enrichment of A•T>C•G and C•G>T•A transversions indicative of peroxide-induced DNA damage, we interrogated the transcriptomes of Rps15-mutpre-leukemic B splenocytes, which showed upregulation of reactive oxygen species (ROS) genes (NES 1.5, p<0.05). We experimentally quantified mitochondrial ROS which confirmed increased oxidative stress in Rps15-mut compared to Rps15-WT splenocytes (p<0.001). We reasoned that unresolved oxidative stress could lead to DNA damage accumulation. Accordingly, we demonstrated increased γH2AX accumulation, a marker of DNA damage, in Rps15-mut B splenocytes compared to their Rps15-WT counterparts (p=0.01), as well as hypersensitivity of Rps15-mut splenocytes to oxidative stress overload with parthenolide (p=0.0009). RPS15 is known to stabilize p53 via MDM2 inhibition, hence Rps15-mut cells exhibited reduced p53 expression. Nevertheless, in response to DNA damage, p53 and p21 were significantly induced, the latter a p53-dependent mediator of G1 checkpoint activation, highlighting the functional importance of residual p53 activity in Rps15-mut B cells.
 Given their heightened level of DNA damage, we further hypothesized that Rps15-mut pre-leukemic B splenocytes are hyper-dependent on DNA damage response (DDR). Conversely, defective DDR could precipitate genomic instability and facilitate acquisition of additional genomic alterations, such as TP53 deletion or loss-of-function mutation, that might promote transition toward a hyperproliferative or malignant state. To determine whether Rps15-mut B splenocytes could effectively respond to further genomic insults, we assessed cellular response to hydroxyurea (HU) and ionizing radiation (IR) that induce replication stress and DNA double-strand breaks (DSBs) respectively. While the ATR-Chk1 response to HU was preserved, the ATM-Chk2 response to IR was defective with diminished phosphorylation of downstream ATM targets and impaired induction of G2/M arrest. Indeed, Rps15-mut B cells subsequently acquired a spectrum of additional mutations and chromosomal alterations (e.g. Myc/Mapk amplification, Trp53 loss) that reflect genomic instability secondary to defective DDR.
 Finally, we postulated that translational defects directly contribute to oxidative DNA damage accumulation and genomic instability in Rps15-mut B cells. We therefore performed ribosome profiling that showed differential translation efficiency (TE) in 342 genes between Rps15-mutand Rps15-WT B splenocytes, with significant enrichment of DNA replication, DDR and cell cycle genes by GSEA that was recapitulated in RPS15-mut vs WT isogenic HG3 CLL cell lines. In particular, Gpx1, a glutathione peroxidase that functions as a key cellular antioxidant, was confirmed by western blot to be downregulated in Rps15-mut B cells secondary to reduced TE (0.6-fold change vs WT, p=0.018), alongside other proteins important for the amelioration of oxidative DNA damage and DSBs (e.g. Cyb5r4, Otud4, Fance, Tlk1). Our analyses thus link RPS15-induced translational alterations with a cascade of cellular defects that compromise genome stability and highlight the critical nature of these defects in B-cell transformation and CLL progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b6feb4689d0ca7cd852af6a8ea529fdb6b2609" target='_blank'>
              Impaired Response to Oxidative DNA Damage Underlies Genomic Instability in B-Cell Leukemia Driven By RPS15 Mutation
              </a>
            </td>
          <td>
            M. Kwok, Catherine Gutierrez, Neil Ruthen, Peyton Waddicor, Tamara Ouspenskaia, Christina Curran, Doris Fu, B. Knisbacher, Anat Biran, Adi Nagler, Aviv Liani, Sarah Chen, G. Lazarian, Elizabeth Witten, Gabriela Brunsting Hoffmann, Emma Lin, Lucas Pomerance, Shuqiang Li, Lili Wang, K. Livak, D. Neuberg, G. Getz, Elisa ten Hacken, Catherine J. Wu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096d8a8360c9f47bd2a3b661c0d89b3fc08440ad" target='_blank'>
              Structural basis for human OGG1 processing 8-oxodGuo within nucleosome core particles
              </a>
            </td>
          <td>
            Mengtian Ren, F. Gut, Yilan Fan, Jingke Ma, Xiajing Shan, Aysenur Yikilmazsoy, Mariia Likhodeeva, K. Hopfner, Chuanzheng Zhou
          </td>
          <td>2024-10-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy.
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Background: B-cell-derived acute lymphoblastic leukemia (B-ALL) is a type of blood cancer characterized by the overproduction of immature lymphocytes and is the most common cancer in children. The current overall survival (OS) rates for children with B-ALL are nearly 90% when diagnosed with good- and intermediate-risk subtypes. However, OS rates for children diagnosed with high-risk subtypes, such as KMT2A-Rearranged (KMT2A-R) and Philadelphia-like (Ph-like), are significantly lower, highlighting the urgent need to identify critical leukemogenic pathways for targeted therapeutic interventions. SMARCA4 is part of the SWI/SNF chromatin-remodeling complex, which is crucial for regulating gene expression by altering chromatin structure. SMARCA4 is mutated in about 10% of non-small cell lung cancer patients, resulting in significantly lower OS. More recently, SMARCA4 has been identified as essential for early B cell development, identity, and growth. However, its importance in ALL has not been studied.
 Results: We performed single-cell DNA sequencing experiments paired with meta-analyses of whole exome and whole genome sequencing data from 1717 patients with B-ALL. We found that SMARCA4 is only mutated in 0.35% of patients with B-ALL, suggesting that, unlike in non-small cell lung cancer, SMARCA4 mutations may not benefit ALL growth. Gene expression analysis of B cells at different developmental stages demonstrated that SMARCA4 is specifically upregulated during early B cell development (early pro-B to small pre-B stages), overlapping with the stages where ALL cells are arrested. Direct comparisons of human pre-B cells, B-ALL, CD34+, and T-ALL showed that pre-B cells and B-ALL cells have the highest SMARCA4 expression levels, indicating a potential dependency of B-ALL on SMARCA4. Analysis using the DepMAP portal confirmed that B-ALL cells are highly dependent on SMARCA4, confirming that a SMARCA4 inhibitor-based therapy may be beneficial for children with B-ALL. However, survival analysis of 207 children from the COG trial P9906 indicated no impact on patient outcomes from varying SMARCA4 expression levels. When we separated the patients based on their B-ALL subtype, we found that SMARCA4 was significantly highly expressed in children with Ph-like ALL than in those with KMT2A-R B-ALL, which we confirmed via Western blotting using Ph-like and KMT2A-R ALL cell lines. A survival analysis focused on these subtypes revealed that poor outcomes correlated with higher SMARCA4 expression in Ph-like ALL, while, surprisingly, high SMARCA4 expression levels correlated with good outcomes in KMT2A-R ALL. Based on these results, we hypothesized that Ph-like ALL, but not KMT2A-R B-ALL, may depend on high SMARCA4 expression levels. Validating our hypothesis, pharmacologic SMARCA4 inhibition using two recently developed inhibitors (BRM014 and FHD-286) showed that Ph-like B-ALL was significantly more sensitive to SMARCA4 inhibition compared to KMT2A-R ALL. KMT2A-R cell lines were highly resistant to both inhibitors, while Ph-like cells were highly sensitive, with IC50s for BRM014 at 10-50nM and 1nM for FHD-286.
 Conclusion: Our results identify that SMARCA4 has a very low mutation frequency in B-ALL and is highly expressed in Ph-like B-ALL. Furthermore, we demonstrate a direct correlation between high SMARCA4 expression and poor patient outcomes, as well as a clear dependency of Ph-like B-ALL on SMARCA4. Based on our data, applying a SMARCA4 inhibitor-based therapy may specifically benefit patients with Ph-like B-ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f699824d4c1221d5494d13936d2cbceaa818ddcc" target='_blank'>
              Targeting the SWI/SNF Chromatin Remodeling Complex Subunit SMARCA4 in ALL
              </a>
            </td>
          <td>
            Venkata Sesha Sai Abhinav Ayyadevara, Ashley Paik, Ria Perencsik, Monika M. Toma, Tomasz Skorski, Rozbeh Jafari, Gerald B Wertheim, H. Geng, C. Hurtz
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Some selfish genetic elements drive at meiosis to achieve transmission distortion, breaking the rules of Mendelian segregation to enhance their own evolutionary success. It has been previously shown that positive modifiers of drive must act in cis in order to gain the selfish benefit of drive and that suppressors of drive will be selected at any unlinked loci. Here, we model the evolution of an autosomal trans-acting gene (Distorter) that causes the Y-chromosome (or even 0-chromosome) to drive. We show that such a gene may spread in the population when linked to a second locus, Assister, whose alleles are transmitted at different frequencies through sperm as compared to eggs, for which we consider various scenarios, e.g. genes subject to sexually antagonistic selection or sex-limited drive. Depending on the mechanistic details of sex-chromosome drive, Distorter’s spread can additionally facilitate transitions between XY and X0 sex determination. We discuss these results in relation to sex allocation theory, and as proof of principle for a novel type of naturally occurring or synthetically useful sex-ratio-distorting selfish genetic element.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613e353dc3f751fb439c900d5ee4616ae023c27e" target='_blank'>
              Remote-control meiotic drive of sex chromosomes
              </a>
            </td>
          <td>
            Naomi L. Greenberg, M. Patten, M. Schenkel
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The memory of cellular identity is crucial for the correct development of an individual and is maintained throughout life by the epigenome. Chromatin marks, such as DNA methylation and histone modifications, ensure the stability of gene expression programmes over time and through cell division. Loss of these marks can lead to severe pathologies, including cancer and developmental syndromes. However, reprogramming of cellular identity is also a natural phenomenon that occurs early in mammalian development, particularly in the germ line, which enables the production of mature and functional gametes. The germ line transmits genetic and epigenetic information to the next generation, contributing to the survival of the species. Primordial germ cells (PGCs) undergo extensive chromatin remodelling, including global DNA demethylation and erasure of the parental imprints. This review introduces the concept of epigenetic reprogramming, its discovery and key steps, as well as the transcriptional and chromatin changes that accompany germ cell formation in mice. Finally, we discuss the epigenetic mechanisms of genomic imprinting, its discovery, regulation and relevance to human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2611fcc1b3dfdfc6647da0d47d718b5970da542" target='_blank'>
              [Epigenetic reprogramming, germline and genomic imprinting].
              </a>
            </td>
          <td>
            Clara Roidor, Karim Chebli, Maud Borensztein
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Using a newly described in vitro murine model of GBM [Bohm et al., Neuro-Oncol 2020 and Omairi et al., Neuro-Oncol 2023] in which P53 null neural progenitor cells (NPCs) divide abnormally and evolve a GBM-like genome during exposure to Platelet-Derived Growth Factor-AA (PDGFA), we asked how a brain-abundant mitogen could transform NPCs. By analyzing gene and protein expression over time, we found that PDGFA fails to induce the transcription of kinetochore and spindle assembly checkpoint genes, while simultaneously driving NPCs to enter mitosis. These dual effects caused chromosome miss-segregation in continuously dividing NPCs; moreover, they occurred in both WT and null NPCs, although only null cells survived defective mitosis. These surviving cells gradually expanded in PDGFA accumulating both random and clonal chromosomal re-arrangements. Transcriptome analysis of NPCs in PDGFA revealed significant under-expression of Foxm1, the major regulator of kinetochore transcription, together with over-expression and phosphorylation of the immediate early response gene, FOS. Analysis of signalling downstream of PDGFRα identified the Ras-MAPK pathway, especially ERK, as responsible for FOS activation. As surviving null cells gradually expanded, they accumulated random and recurrent chromosomal rearrangements. Expansion and subsequent PDGFA-independent proliferation and tumorigenicity were associated with re-expression of Foxm1 and kinetochore proteins, and accompanied by over-expression of Egfr, an RTK-signature that defines human GBM. By stimulating proliferation without setting the stage for error-free mitosis, exposure to PDGFA transforms p53 null NPCs and generates Egfr amplified GBM-like cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25847a5da5fbc11c69b366797c99e2ec170e9736" target='_blank'>
              TMOD-01. EGFR AMPLIFICATION ACCOMPANIES ADAPTATION TO MITOGEN-INDUCED DEFECTIVE MITOSIS IN PDGFA
              </a>
            </td>
          <td>
            Mathieu Meode, Hiba Omairi, Sarah Sait, Sophie Black, J. Cairncross
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Replication-repair deficiency (RRD) stemming from mismatch repair and polymerase proofreading gene defects (MMRD/PPD) lead to hypermutant childhood cancers, most frequently brain tumors. While medulloblastoma is reported, the clinical, genomic and immune landscape remains unknown.



 We analysed the genome, methylome, transcriptome (bulk, single-nuclei) and immune-microenvironment of the largest cohort of RRD-medulloblastoma patients enrolled by the International RRD Consortium and correlated these to clinical outcomes. RRD mice-models were used to preclinically assess response to immunotherapy. Results:



 RRD-medulloblastoma (n=43) were enriched for anaplasia (61%) and localised disease (77%). Methylation/nano-string-based subgrouping failed to classify 40%, while the remaining clustered with the SHH-subgroup. Copy-number changes were notably infrequent (<20%). All tumors harboured hypermutation and microsatellite instability in contrast to non-RRD controls (p<0.0001). Pathogenic variants were frequent in POLE/POLD1 (80%), TP53 (48%) and SHH-pathway genes (PTCH1, SUFU, SMO: 56%). Interestingly, some tumors additionally had glioma-driver alterations (ATRX, NF1: ~50%), similar to RRD cerebellar glioblastoma, but distinct from both non-RRD medulloblastoma (n=791) and hemispheric glioblastoma (n=733) controls. Moreover, although bulk-transcriptome matched non-RRD SHH medulloblastoma, single-nuclei analyses suggested an earlier cell-of-origin potentially explaining the heterogenous phenotypes resulting from the driver mutations. Uniquely, immune analyses (gene expression and immunohistochemistry) demonstrated high intra-tumoral CD8 T-cell infiltration. Three-year progression-free survival was 60%. Outcome was worse for RRD-medulloblastoma harbouring TP53-mutation (p=0.04) and failing subgroup-classification (p=0.004). Nestin/Cre MSH2-/- POLE;p.S459F mice developing medulloblastoma recapitulated human disease and demonstrated response to anti-PD1 monotherapy. Recurrent/progressive human tumors including those harbouring TP53-mutations exhibited radiological responses to anti-PD1 monotherapy, which was associated with prolonged ongoing survival in comparison to those not treated with checkpoint-inhibitors (p=0.02).



 Hypermutant RRD-medulloblastoma reveal heterogenous phenotypes due to unique spectrum of driver mutations and an early cell of origin. Their immune-hot microenvironment allows successful salvage treatment with checkpoint-inhibitors for those failing chemo-radiation, including the high-risk SHH/TP53-mutant and unclassified subgroups.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce04e7f8a2b9f714e7b852d9d98fb0db358fffc" target='_blank'>
              IMMU-14. TRANS-SPECIES ANALYSIS OF REPLICATION-REPAIR DEFICIENT (RRD) MEDULLOBLASTOMA AND RESPONSE TO IMMUNE CHECKPOINT INHIBITION: AN IRRDC REPORT
              </a>
            </td>
          <td>
            Anirban Das, Nick Fernandez, A. Levine, Kyle Smith, Evan Wang, M. Galati, Z. Aamir, Jill Chung, L. Negm, Hope Friedman, Katharine O’Flaherty, Owen Crump, Quang Trinh, N. Nunes, V. Bianchi, L. Stengs, M. Edwards, Lincoln Stein, E. Bouffet, Michael Taylor, P. Northcott, Vijay Ramaswamy, C. Hawkins, U. Tabori
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) broadly impacts host genes, affecting the infected cell population and inducing the development of a disease with a poor prognosis, adult T-cell leukemia-lymphoma (ATL). This study aimed to provide a comprehensive epigenomic characterization of the infected cell population and evaluated the transcriptome and chromatin structures of peripheral blood cells in HTLV-1-infected individuals using RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq). The infected cells showed significant changes in gene expression patterns from the polyclonal stage and before ATL onset while demonstrating similarities to tumor-forming ATL cells. These similarities were a result of large-scale open chromatin changes, supporting the independent early formation of epigenomic aberrations as an underlying mechanism for later clonal propagation. This study also demonstrated that HTLV-1 Tax directly affects the host chromatin structure, thereby developing fundamental epigenomic characteristics. Several Tax target genes, including the RASGRP3-ERK pathway, were recognized, indicating an impact on signaling pathways. This genome-wide variability in chromatin structural property is a novel feature of HTLV-1 infection and may contribute to pathogenic mechanisms. In addition, it has crucial implications for better understanding the impact of HTLV-1 on the host genome and identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f7c450fea8921cdae499b880a5d0560a4c49aa" target='_blank'>
              Rewired chromatin structure and epigenetic gene dysregulation during HTLV-1 infection to leukemogenesis.
              </a>
            </td>
          <td>
            Jun Mizuike, Kako Suzuki, Shu Tosaka, Yuta Kuze, Seiichiro Kobayashi, Makoto Nakashima, Koji Jimbo, Y. Nannya, Yutaka Suzuki, K. Uchimaru, Makoto Yamagishi
          </td>
          <td>2024-11-19</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="In mammals, fertilization is followed by extensive reprogramming and reorganization of the chromatin accompanying the transcriptional activation of the embryo. This reprogramming results in blastomeres with the ability to give rise to all cell types and a complete organism, including extra-embryonic tissues, and is known as totipotency. Transcriptional activation occurs in a process known as zygotic genome activation (ZGA) and is tightly linked to the expression of transposable elements, including endogenous retroviruses (ERVs) such as endogenous retrovirus with leucine tRNA primer (ERVL). Recent studies discovered the importance of ERVs in this process, yet the race to decipher the network surrounding these elements is still ongoing, and the molecular mechanism behind their involvement remains a mystery. Amid a recent surge of studies reporting the discovery of various factors and pathways involved in the regulation of ERVs, this review provides an overview of the knowns and unknowns in the field, with a particular emphasis on the chromatin landscape and how ERVs shape preimplantation development in mammals. In so doing, we highlight recent discoveries that have advanced our understanding of how these elements are involved in transforming the quiescent zygote into the most powerful cell type in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de59ad5f3d58d084f36e3b3323bf2a35634e22da" target='_blank'>
              The impact of retrotransposons on zygotic genome activation and the chromatin landscape of early embryos.
              </a>
            </td>
          <td>
            Therese Solberg, Mie Kobayashi-Ishihara, Haruhiko Siomi
          </td>
          <td>2024-11-22</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In the era of transcriptomics, the role of epigenetics in the study of cancers in females has gained increasing recognition. This article explores the impact of epigenetic modifications, such as DNA methylation, histone modification, and non-coding RNA, on cancers in females, including breast, cervical, and ovarian cancers (1). Our findings suggest that these epigenetic markers not only influence tumor onset, progression, and metastasis but also present novel targets for therapeutic intervention. Detailed analyses of DNA methylation patterns have revealed aberrant events in cancer cells, particularly promoter region hypermethylation, which may lead to silencing of tumor suppressor genes. Furthermore, we examined the complex roles of histone modifications and long non-coding RNAs in regulating the expression of cancer-related genes, thereby providing a scientific basis for developing targeted epigenetic therapies. Our research emphasizes the importance of understanding the functions and mechanisms of epigenetics in cancers in females to develop effective treatment strategies. Future therapeutic approaches may include drugs targeting specific epigenetic markers, which could not only improve therapeutic outcomes but also enhance patient survival and quality of life. Through these efforts, we aim to offer new perspectives and hope for the prevention, diagnosis, and treatment of cancers in females.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cc4082f981f81654c23a63365924b300c8aa1c" target='_blank'>
              Transcriptomic era of cancers in females: new epigenetic perspectives and therapeutic prospects.
              </a>
            </td>
          <td>
            Runhe Zhu, Jiawei Ni, Jiayin Ren, Dongye Li, Jiawei Xu, Xinru Yu, Yingjie Ma, Luan Kou
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4f929a38c9c2e5538148166e84040e0b1407ac" target='_blank'>
              Chemoproteogenomic stratification of the missense variant cysteinome
              </a>
            </td>
          <td>
            H. Desai, Katrina H. Andrews, Kristina V Bergersen, Samuel Ofori, Fengchao Yu, Flowreen Shikwana, M. Arbing, Lisa M Boatner, M. Villanueva, Nicholas Ung, Elaine F. Reed, Alexey I. Nesvizhskii, Keriann M. Backus
          </td>
          <td>2024-10-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) in Glioblastoma (GBM) residual disease and escape from TIS account for resistance and recurrence, but the mechanism of TIS manifestation remains obscure. Here, we demonstrate that replication stress (RS) is critical for the induction of TIS in residual cells by employing an in-vitro GBM therapy-resistance cellular model. Interestingly, we found a 'bi-phasic' mode of DNA damage after radiation treatment and revealed that the second phase of DNA damage arises majorly in the S-phase of residual cells due to RS. Mechanistically, we show that persistent phosphorylated ATR is a safeguard for radiation resilience, while the other canonical RS molecules remain unaltered during the second phase of DNA damage. Importantly, RS precedes the induction of senescence, and ATR inhibition results in TIS reduction, leading to apoptosis. Moreover, ATR inhibition sensitized PARP-1 inhibitor-induced enhanced TIS-mediated resistance, leading to cell death. Our study demonstrates the crucial role of RS in TIS induction and maintenance in GBM residual cells, and targeting ATR alone or in combination with a PARPi will be an effective strategy to eliminate TIS for better treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/371a4a7bd56c651f3ce9205ee0c78d0127cfb00d" target='_blank'>
              Bi-phasic DNA damage and non-canonical replication stress response govern radiation-induced senescence in Glioblastoma.
              </a>
            </td>
          <td>
            Atanu Ghorai, Bhawna Singh, S. Dutt
          </td>
          <td>2024-11-21</td>
          <td>Journal of cell science</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="DNA methylation is a critical epigenetic modification that regulates gene expression and plays a significant role in development and disease processes. Here, we present the Cytosine-phosphate-Guanine Pretrained Transformer (CpGPT), a novel foundation model pretrained on over 1,500 DNA methylation datasets encompassing over 100,000 samples from diverse tissues and conditions. CpGPT leverages an improved transformer architecture to learn comprehensive representations of methylation patterns, allowing it to impute and reconstruct genome-wide methylation profiles from limited input data. By capturing sequence, positional, and epigenetic contexts, CpGPT outperforms specialized models when finetuned for aging-related tasks, including chronological age prediction, mortality risk, and morbidity assessments. The model is highly adaptable across different methylation platforms and tissue types. Furthermore, analysis of sample-specific attention weights enables the identification of the most influential CpG sites for individual predictions. As a foundation model, CpGPT sets a new benchmark for DNA methylation analysis, achieving strong performance in the Biomarkers of Aging Challenge, where it placed second overall in chronological age estimation and first on the public leaderboard in methylation-based mortality prediction. Highlights CpGPT is a novel foundation model for DNA methylation analysis, pretrained on over 1,500 datasets encompassing 100,000+ samples. The model demonstrates strong performance in zero-shot tasks including imputation, array conversion, and reference mapping. CpGPT achieves state-of-the-art results in mortality prediction and chronological age estimation. Sample-specific interpretability is enabled through analysis of attention weights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02c9095a01a40dde5d97c8d18c0ec0e0b28e7f2" target='_blank'>
              CpGPT: a Foundation Model for DNA Methylation
              </a>
            </td>
          <td>
            Lucas Paulo de Lima Camillo, Raghav Sehgal, J. Armstrong, A. Higgins-Chen, S. Horvath, Bo Wang
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92346b55940f48c9910eb89c8a2199b9be5b6633" target='_blank'>
              KRAS G12C mutation in NSCLC in a small genetic center: insights into sotorasib therapy response potential
              </a>
            </td>
          <td>
            Metin Eser, Gulam Hekimoğlu, Murat Hakkı Yarar, Sezin Canbek, Melike Ozcelik
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Megakaryocytes (MKs) are large, hematopoietic cells that primarily reside in the bone marrow and release platelets into the circulation. MKs can reach extreme states of polyploidy (with a DNA content of up to 128n) and contain their nuclear lobes within a single nuclear membrane. While DNA replication in MKs is well studied, the impact of DNA damage and repair inhibition on megakaryopoiesis. To determine if modulating DNA damage within the hematopoietic stem cell and progenitor compartment affects megakaryocyte development, we employed a pharmacological approach using poly-ADP ribose polymerase inhibitors (PARPi). PARP, a sensor within the DNA damage repair pathway, binds to sites of DNA damage and recruits DNA repair effectors. PARPi, such as niraparib and olaparib, can trap PARP at site of lesions, leading to stalling of DNA replication forks and accumulation of DNA damage. Clinically, PARP inhibition is known to cause thrombocytopenia in patients, postulated to be due to effects on megakaryocytes and progenitors as platelets themselves do not have DNA. Higher dosing of PARPi in C57BL/6J mice mimicked this clinical phenotype. Surprisingly, treatment of C57BL/6J mice with lower doses of the FDA-approved PARP inhibitors niraparib and olaparib led to a doubling of the platelet count after 11 days. Flow cytometric analysis of niraparib-treated platelets revealed a larger and younger phenotype, as indicated by increased thiazole orange (TO) incorporation. Functional evaluation showed an increased sensitivity to GPVI stimulation as measured by CD62P expression and GPIIbIIIA activation. Cell surface expression of GPIIb, GPIIIA and GPVI were unaltered. Splenic weights of the mice were unaltered, decreasing the likelihood that the observed increase in platelets is due to splenic platelet pool mobilization. Together, these findings suggests that lower dose niraparib exposure in mice leads to increased functional platelets, by enhancing platelet production. To determine the root cause of this increased production, we evaluated the hematopoietic stem cell and progenitor compartment. Initially, we observed increased MKs after niraparib treatment via 2-photon intravital microscopy (2PIVM) in the calvaria of MK/platelet (vWF-GFP) reporter mice. These findings were validated by immunofluorescent visualization of MKs in femoral cryosections, revealing an increase in both the number, and size of MKs in niraparib-treated mice after 11 days of treatment. In vivo niraparib treatment also resulted in an increase in MK ploidy from day 3 to day 11, after which we observed a stark shift towards high ploidy MKs, at the expense of 2n MKs. While MK ploidy is not directly causative of platelet production, higher ploidy MKs are associated with increased platelet production and more terminal MK maturation. To directly analyze DNA damage in our cellular populations, we isolated native bone marrow MKs from niraparib- and vehicle-treated mice and performed comet assays. This indicated increased DNA damage in MKs isolated from niraparib-treated mice, as these cells showed significantly longer tail lengths when compared to control. In line with our comet assays, we then measured accumulation of phosphorylated histone H2 family member X (γ-H2AX), a marker of DNA breaks, by immunofluorescence. In situ, we observed a higher proportion of γ-H2AX positive MKs as well as more foci per MK after 11 days of treatment with niraparib. Evaluation of the bone marrow compartment revealed that after three days of niraparib treatment, and preceding the increase in platelet counts, there was a pronounced increase in the number of γ-H2AX-positive LT-HSCs, CD41+ HSCs, and MK progenitors (MkP), indicating increased DNA damage accumulation. In addition, after 11 days of treatment, there was an absolute increase in all multipotent progenitor populations (MPP2-4), LT-HSCs, MkPs and MKs. Together, these data suggest that low dose niraparib treatment causes DNA damage in HSPCs and MkPs and enhances canonical megakaryopoiesis, leading to increased platelet production.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c76b94cfe28d815600332ce6c2830addf88a5dc" target='_blank'>
              Priming of Megakaryopoiesis and Platelet Production through DNA Damage Repair Inhibition
              </a>
            </td>
          <td>
            Roelof H. Bekendam, Andrew P. Stone, Virginia Camacho, Isabelle C. Becker, Clementine Payne, Estelle Carminita, Maria N. Barrachina, Jakub Kaplan, Luis Batista, Sharmista Pal, Joseph E. Italiano, Kellie R. Machlus
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Borrelia burgdorferi, a tick-vectored spirochete bacteria best known for causing Lyme disease, has been found to induce physiological and behavioural changes in its tick vector that can increase tick fitness and its ability to transmit the bacteria. The mechanism by which this bacterium modulates these changes remains unknown. Epigenetics plays a central role in transducing external and internal microbiome environmental influences to the organism, so we investigated DNA methylation and the expression of a key histone modification enzyme in Borrelia-infected and uninfected Ixodes scapularis ticks. DNA methylation of the pericentromeric tandem repeats family, Ixodes scapularis Repeats (ISR) were assessed by methylated-DNA immunoprecipitation (MeDIP) followed by qPCR of the ISR regions. DNA methylation of the ISR sequences was found. The different repeats had different levels of DNA methylation, however, these levels were not significantly affected by the presence or absence of B. burgdorferi. The epigenetic regulator euchromatic histone lysine methyltransferase 2 (EHMT2) is recognized as having a key role in modulating the organismal stress response to infections. To assess EHMT2 transcription in Borrelia-infected and uninfected ticks, real-time reverse transcriptase PCR was performed. Uninfected ticks had over 800X lower EHMT2 expression than infected ticks. This study is among the first to identify a gene that may be involved in producing epigenetic differences in ticks depending on infection status and lays the groundwork for future epigenetic studies of I. scapularis in response to B. burgdorferi as well as other pathogens that these ticks transmit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a55683c8198f5a6deae72b7ff3e95f0325a745d" target='_blank'>
              Epigenetic responses in Borrelia-infected Ixodes scapularis ticks: Over-expression of euchromatic histone lysine methyltransferase 2 and no change in DNA methylation
              </a>
            </td>
          <td>
            Grace Hadley MacIntosh, Alexandra C. Nuyens, Jessica L. Vickery, Anne Berthold, V. Lloyd
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA damage, particularly the occurrence of DNA double-strand breaks (DSBs), presents a significant hazard to the integrity and viability of cells. Improper repair of DSBs can result in chromosomal alterations, oncogenic changes, or cell demise. The MRE11-RAD50-NBS1 (MRN) complex plays a crucial role in DNA repair and signaling under the Ataxia Telangiectasia Mutated (ATM) kinase regulation. In this study, we employed comprehensive computational techniques to analyze the structure of RAD50 in Danio rerio (Zebrafish), utilized as a model organism. Additionally, we conducted in silico assessments of RAD50 from both Zebrafish and humans, comparing their characteristics. The substantial sequence resemblance between DrRAD50 and HsRAD50 suggests that DrRAD50 could potentially serve as a valuable model for HsRAD50. However, it is important to acknowledge that sequence similarity alone does not necessarily imply functional equivalence. Further functional studies are needed to confirm the extent of their functional similarities. By examining the secondary and tertiary protein structures of RAD50, we observed a notable likeness between Zebrafish and Human RAD50 proteins. In silico analysis demonstrated that the sequence of RAD50 in zebrafish shares 70% similarity with the human RAD50 protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c711684261abaf353b4564abadc21c525f96de" target='_blank'>
              From Zebrafish To Humans: In Silico Comparative Study of RAD50 Sequences
              </a>
            </td>
          <td>
            N. Khalili, A. Najm, Ibrahim Mahmood, Douglas Law, Chyan Leong Ng, A. Azfaralariff, S. Fazry
          </td>
          <td>2024-10-27</td>
          <td>Malaysian Applied Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9ca2f5b5b7678d5eedf7402c5341d078db68d6" target='_blank'>
              Human HDAC6 senses valine abundancy to regulate DNA damage.
              </a>
            </td>
          <td>
            Jiali Jin, Tong Meng, Yuanyuan Yu, Shuheng Wu, Chen-Chen Jiao, Sihui Song, Ya-Xu Li, Yu Zhang, Yuan-Yuan Zhao, Xinran Li, Zixin Wang, Yu-Fan Liu, Runzhi Huang, Jieling Qin, Yihua Chen, Hao Cao, Xiao Tan, Xin Ge, Cong Jiang, Jianhuang Xue, Jian Yuan, Dianqing Wu, Wei Wu, Ci-Zhong Jiang, Ping Wang
          </td>
          <td>2024-11-20</td>
          <td>Nature</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutagenic processes drive evolutionary progress, with ultraviolet (UV) radiation significantly affecting evolution. Despite extensive research on SOS response-mediated mutagenesis, UV-induced repair mechanisms remain complex, and their effects on cell survival and mutagenesis are not fully understood. We previously observed a near-perfect correlation between RecA-mediated SOS response and mutation levels in Escherichia coli following UV treatment. However, prolonged UV exposure caused transient non-culturability and impaired SOS-mediated mutagenesis. Using fluorescent reporters, flow cytometry, promoter-reporter assays, single-gene deletions, knockouts, and clonogenic assays, we found that excessive UV exposure disrupts cellular translation, reducing SOS gene expression, albeit with minimal impact on membrane permeability or reactive oxygen species levels. While our findings underline the abundance of repair mechanisms in E. coli cells, enabling them to compensate when specific genes are disrupted, they also highlighted the differential impacts of gene deletions on mutagenesis versus culturability, leading to three major outcomes: (i) Disruption of proteins involved in DNA polymerase for trans-lesion synthesis (UmuC and UmuD) or Holliday junction resolution (RuvC) results in significantly decreased mutagenesis levels while maintaining a transient non-culturability pattern after UV exposure. (ii) Disruption of proteins involved in homologous recombination (RecA and RecB) and nucleotide excision repair (UvrA) leads to both significantly reduced mutagenesis and a more severe transient non-culturability pattern after UV exposure, making these cells more sensitive to UV. (iii) Disruption of DNA Helicase II (UvrD), which functions in mismatch repair, does not affect mutagenesis levels from UV radiation but results in a very pronounced transient non-culturability pattern following UV exposure. Overall, our results further advance our understanding of bacterial adaptation mechanisms and the role of DNA repair pathways in shaping mutagenesis. Author Summary Ultraviolet (UV) radiation has been a significant force in driving genetic variation and adaptation throughout billions of years of evolution. By directly damaging DNA and triggering repair mechanisms, UV radiation is a powerful tool for studying mutagenesis. This study aims to provide new insights into the complex processes behind bacterial mutagenesis, a critical topic in microbiology and public health. Although the connection between the SOS response and DNA repair in UV-treated cells is well known, two key phenomena — UV-induced bacterial cell dormancy and mutagenesis — remain poorly understood. Our findings reveal a highly unusual, SOS response-dependent transient non-culturability in Escherichia coli cells following prolonged UV exposure. However, the downstream mechanisms of this phenomenon and its links to mutagenesis remain unclear. This study seeks to thoroughly investigate these phenomena, offering new insights into their underlying molecular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8574f751e70ca946f8a54c7557e261ab9d00bd43" target='_blank'>
              UV-Induced DNA Repair Mechanisms and Their Effects on Mutagenesis and Culturability in Escherichia coli
              </a>
            </td>
          <td>
            Sreyashi Ghosh, Mehmet A. Orman
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) represent one of the most severe forms of genetic damage in organisms, yet vertebrate models capable of monitoring DSBs in real-time remain scarce. BRCA1/BRCA2-containing complex subunit 3 (BRCC3), also known as BRCC36, functions within various multiprotein complexes to mediate diverse biological processes. However, the physiological role of BRCC3 in vertebrates, as well as the underlying mechanisms that govern its activity, are not well understood. To explore these questions, we generated brcc3-knockout zebrafish using CRISPR/Cas9 gene-editing technology. While brcc3 mutant zebrafish appear phenotypically normal and remain fertile, they exhibit significantly increased rates of mortality and deformity following exposure to DNA damage. Furthermore, embryos lacking Brcc3 display heightened p53 signaling, elevated γ-H2AX levels, and increased apoptosis in response to DNA-damaging agents such as ultraviolet (UV) light and Etoposide (ETO). Notably, genetic inactivation of p53 or pharmacological inhibition of Ataxia-telangiectasia mutated (ATM) activity rescues the hypersensitivity to UV and ETO observed in Brcc3-deficient embryos. These findings suggest that Brcc3 plays a critical role in DNA damage response (DDR), promoting cell survival during embryogenesis. Additionally, brcc3-null mutant zebrafish offer a promising vertebrate model for real-time monitoring of DSBs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d1549cbda55b49c02b764216ac718c73d8cdce" target='_blank'>
              Loss of Brcc3 in Zebrafish Embryos Increases Their Susceptibility to DNA Damage Stress
              </a>
            </td>
          <td>
            Zhengyang Wang, Caixia Wang, Yanpeng Zhai, Yanhong Bai, Hongying Wang, Xiaozhi Rong
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The de-ubiquitinase USP33 has been shown to possess either tumour-promoting or inhibitory effect on human cancer cells. However, all these findings are mainly based on in vitro cell culture models, and the in vivo evidence, which is more plausible to digest the functional role of USP33 in carcinogenic process, is still lacking. Here, we demonstrate that USP33 modulates DNA damage responses including cell cycle arrest and apoptosis induction through associating with p53. It directly interacts with p53 to mediate its de-ubiquitination and further  stabilisation under DNA damage condition. Depletion of USP33 induces an enhanced level of p53 ubiquitination, which de-stabilises p53 protein leading to impaired DNA damage responses. Furthermore, USP33 silencing shows either promoted or inhibited effect on cell proliferation in human cancer cells with p53 WT and mutant background, respectively. Consistently, mice with hepatocyte-specific USP33 knockout are more sensitive to nitrosodiethylamine (DEN)-induced hepatocarcinogenesis compared to wild type mice. Thus, our in vitro and in vivo evidences illustrate that USP33 possesses anti-tumour activity via regulating p53 stability and activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf79241c93d773eae434c4f3d8db032442a0c49" target='_blank'>
              USP33 Regulates DNA Damage Response and Carcinogenesis Through Deubiquitylating and Stabilising p53.
              </a>
            </td>
          <td>
            Yuqi Zhu, Zixiang Chen, Kaifeng Niu, Mengge Li, Yuchun Deng, Ji Zhang, D. Wei, Jiaqi Wang, YongLiang Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The nervous system is susceptible to DNA damage and DNA repair defects, and if DNA damage is not repaired, neuronal cells can die, causing neurodegenerative diseases in humans. The overall picture of what is known about DNA repair mechanisms in the nervous system is still unclear. The current challenge is to use the accumulated knowledge of basic science on DNA repair to improve the treatment of neurodegenerative disorders. In this review, we summarize the current understanding of the function of DNA damage repair, in particular, the base excision repair and double-strand break repair pathways as being the most important in nervous system cells. We summarize recent data on the proteins involved in DNA repair associated with neurodegenerative diseases, with particular emphasis on PARP1 and ND-associated proteins, which are involved in DNA repair and have the ability to undergo liquid–liquid phase separation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8042a1ecd637196dc9950d0a43a55a44dd052ef" target='_blank'>
              Proteins Associated with Neurodegenerative Diseases: Link to DNA Repair
              </a>
            </td>
          <td>
            S. Khodyreva, N. Dyrkheeva, O. Lavrik
          </td>
          <td>2024-12-11</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9be8cb3fc8529e67efe318e7f654c3c420ce756" target='_blank'>
              Hop2-Mnd1 functions as a DNA sequence fidelity switch in Dmc1-mediated DNA recombination
              </a>
            </td>
          <td>
            Jo-Ching Peng, Hao-Yen Chang, Yuting Liang Sun, Mara Prentiss, Hung-Wen Li, Peter Chi
          </td>
          <td>2024-10-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Background and Rationale
 Multiple myeloma (MM) is an incurable plasma cell malignancy characterized by clonal expansion in the bone marrow. MM cells exhibit extensive DNA damage, necessitating constant repair for survival and proliferation. Additionally, they rely on an overactive proteasome pathway to degrade the large amount of immunoglobulins they produce. Consequently, DNA-damaging chemotherapeutic drugs and proteasome inhibitors are standard treatments for MM. However, acquired resistance to these drugs is common. RAD23 Homolog A (Rad23A) is a multifunctional protein recognized for its involvement in the nucleotide excision repair (NER) and proteasome pathways. Here, we show that inhibition of Rad23A triggers potent anti-MM activity using both in vitro and in vivo models of MM.
 Results
 We first analyzed MM patient gene expression databases and found that Rad23A was highly expressed in MM patient compared to normal controls. The high expression of Rad23A was inversely correlated with patient overall survival (n=1614; p<1E−16). qPCR and immunoblotting confirmed that Rad23A levels were higher in MM patient cells and MM cell lines versus normal cells. CRISPR/Cas9-mediated knockout (KO) of Rad23A in MM.1R, MM.1S, and AMO1 cells, as well as shRNA knockdown (KD) in H929 and ANBL6-WT cells, resulted in decreased cell growth. Notably, decreased cell growth was also observed in cells resistant to the proteasome inhibitor bortezomib (ANBL6-BR). Western blot analysis revealed that Rad23A KO and KD cells had elevated levels of K48-linked polyubiquitylated proteins, indicating an inhibited proteasome system. Depletion of Rad23A activated the DNA damage pathway, evidenced by elevated gamma-H2AX signals in immunofluorescence staining and increased Rad51 foci formation observed under confocal microscopy. This led to elevated endoplasmic reticulum stress response signaling (BiP, p-eIF2α) and apoptosis (Cleaved-Caspase3, PARP, p21). Increased Caspase 3/7 activity was observed in Rad23A KO and KD cells using live-cell imaging system (IncuCyte), indicating caspase-related apoptosis activation. Flow cytometry showed that Rad23A depletion triggered cell cycle arrest, with cells accumulating in the G2/M phase. Proteomic analysis showed that Rad23A-KO cells downregulated 225 proteins and upregulated 642 proteins compared to WT cells. Upregulated proteins were involved in interferon-alpha response, apoptosis, and the p53 pathway, while downregulated proteins were associated with DNA repair, TNF-α signaling via NF-κB, and cell cycle checkpoints. Finaly, mice xenografted with Rad23A inducible-knockout MM1R cells exhibited reduced tumor growth (p<0.0001) and prolonged survival (p<0.0001) compared to those engrafted with WT-MM1R cells.
 Conclusion
 Our data demonstrate the therapeutic potential of the proteasome shuttle factor and DNA repair protein Rad23A, providing a preclinical basis for developing Rad23A inhibitors to treat MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7771f5541d000bcdc94a06631c8b7b2928b2f733" target='_blank'>
              Proteasome Shuttle Factor and DNA Repair Protein Rad23A Is a Potential Therapeutic Target in Multiple Myeloma
              </a>
            </td>
          <td>
            Xueping Wan, Yan Song, Ting Du, A. Ray, Teng Fang, Minxing Wang, Sindhu C Pillai, Md. Abu Musa, Kenneth Wen, D. Chauhan, Kenneth C. Anderson
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Circular RNA (circRNA) emerges as a significant sub-type of single-stranded non-coding RNA within colorectal cancer (CRC), boasting high abundance. Delving into research, numerous pivotal roles of circRNA in therapeutic contexts within CRC have come to light, encompassing areas such as metastasis, apoptosis, and proliferation. Moreover, circRNAs exhibit significant involvement in the advancement of therapeutic strategies, demonstrating unique correlations with tumor staging, size and overall survival rates in colorectal cancer. These associations position circRNAs as potential candidates for both anticancer interventions and prognostic biomarkers. Among all cancers, colorectal cancer is the second most prevalent cause of cancer-related death and the third most common disease to be diagnosed worldwide. To gain deeper insights into the impact of circRNA-based therapeutic developments on CRC and its progression, this comprehensive review aims to synthesize the roles of specific therapeutic applications targeting circRNAs in CRC. It also aims to evaluate circRNAs' potential as useful therapeutic targets and prognostic indicators in the context of colorectal cancer. The overarching goal of this review is to illuminate the landscape of therapeutic strategies and aid in clinical decision-making processes related to CRC. By elucidating the intricate interplay between circRNAs and therapeutic interventions, this review seeks to contribute to the advancement of therapeutic modalities and improve patient outcomes in the realm of colorectal cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e2a54ee5b48b0544173f0c3c5f63c22c950fe" target='_blank'>
              Circular RNAs: Therapeutic Uses in Colorectal Cancer
              </a>
            </td>
          <td>
            Muthusamy Thangavel, Chalini Vijayakumar, Deepalakshmi Balakrishnan
          </td>
          <td>2024-11-30</td>
          <td>International Journal of Experimental Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f14ce0de021c18e7f8fa32d88be80097576943" target='_blank'>
              BCAS2 and hnRNPH1 orchestrate alternative splicing for DNA double-strand break repair and synapsis in meiotic prophase I
              </a>
            </td>
          <td>
            Longjie Sun, Rong Ye, Changchang Cao, Zheng Lv, Chaofan Wang, Xiaomei Xie, Xuexue Chen, Xiaohong Yao, Shuang Tian, Lu Yan, Yujing Shao, Sheng Cui, Chen Chen, Yuanchao Xue, Lei Li, Juan Chen, Jiali Liu
          </td>
          <td>2024-11-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The normal-tissue and tumor attractors are far apart in gene-expression space [1,2]. As a consequence, there are genes with specific gene-expression intervals characteristic only of normal tissues or tumors [3]. We call them N-genes and T-genes, respectively. They are not only perfect markers, but are also responsible of anchoring the state in the vicinity of the corresponding attractor. According to the higher degree of disorder of tumors [4,5], there are 10-times more T-genes than N-genes [3]. Carcinogenesis, which is the progressive motion from the N-attractor to the T-attractor, can also be viewed as a process of deactivating the N-genes marking and holding the normal tissue and setting new T-markers and anchors for the tumor. In prostate, for example, the process starts with around 500 activated N-genes in any healthy normal tissue that may be dramatically reduced to a few N-genes before the transition to the tumor occurs. The available gene expression data [6,7] shows that even 2 or 3 active N-genes are enough for holding the normal state condition (at least from a pathological perspective). The process is expected to evolve in the form of consecutive deregulation cascades [8], in which a few initial cascades inactivate most of the active N-genes, and a final cascade inactivates the remaining N-markers and sets the new T-markers for the tumor. The whole process shares similarities with the multistep model of cancer [9].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da1f5356432b7442267cd5bf216810a37f8a3d" target='_blank'>
              Dynamics of N-genes and T-genes in cancer
              </a>
            </td>
          <td>
            Gabriel Gil, Augusto Gonzalez
          </td>
          <td>2024-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life.Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity.A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis.By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="During infection, the autonomous parvovirus minute virus of mice (MVM) induces cellular DNA breaks and localizes to such sites, which presumably affords an environment beneficial for genome replication. MVM replication also benefits from the DNA damage response (DDR) mediated by the ataxia-telangiectasia mutated (ATM) kinase, while the ataxia telangiectasia and Rad-3 related (ATR) arm of the DDR is disabled, which prevents activation of its primary target, checkpoint kinase 1 (Chk1). We find here that Chk1 inactivation strongly correlates with dephosphorylation of one of its targets, RAD51, known to play a pivotal role in homologous recombination repair (HRR), thus leading to substantial inhibition of DNA repair in infected cells. We demonstrate colocalization of replicating MVM DNA with cellular double-strand breaks (DSBs) during infection, and show that an agent that exogenously induces cellular DSBs significantly increases viral DNA replication levels, establishing a role for cellular genome damage in facilitating virus DNA replication. Additionally, overexpression of active Chk1 during MVM infection was found to re-establish the activating phosphorylation of RAD51 Thr 309, significantly suppress infection-induced reduction of HRR efficiency with a concomitant increase in cellular genome DSBs, and reduce viral DNA replication levels. Thus, we conclude that during infection, MVM inhibition of Chk1 activation enhances viral replication, at least in part, by inhibiting cellular HRR.IMPORTANCEThe autonomous parvovirus minute virus of mice (MVM) has a compact DNA genome encoding a minimum number of proteins. During infection, it induces cellular DNA damage and both utilizes and modifies the subsequent cellular DNA damage response (DDR) in various ways to facilitate its replication. One of MVM's activities in this regard is to inhibit one of the primary arms of the DDR, the ataxia telangiectasia and Rad-3 related (ATR) pathway, which prevents activation of checkpoint kinase 1 (Chk1), a key protein involved in controlling the cellular DDR and preserving genome integrity. We show that prevention by MVM of Chk1 activation leads to inhibition of homologous recombination repair (HRR) of cellular DNA, which helps sustain viral replication. This work illuminates another way in which autonomous parvoviruses adjust the cellular environment for their replicative advantage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab15a702412827cf231b7b73cc79a81325904c4" target='_blank'>
              Inactivation of checkpoint kinase 1 (Chk1) during parvovirus minute virus of mice (MVM) infection inhibits cellular homologous recombination repair and facilitates viral genome replication.
              </a>
            </td>
          <td>
            Igor Etingov, D. Pintel
          </td>
          <td>2024-11-20</td>
          <td>Journal of virology</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="
 Metabolic reprogramming is a hallmark of cancer, including diffuse gliomas. IDH1 and IDH2 are key enzymes linking cellular metabolism to epigenetic regulation and redox states. IDH1/IDH2 enzymes catalyze the conversion of isocitrate to α-ketoglutarate, while the mutant enzymes increase the production of D-2-hydroxyglutarate, an oncometabolite that induces epigenetic changes, leading to gene expression alterations. Understanding how epigenetic alterations affect metabolic gene expression and metabolite levels in diffuse gliomas could help identify diagnostic biomarkers and therapeutic targets. We studied the expression of metabolism-related genes, metabolites, and genome-wide methylation status, which will facilitate an integral understanding of the metabolic changes in IDH-mutant and IDH-wildtype gliomas. Study design included 47 fresh frozen tissue samples. RNA was extracted and used for gene expression analysis using the NanoString platform (nCounter® Metabolic Pathways Panel), while extracted DNA was used for targeted sequencing with a panel interrogating 600 genes and for genome-wide methylation analysis using the Illumina Epic v2. In addition, unbiased metabolomic analysis by mass spectrometry was performed in 25 tissue and CSF samples out of 47 samples. Methylation status, gene expression and metabolite levels were compared between IDH-mutant and IDH-wildtype gliomas. Results showed significant differences in DNA methylation, gene expression and metabolites between IDH-mutant and IDH-wildtype gliomas. UPP1, an enzyme involved in uracil metabolism, was studied in depth to understand the interplay between DNA methylation, gene expression, and metabolic reprogramming in diffuse glioma. Epigenetic analysis revealed that the CpG sites within the UPP1 gene displayed reduced methylation in IDH wild-type compared to IDH-mutant tumors. IDH wild-type tumors showed higher UPP1 expression compared to their IDH-mutant counterparts. Consequently, we observed higher levels of uracil in tissue and CSF samples from patients with IDH wild-type gliomas. These data provide a framework to study the relationship between mutations, methylation, gene expression and metabolism in diffuse gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a660a3dfbb1329ab34475bab6b46d65e8f2ffd0" target='_blank'>
              PATH-52. DIFFERENCES IN GENE EXPRESSION, GENOME-WIDE METHYLATION AND METABOLITES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            Anjali Vinocha, Valerie Berdecia, Daniel H Wang, Srividya Arjuna, Mauli Shah, A. Dono, N. Putluri, Y. Esquenazi, L. Ballester
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Liquid biopsy continues to be a focus of the cancer diagnostics industry, primarily due to the non-invasive nature of sample collection, however the low levels of circulating tumor DNA (ctDNA) remain a challenge. Methylation has emerged as a promising biomarker for detecting ctDNA, yet detecting the cancerous signal in an excess of non-cancerous, unmethylated cfDNA in the background has been proven difficult. Harbinger Health has developed a novel method that depletes unmethylated background at the CpG-level from specific genomic regions of interest, resulting in an enrichment of the ctDNA fraction. We developed a background cfDNA depletion method that utilizes the endonuclease activity and the specificity of CRISPR/Cas12a to remove unmethylated DNA fragments at ctDNA methylation biomarker sites. First, we identified cancer-informative regions that showed consistent contiguous elevated methylation states in cancer compared to normal samples. We then designed guide RNAs that specifically bind and cut unmethylated CpG sites within the target regions. Cas depletion of bisulfite-converted DNA enriches cancer informative signal for bioanalysis. To test validity of our Cas complex designs, we created model samples that mimic fully unmethylated and fully methylated species. We spiked these cfDNA models into a background of genomic DNA from 5% to 0.01%. In triplicate, we examined cutting efficiency, multiplexing and specificity of Cas to unmethylated species of model DNA. Results were assessed using qPCR by comparing differences in Ct with and without Cas treatment. Additionally, we applied this depletion step during our library preparation process to show utility in methylation analysis of low tumor content liquid biopsy samples. Using both the model sample dilution series and bisulfite converted cfDNA samples, which represent a range of cancerous and background methylation signal, we deplete unmethylated targets using Cas prior to indexing PCR. This rendered unmethylated targets non-amplifiable, removing them from analysis. Results were assessed using both qPCR and sequencing data, such as methylated read counts over the target region. We demonstrated our analytical approach efficiently depletes non-cancer cfDNA by on average 9-fold, which increases the fraction of ctDNA signal and can be multiplexed. This background signal depletion driven by CRISPR is sensitive and specific, maintaining methylated DNA signal detection down to 0.01% at inputs of 10ng DNA. Our Cas-mediated depletion of background signal method has proven useful for enriching rare methylated fragments over background by improving the signal to noise ratio. This technique is novel by harnessing the specificity of CRISPR to target DNA in both a sequence and methylation state specific manner. The depletion of precise methylation states at the CpG-level make it suitable for use in multiple bioassays.
 Citation Format: Caitlin Gilley, Miguel Williams, Emily Neaga, Sarah Falotico, Kade Pettie, Anthony Shuber. A methylation state specific targeted background depletion technique for enrichment of ctDNA fraction [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a7c24917cd5101085373f3d45e3093d6baa4c" target='_blank'>
              Abstract A014: A methylation state specific targeted background depletion technique for enrichment of ctDNA fraction
              </a>
            </td>
          <td>
            C. Gilley, Miguel Williams, Emily Neaga, Sarah Falotico, Kade Pettie, A. Shuber
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f613764be35d55a649a973f37ccea0e0af5d9f7" target='_blank'>
              Epigenetic dynamics of partially methylated domains in human placenta and trophoblast stem cells
              </a>
            </td>
          <td>
            Hidehiro Toh, Hiroaki Okae, K. Shirane, Tetsuya Sato, Hirotaka Hamada, Chie Kikutake, Daisuke Saito, Takahiro Arima, Hiroyuki Sasaki, Mikita Suyama
          </td>
          <td>2024-11-06</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In this study, Clostridium butyricum TO-A culture supernatant (CBCS) or butyric acid was added to a culture medium of human cervical carcinoma HeLa S3 cells, and changes in DNA-repair-related gene promoter activities were investigated. The HeLa S3 cells were transfected with a luciferase (Luc) expression vector containing approximately 500 bp of the 5′-upstream region of several human DNA-repair-related genes and cultured with a medium containing the CBCS (10%) or butyric acid (2.5 mM). The cells were harvested after 19 to 42 h of incubation. A Luc assay revealed that the human ATM, PARG, PARP1, and RB1 gene promoter activities were significantly increased. A Western blot analysis showed that the amounts of the proteins encoded by these genes markedly increased. Furthermore, 8, 24, and 48 h after the addition of the CBCS (10%), total RNA was extracted and subjected to RNAseq analysis. The results showed that the expression of several inflammation- and DNA-replication/repair-related genes, including NFKB and the MCM gene groups, decreased markedly after 8 h. However, the expression of the histone genes increased after 24 h. Elucidation of the mechanism by which the CBCS and butyrate affect the expression of genes that encode DNA-repair-associated proteins may contribute to the prevention of carcinogenesis, the risk of which rises in accordance with aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874bc855919805711ae53b74abe8374a883fe997" target='_blank'>
              Effect of Culture Supernatant of Clostridium butyricum TO-A on Human DNA-Repair-Factor-Encoding Gene Promoters
              </a>
            </td>
          <td>
            Shunsuke Takaoka, T. Ishii, Yuriko Umihara, Ryuji Otani, Sota Akazawa, Takahiro Oda, Yoko Ogino, Yoichi Okino, Dian-Sheng Wang, Fumiaki Uchiumi
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6345a45591e3efdf44b1aa93d340307a041b1c2a" target='_blank'>
              Synthetic rescue of Xeroderma Pigmentosum C phenotype via PIK3C3 downregulation
              </a>
            </td>
          <td>
            F. Kobaisi, E. Sulpice, A. Nasrallah, Patricia Obeïd, Hussein Fayyad-Kazan, Walid Rachidi, Xavier Gidrol
          </td>
          <td>2024-11-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="

 Introduction:
 Present in 30% of individuals with solid tumors, clonal hematopoiesis of indeterminate potential (CHIP) is increasingly recognized for its role in modifying the tumor immune microenvironment (TIME) and modulating immune responses. Given the significance of inflammation in lung cancer development, CHIP could influence prognosis of non-small cell lung cancer (NSCLC), and its subtypes, lung squamous cell carcinoma (LUSC) and lung adenocarcinoma (LUAD). This study explores the molecular and immunological changes associated with CHIP, aiming to provide insights into how CHIP influences tumor progression and immune response in NSCLC.
 Methods:
 Somatic mutations associated with CHIP were identified in peripheral blood whole exome sequences of NSCLC, LUSC and LUAD patients from the Clinical Proteomic Tumor Analysis Consortium (CPTAC), using established methods (PMID: 36652671). CHIP in tumor tissue (CHIP-Tum) was determined by detecting the same peripheral blood driver mutations in tumor DNA. RNA sequencing (RNA-seq) analysis of tumor samples was performed with kallisto for pseudo-alignment and quantification, and DESeq2 for normalization. Differentially expressed genes underwent gene set enrichment analysis (GSEA) of gene ontology (GO) terms. T cell exhaustion markers such as CTLA-4, TIGIT, and BTLA were examined. The MFeaST package was used for feature selection to distinguish CHIP status in lung cancer, using 5-fold cross-validation and the consensus of multiple algorithms. Selected genes underwent over-representation analysis using g:Profiler, and classifiers were trained to predict CHIP status. CIBERSORTx was used to assess immune cell composition, and HALO software was used to analyze cellular densities and ratios in H&E stained images.
 Results:
 CHIP prevalence was 18% (54/298) in NSCLC, 15% (16/108) in LUSC, and 20% (38/190) in LUAD. The total number of CHIP variants were 70 in NSCLC, 19 in LUSC and 51 in LUAD, with DNMT3A being the most common mutation, followed by TET2. RNA-seq analysis identified 101 genes in NSCLC, 59 genes in LUSC, and 38 genes in LUAD, that were differentially expressed by CHIP status (padj<0.05). While CHIP+ NSCLC and LUAD demonstrated significant pathway enrichments linked to tumor growth, CHIP+ LUSC patients revealed significant pathway enrichments involved in both positive and negative regulation of innate and adaptive immune responses. This suggests heterogenous and multidirectional immune dysregulation rather than solely hyperinflammation in the TIME. Similarly, selected discriminative genes by CHIP status in LUSC demonstrated an overrepresentation of immune processes, including antigen processing and presentation, interleukin and interferon signaling, T cell response, and response to bacterial and viral infection. Furthermore, CHIP-Tum+ LUSC demonstrated trending decreased expression of T cell exhaustion markers. Using selected discriminative genes in NSCLC and LUSC by CHIP status, trained classifiers demonstrated moderate accuracy (NSCLC: 71.4%, LUSC: 91.3%), moderate specificity (NSCLC: 76.5%, LUSC: 97.8%), low sensitivity (NSCLC: 48.1%, LUSC: 50.0%), and moderate area under the curve values (NSCLC: 0.677, LUSC: 0.856). In silico immunophenotyping of LUSC revealed enrichment of dendritic cells in DNMT3A-driven CHIP (p<0.05), and trending enrichment of CD8+ T cells seen with CHIP and DNMT3A-driven CHIP. In LUAD, there was an enrichment of regulatory T cells seen with CHIP (p<0.05), CHIP-Tum (p<0.05), and DNMT3A-driven CHIP (p<0.01). Preliminary HALO software analysis confirmed increased lymphocyte infiltration in CHIP+ or CHIP-Tum+ NSCLC.
 Conclusions:
 The role of CHIP in tumor progression and immune response appears to differ across lung cancer subtypes and CHIP drivers. In LUSC, patients with CHIP exhibited altered immune responses and decreased markers of T cell exhaustion, indicating potential sensitivity to immune checkpoint inhibitors. While optimizations are necessary to address class imbalances in classifier training, identifying genes that discriminate CHIP status could enhance detection methods and reveal new targets for immunotherapy. Future studies will focus on exploring the TIME of NSCLC and its subtypes using in situ histological imaging. CHIP is emerging as a key biomarker for understanding tumor progression and preparing for the next era of personalized cancer management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3b18e993dd28730f849b01c6e6f9cfb6b3a40d" target='_blank'>
              Transcriptomic Analysis of Clonal Hematopoiesis in Non-Small Cell Lung Carcinoma Unveils a Tumorigenic and Dysregulated Immune Profile
              </a>
            </td>
          <td>
            Casey K. Wong, M. Buttigieg, Jahanara Rajwani, Caitlin Lee, C. Vlasschaert, M. Rauh
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract LINE-1 (L1) is a parasitic retrotransposable DNA element, active in primates for the last 80–120 Myr. L1 has generated nearly one-third of the human genome by copying its transcripts, and those of other genetic elements (e.g. Alu and SVA), into genomic DNA by target site-primed reverse transcription (TPRT) and remains active in modern humans. L1 encodes two proteins that bind their encoding transcript (cis preference) to form an L1 ribonucleoprotein (RNP) that mediates retrotransposition. ORF2p provides reverse transcriptase and endonuclease activity. ORF1p, its major component, is a homo-trimeric phospho-protein that binds single-stranded nucleic acid (ssNA) with high affinity and exhibits nucleic acid (NA) chaperone activity. We used optical tweezers to examine ORF1p binding to individual single-stranded DNA (ssDNA) molecules and found that the arrangement of ORF1p on the ssDNA depends on their molar ratio. When the concentration of ORF1p is just sufficient to saturate the entire NA molecule, the nucleoprotein (NP) is compact and stable. However, additional ORF1p binds and destabilizes the compacted NP, allowing it to engage a second ssDNA. Our results suggest that ORF1p displaced from its RNA template during TPRT could bind and destabilize remaining downstream L1 RNP, making them susceptible to hijacking by non-L1 templates, and thereby enable retrotransposition of non-L1 transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65999fea4fdf64159b0557e5ff03c0a4195548b9" target='_blank'>
              L1-ORF1p nucleoprotein can rapidly assume distinct conformations and simultaneously bind more than one nucleic acid
              </a>
            </td>
          <td>
            Benjamin A. Cashen, M. N. Naufer, Michael Morse, Micah J McCauley, I. Rouzina, Charlie E. Jones, A. Furano, Mark C. Williams
          </td>
          <td>2024-11-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Karyoptosis is a type of regulated cell death (RCD) characterized by explosive nuclear rupture caused by a loss of nuclear membrane integrity, resulting in the release of genomic DNA and other nuclear components into the cytosol and extracellular environment. The mechanism underlying karyoptosis involves a delicate balance between the following forces: the expansion force exerted by the tightly packed DNA in the nucleus, the resistance provided by the nuclear lamina at the inner nuclear membrane (INM), and the tensile force from the cytoskeleton that helps position the nucleus at the center of the cytoplasm, allowing it to remain maximally expanded. In addition, CREB3, a type II integral membrane protein with DNA-binding ability, tethers chromatin to the INM, providing a tightening force through chromatin interactions that prevent nuclear membrane rupture. UVB radiation can trigger this process, inducing CREB3-FL cleavage and producing CREB3-CF. Therefore, UVB acts as an intrinsic factor in the induction of karyoptosis. Importantly, biochemical analysis of RCD markers shows that karyoptosis is distinct from other forms of cell death, such as apoptosis, autophagy, necroptosis, and pyroptosis. This review explores the mechanisms involved in maintaining nuclear membrane integrity and the role of CREB3 in triggering karyoptosis and provides brief suggestions on the potential implications for targeting cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9b81aa2650efbd4287e4521e56cb949db56f3f" target='_blank'>
              Karyoptosis as a novel type of UVB-induced regulated cell death.
              </a>
            </td>
          <td>
            Weidong Chen, Ji-Young Byun, Han Chang Kang, Hye Suk Lee, Joo Young Lee, Young Jik Kwon, Yong-Yeon Cho
          </td>
          <td>2024-12-03</td>
          <td>Free radical research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 The GATA1 transcription factor is crucial for the regulation of gene expression during the differentiation of hematopoietic progenitors to erythrocytic and megakaryocytic lineages. Mutations in GATA1 are pervasively found in myeloid leukemia of Down syndrome (ML-DS), a subtype of acute myeloid leukemia that is common in children with Down syndrome. These mutations typically occur in the N-terminal region of the GATA1 gene and lead to the production of a truncated isoform known as GATA1s, which retains the DNA-binding capacity. Expression of the truncated form GATA1s disrupts normal hematopoietic differentiation and leads to the over-production of megakaryocytic progenitors in the fetal liver. In Down syndrome children, this results in a non-lethal blood disorder known as transient abnormal myelopoiesis (TAM), which in many cases progresses to myeloid leukemia.
 Our study investigates the impact of the GATA1s isoform on megakaryopoiesis and its underlying molecular mechanisms. We use mouse hematopoietic progenitors and CRISPR-edited human HSPCs and perform transcriptomic and epigenomic analyses to understand how GATA1s differentially controls megakaryocytic gene expression. Our findings indicate that the expression of GATA1s promotes a premature increase of megakaryocyte-specific transcription already in hematopoietic stem cells. The upregulation of megakaryocyte markers is observed in HSCs from both fetal liver and adult bone marrow, though with varying effects across subpopulations. Moreover, the response to megakaryocyte-inducing acute stressors, such as interferon, differs between cells expressing GATA1 and those expressing GATA1s. Despite this early bias, GATA1s-expressing cells show incomplete differentiation into megakaryocytes in culture. ChIP-seq of normal and mutant GATA1 shows that despite both isoforms bind to a majority of shared binding sites, a fraction of these sites is isoform-specific. This is linked to changes in chromatin accessibility, which present different dynamics of gain and loss along megakaryocyte differentiation and which are associated to the number of GATA1 motifs on DNA. Furthermore, Hi-C analysis of global chromatin interactions shows changes in long-range genomic interactions associated to the deregulation of Mk-specific transcripts.
 In summary, our work illustrates how GATA1s disrupts normal megakaryopoiesis by inducing early lineage bias and impairing differentiation through remodeling of the chromatin landscape and gene expression. These findings enhance our understanding of GATA1s-related disorders, such as myeloid leukemia of Down syndrome, and may guide future therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23bb339eef4d240a4f9642c9df617f68b4213777" target='_blank'>
              Chromatin Remodeling By GATA1s Alters Megakaryocytic Gene Expression
              </a>
            </td>
          <td>
            Carini Picardi Morais de Castro, Gerard Martínez-Cebrián, Miguel Quijada-álamo, Grace Freed, Lucía Lorenzi, Sébastien Malinge, Elvin Wagenblast, Sergi Cuartero
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="To overcome genotoxicity, cells have evolved powerful and effective mechanisms to detect and respond to DNA lesions. RecQ Like Helicase-4 (RECQL4) plays a vital role in DNA damage responses. RECQL4 is recruited to DNA double-strand break (DSB) sites in a poly(ADP-ribosyl)ation (PARylation)-dependent manner, but the mechanism and significance of this process remain unclear. Here, we showed that the domain of RECQL4 recruited to DSBs in a PARylation-dependent manner directly interacts with poly(ADP-ribose) (PAR) and contains a PAR-binding motif (PBM). By replacing this PBM with a PBM of hnRNPA2 or its mutated form, we demonstrated that the PBM in RECQL4 is required for PARylation-dependent recruitment and the roles of RECQL4 in the DSB response. These results suggest that the direct interaction of RECQL4 with PAR is critical for proper cellular response to DSBs and provide insights to understand PARylation-dependent control of the DSB response and cancer therapeutics using PARylation inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d28fd83ec5597eeaffb6744f7a6080578ac3e86" target='_blank'>
              Interaction of RECQL4 with poly(ADP-ribose) is critical for the DNA double-strand break response in human cells.
              </a>
            </td>
          <td>
            Sunyoung Shin, Dongmin Kim, Hyemi Kim, Won-Ho Cho, Gyungmin Kim, Joon-Kyu Lee
          </td>
          <td>2024-10-27</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
 Esophageal squamous cell carcinoma (ESCC) is a malignant tumor with high morbidity and mortality, and easy to develop resistance to chemotherapeutic agents. Telomeres are DNA-protein complexes located at the termini of chromosomes in eukaryotic cells, which are unreplaceable in maintaining the stability and integrity of genome. Telomerase, an RNA-dependent DNA polymerase, play vital role in telomere length maintain, targeting telomerase is a promising therapeutic strategy for cancer.
 AIM
 To investigate the efficacy and underlying mechanisms of BIBR1532, a telomerase inhibitor, in ESCC.
 METHODS
 KYSE150 and KYSE410 cells were cultured and exposed to various concentrations of BIBR1532. Cell viability was assessed at 48 hours and 72 hours to determine the IC50 values. The effects of BIBR1532 on ESCC cell proliferation, migration, and cellular senescence were evaluated using the cell counting kit-8 assay, plate colony formation assay, scratch assay, transwell assay, and β-galactosidase staining, respectively. Western blotting was performed to detect the expression of proteins in BIBR1532-treated ESCC cells, such as human telomerase reverse transcriptase (hTERT), key molecules involved in DNA damage response (DDR) or cellular senescence, as well as telomere-binding proteins. Additionally, a tumor-bearing nude mouse model was established to evaluate the anti-cancer effect of BIBR1532 in vivo .
 RESULTS
 The IC50 values for KYSE150 and KYSE410 cells after 48 hours of BIBR1532 exposure were 48.53 μM and 39.59 μM, respectively. These values decreased to 37.22 μM and 22.71 μM, respectively, following a longer exposure of 72 hours. BIBR1532 exhibited dose-dependent effects on KYSE150 and KYSE410 cells, including decreased hTERT expression, inhibition of proliferation and metastasis, and induction of cellular senescence. Mechanistically, BIBR1532 upregulated the expression of the DDR protein, γ-H2AX, and activated the ataxia telangiectasia and Rad3-related protein (ATR)/ check point kinase 1 (CHK-1) and ataxia-telangiectasia mutated gene (ATM)/CHK2 pathways. BIBR1532 downregulated the expression of telomere-binding proteins, including telomeric-repeat binding factor 1 (TRF1), TRF2, protection of telomeres 1, and TIN2-interacting protein 1. In a nude mouse xenograft model, BIBR1532 significantly suppressed tumor growth, reduced hTERT expression, and increased γ-H2AX protein levels. Hematoxylin and eosin staining of various organs, including the heart, liver, spleen, lungs, and kidneys, revealed no apparent adverse effects.
 CONCLUSION
 BIBR1532 exerts anti-cancer effects on ESCC by inducing DDR through the ATR/CHK1 and ATM/CHK2 pathways and downregulating the expression of telomere-binding proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c9a7cb4627655be894378fe9cf7a95af0081d7" target='_blank'>
              BIBR1532 inhibits proliferation and metastasis of esophageal squamous cancer cells by inducing telomere dysregulation
              </a>
            </td>
          <td>
            Qin Wang, Qingrong Li, Lei Xu, Zichun Yuan, Xiao Liu, Mao-Ju Tang, Man Luo, Xiao-Wu Zhong, Qiang Ma, Xiao-Lan Guo
          </td>
          <td>2025-01-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is the most significant DNA modification present in mammalian genomes. Understanding the roles of 5mC in diverse biological processes requires quantitative detection at single-base resolution. In this study, we engineered the repressor of the silencing 1 (ROS1) protein derived from Arabidopsis thaliana to enhance its 5mC glycosylase/lyase activity, resulting in the creation of the engineered ROS1 (eROS1) protein. Leveraging the unique properties of eROS1, we introduced a method termed engineered ROS1 sequencing (eROS1-seq) for bisulfite-free and quantitative detection of 5mC in DNA at single-base resolution. In eROS1-seq, the eROS1 protein selectively cleaves 5mC while leaving unmodified cytosine (C) intact, followed by the incorporation of dTTP, which subsequently results in sequencing as thymine (T). This method effectively differentiates between C and 5mC. Unlike conventional bisulfite sequencing (BS-seq), which predominantly converts cytosines, eROS1-seq specifically transforms 5mC into T, thereby avoiding potential imbalances in the nucleobase composition of the sequencing library. Using eROS1-seq, we successfully achieved quantitative and site-specific detection of 5mC in the genomic DNA of lung cancer tissue. Overall, the eROS1-seq approach is bisulfite-free and straightforward, making it a valuable tool for the quantitative detection of 5mC at single-base resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f4d773df90996d7c3a70b0f0857f61a3499f252" target='_blank'>
              Bisulfite-Free and Quantitative Detection of DNA Methylation at Single-Base Resolution by eROS1-seq.
              </a>
            </td>
          <td>
            Fang-Yin Gang, Neng-Bin Xie, Min Wang, Shan Zhang, T. Ji, Wei Liu, Xia Guo, , Bi-Feng Yuan
          </td>
          <td>2024-12-16</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Long non-coding RNA Rhabdomyosarcoma 2-associated Transcript (RMST) is a crucial regulator in various biological processes, particularly in neurogenesis and cancer progression. This review summarizes current knowledge on structure, expression patterns, and functional roles across different organs and diseases of RMST. RMST exhibits tissue-specific expression, notably in brain tissues and vascular endothelial cells, and plays a significant role in neuronal differentiation through interaction with SRY-box 2 (SOX2). In cancer, RMST predominantly functions as a tumor suppressor, with context-dependent roles observed across different cancer types. RMST is also implicated in neurological disorders, cardiovascular diseases, and Hirschsprung's disease. Mechanistically, RMST acts as a competing endogenous RNA and a transcriptional regulator, interacting with various microRNAs and proteins to modulate gene expression. The potential of RMST as a biomarker and therapeutic target is increasingly recognized, particularly in atherosclerosis and cancer. While current findings are promising, further research is needed to fully elucidate the functions and translate these insights into clinical applications of RMST. This review underscores the significance of RMST in cellular processes and disease pathogenesis, highlighting its potential as a novel target for diagnostic and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274785be6ba4141ed74a6cfcddf7cc469ab67f46" target='_blank'>
              RMST: A long noncoding RNA involved in cancer and disease.
              </a>
            </td>
          <td>
            Hidenori Tani
          </td>
          <td>2024-11-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Suppression of single-stranded DNA (ssDNA) gap accumulation at replication forks has emerged as a potential determinant of chemosensitivity in homologous recombination (HR)-deficient tumors, as ssDNA gaps are transformed into cytotoxic double-stranded DNA breaks. We have previously shown that the histone chaperone CAF-1’s nucleosome deposition function is vital to preventing degradation of stalled replication forks correlating with HR-deficient cells’ response to genotoxic drugs. Here we report that the CAF-1–ASF1 pathway promotes ssDNA gap accumulation at replication forks in both wild-type and breast cancer (BRCA)-deficient backgrounds. We show that this is independent of CAF-1’s nucleosome deposition function but instead may rely on its proper localization to replication forks. Moreover, we show that the efficient localization to nascent DNA of PrimPol, the enzyme responsible for repriming upon replication stress, is dependent on CAF-1. As PrimPol has been shown to be responsible for generating ssDNA gaps as a byproduct of its repriming function, CAF-1’s role in its recruitment could directly impact ssDNA gap formation. We also show that chemoresistance observed in HR-deficient cells when CAF-1 or ASF1A are lost correlates with suppression of ssDNA gaps rather than protection of stalled replication forks. Overall, this work identifies an unexpected role of CAF-1 in regulating PrimPol recruitment and ssDNA gap generation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e923b166479ec807671490eea48162390e320f0b" target='_blank'>
              CAF-1 promotes efficient PrimPol recruitment to nascent DNA for single-stranded DNA gap formation
              </a>
            </td>
          <td>
            Joshua Straka, Jude B. Khatib, Lindsey M Pale, Claudia M. Nicolae, George-Lucian Moldovan
          </td>
          <td>2024-11-18</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Epigenetics which examines the regulation of genes without modification of the DNA sequence, plays a crucial role in various biological processes and disease mechanisms. Among the different forms of epigenetic modifications, histone post-translational modifications (PTMs) are important for modulating chromatin structure and gene expression. Aberrant levels of histone PTMs are implicated in a wide range of diseases, including cancer, making them promising targets for biomarker discovery and therapeutic intervention. In this context, blood, tissues, or cells serve as valuable resources for epigenetic research and analysis. Traditional methods such as mass spectrometry and western blotting are widely used to study histone PTMs, providing qualitative and (semi)quantitative information. However, these techniques often face limitations that could include throughput and scalability, particularly when applied to clinical samples. To overcome these challenges, we developed and validated 13 Nu.Q® immunoassays to detect and quantify specific histone PTM-nucleosomes from K2EDTA plasma samples. Then, we tested these assays on other types of samples, including chromatin extracts from frozen tissues, as well as cell lines and white blood cells Our findings demonstrate that the Nu.Q® assays offer high specificity, sensitivity, precision and linearity, making them effective tools for epigenetic profiling. A comparative analysis of HeLa cells using mass spectrometry, Western blot, and Nu.Q® immunoassays revealed a consistent histone PTMs signature, further validating the effectiveness of these assays. Additionally, we successfully applied Nu.Q® assays across various biological samples, including human tissues from different organs and specific white blood cell subtypes, highlighting their versatility and applicability in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea54269010715f8d7295e217ed70c9663ed44c1" target='_blank'>
              High-Throughput Epigenetic Profiling Immunoassays for Accelerated Disease Research and Clinical Development
              </a>
            </td>
          <td>
            P. Van den Ackerveken, Clotilde Hannart, Dorian Pamart, Robin Varsebroucq, Marion Wargnies, Olivia Thiry, Marie Lurkin, Séverine Vincent, Muriel Chapelier, Guillaume Rommelaere, M. Herzog
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [15, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>